<script src="https://code.jquery.com/jquery-3.7.0.js"></script>
<script src="https://cdn.datatables.net/1.13.6/js/jquery.dataTables.min.js"></script>
<script src="https://cdn.datatables.net/searchbuilder/1.6.0/js/dataTables.searchBuilder.min.js"></script>
<script src="https://cdn.datatables.net/datetime/1.5.1/js/dataTables.dateTime.min.js"></script>

<script src="https://cdn.datatables.net/responsive/2.5.0/js/dataTables.responsive.min.js"></script>

<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/responsive/2.2.3/css/responsive.dataTables.min.css">

<link rel="stylesheet" href="https://cdn.datatables.net/1.13.6/css/jquery.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/searchbuilder/1.6.0/css/searchBuilder.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/datetime/1.5.1/css/dataTables.dateTime.min.css">

<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" />
<script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>

<style type="text/css">
  .tooltip {
    font-family: Georgia;
    font-size: 18px;
  }
  .tooltip .tooltip-inner {
    background-color: #00FFFF;
    color: #000000;
    min-width: 400px;
  }
  </style>

<script>
    $(document).ready(function() {
    $('#dataframe').DataTable( {
        "sScrollX": "100%",
        "dom": "Qlfrtip",

        responsive: {
            details: {
                type: 'column'
            }
        },
        "bAutoWidth": false,
        columnDefs: [ {
            className: 'control',
            orderable: false,
            targets:   [ 0 ],
            width: 100,
        },
        { type: 'natural', targets: 1 } // allow normal sorting
        ],

    });
});
</script>

<div class="display nowrap" style="width:80%" width="100%">

<table class="dataframe display nowrap" id="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th>Expand</th>
      <th>Galaxy wrapper id</th>
      <th>Galaxy wrapper version</th>
      <th>Conda version</th>
      <th>Conda id</th>
      <th>Status</th>
      <th>bio.tool id</th>
      <th>bio.tool name</th>
      <th>EDAM operation</th>
      <th>EDAM topic</th>
      <th>Description</th>
      <th>bio.tool description</th>
      <th>Status</th>
      <th>Source</th>
      <th>ToolShed categories</th>
      <th>ToolShed id</th>
      <th>Galaxy wrapper owner</th>
      <th>Galaxy wrapper source</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td></td>
      <td>TreeBest</td>
      <td>1.9.2.post0</td>
      <td>1.9.2.post1</td>
      <td>treebest</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>TreeBeST best</td>
      <td></td>
      <td>To update</td>
      <td>http://treesoft.sourceforge.net/treebest.shtml</td>
      <td>Phylogenetics</td>
      <td>treebest_best</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/TreeBest</td>
    </tr>
    <tr>
      <td></td>
      <td>ete</td>
      <td>3.1.2</td>
      <td>3.1.1</td>
      <td>ete3</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Analyse phylogenetic trees using the ETE Toolkit</td>
      <td></td>
      <td>To update</td>
      <td>http://etetoolkit.org/</td>
      <td>Phylogenetics</td>
      <td>ete</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/ete</td>
    </tr>
    <tr>
      <td></td>
      <td>lotus2</td>
      <td>2.32</td>
      <td>2.32</td>
      <td>lotus2</td>
      <td>Up-to-date</td>
      <td>lotus2</td>
      <td>lotus2</td>
      <td>Sequence feature detection</td>
      <td>Metagenomics</td>
      <td>LotuS2 OTU processing pipeline</td>
      <td>LotuS2 is a lightweight and user-friendly pipeline that is fast, precise, and streamlined, using extensive pre- and post-ASV/OTU clustering steps to further increase data quality. High data usage rates and reliability enable high-throughput microbiome analysis in minutes.</td>
      <td>Up-to-date</td>
      <td>http://lotus2.earlham.ac.uk/</td>
      <td>Metagenomics</td>
      <td>lotus2</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/lotus2</td>
    </tr>
    <tr>
      <td></td>
      <td>abacas</td>
      <td>1.1</td>
      <td>3.23</td>
      <td>mummer</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Order and Orientate Contigs</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/abacas</td>
      <td>Assembly</td>
      <td>abacas</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/abacas</td>
    </tr>
    <tr>
      <td></td>
      <td>assemblystats</td>
      <td>1.1.0</td>
      <td>1.7.8</td>
      <td>perl-bioperl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Summarise an assembly (e.g. N50 metrics)</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>Assembly</td>
      <td>assemblystats</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
    </tr>
    <tr>
      <td></td>
      <td>biohansel</td>
      <td>2.4.0</td>
      <td>2.6.1</td>
      <td>bio_hansel</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Heidelberg and Enteritidis SNP Elucidation</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/biohansel</td>
      <td>Sequence Analysis</td>
      <td>biohansel</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/biohansel</td>
    </tr>
    <tr>
      <td></td>
      <td>combine_assembly_stats</td>
      <td>1.0</td>
      <td>2.54</td>
      <td>perl-getopt-long</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Combine multiple Assemblystats datasets into a single tabular report</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>Assembly</td>
      <td>combine_assemblystats</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
    </tr>
    <tr>
      <td></td>
      <td>cryptogenotyper</td>
      <td>1.0</td>
      <td>1.0</td>
      <td>cryptogenotyper</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>CryptoGenotyper is a standalone tool to *in-silico*  determine species and subtype based on SSU rRNA and gp60 markers.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/phac-nml/CryptoGenotyper</td>
      <td>Sequence Analysis</td>
      <td>cryptogenotyper</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/CryptoGenotyper</td>
    </tr>
    <tr>
      <td></td>
      <td>ectyper</td>
      <td>1.0.0</td>
      <td>1.0.0</td>
      <td>ectyper</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>EC-Typer - in silico serotyping of Escherichia coli species</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/phac-nml/ecoli_serotyping</td>
      <td>Sequence Analysis</td>
      <td>ectyper</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/ecoli_serotyping</td>
    </tr>
    <tr>
      <td></td>
      <td>filter_spades_repeats</td>
      <td>1.0.1</td>
      <td>1.7.8</td>
      <td>perl-bioperl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Remove short and repeat contigs/scaffolds</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/galaxy_tools/</td>
      <td>Assembly</td>
      <td>filter_spades_repeats</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools/</td>
    </tr>
    <tr>
      <td></td>
      <td>getmlst</td>
      <td>0.1.4.1</td>
      <td>0.2.0</td>
      <td>srst2</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Download MLST datasets by species from pubmlst.org</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>getmlst</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>hivtrace</td>
      <td>1.0.1</td>
      <td>1.5.0</td>
      <td>hivtrace</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>An application that identifies potential transmission clusters within a supplied FASTA file with an option to find potential links against the Los Alamos HIV Sequence Database.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>hivtrace</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/tools/hivtrace</td>
    </tr>
    <tr>
      <td></td>
      <td>kat_filter</td>
      <td>2.3</td>
      <td>2.4.2</td>
      <td>kat</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filtering kmers or reads from a database of kmers hashes</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>kat_filter</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>mob_suite</td>
      <td>3.0.3</td>
      <td>3.1.8</td>
      <td>mob_suite</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MOB-suite is a set of software tools for clustering, reconstruction and typing of plasmids from draft assemblies</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/mob-suite</td>
      <td>Sequence Analysis</td>
      <td>mob_suite</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/mob-suite</td>
    </tr>
    <tr>
      <td></td>
      <td>mrbayes</td>
      <td>1.0.2</td>
      <td>3.2.7</td>
      <td>mrbayes</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A program for the Bayesian estimation of phylogeny.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>mrbayes</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>mykrobe_parser</td>
      <td>0.1.4.1</td>
      <td></td>
      <td>r-base</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>RScript to parse the results of mykrobe predictor.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/mykrobe-parser</td>
      <td>Sequence Analysis</td>
      <td>mykrobe_parser</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/mykrobe-parser</td>
    </tr>
    <tr>
      <td></td>
      <td>plasmidspades</td>
      <td>1.1</td>
      <td>3.15.5</td>
      <td>spades</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Genome assembler for assemblying plasmid</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Assembly</td>
      <td>plasmidspades</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>promer</td>
      <td>1.2</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Aligns two sets of contigs and reports amino acid substitutions between them</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/promer</td>
      <td>Assembly</td>
      <td>promer</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/promer</td>
    </tr>
    <tr>
      <td></td>
      <td>sistr_cmd</td>
      <td>1.1.1</td>
      <td>1.1.1</td>
      <td>sistr_cmd</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SISTR in silico serotyping tool</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/phac-nml/sistr_cmd</td>
      <td>Sequence Analysis</td>
      <td>sistr_cmd</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>smalt</td>
      <td>0.7.6</td>
      <td>0.7.6</td>
      <td>smalt</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SMALT aligns DNA sequencing reads with a reference genome.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://www.sanger.ac.uk/science/tools/smalt-0</td>
      <td>Sequence Analysis</td>
      <td>smalt</td>
      <td>nml</td>
      <td>https://sourceforge.net/projects/smalt/</td>
    </tr>
    <tr>
      <td></td>
      <td>srst2</td>
      <td>0.3.7</td>
      <td>0.2.0</td>
      <td>srst2</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Short Read Sequence Typing for Bacterial Pathogens</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>srst2</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>staramr</td>
      <td>0.10.0</td>
      <td>0.10.0</td>
      <td>staramr</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Scan genome contigs against the ResFinder, PlasmidFinder, and PointFinder antimicrobial resistance databases.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/phac-nml/staramr</td>
      <td>Sequence Analysis</td>
      <td>staramr</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/staramr</td>
    </tr>
    <tr>
      <td></td>
      <td>stringmlst</td>
      <td>1.1.0</td>
      <td>0.6.3</td>
      <td>stringMLST</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Rapid and accurate identification of the sequence type (ST)</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>stringmlst</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>repeatexplorer2</td>
      <td>2.3.8</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tool for annotation of repeats from unassembled shotgun reads.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/repeatexplorer/repex_tarean</td>
      <td>Genome annotation</td>
      <td>repeatexplorer2</td>
      <td>gga</td>
      <td>https://github.com/galaxy-genome-annotation/galaxy-tools/tree/master/tools/repeatexplorer2</td>
    </tr>
    <tr>
      <td></td>
      <td>cd_hit_dup</td>
      <td>0.0.1</td>
      <td>4.8.1</td>
      <td>cd-hit-auxtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>simple tool for removing duplicates from sequencing reads</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Metagenomics, Sequence Analysis</td>
      <td>cd_hit_dup</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/cd_hit_dup</td>
    </tr>
    <tr>
      <td></td>
      <td>multispecies_orthologous_microsats</td>
      <td>1.0.0</td>
      <td></td>
      <td>bx-sputnik</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Extract orthologous microsatellites</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>multispecies_orthologous_microsats</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/multispecies_orthologous_microsats</td>
    </tr>
    <tr>
      <td></td>
      <td>quality_filter</td>
      <td>1.0.1</td>
      <td>0.10.0</td>
      <td>bx-python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter nucleotides based on quality scores</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>quality_filter</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/quality_filter</td>
    </tr>
    <tr>
      <td></td>
      <td>gi2taxonomy</td>
      <td>1.1.1</td>
      <td>0.10.0</td>
      <td>taxonomy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Fetch taxonomic representation</td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>Metagenomics</td>
      <td>gi2taxonomy</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/taxonomy/gi2taxonomy</td>
    </tr>
    <tr>
      <td></td>
      <td>kraken2tax</td>
      <td>1.2+galaxy0</td>
      <td></td>
      <td>gawk</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert Kraken output to Galaxy taxonomy data.</td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>Metagenomics</td>
      <td>kraken2tax</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/blob/master/tool_collections/taxonomy/kraken2tax/</td>
    </tr>
    <tr>
      <td></td>
      <td>lca_wrapper</td>
      <td>1.0.1</td>
      <td>0.10.0</td>
      <td>taxonomy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Find lowest diagnostic rank</td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>Metagenomics</td>
      <td>lca_wrapper</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/taxonomy/lca_wrapper</td>
    </tr>
    <tr>
      <td></td>
      <td>t2ps</td>
      <td>1.0.0</td>
      <td>0.10.0</td>
      <td>taxonomy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Draw phylogeny</td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>Metagenomics</td>
      <td>t2ps</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/taxonomy/t2ps</td>
    </tr>
    <tr>
      <td></td>
      <td>t2t_report</td>
      <td>1.0.0</td>
      <td>0.10.0</td>
      <td>taxonomy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Summarize taxonomy</td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>Metagenomics</td>
      <td>t2t_report</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/taxonomy/t2t_report</td>
    </tr>
    <tr>
      <td></td>
      <td>antismash</td>
      <td>6.1.1</td>
      <td>6.1.1</td>
      <td>antismash</td>
      <td>Up-to-date</td>
      <td>antismash</td>
      <td>antiSMASH</td>
      <td>Sequence clustering, Gene prediction, Differential gene expression analysis</td>
      <td>Molecular interactions, pathways and networks, Gene and protein families</td>
      <td>Antismash allows the genome-wide identification, annotation and analysis of secondary metabolite biosynthesis gene clusters</td>
      <td>Rapid genome-wide identification, annotation and analysis of secondary metabolite biosynthesis gene clusters in bacterial and fungal genomes. It integrates and cross-links with a large number of in silico secondary metabolite analysis tools that have been published earlier.</td>
      <td>Up-to-date</td>
      <td>https://antismash.secondarymetabolites.org</td>
      <td>Sequence Analysis</td>
      <td>antismash</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/antismash</td>
    </tr>
    <tr>
      <td></td>
      <td>combine_metaphlan_humann</td>
      <td>0.3.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td>combine_metaphlan_and_humann</td>
      <td>Combine Metaphlan and HUMAnN</td>
      <td>Aggregation</td>
      <td>Metagenomics, Molecular interactions, pathways and networks</td>
      <td>Combine MetaPhlAn2 and HUMAnN2 outputs to relate genus/species abundances and gene families/pathways abundances</td>
      <td>"This tool combine MetaPhlAn outputs and HUMANnN outputs." - Galaxy tool wrapper</td>
      <td>To update</td>
      <td></td>
      <td>Metagenomics</td>
      <td>combine_metaphlan2_humann2</td>
      <td>bebatut</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/combine_metaphlan2_humann2</td>
    </tr>
    <tr>
      <td></td>
      <td>compare_humann2_output</td>
      <td>0.2.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>compare_humann2_outputs</td>
      <td>Compare HUMAnN2 outputs</td>
      <td>Comparison</td>
      <td>Metagenomics, Gene and protein families</td>
      <td>Compare outputs of HUMAnN2 for several samples and extract similar and specific information</td>
      <td>"This tool compare HUMANnN2 outputs with gene families or pathways and their relative abundances between several samples." - Galaxy tool wrapper</td>
      <td>To update</td>
      <td></td>
      <td>Metagenomics</td>
      <td>compare_humann2_output</td>
      <td>bebatut</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/compare_humann2_output</td>
    </tr>
    <tr>
      <td></td>
      <td>flye</td>
      <td>2.9.1</td>
      <td>2.9.3</td>
      <td>flye</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Assembly of long and error-prone reads.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/fenderglass/Flye/</td>
      <td>Assembly</td>
      <td>flye</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/flye</td>
    </tr>
    <tr>
      <td></td>
      <td>format_metaphlan2_output</td>
      <td>0.2.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>format_metaphlan2_output</td>
      <td>Format metaphlan2 output</td>
      <td>Formatting</td>
      <td>Taxonomy, Metagenomics</td>
      <td>Format MetaPhlAn2 output to extract abundance at different taxonomic levels</td>
      <td>"This tool format output file of MetaPhlan2 containing community content (abundance) at all taxonomic levels (from kingdom to strains)." - Galaxy tool wrapper</td>
      <td>To update</td>
      <td></td>
      <td>Metagenomics</td>
      <td>format_metaphlan2_output</td>
      <td>bebatut</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/format_metaphlan2_output/</td>
    </tr>
    <tr>
      <td></td>
      <td>glimmer_hmm</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>GlimmerHMM is a new gene finder based on a Generalized Hidden Markov Model (GHMM)</td>
      <td></td>
      <td>To update</td>
      <td>https://ccb.jhu.edu/software/glimmerhmm/</td>
      <td>Sequence Analysis</td>
      <td>glimmer_hmm</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/glimmer_hmm</td>
    </tr>
    <tr>
      <td></td>
      <td>graphmap</td>
      <td>0.5.2</td>
      <td>0.6.3</td>
      <td>graphmap</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Mapper for long, error-prone reads.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/isovic/graphmap/</td>
      <td>Assembly</td>
      <td>graphmap</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/graphmap</td>
    </tr>
    <tr>
      <td></td>
      <td>iprscan5</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Interproscan queries the interpro database and provides annotations.</td>
      <td></td>
      <td>To update</td>
      <td>http://www.ebi.ac.uk/Tools/pfa/iprscan5/</td>
      <td>Sequence Analysis</td>
      <td>iprscan5</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/iprscan5</td>
    </tr>
    <tr>
      <td></td>
      <td>itsx</td>
      <td>1.1.3</td>
      <td>1.1.3</td>
      <td>itsx</td>
      <td>Up-to-date</td>
      <td>ITSx</td>
      <td>ITSx</td>
      <td>Sequence feature detection</td>
      <td>Functional, regulatory and non-coding RNA, Microbiology</td>
      <td>ITSx is an open source software utility to extract the highly variable ITS1 and ITS2 subregions from ITS sequences.</td>
      <td>TSx is an open source software utility to extract the highly variable ITS1 and ITS2 subregions from ITS sequences, which is commonly used as a molecular barcode for e.g. fungi. As the inclusion of parts of the neighbouring, very conserved, ribosomal genes (SSU, 5S and LSU rRNA sequences) in the sequence identification process can lead to severely misleading results, ITSx identifies and extracts only the ITS regions themselves.</td>
      <td>Up-to-date</td>
      <td>https://microbiology.se/software/itsx/</td>
      <td>Metagenomics</td>
      <td>itsx</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/itsx</td>
    </tr>
    <tr>
      <td></td>
      <td>lighter</td>
      <td>1.0</td>
      <td>1.1.2</td>
      <td>lighter</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Lighter is a kmer-based error correction method for whole genome sequencing data</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Fasta Manipulation</td>
      <td>lighter</td>
      <td>bgruening</td>
      <td>https://github.com/mourisl/Lighter</td>
    </tr>
    <tr>
      <td></td>
      <td>mafft</td>
      <td>7.508</td>
      <td>7.520</td>
      <td>mafft</td>
      <td>To update</td>
      <td>MAFFT</td>
      <td>MAFFT</td>
      <td>Multiple sequence alignment</td>
      <td>Sequence analysis</td>
      <td>Multiple alignment program for amino acid or nucleotide sequences</td>
      <td>MAFFT (Multiple Alignment using Fast Fourier Transform) is a high speed multiple sequence alignment program.</td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>mafft</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/mafft</td>
    </tr>
    <tr>
      <td></td>
      <td>mcl</td>
      <td>14.137</td>
      <td>22.282</td>
      <td>mcl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Markov Cluster Algorithm</td>
      <td></td>
      <td>To update</td>
      <td>http://micans.org/mcl/</td>
      <td>Sequence Analysis, Metagenomics</td>
      <td>mcl</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/mcl</td>
    </tr>
    <tr>
      <td></td>
      <td>minipolish</td>
      <td>0.1.3</td>
      <td>0.1.3</td>
      <td>minipolish</td>
      <td>Up-to-date</td>
      <td>minipolish</td>
      <td>minipolish</td>
      <td>Localised reassembly, Read depth analysis</td>
      <td>Sequence assembly, Sequencing</td>
      <td>Polishing miniasm assemblies</td>
      <td>A tool that bridges the output of miniasm (long-read assembly) and racon (assembly polishing) together to polish a draft assembly. It also provides read depth information in contigs.</td>
      <td>Up-to-date</td>
      <td>https://github.com/rrwick/Minipolish</td>
      <td>Sequence Analysis</td>
      <td>minipolish</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/minipolish</td>
    </tr>
    <tr>
      <td></td>
      <td>nextdenovo</td>
      <td>2.5.0</td>
      <td>2.5.2</td>
      <td>nextdenovo</td>
      <td>To update</td>
      <td>nextdenovo</td>
      <td>NextDenovo</td>
      <td>De-novo assembly, Genome assembly</td>
      <td>Sequencing, Sequence assembly</td>
      <td>String graph-based de novo assembler for long reads</td>
      <td>NextDenovo is a string graph-based de novo assembler for long reads (CLR, HiFi and ONT). It uses a "correct-then-assemble" strategy similar to canu (no correction step for PacBio Hifi reads), but requires significantly less computing resources and storages.</td>
      <td>To update</td>
      <td>https://github.com/Nextomics/NextDenovo</td>
      <td>Assembly</td>
      <td>nextdenovo</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/nextdenovo</td>
    </tr>
    <tr>
      <td></td>
      <td>nucleosome_prediction</td>
      <td>3.0</td>
      <td>3.0</td>
      <td>nucleosome_prediction</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Prediction of Nucleosomes Positions on the Genome</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://genie.weizmann.ac.il/software/nucleo_exe.html</td>
      <td>Sequence Analysis</td>
      <td>nucleosome_prediction</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/nucleosome_prediction</td>
    </tr>
    <tr>
      <td></td>
      <td>pfamscan</td>
      <td>1.6</td>
      <td>1.6</td>
      <td>pfam_scan</td>
      <td>Up-to-date</td>
      <td>pfamscan</td>
      <td>PfamScan</td>
      <td>Protein sequence analysis</td>
      <td>Sequence analysis</td>
      <td>Search a FASTA sequence against a library of Pfam HMM.</td>
      <td>This tool is used to search a FASTA sequence against a library of Pfam HMM.</td>
      <td>Up-to-date</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/Pfam/Tools/</td>
      <td>Sequence Analysis</td>
      <td>pfamscan</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/pfamscan</td>
    </tr>
    <tr>
      <td></td>
      <td>racon</td>
      <td>1.5.0</td>
      <td>1.5.0</td>
      <td>racon</td>
      <td>Up-to-date</td>
      <td>Racon</td>
      <td>Racon</td>
      <td>Genome assembly, Mapping assembly, Sequence trimming</td>
      <td>Whole genome sequencing, Sequence assembly, Plant biology</td>
      <td>Consensus module for raw de novo DNA assembly of long uncorrected reads.</td>
      <td>The Possibility to Use Oxford Nanopore Technology | Ultrafast consensus module for raw de novo genome assembly of long uncorrected reads. http://genome.cshlp.org/content/early/2017/01/18/gr.214270.116 Note: This was the original repository which will no longer be officially maintained. Please use the new official repository here: https://github.com/isovic/racon| Racon is intended as a standalone consensus module to correct raw contigs generated by rapid assembly methods which do not include a consensus step | Consensus module for raw de novo DNA assembly of long uncorrected reads</td>
      <td>Up-to-date</td>
      <td>https://github.com/isovic/racon</td>
      <td>Sequence Analysis</td>
      <td>racon</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/racon</td>
    </tr>
    <tr>
      <td></td>
      <td>infernal</td>
      <td>1.1.4</td>
      <td>1.1.4</td>
      <td>infernal</td>
      <td>Up-to-date</td>
      <td>infernal</td>
      <td>Infernal</td>
      <td>Nucleic acid feature detection</td>
      <td>Sequence sites, features and motifs</td>
      <td>Infernal ("INFERence of RNA ALignment") is for searching DNA sequence databases for RNA structure and sequence similarities.</td>
      <td>Infernal ("INFERence of RNA ALignment") is for searching DNA sequence databases for RNA structure and sequence similarities. It is an implementation of a special case of profile stochastic context-free grammars called covariance models (CMs). A CM is like a sequence profile, but it scores a combination of sequence consensus and RNA secondary structure consensus, so in many cases, it is more capable of identifying RNA homologs that conserve their secondary structure more than their primary sequence.</td>
      <td>Up-to-date</td>
      <td>http://infernal.janelia.org/</td>
      <td>RNA</td>
      <td>infernal</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/infernal</td>
    </tr>
    <tr>
      <td></td>
      <td>rRNA</td>
      <td>0.1</td>
      <td></td>
      <td>hmmsearch3.0</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Identification of ribosomal RNA genes in metagenomic fragments.</td>
      <td></td>
      <td>To update</td>
      <td>http://weizhong-lab.ucsd.edu/meta_rna/</td>
      <td>RNA</td>
      <td>rrna</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rRNA</td>
    </tr>
    <tr>
      <td></td>
      <td>rbpbench</td>
      <td>0.8.1</td>
      <td>0.8.1</td>
      <td>rbpbench</td>
      <td>Up-to-date</td>
      <td>rbpbench</td>
      <td>RBPBench</td>
      <td></td>
      <td>RNA, Protein interactions, RNA immunoprecipitation, Bioinformatics, Sequence analysis</td>
      <td>Evaluate CLIP-seq and other genomic region data using a comprehensive collection of RBP binding motifs</td>
      <td>Evaluate CLIP-seq and other genomic region data using a comprehensive collection of RBP binding motifs</td>
      <td>Up-to-date</td>
      <td>https://github.com/michauhl/RBPBench</td>
      <td>Sequence Analysis, RNA, CLIP-seq</td>
      <td>rbpbench</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/rbpbench</td>
    </tr>
    <tr>
      <td></td>
      <td>reago</td>
      <td>1.1</td>
      <td>1.1</td>
      <td>reago</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Reago is tool to assembly 16S ribosomal RNA recovery from metagenomic data.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/chengyuan/reago-1.1</td>
      <td>Metagenomics, RNA</td>
      <td>reago</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/reago</td>
    </tr>
    <tr>
      <td></td>
      <td>sortmerna</td>
      <td>4.3.6</td>
      <td>4.3.6</td>
      <td>sortmerna</td>
      <td>Up-to-date</td>
      <td>sortmerna</td>
      <td>SortMeRNA</td>
      <td>Sequence similarity search, Sequence comparison, Sequence alignment analysis</td>
      <td>Metatranscriptomics, Metagenomics</td>
      <td>SortMeRNA is a software designed to rapidly filter ribosomal RNA fragments from metatransriptomic data produced by next-generation sequencers.</td>
      <td>Sequence analysis tool for filtering, mapping and OTU-picking NGS reads.</td>
      <td>Up-to-date</td>
      <td>http://bioinfo.lifl.fr/RNA/sortmerna/</td>
      <td>RNA</td>
      <td>sortmerna</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/sortmerna</td>
    </tr>
    <tr>
      <td></td>
      <td>salmon</td>
      <td>1.10.1</td>
      <td>1.10.2</td>
      <td>salmon</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Salmon is a wicked-fast program to produce a highly-accurate, transcript-level quantification estimates from RNA-seq and single-cell data.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/COMBINE-lab/salmon</td>
      <td>Sequence Analysis, RNA, Transcriptomics</td>
      <td></td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/salmon</td>
    </tr>
    <tr>
      <td></td>
      <td>trim_galore</td>
      <td>0.6.7</td>
      <td>0.6.10</td>
      <td>trim-galore</td>
      <td>To update</td>
      <td>trim_galore</td>
      <td>Trim Galore</td>
      <td>Sequence trimming</td>
      <td>Sequence analysis</td>
      <td>Trim Galore adaptive quality and adapter trimmer</td>
      <td>A wrapper tool around Cutadapt and FastQC to consistently apply quality and adapter trimming to FastQ files, with some extra functionality for MspI-digested RRBS-type (Reduced Representation Bisufite-Seq) libraries.</td>
      <td>To update</td>
      <td>http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/</td>
      <td>Sequence Analysis, Fastq Manipulation</td>
      <td>trim_galore</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/trim_galore</td>
    </tr>
    <tr>
      <td></td>
      <td>wtdbg</td>
      <td>2.5</td>
      <td>2.5</td>
      <td>wtdbg</td>
      <td>Up-to-date</td>
      <td>wtdbg2</td>
      <td>wtdbg2</td>
      <td>Genome assembly, De-novo assembly</td>
      <td>Sequence assembly, Sequencing</td>
      <td>WTDBG is a fuzzy Bruijn graph (FBG) approach to long noisy reads assembly.</td>
      <td>Wtdbg2 is a de novo sequence assembler for long noisy reads produced by PacBio or Oxford Nanopore Technologies (ONT). It assembles raw reads without error correction and then builds the consensus from intermediate assembly output. Wtdbg2 is able to assemble the human and even the 32Gb Axolotl genome at a speed tens of times faster than CANU and FALCON while producing contigs of comparable base accuracy.</td>
      <td>Up-to-date</td>
      <td>https://github.com/ruanjue/wtdbg2</td>
      <td>Assembly</td>
      <td>wtdbg</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/wtdbg</td>
    </tr>
    <tr>
      <td></td>
      <td>clinod</td>
      <td>0.1.0</td>
      <td>1.3</td>
      <td>clinod</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>NoD: a Nucleolar localization sequence detector for eukaryotic and viral proteins</td>
      <td></td>
      <td>To update</td>
      <td>http://www.compbio.dundee.ac.uk/www-nod/</td>
      <td>Sequence Analysis</td>
      <td>clinod</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/clinod</td>
    </tr>
    <tr>
      <td></td>
      <td>effectiveT3</td>
      <td>0.0.20</td>
      <td>1.0.1</td>
      <td>effectiveT3</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Find bacterial type III effectors in protein sequences</td>
      <td></td>
      <td>To update</td>
      <td>http://effectors.org</td>
      <td>Sequence Analysis</td>
      <td>effectivet3</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/effectiveT3</td>
    </tr>
    <tr>
      <td></td>
      <td>abricate</td>
      <td>1.0.1</td>
      <td>1.0.1</td>
      <td>abricate</td>
      <td>Up-to-date</td>
      <td>ABRicate</td>
      <td>ABRicate</td>
      <td>Antimicrobial resistance prediction</td>
      <td>Genomics, Microbiology</td>
      <td>Mass screening of contigs for antiobiotic resistance genes</td>
      <td>Mass screening of contigs for antimicrobial resistance or virulence genes.</td>
      <td>Up-to-date</td>
      <td>https://github.com/tseemann/abricate</td>
      <td>Sequence Analysis</td>
      <td>abricate</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/abricate/</td>
    </tr>
    <tr>
      <td></td>
      <td>abritamr</td>
      <td>1.0.14</td>
      <td>1.0.14</td>
      <td>abritamr</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A pipeline for running AMRfinderPlus and collating results into functional classes</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://zenodo.org/record/7370628</td>
      <td>Sequence Analysis</td>
      <td>abritamr</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/abritamr</td>
    </tr>
    <tr>
      <td></td>
      <td>abyss</td>
      <td>2.3.6</td>
      <td>2.3.7</td>
      <td>abyss</td>
      <td>To update</td>
      <td>abyss</td>
      <td>ABySS</td>
      <td>Genome assembly, De-novo assembly, Scaffolding</td>
      <td>Sequence assembly</td>
      <td>Assembly By Short Sequences - a de novo, parallel, paired-end sequence assembler</td>
      <td>De novo genome sequence assembler using short reads.</td>
      <td>To update</td>
      <td>http://www.bcgsc.ca/platform/bioinfo/software/abyss</td>
      <td>Assembly</td>
      <td>abyss</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/abyss</td>
    </tr>
    <tr>
      <td></td>
      <td>aldex2</td>
      <td>1.26.0</td>
      <td>1.34.0</td>
      <td>bioconductor-aldex2</td>
      <td>To update</td>
      <td>aldex2</td>
      <td>ALDEx2</td>
      <td>Statistical inference</td>
      <td>Gene expression, Statistics and probability</td>
      <td>Performs analysis Of differential abundance taking sample variation into account</td>
      <td>A differential abundance analysis for the comparison of two or more conditions. It uses a Dirichlet-multinomial model to infer abundance from counts, that has been optimized for three or more experimental replicates. Infers sampling variation and calculates the expected FDR given the biological and sampling variation using the Wilcox rank test and Welches t-test, or the glm and Kruskal Wallis tests. Reports both P and fdr values calculated by the Benjamini Hochberg correction.</td>
      <td>To update</td>
      <td>https://github.com/ggloor/ALDEx_bioc</td>
      <td>Metagenomics</td>
      <td>aldex2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/aldex2</td>
    </tr>
    <tr>
      <td></td>
      <td>amplican</td>
      <td>1.14.0</td>
      <td>1.24.0</td>
      <td>bioconductor-amplican</td>
      <td>To update</td>
      <td>amplican</td>
      <td>amplican</td>
      <td>Alignment, Standardisation and normalisation</td>
      <td>PCR experiment, Statistics and probability</td>
      <td>AmpliCan is an analysis tool for genome editing.</td>
      <td>It performs alignment of the amplicon reads, normalizes gathered data, calculates multiple statistics (e.g. cut rates, frameshifts) and presents results in form of aggregated reports. Data and statistics can be broken down by experiments, barcodes, user defined groups, guides and amplicons allowing for quick identification of potential problems.</td>
      <td>To update</td>
      <td>https://github.com/valenlab/amplican</td>
      <td>Sequence Analysis</td>
      <td>amplican</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/amplican</td>
    </tr>
    <tr>
      <td></td>
      <td>amrfinderplus</td>
      <td>3.11.26</td>
      <td>3.12.8</td>
      <td>ncbi-amrfinderplus</td>
      <td>To update</td>
      <td>amrfinderplus</td>
      <td>AMRFinderPlus</td>
      <td>Antimicrobial resistance prediction</td>
      <td>Microbiology, Public health and epidemiology, Infectious disease</td>
      <td>"AMRFinderPlus is designed to find acquired antimicrobial resistance genes and point mutations in protein and/or assembled nucleotide sequences.It can also search "plus", stress, heat, and biocide resistance and virulence factors for some organisms.</td>
      <td>AMRFinderPlus is designed to find acquired antimicrobial resistance genes and point mutations in protein and/or assembled nucleotide sequences.It can also search "plus", stress, heat, and biocide resistance and virulence factors for some organisms</td>
      <td>To update</td>
      <td>https://github.com/ncbi/amr</td>
      <td>Sequence Analysis</td>
      <td>AMRFinderPlus</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/amrfinderplus</td>
    </tr>
    <tr>
      <td></td>
      <td>ancombc</td>
      <td>1.4.0</td>
      <td>2.4.0</td>
      <td>bioconductor-ancombc</td>
      <td>To update</td>
      <td>ancombc</td>
      <td>ANCOMBC</td>
      <td>DNA barcoding</td>
      <td>Microbial ecology, Metagenomics</td>
      <td>Performs analysis of compositions of microbiomes with bias correction.</td>
      <td>Determine taxa whose absolute abundances, per unit volume, of the ecosystem (e.g. gut) are significantly different with changes in the covariate of interest (e.g. group). The current version of ancombc function implements Analysis of Compositions of Microbiomes with Bias Correction (ANCOM-BC) in cross-sectional data while allowing for covariate adjustment.</td>
      <td>To update</td>
      <td>https://github.com/FrederickHuangLin/ANCOMBC</td>
      <td>Metagenomics</td>
      <td>ancombc</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ancombc</td>
    </tr>
    <tr>
      <td></td>
      <td>artic</td>
      <td></td>
      <td>1.2.4</td>
      <td>artic</td>
      <td>To update</td>
      <td>artic</td>
      <td>ARTIC</td>
      <td>Sequence alignment</td>
      <td>Genomics</td>
      <td>The artic pipeline is designed to help run the artic bioinformatics protocols;for example the nCoV-2019 novel coronavirus protocol.Features include: read filtering, primer trimming, amplicon coverage normalisation,variant calling and consensus building</td>
      <td>A bioinformatics pipeline for working with virus sequencing data sequenced with nanopore</td>
      <td>To update</td>
      <td>https://github.com/artic-network/fieldbioinformatics</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/artic</td>
    </tr>
    <tr>
      <td></td>
      <td>bakta</td>
      <td>1.9.2</td>
      <td>1.9.2</td>
      <td>bakta</td>
      <td>Up-to-date</td>
      <td>Bakta</td>
      <td>Bakta</td>
      <td>Genome annotation</td>
      <td>Genomics, Data submission, annotation and curation, Sequence analysis</td>
      <td>"Bakta is a tool for the rapid &amp; standardized annotation of bacterial genomes and plasmids from both isolates and MAGs.It provides dbxref-rich and sORF-including annotations in machine-readable JSON &amp; bioinformatics standard file formats for automatic downstream analysis."</td>
      <td>Rapid &amp; standardized annotation of bacterial genomes, MAGs &amp; plasmids</td>
      <td>Up-to-date</td>
      <td>https://github.com/oschwengers/bakta</td>
      <td>Sequence Analysis</td>
      <td>bakta</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/bakta</td>
    </tr>
    <tr>
      <td></td>
      <td>bandage</td>
      <td>2022.09</td>
      <td>2022.09</td>
      <td>bandage_ng</td>
      <td>Up-to-date</td>
      <td>bandage</td>
      <td>Bandage</td>
      <td>Sequence assembly visualisation</td>
      <td>Genomics, Sequence assembly</td>
      <td>Bandage - A Bioinformatics Application for Navigating De novo Assembly Graphs Easily</td>
      <td>GUI program that allows users to interact with the assembly graphs made by de novo assemblers such as Velvet, SPAdes, MEGAHIT and others. It visualises assembly graphs, with connections, using graph layout algorithms.</td>
      <td>Up-to-date</td>
      <td>https://github.com/rrwick/Bandage</td>
      <td>Visualization</td>
      <td>bandage</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bandage</td>
    </tr>
    <tr>
      <td></td>
      <td>bayescan</td>
      <td>2.1</td>
      <td>2.0.1</td>
      <td>bayescan</td>
      <td>To update</td>
      <td>bayescan</td>
      <td>BayeScan</td>
      <td>Statistical inference</td>
      <td>Genetics, Evolutionary biology, Statistics and probability, DNA polymorphism</td>
      <td>Detecting natural selection from population-based genetic data</td>
      <td>BAYEsian genome SCAN for outliers, aims at identifying candidate loci under natural selection from genetic data, using differences in allele frequencies between populations. It is based on the multinomial-Dirichlet model.</td>
      <td>To update</td>
      <td>http://cmpg.unibe.ch/software/BayeScan/index.html</td>
      <td>Sequence Analysis</td>
      <td>bayescan</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bayescan/</td>
    </tr>
    <tr>
      <td></td>
      <td>binning_refiner</td>
      <td>1.4.3</td>
      <td>1.4.3</td>
      <td>binning_refiner</td>
      <td>Up-to-date</td>
      <td>binning_refiner</td>
      <td>Binning_refiner</td>
      <td>Read binning, Sequence clustering</td>
      <td>Metagenomics</td>
      <td>Reconciles the outputs of different binning programs with the aim to improve the quality of genome bins,especially with respect to contamination levels.</td>
      <td>Improving genome bins through the combination of different binning programs</td>
      <td>Up-to-date</td>
      <td>https://github.com/songweizhi/Binning_refiner</td>
      <td>Metagenomics</td>
      <td>binning_refiner</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/binning_refiner/</td>
    </tr>
    <tr>
      <td></td>
      <td>biom_format</td>
      <td>2.1.15</td>
      <td>2.1.7</td>
      <td>biom-format</td>
      <td>To update</td>
      <td>biomformat</td>
      <td>biomformat</td>
      <td>Formatting</td>
      <td>Laboratory information management, Sequence analysis</td>
      <td>The biom-format package provides a command line interface and Python API for working with BIOM files.</td>
      <td>This package includes basic tools for reading biom-format files, accessing and subsetting data tables from a biom object, as well as limited support for writing a biom-object back to a biom-format file. The design of this API is intended to match the python API and other tools included with the biom-format project, but with a decidedly "R flavor" that should be familiar to R users. This includes S4 classes and methods, as well as extensions of common core functions/methods.</td>
      <td>To update</td>
      <td>https://github.com/biocore/biom-format</td>
      <td>Metagenomics</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/biom_format</td>
    </tr>
    <tr>
      <td></td>
      <td>bracken</td>
      <td>2.9</td>
      <td>2.9</td>
      <td>bracken</td>
      <td>Up-to-date</td>
      <td>bracken</td>
      <td>Bracken</td>
      <td>Statistical calculation</td>
      <td>Metagenomics, Microbial ecology</td>
      <td>Bayesian Reestimation of Abundance with KrakEN</td>
      <td>Statistical method that computes the abundance of species in DNA sequences from a metagenomics sample.</td>
      <td>Up-to-date</td>
      <td>https://ccb.jhu.edu/software/bracken/</td>
      <td>Sequence Analysis, Metagenomics</td>
      <td>bracken</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bracken</td>
    </tr>
    <tr>
      <td></td>
      <td>busco</td>
      <td>5.5.0</td>
      <td>5.6.1</td>
      <td>busco</td>
      <td>To update</td>
      <td>busco</td>
      <td>BUSCO</td>
      <td>Sequence assembly validation, Scaffolding, Genome assembly, Transcriptome assembly</td>
      <td>Sequence assembly, Genomics, Transcriptomics, Sequence analysis</td>
      <td>BUSCO assess genome and annotation completeness</td>
      <td>Provides measures for quantitative assessment of genome assembly, gene set, and transcriptome completeness based on evolutionarily informed expectations of gene content from near-universal single-copy orthologs.</td>
      <td>To update</td>
      <td>https://gitlab.com/ezlab/busco/-/releases</td>
      <td>Sequence Analysis</td>
      <td>busco</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/busco/</td>
    </tr>
    <tr>
      <td></td>
      <td>cactus</td>
      <td>2.7.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>cactus</td>
      <td>Cactus</td>
      <td>Multiple sequence alignment, Genome alignment</td>
      <td>Genomics, Sequence analysis, Phylogeny, Sequence assembly, Mapping, Phylogenetics</td>
      <td>Cactus is a reference-free whole-genome multiple alignment program</td>
      <td>Cactus is a reference-free whole-genome multiple alignment program.</td>
      <td>To update</td>
      <td>https://github.com/ComparativeGenomicsToolkit/cactus</td>
      <td>Sequence Analysis</td>
      <td>cactus</td>
      <td>galaxy-australia</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cactus</td>
    </tr>
    <tr>
      <td></td>
      <td>cat</td>
      <td>5.2.3</td>
      <td>5.3</td>
      <td>cat</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Contig Annotation Tool (CAT)</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/dutilh/CAT</td>
      <td>Metagenomics</td>
      <td>contig_annotation_tool</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/cat</td>
    </tr>
    <tr>
      <td></td>
      <td>cdhit</td>
      <td>4.8.1</td>
      <td>4.8.1</td>
      <td>cd-hit</td>
      <td>Up-to-date</td>
      <td>cd-hit</td>
      <td>cd-hit</td>
      <td>Sequence clustering</td>
      <td>Sequencing</td>
      <td>Cluster or compare biological sequence datasets</td>
      <td>Cluster a nucleotide dataset into representative sequences.</td>
      <td>Up-to-date</td>
      <td>http://weizhongli-lab.org/cd-hit/</td>
      <td>Sequence Analysis, Fasta Manipulation</td>
      <td>cd_hit</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cdhit</td>
    </tr>
    <tr>
      <td></td>
      <td>cemitool</td>
      <td>1.18.1</td>
      <td>1.26.0</td>
      <td>bioconductor-cemitool</td>
      <td>To update</td>
      <td>cemitool</td>
      <td>CEMiTool</td>
      <td>Enrichment analysis, Pathway or network analysis</td>
      <td>Gene expression, Transcriptomics, Microarray experiment</td>
      <td>Gene co-expression network analysis tool</td>
      <td>It unifies the discovery and the analysis of coexpression gene modules in a fully automatic manner, while providing a user-friendly html report with high quality graphs. Our tool evaluates if modules contain genes that are over-represented by specific pathways or that are altered in a specific sample group. Additionally, CEMiTool is able to integrate transcriptomic data with interactome information, identifying the potential hubs on each network.</td>
      <td>To update</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/CEMiTool.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>cemitool</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cemitool</td>
    </tr>
    <tr>
      <td></td>
      <td>checkm</td>
      <td>1.2.0</td>
      <td>1.2.2</td>
      <td>checkm-genome</td>
      <td>To update</td>
      <td>checkm</td>
      <td>CheckM</td>
      <td>Operation</td>
      <td>Genomics, Phylogenomics, Phylogenetics, Taxonomy, Metagenomics</td>
      <td>Assess the quality of microbial genomes recovered from isolates, single cells, and metagenomes</td>
      <td>CheckM provides a set of tools for assessing the quality of genomes recovered from isolates, single cells, or metagenomes.</td>
      <td>To update</td>
      <td>https://github.com/Ecogenomics/CheckM</td>
      <td>Metagenomics</td>
      <td>checkm</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/checkm</td>
    </tr>
    <tr>
      <td></td>
      <td>clustalw</td>
      <td>2.1</td>
      <td>2.1</td>
      <td>clustalw</td>
      <td>Up-to-date</td>
      <td>\n                clustal2\n</td>
      <td></td>
      <td></td>
      <td></td>
      <td>ClustalW multiple sequence alignment program for DNA or proteins</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://www.clustal.org/clustal2/</td>
      <td>Phylogenetics, Sequence Analysis</td>
      <td>clustalw</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/clustalw</td>
    </tr>
    <tr>
      <td></td>
      <td>codeml</td>
      <td>4.9</td>
      <td>4.10.7</td>
      <td>paml</td>
      <td>To update</td>
      <td>paml</td>
      <td>PAML</td>
      <td>Probabilistic sequence generation, Phylogenetic tree generation (maximum likelihood and Bayesian methods), Phylogenetic tree analysis</td>
      <td>Phylogenetics, Sequence analysis</td>
      <td>Detects positive selection</td>
      <td>Package of programs for phylogenetic analyses of DNA or protein sequences using maximum likelihood.</td>
      <td>To update</td>
      <td>http://abacus.gene.ucl.ac.uk/software/paml.html</td>
      <td>Phylogenetics</td>
      <td>codeml</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/codeml</td>
    </tr>
    <tr>
      <td></td>
      <td>cojac</td>
      <td>0.9.1</td>
      <td>0.9.1</td>
      <td>cojac</td>
      <td>Up-to-date</td>
      <td>cojac</td>
      <td>COJAC</td>
      <td></td>
      <td>Genetic variation</td>
      <td>co-occurrence of mutations on amplicons</td>
      <td>CoOccurrence adJusted Analysis and Calling - The cojac package comprises a set of command-line tools to analyse co-occurrence of mutations on amplicons. It is useful, for example, for early detection of viral variants of concern (e.g. Alpha, Delta, Omicron) in environmental samples, and has been designed to scan for multiple SARS-CoV-2 variants in wastewater samples, as analyzed jointly by ETH Zurich, EPFL and Eawag.</td>
      <td>Up-to-date</td>
      <td>https://github.com/cbg-ethz/cojac</td>
      <td>Metagenomics, Sequence Analysis</td>
      <td>cojac</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cojac</td>
    </tr>
    <tr>
      <td></td>
      <td>concoct</td>
      <td>1.1.0</td>
      <td>1.1.0</td>
      <td>concoct</td>
      <td>Up-to-date</td>
      <td>concoct</td>
      <td>CONCOCT</td>
      <td>Sequence clustering, Read binning</td>
      <td>Metagenomics</td>
      <td>CONCOCT (Clustering cONtigs with COverage and ComposiTion) does unsupervised binning of metagenomic contigs byusing nucleotide composition - kmer frequencies - and coverage data for multiple samples. CONCOCT can accurately(up to species level) bin metagenomic contigs.</td>
      <td>A program for unsupervised binning of metagenomic contigs by using nucleotide composition, coverage data in multiple samples and linkage data from paired end reads.</td>
      <td>Up-to-date</td>
      <td>https://github.com/BinPro/CONCOCT</td>
      <td>Metagenomics</td>
      <td>concoct</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/concoct</td>
    </tr>
    <tr>
      <td></td>
      <td>coverm</td>
      <td>0.6.1</td>
      <td>0.7.0</td>
      <td>coverm</td>
      <td>To update</td>
      <td>coverm</td>
      <td>CoverM</td>
      <td>Local alignment</td>
      <td>Bioinformatics</td>
      <td>CoverM genome and contig wrappers</td>
      <td>Read coverage calculator for metagenomics</td>
      <td>To update</td>
      <td>https://github.com/wwood/CoverM</td>
      <td>Sequence Analysis</td>
      <td>coverm</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tools/coverm</td>
    </tr>
    <tr>
      <td></td>
      <td>cutadapt</td>
      <td>4.6</td>
      <td>4.6</td>
      <td>cutadapt</td>
      <td>Up-to-date</td>
      <td>cutadapt</td>
      <td>Cutadapt</td>
      <td>Sequence trimming</td>
      <td>Genomics, Probes and primers, Sequencing</td>
      <td>Flexible tool to remove adapter sequences (and quality trim) high throughput sequencing reads (fasta/fastq).</td>
      <td>Find and remove adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.</td>
      <td>Up-to-date</td>
      <td>https://cutadapt.readthedocs.org/en/stable/</td>
      <td>Fasta Manipulation, Fastq Manipulation, Sequence Analysis</td>
      <td>cutadapt</td>
      <td>lparsons</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cutadapt</td>
    </tr>
    <tr>
      <td></td>
      <td>dada2</td>
      <td></td>
      <td>1.30.0</td>
      <td>bioconductor-dada2</td>
      <td>To update</td>
      <td>dada2</td>
      <td>dada2</td>
      <td>Variant calling, DNA barcoding</td>
      <td>Sequencing, Genetic variation, Microbial ecology, Metagenomics</td>
      <td>DADA2 wrappers</td>
      <td>This package infers exact sequence variants (SVs) from amplicon data, replacing the commonly used and coarser OTU clustering approach. This pipeline inputs demultiplexed fastq files, and outputs the sequence variants and their sample-wise abundances after removing substitution and chimera errors. Taxonomic classification is available via a native implementation of the RDP naive Bayesian classifier.</td>
      <td>To update</td>
      <td>https://benjjneb.github.io/dada2/index.html</td>
      <td>Metagenomics</td>
      <td>dada2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/dada2</td>
    </tr>
    <tr>
      <td></td>
      <td>das_tool</td>
      <td>1.1.6</td>
      <td>1.1.7</td>
      <td>das_tool</td>
      <td>To update</td>
      <td>dastool</td>
      <td>dastool</td>
      <td>Read binning</td>
      <td>Metagenomics</td>
      <td>DAS Tool for genome resolved metagenomics</td>
      <td>DAS Tool is an automated method that integrates the results of a flexible number of binning algorithms to calculate an optimized, non-redundant set of bins from a single assembly.</td>
      <td>To update</td>
      <td>https://github.com/cmks/DAS_Tool</td>
      <td>Metagenomics</td>
      <td>das_tool</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/das_tool</td>
    </tr>
    <tr>
      <td></td>
      <td>diamond</td>
      <td>2.0.15</td>
      <td>2.1.9</td>
      <td>diamond</td>
      <td>To update</td>
      <td>diamond</td>
      <td>Diamond</td>
      <td>Sequence alignment analysis</td>
      <td>Sequence analysis, Proteins</td>
      <td>DIAMOND is a new alignment tool for aligning short DNA sequencing reads to a protein reference database such as NCBI-NR.</td>
      <td>Sequence aligner for protein and translated DNA searches and functions as a drop-in replacement for the NCBI BLAST software tools. It is suitable for protein-protein search as well as DNA-protein search on short reads and longer sequences including contigs and assemblies, providing a speedup of BLAST ranging up to x20,000.</td>
      <td>To update</td>
      <td>https://github.com/bbuchfink/diamond</td>
      <td>Sequence Analysis</td>
      <td>diamond</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/diamond</td>
    </tr>
    <tr>
      <td></td>
      <td>disco</td>
      <td></td>
      <td>1.2</td>
      <td>disco</td>
      <td>To update</td>
      <td>disco</td>
      <td>DISCO</td>
      <td>Protein sequence analysis</td>
      <td>Structure determination</td>
      <td>DISCO is a overlap-layout-consensus (OLC) metagenome assembler</td>
      <td>DISCO is software to perform structure determination of protein homo-oligomers with cyclic symmetry.DISCO computes oligomeric protein structures using geometric constraints derived from RDCs and intermolecular distance restraints such as NOEs or disulfide bonds. When a reliable subunit structure can be calculated from intramolecular restraints, DISCO guarantees that all satisfying oligomer structures will be discovered, yet can run in minutes to hours on only a single desktop-class computer.</td>
      <td>To update</td>
      <td>http://disco.omicsbio.org/</td>
      <td>Metagenomics, Assembly</td>
      <td>disco</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/disco/</td>
    </tr>
    <tr>
      <td></td>
      <td>dram</td>
      <td>1.3.5</td>
      <td>1.4.6</td>
      <td>dram</td>
      <td>To update</td>
      <td>dram</td>
      <td>DRAM</td>
      <td>Gene functional annotation</td>
      <td>Metagenomics, Biological databases, Molecular genetics</td>
      <td>DRAM for distilling microbial metabolism to automate the curation of microbiome function</td>
      <td>Distilled and Refined Annotation of Metabolism: A tool for the annotation and curation of function for microbial and viral genomes</td>
      <td>To update</td>
      <td>https://github.com/WrightonLabCSU/DRAM</td>
      <td>Metagenomics</td>
      <td>dram</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/dram</td>
    </tr>
    <tr>
      <td></td>
      <td>drep</td>
      <td>3.4.5</td>
      <td>3.4.5</td>
      <td>drep</td>
      <td>Up-to-date</td>
      <td>drep</td>
      <td>dRep</td>
      <td>Genome comparison</td>
      <td>Metagenomics, Genomics, Sequence analysis</td>
      <td>dRep compares and dereplicates genome sets</td>
      <td>Fast and accurate genomic comparisons that enables improved genome recovery from metagenomes through de-replication.</td>
      <td>Up-to-date</td>
      <td>https://github.com/MrOlm/drep</td>
      <td>Metagenomics</td>
      <td>drep</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/drep</td>
    </tr>
    <tr>
      <td></td>
      <td>export2graphlan</td>
      <td>0.20</td>
      <td>0.22</td>
      <td>export2graphlan</td>
      <td>To update</td>
      <td>export2graphlan</td>
      <td>export2graphlan</td>
      <td>Conversion</td>
      <td>Taxonomy, Metabolomics, Biomarkers</td>
      <td>export2graphlan is a conversion software tool for producing both annotation and tree file for GraPhlAn</td>
      <td>export2graphlan is a conversion software tool for producing both annotation and tree file for GraPhlAn. In particular, the annotation file tries to highlight specific sub-trees deriving automatically from input file what nodes are important.</td>
      <td>To update</td>
      <td>https://bitbucket.org/CibioCM/export2graphlan/overview</td>
      <td>Metagenomics</td>
      <td>export2graphlan</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/export2graphlan/</td>
    </tr>
    <tr>
      <td></td>
      <td>fargene</td>
      <td>0.1</td>
      <td>0.1</td>
      <td>fargene</td>
      <td>Up-to-date</td>
      <td>fargene</td>
      <td>fARGene</td>
      <td>Antimicrobial resistance prediction</td>
      <td>Metagenomics, Microbiology, Public health and epidemiology</td>
      <td>fARGene (Fragmented Antibiotic Resistance Gene iENntifiEr )</td>
      <td>fARGene (Fragmented Antibiotic Resistance Gene iENntifiEr ) is a tool that takes either fragmented metagenomic data or longer sequences as input and predicts and delivers full-length antiobiotic resistance genes as output.</td>
      <td>Up-to-date</td>
      <td>https://github.com/fannyhb/fargene</td>
      <td>Sequence Analysis</td>
      <td>fargene</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fargene</td>
    </tr>
    <tr>
      <td></td>
      <td>fastp</td>
      <td></td>
      <td>0.23.4</td>
      <td>fastp</td>
      <td>To update</td>
      <td>\n                fastp\n</td>
      <td></td>
      <td></td>
      <td></td>
      <td>Fast all-in-one preprocessing for FASTQ files</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/OpenGene/fastp</td>
      <td>Sequence Analysis</td>
      <td>fastp</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fastp</td>
    </tr>
    <tr>
      <td></td>
      <td>fastqe</td>
      <td>0.3.1+galaxy0</td>
      <td>0.3.1</td>
      <td>fastqe</td>
      <td>To update</td>
      <td>fastqe</td>
      <td>FASTQE</td>
      <td>Sequencing quality control</td>
      <td>Sequence analysis, Sequencing</td>
      <td>FASTQE</td>
      <td>Compute quality stats for FASTQ files and print those stats as emoji... for some reason.</td>
      <td>To update</td>
      <td>https://fastqe.com/</td>
      <td>Sequence Analysis</td>
      <td>fastqe</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fastqe</td>
    </tr>
    <tr>
      <td></td>
      <td>fasttree</td>
      <td>2.1.10</td>
      <td>2.1.11</td>
      <td>fasttree</td>
      <td>To update</td>
      <td>fasttree</td>
      <td>FastTree</td>
      <td>Phylogenetic tree generation (from molecular sequences), Phylogenetic tree generation (maximum likelihood and Bayesian methods)</td>
      <td>Phylogenetics, Sequence analysis</td>
      <td>FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide or protein sequences - GVL</td>
      <td>Infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide or protein sequences.</td>
      <td>To update</td>
      <td>http://www.microbesonline.org/fasttree/</td>
      <td>Phylogenetics</td>
      <td>fasttree</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fasttree</td>
    </tr>
    <tr>
      <td></td>
      <td>filtlong</td>
      <td>0.2.1</td>
      <td>0.2.1</td>
      <td>filtlong</td>
      <td>Up-to-date</td>
      <td>\n                filtlong\n</td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filtlong - Filtering long reads by quality</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/rrwick/Filtlong</td>
      <td>Fastq Manipulation, Sequence Analysis</td>
      <td>filtlong</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/filtlong</td>
    </tr>
    <tr>
      <td></td>
      <td>fraggenescan</td>
      <td></td>
      <td>1.31</td>
      <td>fraggenescan</td>
      <td>To update</td>
      <td>fraggenescan</td>
      <td>FragGeneScan</td>
      <td>Gene prediction</td>
      <td>Genetics, Sequence analysis</td>
      <td>Tool for finding (fragmented) genes in short read</td>
      <td>Application for finding (fragmented) genes in short reads</td>
      <td>To update</td>
      <td>https://sourceforge.net/projects/fraggenescan/</td>
      <td>Sequence Analysis</td>
      <td>fraggenescan</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fraggenescan/</td>
    </tr>
    <tr>
      <td></td>
      <td>freyja</td>
      <td>1.4.4</td>
      <td>1.4.9</td>
      <td>freyja</td>
      <td>To update</td>
      <td>freyja</td>
      <td></td>
      <td></td>
      <td></td>
      <td>lineage abundances estimation</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/andersen-lab/Freyja</td>
      <td>Metagenomics, Sequence Analysis</td>
      <td>freyja</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/freyja</td>
    </tr>
    <tr>
      <td></td>
      <td>funannotate</td>
      <td>1.8.15</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>\n                funannotate\n</td>
      <td></td>
      <td></td>
      <td></td>
      <td>Funannotate is a genome prediction, annotation, and comparison software package.</td>
      <td></td>
      <td>To update</td>
      <td>https://funannotate.readthedocs.io</td>
      <td>Genome annotation</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/funannotate</td>
    </tr>
    <tr>
      <td></td>
      <td>glimmer</td>
      <td></td>
      <td>3.02</td>
      <td>glimmer</td>
      <td>To update</td>
      <td>gemini</td>
      <td>GEMINI</td>
      <td>Sequence analysis, Genetic variation analysis</td>
      <td>Sequence analysis</td>
      <td>Glimmer makes gene predictions.</td>
      <td>GEMINI (GEnome MINIng) is a flexible framework for exploring genetic variation in the context of the wealth of genome annotations available for the human genome. By placing genetic variants, sample phenotypes and genotypes, as well as genome annotations into an integrated database framework, GEMINI provides a simple, flexible, and powerful system for exploring genetic variation for disease and population genetics.</td>
      <td>To update</td>
      <td>https://ccb.jhu.edu/software/glimmer/</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/glimmer</td>
    </tr>
    <tr>
      <td></td>
      <td>goenrichment</td>
      <td>2.0.1</td>
      <td>2.0.1</td>
      <td>goenrichment</td>
      <td>Up-to-date</td>
      <td>goenrichment</td>
      <td>GOEnrichment</td>
      <td>Gene-set enrichment analysis</td>
      <td>Transcriptomics</td>
      <td>Performs GO Enrichment analysis.</td>
      <td>GOEnrichment is a tool for performing GO enrichment analysis of gene sets, such as those obtained from RNA-seq or Microarray experiments, to help characterize them at the functional level. It is available in Galaxy Europe and as a stand-alone tool.GOEnrichment is flexible in that it allows the user to use any version of the Gene Ontology and any GO annotation file they desire. To enable the use of GO slims, it is accompanied by a sister tool GOSlimmer, which can convert annotation files from full GO to any specified GO slim.The tool features an optional graph clustering algorithm to reduce the redundancy in the set of enriched GO terms and simplify its output.It was developed by the BioData.pt / ELIXIR-PT team at the Instituto Gulbenkian de Ciência.</td>
      <td>Up-to-date</td>
      <td>https://github.com/DanFaria/GOEnrichment</td>
      <td>Genome annotation</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/goenrichment</td>
    </tr>
    <tr>
      <td></td>
      <td>goseq</td>
      <td>1.50.0</td>
      <td>1.54.0</td>
      <td>bioconductor-goseq</td>
      <td>To update</td>
      <td>goseq</td>
      <td>GOseq</td>
      <td>Gene functional annotation</td>
      <td>RNA-Seq</td>
      <td>goseq does selection-unbiased testing for category enrichment amongst differentially expressed (DE) genes for RNA-seq data</td>
      <td>Detect Gene Ontology and/or other user defined categories which are over/under represented in RNA-seq data.</td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/goseq.html</td>
      <td>Statistics, RNA, Micro-array Analysis</td>
      <td>goseq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/goseq</td>
    </tr>
    <tr>
      <td></td>
      <td>graphlan</td>
      <td></td>
      <td>1.1.3</td>
      <td>graphlan</td>
      <td>To update</td>
      <td>graphlan</td>
      <td>GraPhlAn</td>
      <td>Phylogenetic inference, Phylogenetic tree visualisation, Phylogenetic tree editing, Taxonomic classification</td>
      <td>Metagenomics, Phylogenetics, Phylogenomics, Cladistics</td>
      <td>GraPhlAn is a software tool for producing high-quality circular representations of taxonomic and phylogenetic trees</td>
      <td>GraPhlAn is a software tool for producing high-quality circular representations of taxonomic and phylogenetic trees. GraPhlAn focuses on concise, integrative, informative, and publication-ready representations of phylogenetically- and taxonomically-driven investigation.</td>
      <td>To update</td>
      <td>https://github.com/biobakery/graphlan</td>
      <td>Metagenomics, Graphics, Phylogenetics</td>
      <td>graphlan</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/humann2/</td>
    </tr>
    <tr>
      <td></td>
      <td>gtdbtk</td>
      <td>2.2.2</td>
      <td>2.3.2</td>
      <td>gtdbtk</td>
      <td>To update</td>
      <td>GTDB-Tk</td>
      <td>GTDB-Tk</td>
      <td>Genome alignment, Taxonomic classification, Sequence assembly, Query and retrieval</td>
      <td>Metagenomics, Taxonomy, Phylogenetics, Database management, Proteins</td>
      <td>GTDB-Tk is a software tool kit for assigning objective taxonomic classifications to bacterial and archaeal genomesbased on the Genome Database Taxonomy GTDB. It is designed to work with recent advances that allow hundreds orthousands of metagenome-assembled genomes (MAGs) to be obtained directly from environmental samples. It can alsobe applied to isolate and single-cell genomes.</td>
      <td>a toolkit to classify genomes with the Genome Taxonomy Database.GTDB-Tk: a toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes.GTDB-Tk is a software toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes based on the Genome Database Taxonomy GTDB. It is designed to work with recent advances that allow hundreds or thousands of metagenome-assembled genomes (MAGs) to be obtained directly from environmental samples. It can also be applied to isolate and single-cell genomes. The GTDB-Tk is open source and released under the GNU General Public License (Version 3).</td>
      <td>To update</td>
      <td>https://github.com/Ecogenomics/GTDBTk</td>
      <td>Metagenomics</td>
      <td>gtdbtk</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gtdbtk</td>
    </tr>
    <tr>
      <td></td>
      <td>gubbins</td>
      <td>3.2.1</td>
      <td>3.3.3</td>
      <td>gubbins</td>
      <td>To update</td>
      <td>gubbins</td>
      <td>Gubbins</td>
      <td>Genotyping, Phylogenetic inference, Ancestral reconstruction</td>
      <td>Phylogeny, Genotype and phenotype, Whole genome sequencing</td>
      <td>Gubbins - bacterial recombination detection</td>
      <td>Gubbins is a tool for rapid phylogenetic analysis of large samples of recombinant bacterial whole genome sequences.</td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>gubbins</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gubbins</td>
    </tr>
    <tr>
      <td></td>
      <td>hamronization</td>
      <td>1.0.3</td>
      <td>1.1.4</td>
      <td>hamronization</td>
      <td>To update</td>
      <td>hamronization</td>
      <td>hAMRonization</td>
      <td>Data handling, Antimicrobial resistance prediction, Parsing</td>
      <td>Public health and epidemiology, Microbiology, Bioinformatics</td>
      <td>Convert AMR gene detection tool output to hAMRonization specification format.</td>
      <td>Parse multiple Antimicrobial Resistance Analysis Reports into a common data structure</td>
      <td>To update</td>
      <td>https://github.com/pha4ge/hAMRonization</td>
      <td>Sequence Analysis</td>
      <td>hamronization</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hamronization</td>
    </tr>
    <tr>
      <td></td>
      <td>hansel</td>
      <td>2.6.1</td>
      <td>2.6.1</td>
      <td>bio_hansel</td>
      <td>Up-to-date</td>
      <td>Biohansel</td>
      <td>BioHansel</td>
      <td>Genotyping, SNP detection, Genome assembly</td>
      <td>Whole genome sequencing, DNA polymorphism, Genotype and phenotype, Infectious disease, Agricultural science</td>
      <td>Heidelberg and Enteritidis SNP Elucidation</td>
      <td>BioHansel is a tool for performing high-resolution genotyping of bacterial isolates by identifying phylogenetically informative single nucleotide polymorphisms (SNPs), also known as canonical SNPs, in whole genome sequencing (WGS) data. The application uses a fast k-mer matching algorithm to map pathogen WGS data to canonical SNPs contained in hierarchically structured schemas and assigns genotypes based on the detected SNP profile.</td>
      <td>Up-to-date</td>
      <td>https://github.com/phac-nml/bio_hansel</td>
      <td>Sequence Analysis</td>
      <td>bio_hansel</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hansel</td>
    </tr>
    <tr>
      <td></td>
      <td>hifiasm_meta</td>
      <td>0.3.1</td>
      <td>hamtv0.3.1</td>
      <td>hifiasm_meta</td>
      <td>To update</td>
      <td>hifiasm-meta</td>
      <td>Hifiasm-meta</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly, Metagenomics</td>
      <td>A hifiasm fork for metagenome assembly using Hifi reads.</td>
      <td>Hifiasm_meta - de novo metagenome assembler, based on hifiasm, a haplotype-resolved de novo assembler for PacBio Hifi reads.</td>
      <td>To update</td>
      <td>https://github.com/xfengnefx/hifiasm-meta</td>
      <td>Metagenomics</td>
      <td>hifiasm_meta</td>
      <td>galaxy-australia</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hifiasm_meta</td>
    </tr>
    <tr>
      <td></td>
      <td>hmmer3</td>
      <td>3.3.2</td>
      <td>3.4</td>
      <td>hmmer</td>
      <td>To update</td>
      <td>hmmer3</td>
      <td>HMMER3</td>
      <td>Formatting, Multiple sequence alignment, Sequence profile generation, Format validation, Conversion, Sequence generation, Data retrieval, Statistical calculation, Database search, Formatting, Database search, Database search, Probabilistic sequence generation, Statistical calculation, Statistical calculation, Sequence database search, Formatting, Sequence database search, Database search, Sequence database search</td>
      <td>Sequence analysis, Sequence sites, features and motifs, Gene and protein families</td>
      <td>HMMER is used for searching sequence databases for homologs of proteinsequences, and for making protein sequence alignments. It implementsmethods using probabilistic models called profile hidden Markov models(profile HMMs).</td>
      <td>This tool is used for searching sequence databases for homologs of protein sequences, and for making protein sequence alignments. It implements methods using probabilistic models called profile hidden Markov models. The new HMMER3 project, HMMER is now as fast as BLAST for protein search.</td>
      <td>To update</td>
      <td>http://hmmer.org/</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hmmer3</td>
    </tr>
    <tr>
      <td></td>
      <td>humann</td>
      <td>3.8</td>
      <td>3.8</td>
      <td>humann</td>
      <td>Up-to-date</td>
      <td>humann</td>
      <td>humann</td>
      <td>Species frequency estimation, Taxonomic classification, Phylogenetic analysis</td>
      <td>Metagenomics, Phylogenomics</td>
      <td>HUMAnN for functionally profiling metagenomes and metatranscriptomes at species-level resolution</td>
      <td>HUMAnN is a pipeline for efficiently and accurately profiling the presence/absence and abundance of microbial pathways in a community from metagenomic or metatranscriptomic sequencing data (typically millions of short DNA/RNA reads). This process, referred to as functional profiling, aims to describe the metabolic potential of a microbial community and its members. More generally, functional profiling answers the question “What are the microbes in my community-of-interest doing (or are capable of doing)?”</td>
      <td>Up-to-date</td>
      <td>http://huttenhower.sph.harvard.edu/humann</td>
      <td>Metagenomics</td>
      <td>humann</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/humann</td>
    </tr>
    <tr>
      <td></td>
      <td>hyphy</td>
      <td>2.5.47</td>
      <td>2.5.59</td>
      <td>hyphy</td>
      <td>To update</td>
      <td>HyPhy</td>
      <td>HyPhy</td>
      <td>Statistical calculation</td>
      <td>Phylogeny, Small molecules, Molecular interactions, pathways and networks</td>
      <td>Hypothesis Testing using Phylogenies</td>
      <td>Software package for the analysis of genetic sequences using techniques in phylogenetics, molecular evolution, and machine learning.</td>
      <td>To update</td>
      <td>http://www.hyphy.org</td>
      <td>Phylogenetics</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hyphy/</td>
    </tr>
    <tr>
      <td></td>
      <td>hypo</td>
      <td>1.0.3</td>
      <td>1.0.3</td>
      <td>hypo</td>
      <td>Up-to-date</td>
      <td>HyPo</td>
      <td>HyPo</td>
      <td>Optimisation and refinement, Genome assembly</td>
      <td>Sequence assembly, Genomics</td>
      <td>Super Fast &amp; Accurate Polisher for Long Read Genome Assemblies</td>
      <td>HyPo, a Hybrid Polisher, utilizes short as well as long reads within a single run to polish a long reads assembly of small and large genomes.</td>
      <td>Up-to-date</td>
      <td>https://github.com/kensung-lab/hypo</td>
      <td>Assembly</td>
      <td>hypo</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hypo</td>
    </tr>
    <tr>
      <td></td>
      <td>icescreen</td>
      <td>1.3.1</td>
      <td>1.3.1</td>
      <td>icescreen</td>
      <td>Up-to-date</td>
      <td>icescreen</td>
      <td>ICEscreen</td>
      <td>Database search, Protein feature detection</td>
      <td>Mobile genetic elements, Sequence sites, features and motifs, Genomics, Molecular interactions, pathways and networks, Structural variation</td>
      <td>ICEscreen identifies Integrative Conjugative Elements (ICEs) and Integrative Mobilizable Elements (IMEs) in Bacillota genomes.</td>
      <td>A tool to detect Firmicute ICEs and IMEs, isolated or enclosed in composite structures.</td>
      <td>Up-to-date</td>
      <td>https://icescreen.migale.inrae.fr/</td>
      <td>Genome annotation</td>
      <td>icescreen</td>
      <td>iuc</td>
      <td>https://forgemia.inra.fr/ices_imes_analysis/icescreen</td>
    </tr>
    <tr>
      <td></td>
      <td>idba_ud</td>
      <td></td>
      <td>1.1.3</td>
      <td>idba</td>
      <td>To update</td>
      <td>idba</td>
      <td>IDBA</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly</td>
      <td>Wrappers for the idba assembler variants.</td>
      <td>A short read assembler based on iterative De Bruijn graph. It is developed under 64-bit Linux, but should be suitable for all unix-like system.</td>
      <td>To update</td>
      <td>https://i.cs.hku.hk/~alse/hkubrg/projects/index.html</td>
      <td>Assembly</td>
      <td>idba</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/idba_ud</td>
    </tr>
    <tr>
      <td></td>
      <td>instrain</td>
      <td>1.5.3</td>
      <td>1.8.0</td>
      <td>instrain</td>
      <td>To update</td>
      <td>instrain</td>
      <td>InStrain</td>
      <td>SNP detection, Genome comparison</td>
      <td>Mapping, Metagenomics</td>
      <td>InStrain is a tool for analysis of co-occurring genome populations from metagenomes</td>
      <td>InStrain is a tool for analysis of co-occurring genome populations from metagenomes that allows highly accurate genome comparisons, analysis of coverage, microdiversity, and linkage, and sensitive SNP detection with gene localization and synonymous non-synonymous identification</td>
      <td>To update</td>
      <td>https://instrain.readthedocs.io/en/latest/#</td>
      <td>Metagenomics</td>
      <td>instrain</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/instrain</td>
    </tr>
    <tr>
      <td></td>
      <td>interproscan</td>
      <td>5.59-91.0</td>
      <td>5.59_91.0</td>
      <td>interproscan</td>
      <td>To update</td>
      <td>interproscan_ebi</td>
      <td>InterProScan (EBI)</td>
      <td>Sequence motif recognition, Protein feature detection</td>
      <td>Gene and protein families, Sequence analysis</td>
      <td>Interproscan queries the interpro database and provides annotations.</td>
      <td>Scan sequences against the InterPro protein signature databases.</td>
      <td>To update</td>
      <td>http://www.ebi.ac.uk/Tools/pfa/iprscan5/</td>
      <td>Sequence Analysis</td>
      <td>interproscan</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/interproscan</td>
    </tr>
    <tr>
      <td></td>
      <td>iqtree</td>
      <td>2.1.2</td>
      <td>2.2.6</td>
      <td>iqtree</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Efficient phylogenomic software by maximum likelihood</td>
      <td></td>
      <td>To update</td>
      <td>http://www.iqtree.org/</td>
      <td>Phylogenetics</td>
      <td>iqtree</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/iqtree/</td>
    </tr>
    <tr>
      <td></td>
      <td>ivar</td>
      <td>1.4.2</td>
      <td>1.4.2</td>
      <td>ivar</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iVar is a computational package that contains functions broadly useful for viral amplicon-based sequencing</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/andersen-lab/ivar</td>
      <td>Sequence Analysis</td>
      <td>ivar</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ivar/</td>
    </tr>
    <tr>
      <td></td>
      <td>jbrowse</td>
      <td>1.16.11</td>
      <td>1.16.11</td>
      <td>jbrowse</td>
      <td>Up-to-date</td>
      <td>jbrowse</td>
      <td>JBrowse</td>
      <td>Genome visualisation</td>
      <td>Genomics</td>
      <td>JBrowse Genome Browser integrated as a Galaxy Tool</td>
      <td>Slick, speedy genome browser with a responsive and dynamic AJAX interface for visualization of genome data. Being developed by the GMOD project as a successor to GBrowse.</td>
      <td>Up-to-date</td>
      <td>https://jbrowse.org</td>
      <td>Sequence Analysis</td>
      <td>jbrowse</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/jbrowse</td>
    </tr>
    <tr>
      <td></td>
      <td>jellyfish</td>
      <td></td>
      <td>2.3.1</td>
      <td>kmer-jellyfish</td>
      <td>To update</td>
      <td>Jellyfish</td>
      <td>Jellyfish</td>
      <td>k-mer counting</td>
      <td>Sequence analysis, Genomics</td>
      <td>Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA</td>
      <td>A command-line algorithm for counting k-mers in DNA sequence.</td>
      <td>To update</td>
      <td>https://github.com/gmarcais/Jellyfish</td>
      <td>Assembly</td>
      <td>jellyfish</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/jellyfish</td>
    </tr>
    <tr>
      <td></td>
      <td>kc-align</td>
      <td>1.0.2</td>
      <td>1.0.2</td>
      <td>kcalign</td>
      <td>Up-to-date</td>
      <td>kc-align</td>
      <td>kc-align</td>
      <td>Multiple sequence alignment</td>
      <td>Mapping</td>
      <td>Kc-Align custom tool</td>
      <td>A fast and accurate tool for performing codon-aware multiple sequence alignments</td>
      <td>Up-to-date</td>
      <td>https://github.com/davebx/kc-align</td>
      <td>Sequence Analysis</td>
      <td>kc_align</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/kc-align</td>
    </tr>
    <tr>
      <td></td>
      <td>khmer</td>
      <td>3.0.0a3</td>
      <td>3.0.0a3</td>
      <td>khmer</td>
      <td>Up-to-date</td>
      <td>khmer</td>
      <td>khmer</td>
      <td>Standardisation and normalisation, De-novo assembly</td>
      <td>Sequence assembly</td>
      <td>In-memory nucleotide sequence k-mer counting, filtering, graph traversal and more</td>
      <td>khmer is a set of command-line tools for working with DNA shotgun sequencing data from genomes, transcriptomes, metagenomes, and single cells. khmer can make de novo assemblies faster, and sometimes better. khmer can also identify (and fix) problems with shotgun data.</td>
      <td>Up-to-date</td>
      <td>https://khmer.readthedocs.org/</td>
      <td>Assembly, Next Gen Mappers</td>
      <td>khmer</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/khmer</td>
    </tr>
    <tr>
      <td></td>
      <td>kleborate</td>
      <td>2.3.2</td>
      <td>2.3.2</td>
      <td>kleborate</td>
      <td>Up-to-date</td>
      <td>kleborate</td>
      <td>Kleborate</td>
      <td>Multilocus sequence typing, Genome assembly, Virulence prediction</td>
      <td>Public health and epidemiology, Metagenomics, Population genomics, Sequence assembly, Whole genome sequencing</td>
      <td>Screen genome assemblies of Klebsiella pneumoniae and the Klebsiella pneumoniae species complex (KpSC)</td>
      <td>Genomic surveillance framework and global population structure for Klebsiella pneumoniae.Kleborate is a tool to screen genome assemblies of Klebsiella pneumoniae and the Klebsiella pneumoniae species complex (KpSC) for:.A manuscript describing the Kleborate software in full is currently in preparation. In the meantime, if you use Kleborate, please cite the preprint: Lam, MMC. et al. Genomic surveillance framework and global population structure for Klebsiella pneumoniae. bioRxiv (2020).</td>
      <td>Up-to-date</td>
      <td>https://github.com/katholt/Kleborate/wiki</td>
      <td>Metagenomics</td>
      <td>kleborate</td>
      <td>iuc</td>
      <td>https://github.com/katholt/Kleborate</td>
    </tr>
    <tr>
      <td></td>
      <td>kofamscan</td>
      <td>1.3.0</td>
      <td>1.3.0</td>
      <td>kofamscan</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Gene function annotation tool based on KEGG Orthology and hidden Markov model</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/takaram/kofam_scan</td>
      <td>Sequence Analysis</td>
      <td>kofamscan</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/kofamscan</td>
    </tr>
    <tr>
      <td></td>
      <td>kraken_biom</td>
      <td>1.2.0</td>
      <td>1.2.0</td>
      <td>kraken-biom</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Create BIOM-format tables (http://biom-format.org) from Kraken output (http://ccb.jhu.edu/software/kraken/)</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/smdabdoub/kraken-biom</td>
      <td>Metagenomics</td>
      <td>kraken_biom</td>
      <td>iuc</td>
      <td>https://github.com/smdabdoub/kraken-biom</td>
    </tr>
    <tr>
      <td></td>
      <td>kraken_taxonomy_report</td>
      <td>0.0.3</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Kraken taxonomy report</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/blankenberg/Kraken-Taxonomy-Report</td>
      <td>Metagenomics</td>
      <td>kraken_taxonomy_report</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/kraken_taxonomy_report</td>
    </tr>
    <tr>
      <td></td>
      <td>krakentools</td>
      <td>1.2</td>
      <td>1.2</td>
      <td>krakentools</td>
      <td>Up-to-date</td>
      <td>krakentools</td>
      <td>KrakenTools</td>
      <td>Visualisation, Aggregation</td>
      <td>Taxonomy, Metagenomics</td>
      <td>KrakenTools is a suite of scripts to be used alongside the Kraken</td>
      <td>KrakenTools provides individual scripts to analyze Kraken/Kraken2/Bracken/KrakenUniq output files</td>
      <td>Up-to-date</td>
      <td>https://github.com/jenniferlu717/KrakenTools</td>
      <td>Metagenomics</td>
      <td>krakentools</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/krakentools</td>
    </tr>
    <tr>
      <td></td>
      <td>krocus</td>
      <td>1.0.1</td>
      <td>1.0.3</td>
      <td>krocus</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Predict MLST directly from uncorrected long reads</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/quadram-institute-bioscience/krocus</td>
      <td>Sequence Analysis</td>
      <td>krocus</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/krocus</td>
    </tr>
    <tr>
      <td></td>
      <td>legsta</td>
      <td>0.5.1</td>
      <td>0.5.1</td>
      <td>legsta</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Performs in silico Legionella pneumophila sequence based typing.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/tseemann/legsta</td>
      <td>Sequence Analysis</td>
      <td>legsta</td>
      <td>iuc</td>
      <td>https://github.com/tseemann/legsta</td>
    </tr>
    <tr>
      <td></td>
      <td>lineagespot</td>
      <td>1.4.0</td>
      <td></td>
      <td>r-base</td>
      <td>To update</td>
      <td>lineagespot</td>
      <td>lineagespot</td>
      <td>Variant calling</td>
      <td>Metagenomics, Gene transcripts, Evolutionary biology, Sequencing, Genetic variation</td>
      <td>Identification of SARS-CoV-2 related metagenomic mutations based on a single (or a list of) variant(s) file(s)</td>
      <td>Lineagespot is a framework written in R, and aims to identify and assign different SARS-CoV-2 lineages based on a single variant file (i.e., variant calling format).</td>
      <td>To update</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/lineagespot.html</td>
      <td>Metagenomics, Sequence Analysis</td>
      <td>lineagespot</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/lineagespot</td>
    </tr>
    <tr>
      <td></td>
      <td>lorikeet</td>
      <td>20</td>
      <td>20</td>
      <td>lorikeet</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tools for M. tuberculosis DNA fingerprinting (spoligotyping)</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/AbeelLab/lorikeet</td>
      <td>Sequence Analysis</td>
      <td>lorikeet_spoligotype</td>
      <td>iuc</td>
      <td>https://github.com/AbeelLab/lorikeet</td>
    </tr>
    <tr>
      <td></td>
      <td>m6anet</td>
      <td>2.1.0</td>
      <td>2.1.0</td>
      <td>m6anet</td>
      <td>Up-to-date</td>
      <td>m6Anet</td>
      <td>m6Anet</td>
      <td>Quantification, Imputation, Gene expression profiling</td>
      <td>RNA-Seq, Transcriptomics, RNA, Machine learning</td>
      <td>m6anet to detect m6A RNA modifications from nanopore data</td>
      <td>Detection of m6A from direct RNA sequencing using a Multiple Instance Learning framework.</td>
      <td>Up-to-date</td>
      <td>https://m6anet.readthedocs.io/en/latest</td>
      <td>Sequence Analysis</td>
      <td>m6anet</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/m6anet</td>
    </tr>
    <tr>
      <td></td>
      <td>maaslin2</td>
      <td>0.99.12</td>
      <td>0.99.12</td>
      <td>maaslin2</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MaAsLin2 is comprehensive R package for efficiently determining multivariable association between microbial meta'omic features and clinical metadata.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://huttenhower.sph.harvard.edu/maaslin</td>
      <td>Metagenomics</td>
      <td>maaslin2</td>
      <td>iuc</td>
      <td>https://github.com/biobakery/Maaslin2</td>
    </tr>
    <tr>
      <td></td>
      <td>maker</td>
      <td>2.31.11</td>
      <td>3.01.03</td>
      <td>maker</td>
      <td>To update</td>
      <td>maker</td>
      <td>MAKER</td>
      <td>Genome annotation</td>
      <td>Genomics, DNA, Sequence analysis</td>
      <td>MAKER is a portable and easily configurable genome annotation pipeline.Its purpose is to allow smaller eukaryotic and prokaryotic genome projects to independently annotate their genomes and to create genome databases.</td>
      <td>Portable and easily configurable genome annotation pipeline. It’s purpose is to allow smaller eukaryotic and prokaryotic genome projects to independently annotate their genomes and to create genome databases.</td>
      <td>To update</td>
      <td>http://www.yandell-lab.org/software/maker.html</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/maker</td>
    </tr>
    <tr>
      <td></td>
      <td>mapseq</td>
      <td>2.1.1</td>
      <td></td>
      <td>perl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>fast and accurate sequence read classification tool designed to assign taxonomy and OTU classifications to ribosomal RNA sequences.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/jfmrod/MAPseq</td>
      <td>Metagenomics</td>
      <td>mapseq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mapseq</td>
    </tr>
    <tr>
      <td></td>
      <td>mash</td>
      <td>2.3</td>
      <td>2.3</td>
      <td>mash</td>
      <td>Up-to-date</td>
      <td>mash</td>
      <td>Mash</td>
      <td>Sequence distance matrix generation</td>
      <td>Genomics, Metagenomics, Statistics and probability, Sequence analysis, DNA mutation</td>
      <td>Fast genome and metagenome distance estimation using MinHash</td>
      <td>Fast genome and metagenome distance estimation using MinHash.</td>
      <td>Up-to-date</td>
      <td>https://github.com/marbl/Mash</td>
      <td>Metagenomics</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mash</td>
    </tr>
    <tr>
      <td></td>
      <td>maxbin2</td>
      <td></td>
      <td>2.2.7</td>
      <td>maxbin2</td>
      <td>To update</td>
      <td>maxbin</td>
      <td>MaxBin</td>
      <td>Sequence assembly</td>
      <td>Metagenomics, Sequence assembly, Microbiology</td>
      <td>clusters metagenomic contigs into bins</td>
      <td>Software for binning assembled metagenomic sequences based on an Expectation-Maximization algorithm.</td>
      <td>To update</td>
      <td>https://downloads.jbei.org/data/microbial_communities/MaxBin/MaxBin.html</td>
      <td>Metagenomics</td>
      <td>maxbin2</td>
      <td>mbernt</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/maxbin2/</td>
    </tr>
    <tr>
      <td></td>
      <td>mcl</td>
      <td>22.282</td>
      <td>22.282</td>
      <td>mcl</td>
      <td>Up-to-date</td>
      <td>mcl</td>
      <td>MCL</td>
      <td>Clustering, Network analysis, Gene regulatory network analysis</td>
      <td>Molecular interactions, pathways and networks</td>
      <td>The Markov Cluster Algorithm, a cluster algorithm for graphs</td>
      <td>MCL is a clustering algorithm widely used in bioinformatics and gaining traction in other fields.</td>
      <td>Up-to-date</td>
      <td>https://micans.org/mcl/man/mcl.html</td>
      <td>Sequence Analysis</td>
      <td>mcl</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/mcl</td>
    </tr>
    <tr>
      <td></td>
      <td>medaka</td>
      <td>1.7.2</td>
      <td>1.11.3</td>
      <td>medaka</td>
      <td>To update</td>
      <td>medaka</td>
      <td>Medaka</td>
      <td>Base-calling, Variant calling, Sequence assembly</td>
      <td>Sequence assembly, Machine learning</td>
      <td>Sequence correction provided by ONT Research</td>
      <td>medaka is a tool to create consensus sequences and variant calls from nanopore sequencing data. This task is performed using neural networks applied a pileup of individual sequencing reads against a draft assembly.</td>
      <td>To update</td>
      <td>https://github.com/nanoporetech/medaka</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/medaka</td>
    </tr>
    <tr>
      <td></td>
      <td>megahit</td>
      <td>1.2.9</td>
      <td>1.2.9</td>
      <td>megahit</td>
      <td>Up-to-date</td>
      <td>megahit</td>
      <td>MEGAHIT</td>
      <td>Genome assembly</td>
      <td>Metagenomics, Sequencing, Ecology, Sequence assembly</td>
      <td>An ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph.</td>
      <td>Single node assembler for large and complex metagenomics NGS reads, such as soil. It makes use of succinct de Bruijn graph to achieve low memory usage, whereas its goal is not to make memory usage as low as possible.</td>
      <td>Up-to-date</td>
      <td>https://github.com/voutcn/megahit</td>
      <td>Sequence Analysis, Assembly, Metagenomics</td>
      <td>megahit</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/megahit</td>
    </tr>
    <tr>
      <td></td>
      <td>megahit_contig2fastg</td>
      <td>1.1.3</td>
      <td>1.2.9</td>
      <td>megahit</td>
      <td>To update</td>
      <td>megahit</td>
      <td>MEGAHIT</td>
      <td>Genome assembly</td>
      <td>Metagenomics, Sequencing, Ecology, Sequence assembly</td>
      <td>A subprogram within the Megahit toolkit for converting contigs to assembly graphs (fastg)</td>
      <td>Single node assembler for large and complex metagenomics NGS reads, such as soil. It makes use of succinct de Bruijn graph to achieve low memory usage, whereas its goal is not to make memory usage as low as possible.</td>
      <td>To update</td>
      <td>https://github.com/voutcn/megahit/blob/master/tools/toolkit.cpp</td>
      <td>Sequence Analysis, Assembly, Metagenomics</td>
      <td>megahit_contig2fastg</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/megahit_contig2fastg</td>
    </tr>
    <tr>
      <td></td>
      <td>megan</td>
      <td>6.21.7</td>
      <td>6.25.9</td>
      <td>megan</td>
      <td>To update</td>
      <td>megan</td>
      <td>MEGAN</td>
      <td>Sequence analysis, Taxonomic classification</td>
      <td>Sequence analysis</td>
      <td>MEGAN Community Edition - Interactive exploration and analysis of large-scale microbiome sequencing data.  MEGAN is a tool for studying the taxonomic content of a set of DNA reads, typically collected in a metagenomics project.In a preprocessing step, a sequence alignment of all reads against a suitable database of reference DNA or proteinsequences must be performed to produce an input file for the program. MEGAN is suitable for DNA reads (metagenomedata), RNA reads (metatranscriptome data), peptide sequences (metaproteomics data) and, using a suitable synonymsfile that maps SILVA ids to taxon ids, for 16S rRNA data (amplicon sequencing).</td>
      <td>Metagenome Analysis Software - MEGAN (MEtaGenome ANalyzer) is a new computer program that allows laptop analysis of large metagenomic datasets. In a preprocessing step, the set of DNA reads (or contigs) is compared against databases of known sequences using BLAST or another comparison tool. MEGAN can then be used to compute and interactively explore the taxonomical content of the dataset, employing the NCBI taxonomy to summarize and order the results.</td>
      <td>To update</td>
      <td>https://github.com/husonlab/megan-ce</td>
      <td>Sequence Analysis</td>
      <td>megan</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/megan</td>
    </tr>
    <tr>
      <td></td>
      <td>meningotype</td>
      <td>0.8.5</td>
      <td>0.8.5</td>
      <td>meningotype</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Assign sequence type to N. meningitidis genome assemblies</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/MDU-PHL/meningotype</td>
      <td>Sequence Analysis</td>
      <td>meningotype</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/meningotype</td>
    </tr>
    <tr>
      <td></td>
      <td>merqury</td>
      <td>1.3</td>
      <td>1.3</td>
      <td>merqury</td>
      <td>Up-to-date</td>
      <td>merqury</td>
      <td>Merqury</td>
      <td>Genome assembly, k-mer counting, Scaffolding, Phasing, De-novo assembly</td>
      <td>Sequence assembly, Whole genome sequencing, Plant biology</td>
      <td>Merqury is a tool for evaluating genomes assemblies based of k-mer operations.</td>
      <td>Reference-free quality, completeness, and phasing assessment for genome assemblies.Evaluate genome assemblies with k-mers and more.Often, genome assembly projects have illumina whole genome sequencing reads available for the assembled individual.Merqury provides a set of tools for this purpose.</td>
      <td>Up-to-date</td>
      <td>https://github.com/marbl/merqury</td>
      <td>Assembly</td>
      <td>merqury</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/merqury</td>
    </tr>
    <tr>
      <td></td>
      <td>meryl</td>
      <td>1.3</td>
      <td>1.3</td>
      <td>merqury</td>
      <td>Up-to-date</td>
      <td>meryl</td>
      <td>Meryl</td>
      <td>k-mer counting</td>
      <td>Whole genome sequencing, Genomics, Sequence analysis, Sequencing</td>
      <td>Meryl a k-mer counter.</td>
      <td>Meryl is a tool for counting and working with sets of k-mers that was originally developed for use in the Celera Assembler and has since been migrated and maintained as part of Canu.</td>
      <td>Up-to-date</td>
      <td>https://github.com/marbl/meryl</td>
      <td>Assembly</td>
      <td>meryl</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/meryl</td>
    </tr>
    <tr>
      <td></td>
      <td>metabat2</td>
      <td>2.15</td>
      <td>2.15</td>
      <td>metabat2</td>
      <td>Up-to-date</td>
      <td>MetaBAT_2</td>
      <td>MetaBAT 2</td>
      <td>Read binning, Sequence assembly, Genome annotation</td>
      <td>Metagenomics, Sequence assembly, Metagenomic sequencing</td>
      <td>MetaBAT2 (Metagenome Binning based on Abundance and Tetranucleotide frequency) is an automated metagenome binningsoftware that integrates empirical probabilistic distances of genome abundance and tetranucleotide frequency.</td>
      <td>an adaptive binning algorithm for robust and efficient genome reconstruction from metagenome assemblies | MetaBAT2 clusters metagenomic contigs into different "bins", each of which should correspond to a putative genome | MetaBAT2 uses nucleotide composition information and source strain abundance (measured by depth-of-coverage by aligning the reads to the contigs) to perform binning</td>
      <td>Up-to-date</td>
      <td>https://bitbucket.org/berkeleylab/metabat/src/master/</td>
      <td>Metagenomics</td>
      <td>metabat2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/metabat2/</td>
    </tr>
    <tr>
      <td></td>
      <td>metaeuk</td>
      <td>5.34c21f2</td>
      <td>6.a5d39d9</td>
      <td>metaeuk</td>
      <td>To update</td>
      <td>MetaEuk</td>
      <td>MetaEuk</td>
      <td>Homology-based gene prediction</td>
      <td>Metagenomics, Gene and protein families</td>
      <td>MetaEuk is a modular toolkit designed for large-scale gene discovery andannotation in eukaryotic metagenomic contigs. Metaeuk combines the fast andsensitive homology search capabilities of MMseqs2 with a dynamic programmingprocedure to recover optimal exons sets. It reduces redundancies in multiplediscoveries of the same gene and resolves conflicting gene predictions onthe same strand.</td>
      <td>MetaEuk - sensitive, high-throughput gene discovery and annotation for large-scale eukaryotic metagenomics</td>
      <td>To update</td>
      <td>https://github.com/soedinglab/metaeuk</td>
      <td>Sequence Analysis, Genome annotation</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/metaeuk</td>
    </tr>
    <tr>
      <td></td>
      <td>metagenomeseq</td>
      <td>1.16.0-0.0.1</td>
      <td>1.43.0</td>
      <td>bioconductor-metagenomeseq</td>
      <td>To update</td>
      <td>metagenomeseq</td>
      <td>metagenomeSeq</td>
      <td>Sequence visualisation, Statistical calculation</td>
      <td>Metagenomics, Sequencing</td>
      <td>metagenomeSeq Normalization</td>
      <td>Designed to determine features (be it Operational Taxanomic Unit (OTU), species, etc.) that are differentially abundant between two or more groups of multiple samples. It is designed to address the effects of both normalization and under-sampling of microbial communities on disease association detection and the testing of feature correlations.</td>
      <td>To update</td>
      <td></td>
      <td>Metagenomics</td>
      <td>metagenomeseq_normalization</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/metagenomeseq</td>
    </tr>
    <tr>
      <td></td>
      <td>metaphlan</td>
      <td>4.0.6</td>
      <td>4.0.6</td>
      <td>metaphlan</td>
      <td>Up-to-date</td>
      <td>metaphlan</td>
      <td>MetaPhlAn</td>
      <td>Nucleic acid sequence analysis, Phylogenetic tree analysis</td>
      <td>Metagenomics, Phylogenomics</td>
      <td>MetaPhlAn for Metagenomic Phylogenetic Analysis</td>
      <td>Computational tool for profiling the composition of microbial communities from metagenomic shotgun sequencing data.</td>
      <td>Up-to-date</td>
      <td>https://github.com/biobakery/MetaPhlAn</td>
      <td>Metagenomics</td>
      <td>metaphlan</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/metaphlan/</td>
    </tr>
    <tr>
      <td></td>
      <td>minia</td>
      <td>3.2.6</td>
      <td>3.2.6</td>
      <td>minia</td>
      <td>Up-to-date</td>
      <td>minia</td>
      <td>Minia</td>
      <td>Genome assembly</td>
      <td>Sequence assembly</td>
      <td>Short-read assembler based on a de Bruijn graph</td>
      <td>Short-read assembler based on a de Bruijn graph, capable of assembling a human genome on a desktop computer in a day.</td>
      <td>Up-to-date</td>
      <td>https://gatb.inria.fr/software/minia/</td>
      <td>Assembly</td>
      <td>minia</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/minia</td>
    </tr>
    <tr>
      <td></td>
      <td>miniasm</td>
      <td>0.3_r179</td>
      <td>0.3</td>
      <td>miniasm</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Miniasm - Ultrafast de novo assembly for long noisy reads (though having no consensus step)</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/lh3/miniasm</td>
      <td>Assembly</td>
      <td>miniasm</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/miniasm</td>
    </tr>
    <tr>
      <td></td>
      <td>miniprot</td>
      <td>0.12</td>
      <td>0.12</td>
      <td>miniprot</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Align a protein sequence against a genome with affine gap penalty, splicing and frameshift.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/lh3/miniprot</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/miniprot</td>
    </tr>
    <tr>
      <td></td>
      <td>mitos</td>
      <td>1.1.5</td>
      <td>2.1.7</td>
      <td>mitos</td>
      <td>To update</td>
      <td>mitos</td>
      <td>MITOS</td>
      <td>Genome annotation</td>
      <td>Zoology, Whole genome sequencing</td>
      <td>de-novo annotation of metazoan mitochondrial genomes</td>
      <td>De novo metazoan mitochondrial genome annotation.</td>
      <td>To update</td>
      <td>http://mitos.bioinf.uni-leipzig.de/</td>
      <td>Sequence Analysis</td>
      <td>mitos</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mitos</td>
    </tr>
    <tr>
      <td></td>
      <td>mlst</td>
      <td>2.22.0</td>
      <td>2.23.0</td>
      <td>mlst</td>
      <td>To update</td>
      <td>mlst</td>
      <td>MLST</td>
      <td>Taxonomic classification</td>
      <td>Immunoproteins, genes and antigens</td>
      <td>Scan contig files against PubMLST typing schemes</td>
      <td>Multi Locus Sequence Typing from an assembled genome or from a set of reads.</td>
      <td>To update</td>
      <td>https://github.com/tseemann/mlst</td>
      <td>Sequence Analysis</td>
      <td>mlst</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mlst</td>
    </tr>
    <tr>
      <td></td>
      <td>mothur</td>
      <td>1.0</td>
      <td>1.48.0</td>
      <td>mothur</td>
      <td>To update</td>
      <td>mothur</td>
      <td>mothur</td>
      <td>DNA barcoding, Sequencing quality control, Sequence clustering, Taxonomic classification, Visualisation, Sequence read processing, Phylogenetic analysis</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Phylogeny</td>
      <td>Mothur wrappers</td>
      <td>Open-source, platform-independent, community-supported software for describing and comparing microbial communities</td>
      <td>To update</td>
      <td>https://www.mothur.org</td>
      <td>Metagenomics</td>
      <td>mothur</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mothur</td>
    </tr>
    <tr>
      <td></td>
      <td>multigsea</td>
      <td>1.8.0</td>
      <td>1.12.0</td>
      <td>bioconductor-multigsea</td>
      <td>To update</td>
      <td>multiGSEA</td>
      <td>multiGSEA</td>
      <td>Gene-set enrichment analysis, Aggregation, Pathway analysis</td>
      <td>Metabolomics, Molecular interactions, pathways and networks, Proteomics, Transcriptomics, Small molecules</td>
      <td>GSEA-based pathway enrichment analysis for multi-omics data</td>
      <td>A GSEA-based pathway enrichment analysis for multi-omics data.multiGSEA: a GSEA-based pathway enrichment analysis for multi-omics data, BMC Bioinformatics 21, 561 (2020).Combining GSEA-based pathway enrichment with multi omics data integration.</td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/devel/bioc/html/multiGSEA.html</td>
      <td>Transcriptomics, Proteomics, Statistics</td>
      <td>multigsea</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/multigsea</td>
    </tr>
    <tr>
      <td></td>
      <td>multiqc</td>
      <td>1.11</td>
      <td>1.19</td>
      <td>multiqc</td>
      <td>To update</td>
      <td>multiqc</td>
      <td>MultiQC</td>
      <td>Validation</td>
      <td>Sequencing, Bioinformatics</td>
      <td>MultiQC aggregates results from bioinformatics analyses across many samples into a single report</td>
      <td>MultiQC aggregates results from multiple bioinformatics analyses across many samples into a single report. It searches a given directory for analysis logs and compiles a HTML report. It's a general use tool, perfect for summarising the output from numerous bioinformatics tools.</td>
      <td>To update</td>
      <td>http://multiqc.info/</td>
      <td>Fastq Manipulation, Statistics, Visualization</td>
      <td>multiqc</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/multiqc</td>
    </tr>
    <tr>
      <td></td>
      <td>mykrobe</td>
      <td>0.10.0</td>
      <td>0.13.0</td>
      <td>mykrobe</td>
      <td>To update</td>
      <td>Mykrobe</td>
      <td>Mykrobe</td>
      <td>Antimicrobial resistance prediction, Variant calling, Genotyping, Sequence trimming</td>
      <td>Whole genome sequencing, Genotype and phenotype, Probes and primers, Genetic variation, Metagenomics</td>
      <td>Antibiotic resistance predictions</td>
      <td>Antibiotic resistance prediction for Mycobacterium tuberculosis from genome sequence data with Mykrobe.Antibiotic resistance prediction in minutes.Table of Contents generated with DocToc.AMR prediction (Mykrobe predictor).Before attempting to install with bioconda, please ensure you have your channels set up as specified in the documentation. If you don't, you may run into issues with an older version of mykrobe being installed</td>
      <td>To update</td>
      <td>https://github.com/Mykrobe-tools/mykrobe</td>
      <td>Sequence Analysis</td>
      <td>mykrobe</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mykrobe</td>
    </tr>
    <tr>
      <td></td>
      <td>nanocompore</td>
      <td>1.0.0rc3.post2</td>
      <td>1.0.4</td>
      <td>nanocompore</td>
      <td>To update</td>
      <td>Nanocompore</td>
      <td>Nanocompore</td>
      <td>Post-translation modification site prediction, PolyA signal detection, Genotyping, k-mer counting</td>
      <td>Functional, regulatory and non-coding RNA, RNA-Seq, Gene transcripts, Transcriptomics, Transcription factors and regulatory sites</td>
      <td>Nanocompore compares 2 ONT nanopore direct RNA sequencing datasets from different experimental conditions expected to have a significant impact on RNA modifications. It is recommended to have at least 2 replicates per condition. For example one can use a control condition with a significantly reduced number of modifications such as a cell line for which a modification writing enzyme was knocked-down or knocked-out. Alternatively, on a smaller scale transcripts of interests could be synthesized in-vitro.</td>
      <td>RNA modifications detection by comparative Nanopore direct RNA sequencing.RNA modifications detection from Nanopore dRNA-Seq data.Nanocompore identifies differences in ONT nanopore sequencing raw signal corresponding to RNA modifications by comparing 2 samples.Analyses performed for the nanocompore paper.Nanocompore compares 2 ONT nanopore direct RNA sequencing datasets from different experimental conditions expected to have a significant impact on RNA modifications. It is recommended to have at least 2 replicates per condition. For example one can use a control condition with a significantly reduced number of modifications such as a cell line for which a modification writing enzyme was knocked-down or knocked-out. Alternatively, on a smaller scale transcripts of interests could be synthesized in-vitro</td>
      <td>To update</td>
      <td>https://nanocompore.rna.rocks/</td>
      <td>Sequence Analysis</td>
      <td>nanocompore</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/nanocompore</td>
    </tr>
    <tr>
      <td></td>
      <td>nanoplot</td>
      <td>1.41.0</td>
      <td>1.42.0</td>
      <td>nanoplot</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Plotting tool for long read sequencing data and alignments</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/wdecoster/NanoPlot</td>
      <td>Visualization</td>
      <td>nanoplot</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/nanoplot/</td>
    </tr>
    <tr>
      <td></td>
      <td>nanopolishcomp</td>
      <td>0.6.11</td>
      <td>0.6.12</td>
      <td>nanopolishcomp</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>NanopolishComp contains 2 modules. Eventalign_collapse collapses the raw file generated by nanopolish eventalign by kmers rather than by event. Freq_meth_calculate methylation frequency at genomic CpG sites from the output of nanopolish call-methylation.</td>
      <td></td>
      <td>To update</td>
      <td>https://a-slide.github.io/NanopolishComp</td>
      <td>Sequence Analysis</td>
      <td>nanopolishcomp</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/nanopolishcomp</td>
    </tr>
    <tr>
      <td></td>
      <td>ncbi_fcs_gx</td>
      <td>0.5.0</td>
      <td>0.5.0</td>
      <td>ncbi-fcs-gx</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FCS-GX detects contamination from foreign organisms in genome sequences using the genome cross-species aligner (GX).</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/ncbi/fcs-gx</td>
      <td>Sequence Analysis</td>
      <td>ncbi_fcs_gx</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ncbi_fcs_gx</td>
    </tr>
    <tr>
      <td></td>
      <td>newick_utils</td>
      <td>1.6+galaxy1</td>
      <td>1.6</td>
      <td>newick_utils</td>
      <td>To update</td>
      <td>newick_utilities</td>
      <td>Newick Utilities</td>
      <td>Phylogenetic tree generation, Phylogenetic tree analysis, Phylogenetic tree reconstruction</td>
      <td>Phylogeny, Genomics, Computer science</td>
      <td>Perform operations on Newick trees</td>
      <td>The Newick Utilities are a set of command-line tools for processing phylogenetic trees. They can process arbitrarily large amounts of data and do not require user interaction, which makes them suitable for automating phylogeny processing tasks.</td>
      <td>To update</td>
      <td>http://cegg.unige.ch/newick_utils</td>
      <td>Visualization, Metagenomics</td>
      <td>newick_utils</td>
      <td>iuc</td>
      <td>https://github.com/tjunier/newick_utils</td>
    </tr>
    <tr>
      <td></td>
      <td>nextclade</td>
      <td>2.7.0</td>
      <td>2.14.0</td>
      <td>nextalign</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Identify differences between your sequences and a reference sequence used by Nextstrain</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/nextstrain/nextclade</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/nextclade</td>
    </tr>
    <tr>
      <td></td>
      <td>nonpareil</td>
      <td>3.1.1</td>
      <td>3.4.1</td>
      <td>nonpareil</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Estimate average coverage in metagenomic datasets</td>
      <td></td>
      <td>To update</td>
      <td>http://nonpareil.readthedocs.io</td>
      <td>Metagenomics</td>
      <td>nonpareil</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/nonpareil</td>
    </tr>
    <tr>
      <td></td>
      <td>nugen_nudup</td>
      <td>2.3.3</td>
      <td>2.3.3</td>
      <td>nudup</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Marks/removes PCR introduced duplicate molecules based on the molecular tagging technology used in NuGEN products.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://nugentechnologies.github.io/nudup/</td>
      <td>SAM, Metagenomics, Sequence Analysis, Transcriptomics</td>
      <td>nugen_nudup</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/nugen_nudup</td>
    </tr>
    <tr>
      <td></td>
      <td>orfipy</td>
      <td>0.0.4</td>
      <td>0.0.4</td>
      <td>orfipy</td>
      <td>Up-to-date</td>
      <td>orfipy</td>
      <td>orfipy</td>
      <td>Coding region prediction, Database search, Transcriptome assembly, De-novo assembly</td>
      <td>Computer science, RNA-Seq, Transcriptomics, Small molecules</td>
      <td>Galaxy wrapper for ORFIPY</td>
      <td>A fast and flexible tool for extracting ORFs.orfipy is a tool written in python/cython to extract ORFs in extremely an fast and flexible manner. Other popular ORF searching tools are OrfM and getorf. Compared to OrfM and getorf, orfipy provides the most options to fine tune ORF searches. orfipy uses multiple CPU cores and is particularly faster for data containing multiple smaller fasta sequences such as de-novo transcriptome assemblies. Please read the preprint here.</td>
      <td>Up-to-date</td>
      <td>https://github.com/urmi-21/orfipy</td>
      <td>Sequence Analysis</td>
      <td>orfipy</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/orfipy</td>
    </tr>
    <tr>
      <td></td>
      <td>orthofinder</td>
      <td>2.5.5</td>
      <td>2.5.5</td>
      <td>orthofinder</td>
      <td>Up-to-date</td>
      <td>OrthoFinder</td>
      <td>OrthoFinder</td>
      <td>Genome comparison, Phylogenetic tree generation (from molecular sequences), Phylogenetic tree analysis, Genome alignment</td>
      <td>Phylogenetics, Phylogenomics, Bioinformatics, Comparative genomics, Sequence analysis</td>
      <td>Accurate inference of orthologous gene groups made easy</td>
      <td>OrthoFinder is a fast, accurate and comprehensive platform for comparative genomics. It finds orthogroups and orthologs, infers rooted gene trees for all orthogroups and identifies all of the gene duplcation events in those gene trees. It also infers a rooted species tree for the species being analysed and maps the gene duplication events from the gene trees to branches in the species tree. OrthoFinder also provides comprehensive statistics for comparative genomic analyses.</td>
      <td>Up-to-date</td>
      <td>https://github.com/davidemms/OrthoFinder</td>
      <td>Phylogenetics, Sequence Analysis</td>
      <td>orthofinder</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/orthofinder</td>
    </tr>
    <tr>
      <td></td>
      <td>pharokka</td>
      <td>1.3.2</td>
      <td></td>
      <td>\n                pharokka\n</td>
      <td>To update</td>
      <td>\n                pharokka\n</td>
      <td></td>
      <td></td>
      <td></td>
      <td>rapid standardised annotation tool for bacteriophage genomes and metagenomes</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/gbouras13/pharokka</td>
      <td>Genome annotation</td>
      <td>pharokka</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pharokka</td>
    </tr>
    <tr>
      <td></td>
      <td>phyloseq</td>
      <td>1.38.0</td>
      <td>1.46.0</td>
      <td>bioconductor-phyloseq</td>
      <td>To update</td>
      <td>phyloseq</td>
      <td>phyloseq</td>
      <td>Deposition, Analysis, Visualisation</td>
      <td>Microbiology, Sequence analysis, Metagenomics</td>
      <td>Handling and analysis of high-throughput microbiome census data</td>
      <td>Provides a set of classes and tools to facilitate the import, storage, analysis, and graphical display of microbiome census data.</td>
      <td>To update</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/phyloseq.html</td>
      <td>Metagenomics</td>
      <td>phyloseq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/phyloseq</td>
    </tr>
    <tr>
      <td></td>
      <td>phyml</td>
      <td>3.3.20220408</td>
      <td>3.3.20220408</td>
      <td>phyml</td>
      <td>Up-to-date</td>
      <td>phyml</td>
      <td>PhyML</td>
      <td>Phylogenetic tree generation (maximum likelihood and Bayesian methods)</td>
      <td>Phylogenetics, Bioinformatics, Phylogenetics</td>
      <td>PhyML is a phylogeny software based on the maximum-likelihood principle.</td>
      <td>Phylogenetic estimation software using Maximum Likelihood</td>
      <td>Up-to-date</td>
      <td>http://www.atgc-montpellier.fr/phyml/</td>
      <td>Phylogenetics</td>
      <td>phyml</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/phyml</td>
    </tr>
    <tr>
      <td></td>
      <td>picrust</td>
      <td>1.1.1</td>
      <td>1.1.4</td>
      <td>picrust</td>
      <td>To update</td>
      <td>picrust</td>
      <td>PICRUSt</td>
      <td>Phylogenetic reconstruction, Expression analysis, Genome annotation, DNA barcoding</td>
      <td>Metagenomics, Microbial ecology, Functional, regulatory and non-coding RNA, Metagenomic sequencing</td>
      <td>PICRUSt wrappers</td>
      <td>PICRUSt (Phylogenetic Investigation of Communities by Reconstruction of Unobserved States) is a bioinformatics software package designed to predict metagenome functional content from marker gene (e.g., 16S rRNA) surveys and full genomes.</td>
      <td>To update</td>
      <td>https://picrust.github.io/picrust/</td>
      <td>Metagenomics</td>
      <td>picrust</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/picrust</td>
    </tr>
    <tr>
      <td></td>
      <td>picrust2</td>
      <td>2.5.1</td>
      <td>2.5.2</td>
      <td>picrust2</td>
      <td>To update</td>
      <td>picrust2</td>
      <td>PICRUSt2</td>
      <td>Phylogenetic reconstruction, Expression analysis, Rarefaction, Pathway analysis</td>
      <td>Metagenomics, Microbiology, Phylogenetics, Metagenomic sequencing</td>
      <td>PICRUSt2: Phylogenetic Investigation of Communities by Reconstruction of Unobserved States</td>
      <td>PICRUSt2 (Phylogenetic Investigation of Communities by Reconstruction of Unobserved States) is a software for predicting functional abundances based only on marker gene sequences.</td>
      <td>To update</td>
      <td>https://github.com/picrust/picrust2/wiki</td>
      <td>Metagenomics</td>
      <td>picrust2</td>
      <td>iuc</td>
      <td>https://github.com/picrust/picrust2</td>
    </tr>
    <tr>
      <td></td>
      <td>plasflow</td>
      <td>1.1.0</td>
      <td>1.1.0</td>
      <td>plasflow</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>PlasFlow - Prediction of plasmid sequences in metagenomic contigs.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/smaegol/PlasFlow</td>
      <td>Sequence Analysis</td>
      <td>plasflow</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/plasflow</td>
    </tr>
    <tr>
      <td></td>
      <td>plasmidfinder</td>
      <td>2.1.6</td>
      <td>2.1.6</td>
      <td>plasmidfinder</td>
      <td>Up-to-date</td>
      <td>PlasmidFinder</td>
      <td>PlasmidFinder</td>
      <td>Genome assembly, Scaffolding, Multilocus sequence typing</td>
      <td>Whole genome sequencing, Sequence assembly, Mapping, Probes and primers</td>
      <td>"PlasmidFinder provides the detection of replicons in the WGSand assigns the plasmids under study to lineages that trace backthe information to the existing knowledge on Inc groups and suggestspossible reference plasmids for each lineage"</td>
      <td>PlasmidFinder is a tool for the identification and typing of Plasmid Replicons in Whole-Genome Sequencing (WGS).</td>
      <td>Up-to-date</td>
      <td>https://bitbucket.org/genomicepidemiology/plasmidfinder/src/master/</td>
      <td>Sequence Analysis</td>
      <td>plasmidfinder</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/plasmidfinder</td>
    </tr>
    <tr>
      <td></td>
      <td>prokka</td>
      <td>1.14.6</td>
      <td>1.14.6</td>
      <td>prokka</td>
      <td>Up-to-date</td>
      <td>prokka</td>
      <td>Prokka</td>
      <td>Gene prediction, Coding region prediction, Genome annotation</td>
      <td>Genomics, Model organisms, Virology</td>
      <td>Rapid annotation of prokaryotic genomes</td>
      <td>Software tool to annotate bacterial, archaeal and viral genomes quickly and produce standards-compliant output files.</td>
      <td>Up-to-date</td>
      <td>http://github.com/tseemann/prokka</td>
      <td>Sequence Analysis</td>
      <td>prokka</td>
      <td>crs4</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/prokka/</td>
    </tr>
    <tr>
      <td></td>
      <td>proteinortho</td>
      <td>6.3.1</td>
      <td>6.3.1</td>
      <td>proteinortho</td>
      <td>Up-to-date</td>
      <td>proteinortho</td>
      <td>Proteinortho</td>
      <td>Homology-based gene prediction</td>
      <td>Phylogeny</td>
      <td>Proteinortho is a tool to detect orthologous proteins/genes within different species.</td>
      <td>Proteinortho is a tool to detect orthologous genes within different species</td>
      <td>Up-to-date</td>
      <td>https://gitlab.com/paulklemm_PHD/proteinortho</td>
      <td>Proteomics</td>
      <td>proteinortho</td>
      <td>iuc</td>
      <td>https://gitlab.com/paulklemm_PHD/proteinortho</td>
    </tr>
    <tr>
      <td></td>
      <td>pycoqc</td>
      <td>2.5.2</td>
      <td>2.5.2</td>
      <td>pycoqc</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>QC metrics for ONT Basecalling</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/tleonardi/pycoQC</td>
      <td>Nanopore</td>
      <td>pycoqc</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pycoqc</td>
    </tr>
    <tr>
      <td></td>
      <td>pygenometracks</td>
      <td>3.8</td>
      <td>3.8</td>
      <td>pygenometracks</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>pyGenomeTracks: Standalone program and library to plot beautiful genome browser tracks.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/deeptools/pyGenomeTracks</td>
      <td>Visualization</td>
      <td>pygenometracks</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pygenometracks</td>
    </tr>
    <tr>
      <td></td>
      <td>qiime_add_on</td>
      <td></td>
      <td>1.9.1</td>
      <td>qiime</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>QIIME to perform microbial community analysis</td>
      <td></td>
      <td>To update</td>
      <td>http://www.qiime.org</td>
      <td>Metagenomics</td>
      <td>qiime</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/qiime/</td>
    </tr>
    <tr>
      <td></td>
      <td>qiime_core</td>
      <td></td>
      <td>1.9.1</td>
      <td>qiime</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>QIIME to perform microbial community analysis</td>
      <td></td>
      <td>To update</td>
      <td>http://www.qiime.org</td>
      <td>Metagenomics</td>
      <td>qiime</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/qiime/</td>
    </tr>
    <tr>
      <td></td>
      <td>qualimap</td>
      <td>2.2.2d</td>
      <td>2.3</td>
      <td>qualimap</td>
      <td>To update</td>
      <td>qualimap</td>
      <td>QualiMap</td>
      <td>Sequencing quality control</td>
      <td>Data quality management</td>
      <td></td>
      <td>Platform-independent application written in Java and R that provides both a Graphical User Inteface (GUI) and a command-line interface to facilitate the quality control of alignment sequencing data.</td>
      <td>To update</td>
      <td>http://qualimap.bioinfo.cipf.es/</td>
      <td>Sequence Analysis, Transcriptomics, SAM</td>
      <td>qualimap</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/qualimap</td>
    </tr>
    <tr>
      <td></td>
      <td>quast</td>
      <td>5.2.0</td>
      <td>5.2.0</td>
      <td>quast</td>
      <td>Up-to-date</td>
      <td>quast</td>
      <td>QUAST</td>
      <td>Visualisation, Sequence assembly validation</td>
      <td>Sequence assembly</td>
      <td>Quast (Quality ASsessment Tool) evaluates genome assemblies.</td>
      <td>QUAST stands for QUality ASsessment Tool.  It evaluates a quality of genome assemblies by computing various metrics and providing nice reports.</td>
      <td>Up-to-date</td>
      <td>http://quast.bioinf.spbau.ru/</td>
      <td>Assembly</td>
      <td>quast</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/quast</td>
    </tr>
    <tr>
      <td></td>
      <td>quickmerge</td>
      <td>0.3</td>
      <td>0.3</td>
      <td>quickmerge</td>
      <td>Up-to-date</td>
      <td>quickmerge</td>
      <td>quickmerge</td>
      <td>Genome assembly, Scaffolding, De-novo assembly, Genotyping</td>
      <td>Structural variation, Sequence assembly, DNA polymorphism, Whole genome sequencing, Genotype and phenotype</td>
      <td>Merge long-read and hybrid assemblies to increase contiguity</td>
      <td>Quickmerge is a program that uses complementary information from genomes assembled with long reads in order to improve contiguity, and works with assemblies derived from both Pacific Biosciences or Oxford Nanopore. Quickmerge will even work with hybrid assemblies made by combining long reads and Illumina short reads.</td>
      <td>Up-to-date</td>
      <td>https://github.com/mahulchak/quickmerge</td>
      <td>Assembly</td>
      <td>quickmerge</td>
      <td>galaxy-australia</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/quickmerge</td>
    </tr>
    <tr>
      <td></td>
      <td>raxml</td>
      <td>8.2.12</td>
      <td>8.2.13</td>
      <td>raxml</td>
      <td>To update</td>
      <td>raxml</td>
      <td>RAxML</td>
      <td>Sequence analysis, Phylogenetic tree analysis</td>
      <td>Phylogenetics, Sequence analysis</td>
      <td>RAxML - A Maximum Likelihood based phylogenetic inference</td>
      <td>A tool for Phylogenetic Analysis and Post-Analysis of Large Phylogenies.</td>
      <td>To update</td>
      <td>http://www.exelixis-lab.org/web/software/raxml/</td>
      <td>Phylogenetics</td>
      <td>raxml</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/raxml</td>
    </tr>
    <tr>
      <td></td>
      <td>read_it_and_keep</td>
      <td>0.2.2</td>
      <td>0.3.0</td>
      <td>read-it-and-keep</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Rapid decontamination of SARS-CoV-2 sequencing reads</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/GenomePathogenAnalysisService/read-it-and-keep</td>
      <td>Sequence Analysis</td>
      <td>read_it_and_keep</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/read-it-and-keep</td>
    </tr>
    <tr>
      <td></td>
      <td>recentrifuge</td>
      <td>1.13.2</td>
      <td>1.14.0</td>
      <td>recentrifuge</td>
      <td>To update</td>
      <td>Recentrifuge</td>
      <td>Recentrifuge</td>
      <td>Taxonomic classification, Expression analysis, Cross-assembly</td>
      <td>Metagenomics, Microbial ecology, Metagenomic sequencing</td>
      <td>"With Recentrifuge, researchers can analyze results from taxonomic classifiers using interactive charts with emphasis on the confidence level of the classifications.In addition to contamination-subtracted samples.Recentrifuge provides shared and exclusive taxa per sample,thus enabling robust contamination removal and comparative analysis in environmental and clinical metagenomics."</td>
      <td>Robust comparative analysis and contamination removal for metagenomics.</td>
      <td>To update</td>
      <td>https://github.com/khyox/recentrifuge</td>
      <td>Metagenomics</td>
      <td>recentrifuge</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/recentrifuge</td>
    </tr>
    <tr>
      <td></td>
      <td>roary</td>
      <td>3.13.0</td>
      <td>3.13.0</td>
      <td>roary</td>
      <td>Up-to-date</td>
      <td>roary</td>
      <td>Roary</td>
      <td>Genome assembly</td>
      <td>DNA, Genomics, Mapping</td>
      <td>Roary the pangenome pipeline</td>
      <td>A high speed stand alone pan genome pipeline, which takes annotated assemblies in GFF3 format (produced by Prokka (Seemann, 2014)) and calculates the pan genome.</td>
      <td>Up-to-date</td>
      <td>https://sanger-pathogens.github.io/Roary/</td>
      <td>Sequence Analysis</td>
      <td>roary</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/roary</td>
    </tr>
    <tr>
      <td></td>
      <td>rseqc</td>
      <td>5.0.1</td>
      <td>5.0.3</td>
      <td>rseqc</td>
      <td>To update</td>
      <td>rseqc</td>
      <td>RSeQC</td>
      <td>Data handling</td>
      <td>Sequencing</td>
      <td>an RNA-seq quality control package</td>
      <td>Provides a number of useful modules that can comprehensively evaluate high throughput sequence data especially RNA-seq data. Some basic modules quickly inspect sequence quality, nucleotide composition bias, PCR bias and GC bias, while RNA-seq specific modules evaluate sequencing saturation, mapped reads distribution, coverage uniformity, strand specificity, transcript level RNA integrity etc.</td>
      <td>To update</td>
      <td>https://code.google.com/p/rseqc/</td>
      <td>Convert Formats, Sequence Analysis, RNA, Transcriptomics, Visualization</td>
      <td>rseqc</td>
      <td>nilesh</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/rseqc</td>
    </tr>
    <tr>
      <td></td>
      <td>sarscov2formatter</td>
      <td>1.0</td>
      <td>1.0</td>
      <td>sarscov2formatter</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>sarscov2formatter custom script</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/nickeener/sarscov2formatter</td>
      <td>Sequence Analysis</td>
      <td>sarscov2formatter</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/sarscov2formatter</td>
    </tr>
    <tr>
      <td></td>
      <td>sarscov2summary</td>
      <td>0.1</td>
      <td>0.5</td>
      <td>sarscov2summary</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>sarscov2summary custom script</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/nickeener/sarscov2summary</td>
      <td>Sequence Analysis</td>
      <td>sarscov2summary</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/sarscov2summary</td>
    </tr>
    <tr>
      <td></td>
      <td>scoary</td>
      <td>1.6.16</td>
      <td>1.6.16</td>
      <td>scoary</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Scoary calculates the assocations between all genes in the accessory genome and the traits.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/AdmiralenOla/Scoary</td>
      <td>Metagenomics</td>
      <td>scoary</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/scoary</td>
    </tr>
    <tr>
      <td></td>
      <td>semibin</td>
      <td>2.0.2</td>
      <td>2.0.2</td>
      <td>semibin</td>
      <td>Up-to-date</td>
      <td>semibin</td>
      <td>SemiBin</td>
      <td>Sequence assembly, Read binning</td>
      <td>Metagenomics, Machine learning, Microbial ecology, Sequence assembly</td>
      <td>SemiBin: Semi-supervised Metagenomic Binning Using Siamese Neural Networks</td>
      <td>Command tool for metagenomic binning with semi-supervised deep learning using information from reference genomes.</td>
      <td>Up-to-date</td>
      <td>https://semibin.readthedocs.io/en/latest/</td>
      <td>Metagenomics</td>
      <td>semibin</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/semibin</td>
    </tr>
    <tr>
      <td></td>
      <td>seqkit</td>
      <td>2.3.1</td>
      <td>2.7.0</td>
      <td>seqkit</td>
      <td>To update</td>
      <td>seqkit</td>
      <td>seqkit</td>
      <td>DNA transcription, Sequence trimming, DNA translation, Sequence conversion</td>
      <td>Database management, Sequence analysis</td>
      <td>A cross-platform and ultrafast toolkit for FASTA/Q file manipulation</td>
      <td>FASTA and FASTQ are basic and ubiquitous formats for storing nucleotide and protein sequences. Common manipulations of FASTA/Q file include converting, searching, filtering, deduplication, splitting, shuffling, and sampling. Existing tools only implement some of these manipulations, and not particularly efficiently, and some are only available for certain operating systems. Furthermore, the complicated installation process of required packages and running environments can render these programs less user friendly. SeqKit demonstrates competitive performance in execution time and memory usage compared to similar tools. The efficiency and usability of SeqKit enable researchers to rapidly accomplish common FASTA/Q file manipulations.</td>
      <td>To update</td>
      <td>https://bioinf.shenwei.me/seqkit/</td>
      <td>Sequence Analysis</td>
      <td>seqkit</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/seqkit</td>
    </tr>
    <tr>
      <td></td>
      <td>seqsero2</td>
      <td>1.2.1</td>
      <td>1.2.1</td>
      <td>seqsero2</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Salmonella serotype prediction from genome sequencing data</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/denglab/SeqSero2</td>
      <td>Sequence Analysis</td>
      <td>seqsero2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/seqsero2</td>
    </tr>
    <tr>
      <td></td>
      <td>shovill</td>
      <td>1.1.0</td>
      <td>1.1.0</td>
      <td>shovill</td>
      <td>Up-to-date</td>
      <td>shovill</td>
      <td>shovill</td>
      <td>Genome assembly</td>
      <td>Genomics, Microbiology, Sequence assembly</td>
      <td>Faster de novo assembly pipeline based around Spades</td>
      <td>Shovill is a pipeline for assembly of bacterial isolate genomes from Illumina paired-end reads.  Shovill uses SPAdes at its core, but alters the steps before and after the primary assembly step to get similar results in less time. Shovill also supports other assemblers like SKESA, Velvet and Megahit, so you can take advantage of the pre- and post-processing the Shovill provides with those too.</td>
      <td>Up-to-date</td>
      <td>https://github.com/tseemann/shovill</td>
      <td>Assembly</td>
      <td>shovill</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/shovill</td>
    </tr>
    <tr>
      <td></td>
      <td>smallgenomeutilities</td>
      <td>0.4.0</td>
      <td>0.4.0</td>
      <td>smallgenomeutilities</td>
      <td>Up-to-date</td>
      <td>v-pipe</td>
      <td>V-pipe</td>
      <td>Read pre-processing, Sequence alignment, Genetic variation analysis</td>
      <td>Genomics, Population genetics, Workflows, Virology, Sequencing</td>
      <td>Set of utilities for manipulating small viral genome data.</td>
      <td>Bioinformatics pipeline for the analysis of next-generation sequencing data derived from intra-host viral populations.</td>
      <td>Up-to-date</td>
      <td>https://github.com/cbg-ethz/smallgenomeutilities</td>
      <td>Sequence Analysis</td>
      <td>smallgenomeutilities</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/smallgenomeutilities</td>
    </tr>
    <tr>
      <td></td>
      <td>snap</td>
      <td>2013_11_29</td>
      <td>2013_11_29</td>
      <td>snap</td>
      <td>Up-to-date</td>
      <td>snap</td>
      <td>SNAP</td>
      <td>Gene prediction</td>
      <td>DNA, DNA polymorphism, Genetics</td>
      <td>SNAP is a general purpose gene finding program suitable for both eukaryotic and prokaryotic genomes.</td>
      <td>The Semi-HMM-based Nucleic Acid Parser is a gene prediction tool.</td>
      <td>Up-to-date</td>
      <td>https://github.com/KorfLab/SNAP</td>
      <td>Sequence Analysis</td>
      <td>snap</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/snap</td>
    </tr>
    <tr>
      <td></td>
      <td>snippy</td>
      <td></td>
      <td>4.6.0</td>
      <td>snippy</td>
      <td>To update</td>
      <td>snippy</td>
      <td>snippy</td>
      <td>Phylogenetic tree visualisation, Phylogenetic tree generation, Variant calling</td>
      <td>Genomics, Model organisms, DNA polymorphism, Phylogenetics</td>
      <td>Contains the snippy tool for characterising microbial snps</td>
      <td>Rapid haploid variant calling and core SNP phylogeny generation.</td>
      <td>To update</td>
      <td>https://github.com/tseemann/snippy</td>
      <td>Sequence Analysis</td>
      <td>snippy</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/snippy</td>
    </tr>
    <tr>
      <td></td>
      <td>sonneityping</td>
      <td>20210201</td>
      <td>20210201</td>
      <td>sonneityping</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Scripts for parsing Mykrobe predict results for Shigella sonnei.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/katholt/sonneityping</td>
      <td>Sequence Analysis</td>
      <td>sonneityping</td>
      <td>iuc</td>
      <td>https://github.com/katholt/sonneityping</td>
    </tr>
    <tr>
      <td></td>
      <td>spades</td>
      <td>3.15.5</td>
      <td>3.15.5</td>
      <td>spades</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SPAdes is an assembly toolkit containing various assembly pipelines. It implements the following 4 stages: assembly graph construction, k-bimer adjustment, construction of paired assembly graph and contig construction.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/ablab/spades</td>
      <td>Assembly, RNA, Metagenomics</td>
      <td>spades</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/spades</td>
    </tr>
    <tr>
      <td></td>
      <td>spotyping</td>
      <td>2.1</td>
      <td>2.1</td>
      <td>spotyping</td>
      <td>Up-to-date</td>
      <td>spotyping</td>
      <td>SpoTyping</td>
      <td>Variant pattern analysis</td>
      <td>Microbiology, Sequencing, Sequence composition, complexity and repeats, Genetic variation</td>
      <td>SpoTyping allows fast and accurate in silico Mycobacterium spoligotyping from sequence reads</td>
      <td>Fast and accurate in silico Mycobacterium spoligotyping from sequence reads.</td>
      <td>Up-to-date</td>
      <td>https://github.com/xiaeryu/SpoTyping-v2.0</td>
      <td>Sequence Analysis</td>
      <td>spotyping</td>
      <td>iuc</td>
      <td>https://github.com/xiaeryu/SpoTyping-v2.0/tree/master/SpoTyping-v2.0-commandLine</td>
    </tr>
    <tr>
      <td></td>
      <td>srst2</td>
      <td>0.2.0</td>
      <td>1.19.2</td>
      <td>samtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SRST2 Short Read Sequence Typing for Bacterial Pathogens</td>
      <td></td>
      <td>To update</td>
      <td>http://katholt.github.io/srst2/</td>
      <td>Metagenomics</td>
      <td>srst2</td>
      <td>iuc</td>
      <td>https://github.com/katholt/srst2</td>
    </tr>
    <tr>
      <td></td>
      <td>structure</td>
      <td>2.3.4</td>
      <td>2.3.4</td>
      <td>structure</td>
      <td>Up-to-date</td>
      <td>structure</td>
      <td>Structure</td>
      <td>Genetic variation analysis</td>
      <td>Population genetics</td>
      <td>for using multi-locus genotype data to investigate population structure.</td>
      <td>The program structureis a free software package for using multi-locus genotype data to investigate population structure. Its uses include inferring the presence of distinct populations, assigning individuals to populations, studying hybrid zones, identifying migrants and admixed individuals, and estimating population allele frequencies in situations where many individuals are migrants or admixed.</td>
      <td>Up-to-date</td>
      <td></td>
      <td>Phylogenetics, Variant Analysis</td>
      <td>structure</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/structure</td>
    </tr>
    <tr>
      <td></td>
      <td>taxonomy_krona_chart</td>
      <td>2.7.1+galaxy0</td>
      <td>2.8.1</td>
      <td>krona</td>
      <td>To update</td>
      <td>krona</td>
      <td>Krona</td>
      <td>Visualisation</td>
      <td>Metagenomics</td>
      <td>Krona pie chart from taxonomic profile</td>
      <td>Krona creates interactive HTML5 charts of hierarchical data (such as taxonomic abundance in a metagenome).</td>
      <td>To update</td>
      <td>http://sourceforge.net/projects/krona/</td>
      <td>Assembly</td>
      <td>taxonomy_krona_chart</td>
      <td>crs4</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/taxonomy_krona_chart</td>
    </tr>
    <tr>
      <td></td>
      <td>tb-profiler</td>
      <td>4.4.1</td>
      <td>5.0.1</td>
      <td>tb-profiler</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Processes M. tuberculosis sequence data to infer strain type and identify known drug resistance markers.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/jodyphelan/TBProfiler</td>
      <td>Sequence Analysis</td>
      <td>tbprofiler</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/tb-profiler</td>
    </tr>
    <tr>
      <td></td>
      <td>transtermhp</td>
      <td></td>
      <td>2.09</td>
      <td>transtermhp</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Finds rho-independent transcription terminators in bacterial genomes</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>transtermhp</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/transtermhp</td>
    </tr>
    <tr>
      <td></td>
      <td>unicycler</td>
      <td>0.5.0</td>
      <td>0.5.0</td>
      <td>unicycler</td>
      <td>Up-to-date</td>
      <td>unicycler</td>
      <td>Unicycler</td>
      <td>Genome assembly, Aggregation</td>
      <td>Microbiology, Genomics, Sequencing, Sequence assembly</td>
      <td>Unicycler is a hybrid assembly pipeline for bacterial genomes.</td>
      <td>A tool for assembling bacterial genomes from a combination of short (2nd generation) and long (3rd generation) sequencing reads.</td>
      <td>Up-to-date</td>
      <td>https://github.com/rrwick/Unicycler</td>
      <td>Assembly</td>
      <td>unicycler</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/unicycler</td>
    </tr>
    <tr>
      <td></td>
      <td>usher</td>
      <td>0.2.1</td>
      <td>0.6.3</td>
      <td>usher</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>UShER toolkit wrappers</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/yatisht/usher</td>
      <td>Phylogenetics</td>
      <td>usher</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/usher</td>
    </tr>
    <tr>
      <td></td>
      <td>valet</td>
      <td></td>
      <td>1.0</td>
      <td>valet</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A pipeline for detecting mis-assemblies in metagenomic assemblies.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/marbl/VALET</td>
      <td>Metagenomics</td>
      <td>valet</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/valet</td>
    </tr>
    <tr>
      <td></td>
      <td>vapor</td>
      <td>1.0.2</td>
      <td>1.0.2</td>
      <td>vapor</td>
      <td>Up-to-date</td>
      <td>vapor</td>
      <td>VAPOR</td>
      <td>Data retrieval, De-novo assembly, Read mapping</td>
      <td>Whole genome sequencing, Mapping, Sequence assembly</td>
      <td>Classify Influenza samples from raw short read sequence data</td>
      <td>VAPOR is a tool for classification of Influenza samples from raw short read sequence data for downstream bioinformatics analysis. VAPOR is provided with a fasta file of full-length sequences (&gt; 20,000) for a given segment, a set of reads, and attempts to retrieve a reference that is closest to the sample strain.</td>
      <td>Up-to-date</td>
      <td>https://github.com/connor-lab/vapor</td>
      <td>Sequence Analysis</td>
      <td>vapor</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/vapor</td>
    </tr>
    <tr>
      <td></td>
      <td>varvamp</td>
      <td>1.1.1</td>
      <td>1.1.1</td>
      <td>varvamp</td>
      <td>Up-to-date</td>
      <td>varvamp</td>
      <td>varVAMP</td>
      <td>PCR primer design</td>
      <td>Virology</td>
      <td>Variable VirusAMPlicons (varVAMP) is a tool to design primers for highly diverse viruses</td>
      <td>variable VirusAMPlicons (varVAMP) is a tool to design primers for highly diverse viruses. The input is an alignment of your viral (full-genome) sequences.</td>
      <td>Up-to-date</td>
      <td>https://github.com/jonas-fuchs/varVAMP/</td>
      <td>Sequence Analysis</td>
      <td>varvamp</td>
      <td>iuc</td>
      <td>https://github.com/jonas-fuchs/varVAMP</td>
    </tr>
    <tr>
      <td></td>
      <td>vegan</td>
      <td>2.4-3</td>
      <td>2.3_4</td>
      <td>r-vegan</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://cran.r-project.org/package=vegan</td>
      <td>Metagenomics</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/vegan/</td>
    </tr>
    <tr>
      <td></td>
      <td>velvet</td>
      <td></td>
      <td>1.2.10</td>
      <td>velvet</td>
      <td>To update</td>
      <td>velvet</td>
      <td>Velvet</td>
      <td>Formatting, De-novo assembly</td>
      <td>Sequence assembly</td>
      <td>de novo genomic assembler specially designed for short read sequencing technologies</td>
      <td>A de novo genomic assembler specially designed for short read sequencing technologies, such as Solexa or 454 or SOLiD.</td>
      <td>To update</td>
      <td>https://www.ebi.ac.uk/~zerbino/velvet/</td>
      <td>Assembly</td>
      <td>velvet</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/velvet</td>
    </tr>
    <tr>
      <td></td>
      <td>velvet_optimiser</td>
      <td>2.2.6+galaxy2</td>
      <td>1.2.10</td>
      <td>velvet</td>
      <td>To update</td>
      <td>velvetoptimiser</td>
      <td>VelvetOptimiser</td>
      <td>Optimisation and refinement, Sequence assembly</td>
      <td>Genomics, Sequence assembly</td>
      <td>Automatically optimize Velvet assemblies</td>
      <td>This tool is designed to run as a wrapper script for the Velvet assembler (Daniel Zerbino, EBI UK) and to assist with optimising the assembly.</td>
      <td>To update</td>
      <td></td>
      <td>Assembly</td>
      <td>velvetoptimiser</td>
      <td>simon-gladman</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/velvetoptimiser</td>
    </tr>
    <tr>
      <td></td>
      <td>vsearch</td>
      <td>2.8.3</td>
      <td>2.27.0</td>
      <td>vsearch</td>
      <td>To update</td>
      <td>vsearch</td>
      <td>VSEARCH</td>
      <td>DNA mapping, Chimera detection</td>
      <td>Metagenomics, Sequence analysis</td>
      <td>VSEARCH including searching, clustering, chimera detection, dereplication, sorting, masking and shuffling of sequences.</td>
      <td>High-throughput search and clustering sequence analysis tool. It supports de novo and reference based chimera detection, clustering, full-length and prefix dereplication, reverse complementation, masking, all-vs-all pairwise global alignment, exact and global alignment searching, shuffling, subsampling and sorting. It also supports FASTQ file analysis, filtering and conversion.</td>
      <td>To update</td>
      <td>https://github.com/torognes/vsearch</td>
      <td>Sequence Analysis</td>
      <td>vsearch</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/vsearch</td>
    </tr>
    <tr>
      <td></td>
      <td>windowmasker</td>
      <td>1.0</td>
      <td>2.15.0</td>
      <td>blast</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Identify repetitive regions using WindowMasker</td>
      <td></td>
      <td>To update</td>
      <td>https://www.ncbi.nlm.nih.gov/IEB/ToolBox/CPP_DOC/lxr/source/src/app/winmasker/</td>
      <td>Sequence Analysis</td>
      <td>windowmasker</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/windowmasker/</td>
    </tr>
    <tr>
      <td></td>
      <td>bamtools</td>
      <td>2.5.2</td>
      <td>2.5.2</td>
      <td>bamtools</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Operate on and transform BAM datasets in various ways using bamtools</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/pezmaster31/bamtools</td>
      <td>Sequence Analysis, SAM</td>
      <td>bamtools</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/bamtools/bamtools</td>
    </tr>
    <tr>
      <td></td>
      <td>biotradis</td>
      <td>1.4.5</td>
      <td>1.4.5</td>
      <td>biotradis</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Bio-Tradis is a tool suite dedicated to essentiality analyses with TraDis data.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://www.sanger.ac.uk/science/tools/bio-tradis</td>
      <td>Genome annotation</td>
      <td>biotradis</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/biotradis</td>
    </tr>
    <tr>
      <td></td>
      <td>kraken</td>
      <td></td>
      <td>1.1.1</td>
      <td>kraken</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Kraken is a system for assigning taxonomic labels to short DNAsequences, usually obtained through metagenomic studies. Previous attempts by otherbioinformatics software to accomplish this task have often used sequence alignmentor machine learning techniques that were quite slow, leading to the developmentof less sensitive but much faster abundance estimation programs. Kraken aims toachieve high sensitivity and high speed by utilizing exact alignments of k-mersand a novel classification algorithm.</td>
      <td></td>
      <td>To update</td>
      <td>http://ccb.jhu.edu/software/kraken/</td>
      <td>Metagenomics</td>
      <td>kraken</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tool_collections/kraken/</td>
    </tr>
    <tr>
      <td></td>
      <td>kraken2</td>
      <td>2.1.1</td>
      <td>2.1.3</td>
      <td>kraken2</td>
      <td>To update</td>
      <td>kraken2</td>
      <td>kraken2</td>
      <td>Taxonomic classification</td>
      <td>Taxonomy, Metagenomics</td>
      <td>Kraken2 for taxonomic designation.</td>
      <td>Kraken 2 is the newest version of Kraken, a taxonomic classification system using exact k-mer matches to achieve high accuracy and fast classification speeds. This classifier matches each k-mer within a query sequence to the lowest common ancestor (LCA) of all genomes containing the given k-mer. The k-mer assignments inform the classification algorithm.</td>
      <td>To update</td>
      <td>http://ccb.jhu.edu/software/kraken/</td>
      <td>Metagenomics</td>
      <td>kraken2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tool_collections/kraken2/kraken2/</td>
    </tr>
    <tr>
      <td></td>
      <td>eggnog_mapper</td>
      <td>2.1.8</td>
      <td>2.1.12</td>
      <td>eggnog-mapper</td>
      <td>To update</td>
      <td>eggnog-mapper-v2</td>
      <td>eggNOG-mapper v2</td>
      <td>Homology-based gene prediction, Genome annotation, Fold recognition, Information extraction, Query and retrieval</td>
      <td>Metagenomics, Phylogeny, Transcriptomics, Workflows, Sequence analysis</td>
      <td>eggnog-mapper fast functional annotation of novel sequences</td>
      <td>EggNOG-mapper is a tool for fast functional annotation of novel sequences. It uses precomputed orthologous groups and phylogenies from the eggNOG database (http://eggnog5.embl.de) to transfer functional information from fine-grained orthologs only.</td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>eggnog_mapper</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/eggnog_mapper/eggnog_mapper</td>
    </tr>
    <tr>
      <td></td>
      <td>metagene_annotator</td>
      <td>1.0.0</td>
      <td>1.0</td>
      <td>metagene_annotator</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MetaGeneAnnotator gene-finding program for prokaryote and phage</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>metagene_annotator</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/metagene_annotator</td>
    </tr>
    <tr>
      <td></td>
      <td>ez_histograms</td>
      <td>3.4.4</td>
      <td>2.2.1</td>
      <td>r-ggplot2</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ggplot2 histograms and density plots</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/tidyverse/ggplot2</td>
      <td>Visualization, Statistics</td>
      <td>ez_histograms</td>
      <td>artbio</td>
      <td>https://github.com/artbio/tools-artbio/tree/main/tools/ez_histograms</td>
    </tr>
    <tr>
      <td></td>
      <td>Geom_mean_workflow</td>
      <td>0.1.0+galaxy0</td>
      <td></td>
      <td>r-base</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tools to compute The evolution of the total volume of very large trees, standing dead wood and dead wood on the ground on an area and the rate of devolution of the volume of wood favorable to biodiversity by large ecological regions (France).</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/PaulineSGN/Galaxy_tool_moyenne_geom</td>
      <td>Ecology</td>
      <td>Geometric means (Dead wood)</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/Geom_mean_workflow</td>
    </tr>
    <tr>
      <td></td>
      <td>PAMPA</td>
      <td>0.0.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tools to compute and analyse biodiversity metrics</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Ecology</td>
      <td>pampa</td>
      <td>ecology</td>
      <td>https://github.com/ColineRoyaux/PAMPA-Galaxy</td>
    </tr>
    <tr>
      <td></td>
      <td>interpolation</td>
      <td>1.0</td>
      <td></td>
      <td>r-getopt</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Run IDW interpolation based on a .csv and .geojson file</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/AquaINFRA/galaxy</td>
      <td>Ecology</td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/interpolation</td>
    </tr>
    <tr>
      <td></td>
      <td>obisindicators</td>
      <td>0.0.2</td>
      <td></td>
      <td>r-base</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute biodiveristy indicators for marine data from obis</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/Marie59/obisindicators</td>
      <td>Ecology</td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/obisindicators</td>
    </tr>
    <tr>
      <td></td>
      <td>ocean</td>
      <td>0.1.15</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Access, process, visualise oceanographic data for the Earth System</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/Marie59/FE-ft-ESG/tree/main/argo</td>
      <td>Ecology</td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/ocean</td>
    </tr>
    <tr>
      <td></td>
      <td>blast2go</td>
      <td>0.0.11</td>
      <td></td>
      <td>b2g4pipe</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Maps BLAST results to GO annotation terms</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/blast2go</td>
      <td>Ontology Manipulation, Sequence Analysis</td>
      <td>blast2go</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/blast2go</td>
    </tr>
    <tr>
      <td></td>
      <td>blast_rbh</td>
      <td>0.3.0</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>BLAST Reciprocal Best Hits (RBH) from two FASTA files</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/blast_rbh</td>
      <td>Fasta Manipulation, Sequence Analysis</td>
      <td>blast_rbh</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/blast_rbh</td>
    </tr>
    <tr>
      <td></td>
      <td>blastxml_to_top_descr</td>
      <td>0.1.2</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Make table of top BLAST match descriptions</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/blastxml_to_top_descr</td>
      <td>Convert Formats, Sequence Analysis, Text Manipulation</td>
      <td>blastxml_to_top_descr</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/blastxml_to_top_descr</td>
    </tr>
    <tr>
      <td></td>
      <td>make_nr</td>
      <td>0.0.2</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Make a FASTA file non-redundant</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/make_nr</td>
      <td>Fasta Manipulation, Sequence Analysis</td>
      <td>make_nr</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/make_nr</td>
    </tr>
    <tr>
      <td></td>
      <td>ncbi_blast_plus</td>
      <td>2.14.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>NCBI BLAST+</td>
      <td></td>
      <td>To update</td>
      <td>https://blast.ncbi.nlm.nih.gov/</td>
      <td>Sequence Analysis</td>
      <td>ncbi_blast_plus</td>
      <td>devteam</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/ncbi_blast_plus</td>
    </tr>
  </tbody>
</table>

</div>