{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Imports"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import numpy as np\n",
    "from io import StringIO\n",
    "import os\n",
    "import requests\n",
    "import pip\n",
    "import matplotlib.pyplot as plt\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Reading tools.tsv to dataframe and Preparing the table for lowering down the EDAM operations"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Galaxy tool ids</th>\n",
       "      <th>SubColumn0</th>\n",
       "      <th>SubColumn1</th>\n",
       "      <th>SubColumn2</th>\n",
       "      <th>SubColumn3</th>\n",
       "      <th>SubColumn4</th>\n",
       "      <th>SubColumn5</th>\n",
       "      <th>SubColumn6</th>\n",
       "      <th>SubColumn7</th>\n",
       "      <th>SubColumn8</th>\n",
       "      <th>...</th>\n",
       "      <th>SubColumn10</th>\n",
       "      <th>SubColumn11</th>\n",
       "      <th>SubColumn12</th>\n",
       "      <th>SubColumn13</th>\n",
       "      <th>SubColumn14</th>\n",
       "      <th>SubColumn15</th>\n",
       "      <th>SubColumn16</th>\n",
       "      <th>SubColumn17</th>\n",
       "      <th>SubColumn18</th>\n",
       "      <th>SubColumn19</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>lotus2</td>\n",
       "      <td>Sequence feature detection</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>119</th>\n",
       "      <td>antismash</td>\n",
       "      <td>Sequence clustering</td>\n",
       "      <td>Gene prediction</td>\n",
       "      <td>Differential gene expression analysis</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>129</th>\n",
       "      <td>combine_metaphlan_humann</td>\n",
       "      <td>Aggregation</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>130</th>\n",
       "      <td>compare_humann2_output</td>\n",
       "      <td>Comparison</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>133</th>\n",
       "      <td>format_metaphlan2_output</td>\n",
       "      <td>Formatting</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows × 21 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "              Galaxy tool ids                  SubColumn0        SubColumn1  \\\n",
       "12                     lotus2  Sequence feature detection              None   \n",
       "119                 antismash         Sequence clustering   Gene prediction   \n",
       "129  combine_metaphlan_humann                 Aggregation              None   \n",
       "130    compare_humann2_output                  Comparison              None   \n",
       "133  format_metaphlan2_output                  Formatting              None   \n",
       "\n",
       "                                 SubColumn2 SubColumn3 SubColumn4 SubColumn5  \\\n",
       "12                                     None       None       None       None   \n",
       "119   Differential gene expression analysis       None       None       None   \n",
       "129                                    None       None       None       None   \n",
       "130                                    None       None       None       None   \n",
       "133                                    None       None       None       None   \n",
       "\n",
       "    SubColumn6 SubColumn7 SubColumn8  ... SubColumn10 SubColumn11 SubColumn12  \\\n",
       "12        None       None       None  ...        None        None        None   \n",
       "119       None       None       None  ...        None        None        None   \n",
       "129       None       None       None  ...        None        None        None   \n",
       "130       None       None       None  ...        None        None        None   \n",
       "133       None       None       None  ...        None        None        None   \n",
       "\n",
       "    SubColumn13 SubColumn14 SubColumn15 SubColumn16 SubColumn17 SubColumn18  \\\n",
       "12         None        None        None        None        None        None   \n",
       "119        None        None        None        None        None        None   \n",
       "129        None        None        None        None        None        None   \n",
       "130        None        None        None        None        None        None   \n",
       "133        None        None        None        None        None        None   \n",
       "\n",
       "    SubColumn19  \n",
       "12         None  \n",
       "119        None  \n",
       "129        None  \n",
       "130        None  \n",
       "133        None  \n",
       "\n",
       "[5 rows x 21 columns]"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "tools_db = pd.read_table('tools.tsv')\n",
    "tools2_db = tools_db[tools_db['To keep']==True]\n",
    "tools3_db =tools2_db[tools2_db['EDAM operation'].notna()]\n",
    "tmp = pd.concat( [tools3_db['Galaxy tool ids'], tools3_db['EDAM operation'].str.split(',', expand=True).add_prefix('SubColumn')], axis=1 )\n",
    "#tmp.set_index('Galaxy tool ids', inplace=True)\n",
    "#tools2_db.head()\n",
    "#print(tools3_db)\n",
    "\n",
    "tmp.head()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Using OWL aginst the tmp dataframe to keep only the leaf node of the same branch for EDAM operations of one tool that belong to the same branch"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "Warning: SQLite3 version 3.40.0 and 3.41.2 have huge performance regressions; please install version 3.41.1 or 3.42!\n",
      "\n"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "from owlready2 import get_ontology, Thing\n",
    "import copy\n",
    "\n",
    "def process_row(row):\n",
    "# Remove extra spaces from each column value in the row\n",
    "    cleaned_row = [str(value).strip() for value in row[1:]]  # Exclude the first column ('Galaxy tool ids')\n",
    "    \n",
    "    # Convert the cleaned row to a list of EDAM terms using the provided ontology\n",
    "    edam_ontology = get_ontology('https://edamontology.org/EDAM_1.25.owl').load()\n",
    "    \n",
    "    terms = cleaned_row\n",
    "    classes = [edam_ontology.search_one(label=term) for term in terms]\n",
    "    check_classes = [cla for cla in classes if cla is not None]  # Remove None values\n",
    "    \n",
    "    new_classes = []\n",
    "    for cla in check_classes:\n",
    "        try:\n",
    "            # get all subclasses\n",
    "            subclasses = list(cla.subclasses())\n",
    "    \n",
    "            # check if any of the other classes is a subclass\n",
    "            include_class = True\n",
    "            for subcla in subclasses:\n",
    "                for cla2 in check_classes:\n",
    "                    if subcla == cla2:\n",
    "                        include_class = False\n",
    "    \n",
    "            # only keep the class if it is not a parent class\n",
    "            if include_class:\n",
    "                new_classes.append(cla)\n",
    "        except Exception as e:\n",
    "            print(f\"Error processing class {cla}: {e}\")\n",
    "\n",
    "    # convert back to terms, skipping None values\n",
    "    new_terms = [cla.label[0] for cla in new_classes if cla is not None]\n",
    "    \n",
    "    # Include the first column ('Galaxy tool ids') in the returned series\n",
    "    return pd.Series([row[0], ', '.join(new_terms)])  # Combine the new terms with commas\n",
    "\n",
    "def process_dataframe(input_df):\n",
    "    # Apply the process_row function to each row in the dataframe\n",
    "    output_df = input_df.apply(process_row, axis=1)\n",
    "    \n",
    "    # Set the header of the output dataframe\n",
    "    output_df.columns = ['Galaxy tool ids', 'EDAM operation']\n",
    "    \n",
    "    return output_df\n",
    "\n",
    "example_df = pd.DataFrame({\n",
    "    'Galaxy tool ids': ['tool1', 'tool2', 'tool3'],\n",
    "    'SubColumn0': ['Differential protein expression profiling', 'Sequence analysis', 'Data retrieval'],\n",
    "    'SubColumn1': [' Gene expression profiling ', '  Phylogenetic tree analysis  ', ' Service invocation ']\n",
    "})\n",
    "\n",
    "\n",
    "processed_df = process_dataframe(tmp)\n",
    "processed_df.to_csv('filteredtoolsEDAMoperations.csv', index=False)\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Adding an updated EDAM operation column (EDAM operation low classes) in the original tool.tsv file (after removing to Kepp = false) and save it to updatedtools.tsv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_24265/1080805737.py:3: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  tools2_db['EDAM operation low classes'] = tools2_db['EDAM operation']\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Galaxy wrapper id</th>\n",
       "      <th>Total tool usage (usegalaxy.eu)</th>\n",
       "      <th>No. of tool users (2022-2023) (usegalaxy.eu)</th>\n",
       "      <th>Galaxy tool ids</th>\n",
       "      <th>Description</th>\n",
       "      <th>bio.tool id</th>\n",
       "      <th>bio.tool name</th>\n",
       "      <th>bio.tool description</th>\n",
       "      <th>EDAM operation</th>\n",
       "      <th>EDAM topic</th>\n",
       "      <th>...</th>\n",
       "      <th>Galaxy wrapper source</th>\n",
       "      <th>Galaxy wrapper version</th>\n",
       "      <th>Conda id</th>\n",
       "      <th>Conda version</th>\n",
       "      <th>https://usegalaxy.org</th>\n",
       "      <th>https://usegalaxy.org.au</th>\n",
       "      <th>https://usegalaxy.eu</th>\n",
       "      <th>Reviewed</th>\n",
       "      <th>To keep</th>\n",
       "      <th>EDAM operation low classes</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>TreeBest</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>treebest_best</td>\n",
       "      <td>TreeBeST best</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>https://github.com/TGAC/earlham-galaxytools/tr...</td>\n",
       "      <td>1.9.2.post0</td>\n",
       "      <td>treebest</td>\n",
       "      <td>1.9.2.post1</td>\n",
       "      <td>(0/1)</td>\n",
       "      <td>(0/1)</td>\n",
       "      <td>(1/1)</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>ete</td>\n",
       "      <td>1255.0</td>\n",
       "      <td>67.0</td>\n",
       "      <td>ete_gene_csv_finder, ete_genetree_splitter, et...</td>\n",
       "      <td>Analyse phylogenetic trees using the ETE Toolkit</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>https://github.com/TGAC/earlham-galaxytools/tr...</td>\n",
       "      <td>3.1.2</td>\n",
       "      <td>ete3</td>\n",
       "      <td>3.1.1</td>\n",
       "      <td>(0/7)</td>\n",
       "      <td>(0/7)</td>\n",
       "      <td>(7/7)</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>lotus2</td>\n",
       "      <td>936.0</td>\n",
       "      <td>114.0</td>\n",
       "      <td>lotus2</td>\n",
       "      <td>LotuS2 OTU processing pipeline</td>\n",
       "      <td>lotus2</td>\n",
       "      <td>lotus2</td>\n",
       "      <td>LotuS2 is a lightweight and user-friendly pipe...</td>\n",
       "      <td>Sequence feature detection</td>\n",
       "      <td>Metagenomics</td>\n",
       "      <td>...</td>\n",
       "      <td>https://github.com/TGAC/earlham-galaxytools/tr...</td>\n",
       "      <td>2.32</td>\n",
       "      <td>lotus2</td>\n",
       "      <td>2.32</td>\n",
       "      <td>(0/1)</td>\n",
       "      <td>(0/1)</td>\n",
       "      <td>(1/1)</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>Sequence feature detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>abacas</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>abacas</td>\n",
       "      <td>Order and Orientate Contigs</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>https://github.com/phac-nml/abacas</td>\n",
       "      <td>1.1</td>\n",
       "      <td>mummer</td>\n",
       "      <td>3.23</td>\n",
       "      <td>(0/1)</td>\n",
       "      <td>(0/1)</td>\n",
       "      <td>(0/1)</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>assemblystats</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>assemblystats</td>\n",
       "      <td>Summarise an assembly (e.g. N50 metrics)</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>https://github.com/phac-nml/galaxy_tools</td>\n",
       "      <td>1.1.0</td>\n",
       "      <td>perl-bioperl</td>\n",
       "      <td>1.7.8</td>\n",
       "      <td>(0/1)</td>\n",
       "      <td>(0/1)</td>\n",
       "      <td>(0/1)</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows × 25 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "   Galaxy wrapper id  Total tool usage (usegalaxy.eu)  \\\n",
       "6           TreeBest                              NaN   \n",
       "8                ete                           1255.0   \n",
       "12            lotus2                            936.0   \n",
       "16            abacas                              NaN   \n",
       "17     assemblystats                              NaN   \n",
       "\n",
       "    No. of tool users (2022-2023) (usegalaxy.eu)  \\\n",
       "6                                            NaN   \n",
       "8                                           67.0   \n",
       "12                                         114.0   \n",
       "16                                           NaN   \n",
       "17                                           NaN   \n",
       "\n",
       "                                      Galaxy tool ids  \\\n",
       "6                                       treebest_best   \n",
       "8   ete_gene_csv_finder, ete_genetree_splitter, et...   \n",
       "12                                             lotus2   \n",
       "16                                             abacas   \n",
       "17                                      assemblystats   \n",
       "\n",
       "                                         Description bio.tool id  \\\n",
       "6                                      TreeBeST best         NaN   \n",
       "8   Analyse phylogenetic trees using the ETE Toolkit         NaN   \n",
       "12                    LotuS2 OTU processing pipeline      lotus2   \n",
       "16                       Order and Orientate Contigs         NaN   \n",
       "17          Summarise an assembly (e.g. N50 metrics)         NaN   \n",
       "\n",
       "   bio.tool name                               bio.tool description  \\\n",
       "6            NaN                                                NaN   \n",
       "8            NaN                                                NaN   \n",
       "12        lotus2  LotuS2 is a lightweight and user-friendly pipe...   \n",
       "16           NaN                                                NaN   \n",
       "17           NaN                                                NaN   \n",
       "\n",
       "                EDAM operation    EDAM topic  ...  \\\n",
       "6                          NaN           NaN  ...   \n",
       "8                          NaN           NaN  ...   \n",
       "12  Sequence feature detection  Metagenomics  ...   \n",
       "16                         NaN           NaN  ...   \n",
       "17                         NaN           NaN  ...   \n",
       "\n",
       "                                Galaxy wrapper source Galaxy wrapper version  \\\n",
       "6   https://github.com/TGAC/earlham-galaxytools/tr...            1.9.2.post0   \n",
       "8   https://github.com/TGAC/earlham-galaxytools/tr...                  3.1.2   \n",
       "12  https://github.com/TGAC/earlham-galaxytools/tr...                   2.32   \n",
       "16                 https://github.com/phac-nml/abacas                    1.1   \n",
       "17           https://github.com/phac-nml/galaxy_tools                  1.1.0   \n",
       "\n",
       "        Conda id Conda version https://usegalaxy.org https://usegalaxy.org.au  \\\n",
       "6       treebest   1.9.2.post1                 (0/1)                    (0/1)   \n",
       "8           ete3         3.1.1                 (0/7)                    (0/7)   \n",
       "12        lotus2          2.32                 (0/1)                    (0/1)   \n",
       "16        mummer          3.23                 (0/1)                    (0/1)   \n",
       "17  perl-bioperl         1.7.8                 (0/1)                    (0/1)   \n",
       "\n",
       "   https://usegalaxy.eu Reviewed To keep  EDAM operation low classes  \n",
       "6                 (1/1)     True    True                         NaN  \n",
       "8                 (7/7)     True    True                         NaN  \n",
       "12                (1/1)     True    True  Sequence feature detection  \n",
       "16                (0/1)     True    True                         NaN  \n",
       "17                (0/1)     True    True                         NaN  \n",
       "\n",
       "[5 rows x 25 columns]"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "if 'EDAM operation' in tools2_db.columns and 'EDAM operation' in processed_df.columns:\n",
    "    # Add the column from df2 to df1 with a new name\n",
    "    tools2_db['EDAM operation low classes'] = tools2_db['EDAM operation']\n",
    "else:\n",
    "    print(\"Column 'EDAM operation' not found in one or both dataframes.\")\n",
    "\n",
    "tools2_db.to_csv('Updatedtools.tsv', index=False)\n",
    "\n",
    "tools2_db.head()"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
