<script src="https://code.jquery.com/jquery-3.7.0.js"></script>
<script src="https://cdn.datatables.net/1.13.6/js/jquery.dataTables.min.js"></script>
<script src="https://cdn.datatables.net/searchbuilder/1.6.0/js/dataTables.searchBuilder.min.js"></script>
<script src="https://cdn.datatables.net/datetime/1.5.1/js/dataTables.dateTime.min.js"></script>

<script src="https://cdn.datatables.net/responsive/2.5.0/js/dataTables.responsive.min.js"></script>

<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/responsive/2.2.3/css/responsive.dataTables.min.css">

<link rel="stylesheet" href="https://cdn.datatables.net/1.13.6/css/jquery.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/searchbuilder/1.6.0/css/searchBuilder.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/datetime/1.5.1/css/dataTables.dateTime.min.css">

<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" />
<script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>

<style type="text/css">
  .tooltip {
    font-family: Georgia;
    font-size: 18px;
  }
  .tooltip .tooltip-inner {
    background-color: #00FFFF;
    color: #000000;
    min-width: 400px;
  }
  </style>

<script>
    $(document).ready(function() {
    $('#dataframe').DataTable( {
        "sScrollX": "100%",
        "dom": "Qlfrtip",

        responsive: {
            details: {
                type: 'column'
            }
        },
        "bAutoWidth": false,
        columnDefs: [ {
            className: 'control',
            orderable: false,
            targets:   [ 0 ],
            width: 100,
        },
        { type: 'natural', targets: 1 } // allow normal sorting
        ],

    });
});
</script>

<div class="display nowrap" style="width:80%" width="100%">

<table class="dataframe display nowrap" id="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th>Expand</th>
      <th>Topic</th>
      <th>Title</th>
      <th>Link</th>
      <th>EDAM topic</th>
      <th>EDAM operation</th>
      <th>Creation</th>
      <th>Last modification</th>
      <th>Version</th>
      <th>Tutorial</th>
      <th>Slides</th>
      <th>Video</th>
      <th>Workflows</th>
      <th>Tools</th>
      <th>Servers with precise tool versions</th>
      <th>Servers with tool but different versions</th>
      <th>Feedback number</th>
      <th>Feedback mean note</th>
      <th>Visitors</th>
      <th>Page views</th>
      <th>Visit duration</th>
      <th>Video views</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Making sense of a newly assembled genome</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/ecoli_comparison/tutorial.html</td>
      <td>Sequence assembly, Genomics, Microbiology</td>
      <td>Genome visualisation, Sequence alignment, Read mapping, Sequence visualisation, Mapping</td>
      <td>2018-06-14</td>
      <td>2024-03-11</td>
      <td>86</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Cut1, tp_replace_in_line, mergeCols1, tp_replace_in_column, fasta_filter_by_length, Grep1, cat1, circos_aln_to_links, jbrowse, fasta_compute_length, collapse_dataset, random_lines1, bedtools_sortbed, lastz_wrapper_2, addValue, circos, join1, Filter1, bedtools_intersectbed, bedtools_complementbed, upload1, tp_sed_tool, tp_cat, datamash_ops, tp_sort_header_tool, circos_interval_to_tiles, tp_grep_tool</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>2.0</td>
      <td>5828</td>
      <td>11935</td>
      <td>3.033333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Assembly of metagenomic sequencing data</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/metagenomics-assembly/tutorial.html</td>
      <td>Metagenomics, Sequence assembly</td>
      <td>Sequence composition calculation, Genome assembly, Visualisation, Primer removal, Sequence assembly visualisation, Sequence file editing, Read mapping, Statistical calculation, Data handling, Sequence contamination filtering, Local alignment, Sequence alignment analysis, Variant calling, Sequencing quality control, Read pre-processing, Sequence assembly validation, Formatting, Sequence trimming</td>
      <td>2024-06-24</td>
      <td>2024-06-24</td>
      <td>0</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>filter_tabular, bandage_image, metaspades, megahit, quast, cutadapt, tp_cat, ngsutils_bam_filter, megahit_contig2fastg, random_lines1, bamtools, bowtie2, bandage_info, collection_column_join, coverm_contig, seqtk_subseq, fastqc, bg_uniq</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td>0</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>0.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Genome Assembly of a bacterial genome (MRSA) sequenced using Illumina MiSeq Data</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/mrsa-illumina/tutorial.html</td>
      <td>Sequence assembly, Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology</td>
      <td>Sequence assembly visualisation, Sequence composition calculation, Sequence contamination filtering, Genome assembly, Statistical calculation, Sequencing quality control, Visualisation, Sequence assembly validation</td>
      <td>2021-03-24</td>
      <td>2024-03-11</td>
      <td>28</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>bandage_image, upload1, quast, bandage_info, fastqc, fastp, shovill</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>Galaxy@AuBi, GalaxyTrakr, UseGalaxy.no</td>
      <td>8</td>
      <td>4.5</td>
      <td>6115</td>
      <td>9890</td>
      <td>3.733333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Genome Assembly of MRSA from Oxford Nanopore MinION data (and optionally Illumina data)</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/mrsa-nanopore/tutorial.html</td>
      <td>Sequence assembly, Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology, Antimicrobial Resistance</td>
      <td>De-novo assembly, Sequence composition calculation, Genome assembly, Sequencing error detection, Visualisation, Sequencing quality control, Mapping assembly, Sequence assembly visualisation, Sequence assembly validation, Cross-assembly, Box-Whisker plot plotting, Scatter plot plotting, Filtering, Read mapping, Statistical calculation, Sequence alignment, Sequence contamination filtering</td>
      <td>2021-03-24</td>
      <td>2024-03-13</td>
      <td>35</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>flye, nanoplot, bandage_image, polypolish, upload1, quast, porechop, CONVERTER_bz2_to_uncompressed, bwa_mem2, filtlong, fastqc, fastp</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz</td>
      <td>2</td>
      <td>4.0</td>
      <td>6353</td>
      <td>9777</td>
      <td>3.416667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Unicycler Assembly</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/unicycler-assembly/tutorial.html</td>
      <td>Sequence assembly, Genomics, Microbiology</td>
      <td>Sequence composition calculation, Genome assembly, Validation, Coding region prediction, Sequencing quality control, Visualisation, Sequence assembly validation, Statistical calculation, Gene prediction, Aggregation, Genome annotation</td>
      <td>2017-10-11</td>
      <td>2024-03-11</td>
      <td>61</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>quast, prokka, unicycler, multiqc, fastqc</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>GalaxyTrakr, HyPhy HIV NGS Tools, MISSISSIPPI, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no</td>
      <td>8</td>
      <td>3.75</td>
      <td>1994916</td>
      <td>5552489</td>
      <td>4.566667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Checking expected species and contamination in bacterial isolate</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/bacterial-isolate-species-contamination-checking/tutorial.html</td>
      <td>Whole genome sequencing, Genomics, Microbiology, Microbial ecology</td>
      <td>Statistical calculation, Cross-assembly, Taxonomic classification, Expression analysis</td>
      <td>2024-03-04</td>
      <td>2024-04-04</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>kraken2, recentrifuge, upload1, est_abundance</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td>Galaxy@AuBi, GalaxyTrakr, UseGalaxy.cz, UseGalaxy.org (Main)</td>
      <td>1</td>
      <td>5.0</td>
      <td>165</td>
      <td>226</td>
      <td>4.866667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Tree thinking for tuberculosis evolution and epidemiology</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/mtb_phylogeny/tutorial.html</td>
      <td>Genomics, Microbiology, Phylogeny, Infectious disease</td>
      <td>Sequence analysis, Phylogenetic tree analysis</td>
      <td>2022-03-16</td>
      <td>2023-02-15</td>
      <td>19</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>raxml, interactive_tool_rstudio, upload1</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>6</td>
      <td>4.333333</td>
      <td>1211</td>
      <td>1857</td>
      <td>2.266667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Identifying tuberculosis transmission links: from SNPs to transmission clusters</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/mtb_transmission/tutorial.html</td>
      <td>Genomics, Microbiology, Infectious disease, DNA polymorphism</td>
      <td>Antimicrobial resistance prediction, Phylogenetic tree generation, Phylogenetic tree visualisation, Variant calling</td>
      <td>2022-03-16</td>
      <td>2024-03-12</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>snippy, tb_variant_filter, tp_replace_in_line, trimmomatic, bcftools_consensus, snp_sites, tp_sed_tool, addName, tp_cat, upload1, collapse_dataset, Grep1, __MERGE_COLLECTION__, tp_easyjoin_tool, tb_profiler_profile, samtools_view, tp_grep_tool, snp_dists</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>7</td>
      <td>4.0</td>
      <td>1377</td>
      <td>2211</td>
      <td>3.183333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Genome annotation with Prokka</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/annotation-with-prokka/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis</td>
      <td>Genome annotation, Coding region prediction, Gene prediction, Genome visualisation</td>
      <td>2018-03-06</td>
      <td>2024-05-29</td>
      <td>43</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>prokka, jbrowse</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>Galaxy@AuBi, GalaxyTrakr, MISSISSIPPI, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>17</td>
      <td>3.764706</td>
      <td>1999341</td>
      <td>5560175</td>
      <td>4.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Bacterial Genome Annotation</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/bacterial-genome-annotation/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis, Whole genome sequencing, Functional genomics, Mobile genetic elements</td>
      <td>Structural variation detection, Genome assembly, Nucleic acid feature detection, Protein feature detection, Scaffolding, Multilocus sequence typing, Sequence motif recognition, Genome visualisation, Genome annotation</td>
      <td>2024-02-01</td>
      <td>2024-03-13</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tp_tail_tool, upload1, bakta, isescan, Grouping1, tp_replace_in_column, tbl2gff3, plasmidfinder, integron_finder, jbrowse</td>
      <td>Galaxy@AuBi, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr</td>
      <td></td>
      <td>1</td>
      <td>1.0</td>
      <td>1237</td>
      <td>1630</td>
      <td>2.483333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Identification of AMR genes in an assembled bacterial genome</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/amr-gene-detection/tutorial.html</td>
      <td>Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology, Sequence analysis, Infectious disease, Antimicrobial Resistance</td>
      <td>Genome annotation, Genome visualisation, Read mapping</td>
      <td>2024-01-23</td>
      <td>2024-03-13</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>upload1, staramr_search, bakta, Grep1, bowtie2, tbl2gff3, jbrowse</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr</td>
      <td>Galaxy@AuBi</td>
      <td>0</td>
      <td></td>
      <td>726</td>
      <td>1061</td>
      <td>3.066667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Refining Genome Annotations with Apollo (prokaryotes)</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/apollo/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis</td>
      <td>Genome visualisation</td>
      <td>2021-06-04</td>
      <td>2024-05-29</td>
      <td>40</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>iframe, create_or_update, list_organism, create_account, jbrowse</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no</td>
      <td>5</td>
      <td>4.6</td>
      <td>1757138</td>
      <td>4755339</td>
      <td>5.016667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Comparative gene analysis in unannotated genomes</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/gene-centric/tutorial.html</td>
      <td>Genomics, Gene and protein families, Sequence analysis, Phylogeny</td>
      <td>De-novo assembly, Sequence alignment analysis, Multiple sequence alignment, Coding region prediction, Phylogenetic tree generation, Transcriptome assembly, Database search</td>
      <td>2022-09-08</td>
      <td>2024-03-18</td>
      <td>30</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Add_a_column1, gops_intersect_1, Cut1, orfipy, bg_diamond_view, tp_split_on_column, bg_diamond, tab2fasta, rbc_mafft, collapse_dataset, cat1, regexColumn1, join1, rapidnj, Filter1, bg_diamond_makedb</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>910</td>
      <td>1395</td>
      <td>3.116667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Essential genes detection with Transposon insertion sequencing</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/tnseq/tutorial.html</td>
      <td>Genomics, Microbiology, Sequence analysis, Mobile genetic elements</td>
      <td>Transposon prediction, Primer removal, Read pre-processing, Sequence trimming</td>
      <td>2019-07-02</td>
      <td>2024-03-15</td>
      <td>52</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>Add_a_column1, Cut1, bowtie_wrapper, transit_gumbel, cutadapt, gff_to_prot, tp_sort_header_tool, __EXTRACT_DATASET__, deeptools_bam_coverage, tp_easyjoin_tool, bg_find_subsequences, Filter1</td>
      <td>UseGalaxy.no</td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td>4</td>
      <td>4.5</td>
      <td>1990676</td>
      <td>5545573</td>
      <td>4.250000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Genome annotation with Prokka</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/annotation-with-prokka/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis</td>
      <td>Genome annotation, Coding region prediction, Gene prediction, Genome visualisation</td>
      <td>2018-03-06</td>
      <td>2024-05-29</td>
      <td>43</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>prokka, jbrowse</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>Galaxy@AuBi, GalaxyTrakr, MISSISSIPPI, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>17</td>
      <td>3.764706</td>
      <td>1999341</td>
      <td>5560176</td>
      <td>4.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Microbial Variant Calling</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/microbial-variants/tutorial.html</td>
      <td>Genomics, Sequence assembly, DNA polymorphism, Microbiology, Sequence analysis, Genetic variation</td>
      <td>Genome visualisation, Phylogenetic tree generation, Phylogenetic tree visualisation, Variant calling</td>
      <td>2018-02-26</td>
      <td>2024-03-14</td>
      <td>36</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>snippy, jbrowse</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>GalaxyTrakr, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>12</td>
      <td>4.416667</td>
      <td>11631</td>
      <td>19648</td>
      <td>1.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Making sense of a newly assembled genome</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/ecoli_comparison/tutorial.html</td>
      <td>Sequence assembly, Genomics, Microbiology</td>
      <td>Genome visualisation, Sequence alignment, Read mapping, Sequence visualisation, Mapping</td>
      <td>2018-06-14</td>
      <td>2024-03-11</td>
      <td>86</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Cut1, tp_replace_in_line, mergeCols1, tp_replace_in_column, fasta_filter_by_length, Grep1, cat1, circos_aln_to_links, jbrowse, fasta_compute_length, collapse_dataset, random_lines1, bedtools_sortbed, lastz_wrapper_2, addValue, circos, join1, Filter1, bedtools_intersectbed, bedtools_complementbed, upload1, tp_sed_tool, tp_cat, datamash_ops, tp_sort_header_tool, circos_interval_to_tiles, tp_grep_tool</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>2.0</td>
      <td>5828</td>
      <td>11935</td>
      <td>3.033333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Genome Assembly of MRSA from Oxford Nanopore MinION data (and optionally Illumina data)</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/mrsa-nanopore/tutorial.html</td>
      <td>Sequence assembly, Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology, Antimicrobial Resistance</td>
      <td>De-novo assembly, Sequence composition calculation, Genome assembly, Sequencing error detection, Visualisation, Sequencing quality control, Mapping assembly, Sequence assembly visualisation, Sequence assembly validation, Cross-assembly, Box-Whisker plot plotting, Scatter plot plotting, Filtering, Read mapping, Statistical calculation, Sequence alignment, Sequence contamination filtering</td>
      <td>2021-03-24</td>
      <td>2024-03-13</td>
      <td>35</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>flye, nanoplot, bandage_image, polypolish, upload1, quast, porechop, CONVERTER_bz2_to_uncompressed, bwa_mem2, filtlong, fastqc, fastp</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz</td>
      <td>2</td>
      <td>4.0</td>
      <td>6353</td>
      <td>9777</td>
      <td>3.416667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Bacterial Genome Annotation</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/bacterial-genome-annotation/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis, Whole genome sequencing, Functional genomics, Mobile genetic elements</td>
      <td>Structural variation detection, Genome assembly, Nucleic acid feature detection, Protein feature detection, Scaffolding, Multilocus sequence typing, Sequence motif recognition, Genome visualisation, Genome annotation</td>
      <td>2024-02-01</td>
      <td>2024-03-13</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tp_tail_tool, upload1, bakta, isescan, Grouping1, tp_replace_in_column, tbl2gff3, plasmidfinder, integron_finder, jbrowse</td>
      <td>Galaxy@AuBi, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr</td>
      <td></td>
      <td>1</td>
      <td>1.0</td>
      <td>1237</td>
      <td>1630</td>
      <td>2.483333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Identification of AMR genes in an assembled bacterial genome</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/amr-gene-detection/tutorial.html</td>
      <td>Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology, Sequence analysis, Infectious disease, Antimicrobial Resistance</td>
      <td>Genome annotation, Genome visualisation, Read mapping</td>
      <td>2024-01-23</td>
      <td>2024-03-13</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>upload1, staramr_search, bakta, Grep1, bowtie2, tbl2gff3, jbrowse</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr</td>
      <td>Galaxy@AuBi</td>
      <td>0</td>
      <td></td>
      <td>726</td>
      <td>1061</td>
      <td>3.066667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Refining Genome Annotations with Apollo (prokaryotes)</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/apollo/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis</td>
      <td>Genome visualisation</td>
      <td>2021-06-04</td>
      <td>2024-05-29</td>
      <td>40</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>iframe, create_or_update, list_organism, create_account, jbrowse</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no</td>
      <td>5</td>
      <td>4.6</td>
      <td>1757138</td>
      <td>4755340</td>
      <td>5.016667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Analyses of metagenomics data - The global picture</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/general-tutorial/tutorial.html</td>
      <td>Metagenomics, Microbial ecology, Taxonomy, Sequence analysis</td>
      <td>Sequence read processing, Sequencing quality control, Visualisation, Taxonomic classification, Phylogenetic analysis, DNA barcoding, Sequence clustering</td>
      <td>2017-06-22</td>
      <td>2024-03-14</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>mothur_summary_seqs, mothur_classify_seqs, humann2_regroup_table, krona-text, mothur_pre_cluster, mothur_align_seqs, mothur_make_group, mothur_unique_seqs, humann2_renorm_table, metaphlan2, metaphlan2krona, mothur_merge_files, mothur_make_biom, mothur_make_shared, mothur_screen_seqs, mothur_count_seqs, mothur_cluster_split, mothur_classify_otu, mothur_filter_seqs, taxonomy_krona_chart, humann2</td>
      <td></td>
      <td>UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>16</td>
      <td>4.0625</td>
      <td>2225</td>
      <td>2984</td>
      <td>2.650000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Pathogen detection from (direct Nanopore) sequencing data using Galaxy - Foodborne Edition</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.html</td>
      <td>Metagenomics, Public health and epidemiology, Taxonomy, Sequence assembly, Pathology, Sequence analysis</td>
      <td>Phylogenetic tree reconstruction, Sequence composition calculation, Base-calling, Genome assembly, Phylogenetic tree generation (from molecular sequences), Validation, Visualisation, Sequence assembly visualisation, Multilocus sequence typing, Scatter plot plotting, Statistical calculation, Data handling, Phylogenetic tree generation (maximum likelihood and Bayesian methods), Pairwise sequence alignment, Sequence contamination filtering, De-novo assembly, Sequence alignment analysis, Variant calling, Mapping, Sequencing quality control, Phylogenetic tree generation, Mapping assembly, Box-Whisker plot plotting, Cross-assembly, Taxonomic classification, Antimicrobial resistance prediction, Multiple sequence alignment, Sequence assembly, Aggregation, Phylogenetic tree analysis</td>
      <td>2023-01-26</td>
      <td>2024-06-19</td>
      <td>18</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>nanoplot, bandage_image, Cut1, CONVERTER_gz_to_uncompressed, bcftools_consensus, samtools_fastx, Grouping1, tp_replace_in_column, Grep1, __BUILD_LIST__, tp_find_and_replace, regex1, clustalw, fastp, tp_sorted_uniq, collection_element_identifiers, Paste1, flye, Add_a_column1, medaka_consensus_pipeline, Count1, minimap2, table_compute, tp_split_on_column, bamtools_split_mapped, clair3, kraken2, collapse_dataset, __FILTER_EMPTY_DATASETS__, compose_text_param, samtools_depth, snpSift_extractFields, regexColumn1, fastqc, abricate, newick_display, krakentools_extract_kraken_reads, bcftools_norm, barchart_gnuplot, Remove beginning1, samtools_coverage, split_file_to_collection, porechop, tp_multijoin_tool, mlst, tp_head_tool, multiqc, collection_column_join, fasta_merge_files_and_filter_unique_sequences, param_value_from_file, add_line_to_file, krakentools_kreport2krona, bedtools_getfastabed, fasttree, tab2fasta, ggplot2_heatmap, snpSift_filter, __FILTER_FAILED_DATASETS__, taxonomy_krona_chart, fasta2tab, tp_cut_tool</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>1</td>
      <td>5.0</td>
      <td>1259</td>
      <td>1884</td>
      <td>2.800000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Taxonomic Profiling and Visualization of Metagenomic Data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/taxonomic-profiling/tutorial.html</td>
      <td>Metagenomics, Microbial ecology, Taxonomy, Sequence analysis</td>
      <td>Taxonomic classification, Aggregation, Nucleic acid sequence analysis, Statistical calculation, Visualisation, Phylogenetic tree analysis</td>
      <td>2023-05-03</td>
      <td>2024-05-29</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>interactive_tool_pavian, interactive_tool_phinch, est_abundance, kraken2, metaphlan, kraken_biom, taxonomy_krona_chart, __UNZIP_COLLECTION__, krakentools_kreport2krona</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.org.au</td>
      <td>3</td>
      <td>4.333333</td>
      <td>2673</td>
      <td>3680</td>
      <td>1.900000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Assembly of metagenomic sequencing data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metagenomics-assembly/tutorial.html</td>
      <td>Metagenomics, Sequence assembly</td>
      <td>Sequence composition calculation, Genome assembly, Visualisation, Primer removal, Sequence assembly visualisation, Sequence file editing, Read mapping, Statistical calculation, Data handling, Sequence contamination filtering, Local alignment, Sequence alignment analysis, Variant calling, Sequencing quality control, Read pre-processing, Sequence assembly validation, Formatting, Sequence trimming</td>
      <td>2022-12-05</td>
      <td>2024-05-29</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>filter_tabular, bandage_image, metaspades, megahit, quast, cutadapt, tp_cat, ngsutils_bam_filter, megahit_contig2fastg, random_lines1, bamtools, bowtie2, bandage_info, collection_column_join, coverm_contig, seqtk_subseq, fastqc, bg_uniq</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td>0</td>
      <td></td>
      <td>1100</td>
      <td>1543</td>
      <td>2.600000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Antibiotic resistance detection</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/plasmid-metagenomics-nanopore/tutorial.html</td>
      <td>Metagenomic sequencing, Public health and epidemiology, Metagenomics, Microbiology, Infectious disease, Sequence analysis, Antimicrobial Resistance</td>
      <td>De-novo assembly, Genome assembly, Mapping assembly, Sequence assembly visualisation, Box-Whisker plot plotting, Scatter plot plotting, Aggregation, Sequence analysis, Pairwise sequence alignment</td>
      <td>2019-06-25</td>
      <td>2024-05-29</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>nanoplot, bandage_image, racon, staramr_search, PlasFlow, gfa_to_fa, unicycler, miniasm, minimap2</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>10</td>
      <td>5.0</td>
      <td>902</td>
      <td>1419</td>
      <td>4.050000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>16S Microbial analysis with Nanopore data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/nanopore-16S-metagenomics/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis</td>
      <td>Taxonomic classification, Sequence composition calculation, Statistical calculation, Validation, Sequencing quality control, Visualisation, Sequence contamination filtering</td>
      <td>2020-11-24</td>
      <td>2024-03-21</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tp_replace_in_line, kraken2, Remove beginning1, porechop, multiqc, datamash_reverse, taxonomy_krona_chart, fastqc, fastp</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>GalaxyTrakr, UseGalaxy.cz</td>
      <td>13</td>
      <td>4.692308</td>
      <td>1835</td>
      <td>2617</td>
      <td>2.900000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Metatranscriptomics analysis using microbiome RNA-seq data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metatranscriptomics/tutorial.html</td>
      <td>Metatranscriptomics, Microbial ecology, Taxonomy, Function analysis, Sequence analysis</td>
      <td>Sequence composition calculation, Phylogenetic tree visualisation, Validation, Phylogenetic inference, Primer removal, Visualisation, Species frequency estimation, Statistical calculation, Phylogenetic analysis, Sequence similarity search, Sequence alignment analysis, Sequence comparison, Sequencing quality control, Read pre-processing, Phylogenetic tree editing, Taxonomic classification, Conversion, Nucleic acid sequence analysis, Phylogenetic tree analysis, Sequence trimming</td>
      <td>2019-11-21</td>
      <td>2024-05-29</td>
      <td>4</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>graphlan_annotate, Cut1, humann_split_stratified_table, bg_sortmerna, Grep1, tp_find_and_replace, graphlan, humann_renorm_table, humann, metaphlan, humann_rename_table, fastqc, multiqc, combine_metaphlan2_humann2, fastq_paired_end_interlacer, cutadapt, export2graphlan, humann_regroup_table, taxonomy_krona_chart, humann_unpack_pathways</td>
      <td>UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>6</td>
      <td>4.5</td>
      <td>1986067</td>
      <td>5535600</td>
      <td>4.300000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Metatranscriptomics analysis using microbiome RNA-seq data (short)</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metatranscriptomics-short/tutorial.html</td>
      <td>Metatranscriptomics, Microbial ecology, Taxonomy, Function analysis, Sequence analysis</td>
      <td>Sequence composition calculation, Phylogenetic tree visualisation, Validation, Phylogenetic inference, Primer removal, Visualisation, Species frequency estimation, Statistical calculation, Phylogenetic analysis, Sequence similarity search, Sequence alignment analysis, Sequence comparison, Sequencing quality control, Read pre-processing, Phylogenetic tree editing, Taxonomic classification, Conversion, Nucleic acid sequence analysis, Phylogenetic tree analysis, Sequence trimming</td>
      <td>2020-02-13</td>
      <td>2024-05-29</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>graphlan_annotate, Cut1, humann_split_stratified_table, bg_sortmerna, Grep1, tp_find_and_replace, graphlan, humann_renorm_table, humann, metaphlan, humann_rename_table, fastqc, multiqc, combine_metaphlan2_humann2, fastq_paired_end_interlacer, cutadapt, export2graphlan, humann_regroup_table, taxonomy_krona_chart, humann_unpack_pathways</td>
      <td>UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>1</td>
      <td>5.0</td>
      <td>184</td>
      <td>236</td>
      <td>2.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Identification of the micro-organisms in a beer using Nanopore sequencing</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/beer-data-analysis/tutorial.html</td>
      <td>Metagenomics, Microbial ecology, Taxonomy, Sequence analysis</td>
      <td>Taxonomic classification, Sequence composition calculation, Aggregation, Statistical calculation, Sequencing quality control, Visualisation, Sequence contamination filtering</td>
      <td>2022-09-29</td>
      <td>2024-05-29</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>kraken2, porechop, taxonomy_krona_chart, fastqc, fastp, Filter1, krakentools_kreport2krona</td>
      <td>GalaxyTrakr, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>3</td>
      <td>4.666667</td>
      <td>969</td>
      <td>1570</td>
      <td>3.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Building an amplicon sequence variant (ASV) table from 16S data using DADA2</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/dada-16S/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis</td>
      <td>Variant calling, DNA barcoding, Deposition, Visualisation, Analysis</td>
      <td>2024-06-05</td>
      <td>2024-06-17</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tp_replace_in_line, dada2_filterAndTrim, dada2_plotQualityProfile, Grouping1, tp_replace_in_column, cat1, interactive_tool_phyloseq, collection_element_identifiers, Add_a_column1, dada2_mergePairs, dada2_seqCounts, __UNZIP_COLLECTION__, __SORTLIST__, dada2_dada, dada2_makeSequenceTable, phyloseq_from_dada2, tp_head_tool, dada2_removeBimeraDenovo, dada2_learnErrors, dada2_assignTaxonomyAddspecies</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>104</td>
      <td>129</td>
      <td>1.483333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>QIIME 2 Cancer Microbiome Intervention</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/qiime2-cancer-microbiome-intervention/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis</td>
      <td></td>
      <td>2024-02-12</td>
      <td>2024-03-14</td>
      <td>6</td>
      <td>external</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>1</td>
      <td>5.0</td>
      <td>83</td>
      <td>152</td>
      <td>1.233333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Binning of metagenomic sequencing data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metagenomics-binning/tutorial.html</td>
      <td>Metagenomics, Sequence assembly</td>
      <td>Sequence composition calculation, Genome assembly, Statistical calculation, Validation, Sequencing quality control, Sequence assembly validation</td>
      <td>2023-12-05</td>
      <td>2024-03-14</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>checkm_lineage_wf, megahit</td>
      <td>GalaxyTrakr, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1134</td>
      <td>1367</td>
      <td>1.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>QIIME 2 Moving Pictures</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/qiime2-moving-pictures/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis</td>
      <td></td>
      <td>2024-03-14</td>
      <td>2024-03-14</td>
      <td>2</td>
      <td>external</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>1</td>
      <td>5.0</td>
      <td>72</td>
      <td>94</td>
      <td>3.416667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>16S Microbial Analysis with mothur (short)</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/mothur-miseq-sop-short/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis</td>
      <td>Phylogenetic tree reconstruction, Sequence read processing, Sequencing quality control, Visualisation, Phylogenetic tree generation, Taxonomic classification, Phylogenetic analysis, DNA barcoding, Sequence clustering, Phylogenetic tree analysis</td>
      <td>2019-05-13</td>
      <td>2024-05-31</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>mothur_summary_seqs, mothur_classify_seqs, mothur_pre_cluster, mothur_rarefaction_single, mothur_remove_lineage, mothur_unique_seqs, collapse_dataset, XY_Plot_1, newick_display, mothur_sub_sample, mothur_summary_single, mothur_venn, mothur_make_shared, mothur_heatmap_sim, mothur_dist_shared, mothur_screen_seqs, mothur_count_seqs, mothur_cluster_split, mothur_remove_seqs, mothur_classify_otu, mothur_filter_seqs, mothur_remove_groups, mothur_cluster, mothur_count_groups, mothur_dist_seqs, mothur_chimera_vsearch, mothur_tree_shared</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.fr</td>
      <td>9</td>
      <td>4.666667</td>
      <td>1485</td>
      <td>2824</td>
      <td>6.050000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>16S Microbial Analysis with mothur (extended)</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/mothur-miseq-sop/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis</td>
      <td>Phylogenetic tree reconstruction, Sequence read processing, Sequencing quality control, Visualisation, Phylogenetic tree generation, Taxonomic classification, Phylogenetic analysis, DNA barcoding, Sequence clustering, Phylogenetic tree analysis</td>
      <td>2017-02-12</td>
      <td>2024-03-14</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>mothur_make_contigs, mothur_seq_error, mothur_summary_seqs, mothur_classify_seqs, mothur_pre_cluster, mothur_align_seqs, mothur_rarefaction_single, mothur_remove_lineage, mothur_unique_seqs, mothur_taxonomy_to_krona, XY_Plot_1, newick_display, mothur_sub_sample, mothur_summary_single, mothur_make_biom, mothur_venn, mothur_make_shared, mothur_heatmap_sim, mothur_dist_shared, mothur_screen_seqs, mothur_get_groups, mothur_count_seqs, mothur_cluster_split, mothur_remove_seqs, mothur_classify_otu, mothur_filter_seqs, mothur_remove_groups, mothur_cluster, taxonomy_krona_chart, mothur_count_groups, mothur_dist_seqs, mothur_chimera_vsearch, mothur_tree_shared</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>44</td>
      <td>4.613636</td>
      <td>3646</td>
      <td>5683</td>
      <td>3.133333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>M. tuberculosis Variant Analysis</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/tb-variant-analysis/tutorial.html</td>
      <td>Genomics, Microbiology, Sequence assembly, Genetic variation, Public health and epidemiology, Infectious disease</td>
      <td>Local alignment, Sequence composition calculation, Sequence alignment analysis, Global alignment, Variant calling, Validation, Sequencing quality control, Phylogenetic tree generation, Antimicrobial resistance prediction, Taxonomic classification, Genome visualisation, Statistical calculation, Sequence alignment, Sequence analysis, Phylogenetic tree visualisation, Sequence contamination filtering</td>
      <td>2020-07-25</td>
      <td>2024-06-12</td>
      <td>75</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tb_variant_filter, bcftools_consensus, jvarkit_wgscoverageplotter, fastp, jbrowse, snippy, kraken2, EMBOSS:%20seqret84, qualimap_bamqc, fastqc, tb_profiler_profile, EMBOSS: seqret84, mosdepth, tbvcfreport, __FLATTEN__, multiqc, upload1, tp_sed_tool, samtools_stats, tp_awk_tool</td>
      <td></td>
      <td></td>
      <td>25</td>
      <td>4.64</td>
      <td>5340</td>
      <td>8587</td>
      <td>2.966667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Identification of AMR genes in an assembled bacterial genome</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/amr-gene-detection/tutorial.html</td>
      <td>Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology, Sequence analysis, Infectious disease, Antimicrobial Resistance</td>
      <td>Genome annotation, Genome visualisation, Read mapping</td>
      <td>2024-01-23</td>
      <td>2024-03-13</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>upload1, staramr_search, bakta, Grep1, bowtie2, tbl2gff3, jbrowse</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr</td>
      <td>Galaxy@AuBi</td>
      <td>0</td>
      <td></td>
      <td>726</td>
      <td>1061</td>
      <td>3.066667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Tree thinking for tuberculosis evolution and epidemiology</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/mtb_phylogeny/tutorial.html</td>
      <td>Genomics, Microbiology, Phylogeny, Infectious disease</td>
      <td>Sequence analysis, Phylogenetic tree analysis</td>
      <td>2022-03-16</td>
      <td>2023-02-15</td>
      <td>19</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>raxml, interactive_tool_rstudio, upload1</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>6</td>
      <td>4.333333</td>
      <td>1211</td>
      <td>1857</td>
      <td>2.266667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Identifying tuberculosis transmission links: from SNPs to transmission clusters</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/mtb_transmission/tutorial.html</td>
      <td>Genomics, Microbiology, Infectious disease, DNA polymorphism</td>
      <td>Antimicrobial resistance prediction, Phylogenetic tree generation, Phylogenetic tree visualisation, Variant calling</td>
      <td>2022-03-16</td>
      <td>2024-03-12</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>snippy, tb_variant_filter, tp_replace_in_line, trimmomatic, bcftools_consensus, snp_sites, tp_sed_tool, addName, tp_cat, upload1, collapse_dataset, Grep1, __MERGE_COLLECTION__, tp_easyjoin_tool, tb_profiler_profile, samtools_view, tp_grep_tool, snp_dists</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>7</td>
      <td>4.0</td>
      <td>1377</td>
      <td>2211</td>
      <td>3.183333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>metaQuantome 3: Taxonomy</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaquantome-taxonomy/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Metatranscriptomics, Microbial ecology, Metagenomics, Taxonomy</td>
      <td>Statistical inference, Quantification, Functional clustering, Visualisation, Filtering, Principal component visualisation, Indexing, Query and retrieval, Differential protein expression analysis, Heat map generation</td>
      <td>2020-10-29</td>
      <td>2024-03-14</td>
      <td>36</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>metaquantome_filter, metaquantome_expand, metaquantome_sample, metaquantome_viz, metaquantome_db, metaquantome_stat</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>357</td>
      <td>487</td>
      <td>2.733333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Metaproteomics tutorial</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaproteomics/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Biodiversity, Taxonomy</td>
      <td>Prediction and recognition, Visualisation</td>
      <td>2017-06-28</td>
      <td>2024-05-29</td>
      <td>57</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>search_gui, peptide_shaker, sqlite_to_tabular, query_tabular, unipept</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>1</td>
      <td>5.0</td>
      <td>2198</td>
      <td>3676</td>
      <td>2.683333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>metaQuantome 1: Data creation</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaquantome-data-creation/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Biodiversity, Taxonomy</td>
      <td>Prediction and recognition, Formatting, Visualisation, Label-free quantification, Filtering</td>
      <td>2020-10-16</td>
      <td>2024-03-14</td>
      <td>90</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>flashlfq, Cut1, tp_replace_in_line, Remove beginning1, tp_replace_in_column, Grep1, unipept, peptide_shaker, regex1, msconvert, query_tabular, Filter1, search_gui</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>757</td>
      <td>1325</td>
      <td>3.866667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>metaQuantome 2: Function</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaquantome-function/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Metatranscriptomics, Microbial ecology, Metagenomics</td>
      <td>Statistical inference, Quantification, Functional clustering, Visualisation, Filtering, Principal component visualisation, Indexing, Query and retrieval, Differential protein expression analysis, Heat map generation</td>
      <td>2020-10-29</td>
      <td>2024-03-14</td>
      <td>38</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>metaquantome_filter, metaquantome_expand, metaquantome_sample, metaquantome_viz, metaquantome_db, metaquantome_stat</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>337</td>
      <td>489</td>
      <td>3.866667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Calling variants in non-diploid systems</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/non-dip/tutorial.html</td>
      <td>Genomics, Sequence assembly, DNA polymorphism, Microbiology, Sequence analysis, Genetic variation</td>
      <td>Sequence composition calculation, Sequence alignment analysis, Generation, Sequence alignment, Variant calling, Genome indexing, Sequencing quality control, Formatting, Read mapping, Statistical calculation, Data handling</td>
      <td>2017-02-16</td>
      <td>2024-03-14</td>
      <td>39</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>picard_MergeSamFiles, fastqc, Cut1, picard_MarkDuplicates, freebayes, vcf2tsv, bamFilter, bwa_mem, bamleftalign, vcffilter2</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz, UseGalaxy.org.au</td>
      <td>3</td>
      <td>2.333333</td>
      <td>6849</td>
      <td>13178</td>
      <td>2.050000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>M. tuberculosis Variant Analysis</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/tb-variant-analysis/tutorial.html</td>
      <td>Genomics, Microbiology, Sequence assembly, Genetic variation, Public health and epidemiology, Infectious disease</td>
      <td>Local alignment, Sequence composition calculation, Sequence alignment analysis, Global alignment, Variant calling, Validation, Sequencing quality control, Phylogenetic tree generation, Antimicrobial resistance prediction, Taxonomic classification, Genome visualisation, Statistical calculation, Sequence alignment, Sequence analysis, Phylogenetic tree visualisation, Sequence contamination filtering</td>
      <td>2020-07-25</td>
      <td>2024-06-12</td>
      <td>75</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tb_variant_filter, bcftools_consensus, jvarkit_wgscoverageplotter, fastp, jbrowse, snippy, kraken2, EMBOSS:%20seqret84, qualimap_bamqc, fastqc, tb_profiler_profile, EMBOSS: seqret84, mosdepth, tbvcfreport, __FLATTEN__, multiqc, upload1, tp_sed_tool, samtools_stats, tp_awk_tool</td>
      <td></td>
      <td></td>
      <td>25</td>
      <td>4.64</td>
      <td>5340</td>
      <td>8587</td>
      <td>2.966667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Microbial Variant Calling</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/microbial-variants/tutorial.html</td>
      <td>Genomics, Sequence assembly, DNA polymorphism, Microbiology, Sequence analysis, Genetic variation</td>
      <td>Genome visualisation, Phylogenetic tree generation, Phylogenetic tree visualisation, Variant calling</td>
      <td>2018-02-26</td>
      <td>2024-03-14</td>
      <td>36</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>snippy, jbrowse</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>GalaxyTrakr, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>12</td>
      <td>4.416667</td>
      <td>11631</td>
      <td>19648</td>
      <td>1.983333</td>
      <td>0</td>
    </tr>
  </tbody>
</table>

</div>