<script src="https://code.jquery.com/jquery-3.7.0.js"></script>
<script src="https://cdn.datatables.net/1.13.6/js/jquery.dataTables.min.js"></script>
<script src="https://cdn.datatables.net/searchbuilder/1.6.0/js/dataTables.searchBuilder.min.js"></script>
<script src="https://cdn.datatables.net/datetime/1.5.1/js/dataTables.dateTime.min.js"></script>

<script src="https://cdn.datatables.net/responsive/2.5.0/js/dataTables.responsive.min.js"></script>

<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/responsive/2.2.3/css/responsive.dataTables.min.css">

<link rel="stylesheet" href="https://cdn.datatables.net/1.13.6/css/jquery.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/searchbuilder/1.6.0/css/searchBuilder.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/datetime/1.5.1/css/dataTables.dateTime.min.css">

<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" />
<script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>

<style type="text/css">
  .tooltip {
    font-family: Georgia;
    font-size: 18px;
  }
  .tooltip .tooltip-inner {
    background-color: #00FFFF;
    color: #000000;
    min-width: 400px;
  }
  </style>

<script>
    $(document).ready(function() {
    $('#dataframe').DataTable( {
        "sScrollX": "100%",
        "dom": "Qlfrtip",

        responsive: {
            details: {
                type: 'column'
            }
        },
        "bAutoWidth": false,
        columnDefs: [ {
            className: 'control',
            orderable: false,
            targets:   [ 0 ],
            width: 100,
        },
        { type: 'natural', targets: 1 } // allow normal sorting
        ],

    });
});
</script>

<div class="display nowrap" style="width:80%" width="100%">

<table class="dataframe display nowrap" id="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th>Expand</th>
      <th>Suite ID</th>
      <th>Tool IDs</th>
      <th>Description</th>
      <th>Suite first commit date</th>
      <th>Homepage</th>
      <th>Suite version</th>
      <th>Suite conda package</th>
      <th>Latest suite conda package version</th>
      <th>Suite version status</th>
      <th>ToolShed categories</th>
      <th>EDAM operations</th>
      <th>EDAM reduced operations</th>
      <th>EDAM topics</th>
      <th>EDAM reduced topics</th>
      <th>Suite owner</th>
      <th>Suite source</th>
      <th>Suite parsed folder</th>
      <th>bio.tool ID</th>
      <th>bio.tool name</th>
      <th>bio.tool description</th>
      <th>biii ID</th>
      <th>Number of tools on UseGalaxy.org (Main)</th>
      <th>Number of tools on UseGalaxy.org.au</th>
      <th>Number of tools on UseGalaxy.eu</th>
      <th>Number of tools on UseGalaxy.fr</th>
      <th>Number of tools on APOSTL</th>
      <th>Number of tools on BioBix</th>
      <th>Number of tools on CIRM-CFBP</th>
      <th>Number of tools on ChemFlow</th>
      <th>Number of tools on Coloc-stats</th>
      <th>Number of tools on CorGAT</th>
      <th>Number of tools on FreeBioinfo</th>
      <th>Number of tools on GASLINI</th>
      <th>Number of tools on Galaxy@AuBi</th>
      <th>Number of tools on Galaxy@Pasteur</th>
      <th>Number of tools on GalaxyTrakr</th>
      <th>Number of tools on Genomic Hyperbrowser</th>
      <th>Number of tools on HyPhy HIV NGS Tools</th>
      <th>Number of tools on IPK Galaxy Blast Suite</th>
      <th>Number of tools on ImmPort Galaxy</th>
      <th>Number of tools on InteractoMIX</th>
      <th>Number of tools on Lebanese University Galaxy</th>
      <th>Number of tools on MISSISSIPPI</th>
      <th>Number of tools on Mandoiu Lab</th>
      <th>Number of tools on MiModD NacreousMap</th>
      <th>Number of tools on Palfinder</th>
      <th>Number of tools on PepSimili</th>
      <th>Number of tools on PhagePromotor</th>
      <th>Number of tools on UseGalaxy.be</th>
      <th>Number of tools on UseGalaxy.cz</th>
      <th>Number of tools on UseGalaxy.no</th>
      <th>Number of tools on Viral Variant Visualizer (VVV)</th>
      <th>Suite users (last 5 years) (usegalaxy.eu)</th>
      <th>Suite users (usegalaxy.eu)</th>
      <th>Suite runs (last 5 years) (usegalaxy.eu)</th>
      <th>Suite runs (usegalaxy.eu)</th>
      <th>Suite users (last 5 years) (usegalaxy.org)</th>
      <th>Suite users (usegalaxy.org)</th>
      <th>Suite runs (last 5 years) (usegalaxy.org)</th>
      <th>Suite runs (usegalaxy.org)</th>
      <th>Suite users (last 5 years) (usegalaxy.org.au)</th>
      <th>Suite users (usegalaxy.org.au)</th>
      <th>Suite runs (last 5 years) (usegalaxy.org.au)</th>
      <th>Suite runs (usegalaxy.org.au)</th>
      <th>Suite users (last 5 years) (usegalaxy.fr)</th>
      <th>Suite users (usegalaxy.fr)</th>
      <th>Suite runs (last 5 years) (usegalaxy.fr)</th>
      <th>Suite runs (usegalaxy.fr)</th>
      <th>Suite runs on main servers</th>
      <th>Suite runs (last 5 years) on main servers</th>
      <th>Suite users on main servers</th>
      <th>Suite users (last 5 years) on main servers</th>
      <th>Related Workflows</th>
      <th>Related Tutorials</th>
      <th>To keep</th>
      <th>Deprecated</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td></td>
      <td>anndata</td>
      <td>anndata_export, anndata_import, anndata_inspect, anndata_manipulate, modify_loom</td>
      <td>Import, Export, Inspect and Manipulate Anndata and Loom objects</td>
      <td>2019-03-21</td>
      <td>https://anndata.readthedocs.io</td>
      <td>0.10.9</td>
      <td>anndata</td>
      <td>0.6.22.post1</td>
      <td>To update</td>
      <td>Single Cell, Spatial Omics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/anndata</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/anndata</td>
      <td>anndata</td>
      <td>anndata</td>
      <td>From https://anndata.readthedocs.io/en/latest/"Python package for handling annotated data matrices in memory and on disk, positioned between pandas and xarray."</td>
      <td></td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>787</td>
      <td>890</td>
      <td>47423</td>
      <td>51315</td>
      <td>459</td>
      <td>467</td>
      <td>23370</td>
      <td>23451</td>
      <td>139</td>
      <td>141</td>
      <td>11736</td>
      <td>12386</td>
      <td>2</td>
      <td>2</td>
      <td>56</td>
      <td>56</td>
      <td>87208</td>
      <td>82585</td>
      <td>1500</td>
      <td>1387</td>
      <td>https://usegalaxy.eu/published/workflow?id=030e978022e19375, https://usegalaxy.eu/published/workflow?id=0e6e181234b48121, https://usegalaxy.eu/published/workflow?id=171d6fca2c3339e2, https://usegalaxy.eu/published/workflow?id=21315ffd2df2f159, https://usegalaxy.eu/published/workflow?id=35c092316cdc27d1, https://usegalaxy.eu/published/workflow?id=37e51e215684bdcc, https://usegalaxy.eu/published/workflow?id=3adf77a0576af98c, https://usegalaxy.eu/published/workflow?id=3fe3b9a57762841e, https://usegalaxy.eu/published/workflow?id=409ac72c9906f16a, https://usegalaxy.eu/published/workflow?id=4756053b2596b6cf, https://usegalaxy.eu/published/workflow?id=49a6ddcba35bf085, https://usegalaxy.eu/published/workflow?id=5b8e4ae3447934d9, https://usegalaxy.eu/published/workflow?id=704e6c9bc51db49f, https://usegalaxy.eu/published/workflow?id=715ca378c1d68de3, https://usegalaxy.eu/published/workflow?id=73aa794916b91bb2, https://usegalaxy.eu/published/workflow?id=767f8c990316b66a, https://usegalaxy.eu/published/workflow?id=77e85bde57477194, https://usegalaxy.eu/published/workflow?id=78dc6fca5b58ec05, https://usegalaxy.eu/published/workflow?id=7d22eea4278570e8, https://usegalaxy.eu/published/workflow?id=921cab3e6faf30be, https://usegalaxy.eu/published/workflow?id=9a657909d6191203, https://usegalaxy.eu/published/workflow?id=9a9bbd8046da7b19, https://usegalaxy.eu/published/workflow?id=9d15b328d8408715, https://usegalaxy.eu/published/workflow?id=b5627201bb4f6943, https://usegalaxy.eu/published/workflow?id=d1a1e6070c0a36ca, https://usegalaxy.eu/published/workflow?id=d20fbb95f15386f5, https://usegalaxy.eu/published/workflow?id=d70dc4cfeedb690b, https://usegalaxy.eu/published/workflow?id=ddced7b107e4707a, https://usegalaxy.eu/published/workflow?id=decd6ad0b41289ef, https://usegalaxy.eu/published/workflow?id=e2391c40d24d403f, https://usegalaxy.eu/published/workflow?id=e472096116cbe392, https://usegalaxy.eu/published/workflow?id=e4e969e9943cdaa5, https://usegalaxy.eu/published/workflow?id=f05754467a439134, https://usegalaxy.eu/published/workflow?id=f689faf267fbe504, https://usegalaxy.org.au/published/workflow?id=102eb726f63f3190, https://usegalaxy.org.au/published/workflow?id=218ffb4438833bdd, https://usegalaxy.org.au/published/workflow?id=471615c412b305f2, https://usegalaxy.org.au/published/workflow?id=57321b64220d5835, https://usegalaxy.org.au/published/workflow?id=8055596d732590d6, https://usegalaxy.org.au/published/workflow?id=94241afd07e8a8b7, https://usegalaxy.org/published/workflow?id=dd43ae9a5b303086, https://usegalaxy.org/published/workflow?id=ea3257e0ef9f505a, https://usegalaxy.org/published/workflow?id=f35ccb693cd874c1, https://workflowhub.eu/workflows/1077?version=1, https://workflowhub.eu/workflows/1078?version=1, https://workflowhub.eu/workflows/1256?version=1, https://workflowhub.eu/workflows/1356?version=1, https://workflowhub.eu/workflows/1362?version=1, https://workflowhub.eu/workflows/1485?version=1, https://workflowhub.eu/workflows/1490?version=1, https://workflowhub.eu/workflows/1496?version=1, https://workflowhub.eu/workflows/1501?version=1, https://workflowhub.eu/workflows/1502?version=2, https://workflowhub.eu/workflows/1506?version=2, https://workflowhub.eu/workflows/1519?version=1, https://workflowhub.eu/workflows/1520?version=1, https://workflowhub.eu/workflows/1523?version=1, https://workflowhub.eu/workflows/1524?version=1, https://workflowhub.eu/workflows/1527?version=1, https://workflowhub.eu/workflows/1529?version=1, https://workflowhub.eu/workflows/1531?version=1, https://workflowhub.eu/workflows/1537?version=1, https://workflowhub.eu/workflows/1539?version=1, https://workflowhub.eu/workflows/1543?version=1, https://workflowhub.eu/workflows/1548?version=1, https://workflowhub.eu/workflows/1550?version=1, https://workflowhub.eu/workflows/1551?version=1, https://workflowhub.eu/workflows/1556?version=1, https://workflowhub.eu/workflows/1561?version=1, https://workflowhub.eu/workflows/1565?version=1, https://workflowhub.eu/workflows/1574?version=1, https://workflowhub.eu/workflows/1745?version=2, https://workflowhub.eu/workflows/468?version=3, https://workflowhub.eu/workflows/512?version=2, https://workflowhub.eu/workflows/513?version=3, https://workflowhub.eu/workflows/646?version=2</td>
      <td>single-cell/bulk-music-2-preparescref, single-cell/pseudobulk-analysis, single-cell/scatac-batch-correction-snapatac2, single-cell/scatac-preprocessing-tenx, single-cell/scatac-standard-processing-snapatac2, single-cell/scrna-case_alevin, single-cell/scrna-case_alevin-combine-datasets, single-cell/scrna-case_basic-pipeline, single-cell/scrna-case_cell-cycle, single-cell/scrna-case_monocle3-trajectories, single-cell/scrna-case_trajectories, single-cell/scrna-data-ingest, single-cell/scrna-ncbi-anndata, single-cell/scrna-plant, single-cell/scrna-scanpy-pbmc3k</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>baredsc</td>
      <td>baredsc_1d, baredsc_2d, baredsc_combine_1d, baredsc_combine_2d</td>
      <td>baredSC is a tool that uses a Monte-Carlo Markov Chain to estimate a confidence interval on the probability density function (PDF) of expression of one or two genes from single-cell RNA-seq data.</td>
      <td>2023-09-06</td>
      <td>https://github.com/lldelisle/baredSC</td>
      <td>1.1.3</td>
      <td>baredsc</td>
      <td>1.1.3</td>
      <td>Up-to-date</td>
      <td>Single Cell, Transcriptomics, Visualization</td>
      <td>Data retrieval, Expression correlation analysis, Differential gene expression profiling</td>
      <td>Data retrieval, Expression correlation analysis, Differential gene expression profiling</td>
      <td>RNA-Seq, Cytometry, Transcriptomics, Gene transcripts, Statistics and probability</td>
      <td>RNA-Seq, Cytometry, Transcriptomics, Gene transcripts, Statistics and probability</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/baredsc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/baredsc</td>
      <td>baredsc</td>
      <td>baredSC</td>
      <td>The baredSC (Bayesian Approach to Retreive Expression Distribution of Single Cell) is a tool that uses a Monte-Carlo Markov Chain to estimate a confidence interval on the probability density function (PDF) of expression of one or two genes from single-cell RNA-seq data.</td>
      <td></td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>115</td>
      <td>115</td>
      <td>12</td>
      <td>12</td>
      <td>180</td>
      <td>180</td>
      <td>1</td>
      <td>1</td>
      <td>336</td>
      <td>336</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>631</td>
      <td>631</td>
      <td>19</td>
      <td>19</td>
      <td>https://usegalaxy.eu/published/workflow?id=2da3c41ad5c11c3b, https://usegalaxy.eu/published/workflow?id=3410a75a2a4899dc, https://usegalaxy.eu/published/workflow?id=4025b5dca188e6a4, https://usegalaxy.eu/published/workflow?id=49d913cefc5daaf3, https://usegalaxy.eu/published/workflow?id=4eb8aec8a5be98b3, https://usegalaxy.eu/published/workflow?id=72c03f7b5bc9fb9d, https://usegalaxy.eu/published/workflow?id=7341ca7572447d60, https://usegalaxy.eu/published/workflow?id=967f890c0883f72a, https://usegalaxy.eu/published/workflow?id=b3d38025c7ff72ed, https://usegalaxy.eu/published/workflow?id=cdf782de6c67477a, https://usegalaxy.eu/published/workflow?id=da41880826c3d758, https://usegalaxy.eu/published/workflow?id=fb08922b0bd812a9, https://usegalaxy.org.au/published/workflow?id=2436bc493e76f438, https://usegalaxy.org.au/published/workflow?id=31a664d151978dc6, https://usegalaxy.org.au/published/workflow?id=481c666fbc7ffa02, https://usegalaxy.org.au/published/workflow?id=51aaafbd00b91717, https://usegalaxy.org.au/published/workflow?id=579da54989f5d55c, https://usegalaxy.org.au/published/workflow?id=5e3ff382ab802baf, https://usegalaxy.org.au/published/workflow?id=ac4799c5009f98d3, https://usegalaxy.org.au/published/workflow?id=b73280eaeb77a0a9, https://usegalaxy.org.au/published/workflow?id=d01ff75a4943fa79, https://usegalaxy.org.au/published/workflow?id=d6521e6c247eeaaf, https://usegalaxy.org.au/published/workflow?id=ecafc3d43b4c63f7, https://usegalaxy.org.au/published/workflow?id=ffeef94c4539c3e5, https://usegalaxy.org/published/workflow?id=0f6bb7847f8769af, https://usegalaxy.org/published/workflow?id=1a08efa0bf83d225, https://usegalaxy.org/published/workflow?id=3d0f92e7f63978b6, https://usegalaxy.org/published/workflow?id=6cad14c78ec330ac, https://usegalaxy.org/published/workflow?id=6f1da4e0830e6051, https://usegalaxy.org/published/workflow?id=84112c29ee53a5db, https://usegalaxy.org/published/workflow?id=92b67534076ae889, https://usegalaxy.org/published/workflow?id=a8fa62e35610d85e, https://usegalaxy.org/published/workflow?id=e3366346808b1f55, https://usegalaxy.org/published/workflow?id=ebdf324634e7243c, https://usegalaxy.org/published/workflow?id=fd511165e30cb62c, https://usegalaxy.org/published/workflow?id=fda56c998bb88e2a, https://usegalaxy.org/published/workflow?id=ff84d87fd3638a3f, https://workflowhub.eu/workflows/615?version=6</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>cemitool</td>
      <td>cemitool</td>
      <td>Gene co-expression network analysis tool</td>
      <td>2022-10-10</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/CEMiTool.html</td>
      <td>1.30.0</td>
      <td>bioconductor-cemitool</td>
      <td>1.30.0</td>
      <td>Up-to-date</td>
      <td>Transcriptomics, RNA, Statistics, Single Cell</td>
      <td>Enrichment analysis, Pathway or network analysis</td>
      <td>Enrichment analysis, Pathway or network analysis</td>
      <td>Gene expression, Transcriptomics, Microarray experiment</td>
      <td>Transcriptomics, Microarray experiment</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cemitool</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cemitool</td>
      <td>cemitool</td>
      <td>CEMiTool</td>
      <td>It unifies the discovery and the analysis of coexpression gene modules in a fully automatic manner, while providing a user-friendly html report with high quality graphs. Our tool evaluates if modules contain genes that are over-represented by specific pathways or that are altered in a specific sample group. Additionally, CEMiTool is able to integrate transcriptomic data with interactome information, identifying the potential hubs on each network.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>45</td>
      <td>45</td>
      <td>354</td>
      <td>354</td>
      <td>87</td>
      <td>87</td>
      <td>584</td>
      <td>584</td>
      <td>6</td>
      <td>6</td>
      <td>154</td>
      <td>154</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1092</td>
      <td>1092</td>
      <td>138</td>
      <td>138</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>cite_seq_count</td>
      <td>cite_seq_count</td>
      <td>Count CMO/HTO</td>
      <td>2023-01-18</td>
      <td>https://github.com/Hoohm/CITE-seq-Count</td>
      <td>1.4.4</td>
      <td>cite-seq-count</td>
      <td>1.4.5</td>
      <td>To update</td>
      <td>Single Cell, Transcriptomics, Proteomics</td>
      <td>RNA-Seq quantification</td>
      <td>RNA-Seq quantification</td>
      <td>Transcriptomics, Immunoproteins and antigens</td>
      <td>Transcriptomics, Immunoproteins and antigens</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cite_seq_count</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cite_seq_count</td>
      <td>CITE-seq-Count</td>
      <td>CITE-seq-Count</td>
      <td>Tool for counting antibody TAGS from a CITE-seq and/or cell hashing experiment.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>23</td>
      <td>23</td>
      <td>6</td>
      <td>6</td>
      <td>21</td>
      <td>21</td>
      <td>1</td>
      <td>1</td>
      <td>290</td>
      <td>290</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>334</td>
      <td>334</td>
      <td>12</td>
      <td>12</td>
      <td>https://usegalaxy.eu/published/workflow?id=4cb8f782994763a4, https://usegalaxy.eu/published/workflow?id=5009d8d7a907a825, https://usegalaxy.eu/published/workflow?id=6ed089fe65347272, https://usegalaxy.eu/published/workflow?id=865715636d40b798, https://usegalaxy.eu/published/workflow?id=a18615f77ed3f371, https://usegalaxy.eu/published/workflow?id=a72e3cb6eb656102, https://usegalaxy.eu/published/workflow?id=c4327bccfa623f7e, https://usegalaxy.eu/published/workflow?id=e296e4ac941bf0de, https://usegalaxy.org.au/published/workflow?id=0f01e24e6c33a834, https://usegalaxy.org.au/published/workflow?id=109f0c68e617e104, https://usegalaxy.org.au/published/workflow?id=aa387984ac9e2f71, https://usegalaxy.org.au/published/workflow?id=b09530351a9e9ffb, https://usegalaxy.org.au/published/workflow?id=b88036506d0be117, https://usegalaxy.org.au/published/workflow?id=bcdbefda8e15e547, https://usegalaxy.org.au/published/workflow?id=d66f1a1c1ffd6275, https://usegalaxy.org/published/workflow?id=1d19cf599e1d4a0f, https://usegalaxy.org/published/workflow?id=3b88ab63573edb0f, https://usegalaxy.org/published/workflow?id=41c40e7aa6b24f51, https://usegalaxy.org/published/workflow?id=586b923221fb5da4, https://usegalaxy.org/published/workflow?id=73a8b48cbd6f1d8c, https://usegalaxy.org/published/workflow?id=90ac9dc2bd9d8092, https://usegalaxy.org/published/workflow?id=a737766dd083b280, https://workflowhub.eu/workflows/694?version=8</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>cosg</td>
      <td>cosg</td>
      <td>Marker gene identification for single-cell sequencing data using COSG.</td>
      <td>2024-05-29</td>
      <td>https://github.com/genecell/COSG</td>
      <td>1.0.1</td>
      <td>cosg</td>
      <td>1.0.3</td>
      <td>To update</td>
      <td>Transcriptomics, Sequence Analysis, Single Cell</td>
      <td>Nucleic acid sequence analysis</td>
      <td>Nucleic acid sequence analysis</td>
      <td>Bioinformatics</td>
      <td>Bioinformatics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cosg</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cosg</td>
      <td>cosg</td>
      <td>COSG</td>
      <td>COSG is a cosine similarity-based method for more accurate and scalable marker gene identification.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>https://usegalaxy.eu/published/workflow?id=9adf0402b641042c</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>crosscontamination_barcode_filter</td>
      <td>crosscontamination_barcode_filter</td>
      <td>Barcode contamination discovery tool</td>
      <td>2018-07-27</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/crosscontamination_barcode_filter</td>
      <td>0.3</td>
      <td>r-ggplot2</td>
      <td>2.2.1</td>
      <td>To update</td>
      <td>Transcriptomics, Visualization, Single Cell</td>
      <td>Filtering, Sequence contamination filtering, Sequencing quality control, RNA-Seq analysis</td>
      <td>Filtering, Sequence contamination filtering, RNA-Seq analysis</td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/crosscontamination_barcode_filter</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/crosscontamination_barcode_filter</td>
      <td>cross-contamination_barcode_filter</td>
      <td>Cross-contamination Barcode Filter</td>
      <td>This tool produces plots enabling the discovery of cross-contamination of samples across the full selection of barcodes used in a sequencing plate. It also filters false barcodes from each batch and retains only the 'real' barcodes.</td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>68</td>
      <td>99</td>
      <td>146</td>
      <td>415</td>
      <td>48</td>
      <td>49</td>
      <td>82</td>
      <td>144</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>559</td>
      <td>228</td>
      <td>148</td>
      <td>116</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>deg_annotate</td>
      <td>deg_annotate</td>
      <td>Annotate DESeq2/DEXSeq output tables</td>
      <td>2018-11-22</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/deg_annotate</td>
      <td>1.1.0+galaxy1</td>
      <td>bedtools</td>
      <td>2.31.1</td>
      <td>To update</td>
      <td>Transcriptomics, Single Cell</td>
      <td>RNA-Seq analysis, Gene functional annotation</td>
      <td>RNA-Seq analysis, Gene functional annotation</td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/deg_annotate</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/deg_annotate</td>
      <td>deg_annotate</td>
      <td>DEG Annotate</td>
      <td>This tool appends the output table of DESeq2 or DEXSeq with gene symbols, biotypes, positions etc. The information you want to add is configurable. This information should present in the input GTF/GFF file as attributes of feature you choose.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>5016</td>
      <td>5484</td>
      <td>28041</td>
      <td>30372</td>
      <td>6635</td>
      <td>6880</td>
      <td>40314</td>
      <td>41985</td>
      <td>1207</td>
      <td>1251</td>
      <td>6149</td>
      <td>6366</td>
      <td>185</td>
      <td>186</td>
      <td>649</td>
      <td>665</td>
      <td>79388</td>
      <td>75153</td>
      <td>13801</td>
      <td>13043</td>
      <td>https://usegalaxy.eu/published/workflow?id=63964ba7d9edce74, https://usegalaxy.eu/published/workflow?id=76389d8978ec26eb, https://usegalaxy.eu/published/workflow?id=81f1927bfe3ffea1, https://usegalaxy.eu/published/workflow?id=95c4d5549e820d88, https://usegalaxy.eu/published/workflow?id=a9469155eee12ff1, https://usegalaxy.eu/published/workflow?id=c5e6047b3b256b06, https://usegalaxy.eu/published/workflow?id=ccef778cfd9024d1, https://usegalaxy.eu/published/workflow?id=e764c594f070f107, https://usegalaxy.eu/published/workflow?id=e7e26967cf6cb8d8, https://usegalaxy.eu/published/workflow?id=f58526da1fd68108, https://usegalaxy.org.au/published/workflow?id=000cdc969e9f0886, https://usegalaxy.org.au/published/workflow?id=01997cb92a2ce2d5, https://usegalaxy.org.au/published/workflow?id=02b8d033d9b1fa05, https://usegalaxy.org.au/published/workflow?id=04ab43fb3e7840ef, https://usegalaxy.org.au/published/workflow?id=05b17b9d70c1db8b, https://usegalaxy.org.au/published/workflow?id=0614f24ebeb680de, https://usegalaxy.org.au/published/workflow?id=068f85b8b137bfd7, https://usegalaxy.org.au/published/workflow?id=0c567180448d8774, https://usegalaxy.org.au/published/workflow?id=0cfc1408fd9ea4e7, https://usegalaxy.org.au/published/workflow?id=0d1db28665d19bda, https://usegalaxy.org.au/published/workflow?id=10acf4f497172400, https://usegalaxy.org.au/published/workflow?id=114fb9589bb9db92, https://usegalaxy.org.au/published/workflow?id=14184bc2580a131f, https://usegalaxy.org.au/published/workflow?id=19315e6573c58239, https://usegalaxy.org.au/published/workflow?id=1c886f008a36b0b8, https://usegalaxy.org.au/published/workflow?id=20e5e6b233ffb951, https://usegalaxy.org.au/published/workflow?id=210ca013cc1cf185, https://usegalaxy.org.au/published/workflow?id=25ac9b4f2e39af97, https://usegalaxy.org.au/published/workflow?id=28705b86e0595ded, https://usegalaxy.org.au/published/workflow?id=2999a1891ffe201b, https://usegalaxy.org.au/published/workflow?id=2a0046582f5f15a1, https://usegalaxy.org.au/published/workflow?id=2c58509f80897972, https://usegalaxy.org.au/published/workflow?id=2cef9a1633e0a067, https://usegalaxy.org.au/published/workflow?id=2f7c0e3221cbca2f, https://usegalaxy.org.au/published/workflow?id=301fdb29c513ffbe, https://usegalaxy.org.au/published/workflow?id=30b2cd025e0300da, https://usegalaxy.org.au/published/workflow?id=31449596b22f81f4, https://usegalaxy.org.au/published/workflow?id=35384818ee1cbac2, https://usegalaxy.org.au/published/workflow?id=374311292d72c9f9, https://usegalaxy.org.au/published/workflow?id=383ce49fa7a367e9, https://usegalaxy.org.au/published/workflow?id=39d56981a6854d82, https://usegalaxy.org.au/published/workflow?id=3bc45aaed5da1058, https://usegalaxy.org.au/published/workflow?id=3c727d5cd349a846, https://usegalaxy.org.au/published/workflow?id=3ef07022656067b1, https://usegalaxy.org.au/published/workflow?id=410d0d19b6589c78, https://usegalaxy.org.au/published/workflow?id=42305ed876d19d08, https://usegalaxy.org.au/published/workflow?id=45c8789533c45abf, https://usegalaxy.org.au/published/workflow?id=4e096a68187ce652, https://usegalaxy.org.au/published/workflow?id=4efed79608f85604, https://usegalaxy.org.au/published/workflow?id=4f2818b6620037af, https://usegalaxy.org.au/published/workflow?id=4f562265cbb14cd6, https://usegalaxy.org.au/published/workflow?id=525612313de7591e, https://usegalaxy.org.au/published/workflow?id=5314a7a4732a0ef1, https://usegalaxy.org.au/published/workflow?id=544cb12a16f283b8, https://usegalaxy.org.au/published/workflow?id=5490661d0153962b, https://usegalaxy.org.au/published/workflow?id=5779029c2c13c75a, https://usegalaxy.org.au/published/workflow?id=5a4e49ef8693d7c4, https://usegalaxy.org.au/published/workflow?id=5cbee52cf03b6f9a, https://usegalaxy.org.au/published/workflow?id=5ea74382fb77eb10, https://usegalaxy.org.au/published/workflow?id=608f3ad8f485e7a8, https://usegalaxy.org.au/published/workflow?id=6204bbcfe56931eb, https://usegalaxy.org.au/published/workflow?id=625f3c52832b0591, https://usegalaxy.org.au/published/workflow?id=6448b82959076a29, https://usegalaxy.org.au/published/workflow?id=6469ade8f6f7c718, https://usegalaxy.org.au/published/workflow?id=66b5c7dd3e58193f, https://usegalaxy.org.au/published/workflow?id=67191ea9fe977e08, https://usegalaxy.org.au/published/workflow?id=69233fb6edfcdde9, https://usegalaxy.org.au/published/workflow?id=6cdbcd1f667d0f16, https://usegalaxy.org.au/published/workflow?id=6f0801fecbba5d3f, https://usegalaxy.org.au/published/workflow?id=71bad95f72c9f30d, https://usegalaxy.org.au/published/workflow?id=721e528c2ae6d8ee, https://usegalaxy.org.au/published/workflow?id=74186b006665daf8, https://usegalaxy.org.au/published/workflow?id=75113a1b41904d2a, https://usegalaxy.org.au/published/workflow?id=79522a1f7b0c53cb, https://usegalaxy.org.au/published/workflow?id=7c866ed4a3552d0d, https://usegalaxy.org.au/published/workflow?id=7cd1acb1da823cdc, https://usegalaxy.org.au/published/workflow?id=7ea616c652d08fb6, https://usegalaxy.org.au/published/workflow?id=7ef245f5ae3b916d, https://usegalaxy.org.au/published/workflow?id=7febc6ac6f106e57, https://usegalaxy.org.au/published/workflow?id=83680c1c6dffb05b, https://usegalaxy.org.au/published/workflow?id=83a535840b388980, https://usegalaxy.org.au/published/workflow?id=8483ae422665a6c1, https://usegalaxy.org.au/published/workflow?id=8483e8d3e8d0ddc9, https://usegalaxy.org.au/published/workflow?id=84a8eb379440656e, https://usegalaxy.org.au/published/workflow?id=8597166abda0a375, https://usegalaxy.org.au/published/workflow?id=85aede91db0122e4, https://usegalaxy.org.au/published/workflow?id=897dfa78c3b25356, https://usegalaxy.org.au/published/workflow?id=8b8a5b50f2edc72c, https://usegalaxy.org.au/published/workflow?id=8cab31db0bd3ff52, https://usegalaxy.org.au/published/workflow?id=8d196c8c62530dc0, https://usegalaxy.org.au/published/workflow?id=8eab5e2b2c683076, https://usegalaxy.org.au/published/workflow?id=8fb691706f47527a, https://usegalaxy.org.au/published/workflow?id=91cfd96f697ba034, https://usegalaxy.org.au/published/workflow?id=95a9070469a895c3, https://usegalaxy.org.au/published/workflow?id=95efb414f5984735, https://usegalaxy.org.au/published/workflow?id=975ddb0eb436f1fd, https://usegalaxy.org.au/published/workflow?id=97628b0604d5ec77, https://usegalaxy.org.au/published/workflow?id=9988bc2402f32dc1, https://usegalaxy.org.au/published/workflow?id=9b1afa59c087eacd, https://usegalaxy.org.au/published/workflow?id=9df67170c9260539, https://usegalaxy.org.au/published/workflow?id=9e8d9b7055b7c863, https://usegalaxy.org.au/published/workflow?id=9ec9fe2715d7cfdc, https://usegalaxy.org.au/published/workflow?id=a1c59e9f5c7327d2, https://usegalaxy.org.au/published/workflow?id=a69db7f914c06407, https://usegalaxy.org.au/published/workflow?id=a7346c25dfdf2e3e, https://usegalaxy.org.au/published/workflow?id=a7726d18d2a5fcf2, https://usegalaxy.org.au/published/workflow?id=a7a30d970584f9ac, https://usegalaxy.org.au/published/workflow?id=a7d1cf2bb636e695, https://usegalaxy.org.au/published/workflow?id=a9e59d38c6c710bc, https://usegalaxy.org.au/published/workflow?id=a9fa6101f1fb13e2, https://usegalaxy.org.au/published/workflow?id=aaaf580eb00459b6, https://usegalaxy.org.au/published/workflow?id=aae07f802002fdd0, https://usegalaxy.org.au/published/workflow?id=ad13b4a61e8987b1, https://usegalaxy.org.au/published/workflow?id=ae6892f1df022097, https://usegalaxy.org.au/published/workflow?id=ae773dfaa074cb72, https://usegalaxy.org.au/published/workflow?id=af3f40cba4d7afa6, https://usegalaxy.org.au/published/workflow?id=af990cca72794142, https://usegalaxy.org.au/published/workflow?id=afd9a8bbc07f9c3f, https://usegalaxy.org.au/published/workflow?id=b57bf134896c30ea, https://usegalaxy.org.au/published/workflow?id=b6db525b1b73394b, https://usegalaxy.org.au/published/workflow?id=bda98c5a0ea70dd0, https://usegalaxy.org.au/published/workflow?id=c00b5bc4d20f047d, https://usegalaxy.org.au/published/workflow?id=c1a0a3888d9ee3ac, https://usegalaxy.org.au/published/workflow?id=c296b3848708f6f1, https://usegalaxy.org.au/published/workflow?id=c2e5a43356042b30, https://usegalaxy.org.au/published/workflow?id=c3b3717502ff6f76, https://usegalaxy.org.au/published/workflow?id=c3bb21f20a3ce689, https://usegalaxy.org.au/published/workflow?id=c3e198bc2146b4ad, https://usegalaxy.org.au/published/workflow?id=c65a5af6c0383f8c, https://usegalaxy.org.au/published/workflow?id=ce7f6a1c824920e2, https://usegalaxy.org.au/published/workflow?id=cef68481b305a516, https://usegalaxy.org.au/published/workflow?id=d126940d33e63036, https://usegalaxy.org.au/published/workflow?id=d5717afa6cce6fb4, https://usegalaxy.org.au/published/workflow?id=d5ecfdbc04b24227, https://usegalaxy.org.au/published/workflow?id=db0bc7357c181222, https://usegalaxy.org.au/published/workflow?id=db3370a412c4de78, https://usegalaxy.org.au/published/workflow?id=dce29c35dfac48b1, https://usegalaxy.org.au/published/workflow?id=de1c04a2caf70647, https://usegalaxy.org.au/published/workflow?id=e08ef6b519f5fba0, https://usegalaxy.org.au/published/workflow?id=e1fd89e7248b6661, https://usegalaxy.org.au/published/workflow?id=e3cda847fe727b61, https://usegalaxy.org.au/published/workflow?id=e4ddf33ee5b5e259, https://usegalaxy.org.au/published/workflow?id=e52a3479c8216291, https://usegalaxy.org.au/published/workflow?id=e826584d78b69467, https://usegalaxy.org.au/published/workflow?id=e8d3651c07ce0bec, https://usegalaxy.org.au/published/workflow?id=eb3a4535204383fa, https://usegalaxy.org.au/published/workflow?id=ec79c2fab8eac15d, https://usegalaxy.org.au/published/workflow?id=ed2f72e3537e8a4a, https://usegalaxy.org.au/published/workflow?id=f0ced9a5d34b2aff, https://usegalaxy.org.au/published/workflow?id=f19d63d7c9a48160, https://usegalaxy.org.au/published/workflow?id=f268c181a47d0cab, https://usegalaxy.org.au/published/workflow?id=f5c0b0bc5e023e42, https://usegalaxy.org.au/published/workflow?id=f944f2d3fbd1f062, https://usegalaxy.org.au/published/workflow?id=fc64a77321752a3c, https://usegalaxy.org.au/published/workflow?id=fc6f8afdee980f0e, https://usegalaxy.org.au/published/workflow?id=fd8fb237f8dc24c3, https://usegalaxy.org.au/published/workflow?id=fe1de81a97578368, https://usegalaxy.org.au/published/workflow?id=fe893d72526eba71, https://usegalaxy.org.au/published/workflow?id=ff42213dda7d3e59, https://usegalaxy.org/published/workflow?id=098375608a1223aa, https://usegalaxy.org/published/workflow?id=19fcce20ecaf2ef6, https://usegalaxy.org/published/workflow?id=25cdc47baba556c5, https://usegalaxy.org/published/workflow?id=27144c7e44d3f56b, https://usegalaxy.org/published/workflow?id=35209d092e636cd8, https://usegalaxy.org/published/workflow?id=40a65b5bc1dcc0b4, https://usegalaxy.org/published/workflow?id=4c840284d2984fa0, https://usegalaxy.org/published/workflow?id=667493d318b86b7d, https://usegalaxy.org/published/workflow?id=68832752b883c226, https://usegalaxy.org/published/workflow?id=abf1aec0f3d48d03, https://usegalaxy.org/published/workflow?id=b0e19e7315b64c87, https://usegalaxy.org/published/workflow?id=b77fa73d4fbde131, https://usegalaxy.org/published/workflow?id=b882703ca6fea3a8, https://usegalaxy.org/published/workflow?id=dcc7c5fdef8780fd, https://usegalaxy.org/published/workflow?id=e40c2e7f6b73fb11, https://usegalaxy.org/published/workflow?id=e790587f0a61fbad, https://usegalaxy.org/published/workflow?id=eb4b32fcf4a9f64a, https://usegalaxy.org/published/workflow?id=f82f893b94cfa1ac, https://usegalaxy.org/published/workflow?id=f92c83df28d4896f, https://workflowhub.eu/workflows/1201?version=5, https://workflowhub.eu/workflows/1708?version=1, https://workflowhub.eu/workflows/1713?version=2</td>
      <td>transcriptomics/ref-based, transcriptomics/rna-seq-analysis-clustering-viz</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>deseq2</td>
      <td>deseq2</td>
      <td>Differential gene expression analysis based on the negative binomial distribution</td>
      <td>2014-12-03</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/DESeq2.html</td>
      <td>2.11.40.8</td>
      <td>bioconductor-deseq2</td>
      <td>1.46.0</td>
      <td>To update</td>
      <td>Transcriptomics, RNA, Statistics, Single Cell</td>
      <td>Differential gene expression analysis, RNA-Seq analysis</td>
      <td>Differential gene expression analysis, RNA-Seq analysis</td>
      <td>RNA-Seq</td>
      <td>RNA-Seq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/deseq2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/deseq2</td>
      <td>DESeq2</td>
      <td>DESeq2</td>
      <td>R/Bioconductor package for differential gene expression analysis based on the negative binomial distribution. Estimate variance-mean dependence in count data from high-throughput sequencing assays and test for differential expression based on a model using the negative binomial distribution.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>13707</td>
      <td>15994</td>
      <td>110682</td>
      <td>133936</td>
      <td>27214</td>
      <td>37996</td>
      <td>191581</td>
      <td>286575</td>
      <td>2980</td>
      <td>3614</td>
      <td>19899</td>
      <td>24448</td>
      <td>820</td>
      <td>824</td>
      <td>4511</td>
      <td>4530</td>
      <td>449489</td>
      <td>326673</td>
      <td>58428</td>
      <td>44721</td>
      <td>https://usegalaxy.eu/published/workflow?id=02a212cc9f134570, https://usegalaxy.eu/published/workflow?id=0de150b19102cb7e, https://usegalaxy.eu/published/workflow?id=0fa523529c4f73cf, https://usegalaxy.eu/published/workflow?id=12c80c5b5e2305d8, https://usegalaxy.eu/published/workflow?id=17ad465d9515cd25, https://usegalaxy.eu/published/workflow?id=1fe02012c96b731f, https://usegalaxy.eu/published/workflow?id=1ffc058273ab357e, https://usegalaxy.eu/published/workflow?id=20ada53472782c9b, https://usegalaxy.eu/published/workflow?id=2287afbf248d9c89, https://usegalaxy.eu/published/workflow?id=250e6058f160825f, https://usegalaxy.eu/published/workflow?id=2d2d0c2231babc50, https://usegalaxy.eu/published/workflow?id=350d7e6976e15d30, https://usegalaxy.eu/published/workflow?id=5b46c8533cd95447, https://usegalaxy.eu/published/workflow?id=5c8186f1e342c522, https://usegalaxy.eu/published/workflow?id=63964ba7d9edce74, https://usegalaxy.eu/published/workflow?id=68e8042f11d5a3fd, https://usegalaxy.eu/published/workflow?id=7136ce9f18c74853, https://usegalaxy.eu/published/workflow?id=76389d8978ec26eb, https://usegalaxy.eu/published/workflow?id=81f1927bfe3ffea1, https://usegalaxy.eu/published/workflow?id=82ef590b5821805d, https://usegalaxy.eu/published/workflow?id=8b1e2fe9f4a07de0, https://usegalaxy.eu/published/workflow?id=95c4d5549e820d88, https://usegalaxy.eu/published/workflow?id=9c7a218993788493, https://usegalaxy.eu/published/workflow?id=9e01e24c5e749365, https://usegalaxy.eu/published/workflow?id=a9469155eee12ff1, https://usegalaxy.eu/published/workflow?id=bc79a9f797334b01, https://usegalaxy.eu/published/workflow?id=c5e6047b3b256b06, https://usegalaxy.eu/published/workflow?id=ca286c2fdc71e960, https://usegalaxy.eu/published/workflow?id=ccef778cfd9024d1, https://usegalaxy.eu/published/workflow?id=d41b52a15ba56f54, https://usegalaxy.eu/published/workflow?id=d6f7e86d52d43555, https://usegalaxy.eu/published/workflow?id=d72edb9f8c529f47, https://usegalaxy.eu/published/workflow?id=e7363b9f450c56b4, https://usegalaxy.eu/published/workflow?id=e764c594f070f107, https://usegalaxy.eu/published/workflow?id=e7e26967cf6cb8d8, https://usegalaxy.eu/published/workflow?id=e8202b7b6338005d, https://usegalaxy.eu/published/workflow?id=e95729ed24f7c7d0, https://usegalaxy.eu/published/workflow?id=f026c4b8341ff94c, https://usegalaxy.eu/published/workflow?id=f33b5135ee99e0dc, https://usegalaxy.eu/published/workflow?id=f58526da1fd68108, https://usegalaxy.fr/published/workflow?id=ff14dc4005e92cfd, https://usegalaxy.org.au/published/workflow?id=000cdc969e9f0886, https://usegalaxy.org.au/published/workflow?id=01997cb92a2ce2d5, https://usegalaxy.org.au/published/workflow?id=02b8d033d9b1fa05, https://usegalaxy.org.au/published/workflow?id=046a1c600a6b77c2, https://usegalaxy.org.au/published/workflow?id=04ab43fb3e7840ef, https://usegalaxy.org.au/published/workflow?id=05950987aa2eba5b, https://usegalaxy.org.au/published/workflow?id=05b17b9d70c1db8b, https://usegalaxy.org.au/published/workflow?id=0614f24ebeb680de, https://usegalaxy.org.au/published/workflow?id=068f85b8b137bfd7, https://usegalaxy.org.au/published/workflow?id=07c97b9e59c0218f, https://usegalaxy.org.au/published/workflow?id=0c567180448d8774, https://usegalaxy.org.au/published/workflow?id=0cfc1408fd9ea4e7, https://usegalaxy.org.au/published/workflow?id=0d1db28665d19bda, https://usegalaxy.org.au/published/workflow?id=0fc7c14c4d1ac8f5, https://usegalaxy.org.au/published/workflow?id=10acf4f497172400, https://usegalaxy.org.au/published/workflow?id=114fb9589bb9db92, https://usegalaxy.org.au/published/workflow?id=13a2c5bd07705985, https://usegalaxy.org.au/published/workflow?id=14184bc2580a131f, https://usegalaxy.org.au/published/workflow?id=16f4846b4d6bc155, https://usegalaxy.org.au/published/workflow?id=19315e6573c58239, https://usegalaxy.org.au/published/workflow?id=1ad81f95ace878ea, https://usegalaxy.org.au/published/workflow?id=1c886f008a36b0b8, https://usegalaxy.org.au/published/workflow?id=20e5e6b233ffb951, https://usegalaxy.org.au/published/workflow?id=210ca013cc1cf185, https://usegalaxy.org.au/published/workflow?id=25ac9b4f2e39af97, https://usegalaxy.org.au/published/workflow?id=28705b86e0595ded, https://usegalaxy.org.au/published/workflow?id=2999a1891ffe201b, https://usegalaxy.org.au/published/workflow?id=2a0046582f5f15a1, https://usegalaxy.org.au/published/workflow?id=2c58509f80897972, https://usegalaxy.org.au/published/workflow?id=2cef9a1633e0a067, https://usegalaxy.org.au/published/workflow?id=2f7c0e3221cbca2f, https://usegalaxy.org.au/published/workflow?id=301fdb29c513ffbe, https://usegalaxy.org.au/published/workflow?id=30b2cd025e0300da, https://usegalaxy.org.au/published/workflow?id=31449596b22f81f4, https://usegalaxy.org.au/published/workflow?id=35384818ee1cbac2, https://usegalaxy.org.au/published/workflow?id=374311292d72c9f9, https://usegalaxy.org.au/published/workflow?id=383ce49fa7a367e9, https://usegalaxy.org.au/published/workflow?id=39d56981a6854d82, https://usegalaxy.org.au/published/workflow?id=3bc45aaed5da1058, https://usegalaxy.org.au/published/workflow?id=3c727d5cd349a846, https://usegalaxy.org.au/published/workflow?id=3ce9dc804780e4a1, https://usegalaxy.org.au/published/workflow?id=3ef07022656067b1, https://usegalaxy.org.au/published/workflow?id=3f45685df9b9e792, https://usegalaxy.org.au/published/workflow?id=410d0d19b6589c78, https://usegalaxy.org.au/published/workflow?id=421adea2f259ec26, https://usegalaxy.org.au/published/workflow?id=42305ed876d19d08, https://usegalaxy.org.au/published/workflow?id=45c8789533c45abf, https://usegalaxy.org.au/published/workflow?id=4e096a68187ce652, https://usegalaxy.org.au/published/workflow?id=4efed79608f85604, https://usegalaxy.org.au/published/workflow?id=4f2818b6620037af, https://usegalaxy.org.au/published/workflow?id=4f562265cbb14cd6, https://usegalaxy.org.au/published/workflow?id=525612313de7591e, https://usegalaxy.org.au/published/workflow?id=53010c2f1f207429, https://usegalaxy.org.au/published/workflow?id=5314a7a4732a0ef1, https://usegalaxy.org.au/published/workflow?id=544cb12a16f283b8, https://usegalaxy.org.au/published/workflow?id=5490661d0153962b, https://usegalaxy.org.au/published/workflow?id=56599572e0364e2a, https://usegalaxy.org.au/published/workflow?id=5779029c2c13c75a, https://usegalaxy.org.au/published/workflow?id=5a4e49ef8693d7c4, https://usegalaxy.org.au/published/workflow?id=5be983b903469f07, https://usegalaxy.org.au/published/workflow?id=5cbee52cf03b6f9a, https://usegalaxy.org.au/published/workflow?id=5ce00d3eab862f4e, https://usegalaxy.org.au/published/workflow?id=5ea74382fb77eb10, https://usegalaxy.org.au/published/workflow?id=608f3ad8f485e7a8, https://usegalaxy.org.au/published/workflow?id=6204bbcfe56931eb, https://usegalaxy.org.au/published/workflow?id=625f3c52832b0591, https://usegalaxy.org.au/published/workflow?id=6448b82959076a29, https://usegalaxy.org.au/published/workflow?id=6469ade8f6f7c718, https://usegalaxy.org.au/published/workflow?id=66b5c7dd3e58193f, https://usegalaxy.org.au/published/workflow?id=67191ea9fe977e08, https://usegalaxy.org.au/published/workflow?id=677ef3322366c953, https://usegalaxy.org.au/published/workflow?id=69233fb6edfcdde9, https://usegalaxy.org.au/published/workflow?id=6a161c87d19a543c, https://usegalaxy.org.au/published/workflow?id=6b34c95a247d15e3, https://usegalaxy.org.au/published/workflow?id=6cdbcd1f667d0f16, https://usegalaxy.org.au/published/workflow?id=6f0801fecbba5d3f, https://usegalaxy.org.au/published/workflow?id=71bad95f72c9f30d, https://usegalaxy.org.au/published/workflow?id=721e528c2ae6d8ee, https://usegalaxy.org.au/published/workflow?id=73ff1aa9acc0ad2a, https://usegalaxy.org.au/published/workflow?id=74186b006665daf8, https://usegalaxy.org.au/published/workflow?id=75113a1b41904d2a, https://usegalaxy.org.au/published/workflow?id=76323f103844f75b, https://usegalaxy.org.au/published/workflow?id=79522a1f7b0c53cb, https://usegalaxy.org.au/published/workflow?id=7ad277dd448e57b3, https://usegalaxy.org.au/published/workflow?id=7becafd394472dfb, https://usegalaxy.org.au/published/workflow?id=7c866ed4a3552d0d, https://usegalaxy.org.au/published/workflow?id=7cd1acb1da823cdc, https://usegalaxy.org.au/published/workflow?id=7ea616c652d08fb6, https://usegalaxy.org.au/published/workflow?id=7ef245f5ae3b916d, https://usegalaxy.org.au/published/workflow?id=7febc6ac6f106e57, https://usegalaxy.org.au/published/workflow?id=83680c1c6dffb05b, https://usegalaxy.org.au/published/workflow?id=83a535840b388980, https://usegalaxy.org.au/published/workflow?id=8483ae422665a6c1, https://usegalaxy.org.au/published/workflow?id=8483e8d3e8d0ddc9, https://usegalaxy.org.au/published/workflow?id=84a8eb379440656e, https://usegalaxy.org.au/published/workflow?id=8597166abda0a375, https://usegalaxy.org.au/published/workflow?id=85aede91db0122e4, https://usegalaxy.org.au/published/workflow?id=863eee1b041b6c58, https://usegalaxy.org.au/published/workflow?id=879d4fb110a61f7a, https://usegalaxy.org.au/published/workflow?id=897dfa78c3b25356, https://usegalaxy.org.au/published/workflow?id=897e9d7432d231e9, https://usegalaxy.org.au/published/workflow?id=8b8a5b50f2edc72c, https://usegalaxy.org.au/published/workflow?id=8c40b4a911517cc5, https://usegalaxy.org.au/published/workflow?id=8cab31db0bd3ff52, https://usegalaxy.org.au/published/workflow?id=8d196c8c62530dc0, https://usegalaxy.org.au/published/workflow?id=8eab5e2b2c683076, https://usegalaxy.org.au/published/workflow?id=8fb691706f47527a, https://usegalaxy.org.au/published/workflow?id=91cfd96f697ba034, https://usegalaxy.org.au/published/workflow?id=925c167abd5610ae, https://usegalaxy.org.au/published/workflow?id=95a9070469a895c3, https://usegalaxy.org.au/published/workflow?id=95efb414f5984735, https://usegalaxy.org.au/published/workflow?id=975ddb0eb436f1fd, https://usegalaxy.org.au/published/workflow?id=97628b0604d5ec77, https://usegalaxy.org.au/published/workflow?id=978953f492b2b914, https://usegalaxy.org.au/published/workflow?id=981d7b02416b7734, https://usegalaxy.org.au/published/workflow?id=9898348d95033af2, https://usegalaxy.org.au/published/workflow?id=9988bc2402f32dc1, https://usegalaxy.org.au/published/workflow?id=9b1afa59c087eacd, https://usegalaxy.org.au/published/workflow?id=9df67170c9260539, https://usegalaxy.org.au/published/workflow?id=9e8d9b7055b7c863, https://usegalaxy.org.au/published/workflow?id=9ec9fe2715d7cfdc, https://usegalaxy.org.au/published/workflow?id=a1c59e9f5c7327d2, https://usegalaxy.org.au/published/workflow?id=a69db7f914c06407, https://usegalaxy.org.au/published/workflow?id=a7346c25dfdf2e3e, https://usegalaxy.org.au/published/workflow?id=a7726d18d2a5fcf2, https://usegalaxy.org.au/published/workflow?id=a7a30d970584f9ac, https://usegalaxy.org.au/published/workflow?id=a7d1cf2bb636e695, https://usegalaxy.org.au/published/workflow?id=a9e59d38c6c710bc, https://usegalaxy.org.au/published/workflow?id=a9fa6101f1fb13e2, https://usegalaxy.org.au/published/workflow?id=aa94c77b720a2810, https://usegalaxy.org.au/published/workflow?id=aaaf580eb00459b6, https://usegalaxy.org.au/published/workflow?id=aae07f802002fdd0, https://usegalaxy.org.au/published/workflow?id=ad13b4a61e8987b1, https://usegalaxy.org.au/published/workflow?id=ae5dca8108d1a8a6, https://usegalaxy.org.au/published/workflow?id=ae6892f1df022097, https://usegalaxy.org.au/published/workflow?id=ae773dfaa074cb72, https://usegalaxy.org.au/published/workflow?id=aed93da6fd6e4e13, https://usegalaxy.org.au/published/workflow?id=af3f40cba4d7afa6, https://usegalaxy.org.au/published/workflow?id=af990cca72794142, https://usegalaxy.org.au/published/workflow?id=afd9a8bbc07f9c3f, https://usegalaxy.org.au/published/workflow?id=b57bf134896c30ea, https://usegalaxy.org.au/published/workflow?id=b6db525b1b73394b, https://usegalaxy.org.au/published/workflow?id=bda98c5a0ea70dd0, https://usegalaxy.org.au/published/workflow?id=c00b5bc4d20f047d, https://usegalaxy.org.au/published/workflow?id=c1a0a3888d9ee3ac, https://usegalaxy.org.au/published/workflow?id=c296b3848708f6f1, https://usegalaxy.org.au/published/workflow?id=c2e5a43356042b30, https://usegalaxy.org.au/published/workflow?id=c3b3717502ff6f76, https://usegalaxy.org.au/published/workflow?id=c3bb21f20a3ce689, https://usegalaxy.org.au/published/workflow?id=c3e198bc2146b4ad, https://usegalaxy.org.au/published/workflow?id=c65a5af6c0383f8c, https://usegalaxy.org.au/published/workflow?id=cb960edf95d2dc35, https://usegalaxy.org.au/published/workflow?id=ce7f6a1c824920e2, https://usegalaxy.org.au/published/workflow?id=cef4a514f35576e5, https://usegalaxy.org.au/published/workflow?id=cef68481b305a516, https://usegalaxy.org.au/published/workflow?id=d126940d33e63036, https://usegalaxy.org.au/published/workflow?id=d5717afa6cce6fb4, https://usegalaxy.org.au/published/workflow?id=d5ecfdbc04b24227, https://usegalaxy.org.au/published/workflow?id=d7a65f577fe5f16b, https://usegalaxy.org.au/published/workflow?id=db0bc7357c181222, https://usegalaxy.org.au/published/workflow?id=db3370a412c4de78, https://usegalaxy.org.au/published/workflow?id=dce29c35dfac48b1, https://usegalaxy.org.au/published/workflow?id=de1c04a2caf70647, https://usegalaxy.org.au/published/workflow?id=e08ef6b519f5fba0, https://usegalaxy.org.au/published/workflow?id=e1fd89e7248b6661, https://usegalaxy.org.au/published/workflow?id=e3cda847fe727b61, https://usegalaxy.org.au/published/workflow?id=e4ddf33ee5b5e259, https://usegalaxy.org.au/published/workflow?id=e52a3479c8216291, https://usegalaxy.org.au/published/workflow?id=e826584d78b69467, https://usegalaxy.org.au/published/workflow?id=e8d3651c07ce0bec, https://usegalaxy.org.au/published/workflow?id=eb3a4535204383fa, https://usegalaxy.org.au/published/workflow?id=ec79c2fab8eac15d, https://usegalaxy.org.au/published/workflow?id=ed2f72e3537e8a4a, https://usegalaxy.org.au/published/workflow?id=f0ced9a5d34b2aff, https://usegalaxy.org.au/published/workflow?id=f19d63d7c9a48160, https://usegalaxy.org.au/published/workflow?id=f268c181a47d0cab, https://usegalaxy.org.au/published/workflow?id=f5c0b0bc5e023e42, https://usegalaxy.org.au/published/workflow?id=f944f2d3fbd1f062, https://usegalaxy.org.au/published/workflow?id=fb644a9649f36f40, https://usegalaxy.org.au/published/workflow?id=fc64a77321752a3c, https://usegalaxy.org.au/published/workflow?id=fc6f8afdee980f0e, https://usegalaxy.org.au/published/workflow?id=fd477917a3d27f61, https://usegalaxy.org.au/published/workflow?id=fd8fb237f8dc24c3, https://usegalaxy.org.au/published/workflow?id=fe1de81a97578368, https://usegalaxy.org.au/published/workflow?id=fe893d72526eba71, https://usegalaxy.org.au/published/workflow?id=ff42213dda7d3e59, https://usegalaxy.org/published/workflow?id=030814e5329ea62d, https://usegalaxy.org/published/workflow?id=092be3cf6e40282c, https://usegalaxy.org/published/workflow?id=0930f955bdac93e8, https://usegalaxy.org/published/workflow?id=098375608a1223aa, https://usegalaxy.org/published/workflow?id=19fcce20ecaf2ef6, https://usegalaxy.org/published/workflow?id=1bd0d7ef0aca1363, https://usegalaxy.org/published/workflow?id=25cdc47baba556c5, https://usegalaxy.org/published/workflow?id=27144c7e44d3f56b, https://usegalaxy.org/published/workflow?id=2d50de9b95ad0808, https://usegalaxy.org/published/workflow?id=36a92f1f6d2212a4, https://usegalaxy.org/published/workflow?id=4061e061f659449b, https://usegalaxy.org/published/workflow?id=40a65b5bc1dcc0b4, https://usegalaxy.org/published/workflow?id=420a332da24fb9e9, https://usegalaxy.org/published/workflow?id=4c0785fe74ab0a24, https://usegalaxy.org/published/workflow?id=4c840284d2984fa0, https://usegalaxy.org/published/workflow?id=53834a5fb46ca0f6, https://usegalaxy.org/published/workflow?id=5a272f266a1fa4ac, https://usegalaxy.org/published/workflow?id=667493d318b86b7d, https://usegalaxy.org/published/workflow?id=68832752b883c226, https://usegalaxy.org/published/workflow?id=69af7dd51aff958a, https://usegalaxy.org/published/workflow?id=6dada297572d77bd, https://usegalaxy.org/published/workflow?id=7e81153760fb4c90, https://usegalaxy.org/published/workflow?id=82681bd5f059e39a, https://usegalaxy.org/published/workflow?id=8700f593a3d642f5, https://usegalaxy.org/published/workflow?id=9bace540b35b1e61, https://usegalaxy.org/published/workflow?id=9e74e3455e47c813, https://usegalaxy.org/published/workflow?id=a743746151101352, https://usegalaxy.org/published/workflow?id=a8ddab36010ed4c3, https://usegalaxy.org/published/workflow?id=aae5b6d9beb860ba, https://usegalaxy.org/published/workflow?id=abf1aec0f3d48d03, https://usegalaxy.org/published/workflow?id=ac8b74b911778fbc, https://usegalaxy.org/published/workflow?id=b0e19e7315b64c87, https://usegalaxy.org/published/workflow?id=b77fa73d4fbde131, https://usegalaxy.org/published/workflow?id=b882703ca6fea3a8, https://usegalaxy.org/published/workflow?id=bc83c42b44c4b03f, https://usegalaxy.org/published/workflow?id=c8ee2da45b6889b9, https://usegalaxy.org/published/workflow?id=cd56ed91856fcc6f, https://usegalaxy.org/published/workflow?id=ce99c56e45d982cb, https://usegalaxy.org/published/workflow?id=cff93a6f24af5fee, https://usegalaxy.org/published/workflow?id=d2f7fd589a91124a, https://usegalaxy.org/published/workflow?id=d72e33effe07de02, https://usegalaxy.org/published/workflow?id=d761637d43ee0324, https://usegalaxy.org/published/workflow?id=d8afd017905d6702, https://usegalaxy.org/published/workflow?id=dab54002b3c0e351, https://usegalaxy.org/published/workflow?id=dcc7c5fdef8780fd, https://usegalaxy.org/published/workflow?id=dcf6f201b787d148, https://usegalaxy.org/published/workflow?id=dec0cf06fa059e65, https://usegalaxy.org/published/workflow?id=e26fabceece3b858, https://usegalaxy.org/published/workflow?id=e40c2e7f6b73fb11, https://usegalaxy.org/published/workflow?id=e790587f0a61fbad, https://usegalaxy.org/published/workflow?id=eb01b539d17580c3, https://usegalaxy.org/published/workflow?id=eb4b32fcf4a9f64a, https://usegalaxy.org/published/workflow?id=f82f893b94cfa1ac, https://usegalaxy.org/published/workflow?id=f92c83df28d4896f, https://usegalaxy.org/published/workflow?id=fb903de4aa4d5090, https://workflowhub.eu/workflows/1201?version=5, https://workflowhub.eu/workflows/1676?version=1, https://workflowhub.eu/workflows/1677?version=1, https://workflowhub.eu/workflows/1685?version=1, https://workflowhub.eu/workflows/1708?version=1, https://workflowhub.eu/workflows/1713?version=2, https://workflowhub.eu/workflows/1717?version=2, https://workflowhub.eu/workflows/338?version=1, https://workflowhub.eu/workflows/412?version=1</td>
      <td>transcriptomics/de-novo, transcriptomics/mirna-target-finder, transcriptomics/network-analysis-with-heinz, transcriptomics/ref-based, transcriptomics/rna-seq-analysis-clustering-viz, transcriptomics/srna</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>deseq2_normalization</td>
      <td>deseq2_normalization</td>
      <td>Normalizes gene hitlists</td>
      <td>2018-01-05</td>
      <td>http://artbio.fr</td>
      <td>1.40.2+galaxy1</td>
      <td>bioconductor-deseq2</td>
      <td>1.46.0</td>
      <td>To update</td>
      <td>Transcriptomics, Statistics, Single Cell</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/deseq2_normalization</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/deseq2_normalization</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>dexseq</td>
      <td>dexseq, dexseq_count, plotdexseq</td>
      <td>Inference of differential exon usage in RNA-Seq</td>
      <td>2015-09-25</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/DEXSeq.html</td>
      <td>1.48.0</td>
      <td>bioconductor-dexseq</td>
      <td>1.52.0</td>
      <td>To update</td>
      <td>Transcriptomics, RNA, Statistics, Single Cell</td>
      <td>Enrichment analysis, Exonic splicing enhancer prediction</td>
      <td>Enrichment analysis, Exonic splicing enhancer prediction</td>
      <td>RNA-Seq</td>
      <td>RNA-Seq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/dexseq</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/dexseq</td>
      <td>dexseq</td>
      <td>DEXSeq</td>
      <td>The package is focused on finding differential exon usage using RNA-seq exon counts between samples with different experimental designs. It provides functions that allows the user to make the necessary statistical tests based on a model that uses the negative binomial distribution to estimate the variance between biological replicates and generalized linear models for testing. The package also provides functions for the visualization and exploration of the results.</td>
      <td></td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>274</td>
      <td>481</td>
      <td>14442</td>
      <td>18733</td>
      <td>646</td>
      <td>1620</td>
      <td>16356</td>
      <td>30696</td>
      <td>65</td>
      <td>101</td>
      <td>2756</td>
      <td>4812</td>
      <td>8</td>
      <td>9</td>
      <td>97</td>
      <td>98</td>
      <td>54339</td>
      <td>33651</td>
      <td>2211</td>
      <td>993</td>
      <td>https://usegalaxy.org/published/workflow?id=82681bd5f059e39a</td>
      <td>transcriptomics/ref-based</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>dropletutils</td>
      <td>dropletutils</td>
      <td>DropletUtils - Utilities for handling droplet-based single-cell RNA-seq data</td>
      <td>2019-06-04</td>
      <td>https://bioconductor.org/packages/devel/bioc/html/DropletUtils.html</td>
      <td>1.10.0</td>
      <td>bioconductor-dropletutils</td>
      <td>1.26.0</td>
      <td>To update</td>
      <td>Single Cell, Sequence Analysis</td>
      <td>Loading, Community profiling</td>
      <td></td>
      <td>Gene expression, RNA-seq, Sequencing, Transcriptomics</td>
      <td>Sequencing, Transcriptomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/dropletutils</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/dropletutils</td>
      <td>dropletutils</td>
      <td>DropletUtils</td>
      <td>Provides a number of utility functions for handling single-cell (RNA-seq) data from droplet technologies such as 10X Genomics. This includes data loading, identification of cells from empty droplets, removal of barcode-swapped pseudo-cells, and downsampling of the count matrix.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>450</td>
      <td>517</td>
      <td>4128</td>
      <td>4902</td>
      <td>381</td>
      <td>385</td>
      <td>5328</td>
      <td>5345</td>
      <td>42</td>
      <td>46</td>
      <td>2038</td>
      <td>2190</td>
      <td>34</td>
      <td>34</td>
      <td>110</td>
      <td>110</td>
      <td>12547</td>
      <td>11604</td>
      <td>982</td>
      <td>907</td>
      <td>https://usegalaxy.eu/published/workflow?id=0f563f176bb245af, https://usegalaxy.eu/published/workflow?id=3403d5a8a71e0c3c, https://usegalaxy.eu/published/workflow?id=49a6ddcba35bf085, https://usegalaxy.eu/published/workflow?id=6a6e2c2335051fb5, https://usegalaxy.eu/published/workflow?id=704e6c9bc51db49f, https://usegalaxy.eu/published/workflow?id=7d22eea4278570e8, https://usegalaxy.eu/published/workflow?id=83ae28e23099a3f6, https://usegalaxy.eu/published/workflow?id=90a32c95b60f5221, https://usegalaxy.eu/published/workflow?id=a1cfb476e849fce3, https://usegalaxy.eu/published/workflow?id=a3cb75fd31ba0689, https://usegalaxy.eu/published/workflow?id=af7c349497a75314, https://usegalaxy.eu/published/workflow?id=b5627201bb4f6943, https://usegalaxy.eu/published/workflow?id=d79309343e2a5d62, https://usegalaxy.eu/published/workflow?id=e7494f29c8672f34, https://usegalaxy.eu/published/workflow?id=e7a9e1a7728848cd, https://usegalaxy.eu/published/workflow?id=fa960b78dfc0c2f8, https://usegalaxy.eu/published/workflow?id=fd61d070d9744f95, https://usegalaxy.org.au/published/workflow?id=00da6615bca12a3c, https://usegalaxy.org.au/published/workflow?id=0458ecd2761307d1, https://usegalaxy.org.au/published/workflow?id=10296e860e9ef4bb, https://usegalaxy.org.au/published/workflow?id=281ea78467e73147, https://usegalaxy.org.au/published/workflow?id=3fd10af90a07f771, https://usegalaxy.org.au/published/workflow?id=41b115a9be4b6002, https://usegalaxy.org.au/published/workflow?id=471615c412b305f2, https://usegalaxy.org.au/published/workflow?id=644d8e4c78df8e36, https://usegalaxy.org.au/published/workflow?id=738350fc2ca69c0b, https://usegalaxy.org.au/published/workflow?id=c64dcc1e6a67f3e4, https://usegalaxy.org.au/published/workflow?id=f627023a3401bf9b, https://usegalaxy.org/published/workflow?id=4bbe4c8a6c52d03a, https://usegalaxy.org/published/workflow?id=516a2181b41af94e, https://usegalaxy.org/published/workflow?id=69712265fdb10ed5, https://usegalaxy.org/published/workflow?id=7aa36b1722ff4c11, https://usegalaxy.org/published/workflow?id=8e6d24a14d2908c2, https://usegalaxy.org/published/workflow?id=a6d6a2e8867fa566, https://usegalaxy.org/published/workflow?id=b73e416c06790ac5, https://usegalaxy.org/published/workflow?id=be7dab2274c1acdd, https://usegalaxy.org/published/workflow?id=cdb6c9c7037a95b2, https://workflowhub.eu/workflows/1523?version=1, https://workflowhub.eu/workflows/1527?version=1, https://workflowhub.eu/workflows/1546?version=1</td>
      <td>single-cell/scrna-data-ingest, single-cell/scrna-preprocessing-tenx</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>edger</td>
      <td>edger</td>
      <td>Perform RNA-Seq differential expression analysis using edgeR pipeline</td>
      <td>2017-09-11</td>
      <td>http://bioconductor.org/packages/release/bioc/html/edgeR.html</td>
      <td>3.36.0</td>
      <td>bioconductor-edger</td>
      <td>4.4.0</td>
      <td>To update</td>
      <td>Transcriptomics, RNA, Statistics, Single Cell</td>
      <td>Differential gene expression analysis</td>
      <td>Differential gene expression analysis</td>
      <td>Genetics, RNA-Seq, ChIP-seq</td>
      <td>Genetics, RNA-Seq, ChIP-seq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/edger</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/edger</td>
      <td>edger</td>
      <td>edgeR</td>
      <td>Differential expression analysis of RNA-seq expression profiles with biological replication. Implements a range of statistical methodology based on the negative binomial distributions, including empirical Bayes estimation, exact tests, generalized linear models and quasi-likelihood tests. As well as RNA-seq, it be applied to differential signal analysis of other types of genomic data that produce counts, including ChIP-seq, SAGE and CAGE.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2767</td>
      <td>3276</td>
      <td>20681</td>
      <td>25641</td>
      <td>6109</td>
      <td>8376</td>
      <td>43822</td>
      <td>64826</td>
      <td>928</td>
      <td>1075</td>
      <td>7956</td>
      <td>10252</td>
      <td>29</td>
      <td>30</td>
      <td>172</td>
      <td>173</td>
      <td>100892</td>
      <td>72631</td>
      <td>12757</td>
      <td>9833</td>
      <td>https://usegalaxy.eu/published/workflow?id=0229f3f3835c2d53, https://usegalaxy.eu/published/workflow?id=04df470a38b10e94, https://usegalaxy.eu/published/workflow?id=c3a11e1ac1aa8383, https://usegalaxy.eu/published/workflow?id=c7b9fb6872444809, https://usegalaxy.eu/published/workflow?id=d1b3b62e7f91e217, https://usegalaxy.eu/published/workflow?id=e04be1ab5a16eb9f, https://usegalaxy.org.au/published/workflow?id=0cfc1408fd9ea4e7, https://usegalaxy.org.au/published/workflow?id=0fc7c14c4d1ac8f5, https://usegalaxy.org.au/published/workflow?id=5fe66b29dcfb22ab, https://usegalaxy.org.au/published/workflow?id=863eee1b041b6c58, https://usegalaxy.org/published/workflow?id=03258e0321162584, https://usegalaxy.org/published/workflow?id=13a3954b8894e1b1, https://usegalaxy.org/published/workflow?id=ce232859a466c75c, https://usegalaxy.org/published/workflow?id=d761637d43ee0324, https://workflowhub.eu/workflows/1207?version=2, https://workflowhub.eu/workflows/1556?version=1</td>
      <td>single-cell/pseudobulk-analysis</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>egsea</td>
      <td>egsea</td>
      <td>This tool implements the Ensemble of Gene Set Enrichment Analyses (EGSEA) method for gene set testing</td>
      <td>2017-12-28</td>
      <td>https://bioconductor.org/packages/release/bioc/html/EGSEA.html</td>
      <td>1.20.0</td>
      <td>bioconductor-egsea</td>
      <td>1.34.0</td>
      <td>To update</td>
      <td>Transcriptomics, RNA, Statistics, Single Cell</td>
      <td>Gene set testing</td>
      <td>Gene set testing</td>
      <td>Systems biology</td>
      <td>Systems biology</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/egsea</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/egsea</td>
      <td>egsea</td>
      <td>EGSEA</td>
      <td>This package implements the Ensemble of Gene Set Enrichment Analyses method for gene set testing.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>365</td>
      <td>467</td>
      <td>2481</td>
      <td>3462</td>
      <td>783</td>
      <td>865</td>
      <td>5041</td>
      <td>5849</td>
      <td>259</td>
      <td>282</td>
      <td>2025</td>
      <td>2377</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>11688</td>
      <td>9547</td>
      <td>1614</td>
      <td>1407</td>
      <td>https://usegalaxy.eu/published/workflow?id=12bf85cdc8102bb5, https://usegalaxy.eu/published/workflow?id=63c91ea322cd2819, https://usegalaxy.eu/published/workflow?id=732fa536cdc70c96, https://usegalaxy.eu/published/workflow?id=8b1e2fe9f4a07de0, https://usegalaxy.org.au/published/workflow?id=9f6e62f465589ca3, https://workflowhub.eu/workflows/1699?version=1</td>
      <td>transcriptomics/rna-seq-genes-to-pathways</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>episcanpy</td>
      <td>episcanpy_build_matrix, episcanpy_cluster_embed, episcanpy_preprocess</td>
      <td>EpiScanpy  Epigenomics single cell analysis in python</td>
      <td>2023-03-13</td>
      <td>https://github.com/colomemaria/epiScanpy</td>
      <td>0.3.2</td>
      <td>episcanpy</td>
      <td>0.4.0</td>
      <td>To update</td>
      <td>Single Cell, Epigenetics</td>
      <td>Enrichment analysis, Imputation</td>
      <td>Enrichment analysis, Imputation</td>
      <td>Epigenomics, Cell biology, DNA</td>
      <td>Epigenomics, Cell biology, DNA</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/episcanpy</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/episcanpy</td>
      <td>episcanpy</td>
      <td>epiScanpy</td>
      <td>Epigenomics Single Cell Analysis in Python.</td>
      <td></td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>53</td>
      <td>53</td>
      <td>1075</td>
      <td>1075</td>
      <td>53</td>
      <td>53</td>
      <td>406</td>
      <td>406</td>
      <td>2</td>
      <td>2</td>
      <td>766</td>
      <td>766</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2247</td>
      <td>2247</td>
      <td>108</td>
      <td>108</td>
      <td>https://workflowhub.eu/workflows/1502?version=2, https://workflowhub.eu/workflows/1506?version=2</td>
      <td>single-cell/scatac-preprocessing-tenx</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>fgsea</td>
      <td>fgsea</td>
      <td>Perform gene set testing using fgsea</td>
      <td>2018-10-19</td>
      <td>https://bioconductor.org/packages/release/bioc/html/fgsea.html</td>
      <td>1.8.0+galaxy1</td>
      <td>bioconductor-fgsea</td>
      <td>1.32.2</td>
      <td>To update</td>
      <td>Visualization, Transcriptomics, Statistics, Single Cell</td>
      <td>Gene-set enrichment analysis</td>
      <td>Gene-set enrichment analysis</td>
      <td>Genetics</td>
      <td>Genetics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fgsea</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fgsea</td>
      <td>fgsea</td>
      <td>fgsea</td>
      <td>The package implements an algorithm for fast gene set enrichment analysis. Using the fast algorithm allows to make more permutations and get more fine grained p-values, which allows to use accurate stantard approaches to multiple hypothesis correction.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>632</td>
      <td>734</td>
      <td>7520</td>
      <td>8942</td>
      <td>1072</td>
      <td>1156</td>
      <td>22759</td>
      <td>24664</td>
      <td>121</td>
      <td>136</td>
      <td>1144</td>
      <td>1277</td>
      <td>9</td>
      <td>9</td>
      <td>115</td>
      <td>115</td>
      <td>34998</td>
      <td>31538</td>
      <td>2035</td>
      <td>1834</td>
      <td>https://usegalaxy.eu/published/workflow?id=12bf85cdc8102bb5, https://usegalaxy.eu/published/workflow?id=63c91ea322cd2819, https://usegalaxy.eu/published/workflow?id=732fa536cdc70c96, https://usegalaxy.eu/published/workflow?id=73a9ab4f0f0b9907, https://usegalaxy.eu/published/workflow?id=9a83043d5e847318, https://usegalaxy.eu/published/workflow?id=c3a11e1ac1aa8383, https://usegalaxy.eu/published/workflow?id=e04be1ab5a16eb9f, https://usegalaxy.eu/published/workflow?id=e95729ed24f7c7d0, https://usegalaxy.eu/published/workflow?id=fa83e26e05418370, https://usegalaxy.org.au/published/workflow?id=9f6e62f465589ca3, https://usegalaxy.org/published/workflow?id=68832752b883c226, https://workflowhub.eu/workflows/1699?version=1</td>
      <td>transcriptomics/rna-seq-genes-to-pathways</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>gsc_filter_cells</td>
      <td>filter_cells</td>
      <td>Filter single cell RNAseq data on library depth and number of detected genes</td>
      <td>2019-06-23</td>
      <td>http://artbio.fr</td>
      <td>4.3.1+galaxy1</td>
      <td>r-optparse</td>
      <td>1.3.2</td>
      <td>To update</td>
      <td>Transcriptomics, Single Cell</td>
      <td>RNA-Seq analysis, Clustering, Visualisation, Expression analysis</td>
      <td>RNA-Seq analysis, Clustering, Visualisation</td>
      <td></td>
      <td></td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_filter_cells</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_filter_cells</td>
      <td>galaxy_single_cell_suite</td>
      <td>Galaxy Single Cell suite</td>
      <td>Suite of Galaxy tools from artbio.fr for end-to-end single-cell RNA-seq analysis</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>gsc_filter_genes</td>
      <td>filter_genes</td>
      <td>Filter genes that are detected in less than a fraction of libraries in single cell RNAseq data</td>
      <td>2019-06-23</td>
      <td>http://artbio.fr</td>
      <td>4.3.1+galaxy1</td>
      <td>r-optparse</td>
      <td>1.3.2</td>
      <td>To update</td>
      <td>Transcriptomics, Single Cell</td>
      <td>RNA-Seq analysis, Clustering, Visualisation, Expression analysis</td>
      <td>RNA-Seq analysis, Clustering, Visualisation</td>
      <td></td>
      <td></td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_filter_genes</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_filter_genes</td>
      <td>galaxy_single_cell_suite</td>
      <td>Galaxy Single Cell suite</td>
      <td>Suite of Galaxy tools from artbio.fr for end-to-end single-cell RNA-seq analysis</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>gsc_gene_expression_correlations</td>
      <td>single_cell_gene_expression_correlations</td>
      <td>Compute single-cell paire-wise gene expressions correlations</td>
      <td>2019-06-24</td>
      <td>http://artbio.fr</td>
      <td>4.3.1+galaxy1</td>
      <td>r-optparse</td>
      <td>1.3.2</td>
      <td>To update</td>
      <td>Transcriptomics, Single Cell</td>
      <td>RNA-Seq analysis, Clustering, Visualisation, Expression analysis</td>
      <td>RNA-Seq analysis, Clustering, Visualisation</td>
      <td></td>
      <td></td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/gsc_gene_expression_correlations</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_gene_expression_correlations</td>
      <td>galaxy_single_cell_suite</td>
      <td>Galaxy Single Cell suite</td>
      <td>Suite of Galaxy tools from artbio.fr for end-to-end single-cell RNA-seq analysis</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>gsc_high_dimensions_visualisation</td>
      <td>high_dimensions_visualisation</td>
      <td>Generates PCA, t-SNE and HCPC visualisation</td>
      <td>2019-06-23</td>
      <td>http://artbio.fr</td>
      <td>4.3+galaxy1</td>
      <td>r-optparse</td>
      <td>1.3.2</td>
      <td>To update</td>
      <td>Transcriptomics, Visualization, Single Cell</td>
      <td>RNA-Seq analysis, Clustering, Visualisation, Expression analysis</td>
      <td>RNA-Seq analysis, Clustering, Visualisation</td>
      <td></td>
      <td></td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_high_dimension_visualization</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_high_dimensions_visualisation</td>
      <td>galaxy_single_cell_suite</td>
      <td>Galaxy Single Cell suite</td>
      <td>Suite of Galaxy tools from artbio.fr for end-to-end single-cell RNA-seq analysis</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>gsc_mannwhitney_de</td>
      <td>mannwhitney_de</td>
      <td>Perform a mann-whitney differential testing between two sets of gene expression data</td>
      <td>2019-06-23</td>
      <td>http://artbio.fr</td>
      <td>4.1.3+galaxy1</td>
      <td>r-optparse</td>
      <td>1.3.2</td>
      <td>To update</td>
      <td>Transcriptomics, Single Cell</td>
      <td>RNA-Seq analysis, Clustering, Visualisation, Expression analysis</td>
      <td>RNA-Seq analysis, Clustering, Visualisation</td>
      <td></td>
      <td></td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_mannwhitney_de</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_mannwhitney_de</td>
      <td>galaxy_single_cell_suite</td>
      <td>Galaxy Single Cell suite</td>
      <td>Suite of Galaxy tools from artbio.fr for end-to-end single-cell RNA-seq analysis</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>gsc_scran_normalize</td>
      <td>scran_normalize</td>
      <td>Normalize raw counts using scran</td>
      <td>2019-09-23</td>
      <td>http://artbio.fr</td>
      <td>1.28.1+galaxy1</td>
      <td>bioconductor-scran</td>
      <td>1.34.0</td>
      <td>To update</td>
      <td>Transcriptomics, Single Cell</td>
      <td>RNA-Seq analysis, Clustering, Visualisation, Expression analysis</td>
      <td>RNA-Seq analysis, Clustering, Visualisation</td>
      <td></td>
      <td></td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_scran_normalize</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_scran_normalize</td>
      <td>galaxy_single_cell_suite</td>
      <td>Galaxy Single Cell suite</td>
      <td>Suite of Galaxy tools from artbio.fr for end-to-end single-cell RNA-seq analysis</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>34</td>
      <td>34</td>
      <td>125</td>
      <td>125</td>
      <td>18</td>
      <td>18</td>
      <td>334</td>
      <td>334</td>
      <td>5</td>
      <td>5</td>
      <td>157</td>
      <td>157</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>616</td>
      <td>616</td>
      <td>57</td>
      <td>57</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>gsc_signature_score</td>
      <td>signature_score</td>
      <td>Compute signature scores from single cell RNAseq data</td>
      <td>2019-06-23</td>
      <td>http://artbio.fr</td>
      <td>2.3.9+galaxy1</td>
      <td>r-optparse</td>
      <td>1.3.2</td>
      <td>To update</td>
      <td>Transcriptomics, Single Cell</td>
      <td>RNA-Seq analysis, Clustering, Visualisation, Expression analysis</td>
      <td>RNA-Seq analysis, Clustering, Visualisation</td>
      <td></td>
      <td></td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_signature_score</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_signature_score</td>
      <td>galaxy_single_cell_suite</td>
      <td>Galaxy Single Cell suite</td>
      <td>Suite of Galaxy tools from artbio.fr for end-to-end single-cell RNA-seq analysis</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>heinz</td>
      <td>heinz_bum, heinz, heinz_scoring, heinz_visualization</td>
      <td>An algorithm for identification of the optimal scoring subnetwork.</td>
      <td>2018-06-11</td>
      <td>https://github.com/ls-cwi/heinz</td>
      <td>1.0</td>
      <td>bioconductor-bionet</td>
      <td>1.66.0</td>
      <td>To update</td>
      <td>Transcriptomics, Visualization, Statistics, Single Cell</td>
      <td>Pathway or network analysis</td>
      <td>Pathway or network analysis</td>
      <td>Genetics, Gene expression, Molecular interactions, pathways and networks</td>
      <td>Genetics, Gene expression, Molecular interactions, pathways and networks</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/heinz</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/heinz</td>
      <td>heinz</td>
      <td>Heinz</td>
      <td>Tool for single-species active module discovery.</td>
      <td></td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>123</td>
      <td>160</td>
      <td>823</td>
      <td>1460</td>
      <td>318</td>
      <td>345</td>
      <td>1490</td>
      <td>1678</td>
      <td>36</td>
      <td>36</td>
      <td>847</td>
      <td>913</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4051</td>
      <td>3160</td>
      <td>541</td>
      <td>477</td>
      <td>https://usegalaxy.eu/published/workflow?id=12c80c5b5e2305d8, https://usegalaxy.org.au/published/workflow?id=6a161c87d19a543c, https://workflowhub.eu/workflows/1677?version=1</td>
      <td>transcriptomics/network-analysis-with-heinz</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>infercnv</td>
      <td>infercnv</td>
      <td>Infer Copy Number Variation from Single-Cell RNA-Seq Data</td>
      <td>2024-07-23</td>
      <td>https://github.com/broadinstitute/infercnv</td>
      <td>1.22.0</td>
      <td>bioconductor-infercnv</td>
      <td>1.22.0</td>
      <td>Up-to-date</td>
      <td>Transcriptomics, Variant Analysis, Single Cell</td>
      <td>Heat map generation, Expression analysis, RNA-Seq analysis, Copy number variation detection, Clustering</td>
      <td>Heat map generation, RNA-Seq analysis, Copy number variation detection, Clustering</td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/infercnv</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/infercnv</td>
      <td>infercnv</td>
      <td>InferCNV</td>
      <td>Inferring copy number alterations from tumor single cell RNA-Seq data</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>31</td>
      <td>31</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>31</td>
      <td>31</td>
      <td>7</td>
      <td>7</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>multigsea</td>
      <td>multigsea</td>
      <td>GSEA-based pathway enrichment analysis for multi-omics data</td>
      <td>2023-06-07</td>
      <td>https://bioconductor.org/packages/devel/bioc/html/multiGSEA.html</td>
      <td>1.12.0</td>
      <td>bioconductor-multigsea</td>
      <td>1.16.0</td>
      <td>To update</td>
      <td>Transcriptomics, Proteomics, Statistics, Single Cell</td>
      <td>Gene-set enrichment analysis, Aggregation, Pathway analysis</td>
      <td>Gene-set enrichment analysis, Aggregation, Pathway analysis</td>
      <td>Metabolomics, Molecular interactions, pathways and networks, Proteomics, Transcriptomics, Small molecules</td>
      <td>Metabolomics, Molecular interactions, pathways and networks, Proteomics, Transcriptomics, Small molecules</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/multigsea</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/multigsea</td>
      <td>multiGSEA</td>
      <td>multiGSEA</td>
      <td>A GSEA-based pathway enrichment analysis for multi-omics data.multiGSEA: a GSEA-based pathway enrichment analysis for multi-omics data, BMC Bioinformatics 21, 561 (2020).Combining GSEA-based pathway enrichment with multi omics data integration.</td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>65</td>
      <td>65</td>
      <td>212</td>
      <td>212</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>46</td>
      <td>46</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>258</td>
      <td>258</td>
      <td>70</td>
      <td>70</td>
      <td>https://workflowhub.eu/workflows/1388?version=1</td>
      <td>proteomics/multiGSEA-tutorial</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>rgrnastar</td>
      <td>rna_star, rna_starsolo</td>
      <td>RNA STAR is an ultra fast universal RNA and scRNA-seq aligner and mapper</td>
      <td>2015-05-18</td>
      <td>https://github.com/alexdobin/STAR</td>
      <td>2.7.11b</td>
      <td>star</td>
      <td>2.7.11b</td>
      <td>Up-to-date</td>
      <td>Next Gen Mappers, Transcriptomics, Single Cell</td>
      <td>Sequence alignment</td>
      <td>Sequence alignment</td>
      <td>RNA-Seq, Transcriptomics</td>
      <td>RNA-Seq, Transcriptomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/rgrnastar</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/rgrnastar</td>
      <td>star</td>
      <td>STAR</td>
      <td>Ultrafast universal RNA-seq data aligner</td>
      <td></td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>2</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>11597</td>
      <td>13300</td>
      <td>454740</td>
      <td>555235</td>
      <td>17153</td>
      <td>22174</td>
      <td>406678</td>
      <td>513360</td>
      <td>1461</td>
      <td>1710</td>
      <td>55618</td>
      <td>62678</td>
      <td>1016</td>
      <td>1019</td>
      <td>12202</td>
      <td>12241</td>
      <td>1143514</td>
      <td>929238</td>
      <td>38203</td>
      <td>31227</td>
      <td>https://usegalaxy.cz/published/workflow?id=932f4901960a595a, https://usegalaxy.eu/published/workflow?id=0229f3f3835c2d53, https://usegalaxy.eu/published/workflow?id=046e5c9726c75a3c, https://usegalaxy.eu/published/workflow?id=06fe33c4420906bf, https://usegalaxy.eu/published/workflow?id=08cfb9b21c1ec808, https://usegalaxy.eu/published/workflow?id=0a0b5aff6bd9689d, https://usegalaxy.eu/published/workflow?id=0c619b25c1394a09, https://usegalaxy.eu/published/workflow?id=0d6bed63a2e4c3d6, https://usegalaxy.eu/published/workflow?id=0eb137cd936bbfe0, https://usegalaxy.eu/published/workflow?id=0f563f176bb245af, https://usegalaxy.eu/published/workflow?id=0fa523529c4f73cf, https://usegalaxy.eu/published/workflow?id=153ba1581743d31f, https://usegalaxy.eu/published/workflow?id=1f9a9a7998ef5ab3, https://usegalaxy.eu/published/workflow?id=20ada53472782c9b, https://usegalaxy.eu/published/workflow?id=23821870734575e9, https://usegalaxy.eu/published/workflow?id=249c5d6698c8b98c, https://usegalaxy.eu/published/workflow?id=24ddd188da35b061, https://usegalaxy.eu/published/workflow?id=250e6058f160825f, https://usegalaxy.eu/published/workflow?id=2d08a73dd8ff99e9, https://usegalaxy.eu/published/workflow?id=3403d5a8a71e0c3c, https://usegalaxy.eu/published/workflow?id=406e6ef0dfb38628, https://usegalaxy.eu/published/workflow?id=409336bf210033a2, https://usegalaxy.eu/published/workflow?id=47a0407867a90f95, https://usegalaxy.eu/published/workflow?id=48071c1720b2fd8d, https://usegalaxy.eu/published/workflow?id=49073a24429d93d6, https://usegalaxy.eu/published/workflow?id=4e215c9b50650d5d, https://usegalaxy.eu/published/workflow?id=6514e610c79df72c, https://usegalaxy.eu/published/workflow?id=6a6e2c2335051fb5, https://usegalaxy.eu/published/workflow?id=6ebd09100fc98878, https://usegalaxy.eu/published/workflow?id=7117dbcbd5d12de5, https://usegalaxy.eu/published/workflow?id=7136ce9f18c74853, https://usegalaxy.eu/published/workflow?id=7d22eea4278570e8, https://usegalaxy.eu/published/workflow?id=86e2e2faa2e0bd46, https://usegalaxy.eu/published/workflow?id=873e6ac4699ed7f6, https://usegalaxy.eu/published/workflow?id=87fea062a9646a31, https://usegalaxy.eu/published/workflow?id=8fb156020285e049, https://usegalaxy.eu/published/workflow?id=90a32c95b60f5221, https://usegalaxy.eu/published/workflow?id=94bbf8fa7c1f030d, https://usegalaxy.eu/published/workflow?id=96c61a584cb2e5e9, https://usegalaxy.eu/published/workflow?id=985044b4b2c51425, https://usegalaxy.eu/published/workflow?id=9a52516ec0b539ea, https://usegalaxy.eu/published/workflow?id=9b025479fade745d, https://usegalaxy.eu/published/workflow?id=9c7a218993788493, https://usegalaxy.eu/published/workflow?id=9da26c5131603b9f, https://usegalaxy.eu/published/workflow?id=9e03cf8cd5e5de08, https://usegalaxy.eu/published/workflow?id=a1cfb476e849fce3, https://usegalaxy.eu/published/workflow?id=a28c207730e7e13d, https://usegalaxy.eu/published/workflow?id=a2d4221ad117c378, https://usegalaxy.eu/published/workflow?id=a3cb75fd31ba0689, https://usegalaxy.eu/published/workflow?id=a3da32b6b45da639, https://usegalaxy.eu/published/workflow?id=a461ea397418f3aa, https://usegalaxy.eu/published/workflow?id=af7c349497a75314, https://usegalaxy.eu/published/workflow?id=aff44f1665a14e23, https://usegalaxy.eu/published/workflow?id=b26fee38f6f796d8, https://usegalaxy.eu/published/workflow?id=b426e137396acb14, https://usegalaxy.eu/published/workflow?id=b5627201bb4f6943, https://usegalaxy.eu/published/workflow?id=b619339f4e1c6fdf, https://usegalaxy.eu/published/workflow?id=b7a3cbcd32734e8b, https://usegalaxy.eu/published/workflow?id=bace9aebab9d4ce3, https://usegalaxy.eu/published/workflow?id=be3d1d7d4afd96b1, https://usegalaxy.eu/published/workflow?id=c2d814f2182c0082, https://usegalaxy.eu/published/workflow?id=c4fa05e209a2f082, https://usegalaxy.eu/published/workflow?id=c5e6047b3b256b06, https://usegalaxy.eu/published/workflow?id=c64b219b4031fabd, https://usegalaxy.eu/published/workflow?id=c6b22fc33b569c8f, https://usegalaxy.eu/published/workflow?id=c98a14fcf5998159, https://usegalaxy.eu/published/workflow?id=ca9d3233b0912765, https://usegalaxy.eu/published/workflow?id=cc37667c2d1f9056, https://usegalaxy.eu/published/workflow?id=ccef778cfd9024d1, https://usegalaxy.eu/published/workflow?id=d0a809f0880d7ec0, https://usegalaxy.eu/published/workflow?id=d3494c6124f820b2, https://usegalaxy.eu/published/workflow?id=d3edc3f0659a5e5b, https://usegalaxy.eu/published/workflow?id=d4b3da4cc5a50044, https://usegalaxy.eu/published/workflow?id=d6809603f46de1b3, https://usegalaxy.eu/published/workflow?id=d79309343e2a5d62, https://usegalaxy.eu/published/workflow?id=deec04097a871646, https://usegalaxy.eu/published/workflow?id=e5becc6d66b2568c, https://usegalaxy.eu/published/workflow?id=e7494f29c8672f34, https://usegalaxy.eu/published/workflow?id=e7a9e1a7728848cd, https://usegalaxy.eu/published/workflow?id=e94e307375a92c94, https://usegalaxy.eu/published/workflow?id=ead0f264798f9418, https://usegalaxy.eu/published/workflow?id=ef1d11b79a2e1727, https://usegalaxy.eu/published/workflow?id=f5be5bcf9b9f171c, https://usegalaxy.eu/published/workflow?id=f841e1207611f28f, https://usegalaxy.eu/published/workflow?id=fa53b172def3fe75, https://usegalaxy.eu/published/workflow?id=fa960b78dfc0c2f8, https://usegalaxy.eu/published/workflow?id=fd61d070d9744f95, https://usegalaxy.fr/published/workflow?id=e266db902aa1b0e8, https://usegalaxy.org.au/published/workflow?id=00da6615bca12a3c, https://usegalaxy.org.au/published/workflow?id=0458ecd2761307d1, https://usegalaxy.org.au/published/workflow?id=0dff588f31f1dfd6, https://usegalaxy.org.au/published/workflow?id=13ce18408ee7acc9, https://usegalaxy.org.au/published/workflow?id=16f4846b4d6bc155, https://usegalaxy.org.au/published/workflow?id=1a107e6611fb9c15, https://usegalaxy.org.au/published/workflow?id=1ad81f95ace878ea, https://usegalaxy.org.au/published/workflow?id=281ea78467e73147, https://usegalaxy.org.au/published/workflow?id=2cef9a1633e0a067, https://usegalaxy.org.au/published/workflow?id=3fd10af90a07f771, https://usegalaxy.org.au/published/workflow?id=41b115a9be4b6002, https://usegalaxy.org.au/published/workflow?id=4e161a8f5098cc49, https://usegalaxy.org.au/published/workflow?id=56bda8e4b0018e03, https://usegalaxy.org.au/published/workflow?id=5aff01d0b5fabd84, https://usegalaxy.org.au/published/workflow?id=5cd9b9338e75f254, https://usegalaxy.org.au/published/workflow?id=644d8e4c78df8e36, https://usegalaxy.org.au/published/workflow?id=65250d6b30d4bc7c, https://usegalaxy.org.au/published/workflow?id=6e1e24f9e0326930, https://usegalaxy.org.au/published/workflow?id=6f0801fecbba5d3f, https://usegalaxy.org.au/published/workflow?id=71bad95f72c9f30d, https://usegalaxy.org.au/published/workflow?id=738350fc2ca69c0b, https://usegalaxy.org.au/published/workflow?id=749d8f2d26e0a111, https://usegalaxy.org.au/published/workflow?id=75113a1b41904d2a, https://usegalaxy.org.au/published/workflow?id=7ad277dd448e57b3, https://usegalaxy.org.au/published/workflow?id=7ae06a27188b9438, https://usegalaxy.org.au/published/workflow?id=7becafd394472dfb, https://usegalaxy.org.au/published/workflow?id=7c22d726e1716f4b, https://usegalaxy.org.au/published/workflow?id=85aede91db0122e4, https://usegalaxy.org.au/published/workflow?id=897e9d7432d231e9, https://usegalaxy.org.au/published/workflow?id=8d6d4f848067f3d9, https://usegalaxy.org.au/published/workflow?id=8fb691706f47527a, https://usegalaxy.org.au/published/workflow?id=9b18123afbe1264b, https://usegalaxy.org.au/published/workflow?id=9cf3860659285656, https://usegalaxy.org.au/published/workflow?id=a763e53653b10ce0, https://usegalaxy.org.au/published/workflow?id=ae773dfaa074cb72, https://usegalaxy.org.au/published/workflow?id=b03b577156e4db5b, https://usegalaxy.org.au/published/workflow?id=b517cabb697491a0, https://usegalaxy.org.au/published/workflow?id=b7226d37b25df833, https://usegalaxy.org.au/published/workflow?id=bb0feff5d764df60, https://usegalaxy.org.au/published/workflow?id=c2dee690320f9845, https://usegalaxy.org.au/published/workflow?id=c64dcc1e6a67f3e4, https://usegalaxy.org.au/published/workflow?id=c8d18e2c03dca796, https://usegalaxy.org.au/published/workflow?id=ccc6602623583bcc, https://usegalaxy.org.au/published/workflow?id=ccfae75390038ced, https://usegalaxy.org.au/published/workflow?id=ce7f6a1c824920e2, https://usegalaxy.org.au/published/workflow?id=d5a3fd1faa3f0a41, https://usegalaxy.org.au/published/workflow?id=e1fd89e7248b6661, https://usegalaxy.org.au/published/workflow?id=f627023a3401bf9b, https://usegalaxy.org/published/workflow?id=03a1cd225af6de69, https://usegalaxy.org/published/workflow?id=044c5ce43f2e1829, https://usegalaxy.org/published/workflow?id=098375608a1223aa, https://usegalaxy.org/published/workflow?id=10fa713ca3076a47, https://usegalaxy.org/published/workflow?id=1234a128adc25b03, https://usegalaxy.org/published/workflow?id=1f28e673ea725541, https://usegalaxy.org/published/workflow?id=2125db88b4a71c09, https://usegalaxy.org/published/workflow?id=26019c2b056d3a1c, https://usegalaxy.org/published/workflow?id=2f774500d5bec9a6, https://usegalaxy.org/published/workflow?id=39810594e5c332d5, https://usegalaxy.org/published/workflow?id=3ab66315f50501f0, https://usegalaxy.org/published/workflow?id=3ad9cdcda44b7dfb, https://usegalaxy.org/published/workflow?id=3e57e668cf698d19, https://usegalaxy.org/published/workflow?id=3e6c9ab5ef3250ae, https://usegalaxy.org/published/workflow?id=413d7e5aa7378cf1, https://usegalaxy.org/published/workflow?id=42bc3a770f0ab6b8, https://usegalaxy.org/published/workflow?id=467d3e260c21de98, https://usegalaxy.org/published/workflow?id=4bbe4c8a6c52d03a, https://usegalaxy.org/published/workflow?id=516a2181b41af94e, https://usegalaxy.org/published/workflow?id=5336b9d26a083983, https://usegalaxy.org/published/workflow?id=53fdff74729a6780, https://usegalaxy.org/published/workflow?id=5497e8630ebfeb42, https://usegalaxy.org/published/workflow?id=582f3de3971fc38c, https://usegalaxy.org/published/workflow?id=5fc0c7c65edc7457, https://usegalaxy.org/published/workflow?id=62316da77b1bfa5a, https://usegalaxy.org/published/workflow?id=63b3b266bde77563, https://usegalaxy.org/published/workflow?id=676b88d0aac13fa5, https://usegalaxy.org/published/workflow?id=69712265fdb10ed5, https://usegalaxy.org/published/workflow?id=76b8ed1c68678b62, https://usegalaxy.org/published/workflow?id=778a8867b0a6ea4f, https://usegalaxy.org/published/workflow?id=7a26069c2939bef9, https://usegalaxy.org/published/workflow?id=7aa36b1722ff4c11, https://usegalaxy.org/published/workflow?id=82681bd5f059e39a, https://usegalaxy.org/published/workflow?id=868acaf49d0743ec, https://usegalaxy.org/published/workflow?id=87eb1a8ee7a2c984, https://usegalaxy.org/published/workflow?id=89f52c97b3aee938, https://usegalaxy.org/published/workflow?id=8bcab83874cbc75c, https://usegalaxy.org/published/workflow?id=8e6d24a14d2908c2, https://usegalaxy.org/published/workflow?id=8ebcae16a7822d93, https://usegalaxy.org/published/workflow?id=8fb8c80c93ad694e, https://usegalaxy.org/published/workflow?id=93e69c7e7a857ca3, https://usegalaxy.org/published/workflow?id=a6d6a2e8867fa566, https://usegalaxy.org/published/workflow?id=a8ddab36010ed4c3, https://usegalaxy.org/published/workflow?id=b1b75a9323d25085, https://usegalaxy.org/published/workflow?id=b28789b6a0e1f5c9, https://usegalaxy.org/published/workflow?id=b73e416c06790ac5, https://usegalaxy.org/published/workflow?id=bc83c42b44c4b03f, https://usegalaxy.org/published/workflow?id=bd2ce9a97c1598df, https://usegalaxy.org/published/workflow?id=bdfb27e4ef7b6138, https://usegalaxy.org/published/workflow?id=be7dab2274c1acdd, https://usegalaxy.org/published/workflow?id=c74e3a24751d9924, https://usegalaxy.org/published/workflow?id=cdb6c9c7037a95b2, https://usegalaxy.org/published/workflow?id=d2f7fd589a91124a, https://usegalaxy.org/published/workflow?id=d82db9c29b998f22, https://usegalaxy.org/published/workflow?id=dc7f05cb5a62da77, https://usegalaxy.org/published/workflow?id=e035be74e3c85bf5, https://usegalaxy.org/published/workflow?id=e33f6a058c5f57e5, https://usegalaxy.org/published/workflow?id=e55ca98817c1855e, https://usegalaxy.org/published/workflow?id=f3c8093c5b8a7233, https://usegalaxy.org/published/workflow?id=fb789920b057fe8d, https://workflowhub.eu/workflows/1442?version=1, https://workflowhub.eu/workflows/1494?version=1, https://workflowhub.eu/workflows/1525?version=1, https://workflowhub.eu/workflows/1546?version=1, https://workflowhub.eu/workflows/1688?version=1, https://workflowhub.eu/workflows/1706?version=1, https://workflowhub.eu/workflows/1715?version=3, https://workflowhub.eu/workflows/1792?version=1, https://workflowhub.eu/workflows/400?version=12, https://workflowhub.eu/workflows/401?version=13, https://workflowhub.eu/workflows/513?version=3, https://workflowhub.eu/workflows/754?version=1</td>
      <td>genome-annotation/braker3, genome-annotation/comparison-braker-helixer-annotation, genome-annotation/funannotate, proteomics/neoantigen-fusion-database-generation, single-cell/scrna-preprocessing, single-cell/scrna-preprocessing-tenx, transcriptomics/clipseq, transcriptomics/differential-isoform-expression, transcriptomics/ref-based</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>scanpy</td>
      <td>scanpy_cluster_reduce_dimension, scanpy_filter, scanpy_inspect, scanpy_normalize, scanpy_plot, scanpy_remove_confounders</td>
      <td>Scanpy  Single-Cell Analysis in Python</td>
      <td>2019-03-04</td>
      <td>https://scanpy.readthedocs.io</td>
      <td>1.10.2</td>
      <td>scanpy</td>
      <td>1.7.2</td>
      <td>To update</td>
      <td>Single Cell, Spatial Omics, Transcriptomics</td>
      <td>Differential gene expression analysis, Principal component visualisation, Expression profile clustering</td>
      <td>Differential gene expression analysis, Principal component visualisation, Expression profile clustering</td>
      <td>Gene expression, Cell biology, Genetics, Single-cell sequencing</td>
      <td>Gene expression, Cell biology, Genetics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/scanpy</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/scanpy</td>
      <td>scanpy</td>
      <td>SCANPY</td>
      <td>Scalable toolkit for analyzing single-cell gene expression data. It includes preprocessing, visualization, clustering, pseudotime and trajectory inference and differential expression testing. The Python-based implementation efficiently deals with datasets of more than one million cells.</td>
      <td></td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>771</td>
      <td>868</td>
      <td>70187</td>
      <td>77433</td>
      <td>456</td>
      <td>465</td>
      <td>42066</td>
      <td>42163</td>
      <td>120</td>
      <td>121</td>
      <td>22195</td>
      <td>23258</td>
      <td>2</td>
      <td>2</td>
      <td>114</td>
      <td>114</td>
      <td>142968</td>
      <td>134562</td>
      <td>1456</td>
      <td>1349</td>
      <td>https://usegalaxy.eu/published/workflow?id=0d7c643d7720ee31, https://usegalaxy.eu/published/workflow?id=171d6fca2c3339e2, https://usegalaxy.eu/published/workflow?id=21315ffd2df2f159, https://usegalaxy.eu/published/workflow?id=37e51e215684bdcc, https://usegalaxy.eu/published/workflow?id=4756053b2596b6cf, https://usegalaxy.eu/published/workflow?id=715ca378c1d68de3, https://usegalaxy.eu/published/workflow?id=77e85bde57477194, https://usegalaxy.eu/published/workflow?id=78dc6fca5b58ec05, https://usegalaxy.eu/published/workflow?id=921cab3e6faf30be, https://usegalaxy.eu/published/workflow?id=9adf0402b641042c, https://usegalaxy.eu/published/workflow?id=9d15b328d8408715, https://usegalaxy.eu/published/workflow?id=ae79275d3568ad31, https://usegalaxy.eu/published/workflow?id=b3cf397fd4d1bea8, https://usegalaxy.eu/published/workflow?id=b5627201bb4f6943, https://usegalaxy.eu/published/workflow?id=d70dc4cfeedb690b, https://usegalaxy.eu/published/workflow?id=d7c0c972c471d514, https://usegalaxy.eu/published/workflow?id=ddced7b107e4707a, https://usegalaxy.eu/published/workflow?id=e2391c40d24d403f, https://usegalaxy.eu/published/workflow?id=e472096116cbe392, https://usegalaxy.eu/published/workflow?id=e4e969e9943cdaa5, https://usegalaxy.eu/published/workflow?id=f05754467a439134, https://usegalaxy.org.au/published/workflow?id=102eb726f63f3190, https://usegalaxy.org.au/published/workflow?id=106080375ad4eaaa, https://usegalaxy.org.au/published/workflow?id=218ffb4438833bdd, https://usegalaxy.org.au/published/workflow?id=57321b64220d5835, https://usegalaxy.org.au/published/workflow?id=8055596d732590d6, https://usegalaxy.org.au/published/workflow?id=8bc01c1fdd62ad12, https://usegalaxy.org/published/workflow?id=dd43ae9a5b303086, https://usegalaxy.org/published/workflow?id=ea3257e0ef9f505a, https://usegalaxy.org/published/workflow?id=f35ccb693cd874c1, https://workflowhub.eu/workflows/1077?version=1, https://workflowhub.eu/workflows/1256?version=1, https://workflowhub.eu/workflows/1356?version=1, https://workflowhub.eu/workflows/1362?version=1, https://workflowhub.eu/workflows/1485?version=1, https://workflowhub.eu/workflows/1490?version=1, https://workflowhub.eu/workflows/1496?version=1, https://workflowhub.eu/workflows/1501?version=1, https://workflowhub.eu/workflows/1502?version=2, https://workflowhub.eu/workflows/1519?version=1, https://workflowhub.eu/workflows/1550?version=1, https://workflowhub.eu/workflows/1551?version=1, https://workflowhub.eu/workflows/1556?version=1, https://workflowhub.eu/workflows/1574?version=1, https://workflowhub.eu/workflows/467?version=4, https://workflowhub.eu/workflows/468?version=3</td>
      <td>single-cell/pseudobulk-analysis, single-cell/scatac-preprocessing-tenx, single-cell/scatac-standard-processing-snapatac2, single-cell/scrna-case_basic-pipeline, single-cell/scrna-case_cell-cycle, single-cell/scrna-plant, single-cell/scrna-scanpy-pbmc3k</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>sceasy</td>
      <td>sceasy_convert</td>
      <td>Convert scRNA data object between popular formats</td>
      <td>2019-10-14</td>
      <td></td>
      <td>0.0.5</td>
      <td>r-sceasy</td>
      <td>0.0.7</td>
      <td>To update</td>
      <td>Transcriptomics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/sceasy</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>301</td>
      <td>310</td>
      <td>1543</td>
      <td>1615</td>
      <td>103</td>
      <td>104</td>
      <td>521</td>
      <td>522</td>
      <td>5</td>
      <td>5</td>
      <td>349</td>
      <td>349</td>
      <td>1</td>
      <td>1</td>
      <td>2</td>
      <td>2</td>
      <td>2488</td>
      <td>2415</td>
      <td>420</td>
      <td>410</td>
      <td>https://usegalaxy.eu/published/workflow?id=0e6e181234b48121, https://usegalaxy.eu/published/workflow?id=1e2d27fae8a41ac1, https://usegalaxy.eu/published/workflow?id=9a9bbd8046da7b19, https://usegalaxy.eu/published/workflow?id=a0f1959a9f4533f7, https://usegalaxy.eu/published/workflow?id=bcec217b62ccc7af, https://usegalaxy.eu/published/workflow?id=c6adc3b665828a6f, https://usegalaxy.eu/published/workflow?id=df455b86fff16116, https://usegalaxy.eu/published/workflow?id=e06934a35fc18f2c, https://usegalaxy.eu/published/workflow?id=f944ad0c35d1a472, https://usegalaxy.org/published/workflow?id=1fe6b25825d9b736, https://usegalaxy.org/published/workflow?id=a2a924b871b1c96d, https://workflowhub.eu/workflows/1533?version=1, https://workflowhub.eu/workflows/1537?version=1</td>
      <td>single-cell/EBI-retrieval, single-cell/scrna-case_alevin, single-cell/scrna-data-ingest</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>schicexplorer</td>
      <td>schicexplorer_schicadjustmatrix, schicexplorer_schiccluster, schicexplorer_schicclustercompartments, schicexplorer_schicclusterminhash, schicexplorer_schicclustersvl, schicexplorer_schicconsensusmatrices, schicexplorer_schiccorrectmatrices, schicexplorer_schiccreatebulkmatrix, schicexplorer_schicdemultiplex, schicexplorer_schicinfo, schicexplorer_schicmergematrixbins, schicexplorer_schicmergetoscool, schicexplorer_schicnormalize, schicexplorer_schicplotclusterprofiles, schicexplorer_schicplotconsensusmatrices, schicexplorer_schicqualitycontrol</td>
      <td>scHiCExplorer: Set of programs to process, analyze and visualize scHi-C data.</td>
      <td>2019-12-16</td>
      <td>https://github.com/joachimwolff/schicexplorer</td>
      <td>4</td>
      <td>schicexplorer</td>
      <td>7</td>
      <td>To update</td>
      <td>Sequence Analysis, Transcriptomics, Visualization, Single Cell</td>
      <td>Validation, Visualisation, Standardisation and normalisation</td>
      <td>Validation, Visualisation, Standardisation and normalisation</td>
      <td>Workflows, DNA, Chromosome conformation capture, ChIP-seq, Epigenetics</td>
      <td>Workflows, DNA, Chromosome conformation capture, ChIP-seq, Epigenetics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/schicexplorer</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/schicexplorer</td>
      <td>hicexplorer</td>
      <td>HiCExplorer</td>
      <td>A web server for reproducible Hi-C, capture Hi-C and single-cell Hi-C data analysis, quality control and visualization.HiCExplorer  HiCExplorer 3.6 documentation.scHiCExplorer  scHiCExplorer 7 documentation.Free document hosting provided by Read the Docs.</td>
      <td></td>
      <td>16</td>
      <td>16</td>
      <td>16</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>16</td>
      <td>16</td>
      <td>16</td>
      <td>0</td>
      <td>8</td>
      <td>11</td>
      <td>99</td>
      <td>821</td>
      <td>12</td>
      <td>12</td>
      <td>32</td>
      <td>32</td>
      <td>2</td>
      <td>2</td>
      <td>1351</td>
      <td>1351</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2204</td>
      <td>1482</td>
      <td>25</td>
      <td>22</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>scpipe</td>
      <td>scpipe</td>
      <td>A flexible preprocessing pipeline for single-cell RNA-sequencing data</td>
      <td>2018-08-12</td>
      <td>http://bioconductor.org/packages/release/bioc/html/scPipe.html</td>
      <td>1.0.0+galaxy2</td>
      <td>bioconductor-scpipe</td>
      <td>2.6.0</td>
      <td>To update</td>
      <td>Transcriptomics, Single Cell</td>
      <td>Genome annotation, Validation, Alignment, Visualisation</td>
      <td>Genome annotation, Validation, Alignment, Visualisation</td>
      <td>Gene expression, RNA-Seq, Sequencing</td>
      <td>Gene expression, RNA-Seq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/scpipe</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/scpipe</td>
      <td>scpipe</td>
      <td>scPipe</td>
      <td>A preprocessing pipeline for single cell RNA-seq data that starts from the fastq files and produces a gene count matrix with associated quality control information. It can process fastq data generated by CEL-seq, MARS-seq, Drop-seq, Chromium 10x and SMART-seq protocols.</td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>25</td>
      <td>40</td>
      <td>149</td>
      <td>643</td>
      <td>50</td>
      <td>63</td>
      <td>656</td>
      <td>799</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1443</td>
      <td>806</td>
      <td>104</td>
      <td>76</td>
      <td>https://usegalaxy.eu/published/workflow?id=0a692a2e643b9e1a</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>seurat</td>
      <td>seurat_create, seurat_data, seurat_integrate, seurat_clustering, seurat_preprocessing, seurat_plot, seurat_reduce_dimension</td>
      <td>Seurat - R toolkit for single cell genomics</td>
      <td>2018-08-20</td>
      <td>https://satijalab.org/seurat/</td>
      <td>5.0</td>
      <td>r-seurat</td>
      <td>3.0.2</td>
      <td>To update</td>
      <td>Single Cell, Transcriptomics, Sequence Analysis</td>
      <td></td>
      <td></td>
      <td>RNA-Seq, Transcriptomics</td>
      <td>RNA-Seq, Transcriptomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/seurat</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/seurat</td>
      <td>seurat</td>
      <td>Seurat</td>
      <td>Seurat is an R package designed for QC, analysis, and exploration of single-cell RNA-seq data. Seurat aims to enable users to identify and interpret sources of heterogeneity from single-cell transcriptomic measurements, and to integrate diverse types of single-cell data.</td>
      <td></td>
      <td>7</td>
      <td>7</td>
      <td>7</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>120</td>
      <td>120</td>
      <td>6468</td>
      <td>6468</td>
      <td>109</td>
      <td>109</td>
      <td>2871</td>
      <td>2871</td>
      <td>9</td>
      <td>9</td>
      <td>1332</td>
      <td>1332</td>
      <td>4</td>
      <td>4</td>
      <td>105</td>
      <td>105</td>
      <td>10776</td>
      <td>10776</td>
      <td>242</td>
      <td>242</td>
      <td>https://usegalaxy.eu/published/workflow?id=10c5d0560b977a6f, https://usegalaxy.eu/published/workflow?id=26deb6f2521742ea, https://usegalaxy.eu/published/workflow?id=290373b2f5fb8c6c, https://usegalaxy.eu/published/workflow?id=c52e8b5ab3e8e958, https://usegalaxy.eu/published/workflow?id=d773cbc76253afbf, https://usegalaxy.org/published/workflow?id=59554bfcad77e307, https://workflowhub.eu/workflows/1555?version=1, https://workflowhub.eu/workflows/1560?version=1, https://workflowhub.eu/workflows/1570?version=1</td>
      <td>single-cell/scrna-case_FilterPlotandExplore_SeuratTools, single-cell/scrna-seurat-pbmc3k</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>snapatac2</td>
      <td>snapatac2_clustering, snapatac2_metrics, snapatac2_network, snapatac2_peaks_and_motif, snapatac2_plotting, snapatac2_preprocessing</td>
      <td>SnapATAC2  A Python/Rust package for single-cell epigenomics analysis</td>
      <td>2024-05-16</td>
      <td>https://kzhang.org/SnapATAC2/</td>
      <td>2.8.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>Single Cell, Epigenetics</td>
      <td>Essential dynamics, Dimensionality reduction, Gene expression profiling</td>
      <td>Essential dynamics, Dimensionality reduction, Gene expression profiling</td>
      <td>Epigenomics, Cell biology, Chromosome conformation capture, Mapping, Sequencing</td>
      <td>Epigenomics, Cell biology, Chromosome conformation capture, Mapping, Sequencing</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/snapatac2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/snapatac2</td>
      <td>snapatac</td>
      <td>SnapATAC</td>
      <td>SnapATAC (Single Nucleus Analysis Pipeline for ATAC-seq) is a fast, accurate and comprehensive method for analyzing single cell ATAC-seq datasets.</td>
      <td></td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>14</td>
      <td>14</td>
      <td>2521</td>
      <td>2521</td>
      <td>14</td>
      <td>14</td>
      <td>65</td>
      <td>65</td>
      <td>1</td>
      <td>1</td>
      <td>1633</td>
      <td>1633</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4219</td>
      <td>4219</td>
      <td>29</td>
      <td>29</td>
      <td>https://usegalaxy.eu/published/workflow?id=15cb6b690d23e6ef, https://usegalaxy.eu/published/workflow?id=d20fbb95f15386f5, https://usegalaxy.eu/published/workflow?id=e4e969e9943cdaa5, https://workflowhub.eu/workflows/1077?version=1, https://workflowhub.eu/workflows/1078?version=1, https://workflowhub.eu/workflows/1089?version=1, https://workflowhub.eu/workflows/1551?version=1, https://workflowhub.eu/workflows/1561?version=1</td>
      <td>single-cell/scatac-batch-correction-snapatac2, single-cell/scatac-standard-processing-snapatac2</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>spapros</td>
      <td>spapros_evaluation, spapros_selection</td>
      <td>Select and evaluate probe sets for targeted spatial transcriptomics.</td>
      <td>2024-08-25</td>
      <td>https://github.com/theislab/spapros</td>
      <td>0.1.5</td>
      <td>spapros</td>
      <td>0.1.6</td>
      <td>To update</td>
      <td>Single Cell, Transcriptomics, Sequence Analysis</td>
      <td>Primer and probe design, Visualisation</td>
      <td>Primer and probe design, Visualisation</td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/spapros</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/spapros</td>
      <td>spapros</td>
      <td>spapros</td>
      <td>Spapros is a python package that provides a pipeline for probe set selection and evaluation for targeted spatial transcriptomics data.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>1</td>
      <td>1</td>
      <td>https://usegalaxy.eu/published/workflow?id=9adf0402b641042c</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>velocyto</td>
      <td>velocyto_cli</td>
      <td>Velocyto is a library for the analysis of RNA velocity.</td>
      <td>2021-04-30</td>
      <td>http://velocyto.org/</td>
      <td>0.17.17</td>
      <td>velocyto.py</td>
      <td>0.17.17</td>
      <td>Up-to-date</td>
      <td>Transcriptomics, Single Cell</td>
      <td>RNA-Seq analysis, Splicing analysis</td>
      <td>RNA-Seq analysis, Splicing analysis</td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/velocyto</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/velocyto</td>
      <td>velocyto</td>
      <td>Velocyto</td>
      <td>Estimating RNA velocity in single cell RNA sequencing datasets</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>14</td>
      <td>14</td>
      <td>131</td>
      <td>131</td>
      <td>12</td>
      <td>12</td>
      <td>75</td>
      <td>75</td>
      <td>1</td>
      <td>1</td>
      <td>192</td>
      <td>192</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>398</td>
      <td>398</td>
      <td>27</td>
      <td>27</td>
      <td>https://usegalaxy.eu/published/workflow?id=c5841e0458a4ebf7, https://usegalaxy.eu/published/workflow?id=c58bb47e04157ee6, https://usegalaxy.org.au/published/workflow?id=1e5a77d9e9630115, https://usegalaxy.org.au/published/workflow?id=77e0ca6492f40c9c, https://usegalaxy.org/published/workflow?id=29d51c37d66deeeb, https://usegalaxy.org/published/workflow?id=6592430b80345167</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>volcanoplot</td>
      <td>volcanoplot</td>
      <td>Tool to create a Volcano Plot</td>
      <td>2018-10-01</td>
      <td>https://ggplot2.tidyverse.org/</td>
      <td>4.0.1</td>
      <td>r-ggplot2</td>
      <td>2.2.1</td>
      <td>To update</td>
      <td>Visualization, Transcriptomics, Statistics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/volcanoplot</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/volcanoplot</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>4591</td>
      <td>5038</td>
      <td>38589</td>
      <td>45177</td>
      <td>6962</td>
      <td>7279</td>
      <td>66385</td>
      <td>70667</td>
      <td>1001</td>
      <td>1068</td>
      <td>8606</td>
      <td>9348</td>
      <td>266</td>
      <td>267</td>
      <td>1555</td>
      <td>1558</td>
      <td>126750</td>
      <td>115135</td>
      <td>13652</td>
      <td>12820</td>
      <td>https://usegalaxy.eu/published/workflow?id=04df470a38b10e94, https://usegalaxy.eu/published/workflow?id=17ad465d9515cd25, https://usegalaxy.eu/published/workflow?id=20ada53472782c9b, https://usegalaxy.eu/published/workflow?id=350d7e6976e15d30, https://usegalaxy.eu/published/workflow?id=3ac10b9887fd0fb2, https://usegalaxy.eu/published/workflow?id=466bdd8ba7b67264, https://usegalaxy.eu/published/workflow?id=4ddf2ec5c460d622, https://usegalaxy.eu/published/workflow?id=68e8042f11d5a3fd, https://usegalaxy.eu/published/workflow?id=7136ce9f18c74853, https://usegalaxy.eu/published/workflow?id=76389d8978ec26eb, https://usegalaxy.eu/published/workflow?id=81f1927bfe3ffea1, https://usegalaxy.eu/published/workflow?id=82ef590b5821805d, https://usegalaxy.eu/published/workflow?id=8b1e2fe9f4a07de0, https://usegalaxy.eu/published/workflow?id=95c4d5549e820d88, https://usegalaxy.eu/published/workflow?id=a9469155eee12ff1, https://usegalaxy.eu/published/workflow?id=bc79a9f797334b01, https://usegalaxy.eu/published/workflow?id=c3a11e1ac1aa8383, https://usegalaxy.eu/published/workflow?id=c7b9fb6872444809, https://usegalaxy.eu/published/workflow?id=d1b3b62e7f91e217, https://usegalaxy.eu/published/workflow?id=d41b52a15ba56f54, https://usegalaxy.eu/published/workflow?id=e04be1ab5a16eb9f, https://usegalaxy.eu/published/workflow?id=e7363b9f450c56b4, https://usegalaxy.eu/published/workflow?id=e7e26967cf6cb8d8, https://usegalaxy.eu/published/workflow?id=f58526da1fd68108, https://usegalaxy.eu/published/workflow?id=fd156028b09d213a, https://usegalaxy.org.au/published/workflow?id=04ab43fb3e7840ef, https://usegalaxy.org.au/published/workflow?id=28705b86e0595ded, https://usegalaxy.org.au/published/workflow?id=5fe66b29dcfb22ab, https://usegalaxy.org.au/published/workflow?id=897e9d7432d231e9, https://usegalaxy.org.au/published/workflow?id=8eab5e2b2c683076, https://usegalaxy.org.au/published/workflow?id=9988bc2402f32dc1, https://usegalaxy.org.au/published/workflow?id=a8d9a16eea69b812, https://usegalaxy.org.au/published/workflow?id=a9e59d38c6c710bc, https://usegalaxy.org.au/published/workflow?id=c00b5bc4d20f047d, https://usegalaxy.org/published/workflow?id=0930f955bdac93e8, https://usegalaxy.org/published/workflow?id=13a3954b8894e1b1, https://usegalaxy.org/published/workflow?id=25cdc47baba556c5, https://usegalaxy.org/published/workflow?id=40a65b5bc1dcc0b4, https://usegalaxy.org/published/workflow?id=420a332da24fb9e9, https://usegalaxy.org/published/workflow?id=4c840284d2984fa0, https://usegalaxy.org/published/workflow?id=5944c5effd56b0d6, https://usegalaxy.org/published/workflow?id=667493d318b86b7d, https://usegalaxy.org/published/workflow?id=68832752b883c226, https://usegalaxy.org/published/workflow?id=7497bfdd2085b6e0, https://usegalaxy.org/published/workflow?id=b0e19e7315b64c87, https://usegalaxy.org/published/workflow?id=e40c2e7f6b73fb11, https://usegalaxy.org/published/workflow?id=f92c83df28d4896f, https://workflowhub.eu/workflows/1201?version=5, https://workflowhub.eu/workflows/1207?version=2, https://workflowhub.eu/workflows/1556?version=1, https://workflowhub.eu/workflows/1708?version=1, https://workflowhub.eu/workflows/1709?version=1</td>
      <td>genome-annotation/crispr-screen, single-cell/pseudobulk-analysis, transcriptomics/rna-seq-analysis-clustering-viz, transcriptomics/rna-seq-viz-with-volcanoplot, transcriptomics/rna-seq-viz-with-volcanoplot-r</td>
      <td>True</td>
      <td>False</td>
    </tr>
  </tbody>
</table>

</div>