<script src="https://code.jquery.com/jquery-3.7.0.js"></script>
<script src="https://cdn.datatables.net/1.13.6/js/jquery.dataTables.min.js"></script>
<script src="https://cdn.datatables.net/searchbuilder/1.6.0/js/dataTables.searchBuilder.min.js"></script>
<script src="https://cdn.datatables.net/datetime/1.5.1/js/dataTables.dateTime.min.js"></script>

<script src="https://cdn.datatables.net/responsive/2.5.0/js/dataTables.responsive.min.js"></script>

<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/responsive/2.2.3/css/responsive.dataTables.min.css">

<link rel="stylesheet" href="https://cdn.datatables.net/1.13.6/css/jquery.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/searchbuilder/1.6.0/css/searchBuilder.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/datetime/1.5.1/css/dataTables.dateTime.min.css">

<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" />
<script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>

<style type="text/css">
  .tooltip {
    font-family: Georgia;
    font-size: 18px;
  }
  .tooltip .tooltip-inner {
    background-color: #00FFFF;
    color: #000000;
    min-width: 400px;
  }
  </style>

<script>
    $(document).ready(function() {
    $('#dataframe').DataTable( {
        "sScrollX": "100%",
        "dom": "Qlfrtip",

        responsive: {
            details: {
                type: 'column'
            }
        },
        "bAutoWidth": false,
        columnDefs: [ {
            className: 'control',
            orderable: false,
            targets:   [ 0 ],
            width: 100,
        },
        { type: 'natural', targets: 1 } // allow normal sorting
        ],

    });
});
</script>

<div class="display nowrap" style="width:80%" width="100%">

<table class="dataframe display nowrap" id="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th>Expand</th>
      <th>Galaxy wrapper id</th>
      <th>Galaxy tool ids</th>
      <th>Description</th>
      <th>bio.tool id</th>
      <th>bio.tool ids</th>
      <th>biii</th>
      <th>bio.tool name</th>
      <th>bio.tool description</th>
      <th>EDAM operation</th>
      <th>EDAM topic</th>
      <th>Status</th>
      <th>Source</th>
      <th>ToolShed categories</th>
      <th>ToolShed id</th>
      <th>Date of first commit of the suite</th>
      <th>Galaxy wrapper owner</th>
      <th>Galaxy wrapper source</th>
      <th>Galaxy wrapper parsed folder</th>
      <th>Galaxy wrapper version</th>
      <th>Conda id</th>
      <th>Conda version</th>
      <th>EDAM operation (no superclasses)</th>
      <th>EDAM topic (no superclasses)</th>
      <th>Available on UseGalaxy.org (Main)</th>
      <th>Available on UseGalaxy.org.au</th>
      <th>Available on UseGalaxy.eu</th>
      <th>Available on UseGalaxy.fr</th>
      <th>Tools available on UseGalaxy.org (Main)</th>
      <th>Tools available on UseGalaxy.org.au</th>
      <th>Tools available on UseGalaxy.eu</th>
      <th>Tools available on UseGalaxy.fr</th>
      <th>Tools available on APOSTL</th>
      <th>Tools available on ARGs-OAP</th>
      <th>Tools available on CIRM-CFBP</th>
      <th>Tools available on ChemFlow</th>
      <th>Tools available on Coloc-stats</th>
      <th>Tools available on CropGalaxy</th>
      <th>Tools available on Dintor</th>
      <th>Tools available on GASLINI</th>
      <th>Tools available on Galaxy@AuBi</th>
      <th>Tools available on Galaxy@Pasteur</th>
      <th>Tools available on GalaxyTrakr</th>
      <th>Tools available on Genomic Hyperbrowser</th>
      <th>Tools available on GigaGalaxy</th>
      <th>Tools available on HyPhy HIV NGS Tools</th>
      <th>Tools available on IPK Galaxy Blast Suite</th>
      <th>Tools available on ImmPort Galaxy</th>
      <th>Tools available on InteractoMIX</th>
      <th>Tools available on MISSISSIPPI</th>
      <th>Tools available on Mandoiu Lab</th>
      <th>Tools available on MiModD NacreousMap</th>
      <th>Tools available on Oqtans</th>
      <th>Tools available on Palfinder</th>
      <th>Tools available on PepSimili</th>
      <th>Tools available on PhagePromotor</th>
      <th>Tools available on UseGalaxy.be</th>
      <th>Tools available on UseGalaxy.cz</th>
      <th>Tools available on UseGalaxy.no</th>
      <th>Tools available on Viral Variant Visualizer (VVV)</th>
      <th>No. of tool users (5 years) (usegalaxy.eu)</th>
      <th>No. of tool users (all time) (usegalaxy.eu)</th>
      <th>Tool usage (5 years) (usegalaxy.eu)</th>
      <th>Tool usage (all time) (usegalaxy.eu)</th>
      <th>No. of tool users (5 years) (usegalaxy.org)</th>
      <th>No. of tool users (all time) (usegalaxy.org)</th>
      <th>Tool usage (5 years) (usegalaxy.org)</th>
      <th>Tool usage (all time) (usegalaxy.org)</th>
      <th>No. of tool users (5 years) (usegalaxy.org.au)</th>
      <th>No. of tool users (all time) (usegalaxy.org.au)</th>
      <th>Tool usage (5 years) (usegalaxy.org.au)</th>
      <th>Tool usage (all time) (usegalaxy.org.au)</th>
      <th>No. of tool users (5 years) - all main servers</th>
      <th>No. of tool users (all time) - all main servers</th>
      <th>Tool usage (5 years) - all main servers</th>
      <th>Tool usage (all time) - all main servers</th>
      <th>Deprecated</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td></td>
      <td>agat</td>
      <td>agat</td>
      <td>GTF/GFF analysis toolkit</td>
      <td>agat</td>
      <td>agat</td>
      <td></td>
      <td>AGAT</td>
      <td>Another Gff Analysis Toolkit (AGAT)Suite of tools to handle gene annotations in any GTF/GFF format.</td>
      <td>Data handling, Genome annotation</td>
      <td>Genomics</td>
      <td>To update</td>
      <td>https://github.com/NBISweden/AGAT</td>
      <td>Convert Formats, Statistics, Fasta Manipulation</td>
      <td>agat</td>
      <td>2023-05-22</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/agat</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/agat</td>
      <td>1.4.0</td>
      <td>agat</td>
      <td>1.4.1</td>
      <td>Data handling, Genome annotation</td>
      <td>Genomics</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1436</td>
      <td>1436</td>
      <td>244</td>
      <td>244</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>119</td>
      <td>119</td>
      <td>10</td>
      <td>10</td>
      <td>1555</td>
      <td>1555</td>
      <td>254</td>
      <td>254</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>ambertools</td>
      <td>ambertools_acpype, acpype_Amber2Gromacs, ambertools_antechamber, mmpbsa_mmgbsa, ambertools_parmchk2, parmconv, tleap</td>
      <td>Ambertools is a set of packages for preparing systems for molecular dynamics (MD) simulations and analyzing trajectories.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://ambermd.org/AmberTools.php</td>
      <td>Molecular Dynamics, Computational chemistry</td>
      <td>ambertools</td>
      <td>2020-04-03</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/buildtools/ambertools</td>
      <td>21.10</td>
      <td>ambertools</td>
      <td></td>
      <td></td>
      <td></td>
      <td>7</td>
      <td>2</td>
      <td>7</td>
      <td>0</td>
      <td>7</td>
      <td>2</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>7</td>
      <td>5</td>
      <td>0</td>
      <td>41793</td>
      <td>41881</td>
      <td>510</td>
      <td>511</td>
      <td>1126</td>
      <td>1126</td>
      <td>159</td>
      <td>159</td>
      <td>1552</td>
      <td>1552</td>
      <td>85</td>
      <td>85</td>
      <td>44471</td>
      <td>44559</td>
      <td>754</td>
      <td>755</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>apoc</td>
      <td>apoc</td>
      <td>Large-scale structural comparison of protein pockets</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://cssb.biology.gatech.edu/APoc</td>
      <td>Computational chemistry</td>
      <td>apoc</td>
      <td>2017-07-04</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/apoc/</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/apoc</td>
      <td>1.0+galaxy1</td>
      <td>apoc</td>
      <td>1b16</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>b2btools</td>
      <td>b2btools_single_sequence</td>
      <td>This software suite provides structural predictions for protein sequences made by Bio2Byte group.About Bio2Byte: We investigate how the dynamics, conformational states, and available experimental data of proteins relate to their amino acid sequence.Underlying physical and chemical principles are computationally unraveled through data integration, analysis, and machine learning, so connecting themto biological events and improving our understanding of the way proteins work.</td>
      <td>b2btools</td>
      <td>b2btools</td>
      <td></td>
      <td>b2bTools</td>
      <td>The bio2byte tools server (b2btools) offers the following single protein sequence based predictions:- Backbone and sidechain dynamics (DynaMine)- Helix, sheet, coil and polyproline-II propensity- Early folding propensity (EFoldMine)- Disorder (DisoMine)- Beta-sheet aggregation (Agmata)In addition, multiple sequence alignments (MSAs) can be uploaded to scan the 'biophysical limits' of a protein family as defined in the MSA</td>
      <td>Protein disorder prediction, Protein secondary structure prediction, Protein feature detection</td>
      <td></td>
      <td>To update</td>
      <td>https://bio2byte.be</td>
      <td>Computational chemistry, Molecular Dynamics, Proteomics, Sequence Analysis, Synthetic Biology</td>
      <td></td>
      <td>2022-08-09</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ucsc-tools/b2tools</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/b2btools</td>
      <td>3.0.5+galaxy0</td>
      <td>b2btools</td>
      <td>3.0.7</td>
      <td>Protein disorder prediction, Protein secondary structure prediction</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>369</td>
      <td>369</td>
      <td>19</td>
      <td>19</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>369</td>
      <td>369</td>
      <td>19</td>
      <td>19</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>bio3d</td>
      <td>bio3d_dccm, bio3d_pca, bio3d_rmsd, bio3d_rmsf, bio3d_pca_visualize</td>
      <td>Bio3d is a program that can be used to analyse molecular dynamics trajectories.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://thegrantlab.org/bio3d/index.php</td>
      <td>Computational chemistry</td>
      <td>bio3d</td>
      <td>2018-10-03</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/bio3d</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/bio3d</td>
      <td>2.4_1</td>
      <td>r-bio3d</td>
      <td>2.3_3</td>
      <td></td>
      <td></td>
      <td>5</td>
      <td>4</td>
      <td>5</td>
      <td>0</td>
      <td>5</td>
      <td>4</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>10600</td>
      <td>10729</td>
      <td>1150</td>
      <td>1159</td>
      <td>1042</td>
      <td>1042</td>
      <td>255</td>
      <td>255</td>
      <td>2756</td>
      <td>2756</td>
      <td>215</td>
      <td>215</td>
      <td>14398</td>
      <td>14527</td>
      <td>1620</td>
      <td>1629</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>biomoldyn</td>
      <td>biomd_neqgamma, fastpca, biomd_extract_clusters, biomd_rmsd_clustering</td>
      <td>Tools for MD analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/moldyn/</td>
      <td>Molecular Dynamics, Computational chemistry</td>
      <td>biomoldyn</td>
      <td>2020-01-30</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/biomoldyn</td>
      <td>1.5.2</td>
      <td>scipy</td>
      <td></td>
      <td></td>
      <td></td>
      <td>4</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>3</td>
      <td>0</td>
      <td>493</td>
      <td>493</td>
      <td>119</td>
      <td>119</td>
      <td>441</td>
      <td>441</td>
      <td>148</td>
      <td>148</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>934</td>
      <td>934</td>
      <td>267</td>
      <td>267</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>biotransformer</td>
      <td>biotransformer</td>
      <td>BioTransformer is a tool for prediction of small molecule metabolism in mammals.</td>
      <td>biotransformer</td>
      <td>biotransformer</td>
      <td></td>
      <td>BioTransformer</td>
      <td>BioTransformer is a freely available web server that supports accurate, rapid and comprehensive in silico metabolism prediction.</td>
      <td>Metabolic pathway prediction, PTM site prediction, Natural product identification</td>
      <td>Small molecules, Endocrinology and metabolism, Metabolomics, Carbohydrates, NMR</td>
      <td>Up-to-date</td>
      <td>https://bitbucket.org/djoumbou/biotransformerjar/src/master/</td>
      <td>Metabolomics</td>
      <td>biotransformer</td>
      <td>2020-12-11</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/biotransformer</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/biotransformer</td>
      <td>3.0.20230403</td>
      <td>biotransformer</td>
      <td>3.0.20230403</td>
      <td>Metabolic pathway prediction, PTM site prediction, Natural product identification</td>
      <td>Small molecules, Endocrinology and metabolism, Metabolomics, Carbohydrates, NMR</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>48</td>
      <td>48</td>
      <td>1</td>
      <td>1</td>
      <td>48</td>
      <td>48</td>
      <td>1</td>
      <td>1</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>camera</td>
      <td>abims_CAMERA_annotateDiffreport, abims_CAMERA_combinexsAnnos</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Metabolomics</td>
      <td>camera</td>
      <td>2023-05-25</td>
      <td>workflow4metabolomics</td>
      <td></td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/camera</td>
      <td>1.48.0</td>
      <td>r-snow</td>
      <td>0.4_1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>1</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>677</td>
      <td>787</td>
      <td>114</td>
      <td>120</td>
      <td>60</td>
      <td>60</td>
      <td>30</td>
      <td>30</td>
      <td>734</td>
      <td>746</td>
      <td>26</td>
      <td>26</td>
      <td>1471</td>
      <td>1593</td>
      <td>170</td>
      <td>176</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>canonical_correlation_analysis</td>
      <td>cca1</td>
      <td>Canonical Correlation Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>canonical_correlation_analysis</td>
      <td>2014-05-19</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/canonical_correlation_analysis</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/canonical_correlation_analysis</td>
      <td>1.0.0</td>
      <td>R</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>66</td>
      <td>75</td>
      <td>16</td>
      <td>18</td>
      <td>141</td>
      <td>505</td>
      <td>31</td>
      <td>147</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>207</td>
      <td>580</td>
      <td>47</td>
      <td>165</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>cardinal</td>
      <td>cardinal_classification, cardinal_colocalization, cardinal_combine, cardinal_data_exporter, cardinal_filtering, cardinal_mz_images, cardinal_preprocessing, cardinal_quality_report, cardinal_segmentations, cardinal_single_ion_segmentation, cardinal_spectra_plots</td>
      <td>Statistical and computational tools for analyzing mass spectrometry imaging datasets</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://cardinalmsi.org</td>
      <td>Proteomics, Metabolomics</td>
      <td></td>
      <td>2020-12-23</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/cardinal</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/cardinal</td>
      <td>3.4.3</td>
      <td>bioconductor-cardinal</td>
      <td>3.4.3</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>9</td>
      <td>11</td>
      <td>11</td>
      <td>0</td>
      <td>9</td>
      <td>11</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>11</td>
      <td>9</td>
      <td>0</td>
      <td>41875</td>
      <td>50702</td>
      <td>663</td>
      <td>713</td>
      <td>12</td>
      <td>12</td>
      <td>1</td>
      <td>1</td>
      <td>7512</td>
      <td>7512</td>
      <td>231</td>
      <td>231</td>
      <td>49399</td>
      <td>58226</td>
      <td>895</td>
      <td>945</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>categorize_elements_satisfying_criteria</td>
      <td>categorize_elements_satisfying_criteria</td>
      <td>Categorize Elements satisfying criteria.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>categorize_elements_satisfying_criteria</td>
      <td>2013-09-25</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/categorize_elements_satisfying_criteria</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/categorize_elements_satisfying_criteria</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>37</td>
      <td>41</td>
      <td>4</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>168</td>
      <td>186</td>
      <td>12</td>
      <td>14</td>
      <td>205</td>
      <td>227</td>
      <td>16</td>
      <td>19</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>cell-types-analysis</td>
      <td>ct_build_cell_ontology_dict, ct_check_labels, ct_combine_tool_outputs, ct_downsample_cells, ct_get_consensus_outputs, ct_get_empirical_dist, ct_get_tool_perf_table, ct_get_tool_pvals</td>
      <td>Tools for analysis of predictions from scRNAseq cell type classification tools, see https://github.com/ebi-gene-expression-group/cell-types-analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>suite_cell_types_analysis</td>
      <td>2020-04-07</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/cell-types-analysis</td>
      <td>1.1.1</td>
      <td>cell-types-analysis</td>
      <td>0.1.11</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>4</td>
      <td>4</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>cemitool</td>
      <td>cemitool</td>
      <td>Gene co-expression network analysis tool</td>
      <td>cemitool</td>
      <td>cemitool</td>
      <td></td>
      <td>CEMiTool</td>
      <td>It unifies the discovery and the analysis of coexpression gene modules in a fully automatic manner, while providing a user-friendly html report with high quality graphs. Our tool evaluates if modules contain genes that are over-represented by specific pathways or that are altered in a specific sample group. Additionally, CEMiTool is able to integrate transcriptomic data with interactome information, identifying the potential hubs on each network.</td>
      <td>Enrichment analysis, Pathway or network analysis</td>
      <td>Gene expression, Transcriptomics, Microarray experiment</td>
      <td>Up-to-date</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/CEMiTool.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>cemitool</td>
      <td>2022-10-10</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cemitool</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cemitool</td>
      <td>1.26.0</td>
      <td>bioconductor-cemitool</td>
      <td>1.26.0</td>
      <td>Enrichment analysis, Pathway or network analysis</td>
      <td>Transcriptomics, Microarray experiment</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>225</td>
      <td>225</td>
      <td>32</td>
      <td>32</td>
      <td>271</td>
      <td>271</td>
      <td>61</td>
      <td>61</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>496</td>
      <td>496</td>
      <td>93</td>
      <td>93</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>cfm</td>
      <td>cfmid</td>
      <td>Competitive Fragmentation Modeling (CFM)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://sourceforge.net/p/cfm-id/</td>
      <td>Metabolomics</td>
      <td>cfmid</td>
      <td>2019-03-07</td>
      <td>computational-metabolomics</td>
      <td>https://github.com/computational-metabolomics/cfm-galaxy/tree/master/tools/cfm</td>
      <td>https://github.com/computational-metabolomics/cfm-galaxy/tree/master/tools/cfm</td>
      <td>33</td>
      <td>cfm</td>
      <td>33</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>clustering_from_distmat</td>
      <td>clustering_from_distmat</td>
      <td>Distance matrix-based hierarchical clustering using SciPy</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/clustering_from_distmat/</td>
      <td>Statistics</td>
      <td>clustering_from_distmat</td>
      <td>2024-08-08</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/clustering_from_distmat/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/clustering_from_distmat</td>
      <td>1.1.1</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>12</td>
      <td>12</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>12</td>
      <td>12</td>
      <td>5</td>
      <td>5</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>compute_motif_frequencies_for_all_motifs</td>
      <td>compute_motif_frequencies_for_all_motifs</td>
      <td>Compute Motif Frequencies For All Motifs, motif by motif.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Statistics</td>
      <td>compute_motif_frequencies_for_all_motifs</td>
      <td>2013-09-25</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/compute_motif_frequencies_for_all_motifs</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/compute_motif_frequencies_for_all_motifs</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>85</td>
      <td>96</td>
      <td>9</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>316</td>
      <td>334</td>
      <td>8</td>
      <td>8</td>
      <td>401</td>
      <td>430</td>
      <td>17</td>
      <td>18</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>compute_motifs_frequency</td>
      <td>compute_motifs_frequency</td>
      <td>Compute Motif Frequencies in indel flanking regions.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Statistics</td>
      <td>compute_motifs_frequency</td>
      <td>2013-09-25</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/compute_motifs_frequency</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/compute_motifs_frequency</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>55</td>
      <td>65</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>317</td>
      <td>335</td>
      <td>5</td>
      <td>5</td>
      <td>372</td>
      <td>400</td>
      <td>7</td>
      <td>7</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>compute_q_values</td>
      <td>compute_q_values</td>
      <td>Compute q-values based on multiple simultaneous tests p-values</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>compute_q_values</td>
      <td>2014-04-01</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/compute_q_values</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/compute_q_values</td>
      <td>1.0.1</td>
      <td>R</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>35</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>35</td>
      <td>0</td>
      <td>3</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>correlation</td>
      <td>cor2</td>
      <td>Correlation for numeric columns</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>correlation</td>
      <td>2014-07-28</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/correlation</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/correlation</td>
      <td>1.0.0</td>
      <td>rpy</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>398</td>
      <td>876</td>
      <td>106</td>
      <td>134</td>
      <td>412</td>
      <td>5568</td>
      <td>105</td>
      <td>755</td>
      <td>192</td>
      <td>239</td>
      <td>21</td>
      <td>27</td>
      <td>1002</td>
      <td>6683</td>
      <td>232</td>
      <td>916</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>correlation_analysis</td>
      <td>correlation_analysis</td>
      <td>[Metabolomics][W4M] Metabolites Correlation Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://workflow4metabolomics.org</td>
      <td>Metabolomics</td>
      <td>correlation_analysis</td>
      <td>2020-12-08</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/blob/master/tools/correlation_analysis/</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/correlation_analysis</td>
      <td>1.0.1+galaxy0</td>
      <td>r-batch</td>
      <td>1.1_4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>deseq2</td>
      <td>deseq2</td>
      <td>Differential gene expression analysis based on the negative binomial distribution</td>
      <td>DESeq2</td>
      <td>DESeq2</td>
      <td></td>
      <td>DESeq2</td>
      <td>R/Bioconductor package for differential gene expression analysis based on the negative binomial distribution. Estimate variance-mean dependence in count data from high-throughput sequencing assays and test for differential expression based on a model using the negative binomial distribution.</td>
      <td>Differential gene expression analysis, RNA-Seq analysis</td>
      <td>RNA-Seq</td>
      <td>To update</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/DESeq2.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>deseq2</td>
      <td>2018-11-15</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/deseq2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/deseq2</td>
      <td>2.11.40.8</td>
      <td>bioconductor-deseq2</td>
      <td>1.42.0</td>
      <td>Differential gene expression analysis, RNA-Seq analysis</td>
      <td>RNA-Seq</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>97434</td>
      <td>109474</td>
      <td>11355</td>
      <td>12514</td>
      <td>183905</td>
      <td>246475</td>
      <td>24951</td>
      <td>31957</td>
      <td>18551</td>
      <td>20166</td>
      <td>2647</td>
      <td>2905</td>
      <td>299890</td>
      <td>376115</td>
      <td>38953</td>
      <td>47376</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>deseq2_normalization</td>
      <td>deseq2_normalization</td>
      <td>Normalizes gene hitlists</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, Transcriptomics, Sequence Analysis, Statistics</td>
      <td>deseq2_normalization</td>
      <td>2018-01-05</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/deseq2_normalization</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/deseq2_normalization</td>
      <td>1.40.2+galaxy0</td>
      <td>bioconductor-deseq2</td>
      <td>1.42.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>dexseq</td>
      <td>dexseq, dexseq_count, plotdexseq</td>
      <td>Inference of differential exon usage in RNA-Seq</td>
      <td>dexseq</td>
      <td>dexseq</td>
      <td></td>
      <td>DEXSeq</td>
      <td>The package is focused on finding differential exon usage using RNA-seq exon counts between samples with different experimental designs. It provides functions that allows the user to make the necessary statistical tests based on a model that uses the negative binomial distribution to estimate the variance between biological replicates and generalized linear models for testing. The package also provides functions for the visualization and exploration of the results.</td>
      <td>Enrichment analysis, Exonic splicing enhancer prediction</td>
      <td>RNA-Seq</td>
      <td>Up-to-date</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/DEXSeq.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>dexseq</td>
      <td>2018-05-02</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/dexseq</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/dexseq</td>
      <td>1.48.0</td>
      <td>bioconductor-dexseq</td>
      <td>1.48.0</td>
      <td>Enrichment analysis, Exonic splicing enhancer prediction</td>
      <td>RNA-Seq</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>14874</td>
      <td>16927</td>
      <td>533</td>
      <td>837</td>
      <td>18475</td>
      <td>28854</td>
      <td>1403</td>
      <td>2568</td>
      <td>3854</td>
      <td>4271</td>
      <td>141</td>
      <td>163</td>
      <td>37203</td>
      <td>50052</td>
      <td>2077</td>
      <td>3568</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>dimet</td>
      <td>dimet_@EXECUTABLE@, dimet_@EXECUTABLE@, dimet_@EXECUTABLE@, dimet_@EXECUTABLE@, dimet_@EXECUTABLE@, dimet_@EXECUTABLE@, dimet_@EXECUTABLE@, dimet_@EXECUTABLE@, dimet_@EXECUTABLE@, dimet_@EXECUTABLE@</td>
      <td>DIMet is a bioinformatics pipeline for differential analysis of isotopic targeted labeling data.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/cbib/DIMet</td>
      <td>Metabolomics</td>
      <td></td>
      <td>2023-10-09</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/DIMet</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/dimet</td>
      <td>0.2.4</td>
      <td>dimet</td>
      <td>0.2.4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>draw_stacked_barplots</td>
      <td>draw_stacked_barplots</td>
      <td>Draw Stacked Bar Plots for different categories and different criteria</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Graphics, Statistics</td>
      <td>draw_stacked_barplots</td>
      <td>2013-09-25</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/draw_stacked_barplots</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/draw_stacked_barplots</td>
      <td>1.0.0</td>
      <td>R</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>189</td>
      <td>216</td>
      <td>47</td>
      <td>52</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>183</td>
      <td>213</td>
      <td>22</td>
      <td>31</td>
      <td>372</td>
      <td>429</td>
      <td>69</td>
      <td>83</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>dropletutils</td>
      <td>dropletutils_empty_drops, dropletutils_read_10x</td>
      <td>De-composed DropletUtils functionality tools, based on https://github.com/ebi-gene-expression-group/dropletutils-scripts and DropletUtils 1.0.3</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_dropletutils</td>
      <td>2019-01-22</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/dropletutils</td>
      <td>1.0.4</td>
      <td>dropletutils-scripts</td>
      <td>0.0.5</td>
      <td></td>
      <td></td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>1900</td>
      <td>1901</td>
      <td>412</td>
      <td>413</td>
      <td>609</td>
      <td>609</td>
      <td>131</td>
      <td>131</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2509</td>
      <td>2510</td>
      <td>543</td>
      <td>544</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>dwt_cor_ava_perclass</td>
      <td>compute_p-values_correlation_coefficients_feature_occurrences_between_two_datasets_using_discrete_wavelet_transfom</td>
      <td>Compute P-values and Correlation Coefficients for Feature Occurrences</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>dwt_cor_ava_perclass</td>
      <td>2014-01-27</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/dwt_cor_ava_perclass</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/dwt_cor_ava_perclass</td>
      <td>1.0.1</td>
      <td>r-waveslim</td>
      <td>1.7.5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>136</td>
      <td>141</td>
      <td>8</td>
      <td>11</td>
      <td>136</td>
      <td>141</td>
      <td>8</td>
      <td>11</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>dwt_cor_avb_all</td>
      <td>compute_p-values_correlation_coefficients_featureA_featureB_occurrences_between_two_datasets_using_discrete_wavelet_transfom</td>
      <td>Compute P-values and Correlation Coefficients for Occurrences of Two Set of Features</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>dwt_cor_avb_all</td>
      <td>2014-01-27</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/dwt_cor_avb_all</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/dwt_cor_avb_all</td>
      <td>1.0.1</td>
      <td>r-waveslim</td>
      <td>1.7.5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>124</td>
      <td>124</td>
      <td>7</td>
      <td>7</td>
      <td>124</td>
      <td>124</td>
      <td>7</td>
      <td>7</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>dwt_ivc_all</td>
      <td>compute_p-values_second_moments_feature_occurrences_between_two_datasets_using_discrete_wavelet_transfom</td>
      <td>Compute P-values and Second Moments for Feature Occurrences</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>dwt_ivc_all</td>
      <td>2014-01-27</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/dwt_ivc_all</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/dwt_ivc_all</td>
      <td>1.0.1</td>
      <td>r-waveslim</td>
      <td>1.7.5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>121</td>
      <td>123</td>
      <td>5</td>
      <td>7</td>
      <td>121</td>
      <td>123</td>
      <td>5</td>
      <td>7</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>dwt_var_perclass</td>
      <td>compute_p-values_max_variances_feature_occurrences_in_one_dataset_using_discrete_wavelet_transfom</td>
      <td>Compute P-values and Max Variances for Feature Occurrences</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>dwt_var_perclass</td>
      <td>2014-01-27</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/dwt_var_perclass</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/dwt_var_perclass</td>
      <td>1.0.1</td>
      <td>r-waveslim</td>
      <td>1.7.5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>131</td>
      <td>132</td>
      <td>7</td>
      <td>8</td>
      <td>131</td>
      <td>132</td>
      <td>7</td>
      <td>8</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>dwt_var_perfeature</td>
      <td>dwt_var1</td>
      <td>Wavelet variance using Discrete Wavelet Transfoms</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>dwt_var_perfeature</td>
      <td>2014-01-27</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/dwt_var_perfeature</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/dwt_var_perfeature</td>
      <td>1.0.2</td>
      <td>r-bitops</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>30</td>
      <td>36</td>
      <td>2</td>
      <td>4</td>
      <td>11</td>
      <td>11</td>
      <td>4</td>
      <td>4</td>
      <td>206</td>
      <td>220</td>
      <td>6</td>
      <td>7</td>
      <td>247</td>
      <td>267</td>
      <td>12</td>
      <td>15</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>edger</td>
      <td>edger</td>
      <td>Perform RNA-Seq differential expression analysis using edgeR pipeline</td>
      <td>edger</td>
      <td>edger</td>
      <td></td>
      <td>edgeR</td>
      <td>Differential expression analysis of RNA-seq expression profiles with biological replication. Implements a range of statistical methodology based on the negative binomial distributions, including empirical Bayes estimation, exact tests, generalized linear models and quasi-likelihood tests. As well as RNA-seq, it be applied to differential signal analysis of other types of genomic data that produce counts, including ChIP-seq, SAGE and CAGE.</td>
      <td>Differential gene expression analysis</td>
      <td>Genetics, RNA-Seq, ChIP-seq</td>
      <td>To update</td>
      <td>http://bioconductor.org/packages/release/bioc/html/edgeR.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>edger</td>
      <td>2019-02-08</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/edger</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/edger</td>
      <td>3.36.0</td>
      <td>bioconductor-edger</td>
      <td>4.0.16</td>
      <td>Differential gene expression analysis</td>
      <td>Genetics, RNA-Seq, ChIP-seq</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>19294</td>
      <td>20905</td>
      <td>2423</td>
      <td>2618</td>
      <td>48876</td>
      <td>58723</td>
      <td>6341</td>
      <td>7527</td>
      <td>8010</td>
      <td>8705</td>
      <td>846</td>
      <td>892</td>
      <td>76180</td>
      <td>88333</td>
      <td>9610</td>
      <td>11037</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>egsea</td>
      <td>egsea</td>
      <td>This tool implements the Ensemble of Gene Set Enrichment Analyses (EGSEA) method for gene set testing</td>
      <td>egsea</td>
      <td>egsea</td>
      <td></td>
      <td>EGSEA</td>
      <td>This package implements the Ensemble of Gene Set Enrichment Analyses method for gene set testing.</td>
      <td>Gene set testing</td>
      <td>Systems biology</td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/EGSEA.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>egsea</td>
      <td>2018-01-01</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/egsea</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/egsea</td>
      <td>1.20.0</td>
      <td>bioconductor-egsea</td>
      <td>1.28.0</td>
      <td>Gene set testing</td>
      <td>Systems biology</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2809</td>
      <td>3060</td>
      <td>371</td>
      <td>402</td>
      <td>5179</td>
      <td>5179</td>
      <td>759</td>
      <td>759</td>
      <td>1931</td>
      <td>1931</td>
      <td>226</td>
      <td>226</td>
      <td>9919</td>
      <td>10170</td>
      <td>1356</td>
      <td>1387</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>ez_histograms</td>
      <td>ez_histograms</td>
      <td>ggplot2 histograms and density plots</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/tidyverse/ggplot2</td>
      <td>Visualization, Statistics</td>
      <td>ez_histograms</td>
      <td>2024-02-07</td>
      <td>artbio</td>
      <td>https://github.com/artbio/tools-artbio/tree/main/tools/ez_histograms</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/ez_histograms</td>
      <td>3.4.4</td>
      <td>r-ggplot2</td>
      <td>2.2.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>fasta_clipping_histogram</td>
      <td>cshl_fasta_clipping_histogram</td>
      <td>Length Distribution chart</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hannonlab.cshl.edu/fastx_toolkit/</td>
      <td>Fasta Manipulation, Graphics, Statistics</td>
      <td>fasta_clipping_histogram</td>
      <td>2017-07-10</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/fastx_toolkit/fasta_clipping_histogram</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/fastx_toolkit/fasta_clipping_histogram</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>0.0.14</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5191</td>
      <td>5367</td>
      <td>104</td>
      <td>130</td>
      <td>5191</td>
      <td>5367</td>
      <td>104</td>
      <td>130</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_quality_boxplot</td>
      <td>cshl_fastq_quality_boxplot</td>
      <td>Draw quality score boxplot</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hannonlab.cshl.edu/fastx_toolkit/</td>
      <td>Fastq Manipulation, Graphics, Statistics</td>
      <td>fastq_quality_boxplot</td>
      <td>2017-07-10</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/fastx_toolkit/fastq_quality_boxplot</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/fastx_toolkit/fastq_quality_boxplot</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>0.0.14</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1271</td>
      <td>1393</td>
      <td>131</td>
      <td>176</td>
      <td>12763</td>
      <td>58819</td>
      <td>2073</td>
      <td>8850</td>
      <td>430</td>
      <td>479</td>
      <td>32</td>
      <td>51</td>
      <td>14464</td>
      <td>60691</td>
      <td>2236</td>
      <td>9077</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>fastx_quality_statistics</td>
      <td>cshl_fastx_quality_statistics</td>
      <td>Compute quality statistics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hannonlab.cshl.edu/fastx_toolkit/</td>
      <td>Fastq Manipulation, Statistics</td>
      <td>fastx_quality_statistics</td>
      <td>2018-04-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/fastx_toolkit/fastx_quality_statistics</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/fastx_toolkit/fastx_quality_statistics</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>0.0.14</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2218</td>
      <td>2344</td>
      <td>168</td>
      <td>206</td>
      <td>20551</td>
      <td>60032</td>
      <td>3784</td>
      <td>9866</td>
      <td>2347</td>
      <td>2460</td>
      <td>234</td>
      <td>256</td>
      <td>25116</td>
      <td>64836</td>
      <td>4186</td>
      <td>10328</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>fgsea</td>
      <td>fgsea</td>
      <td>Perform gene set testing using fgsea</td>
      <td>fgsea</td>
      <td>fgsea</td>
      <td></td>
      <td>fgsea</td>
      <td>The package implements an algorithm for fast gene set enrichment analysis. Using the fast algorithm allows to make more permutations and get more fine grained p-values, which allows to use accurate stantard approaches to multiple hypothesis correction.</td>
      <td>Gene-set enrichment analysis</td>
      <td>Genetics</td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/fgsea.html</td>
      <td>Visualization, Transcriptomics, Statistics</td>
      <td>fgsea</td>
      <td>2018-10-19</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fgsea</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fgsea</td>
      <td>1.8.0+galaxy1</td>
      <td>bioconductor-fgsea</td>
      <td>1.28.0</td>
      <td>Gene-set enrichment analysis</td>
      <td>Genetics</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>5618</td>
      <td>5820</td>
      <td>582</td>
      <td>607</td>
      <td>21673</td>
      <td>21673</td>
      <td>938</td>
      <td>938</td>
      <td>869</td>
      <td>869</td>
      <td>110</td>
      <td>110</td>
      <td>28160</td>
      <td>28362</td>
      <td>1630</td>
      <td>1655</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>filter_compounds</td>
      <td>filter_orgmet_anorg</td>
      <td>Tool for filtering organometallics/anorganic compounds from a list of compounds.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/RECETOX/galaxytools/</td>
      <td>Metabolomics</td>
      <td>filter_compounds</td>
      <td>2021-01-06</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/filter_compounds</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/filter_compounds</td>
      <td>3.1.1</td>
      <td>openbabel</td>
      <td>2.3.90dev7d621d9</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>fisher_test</td>
      <td>fishertest</td>
      <td>Fisher's exact test on two-column hit lists.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, Statistics</td>
      <td>fishertest</td>
      <td>2015-06-17</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/fisher_test</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/fisher_test</td>
      <td>2.32.0+galaxy0</td>
      <td>bioconductor-qvalue</td>
      <td>2.34.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>flexynesis</td>
      <td>flexynesis</td>
      <td>This is a deep-learning based multi-omics bulk sequencing data integration suite with a focus on (pre-)clinical endpoint prediction.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/BIMSBbioinfo/flexynesis/tree/main</td>
      <td>Machine Learning, Statistics, Systems Biology</td>
      <td>flexynesis</td>
      <td>2024-08-12</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/flexynesis</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/flexynesis</td>
      <td>0.2.10</td>
      <td>flexynesis</td>
      <td>0.2.10</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>9</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>9</td>
      <td>2</td>
      <td>2</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>free_energy</td>
      <td></td>
      <td>Free energy tools of BRIDGE.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Molecular Dynamics, Computational chemistry</td>
      <td>freeenergy</td>
      <td>2019-11-11</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tools/free_energy</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/free_energy</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>garnett</td>
      <td>garnett_check_markers, garnett_classify_cells, garnett_get_feature_genes, garnett_get_std_output, garnett_train_classifier, garnett_transform_markers, update_marker_file</td>
      <td>De-composed Garnett functionality tools, see https://github.com/ebi-gene-expression-group/garnett-cli and r-garnett 0.2.8</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_garnett</td>
      <td>2020-04-03</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/garnett</td>
      <td>0.2.8</td>
      <td>garnett-cli</td>
      <td>0.0.5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>27</td>
      <td>27</td>
      <td>11</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>27</td>
      <td>27</td>
      <td>11</td>
      <td>11</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>gc_derivatization</td>
      <td>gc_derivatization</td>
      <td>In silico derivatization for GC.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/RECETOX/gc-meox-tms</td>
      <td>Metabolomics</td>
      <td>gc_derivatization</td>
      <td>2024-03-15</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/gc_derivatization</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/gc_derivatization</td>
      <td>1.0.1</td>
      <td>gc-meox-tms</td>
      <td>1.0.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>gcms2isocor</td>
      <td>gcms2isocor</td>
      <td>Conversion from GCMS PostRun Analysis to Isocor</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Metabolomics</td>
      <td>gcms2isocor</td>
      <td>2021-09-16</td>
      <td>genouest</td>
      <td></td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/gcms2isocor</td>
      <td>0.1.0</td>
      <td>openjdk</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>gcms2isocor</td>
      <td>gcms2isocor</td>
      <td>Corrective method dedicated to Isocor for calculating carbon isotopologue distribution from GCMS runs</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/p2m2/p2m2tools</td>
      <td>Metabolomics</td>
      <td>gcms2isocor</td>
      <td>2023-03-15</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/blob/master/tools/gcms2isocor/</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/gcms2isocor</td>
      <td>0.2.1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>genform</td>
      <td>genform</td>
      <td>genform: generation of molecular formulas by high-resolution MS and MS/MS data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://sourceforge.net/projects/genform/</td>
      <td>Metabolomics</td>
      <td>genform</td>
      <td>2019-09-12</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/blob/master/tools/genform/</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/genform</td>
      <td></td>
      <td>genform</td>
      <td>r8</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>68</td>
      <td>68</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>68</td>
      <td>68</td>
      <td>4</td>
      <td>4</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>genomescope</td>
      <td>genomescope</td>
      <td>Analyze unassembled short reads</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/tbenavi1/genomescope2.0</td>
      <td>Statistics</td>
      <td>genomescope</td>
      <td>2021-04-30</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/genomescope</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/genomescope</td>
      <td>2.0.1</td>
      <td>genomescope2</td>
      <td>2.0.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>3181</td>
      <td>3181</td>
      <td>665</td>
      <td>665</td>
      <td>2112</td>
      <td>2112</td>
      <td>435</td>
      <td>435</td>
      <td>1622</td>
      <td>1622</td>
      <td>148</td>
      <td>148</td>
      <td>6915</td>
      <td>6915</td>
      <td>1248</td>
      <td>1248</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>goseq</td>
      <td>goseq</td>
      <td>goseq does selection-unbiased testing for category enrichment amongst differentially expressed (DE) genes for RNA-seq data</td>
      <td>goseq</td>
      <td>goseq</td>
      <td></td>
      <td>GOseq</td>
      <td>Detect Gene Ontology and/or other user defined categories which are over/under represented in RNA-seq data.</td>
      <td>Gene functional annotation</td>
      <td>RNA-Seq</td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/goseq.html</td>
      <td>Statistics, RNA, Micro-array Analysis</td>
      <td>goseq</td>
      <td>2018-09-30</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/goseq</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/goseq</td>
      <td>1.50.0</td>
      <td>bioconductor-goseq</td>
      <td>1.54.0</td>
      <td>Gene functional annotation</td>
      <td>RNA-Seq</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>20207</td>
      <td>21960</td>
      <td>2807</td>
      <td>3139</td>
      <td>24488</td>
      <td>26198</td>
      <td>3066</td>
      <td>3321</td>
      <td>4155</td>
      <td>4155</td>
      <td>545</td>
      <td>545</td>
      <td>48850</td>
      <td>52313</td>
      <td>6418</td>
      <td>7005</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>gprofiler</td>
      <td>gprofiler_convert, gprofiler_gost, gprofiler_orth, gprofiler_random, gprofiler_snpense</td>
      <td>functional enrichment analysis of gene lists, convertion between various types of namespaces, translation gene identifiers between organisms and more</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://biit.cs.ut.ee/gprofiler</td>
      <td>Statistics, Web Services</td>
      <td>gprofiler</td>
      <td>2019-10-30</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gprofiler/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/gprofiler</td>
      <td>@TOOL_VERSION@+galaxy11</td>
      <td>r-gprofiler2</td>
      <td></td>
      <td></td>
      <td></td>
      <td>5</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>4001</td>
      <td>4001</td>
      <td>614</td>
      <td>614</td>
      <td>7882</td>
      <td>7882</td>
      <td>1238</td>
      <td>1238</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>11883</td>
      <td>11883</td>
      <td>1852</td>
      <td>1852</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>graphembed</td>
      <td>graphembed</td>
      <td>Compute a 2D embedding of a data matrix given supervised class information</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/fabriziocosta/GraphEmbed</td>
      <td>Statistics, Graphics</td>
      <td>graphembed</td>
      <td>2019-12-10</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/graphembed/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/graphembed</td>
      <td>2.4</td>
      <td>graph_embed</td>
      <td>2.4</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>78</td>
      <td>78</td>
      <td>6</td>
      <td>6</td>
      <td>120</td>
      <td>120</td>
      <td>43</td>
      <td>43</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>198</td>
      <td>198</td>
      <td>49</td>
      <td>49</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>gromacs</td>
      <td>gmx_check, gmx_editconf, gmx_energy, gmx_get_builtin_file, gmx_rg, gmx_makendx, gmx_merge_topology_files, gmx_em, gmx_restraints, gmx_rmsd, gmx_rmsf, gmx_setup, gmx_sim, gmx_solvate, gmx_trj</td>
      <td>GROMACS is a package for performing molecular dynamics, primarily designed for biochemical molecules such as proteins, lipids and nucleic acids.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/gromacs</td>
      <td>Molecular Dynamics, Computational chemistry</td>
      <td>gromacs</td>
      <td>2022-03-22</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/gromacs</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/gromacs</td>
      <td>2022</td>
      <td>gromacs</td>
      <td>2021.3</td>
      <td></td>
      <td></td>
      <td>14</td>
      <td>8</td>
      <td>15</td>
      <td>0</td>
      <td>14</td>
      <td>8</td>
      <td>15</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>13</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>15</td>
      <td>9</td>
      <td>0</td>
      <td>184067</td>
      <td>184203</td>
      <td>3441</td>
      <td>3451</td>
      <td>9382</td>
      <td>9382</td>
      <td>1438</td>
      <td>1438</td>
      <td>10325</td>
      <td>10325</td>
      <td>648</td>
      <td>648</td>
      <td>203774</td>
      <td>203910</td>
      <td>5527</td>
      <td>5537</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>gsc_center_scale</td>
      <td>center_scale</td>
      <td>Center or scale (standardize) data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Statistics</td>
      <td>gsc_center_scale</td>
      <td>2019-07-09</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_center_scale</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_center_scale</td>
      <td>4.3.1+galaxy0</td>
      <td>r-optparse</td>
      <td>1.3.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>heinz</td>
      <td>heinz_bum, heinz, heinz_scoring, heinz_visualization</td>
      <td>An algorithm for identification of the optimal scoring subnetwork.</td>
      <td>heinz</td>
      <td>heinz, bionet</td>
      <td></td>
      <td>Heinz</td>
      <td>Tool for single-species active module discovery.</td>
      <td>Pathway or network analysis</td>
      <td>Genetics, Gene expression, Molecular interactions, pathways and networks</td>
      <td>To update</td>
      <td>https://github.com/ls-cwi/heinz</td>
      <td>Transcriptomics, Visualization, Statistics</td>
      <td>heinz</td>
      <td>2018-06-11</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/heinz</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/heinz</td>
      <td>1.0</td>
      <td>bioconductor-bionet</td>
      <td>1.62.0</td>
      <td>Pathway or network analysis</td>
      <td>Genetics, Gene expression, Molecular interactions, pathways and networks</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>1281</td>
      <td>1434</td>
      <td>367</td>
      <td>442</td>
      <td>1559</td>
      <td>1559</td>
      <td>636</td>
      <td>636</td>
      <td>678</td>
      <td>678</td>
      <td>52</td>
      <td>52</td>
      <td>3518</td>
      <td>3671</td>
      <td>1055</td>
      <td>1130</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>histogram</td>
      <td>histogram_rpy</td>
      <td>Histogram of a numeric column</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Graphics, Statistics</td>
      <td>histogram</td>
      <td>2014-07-28</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/histogram</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/histogram</td>
      <td>1.0.4</td>
      <td>rpy2</td>
      <td>2.7.8</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>3748</td>
      <td>4661</td>
      <td>866</td>
      <td>1002</td>
      <td>5993</td>
      <td>48272</td>
      <td>1507</td>
      <td>7101</td>
      <td>549</td>
      <td>746</td>
      <td>152</td>
      <td>253</td>
      <td>10290</td>
      <td>53679</td>
      <td>2525</td>
      <td>8356</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>influx_data_manager</td>
      <td>influx_data_manager</td>
      <td>Handling influx_si data inputs in Galaxy workflows</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/MetaboHUB-MetaToul-FluxoMet/tools_w4m/</td>
      <td>Metabolomics</td>
      <td>influx_si_data_manager</td>
      <td>2024-02-09</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/MetaboHUB-MetaToul-FluxoMet/tools_w4m/tree/main/tools/influx_data_manager</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/influx_data_manager</td>
      <td>1.0.2</td>
      <td>influx-si-data-manager</td>
      <td>1.0.3</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>influx_si</td>
      <td>influx_si</td>
      <td>metabolic flux estimation based on [in]stationary labeling</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/sgsokol/influx</td>
      <td>Metabolomics</td>
      <td>influx_si</td>
      <td>2020-04-17</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/blob/master/tools/influx_si/</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/influx_si</td>
      <td>7.0.1</td>
      <td>influx_si</td>
      <td>7.1.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>intervene</td>
      <td>intervene_pairwise, intervene_upset</td>
      <td>Create pairwise and upset plots</td>
      <td>intervene</td>
      <td>intervene</td>
      <td></td>
      <td>Intervene</td>
      <td>Tool for intersection and visualization of multiple gene or genomic region sets. Intervene contains three modules: venn to generate Venn diagrams of up to six sets, upset to generate UpSet plots of multiple sets, and pairwise to compute and visualize intersections of multiple sets as clustered heat maps.</td>
      <td>Sequence comparison, Sequence visualisation</td>
      <td>Computational biology</td>
      <td>Up-to-date</td>
      <td>https://intervene.readthedocs.io</td>
      <td>Statistics</td>
      <td>intervene</td>
      <td>2018-09-11</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/intervene</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/intervene</td>
      <td>0.6.5</td>
      <td>intervene</td>
      <td>0.6.5</td>
      <td>Sequence comparison, Sequence visualisation</td>
      <td>Computational biology</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>1557</td>
      <td>1607</td>
      <td>278</td>
      <td>292</td>
      <td>4458</td>
      <td>4458</td>
      <td>458</td>
      <td>458</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6015</td>
      <td>6065</td>
      <td>736</td>
      <td>750</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>ipo</td>
      <td>ipo4retgroup, ipo4xcmsSet</td>
      <td>[W4M][LC-MS] IPO</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/rietho/IPO</td>
      <td>Metabolomics</td>
      <td>ipo</td>
      <td>2019-10-03</td>
      <td>lecorguille</td>
      <td>https://github.com/rietho/IPO</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/ipo</td>
      <td>1.10.0</td>
      <td>bioconductor-ipo</td>
      <td>1.28.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>isoformswitchanalyzer</td>
      <td>isoformswitchanalyzer</td>
      <td>Statistical identification of isoform switching from RNA-seq derived quantification of novel and/or annotated full-length isoforms.</td>
      <td>IsoformSwitchAnalyzeR</td>
      <td>IsoformSwitchAnalyzeR</td>
      <td></td>
      <td>IsoformSwitchAnalyzeR</td>
      <td>Enables identification of isoform switches with predicted functional consequences from RNA-seq data. Consequences can be chosen from a long list but includes protein domains gain/loss changes in NMD sensitivity etc. It directly supports import of data from Cufflinks/Cuffdiff, Kallisto, Salmon and RSEM but other transcript qunatification tools are easy to import as well.</td>
      <td>Sequence comparison, Sequence analysis</td>
      <td>Computational biology, Gene transcripts</td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/devel/bioc/html/IsoformSwitchAnalyzeR.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>isoformswitchanalyzer</td>
      <td>2023-05-24</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/isoformswitchanalyzer</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/isoformswitchanalyzer</td>
      <td>1.20.0</td>
      <td>bioconductor-isoformswitchanalyzer</td>
      <td>2.2.0</td>
      <td>Sequence comparison, Sequence analysis</td>
      <td>Computational biology, Gene transcripts</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1301</td>
      <td>1301</td>
      <td>59</td>
      <td>59</td>
      <td>1331</td>
      <td>1331</td>
      <td>60</td>
      <td>60</td>
      <td>507</td>
      <td>507</td>
      <td>9</td>
      <td>9</td>
      <td>3139</td>
      <td>3139</td>
      <td>128</td>
      <td>128</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>isolib</td>
      <td>isolib</td>
      <td>Create an isotopic pattern library for given compounds and adducts.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/RECETOX/galaxytools/</td>
      <td>Metabolomics</td>
      <td>isolib</td>
      <td>2023-10-26</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/isolib</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/isolib</td>
      <td>1.0.1+galaxy0</td>
      <td>bioconductor-metabocoreutils</td>
      <td>1.10.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>isoplot</td>
      <td>isoplot</td>
      <td>Isoplot is a software for the visualisation of MS data from C13 labelling experiments</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Metabolomics, Visualization</td>
      <td>isoplot</td>
      <td>2021-07-12</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/llegregam/Isoplot/tree/main</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/isoplot</td>
      <td>1.3.0+galaxy1</td>
      <td>isoplot</td>
      <td>1.3.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>3</td>
      <td>3</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>iwtomics</td>
      <td>iwtomics_loadandplot, iwtomics_plotwithscale, iwtomics_testandplot</td>
      <td>Interval-Wise Testing for Omics Data</td>
      <td>iwtomics</td>
      <td>iwtomics</td>
      <td></td>
      <td>IWTomics</td>
      <td>Implementation of the Interval-Wise Testing (IWT) for omics data. This inferential procedure tests for differences in "Omics" data between two groups of genomic regions (or between a group of genomic regions and a reference center of symmetry), and does not require fixing location and scale at the outset.</td>
      <td>Differential gene expression analysis, Differentially-methylated region identification, Peak calling, Genome annotation, Comparison</td>
      <td>Statistics and probability</td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/IWTomics.html</td>
      <td>Statistics</td>
      <td>iwtomics</td>
      <td>2017-06-08</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/iwtomics</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/iwtomics</td>
      <td>1.0.0</td>
      <td>bioconductor-iwtomics</td>
      <td>1.26.0</td>
      <td>Differential gene expression analysis, Peak calling, Genome annotation, Comparison</td>
      <td>Statistics and probability</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>675</td>
      <td>690</td>
      <td>27</td>
      <td>28</td>
      <td>159</td>
      <td>251</td>
      <td>29</td>
      <td>49</td>
      <td>383</td>
      <td>383</td>
      <td>12</td>
      <td>12</td>
      <td>1217</td>
      <td>1324</td>
      <td>68</td>
      <td>89</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>kernel_canonical_correlation_analysis</td>
      <td>kcca1</td>
      <td>Kernel Canonical Correlation Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>kernel_canonical_correlation_analysis</td>
      <td>2014-05-19</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/kernel_canonical_correlation_analysis</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/kernel_canonical_correlation_analysis</td>
      <td>1.0.0</td>
      <td>rpy</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>62</td>
      <td>70</td>
      <td>4</td>
      <td>4</td>
      <td>207</td>
      <td>334</td>
      <td>27</td>
      <td>71</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>269</td>
      <td>404</td>
      <td>31</td>
      <td>75</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>kernel_principal_component_analysis</td>
      <td>kpca1</td>
      <td>Kernel Principal Component Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>kernel_principal_component_analysis</td>
      <td>2014-05-19</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/kernel_principal_component_analysis</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/kernel_principal_component_analysis</td>
      <td>1.0.0</td>
      <td>rpy</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>144</td>
      <td>157</td>
      <td>26</td>
      <td>30</td>
      <td>268</td>
      <td>707</td>
      <td>83</td>
      <td>248</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>412</td>
      <td>864</td>
      <td>109</td>
      <td>278</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>kmd_hmdb_data_plot</td>
      <td>kmd_hmdb_data_plot</td>
      <td>retrieves data from KMD HMDB API and produce plot and csv file</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics</td>
      <td>Metabolomics</td>
      <td>kmd_hmdb_data_plot</td>
      <td>2023-07-10</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/blob/master/tools/kmd_hmdb_data_plot/</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/kmd_hmdb_data_plot</td>
      <td>1.0.0</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>lda_analysis</td>
      <td>lda_analy1</td>
      <td>Perform Linear Discriminant Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Graphics, Statistics</td>
      <td>lda_analysis</td>
      <td>2014-07-28</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/lda_analysis</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/lda_analysis</td>
      <td>1.0.1</td>
      <td>R</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>147</td>
      <td>159</td>
      <td>46</td>
      <td>48</td>
      <td>793</td>
      <td>1538</td>
      <td>101</td>
      <td>314</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>3</td>
      <td>940</td>
      <td>1701</td>
      <td>147</td>
      <td>365</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>length_and_gc_content</td>
      <td>length_and_gc_content</td>
      <td>Gets gene length and gc content from a fasta and a GTF file</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/length_and_gc_content</td>
      <td>Fasta Manipulation, Statistics, RNA, Micro-array Analysis</td>
      <td>length_and_gc_content</td>
      <td>2016-11-13</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/length_and_gc_content</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/length_and_gc_content</td>
      <td>0.1.2</td>
      <td>r-optparse</td>
      <td>1.3.2</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>4349</td>
      <td>4466</td>
      <td>666</td>
      <td>698</td>
      <td>2093</td>
      <td>2286</td>
      <td>928</td>
      <td>998</td>
      <td>201</td>
      <td>201</td>
      <td>77</td>
      <td>77</td>
      <td>6643</td>
      <td>6953</td>
      <td>1671</td>
      <td>1773</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>limma_voom</td>
      <td>limma_voom</td>
      <td>Perform RNA-Seq differential expression analysis using limma voom pipeline</td>
      <td>limma</td>
      <td>limma</td>
      <td></td>
      <td>limma</td>
      <td>Data analysis, linear models and differential expression for microarray data.</td>
      <td>RNA-Seq analysis</td>
      <td>Molecular biology, Genetics</td>
      <td>Up-to-date</td>
      <td>http://bioconductor.org/packages/release/bioc/html/limma.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>limma_voom</td>
      <td>2019-02-17</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/limma_voom</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/limma_voom</td>
      <td>3.58.1</td>
      <td>bioconductor-limma</td>
      <td>3.58.1</td>
      <td>RNA-Seq analysis</td>
      <td>Molecular biology, Genetics</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>23442</td>
      <td>24466</td>
      <td>2379</td>
      <td>2475</td>
      <td>45734</td>
      <td>49342</td>
      <td>4565</td>
      <td>5032</td>
      <td>9327</td>
      <td>9517</td>
      <td>835</td>
      <td>842</td>
      <td>78503</td>
      <td>83325</td>
      <td>7779</td>
      <td>8349</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>linear_regression</td>
      <td>LinearRegression1</td>
      <td>Perform Linear Regression</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>linear_regression</td>
      <td>2014-04-01</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/linear_regression</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/linear_regression</td>
      <td>1.0.1</td>
      <td>R</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>146</td>
      <td>1103</td>
      <td>31</td>
      <td>271</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>146</td>
      <td>1103</td>
      <td>31</td>
      <td>271</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>logistic_regression_vif</td>
      <td>LogisticRegression</td>
      <td>Perform Logistic Regression with vif</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis, Statistics</td>
      <td>logistic_regression_vif</td>
      <td>2014-04-01</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/logistic_regression_vif</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/logistic_regression_vif</td>
      <td>1.0.1</td>
      <td>numpy</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>64</td>
      <td>173</td>
      <td>12</td>
      <td>48</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>64</td>
      <td>173</td>
      <td>12</td>
      <td>48</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>macs2</td>
      <td>macs2_bdgbroadcall, macs2_bdgcmp, macs2_bdgdiff, macs2_bdgpeakcall, macs2_callpeak, macs2_filterdup, macs2_predictd, macs2_randsample, macs2_refinepeak</td>
      <td>MACS - Model-based Analysis of ChIP-Seq</td>
      <td>macs</td>
      <td>macs</td>
      <td></td>
      <td>MACS</td>
      <td>Model-based Analysis of ChIP-seq data.</td>
      <td>Peak calling, Enrichment analysis, Gene regulatory network analysis</td>
      <td>ChIP-seq, Molecular interactions, pathways and networks, Transcription factors and regulatory sites</td>
      <td>Up-to-date</td>
      <td>https://github.com/taoliu/MACS</td>
      <td>Sequence Analysis, Statistics</td>
      <td>macs2</td>
      <td>2018-01-21</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/macs2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/macs2</td>
      <td>2.2.9.1</td>
      <td>macs2</td>
      <td>2.2.9.1</td>
      <td>Peak calling, Enrichment analysis, Gene regulatory network analysis</td>
      <td>ChIP-seq, Molecular interactions, pathways and networks, Transcription factors and regulatory sites</td>
      <td>9</td>
      <td>9</td>
      <td>9</td>
      <td>9</td>
      <td>9</td>
      <td>9</td>
      <td>9</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>9</td>
      <td>9</td>
      <td>0</td>
      <td>87810</td>
      <td>96069</td>
      <td>4980</td>
      <td>5629</td>
      <td>153500</td>
      <td>227569</td>
      <td>12946</td>
      <td>18825</td>
      <td>10988</td>
      <td>12301</td>
      <td>692</td>
      <td>807</td>
      <td>252298</td>
      <td>335939</td>
      <td>18618</td>
      <td>25261</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>mapping_quality_stats</td>
      <td>mapqstatistics</td>
      <td>Collects and shows the distribution of MAPQ values in a BAM alignment file</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Sequence Analysis, Statistics</td>
      <td>mapping_quality_stats</td>
      <td>2022-06-15</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/mapping_quality_stats</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/mapping_quality_stats</td>
      <td>0.22.0</td>
      <td>r-optparse</td>
      <td>1.3.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>masigpro</td>
      <td>masigpro</td>
      <td>Identify significantly differential expression profiles in time-course microarray experiments</td>
      <td>masigpro</td>
      <td>masigpro</td>
      <td></td>
      <td>maSigPro</td>
      <td>Regression based approach to find genes for which there are significant gene expression profile differences between experimental groups in time course microarray and RNA-Seq experiments.</td>
      <td>Regression analysis</td>
      <td>Gene expression, Molecular genetics, Microarray experiment, RNA-Seq</td>
      <td>To update</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/maSigPro.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>masigpro</td>
      <td>2017-05-09</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/masigpro</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/masigpro</td>
      <td>1.49.3</td>
      <td>coreutils</td>
      <td>8.25</td>
      <td>Regression analysis</td>
      <td>Gene expression, Microarray experiment, RNA-Seq</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>471</td>
      <td>582</td>
      <td>35</td>
      <td>40</td>
      <td>589</td>
      <td>589</td>
      <td>80</td>
      <td>80</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1060</td>
      <td>1171</td>
      <td>115</td>
      <td>120</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>matchms</td>
      <td>matchms_add_key, matchms_convert, matchms_filtering, matchms_fingerprint_similarity, matchms_formatter, matchms_metadata_export, matchms_metadata_match, matchms_metadata_merge, matchms_networking, matchms_remove_key, matchms_remove_spectra, matchms_spectral_similarity, matchms_split, matchms_subsetting</td>
      <td>Searching, filtering and converting mass spectral libraries.</td>
      <td>matchms</td>
      <td>matchms</td>
      <td></td>
      <td>Matchms</td>
      <td>Tool to import, process, clean, and compare mass spectrometry data.</td>
      <td>Spectral library search, Format validation, Filtering</td>
      <td>Metabolomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/matchms/matchms</td>
      <td>Metabolomics</td>
      <td>matchms</td>
      <td>2023-12-12</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/matchms</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/matchms</td>
      <td>0.27.0</td>
      <td>matchms</td>
      <td>0.27.0</td>
      <td>Spectral library search, Format validation, Filtering</td>
      <td>Metabolomics</td>
      <td>2</td>
      <td>14</td>
      <td>14</td>
      <td>11</td>
      <td>2</td>
      <td>14</td>
      <td>14</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>1620</td>
      <td>1620</td>
      <td>70</td>
      <td>70</td>
      <td>27</td>
      <td>27</td>
      <td>8</td>
      <td>8</td>
      <td>1685</td>
      <td>1685</td>
      <td>12</td>
      <td>12</td>
      <td>3332</td>
      <td>3332</td>
      <td>90</td>
      <td>90</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>mdanalysis</td>
      <td>mdanalysis_angle, mdanalysis_dihedral, mdanalysis_distance, mdanalysis_endtoend, mdanalysis_extract_rmsd, mdanalysis_hbonds, mdanalysis_cosine_analysis, mdanalysis_ramachandran_protein, mdanalysis_ramachandran_plot, mdanalysis_rdf</td>
      <td>MDAnalysis is a package for analyzing trajectories from molecular dynamics (MD) simulations</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/MDAnalysis/mdanalysis</td>
      <td>Computational chemistry</td>
      <td>mdanalysis</td>
      <td>2018-10-08</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/mdanalysis</td>
      <td>1.0.0</td>
      <td>mdanalysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td>10</td>
      <td>5</td>
      <td>10</td>
      <td>0</td>
      <td>10</td>
      <td>5</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>3209</td>
      <td>3268</td>
      <td>318</td>
      <td>326</td>
      <td>182</td>
      <td>182</td>
      <td>51</td>
      <td>51</td>
      <td>397</td>
      <td>397</td>
      <td>39</td>
      <td>39</td>
      <td>3788</td>
      <td>3847</td>
      <td>408</td>
      <td>416</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>mdfileconverter</td>
      <td>md_converter</td>
      <td>A tool for interconverting between different MD structure and trajectory file formats.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Molecular Dynamics, Computational chemistry</td>
      <td>md_converter</td>
      <td>2018-10-13</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tools/mdfileconverter</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/mdfileconverter</td>
      <td>1.9.7</td>
      <td>mdtraj</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>26720</td>
      <td>26762</td>
      <td>251</td>
      <td>253</td>
      <td>784</td>
      <td>784</td>
      <td>54</td>
      <td>54</td>
      <td>977</td>
      <td>977</td>
      <td>53</td>
      <td>53</td>
      <td>28481</td>
      <td>28523</td>
      <td>358</td>
      <td>360</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>mdslicer</td>
      <td>md_slicer</td>
      <td>A tool for slicing trajectory files using MDTraj.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Molecular Dynamics, Computational chemistry</td>
      <td>md_converter</td>
      <td>2019-10-07</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tools/mdslicer</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/mdslicer</td>
      <td>1.9.9</td>
      <td>mdtraj</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>174</td>
      <td>174</td>
      <td>19</td>
      <td>19</td>
      <td>6</td>
      <td>6</td>
      <td>4</td>
      <td>4</td>
      <td>441</td>
      <td>441</td>
      <td>7</td>
      <td>7</td>
      <td>621</td>
      <td>621</td>
      <td>30</td>
      <td>30</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>mdtraj</td>
      <td>traj_selections_and_merge</td>
      <td>MDTraj is a python library that allows users to manipulate molecular dynamics (MD) trajectories</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/mdtraj/mdtraj</td>
      <td>Computational chemistry</td>
      <td>mdtraj</td>
      <td>2020-06-24</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/mdtraj</td>
      <td>1.9.7</td>
      <td>mdtraj</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>36</td>
      <td>36</td>
      <td>8</td>
      <td>8</td>
      <td>63</td>
      <td>63</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>99</td>
      <td>99</td>
      <td>12</td>
      <td>12</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>metfrag</td>
      <td>metfrag</td>
      <td>[Metabolomics][LC-MS][MS/MS] MetFrag - Perform metfrag analysis (can work with the output from msPurity)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/computational-metabolomics/metfrag-galaxy</td>
      <td>Metabolomics</td>
      <td></td>
      <td>2019-08-01</td>
      <td>computational-metabolomics</td>
      <td>https://github.com/computational-metabolomics/metfrag-galaxy</td>
      <td>https://github.com/computational-metabolomics/metfrag-galaxy/tree/master/tools/metfrag</td>
      <td>2.4.5+galaxy3</td>
      <td>metfrag</td>
      <td>2.4.5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>192</td>
      <td>192</td>
      <td>20</td>
      <td>20</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>192</td>
      <td>192</td>
      <td>20</td>
      <td>20</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>metfrag-vis</td>
      <td>metfrag_vis</td>
      <td>[Metabolomics][LC-MS][MS/MS] This tool summarises the results generated by MetFrag</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/computational-metabolomics/metfrag-galaxy</td>
      <td>Metabolomics</td>
      <td></td>
      <td>2020-06-30</td>
      <td>computational-metabolomics</td>
      <td>https://github.com/computational-metabolomics/metfrag-galaxy</td>
      <td>https://github.com/computational-metabolomics/metfrag-galaxy/tree/master/tools/metfrag-vis</td>
      <td>2.4.5+galaxy0</td>
      <td>metfrag</td>
      <td>2.4.5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>50</td>
      <td>50</td>
      <td>14</td>
      <td>14</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>50</td>
      <td>50</td>
      <td>14</td>
      <td>14</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>metilene</td>
      <td>metilene</td>
      <td>Differential DNA methylation calling</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA, Statistics</td>
      <td>metilene</td>
      <td>2015-12-14</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/metilene</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/metilene</td>
      <td>0.2.6.1</td>
      <td>metilene</td>
      <td>0.2.8</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>3209</td>
      <td>4572</td>
      <td>312</td>
      <td>442</td>
      <td>479</td>
      <td>479</td>
      <td>79</td>
      <td>79</td>
      <td>315</td>
      <td>315</td>
      <td>51</td>
      <td>51</td>
      <td>4003</td>
      <td>5366</td>
      <td>442</td>
      <td>572</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>mfassignr</td>
      <td>mfassignr_findRecalSeries, mfassignr_histnoise, mfassignr_isofiltr, mfassignr_kmdnoise, mfassignr_mfassign, mfassignr_mfassignCHO, mfassignr_recal, mfassignr_recallist, mfassignr_snplot</td>
      <td>The MFAssignR package was designed for multi-element molecular formula (MF) assignment of ultrahigh resolution mass spectrometry measurements</td>
      <td>mfassignr</td>
      <td>mfassignr</td>
      <td></td>
      <td>MFAssignR</td>
      <td>Molecular formula assignment software for ultrahigh resolution mass spectrometry analysis of environmental complex mixtures.Ultrahigh resolution mass spectrometry is widely used for nontargeted analysis of complex environmental and biological mixtures, such as dissolved organic matter, due to its unparalleled ability to provide accurate mass measurements.</td>
      <td>Visualisation</td>
      <td>Proteomics experiment, Molecular interactions, pathways and networks, Workflows</td>
      <td>Up-to-date</td>
      <td>https://github.com/RECETOX/MFAssignR</td>
      <td>Metabolomics</td>
      <td>MFAssignR</td>
      <td>2024-08-08</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/mfassignr</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/mfassignr</td>
      <td>1.1.1</td>
      <td>r-mfassignr</td>
      <td>1.1.1</td>
      <td>Visualisation</td>
      <td>Proteomics experiment, Molecular interactions, pathways and networks, Workflows</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>10</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>10</td>
      <td>8</td>
      <td>8</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>misc</td>
      <td>target_screen, use_theoretical_mz_annotations</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/RECETOX/galaxytools</td>
      <td>Metabolomics</td>
      <td></td>
      <td>2024-02-16</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/misc</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/misc</td>
      <td>1.0.0</td>
      <td>pandas</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>2</td>
      <td>2</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>mixmodel4repeated_measures</td>
      <td>mixmodel4repeated_measures</td>
      <td>[Metabolomics][W4M][Statistics] Mixed models - Analysis of variance for repeated measures using mixed model</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://workflow4metabolomics.org</td>
      <td>Metabolomics</td>
      <td>mixmodel4repeated_measures</td>
      <td>2022-05-16</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/mixmodel4repeated_measures</td>
      <td>3.1.0</td>
      <td>r-lme4</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>monocle3</td>
      <td>monocle3_create, monocle3_diffExp, monocle3_learnGraph, monocle3_orderCells, monocle3_partition, monocle3_plotCells, monocle3_preprocess, monocle3_reduceDim, monocle3_topmarkers</td>
      <td>De-composed monocle3 functionality tools, based on https://github.com/ebi-gene-expression-group/monocle-scripts and monocle3 0.1.2.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_monocle3</td>
      <td>2019-09-09</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/monocle3</td>
      <td>0.1.4</td>
      <td>monocle3-cli</td>
      <td>0.0.9</td>
      <td></td>
      <td></td>
      <td>9</td>
      <td>0</td>
      <td>9</td>
      <td>0</td>
      <td>9</td>
      <td>0</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3610</td>
      <td>3610</td>
      <td>559</td>
      <td>559</td>
      <td>1728</td>
      <td>1728</td>
      <td>434</td>
      <td>434</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5338</td>
      <td>5338</td>
      <td>993</td>
      <td>993</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>ms2deepscore</td>
      <td>ms2deepscore_config_generator, ms2deepscore_similarity, ms2deepscore_training</td>
      <td>Mass spectra similarity scoring using a trained ms2deepscore model.</td>
      <td>ms2deepscore</td>
      <td>ms2deepscore</td>
      <td></td>
      <td>MS2DeepScore</td>
      <td>MS2DeepScore is a deep learning similarity measure for mass fragmentation spectrum comparisons. MS2DeepScore provides a Siamese neural network that is trained to predict molecular structural similarities (Tanimoto scores) from pairs of mass spectrometry spectra.</td>
      <td>Spectrum calculation, Spectral library search, Network analysis</td>
      <td>Proteomics experiment, Machine learning, Metabolomics, Small molecules, Compound libraries and screening</td>
      <td>To update</td>
      <td>https://github.com/matchms/ms2deepscore</td>
      <td>Metabolomics</td>
      <td>ms2deepscore</td>
      <td>2024-08-15</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/ms2deepscore</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/ms2deepscore</td>
      <td>2.0.0</td>
      <td>ms2deepscore</td>
      <td>2.2.0</td>
      <td>Spectrum calculation, Spectral library search, Network analysis</td>
      <td>Proteomics experiment, Machine learning, Metabolomics, Small molecules, Compound libraries and screening</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>ms2snoop</td>
      <td>ms2snoop</td>
      <td>[W4M][Utils] Extraction et nettoyage des spectre MS1/2 post-traitement MSPurity.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://workflow4metabolomics.org</td>
      <td>Metabolomics</td>
      <td>ms2snoop</td>
      <td>2022-03-15</td>
      <td>workflow4metabolomics</td>
      <td></td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/ms2snoop</td>
      <td>2.2.1</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>msPurity</td>
      <td>mspurity_averagefragspectra, mspurity_combineannotations, mspurity_createdatabase, mspurity_createmsp, mspurity_dimspredictpuritysingle, mspurity_filterfragspectra, mspurity_flagremove, mspurity_frag4feature, mspurity_puritya, mspurity_purityx, mspurity_spectralmatching</td>
      <td>[Metabolomics][LC-MS][DIMS][MS/MS] msPurity -  R package for precursor ion purity assessments, data processing and spectral matching for LC-MS(/MS) and DI-MS(/MS) data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/computational-metabolomics/mspurity-galaxy</td>
      <td>Metabolomics</td>
      <td></td>
      <td>2019-05-24</td>
      <td>computational-metabolomics</td>
      <td>https://github.com/computational-metabolomics/mspurity-galaxy</td>
      <td>https://github.com/computational-metabolomics/mspurity-galaxy/tree/master/tools/msPurity</td>
      <td>1.28.0</td>
      <td>bioconductor-mspurity</td>
      <td>1.28.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>848</td>
      <td>848</td>
      <td>91</td>
      <td>91</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>848</td>
      <td>848</td>
      <td>91</td>
      <td>91</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>msmetaenhancer</td>
      <td>msmetaenhancer</td>
      <td></td>
      <td>msmetaenhancer</td>
      <td>msmetaenhancer</td>
      <td></td>
      <td>MSMetaEnhancer</td>
      <td>Tool for mass spectra metadata annotation.</td>
      <td>Annotation, Standardisation and normalisation</td>
      <td>Metabolomics, Compound libraries and screening, Data submission, annotation and curation</td>
      <td>Up-to-date</td>
      <td>https://github.com/RECETOX/MSMetaEnhancer</td>
      <td>Metabolomics</td>
      <td></td>
      <td>2022-04-22</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/msmetaenhancer</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/msmetaenhancer</td>
      <td>0.4.0</td>
      <td>msmetaenhancer</td>
      <td>0.4.0</td>
      <td>Annotation, Standardisation and normalisation</td>
      <td>Metabolomics, Compound libraries and screening, Data submission, annotation and curation</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>203</td>
      <td>203</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>48</td>
      <td>48</td>
      <td>1</td>
      <td>1</td>
      <td>251</td>
      <td>251</td>
      <td>11</td>
      <td>11</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>msp_merge</td>
      <td>msp_merge</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/RECETOX/galaxytools</td>
      <td>Metabolomics</td>
      <td></td>
      <td>2022-05-05</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/msp_merge</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/msp_merge</td>
      <td>0.1.0</td>
      <td>matchms</td>
      <td>0.27.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>msp_split</td>
      <td>msp_split</td>
      <td>[Metabolomics][LC-MS][DIMS][MS/MS] msPurity -  R package for precursor ion purity assessments, data processing and spectral matching for LC-MS(/MS) and DI-MS(/MS) data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/computational-metabolomics/mspurity-galaxy</td>
      <td>Metabolomics</td>
      <td></td>
      <td>2019-09-24</td>
      <td>tomnl</td>
      <td>https://github.com/computational-metabolomics/mspurity-galaxy</td>
      <td>https://github.com/computational-metabolomics/mspurity-galaxy/tree/master/tools/utils/msp_split</td>
      <td>0.0.2</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>multigsea</td>
      <td>multigsea</td>
      <td>GSEA-based pathway enrichment analysis for multi-omics data</td>
      <td>multiGSEA</td>
      <td>multiGSEA</td>
      <td></td>
      <td>multiGSEA</td>
      <td>A GSEA-based pathway enrichment analysis for multi-omics data.multiGSEA: a GSEA-based pathway enrichment analysis for multi-omics data, BMC Bioinformatics 21, 561 (2020).Combining GSEA-based pathway enrichment with multi omics data integration.</td>
      <td>Gene-set enrichment analysis, Aggregation, Pathway analysis</td>
      <td>Metabolomics, Molecular interactions, pathways and networks, Proteomics, Transcriptomics, Small molecules</td>
      <td>Up-to-date</td>
      <td>https://bioconductor.org/packages/devel/bioc/html/multiGSEA.html</td>
      <td>Transcriptomics, Proteomics, Statistics</td>
      <td>multigsea</td>
      <td>2023-06-07</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/multigsea</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/multigsea</td>
      <td>1.12.0</td>
      <td>bioconductor-multigsea</td>
      <td>1.12.0</td>
      <td>Gene-set enrichment analysis, Aggregation, Pathway analysis</td>
      <td>Metabolomics, Molecular interactions, pathways and networks, Proteomics, Transcriptomics, Small molecules</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>106</td>
      <td>106</td>
      <td>25</td>
      <td>25</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>106</td>
      <td>106</td>
      <td>25</td>
      <td>25</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>multiqc</td>
      <td>multiqc</td>
      <td>MultiQC aggregates results from bioinformatics analyses across many samples into a single report</td>
      <td>multiqc</td>
      <td>multiqc</td>
      <td></td>
      <td>MultiQC</td>
      <td>MultiQC aggregates results from multiple bioinformatics analyses across many samples into a single report. It searches a given directory for analysis logs and compiles a HTML report. It's a general use tool, perfect for summarising the output from numerous bioinformatics tools.</td>
      <td>Validation, Sequencing quality control</td>
      <td>Sequencing, Bioinformatics, Sequence analysis, Genomics</td>
      <td>To update</td>
      <td>http://multiqc.info/</td>
      <td>Fastq Manipulation, Statistics, Visualization</td>
      <td>multiqc</td>
      <td>2020-04-28</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/multiqc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/multiqc</td>
      <td>1.24.1</td>
      <td>multiqc</td>
      <td>1.25.1</td>
      <td>Sequencing quality control</td>
      <td>Sequencing, Bioinformatics, Sequence analysis, Genomics</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>182221</td>
      <td>192389</td>
      <td>19343</td>
      <td>20494</td>
      <td>163961</td>
      <td>182413</td>
      <td>26449</td>
      <td>29758</td>
      <td>40663</td>
      <td>41855</td>
      <td>5974</td>
      <td>6157</td>
      <td>386845</td>
      <td>416657</td>
      <td>51766</td>
      <td>56409</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>mzmine</td>
      <td>mzmine_batch</td>
      <td>mass-spectrometry data processing, with the main focus on LC-MS data</td>
      <td>mzmine</td>
      <td>mzmine</td>
      <td></td>
      <td>MZmine</td>
      <td>Toolbox for visualization and analysis of LC-MS data in netCDF or mzXML.</td>
      <td>Natural product identification, Standardisation and normalisation, Peptide database search, Deisotoping, Clustering, Filtering, Chromatographic alignment, Peak detection, Peptide identification, Chromatogram visualisation, Mass spectrum visualisation, Structure visualisation, Plotting, Heat map generation</td>
      <td>Proteomics, Metabolomics, Proteomics experiment, Small molecules</td>
      <td>Up-to-date</td>
      <td>http://mzmine.github.io/</td>
      <td>Metabolomics</td>
      <td>mzmine_batch</td>
      <td>2023-10-21</td>
      <td>iuc</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/blob/master/tools/mzmine/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mzmine</td>
      <td>3.9.0</td>
      <td>mzmine</td>
      <td>3.9.0</td>
      <td>Natural product identification, Standardisation and normalisation, Peptide database search, Deisotoping, Clustering, Filtering, Chromatographic alignment, Peak detection, Peptide identification, Chromatogram visualisation, Mass spectrum visualisation, Structure visualisation, Plotting, Heat map generation</td>
      <td>Proteomics, Metabolomics, Proteomics experiment, Small molecules</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>9</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>9</td>
      <td>5</td>
      <td>5</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>mzml_validator</td>
      <td>mzml_validator</td>
      <td>mzML Validator checks if mzML file validates against XML Schema Definition of HUPO Proteomics Standard Initiative.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/RECETOX/galaxytools</td>
      <td>Metabolomics, Proteomics</td>
      <td></td>
      <td>2023-01-12</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/mzml_validator</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/mzml_validator</td>
      <td>0.1.0+galaxy2</td>
      <td>lxml</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>mzxmlshaper</td>
      <td>mzxmlshaper</td>
      <td>Convert mzML, mzXML or netCDF files to mzML or mzXML</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://rdrr.io/bioc/mzR/</td>
      <td>Metabolomics</td>
      <td>mzxmlshaper</td>
      <td>2024-09-20</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/mzxmlshaper</td>
      <td>1.0.0+galaxy0</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>nmr_annotation</td>
      <td>NmrAnnotation</td>
      <td>[Metabolomics][W4M][NMR] NMR Annotation - Annotation of complex mixture NMR spectra and metabolite proportion estimation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://workflow4metabolomics.org</td>
      <td>Metabolomics</td>
      <td>nmr_annotation</td>
      <td>2019-07-30</td>
      <td>marie-tremblay-metatoul</td>
      <td>https://github.com/workflow4metabolomics/nmr_annotation</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/nmr_annotation</td>
      <td>3.0.0</td>
      <td>r-batch</td>
      <td>1.1_4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>49</td>
      <td>49</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>49</td>
      <td>49</td>
      <td>5</td>
      <td>5</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>nmr_annotation2d</td>
      <td>2DNmrAnnotation</td>
      <td>[Metabolomics][W4M][NMR] NMR Annotation2D - Automatic annotation of bi-dimensional NMR spectra</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://workflow4metabolomics.org</td>
      <td>Metabolomics</td>
      <td>2dnmrannotation</td>
      <td>2021-01-21</td>
      <td>marie-tremblay-metatoul</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/nmr_annotation2d</td>
      <td>2.0.0</td>
      <td>r-batch</td>
      <td>1.1_4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>nmr_preprocessing</td>
      <td>NMR_Preprocessing, NMR_Read</td>
      <td>[Metabolomics][W4M][NMR] NMR Preprocessing - Preprocessing of 1D NMR spectra from FID to baseline correction</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://workflow4metabolomics.org</td>
      <td>Metabolomics</td>
      <td>nmr_preprocessing</td>
      <td>2019-07-29</td>
      <td>marie-tremblay-metatoul</td>
      <td>https://github.com/workflow4metabolomics/nmr_preprocessing</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/nmr_preprocessing</td>
      <td></td>
      <td>r-batch</td>
      <td>1.1_4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>142</td>
      <td>142</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>335</td>
      <td>341</td>
      <td>9</td>
      <td>9</td>
      <td>477</td>
      <td>483</td>
      <td>17</td>
      <td>17</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>normalization</td>
      <td>normalization</td>
      <td>[Metabolomics][W4M][ALL] Normalization (operation applied on each individual spectrum) of preprocessed data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://workflow4metabolomics.org</td>
      <td>Metabolomics</td>
      <td>normalization</td>
      <td>2019-07-30</td>
      <td>marie-tremblay-metatoul</td>
      <td>https://github.com/workflow4metabolomics/normalization</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/normalization</td>
      <td>1.0.7</td>
      <td>r-batch</td>
      <td>1.1_4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>152</td>
      <td>162</td>
      <td>39</td>
      <td>40</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>233</td>
      <td>236</td>
      <td>12</td>
      <td>12</td>
      <td>385</td>
      <td>398</td>
      <td>51</td>
      <td>52</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>numeric_clustering</td>
      <td>numeric_clustering</td>
      <td>Clustering tool for numberic values</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://scikit-learn.org/stable/index.html</td>
      <td>Statistics</td>
      <td>numeric_clustering</td>
      <td>2015-12-19</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/numeric_clustering</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/numeric_clustering</td>
      <td>0.9</td>
      <td>anaconda</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>391</td>
      <td>486</td>
      <td>5</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>391</td>
      <td>486</td>
      <td>5</td>
      <td>11</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>openlabcds2csv</td>
      <td>openlabcds2csv</td>
      <td>Creates a summary of several "Internal Standard Report" OpenLabCDS results.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Metabolomics</td>
      <td>openlabcds2csv</td>
      <td>2021-10-28</td>
      <td>genouest</td>
      <td></td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/openlabcds2csv</td>
      <td>0.1.0</td>
      <td>openjdk</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>openmm</td>
      <td>pdbfixer</td>
      <td>OpenMM is a toolkit for molecular simulation using high performance GPU code.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/openmm</td>
      <td>Molecular Dynamics, Computational chemistry</td>
      <td>openmm</td>
      <td>2022-03-25</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/openmm</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/openmm</td>
      <td>1.8.1</td>
      <td>pdbfixer</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>491</td>
      <td>491</td>
      <td>52</td>
      <td>52</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>491</td>
      <td>491</td>
      <td>52</td>
      <td>52</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>packmol</td>
      <td>packmol</td>
      <td>PACKMOL is a package for creating starting structures for Molecular Dynamics simulations</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://m3g.iqm.unicamp.br/packmol/home.shtml</td>
      <td>Molecular Dynamics, Computational chemistry</td>
      <td>packmol</td>
      <td>2018-10-04</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/buildtools/packmol</td>
      <td>18.169.1</td>
      <td>packmol</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>282</td>
      <td>295</td>
      <td>32</td>
      <td>33</td>
      <td>420</td>
      <td>420</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>702</td>
      <td>715</td>
      <td>40</td>
      <td>41</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>pandas_rolling_window</td>
      <td>pandas_rolling_window</td>
      <td>Rolling window calculations</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.rolling.html</td>
      <td>Statistics</td>
      <td>pandas_rolling_window</td>
      <td>2019-05-20</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/pandas_rolling_window</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/pandas_rolling_window</td>
      <td>0.1</td>
      <td>numpy</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>261</td>
      <td>275</td>
      <td>12</td>
      <td>12</td>
      <td>63</td>
      <td>63</td>
      <td>22</td>
      <td>22</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>324</td>
      <td>338</td>
      <td>34</td>
      <td>34</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>partialr_square</td>
      <td>partialRsq</td>
      <td>Compute partial R square</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>partialr_square</td>
      <td>2014-04-01</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/partialr_square</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/partialr_square</td>
      <td>1.0.0</td>
      <td>R</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>55</td>
      <td>142</td>
      <td>8</td>
      <td>42</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>55</td>
      <td>142</td>
      <td>8</td>
      <td>42</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>pathifier</td>
      <td>pathifier</td>
      <td>pathifier</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://</td>
      <td>Transcriptomics, Statistics</td>
      <td>pathifier</td>
      <td>2019-11-08</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/pathifier</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/pathifier</td>
      <td>1.40.0</td>
      <td>bioconductor-pathifier</td>
      <td>1.40.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>245</td>
      <td>245</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>245</td>
      <td>245</td>
      <td>10</td>
      <td>10</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>pathview</td>
      <td>pathview</td>
      <td>Pathview is a tool set for pathway based data integration and visualization.</td>
      <td>pathview</td>
      <td>pathview</td>
      <td></td>
      <td>pathview</td>
      <td>Tool set for pathway based data integration and visualization that maps and renders a wide variety of biological data on relevant pathway graphs. It downloads the pathway graph data, parses the data file, maps user data to the pathway, and render pathway graph with the mapped data. In addition, it integrates with pathway and gene set (enrichment) analysis tools for large-scale and fully automated analysis.</td>
      <td>Pathway or network analysis, Pathway or network visualisation</td>
      <td>Molecular interactions, pathways and networks, Systems biology, Data visualisation</td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/pathview.html</td>
      <td>Statistics, RNA, Micro-array Analysis</td>
      <td>pathview</td>
      <td>2019-08-26</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pathview</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pathview</td>
      <td>1.34.0</td>
      <td>bioconductor-pathview</td>
      <td>1.42.0</td>
      <td>Pathway or network analysis, Pathway or network visualisation</td>
      <td>Molecular interactions, pathways and networks, Systems biology, Data visualisation</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>6204</td>
      <td>6211</td>
      <td>1261</td>
      <td>1261</td>
      <td>6667</td>
      <td>6667</td>
      <td>1143</td>
      <td>1143</td>
      <td>1160</td>
      <td>1160</td>
      <td>189</td>
      <td>189</td>
      <td>14031</td>
      <td>14038</td>
      <td>2593</td>
      <td>2593</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>pearson_correlation</td>
      <td>Pearson_and_apos_Correlation1</td>
      <td>Pearson and apos Correlation between any two numeric columns</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>pearson_correlation</td>
      <td>2014-07-28</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/pearson_correlation</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/pearson_correlation</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>physiofit</td>
      <td>physiofit</td>
      <td>PhysioFit is a scientific tool designed to quantify cell growth parameters and uptake &amp; production fluxes</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>physiofit.readthedocs.io</td>
      <td>Metabolomics</td>
      <td>physiofit</td>
      <td>2022-09-28</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/MetaSys-LISBP/PhysioFit</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/physiofit</td>
      <td>3.4.0</td>
      <td>physiofit</td>
      <td>3.4.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>physiofit_manager</td>
      <td>physiofit_data_manager</td>
      <td>Handling of physiofit input files</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>physiofit.readthedocs.io</td>
      <td>Metabolomics</td>
      <td>physiofit_manager</td>
      <td>2022-11-03</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/MetaboHUB-MetaToul-FluxoMet/PhysioFit_Data_Manager</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/physiofit_manager</td>
      <td>1.0.1</td>
      <td>physiofit_data_manager</td>
      <td>1.0.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>plot_from_lda</td>
      <td>plot_for_lda_output1</td>
      <td>Draw ROC plot on "Perform LDA" output</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Graphics, Statistics</td>
      <td>plot_from_lda</td>
      <td>2014-07-28</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/plot_from_lda</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/plot_from_lda</td>
      <td>1.0.1</td>
      <td>R</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>65</td>
      <td>77</td>
      <td>19</td>
      <td>25</td>
      <td>153</td>
      <td>886</td>
      <td>38</td>
      <td>343</td>
      <td>142</td>
      <td>155</td>
      <td>8</td>
      <td>10</td>
      <td>360</td>
      <td>1118</td>
      <td>65</td>
      <td>378</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>plotheatmap</td>
      <td>plotheatmap</td>
      <td>This tool can be used to plot heatmap of gene expression data. The genes are chosen based on p-value, FDR, log FC and log CPM from edgeR output.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Computational chemistry</td>
      <td>plotheatmap</td>
      <td>2017-07-04</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/plotheatmap</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/plotheatmap</td>
      <td>1.0</td>
      <td>bioconductor-preprocesscore</td>
      <td>1.64.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>poisson2test</td>
      <td>poisson2test</td>
      <td>Poisson two-sample test</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>Statistics, Metagenomics</td>
      <td>poisson2test</td>
      <td>2014-01-27</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/taxonomy/poisson2test</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/taxonomy/poisson2test</td>
      <td>1.0.0</td>
      <td>taxonomy</td>
      <td>0.10.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>112</td>
      <td>120</td>
      <td>16</td>
      <td>16</td>
      <td>4</td>
      <td>562</td>
      <td>1</td>
      <td>140</td>
      <td>320</td>
      <td>347</td>
      <td>7</td>
      <td>8</td>
      <td>436</td>
      <td>1029</td>
      <td>24</td>
      <td>164</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>principal_component_analysis</td>
      <td>pca1</td>
      <td>Principal Component Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>principal_component_analysis</td>
      <td>2014-05-19</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/principal_component_analysis</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/principal_component_analysis</td>
      <td>1.0.2</td>
      <td>rpy</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>9788</td>
      <td>10032</td>
      <td>207</td>
      <td>244</td>
      <td>3072</td>
      <td>8054</td>
      <td>607</td>
      <td>1488</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>12860</td>
      <td>18086</td>
      <td>814</td>
      <td>1732</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>probecoverage</td>
      <td>probecoverage</td>
      <td>computes and plots read coverage of genomic regions by sequencing datasets</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Sequence Analysis, Genomic Interval Operations, Graphics, Statistics</td>
      <td>probecoverage</td>
      <td>2017-09-22</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/probecoverage</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/probecoverage</td>
      <td>0.22.0</td>
      <td>pysam</td>
      <td>0.22.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>pyteomics</td>
      <td>mztab2tsv</td>
      <td>Tools using the pyteomics library</td>
      <td>pyteomics</td>
      <td>pyteomics</td>
      <td></td>
      <td>Pyteomics</td>
      <td>Framework for proteomics data analysis, supporting mzML, MGF, pepXML and more.</td>
      <td>Protein identification</td>
      <td>Proteomics, Proteomics experiment</td>
      <td>To update</td>
      <td>https://pyteomics.readthedocs.io/en/latest/</td>
      <td>Proteomics, Metabolomics</td>
      <td>pyteomics</td>
      <td>2021-01-15</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pyteomics</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pyteomics</td>
      <td>4.4.1</td>
      <td>pyteomics</td>
      <td>4.7.5</td>
      <td>Protein identification</td>
      <td>Proteomics, Proteomics experiment</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>92</td>
      <td>92</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>92</td>
      <td>92</td>
      <td>4</td>
      <td>4</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>qcxms</td>
      <td>qcxms_getres, qcxms_neutral_run, qcxms_production_run</td>
      <td>QCxMS is a quantum chemical (QC) based program that enables users to calculate mass spectra (MS) using Born-Oppenheimer Molecular Dynamics (MD).</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/grimme-lab/QCxMS</td>
      <td>Computational chemistry, Molecular Dynamics</td>
      <td>QCxMS</td>
      <td>2023-10-21</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/qcxms</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/qcxms</td>
      <td>5.2.1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>11246</td>
      <td>11246</td>
      <td>13</td>
      <td>13</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>11246</td>
      <td>11246</td>
      <td>13</td>
      <td>13</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>ramclustr</td>
      <td>ramclustr, ramclustr_define_experiment</td>
      <td></td>
      <td>ramclustr</td>
      <td>ramclustr</td>
      <td></td>
      <td>RAMClustR</td>
      <td>A feature clustering algorithm for non-targeted mass spectrometric metabolomics data.</td>
      <td>Imputation, Standardisation and normalisation, Clustering, Correlation</td>
      <td>Metabolomics</td>
      <td>To update</td>
      <td>https://rdrr.io/cran/RAMClustR/</td>
      <td>Metabolomics</td>
      <td></td>
      <td>2023-05-31</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/ramclustr</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/ramclustr</td>
      <td>1.3.0</td>
      <td>r-ramclustr</td>
      <td>1.3.1</td>
      <td>Imputation, Standardisation and normalisation, Clustering, Correlation</td>
      <td>Metabolomics</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>58</td>
      <td>58</td>
      <td>22</td>
      <td>22</td>
      <td>29</td>
      <td>29</td>
      <td>14</td>
      <td>14</td>
      <td>344</td>
      <td>344</td>
      <td>4</td>
      <td>4</td>
      <td>431</td>
      <td>431</td>
      <td>40</td>
      <td>40</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>rcx-tk</td>
      <td>rcx-tk</td>
      <td>Tools to process metadata or alkane files.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/RECETOX/rcx-tk</td>
      <td>Metabolomics</td>
      <td>rcx_tk</td>
      <td>2024-08-01</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/rcx-tk/</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/rcx-tk</td>
      <td>0.1.0+galaxy0</td>
      <td>rcx-tk</td>
      <td>0.1.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>rdock</td>
      <td>rdock</td>
      <td>Docking ligands to proteins and nucleic acids</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://rdock.sourceforge.net/</td>
      <td>Computational chemistry</td>
      <td>rdock</td>
      <td>2016-12-14</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/rdock/</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/rdock</td>
      <td>1.0</td>
      <td>rDock</td>
      <td>2013.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>recetox_aplcms</td>
      <td>recetox_aplcms_align_features, recetox_aplcms_compute_clusters, recetox_aplcms_compute_template, recetox_aplcms_correct_time, recetox_aplcms_generate_feature_table, recetox_aplcms_merge_known_table, recetox_aplcms_recover_weaker_signals, recetox_aplcms_remove_noise</td>
      <td>Peak detection tool for HRMS profile data.</td>
      <td>recetox-aplcms</td>
      <td>recetox-aplcms</td>
      <td></td>
      <td>recetox-aplcms</td>
      <td>recetox-aplcms is a tool for peak detection in mass spectrometry data. The tool performs (1) noise removal, (2) peak detection, (3) retention time drift correction, (4) peak alignment and (5) weaker signal recovery as well as (6) suspect screening.</td>
      <td>Chromatographic alignment, Quantification, Peak detection, Feature extraction, Alignment</td>
      <td>Metabolomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/RECETOX/recetox-aplcms</td>
      <td>Metabolomics</td>
      <td>recetox-aplcms</td>
      <td>2023-05-18</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/recetox_aplcms</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/recetox_aplcms</td>
      <td>0.13.3</td>
      <td>r-recetox-aplcms</td>
      <td>0.13.3</td>
      <td>Chromatographic alignment, Quantification, Peak detection, Feature extraction, Alignment</td>
      <td>Metabolomics</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>862</td>
      <td>862</td>
      <td>12</td>
      <td>12</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>48</td>
      <td>48</td>
      <td>1</td>
      <td>1</td>
      <td>910</td>
      <td>910</td>
      <td>13</td>
      <td>13</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>recetox_msfinder</td>
      <td>recetox_msfinder</td>
      <td></td>
      <td>recetox-msfinder</td>
      <td>recetox-msfinder</td>
      <td></td>
      <td>recetox-msfinder</td>
      <td>This is a modified copy of MS-FINDER with source code modifications to make the tool accessible in Galaxy.MS-FINDER - software for structure elucidation of unknown spectra with hydrogen rearrangement (HR) rulesThe program supports molecular formula prediction, metabolie class prediction, and structure elucidation for EI-MS and MS/MS spectra, and the assembly is licensed under the CC-BY 4.0.</td>
      <td>Annotation</td>
      <td>Metabolomics</td>
      <td>To update</td>
      <td>https://github.com/RECETOX/recetox-msfinder</td>
      <td>Metabolomics</td>
      <td></td>
      <td>2022-12-02</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/recetox_msfinder</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/recetox_msfinder</td>
      <td>v3.5.2</td>
      <td></td>
      <td></td>
      <td>Annotation</td>
      <td>Metabolomics</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>47</td>
      <td>47</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>47</td>
      <td>47</td>
      <td>4</td>
      <td>4</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>recetox_xmsannotator</td>
      <td>recetox_xmsannotator_advanced</td>
      <td></td>
      <td>recetox-xmsannotator</td>
      <td>recetox-xmsannotator</td>
      <td></td>
      <td>recetox-xMSannotator</td>
      <td>Annotation tool for untargeted LCMS1 data. Uses a database and adduct list for compound annotation and intensity networks, isotopic patterns and pathways for annotation scoring.</td>
      <td>Expression profile pathway mapping, Structure comparison, Isotopic distributions calculation, Annotation</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/RECETOX/recetox-xMSannotator</td>
      <td>Metabolomics</td>
      <td></td>
      <td>2021-07-21</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/recetox-xmsannotator</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/recetox_xmsannotator</td>
      <td>0.10.0</td>
      <td>r-recetox-xmsannotator</td>
      <td>0.10.0</td>
      <td>Expression profile pathway mapping, Structure comparison, Isotopic distributions calculation, Annotation</td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>93</td>
      <td>93</td>
      <td>2</td>
      <td>2</td>
      <td>93</td>
      <td>93</td>
      <td>2</td>
      <td>2</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>rem_complex</td>
      <td>rem_complex</td>
      <td>Removes molecular coordination complexes.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/RECETOX/galaxytools</td>
      <td>Metabolomics</td>
      <td>rem_complex</td>
      <td>2023-11-21</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/rem_complex</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/rem_complex</td>
      <td>1.0.0</td>
      <td>pandas</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>18</td>
      <td>18</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>18</td>
      <td>18</td>
      <td>2</td>
      <td>2</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>rename_annotated_feature</td>
      <td>rename_annotated_feature</td>
      <td>Update column names in an abundance table using a annotation table with spectral matching results</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/RECETOX/galaxytools/</td>
      <td>Metabolomics</td>
      <td>rename_annotated_feature</td>
      <td>2024-05-13</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/rename_annotated_feature</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/rename_annotated_feature</td>
      <td>1.0.0</td>
      <td>pandas</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>retip</td>
      <td>retip_apply, retip_descriptors, retip_filter_rt, retip_train</td>
      <td></td>
      <td>retip</td>
      <td>retip</td>
      <td></td>
      <td>Retip</td>
      <td>Retention Time Prediction for Compound Annotation in Untargeted Metabolomics.Retip is an R package for predicting Retention Time (RT) for small molecules in a high pressure liquid chromatography (HPLC) Mass Spectrometry analysis.Retip - Retention Time prediction for Metabolomics.Retip: Retention Time Prediction for Compound Annotation in Untargeted Metabolomics Paolo Bonini, Tobias Kind, Hiroshi Tsugawa, Dinesh Kumar Barupal, and Oliver Fiehn Analytical Chemistry 2020 92 (11), 7515-7522 DOI: 10.1021/acs.analchem.9b05765.</td>
      <td>Retention time prediction, Spectrum calculation, Deisotoping, Formatting, Deposition</td>
      <td>Metabolomics, Proteomics experiment, Machine learning, Cheminformatics, Chemistry</td>
      <td>To update</td>
      <td>https://github.com/PaoloBnn/Retip</td>
      <td>Metabolomics</td>
      <td></td>
      <td>2020-09-07</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/retip</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/retip</td>
      <td>0.5.4</td>
      <td></td>
      <td></td>
      <td>Retention time prediction, Spectrum calculation, Deisotoping, Formatting, Deposition</td>
      <td>Metabolomics, Proteomics experiment, Machine learning, Cheminformatics, Chemistry</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>rgcca</td>
      <td>rgcca</td>
      <td>multi-block analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/rgcca-factory/RGCCA</td>
      <td>Statistics</td>
      <td>rgcca</td>
      <td>2020-09-02</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/rgcca</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/rgcca</td>
      <td>3.0.2</td>
      <td>rgccacmd</td>
      <td>3.0.3</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>44</td>
      <td>44</td>
      <td>6</td>
      <td>6</td>
      <td>77</td>
      <td>77</td>
      <td>35</td>
      <td>35</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>121</td>
      <td>121</td>
      <td>41</td>
      <td>41</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>riassigner</td>
      <td>riassigner, riassigner_from_comment</td>
      <td></td>
      <td>riassigner</td>
      <td>riassigner</td>
      <td></td>
      <td>RIAssigner</td>
      <td>RIAssigner is a python tool for retention index (RI) computation for GC-MS data.</td>
      <td>Standardisation and normalisation</td>
      <td>Metabolomics, Compound libraries and screening, Data submission, annotation and curation</td>
      <td>Up-to-date</td>
      <td>https://github.com/RECETOX/RIAssigner</td>
      <td>Metabolomics</td>
      <td></td>
      <td>2021-08-19</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/riassigner</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/riassigner</td>
      <td>0.4.1</td>
      <td>riassigner</td>
      <td>0.4.1</td>
      <td>Standardisation and normalisation</td>
      <td>Metabolomics, Compound libraries and screening, Data submission, annotation and curation</td>
      <td>1</td>
      <td>2</td>
      <td>2</td>
      <td>1</td>
      <td>1</td>
      <td>2</td>
      <td>2</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>46</td>
      <td>46</td>
      <td>19</td>
      <td>19</td>
      <td>15</td>
      <td>15</td>
      <td>4</td>
      <td>4</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>62</td>
      <td>62</td>
      <td>24</td>
      <td>24</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>rmassbank</td>
      <td>rmassbank</td>
      <td>RMassBank is an R package for processing tandem MS files and building of MassBank records.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/MassBank/RMassBank</td>
      <td>Metabolomics</td>
      <td>rmassbank</td>
      <td>2021-02-25</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/rmassbank</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/rmassbank</td>
      <td>3.0.0</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>48</td>
      <td>48</td>
      <td>1</td>
      <td>1</td>
      <td>48</td>
      <td>48</td>
      <td>1</td>
      <td>1</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>ruvseq</td>
      <td>ruvseq</td>
      <td>Remove Unwanted Variation from RNA-Seq Data</td>
      <td>ruvseq</td>
      <td>ruvseq</td>
      <td></td>
      <td>RUVSeq</td>
      <td>This package implements the remove unwanted variation (RUV) methods for the normalization of RNA-Seq read counts between samples.</td>
      <td>Differential gene expression analysis</td>
      <td>Gene expression, RNA-seq</td>
      <td>To update</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/DESeq2.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>ruvseq</td>
      <td>2018-08-30</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ruvseq</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ruvseq</td>
      <td>1.26.0</td>
      <td>bioconductor-ruvseq</td>
      <td>1.36.0</td>
      <td>Differential gene expression analysis</td>
      <td>Gene expression</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1286</td>
      <td>1364</td>
      <td>178</td>
      <td>193</td>
      <td>2220</td>
      <td>2220</td>
      <td>299</td>
      <td>299</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3506</td>
      <td>3584</td>
      <td>477</td>
      <td>492</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>sc3</td>
      <td>sc3_calc_biology, sc3_calc_consens, sc3_calc_dists, sc3_calc_transfs, sc3_estimate_k, sc3_kmeans, sc3_prepare</td>
      <td>De-composed SC3 functionality tools, based on https://github.com/ebi-gene-expression-group/bioconductor-sc3-scripts and SC3 1.8.0.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_sc3</td>
      <td>2018-11-05</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/sc3</td>
      <td>1.8.0</td>
      <td>sc3-scripts</td>
      <td>0.0.6</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>13</td>
      <td>13</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>13</td>
      <td>13</td>
      <td>10</td>
      <td>10</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>scater</td>
      <td>scater_calculate_cpm, scater_calculate_qc_metrics, scater_filter, scater_is_outlier, scater_normalize, scater_read_10x_results</td>
      <td>De-composed Scater functionality tools, based on https://github.com/ebi-gene-expression-group/bioconductor-scater-scripts and Scater 1.8.4.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_scater</td>
      <td>2018-10-11</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/scater</td>
      <td>1.10.0</td>
      <td>scater-scripts</td>
      <td>0.0.5</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>2</td>
      <td>6</td>
      <td>0</td>
      <td>1</td>
      <td>2</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>381</td>
      <td>388</td>
      <td>99</td>
      <td>100</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>297</td>
      <td>297</td>
      <td>9</td>
      <td>9</td>
      <td>678</td>
      <td>685</td>
      <td>108</td>
      <td>109</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>scatterplot</td>
      <td>scatterplot_rpy</td>
      <td>Scatterplot of two numeric columns</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Graphics, Statistics</td>
      <td>scatterplot</td>
      <td>2014-07-28</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/scatterplot</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/scatterplot</td>
      <td>1.0.3</td>
      <td>numpy</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1234</td>
      <td>1752</td>
      <td>179</td>
      <td>261</td>
      <td>680</td>
      <td>9223</td>
      <td>231</td>
      <td>1927</td>
      <td>0</td>
      <td>49</td>
      <td>0</td>
      <td>18</td>
      <td>1914</td>
      <td>11024</td>
      <td>410</td>
      <td>2206</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>scmap</td>
      <td>scmap_get_std_output, scmap_index_cell, scmap_index_cluster, scmap_preprocess_sce, scmap_scmap_cell, scmap_scmap_cluster, scmap_select_features</td>
      <td>De-composed scmap functionality tools, based on https://github.com/ebi-gene-expression-group/scmap-cli and scmap 1.6.0.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_scmap</td>
      <td>2019-08-30</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/scmap</td>
      <td>1.6.4</td>
      <td>scmap-cli</td>
      <td>0.1.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>97</td>
      <td>97</td>
      <td>41</td>
      <td>41</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>97</td>
      <td>97</td>
      <td>41</td>
      <td>41</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>scpred</td>
      <td>scpred_get_feature_space, scpred_get_std_output, scpred_predict_labels, scpred_train_model</td>
      <td>De-composed scPred functionality tools, see https://github.com/ebi-gene-expression-group/scpred-cli and r-scPred 1.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_scpred</td>
      <td>2020-04-07</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/scpred</td>
      <td>1.0.2</td>
      <td>scpred-cli</td>
      <td>0.1.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>25</td>
      <td>25</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>25</td>
      <td>25</td>
      <td>5</td>
      <td>5</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>seurat</td>
      <td>seurat_convert, seurat_dim_plot, seurat_export_cellbrowser, seurat_filter_cells, seurat_find_clusters, seurat_find_markers, seurat_find_neighbours, seurat_find_variable_genes, seurat_hover_locator, seurat_integration, seurat_map_query, seurat_normalise_data, seurat_plot, seurat_read10x, seurat_run_pca, seurat_run_tsne, seurat_run_umap, seurat_scale_data, seurat_select_integration_features</td>
      <td>De-composed Seurat functionality tools, based on https://github.com/ebi-gene-expression-group/r-seurat-scripts and Seurat 2.3.1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_seurat</td>
      <td>2018-07-09</td>
      <td>ebi-gxa</td>
      <td></td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/seurat</td>
      <td>4.0.0</td>
      <td>seurat-scripts</td>
      <td>4.4.0</td>
      <td></td>
      <td></td>
      <td>14</td>
      <td>14</td>
      <td>14</td>
      <td>11</td>
      <td>14</td>
      <td>14</td>
      <td>14</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>19</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2296</td>
      <td>2328</td>
      <td>673</td>
      <td>676</td>
      <td>141</td>
      <td>141</td>
      <td>39</td>
      <td>39</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2437</td>
      <td>2469</td>
      <td>712</td>
      <td>715</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>sklearn</td>
      <td>sklearn_mlxtend_association_rules, sklearn_clf_metrics, sklearn_discriminant_classifier, sklearn_ensemble, sklearn_estimator_attributes, sklearn_feature_selection, sklearn_fitted_model_eval, sklearn_generalized_linear, keras_batch_models, keras_model_builder, keras_model_config, keras_train_and_eval, sklearn_label_encoder, sklearn_lightgbm, ml_visualization_ex, model_prediction, sklearn_model_validation, sklearn_nn_classifier, sklearn_numeric_clustering, sklearn_pairwise_metrics, sklearn_pca, sklearn_build_pipeline, sklearn_data_preprocess, sklearn_regression_metrics, sklearn_sample_generator, sklearn_searchcv, sklearn_model_fit, scipy_sparse, stacking_ensemble_models, sklearn_svm_classifier, sklearn_to_categorical, sklearn_train_test_eval, sklearn_train_test_split</td>
      <td>Machine Learning tool suite from Scikit-learn</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://scikit-learn.org</td>
      <td>Machine Learning, Statistics</td>
      <td>sklearn</td>
      <td>2021-04-12</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/sklearn</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/sklearn</td>
      <td>1.0.11.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>30</td>
      <td>14</td>
      <td>31</td>
      <td>16</td>
      <td>30</td>
      <td>14</td>
      <td>31</td>
      <td>16</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>29</td>
      <td>31</td>
      <td>27</td>
      <td>0</td>
      <td>82169</td>
      <td>116810</td>
      <td>2402</td>
      <td>2666</td>
      <td>18101</td>
      <td>18101</td>
      <td>2375</td>
      <td>2375</td>
      <td>17549</td>
      <td>17549</td>
      <td>321</td>
      <td>321</td>
      <td>117819</td>
      <td>152460</td>
      <td>5098</td>
      <td>5362</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>skyline2isocor</td>
      <td>skyline2isocor</td>
      <td>Converting skyline output to IsoCor input</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/MetaboHUB-MetaToul-FluxoMet/Skyline2IsoCor</td>
      <td>Metabolomics</td>
      <td>skyline2isocor</td>
      <td>2024-02-07</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/MetaboHUB-MetaToul-FluxoMet/tools_w4m</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/skyline2isocor</td>
      <td>1.0.0</td>
      <td>skyline2isocor</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>sleuth</td>
      <td>sleuth</td>
      <td>Sleuth is a program for differential analysis of RNA-Seq data.</td>
      <td>sleuth</td>
      <td>sleuth</td>
      <td></td>
      <td>sleuth</td>
      <td>A statistical model and software application for RNA-seq differential expression analysis.</td>
      <td>Expression data visualisation, Differential gene expression analysis, Gene expression profiling, Statistical calculation</td>
      <td>RNA-seq, Gene expression, Statistics and probability</td>
      <td>Up-to-date</td>
      <td>https://github.com/pachterlab/sleuth</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>sleuth</td>
      <td>2023-05-31</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/sleuth</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/sleuth</td>
      <td>0.30.1</td>
      <td>r-sleuth</td>
      <td>0.30.1</td>
      <td>Expression data visualisation, Differential gene expression analysis, Gene expression profiling, Statistical calculation</td>
      <td>Gene expression, Statistics and probability</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>141</td>
      <td>141</td>
      <td>26</td>
      <td>26</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>141</td>
      <td>141</td>
      <td>26</td>
      <td>26</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>smina</td>
      <td>smina</td>
      <td>smina is a fork of Autodock Vina (http://vina.scripps.edu/) that focuses on improving scoring and minimization</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://sourceforge.net/projects/smina/</td>
      <td>Computational chemistry</td>
      <td>smina</td>
      <td>2017-07-07</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/smina/</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/smina</td>
      <td>1.0</td>
      <td>smina</td>
      <td>2017.11.9</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>499</td>
      <td>499</td>
      <td>16</td>
      <td>16</td>
      <td>44</td>
      <td>44</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>543</td>
      <td>543</td>
      <td>26</td>
      <td>26</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>snpfreq</td>
      <td>hgv_snpFreq</td>
      <td>snpFreq significant SNPs in case-control data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Variant Analysis, Statistics</td>
      <td>snpfreq</td>
      <td>2014-07-28</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/hgv/snpfreq</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/hgv/snpfreq</td>
      <td>1.0.1</td>
      <td>R</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>35</td>
      <td>44</td>
      <td>10</td>
      <td>12</td>
      <td>307</td>
      <td>649</td>
      <td>68</td>
      <td>181</td>
      <td>227</td>
      <td>260</td>
      <td>16</td>
      <td>23</td>
      <td>569</td>
      <td>953</td>
      <td>94</td>
      <td>216</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>spec2vec</td>
      <td>spec2vec_similarity, spec2vec_training</td>
      <td>Mass spectra similarity scoring using a trained Spec2Vec model.</td>
      <td>spec2vec</td>
      <td>spec2vec</td>
      <td></td>
      <td>Spec2Vec</td>
      <td>Improved mass spectral similarity scoring through learning of structural relationships.Spec2vec is a novel spectral similarity score inspired by a natural language processing algorithm -- Word2Vec. Where Word2Vec learns relationships between words in sentences, spec2vec does so for mass fragments and neutral losses in MS/MS spectra. The spectral similarity score is based on spectral embeddings learnt from the fragmental relationships within a large set of spectral data.Analysis and benchmarking of mass spectra similarity measures using gnps data set.</td>
      <td>Spectrum calculation, Spectral library search, Database search, Natural product identification</td>
      <td>Proteomics experiment, Metabolomics, Natural language processing, Proteomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/iomega/spec2vec</td>
      <td>Metabolomics</td>
      <td>spec2vec</td>
      <td>2022-12-07</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/spec2vec</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/spec2vec</td>
      <td>0.8.0</td>
      <td>spec2vec</td>
      <td>0.8.0</td>
      <td>Spectrum calculation, Spectral library search, Database search, Natural product identification</td>
      <td>Proteomics experiment, Metabolomics, Natural language processing, Proteomics</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>statistics</td>
      <td>bg_statistical_hypothesis_testing</td>
      <td>Tool for computing statistical tests.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/statistics</td>
      <td>Statistics</td>
      <td>bg_statistical_hypothesis_testing</td>
      <td>2015-02-28</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/statistics</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/statistics</td>
      <td>0.3</td>
      <td>numpy</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>169</td>
      <td>346</td>
      <td>46</td>
      <td>61</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>169</td>
      <td>346</td>
      <td>46</td>
      <td>61</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__alignment</td>
      <td>qiime2__alignment__mafft, qiime2__alignment__mafft_add, qiime2__alignment__mask</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/qiime2/q2-alignment</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__alignment</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__alignment</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>249</td>
      <td>249</td>
      <td>32</td>
      <td>32</td>
      <td>16</td>
      <td>16</td>
      <td>10</td>
      <td>10</td>
      <td>102</td>
      <td>102</td>
      <td>16</td>
      <td>16</td>
      <td>367</td>
      <td>367</td>
      <td>58</td>
      <td>58</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__composition</td>
      <td>qiime2__composition__add_pseudocount, qiime2__composition__ancom, qiime2__composition__ancombc, qiime2__composition__da_barplot, qiime2__composition__tabulate</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/qiime2/q2-composition</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__composition</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__composition</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>2</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>117</td>
      <td>117</td>
      <td>41</td>
      <td>41</td>
      <td>84</td>
      <td>84</td>
      <td>24</td>
      <td>24</td>
      <td>161</td>
      <td>161</td>
      <td>13</td>
      <td>13</td>
      <td>362</td>
      <td>362</td>
      <td>78</td>
      <td>78</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__cutadapt</td>
      <td>qiime2__cutadapt__demux_paired, qiime2__cutadapt__demux_single, qiime2__cutadapt__trim_paired, qiime2__cutadapt__trim_single</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/qiime2/q2-cutadapt</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__cutadapt</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__cutadapt</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>217</td>
      <td>217</td>
      <td>47</td>
      <td>47</td>
      <td>42</td>
      <td>42</td>
      <td>20</td>
      <td>20</td>
      <td>96</td>
      <td>96</td>
      <td>12</td>
      <td>12</td>
      <td>355</td>
      <td>355</td>
      <td>79</td>
      <td>79</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__dada2</td>
      <td>qiime2__dada2__denoise_ccs, qiime2__dada2__denoise_paired, qiime2__dada2__denoise_pyro, qiime2__dada2__denoise_single</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://benjjneb.github.io/dada2/</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__dada2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__dada2</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>721</td>
      <td>721</td>
      <td>160</td>
      <td>160</td>
      <td>350</td>
      <td>350</td>
      <td>91</td>
      <td>91</td>
      <td>214</td>
      <td>214</td>
      <td>35</td>
      <td>35</td>
      <td>1285</td>
      <td>1285</td>
      <td>286</td>
      <td>286</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__deblur</td>
      <td>qiime2__deblur__denoise_16S, qiime2__deblur__denoise_other, qiime2__deblur__visualize_stats</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/biocore/deblur</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__deblur</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__deblur</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>125</td>
      <td>125</td>
      <td>30</td>
      <td>30</td>
      <td>13</td>
      <td>13</td>
      <td>9</td>
      <td>9</td>
      <td>83</td>
      <td>83</td>
      <td>8</td>
      <td>8</td>
      <td>221</td>
      <td>221</td>
      <td>47</td>
      <td>47</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__demux</td>
      <td>qiime2__demux__emp_paired, qiime2__demux__emp_single, qiime2__demux__filter_samples, qiime2__demux__partition_samples_paired, qiime2__demux__partition_samples_single, qiime2__demux__subsample_paired, qiime2__demux__subsample_single, qiime2__demux__summarize, qiime2__demux__tabulate_read_counts</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/qiime2/q2-demux</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__demux</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__demux</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>396</td>
      <td>396</td>
      <td>134</td>
      <td>134</td>
      <td>121</td>
      <td>121</td>
      <td>57</td>
      <td>57</td>
      <td>144</td>
      <td>144</td>
      <td>39</td>
      <td>39</td>
      <td>661</td>
      <td>661</td>
      <td>230</td>
      <td>230</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__diversity</td>
      <td>qiime2__diversity__adonis, qiime2__diversity__alpha, qiime2__diversity__alpha_correlation, qiime2__diversity__alpha_group_significance, qiime2__diversity__alpha_phylogenetic, qiime2__diversity__alpha_rarefaction, qiime2__diversity__beta, qiime2__diversity__beta_correlation, qiime2__diversity__beta_group_significance, qiime2__diversity__beta_phylogenetic, qiime2__diversity__beta_rarefaction, qiime2__diversity__bioenv, qiime2__diversity__core_metrics, qiime2__diversity__core_metrics_phylogenetic, qiime2__diversity__filter_alpha_diversity, qiime2__diversity__filter_distance_matrix, qiime2__diversity__mantel, qiime2__diversity__partial_procrustes, qiime2__diversity__pcoa, qiime2__diversity__pcoa_biplot, qiime2__diversity__procrustes_analysis, qiime2__diversity__tsne, qiime2__diversity__umap</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/qiime2/q2-diversity</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__diversity</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__diversity</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>21</td>
      <td>21</td>
      <td>22</td>
      <td>21</td>
      <td>21</td>
      <td>21</td>
      <td>22</td>
      <td>21</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>21</td>
      <td>0</td>
      <td>0</td>
      <td>964</td>
      <td>964</td>
      <td>318</td>
      <td>318</td>
      <td>249</td>
      <td>249</td>
      <td>118</td>
      <td>118</td>
      <td>475</td>
      <td>475</td>
      <td>108</td>
      <td>108</td>
      <td>1688</td>
      <td>1688</td>
      <td>544</td>
      <td>544</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__diversity_lib</td>
      <td>qiime2__diversity_lib__alpha_passthrough, qiime2__diversity_lib__beta_passthrough, qiime2__diversity_lib__beta_phylogenetic_meta_passthrough, qiime2__diversity_lib__beta_phylogenetic_passthrough, qiime2__diversity_lib__bray_curtis, qiime2__diversity_lib__faith_pd, qiime2__diversity_lib__jaccard, qiime2__diversity_lib__observed_features, qiime2__diversity_lib__pielou_evenness, qiime2__diversity_lib__shannon_entropy, qiime2__diversity_lib__unweighted_unifrac, qiime2__diversity_lib__weighted_unifrac</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/qiime2/q2-diversity-lib</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__diversity_lib</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__diversity_lib</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>12</td>
      <td>12</td>
      <td>12</td>
      <td>12</td>
      <td>12</td>
      <td>12</td>
      <td>12</td>
      <td>12</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>12</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>13</td>
      <td>13</td>
      <td>11</td>
      <td>11</td>
      <td>1302</td>
      <td>1302</td>
      <td>19</td>
      <td>19</td>
      <td>1323</td>
      <td>1323</td>
      <td>38</td>
      <td>38</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__emperor</td>
      <td>qiime2__emperor__biplot, qiime2__emperor__plot, qiime2__emperor__procrustes_plot</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://emperor.microbio.me</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__emperor</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__emperor</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>58</td>
      <td>58</td>
      <td>23</td>
      <td>23</td>
      <td>24</td>
      <td>24</td>
      <td>11</td>
      <td>11</td>
      <td>92</td>
      <td>92</td>
      <td>11</td>
      <td>11</td>
      <td>174</td>
      <td>174</td>
      <td>45</td>
      <td>45</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__feature_classifier</td>
      <td>qiime2__feature_classifier__blast, qiime2__feature_classifier__classify_consensus_blast, qiime2__feature_classifier__classify_consensus_vsearch, qiime2__feature_classifier__classify_hybrid_vsearch_sklearn, qiime2__feature_classifier__classify_sklearn, qiime2__feature_classifier__extract_reads, qiime2__feature_classifier__find_consensus_annotation, qiime2__feature_classifier__fit_classifier_naive_bayes, qiime2__feature_classifier__fit_classifier_sklearn, qiime2__feature_classifier__makeblastdb, qiime2__feature_classifier__vsearch_global</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/qiime2/q2-feature-classifier</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__feature_classifier</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__feature_classifier</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>10</td>
      <td>10</td>
      <td>10</td>
      <td>10</td>
      <td>10</td>
      <td>10</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>863</td>
      <td>863</td>
      <td>220</td>
      <td>220</td>
      <td>237</td>
      <td>237</td>
      <td>98</td>
      <td>98</td>
      <td>181</td>
      <td>181</td>
      <td>41</td>
      <td>41</td>
      <td>1281</td>
      <td>1281</td>
      <td>359</td>
      <td>359</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__feature_table</td>
      <td>qiime2__feature_table__core_features, qiime2__feature_table__filter_features, qiime2__feature_table__filter_features_conditionally, qiime2__feature_table__filter_samples, qiime2__feature_table__filter_seqs, qiime2__feature_table__group, qiime2__feature_table__heatmap, qiime2__feature_table__merge, qiime2__feature_table__merge_seqs, qiime2__feature_table__merge_taxa, qiime2__feature_table__presence_absence, qiime2__feature_table__rarefy, qiime2__feature_table__relative_frequency, qiime2__feature_table__rename_ids, qiime2__feature_table__split, qiime2__feature_table__subsample_ids, qiime2__feature_table__summarize, qiime2__feature_table__summarize_plus, qiime2__feature_table__tabulate_feature_frequencies, qiime2__feature_table__tabulate_sample_frequencies, qiime2__feature_table__tabulate_seqs, qiime2__feature_table__transpose</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/qiime2/q2-feature-table</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__feature_table</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__feature_table</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>17</td>
      <td>17</td>
      <td>17</td>
      <td>17</td>
      <td>17</td>
      <td>17</td>
      <td>17</td>
      <td>17</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>17</td>
      <td>0</td>
      <td>0</td>
      <td>983</td>
      <td>983</td>
      <td>287</td>
      <td>287</td>
      <td>375</td>
      <td>375</td>
      <td>144</td>
      <td>144</td>
      <td>907</td>
      <td>907</td>
      <td>106</td>
      <td>106</td>
      <td>2265</td>
      <td>2265</td>
      <td>537</td>
      <td>537</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__fragment_insertion</td>
      <td>qiime2__fragment_insertion__classify_otus_experimental, qiime2__fragment_insertion__filter_features, qiime2__fragment_insertion__sepp</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/qiime2/q2-fragment-insertion</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__fragment_insertion</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__fragment_insertion</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>46</td>
      <td>46</td>
      <td>13</td>
      <td>13</td>
      <td>9</td>
      <td>9</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>55</td>
      <td>55</td>
      <td>18</td>
      <td>18</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__longitudinal</td>
      <td>qiime2__longitudinal__anova, qiime2__longitudinal__feature_volatility, qiime2__longitudinal__first_differences, qiime2__longitudinal__first_distances, qiime2__longitudinal__linear_mixed_effects, qiime2__longitudinal__maturity_index, qiime2__longitudinal__nmit, qiime2__longitudinal__pairwise_differences, qiime2__longitudinal__pairwise_distances, qiime2__longitudinal__plot_feature_volatility, qiime2__longitudinal__volatility</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/qiime2/q2-longitudinal</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__longitudinal</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__longitudinal</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>11</td>
      <td>11</td>
      <td>11</td>
      <td>11</td>
      <td>11</td>
      <td>11</td>
      <td>11</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>37</td>
      <td>37</td>
      <td>16</td>
      <td>16</td>
      <td>22</td>
      <td>22</td>
      <td>7</td>
      <td>7</td>
      <td>89</td>
      <td>89</td>
      <td>14</td>
      <td>14</td>
      <td>148</td>
      <td>148</td>
      <td>37</td>
      <td>37</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__metadata</td>
      <td>qiime2__metadata__distance_matrix, qiime2__metadata__merge, qiime2__metadata__shuffle_groups, qiime2__metadata__tabulate</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/qiime2/q2-metadata</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__metadata</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__metadata</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>615</td>
      <td>615</td>
      <td>143</td>
      <td>143</td>
      <td>246</td>
      <td>246</td>
      <td>75</td>
      <td>75</td>
      <td>115</td>
      <td>115</td>
      <td>27</td>
      <td>27</td>
      <td>976</td>
      <td>976</td>
      <td>245</td>
      <td>245</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__phylogeny</td>
      <td>qiime2__phylogeny__align_to_tree_mafft_fasttree, qiime2__phylogeny__align_to_tree_mafft_iqtree, qiime2__phylogeny__align_to_tree_mafft_raxml, qiime2__phylogeny__fasttree, qiime2__phylogeny__filter_table, qiime2__phylogeny__filter_tree, qiime2__phylogeny__iqtree, qiime2__phylogeny__iqtree_ultrafast_bootstrap, qiime2__phylogeny__midpoint_root, qiime2__phylogeny__raxml, qiime2__phylogeny__raxml_rapid_bootstrap, qiime2__phylogeny__robinson_foulds</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/qiime2/q2-phylogeny</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__phylogeny</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__phylogeny</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>12</td>
      <td>12</td>
      <td>12</td>
      <td>12</td>
      <td>12</td>
      <td>12</td>
      <td>12</td>
      <td>12</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>12</td>
      <td>0</td>
      <td>0</td>
      <td>330</td>
      <td>330</td>
      <td>83</td>
      <td>83</td>
      <td>64</td>
      <td>64</td>
      <td>40</td>
      <td>40</td>
      <td>195</td>
      <td>195</td>
      <td>49</td>
      <td>49</td>
      <td>589</td>
      <td>589</td>
      <td>172</td>
      <td>172</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__quality_control</td>
      <td>qiime2__quality_control__bowtie2_build, qiime2__quality_control__decontam_identify, qiime2__quality_control__decontam_identify_batches, qiime2__quality_control__decontam_remove, qiime2__quality_control__decontam_score_viz, qiime2__quality_control__evaluate_composition, qiime2__quality_control__evaluate_seqs, qiime2__quality_control__evaluate_taxonomy, qiime2__quality_control__exclude_seqs, qiime2__quality_control__filter_reads</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/qiime2/q2-quality-control</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__quality_control</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__quality_control</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>10</td>
      <td>10</td>
      <td>10</td>
      <td>10</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__quality_filter</td>
      <td>qiime2__quality_filter__q_score</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/qiime2/q2-quality-filter</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__quality_filter</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__quality_filter</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>29</td>
      <td>29</td>
      <td>14</td>
      <td>14</td>
      <td>16</td>
      <td>16</td>
      <td>6</td>
      <td>6</td>
      <td>37</td>
      <td>37</td>
      <td>2</td>
      <td>2</td>
      <td>82</td>
      <td>82</td>
      <td>22</td>
      <td>22</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__rescript</td>
      <td>qiime2__rescript__cull_seqs, qiime2__rescript__degap_seqs, qiime2__rescript__dereplicate, qiime2__rescript__edit_taxonomy, qiime2__rescript__evaluate_classifications, qiime2__rescript__evaluate_cross_validate, qiime2__rescript__evaluate_fit_classifier, qiime2__rescript__evaluate_seqs, qiime2__rescript__evaluate_taxonomy, qiime2__rescript__extract_seq_segments, qiime2__rescript__filter_seqs_length, qiime2__rescript__filter_seqs_length_by_taxon, qiime2__rescript__filter_taxa, qiime2__rescript__get_gtdb_data, qiime2__rescript__get_ncbi_data, qiime2__rescript__get_ncbi_data_protein, qiime2__rescript__get_ncbi_genomes, qiime2__rescript__get_silva_data, qiime2__rescript__get_unite_data, qiime2__rescript__merge_taxa, qiime2__rescript__orient_seqs, qiime2__rescript__parse_silva_taxonomy, qiime2__rescript__reverse_transcribe, qiime2__rescript__subsample_fasta, qiime2__rescript__trim_alignment</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/nbokulich/RESCRIPt</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2024-04-25</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__rescript</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__rescript</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__sample_classifier</td>
      <td>qiime2__sample_classifier__classify_samples, qiime2__sample_classifier__classify_samples_from_dist, qiime2__sample_classifier__classify_samples_ncv, qiime2__sample_classifier__confusion_matrix, qiime2__sample_classifier__fit_classifier, qiime2__sample_classifier__fit_regressor, qiime2__sample_classifier__heatmap, qiime2__sample_classifier__metatable, qiime2__sample_classifier__predict_classification, qiime2__sample_classifier__predict_regression, qiime2__sample_classifier__regress_samples, qiime2__sample_classifier__regress_samples_ncv, qiime2__sample_classifier__scatterplot, qiime2__sample_classifier__split_table, qiime2__sample_classifier__summarize</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/qiime2/q2-sample-classifier</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__sample_classifier</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__sample_classifier</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>15</td>
      <td>15</td>
      <td>15</td>
      <td>15</td>
      <td>15</td>
      <td>15</td>
      <td>15</td>
      <td>15</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>15</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>10</td>
      <td>3</td>
      <td>3</td>
      <td>14</td>
      <td>14</td>
      <td>10</td>
      <td>10</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>29</td>
      <td>29</td>
      <td>18</td>
      <td>18</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__taxa</td>
      <td>qiime2__taxa__barplot, qiime2__taxa__collapse, qiime2__taxa__filter_seqs, qiime2__taxa__filter_table</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/qiime2/q2-taxa</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__taxa</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__taxa</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>664</td>
      <td>664</td>
      <td>145</td>
      <td>145</td>
      <td>113</td>
      <td>113</td>
      <td>43</td>
      <td>43</td>
      <td>255</td>
      <td>255</td>
      <td>50</td>
      <td>50</td>
      <td>1032</td>
      <td>1032</td>
      <td>238</td>
      <td>238</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__vsearch</td>
      <td>qiime2__vsearch__cluster_features_closed_reference, qiime2__vsearch__cluster_features_de_novo, qiime2__vsearch__cluster_features_open_reference, qiime2__vsearch__dereplicate_sequences, qiime2__vsearch__fastq_stats, qiime2__vsearch__merge_pairs, qiime2__vsearch__uchime_denovo, qiime2__vsearch__uchime_ref</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/qiime2/q2-vsearch</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__vsearch</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__vsearch</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>7</td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>84</td>
      <td>84</td>
      <td>52</td>
      <td>52</td>
      <td>25</td>
      <td>25</td>
      <td>16</td>
      <td>16</td>
      <td>9</td>
      <td>9</td>
      <td>5</td>
      <td>5</td>
      <td>118</td>
      <td>118</td>
      <td>73</td>
      <td>73</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2_core</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics, Metagenomics, Sequence Analysis</td>
      <td></td>
      <td>2022-08-29</td>
      <td>q2d2</td>
      <td></td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tool_collections/suite_qiime2_core</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2_core__tools</td>
      <td>qiime2_core__tools__export, qiime2_core__tools__import, qiime2_core__tools__import_fastq</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://qiime2.org</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2_core__tools</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2_core__tools</td>
      <td>2024.5.0+dist.he540b0b0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>14941</td>
      <td>14941</td>
      <td>383</td>
      <td>383</td>
      <td>25941</td>
      <td>25941</td>
      <td>255</td>
      <td>255</td>
      <td>2856</td>
      <td>2856</td>
      <td>112</td>
      <td>112</td>
      <td>43738</td>
      <td>43738</td>
      <td>750</td>
      <td>750</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>t_test_two_samples</td>
      <td>t_test_two_samples</td>
      <td>T Test for Two Samples</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>t_test_two_samples</td>
      <td>2013-09-25</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/t_test_two_samples</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/t_test_two_samples</td>
      <td>1.0.1</td>
      <td>R</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>141</td>
      <td>215</td>
      <td>20</td>
      <td>31</td>
      <td>520</td>
      <td>1166</td>
      <td>79</td>
      <td>274</td>
      <td>1025</td>
      <td>1118</td>
      <td>11</td>
      <td>12</td>
      <td>1686</td>
      <td>2499</td>
      <td>110</td>
      <td>317</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>tag_pileup_frequency</td>
      <td>tag_pileup_frequency</td>
      <td>Contains a tool that generates a frequency pileup of the 5' ends of aligned reads in a BAM filerelative to reference points in a BED file.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/seqcode/cegr-tools/tree/master/src/org/seqcode/cegrtools/tagpileup</td>
      <td>Statistics, SAM, Genomic Interval Operations</td>
      <td>tag_pileup_frequency</td>
      <td>2016-05-05</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/tag_pileup_frequency</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/tag_pileup_frequency</td>
      <td>1.0.2</td>
      <td>openjdk</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>158</td>
      <td>164</td>
      <td>15</td>
      <td>16</td>
      <td>722</td>
      <td>865</td>
      <td>40</td>
      <td>73</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>880</td>
      <td>1029</td>
      <td>55</td>
      <td>89</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>topologyeditors</td>
      <td>gromacs_modify_topology, gromacs_extract_topology</td>
      <td>Set of python scripts and associated tool files that can be used to modify topology files.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/buildtools/topologyeditors</td>
      <td>Molecular Dynamics, Computational chemistry</td>
      <td>topologyeditors</td>
      <td>2021-12-23</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/buildtools/topologyeditors</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/buildtools/topologyeditors</td>
      <td>0</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>57</td>
      <td>57</td>
      <td>10</td>
      <td>10</td>
      <td>20</td>
      <td>20</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>77</td>
      <td>77</td>
      <td>18</td>
      <td>18</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>tracegroomer</td>
      <td>tracegroomer</td>
      <td>TraceGroomer is a solution for formatting and normalising Tracer metabolomics given file(s), to produce the .csv files which are ready for DIMet tool.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/cbib/TraceGroomer</td>
      <td>Metabolomics</td>
      <td></td>
      <td>2024-03-28</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/Tracegroomer</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/tracegroomer</td>
      <td>0.1.4</td>
      <td>tracegroomer</td>
      <td>0.1.4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>1</td>
      <td>1</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>vmd</td>
      <td></td>
      <td>vmd is a package for  visualizing and analyzing trajectories from molecular dynamics (MD) simulations</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.ks.uiuc.edu/Research/vmd/</td>
      <td>Computational chemistry</td>
      <td>vmd</td>
      <td>2019-10-24</td>
      <td>chemteam</td>
      <td>https://github.com/thatchristoph/vmd-cvs-github/tree/master/vmd</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/vmd</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>volcanoplot</td>
      <td>volcanoplot</td>
      <td>Tool to create a Volcano Plot</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://ggplot2.tidyverse.org/</td>
      <td>Visualization, Transcriptomics, Statistics</td>
      <td>volcanoplot</td>
      <td>2018-10-14</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/volcanoplot</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/volcanoplot</td>
      <td>0.0.6</td>
      <td>r-ggplot2</td>
      <td>2.2.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>34917</td>
      <td>36852</td>
      <td>3670</td>
      <td>3786</td>
      <td>54931</td>
      <td>54931</td>
      <td>5597</td>
      <td>5597</td>
      <td>7183</td>
      <td>7183</td>
      <td>905</td>
      <td>905</td>
      <td>97031</td>
      <td>98966</td>
      <td>10172</td>
      <td>10288</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>w4mclassfilter</td>
      <td>w4mclassfilter</td>
      <td>Filter W4M data by values or metadata</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/HegemanLab/w4mclassfilter_galaxy_wrapper</td>
      <td>Metabolomics</td>
      <td>w4mclassfilter</td>
      <td>2019-09-26</td>
      <td>eschen42</td>
      <td>https://github.com/HegemanLab/w4mclassfilter_galaxy_wrapper/tree/master</td>
      <td>https://github.com/HegemanLab/w4mclassfilter_galaxy_wrapper/tree/master/tools/w4mclassfilter</td>
      <td>0.98.19</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2370</td>
      <td>2836</td>
      <td>9</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1482</td>
      <td>1509</td>
      <td>4</td>
      <td>4</td>
      <td>3852</td>
      <td>4345</td>
      <td>13</td>
      <td>14</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>w4mconcatenate</td>
      <td>W4Mconcatenate</td>
      <td>[W4M][Utils] concatenate two Metadata tables</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://workflow4metabolomics.org</td>
      <td>Metabolomics</td>
      <td>w4mconcatenate</td>
      <td>2024-07-10</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/w4mconcatenate</td>
      <td>1.0.0+galaxy0</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>w4mcorcov</td>
      <td>w4mcorcov</td>
      <td>OPLS-DA Contrasts of Univariate Results</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/HegemanLab/w4mcorcov_galaxy_wrapper</td>
      <td>Metabolomics</td>
      <td>w4mcorcov</td>
      <td>2018-09-05</td>
      <td>eschen42</td>
      <td>https://github.com/HegemanLab/w4mcorcov_galaxy_wrapper/tree/master</td>
      <td>https://github.com/HegemanLab/w4mcorcov_galaxy_wrapper/tree/master/tools/w4mcorcov</td>
      <td>0.98.18</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>238</td>
      <td>375</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>10</td>
      <td>2</td>
      <td>2</td>
      <td>248</td>
      <td>385</td>
      <td>12</td>
      <td>12</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>w4mjoinpn</td>
      <td>w4mjoinpn</td>
      <td>Join positive- and negative-mode W4M datasets</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/HegemanLab/w4mjoinpn_galaxy_wrapper</td>
      <td>Metabolomics</td>
      <td>w4mjoinpn</td>
      <td>2018-08-06</td>
      <td>eschen42</td>
      <td>https://github.com/HegemanLab/w4mjoinpn_galaxy_wrapper/tree/master</td>
      <td>https://github.com/HegemanLab/w4mjoinpn_galaxy_wrapper/tree/master/tools/w4mjoinpn</td>
      <td>0.98.2</td>
      <td>coreutils</td>
      <td>8.25</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>150</td>
      <td>171</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>2</td>
      <td>2</td>
      <td>157</td>
      <td>178</td>
      <td>9</td>
      <td>9</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>waveica</td>
      <td>waveica</td>
      <td>Removal of batch effects for large-scale untargeted metabolomics data based on wavelet analysis.</td>
      <td>waveica</td>
      <td>waveica</td>
      <td></td>
      <td>WaveICA</td>
      <td>Removal of batch effects for large-scale untargeted metabolomics data based on wavelet transform.</td>
      <td>Standardisation and normalisation</td>
      <td>Metabolomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/RECETOX/WaveICA</td>
      <td>Metabolomics</td>
      <td></td>
      <td>2022-09-19</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/waveica</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/waveica</td>
      <td>0.2.0</td>
      <td>r-recetox-waveica</td>
      <td>0.2.0</td>
      <td>Standardisation and normalisation</td>
      <td>Metabolomics</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>39</td>
      <td>39</td>
      <td>6</td>
      <td>6</td>
      <td>4</td>
      <td>4</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>43</td>
      <td>43</td>
      <td>9</td>
      <td>9</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>xcms</td>
      <td>abims_xcms_fillPeaks, abims_xcms_group, abims_xcms_refine, abims_xcms_retcor, abims_xcms_summary, abims_xcms_xcmsSet, msnbase_readmsdata, xcms_export_samplemetadata, xcms_merge, xcms_plot_chromatogram, xcms_plot_eic, xcms_plot_raw</td>
      <td></td>
      <td>XCMS</td>
      <td>XCMS</td>
      <td></td>
      <td>xcms</td>
      <td>Framework for processing and visualization of chromatographically separated and single-spectra mass spectral data.  The packages enables imports from AIA/ANDI NetCDF, mzXML, mzData and mzML files and preprocesses data for high-throughput, untargeted analyte profiling.</td>
      <td>Mass spectrum visualisation, Label-free quantification, Validation, Filtering, Chromatographic alignment, Peak detection, Chromatogram visualisation</td>
      <td>Biological imaging, Data visualisation, Metabolomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/sneumann/xcms</td>
      <td>Metabolomics</td>
      <td>xcms</td>
      <td>2023-05-24</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/xcms</td>
      <td>4.0.0</td>
      <td>bioconductor-xcms</td>
      <td>4.0.0</td>
      <td>Mass spectrum visualisation, Label-free quantification, Validation, Filtering, Chromatographic alignment, Peak detection, Chromatogram visualisation</td>
      <td>Data visualisation, Metabolomics</td>
      <td>9</td>
      <td>11</td>
      <td>12</td>
      <td>12</td>
      <td>9</td>
      <td>11</td>
      <td>12</td>
      <td>12</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>10</td>
      <td>9</td>
      <td>0</td>
      <td>31909</td>
      <td>35790</td>
      <td>1357</td>
      <td>1384</td>
      <td>7099</td>
      <td>7099</td>
      <td>413</td>
      <td>413</td>
      <td>5333</td>
      <td>5357</td>
      <td>330</td>
      <td>330</td>
      <td>44341</td>
      <td>48246</td>
      <td>2100</td>
      <td>2127</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>xtb</td>
      <td>xtb_molecular_optimization</td>
      <td>Performs semiempirical molecular optimization.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/grimme-lab/xtb</td>
      <td>Metabolomics</td>
      <td>xtb_molecular_optimization</td>
      <td>2023-09-06</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/xtb</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/xtb</td>
      <td>6.6.1</td>
      <td>xtb</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>143</td>
      <td>143</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>143</td>
      <td>143</td>
      <td>4</td>
      <td>4</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>xy_plot</td>
      <td>XY_Plot_1</td>
      <td>Plotting tool for multiple series and graph types</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Graphics, Statistics</td>
      <td>xy_plot</td>
      <td>2013-09-25</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/xy_plot</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/xy_plot</td>
      <td>1.0.2</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>4490</td>
      <td>6169</td>
      <td>768</td>
      <td>889</td>
      <td>6120</td>
      <td>15516</td>
      <td>1921</td>
      <td>3653</td>
      <td>2033</td>
      <td>2212</td>
      <td>556</td>
      <td>590</td>
      <td>12643</td>
      <td>23897</td>
      <td>3245</td>
      <td>5132</td>
      <td></td>
    </tr>
  </tbody>
</table>

</div>