<script src="https://code.jquery.com/jquery-3.7.0.js"></script>
<script src="https://cdn.datatables.net/1.13.6/js/jquery.dataTables.min.js"></script>
<script src="https://cdn.datatables.net/searchbuilder/1.6.0/js/dataTables.searchBuilder.min.js"></script>
<script src="https://cdn.datatables.net/datetime/1.5.1/js/dataTables.dateTime.min.js"></script>

<script src="https://cdn.datatables.net/responsive/2.5.0/js/dataTables.responsive.min.js"></script>

<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/responsive/2.2.3/css/responsive.dataTables.min.css">

<link rel="stylesheet" href="https://cdn.datatables.net/1.13.6/css/jquery.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/searchbuilder/1.6.0/css/searchBuilder.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/datetime/1.5.1/css/dataTables.dateTime.min.css">

<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" />
<script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>

<style type="text/css">
  .tooltip {
    font-family: Georgia;
    font-size: 18px;
  }
  .tooltip .tooltip-inner {
    background-color: #00FFFF;
    color: #000000;
    min-width: 400px;
  }
  </style>

<script>
    $(document).ready(function() {
    $('#dataframe').DataTable( {
        "sScrollX": "100%",
        "dom": "Qlfrtip",

        responsive: {
            details: {
                type: 'column'
            }
        },
        "bAutoWidth": false,
        columnDefs: [ {
            className: 'control',
            orderable: false,
            targets:   [ 0 ],
            width: 100,
        },
        { type: 'natural', targets: 1 } // allow normal sorting
        ],

    });
});
</script>

<div class="display nowrap" style="width:80%" width="100%">

<table class="dataframe display nowrap" id="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th>Expand</th>
      <th>Galaxy wrapper id</th>
      <th>Galaxy tool ids</th>
      <th>Description</th>
      <th>bio.tool id</th>
      <th>bio.tool ids</th>
      <th>biii</th>
      <th>bio.tool name</th>
      <th>bio.tool description</th>
      <th>EDAM operation</th>
      <th>EDAM topic</th>
      <th>Status</th>
      <th>Source</th>
      <th>ToolShed categories</th>
      <th>ToolShed id</th>
      <th>Date of first commit of the suite</th>
      <th>Galaxy wrapper owner</th>
      <th>Galaxy wrapper source</th>
      <th>Galaxy wrapper parsed folder</th>
      <th>Galaxy wrapper version</th>
      <th>Conda id</th>
      <th>Conda version</th>
      <th>EDAM operation (no superclasses)</th>
      <th>EDAM topic (no superclasses)</th>
      <th>Available on UseGalaxy.org (Main)</th>
      <th>Available on UseGalaxy.org.au</th>
      <th>Available on UseGalaxy.eu</th>
      <th>Available on UseGalaxy.fr</th>
      <th>Tools available on UseGalaxy.org (Main)</th>
      <th>Tools available on UseGalaxy.org.au</th>
      <th>Tools available on UseGalaxy.eu</th>
      <th>Tools available on UseGalaxy.fr</th>
      <th>Tools available on APOSTL</th>
      <th>Tools available on ARGs-OAP</th>
      <th>Tools available on CIRM-CFBP</th>
      <th>Tools available on ChemFlow</th>
      <th>Tools available on Coloc-stats</th>
      <th>Tools available on CoralSNP</th>
      <th>Tools available on CropGalaxy</th>
      <th>Tools available on Dintor</th>
      <th>Tools available on GASLINI</th>
      <th>Tools available on Galaxy@AuBi</th>
      <th>Tools available on Galaxy@Pasteur</th>
      <th>Tools available on GalaxyTrakr</th>
      <th>Tools available on Genomic Hyperbrowser</th>
      <th>Tools available on GigaGalaxy</th>
      <th>Tools available on HyPhy HIV NGS Tools</th>
      <th>Tools available on IPK Galaxy Blast Suite</th>
      <th>Tools available on ImmPort Galaxy</th>
      <th>Tools available on InteractoMIX</th>
      <th>Tools available on MISSISSIPPI</th>
      <th>Tools available on Mandoiu Lab</th>
      <th>Tools available on MiModD NacreousMap</th>
      <th>Tools available on Oqtans</th>
      <th>Tools available on Palfinder</th>
      <th>Tools available on PepSimili</th>
      <th>Tools available on UseGalaxy.be</th>
      <th>Tools available on UseGalaxy.cz</th>
      <th>Tools available on UseGalaxy.no</th>
      <th>Tools available on Viral Variant Visualizer (VVV)</th>
      <th>No. of tool users (5 years) (usegalaxy.eu)</th>
      <th>No. of tool users (all time) (usegalaxy.eu)</th>
      <th>Tool usage (5 years) (usegalaxy.eu)</th>
      <th>Tool usage (all time) (usegalaxy.eu)</th>
      <th>No. of tool users (5 years) (usegalaxy.org)</th>
      <th>No. of tool users (all time) (usegalaxy.org)</th>
      <th>Tool usage (5 years) (usegalaxy.org)</th>
      <th>Tool usage (all time) (usegalaxy.org)</th>
      <th>No. of tool users (5 years) (usegalaxy.org.au)</th>
      <th>No. of tool users (all time) (usegalaxy.org.au)</th>
      <th>Tool usage (5 years) (usegalaxy.org.au)</th>
      <th>Tool usage (all time) (usegalaxy.org.au)</th>
      <th>No. of tool users (5 years) - all main servers</th>
      <th>No. of tool users (all time) - all main servers</th>
      <th>Tool usage (5 years) - all main servers</th>
      <th>Tool usage (all time) - all main servers</th>
      <th>Deprecated</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td></td>
      <td>ThermoRawFileParser</td>
      <td>thermo_raw_file_converter</td>
      <td>Thermo RAW file converter</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/compomics/ThermoRawFileParser</td>
      <td>Proteomics</td>
      <td>thermo_raw_file_converter</td>
      <td>2019-08-06</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/ThermoRawFileParser</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/ThermoRawFileParser</td>
      <td>1.3.4</td>
      <td>thermorawfileparser</td>
      <td>1.4.5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2467</td>
      <td>2959</td>
      <td>88</td>
      <td>124</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1097</td>
      <td>1097</td>
      <td>18</td>
      <td>18</td>
      <td>3564</td>
      <td>4056</td>
      <td>106</td>
      <td>142</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>b2btools</td>
      <td>b2btools_single_sequence</td>
      <td>This software suite provides structural predictions for protein sequences made by Bio2Byte group.About Bio2Byte: We investigate how the dynamics, conformational states, and available experimental data of proteins relate to their amino acid sequence.Underlying physical and chemical principles are computationally unraveled through data integration, analysis, and machine learning, so connecting themto biological events and improving our understanding of the way proteins work.</td>
      <td>b2btools</td>
      <td>b2btools</td>
      <td></td>
      <td>b2bTools</td>
      <td>The bio2byte tools server (b2btools) offers the following single protein sequence based predictions:- Backbone and sidechain dynamics (DynaMine)- Helix, sheet, coil and polyproline-II propensity- Early folding propensity (EFoldMine)- Disorder (DisoMine)- Beta-sheet aggregation (Agmata)In addition, multiple sequence alignments (MSAs) can be uploaded to scan the 'biophysical limits' of a protein family as defined in the MSA</td>
      <td>Protein disorder prediction, Protein secondary structure prediction, Protein feature detection</td>
      <td></td>
      <td>To update</td>
      <td>https://bio2byte.be</td>
      <td>Computational chemistry, Molecular Dynamics, Proteomics, Sequence Analysis, Synthetic Biology</td>
      <td></td>
      <td>2022-08-09</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ucsc-tools/b2tools</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/b2btools</td>
      <td>3.0.5+galaxy0</td>
      <td>b2btools</td>
      <td>3.0.7</td>
      <td>Protein disorder prediction, Protein secondary structure prediction</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>369</td>
      <td>369</td>
      <td>19</td>
      <td>19</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>369</td>
      <td>369</td>
      <td>19</td>
      <td>19</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>bed_to_protein_map</td>
      <td>bed_to_protein_map</td>
      <td>Converts a BED file to a tabular list of exon locations</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>bed_to_protein_map</td>
      <td>2018-01-04</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/bed_to_protein_map</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/bed_to_protein_map</td>
      <td>0.2.0</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>410</td>
      <td>419</td>
      <td>134</td>
      <td>137</td>
      <td>363</td>
      <td>363</td>
      <td>99</td>
      <td>99</td>
      <td>158</td>
      <td>158</td>
      <td>13</td>
      <td>13</td>
      <td>931</td>
      <td>940</td>
      <td>246</td>
      <td>249</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>bumbershoot</td>
      <td>idpqonvertEmbedder, idpassemble, idpqonvert, idpquery, myrimatch</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://proteowizard.sourceforge.net/</td>
      <td>Proteomics</td>
      <td></td>
      <td>2017-06-08</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tools/bumbershoot</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/bumbershoot</td>
      <td>3.0.21142</td>
      <td>bumbershoot</td>
      <td>3_0_21142_0e4f4a4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>1186</td>
      <td>1356</td>
      <td>21</td>
      <td>32</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1186</td>
      <td>1356</td>
      <td>21</td>
      <td>32</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>calisp</td>
      <td>calisp</td>
      <td>Calgary approach to isotopes in proteomics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/kinestetika/Calisp/</td>
      <td>Proteomics</td>
      <td>calisp</td>
      <td>2023-06-01</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tools/calisp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/calisp</td>
      <td>3.0.13</td>
      <td>calisp</td>
      <td>3.0.13</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>cardinal</td>
      <td>cardinal_classification, cardinal_colocalization, cardinal_combine, cardinal_data_exporter, cardinal_filtering, cardinal_mz_images, cardinal_preprocessing, cardinal_quality_report, cardinal_segmentations, cardinal_single_ion_segmentation, cardinal_spectra_plots</td>
      <td>Statistical and computational tools for analyzing mass spectrometry imaging datasets</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://cardinalmsi.org</td>
      <td>Proteomics, Metabolomics</td>
      <td></td>
      <td>2020-12-23</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/cardinal</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/cardinal</td>
      <td>3.4.3</td>
      <td>bioconductor-cardinal</td>
      <td>3.4.3</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>9</td>
      <td>11</td>
      <td>11</td>
      <td>0</td>
      <td>9</td>
      <td>11</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>11</td>
      <td>9</td>
      <td>0</td>
      <td>41875</td>
      <td>50702</td>
      <td>663</td>
      <td>713</td>
      <td>12</td>
      <td>12</td>
      <td>1</td>
      <td>1</td>
      <td>7512</td>
      <td>7512</td>
      <td>231</td>
      <td>231</td>
      <td>49399</td>
      <td>58226</td>
      <td>895</td>
      <td>945</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>cite_seq_count</td>
      <td>cite_seq_count</td>
      <td>Count CMO/HTO</td>
      <td>CITE-seq-Count</td>
      <td>CITE-seq-Count</td>
      <td></td>
      <td>CITE-seq-Count</td>
      <td>Tool for counting antibody TAGS from a CITE-seq and/or cell hashing experiment.</td>
      <td>RNA-Seq quantification</td>
      <td>Transcriptomics, Immunoproteins and antigens</td>
      <td>To update</td>
      <td>https://github.com/Hoohm/CITE-seq-Count</td>
      <td>Single Cell, Transcriptomics, Proteomics</td>
      <td>cite_seq_count</td>
      <td>2023-01-18</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cite_seq_count</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cite_seq_count</td>
      <td>1.4.4</td>
      <td>cite-seq-count</td>
      <td>1.4.5</td>
      <td>RNA-Seq quantification</td>
      <td>Transcriptomics, Immunoproteins and antigens</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>21</td>
      <td>21</td>
      <td>4</td>
      <td>4</td>
      <td>9</td>
      <td>9</td>
      <td>5</td>
      <td>5</td>
      <td>18</td>
      <td>18</td>
      <td>1</td>
      <td>1</td>
      <td>48</td>
      <td>48</td>
      <td>10</td>
      <td>10</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>colabfold</td>
      <td>colabfold_alphafold, colabfold_msa</td>
      <td>Protein prediction based on AlphaFold2</td>
      <td>Colabfold</td>
      <td>Colabfold</td>
      <td></td>
      <td>ColabFold</td>
      <td>ColabFold databases are MMseqs2 expandable profile databases to generate diverse multiple sequence alignments to predict protein structures.</td>
      <td>Database search, Protein structure prediction, Fold recognition</td>
      <td>Protein folds and structural domains, Protein folding, stability and design, Structure prediction, Sequence sites, features and motifs, Metagenomics</td>
      <td>To update</td>
      <td>https://github.com/sokrypton/ColabFold</td>
      <td>Proteomics, Graphics</td>
      <td>colabfold</td>
      <td>2024-03-24</td>
      <td>iuc</td>
      <td>https://github.com/sokrypton/ColabFold</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/colabfold</td>
      <td>1.5.5</td>
      <td></td>
      <td></td>
      <td>Database search, Protein structure prediction, Fold recognition</td>
      <td>Protein folds and structural domains, Protein folding, stability and design, Structure prediction, Sequence sites, features and motifs, Metagenomics</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6771</td>
      <td>6771</td>
      <td>50</td>
      <td>50</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6771</td>
      <td>6771</td>
      <td>50</td>
      <td>50</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>custom_pro_db</td>
      <td>custom_pro_db</td>
      <td>CustomProDB</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/customProDB.html</td>
      <td>Proteomics</td>
      <td>custom_pro_db</td>
      <td>2017-10-06</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tools/bumbershoot/custom_pro_db</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/probam_suite/custom_pro_db</td>
      <td>1.22.0</td>
      <td>bioconductor-rgalaxy</td>
      <td>1.37.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1729</td>
      <td>1744</td>
      <td>144</td>
      <td>146</td>
      <td>450</td>
      <td>450</td>
      <td>72</td>
      <td>72</td>
      <td>361</td>
      <td>361</td>
      <td>12</td>
      <td>12</td>
      <td>2540</td>
      <td>2555</td>
      <td>228</td>
      <td>230</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>custom_pro_db_annotation_data_manager</td>
      <td></td>
      <td>CustomProDB Annotation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/customProDB.html</td>
      <td>Proteomics</td>
      <td>custom_pro_db_annotation_data_manager</td>
      <td>2017-10-06</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tools/bumbershoot/custom_pro_db</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/probam_suite/custom_pro_db_annotation_data_manager</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>data_manager_eggnog_mapper</td>
      <td></td>
      <td>downloads eggnog data for eggnog-mapper</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>data_manager_eggnog_mapper</td>
      <td>2019-11-11</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/eggnog_mapper/eggnog_mapper_data_manager</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/eggnog_mapper/data_manager_eggnog_mapper</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>data_manager_eggnog_mapper_abspath</td>
      <td></td>
      <td>download eggnog data for eggnog-mapper</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>data_manager_eggnog_mapper_abspath</td>
      <td>2019-11-11</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/eggnog_mapper/data_manager_eggnog_mapper_abspath</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/eggnog_mapper/data_manager_eggnog_mapper_abspath</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>dbbuilder</td>
      <td>dbbuilder</td>
      <td>Protein Database Downloader</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/dbbuilder</td>
      <td>Proteomics</td>
      <td>dbbuilder</td>
      <td>2015-01-24</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/dbbuilder</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/dbbuilder</td>
      <td>0.3.4</td>
      <td>wget</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>4813</td>
      <td>5555</td>
      <td>687</td>
      <td>805</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>961</td>
      <td>961</td>
      <td>87</td>
      <td>87</td>
      <td>5774</td>
      <td>6516</td>
      <td>774</td>
      <td>892</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>decoyfasta</td>
      <td></td>
      <td>Galaxy tool wrapper for the transproteomic pipeline decoyFASTA tool.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>decoyfasta</td>
      <td>2015-01-24</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/decoyfasta</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/decoyfasta</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>dia_umpire</td>
      <td>dia_umpire_se</td>
      <td>DIA-Umpire analysis for data independent acquisition (DIA) mass spectrometry-based proteomics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://diaumpire.sourceforge.net/</td>
      <td>Proteomics</td>
      <td>dia_umpire</td>
      <td>2015-08-26</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/dia_umpire</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/dia_umpire</td>
      <td>2.1.3</td>
      <td>dia_umpire</td>
      <td>2.1.6</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>28</td>
      <td>33</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>134</td>
      <td>134</td>
      <td>4</td>
      <td>4</td>
      <td>162</td>
      <td>167</td>
      <td>8</td>
      <td>8</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>dialignr</td>
      <td>dialignr</td>
      <td>DIAlignR is an R package for retention time alignment of targeted mass spectrometric data, including DIA and SWATH-MS data. This tool works with MS2 chromatograms directly and uses dynamic programming for alignment of raw chromatographic traces. DIAlignR uses a hybrid approach of global (feature-based) and local (raw data-based) alignment to establish correspondence between peaks.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/shubham1637/DIAlignR</td>
      <td>Proteomics</td>
      <td>dialignr</td>
      <td>2020-12-23</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/dialignr</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/dialignr</td>
      <td>1.2.0</td>
      <td>bioconductor-dialignr</td>
      <td>2.10.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>40</td>
      <td>40</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>40</td>
      <td>40</td>
      <td>2</td>
      <td>2</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>diann</td>
      <td>diann</td>
      <td>DiaNN (DIA-based Neural Networks) is a software for DIA/SWATH data processing.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/vdemichev/DiaNN</td>
      <td>Proteomics</td>
      <td>diann</td>
      <td>2023-06-26</td>
      <td>galaxyp</td>
      <td>https://github.com/vdemichev/DiaNN</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/diann</td>
      <td>1.8.1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>72</td>
      <td>72</td>
      <td>14</td>
      <td>14</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>171</td>
      <td>171</td>
      <td>14</td>
      <td>14</td>
      <td>243</td>
      <td>243</td>
      <td>28</td>
      <td>28</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>diapysef</td>
      <td>diapysef</td>
      <td>diapysef is a convenience package for working with DIA-PASEF data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://pypi.org/project/diapysef/</td>
      <td>Proteomics</td>
      <td>diapysef</td>
      <td>2020-02-27</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/diapysef</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/diapysef</td>
      <td>0.3.5.0</td>
      <td>diapysef</td>
      <td>1.0.10</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>262</td>
      <td>262</td>
      <td>38</td>
      <td>38</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>256</td>
      <td>256</td>
      <td>6</td>
      <td>6</td>
      <td>518</td>
      <td>518</td>
      <td>44</td>
      <td>44</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>diffacto</td>
      <td>diffacto</td>
      <td>Diffacto comparative protein abundance estimation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/statisticalbiotechnology/diffacto</td>
      <td>Proteomics</td>
      <td>diffacto</td>
      <td>2021-06-20</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/diffacto</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/diffacto</td>
      <td>1.0.6</td>
      <td>diffacto</td>
      <td>1.0.7</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>11</td>
      <td>11</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>11</td>
      <td>11</td>
      <td>7</td>
      <td>7</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>dotknot</td>
      <td>dotknot</td>
      <td>DotKnot is a heuristic method for pseudoknot prediction in a given RNA sequence</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://dotknot.csse.uwa.edu.au/</td>
      <td>RNA, Proteomics</td>
      <td>dotknot</td>
      <td>2015-09-17</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna/dotknot</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/dotknot</td>
      <td>1.3.1</td>
      <td>vienna_rna</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>74</td>
      <td>88</td>
      <td>12</td>
      <td>15</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>74</td>
      <td>88</td>
      <td>12</td>
      <td>15</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>eggnog_mapper</td>
      <td>eggnog_mapper, eggnog_mapper_annotate, eggnog_mapper_search</td>
      <td>eggnog-mapper fast functional annotation of novel sequences</td>
      <td>eggnog-mapper-v2</td>
      <td>eggnog-mapper-v2</td>
      <td></td>
      <td>eggNOG-mapper v2</td>
      <td>EggNOG-mapper is a tool for fast functional annotation of novel sequences. It uses precomputed orthologous groups and phylogenies from the eggNOG database (http://eggnog5.embl.de) to transfer functional information from fine-grained orthologs only.</td>
      <td>Homology-based gene prediction, Genome annotation, Fold recognition, Information extraction, Query and retrieval</td>
      <td>Metagenomics, Phylogeny, Transcriptomics, Workflows, Sequence analysis</td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>eggnog_mapper</td>
      <td>2019-11-11</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/eggnog_mapper/eggnog_mapper</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/eggnog_mapper/eggnog_mapper</td>
      <td>2.1.8</td>
      <td>eggnog-mapper</td>
      <td>2.1.12</td>
      <td>Homology-based gene prediction, Genome annotation, Fold recognition, Information extraction, Query and retrieval</td>
      <td>Metagenomics, Phylogeny, Transcriptomics, Workflows, Sequence analysis</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>1</td>
      <td>3</td>
      <td>1</td>
      <td>0</td>
      <td>34432</td>
      <td>34437</td>
      <td>1149</td>
      <td>1152</td>
      <td>42</td>
      <td>42</td>
      <td>2</td>
      <td>2</td>
      <td>21027</td>
      <td>21027</td>
      <td>413</td>
      <td>413</td>
      <td>55501</td>
      <td>55506</td>
      <td>1564</td>
      <td>1567</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>encyclopedia</td>
      <td>encyclopedia_encyclopedia, encyclopedia_fasta_to_prosit_csv, encyclopedia_library_to_blib, encyclopedia_prosit_csv_to_library, encyclopedia_quantify, encyclopedia_searchtolib, encyclopedia_walnut</td>
      <td>Mass Spec Data-Independent Acquisition (DIA) MS/MS analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/searleb/encyclopedia/wiki/Home</td>
      <td>Proteomics</td>
      <td>encyclopedia</td>
      <td>2020-09-11</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/encyclopedia/tools/encyclopedia</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/encyclopedia</td>
      <td>1.12.34</td>
      <td>encyclopedia</td>
      <td>2.12.30</td>
      <td></td>
      <td></td>
      <td>2</td>
      <td>4</td>
      <td>7</td>
      <td>0</td>
      <td>2</td>
      <td>4</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>738</td>
      <td>738</td>
      <td>98</td>
      <td>98</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>42</td>
      <td>42</td>
      <td>4</td>
      <td>4</td>
      <td>781</td>
      <td>781</td>
      <td>103</td>
      <td>103</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>fastg2protlib</td>
      <td>fastg2protlib-peptides, fastg2protlib-validate</td>
      <td>Generate FASTA from FASTG</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproteomics/fastg2protlib.git</td>
      <td>Proteomics</td>
      <td>fastg2protlib</td>
      <td>2020-07-23</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/fastg2protlib</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/fastg2protlib</td>
      <td>1.0.2</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>29</td>
      <td>29</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>29</td>
      <td>29</td>
      <td>8</td>
      <td>8</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>feature_alignment</td>
      <td>feature_alignment</td>
      <td>TRIC integrates information from all available runs via a graph-based alignment strategy</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td></td>
      <td>Proteomics</td>
      <td>feature_alignment</td>
      <td>2021-11-22</td>
      <td>galaxyp</td>
      <td>https://github.com/msproteomicstools/msproteomicstools/blob/master/TRIC-README.md</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/feature_alignment</td>
      <td>0.11.0</td>
      <td>msproteomicstools</td>
      <td>0.11.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>18</td>
      <td>18</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>18</td>
      <td>18</td>
      <td>1</td>
      <td>1</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>filter_by_fasta_ids</td>
      <td>filter_by_fasta_ids</td>
      <td>Filter FASTA on the headers and/or the sequences</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Fasta Manipulation, Proteomics</td>
      <td>filter_by_fasta_ids</td>
      <td>2015-01-24</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/filter_by_fasta_ids</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/filter_by_fasta_ids</td>
      <td>2.3</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>26717</td>
      <td>29932</td>
      <td>1026</td>
      <td>1072</td>
      <td>42990</td>
      <td>42990</td>
      <td>1176</td>
      <td>1176</td>
      <td>9619</td>
      <td>9619</td>
      <td>230</td>
      <td>230</td>
      <td>79326</td>
      <td>82541</td>
      <td>2432</td>
      <td>2478</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>flashlfq</td>
      <td>flashlfq</td>
      <td>FlashLFQ mass-spectrometry proteomics label-free quantification</td>
      <td>flashlfq</td>
      <td>flashlfq</td>
      <td></td>
      <td>FlashLFQ</td>
      <td>FlashLFQ is an ultrafast label-free quantification algorithm for mass-spectrometry proteomics.</td>
      <td>Label-free quantification</td>
      <td>Proteomics experiment, Proteomics</td>
      <td>To update</td>
      <td>https://github.com/smith-chem-wisc/FlashLFQ</td>
      <td>Proteomics</td>
      <td>flashlfq</td>
      <td>2017-12-06</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/flashlfq</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/flashlfq</td>
      <td>1.0.3.1</td>
      <td>flashlfq</td>
      <td>1.2.6</td>
      <td>Label-free quantification</td>
      <td>Proteomics experiment, Proteomics</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>661</td>
      <td>715</td>
      <td>52</td>
      <td>54</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>167</td>
      <td>167</td>
      <td>15</td>
      <td>15</td>
      <td>828</td>
      <td>882</td>
      <td>67</td>
      <td>69</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>fragpipe</td>
      <td>fragpipe</td>
      <td>Data analysis for mass spectrometry-based proteomics.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://fragpipe.nesvilab.org/</td>
      <td>Proteomics</td>
      <td>fragpipe</td>
      <td>2023-05-10</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/fragpipe</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/fragpipe</td>
      <td>20.0</td>
      <td>fragpipe</td>
      <td>20.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>64</td>
      <td>64</td>
      <td>9</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>64</td>
      <td>64</td>
      <td>9</td>
      <td>9</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>hardklor</td>
      <td>hardklor, kronik</td>
      <td>Hardklör</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>hardklor</td>
      <td>2016-04-26</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tools/hardklor</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/hardklor</td>
      <td>2.30.1+galaxy1</td>
      <td>hardklor</td>
      <td>2.3.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>98</td>
      <td>111</td>
      <td>5</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>98</td>
      <td>111</td>
      <td>5</td>
      <td>8</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>idconvert</td>
      <td>idconvert</td>
      <td>Convert mass spectrometry identification files on linux or MacOSX</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>idconvert</td>
      <td>2019-02-22</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msconvert</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/idconvert</td>
      <td></td>
      <td>proteowizard</td>
      <td>3_0_9992</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>241</td>
      <td>276</td>
      <td>24</td>
      <td>33</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>241</td>
      <td>276</td>
      <td>24</td>
      <td>33</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>improviser</td>
      <td>proteomics_improviser</td>
      <td>Visualisation of PepXML files</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.improviser.uni-freiburg.de/</td>
      <td>Proteomics</td>
      <td>proteomics_improviser</td>
      <td>2015-02-28</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/proteomics/improviser</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/proteomics/improviser</td>
      <td>1.1.0.1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>lfq_protein_quant</td>
      <td>lfq_protein_quant</td>
      <td>Enable protein summarisation and quantitation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/compomics/LFQ_galaxy_p</td>
      <td>Proteomics</td>
      <td>lfq_protein_quant</td>
      <td>2018-10-02</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/lfq_protein_quant</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/lfq_protein_quant</td>
      <td>1.0</td>
      <td>bioconductor-msnbase</td>
      <td>2.28.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>78</td>
      <td>116</td>
      <td>12</td>
      <td>19</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>78</td>
      <td>116</td>
      <td>12</td>
      <td>19</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>maldiquant</td>
      <td>maldi_quant_peak_detection, maldi_quant_preprocessing</td>
      <td>MALDIquant provides a complete analysis pipeline for MALDI-TOF and other 2D mass spectrometry data.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://strimmerlab.org/software/maldiquant/</td>
      <td>Proteomics</td>
      <td>MALDIquant</td>
      <td>2018-08-22</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/MALDIquant</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/maldiquant</td>
      <td>1.22.0</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>1452</td>
      <td>3822</td>
      <td>61</td>
      <td>68</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1065</td>
      <td>1065</td>
      <td>39</td>
      <td>39</td>
      <td>2517</td>
      <td>4887</td>
      <td>100</td>
      <td>107</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>map_peptides_to_bed</td>
      <td>map_peptides_to_bed</td>
      <td>Map peptides to a reference genome for display by a genome browser</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>map_peptides_to_bed</td>
      <td>2016-01-13</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/map_peptides_to_bed</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/map_peptides_to_bed</td>
      <td>0.2</td>
      <td>biopython</td>
      <td>1.70</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>38</td>
      <td>44</td>
      <td>6</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>38</td>
      <td>44</td>
      <td>6</td>
      <td>7</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>maxquant</td>
      <td>maxquant, maxquant_mqpar</td>
      <td>wrapper for MaxQuant</td>
      <td>maxquant</td>
      <td>maxquant</td>
      <td></td>
      <td>MaxQuant</td>
      <td>Quantitative proteomics software package designed for analyzing large mass-spectrometric data sets. It is specifically aimed at high-resolution MS data.</td>
      <td>Imputation, Visualisation, Protein quantification, Statistical calculation, Standardisation and normalisation, Heat map generation, Clustering, Principal component plotting</td>
      <td>Proteomics experiment, Proteomics, Statistics and probability</td>
      <td>Up-to-date</td>
      <td>https://www.maxquant.org/</td>
      <td>Proteomics</td>
      <td>maxquant</td>
      <td>2021-08-05</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/maxquant</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/maxquant</td>
      <td>2.0.3.0</td>
      <td>maxquant</td>
      <td>2.0.3.0</td>
      <td>Imputation, Visualisation, Protein quantification, Standardisation and normalisation, Heat map generation, Clustering</td>
      <td>Proteomics experiment, Proteomics, Statistics and probability</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>6863</td>
      <td>7021</td>
      <td>1251</td>
      <td>1264</td>
      <td>2798</td>
      <td>2798</td>
      <td>543</td>
      <td>543</td>
      <td>3189</td>
      <td>3189</td>
      <td>352</td>
      <td>352</td>
      <td>12850</td>
      <td>13008</td>
      <td>2146</td>
      <td>2159</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>meta_proteome_analyzer</td>
      <td>meta_proteome_analyzer</td>
      <td>MetaProteomeAnalyzer</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/compomics/meta-proteome-analyzer/</td>
      <td>Proteomics</td>
      <td>meta_proteome_analyzer</td>
      <td>2017-03-03</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/meta_proteome_analyzer</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/meta_proteome_analyzer</td>
      <td>2.0.0</td>
      <td>mpa-portable</td>
      <td>2.0.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>126</td>
      <td>126</td>
      <td>20</td>
      <td>20</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>126</td>
      <td>126</td>
      <td>20</td>
      <td>20</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>metanovo</td>
      <td>metanovo</td>
      <td>Produce targeted databases for mass spectrometry analysis.</td>
      <td>metanovo</td>
      <td>metanovo</td>
      <td></td>
      <td>MetaNovo</td>
      <td>An open-source pipeline for probabilistic peptide discovery in complex metaproteomic datasets.</td>
      <td>Target-Decoy, de Novo sequencing, Tag-based peptide identification, Protein identification, Expression analysis</td>
      <td>Proteomics, Microbial ecology, Metagenomics, Proteomics experiment, Small molecules</td>
      <td>Up-to-date</td>
      <td>https://github.com/uct-cbio/proteomics-pipelines</td>
      <td>Proteomics</td>
      <td>metanovo</td>
      <td>2022-03-29</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/metanovo</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/metanovo</td>
      <td>1.9.4</td>
      <td>metanovo</td>
      <td>1.9.4</td>
      <td>Target-Decoy, de Novo sequencing, Tag-based peptide identification, Protein identification, Expression analysis</td>
      <td>Proteomics, Microbial ecology, Metagenomics, Proteomics experiment, Small molecules</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>4471</td>
      <td>4471</td>
      <td>29</td>
      <td>29</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4471</td>
      <td>4471</td>
      <td>29</td>
      <td>29</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>metaquantome</td>
      <td>metaquantome_db, metaquantome_expand, metaquantome_filter, metaquantome_sample, metaquantome_stat, metaquantome_viz</td>
      <td>quantitative analysis of microbiome taxonomy and function</td>
      <td>metaQuantome</td>
      <td>metaQuantome</td>
      <td></td>
      <td>metaQuantome</td>
      <td>metaQuantome software suite analyzes the state of a microbiome by leveraging complex taxonomic and functional hierarchies to summarize peptide-level quantitative information. metaQuantome offers differential abundance analysis, principal components analysis, and clustered heat map visualizations, as well as exploratory analysis for a single sample or experimental condition.</td>
      <td>Principal component visualisation, Visualisation, Functional clustering, Query and retrieval, Differential protein expression analysis, Heat map generation, Quantification, Indexing, Filtering, Statistical inference</td>
      <td>Proteomics, Metatranscriptomics, Microbial ecology, Proteomics experiment, Metagenomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproteomics/metaquantome/</td>
      <td>Proteomics</td>
      <td>metaquantome</td>
      <td>2019-04-05</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/metaquantome</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/metaquantome</td>
      <td>2.0.2</td>
      <td>metaquantome</td>
      <td>2.0.2</td>
      <td>Principal component visualisation, Functional clustering, Query and retrieval, Heat map generation, Quantification, Indexing, Filtering, Statistical inference</td>
      <td>Proteomics, Metatranscriptomics, Microbial ecology, Proteomics experiment, Metagenomics</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>2730</td>
      <td>2831</td>
      <td>260</td>
      <td>262</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1540</td>
      <td>1540</td>
      <td>71</td>
      <td>71</td>
      <td>4270</td>
      <td>4371</td>
      <td>331</td>
      <td>333</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>moFF</td>
      <td>proteomics_moff</td>
      <td>moFF (a modest Feature Finder) extracts MS1 intensities from RAW and mzML spectrum files.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/compomics/moFF</td>
      <td>Proteomics</td>
      <td>proteomics_moff</td>
      <td>2019-01-28</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/moFF</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/moFF</td>
      <td>2.0.3</td>
      <td>moff</td>
      <td>2.0.3</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>331</td>
      <td>762</td>
      <td>20</td>
      <td>34</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>137</td>
      <td>137</td>
      <td>4</td>
      <td>4</td>
      <td>468</td>
      <td>899</td>
      <td>24</td>
      <td>38</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>morpheus</td>
      <td>morpheus</td>
      <td>Morpheus MS Search Application</td>
      <td>morpheus</td>
      <td>morpheus</td>
      <td></td>
      <td>Morpheus</td>
      <td>A proteomics search algorithm specifically designed for high-resolution tandem mass spectra.</td>
      <td>Peptide database search</td>
      <td>Proteomics</td>
      <td>To update</td>
      <td>https://cwenger.github.io/Morpheus</td>
      <td>Proteomics</td>
      <td>morpheus</td>
      <td>2015-10-29</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/morpheus</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/morpheus</td>
      <td>288</td>
      <td>morpheus</td>
      <td>290</td>
      <td>Peptide database search</td>
      <td>Proteomics</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>105</td>
      <td>153</td>
      <td>12</td>
      <td>19</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>833</td>
      <td>833</td>
      <td>93</td>
      <td>93</td>
      <td>938</td>
      <td>986</td>
      <td>105</td>
      <td>112</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>mqppep</td>
      <td>mqppep_anova, mqppep_preproc</td>
      <td>MaxQuant Phosphoproteomic Enrichment Pipeline - Preprocessing and ANOVA</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/</td>
      <td>Proteomics</td>
      <td>mqppep</td>
      <td>2022-03-31</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/mqppep</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/mqppep</td>
      <td>0.1.19</td>
      <td>bioconductor-preprocesscore</td>
      <td>1.64.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>13</td>
      <td>13</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>13</td>
      <td>13</td>
      <td>3</td>
      <td>3</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>msconvert</td>
      <td>msconvert</td>
      <td>msconvert Convert and/or filter mass spectrometry files (including vendor formats) using the official Docker container</td>
      <td>msconvert</td>
      <td>msconvert</td>
      <td></td>
      <td>msConvert</td>
      <td>msConvert is a command-line utility for converting between various mass spectrometry data formats, including from raw data from several commercial companies (with vendor libraries, Windows-only). For Windows users, there is also a GUI, msConvertGUI.</td>
      <td>Filtering, Formatting</td>
      <td>Proteomics, Proteomics experiment</td>
      <td>To update</td>
      <td>http://proteowizard.sourceforge.net/tools.shtml</td>
      <td>Proteomics</td>
      <td>msconvert</td>
      <td>2019-02-14</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msconvert</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msconvert</td>
      <td>3.0.20287</td>
      <td></td>
      <td></td>
      <td>Filtering, Formatting</td>
      <td>Proteomics, Proteomics experiment</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>20749</td>
      <td>22184</td>
      <td>455</td>
      <td>469</td>
      <td>1239</td>
      <td>1239</td>
      <td>71</td>
      <td>71</td>
      <td>2666</td>
      <td>2666</td>
      <td>53</td>
      <td>53</td>
      <td>24654</td>
      <td>26089</td>
      <td>579</td>
      <td>593</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>msgfplus</td>
      <td>msgfplus</td>
      <td>MSGF+</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>msgfplus</td>
      <td>2017-01-12</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msgfplus</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msgfplus</td>
      <td>0.5</td>
      <td>msgf_plus</td>
      <td>2024.03.26</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>473</td>
      <td>529</td>
      <td>33</td>
      <td>43</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>473</td>
      <td>529</td>
      <td>33</td>
      <td>43</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>msms_extractor</td>
      <td>msms_extractor</td>
      <td>Extract MS/MS scans from the mzML file(s) based on PSM report.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>msms_extractor</td>
      <td>2019-10-23</td>
      <td>galaxyp</td>
      <td></td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msms_extractor</td>
      <td>1.0.0</td>
      <td>proteowizard</td>
      <td>3_0_9992</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>110</td>
      <td>110</td>
      <td>11</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>110</td>
      <td>110</td>
      <td>11</td>
      <td>11</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>msstats</td>
      <td>msstats</td>
      <td>MSstats tool for analyzing mass spectrometry proteomic datasets</td>
      <td>msstatstmt</td>
      <td>msstatstmt</td>
      <td></td>
      <td>MSstatsTMT</td>
      <td>Tools for detecting differentially abundant peptides and proteins in shotgun mass spectrometry-based proteomic experiments with tandem mass tag (TMT) labeling</td>
      <td>Spectrum calculation, Tag-based peptide identification, Differential protein expression profiling</td>
      <td>Proteomics, Proteomics experiment, Protein expression</td>
      <td>To update</td>
      <td>https://github.com/MeenaChoi/MSstats</td>
      <td>Proteomics</td>
      <td></td>
      <td>2020-07-25</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msstats</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msstats</td>
      <td>4.0.0</td>
      <td>bioconductor-msstats</td>
      <td>4.10.0</td>
      <td>Spectrum calculation, Tag-based peptide identification, Differential protein expression profiling</td>
      <td>Proteomics, Proteomics experiment, Protein expression</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2353</td>
      <td>2353</td>
      <td>559</td>
      <td>559</td>
      <td>1122</td>
      <td>1122</td>
      <td>211</td>
      <td>211</td>
      <td>1091</td>
      <td>1091</td>
      <td>35</td>
      <td>35</td>
      <td>4566</td>
      <td>4566</td>
      <td>805</td>
      <td>805</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>msstatstmt</td>
      <td>msstatstmt</td>
      <td>MSstatsTMT protein significance analysis in shotgun mass spectrometry-based proteomic experiments with tandem mass tag (TMT) labeling</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://msstats.org/msstatstmt/</td>
      <td>Proteomics</td>
      <td>msstatstmt</td>
      <td>2021-02-26</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msstatstmt</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msstatstmt</td>
      <td>2.0.0</td>
      <td>bioconductor-msstatstmt</td>
      <td>2.10.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>795</td>
      <td>795</td>
      <td>104</td>
      <td>104</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>461</td>
      <td>461</td>
      <td>12</td>
      <td>12</td>
      <td>1256</td>
      <td>1256</td>
      <td>116</td>
      <td>116</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>mt2mq</td>
      <td>mt2mq</td>
      <td>Tool to prepare metatranscriptomic outputs from ASaiM for Metaquantome</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>mt2mq</td>
      <td>2020-06-23</td>
      <td>galaxyp</td>
      <td></td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/mt2mq</td>
      <td>1.1.0</td>
      <td>r-tidyverse</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>275</td>
      <td>275</td>
      <td>38</td>
      <td>38</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>275</td>
      <td>275</td>
      <td>38</td>
      <td>38</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>multigsea</td>
      <td>multigsea</td>
      <td>GSEA-based pathway enrichment analysis for multi-omics data</td>
      <td>multiGSEA</td>
      <td>multiGSEA</td>
      <td></td>
      <td>multiGSEA</td>
      <td>A GSEA-based pathway enrichment analysis for multi-omics data.multiGSEA: a GSEA-based pathway enrichment analysis for multi-omics data, BMC Bioinformatics 21, 561 (2020).Combining GSEA-based pathway enrichment with multi omics data integration.</td>
      <td>Gene-set enrichment analysis, Aggregation, Pathway analysis</td>
      <td>Metabolomics, Molecular interactions, pathways and networks, Proteomics, Transcriptomics, Small molecules</td>
      <td>Up-to-date</td>
      <td>https://bioconductor.org/packages/devel/bioc/html/multiGSEA.html</td>
      <td>Transcriptomics, Proteomics, Statistics</td>
      <td>multigsea</td>
      <td>2023-06-07</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/multigsea</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/multigsea</td>
      <td>1.12.0</td>
      <td>bioconductor-multigsea</td>
      <td>1.12.0</td>
      <td>Gene-set enrichment analysis, Aggregation, Pathway analysis</td>
      <td>Metabolomics, Molecular interactions, pathways and networks, Proteomics, Transcriptomics, Small molecules</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>106</td>
      <td>106</td>
      <td>25</td>
      <td>25</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>106</td>
      <td>106</td>
      <td>25</td>
      <td>25</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>mz_to_sqlite</td>
      <td>mz_to_sqlite</td>
      <td>Creates a SQLite database for proteomics data</td>
      <td>mztosqlite</td>
      <td>mztosqlite</td>
      <td></td>
      <td>mzToSQLite</td>
      <td>Convert proteomics data files into a SQLite database</td>
      <td>Conversion, Peptide database search</td>
      <td>Proteomics, Biological databases</td>
      <td>To update</td>
      <td>https://github.com/galaxyproteomics/mzToSQLite</td>
      <td>Proteomics</td>
      <td>mz_to_sqlite</td>
      <td>2015-06-01</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/mz_to_sqlite</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/mz_to_sqlite</td>
      <td>2.1.1+galaxy0</td>
      <td>mztosqlite</td>
      <td>2.1.1</td>
      <td>Conversion, Peptide database search</td>
      <td>Proteomics, Biological databases</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>854</td>
      <td>860</td>
      <td>110</td>
      <td>111</td>
      <td>79</td>
      <td>79</td>
      <td>15</td>
      <td>15</td>
      <td>274</td>
      <td>274</td>
      <td>13</td>
      <td>13</td>
      <td>1207</td>
      <td>1213</td>
      <td>138</td>
      <td>139</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>mzml_validator</td>
      <td>mzml_validator</td>
      <td>mzML Validator checks if mzML file validates against XML Schema Definition of HUPO Proteomics Standard Initiative.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/RECETOX/galaxytools</td>
      <td>Metabolomics, Proteomics</td>
      <td></td>
      <td>2023-01-12</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/mzml_validator</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/mzml_validator</td>
      <td>0.1.0+galaxy2</td>
      <td>lxml</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>openms</td>
      <td>AccurateMassSearch, AdditiveSeries, BaselineFilter, CVInspector, CompNovo, CompNovoCID, ConsensusID, ConsensusMapNormalizer, ConvertTSVToTraML, ConvertTraMLToTSV, DTAExtractor, DeMeanderize, Decharger, DecoyDatabase, Digestor, DigestorMotif, EICExtractor, ERPairFinder, ExternalCalibration, FFEval, FalseDiscoveryRate, FeatureFinderCentroided, FeatureFinderIdentification, FeatureFinderIsotopeWavelet, FeatureFinderMRM, FeatureFinderMetabo, FeatureFinderMultiplex, FeatureFinderSuperHirn, FeatureLinkerLabeled, FeatureLinkerUnlabeled, FeatureLinkerUnlabeledQT, FidoAdapter, FileConverter, FileFilter, FileInfo, FileMerger, FuzzyDiff, HighResPrecursorMassCorrector, IDConflictResolver, IDDecoyProbability, IDExtractor, IDFileConverter, IDFilter, IDMapper, IDMassAccuracy, IDMerger, IDPosteriorErrorProbability, IDRTCalibration, IDRipper, IDScoreSwitcher, IDSplitter, ITRAQAnalyzer, InclusionExclusionListCreator, InspectAdapter, InternalCalibration, IsobaricAnalyzer, LabeledEval, LowMemPeakPickerHiRes, LowMemPeakPickerHiRes_RandomAccess, LuciphorAdapter, MRMMapper, MRMPairFinder, MRMTransitionGroupPicker, MSGFPlusAdapter, MSSimulator, MapAlignmentEvaluation, MapNormalizer, MapRTTransformer, MapStatistics, MascotAdapter, MascotAdapterOnline, MassCalculator, MassTraceExtractor, MetaProSIP, MetaboliteSpectralMatcher, MultiplexResolver, MzMLSplitter, MzTabExporter, NoiseFilterGaussian, NoiseFilterSGolay, OpenSwathAnalyzer, OpenSwathAssayGenerator, OpenSwathChromatogramExtractor, OpenSwathConfidenceScoring, OpenSwathDIAPreScoring, OpenSwathDecoyGenerator, OpenSwathFeatureXMLToTSV, OpenSwathFileSplitter, OpenSwathMzMLFileCacher, OpenSwathRTNormalizer, OpenSwathRewriteToFeatureXML, OpenSwathWorkflow, PTModel, PTPredict, PeakPickerHiRes, PeakPickerIterative, PeakPickerWavelet, PepNovoAdapter, PeptideIndexer, PhosphoScoring, PrecursorIonSelector, PrecursorMassCorrector, ProteinInference, ProteinQuantifier, ProteinResolver, QCCalculator, QCEmbedder, QCExporter, QCExtractor, QCImporter, QCMerger, QCShrinker, RNPxl, RNPxlXICFilter, RTEvaluation, RTModel, RTPredict, SemanticValidator, SequenceCoverageCalculator, SimpleSearchEngine, SpecLibCreator, SpectraFilterBernNorm, SpectraFilterMarkerMower, SpectraFilterNLargest, SpectraFilterNormalizer, SpectraFilterParentPeakMower, SpectraFilterScaler, SpectraFilterSqrtMower, SpectraFilterThresholdMower, SpectraFilterWindowMower, SpectraMerger, SvmTheoreticalSpectrumGeneratorTrainer, TICCalculator, TMTAnalyzer, TOFCalibration, TextExporter, TopPerc, TransformationEvaluation, XMLValidator, XTandemAdapter</td>
      <td>OpenMS in version 2.1.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>openms</td>
      <td>2015-10-09</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/openms</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/openms</td>
      <td>2.1.0</td>
      <td>openms</td>
      <td>3.2.0</td>
      <td></td>
      <td></td>
      <td>7</td>
      <td>34</td>
      <td>135</td>
      <td>0</td>
      <td>7</td>
      <td>34</td>
      <td>135</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>19</td>
      <td>135</td>
      <td>134</td>
      <td>0</td>
      <td>31121</td>
      <td>108907</td>
      <td>3369</td>
      <td>5029</td>
      <td>50</td>
      <td>50</td>
      <td>15</td>
      <td>15</td>
      <td>933</td>
      <td>933</td>
      <td>242</td>
      <td>242</td>
      <td>32104</td>
      <td>109890</td>
      <td>3626</td>
      <td>5286</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>openms</td>
      <td>AccurateMassSearch, AssayGeneratorMetabo, BaselineFilter, CVInspector, ClusterMassTraces, ClusterMassTracesByPrecursor, CometAdapter, ConsensusID, ConsensusMapNormalizer, DTAExtractor, DatabaseFilter, DatabaseSuitability, DeMeanderize, DecoyDatabase, Digestor, DigestorMotif, EICExtractor, ERPairFinder, Epifany, ExternalCalibration, FalseDiscoveryRate, FeatureFinderCentroided, FeatureFinderIdentification, FeatureFinderIsotopeWavelet, FeatureFinderMRM, FeatureFinderMetabo, FeatureFinderMetaboIdent, FeatureFinderMultiplex, FeatureLinkerLabeled, FeatureLinkerUnlabeled, FeatureLinkerUnlabeledKD, FeatureLinkerUnlabeledQT, FileConverter, FileFilter, FileInfo, FileMerger, FuzzyDiff, GNPSExport, HighResPrecursorMassCorrector, IDConflictResolver, IDExtractor, IDFileConverter, IDFilter, IDMapper, IDMassAccuracy, IDMerger, IDPosteriorErrorProbability, IDRTCalibration, IDRipper, IDScoreSwitcher, IDSplitter, InternalCalibration, IsobaricAnalyzer, JSONExporter, LuciphorAdapter, MRMMapper, MRMPairFinder, MRMTransitionGroupPicker, MSFraggerAdapter, MSGFPlusAdapter, MSstatsConverter, MapAlignerIdentification, MapAlignerPoseClustering, MapAlignerSpectrum, MapAlignerTreeGuided, MapNormalizer, MapRTTransformer, MapStatistics, MascotAdapter, MascotAdapterOnline, MassCalculator, MassTraceExtractor, MetaProSIP, MetaboliteSpectralMatcher, MultiplexResolver, MzMLSplitter, MzTabExporter, NoiseFilterGaussian, NoiseFilterSGolay, NovorAdapter, NucleicAcidSearchEngine, OpenMSDatabasesInfo, OpenMSInfo, OpenPepXL, OpenPepXLLF, OpenSwathAnalyzer, OpenSwathAssayGenerator, OpenSwathChromatogramExtractor, OpenSwathConfidenceScoring, OpenSwathDIAPreScoring, OpenSwathDecoyGenerator, OpenSwathFeatureXMLToTSV, OpenSwathFileSplitter, OpenSwathMzMLFileCacher, OpenSwathRTNormalizer, OpenSwathRewriteToFeatureXML, OpenSwathWorkflow, PSMFeatureExtractor, PeakPickerHiRes, PeakPickerIterative, PeakPickerWavelet, PeptideIndexer, PercolatorAdapter, PhosphoScoring, PrecursorMassCorrector, ProteinInference, ProteinQuantifier, ProteinResolver, QCCalculator, QCEmbedder, QCExporter, QCExtractor, QCImporter, QCMerger, QCShrinker, QualityControl, RNADigestor, RNAMassCalculator, RNPxlSearch, RNPxlXICFilter, SageAdapter, SeedListGenerator, SemanticValidator, SequenceCoverageCalculator, SimpleSearchEngine, SiriusAdapter, SpecLibCreator, SpecLibSearcher, SpectraFilterBernNorm, SpectraFilterMarkerMower, SpectraFilterNLargest, SpectraFilterNormalizer, SpectraFilterParentPeakMower, SpectraFilterScaler, SpectraFilterSqrtMower, SpectraFilterThresholdMower, SpectraFilterWindowMower, SpectraMerger, SpectraSTSearchAdapter, StaticModification, TICCalculator, TOFCalibration, TargetedFileConverter, TextExporter, TriqlerConverter, XFDR, XMLValidator, XTandemAdapter</td>
      <td>OpenMS Suite for LC/MS data management and analyses</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.openms.de/</td>
      <td>Proteomics</td>
      <td>openms</td>
      <td>2024-04-09</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/openms</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/openms</td>
      <td>3.1</td>
      <td>openms</td>
      <td>3.2.0</td>
      <td></td>
      <td></td>
      <td>8</td>
      <td>34</td>
      <td>144</td>
      <td>0</td>
      <td>8</td>
      <td>34</td>
      <td>144</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>18</td>
      <td>141</td>
      <td>124</td>
      <td>0</td>
      <td>32169</td>
      <td>108654</td>
      <td>3316</td>
      <td>4889</td>
      <td>50</td>
      <td>50</td>
      <td>15</td>
      <td>15</td>
      <td>940</td>
      <td>940</td>
      <td>245</td>
      <td>245</td>
      <td>33159</td>
      <td>109644</td>
      <td>3576</td>
      <td>5149</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>pathwaymatcher</td>
      <td>reactome_pathwaymatcher</td>
      <td>Reactome Pathway Matcher</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/LuisFranciscoHS/PathwayMatcher</td>
      <td>Proteomics</td>
      <td>reactome_pathwaymatcher</td>
      <td>2018-06-08</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pathwaymatcher</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pathwaymatcher</td>
      <td></td>
      <td>pathwaymatcher</td>
      <td>1.9.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>174</td>
      <td>244</td>
      <td>65</td>
      <td>76</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>174</td>
      <td>244</td>
      <td>65</td>
      <td>76</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>pep_pointer</td>
      <td>pep_pointer</td>
      <td>PepPointer categorizes peptides by their genomic coordinates.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Genomic Interval Operations, Proteomics</td>
      <td>pep_pointer</td>
      <td>2017-12-19</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pep_pointer</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pep_pointer</td>
      <td>0.1.3+galaxy1</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>506</td>
      <td>511</td>
      <td>54</td>
      <td>55</td>
      <td>21</td>
      <td>21</td>
      <td>12</td>
      <td>12</td>
      <td>163</td>
      <td>163</td>
      <td>8</td>
      <td>8</td>
      <td>690</td>
      <td>695</td>
      <td>74</td>
      <td>75</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>pepquery</td>
      <td>pepquery</td>
      <td>A peptide-centric MS search engine for novel peptide identification and validation.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://pepquery.org</td>
      <td>Proteomics</td>
      <td>pepquery</td>
      <td>2020-01-22</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pepquery</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pepquery</td>
      <td>1.6.2</td>
      <td>pepquery</td>
      <td>2.0.2</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>7047</td>
      <td>7047</td>
      <td>76</td>
      <td>76</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7047</td>
      <td>7047</td>
      <td>76</td>
      <td>76</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>pepquery2</td>
      <td>pepquery2, pepquery2_index, pepquery2_show_sets</td>
      <td>PepQuery2 peptide-centric MS search for peptide identification and validation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://pepquery.org</td>
      <td>Proteomics</td>
      <td>pepquery2</td>
      <td>2022-10-02</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pepquery2</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pepquery2</td>
      <td>2.0.2</td>
      <td>pepquery</td>
      <td>2.0.2</td>
      <td></td>
      <td></td>
      <td>3</td>
      <td>1</td>
      <td>3</td>
      <td>0</td>
      <td>3</td>
      <td>1</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>1222</td>
      <td>1222</td>
      <td>28</td>
      <td>28</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1222</td>
      <td>1222</td>
      <td>28</td>
      <td>28</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>peptide_genomic_coordinate</td>
      <td>peptide_genomic_coordinate</td>
      <td>Gets genomic coordinate of peptides based on the information in mzsqlite and genomic mapping sqlite files</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>peptide_genomic_coordinate</td>
      <td>2019-03-29</td>
      <td>galaxyp</td>
      <td></td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/peptide_genomic_coordinate</td>
      <td>1.0.0</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>472</td>
      <td>472</td>
      <td>56</td>
      <td>56</td>
      <td>16</td>
      <td>16</td>
      <td>10</td>
      <td>10</td>
      <td>149</td>
      <td>149</td>
      <td>5</td>
      <td>5</td>
      <td>637</td>
      <td>637</td>
      <td>71</td>
      <td>71</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>peptideshaker</td>
      <td>fasta_cli, ident_params, peptide_shaker, search_gui</td>
      <td>PeptideShaker and SearchGUI</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://compomics.github.io</td>
      <td>Proteomics</td>
      <td>peptideshaker</td>
      <td>2021-04-02</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/peptideshaker</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/peptideshaker</td>
      <td></td>
      <td>searchgui</td>
      <td>4.3.11</td>
      <td></td>
      <td></td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>4</td>
      <td>2</td>
      <td>0</td>
      <td>11986</td>
      <td>17898</td>
      <td>1046</td>
      <td>1153</td>
      <td>1314</td>
      <td>1314</td>
      <td>389</td>
      <td>389</td>
      <td>3034</td>
      <td>3034</td>
      <td>118</td>
      <td>118</td>
      <td>16334</td>
      <td>22246</td>
      <td>1553</td>
      <td>1660</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>peptimapper</td>
      <td>peptimapper_clustqualify, peptimapper_clust_to_gff, peptimapper_pep_match, peptimapper_pep_novo_tag</td>
      <td>Proteogenomics workflow for the expert annotation of eukaryotic genomes</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-019-5431-9</td>
      <td>Proteomics</td>
      <td></td>
      <td>2021-11-03</td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/peptimapper</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/peptimapper</td>
      <td>2.1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>pepxml_to_xls</td>
      <td></td>
      <td>Convert PepXML to Tabular</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>pepxml_to_xls</td>
      <td>2015-01-24</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pepxml_to_xls</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pepxml_to_xls</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>percolator</td>
      <td>batched_set_list_creator, percolator, percolator_input_converters, pout2mzid</td>
      <td>Percolator</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>percolator</td>
      <td>2017-03-03</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tools/percolator</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/percolator</td>
      <td>3.5</td>
      <td>percolator</td>
      <td>3.7.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>338</td>
      <td>378</td>
      <td>15</td>
      <td>16</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1059</td>
      <td>1059</td>
      <td>13</td>
      <td>13</td>
      <td>1397</td>
      <td>1437</td>
      <td>28</td>
      <td>29</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>pi_db_tools</td>
      <td>calc_delta_pi, pi_db_split, pi_dbspec_align</td>
      <td>HiRIEF tools</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>hirieftools</td>
      <td>2017-04-11</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tools/pi_db_tools</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pi_db_tools</td>
      <td>1.3</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>pmd_fdr</td>
      <td>pmd_fdr</td>
      <td>Calculate Precursor Mass Discrepancy (PMD) for MS/MS</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/slhubler/PMD-FDR-for-Galaxy-P</td>
      <td>Proteomics</td>
      <td>pmd_fdr</td>
      <td>2019-10-01</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pmd_fdr</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pmd_fdr</td>
      <td>1.4.0</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>prot-scriber</td>
      <td>prot_scriber</td>
      <td>Protein annotation of short human readable descriptions</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/usadellab/prot-scriber</td>
      <td>Proteomics</td>
      <td>prot_scriber</td>
      <td>2022-05-10</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/prot-scriber</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/prot-scriber</td>
      <td>0.1.6</td>
      <td>prot-scriber</td>
      <td>0.1.6</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>10</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>10</td>
      <td>7</td>
      <td>7</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>protease_prediction</td>
      <td>eden_protease_prediction</td>
      <td>This tool can learn the cleavage specificity of a given class of proteases.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/fabriziocosta/eden</td>
      <td>Sequence Analysis, Proteomics</td>
      <td>protease_prediction</td>
      <td>2016-03-13</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/protease_prediction</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/protease_prediction</td>
      <td>0.9</td>
      <td>eden</td>
      <td>2.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>129</td>
      <td>155</td>
      <td>15</td>
      <td>21</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>129</td>
      <td>155</td>
      <td>15</td>
      <td>21</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>proteinortho</td>
      <td>proteinortho, proteinortho_grab_proteins, proteinortho_summary</td>
      <td>Proteinortho is a tool to detect orthologous proteins/genes within different species.</td>
      <td>proteinortho</td>
      <td>proteinortho</td>
      <td></td>
      <td>Proteinortho</td>
      <td>Proteinortho is a tool to detect orthologous genes within different species</td>
      <td>Sequence clustering, Sequence analysis</td>
      <td>Comparative genomics</td>
      <td>To update</td>
      <td>https://gitlab.com/paulklemm_PHD/proteinortho</td>
      <td>Proteomics</td>
      <td>proteinortho</td>
      <td>2020-10-02</td>
      <td>iuc</td>
      <td>https://gitlab.com/paulklemm_PHD/proteinortho</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/proteinortho</td>
      <td>6.3.1</td>
      <td>proteinortho</td>
      <td>6.3.2</td>
      <td>Sequence clustering, Sequence analysis</td>
      <td>Comparative genomics</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>3962</td>
      <td>3962</td>
      <td>501</td>
      <td>501</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3962</td>
      <td>3962</td>
      <td>501</td>
      <td>501</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_joinquantpepionswithproteins</td>
      <td>proteomiqon_joinquantpepionswithproteins</td>
      <td>The tool JoinQuantPepIonsWithProteins combines results from ProteinInference and PSMBasedQuantification.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/JoinQuantPepIonsWithProteins.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_joinquantpepionswithproteins</td>
      <td>2021-09-16</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_joinquantpepionswithproteins</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_joinquantpepionswithproteins</td>
      <td>0.0.1</td>
      <td>proteomiqon-joinquantpepionswithproteins</td>
      <td>0.0.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>366</td>
      <td>366</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>366</td>
      <td>366</td>
      <td>4</td>
      <td>4</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_labeledproteinquantification</td>
      <td>proteomiqon_labeledproteinquantification</td>
      <td>The tool LabeledProteinQuantification estimates protein abundances using quantified peptide ions.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/LabeledProteinQuantification.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_labeledproteinquantification</td>
      <td>2021-09-16</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_labeledproteinquantification</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_labeledproteinquantification</td>
      <td>0.0.1</td>
      <td>proteomiqon-labeledproteinquantification</td>
      <td>0.0.3</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>16</td>
      <td>16</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>16</td>
      <td>16</td>
      <td>8</td>
      <td>8</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_labelfreeproteinquantification</td>
      <td>proteomiqon_labelfreeproteinquantification</td>
      <td>The tool LabelFreeProteinQuantification estimates protein abundances using quantified peptide ions.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/LabelfreeProteinQuantification.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_labelfreeproteinquantification</td>
      <td>2021-10-13</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_labelfreeproteinquantification</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_labelfreeproteinquantification</td>
      <td>0.0.1</td>
      <td>proteomiqon-labelfreeproteinquantification</td>
      <td>0.0.3</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>4</td>
      <td>4</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_mzmltomzlite</td>
      <td>proteomiqon_mzmltomzlite</td>
      <td>The tool MzMLToMzLite allows to convert mzML files to mzLite files.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/MzMLToMzLite.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_mzmltomzlite</td>
      <td>2021-06-30</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomIQon_MzMLToMzLite</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_mzmltomzlite</td>
      <td>0.0.8</td>
      <td>proteomiqon-mzmltomzlite</td>
      <td>0.0.8</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>721</td>
      <td>721</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>721</td>
      <td>721</td>
      <td>8</td>
      <td>8</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_peptidedb</td>
      <td>proteomiqon_peptidedb</td>
      <td>The tool ProteomIQon PeptideDB creates a peptide database in the SQLite format.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/PeptideDB.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_peptidedb</td>
      <td>2021-07-04</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_peptidedb</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_peptidedb</td>
      <td>0.0.7</td>
      <td>proteomiqon-peptidedb</td>
      <td>0.0.7</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>96</td>
      <td>96</td>
      <td>12</td>
      <td>12</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>96</td>
      <td>96</td>
      <td>12</td>
      <td>12</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_peptidespectrummatching</td>
      <td>proteomiqon_peptidespectrummatching</td>
      <td>Given raw an MS run in the mzLite format, this tool iterates across all MS/MS scans, determines precursor charge states and possible peptide spectrum matches using reimplementations of SEQUEST,Andromeda and XTandem.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/PeptideSpectrumMatching.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_peptidespectrummatching</td>
      <td>2021-07-15</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_peptidespectrummatching</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_peptidespectrummatching</td>
      <td>0.0.7</td>
      <td>proteomiqon-peptidespectrummatching</td>
      <td>0.0.7</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>686</td>
      <td>686</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>686</td>
      <td>686</td>
      <td>5</td>
      <td>5</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_proteininference</td>
      <td>proteomiqon_proteininference</td>
      <td>MS-based shotgun proteomics estimates protein abundances using a proxy: peptides. The process of 'Protein Inference' is concerned with the mapping of identified peptides to the proteins they putatively originated from.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/ProteinInference.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_proteininference</td>
      <td>2021-07-09</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_proteininference</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_proteininference</td>
      <td>0.0.7</td>
      <td>proteomiqon-proteininference</td>
      <td>0.0.7</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>89</td>
      <td>89</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>89</td>
      <td>89</td>
      <td>6</td>
      <td>6</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_psmbasedquantification</td>
      <td>proteomiqon_psmbasedquantification</td>
      <td>The PSMBasedQuantification tool was designed to allow label-free quantification as well as quantification of full metabolic labeled samples.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/PSMBasedQuantification.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_psmbasedquantification</td>
      <td>2021-07-09</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_psmbasedquantification</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_psmbasedquantification</td>
      <td>0.0.8</td>
      <td>proteomiqon-psmbasedquantification</td>
      <td>0.0.9</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>604</td>
      <td>604</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>604</td>
      <td>604</td>
      <td>4</td>
      <td>4</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_psmstatistics</td>
      <td>proteomiqon_psmstatistics</td>
      <td>The PSMStatistics tool utilizes semi supervised machine learning techniques to integrate search engine scores as well as the mentioned quality scores into one single consensus score.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/PSMStatistics.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_psmstatistics</td>
      <td>2021-07-09</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_psmstatistics</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_psmstatistics</td>
      <td>0.0.8</td>
      <td>proteomiqon-psmstatistics</td>
      <td>0.0.8</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>694</td>
      <td>694</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>694</td>
      <td>694</td>
      <td>6</td>
      <td>6</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>proteore_venn_diagram</td>
      <td>proteore_venn_diagram</td>
      <td>ProteoRE JVenn Diagram</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>proteore_venn_diagram</td>
      <td>2021-05-17</td>
      <td>galaxyp</td>
      <td></td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteore_venn_diagram</td>
      <td>2021.06.08</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>15</td>
      <td>15</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>15</td>
      <td>15</td>
      <td>1</td>
      <td>1</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>psm2sam</td>
      <td>PSMtoSAM</td>
      <td>PSM to SAM</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/proBAMr.html</td>
      <td>Proteomics</td>
      <td>psm_to_sam</td>
      <td>2017-10-06</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tools/bumbershoot/psm2sam</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/probam_suite/psm2sam</td>
      <td>1.3.2.1</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>psm_validation</td>
      <td>psmvalidator</td>
      <td>Validate PSM from Ion Fragmentation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproteomics/psm_fragments.git</td>
      <td>Proteomics</td>
      <td>psm_validation</td>
      <td>2020-10-13</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/psm_validation</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/psm_validation</td>
      <td>1.0.3</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>20</td>
      <td>20</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>20</td>
      <td>20</td>
      <td>4</td>
      <td>4</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>pyprophet</td>
      <td>pyprophet_export, pyprophet_merge, pyprophet_peptide, pyprophet_protein, pyprophet_score, pyprophet_subsample</td>
      <td>Semi-supervised learning and scoring of OpenSWATH results.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/PyProphet/pyprophet</td>
      <td>Proteomics</td>
      <td></td>
      <td>2015-02-06</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pyprophet</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pyprophet</td>
      <td>2.1.4</td>
      <td>pyprophet</td>
      <td>2.2.5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>5</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>2626</td>
      <td>2626</td>
      <td>150</td>
      <td>150</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>609</td>
      <td>609</td>
      <td>9</td>
      <td>9</td>
      <td>3235</td>
      <td>3235</td>
      <td>159</td>
      <td>159</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>pyteomics</td>
      <td>mztab2tsv</td>
      <td>Tools using the pyteomics library</td>
      <td>pyteomics</td>
      <td>pyteomics</td>
      <td></td>
      <td>Pyteomics</td>
      <td>Framework for proteomics data analysis, supporting mzML, MGF, pepXML and more.</td>
      <td>Protein identification</td>
      <td>Proteomics, Proteomics experiment</td>
      <td>To update</td>
      <td>https://pyteomics.readthedocs.io/en/latest/</td>
      <td>Proteomics, Metabolomics</td>
      <td>pyteomics</td>
      <td>2021-01-15</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pyteomics</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pyteomics</td>
      <td>4.4.1</td>
      <td>pyteomics</td>
      <td>4.7.5</td>
      <td>Protein identification</td>
      <td>Proteomics, Proteomics experiment</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>92</td>
      <td>92</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>92</td>
      <td>92</td>
      <td>4</td>
      <td>4</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>quantp</td>
      <td>quantp</td>
      <td>Correlation between protein and transcript abundance</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>quantp</td>
      <td>2018-09-14</td>
      <td>galaxyp</td>
      <td></td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/quantp</td>
      <td>1.1.2</td>
      <td>r-data.table</td>
      <td>1.11.6</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>197</td>
      <td>241</td>
      <td>17</td>
      <td>22</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>197</td>
      <td>241</td>
      <td>17</td>
      <td>22</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>quantwiz_iq</td>
      <td>quantwiz_iq</td>
      <td>Isobaric Quantitation using QuantWiz-IQ</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://sourceforge.net/projects/quantwiz/</td>
      <td>Proteomics</td>
      <td>quantwiz_iq</td>
      <td>2020-01-21</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/quantwiz_iq</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/quantwiz_iq</td>
      <td>2.0</td>
      <td>quantwiz-iq</td>
      <td>2.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>32</td>
      <td>32</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>32</td>
      <td>32</td>
      <td>3</td>
      <td>3</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>rawtools</td>
      <td>rawtools</td>
      <td>Raw Tools</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/kevinkovalchik/RawTools</td>
      <td>Proteomics</td>
      <td>rawtools</td>
      <td>2019-02-28</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/rawtools</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/rawtools</td>
      <td></td>
      <td>rawtools</td>
      <td>2.0.4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>166</td>
      <td>179</td>
      <td>32</td>
      <td>33</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>414</td>
      <td>414</td>
      <td>13</td>
      <td>13</td>
      <td>580</td>
      <td>593</td>
      <td>45</td>
      <td>46</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>sixgill</td>
      <td>sixgill_build, sixgill_filter, sixgill_makefasta, sixgill_merge</td>
      <td>Six-frame Genome-Inferred Libraries for LC-MS/MS</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td></td>
      <td>Proteomics, MetaProteomics</td>
      <td>sixgill</td>
      <td>2016-10-11</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/sixgill</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/sixgill</td>
      <td>0.2.4</td>
      <td>sixgill</td>
      <td>0.2.4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>312</td>
      <td>312</td>
      <td>71</td>
      <td>71</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>312</td>
      <td>312</td>
      <td>71</td>
      <td>71</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>spectrast2spectrast_irt</td>
      <td>gp_spectrast2spectrast_irt</td>
      <td>Filter from spectraST files to swath input files</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>spectrast2spectrast_irt</td>
      <td>2015-04-13</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/spectrast2spectrast_irt</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/spectrast2spectrast_irt</td>
      <td>0.1.0</td>
      <td>msproteomicstools</td>
      <td>0.11.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>spectrast2tsv</td>
      <td>gp_spectrast2tsv</td>
      <td>Filter from spectraST files to swath input files</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>spectrast2tsv</td>
      <td>2015-02-06</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/spectrast2tsv</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/spectrast2tsv</td>
      <td>0.1.0</td>
      <td>msproteomicstools</td>
      <td>0.11.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>syndiva</td>
      <td>syndiva</td>
      <td>SynDivA was developed to analyze the diversity of synthetic libraries of a Fibronectin domain.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>syndiva</td>
      <td>2022-06-23</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/SynDivA</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/syndiva</td>
      <td>1.0</td>
      <td>clustalo</td>
      <td>1.2.4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>30</td>
      <td>30</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>30</td>
      <td>30</td>
      <td>2</td>
      <td>2</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>tapscan</td>
      <td>tapscan_classify</td>
      <td>Search for transcription associated proteins (TAPs)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://plantcode.cup.uni-freiburg.de/tapscan/</td>
      <td>Proteomics</td>
      <td>tapscan</td>
      <td>2024-02-14</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/tapscan</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/tapscan</td>
      <td>4.76+galaxy0</td>
      <td>hmmer</td>
      <td>3.4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>19</td>
      <td>19</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>19</td>
      <td>19</td>
      <td>6</td>
      <td>6</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>translate_bed</td>
      <td>translate_bed</td>
      <td>Translate BED transcript CDS or cDNA in 3 frames</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://rest.ensembl.org/</td>
      <td>Proteomics</td>
      <td>translate_bed</td>
      <td>2018-01-14</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteogenomics/translate_bed</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteogenomics/translate_bed</td>
      <td>0.1.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>649</td>
      <td>685</td>
      <td>138</td>
      <td>142</td>
      <td>473</td>
      <td>473</td>
      <td>105</td>
      <td>105</td>
      <td>761</td>
      <td>761</td>
      <td>18</td>
      <td>18</td>
      <td>1883</td>
      <td>1919</td>
      <td>261</td>
      <td>265</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>translate_bed_sequences</td>
      <td>translate_bed_sequences</td>
      <td>Perform 3 frame translation of BED file augmented with a sequence column</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>translate_bed_sequences</td>
      <td>2016-01-13</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/translate_bed_sequences</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/translate_bed_sequences</td>
      <td>0.2.0</td>
      <td>biopython</td>
      <td>1.70</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>57</td>
      <td>69</td>
      <td>16</td>
      <td>21</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>57</td>
      <td>69</td>
      <td>16</td>
      <td>21</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>unipept</td>
      <td>unipept</td>
      <td>Unipept retrieves metaproteomics information</td>
      <td>unipept</td>
      <td>unipept</td>
      <td></td>
      <td>Unipept</td>
      <td>Metaproteomics data analysis with a focus on interactive data visualizations.</td>
      <td>Prediction and recognition, Visualisation</td>
      <td>Proteomics, Proteogenomics, Biodiversity, Workflows</td>
      <td>To update</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp</td>
      <td>Proteomics</td>
      <td>unipept</td>
      <td>2015-04-03</td>
      <td>galaxyp</td>
      <td>https://unipept.ugent.be/apidocs</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/unipept</td>
      <td>4.5.1</td>
      <td>python</td>
      <td></td>
      <td>Prediction and recognition, Visualisation</td>
      <td>Proteomics, Proteogenomics, Biodiversity, Workflows</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>4900</td>
      <td>5146</td>
      <td>257</td>
      <td>270</td>
      <td>251</td>
      <td>251</td>
      <td>81</td>
      <td>81</td>
      <td>1069</td>
      <td>1069</td>
      <td>29</td>
      <td>29</td>
      <td>6220</td>
      <td>6466</td>
      <td>367</td>
      <td>380</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>uniprot_rest_interface</td>
      <td>uniprot</td>
      <td>UniProt ID mapping and sequence retrieval</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/jdrudolph/uniprot</td>
      <td>Proteomics, Sequence Analysis</td>
      <td>uniprot_rest_interface</td>
      <td>2015-10-09</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/uniprot_rest_interface</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/uniprot_rest_interface</td>
      <td>0.6</td>
      <td>requests</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2117</td>
      <td>2725</td>
      <td>341</td>
      <td>412</td>
      <td>3111</td>
      <td>5779</td>
      <td>594</td>
      <td>1011</td>
      <td>806</td>
      <td>806</td>
      <td>50</td>
      <td>50</td>
      <td>6034</td>
      <td>9310</td>
      <td>985</td>
      <td>1473</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>uniprotxml_downloader</td>
      <td>uniprotxml_downloader</td>
      <td>Download UniProt proteome in XML or fasta format</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>uniprotxml_downloader</td>
      <td>2016-03-08</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/uniprotxml_downloader</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/uniprotxml_downloader</td>
      <td>2.4.0</td>
      <td>requests</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1488</td>
      <td>1596</td>
      <td>230</td>
      <td>266</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>743</td>
      <td>743</td>
      <td>35</td>
      <td>35</td>
      <td>2231</td>
      <td>2339</td>
      <td>265</td>
      <td>301</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>validate_fasta_database</td>
      <td>validate_fasta_database</td>
      <td>runs Compomics database identification tool on any FASTA database, and separates valid and invalid entries based on a series of checks.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Fasta Manipulation, Proteomics</td>
      <td>validate_fasta_database</td>
      <td>2017-09-14</td>
      <td>galaxyp</td>
      <td></td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/validate_fasta_database</td>
      <td>0.1.5</td>
      <td>validate-fasta-database</td>
      <td>1.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>112</td>
      <td>112</td>
      <td>38</td>
      <td>38</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>112</td>
      <td>112</td>
      <td>38</td>
      <td>38</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>zeiss_lmd_converter</td>
      <td>ZeissLMDconverter</td>
      <td>Converts coordinates from a tabular file into a formatted text file readable by Zeiss laser-capture microdissection systems</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/zeiss_lmd_converter</td>
      <td>Proteomics</td>
      <td></td>
      <td>2024-06-29</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/zeiss_lmd_converter</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/zeiss_lmd_converter</td>
      <td>2.0.4</td>
      <td>shapely</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>1</td>
      <td>1</td>
      <td></td>
    </tr>
  </tbody>
</table>

</div>