<script src="https://code.jquery.com/jquery-3.7.0.js"></script>
<script src="https://cdn.datatables.net/1.13.6/js/jquery.dataTables.min.js"></script>
<script src="https://cdn.datatables.net/searchbuilder/1.6.0/js/dataTables.searchBuilder.min.js"></script>
<script src="https://cdn.datatables.net/datetime/1.5.1/js/dataTables.dateTime.min.js"></script>

<script src="https://cdn.datatables.net/responsive/2.5.0/js/dataTables.responsive.min.js"></script>

<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/responsive/2.2.3/css/responsive.dataTables.min.css">

<link rel="stylesheet" href="https://cdn.datatables.net/1.13.6/css/jquery.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/searchbuilder/1.6.0/css/searchBuilder.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/datetime/1.5.1/css/dataTables.dateTime.min.css">

<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" />
<script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>

<style type="text/css">
  .tooltip {
    font-family: Georgia;
    font-size: 18px;
  }
  .tooltip .tooltip-inner {
    background-color: #00FFFF;
    color: #000000;
    min-width: 400px;
  }
  </style>

<script>
    $(document).ready(function() {
    $('#dataframe').DataTable( {
        "sScrollX": "100%",
        "dom": "Qlfrtip",

        responsive: {
            details: {
                type: 'column'
            }
        },
        "bAutoWidth": false,
        columnDefs: [ {
            className: 'control',
            orderable: false,
            targets:   [ 0 ],
            width: 100,
        },
        { type: 'natural', targets: 1 } // allow normal sorting
        ],

    });
});
</script>

<div class="display nowrap" style="width:80%" width="100%">

<table class="dataframe display nowrap" id="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th>Expand</th>
      <th>Suite ID</th>
      <th>Tool IDs</th>
      <th>Description</th>
      <th>Suite first commit date</th>
      <th>Homepage</th>
      <th>Suite version</th>
      <th>Suite conda package</th>
      <th>Latest suite conda package version</th>
      <th>Suite version status</th>
      <th>ToolShed categories</th>
      <th>EDAM operations</th>
      <th>EDAM reduced operations</th>
      <th>EDAM topics</th>
      <th>EDAM reduced topics</th>
      <th>Suite owner</th>
      <th>Suite source</th>
      <th>Suite parsed folder</th>
      <th>bio.tool ID</th>
      <th>bio.tool name</th>
      <th>bio.tool description</th>
      <th>biii ID</th>
      <th>Number of tools on UseGalaxy.org (Main)</th>
      <th>Number of tools on UseGalaxy.org.au</th>
      <th>Number of tools on UseGalaxy.eu</th>
      <th>Number of tools on UseGalaxy.fr</th>
      <th>Number of tools on APOSTL</th>
      <th>Number of tools on BioBix</th>
      <th>Number of tools on CIRM-CFBP</th>
      <th>Number of tools on ChemFlow</th>
      <th>Number of tools on Coloc-stats</th>
      <th>Number of tools on CorGAT</th>
      <th>Number of tools on FreeBioinfo</th>
      <th>Number of tools on GASLINI</th>
      <th>Number of tools on Galaxy@AuBi</th>
      <th>Number of tools on Galaxy@Pasteur</th>
      <th>Number of tools on GalaxyTrakr</th>
      <th>Number of tools on Genomic Hyperbrowser</th>
      <th>Number of tools on HyPhy HIV NGS Tools</th>
      <th>Number of tools on IPK Galaxy Blast Suite</th>
      <th>Number of tools on ImmPort Galaxy</th>
      <th>Number of tools on InteractoMIX</th>
      <th>Number of tools on Lebanese University Galaxy</th>
      <th>Number of tools on MISSISSIPPI</th>
      <th>Number of tools on Mandoiu Lab</th>
      <th>Number of tools on MiModD NacreousMap</th>
      <th>Number of tools on Palfinder</th>
      <th>Number of tools on PepSimili</th>
      <th>Number of tools on PhagePromotor</th>
      <th>Number of tools on UseGalaxy.be</th>
      <th>Number of tools on UseGalaxy.cz</th>
      <th>Number of tools on UseGalaxy.no</th>
      <th>Number of tools on Viral Variant Visualizer (VVV)</th>
      <th>Suite users (last 5 years) (usegalaxy.eu)</th>
      <th>Suite users (usegalaxy.eu)</th>
      <th>Suite runs (last 5 years) (usegalaxy.eu)</th>
      <th>Suite runs (usegalaxy.eu)</th>
      <th>Suite users (last 5 years) (usegalaxy.org)</th>
      <th>Suite users (usegalaxy.org)</th>
      <th>Suite runs (last 5 years) (usegalaxy.org)</th>
      <th>Suite runs (usegalaxy.org)</th>
      <th>Suite users (last 5 years) (usegalaxy.org.au)</th>
      <th>Suite users (usegalaxy.org.au)</th>
      <th>Suite runs (last 5 years) (usegalaxy.org.au)</th>
      <th>Suite runs (usegalaxy.org.au)</th>
      <th>Suite users (last 5 years) (usegalaxy.fr)</th>
      <th>Suite users (usegalaxy.fr)</th>
      <th>Suite runs (last 5 years) (usegalaxy.fr)</th>
      <th>Suite runs (usegalaxy.fr)</th>
      <th>Suite runs on main servers</th>
      <th>Suite runs (last 5 years) on main servers</th>
      <th>Suite users on main servers</th>
      <th>Suite users (last 5 years) on main servers</th>
      <th>Related Workflows</th>
      <th>Related Tutorials</th>
      <th>To keep</th>
      <th>Deprecated</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td></td>
      <td>AMRFinderPlus</td>
      <td>amrfinderplus</td>
      <td>"AMRFinderPlus is designed to find acquired antimicrobial resistance genes and point mutations in protein and/or assembled nucleotide sequences.It can also search "plus", stress, heat, and biocide resistance and virulence factors for some organisms.</td>
      <td>2023-05-12</td>
      <td>https://github.com/ncbi/amr</td>
      <td>3.12.8</td>
      <td>ncbi-amrfinderplus</td>
      <td>4.2.5</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Antimicrobial resistance prediction</td>
      <td>Antimicrobial resistance prediction</td>
      <td>Microbiology, Public health and epidemiology, Infectious disease</td>
      <td>Microbiology, Public health and epidemiology, Infectious disease</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/amrfinderplus</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/amrfinderplus</td>
      <td>amrfinderplus</td>
      <td>AMRFinderPlus</td>
      <td>AMRFinderPlus is designed to find acquired antimicrobial resistance genes and point mutations in protein and/or assembled nucleotide sequences.It can also search "plus", stress, heat, and biocide resistance and virulence factors for some organisms</td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>953</td>
      <td>953</td>
      <td>45130</td>
      <td>45130</td>
      <td>689</td>
      <td>689</td>
      <td>17362</td>
      <td>17362</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>28</td>
      <td>28</td>
      <td>1863</td>
      <td>1863</td>
      <td>64355</td>
      <td>64355</td>
      <td>1670</td>
      <td>1670</td>
      <td>https://usegalaxy.eu/published/workflow?id=1ce523ef51f47134, https://usegalaxy.eu/published/workflow?id=309fc4fc2e0da89c, https://usegalaxy.eu/published/workflow?id=32de10b9de8a999a, https://usegalaxy.eu/published/workflow?id=34df701aa23a9fb3, https://usegalaxy.eu/published/workflow?id=368d1ac0150ea0b6, https://usegalaxy.eu/published/workflow?id=41d207d06a32ba05, https://usegalaxy.eu/published/workflow?id=46b6812817c57a01, https://usegalaxy.eu/published/workflow?id=5b940b283d91d99d, https://usegalaxy.eu/published/workflow?id=a448400f69594135, https://usegalaxy.eu/published/workflow?id=abe85758e77c78e6, https://usegalaxy.eu/published/workflow?id=b203dec6ee6da2fd, https://usegalaxy.eu/published/workflow?id=cc59ae1a9878942c, https://usegalaxy.eu/published/workflow?id=e37cc0b64774561b, https://usegalaxy.eu/published/workflow?id=eb5ab8bc82569eee, https://usegalaxy.eu/published/workflow?id=f30f8e7dcd50cd0d, https://usegalaxy.eu/published/workflow?id=fde21b5f5c169114, https://usegalaxy.org/published/workflow?id=2078a54de1f2628d, https://usegalaxy.org/published/workflow?id=2b4b1fe82258f303, https://usegalaxy.org/published/workflow?id=3c177fdf38311118, https://usegalaxy.org/published/workflow?id=9c0fbd9f51a9857f, https://usegalaxy.org/published/workflow?id=aaf8e740b3f8e2c0, https://usegalaxy.org/published/workflow?id=d3dc805cd43b2737, https://usegalaxy.org/published/workflow?id=ef187f85b103d858, https://usegalaxy.org/published/workflow?id=f8fab0cd6fc30d92, https://usegalaxy.org/published/workflow?id=fde774a049875466, https://workflowhub.eu/workflows/1049?version=9, https://workflowhub.eu/workflows/2024?version=2</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>ISEScan</td>
      <td>isescan</td>
      <td>"ISEScan is a pipeline to identify IS (Insertion Sequence) elements in genome and metagenomebased on profile hidden Markov models constructed from manually curated IS elements."</td>
      <td>2022-09-01</td>
      <td>https://github.com/xiezhq/ISEScan</td>
      <td>1.7.3</td>
      <td>isescan</td>
      <td>1.7.3</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Structural variation detection</td>
      <td>Structural variation detection</td>
      <td>Genomics, DNA structural variation, Sequence analysis, Genetic variation</td>
      <td>Genomics, Sequence analysis, Genetic variation</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/isescan</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/isescan</td>
      <td>ISEScan</td>
      <td>ISEScan</td>
      <td>Automated identification of insertion sequence elements in prokaryotic genomes.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>392</td>
      <td>392</td>
      <td>67516</td>
      <td>67516</td>
      <td>237</td>
      <td>237</td>
      <td>1535</td>
      <td>1535</td>
      <td>116</td>
      <td>116</td>
      <td>7464</td>
      <td>7464</td>
      <td>37</td>
      <td>37</td>
      <td>1759</td>
      <td>1759</td>
      <td>78274</td>
      <td>78274</td>
      <td>782</td>
      <td>782</td>
      <td>https://usegalaxy.eu/published/workflow?id=1306ed083d91a852, https://usegalaxy.eu/published/workflow?id=2c1a6da84d891cf2, https://usegalaxy.eu/published/workflow?id=3081569659a45b9e, https://usegalaxy.eu/published/workflow?id=36ea584e0f60893e, https://usegalaxy.eu/published/workflow?id=3f5fe28793ed0c6c, https://usegalaxy.eu/published/workflow?id=4a218fb93d78ba67, https://usegalaxy.eu/published/workflow?id=61948fcaefa1418a, https://usegalaxy.eu/published/workflow?id=6b69ab27c565257b, https://usegalaxy.eu/published/workflow?id=8792eaecb69fbc26, https://usegalaxy.eu/published/workflow?id=a18aed4b03ffd27e, https://usegalaxy.eu/published/workflow?id=b1afc8d1e81fc904, https://usegalaxy.eu/published/workflow?id=bcd50e655809cfcb, https://usegalaxy.eu/published/workflow?id=d8db0b2e30370b84, https://usegalaxy.eu/published/workflow?id=dfb8e0f3c534000e, https://usegalaxy.eu/published/workflow?id=fd63b9884fda75b6, https://usegalaxy.fr/published/workflow?id=7964ddc68853458f, https://usegalaxy.org/published/workflow?id=04c5ea133de3204c, https://usegalaxy.org/published/workflow?id=3940186d02245b1d, https://usegalaxy.org/published/workflow?id=49da012e2839c046, https://usegalaxy.org/published/workflow?id=4f8bdd61c524b901, https://usegalaxy.org/published/workflow?id=649cdc14757e5755, https://usegalaxy.org/published/workflow?id=67ca17ab34b10772, https://usegalaxy.org/published/workflow?id=8cb8a588a0495cfa, https://usegalaxy.org/published/workflow?id=993534c11310a207, https://usegalaxy.org/published/workflow?id=d2a5f6a5181aada5, https://workflowhub.eu/workflows/1050?version=13, https://workflowhub.eu/workflows/1513?version=1</td>
      <td>genome-annotation/bacterial-genome-annotation</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>abacas</td>
      <td>abacas</td>
      <td>Order and Orientate Contigs</td>
      <td>2019-11-20</td>
      <td>https://github.com/phac-nml/abacas</td>
      <td>1.1</td>
      <td>mummer</td>
      <td>3.23</td>
      <td>To update</td>
      <td>Assembly</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>nml</td>
      <td>https://github.com/phac-nml/abacas</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/abacas</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>2</td>
      <td>2</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>abricate</td>
      <td>abricate, abricate_list, abricate_summary</td>
      <td>Mass screening of contigs for antiobiotic resistance genes</td>
      <td>2016-07-29</td>
      <td>https://github.com/tseemann/abricate</td>
      <td>1.0.1</td>
      <td>abricate</td>
      <td>1.2.0</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Antimicrobial resistance prediction</td>
      <td>Antimicrobial resistance prediction</td>
      <td>Genomics, Microbiology</td>
      <td>Genomics, Microbiology</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/abricate</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/abricate</td>
      <td>ABRicate</td>
      <td>ABRicate</td>
      <td>Mass screening of contigs for antimicrobial resistance or virulence genes.</td>
      <td></td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>4522</td>
      <td>4679</td>
      <td>799841</td>
      <td>821957</td>
      <td>3401</td>
      <td>3468</td>
      <td>401550</td>
      <td>405363</td>
      <td>1867</td>
      <td>2216</td>
      <td>578174</td>
      <td>600408</td>
      <td>85</td>
      <td>85</td>
      <td>19178</td>
      <td>19178</td>
      <td>1846906</td>
      <td>1798743</td>
      <td>10448</td>
      <td>9875</td>
      <td>https://usegalaxy.eu/published/workflow?id=1ce523ef51f47134, https://usegalaxy.eu/published/workflow?id=2b888e200dfd91bc, https://usegalaxy.eu/published/workflow?id=2e0075791f7fa384, https://usegalaxy.eu/published/workflow?id=3081569659a45b9e, https://usegalaxy.eu/published/workflow?id=309fc4fc2e0da89c, https://usegalaxy.eu/published/workflow?id=32de10b9de8a999a, https://usegalaxy.eu/published/workflow?id=34df701aa23a9fb3, https://usegalaxy.eu/published/workflow?id=368d1ac0150ea0b6, https://usegalaxy.eu/published/workflow?id=37da7b42cb063a3a, https://usegalaxy.eu/published/workflow?id=3f5fe28793ed0c6c, https://usegalaxy.eu/published/workflow?id=41d207d06a32ba05, https://usegalaxy.eu/published/workflow?id=42c0da2e86318b37, https://usegalaxy.eu/published/workflow?id=46b6812817c57a01, https://usegalaxy.eu/published/workflow?id=49c46d7653d6c349, https://usegalaxy.eu/published/workflow?id=4f245ac304ab76d9, https://usegalaxy.eu/published/workflow?id=5826e573b6f93762, https://usegalaxy.eu/published/workflow?id=585c21b7b1d864fc, https://usegalaxy.eu/published/workflow?id=5b940b283d91d99d, https://usegalaxy.eu/published/workflow?id=628062a8cf93261f, https://usegalaxy.eu/published/workflow?id=6984990d0ac2e1e6, https://usegalaxy.eu/published/workflow?id=6a5c39c1d38a10c2, https://usegalaxy.eu/published/workflow?id=6d87481be47902e1, https://usegalaxy.eu/published/workflow?id=77e5bbd317750915, https://usegalaxy.eu/published/workflow?id=94c3d329c31c712f, https://usegalaxy.eu/published/workflow?id=a337bfb2eb1e583f, https://usegalaxy.eu/published/workflow?id=a84bac86ec183d46, https://usegalaxy.eu/published/workflow?id=abe85758e77c78e6, https://usegalaxy.eu/published/workflow?id=b203dec6ee6da2fd, https://usegalaxy.eu/published/workflow?id=bf48b0eb15e8ff75, https://usegalaxy.eu/published/workflow?id=c62d65832377e376, https://usegalaxy.eu/published/workflow?id=c9ef2d2ce92e2dfb, https://usegalaxy.eu/published/workflow?id=e37cc0b64774561b, https://usegalaxy.eu/published/workflow?id=eb5ab8bc82569eee, https://usegalaxy.eu/published/workflow?id=f30f8e7dcd50cd0d, https://usegalaxy.eu/published/workflow?id=fde21b5f5c169114, https://usegalaxy.org.au/published/workflow?id=05f3abecd4ddd4a1, https://usegalaxy.org.au/published/workflow?id=084bb76cf47d7060, https://usegalaxy.org.au/published/workflow?id=35f39173f0ac9179, https://usegalaxy.org.au/published/workflow?id=59618ed15510f531, https://usegalaxy.org.au/published/workflow?id=7a0c4804e4d4c905, https://usegalaxy.org.au/published/workflow?id=914330feb75ea9ab, https://usegalaxy.org.au/published/workflow?id=c997a34b55425155, https://usegalaxy.org.au/published/workflow?id=c9b7de574fb2f2d5, https://usegalaxy.org.au/published/workflow?id=d78f885b4b86635f, https://usegalaxy.org.au/published/workflow?id=e20172eb0f30d6cd, https://usegalaxy.org.au/published/workflow?id=ef8c22c2525063a2, https://usegalaxy.org/published/workflow?id=00596e862edb23f2, https://usegalaxy.org/published/workflow?id=2078a54de1f2628d, https://usegalaxy.org/published/workflow?id=2b4b1fe82258f303, https://usegalaxy.org/published/workflow?id=3c177fdf38311118, https://usegalaxy.org/published/workflow?id=3f252e077e0bcce5, https://usegalaxy.org/published/workflow?id=562bcaa9b4d2e8a4, https://usegalaxy.org/published/workflow?id=69d8863d40524120, https://usegalaxy.org/published/workflow?id=7dd6e4150bb81ad1, https://usegalaxy.org/published/workflow?id=80565301ada0d3eb, https://usegalaxy.org/published/workflow?id=80ac6adead9f97d8, https://usegalaxy.org/published/workflow?id=8111bb9bb26bb665, https://usegalaxy.org/published/workflow?id=9ba6da7dae465d3e, https://usegalaxy.org/published/workflow?id=9c0fbd9f51a9857f, https://usegalaxy.org/published/workflow?id=aaf8e740b3f8e2c0, https://usegalaxy.org/published/workflow?id=c65bec6c332bf740, https://usegalaxy.org/published/workflow?id=cce88bc57b180d09, https://usegalaxy.org/published/workflow?id=d3dc805cd43b2737, https://usegalaxy.org/published/workflow?id=e12324f5e3b930fd, https://usegalaxy.org/published/workflow?id=ef187f85b103d858, https://usegalaxy.org/published/workflow?id=f8fab0cd6fc30d92, https://usegalaxy.org/published/workflow?id=fb01e1a49793c8be, https://usegalaxy.org/published/workflow?id=fde774a049875466, https://workflowhub.eu/workflows/1049?version=9, https://workflowhub.eu/workflows/1062?version=1, https://workflowhub.eu/workflows/1189?version=2, https://workflowhub.eu/workflows/1495?version=1, https://workflowhub.eu/workflows/2024?version=2, https://workflowhub.eu/workflows/470?version=1</td>
      <td>microbiome/pathogen-detection-from-nanopore-foodborne-data</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>abritamr</td>
      <td>abritamr</td>
      <td>A pipeline for running AMRfinderPlus and collating results into functional classes</td>
      <td>2023-04-03</td>
      <td>https://zenodo.org/record/7370628</td>
      <td>1.0.20</td>
      <td>abritamr</td>
      <td>1.0.20</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Antimicrobial resistance prediction</td>
      <td>Antimicrobial resistance prediction</td>
      <td>Microbiology, Public health and epidemiology, Infectious disease, Antimicrobial resistance</td>
      <td>Microbiology, Public health and epidemiology, Infectious disease</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/abritamr</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/abritamr</td>
      <td>abritamr</td>
      <td>abriTAMR</td>
      <td>an AMR gene detection pipeline that runs AMRFinderPlus on a single (or list ) of given isolates and collates the results into a table, separating genes identified into functionally relevant groups.</td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>319</td>
      <td>319</td>
      <td>2633</td>
      <td>2633</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>97</td>
      <td>97</td>
      <td>880</td>
      <td>880</td>
      <td>3</td>
      <td>3</td>
      <td>5</td>
      <td>5</td>
      <td>3518</td>
      <td>3518</td>
      <td>419</td>
      <td>419</td>
      <td>https://usegalaxy.eu/published/workflow?id=eb5ab8bc82569eee, https://usegalaxy.org.au/published/workflow?id=7a0c4804e4d4c905, https://workflowhub.eu/workflows/634?version=1</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>abyss</td>
      <td>abyss-pe</td>
      <td>Assembly By Short Sequences - a de novo, parallel, paired-end sequence assembler</td>
      <td>2015-04-14</td>
      <td>http://www.bcgsc.ca/platform/bioinfo/software/abyss</td>
      <td>2.3.10</td>
      <td>abyss</td>
      <td>2.3.10</td>
      <td>Up-to-date</td>
      <td>Assembly</td>
      <td>Genome assembly, De-novo assembly, Scaffolding</td>
      <td>Genome assembly, De-novo assembly, Scaffolding</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/abyss</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/abyss</td>
      <td>abyss</td>
      <td>ABySS</td>
      <td>De novo genome sequence assembler using short reads.</td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1258</td>
      <td>1368</td>
      <td>7148</td>
      <td>7779</td>
      <td>183</td>
      <td>414</td>
      <td>505</td>
      <td>1305</td>
      <td>398</td>
      <td>398</td>
      <td>2608</td>
      <td>2608</td>
      <td>11</td>
      <td>11</td>
      <td>16</td>
      <td>16</td>
      <td>11708</td>
      <td>10277</td>
      <td>2191</td>
      <td>1850</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>aldex2</td>
      <td>aldex2</td>
      <td>Performs analysis Of differential abundance taking sample variation into account</td>
      <td>2022-06-29</td>
      <td>https://github.com/ggloor/ALDEx_bioc</td>
      <td>1.26.0</td>
      <td>bioconductor-aldex2</td>
      <td>1.38.0</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Statistical inference</td>
      <td>Statistical inference</td>
      <td>Gene expression, Statistics and probability</td>
      <td>Gene expression, Statistics and probability</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/aldex2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/aldex2</td>
      <td>aldex2</td>
      <td>ALDEx2</td>
      <td>A differential abundance analysis for the comparison of two or more conditions. It uses a Dirichlet-multinomial model to infer abundance from counts, that has been optimized for three or more experimental replicates. Infers sampling variation and calculates the expected FDR given the biological and sampling variation using the Wilcox rank test and Welches t-test, or the glm and Kruskal Wallis tests. Reports both P and fdr values calculated by the Benjamini Hochberg correction.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>64</td>
      <td>64</td>
      <td>356</td>
      <td>356</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>356</td>
      <td>356</td>
      <td>64</td>
      <td>64</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>amas</td>
      <td>amas_concat, amas_remove, amas_replicate, amas_split, amas_summary</td>
      <td>AMAS high-throughput alignment manipulation and summaries for phylogenomics</td>
      <td>2025-12-02</td>
      <td>https://github.com/marekborowiec/AMAS</td>
      <td>1.0</td>
      <td>amas</td>
      <td>1.0</td>
      <td>Up-to-date</td>
      <td>Phylogenetics, Sequence Analysis, Statistics</td>
      <td>Nucleic acid sequence analysis, Protein sequence analysis</td>
      <td>Nucleic acid sequence analysis, Protein sequence analysis</td>
      <td>Phylogenomics, Phylogenetics, Sequence analysis, Statistics and probability</td>
      <td>Phylogenomics, Phylogenetics, Statistics and probability</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/amas</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/amas</td>
      <td>amas</td>
      <td>AMAS</td>
      <td>AMAS (Alignment Manipulation And Summary), a tool that works on amino acid and nucleotide alignments and combines capabilities of sequence manipulation with a function that calculates basic statistics.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>amplican</td>
      <td>amplican</td>
      <td>AmpliCan is an analysis tool for genome editing.</td>
      <td>2021-09-22</td>
      <td>https://github.com/valenlab/amplican</td>
      <td>1.14.0</td>
      <td>bioconductor-amplican</td>
      <td>1.28.0</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Alignment, Standardisation and normalisation</td>
      <td>Alignment, Standardisation and normalisation</td>
      <td>PCR experiment, Statistics and probability</td>
      <td>PCR experiment, Statistics and probability</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/amplican</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/amplican</td>
      <td>amplican</td>
      <td>amplican</td>
      <td>It performs alignment of the amplicon reads, normalizes gathered data, calculates multiple statistics (e.g. cut rates, frameshifts) and presents results in form of aggregated reports. Data and statistics can be broken down by experiments, barcodes, user defined groups, guides and amplicons allowing for quick identification of potential problems.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>18</td>
      <td>18</td>
      <td>85</td>
      <td>85</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>85</td>
      <td>85</td>
      <td>18</td>
      <td>18</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>ampligone</td>
      <td>ampligone</td>
      <td>AmpliGone is a tool which accurately finds and removes primer sequences from NGS reads in an amplicon experiment.</td>
      <td>2025-07-29</td>
      <td>https://rivm-bioinformatics.github.io/AmpliGone/latest/</td>
      <td>2.0.1</td>
      <td>AmpliGone</td>
      <td>2.0.2</td>
      <td>To update</td>
      <td>Sequence Analysis, Fastq Manipulation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ampligone</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ampligone</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>ancombc</td>
      <td>ancombc</td>
      <td>Performs analysis of compositions of microbiomes with bias correction.</td>
      <td>2022-07-16</td>
      <td>https://github.com/FrederickHuangLin/ANCOMBC</td>
      <td>1.4.0</td>
      <td>bioconductor-ancombc</td>
      <td>2.8.0</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>DNA barcoding</td>
      <td>DNA barcoding</td>
      <td>Microbial ecology, Metagenomics, Taxonomy</td>
      <td>Microbial ecology, Metagenomics, Taxonomy</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ancombc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ancombc</td>
      <td>ancombc</td>
      <td>ANCOMBC</td>
      <td>Determine taxa whose absolute abundances, per unit volume, of the ecosystem (e.g. gut) are significantly different with changes in the covariate of interest (e.g. group). The current version of ancombc function implements Analysis of Compositions of Microbiomes with Bias Correction (ANCOM-BC) in cross-sectional data while allowing for covariate adjustment.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>20</td>
      <td>20</td>
      <td>64</td>
      <td>64</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>64</td>
      <td>64</td>
      <td>20</td>
      <td>20</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>antismash</td>
      <td>antismash</td>
      <td>Antismash allows the genome-wide identification, annotation and analysis of secondary metabolite biosynthesis gene clusters</td>
      <td>2015-02-28</td>
      <td>https://antismash.secondarymetabolites.org</td>
      <td>6.1.1</td>
      <td>antismash</td>
      <td>8.0.4</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Sequence clustering, Gene prediction, Differential gene expression analysis</td>
      <td>Sequence clustering, Gene prediction, Differential gene expression analysis</td>
      <td>Molecular interactions, pathways and networks, Gene and protein families</td>
      <td>Molecular interactions, pathways and networks, Gene and protein families</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/antismash</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/antismash</td>
      <td>antismash</td>
      <td>antiSMASH</td>
      <td>Rapid genome-wide identification, annotation and analysis of secondary metabolite biosynthesis gene clusters in bacterial and fungal genomes. It integrates and cross-links with a large number of in silico secondary metabolite analysis tools that have been published earlier.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1083</td>
      <td>1236</td>
      <td>29269</td>
      <td>32703</td>
      <td>1017</td>
      <td>1017</td>
      <td>12499</td>
      <td>12499</td>
      <td>280</td>
      <td>280</td>
      <td>10622</td>
      <td>10622</td>
      <td>6</td>
      <td>6</td>
      <td>170</td>
      <td>170</td>
      <td>55994</td>
      <td>52560</td>
      <td>2539</td>
      <td>2386</td>
      <td>https://usegalaxy.eu/published/workflow?id=0461bbd1f96292e2, https://usegalaxy.eu/published/workflow?id=0ab3e47ebd3b520e, https://usegalaxy.eu/published/workflow?id=24eaa682897379c8, https://usegalaxy.eu/published/workflow?id=33312e5643279e8a, https://usegalaxy.eu/published/workflow?id=5be158931abca947, https://usegalaxy.eu/published/workflow?id=6984990d0ac2e1e6, https://usegalaxy.eu/published/workflow?id=6a5c39c1d38a10c2, https://usegalaxy.eu/published/workflow?id=6d87481be47902e1, https://usegalaxy.eu/published/workflow?id=9a60ecf98d4f8134, https://usegalaxy.eu/published/workflow?id=a448400f69594135, https://usegalaxy.eu/published/workflow?id=bf48b0eb15e8ff75, https://usegalaxy.eu/published/workflow?id=cc59ae1a9878942c, https://usegalaxy.eu/published/workflow?id=edbaf66c1e97ebdb, https://usegalaxy.eu/published/workflow?id=f141db5179ea8422, https://usegalaxy.org.au/published/workflow?id=35f39173f0ac9179, https://usegalaxy.org/published/workflow?id=c451e4dca9981ed0, https://workflowhub.eu/workflows/1558?version=1, https://workflowhub.eu/workflows/37?version=1, https://workflowhub.eu/workflows/38?version=1, https://workflowhub.eu/workflows/39?version=1, https://workflowhub.eu/workflows/40?version=1</td>
      <td>genome-annotation/secondary-metabolite-discovery</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>argnorm</td>
      <td>argnorm</td>
      <td>argNorm is a tool to normalize antibiotic resistance genes (ARGs) by mapping them to the antibiotic resistance ontology (ARO) created by the CARD database</td>
      <td>2024-08-28</td>
      <td>https://github.com/BigDataBiology/argNorm</td>
      <td>1.0.0</td>
      <td>argnorm</td>
      <td>1.1.0</td>
      <td>To update</td>
      <td>Genome annotation</td>
      <td>Gene functional annotation</td>
      <td>Gene functional annotation</td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/argnorm</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/argnorm</td>
      <td>argnorm</td>
      <td>argNorm</td>
      <td>argNorm is a tool to normalize antibiotic resistance genes (ARGs) by mapping them to the antibiotic resistance ontology (ARO) created by the CARD database.argNorm also enhances antibiotic resistance gene annotations by providing drug categorization of the drugs that antibiotic resistance genes confer resistance to.</td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>48</td>
      <td>48</td>
      <td>211</td>
      <td>211</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>91</td>
      <td>91</td>
      <td>302</td>
      <td>302</td>
      <td>51</td>
      <td>51</td>
      <td>https://workflowhub.eu/workflows/2024?version=2</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>artic</td>
      <td>artic_guppyplex, artic_minion</td>
      <td>The artic pipeline is designed to help run the artic bioinformatics protocols;for example the nCoV-2019 novel coronavirus protocol.Features include: read filtering, primer trimming, amplicon coverage normalisation,variant calling and consensus building</td>
      <td>2020-04-17</td>
      <td>https://github.com/artic-network/fieldbioinformatics</td>
      <td>1.7.3</td>
      <td>artic</td>
      <td>1.8.5</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Sequence alignment</td>
      <td>Sequence alignment</td>
      <td>Genomics</td>
      <td>Genomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/artic</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/artic</td>
      <td>artic</td>
      <td>ARTIC</td>
      <td>A bioinformatics pipeline for working with virus sequencing data sequenced with nanopore</td>
      <td></td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>2</td>
      <td>1</td>
      <td>0</td>
      <td>206</td>
      <td>206</td>
      <td>13444</td>
      <td>13459</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>34</td>
      <td>34</td>
      <td>1812</td>
      <td>1812</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>15271</td>
      <td>15256</td>
      <td>240</td>
      <td>240</td>
      <td>https://usegalaxy.org.au/published/workflow?id=aa1d123c0a0929f3, https://workflowhub.eu/workflows/521?version=1</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>assembly_stats</td>
      <td>assembly_stats</td>
      <td>Assembly metric visualisations to facilitate rapid assessment and comparison of assembly quality.</td>
      <td>2023-06-21</td>
      <td>https://github.com/rjchallis/assembly-stats</td>
      <td>17.02</td>
      <td>rjchallis-assembly-stats</td>
      <td>17.02</td>
      <td>Up-to-date</td>
      <td>Assembly</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/assembly-stats</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/assembly-stats</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>18</td>
      <td>18</td>
      <td>32</td>
      <td>32</td>
      <td>32</td>
      <td>32</td>
      <td>18</td>
      <td>18</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>True</td>
    </tr>
    <tr>
      <td></td>
      <td>assemblystats</td>
      <td>assemblystats</td>
      <td>Summarise an assembly (e.g. N50 metrics)</td>
      <td>2017-10-13</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>1.1.0</td>
      <td>perl-bioperl</td>
      <td>1.7.8</td>
      <td>To update</td>
      <td>Assembly</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/assemblystats</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>autobigs</td>
      <td>autobigs-cli</td>
      <td>Automated MLST typing from PubMLST and InstitutPasteur.</td>
      <td>2025-02-24</td>
      <td>https://github.com/Syph-and-VPD-Lab/autoBIGS.cli</td>
      <td>0.6.2</td>
      <td>autobigs-cli</td>
      <td>0.6.5</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/autobigs</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/autobigs</td>
      <td>AutoBIGS.CLI</td>
      <td>AutoBIGS.CLI</td>
      <td>A command-line interface (CLI) based program that allows quickly batched requests for obtaining MLST profiles on multiple FASTA sequences and exporting it as a convenient CSV. Capable of querying a variety of MLST databases from both Institut Pasteur and PubMLST.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>31</td>
      <td>31</td>
      <td>53</td>
      <td>53</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>53</td>
      <td>53</td>
      <td>31</td>
      <td>31</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>bakta</td>
      <td>bakta</td>
      <td>"Bakta is a tool for the rapid &amp; standardized annotation of bacterial genomes and plasmids from both isolates and MAGs.It provides dbxref-rich and sORF-including annotations in machine-readable JSON &amp; bioinformatics standard file formats for automatic downstream analysis."</td>
      <td>2022-09-01</td>
      <td>https://github.com/oschwengers/bakta</td>
      <td>1.9.4</td>
      <td>bakta</td>
      <td>1.11.4</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Genome annotation</td>
      <td>Genome annotation</td>
      <td>Genomics, Sequence analysis, Bioinformatics</td>
      <td>Genomics, Sequence analysis, Bioinformatics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bakta</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bakta</td>
      <td>bakta</td>
      <td>Bakta</td>
      <td>Rapid &amp; standardized annotation of bacterial genomes, MAGs &amp; plasmids</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1503</td>
      <td>1503</td>
      <td>67465</td>
      <td>67465</td>
      <td>1017</td>
      <td>1017</td>
      <td>14351</td>
      <td>14351</td>
      <td>399</td>
      <td>399</td>
      <td>33530</td>
      <td>33530</td>
      <td>84</td>
      <td>84</td>
      <td>3664</td>
      <td>3664</td>
      <td>119010</td>
      <td>119010</td>
      <td>3003</td>
      <td>3003</td>
      <td>https://usegalaxy.eu/published/workflow?id=1306ed083d91a852, https://usegalaxy.eu/published/workflow?id=29c565eff0080e42, https://usegalaxy.eu/published/workflow?id=2c1a6da84d891cf2, https://usegalaxy.eu/published/workflow?id=309fc4fc2e0da89c, https://usegalaxy.eu/published/workflow?id=36ea584e0f60893e, https://usegalaxy.eu/published/workflow?id=3f5fe28793ed0c6c, https://usegalaxy.eu/published/workflow?id=4a218fb93d78ba67, https://usegalaxy.eu/published/workflow?id=4f9f7065975b3f97, https://usegalaxy.eu/published/workflow?id=61948fcaefa1418a, https://usegalaxy.eu/published/workflow?id=6b69ab27c565257b, https://usegalaxy.eu/published/workflow?id=7ecd92cb382d8168, https://usegalaxy.eu/published/workflow?id=8792eaecb69fbc26, https://usegalaxy.eu/published/workflow?id=968f0ac2617aa524, https://usegalaxy.eu/published/workflow?id=a18aed4b03ffd27e, https://usegalaxy.eu/published/workflow?id=b1afc8d1e81fc904, https://usegalaxy.eu/published/workflow?id=bcd50e655809cfcb, https://usegalaxy.eu/published/workflow?id=cc59ae1a9878942c, https://usegalaxy.eu/published/workflow?id=d8db0b2e30370b84, https://usegalaxy.eu/published/workflow?id=dfb8e0f3c534000e, https://usegalaxy.eu/published/workflow?id=fd63b9884fda75b6, https://usegalaxy.eu/published/workflow?id=fde21b5f5c169114, https://usegalaxy.fr/published/workflow?id=7964ddc68853458f, https://usegalaxy.org.au/published/workflow?id=b6bad1f167257248, https://usegalaxy.org/published/workflow?id=00596e862edb23f2, https://usegalaxy.org/published/workflow?id=04c5ea133de3204c, https://usegalaxy.org/published/workflow?id=3940186d02245b1d, https://usegalaxy.org/published/workflow?id=49da012e2839c046, https://usegalaxy.org/published/workflow?id=4f8bdd61c524b901, https://usegalaxy.org/published/workflow?id=547f86db467bf620, https://usegalaxy.org/published/workflow?id=562bcaa9b4d2e8a4, https://usegalaxy.org/published/workflow?id=649cdc14757e5755, https://usegalaxy.org/published/workflow?id=67ca17ab34b10772, https://usegalaxy.org/published/workflow?id=69d8863d40524120, https://usegalaxy.org/published/workflow?id=7dd6e4150bb81ad1, https://usegalaxy.org/published/workflow?id=80ac6adead9f97d8, https://usegalaxy.org/published/workflow?id=8cb8a588a0495cfa, https://usegalaxy.org/published/workflow?id=993534c11310a207, https://usegalaxy.org/published/workflow?id=9c0fbd9f51a9857f, https://usegalaxy.org/published/workflow?id=d2a5f6a5181aada5, https://workflowhub.eu/workflows/1050?version=13, https://workflowhub.eu/workflows/1352?version=4, https://workflowhub.eu/workflows/1488?version=2, https://workflowhub.eu/workflows/1513?version=1</td>
      <td>genome-annotation/amr-gene-detection, genome-annotation/bacterial-genome-annotation, microbiome/mags-building</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>bamtools</td>
      <td>bamtools</td>
      <td>Operate on and transform BAM datasets in various ways using bamtools</td>
      <td>2017-06-09</td>
      <td>https://github.com/pezmaster31/bamtools</td>
      <td>2.5.2</td>
      <td>bamtools</td>
      <td>2.5.3</td>
      <td>To update</td>
      <td>Sequence Analysis, SAM</td>
      <td>Data handling, Sequence alignment analysis</td>
      <td>Data handling, Sequence alignment analysis</td>
      <td>Sequencing, Data management, Sequence analysis</td>
      <td>Sequencing, Sequence analysis</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/bamtools/bamtools</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/bamtools/bamtools</td>
      <td>bamtools</td>
      <td>BamTools</td>
      <td>BamTools provides a fast, flexible C++ API &amp; toolkit for reading, writing, and managing BAM files.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>583</td>
      <td>746</td>
      <td>13272</td>
      <td>15936</td>
      <td>1177</td>
      <td>5735</td>
      <td>56981</td>
      <td>102230</td>
      <td>6</td>
      <td>6</td>
      <td>283</td>
      <td>283</td>
      <td>14</td>
      <td>14</td>
      <td>56</td>
      <td>56</td>
      <td>118505</td>
      <td>70592</td>
      <td>6501</td>
      <td>1780</td>
      <td>https://usegalaxy.eu/published/workflow?id=36a4646959829018, https://usegalaxy.eu/published/workflow?id=99022559adf01c4a, https://usegalaxy.eu/published/workflow?id=ce5c16ac23c4fbb1, https://workflowhub.eu/workflows/1480?version=1, https://workflowhub.eu/workflows/1631?version=1</td>
      <td>assembly/metagenomics-assembly, microbiome/metagenomics-assembly</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>bandage</td>
      <td>bandage_image, bandage_info</td>
      <td>Bandage - A Bioinformatics Application for Navigating De novo Assembly Graphs Easily</td>
      <td>2018-09-06</td>
      <td>https://github.com/rrwick/Bandage</td>
      <td>2022.09</td>
      <td>bandage_ng</td>
      <td>2026.1.1</td>
      <td>To update</td>
      <td>Visualization</td>
      <td>Sequence assembly visualisation</td>
      <td>Sequence assembly visualisation</td>
      <td>Genomics, Sequence assembly</td>
      <td>Genomics, Sequence assembly</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bandage</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bandage</td>
      <td>bandage</td>
      <td>Bandage</td>
      <td>GUI program that allows users to interact with the assembly graphs made by de novo assemblers such as Velvet, SPAdes, MEGAHIT and others. It visualises assembly graphs, with connections, using graph layout algorithms.</td>
      <td></td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>5065</td>
      <td>5359</td>
      <td>74766</td>
      <td>78507</td>
      <td>4846</td>
      <td>5038</td>
      <td>63998</td>
      <td>65491</td>
      <td>3140</td>
      <td>3234</td>
      <td>25266</td>
      <td>26126</td>
      <td>317</td>
      <td>319</td>
      <td>9763</td>
      <td>9780</td>
      <td>179904</td>
      <td>173793</td>
      <td>13950</td>
      <td>13368</td>
      <td>https://usegalaxy.eu/published/workflow?id=002a51bdef00e14d, https://usegalaxy.eu/published/workflow?id=01fe4c590e608103, https://usegalaxy.eu/published/workflow?id=03be9d62e8e4a129, https://usegalaxy.eu/published/workflow?id=0404013d5c27210d, https://usegalaxy.eu/published/workflow?id=085c3a4c89d137b5, https://usegalaxy.eu/published/workflow?id=089e5e83b1de0511, https://usegalaxy.eu/published/workflow?id=0ebfab0e957b7267, https://usegalaxy.eu/published/workflow?id=1151052f4393852a, https://usegalaxy.eu/published/workflow?id=1483534f49b826bf, https://usegalaxy.eu/published/workflow?id=152223c8aed27357, https://usegalaxy.eu/published/workflow?id=17b0a6d3379a4392, https://usegalaxy.eu/published/workflow?id=194a05d42687b524, https://usegalaxy.eu/published/workflow?id=1ac0a15758277730, https://usegalaxy.eu/published/workflow?id=1bb084f1bc102734, https://usegalaxy.eu/published/workflow?id=1cba6eb4aff8dc61, https://usegalaxy.eu/published/workflow?id=1da7ac8eaf891f39, https://usegalaxy.eu/published/workflow?id=1e02a7292e9974b1, https://usegalaxy.eu/published/workflow?id=1ea8bfbe6cb04531, https://usegalaxy.eu/published/workflow?id=1f2a1438e41188f2, https://usegalaxy.eu/published/workflow?id=21a3f72851f29925, https://usegalaxy.eu/published/workflow?id=249c115413f17f28, https://usegalaxy.eu/published/workflow?id=2671a5f85b2cccfe, https://usegalaxy.eu/published/workflow?id=270b6f82efab28b2, https://usegalaxy.eu/published/workflow?id=2b888e200dfd91bc, https://usegalaxy.eu/published/workflow?id=2e0075791f7fa384, https://usegalaxy.eu/published/workflow?id=32d2f5340098bcbd, https://usegalaxy.eu/published/workflow?id=3303a04887a47675, https://usegalaxy.eu/published/workflow?id=342a5df69951e3d7, https://usegalaxy.eu/published/workflow?id=346b46e6316c6950, https://usegalaxy.eu/published/workflow?id=374797e58377c40f, https://usegalaxy.eu/published/workflow?id=379642036b6f5a66, https://usegalaxy.eu/published/workflow?id=39837ca09374598b, https://usegalaxy.eu/published/workflow?id=3a3e2d50fe7a9ee3, https://usegalaxy.eu/published/workflow?id=3d937648809ca4e7, https://usegalaxy.eu/published/workflow?id=3e255a5b7026d26e, https://usegalaxy.eu/published/workflow?id=3eb6663a66f8e69a, https://usegalaxy.eu/published/workflow?id=3f5414bb38b47fcf, https://usegalaxy.eu/published/workflow?id=416141b245073406, https://usegalaxy.eu/published/workflow?id=4216424f032c055f, https://usegalaxy.eu/published/workflow?id=44e53ea319e07daa, https://usegalaxy.eu/published/workflow?id=45ae834d842302c3, https://usegalaxy.eu/published/workflow?id=471eb8b4c54f897d, https://usegalaxy.eu/published/workflow?id=475ea16f943cc013, https://usegalaxy.eu/published/workflow?id=4856e3522b2ee059, https://usegalaxy.eu/published/workflow?id=497b82d3b92dc076, https://usegalaxy.eu/published/workflow?id=4c3a9b574f37eebb, https://usegalaxy.eu/published/workflow?id=4d41ee83fc0040d4, https://usegalaxy.eu/published/workflow?id=4ddbffe4b3fef275, https://usegalaxy.eu/published/workflow?id=54590ccd11ffbf65, https://usegalaxy.eu/published/workflow?id=56523b41ff7eec76, https://usegalaxy.eu/published/workflow?id=5826e573b6f93762, https://usegalaxy.eu/published/workflow?id=585c21b7b1d864fc, https://usegalaxy.eu/published/workflow?id=5ad406442ec21345, https://usegalaxy.eu/published/workflow?id=5b893b69a1b6748f, https://usegalaxy.eu/published/workflow?id=5e1906290f5b4d8a, https://usegalaxy.eu/published/workflow?id=62c47e3c9d4f3aa3, https://usegalaxy.eu/published/workflow?id=6573f3b1606b7f09, https://usegalaxy.eu/published/workflow?id=69be516a48761365, https://usegalaxy.eu/published/workflow?id=69cdd404592535ec, https://usegalaxy.eu/published/workflow?id=7137e45ca8dc6e4c, https://usegalaxy.eu/published/workflow?id=71a6d128a09210f0, https://usegalaxy.eu/published/workflow?id=723e718f37e9ab91, https://usegalaxy.eu/published/workflow?id=7337d15d121596da, https://usegalaxy.eu/published/workflow?id=77e5bbd317750915, https://usegalaxy.eu/published/workflow?id=79b5afb125e3c3da, https://usegalaxy.eu/published/workflow?id=7aa32aba3fde0708, https://usegalaxy.eu/published/workflow?id=7b755497d168cb13, https://usegalaxy.eu/published/workflow?id=7e48134082dab0a3, https://usegalaxy.eu/published/workflow?id=847403b72dd346ff, https://usegalaxy.eu/published/workflow?id=86cf02cc0868d238, https://usegalaxy.eu/published/workflow?id=8b2f53745e720d0f, https://usegalaxy.eu/published/workflow?id=8be366ad6a42408b, https://usegalaxy.eu/published/workflow?id=9207e74a5bf7d8e1, https://usegalaxy.eu/published/workflow?id=9219557de92e0203, https://usegalaxy.eu/published/workflow?id=92dafb3d1c06e074, https://usegalaxy.eu/published/workflow?id=93d1035a307fe63d, https://usegalaxy.eu/published/workflow?id=95f420d8856bf725, https://usegalaxy.eu/published/workflow?id=968f0ac2617aa524, https://usegalaxy.eu/published/workflow?id=98bfad0c684011e9, https://usegalaxy.eu/published/workflow?id=9c0b67496dfe52b2, https://usegalaxy.eu/published/workflow?id=9ddb703e1db13504, https://usegalaxy.eu/published/workflow?id=9f2d18f0d0896e8b, https://usegalaxy.eu/published/workflow?id=a337bfb2eb1e583f, https://usegalaxy.eu/published/workflow?id=a6dd62ab4b9a0725, https://usegalaxy.eu/published/workflow?id=a727e22dc0431614, https://usegalaxy.eu/published/workflow?id=a847773b563b2328, https://usegalaxy.eu/published/workflow?id=a84bac86ec183d46, https://usegalaxy.eu/published/workflow?id=a9c1dca5da7170dd, https://usegalaxy.eu/published/workflow?id=aa6dc57f5c67fb4b, https://usegalaxy.eu/published/workflow?id=ac6eb99f70d35999, https://usegalaxy.eu/published/workflow?id=ae74f60cd20d56bc, https://usegalaxy.eu/published/workflow?id=b1d9e56aef03672b, https://usegalaxy.eu/published/workflow?id=b211fceaecf41c9e, https://usegalaxy.eu/published/workflow?id=b4948afacccfefdb, https://usegalaxy.eu/published/workflow?id=b4de76c0a4d0e23a, https://usegalaxy.eu/published/workflow?id=b93333efdab316d8, https://usegalaxy.eu/published/workflow?id=ba1fda2318046543, https://usegalaxy.eu/published/workflow?id=bc32937d505135ab, https://usegalaxy.eu/published/workflow?id=bcf99cc0469db276, https://usegalaxy.eu/published/workflow?id=bf48b0eb15e8ff75, https://usegalaxy.eu/published/workflow?id=c48807045e258dd8, https://usegalaxy.eu/published/workflow?id=c4c2e669b86cffbc, https://usegalaxy.eu/published/workflow?id=c508c00e81018dd8, https://usegalaxy.eu/published/workflow?id=c702c76cadebac30, https://usegalaxy.eu/published/workflow?id=c9ef2d2ce92e2dfb, https://usegalaxy.eu/published/workflow?id=caba6d3d7db648e9, https://usegalaxy.eu/published/workflow?id=d2f75f70c33394ee, https://usegalaxy.eu/published/workflow?id=dc27453c438577f6, https://usegalaxy.eu/published/workflow?id=dc7107896a3f2964, https://usegalaxy.eu/published/workflow?id=de9cf573314f416c, https://usegalaxy.eu/published/workflow?id=dfe3b555856c7b44, https://usegalaxy.eu/published/workflow?id=e138640778b7414c, https://usegalaxy.eu/published/workflow?id=e1947b059ef131d2, https://usegalaxy.eu/published/workflow?id=e2f936f1aa43a629, https://usegalaxy.eu/published/workflow?id=e532d98f4b368666, https://usegalaxy.eu/published/workflow?id=e7aebe58411d97ed, https://usegalaxy.eu/published/workflow?id=e8f9e1d911683b22, https://usegalaxy.eu/published/workflow?id=ea485dbabc76488d, https://usegalaxy.eu/published/workflow?id=eb5ab8bc82569eee, https://usegalaxy.eu/published/workflow?id=ec79f14ab91cd933, https://usegalaxy.eu/published/workflow?id=ecf48fa50af560a0, https://usegalaxy.eu/published/workflow?id=ef83f7fd2176a244, https://usegalaxy.eu/published/workflow?id=f1ca4289ee4f67bb, https://usegalaxy.eu/published/workflow?id=f38745208494d4ab, https://usegalaxy.eu/published/workflow?id=f578a097b4339025, https://usegalaxy.eu/published/workflow?id=fb015e398373ea8a, https://usegalaxy.eu/published/workflow?id=fc0b9efa7d2e61e6, https://usegalaxy.eu/published/workflow?id=fc25c1f8c6667c87, https://usegalaxy.eu/published/workflow?id=fca458178b3c794a, https://usegalaxy.eu/published/workflow?id=fd649ac4d4872c81, https://usegalaxy.fr/published/workflow?id=5bb882218afda132, https://usegalaxy.org.au/published/workflow?id=02eebe3d909206ab, https://usegalaxy.org.au/published/workflow?id=044b6b475b4db27a, https://usegalaxy.org.au/published/workflow?id=084bb76cf47d7060, https://usegalaxy.org.au/published/workflow?id=08fac3b4c128e597, https://usegalaxy.org.au/published/workflow?id=0af0ea50f1237952, https://usegalaxy.org.au/published/workflow?id=0cc8c8875df6bccb, https://usegalaxy.org.au/published/workflow?id=10355b2680374a03, https://usegalaxy.org.au/published/workflow?id=1ad4c47cee851dda, https://usegalaxy.org.au/published/workflow?id=1eb3e1452c672480, https://usegalaxy.org.au/published/workflow?id=22bd45f60c60a338, https://usegalaxy.org.au/published/workflow?id=2470cb61ba51288c, https://usegalaxy.org.au/published/workflow?id=2cb16d321a804b1a, https://usegalaxy.org.au/published/workflow?id=2d45d7138d8715b4, https://usegalaxy.org.au/published/workflow?id=2e29b21cdec7d0cc, https://usegalaxy.org.au/published/workflow?id=319e9d693fac8dce, https://usegalaxy.org.au/published/workflow?id=35f39173f0ac9179, https://usegalaxy.org.au/published/workflow?id=3604307ec2835628, https://usegalaxy.org.au/published/workflow?id=36604d0ab0dee988, https://usegalaxy.org.au/published/workflow?id=36f54eda8568355d, https://usegalaxy.org.au/published/workflow?id=371c9892d261cee2, https://usegalaxy.org.au/published/workflow?id=38d992baf9a2a2bc, https://usegalaxy.org.au/published/workflow?id=3c30c822058187be, https://usegalaxy.org.au/published/workflow?id=3df7f45b7b388f8c, https://usegalaxy.org.au/published/workflow?id=40666ceb68d101ac, https://usegalaxy.org.au/published/workflow?id=41795303a88d9e94, https://usegalaxy.org.au/published/workflow?id=4267bd1e4cfd1901, https://usegalaxy.org.au/published/workflow?id=43343fa1896f8278, https://usegalaxy.org.au/published/workflow?id=447d8daefdefafee, https://usegalaxy.org.au/published/workflow?id=479c6b0a96ea1610, https://usegalaxy.org.au/published/workflow?id=47cd0a90609fd557, https://usegalaxy.org.au/published/workflow?id=4b5f80656fc94b88, https://usegalaxy.org.au/published/workflow?id=51bad198f1fd1215, https://usegalaxy.org.au/published/workflow?id=52065a4100b11184, https://usegalaxy.org.au/published/workflow?id=5550585ffc3d8d0e, https://usegalaxy.org.au/published/workflow?id=5b0c0e34394afed1, https://usegalaxy.org.au/published/workflow?id=5b32e2f0d390a4ba, https://usegalaxy.org.au/published/workflow?id=5f4f874e49515555, https://usegalaxy.org.au/published/workflow?id=60c0dec52cdca708, https://usegalaxy.org.au/published/workflow?id=63789b6f82172679, https://usegalaxy.org.au/published/workflow?id=68b2961117e62890, https://usegalaxy.org.au/published/workflow?id=68f5c371adbe5490, https://usegalaxy.org.au/published/workflow?id=6c27019d03dd49d8, https://usegalaxy.org.au/published/workflow?id=6c9129acb9d98afe, https://usegalaxy.org.au/published/workflow?id=6ec0031406b10635, https://usegalaxy.org.au/published/workflow?id=7c343e5b55474d19, https://usegalaxy.org.au/published/workflow?id=803ef29108e36e58, https://usegalaxy.org.au/published/workflow?id=813a4938397cf6ee, https://usegalaxy.org.au/published/workflow?id=8926e592f2e60863, https://usegalaxy.org.au/published/workflow?id=8a40059343cf4b87, https://usegalaxy.org.au/published/workflow?id=8a6233015db6dd24, https://usegalaxy.org.au/published/workflow?id=8bf8c9929249f37d, https://usegalaxy.org.au/published/workflow?id=8dd86497bc15d981, https://usegalaxy.org.au/published/workflow?id=8e8a5cce9f3f2f67, https://usegalaxy.org.au/published/workflow?id=91ee716be8bfd508, https://usegalaxy.org.au/published/workflow?id=9642aadb83c4f752, https://usegalaxy.org.au/published/workflow?id=970dc0d8bd74707f, https://usegalaxy.org.au/published/workflow?id=98f660971010d11d, https://usegalaxy.org.au/published/workflow?id=9c60b83feaa22415, https://usegalaxy.org.au/published/workflow?id=9e0bae0505ee7c07, https://usegalaxy.org.au/published/workflow?id=a02c102e179596cb, https://usegalaxy.org.au/published/workflow?id=a345bf84eb9bf6b2, https://usegalaxy.org.au/published/workflow?id=a525966251895f8a, https://usegalaxy.org.au/published/workflow?id=ad10cd704268dcf0, https://usegalaxy.org.au/published/workflow?id=b19fa49405117317, https://usegalaxy.org.au/published/workflow?id=b1f45d2f8d8343e9, https://usegalaxy.org.au/published/workflow?id=b5131e3a0ef2676e, https://usegalaxy.org.au/published/workflow?id=b5403e4fff2961f9, https://usegalaxy.org.au/published/workflow?id=b59fada651043735, https://usegalaxy.org.au/published/workflow?id=b6bad1f167257248, https://usegalaxy.org.au/published/workflow?id=b9faf6a3f48fc11b, https://usegalaxy.org.au/published/workflow?id=be96bb47b6d6435a, https://usegalaxy.org.au/published/workflow?id=c43744d7dd9c0c0b, https://usegalaxy.org.au/published/workflow?id=c4d23297f6fbf5f3, https://usegalaxy.org.au/published/workflow?id=c6a71573a299eb3a, https://usegalaxy.org.au/published/workflow?id=c7e1f666a3366823, https://usegalaxy.org.au/published/workflow?id=c7f2d6490128dc3c, https://usegalaxy.org.au/published/workflow?id=c9b7de574fb2f2d5, https://usegalaxy.org.au/published/workflow?id=d298edda13b382e7, https://usegalaxy.org.au/published/workflow?id=d42d7b1aa47e870f, https://usegalaxy.org.au/published/workflow?id=d78879cfaa5948b3, https://usegalaxy.org.au/published/workflow?id=d7f3fc810261e419, https://usegalaxy.org.au/published/workflow?id=d832905fbaf2ca31, https://usegalaxy.org.au/published/workflow?id=d9571b3dfef34e5e, https://usegalaxy.org.au/published/workflow?id=d9c181fd954b629a, https://usegalaxy.org.au/published/workflow?id=df4e186069552649, https://usegalaxy.org.au/published/workflow?id=e031af90daa2cadf, https://usegalaxy.org.au/published/workflow?id=e28ca83f14f4ede5, https://usegalaxy.org.au/published/workflow?id=e5f1ce106ea7af12, https://usegalaxy.org.au/published/workflow?id=ea687c5613fcce27, https://usegalaxy.org.au/published/workflow?id=ef8c22c2525063a2, https://usegalaxy.org.au/published/workflow?id=f2541030f6d70b59, https://usegalaxy.org.au/published/workflow?id=f7a866159f1eaecf, https://usegalaxy.org.au/published/workflow?id=f83a3cda33131fb8, https://usegalaxy.org.au/published/workflow?id=fc3c83b11f527977, https://usegalaxy.org.au/published/workflow?id=fc664a96d8099652, https://usegalaxy.org.au/published/workflow?id=fe1e7fa0bc437ed8, https://usegalaxy.org.au/published/workflow?id=fff2e49bc90e38ca, https://usegalaxy.org/published/workflow?id=00596e862edb23f2, https://usegalaxy.org/published/workflow?id=0602c14704dbe1ff, https://usegalaxy.org/published/workflow?id=07e873a6d75dfe9f, https://usegalaxy.org/published/workflow?id=0993daadf2db5f51, https://usegalaxy.org/published/workflow?id=0e30324072570abb, https://usegalaxy.org/published/workflow?id=109092d8dbda4398, https://usegalaxy.org/published/workflow?id=1245e9435cbc3985, https://usegalaxy.org/published/workflow?id=137d0f05a71d7acb, https://usegalaxy.org/published/workflow?id=1c7af480caf150bd, https://usegalaxy.org/published/workflow?id=1eec2586d1ff908b, https://usegalaxy.org/published/workflow?id=25a9f0693ff6d327, https://usegalaxy.org/published/workflow?id=284150bb1f296378, https://usegalaxy.org/published/workflow?id=2c5c627c08563398, https://usegalaxy.org/published/workflow?id=2d994cc02ec05ede, https://usegalaxy.org/published/workflow?id=305fb9f84b053713, https://usegalaxy.org/published/workflow?id=3ba73f679558017c, https://usegalaxy.org/published/workflow?id=3cf46c167ccb374f, https://usegalaxy.org/published/workflow?id=3db4eb1ecb588d44, https://usegalaxy.org/published/workflow?id=3f252e077e0bcce5, https://usegalaxy.org/published/workflow?id=3f7550d5d2bf0b96, https://usegalaxy.org/published/workflow?id=408c799822cbf6ea, https://usegalaxy.org/published/workflow?id=4161cfd67ed94b5d, https://usegalaxy.org/published/workflow?id=435f0340bf00c940, https://usegalaxy.org/published/workflow?id=4403ca7fe447a5d7, https://usegalaxy.org/published/workflow?id=44781c8b0bd667ed, https://usegalaxy.org/published/workflow?id=484f8fcf21d67376, https://usegalaxy.org/published/workflow?id=498a8c6329ca5728, https://usegalaxy.org/published/workflow?id=4a0ad87aa5aa558a, https://usegalaxy.org/published/workflow?id=4bf73376661baafd, https://usegalaxy.org/published/workflow?id=4d1279007d0137fa, https://usegalaxy.org/published/workflow?id=4d5cc5010537f172, https://usegalaxy.org/published/workflow?id=5039ae2c81d0aba8, https://usegalaxy.org/published/workflow?id=524602abe59b18f7, https://usegalaxy.org/published/workflow?id=547f86db467bf620, https://usegalaxy.org/published/workflow?id=562bcaa9b4d2e8a4, https://usegalaxy.org/published/workflow?id=578e634b63923e9b, https://usegalaxy.org/published/workflow?id=57e6345fda3e2682, https://usegalaxy.org/published/workflow?id=597c4d54f8a8ebe7, https://usegalaxy.org/published/workflow?id=5cd56232e873beab, https://usegalaxy.org/published/workflow?id=5dfec2a3ae5fa765, https://usegalaxy.org/published/workflow?id=653d553ac2fa51c8, https://usegalaxy.org/published/workflow?id=67252039bce060b2, https://usegalaxy.org/published/workflow?id=683e725f672b7d9b, https://usegalaxy.org/published/workflow?id=69d8863d40524120, https://usegalaxy.org/published/workflow?id=6b511b47176893e3, https://usegalaxy.org/published/workflow?id=7072731b51e16923, https://usegalaxy.org/published/workflow?id=7dba74c915f63dc1, https://usegalaxy.org/published/workflow?id=7dd6e4150bb81ad1, https://usegalaxy.org/published/workflow?id=7f6900852a81d121, https://usegalaxy.org/published/workflow?id=80ac6adead9f97d8, https://usegalaxy.org/published/workflow?id=8111bb9bb26bb665, https://usegalaxy.org/published/workflow?id=81f1f0fd6d9235fd, https://usegalaxy.org/published/workflow?id=84161f4f8bf77474, https://usegalaxy.org/published/workflow?id=864da6858d34df08, https://usegalaxy.org/published/workflow?id=8a350e0d96e937a5, https://usegalaxy.org/published/workflow?id=8cfbaa14cff5a633, https://usegalaxy.org/published/workflow?id=8ebc2b1ae24ed884, https://usegalaxy.org/published/workflow?id=8f7c9461910b08f1, https://usegalaxy.org/published/workflow?id=90f5b512997c70f8, https://usegalaxy.org/published/workflow?id=932b3bc10186f73e, https://usegalaxy.org/published/workflow?id=936ed7447c8e0a2d, https://usegalaxy.org/published/workflow?id=93c1fef4472aa7a8, https://usegalaxy.org/published/workflow?id=961dac8100c6c70c, https://usegalaxy.org/published/workflow?id=9938a81fb6911412, https://usegalaxy.org/published/workflow?id=9ac1e4efe310b3b3, https://usegalaxy.org/published/workflow?id=9ba6da7dae465d3e, https://usegalaxy.org/published/workflow?id=9ddfdc9d6cacea88, https://usegalaxy.org/published/workflow?id=9e5c5ddb60a24889, https://usegalaxy.org/published/workflow?id=a8e0f77cd6ef49f9, https://usegalaxy.org/published/workflow?id=ab0be4897d5355ac, https://usegalaxy.org/published/workflow?id=acada5044d64a30a, https://usegalaxy.org/published/workflow?id=b11c6268704dcc1b, https://usegalaxy.org/published/workflow?id=b5644a5dcf237ef9, https://usegalaxy.org/published/workflow?id=b661fb850c4b4b01, https://usegalaxy.org/published/workflow?id=b6d535f210efa18e, https://usegalaxy.org/published/workflow?id=b76a9ce44ea019f1, https://usegalaxy.org/published/workflow?id=bb1cb6abfe84a646, https://usegalaxy.org/published/workflow?id=bbc894a79eb0e769, https://usegalaxy.org/published/workflow?id=bc72dcbca89ca35d, https://usegalaxy.org/published/workflow?id=bd6bfd338ffb9637, https://usegalaxy.org/published/workflow?id=bfd2e7e63d3e6856, https://usegalaxy.org/published/workflow?id=bff206b4625375bc, https://usegalaxy.org/published/workflow?id=c0a9a2d53b8660de, https://usegalaxy.org/published/workflow?id=c454c930ccfb0d3f, https://usegalaxy.org/published/workflow?id=c65bec6c332bf740, https://usegalaxy.org/published/workflow?id=c8d74054c1820087, https://usegalaxy.org/published/workflow?id=c983ea8a32662cd6, https://usegalaxy.org/published/workflow?id=cce88bc57b180d09, https://usegalaxy.org/published/workflow?id=cdcf91a8ddb5dcd1, https://usegalaxy.org/published/workflow?id=ce04658f05029be2, https://usegalaxy.org/published/workflow?id=d08135fa13d1745e, https://usegalaxy.org/published/workflow?id=d36765ac28c11e1a, https://usegalaxy.org/published/workflow?id=d3f35215e6f45d79, https://usegalaxy.org/published/workflow?id=d4c790a45978bb9e, https://usegalaxy.org/published/workflow?id=daaded719dcfd53b, https://usegalaxy.org/published/workflow?id=dae5d5e5df5c4dd7, https://usegalaxy.org/published/workflow?id=e2ab3d3219a648a7, https://usegalaxy.org/published/workflow?id=e4dd39237df58884, https://usegalaxy.org/published/workflow?id=f1a54b10d4afce8f, https://usegalaxy.org/published/workflow?id=f4d623a19b32370e, https://usegalaxy.org/published/workflow?id=f60027545d40b82d, https://usegalaxy.org/published/workflow?id=fa26fd51caea53c7, https://usegalaxy.org/published/workflow?id=fb01e1a49793c8be, https://usegalaxy.org/published/workflow?id=fc7b14aafa0b421e, https://usegalaxy.org/published/workflow?id=fd40e50c9f1545ce, https://workflowhub.eu/workflows/1043?version=12, https://workflowhub.eu/workflows/1062?version=1, https://workflowhub.eu/workflows/1114?version=1, https://workflowhub.eu/workflows/1162?version=2, https://workflowhub.eu/workflows/1390?version=2, https://workflowhub.eu/workflows/1477?version=1, https://workflowhub.eu/workflows/1495?version=1, https://workflowhub.eu/workflows/1595?version=1, https://workflowhub.eu/workflows/1602?version=2, https://workflowhub.eu/workflows/1612?version=1, https://workflowhub.eu/workflows/1613?version=2, https://workflowhub.eu/workflows/1616?version=1, https://workflowhub.eu/workflows/1617?version=1, https://workflowhub.eu/workflows/1634?version=1, https://workflowhub.eu/workflows/1641?version=1, https://workflowhub.eu/workflows/221?version=3, https://workflowhub.eu/workflows/225?version=1, https://workflowhub.eu/workflows/406?version=1, https://workflowhub.eu/workflows/52?version=1, https://workflowhub.eu/workflows/605?version=1, https://workflowhub.eu/workflows/612?version=25, https://workflowhub.eu/workflows/641?version=32, https://workflowhub.eu/workflows/642?version=25, https://workflowhub.eu/workflows/750?version=3, https://workflowhub.eu/workflows/788?version=1, https://workflowhub.eu/workflows/789?version=1</td>
      <td>assembly/ERGA-post-assembly-QC, assembly/assembly-with-preprocessing, assembly/chloroplast-assembly, assembly/debruijn-graph-assembly, assembly/largegenome, assembly/metagenomics-assembly, assembly/mrsa-illumina, assembly/mrsa-nanopore, microbiome/metagenomics-assembly, microbiome/pathogen-detection-from-nanopore-foodborne-data, microbiome/plasmid-metagenomics-nanopore</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>bayescan</td>
      <td>BayeScan</td>
      <td>Detecting natural selection from population-based genetic data</td>
      <td>2017-03-09</td>
      <td>http://cmpg.unibe.ch/software/BayeScan/index.html</td>
      <td>2.1</td>
      <td>bayescan</td>
      <td>2.0.1</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Statistical inference</td>
      <td>Statistical inference</td>
      <td>Genetics, Evolutionary biology, Statistics and probability, DNA polymorphism</td>
      <td>Genetics, Evolutionary biology, Statistics and probability, DNA polymorphism</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bayescan</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bayescan</td>
      <td>bayescan</td>
      <td>BayeScan</td>
      <td>BAYEsian genome SCAN for outliers, aims at identifying candidate loci under natural selection from genetic data, using differences in allele frequencies between populations. It is based on the multinomial-Dirichlet model.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>28</td>
      <td>35</td>
      <td>109</td>
      <td>136</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>136</td>
      <td>109</td>
      <td>35</td>
      <td>28</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>bbtools</td>
      <td>bbtools_bbduk, bbtools_bbmap, bbtools_bbmerge, bbtools_bbnorm, bbtools_callvariants, bbtools_tadpole</td>
      <td>BBTools is a suite of fast, multithreaded bioinformatics tools designed for analysis of DNA and RNA sequence data.BBTools can handle common sequencing file formats such as fastq, fasta, sam, scarf, fasta+qual, compressed or raw,with autodetection of quality encoding and interleaving. It is written in Java and works on any platform supportingJava, including Linux, MacOS, and Microsoft Windows and Linux; there are no dependencies other than Java (version7 or higher). Program descriptions and options are shown when running the shell scripts with no parameters.</td>
      <td>2021-10-04</td>
      <td>https://jgi.doe.gov/data-and-tools/bbtools/</td>
      <td>39.08</td>
      <td>bbmap</td>
      <td>39.52</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>RNA-Seq analysis, Sequence trimming, Read mapping, Sequence contamination filtering, Read binning, Sequence alignment</td>
      <td>RNA-Seq analysis, Sequence trimming, Read mapping, Sequence contamination filtering, Read binning, Sequence alignment</td>
      <td>Sequencing, RNA splicing, Whole genome sequencing, Phylogenetics, Metagenomics, RNA-Seq</td>
      <td>RNA splicing, Whole genome sequencing, Phylogenetics, Metagenomics, RNA-Seq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bbtools</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bbtools</td>
      <td>bbmap</td>
      <td>BBMap</td>
      <td>BBMap is a fast splice-aware aligner for RNA and DNA.  It is faster than almost all short-read aligners, yet retains unrivaled sensitivity and specificity, particularly for reads with many errors and indels.</td>
      <td></td>
      <td>6</td>
      <td>3</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>413</td>
      <td>413</td>
      <td>25287</td>
      <td>25287</td>
      <td>562</td>
      <td>562</td>
      <td>10487</td>
      <td>10487</td>
      <td>172</td>
      <td>172</td>
      <td>3537</td>
      <td>3537</td>
      <td>5</td>
      <td>5</td>
      <td>134</td>
      <td>134</td>
      <td>39445</td>
      <td>39445</td>
      <td>1152</td>
      <td>1152</td>
      <td>https://usegalaxy.eu/published/workflow?id=d967194ca51ff11e, https://usegalaxy.org.au/published/workflow?id=25f65a29ce3c3ee0, https://workflowhub.eu/workflows/407?version=1, https://workflowhub.eu/workflows/519?version=1</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>bigscape</td>
      <td>bigscape</td>
      <td>Construct sequence similarity networks of BGCs and groups them into GCF</td>
      <td>2024-02-18</td>
      <td>https://github.com/medema-group/BiG-SCAPE</td>
      <td>1.1.9</td>
      <td>bigscape</td>
      <td>2.0.0</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Clustering, Global alignment, Fold recognition</td>
      <td>Clustering, Global alignment, Fold recognition</td>
      <td>Phylogeny, Microbial ecology, Mapping, Metabolomics, Bioinformatics, Gene and protein families</td>
      <td>Phylogeny, Microbial ecology, Mapping, Metabolomics, Bioinformatics, Gene and protein families</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bigscape</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bigscape</td>
      <td>BiG-SCAPE</td>
      <td>BiG-SCAPE</td>
      <td>A computational framework to explore large-scale biosynthetic diversity.BiG-SCAPE (Biosynthetic Gene Similarity Clustering and Prospecting Engine) is a software package, written in Python, that constructs sequence similarity networks of Biosynthetic Gene Clusters (BGCs) and groups them into Gene Cluster Families (GCFs). BiG-SCAPE does this by rapidly calculating a distance matrix between gene clusters based on a comparison of their protein domain content, order, copy number and sequence identity.It defines a distance metric between Gene Clusters using a combination of three indices (Jaccard Index of domain types, Domain Sequence Similarity the Adjacency...BiG-SCAPE and CORASON provide a set of tools to explore the diversity of biosynthetic gene clusters (BGCs) across large numbers of genomes, by constructing BGC sequence similarity networks, grouping BGCs into gene cluster families, and exploring gene cluster diversity linked to enzyme phylogenies.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>95</td>
      <td>95</td>
      <td>2449</td>
      <td>2449</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2449</td>
      <td>2449</td>
      <td>95</td>
      <td>95</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>binette</td>
      <td>binette</td>
      <td>Binning refinement tool</td>
      <td>2025-01-14</td>
      <td>https://github.com/genotoul-bioinfo/Binette</td>
      <td>1.2.1</td>
      <td>binette</td>
      <td>1.2.1</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Read binning</td>
      <td>Read binning</td>
      <td>Metagenomics</td>
      <td>Metagenomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/binette</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/binette</td>
      <td>binette</td>
      <td>Binette</td>
      <td>Binette is a fast and accurate binning refinement tool to constructs high quality MAGs from the output of multiple binning tools.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>11</td>
      <td>11</td>
      <td>359</td>
      <td>359</td>
      <td>8</td>
      <td>8</td>
      <td>17</td>
      <td>17</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>49</td>
      <td>49</td>
      <td>425</td>
      <td>425</td>
      <td>20</td>
      <td>20</td>
      <td>https://usegalaxy.eu/published/workflow?id=00f15824c7a9b196, https://usegalaxy.eu/published/workflow?id=7ecd92cb382d8168, https://usegalaxy.eu/published/workflow?id=ce5c9287a67e6085, https://usegalaxy.org/published/workflow?id=7251c6e0dad8f927, https://workflowhub.eu/workflows/1352?version=4, https://workflowhub.eu/workflows/2028?version=1</td>
      <td>microbiome/metagenomics-binning</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>binning_refiner</td>
      <td>bin_refiner</td>
      <td>Reconciles the outputs of different binning programs with the aim to improve the quality of genome bins,especially with respect to contamination levels.</td>
      <td>2022-02-18</td>
      <td>https://github.com/songweizhi/Binning_refiner</td>
      <td>1.4.3</td>
      <td>binning_refiner</td>
      <td>1.4.3</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Read binning, Sequence clustering</td>
      <td>Read binning, Sequence clustering</td>
      <td>Metagenomics, Sequence assembly, Microbial ecology</td>
      <td>Metagenomics, Sequence assembly, Microbial ecology</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/binning_refiner</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/binning_refiner</td>
      <td>binning_refiner</td>
      <td>Binning_refiner</td>
      <td>Improving genome bins through the combination of different binning programs</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>64</td>
      <td>64</td>
      <td>236</td>
      <td>236</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>236</td>
      <td>236</td>
      <td>64</td>
      <td>64</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>bio_hansel</td>
      <td>bio_hansel</td>
      <td>Heidelberg and Enteritidis SNP Elucidation</td>
      <td>2017-09-29</td>
      <td>https://github.com/phac-nml/bio_hansel</td>
      <td>2.6.1</td>
      <td>bio_hansel</td>
      <td>2.6.1</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Genotyping, SNP detection, Genome assembly</td>
      <td>Genotyping, SNP detection, Genome assembly</td>
      <td>Whole genome sequencing, DNA polymorphism, Genotype and phenotype, Infectious disease, Agricultural science</td>
      <td>Whole genome sequencing, DNA polymorphism, Genotype and phenotype, Infectious disease, Agricultural science</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hansel</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hansel</td>
      <td>Biohansel</td>
      <td>BioHansel</td>
      <td>BioHansel is a tool for performing high-resolution genotyping of bacterial isolates by identifying phylogenetically informative single nucleotide polymorphisms (SNPs), also known as canonical SNPs, in whole genome sequencing (WGS) data. The application uses a fast k-mer matching algorithm to map pathogen WGS data to canonical SNPs contained in hierarchically structured schemas and assigns genotypes based on the detected SNP profile.</td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>36</td>
      <td>39</td>
      <td>312</td>
      <td>397</td>
      <td>57</td>
      <td>63</td>
      <td>538</td>
      <td>592</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>989</td>
      <td>850</td>
      <td>102</td>
      <td>93</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>biohansel</td>
      <td>biohansel</td>
      <td>Heidelberg and Enteritidis SNP Elucidation</td>
      <td>2018-08-01</td>
      <td>https://github.com/phac-nml/biohansel</td>
      <td>2.4.0</td>
      <td>bio_hansel</td>
      <td>2.6.1</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>nml</td>
      <td>https://github.com/phac-nml/biohansel</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/biohansel</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>biomformat</td>
      <td>biom_add_metadata, biom_convert, biom_from_uc, biom_normalize_table, biom_subset_table, biom_summarize_table</td>
      <td>The biom-format package provides a command line interface and Python API for working with BIOM files.</td>
      <td>2016-06-07</td>
      <td>https://github.com/biocore/biom-format</td>
      <td>2.1.15</td>
      <td>biom-format</td>
      <td>2.1.7</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Formatting</td>
      <td>Formatting</td>
      <td>Laboratory information management, Sequence analysis</td>
      <td>Laboratory information management, Sequence analysis</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/biom_format</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/biom_format</td>
      <td>biomformat</td>
      <td>biomformat</td>
      <td>This package includes basic tools for reading biom-format files, accessing and subsetting data tables from a biom object, as well as limited support for writing a biom-object back to a biom-format file. The design of this API is intended to match the python API and other tools included with the biom-format project, but with a decidedly "R flavor" that should be familiar to R users. This includes S4 classes and methods, as well as extensions of common core functions/methods.</td>
      <td></td>
      <td>2</td>
      <td>2</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>342</td>
      <td>356</td>
      <td>16688</td>
      <td>16852</td>
      <td>374</td>
      <td>593</td>
      <td>2736</td>
      <td>5533</td>
      <td>166</td>
      <td>235</td>
      <td>2283</td>
      <td>2833</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>25218</td>
      <td>21707</td>
      <td>1184</td>
      <td>882</td>
      <td>https://usegalaxy.eu/published/workflow?id=08e25b733cd796e6, https://usegalaxy.eu/published/workflow?id=437704898229dfb6, https://usegalaxy.eu/published/workflow?id=466bdd8ba7b67264, https://usegalaxy.eu/published/workflow?id=8f747dbff3216993, https://usegalaxy.eu/published/workflow?id=e14b81a8c0b1b7a4, https://usegalaxy.eu/published/workflow?id=fb806b88850f3427, https://usegalaxy.org.au/published/workflow?id=14e80b7f9f68ec2a, https://usegalaxy.org.au/published/workflow?id=20d472fc5563a640, https://usegalaxy.org.au/published/workflow?id=4f837b7f90d5ebac, https://usegalaxy.org.au/published/workflow?id=869440d35a618836, https://usegalaxy.org.au/published/workflow?id=ca6f29110df8dadc, https://usegalaxy.org/published/workflow?id=45f959d5bf505b15, https://usegalaxy.org/published/workflow?id=8151be18aab3a3c6, https://workflowhub.eu/workflows/1270?version=2, https://workflowhub.eu/workflows/1273?version=2, https://workflowhub.eu/workflows/142?version=1, https://workflowhub.eu/workflows/1842?version=1, https://workflowhub.eu/workflows/1856?version=1, https://workflowhub.eu/workflows/232?version=1, https://workflowhub.eu/workflows/233?version=1</td>
      <td>microbiome/mgnify-amplicon</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>biotradis</td>
      <td>bacteria_tradis, tradis_essentiality, tradis_gene_insert_sites</td>
      <td>Bio-Tradis is a tool suite dedicated to essentiality analyses with TraDis data.</td>
      <td>2020-01-17</td>
      <td>https://www.sanger.ac.uk/science/tools/bio-tradis</td>
      <td>1.4.5</td>
      <td>biotradis</td>
      <td>1.4.5</td>
      <td>Up-to-date</td>
      <td>Genome annotation</td>
      <td>Sequence analysis</td>
      <td>Sequence analysis</td>
      <td>Mobile genetic elements, Workflows</td>
      <td>Mobile genetic elements, Workflows</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/biotradis</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/biotradis</td>
      <td>biotradis</td>
      <td>biotradis</td>
      <td>The Bio::TraDIS pipeline provides software utilities for the processing, mapping, and analysis of transposon insertion sequencing data. The pipeline was designed with the data from the TraDIS sequencing protocol in mind, but should work with a variety of transposon insertion sequencing protocols as long as they produce data in the expected format.</td>
      <td></td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>165</td>
      <td>176</td>
      <td>6369</td>
      <td>6769</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6769</td>
      <td>6369</td>
      <td>176</td>
      <td>165</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>blast2go</td>
      <td>blast2go</td>
      <td>Maps BLAST results to GO annotation terms</td>
      <td>2013-09-23</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/blast2go</td>
      <td>0.0.11</td>
      <td>b2g4pipe</td>
      <td></td>
      <td>To update</td>
      <td>Ontology Manipulation, Sequence Analysis</td>
      <td>Gene functional annotation</td>
      <td>Gene functional annotation</td>
      <td>Nucleic acid sites, features and motifs</td>
      <td>Nucleic acid sites, features and motifs</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/blast2go</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/blast2go</td>
      <td>Blast2GO</td>
      <td>Blast2GO</td>
      <td>Bioinformatics platform for high-quality functional annotation and analysis of genomic datasets. It allows analyzing and visualizing newly sequenced genomes by combining state-of-the-art methodologies, standard resources and algorithms. It allows to gain biological insights fast and easy even for completely novel genomes. Perform out-of-the-box the entire workflow of functional annotation of your transcriptomic datasets including its analysis and biological interpretation.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>205</td>
      <td>303</td>
      <td>851</td>
      <td>1255</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>10</td>
      <td>1265</td>
      <td>851</td>
      <td>304</td>
      <td>205</td>
      <td>https://usegalaxy.eu/published/workflow?id=33312e5643279e8a</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>blast_rbh</td>
      <td>blast_reciprocal_best_hits</td>
      <td>BLAST Reciprocal Best Hits (RBH) from two FASTA files</td>
      <td>2014-05-15</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/blast_rbh</td>
      <td>0.3.0</td>
      <td>biopython</td>
      <td>1.70</td>
      <td>To update</td>
      <td>Fasta Manipulation, Sequence Analysis</td>
      <td>Gene functional annotation, Nucleic acid sequence analysis</td>
      <td>Gene functional annotation, Nucleic acid sequence analysis</td>
      <td>Nucleic acid sites, features and motifs</td>
      <td>Nucleic acid sites, features and motifs</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/blast_rbh</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/blast_rbh</td>
      <td>blast_rbh</td>
      <td>blast_rbh</td>
      <td>This tool is a short Python script to run reciprocal BLAST searches on a pair of sequence files, and extract the reciprocal best hits.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>413</td>
      <td>502</td>
      <td>25833</td>
      <td>37031</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>37031</td>
      <td>25833</td>
      <td>502</td>
      <td>413</td>
      <td>https://usegalaxy.eu/published/workflow?id=7f6950394a8c9c4a</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>blastxml_to_top_descr</td>
      <td>blastxml_to_top_descr</td>
      <td>Make table of top BLAST match descriptions</td>
      <td>2013-09-23</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/blastxml_to_top_descr</td>
      <td>0.1.2</td>
      <td>python</td>
      <td></td>
      <td>To update</td>
      <td>Convert Formats, Sequence Analysis, Text Manipulation</td>
      <td>Sequence alignment, Sequence annotation</td>
      <td>Sequence alignment, Sequence annotation</td>
      <td>Sequence analysis</td>
      <td>Sequence analysis</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/blastxml_to_top_descr</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/blastxml_to_top_descr</td>
      <td>blastxml_to_top_descr</td>
      <td>blastxml_to_top_descr</td>
      <td>This tool is a short Python script to parse a BLAST XML file, and extract the identifiers with description for the top matches (by default the top 3), and output these as a simple tabular file along with the query identifiers.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>530</td>
      <td>653</td>
      <td>263613</td>
      <td>265902</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>265902</td>
      <td>263613</td>
      <td>653</td>
      <td>530</td>
      <td>https://usegalaxy.eu/published/workflow?id=33312e5643279e8a, https://usegalaxy.eu/published/workflow?id=8f747dbff3216993, https://usegalaxy.eu/published/workflow?id=bebcbd6f0ffa55c5, https://usegalaxy.eu/published/workflow?id=e14b81a8c0b1b7a4</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>bmtagger</td>
      <td>bmtagger</td>
      <td>remove contaminant reads</td>
      <td>2025-11-12</td>
      <td>https://ftp.ncbi.nlm.nih.gov/pub/agarwala/bmtagger/</td>
      <td>3.101</td>
      <td>bmtagger</td>
      <td>3.101</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Data filtering, Read pre-processing</td>
      <td>Read pre-processing</td>
      <td>Metagenomics</td>
      <td>Metagenomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bmtagger</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bmtagger</td>
      <td>bmtagger</td>
      <td>bmtagger</td>
      <td>Best Match Tagger for removing contaminant reads from metagenomics datasets</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>bracken</td>
      <td>est_abundance</td>
      <td>Bayesian Reestimation of Abundance with KrakEN</td>
      <td>2019-10-15</td>
      <td>https://github.com/jenniferlu717/Bracken/releases</td>
      <td>3.1</td>
      <td>bracken</td>
      <td>3.1</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis, Metagenomics</td>
      <td>Statistical calculation</td>
      <td>Statistical calculation</td>
      <td>Metagenomics, Microbial ecology</td>
      <td>Metagenomics, Microbial ecology</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bracken</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bracken</td>
      <td>bracken</td>
      <td>Bracken</td>
      <td>Bracken is a companion program to Kraken 1, KrakenUniq, or Kraken 2 While Kraken classifies reads to multiple levels in the taxonomic tree, Bracken allows estimation of abundance at a single level using those classifications (e.g. Bracken can estimate abundance of species within a sample).</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1714</td>
      <td>1714</td>
      <td>108307</td>
      <td>108307</td>
      <td>804</td>
      <td>804</td>
      <td>27569</td>
      <td>27569</td>
      <td>265</td>
      <td>265</td>
      <td>8873</td>
      <td>8873</td>
      <td>59</td>
      <td>59</td>
      <td>12432</td>
      <td>12432</td>
      <td>157181</td>
      <td>157181</td>
      <td>2842</td>
      <td>2842</td>
      <td>https://usegalaxy.eu/published/workflow?id=0a82eb8b44d9d502, https://usegalaxy.eu/published/workflow?id=0abd32dbc5207b88, https://usegalaxy.eu/published/workflow?id=0c47571d20535426, https://usegalaxy.eu/published/workflow?id=2fa2f67603772413, https://usegalaxy.eu/published/workflow?id=39d36a7ce0ef268d, https://usegalaxy.eu/published/workflow?id=47e14222073e7192, https://usegalaxy.eu/published/workflow?id=4f0daf8e9dd54127, https://usegalaxy.eu/published/workflow?id=6f878d8b43ef1521, https://usegalaxy.eu/published/workflow?id=7491883694fff308, https://usegalaxy.eu/published/workflow?id=884a4415b669690c, https://usegalaxy.eu/published/workflow?id=aafcfcb585c5ca75, https://usegalaxy.eu/published/workflow?id=b14845359b702444, https://usegalaxy.eu/published/workflow?id=bb63b5bb983fd49c, https://usegalaxy.eu/published/workflow?id=c14d2e8b3c191bc4, https://usegalaxy.eu/published/workflow?id=c930a54a30dcc7e3, https://usegalaxy.eu/published/workflow?id=cb2df493f5e334f7, https://usegalaxy.eu/published/workflow?id=ebb131f2a6172081, https://usegalaxy.fr/published/workflow?id=3d3b40398d1cd1ae, https://usegalaxy.org.au/published/workflow?id=92fcdd490831c6c5, https://usegalaxy.org.au/published/workflow?id=a335b9c9103b57ad, https://usegalaxy.org/published/workflow?id=1be5c982c2d36a6c, https://usegalaxy.org/published/workflow?id=2a782ad24271f570, https://usegalaxy.org/published/workflow?id=323c651ca583c9e5, https://usegalaxy.org/published/workflow?id=4f698eaeef30e364, https://usegalaxy.org/published/workflow?id=623fc80c7f91a33a, https://usegalaxy.org/published/workflow?id=9ba6da7dae465d3e, https://usegalaxy.org/published/workflow?id=9f68718f025bed30, https://usegalaxy.org/published/workflow?id=b6a795834009ce06, https://usegalaxy.org/published/workflow?id=bdd7011a908bc1aa, https://usegalaxy.org/published/workflow?id=de9bf4c1024f8d53, https://usegalaxy.org/published/workflow?id=dfd73857c0eb2a21, https://usegalaxy.org/published/workflow?id=f378960ee55f567e, https://workflowhub.eu/workflows/1052?version=11, https://workflowhub.eu/workflows/1199?version=2, https://workflowhub.eu/workflows/1470?version=2, https://workflowhub.eu/workflows/1644?version=1, https://workflowhub.eu/workflows/1674?version=2, https://workflowhub.eu/workflows/1881?version=2, https://workflowhub.eu/workflows/1882?version=3, https://workflowhub.eu/workflows/2043?version=1</td>
      <td>ecology/bacterial-isolate-species-contamination-checking, microbiome/taxonomic-profiling, sequence-analysis/quality-contamination-control</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>busco</td>
      <td>busco</td>
      <td>BUSCO assess genome and annotation completeness</td>
      <td>2017-01-12</td>
      <td>https://gitlab.com/ezlab/busco/-/releases</td>
      <td>5.8.0</td>
      <td>busco</td>
      <td>6.0.0</td>
      <td>To update</td>
      <td>Sequence Analysis, Genome annotation</td>
      <td>Sequence assembly validation</td>
      <td>Sequence assembly validation</td>
      <td>Sequence assembly, Genomics, Transcriptomics, Sequence analysis</td>
      <td>Sequence assembly, Transcriptomics, Sequence analysis</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/busco</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/busco</td>
      <td>busco</td>
      <td>BUSCO</td>
      <td>Provides measures for quantitative assessment of genome assembly, gene set, and transcriptome completeness based on evolutionarily informed expectations of gene content from near-universal single-copy orthologs.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>5636</td>
      <td>5838</td>
      <td>134029</td>
      <td>137202</td>
      <td>5194</td>
      <td>5212</td>
      <td>58403</td>
      <td>58497</td>
      <td>3096</td>
      <td>3288</td>
      <td>49227</td>
      <td>52147</td>
      <td>262</td>
      <td>262</td>
      <td>8860</td>
      <td>8866</td>
      <td>256712</td>
      <td>250519</td>
      <td>14600</td>
      <td>14188</td>
      <td>https://usegalaxy.cz/published/workflow?id=932f4901960a595a, https://usegalaxy.eu/published/workflow?id=002a51bdef00e14d, https://usegalaxy.eu/published/workflow?id=007ce6405d196df3, https://usegalaxy.eu/published/workflow?id=01fe4c590e608103, https://usegalaxy.eu/published/workflow?id=023a0ee62dd1f593, https://usegalaxy.eu/published/workflow?id=032a7281a86eba0d, https://usegalaxy.eu/published/workflow?id=03bb4c20a1f4b4de, https://usegalaxy.eu/published/workflow?id=03be9d62e8e4a129, https://usegalaxy.eu/published/workflow?id=0444c98a3ca2a3ca, https://usegalaxy.eu/published/workflow?id=04844dddcc4e8aa7, https://usegalaxy.eu/published/workflow?id=069d99391a69baf4, https://usegalaxy.eu/published/workflow?id=082bfd9b36e921e8, https://usegalaxy.eu/published/workflow?id=085c227c9a56fcae, https://usegalaxy.eu/published/workflow?id=085c3a4c89d137b5, https://usegalaxy.eu/published/workflow?id=089e5e83b1de0511, https://usegalaxy.eu/published/workflow?id=0a3ebee985c26060, https://usegalaxy.eu/published/workflow?id=0c3ce9feb83f7920, https://usegalaxy.eu/published/workflow?id=0cf3c2114ab65858, https://usegalaxy.eu/published/workflow?id=0f207a15b6f7fa10, https://usegalaxy.eu/published/workflow?id=1151052f4393852a, https://usegalaxy.eu/published/workflow?id=122da0054fd40adf, https://usegalaxy.eu/published/workflow?id=12b0bd336c76a874, https://usegalaxy.eu/published/workflow?id=1483534f49b826bf, https://usegalaxy.eu/published/workflow?id=165089132248a0b0, https://usegalaxy.eu/published/workflow?id=17b0a6d3379a4392, https://usegalaxy.eu/published/workflow?id=194a05d42687b524, https://usegalaxy.eu/published/workflow?id=19bb57f8cfa48bb9, https://usegalaxy.eu/published/workflow?id=1ac0a15758277730, https://usegalaxy.eu/published/workflow?id=1af73dc855d0a5ed, https://usegalaxy.eu/published/workflow?id=1bb084f1bc102734, https://usegalaxy.eu/published/workflow?id=1cba6eb4aff8dc61, https://usegalaxy.eu/published/workflow?id=1d1ae39386b5c318, https://usegalaxy.eu/published/workflow?id=1da7ac8eaf891f39, https://usegalaxy.eu/published/workflow?id=1e02a7292e9974b1, https://usegalaxy.eu/published/workflow?id=1e7f4ba6261f979c, https://usegalaxy.eu/published/workflow?id=1f2a1438e41188f2, https://usegalaxy.eu/published/workflow?id=21a3f72851f29925, https://usegalaxy.eu/published/workflow?id=2294ef9515bcf972, https://usegalaxy.eu/published/workflow?id=2671a5f85b2cccfe, https://usegalaxy.eu/published/workflow?id=26ad77bc07a7e512, https://usegalaxy.eu/published/workflow?id=270b6f82efab28b2, https://usegalaxy.eu/published/workflow?id=29c97d43eeb8ed63, https://usegalaxy.eu/published/workflow?id=2e418c5810012e5a, https://usegalaxy.eu/published/workflow?id=3010ecad1a342331, https://usegalaxy.eu/published/workflow?id=3069c17b52decab4, https://usegalaxy.eu/published/workflow?id=30b514cfb9cce524, https://usegalaxy.eu/published/workflow?id=3192f92ac4d389b8, https://usegalaxy.eu/published/workflow?id=31bc2a44fb89aa89, https://usegalaxy.eu/published/workflow?id=31f8c7c46207432f, https://usegalaxy.eu/published/workflow?id=327beb1edda305ff, https://usegalaxy.eu/published/workflow?id=342a5df69951e3d7, https://usegalaxy.eu/published/workflow?id=379642036b6f5a66, https://usegalaxy.eu/published/workflow?id=399048afb4acef3d, https://usegalaxy.eu/published/workflow?id=3a3e2d50fe7a9ee3, https://usegalaxy.eu/published/workflow?id=3b57fc81001667b5, https://usegalaxy.eu/published/workflow?id=3d604dad4f1f5b3d, https://usegalaxy.eu/published/workflow?id=3e255a5b7026d26e, https://usegalaxy.eu/published/workflow?id=3eb6663a66f8e69a, https://usegalaxy.eu/published/workflow?id=3f5414bb38b47fcf, https://usegalaxy.eu/published/workflow?id=4077e77b6af919c4, https://usegalaxy.eu/published/workflow?id=416141b245073406, https://usegalaxy.eu/published/workflow?id=44c92c04f1d14631, https://usegalaxy.eu/published/workflow?id=45ae834d842302c3, https://usegalaxy.eu/published/workflow?id=471eb8b4c54f897d, https://usegalaxy.eu/published/workflow?id=475ea16f943cc013, https://usegalaxy.eu/published/workflow?id=4856e3522b2ee059, https://usegalaxy.eu/published/workflow?id=4949751217c0e211, https://usegalaxy.eu/published/workflow?id=497b82d3b92dc076, https://usegalaxy.eu/published/workflow?id=4c3a9b574f37eebb, https://usegalaxy.eu/published/workflow?id=4d41ee83fc0040d4, https://usegalaxy.eu/published/workflow?id=5115299e77df8104, https://usegalaxy.eu/published/workflow?id=5395e76fa828c343, https://usegalaxy.eu/published/workflow?id=54590ccd11ffbf65, https://usegalaxy.eu/published/workflow?id=56523b41ff7eec76, https://usegalaxy.eu/published/workflow?id=57cc33ed64d71982, https://usegalaxy.eu/published/workflow?id=5ad406442ec21345, https://usegalaxy.eu/published/workflow?id=5bf6718e7d6158d2, https://usegalaxy.eu/published/workflow?id=5c51f26ef8b20633, https://usegalaxy.eu/published/workflow?id=5e1906290f5b4d8a, https://usegalaxy.eu/published/workflow?id=652433160b04b232, https://usegalaxy.eu/published/workflow?id=6550f72070b24082, https://usegalaxy.eu/published/workflow?id=6573f3b1606b7f09, https://usegalaxy.eu/published/workflow?id=6865f33cffca4922, https://usegalaxy.eu/published/workflow?id=6984990d0ac2e1e6, https://usegalaxy.eu/published/workflow?id=69cdd404592535ec, https://usegalaxy.eu/published/workflow?id=6c14b08a7ef7e79a, https://usegalaxy.eu/published/workflow?id=6ca20e710f4f3ffb, https://usegalaxy.eu/published/workflow?id=700fa3ab11a8f3e2, https://usegalaxy.eu/published/workflow?id=7137e45ca8dc6e4c, https://usegalaxy.eu/published/workflow?id=71a6d128a09210f0, https://usegalaxy.eu/published/workflow?id=7337d15d121596da, https://usegalaxy.eu/published/workflow?id=792b24fc23bd3071, https://usegalaxy.eu/published/workflow?id=79b5afb125e3c3da, https://usegalaxy.eu/published/workflow?id=7b4a9aebb09b51ca, https://usegalaxy.eu/published/workflow?id=7e383fe34031a9d0, https://usegalaxy.eu/published/workflow?id=7e68cf8a67efa8e9, https://usegalaxy.eu/published/workflow?id=83a1b8949d87e3da, https://usegalaxy.eu/published/workflow?id=84236d781f7dd2ad, https://usegalaxy.eu/published/workflow?id=847403b72dd346ff, https://usegalaxy.eu/published/workflow?id=88f80a31e3ff8663, https://usegalaxy.eu/published/workflow?id=8976041ef89e4548, https://usegalaxy.eu/published/workflow?id=8be366ad6a42408b, https://usegalaxy.eu/published/workflow?id=8f105de48f7f1d9d, https://usegalaxy.eu/published/workflow?id=915ffef5b14c871e, https://usegalaxy.eu/published/workflow?id=9207e74a5bf7d8e1, https://usegalaxy.eu/published/workflow?id=92760fc19ef120ce, https://usegalaxy.eu/published/workflow?id=92dafb3d1c06e074, https://usegalaxy.eu/published/workflow?id=95f420d8856bf725, https://usegalaxy.eu/published/workflow?id=9667bf36217083b7, https://usegalaxy.eu/published/workflow?id=9c0b67496dfe52b2, https://usegalaxy.eu/published/workflow?id=9d209bcdae1a475e, https://usegalaxy.eu/published/workflow?id=9f2d18f0d0896e8b, https://usegalaxy.eu/published/workflow?id=a2daeb488bee0152, https://usegalaxy.eu/published/workflow?id=a37d1e0db16cfb67, https://usegalaxy.eu/published/workflow?id=a3f082ff30357040, https://usegalaxy.eu/published/workflow?id=a53f78aa6866b811, https://usegalaxy.eu/published/workflow?id=a6dd62ab4b9a0725, https://usegalaxy.eu/published/workflow?id=a9c1dca5da7170dd, https://usegalaxy.eu/published/workflow?id=aa6dc57f5c67fb4b, https://usegalaxy.eu/published/workflow?id=ac6eb99f70d35999, https://usegalaxy.eu/published/workflow?id=ade6f36f6a035a73, https://usegalaxy.eu/published/workflow?id=ae74f60cd20d56bc, https://usegalaxy.eu/published/workflow?id=aea88bfca6d1d476, https://usegalaxy.eu/published/workflow?id=aef62f4d0bba003b, https://usegalaxy.eu/published/workflow?id=b211fceaecf41c9e, https://usegalaxy.eu/published/workflow?id=b287bb779cf686ef, https://usegalaxy.eu/published/workflow?id=b4948afacccfefdb, https://usegalaxy.eu/published/workflow?id=b5de94e9cde071ab, https://usegalaxy.eu/published/workflow?id=b627aee461e50756, https://usegalaxy.eu/published/workflow?id=b8baade87e733fce, https://usegalaxy.eu/published/workflow?id=bac84a7033c3423f, https://usegalaxy.eu/published/workflow?id=bc32937d505135ab, https://usegalaxy.eu/published/workflow?id=bc87309a8763d0b4, https://usegalaxy.eu/published/workflow?id=bc8ccd13a07bd9ec, https://usegalaxy.eu/published/workflow?id=bcf99cc0469db276, https://usegalaxy.eu/published/workflow?id=bef0a7d604f2ebef, https://usegalaxy.eu/published/workflow?id=bf48b0eb15e8ff75, https://usegalaxy.eu/published/workflow?id=c4c2e669b86cffbc, https://usegalaxy.eu/published/workflow?id=c508c00e81018dd8, https://usegalaxy.eu/published/workflow?id=c8f94a2e838fc0f1, https://usegalaxy.eu/published/workflow?id=c9a8fba4c56f5c83, https://usegalaxy.eu/published/workflow?id=ca39822372847670, https://usegalaxy.eu/published/workflow?id=cab2aa791a814d55, https://usegalaxy.eu/published/workflow?id=caba6d3d7db648e9, https://usegalaxy.eu/published/workflow?id=cf8591391a17a914, https://usegalaxy.eu/published/workflow?id=d02458f1d7d9c3f4, https://usegalaxy.eu/published/workflow?id=d172f927a489e9f8, https://usegalaxy.eu/published/workflow?id=d17d6029914d666b, https://usegalaxy.eu/published/workflow?id=d1bd5d8597c1d31d, https://usegalaxy.eu/published/workflow?id=d261bbd05889ce75, https://usegalaxy.eu/published/workflow?id=d649ec485c4b3994, https://usegalaxy.eu/published/workflow?id=d67299c34e63f3d9, https://usegalaxy.eu/published/workflow?id=d8492b49aed84d4f, https://usegalaxy.eu/published/workflow?id=d924fafd8e042952, https://usegalaxy.eu/published/workflow?id=dc27453c438577f6, https://usegalaxy.eu/published/workflow?id=de9cf573314f416c, https://usegalaxy.eu/published/workflow?id=dfe3b555856c7b44, https://usegalaxy.eu/published/workflow?id=e138640778b7414c, https://usegalaxy.eu/published/workflow?id=e1947b059ef131d2, https://usegalaxy.eu/published/workflow?id=e2f936f1aa43a629, https://usegalaxy.eu/published/workflow?id=e5becc6d66b2568c, https://usegalaxy.eu/published/workflow?id=e7aebe58411d97ed, https://usegalaxy.eu/published/workflow?id=e8eb80ee09159bac, https://usegalaxy.eu/published/workflow?id=e8f9e1d911683b22, https://usegalaxy.eu/published/workflow?id=e935d2c8ffb52588, https://usegalaxy.eu/published/workflow?id=ec79f14ab91cd933, https://usegalaxy.eu/published/workflow?id=f08b29e4cef76cd5, https://usegalaxy.eu/published/workflow?id=f1ca4289ee4f67bb, https://usegalaxy.eu/published/workflow?id=f1e2577699125607, https://usegalaxy.eu/published/workflow?id=f38745208494d4ab, https://usegalaxy.eu/published/workflow?id=f3e17c8dd80def27, https://usegalaxy.eu/published/workflow?id=f578a097b4339025, https://usegalaxy.eu/published/workflow?id=f63127675a07e00a, https://usegalaxy.eu/published/workflow?id=f6a625946dea60fa, https://usegalaxy.eu/published/workflow?id=f7cdaf862e6599c0, https://usegalaxy.eu/published/workflow?id=f882d14fc5dabbb9, https://usegalaxy.eu/published/workflow?id=f91245e580319e65, https://usegalaxy.eu/published/workflow?id=fb015e398373ea8a, https://usegalaxy.eu/published/workflow?id=fba1671d835b6dd3, https://usegalaxy.eu/published/workflow?id=fc25c1f8c6667c87, https://usegalaxy.eu/published/workflow?id=fd649ac4d4872c81, https://usegalaxy.eu/published/workflow?id=fe43cf59e48bf556, https://usegalaxy.eu/published/workflow?id=feeb1fa358ccf328, https://usegalaxy.fr/published/workflow?id=ae05ff8487f58fd6, https://usegalaxy.org.au/published/workflow?id=000c2f1884bc4687, https://usegalaxy.org.au/published/workflow?id=02c74095a7128c9d, https://usegalaxy.org.au/published/workflow?id=02eebe3d909206ab, https://usegalaxy.org.au/published/workflow?id=06564b19b0655c19, https://usegalaxy.org.au/published/workflow?id=0810be3d7d2b0322, https://usegalaxy.org.au/published/workflow?id=08fac3b4c128e597, https://usegalaxy.org.au/published/workflow?id=0a3fb089674de144, https://usegalaxy.org.au/published/workflow?id=0aa316f8c6cff444, https://usegalaxy.org.au/published/workflow?id=0af0ea50f1237952, https://usegalaxy.org.au/published/workflow?id=0c11e1ba5e8fa21b, https://usegalaxy.org.au/published/workflow?id=0cc8c8875df6bccb, https://usegalaxy.org.au/published/workflow?id=0dbb85f727302a72, https://usegalaxy.org.au/published/workflow?id=0df25e2dbd5e8ed5, https://usegalaxy.org.au/published/workflow?id=10355b2680374a03, https://usegalaxy.org.au/published/workflow?id=1905ec04d5841f09, https://usegalaxy.org.au/published/workflow?id=1ce2a725df3adb74, https://usegalaxy.org.au/published/workflow?id=1dcc1b25c9091fbf, https://usegalaxy.org.au/published/workflow?id=1eb3e1452c672480, https://usegalaxy.org.au/published/workflow?id=1f49c65625b9d2de, https://usegalaxy.org.au/published/workflow?id=2356584eac7fa234, https://usegalaxy.org.au/published/workflow?id=243c67fa4312855e, https://usegalaxy.org.au/published/workflow?id=2470cb61ba51288c, https://usegalaxy.org.au/published/workflow?id=269b42605a2a87d2, https://usegalaxy.org.au/published/workflow?id=29f0c07c8a24b542, https://usegalaxy.org.au/published/workflow?id=2d45d7138d8715b4, https://usegalaxy.org.au/published/workflow?id=319e9d693fac8dce, https://usegalaxy.org.au/published/workflow?id=328adfa7d90b1d54, https://usegalaxy.org.au/published/workflow?id=35f39173f0ac9179, https://usegalaxy.org.au/published/workflow?id=3604307ec2835628, https://usegalaxy.org.au/published/workflow?id=36f54eda8568355d, https://usegalaxy.org.au/published/workflow?id=371c9892d261cee2, https://usegalaxy.org.au/published/workflow?id=3a64c1031340d860, https://usegalaxy.org.au/published/workflow?id=3b6e9f03b088bf01, https://usegalaxy.org.au/published/workflow?id=3c30c822058187be, https://usegalaxy.org.au/published/workflow?id=3df7f45b7b388f8c, https://usegalaxy.org.au/published/workflow?id=4023bca282ca9315, https://usegalaxy.org.au/published/workflow?id=40666ceb68d101ac, https://usegalaxy.org.au/published/workflow?id=40740c32b9a550c9, https://usegalaxy.org.au/published/workflow?id=41282d51f9112552, https://usegalaxy.org.au/published/workflow?id=41795303a88d9e94, https://usegalaxy.org.au/published/workflow?id=4267bd1e4cfd1901, https://usegalaxy.org.au/published/workflow?id=43343fa1896f8278, https://usegalaxy.org.au/published/workflow?id=434127690544d0e9, https://usegalaxy.org.au/published/workflow?id=447d8daefdefafee, https://usegalaxy.org.au/published/workflow?id=455e68f185fec41b, https://usegalaxy.org.au/published/workflow?id=466809eb3392f7fb, https://usegalaxy.org.au/published/workflow?id=478c79365fc9677f, https://usegalaxy.org.au/published/workflow?id=47cd0a90609fd557, https://usegalaxy.org.au/published/workflow?id=49b253f75ab6f38f, https://usegalaxy.org.au/published/workflow?id=4a404aa3e9f29e5a, https://usegalaxy.org.au/published/workflow?id=4b5f80656fc94b88, https://usegalaxy.org.au/published/workflow?id=4cb69c84b73ea9dd, https://usegalaxy.org.au/published/workflow?id=4e161a8f5098cc49, https://usegalaxy.org.au/published/workflow?id=4e7930c7286c6e9e, https://usegalaxy.org.au/published/workflow?id=4ea1049371de111c, https://usegalaxy.org.au/published/workflow?id=4ebf1297793870e7, https://usegalaxy.org.au/published/workflow?id=51bad198f1fd1215, https://usegalaxy.org.au/published/workflow?id=543fb2d9431a7e13, https://usegalaxy.org.au/published/workflow?id=5550585ffc3d8d0e, https://usegalaxy.org.au/published/workflow?id=5b0c0e34394afed1, https://usegalaxy.org.au/published/workflow?id=5cd804644a34c8fd, https://usegalaxy.org.au/published/workflow?id=5f4f874e49515555, https://usegalaxy.org.au/published/workflow?id=60c0dec52cdca708, https://usegalaxy.org.au/published/workflow?id=63789b6f82172679, https://usegalaxy.org.au/published/workflow?id=68b2961117e62890, https://usegalaxy.org.au/published/workflow?id=6c1c11a81a2ab1b8, https://usegalaxy.org.au/published/workflow?id=6c27019d03dd49d8, https://usegalaxy.org.au/published/workflow?id=6c9129acb9d98afe, https://usegalaxy.org.au/published/workflow?id=6de8899c40a90b98, https://usegalaxy.org.au/published/workflow?id=6ec0031406b10635, https://usegalaxy.org.au/published/workflow?id=6ec5728eec9c26d7, https://usegalaxy.org.au/published/workflow?id=6f76c43e6d2d7f74, https://usegalaxy.org.au/published/workflow?id=7028436ef03171cc, https://usegalaxy.org.au/published/workflow?id=7168c01be9997823, https://usegalaxy.org.au/published/workflow?id=77b2fcf8c805ad5d, https://usegalaxy.org.au/published/workflow?id=7909da83448d8bf4, https://usegalaxy.org.au/published/workflow?id=79bf7b0f5e9052d7, https://usegalaxy.org.au/published/workflow?id=7a97e8ccc332c1e8, https://usegalaxy.org.au/published/workflow?id=7e40485d1ed45b7c, https://usegalaxy.org.au/published/workflow?id=7f37c5366bce5489, https://usegalaxy.org.au/published/workflow?id=81a5b96452c71a52, https://usegalaxy.org.au/published/workflow?id=82d0876385b5ac60, https://usegalaxy.org.au/published/workflow?id=868775ef66d174a5, https://usegalaxy.org.au/published/workflow?id=86eb0ea9dc230f04, https://usegalaxy.org.au/published/workflow?id=8926e592f2e60863, https://usegalaxy.org.au/published/workflow?id=8a40059343cf4b87, https://usegalaxy.org.au/published/workflow?id=8a6233015db6dd24, https://usegalaxy.org.au/published/workflow?id=8bf8c9929249f37d, https://usegalaxy.org.au/published/workflow?id=8dd86497bc15d981, https://usegalaxy.org.au/published/workflow?id=8e8a5cce9f3f2f67, https://usegalaxy.org.au/published/workflow?id=91ee716be8bfd508, https://usegalaxy.org.au/published/workflow?id=9642aadb83c4f752, https://usegalaxy.org.au/published/workflow?id=970dc0d8bd74707f, https://usegalaxy.org.au/published/workflow?id=9804030358c3f844, https://usegalaxy.org.au/published/workflow?id=98d3def7da131b4c, https://usegalaxy.org.au/published/workflow?id=98f660971010d11d, https://usegalaxy.org.au/published/workflow?id=9e0bae0505ee7c07, https://usegalaxy.org.au/published/workflow?id=a02c102e179596cb, https://usegalaxy.org.au/published/workflow?id=a1015b83baa0eb6d, https://usegalaxy.org.au/published/workflow?id=a209f43d8a3cc853, https://usegalaxy.org.au/published/workflow?id=a345bf84eb9bf6b2, https://usegalaxy.org.au/published/workflow?id=a40dadc943a0db1e, https://usegalaxy.org.au/published/workflow?id=a525966251895f8a, https://usegalaxy.org.au/published/workflow?id=a740543c4e9808f1, https://usegalaxy.org.au/published/workflow?id=ac8bdc5a0b2d70cd, https://usegalaxy.org.au/published/workflow?id=ad10cd704268dcf0, https://usegalaxy.org.au/published/workflow?id=ad556db35858e7ec, https://usegalaxy.org.au/published/workflow?id=ae2a397837b232b1, https://usegalaxy.org.au/published/workflow?id=ae6a03240680712a, https://usegalaxy.org.au/published/workflow?id=afab89bc468da8b3, https://usegalaxy.org.au/published/workflow?id=b19fa49405117317, https://usegalaxy.org.au/published/workflow?id=b1f45d2f8d8343e9, https://usegalaxy.org.au/published/workflow?id=b310af289ada88bc, https://usegalaxy.org.au/published/workflow?id=b5131e3a0ef2676e, https://usegalaxy.org.au/published/workflow?id=b53f82aaf32fbaa9, https://usegalaxy.org.au/published/workflow?id=b7aa22291322e505, https://usegalaxy.org.au/published/workflow?id=b9e89401313a0cca, https://usegalaxy.org.au/published/workflow?id=b9faf6a3f48fc11b, https://usegalaxy.org.au/published/workflow?id=be35847680078c90, https://usegalaxy.org.au/published/workflow?id=be96bb47b6d6435a, https://usegalaxy.org.au/published/workflow?id=c04a62f528a69e79, https://usegalaxy.org.au/published/workflow?id=c43744d7dd9c0c0b, https://usegalaxy.org.au/published/workflow?id=c556c2270200801d, https://usegalaxy.org.au/published/workflow?id=c6a71573a299eb3a, https://usegalaxy.org.au/published/workflow?id=c7e1f666a3366823, https://usegalaxy.org.au/published/workflow?id=c7f2d6490128dc3c, https://usegalaxy.org.au/published/workflow?id=c853aa707fcddf4e, https://usegalaxy.org.au/published/workflow?id=cac0b8a09d59044b, https://usegalaxy.org.au/published/workflow?id=cbf022e349d726d9, https://usegalaxy.org.au/published/workflow?id=d06665e4bc7fff6d, https://usegalaxy.org.au/published/workflow?id=d298edda13b382e7, https://usegalaxy.org.au/published/workflow?id=d42d7b1aa47e870f, https://usegalaxy.org.au/published/workflow?id=d678e57c7f1baf5d, https://usegalaxy.org.au/published/workflow?id=d78879cfaa5948b3, https://usegalaxy.org.au/published/workflow?id=d7f3fc810261e419, https://usegalaxy.org.au/published/workflow?id=d83d5ecf31c20490, https://usegalaxy.org.au/published/workflow?id=d957bfeb3cd03db0, https://usegalaxy.org.au/published/workflow?id=d9c181fd954b629a, https://usegalaxy.org.au/published/workflow?id=dd9272826e6dd98d, https://usegalaxy.org.au/published/workflow?id=df11f09771e73e04, https://usegalaxy.org.au/published/workflow?id=df4e186069552649, https://usegalaxy.org.au/published/workflow?id=e031af90daa2cadf, https://usegalaxy.org.au/published/workflow?id=e07866576cc2f467, https://usegalaxy.org.au/published/workflow?id=e263fcb6c98e231b, https://usegalaxy.org.au/published/workflow?id=e28ca83f14f4ede5, https://usegalaxy.org.au/published/workflow?id=e5f1ce106ea7af12, https://usegalaxy.org.au/published/workflow?id=ea687c5613fcce27, https://usegalaxy.org.au/published/workflow?id=ea98be0f99aaceb9, https://usegalaxy.org.au/published/workflow?id=ec7efff3d3249177, https://usegalaxy.org.au/published/workflow?id=ecfb8a08e5a40e15, https://usegalaxy.org.au/published/workflow?id=f06370394b061e08, https://usegalaxy.org.au/published/workflow?id=f38f610e0d95915f, https://usegalaxy.org.au/published/workflow?id=f53d8ade897b3f46, https://usegalaxy.org.au/published/workflow?id=f7a866159f1eaecf, https://usegalaxy.org.au/published/workflow?id=f895914e322c8b77, https://usegalaxy.org.au/published/workflow?id=fa37223950a91f34, https://usegalaxy.org.au/published/workflow?id=fdec6e2fcd5e2cf5, https://usegalaxy.org.au/published/workflow?id=fe1e7fa0bc437ed8, https://usegalaxy.org.au/published/workflow?id=fea0e6c02c2f6274, https://usegalaxy.org.au/published/workflow?id=fff2e49bc90e38ca, https://usegalaxy.org/published/workflow?id=07e873a6d75dfe9f, https://usegalaxy.org/published/workflow?id=0993daadf2db5f51, https://usegalaxy.org/published/workflow?id=10d5e195501603bc, https://usegalaxy.org/published/workflow?id=1245e9435cbc3985, https://usegalaxy.org/published/workflow?id=137d0f05a71d7acb, https://usegalaxy.org/published/workflow?id=155839aeb6340f54, https://usegalaxy.org/published/workflow?id=15cca0e249f40aea, https://usegalaxy.org/published/workflow?id=1b434bfc3aac0022, https://usegalaxy.org/published/workflow?id=1d24cc949c52e475, https://usegalaxy.org/published/workflow?id=1ea88ca80a7f0038, https://usegalaxy.org/published/workflow?id=1eec2586d1ff908b, https://usegalaxy.org/published/workflow?id=237ca9f36ca266e7, https://usegalaxy.org/published/workflow?id=23eeee48217654d4, https://usegalaxy.org/published/workflow?id=242ef2e9dee35702, https://usegalaxy.org/published/workflow?id=25a9f0693ff6d327, https://usegalaxy.org/published/workflow?id=283bc048cc7a80f5, https://usegalaxy.org/published/workflow?id=284150bb1f296378, https://usegalaxy.org/published/workflow?id=28dbbfb3d480f4b1, https://usegalaxy.org/published/workflow?id=28ddea356ae57da1, https://usegalaxy.org/published/workflow?id=2acbb8c5f72b9f1c, https://usegalaxy.org/published/workflow?id=2b41752b8518a87b, https://usegalaxy.org/published/workflow?id=2c5c627c08563398, https://usegalaxy.org/published/workflow?id=2d994cc02ec05ede, https://usegalaxy.org/published/workflow?id=2f1469dab87a3649, https://usegalaxy.org/published/workflow?id=305fb9f84b053713, https://usegalaxy.org/published/workflow?id=31387ed5ebc88cd2, https://usegalaxy.org/published/workflow?id=3179c157fa5a2fdd, https://usegalaxy.org/published/workflow?id=31a6145b0832fa48, https://usegalaxy.org/published/workflow?id=325783f21bc5fef6, https://usegalaxy.org/published/workflow?id=349f1bcdbc84045f, https://usegalaxy.org/published/workflow?id=3753dcca4100e976, https://usegalaxy.org/published/workflow?id=3b0d42747f598f82, https://usegalaxy.org/published/workflow?id=3ba73f679558017c, https://usegalaxy.org/published/workflow?id=3d805d7e0f34c727, https://usegalaxy.org/published/workflow?id=3db4eb1ecb588d44, https://usegalaxy.org/published/workflow?id=3db51822f04f4ff7, https://usegalaxy.org/published/workflow?id=3f7550d5d2bf0b96, https://usegalaxy.org/published/workflow?id=408c799822cbf6ea, https://usegalaxy.org/published/workflow?id=426358bd988cb39d, https://usegalaxy.org/published/workflow?id=435f0340bf00c940, https://usegalaxy.org/published/workflow?id=4403ca7fe447a5d7, https://usegalaxy.org/published/workflow?id=44781c8b0bd667ed, https://usegalaxy.org/published/workflow?id=45c0b677c4920d03, https://usegalaxy.org/published/workflow?id=4605309321799e9b, https://usegalaxy.org/published/workflow?id=484f8fcf21d67376, https://usegalaxy.org/published/workflow?id=48dafc35989f13b1, https://usegalaxy.org/published/workflow?id=498a8c6329ca5728, https://usegalaxy.org/published/workflow?id=49a07807fcea8158, https://usegalaxy.org/published/workflow?id=49c9e8161a46667f, https://usegalaxy.org/published/workflow?id=4a0ad87aa5aa558a, https://usegalaxy.org/published/workflow?id=4bf38f5b45faa8a1, https://usegalaxy.org/published/workflow?id=4bf73376661baafd, https://usegalaxy.org/published/workflow?id=4d5cc5010537f172, https://usegalaxy.org/published/workflow?id=578e634b63923e9b, https://usegalaxy.org/published/workflow?id=57e6345fda3e2682, https://usegalaxy.org/published/workflow?id=597c4d54f8a8ebe7, https://usegalaxy.org/published/workflow?id=5dfec2a3ae5fa765, https://usegalaxy.org/published/workflow?id=5edabfc62f58d8cf, https://usegalaxy.org/published/workflow?id=6341c384ceaf47ed, https://usegalaxy.org/published/workflow?id=653d553ac2fa51c8, https://usegalaxy.org/published/workflow?id=65b4051e722228b8, https://usegalaxy.org/published/workflow?id=65ccf48b7872e36c, https://usegalaxy.org/published/workflow?id=683e725f672b7d9b, https://usegalaxy.org/published/workflow?id=6a27c0d224683f73, https://usegalaxy.org/published/workflow?id=6b511b47176893e3, https://usegalaxy.org/published/workflow?id=6fc7497c86ebd8ee, https://usegalaxy.org/published/workflow?id=7072731b51e16923, https://usegalaxy.org/published/workflow?id=73c20d784b7ef81a, https://usegalaxy.org/published/workflow?id=74049914e4a6c791, https://usegalaxy.org/published/workflow?id=771345c66d5b9700, https://usegalaxy.org/published/workflow?id=7adf904dea83e699, https://usegalaxy.org/published/workflow?id=7b8ee08ba291d231, https://usegalaxy.org/published/workflow?id=7dba74c915f63dc1, https://usegalaxy.org/published/workflow?id=7f6900852a81d121, https://usegalaxy.org/published/workflow?id=80a7124c40438cc9, https://usegalaxy.org/published/workflow?id=81f1f0fd6d9235fd, https://usegalaxy.org/published/workflow?id=84161f4f8bf77474, https://usegalaxy.org/published/workflow?id=864da6858d34df08, https://usegalaxy.org/published/workflow?id=86c67855acbf6124, https://usegalaxy.org/published/workflow?id=88327cb2e0fe3c25, https://usegalaxy.org/published/workflow?id=8ccb42012dcb5db2, https://usegalaxy.org/published/workflow?id=8cfbaa14cff5a633, https://usegalaxy.org/published/workflow?id=8d605fbd33003d26, https://usegalaxy.org/published/workflow?id=8f7c9461910b08f1, https://usegalaxy.org/published/workflow?id=903e35b62946ed76, https://usegalaxy.org/published/workflow?id=90f5b512997c70f8, https://usegalaxy.org/published/workflow?id=91a4daecbb97d03e, https://usegalaxy.org/published/workflow?id=932b3bc10186f73e, https://usegalaxy.org/published/workflow?id=936ed7447c8e0a2d, https://usegalaxy.org/published/workflow?id=93c1fef4472aa7a8, https://usegalaxy.org/published/workflow?id=950c7245a6c3f6e9, https://usegalaxy.org/published/workflow?id=951575e9a352301c, https://usegalaxy.org/published/workflow?id=961dac8100c6c70c, https://usegalaxy.org/published/workflow?id=96bd0007dceec123, https://usegalaxy.org/published/workflow?id=9938a81fb6911412, https://usegalaxy.org/published/workflow?id=9ab68c65337b893b, https://usegalaxy.org/published/workflow?id=9ddfdc9d6cacea88, https://usegalaxy.org/published/workflow?id=9f4c4038d1c519e2, https://usegalaxy.org/published/workflow?id=a4a6d6846c1692f9, https://usegalaxy.org/published/workflow?id=a8e0f77cd6ef49f9, https://usegalaxy.org/published/workflow?id=ab0be4897d5355ac, https://usegalaxy.org/published/workflow?id=acada5044d64a30a, https://usegalaxy.org/published/workflow?id=afa89bb0e8966753, https://usegalaxy.org/published/workflow?id=b11c6268704dcc1b, https://usegalaxy.org/published/workflow?id=b12f99e2233fcca6, https://usegalaxy.org/published/workflow?id=b5644a5dcf237ef9, https://usegalaxy.org/published/workflow?id=b661fb850c4b4b01, https://usegalaxy.org/published/workflow?id=b6d535f210efa18e, https://usegalaxy.org/published/workflow?id=bb1cb6abfe84a646, https://usegalaxy.org/published/workflow?id=bbc894a79eb0e769, https://usegalaxy.org/published/workflow?id=bc72dcbca89ca35d, https://usegalaxy.org/published/workflow?id=bd6bfd338ffb9637, https://usegalaxy.org/published/workflow?id=bfd2e7e63d3e6856, https://usegalaxy.org/published/workflow?id=bfef75672b9c3ea4, https://usegalaxy.org/published/workflow?id=c03ada2070e9ca8a, https://usegalaxy.org/published/workflow?id=c09b8a0c96123cbb, https://usegalaxy.org/published/workflow?id=c0a9a2d53b8660de, https://usegalaxy.org/published/workflow?id=c451e4dca9981ed0, https://usegalaxy.org/published/workflow?id=c694856c01ce7cc5, https://usegalaxy.org/published/workflow?id=c95e9116e4b7982b, https://usegalaxy.org/published/workflow?id=c983ea8a32662cd6, https://usegalaxy.org/published/workflow?id=cb4ccd3068819418, https://usegalaxy.org/published/workflow?id=cc50da71cdf6f244, https://usegalaxy.org/published/workflow?id=ce04658f05029be2, https://usegalaxy.org/published/workflow?id=cfd5dc3c7bc1e832, https://usegalaxy.org/published/workflow?id=d08135fa13d1745e, https://usegalaxy.org/published/workflow?id=d2e8a6b39fe88f4c, https://usegalaxy.org/published/workflow?id=d3f35215e6f45d79, https://usegalaxy.org/published/workflow?id=d4b2b85db47c405e, https://usegalaxy.org/published/workflow?id=daaded719dcfd53b, https://usegalaxy.org/published/workflow?id=dae5d5e5df5c4dd7, https://usegalaxy.org/published/workflow?id=dbe08700353dfc43, https://usegalaxy.org/published/workflow?id=e12e642b88a0fe08, https://usegalaxy.org/published/workflow?id=e4dd39237df58884, https://usegalaxy.org/published/workflow?id=eaf96fa3c39c210b, https://usegalaxy.org/published/workflow?id=ec3cc032a90c44f1, https://usegalaxy.org/published/workflow?id=f09033420c6267ab, https://usegalaxy.org/published/workflow?id=f1a54b10d4afce8f, https://usegalaxy.org/published/workflow?id=f4111a169255535c, https://usegalaxy.org/published/workflow?id=f60027545d40b82d, https://usegalaxy.org/published/workflow?id=f93a4f6b5f3aaaed, https://usegalaxy.org/published/workflow?id=fa26fd51caea53c7, https://usegalaxy.org/published/workflow?id=fa36d9b50e956917, https://usegalaxy.org/published/workflow?id=fb98ef955b3dbd4b, https://usegalaxy.org/published/workflow?id=fc7b14aafa0b421e, https://usegalaxy.org/published/workflow?id=fd155fcba9bed7ff, https://usegalaxy.org/published/workflow?id=fd40e50c9f1545ce, https://workflowhub.eu/workflows/1054?version=1, https://workflowhub.eu/workflows/1096?version=2, https://workflowhub.eu/workflows/1103?version=3, https://workflowhub.eu/workflows/1104?version=2, https://workflowhub.eu/workflows/1162?version=2, https://workflowhub.eu/workflows/1163?version=2, https://workflowhub.eu/workflows/1164?version=2, https://workflowhub.eu/workflows/1255?version=3, https://workflowhub.eu/workflows/1323?version=1, https://workflowhub.eu/workflows/1500?version=1, https://workflowhub.eu/workflows/1504?version=1, https://workflowhub.eu/workflows/1509?version=1, https://workflowhub.eu/workflows/1525?version=1, https://workflowhub.eu/workflows/1549?version=1, https://workflowhub.eu/workflows/1580?version=1, https://workflowhub.eu/workflows/1607?version=1, https://workflowhub.eu/workflows/1621?version=1, https://workflowhub.eu/workflows/1641?version=1, https://workflowhub.eu/workflows/1646?version=1, https://workflowhub.eu/workflows/1656?version=1, https://workflowhub.eu/workflows/1746?version=1, https://workflowhub.eu/workflows/2011?version=1, https://workflowhub.eu/workflows/2029?version=1, https://workflowhub.eu/workflows/229?version=1, https://workflowhub.eu/workflows/358?version=1, https://workflowhub.eu/workflows/403?version=9, https://workflowhub.eu/workflows/605?version=1, https://workflowhub.eu/workflows/606?version=2, https://workflowhub.eu/workflows/608?version=1, https://workflowhub.eu/workflows/612?version=25, https://workflowhub.eu/workflows/633?version=23, https://workflowhub.eu/workflows/642?version=25, https://workflowhub.eu/workflows/701?version=1, https://workflowhub.eu/workflows/702?version=1, https://workflowhub.eu/workflows/751?version=18, https://workflowhub.eu/workflows/754?version=1, https://workflowhub.eu/workflows/788?version=1, https://workflowhub.eu/workflows/789?version=1, https://workflowhub.eu/workflows/794?version=2, https://workflowhub.eu/workflows/879?version=1, https://workflowhub.eu/workflows/881?version=5</td>
      <td>assembly/ERGA-post-assembly-QC, assembly/assembly-quality-control, assembly/hifi-assembly, assembly/hybrid_denovo_assembly, assembly/largegenome, assembly/vgp_genome_assembly, ecology/phylogeny-data-prep, genome-annotation/annotation-with-maker, genome-annotation/annotation-with-maker-short, genome-annotation/braker3, genome-annotation/comparison-braker-helixer-annotation, genome-annotation/funannotate, genome-annotation/helixer</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>cami_amber</td>
      <td>biobox_add_taxid, cami_amber, cami_amber_add, cami_amber_convert</td>
      <td>Evaluation package for the comparative assessment of genome reconstructions and taxonomic assignments</td>
      <td>2024-05-20</td>
      <td>https://github.com/CAMI-challenge/AMBER</td>
      <td>2.0.7</td>
      <td>cami-amber</td>
      <td>2.0.7</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Read binning</td>
      <td>Read binning</td>
      <td>Metagenomics</td>
      <td>Metagenomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cami_amber</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cami_amber</td>
      <td>cami-amber</td>
      <td>AMBER</td>
      <td>AMBER is an evaluation package for the comparative assessment of genome reconstructions and taxonomic assignments from metagenome benchmark datasets. It provides performance metrics, results rankings, and comparative visualizations for assessing multiple programs or parameter effects. The provided metrics were used in the first community benchmarking challenge of the initiative for the Critical Assessment of Metagenomic Interpretation.</td>
      <td></td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>10522</td>
      <td>10522</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10522</td>
      <td>10522</td>
      <td>8</td>
      <td>8</td>
      <td>https://usegalaxy.eu/published/workflow?id=00f15824c7a9b196, https://usegalaxy.eu/published/workflow?id=17d2e76147fc940c, https://usegalaxy.eu/published/workflow?id=43098b8afe78b071, https://usegalaxy.eu/published/workflow?id=ce5c9287a67e6085, https://usegalaxy.eu/published/workflow?id=e4dabaf57668dbd1</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>canu</td>
      <td>canu</td>
      <td>Canu is a hierarchical assembly pipeline designed for high-noise single-molecule sequencing (such as the PacBio RS II/Sequel or Oxford Nanopore MinION).</td>
      <td>2018-06-07</td>
      <td>https://github.com/marbl/canu</td>
      <td>2.3</td>
      <td>canu</td>
      <td>2.3</td>
      <td>Up-to-date</td>
      <td>Assembly</td>
      <td>De-novo assembly</td>
      <td>De-novo assembly</td>
      <td>Genomics</td>
      <td>Genomics</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/canu</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/canu</td>
      <td>canu</td>
      <td>CANU</td>
      <td>De-novo assembly tool for long read chemistry like Nanopore data and PacBio data.</td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1644</td>
      <td>1843</td>
      <td>13768</td>
      <td>17494</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>454</td>
      <td>485</td>
      <td>3744</td>
      <td>4325</td>
      <td>37</td>
      <td>37</td>
      <td>480</td>
      <td>480</td>
      <td>22299</td>
      <td>17992</td>
      <td>2365</td>
      <td>2135</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>cawlign</td>
      <td>cawlign</td>
      <td>Codon-aware alignment of sequences to a reference.</td>
      <td>2025-09-30</td>
      <td>https://github.com/veg/cawlign</td>
      <td>0.1.15</td>
      <td>cawlign</td>
      <td>0.1.16</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cawlign</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cawlign</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>cd_hit</td>
      <td>cd_hit</td>
      <td>Cluster or compare biological sequence datasets</td>
      <td>2018-02-26</td>
      <td>http://weizhongli-lab.org/cd-hit/</td>
      <td>4.8.1</td>
      <td>cd-hit</td>
      <td>4.8.1</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis, Fasta Manipulation</td>
      <td>Sequence clustering</td>
      <td>Sequence clustering</td>
      <td>Sequencing</td>
      <td>Sequencing</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cdhit</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cdhit</td>
      <td>cd-hit</td>
      <td>cd-hit</td>
      <td>Cluster a nucleotide dataset into representative sequences.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>899</td>
      <td>899</td>
      <td>20148</td>
      <td>20148</td>
      <td>1</td>
      <td>1</td>
      <td>2</td>
      <td>2</td>
      <td>1</td>
      <td>1</td>
      <td>2</td>
      <td>2</td>
      <td>57</td>
      <td>57</td>
      <td>1174</td>
      <td>1174</td>
      <td>21326</td>
      <td>21326</td>
      <td>958</td>
      <td>958</td>
      <td>https://usegalaxy.eu/published/workflow?id=2ee0eb74e839b476, https://usegalaxy.eu/published/workflow?id=974cc681d07ec788, https://usegalaxy.eu/published/workflow?id=9b9e1e28f3683f92, https://usegalaxy.eu/published/workflow?id=bebcbd6f0ffa55c5</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>cd_hit_dup</td>
      <td>cd_hit_dup</td>
      <td>simple tool for removing duplicates from sequencing reads</td>
      <td>2015-04-28</td>
      <td></td>
      <td>0.0.1</td>
      <td>cd-hit-auxtools</td>
      <td>4.8.1</td>
      <td>To update</td>
      <td>Metagenomics, Sequence Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/cd_hit_dup</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/cd_hit_dup</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>501</td>
      <td>770</td>
      <td>7542</td>
      <td>9437</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9437</td>
      <td>7542</td>
      <td>770</td>
      <td>501</td>
      <td>https://usegalaxy.org/published/workflow?id=35c2626849348f2c</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>cemitool</td>
      <td>cemitool</td>
      <td>Gene co-expression network analysis tool</td>
      <td>2022-10-10</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/CEMiTool.html</td>
      <td>1.30.0</td>
      <td>bioconductor-cemitool</td>
      <td>1.30.0</td>
      <td>Up-to-date</td>
      <td>Transcriptomics, RNA, Statistics, Single Cell</td>
      <td>Enrichment analysis, Pathway or network analysis</td>
      <td>Enrichment analysis, Pathway or network analysis</td>
      <td>Gene expression, Transcriptomics, Microarray experiment</td>
      <td>Transcriptomics, Microarray experiment</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cemitool</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cemitool</td>
      <td>cemitool</td>
      <td>CEMiTool</td>
      <td>It unifies the discovery and the analysis of coexpression gene modules in a fully automatic manner, while providing a user-friendly html report with high quality graphs. Our tool evaluates if modules contain genes that are over-represented by specific pathways or that are altered in a specific sample group. Additionally, CEMiTool is able to integrate transcriptomic data with interactome information, identifying the potential hubs on each network.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>45</td>
      <td>45</td>
      <td>354</td>
      <td>354</td>
      <td>87</td>
      <td>87</td>
      <td>584</td>
      <td>584</td>
      <td>6</td>
      <td>6</td>
      <td>154</td>
      <td>154</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1092</td>
      <td>1092</td>
      <td>138</td>
      <td>138</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>checkm</td>
      <td>checkm_analyze, checkm_lineage_set, checkm_lineage_wf, checkm_plot, checkm_qa, checkm_taxon_set, checkm_taxonomy_wf, checkm_tetra, checkm_tree, checkm_tree_qa</td>
      <td>Assess the quality of microbial genomes recovered from isolates, single cells, and metagenomes</td>
      <td>2022-07-29</td>
      <td>https://github.com/Ecogenomics/CheckM</td>
      <td>1.2.4</td>
      <td>checkm-genome</td>
      <td>1.2.4</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Sequence assembly validation, Validation, Sequence composition calculation, Sequencing quality control, Statistical calculation</td>
      <td>Sequence assembly validation, Sequence composition calculation, Statistical calculation</td>
      <td>Genomics, Phylogenomics, Phylogenetics, Taxonomy, Metagenomics, Data quality management</td>
      <td>Phylogenomics, Phylogenetics, Taxonomy, Metagenomics, Data quality management</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/checkm</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/checkm</td>
      <td>checkm</td>
      <td>CheckM</td>
      <td>CheckM provides a set of tools for assessing the quality of genomes recovered from isolates, single cells, or metagenomes.</td>
      <td></td>
      <td>10</td>
      <td>10</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>717</td>
      <td>717</td>
      <td>12189</td>
      <td>12189</td>
      <td>179</td>
      <td>179</td>
      <td>1510</td>
      <td>1510</td>
      <td>96</td>
      <td>96</td>
      <td>2254</td>
      <td>2254</td>
      <td>50</td>
      <td>50</td>
      <td>1469</td>
      <td>1469</td>
      <td>17422</td>
      <td>17422</td>
      <td>1042</td>
      <td>1042</td>
      <td>https://usegalaxy.eu/published/workflow?id=0563f58718be932d, https://usegalaxy.eu/published/workflow?id=1e5f1d9a51032e44, https://usegalaxy.eu/published/workflow?id=3eaa913306c16146, https://usegalaxy.eu/published/workflow?id=5b940b283d91d99d, https://usegalaxy.eu/published/workflow?id=6158c3b5fc12044e, https://usegalaxy.eu/published/workflow?id=6416cfdbd5273063, https://usegalaxy.eu/published/workflow?id=667bac8d7453e5f1, https://usegalaxy.eu/published/workflow?id=8deb08d4f228e198, https://usegalaxy.eu/published/workflow?id=97312d273b6e8bd9, https://usegalaxy.eu/published/workflow?id=d57d41e306241396, https://usegalaxy.org.au/published/workflow?id=92fcdd490831c6c5, https://usegalaxy.org/published/workflow?id=7251c6e0dad8f927, https://usegalaxy.org/published/workflow?id=80565301ada0d3eb, https://workflowhub.eu/workflows/1352?version=4</td>
      <td>microbiome/mags-building, microbiome/metagenomics-binning</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>checkm2</td>
      <td>checkm2</td>
      <td>Rapid assessment of genome bin quality using machine learning</td>
      <td>2024-11-08</td>
      <td>https://github.com/chklovski/CheckM2</td>
      <td>1.1.0</td>
      <td>checkm2</td>
      <td>1.1.0</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Sequence assembly validation, Validation, Sequence composition calculation, Sequencing quality control, Statistical calculation</td>
      <td>Sequence assembly validation, Sequence composition calculation, Statistical calculation</td>
      <td>Genomics, Phylogenomics, Phylogenetics, Taxonomy, Metagenomics, Data quality management</td>
      <td>Phylogenomics, Phylogenetics, Taxonomy, Metagenomics, Data quality management</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/checkm2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/checkm2</td>
      <td>checkm</td>
      <td>CheckM</td>
      <td>CheckM provides a set of tools for assessing the quality of genomes recovered from isolates, single cells, or metagenomes.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>228</td>
      <td>228</td>
      <td>1602</td>
      <td>1602</td>
      <td>187</td>
      <td>187</td>
      <td>1322</td>
      <td>1322</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>15</td>
      <td>15</td>
      <td>1268</td>
      <td>1268</td>
      <td>4192</td>
      <td>4192</td>
      <td>430</td>
      <td>430</td>
      <td>https://usegalaxy.eu/published/workflow?id=3303a04887a47675, https://usegalaxy.eu/published/workflow?id=368d1ac0150ea0b6, https://usegalaxy.eu/published/workflow?id=7371b6918e895e0c, https://usegalaxy.eu/published/workflow?id=7ecd92cb382d8168, https://usegalaxy.eu/published/workflow?id=ba1fda2318046543, https://usegalaxy.eu/published/workflow?id=c62d65832377e376, https://usegalaxy.eu/published/workflow?id=e532d98f4b368666, https://usegalaxy.fr/published/workflow?id=f0776f7a890b523a, https://usegalaxy.org.au/published/workflow?id=f05ed1e5c5dfffa9, https://usegalaxy.org/published/workflow?id=524602abe59b18f7, https://usegalaxy.org/published/workflow?id=5cd56232e873beab, https://usegalaxy.org/published/workflow?id=a8aee61c2cbaf6ea, https://usegalaxy.org/published/workflow?id=b76a9ce44ea019f1, https://workflowhub.eu/workflows/1352?version=4, https://workflowhub.eu/workflows/1882?version=3</td>
      <td>genome-annotation/bacterial-genome-quality-control</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>checkv</td>
      <td>checkv_end_to_end</td>
      <td>Assess quality of single-contig viral genomes</td>
      <td>2024-09-13</td>
      <td>https://bitbucket.org/berkeleylab/checkv/</td>
      <td>1.0.3</td>
      <td>checkv</td>
      <td>1.0.3</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Sequence assembly, Validation, Read mapping</td>
      <td>Sequence assembly, Validation, Read mapping</td>
      <td>Metagenomics, Sequence assembly, Metagenomic sequencing, Mapping</td>
      <td>Metagenomics, Sequence assembly, Metagenomic sequencing, Mapping</td>
      <td>ufz</td>
      <td>https://github.com/Helmholtz-UFZ/ufz-galaxy-tools/blob/main/tools/checkv/</td>
      <td>https://github.com/Helmholtz-UFZ/galaxy-tools/tree/main/tools/checkv</td>
      <td>checkv</td>
      <td>CheckV</td>
      <td>CheckV is a fully automated command-line pipeline for assessing the quality of single-contig viral genomes, including identification of host contamination for integrated proviruses, estimating completeness for genome fragments, and identification of closed genomes.</td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>83</td>
      <td>83</td>
      <td>4527</td>
      <td>4527</td>
      <td>1</td>
      <td>1</td>
      <td>21</td>
      <td>21</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4548</td>
      <td>4548</td>
      <td>84</td>
      <td>84</td>
      <td>https://usegalaxy.eu/published/workflow?id=36de05d50c703164, https://usegalaxy.eu/published/workflow?id=62392daea25a3980, https://usegalaxy.eu/published/workflow?id=87231a6a7938cb95, https://usegalaxy.eu/published/workflow?id=c62d65832377e376</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>chopper</td>
      <td>chopper</td>
      <td>Filtering and trimming of fastq files with long read sequencing such as PacBio or ONT.</td>
      <td>2025-07-22</td>
      <td>https://github.com/wdecoster/chopper</td>
      <td>0.12.0</td>
      <td>chopper</td>
      <td>0.12.0</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis, Fastq Manipulation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/chopper</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/chopper</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>10</td>
      <td>6</td>
      <td>6</td>
      <td>https://usegalaxy.eu/published/workflow?id=4eb9dfe91fc2dc67</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>cialign</td>
      <td>cialign</td>
      <td>Tool to clean, visualise and analyse a multiple sequence alignment</td>
      <td>2025-08-04</td>
      <td>https://github.com/KatyBrown/CIAlign</td>
      <td>1.1.4</td>
      <td>cialign</td>
      <td>1.1.4</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cialign</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cialign</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>3</td>
      <td>3</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>clair3</td>
      <td>clair3</td>
      <td>Symphonizing pileup and full-alignment for high-performance long-read variant calling</td>
      <td>2022-06-15</td>
      <td>https://github.com/HKU-BAL/Clair3</td>
      <td>1.0.10</td>
      <td>clair3</td>
      <td>1.2.0</td>
      <td>To update</td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>Variant calling</td>
      <td>Variant calling</td>
      <td>Molecular genetics</td>
      <td>Molecular genetics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/clair3</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/clair3</td>
      <td>clair3</td>
      <td>Clair3</td>
      <td>Clair3 is a germline small variant caller for long-reads. Clair3 makes the best of two major method categories: pileup calling handles most variant candidates with speed, and full-alignment tackles complicated candidates to maximize precision and recall. Clair3 runs fast and has superior performance, especially at lower coverage. Clair3 is simple and modular for easy deployment and integration.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>328</td>
      <td>328</td>
      <td>5269</td>
      <td>5269</td>
      <td>52</td>
      <td>52</td>
      <td>1858</td>
      <td>1858</td>
      <td>13</td>
      <td>13</td>
      <td>246</td>
      <td>246</td>
      <td>2</td>
      <td>2</td>
      <td>54</td>
      <td>54</td>
      <td>7427</td>
      <td>7427</td>
      <td>395</td>
      <td>395</td>
      <td>https://usegalaxy.eu/published/workflow?id=09c7069ae409c362, https://usegalaxy.eu/published/workflow?id=3f0b104a74acf869, https://usegalaxy.eu/published/workflow?id=60e0b2a28a900178, https://usegalaxy.eu/published/workflow?id=64abcddeb516a712, https://usegalaxy.eu/published/workflow?id=6aa2891bd2b41f32, https://usegalaxy.eu/published/workflow?id=cb54883452a73019, https://usegalaxy.eu/published/workflow?id=cdf2f02428f5f83f, https://usegalaxy.eu/published/workflow?id=d215a2e554fd83bc, https://usegalaxy.org.au/published/workflow?id=0e11619979c2830c, https://usegalaxy.org.au/published/workflow?id=244ea5e94237ebad, https://usegalaxy.org.au/published/workflow?id=76a0a3af8de11622, https://usegalaxy.org.au/published/workflow?id=857e482acfcb2dba, https://usegalaxy.org/published/workflow?id=29ba77708cdafe0f, https://usegalaxy.org/published/workflow?id=38911ba6f66d80f6, https://usegalaxy.org/published/workflow?id=3e703abaeb870f6f, https://usegalaxy.org/published/workflow?id=6d778eeb176d563c, https://usegalaxy.org/published/workflow?id=cb810cdaa4915f4b, https://workflowhub.eu/workflows/1063?version=5, https://workflowhub.eu/workflows/1479?version=1</td>
      <td>microbiome/pathogen-detection-from-nanopore-foodborne-data</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>clinod</td>
      <td>clinod</td>
      <td>NoD: a Nucleolar localization sequence detector for eukaryotic and viral proteins</td>
      <td>2013-05-02</td>
      <td>http://www.compbio.dundee.ac.uk/www-nod/</td>
      <td>0.1.0</td>
      <td>clinod</td>
      <td>1.3</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Nucleic acid sequence analysis</td>
      <td>Nucleic acid sequence analysis</td>
      <td>Sequence analysis</td>
      <td>Sequence analysis</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/clinod</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/clinod</td>
      <td>clinod</td>
      <td>clinod</td>
      <td>The command line NoD predictor (clinod) can be run from the command line to predict Nucleolar localization sequences (NoLSs) that are short targeting sequences responsible for the localization of proteins to the nucleolus.The predictor accepts a list of FASTA formatted sequences as an input and outputs the NOLS predictions as a result.Please note that currently, JPred secondary structure predictions are not supported by clinod. However, we are working on it.</td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>31</td>
      <td>31</td>
      <td>585</td>
      <td>585</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>585</td>
      <td>585</td>
      <td>31</td>
      <td>31</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>clustalw</td>
      <td>clustalw</td>
      <td>ClustalW multiple sequence alignment program for DNA or proteins</td>
      <td>2022-10-02</td>
      <td>http://www.clustal.org/clustal2/</td>
      <td>2.1</td>
      <td>clustalw</td>
      <td>2.1</td>
      <td>Up-to-date</td>
      <td>Phylogenetics, Sequence Analysis</td>
      <td>Multiple sequence alignment</td>
      <td>Multiple sequence alignment</td>
      <td>Phylogeny, Sequence analysis</td>
      <td>Phylogeny, Sequence analysis</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/clustalw</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/clustalw</td>
      <td>clustal2</td>
      <td>Clustal 2 (Clustal W, Clustal X)</td>
      <td>Multiple sequence alignment program with a command-line interface (Clustal W) and a graphical user interface (Clustal X). The display colours allow conserved features to be highlighted for easy viewing in the alignment. It is available for several platforms, including Windows, Macintosh PowerMac, Linux and Solaris.Names occassionally spelled also as Clustal W2, ClustalW2, ClustalW, ClustalX, Clustal2.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2268</td>
      <td>2506</td>
      <td>114395</td>
      <td>117339</td>
      <td>3966</td>
      <td>8263</td>
      <td>54653</td>
      <td>99642</td>
      <td>837</td>
      <td>966</td>
      <td>11073</td>
      <td>13924</td>
      <td>42</td>
      <td>43</td>
      <td>726</td>
      <td>730</td>
      <td>231635</td>
      <td>180847</td>
      <td>11778</td>
      <td>7113</td>
      <td>https://usegalaxy.eu/published/workflow?id=36de05d50c703164, https://usegalaxy.eu/published/workflow?id=376119528377a3ae, https://usegalaxy.eu/published/workflow?id=3e0246389fb70340, https://usegalaxy.eu/published/workflow?id=974cc681d07ec788, https://usegalaxy.eu/published/workflow?id=c8bc4af6dd3b5d02, https://usegalaxy.eu/published/workflow?id=f6a763951d815944, https://usegalaxy.org.au/published/workflow?id=b60922a253df6654, https://usegalaxy.org.au/published/workflow?id=eda40b58616a0fe4, https://usegalaxy.org/published/workflow?id=042758b8a8c5ac80, https://usegalaxy.org/published/workflow?id=07f7198b63108131, https://usegalaxy.org/published/workflow?id=18e7a954c0b6e4bc, https://usegalaxy.org/published/workflow?id=2d3063882d8239ff, https://usegalaxy.org/published/workflow?id=3d7baebefa7c9b81, https://usegalaxy.org/published/workflow?id=3ef08553982f6ffc, https://usegalaxy.org/published/workflow?id=603e5c61c57bcf3f, https://usegalaxy.org/published/workflow?id=6a1eb7e7bececba1, https://usegalaxy.org/published/workflow?id=75c2cf36253194c2, https://usegalaxy.org/published/workflow?id=7b68f97bf88f99b9, https://usegalaxy.org/published/workflow?id=87685c6247521b54, https://usegalaxy.org/published/workflow?id=8b7e03f09b0183d9, https://usegalaxy.org/published/workflow?id=a484a1e1a7335696, https://usegalaxy.org/published/workflow?id=b2fb90873471f4e7, https://usegalaxy.org/published/workflow?id=be9bd7286d38c210, https://workflowhub.eu/workflows/1060?version=1, https://workflowhub.eu/workflows/1487?version=2, https://workflowhub.eu/workflows/1541?version=1, https://workflowhub.eu/workflows/1656?version=1, https://workflowhub.eu/workflows/358?version=1</td>
      <td>ecology/phylogeny-data-prep, genome-annotation/hpc-for-lsgc, microbiome/pathogen-detection-from-nanopore-foodborne-data</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>codeml</td>
      <td>codeml</td>
      <td>Detects positive selection</td>
      <td>2017-07-19</td>
      <td>http://abacus.gene.ucl.ac.uk/software/paml.html</td>
      <td>4.9</td>
      <td>paml</td>
      <td>4.10.9</td>
      <td>To update</td>
      <td>Phylogenetics</td>
      <td>Probabilistic sequence generation, Phylogenetic tree generation (maximum likelihood and Bayesian methods), Phylogenetic tree analysis</td>
      <td>Probabilistic sequence generation</td>
      <td>Phylogenetics, Sequence analysis</td>
      <td>Phylogenetics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/codeml</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/codeml</td>
      <td>paml</td>
      <td>PAML</td>
      <td>Package of programs for phylogenetic analyses of DNA or protein sequences using maximum likelihood.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>72</td>
      <td>84</td>
      <td>62891</td>
      <td>63000</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>63000</td>
      <td>62891</td>
      <td>84</td>
      <td>72</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>cojac</td>
      <td>cooc_mutbamscan, cooc_pubmut, cooc_tabmut</td>
      <td>co-occurrence of mutations on amplicons</td>
      <td>2022-08-11</td>
      <td>https://github.com/cbg-ethz/cojac</td>
      <td>0.9.3</td>
      <td>cojac</td>
      <td>0.9.3</td>
      <td>Up-to-date</td>
      <td>Metagenomics, Sequence Analysis</td>
      <td></td>
      <td></td>
      <td>Genetic variation</td>
      <td>Genetic variation</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cojac</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cojac</td>
      <td>cojac</td>
      <td>COJAC</td>
      <td>CoOccurrence adJusted Analysis and Calling - The cojac package comprises a set of command-line tools to analyse co-occurrence of mutations on amplicons. It is useful, for example, for early detection of viral variants of concern (e.g. Alpha, Delta, Omicron) in environmental samples, and has been designed to scan for multiple SARS-CoV-2 variants in wastewater samples, as analyzed jointly by ETH Zurich, EPFL and Eawag.</td>
      <td></td>
      <td>2</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>13</td>
      <td>13</td>
      <td>2763</td>
      <td>2763</td>
      <td>7</td>
      <td>7</td>
      <td>20</td>
      <td>20</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2783</td>
      <td>2783</td>
      <td>20</td>
      <td>20</td>
      <td>https://usegalaxy.eu/published/workflow?id=d88145e7475e0b7b</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>combine_metaphlan2_humann2</td>
      <td>combine_metaphlan_humann</td>
      <td>Combine MetaPhlAn2 and HUMAnN2 outputs to relate genus/species abundances and gene families/pathways abundances</td>
      <td>2023-07-20</td>
      <td></td>
      <td>0.3.0</td>
      <td>python</td>
      <td></td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Aggregation</td>
      <td>Aggregation</td>
      <td>Metagenomics, Molecular interactions, pathways and networks</td>
      <td>Metagenomics, Molecular interactions, pathways and networks</td>
      <td>bebatut</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/combine_metaphlan2_humann2</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/combine_metaphlan_humann</td>
      <td>combine_metaphlan_and_humann</td>
      <td>Combine Metaphlan and HUMAnN</td>
      <td>This tool combine MetaPhlAn outputs and HUMANnN outputs</td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>112</td>
      <td>112</td>
      <td>1972</td>
      <td>1972</td>
      <td>31</td>
      <td>31</td>
      <td>128</td>
      <td>128</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>12</td>
      <td>12</td>
      <td>25</td>
      <td>25</td>
      <td>2125</td>
      <td>2125</td>
      <td>155</td>
      <td>155</td>
      <td>https://usegalaxy.eu/published/workflow?id=9472be79da012999, https://usegalaxy.fr/published/workflow?id=c4f6062e51d0f45e, https://workflowhub.eu/workflows/1466?version=2</td>
      <td>microbiome/metatranscriptomics</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>comebin</td>
      <td>comebin</td>
      <td>Binning of metagenomic contigs using COntrastive Multi-viEw representation learning</td>
      <td>2025-07-03</td>
      <td>https://github.com/ziyewang/COMEBin</td>
      <td>1.0.4</td>
      <td>comebin</td>
      <td>1.0.4</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/comebin</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/comebin</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>comebin_bam</td>
      <td>comebin_bam</td>
      <td>Generate bam file for COMEBin</td>
      <td>2025-11-07</td>
      <td>https://github.com/ziyewang/COMEBin</td>
      <td>1.0.4</td>
      <td>comebin</td>
      <td>1.0.4</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/comebin_bam/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/comebin_bam</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>compare_humann2_output</td>
      <td>compare_humann2_output</td>
      <td>Compare outputs of HUMAnN2 for several samples and extract similar and specific information</td>
      <td>2022-10-19</td>
      <td></td>
      <td>0.2.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Comparison</td>
      <td>Comparison</td>
      <td>Metagenomics, Gene and protein families</td>
      <td>Metagenomics, Gene and protein families</td>
      <td>bebatut</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/compare_humann2_output</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/compare_humann2_output</td>
      <td>compare_humann2_outputs</td>
      <td>Compare HUMAnN2 outputs</td>
      <td>This tool compare HUMANnN2 outputs with gene families or pathways and their relative abundances between several samples</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>23</td>
      <td>32</td>
      <td>248</td>
      <td>332</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>332</td>
      <td>248</td>
      <td>32</td>
      <td>23</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>compleasm</td>
      <td>compleasm</td>
      <td>Compleasm: a faster and more accurate reimplementation of BUSCO</td>
      <td>2023-12-04</td>
      <td>https://github.com/huangnengCSU/compleasm</td>
      <td>0.2.6</td>
      <td>compleasm</td>
      <td>0.2.7</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Sequence assembly validation</td>
      <td>Sequence assembly validation</td>
      <td>Sequence assembly, Genomics, Transcriptomics, Sequence analysis</td>
      <td>Sequence assembly, Transcriptomics, Sequence analysis</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/compleasm</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/compleasm</td>
      <td>compleasm</td>
      <td>Compleasm</td>
      <td>Compleasm: a faster and more accurate reimplementation of BUSCO.It provides measures for quantitative assessment of genome assembly, gene set, and transcriptome completeness based on evolutionarily informed expectations of gene content from near-universal single-copy orthologs.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>99</td>
      <td>99</td>
      <td>675</td>
      <td>675</td>
      <td>92</td>
      <td>92</td>
      <td>638</td>
      <td>638</td>
      <td>20</td>
      <td>20</td>
      <td>269</td>
      <td>269</td>
      <td>18</td>
      <td>18</td>
      <td>449</td>
      <td>449</td>
      <td>2031</td>
      <td>2031</td>
      <td>229</td>
      <td>229</td>
      <td>https://usegalaxy.eu/published/workflow?id=04844dddcc4e8aa7, https://usegalaxy.eu/published/workflow?id=082bfd9b36e921e8, https://usegalaxy.eu/published/workflow?id=122da0054fd40adf, https://usegalaxy.eu/published/workflow?id=194a05d42687b524, https://usegalaxy.eu/published/workflow?id=1cba6eb4aff8dc61, https://usegalaxy.eu/published/workflow?id=1d1ae39386b5c318, https://usegalaxy.eu/published/workflow?id=1da7ac8eaf891f39, https://usegalaxy.eu/published/workflow?id=1e7f4ba6261f979c, https://usegalaxy.eu/published/workflow?id=327beb1edda305ff, https://usegalaxy.eu/published/workflow?id=3d604dad4f1f5b3d, https://usegalaxy.eu/published/workflow?id=3e255a5b7026d26e, https://usegalaxy.eu/published/workflow?id=4077e77b6af919c4, https://usegalaxy.eu/published/workflow?id=45ae834d842302c3, https://usegalaxy.eu/published/workflow?id=475ea16f943cc013, https://usegalaxy.eu/published/workflow?id=54590ccd11ffbf65, https://usegalaxy.eu/published/workflow?id=56523b41ff7eec76, https://usegalaxy.eu/published/workflow?id=5c51f26ef8b20633, https://usegalaxy.eu/published/workflow?id=6573f3b1606b7f09, https://usegalaxy.eu/published/workflow?id=6865f33cffca4922, https://usegalaxy.eu/published/workflow?id=7137e45ca8dc6e4c, https://usegalaxy.eu/published/workflow?id=7337d15d121596da, https://usegalaxy.eu/published/workflow?id=7e68cf8a67efa8e9, https://usegalaxy.eu/published/workflow?id=9207e74a5bf7d8e1, https://usegalaxy.eu/published/workflow?id=9d209bcdae1a475e, https://usegalaxy.eu/published/workflow?id=a9c1dca5da7170dd, https://usegalaxy.eu/published/workflow?id=ae74f60cd20d56bc, https://usegalaxy.eu/published/workflow?id=aea88bfca6d1d476, https://usegalaxy.eu/published/workflow?id=c4c2e669b86cffbc, https://usegalaxy.eu/published/workflow?id=cf4638fbca7c460b, https://usegalaxy.eu/published/workflow?id=d17d6029914d666b, https://usegalaxy.eu/published/workflow?id=d649ec485c4b3994, https://usegalaxy.eu/published/workflow?id=dfe3b555856c7b44, https://usegalaxy.eu/published/workflow?id=e1947b059ef131d2, https://usegalaxy.eu/published/workflow?id=e2f936f1aa43a629, https://usegalaxy.eu/published/workflow?id=f08b29e4cef76cd5, https://usegalaxy.eu/published/workflow?id=f1e2577699125607, https://usegalaxy.eu/published/workflow?id=f63127675a07e00a, https://usegalaxy.eu/published/workflow?id=fd649ac4d4872c81, https://usegalaxy.fr/published/workflow?id=ae05ff8487f58fd6, https://usegalaxy.org.au/published/workflow?id=2356584eac7fa234, https://usegalaxy.org.au/published/workflow?id=2470cb61ba51288c, https://usegalaxy.org.au/published/workflow?id=269b42605a2a87d2, https://usegalaxy.org.au/published/workflow?id=328adfa7d90b1d54, https://usegalaxy.org.au/published/workflow?id=3a64c1031340d860, https://usegalaxy.org.au/published/workflow?id=3c30c822058187be, https://usegalaxy.org.au/published/workflow?id=40666ceb68d101ac, https://usegalaxy.org.au/published/workflow?id=41282d51f9112552, https://usegalaxy.org.au/published/workflow?id=43343fa1896f8278, https://usegalaxy.org.au/published/workflow?id=447d8daefdefafee, https://usegalaxy.org.au/published/workflow?id=466809eb3392f7fb, https://usegalaxy.org.au/published/workflow?id=47cd0a90609fd557, https://usegalaxy.org.au/published/workflow?id=4a404aa3e9f29e5a, https://usegalaxy.org.au/published/workflow?id=4b5f80656fc94b88, https://usegalaxy.org.au/published/workflow?id=4cb69c84b73ea9dd, https://usegalaxy.org.au/published/workflow?id=51bad198f1fd1215, https://usegalaxy.org.au/published/workflow?id=5550585ffc3d8d0e, https://usegalaxy.org.au/published/workflow?id=5b0c0e34394afed1, https://usegalaxy.org.au/published/workflow?id=6ec0031406b10635, https://usegalaxy.org.au/published/workflow?id=6f76c43e6d2d7f74, https://usegalaxy.org.au/published/workflow?id=970dc0d8bd74707f, https://usegalaxy.org.au/published/workflow?id=98f660971010d11d, https://usegalaxy.org.au/published/workflow?id=ad556db35858e7ec, https://usegalaxy.org.au/published/workflow?id=ae2a397837b232b1, https://usegalaxy.org.au/published/workflow?id=be96bb47b6d6435a, https://usegalaxy.org.au/published/workflow?id=c43744d7dd9c0c0b, https://usegalaxy.org.au/published/workflow?id=c7e1f666a3366823, https://usegalaxy.org.au/published/workflow?id=cbf022e349d726d9, https://usegalaxy.org.au/published/workflow?id=d678e57c7f1baf5d, https://usegalaxy.org.au/published/workflow?id=d78879cfaa5948b3, https://usegalaxy.org.au/published/workflow?id=df4e186069552649, https://usegalaxy.org.au/published/workflow?id=e5f1ce106ea7af12, https://usegalaxy.org.au/published/workflow?id=f38f610e0d95915f, https://usegalaxy.org.au/published/workflow?id=f53d8ade897b3f46, https://usegalaxy.org/published/workflow?id=1b434bfc3aac0022, https://usegalaxy.org/published/workflow?id=2acbb8c5f72b9f1c, https://usegalaxy.org/published/workflow?id=2f1469dab87a3649, https://usegalaxy.org/published/workflow?id=2ff2961f044cac0f, https://usegalaxy.org/published/workflow?id=31387ed5ebc88cd2, https://usegalaxy.org/published/workflow?id=3a92253f57e553bd, https://usegalaxy.org/published/workflow?id=3b0d42747f598f82, https://usegalaxy.org/published/workflow?id=3ba73f679558017c, https://usegalaxy.org/published/workflow?id=435f0340bf00c940, https://usegalaxy.org/published/workflow?id=4403ca7fe447a5d7, https://usegalaxy.org/published/workflow?id=49a07807fcea8158, https://usegalaxy.org/published/workflow?id=4bf38f5b45faa8a1, https://usegalaxy.org/published/workflow?id=4d5cc5010537f172, https://usegalaxy.org/published/workflow?id=578e634b63923e9b, https://usegalaxy.org/published/workflow?id=597c4d54f8a8ebe7, https://usegalaxy.org/published/workflow?id=67252039bce060b2, https://usegalaxy.org/published/workflow?id=6a27c0d224683f73, https://usegalaxy.org/published/workflow?id=7fd68212ce020e6a, https://usegalaxy.org/published/workflow?id=81f1f0fd6d9235fd, https://usegalaxy.org/published/workflow?id=84161f4f8bf77474, https://usegalaxy.org/published/workflow?id=8f7c9461910b08f1, https://usegalaxy.org/published/workflow?id=90f5b512997c70f8, https://usegalaxy.org/published/workflow?id=91a4daecbb97d03e, https://usegalaxy.org/published/workflow?id=936ed7447c8e0a2d, https://usegalaxy.org/published/workflow?id=950c7245a6c3f6e9, https://usegalaxy.org/published/workflow?id=b11c6268704dcc1b, https://usegalaxy.org/published/workflow?id=b5644a5dcf237ef9, https://usegalaxy.org/published/workflow?id=b6d535f210efa18e, https://usegalaxy.org/published/workflow?id=bbc894a79eb0e769, https://usegalaxy.org/published/workflow?id=bfd2e7e63d3e6856, https://usegalaxy.org/published/workflow?id=c03ada2070e9ca8a, https://usegalaxy.org/published/workflow?id=c694856c01ce7cc5, https://usegalaxy.org/published/workflow?id=c95e9116e4b7982b, https://usegalaxy.org/published/workflow?id=ce04658f05029be2, https://usegalaxy.org/published/workflow?id=d08135fa13d1745e, https://usegalaxy.org/published/workflow?id=d36765ac28c11e1a, https://usegalaxy.org/published/workflow?id=d4b2b85db47c405e, https://usegalaxy.org/published/workflow?id=e12e642b88a0fe08, https://usegalaxy.org/published/workflow?id=f60027545d40b82d, https://usegalaxy.org/published/workflow?id=fb98ef955b3dbd4b, https://workflowhub.eu/workflows/612?version=25, https://workflowhub.eu/workflows/625?version=29, https://workflowhub.eu/workflows/633?version=23, https://workflowhub.eu/workflows/641?version=32, https://workflowhub.eu/workflows/642?version=25, https://workflowhub.eu/workflows/751?version=18</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>concoct</td>
      <td>concoct, concoct_coverage_table, concoct_cut_up_fasta, concoct_extract_fasta_bins, concoct_merge_cut_up_clustering</td>
      <td>CONCOCT (Clustering cONtigs with COverage and ComposiTion) does unsupervised binning of metagenomic contigs byusing nucleotide composition - kmer frequencies - and coverage data for multiple samples. CONCOCT can accurately(up to species level) bin metagenomic contigs.</td>
      <td>2022-02-18</td>
      <td>https://github.com/BinPro/CONCOCT</td>
      <td>1.1.0</td>
      <td>concoct</td>
      <td>1.1.0</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Sequence clustering, Read binning</td>
      <td>Sequence clustering, Read binning</td>
      <td>Metagenomics</td>
      <td>Metagenomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/concoct</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/concoct</td>
      <td>concoct</td>
      <td>CONCOCT</td>
      <td>A program for unsupervised binning of metagenomic contigs by using nucleotide composition, coverage data in multiple samples and linkage data from paired end reads.</td>
      <td></td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>107</td>
      <td>107</td>
      <td>9955</td>
      <td>9955</td>
      <td>19</td>
      <td>19</td>
      <td>187</td>
      <td>187</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>3085</td>
      <td>3085</td>
      <td>13227</td>
      <td>13227</td>
      <td>130</td>
      <td>130</td>
      <td>https://usegalaxy.eu/published/workflow?id=00f15824c7a9b196, https://usegalaxy.eu/published/workflow?id=0563f58718be932d, https://usegalaxy.eu/published/workflow?id=1e5f1d9a51032e44, https://usegalaxy.eu/published/workflow?id=43098b8afe78b071, https://usegalaxy.eu/published/workflow?id=58f779d238d569dc, https://usegalaxy.eu/published/workflow?id=7ecd92cb382d8168, https://usegalaxy.eu/published/workflow?id=97312d273b6e8bd9, https://usegalaxy.eu/published/workflow?id=ce5c9287a67e6085, https://workflowhub.eu/workflows/1352?version=4, https://workflowhub.eu/workflows/2028?version=1</td>
      <td>microbiome/metagenomics-binning</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>contig_annotation_tool</td>
      <td>cat_add_names, cat_bins, cat_contigs, cat_prepare, cat_summarise</td>
      <td>Contig Annotation Tool (CAT)</td>
      <td>2019-11-27</td>
      <td>https://github.com/dutilh/CAT</td>
      <td>5.2.3</td>
      <td>cat</td>
      <td>6.0.1</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Taxonomic classification, Sequence assembly, Coding region prediction</td>
      <td>Taxonomic classification, Sequence assembly, Coding region prediction</td>
      <td>Metagenomics, Metagenomic sequencing, Taxonomy, Sequence assembly</td>
      <td>Metagenomics, Metagenomic sequencing, Taxonomy, Sequence assembly</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cat</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cat</td>
      <td>cat_bins</td>
      <td>CAT and BAT</td>
      <td>Contig Annotation Tool (CAT) and Bin Annotation Tool (BAT) are pipelines for the taxonomic classification of long DNA sequences and metagenome assembled genomes (MAGs/bins) of both known and (highly) unknown microorganisms, as generated by contemporary metagenomics studies. The core algorithm of both programs involves gene calling, mapping of predicted ORFs against the nr protein database, and voting-based classification of the entire contig / MAG based on classification of the individual ORFs.</td>
      <td></td>
      <td>5</td>
      <td>2</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>257</td>
      <td>267</td>
      <td>17418</td>
      <td>17676</td>
      <td>216</td>
      <td>231</td>
      <td>1590</td>
      <td>1750</td>
      <td>84</td>
      <td>85</td>
      <td>848</td>
      <td>875</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>20301</td>
      <td>19856</td>
      <td>583</td>
      <td>557</td>
      <td>https://usegalaxy.eu/published/workflow?id=02631612a43a4d9b, https://usegalaxy.eu/published/workflow?id=3eaa913306c16146, https://usegalaxy.eu/published/workflow?id=6158c3b5fc12044e, https://usegalaxy.eu/published/workflow?id=6416cfdbd5273063, https://usegalaxy.eu/published/workflow?id=667bac8d7453e5f1, https://usegalaxy.eu/published/workflow?id=8deb08d4f228e198, https://usegalaxy.eu/published/workflow?id=9a97c6eb4fe015e7, https://usegalaxy.eu/published/workflow?id=d57d41e306241396, https://usegalaxy.org/published/workflow?id=33d90e718ce500ef</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>coverm</td>
      <td>coverm_contig, coverm_genome</td>
      <td>CoverM genome and contig wrappers</td>
      <td>2022-04-26</td>
      <td>https://github.com/wwood/CoverM</td>
      <td>0.7.0</td>
      <td>coverm</td>
      <td>0.7.0</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Local alignment</td>
      <td>Local alignment</td>
      <td>Metagenomics, Sequence analysis</td>
      <td>Metagenomics, Sequence analysis</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/coverm</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/coverm</td>
      <td>coverm</td>
      <td>CoverM</td>
      <td>Read coverage calculator for metagenomics</td>
      <td></td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>147</td>
      <td>147</td>
      <td>3594</td>
      <td>3594</td>
      <td>14</td>
      <td>14</td>
      <td>414</td>
      <td>414</td>
      <td>11</td>
      <td>11</td>
      <td>125</td>
      <td>125</td>
      <td>7</td>
      <td>7</td>
      <td>565</td>
      <td>565</td>
      <td>4698</td>
      <td>4698</td>
      <td>179</td>
      <td>179</td>
      <td>https://usegalaxy.eu/published/workflow?id=0563f58718be932d, https://usegalaxy.eu/published/workflow?id=1e5f1d9a51032e44, https://usegalaxy.eu/published/workflow?id=2ee0eb74e839b476, https://usegalaxy.eu/published/workflow?id=6158c3b5fc12044e, https://usegalaxy.eu/published/workflow?id=62392daea25a3980, https://usegalaxy.eu/published/workflow?id=667bac8d7453e5f1, https://usegalaxy.eu/published/workflow?id=7ecd92cb382d8168, https://usegalaxy.eu/published/workflow?id=8deb08d4f228e198, https://usegalaxy.eu/published/workflow?id=97312d273b6e8bd9, https://usegalaxy.eu/published/workflow?id=d57d41e306241396, https://workflowhub.eu/workflows/1352?version=4, https://workflowhub.eu/workflows/1390?version=2, https://workflowhub.eu/workflows/2024?version=2</td>
      <td>assembly/metagenomics-assembly, microbiome/metagenomics-assembly</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>cryptogenotyper</td>
      <td>CryptoGenotyper</td>
      <td>CryptoGenotyper is a standalone tool to *in-silico*  determine species and subtype based on SSU rRNA (18S) and gp60 markers.</td>
      <td>2020-10-14</td>
      <td>https://github.com/phac-nml/CryptoGenotyper</td>
      <td>1.5.0</td>
      <td>cryptogenotyper</td>
      <td>1.5.0</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>nml</td>
      <td>https://github.com/phac-nml/CryptoGenotyper</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/cryptogenotyper</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>31</td>
      <td>31</td>
      <td>12430</td>
      <td>12430</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>12430</td>
      <td>12430</td>
      <td>31</td>
      <td>31</td>
      <td>https://usegalaxy.eu/published/workflow?id=07865c770825c54a, https://usegalaxy.eu/published/workflow?id=280877e42b09f49f, https://usegalaxy.eu/published/workflow?id=3d7659452e3af984, https://usegalaxy.eu/published/workflow?id=4ba4da05a7696923, https://usegalaxy.eu/published/workflow?id=aefe927f13bc9374, https://usegalaxy.eu/published/workflow?id=f7558edf450e781f</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>cutadapt</td>
      <td>cutadapt</td>
      <td>Flexible tool to remove adapter sequences (and quality trim) high throughput sequencing reads (fasta/fastq).</td>
      <td>2018-02-20</td>
      <td>https://cutadapt.readthedocs.org/en/stable/</td>
      <td>5.2</td>
      <td>cutadapt</td>
      <td>5.2</td>
      <td>Up-to-date</td>
      <td>Fasta Manipulation, Fastq Manipulation, Sequence Analysis</td>
      <td>Sequence trimming, Primer removal, Read pre-processing</td>
      <td>Sequence trimming, Primer removal, Read pre-processing</td>
      <td>Genomics, Probes and primers, Sequencing</td>
      <td>Genomics, Probes and primers, Sequencing</td>
      <td>lparsons</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cutadapt</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cutadapt</td>
      <td>cutadapt</td>
      <td>Cutadapt</td>
      <td>Find and remove adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>14131</td>
      <td>16253</td>
      <td>330892</td>
      <td>364942</td>
      <td>22476</td>
      <td>26008</td>
      <td>409500</td>
      <td>482165</td>
      <td>4256</td>
      <td>4631</td>
      <td>79748</td>
      <td>86014</td>
      <td>1579</td>
      <td>1582</td>
      <td>59464</td>
      <td>59550</td>
      <td>992671</td>
      <td>879604</td>
      <td>48474</td>
      <td>42442</td>
      <td>https://usegalaxy.eu/published/workflow?id=002a51bdef00e14d, https://usegalaxy.eu/published/workflow?id=01fe4c590e608103, https://usegalaxy.eu/published/workflow?id=023a0ee62dd1f593, https://usegalaxy.eu/published/workflow?id=02655d0e4af0ee0d, https://usegalaxy.eu/published/workflow?id=03be9d62e8e4a129, https://usegalaxy.eu/published/workflow?id=06fe33c4420906bf, https://usegalaxy.eu/published/workflow?id=07aefca769095616, https://usegalaxy.eu/published/workflow?id=085c3a4c89d137b5, https://usegalaxy.eu/published/workflow?id=089e5e83b1de0511, https://usegalaxy.eu/published/workflow?id=096b75501c8e0888, https://usegalaxy.eu/published/workflow?id=0c619b25c1394a09, https://usegalaxy.eu/published/workflow?id=0cf3c2114ab65858, https://usegalaxy.eu/published/workflow?id=0eb137cd936bbfe0, https://usegalaxy.eu/published/workflow?id=1089e5c66dbcc1a3, https://usegalaxy.eu/published/workflow?id=111527fe542846af, https://usegalaxy.eu/published/workflow?id=1151052f4393852a, https://usegalaxy.eu/published/workflow?id=146cef3acb2eba75, https://usegalaxy.eu/published/workflow?id=1483534f49b826bf, https://usegalaxy.eu/published/workflow?id=14b9753269cd368f, https://usegalaxy.eu/published/workflow?id=153ba1581743d31f, https://usegalaxy.eu/published/workflow?id=165089132248a0b0, https://usegalaxy.eu/published/workflow?id=166eb3f14feffa1b, https://usegalaxy.eu/published/workflow?id=17b0a6d3379a4392, https://usegalaxy.eu/published/workflow?id=194a05d42687b524, https://usegalaxy.eu/published/workflow?id=19bb57f8cfa48bb9, https://usegalaxy.eu/published/workflow?id=1ac0a15758277730, https://usegalaxy.eu/published/workflow?id=1bb084f1bc102734, https://usegalaxy.eu/published/workflow?id=1c44bbaa5f4f8b54, https://usegalaxy.eu/published/workflow?id=1cba6eb4aff8dc61, https://usegalaxy.eu/published/workflow?id=1d1092f57e418c57, https://usegalaxy.eu/published/workflow?id=1da7ac8eaf891f39, https://usegalaxy.eu/published/workflow?id=1dba8278fb2786fb, https://usegalaxy.eu/published/workflow?id=1e02a7292e9974b1, https://usegalaxy.eu/published/workflow?id=1ea8bfbe6cb04531, https://usegalaxy.eu/published/workflow?id=1eac2ebffcbb8d59, https://usegalaxy.eu/published/workflow?id=1f2a1438e41188f2, https://usegalaxy.eu/published/workflow?id=21a3f72851f29925, https://usegalaxy.eu/published/workflow?id=21c8ee4af52a3931, https://usegalaxy.eu/published/workflow?id=249c5d6698c8b98c, https://usegalaxy.eu/published/workflow?id=24ddd188da35b061, https://usegalaxy.eu/published/workflow?id=25a209bda998412f, https://usegalaxy.eu/published/workflow?id=25aa944237d7fb82, https://usegalaxy.eu/published/workflow?id=2671a5f85b2cccfe, https://usegalaxy.eu/published/workflow?id=26c4882e320ed7b3, https://usegalaxy.eu/published/workflow?id=270b6f82efab28b2, https://usegalaxy.eu/published/workflow?id=29a855d928337f8f, https://usegalaxy.eu/published/workflow?id=2c5cae1389c11881, https://usegalaxy.eu/published/workflow?id=2cf9f0e2ab5a7f8b, https://usegalaxy.eu/published/workflow?id=2e418c5810012e5a, https://usegalaxy.eu/published/workflow?id=2f51a82838722bec, https://usegalaxy.eu/published/workflow?id=2fd462d85412444c, https://usegalaxy.eu/published/workflow?id=300301c23327f797, https://usegalaxy.eu/published/workflow?id=30e7dc81e323cbdb, https://usegalaxy.eu/published/workflow?id=3183c2abca463bb4, https://usegalaxy.eu/published/workflow?id=326d4102190b74fc, https://usegalaxy.eu/published/workflow?id=342a5df69951e3d7, https://usegalaxy.eu/published/workflow?id=379642036b6f5a66, https://usegalaxy.eu/published/workflow?id=39b195a36408277c, https://usegalaxy.eu/published/workflow?id=3a3e2d50fe7a9ee3, https://usegalaxy.eu/published/workflow?id=3b8ef7905c902b27, https://usegalaxy.eu/published/workflow?id=3ce75b5de4922cba, https://usegalaxy.eu/published/workflow?id=3e255a5b7026d26e, https://usegalaxy.eu/published/workflow?id=3eb6663a66f8e69a, https://usegalaxy.eu/published/workflow?id=3f5414bb38b47fcf, https://usegalaxy.eu/published/workflow?id=406e6ef0dfb38628, https://usegalaxy.eu/published/workflow?id=409336bf210033a2, https://usegalaxy.eu/published/workflow?id=416141b245073406, https://usegalaxy.eu/published/workflow?id=4176b9d43325a8d8, https://usegalaxy.eu/published/workflow?id=44a2b6c8ae54eb31, https://usegalaxy.eu/published/workflow?id=44c5999436da5f1e, https://usegalaxy.eu/published/workflow?id=45ae834d842302c3, https://usegalaxy.eu/published/workflow?id=45f63151bf4f1a49, https://usegalaxy.eu/published/workflow?id=471eb8b4c54f897d, https://usegalaxy.eu/published/workflow?id=475ea16f943cc013, https://usegalaxy.eu/published/workflow?id=478190607a114390, https://usegalaxy.eu/published/workflow?id=4856e3522b2ee059, https://usegalaxy.eu/published/workflow?id=48f63ef14a12f306, https://usegalaxy.eu/published/workflow?id=497b82d3b92dc076, https://usegalaxy.eu/published/workflow?id=4a51e616d96338ed, https://usegalaxy.eu/published/workflow?id=4c3a9b574f37eebb, https://usegalaxy.eu/published/workflow?id=4d32b1b3ac0a7161, https://usegalaxy.eu/published/workflow?id=4d41ee83fc0040d4, https://usegalaxy.eu/published/workflow?id=4daa0c33d6d50379, https://usegalaxy.eu/published/workflow?id=503a2f4caacaf63e, https://usegalaxy.eu/published/workflow?id=5115299e77df8104, https://usegalaxy.eu/published/workflow?id=5138874b208adb4b, https://usegalaxy.eu/published/workflow?id=519e0ffdfd1790bc, https://usegalaxy.eu/published/workflow?id=5279af5c95ae4f7a, https://usegalaxy.eu/published/workflow?id=54590ccd11ffbf65, https://usegalaxy.eu/published/workflow?id=54b91dd86f463391, https://usegalaxy.eu/published/workflow?id=55ead91471bf1008, https://usegalaxy.eu/published/workflow?id=56523b41ff7eec76, https://usegalaxy.eu/published/workflow?id=572091280033312e, https://usegalaxy.eu/published/workflow?id=57cc33ed64d71982, https://usegalaxy.eu/published/workflow?id=5ad406442ec21345, https://usegalaxy.eu/published/workflow?id=5e1906290f5b4d8a, https://usegalaxy.eu/published/workflow?id=5f0faac42b0a5062, https://usegalaxy.eu/published/workflow?id=6514e610c79df72c, https://usegalaxy.eu/published/workflow?id=6573f3b1606b7f09, https://usegalaxy.eu/published/workflow?id=66834f10c86362b9, https://usegalaxy.eu/published/workflow?id=67be21795d403ca3, https://usegalaxy.eu/published/workflow?id=69cdd404592535ec, https://usegalaxy.eu/published/workflow?id=6ebd09100fc98878, https://usegalaxy.eu/published/workflow?id=7137e45ca8dc6e4c, https://usegalaxy.eu/published/workflow?id=71a6d128a09210f0, https://usegalaxy.eu/published/workflow?id=7337d15d121596da, https://usegalaxy.eu/published/workflow?id=74e4a8f2edb25f1a, https://usegalaxy.eu/published/workflow?id=77d2adf5249eb826, https://usegalaxy.eu/published/workflow?id=79b5afb125e3c3da, https://usegalaxy.eu/published/workflow?id=79fd2517af48c594, https://usegalaxy.eu/published/workflow?id=7ab70660e6235cf0, https://usegalaxy.eu/published/workflow?id=7d84b0419296857d, https://usegalaxy.eu/published/workflow?id=7e0ba4c14803448e, https://usegalaxy.eu/published/workflow?id=8132c3b710fd6d48, https://usegalaxy.eu/published/workflow?id=830eb4faff21eb0c, https://usegalaxy.eu/published/workflow?id=847403b72dd346ff, https://usegalaxy.eu/published/workflow?id=86e2e2faa2e0bd46, https://usegalaxy.eu/published/workflow?id=873e6ac4699ed7f6, https://usegalaxy.eu/published/workflow?id=87cc0cd143a12c41, https://usegalaxy.eu/published/workflow?id=8b1e2fe9f4a07de0, https://usegalaxy.eu/published/workflow?id=8be366ad6a42408b, https://usegalaxy.eu/published/workflow?id=8fb156020285e049, https://usegalaxy.eu/published/workflow?id=906a64781295b08c, https://usegalaxy.eu/published/workflow?id=9207e74a5bf7d8e1, https://usegalaxy.eu/published/workflow?id=92760fc19ef120ce, https://usegalaxy.eu/published/workflow?id=92dafb3d1c06e074, https://usegalaxy.eu/published/workflow?id=93a7047396080fda, https://usegalaxy.eu/published/workflow?id=9472be79da012999, https://usegalaxy.eu/published/workflow?id=94b9287365ea4d0d, https://usegalaxy.eu/published/workflow?id=95f420d8856bf725, https://usegalaxy.eu/published/workflow?id=985044b4b2c51425, https://usegalaxy.eu/published/workflow?id=994e5ff1df7d6c9e, https://usegalaxy.eu/published/workflow?id=9ad5aff92e185907, https://usegalaxy.eu/published/workflow?id=9c0b67496dfe52b2, https://usegalaxy.eu/published/workflow?id=9c24df68dc7d25a4, https://usegalaxy.eu/published/workflow?id=9c7a218993788493, https://usegalaxy.eu/published/workflow?id=9da26c5131603b9f, https://usegalaxy.eu/published/workflow?id=9e03cf8cd5e5de08, https://usegalaxy.eu/published/workflow?id=9f2d18f0d0896e8b, https://usegalaxy.eu/published/workflow?id=a1e09c648d8f1fc7, https://usegalaxy.eu/published/workflow?id=a28c207730e7e13d, https://usegalaxy.eu/published/workflow?id=a3da32b6b45da639, https://usegalaxy.eu/published/workflow?id=a461ea397418f3aa, https://usegalaxy.eu/published/workflow?id=a6dd62ab4b9a0725, https://usegalaxy.eu/published/workflow?id=a76629eccfdeafb8, https://usegalaxy.eu/published/workflow?id=a8f829deb69b30ab, https://usegalaxy.eu/published/workflow?id=a9c1dca5da7170dd, https://usegalaxy.eu/published/workflow?id=aa6dc57f5c67fb4b, https://usegalaxy.eu/published/workflow?id=ac6eb99f70d35999, https://usegalaxy.eu/published/workflow?id=ae74f60cd20d56bc, https://usegalaxy.eu/published/workflow?id=b211fceaecf41c9e, https://usegalaxy.eu/published/workflow?id=b323867a9efcc6c6, https://usegalaxy.eu/published/workflow?id=b4948afacccfefdb, https://usegalaxy.eu/published/workflow?id=b5e3360f933d1950, https://usegalaxy.eu/published/workflow?id=b619339f4e1c6fdf, https://usegalaxy.eu/published/workflow?id=b84fd6a0e2632ad9, https://usegalaxy.eu/published/workflow?id=b8baade87e733fce, https://usegalaxy.eu/published/workflow?id=b9ad204395f7d931, https://usegalaxy.eu/published/workflow?id=bace9aebab9d4ce3, https://usegalaxy.eu/published/workflow?id=bc87309a8763d0b4, https://usegalaxy.eu/published/workflow?id=bcf99cc0469db276, https://usegalaxy.eu/published/workflow?id=bef0a7d604f2ebef, https://usegalaxy.eu/published/workflow?id=c0d1e965daf89006, https://usegalaxy.eu/published/workflow?id=c2d814f2182c0082, https://usegalaxy.eu/published/workflow?id=c3eba26c5d7febff, https://usegalaxy.eu/published/workflow?id=c4c2e669b86cffbc, https://usegalaxy.eu/published/workflow?id=c4fa05e209a2f082, https://usegalaxy.eu/published/workflow?id=c508c00e81018dd8, https://usegalaxy.eu/published/workflow?id=c6ca9e6fd28979f8, https://usegalaxy.eu/published/workflow?id=c7a8f7aabc219831, https://usegalaxy.eu/published/workflow?id=c7f34c67c9054234, https://usegalaxy.eu/published/workflow?id=c98a14fcf5998159, https://usegalaxy.eu/published/workflow?id=c9aff55e39e195e3, https://usegalaxy.eu/published/workflow?id=caba6d3d7db648e9, https://usegalaxy.eu/published/workflow?id=cc37667c2d1f9056, https://usegalaxy.eu/published/workflow?id=ccef778cfd9024d1, https://usegalaxy.eu/published/workflow?id=ce5c16ac23c4fbb1, https://usegalaxy.eu/published/workflow?id=cfe47f0c78ed5ec4, https://usegalaxy.eu/published/workflow?id=d05c6f0f3205fbd2, https://usegalaxy.eu/published/workflow?id=d12617d0df5f81d0, https://usegalaxy.eu/published/workflow?id=d3494c6124f820b2, https://usegalaxy.eu/published/workflow?id=d3edc3f0659a5e5b, https://usegalaxy.eu/published/workflow?id=d41b52a15ba56f54, https://usegalaxy.eu/published/workflow?id=d67299c34e63f3d9, https://usegalaxy.eu/published/workflow?id=d6809603f46de1b3, https://usegalaxy.eu/published/workflow?id=d768c0e0df4d432e, https://usegalaxy.eu/published/workflow?id=d83280cd72ec0f33, https://usegalaxy.eu/published/workflow?id=dc27453c438577f6, https://usegalaxy.eu/published/workflow?id=de9003e589631029, https://usegalaxy.eu/published/workflow?id=de9cf573314f416c, https://usegalaxy.eu/published/workflow?id=dfc2c53ac3103eff, https://usegalaxy.eu/published/workflow?id=dfe3b555856c7b44, https://usegalaxy.eu/published/workflow?id=e138640778b7414c, https://usegalaxy.eu/published/workflow?id=e1947b059ef131d2, https://usegalaxy.eu/published/workflow?id=e2f936f1aa43a629, https://usegalaxy.eu/published/workflow?id=e7aebe58411d97ed, https://usegalaxy.eu/published/workflow?id=e8eb80ee09159bac, https://usegalaxy.eu/published/workflow?id=e8f9e1d911683b22, https://usegalaxy.eu/published/workflow?id=e935d2c8ffb52588, https://usegalaxy.eu/published/workflow?id=ead0f264798f9418, https://usegalaxy.eu/published/workflow?id=ebbf5cfa946d11d3, https://usegalaxy.eu/published/workflow?id=ec79f14ab91cd933, https://usegalaxy.eu/published/workflow?id=ef1d11b79a2e1727, https://usegalaxy.eu/published/workflow?id=f14ebad1c3c20067, https://usegalaxy.eu/published/workflow?id=f1ca4289ee4f67bb, https://usegalaxy.eu/published/workflow?id=f33b5135ee99e0dc, https://usegalaxy.eu/published/workflow?id=f38745208494d4ab, https://usegalaxy.eu/published/workflow?id=f5005e80d9ffa109, https://usegalaxy.eu/published/workflow?id=f578a097b4339025, https://usegalaxy.eu/published/workflow?id=f5be5bcf9b9f171c, https://usegalaxy.eu/published/workflow?id=f6a625946dea60fa, https://usegalaxy.eu/published/workflow?id=f6b55599db1ab056, https://usegalaxy.eu/published/workflow?id=f6e8a63283262518, https://usegalaxy.eu/published/workflow?id=fa0ff9a0db8bbddb, https://usegalaxy.eu/published/workflow?id=fa53b172def3fe75, https://usegalaxy.eu/published/workflow?id=fb015e398373ea8a, https://usegalaxy.eu/published/workflow?id=fba1671d835b6dd3, https://usegalaxy.eu/published/workflow?id=fc25c1f8c6667c87, https://usegalaxy.eu/published/workflow?id=fc956fb42ef2852e, https://usegalaxy.eu/published/workflow?id=fd6429a788660c8d, https://usegalaxy.eu/published/workflow?id=fd649ac4d4872c81, https://usegalaxy.eu/published/workflow?id=fdca6c9d394986bf, https://usegalaxy.eu/published/workflow?id=ff4d5beb6ce1809d, https://usegalaxy.fr/published/workflow?id=5d83e0886f2a7465, https://usegalaxy.fr/published/workflow?id=826cffb2458e4067, https://usegalaxy.fr/published/workflow?id=b5d86d505c09d170, https://usegalaxy.fr/published/workflow?id=c4f6062e51d0f45e, https://usegalaxy.fr/published/workflow?id=e6991d64e851b09e, https://usegalaxy.org.au/published/workflow?id=000cdc969e9f0886, https://usegalaxy.org.au/published/workflow?id=01997cb92a2ce2d5, https://usegalaxy.org.au/published/workflow?id=0220e60c66de9065, https://usegalaxy.org.au/published/workflow?id=02b8d033d9b1fa05, https://usegalaxy.org.au/published/workflow?id=02eebe3d909206ab, https://usegalaxy.org.au/published/workflow?id=046a1c600a6b77c2, https://usegalaxy.org.au/published/workflow?id=05b17b9d70c1db8b, https://usegalaxy.org.au/published/workflow?id=0614f24ebeb680de, https://usegalaxy.org.au/published/workflow?id=068f85b8b137bfd7, https://usegalaxy.org.au/published/workflow?id=07c97b9e59c0218f, https://usegalaxy.org.au/published/workflow?id=08fac3b4c128e597, https://usegalaxy.org.au/published/workflow?id=0af0ea50f1237952, https://usegalaxy.org.au/published/workflow?id=0c567180448d8774, https://usegalaxy.org.au/published/workflow?id=0cc8c8875df6bccb, https://usegalaxy.org.au/published/workflow?id=0cfc1408fd9ea4e7, https://usegalaxy.org.au/published/workflow?id=0d1db28665d19bda, https://usegalaxy.org.au/published/workflow?id=0f7406ada6706c91, https://usegalaxy.org.au/published/workflow?id=10355b2680374a03, https://usegalaxy.org.au/published/workflow?id=10acf4f497172400, https://usegalaxy.org.au/published/workflow?id=114fb9589bb9db92, https://usegalaxy.org.au/published/workflow?id=115151c2c8998d44, https://usegalaxy.org.au/published/workflow?id=126116a92f2e9a68, https://usegalaxy.org.au/published/workflow?id=14b17f17d03c8424, https://usegalaxy.org.au/published/workflow?id=158b5d9107a05d95, https://usegalaxy.org.au/published/workflow?id=16f4846b4d6bc155, https://usegalaxy.org.au/published/workflow?id=178db14511c8f293, https://usegalaxy.org.au/published/workflow?id=18d48f1b26811372, https://usegalaxy.org.au/published/workflow?id=19315e6573c58239, https://usegalaxy.org.au/published/workflow?id=1a107e6611fb9c15, https://usegalaxy.org.au/published/workflow?id=1ab66d382630afa7, https://usegalaxy.org.au/published/workflow?id=1ad81f95ace878ea, https://usegalaxy.org.au/published/workflow?id=1c886f008a36b0b8, https://usegalaxy.org.au/published/workflow?id=1eb3e1452c672480, https://usegalaxy.org.au/published/workflow?id=1f63ec596e578b1c, https://usegalaxy.org.au/published/workflow?id=1ff259845498b818, https://usegalaxy.org.au/published/workflow?id=20e5e6b233ffb951, https://usegalaxy.org.au/published/workflow?id=210ca013cc1cf185, https://usegalaxy.org.au/published/workflow?id=2470cb61ba51288c, https://usegalaxy.org.au/published/workflow?id=2543e425fa6d5144, https://usegalaxy.org.au/published/workflow?id=25ac9b4f2e39af97, https://usegalaxy.org.au/published/workflow?id=25f65a29ce3c3ee0, https://usegalaxy.org.au/published/workflow?id=2617fee24459e680, https://usegalaxy.org.au/published/workflow?id=2789fb3590727342, https://usegalaxy.org.au/published/workflow?id=2980172ed0226b59, https://usegalaxy.org.au/published/workflow?id=2999a1891ffe201b, https://usegalaxy.org.au/published/workflow?id=2a0046582f5f15a1, https://usegalaxy.org.au/published/workflow?id=2afeae1ec03c0b57, https://usegalaxy.org.au/published/workflow?id=2c58509f80897972, https://usegalaxy.org.au/published/workflow?id=2cef9a1633e0a067, https://usegalaxy.org.au/published/workflow?id=2d45d7138d8715b4, https://usegalaxy.org.au/published/workflow?id=2f520cb446adf726, https://usegalaxy.org.au/published/workflow?id=2f7c0e3221cbca2f, https://usegalaxy.org.au/published/workflow?id=301fdb29c513ffbe, https://usegalaxy.org.au/published/workflow?id=30b2cd025e0300da, https://usegalaxy.org.au/published/workflow?id=31449596b22f81f4, https://usegalaxy.org.au/published/workflow?id=319e9d693fac8dce, https://usegalaxy.org.au/published/workflow?id=32d28f800adcea42, https://usegalaxy.org.au/published/workflow?id=35384818ee1cbac2, https://usegalaxy.org.au/published/workflow?id=3604307ec2835628, https://usegalaxy.org.au/published/workflow?id=36f54eda8568355d, https://usegalaxy.org.au/published/workflow?id=371c9892d261cee2, https://usegalaxy.org.au/published/workflow?id=374311292d72c9f9, https://usegalaxy.org.au/published/workflow?id=383ce49fa7a367e9, https://usegalaxy.org.au/published/workflow?id=39d56981a6854d82, https://usegalaxy.org.au/published/workflow?id=3bc45aaed5da1058, https://usegalaxy.org.au/published/workflow?id=3c30c822058187be, https://usegalaxy.org.au/published/workflow?id=3c727d5cd349a846, https://usegalaxy.org.au/published/workflow?id=3ce9dc804780e4a1, https://usegalaxy.org.au/published/workflow?id=3df7f45b7b388f8c, https://usegalaxy.org.au/published/workflow?id=3e7fcfd23872ad0d, https://usegalaxy.org.au/published/workflow?id=3ef07022656067b1, https://usegalaxy.org.au/published/workflow?id=40666ceb68d101ac, https://usegalaxy.org.au/published/workflow?id=408a31694e9c85e6, https://usegalaxy.org.au/published/workflow?id=410d0d19b6589c78, https://usegalaxy.org.au/published/workflow?id=41795303a88d9e94, https://usegalaxy.org.au/published/workflow?id=41e005d4ebb5885a, https://usegalaxy.org.au/published/workflow?id=421adea2f259ec26, https://usegalaxy.org.au/published/workflow?id=4267bd1e4cfd1901, https://usegalaxy.org.au/published/workflow?id=43343fa1896f8278, https://usegalaxy.org.au/published/workflow?id=434127690544d0e9, https://usegalaxy.org.au/published/workflow?id=43f2dabbc1932e30, https://usegalaxy.org.au/published/workflow?id=447d8daefdefafee, https://usegalaxy.org.au/published/workflow?id=45c8789533c45abf, https://usegalaxy.org.au/published/workflow?id=47cacc06d404a8c6, https://usegalaxy.org.au/published/workflow?id=47cd0a90609fd557, https://usegalaxy.org.au/published/workflow?id=48a25f8e58d068e6, https://usegalaxy.org.au/published/workflow?id=4b5f80656fc94b88, https://usegalaxy.org.au/published/workflow?id=4bd6c5c12fb1a69a, https://usegalaxy.org.au/published/workflow?id=4e096a68187ce652, https://usegalaxy.org.au/published/workflow?id=4efed79608f85604, https://usegalaxy.org.au/published/workflow?id=4f2818b6620037af, https://usegalaxy.org.au/published/workflow?id=4f562265cbb14cd6, https://usegalaxy.org.au/published/workflow?id=51bad198f1fd1215, https://usegalaxy.org.au/published/workflow?id=52056336f24eccf6, https://usegalaxy.org.au/published/workflow?id=525612313de7591e, https://usegalaxy.org.au/published/workflow?id=5314a7a4732a0ef1, https://usegalaxy.org.au/published/workflow?id=544cb12a16f283b8, https://usegalaxy.org.au/published/workflow?id=5550585ffc3d8d0e, https://usegalaxy.org.au/published/workflow?id=56599572e0364e2a, https://usegalaxy.org.au/published/workflow?id=56bda8e4b0018e03, https://usegalaxy.org.au/published/workflow?id=57b558e1a76c09e1, https://usegalaxy.org.au/published/workflow?id=5a4e49ef8693d7c4, https://usegalaxy.org.au/published/workflow?id=5b0c0e34394afed1, https://usegalaxy.org.au/published/workflow?id=5be983b903469f07, https://usegalaxy.org.au/published/workflow?id=5cbee52cf03b6f9a, https://usegalaxy.org.au/published/workflow?id=5cd9b9338e75f254, https://usegalaxy.org.au/published/workflow?id=5ce00d3eab862f4e, https://usegalaxy.org.au/published/workflow?id=5d6773ed5b2f41cb, https://usegalaxy.org.au/published/workflow?id=5ea74382fb77eb10, https://usegalaxy.org.au/published/workflow?id=5f4f874e49515555, https://usegalaxy.org.au/published/workflow?id=608f3ad8f485e7a8, https://usegalaxy.org.au/published/workflow?id=60c0dec52cdca708, https://usegalaxy.org.au/published/workflow?id=6204bbcfe56931eb, https://usegalaxy.org.au/published/workflow?id=625f3c52832b0591, https://usegalaxy.org.au/published/workflow?id=63789b6f82172679, https://usegalaxy.org.au/published/workflow?id=6448b82959076a29, https://usegalaxy.org.au/published/workflow?id=6469ade8f6f7c718, https://usegalaxy.org.au/published/workflow?id=67191ea9fe977e08, https://usegalaxy.org.au/published/workflow?id=677ef3322366c953, https://usegalaxy.org.au/published/workflow?id=68b2961117e62890, https://usegalaxy.org.au/published/workflow?id=69233fb6edfcdde9, https://usegalaxy.org.au/published/workflow?id=6b2716309947836d, https://usegalaxy.org.au/published/workflow?id=6b34c95a247d15e3, https://usegalaxy.org.au/published/workflow?id=6b3df543536faf8c, https://usegalaxy.org.au/published/workflow?id=6c1c11a81a2ab1b8, https://usegalaxy.org.au/published/workflow?id=6c27019d03dd49d8, https://usegalaxy.org.au/published/workflow?id=6c9129acb9d98afe, https://usegalaxy.org.au/published/workflow?id=6cdbcd1f667d0f16, https://usegalaxy.org.au/published/workflow?id=6ec0031406b10635, https://usegalaxy.org.au/published/workflow?id=6efcacf14bfc18b3, https://usegalaxy.org.au/published/workflow?id=6f0801fecbba5d3f, https://usegalaxy.org.au/published/workflow?id=71bad95f72c9f30d, https://usegalaxy.org.au/published/workflow?id=723513b09d6690b8, https://usegalaxy.org.au/published/workflow?id=73ff1aa9acc0ad2a, https://usegalaxy.org.au/published/workflow?id=74186b006665daf8, https://usegalaxy.org.au/published/workflow?id=75113a1b41904d2a, https://usegalaxy.org.au/published/workflow?id=754149375c3069c4, https://usegalaxy.org.au/published/workflow?id=76323f103844f75b, https://usegalaxy.org.au/published/workflow?id=79522a1f7b0c53cb, https://usegalaxy.org.au/published/workflow?id=79ca45e45d76871f, https://usegalaxy.org.au/published/workflow?id=7a4ee63a04362967, https://usegalaxy.org.au/published/workflow?id=7ad9410e36ee9ea1, https://usegalaxy.org.au/published/workflow?id=7b203d52f496e2c7, https://usegalaxy.org.au/published/workflow?id=7ba13e928de65e61, https://usegalaxy.org.au/published/workflow?id=7becafd394472dfb, https://usegalaxy.org.au/published/workflow?id=7c2e7b292803b3e9, https://usegalaxy.org.au/published/workflow?id=7c866ed4a3552d0d, https://usegalaxy.org.au/published/workflow?id=7d268e7209a68e2e, https://usegalaxy.org.au/published/workflow?id=7ea616c652d08fb6, https://usegalaxy.org.au/published/workflow?id=7ef245f5ae3b916d, https://usegalaxy.org.au/published/workflow?id=7febc6ac6f106e57, https://usegalaxy.org.au/published/workflow?id=83680c1c6dffb05b, https://usegalaxy.org.au/published/workflow?id=83a535840b388980, https://usegalaxy.org.au/published/workflow?id=8433a2f79b2e5894, https://usegalaxy.org.au/published/workflow?id=8483ae422665a6c1, https://usegalaxy.org.au/published/workflow?id=8483e8d3e8d0ddc9, https://usegalaxy.org.au/published/workflow?id=8597166abda0a375, https://usegalaxy.org.au/published/workflow?id=85aede91db0122e4, https://usegalaxy.org.au/published/workflow?id=863eee1b041b6c58, https://usegalaxy.org.au/published/workflow?id=879d4fb110a61f7a, https://usegalaxy.org.au/published/workflow?id=88c0d875eea2ee49, https://usegalaxy.org.au/published/workflow?id=8926e592f2e60863, https://usegalaxy.org.au/published/workflow?id=897dfa78c3b25356, https://usegalaxy.org.au/published/workflow?id=897e9d7432d231e9, https://usegalaxy.org.au/published/workflow?id=8a6233015db6dd24, https://usegalaxy.org.au/published/workflow?id=8b8a5b50f2edc72c, https://usegalaxy.org.au/published/workflow?id=8bf8c9929249f37d, https://usegalaxy.org.au/published/workflow?id=8c40b4a911517cc5, https://usegalaxy.org.au/published/workflow?id=8cab31db0bd3ff52, https://usegalaxy.org.au/published/workflow?id=8d196c8c62530dc0, https://usegalaxy.org.au/published/workflow?id=8d47811c6154d347, https://usegalaxy.org.au/published/workflow?id=8d6010d07cc2e170, https://usegalaxy.org.au/published/workflow?id=8d6d4f848067f3d9, https://usegalaxy.org.au/published/workflow?id=8dd86497bc15d981, https://usegalaxy.org.au/published/workflow?id=8e8a5cce9f3f2f67, https://usegalaxy.org.au/published/workflow?id=8fb691706f47527a, https://usegalaxy.org.au/published/workflow?id=8fe84de499e96485, https://usegalaxy.org.au/published/workflow?id=91cfd96f697ba034, https://usegalaxy.org.au/published/workflow?id=91ee716be8bfd508, https://usegalaxy.org.au/published/workflow?id=925c167abd5610ae, https://usegalaxy.org.au/published/workflow?id=94c7fdcbbc7b428a, https://usegalaxy.org.au/published/workflow?id=95a9070469a895c3, https://usegalaxy.org.au/published/workflow?id=95efb414f5984735, https://usegalaxy.org.au/published/workflow?id=9642aadb83c4f752, https://usegalaxy.org.au/published/workflow?id=970dc0d8bd74707f, https://usegalaxy.org.au/published/workflow?id=975ddb0eb436f1fd, https://usegalaxy.org.au/published/workflow?id=97628b0604d5ec77, https://usegalaxy.org.au/published/workflow?id=9787c8b7adeedb0c, https://usegalaxy.org.au/published/workflow?id=978953f492b2b914, https://usegalaxy.org.au/published/workflow?id=9804030358c3f844, https://usegalaxy.org.au/published/workflow?id=981d7b02416b7734, https://usegalaxy.org.au/published/workflow?id=9898348d95033af2, https://usegalaxy.org.au/published/workflow?id=98d3def7da131b4c, https://usegalaxy.org.au/published/workflow?id=98f660971010d11d, https://usegalaxy.org.au/published/workflow?id=99821674db7682bb, https://usegalaxy.org.au/published/workflow?id=9988bc2402f32dc1, https://usegalaxy.org.au/published/workflow?id=9b18123afbe1264b, https://usegalaxy.org.au/published/workflow?id=9b1afa59c087eacd, https://usegalaxy.org.au/published/workflow?id=9bc31dff5bcec42b, https://usegalaxy.org.au/published/workflow?id=9df67170c9260539, https://usegalaxy.org.au/published/workflow?id=9e0bae0505ee7c07, https://usegalaxy.org.au/published/workflow?id=9e8d9b7055b7c863, https://usegalaxy.org.au/published/workflow?id=9ec9fe2715d7cfdc, https://usegalaxy.org.au/published/workflow?id=a02c102e179596cb, https://usegalaxy.org.au/published/workflow?id=a345bf84eb9bf6b2, https://usegalaxy.org.au/published/workflow?id=a484cdc9ff9d7e9e, https://usegalaxy.org.au/published/workflow?id=a525966251895f8a, https://usegalaxy.org.au/published/workflow?id=a5ccbb9aced9ac53, https://usegalaxy.org.au/published/workflow?id=a69db7f914c06407, https://usegalaxy.org.au/published/workflow?id=a7346c25dfdf2e3e, https://usegalaxy.org.au/published/workflow?id=a763e53653b10ce0, https://usegalaxy.org.au/published/workflow?id=a7726d18d2a5fcf2, https://usegalaxy.org.au/published/workflow?id=a7a30d970584f9ac, https://usegalaxy.org.au/published/workflow?id=a7d1cf2bb636e695, https://usegalaxy.org.au/published/workflow?id=a820d57cc8300253, https://usegalaxy.org.au/published/workflow?id=a8ec08a1408cd6e7, https://usegalaxy.org.au/published/workflow?id=a9fa6101f1fb13e2, https://usegalaxy.org.au/published/workflow?id=aa94c77b720a2810, https://usegalaxy.org.au/published/workflow?id=aaaf580eb00459b6, https://usegalaxy.org.au/published/workflow?id=aae07f802002fdd0, https://usegalaxy.org.au/published/workflow?id=ad10cd704268dcf0, https://usegalaxy.org.au/published/workflow?id=ad13b4a61e8987b1, https://usegalaxy.org.au/published/workflow?id=ae5dca8108d1a8a6, https://usegalaxy.org.au/published/workflow?id=ae6892f1df022097, https://usegalaxy.org.au/published/workflow?id=ae773dfaa074cb72, https://usegalaxy.org.au/published/workflow?id=aed93da6fd6e4e13, https://usegalaxy.org.au/published/workflow?id=af3f40cba4d7afa6, https://usegalaxy.org.au/published/workflow?id=af7b95f46485308c, https://usegalaxy.org.au/published/workflow?id=af990cca72794142, https://usegalaxy.org.au/published/workflow?id=afab89bc468da8b3, https://usegalaxy.org.au/published/workflow?id=afd9a8bbc07f9c3f, https://usegalaxy.org.au/published/workflow?id=b191da6e594747fb, https://usegalaxy.org.au/published/workflow?id=b19fa49405117317, https://usegalaxy.org.au/published/workflow?id=b1f45d2f8d8343e9, https://usegalaxy.org.au/published/workflow?id=b20f28151e46d3e0, https://usegalaxy.org.au/published/workflow?id=b5131e3a0ef2676e, https://usegalaxy.org.au/published/workflow?id=b57bf134896c30ea, https://usegalaxy.org.au/published/workflow?id=b6db525b1b73394b, https://usegalaxy.org.au/published/workflow?id=b7226d37b25df833, https://usegalaxy.org.au/published/workflow?id=b8968fb0bf796e53, https://usegalaxy.org.au/published/workflow?id=b8c661ff6b79383b, https://usegalaxy.org.au/published/workflow?id=b9faf6a3f48fc11b, https://usegalaxy.org.au/published/workflow?id=bb0feff5d764df60, https://usegalaxy.org.au/published/workflow?id=bd251174d94cde9d, https://usegalaxy.org.au/published/workflow?id=bda98c5a0ea70dd0, https://usegalaxy.org.au/published/workflow?id=be96bb47b6d6435a, https://usegalaxy.org.au/published/workflow?id=c1a0a3888d9ee3ac, https://usegalaxy.org.au/published/workflow?id=c296b3848708f6f1, https://usegalaxy.org.au/published/workflow?id=c2dee690320f9845, https://usegalaxy.org.au/published/workflow?id=c2e5a43356042b30, https://usegalaxy.org.au/published/workflow?id=c3b3717502ff6f76, https://usegalaxy.org.au/published/workflow?id=c3bb21f20a3ce689, https://usegalaxy.org.au/published/workflow?id=c3e198bc2146b4ad, https://usegalaxy.org.au/published/workflow?id=c43744d7dd9c0c0b, https://usegalaxy.org.au/published/workflow?id=c65a5af6c0383f8c, https://usegalaxy.org.au/published/workflow?id=c6a71573a299eb3a, https://usegalaxy.org.au/published/workflow?id=c7e1f666a3366823, https://usegalaxy.org.au/published/workflow?id=c7f2d6490128dc3c, https://usegalaxy.org.au/published/workflow?id=c87c1f8c4eff415a, https://usegalaxy.org.au/published/workflow?id=c9d04bf6207c0b32, https://usegalaxy.org.au/published/workflow?id=cb960edf95d2dc35, https://usegalaxy.org.au/published/workflow?id=ccc6602623583bcc, https://usegalaxy.org.au/published/workflow?id=ce7106593536f900, https://usegalaxy.org.au/published/workflow?id=ce7f6a1c824920e2, https://usegalaxy.org.au/published/workflow?id=cef68481b305a516, https://usegalaxy.org.au/published/workflow?id=d126940d33e63036, https://usegalaxy.org.au/published/workflow?id=d298edda13b382e7, https://usegalaxy.org.au/published/workflow?id=d42d7b1aa47e870f, https://usegalaxy.org.au/published/workflow?id=d5ecfdbc04b24227, https://usegalaxy.org.au/published/workflow?id=d78879cfaa5948b3, https://usegalaxy.org.au/published/workflow?id=d7a65f577fe5f16b, https://usegalaxy.org.au/published/workflow?id=d7f3fc810261e419, https://usegalaxy.org.au/published/workflow?id=d8ee4c7ef846e30f, https://usegalaxy.org.au/published/workflow?id=d9c181fd954b629a, https://usegalaxy.org.au/published/workflow?id=db0bc7357c181222, https://usegalaxy.org.au/published/workflow?id=db3370a412c4de78, https://usegalaxy.org.au/published/workflow?id=dccc8fb49d61ac33, https://usegalaxy.org.au/published/workflow?id=dce29c35dfac48b1, https://usegalaxy.org.au/published/workflow?id=dd6f465ab1eb10b0, https://usegalaxy.org.au/published/workflow?id=de1c04a2caf70647, https://usegalaxy.org.au/published/workflow?id=df4e186069552649, https://usegalaxy.org.au/published/workflow?id=dfc5d454becbc956, https://usegalaxy.org.au/published/workflow?id=e031af90daa2cadf, https://usegalaxy.org.au/published/workflow?id=e08b04291839e79a, https://usegalaxy.org.au/published/workflow?id=e08ef6b519f5fba0, https://usegalaxy.org.au/published/workflow?id=e0b89b4e50d1a99b, https://usegalaxy.org.au/published/workflow?id=e1fd89e7248b6661, https://usegalaxy.org.au/published/workflow?id=e28ca83f14f4ede5, https://usegalaxy.org.au/published/workflow?id=e3cda847fe727b61, https://usegalaxy.org.au/published/workflow?id=e52a3479c8216291, https://usegalaxy.org.au/published/workflow?id=e57c6d05b560e206, https://usegalaxy.org.au/published/workflow?id=e5f1ce106ea7af12, https://usegalaxy.org.au/published/workflow?id=e62f743342851151, https://usegalaxy.org.au/published/workflow?id=e826584d78b69467, https://usegalaxy.org.au/published/workflow?id=e92b4d8dfa2f1cf7, https://usegalaxy.org.au/published/workflow?id=ea687c5613fcce27, https://usegalaxy.org.au/published/workflow?id=eaa841e6b7384633, https://usegalaxy.org.au/published/workflow?id=eb3a4535204383fa, https://usegalaxy.org.au/published/workflow?id=ec79c2fab8eac15d, https://usegalaxy.org.au/published/workflow?id=ed2f72e3537e8a4a, https://usegalaxy.org.au/published/workflow?id=efd39a4be0a9a4e6, https://usegalaxy.org.au/published/workflow?id=f0ced9a5d34b2aff, https://usegalaxy.org.au/published/workflow?id=f268c181a47d0cab, https://usegalaxy.org.au/published/workflow?id=f5c0b0bc5e023e42, https://usegalaxy.org.au/published/workflow?id=f7a866159f1eaecf, https://usegalaxy.org.au/published/workflow?id=f944f2d3fbd1f062, https://usegalaxy.org.au/published/workflow?id=fa37223950a91f34, https://usegalaxy.org.au/published/workflow?id=fa4fa7353dffef9a, https://usegalaxy.org.au/published/workflow?id=fa87a22c8dcd8d78, https://usegalaxy.org.au/published/workflow?id=fb644a9649f36f40, https://usegalaxy.org.au/published/workflow?id=fc64a77321752a3c, https://usegalaxy.org.au/published/workflow?id=fc6f8afdee980f0e, https://usegalaxy.org.au/published/workflow?id=fd1b3d7f2d34ff09, https://usegalaxy.org.au/published/workflow?id=fd477917a3d27f61, https://usegalaxy.org.au/published/workflow?id=fd8fb237f8dc24c3, https://usegalaxy.org.au/published/workflow?id=fe1de81a97578368, https://usegalaxy.org.au/published/workflow?id=fe1e7fa0bc437ed8, https://usegalaxy.org.au/published/workflow?id=fe893d72526eba71, https://usegalaxy.org.au/published/workflow?id=ff42213dda7d3e59, https://usegalaxy.org.au/published/workflow?id=fff2e49bc90e38ca, https://usegalaxy.org/published/workflow?id=00bc4e99626da1ba, https://usegalaxy.org/published/workflow?id=028f1beead539975, https://usegalaxy.org/published/workflow?id=02e29499b946f3bb, https://usegalaxy.org/published/workflow?id=044c5ce43f2e1829, https://usegalaxy.org/published/workflow?id=052ba366e32132ec, https://usegalaxy.org/published/workflow?id=05e86b4df0d14800, https://usegalaxy.org/published/workflow?id=063173d36c254105, https://usegalaxy.org/published/workflow?id=06b489dc859f6e39, https://usegalaxy.org/published/workflow?id=06f3ca70b542556e, https://usegalaxy.org/published/workflow?id=07dca0dc191fb1de, https://usegalaxy.org/published/workflow?id=07e4414905458c86, https://usegalaxy.org/published/workflow?id=07e873a6d75dfe9f, https://usegalaxy.org/published/workflow?id=08da44d18689dffc, https://usegalaxy.org/published/workflow?id=08edef22fa132799, https://usegalaxy.org/published/workflow?id=091f788edad78956, https://usegalaxy.org/published/workflow?id=098fd236c5e2895d, https://usegalaxy.org/published/workflow?id=0993daadf2db5f51, https://usegalaxy.org/published/workflow?id=0d47009eb2d7bdc1, https://usegalaxy.org/published/workflow?id=0ebb1f5b6c9c8ab9, https://usegalaxy.org/published/workflow?id=1010a9b57c77239c, https://usegalaxy.org/published/workflow?id=10c5f3c5cadd7417, https://usegalaxy.org/published/workflow?id=1234a128adc25b03, https://usegalaxy.org/published/workflow?id=1245e9435cbc3985, https://usegalaxy.org/published/workflow?id=1250ec3ad6199cda, https://usegalaxy.org/published/workflow?id=12a52ae52e0f1053, https://usegalaxy.org/published/workflow?id=137d0f05a71d7acb, https://usegalaxy.org/published/workflow?id=13a1fa4f6daf65cc, https://usegalaxy.org/published/workflow?id=148c071f5c5a4586, https://usegalaxy.org/published/workflow?id=150e288c2c2fe9e9, https://usegalaxy.org/published/workflow?id=156e245e3d9f22ce, https://usegalaxy.org/published/workflow?id=1598b3fbd1dd2439, https://usegalaxy.org/published/workflow?id=15ea806eaf077617, https://usegalaxy.org/published/workflow?id=162fda90525220de, https://usegalaxy.org/published/workflow?id=172ee5e5bef33d2e, https://usegalaxy.org/published/workflow?id=17f3d7e52777d2e8, https://usegalaxy.org/published/workflow?id=1829c91a30138721, https://usegalaxy.org/published/workflow?id=19c1ba21e82d8615, https://usegalaxy.org/published/workflow?id=1a23d8c795846251, https://usegalaxy.org/published/workflow?id=1cad9a0b375ed3d1, https://usegalaxy.org/published/workflow?id=1cbd946c4cc0e7b9, https://usegalaxy.org/published/workflow?id=1d24cc949c52e475, https://usegalaxy.org/published/workflow?id=1d56a0975fca0e99, https://usegalaxy.org/published/workflow?id=1d69ab87c94d68fd, https://usegalaxy.org/published/workflow?id=1d8227ad3f3651a0, https://usegalaxy.org/published/workflow?id=1dd63a7b4c6f4c83, https://usegalaxy.org/published/workflow?id=1e0a55635f5b04cf, https://usegalaxy.org/published/workflow?id=1e1d7fac7e241768, https://usegalaxy.org/published/workflow?id=1eec2586d1ff908b, https://usegalaxy.org/published/workflow?id=1f00e44f590b8369, https://usegalaxy.org/published/workflow?id=1f28e673ea725541, https://usegalaxy.org/published/workflow?id=1ffc71df8eeed57e, https://usegalaxy.org/published/workflow?id=2078abc4c99c3344, https://usegalaxy.org/published/workflow?id=20da2dcce0ae2ca2, https://usegalaxy.org/published/workflow?id=25a9f0693ff6d327, https://usegalaxy.org/published/workflow?id=26019c2b056d3a1c, https://usegalaxy.org/published/workflow?id=27ee9da9121d0522, https://usegalaxy.org/published/workflow?id=283bc048cc7a80f5, https://usegalaxy.org/published/workflow?id=284150bb1f296378, https://usegalaxy.org/published/workflow?id=291829fed6fa7c44, https://usegalaxy.org/published/workflow?id=2957924403998f3a, https://usegalaxy.org/published/workflow?id=2aa84a6106bfc969, https://usegalaxy.org/published/workflow?id=2b41752b8518a87b, https://usegalaxy.org/published/workflow?id=2b5346b94358df88, https://usegalaxy.org/published/workflow?id=2c5c627c08563398, https://usegalaxy.org/published/workflow?id=2cc5472ad7c039d1, https://usegalaxy.org/published/workflow?id=2d6d74294e911fa3, https://usegalaxy.org/published/workflow?id=2d994cc02ec05ede, https://usegalaxy.org/published/workflow?id=2e029537d2f4ab50, https://usegalaxy.org/published/workflow?id=2e4918ad409f7f4e, https://usegalaxy.org/published/workflow?id=2f4d5f833aa1c981, https://usegalaxy.org/published/workflow?id=2f774500d5bec9a6, https://usegalaxy.org/published/workflow?id=305fb9f84b053713, https://usegalaxy.org/published/workflow?id=30ba39be60ec29be, https://usegalaxy.org/published/workflow?id=313d50268e1e22e4, https://usegalaxy.org/published/workflow?id=31a6145b0832fa48, https://usegalaxy.org/published/workflow?id=326ced2230a49c26, https://usegalaxy.org/published/workflow?id=32c769241ffce3d1, https://usegalaxy.org/published/workflow?id=336795e890851c21, https://usegalaxy.org/published/workflow?id=33e827d75a0074df, https://usegalaxy.org/published/workflow?id=35c022174ce79c1e, https://usegalaxy.org/published/workflow?id=35c2626849348f2c, https://usegalaxy.org/published/workflow?id=3673a1ced343d0d0, https://usegalaxy.org/published/workflow?id=3823773ef4ecaaf4, https://usegalaxy.org/published/workflow?id=38f9fc867c4d1f40, https://usegalaxy.org/published/workflow?id=3aa1b79aa6cca06b, https://usegalaxy.org/published/workflow?id=3ab66315f50501f0, https://usegalaxy.org/published/workflow?id=3b2bee20f0cfd957, https://usegalaxy.org/published/workflow?id=3b5437544ca3b5dc, https://usegalaxy.org/published/workflow?id=3ba73f679558017c, https://usegalaxy.org/published/workflow?id=3c076bc8551a23d2, https://usegalaxy.org/published/workflow?id=3c0af11dbeeee506, https://usegalaxy.org/published/workflow?id=3d0ba2f11f370e05, https://usegalaxy.org/published/workflow?id=3db4eb1ecb588d44, https://usegalaxy.org/published/workflow?id=3e57e668cf698d19, https://usegalaxy.org/published/workflow?id=3e6c9ab5ef3250ae, https://usegalaxy.org/published/workflow?id=3f7550d5d2bf0b96, https://usegalaxy.org/published/workflow?id=3f8e0395007c8e9e, https://usegalaxy.org/published/workflow?id=3fc5bdbd6355477c, https://usegalaxy.org/published/workflow?id=408c799822cbf6ea, https://usegalaxy.org/published/workflow?id=4238587f2feb1627, https://usegalaxy.org/published/workflow?id=424acfc2ded46f9f, https://usegalaxy.org/published/workflow?id=435f0340bf00c940, https://usegalaxy.org/published/workflow?id=4403ca7fe447a5d7, https://usegalaxy.org/published/workflow?id=442949862ba62462, https://usegalaxy.org/published/workflow?id=444c856f03b6d2a7, https://usegalaxy.org/published/workflow?id=44781c8b0bd667ed, https://usegalaxy.org/published/workflow?id=448a0797f9150198, https://usegalaxy.org/published/workflow?id=45cb3e98b1319c7c, https://usegalaxy.org/published/workflow?id=4612ec88a57ab9e7, https://usegalaxy.org/published/workflow?id=467d3e260c21de98, https://usegalaxy.org/published/workflow?id=46e718fb8eeb4d54, https://usegalaxy.org/published/workflow?id=472f73c3e81da1d9, https://usegalaxy.org/published/workflow?id=4790266996f12979, https://usegalaxy.org/published/workflow?id=47a2458ee9d57b24, https://usegalaxy.org/published/workflow?id=484f8fcf21d67376, https://usegalaxy.org/published/workflow?id=498a8c6329ca5728, https://usegalaxy.org/published/workflow?id=49c61c378cacfe02, https://usegalaxy.org/published/workflow?id=4a01263e3d4b4edb, https://usegalaxy.org/published/workflow?id=4a0ad87aa5aa558a, https://usegalaxy.org/published/workflow?id=4b7daa3c73c8ef30, https://usegalaxy.org/published/workflow?id=4bf73376661baafd, https://usegalaxy.org/published/workflow?id=4ca8b023f9a33440, https://usegalaxy.org/published/workflow?id=4cc70b699ea8bc63, https://usegalaxy.org/published/workflow?id=4d5cc5010537f172, https://usegalaxy.org/published/workflow?id=506acac8c6a58d5e, https://usegalaxy.org/published/workflow?id=5199f404ad7ecef8, https://usegalaxy.org/published/workflow?id=51d80a44d7d6fff7, https://usegalaxy.org/published/workflow?id=52592de9687fc6de, https://usegalaxy.org/published/workflow?id=527adadecbab2eb6, https://usegalaxy.org/published/workflow?id=52b7524a3941885c, https://usegalaxy.org/published/workflow?id=52bde903adbbaf60, https://usegalaxy.org/published/workflow?id=52e2b69787ca7a17, https://usegalaxy.org/published/workflow?id=533b1d5d08f9943e, https://usegalaxy.org/published/workflow?id=53fdff74729a6780, https://usegalaxy.org/published/workflow?id=553125cd799e5982, https://usegalaxy.org/published/workflow?id=5532e6f775adb0d7, https://usegalaxy.org/published/workflow?id=570172fbcf865e1d, https://usegalaxy.org/published/workflow?id=578e634b63923e9b, https://usegalaxy.org/published/workflow?id=57e6345fda3e2682, https://usegalaxy.org/published/workflow?id=582f3de3971fc38c, https://usegalaxy.org/published/workflow?id=58a7a02dade085cf, https://usegalaxy.org/published/workflow?id=58eeb06f911e8410, https://usegalaxy.org/published/workflow?id=59007e2bc9c807ee, https://usegalaxy.org/published/workflow?id=5944c5effd56b0d6, https://usegalaxy.org/published/workflow?id=597c4d54f8a8ebe7, https://usegalaxy.org/published/workflow?id=5a16ff38eb80da82, https://usegalaxy.org/published/workflow?id=5a55d9bd9701b1ce, https://usegalaxy.org/published/workflow?id=5b2bb861501fa248, https://usegalaxy.org/published/workflow?id=5b8497a32bf01c4f, https://usegalaxy.org/published/workflow?id=5cb74afcca56a453, https://usegalaxy.org/published/workflow?id=5ccc7d708e13de60, https://usegalaxy.org/published/workflow?id=5d1208bd49f97aeb, https://usegalaxy.org/published/workflow?id=5dd0a672cceb15ec, https://usegalaxy.org/published/workflow?id=5df593510c7d944d, https://usegalaxy.org/published/workflow?id=5dfec2a3ae5fa765, https://usegalaxy.org/published/workflow?id=5e18bbbe9ff2d9f5, https://usegalaxy.org/published/workflow?id=607d1683d6e26cef, https://usegalaxy.org/published/workflow?id=60b9910b7bb69f47, https://usegalaxy.org/published/workflow?id=6229546fb8b0eba9, https://usegalaxy.org/published/workflow?id=62316da77b1bfa5a, https://usegalaxy.org/published/workflow?id=62436fe121e035e7, https://usegalaxy.org/published/workflow?id=62c842e67fca5ab7, https://usegalaxy.org/published/workflow?id=62ecc8078a396e9e, https://usegalaxy.org/published/workflow?id=63b3b266bde77563, https://usegalaxy.org/published/workflow?id=63f61ad7cae40646, https://usegalaxy.org/published/workflow?id=642e71c64a975461, https://usegalaxy.org/published/workflow?id=64a329cf75834d6b, https://usegalaxy.org/published/workflow?id=64af2cd469fee16a, https://usegalaxy.org/published/workflow?id=653d553ac2fa51c8, https://usegalaxy.org/published/workflow?id=66e6b41f6c6d52b7, https://usegalaxy.org/published/workflow?id=672125473ec700bc, https://usegalaxy.org/published/workflow?id=67252039bce060b2, https://usegalaxy.org/published/workflow?id=676b88d0aac13fa5, https://usegalaxy.org/published/workflow?id=676fa9af68f6a763, https://usegalaxy.org/published/workflow?id=67992bf894a82587, https://usegalaxy.org/published/workflow?id=683e725f672b7d9b, https://usegalaxy.org/published/workflow?id=68495cc8ec3d46df, https://usegalaxy.org/published/workflow?id=69edab5d6ad91556, https://usegalaxy.org/published/workflow?id=6a174e7498434bd5, https://usegalaxy.org/published/workflow?id=6b511b47176893e3, https://usegalaxy.org/published/workflow?id=6c68072bf72e6c4a, https://usegalaxy.org/published/workflow?id=6c95a357a3c1f399, https://usegalaxy.org/published/workflow?id=6d68f56abf9c624d, https://usegalaxy.org/published/workflow?id=6db96a9c7e4b8d14, https://usegalaxy.org/published/workflow?id=6dfe0c6e87c6cb7b, https://usegalaxy.org/published/workflow?id=6e47a35cc5f2bdce, https://usegalaxy.org/published/workflow?id=6ea1bd64873b6c18, https://usegalaxy.org/published/workflow?id=6ec8b101d24a0f4f, https://usegalaxy.org/published/workflow?id=6ecfa8ce19ed86da, https://usegalaxy.org/published/workflow?id=6fa88ca75caba8e4, https://usegalaxy.org/published/workflow?id=7022c4d9af725e5c, https://usegalaxy.org/published/workflow?id=7072731b51e16923, https://usegalaxy.org/published/workflow?id=70f9c7cc75a1d3c0, https://usegalaxy.org/published/workflow?id=716dc497d7d2e1c3, https://usegalaxy.org/published/workflow?id=72d134a60c694b59, https://usegalaxy.org/published/workflow?id=74c98eafd363cc2b, https://usegalaxy.org/published/workflow?id=75324d68ed124ff5, https://usegalaxy.org/published/workflow?id=75e23bc4e1ac2f5b, https://usegalaxy.org/published/workflow?id=75fffbce810acaf3, https://usegalaxy.org/published/workflow?id=761675cb2c9ad2ef, https://usegalaxy.org/published/workflow?id=76b8ed1c68678b62, https://usegalaxy.org/published/workflow?id=76d5143860fac947, https://usegalaxy.org/published/workflow?id=76fb573f1372ae86, https://usegalaxy.org/published/workflow?id=7757cd56fa4d060c, https://usegalaxy.org/published/workflow?id=778a8867b0a6ea4f, https://usegalaxy.org/published/workflow?id=77d1505c0dd65e26, https://usegalaxy.org/published/workflow?id=7832cfdeabe47696, https://usegalaxy.org/published/workflow?id=789647a199e563fd, https://usegalaxy.org/published/workflow?id=7a33bc2d0970b502, https://usegalaxy.org/published/workflow?id=7b9e17cb1d72d7e4, https://usegalaxy.org/published/workflow?id=7b9f36ae2174164a, https://usegalaxy.org/published/workflow?id=7c4a2339b17a8930, https://usegalaxy.org/published/workflow?id=7cee34a6c06c21e4, https://usegalaxy.org/published/workflow?id=7d11026214dd8c93, https://usegalaxy.org/published/workflow?id=7dba74c915f63dc1, https://usegalaxy.org/published/workflow?id=7df3246858cd83d1, https://usegalaxy.org/published/workflow?id=7f6900852a81d121, https://usegalaxy.org/published/workflow?id=7fb4dae5daab0522, https://usegalaxy.org/published/workflow?id=803aec95399f7073, https://usegalaxy.org/published/workflow?id=8095deba606ebd5e, https://usegalaxy.org/published/workflow?id=81f1f0fd6d9235fd, https://usegalaxy.org/published/workflow?id=824730064b5014e5, https://usegalaxy.org/published/workflow?id=84161f4f8bf77474, https://usegalaxy.org/published/workflow?id=842c09b49c456429, https://usegalaxy.org/published/workflow?id=85011076892808e0, https://usegalaxy.org/published/workflow?id=85bc37a59a6ea443, https://usegalaxy.org/published/workflow?id=864da6858d34df08, https://usegalaxy.org/published/workflow?id=868acaf49d0743ec, https://usegalaxy.org/published/workflow?id=8756aabc654b1050, https://usegalaxy.org/published/workflow?id=875ff758068e1c2f, https://usegalaxy.org/published/workflow?id=87756e064ab1e28a, https://usegalaxy.org/published/workflow?id=87c1bdc00b975377, https://usegalaxy.org/published/workflow?id=87d7a69106ca663d, https://usegalaxy.org/published/workflow?id=87eb1a8ee7a2c984, https://usegalaxy.org/published/workflow?id=87fab09e4c592435, https://usegalaxy.org/published/workflow?id=885f0e66fa6b8790, https://usegalaxy.org/published/workflow?id=88e10493f7e6b9d5, https://usegalaxy.org/published/workflow?id=8990ae43ae855bc7, https://usegalaxy.org/published/workflow?id=899f4f371b4c958c, https://usegalaxy.org/published/workflow?id=89f48993bb8f94bd, https://usegalaxy.org/published/workflow?id=89f52c97b3aee938, https://usegalaxy.org/published/workflow?id=8a215695e7b57cf5, https://usegalaxy.org/published/workflow?id=8bf608c6fd0dabd0, https://usegalaxy.org/published/workflow?id=8cbb9936e88d4f49, https://usegalaxy.org/published/workflow?id=8cfbaa14cff5a633, https://usegalaxy.org/published/workflow?id=8d605fbd33003d26, https://usegalaxy.org/published/workflow?id=8e2b2a40831dfb08, https://usegalaxy.org/published/workflow?id=8e509c87d143022c, https://usegalaxy.org/published/workflow?id=8e918c12d2f0d1fc, https://usegalaxy.org/published/workflow?id=8ebcae16a7822d93, https://usegalaxy.org/published/workflow?id=8eeee8ae24fb37b2, https://usegalaxy.org/published/workflow?id=8f392d06d32bdf14, https://usegalaxy.org/published/workflow?id=8f7c9461910b08f1, https://usegalaxy.org/published/workflow?id=907aaf55487ddf43, https://usegalaxy.org/published/workflow?id=90ed607bedd65524, https://usegalaxy.org/published/workflow?id=90f5b512997c70f8, https://usegalaxy.org/published/workflow?id=9124c2ec80b35680, https://usegalaxy.org/published/workflow?id=914ec4d8fc0022e4, https://usegalaxy.org/published/workflow?id=917e5a21d51a61ce, https://usegalaxy.org/published/workflow?id=91806144e90637e8, https://usegalaxy.org/published/workflow?id=9255c0c9017b139f, https://usegalaxy.org/published/workflow?id=92aab10842ff610d, https://usegalaxy.org/published/workflow?id=932b3bc10186f73e, https://usegalaxy.org/published/workflow?id=936ed7447c8e0a2d, https://usegalaxy.org/published/workflow?id=9373050c3ce80d56, https://usegalaxy.org/published/workflow?id=93c1fef4472aa7a8, https://usegalaxy.org/published/workflow?id=93cb976bd4ae546e, https://usegalaxy.org/published/workflow?id=93e69c7e7a857ca3, https://usegalaxy.org/published/workflow?id=94ab68d27de2e60f, https://usegalaxy.org/published/workflow?id=961dac8100c6c70c, https://usegalaxy.org/published/workflow?id=96ae2544e22d34c5, https://usegalaxy.org/published/workflow?id=96cdba1fe599ada8, https://usegalaxy.org/published/workflow?id=9763787d7b102f39, https://usegalaxy.org/published/workflow?id=97c20a819394e381, https://usegalaxy.org/published/workflow?id=98a4b90f7fc2bfce, https://usegalaxy.org/published/workflow?id=9938a81fb6911412, https://usegalaxy.org/published/workflow?id=995cc428eace3bc4, https://usegalaxy.org/published/workflow?id=9990b6e2facf6dca, https://usegalaxy.org/published/workflow?id=99ee567c092fb34c, https://usegalaxy.org/published/workflow?id=9b5ae15bb9aaba48, https://usegalaxy.org/published/workflow?id=9c688a2fc8d1bf4e, https://usegalaxy.org/published/workflow?id=9da9ddae8d5dc92f, https://usegalaxy.org/published/workflow?id=9db607d8d1cdcc25, https://usegalaxy.org/published/workflow?id=9ddfdc9d6cacea88, https://usegalaxy.org/published/workflow?id=9ded318097694dbc, https://usegalaxy.org/published/workflow?id=9e74e3455e47c813, https://usegalaxy.org/published/workflow?id=9e9a027e0c294a79, https://usegalaxy.org/published/workflow?id=9eb29b71d7effefc, https://usegalaxy.org/published/workflow?id=9edc78f665579559, https://usegalaxy.org/published/workflow?id=9f1e8e82fb3e2033, https://usegalaxy.org/published/workflow?id=9f55643d3fd6f28c, https://usegalaxy.org/published/workflow?id=a164d857439a126f, https://usegalaxy.org/published/workflow?id=a17af8b7b2b39c1a, https://usegalaxy.org/published/workflow?id=a2046dd883227b49, https://usegalaxy.org/published/workflow?id=a4a56efb37d845bf, https://usegalaxy.org/published/workflow?id=a57a4249a8a21a6d, https://usegalaxy.org/published/workflow?id=a63d3ee4a2a4a20b, https://usegalaxy.org/published/workflow?id=a719660eb1877f54, https://usegalaxy.org/published/workflow?id=a89008fccc1f1d17, https://usegalaxy.org/published/workflow?id=a89d1b74566d516d, https://usegalaxy.org/published/workflow?id=a8a38b594183db2b, https://usegalaxy.org/published/workflow?id=a8e0f77cd6ef49f9, https://usegalaxy.org/published/workflow?id=a8e5adb65d064641, https://usegalaxy.org/published/workflow?id=a8f633b79b20c760, https://usegalaxy.org/published/workflow?id=a986e4991fc61c92, https://usegalaxy.org/published/workflow?id=a99a938953be5fd6, https://usegalaxy.org/published/workflow?id=ab0be4897d5355ac, https://usegalaxy.org/published/workflow?id=ac87b6133de6a03f, https://usegalaxy.org/published/workflow?id=ac8b74b911778fbc, https://usegalaxy.org/published/workflow?id=acada5044d64a30a, https://usegalaxy.org/published/workflow?id=acae55e8f37fea59, https://usegalaxy.org/published/workflow?id=ace738014a7278b0, https://usegalaxy.org/published/workflow?id=adb86df9061dca2b, https://usegalaxy.org/published/workflow?id=adbed5c956a71543, https://usegalaxy.org/published/workflow?id=ae1de4267e4da154, https://usegalaxy.org/published/workflow?id=ae523fe6c9183e58, https://usegalaxy.org/published/workflow?id=ae5327f6f30b6138, https://usegalaxy.org/published/workflow?id=af5d714396856e9a, https://usegalaxy.org/published/workflow?id=af71519c81429287, https://usegalaxy.org/published/workflow?id=afd7633a34d68704, https://usegalaxy.org/published/workflow?id=b0b6dfa166b0d873, https://usegalaxy.org/published/workflow?id=b11c6268704dcc1b, https://usegalaxy.org/published/workflow?id=b19a9163c26f1270, https://usegalaxy.org/published/workflow?id=b1dc9cf054a63be5, https://usegalaxy.org/published/workflow?id=b232caf4a1e1b7ee, https://usegalaxy.org/published/workflow?id=b3c8d64eca750ecf, https://usegalaxy.org/published/workflow?id=b5644a5dcf237ef9, https://usegalaxy.org/published/workflow?id=b5e8f4a0ff226921, https://usegalaxy.org/published/workflow?id=b63d1e1024cc3295, https://usegalaxy.org/published/workflow?id=b661fb850c4b4b01, https://usegalaxy.org/published/workflow?id=b6d535f210efa18e, https://usegalaxy.org/published/workflow?id=b78a75ae89ae4c79, https://usegalaxy.org/published/workflow?id=b86d24a103181742, https://usegalaxy.org/published/workflow?id=b882703ca6fea3a8, https://usegalaxy.org/published/workflow?id=b893662378ed3da0, https://usegalaxy.org/published/workflow?id=ba5889301928907f, https://usegalaxy.org/published/workflow?id=bae2d6b436cd80ef, https://usegalaxy.org/published/workflow?id=bb1cb6abfe84a646, https://usegalaxy.org/published/workflow?id=bb25ac3ba4863424, https://usegalaxy.org/published/workflow?id=bbc894a79eb0e769, https://usegalaxy.org/published/workflow?id=bcb798fb718968ce, https://usegalaxy.org/published/workflow?id=bceb6ae418bef9b8, https://usegalaxy.org/published/workflow?id=bd2ce9a97c1598df, https://usegalaxy.org/published/workflow?id=bd3bed78eb99e294, https://usegalaxy.org/published/workflow?id=bd6bfd338ffb9637, https://usegalaxy.org/published/workflow?id=bdfb27e4ef7b6138, https://usegalaxy.org/published/workflow?id=bf3f71a748d88490, https://usegalaxy.org/published/workflow?id=bfd2e7e63d3e6856, https://usegalaxy.org/published/workflow?id=bfef75672b9c3ea4, https://usegalaxy.org/published/workflow?id=c0746df296544a3a, https://usegalaxy.org/published/workflow?id=c0a108e53ab05b0d, https://usegalaxy.org/published/workflow?id=c0a9a2d53b8660de, https://usegalaxy.org/published/workflow?id=c1339c998a55431c, https://usegalaxy.org/published/workflow?id=c156e67ad1ae4a97, https://usegalaxy.org/published/workflow?id=c1ec12905944ca7b, https://usegalaxy.org/published/workflow?id=c20a68563e80d401, https://usegalaxy.org/published/workflow?id=c2898d28a3b380d6, https://usegalaxy.org/published/workflow?id=c30bfd545ef3a228, https://usegalaxy.org/published/workflow?id=c4ac09605633655c, https://usegalaxy.org/published/workflow?id=c57f70d686b6062e, https://usegalaxy.org/published/workflow?id=c69f8c26ec5fb8a6, https://usegalaxy.org/published/workflow?id=c707871e1dc9e21d, https://usegalaxy.org/published/workflow?id=c712b85a55da2a2d, https://usegalaxy.org/published/workflow?id=c74e3a24751d9924, https://usegalaxy.org/published/workflow?id=c81833a367445f98, https://usegalaxy.org/published/workflow?id=c96b12be162f744b, https://usegalaxy.org/published/workflow?id=c983ea8a32662cd6, https://usegalaxy.org/published/workflow?id=ca36eebd59575df3, https://usegalaxy.org/published/workflow?id=cb011900fc0ee12a, https://usegalaxy.org/published/workflow?id=cbbd7e95041d9429, https://usegalaxy.org/published/workflow?id=cd4d88429f096fbc, https://usegalaxy.org/published/workflow?id=cd5f803c80875b9c, https://usegalaxy.org/published/workflow?id=cd824d8631d6a55e, https://usegalaxy.org/published/workflow?id=ce04658f05029be2, https://usegalaxy.org/published/workflow?id=ce815d36cbb57304, https://usegalaxy.org/published/workflow?id=cf2dbc29ca20ebee, https://usegalaxy.org/published/workflow?id=cf3320614654adfe, https://usegalaxy.org/published/workflow?id=d00ac64dcd07f91a, https://usegalaxy.org/published/workflow?id=d08135fa13d1745e, https://usegalaxy.org/published/workflow?id=d2759906e4124964, https://usegalaxy.org/published/workflow?id=d2f7fd589a91124a, https://usegalaxy.org/published/workflow?id=d36765ac28c11e1a, https://usegalaxy.org/published/workflow?id=d3f35215e6f45d79, https://usegalaxy.org/published/workflow?id=d4029ce1344e874a, https://usegalaxy.org/published/workflow?id=d768a4a06d3b1a90, https://usegalaxy.org/published/workflow?id=d82db9c29b998f22, https://usegalaxy.org/published/workflow?id=d903c7215a88c79f, https://usegalaxy.org/published/workflow?id=d9840d6ecf2b4d2f, https://usegalaxy.org/published/workflow?id=daaded719dcfd53b, https://usegalaxy.org/published/workflow?id=dae5d5e5df5c4dd7, https://usegalaxy.org/published/workflow?id=dbc650d226e89013, https://usegalaxy.org/published/workflow?id=dc006ebf798f1f2f, https://usegalaxy.org/published/workflow?id=dc7f05cb5a62da77, https://usegalaxy.org/published/workflow?id=dcf658333596d76a, https://usegalaxy.org/published/workflow?id=ddf222b797082436, https://usegalaxy.org/published/workflow?id=de208a900880dcdc, https://usegalaxy.org/published/workflow?id=de24e5678f541584, https://usegalaxy.org/published/workflow?id=de29cd965a2e93d3, https://usegalaxy.org/published/workflow?id=de9c2db125a5ea27, https://usegalaxy.org/published/workflow?id=df24d76ebf13e3c1, https://usegalaxy.org/published/workflow?id=df8b49863e83824d, https://usegalaxy.org/published/workflow?id=dff3cf6921b3d4d6, https://usegalaxy.org/published/workflow?id=e035be74e3c85bf5, https://usegalaxy.org/published/workflow?id=e0ec503626fa0d58, https://usegalaxy.org/published/workflow?id=e12296df7cf33c69, https://usegalaxy.org/published/workflow?id=e1aad86fdf9e6616, https://usegalaxy.org/published/workflow?id=e1dcdc125122f561, https://usegalaxy.org/published/workflow?id=e33f6a058c5f57e5, https://usegalaxy.org/published/workflow?id=e42a07b777c9f9dc, https://usegalaxy.org/published/workflow?id=e4dd39237df58884, https://usegalaxy.org/published/workflow?id=e58a87508a337b91, https://usegalaxy.org/published/workflow?id=e5f79f576548f7c5, https://usegalaxy.org/published/workflow?id=e65eb8a3bafde22b, https://usegalaxy.org/published/workflow?id=e7586d70715207b5, https://usegalaxy.org/published/workflow?id=e7fa48d7fd40908b, https://usegalaxy.org/published/workflow?id=e899acc664c8b808, https://usegalaxy.org/published/workflow?id=e9927c058ec0661f, https://usegalaxy.org/published/workflow?id=e9f4de639f4cfe02, https://usegalaxy.org/published/workflow?id=e9ffad95baa43478, https://usegalaxy.org/published/workflow?id=ea248022a4217e0e, https://usegalaxy.org/published/workflow?id=ea4c2defce2f5258, https://usegalaxy.org/published/workflow?id=ea5d2401f6711dff, https://usegalaxy.org/published/workflow?id=ea61cd21e3fbad2b, https://usegalaxy.org/published/workflow?id=eafe4aa62a600e1d, https://usegalaxy.org/published/workflow?id=ec2f77224324f22e, https://usegalaxy.org/published/workflow?id=ec66767cab0022ee, https://usegalaxy.org/published/workflow?id=edf9be785ab9eb65, https://usegalaxy.org/published/workflow?id=ee27581327d147da, https://usegalaxy.org/published/workflow?id=ee3b8d7ba29da11c, https://usegalaxy.org/published/workflow?id=efe7ca87323773c0, https://usegalaxy.org/published/workflow?id=f062eda543541ba8, https://usegalaxy.org/published/workflow?id=f09033420c6267ab, https://usegalaxy.org/published/workflow?id=f0ee8bcd7c8fadca, https://usegalaxy.org/published/workflow?id=f18946db4c93e88b, https://usegalaxy.org/published/workflow?id=f1a54b10d4afce8f, https://usegalaxy.org/published/workflow?id=f1c2d920bd41cebb, https://usegalaxy.org/published/workflow?id=f29d1bcfeaacba04, https://usegalaxy.org/published/workflow?id=f3b417763495221b, https://usegalaxy.org/published/workflow?id=f3c8093c5b8a7233, https://usegalaxy.org/published/workflow?id=f5805e66642fed04, https://usegalaxy.org/published/workflow?id=f5bd2da36ab0a870, https://usegalaxy.org/published/workflow?id=f60027545d40b82d, https://usegalaxy.org/published/workflow?id=f733ad0a6cc50cb9, https://usegalaxy.org/published/workflow?id=f765c8aa4bc3176c, https://usegalaxy.org/published/workflow?id=f7975ae799daf463, https://usegalaxy.org/published/workflow?id=f93b5864d6d2659c, https://usegalaxy.org/published/workflow?id=f9cd2289c007f103, https://usegalaxy.org/published/workflow?id=fa226c044fa026ef, https://usegalaxy.org/published/workflow?id=fa26fd51caea53c7, https://usegalaxy.org/published/workflow?id=fb588edfd343b24d, https://usegalaxy.org/published/workflow?id=fb789920b057fe8d, https://usegalaxy.org/published/workflow?id=fc130cb411aa7d06, https://usegalaxy.org/published/workflow?id=fc7b14aafa0b421e, https://usegalaxy.org/published/workflow?id=fcae9dfcdca70798, https://usegalaxy.org/published/workflow?id=fcbd4767298488b9, https://usegalaxy.org/published/workflow?id=fcbe12fc4eaae12e, https://usegalaxy.org/published/workflow?id=fce1837144d3a86e, https://usegalaxy.org/published/workflow?id=fd30cdfe7e38b508, https://usegalaxy.org/published/workflow?id=fd40e50c9f1545ce, https://usegalaxy.org/published/workflow?id=fd75347cfea06651, https://usegalaxy.org/published/workflow?id=fe262214fba1cb8e, https://usegalaxy.org/published/workflow?id=fe4c2791508c56ab, https://workflowhub.eu/workflows/1104?version=2, https://workflowhub.eu/workflows/1444?version=2, https://workflowhub.eu/workflows/1466?version=2, https://workflowhub.eu/workflows/1480?version=1, https://workflowhub.eu/workflows/1530?version=1, https://workflowhub.eu/workflows/1534?version=1, https://workflowhub.eu/workflows/1568?version=2, https://workflowhub.eu/workflows/1607?version=1, https://workflowhub.eu/workflows/1629?version=1, https://workflowhub.eu/workflows/1631?version=1, https://workflowhub.eu/workflows/1640?version=1, https://workflowhub.eu/workflows/1675?version=1, https://workflowhub.eu/workflows/1681?version=1, https://workflowhub.eu/workflows/1692?version=1, https://workflowhub.eu/workflows/1706?version=1, https://workflowhub.eu/workflows/1715?version=3, https://workflowhub.eu/workflows/2036?version=1, https://workflowhub.eu/workflows/2038?version=1, https://workflowhub.eu/workflows/346?version=1, https://workflowhub.eu/workflows/395?version=15, https://workflowhub.eu/workflows/399?version=19, https://workflowhub.eu/workflows/602?version=2, https://workflowhub.eu/workflows/612?version=25, https://workflowhub.eu/workflows/641?version=32, https://workflowhub.eu/workflows/642?version=25, https://workflowhub.eu/workflows/66?version=1, https://workflowhub.eu/workflows/67?version=1, https://workflowhub.eu/workflows/688?version=1</td>
      <td>assembly/metagenomics-assembly, assembly/vgp_genome_assembly, contributing/create-new-tutorial, ecology/ENA_Biodiv_submission, epigenetics/atac-seq, genome-annotation/crispr-screen, genome-annotation/tnseq, microbiome/metagenomics-assembly, microbiome/metatranscriptomics, microbiome/metatranscriptomics-short, sequence-analysis/quality-control, transcriptomics/clipseq, transcriptomics/minerva-pathways, transcriptomics/ref-based, transcriptomics/rna-interactome, transcriptomics/rna-seq-reads-to-counts</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>dada2</td>
      <td>dada2_assignTaxonomyAddspecies, dada2_dada, dada2_filterAndTrim, dada2_learnErrors, dada2_makeSequenceTable, dada2_mergePairs, dada2_plotComplexity, dada2_plotQualityProfile, dada2_primerCheck, dada2_removeBimeraDenovo, dada2_seqCounts</td>
      <td>DADA2 wrappers</td>
      <td>2019-07-05</td>
      <td>https://benjjneb.github.io/dada2/index.html</td>
      <td>1.34.0</td>
      <td>bioconductor-dada2</td>
      <td>1.34.0</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Variant calling, DNA barcoding</td>
      <td>Variant calling, DNA barcoding</td>
      <td>Sequencing, Genetic variation, Microbial ecology, Metagenomics</td>
      <td>Sequencing, Genetic variation, Microbial ecology, Metagenomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/dada2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/dada2</td>
      <td>dada2</td>
      <td>dada2</td>
      <td>This package infers exact sequence variants (SVs) from amplicon data, replacing the commonly used and coarser OTU clustering approach. This pipeline inputs demultiplexed fastq files, and outputs the sequence variants and their sample-wise abundances after removing substitution and chimera errors. Taxonomic classification is available via a native implementation of the RDP naive Bayesian classifier.</td>
      <td></td>
      <td>10</td>
      <td>10</td>
      <td>11</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>865</td>
      <td>885</td>
      <td>205808</td>
      <td>209081</td>
      <td>1301</td>
      <td>1344</td>
      <td>254475</td>
      <td>270135</td>
      <td>179</td>
      <td>179</td>
      <td>52639</td>
      <td>52639</td>
      <td>72</td>
      <td>72</td>
      <td>38131</td>
      <td>38131</td>
      <td>569986</td>
      <td>551053</td>
      <td>2480</td>
      <td>2417</td>
      <td>https://usegalaxy.eu/published/workflow?id=48d0dc424e71d8c2, https://usegalaxy.eu/published/workflow?id=642f4c47028031d7, https://usegalaxy.eu/published/workflow?id=7b557efda89bb998, https://usegalaxy.eu/published/workflow?id=b483cf6a138d2372, https://usegalaxy.eu/published/workflow?id=e132370adf8f2838, https://usegalaxy.eu/published/workflow?id=e3918808605cf8cd, https://usegalaxy.fr/published/workflow?id=b5d86d505c09d170, https://usegalaxy.org.au/published/workflow?id=1016f17b77a8216d, https://usegalaxy.org.au/published/workflow?id=2d05f15a9616cc47, https://usegalaxy.org.au/published/workflow?id=400fbd96fb04c60e, https://usegalaxy.org/published/workflow?id=0942f2f3386e6b41, https://usegalaxy.org/published/workflow?id=38f9fc867c4d1f40, https://usegalaxy.org/published/workflow?id=4faff660f6defa10, https://usegalaxy.org/published/workflow?id=5df593510c7d944d, https://usegalaxy.org/published/workflow?id=78e60413358273bc, https://usegalaxy.org/published/workflow?id=806ef8816168916f, https://usegalaxy.org/published/workflow?id=e067c4e66f0af362, https://usegalaxy.org/published/workflow?id=ecf3a7f9da7409ef, https://workflowhub.eu/workflows/1395?version=2, https://workflowhub.eu/workflows/790?version=3</td>
      <td>microbiome/dada-16S</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>das_tool</td>
      <td>Fasta_to_Contig2Bin, das_tool</td>
      <td>DAS Tool for genome resolved metagenomics</td>
      <td>2022-06-23</td>
      <td>https://github.com/cmks/DAS_Tool</td>
      <td>1.1.7</td>
      <td>das_tool</td>
      <td>1.1.7</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Read binning</td>
      <td>Read binning</td>
      <td>Metagenomics</td>
      <td>Metagenomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/das_tool</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/das_tool</td>
      <td>dastool</td>
      <td>dastool</td>
      <td>DAS Tool is an automated method that integrates the results of a flexible number of binning algorithms to calculate an optimized, non-redundant set of bins from a single assembly.</td>
      <td></td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>77</td>
      <td>77</td>
      <td>6906</td>
      <td>6906</td>
      <td>12</td>
      <td>12</td>
      <td>283</td>
      <td>283</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>1185</td>
      <td>1185</td>
      <td>8374</td>
      <td>8374</td>
      <td>92</td>
      <td>92</td>
      <td>https://usegalaxy.eu/published/workflow?id=00f15824c7a9b196, https://usegalaxy.eu/published/workflow?id=0563f58718be932d, https://usegalaxy.eu/published/workflow?id=1e5f1d9a51032e44, https://usegalaxy.eu/published/workflow?id=43098b8afe78b071, https://usegalaxy.eu/published/workflow?id=7ecd92cb382d8168, https://usegalaxy.eu/published/workflow?id=97312d273b6e8bd9, https://usegalaxy.eu/published/workflow?id=ce5c9287a67e6085, https://workflowhub.eu/workflows/1352?version=4, https://workflowhub.eu/workflows/2028?version=1</td>
      <td>microbiome/metagenomics-binning</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>decontam</td>
      <td>decontam</td>
      <td>Removes decontamination features (ASVs/OTUs) using control samples</td>
      <td>2024-09-13</td>
      <td>https://github.com/benjjneb/decontam</td>
      <td>1.22.0</td>
      <td>bioconductor-decontam</td>
      <td>1.30.0</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Classification</td>
      <td>Classification</td>
      <td>Metagenomics, Microbial ecology, Sequencing</td>
      <td>Metagenomics, Microbial ecology, Sequencing</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/decontam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/decontam</td>
      <td>decontam</td>
      <td>decontam</td>
      <td>Simple statistical identification of contaminating sequence features in marker-gene or metagenomics data. Works on any kind of feature derived from environmental sequencing data. Requires DNA quantitation data or sequenced negative control samples.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>32</td>
      <td>32</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>32</td>
      <td>32</td>
      <td>8</td>
      <td>8</td>
      <td>https://usegalaxy.eu/published/workflow?id=466bdd8ba7b67264</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>deeparg</td>
      <td>deeparg_predict, deeparg_short_reads</td>
      <td>A deep learning based approach to predict Antibiotic Resistance Genes (ARGs) from metagenomes</td>
      <td>2024-10-07</td>
      <td>https://github.com/gaarangoa/deeparg</td>
      <td>1.0.4</td>
      <td>deeparg</td>
      <td>1.0.4</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Antimicrobial resistance prediction, Antimicrobial resistance prediction</td>
      <td>Antimicrobial resistance prediction, Antimicrobial resistance prediction</td>
      <td>Microbiology</td>
      <td>Microbiology</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/deeparg</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/deeparg</td>
      <td>deeparg</td>
      <td>DeepARG</td>
      <td>A deep learning based approach to predict Antibiotic Resistance Genes (ARGs) from metagenomes.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>20</td>
      <td>20</td>
      <td>108</td>
      <td>108</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>65</td>
      <td>65</td>
      <td>173</td>
      <td>173</td>
      <td>24</td>
      <td>24</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>deseq2</td>
      <td>deseq2</td>
      <td>Differential gene expression analysis based on the negative binomial distribution</td>
      <td>2014-12-03</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/DESeq2.html</td>
      <td>2.11.40.8</td>
      <td>bioconductor-deseq2</td>
      <td>1.46.0</td>
      <td>To update</td>
      <td>Transcriptomics, RNA, Statistics, Single Cell</td>
      <td>Differential gene expression analysis, RNA-Seq analysis</td>
      <td>Differential gene expression analysis, RNA-Seq analysis</td>
      <td>RNA-Seq</td>
      <td>RNA-Seq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/deseq2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/deseq2</td>
      <td>DESeq2</td>
      <td>DESeq2</td>
      <td>R/Bioconductor package for differential gene expression analysis based on the negative binomial distribution. Estimate variance-mean dependence in count data from high-throughput sequencing assays and test for differential expression based on a model using the negative binomial distribution.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>13707</td>
      <td>15994</td>
      <td>110682</td>
      <td>133936</td>
      <td>27214</td>
      <td>37996</td>
      <td>191581</td>
      <td>286575</td>
      <td>2980</td>
      <td>3614</td>
      <td>19899</td>
      <td>24448</td>
      <td>820</td>
      <td>824</td>
      <td>4511</td>
      <td>4530</td>
      <td>449489</td>
      <td>326673</td>
      <td>58428</td>
      <td>44721</td>
      <td>https://usegalaxy.eu/published/workflow?id=02a212cc9f134570, https://usegalaxy.eu/published/workflow?id=0de150b19102cb7e, https://usegalaxy.eu/published/workflow?id=0fa523529c4f73cf, https://usegalaxy.eu/published/workflow?id=12c80c5b5e2305d8, https://usegalaxy.eu/published/workflow?id=17ad465d9515cd25, https://usegalaxy.eu/published/workflow?id=1fe02012c96b731f, https://usegalaxy.eu/published/workflow?id=1ffc058273ab357e, https://usegalaxy.eu/published/workflow?id=20ada53472782c9b, https://usegalaxy.eu/published/workflow?id=2287afbf248d9c89, https://usegalaxy.eu/published/workflow?id=250e6058f160825f, https://usegalaxy.eu/published/workflow?id=2d2d0c2231babc50, https://usegalaxy.eu/published/workflow?id=350d7e6976e15d30, https://usegalaxy.eu/published/workflow?id=5b46c8533cd95447, https://usegalaxy.eu/published/workflow?id=5c8186f1e342c522, https://usegalaxy.eu/published/workflow?id=63964ba7d9edce74, https://usegalaxy.eu/published/workflow?id=68e8042f11d5a3fd, https://usegalaxy.eu/published/workflow?id=7136ce9f18c74853, https://usegalaxy.eu/published/workflow?id=76389d8978ec26eb, https://usegalaxy.eu/published/workflow?id=81f1927bfe3ffea1, https://usegalaxy.eu/published/workflow?id=82ef590b5821805d, https://usegalaxy.eu/published/workflow?id=8b1e2fe9f4a07de0, https://usegalaxy.eu/published/workflow?id=95c4d5549e820d88, https://usegalaxy.eu/published/workflow?id=9c7a218993788493, https://usegalaxy.eu/published/workflow?id=9e01e24c5e749365, https://usegalaxy.eu/published/workflow?id=a9469155eee12ff1, https://usegalaxy.eu/published/workflow?id=bc79a9f797334b01, https://usegalaxy.eu/published/workflow?id=c5e6047b3b256b06, https://usegalaxy.eu/published/workflow?id=ca286c2fdc71e960, https://usegalaxy.eu/published/workflow?id=ccef778cfd9024d1, https://usegalaxy.eu/published/workflow?id=d41b52a15ba56f54, https://usegalaxy.eu/published/workflow?id=d6f7e86d52d43555, https://usegalaxy.eu/published/workflow?id=d72edb9f8c529f47, https://usegalaxy.eu/published/workflow?id=e7363b9f450c56b4, https://usegalaxy.eu/published/workflow?id=e764c594f070f107, https://usegalaxy.eu/published/workflow?id=e7e26967cf6cb8d8, https://usegalaxy.eu/published/workflow?id=e8202b7b6338005d, https://usegalaxy.eu/published/workflow?id=e95729ed24f7c7d0, https://usegalaxy.eu/published/workflow?id=f026c4b8341ff94c, https://usegalaxy.eu/published/workflow?id=f33b5135ee99e0dc, https://usegalaxy.eu/published/workflow?id=f58526da1fd68108, https://usegalaxy.fr/published/workflow?id=ff14dc4005e92cfd, https://usegalaxy.org.au/published/workflow?id=000cdc969e9f0886, https://usegalaxy.org.au/published/workflow?id=01997cb92a2ce2d5, https://usegalaxy.org.au/published/workflow?id=02b8d033d9b1fa05, https://usegalaxy.org.au/published/workflow?id=046a1c600a6b77c2, https://usegalaxy.org.au/published/workflow?id=04ab43fb3e7840ef, https://usegalaxy.org.au/published/workflow?id=05950987aa2eba5b, https://usegalaxy.org.au/published/workflow?id=05b17b9d70c1db8b, https://usegalaxy.org.au/published/workflow?id=0614f24ebeb680de, https://usegalaxy.org.au/published/workflow?id=068f85b8b137bfd7, https://usegalaxy.org.au/published/workflow?id=07c97b9e59c0218f, https://usegalaxy.org.au/published/workflow?id=0c567180448d8774, https://usegalaxy.org.au/published/workflow?id=0cfc1408fd9ea4e7, https://usegalaxy.org.au/published/workflow?id=0d1db28665d19bda, https://usegalaxy.org.au/published/workflow?id=0fc7c14c4d1ac8f5, https://usegalaxy.org.au/published/workflow?id=10acf4f497172400, https://usegalaxy.org.au/published/workflow?id=114fb9589bb9db92, https://usegalaxy.org.au/published/workflow?id=13a2c5bd07705985, https://usegalaxy.org.au/published/workflow?id=14184bc2580a131f, https://usegalaxy.org.au/published/workflow?id=16f4846b4d6bc155, https://usegalaxy.org.au/published/workflow?id=19315e6573c58239, https://usegalaxy.org.au/published/workflow?id=1ad81f95ace878ea, https://usegalaxy.org.au/published/workflow?id=1c886f008a36b0b8, https://usegalaxy.org.au/published/workflow?id=20e5e6b233ffb951, https://usegalaxy.org.au/published/workflow?id=210ca013cc1cf185, https://usegalaxy.org.au/published/workflow?id=25ac9b4f2e39af97, https://usegalaxy.org.au/published/workflow?id=28705b86e0595ded, https://usegalaxy.org.au/published/workflow?id=2999a1891ffe201b, https://usegalaxy.org.au/published/workflow?id=2a0046582f5f15a1, https://usegalaxy.org.au/published/workflow?id=2c58509f80897972, https://usegalaxy.org.au/published/workflow?id=2cef9a1633e0a067, https://usegalaxy.org.au/published/workflow?id=2f7c0e3221cbca2f, https://usegalaxy.org.au/published/workflow?id=301fdb29c513ffbe, https://usegalaxy.org.au/published/workflow?id=30b2cd025e0300da, https://usegalaxy.org.au/published/workflow?id=31449596b22f81f4, https://usegalaxy.org.au/published/workflow?id=35384818ee1cbac2, https://usegalaxy.org.au/published/workflow?id=374311292d72c9f9, https://usegalaxy.org.au/published/workflow?id=383ce49fa7a367e9, https://usegalaxy.org.au/published/workflow?id=39d56981a6854d82, https://usegalaxy.org.au/published/workflow?id=3bc45aaed5da1058, https://usegalaxy.org.au/published/workflow?id=3c727d5cd349a846, https://usegalaxy.org.au/published/workflow?id=3ce9dc804780e4a1, https://usegalaxy.org.au/published/workflow?id=3ef07022656067b1, https://usegalaxy.org.au/published/workflow?id=3f45685df9b9e792, https://usegalaxy.org.au/published/workflow?id=410d0d19b6589c78, https://usegalaxy.org.au/published/workflow?id=421adea2f259ec26, https://usegalaxy.org.au/published/workflow?id=42305ed876d19d08, https://usegalaxy.org.au/published/workflow?id=45c8789533c45abf, https://usegalaxy.org.au/published/workflow?id=4e096a68187ce652, https://usegalaxy.org.au/published/workflow?id=4efed79608f85604, https://usegalaxy.org.au/published/workflow?id=4f2818b6620037af, https://usegalaxy.org.au/published/workflow?id=4f562265cbb14cd6, https://usegalaxy.org.au/published/workflow?id=525612313de7591e, https://usegalaxy.org.au/published/workflow?id=53010c2f1f207429, https://usegalaxy.org.au/published/workflow?id=5314a7a4732a0ef1, https://usegalaxy.org.au/published/workflow?id=544cb12a16f283b8, https://usegalaxy.org.au/published/workflow?id=5490661d0153962b, https://usegalaxy.org.au/published/workflow?id=56599572e0364e2a, https://usegalaxy.org.au/published/workflow?id=5779029c2c13c75a, https://usegalaxy.org.au/published/workflow?id=5a4e49ef8693d7c4, https://usegalaxy.org.au/published/workflow?id=5be983b903469f07, https://usegalaxy.org.au/published/workflow?id=5cbee52cf03b6f9a, https://usegalaxy.org.au/published/workflow?id=5ce00d3eab862f4e, https://usegalaxy.org.au/published/workflow?id=5ea74382fb77eb10, https://usegalaxy.org.au/published/workflow?id=608f3ad8f485e7a8, https://usegalaxy.org.au/published/workflow?id=6204bbcfe56931eb, https://usegalaxy.org.au/published/workflow?id=625f3c52832b0591, https://usegalaxy.org.au/published/workflow?id=6448b82959076a29, https://usegalaxy.org.au/published/workflow?id=6469ade8f6f7c718, https://usegalaxy.org.au/published/workflow?id=66b5c7dd3e58193f, https://usegalaxy.org.au/published/workflow?id=67191ea9fe977e08, https://usegalaxy.org.au/published/workflow?id=677ef3322366c953, https://usegalaxy.org.au/published/workflow?id=69233fb6edfcdde9, https://usegalaxy.org.au/published/workflow?id=6a161c87d19a543c, https://usegalaxy.org.au/published/workflow?id=6b34c95a247d15e3, https://usegalaxy.org.au/published/workflow?id=6cdbcd1f667d0f16, https://usegalaxy.org.au/published/workflow?id=6f0801fecbba5d3f, https://usegalaxy.org.au/published/workflow?id=71bad95f72c9f30d, https://usegalaxy.org.au/published/workflow?id=721e528c2ae6d8ee, https://usegalaxy.org.au/published/workflow?id=73ff1aa9acc0ad2a, https://usegalaxy.org.au/published/workflow?id=74186b006665daf8, https://usegalaxy.org.au/published/workflow?id=75113a1b41904d2a, https://usegalaxy.org.au/published/workflow?id=76323f103844f75b, https://usegalaxy.org.au/published/workflow?id=79522a1f7b0c53cb, https://usegalaxy.org.au/published/workflow?id=7ad277dd448e57b3, https://usegalaxy.org.au/published/workflow?id=7becafd394472dfb, https://usegalaxy.org.au/published/workflow?id=7c866ed4a3552d0d, https://usegalaxy.org.au/published/workflow?id=7cd1acb1da823cdc, https://usegalaxy.org.au/published/workflow?id=7ea616c652d08fb6, https://usegalaxy.org.au/published/workflow?id=7ef245f5ae3b916d, https://usegalaxy.org.au/published/workflow?id=7febc6ac6f106e57, https://usegalaxy.org.au/published/workflow?id=83680c1c6dffb05b, https://usegalaxy.org.au/published/workflow?id=83a535840b388980, https://usegalaxy.org.au/published/workflow?id=8483ae422665a6c1, https://usegalaxy.org.au/published/workflow?id=8483e8d3e8d0ddc9, https://usegalaxy.org.au/published/workflow?id=84a8eb379440656e, https://usegalaxy.org.au/published/workflow?id=8597166abda0a375, https://usegalaxy.org.au/published/workflow?id=85aede91db0122e4, https://usegalaxy.org.au/published/workflow?id=863eee1b041b6c58, https://usegalaxy.org.au/published/workflow?id=879d4fb110a61f7a, https://usegalaxy.org.au/published/workflow?id=897dfa78c3b25356, https://usegalaxy.org.au/published/workflow?id=897e9d7432d231e9, https://usegalaxy.org.au/published/workflow?id=8b8a5b50f2edc72c, https://usegalaxy.org.au/published/workflow?id=8c40b4a911517cc5, https://usegalaxy.org.au/published/workflow?id=8cab31db0bd3ff52, https://usegalaxy.org.au/published/workflow?id=8d196c8c62530dc0, https://usegalaxy.org.au/published/workflow?id=8eab5e2b2c683076, https://usegalaxy.org.au/published/workflow?id=8fb691706f47527a, https://usegalaxy.org.au/published/workflow?id=91cfd96f697ba034, https://usegalaxy.org.au/published/workflow?id=925c167abd5610ae, https://usegalaxy.org.au/published/workflow?id=95a9070469a895c3, https://usegalaxy.org.au/published/workflow?id=95efb414f5984735, https://usegalaxy.org.au/published/workflow?id=975ddb0eb436f1fd, https://usegalaxy.org.au/published/workflow?id=97628b0604d5ec77, https://usegalaxy.org.au/published/workflow?id=978953f492b2b914, https://usegalaxy.org.au/published/workflow?id=981d7b02416b7734, https://usegalaxy.org.au/published/workflow?id=9898348d95033af2, https://usegalaxy.org.au/published/workflow?id=9988bc2402f32dc1, https://usegalaxy.org.au/published/workflow?id=9b1afa59c087eacd, https://usegalaxy.org.au/published/workflow?id=9df67170c9260539, https://usegalaxy.org.au/published/workflow?id=9e8d9b7055b7c863, https://usegalaxy.org.au/published/workflow?id=9ec9fe2715d7cfdc, https://usegalaxy.org.au/published/workflow?id=a1c59e9f5c7327d2, https://usegalaxy.org.au/published/workflow?id=a69db7f914c06407, https://usegalaxy.org.au/published/workflow?id=a7346c25dfdf2e3e, https://usegalaxy.org.au/published/workflow?id=a7726d18d2a5fcf2, https://usegalaxy.org.au/published/workflow?id=a7a30d970584f9ac, https://usegalaxy.org.au/published/workflow?id=a7d1cf2bb636e695, https://usegalaxy.org.au/published/workflow?id=a9e59d38c6c710bc, https://usegalaxy.org.au/published/workflow?id=a9fa6101f1fb13e2, https://usegalaxy.org.au/published/workflow?id=aa94c77b720a2810, https://usegalaxy.org.au/published/workflow?id=aaaf580eb00459b6, https://usegalaxy.org.au/published/workflow?id=aae07f802002fdd0, https://usegalaxy.org.au/published/workflow?id=ad13b4a61e8987b1, https://usegalaxy.org.au/published/workflow?id=ae5dca8108d1a8a6, https://usegalaxy.org.au/published/workflow?id=ae6892f1df022097, https://usegalaxy.org.au/published/workflow?id=ae773dfaa074cb72, https://usegalaxy.org.au/published/workflow?id=aed93da6fd6e4e13, https://usegalaxy.org.au/published/workflow?id=af3f40cba4d7afa6, https://usegalaxy.org.au/published/workflow?id=af990cca72794142, https://usegalaxy.org.au/published/workflow?id=afd9a8bbc07f9c3f, https://usegalaxy.org.au/published/workflow?id=b57bf134896c30ea, https://usegalaxy.org.au/published/workflow?id=b6db525b1b73394b, https://usegalaxy.org.au/published/workflow?id=bda98c5a0ea70dd0, https://usegalaxy.org.au/published/workflow?id=c00b5bc4d20f047d, https://usegalaxy.org.au/published/workflow?id=c1a0a3888d9ee3ac, https://usegalaxy.org.au/published/workflow?id=c296b3848708f6f1, https://usegalaxy.org.au/published/workflow?id=c2e5a43356042b30, https://usegalaxy.org.au/published/workflow?id=c3b3717502ff6f76, https://usegalaxy.org.au/published/workflow?id=c3bb21f20a3ce689, https://usegalaxy.org.au/published/workflow?id=c3e198bc2146b4ad, https://usegalaxy.org.au/published/workflow?id=c65a5af6c0383f8c, https://usegalaxy.org.au/published/workflow?id=cb960edf95d2dc35, https://usegalaxy.org.au/published/workflow?id=ce7f6a1c824920e2, https://usegalaxy.org.au/published/workflow?id=cef4a514f35576e5, https://usegalaxy.org.au/published/workflow?id=cef68481b305a516, https://usegalaxy.org.au/published/workflow?id=d126940d33e63036, https://usegalaxy.org.au/published/workflow?id=d5717afa6cce6fb4, https://usegalaxy.org.au/published/workflow?id=d5ecfdbc04b24227, https://usegalaxy.org.au/published/workflow?id=d7a65f577fe5f16b, https://usegalaxy.org.au/published/workflow?id=db0bc7357c181222, https://usegalaxy.org.au/published/workflow?id=db3370a412c4de78, https://usegalaxy.org.au/published/workflow?id=dce29c35dfac48b1, https://usegalaxy.org.au/published/workflow?id=de1c04a2caf70647, https://usegalaxy.org.au/published/workflow?id=e08ef6b519f5fba0, https://usegalaxy.org.au/published/workflow?id=e1fd89e7248b6661, https://usegalaxy.org.au/published/workflow?id=e3cda847fe727b61, https://usegalaxy.org.au/published/workflow?id=e4ddf33ee5b5e259, https://usegalaxy.org.au/published/workflow?id=e52a3479c8216291, https://usegalaxy.org.au/published/workflow?id=e826584d78b69467, https://usegalaxy.org.au/published/workflow?id=e8d3651c07ce0bec, https://usegalaxy.org.au/published/workflow?id=eb3a4535204383fa, https://usegalaxy.org.au/published/workflow?id=ec79c2fab8eac15d, https://usegalaxy.org.au/published/workflow?id=ed2f72e3537e8a4a, https://usegalaxy.org.au/published/workflow?id=f0ced9a5d34b2aff, https://usegalaxy.org.au/published/workflow?id=f19d63d7c9a48160, https://usegalaxy.org.au/published/workflow?id=f268c181a47d0cab, https://usegalaxy.org.au/published/workflow?id=f5c0b0bc5e023e42, https://usegalaxy.org.au/published/workflow?id=f944f2d3fbd1f062, https://usegalaxy.org.au/published/workflow?id=fb644a9649f36f40, https://usegalaxy.org.au/published/workflow?id=fc64a77321752a3c, https://usegalaxy.org.au/published/workflow?id=fc6f8afdee980f0e, https://usegalaxy.org.au/published/workflow?id=fd477917a3d27f61, https://usegalaxy.org.au/published/workflow?id=fd8fb237f8dc24c3, https://usegalaxy.org.au/published/workflow?id=fe1de81a97578368, https://usegalaxy.org.au/published/workflow?id=fe893d72526eba71, https://usegalaxy.org.au/published/workflow?id=ff42213dda7d3e59, https://usegalaxy.org/published/workflow?id=030814e5329ea62d, https://usegalaxy.org/published/workflow?id=092be3cf6e40282c, https://usegalaxy.org/published/workflow?id=0930f955bdac93e8, https://usegalaxy.org/published/workflow?id=098375608a1223aa, https://usegalaxy.org/published/workflow?id=19fcce20ecaf2ef6, https://usegalaxy.org/published/workflow?id=1bd0d7ef0aca1363, https://usegalaxy.org/published/workflow?id=25cdc47baba556c5, https://usegalaxy.org/published/workflow?id=27144c7e44d3f56b, https://usegalaxy.org/published/workflow?id=2d50de9b95ad0808, https://usegalaxy.org/published/workflow?id=36a92f1f6d2212a4, https://usegalaxy.org/published/workflow?id=4061e061f659449b, https://usegalaxy.org/published/workflow?id=40a65b5bc1dcc0b4, https://usegalaxy.org/published/workflow?id=420a332da24fb9e9, https://usegalaxy.org/published/workflow?id=4c0785fe74ab0a24, https://usegalaxy.org/published/workflow?id=4c840284d2984fa0, https://usegalaxy.org/published/workflow?id=53834a5fb46ca0f6, https://usegalaxy.org/published/workflow?id=5a272f266a1fa4ac, https://usegalaxy.org/published/workflow?id=667493d318b86b7d, https://usegalaxy.org/published/workflow?id=68832752b883c226, https://usegalaxy.org/published/workflow?id=69af7dd51aff958a, https://usegalaxy.org/published/workflow?id=6dada297572d77bd, https://usegalaxy.org/published/workflow?id=7e81153760fb4c90, https://usegalaxy.org/published/workflow?id=82681bd5f059e39a, https://usegalaxy.org/published/workflow?id=8700f593a3d642f5, https://usegalaxy.org/published/workflow?id=9bace540b35b1e61, https://usegalaxy.org/published/workflow?id=9e74e3455e47c813, https://usegalaxy.org/published/workflow?id=a743746151101352, https://usegalaxy.org/published/workflow?id=a8ddab36010ed4c3, https://usegalaxy.org/published/workflow?id=aae5b6d9beb860ba, https://usegalaxy.org/published/workflow?id=abf1aec0f3d48d03, https://usegalaxy.org/published/workflow?id=ac8b74b911778fbc, https://usegalaxy.org/published/workflow?id=b0e19e7315b64c87, https://usegalaxy.org/published/workflow?id=b77fa73d4fbde131, https://usegalaxy.org/published/workflow?id=b882703ca6fea3a8, https://usegalaxy.org/published/workflow?id=bc83c42b44c4b03f, https://usegalaxy.org/published/workflow?id=c8ee2da45b6889b9, https://usegalaxy.org/published/workflow?id=cd56ed91856fcc6f, https://usegalaxy.org/published/workflow?id=ce99c56e45d982cb, https://usegalaxy.org/published/workflow?id=cff93a6f24af5fee, https://usegalaxy.org/published/workflow?id=d2f7fd589a91124a, https://usegalaxy.org/published/workflow?id=d72e33effe07de02, https://usegalaxy.org/published/workflow?id=d761637d43ee0324, https://usegalaxy.org/published/workflow?id=d8afd017905d6702, https://usegalaxy.org/published/workflow?id=dab54002b3c0e351, https://usegalaxy.org/published/workflow?id=dcc7c5fdef8780fd, https://usegalaxy.org/published/workflow?id=dcf6f201b787d148, https://usegalaxy.org/published/workflow?id=dec0cf06fa059e65, https://usegalaxy.org/published/workflow?id=e26fabceece3b858, https://usegalaxy.org/published/workflow?id=e40c2e7f6b73fb11, https://usegalaxy.org/published/workflow?id=e790587f0a61fbad, https://usegalaxy.org/published/workflow?id=eb01b539d17580c3, https://usegalaxy.org/published/workflow?id=eb4b32fcf4a9f64a, https://usegalaxy.org/published/workflow?id=f82f893b94cfa1ac, https://usegalaxy.org/published/workflow?id=f92c83df28d4896f, https://usegalaxy.org/published/workflow?id=fb903de4aa4d5090, https://workflowhub.eu/workflows/1201?version=5, https://workflowhub.eu/workflows/1676?version=1, https://workflowhub.eu/workflows/1677?version=1, https://workflowhub.eu/workflows/1685?version=1, https://workflowhub.eu/workflows/1708?version=1, https://workflowhub.eu/workflows/1713?version=2, https://workflowhub.eu/workflows/1717?version=2, https://workflowhub.eu/workflows/338?version=1, https://workflowhub.eu/workflows/412?version=1</td>
      <td>transcriptomics/de-novo, transcriptomics/mirna-target-finder, transcriptomics/network-analysis-with-heinz, transcriptomics/ref-based, transcriptomics/rna-seq-analysis-clustering-viz, transcriptomics/srna</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>dfast</td>
      <td>dfast</td>
      <td>Prokaryotic genome annotation</td>
      <td>2025-04-16</td>
      <td>https://github.com/nigyta/dfast_core/</td>
      <td>1.3.6</td>
      <td>dfast</td>
      <td>1.3.7</td>
      <td>To update</td>
      <td>Genome annotation</td>
      <td>Genome annotation</td>
      <td>Genome annotation</td>
      <td>Nucleic acid structure analysis, Genomics, Sequence analysis</td>
      <td>Nucleic acid structure analysis, Genomics, Sequence analysis</td>
      <td>ufz</td>
      <td>https://github.com/Helmholtz-UFZ/ufz-galaxy-tools/blob/main/tools/dfast/</td>
      <td>https://github.com/Helmholtz-UFZ/galaxy-tools/tree/main/tools/dfast</td>
      <td>dfast</td>
      <td>DFAST</td>
      <td>Flexible prokaryotic genome annotation pipeline for faster genome publication.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>diamond</td>
      <td>bg_diamond, bg_diamond_makedb, bg_diamond_view</td>
      <td>DIAMOND is a new alignment tool for aligning short DNA sequencing reads to a protein reference database such as NCBI-NR.</td>
      <td>2021-03-21</td>
      <td>https://github.com/bbuchfink/diamond</td>
      <td>2.1.16</td>
      <td>diamond</td>
      <td>2.1.19</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Sequence alignment analysis</td>
      <td>Sequence alignment analysis</td>
      <td>Sequence analysis, Proteins</td>
      <td>Sequence analysis, Proteins</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/diamond</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/diamond</td>
      <td>diamond</td>
      <td>Diamond</td>
      <td>Sequence aligner for protein and translated DNA searches and functions as a drop-in replacement for the NCBI BLAST software tools. It is suitable for protein-protein search as well as DNA-protein search on short reads and longer sequences including contigs and assemblies, providing a speedup of BLAST ranging up to x20,000.</td>
      <td></td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>1620</td>
      <td>1706</td>
      <td>112648</td>
      <td>114376</td>
      <td>665</td>
      <td>1048</td>
      <td>35921</td>
      <td>43599</td>
      <td>446</td>
      <td>448</td>
      <td>65850</td>
      <td>66062</td>
      <td>125</td>
      <td>125</td>
      <td>2180</td>
      <td>2208</td>
      <td>226245</td>
      <td>216599</td>
      <td>3327</td>
      <td>2856</td>
      <td>https://usegalaxy.eu/published/workflow?id=2ee0eb74e839b476, https://usegalaxy.eu/published/workflow?id=559f079f6510aa24, https://usegalaxy.eu/published/workflow?id=86900be4282d4b1d, https://usegalaxy.eu/published/workflow?id=9b9e1e28f3683f92, https://usegalaxy.eu/published/workflow?id=a2c46deea34d9d80, https://usegalaxy.eu/published/workflow?id=bc32937d505135ab, https://usegalaxy.eu/published/workflow?id=bebcbd6f0ffa55c5, https://usegalaxy.eu/published/workflow?id=ff4d5beb6ce1809d, https://usegalaxy.org.au/published/workflow?id=0a3fb089674de144, https://usegalaxy.org/published/workflow?id=b0f85c802c1ea7f8, https://usegalaxy.org/published/workflow?id=e0c33057ac2670ba, https://workflowhub.eu/workflows/1103?version=3, https://workflowhub.eu/workflows/1104?version=2, https://workflowhub.eu/workflows/1518?version=1, https://workflowhub.eu/workflows/1641?version=1</td>
      <td>assembly/ERGA-post-assembly-QC, genome-annotation/functional, genome-annotation/gene-centric, sequence-analysis/ncbi-blast-against-the-madland</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>disco</td>
      <td>disco</td>
      <td>DISCO is a overlap-layout-consensus (OLC) metagenome assembler</td>
      <td>2017-10-26</td>
      <td>http://disco.omicsbio.org/</td>
      <td></td>
      <td>disco</td>
      <td>1.3</td>
      <td>To update</td>
      <td>Metagenomics, Assembly</td>
      <td>Protein sequence analysis</td>
      <td>Protein sequence analysis</td>
      <td>Structure determination</td>
      <td>Structure determination</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/disco</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/disco</td>
      <td>disco</td>
      <td>DISCO</td>
      <td>DISCO is software to perform structure determination of protein homo-oligomers with cyclic symmetry.DISCO computes oligomeric protein structures using geometric constraints derived from RDCs and intermolecular distance restraints such as NOEs or disulfide bonds. When a reliable subunit structure can be calculated from intramolecular restraints, DISCO guarantees that all satisfying oligomer structures will be discovered, yet can run in minutes to hours on only a single desktop-class computer.</td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>127</td>
      <td>163</td>
      <td>280</td>
      <td>453</td>
      <td>406</td>
      <td>435</td>
      <td>1525</td>
      <td>1690</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2143</td>
      <td>1805</td>
      <td>598</td>
      <td>533</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>dose_response_analysis_tool</td>
      <td>dr_curve</td>
      <td>A  tool for analyzing and visualizing the relationship between various doses and their corresponding biological responses</td>
      <td>2024-04-06</td>
      <td>https://github.com/bernt-matthias/mb-galaxy-tools</td>
      <td>3.0.1</td>
      <td>r-drc</td>
      <td>3.0_1</td>
      <td>To update</td>
      <td>Ecology</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ufz</td>
      <td>https://github.com/Helmholtz-UFZ/galaxy-tools/tree/main/tools/tox_tools/dose_responses</td>
      <td>https://github.com/Helmholtz-UFZ/galaxy-tools/tree/main/tools/tox_tools/dose_responses</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>dram</td>
      <td>dram_annotate, dram_distill, dram_merge_annotations, dram_neighborhoods, dram_strainer</td>
      <td>DRAM for distilling microbial metabolism to automate the curation of microbiome function</td>
      <td>2022-09-01</td>
      <td>https://github.com/WrightonLabCSU/DRAM</td>
      <td>1.5.0</td>
      <td>dram</td>
      <td>1.5.0</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Gene functional annotation</td>
      <td>Gene functional annotation</td>
      <td>Metagenomics, Biological databases, Molecular genetics</td>
      <td>Metagenomics, Biological databases, Molecular genetics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/dram</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/dram</td>
      <td>dram</td>
      <td>DRAM</td>
      <td>Distilled and Refined Annotation of Metabolism: A tool for the annotation and curation of function for microbial and viral genomes</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>120</td>
      <td>120</td>
      <td>13633</td>
      <td>13633</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>13633</td>
      <td>13633</td>
      <td>120</td>
      <td>120</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>drep</td>
      <td>drep_compare, drep_dereplicate</td>
      <td>dRep compares and dereplicates genome sets</td>
      <td>2020-01-06</td>
      <td>https://github.com/MrOlm/drep</td>
      <td>3.6.2</td>
      <td>drep</td>
      <td>3.6.2</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Genome comparison</td>
      <td>Genome comparison</td>
      <td>Metagenomics, Genomics, Sequence analysis</td>
      <td>Metagenomics, Sequence analysis</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/drep</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/drep</td>
      <td>drep</td>
      <td>dRep</td>
      <td>Fast and accurate genomic comparisons that enables improved genome recovery from metagenomes through de-replication.</td>
      <td></td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>113</td>
      <td>115</td>
      <td>1708</td>
      <td>1749</td>
      <td>13</td>
      <td>13</td>
      <td>47</td>
      <td>47</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>38</td>
      <td>38</td>
      <td>1273</td>
      <td>1273</td>
      <td>3069</td>
      <td>3028</td>
      <td>166</td>
      <td>164</td>
      <td>https://usegalaxy.eu/published/workflow?id=0563f58718be932d, https://usegalaxy.eu/published/workflow?id=1e5f1d9a51032e44, https://usegalaxy.eu/published/workflow?id=7ecd92cb382d8168, https://usegalaxy.eu/published/workflow?id=97312d273b6e8bd9, https://usegalaxy.eu/published/workflow?id=c62d65832377e376, https://workflowhub.eu/workflows/1352?version=4</td>
      <td>genome-annotation/bacterial-genome-quality-control, microbiome/mags-building, microbiome/metagenomics-binning</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>ectyper</td>
      <td>ectyper</td>
      <td>EC-Typer - in silico serotyping of Escherichia coli species</td>
      <td>2018-12-21</td>
      <td>https://github.com/phac-nml/ecoli_serotyping</td>
      <td>2.0.0</td>
      <td>ectyper</td>
      <td>2.0.0</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Sequence assembly, Read pre-processing, Variant calling</td>
      <td>Sequence assembly, Read pre-processing, Variant calling</td>
      <td>Whole genome sequencing, Public health and epidemiology, Sequence assembly, Genotype and phenotype, Workflows</td>
      <td>Whole genome sequencing, Public health and epidemiology, Sequence assembly, Genotype and phenotype, Workflows</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/ectyper</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/ectyper</td>
      <td>ectyper</td>
      <td>ECTyper</td>
      <td>Ectyper is a standalone serotyping module for Escherichia coli. It supports fasta and fastq file formats. (Galaxy Version 1.0.0)</td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>218</td>
      <td>218</td>
      <td>30463</td>
      <td>30463</td>
      <td>1</td>
      <td>1</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>30473</td>
      <td>30473</td>
      <td>219</td>
      <td>219</td>
      <td>https://usegalaxy.eu/published/workflow?id=b90585626a793502</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>effectivet3</td>
      <td>effectiveT3</td>
      <td>Find bacterial type III effectors in protein sequences</td>
      <td>2013-04-30</td>
      <td>http://effectors.org</td>
      <td>0.0.21</td>
      <td>effectiveT3</td>
      <td>1.0.1</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Sequence classification</td>
      <td>Sequence classification</td>
      <td>Sequence analysis</td>
      <td>Sequence analysis</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/effectiveT3</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/effectiveT3</td>
      <td>effectivet3</td>
      <td>EffectiveT3</td>
      <td>Prediction of putative Type-III secreted proteins.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>eggnog_mapper</td>
      <td>eggnog_mapper, eggnog_mapper_annotate, eggnog_mapper_search</td>
      <td>eggnog-mapper fast functional annotation of novel sequences</td>
      <td>2019-11-11</td>
      <td>https://github.com/eggnogdb/eggnog-mapper</td>
      <td>2.1.13</td>
      <td>eggnog-mapper</td>
      <td>2.1.13</td>
      <td>Up-to-date</td>
      <td>Proteomics</td>
      <td>Homology-based gene prediction, Genome annotation, Fold recognition, Information extraction, Query and retrieval</td>
      <td>Homology-based gene prediction, Genome annotation, Fold recognition, Information extraction, Query and retrieval</td>
      <td>Metagenomics, Phylogeny, Transcriptomics, Workflows, Sequence analysis</td>
      <td>Metagenomics, Phylogeny, Transcriptomics, Workflows, Sequence analysis</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/eggnog_mapper/eggnog_mapper</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/eggnog_mapper/eggnog_mapper</td>
      <td>eggnog-mapper-v2</td>
      <td>eggNOG-mapper v2</td>
      <td>EggNOG-mapper is a tool for fast functional annotation of novel sequences. It uses precomputed orthologous groups and phylogenies from the eggNOG database (http://eggnog5.embl.de) to transfer functional information from fine-grained orthologs only.</td>
      <td></td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>1977</td>
      <td>2018</td>
      <td>64994</td>
      <td>65580</td>
      <td>518</td>
      <td>518</td>
      <td>9634</td>
      <td>9634</td>
      <td>541</td>
      <td>541</td>
      <td>25986</td>
      <td>25986</td>
      <td>144</td>
      <td>144</td>
      <td>66224</td>
      <td>66224</td>
      <td>167424</td>
      <td>166838</td>
      <td>3221</td>
      <td>3180</td>
      <td>https://usegalaxy.cz/published/workflow?id=932f4901960a595a, https://usegalaxy.eu/published/workflow?id=2d2862b48918b22f, https://usegalaxy.eu/published/workflow?id=2ee0eb74e839b476, https://usegalaxy.eu/published/workflow?id=87231a6a7938cb95, https://usegalaxy.eu/published/workflow?id=9b9e1e28f3683f92, https://usegalaxy.eu/published/workflow?id=a448400f69594135, https://usegalaxy.eu/published/workflow?id=c5e6047b3b256b06, https://usegalaxy.eu/published/workflow?id=cc59ae1a9878942c, https://usegalaxy.eu/published/workflow?id=e5becc6d66b2568c, https://usegalaxy.org.au/published/workflow?id=35f39173f0ac9179, https://usegalaxy.org.au/published/workflow?id=4e161a8f5098cc49, https://usegalaxy.org.au/published/workflow?id=876163677f1cccc4, https://usegalaxy.org/published/workflow?id=80565301ada0d3eb, https://usegalaxy.org/published/workflow?id=92613e1e0b13aa0a, https://workflowhub.eu/workflows/1262?version=1, https://workflowhub.eu/workflows/1521?version=1, https://workflowhub.eu/workflows/1525?version=1, https://workflowhub.eu/workflows/2024?version=2, https://workflowhub.eu/workflows/754?version=1, https://workflowhub.eu/workflows/755?version=1</td>
      <td>genome-annotation/funannotate, genome-annotation/functional</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>emboss</td>
      <td>emboss_needle, emboss_needleall</td>
      <td>Galaxy wrappers for EMBOSS6 tools</td>
      <td>2025-01-17</td>
      <td>http://emboss.open-bio.org/</td>
      <td>6.6.0</td>
      <td>emboss</td>
      <td>6.6.0</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis, Fasta Manipulation</td>
      <td>Sequence analysis, Local alignment, Sequence alignment analysis, Global alignment, Sequence alignment</td>
      <td>Sequence analysis, Local alignment, Sequence alignment analysis, Global alignment, Sequence alignment</td>
      <td>Molecular biology, Sequence analysis, Biology</td>
      <td>Molecular biology, Sequence analysis</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/emboss</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/emboss</td>
      <td>emboss</td>
      <td>EMBOSS</td>
      <td>Diverse suite of tools for sequence analysis; many programs analagous to GCG; context-sensitive help for each tool.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>62</td>
      <td>62</td>
      <td>419</td>
      <td>419</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>419</td>
      <td>419</td>
      <td>62</td>
      <td>62</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>emboss_5</td>
      <td>EMBOSS: antigenic1, EMBOSS: backtranseq2, EMBOSS: banana3, EMBOSS: biosed4, EMBOSS: btwisted5, EMBOSS: cai6, EMBOSS: cai_custom6, EMBOSS: chaos7, EMBOSS: charge8, EMBOSS: checktrans9, EMBOSS: chips10, EMBOSS: cirdna11, EMBOSS: codcmp12, EMBOSS: coderet13, EMBOSS: compseq14, EMBOSS: cpgplot15, EMBOSS: cpgreport16, EMBOSS: cusp17, EMBOSS: cutseq18, EMBOSS: dan19, EMBOSS: degapseq20, EMBOSS: descseq21, EMBOSS: diffseq22, EMBOSS: digest23, EMBOSS: dotmatcher24, EMBOSS: dotpath25, EMBOSS: dottup26, EMBOSS: dreg27, EMBOSS: einverted28, EMBOSS: epestfind29, EMBOSS: equicktandem31, EMBOSS: est2genome32, EMBOSS: etandem33, EMBOSS: extractfeat34, EMBOSS: extractseq35, EMBOSS: freak36, EMBOSS: fuzznuc37, EMBOSS: fuzzpro38, EMBOSS: fuzztran39, EMBOSS: garnier40, EMBOSS: geecee41, EMBOSS: getorf42, EMBOSS: helixturnhelix43, EMBOSS: hmoment44, EMBOSS: iep45, EMBOSS: infoseq46, EMBOSS: isochore47, EMBOSS: lindna48, EMBOSS: marscan49, EMBOSS: maskfeat50, EMBOSS: maskseq51, EMBOSS: matcher52, EMBOSS: megamerger53, EMBOSS: merger54, EMBOSS: msbar55, EMBOSS: needle56, EMBOSS: newcpgreport57, EMBOSS: newcpgseek58, EMBOSS: newseq59, EMBOSS: noreturn60, EMBOSS: notseq61, EMBOSS: nthseq62, EMBOSS: octanol63, EMBOSS: oddcomp64, EMBOSS: palindrome65, EMBOSS: pasteseq66, EMBOSS: patmatdb67, EMBOSS: pepcoil68, EMBOSS: pepinfo69, EMBOSS: pepnet70, EMBOSS: pepstats71, EMBOSS: pepwheel72, EMBOSS: pepwindow73, EMBOSS: pepwindowall74, EMBOSS: plotcon75, EMBOSS: plotorf76, EMBOSS: polydot77, EMBOSS: preg78, EMBOSS: prettyplot79, EMBOSS: prettyseq80, EMBOSS: primersearch81, EMBOSS: revseq82, EMBOSS: seqmatchall83, EMBOSS: seqret84, EMBOSS: showfeat85, EMBOSS: shuffleseq87, EMBOSS: sigcleave88, EMBOSS: sirna89, EMBOSS: sixpack90, EMBOSS: skipseq91, EMBOSS: splitter92, EMBOSS: supermatcher95, EMBOSS: syco96, EMBOSS: tcode97, EMBOSS: textsearch98, EMBOSS: tmap99, EMBOSS: tranalign100, EMBOSS: transeq101, EMBOSS: trimest102, EMBOSS: trimseq103, EMBOSS: twofeat104, EMBOSS: union105, EMBOSS: vectorstrip106, EMBOSS: water107, EMBOSS: wobble108, EMBOSS: wordcount109, EMBOSS: wordmatch110</td>
      <td>Galaxy wrappers for EMBOSS version 5.0.0 tools</td>
      <td>2017-01-11</td>
      <td>http://emboss.open-bio.org/</td>
      <td>5.0.0</td>
      <td>emboss</td>
      <td>6.6.0</td>
      <td>To update</td>
      <td>Sequence Analysis, Fasta Manipulation</td>
      <td>Sequence analysis, Local alignment, Sequence alignment analysis, Global alignment, Sequence alignment</td>
      <td>Sequence analysis, Local alignment, Sequence alignment analysis, Global alignment, Sequence alignment</td>
      <td>Molecular biology, Sequence analysis, Biology</td>
      <td>Molecular biology, Sequence analysis</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/emboss_5</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/emboss_5</td>
      <td>emboss</td>
      <td>EMBOSS</td>
      <td>Diverse suite of tools for sequence analysis; many programs analagous to GCG; context-sensitive help for each tool.</td>
      <td></td>
      <td>107</td>
      <td>107</td>
      <td>107</td>
      <td>107</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>107</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>107</td>
      <td>107</td>
      <td>107</td>
      <td>0</td>
      <td>444</td>
      <td>513</td>
      <td>264479</td>
      <td>273772</td>
      <td>1350</td>
      <td>2518</td>
      <td>169846</td>
      <td>325994</td>
      <td>557</td>
      <td>810</td>
      <td>29757</td>
      <td>35329</td>
      <td>51</td>
      <td>51</td>
      <td>997</td>
      <td>1037</td>
      <td>636132</td>
      <td>465079</td>
      <td>3892</td>
      <td>2402</td>
      <td>https://usegalaxy.eu/published/workflow?id=04a4dcac3fd922c5, https://usegalaxy.eu/published/workflow?id=0ed8817a90caa2b1, https://usegalaxy.eu/published/workflow?id=1031e4c4f18e020e, https://usegalaxy.eu/published/workflow?id=1f88311a8e365c17, https://usegalaxy.eu/published/workflow?id=2c79eee5ca5a3b09, https://usegalaxy.eu/published/workflow?id=2cec6a39cd496f09, https://usegalaxy.eu/published/workflow?id=33312e5643279e8a, https://usegalaxy.eu/published/workflow?id=402b14906a03b2ef, https://usegalaxy.eu/published/workflow?id=6e1fe7372e93fae0, https://usegalaxy.eu/published/workflow?id=90f993a1b245eb68, https://usegalaxy.eu/published/workflow?id=9275d0c90a8713d6, https://usegalaxy.eu/published/workflow?id=93d1035a307fe63d, https://usegalaxy.eu/published/workflow?id=a31100610fba17a1, https://usegalaxy.eu/published/workflow?id=ada52ba09b462567, https://usegalaxy.eu/published/workflow?id=b681f7b677b23168, https://usegalaxy.eu/published/workflow?id=b77e3b08092fa73e, https://usegalaxy.eu/published/workflow?id=d22dac11eba8c5a3, https://usegalaxy.eu/published/workflow?id=e7f03b4bd9b3d6fd, https://usegalaxy.eu/published/workflow?id=eaf01ccb13aad3ec, https://usegalaxy.eu/published/workflow?id=eaf3c4d97399e673, https://usegalaxy.fr/published/workflow?id=297b71156b297a73, https://usegalaxy.fr/published/workflow?id=b3974acb70608e38, https://usegalaxy.org.au/published/workflow?id=203f9e69152fbb2a, https://usegalaxy.org.au/published/workflow?id=42137e36791652bc, https://usegalaxy.org.au/published/workflow?id=6b1b0a945d367b4c, https://usegalaxy.org.au/published/workflow?id=94c7fdcbbc7b428a, https://usegalaxy.org.au/published/workflow?id=b432a1312859edd2, https://usegalaxy.org.au/published/workflow?id=dccc8fb49d61ac33, https://usegalaxy.org.au/published/workflow?id=f21446cc44da50ce, https://usegalaxy.org/published/workflow?id=00381bbae3fd7e78, https://usegalaxy.org/published/workflow?id=17d9b233640aefec, https://usegalaxy.org/published/workflow?id=2d1ca299c1b058f6, https://usegalaxy.org/published/workflow?id=3b37e1cb61ec6f8e, https://usegalaxy.org/published/workflow?id=3c7349268d74c576, https://usegalaxy.org/published/workflow?id=4884918895db9af0, https://usegalaxy.org/published/workflow?id=62316da77b1bfa5a, https://usegalaxy.org/published/workflow?id=63adc811e3ae529d, https://usegalaxy.org/published/workflow?id=6b4ad51307a28c7f, https://usegalaxy.org/published/workflow?id=6f836f4b71d399e7, https://usegalaxy.org/published/workflow?id=8d8e8927d63803ef, https://usegalaxy.org/published/workflow?id=8e1c6ad6ff623015, https://usegalaxy.org/published/workflow?id=94da3d448bb8a137, https://usegalaxy.org/published/workflow?id=956987e628530c79, https://usegalaxy.org/published/workflow?id=a666aaaa32f62487, https://usegalaxy.org/published/workflow?id=cc6ee582a6e5cae2, https://usegalaxy.org/published/workflow?id=cdf5dffbed938d49, https://usegalaxy.org/published/workflow?id=d3a69b0b826331ef, https://usegalaxy.org/published/workflow?id=e1dcdc125122f561, https://usegalaxy.org/published/workflow?id=e7ce46762faccaac, https://usegalaxy.org/published/workflow?id=f737593a1db217c8, https://workflowhub.eu/workflows/1035?version=1, https://workflowhub.eu/workflows/1632?version=1, https://workflowhub.eu/workflows/1647?version=1, https://workflowhub.eu/workflows/439?version=4</td>
      <td>sequence-analysis/viral_primer_design, variant-analysis/pox-tiled-amplicon, variant-analysis/tb-variant-analysis</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>ete</td>
      <td>ete_gene_csv_finder, ete_genetree_splitter, ete_homology_classifier, ete_init_taxdb, ete_lineage_generator, ete3_mod, ete_species_tree_generator</td>
      <td>Analyse phylogenetic trees using the ETE Toolkit</td>
      <td>2016-06-01</td>
      <td>https://etetoolkit.org/</td>
      <td>3.1.3</td>
      <td>ete3</td>
      <td>3.1.1</td>
      <td>To update</td>
      <td>Phylogenetics</td>
      <td>Phylogenetic analysis, Phylogenetic tree editing</td>
      <td>Phylogenetic tree editing</td>
      <td>Phylogenetics</td>
      <td>Phylogenetics</td>
      <td>earlhaminst</td>
      <td>https://github.com/EarlhamInst/earlham-galaxytools/tree/main/tools/ete</td>
      <td>https://github.com/EarlhamInst/earlham-galaxytools/tree/main/tools/ete</td>
      <td>ete</td>
      <td>ete</td>
      <td>The Environment for Tree Exploration (ETE) is a computational framework that simplifies the reconstruction, analysis, and visualization of phylogenetic trees and multiple sequence alignments. Here, we present ETE v3, featuring numerous improvements in the underlying library of methods, and providing a novel set of standalone tools to perform common tasks in comparative genomics and phylogenetics. The new features include (i) building gene-based and supermatrix-based phylogenies using a single command, (ii) testing and visualizing evolutionary models, (iii) calculating distances between trees of different size or including duplications, and (iv) providing seamless integration with the NCBI taxonomy database. ETE is freely available at http://etetoolkit.org</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>30</td>
      <td>39</td>
      <td>279</td>
      <td>1030</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>13</td>
      <td>13</td>
      <td>1043</td>
      <td>292</td>
      <td>41</td>
      <td>32</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>ete_treeviewer</td>
      <td>ete_treeviewer</td>
      <td>A toolkit for the analysis and visualization of trees.</td>
      <td>2024-11-25</td>
      <td>http://etetoolkit.org</td>
      <td>3.1.3</td>
      <td>curl</td>
      <td></td>
      <td>To update</td>
      <td>Phylogenetics</td>
      <td>Phylogenetic analysis, Phylogenetic tree editing</td>
      <td>Phylogenetic tree editing</td>
      <td>Phylogenetics</td>
      <td>Phylogenetics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ete-toolkit</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ete-toolkit</td>
      <td>ete</td>
      <td>ete</td>
      <td>The Environment for Tree Exploration (ETE) is a computational framework that simplifies the reconstruction, analysis, and visualization of phylogenetic trees and multiple sequence alignments. Here, we present ETE v3, featuring numerous improvements in the underlying library of methods, and providing a novel set of standalone tools to perform common tasks in comparative genomics and phylogenetics. The new features include (i) building gene-based and supermatrix-based phylogenies using a single command, (ii) testing and visualizing evolutionary models, (iii) calculating distances between trees of different size or including duplications, and (iv) providing seamless integration with the NCBI taxonomy database. ETE is freely available at http://etetoolkit.org</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>130</td>
      <td>130</td>
      <td>1626</td>
      <td>1626</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>5</td>
      <td>5</td>
      <td>1631</td>
      <td>1631</td>
      <td>134</td>
      <td>134</td>
      <td>https://workflowhub.eu/workflows/1650?version=1</td>
      <td>sequence-analysis/tapscan-streptophyte-algae</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>eukcc</td>
      <td>eukcc_single</td>
      <td>Galaxy wrapper for EukCC</td>
      <td>2025-07-25</td>
      <td>https://github.com/EBI-Metagenomics/EukCC</td>
      <td>2.1.3</td>
      <td>eukcc</td>
      <td>2.1.3</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Sequence assembly validation</td>
      <td>Sequence assembly validation</td>
      <td>Metagenomics</td>
      <td>Metagenomics</td>
      <td>ufz</td>
      <td>https://github.com/Helmholtz-UFZ/galaxy-tools/tree/main/tools/eukcc</td>
      <td>https://github.com/Helmholtz-UFZ/galaxy-tools/tree/main/tools/eukcc</td>
      <td>eukcc</td>
      <td>EukCC</td>
      <td>EukCC is a completeness and contamination estimator for metagenomic assembled microbial eukaryotic genomes.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>eukrep</td>
      <td>eukrep</td>
      <td>Classification of Eukaryotic and Prokaryotic sequences from metagenomic datasets</td>
      <td>2025-07-30</td>
      <td>https://github.com/patrickwest/EukRep</td>
      <td>0.6.7</td>
      <td>eukrep</td>
      <td>0.6.7</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Nucleic acid sequence analysis</td>
      <td>Nucleic acid sequence analysis</td>
      <td>Metagenomics</td>
      <td>Metagenomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/eukrep</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/eukrep</td>
      <td>eukrep</td>
      <td>EukRep</td>
      <td>Classification of Eukaryotic and Prokaryotic sequences from metagenomic datasets</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>export2graphlan</td>
      <td>export2graphlan</td>
      <td>export2graphlan is a conversion software tool for producing both annotation and tree file for GraPhlAn</td>
      <td>2017-03-03</td>
      <td>https://bitbucket.org/CibioCM/export2graphlan/overview</td>
      <td>0.20</td>
      <td>export2graphlan</td>
      <td>0.22</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Conversion</td>
      <td>Conversion</td>
      <td>Taxonomy, Metabolomics, Biomarkers</td>
      <td>Taxonomy, Metabolomics, Biomarkers</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/export2graphlan</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/export2graphlan</td>
      <td>export2graphlan</td>
      <td>export2graphlan</td>
      <td>export2graphlan is a conversion software tool for producing both annotation and tree file for GraPhlAn. In particular, the annotation file tries to highlight specific sub-trees deriving automatically from input file what nodes are important.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>589</td>
      <td>710</td>
      <td>6276</td>
      <td>7303</td>
      <td>193</td>
      <td>213</td>
      <td>712</td>
      <td>791</td>
      <td>122</td>
      <td>123</td>
      <td>616</td>
      <td>632</td>
      <td>16</td>
      <td>16</td>
      <td>43</td>
      <td>43</td>
      <td>8769</td>
      <td>7647</td>
      <td>1062</td>
      <td>920</td>
      <td>https://usegalaxy.eu/published/workflow?id=05f5f040337786bf, https://usegalaxy.eu/published/workflow?id=096b75501c8e0888, https://usegalaxy.eu/published/workflow?id=19de5cbae7f48103, https://usegalaxy.eu/published/workflow?id=2cf09adb1ed6c3ec, https://usegalaxy.eu/published/workflow?id=4d32b1b3ac0a7161, https://usegalaxy.eu/published/workflow?id=4f21205e71cabd37, https://usegalaxy.eu/published/workflow?id=5f0faac42b0a5062, https://usegalaxy.eu/published/workflow?id=713a5485e2007f90, https://usegalaxy.eu/published/workflow?id=79b4ae4403772098, https://usegalaxy.eu/published/workflow?id=7ab70660e6235cf0, https://usegalaxy.eu/published/workflow?id=87cc0cd143a12c41, https://usegalaxy.eu/published/workflow?id=994e5ff1df7d6c9e, https://usegalaxy.eu/published/workflow?id=9cbba5de2cbeeb0f, https://usegalaxy.eu/published/workflow?id=a8f829deb69b30ab, https://usegalaxy.eu/published/workflow?id=b5f848dc8e96663c, https://usegalaxy.eu/published/workflow?id=c01ef8354b585a1f, https://usegalaxy.eu/published/workflow?id=c3aa516e78ca4191, https://usegalaxy.eu/published/workflow?id=d05c6f0f3205fbd2, https://usegalaxy.eu/published/workflow?id=dfc2c53ac3103eff, https://usegalaxy.eu/published/workflow?id=ed0ad728cf21702b, https://usegalaxy.eu/published/workflow?id=fdca6c9d394986bf, https://usegalaxy.fr/published/workflow?id=09e8293456ee6d99, https://usegalaxy.fr/published/workflow?id=c4f6062e51d0f45e, https://workflowhub.eu/workflows/1451?version=2, https://workflowhub.eu/workflows/1466?version=2</td>
      <td>microbiome/metatranscriptomics, microbiome/metatranscriptomics-short</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>ez_histograms</td>
      <td>ez_histograms</td>
      <td>ggplot2 histograms and density plots</td>
      <td>2024-02-07</td>
      <td>https://github.com/tidyverse/ggplot2</td>
      <td>3.4.4</td>
      <td>r-ggplot2</td>
      <td>2.2.1</td>
      <td>To update</td>
      <td>Visualization, Statistics</td>
      <td>Data analysis, Statistical calculation, Visualisation</td>
      <td>Statistical calculation, Visualisation</td>
      <td>Ecology</td>
      <td>Ecology</td>
      <td>artbio</td>
      <td>https://github.com/artbio/tools-artbio/tree/main/tools/ez_histograms</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/ez_histograms</td>
      <td>ez_histograms</td>
      <td>ez_histograms</td>
      <td>This tool produces histograms or density plots from any tabular datasets, using ggplot2.You may provide headerless tables as well as tables with column headers. Any non-numeric columns will be filtered out. Your first column may contains row names, but it will also work if your first column already corresponds to the numeric values of a variable.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>fairy</td>
      <td>fairy_cov, fairy_sketch</td>
      <td>Fast approximate contig coverage for metagenomic binning</td>
      <td>2024-12-20</td>
      <td>https://github.com/bluenote-1577/fairy</td>
      <td>0.5.8</td>
      <td>fairy</td>
      <td>0.5.8</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Read binning, Sequence assembly validation</td>
      <td>Read binning, Sequence assembly validation</td>
      <td>Metagenomics</td>
      <td>Metagenomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fairy</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fairy</td>
      <td>fairy</td>
      <td>fairy</td>
      <td>fast approximate contig coverage for metagenomic binningFairy computes multi-sample contig coverage for metagenome-assembled genome (MAG) binning.Fairy is used after metagenomic assembly and before binning. It can- Calculate coverage 100x-1000x faster than read alignment (e.g. BWA)- Give comparable bins for multi-sample binning (short read or nanopore reads)- Output formats that are compatible with MetaBAT2, MaxBin2, SemiBin2, and more</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>148</td>
      <td>148</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>148</td>
      <td>148</td>
      <td>5</td>
      <td>5</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>falco</td>
      <td>falco</td>
      <td>A high throughput sequence QC analysis tool</td>
      <td>2024-04-17</td>
      <td>https://github.com/smithlabcode/falco/</td>
      <td>1.2.4</td>
      <td>falco</td>
      <td>1.2.5</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Sequencing quality control, Visualisation, Read mapping</td>
      <td>Sequencing quality control, Visualisation, Read mapping</td>
      <td>Workflows, Mapping, Imaging</td>
      <td>Workflows, Mapping, Imaging</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/falco</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/falco</td>
      <td>falco</td>
      <td>Falco</td>
      <td>A high-speed FastQC emulation for quality control of sequencing data.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>3878</td>
      <td>3878</td>
      <td>162045</td>
      <td>162045</td>
      <td>965</td>
      <td>965</td>
      <td>16492</td>
      <td>16492</td>
      <td>274</td>
      <td>274</td>
      <td>5029</td>
      <td>5029</td>
      <td>24</td>
      <td>24</td>
      <td>357</td>
      <td>357</td>
      <td>183923</td>
      <td>183923</td>
      <td>5141</td>
      <td>5141</td>
      <td>https://usegalaxy.eu/published/workflow?id=046e5c9726c75a3c, https://usegalaxy.eu/published/workflow?id=259f459a45e17e46, https://usegalaxy.eu/published/workflow?id=3d0039098a87642c, https://usegalaxy.eu/published/workflow?id=4e2dc70a35ada981, https://usegalaxy.eu/published/workflow?id=6368019ffdf6c5ee, https://usegalaxy.eu/published/workflow?id=6f878d8b43ef1521, https://usegalaxy.eu/published/workflow?id=79fa095c9fce9923, https://usegalaxy.eu/published/workflow?id=8b8e89b9b8801f2f, https://usegalaxy.eu/published/workflow?id=b144287d56b3291b, https://usegalaxy.eu/published/workflow?id=c9ef2d2ce92e2dfb, https://usegalaxy.eu/published/workflow?id=d17839eaeac6c4c8, https://usegalaxy.org.au/published/workflow?id=71bad95f72c9f30d, https://usegalaxy.org.au/published/workflow?id=7cd1acb1da823cdc, https://usegalaxy.org/published/workflow?id=38f9fc867c4d1f40, https://usegalaxy.org/published/workflow?id=467d3e260c21de98, https://usegalaxy.org/published/workflow?id=fb789920b057fe8d, https://workflowhub.eu/workflows/1506?version=2, https://workflowhub.eu/workflows/1562?version=1, https://workflowhub.eu/workflows/1602?version=2, https://workflowhub.eu/workflows/1644?version=1, https://workflowhub.eu/workflows/1715?version=3, https://workflowhub.eu/workflows/1717?version=2, https://workflowhub.eu/workflows/1845?version=1, https://workflowhub.eu/workflows/2043?version=1</td>
      <td>assembly/mrsa-illumina, ecology/ENA_Biodiv_submission, epigenetics/cut_and_run, epigenetics/methylation-seq, galaxy-interface/workflow-automation, galaxy-interface/workflow-fairification, sequence-analysis/quality-contamination-control, single-cell/scatac-preprocessing-tenx, transcriptomics/mirna-target-finder, transcriptomics/ref-based, variant-analysis/somatic-variant-discovery, variant-analysis/somatic-variants</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>fargene</td>
      <td>fargene</td>
      <td>fARGene (Fragmented Antibiotic Resistance Gene iENntifiEr )</td>
      <td>2019-08-30</td>
      <td>https://github.com/fannyhb/fargene</td>
      <td>0.1</td>
      <td>fargene</td>
      <td>0.1</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Antimicrobial resistance prediction</td>
      <td>Antimicrobial resistance prediction</td>
      <td>Metagenomics, Microbiology, Public health and epidemiology, Antimicrobial resistance</td>
      <td>Metagenomics, Microbiology, Public health and epidemiology</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fargene</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fargene</td>
      <td>fargene</td>
      <td>fARGene</td>
      <td>fARGene (Fragmented Antibiotic Resistance Gene iENntifiEr ) is a tool that takes either fragmented metagenomic data or longer sequences as input and predicts and delivers full-length antiobiotic resistance genes as output.</td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>150</td>
      <td>159</td>
      <td>895</td>
      <td>1011</td>
      <td>262</td>
      <td>287</td>
      <td>1203</td>
      <td>1346</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2359</td>
      <td>2100</td>
      <td>448</td>
      <td>414</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>fastani</td>
      <td>fastani</td>
      <td>Fast alignment-free computation of whole-genome Average Nucleotide Identity</td>
      <td>2020-02-17</td>
      <td>https://github.com/ParBLiSS/FastANI</td>
      <td>1.3</td>
      <td>fastani</td>
      <td>1.34</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Genome alignment, Sequence similarity search</td>
      <td>Genome alignment, Sequence similarity search</td>
      <td>Microbiology, Genetic variation</td>
      <td>Microbiology, Genetic variation</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fastani</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fastani</td>
      <td>fastani</td>
      <td>FastANI</td>
      <td>FastANI is developed for fast alignment-free computation of whole-genome Average Nucleotide Identity (ANI). ANI is defined as mean nucleotide identity of orthologous gene pairs shared between two microbial genomes. FastANI supports pairwise comparison of both complete and draft genome assemblies.</td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1182</td>
      <td>1205</td>
      <td>29436</td>
      <td>29626</td>
      <td>1</td>
      <td>1</td>
      <td>18</td>
      <td>18</td>
      <td>133</td>
      <td>133</td>
      <td>1799</td>
      <td>1799</td>
      <td>64</td>
      <td>64</td>
      <td>949</td>
      <td>949</td>
      <td>32392</td>
      <td>32202</td>
      <td>1403</td>
      <td>1380</td>
      <td>https://usegalaxy.eu/published/workflow?id=bf48b0eb15e8ff75</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>fastk</td>
      <td>fastk_fastk, fastk_histex, fastk_logex</td>
      <td>FastK: A K-mer counter (for HQ assembly data sets)</td>
      <td>2024-05-03</td>
      <td>https://github.com/thegenemyers/FASTK</td>
      <td>1.1.0</td>
      <td>fastk</td>
      <td>1.2</td>
      <td>To update</td>
      <td>Assembly</td>
      <td>k-mer counting, Sequence analysis</td>
      <td>k-mer counting, Sequence analysis</td>
      <td>Genomics, Sequence analysis</td>
      <td>Genomics, Sequence analysis</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fastk</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fastk</td>
      <td>fastk</td>
      <td>FASTK</td>
      <td>FastK is a kmer counter that is optimized for processing high quality DNA assembly data sets such as those produced with an Illumina instrument or a PacBio run in HiFi mode.</td>
      <td></td>
      <td>3</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>46</td>
      <td>46</td>
      <td>742</td>
      <td>742</td>
      <td>45</td>
      <td>45</td>
      <td>389</td>
      <td>389</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1131</td>
      <td>1131</td>
      <td>91</td>
      <td>91</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>fastoma</td>
      <td>fastoma</td>
      <td>FastOMA is a scalable tool for orthology inference among many genomes.</td>
      <td>2025-12-26</td>
      <td>https://github.com/dessimozlab/FastOMA</td>
      <td>0.5.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>Phylogenetics, Sequence Analysis</td>
      <td>Phylogenetic tree reconciliation, Protein sequence analysis, Genome comparison, Sequence clustering, Phylogenetic inference (from molecular sequences)</td>
      <td>Phylogenetic tree reconciliation, Protein sequence analysis, Genome comparison, Sequence clustering, Phylogenetic inference (from molecular sequences)</td>
      <td>Comparative genomics</td>
      <td>Comparative genomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fastoma</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fastoma</td>
      <td>fastoma</td>
      <td>FastOMA</td>
      <td>FastOMA is a scalable software package to infer orthology relationship.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>fastp</td>
      <td>fastp</td>
      <td>Fast all-in-one preprocessing for FASTQ files</td>
      <td>2018-03-07</td>
      <td>https://github.com/OpenGene/fastp</td>
      <td>1.0.1</td>
      <td>fastp</td>
      <td>1.0.1</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Sequencing quality control, Sequence contamination filtering</td>
      <td>Sequence contamination filtering</td>
      <td>Sequence analysis, Probes and primers</td>
      <td>Probes and primers</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fastp</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fastp</td>
      <td>fastp</td>
      <td>fastp</td>
      <td>A tool designed to provide fast all-in-one preprocessing for FastQ files. This tool is developed in C++ with multithreading supported to afford high performance.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>10510</td>
      <td>10845</td>
      <td>1229203</td>
      <td>1372687</td>
      <td>13972</td>
      <td>14837</td>
      <td>495065</td>
      <td>537410</td>
      <td>3030</td>
      <td>3115</td>
      <td>99990</td>
      <td>109532</td>
      <td>303</td>
      <td>306</td>
      <td>30703</td>
      <td>30747</td>
      <td>2050376</td>
      <td>1854961</td>
      <td>29103</td>
      <td>27815</td>
      <td>https://usegalaxy.eu/published/workflow?id=007ce6405d196df3, https://usegalaxy.eu/published/workflow?id=0189bab68f828d86, https://usegalaxy.eu/published/workflow?id=03d1d2e0dc6e2024, https://usegalaxy.eu/published/workflow?id=04ed3f21de17aa1d, https://usegalaxy.eu/published/workflow?id=0590554db9794d13, https://usegalaxy.eu/published/workflow?id=0599f049e4697839, https://usegalaxy.eu/published/workflow?id=0609a4b1941d1a68, https://usegalaxy.eu/published/workflow?id=07b74e4cf53cb633, https://usegalaxy.eu/published/workflow?id=0882300f7b399587, https://usegalaxy.eu/published/workflow?id=08eb5e55109c4d9c, https://usegalaxy.eu/published/workflow?id=0a82eb8b44d9d502, https://usegalaxy.eu/published/workflow?id=0ab3e47ebd3b520e, https://usegalaxy.eu/published/workflow?id=0abd32dbc5207b88, https://usegalaxy.eu/published/workflow?id=0be293debca8fb9b, https://usegalaxy.eu/published/workflow?id=0c47571d20535426, https://usegalaxy.eu/published/workflow?id=0e5b701ed5ca69a7, https://usegalaxy.eu/published/workflow?id=0f6a7bd5362e4a2b, https://usegalaxy.eu/published/workflow?id=112422839b68db02, https://usegalaxy.eu/published/workflow?id=135e8888872e8413, https://usegalaxy.eu/published/workflow?id=1761092d0e7deece, https://usegalaxy.eu/published/workflow?id=181e751be8dd9abc, https://usegalaxy.eu/published/workflow?id=1842a61fe5f6965f, https://usegalaxy.eu/published/workflow?id=1860e02bcb894a64, https://usegalaxy.eu/published/workflow?id=18c9095fbf5c4517, https://usegalaxy.eu/published/workflow?id=1b43449ed8481892, https://usegalaxy.eu/published/workflow?id=1bc7b5eabea972fd, https://usegalaxy.eu/published/workflow?id=1c5588d6ac7f773f, https://usegalaxy.eu/published/workflow?id=1ccc0c9e1c6821bb, https://usegalaxy.eu/published/workflow?id=1d1ebdeddf23bdb9, https://usegalaxy.eu/published/workflow?id=1f874262c368fe7d, https://usegalaxy.eu/published/workflow?id=1f9a9a7998ef5ab3, https://usegalaxy.eu/published/workflow?id=212c213f1d8e0154, https://usegalaxy.eu/published/workflow?id=24eaa682897379c8, https://usegalaxy.eu/published/workflow?id=2573209308aaad23, https://usegalaxy.eu/published/workflow?id=25850714d13f0b0f, https://usegalaxy.eu/published/workflow?id=2a39dbc8fc6dd5d3, https://usegalaxy.eu/published/workflow?id=2b25eaebd3987612, https://usegalaxy.eu/published/workflow?id=2c79eee5ca5a3b09, https://usegalaxy.eu/published/workflow?id=2d08a73dd8ff99e9, https://usegalaxy.eu/published/workflow?id=2d64d6571e01809d, https://usegalaxy.eu/published/workflow?id=2d9d4445a1871230, https://usegalaxy.eu/published/workflow?id=2fa2f67603772413, https://usegalaxy.eu/published/workflow?id=309fc4fc2e0da89c, https://usegalaxy.eu/published/workflow?id=318be2b30bf7884c, https://usegalaxy.eu/published/workflow?id=344cd3a3c2ce5302, https://usegalaxy.eu/published/workflow?id=374797e58377c40f, https://usegalaxy.eu/published/workflow?id=37da7b42cb063a3a, https://usegalaxy.eu/published/workflow?id=3949d1d467224b0b, https://usegalaxy.eu/published/workflow?id=39876e9c04aad543, https://usegalaxy.eu/published/workflow?id=3aeb0e55d82992d2, https://usegalaxy.eu/published/workflow?id=3d5b39e454df584b, https://usegalaxy.eu/published/workflow?id=3f046b416b10b871, https://usegalaxy.eu/published/workflow?id=3fb550d7c2bc390a, https://usegalaxy.eu/published/workflow?id=40d2c7ac42bd5302, https://usegalaxy.eu/published/workflow?id=429ae28a178cd07c, https://usegalaxy.eu/published/workflow?id=45fc2dada84d251e, https://usegalaxy.eu/published/workflow?id=461d71caea871db8, https://usegalaxy.eu/published/workflow?id=4643edf74f3eaec3, https://usegalaxy.eu/published/workflow?id=476db04368d5e90c, https://usegalaxy.eu/published/workflow?id=47a0407867a90f95, https://usegalaxy.eu/published/workflow?id=47e14222073e7192, https://usegalaxy.eu/published/workflow?id=487929c7b5be8dc3, https://usegalaxy.eu/published/workflow?id=49ce9e738fadef4a, https://usegalaxy.eu/published/workflow?id=4a39d38fde81e35d, https://usegalaxy.eu/published/workflow?id=4a764275d4bf9f31, https://usegalaxy.eu/published/workflow?id=4b4bfc79205d83f1, https://usegalaxy.eu/published/workflow?id=4c75ab0be3c56494, https://usegalaxy.eu/published/workflow?id=4d97068a725c975c, https://usegalaxy.eu/published/workflow?id=4dd7f6e1c1e7f96a, https://usegalaxy.eu/published/workflow?id=50758631a0e81a7a, https://usegalaxy.eu/published/workflow?id=529f30e894beacc2, https://usegalaxy.eu/published/workflow?id=52bdf400f97c5770, https://usegalaxy.eu/published/workflow?id=52cafb62f5bf6e25, https://usegalaxy.eu/published/workflow?id=534b81d48df6b507, https://usegalaxy.eu/published/workflow?id=53de0e742573e588, https://usegalaxy.eu/published/workflow?id=56868483be18f1a1, https://usegalaxy.eu/published/workflow?id=57769c948e14f9ac, https://usegalaxy.eu/published/workflow?id=579dd1e95e76402f, https://usegalaxy.eu/published/workflow?id=57d3109f32defa3b, https://usegalaxy.eu/published/workflow?id=5b00ef046439d757, https://usegalaxy.eu/published/workflow?id=5b940b283d91d99d, https://usegalaxy.eu/published/workflow?id=5be158931abca947, https://usegalaxy.eu/published/workflow?id=5c15f251642bfa1f, https://usegalaxy.eu/published/workflow?id=5ec0d6d108e02e1a, https://usegalaxy.eu/published/workflow?id=606a04c4aa95d010, https://usegalaxy.eu/published/workflow?id=60e4d0db3164f037, https://usegalaxy.eu/published/workflow?id=614807d57979554c, https://usegalaxy.eu/published/workflow?id=61e592bafd9d2b58, https://usegalaxy.eu/published/workflow?id=625aafb9cf16b03f, https://usegalaxy.eu/published/workflow?id=628062a8cf93261f, https://usegalaxy.eu/published/workflow?id=629e57958afed44c, https://usegalaxy.eu/published/workflow?id=63eed9c2e025a723, https://usegalaxy.eu/published/workflow?id=69be516a48761365, https://usegalaxy.eu/published/workflow?id=6be946c7ec91835e, https://usegalaxy.eu/published/workflow?id=6dbac47941a804da, https://usegalaxy.eu/published/workflow?id=6e1fe7372e93fae0, https://usegalaxy.eu/published/workflow?id=71a9224fa6f3da9d, https://usegalaxy.eu/published/workflow?id=72e3e5fdc766e24a, https://usegalaxy.eu/published/workflow?id=74f19aff1b9cdd6b, https://usegalaxy.eu/published/workflow?id=762c64a9f588a420, https://usegalaxy.eu/published/workflow?id=766ebba7a2aee494, https://usegalaxy.eu/published/workflow?id=79fa095c9fce9923, https://usegalaxy.eu/published/workflow?id=7b7ef222ac4f9887, https://usegalaxy.eu/published/workflow?id=7baae0775071c7ec, https://usegalaxy.eu/published/workflow?id=7bdd9ad918edab0b, https://usegalaxy.eu/published/workflow?id=7c986f4d14c6a539, https://usegalaxy.eu/published/workflow?id=7c9b182da21748c7, https://usegalaxy.eu/published/workflow?id=81122d30c64a52a6, https://usegalaxy.eu/published/workflow?id=87fea062a9646a31, https://usegalaxy.eu/published/workflow?id=884a4415b669690c, https://usegalaxy.eu/published/workflow?id=889f2782ad82bd69, https://usegalaxy.eu/published/workflow?id=88b382a6f56a6329, https://usegalaxy.eu/published/workflow?id=8ee3b635c1c380cd, https://usegalaxy.eu/published/workflow?id=8f415b0c409f56f7, https://usegalaxy.eu/published/workflow?id=910570a2caafe462, https://usegalaxy.eu/published/workflow?id=9141c6449595c6a1, https://usegalaxy.eu/published/workflow?id=9219557de92e0203, https://usegalaxy.eu/published/workflow?id=93d1035a307fe63d, https://usegalaxy.eu/published/workflow?id=94c3d329c31c712f, https://usegalaxy.eu/published/workflow?id=96c61a584cb2e5e9, https://usegalaxy.eu/published/workflow?id=987a2abb5cd56e57, https://usegalaxy.eu/published/workflow?id=98bfad0c684011e9, https://usegalaxy.eu/published/workflow?id=98d4adf5774e274e, https://usegalaxy.eu/published/workflow?id=9a60ecf98d4f8134, https://usegalaxy.eu/published/workflow?id=9b025479fade745d, https://usegalaxy.eu/published/workflow?id=9eda857e4141d56e, https://usegalaxy.eu/published/workflow?id=9fddc44294236e8f, https://usegalaxy.eu/published/workflow?id=9feebfc6c34da287, https://usegalaxy.eu/published/workflow?id=a15dabcadb63b46d, https://usegalaxy.eu/published/workflow?id=a2d4221ad117c378, https://usegalaxy.eu/published/workflow?id=a31100610fba17a1, https://usegalaxy.eu/published/workflow?id=a4e3667f6879bfaf, https://usegalaxy.eu/published/workflow?id=a6c666f3028ea440, https://usegalaxy.eu/published/workflow?id=a6f4371ef9bc1aab, https://usegalaxy.eu/published/workflow?id=a705370bc2c13d5c, https://usegalaxy.eu/published/workflow?id=a72d2667a53830f7, https://usegalaxy.eu/published/workflow?id=a9e1a6b90dbd2518, https://usegalaxy.eu/published/workflow?id=aafcfcb585c5ca75, https://usegalaxy.eu/published/workflow?id=ab8924d78fa5ef8f, https://usegalaxy.eu/published/workflow?id=ada52ba09b462567, https://usegalaxy.eu/published/workflow?id=ae1ca223d6667070, https://usegalaxy.eu/published/workflow?id=aff44f1665a14e23, https://usegalaxy.eu/published/workflow?id=b02d90c32dc67730, https://usegalaxy.eu/published/workflow?id=b144287d56b3291b, https://usegalaxy.eu/published/workflow?id=b14845359b702444, https://usegalaxy.eu/published/workflow?id=b37800d990cceb83, https://usegalaxy.eu/published/workflow?id=b41ff6d8fb5a0878, https://usegalaxy.eu/published/workflow?id=b426e137396acb14, https://usegalaxy.eu/published/workflow?id=b5ffba506152b1a1, https://usegalaxy.eu/published/workflow?id=b6eb5fe7e37f680a, https://usegalaxy.eu/published/workflow?id=b77e3b08092fa73e, https://usegalaxy.eu/published/workflow?id=b7a3cbcd32734e8b, https://usegalaxy.eu/published/workflow?id=b801e745205892f5, https://usegalaxy.eu/published/workflow?id=b97532c398b821a2, https://usegalaxy.eu/published/workflow?id=ba01694e45595b6b, https://usegalaxy.eu/published/workflow?id=bb63b5bb983fd49c, https://usegalaxy.eu/published/workflow?id=bcb8a8c5bff790ab, https://usegalaxy.eu/published/workflow?id=bfb762c96328e3c1, https://usegalaxy.eu/published/workflow?id=c084c2009f6b2d0a, https://usegalaxy.eu/published/workflow?id=c14d2e8b3c191bc4, https://usegalaxy.eu/published/workflow?id=c2af7a5dbfb5b023, https://usegalaxy.eu/published/workflow?id=c2f0decb06f0bb90, https://usegalaxy.eu/published/workflow?id=c38e3ac59d8b86d6, https://usegalaxy.eu/published/workflow?id=c789ab71876c6ead, https://usegalaxy.eu/published/workflow?id=c885eaabae612d6d, https://usegalaxy.eu/published/workflow?id=c930a54a30dcc7e3, https://usegalaxy.eu/published/workflow?id=cbc16cb5db98ab21, https://usegalaxy.eu/published/workflow?id=cd25f70f71683ab6, https://usegalaxy.eu/published/workflow?id=cd926dd6b70e72d6, https://usegalaxy.eu/published/workflow?id=d2f75f70c33394ee, https://usegalaxy.eu/published/workflow?id=d503dee6f9b35a18, https://usegalaxy.eu/published/workflow?id=d5333903ba947adf, https://usegalaxy.eu/published/workflow?id=d88145e7475e0b7b, https://usegalaxy.eu/published/workflow?id=d967194ca51ff11e, https://usegalaxy.eu/published/workflow?id=daf47855afa73de8, https://usegalaxy.eu/published/workflow?id=dd3193aaaf08e798, https://usegalaxy.eu/published/workflow?id=deec04097a871646, https://usegalaxy.eu/published/workflow?id=e16e524c557d7928, https://usegalaxy.eu/published/workflow?id=e62b659a5da9734e, https://usegalaxy.eu/published/workflow?id=eaf01ccb13aad3ec, https://usegalaxy.eu/published/workflow?id=ebb131f2a6172081, https://usegalaxy.eu/published/workflow?id=ecab5b2c3ff3df9d, https://usegalaxy.eu/published/workflow?id=ecf48fa50af560a0, https://usegalaxy.eu/published/workflow?id=ed20bd120064bab0, https://usegalaxy.eu/published/workflow?id=ee6dc47660540ba6, https://usegalaxy.eu/published/workflow?id=f0ac222d3e965ab0, https://usegalaxy.eu/published/workflow?id=f12fb4a95954c448, https://usegalaxy.eu/published/workflow?id=f54b4d76f7f9a589, https://usegalaxy.eu/published/workflow?id=f97e6fe1fcbc53a7, https://usegalaxy.eu/published/workflow?id=fb18c5d14acb1b36, https://usegalaxy.eu/published/workflow?id=fc216dfb0ec9ed41, https://usegalaxy.eu/published/workflow?id=fcf745185d6e0ea4, https://usegalaxy.eu/published/workflow?id=fde21b5f5c169114, https://usegalaxy.eu/published/workflow?id=fe8a1a970fd0d4cd, https://usegalaxy.eu/published/workflow?id=ff4d5beb6ce1809d, https://usegalaxy.fr/published/workflow?id=2345fad7dac13daa, https://usegalaxy.fr/published/workflow?id=54b48eeeed3564dd, https://usegalaxy.fr/published/workflow?id=57b5a45c9f71a09c, https://usegalaxy.fr/published/workflow?id=591c57173e259ca6, https://usegalaxy.fr/published/workflow?id=61235264d5d2f666, https://usegalaxy.fr/published/workflow?id=727cfdb7c79287d1, https://usegalaxy.fr/published/workflow?id=776ef2fa51094ba6, https://usegalaxy.fr/published/workflow?id=7d27dd394b921846, https://usegalaxy.fr/published/workflow?id=98529cff0ef36b91, https://usegalaxy.fr/published/workflow?id=bffee40c87c7b20a, https://usegalaxy.fr/published/workflow?id=ca0bd0e42cb774f9, https://usegalaxy.fr/published/workflow?id=d0d12c0b80c286f9, https://usegalaxy.fr/published/workflow?id=e0b225d4b2556365, https://usegalaxy.fr/published/workflow?id=e3c04a30258421aa, https://usegalaxy.org.au/published/workflow?id=05f3abecd4ddd4a1, https://usegalaxy.org.au/published/workflow?id=06ba34da322f46f1, https://usegalaxy.org.au/published/workflow?id=094a545d44ce28cc, https://usegalaxy.org.au/published/workflow?id=0c6f7bfc826433c4, https://usegalaxy.org.au/published/workflow?id=13ce18408ee7acc9, https://usegalaxy.org.au/published/workflow?id=17303f96fb40897c, https://usegalaxy.org.au/published/workflow?id=1dd4392c51766383, https://usegalaxy.org.au/published/workflow?id=1e033cad62e1bd1e, https://usegalaxy.org.au/published/workflow?id=203f9e69152fbb2a, https://usegalaxy.org.au/published/workflow?id=25403321a649a92b, https://usegalaxy.org.au/published/workflow?id=25d52afddaa3451b, https://usegalaxy.org.au/published/workflow?id=27205feb28b0ce06, https://usegalaxy.org.au/published/workflow?id=2b2ce7e1206d177d, https://usegalaxy.org.au/published/workflow?id=30a601039d24e2eb, https://usegalaxy.org.au/published/workflow?id=31dbd313e5c8160b, https://usegalaxy.org.au/published/workflow?id=3223ceaa2b58d940, https://usegalaxy.org.au/published/workflow?id=39b464084d930a33, https://usegalaxy.org.au/published/workflow?id=3c1b1f210c7c292b, https://usegalaxy.org.au/published/workflow?id=3c657dd766d02f6a, https://usegalaxy.org.au/published/workflow?id=3c7b55daae7c119b, https://usegalaxy.org.au/published/workflow?id=3f39cf407c185724, https://usegalaxy.org.au/published/workflow?id=41bd0340dcde60ec, https://usegalaxy.org.au/published/workflow?id=47ce6a2d30887637, https://usegalaxy.org.au/published/workflow?id=48a60965b6b3ff54, https://usegalaxy.org.au/published/workflow?id=49b5c35b9723fa62, https://usegalaxy.org.au/published/workflow?id=4bc8ac41ecff7daf, https://usegalaxy.org.au/published/workflow?id=4c7e0703aace6aa3, https://usegalaxy.org.au/published/workflow?id=51b745f8c964e5c6, https://usegalaxy.org.au/published/workflow?id=557601292ea0f6f3, https://usegalaxy.org.au/published/workflow?id=69d91340fc7effa2, https://usegalaxy.org.au/published/workflow?id=6a04c54236f30e8d, https://usegalaxy.org.au/published/workflow?id=6b1b0a945d367b4c, https://usegalaxy.org.au/published/workflow?id=6e1e24f9e0326930, https://usegalaxy.org.au/published/workflow?id=749d8f2d26e0a111, https://usegalaxy.org.au/published/workflow?id=7563b3ef697a69fd, https://usegalaxy.org.au/published/workflow?id=76bb6bde19975e05, https://usegalaxy.org.au/published/workflow?id=77951619443e6f87, https://usegalaxy.org.au/published/workflow?id=7a0c4804e4d4c905, https://usegalaxy.org.au/published/workflow?id=7b203d52f496e2c7, https://usegalaxy.org.au/published/workflow?id=7c22d726e1716f4b, https://usegalaxy.org.au/published/workflow?id=7fd58f5fc93f414e, https://usegalaxy.org.au/published/workflow?id=810242a3dc4f6170, https://usegalaxy.org.au/published/workflow?id=811bbada546850bd, https://usegalaxy.org.au/published/workflow?id=813a4938397cf6ee, https://usegalaxy.org.au/published/workflow?id=84c98a23187d2a36, https://usegalaxy.org.au/published/workflow?id=888eb87b62c4e6dd, https://usegalaxy.org.au/published/workflow?id=888f8377eb6eabf2, https://usegalaxy.org.au/published/workflow?id=8b955274ca3ca3dc, https://usegalaxy.org.au/published/workflow?id=92fcdd490831c6c5, https://usegalaxy.org.au/published/workflow?id=932bad88f8dc8d33, https://usegalaxy.org.au/published/workflow?id=94c7fdcbbc7b428a, https://usegalaxy.org.au/published/workflow?id=9c60b83feaa22415, https://usegalaxy.org.au/published/workflow?id=a29670ff0ecb1a38, https://usegalaxy.org.au/published/workflow?id=ae894cff1d1aba01, https://usegalaxy.org.au/published/workflow?id=af12a3ec85859563, https://usegalaxy.org.au/published/workflow?id=b03b577156e4db5b, https://usegalaxy.org.au/published/workflow?id=b432a1312859edd2, https://usegalaxy.org.au/published/workflow?id=b517cabb697491a0, https://usegalaxy.org.au/published/workflow?id=b59fada651043735, https://usegalaxy.org.au/published/workflow?id=b7aa22291322e505, https://usegalaxy.org.au/published/workflow?id=ba30cc4a84c449f9, https://usegalaxy.org.au/published/workflow?id=ba7a9d4c857d26b7, https://usegalaxy.org.au/published/workflow?id=baa763dbdf6afb71, https://usegalaxy.org.au/published/workflow?id=bcf230e526632064, https://usegalaxy.org.au/published/workflow?id=bd1cf22d47389742, https://usegalaxy.org.au/published/workflow?id=be9213411c520096, https://usegalaxy.org.au/published/workflow?id=bf80ef6e5f37deb7, https://usegalaxy.org.au/published/workflow?id=c1ce3f2ae7c81c3e, https://usegalaxy.org.au/published/workflow?id=c38b998d3e90c329, https://usegalaxy.org.au/published/workflow?id=c5f000229dd93599, https://usegalaxy.org.au/published/workflow?id=c8d18e2c03dca796, https://usegalaxy.org.au/published/workflow?id=c9156ca3df2b595c, https://usegalaxy.org.au/published/workflow?id=ca7fc6e2adbf1f26, https://usegalaxy.org.au/published/workflow?id=ccf936c5b2f2c2f3, https://usegalaxy.org.au/published/workflow?id=ccfae75390038ced, https://usegalaxy.org.au/published/workflow?id=d3521c05077192df, https://usegalaxy.org.au/published/workflow?id=d35d05be576e8b13, https://usegalaxy.org.au/published/workflow?id=d78f885b4b86635f, https://usegalaxy.org.au/published/workflow?id=d7c329b354f4426d, https://usegalaxy.org.au/published/workflow?id=dccc8fb49d61ac33, https://usegalaxy.org.au/published/workflow?id=e439a0e17a90d24c, https://usegalaxy.org.au/published/workflow?id=f2541030f6d70b59, https://usegalaxy.org.au/published/workflow?id=fc3c83b11f527977, https://usegalaxy.org.au/published/workflow?id=fca05a51c9c91df4, https://usegalaxy.org.au/published/workflow?id=fe4e34a407157389, https://usegalaxy.org.au/published/workflow?id=fe5542c99cdc57ce, https://usegalaxy.org.au/published/workflow?id=fedf2874752ac1f6, https://usegalaxy.org/published/workflow?id=00596e862edb23f2, https://usegalaxy.org/published/workflow?id=0e5a3899ae97076b, https://usegalaxy.org/published/workflow?id=0f1659b3290c3492, https://usegalaxy.org/published/workflow?id=0f8b4fd0bf276968, https://usegalaxy.org/published/workflow?id=0fa2ce12c2874b1d, https://usegalaxy.org/published/workflow?id=0fd63eaea85d7002, https://usegalaxy.org/published/workflow?id=109092d8dbda4398, https://usegalaxy.org/published/workflow?id=10c2136973c88d8a, https://usegalaxy.org/published/workflow?id=1518c8c50747a1c1, https://usegalaxy.org/published/workflow?id=15a0e22570f71098, https://usegalaxy.org/published/workflow?id=17d9b233640aefec, https://usegalaxy.org/published/workflow?id=195f7acb16c5a2e2, https://usegalaxy.org/published/workflow?id=1be5c982c2d36a6c, https://usegalaxy.org/published/workflow?id=1c7af480caf150bd, https://usegalaxy.org/published/workflow?id=1ddabc7d272dc702, https://usegalaxy.org/published/workflow?id=1f48b44fa22ac5fc, https://usegalaxy.org/published/workflow?id=2060fa13feff512b, https://usegalaxy.org/published/workflow?id=20b7ae4b517a80fd, https://usegalaxy.org/published/workflow?id=21161075002d6778, https://usegalaxy.org/published/workflow?id=2125db88b4a71c09, https://usegalaxy.org/published/workflow?id=24663ad42c78106c, https://usegalaxy.org/published/workflow?id=26b35874b9e32355, https://usegalaxy.org/published/workflow?id=2a782ad24271f570, https://usegalaxy.org/published/workflow?id=307d19833ffb5395, https://usegalaxy.org/published/workflow?id=323c651ca583c9e5, https://usegalaxy.org/published/workflow?id=33ae58f05e5e1d60, https://usegalaxy.org/published/workflow?id=34ab5572a9547bfe, https://usegalaxy.org/published/workflow?id=35c2626849348f2c, https://usegalaxy.org/published/workflow?id=3ad9cdcda44b7dfb, https://usegalaxy.org/published/workflow?id=3cf46c167ccb374f, https://usegalaxy.org/published/workflow?id=3f92474cd16ca314, https://usegalaxy.org/published/workflow?id=426e3951dac0704e, https://usegalaxy.org/published/workflow?id=42bc3a770f0ab6b8, https://usegalaxy.org/published/workflow?id=4357180d40560019, https://usegalaxy.org/published/workflow?id=4458439388155d92, https://usegalaxy.org/published/workflow?id=4b41a8247f52c463, https://usegalaxy.org/published/workflow?id=4cf8cc42e97b2da6, https://usegalaxy.org/published/workflow?id=4de095b92ed661ac, https://usegalaxy.org/published/workflow?id=4f698eaeef30e364, https://usegalaxy.org/published/workflow?id=5336b9d26a083983, https://usegalaxy.org/published/workflow?id=53dda64a5a6ebd54, https://usegalaxy.org/published/workflow?id=53fea86eb74c224b, https://usegalaxy.org/published/workflow?id=5415e85a86898e41, https://usegalaxy.org/published/workflow?id=562bcaa9b4d2e8a4, https://usegalaxy.org/published/workflow?id=574e42683dc3961b, https://usegalaxy.org/published/workflow?id=57f4c6f5206edfcb, https://usegalaxy.org/published/workflow?id=58a7a02dade085cf, https://usegalaxy.org/published/workflow?id=5a610d5a42d50cf3, https://usegalaxy.org/published/workflow?id=5b3e2ddf525da25d, https://usegalaxy.org/published/workflow?id=5bcd1aade71a8035, https://usegalaxy.org/published/workflow?id=60f0d0d5d6e93941, https://usegalaxy.org/published/workflow?id=612452c925d723b7, https://usegalaxy.org/published/workflow?id=623fc80c7f91a33a, https://usegalaxy.org/published/workflow?id=668df61f85b5b166, https://usegalaxy.org/published/workflow?id=6961a64d6f6b782d, https://usegalaxy.org/published/workflow?id=69d8863d40524120, https://usegalaxy.org/published/workflow?id=6b11d738ee57f245, https://usegalaxy.org/published/workflow?id=6f85b9fc84e3bf0c, https://usegalaxy.org/published/workflow?id=786aa9e2bd723629, https://usegalaxy.org/published/workflow?id=7a26069c2939bef9, https://usegalaxy.org/published/workflow?id=7d6617b8316cee19, https://usegalaxy.org/published/workflow?id=7dd6e4150bb81ad1, https://usegalaxy.org/published/workflow?id=7dde0721a382ea3e, https://usegalaxy.org/published/workflow?id=7e2651e5ed17e3db, https://usegalaxy.org/published/workflow?id=80ac6adead9f97d8, https://usegalaxy.org/published/workflow?id=88d0b64011c3148c, https://usegalaxy.org/published/workflow?id=8b81b9a3b49bbf98, https://usegalaxy.org/published/workflow?id=8bcab83874cbc75c, https://usegalaxy.org/published/workflow?id=8e1c6ad6ff623015, https://usegalaxy.org/published/workflow?id=92875b13082b5568, https://usegalaxy.org/published/workflow?id=953d9cbe05e632a8, https://usegalaxy.org/published/workflow?id=97263f3308077ff0, https://usegalaxy.org/published/workflow?id=98286b6154e1b57f, https://usegalaxy.org/published/workflow?id=98b2ac8fc75781f8, https://usegalaxy.org/published/workflow?id=9adf8691809326e5, https://usegalaxy.org/published/workflow?id=9f68718f025bed30, https://usegalaxy.org/published/workflow?id=9fc9f1db50019929, https://usegalaxy.org/published/workflow?id=a71fcbc60caae8b6, https://usegalaxy.org/published/workflow?id=a743746151101352, https://usegalaxy.org/published/workflow?id=a8b9252fad5fe06a, https://usegalaxy.org/published/workflow?id=aadccbe72752c129, https://usegalaxy.org/published/workflow?id=b26501a4a85bfdeb, https://usegalaxy.org/published/workflow?id=b300d028da9b8396, https://usegalaxy.org/published/workflow?id=b34ba9b2e4d8f050, https://usegalaxy.org/published/workflow?id=b6a795834009ce06, https://usegalaxy.org/published/workflow?id=b7817a12dbee583c, https://usegalaxy.org/published/workflow?id=badd4b56dbdeaba7, https://usegalaxy.org/published/workflow?id=bbe6f5a45ea5ee5e, https://usegalaxy.org/published/workflow?id=bd355e8ce58a139e, https://usegalaxy.org/published/workflow?id=bdd7011a908bc1aa, https://usegalaxy.org/published/workflow?id=bf3ccae8d5b6ef58, https://usegalaxy.org/published/workflow?id=c092a3631d68ce38, https://usegalaxy.org/published/workflow?id=c12e6b7896f8e114, https://usegalaxy.org/published/workflow?id=c7b2d2ccc4b0dec5, https://usegalaxy.org/published/workflow?id=cdea4f88ccbd155f, https://usegalaxy.org/published/workflow?id=cdf5dffbed938d49, https://usegalaxy.org/published/workflow?id=d3428dd172a78102, https://usegalaxy.org/published/workflow?id=d4ea6cdd40522eb1, https://usegalaxy.org/published/workflow?id=d4f0a7ccb09f1e55, https://usegalaxy.org/published/workflow?id=d5436cee11a34de9, https://usegalaxy.org/published/workflow?id=d817e98cef94bc96, https://usegalaxy.org/published/workflow?id=d96bd5c9fb553f2c, https://usegalaxy.org/published/workflow?id=db01ac5b0dc50cd4, https://usegalaxy.org/published/workflow?id=de9bf4c1024f8d53, https://usegalaxy.org/published/workflow?id=dfa8c64634aec75c, https://usegalaxy.org/published/workflow?id=dfd73857c0eb2a21, https://usegalaxy.org/published/workflow?id=e12324f5e3b930fd, https://usegalaxy.org/published/workflow?id=e282f505ba9f729f, https://usegalaxy.org/published/workflow?id=e2ab3d3219a648a7, https://usegalaxy.org/published/workflow?id=e55ca98817c1855e, https://usegalaxy.org/published/workflow?id=e67b6da41e1e6932, https://usegalaxy.org/published/workflow?id=e7ce46762faccaac, https://usegalaxy.org/published/workflow?id=eb3409cc73e392a8, https://usegalaxy.org/published/workflow?id=eb4f7dbca0114270, https://usegalaxy.org/published/workflow?id=f378960ee55f567e, https://usegalaxy.org/published/workflow?id=ffae9c47980a5170, https://workflowhub.eu/workflows/100?version=1, https://workflowhub.eu/workflows/1035?version=1, https://workflowhub.eu/workflows/103?version=1, https://workflowhub.eu/workflows/1052?version=11, https://workflowhub.eu/workflows/1061?version=1, https://workflowhub.eu/workflows/1103?version=3, https://workflowhub.eu/workflows/1104?version=2, https://workflowhub.eu/workflows/110?version=11, https://workflowhub.eu/workflows/111?version=5, https://workflowhub.eu/workflows/112?version=6, https://workflowhub.eu/workflows/113?version=4, https://workflowhub.eu/workflows/1190?version=1, https://workflowhub.eu/workflows/1260?version=2, https://workflowhub.eu/workflows/1272?version=2, https://workflowhub.eu/workflows/1439?version=1, https://workflowhub.eu/workflows/1473?version=1, https://workflowhub.eu/workflows/1492?version=2, https://workflowhub.eu/workflows/1540?version=1, https://workflowhub.eu/workflows/155?version=6, https://workflowhub.eu/workflows/1589?version=1, https://workflowhub.eu/workflows/1602?version=2, https://workflowhub.eu/workflows/1612?version=1, https://workflowhub.eu/workflows/1613?version=2, https://workflowhub.eu/workflows/1616?version=1, https://workflowhub.eu/workflows/1642?version=1, https://workflowhub.eu/workflows/1644?version=1, https://workflowhub.eu/workflows/1647?version=1, https://workflowhub.eu/workflows/1688?version=1, https://workflowhub.eu/workflows/1723?version=1, https://workflowhub.eu/workflows/1738?version=1, https://workflowhub.eu/workflows/1739?version=1, https://workflowhub.eu/workflows/1740?version=1, https://workflowhub.eu/workflows/1741?version=1, https://workflowhub.eu/workflows/1854?version=1, https://workflowhub.eu/workflows/1876?version=2, https://workflowhub.eu/workflows/1881?version=2, https://workflowhub.eu/workflows/2025?version=2, https://workflowhub.eu/workflows/2043?version=1, https://workflowhub.eu/workflows/224?version=1, https://workflowhub.eu/workflows/346?version=1, https://workflowhub.eu/workflows/353?version=1, https://workflowhub.eu/workflows/357?version=1, https://workflowhub.eu/workflows/35?version=1, https://workflowhub.eu/workflows/36?version=1, https://workflowhub.eu/workflows/37?version=1, https://workflowhub.eu/workflows/38?version=1, https://workflowhub.eu/workflows/397?version=16, https://workflowhub.eu/workflows/398?version=17, https://workflowhub.eu/workflows/39?version=1, https://workflowhub.eu/workflows/400?version=12, https://workflowhub.eu/workflows/401?version=13, https://workflowhub.eu/workflows/40?version=1, https://workflowhub.eu/workflows/439?version=4, https://workflowhub.eu/workflows/517?version=1, https://workflowhub.eu/workflows/518?version=1, https://workflowhub.eu/workflows/519?version=1, https://workflowhub.eu/workflows/601?version=2, https://workflowhub.eu/workflows/68?version=1</td>
      <td>assembly/assembly-with-preprocessing, assembly/largegenome, assembly/mrsa-illumina, assembly/mrsa-nanopore, ecology/eDNA-taxonomic-analysis, galaxy-interface/workflow-automation, introduction/galaxy-intro-ngs-data-managment, introduction/vsi_qc, microbiome/beer-data-analysis, microbiome/mgnify-amplicon, microbiome/nanopore-16S-metagenomics, microbiome/pathogen-detection-from-nanopore-foodborne-data, sequence-analysis/quality-contamination-control, transcriptomics/differential-isoform-expression, variant-analysis/aiv-analysis, variant-analysis/pox-tiled-amplicon, variant-analysis/sars-cov-2, variant-analysis/tb-variant-analysis</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>fastplong</td>
      <td>fastplong</td>
      <td>Fastplong is used for ultrafast preprocessing and quality control for long reads (Nanopore, PacBio, Cyclone, etc.).</td>
      <td>2025-05-06</td>
      <td>https://github.com/OpenGene/fastplong</td>
      <td>0.4.1</td>
      <td>fastplong</td>
      <td>0.4.1</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis, Assembly</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fastplong</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fastplong</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>40</td>
      <td>40</td>
      <td>309</td>
      <td>309</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>217</td>
      <td>217</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>526</td>
      <td>526</td>
      <td>48</td>
      <td>48</td>
      <td>https://usegalaxy.org.au/published/workflow?id=aa1d123c0a0929f3</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>fastqe</td>
      <td>fastqe</td>
      <td>FASTQE</td>
      <td>2020-07-13</td>
      <td>https://fastqe.com/</td>
      <td>0.3.1+galaxy0</td>
      <td>fastqe</td>
      <td>0.5.2</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Sequencing quality control</td>
      <td>Sequencing quality control</td>
      <td>Sequence analysis, Sequencing</td>
      <td>Sequence analysis, Sequencing</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fastqe</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fastqe</td>
      <td>fastqe</td>
      <td>FASTQE</td>
      <td>Compute quality stats for FASTQ files and print those stats as emoji... for some reason.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>3154</td>
      <td>3177</td>
      <td>6904</td>
      <td>6978</td>
      <td>6143</td>
      <td>6198</td>
      <td>13534</td>
      <td>13627</td>
      <td>1316</td>
      <td>1323</td>
      <td>3505</td>
      <td>3549</td>
      <td>291</td>
      <td>291</td>
      <td>510</td>
      <td>510</td>
      <td>24664</td>
      <td>24453</td>
      <td>10989</td>
      <td>10904</td>
      <td>https://usegalaxy.eu/published/workflow?id=055eae88be2d4dd7, https://usegalaxy.eu/published/workflow?id=2a274e0e4ba45c77, https://usegalaxy.eu/published/workflow?id=340aa3ae5468deb0, https://usegalaxy.eu/published/workflow?id=4c736e255f4a740f, https://usegalaxy.eu/published/workflow?id=519e0ffdfd1790bc, https://usegalaxy.eu/published/workflow?id=76a63b1710da5e60, https://usegalaxy.eu/published/workflow?id=a8d814a6d86e9661, https://usegalaxy.eu/published/workflow?id=c65d98740ca64104, https://usegalaxy.eu/published/workflow?id=d3ae8c51061330f3, https://usegalaxy.eu/published/workflow?id=d5780c37232e3181, https://usegalaxy.org.au/published/workflow?id=115151c2c8998d44, https://usegalaxy.org.au/published/workflow?id=178db14511c8f293, https://usegalaxy.org.au/published/workflow?id=1f63ec596e578b1c, https://usegalaxy.org.au/published/workflow?id=2afeae1ec03c0b57, https://usegalaxy.org.au/published/workflow?id=43f2dabbc1932e30, https://usegalaxy.org.au/published/workflow?id=47cacc06d404a8c6, https://usegalaxy.org.au/published/workflow?id=5d6773ed5b2f41cb, https://usegalaxy.org.au/published/workflow?id=8fe84de499e96485, https://usegalaxy.org.au/published/workflow?id=bd251174d94cde9d, https://usegalaxy.org.au/published/workflow?id=dd6f465ab1eb10b0, https://usegalaxy.org.au/published/workflow?id=e08b04291839e79a, https://usegalaxy.org/published/workflow?id=20da2dcce0ae2ca2, https://usegalaxy.org/published/workflow?id=7a33bc2d0970b502, https://usegalaxy.org/published/workflow?id=cf2dbc29ca20ebee, https://usegalaxy.org/published/workflow?id=fe262214fba1cb8e</td>
      <td>sequence-analysis/quality-control</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>fastreer</td>
      <td>fastreer_dist2tree, fastreer_fasta2dist, fastreer_vcf2dist, fastreer_vcf2tree</td>
      <td>fastreeR toolkit wrappers for phylogenetic trees and distance matrices from VCF and FASTA</td>
      <td>2025-11-01</td>
      <td>https://github.com/gkanogiannis/fastreeR</td>
      <td>2.1.2</td>
      <td>fastreer</td>
      <td>2.1.2</td>
      <td>Up-to-date</td>
      <td>Phylogenetics, Variant Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fastreer</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fastreer</td>
      <td>fastreer</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>fastspar</td>
      <td>fastspar, fastspar_pvalues, fastspar_reduce</td>
      <td>Tool for rapid and scalable correlation estimation for compositional data.</td>
      <td>2025-06-16</td>
      <td>https://github.com/scwatts/fastspar/</td>
      <td>1.0.0</td>
      <td>fastspar</td>
      <td>1.0.0</td>
      <td>Up-to-date</td>
      <td>Metagenomics, Statistics</td>
      <td></td>
      <td></td>
      <td>Metagenomics, Microbial ecology</td>
      <td>Metagenomics, Microbial ecology</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fastspar</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fastspar</td>
      <td>FastSpar</td>
      <td>FastSpar</td>
      <td>FastSpar is a C++ implementation of the SparCC algorithm for fast correlation estimation from compositional count data such as microbiome OTU tables.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>fasttree</td>
      <td>fasttree</td>
      <td>FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide or protein sequences - GVL</td>
      <td>2018-02-02</td>
      <td>http://www.microbesonline.org/fasttree/</td>
      <td>2.1.10</td>
      <td>fasttree</td>
      <td>2.2.0</td>
      <td>To update</td>
      <td>Phylogenetics</td>
      <td>Phylogenetic tree generation (from molecular sequences), Phylogenetic tree generation (maximum likelihood and Bayesian methods)</td>
      <td></td>
      <td>Phylogenetics, Sequence analysis</td>
      <td>Phylogenetics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fasttree</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fasttree</td>
      <td>fasttree</td>
      <td>FastTree</td>
      <td>Infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide or protein sequences.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1826</td>
      <td>1923</td>
      <td>84447</td>
      <td>85175</td>
      <td>2879</td>
      <td>3069</td>
      <td>29857</td>
      <td>32881</td>
      <td>829</td>
      <td>967</td>
      <td>8573</td>
      <td>9437</td>
      <td>82</td>
      <td>84</td>
      <td>411</td>
      <td>416</td>
      <td>127909</td>
      <td>123288</td>
      <td>6043</td>
      <td>5616</td>
      <td>https://usegalaxy.eu/published/workflow?id=36de05d50c703164, https://usegalaxy.eu/published/workflow?id=376119528377a3ae, https://usegalaxy.eu/published/workflow?id=9155ed0e7a6a7b60, https://usegalaxy.eu/published/workflow?id=974cc681d07ec788, https://usegalaxy.eu/published/workflow?id=b1a799497c366eef, https://usegalaxy.eu/published/workflow?id=c8bc4af6dd3b5d02, https://usegalaxy.eu/published/workflow?id=ce21961260fe4351, https://usegalaxy.eu/published/workflow?id=d07fe8b5a045c176, https://usegalaxy.eu/published/workflow?id=dc57118a95b10a80, https://usegalaxy.eu/published/workflow?id=e16e524c557d7928, https://usegalaxy.eu/published/workflow?id=e7f03b4bd9b3d6fd, https://usegalaxy.eu/published/workflow?id=f6a763951d815944, https://usegalaxy.fr/published/workflow?id=297b71156b297a73, https://usegalaxy.fr/published/workflow?id=c7be6e229a045fad, https://usegalaxy.org.au/published/workflow?id=42137e36791652bc, https://usegalaxy.org.au/published/workflow?id=a01c3004bcef9dd3, https://usegalaxy.org.au/published/workflow?id=b60922a253df6654, https://usegalaxy.org.au/published/workflow?id=d62b6717202f42c9, https://usegalaxy.org.au/published/workflow?id=eda40b58616a0fe4, https://usegalaxy.org/published/workflow?id=07f7198b63108131, https://usegalaxy.org/published/workflow?id=2d1ca299c1b058f6, https://usegalaxy.org/published/workflow?id=2d3063882d8239ff, https://usegalaxy.org/published/workflow?id=3d7baebefa7c9b81, https://usegalaxy.org/published/workflow?id=3ef08553982f6ffc, https://usegalaxy.org/published/workflow?id=80565301ada0d3eb, https://usegalaxy.org/published/workflow?id=87685c6247521b54, https://usegalaxy.org/published/workflow?id=a484a1e1a7335696, https://usegalaxy.org/published/workflow?id=b2fb90873471f4e7, https://usegalaxy.org/published/workflow?id=be54e57345ce7ca5, https://usegalaxy.org/published/workflow?id=be9bd7286d38c210, https://workflowhub.eu/workflows/1060?version=1, https://workflowhub.eu/workflows/1487?version=2, https://workflowhub.eu/workflows/1559?version=1</td>
      <td>evolution/abc_intro_phylo, evolution/bacterial-comparative-genomics, microbiome/pathogen-detection-from-nanopore-foodborne-data</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>featurecounts</td>
      <td>featurecounts</td>
      <td>featureCounts counts the number of reads aligned to defined masked regions in a reference genome</td>
      <td>2016-02-09</td>
      <td>https://subread.sourceforge.net/</td>
      <td>2.1.1</td>
      <td>subread</td>
      <td>2.1.1</td>
      <td>Up-to-date</td>
      <td>RNA, Transcriptomics, Sequence Analysis</td>
      <td>Read summarisation, RNA-Seq quantification</td>
      <td>Read summarisation, RNA-Seq quantification</td>
      <td>RNA-Seq</td>
      <td>RNA-Seq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/featurecounts</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/featurecounts</td>
      <td>featurecounts</td>
      <td>FeatureCounts</td>
      <td>featureCounts is a very efficient read quantifier. It can be used to summarize RNA-seq reads and gDNA-seq reads to a variety of genomic features such as genes, exons, promoters, gene bodies and genomic bins. It is included in the Bioconductor Rsubread package and also in the SourceForge Subread package.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>14615</td>
      <td>16504</td>
      <td>807686</td>
      <td>914668</td>
      <td>26510</td>
      <td>33788</td>
      <td>760980</td>
      <td>995817</td>
      <td>2872</td>
      <td>3227</td>
      <td>79521</td>
      <td>90365</td>
      <td>1581</td>
      <td>1583</td>
      <td>52228</td>
      <td>52279</td>
      <td>2053129</td>
      <td>1700415</td>
      <td>55102</td>
      <td>45578</td>
      <td>https://usegalaxy.eu/published/workflow?id=0229f3f3835c2d53, https://usegalaxy.eu/published/workflow?id=08cfb9b21c1ec808, https://usegalaxy.eu/published/workflow?id=0de150b19102cb7e, https://usegalaxy.eu/published/workflow?id=135e8888872e8413, https://usegalaxy.eu/published/workflow?id=1502fd3fa90d32d8, https://usegalaxy.eu/published/workflow?id=17ad465d9515cd25, https://usegalaxy.eu/published/workflow?id=1842a61fe5f6965f, https://usegalaxy.eu/published/workflow?id=18c9095fbf5c4517, https://usegalaxy.eu/published/workflow?id=1b49eda539658e53, https://usegalaxy.eu/published/workflow?id=1bc7b5eabea972fd, https://usegalaxy.eu/published/workflow?id=1ccc0c9e1c6821bb, https://usegalaxy.eu/published/workflow?id=1ee43d90b69829e9, https://usegalaxy.eu/published/workflow?id=1fe02012c96b731f, https://usegalaxy.eu/published/workflow?id=20ada53472782c9b, https://usegalaxy.eu/published/workflow?id=212c213f1d8e0154, https://usegalaxy.eu/published/workflow?id=2287afbf248d9c89, https://usegalaxy.eu/published/workflow?id=250e6058f160825f, https://usegalaxy.eu/published/workflow?id=26185e8fa5a9b988, https://usegalaxy.eu/published/workflow?id=29a855d928337f8f, https://usegalaxy.eu/published/workflow?id=2a123e08436a89f1, https://usegalaxy.eu/published/workflow?id=2d08a73dd8ff99e9, https://usegalaxy.eu/published/workflow?id=2d64d6571e01809d, https://usegalaxy.eu/published/workflow?id=300301c23327f797, https://usegalaxy.eu/published/workflow?id=318be2b30bf7884c, https://usegalaxy.eu/published/workflow?id=31d9eb6a120411d6, https://usegalaxy.eu/published/workflow?id=3550384e7c062a60, https://usegalaxy.eu/published/workflow?id=39876e9c04aad543, https://usegalaxy.eu/published/workflow?id=3d5b39e454df584b, https://usegalaxy.eu/published/workflow?id=443ea80b36d862f5, https://usegalaxy.eu/published/workflow?id=4566262dd0acc8ab, https://usegalaxy.eu/published/workflow?id=487929c7b5be8dc3, https://usegalaxy.eu/published/workflow?id=49073a24429d93d6, https://usegalaxy.eu/published/workflow?id=4dd7f6e1c1e7f96a, https://usegalaxy.eu/published/workflow?id=4e215c9b50650d5d, https://usegalaxy.eu/published/workflow?id=50758631a0e81a7a, https://usegalaxy.eu/published/workflow?id=52cafb62f5bf6e25, https://usegalaxy.eu/published/workflow?id=534b81d48df6b507, https://usegalaxy.eu/published/workflow?id=535f777e5e42e98c, https://usegalaxy.eu/published/workflow?id=54467870b7c635f2, https://usegalaxy.eu/published/workflow?id=56868483be18f1a1, https://usegalaxy.eu/published/workflow?id=5aba0f20d994b1a4, https://usegalaxy.eu/published/workflow?id=5b00ef046439d757, https://usegalaxy.eu/published/workflow?id=5c8186f1e342c522, https://usegalaxy.eu/published/workflow?id=60a45de522b8b695, https://usegalaxy.eu/published/workflow?id=60e4d0db3164f037, https://usegalaxy.eu/published/workflow?id=61e592bafd9d2b58, https://usegalaxy.eu/published/workflow?id=625aafb9cf16b03f, https://usegalaxy.eu/published/workflow?id=63478edcea3f449a, https://usegalaxy.eu/published/workflow?id=63749bd3045ccd4a, https://usegalaxy.eu/published/workflow?id=63964ba7d9edce74, https://usegalaxy.eu/published/workflow?id=68e8042f11d5a3fd, https://usegalaxy.eu/published/workflow?id=71a9224fa6f3da9d, https://usegalaxy.eu/published/workflow?id=733650962d379da2, https://usegalaxy.eu/published/workflow?id=762c64a9f588a420, https://usegalaxy.eu/published/workflow?id=766ebba7a2aee494, https://usegalaxy.eu/published/workflow?id=7baae0775071c7ec, https://usegalaxy.eu/published/workflow?id=7c9b182da21748c7, https://usegalaxy.eu/published/workflow?id=87fea062a9646a31, https://usegalaxy.eu/published/workflow?id=88b382a6f56a6329, https://usegalaxy.eu/published/workflow?id=8b1e2fe9f4a07de0, https://usegalaxy.eu/published/workflow?id=910570a2caafe462, https://usegalaxy.eu/published/workflow?id=9141c6449595c6a1, https://usegalaxy.eu/published/workflow?id=927d24dcde139b9c, https://usegalaxy.eu/published/workflow?id=94bbf8fa7c1f030d, https://usegalaxy.eu/published/workflow?id=96c61a584cb2e5e9, https://usegalaxy.eu/published/workflow?id=985044b4b2c51425, https://usegalaxy.eu/published/workflow?id=98d4adf5774e274e, https://usegalaxy.eu/published/workflow?id=9a1d17c12b139bda, https://usegalaxy.eu/published/workflow?id=9a52516ec0b539ea, https://usegalaxy.eu/published/workflow?id=9c7a218993788493, https://usegalaxy.eu/published/workflow?id=9e01e24c5e749365, https://usegalaxy.eu/published/workflow?id=9e31c51911987687, https://usegalaxy.eu/published/workflow?id=a28c207730e7e13d, https://usegalaxy.eu/published/workflow?id=a35e0c2fcba4e95f, https://usegalaxy.eu/published/workflow?id=a461ea397418f3aa, https://usegalaxy.eu/published/workflow?id=a6c666f3028ea440, https://usegalaxy.eu/published/workflow?id=a72d2667a53830f7, https://usegalaxy.eu/published/workflow?id=ab6e6ffaaebab45b, https://usegalaxy.eu/published/workflow?id=ab8924d78fa5ef8f, https://usegalaxy.eu/published/workflow?id=ae1ca223d6667070, https://usegalaxy.eu/published/workflow?id=af7c349497a75314, https://usegalaxy.eu/published/workflow?id=aff44f1665a14e23, https://usegalaxy.eu/published/workflow?id=b41ff6d8fb5a0878, https://usegalaxy.eu/published/workflow?id=b426e137396acb14, https://usegalaxy.eu/published/workflow?id=b801e745205892f5, https://usegalaxy.eu/published/workflow?id=b86f3775aa1f13bc, https://usegalaxy.eu/published/workflow?id=bbe2a1615c8faaae, https://usegalaxy.eu/published/workflow?id=bc79a9f797334b01, https://usegalaxy.eu/published/workflow?id=bcb8a8c5bff790ab, https://usegalaxy.eu/published/workflow?id=bfb762c96328e3c1, https://usegalaxy.eu/published/workflow?id=c5e6047b3b256b06, https://usegalaxy.eu/published/workflow?id=c64b219b4031fabd, https://usegalaxy.eu/published/workflow?id=c6b22fc33b569c8f, https://usegalaxy.eu/published/workflow?id=c885eaabae612d6d, https://usegalaxy.eu/published/workflow?id=ca9d3233b0912765, https://usegalaxy.eu/published/workflow?id=cae527ce845e770e, https://usegalaxy.eu/published/workflow?id=cbc16cb5db98ab21, https://usegalaxy.eu/published/workflow?id=cc37667c2d1f9056, https://usegalaxy.eu/published/workflow?id=ccef778cfd9024d1, https://usegalaxy.eu/published/workflow?id=d1338e2b4002a364, https://usegalaxy.eu/published/workflow?id=d3dfe7ba11508c72, https://usegalaxy.eu/published/workflow?id=d41b52a15ba56f54, https://usegalaxy.eu/published/workflow?id=d4b3da4cc5a50044, https://usegalaxy.eu/published/workflow?id=d503dee6f9b35a18, https://usegalaxy.eu/published/workflow?id=d5333903ba947adf, https://usegalaxy.eu/published/workflow?id=d72edb9f8c529f47, https://usegalaxy.eu/published/workflow?id=d9f8c5d575da02b1, https://usegalaxy.eu/published/workflow?id=daf47855afa73de8, https://usegalaxy.eu/published/workflow?id=dbcf7af09e036754, https://usegalaxy.eu/published/workflow?id=de3eafbcf42c75bb, https://usegalaxy.eu/published/workflow?id=deec04097a871646, https://usegalaxy.eu/published/workflow?id=e62b659a5da9734e, https://usegalaxy.eu/published/workflow?id=e6c5fb1d37d29ec0, https://usegalaxy.eu/published/workflow?id=e7363b9f450c56b4, https://usegalaxy.eu/published/workflow?id=e8202b7b6338005d, https://usegalaxy.eu/published/workflow?id=e85c9385870aadcc, https://usegalaxy.eu/published/workflow?id=e94e307375a92c94, https://usegalaxy.eu/published/workflow?id=eb8168bc01249596, https://usegalaxy.eu/published/workflow?id=ecab5b2c3ff3df9d, https://usegalaxy.eu/published/workflow?id=f026c4b8341ff94c, https://usegalaxy.eu/published/workflow?id=f0ac222d3e965ab0, https://usegalaxy.eu/published/workflow?id=f12fb4a95954c448, https://usegalaxy.eu/published/workflow?id=f841e1207611f28f, https://usegalaxy.eu/published/workflow?id=fb18c5d14acb1b36, https://usegalaxy.eu/published/workflow?id=fe8a1a970fd0d4cd, https://usegalaxy.fr/published/workflow?id=0f6714dbdb90b6aa, https://usegalaxy.fr/published/workflow?id=5d83e0886f2a7465, https://usegalaxy.fr/published/workflow?id=5e49933a687c2b46, https://usegalaxy.fr/published/workflow?id=6d2945d35fa956c8, https://usegalaxy.fr/published/workflow?id=72b531d980acaa6c, https://usegalaxy.fr/published/workflow?id=826cffb2458e4067, https://usegalaxy.fr/published/workflow?id=e0bad0b97ee71577, https://usegalaxy.org.au/published/workflow?id=000cdc969e9f0886, https://usegalaxy.org.au/published/workflow?id=01997cb92a2ce2d5, https://usegalaxy.org.au/published/workflow?id=02b8d033d9b1fa05, https://usegalaxy.org.au/published/workflow?id=046a1c600a6b77c2, https://usegalaxy.org.au/published/workflow?id=05950987aa2eba5b, https://usegalaxy.org.au/published/workflow?id=05b17b9d70c1db8b, https://usegalaxy.org.au/published/workflow?id=0614f24ebeb680de, https://usegalaxy.org.au/published/workflow?id=068f85b8b137bfd7, https://usegalaxy.org.au/published/workflow?id=07c97b9e59c0218f, https://usegalaxy.org.au/published/workflow?id=0c567180448d8774, https://usegalaxy.org.au/published/workflow?id=0cfc1408fd9ea4e7, https://usegalaxy.org.au/published/workflow?id=0d1db28665d19bda, https://usegalaxy.org.au/published/workflow?id=0fc7c14c4d1ac8f5, https://usegalaxy.org.au/published/workflow?id=10acf4f497172400, https://usegalaxy.org.au/published/workflow?id=114fb9589bb9db92, https://usegalaxy.org.au/published/workflow?id=14184bc2580a131f, https://usegalaxy.org.au/published/workflow?id=16f4846b4d6bc155, https://usegalaxy.org.au/published/workflow?id=19315e6573c58239, https://usegalaxy.org.au/published/workflow?id=1ab66d382630afa7, https://usegalaxy.org.au/published/workflow?id=1ad81f95ace878ea, https://usegalaxy.org.au/published/workflow?id=1c886f008a36b0b8, https://usegalaxy.org.au/published/workflow?id=20e5e6b233ffb951, https://usegalaxy.org.au/published/workflow?id=210ca013cc1cf185, https://usegalaxy.org.au/published/workflow?id=25ac9b4f2e39af97, https://usegalaxy.org.au/published/workflow?id=2999a1891ffe201b, https://usegalaxy.org.au/published/workflow?id=2a0046582f5f15a1, https://usegalaxy.org.au/published/workflow?id=2c58509f80897972, https://usegalaxy.org.au/published/workflow?id=2cef9a1633e0a067, https://usegalaxy.org.au/published/workflow?id=2f7c0e3221cbca2f, https://usegalaxy.org.au/published/workflow?id=301fdb29c513ffbe, https://usegalaxy.org.au/published/workflow?id=30b2cd025e0300da, https://usegalaxy.org.au/published/workflow?id=31449596b22f81f4, https://usegalaxy.org.au/published/workflow?id=35384818ee1cbac2, https://usegalaxy.org.au/published/workflow?id=374311292d72c9f9, https://usegalaxy.org.au/published/workflow?id=383ce49fa7a367e9, https://usegalaxy.org.au/published/workflow?id=39d56981a6854d82, https://usegalaxy.org.au/published/workflow?id=3bc45aaed5da1058, https://usegalaxy.org.au/published/workflow?id=3c727d5cd349a846, https://usegalaxy.org.au/published/workflow?id=3ce9dc804780e4a1, https://usegalaxy.org.au/published/workflow?id=3ef07022656067b1, https://usegalaxy.org.au/published/workflow?id=410d0d19b6589c78, https://usegalaxy.org.au/published/workflow?id=421adea2f259ec26, https://usegalaxy.org.au/published/workflow?id=42305ed876d19d08, https://usegalaxy.org.au/published/workflow?id=45c8789533c45abf, https://usegalaxy.org.au/published/workflow?id=49b5c35b9723fa62, https://usegalaxy.org.au/published/workflow?id=4e096a68187ce652, https://usegalaxy.org.au/published/workflow?id=4efed79608f85604, https://usegalaxy.org.au/published/workflow?id=4f2818b6620037af, https://usegalaxy.org.au/published/workflow?id=4f562265cbb14cd6, https://usegalaxy.org.au/published/workflow?id=525612313de7591e, https://usegalaxy.org.au/published/workflow?id=53010c2f1f207429, https://usegalaxy.org.au/published/workflow?id=5314a7a4732a0ef1, https://usegalaxy.org.au/published/workflow?id=544cb12a16f283b8, https://usegalaxy.org.au/published/workflow?id=56599572e0364e2a, https://usegalaxy.org.au/published/workflow?id=5a4e49ef8693d7c4, https://usegalaxy.org.au/published/workflow?id=5be983b903469f07, https://usegalaxy.org.au/published/workflow?id=5cbee52cf03b6f9a, https://usegalaxy.org.au/published/workflow?id=5ce00d3eab862f4e, https://usegalaxy.org.au/published/workflow?id=5ea74382fb77eb10, https://usegalaxy.org.au/published/workflow?id=608f3ad8f485e7a8, https://usegalaxy.org.au/published/workflow?id=6204bbcfe56931eb, https://usegalaxy.org.au/published/workflow?id=625f3c52832b0591, https://usegalaxy.org.au/published/workflow?id=6448b82959076a29, https://usegalaxy.org.au/published/workflow?id=6469ade8f6f7c718, https://usegalaxy.org.au/published/workflow?id=66b5c7dd3e58193f, https://usegalaxy.org.au/published/workflow?id=67191ea9fe977e08, https://usegalaxy.org.au/published/workflow?id=677ef3322366c953, https://usegalaxy.org.au/published/workflow?id=69233fb6edfcdde9, https://usegalaxy.org.au/published/workflow?id=6b34c95a247d15e3, https://usegalaxy.org.au/published/workflow?id=6cdbcd1f667d0f16, https://usegalaxy.org.au/published/workflow?id=6f0801fecbba5d3f, https://usegalaxy.org.au/published/workflow?id=71bad95f72c9f30d, https://usegalaxy.org.au/published/workflow?id=723513b09d6690b8, https://usegalaxy.org.au/published/workflow?id=73ff1aa9acc0ad2a, https://usegalaxy.org.au/published/workflow?id=74186b006665daf8, https://usegalaxy.org.au/published/workflow?id=749d8f2d26e0a111, https://usegalaxy.org.au/published/workflow?id=75113a1b41904d2a, https://usegalaxy.org.au/published/workflow?id=76323f103844f75b, https://usegalaxy.org.au/published/workflow?id=79522a1f7b0c53cb, https://usegalaxy.org.au/published/workflow?id=7ad277dd448e57b3, https://usegalaxy.org.au/published/workflow?id=7ae06a27188b9438, https://usegalaxy.org.au/published/workflow?id=7becafd394472dfb, https://usegalaxy.org.au/published/workflow?id=7c22d726e1716f4b, https://usegalaxy.org.au/published/workflow?id=7c866ed4a3552d0d, https://usegalaxy.org.au/published/workflow?id=7cd1acb1da823cdc, https://usegalaxy.org.au/published/workflow?id=7ea616c652d08fb6, https://usegalaxy.org.au/published/workflow?id=7ef245f5ae3b916d, https://usegalaxy.org.au/published/workflow?id=7febc6ac6f106e57, https://usegalaxy.org.au/published/workflow?id=811bbada546850bd, https://usegalaxy.org.au/published/workflow?id=83680c1c6dffb05b, https://usegalaxy.org.au/published/workflow?id=83a535840b388980, https://usegalaxy.org.au/published/workflow?id=8483ae422665a6c1, https://usegalaxy.org.au/published/workflow?id=8483e8d3e8d0ddc9, https://usegalaxy.org.au/published/workflow?id=8597166abda0a375, https://usegalaxy.org.au/published/workflow?id=85aede91db0122e4, https://usegalaxy.org.au/published/workflow?id=863eee1b041b6c58, https://usegalaxy.org.au/published/workflow?id=879d4fb110a61f7a, https://usegalaxy.org.au/published/workflow?id=88c0d875eea2ee49, https://usegalaxy.org.au/published/workflow?id=897dfa78c3b25356, https://usegalaxy.org.au/published/workflow?id=897e9d7432d231e9, https://usegalaxy.org.au/published/workflow?id=8b8a5b50f2edc72c, https://usegalaxy.org.au/published/workflow?id=8c40b4a911517cc5, https://usegalaxy.org.au/published/workflow?id=8cab31db0bd3ff52, https://usegalaxy.org.au/published/workflow?id=8d196c8c62530dc0, https://usegalaxy.org.au/published/workflow?id=8d6010d07cc2e170, https://usegalaxy.org.au/published/workflow?id=8fb691706f47527a, https://usegalaxy.org.au/published/workflow?id=9157a7438977b9e9, https://usegalaxy.org.au/published/workflow?id=91cfd96f697ba034, https://usegalaxy.org.au/published/workflow?id=925c167abd5610ae, https://usegalaxy.org.au/published/workflow?id=95a9070469a895c3, https://usegalaxy.org.au/published/workflow?id=95efb414f5984735, https://usegalaxy.org.au/published/workflow?id=975ddb0eb436f1fd, https://usegalaxy.org.au/published/workflow?id=97628b0604d5ec77, https://usegalaxy.org.au/published/workflow?id=978953f492b2b914, https://usegalaxy.org.au/published/workflow?id=981d7b02416b7734, https://usegalaxy.org.au/published/workflow?id=9898348d95033af2, https://usegalaxy.org.au/published/workflow?id=9988bc2402f32dc1, https://usegalaxy.org.au/published/workflow?id=9b1afa59c087eacd, https://usegalaxy.org.au/published/workflow?id=9df67170c9260539, https://usegalaxy.org.au/published/workflow?id=9e8d9b7055b7c863, https://usegalaxy.org.au/published/workflow?id=9ec9fe2715d7cfdc, https://usegalaxy.org.au/published/workflow?id=a1c59e9f5c7327d2, https://usegalaxy.org.au/published/workflow?id=a69db7f914c06407, https://usegalaxy.org.au/published/workflow?id=a7346c25dfdf2e3e, https://usegalaxy.org.au/published/workflow?id=a763e53653b10ce0, https://usegalaxy.org.au/published/workflow?id=a7726d18d2a5fcf2, https://usegalaxy.org.au/published/workflow?id=a7a30d970584f9ac, https://usegalaxy.org.au/published/workflow?id=a7d1cf2bb636e695, https://usegalaxy.org.au/published/workflow?id=a9fa6101f1fb13e2, https://usegalaxy.org.au/published/workflow?id=aa94c77b720a2810, https://usegalaxy.org.au/published/workflow?id=aaaf580eb00459b6, https://usegalaxy.org.au/published/workflow?id=aae07f802002fdd0, https://usegalaxy.org.au/published/workflow?id=ad13b4a61e8987b1, https://usegalaxy.org.au/published/workflow?id=ae5dca8108d1a8a6, https://usegalaxy.org.au/published/workflow?id=ae6892f1df022097, https://usegalaxy.org.au/published/workflow?id=ae773dfaa074cb72, https://usegalaxy.org.au/published/workflow?id=aed93da6fd6e4e13, https://usegalaxy.org.au/published/workflow?id=af3f40cba4d7afa6, https://usegalaxy.org.au/published/workflow?id=af7b95f46485308c, https://usegalaxy.org.au/published/workflow?id=af990cca72794142, https://usegalaxy.org.au/published/workflow?id=afd9a8bbc07f9c3f, https://usegalaxy.org.au/published/workflow?id=b03b577156e4db5b, https://usegalaxy.org.au/published/workflow?id=b517cabb697491a0, https://usegalaxy.org.au/published/workflow?id=b57bf134896c30ea, https://usegalaxy.org.au/published/workflow?id=b6db525b1b73394b, https://usegalaxy.org.au/published/workflow?id=b8c661ff6b79383b, https://usegalaxy.org.au/published/workflow?id=bda98c5a0ea70dd0, https://usegalaxy.org.au/published/workflow?id=c1a0a3888d9ee3ac, https://usegalaxy.org.au/published/workflow?id=c296b3848708f6f1, https://usegalaxy.org.au/published/workflow?id=c2e5a43356042b30, https://usegalaxy.org.au/published/workflow?id=c3b3717502ff6f76, https://usegalaxy.org.au/published/workflow?id=c3bb21f20a3ce689, https://usegalaxy.org.au/published/workflow?id=c3e198bc2146b4ad, https://usegalaxy.org.au/published/workflow?id=c65a5af6c0383f8c, https://usegalaxy.org.au/published/workflow?id=c8d18e2c03dca796, https://usegalaxy.org.au/published/workflow?id=cb960edf95d2dc35, https://usegalaxy.org.au/published/workflow?id=ccfae75390038ced, https://usegalaxy.org.au/published/workflow?id=ce7f6a1c824920e2, https://usegalaxy.org.au/published/workflow?id=cef4a514f35576e5, https://usegalaxy.org.au/published/workflow?id=cef68481b305a516, https://usegalaxy.org.au/published/workflow?id=d126940d33e63036, https://usegalaxy.org.au/published/workflow?id=d5ecfdbc04b24227, https://usegalaxy.org.au/published/workflow?id=d7a65f577fe5f16b, https://usegalaxy.org.au/published/workflow?id=db0bc7357c181222, https://usegalaxy.org.au/published/workflow?id=db3370a412c4de78, https://usegalaxy.org.au/published/workflow?id=dce29c35dfac48b1, https://usegalaxy.org.au/published/workflow?id=de1c04a2caf70647, https://usegalaxy.org.au/published/workflow?id=e08ef6b519f5fba0, https://usegalaxy.org.au/published/workflow?id=e1fd89e7248b6661, https://usegalaxy.org.au/published/workflow?id=e3cda847fe727b61, https://usegalaxy.org.au/published/workflow?id=e4ddf33ee5b5e259, https://usegalaxy.org.au/published/workflow?id=e52a3479c8216291, https://usegalaxy.org.au/published/workflow?id=e62f743342851151, https://usegalaxy.org.au/published/workflow?id=e826584d78b69467, https://usegalaxy.org.au/published/workflow?id=e8d3651c07ce0bec, https://usegalaxy.org.au/published/workflow?id=eb3a4535204383fa, https://usegalaxy.org.au/published/workflow?id=ec79c2fab8eac15d, https://usegalaxy.org.au/published/workflow?id=ed2f72e3537e8a4a, https://usegalaxy.org.au/published/workflow?id=f0ced9a5d34b2aff, https://usegalaxy.org.au/published/workflow?id=f19d63d7c9a48160, https://usegalaxy.org.au/published/workflow?id=f268c181a47d0cab, https://usegalaxy.org.au/published/workflow?id=f5c0b0bc5e023e42, https://usegalaxy.org.au/published/workflow?id=f944f2d3fbd1f062, https://usegalaxy.org.au/published/workflow?id=fb644a9649f36f40, https://usegalaxy.org.au/published/workflow?id=fc64a77321752a3c, https://usegalaxy.org.au/published/workflow?id=fc6f8afdee980f0e, https://usegalaxy.org.au/published/workflow?id=fd477917a3d27f61, https://usegalaxy.org.au/published/workflow?id=fd8fb237f8dc24c3, https://usegalaxy.org.au/published/workflow?id=fe1de81a97578368, https://usegalaxy.org.au/published/workflow?id=fe893d72526eba71, https://usegalaxy.org.au/published/workflow?id=ff42213dda7d3e59, https://usegalaxy.org/published/workflow?id=00a501a1fada4104, https://usegalaxy.org/published/workflow?id=03a1cd225af6de69, https://usegalaxy.org/published/workflow?id=044c5ce43f2e1829, https://usegalaxy.org/published/workflow?id=0a009cf9e88171f1, https://usegalaxy.org/published/workflow?id=13460c791ca97689, https://usegalaxy.org/published/workflow?id=1dd60e57d1d00406, https://usegalaxy.org/published/workflow?id=2125db88b4a71c09, https://usegalaxy.org/published/workflow?id=2375a915a5b33b39, https://usegalaxy.org/published/workflow?id=271b97e5c00c4da4, https://usegalaxy.org/published/workflow?id=2a7fbfc1c17e38bc, https://usegalaxy.org/published/workflow?id=2c36a7ab1979ee8a, https://usegalaxy.org/published/workflow?id=30883502ec515fe2, https://usegalaxy.org/published/workflow?id=3ad9cdcda44b7dfb, https://usegalaxy.org/published/workflow?id=4061e061f659449b, https://usegalaxy.org/published/workflow?id=413d7e5aa7378cf1, https://usegalaxy.org/published/workflow?id=41c09a87c6cd5f75, https://usegalaxy.org/published/workflow?id=42bc3a770f0ab6b8, https://usegalaxy.org/published/workflow?id=48adcc44fe62a99c, https://usegalaxy.org/published/workflow?id=4a50ce0e9f4d4bb8, https://usegalaxy.org/published/workflow?id=4e6005819e57ed35, https://usegalaxy.org/published/workflow?id=5336b9d26a083983, https://usegalaxy.org/published/workflow?id=544ce8ff6c9f6d4c, https://usegalaxy.org/published/workflow?id=5497e8630ebfeb42, https://usegalaxy.org/published/workflow?id=582f3de3971fc38c, https://usegalaxy.org/published/workflow?id=5944c5effd56b0d6, https://usegalaxy.org/published/workflow?id=5a272f266a1fa4ac, https://usegalaxy.org/published/workflow?id=63d590e8567f7157, https://usegalaxy.org/published/workflow?id=69af7dd51aff958a, https://usegalaxy.org/published/workflow?id=7022c4d9af725e5c, https://usegalaxy.org/published/workflow?id=73d98e5f821a78b3, https://usegalaxy.org/published/workflow?id=75f14618020c922a, https://usegalaxy.org/published/workflow?id=7a203cf1678df56d, https://usegalaxy.org/published/workflow?id=7a26069c2939bef9, https://usegalaxy.org/published/workflow?id=7b19452fe35b8b17, https://usegalaxy.org/published/workflow?id=7e81153760fb4c90, https://usegalaxy.org/published/workflow?id=812bc1b7a52f4f6f, https://usegalaxy.org/published/workflow?id=82681bd5f059e39a, https://usegalaxy.org/published/workflow?id=8700f593a3d642f5, https://usegalaxy.org/published/workflow?id=8bcab83874cbc75c, https://usegalaxy.org/published/workflow?id=8d692df647e2aef3, https://usegalaxy.org/published/workflow?id=8ebcae16a7822d93, https://usegalaxy.org/published/workflow?id=93bfbd7db42db543, https://usegalaxy.org/published/workflow?id=9bace540b35b1e61, https://usegalaxy.org/published/workflow?id=9c022f4028de0bc6, https://usegalaxy.org/published/workflow?id=9e74e3455e47c813, https://usegalaxy.org/published/workflow?id=a743746151101352, https://usegalaxy.org/published/workflow?id=ac8b74b911778fbc, https://usegalaxy.org/published/workflow?id=afebfed5df59dc18, https://usegalaxy.org/published/workflow?id=b2e81f82ccf3afd6, https://usegalaxy.org/published/workflow?id=b882703ca6fea3a8, https://usegalaxy.org/published/workflow?id=bb2ac86b0440ae86, https://usegalaxy.org/published/workflow?id=bc83c42b44c4b03f, https://usegalaxy.org/published/workflow?id=bdfb27e4ef7b6138, https://usegalaxy.org/published/workflow?id=c24cfddfc88a97a3, https://usegalaxy.org/published/workflow?id=cd431e5f912d3616, https://usegalaxy.org/published/workflow?id=cd56ed91856fcc6f, https://usegalaxy.org/published/workflow?id=cf3f0760611f22f9, https://usegalaxy.org/published/workflow?id=d761637d43ee0324, https://usegalaxy.org/published/workflow?id=d82db9c29b998f22, https://usegalaxy.org/published/workflow?id=d97a88a8f9061e25, https://usegalaxy.org/published/workflow?id=dab54002b3c0e351, https://usegalaxy.org/published/workflow?id=e04f443a8ceca0d2, https://usegalaxy.org/published/workflow?id=e22ea88bc0fdc146, https://usegalaxy.org/published/workflow?id=e26fabceece3b858, https://usegalaxy.org/published/workflow?id=e33f6a058c5f57e5, https://usegalaxy.org/published/workflow?id=e55ca98817c1855e, https://usegalaxy.org/published/workflow?id=e5a01f746bc9e9b5, https://usegalaxy.org/published/workflow?id=eb01b539d17580c3, https://usegalaxy.org/published/workflow?id=eb4b32fcf4a9f64a, https://usegalaxy.org/published/workflow?id=f961fc46d4fddecc, https://usegalaxy.org/published/workflow?id=fb789920b057fe8d, https://usegalaxy.org/published/workflow?id=fb903de4aa4d5090, https://usegalaxy.org/published/workflow?id=fc09d17e0bce18f8, https://workflowhub.eu/workflows/1494?version=1, https://workflowhub.eu/workflows/1675?version=1, https://workflowhub.eu/workflows/1685?version=1, https://workflowhub.eu/workflows/1692?version=1, https://workflowhub.eu/workflows/1715?version=3, https://workflowhub.eu/workflows/338?version=1, https://workflowhub.eu/workflows/400?version=12, https://workflowhub.eu/workflows/401?version=13, https://workflowhub.eu/workflows/688?version=1</td>
      <td>single-cell/scrna-preprocessing, transcriptomics/de-novo, transcriptomics/minerva-pathways, transcriptomics/ref-based, transcriptomics/rna-seq-reads-to-counts</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>filter_spades_repeats</td>
      <td>filter_spades_repeat</td>
      <td>Remove short and repeat contigs/scaffolds</td>
      <td>2017-10-12</td>
      <td>https://github.com/phac-nml/galaxy_tools/</td>
      <td>1.0.1</td>
      <td>perl-bioperl</td>
      <td>1.7.8</td>
      <td>To update</td>
      <td>Assembly</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools/</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/filter_spades_repeats</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>filtlong</td>
      <td>filtlong</td>
      <td>Filtlong - Filtering long reads by quality</td>
      <td>2018-09-14</td>
      <td>https://github.com/rrwick/Filtlong</td>
      <td>0.3.1</td>
      <td>filtlong</td>
      <td>0.3.1</td>
      <td>Up-to-date</td>
      <td>Fastq Manipulation, Sequence Analysis</td>
      <td>Filtering, Sequencing quality control</td>
      <td>Filtering, Sequencing quality control</td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/filtlong</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/filtlong</td>
      <td>filtlong</td>
      <td>Filtlong</td>
      <td>Filtlong is a tool for filtering long reads by quality. It can take a set of long reads and produce a smaller, better subset. It uses both read length (longer is better) and read identity (higher is better) when choosing which reads pass the filter.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2169</td>
      <td>2318</td>
      <td>79820</td>
      <td>82849</td>
      <td>1410</td>
      <td>1424</td>
      <td>72145</td>
      <td>72255</td>
      <td>830</td>
      <td>830</td>
      <td>17538</td>
      <td>17538</td>
      <td>27</td>
      <td>27</td>
      <td>150</td>
      <td>150</td>
      <td>172792</td>
      <td>169653</td>
      <td>4599</td>
      <td>4436</td>
      <td>https://usegalaxy.eu/published/workflow?id=07c92110c3fffcf8, https://usegalaxy.eu/published/workflow?id=30dc526dcfa500df, https://usegalaxy.eu/published/workflow?id=346b46e6316c6950, https://usegalaxy.eu/published/workflow?id=4ddbffe4b3fef275, https://usegalaxy.eu/published/workflow?id=5500de2961f7470c, https://usegalaxy.eu/published/workflow?id=5826e573b6f93762, https://usegalaxy.eu/published/workflow?id=6f878d8b43ef1521, https://usegalaxy.eu/published/workflow?id=83a1b8949d87e3da, https://usegalaxy.eu/published/workflow?id=86cf02cc0868d238, https://usegalaxy.eu/published/workflow?id=968f0ac2617aa524, https://usegalaxy.eu/published/workflow?id=a337bfb2eb1e583f, https://usegalaxy.eu/published/workflow?id=a84bac86ec183d46, https://usegalaxy.eu/published/workflow?id=ab646c6ab6de7061, https://usegalaxy.eu/published/workflow?id=bf48b0eb15e8ff75, https://usegalaxy.eu/published/workflow?id=c3bfedf33bb00069, https://usegalaxy.eu/published/workflow?id=c9ef2d2ce92e2dfb, https://usegalaxy.eu/published/workflow?id=ea485dbabc76488d, https://usegalaxy.eu/published/workflow?id=eb5ab8bc82569eee, https://usegalaxy.eu/published/workflow?id=ecf48fa50af560a0, https://usegalaxy.org.au/published/workflow?id=2e29b21cdec7d0cc, https://usegalaxy.org.au/published/workflow?id=35f39173f0ac9179, https://usegalaxy.org.au/published/workflow?id=4871f853090e8be8, https://usegalaxy.org.au/published/workflow?id=b7aa22291322e505, https://usegalaxy.org.au/published/workflow?id=c997a34b55425155, https://usegalaxy.org.au/published/workflow?id=d69a765cfc82a399, https://usegalaxy.org.au/published/workflow?id=fc664a96d8099652, https://usegalaxy.org/published/workflow?id=2ded909e7e3a9962, https://usegalaxy.org/published/workflow?id=48dafc35989f13b1, https://usegalaxy.org/published/workflow?id=547f86db467bf620, https://usegalaxy.org/published/workflow?id=7fc8b2398b6f093f, https://usegalaxy.org/published/workflow?id=c451e4dca9981ed0, https://usegalaxy.org/published/workflow?id=cc6ee582a6e5cae2, https://usegalaxy.org/published/workflow?id=d5436cee11a34de9, https://usegalaxy.org/published/workflow?id=e1dcdc125122f561, https://workflowhub.eu/workflows/1613?version=2, https://workflowhub.eu/workflows/51?version=1</td>
      <td>assembly/mrsa-nanopore</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>flashlfq</td>
      <td>flashlfq</td>
      <td>FlashLFQ mass-spectrometry proteomics label-free quantification</td>
      <td>2017-12-06</td>
      <td>https://github.com/smith-chem-wisc/FlashLFQ</td>
      <td>1.0.3.1</td>
      <td>flashlfq</td>
      <td>2.1.4</td>
      <td>To update</td>
      <td>Proteomics</td>
      <td>Label-free quantification</td>
      <td>Label-free quantification</td>
      <td>Proteomics experiment, Proteomics</td>
      <td>Proteomics experiment, Proteomics</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/flashlfq</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/flashlfq</td>
      <td>flashlfq</td>
      <td>FlashLFQ</td>
      <td>FlashLFQ is an ultrafast label-free quantification algorithm for mass-spectrometry proteomics.</td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>47</td>
      <td>59</td>
      <td>516</td>
      <td>760</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>16</td>
      <td>16</td>
      <td>216</td>
      <td>226</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>986</td>
      <td>732</td>
      <td>75</td>
      <td>63</td>
      <td>https://usegalaxy.eu/published/workflow?id=ba3a5d59b3879764, https://usegalaxy.eu/published/workflow?id=cd675ab32d3e7833, https://usegalaxy.eu/published/workflow?id=d0e062aee910086a, https://usegalaxy.eu/published/workflow?id=fc42cc9467c6ee8d, https://workflowhub.eu/workflows/1450?version=1</td>
      <td>proteomics/metaquantome-data-creation</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>flye</td>
      <td>flye</td>
      <td>Assembly of long and error-prone reads.</td>
      <td>2018-09-22</td>
      <td>https://github.com/fenderglass/Flye/</td>
      <td>2.9.6</td>
      <td>flye</td>
      <td>2.9.6</td>
      <td>Up-to-date</td>
      <td>Assembly</td>
      <td>Genome assembly, De-novo assembly, Mapping assembly, Cross-assembly</td>
      <td>Genome assembly, De-novo assembly, Mapping assembly, Cross-assembly</td>
      <td>Sequence assembly, Metagenomics, Whole genome sequencing, Genomics</td>
      <td>Sequence assembly, Metagenomics, Whole genome sequencing</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/flye</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/flye</td>
      <td>Flye</td>
      <td>Flye</td>
      <td>Flye is a de novo assembler for single molecule sequencing reads, such as those produced by PacBio and Oxford Nanopore Technologies. It is designed for a wide range of datasets, from small bacterial projects to large mammalian-scale assemblies. The package represents a complete pipeline: it takes raw PB / ONT reads as input and outputs polished contigs.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>4818</td>
      <td>4961</td>
      <td>49679</td>
      <td>51133</td>
      <td>4110</td>
      <td>4110</td>
      <td>36053</td>
      <td>36053</td>
      <td>2250</td>
      <td>2274</td>
      <td>13882</td>
      <td>14419</td>
      <td>185</td>
      <td>185</td>
      <td>1088</td>
      <td>1088</td>
      <td>102693</td>
      <td>100702</td>
      <td>11530</td>
      <td>11363</td>
      <td>https://usegalaxy.eu/published/workflow?id=0ebfab0e957b7267, https://usegalaxy.eu/published/workflow?id=152223c8aed27357, https://usegalaxy.eu/published/workflow?id=2b888e200dfd91bc, https://usegalaxy.eu/published/workflow?id=30b514cfb9cce524, https://usegalaxy.eu/published/workflow?id=34a07bf6407510ef, https://usegalaxy.eu/published/workflow?id=368d1ac0150ea0b6, https://usegalaxy.eu/published/workflow?id=4ddbffe4b3fef275, https://usegalaxy.eu/published/workflow?id=5500de2961f7470c, https://usegalaxy.eu/published/workflow?id=585c21b7b1d864fc, https://usegalaxy.eu/published/workflow?id=77e5bbd317750915, https://usegalaxy.eu/published/workflow?id=83a1b8949d87e3da, https://usegalaxy.eu/published/workflow?id=86cf02cc0868d238, https://usegalaxy.eu/published/workflow?id=8b2f53745e720d0f, https://usegalaxy.eu/published/workflow?id=968f0ac2617aa524, https://usegalaxy.eu/published/workflow?id=9ddb703e1db13504, https://usegalaxy.eu/published/workflow?id=a337bfb2eb1e583f, https://usegalaxy.eu/published/workflow?id=a84bac86ec183d46, https://usegalaxy.eu/published/workflow?id=b1d9e56aef03672b, https://usegalaxy.eu/published/workflow?id=b2dfa74f55a73409, https://usegalaxy.eu/published/workflow?id=bf48b0eb15e8ff75, https://usegalaxy.eu/published/workflow?id=c9ef2d2ce92e2dfb, https://usegalaxy.eu/published/workflow?id=cf8591391a17a914, https://usegalaxy.eu/published/workflow?id=d2f75f70c33394ee, https://usegalaxy.eu/published/workflow?id=eb5ab8bc82569eee, https://usegalaxy.org.au/published/workflow?id=02c74095a7128c9d, https://usegalaxy.org.au/published/workflow?id=084bb76cf47d7060, https://usegalaxy.org.au/published/workflow?id=1ad4c47cee851dda, https://usegalaxy.org.au/published/workflow?id=22bd45f60c60a338, https://usegalaxy.org.au/published/workflow?id=2cb16d321a804b1a, https://usegalaxy.org.au/published/workflow?id=2e29b21cdec7d0cc, https://usegalaxy.org.au/published/workflow?id=35f39173f0ac9179, https://usegalaxy.org.au/published/workflow?id=479c6b0a96ea1610, https://usegalaxy.org.au/published/workflow?id=59618ed15510f531, https://usegalaxy.org.au/published/workflow?id=5b32e2f0d390a4ba, https://usegalaxy.org.au/published/workflow?id=68f5c371adbe5490, https://usegalaxy.org.au/published/workflow?id=7c343e5b55474d19, https://usegalaxy.org.au/published/workflow?id=803ef29108e36e58, https://usegalaxy.org.au/published/workflow?id=8a40059343cf4b87, https://usegalaxy.org.au/published/workflow?id=b5403e4fff2961f9, https://usegalaxy.org.au/published/workflow?id=b6bad1f167257248, https://usegalaxy.org.au/published/workflow?id=c4d23297f6fbf5f3, https://usegalaxy.org.au/published/workflow?id=c997a34b55425155, https://usegalaxy.org.au/published/workflow?id=d832905fbaf2ca31, https://usegalaxy.org.au/published/workflow?id=ef8c22c2525063a2, https://usegalaxy.org.au/published/workflow?id=fc664a96d8099652, https://usegalaxy.org/published/workflow?id=00596e862edb23f2, https://usegalaxy.org/published/workflow?id=0602c14704dbe1ff, https://usegalaxy.org/published/workflow?id=2ded909e7e3a9962, https://usegalaxy.org/published/workflow?id=3f252e077e0bcce5, https://usegalaxy.org/published/workflow?id=48dafc35989f13b1, https://usegalaxy.org/published/workflow?id=547f86db467bf620, https://usegalaxy.org/published/workflow?id=5626a440a5798458, https://usegalaxy.org/published/workflow?id=562bcaa9b4d2e8a4, https://usegalaxy.org/published/workflow?id=69d8863d40524120, https://usegalaxy.org/published/workflow?id=7dd6e4150bb81ad1, https://usegalaxy.org/published/workflow?id=7fc8b2398b6f093f, https://usegalaxy.org/published/workflow?id=80ac6adead9f97d8, https://usegalaxy.org/published/workflow?id=bff206b4625375bc, https://usegalaxy.org/published/workflow?id=c451e4dca9981ed0, https://usegalaxy.org/published/workflow?id=c8d74054c1820087, https://usegalaxy.org/published/workflow?id=cce88bc57b180d09, https://usegalaxy.org/published/workflow?id=cdcf91a8ddb5dcd1, https://usegalaxy.org/published/workflow?id=e12324f5e3b930fd, https://workflowhub.eu/workflows/1062?version=1, https://workflowhub.eu/workflows/1114?version=1, https://workflowhub.eu/workflows/1495?version=1, https://workflowhub.eu/workflows/1595?version=1, https://workflowhub.eu/workflows/1613?version=2, https://workflowhub.eu/workflows/1617?version=1, https://workflowhub.eu/workflows/1621?version=1, https://workflowhub.eu/workflows/2011?version=1, https://workflowhub.eu/workflows/225?version=1, https://workflowhub.eu/workflows/51?version=1, https://workflowhub.eu/workflows/750?version=3, https://workflowhub.eu/workflows/788?version=1</td>
      <td>assembly/chloroplast-assembly, assembly/hifi-assembly, assembly/hybrid_denovo_assembly, assembly/largegenome, assembly/mrsa-nanopore, microbiome/pathogen-detection-from-nanopore-foodborne-data</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>format_metaphlan2_output</td>
      <td>format_metaphlan2_output</td>
      <td>Format MetaPhlAn2 output to extract abundance at different taxonomic levels</td>
      <td>2022-10-19</td>
      <td></td>
      <td>0.2.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Formatting</td>
      <td>Formatting</td>
      <td>Taxonomy, Metagenomics</td>
      <td>Taxonomy, Metagenomics</td>
      <td>bebatut</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/format_metaphlan2_output/</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/format_metaphlan2_output</td>
      <td>format_metaphlan2_output</td>
      <td>Format metaphlan2 output</td>
      <td>This tool format output file of MetaPhlan2 containing community content (abundance) at all taxonomic levels (from kingdom to strains).</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>555</td>
      <td>731</td>
      <td>6485</td>
      <td>7678</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7678</td>
      <td>6485</td>
      <td>731</td>
      <td>555</td>
      <td>https://usegalaxy.eu/published/workflow?id=096b75501c8e0888, https://usegalaxy.eu/published/workflow?id=2cf09adb1ed6c3ec, https://usegalaxy.eu/published/workflow?id=7ab70660e6235cf0, https://usegalaxy.eu/published/workflow?id=994e5ff1df7d6c9e, https://usegalaxy.eu/published/workflow?id=9cbba5de2cbeeb0f, https://usegalaxy.eu/published/workflow?id=a8f829deb69b30ab, https://usegalaxy.eu/published/workflow?id=c01ef8354b585a1f, https://usegalaxy.eu/published/workflow?id=c3aa516e78ca4191, https://usegalaxy.eu/published/workflow?id=d05c6f0f3205fbd2, https://usegalaxy.eu/published/workflow?id=dfc2c53ac3103eff, https://usegalaxy.eu/published/workflow?id=ed0ad728cf21702b</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>fraggenescan</td>
      <td>fraggenescan</td>
      <td>Tool for finding (fragmented) genes in short read</td>
      <td>2017-09-04</td>
      <td>https://sourceforge.net/projects/fraggenescan/</td>
      <td></td>
      <td>fraggenescan</td>
      <td>1.32</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Gene prediction</td>
      <td>Gene prediction</td>
      <td>Genetics, Sequence analysis</td>
      <td>Genetics, Sequence analysis</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fraggenescan</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fraggenescan</td>
      <td>fraggenescan</td>
      <td>FragGeneScan</td>
      <td>Application for finding (fragmented) genes in short reads</td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>160</td>
      <td>193</td>
      <td>1267</td>
      <td>1643</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>48</td>
      <td>48</td>
      <td>1171</td>
      <td>1208</td>
      <td>2</td>
      <td>2</td>
      <td>12</td>
      <td>12</td>
      <td>2863</td>
      <td>2450</td>
      <td>243</td>
      <td>210</td>
      <td>https://usegalaxy.eu/published/workflow?id=02631612a43a4d9b, https://usegalaxy.eu/published/workflow?id=3eaa913306c16146, https://usegalaxy.eu/published/workflow?id=401746888e8b3aa8, https://usegalaxy.eu/published/workflow?id=534f5fc25bc887ca, https://usegalaxy.eu/published/workflow?id=58955868b504c60d, https://usegalaxy.eu/published/workflow?id=6158c3b5fc12044e, https://usegalaxy.eu/published/workflow?id=6416cfdbd5273063, https://usegalaxy.eu/published/workflow?id=667bac8d7453e5f1, https://usegalaxy.eu/published/workflow?id=8c7958cb05fba814, https://usegalaxy.eu/published/workflow?id=8deb08d4f228e198, https://usegalaxy.eu/published/workflow?id=9382f9bbbdddb5ad, https://usegalaxy.eu/published/workflow?id=9a97c6eb4fe015e7, https://usegalaxy.eu/published/workflow?id=a8417c63e9e22afb, https://usegalaxy.eu/published/workflow?id=d4a95588e6e50893, https://usegalaxy.eu/published/workflow?id=d57d41e306241396</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>fragpipe</td>
      <td>fragpipe, fragpipe_manifest_generator</td>
      <td>Data analysis for mass spectrometry-based proteomics.</td>
      <td>2022-07-21</td>
      <td>https://fragpipe.nesvilab.org/</td>
      <td>23.0</td>
      <td>fragpipe</td>
      <td>24.0</td>
      <td>To update</td>
      <td>Proteomics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/fragpipe</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/fragpipe</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>41</td>
      <td>41</td>
      <td>435</td>
      <td>435</td>
      <td>23</td>
      <td>23</td>
      <td>70</td>
      <td>70</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>505</td>
      <td>505</td>
      <td>64</td>
      <td>64</td>
      <td>https://usegalaxy.eu/published/workflow?id=4c7fabcb345f5990, https://usegalaxy.eu/published/workflow?id=8ef3379dded2490a, https://usegalaxy.eu/published/workflow?id=bee2a7669a7dea5b, https://usegalaxy.org.au/published/workflow?id=9365a7de314c9f85, https://usegalaxy.org/published/workflow?id=e3a30c0583183c1e, https://workflowhub.eu/workflows/1410?version=1, https://workflowhub.eu/workflows/1793?version=1</td>
      <td>proteomics/neoantigen-fragpipe-discovery</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>freqsap</td>
      <td>freqsap</td>
      <td>Get frequencies of single amino-acid polymorphisms based on nucleid-acid polymorphism for different populations from UniProt and DbSNP.</td>
      <td>2025-07-18</td>
      <td>https://github.com/RECETOX/galaxytools</td>
      <td>1.0.0+galaxy0</td>
      <td>pandas</td>
      <td></td>
      <td>To update</td>
      <td>Proteomics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/main/tools/freqsap</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/freqsap</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>freyja</td>
      <td>freyja_aggregate_plot, freyja_boot, freyja_demix, freyja_variants</td>
      <td>lineage abundances estimation</td>
      <td>2022-07-05</td>
      <td>https://github.com/andersen-lab/Freyja</td>
      <td>2.0.1</td>
      <td>freyja</td>
      <td>2.0.2</td>
      <td>To update</td>
      <td>Metagenomics, Sequence Analysis</td>
      <td>RNA-Seq quantification</td>
      <td>RNA-Seq quantification</td>
      <td>Metagenomics</td>
      <td>Metagenomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/freyja</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/freyja</td>
      <td>freyja</td>
      <td>Freyja</td>
      <td>Recover relative lineage abundances from mixed SARS-CoV-2 samples from a sequencing dataset</td>
      <td></td>
      <td>2</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>58</td>
      <td>58</td>
      <td>24576</td>
      <td>24576</td>
      <td>15</td>
      <td>15</td>
      <td>327</td>
      <td>327</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>24903</td>
      <td>24903</td>
      <td>73</td>
      <td>73</td>
      <td>https://usegalaxy.eu/published/workflow?id=4a764275d4bf9f31, https://usegalaxy.eu/published/workflow?id=d88145e7475e0b7b</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>frogs</td>
      <td>FROGS_affiliation_filters, FROGS_affiliation_postprocess, FROGS_affiliation_stats, FROGS_biom_to_stdBiom, FROGS_biom_to_tsv, FROGS_cluster_filters, FROGS_cluster_stats, FROGS_clustering, FROGS_demultiplex, FROGSSTAT_DESeq2_Preprocess, FROGSSTAT_DESeq2_Visualisation, FROGSFUNC_step2_functions, FROGSFUNC_step3_pathways, FROGSFUNC_step1_placeseqs, FROGS_itsx, FROGS_normalisation, FROGSSTAT_Phyloseq_Alpha_Diversity, FROGSSTAT_Phyloseq_Beta_Diversity, FROGSSTAT_Phyloseq_Sample_Clustering, FROGSSTAT_Phyloseq_Composition_Visualisation, FROGSSTAT_Phyloseq_Import_Data, FROGSSTAT_Phyloseq_Multivariate_Analysis_Of_Variance, FROGSSTAT_Phyloseq_Structure_Visualisation, FROGS_preprocess, FROGS_remove_chimera, FROGS_taxonomic_affiliation, FROGS_Tree, FROGS_tsv_to_biom</td>
      <td>Suite for metabarcoding analysis</td>
      <td>2021-08-16</td>
      <td>http://frogs.toulouse.inrae.fr/</td>
      <td>4.2.0</td>
      <td>frogs</td>
      <td>5.1.0</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>frogs</td>
      <td>https://github.com/geraldinepascal/FROGS-wrappers/</td>
      <td>https://github.com/geraldinepascal/FROGS-wrappers/tree/master/tools/frogs</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>28</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>18</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1271</td>
      <td>1272</td>
      <td>73623</td>
      <td>73661</td>
      <td>73661</td>
      <td>73623</td>
      <td>1272</td>
      <td>1271</td>
      <td>https://usegalaxy.fr/published/workflow?id=048cb90ec1ac7120, https://usegalaxy.fr/published/workflow?id=2ac84b2bd45876cf, https://usegalaxy.fr/published/workflow?id=a5f8c5d37ddfcf0f, https://usegalaxy.fr/published/workflow?id=b46954345641a3b8, https://usegalaxy.fr/published/workflow?id=d68b4aa5337f016b</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>funannotate</td>
      <td>funannotate_annotate, funannotate_clean, funannotate_compare, funannotate_predict, funannotate_sort</td>
      <td>Funannotate is a genome prediction, annotation, and comparison software package.</td>
      <td>2021-08-26</td>
      <td>https://funannotate.readthedocs.io</td>
      <td>1.8.17</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>Genome annotation</td>
      <td>Genome annotation</td>
      <td>Genome annotation</td>
      <td>Genomics</td>
      <td>Genomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/funannotate</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/funannotate</td>
      <td>funannotate</td>
      <td>funannotate</td>
      <td>funannotate is a pipeline for genome annotation (built specifically for fungi, but will also work with higher eukaryotes).</td>
      <td></td>
      <td>3</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>891</td>
      <td>891</td>
      <td>14118</td>
      <td>14118</td>
      <td>209</td>
      <td>209</td>
      <td>1453</td>
      <td>1453</td>
      <td>310</td>
      <td>310</td>
      <td>8597</td>
      <td>8597</td>
      <td>88</td>
      <td>88</td>
      <td>1505</td>
      <td>1505</td>
      <td>25673</td>
      <td>25673</td>
      <td>1498</td>
      <td>1498</td>
      <td>https://usegalaxy.cz/published/workflow?id=932f4901960a595a, https://usegalaxy.eu/published/workflow?id=07c92110c3fffcf8, https://usegalaxy.eu/published/workflow?id=e5becc6d66b2568c, https://usegalaxy.org.au/published/workflow?id=4e161a8f5098cc49, https://usegalaxy.org.au/published/workflow?id=52065a4100b11184, https://usegalaxy.org.au/published/workflow?id=a9068ea858222e2b, https://usegalaxy.org/published/workflow?id=7251c6e0dad8f927, https://workflowhub.eu/workflows/1525?version=1, https://workflowhub.eu/workflows/1656?version=1, https://workflowhub.eu/workflows/358?version=1, https://workflowhub.eu/workflows/754?version=1</td>
      <td>ecology/phylogeny-data-prep, genome-annotation/funannotate</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>gamma</td>
      <td>gamma_s, gamma</td>
      <td>Gene Allele Mutation Microbial Assessment</td>
      <td>2025-08-06</td>
      <td>https://github.com/rastanton/GAMMA</td>
      <td>2.2</td>
      <td>GAMMA</td>
      <td>2.2</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/gamma</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/gamma</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>genomad</td>
      <td>genomad_end_to_end</td>
      <td>Identify virus and plasmid genomes from nucleotide sequences</td>
      <td>2024-06-17</td>
      <td>https://github.com/apcamargo/genomad/</td>
      <td>1.11.1</td>
      <td>genomad</td>
      <td>1.11.2</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Sequence annotation, Taxonomic classification</td>
      <td>Sequence annotation, Taxonomic classification</td>
      <td>Sequence analysis</td>
      <td>Sequence analysis</td>
      <td>ufz</td>
      <td>https://github.com/Helmholtz-UFZ/ufz-galaxy-tools/blob/main/tools/genomad/</td>
      <td>https://github.com/Helmholtz-UFZ/galaxy-tools/tree/main/tools/genomad</td>
      <td>genomad</td>
      <td>geNomad</td>
      <td>geNomad is a tool that identifies virus and plasmid genomes from nucleotide sequences. It provides state-of-the-art classification performance and can be used to quickly find mobile genetic elements from genomes, metagenomes, or metatranscriptomes.</td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>119</td>
      <td>119</td>
      <td>2576</td>
      <td>2576</td>
      <td>20</td>
      <td>20</td>
      <td>71</td>
      <td>71</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2647</td>
      <td>2647</td>
      <td>139</td>
      <td>139</td>
      <td>https://usegalaxy.eu/published/workflow?id=36de05d50c703164, https://usegalaxy.eu/published/workflow?id=62392daea25a3980, https://usegalaxy.eu/published/workflow?id=c62d65832377e376</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>getmlst</td>
      <td>getmlst</td>
      <td>Download MLST datasets by species from pubmlst.org</td>
      <td>2015-12-03</td>
      <td></td>
      <td>0.1.4.1</td>
      <td>srst2</td>
      <td>0.2.0</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>nml</td>
      <td></td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/getmlst</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>gfastats</td>
      <td>gfastats</td>
      <td>Tool for generating sequence statistics and simultaneous genome assembly file manipulation.</td>
      <td>2022-03-08</td>
      <td>https://github.com/vgl-hub/gfastats</td>
      <td>1.3.11</td>
      <td>gfastats</td>
      <td>1.3.11</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Data handling</td>
      <td>Data handling</td>
      <td>Computational biology</td>
      <td>Computational biology</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/gfastats</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/gfastats</td>
      <td>gfastats</td>
      <td>gfastats</td>
      <td>gfastats is a single fast and exhaustive tool for summary statistics and simultaneous genome assembly file manipulation. gfastats also allows seamless fasta/fastq/gfa conversion.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1105</td>
      <td>1105</td>
      <td>19582</td>
      <td>19582</td>
      <td>735</td>
      <td>735</td>
      <td>15352</td>
      <td>15352</td>
      <td>183</td>
      <td>183</td>
      <td>4128</td>
      <td>4128</td>
      <td>5</td>
      <td>5</td>
      <td>453</td>
      <td>453</td>
      <td>39515</td>
      <td>39515</td>
      <td>2028</td>
      <td>2028</td>
      <td>https://usegalaxy.eu/published/workflow?id=002a51bdef00e14d, https://usegalaxy.eu/published/workflow?id=012762cb4acb068b, https://usegalaxy.eu/published/workflow?id=01d42b604ab92d49, https://usegalaxy.eu/published/workflow?id=01fe4c590e608103, https://usegalaxy.eu/published/workflow?id=023a0ee62dd1f593, https://usegalaxy.eu/published/workflow?id=032a7281a86eba0d, https://usegalaxy.eu/published/workflow?id=03bb4c20a1f4b4de, https://usegalaxy.eu/published/workflow?id=03be9d62e8e4a129, https://usegalaxy.eu/published/workflow?id=0444c98a3ca2a3ca, https://usegalaxy.eu/published/workflow?id=04844dddcc4e8aa7, https://usegalaxy.eu/published/workflow?id=069d99391a69baf4, https://usegalaxy.eu/published/workflow?id=075c4126e66def01, https://usegalaxy.eu/published/workflow?id=082bfd9b36e921e8, https://usegalaxy.eu/published/workflow?id=085c3a4c89d137b5, https://usegalaxy.eu/published/workflow?id=089e5e83b1de0511, https://usegalaxy.eu/published/workflow?id=098f43ded1b5fa7f, https://usegalaxy.eu/published/workflow?id=0a3ebee985c26060, https://usegalaxy.eu/published/workflow?id=0c3ce9feb83f7920, https://usegalaxy.eu/published/workflow?id=0cf3c2114ab65858, https://usegalaxy.eu/published/workflow?id=0f044c9f62773a88, https://usegalaxy.eu/published/workflow?id=0f207a15b6f7fa10, https://usegalaxy.eu/published/workflow?id=10150b1a188f6d07, https://usegalaxy.eu/published/workflow?id=1151052f4393852a, https://usegalaxy.eu/published/workflow?id=122da0054fd40adf, https://usegalaxy.eu/published/workflow?id=12a018217ee146ca, https://usegalaxy.eu/published/workflow?id=12b0bd336c76a874, https://usegalaxy.eu/published/workflow?id=1483534f49b826bf, https://usegalaxy.eu/published/workflow?id=16418e8b023ff226, https://usegalaxy.eu/published/workflow?id=17b0a6d3379a4392, https://usegalaxy.eu/published/workflow?id=194a05d42687b524, https://usegalaxy.eu/published/workflow?id=19bb57f8cfa48bb9, https://usegalaxy.eu/published/workflow?id=1ac0a15758277730, https://usegalaxy.eu/published/workflow?id=1af73dc855d0a5ed, https://usegalaxy.eu/published/workflow?id=1bb084f1bc102734, https://usegalaxy.eu/published/workflow?id=1cba6eb4aff8dc61, https://usegalaxy.eu/published/workflow?id=1ce406de61de8492, https://usegalaxy.eu/published/workflow?id=1d1ae39386b5c318, https://usegalaxy.eu/published/workflow?id=1da7ac8eaf891f39, https://usegalaxy.eu/published/workflow?id=1e02a7292e9974b1, https://usegalaxy.eu/published/workflow?id=1e7f4ba6261f979c, https://usegalaxy.eu/published/workflow?id=1f2a1438e41188f2, https://usegalaxy.eu/published/workflow?id=21a3f72851f29925, https://usegalaxy.eu/published/workflow?id=2294ef9515bcf972, https://usegalaxy.eu/published/workflow?id=2497292e98937a5f, https://usegalaxy.eu/published/workflow?id=2671a5f85b2cccfe, https://usegalaxy.eu/published/workflow?id=270b6f82efab28b2, https://usegalaxy.eu/published/workflow?id=2880a32a52d27743, https://usegalaxy.eu/published/workflow?id=29c97d43eeb8ed63, https://usegalaxy.eu/published/workflow?id=2c2f0b4312bae4f4, https://usegalaxy.eu/published/workflow?id=2e418c5810012e5a, https://usegalaxy.eu/published/workflow?id=2e6ed783bdf05d16, https://usegalaxy.eu/published/workflow?id=2e93613c688ea52e, https://usegalaxy.eu/published/workflow?id=3010ecad1a342331, https://usegalaxy.eu/published/workflow?id=3069c17b52decab4, https://usegalaxy.eu/published/workflow?id=309969a261189b5f, https://usegalaxy.eu/published/workflow?id=3192f92ac4d389b8, https://usegalaxy.eu/published/workflow?id=319bc351f7a8e358, https://usegalaxy.eu/published/workflow?id=31c6dcd3a25620df, https://usegalaxy.eu/published/workflow?id=31f8c7c46207432f, https://usegalaxy.eu/published/workflow?id=33a052f9a3d5551d, https://usegalaxy.eu/published/workflow?id=342a5df69951e3d7, https://usegalaxy.eu/published/workflow?id=379642036b6f5a66, https://usegalaxy.eu/published/workflow?id=389d992ff7268247, https://usegalaxy.eu/published/workflow?id=38f12705279b5dfa, https://usegalaxy.eu/published/workflow?id=399048afb4acef3d, https://usegalaxy.eu/published/workflow?id=3a3e2d50fe7a9ee3, https://usegalaxy.eu/published/workflow?id=3b57fc81001667b5, https://usegalaxy.eu/published/workflow?id=3cf3dd9727b7ac92, https://usegalaxy.eu/published/workflow?id=3d047ed984762ce8, https://usegalaxy.eu/published/workflow?id=3dec7c9b02c3d1ee, https://usegalaxy.eu/published/workflow?id=3e255a5b7026d26e, https://usegalaxy.eu/published/workflow?id=3eb6663a66f8e69a, https://usegalaxy.eu/published/workflow?id=3f5414bb38b47fcf, https://usegalaxy.eu/published/workflow?id=416141b245073406, https://usegalaxy.eu/published/workflow?id=44c92c04f1d14631, https://usegalaxy.eu/published/workflow?id=45ae834d842302c3, https://usegalaxy.eu/published/workflow?id=471eb8b4c54f897d, https://usegalaxy.eu/published/workflow?id=475ea16f943cc013, https://usegalaxy.eu/published/workflow?id=4856e3522b2ee059, https://usegalaxy.eu/published/workflow?id=4949751217c0e211, https://usegalaxy.eu/published/workflow?id=497b82d3b92dc076, https://usegalaxy.eu/published/workflow?id=4c3a9b574f37eebb, https://usegalaxy.eu/published/workflow?id=4d41ee83fc0040d4, https://usegalaxy.eu/published/workflow?id=5115299e77df8104, https://usegalaxy.eu/published/workflow?id=5395e76fa828c343, https://usegalaxy.eu/published/workflow?id=54590ccd11ffbf65, https://usegalaxy.eu/published/workflow?id=56523b41ff7eec76, https://usegalaxy.eu/published/workflow?id=57cc33ed64d71982, https://usegalaxy.eu/published/workflow?id=5ad406442ec21345, https://usegalaxy.eu/published/workflow?id=5bb7691b176d1010, https://usegalaxy.eu/published/workflow?id=5bf6718e7d6158d2, https://usegalaxy.eu/published/workflow?id=5c51f26ef8b20633, https://usegalaxy.eu/published/workflow?id=5e1906290f5b4d8a, https://usegalaxy.eu/published/workflow?id=5f29105e842716db, https://usegalaxy.eu/published/workflow?id=652433160b04b232, https://usegalaxy.eu/published/workflow?id=6573f3b1606b7f09, https://usegalaxy.eu/published/workflow?id=6865f33cffca4922, https://usegalaxy.eu/published/workflow?id=69cdd404592535ec, https://usegalaxy.eu/published/workflow?id=6c14b08a7ef7e79a, https://usegalaxy.eu/published/workflow?id=700fa3ab11a8f3e2, https://usegalaxy.eu/published/workflow?id=7137e45ca8dc6e4c, https://usegalaxy.eu/published/workflow?id=71a6d128a09210f0, https://usegalaxy.eu/published/workflow?id=7337d15d121596da, https://usegalaxy.eu/published/workflow?id=792b24fc23bd3071, https://usegalaxy.eu/published/workflow?id=79b5afb125e3c3da, https://usegalaxy.eu/published/workflow?id=7b4a9aebb09b51ca, https://usegalaxy.eu/published/workflow?id=7e383fe34031a9d0, https://usegalaxy.eu/published/workflow?id=7e68cf8a67efa8e9, https://usegalaxy.eu/published/workflow?id=83a1b8949d87e3da, https://usegalaxy.eu/published/workflow?id=84236d781f7dd2ad, https://usegalaxy.eu/published/workflow?id=847403b72dd346ff, https://usegalaxy.eu/published/workflow?id=8a14cda6140acab7, https://usegalaxy.eu/published/workflow?id=8be366ad6a42408b, https://usegalaxy.eu/published/workflow?id=8c7e972020799d57, https://usegalaxy.eu/published/workflow?id=8d00a8b70b0c019c, https://usegalaxy.eu/published/workflow?id=8d64614d1971065a, https://usegalaxy.eu/published/workflow?id=8f105de48f7f1d9d, https://usegalaxy.eu/published/workflow?id=915ffef5b14c871e, https://usegalaxy.eu/published/workflow?id=9207e74a5bf7d8e1, https://usegalaxy.eu/published/workflow?id=92760fc19ef120ce, https://usegalaxy.eu/published/workflow?id=92dafb3d1c06e074, https://usegalaxy.eu/published/workflow?id=9588311136ac8794, https://usegalaxy.eu/published/workflow?id=95f420d8856bf725, https://usegalaxy.eu/published/workflow?id=9667bf36217083b7, https://usegalaxy.eu/published/workflow?id=9aa290ce1927d700, https://usegalaxy.eu/published/workflow?id=9c0b67496dfe52b2, https://usegalaxy.eu/published/workflow?id=9c5a0596956064a6, https://usegalaxy.eu/published/workflow?id=9d209bcdae1a475e, https://usegalaxy.eu/published/workflow?id=9f2d18f0d0896e8b, https://usegalaxy.eu/published/workflow?id=a2daeb488bee0152, https://usegalaxy.eu/published/workflow?id=a37d1e0db16cfb67, https://usegalaxy.eu/published/workflow?id=a3f082ff30357040, https://usegalaxy.eu/published/workflow?id=a53f78aa6866b811, https://usegalaxy.eu/published/workflow?id=a6dd62ab4b9a0725, https://usegalaxy.eu/published/workflow?id=a9c1dca5da7170dd, https://usegalaxy.eu/published/workflow?id=aa6dc57f5c67fb4b, https://usegalaxy.eu/published/workflow?id=aada0111f835ce49, https://usegalaxy.eu/published/workflow?id=ac6eb99f70d35999, https://usegalaxy.eu/published/workflow?id=ade6f36f6a035a73, https://usegalaxy.eu/published/workflow?id=ae74f60cd20d56bc, https://usegalaxy.eu/published/workflow?id=aea88bfca6d1d476, https://usegalaxy.eu/published/workflow?id=aef62f4d0bba003b, https://usegalaxy.eu/published/workflow?id=b211fceaecf41c9e, https://usegalaxy.eu/published/workflow?id=b287bb779cf686ef, https://usegalaxy.eu/published/workflow?id=b4948afacccfefdb, https://usegalaxy.eu/published/workflow?id=b5de94e9cde071ab, https://usegalaxy.eu/published/workflow?id=b8baade87e733fce, https://usegalaxy.eu/published/workflow?id=bac84a7033c3423f, https://usegalaxy.eu/published/workflow?id=bb13f30a5562d30d, https://usegalaxy.eu/published/workflow?id=bc32937d505135ab, https://usegalaxy.eu/published/workflow?id=bc87309a8763d0b4, https://usegalaxy.eu/published/workflow?id=bc8ccd13a07bd9ec, https://usegalaxy.eu/published/workflow?id=bcf99cc0469db276, https://usegalaxy.eu/published/workflow?id=bef0a7d604f2ebef, https://usegalaxy.eu/published/workflow?id=c36560c74416fbbf, https://usegalaxy.eu/published/workflow?id=c4c2e669b86cffbc, https://usegalaxy.eu/published/workflow?id=c4fc3adf8f7ff2bc, https://usegalaxy.eu/published/workflow?id=c508c00e81018dd8, https://usegalaxy.eu/published/workflow?id=c8f94a2e838fc0f1, https://usegalaxy.eu/published/workflow?id=ca39822372847670, https://usegalaxy.eu/published/workflow?id=cab2aa791a814d55, https://usegalaxy.eu/published/workflow?id=caba6d3d7db648e9, https://usegalaxy.eu/published/workflow?id=cc2bc81352d2e2cd, https://usegalaxy.eu/published/workflow?id=cc385bd4376a907c, https://usegalaxy.eu/published/workflow?id=cf4638fbca7c460b, https://usegalaxy.eu/published/workflow?id=d02458f1d7d9c3f4, https://usegalaxy.eu/published/workflow?id=d172f927a489e9f8, https://usegalaxy.eu/published/workflow?id=d17d6029914d666b, https://usegalaxy.eu/published/workflow?id=d1bd5d8597c1d31d, https://usegalaxy.eu/published/workflow?id=d261bbd05889ce75, https://usegalaxy.eu/published/workflow?id=d649ec485c4b3994, https://usegalaxy.eu/published/workflow?id=d924fafd8e042952, https://usegalaxy.eu/published/workflow?id=dc27453c438577f6, https://usegalaxy.eu/published/workflow?id=de9cf573314f416c, https://usegalaxy.eu/published/workflow?id=df99c8a88309918e, https://usegalaxy.eu/published/workflow?id=dfe3b555856c7b44, https://usegalaxy.eu/published/workflow?id=e035ac55551b2744, https://usegalaxy.eu/published/workflow?id=e138640778b7414c, https://usegalaxy.eu/published/workflow?id=e1947b059ef131d2, https://usegalaxy.eu/published/workflow?id=e2f936f1aa43a629, https://usegalaxy.eu/published/workflow?id=e5becc6d66b2568c, https://usegalaxy.eu/published/workflow?id=e7aebe58411d97ed, https://usegalaxy.eu/published/workflow?id=e8f9e1d911683b22, https://usegalaxy.eu/published/workflow?id=e935d2c8ffb52588, https://usegalaxy.eu/published/workflow?id=ec6b2c926fffab5f, https://usegalaxy.eu/published/workflow?id=ec79f14ab91cd933, https://usegalaxy.eu/published/workflow?id=efb58bba65393347, https://usegalaxy.eu/published/workflow?id=f08b29e4cef76cd5, https://usegalaxy.eu/published/workflow?id=f1ca4289ee4f67bb, https://usegalaxy.eu/published/workflow?id=f1e2577699125607, https://usegalaxy.eu/published/workflow?id=f38745208494d4ab, https://usegalaxy.eu/published/workflow?id=f3e17c8dd80def27, https://usegalaxy.eu/published/workflow?id=f578a097b4339025, https://usegalaxy.eu/published/workflow?id=f63127675a07e00a, https://usegalaxy.eu/published/workflow?id=f6a625946dea60fa, https://usegalaxy.eu/published/workflow?id=f6bd10486a0ba17d, https://usegalaxy.eu/published/workflow?id=f7cdaf862e6599c0, https://usegalaxy.eu/published/workflow?id=f882d14fc5dabbb9, https://usegalaxy.eu/published/workflow?id=f91245e580319e65, https://usegalaxy.eu/published/workflow?id=fb015e398373ea8a, https://usegalaxy.eu/published/workflow?id=fc25c1f8c6667c87, https://usegalaxy.eu/published/workflow?id=fd649ac4d4872c81, https://usegalaxy.eu/published/workflow?id=feeb1fa358ccf328, https://usegalaxy.org.au/published/workflow?id=000c2f1884bc4687, https://usegalaxy.org.au/published/workflow?id=02eebe3d909206ab, https://usegalaxy.org.au/published/workflow?id=02f2a556e77eed14, https://usegalaxy.org.au/published/workflow?id=06564b19b0655c19, https://usegalaxy.org.au/published/workflow?id=0810be3d7d2b0322, https://usegalaxy.org.au/published/workflow?id=08fac3b4c128e597, https://usegalaxy.org.au/published/workflow?id=094655df3ccd0b00, https://usegalaxy.org.au/published/workflow?id=0aa316f8c6cff444, https://usegalaxy.org.au/published/workflow?id=0af0ea50f1237952, https://usegalaxy.org.au/published/workflow?id=0c11e1ba5e8fa21b, https://usegalaxy.org.au/published/workflow?id=0cc8c8875df6bccb, https://usegalaxy.org.au/published/workflow?id=0dbb85f727302a72, https://usegalaxy.org.au/published/workflow?id=0df25e2dbd5e8ed5, https://usegalaxy.org.au/published/workflow?id=0edb9c8b767fa74b, https://usegalaxy.org.au/published/workflow?id=10355b2680374a03, https://usegalaxy.org.au/published/workflow?id=11f1564f2bd815d8, https://usegalaxy.org.au/published/workflow?id=1905ec04d5841f09, https://usegalaxy.org.au/published/workflow?id=1ce2a725df3adb74, https://usegalaxy.org.au/published/workflow?id=1dcc1b25c9091fbf, https://usegalaxy.org.au/published/workflow?id=1eb3e1452c672480, https://usegalaxy.org.au/published/workflow?id=1f49c65625b9d2de, https://usegalaxy.org.au/published/workflow?id=2356584eac7fa234, https://usegalaxy.org.au/published/workflow?id=2470cb61ba51288c, https://usegalaxy.org.au/published/workflow?id=269b42605a2a87d2, https://usegalaxy.org.au/published/workflow?id=29ce205d98faf47e, https://usegalaxy.org.au/published/workflow?id=29f0c07c8a24b542, https://usegalaxy.org.au/published/workflow?id=2d45d7138d8715b4, https://usegalaxy.org.au/published/workflow?id=2fee25e188531691, https://usegalaxy.org.au/published/workflow?id=319e9d693fac8dce, https://usegalaxy.org.au/published/workflow?id=328adfa7d90b1d54, https://usegalaxy.org.au/published/workflow?id=3604307ec2835628, https://usegalaxy.org.au/published/workflow?id=36f54eda8568355d, https://usegalaxy.org.au/published/workflow?id=371c9892d261cee2, https://usegalaxy.org.au/published/workflow?id=3a64c1031340d860, https://usegalaxy.org.au/published/workflow?id=3b6e9f03b088bf01, https://usegalaxy.org.au/published/workflow?id=3c30c822058187be, https://usegalaxy.org.au/published/workflow?id=3d47e0de658b3304, https://usegalaxy.org.au/published/workflow?id=3df7f45b7b388f8c, https://usegalaxy.org.au/published/workflow?id=4047f9f242542716, https://usegalaxy.org.au/published/workflow?id=40666ceb68d101ac, https://usegalaxy.org.au/published/workflow?id=40740c32b9a550c9, https://usegalaxy.org.au/published/workflow?id=41282d51f9112552, https://usegalaxy.org.au/published/workflow?id=41795303a88d9e94, https://usegalaxy.org.au/published/workflow?id=4267bd1e4cfd1901, https://usegalaxy.org.au/published/workflow?id=43343fa1896f8278, https://usegalaxy.org.au/published/workflow?id=446ba22c3d2f3367, https://usegalaxy.org.au/published/workflow?id=447d8daefdefafee, https://usegalaxy.org.au/published/workflow?id=455e68f185fec41b, https://usegalaxy.org.au/published/workflow?id=466809eb3392f7fb, https://usegalaxy.org.au/published/workflow?id=47cd0a90609fd557, https://usegalaxy.org.au/published/workflow?id=499245b3a113e06d, https://usegalaxy.org.au/published/workflow?id=4a404aa3e9f29e5a, https://usegalaxy.org.au/published/workflow?id=4b5f80656fc94b88, https://usegalaxy.org.au/published/workflow?id=4cb69c84b73ea9dd, https://usegalaxy.org.au/published/workflow?id=4e7930c7286c6e9e, https://usegalaxy.org.au/published/workflow?id=4ea1049371de111c, https://usegalaxy.org.au/published/workflow?id=4ebf1297793870e7, https://usegalaxy.org.au/published/workflow?id=518d27f68e05a356, https://usegalaxy.org.au/published/workflow?id=51bad198f1fd1215, https://usegalaxy.org.au/published/workflow?id=53cbacc71790c41b, https://usegalaxy.org.au/published/workflow?id=543fb2d9431a7e13, https://usegalaxy.org.au/published/workflow?id=5550585ffc3d8d0e, https://usegalaxy.org.au/published/workflow?id=5b0c0e34394afed1, https://usegalaxy.org.au/published/workflow?id=5f4f874e49515555, https://usegalaxy.org.au/published/workflow?id=5fb44f257e445f56, https://usegalaxy.org.au/published/workflow?id=60c0dec52cdca708, https://usegalaxy.org.au/published/workflow?id=63789b6f82172679, https://usegalaxy.org.au/published/workflow?id=66e530368fd40abd, https://usegalaxy.org.au/published/workflow?id=68b2961117e62890, https://usegalaxy.org.au/published/workflow?id=6c1c11a81a2ab1b8, https://usegalaxy.org.au/published/workflow?id=6c27019d03dd49d8, https://usegalaxy.org.au/published/workflow?id=6c9129acb9d98afe, https://usegalaxy.org.au/published/workflow?id=6de8899c40a90b98, https://usegalaxy.org.au/published/workflow?id=6ec0031406b10635, https://usegalaxy.org.au/published/workflow?id=6ec5728eec9c26d7, https://usegalaxy.org.au/published/workflow?id=6f76c43e6d2d7f74, https://usegalaxy.org.au/published/workflow?id=7028436ef03171cc, https://usegalaxy.org.au/published/workflow?id=7168c01be9997823, https://usegalaxy.org.au/published/workflow?id=73565f6cb4136370, https://usegalaxy.org.au/published/workflow?id=77b2fcf8c805ad5d, https://usegalaxy.org.au/published/workflow?id=7909da83448d8bf4, https://usegalaxy.org.au/published/workflow?id=79bf7b0f5e9052d7, https://usegalaxy.org.au/published/workflow?id=7a97e8ccc332c1e8, https://usegalaxy.org.au/published/workflow?id=8034bd0690b4cbe8, https://usegalaxy.org.au/published/workflow?id=81a5b96452c71a52, https://usegalaxy.org.au/published/workflow?id=82d0876385b5ac60, https://usegalaxy.org.au/published/workflow?id=83441e3de699d157, https://usegalaxy.org.au/published/workflow?id=868775ef66d174a5, https://usegalaxy.org.au/published/workflow?id=86eb0ea9dc230f04, https://usegalaxy.org.au/published/workflow?id=87bbf9408b3f2a32, https://usegalaxy.org.au/published/workflow?id=8926e592f2e60863, https://usegalaxy.org.au/published/workflow?id=897a1255cef5c310, https://usegalaxy.org.au/published/workflow?id=8a6233015db6dd24, https://usegalaxy.org.au/published/workflow?id=8bf8c9929249f37d, https://usegalaxy.org.au/published/workflow?id=8dd86497bc15d981, https://usegalaxy.org.au/published/workflow?id=8e8a5cce9f3f2f67, https://usegalaxy.org.au/published/workflow?id=9016b2be61f89053, https://usegalaxy.org.au/published/workflow?id=91ee716be8bfd508, https://usegalaxy.org.au/published/workflow?id=9642aadb83c4f752, https://usegalaxy.org.au/published/workflow?id=970dc0d8bd74707f, https://usegalaxy.org.au/published/workflow?id=97e55a900db3863b, https://usegalaxy.org.au/published/workflow?id=9804030358c3f844, https://usegalaxy.org.au/published/workflow?id=98d3def7da131b4c, https://usegalaxy.org.au/published/workflow?id=98f660971010d11d, https://usegalaxy.org.au/published/workflow?id=9e0bae0505ee7c07, https://usegalaxy.org.au/published/workflow?id=a02c102e179596cb, https://usegalaxy.org.au/published/workflow?id=a1015b83baa0eb6d, https://usegalaxy.org.au/published/workflow?id=a209f43d8a3cc853, https://usegalaxy.org.au/published/workflow?id=a345bf84eb9bf6b2, https://usegalaxy.org.au/published/workflow?id=a40dadc943a0db1e, https://usegalaxy.org.au/published/workflow?id=a525966251895f8a, https://usegalaxy.org.au/published/workflow?id=a740543c4e9808f1, https://usegalaxy.org.au/published/workflow?id=ac8bdc5a0b2d70cd, https://usegalaxy.org.au/published/workflow?id=ad10cd704268dcf0, https://usegalaxy.org.au/published/workflow?id=ad556db35858e7ec, https://usegalaxy.org.au/published/workflow?id=ae2a397837b232b1, https://usegalaxy.org.au/published/workflow?id=ae6a03240680712a, https://usegalaxy.org.au/published/workflow?id=afab89bc468da8b3, https://usegalaxy.org.au/published/workflow?id=b19fa49405117317, https://usegalaxy.org.au/published/workflow?id=b1f45d2f8d8343e9, https://usegalaxy.org.au/published/workflow?id=b310af289ada88bc, https://usegalaxy.org.au/published/workflow?id=b5131e3a0ef2676e, https://usegalaxy.org.au/published/workflow?id=b53f82aaf32fbaa9, https://usegalaxy.org.au/published/workflow?id=b9e89401313a0cca, https://usegalaxy.org.au/published/workflow?id=b9faf6a3f48fc11b, https://usegalaxy.org.au/published/workflow?id=bcb0be4f4a0e369f, https://usegalaxy.org.au/published/workflow?id=be35847680078c90, https://usegalaxy.org.au/published/workflow?id=be96bb47b6d6435a, https://usegalaxy.org.au/published/workflow?id=c43744d7dd9c0c0b, https://usegalaxy.org.au/published/workflow?id=c556c2270200801d, https://usegalaxy.org.au/published/workflow?id=c6a71573a299eb3a, https://usegalaxy.org.au/published/workflow?id=c7e1f666a3366823, https://usegalaxy.org.au/published/workflow?id=c7f2d6490128dc3c, https://usegalaxy.org.au/published/workflow?id=c853aa707fcddf4e, https://usegalaxy.org.au/published/workflow?id=c92fead7c30c7497, https://usegalaxy.org.au/published/workflow?id=cac0b8a09d59044b, https://usegalaxy.org.au/published/workflow?id=cbf022e349d726d9, https://usegalaxy.org.au/published/workflow?id=cf775b8ff0745456, https://usegalaxy.org.au/published/workflow?id=d298edda13b382e7, https://usegalaxy.org.au/published/workflow?id=d2dcd3b54e5082a6, https://usegalaxy.org.au/published/workflow?id=d417e920ff97e4c1, https://usegalaxy.org.au/published/workflow?id=d42d7b1aa47e870f, https://usegalaxy.org.au/published/workflow?id=d678e57c7f1baf5d, https://usegalaxy.org.au/published/workflow?id=d720c98f7227b6d5, https://usegalaxy.org.au/published/workflow?id=d78879cfaa5948b3, https://usegalaxy.org.au/published/workflow?id=d7f3fc810261e419, https://usegalaxy.org.au/published/workflow?id=d83d5ecf31c20490, https://usegalaxy.org.au/published/workflow?id=d9c181fd954b629a, https://usegalaxy.org.au/published/workflow?id=df4e186069552649, https://usegalaxy.org.au/published/workflow?id=e031af90daa2cadf, https://usegalaxy.org.au/published/workflow?id=e07866576cc2f467, https://usegalaxy.org.au/published/workflow?id=e28ca83f14f4ede5, https://usegalaxy.org.au/published/workflow?id=e2c49586dc941f38, https://usegalaxy.org.au/published/workflow?id=e5f1ce106ea7af12, https://usegalaxy.org.au/published/workflow?id=e6552643a3713b50, https://usegalaxy.org.au/published/workflow?id=ea687c5613fcce27, https://usegalaxy.org.au/published/workflow?id=ea98be0f99aaceb9, https://usegalaxy.org.au/published/workflow?id=ecfb8a08e5a40e15, https://usegalaxy.org.au/published/workflow?id=f06370394b061e08, https://usegalaxy.org.au/published/workflow?id=f38f610e0d95915f, https://usegalaxy.org.au/published/workflow?id=f53d8ade897b3f46, https://usegalaxy.org.au/published/workflow?id=f7a866159f1eaecf, https://usegalaxy.org.au/published/workflow?id=f895914e322c8b77, https://usegalaxy.org.au/published/workflow?id=fa37223950a91f34, https://usegalaxy.org.au/published/workflow?id=fdec6e2fcd5e2cf5, https://usegalaxy.org.au/published/workflow?id=fe1e7fa0bc437ed8, https://usegalaxy.org.au/published/workflow?id=fea0e6c02c2f6274, https://usegalaxy.org.au/published/workflow?id=fff2e49bc90e38ca, https://usegalaxy.org/published/workflow?id=07e873a6d75dfe9f, https://usegalaxy.org/published/workflow?id=0993daadf2db5f51, https://usegalaxy.org/published/workflow?id=10d5e195501603bc, https://usegalaxy.org/published/workflow?id=1245e9435cbc3985, https://usegalaxy.org/published/workflow?id=137d0f05a71d7acb, https://usegalaxy.org/published/workflow?id=1473cdcbd664d147, https://usegalaxy.org/published/workflow?id=155839aeb6340f54, https://usegalaxy.org/published/workflow?id=1b434bfc3aac0022, https://usegalaxy.org/published/workflow?id=1b80302437db5937, https://usegalaxy.org/published/workflow?id=1d24cc949c52e475, https://usegalaxy.org/published/workflow?id=1e264fc06d87899f, https://usegalaxy.org/published/workflow?id=1ea88ca80a7f0038, https://usegalaxy.org/published/workflow?id=1eec2586d1ff908b, https://usegalaxy.org/published/workflow?id=237ca9f36ca266e7, https://usegalaxy.org/published/workflow?id=23eeee48217654d4, https://usegalaxy.org/published/workflow?id=242ef2e9dee35702, https://usegalaxy.org/published/workflow?id=25a9f0693ff6d327, https://usegalaxy.org/published/workflow?id=283bc048cc7a80f5, https://usegalaxy.org/published/workflow?id=284150bb1f296378, https://usegalaxy.org/published/workflow?id=28dbbfb3d480f4b1, https://usegalaxy.org/published/workflow?id=28ddea356ae57da1, https://usegalaxy.org/published/workflow?id=2acbb8c5f72b9f1c, https://usegalaxy.org/published/workflow?id=2b41752b8518a87b, https://usegalaxy.org/published/workflow?id=2c5c627c08563398, https://usegalaxy.org/published/workflow?id=2d994cc02ec05ede, https://usegalaxy.org/published/workflow?id=2f1469dab87a3649, https://usegalaxy.org/published/workflow?id=2ff2961f044cac0f, https://usegalaxy.org/published/workflow?id=305fb9f84b053713, https://usegalaxy.org/published/workflow?id=31387ed5ebc88cd2, https://usegalaxy.org/published/workflow?id=3179c157fa5a2fdd, https://usegalaxy.org/published/workflow?id=31a6145b0832fa48, https://usegalaxy.org/published/workflow?id=325783f21bc5fef6, https://usegalaxy.org/published/workflow?id=349f1bcdbc84045f, https://usegalaxy.org/published/workflow?id=3753dcca4100e976, https://usegalaxy.org/published/workflow?id=3a92253f57e553bd, https://usegalaxy.org/published/workflow?id=3b0d42747f598f82, https://usegalaxy.org/published/workflow?id=3ba73f679558017c, https://usegalaxy.org/published/workflow?id=3d805d7e0f34c727, https://usegalaxy.org/published/workflow?id=3db4eb1ecb588d44, https://usegalaxy.org/published/workflow?id=3db51822f04f4ff7, https://usegalaxy.org/published/workflow?id=3f7550d5d2bf0b96, https://usegalaxy.org/published/workflow?id=408c799822cbf6ea, https://usegalaxy.org/published/workflow?id=426358bd988cb39d, https://usegalaxy.org/published/workflow?id=435f0340bf00c940, https://usegalaxy.org/published/workflow?id=4403ca7fe447a5d7, https://usegalaxy.org/published/workflow?id=44781c8b0bd667ed, https://usegalaxy.org/published/workflow?id=44bcae0e352563f9, https://usegalaxy.org/published/workflow?id=450c2181f89e6b23, https://usegalaxy.org/published/workflow?id=45c0b677c4920d03, https://usegalaxy.org/published/workflow?id=484f8fcf21d67376, https://usegalaxy.org/published/workflow?id=48dd4a4cb5536670, https://usegalaxy.org/published/workflow?id=498a8c6329ca5728, https://usegalaxy.org/published/workflow?id=49a07807fcea8158, https://usegalaxy.org/published/workflow?id=49c9e8161a46667f, https://usegalaxy.org/published/workflow?id=4a0ad87aa5aa558a, https://usegalaxy.org/published/workflow?id=4bf38f5b45faa8a1, https://usegalaxy.org/published/workflow?id=4bf73376661baafd, https://usegalaxy.org/published/workflow?id=4d5cc5010537f172, https://usegalaxy.org/published/workflow?id=529b410bd03b66aa, https://usegalaxy.org/published/workflow?id=578e634b63923e9b, https://usegalaxy.org/published/workflow?id=57e6345fda3e2682, https://usegalaxy.org/published/workflow?id=597c4d54f8a8ebe7, https://usegalaxy.org/published/workflow?id=5c259623a010fe77, https://usegalaxy.org/published/workflow?id=5d1cc6450f699345, https://usegalaxy.org/published/workflow?id=5dfec2a3ae5fa765, https://usegalaxy.org/published/workflow?id=5e38c7b36e4918ad, https://usegalaxy.org/published/workflow?id=5edabfc62f58d8cf, https://usegalaxy.org/published/workflow?id=605160c81d098703, https://usegalaxy.org/published/workflow?id=6341c384ceaf47ed, https://usegalaxy.org/published/workflow?id=653d553ac2fa51c8, https://usegalaxy.org/published/workflow?id=65ccf48b7872e36c, https://usegalaxy.org/published/workflow?id=67252039bce060b2, https://usegalaxy.org/published/workflow?id=683e725f672b7d9b, https://usegalaxy.org/published/workflow?id=6a27c0d224683f73, https://usegalaxy.org/published/workflow?id=6b511b47176893e3, https://usegalaxy.org/published/workflow?id=6fc7497c86ebd8ee, https://usegalaxy.org/published/workflow?id=7072731b51e16923, https://usegalaxy.org/published/workflow?id=73c20d784b7ef81a, https://usegalaxy.org/published/workflow?id=74049914e4a6c791, https://usegalaxy.org/published/workflow?id=768cee70d9aa8cc6, https://usegalaxy.org/published/workflow?id=771345c66d5b9700, https://usegalaxy.org/published/workflow?id=7adf904dea83e699, https://usegalaxy.org/published/workflow?id=7b8ee08ba291d231, https://usegalaxy.org/published/workflow?id=7c9a55db0b0eb178, https://usegalaxy.org/published/workflow?id=7dba74c915f63dc1, https://usegalaxy.org/published/workflow?id=7f6900852a81d121, https://usegalaxy.org/published/workflow?id=7fd68212ce020e6a, https://usegalaxy.org/published/workflow?id=81f1f0fd6d9235fd, https://usegalaxy.org/published/workflow?id=84161f4f8bf77474, https://usegalaxy.org/published/workflow?id=85f521f5c7351406, https://usegalaxy.org/published/workflow?id=864da6858d34df08, https://usegalaxy.org/published/workflow?id=86c67855acbf6124, https://usegalaxy.org/published/workflow?id=88327cb2e0fe3c25, https://usegalaxy.org/published/workflow?id=8ccb42012dcb5db2, https://usegalaxy.org/published/workflow?id=8cfbaa14cff5a633, https://usegalaxy.org/published/workflow?id=8d605fbd33003d26, https://usegalaxy.org/published/workflow?id=8f7c9461910b08f1, https://usegalaxy.org/published/workflow?id=903e35b62946ed76, https://usegalaxy.org/published/workflow?id=90f5b512997c70f8, https://usegalaxy.org/published/workflow?id=91a4daecbb97d03e, https://usegalaxy.org/published/workflow?id=927b8f90ad9603a7, https://usegalaxy.org/published/workflow?id=932b3bc10186f73e, https://usegalaxy.org/published/workflow?id=936ed7447c8e0a2d, https://usegalaxy.org/published/workflow?id=93c1fef4472aa7a8, https://usegalaxy.org/published/workflow?id=950c7245a6c3f6e9, https://usegalaxy.org/published/workflow?id=961dac8100c6c70c, https://usegalaxy.org/published/workflow?id=96bd0007dceec123, https://usegalaxy.org/published/workflow?id=9938a81fb6911412, https://usegalaxy.org/published/workflow?id=9ab68c65337b893b, https://usegalaxy.org/published/workflow?id=9ddfdc9d6cacea88, https://usegalaxy.org/published/workflow?id=9f4c4038d1c519e2, https://usegalaxy.org/published/workflow?id=a4a6d6846c1692f9, https://usegalaxy.org/published/workflow?id=a8cd62e1924a64e1, https://usegalaxy.org/published/workflow?id=a8e0f77cd6ef49f9, https://usegalaxy.org/published/workflow?id=ab0be4897d5355ac, https://usegalaxy.org/published/workflow?id=acada5044d64a30a, https://usegalaxy.org/published/workflow?id=afa89bb0e8966753, https://usegalaxy.org/published/workflow?id=b11c6268704dcc1b, https://usegalaxy.org/published/workflow?id=b12f99e2233fcca6, https://usegalaxy.org/published/workflow?id=b1b4fc510099d592, https://usegalaxy.org/published/workflow?id=b1cfa3c239ef2523, https://usegalaxy.org/published/workflow?id=b5644a5dcf237ef9, https://usegalaxy.org/published/workflow?id=b661fb850c4b4b01, https://usegalaxy.org/published/workflow?id=b6d535f210efa18e, https://usegalaxy.org/published/workflow?id=b7fe3873ee5e3a02, https://usegalaxy.org/published/workflow?id=bb1cb6abfe84a646, https://usegalaxy.org/published/workflow?id=bbc894a79eb0e769, https://usegalaxy.org/published/workflow?id=bd6bfd338ffb9637, https://usegalaxy.org/published/workflow?id=bf79271a438ffa54, https://usegalaxy.org/published/workflow?id=bfd2e7e63d3e6856, https://usegalaxy.org/published/workflow?id=bfef75672b9c3ea4, https://usegalaxy.org/published/workflow?id=bff8bc8d0059274f, https://usegalaxy.org/published/workflow?id=c03ada2070e9ca8a, https://usegalaxy.org/published/workflow?id=c09b8a0c96123cbb, https://usegalaxy.org/published/workflow?id=c0a9a2d53b8660de, https://usegalaxy.org/published/workflow?id=c694856c01ce7cc5, https://usegalaxy.org/published/workflow?id=c8f9e048ddae7d03, https://usegalaxy.org/published/workflow?id=c95e9116e4b7982b, https://usegalaxy.org/published/workflow?id=c983ea8a32662cd6, https://usegalaxy.org/published/workflow?id=ca3f2d9554e9e1c8, https://usegalaxy.org/published/workflow?id=cace9f1359874352, https://usegalaxy.org/published/workflow?id=cb4ccd3068819418, https://usegalaxy.org/published/workflow?id=cc50da71cdf6f244, https://usegalaxy.org/published/workflow?id=ce04658f05029be2, https://usegalaxy.org/published/workflow?id=cfa9e6e722b086c8, https://usegalaxy.org/published/workflow?id=cfd5dc3c7bc1e832, https://usegalaxy.org/published/workflow?id=d08135fa13d1745e, https://usegalaxy.org/published/workflow?id=d2c408df40dc311b, https://usegalaxy.org/published/workflow?id=d2e8a6b39fe88f4c, https://usegalaxy.org/published/workflow?id=d36765ac28c11e1a, https://usegalaxy.org/published/workflow?id=d3f35215e6f45d79, https://usegalaxy.org/published/workflow?id=d4b2b85db47c405e, https://usegalaxy.org/published/workflow?id=d5183b025d0e78dd, https://usegalaxy.org/published/workflow?id=daaded719dcfd53b, https://usegalaxy.org/published/workflow?id=dae5d5e5df5c4dd7, https://usegalaxy.org/published/workflow?id=ddf7a594b9609d68, https://usegalaxy.org/published/workflow?id=e12e642b88a0fe08, https://usegalaxy.org/published/workflow?id=e4dd39237df58884, https://usegalaxy.org/published/workflow?id=e50e36a25bae863c, https://usegalaxy.org/published/workflow?id=e887adf3afc3253d, https://usegalaxy.org/published/workflow?id=ea206593698adbfd, https://usegalaxy.org/published/workflow?id=eaf96fa3c39c210b, https://usegalaxy.org/published/workflow?id=f09033420c6267ab, https://usegalaxy.org/published/workflow?id=f18e6fa5270fcd37, https://usegalaxy.org/published/workflow?id=f1a54b10d4afce8f, https://usegalaxy.org/published/workflow?id=f60027545d40b82d, https://usegalaxy.org/published/workflow?id=f93a4f6b5f3aaaed, https://usegalaxy.org/published/workflow?id=fa26fd51caea53c7, https://usegalaxy.org/published/workflow?id=fa36d9b50e956917, https://usegalaxy.org/published/workflow?id=fb98ef955b3dbd4b, https://usegalaxy.org/published/workflow?id=fc2b48600853f3ea, https://usegalaxy.org/published/workflow?id=fc7b14aafa0b421e, https://usegalaxy.org/published/workflow?id=fd155fcba9bed7ff, https://usegalaxy.org/published/workflow?id=fd40e50c9f1545ce, https://workflowhub.eu/workflows/1054?version=1, https://workflowhub.eu/workflows/1057?version=9, https://workflowhub.eu/workflows/1103?version=3, https://workflowhub.eu/workflows/1104?version=2, https://workflowhub.eu/workflows/1162?version=2, https://workflowhub.eu/workflows/1163?version=2, https://workflowhub.eu/workflows/1164?version=2, https://workflowhub.eu/workflows/1327?version=6, https://workflowhub.eu/workflows/1594?version=1, https://workflowhub.eu/workflows/1641?version=1, https://workflowhub.eu/workflows/605?version=1, https://workflowhub.eu/workflows/606?version=2, https://workflowhub.eu/workflows/608?version=1, https://workflowhub.eu/workflows/612?version=25, https://workflowhub.eu/workflows/625?version=29, https://workflowhub.eu/workflows/633?version=23, https://workflowhub.eu/workflows/641?version=32, https://workflowhub.eu/workflows/642?version=25, https://workflowhub.eu/workflows/643?version=5, https://workflowhub.eu/workflows/645?version=16, https://workflowhub.eu/workflows/701?version=1, https://workflowhub.eu/workflows/702?version=1, https://workflowhub.eu/workflows/751?version=18, https://workflowhub.eu/workflows/788?version=1, https://workflowhub.eu/workflows/789?version=1</td>
      <td>assembly/ERGA-post-assembly-QC, assembly/assembly-decontamination, assembly/vgp_genome_assembly</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>gi2taxonomy</td>
      <td>Fetch Taxonomic Ranks</td>
      <td>Fetch taxonomic representation</td>
      <td>2014-01-27</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>1.1.1</td>
      <td>taxonomy</td>
      <td>0.10.2</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Database search, ID mapping</td>
      <td>Database search, ID mapping</td>
      <td>Taxonomy</td>
      <td>Taxonomy</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/taxonomy/gi2taxonomy</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/taxonomy/gi2taxonomy</td>
      <td>gi2taxonomy</td>
      <td>gi2taxonomy</td>
      <td>The tool fetches taxonomic information for a list of sequence identifiers (i.e. GI numbers, as used by the National Center for Biotechnology Information (NCBI).</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>68</td>
      <td>116</td>
      <td>244</td>
      <td>713</td>
      <td>17</td>
      <td>1798</td>
      <td>113</td>
      <td>14058</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>6</td>
      <td>14777</td>
      <td>357</td>
      <td>1915</td>
      <td>85</td>
      <td>https://usegalaxy.org/published/workflow?id=19c9c8cc3c813bda, https://usegalaxy.org/published/workflow?id=2f7638035fb889ac, https://usegalaxy.org/published/workflow?id=55aa743125c2465d, https://usegalaxy.org/published/workflow?id=59347cc210c4cdbc, https://usegalaxy.org/published/workflow?id=7294f75ad01ca7ed, https://usegalaxy.org/published/workflow?id=7b67ffcf1b7c2232, https://usegalaxy.org/published/workflow?id=85d825931edf4a64, https://usegalaxy.org/published/workflow?id=896d29573ea3e574, https://usegalaxy.org/published/workflow?id=d851f8436497dd1c</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>glimmer_hmm</td>
      <td></td>
      <td>GlimmerHMM is a new gene finder based on a Generalized Hidden Markov Model (GHMM)</td>
      <td>2015-02-28</td>
      <td>https://ccb.jhu.edu/software/glimmerhmm/</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/glimmer_hmm</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/glimmer_hmm</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>goenrichment</td>
      <td>goenrichment, goslimmer</td>
      <td>Performs GO Enrichment analysis.</td>
      <td>2018-12-28</td>
      <td>https://github.com/DanFaria/GOEnrichment</td>
      <td>2.0.1</td>
      <td>goenrichment</td>
      <td>2.0.1</td>
      <td>Up-to-date</td>
      <td>Genome annotation</td>
      <td>Gene-set enrichment analysis</td>
      <td>Gene-set enrichment analysis</td>
      <td>Transcriptomics</td>
      <td>Transcriptomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/goenrichment</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/goenrichment</td>
      <td>goenrichment</td>
      <td>GOEnrichment</td>
      <td>GOEnrichment is a tool for performing GO enrichment analysis of gene sets, such as those obtained from RNA-seq or Microarray experiments, to help characterize them at the functional level. It is available in Galaxy Europe and as a stand-alone tool.GOEnrichment is flexible in that it allows the user to use any version of the Gene Ontology and any GO annotation file they desire. To enable the use of GO slims, it is accompanied by a sister tool GOSlimmer, which can convert annotation files from full GO to any specified GO slim.The tool features an optional graph clustering algorithm to reduce the redundancy in the set of enriched GO terms and simplify its output.It was developed by the BioData.pt / ELIXIR-PT team at the Instituto Gulbenkian de Cincia.</td>
      <td></td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>799</td>
      <td>928</td>
      <td>6000</td>
      <td>7557</td>
      <td>1081</td>
      <td>1196</td>
      <td>8060</td>
      <td>9309</td>
      <td>230</td>
      <td>233</td>
      <td>1602</td>
      <td>1647</td>
      <td>5</td>
      <td>6</td>
      <td>39</td>
      <td>44</td>
      <td>18557</td>
      <td>15701</td>
      <td>2363</td>
      <td>2115</td>
      <td>https://usegalaxy.eu/published/workflow?id=5f053303601cc630, https://usegalaxy.eu/published/workflow?id=7136ce9f18c74853, https://usegalaxy.org.au/published/workflow?id=d04b26372b28e8ed, https://workflowhub.eu/workflows/1515?version=1, https://workflowhub.eu/workflows/1667?version=1</td>
      <td>single-cell/GO-enrichment, transcriptomics/goenrichment</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>goseq</td>
      <td>goseq</td>
      <td>goseq does selection-unbiased testing for category enrichment amongst differentially expressed (DE) genes for RNA-seq data</td>
      <td>2016-11-12</td>
      <td>https://bioconductor.org/packages/release/bioc/html/goseq.html</td>
      <td>1.50.0</td>
      <td>bioconductor-goseq</td>
      <td>1.58.0</td>
      <td>To update</td>
      <td>Statistics, RNA, Micro-array Analysis</td>
      <td>Gene functional annotation</td>
      <td>Gene functional annotation</td>
      <td>RNA-Seq</td>
      <td>RNA-Seq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/goseq</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/goseq</td>
      <td>goseq</td>
      <td>GOseq</td>
      <td>Detect Gene Ontology and/or other user defined categories which are over/under represented in RNA-seq data.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>3196</td>
      <td>3921</td>
      <td>23708</td>
      <td>28273</td>
      <td>3528</td>
      <td>4089</td>
      <td>26762</td>
      <td>31254</td>
      <td>640</td>
      <td>680</td>
      <td>4461</td>
      <td>5015</td>
      <td>103</td>
      <td>104</td>
      <td>751</td>
      <td>784</td>
      <td>65326</td>
      <td>55682</td>
      <td>8794</td>
      <td>7467</td>
      <td>https://usegalaxy.eu/published/workflow?id=032ad839dab7ae70, https://usegalaxy.eu/published/workflow?id=12bf85cdc8102bb5, https://usegalaxy.eu/published/workflow?id=20ada53472782c9b, https://usegalaxy.eu/published/workflow?id=63c91ea322cd2819, https://usegalaxy.eu/published/workflow?id=732fa536cdc70c96, https://usegalaxy.eu/published/workflow?id=9c7a218993788493, https://usegalaxy.eu/published/workflow?id=c1a6d11e1eab6a42, https://usegalaxy.eu/published/workflow?id=e7e26967cf6cb8d8, https://usegalaxy.eu/published/workflow?id=fa83e26e05418370, https://usegalaxy.org.au/published/workflow?id=4f0814ec9f9f8425, https://usegalaxy.org.au/published/workflow?id=5490661d0153962b, https://usegalaxy.org.au/published/workflow?id=71bad95f72c9f30d, https://usegalaxy.org.au/published/workflow?id=75113a1b41904d2a, https://usegalaxy.org.au/published/workflow?id=9f6e62f465589ca3, https://usegalaxy.org.au/published/workflow?id=e1fd89e7248b6661, https://usegalaxy.org/published/workflow?id=420a332da24fb9e9, https://usegalaxy.org/published/workflow?id=742120336d5ff997, https://workflowhub.eu/workflows/1194?version=1, https://workflowhub.eu/workflows/1684?version=1, https://workflowhub.eu/workflows/1699?version=1, https://workflowhub.eu/workflows/1708?version=1, https://workflowhub.eu/workflows/1713?version=2, https://workflowhub.eu/workflows/689?version=1</td>
      <td>transcriptomics/minerva-pathways, transcriptomics/ref-based, transcriptomics/rna-seq-analysis-clustering-viz, transcriptomics/rna-seq-genes-to-pathways</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>graphlan</td>
      <td>graphlan, graphlan_annotate</td>
      <td>GraPhlAn is a software tool for producing high-quality circular representations of taxonomic and phylogenetic trees</td>
      <td>2017-03-03</td>
      <td>https://github.com/biobakery/graphlan</td>
      <td></td>
      <td>graphlan</td>
      <td>1.1.3</td>
      <td>To update</td>
      <td>Metagenomics, Graphics, Phylogenetics</td>
      <td>Phylogenetic inference, Phylogenetic tree visualisation, Phylogenetic tree editing, Taxonomic classification</td>
      <td>Phylogenetic inference, Phylogenetic tree visualisation, Phylogenetic tree editing, Taxonomic classification</td>
      <td>Metagenomics, Phylogenetics, Phylogenomics, Cladistics</td>
      <td>Metagenomics, Phylogenetics, Phylogenomics, Cladistics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/graphlan</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/graphlan</td>
      <td>graphlan</td>
      <td>GraPhlAn</td>
      <td>GraPhlAn is a software tool for producing high-quality circular representations of taxonomic and phylogenetic trees. GraPhlAn focuses on concise, integrative, informative, and publication-ready representations of phylogenetically- and taxonomically-driven investigation.</td>
      <td></td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>799</td>
      <td>940</td>
      <td>11872</td>
      <td>13928</td>
      <td>325</td>
      <td>325</td>
      <td>1773</td>
      <td>1773</td>
      <td>202</td>
      <td>207</td>
      <td>1432</td>
      <td>1470</td>
      <td>21</td>
      <td>21</td>
      <td>90</td>
      <td>90</td>
      <td>17261</td>
      <td>15167</td>
      <td>1493</td>
      <td>1347</td>
      <td>https://usegalaxy.eu/published/workflow?id=05f5f040337786bf, https://usegalaxy.eu/published/workflow?id=096b75501c8e0888, https://usegalaxy.eu/published/workflow?id=19de5cbae7f48103, https://usegalaxy.eu/published/workflow?id=2cf09adb1ed6c3ec, https://usegalaxy.eu/published/workflow?id=4d32b1b3ac0a7161, https://usegalaxy.eu/published/workflow?id=4f21205e71cabd37, https://usegalaxy.eu/published/workflow?id=5f0faac42b0a5062, https://usegalaxy.eu/published/workflow?id=713a5485e2007f90, https://usegalaxy.eu/published/workflow?id=79b4ae4403772098, https://usegalaxy.eu/published/workflow?id=7ab70660e6235cf0, https://usegalaxy.eu/published/workflow?id=87cc0cd143a12c41, https://usegalaxy.eu/published/workflow?id=994e5ff1df7d6c9e, https://usegalaxy.eu/published/workflow?id=9cbba5de2cbeeb0f, https://usegalaxy.eu/published/workflow?id=a8f829deb69b30ab, https://usegalaxy.eu/published/workflow?id=b5f848dc8e96663c, https://usegalaxy.eu/published/workflow?id=c01ef8354b585a1f, https://usegalaxy.eu/published/workflow?id=c3aa516e78ca4191, https://usegalaxy.eu/published/workflow?id=d05c6f0f3205fbd2, https://usegalaxy.eu/published/workflow?id=dfc2c53ac3103eff, https://usegalaxy.eu/published/workflow?id=ec50dd6d56f50db1, https://usegalaxy.eu/published/workflow?id=ed0ad728cf21702b, https://usegalaxy.eu/published/workflow?id=fdca6c9d394986bf, https://usegalaxy.fr/published/workflow?id=09e8293456ee6d99, https://usegalaxy.fr/published/workflow?id=c4f6062e51d0f45e, https://workflowhub.eu/workflows/1451?version=2, https://workflowhub.eu/workflows/1466?version=2</td>
      <td>microbiome/metatranscriptomics, microbiome/metatranscriptomics-short</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>groot</td>
      <td>groot</td>
      <td>GROOT is a tool to type Antibiotic Resistance Genes (ARGs) in metagenomic samples</td>
      <td>2024-05-14</td>
      <td>https://github.com/will-rowe/groot</td>
      <td>1.1.2</td>
      <td>groot</td>
      <td>1.1.2</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Antimicrobial resistance prediction</td>
      <td>Antimicrobial resistance prediction</td>
      <td>Metagenomics, Microbiology, Genomics</td>
      <td>Metagenomics, Microbiology</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/groot</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/groot</td>
      <td>groot</td>
      <td>GROOT</td>
      <td>Indexed variation graphs for efficient and accurate resistome profiling.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>37</td>
      <td>37</td>
      <td>44</td>
      <td>44</td>
      <td>8</td>
      <td>8</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>gtdb_to_taxdump</td>
      <td>gtdb_to_taxdump</td>
      <td>Convert GTDB taxonomy to NCBI taxdump format</td>
      <td>2024-08-25</td>
      <td>https://github.com/nick-youngblut/gtdb_to_taxdump</td>
      <td>0.1.9</td>
      <td>gtdb_to_taxdump</td>
      <td>0.1.9</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Data handling, Mapping, Generation</td>
      <td>Data handling, Mapping, Generation</td>
      <td>Computational biology</td>
      <td>Computational biology</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/gtdb_to_taxdump</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/gtdb_to_taxdump</td>
      <td>gtdb_to_taxdump</td>
      <td>gtdb_to_taxdump</td>
      <td>Tool with multiple functions. Main functions are to create a DIAMOND database from the GTDB taxonomy data or create a NCBI taxdump format out of this data. This tool can also create a mapping between the taxonomy classification between GTDB and NCBI.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>27</td>
      <td>27</td>
      <td>1127</td>
      <td>1127</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1127</td>
      <td>1127</td>
      <td>27</td>
      <td>27</td>
      <td>https://usegalaxy.eu/published/workflow?id=cf0159c14418525e, https://usegalaxy.eu/published/workflow?id=e4dabaf57668dbd1</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>gtdbtk</td>
      <td>gtdbtk_classify_wf</td>
      <td>GTDB-Tk is a software tool kit for assigning objective taxonomic classifications to bacterial and archaeal genomesbased on the Genome Database Taxonomy GTDB. It is designed to work with recent advances that allow hundreds orthousands of metagenome-assembled genomes (MAGs) to be obtained directly from environmental samples. It can alsobe applied to isolate and single-cell genomes.</td>
      <td>2022-12-13</td>
      <td>https://github.com/Ecogenomics/GTDBTk</td>
      <td>2.6.1</td>
      <td>gtdbtk</td>
      <td>2.6.1</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Genome alignment, Taxonomic classification, Sequence assembly, Query and retrieval</td>
      <td>Genome alignment, Taxonomic classification, Sequence assembly, Query and retrieval</td>
      <td>Metagenomics, Taxonomy, Phylogenetics, Database management, Proteins</td>
      <td>Metagenomics, Taxonomy, Phylogenetics, Database management, Proteins</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/gtdbtk</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/gtdbtk</td>
      <td>GTDB-Tk</td>
      <td>GTDB-Tk</td>
      <td>a toolkit to classify genomes with the Genome Taxonomy Database.GTDB-Tk: a toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes.GTDB-Tk is a software toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes based on the Genome Database Taxonomy GTDB. It is designed to work with recent advances that allow hundreds or thousands of metagenome-assembled genomes (MAGs) to be obtained directly from environmental samples. It can also be applied to isolate and single-cell genomes. The GTDB-Tk is open source and released under the GNU General Public License (Version 3).</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>462</td>
      <td>462</td>
      <td>3982</td>
      <td>3982</td>
      <td>305</td>
      <td>305</td>
      <td>1566</td>
      <td>1566</td>
      <td>177</td>
      <td>177</td>
      <td>1095</td>
      <td>1095</td>
      <td>15</td>
      <td>15</td>
      <td>1439</td>
      <td>1439</td>
      <td>8082</td>
      <td>8082</td>
      <td>959</td>
      <td>959</td>
      <td>https://usegalaxy.eu/published/workflow?id=0563f58718be932d, https://usegalaxy.eu/published/workflow?id=17d2e76147fc940c, https://usegalaxy.eu/published/workflow?id=1e5f1d9a51032e44, https://usegalaxy.eu/published/workflow?id=7ecd92cb382d8168, https://usegalaxy.eu/published/workflow?id=c62d65832377e376, https://usegalaxy.eu/published/workflow?id=e4dabaf57668dbd1, https://usegalaxy.org/published/workflow?id=33d90e718ce500ef, https://usegalaxy.org/published/workflow?id=8d229c0f58a07ebe, https://workflowhub.eu/workflows/1352?version=4</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>gubbins</td>
      <td>gubbins</td>
      <td>Gubbins - bacterial recombination detection</td>
      <td>2017-06-22</td>
      <td></td>
      <td>3.2.1</td>
      <td>gubbins</td>
      <td>3.4.3</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Genotyping, Phylogenetic inference, Ancestral reconstruction</td>
      <td>Genotyping, Phylogenetic inference, Ancestral reconstruction</td>
      <td>Phylogeny, Genotype and phenotype, Whole genome sequencing</td>
      <td>Phylogeny, Genotype and phenotype, Whole genome sequencing</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/gubbins</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/gubbins</td>
      <td>gubbins</td>
      <td>Gubbins</td>
      <td>Gubbins is a tool for rapid phylogenetic analysis of large samples of recombinant bacterial whole genome sequences.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>402</td>
      <td>428</td>
      <td>4731</td>
      <td>5118</td>
      <td>268</td>
      <td>288</td>
      <td>3847</td>
      <td>4061</td>
      <td>250</td>
      <td>317</td>
      <td>1977</td>
      <td>2858</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>12037</td>
      <td>10555</td>
      <td>1033</td>
      <td>920</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>guppy_basecaller</td>
      <td>guppy-basecaller</td>
      <td>A wrapper for the guppy basecaller tool from Oxford Nanopore Technologies</td>
      <td>2020-11-18</td>
      <td>http://artbio.fr</td>
      <td>6.5.7+galaxy0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>Nanopore</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/guppy</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/guppy</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>halla</td>
      <td>halla</td>
      <td>Hierarchical All-against-All association</td>
      <td>2025-11-05</td>
      <td>https://github.com/biobakery/halla</td>
      <td>0.8.40</td>
      <td>halla</td>
      <td>0.8.40</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Statistical calculation, Gene expression profiling, Essential dynamics</td>
      <td>Statistical calculation, Gene expression profiling, Essential dynamics</td>
      <td>Microbial ecology, Microarray experiment, Metabolomics, Phylogeny, Epigenetics</td>
      <td>Microbial ecology, Microarray experiment, Metabolomics, Phylogeny, Epigenetics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/halla</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/halla</td>
      <td>halla</td>
      <td>HAllA</td>
      <td>High-sensitivity pattern discovery in large, paired multi-omic datasets.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>hamronization</td>
      <td>hamronize_summarize, hamronize_tool</td>
      <td>Convert AMR gene detection tool output to hAMRonization specification format.</td>
      <td>2021-02-02</td>
      <td>https://github.com/pha4ge/hAMRonization</td>
      <td>1.1.4</td>
      <td>hamronization</td>
      <td>1.1.9</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Data handling, Antimicrobial resistance prediction, Parsing</td>
      <td>Antimicrobial resistance prediction, Parsing</td>
      <td>Public health and epidemiology, Microbiology, Bioinformatics</td>
      <td>Public health and epidemiology, Microbiology, Bioinformatics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hamronization</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hamronization</td>
      <td>hamronization</td>
      <td>hAMRonization</td>
      <td>Parse multiple Antimicrobial Resistance Analysis Reports into a common data structure</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>254</td>
      <td>254</td>
      <td>18459</td>
      <td>18459</td>
      <td>1</td>
      <td>1</td>
      <td>20</td>
      <td>20</td>
      <td>1</td>
      <td>1</td>
      <td>21</td>
      <td>21</td>
      <td>6</td>
      <td>6</td>
      <td>191</td>
      <td>191</td>
      <td>18691</td>
      <td>18691</td>
      <td>262</td>
      <td>262</td>
      <td>https://usegalaxy.eu/published/workflow?id=eb5ab8bc82569eee, https://workflowhub.eu/workflows/1189?version=2, https://workflowhub.eu/workflows/407?version=1, https://workflowhub.eu/workflows/470?version=1</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>hifiasm_meta</td>
      <td>hifiasm_meta</td>
      <td>A hifiasm fork for metagenome assembly using Hifi reads.</td>
      <td>2023-01-18</td>
      <td>https://github.com/xfengnefx/hifiasm-meta</td>
      <td>0.3.1</td>
      <td>hifiasm_meta</td>
      <td>hamtv0.3.1</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly, Metagenomics</td>
      <td>Sequence assembly, Metagenomics</td>
      <td>galaxy-australia</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hifiasm_meta</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hifiasm_meta</td>
      <td>hifiasm-meta</td>
      <td>Hifiasm-meta</td>
      <td>Hifiasm_meta - de novo metagenome assembler, based on hifiasm, a haplotype-resolved de novo assembler for PacBio Hifi reads.</td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>40</td>
      <td>40</td>
      <td>229</td>
      <td>229</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>16</td>
      <td>16</td>
      <td>368</td>
      <td>368</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>597</td>
      <td>597</td>
      <td>56</td>
      <td>56</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>hivtrace</td>
      <td>hivtrace</td>
      <td>An application that identifies potential transmission clusters within a supplied FASTA file with an option to find potential links against the Los Alamos HIV Sequence Database.</td>
      <td>2015-12-10</td>
      <td></td>
      <td>1.0.1</td>
      <td>hivtrace</td>
      <td>1.5.0</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/tools/hivtrace</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/hivtrace</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>hmmer3</td>
      <td>hmmer_alimask, hmmer_hmmalign, hmmer_hmmbuild, hmmer_hmmconvert, hmmer_hmmemit, hmmer_hmmfetch, hmmer_hmmscan, hmmer_hmmsearch, hmmer_jackhmmer, hmmer_nhmmer, hmmer_nhmmscan, hmmer_phmmer</td>
      <td>HMMER is used for searching sequence databases for homologs of proteinsequences, and for making protein sequence alignments. It implementsmethods using probabilistic models called profile hidden Markov models(profile HMMs).</td>
      <td>2015-02-16</td>
      <td>http://hmmer.org/</td>
      <td>3.4</td>
      <td>hmmer</td>
      <td>3.4</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Formatting, Multiple sequence alignment, Sequence profile generation, Format validation, Conversion, Sequence generation, Data retrieval, Statistical calculation, Database search, Formatting, Database search, Database search, Probabilistic sequence generation, Statistical calculation, Statistical calculation, Sequence database search, Formatting, Sequence database search, Database search, Sequence database search</td>
      <td>Formatting, Multiple sequence alignment, Sequence profile generation, Format validation, Conversion, Data retrieval, Statistical calculation, Formatting, Probabilistic sequence generation, Statistical calculation, Statistical calculation, Sequence database search, Formatting, Sequence database search, Sequence database search</td>
      <td>Sequence analysis, Sequence sites, features and motifs, Gene and protein families</td>
      <td>Sequence analysis, Sequence sites, features and motifs, Gene and protein families</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hmmer3</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hmmer3</td>
      <td>hmmer3</td>
      <td>HMMER3</td>
      <td>This tool is used for searching sequence databases for homologs of protein sequences, and for making protein sequence alignments. It implements methods using probabilistic models called profile hidden Markov models. The new HMMER3 project, HMMER is now as fast as BLAST for protein search.</td>
      <td></td>
      <td>0</td>
      <td>12</td>
      <td>12</td>
      <td>12</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>12</td>
      <td>12</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>12</td>
      <td>12</td>
      <td>12</td>
      <td>0</td>
      <td>380</td>
      <td>403</td>
      <td>36302</td>
      <td>43262</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>83</td>
      <td>84</td>
      <td>4731</td>
      <td>4887</td>
      <td>97</td>
      <td>97</td>
      <td>540</td>
      <td>543</td>
      <td>48692</td>
      <td>41573</td>
      <td>584</td>
      <td>560</td>
      <td>https://usegalaxy.eu/published/workflow?id=02631612a43a4d9b, https://usegalaxy.eu/published/workflow?id=0ab3e47ebd3b520e, https://usegalaxy.eu/published/workflow?id=24eaa682897379c8, https://usegalaxy.eu/published/workflow?id=3eaa913306c16146, https://usegalaxy.eu/published/workflow?id=58955868b504c60d, https://usegalaxy.eu/published/workflow?id=5be158931abca947, https://usegalaxy.eu/published/workflow?id=6158c3b5fc12044e, https://usegalaxy.eu/published/workflow?id=6416cfdbd5273063, https://usegalaxy.eu/published/workflow?id=667bac8d7453e5f1, https://usegalaxy.eu/published/workflow?id=8deb08d4f228e198, https://usegalaxy.eu/published/workflow?id=9a60ecf98d4f8134, https://usegalaxy.eu/published/workflow?id=d57d41e306241396, https://usegalaxy.fr/published/workflow?id=5ec6d378e60f7ccc, https://usegalaxy.fr/published/workflow?id=7ebb54181b404733, https://usegalaxy.fr/published/workflow?id=b131218bec7eacb4, https://usegalaxy.fr/published/workflow?id=d425a77157cae2ee, https://workflowhub.eu/workflows/100?version=1, https://workflowhub.eu/workflows/37?version=1, https://workflowhub.eu/workflows/38?version=1, https://workflowhub.eu/workflows/39?version=1, https://workflowhub.eu/workflows/40?version=1</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>humann</td>
      <td>humann, humann_barplot, humann_join_tables, humann_reduce_table, humann_regroup_table, humann_rename_table, humann_renorm_table, humann_split_stratified_table, humann_split_table, humann_unpack_pathways</td>
      <td>HUMAnN for functionally profiling metagenomes and metatranscriptomes at species-level resolution</td>
      <td>2021-05-12</td>
      <td>http://huttenhower.sph.harvard.edu/humann</td>
      <td>3.9</td>
      <td>humann</td>
      <td>3.9</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Species frequency estimation, Taxonomic classification, Phylogenetic analysis</td>
      <td>Species frequency estimation, Taxonomic classification, Phylogenetic analysis</td>
      <td>Metagenomics, Phylogenomics</td>
      <td>Metagenomics, Phylogenomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/humann</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/humann</td>
      <td>humann</td>
      <td>humann</td>
      <td>HUMAnN is a pipeline for efficiently and accurately profiling the presence/absence and abundance of microbial pathways in a community from metagenomic or metatranscriptomic sequencing data (typically millions of short DNA/RNA reads). This process, referred to as functional profiling, aims to describe the metabolic potential of a microbial community and its members. More generally, functional profiling answers the question What are the microbes in my community-of-interest doing (or are capable of doing)?</td>
      <td></td>
      <td>6</td>
      <td>10</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>0</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>795</td>
      <td>795</td>
      <td>52483</td>
      <td>52483</td>
      <td>123</td>
      <td>123</td>
      <td>3171</td>
      <td>3171</td>
      <td>122</td>
      <td>122</td>
      <td>4812</td>
      <td>4812</td>
      <td>17</td>
      <td>17</td>
      <td>987</td>
      <td>987</td>
      <td>61453</td>
      <td>61453</td>
      <td>1057</td>
      <td>1057</td>
      <td>https://usegalaxy.eu/published/workflow?id=21a17d6c473a88c0, https://usegalaxy.eu/published/workflow?id=23dd2253da51e22e, https://usegalaxy.eu/published/workflow?id=3118d383c81cf805, https://usegalaxy.eu/published/workflow?id=4d32b1b3ac0a7161, https://usegalaxy.eu/published/workflow?id=5ce8a472e6e95c60, https://usegalaxy.eu/published/workflow?id=5f0faac42b0a5062, https://usegalaxy.eu/published/workflow?id=713a5485e2007f90, https://usegalaxy.eu/published/workflow?id=87cc0cd143a12c41, https://usegalaxy.eu/published/workflow?id=9472be79da012999, https://usegalaxy.eu/published/workflow?id=a7a6d8ecc2795f58, https://usegalaxy.eu/published/workflow?id=a8f829deb69b30ab, https://usegalaxy.eu/published/workflow?id=b14845359b702444, https://usegalaxy.eu/published/workflow?id=cb2df493f5e334f7, https://usegalaxy.eu/published/workflow?id=fdca6c9d394986bf, https://usegalaxy.fr/published/workflow?id=c4f6062e51d0f45e, https://usegalaxy.fr/published/workflow?id=fd733565098d76c9, https://workflowhub.eu/workflows/1447?version=2, https://workflowhub.eu/workflows/1456?version=2, https://workflowhub.eu/workflows/1466?version=2</td>
      <td>microbiome/metatranscriptomics, microbiome/metatranscriptomics-short</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>hyphy</td>
      <td>hyphy_absrel, hyphy_annotate, hyphy_bgm, hyphy_busted, hyphy_cfel, hyphy_conv, hyphy_fade, hyphy_fel, hyphy_fubar, hyphy_gard, hyphy_meme, hyphy_prime, hyphy_relax, hyphy_slac, hyphy_sm19, hyphy_strike_ambigs</td>
      <td>Hypothesis Testing using Phylogenies</td>
      <td>2018-06-08</td>
      <td>http://www.hyphy.org</td>
      <td>2.5.83</td>
      <td>hyphy</td>
      <td>2.5.93</td>
      <td>To update</td>
      <td>Phylogenetics</td>
      <td>Statistical calculation</td>
      <td>Statistical calculation</td>
      <td>Phylogeny, Small molecules, Molecular interactions, pathways and networks</td>
      <td>Phylogeny, Small molecules, Molecular interactions, pathways and networks</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hyphy</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hyphy</td>
      <td>HyPhy</td>
      <td>HyPhy</td>
      <td>Software package for the analysis of genetic sequences using techniques in phylogenetics, molecular evolution, and machine learning.</td>
      <td></td>
      <td>16</td>
      <td>2</td>
      <td>16</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>12</td>
      <td>16</td>
      <td>12</td>
      <td>0</td>
      <td>88</td>
      <td>94</td>
      <td>10953</td>
      <td>11906</td>
      <td>78</td>
      <td>108</td>
      <td>4725</td>
      <td>8358</td>
      <td>33</td>
      <td>42</td>
      <td>815</td>
      <td>909</td>
      <td>3</td>
      <td>5</td>
      <td>22</td>
      <td>27</td>
      <td>21200</td>
      <td>16515</td>
      <td>249</td>
      <td>202</td>
      <td>https://usegalaxy.eu/published/workflow?id=1d40d6deb2da04cc, https://usegalaxy.eu/published/workflow?id=e7f03b4bd9b3d6fd, https://usegalaxy.fr/published/workflow?id=297b71156b297a73, https://usegalaxy.org.au/published/workflow?id=42137e36791652bc, https://usegalaxy.org/published/workflow?id=2d1ca299c1b058f6, https://usegalaxy.org/published/workflow?id=a46ad0b9240d1a13, https://usegalaxy.org/published/workflow?id=e1cc0886cd27b92e</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>hypo</td>
      <td>hypo</td>
      <td>Super Fast &amp; Accurate Polisher for Long Read Genome Assemblies</td>
      <td>2021-11-15</td>
      <td>https://github.com/kensung-lab/hypo</td>
      <td>1.0.3</td>
      <td>hypo</td>
      <td>1.0.3</td>
      <td>Up-to-date</td>
      <td>Assembly</td>
      <td>Optimisation and refinement, Genome assembly</td>
      <td>Optimisation and refinement, Genome assembly</td>
      <td>Sequence assembly, Genomics</td>
      <td>Sequence assembly, Genomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hypo</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hypo</td>
      <td>HyPo</td>
      <td>HyPo</td>
      <td>HyPo, a Hybrid Polisher, utilizes short as well as long reads within a single run to polish a long reads assembly of small and large genomes.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>107</td>
      <td>107</td>
      <td>575</td>
      <td>575</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>575</td>
      <td>575</td>
      <td>107</td>
      <td>107</td>
      <td>https://usegalaxy.eu/published/workflow?id=83a1b8949d87e3da, https://workflowhub.eu/workflows/788?version=1, https://workflowhub.eu/workflows/789?version=1</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>icescreen</td>
      <td>icescreen</td>
      <td>identification of ICEs and IMEs in Bacillota genomes.</td>
      <td>2022-01-04</td>
      <td>https://forgemia.inra.fr/ices_imes_analysis/icescreen</td>
      <td>1.3.3</td>
      <td>icescreen</td>
      <td>1.3.3</td>
      <td>Up-to-date</td>
      <td>Genome annotation</td>
      <td>Database search, Protein feature detection</td>
      <td>Database search, Protein feature detection</td>
      <td>Mobile genetic elements, Sequence sites, features and motifs, Genomics, Molecular interactions, pathways and networks, Structural variation</td>
      <td>Mobile genetic elements, Sequence sites, features and motifs, Genomics, Molecular interactions, pathways and networks, Structural variation</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/icescreen</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/icescreen</td>
      <td>icescreen</td>
      <td>ICEscreen</td>
      <td>A tool to detect Firmicute ICEs and IMEs, isolated or enclosed in composite structures.</td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>153</td>
      <td>153</td>
      <td>4</td>
      <td>4</td>
      <td>15</td>
      <td>15</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>168</td>
      <td>168</td>
      <td>7</td>
      <td>7</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>idba</td>
      <td>idba_hybrid, idba_tran, idba_ud</td>
      <td>Wrappers for the idba assembler variants.</td>
      <td>2018-04-26</td>
      <td>https://i.cs.hku.hk/~alse/hkubrg/projects/index.html</td>
      <td></td>
      <td>idba</td>
      <td>1.1.3</td>
      <td>To update</td>
      <td>Assembly</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/idba_ud</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/idba_ud</td>
      <td>idba</td>
      <td>IDBA</td>
      <td>A short read assembler based on iterative De Bruijn graph. It is developed under 64-bit Linux, but should be suitable for all unix-like system.</td>
      <td></td>
      <td>3</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>106</td>
      <td>126</td>
      <td>944</td>
      <td>1353</td>
      <td>146</td>
      <td>169</td>
      <td>1199</td>
      <td>1700</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>3</td>
      <td>3</td>
      <td>3056</td>
      <td>2146</td>
      <td>296</td>
      <td>253</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>instrain</td>
      <td>instrain_compare, instrain_profile</td>
      <td>InStrain is a tool for analysis of co-occurring genome populations from metagenomes</td>
      <td>2021-08-11</td>
      <td>https://instrain.readthedocs.io/</td>
      <td>1.5.3</td>
      <td>instrain</td>
      <td>1.10.0</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>SNP detection, Genome comparison</td>
      <td>SNP detection, Genome comparison</td>
      <td>Mapping, Metagenomics</td>
      <td>Mapping, Metagenomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/instrain</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/instrain</td>
      <td>instrain</td>
      <td>InStrain</td>
      <td>InStrain is a tool for analysis of co-occurring genome populations from metagenomes that allows highly accurate genome comparisons, analysis of coverage, microdiversity, and linkage, and sensitive SNP detection with gene localization and synonymous non-synonymous identification</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>22</td>
      <td>22</td>
      <td>208</td>
      <td>208</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>208</td>
      <td>208</td>
      <td>22</td>
      <td>22</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>integronfinder</td>
      <td>integron_finder</td>
      <td>"IntegronFinder identify integrons with high accuracy and sensitivity.It searches for attC sites using covariance models, for integron-integrases using HMM profiles, and for other features (promoters, attI site) using pattern matching"</td>
      <td>2022-09-22</td>
      <td>https://github.com/gem-pasteur/Integron_Finder</td>
      <td>2.0.5</td>
      <td>integron_finder</td>
      <td>2.0.6</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Nucleic acid feature detection, Sequence motif recognition, Protein feature detection, Genome annotation</td>
      <td>Nucleic acid feature detection, Sequence motif recognition, Protein feature detection, Genome annotation</td>
      <td>Functional genomics, Mobile genetic elements, Molecular biology, Sequence analysis</td>
      <td>Functional genomics, Mobile genetic elements, Molecular biology, Sequence analysis</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/integron_finder</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/integron_finder</td>
      <td>integron_finder</td>
      <td>Integron Finder</td>
      <td>A tool to detect Integron in DNA sequences.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>390</td>
      <td>390</td>
      <td>64428</td>
      <td>64428</td>
      <td>252</td>
      <td>252</td>
      <td>7167</td>
      <td>7167</td>
      <td>132</td>
      <td>132</td>
      <td>12061</td>
      <td>12061</td>
      <td>34</td>
      <td>34</td>
      <td>2504</td>
      <td>2504</td>
      <td>86160</td>
      <td>86160</td>
      <td>808</td>
      <td>808</td>
      <td>https://usegalaxy.eu/published/workflow?id=1306ed083d91a852, https://usegalaxy.eu/published/workflow?id=2c1a6da84d891cf2, https://usegalaxy.eu/published/workflow?id=36ea584e0f60893e, https://usegalaxy.eu/published/workflow?id=3f5fe28793ed0c6c, https://usegalaxy.eu/published/workflow?id=4a218fb93d78ba67, https://usegalaxy.eu/published/workflow?id=61948fcaefa1418a, https://usegalaxy.eu/published/workflow?id=6b69ab27c565257b, https://usegalaxy.eu/published/workflow?id=8792eaecb69fbc26, https://usegalaxy.eu/published/workflow?id=a18aed4b03ffd27e, https://usegalaxy.eu/published/workflow?id=b1afc8d1e81fc904, https://usegalaxy.eu/published/workflow?id=bcd50e655809cfcb, https://usegalaxy.eu/published/workflow?id=d8db0b2e30370b84, https://usegalaxy.eu/published/workflow?id=dfb8e0f3c534000e, https://usegalaxy.eu/published/workflow?id=fd63b9884fda75b6, https://usegalaxy.fr/published/workflow?id=7964ddc68853458f, https://usegalaxy.org/published/workflow?id=04c5ea133de3204c, https://usegalaxy.org/published/workflow?id=3940186d02245b1d, https://usegalaxy.org/published/workflow?id=49da012e2839c046, https://usegalaxy.org/published/workflow?id=4f8bdd61c524b901, https://usegalaxy.org/published/workflow?id=649cdc14757e5755, https://usegalaxy.org/published/workflow?id=67ca17ab34b10772, https://usegalaxy.org/published/workflow?id=8cb8a588a0495cfa, https://usegalaxy.org/published/workflow?id=993534c11310a207, https://usegalaxy.org/published/workflow?id=d2a5f6a5181aada5, https://workflowhub.eu/workflows/1050?version=13, https://workflowhub.eu/workflows/1513?version=1</td>
      <td>genome-annotation/bacterial-genome-annotation</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>interproscan</td>
      <td>interproscan</td>
      <td>Interproscan queries the interpro database and provides annotations.</td>
      <td>2021-11-15</td>
      <td>http://www.ebi.ac.uk/Tools/pfa/iprscan5/</td>
      <td>5.59-91.0</td>
      <td>interproscan</td>
      <td>5.59_91.0</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Sequence motif recognition, Protein feature detection</td>
      <td>Sequence motif recognition, Protein feature detection</td>
      <td>Gene and protein families, Sequence analysis</td>
      <td>Gene and protein families, Sequence analysis</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/interproscan</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/interproscan</td>
      <td>interproscan_ebi</td>
      <td>InterProScan (EBI)</td>
      <td>Scan sequences against the InterPro protein signature databases.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2350</td>
      <td>2550</td>
      <td>44154</td>
      <td>69658</td>
      <td>486</td>
      <td>486</td>
      <td>3914</td>
      <td>3914</td>
      <td>799</td>
      <td>804</td>
      <td>13188</td>
      <td>13204</td>
      <td>144</td>
      <td>144</td>
      <td>9435</td>
      <td>9435</td>
      <td>96211</td>
      <td>70691</td>
      <td>3984</td>
      <td>3779</td>
      <td>https://usegalaxy.cz/published/workflow?id=932f4901960a595a, https://usegalaxy.eu/published/workflow?id=02631612a43a4d9b, https://usegalaxy.eu/published/workflow?id=0b9071d5dfa38559, https://usegalaxy.eu/published/workflow?id=2d2862b48918b22f, https://usegalaxy.eu/published/workflow?id=33312e5643279e8a, https://usegalaxy.eu/published/workflow?id=3eaa913306c16146, https://usegalaxy.eu/published/workflow?id=58955868b504c60d, https://usegalaxy.eu/published/workflow?id=6158c3b5fc12044e, https://usegalaxy.eu/published/workflow?id=6416cfdbd5273063, https://usegalaxy.eu/published/workflow?id=667bac8d7453e5f1, https://usegalaxy.eu/published/workflow?id=8deb08d4f228e198, https://usegalaxy.eu/published/workflow?id=9a95556ec171d405, https://usegalaxy.eu/published/workflow?id=9a97c6eb4fe015e7, https://usegalaxy.eu/published/workflow?id=a448400f69594135, https://usegalaxy.eu/published/workflow?id=b9c938d1af08124b, https://usegalaxy.eu/published/workflow?id=cc59ae1a9878942c, https://usegalaxy.eu/published/workflow?id=d57d41e306241396, https://usegalaxy.eu/published/workflow?id=df7d9943423549df, https://usegalaxy.eu/published/workflow?id=e5becc6d66b2568c, https://usegalaxy.eu/published/workflow?id=f62d7c0c7cc57eb6, https://usegalaxy.org.au/published/workflow?id=35f39173f0ac9179, https://usegalaxy.org.au/published/workflow?id=7559224fc32dce3a, https://usegalaxy.org.au/published/workflow?id=876163677f1cccc4, https://usegalaxy.org.au/published/workflow?id=a9068ea858222e2b, https://usegalaxy.org/published/workflow?id=92613e1e0b13aa0a, https://usegalaxy.org/published/workflow?id=c451e4dca9981ed0, https://workflowhub.eu/workflows/1262?version=1, https://workflowhub.eu/workflows/1521?version=1, https://workflowhub.eu/workflows/1525?version=1, https://workflowhub.eu/workflows/1663?version=1, https://workflowhub.eu/workflows/754?version=1, https://workflowhub.eu/workflows/755?version=1</td>
      <td>ecology/marine_omics_bgc, genome-annotation/funannotate, genome-annotation/functional</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>iphop</td>
      <td>iphop_predict</td>
      <td>Integrated Phage Host Prediction</td>
      <td>2024-10-14</td>
      <td>https://bitbucket.org/srouxjgi/iphop/</td>
      <td>1.3.3</td>
      <td>iphop</td>
      <td>1.4.2</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Sample comparison</td>
      <td>Sample comparison</td>
      <td></td>
      <td></td>
      <td>ufz</td>
      <td>https://github.com/Helmholtz-UFZ/ufz-galaxy-tools/blob/main/tools/iphop/</td>
      <td>https://github.com/Helmholtz-UFZ/galaxy-tools/tree/main/tools/iphop</td>
      <td>iphop</td>
      <td>iPHoP</td>
      <td>An automated command-line pipeline for predicting host genus of novel bacteriophages and archaeoviruses based on their genome sequences.</td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>9</td>
      <td>26</td>
      <td>26</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>26</td>
      <td>26</td>
      <td>9</td>
      <td>9</td>
      <td>https://usegalaxy.eu/published/workflow?id=62392daea25a3980, https://usegalaxy.eu/published/workflow?id=c62d65832377e376</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>iprscan5</td>
      <td></td>
      <td>Interproscan queries the interpro database and provides annotations.</td>
      <td>2015-02-28</td>
      <td>http://www.ebi.ac.uk/Tools/pfa/iprscan5/</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/iprscan5</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/iprscan5</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>iqtree</td>
      <td>iqtree</td>
      <td>Efficient phylogenomic software by maximum likelihood</td>
      <td>2017-09-26</td>
      <td>http://www.iqtree.org/</td>
      <td>2.4.0</td>
      <td>iqtree</td>
      <td>3.0.1</td>
      <td>To update</td>
      <td>Phylogenetics</td>
      <td>Phylogenetic analysis, Sequence analysis</td>
      <td>Phylogenetic analysis, Sequence analysis</td>
      <td>Phylogenetics</td>
      <td>Phylogenetics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/iqtree</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/iqtree</td>
      <td>iqtree</td>
      <td>iqtree</td>
      <td>A fast and effective stochastic algorithm to infer phylogenetic trees by maximum likelihood. IQ-TREE compares favorably to RAxML and PhyML in terms of likelihoods with similar computing time</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2588</td>
      <td>2695</td>
      <td>44137</td>
      <td>45132</td>
      <td>2814</td>
      <td>3035</td>
      <td>34112</td>
      <td>35898</td>
      <td>1165</td>
      <td>1337</td>
      <td>16490</td>
      <td>18756</td>
      <td>17</td>
      <td>17</td>
      <td>229</td>
      <td>229</td>
      <td>100015</td>
      <td>94968</td>
      <td>7084</td>
      <td>6584</td>
      <td>https://usegalaxy.eu/published/workflow?id=628062a8cf93261f, https://usegalaxy.eu/published/workflow?id=94c3d329c31c712f, https://usegalaxy.eu/published/workflow?id=a15dabcadb63b46d, https://usegalaxy.eu/published/workflow?id=c789ab71876c6ead, https://usegalaxy.eu/published/workflow?id=cd25f70f71683ab6, https://usegalaxy.eu/published/workflow?id=d5cf994c03a60fa4, https://usegalaxy.eu/published/workflow?id=e97b938ca4723827, https://usegalaxy.org.au/published/workflow?id=3242ec69dec6e13f, https://usegalaxy.org.au/published/workflow?id=3fd2195b6928d5e0, https://usegalaxy.org/published/workflow?id=1518c8c50747a1c1, https://usegalaxy.org/published/workflow?id=23212ae74fc2d2b5, https://usegalaxy.org/published/workflow?id=603e5c61c57bcf3f, https://usegalaxy.org/published/workflow?id=6a1eb7e7bececba1, https://usegalaxy.org/published/workflow?id=78d8b738f5715a8a, https://usegalaxy.org/published/workflow?id=952b83e52f7d7cff, https://usegalaxy.org/published/workflow?id=97263f3308077ff0, https://usegalaxy.org/published/workflow?id=9c4038fa15af6995, https://usegalaxy.org/published/workflow?id=e1cc0886cd27b92e, https://workflowhub.eu/workflows/1260?version=2, https://workflowhub.eu/workflows/1559?version=1, https://workflowhub.eu/workflows/1839?version=1, https://workflowhub.eu/workflows/359?version=1</td>
      <td>evolution/abc_intro_phylo, sequence-analysis/tapscan-streptophyte-algae, variant-analysis/aiv-analysis</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>irma</td>
      <td>irma</td>
      <td>Iterative Refinement Meta-Assembler (IRMA) is a tool to construct assembly of highly variable RNA viruses.</td>
      <td>2024-11-09</td>
      <td>https://wonder.cdc.gov/amd/flu/irma/irma.html</td>
      <td>1.2.0</td>
      <td>irma</td>
      <td>1.2.0</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Mapping assembly</td>
      <td>Mapping assembly</td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/irma</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/irma</td>
      <td>irma-virus</td>
      <td>IRMA</td>
      <td>Iterative Refinement Meta-Assembler for robust assembly, variant calling, and phasing of highly variable RNA virus sequencing data</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>12</td>
      <td>12</td>
      <td>16450</td>
      <td>16450</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>16450</td>
      <td>16450</td>
      <td>12</td>
      <td>12</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>itsx</td>
      <td>itsx</td>
      <td>ITSx is an open source software utility to extract the highly variable ITS1 and ITS2 subregions from ITS sequences.</td>
      <td>2022-05-02</td>
      <td>https://microbiology.se/software/itsx/</td>
      <td>1.1.3</td>
      <td>itsx</td>
      <td>1.1.3</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Sequence feature detection</td>
      <td>Sequence feature detection</td>
      <td>Functional, regulatory and non-coding RNA, Microbiology</td>
      <td>Functional, regulatory and non-coding RNA, Microbiology</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/itsx</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/itsx</td>
      <td>ITSx</td>
      <td>ITSx</td>
      <td>TSx is an open source software utility to extract the highly variable ITS1 and ITS2 subregions from ITS sequences, which is commonly used as a molecular barcode for e.g. fungi. As the inclusion of parts of the neighbouring, very conserved, ribosomal genes (SSU, 5S and LSU rRNA sequences) in the sequence identification process can lead to severely misleading results, ITSx identifies and extracts only the ITS regions themselves.</td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>140</td>
      <td>140</td>
      <td>2878</td>
      <td>2878</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>18</td>
      <td>18</td>
      <td>198</td>
      <td>198</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3076</td>
      <td>3076</td>
      <td>158</td>
      <td>158</td>
      <td>https://usegalaxy.eu/published/workflow?id=4eb9dfe91fc2dc67</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>ivar</td>
      <td>ivar_consensus, ivar_filtervariants, ivar_removereads, ivar_trim, ivar_variants</td>
      <td>iVar is a computational package that contains functions broadly useful for viral amplicon-based sequencing</td>
      <td>2020-03-21</td>
      <td>https://github.com/andersen-lab/ivar</td>
      <td>1.4.4</td>
      <td>ivar</td>
      <td>1.4.4</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Primer removal</td>
      <td>Primer removal</td>
      <td>Virology</td>
      <td>Virology</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ivar</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ivar</td>
      <td>andersen-lab_ivar</td>
      <td>iVar</td>
      <td>iVar is a computational package that contains functions broadly useful for viral amplicon-based sequencing.</td>
      <td></td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>1081</td>
      <td>1087</td>
      <td>1414011</td>
      <td>1417441</td>
      <td>1310</td>
      <td>1330</td>
      <td>147224</td>
      <td>148408</td>
      <td>374</td>
      <td>379</td>
      <td>43473</td>
      <td>54915</td>
      <td>19</td>
      <td>19</td>
      <td>7502</td>
      <td>7502</td>
      <td>1628266</td>
      <td>1612210</td>
      <td>2815</td>
      <td>2784</td>
      <td>https://usegalaxy.eu/published/workflow?id=0189bab68f828d86, https://usegalaxy.eu/published/workflow?id=0be293debca8fb9b, https://usegalaxy.eu/published/workflow?id=0f6a7bd5362e4a2b, https://usegalaxy.eu/published/workflow?id=2573209308aaad23, https://usegalaxy.eu/published/workflow?id=2d9d4445a1871230, https://usegalaxy.eu/published/workflow?id=3f046b416b10b871, https://usegalaxy.eu/published/workflow?id=3fb550d7c2bc390a, https://usegalaxy.eu/published/workflow?id=40d2c7ac42bd5302, https://usegalaxy.eu/published/workflow?id=429ae28a178cd07c, https://usegalaxy.eu/published/workflow?id=45fc2dada84d251e, https://usegalaxy.eu/published/workflow?id=476db04368d5e90c, https://usegalaxy.eu/published/workflow?id=4b4bfc79205d83f1, https://usegalaxy.eu/published/workflow?id=4c75ab0be3c56494, https://usegalaxy.eu/published/workflow?id=4d97068a725c975c, https://usegalaxy.eu/published/workflow?id=4f40ae432d8a4259, https://usegalaxy.eu/published/workflow?id=529f30e894beacc2, https://usegalaxy.eu/published/workflow?id=52bdf400f97c5770, https://usegalaxy.eu/published/workflow?id=53de0e742573e588, https://usegalaxy.eu/published/workflow?id=579dd1e95e76402f, https://usegalaxy.eu/published/workflow?id=5c15f251642bfa1f, https://usegalaxy.eu/published/workflow?id=629e57958afed44c, https://usegalaxy.eu/published/workflow?id=6dbac47941a804da, https://usegalaxy.eu/published/workflow?id=6e1fe7372e93fae0, https://usegalaxy.eu/published/workflow?id=74f19aff1b9cdd6b, https://usegalaxy.eu/published/workflow?id=7bdd9ad918edab0b, https://usegalaxy.eu/published/workflow?id=7c986f4d14c6a539, https://usegalaxy.eu/published/workflow?id=98bfad0c684011e9, https://usegalaxy.eu/published/workflow?id=9feebfc6c34da287, https://usegalaxy.eu/published/workflow?id=a15dabcadb63b46d, https://usegalaxy.eu/published/workflow?id=a31100610fba17a1, https://usegalaxy.eu/published/workflow?id=a6f4371ef9bc1aab, https://usegalaxy.eu/published/workflow?id=ada52ba09b462567, https://usegalaxy.eu/published/workflow?id=b77e3b08092fa73e, https://usegalaxy.eu/published/workflow?id=b97532c398b821a2, https://usegalaxy.eu/published/workflow?id=c789ab71876c6ead, https://usegalaxy.eu/published/workflow?id=cd25f70f71683ab6, https://usegalaxy.eu/published/workflow?id=d88145e7475e0b7b, https://usegalaxy.eu/published/workflow?id=d967194ca51ff11e, https://usegalaxy.eu/published/workflow?id=eaf01ccb13aad3ec, https://usegalaxy.eu/published/workflow?id=ed20bd120064bab0, https://usegalaxy.eu/published/workflow?id=ee6dc47660540ba6, https://usegalaxy.eu/published/workflow?id=fc0b9efa7d2e61e6, https://usegalaxy.eu/published/workflow?id=fc216dfb0ec9ed41, https://usegalaxy.eu/published/workflow?id=fcf745185d6e0ea4, https://usegalaxy.fr/published/workflow?id=2345fad7dac13daa, https://usegalaxy.fr/published/workflow?id=591c57173e259ca6, https://usegalaxy.fr/published/workflow?id=61235264d5d2f666, https://usegalaxy.fr/published/workflow?id=727cfdb7c79287d1, https://usegalaxy.fr/published/workflow?id=7d27dd394b921846, https://usegalaxy.fr/published/workflow?id=98529cff0ef36b91, https://usegalaxy.fr/published/workflow?id=d0d12c0b80c286f9, https://usegalaxy.fr/published/workflow?id=e0b225d4b2556365, https://usegalaxy.fr/published/workflow?id=e3c04a30258421aa, https://usegalaxy.org.au/published/workflow?id=094a545d44ce28cc, https://usegalaxy.org.au/published/workflow?id=0c6f7bfc826433c4, https://usegalaxy.org.au/published/workflow?id=203f9e69152fbb2a, https://usegalaxy.org.au/published/workflow?id=265a3ce9305702f5, https://usegalaxy.org.au/published/workflow?id=2b2ce7e1206d177d, https://usegalaxy.org.au/published/workflow?id=3223ceaa2b58d940, https://usegalaxy.org.au/published/workflow?id=3c657dd766d02f6a, https://usegalaxy.org.au/published/workflow?id=47ce6a2d30887637, https://usegalaxy.org.au/published/workflow?id=48a60965b6b3ff54, https://usegalaxy.org.au/published/workflow?id=4bc8ac41ecff7daf, https://usegalaxy.org.au/published/workflow?id=6b1b0a945d367b4c, https://usegalaxy.org.au/published/workflow?id=71f90428fbf8755b, https://usegalaxy.org.au/published/workflow?id=7563b3ef697a69fd, https://usegalaxy.org.au/published/workflow?id=7fd58f5fc93f414e, https://usegalaxy.org.au/published/workflow?id=810242a3dc4f6170, https://usegalaxy.org.au/published/workflow?id=888eb87b62c4e6dd, https://usegalaxy.org.au/published/workflow?id=888f8377eb6eabf2, https://usegalaxy.org.au/published/workflow?id=8ffcaa3f4a56ce35, https://usegalaxy.org.au/published/workflow?id=932bad88f8dc8d33, https://usegalaxy.org.au/published/workflow?id=af12a3ec85859563, https://usegalaxy.org.au/published/workflow?id=b432a1312859edd2, https://usegalaxy.org.au/published/workflow?id=ba7a9d4c857d26b7, https://usegalaxy.org.au/published/workflow?id=baa763dbdf6afb71, https://usegalaxy.org.au/published/workflow?id=bcf230e526632064, https://usegalaxy.org.au/published/workflow?id=be9213411c520096, https://usegalaxy.org.au/published/workflow?id=c1ce3f2ae7c81c3e, https://usegalaxy.org.au/published/workflow?id=c9156ca3df2b595c, https://usegalaxy.org.au/published/workflow?id=ccf936c5b2f2c2f3, https://usegalaxy.org.au/published/workflow?id=d35d05be576e8b13, https://usegalaxy.org.au/published/workflow?id=d8a4a8bc4fa2f9ee, https://usegalaxy.org.au/published/workflow?id=e439a0e17a90d24c, https://usegalaxy.org.au/published/workflow?id=f52fb12383c8a214, https://usegalaxy.org.au/published/workflow?id=fca05a51c9c91df4, https://usegalaxy.org.au/published/workflow?id=fe5542c99cdc57ce, https://usegalaxy.org/published/workflow?id=0ee0493f3168cbb3, https://usegalaxy.org/published/workflow?id=0fa2ce12c2874b1d, https://usegalaxy.org/published/workflow?id=1518c8c50747a1c1, https://usegalaxy.org/published/workflow?id=17d9b233640aefec, https://usegalaxy.org/published/workflow?id=1f48b44fa22ac5fc, https://usegalaxy.org/published/workflow?id=2ded909e7e3a9962, https://usegalaxy.org/published/workflow?id=426e3951dac0704e, https://usegalaxy.org/published/workflow?id=4357180d40560019, https://usegalaxy.org/published/workflow?id=4b41a8247f52c463, https://usegalaxy.org/published/workflow?id=4cf8cc42e97b2da6, https://usegalaxy.org/published/workflow?id=6b11d738ee57f245, https://usegalaxy.org/published/workflow?id=7dde0721a382ea3e, https://usegalaxy.org/published/workflow?id=7e2651e5ed17e3db, https://usegalaxy.org/published/workflow?id=88d0b64011c3148c, https://usegalaxy.org/published/workflow?id=8b81b9a3b49bbf98, https://usegalaxy.org/published/workflow?id=8e1c6ad6ff623015, https://usegalaxy.org/published/workflow?id=953d9cbe05e632a8, https://usegalaxy.org/published/workflow?id=97263f3308077ff0, https://usegalaxy.org/published/workflow?id=98b2ac8fc75781f8, https://usegalaxy.org/published/workflow?id=9adf8691809326e5, https://usegalaxy.org/published/workflow?id=9fc9f1db50019929, https://usegalaxy.org/published/workflow?id=b34ba9b2e4d8f050, https://usegalaxy.org/published/workflow?id=bd355e8ce58a139e, https://usegalaxy.org/published/workflow?id=bf3ccae8d5b6ef58, https://usegalaxy.org/published/workflow?id=bf90f2bd20fbff21, https://usegalaxy.org/published/workflow?id=c12e6b7896f8e114, https://usegalaxy.org/published/workflow?id=c7b2d2ccc4b0dec5, https://usegalaxy.org/published/workflow?id=cdf5dffbed938d49, https://usegalaxy.org/published/workflow?id=d3428dd172a78102, https://usegalaxy.org/published/workflow?id=d4ea6cdd40522eb1, https://usegalaxy.org/published/workflow?id=d96bd5c9fb553f2c, https://usegalaxy.org/published/workflow?id=e282f505ba9f729f, https://workflowhub.eu/workflows/110?version=11, https://workflowhub.eu/workflows/111?version=5, https://workflowhub.eu/workflows/1260?version=2, https://workflowhub.eu/workflows/155?version=6, https://workflowhub.eu/workflows/1876?version=2, https://workflowhub.eu/workflows/439?version=4, https://workflowhub.eu/workflows/519?version=1</td>
      <td>variant-analysis/aiv-analysis, variant-analysis/pox-tiled-amplicon</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>jbrowse</td>
      <td>jbrowse_to_standalone, jbrowse</td>
      <td>JBrowse Genome Browser integrated as a Galaxy Tool</td>
      <td>2015-05-01</td>
      <td>https://jbrowse.org</td>
      <td>1.16.11</td>
      <td>jbrowse</td>
      <td>1.16.11</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Genome visualisation</td>
      <td>Genome visualisation</td>
      <td>Genomics</td>
      <td>Genomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/jbrowse</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/jbrowse</td>
      <td>jbrowse</td>
      <td>JBrowse</td>
      <td>Slick, speedy genome browser with a responsive and dynamic AJAX interface for visualization of genome data. Being developed by the GMOD project as a successor to GBrowse.</td>
      <td></td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>6793</td>
      <td>7423</td>
      <td>26043</td>
      <td>28614</td>
      <td>7372</td>
      <td>7751</td>
      <td>22125</td>
      <td>23288</td>
      <td>3208</td>
      <td>4288</td>
      <td>18197</td>
      <td>22221</td>
      <td>299</td>
      <td>300</td>
      <td>1697</td>
      <td>1712</td>
      <td>75835</td>
      <td>68062</td>
      <td>19762</td>
      <td>17672</td>
      <td>https://usegalaxy.cz/published/workflow?id=932f4901960a595a, https://usegalaxy.eu/published/workflow?id=0b9071d5dfa38559, https://usegalaxy.eu/published/workflow?id=12653a1e65a20304, https://usegalaxy.eu/published/workflow?id=152223c8aed27357, https://usegalaxy.eu/published/workflow?id=1ea8bfbe6cb04531, https://usegalaxy.eu/published/workflow?id=26ad77bc07a7e512, https://usegalaxy.eu/published/workflow?id=2b25eaebd3987612, https://usegalaxy.eu/published/workflow?id=31bc2a44fb89aa89, https://usegalaxy.eu/published/workflow?id=327beb1edda305ff, https://usegalaxy.eu/published/workflow?id=3c233cc56d4df8b0, https://usegalaxy.eu/published/workflow?id=3d604dad4f1f5b3d, https://usegalaxy.eu/published/workflow?id=3dcf09e75d707573, https://usegalaxy.eu/published/workflow?id=3ff4dbcc7a602e6b, https://usegalaxy.eu/published/workflow?id=4077e77b6af919c4, https://usegalaxy.eu/published/workflow?id=41268469c6616e42, https://usegalaxy.eu/published/workflow?id=44189803f223acd1, https://usegalaxy.eu/published/workflow?id=461d71caea871db8, https://usegalaxy.eu/published/workflow?id=4ddbffe4b3fef275, https://usegalaxy.eu/published/workflow?id=515ef1601bc24a6c, https://usegalaxy.eu/published/workflow?id=5826e573b6f93762, https://usegalaxy.eu/published/workflow?id=5aba0f20d994b1a4, https://usegalaxy.eu/published/workflow?id=6550f72070b24082, https://usegalaxy.eu/published/workflow?id=6ca20e710f4f3ffb, https://usegalaxy.eu/published/workflow?id=733650962d379da2, https://usegalaxy.eu/published/workflow?id=7aa32aba3fde0708, https://usegalaxy.eu/published/workflow?id=7f6950394a8c9c4a, https://usegalaxy.eu/published/workflow?id=8048cadfd1ed1661, https://usegalaxy.eu/published/workflow?id=8976041ef89e4548, https://usegalaxy.eu/published/workflow?id=920aab45ff028ed4, https://usegalaxy.eu/published/workflow?id=93d1035a307fe63d, https://usegalaxy.eu/published/workflow?id=968f0ac2617aa524, https://usegalaxy.eu/published/workflow?id=a337bfb2eb1e583f, https://usegalaxy.eu/published/workflow?id=a84bac86ec183d46, https://usegalaxy.eu/published/workflow?id=c0f225536e78a178, https://usegalaxy.eu/published/workflow?id=c38e3ac59d8b86d6, https://usegalaxy.eu/published/workflow?id=c9ef2d2ce92e2dfb, https://usegalaxy.eu/published/workflow?id=cae527ce845e770e, https://usegalaxy.eu/published/workflow?id=d2f75f70c33394ee, https://usegalaxy.eu/published/workflow?id=d3dfe7ba11508c72, https://usegalaxy.eu/published/workflow?id=df7d9943423549df, https://usegalaxy.eu/published/workflow?id=e1894c36f1ab4ab4, https://usegalaxy.eu/published/workflow?id=f62d7c0c7cc57eb6, https://usegalaxy.fr/published/workflow?id=ae05ff8487f58fd6, https://usegalaxy.org.au/published/workflow?id=0dbb0d1ae1222bd0, https://usegalaxy.org.au/published/workflow?id=0f71c97671bb401f, https://usegalaxy.org.au/published/workflow?id=16a2c2e8b3c1ffe6, https://usegalaxy.org.au/published/workflow?id=1719e0eb54b41f68, https://usegalaxy.org.au/published/workflow?id=22bd45f60c60a338, https://usegalaxy.org.au/published/workflow?id=2e29b21cdec7d0cc, https://usegalaxy.org.au/published/workflow?id=33401b29d277b447, https://usegalaxy.org.au/published/workflow?id=4e161a8f5098cc49, https://usegalaxy.org.au/published/workflow?id=5693811cd933d2a2, https://usegalaxy.org.au/published/workflow?id=59d6ee23bff41eec, https://usegalaxy.org.au/published/workflow?id=6c67e22d18566af3, https://usegalaxy.org.au/published/workflow?id=7c343e5b55474d19, https://usegalaxy.org.au/published/workflow?id=7e40485d1ed45b7c, https://usegalaxy.org.au/published/workflow?id=8031655d71b59d15, https://usegalaxy.org.au/published/workflow?id=87cdeb05b1ebeab0, https://usegalaxy.org.au/published/workflow?id=914330feb75ea9ab, https://usegalaxy.org.au/published/workflow?id=a35e5a238a70176a, https://usegalaxy.org.au/published/workflow?id=a61d4bda54de7cd1, https://usegalaxy.org.au/published/workflow?id=b2818098da6e2f82, https://usegalaxy.org.au/published/workflow?id=b5403e4fff2961f9, https://usegalaxy.org.au/published/workflow?id=c456492ce4f963d0, https://usegalaxy.org.au/published/workflow?id=cb6f49d4ebfa0891, https://usegalaxy.org.au/published/workflow?id=d8a4a8bc4fa2f9ee, https://usegalaxy.org.au/published/workflow?id=dd9272826e6dd98d, https://usegalaxy.org.au/published/workflow?id=e263fcb6c98e231b, https://usegalaxy.org.au/published/workflow?id=eba41e98316b8a08, https://usegalaxy.org.au/published/workflow?id=ec7efff3d3249177, https://usegalaxy.org/published/workflow?id=00596e862edb23f2, https://usegalaxy.org/published/workflow?id=0795dd1d83662bae, https://usegalaxy.org/published/workflow?id=0f308c521abfe2c9, https://usegalaxy.org/published/workflow?id=4605309321799e9b, https://usegalaxy.org/published/workflow?id=49c61c378cacfe02, https://usegalaxy.org/published/workflow?id=53d65fd38b9b8c1b, https://usegalaxy.org/published/workflow?id=547f86db467bf620, https://usegalaxy.org/published/workflow?id=562bcaa9b4d2e8a4, https://usegalaxy.org/published/workflow?id=5674956f5c18b85b, https://usegalaxy.org/published/workflow?id=5b2bb861501fa248, https://usegalaxy.org/published/workflow?id=5bcd1aade71a8035, https://usegalaxy.org/published/workflow?id=5efe288b18c7d0ab, https://usegalaxy.org/published/workflow?id=612452c925d723b7, https://usegalaxy.org/published/workflow?id=62436fe121e035e7, https://usegalaxy.org/published/workflow?id=69d8863d40524120, https://usegalaxy.org/published/workflow?id=6d68f56abf9c624d, https://usegalaxy.org/published/workflow?id=7dd6e4150bb81ad1, https://usegalaxy.org/published/workflow?id=80565301ada0d3eb, https://usegalaxy.org/published/workflow?id=80a7124c40438cc9, https://usegalaxy.org/published/workflow?id=80ac6adead9f97d8, https://usegalaxy.org/published/workflow?id=c9a8fd89224e7414, https://usegalaxy.org/published/workflow?id=ce232859a466c75c, https://usegalaxy.org/published/workflow?id=dbe08700353dfc43, https://usegalaxy.org/published/workflow?id=ec3cc032a90c44f1, https://usegalaxy.org/published/workflow?id=ee4679b3e9e830b7, https://usegalaxy.org/published/workflow?id=f4111a169255535c, https://workflowhub.eu/workflows/1255?version=3, https://workflowhub.eu/workflows/1323?version=1, https://workflowhub.eu/workflows/1488?version=2, https://workflowhub.eu/workflows/1497?version=1, https://workflowhub.eu/workflows/1500?version=1, https://workflowhub.eu/workflows/1504?version=1, https://workflowhub.eu/workflows/1509?version=1, https://workflowhub.eu/workflows/1513?version=1, https://workflowhub.eu/workflows/1525?version=1, https://workflowhub.eu/workflows/1549?version=1, https://workflowhub.eu/workflows/1554?version=1, https://workflowhub.eu/workflows/1617?version=1, https://workflowhub.eu/workflows/1639?version=1, https://workflowhub.eu/workflows/1657?version=1, https://workflowhub.eu/workflows/1661?version=2, https://workflowhub.eu/workflows/1746?version=1, https://workflowhub.eu/workflows/2029?version=1, https://workflowhub.eu/workflows/520?version=1, https://workflowhub.eu/workflows/749?version=1, https://workflowhub.eu/workflows/754?version=1</td>
      <td>assembly/chloroplast-assembly, assembly/ecoli_comparison, genome-annotation/amr-gene-detection, genome-annotation/annotation-with-maker, genome-annotation/annotation-with-maker-short, genome-annotation/annotation-with-prokka, genome-annotation/apollo, genome-annotation/apollo-euk, genome-annotation/bacterial-genome-annotation, genome-annotation/braker3, genome-annotation/comparison-braker-helixer-annotation, genome-annotation/funannotate, genome-annotation/helixer, sequence-analysis/mapping, sequence-analysis/sars-with-galaxy-on-anvil, variant-analysis/microbial-variants, variant-analysis/tb-variant-analysis, visualisation/jbrowse</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>jbrowse2</td>
      <td>jbrowse2</td>
      <td>JBrowse2 Genome Browser integrated as a Galaxy Tool</td>
      <td>2021-09-30</td>
      <td>https://jbrowse.org</td>
      <td>3.7.0</td>
      <td>jbrowse2</td>
      <td>4.0.4</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Genome visualisation, Structure visualisation, Pathway visualisation</td>
      <td>Genome visualisation, Structure visualisation, Pathway visualisation</td>
      <td>Mapping, Structural variation, Genomics, Gene structure, Sequence assembly</td>
      <td>Mapping, Structural variation, Genomics, Gene structure, Sequence assembly</td>
      <td>fubar</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/jbrowse2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/jbrowse2</td>
      <td>jbrowse_2</td>
      <td>JBrowse 2</td>
      <td>Modular genome browser with views of synteny and structural variation.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>626</td>
      <td>626</td>
      <td>3765</td>
      <td>3765</td>
      <td>349</td>
      <td>349</td>
      <td>1248</td>
      <td>1248</td>
      <td>95</td>
      <td>95</td>
      <td>603</td>
      <td>603</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5616</td>
      <td>5616</td>
      <td>1070</td>
      <td>1070</td>
      <td>https://usegalaxy.eu/published/workflow?id=07c92110c3fffcf8, https://usegalaxy.eu/published/workflow?id=1ce406de61de8492, https://usegalaxy.eu/published/workflow?id=2cd5670f075704ab, https://usegalaxy.eu/published/workflow?id=2e93613c688ea52e, https://usegalaxy.eu/published/workflow?id=309969a261189b5f, https://usegalaxy.eu/published/workflow?id=36de05d50c703164, https://usegalaxy.eu/published/workflow?id=38f12705279b5dfa, https://usegalaxy.eu/published/workflow?id=3cf3dd9727b7ac92, https://usegalaxy.eu/published/workflow?id=5f29105e842716db, https://usegalaxy.eu/published/workflow?id=84414f3441946702, https://usegalaxy.eu/published/workflow?id=972db4abc51bb3ab, https://usegalaxy.eu/published/workflow?id=aada0111f835ce49, https://usegalaxy.eu/published/workflow?id=cc2bc81352d2e2cd, https://usegalaxy.eu/published/workflow?id=ce8bf03d3df24806, https://usegalaxy.eu/published/workflow?id=df08e520ec3dbcb7, https://usegalaxy.eu/published/workflow?id=e5becc6d66b2568c, https://usegalaxy.eu/published/workflow?id=efb58bba65393347, https://usegalaxy.org.au/published/workflow?id=a9068ea858222e2b, https://usegalaxy.org/published/workflow?id=386d3b300cf05305, https://usegalaxy.org/published/workflow?id=505c826e8c99d7bf, https://usegalaxy.org/published/workflow?id=89d3c800d3be6050, https://workflowhub.eu/workflows/1711?version=3, https://workflowhub.eu/workflows/1715?version=3, https://workflowhub.eu/workflows/2036?version=1, https://workflowhub.eu/workflows/2038?version=1</td>
      <td>epigenetics/formation_of_super-structures_on_xi, transcriptomics/ref-based</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>jbrowse2</td>
      <td>jbrowse2</td>
      <td>JBrowse2 Genome Browser integrated as a Galaxy Tool</td>
      <td>2024-06-04</td>
      <td>https://jbrowse.org</td>
      <td>2.17.0</td>
      <td>jbrowse2</td>
      <td>4.0.4</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Genome visualisation, Structure visualisation, Pathway visualisation</td>
      <td>Genome visualisation, Structure visualisation, Pathway visualisation</td>
      <td>Mapping, Structural variation, Genomics, Gene structure, Sequence assembly</td>
      <td>Mapping, Structural variation, Genomics, Gene structure, Sequence assembly</td>
      <td>fubar</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/jbrowse2</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/jbrowse2</td>
      <td>jbrowse_2</td>
      <td>JBrowse 2</td>
      <td>Modular genome browser with views of synteny and structural variation.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>626</td>
      <td>626</td>
      <td>3765</td>
      <td>3765</td>
      <td>349</td>
      <td>349</td>
      <td>1248</td>
      <td>1248</td>
      <td>95</td>
      <td>95</td>
      <td>603</td>
      <td>603</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5616</td>
      <td>5616</td>
      <td>1070</td>
      <td>1070</td>
      <td>https://usegalaxy.eu/published/workflow?id=07c92110c3fffcf8, https://usegalaxy.eu/published/workflow?id=1ce406de61de8492, https://usegalaxy.eu/published/workflow?id=2cd5670f075704ab, https://usegalaxy.eu/published/workflow?id=2e93613c688ea52e, https://usegalaxy.eu/published/workflow?id=309969a261189b5f, https://usegalaxy.eu/published/workflow?id=36de05d50c703164, https://usegalaxy.eu/published/workflow?id=38f12705279b5dfa, https://usegalaxy.eu/published/workflow?id=3cf3dd9727b7ac92, https://usegalaxy.eu/published/workflow?id=5f29105e842716db, https://usegalaxy.eu/published/workflow?id=84414f3441946702, https://usegalaxy.eu/published/workflow?id=972db4abc51bb3ab, https://usegalaxy.eu/published/workflow?id=aada0111f835ce49, https://usegalaxy.eu/published/workflow?id=cc2bc81352d2e2cd, https://usegalaxy.eu/published/workflow?id=ce8bf03d3df24806, https://usegalaxy.eu/published/workflow?id=df08e520ec3dbcb7, https://usegalaxy.eu/published/workflow?id=e5becc6d66b2568c, https://usegalaxy.eu/published/workflow?id=efb58bba65393347, https://usegalaxy.org.au/published/workflow?id=a9068ea858222e2b, https://usegalaxy.org/published/workflow?id=386d3b300cf05305, https://usegalaxy.org/published/workflow?id=505c826e8c99d7bf, https://usegalaxy.org/published/workflow?id=89d3c800d3be6050, https://workflowhub.eu/workflows/1711?version=3, https://workflowhub.eu/workflows/1715?version=3, https://workflowhub.eu/workflows/2036?version=1, https://workflowhub.eu/workflows/2038?version=1</td>
      <td>epigenetics/formation_of_super-structures_on_xi, transcriptomics/ref-based</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>jellyfish</td>
      <td>jellyfish</td>
      <td>Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA</td>
      <td>2021-04-07</td>
      <td>https://github.com/gmarcais/Jellyfish</td>
      <td></td>
      <td>kmer-jellyfish</td>
      <td>2.3.1</td>
      <td>To update</td>
      <td>Assembly</td>
      <td>k-mer counting</td>
      <td>k-mer counting</td>
      <td>Sequence analysis, Genomics</td>
      <td>Sequence analysis, Genomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/jellyfish</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/jellyfish</td>
      <td>Jellyfish</td>
      <td>Jellyfish</td>
      <td>A command-line algorithm for counting k-mers in DNA sequence.</td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>369</td>
      <td>369</td>
      <td>4901</td>
      <td>4901</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>87</td>
      <td>87</td>
      <td>1492</td>
      <td>1492</td>
      <td>35</td>
      <td>35</td>
      <td>1519</td>
      <td>1519</td>
      <td>7912</td>
      <td>7912</td>
      <td>491</td>
      <td>491</td>
      <td>https://usegalaxy.eu/published/workflow?id=bc32937d505135ab</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>kaiju</td>
      <td>kaiju_addtaxonnames, kaiju_mergeoutputs, kaiju_kaiju, kaiju_kaiju2krona, kaiju_kaiju2table</td>
      <td>taxonomic classification of high-throughput sequencing reads</td>
      <td>2025-04-22</td>
      <td>https://github.com/bioinformatics-centre/kaiju</td>
      <td>1.10.1</td>
      <td>kaiju</td>
      <td>1.10.1</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Taxonomic classification</td>
      <td>Taxonomic classification</td>
      <td>Metagenomics</td>
      <td>Metagenomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/kaiju</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/kaiju</td>
      <td>kaiju</td>
      <td>Kaiju</td>
      <td>Program for the taxonomic assignment of high-throughput sequencing reads, e.g., Illumina or Roche/454, from whole-genome sequencing of metagenomic DNA. Reads are directly assigned to taxa using the NCBI taxonomy and a reference database of protein sequences from Bacteria, Archaea, Fungi, microbial eukaryotes and viruses.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>kat_filter</td>
      <td>kat_@EXECUTABLE@</td>
      <td>Filtering kmers or reads from a database of kmers hashes</td>
      <td>2017-03-13</td>
      <td></td>
      <td>2.3</td>
      <td>kat</td>
      <td>2.4.2</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>nml</td>
      <td></td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/kat_filter</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>khmer</td>
      <td>khmer_abundance_distribution_single, khmer_abundance_distribution, khmer_count_median, khmer_partition, khmer_extract_partitions, khmer_filter_abundance, khmer_filter_below_abundance_cutoff, khmer_normalize_by_median</td>
      <td>In-memory nucleotide sequence k-mer counting, filtering, graph traversal and more</td>
      <td>2015-03-24</td>
      <td>https://khmer.readthedocs.org/</td>
      <td>3.0.0a3</td>
      <td>khmer</td>
      <td>3.0.0a3</td>
      <td>Up-to-date</td>
      <td>Assembly, Next Gen Mappers</td>
      <td>Standardisation and normalisation, De-novo assembly</td>
      <td>Standardisation and normalisation, De-novo assembly</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/khmer</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/khmer</td>
      <td>khmer</td>
      <td>khmer</td>
      <td>khmer is a set of command-line tools for working with DNA shotgun sequencing data from genomes, transcriptomes, metagenomes, and single cells. khmer can make de novo assemblies faster, and sometimes better. khmer can also identify (and fix) problems with shotgun data.</td>
      <td></td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>98</td>
      <td>116</td>
      <td>1572</td>
      <td>2262</td>
      <td>169</td>
      <td>185</td>
      <td>1179</td>
      <td>1564</td>
      <td>25</td>
      <td>25</td>
      <td>2179</td>
      <td>2179</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6005</td>
      <td>4930</td>
      <td>326</td>
      <td>292</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>kleborate</td>
      <td>kleborate</td>
      <td>Screen genome assemblies of Klebsiella pneumoniae and the Klebsiella pneumoniae species complex (KpSC)</td>
      <td>2022-09-09</td>
      <td>https://github.com/katholt/Kleborate</td>
      <td>2.3.2</td>
      <td>kleborate</td>
      <td>3.2.4</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Multilocus sequence typing, Genome assembly, Virulence prediction</td>
      <td>Multilocus sequence typing, Genome assembly, Virulence prediction</td>
      <td>Public health and epidemiology, Metagenomics, Population genomics, Sequence assembly, Whole genome sequencing</td>
      <td>Public health and epidemiology, Metagenomics, Population genomics, Sequence assembly, Whole genome sequencing</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/kleborate</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/kleborate</td>
      <td>kleborate</td>
      <td>Kleborate</td>
      <td>Genomic surveillance framework and global population structure for Klebsiella pneumoniae.Kleborate is a tool to screen genome assemblies of Klebsiella pneumoniae and the Klebsiella pneumoniae species complex (KpSC) for:.A manuscript describing the Kleborate software in full is currently in preparation. In the meantime, if you use Kleborate, please cite the preprint: Lam, MMC. et al. Genomic surveillance framework and global population structure for Klebsiella pneumoniae. bioRxiv (2020).</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>123</td>
      <td>123</td>
      <td>1479</td>
      <td>1479</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1479</td>
      <td>1479</td>
      <td>123</td>
      <td>123</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>kmer2stats</td>
      <td>kmer2stats</td>
      <td>A tool for creating data files for statistics based on kmers</td>
      <td>2025-04-16</td>
      <td>https://github.com/SantaMcCloud/kmer2stats</td>
      <td>1.0.1</td>
      <td>kmer2stats</td>
      <td>1.0.1</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/kmer2stats</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/kmer2stats</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>310</td>
      <td>310</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>310</td>
      <td>310</td>
      <td>8</td>
      <td>8</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>kmetashot</td>
      <td>kmetashot</td>
      <td>an alignment-free taxonomic classifier based on k-mer/minimizer counting</td>
      <td>2025-12-18</td>
      <td>https://github.com/gdefazio/kMetaShot</td>
      <td>2.0</td>
      <td>kmetashot</td>
      <td>2.0</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/kmetashot</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/kmetashot</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>kmindex</td>
      <td>kmindex_build, kmindex_query</td>
      <td>Index and query k-mers from sequencing samples using Bloom filters</td>
      <td>2025-12-10</td>
      <td>https://github.com/tlemane/kmindex</td>
      <td>0.6.0</td>
      <td>kmindex</td>
      <td>0.6.0</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/kmindex</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/kmindex</td>
      <td>kmindex</td>
      <td>kmindex</td>
      <td>kmindex is a tool for indexing and querying sequencing samples. It is built on top of kmtricks.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>kofamscan</td>
      <td>kofamscan</td>
      <td>Gene function annotation tool based on KEGG Orthology and hidden Markov model</td>
      <td>2020-11-12</td>
      <td>https://github.com/takaram/kofam_scan</td>
      <td>1.3.0</td>
      <td>kofamscan</td>
      <td>1.3.0</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Sequence analysis, Gene functional annotation</td>
      <td>Sequence analysis, Gene functional annotation</td>
      <td>Genomics, Structure analysis</td>
      <td>Genomics, Structure analysis</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/kofamscan</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/kofamscan</td>
      <td>kofamscan</td>
      <td>kofamscan</td>
      <td>KofamScan is a gene function annotation tool based on KEGG Orthology and hidden Markov model. You need KOfam database to use this tool.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>94</td>
      <td>94</td>
      <td>2904</td>
      <td>2904</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>3</td>
      <td>3</td>
      <td>2907</td>
      <td>2907</td>
      <td>96</td>
      <td>96</td>
      <td>https://usegalaxy.eu/published/workflow?id=3eaa913306c16146, https://usegalaxy.eu/published/workflow?id=6158c3b5fc12044e, https://usegalaxy.eu/published/workflow?id=6416cfdbd5273063, https://usegalaxy.eu/published/workflow?id=667bac8d7453e5f1, https://usegalaxy.eu/published/workflow?id=8deb08d4f228e198, https://usegalaxy.eu/published/workflow?id=9b9e1e28f3683f92, https://usegalaxy.eu/published/workflow?id=d57d41e306241396</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>kraken</td>
      <td>kraken-filter, kraken-mpa-report, kraken-report, kraken-translate, kraken</td>
      <td>Kraken is a system for assigning taxonomic labels to short DNAsequences, usually obtained through metagenomic studies. Previous attempts by otherbioinformatics software to accomplish this task have often used sequence alignmentor machine learning techniques that were quite slow, leading to the developmentof less sensitive but much faster abundance estimation programs. Kraken aims toachieve high sensitivity and high speed by utilizing exact alignments of k-mersand a novel classification algorithm.</td>
      <td>2017-01-24</td>
      <td>http://ccb.jhu.edu/software/kraken/</td>
      <td></td>
      <td>kraken</td>
      <td>1.1.1</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Taxonomic classification</td>
      <td>Taxonomic classification</td>
      <td>Taxonomy, Metagenomics</td>
      <td>Taxonomy, Metagenomics</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/kraken</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/kraken</td>
      <td>kraken</td>
      <td>Kraken</td>
      <td>System for assigning taxonomic labels to short DNA sequences, usually obtained through metagenomic studies. Previous attempts by other bioinformatics software to accomplish this task have often used sequence alignment or machine learning techniques that were quite slow, leading to the development of less sensitive but much faster abundance estimation programs. It aims to achieve high sensitivity and high speed by utilizing exact alignments of k-mers and a novel classification algorithm.</td>
      <td></td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>1452</td>
      <td>1453</td>
      <td>75318</td>
      <td>75385</td>
      <td>2535</td>
      <td>4229</td>
      <td>59050</td>
      <td>144905</td>
      <td>904</td>
      <td>1133</td>
      <td>38078</td>
      <td>44609</td>
      <td>114</td>
      <td>114</td>
      <td>3720</td>
      <td>3720</td>
      <td>268619</td>
      <td>176166</td>
      <td>6929</td>
      <td>5005</td>
      <td>https://usegalaxy.org.au/published/workflow?id=b6bad1f167257248, https://usegalaxy.org/published/workflow?id=2ec3e326c51d3147, https://usegalaxy.org/published/workflow?id=33d90e718ce500ef, https://usegalaxy.org/published/workflow?id=7321c6ad5425d347, https://usegalaxy.org/published/workflow?id=87b3305ea433b4e2, https://usegalaxy.org/published/workflow?id=a89e1018a00b883c, https://usegalaxy.org/published/workflow?id=e0c33057ac2670ba, https://usegalaxy.org/published/workflow?id=fb01e1a49793c8be</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>kraken2</td>
      <td>kraken2</td>
      <td>Kraken2 for taxonomic designation.</td>
      <td>2019-03-05</td>
      <td>http://ccb.jhu.edu/software/kraken/</td>
      <td>2.17.1</td>
      <td>kraken2</td>
      <td>2.17.1</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Taxonomic classification</td>
      <td>Taxonomic classification</td>
      <td>Taxonomy, Metagenomics, Microbial ecology, Microbiology</td>
      <td>Taxonomy, Metagenomics, Microbial ecology</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/kraken2/kraken2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/kraken2/kraken2</td>
      <td>kraken2</td>
      <td>kraken2</td>
      <td>Kraken 2 is the newest version of Kraken, a taxonomic classification system using exact k-mer matches to achieve high accuracy and fast classification speeds. This classifier matches each k-mer within a query sequence to the lowest common ancestor (LCA) of all genomes containing the given k-mer. The k-mer assignments inform the classification algorithm.Any assumption that Krakens raw read assignments can be directly translated into species or strain-level abundance estimates is flawed. Bracken (Bayesian Reestimation of Abundance after Classification with KrakEN), estimates species abundances in metagenomics samples by probabilistically re-distributing reads in the taxonomic tree. (Lu, Jennifer et al. Bracken: estimating species abundance in metagenomics data.)</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>9806</td>
      <td>9999</td>
      <td>519669</td>
      <td>530279</td>
      <td>7561</td>
      <td>7562</td>
      <td>224146</td>
      <td>224149</td>
      <td>2412</td>
      <td>2459</td>
      <td>87806</td>
      <td>88898</td>
      <td>216</td>
      <td>216</td>
      <td>26628</td>
      <td>26628</td>
      <td>869954</td>
      <td>858249</td>
      <td>20236</td>
      <td>19995</td>
      <td>https://usegalaxy.eu/published/workflow?id=007ce6405d196df3, https://usegalaxy.eu/published/workflow?id=012762cb4acb068b, https://usegalaxy.eu/published/workflow?id=04ed3f21de17aa1d, https://usegalaxy.eu/published/workflow?id=0590554db9794d13, https://usegalaxy.eu/published/workflow?id=08eb5e55109c4d9c, https://usegalaxy.eu/published/workflow?id=0a82eb8b44d9d502, https://usegalaxy.eu/published/workflow?id=0abd32dbc5207b88, https://usegalaxy.eu/published/workflow?id=0bafeb17c21ebc9c, https://usegalaxy.eu/published/workflow?id=0c47571d20535426, https://usegalaxy.eu/published/workflow?id=0f044c9f62773a88, https://usegalaxy.eu/published/workflow?id=10101558b211a782, https://usegalaxy.eu/published/workflow?id=10150b1a188f6d07, https://usegalaxy.eu/published/workflow?id=12a018217ee146ca, https://usegalaxy.eu/published/workflow?id=14714876dda7186d, https://usegalaxy.eu/published/workflow?id=181e751be8dd9abc, https://usegalaxy.eu/published/workflow?id=2497292e98937a5f, https://usegalaxy.eu/published/workflow?id=2b25eaebd3987612, https://usegalaxy.eu/published/workflow?id=2c2f0b4312bae4f4, https://usegalaxy.eu/published/workflow?id=2c79eee5ca5a3b09, https://usegalaxy.eu/published/workflow?id=2e0075791f7fa384, https://usegalaxy.eu/published/workflow?id=2e5e6291cd112225, https://usegalaxy.eu/published/workflow?id=2fa2f67603772413, https://usegalaxy.eu/published/workflow?id=309fc4fc2e0da89c, https://usegalaxy.eu/published/workflow?id=33a052f9a3d5551d, https://usegalaxy.eu/published/workflow?id=344cd3a3c2ce5302, https://usegalaxy.eu/published/workflow?id=39d36a7ce0ef268d, https://usegalaxy.eu/published/workflow?id=3dec7c9b02c3d1ee, https://usegalaxy.eu/published/workflow?id=3ff4dbcc7a602e6b, https://usegalaxy.eu/published/workflow?id=44e16f0782f54111, https://usegalaxy.eu/published/workflow?id=461d71caea871db8, https://usegalaxy.eu/published/workflow?id=47e14222073e7192, https://usegalaxy.eu/published/workflow?id=49c46d7653d6c349, https://usegalaxy.eu/published/workflow?id=4a39d38fde81e35d, https://usegalaxy.eu/published/workflow?id=4a764275d4bf9f31, https://usegalaxy.eu/published/workflow?id=4f0daf8e9dd54127, https://usegalaxy.eu/published/workflow?id=57cfe5062c8e01bf, https://usegalaxy.eu/published/workflow?id=57d3109f32defa3b, https://usegalaxy.eu/published/workflow?id=5826e573b6f93762, https://usegalaxy.eu/published/workflow?id=628062a8cf93261f, https://usegalaxy.eu/published/workflow?id=6a5c39c1d38a10c2, https://usegalaxy.eu/published/workflow?id=6f878d8b43ef1521, https://usegalaxy.eu/published/workflow?id=7491883694fff308, https://usegalaxy.eu/published/workflow?id=7d84b0419296857d, https://usegalaxy.eu/published/workflow?id=884a4415b669690c, https://usegalaxy.eu/published/workflow?id=889f2782ad82bd69, https://usegalaxy.eu/published/workflow?id=8a14cda6140acab7, https://usegalaxy.eu/published/workflow?id=8b2f53745e720d0f, https://usegalaxy.eu/published/workflow?id=8c7e972020799d57, https://usegalaxy.eu/published/workflow?id=8d64614d1971065a, https://usegalaxy.eu/published/workflow?id=8deb08d4f228e198, https://usegalaxy.eu/published/workflow?id=8ee3b635c1c380cd, https://usegalaxy.eu/published/workflow?id=94c3d329c31c712f, https://usegalaxy.eu/published/workflow?id=9588311136ac8794, https://usegalaxy.eu/published/workflow?id=98bfad0c684011e9, https://usegalaxy.eu/published/workflow?id=a0ea6fc756dd17ea, https://usegalaxy.eu/published/workflow?id=a337bfb2eb1e583f, https://usegalaxy.eu/published/workflow?id=a403c661bf1bc4fa, https://usegalaxy.eu/published/workflow?id=a459c311f461adce, https://usegalaxy.eu/published/workflow?id=a705370bc2c13d5c, https://usegalaxy.eu/published/workflow?id=a84bac86ec183d46, https://usegalaxy.eu/published/workflow?id=aafcfcb585c5ca75, https://usegalaxy.eu/published/workflow?id=ad8b6d73c9654305, https://usegalaxy.eu/published/workflow?id=b14845359b702444, https://usegalaxy.eu/published/workflow?id=b2b46e71046b1999, https://usegalaxy.eu/published/workflow?id=b37800d990cceb83, https://usegalaxy.eu/published/workflow?id=b4809f13c8f0c0be, https://usegalaxy.eu/published/workflow?id=b659069edd9b1d4e, https://usegalaxy.eu/published/workflow?id=b67b5ecd3305f830, https://usegalaxy.eu/published/workflow?id=bb63b5bb983fd49c, https://usegalaxy.eu/published/workflow?id=bd4e3cdc7f29049b, https://usegalaxy.eu/published/workflow?id=bdc7cfaac0db2eab, https://usegalaxy.eu/published/workflow?id=c09159d7aad0f264, https://usegalaxy.eu/published/workflow?id=c14d2e8b3c191bc4, https://usegalaxy.eu/published/workflow?id=c38e3ac59d8b86d6, https://usegalaxy.eu/published/workflow?id=c4fc3adf8f7ff2bc, https://usegalaxy.eu/published/workflow?id=c930a54a30dcc7e3, https://usegalaxy.eu/published/workflow?id=ca80039c4933aa0a, https://usegalaxy.eu/published/workflow?id=cb2df493f5e334f7, https://usegalaxy.eu/published/workflow?id=cfd37e9e307e5d3d, https://usegalaxy.eu/published/workflow?id=d0ae255151450dad, https://usegalaxy.eu/published/workflow?id=d88145e7475e0b7b, https://usegalaxy.eu/published/workflow?id=e69b555faa6bb962, https://usegalaxy.eu/published/workflow?id=ebb131f2a6172081, https://usegalaxy.eu/published/workflow?id=ec6b2c926fffab5f, https://usegalaxy.eu/published/workflow?id=ed2df13cfbc86b99, https://usegalaxy.eu/published/workflow?id=ee0005a5487653ba, https://usegalaxy.eu/published/workflow?id=ef6556d65ce5d72c, https://usegalaxy.eu/published/workflow?id=f97e6fe1fcbc53a7, https://usegalaxy.eu/published/workflow?id=fde21b5f5c169114, https://usegalaxy.fr/published/workflow?id=3d3b40398d1cd1ae, https://usegalaxy.org.au/published/workflow?id=0edb9c8b767fa74b, https://usegalaxy.org.au/published/workflow?id=131636a795bac485, https://usegalaxy.org.au/published/workflow?id=25d52afddaa3451b, https://usegalaxy.org.au/published/workflow?id=29ce205d98faf47e, https://usegalaxy.org.au/published/workflow?id=3d47e0de658b3304, https://usegalaxy.org.au/published/workflow?id=4047f9f242542716, https://usegalaxy.org.au/published/workflow?id=4c7e0703aace6aa3, https://usegalaxy.org.au/published/workflow?id=52065a4100b11184, https://usegalaxy.org.au/published/workflow?id=5693811cd933d2a2, https://usegalaxy.org.au/published/workflow?id=59618ed15510f531, https://usegalaxy.org.au/published/workflow?id=5fb44f257e445f56, https://usegalaxy.org.au/published/workflow?id=69d91340fc7effa2, https://usegalaxy.org.au/published/workflow?id=7a0c4804e4d4c905, https://usegalaxy.org.au/published/workflow?id=8034bd0690b4cbe8, https://usegalaxy.org.au/published/workflow?id=83441e3de699d157, https://usegalaxy.org.au/published/workflow?id=897a1255cef5c310, https://usegalaxy.org.au/published/workflow?id=92fcdd490831c6c5, https://usegalaxy.org.au/published/workflow?id=97e55a900db3863b, https://usegalaxy.org.au/published/workflow?id=a335b9c9103b57ad, https://usegalaxy.org.au/published/workflow?id=a61d4bda54de7cd1, https://usegalaxy.org.au/published/workflow?id=c92fead7c30c7497, https://usegalaxy.org.au/published/workflow?id=c997a34b55425155, https://usegalaxy.org.au/published/workflow?id=d2dcd3b54e5082a6, https://usegalaxy.org.au/published/workflow?id=d417e920ff97e4c1, https://usegalaxy.org.au/published/workflow?id=d720c98f7227b6d5, https://usegalaxy.org.au/published/workflow?id=d9ba165e6ae55417, https://usegalaxy.org.au/published/workflow?id=e20172eb0f30d6cd, https://usegalaxy.org.au/published/workflow?id=ed3e48a73b5aaa56, https://usegalaxy.org/published/workflow?id=1473cdcbd664d147, https://usegalaxy.org/published/workflow?id=1b80302437db5937, https://usegalaxy.org/published/workflow?id=1be5c982c2d36a6c, https://usegalaxy.org/published/workflow?id=21161075002d6778, https://usegalaxy.org/published/workflow?id=2a782ad24271f570, https://usegalaxy.org/published/workflow?id=3011f5dbf884c9bc, https://usegalaxy.org/published/workflow?id=323c651ca583c9e5, https://usegalaxy.org/published/workflow?id=4b5f6b5d4f36e38b, https://usegalaxy.org/published/workflow?id=4f698eaeef30e364, https://usegalaxy.org/published/workflow?id=574e42683dc3961b, https://usegalaxy.org/published/workflow?id=5b74b780d3955eb3, https://usegalaxy.org/published/workflow?id=5c259623a010fe77, https://usegalaxy.org/published/workflow?id=5e38c7b36e4918ad, https://usegalaxy.org/published/workflow?id=605160c81d098703, https://usegalaxy.org/published/workflow?id=60f0d0d5d6e93941, https://usegalaxy.org/published/workflow?id=623fc80c7f91a33a, https://usegalaxy.org/published/workflow?id=7321c6ad5425d347, https://usegalaxy.org/published/workflow?id=7c9a55db0b0eb178, https://usegalaxy.org/published/workflow?id=8f5904693b5f74f4, https://usegalaxy.org/published/workflow?id=9ba6da7dae465d3e, https://usegalaxy.org/published/workflow?id=9f68718f025bed30, https://usegalaxy.org/published/workflow?id=a63927dd19759714, https://usegalaxy.org/published/workflow?id=a8b9252fad5fe06a, https://usegalaxy.org/published/workflow?id=b68ec42db8953569, https://usegalaxy.org/published/workflow?id=b6a795834009ce06, https://usegalaxy.org/published/workflow?id=b7817a12dbee583c, https://usegalaxy.org/published/workflow?id=b7fe3873ee5e3a02, https://usegalaxy.org/published/workflow?id=badd4b56dbdeaba7, https://usegalaxy.org/published/workflow?id=bdd7011a908bc1aa, https://usegalaxy.org/published/workflow?id=c3ea8fa3bb202dc6, https://usegalaxy.org/published/workflow?id=c8f9e048ddae7d03, https://usegalaxy.org/published/workflow?id=cace9f1359874352, https://usegalaxy.org/published/workflow?id=d2c408df40dc311b, https://usegalaxy.org/published/workflow?id=d4f0a7ccb09f1e55, https://usegalaxy.org/published/workflow?id=d5436cee11a34de9, https://usegalaxy.org/published/workflow?id=ddf7a594b9609d68, https://usegalaxy.org/published/workflow?id=de9bf4c1024f8d53, https://usegalaxy.org/published/workflow?id=dfd73857c0eb2a21, https://usegalaxy.org/published/workflow?id=e12324f5e3b930fd, https://usegalaxy.org/published/workflow?id=e7ce46762faccaac, https://usegalaxy.org/published/workflow?id=e887adf3afc3253d, https://usegalaxy.org/published/workflow?id=ea206593698adbfd, https://usegalaxy.org/published/workflow?id=f378960ee55f567e, https://usegalaxy.org/published/workflow?id=fc2b48600853f3ea, https://workflowhub.eu/workflows/100?version=1, https://workflowhub.eu/workflows/101?version=1, https://workflowhub.eu/workflows/1035?version=1, https://workflowhub.eu/workflows/1052?version=11, https://workflowhub.eu/workflows/1059?version=1, https://workflowhub.eu/workflows/1061?version=1, https://workflowhub.eu/workflows/1199?version=2, https://workflowhub.eu/workflows/124?version=1, https://workflowhub.eu/workflows/1439?version=1, https://workflowhub.eu/workflows/1470?version=2, https://workflowhub.eu/workflows/1473?version=1, https://workflowhub.eu/workflows/1483?version=1, https://workflowhub.eu/workflows/1492?version=2, https://workflowhub.eu/workflows/1594?version=1, https://workflowhub.eu/workflows/1644?version=1, https://workflowhub.eu/workflows/1647?version=1, https://workflowhub.eu/workflows/1674?version=2, https://workflowhub.eu/workflows/1881?version=2, https://workflowhub.eu/workflows/1882?version=3, https://workflowhub.eu/workflows/2043?version=1, https://workflowhub.eu/workflows/53?version=1</td>
      <td>assembly/assembly-decontamination, ecology/bacterial-isolate-species-contamination-checking, microbiome/beer-data-analysis, microbiome/nanopore-16S-metagenomics, microbiome/pathogen-detection-from-nanopore-foodborne-data, microbiome/taxonomic-profiling, sequence-analysis/quality-contamination-control, variant-analysis/tb-variant-analysis</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>kraken2tax</td>
      <td>Kraken2Tax</td>
      <td>Convert Kraken output to Galaxy taxonomy data.</td>
      <td>2015-08-05</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>1.2+galaxy0</td>
      <td>gawk</td>
      <td></td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/blob/master/tool_collections/taxonomy/kraken2tax/</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/taxonomy/kraken2tax</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1365</td>
      <td>1376</td>
      <td>33158</td>
      <td>33186</td>
      <td>1892</td>
      <td>2555</td>
      <td>18299</td>
      <td>28718</td>
      <td>564</td>
      <td>576</td>
      <td>19721</td>
      <td>19871</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>81775</td>
      <td>71178</td>
      <td>4507</td>
      <td>3821</td>
      <td>https://usegalaxy.eu/published/workflow?id=0bafeb17c21ebc9c, https://usegalaxy.eu/published/workflow?id=2e0075791f7fa384, https://usegalaxy.eu/published/workflow?id=2ee0eb74e839b476, https://usegalaxy.eu/published/workflow?id=4a764275d4bf9f31, https://usegalaxy.eu/published/workflow?id=628062a8cf93261f, https://usegalaxy.eu/published/workflow?id=a459c311f461adce, https://usegalaxy.eu/published/workflow?id=b2b46e71046b1999, https://usegalaxy.eu/published/workflow?id=bd4e3cdc7f29049b, https://usegalaxy.eu/published/workflow?id=ca80039c4933aa0a, https://usegalaxy.eu/published/workflow?id=d88145e7475e0b7b, https://usegalaxy.eu/published/workflow?id=ee0005a5487653ba, https://usegalaxy.org/published/workflow?id=a63927dd19759714, https://usegalaxy.org/published/workflow?id=b68ec42db8953569, https://usegalaxy.org/published/workflow?id=d4f0a7ccb09f1e55, https://usegalaxy.org/published/workflow?id=e12324f5e3b930fd, https://workflowhub.eu/workflows/100?version=1, https://workflowhub.eu/workflows/101?version=1, https://workflowhub.eu/workflows/124?version=1</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>kraken_biom</td>
      <td>kraken_biom</td>
      <td>Create BIOM-format tables (http://biom-format.org) from Kraken output (http://ccb.jhu.edu/software/kraken/)</td>
      <td>2022-09-03</td>
      <td>https://github.com/smdabdoub/kraken-biom</td>
      <td>1.2.0</td>
      <td>kraken-biom</td>
      <td>1.2.0</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Genome annotation, Taxonomic classification</td>
      <td>Genome annotation, Taxonomic classification</td>
      <td>Metagenomics, Taxonomy</td>
      <td>Metagenomics, Taxonomy</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/kraken_biom</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/kraken_biom</td>
      <td>kraken-biom</td>
      <td>kraken-biom</td>
      <td>Create BIOM-format tables from Kraken output</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>857</td>
      <td>857</td>
      <td>4747</td>
      <td>4747</td>
      <td>47</td>
      <td>47</td>
      <td>150</td>
      <td>150</td>
      <td>95</td>
      <td>95</td>
      <td>963</td>
      <td>963</td>
      <td>9</td>
      <td>9</td>
      <td>17</td>
      <td>17</td>
      <td>5877</td>
      <td>5877</td>
      <td>1008</td>
      <td>1008</td>
      <td>https://usegalaxy.eu/published/workflow?id=39d36a7ce0ef268d, https://usegalaxy.eu/published/workflow?id=4f0daf8e9dd54127, https://usegalaxy.eu/published/workflow?id=a0ea6fc756dd17ea, https://usegalaxy.eu/published/workflow?id=b67b5ecd3305f830, https://usegalaxy.eu/published/workflow?id=e69b555faa6bb962, https://workflowhub.eu/workflows/1470?version=2</td>
      <td>microbiome/taxonomic-profiling</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>kraken_taxonomy_report</td>
      <td>kraken_taxonomy_report</td>
      <td>Kraken taxonomy report</td>
      <td>2016-06-01</td>
      <td>https://github.com/blankenberg/Kraken-Taxonomy-Report</td>
      <td>0.0.3</td>
      <td>biopython</td>
      <td>1.70</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Visualisation, Classification</td>
      <td>Visualisation, Classification</td>
      <td>Metagenomics, Taxonomy</td>
      <td>Metagenomics, Taxonomy</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/kraken_taxonomy_report</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/kraken_taxonomy_report</td>
      <td>Kraken-Taxonomy-Report</td>
      <td>Kraken-Taxonomy-Report</td>
      <td>view report of classification for multiple samples</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1148</td>
      <td>1155</td>
      <td>5719</td>
      <td>5735</td>
      <td>1291</td>
      <td>1717</td>
      <td>5613</td>
      <td>7684</td>
      <td>148</td>
      <td>148</td>
      <td>587</td>
      <td>587</td>
      <td>10</td>
      <td>10</td>
      <td>25</td>
      <td>25</td>
      <td>14031</td>
      <td>11944</td>
      <td>3030</td>
      <td>2597</td>
      <td>https://usegalaxy.eu/published/workflow?id=2e5e6291cd112225, https://usegalaxy.eu/published/workflow?id=b659069edd9b1d4e</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>krakentools</td>
      <td>krakentools_alpha_diversity, krakentools_beta_diversity, krakentools_combine_kreports, krakentools_extract_kraken_reads, krakentools_kreport2krona, krakentools_kreport2mpa</td>
      <td>KrakenTools is a suite of scripts to be used alongside the Kraken</td>
      <td>2023-01-13</td>
      <td>https://github.com/jenniferlu717/KrakenTools</td>
      <td>1.2.1</td>
      <td>krakentools</td>
      <td>1.2.1</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Visualisation, Aggregation</td>
      <td>Visualisation, Aggregation</td>
      <td>Taxonomy, Metagenomics</td>
      <td>Taxonomy, Metagenomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/krakentools</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/krakentools</td>
      <td>krakentools</td>
      <td>KrakenTools</td>
      <td>KrakenTools provides individual scripts to analyze Kraken/Kraken2/Bracken/KrakenUniq output files</td>
      <td></td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>6</td>
      <td>1</td>
      <td>0</td>
      <td>2603</td>
      <td>2603</td>
      <td>161915</td>
      <td>161915</td>
      <td>1861</td>
      <td>1861</td>
      <td>40010</td>
      <td>40010</td>
      <td>488</td>
      <td>488</td>
      <td>18602</td>
      <td>18602</td>
      <td>64</td>
      <td>64</td>
      <td>5824</td>
      <td>5824</td>
      <td>226351</td>
      <td>226351</td>
      <td>5016</td>
      <td>5016</td>
      <td>https://usegalaxy.eu/published/workflow?id=0590554db9794d13, https://usegalaxy.eu/published/workflow?id=10101558b211a782, https://usegalaxy.eu/published/workflow?id=181e751be8dd9abc, https://usegalaxy.eu/published/workflow?id=2b25eaebd3987612, https://usegalaxy.eu/published/workflow?id=309fc4fc2e0da89c, https://usegalaxy.eu/published/workflow?id=344cd3a3c2ce5302, https://usegalaxy.eu/published/workflow?id=39d36a7ce0ef268d, https://usegalaxy.eu/published/workflow?id=461d71caea871db8, https://usegalaxy.eu/published/workflow?id=4f0daf8e9dd54127, https://usegalaxy.eu/published/workflow?id=6f878d8b43ef1521, https://usegalaxy.eu/published/workflow?id=7491883694fff308, https://usegalaxy.eu/published/workflow?id=8deb08d4f228e198, https://usegalaxy.eu/published/workflow?id=94c3d329c31c712f, https://usegalaxy.eu/published/workflow?id=98bfad0c684011e9, https://usegalaxy.eu/published/workflow?id=a705370bc2c13d5c, https://usegalaxy.eu/published/workflow?id=ad8b6d73c9654305, https://usegalaxy.eu/published/workflow?id=b14845359b702444, https://usegalaxy.eu/published/workflow?id=b37800d990cceb83, https://usegalaxy.eu/published/workflow?id=b4809f13c8f0c0be, https://usegalaxy.eu/published/workflow?id=c38e3ac59d8b86d6, https://usegalaxy.eu/published/workflow?id=cb2df493f5e334f7, https://usegalaxy.eu/published/workflow?id=ed2df13cfbc86b99, https://usegalaxy.eu/published/workflow?id=f97e6fe1fcbc53a7, https://usegalaxy.eu/published/workflow?id=fde21b5f5c169114, https://usegalaxy.org.au/published/workflow?id=131636a795bac485, https://usegalaxy.org.au/published/workflow?id=25d52afddaa3451b, https://usegalaxy.org.au/published/workflow?id=69d91340fc7effa2, https://usegalaxy.org.au/published/workflow?id=a335b9c9103b57ad, https://usegalaxy.org.au/published/workflow?id=d9ba165e6ae55417, https://usegalaxy.org.au/published/workflow?id=ed3e48a73b5aaa56, https://usegalaxy.org/published/workflow?id=3011f5dbf884c9bc, https://usegalaxy.org/published/workflow?id=4b5f6b5d4f36e38b, https://usegalaxy.org/published/workflow?id=574e42683dc3961b, https://usegalaxy.org/published/workflow?id=60f0d0d5d6e93941, https://usegalaxy.org/published/workflow?id=7321c6ad5425d347, https://usegalaxy.org/published/workflow?id=8f5904693b5f74f4, https://usegalaxy.org/published/workflow?id=9ba6da7dae465d3e, https://usegalaxy.org/published/workflow?id=a8b9252fad5fe06a, https://usegalaxy.org/published/workflow?id=d5436cee11a34de9, https://usegalaxy.org/published/workflow?id=e12324f5e3b930fd, https://workflowhub.eu/workflows/1059?version=1, https://workflowhub.eu/workflows/1061?version=1, https://workflowhub.eu/workflows/1199?version=2, https://workflowhub.eu/workflows/1431?version=1, https://workflowhub.eu/workflows/1439?version=1, https://workflowhub.eu/workflows/1470?version=2, https://workflowhub.eu/workflows/1483?version=1, https://workflowhub.eu/workflows/1492?version=2, https://workflowhub.eu/workflows/2024?version=2</td>
      <td>microbiome/beer-data-analysis, microbiome/diversity, microbiome/pathogen-detection-from-nanopore-foodborne-data, microbiome/taxonomic-profiling</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>krocus</td>
      <td>krocus</td>
      <td>Predict MLST directly from uncorrected long reads</td>
      <td>2019-09-03</td>
      <td>https://github.com/quadram-institute-bioscience/krocus</td>
      <td>1.0.1</td>
      <td>krocus</td>
      <td>1.0.3</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Multilocus sequence typing, k-mer counting</td>
      <td>Multilocus sequence typing, k-mer counting</td>
      <td>Public health and epidemiology</td>
      <td>Public health and epidemiology</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/krocus</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/krocus</td>
      <td>krocus</td>
      <td>krocus</td>
      <td>Predict MLST directly from uncorrected long reads</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>lca_wrapper</td>
      <td>lca1</td>
      <td>Find lowest diagnostic rank</td>
      <td>2014-01-27</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>1.0.1</td>
      <td>taxonomy</td>
      <td>0.10.2</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/taxonomy/lca_wrapper</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/taxonomy/lca_wrapper</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>3</td>
      <td>20</td>
      <td>11</td>
      <td>137</td>
      <td>14</td>
      <td>1112</td>
      <td>112</td>
      <td>6136</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6273</td>
      <td>123</td>
      <td>1132</td>
      <td>17</td>
      <td>https://usegalaxy.org/published/workflow?id=19c9c8cc3c813bda, https://usegalaxy.org/published/workflow?id=2f7638035fb889ac, https://usegalaxy.org/published/workflow?id=55aa743125c2465d, https://usegalaxy.org/published/workflow?id=59347cc210c4cdbc, https://usegalaxy.org/published/workflow?id=7294f75ad01ca7ed, https://usegalaxy.org/published/workflow?id=7b67ffcf1b7c2232, https://usegalaxy.org/published/workflow?id=85d825931edf4a64, https://usegalaxy.org/published/workflow?id=896d29573ea3e574, https://usegalaxy.org/published/workflow?id=d851f8436497dd1c</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>legsta</td>
      <td>legsta</td>
      <td>Performs in silico Legionella pneumophila sequence based typing.</td>
      <td>2022-02-21</td>
      <td>https://github.com/tseemann/legsta</td>
      <td>0.5.1</td>
      <td>legsta</td>
      <td>0.5.1</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Sequence analysis</td>
      <td>Sequence analysis</td>
      <td>Public health and epidemiology</td>
      <td>Public health and epidemiology</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/legsta</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/legsta</td>
      <td>legsta</td>
      <td>legsta</td>
      <td>Performs in silico Legionella pneumophila sequence based typing</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>13</td>
      <td>13</td>
      <td>134</td>
      <td>134</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>134</td>
      <td>134</td>
      <td>13</td>
      <td>13</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>lighter</td>
      <td>lighter</td>
      <td>Lighter is a kmer-based error correction method for whole genome sequencing data</td>
      <td>2016-06-04</td>
      <td>https://github.com/mourisl/Lighter</td>
      <td>1.0</td>
      <td>lighter</td>
      <td>1.1.3</td>
      <td>To update</td>
      <td>Sequence Analysis, Fasta Manipulation</td>
      <td>k-mer counting, Sequence read processing, Sequencing quality control, Sequencing error detection</td>
      <td>k-mer counting, Sequence read processing, Sequencing quality control, Sequencing error detection</td>
      <td>Sequencing, Whole genome sequencing, DNA, Genomics</td>
      <td>Whole genome sequencing, DNA, Genomics</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/lighter</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/lighter</td>
      <td>lighter</td>
      <td>Lighter</td>
      <td>Kmer-based error correction method for whole genome sequencing data. Lighter uses sampling (rather than counting) to obtain a set of kmers that are likely from the genome. Using this information, Lighter can correct the reads containing sequence errors.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>31</td>
      <td>37</td>
      <td>106</td>
      <td>196</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>196</td>
      <td>106</td>
      <td>37</td>
      <td>31</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>limma_voom</td>
      <td>limma_voom</td>
      <td>Perform RNA-Seq differential expression analysis using limma voom pipeline</td>
      <td>2017-05-24</td>
      <td>http://bioconductor.org/packages/release/bioc/html/limma.html</td>
      <td>3.58.1</td>
      <td>bioconductor-limma</td>
      <td>3.62.1</td>
      <td>To update</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>RNA-Seq analysis</td>
      <td>RNA-Seq analysis</td>
      <td>Molecular biology, Genetics</td>
      <td>Molecular biology, Genetics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/limma_voom</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/limma_voom</td>
      <td>limma</td>
      <td>limma</td>
      <td>Data analysis, linear models and differential expression for microarray data.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2902</td>
      <td>3166</td>
      <td>27013</td>
      <td>30001</td>
      <td>5247</td>
      <td>6265</td>
      <td>50381</td>
      <td>59995</td>
      <td>921</td>
      <td>1003</td>
      <td>10130</td>
      <td>11326</td>
      <td>26</td>
      <td>26</td>
      <td>329</td>
      <td>329</td>
      <td>101651</td>
      <td>87853</td>
      <td>10460</td>
      <td>9096</td>
      <td>https://usegalaxy.eu/published/workflow?id=09c77b6fc5add851, https://usegalaxy.eu/published/workflow?id=14226641bba951f3, https://usegalaxy.eu/published/workflow?id=1590085d315aa80e, https://usegalaxy.eu/published/workflow?id=1c1083b159e5bfb2, https://usegalaxy.eu/published/workflow?id=1d2e17a4f71f2455, https://usegalaxy.eu/published/workflow?id=21dd4bd16579fc12, https://usegalaxy.eu/published/workflow?id=227d97974573aaf2, https://usegalaxy.eu/published/workflow?id=263eed267aa3e76b, https://usegalaxy.eu/published/workflow?id=2a73d658446747de, https://usegalaxy.eu/published/workflow?id=2ecf737b53f77cf8, https://usegalaxy.eu/published/workflow?id=2f37c35ade97aa8d, https://usegalaxy.eu/published/workflow?id=2f7c04ad219a9d2d, https://usegalaxy.eu/published/workflow?id=3386856a60b35897, https://usegalaxy.eu/published/workflow?id=47f2c44d79896f08, https://usegalaxy.eu/published/workflow?id=48fe9c4de045d414, https://usegalaxy.eu/published/workflow?id=67e20e3082d24fbd, https://usegalaxy.eu/published/workflow?id=6d47a24218d08930, https://usegalaxy.eu/published/workflow?id=6fc2b9bea5b75a49, https://usegalaxy.eu/published/workflow?id=7036a6cf67d53bc7, https://usegalaxy.eu/published/workflow?id=732fa536cdc70c96, https://usegalaxy.eu/published/workflow?id=8ab363878cd34ef0, https://usegalaxy.eu/published/workflow?id=8bc1c3b9e665ea37, https://usegalaxy.eu/published/workflow?id=8f095e5b9594b098, https://usegalaxy.eu/published/workflow?id=9a83043d5e847318, https://usegalaxy.eu/published/workflow?id=ad3be77eb2e8c10e, https://usegalaxy.eu/published/workflow?id=bb4ecac751a94c79, https://usegalaxy.eu/published/workflow?id=c8deb8952abce1ec, https://usegalaxy.eu/published/workflow?id=d6eb0562d0036940, https://usegalaxy.eu/published/workflow?id=dd8b55e7768843b1, https://usegalaxy.eu/published/workflow?id=f28459c5b16549b7, https://usegalaxy.eu/published/workflow?id=f9ddc4dd9140b293, https://usegalaxy.eu/published/workflow?id=fa83e26e05418370, https://usegalaxy.eu/published/workflow?id=fe64d430d50faf0b, https://usegalaxy.eu/published/workflow?id=fe6c64676be56ee9, https://usegalaxy.org.au/published/workflow?id=bdb498b1f0e8944f, https://usegalaxy.org/published/workflow?id=07fdeadce7a6d5cb, https://usegalaxy.org/published/workflow?id=092be3cf6e40282c, https://usegalaxy.org/published/workflow?id=0a009cf9e88171f1, https://usegalaxy.org/published/workflow?id=0c052e50594f6d2d, https://usegalaxy.org/published/workflow?id=173ecc0c88090d0e, https://usegalaxy.org/published/workflow?id=204d0da8a023d7da, https://usegalaxy.org/published/workflow?id=228ae7f537a60226, https://usegalaxy.org/published/workflow?id=3f8e0395007c8e9e, https://usegalaxy.org/published/workflow?id=427fce51fd9508eb, https://usegalaxy.org/published/workflow?id=43c39bc86399188f, https://usegalaxy.org/published/workflow?id=5203bc68df3cf819, https://usegalaxy.org/published/workflow?id=5944c5effd56b0d6, https://usegalaxy.org/published/workflow?id=6d0a3f36d2736087, https://usegalaxy.org/published/workflow?id=734e08331f8f34b7, https://usegalaxy.org/published/workflow?id=93650de42fb8c015, https://usegalaxy.org/published/workflow?id=a89def2c2462674a, https://usegalaxy.org/published/workflow?id=aac2ae0419f7d515, https://usegalaxy.org/published/workflow?id=b094b6e1e7d48416, https://usegalaxy.org/published/workflow?id=b3823d276917baf0, https://usegalaxy.org/published/workflow?id=cd56ed91856fcc6f, https://usegalaxy.org/published/workflow?id=d5514e327878b531, https://usegalaxy.org/published/workflow?id=ddb737f2ad0589e8, https://usegalaxy.org/published/workflow?id=dec0cf06fa059e65, https://workflowhub.eu/workflows/1684?version=1, https://workflowhub.eu/workflows/1703?version=1, https://workflowhub.eu/workflows/689?version=1</td>
      <td>transcriptomics/minerva-pathways, transcriptomics/rna-seq-counts-to-genes</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>lineagespot</td>
      <td>lineagespot</td>
      <td>Identification of SARS-CoV-2 related metagenomic mutations based on a single (or a list of) variant(s) file(s)</td>
      <td>2023-07-04</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/lineagespot.html</td>
      <td>1.6.0</td>
      <td>r-base</td>
      <td></td>
      <td>To update</td>
      <td>Metagenomics, Sequence Analysis</td>
      <td>Variant calling</td>
      <td>Variant calling</td>
      <td>Metagenomics, Gene transcripts, Evolutionary biology, Sequencing, Genetic variation</td>
      <td>Metagenomics, Gene transcripts, Evolutionary biology, Sequencing, Genetic variation</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/lineagespot</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/lineagespot</td>
      <td>lineagespot</td>
      <td>lineagespot</td>
      <td>Lineagespot is a framework written in R, and aims to identify and assign different SARS-CoV-2 lineages based on a single variant file (i.e., variant calling format).</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>39</td>
      <td>39</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>39</td>
      <td>39</td>
      <td>4</td>
      <td>4</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>logol</td>
      <td>logol_wrapper</td>
      <td>Logol is a pattern matching grammar language and a set of tools to search a pattern in a sequence</td>
      <td>2018-06-06</td>
      <td>http://logol.genouest.org/web/app.php/logol</td>
      <td>1.7.8</td>
      <td>logol</td>
      <td>1.7.8</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/logol</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/logol</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>longdust</td>
      <td>longdust</td>
      <td>Detect low-complexity regions in long DNA sequences using k-mer statistics.</td>
      <td>2025-10-01</td>
      <td>https://github.com/lh3/longdust</td>
      <td>1.4</td>
      <td>longdust</td>
      <td>1.4</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/longdust</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/longdust</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>longorf</td>
      <td>longORF</td>
      <td>obtain longest ORF in six-frame translations</td>
      <td>2018-06-01</td>
      <td></td>
      <td>0.3.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>mbernt</td>
      <td>https://github.com/bernt-matthias/mb-galaxy-tools/blob/master/tools/longorf/</td>
      <td>https://github.com/Helmholtz-UFZ/galaxy-tools/tree/main/tools/longorf</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>lorikeet_spoligotype</td>
      <td>lorikeet_spoligotype</td>
      <td>Tools for M. tuberculosis DNA fingerprinting (spoligotyping)</td>
      <td>2018-05-07</td>
      <td>https://github.com/AbeelLab/lorikeet</td>
      <td>20</td>
      <td>lorikeet</td>
      <td>20</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Sequence analysis, Genotyping</td>
      <td>Sequence analysis, Genotyping</td>
      <td>Genotype and phenotype</td>
      <td>Genotype and phenotype</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/lorikeet</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/lorikeet</td>
      <td>lorikeet</td>
      <td>lorikeet</td>
      <td>Tools for M. tuberculosis DNA fingerprinting (spoligotyping)</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>18</td>
      <td>18</td>
      <td>930</td>
      <td>930</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>930</td>
      <td>930</td>
      <td>18</td>
      <td>18</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>lotus2</td>
      <td>lotus2</td>
      <td>LotuS2 OTU processing pipeline</td>
      <td>2021-05-13</td>
      <td>https://lotus2.earlham.ac.uk/</td>
      <td>2.32</td>
      <td>lotus2</td>
      <td>2.34.1</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Sequence feature detection, DNA barcoding</td>
      <td>Sequence feature detection, DNA barcoding</td>
      <td>Metagenomics, Taxonomy, Microbial ecology</td>
      <td>Metagenomics, Taxonomy, Microbial ecology</td>
      <td>earlhaminst</td>
      <td>https://github.com/EarlhamInst/earlham-galaxytools/tree/main/tools/lotus2</td>
      <td>https://github.com/EarlhamInst/earlham-galaxytools/tree/main/tools/lotus2</td>
      <td>lotus2</td>
      <td>lotus2</td>
      <td>LotuS2 is a lightweight and user-friendly pipeline that is fast, precise, and streamlined, using extensive pre- and post-ASV/OTU clustering steps to further increase data quality. High data usage rates and reliability enable high-throughput microbiome analysis in minutes.</td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>249</td>
      <td>249</td>
      <td>1979</td>
      <td>1979</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>22</td>
      <td>22</td>
      <td>417</td>
      <td>417</td>
      <td>2</td>
      <td>2</td>
      <td>5</td>
      <td>5</td>
      <td>2401</td>
      <td>2401</td>
      <td>273</td>
      <td>273</td>
      <td>https://usegalaxy.eu/published/workflow?id=466bdd8ba7b67264, https://workflowhub.eu/workflows/1460?version=1</td>
      <td>microbiome/lotus2-identifying-fungi, microbiome/mgnify-amplicon</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>m6anet</td>
      <td>m6anet</td>
      <td>m6anet to detect m6A RNA modifications from nanopore data</td>
      <td>2023-10-25</td>
      <td>https://m6anet.readthedocs.io/en/latest</td>
      <td>2.1.0</td>
      <td>m6anet</td>
      <td>2.1.0</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Quantification, Imputation, Gene expression profiling</td>
      <td>Quantification, Imputation, Gene expression profiling</td>
      <td>RNA-Seq, Transcriptomics, RNA, Machine learning</td>
      <td>RNA-Seq, Transcriptomics, RNA, Machine learning</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/m6anet</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/m6anet</td>
      <td>m6Anet</td>
      <td>m6Anet</td>
      <td>Detection of m6A from direct RNA sequencing using a Multiple Instance Learning framework.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>12</td>
      <td>12</td>
      <td>47</td>
      <td>47</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>47</td>
      <td>47</td>
      <td>12</td>
      <td>12</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>maaslin2</td>
      <td>maaslin2</td>
      <td>MaAsLin2 is comprehensive R package for efficiently determining multivariable association between microbial meta'omic features and clinical metadata.</td>
      <td>2021-11-05</td>
      <td>http://huttenhower.sph.harvard.edu/maaslin</td>
      <td>1.18.0</td>
      <td>bioconductor-maaslin2</td>
      <td>1.18.0</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Filtering, Statistical calculation, Standardisation and normalisation, Visualisation</td>
      <td>Filtering, Standardisation and normalisation, Visualisation</td>
      <td>Metagenomics, Statistics and probability</td>
      <td>Metagenomics, Statistics and probability</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/maaslin2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/maaslin2</td>
      <td>maaslin2</td>
      <td>MaAsLin2</td>
      <td>MaAsLin2 is comprehensive R package for efficiently determining multivariable association between phenotypes, environments, exposures, covariates and microbial metaomic features. MaAsLin2 relies on general linear models to accommodate most modern epidemiological study designs, including cross-sectional and longitudinal, and offers a variety of data exploration, normalization, and transformation methods.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>100</td>
      <td>100</td>
      <td>3676</td>
      <td>3676</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>4</td>
      <td>4</td>
      <td>3680</td>
      <td>3680</td>
      <td>102</td>
      <td>102</td>
      <td>https://usegalaxy.eu/published/workflow?id=466bdd8ba7b67264, https://usegalaxy.eu/published/workflow?id=4ddf2ec5c460d622, https://usegalaxy.eu/published/workflow?id=e1296dbcf34499b0, https://usegalaxy.eu/published/workflow?id=e69b555faa6bb962</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>maaslin3</td>
      <td>maaslin3</td>
      <td>MaAsLin 3 is the next generation of MaAsLin. This comprehensive R package efficiently determines multivariable associations between clinical metadata and microbial meta-omics features.</td>
      <td>2025-11-05</td>
      <td>https://github.com/biobakery/maaslin3</td>
      <td>0.99.16</td>
      <td>maaslin3</td>
      <td>0.99.16</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/maaslin3</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/maaslin3</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>mafft</td>
      <td>rbc_mafft_add, rbc_mafft</td>
      <td>Multiple alignment program for amino acid or nucleotide sequences</td>
      <td>2015-06-16</td>
      <td>https://mafft.cbrc.jp/alignment/software/</td>
      <td>7.526</td>
      <td>mafft</td>
      <td>7.525</td>
      <td>To update</td>
      <td>RNA</td>
      <td>Multiple sequence alignment</td>
      <td>Multiple sequence alignment</td>
      <td>Sequence analysis</td>
      <td>Sequence analysis</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/mafft</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/mafft</td>
      <td>MAFFT</td>
      <td>MAFFT</td>
      <td>MAFFT (Multiple Alignment using Fast Fourier Transform) is a high speed multiple sequence alignment program.</td>
      <td></td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>2844</td>
      <td>3060</td>
      <td>164403</td>
      <td>217226</td>
      <td>4659</td>
      <td>6641</td>
      <td>99571</td>
      <td>145229</td>
      <td>1279</td>
      <td>1462</td>
      <td>23332</td>
      <td>25716</td>
      <td>158</td>
      <td>160</td>
      <td>5367</td>
      <td>5382</td>
      <td>393553</td>
      <td>292673</td>
      <td>11323</td>
      <td>8940</td>
      <td>https://usegalaxy.eu/published/workflow?id=402b14906a03b2ef, https://usegalaxy.eu/published/workflow?id=5cd167ed9e159e73, https://usegalaxy.eu/published/workflow?id=86900be4282d4b1d, https://usegalaxy.eu/published/workflow?id=9155ed0e7a6a7b60, https://usegalaxy.eu/published/workflow?id=941400d10c346b2c, https://usegalaxy.eu/published/workflow?id=a06b1ed5cd25922f, https://usegalaxy.eu/published/workflow?id=a15dabcadb63b46d, https://usegalaxy.eu/published/workflow?id=a2c46deea34d9d80, https://usegalaxy.eu/published/workflow?id=b1a799497c366eef, https://usegalaxy.eu/published/workflow?id=c789ab71876c6ead, https://usegalaxy.eu/published/workflow?id=cd25f70f71683ab6, https://usegalaxy.eu/published/workflow?id=d102a20c31d4673c, https://usegalaxy.eu/published/workflow?id=e7f03b4bd9b3d6fd, https://usegalaxy.eu/published/workflow?id=e97b938ca4723827, https://usegalaxy.eu/published/workflow?id=ec50dd6d56f50db1, https://usegalaxy.fr/published/workflow?id=297b71156b297a73, https://usegalaxy.fr/published/workflow?id=5ec6d378e60f7ccc, https://usegalaxy.fr/published/workflow?id=63b49f322ec366b5, https://usegalaxy.fr/published/workflow?id=7ebb54181b404733, https://usegalaxy.fr/published/workflow?id=894143d81af8dff0, https://usegalaxy.fr/published/workflow?id=b131218bec7eacb4, https://usegalaxy.fr/published/workflow?id=b3974acb70608e38, https://usegalaxy.fr/published/workflow?id=c7be6e229a045fad, https://usegalaxy.fr/published/workflow?id=d425a77157cae2ee, https://usegalaxy.fr/published/workflow?id=e134e9637adfc5b6, https://usegalaxy.org.au/published/workflow?id=3242ec69dec6e13f, https://usegalaxy.org.au/published/workflow?id=42137e36791652bc, https://usegalaxy.org.au/published/workflow?id=7559224fc32dce3a, https://usegalaxy.org.au/published/workflow?id=a01c3004bcef9dd3, https://usegalaxy.org.au/published/workflow?id=d62b6717202f42c9, https://usegalaxy.org.au/published/workflow?id=f21446cc44da50ce, https://usegalaxy.org/published/workflow?id=1518c8c50747a1c1, https://usegalaxy.org/published/workflow?id=23212ae74fc2d2b5, https://usegalaxy.org/published/workflow?id=2d1ca299c1b058f6, https://usegalaxy.org/published/workflow?id=603e5c61c57bcf3f, https://usegalaxy.org/published/workflow?id=6a1eb7e7bececba1, https://usegalaxy.org/published/workflow?id=7251c6e0dad8f927, https://usegalaxy.org/published/workflow?id=75c2cf36253194c2, https://usegalaxy.org/published/workflow?id=8d8e8927d63803ef, https://usegalaxy.org/published/workflow?id=956987e628530c79, https://usegalaxy.org/published/workflow?id=97263f3308077ff0, https://usegalaxy.org/published/workflow?id=9c4038fa15af6995, https://usegalaxy.org/published/workflow?id=b0f85c802c1ea7f8, https://usegalaxy.org/published/workflow?id=be54e57345ce7ca5, https://usegalaxy.org/published/workflow?id=f737593a1db217c8, https://workflowhub.eu/workflows/1260?version=2, https://workflowhub.eu/workflows/1518?version=1, https://workflowhub.eu/workflows/1559?version=1, https://workflowhub.eu/workflows/1627?version=1, https://workflowhub.eu/workflows/1650?version=1, https://workflowhub.eu/workflows/1839?version=1</td>
      <td>evolution/abc_intro_phylo, genome-annotation/gene-centric, sequence-analysis/tapscan-streptophyte-algae, sequence-analysis/viral_primer_design, variant-analysis/aiv-analysis, variant-analysis/pox-tiled-amplicon</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>make_nr</td>
      <td>make_nr</td>
      <td>Make a FASTA file non-redundant</td>
      <td>2018-11-06</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/make_nr</td>
      <td>0.0.3</td>
      <td>biopython</td>
      <td>1.70</td>
      <td>To update</td>
      <td>Fasta Manipulation, Sequence Analysis</td>
      <td>Sequence alignment, Genome annotation</td>
      <td>Sequence alignment, Genome annotation</td>
      <td>Molecular biology, Sequence analysis</td>
      <td>Molecular biology, Sequence analysis</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/make_nr</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/make_nr</td>
      <td>make_nr</td>
      <td>make_nr</td>
      <td>Make a FASTA file non-redundant, with a Galaxy wrapperThis tool is copyright 2018 by Peter Cock, The James Hutton Institute, UK. All rights reserved.This tool is a short Python script intended to be run prior to calling the NCBI BLAST+ command line tool makeblastdb or in other settings where you want to collapse duplicated sequences in a FASTA file to a single representative.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>maker</td>
      <td>maker, maker_map_ids</td>
      <td>MAKER is a portable and easily configurable genome annotation pipeline.Its purpose is to allow smaller eukaryotic and prokaryotic genome projects to independently annotate their genomes and to create genome databases.</td>
      <td>2017-10-04</td>
      <td>http://www.yandell-lab.org/software/maker.html</td>
      <td>2.31.11</td>
      <td>maker</td>
      <td>3.01.04</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Genome annotation</td>
      <td>Genome annotation</td>
      <td>Genomics, DNA, Sequence analysis</td>
      <td>Genomics, DNA, Sequence analysis</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/maker</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/maker</td>
      <td>maker</td>
      <td>MAKER</td>
      <td>Portable and easily configurable genome annotation pipeline. Its purpose is to allow smaller eukaryotic and prokaryotic genome projects to independently annotate their genomes and to create genome databases.</td>
      <td></td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>1323</td>
      <td>1455</td>
      <td>7332</td>
      <td>8835</td>
      <td>1701</td>
      <td>1808</td>
      <td>9536</td>
      <td>10359</td>
      <td>904</td>
      <td>904</td>
      <td>4968</td>
      <td>4989</td>
      <td>33</td>
      <td>33</td>
      <td>211</td>
      <td>211</td>
      <td>24394</td>
      <td>22047</td>
      <td>4200</td>
      <td>3961</td>
      <td>https://usegalaxy.eu/published/workflow?id=26ad77bc07a7e512, https://usegalaxy.eu/published/workflow?id=6ca20e710f4f3ffb, https://usegalaxy.org.au/published/workflow?id=7e40485d1ed45b7c, https://usegalaxy.org/published/workflow?id=65b4051e722228b8, https://usegalaxy.org/published/workflow?id=dbe08700353dfc43, https://workflowhub.eu/workflows/1323?version=1, https://workflowhub.eu/workflows/1504?version=1, https://workflowhub.eu/workflows/1549?version=1</td>
      <td>genome-annotation/annotation-with-maker, genome-annotation/annotation-with-maker-short</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>mapseq</td>
      <td>mapseq</td>
      <td>fast and accurate sequence read classification tool designed to assign taxonomy and OTU classifications to ribosomal RNA sequences.</td>
      <td>2023-08-02</td>
      <td>https://github.com/jfmrod/MAPseq</td>
      <td>2.1.1b</td>
      <td>perl</td>
      <td></td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>k-mer counting</td>
      <td>k-mer counting</td>
      <td>Functional, regulatory and non-coding RNA, Sequence analysis, Sequence sites, features and motifs</td>
      <td>Functional, regulatory and non-coding RNA, Sequence analysis, Sequence sites, features and motifs</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mapseq</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mapseq</td>
      <td>mapseq</td>
      <td>MAPseq</td>
      <td>Highly efficient k-mer search with confidence estimates, for rRNA sequence analysis .</td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>64</td>
      <td>64</td>
      <td>9612</td>
      <td>9612</td>
      <td>24</td>
      <td>24</td>
      <td>434</td>
      <td>434</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10046</td>
      <td>10046</td>
      <td>88</td>
      <td>88</td>
      <td>https://usegalaxy.eu/published/workflow?id=08e25b733cd796e6, https://usegalaxy.eu/published/workflow?id=437704898229dfb6, https://usegalaxy.eu/published/workflow?id=fb806b88850f3427, https://usegalaxy.org/published/workflow?id=45f959d5bf505b15, https://usegalaxy.org/published/workflow?id=8151be18aab3a3c6, https://workflowhub.eu/workflows/1270?version=2, https://workflowhub.eu/workflows/1273?version=2, https://workflowhub.eu/workflows/1842?version=1, https://workflowhub.eu/workflows/1856?version=1</td>
      <td>microbiome/mgnify-amplicon</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>marine_omics</td>
      <td>sanntis_marine</td>
      <td>The Sanntis tool identify biosynthetic gene clusters (BGCs) in genomic &amp; metagenomic data</td>
      <td>2024-07-26</td>
      <td>https://github.com/Finn-Lab/SanntiS</td>
      <td>0.9.3.5</td>
      <td>sanntis</td>
      <td>0.9.4.1</td>
      <td>To update</td>
      <td>Ecology</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/marine_omics</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/marine_omics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>22</td>
      <td>22</td>
      <td>198</td>
      <td>198</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>112</td>
      <td>112</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>310</td>
      <td>310</td>
      <td>27</td>
      <td>27</td>
      <td>https://usegalaxy.eu/published/workflow?id=a448400f69594135, https://usegalaxy.eu/published/workflow?id=b9c938d1af08124b, https://usegalaxy.eu/published/workflow?id=cc59ae1a9878942c, https://workflowhub.eu/workflows/1663?version=1</td>
      <td>ecology/marine_omics_bgc</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>mash</td>
      <td>mash_dist, mash_paste, mash_screen, mash_sketch</td>
      <td>Fast genome and metagenome distance estimation using MinHash</td>
      <td>2018-12-07</td>
      <td>https://github.com/marbl/Mash</td>
      <td>2.3</td>
      <td>mash</td>
      <td>2.3</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Sequence distance matrix generation</td>
      <td>Sequence distance matrix generation</td>
      <td>Genomics, Metagenomics, Statistics and probability, Sequence analysis, DNA mutation</td>
      <td>Metagenomics, Statistics and probability, Sequence analysis, DNA mutation</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mash</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mash</td>
      <td>mash</td>
      <td>Mash</td>
      <td>Fast genome and metagenome distance estimation using MinHash.</td>
      <td></td>
      <td>4</td>
      <td>2</td>
      <td>4</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>4</td>
      <td>2</td>
      <td>0</td>
      <td>49</td>
      <td>57</td>
      <td>10768</td>
      <td>10867</td>
      <td>18</td>
      <td>18</td>
      <td>922</td>
      <td>922</td>
      <td>18</td>
      <td>18</td>
      <td>3187</td>
      <td>3187</td>
      <td>3</td>
      <td>3</td>
      <td>29</td>
      <td>29</td>
      <td>15005</td>
      <td>14906</td>
      <td>96</td>
      <td>88</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>maxbin2</td>
      <td>maxbin2</td>
      <td>clusters metagenomic contigs into bins</td>
      <td>2019-10-24</td>
      <td>https://downloads.jbei.org/data/microbial_communities/MaxBin/MaxBin.html</td>
      <td></td>
      <td>maxbin2</td>
      <td>2.2.7</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly</td>
      <td>Metagenomics, Sequence assembly, Microbiology</td>
      <td>Metagenomics, Sequence assembly, Microbiology</td>
      <td>mbernt</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/maxbin2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/maxbin2</td>
      <td>maxbin</td>
      <td>MaxBin</td>
      <td>Software for binning assembled metagenomic sequences based on an Expectation-Maximization algorithm.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>408</td>
      <td>433</td>
      <td>5244</td>
      <td>5537</td>
      <td>634</td>
      <td>675</td>
      <td>2694</td>
      <td>2993</td>
      <td>106</td>
      <td>108</td>
      <td>1135</td>
      <td>1198</td>
      <td>6</td>
      <td>6</td>
      <td>648</td>
      <td>648</td>
      <td>10376</td>
      <td>9721</td>
      <td>1222</td>
      <td>1154</td>
      <td>https://usegalaxy.eu/published/workflow?id=00f15824c7a9b196, https://usegalaxy.eu/published/workflow?id=02631612a43a4d9b, https://usegalaxy.eu/published/workflow?id=0563f58718be932d, https://usegalaxy.eu/published/workflow?id=1e5f1d9a51032e44, https://usegalaxy.eu/published/workflow?id=6158c3b5fc12044e, https://usegalaxy.eu/published/workflow?id=667bac8d7453e5f1, https://usegalaxy.eu/published/workflow?id=7ecd92cb382d8168, https://usegalaxy.eu/published/workflow?id=8deb08d4f228e198, https://usegalaxy.eu/published/workflow?id=97312d273b6e8bd9, https://usegalaxy.eu/published/workflow?id=9a97c6eb4fe015e7, https://usegalaxy.eu/published/workflow?id=ce5c9287a67e6085, https://usegalaxy.eu/published/workflow?id=d57d41e306241396, https://usegalaxy.org/published/workflow?id=33d90e718ce500ef, https://workflowhub.eu/workflows/1352?version=4, https://workflowhub.eu/workflows/2028?version=1</td>
      <td>microbiome/metagenomics-binning</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>maxquant</td>
      <td>maxquant, maxquant_mqpar</td>
      <td>wrapper for MaxQuant</td>
      <td>2015-01-24</td>
      <td>https://www.maxquant.org/</td>
      <td>2.0.3.0</td>
      <td>maxquant</td>
      <td>2.0.3.0</td>
      <td>Up-to-date</td>
      <td>Proteomics</td>
      <td>Imputation, Visualisation, Protein quantification, Statistical calculation, Standardisation and normalisation, Heat map generation, Clustering, Principal component plotting</td>
      <td>Imputation, Visualisation, Protein quantification, Standardisation and normalisation, Heat map generation, Clustering</td>
      <td>Proteomics experiment, Proteomics, Statistics and probability</td>
      <td>Proteomics experiment, Proteomics, Statistics and probability</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/maxquant</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/maxquant</td>
      <td>maxquant</td>
      <td>MaxQuant</td>
      <td>Quantitative proteomics software package designed for analyzing large mass-spectrometric data sets. It is specifically aimed at high-resolution MS data.</td>
      <td></td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>1628</td>
      <td>1682</td>
      <td>7932</td>
      <td>8757</td>
      <td>775</td>
      <td>775</td>
      <td>3791</td>
      <td>3791</td>
      <td>407</td>
      <td>416</td>
      <td>3720</td>
      <td>3831</td>
      <td>3</td>
      <td>3</td>
      <td>33</td>
      <td>33</td>
      <td>16412</td>
      <td>15476</td>
      <td>2876</td>
      <td>2813</td>
      <td>https://usegalaxy.eu/published/workflow?id=0135ee4b3fa0cbce, https://usegalaxy.eu/published/workflow?id=41e9882300fa2386, https://usegalaxy.eu/published/workflow?id=607ca3a9d45bd080, https://usegalaxy.eu/published/workflow?id=8af41219411062ad, https://usegalaxy.eu/published/workflow?id=912b81c444e35045, https://usegalaxy.eu/published/workflow?id=96d198ed953449fd, https://usegalaxy.eu/published/workflow?id=9e0d784860d6cc91, https://usegalaxy.eu/published/workflow?id=a49275eb8b2b2ecb, https://usegalaxy.eu/published/workflow?id=c4119ea7ffd29094, https://usegalaxy.eu/published/workflow?id=d789165f4e0c50eb, https://usegalaxy.eu/published/workflow?id=eb54512829d8509c, https://usegalaxy.org.au/published/workflow?id=1727d16fa4dec19c, https://usegalaxy.org.au/published/workflow?id=43a951ec514498f6, https://usegalaxy.org.au/published/workflow?id=781457327e644c7e, https://usegalaxy.org.au/published/workflow?id=b0a79310e0dc3fb4, https://usegalaxy.org.au/published/workflow?id=ba217a34d7e1b5ee, https://usegalaxy.org/published/workflow?id=2404d5e5b1817825, https://usegalaxy.org/published/workflow?id=e97da2da9e0290a7, https://workflowhub.eu/workflows/1177?version=1, https://workflowhub.eu/workflows/1225?version=1, https://workflowhub.eu/workflows/1396?version=1, https://workflowhub.eu/workflows/1401?version=1, https://workflowhub.eu/workflows/1420?version=1, https://workflowhub.eu/workflows/1421?version=1, https://workflowhub.eu/workflows/1435?version=1, https://workflowhub.eu/workflows/1448?version=1, https://workflowhub.eu/workflows/1468?version=1</td>
      <td>microbiome/clinical-mp-2-discovery, microbiome/clinical-mp-4-quantitation, proteomics/DIA_lib_OSW, proteomics/clinical-mp-2-discovery, proteomics/clinical-mp-4-quantitation, proteomics/maxquant-label-free, proteomics/maxquant-msstats-dda-lfq, proteomics/maxquant-msstats-tmt</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>mcl</td>
      <td>mcl</td>
      <td>The Markov Cluster Algorithm, a cluster algorithm for graphs</td>
      <td>2022-05-12</td>
      <td>https://micans.org/mcl/man/mcl.html</td>
      <td>22.282</td>
      <td>mcl</td>
      <td>22.282</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Clustering, Network analysis, Gene regulatory network analysis</td>
      <td>Clustering, Gene regulatory network analysis</td>
      <td>Molecular interactions, pathways and networks</td>
      <td>Molecular interactions, pathways and networks</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mcl</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mcl</td>
      <td>mcl</td>
      <td>MCL</td>
      <td>MCL is a clustering algorithm widely used in bioinformatics and gaining traction in other fields.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>18</td>
      <td>18</td>
      <td>153</td>
      <td>153</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>153</td>
      <td>153</td>
      <td>18</td>
      <td>18</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>medaka</td>
      <td>medaka_consensus, medaka_consensus_pipeline, medaka_variant</td>
      <td>Sequence correction provided by ONT Research</td>
      <td>2020-02-23</td>
      <td>https://github.com/nanoporetech/medaka</td>
      <td>2.1.1</td>
      <td>medaka</td>
      <td>2.1.1</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Base-calling, Variant calling, Sequence assembly</td>
      <td>Base-calling, Variant calling, Sequence assembly</td>
      <td>Sequence assembly, Machine learning</td>
      <td>Sequence assembly, Machine learning</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/medaka</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/medaka</td>
      <td>medaka</td>
      <td>Medaka</td>
      <td>medaka is a tool to create consensus sequences and variant calls from nanopore sequencing data. This task is performed using neural networks applied a pileup of individual sequencing reads against a draft assembly.</td>
      <td></td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>1728</td>
      <td>1734</td>
      <td>155538</td>
      <td>156220</td>
      <td>735</td>
      <td>735</td>
      <td>76698</td>
      <td>76698</td>
      <td>421</td>
      <td>421</td>
      <td>11622</td>
      <td>11622</td>
      <td>17</td>
      <td>17</td>
      <td>2659</td>
      <td>2659</td>
      <td>247199</td>
      <td>246517</td>
      <td>2907</td>
      <td>2901</td>
      <td>https://usegalaxy.eu/published/workflow?id=0f6a7bd5362e4a2b, https://usegalaxy.eu/published/workflow?id=152223c8aed27357, https://usegalaxy.eu/published/workflow?id=2b888e200dfd91bc, https://usegalaxy.eu/published/workflow?id=2d9d4445a1871230, https://usegalaxy.eu/published/workflow?id=30dc526dcfa500df, https://usegalaxy.eu/published/workflow?id=34a07bf6407510ef, https://usegalaxy.eu/published/workflow?id=429ae28a178cd07c, https://usegalaxy.eu/published/workflow?id=476db04368d5e90c, https://usegalaxy.eu/published/workflow?id=4d97068a725c975c, https://usegalaxy.eu/published/workflow?id=529f30e894beacc2, https://usegalaxy.eu/published/workflow?id=585c21b7b1d864fc, https://usegalaxy.eu/published/workflow?id=70d3ebf31940bfb3, https://usegalaxy.eu/published/workflow?id=77e5bbd317750915, https://usegalaxy.eu/published/workflow?id=7bdd9ad918edab0b, https://usegalaxy.eu/published/workflow?id=86cf02cc0868d238, https://usegalaxy.eu/published/workflow?id=b97532c398b821a2, https://usegalaxy.eu/published/workflow?id=bf48b0eb15e8ff75, https://usegalaxy.eu/published/workflow?id=cf8591391a17a914, https://usegalaxy.eu/published/workflow?id=eb5ab8bc82569eee, https://usegalaxy.fr/published/workflow?id=61235264d5d2f666, https://usegalaxy.fr/published/workflow?id=727cfdb7c79287d1, https://usegalaxy.fr/published/workflow?id=e3c04a30258421aa, https://usegalaxy.org.au/published/workflow?id=084bb76cf47d7060, https://usegalaxy.org.au/published/workflow?id=0c6f7bfc826433c4, https://usegalaxy.org.au/published/workflow?id=47ce6a2d30887637, https://usegalaxy.org.au/published/workflow?id=48a60965b6b3ff54, https://usegalaxy.org.au/published/workflow?id=4bc8ac41ecff7daf, https://usegalaxy.org.au/published/workflow?id=59618ed15510f531, https://usegalaxy.org.au/published/workflow?id=705658f68ae73930, https://usegalaxy.org.au/published/workflow?id=888f8377eb6eabf2, https://usegalaxy.org.au/published/workflow?id=932bad88f8dc8d33, https://usegalaxy.org.au/published/workflow?id=ba7a9d4c857d26b7, https://usegalaxy.org.au/published/workflow?id=c997a34b55425155, https://usegalaxy.org.au/published/workflow?id=ef8c22c2525063a2, https://usegalaxy.org/published/workflow?id=2ded909e7e3a9962, https://usegalaxy.org/published/workflow?id=3f252e077e0bcce5, https://usegalaxy.org/published/workflow?id=4357180d40560019, https://usegalaxy.org/published/workflow?id=48dafc35989f13b1, https://usegalaxy.org/published/workflow?id=547f86db467bf620, https://usegalaxy.org/published/workflow?id=88d0b64011c3148c, https://usegalaxy.org/published/workflow?id=953d9cbe05e632a8, https://usegalaxy.org/published/workflow?id=9adf8691809326e5, https://usegalaxy.org/published/workflow?id=c12e6b7896f8e114, https://usegalaxy.org/published/workflow?id=c7b2d2ccc4b0dec5, https://usegalaxy.org/published/workflow?id=cce88bc57b180d09, https://usegalaxy.org/published/workflow?id=d3428dd172a78102, https://usegalaxy.org/published/workflow?id=e12324f5e3b930fd, https://workflowhub.eu/workflows/1062?version=1, https://workflowhub.eu/workflows/111?version=5, https://workflowhub.eu/workflows/1495?version=1, https://workflowhub.eu/workflows/1578?version=1, https://workflowhub.eu/workflows/226?version=1</td>
      <td>assembly/largegenome, microbiome/pathogen-detection-from-nanopore-foodborne-data</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>megahit</td>
      <td>megahit</td>
      <td>An ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph.</td>
      <td>2017-09-22</td>
      <td>https://github.com/voutcn/megahit</td>
      <td>1.2.9</td>
      <td>megahit</td>
      <td>1.2.9</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis, Assembly, Metagenomics</td>
      <td>Genome assembly</td>
      <td>Genome assembly</td>
      <td>Metagenomics, Sequencing, Ecology, Sequence assembly</td>
      <td>Metagenomics, Sequencing, Sequence assembly</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/megahit</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/megahit</td>
      <td>megahit</td>
      <td>MEGAHIT</td>
      <td>Single node assembler for large and complex metagenomics NGS reads, such as soil. It makes use of succinct de Bruijn graph to achieve low memory usage, whereas its goal is not to make memory usage as low as possible.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2052</td>
      <td>2197</td>
      <td>22340</td>
      <td>24371</td>
      <td>2823</td>
      <td>2929</td>
      <td>23973</td>
      <td>24874</td>
      <td>615</td>
      <td>632</td>
      <td>9338</td>
      <td>9643</td>
      <td>54</td>
      <td>54</td>
      <td>1387</td>
      <td>1387</td>
      <td>60275</td>
      <td>57038</td>
      <td>5812</td>
      <td>5544</td>
      <td>https://usegalaxy.eu/published/workflow?id=0563f58718be932d, https://usegalaxy.eu/published/workflow?id=1524b189069b42a5, https://usegalaxy.eu/published/workflow?id=1e5f1d9a51032e44, https://usegalaxy.eu/published/workflow?id=2ee0eb74e839b476, https://usegalaxy.eu/published/workflow?id=401746888e8b3aa8, https://usegalaxy.eu/published/workflow?id=534f5fc25bc887ca, https://usegalaxy.eu/published/workflow?id=62c47e3c9d4f3aa3, https://usegalaxy.eu/published/workflow?id=7d84b0419296857d, https://usegalaxy.eu/published/workflow?id=7ecd92cb382d8168, https://usegalaxy.eu/published/workflow?id=97312d273b6e8bd9, https://usegalaxy.eu/published/workflow?id=98bfad0c684011e9, https://usegalaxy.eu/published/workflow?id=9b9e1e28f3683f92, https://usegalaxy.eu/published/workflow?id=a8417c63e9e22afb, https://usegalaxy.eu/published/workflow?id=b37800d990cceb83, https://usegalaxy.eu/published/workflow?id=bebcbd6f0ffa55c5, https://usegalaxy.eu/published/workflow?id=ce5c16ac23c4fbb1, https://usegalaxy.eu/published/workflow?id=d4a95588e6e50893, https://usegalaxy.eu/published/workflow?id=f141db5179ea8422, https://usegalaxy.org/published/workflow?id=7251c6e0dad8f927, https://workflowhub.eu/workflows/1352?version=4, https://workflowhub.eu/workflows/1390?version=2, https://workflowhub.eu/workflows/1480?version=1, https://workflowhub.eu/workflows/1631?version=1, https://workflowhub.eu/workflows/1634?version=1, https://workflowhub.eu/workflows/2024?version=2, https://workflowhub.eu/workflows/2036?version=1, https://workflowhub.eu/workflows/2038?version=1</td>
      <td>assembly/metagenomics-assembly, microbiome/metagenomics-assembly</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>megahit_contig2fastg</td>
      <td>megahit_contig2fastg</td>
      <td>A subprogram within the Megahit toolkit for converting contigs to assembly graphs (fastg)</td>
      <td>2018-11-09</td>
      <td>https://github.com/voutcn/megahit/blob/master/tools/toolkit.cpp</td>
      <td>1.1.3</td>
      <td>megahit</td>
      <td>1.2.9</td>
      <td>To update</td>
      <td>Sequence Analysis, Assembly, Metagenomics</td>
      <td>Genome assembly</td>
      <td>Genome assembly</td>
      <td>Metagenomics, Sequencing, Ecology, Sequence assembly</td>
      <td>Metagenomics, Sequencing, Sequence assembly</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/megahit_contig2fastg</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/megahit_contig2fastg</td>
      <td>megahit</td>
      <td>MEGAHIT</td>
      <td>Single node assembler for large and complex metagenomics NGS reads, such as soil. It makes use of succinct de Bruijn graph to achieve low memory usage, whereas its goal is not to make memory usage as low as possible.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>200</td>
      <td>212</td>
      <td>763</td>
      <td>862</td>
      <td>113</td>
      <td>113</td>
      <td>338</td>
      <td>338</td>
      <td>33</td>
      <td>33</td>
      <td>150</td>
      <td>150</td>
      <td>6</td>
      <td>6</td>
      <td>80</td>
      <td>80</td>
      <td>1430</td>
      <td>1331</td>
      <td>364</td>
      <td>352</td>
      <td>https://usegalaxy.eu/published/workflow?id=62c47e3c9d4f3aa3, https://workflowhub.eu/workflows/1390?version=2, https://workflowhub.eu/workflows/1634?version=1</td>
      <td>assembly/metagenomics-assembly, microbiome/metagenomics-assembly</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>megan</td>
      <td>megan_blast2lca, megan_blast2rma, megan_daa2info, megan_daa2rma, megan_daa_meganizer, megan_read_extractor, megan_sam2rma</td>
      <td>MEGAN Community Edition - Interactive exploration and analysis of large-scale microbiome sequencing data.  MEGAN is a tool for studying the taxonomic content of a set of DNA reads, typically collected in a metagenomics project.In a preprocessing step, a sequence alignment of all reads against a suitable database of reference DNA or proteinsequences must be performed to produce an input file for the program. MEGAN is suitable for DNA reads (metagenomedata), RNA reads (metatranscriptome data), peptide sequences (metaproteomics data) and, using a suitable synonymsfile that maps SILVA ids to taxon ids, for 16S rRNA data (amplicon sequencing).</td>
      <td>2021-11-24</td>
      <td>https://github.com/husonlab/megan-ce</td>
      <td>6.21.7</td>
      <td>megan</td>
      <td>6.25.10</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Sequence analysis, Taxonomic classification</td>
      <td>Sequence analysis, Taxonomic classification</td>
      <td>Sequence analysis</td>
      <td>Sequence analysis</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/megan</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/megan</td>
      <td>megan</td>
      <td>MEGAN</td>
      <td>Metagenome Analysis Software - MEGAN (MEtaGenome ANalyzer) is a new computer program that allows laptop analysis of large metagenomic datasets. In a preprocessing step, the set of DNA reads (or contigs) is compared against databases of known sequences using BLAST or another comparison tool. MEGAN can then be used to compute and interactively explore the taxonomical content of the dataset, employing the NCBI taxonomy to summarize and order the results.</td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>250</td>
      <td>250</td>
      <td>5164</td>
      <td>5164</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>72</td>
      <td>72</td>
      <td>3</td>
      <td>3</td>
      <td>4</td>
      <td>4</td>
      <td>5240</td>
      <td>5240</td>
      <td>258</td>
      <td>258</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>meningotype</td>
      <td>meningotype</td>
      <td>Assign sequence type to N. meningitidis genome assemblies</td>
      <td>2023-06-06</td>
      <td>https://github.com/MDU-PHL/meningotype</td>
      <td>0.8.5</td>
      <td>meningotype</td>
      <td>0.8.5</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Genotyping, Multilocus sequence typing</td>
      <td>Multilocus sequence typing</td>
      <td>Microbiology, Genotype and phenotype</td>
      <td>Microbiology, Genotype and phenotype</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/meningotype</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/meningotype</td>
      <td>meningotype</td>
      <td>meningotype</td>
      <td>In silico typing of Neisseria meningitidis contigs.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>merqury</td>
      <td>merqury, merquryplot</td>
      <td>Merqury is a tool for evaluating genomes assemblies based of k-mer operations.</td>
      <td>2021-04-18</td>
      <td>https://github.com/marbl/merqury</td>
      <td>1.3</td>
      <td>merqury</td>
      <td>1.3</td>
      <td>Up-to-date</td>
      <td>Assembly</td>
      <td>Genome assembly, k-mer counting, Scaffolding, Phasing, De-novo assembly</td>
      <td>Genome assembly, k-mer counting, Scaffolding, Phasing, De-novo assembly</td>
      <td>Sequence assembly, Whole genome sequencing, Plant biology</td>
      <td>Sequence assembly, Whole genome sequencing, Plant biology</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/merqury</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/merqury</td>
      <td>merqury</td>
      <td>Merqury</td>
      <td>Reference-free quality, completeness, and phasing assessment for genome assemblies.Evaluate genome assemblies with k-mers and more.Often, genome assembly projects have illumina whole genome sequencing reads available for the assembled individual.Merqury provides a set of tools for this purpose.</td>
      <td></td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>661</td>
      <td>661</td>
      <td>4657</td>
      <td>4657</td>
      <td>264</td>
      <td>264</td>
      <td>1645</td>
      <td>1645</td>
      <td>243</td>
      <td>243</td>
      <td>1945</td>
      <td>1945</td>
      <td>77</td>
      <td>77</td>
      <td>440</td>
      <td>440</td>
      <td>8687</td>
      <td>8687</td>
      <td>1245</td>
      <td>1245</td>
      <td>https://usegalaxy.eu/published/workflow?id=002a51bdef00e14d, https://usegalaxy.eu/published/workflow?id=01fe4c590e608103, https://usegalaxy.eu/published/workflow?id=023a0ee62dd1f593, https://usegalaxy.eu/published/workflow?id=032a7281a86eba0d, https://usegalaxy.eu/published/workflow?id=03bb4c20a1f4b4de, https://usegalaxy.eu/published/workflow?id=03be9d62e8e4a129, https://usegalaxy.eu/published/workflow?id=0444c98a3ca2a3ca, https://usegalaxy.eu/published/workflow?id=04844dddcc4e8aa7, https://usegalaxy.eu/published/workflow?id=085c3a4c89d137b5, https://usegalaxy.eu/published/workflow?id=089e5e83b1de0511, https://usegalaxy.eu/published/workflow?id=0a3ebee985c26060, https://usegalaxy.eu/published/workflow?id=0f207a15b6f7fa10, https://usegalaxy.eu/published/workflow?id=1151052f4393852a, https://usegalaxy.eu/published/workflow?id=122da0054fd40adf, https://usegalaxy.eu/published/workflow?id=12b0bd336c76a874, https://usegalaxy.eu/published/workflow?id=1483534f49b826bf, https://usegalaxy.eu/published/workflow?id=17b0a6d3379a4392, https://usegalaxy.eu/published/workflow?id=194a05d42687b524, https://usegalaxy.eu/published/workflow?id=19bb57f8cfa48bb9, https://usegalaxy.eu/published/workflow?id=1ac0a15758277730, https://usegalaxy.eu/published/workflow?id=1af73dc855d0a5ed, https://usegalaxy.eu/published/workflow?id=1bb084f1bc102734, https://usegalaxy.eu/published/workflow?id=1cba6eb4aff8dc61, https://usegalaxy.eu/published/workflow?id=1d1ae39386b5c318, https://usegalaxy.eu/published/workflow?id=1da7ac8eaf891f39, https://usegalaxy.eu/published/workflow?id=1e02a7292e9974b1, https://usegalaxy.eu/published/workflow?id=1e7f4ba6261f979c, https://usegalaxy.eu/published/workflow?id=1f2a1438e41188f2, https://usegalaxy.eu/published/workflow?id=21a3f72851f29925, https://usegalaxy.eu/published/workflow?id=2294ef9515bcf972, https://usegalaxy.eu/published/workflow?id=2671a5f85b2cccfe, https://usegalaxy.eu/published/workflow?id=270b6f82efab28b2, https://usegalaxy.eu/published/workflow?id=29c97d43eeb8ed63, https://usegalaxy.eu/published/workflow?id=2e418c5810012e5a, https://usegalaxy.eu/published/workflow?id=3069c17b52decab4, https://usegalaxy.eu/published/workflow?id=3192f92ac4d389b8, https://usegalaxy.eu/published/workflow?id=342a5df69951e3d7, https://usegalaxy.eu/published/workflow?id=3480204a99bf4a35, https://usegalaxy.eu/published/workflow?id=379642036b6f5a66, https://usegalaxy.eu/published/workflow?id=399048afb4acef3d, https://usegalaxy.eu/published/workflow?id=3a3e2d50fe7a9ee3, https://usegalaxy.eu/published/workflow?id=3e255a5b7026d26e, https://usegalaxy.eu/published/workflow?id=3eb6663a66f8e69a, https://usegalaxy.eu/published/workflow?id=3f5414bb38b47fcf, https://usegalaxy.eu/published/workflow?id=416141b245073406, https://usegalaxy.eu/published/workflow?id=45ae834d842302c3, https://usegalaxy.eu/published/workflow?id=471eb8b4c54f897d, https://usegalaxy.eu/published/workflow?id=475ea16f943cc013, https://usegalaxy.eu/published/workflow?id=4856e3522b2ee059, https://usegalaxy.eu/published/workflow?id=497b82d3b92dc076, https://usegalaxy.eu/published/workflow?id=4c3a9b574f37eebb, https://usegalaxy.eu/published/workflow?id=4d41ee83fc0040d4, https://usegalaxy.eu/published/workflow?id=5115299e77df8104, https://usegalaxy.eu/published/workflow?id=54590ccd11ffbf65, https://usegalaxy.eu/published/workflow?id=56523b41ff7eec76, https://usegalaxy.eu/published/workflow?id=57cc33ed64d71982, https://usegalaxy.eu/published/workflow?id=5ad406442ec21345, https://usegalaxy.eu/published/workflow?id=5bf6718e7d6158d2, https://usegalaxy.eu/published/workflow?id=5c51f26ef8b20633, https://usegalaxy.eu/published/workflow?id=5e1906290f5b4d8a, https://usegalaxy.eu/published/workflow?id=6573f3b1606b7f09, https://usegalaxy.eu/published/workflow?id=6865f33cffca4922, https://usegalaxy.eu/published/workflow?id=69be0fb8276753ec, https://usegalaxy.eu/published/workflow?id=69cdd404592535ec, https://usegalaxy.eu/published/workflow?id=6c14b08a7ef7e79a, https://usegalaxy.eu/published/workflow?id=700fa3ab11a8f3e2, https://usegalaxy.eu/published/workflow?id=7137e45ca8dc6e4c, https://usegalaxy.eu/published/workflow?id=71a6d128a09210f0, https://usegalaxy.eu/published/workflow?id=7337d15d121596da, https://usegalaxy.eu/published/workflow?id=79b5afb125e3c3da, https://usegalaxy.eu/published/workflow?id=847403b72dd346ff, https://usegalaxy.eu/published/workflow?id=8be366ad6a42408b, https://usegalaxy.eu/published/workflow?id=8f105de48f7f1d9d, https://usegalaxy.eu/published/workflow?id=915ffef5b14c871e, https://usegalaxy.eu/published/workflow?id=9207e74a5bf7d8e1, https://usegalaxy.eu/published/workflow?id=92760fc19ef120ce, https://usegalaxy.eu/published/workflow?id=92dafb3d1c06e074, https://usegalaxy.eu/published/workflow?id=95f420d8856bf725, https://usegalaxy.eu/published/workflow?id=9667bf36217083b7, https://usegalaxy.eu/published/workflow?id=9c0b67496dfe52b2, https://usegalaxy.eu/published/workflow?id=9d209bcdae1a475e, https://usegalaxy.eu/published/workflow?id=9f2d18f0d0896e8b, https://usegalaxy.eu/published/workflow?id=a3f082ff30357040, https://usegalaxy.eu/published/workflow?id=a6dd62ab4b9a0725, https://usegalaxy.eu/published/workflow?id=a9c1dca5da7170dd, https://usegalaxy.eu/published/workflow?id=aa6dc57f5c67fb4b, https://usegalaxy.eu/published/workflow?id=ac6eb99f70d35999, https://usegalaxy.eu/published/workflow?id=ade6f36f6a035a73, https://usegalaxy.eu/published/workflow?id=ae74f60cd20d56bc, https://usegalaxy.eu/published/workflow?id=aea88bfca6d1d476, https://usegalaxy.eu/published/workflow?id=aef62f4d0bba003b, https://usegalaxy.eu/published/workflow?id=b211fceaecf41c9e, https://usegalaxy.eu/published/workflow?id=b4948afacccfefdb, https://usegalaxy.eu/published/workflow?id=b5de94e9cde071ab, https://usegalaxy.eu/published/workflow?id=b8baade87e733fce, https://usegalaxy.eu/published/workflow?id=bac84a7033c3423f, https://usegalaxy.eu/published/workflow?id=bc32937d505135ab, https://usegalaxy.eu/published/workflow?id=bc87309a8763d0b4, https://usegalaxy.eu/published/workflow?id=bcf99cc0469db276, https://usegalaxy.eu/published/workflow?id=bef0a7d604f2ebef, https://usegalaxy.eu/published/workflow?id=c4c2e669b86cffbc, https://usegalaxy.eu/published/workflow?id=c508c00e81018dd8, https://usegalaxy.eu/published/workflow?id=ca39822372847670, https://usegalaxy.eu/published/workflow?id=caba6d3d7db648e9, https://usegalaxy.eu/published/workflow?id=d02458f1d7d9c3f4, https://usegalaxy.eu/published/workflow?id=d172f927a489e9f8, https://usegalaxy.eu/published/workflow?id=d17d6029914d666b, https://usegalaxy.eu/published/workflow?id=d1bd5d8597c1d31d, https://usegalaxy.eu/published/workflow?id=d261bbd05889ce75, https://usegalaxy.eu/published/workflow?id=d67299c34e63f3d9, https://usegalaxy.eu/published/workflow?id=d924fafd8e042952, https://usegalaxy.eu/published/workflow?id=dc27453c438577f6, https://usegalaxy.eu/published/workflow?id=de9cf573314f416c, https://usegalaxy.eu/published/workflow?id=dfe3b555856c7b44, https://usegalaxy.eu/published/workflow?id=e138640778b7414c, https://usegalaxy.eu/published/workflow?id=e1947b059ef131d2, https://usegalaxy.eu/published/workflow?id=e2f936f1aa43a629, https://usegalaxy.eu/published/workflow?id=e7aebe58411d97ed, https://usegalaxy.eu/published/workflow?id=e8f9e1d911683b22, https://usegalaxy.eu/published/workflow?id=e935d2c8ffb52588, https://usegalaxy.eu/published/workflow?id=ec79f14ab91cd933, https://usegalaxy.eu/published/workflow?id=f08b29e4cef76cd5, https://usegalaxy.eu/published/workflow?id=f1ca4289ee4f67bb, https://usegalaxy.eu/published/workflow?id=f38745208494d4ab, https://usegalaxy.eu/published/workflow?id=f3e17c8dd80def27, https://usegalaxy.eu/published/workflow?id=f578a097b4339025, https://usegalaxy.eu/published/workflow?id=f6a625946dea60fa, https://usegalaxy.eu/published/workflow?id=f7cdaf862e6599c0, https://usegalaxy.eu/published/workflow?id=f882d14fc5dabbb9, https://usegalaxy.eu/published/workflow?id=f91245e580319e65, https://usegalaxy.eu/published/workflow?id=fb015e398373ea8a, https://usegalaxy.eu/published/workflow?id=fba1671d835b6dd3, https://usegalaxy.eu/published/workflow?id=fc25c1f8c6667c87, https://usegalaxy.eu/published/workflow?id=fd649ac4d4872c81, https://usegalaxy.org.au/published/workflow?id=000c2f1884bc4687, https://usegalaxy.org.au/published/workflow?id=02eebe3d909206ab, https://usegalaxy.org.au/published/workflow?id=06564b19b0655c19, https://usegalaxy.org.au/published/workflow?id=0810be3d7d2b0322, https://usegalaxy.org.au/published/workflow?id=08fac3b4c128e597, https://usegalaxy.org.au/published/workflow?id=094655df3ccd0b00, https://usegalaxy.org.au/published/workflow?id=0af0ea50f1237952, https://usegalaxy.org.au/published/workflow?id=0c11e1ba5e8fa21b, https://usegalaxy.org.au/published/workflow?id=0cc8c8875df6bccb, https://usegalaxy.org.au/published/workflow?id=0df25e2dbd5e8ed5, https://usegalaxy.org.au/published/workflow?id=10355b2680374a03, https://usegalaxy.org.au/published/workflow?id=1905ec04d5841f09, https://usegalaxy.org.au/published/workflow?id=1ce2a725df3adb74, https://usegalaxy.org.au/published/workflow?id=1dcc1b25c9091fbf, https://usegalaxy.org.au/published/workflow?id=1eb3e1452c672480, https://usegalaxy.org.au/published/workflow?id=1f49c65625b9d2de, https://usegalaxy.org.au/published/workflow?id=2356584eac7fa234, https://usegalaxy.org.au/published/workflow?id=2470cb61ba51288c, https://usegalaxy.org.au/published/workflow?id=269b42605a2a87d2, https://usegalaxy.org.au/published/workflow?id=2d45d7138d8715b4, https://usegalaxy.org.au/published/workflow?id=319e9d693fac8dce, https://usegalaxy.org.au/published/workflow?id=328adfa7d90b1d54, https://usegalaxy.org.au/published/workflow?id=3604307ec2835628, https://usegalaxy.org.au/published/workflow?id=36f54eda8568355d, https://usegalaxy.org.au/published/workflow?id=371c9892d261cee2, https://usegalaxy.org.au/published/workflow?id=3c30c822058187be, https://usegalaxy.org.au/published/workflow?id=3df7f45b7b388f8c, https://usegalaxy.org.au/published/workflow?id=40666ceb68d101ac, https://usegalaxy.org.au/published/workflow?id=40740c32b9a550c9, https://usegalaxy.org.au/published/workflow?id=41282d51f9112552, https://usegalaxy.org.au/published/workflow?id=41795303a88d9e94, https://usegalaxy.org.au/published/workflow?id=4267bd1e4cfd1901, https://usegalaxy.org.au/published/workflow?id=43343fa1896f8278, https://usegalaxy.org.au/published/workflow?id=434127690544d0e9, https://usegalaxy.org.au/published/workflow?id=447d8daefdefafee, https://usegalaxy.org.au/published/workflow?id=478c79365fc9677f, https://usegalaxy.org.au/published/workflow?id=47cd0a90609fd557, https://usegalaxy.org.au/published/workflow?id=4a404aa3e9f29e5a, https://usegalaxy.org.au/published/workflow?id=4b5f80656fc94b88, https://usegalaxy.org.au/published/workflow?id=4cb69c84b73ea9dd, https://usegalaxy.org.au/published/workflow?id=4e7930c7286c6e9e, https://usegalaxy.org.au/published/workflow?id=4ebf1297793870e7, https://usegalaxy.org.au/published/workflow?id=51bad198f1fd1215, https://usegalaxy.org.au/published/workflow?id=543fb2d9431a7e13, https://usegalaxy.org.au/published/workflow?id=5550585ffc3d8d0e, https://usegalaxy.org.au/published/workflow?id=5b0c0e34394afed1, https://usegalaxy.org.au/published/workflow?id=5cd804644a34c8fd, https://usegalaxy.org.au/published/workflow?id=5f4f874e49515555, https://usegalaxy.org.au/published/workflow?id=60c0dec52cdca708, https://usegalaxy.org.au/published/workflow?id=63789b6f82172679, https://usegalaxy.org.au/published/workflow?id=68b2961117e62890, https://usegalaxy.org.au/published/workflow?id=6c27019d03dd49d8, https://usegalaxy.org.au/published/workflow?id=6c9129acb9d98afe, https://usegalaxy.org.au/published/workflow?id=6ec0031406b10635, https://usegalaxy.org.au/published/workflow?id=6f76c43e6d2d7f74, https://usegalaxy.org.au/published/workflow?id=7168c01be9997823, https://usegalaxy.org.au/published/workflow?id=7909da83448d8bf4, https://usegalaxy.org.au/published/workflow?id=79bf7b0f5e9052d7, https://usegalaxy.org.au/published/workflow?id=81a5b96452c71a52, https://usegalaxy.org.au/published/workflow?id=82d0876385b5ac60, https://usegalaxy.org.au/published/workflow?id=8926e592f2e60863, https://usegalaxy.org.au/published/workflow?id=8a6233015db6dd24, https://usegalaxy.org.au/published/workflow?id=8bf8c9929249f37d, https://usegalaxy.org.au/published/workflow?id=8dd86497bc15d981, https://usegalaxy.org.au/published/workflow?id=8e8a5cce9f3f2f67, https://usegalaxy.org.au/published/workflow?id=91ee716be8bfd508, https://usegalaxy.org.au/published/workflow?id=9642aadb83c4f752, https://usegalaxy.org.au/published/workflow?id=970dc0d8bd74707f, https://usegalaxy.org.au/published/workflow?id=9804030358c3f844, https://usegalaxy.org.au/published/workflow?id=98d3def7da131b4c, https://usegalaxy.org.au/published/workflow?id=98f660971010d11d, https://usegalaxy.org.au/published/workflow?id=9e0bae0505ee7c07, https://usegalaxy.org.au/published/workflow?id=a02c102e179596cb, https://usegalaxy.org.au/published/workflow?id=a1015b83baa0eb6d, https://usegalaxy.org.au/published/workflow?id=a209f43d8a3cc853, https://usegalaxy.org.au/published/workflow?id=a345bf84eb9bf6b2, https://usegalaxy.org.au/published/workflow?id=a40dadc943a0db1e, https://usegalaxy.org.au/published/workflow?id=a525966251895f8a, https://usegalaxy.org.au/published/workflow?id=a740543c4e9808f1, https://usegalaxy.org.au/published/workflow?id=ac8bdc5a0b2d70cd, https://usegalaxy.org.au/published/workflow?id=ad10cd704268dcf0, https://usegalaxy.org.au/published/workflow?id=ad556db35858e7ec, https://usegalaxy.org.au/published/workflow?id=afab89bc468da8b3, https://usegalaxy.org.au/published/workflow?id=b19fa49405117317, https://usegalaxy.org.au/published/workflow?id=b1f45d2f8d8343e9, https://usegalaxy.org.au/published/workflow?id=b310af289ada88bc, https://usegalaxy.org.au/published/workflow?id=b5131e3a0ef2676e, https://usegalaxy.org.au/published/workflow?id=b9faf6a3f48fc11b, https://usegalaxy.org.au/published/workflow?id=be96bb47b6d6435a, https://usegalaxy.org.au/published/workflow?id=c43744d7dd9c0c0b, https://usegalaxy.org.au/published/workflow?id=c556c2270200801d, https://usegalaxy.org.au/published/workflow?id=c6a71573a299eb3a, https://usegalaxy.org.au/published/workflow?id=c7e1f666a3366823, https://usegalaxy.org.au/published/workflow?id=c7f2d6490128dc3c, https://usegalaxy.org.au/published/workflow?id=c853aa707fcddf4e, https://usegalaxy.org.au/published/workflow?id=cbf022e349d726d9, https://usegalaxy.org.au/published/workflow?id=d06665e4bc7fff6d, https://usegalaxy.org.au/published/workflow?id=d298edda13b382e7, https://usegalaxy.org.au/published/workflow?id=d42d7b1aa47e870f, https://usegalaxy.org.au/published/workflow?id=d678e57c7f1baf5d, https://usegalaxy.org.au/published/workflow?id=d78879cfaa5948b3, https://usegalaxy.org.au/published/workflow?id=d7f3fc810261e419, https://usegalaxy.org.au/published/workflow?id=d83d5ecf31c20490, https://usegalaxy.org.au/published/workflow?id=d957bfeb3cd03db0, https://usegalaxy.org.au/published/workflow?id=d9c181fd954b629a, https://usegalaxy.org.au/published/workflow?id=df11f09771e73e04, https://usegalaxy.org.au/published/workflow?id=df4e186069552649, https://usegalaxy.org.au/published/workflow?id=e031af90daa2cadf, https://usegalaxy.org.au/published/workflow?id=e28ca83f14f4ede5, https://usegalaxy.org.au/published/workflow?id=e5f1ce106ea7af12, https://usegalaxy.org.au/published/workflow?id=ea687c5613fcce27, https://usegalaxy.org.au/published/workflow?id=ecfb8a08e5a40e15, https://usegalaxy.org.au/published/workflow?id=f06370394b061e08, https://usegalaxy.org.au/published/workflow?id=f7a866159f1eaecf, https://usegalaxy.org.au/published/workflow?id=f895914e322c8b77, https://usegalaxy.org.au/published/workflow?id=fa37223950a91f34, https://usegalaxy.org.au/published/workflow?id=fe1e7fa0bc437ed8, https://usegalaxy.org.au/published/workflow?id=fea0e6c02c2f6274, https://usegalaxy.org.au/published/workflow?id=fff2e49bc90e38ca, https://usegalaxy.org/published/workflow?id=07e873a6d75dfe9f, https://usegalaxy.org/published/workflow?id=0993daadf2db5f51, https://usegalaxy.org/published/workflow?id=10d5e195501603bc, https://usegalaxy.org/published/workflow?id=1245e9435cbc3985, https://usegalaxy.org/published/workflow?id=137d0f05a71d7acb, https://usegalaxy.org/published/workflow?id=1b434bfc3aac0022, https://usegalaxy.org/published/workflow?id=1d24cc949c52e475, https://usegalaxy.org/published/workflow?id=1eec2586d1ff908b, https://usegalaxy.org/published/workflow?id=237ca9f36ca266e7, https://usegalaxy.org/published/workflow?id=242ef2e9dee35702, https://usegalaxy.org/published/workflow?id=25a9f0693ff6d327, https://usegalaxy.org/published/workflow?id=283bc048cc7a80f5, https://usegalaxy.org/published/workflow?id=284150bb1f296378, https://usegalaxy.org/published/workflow?id=2acbb8c5f72b9f1c, https://usegalaxy.org/published/workflow?id=2b41752b8518a87b, https://usegalaxy.org/published/workflow?id=2c5c627c08563398, https://usegalaxy.org/published/workflow?id=2d994cc02ec05ede, https://usegalaxy.org/published/workflow?id=2f1469dab87a3649, https://usegalaxy.org/published/workflow?id=305fb9f84b053713, https://usegalaxy.org/published/workflow?id=31387ed5ebc88cd2, https://usegalaxy.org/published/workflow?id=3179c157fa5a2fdd, https://usegalaxy.org/published/workflow?id=31a6145b0832fa48, https://usegalaxy.org/published/workflow?id=3753dcca4100e976, https://usegalaxy.org/published/workflow?id=3ba73f679558017c, https://usegalaxy.org/published/workflow?id=3db4eb1ecb588d44, https://usegalaxy.org/published/workflow?id=3f7550d5d2bf0b96, https://usegalaxy.org/published/workflow?id=4029a0c8fa23d375, https://usegalaxy.org/published/workflow?id=408c799822cbf6ea, https://usegalaxy.org/published/workflow?id=426358bd988cb39d, https://usegalaxy.org/published/workflow?id=435f0340bf00c940, https://usegalaxy.org/published/workflow?id=4403ca7fe447a5d7, https://usegalaxy.org/published/workflow?id=44781c8b0bd667ed, https://usegalaxy.org/published/workflow?id=45c0b677c4920d03, https://usegalaxy.org/published/workflow?id=484f8fcf21d67376, https://usegalaxy.org/published/workflow?id=498a8c6329ca5728, https://usegalaxy.org/published/workflow?id=49c9e8161a46667f, https://usegalaxy.org/published/workflow?id=4a0ad87aa5aa558a, https://usegalaxy.org/published/workflow?id=4bf38f5b45faa8a1, https://usegalaxy.org/published/workflow?id=4bf73376661baafd, https://usegalaxy.org/published/workflow?id=4d5cc5010537f172, https://usegalaxy.org/published/workflow?id=578e634b63923e9b, https://usegalaxy.org/published/workflow?id=57e6345fda3e2682, https://usegalaxy.org/published/workflow?id=597c4d54f8a8ebe7, https://usegalaxy.org/published/workflow?id=5dfec2a3ae5fa765, https://usegalaxy.org/published/workflow?id=5edabfc62f58d8cf, https://usegalaxy.org/published/workflow?id=653d553ac2fa51c8, https://usegalaxy.org/published/workflow?id=65ccf48b7872e36c, https://usegalaxy.org/published/workflow?id=67252039bce060b2, https://usegalaxy.org/published/workflow?id=683e725f672b7d9b, https://usegalaxy.org/published/workflow?id=6a27c0d224683f73, https://usegalaxy.org/published/workflow?id=6b511b47176893e3, https://usegalaxy.org/published/workflow?id=6fc7497c86ebd8ee, https://usegalaxy.org/published/workflow?id=7072731b51e16923, https://usegalaxy.org/published/workflow?id=73c20d784b7ef81a, https://usegalaxy.org/published/workflow?id=771345c66d5b9700, https://usegalaxy.org/published/workflow?id=7adf904dea83e699, https://usegalaxy.org/published/workflow?id=7dba74c915f63dc1, https://usegalaxy.org/published/workflow?id=7f6900852a81d121, https://usegalaxy.org/published/workflow?id=81f1f0fd6d9235fd, https://usegalaxy.org/published/workflow?id=84161f4f8bf77474, https://usegalaxy.org/published/workflow?id=864da6858d34df08, https://usegalaxy.org/published/workflow?id=86c67855acbf6124, https://usegalaxy.org/published/workflow?id=8ccb42012dcb5db2, https://usegalaxy.org/published/workflow?id=8cfbaa14cff5a633, https://usegalaxy.org/published/workflow?id=8d605fbd33003d26, https://usegalaxy.org/published/workflow?id=8f7c9461910b08f1, https://usegalaxy.org/published/workflow?id=90f5b512997c70f8, https://usegalaxy.org/published/workflow?id=932b3bc10186f73e, https://usegalaxy.org/published/workflow?id=936ed7447c8e0a2d, https://usegalaxy.org/published/workflow?id=93c1fef4472aa7a8, https://usegalaxy.org/published/workflow?id=950c7245a6c3f6e9, https://usegalaxy.org/published/workflow?id=961dac8100c6c70c, https://usegalaxy.org/published/workflow?id=96bd0007dceec123, https://usegalaxy.org/published/workflow?id=9938a81fb6911412, https://usegalaxy.org/published/workflow?id=9ab68c65337b893b, https://usegalaxy.org/published/workflow?id=9ddfdc9d6cacea88, https://usegalaxy.org/published/workflow?id=a8e0f77cd6ef49f9, https://usegalaxy.org/published/workflow?id=ab0be4897d5355ac, https://usegalaxy.org/published/workflow?id=acada5044d64a30a, https://usegalaxy.org/published/workflow?id=b11c6268704dcc1b, https://usegalaxy.org/published/workflow?id=b12f99e2233fcca6, https://usegalaxy.org/published/workflow?id=b5644a5dcf237ef9, https://usegalaxy.org/published/workflow?id=b661fb850c4b4b01, https://usegalaxy.org/published/workflow?id=b6d535f210efa18e, https://usegalaxy.org/published/workflow?id=bb1cb6abfe84a646, https://usegalaxy.org/published/workflow?id=bbc894a79eb0e769, https://usegalaxy.org/published/workflow?id=bd6bfd338ffb9637, https://usegalaxy.org/published/workflow?id=bfd2e7e63d3e6856, https://usegalaxy.org/published/workflow?id=bfef75672b9c3ea4, https://usegalaxy.org/published/workflow?id=c09b8a0c96123cbb, https://usegalaxy.org/published/workflow?id=c0a9a2d53b8660de, https://usegalaxy.org/published/workflow?id=c694856c01ce7cc5, https://usegalaxy.org/published/workflow?id=c983ea8a32662cd6, https://usegalaxy.org/published/workflow?id=cb4ccd3068819418, https://usegalaxy.org/published/workflow?id=cc50da71cdf6f244, https://usegalaxy.org/published/workflow?id=ce04658f05029be2, https://usegalaxy.org/published/workflow?id=cfd5dc3c7bc1e832, https://usegalaxy.org/published/workflow?id=d08135fa13d1745e, https://usegalaxy.org/published/workflow?id=d2e8a6b39fe88f4c, https://usegalaxy.org/published/workflow?id=d36765ac28c11e1a, https://usegalaxy.org/published/workflow?id=d3f35215e6f45d79, https://usegalaxy.org/published/workflow?id=d4b2b85db47c405e, https://usegalaxy.org/published/workflow?id=d4f0a7ccb09f1e55, https://usegalaxy.org/published/workflow?id=daaded719dcfd53b, https://usegalaxy.org/published/workflow?id=dae5d5e5df5c4dd7, https://usegalaxy.org/published/workflow?id=e12e642b88a0fe08, https://usegalaxy.org/published/workflow?id=e4dd39237df58884, https://usegalaxy.org/published/workflow?id=eaf96fa3c39c210b, https://usegalaxy.org/published/workflow?id=f1a54b10d4afce8f, https://usegalaxy.org/published/workflow?id=f60027545d40b82d, https://usegalaxy.org/published/workflow?id=f93a4f6b5f3aaaed, https://usegalaxy.org/published/workflow?id=fa26fd51caea53c7, https://usegalaxy.org/published/workflow?id=fc7b14aafa0b421e, https://usegalaxy.org/published/workflow?id=fd155fcba9bed7ff, https://usegalaxy.org/published/workflow?id=fd40e50c9f1545ce, https://workflowhub.eu/workflows/1103?version=3, https://workflowhub.eu/workflows/1104?version=2, https://workflowhub.eu/workflows/1162?version=2, https://workflowhub.eu/workflows/1163?version=2, https://workflowhub.eu/workflows/1164?version=2, https://workflowhub.eu/workflows/1607?version=1, https://workflowhub.eu/workflows/1641?version=1, https://workflowhub.eu/workflows/1646?version=1, https://workflowhub.eu/workflows/403?version=9, https://workflowhub.eu/workflows/605?version=1, https://workflowhub.eu/workflows/606?version=2, https://workflowhub.eu/workflows/608?version=1, https://workflowhub.eu/workflows/612?version=25, https://workflowhub.eu/workflows/633?version=23, https://workflowhub.eu/workflows/641?version=32, https://workflowhub.eu/workflows/642?version=25, https://workflowhub.eu/workflows/701?version=1, https://workflowhub.eu/workflows/702?version=1, https://workflowhub.eu/workflows/751?version=18, https://workflowhub.eu/workflows/788?version=1, https://workflowhub.eu/workflows/789?version=1, https://workflowhub.eu/workflows/794?version=2</td>
      <td>assembly/ERGA-post-assembly-QC, assembly/assembly-quality-control, assembly/vgp_genome_assembly</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>meryl</td>
      <td>meryl_arithmetic_kmers, meryl_count_kmers, meryl_filter_kmers, meryl_groups_kmers, meryl_histogram_kmers, meryl_print, meryl_trio_mode</td>
      <td>Meryl a k-mer counter.</td>
      <td>2021-04-05</td>
      <td>https://github.com/marbl/meryl</td>
      <td>1.4.1</td>
      <td>merqury</td>
      <td>1.3</td>
      <td>To update</td>
      <td>Assembly</td>
      <td>k-mer counting</td>
      <td>k-mer counting</td>
      <td>Whole genome sequencing, Genomics, Sequence analysis, Sequencing</td>
      <td>Whole genome sequencing, Genomics, Sequence analysis</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/meryl</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/meryl</td>
      <td>meryl</td>
      <td>Meryl</td>
      <td>Meryl is a tool for counting and working with sets of k-mers that was originally developed for use in the Celera Assembler and has since been migrated and maintained as part of Canu.</td>
      <td></td>
      <td>7</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>55</td>
      <td>55</td>
      <td>1850</td>
      <td>1850</td>
      <td>77</td>
      <td>77</td>
      <td>1478</td>
      <td>1478</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3328</td>
      <td>3328</td>
      <td>132</td>
      <td>132</td>
      <td>https://usegalaxy.eu/published/workflow?id=0ccb3838765a8b84, https://usegalaxy.eu/published/workflow?id=10bde4e7f8dbd0c4, https://usegalaxy.eu/published/workflow?id=41c48e4632684f76, https://usegalaxy.eu/published/workflow?id=59f35cb60b3d989d, https://usegalaxy.org/published/workflow?id=4b919e575a16d062, https://usegalaxy.org/published/workflow?id=5503f7835226bc04, https://usegalaxy.org/published/workflow?id=6988c29ae84d2788, https://usegalaxy.org/published/workflow?id=83898f7cbfb7f88e, https://usegalaxy.org/published/workflow?id=9f1aabc6f5d18d13, https://usegalaxy.org/published/workflow?id=af1f4dba9aaa0b52, https://usegalaxy.org/published/workflow?id=b4f9d9aafe87af82, https://usegalaxy.org/published/workflow?id=beefb67de2b0728c, https://workflowhub.eu/workflows/631?version=6, https://workflowhub.eu/workflows/632?version=13</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>metabat2</td>
      <td>metabat2_jgi_summarize_bam_contig_depths, metabat2</td>
      <td>MetaBAT2 (Metagenome Binning based on Abundance and Tetranucleotide frequency) is an automated metagenome binningsoftware that integrates empirical probabilistic distances of genome abundance and tetranucleotide frequency.</td>
      <td>2022-01-28</td>
      <td>https://bitbucket.org/berkeleylab/metabat/src/master/</td>
      <td>2.17</td>
      <td>metabat2</td>
      <td>2.18</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Read binning, Sequence assembly, Genome annotation</td>
      <td>Read binning, Sequence assembly, Genome annotation</td>
      <td>Metagenomics, Sequence assembly, Metagenomic sequencing</td>
      <td>Metagenomics, Sequence assembly, Metagenomic sequencing</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/metabat2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/metabat2</td>
      <td>MetaBAT_2</td>
      <td>MetaBAT 2</td>
      <td>an adaptive binning algorithm for robust and efficient genome reconstruction from metagenome assemblies | MetaBAT2 clusters metagenomic contigs into different "bins", each of which should correspond to a putative genome | MetaBAT2 uses nucleotide composition information and source strain abundance (measured by depth-of-coverage by aligning the reads to the contigs) to perform binning</td>
      <td></td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>670</td>
      <td>670</td>
      <td>14194</td>
      <td>14194</td>
      <td>375</td>
      <td>375</td>
      <td>2591</td>
      <td>2591</td>
      <td>167</td>
      <td>167</td>
      <td>1738</td>
      <td>1738</td>
      <td>18</td>
      <td>18</td>
      <td>1359</td>
      <td>1359</td>
      <td>19882</td>
      <td>19882</td>
      <td>1230</td>
      <td>1230</td>
      <td>https://usegalaxy.eu/published/workflow?id=00f15824c7a9b196, https://usegalaxy.eu/published/workflow?id=0563f58718be932d, https://usegalaxy.eu/published/workflow?id=1e5f1d9a51032e44, https://usegalaxy.eu/published/workflow?id=43098b8afe78b071, https://usegalaxy.eu/published/workflow?id=7ecd92cb382d8168, https://usegalaxy.eu/published/workflow?id=97312d273b6e8bd9, https://usegalaxy.eu/published/workflow?id=ce5c9287a67e6085, https://workflowhub.eu/workflows/1352?version=4, https://workflowhub.eu/workflows/2028?version=1</td>
      <td>microbiome/metagenomics-binning</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>metabuli</td>
      <td>metabuli_classify</td>
      <td>Classifying metagenomes by jointly analysing both DNA and amino acid (AA) sequences</td>
      <td>2024-06-04</td>
      <td>https://github.com/steineggerlab/Metabuli</td>
      <td>1.0.5</td>
      <td>metabuli</td>
      <td>1.1.0</td>
      <td>To update</td>
      <td>Sequence Analysis, Metagenomics</td>
      <td>Taxonomic classification</td>
      <td>Taxonomic classification</td>
      <td>Taxonomy</td>
      <td>Taxonomy</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/metabuli</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/metabuli</td>
      <td>metabuli</td>
      <td>metabuli</td>
      <td>Metabuli: specific and sensitive metagenomic classification via joint analysis of DNA and amino acid</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>metaeuk</td>
      <td>metaeuk_easy_predict</td>
      <td>MetaEuk is a modular toolkit designed for large-scale gene discovery andannotation in eukaryotic metagenomic contigs. Metaeuk combines the fast andsensitive homology search capabilities of MMseqs2 with a dynamic programmingprocedure to recover optimal exons sets. It reduces redundancies in multiplediscoveries of the same gene and resolves conflicting gene predictions onthe same strand.</td>
      <td>2020-08-04</td>
      <td>https://github.com/soedinglab/metaeuk</td>
      <td>7.bba0d80</td>
      <td>metaeuk</td>
      <td>6.a5d39d9</td>
      <td>To update</td>
      <td>Sequence Analysis, Genome annotation</td>
      <td>Homology-based gene prediction</td>
      <td>Homology-based gene prediction</td>
      <td>Metagenomics, Gene and protein families</td>
      <td>Metagenomics, Gene and protein families</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/metaeuk</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/metaeuk</td>
      <td>MetaEuk</td>
      <td>MetaEuk</td>
      <td>MetaEuk - sensitive, high-throughput gene discovery and annotation for large-scale eukaryotic metagenomics</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>57</td>
      <td>58</td>
      <td>507</td>
      <td>509</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>509</td>
      <td>507</td>
      <td>58</td>
      <td>57</td>
      <td>https://usegalaxy.eu/published/workflow?id=9b9e1e28f3683f92</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>metagene_annotator</td>
      <td>metagene_annotator</td>
      <td>MetaGeneAnnotator gene-finding program for prokaryote and phage</td>
      <td>2018-03-21</td>
      <td>http://metagene.nig.ac.jp/</td>
      <td>1.0</td>
      <td>metagene_annotator</td>
      <td>1.0</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Sequence annotation</td>
      <td>Sequence annotation</td>
      <td>Genomics, Model organisms, Data submission, annotation and curation</td>
      <td>Genomics, Model organisms, Data submission, annotation and curation</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/metagene_annotator</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/metagene_annotator</td>
      <td>metageneannotator</td>
      <td>MetaGeneAnnotator</td>
      <td>Prokaryotic gene finding program from environmental genome shotgun sequences or metagenomic sequences.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>439</td>
      <td>457</td>
      <td>20971</td>
      <td>21069</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>21069</td>
      <td>20971</td>
      <td>457</td>
      <td>439</td>
      <td>https://usegalaxy.eu/published/workflow?id=3dcf09e75d707573, https://usegalaxy.eu/published/workflow?id=799b924edd3c401b, https://usegalaxy.eu/published/workflow?id=9b9e1e28f3683f92, https://usegalaxy.eu/published/workflow?id=c0f225536e78a178</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>metagenomeseq_normalization</td>
      <td>metagenomeseq_normalizaton</td>
      <td>metagenomeSeq Normalization</td>
      <td>2017-03-27</td>
      <td>https://bioconductor.org/packages/3.18/bioc/html/metagenomeSeq.html</td>
      <td>1.16.0-0.0.1</td>
      <td>bioconductor-metagenomeseq</td>
      <td>1.43.0</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Sequence visualisation, Statistical calculation</td>
      <td>Sequence visualisation, Statistical calculation</td>
      <td>Metagenomics, Sequencing</td>
      <td>Metagenomics, Sequencing</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/metagenomeseq</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/metagenomeseq</td>
      <td>metagenomeseq</td>
      <td>metagenomeSeq</td>
      <td>Designed to determine features (be it Operational Taxanomic Unit (OTU), species, etc.) that are differentially abundant between two or more groups of multiple samples. It is designed to address the effects of both normalization and under-sampling of microbial communities on disease association detection and the testing of feature correlations.</td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>15</td>
      <td>23</td>
      <td>521</td>
      <td>763</td>
      <td>28</td>
      <td>56</td>
      <td>261</td>
      <td>530</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1293</td>
      <td>782</td>
      <td>79</td>
      <td>43</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>metamdbg</td>
      <td>metamdbg_asm, metamdbg_gfa</td>
      <td>MetaMDBG is a fast and low-memory assembler for long and accurate metagenomics reads.</td>
      <td>2026-01-21</td>
      <td>https://github.com/GaetanBenoitDev/metaMDBG</td>
      <td>1.2</td>
      <td>metamdbg</td>
      <td>1.2</td>
      <td>Up-to-date</td>
      <td>Assembly, Metagenomics</td>
      <td>Genome assembly</td>
      <td>Genome assembly</td>
      <td>Metagenomics</td>
      <td>Metagenomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/metamdbg</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/metamdbg</td>
      <td>metamdbg</td>
      <td>metaMDBG</td>
      <td>MetaMDBG is a fast and low-memory assembler for long and accurate metagenomics reads (e.g. PacBio HiFi, Nanopore r10.4). It is based on the minimizer de-Brujin graph (MDBG), which have been reimplemetend specifically for metagenomics assembly. MetaMDBG combines an efficient multi-k approach in minimizer-space for dealing with uneven species coverages, and a novel abundance-based filtering method for simplifying strain complexity.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>metanovo</td>
      <td>metanovo</td>
      <td>Produce targeted databases for mass spectrometry analysis.</td>
      <td>2022-03-29</td>
      <td>https://github.com/uct-cbio/proteomics-pipelines</td>
      <td>1.9.4</td>
      <td>metanovo</td>
      <td>1.9.4</td>
      <td>Up-to-date</td>
      <td>Proteomics</td>
      <td>Target-Decoy, de Novo sequencing, Tag-based peptide identification, Protein identification, Expression analysis</td>
      <td>Target-Decoy, de Novo sequencing, Tag-based peptide identification, Protein identification, Expression analysis</td>
      <td>Proteomics, Microbial ecology, Metagenomics, Proteomics experiment, Small molecules</td>
      <td>Proteomics, Microbial ecology, Metagenomics, Proteomics experiment, Small molecules</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/metanovo</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/metanovo</td>
      <td>metanovo</td>
      <td>MetaNovo</td>
      <td>An open-source pipeline for probabilistic peptide discovery in complex metaproteomic datasets.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>40</td>
      <td>40</td>
      <td>5000</td>
      <td>5000</td>
      <td>5</td>
      <td>5</td>
      <td>34</td>
      <td>34</td>
      <td>4</td>
      <td>4</td>
      <td>92</td>
      <td>92</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5126</td>
      <td>5126</td>
      <td>49</td>
      <td>49</td>
      <td>https://usegalaxy.eu/published/workflow?id=2f7e13eb26c8cd18, https://usegalaxy.eu/published/workflow?id=7cfbf4b7bd3f30d3, https://usegalaxy.eu/published/workflow?id=c6aecbdbeb27585f, https://usegalaxy.eu/published/workflow?id=d7c7a4d6c8072167, https://usegalaxy.eu/published/workflow?id=f2b45a7ff4f449e6, https://usegalaxy.org.au/published/workflow?id=351f3fab960c013b, https://usegalaxy.org/published/workflow?id=2be8ecd5922d23d1, https://workflowhub.eu/workflows/1216?version=1, https://workflowhub.eu/workflows/1461?version=1, https://workflowhub.eu/workflows/1474?version=1</td>
      <td>microbiome/clinical-mp-1-database-generation, proteomics/clinical-mp-1-database-generation</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>metaphlan</td>
      <td>customize_metaphlan_database, extract_metaphlan_database, merge_metaphlan_tables, metaphlan</td>
      <td>MetaPhlAn for Metagenomic Phylogenetic Analysis</td>
      <td>2021-03-11</td>
      <td>https://github.com/biobakery/MetaPhlAn</td>
      <td>4.2.4</td>
      <td>metaphlan</td>
      <td>4.2.4</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Taxonomic classification, Taxonomic classification</td>
      <td>Taxonomic classification, Taxonomic classification</td>
      <td>Metagenomics, Phylogenomics</td>
      <td>Metagenomics, Phylogenomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/metaphlan</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/metaphlan</td>
      <td>metaphlan</td>
      <td>MetaPhlAn</td>
      <td>Computational tool for profiling the composition of microbial communities from metagenomic shotgun sequencing data.</td>
      <td></td>
      <td>1</td>
      <td>2</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>1552</td>
      <td>1552</td>
      <td>30934</td>
      <td>31014</td>
      <td>1122</td>
      <td>1122</td>
      <td>20021</td>
      <td>20021</td>
      <td>314</td>
      <td>314</td>
      <td>6347</td>
      <td>6347</td>
      <td>28</td>
      <td>28</td>
      <td>211</td>
      <td>211</td>
      <td>57593</td>
      <td>57513</td>
      <td>3016</td>
      <td>3016</td>
      <td>https://usegalaxy.eu/published/workflow?id=05f5f040337786bf, https://usegalaxy.eu/published/workflow?id=19de5cbae7f48103, https://usegalaxy.eu/published/workflow?id=21a17d6c473a88c0, https://usegalaxy.eu/published/workflow?id=39d36a7ce0ef268d, https://usegalaxy.eu/published/workflow?id=4d32b1b3ac0a7161, https://usegalaxy.eu/published/workflow?id=4f0daf8e9dd54127, https://usegalaxy.eu/published/workflow?id=4f21205e71cabd37, https://usegalaxy.eu/published/workflow?id=5f0faac42b0a5062, https://usegalaxy.eu/published/workflow?id=713a5485e2007f90, https://usegalaxy.eu/published/workflow?id=7491883694fff308, https://usegalaxy.eu/published/workflow?id=79b4ae4403772098, https://usegalaxy.eu/published/workflow?id=87cc0cd143a12c41, https://usegalaxy.eu/published/workflow?id=9472be79da012999, https://usegalaxy.eu/published/workflow?id=a8f829deb69b30ab, https://usegalaxy.eu/published/workflow?id=b14845359b702444, https://usegalaxy.eu/published/workflow?id=cb2df493f5e334f7, https://usegalaxy.eu/published/workflow?id=fdca6c9d394986bf, https://usegalaxy.fr/published/workflow?id=09e8293456ee6d99, https://usegalaxy.fr/published/workflow?id=c4f6062e51d0f45e, https://usegalaxy.org.au/published/workflow?id=6b7e565258a9da2a, https://workflowhub.eu/workflows/1451?version=2, https://workflowhub.eu/workflows/1466?version=2, https://workflowhub.eu/workflows/1470?version=2, https://workflowhub.eu/workflows/624?version=1</td>
      <td>microbiome/metatranscriptomics, microbiome/metatranscriptomics-short, microbiome/taxonomic-profiling</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>metaquantome</td>
      <td>metaquantome_db, metaquantome_expand, metaquantome_filter, metaquantome_sample, metaquantome_stat, metaquantome_viz</td>
      <td>quantitative analysis of microbiome taxonomy and function</td>
      <td>2018-11-27</td>
      <td>https://github.com/galaxyproteomics/metaquantome/</td>
      <td>2.0.2</td>
      <td>metaquantome</td>
      <td>2.0.2</td>
      <td>Up-to-date</td>
      <td>Proteomics</td>
      <td>Principal component visualisation, Visualisation, Functional clustering, Query and retrieval, Differential protein expression analysis, Heat map generation, Quantification, Indexing, Filtering, Statistical inference</td>
      <td>Principal component visualisation, Functional clustering, Query and retrieval, Heat map generation, Quantification, Indexing, Filtering, Statistical inference</td>
      <td>Proteomics, Metatranscriptomics, Microbial ecology, Proteomics experiment, Metagenomics</td>
      <td>Proteomics, Metatranscriptomics, Microbial ecology, Proteomics experiment, Metagenomics</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/metaquantome</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/metaquantome</td>
      <td>metaQuantome</td>
      <td>metaQuantome</td>
      <td>metaQuantome software suite analyzes the state of a microbiome by leveraging complex taxonomic and functional hierarchies to summarize peptide-level quantitative information. metaQuantome offers differential abundance analysis, principal components analysis, and clustered heat map visualizations, as well as exploratory analysis for a single sample or experimental condition.</td>
      <td></td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>80</td>
      <td>89</td>
      <td>2277</td>
      <td>3166</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>27</td>
      <td>27</td>
      <td>2323</td>
      <td>2323</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5489</td>
      <td>4600</td>
      <td>116</td>
      <td>107</td>
      <td>https://usegalaxy.eu/published/workflow?id=4c364c6be27981ba, https://usegalaxy.eu/published/workflow?id=695f7defb516d80d, https://usegalaxy.eu/published/workflow?id=ce7f7c6eef526d67, https://usegalaxy.eu/published/workflow?id=dc438d4ae1866df8, https://usegalaxy.eu/published/workflow?id=e5a89ef7b5f1c1d9, https://usegalaxy.org.au/published/workflow?id=cada735fa0716570, https://usegalaxy.org.au/published/workflow?id=f8183bba3a1d150d, https://workflowhub.eu/workflows/1453?version=1, https://workflowhub.eu/workflows/1459?version=1</td>
      <td>proteomics/metaquantome-function, proteomics/metaquantome-taxonomy</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>metasbt</td>
      <td>metasbt_index, metasbt_profile</td>
      <td>Microbial genomes characterization with Sequence Bloom Trees</td>
      <td>2025-08-22</td>
      <td>https://github.com/cumbof/MetaSBT</td>
      <td>0.1.5</td>
      <td>sed</td>
      <td></td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/metasbt</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/metasbt</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>metawrapmg_binning</td>
      <td>metawrapmg_binning, metawrapmg_bin_refinement</td>
      <td>A flexible pipeline for genome-resolved metagenomic data analysis</td>
      <td>2024-04-11</td>
      <td>https://github.com/bxlab/metaWRAP</td>
      <td>1.3.0</td>
      <td>metawrap-mg</td>
      <td>1.3.0</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Read binning, Sequence assembly, Genome annotation, Sequence trimming, Demultiplexing</td>
      <td>Read binning, Sequence assembly, Genome annotation, Sequence trimming, Demultiplexing</td>
      <td>Whole genome sequencing, Metagenomic sequencing, Metagenomics</td>
      <td>Whole genome sequencing, Metagenomic sequencing, Metagenomics</td>
      <td>galaxy-australia</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/metawrapmg</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/metawrapmg</td>
      <td>metawrap</td>
      <td>MetaWRAP</td>
      <td>MetaWRAP aims to be an easy-to-use metagenomic wrapper suite that accomplishes the core tasks of metagenomic analysis from start to finish: read quality control, assembly, visualization, taxonomic profiling, extracting draft genomes (binning), and functional annotation.</td>
      <td></td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>100</td>
      <td>100</td>
      <td>1596</td>
      <td>1596</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>67</td>
      <td>67</td>
      <td>587</td>
      <td>587</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2183</td>
      <td>2183</td>
      <td>167</td>
      <td>167</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>mg_toolkit</td>
      <td>mg_toolkit_bulk_download, mg_toolkit_original_metadata</td>
      <td>Metagenomics toolkit enables scientists to download all of the sample metadata for a given study or sequence to a single csv file.</td>
      <td>2024-07-12</td>
      <td>https://github.com/EBI-Metagenomics/emg-toolkit</td>
      <td>0.10.4</td>
      <td>mg-toolkit</td>
      <td>0.10.4</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Data retrieval</td>
      <td>Data retrieval</td>
      <td>Metagenomics</td>
      <td>Metagenomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mg_toolkit</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mg_toolkit</td>
      <td>mg-toolkit</td>
      <td>mg-toolkit</td>
      <td>Metagenomics toolkit enables scientists to download all of the sample metadata for a given study or sequence to a single csv file.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>10</td>
      <td>21</td>
      <td>21</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>21</td>
      <td>21</td>
      <td>10</td>
      <td>10</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>mgnify_seqprep</td>
      <td>mgnify_seqprep</td>
      <td>A modifiied vesion of SeqPrep. Made for use with the MGnify pipelines.</td>
      <td>2024-05-14</td>
      <td>https://github.com/jstjohn/SeqPrep</td>
      <td>1.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>Fastq Manipulation, Sequence Analysis</td>
      <td>Nucleic acid design</td>
      <td>Nucleic acid design</td>
      <td>Genomics, Sequence assembly, Sequencing, Probes and primers</td>
      <td>Genomics, Sequence assembly, Sequencing, Probes and primers</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/mgnify_seqprep</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/mgnify_seqprep</td>
      <td>seqprep</td>
      <td>SeqPrep</td>
      <td>Strips adapters and optionally merges overlapping paired-end (or paired-end contamination in mate-pair libraries) illumina style reads.</td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>1348</td>
      <td>1348</td>
      <td>109</td>
      <td>109</td>
      <td>444</td>
      <td>444</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1792</td>
      <td>1792</td>
      <td>115</td>
      <td>115</td>
      <td>https://usegalaxy.eu/published/workflow?id=0609a4b1941d1a68, https://usegalaxy.eu/published/workflow?id=9fddc44294236e8f, https://usegalaxy.org/published/workflow?id=20b7ae4b517a80fd, https://usegalaxy.org/published/workflow?id=a71fcbc60caae8b6, https://workflowhub.eu/workflows/1272?version=2, https://workflowhub.eu/workflows/1854?version=1</td>
      <td>microbiome/mgnify-amplicon</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>micro_decon</td>
      <td>micro_decon</td>
      <td>An R package for removing contamination from metabarcoding (e.g., microbiome) datasets post-sequencing</td>
      <td>2025-08-05</td>
      <td>https://github.com/donaldtmcknight/microdecon</td>
      <td>1.0.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/micro_decon</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/micro_decon</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>minia</td>
      <td>minia</td>
      <td>Short-read assembler based on a de Bruijn graph</td>
      <td>2020-04-08</td>
      <td>https://gatb.inria.fr/software/minia/</td>
      <td>3.2.6</td>
      <td>minia</td>
      <td>3.2.6</td>
      <td>Up-to-date</td>
      <td>Assembly</td>
      <td>Genome assembly</td>
      <td>Genome assembly</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/minia</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/minia</td>
      <td>minia</td>
      <td>Minia</td>
      <td>Short-read assembler based on a de Bruijn graph, capable of assembling a human genome on a desktop computer in a day.</td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>272</td>
      <td>287</td>
      <td>6758</td>
      <td>7224</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>77</td>
      <td>77</td>
      <td>357</td>
      <td>357</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7581</td>
      <td>7115</td>
      <td>364</td>
      <td>349</td>
      <td>https://usegalaxy.eu/published/workflow?id=9ba580ba8b01db40</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>miniasm</td>
      <td>miniasm</td>
      <td>Miniasm - Ultrafast de novo assembly for long noisy reads (though having no consensus step)</td>
      <td>2019-06-18</td>
      <td>https://github.com/lh3/miniasm</td>
      <td>0.3_r179</td>
      <td>miniasm</td>
      <td>0.3</td>
      <td>To update</td>
      <td>Assembly</td>
      <td>De-novo assembly</td>
      <td>De-novo assembly</td>
      <td>Genomics, Sequence assembly</td>
      <td>Genomics, Sequence assembly</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/miniasm</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/miniasm</td>
      <td>miniasm</td>
      <td>miniasm</td>
      <td>Miniasm is a very fast OLC-based de novo assembler for noisy long reads. It takes all-vs-all read self-mappings (typically by minimap) as input and outputs an assembly graph in the GFA format.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>571</td>
      <td>687</td>
      <td>11506</td>
      <td>14702</td>
      <td>474</td>
      <td>538</td>
      <td>9124</td>
      <td>10016</td>
      <td>88</td>
      <td>96</td>
      <td>1235</td>
      <td>1277</td>
      <td>16</td>
      <td>16</td>
      <td>166</td>
      <td>166</td>
      <td>26161</td>
      <td>22031</td>
      <td>1337</td>
      <td>1149</td>
      <td>https://usegalaxy.eu/published/workflow?id=249c115413f17f28, https://usegalaxy.eu/published/workflow?id=32d2f5340098bcbd, https://usegalaxy.eu/published/workflow?id=39837ca09374598b, https://usegalaxy.eu/published/workflow?id=44e53ea319e07daa, https://usegalaxy.eu/published/workflow?id=b4de76c0a4d0e23a, https://usegalaxy.eu/published/workflow?id=c48807045e258dd8, https://usegalaxy.eu/published/workflow?id=dc7107896a3f2964, https://usegalaxy.org.au/published/workflow?id=d9571b3dfef34e5e, https://workflowhub.eu/workflows/1477?version=1, https://workflowhub.eu/workflows/406?version=1</td>
      <td>microbiome/plasmid-metagenomics-nanopore</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>minipolish</td>
      <td>minipolish</td>
      <td>Polishing miniasm assemblies</td>
      <td>2022-10-19</td>
      <td>https://github.com/rrwick/Minipolish</td>
      <td>0.2.0</td>
      <td>minipolish</td>
      <td>0.2.0</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Localised reassembly, Read depth analysis</td>
      <td>Localised reassembly, Read depth analysis</td>
      <td>Sequence assembly, Sequencing</td>
      <td>Sequence assembly, Sequencing</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/minipolish</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/minipolish</td>
      <td>minipolish</td>
      <td>minipolish</td>
      <td>A tool that bridges the output of miniasm (long-read assembly) and racon (assembly polishing) together to polish a draft assembly. It also provides read depth information in contigs.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>43</td>
      <td>43</td>
      <td>278</td>
      <td>278</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>278</td>
      <td>278</td>
      <td>43</td>
      <td>43</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>miniprot</td>
      <td>miniprot, miniprot_index</td>
      <td>Align a protein sequence against a genome with affine gap penalty, splicing and frameshift.</td>
      <td>2022-09-19</td>
      <td>https://github.com/lh3/miniprot</td>
      <td>0.18</td>
      <td>miniprot</td>
      <td>0.18</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Sequence alignment, Protein sequence analysis</td>
      <td>Sequence alignment, Protein sequence analysis</td>
      <td>Sequence sites, features and motifs, Sequence analysis</td>
      <td>Sequence sites, features and motifs, Sequence analysis</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/miniprot</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/miniprot</td>
      <td>miniprot</td>
      <td>miniprot</td>
      <td>Miniprot aligns a protein sequence against a genome with affine gap penalty, splicing and frameshift. It is primarily intended for annotating protein-coding genes in a new species using known genes from other species.</td>
      <td></td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>100</td>
      <td>100</td>
      <td>5038</td>
      <td>5038</td>
      <td>42</td>
      <td>42</td>
      <td>364</td>
      <td>364</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>25</td>
      <td>25</td>
      <td>283</td>
      <td>283</td>
      <td>5685</td>
      <td>5685</td>
      <td>167</td>
      <td>167</td>
      <td>https://usegalaxy.eu/published/workflow?id=0c92c3c322c3effe</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>mitos</td>
      <td>mitos, mitos2</td>
      <td>de-novo annotation of metazoan mitochondrial genomes</td>
      <td>2020-02-18</td>
      <td>http://mitos.bioinf.uni-leipzig.de/</td>
      <td>1.1.7</td>
      <td>mitos</td>
      <td>2.1.10</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Genome annotation</td>
      <td>Genome annotation</td>
      <td>Zoology, Whole genome sequencing</td>
      <td>Zoology, Whole genome sequencing</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mitos</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mitos</td>
      <td>mitos</td>
      <td>MITOS</td>
      <td>De novo metazoan mitochondrial genome annotation.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>1</td>
      <td>0</td>
      <td>1326</td>
      <td>1336</td>
      <td>267673</td>
      <td>268177</td>
      <td>772</td>
      <td>772</td>
      <td>84561</td>
      <td>84561</td>
      <td>151</td>
      <td>151</td>
      <td>15492</td>
      <td>15492</td>
      <td>6</td>
      <td>6</td>
      <td>109</td>
      <td>109</td>
      <td>368339</td>
      <td>367835</td>
      <td>2265</td>
      <td>2255</td>
      <td>https://usegalaxy.eu/published/workflow?id=0de5fcc38ef1501d, https://usegalaxy.eu/published/workflow?id=2d0400b5d05c6c10, https://usegalaxy.org/published/workflow?id=110ad20060b03e02</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>mlst</td>
      <td>mlst, mlst_list</td>
      <td>Scan contig files against PubMLST typing schemes</td>
      <td>2016-12-12</td>
      <td>https://github.com/tseemann/mlst</td>
      <td>2.22.0</td>
      <td>mlst</td>
      <td>2.32.2</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Multilocus sequence typing</td>
      <td>Multilocus sequence typing</td>
      <td>Immunoproteins and antigens</td>
      <td>Immunoproteins and antigens</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mlst</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mlst</td>
      <td>mlst</td>
      <td>MLST</td>
      <td>Multi Locus Sequence Typing from an assembled genome or from a set of reads.</td>
      <td></td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>1786</td>
      <td>1918</td>
      <td>24365</td>
      <td>26341</td>
      <td>1356</td>
      <td>1420</td>
      <td>12903</td>
      <td>13604</td>
      <td>1137</td>
      <td>1439</td>
      <td>12595</td>
      <td>15564</td>
      <td>22</td>
      <td>22</td>
      <td>99</td>
      <td>99</td>
      <td>55608</td>
      <td>49962</td>
      <td>4799</td>
      <td>4301</td>
      <td>https://usegalaxy.eu/published/workflow?id=2b888e200dfd91bc, https://usegalaxy.eu/published/workflow?id=309fc4fc2e0da89c, https://usegalaxy.eu/published/workflow?id=368d1ac0150ea0b6, https://usegalaxy.eu/published/workflow?id=37da7b42cb063a3a, https://usegalaxy.eu/published/workflow?id=42c0da2e86318b37, https://usegalaxy.eu/published/workflow?id=49c46d7653d6c349, https://usegalaxy.eu/published/workflow?id=5826e573b6f93762, https://usegalaxy.eu/published/workflow?id=5b940b283d91d99d, https://usegalaxy.eu/published/workflow?id=9219557de92e0203, https://usegalaxy.eu/published/workflow?id=a337bfb2eb1e583f, https://usegalaxy.eu/published/workflow?id=a84bac86ec183d46, https://usegalaxy.eu/published/workflow?id=eb5ab8bc82569eee, https://usegalaxy.eu/published/workflow?id=fde21b5f5c169114, https://usegalaxy.org.au/published/workflow?id=59618ed15510f531, https://usegalaxy.org.au/published/workflow?id=7a0c4804e4d4c905, https://usegalaxy.org.au/published/workflow?id=914330feb75ea9ab, https://usegalaxy.org.au/published/workflow?id=92fcdd490831c6c5, https://usegalaxy.org.au/published/workflow?id=c997a34b55425155, https://usegalaxy.org.au/published/workflow?id=e20172eb0f30d6cd, https://usegalaxy.org/published/workflow?id=80565301ada0d3eb, https://usegalaxy.org/published/workflow?id=8111bb9bb26bb665, https://usegalaxy.org/published/workflow?id=c65bec6c332bf740, https://usegalaxy.org/published/workflow?id=fb01e1a49793c8be</td>
      <td>microbiome/pathogen-detection-from-nanopore-foodborne-data</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>mmseqs2</td>
      <td>mmseqs2_easy_linclust_clustering, mmseqs2_taxonomy_assignment</td>
      <td>MMseqs2 is an ultra fast and sensitive sequence search and clustering suite</td>
      <td>2025-03-27</td>
      <td>https://github.com/soedinglab/MMseqs2</td>
      <td>17-b804f</td>
      <td>mmseqs2</td>
      <td>17.b804f</td>
      <td>To update</td>
      <td>Sequence Analysis, Metagenomics</td>
      <td>Sequence similarity search, Sequence alignment, Sequence clustering, Taxonomic classification</td>
      <td>Sequence similarity search, Sequence alignment, Sequence clustering, Taxonomic classification</td>
      <td>Metagenomics, Sequence analysis, Proteins, Nucleic acids, Gene and protein families, Taxonomy</td>
      <td>Metagenomics, Sequence analysis, Nucleic acids, Gene and protein families, Taxonomy</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mmseqs2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mmseqs2</td>
      <td>MMseqs2</td>
      <td>MMseqs2</td>
      <td>MMseqs2 (Many-against-Many sequence searching) is a software suite to search and cluster huge protein and nucleotide sequence sets. MMseqs2 is open source software implemented in C++ for Linux, MacOS, and (as beta version, via cygwin) Windows. The software is designed to run on multiple cores and servers and exhibits very good scalability. MMseqs2 can run 10000 times faster than BLAST. At 100 times its speed it achieves almost the same sensitivity. It can perform profile searches with the same sensitivity as PSI-BLAST at over 400 times its speed.MMseqs2 includes Linclust, the first clustering algorithm whose runtime scales linearly With Linclust we clustered 1.6 billion metagenomic sequence fragments in 10h on a single server to 50% sequence identity.</td>
      <td></td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>42</td>
      <td>42</td>
      <td>260</td>
      <td>260</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>29</td>
      <td>29</td>
      <td>289</td>
      <td>289</td>
      <td>45</td>
      <td>45</td>
      <td>https://usegalaxy.eu/published/workflow?id=62392daea25a3980, https://workflowhub.eu/workflows/2024?version=2</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>mmuphin</td>
      <td>mmuphin</td>
      <td>MMUPHin is an R package implementing meta-analysis methods for microbial community profiles</td>
      <td>2024-11-22</td>
      <td>https://huttenhower.sph.harvard.edu/mmuphin</td>
      <td>1.16.0</td>
      <td>bioconductor-mmuphin</td>
      <td>1.16.0</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mmuphin</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mmuphin</td>
      <td>mmuphin</td>
      <td>mmuphin</td>
      <td>MMUPHin is an R package for meta-analysis tasks of microbiome cohorts. It has function interfaces for: a) covariate-controlled batch- and cohort effect adjustment, b) meta-analysis differential abundance testing, c) meta-analysis unsupervised discrete structure (clustering) discovery, and d) meta-analysis unsupervised continuous structure discovery.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>14</td>
      <td>14</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>14</td>
      <td>14</td>
      <td>3</td>
      <td>3</td>
      <td>https://usegalaxy.eu/published/workflow?id=466bdd8ba7b67264</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>mob_suite</td>
      <td>mob_recon, mob_typer</td>
      <td>MOB-suite is a set of software tools for clustering, reconstruction and typing of plasmids from draft assemblies</td>
      <td>2018-04-30</td>
      <td>https://github.com/phac-nml/mob-suite</td>
      <td>3.1.9</td>
      <td>mob_suite</td>
      <td>3.1.9</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Clustering, Sequence analysis</td>
      <td>Clustering, Sequence analysis</td>
      <td>Metagenomics, Sequence assembly, Genotype and phenotype, Mobile genetic elements, Genomics</td>
      <td>Metagenomics, Sequence assembly, Genotype and phenotype, Mobile genetic elements</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/mob-suite</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/mob_suite</td>
      <td>mob_suite</td>
      <td>mob_suite</td>
      <td>MOB-suite: Software tools for clustering, reconstruction and typing of plasmids from draft assembliesThe MOB-suite depends on a series of databases which are too large to be hosted in git-hub. They can be downloaded or updated by running mob_init or if running any of the tools for the first time, the databases will download and initialize automatically if you do not specify an alternate database location.</td>
      <td></td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>620</td>
      <td>620</td>
      <td>162457</td>
      <td>162457</td>
      <td>1</td>
      <td>1</td>
      <td>42</td>
      <td>42</td>
      <td>199</td>
      <td>199</td>
      <td>66525</td>
      <td>66525</td>
      <td>12</td>
      <td>12</td>
      <td>58</td>
      <td>58</td>
      <td>229082</td>
      <td>229082</td>
      <td>832</td>
      <td>832</td>
      <td>https://workflowhub.eu/workflows/407?version=1</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>mothur</td>
      <td>mothur_align_check, mothur_align_seqs, mothur_amova, mothur_anosim, mothur_bin_seqs, mothur_biom_info, mothur_chimera_bellerophon, mothur_chimera_ccode, mothur_chimera_check, mothur_chimera_perseus, mothur_chimera_pintail, mothur_chimera_slayer, mothur_chimera_uchime, mothur_chimera_vsearch, mothur_chop_seqs, mothur_classify_otu, mothur_classify_seqs, mothur_classify_tree, mothur_clearcut, mothur_cluster_classic, mothur_cluster_fragments, mothur_cluster_split, mothur_cluster, mothur_collect_shared, mothur_collect_single, mothur_consensus_seqs, mothur_cooccurrence, mothur_corr_axes, mothur_count_groups, mothur_count_seqs, mothur_create_database, mothur_degap_seqs, mothur_deunique_seqs, mothur_deunique_tree, mothur_dist_seqs, mothur_dist_shared, mothur_fastq_info, mothur_filter_seqs, mothur_filter_shared, mothur_get_communitytype, mothur_get_coremicrobiome, mothur_get_dists, mothur_get_group, mothur_get_groups, mothur_get_label, mothur_get_lineage, mothur_get_mimarkspackage, mothur_get_otulabels, mothur_get_otulist, mothur_get_oturep, mothur_get_otus, mothur_get_rabund, mothur_get_relabund, mothur_get_sabund, mothur_get_seqs, mothur_get_sharedseqs, mothur_heatmap_bin, mothur_heatmap_sim, mothur_homova, mothur_indicator, mothur_lefse, mothur_libshuff, mothur_list_otulabels, mothur_list_seqs, mothur_make_biom, mothur_make_contigs, mothur_make_design, mothur_make_fastq, mothur_make_group, mothur_make_lefse, mothur_make_lookup, mothur_make_shared, mothur_make_sra, mothur_mantel, mothur_merge_count, mothur_merge_files, mothur_merge_groups, mothur_merge_sfffiles, mothur_merge_taxsummary, mothur_metastats, mothur_mimarks_attributes, mothur_nmds, mothur_normalize_shared, mothur_otu_association, mothur_otu_hierarchy, mothur_pairwise_seqs, mothur_parse_list, mothur_parsimony, mothur_pca, mothur_pcoa, mothur_pcr_seqs, mothur_phylo_diversity, mothur_phylotype, mothur_pre_cluster, mothur_primer_design, mothur_rarefaction_shared, mothur_rarefaction_single, mothur_remove_dists, mothur_remove_groups, mothur_remove_lineage, mothur_remove_otulabels, mothur_remove_otus, mothur_remove_rare, mothur_remove_seqs, mothur_rename_seqs, mothur_reverse_seqs, mothur_screen_seqs, mothur_sens_spec, mothur_seq_error, mothur_sffinfo, mothur_shhh_flows, mothur_shhh_seqs, mothur_sort_seqs, mothur_split_abund, mothur_split_groups, mothur_sub_sample, mothur_summary_qual, mothur_summary_seqs, mothur_summary_shared, mothur_summary_single, mothur_summary_tax, mothur_taxonomy_to_krona, mothur_tree_shared, mothur_trim_flows, mothur_trim_seqs, mothur_unifrac_unweighted, mothur_unifrac_weighted, mothur_unique_seqs, mothur_venn</td>
      <td>Mothur wrappers</td>
      <td>2016-06-24</td>
      <td>https://www.mothur.org</td>
      <td>1.0</td>
      <td>mothur</td>
      <td>1.48.5</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>DNA barcoding, Sequencing quality control, Sequence clustering, Taxonomic classification, Visualisation, Sequence read processing, Phylogenetic analysis</td>
      <td>DNA barcoding, Sequencing quality control, Sequence clustering, Taxonomic classification, Visualisation, Sequence read processing, Phylogenetic analysis</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Phylogeny</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Phylogeny</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mothur</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mothur</td>
      <td>mothur</td>
      <td>mothur</td>
      <td>Open-source, platform-independent, community-supported software for describing and comparing microbial communities</td>
      <td></td>
      <td>129</td>
      <td>129</td>
      <td>129</td>
      <td>129</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>129</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>126</td>
      <td>129</td>
      <td>129</td>
      <td>0</td>
      <td>2291</td>
      <td>2805</td>
      <td>265024</td>
      <td>361038</td>
      <td>4027</td>
      <td>5455</td>
      <td>423935</td>
      <td>750968</td>
      <td>1447</td>
      <td>1742</td>
      <td>164152</td>
      <td>213607</td>
      <td>147</td>
      <td>149</td>
      <td>14583</td>
      <td>14948</td>
      <td>1340561</td>
      <td>867694</td>
      <td>10151</td>
      <td>7912</td>
      <td>https://usegalaxy.eu/published/workflow?id=565c989730f68362, https://usegalaxy.eu/published/workflow?id=6d06bd22f206e176, https://usegalaxy.eu/published/workflow?id=6efd80daabf518bf, https://usegalaxy.eu/published/workflow?id=7783eeebfa264bf5, https://usegalaxy.eu/published/workflow?id=83a0c1ba280785e6, https://usegalaxy.eu/published/workflow?id=8f747dbff3216993, https://usegalaxy.eu/published/workflow?id=b66e1d8171e6dac2, https://usegalaxy.eu/published/workflow?id=b86f24473a282ef9, https://usegalaxy.eu/published/workflow?id=b91a380ceaa0342b, https://usegalaxy.eu/published/workflow?id=c1e26abe61c69688, https://usegalaxy.eu/published/workflow?id=c55388a8fbfd7381, https://usegalaxy.eu/published/workflow?id=c67d2e75eeb4fd19, https://usegalaxy.eu/published/workflow?id=ce21961260fe4351, https://usegalaxy.eu/published/workflow?id=dadca1020f586338, https://usegalaxy.eu/published/workflow?id=e14b81a8c0b1b7a4, https://usegalaxy.eu/published/workflow?id=f21d6a40a10a8ae8, https://usegalaxy.fr/published/workflow?id=41264142d3c00016, https://usegalaxy.fr/published/workflow?id=8b1678200a1d157a, https://usegalaxy.org.au/published/workflow?id=177579096e009178, https://usegalaxy.org.au/published/workflow?id=1954437da1e0c4de, https://usegalaxy.org.au/published/workflow?id=2717e90438c7fb8c, https://usegalaxy.org.au/published/workflow?id=2a6324600f159061, https://usegalaxy.org.au/published/workflow?id=57fe85f947217994, https://usegalaxy.org.au/published/workflow?id=5877e2d5eba79f36, https://usegalaxy.org.au/published/workflow?id=58bd08b1530e7cae, https://usegalaxy.org.au/published/workflow?id=670e100ed3c039e2, https://usegalaxy.org.au/published/workflow?id=74bd939615090c2c, https://usegalaxy.org.au/published/workflow?id=7e381c150820e506, https://usegalaxy.org.au/published/workflow?id=832fe9dfc67fd8ed, https://usegalaxy.org.au/published/workflow?id=a52609119b30e64d, https://usegalaxy.org.au/published/workflow?id=fa8d1de86aa944a2, https://usegalaxy.org/published/workflow?id=2d149545a4920326, https://usegalaxy.org/published/workflow?id=4b7e73bb7ac93c95, https://usegalaxy.org/published/workflow?id=603e5c61c57bcf3f, https://usegalaxy.org/published/workflow?id=64fa13bb48fcd7df, https://usegalaxy.org/published/workflow?id=6a1eb7e7bececba1, https://usegalaxy.org/published/workflow?id=6f090b2c259847d2, https://usegalaxy.org/published/workflow?id=7251c6e0dad8f927, https://usegalaxy.org/published/workflow?id=775335812a6b7a61, https://usegalaxy.org/published/workflow?id=7d005f30cd4853c4, https://usegalaxy.org/published/workflow?id=a383ce2e2bc3c7b9, https://usegalaxy.org/published/workflow?id=b2f3d7e4cd4d0f11, https://usegalaxy.org/published/workflow?id=bc01b8b203ecc754, https://usegalaxy.org/published/workflow?id=cade0c5c11978323, https://usegalaxy.org/published/workflow?id=d653eaefcf6b4697, https://usegalaxy.org/published/workflow?id=d7a11a4dfda3dab4, https://workflowhub.eu/workflows/1400?version=1, https://workflowhub.eu/workflows/1404?version=1, https://workflowhub.eu/workflows/1408?version=1, https://workflowhub.eu/workflows/1412?version=1, https://workflowhub.eu/workflows/1418?version=1, https://workflowhub.eu/workflows/1422?version=1, https://workflowhub.eu/workflows/1428?version=1, https://workflowhub.eu/workflows/1465?version=1, https://workflowhub.eu/workflows/1476?version=1, https://workflowhub.eu/workflows/1630?version=1, https://workflowhub.eu/workflows/1635?version=1, https://workflowhub.eu/workflows/2038?version=1, https://workflowhub.eu/workflows/648?version=1, https://workflowhub.eu/workflows/649?version=1, https://workflowhub.eu/workflows/650?version=1, https://workflowhub.eu/workflows/651?version=1, https://workflowhub.eu/workflows/652?version=1, https://workflowhub.eu/workflows/653?version=1</td>
      <td>microbiome/general-tutorial, microbiome/mothur-miseq-sop, microbiome/mothur-miseq-sop-short, sequence-analysis/Manage_AB1_Sanger</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>motus</td>
      <td>mereg_mOTUs_tables, motus_profiler</td>
      <td>Tool for profiling the abundance of microbial taxa.</td>
      <td>2024-09-06</td>
      <td>https://github.com/motu-tool/mOTUs</td>
      <td>3.1.0</td>
      <td>motus</td>
      <td>3.1.0</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Taxonomic classification</td>
      <td>Taxonomic classification</td>
      <td>Metagenomics</td>
      <td>Metagenomics</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/mOTUs</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/motus</td>
      <td>mOTUs</td>
      <td>Metagenomic operational taxonomic units (mOTUs)</td>
      <td>Metagenomic operational taxonomic units (mOTUs) enable high-accuracy taxonomic profiling of known (sequenced) and unknown microorganisms at species-level resolution from shotgun metagenomic or metatranscriptomic data. The method clusters single-copy phylogenetic marker gene sequences from metagenomes and reference genomes into mOTUs to quantify their abundances in meta-omics data with very high precision and recall.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>40</td>
      <td>40</td>
      <td>377</td>
      <td>377</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>377</td>
      <td>377</td>
      <td>40</td>
      <td>40</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>mrbayes</td>
      <td>mrbayes</td>
      <td>A program for the Bayesian estimation of phylogeny.</td>
      <td>2015-12-04</td>
      <td></td>
      <td>1.0.2</td>
      <td>mrbayes</td>
      <td>3.2.7</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>nml</td>
      <td></td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/mrbayes</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>18</td>
      <td>18</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>18</td>
      <td>18</td>
      <td>4</td>
      <td>4</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>msconvert</td>
      <td>msconvert</td>
      <td>msconvert Convert and/or filter mass spectrometry files (including vendor formats) using the official Docker container</td>
      <td>2015-01-24</td>
      <td>http://proteowizard.sourceforge.net/tools.shtml</td>
      <td>3.0.20287</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>Proteomics</td>
      <td>Filtering, Formatting</td>
      <td>Filtering, Formatting</td>
      <td>Proteomics, Proteomics experiment</td>
      <td>Proteomics, Proteomics experiment</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msconvert</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msconvert</td>
      <td>msconvert</td>
      <td>msConvert</td>
      <td>msConvert is a command-line utility for converting between various mass spectrometry data formats, including from raw data from several commercial companies (with vendor libraries, Windows-only). For Windows users, there is also a GUI, msConvertGUI.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>550</td>
      <td>604</td>
      <td>58574</td>
      <td>66339</td>
      <td>133</td>
      <td>133</td>
      <td>2254</td>
      <td>2254</td>
      <td>76</td>
      <td>76</td>
      <td>4844</td>
      <td>4844</td>
      <td>108</td>
      <td>113</td>
      <td>7765</td>
      <td>7906</td>
      <td>81343</td>
      <td>73437</td>
      <td>926</td>
      <td>867</td>
      <td>https://usegalaxy.eu/published/workflow?id=02700454f57eb556, https://usegalaxy.eu/published/workflow?id=14f9c2099db30570, https://usegalaxy.eu/published/workflow?id=1809fcaec264eac9, https://usegalaxy.eu/published/workflow?id=3676aafc35dc2fe7, https://usegalaxy.eu/published/workflow?id=3e01a2c6d954ae6e, https://usegalaxy.eu/published/workflow?id=41df8c7821b5f2fc, https://usegalaxy.eu/published/workflow?id=41e9882300fa2386, https://usegalaxy.eu/published/workflow?id=4c81726bde25c36a, https://usegalaxy.eu/published/workflow?id=5d4e4b9f07a1ec20, https://usegalaxy.eu/published/workflow?id=5dec0e5a2d75cdc1, https://usegalaxy.eu/published/workflow?id=5e4774036aa20b80, https://usegalaxy.eu/published/workflow?id=607ca3a9d45bd080, https://usegalaxy.eu/published/workflow?id=612dc5bdf388585f, https://usegalaxy.eu/published/workflow?id=6b7b86756060659b, https://usegalaxy.eu/published/workflow?id=6cb85fe1ad0c9f3b, https://usegalaxy.eu/published/workflow?id=6d3332794ceca465, https://usegalaxy.eu/published/workflow?id=6d494e2f634eba70, https://usegalaxy.eu/published/workflow?id=82b6871682f1659a, https://usegalaxy.eu/published/workflow?id=87c21b682af78ec9, https://usegalaxy.eu/published/workflow?id=8af41219411062ad, https://usegalaxy.eu/published/workflow?id=912b81c444e35045, https://usegalaxy.eu/published/workflow?id=9ac15b644c805a2b, https://usegalaxy.eu/published/workflow?id=9ef76d764d9961b2, https://usegalaxy.eu/published/workflow?id=b73fa5f8c0403454, https://usegalaxy.eu/published/workflow?id=ba3a5d59b3879764, https://usegalaxy.eu/published/workflow?id=c4119ea7ffd29094, https://usegalaxy.eu/published/workflow?id=c750484145c9903b, https://usegalaxy.eu/published/workflow?id=cd675ab32d3e7833, https://usegalaxy.eu/published/workflow?id=d0e062aee910086a, https://usegalaxy.eu/published/workflow?id=d5756e3ebfa285ef, https://usegalaxy.eu/published/workflow?id=eb54512829d8509c, https://usegalaxy.eu/published/workflow?id=f0bec337f9998289, https://usegalaxy.eu/published/workflow?id=fc42cc9467c6ee8d, https://usegalaxy.fr/published/workflow?id=1407c1703f360f07, https://usegalaxy.org.au/published/workflow?id=049c52f685984dec, https://usegalaxy.org.au/published/workflow?id=762791e094457bf2, https://usegalaxy.org.au/published/workflow?id=8c874ea77827448c, https://usegalaxy.org.au/published/workflow?id=ba217a34d7e1b5ee, https://usegalaxy.org/published/workflow?id=8ecc114eb9097855, https://usegalaxy.org/published/workflow?id=e97da2da9e0290a7, https://workflowhub.eu/workflows/1225?version=1, https://workflowhub.eu/workflows/1396?version=1, https://workflowhub.eu/workflows/1416?version=1, https://workflowhub.eu/workflows/1435?version=1, https://workflowhub.eu/workflows/1450?version=1, https://workflowhub.eu/workflows/1463?version=1, https://workflowhub.eu/workflows/1471?version=1, https://workflowhub.eu/workflows/1582?version=1, https://workflowhub.eu/workflows/1791?version=1</td>
      <td>metabolomics/gc_ms_with_xcms, microbiome/clinical-mp-2-discovery, proteomics/DIA_Analysis_OSW, proteomics/clinical-mp-2-discovery, proteomics/encyclopedia, proteomics/metaquantome-data-creation, proteomics/neoantigen-peptide-verification, proteomics/protein-id-oms</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>msstatstmt</td>
      <td>msstats</td>
      <td>MSstats tool for analyzing mass spectrometry proteomic datasets</td>
      <td>2020-07-25</td>
      <td>https://github.com/MeenaChoi/MSstats</td>
      <td>4.0.0</td>
      <td>bioconductor-msstats</td>
      <td>4.14.0</td>
      <td>To update</td>
      <td>Proteomics</td>
      <td>Spectrum calculation, Tag-based peptide identification, Differential protein expression profiling</td>
      <td>Spectrum calculation, Tag-based peptide identification, Differential protein expression profiling</td>
      <td>Proteomics, Proteomics experiment, Protein expression</td>
      <td>Proteomics, Proteomics experiment, Protein expression</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msstats</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msstats</td>
      <td>msstatstmt</td>
      <td>MSstatsTMT</td>
      <td>Tools for detecting differentially abundant peptides and proteins in shotgun mass spectrometry-based proteomic experiments with tandem mass tag (TMT) labeling</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>800</td>
      <td>803</td>
      <td>2969</td>
      <td>3004</td>
      <td>283</td>
      <td>283</td>
      <td>1456</td>
      <td>1456</td>
      <td>99</td>
      <td>99</td>
      <td>1605</td>
      <td>1605</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6065</td>
      <td>6030</td>
      <td>1185</td>
      <td>1182</td>
      <td>https://usegalaxy.org.au/published/workflow?id=1727d16fa4dec19c, https://workflowhub.eu/workflows/1426?version=1, https://workflowhub.eu/workflows/1458?version=1, https://workflowhub.eu/workflows/1462?version=1</td>
      <td>proteomics/DIA_Analysis_MSstats, proteomics/maxquant-label-free, proteomics/maxquant-msstats-dda-lfq</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>msstatstmt</td>
      <td>msstatstmt</td>
      <td>MSstatsTMT protein significance analysis in shotgun mass spectrometry-based proteomic experiments with tandem mass tag (TMT) labeling</td>
      <td>2021-01-20</td>
      <td>http://msstats.org/msstatstmt/</td>
      <td>2.0.0</td>
      <td>bioconductor-msstatstmt</td>
      <td>2.10.0</td>
      <td>To update</td>
      <td>Proteomics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msstatstmt</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msstatstmt</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>113</td>
      <td>113</td>
      <td>844</td>
      <td>844</td>
      <td>10</td>
      <td>10</td>
      <td>47</td>
      <td>47</td>
      <td>18</td>
      <td>18</td>
      <td>693</td>
      <td>693</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1584</td>
      <td>1584</td>
      <td>141</td>
      <td>141</td>
      <td>https://usegalaxy.eu/published/workflow?id=755ede37d94dc456, https://usegalaxy.eu/published/workflow?id=8f3cafa3bc59ff82, https://usegalaxy.eu/published/workflow?id=d148459a4d4b91f1, https://usegalaxy.org.au/published/workflow?id=050a66ff6d4c2b53, https://usegalaxy.org/published/workflow?id=3f797bfb67130883, https://workflowhub.eu/workflows/1219?version=1, https://workflowhub.eu/workflows/1401?version=1, https://workflowhub.eu/workflows/1433?version=1, https://workflowhub.eu/workflows/1482?version=1</td>
      <td>microbiome/clinical-mp-5-data-interpretation, proteomics/clinical-mp-5-data-interpretation, proteomics/maxquant-msstats-tmt</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>multigsea</td>
      <td>multigsea</td>
      <td>GSEA-based pathway enrichment analysis for multi-omics data</td>
      <td>2023-06-07</td>
      <td>https://bioconductor.org/packages/devel/bioc/html/multiGSEA.html</td>
      <td>1.12.0</td>
      <td>bioconductor-multigsea</td>
      <td>1.16.0</td>
      <td>To update</td>
      <td>Transcriptomics, Proteomics, Statistics, Single Cell</td>
      <td>Gene-set enrichment analysis, Aggregation, Pathway analysis</td>
      <td>Gene-set enrichment analysis, Aggregation, Pathway analysis</td>
      <td>Metabolomics, Molecular interactions, pathways and networks, Proteomics, Transcriptomics, Small molecules</td>
      <td>Metabolomics, Molecular interactions, pathways and networks, Proteomics, Transcriptomics, Small molecules</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/multigsea</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/multigsea</td>
      <td>multiGSEA</td>
      <td>multiGSEA</td>
      <td>A GSEA-based pathway enrichment analysis for multi-omics data.multiGSEA: a GSEA-based pathway enrichment analysis for multi-omics data, BMC Bioinformatics 21, 561 (2020).Combining GSEA-based pathway enrichment with multi omics data integration.</td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>65</td>
      <td>65</td>
      <td>212</td>
      <td>212</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>46</td>
      <td>46</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>258</td>
      <td>258</td>
      <td>70</td>
      <td>70</td>
      <td>https://workflowhub.eu/workflows/1388?version=1</td>
      <td>proteomics/multiGSEA-tutorial</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>multiqc</td>
      <td>multiqc</td>
      <td>MultiQC aggregates results from bioinformatics analyses across many samples into a single report</td>
      <td>2017-09-21</td>
      <td>http://multiqc.info/</td>
      <td>1.33</td>
      <td>multiqc</td>
      <td>1.33</td>
      <td>Up-to-date</td>
      <td>Fastq Manipulation, Statistics, Visualization</td>
      <td>Validation, Sequencing quality control</td>
      <td>Sequencing quality control</td>
      <td>Sequencing, Bioinformatics, Sequence analysis, Genomics</td>
      <td>Sequencing, Bioinformatics, Sequence analysis, Genomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/multiqc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/multiqc</td>
      <td>multiqc</td>
      <td>MultiQC</td>
      <td>MultiQC aggregates results from multiple bioinformatics analyses across many samples into a single report. It searches a given directory for analysis logs and compiles a HTML report. It's a general use tool, perfect for summarising the output from numerous bioinformatics tools.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>24162</td>
      <td>26916</td>
      <td>223019</td>
      <td>248625</td>
      <td>32289</td>
      <td>38522</td>
      <td>203104</td>
      <td>241071</td>
      <td>7040</td>
      <td>7764</td>
      <td>49577</td>
      <td>54957</td>
      <td>2276</td>
      <td>2280</td>
      <td>43768</td>
      <td>43966</td>
      <td>588619</td>
      <td>519468</td>
      <td>75482</td>
      <td>65767</td>
      <td>https://usegalaxy.eu/published/workflow?id=002a51bdef00e14d, https://usegalaxy.eu/published/workflow?id=0189bab68f828d86, https://usegalaxy.eu/published/workflow?id=01fe4c590e608103, https://usegalaxy.eu/published/workflow?id=020f877513f2d0d1, https://usegalaxy.eu/published/workflow?id=023a0ee62dd1f593, https://usegalaxy.eu/published/workflow?id=03be9d62e8e4a129, https://usegalaxy.eu/published/workflow?id=03d1d2e0dc6e2024, https://usegalaxy.eu/published/workflow?id=04ed3f21de17aa1d, https://usegalaxy.eu/published/workflow?id=0563f58718be932d, https://usegalaxy.eu/published/workflow?id=0599f049e4697839, https://usegalaxy.eu/published/workflow?id=0609a4b1941d1a68, https://usegalaxy.eu/published/workflow?id=079e2a968d8e6176, https://usegalaxy.eu/published/workflow?id=07aefca769095616, https://usegalaxy.eu/published/workflow?id=07b74e4cf53cb633, https://usegalaxy.eu/published/workflow?id=082bfd9b36e921e8, https://usegalaxy.eu/published/workflow?id=085c3a4c89d137b5, https://usegalaxy.eu/published/workflow?id=089e5e83b1de0511, https://usegalaxy.eu/published/workflow?id=08eb5e55109c4d9c, https://usegalaxy.eu/published/workflow?id=096b75501c8e0888, https://usegalaxy.eu/published/workflow?id=0ab3e47ebd3b520e, https://usegalaxy.eu/published/workflow?id=0ae360bfbb3b2494, https://usegalaxy.eu/published/workflow?id=0be293debca8fb9b, https://usegalaxy.eu/published/workflow?id=0c619b25c1394a09, https://usegalaxy.eu/published/workflow?id=0e5b701ed5ca69a7, https://usegalaxy.eu/published/workflow?id=0eb137cd936bbfe0, https://usegalaxy.eu/published/workflow?id=0ec684b341ad172d, https://usegalaxy.eu/published/workflow?id=0f563f176bb245af, https://usegalaxy.eu/published/workflow?id=0f6a7bd5362e4a2b, https://usegalaxy.eu/published/workflow?id=0fa523529c4f73cf, https://usegalaxy.eu/published/workflow?id=0fb12e3a127fc4be, https://usegalaxy.eu/published/workflow?id=1089e5c66dbcc1a3, https://usegalaxy.eu/published/workflow?id=111527fe542846af, https://usegalaxy.eu/published/workflow?id=112422839b68db02, https://usegalaxy.eu/published/workflow?id=1151052f4393852a, https://usegalaxy.eu/published/workflow?id=117177b0c231dc55, https://usegalaxy.eu/published/workflow?id=133b299125ff9905, https://usegalaxy.eu/published/workflow?id=146cef3acb2eba75, https://usegalaxy.eu/published/workflow?id=1483534f49b826bf, https://usegalaxy.eu/published/workflow?id=14b9753269cd368f, https://usegalaxy.eu/published/workflow?id=1502fd3fa90d32d8, https://usegalaxy.eu/published/workflow?id=1524b189069b42a5, https://usegalaxy.eu/published/workflow?id=153ba1581743d31f, https://usegalaxy.eu/published/workflow?id=16080c3c1488a8b5, https://usegalaxy.eu/published/workflow?id=166eb3f14feffa1b, https://usegalaxy.eu/published/workflow?id=1761092d0e7deece, https://usegalaxy.eu/published/workflow?id=17ad465d9515cd25, https://usegalaxy.eu/published/workflow?id=17b0a6d3379a4392, https://usegalaxy.eu/published/workflow?id=1860e02bcb894a64, https://usegalaxy.eu/published/workflow?id=194a05d42687b524, https://usegalaxy.eu/published/workflow?id=1ac0a15758277730, https://usegalaxy.eu/published/workflow?id=1b49eda539658e53, https://usegalaxy.eu/published/workflow?id=1bb084f1bc102734, https://usegalaxy.eu/published/workflow?id=1c267e93aa8015e6, https://usegalaxy.eu/published/workflow?id=1c3e7b3af6ff79fa, https://usegalaxy.eu/published/workflow?id=1c44bbaa5f4f8b54, https://usegalaxy.eu/published/workflow?id=1c5588d6ac7f773f, https://usegalaxy.eu/published/workflow?id=1cba6eb4aff8dc61, https://usegalaxy.eu/published/workflow?id=1da7ac8eaf891f39, https://usegalaxy.eu/published/workflow?id=1dba8278fb2786fb, https://usegalaxy.eu/published/workflow?id=1e02a7292e9974b1, https://usegalaxy.eu/published/workflow?id=1e5f1d9a51032e44, https://usegalaxy.eu/published/workflow?id=1eac2ebffcbb8d59, https://usegalaxy.eu/published/workflow?id=1ee43d90b69829e9, https://usegalaxy.eu/published/workflow?id=1ef76b7b86e15792, https://usegalaxy.eu/published/workflow?id=1f2a1438e41188f2, https://usegalaxy.eu/published/workflow?id=1f874262c368fe7d, https://usegalaxy.eu/published/workflow?id=1f9a9a7998ef5ab3, https://usegalaxy.eu/published/workflow?id=1fe02012c96b731f, https://usegalaxy.eu/published/workflow?id=20ada53472782c9b, https://usegalaxy.eu/published/workflow?id=21a3f72851f29925, https://usegalaxy.eu/published/workflow?id=220961d803697f54, https://usegalaxy.eu/published/workflow?id=23821870734575e9, https://usegalaxy.eu/published/workflow?id=249c5d6698c8b98c, https://usegalaxy.eu/published/workflow?id=24ddd188da35b061, https://usegalaxy.eu/published/workflow?id=24eaa682897379c8, https://usegalaxy.eu/published/workflow?id=250e6058f160825f, https://usegalaxy.eu/published/workflow?id=2573209308aaad23, https://usegalaxy.eu/published/workflow?id=259f459a45e17e46, https://usegalaxy.eu/published/workflow?id=25aa944237d7fb82, https://usegalaxy.eu/published/workflow?id=26185e8fa5a9b988, https://usegalaxy.eu/published/workflow?id=2671a5f85b2cccfe, https://usegalaxy.eu/published/workflow?id=270b6f82efab28b2, https://usegalaxy.eu/published/workflow?id=29a855d928337f8f, https://usegalaxy.eu/published/workflow?id=2a123e08436a89f1, https://usegalaxy.eu/published/workflow?id=2a33d4fcb806e942, https://usegalaxy.eu/published/workflow?id=2a39dbc8fc6dd5d3, https://usegalaxy.eu/published/workflow?id=2aede5c6577b28f5, https://usegalaxy.eu/published/workflow?id=2b25eaebd3987612, https://usegalaxy.eu/published/workflow?id=2c3d05023c02113e, https://usegalaxy.eu/published/workflow?id=2c5cae1389c11881, https://usegalaxy.eu/published/workflow?id=2c79eee5ca5a3b09, https://usegalaxy.eu/published/workflow?id=2cf9f0e2ab5a7f8b, https://usegalaxy.eu/published/workflow?id=2d08a73dd8ff99e9, https://usegalaxy.eu/published/workflow?id=2d2d0c2231babc50, https://usegalaxy.eu/published/workflow?id=2d9d4445a1871230, https://usegalaxy.eu/published/workflow?id=2e0075791f7fa384, https://usegalaxy.eu/published/workflow?id=2e418c5810012e5a, https://usegalaxy.eu/published/workflow?id=2e6561d8dadc9e17, https://usegalaxy.eu/published/workflow?id=2f094f5e1f391529, https://usegalaxy.eu/published/workflow?id=2f51a82838722bec, https://usegalaxy.eu/published/workflow?id=2fd462d85412444c, https://usegalaxy.eu/published/workflow?id=300301c23327f797, https://usegalaxy.eu/published/workflow?id=309fc4fc2e0da89c, https://usegalaxy.eu/published/workflow?id=30e7dc81e323cbdb, https://usegalaxy.eu/published/workflow?id=3183c2abca463bb4, https://usegalaxy.eu/published/workflow?id=31d9eb6a120411d6, https://usegalaxy.eu/published/workflow?id=326d4102190b74fc, https://usegalaxy.eu/published/workflow?id=3403d5a8a71e0c3c, https://usegalaxy.eu/published/workflow?id=342a5df69951e3d7, https://usegalaxy.eu/published/workflow?id=344cd3a3c2ce5302, https://usegalaxy.eu/published/workflow?id=350d7e6976e15d30, https://usegalaxy.eu/published/workflow?id=3550384e7c062a60, https://usegalaxy.eu/published/workflow?id=374797e58377c40f, https://usegalaxy.eu/published/workflow?id=379642036b6f5a66, https://usegalaxy.eu/published/workflow?id=37da7b42cb063a3a, https://usegalaxy.eu/published/workflow?id=39b195a36408277c, https://usegalaxy.eu/published/workflow?id=3a3e2d50fe7a9ee3, https://usegalaxy.eu/published/workflow?id=3b8ef7905c902b27, https://usegalaxy.eu/published/workflow?id=3ce75b5de4922cba, https://usegalaxy.eu/published/workflow?id=3d0039098a87642c, https://usegalaxy.eu/published/workflow?id=3e255a5b7026d26e, https://usegalaxy.eu/published/workflow?id=3eb6663a66f8e69a, https://usegalaxy.eu/published/workflow?id=3f046b416b10b871, https://usegalaxy.eu/published/workflow?id=3f5414bb38b47fcf, https://usegalaxy.eu/published/workflow?id=3ff4dbcc7a602e6b, https://usegalaxy.eu/published/workflow?id=406e6ef0dfb38628, https://usegalaxy.eu/published/workflow?id=409336bf210033a2, https://usegalaxy.eu/published/workflow?id=40d2c7ac42bd5302, https://usegalaxy.eu/published/workflow?id=416141b245073406, https://usegalaxy.eu/published/workflow?id=4176b9d43325a8d8, https://usegalaxy.eu/published/workflow?id=429ae28a178cd07c, https://usegalaxy.eu/published/workflow?id=42c0da2e86318b37, https://usegalaxy.eu/published/workflow?id=44a2b6c8ae54eb31, https://usegalaxy.eu/published/workflow?id=44c5999436da5f1e, https://usegalaxy.eu/published/workflow?id=4566262dd0acc8ab, https://usegalaxy.eu/published/workflow?id=45ae834d842302c3, https://usegalaxy.eu/published/workflow?id=45c5e0a1e10edc54, https://usegalaxy.eu/published/workflow?id=45f63151bf4f1a49, https://usegalaxy.eu/published/workflow?id=461d71caea871db8, https://usegalaxy.eu/published/workflow?id=466bdd8ba7b67264, https://usegalaxy.eu/published/workflow?id=475ea16f943cc013, https://usegalaxy.eu/published/workflow?id=476db04368d5e90c, https://usegalaxy.eu/published/workflow?id=478190607a114390, https://usegalaxy.eu/published/workflow?id=47a0407867a90f95, https://usegalaxy.eu/published/workflow?id=48071c1720b2fd8d, https://usegalaxy.eu/published/workflow?id=4856e3522b2ee059, https://usegalaxy.eu/published/workflow?id=48f63ef14a12f306, https://usegalaxy.eu/published/workflow?id=49073a24429d93d6, https://usegalaxy.eu/published/workflow?id=497b82d3b92dc076, https://usegalaxy.eu/published/workflow?id=49ce9e738fadef4a, https://usegalaxy.eu/published/workflow?id=4a51e616d96338ed, https://usegalaxy.eu/published/workflow?id=4a764275d4bf9f31, https://usegalaxy.eu/published/workflow?id=4b4bfc79205d83f1, https://usegalaxy.eu/published/workflow?id=4c3a9b574f37eebb, https://usegalaxy.eu/published/workflow?id=4c75ab0be3c56494, https://usegalaxy.eu/published/workflow?id=4cb25c0bb103f54f, https://usegalaxy.eu/published/workflow?id=4d32b1b3ac0a7161, https://usegalaxy.eu/published/workflow?id=4d41ee83fc0040d4, https://usegalaxy.eu/published/workflow?id=4d97068a725c975c, https://usegalaxy.eu/published/workflow?id=4daa0c33d6d50379, https://usegalaxy.eu/published/workflow?id=4e2dc70a35ada981, https://usegalaxy.eu/published/workflow?id=4f245ac304ab76d9, https://usegalaxy.eu/published/workflow?id=4f62794453f65cba, https://usegalaxy.eu/published/workflow?id=503a2f4caacaf63e, https://usegalaxy.eu/published/workflow?id=510bcef37f385495, https://usegalaxy.eu/published/workflow?id=5115299e77df8104, https://usegalaxy.eu/published/workflow?id=5138874b208adb4b, https://usegalaxy.eu/published/workflow?id=519e0ffdfd1790bc, https://usegalaxy.eu/published/workflow?id=5279af5c95ae4f7a, https://usegalaxy.eu/published/workflow?id=529f30e894beacc2, https://usegalaxy.eu/published/workflow?id=52bdf400f97c5770, https://usegalaxy.eu/published/workflow?id=535f777e5e42e98c, https://usegalaxy.eu/published/workflow?id=5385abe9c2589b69, https://usegalaxy.eu/published/workflow?id=53de0e742573e588, https://usegalaxy.eu/published/workflow?id=54467870b7c635f2, https://usegalaxy.eu/published/workflow?id=54590ccd11ffbf65, https://usegalaxy.eu/published/workflow?id=54b91dd86f463391, https://usegalaxy.eu/published/workflow?id=55a9fc5d30392777, https://usegalaxy.eu/published/workflow?id=55ead91471bf1008, https://usegalaxy.eu/published/workflow?id=56523b41ff7eec76, https://usegalaxy.eu/published/workflow?id=56edd22d14392691, https://usegalaxy.eu/published/workflow?id=572091280033312e, https://usegalaxy.eu/published/workflow?id=57769c948e14f9ac, https://usegalaxy.eu/published/workflow?id=579dd1e95e76402f, https://usegalaxy.eu/published/workflow?id=579ff83177c5ee09, https://usegalaxy.eu/published/workflow?id=57cc33ed64d71982, https://usegalaxy.eu/published/workflow?id=57d3109f32defa3b, https://usegalaxy.eu/published/workflow?id=587e4d5ce25f48da, https://usegalaxy.eu/published/workflow?id=59b274da8df01ead, https://usegalaxy.eu/published/workflow?id=5aba0f20d994b1a4, https://usegalaxy.eu/published/workflow?id=5ad406442ec21345, https://usegalaxy.eu/published/workflow?id=5be158931abca947, https://usegalaxy.eu/published/workflow?id=5c15f251642bfa1f, https://usegalaxy.eu/published/workflow?id=5c8186f1e342c522, https://usegalaxy.eu/published/workflow?id=5cc732a6b4338bf5, https://usegalaxy.eu/published/workflow?id=5e1906290f5b4d8a, https://usegalaxy.eu/published/workflow?id=5ed38d8114bcd6f7, https://usegalaxy.eu/published/workflow?id=5f0faac42b0a5062, https://usegalaxy.eu/published/workflow?id=614807d57979554c, https://usegalaxy.eu/published/workflow?id=628062a8cf93261f, https://usegalaxy.eu/published/workflow?id=629e57958afed44c, https://usegalaxy.eu/published/workflow?id=63478edcea3f449a, https://usegalaxy.eu/published/workflow?id=6368019ffdf6c5ee, https://usegalaxy.eu/published/workflow?id=63749bd3045ccd4a, https://usegalaxy.eu/published/workflow?id=63964ba7d9edce74, https://usegalaxy.eu/published/workflow?id=63eed9c2e025a723, https://usegalaxy.eu/published/workflow?id=6514e610c79df72c, https://usegalaxy.eu/published/workflow?id=6573f3b1606b7f09, https://usegalaxy.eu/published/workflow?id=66f2a58637054c16, https://usegalaxy.eu/published/workflow?id=67be21795d403ca3, https://usegalaxy.eu/published/workflow?id=68e8042f11d5a3fd, https://usegalaxy.eu/published/workflow?id=69be516a48761365, https://usegalaxy.eu/published/workflow?id=69cdd404592535ec, https://usegalaxy.eu/published/workflow?id=6a5c39c1d38a10c2, https://usegalaxy.eu/published/workflow?id=6a6e2c2335051fb5, https://usegalaxy.eu/published/workflow?id=6bce1d419a179c50, https://usegalaxy.eu/published/workflow?id=6be946c7ec91835e, https://usegalaxy.eu/published/workflow?id=6dbac47941a804da, https://usegalaxy.eu/published/workflow?id=6dbc84168feea337, https://usegalaxy.eu/published/workflow?id=6e1fe7372e93fae0, https://usegalaxy.eu/published/workflow?id=6ebd09100fc98878, https://usegalaxy.eu/published/workflow?id=6ed1016dfdf9f928, https://usegalaxy.eu/published/workflow?id=70e93addf6d1b08e, https://usegalaxy.eu/published/workflow?id=7137e45ca8dc6e4c, https://usegalaxy.eu/published/workflow?id=71a6d128a09210f0, https://usegalaxy.eu/published/workflow?id=71c4ec4805f83cd5, https://usegalaxy.eu/published/workflow?id=72e3e5fdc766e24a, https://usegalaxy.eu/published/workflow?id=733650962d379da2, https://usegalaxy.eu/published/workflow?id=7337d15d121596da, https://usegalaxy.eu/published/workflow?id=7371b6918e895e0c, https://usegalaxy.eu/published/workflow?id=74e4a8f2edb25f1a, https://usegalaxy.eu/published/workflow?id=74f19aff1b9cdd6b, https://usegalaxy.eu/published/workflow?id=7582a48054c44d6e, https://usegalaxy.eu/published/workflow?id=75e88603745e069e, https://usegalaxy.eu/published/workflow?id=76e9cd837ea5e7da, https://usegalaxy.eu/published/workflow?id=77d2adf5249eb826, https://usegalaxy.eu/published/workflow?id=79b5afb125e3c3da, https://usegalaxy.eu/published/workflow?id=79fa095c9fce9923, https://usegalaxy.eu/published/workflow?id=79fd2517af48c594, https://usegalaxy.eu/published/workflow?id=7aa32aba3fde0708, https://usegalaxy.eu/published/workflow?id=7ae80d7e4d374b26, https://usegalaxy.eu/published/workflow?id=7b7ef222ac4f9887, https://usegalaxy.eu/published/workflow?id=7bdd9ad918edab0b, https://usegalaxy.eu/published/workflow?id=7c986f4d14c6a539, https://usegalaxy.eu/published/workflow?id=7cc253b53379b727, https://usegalaxy.eu/published/workflow?id=7d44ea091d437edf, https://usegalaxy.eu/published/workflow?id=7e0ba4c14803448e, https://usegalaxy.eu/published/workflow?id=7ea91d562a02eb2f, https://usegalaxy.eu/published/workflow?id=7ecd92cb382d8168, https://usegalaxy.eu/published/workflow?id=81122d30c64a52a6, https://usegalaxy.eu/published/workflow?id=8132c3b710fd6d48, https://usegalaxy.eu/published/workflow?id=82ef590b5821805d, https://usegalaxy.eu/published/workflow?id=83049910e2cd5128, https://usegalaxy.eu/published/workflow?id=830eb4faff21eb0c, https://usegalaxy.eu/published/workflow?id=847403b72dd346ff, https://usegalaxy.eu/published/workflow?id=86e2e2faa2e0bd46, https://usegalaxy.eu/published/workflow?id=873e6ac4699ed7f6, https://usegalaxy.eu/published/workflow?id=87cc0cd143a12c41, https://usegalaxy.eu/published/workflow?id=87fea062a9646a31, https://usegalaxy.eu/published/workflow?id=88064e2231165fd2, https://usegalaxy.eu/published/workflow?id=889f2782ad82bd69, https://usegalaxy.eu/published/workflow?id=895b8f5736bb92bb, https://usegalaxy.eu/published/workflow?id=8be366ad6a42408b, https://usegalaxy.eu/published/workflow?id=8c4331f1bac67aba, https://usegalaxy.eu/published/workflow?id=8f3ce55d44d9f369, https://usegalaxy.eu/published/workflow?id=8f415b0c409f56f7, https://usegalaxy.eu/published/workflow?id=8f9dc5ba61a02815, https://usegalaxy.eu/published/workflow?id=8fb156020285e049, https://usegalaxy.eu/published/workflow?id=8ff9d3ad20e87819, https://usegalaxy.eu/published/workflow?id=90a32c95b60f5221, https://usegalaxy.eu/published/workflow?id=91d4aabba7ad40c8, https://usegalaxy.eu/published/workflow?id=9207e74a5bf7d8e1, https://usegalaxy.eu/published/workflow?id=9219557de92e0203, https://usegalaxy.eu/published/workflow?id=92760fc19ef120ce, https://usegalaxy.eu/published/workflow?id=927d24dcde139b9c, https://usegalaxy.eu/published/workflow?id=92dafb3d1c06e074, https://usegalaxy.eu/published/workflow?id=93a7047396080fda, https://usegalaxy.eu/published/workflow?id=9472be79da012999, https://usegalaxy.eu/published/workflow?id=94b9287365ea4d0d, https://usegalaxy.eu/published/workflow?id=94bbf8fa7c1f030d, https://usegalaxy.eu/published/workflow?id=94c3d329c31c712f, https://usegalaxy.eu/published/workflow?id=95cb04ffb473623b, https://usegalaxy.eu/published/workflow?id=95f420d8856bf725, https://usegalaxy.eu/published/workflow?id=96c61a584cb2e5e9, https://usegalaxy.eu/published/workflow?id=985044b4b2c51425, https://usegalaxy.eu/published/workflow?id=9a1d17c12b139bda, https://usegalaxy.eu/published/workflow?id=9a60ecf98d4f8134, https://usegalaxy.eu/published/workflow?id=9ad5aff92e185907, https://usegalaxy.eu/published/workflow?id=9b025479fade745d, https://usegalaxy.eu/published/workflow?id=9b9e1e28f3683f92, https://usegalaxy.eu/published/workflow?id=9c0b67496dfe52b2, https://usegalaxy.eu/published/workflow?id=9c24df68dc7d25a4, https://usegalaxy.eu/published/workflow?id=9c7a218993788493, https://usegalaxy.eu/published/workflow?id=9da26c5131603b9f, https://usegalaxy.eu/published/workflow?id=9e31c51911987687, https://usegalaxy.eu/published/workflow?id=9e4df2893767591a, https://usegalaxy.eu/published/workflow?id=9eda857e4141d56e, https://usegalaxy.eu/published/workflow?id=9f2d18f0d0896e8b, https://usegalaxy.eu/published/workflow?id=9fddc44294236e8f, https://usegalaxy.eu/published/workflow?id=9feebfc6c34da287, https://usegalaxy.eu/published/workflow?id=a1e09c648d8f1fc7, https://usegalaxy.eu/published/workflow?id=a28c207730e7e13d, https://usegalaxy.eu/published/workflow?id=a31100610fba17a1, https://usegalaxy.eu/published/workflow?id=a35e0c2fcba4e95f, https://usegalaxy.eu/published/workflow?id=a3cb75fd31ba0689, https://usegalaxy.eu/published/workflow?id=a3da32b6b45da639, https://usegalaxy.eu/published/workflow?id=a461ea397418f3aa, https://usegalaxy.eu/published/workflow?id=a4e3667f6879bfaf, https://usegalaxy.eu/published/workflow?id=a639ea0419de1078, https://usegalaxy.eu/published/workflow?id=a6f4371ef9bc1aab, https://usegalaxy.eu/published/workflow?id=a705370bc2c13d5c, https://usegalaxy.eu/published/workflow?id=a76629eccfdeafb8, https://usegalaxy.eu/published/workflow?id=a8f829deb69b30ab, https://usegalaxy.eu/published/workflow?id=a9c1dca5da7170dd, https://usegalaxy.eu/published/workflow?id=a9e1a6b90dbd2518, https://usegalaxy.eu/published/workflow?id=aa6dc57f5c67fb4b, https://usegalaxy.eu/published/workflow?id=ab6e6ffaaebab45b, https://usegalaxy.eu/published/workflow?id=ac6eb99f70d35999, https://usegalaxy.eu/published/workflow?id=ada52ba09b462567, https://usegalaxy.eu/published/workflow?id=ae16643c04ad1216, https://usegalaxy.eu/published/workflow?id=ae74f60cd20d56bc, https://usegalaxy.eu/published/workflow?id=af30010868c97316, https://usegalaxy.eu/published/workflow?id=af472984b3f3d65f, https://usegalaxy.eu/published/workflow?id=aff44f1665a14e23, https://usegalaxy.eu/published/workflow?id=b02d90c32dc67730, https://usegalaxy.eu/published/workflow?id=b144287d56b3291b, https://usegalaxy.eu/published/workflow?id=b1d1694b95ed10e1, https://usegalaxy.eu/published/workflow?id=b30c294a094c90bc, https://usegalaxy.eu/published/workflow?id=b323867a9efcc6c6, https://usegalaxy.eu/published/workflow?id=b426e137396acb14, https://usegalaxy.eu/published/workflow?id=b4948afacccfefdb, https://usegalaxy.eu/published/workflow?id=b57c3522bdd7c2d5, https://usegalaxy.eu/published/workflow?id=b5e3360f933d1950, https://usegalaxy.eu/published/workflow?id=b5ffba506152b1a1, https://usegalaxy.eu/published/workflow?id=b619339f4e1c6fdf, https://usegalaxy.eu/published/workflow?id=b77e3b08092fa73e, https://usegalaxy.eu/published/workflow?id=b7a3cbcd32734e8b, https://usegalaxy.eu/published/workflow?id=b7c2440621968526, https://usegalaxy.eu/published/workflow?id=b84fd6a0e2632ad9, https://usegalaxy.eu/published/workflow?id=b8baade87e733fce, https://usegalaxy.eu/published/workflow?id=b97532c398b821a2, https://usegalaxy.eu/published/workflow?id=b9ad204395f7d931, https://usegalaxy.eu/published/workflow?id=ba01694e45595b6b, https://usegalaxy.eu/published/workflow?id=bace9aebab9d4ce3, https://usegalaxy.eu/published/workflow?id=bb604ff5e010e7ab, https://usegalaxy.eu/published/workflow?id=bc87309a8763d0b4, https://usegalaxy.eu/published/workflow?id=bcf99cc0469db276, https://usegalaxy.eu/published/workflow?id=be3d1d7d4afd96b1, https://usegalaxy.eu/published/workflow?id=bef0a7d604f2ebef, https://usegalaxy.eu/published/workflow?id=c084c2009f6b2d0a, https://usegalaxy.eu/published/workflow?id=c0d1e965daf89006, https://usegalaxy.eu/published/workflow?id=c2af7a5dbfb5b023, https://usegalaxy.eu/published/workflow?id=c2d814f2182c0082, https://usegalaxy.eu/published/workflow?id=c2f0decb06f0bb90, https://usegalaxy.eu/published/workflow?id=c38e3ac59d8b86d6, https://usegalaxy.eu/published/workflow?id=c3eba26c5d7febff, https://usegalaxy.eu/published/workflow?id=c4c2e669b86cffbc, https://usegalaxy.eu/published/workflow?id=c4fa05e209a2f082, https://usegalaxy.eu/published/workflow?id=c508c00e81018dd8, https://usegalaxy.eu/published/workflow?id=c5e6047b3b256b06, https://usegalaxy.eu/published/workflow?id=c6b22fc33b569c8f, https://usegalaxy.eu/published/workflow?id=c6ca9e6fd28979f8, https://usegalaxy.eu/published/workflow?id=c7a8f7aabc219831, https://usegalaxy.eu/published/workflow?id=c7f34c67c9054234, https://usegalaxy.eu/published/workflow?id=c98a14fcf5998159, https://usegalaxy.eu/published/workflow?id=c9aff55e39e195e3, https://usegalaxy.eu/published/workflow?id=ca9d3233b0912765, https://usegalaxy.eu/published/workflow?id=caba6d3d7db648e9, https://usegalaxy.eu/published/workflow?id=cae527ce845e770e, https://usegalaxy.eu/published/workflow?id=cc37667c2d1f9056, https://usegalaxy.eu/published/workflow?id=ccef778cfd9024d1, https://usegalaxy.eu/published/workflow?id=cd926dd6b70e72d6, https://usegalaxy.eu/published/workflow?id=cf4638fbca7c460b, https://usegalaxy.eu/published/workflow?id=cfe47f0c78ed5ec4, https://usegalaxy.eu/published/workflow?id=d05c6f0f3205fbd2, https://usegalaxy.eu/published/workflow?id=d086340422009ba9, https://usegalaxy.eu/published/workflow?id=d12617d0df5f81d0, https://usegalaxy.eu/published/workflow?id=d1338e2b4002a364, https://usegalaxy.eu/published/workflow?id=d17839eaeac6c4c8, https://usegalaxy.eu/published/workflow?id=d3494c6124f820b2, https://usegalaxy.eu/published/workflow?id=d3dfe7ba11508c72, https://usegalaxy.eu/published/workflow?id=d3edc3f0659a5e5b, https://usegalaxy.eu/published/workflow?id=d649ec485c4b3994, https://usegalaxy.eu/published/workflow?id=d6809603f46de1b3, https://usegalaxy.eu/published/workflow?id=d6ff6be33b8bfe85, https://usegalaxy.eu/published/workflow?id=d768c0e0df4d432e, https://usegalaxy.eu/published/workflow?id=d83280cd72ec0f33, https://usegalaxy.eu/published/workflow?id=d88145e7475e0b7b, https://usegalaxy.eu/published/workflow?id=d967194ca51ff11e, https://usegalaxy.eu/published/workflow?id=d9f8c5d575da02b1, https://usegalaxy.eu/published/workflow?id=da19c0eaa757fa39, https://usegalaxy.eu/published/workflow?id=db03a18167f26399, https://usegalaxy.eu/published/workflow?id=db10c092fad70d69, https://usegalaxy.eu/published/workflow?id=dbcf7af09e036754, https://usegalaxy.eu/published/workflow?id=dc27453c438577f6, https://usegalaxy.eu/published/workflow?id=dd3193aaaf08e798, https://usegalaxy.eu/published/workflow?id=ddd604341afd9cf7, https://usegalaxy.eu/published/workflow?id=de3eafbcf42c75bb, https://usegalaxy.eu/published/workflow?id=de9003e589631029, https://usegalaxy.eu/published/workflow?id=de9cf573314f416c, https://usegalaxy.eu/published/workflow?id=deec04097a871646, https://usegalaxy.eu/published/workflow?id=dfc2c53ac3103eff, https://usegalaxy.eu/published/workflow?id=dfe3b555856c7b44, https://usegalaxy.eu/published/workflow?id=e138640778b7414c, https://usegalaxy.eu/published/workflow?id=e16e524c557d7928, https://usegalaxy.eu/published/workflow?id=e1947b059ef131d2, https://usegalaxy.eu/published/workflow?id=e2f936f1aa43a629, https://usegalaxy.eu/published/workflow?id=e6c5fb1d37d29ec0, https://usegalaxy.eu/published/workflow?id=e7363b9f450c56b4, https://usegalaxy.eu/published/workflow?id=e7a9e1a7728848cd, https://usegalaxy.eu/published/workflow?id=e7aebe58411d97ed, https://usegalaxy.eu/published/workflow?id=e85c9385870aadcc, https://usegalaxy.eu/published/workflow?id=e8f9e1d911683b22, https://usegalaxy.eu/published/workflow?id=e935d2c8ffb52588, https://usegalaxy.eu/published/workflow?id=ea8bb8eefdb4ba7a, https://usegalaxy.eu/published/workflow?id=ead0f264798f9418, https://usegalaxy.eu/published/workflow?id=eaf01ccb13aad3ec, https://usegalaxy.eu/published/workflow?id=eb3cfa4d1f05fc0e, https://usegalaxy.eu/published/workflow?id=ebbf5cfa946d11d3, https://usegalaxy.eu/published/workflow?id=ec79f14ab91cd933, https://usegalaxy.eu/published/workflow?id=ecf48fa50af560a0, https://usegalaxy.eu/published/workflow?id=ed20bd120064bab0, https://usegalaxy.eu/published/workflow?id=ee6dc47660540ba6, https://usegalaxy.eu/published/workflow?id=ef1d11b79a2e1727, https://usegalaxy.eu/published/workflow?id=f14ebad1c3c20067, https://usegalaxy.eu/published/workflow?id=f1ca4289ee4f67bb, https://usegalaxy.eu/published/workflow?id=f1e2577699125607, https://usegalaxy.eu/published/workflow?id=f33b5135ee99e0dc, https://usegalaxy.eu/published/workflow?id=f38745208494d4ab, https://usegalaxy.eu/published/workflow?id=f5005e80d9ffa109, https://usegalaxy.eu/published/workflow?id=f54b4d76f7f9a589, https://usegalaxy.eu/published/workflow?id=f578a097b4339025, https://usegalaxy.eu/published/workflow?id=f63127675a07e00a, https://usegalaxy.eu/published/workflow?id=f6a625946dea60fa, https://usegalaxy.eu/published/workflow?id=f6b55599db1ab056, https://usegalaxy.eu/published/workflow?id=f6e8a63283262518, https://usegalaxy.eu/published/workflow?id=f841e1207611f28f, https://usegalaxy.eu/published/workflow?id=f97e6fe1fcbc53a7, https://usegalaxy.eu/published/workflow?id=f9b705e53b6d6539, https://usegalaxy.eu/published/workflow?id=fa0ff9a0db8bbddb, https://usegalaxy.eu/published/workflow?id=fa53b172def3fe75, https://usegalaxy.eu/published/workflow?id=fa960b78dfc0c2f8, https://usegalaxy.eu/published/workflow?id=fb015e398373ea8a, https://usegalaxy.eu/published/workflow?id=fc216dfb0ec9ed41, https://usegalaxy.eu/published/workflow?id=fc25c1f8c6667c87, https://usegalaxy.eu/published/workflow?id=fc956fb42ef2852e, https://usegalaxy.eu/published/workflow?id=fcf745185d6e0ea4, https://usegalaxy.eu/published/workflow?id=fd61d070d9744f95, https://usegalaxy.eu/published/workflow?id=fd6429a788660c8d, https://usegalaxy.eu/published/workflow?id=fd649ac4d4872c81, https://usegalaxy.eu/published/workflow?id=fdca6c9d394986bf, https://usegalaxy.eu/published/workflow?id=fde21b5f5c169114, https://usegalaxy.eu/published/workflow?id=ff2fd4a4bb27d859, https://usegalaxy.fr/published/workflow?id=2345fad7dac13daa, https://usegalaxy.fr/published/workflow?id=54b48eeeed3564dd, https://usegalaxy.fr/published/workflow?id=57b5a45c9f71a09c, https://usegalaxy.fr/published/workflow?id=591c57173e259ca6, https://usegalaxy.fr/published/workflow?id=5d83e0886f2a7465, https://usegalaxy.fr/published/workflow?id=61235264d5d2f666, https://usegalaxy.fr/published/workflow?id=727cfdb7c79287d1, https://usegalaxy.fr/published/workflow?id=776ef2fa51094ba6, https://usegalaxy.fr/published/workflow?id=7d27dd394b921846, https://usegalaxy.fr/published/workflow?id=826cffb2458e4067, https://usegalaxy.fr/published/workflow?id=98529cff0ef36b91, https://usegalaxy.fr/published/workflow?id=aa96487c5a1b29bf, https://usegalaxy.fr/published/workflow?id=bffee40c87c7b20a, https://usegalaxy.fr/published/workflow?id=c4f6062e51d0f45e, https://usegalaxy.fr/published/workflow?id=ca0bd0e42cb774f9, https://usegalaxy.fr/published/workflow?id=d0d12c0b80c286f9, https://usegalaxy.fr/published/workflow?id=e0b225d4b2556365, https://usegalaxy.fr/published/workflow?id=e3c04a30258421aa, https://usegalaxy.fr/published/workflow?id=e6991d64e851b09e, https://usegalaxy.fr/published/workflow?id=f0776f7a890b523a, https://usegalaxy.org.au/published/workflow?id=000cdc969e9f0886, https://usegalaxy.org.au/published/workflow?id=00da6615bca12a3c, https://usegalaxy.org.au/published/workflow?id=01997cb92a2ce2d5, https://usegalaxy.org.au/published/workflow?id=0220e60c66de9065, https://usegalaxy.org.au/published/workflow?id=02b8d033d9b1fa05, https://usegalaxy.org.au/published/workflow?id=02eebe3d909206ab, https://usegalaxy.org.au/published/workflow?id=0458ecd2761307d1, https://usegalaxy.org.au/published/workflow?id=046a1c600a6b77c2, https://usegalaxy.org.au/published/workflow?id=05950987aa2eba5b, https://usegalaxy.org.au/published/workflow?id=05b17b9d70c1db8b, https://usegalaxy.org.au/published/workflow?id=05f3abecd4ddd4a1, https://usegalaxy.org.au/published/workflow?id=0614f24ebeb680de, https://usegalaxy.org.au/published/workflow?id=068f85b8b137bfd7, https://usegalaxy.org.au/published/workflow?id=06ba34da322f46f1, https://usegalaxy.org.au/published/workflow?id=07c97b9e59c0218f, https://usegalaxy.org.au/published/workflow?id=08fac3b4c128e597, https://usegalaxy.org.au/published/workflow?id=094a545d44ce28cc, https://usegalaxy.org.au/published/workflow?id=0a477899db53d098, https://usegalaxy.org.au/published/workflow?id=0af0ea50f1237952, https://usegalaxy.org.au/published/workflow?id=0c567180448d8774, https://usegalaxy.org.au/published/workflow?id=0c6f7bfc826433c4, https://usegalaxy.org.au/published/workflow?id=0cfc1408fd9ea4e7, https://usegalaxy.org.au/published/workflow?id=0d1db28665d19bda, https://usegalaxy.org.au/published/workflow?id=0f7406ada6706c91, https://usegalaxy.org.au/published/workflow?id=10355b2680374a03, https://usegalaxy.org.au/published/workflow?id=114fb9589bb9db92, https://usegalaxy.org.au/published/workflow?id=115151c2c8998d44, https://usegalaxy.org.au/published/workflow?id=13ce18408ee7acc9, https://usegalaxy.org.au/published/workflow?id=14184bc2580a131f, https://usegalaxy.org.au/published/workflow?id=14b17f17d03c8424, https://usegalaxy.org.au/published/workflow?id=158b5d9107a05d95, https://usegalaxy.org.au/published/workflow?id=16f4846b4d6bc155, https://usegalaxy.org.au/published/workflow?id=17303f96fb40897c, https://usegalaxy.org.au/published/workflow?id=178db14511c8f293, https://usegalaxy.org.au/published/workflow?id=18d48f1b26811372, https://usegalaxy.org.au/published/workflow?id=19315e6573c58239, https://usegalaxy.org.au/published/workflow?id=1a107e6611fb9c15, https://usegalaxy.org.au/published/workflow?id=1ab66d382630afa7, https://usegalaxy.org.au/published/workflow?id=1ad81f95ace878ea, https://usegalaxy.org.au/published/workflow?id=1c886f008a36b0b8, https://usegalaxy.org.au/published/workflow?id=1dd4392c51766383, https://usegalaxy.org.au/published/workflow?id=1e033cad62e1bd1e, https://usegalaxy.org.au/published/workflow?id=1eb3e1452c672480, https://usegalaxy.org.au/published/workflow?id=1f63ec596e578b1c, https://usegalaxy.org.au/published/workflow?id=1ff259845498b818, https://usegalaxy.org.au/published/workflow?id=203f9e69152fbb2a, https://usegalaxy.org.au/published/workflow?id=20e5e6b233ffb951, https://usegalaxy.org.au/published/workflow?id=210ca013cc1cf185, https://usegalaxy.org.au/published/workflow?id=2470cb61ba51288c, https://usegalaxy.org.au/published/workflow?id=25403321a649a92b, https://usegalaxy.org.au/published/workflow?id=2543e425fa6d5144, https://usegalaxy.org.au/published/workflow?id=25ac9b4f2e39af97, https://usegalaxy.org.au/published/workflow?id=25d52afddaa3451b, https://usegalaxy.org.au/published/workflow?id=2617fee24459e680, https://usegalaxy.org.au/published/workflow?id=27205feb28b0ce06, https://usegalaxy.org.au/published/workflow?id=2789fb3590727342, https://usegalaxy.org.au/published/workflow?id=281ea78467e73147, https://usegalaxy.org.au/published/workflow?id=2980172ed0226b59, https://usegalaxy.org.au/published/workflow?id=2999a1891ffe201b, https://usegalaxy.org.au/published/workflow?id=2a0046582f5f15a1, https://usegalaxy.org.au/published/workflow?id=2afeae1ec03c0b57, https://usegalaxy.org.au/published/workflow?id=2b2ce7e1206d177d, https://usegalaxy.org.au/published/workflow?id=2c58509f80897972, https://usegalaxy.org.au/published/workflow?id=2cef9a1633e0a067, https://usegalaxy.org.au/published/workflow?id=2d45d7138d8715b4, https://usegalaxy.org.au/published/workflow?id=2f520cb446adf726, https://usegalaxy.org.au/published/workflow?id=2f7c0e3221cbca2f, https://usegalaxy.org.au/published/workflow?id=301fdb29c513ffbe, https://usegalaxy.org.au/published/workflow?id=30430b75d05f54eb, https://usegalaxy.org.au/published/workflow?id=30a601039d24e2eb, https://usegalaxy.org.au/published/workflow?id=30b2cd025e0300da, https://usegalaxy.org.au/published/workflow?id=31449596b22f81f4, https://usegalaxy.org.au/published/workflow?id=319e9d693fac8dce, https://usegalaxy.org.au/published/workflow?id=31dbd313e5c8160b, https://usegalaxy.org.au/published/workflow?id=3223ceaa2b58d940, https://usegalaxy.org.au/published/workflow?id=32d28f800adcea42, https://usegalaxy.org.au/published/workflow?id=35384818ee1cbac2, https://usegalaxy.org.au/published/workflow?id=3604307ec2835628, https://usegalaxy.org.au/published/workflow?id=36f54eda8568355d, https://usegalaxy.org.au/published/workflow?id=371c9892d261cee2, https://usegalaxy.org.au/published/workflow?id=374311292d72c9f9, https://usegalaxy.org.au/published/workflow?id=383ce49fa7a367e9, https://usegalaxy.org.au/published/workflow?id=39b464084d930a33, https://usegalaxy.org.au/published/workflow?id=39d56981a6854d82, https://usegalaxy.org.au/published/workflow?id=3a64c1031340d860, https://usegalaxy.org.au/published/workflow?id=3ac73a2f00423326, https://usegalaxy.org.au/published/workflow?id=3bc45aaed5da1058, https://usegalaxy.org.au/published/workflow?id=3c1b1f210c7c292b, https://usegalaxy.org.au/published/workflow?id=3c30c822058187be, https://usegalaxy.org.au/published/workflow?id=3c657dd766d02f6a, https://usegalaxy.org.au/published/workflow?id=3c727d5cd349a846, https://usegalaxy.org.au/published/workflow?id=3c7b55daae7c119b, https://usegalaxy.org.au/published/workflow?id=3ce9dc804780e4a1, https://usegalaxy.org.au/published/workflow?id=3df7f45b7b388f8c, https://usegalaxy.org.au/published/workflow?id=3e7fcfd23872ad0d, https://usegalaxy.org.au/published/workflow?id=3ef07022656067b1, https://usegalaxy.org.au/published/workflow?id=3f39cf407c185724, https://usegalaxy.org.au/published/workflow?id=3fd10af90a07f771, https://usegalaxy.org.au/published/workflow?id=40666ceb68d101ac, https://usegalaxy.org.au/published/workflow?id=408a31694e9c85e6, https://usegalaxy.org.au/published/workflow?id=410d0d19b6589c78, https://usegalaxy.org.au/published/workflow?id=41795303a88d9e94, https://usegalaxy.org.au/published/workflow?id=41b115a9be4b6002, https://usegalaxy.org.au/published/workflow?id=41bd0340dcde60ec, https://usegalaxy.org.au/published/workflow?id=41e005d4ebb5885a, https://usegalaxy.org.au/published/workflow?id=421adea2f259ec26, https://usegalaxy.org.au/published/workflow?id=4267bd1e4cfd1901, https://usegalaxy.org.au/published/workflow?id=43343fa1896f8278, https://usegalaxy.org.au/published/workflow?id=43f2dabbc1932e30, https://usegalaxy.org.au/published/workflow?id=447d8daefdefafee, https://usegalaxy.org.au/published/workflow?id=44bad90b676d58f6, https://usegalaxy.org.au/published/workflow?id=45c8789533c45abf, https://usegalaxy.org.au/published/workflow?id=466809eb3392f7fb, https://usegalaxy.org.au/published/workflow?id=47cacc06d404a8c6, https://usegalaxy.org.au/published/workflow?id=47cd0a90609fd557, https://usegalaxy.org.au/published/workflow?id=47ce6a2d30887637, https://usegalaxy.org.au/published/workflow?id=48a25f8e58d068e6, https://usegalaxy.org.au/published/workflow?id=48a60965b6b3ff54, https://usegalaxy.org.au/published/workflow?id=49b5c35b9723fa62, https://usegalaxy.org.au/published/workflow?id=4b5f80656fc94b88, https://usegalaxy.org.au/published/workflow?id=4bc8ac41ecff7daf, https://usegalaxy.org.au/published/workflow?id=4bd6c5c12fb1a69a, https://usegalaxy.org.au/published/workflow?id=4c7e0703aace6aa3, https://usegalaxy.org.au/published/workflow?id=4e096a68187ce652, https://usegalaxy.org.au/published/workflow?id=4efed79608f85604, https://usegalaxy.org.au/published/workflow?id=4f2818b6620037af, https://usegalaxy.org.au/published/workflow?id=4f562265cbb14cd6, https://usegalaxy.org.au/published/workflow?id=51b745f8c964e5c6, https://usegalaxy.org.au/published/workflow?id=51bad198f1fd1215, https://usegalaxy.org.au/published/workflow?id=52056336f24eccf6, https://usegalaxy.org.au/published/workflow?id=52065a4100b11184, https://usegalaxy.org.au/published/workflow?id=525612313de7591e, https://usegalaxy.org.au/published/workflow?id=5314a7a4732a0ef1, https://usegalaxy.org.au/published/workflow?id=533dbb3242ee4255, https://usegalaxy.org.au/published/workflow?id=5442f0437ade82c4, https://usegalaxy.org.au/published/workflow?id=544cb12a16f283b8, https://usegalaxy.org.au/published/workflow?id=5550585ffc3d8d0e, https://usegalaxy.org.au/published/workflow?id=557601292ea0f6f3, https://usegalaxy.org.au/published/workflow?id=56599572e0364e2a, https://usegalaxy.org.au/published/workflow?id=5693811cd933d2a2, https://usegalaxy.org.au/published/workflow?id=56bda8e4b0018e03, https://usegalaxy.org.au/published/workflow?id=57b558e1a76c09e1, https://usegalaxy.org.au/published/workflow?id=5a4e49ef8693d7c4, https://usegalaxy.org.au/published/workflow?id=5b0c0e34394afed1, https://usegalaxy.org.au/published/workflow?id=5be983b903469f07, https://usegalaxy.org.au/published/workflow?id=5cbee52cf03b6f9a, https://usegalaxy.org.au/published/workflow?id=5cd9b9338e75f254, https://usegalaxy.org.au/published/workflow?id=5ce00d3eab862f4e, https://usegalaxy.org.au/published/workflow?id=5d6773ed5b2f41cb, https://usegalaxy.org.au/published/workflow?id=5ea74382fb77eb10, https://usegalaxy.org.au/published/workflow?id=5f4f874e49515555, https://usegalaxy.org.au/published/workflow?id=608f3ad8f485e7a8, https://usegalaxy.org.au/published/workflow?id=60c0dec52cdca708, https://usegalaxy.org.au/published/workflow?id=6204bbcfe56931eb, https://usegalaxy.org.au/published/workflow?id=625f3c52832b0591, https://usegalaxy.org.au/published/workflow?id=63789b6f82172679, https://usegalaxy.org.au/published/workflow?id=6448b82959076a29, https://usegalaxy.org.au/published/workflow?id=644d8e4c78df8e36, https://usegalaxy.org.au/published/workflow?id=6469ade8f6f7c718, https://usegalaxy.org.au/published/workflow?id=67191ea9fe977e08, https://usegalaxy.org.au/published/workflow?id=677ef3322366c953, https://usegalaxy.org.au/published/workflow?id=68b2961117e62890, https://usegalaxy.org.au/published/workflow?id=69233fb6edfcdde9, https://usegalaxy.org.au/published/workflow?id=69d91340fc7effa2, https://usegalaxy.org.au/published/workflow?id=6a04c54236f30e8d, https://usegalaxy.org.au/published/workflow?id=6b1b0a945d367b4c, https://usegalaxy.org.au/published/workflow?id=6b2716309947836d, https://usegalaxy.org.au/published/workflow?id=6b34c95a247d15e3, https://usegalaxy.org.au/published/workflow?id=6b3df543536faf8c, https://usegalaxy.org.au/published/workflow?id=6c27019d03dd49d8, https://usegalaxy.org.au/published/workflow?id=6c9129acb9d98afe, https://usegalaxy.org.au/published/workflow?id=6cdbcd1f667d0f16, https://usegalaxy.org.au/published/workflow?id=6e1e24f9e0326930, https://usegalaxy.org.au/published/workflow?id=6ec0031406b10635, https://usegalaxy.org.au/published/workflow?id=6efcacf14bfc18b3, https://usegalaxy.org.au/published/workflow?id=6f0801fecbba5d3f, https://usegalaxy.org.au/published/workflow?id=71bad95f72c9f30d, https://usegalaxy.org.au/published/workflow?id=723513b09d6690b8, https://usegalaxy.org.au/published/workflow?id=738350fc2ca69c0b, https://usegalaxy.org.au/published/workflow?id=73ff1aa9acc0ad2a, https://usegalaxy.org.au/published/workflow?id=74186b006665daf8, https://usegalaxy.org.au/published/workflow?id=749d8f2d26e0a111, https://usegalaxy.org.au/published/workflow?id=75113a1b41904d2a, https://usegalaxy.org.au/published/workflow?id=7563b3ef697a69fd, https://usegalaxy.org.au/published/workflow?id=76323f103844f75b, https://usegalaxy.org.au/published/workflow?id=76bb6bde19975e05, https://usegalaxy.org.au/published/workflow?id=79522a1f7b0c53cb, https://usegalaxy.org.au/published/workflow?id=7a4ee63a04362967, https://usegalaxy.org.au/published/workflow?id=7ad277dd448e57b3, https://usegalaxy.org.au/published/workflow?id=7ad9410e36ee9ea1, https://usegalaxy.org.au/published/workflow?id=7b203d52f496e2c7, https://usegalaxy.org.au/published/workflow?id=7ba13e928de65e61, https://usegalaxy.org.au/published/workflow?id=7becafd394472dfb, https://usegalaxy.org.au/published/workflow?id=7c22d726e1716f4b, https://usegalaxy.org.au/published/workflow?id=7c2e7b292803b3e9, https://usegalaxy.org.au/published/workflow?id=7c866ed4a3552d0d, https://usegalaxy.org.au/published/workflow?id=7d268e7209a68e2e, https://usegalaxy.org.au/published/workflow?id=7ea616c652d08fb6, https://usegalaxy.org.au/published/workflow?id=7ef245f5ae3b916d, https://usegalaxy.org.au/published/workflow?id=7fa13ccbf044edc5, https://usegalaxy.org.au/published/workflow?id=7fd58f5fc93f414e, https://usegalaxy.org.au/published/workflow?id=7febc6ac6f106e57, https://usegalaxy.org.au/published/workflow?id=810242a3dc4f6170, https://usegalaxy.org.au/published/workflow?id=83680c1c6dffb05b, https://usegalaxy.org.au/published/workflow?id=83a535840b388980, https://usegalaxy.org.au/published/workflow?id=83c20d8e5fd97313, https://usegalaxy.org.au/published/workflow?id=83e3ceceb1a92493, https://usegalaxy.org.au/published/workflow?id=8433a2f79b2e5894, https://usegalaxy.org.au/published/workflow?id=8483ae422665a6c1, https://usegalaxy.org.au/published/workflow?id=8483e8d3e8d0ddc9, https://usegalaxy.org.au/published/workflow?id=84c98a23187d2a36, https://usegalaxy.org.au/published/workflow?id=8597166abda0a375, https://usegalaxy.org.au/published/workflow?id=85aede91db0122e4, https://usegalaxy.org.au/published/workflow?id=863eee1b041b6c58, https://usegalaxy.org.au/published/workflow?id=874492eb776ba813, https://usegalaxy.org.au/published/workflow?id=879d4fb110a61f7a, https://usegalaxy.org.au/published/workflow?id=888eb87b62c4e6dd, https://usegalaxy.org.au/published/workflow?id=888f8377eb6eabf2, https://usegalaxy.org.au/published/workflow?id=88c0d875eea2ee49, https://usegalaxy.org.au/published/workflow?id=8926e592f2e60863, https://usegalaxy.org.au/published/workflow?id=897dfa78c3b25356, https://usegalaxy.org.au/published/workflow?id=897e9d7432d231e9, https://usegalaxy.org.au/published/workflow?id=8a6233015db6dd24, https://usegalaxy.org.au/published/workflow?id=8b5897305dcaf076, https://usegalaxy.org.au/published/workflow?id=8b6739a000780d99, https://usegalaxy.org.au/published/workflow?id=8b8a5b50f2edc72c, https://usegalaxy.org.au/published/workflow?id=8b955274ca3ca3dc, https://usegalaxy.org.au/published/workflow?id=8bf8c9929249f37d, https://usegalaxy.org.au/published/workflow?id=8c40b4a911517cc5, https://usegalaxy.org.au/published/workflow?id=8cab31db0bd3ff52, https://usegalaxy.org.au/published/workflow?id=8d196c8c62530dc0, https://usegalaxy.org.au/published/workflow?id=8d47811c6154d347, https://usegalaxy.org.au/published/workflow?id=8d6010d07cc2e170, https://usegalaxy.org.au/published/workflow?id=8d6d4f848067f3d9, https://usegalaxy.org.au/published/workflow?id=8dd86497bc15d981, https://usegalaxy.org.au/published/workflow?id=8e8a5cce9f3f2f67, https://usegalaxy.org.au/published/workflow?id=8fe84de499e96485, https://usegalaxy.org.au/published/workflow?id=9157a7438977b9e9, https://usegalaxy.org.au/published/workflow?id=91cfd96f697ba034, https://usegalaxy.org.au/published/workflow?id=91ee716be8bfd508, https://usegalaxy.org.au/published/workflow?id=925c167abd5610ae, https://usegalaxy.org.au/published/workflow?id=932bad88f8dc8d33, https://usegalaxy.org.au/published/workflow?id=954318db8c8d9219, https://usegalaxy.org.au/published/workflow?id=95a9070469a895c3, https://usegalaxy.org.au/published/workflow?id=95efb414f5984735, https://usegalaxy.org.au/published/workflow?id=9642aadb83c4f752, https://usegalaxy.org.au/published/workflow?id=970dc0d8bd74707f, https://usegalaxy.org.au/published/workflow?id=975ddb0eb436f1fd, https://usegalaxy.org.au/published/workflow?id=97628b0604d5ec77, https://usegalaxy.org.au/published/workflow?id=9787c8b7adeedb0c, https://usegalaxy.org.au/published/workflow?id=978953f492b2b914, https://usegalaxy.org.au/published/workflow?id=9804030358c3f844, https://usegalaxy.org.au/published/workflow?id=981d7b02416b7734, https://usegalaxy.org.au/published/workflow?id=9898348d95033af2, https://usegalaxy.org.au/published/workflow?id=98f660971010d11d, https://usegalaxy.org.au/published/workflow?id=9988bc2402f32dc1, https://usegalaxy.org.au/published/workflow?id=9b18123afbe1264b, https://usegalaxy.org.au/published/workflow?id=9b1afa59c087eacd, https://usegalaxy.org.au/published/workflow?id=9b75924c483b5f6b, https://usegalaxy.org.au/published/workflow?id=9bc31dff5bcec42b, https://usegalaxy.org.au/published/workflow?id=9c60b83feaa22415, https://usegalaxy.org.au/published/workflow?id=9df67170c9260539, https://usegalaxy.org.au/published/workflow?id=9e0bae0505ee7c07, https://usegalaxy.org.au/published/workflow?id=9e8d9b7055b7c863, https://usegalaxy.org.au/published/workflow?id=9ec9fe2715d7cfdc, https://usegalaxy.org.au/published/workflow?id=a02c102e179596cb, https://usegalaxy.org.au/published/workflow?id=a1c59e9f5c7327d2, https://usegalaxy.org.au/published/workflow?id=a29670ff0ecb1a38, https://usegalaxy.org.au/published/workflow?id=a345bf84eb9bf6b2, https://usegalaxy.org.au/published/workflow?id=a484cdc9ff9d7e9e, https://usegalaxy.org.au/published/workflow?id=a525966251895f8a, https://usegalaxy.org.au/published/workflow?id=a5ccbb9aced9ac53, https://usegalaxy.org.au/published/workflow?id=a61d4bda54de7cd1, https://usegalaxy.org.au/published/workflow?id=a69db7f914c06407, https://usegalaxy.org.au/published/workflow?id=a7346c25dfdf2e3e, https://usegalaxy.org.au/published/workflow?id=a763e53653b10ce0, https://usegalaxy.org.au/published/workflow?id=a7726d18d2a5fcf2, https://usegalaxy.org.au/published/workflow?id=a7a30d970584f9ac, https://usegalaxy.org.au/published/workflow?id=a7d1cf2bb636e695, https://usegalaxy.org.au/published/workflow?id=a820d57cc8300253, https://usegalaxy.org.au/published/workflow?id=a8ec08a1408cd6e7, https://usegalaxy.org.au/published/workflow?id=a9fa6101f1fb13e2, https://usegalaxy.org.au/published/workflow?id=aa1d123c0a0929f3, https://usegalaxy.org.au/published/workflow?id=aa94c77b720a2810, https://usegalaxy.org.au/published/workflow?id=aaaf580eb00459b6, https://usegalaxy.org.au/published/workflow?id=aae07f802002fdd0, https://usegalaxy.org.au/published/workflow?id=ad10cd704268dcf0, https://usegalaxy.org.au/published/workflow?id=ad13b4a61e8987b1, https://usegalaxy.org.au/published/workflow?id=ae5dca8108d1a8a6, https://usegalaxy.org.au/published/workflow?id=ae6892f1df022097, https://usegalaxy.org.au/published/workflow?id=ae773dfaa074cb72, https://usegalaxy.org.au/published/workflow?id=ae894cff1d1aba01, https://usegalaxy.org.au/published/workflow?id=aed93da6fd6e4e13, https://usegalaxy.org.au/published/workflow?id=aee5b5645c51cac3, https://usegalaxy.org.au/published/workflow?id=af12a3ec85859563, https://usegalaxy.org.au/published/workflow?id=af3f40cba4d7afa6, https://usegalaxy.org.au/published/workflow?id=af7b95f46485308c, https://usegalaxy.org.au/published/workflow?id=af990cca72794142, https://usegalaxy.org.au/published/workflow?id=afd9a8bbc07f9c3f, https://usegalaxy.org.au/published/workflow?id=b03b577156e4db5b, https://usegalaxy.org.au/published/workflow?id=b08342866a118a5e, https://usegalaxy.org.au/published/workflow?id=b191da6e594747fb, https://usegalaxy.org.au/published/workflow?id=b19fa49405117317, https://usegalaxy.org.au/published/workflow?id=b20f28151e46d3e0, https://usegalaxy.org.au/published/workflow?id=b432a1312859edd2, https://usegalaxy.org.au/published/workflow?id=b5131e3a0ef2676e, https://usegalaxy.org.au/published/workflow?id=b517cabb697491a0, https://usegalaxy.org.au/published/workflow?id=b57bf134896c30ea, https://usegalaxy.org.au/published/workflow?id=b59fada651043735, https://usegalaxy.org.au/published/workflow?id=b6db525b1b73394b, https://usegalaxy.org.au/published/workflow?id=b7226d37b25df833, https://usegalaxy.org.au/published/workflow?id=b7aa22291322e505, https://usegalaxy.org.au/published/workflow?id=b8968fb0bf796e53, https://usegalaxy.org.au/published/workflow?id=b8c661ff6b79383b, https://usegalaxy.org.au/published/workflow?id=b9faf6a3f48fc11b, https://usegalaxy.org.au/published/workflow?id=ba30cc4a84c449f9, https://usegalaxy.org.au/published/workflow?id=ba7a9d4c857d26b7, https://usegalaxy.org.au/published/workflow?id=baa763dbdf6afb71, https://usegalaxy.org.au/published/workflow?id=bb0feff5d764df60, https://usegalaxy.org.au/published/workflow?id=bcf230e526632064, https://usegalaxy.org.au/published/workflow?id=bd1cf22d47389742, https://usegalaxy.org.au/published/workflow?id=bd251174d94cde9d, https://usegalaxy.org.au/published/workflow?id=bda98c5a0ea70dd0, https://usegalaxy.org.au/published/workflow?id=be9213411c520096, https://usegalaxy.org.au/published/workflow?id=be96bb47b6d6435a, https://usegalaxy.org.au/published/workflow?id=bf80ef6e5f37deb7, https://usegalaxy.org.au/published/workflow?id=c1a0a3888d9ee3ac, https://usegalaxy.org.au/published/workflow?id=c1ce3f2ae7c81c3e, https://usegalaxy.org.au/published/workflow?id=c296b3848708f6f1, https://usegalaxy.org.au/published/workflow?id=c2dee690320f9845, https://usegalaxy.org.au/published/workflow?id=c2e5a43356042b30, https://usegalaxy.org.au/published/workflow?id=c38b998d3e90c329, https://usegalaxy.org.au/published/workflow?id=c3b3717502ff6f76, https://usegalaxy.org.au/published/workflow?id=c3bb21f20a3ce689, https://usegalaxy.org.au/published/workflow?id=c3e198bc2146b4ad, https://usegalaxy.org.au/published/workflow?id=c43744d7dd9c0c0b, https://usegalaxy.org.au/published/workflow?id=c456492ce4f963d0, https://usegalaxy.org.au/published/workflow?id=c64dcc1e6a67f3e4, https://usegalaxy.org.au/published/workflow?id=c65a5af6c0383f8c, https://usegalaxy.org.au/published/workflow?id=c6a71573a299eb3a, https://usegalaxy.org.au/published/workflow?id=c7e1f666a3366823, https://usegalaxy.org.au/published/workflow?id=c7f2d6490128dc3c, https://usegalaxy.org.au/published/workflow?id=c87c1f8c4eff415a, https://usegalaxy.org.au/published/workflow?id=c8d18e2c03dca796, https://usegalaxy.org.au/published/workflow?id=c9156ca3df2b595c, https://usegalaxy.org.au/published/workflow?id=c9d04bf6207c0b32, https://usegalaxy.org.au/published/workflow?id=ca7fc6e2adbf1f26, https://usegalaxy.org.au/published/workflow?id=cb960edf95d2dc35, https://usegalaxy.org.au/published/workflow?id=ccc6602623583bcc, https://usegalaxy.org.au/published/workflow?id=ccf936c5b2f2c2f3, https://usegalaxy.org.au/published/workflow?id=ccfae75390038ced, https://usegalaxy.org.au/published/workflow?id=ce7106593536f900, https://usegalaxy.org.au/published/workflow?id=ce7f6a1c824920e2, https://usegalaxy.org.au/published/workflow?id=cef4a514f35576e5, https://usegalaxy.org.au/published/workflow?id=cef68481b305a516, https://usegalaxy.org.au/published/workflow?id=d126940d33e63036, https://usegalaxy.org.au/published/workflow?id=d298edda13b382e7, https://usegalaxy.org.au/published/workflow?id=d3521c05077192df, https://usegalaxy.org.au/published/workflow?id=d35d05be576e8b13, https://usegalaxy.org.au/published/workflow?id=d3989c6c2c663cd3, https://usegalaxy.org.au/published/workflow?id=d42d7b1aa47e870f, https://usegalaxy.org.au/published/workflow?id=d5ecfdbc04b24227, https://usegalaxy.org.au/published/workflow?id=d78879cfaa5948b3, https://usegalaxy.org.au/published/workflow?id=d7a65f577fe5f16b, https://usegalaxy.org.au/published/workflow?id=d7c329b354f4426d, https://usegalaxy.org.au/published/workflow?id=d7f3fc810261e419, https://usegalaxy.org.au/published/workflow?id=d9c181fd954b629a, https://usegalaxy.org.au/published/workflow?id=db0867756cb233b7, https://usegalaxy.org.au/published/workflow?id=db0bc7357c181222, https://usegalaxy.org.au/published/workflow?id=db3370a412c4de78, https://usegalaxy.org.au/published/workflow?id=dce29c35dfac48b1, https://usegalaxy.org.au/published/workflow?id=dd6f465ab1eb10b0, https://usegalaxy.org.au/published/workflow?id=de1c04a2caf70647, https://usegalaxy.org.au/published/workflow?id=df4e186069552649, https://usegalaxy.org.au/published/workflow?id=dfc5d454becbc956, https://usegalaxy.org.au/published/workflow?id=e08b04291839e79a, https://usegalaxy.org.au/published/workflow?id=e08ef6b519f5fba0, https://usegalaxy.org.au/published/workflow?id=e0b89b4e50d1a99b, https://usegalaxy.org.au/published/workflow?id=e1fd89e7248b6661, https://usegalaxy.org.au/published/workflow?id=e28ca83f14f4ede5, https://usegalaxy.org.au/published/workflow?id=e3cda847fe727b61, https://usegalaxy.org.au/published/workflow?id=e439a0e17a90d24c, https://usegalaxy.org.au/published/workflow?id=e4ddf33ee5b5e259, https://usegalaxy.org.au/published/workflow?id=e52a3479c8216291, https://usegalaxy.org.au/published/workflow?id=e57c6d05b560e206, https://usegalaxy.org.au/published/workflow?id=e5f1ce106ea7af12, https://usegalaxy.org.au/published/workflow?id=e62f743342851151, https://usegalaxy.org.au/published/workflow?id=e826584d78b69467, https://usegalaxy.org.au/published/workflow?id=e8d3651c07ce0bec, https://usegalaxy.org.au/published/workflow?id=e92b4d8dfa2f1cf7, https://usegalaxy.org.au/published/workflow?id=ea687c5613fcce27, https://usegalaxy.org.au/published/workflow?id=eaa841e6b7384633, https://usegalaxy.org.au/published/workflow?id=eb3a4535204383fa, https://usegalaxy.org.au/published/workflow?id=ec79c2fab8eac15d, https://usegalaxy.org.au/published/workflow?id=ed2f72e3537e8a4a, https://usegalaxy.org.au/published/workflow?id=ee7a1615f187391d, https://usegalaxy.org.au/published/workflow?id=efd39a4be0a9a4e6, https://usegalaxy.org.au/published/workflow?id=f01be6a7685ac476, https://usegalaxy.org.au/published/workflow?id=f05ed1e5c5dfffa9, https://usegalaxy.org.au/published/workflow?id=f0ced9a5d34b2aff, https://usegalaxy.org.au/published/workflow?id=f19d63d7c9a48160, https://usegalaxy.org.au/published/workflow?id=f25a05070bf97ded, https://usegalaxy.org.au/published/workflow?id=f268c181a47d0cab, https://usegalaxy.org.au/published/workflow?id=f38f610e0d95915f, https://usegalaxy.org.au/published/workflow?id=f53d8ade897b3f46, https://usegalaxy.org.au/published/workflow?id=f5c0b0bc5e023e42, https://usegalaxy.org.au/published/workflow?id=f627023a3401bf9b, https://usegalaxy.org.au/published/workflow?id=f7a866159f1eaecf, https://usegalaxy.org.au/published/workflow?id=f7adc76331620f34, https://usegalaxy.org.au/published/workflow?id=f944f2d3fbd1f062, https://usegalaxy.org.au/published/workflow?id=fa37223950a91f34, https://usegalaxy.org.au/published/workflow?id=fa4fa7353dffef9a, https://usegalaxy.org.au/published/workflow?id=fa87a22c8dcd8d78, https://usegalaxy.org.au/published/workflow?id=fb0986a952c35e0a, https://usegalaxy.org.au/published/workflow?id=fb644a9649f36f40, https://usegalaxy.org.au/published/workflow?id=fc3c83b11f527977, https://usegalaxy.org.au/published/workflow?id=fc64a77321752a3c, https://usegalaxy.org.au/published/workflow?id=fc6f8afdee980f0e, https://usegalaxy.org.au/published/workflow?id=fca05a51c9c91df4, https://usegalaxy.org.au/published/workflow?id=fd477917a3d27f61, https://usegalaxy.org.au/published/workflow?id=fd8fb237f8dc24c3, https://usegalaxy.org.au/published/workflow?id=fe1de81a97578368, https://usegalaxy.org.au/published/workflow?id=fe1e7fa0bc437ed8, https://usegalaxy.org.au/published/workflow?id=fe4e34a407157389, https://usegalaxy.org.au/published/workflow?id=fe5542c99cdc57ce, https://usegalaxy.org.au/published/workflow?id=fe893d72526eba71, https://usegalaxy.org.au/published/workflow?id=fedf2874752ac1f6, https://usegalaxy.org.au/published/workflow?id=ff42213dda7d3e59, https://usegalaxy.org.au/published/workflow?id=fff2e49bc90e38ca, https://usegalaxy.org/published/workflow?id=00a501a1fada4104, https://usegalaxy.org/published/workflow?id=00bc4e99626da1ba, https://usegalaxy.org/published/workflow?id=019be65613876562, https://usegalaxy.org/published/workflow?id=028f1beead539975, https://usegalaxy.org/published/workflow?id=02e29499b946f3bb, https://usegalaxy.org/published/workflow?id=03ce6eb4d9ee4309, https://usegalaxy.org/published/workflow?id=044c5ce43f2e1829, https://usegalaxy.org/published/workflow?id=05e86b4df0d14800, https://usegalaxy.org/published/workflow?id=05ef809e2b11154c, https://usegalaxy.org/published/workflow?id=063173d36c254105, https://usegalaxy.org/published/workflow?id=06b489dc859f6e39, https://usegalaxy.org/published/workflow?id=06f3ca70b542556e, https://usegalaxy.org/published/workflow?id=0775d8efad0dac10, https://usegalaxy.org/published/workflow?id=07dca0dc191fb1de, https://usegalaxy.org/published/workflow?id=07e873a6d75dfe9f, https://usegalaxy.org/published/workflow?id=08da44d18689dffc, https://usegalaxy.org/published/workflow?id=08f58af7b6cc0ab6, https://usegalaxy.org/published/workflow?id=091f788edad78956, https://usegalaxy.org/published/workflow?id=0930f955bdac93e8, https://usegalaxy.org/published/workflow?id=098375608a1223aa, https://usegalaxy.org/published/workflow?id=098fd236c5e2895d, https://usegalaxy.org/published/workflow?id=0993daadf2db5f51, https://usegalaxy.org/published/workflow?id=0b85fb10ed25d34e, https://usegalaxy.org/published/workflow?id=0d47009eb2d7bdc1, https://usegalaxy.org/published/workflow?id=0dec1b9dbbb2f425, https://usegalaxy.org/published/workflow?id=0e5a3899ae97076b, https://usegalaxy.org/published/workflow?id=0e6a842034ca9aec, https://usegalaxy.org/published/workflow?id=0ebb1f5b6c9c8ab9, https://usegalaxy.org/published/workflow?id=0ee0493f3168cbb3, https://usegalaxy.org/published/workflow?id=0f1659b3290c3492, https://usegalaxy.org/published/workflow?id=0f8b4fd0bf276968, https://usegalaxy.org/published/workflow?id=0fa2ce12c2874b1d, https://usegalaxy.org/published/workflow?id=0fd63eaea85d7002, https://usegalaxy.org/published/workflow?id=1010a9b57c77239c, https://usegalaxy.org/published/workflow?id=10c2136973c88d8a, https://usegalaxy.org/published/workflow?id=10c5f3c5cadd7417, https://usegalaxy.org/published/workflow?id=1234a128adc25b03, https://usegalaxy.org/published/workflow?id=1245e9435cbc3985, https://usegalaxy.org/published/workflow?id=1250ec3ad6199cda, https://usegalaxy.org/published/workflow?id=12a52ae52e0f1053, https://usegalaxy.org/published/workflow?id=137d0f05a71d7acb, https://usegalaxy.org/published/workflow?id=1388fe5d8b50bc7c, https://usegalaxy.org/published/workflow?id=13a1fa4f6daf65cc, https://usegalaxy.org/published/workflow?id=13d9db3715f32b37, https://usegalaxy.org/published/workflow?id=148c071f5c5a4586, https://usegalaxy.org/published/workflow?id=150e288c2c2fe9e9, https://usegalaxy.org/published/workflow?id=156e245e3d9f22ce, https://usegalaxy.org/published/workflow?id=1598b3fbd1dd2439, https://usegalaxy.org/published/workflow?id=15a0e22570f71098, https://usegalaxy.org/published/workflow?id=15ea806eaf077617, https://usegalaxy.org/published/workflow?id=162fda90525220de, https://usegalaxy.org/published/workflow?id=16910402667eec78, https://usegalaxy.org/published/workflow?id=170287dcfb9de475, https://usegalaxy.org/published/workflow?id=172ee5e5bef33d2e, https://usegalaxy.org/published/workflow?id=17d9b233640aefec, https://usegalaxy.org/published/workflow?id=17f3d7e52777d2e8, https://usegalaxy.org/published/workflow?id=1829c91a30138721, https://usegalaxy.org/published/workflow?id=195f7acb16c5a2e2, https://usegalaxy.org/published/workflow?id=19c1ba21e82d8615, https://usegalaxy.org/published/workflow?id=19ce6cf93298e268, https://usegalaxy.org/published/workflow?id=1a23d8c795846251, https://usegalaxy.org/published/workflow?id=1b57987895f8ca9e, https://usegalaxy.org/published/workflow?id=1c5a7196a442dd02, https://usegalaxy.org/published/workflow?id=1cad9a0b375ed3d1, https://usegalaxy.org/published/workflow?id=1cbd946c4cc0e7b9, https://usegalaxy.org/published/workflow?id=1d24cc949c52e475, https://usegalaxy.org/published/workflow?id=1d56a0975fca0e99, https://usegalaxy.org/published/workflow?id=1d69ab87c94d68fd, https://usegalaxy.org/published/workflow?id=1d8227ad3f3651a0, https://usegalaxy.org/published/workflow?id=1dd63a7b4c6f4c83, https://usegalaxy.org/published/workflow?id=1ddabc7d272dc702, https://usegalaxy.org/published/workflow?id=1e0a55635f5b04cf, https://usegalaxy.org/published/workflow?id=1e1a299825f27f11, https://usegalaxy.org/published/workflow?id=1e1d7fac7e241768, https://usegalaxy.org/published/workflow?id=1eec2586d1ff908b, https://usegalaxy.org/published/workflow?id=1f00e44f590b8369, https://usegalaxy.org/published/workflow?id=1f28e673ea725541, https://usegalaxy.org/published/workflow?id=1f4dc0488f0554c6, https://usegalaxy.org/published/workflow?id=1f98df1adba71b98, https://usegalaxy.org/published/workflow?id=1ffc71df8eeed57e, https://usegalaxy.org/published/workflow?id=2060fa13feff512b, https://usegalaxy.org/published/workflow?id=2078abc4c99c3344, https://usegalaxy.org/published/workflow?id=20b7ae4b517a80fd, https://usegalaxy.org/published/workflow?id=2125db88b4a71c09, https://usegalaxy.org/published/workflow?id=226b506d1fc30e96, https://usegalaxy.org/published/workflow?id=23468eea382e6bb4, https://usegalaxy.org/published/workflow?id=25a9f0693ff6d327, https://usegalaxy.org/published/workflow?id=25b8b5af30042d76, https://usegalaxy.org/published/workflow?id=26019c2b056d3a1c, https://usegalaxy.org/published/workflow?id=26b35874b9e32355, https://usegalaxy.org/published/workflow?id=271b97e5c00c4da4, https://usegalaxy.org/published/workflow?id=27327a985fa9147f, https://usegalaxy.org/published/workflow?id=27ee9da9121d0522, https://usegalaxy.org/published/workflow?id=283bc048cc7a80f5, https://usegalaxy.org/published/workflow?id=284150bb1f296378, https://usegalaxy.org/published/workflow?id=28b2550248a54d74, https://usegalaxy.org/published/workflow?id=291829fed6fa7c44, https://usegalaxy.org/published/workflow?id=2957924403998f3a, https://usegalaxy.org/published/workflow?id=2b41752b8518a87b, https://usegalaxy.org/published/workflow?id=2b5346b94358df88, https://usegalaxy.org/published/workflow?id=2b6245843e632c05, https://usegalaxy.org/published/workflow?id=2b820beaeb49dd64, https://usegalaxy.org/published/workflow?id=2c5c627c08563398, https://usegalaxy.org/published/workflow?id=2cc5472ad7c039d1, https://usegalaxy.org/published/workflow?id=2d6d74294e911fa3, https://usegalaxy.org/published/workflow?id=2d994cc02ec05ede, https://usegalaxy.org/published/workflow?id=2e029537d2f4ab50, https://usegalaxy.org/published/workflow?id=2f4d5f833aa1c981, https://usegalaxy.org/published/workflow?id=2f774500d5bec9a6, https://usegalaxy.org/published/workflow?id=2ff2961f044cac0f, https://usegalaxy.org/published/workflow?id=305eafecadfa8815, https://usegalaxy.org/published/workflow?id=305fb9f84b053713, https://usegalaxy.org/published/workflow?id=307d19833ffb5395, https://usegalaxy.org/published/workflow?id=30ba39be60ec29be, https://usegalaxy.org/published/workflow?id=313d50268e1e22e4, https://usegalaxy.org/published/workflow?id=31a6145b0832fa48, https://usegalaxy.org/published/workflow?id=31ed5a2bcc8da51a, https://usegalaxy.org/published/workflow?id=326ced2230a49c26, https://usegalaxy.org/published/workflow?id=32c769241ffce3d1, https://usegalaxy.org/published/workflow?id=336795e890851c21, https://usegalaxy.org/published/workflow?id=33ae58f05e5e1d60, https://usegalaxy.org/published/workflow?id=33e827d75a0074df, https://usegalaxy.org/published/workflow?id=34ab5572a9547bfe, https://usegalaxy.org/published/workflow?id=357225c624e796b5, https://usegalaxy.org/published/workflow?id=35c022174ce79c1e, https://usegalaxy.org/published/workflow?id=3673a1ced343d0d0, https://usegalaxy.org/published/workflow?id=36c885b0bc829da3, https://usegalaxy.org/published/workflow?id=371b242d97f2f030, https://usegalaxy.org/published/workflow?id=375f5142c207a768, https://usegalaxy.org/published/workflow?id=3823773ef4ecaaf4, https://usegalaxy.org/published/workflow?id=3a51f1f54b2ef024, https://usegalaxy.org/published/workflow?id=3a92253f57e553bd, https://usegalaxy.org/published/workflow?id=3aa1b79aa6cca06b, https://usegalaxy.org/published/workflow?id=3ab66315f50501f0, https://usegalaxy.org/published/workflow?id=3ad9cdcda44b7dfb, https://usegalaxy.org/published/workflow?id=3b0d42747f598f82, https://usegalaxy.org/published/workflow?id=3b2bee20f0cfd957, https://usegalaxy.org/published/workflow?id=3b5437544ca3b5dc, https://usegalaxy.org/published/workflow?id=3ba73f679558017c, https://usegalaxy.org/published/workflow?id=3c076bc8551a23d2, https://usegalaxy.org/published/workflow?id=3c0af11dbeeee506, https://usegalaxy.org/published/workflow?id=3d0ba2f11f370e05, https://usegalaxy.org/published/workflow?id=3d7a9694bcee6db5, https://usegalaxy.org/published/workflow?id=3db4eb1ecb588d44, https://usegalaxy.org/published/workflow?id=3e57e668cf698d19, https://usegalaxy.org/published/workflow?id=3e6c9ab5ef3250ae, https://usegalaxy.org/published/workflow?id=3f7550d5d2bf0b96, https://usegalaxy.org/published/workflow?id=3fc5bdbd6355477c, https://usegalaxy.org/published/workflow?id=3fdd7a7907233d58, https://usegalaxy.org/published/workflow?id=405dc14eb02ff234, https://usegalaxy.org/published/workflow?id=408c799822cbf6ea, https://usegalaxy.org/published/workflow?id=424acfc2ded46f9f, https://usegalaxy.org/published/workflow?id=426e3951dac0704e, https://usegalaxy.org/published/workflow?id=42bc3a770f0ab6b8, https://usegalaxy.org/published/workflow?id=4357180d40560019, https://usegalaxy.org/published/workflow?id=435f0340bf00c940, https://usegalaxy.org/published/workflow?id=438761debd08a9be, https://usegalaxy.org/published/workflow?id=4403ca7fe447a5d7, https://usegalaxy.org/published/workflow?id=442949862ba62462, https://usegalaxy.org/published/workflow?id=444c856f03b6d2a7, https://usegalaxy.org/published/workflow?id=4458439388155d92, https://usegalaxy.org/published/workflow?id=44781c8b0bd667ed, https://usegalaxy.org/published/workflow?id=448a0797f9150198, https://usegalaxy.org/published/workflow?id=45cb3e98b1319c7c, https://usegalaxy.org/published/workflow?id=4612ec88a57ab9e7, https://usegalaxy.org/published/workflow?id=462a1176b48eef4a, https://usegalaxy.org/published/workflow?id=467d3e260c21de98, https://usegalaxy.org/published/workflow?id=46894456a9bec601, https://usegalaxy.org/published/workflow?id=46e718fb8eeb4d54, https://usegalaxy.org/published/workflow?id=46f184a0e95f3c1c, https://usegalaxy.org/published/workflow?id=472f73c3e81da1d9, https://usegalaxy.org/published/workflow?id=4790266996f12979, https://usegalaxy.org/published/workflow?id=47a2458ee9d57b24, https://usegalaxy.org/published/workflow?id=484f8fcf21d67376, https://usegalaxy.org/published/workflow?id=48963532e030de6c, https://usegalaxy.org/published/workflow?id=496b0e8a0e5a4191, https://usegalaxy.org/published/workflow?id=498a8c6329ca5728, https://usegalaxy.org/published/workflow?id=49a07807fcea8158, https://usegalaxy.org/published/workflow?id=4a01263e3d4b4edb, https://usegalaxy.org/published/workflow?id=4aae99b2ca56a36b, https://usegalaxy.org/published/workflow?id=4b41a8247f52c463, https://usegalaxy.org/published/workflow?id=4b7daa3c73c8ef30, https://usegalaxy.org/published/workflow?id=4b92dfd20d48d479, https://usegalaxy.org/published/workflow?id=4ba61941f31f0215, https://usegalaxy.org/published/workflow?id=4bbe4c8a6c52d03a, https://usegalaxy.org/published/workflow?id=4bf73376661baafd, https://usegalaxy.org/published/workflow?id=4ca8b023f9a33440, https://usegalaxy.org/published/workflow?id=4cc70b699ea8bc63, https://usegalaxy.org/published/workflow?id=4cf8cc42e97b2da6, https://usegalaxy.org/published/workflow?id=4d5cc5010537f172, https://usegalaxy.org/published/workflow?id=4de095b92ed661ac, https://usegalaxy.org/published/workflow?id=4e1ae78405034ad3, https://usegalaxy.org/published/workflow?id=4f6e8ff94ca4f164, https://usegalaxy.org/published/workflow?id=506acac8c6a58d5e, https://usegalaxy.org/published/workflow?id=516a2181b41af94e, https://usegalaxy.org/published/workflow?id=5199f404ad7ecef8, https://usegalaxy.org/published/workflow?id=51c485af11d90700, https://usegalaxy.org/published/workflow?id=51d80a44d7d6fff7, https://usegalaxy.org/published/workflow?id=52592de9687fc6de, https://usegalaxy.org/published/workflow?id=526b05cbffecbc85, https://usegalaxy.org/published/workflow?id=527adadecbab2eb6, https://usegalaxy.org/published/workflow?id=52b7524a3941885c, https://usegalaxy.org/published/workflow?id=52bde903adbbaf60, https://usegalaxy.org/published/workflow?id=52e2b69787ca7a17, https://usegalaxy.org/published/workflow?id=5336b9d26a083983, https://usegalaxy.org/published/workflow?id=533b1d5d08f9943e, https://usegalaxy.org/published/workflow?id=5361db01e3b6cc0c, https://usegalaxy.org/published/workflow?id=53834a5fb46ca0f6, https://usegalaxy.org/published/workflow?id=53dda64a5a6ebd54, https://usegalaxy.org/published/workflow?id=53fdff74729a6780, https://usegalaxy.org/published/workflow?id=53fea86eb74c224b, https://usegalaxy.org/published/workflow?id=5415e85a86898e41, https://usegalaxy.org/published/workflow?id=553125cd799e5982, https://usegalaxy.org/published/workflow?id=5532e6f775adb0d7, https://usegalaxy.org/published/workflow?id=56427a9b9c3848bf, https://usegalaxy.org/published/workflow?id=570172fbcf865e1d, https://usegalaxy.org/published/workflow?id=574e42683dc3961b, https://usegalaxy.org/published/workflow?id=578e634b63923e9b, https://usegalaxy.org/published/workflow?id=57e6345fda3e2682, https://usegalaxy.org/published/workflow?id=582f3de3971fc38c, https://usegalaxy.org/published/workflow?id=58eeb06f911e8410, https://usegalaxy.org/published/workflow?id=59007e2bc9c807ee, https://usegalaxy.org/published/workflow?id=5944c5effd56b0d6, https://usegalaxy.org/published/workflow?id=597c4d54f8a8ebe7, https://usegalaxy.org/published/workflow?id=5a16ff38eb80da82, https://usegalaxy.org/published/workflow?id=5a55d9bd9701b1ce, https://usegalaxy.org/published/workflow?id=5a610d5a42d50cf3, https://usegalaxy.org/published/workflow?id=5aa4b9131c31cb79, https://usegalaxy.org/published/workflow?id=5b3e2ddf525da25d, https://usegalaxy.org/published/workflow?id=5b540de61a70df64, https://usegalaxy.org/published/workflow?id=5b8497a32bf01c4f, https://usegalaxy.org/published/workflow?id=5bcd1aade71a8035, https://usegalaxy.org/published/workflow?id=5bd6d7af64192b86, https://usegalaxy.org/published/workflow?id=5c85cd8c56b9f6ab, https://usegalaxy.org/published/workflow?id=5cb74afcca56a453, https://usegalaxy.org/published/workflow?id=5ccc7d708e13de60, https://usegalaxy.org/published/workflow?id=5d1208bd49f97aeb, https://usegalaxy.org/published/workflow?id=5dd0a672cceb15ec, https://usegalaxy.org/published/workflow?id=5dfec2a3ae5fa765, https://usegalaxy.org/published/workflow?id=5e18bbbe9ff2d9f5, https://usegalaxy.org/published/workflow?id=5efe288b18c7d0ab, https://usegalaxy.org/published/workflow?id=5f878c4cc3bff68c, https://usegalaxy.org/published/workflow?id=607d1683d6e26cef, https://usegalaxy.org/published/workflow?id=60b9910b7bb69f47, https://usegalaxy.org/published/workflow?id=612452c925d723b7, https://usegalaxy.org/published/workflow?id=6229546fb8b0eba9, https://usegalaxy.org/published/workflow?id=62316da77b1bfa5a, https://usegalaxy.org/published/workflow?id=626260e3c3ee5c08, https://usegalaxy.org/published/workflow?id=62c842e67fca5ab7, https://usegalaxy.org/published/workflow?id=62ecc8078a396e9e, https://usegalaxy.org/published/workflow?id=63b0ce2faec558d5, https://usegalaxy.org/published/workflow?id=63b3b266bde77563, https://usegalaxy.org/published/workflow?id=63f61ad7cae40646, https://usegalaxy.org/published/workflow?id=642e71c64a975461, https://usegalaxy.org/published/workflow?id=64a329cf75834d6b, https://usegalaxy.org/published/workflow?id=64af2cd469fee16a, https://usegalaxy.org/published/workflow?id=653d553ac2fa51c8, https://usegalaxy.org/published/workflow?id=66e6b41f6c6d52b7, https://usegalaxy.org/published/workflow?id=672125473ec700bc, https://usegalaxy.org/published/workflow?id=67252039bce060b2, https://usegalaxy.org/published/workflow?id=672ca09dce4b0dc3, https://usegalaxy.org/published/workflow?id=676b88d0aac13fa5, https://usegalaxy.org/published/workflow?id=676fa9af68f6a763, https://usegalaxy.org/published/workflow?id=67992bf894a82587, https://usegalaxy.org/published/workflow?id=683e725f672b7d9b, https://usegalaxy.org/published/workflow?id=68495cc8ec3d46df, https://usegalaxy.org/published/workflow?id=68b995393b3142ec, https://usegalaxy.org/published/workflow?id=68be9cfc0c58cb44, https://usegalaxy.org/published/workflow?id=6960b8f6c98a9b9e, https://usegalaxy.org/published/workflow?id=69712265fdb10ed5, https://usegalaxy.org/published/workflow?id=69edab5d6ad91556, https://usegalaxy.org/published/workflow?id=6a174e7498434bd5, https://usegalaxy.org/published/workflow?id=6a2dace98986992a, https://usegalaxy.org/published/workflow?id=6b11d738ee57f245, https://usegalaxy.org/published/workflow?id=6b511b47176893e3, https://usegalaxy.org/published/workflow?id=6c68072bf72e6c4a, https://usegalaxy.org/published/workflow?id=6c95a357a3c1f399, https://usegalaxy.org/published/workflow?id=6d68f56abf9c624d, https://usegalaxy.org/published/workflow?id=6dada297572d77bd, https://usegalaxy.org/published/workflow?id=6db96a9c7e4b8d14, https://usegalaxy.org/published/workflow?id=6dcb43f86b9cd7dc, https://usegalaxy.org/published/workflow?id=6dfe0c6e87c6cb7b, https://usegalaxy.org/published/workflow?id=6e47a35cc5f2bdce, https://usegalaxy.org/published/workflow?id=6ea1bd64873b6c18, https://usegalaxy.org/published/workflow?id=6ec8b101d24a0f4f, https://usegalaxy.org/published/workflow?id=6ecfa8ce19ed86da, https://usegalaxy.org/published/workflow?id=6f2e4b4860413153, https://usegalaxy.org/published/workflow?id=6f85b9fc84e3bf0c, https://usegalaxy.org/published/workflow?id=6fa88ca75caba8e4, https://usegalaxy.org/published/workflow?id=7022c4d9af725e5c, https://usegalaxy.org/published/workflow?id=7072731b51e16923, https://usegalaxy.org/published/workflow?id=7090bb14a7af15bd, https://usegalaxy.org/published/workflow?id=70dacee3da375e2b, https://usegalaxy.org/published/workflow?id=70f9c7cc75a1d3c0, https://usegalaxy.org/published/workflow?id=716dc497d7d2e1c3, https://usegalaxy.org/published/workflow?id=7185085cbe953cd9, https://usegalaxy.org/published/workflow?id=72d134a60c694b59, https://usegalaxy.org/published/workflow?id=7321c6ad5425d347, https://usegalaxy.org/published/workflow?id=74c98eafd363cc2b, https://usegalaxy.org/published/workflow?id=75324d68ed124ff5, https://usegalaxy.org/published/workflow?id=7551458d929355c2, https://usegalaxy.org/published/workflow?id=75e23bc4e1ac2f5b, https://usegalaxy.org/published/workflow?id=75fffbce810acaf3, https://usegalaxy.org/published/workflow?id=761675cb2c9ad2ef, https://usegalaxy.org/published/workflow?id=7643e66a9b4e9c11, https://usegalaxy.org/published/workflow?id=769ec5a187095b38, https://usegalaxy.org/published/workflow?id=76b8ed1c68678b62, https://usegalaxy.org/published/workflow?id=76d5143860fac947, https://usegalaxy.org/published/workflow?id=76fb573f1372ae86, https://usegalaxy.org/published/workflow?id=76fff93650152579, https://usegalaxy.org/published/workflow?id=7757cd56fa4d060c, https://usegalaxy.org/published/workflow?id=778a8867b0a6ea4f, https://usegalaxy.org/published/workflow?id=77d1505c0dd65e26, https://usegalaxy.org/published/workflow?id=78249d353038002b, https://usegalaxy.org/published/workflow?id=7832cfdeabe47696, https://usegalaxy.org/published/workflow?id=786aa9e2bd723629, https://usegalaxy.org/published/workflow?id=789647a199e563fd, https://usegalaxy.org/published/workflow?id=79527e2b83fd5c64, https://usegalaxy.org/published/workflow?id=7a23e64dfa6957ce, https://usegalaxy.org/published/workflow?id=7a26069c2939bef9, https://usegalaxy.org/published/workflow?id=7a33bc2d0970b502, https://usegalaxy.org/published/workflow?id=7a4d834e794407d9, https://usegalaxy.org/published/workflow?id=7aa36b1722ff4c11, https://usegalaxy.org/published/workflow?id=7b9e17cb1d72d7e4, https://usegalaxy.org/published/workflow?id=7c3faadda8f89749, https://usegalaxy.org/published/workflow?id=7c4a2339b17a8930, https://usegalaxy.org/published/workflow?id=7cee34a6c06c21e4, https://usegalaxy.org/published/workflow?id=7d11026214dd8c93, https://usegalaxy.org/published/workflow?id=7d6617b8316cee19, https://usegalaxy.org/published/workflow?id=7dba74c915f63dc1, https://usegalaxy.org/published/workflow?id=7dde0721a382ea3e, https://usegalaxy.org/published/workflow?id=7df3246858cd83d1, https://usegalaxy.org/published/workflow?id=7e2651e5ed17e3db, https://usegalaxy.org/published/workflow?id=7e56bee496344ddc, https://usegalaxy.org/published/workflow?id=7e9269e78e989480, https://usegalaxy.org/published/workflow?id=7f6900852a81d121, https://usegalaxy.org/published/workflow?id=7fb4dae5daab0522, https://usegalaxy.org/published/workflow?id=7fd68212ce020e6a, https://usegalaxy.org/published/workflow?id=803aec95399f7073, https://usegalaxy.org/published/workflow?id=80565301ada0d3eb, https://usegalaxy.org/published/workflow?id=8095deba606ebd5e, https://usegalaxy.org/published/workflow?id=8182055f5ef561f3, https://usegalaxy.org/published/workflow?id=81f1f0fd6d9235fd, https://usegalaxy.org/published/workflow?id=824730064b5014e5, https://usegalaxy.org/published/workflow?id=839cac17365affb7, https://usegalaxy.org/published/workflow?id=84161f4f8bf77474, https://usegalaxy.org/published/workflow?id=842c09b49c456429, https://usegalaxy.org/published/workflow?id=85011076892808e0, https://usegalaxy.org/published/workflow?id=85bc37a59a6ea443, https://usegalaxy.org/published/workflow?id=85e2a1d6b1d4e3d7, https://usegalaxy.org/published/workflow?id=864da6858d34df08, https://usegalaxy.org/published/workflow?id=868acaf49d0743ec, https://usegalaxy.org/published/workflow?id=870f4988a5cc815e, https://usegalaxy.org/published/workflow?id=8756aabc654b1050, https://usegalaxy.org/published/workflow?id=875ff758068e1c2f, https://usegalaxy.org/published/workflow?id=87756e064ab1e28a, https://usegalaxy.org/published/workflow?id=87c1bdc00b975377, https://usegalaxy.org/published/workflow?id=87d7a69106ca663d, https://usegalaxy.org/published/workflow?id=87eb1a8ee7a2c984, https://usegalaxy.org/published/workflow?id=87ebf1b5956a61b5, https://usegalaxy.org/published/workflow?id=885f0e66fa6b8790, https://usegalaxy.org/published/workflow?id=889b394b38b259eb, https://usegalaxy.org/published/workflow?id=88d0b64011c3148c, https://usegalaxy.org/published/workflow?id=88e10493f7e6b9d5, https://usegalaxy.org/published/workflow?id=8990ae43ae855bc7, https://usegalaxy.org/published/workflow?id=899f4f371b4c958c, https://usegalaxy.org/published/workflow?id=89f48993bb8f94bd, https://usegalaxy.org/published/workflow?id=89f52c97b3aee938, https://usegalaxy.org/published/workflow?id=8a15bda6ce7a63ec, https://usegalaxy.org/published/workflow?id=8a215695e7b57cf5, https://usegalaxy.org/published/workflow?id=8ae311282a0d27a1, https://usegalaxy.org/published/workflow?id=8b81b9a3b49bbf98, https://usegalaxy.org/published/workflow?id=8bcab83874cbc75c, https://usegalaxy.org/published/workflow?id=8bf608c6fd0dabd0, https://usegalaxy.org/published/workflow?id=8cbb9936e88d4f49, https://usegalaxy.org/published/workflow?id=8cf06b14037a8d5a, https://usegalaxy.org/published/workflow?id=8cfbaa14cff5a633, https://usegalaxy.org/published/workflow?id=8d605fbd33003d26, https://usegalaxy.org/published/workflow?id=8db9ccf14567d9ba, https://usegalaxy.org/published/workflow?id=8e1c6ad6ff623015, https://usegalaxy.org/published/workflow?id=8e2b2a40831dfb08, https://usegalaxy.org/published/workflow?id=8e509c87d143022c, https://usegalaxy.org/published/workflow?id=8e6d24a14d2908c2, https://usegalaxy.org/published/workflow?id=8e918c12d2f0d1fc, https://usegalaxy.org/published/workflow?id=8ebcae16a7822d93, https://usegalaxy.org/published/workflow?id=8eeee8ae24fb37b2, https://usegalaxy.org/published/workflow?id=8f392d06d32bdf14, https://usegalaxy.org/published/workflow?id=8f7c9461910b08f1, https://usegalaxy.org/published/workflow?id=907aaf55487ddf43, https://usegalaxy.org/published/workflow?id=90ed607bedd65524, https://usegalaxy.org/published/workflow?id=90f5b512997c70f8, https://usegalaxy.org/published/workflow?id=9124c2ec80b35680, https://usegalaxy.org/published/workflow?id=914ec4d8fc0022e4, https://usegalaxy.org/published/workflow?id=917e5a21d51a61ce, https://usegalaxy.org/published/workflow?id=91806144e90637e8, https://usegalaxy.org/published/workflow?id=918863756ce93725, https://usegalaxy.org/published/workflow?id=9192807702bc3de2, https://usegalaxy.org/published/workflow?id=91a4daecbb97d03e, https://usegalaxy.org/published/workflow?id=91ed290805f590de, https://usegalaxy.org/published/workflow?id=9255c0c9017b139f, https://usegalaxy.org/published/workflow?id=92aab10842ff610d, https://usegalaxy.org/published/workflow?id=931dd3e6b1fae4f5, https://usegalaxy.org/published/workflow?id=932b3bc10186f73e, https://usegalaxy.org/published/workflow?id=936ed7447c8e0a2d, https://usegalaxy.org/published/workflow?id=9373050c3ce80d56, https://usegalaxy.org/published/workflow?id=93bfbd7db42db543, https://usegalaxy.org/published/workflow?id=93c1fef4472aa7a8, https://usegalaxy.org/published/workflow?id=93cb976bd4ae546e, https://usegalaxy.org/published/workflow?id=93e69c7e7a857ca3, https://usegalaxy.org/published/workflow?id=94ab68d27de2e60f, https://usegalaxy.org/published/workflow?id=953d9cbe05e632a8, https://usegalaxy.org/published/workflow?id=958b64b05011d8c0, https://usegalaxy.org/published/workflow?id=958f78270fc8f55c, https://usegalaxy.org/published/workflow?id=961dac8100c6c70c, https://usegalaxy.org/published/workflow?id=962f784745c1b740, https://usegalaxy.org/published/workflow?id=96ae2544e22d34c5, https://usegalaxy.org/published/workflow?id=96cdba1fe599ada8, https://usegalaxy.org/published/workflow?id=9763787d7b102f39, https://usegalaxy.org/published/workflow?id=97c20a819394e381, https://usegalaxy.org/published/workflow?id=97dcd7285ebd5817, https://usegalaxy.org/published/workflow?id=98a4b90f7fc2bfce, https://usegalaxy.org/published/workflow?id=98b2ac8fc75781f8, https://usegalaxy.org/published/workflow?id=9938a81fb6911412, https://usegalaxy.org/published/workflow?id=995cc428eace3bc4, https://usegalaxy.org/published/workflow?id=9990b6e2facf6dca, https://usegalaxy.org/published/workflow?id=99ee567c092fb34c, https://usegalaxy.org/published/workflow?id=9adf8691809326e5, https://usegalaxy.org/published/workflow?id=9b384b774485b2f2, https://usegalaxy.org/published/workflow?id=9b5ae15bb9aaba48, https://usegalaxy.org/published/workflow?id=9bace540b35b1e61, https://usegalaxy.org/published/workflow?id=9bf1d223a57f88da, https://usegalaxy.org/published/workflow?id=9c0fbd9f51a9857f, https://usegalaxy.org/published/workflow?id=9c688a2fc8d1bf4e, https://usegalaxy.org/published/workflow?id=9da9ddae8d5dc92f, https://usegalaxy.org/published/workflow?id=9db607d8d1cdcc25, https://usegalaxy.org/published/workflow?id=9ddfdc9d6cacea88, https://usegalaxy.org/published/workflow?id=9ded318097694dbc, https://usegalaxy.org/published/workflow?id=9e74e3455e47c813, https://usegalaxy.org/published/workflow?id=9e9a027e0c294a79, https://usegalaxy.org/published/workflow?id=9eb29b71d7effefc, https://usegalaxy.org/published/workflow?id=9ecfaa613a112d78, https://usegalaxy.org/published/workflow?id=9edc78f665579559, https://usegalaxy.org/published/workflow?id=9f1e8e82fb3e2033, https://usegalaxy.org/published/workflow?id=9f55643d3fd6f28c, https://usegalaxy.org/published/workflow?id=9fc9f1db50019929, https://usegalaxy.org/published/workflow?id=a124d41da896779c, https://usegalaxy.org/published/workflow?id=a164d857439a126f, https://usegalaxy.org/published/workflow?id=a17af8b7b2b39c1a, https://usegalaxy.org/published/workflow?id=a2046dd883227b49, https://usegalaxy.org/published/workflow?id=a2b4a868fa9795d1, https://usegalaxy.org/published/workflow?id=a37961bf3d435918, https://usegalaxy.org/published/workflow?id=a3ff7dbfadbd96e2, https://usegalaxy.org/published/workflow?id=a4a56efb37d845bf, https://usegalaxy.org/published/workflow?id=a5bdab0b219b3797, https://usegalaxy.org/published/workflow?id=a63d3ee4a2a4a20b, https://usegalaxy.org/published/workflow?id=a6d6a2e8867fa566, https://usegalaxy.org/published/workflow?id=a71fcbc60caae8b6, https://usegalaxy.org/published/workflow?id=a743746151101352, https://usegalaxy.org/published/workflow?id=a8a38b594183db2b, https://usegalaxy.org/published/workflow?id=a8aee61c2cbaf6ea, https://usegalaxy.org/published/workflow?id=a8b9252fad5fe06a, https://usegalaxy.org/published/workflow?id=a8cd62e1924a64e1, https://usegalaxy.org/published/workflow?id=a8e0f77cd6ef49f9, https://usegalaxy.org/published/workflow?id=a8e5adb65d064641, https://usegalaxy.org/published/workflow?id=a8f633b79b20c760, https://usegalaxy.org/published/workflow?id=a986e4991fc61c92, https://usegalaxy.org/published/workflow?id=a99a938953be5fd6, https://usegalaxy.org/published/workflow?id=aa9def6ec8f13b3a, https://usegalaxy.org/published/workflow?id=aadccbe72752c129, https://usegalaxy.org/published/workflow?id=ab0be4897d5355ac, https://usegalaxy.org/published/workflow?id=ab490af27c9ccc80, https://usegalaxy.org/published/workflow?id=ac87b6133de6a03f, https://usegalaxy.org/published/workflow?id=acada5044d64a30a, https://usegalaxy.org/published/workflow?id=acae55e8f37fea59, https://usegalaxy.org/published/workflow?id=ace738014a7278b0, https://usegalaxy.org/published/workflow?id=adb86df9061dca2b, https://usegalaxy.org/published/workflow?id=adbed5c956a71543, https://usegalaxy.org/published/workflow?id=adca3f2ca63730fe, https://usegalaxy.org/published/workflow?id=ae1de4267e4da154, https://usegalaxy.org/published/workflow?id=ae523fe6c9183e58, https://usegalaxy.org/published/workflow?id=ae5327f6f30b6138, https://usegalaxy.org/published/workflow?id=af5d714396856e9a, https://usegalaxy.org/published/workflow?id=af71519c81429287, https://usegalaxy.org/published/workflow?id=afb67b4b9952f57d, https://usegalaxy.org/published/workflow?id=afd7633a34d68704, https://usegalaxy.org/published/workflow?id=affdc1bf42475cc9, https://usegalaxy.org/published/workflow?id=b0b6dfa166b0d873, https://usegalaxy.org/published/workflow?id=b0e19e7315b64c87, https://usegalaxy.org/published/workflow?id=b11796364c22235e, https://usegalaxy.org/published/workflow?id=b11c6268704dcc1b, https://usegalaxy.org/published/workflow?id=b19a9163c26f1270, https://usegalaxy.org/published/workflow?id=b1dc9cf054a63be5, https://usegalaxy.org/published/workflow?id=b232caf4a1e1b7ee, https://usegalaxy.org/published/workflow?id=b26501a4a85bfdeb, https://usegalaxy.org/published/workflow?id=b34ba9b2e4d8f050, https://usegalaxy.org/published/workflow?id=b3c8d64eca750ecf, https://usegalaxy.org/published/workflow?id=b5644a5dcf237ef9, https://usegalaxy.org/published/workflow?id=b5e8f4a0ff226921, https://usegalaxy.org/published/workflow?id=b63d1e1024cc3295, https://usegalaxy.org/published/workflow?id=b661fb850c4b4b01, https://usegalaxy.org/published/workflow?id=b6d535f210efa18e, https://usegalaxy.org/published/workflow?id=b73e416c06790ac5, https://usegalaxy.org/published/workflow?id=b78a75ae89ae4c79, https://usegalaxy.org/published/workflow?id=b7b69aaee5202144, https://usegalaxy.org/published/workflow?id=b7ec8d79dc563a93, https://usegalaxy.org/published/workflow?id=b86d24a103181742, https://usegalaxy.org/published/workflow?id=b882703ca6fea3a8, https://usegalaxy.org/published/workflow?id=b893662378ed3da0, https://usegalaxy.org/published/workflow?id=b8ed069eea7879e7, https://usegalaxy.org/published/workflow?id=b93a475af82a3d36, https://usegalaxy.org/published/workflow?id=ba347b695dbff2ea, https://usegalaxy.org/published/workflow?id=ba5889301928907f, https://usegalaxy.org/published/workflow?id=badd4b56dbdeaba7, https://usegalaxy.org/published/workflow?id=bae2d6b436cd80ef, https://usegalaxy.org/published/workflow?id=bb1cb6abfe84a646, https://usegalaxy.org/published/workflow?id=bb25ac3ba4863424, https://usegalaxy.org/published/workflow?id=bbc894a79eb0e769, https://usegalaxy.org/published/workflow?id=bbe6f5a45ea5ee5e, https://usegalaxy.org/published/workflow?id=bcb798fb718968ce, https://usegalaxy.org/published/workflow?id=bceb6ae418bef9b8, https://usegalaxy.org/published/workflow?id=bd2ce9a97c1598df, https://usegalaxy.org/published/workflow?id=bd355e8ce58a139e, https://usegalaxy.org/published/workflow?id=bd3bed78eb99e294, https://usegalaxy.org/published/workflow?id=bd6bfd338ffb9637, https://usegalaxy.org/published/workflow?id=bdd7011a908bc1aa, https://usegalaxy.org/published/workflow?id=be46c5bb8c06a470, https://usegalaxy.org/published/workflow?id=be7dab2274c1acdd, https://usegalaxy.org/published/workflow?id=bf3ccae8d5b6ef58, https://usegalaxy.org/published/workflow?id=bf3f71a748d88490, https://usegalaxy.org/published/workflow?id=bfd2e7e63d3e6856, https://usegalaxy.org/published/workflow?id=bfef75672b9c3ea4, https://usegalaxy.org/published/workflow?id=c03ada2070e9ca8a, https://usegalaxy.org/published/workflow?id=c0746df296544a3a, https://usegalaxy.org/published/workflow?id=c07476a937b4e52e, https://usegalaxy.org/published/workflow?id=c092a3631d68ce38, https://usegalaxy.org/published/workflow?id=c0a108e53ab05b0d, https://usegalaxy.org/published/workflow?id=c0a9a2d53b8660de, https://usegalaxy.org/published/workflow?id=c12e6b7896f8e114, https://usegalaxy.org/published/workflow?id=c1339c998a55431c, https://usegalaxy.org/published/workflow?id=c156e67ad1ae4a97, https://usegalaxy.org/published/workflow?id=c1ec12905944ca7b, https://usegalaxy.org/published/workflow?id=c20a68563e80d401, https://usegalaxy.org/published/workflow?id=c24cfddfc88a97a3, https://usegalaxy.org/published/workflow?id=c2898d28a3b380d6, https://usegalaxy.org/published/workflow?id=c30bfd545ef3a228, https://usegalaxy.org/published/workflow?id=c30f6ca171896908, https://usegalaxy.org/published/workflow?id=c478d8de43a3f30f, https://usegalaxy.org/published/workflow?id=c4ac09605633655c, https://usegalaxy.org/published/workflow?id=c57f70d686b6062e, https://usegalaxy.org/published/workflow?id=c5f4b7b2463f338b, https://usegalaxy.org/published/workflow?id=c66e2738f37a8c16, https://usegalaxy.org/published/workflow?id=c69f8c26ec5fb8a6, https://usegalaxy.org/published/workflow?id=c707871e1dc9e21d, https://usegalaxy.org/published/workflow?id=c712b85a55da2a2d, https://usegalaxy.org/published/workflow?id=c74e3a24751d9924, https://usegalaxy.org/published/workflow?id=c75acc1ab44c121f, https://usegalaxy.org/published/workflow?id=c75fcf2b7ba057fd, https://usegalaxy.org/published/workflow?id=c7b2d2ccc4b0dec5, https://usegalaxy.org/published/workflow?id=c81833a367445f98, https://usegalaxy.org/published/workflow?id=c9474fcce2a21188, https://usegalaxy.org/published/workflow?id=c95e9116e4b7982b, https://usegalaxy.org/published/workflow?id=c96b12be162f744b, https://usegalaxy.org/published/workflow?id=c983ea8a32662cd6, https://usegalaxy.org/published/workflow?id=ca36eebd59575df3, https://usegalaxy.org/published/workflow?id=cb011900fc0ee12a, https://usegalaxy.org/published/workflow?id=cbbd7e95041d9429, https://usegalaxy.org/published/workflow?id=cd4b5b62f1eed51b, https://usegalaxy.org/published/workflow?id=cd4d88429f096fbc, https://usegalaxy.org/published/workflow?id=cd5f803c80875b9c, https://usegalaxy.org/published/workflow?id=cd824d8631d6a55e, https://usegalaxy.org/published/workflow?id=cdea4f88ccbd155f, https://usegalaxy.org/published/workflow?id=cdf5dffbed938d49, https://usegalaxy.org/published/workflow?id=ce04658f05029be2, https://usegalaxy.org/published/workflow?id=ce815d36cbb57304, https://usegalaxy.org/published/workflow?id=cf2dbc29ca20ebee, https://usegalaxy.org/published/workflow?id=cf3320614654adfe, https://usegalaxy.org/published/workflow?id=cfa7c3d7e69e91d7, https://usegalaxy.org/published/workflow?id=d00ac64dcd07f91a, https://usegalaxy.org/published/workflow?id=d02b4d7483936b73, https://usegalaxy.org/published/workflow?id=d08135fa13d1745e, https://usegalaxy.org/published/workflow?id=d2759906e4124964, https://usegalaxy.org/published/workflow?id=d2dec03b33af1b2f, https://usegalaxy.org/published/workflow?id=d2f7fd589a91124a, https://usegalaxy.org/published/workflow?id=d341660f3d1ce8bf, https://usegalaxy.org/published/workflow?id=d3428dd172a78102, https://usegalaxy.org/published/workflow?id=d36765ac28c11e1a, https://usegalaxy.org/published/workflow?id=d3704f2927e3065e, https://usegalaxy.org/published/workflow?id=d4029ce1344e874a, https://usegalaxy.org/published/workflow?id=d4ea6cdd40522eb1, https://usegalaxy.org/published/workflow?id=d4f0a7ccb09f1e55, https://usegalaxy.org/published/workflow?id=d5062ef48aa246d8, https://usegalaxy.org/published/workflow?id=d6cf3be79a4b7db4, https://usegalaxy.org/published/workflow?id=d768a4a06d3b1a90, https://usegalaxy.org/published/workflow?id=d817e98cef94bc96, https://usegalaxy.org/published/workflow?id=d82db9c29b998f22, https://usegalaxy.org/published/workflow?id=d86b28c8c2e0e64d, https://usegalaxy.org/published/workflow?id=d903c7215a88c79f, https://usegalaxy.org/published/workflow?id=d96bd5c9fb553f2c, https://usegalaxy.org/published/workflow?id=d9840d6ecf2b4d2f, https://usegalaxy.org/published/workflow?id=daaded719dcfd53b, https://usegalaxy.org/published/workflow?id=db01ac5b0dc50cd4, https://usegalaxy.org/published/workflow?id=dbc650d226e89013, https://usegalaxy.org/published/workflow?id=dc006ebf798f1f2f, https://usegalaxy.org/published/workflow?id=dc7f05cb5a62da77, https://usegalaxy.org/published/workflow?id=dcf658333596d76a, https://usegalaxy.org/published/workflow?id=ddf222b797082436, https://usegalaxy.org/published/workflow?id=de208a900880dcdc, https://usegalaxy.org/published/workflow?id=de24e5678f541584, https://usegalaxy.org/published/workflow?id=de29cd965a2e93d3, https://usegalaxy.org/published/workflow?id=deea71ccac30c7ea, https://usegalaxy.org/published/workflow?id=df24d76ebf13e3c1, https://usegalaxy.org/published/workflow?id=df8b49863e83824d, https://usegalaxy.org/published/workflow?id=dfa8c64634aec75c, https://usegalaxy.org/published/workflow?id=dff3cf6921b3d4d6, https://usegalaxy.org/published/workflow?id=e035be74e3c85bf5, https://usegalaxy.org/published/workflow?id=e0ec503626fa0d58, https://usegalaxy.org/published/workflow?id=e12296df7cf33c69, https://usegalaxy.org/published/workflow?id=e12324f5e3b930fd, https://usegalaxy.org/published/workflow?id=e1aad86fdf9e6616, https://usegalaxy.org/published/workflow?id=e282f505ba9f729f, https://usegalaxy.org/published/workflow?id=e31abcc04c5c69bc, https://usegalaxy.org/published/workflow?id=e325b97b81b10edb, https://usegalaxy.org/published/workflow?id=e33f6a058c5f57e5, https://usegalaxy.org/published/workflow?id=e42a07b777c9f9dc, https://usegalaxy.org/published/workflow?id=e4dd39237df58884, https://usegalaxy.org/published/workflow?id=e500078649dd026c, https://usegalaxy.org/published/workflow?id=e53b695ff681b675, https://usegalaxy.org/published/workflow?id=e55ca98817c1855e, https://usegalaxy.org/published/workflow?id=e58a87508a337b91, https://usegalaxy.org/published/workflow?id=e5f79f576548f7c5, https://usegalaxy.org/published/workflow?id=e64a6119053cb126, https://usegalaxy.org/published/workflow?id=e67b6da41e1e6932, https://usegalaxy.org/published/workflow?id=e745c45651d829b8, https://usegalaxy.org/published/workflow?id=e7586d70715207b5, https://usegalaxy.org/published/workflow?id=e7ce46762faccaac, https://usegalaxy.org/published/workflow?id=e7fa48d7fd40908b, https://usegalaxy.org/published/workflow?id=e817b5b275b4952f, https://usegalaxy.org/published/workflow?id=e899acc664c8b808, https://usegalaxy.org/published/workflow?id=e9927c058ec0661f, https://usegalaxy.org/published/workflow?id=e9aaa56af9d230d6, https://usegalaxy.org/published/workflow?id=e9f4de639f4cfe02, https://usegalaxy.org/published/workflow?id=e9ffad95baa43478, https://usegalaxy.org/published/workflow?id=ea248022a4217e0e, https://usegalaxy.org/published/workflow?id=ea4c2defce2f5258, https://usegalaxy.org/published/workflow?id=ea5d2401f6711dff, https://usegalaxy.org/published/workflow?id=ea61cd21e3fbad2b, https://usegalaxy.org/published/workflow?id=eab7347464872b4c, https://usegalaxy.org/published/workflow?id=eae5ae951abe6761, https://usegalaxy.org/published/workflow?id=eafe4aa62a600e1d, https://usegalaxy.org/published/workflow?id=eb3409cc73e392a8, https://usegalaxy.org/published/workflow?id=eb4b32fcf4a9f64a, https://usegalaxy.org/published/workflow?id=eb4f7dbca0114270, https://usegalaxy.org/published/workflow?id=ebe7f93204402a5a, https://usegalaxy.org/published/workflow?id=ec2f77224324f22e, https://usegalaxy.org/published/workflow?id=ec66767cab0022ee, https://usegalaxy.org/published/workflow?id=edf9be785ab9eb65, https://usegalaxy.org/published/workflow?id=ee27581327d147da, https://usegalaxy.org/published/workflow?id=ee3b8d7ba29da11c, https://usegalaxy.org/published/workflow?id=ef6036efe8a03a12, https://usegalaxy.org/published/workflow?id=efe7ca87323773c0, https://usegalaxy.org/published/workflow?id=f062eda543541ba8, https://usegalaxy.org/published/workflow?id=f0ee8bcd7c8fadca, https://usegalaxy.org/published/workflow?id=f18946db4c93e88b, https://usegalaxy.org/published/workflow?id=f1a54b10d4afce8f, https://usegalaxy.org/published/workflow?id=f1c2d920bd41cebb, https://usegalaxy.org/published/workflow?id=f29d1bcfeaacba04, https://usegalaxy.org/published/workflow?id=f2d3abc6c45c4839, https://usegalaxy.org/published/workflow?id=f3aba99a8204081e, https://usegalaxy.org/published/workflow?id=f3b417763495221b, https://usegalaxy.org/published/workflow?id=f3c8093c5b8a7233, https://usegalaxy.org/published/workflow?id=f5200b7bcceed5d6, https://usegalaxy.org/published/workflow?id=f5805e66642fed04, https://usegalaxy.org/published/workflow?id=f58aa85ceaa19534, https://usegalaxy.org/published/workflow?id=f5bd2da36ab0a870, https://usegalaxy.org/published/workflow?id=f60027545d40b82d, https://usegalaxy.org/published/workflow?id=f733ad0a6cc50cb9, https://usegalaxy.org/published/workflow?id=f765c8aa4bc3176c, https://usegalaxy.org/published/workflow?id=f794b4b9bb990c80, https://usegalaxy.org/published/workflow?id=f7975ae799daf463, https://usegalaxy.org/published/workflow?id=f7a3567ba6b57c9c, https://usegalaxy.org/published/workflow?id=f8d85cda81baa77d, https://usegalaxy.org/published/workflow?id=f8ed1ae9d501bb87, https://usegalaxy.org/published/workflow?id=f926e70bee242785, https://usegalaxy.org/published/workflow?id=f93b5864d6d2659c, https://usegalaxy.org/published/workflow?id=f9cd2289c007f103, https://usegalaxy.org/published/workflow?id=fa226c044fa026ef, https://usegalaxy.org/published/workflow?id=fa26fd51caea53c7, https://usegalaxy.org/published/workflow?id=fb588edfd343b24d, https://usegalaxy.org/published/workflow?id=fb789920b057fe8d, https://usegalaxy.org/published/workflow?id=fb903de4aa4d5090, https://usegalaxy.org/published/workflow?id=fc130cb411aa7d06, https://usegalaxy.org/published/workflow?id=fc7b14aafa0b421e, https://usegalaxy.org/published/workflow?id=fcae9dfcdca70798, https://usegalaxy.org/published/workflow?id=fcbd4767298488b9, https://usegalaxy.org/published/workflow?id=fcbe12fc4eaae12e, https://usegalaxy.org/published/workflow?id=fce1837144d3a86e, https://usegalaxy.org/published/workflow?id=fd40e50c9f1545ce, https://usegalaxy.org/published/workflow?id=fd75347cfea06651, https://usegalaxy.org/published/workflow?id=fda29ba2b6c98f8b, https://usegalaxy.org/published/workflow?id=fe4c2791508c56ab, https://usegalaxy.org/published/workflow?id=ff35347c4e2f0e81, https://usegalaxy.org/published/workflow?id=ffae9c47980a5170, https://usegalaxy.org/published/workflow?id=fff1a5c7475b3eeb, https://workflowhub.eu/workflows/1035?version=1, https://workflowhub.eu/workflows/1061?version=1, https://workflowhub.eu/workflows/110?version=11, https://workflowhub.eu/workflows/111?version=5, https://workflowhub.eu/workflows/112?version=6, https://workflowhub.eu/workflows/113?version=4, https://workflowhub.eu/workflows/1190?version=1, https://workflowhub.eu/workflows/1271?version=3, https://workflowhub.eu/workflows/1272?version=2, https://workflowhub.eu/workflows/1352?version=4, https://workflowhub.eu/workflows/1444?version=2, https://workflowhub.eu/workflows/1466?version=2, https://workflowhub.eu/workflows/1473?version=1, https://workflowhub.eu/workflows/1492?version=2, https://workflowhub.eu/workflows/1534?version=1, https://workflowhub.eu/workflows/1540?version=1, https://workflowhub.eu/workflows/155?version=6, https://workflowhub.eu/workflows/1562?version=1, https://workflowhub.eu/workflows/1585?version=1, https://workflowhub.eu/workflows/1598?version=1, https://workflowhub.eu/workflows/1612?version=1, https://workflowhub.eu/workflows/1616?version=1, https://workflowhub.eu/workflows/1622?version=1, https://workflowhub.eu/workflows/1640?version=1, https://workflowhub.eu/workflows/1642?version=1, https://workflowhub.eu/workflows/1647?version=1, https://workflowhub.eu/workflows/1651?version=1, https://workflowhub.eu/workflows/1657?version=1, https://workflowhub.eu/workflows/1661?version=2, https://workflowhub.eu/workflows/1665?version=1, https://workflowhub.eu/workflows/1668?version=1, https://workflowhub.eu/workflows/1675?version=1, https://workflowhub.eu/workflows/1678?version=1, https://workflowhub.eu/workflows/1688?version=1, https://workflowhub.eu/workflows/1692?version=1, https://workflowhub.eu/workflows/1695?version=1, https://workflowhub.eu/workflows/1715?version=3, https://workflowhub.eu/workflows/1716?version=1, https://workflowhub.eu/workflows/1717?version=2, https://workflowhub.eu/workflows/1850?version=1, https://workflowhub.eu/workflows/1854?version=1, https://workflowhub.eu/workflows/1876?version=2, https://workflowhub.eu/workflows/2011?version=1, https://workflowhub.eu/workflows/2025?version=2, https://workflowhub.eu/workflows/2026?version=2, https://workflowhub.eu/workflows/2027?version=2, https://workflowhub.eu/workflows/2042?version=1, https://workflowhub.eu/workflows/222?version=1, https://workflowhub.eu/workflows/346?version=1, https://workflowhub.eu/workflows/347?version=1, https://workflowhub.eu/workflows/351?version=1, https://workflowhub.eu/workflows/353?version=1, https://workflowhub.eu/workflows/357?version=1, https://workflowhub.eu/workflows/35?version=1, https://workflowhub.eu/workflows/36?version=1, https://workflowhub.eu/workflows/37?version=1, https://workflowhub.eu/workflows/38?version=1, https://workflowhub.eu/workflows/395?version=15, https://workflowhub.eu/workflows/397?version=16, https://workflowhub.eu/workflows/398?version=17, https://workflowhub.eu/workflows/399?version=19, https://workflowhub.eu/workflows/39?version=1, https://workflowhub.eu/workflows/400?version=12, https://workflowhub.eu/workflows/401?version=13, https://workflowhub.eu/workflows/40?version=1, https://workflowhub.eu/workflows/439?version=4, https://workflowhub.eu/workflows/519?version=1, https://workflowhub.eu/workflows/561?version=11, https://workflowhub.eu/workflows/601?version=2, https://workflowhub.eu/workflows/602?version=2, https://workflowhub.eu/workflows/612?version=25, https://workflowhub.eu/workflows/625?version=29, https://workflowhub.eu/workflows/628?version=1, https://workflowhub.eu/workflows/641?version=32, https://workflowhub.eu/workflows/642?version=25, https://workflowhub.eu/workflows/676?version=1, https://workflowhub.eu/workflows/688?version=1, https://workflowhub.eu/workflows/876?version=1</td>
      <td>assembly/assembly-with-preprocessing, assembly/general-introduction, assembly/hybrid_denovo_assembly, assembly/largegenome, assembly/unicycler-assembly, contributing/create-new-tutorial-content, ecology/ENA_Biodiv_submission, ecology/ref-based-rad-seq, epigenetics/cut_and_run, galaxy-interface/workflow-automation, galaxy-interface/workflow-fairification, genome-annotation/crispr-screen, introduction/galaxy-intro-ngs-data-managment, introduction/vsi_qc, microbiome/host-removal, microbiome/mags-building, microbiome/metatranscriptomics, microbiome/metatranscriptomics-short, microbiome/mgnify-amplicon, microbiome/nanopore-16S-metagenomics, microbiome/pathogen-detection-from-nanopore-foodborne-data, sequence-analysis/mapping, sequence-analysis/quality-control, single-cell/scrna-preprocessing, single-cell/scrna-preprocessing-tenx, transcriptomics/differential-isoform-expression, transcriptomics/full-de-novo, transcriptomics/minerva-pathways, transcriptomics/mirna-target-finder, transcriptomics/ref-based, transcriptomics/rna-seq-reads-to-counts, variant-analysis/exome-seq, variant-analysis/sars-cov-2, variant-analysis/somatic-variant-discovery, variant-analysis/somatic-variants, variant-analysis/tb-variant-analysis</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>mykrobe</td>
      <td>mykrobe_predict</td>
      <td>Antibiotic resistance predictions</td>
      <td>2017-12-18</td>
      <td>https://github.com/Mykrobe-tools/mykrobe</td>
      <td>0.10.0</td>
      <td>mykrobe</td>
      <td>0.13.0</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Antimicrobial resistance prediction, Variant calling, Genotyping, Sequence trimming</td>
      <td>Antimicrobial resistance prediction, Variant calling, Genotyping, Sequence trimming</td>
      <td>Whole genome sequencing, Genotype and phenotype, Probes and primers, Genetic variation, Metagenomics</td>
      <td>Whole genome sequencing, Genotype and phenotype, Probes and primers, Genetic variation, Metagenomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mykrobe</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mykrobe</td>
      <td>Mykrobe</td>
      <td>Mykrobe</td>
      <td>Antibiotic resistance prediction for Mycobacterium tuberculosis from genome sequence data with Mykrobe.Antibiotic resistance prediction in minutes.Table of Contents generated with DocToc.AMR prediction (Mykrobe predictor).Before attempting to install with bioconda, please ensure you have your channels set up as specified in the documentation. If you don't, you may run into issues with an older version of mykrobe being installed</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>mykrobe_parser</td>
      <td>mykrobe_parseR</td>
      <td>RScript to parse the results of mykrobe predictor.</td>
      <td>2018-09-28</td>
      <td>https://github.com/phac-nml/mykrobe-parser</td>
      <td>0.1.4.1</td>
      <td>r-base</td>
      <td></td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>nml</td>
      <td>https://github.com/phac-nml/mykrobe-parser</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/mykrobe_parser</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>mz_to_sqlite</td>
      <td>mz_to_sqlite</td>
      <td>Creates a SQLite database for proteomics data</td>
      <td>2015-06-01</td>
      <td>https://github.com/galaxyproteomics/mzToSQLite</td>
      <td>2.1.1+galaxy0</td>
      <td>mztosqlite</td>
      <td>2.1.1</td>
      <td>To update</td>
      <td>Proteomics</td>
      <td>Conversion, Peptide database search</td>
      <td>Conversion, Peptide database search</td>
      <td>Proteomics, Biological databases</td>
      <td>Proteomics, Biological databases</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/mz_to_sqlite</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/mz_to_sqlite</td>
      <td>mztosqlite</td>
      <td>mzToSQLite</td>
      <td>Convert proteomics data files into a SQLite databaseExtract data from proteomics mzIdentML and mass spec scan files: mzML, MGF, etc and store in a SQLite database.The intended purpose is to provide a Galaxy dataset that can support an interactive Galaxy visualization plugin.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>136</td>
      <td>146</td>
      <td>868</td>
      <td>959</td>
      <td>17</td>
      <td>17</td>
      <td>83</td>
      <td>83</td>
      <td>18</td>
      <td>18</td>
      <td>336</td>
      <td>336</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1378</td>
      <td>1287</td>
      <td>181</td>
      <td>171</td>
      <td>https://usegalaxy.eu/published/workflow?id=03e52c8c71710cae, https://usegalaxy.eu/published/workflow?id=0aee7ffa5f6384bd, https://usegalaxy.eu/published/workflow?id=2a7e63d1cc583002, https://usegalaxy.eu/published/workflow?id=41d096891bada138, https://usegalaxy.eu/published/workflow?id=41e9882300fa2386, https://usegalaxy.eu/published/workflow?id=6e5f0d05c91876c2, https://usegalaxy.eu/published/workflow?id=8a85663218676b59, https://usegalaxy.eu/published/workflow?id=9a8b5e99ca9500e3, https://usegalaxy.eu/published/workflow?id=b89cf3cb6a25c999, https://usegalaxy.eu/published/workflow?id=eb54512829d8509c, https://usegalaxy.eu/published/workflow?id=edea08c943ff9c5e, https://usegalaxy.org.au/published/workflow?id=1b9bfe7ce484fdc2, https://workflowhub.eu/workflows/1429?version=1</td>
      <td>proteomics/proteogenomics-dbsearch</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>nanocompore</td>
      <td>nanocompore_db, nanocompore_sampcomp</td>
      <td>Nanocompore compares 2 ONT nanopore direct RNA sequencing datasets from different experimental conditions expected to have a significant impact on RNA modifications. It is recommended to have at least 2 replicates per condition. For example one can use a control condition with a significantly reduced number of modifications such as a cell line for which a modification writing enzyme was knocked-down or knocked-out. Alternatively, on a smaller scale transcripts of interests could be synthesized in-vitro.</td>
      <td>2020-04-27</td>
      <td>https://nanocompore.rna.rocks/</td>
      <td>1.0.0rc3.post2</td>
      <td>nanocompore</td>
      <td>1.0.4</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Post-translation modification site prediction, PolyA signal detection, Genotyping, k-mer counting</td>
      <td>PolyA signal detection, Genotyping, k-mer counting</td>
      <td>Functional, regulatory and non-coding RNA, RNA-Seq, Gene transcripts, Transcriptomics, Transcription factors and regulatory sites</td>
      <td>Functional, regulatory and non-coding RNA, RNA-Seq, Gene transcripts, Transcriptomics, Transcription factors and regulatory sites</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/nanocompore</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/nanocompore</td>
      <td>Nanocompore</td>
      <td>Nanocompore</td>
      <td>RNA modifications detection by comparative Nanopore direct RNA sequencing.RNA modifications detection from Nanopore dRNA-Seq data.Nanocompore identifies differences in ONT nanopore sequencing raw signal corresponding to RNA modifications by comparing 2 samples.Analyses performed for the nanocompore paper.Nanocompore compares 2 ONT nanopore direct RNA sequencing datasets from different experimental conditions expected to have a significant impact on RNA modifications. It is recommended to have at least 2 replicates per condition. For example one can use a control condition with a significantly reduced number of modifications such as a cell line for which a modification writing enzyme was knocked-down or knocked-out. Alternatively, on a smaller scale transcripts of interests could be synthesized in-vitro</td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>5</td>
      <td>7</td>
      <td>27</td>
      <td>839</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>365</td>
      <td>371</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1210</td>
      <td>392</td>
      <td>12</td>
      <td>10</td>
      <td>https://usegalaxy.eu/published/workflow?id=778923dd04996e07, https://usegalaxy.eu/published/workflow?id=77db45ad90b4033d</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>nanoplot</td>
      <td>nanoplot</td>
      <td>Plotting tool for long read sequencing data and alignments</td>
      <td>2018-09-24</td>
      <td>https://github.com/wdecoster/NanoPlot</td>
      <td>1.46.2</td>
      <td>nanoplot</td>
      <td>1.46.2</td>
      <td>Up-to-date</td>
      <td>Visualization</td>
      <td>Scatter plot plotting, Box-Whisker plot plotting</td>
      <td>Scatter plot plotting, Box-Whisker plot plotting</td>
      <td>Genomics</td>
      <td>Genomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/nanoplot</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/nanoplot</td>
      <td>nanoplot</td>
      <td>NanoPlot</td>
      <td>NanoPlot is a tool with various visualizations of sequencing data in bam, cram, fastq, fasta or platform-specific TSV summaries, mainly intended for long-read sequencing from Oxford Nanopore Technologies and Pacific Biosciences</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>7663</td>
      <td>7942</td>
      <td>168514</td>
      <td>173861</td>
      <td>4903</td>
      <td>5029</td>
      <td>72856</td>
      <td>74900</td>
      <td>2940</td>
      <td>2983</td>
      <td>25741</td>
      <td>26566</td>
      <td>295</td>
      <td>297</td>
      <td>3814</td>
      <td>3828</td>
      <td>279155</td>
      <td>270925</td>
      <td>16251</td>
      <td>15801</td>
      <td>https://usegalaxy.eu/published/workflow?id=07c92110c3fffcf8, https://usegalaxy.eu/published/workflow?id=0bafeb17c21ebc9c, https://usegalaxy.eu/published/workflow?id=14714876dda7186d, https://usegalaxy.eu/published/workflow?id=152223c8aed27357, https://usegalaxy.eu/published/workflow?id=181e751be8dd9abc, https://usegalaxy.eu/published/workflow?id=249c115413f17f28, https://usegalaxy.eu/published/workflow?id=3081569659a45b9e, https://usegalaxy.eu/published/workflow?id=32d2f5340098bcbd, https://usegalaxy.eu/published/workflow?id=344cd3a3c2ce5302, https://usegalaxy.eu/published/workflow?id=34a07bf6407510ef, https://usegalaxy.eu/published/workflow?id=368d1ac0150ea0b6, https://usegalaxy.eu/published/workflow?id=39837ca09374598b, https://usegalaxy.eu/published/workflow?id=44e53ea319e07daa, https://usegalaxy.eu/published/workflow?id=49ce9e738fadef4a, https://usegalaxy.eu/published/workflow?id=4ddbffe4b3fef275, https://usegalaxy.eu/published/workflow?id=4eb9dfe91fc2dc67, https://usegalaxy.eu/published/workflow?id=57cfe5062c8e01bf, https://usegalaxy.eu/published/workflow?id=57d3109f32defa3b, https://usegalaxy.eu/published/workflow?id=5826e573b6f93762, https://usegalaxy.eu/published/workflow?id=69be516a48761365, https://usegalaxy.eu/published/workflow?id=86cf02cc0868d238, https://usegalaxy.eu/published/workflow?id=889f2782ad82bd69, https://usegalaxy.eu/published/workflow?id=968f0ac2617aa524, https://usegalaxy.eu/published/workflow?id=a337bfb2eb1e583f, https://usegalaxy.eu/published/workflow?id=a705370bc2c13d5c, https://usegalaxy.eu/published/workflow?id=a84bac86ec183d46, https://usegalaxy.eu/published/workflow?id=ab646c6ab6de7061, https://usegalaxy.eu/published/workflow?id=b4de76c0a4d0e23a, https://usegalaxy.eu/published/workflow?id=bf48b0eb15e8ff75, https://usegalaxy.eu/published/workflow?id=c3bfedf33bb00069, https://usegalaxy.eu/published/workflow?id=c48807045e258dd8, https://usegalaxy.eu/published/workflow?id=c9ef2d2ce92e2dfb, https://usegalaxy.eu/published/workflow?id=cf8591391a17a914, https://usegalaxy.eu/published/workflow?id=d0ae255151450dad, https://usegalaxy.eu/published/workflow?id=dc7107896a3f2964, https://usegalaxy.eu/published/workflow?id=dd3193aaaf08e798, https://usegalaxy.eu/published/workflow?id=ea485dbabc76488d, https://usegalaxy.eu/published/workflow?id=eb5ab8bc82569eee, https://usegalaxy.eu/published/workflow?id=ecf48fa50af560a0, https://usegalaxy.eu/published/workflow?id=fc0b9efa7d2e61e6, https://usegalaxy.fr/published/workflow?id=776ef2fa51094ba6, https://usegalaxy.org.au/published/workflow?id=03b090e75c61c1c3, https://usegalaxy.org.au/published/workflow?id=143d4cc586aff5a0, https://usegalaxy.org.au/published/workflow?id=22bd45f60c60a338, https://usegalaxy.org.au/published/workflow?id=24f0d1f7a28756b5, https://usegalaxy.org.au/published/workflow?id=25d52afddaa3451b, https://usegalaxy.org.au/published/workflow?id=2cb16d321a804b1a, https://usegalaxy.org.au/published/workflow?id=2e29b21cdec7d0cc, https://usegalaxy.org.au/published/workflow?id=35f39173f0ac9179, https://usegalaxy.org.au/published/workflow?id=3ac73a2f00423326, https://usegalaxy.org.au/published/workflow?id=3efbc617883dd1af, https://usegalaxy.org.au/published/workflow?id=4871f853090e8be8, https://usegalaxy.org.au/published/workflow?id=4ff39e947c470103, https://usegalaxy.org.au/published/workflow?id=5169ba49122ad39a, https://usegalaxy.org.au/published/workflow?id=557601292ea0f6f3, https://usegalaxy.org.au/published/workflow?id=69d91340fc7effa2, https://usegalaxy.org.au/published/workflow?id=7c343e5b55474d19, https://usegalaxy.org.au/published/workflow?id=8ded019070c6687b, https://usegalaxy.org.au/published/workflow?id=8fe84de499e96485, https://usegalaxy.org.au/published/workflow?id=9c60b83feaa22415, https://usegalaxy.org.au/published/workflow?id=a804d976d351f2fc, https://usegalaxy.org.au/published/workflow?id=b164468b137b73b9, https://usegalaxy.org.au/published/workflow?id=b5403e4fff2961f9, https://usegalaxy.org.au/published/workflow?id=b59fada651043735, https://usegalaxy.org.au/published/workflow?id=b7aa22291322e505, https://usegalaxy.org.au/published/workflow?id=bf80ef6e5f37deb7, https://usegalaxy.org.au/published/workflow?id=c9b7de574fb2f2d5, https://usegalaxy.org.au/published/workflow?id=d69a765cfc82a399, https://usegalaxy.org.au/published/workflow?id=d9571b3dfef34e5e, https://usegalaxy.org.au/published/workflow?id=e7c5dfc703f66403, https://usegalaxy.org.au/published/workflow?id=f9719e48b907f35c, https://usegalaxy.org.au/published/workflow?id=fc3c83b11f527977, https://usegalaxy.org.au/published/workflow?id=fc664a96d8099652, https://usegalaxy.org/published/workflow?id=00381bbae3fd7e78, https://usegalaxy.org/published/workflow?id=00596e862edb23f2, https://usegalaxy.org/published/workflow?id=03ce6eb4d9ee4309, https://usegalaxy.org/published/workflow?id=21161075002d6778, https://usegalaxy.org/published/workflow?id=4458439388155d92, https://usegalaxy.org/published/workflow?id=547f86db467bf620, https://usegalaxy.org/published/workflow?id=562bcaa9b4d2e8a4, https://usegalaxy.org/published/workflow?id=574e42683dc3961b, https://usegalaxy.org/published/workflow?id=69d8863d40524120, https://usegalaxy.org/published/workflow?id=7dd6e4150bb81ad1, https://usegalaxy.org/published/workflow?id=80ac6adead9f97d8, https://usegalaxy.org/published/workflow?id=a8b9252fad5fe06a, https://usegalaxy.org/published/workflow?id=b7817a12dbee583c, https://usegalaxy.org/published/workflow?id=bff206b4625375bc, https://usegalaxy.org/published/workflow?id=c3ea8fa3bb202dc6, https://usegalaxy.org/published/workflow?id=cfa9e6e722b086c8, https://usegalaxy.org/published/workflow?id=e12324f5e3b930fd, https://workflowhub.eu/workflows/1061?version=1, https://workflowhub.eu/workflows/1477?version=1, https://workflowhub.eu/workflows/1492?version=2, https://workflowhub.eu/workflows/1585?version=1, https://workflowhub.eu/workflows/1612?version=1, https://workflowhub.eu/workflows/1613?version=2, https://workflowhub.eu/workflows/1616?version=1, https://workflowhub.eu/workflows/1617?version=1, https://workflowhub.eu/workflows/222?version=1, https://workflowhub.eu/workflows/406?version=1, https://workflowhub.eu/workflows/521?version=1, https://workflowhub.eu/workflows/697?version=1</td>
      <td>assembly/assembly-with-preprocessing, assembly/chloroplast-assembly, assembly/largegenome, assembly/mrsa-nanopore, microbiome/pathogen-detection-from-nanopore-foodborne-data, microbiome/plasmid-metagenomics-nanopore, sequence-analysis/quality-control</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>nanopolishcomp</td>
      <td>nanopolishcomp_eventaligncollapse, nanopolishcomp_freqmethcalculate</td>
      <td>NanopolishComp contains 2 modules. Eventalign_collapse collapses the raw file generated by nanopolish eventalign by kmers rather than by event. Freq_meth_calculate methylation frequency at genomic CpG sites from the output of nanopolish call-methylation.</td>
      <td>2020-04-27</td>
      <td>https://a-slide.github.io/NanopolishComp</td>
      <td>0.6.11</td>
      <td>nanopolishcomp</td>
      <td>0.6.12</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Methylation analysis, Collapsing methods</td>
      <td>Methylation analysis, Collapsing methods</td>
      <td>Sequence analysis, Sequencing, Genetic variation</td>
      <td>Sequence analysis, Sequencing, Genetic variation</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/nanopolishcomp</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/nanopolishcomp</td>
      <td>nanopolishcomp</td>
      <td>NanopolishComp</td>
      <td>NanopolishComp is a Python3 package for downstream analyses of Nanopolish output files.It is a companion package for Nanopolish.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>36</td>
      <td>38</td>
      <td>709</td>
      <td>1129</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1130</td>
      <td>710</td>
      <td>39</td>
      <td>37</td>
      <td>https://usegalaxy.eu/published/workflow?id=77db45ad90b4033d</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>ncbi_blast_plus</td>
      <td>blastxml_to_tabular, get_species_taxids, ncbi_blastdbcmd_info, ncbi_blastdbcmd_wrapper, ncbi_blastn_wrapper, ncbi_blastp_wrapper, ncbi_blastx_wrapper, ncbi_convert2blastmask_wrapper, ncbi_deltablast_wrapper, ncbi_dustmasker_wrapper, ncbi_makeblastdb, ncbi_makeprofiledb, ncbi_psiblast_wrapper, ncbi_rpsblast_wrapper, ncbi_rpstblastn_wrapper, ncbi_segmasker_wrapper, ncbi_tblastn_wrapper, ncbi_tblastx_wrapper</td>
      <td>NCBI BLAST+</td>
      <td>2013-09-23</td>
      <td>https://blast.ncbi.nlm.nih.gov/</td>
      <td>2.16.0</td>
      <td>blast</td>
      <td>2.17.0</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>devteam</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/ncbi_blast_plus</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/ncbi_blast_plus</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>16</td>
      <td>16</td>
      <td>16</td>
      <td>16</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>16</td>
      <td>16</td>
      <td>16</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>16</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>16</td>
      <td>15</td>
      <td>16</td>
      <td>16</td>
      <td>15</td>
      <td>0</td>
      <td>4564</td>
      <td>5012</td>
      <td>608297</td>
      <td>687163</td>
      <td>5091</td>
      <td>6262</td>
      <td>690841</td>
      <td>770815</td>
      <td>1316</td>
      <td>1731</td>
      <td>262398</td>
      <td>287817</td>
      <td>159</td>
      <td>164</td>
      <td>12993</td>
      <td>13154</td>
      <td>1758949</td>
      <td>1574529</td>
      <td>13169</td>
      <td>11130</td>
      <td>https://usegalaxy.eu/published/workflow?id=00e246ea459b0991, https://usegalaxy.eu/published/workflow?id=012762cb4acb068b, https://usegalaxy.eu/published/workflow?id=013cc98f55b01d5d, https://usegalaxy.eu/published/workflow?id=01d42b604ab92d49, https://usegalaxy.eu/published/workflow?id=08f41680101a1a45, https://usegalaxy.eu/published/workflow?id=0ab3e47ebd3b520e, https://usegalaxy.eu/published/workflow?id=0aee7ffa5f6384bd, https://usegalaxy.eu/published/workflow?id=0b9071d5dfa38559, https://usegalaxy.eu/published/workflow?id=0f044c9f62773a88, https://usegalaxy.eu/published/workflow?id=10150b1a188f6d07, https://usegalaxy.eu/published/workflow?id=12a018217ee146ca, https://usegalaxy.eu/published/workflow?id=152223c8aed27357, https://usegalaxy.eu/published/workflow?id=17f73aba442315e6, https://usegalaxy.eu/published/workflow?id=1860e02bcb894a64, https://usegalaxy.eu/published/workflow?id=1b43449ed8481892, https://usegalaxy.eu/published/workflow?id=2497292e98937a5f, https://usegalaxy.eu/published/workflow?id=24eaa682897379c8, https://usegalaxy.eu/published/workflow?id=28af61ead68e0976, https://usegalaxy.eu/published/workflow?id=290bc4120c2c5078, https://usegalaxy.eu/published/workflow?id=2addb042071557e3, https://usegalaxy.eu/published/workflow?id=2c2f0b4312bae4f4, https://usegalaxy.eu/published/workflow?id=2cec6a39cd496f09, https://usegalaxy.eu/published/workflow?id=3069c17b52decab4, https://usegalaxy.eu/published/workflow?id=33312e5643279e8a, https://usegalaxy.eu/published/workflow?id=33a052f9a3d5551d, https://usegalaxy.eu/published/workflow?id=3a178ff6884a5d6d, https://usegalaxy.eu/published/workflow?id=3dec7c9b02c3d1ee, https://usegalaxy.eu/published/workflow?id=3e0246389fb70340, https://usegalaxy.eu/published/workflow?id=44189803f223acd1, https://usegalaxy.eu/published/workflow?id=4eb9dfe91fc2dc67, https://usegalaxy.eu/published/workflow?id=50d85103f2275d38, https://usegalaxy.eu/published/workflow?id=513389e1a8c6bba9, https://usegalaxy.eu/published/workflow?id=52cb9bacc26b013e, https://usegalaxy.eu/published/workflow?id=559f079f6510aa24, https://usegalaxy.eu/published/workflow?id=58db3e1996a8f093, https://usegalaxy.eu/published/workflow?id=5be158931abca947, https://usegalaxy.eu/published/workflow?id=6201ec86d380aef1, https://usegalaxy.eu/published/workflow?id=6d3332794ceca465, https://usegalaxy.eu/published/workflow?id=6da171871d4f3fd5, https://usegalaxy.eu/published/workflow?id=79eb28066236e836, https://usegalaxy.eu/published/workflow?id=7c80cd57b2de54d4, https://usegalaxy.eu/published/workflow?id=86465932556bc96c, https://usegalaxy.eu/published/workflow?id=8a14cda6140acab7, https://usegalaxy.eu/published/workflow?id=8c7e972020799d57, https://usegalaxy.eu/published/workflow?id=8d64614d1971065a, https://usegalaxy.eu/published/workflow?id=8f747dbff3216993, https://usegalaxy.eu/published/workflow?id=933ab5e7fe13a8e0, https://usegalaxy.eu/published/workflow?id=9588311136ac8794, https://usegalaxy.eu/published/workflow?id=974cc681d07ec788, https://usegalaxy.eu/published/workflow?id=9a60ecf98d4f8134, https://usegalaxy.eu/published/workflow?id=9a95556ec171d405, https://usegalaxy.eu/published/workflow?id=ab646c6ab6de7061, https://usegalaxy.eu/published/workflow?id=afad9a17a80bb26d, https://usegalaxy.eu/published/workflow?id=bebcbd6f0ffa55c5, https://usegalaxy.eu/published/workflow?id=c36560c74416fbbf, https://usegalaxy.eu/published/workflow?id=c4fc3adf8f7ff2bc, https://usegalaxy.eu/published/workflow?id=c55388a8fbfd7381, https://usegalaxy.eu/published/workflow?id=c7f34c67c9054234, https://usegalaxy.eu/published/workflow?id=d07fe8b5a045c176, https://usegalaxy.eu/published/workflow?id=d22dac11eba8c5a3, https://usegalaxy.eu/published/workflow?id=df7d9943423549df, https://usegalaxy.eu/published/workflow?id=e035ac55551b2744, https://usegalaxy.eu/published/workflow?id=e14b81a8c0b1b7a4, https://usegalaxy.eu/published/workflow?id=e2d0d5340975c87f, https://usegalaxy.eu/published/workflow?id=eb5ab8bc82569eee, https://usegalaxy.eu/published/workflow?id=ec6b2c926fffab5f, https://usegalaxy.eu/published/workflow?id=edea08c943ff9c5e, https://usegalaxy.eu/published/workflow?id=edf7e2128f5aef8d, https://usegalaxy.eu/published/workflow?id=f62d7c0c7cc57eb6, https://usegalaxy.eu/published/workflow?id=f946efa49bd41a43, https://usegalaxy.eu/published/workflow?id=fc0b9efa7d2e61e6, https://usegalaxy.fr/published/workflow?id=41264142d3c00016, https://usegalaxy.fr/published/workflow?id=8b1678200a1d157a, https://usegalaxy.org.au/published/workflow?id=0edb9c8b767fa74b, https://usegalaxy.org.au/published/workflow?id=29ce205d98faf47e, https://usegalaxy.org.au/published/workflow?id=3d47e0de658b3304, https://usegalaxy.org.au/published/workflow?id=4047f9f242542716, https://usegalaxy.org.au/published/workflow?id=470eb26144325590, https://usegalaxy.org.au/published/workflow?id=5cf5334877cc1297, https://usegalaxy.org.au/published/workflow?id=5fb44f257e445f56, https://usegalaxy.org.au/published/workflow?id=762791e094457bf2, https://usegalaxy.org.au/published/workflow?id=7f37c5366bce5489, https://usegalaxy.org.au/published/workflow?id=8034bd0690b4cbe8, https://usegalaxy.org.au/published/workflow?id=83441e3de699d157, https://usegalaxy.org.au/published/workflow?id=897a1255cef5c310, https://usegalaxy.org.au/published/workflow?id=94c7fdcbbc7b428a, https://usegalaxy.org.au/published/workflow?id=97e55a900db3863b, https://usegalaxy.org.au/published/workflow?id=c049ae8f729b2d7e, https://usegalaxy.org.au/published/workflow?id=c92fead7c30c7497, https://usegalaxy.org.au/published/workflow?id=d2dcd3b54e5082a6, https://usegalaxy.org.au/published/workflow?id=d417e920ff97e4c1, https://usegalaxy.org.au/published/workflow?id=d720c98f7227b6d5, https://usegalaxy.org.au/published/workflow?id=dccc8fb49d61ac33, https://usegalaxy.org/published/workflow?id=1473cdcbd664d147, https://usegalaxy.org/published/workflow?id=1b80302437db5937, https://usegalaxy.org/published/workflow?id=529b410bd03b66aa, https://usegalaxy.org/published/workflow?id=5c259623a010fe77, https://usegalaxy.org/published/workflow?id=5d1cc6450f699345, https://usegalaxy.org/published/workflow?id=5e38c7b36e4918ad, https://usegalaxy.org/published/workflow?id=605160c81d098703, https://usegalaxy.org/published/workflow?id=7c9a55db0b0eb178, https://usegalaxy.org/published/workflow?id=7d005f30cd4853c4, https://usegalaxy.org/published/workflow?id=7fc8b2398b6f093f, https://usegalaxy.org/published/workflow?id=8ecc114eb9097855, https://usegalaxy.org/published/workflow?id=927b8f90ad9603a7, https://usegalaxy.org/published/workflow?id=a67afbea4b2eb26b, https://usegalaxy.org/published/workflow?id=b7fe3873ee5e3a02, https://usegalaxy.org/published/workflow?id=c8f9e048ddae7d03, https://usegalaxy.org/published/workflow?id=cace9f1359874352, https://usegalaxy.org/published/workflow?id=d2c408df40dc311b, https://usegalaxy.org/published/workflow?id=ddf7a594b9609d68, https://usegalaxy.org/published/workflow?id=e1dcdc125122f561, https://usegalaxy.org/published/workflow?id=e887adf3afc3253d, https://usegalaxy.org/published/workflow?id=ea206593698adbfd, https://usegalaxy.org/published/workflow?id=fc2b48600853f3ea, https://workflowhub.eu/workflows/100?version=1, https://workflowhub.eu/workflows/102?version=1, https://workflowhub.eu/workflows/1416?version=1, https://workflowhub.eu/workflows/1467?version=1, https://workflowhub.eu/workflows/1594?version=1, https://workflowhub.eu/workflows/1630?version=1, https://workflowhub.eu/workflows/1635?version=1, https://workflowhub.eu/workflows/1702?version=1, https://workflowhub.eu/workflows/1723?version=1, https://workflowhub.eu/workflows/1791?version=1, https://workflowhub.eu/workflows/2038?version=1, https://workflowhub.eu/workflows/37?version=1, https://workflowhub.eu/workflows/38?version=1, https://workflowhub.eu/workflows/39?version=1, https://workflowhub.eu/workflows/40?version=1, https://workflowhub.eu/workflows/645?version=16, https://workflowhub.eu/workflows/655?version=1</td>
      <td>assembly/assembly-decontamination, data-science/online-resources-gene, ecology/ENA_Biodiv_submission, ecology/Obitools-metabarcoding, ecology/eDNA-taxonomic-analysis, genome-annotation/functional, proteomics/neoantigen-peptide-verification, proteomics/proteogenomics-novel-peptide-analysis, sequence-analysis/Manage_AB1_Sanger, sequence-analysis/ncbi-blast-against-the-madland, sequence-analysis/viral_primer_design</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>ncbi_egapx</td>
      <td>ncbi_egapx</td>
      <td>Eukaryotic Genome Annotation Pipeline - External (EGAPx)</td>
      <td>2024-08-19</td>
      <td>https://github.com/ncbi/egapx</td>
      <td>0.4.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>Genome annotation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>richard-burhans</td>
      <td>https://github.com/richard-burhans/galaxytools/tree/main/tools/ncbi_egapx</td>
      <td>https://github.com/richard-burhans/galaxytools/tree/main/tools/ncbi_egapx</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>7</td>
      <td>7</td>
      <td>49</td>
      <td>49</td>
      <td>1037</td>
      <td>1037</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1044</td>
      <td>1044</td>
      <td>51</td>
      <td>51</td>
      <td>https://usegalaxy.org/published/workflow?id=7251c6e0dad8f927, https://usegalaxy.org/published/workflow?id=951575e9a352301c, https://usegalaxy.org/published/workflow?id=ce488732866ba48f, https://usegalaxy.org/published/workflow?id=e93a0522ec3a35e6</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>ncbi_fcs_gx</td>
      <td>ncbi_fcs_gx</td>
      <td>FCS-GX detects contamination from foreign organisms in genome sequences using the genome cross-species aligner (GX).</td>
      <td>2023-09-21</td>
      <td>https://github.com/ncbi/fcs-gx</td>
      <td>0.5.5</td>
      <td>ncbi-fcs-gx</td>
      <td>0.5.5</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Sequence assembly validation, Sequence trimming, Sequence contamination filtering</td>
      <td>Sequence assembly validation, Sequence trimming, Sequence contamination filtering</td>
      <td>Sequence analysis, Sequence assembly</td>
      <td>Sequence analysis, Sequence assembly</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ncbi_fcs_gx</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ncbi_fcs_gx</td>
      <td>ncbi_fcs</td>
      <td>NCBI fcs</td>
      <td>The NCBI Foreign Contamination Screen (FCS) is a tool suite for identifying and removing contaminant sequences in genome assemblies. Contaminants are defined as sequences in a dataset that do not originate from the biological source organism and can arise from a variety of environmental and laboratory sources. FCS will help you remove contaminants from genomes before submission to GenBank.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>175</td>
      <td>175</td>
      <td>2259</td>
      <td>2259</td>
      <td>485</td>
      <td>485</td>
      <td>26859</td>
      <td>26859</td>
      <td>27</td>
      <td>27</td>
      <td>955</td>
      <td>955</td>
      <td>1</td>
      <td>1</td>
      <td>9</td>
      <td>9</td>
      <td>30082</td>
      <td>30082</td>
      <td>688</td>
      <td>688</td>
      <td>https://usegalaxy.eu/published/workflow?id=01d42b604ab92d49, https://usegalaxy.eu/published/workflow?id=c36560c74416fbbf, https://usegalaxy.eu/published/workflow?id=e035ac55551b2744, https://usegalaxy.org/published/workflow?id=529b410bd03b66aa, https://usegalaxy.org/published/workflow?id=5d1cc6450f699345, https://usegalaxy.org/published/workflow?id=927b8f90ad9603a7, https://workflowhub.eu/workflows/645?version=16</td>
      <td>sequence-analysis/ncbi-fcs</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>newick_utils</td>
      <td>newick_display</td>
      <td>Perform operations on Newick trees</td>
      <td>2018-10-01</td>
      <td>https://github.com/tjunier/newick_utils</td>
      <td>1.6+galaxy1</td>
      <td>newick_utils</td>
      <td>1.6</td>
      <td>To update</td>
      <td>Visualization, Metagenomics</td>
      <td>Phylogenetic tree generation, Phylogenetic tree analysis, Phylogenetic tree reconstruction</td>
      <td></td>
      <td>Phylogeny, Genomics, Computer science</td>
      <td>Phylogeny, Genomics, Computer science</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/newick_utils</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/newick_utils</td>
      <td>newick_utilities</td>
      <td>Newick Utilities</td>
      <td>The Newick Utilities are a set of command-line tools for processing phylogenetic trees. They can process arbitrarily large amounts of data and do not require user interaction, which makes them suitable for automating phylogeny processing tasks.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1756</td>
      <td>1893</td>
      <td>51682</td>
      <td>53373</td>
      <td>2196</td>
      <td>2332</td>
      <td>18983</td>
      <td>20195</td>
      <td>1114</td>
      <td>1145</td>
      <td>9635</td>
      <td>9912</td>
      <td>48</td>
      <td>49</td>
      <td>310</td>
      <td>312</td>
      <td>83792</td>
      <td>80610</td>
      <td>5419</td>
      <td>5114</td>
      <td>https://usegalaxy.eu/published/workflow?id=36de05d50c703164, https://usegalaxy.eu/published/workflow?id=376119528377a3ae, https://usegalaxy.eu/published/workflow?id=974cc681d07ec788, https://usegalaxy.eu/published/workflow?id=b1a799497c366eef, https://usegalaxy.eu/published/workflow?id=c1e26abe61c69688, https://usegalaxy.eu/published/workflow?id=c67d2e75eeb4fd19, https://usegalaxy.eu/published/workflow?id=c8bc4af6dd3b5d02, https://usegalaxy.eu/published/workflow?id=dadca1020f586338, https://usegalaxy.eu/published/workflow?id=dc57118a95b10a80, https://usegalaxy.eu/published/workflow?id=e16e524c557d7928, https://usegalaxy.eu/published/workflow?id=ec50dd6d56f50db1, https://usegalaxy.eu/published/workflow?id=f21d6a40a10a8ae8, https://usegalaxy.eu/published/workflow?id=f6a763951d815944, https://usegalaxy.org.au/published/workflow?id=2717e90438c7fb8c, https://usegalaxy.org.au/published/workflow?id=3fd2195b6928d5e0, https://usegalaxy.org.au/published/workflow?id=a01c3004bcef9dd3, https://usegalaxy.org.au/published/workflow?id=a52609119b30e64d, https://usegalaxy.org.au/published/workflow?id=b60922a253df6654, https://usegalaxy.org.au/published/workflow?id=eda40b58616a0fe4, https://usegalaxy.org/published/workflow?id=005924662f68a480, https://usegalaxy.org/published/workflow?id=0596162a499b4074, https://usegalaxy.org/published/workflow?id=2d3063882d8239ff, https://usegalaxy.org/published/workflow?id=3ef08553982f6ffc, https://usegalaxy.org/published/workflow?id=603e5c61c57bcf3f, https://usegalaxy.org/published/workflow?id=6a1eb7e7bececba1, https://usegalaxy.org/published/workflow?id=6f090b2c259847d2, https://usegalaxy.org/published/workflow?id=775335812a6b7a61, https://usegalaxy.org/published/workflow?id=80565301ada0d3eb, https://usegalaxy.org/published/workflow?id=9c4038fa15af6995, https://usegalaxy.org/published/workflow?id=bc01b8b203ecc754, https://usegalaxy.org/published/workflow?id=d653eaefcf6b4697, https://workflowhub.eu/workflows/1060?version=1, https://workflowhub.eu/workflows/1418?version=1, https://workflowhub.eu/workflows/1465?version=1, https://workflowhub.eu/workflows/1487?version=2, https://workflowhub.eu/workflows/653?version=1</td>
      <td>evolution/abc_intro_phylo, microbiome/mothur-miseq-sop, microbiome/mothur-miseq-sop-short, microbiome/pathogen-detection-from-nanopore-foodborne-data</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>nextclade</td>
      <td>nextalign, nextclade</td>
      <td>Identify differences between your sequences and a reference sequence used by Nextstrain</td>
      <td>2021-03-29</td>
      <td>https://github.com/nextstrain/nextclade</td>
      <td>2.7.0</td>
      <td>nextalign</td>
      <td>2.14.0</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Methylation analysis, Variant calling</td>
      <td>Methylation analysis, Variant calling</td>
      <td>Genomics, Sequence analysis, Cladistics</td>
      <td>Genomics, Cladistics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/nextclade</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/nextclade</td>
      <td>nextclade</td>
      <td>Nextclade</td>
      <td>Nextclade is an open-source project for viral genome alignment, mutation calling, clade assignment, quality checks and phylogenetic placement.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>373</td>
      <td>373</td>
      <td>4718</td>
      <td>4718</td>
      <td>309</td>
      <td>309</td>
      <td>11632</td>
      <td>11632</td>
      <td>258</td>
      <td>258</td>
      <td>2460</td>
      <td>2460</td>
      <td>11</td>
      <td>11</td>
      <td>723</td>
      <td>723</td>
      <td>19533</td>
      <td>19533</td>
      <td>951</td>
      <td>951</td>
      <td>https://usegalaxy.eu/published/workflow?id=0189bab68f828d86, https://usegalaxy.eu/published/workflow?id=1d1ebdeddf23bdb9, https://usegalaxy.eu/published/workflow?id=3f046b416b10b871, https://usegalaxy.eu/published/workflow?id=4c75ab0be3c56494, https://usegalaxy.eu/published/workflow?id=5c15f251642bfa1f, https://usegalaxy.eu/published/workflow?id=74f19aff1b9cdd6b, https://usegalaxy.eu/published/workflow?id=9feebfc6c34da287, https://usegalaxy.fr/published/workflow?id=2345fad7dac13daa, https://usegalaxy.fr/published/workflow?id=591c57173e259ca6, https://usegalaxy.fr/published/workflow?id=e4735d214a68b790, https://usegalaxy.fr/published/workflow?id=feb57548b3ce914d, https://usegalaxy.org.au/published/workflow?id=7563b3ef697a69fd, https://usegalaxy.org.au/published/workflow?id=af12a3ec85859563, https://usegalaxy.org/published/workflow?id=0fa2ce12c2874b1d, https://usegalaxy.org/published/workflow?id=426e3951dac0704e, https://usegalaxy.org/published/workflow?id=4b41a8247f52c463, https://usegalaxy.org/published/workflow?id=8b81b9a3b49bbf98, https://usegalaxy.org/published/workflow?id=d96bd5c9fb553f2c, https://workflowhub.eu/workflows/155?version=6, https://workflowhub.eu/workflows/520?version=1</td>
      <td>variant-analysis/sars-cov-2-variant-discovery</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>nonpareil</td>
      <td>nonpareil</td>
      <td>Estimate average coverage in metagenomic datasets</td>
      <td>2017-11-03</td>
      <td>http://nonpareil.readthedocs.io</td>
      <td>3.5.5</td>
      <td>nonpareil</td>
      <td>3.5.5</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Operation</td>
      <td>Operation</td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/nonpareil</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/nonpareil</td>
      <td>nonpareil</td>
      <td>nonpareil</td>
      <td>Estimate metagenomic coverage and sequence diversity</td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>16</td>
      <td>19</td>
      <td>105</td>
      <td>195</td>
      <td>23</td>
      <td>26</td>
      <td>122</td>
      <td>168</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>363</td>
      <td>227</td>
      <td>45</td>
      <td>39</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>nugen_nudup</td>
      <td>nugen_nudup</td>
      <td>Marks/removes PCR introduced duplicate molecules based on the molecular tagging technology used in NuGEN products.</td>
      <td>2016-11-18</td>
      <td>https://github.com/tecangenomics/nudup</td>
      <td>2.3.3</td>
      <td>nudup</td>
      <td>2.3.3</td>
      <td>Up-to-date</td>
      <td>SAM, Metagenomics, Sequence Analysis, Transcriptomics</td>
      <td>Duplication detection, Sequence analysis</td>
      <td>Duplication detection, Sequence analysis</td>
      <td>Sequencing, Genomics</td>
      <td>Sequencing, Genomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/nugen_nudup</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/nugen_nudup</td>
      <td>nudup</td>
      <td>NuDup</td>
      <td>Marks/removes duplicate molecules based on the molecular tagging technology used in Tecan products.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>obitools</td>
      <td>obi_illumina_pairend, obi_ngsfilter, obi_annotate, obi_clean, obi_convert, obi_grep, obi_sort, obi_stat, obi_tab, obi_uniq</td>
      <td>OBITools is a set of programs developed to simplify the manipulation of sequence files</td>
      <td>2017-03-22</td>
      <td>http://metabarcoding.org/obitools</td>
      <td>1.2.13</td>
      <td>obitools</td>
      <td>1.2.13</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Sequence analysis, Sequence analysis</td>
      <td>Sequence analysis, Sequence analysis</td>
      <td>Sequence analysis, DNA, Sequencing</td>
      <td>Sequence analysis, DNA, Sequencing</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/obitools</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/obitools</td>
      <td>obitools</td>
      <td>OBITools</td>
      <td>Set of python programs developed to simplify the manipulation of sequence files. They were mainly designed to help us for analyzing Next Generation Sequencer outputs (454 or Illumina) in the context of DNA Metabarcoding.</td>
      <td></td>
      <td>0</td>
      <td>10</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>483</td>
      <td>493</td>
      <td>45330</td>
      <td>45705</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>107</td>
      <td>107</td>
      <td>5011</td>
      <td>5011</td>
      <td>25</td>
      <td>25</td>
      <td>1398</td>
      <td>1398</td>
      <td>52114</td>
      <td>51739</td>
      <td>625</td>
      <td>615</td>
      <td>https://usegalaxy.eu/published/workflow?id=513389e1a8c6bba9, https://usegalaxy.eu/published/workflow?id=afad9a17a80bb26d, https://usegalaxy.eu/published/workflow?id=edf7e2128f5aef8d, https://usegalaxy.org.au/published/workflow?id=470eb26144325590, https://workflowhub.eu/workflows/1702?version=1, https://workflowhub.eu/workflows/655?version=1</td>
      <td>ecology/Obitools-metabarcoding</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>omark</td>
      <td>omark</td>
      <td>Proteome quality assessment software</td>
      <td>2023-11-15</td>
      <td>https://github.com/DessimozLab/OMArk</td>
      <td>0.3.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Sequence assembly validation, Differential protein expression profiling</td>
      <td>Sequence assembly validation, Differential protein expression profiling</td>
      <td>Proteomics, Sequence analysis, Statistics and probability</td>
      <td>Proteomics, Sequence analysis, Statistics and probability</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/omark</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/omark</td>
      <td>omark</td>
      <td>OMArk</td>
      <td>OMArk is a software for proteome (protein-coding gene repertoire) quality assessment. It provides measures of proteome completeness, characterizes the consistency of all protein coding genes with regard to their homologs, and identifies the presence of contamination from other species. OMArk relies on the OMA orthology database, from which it exploits orthology relationships, and on the OMAmer software for fast placement of all proteins into gene families.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>89</td>
      <td>89</td>
      <td>707</td>
      <td>707</td>
      <td>17</td>
      <td>17</td>
      <td>44</td>
      <td>44</td>
      <td>6</td>
      <td>6</td>
      <td>11</td>
      <td>11</td>
      <td>15</td>
      <td>15</td>
      <td>347</td>
      <td>347</td>
      <td>1109</td>
      <td>1109</td>
      <td>127</td>
      <td>127</td>
      <td>https://usegalaxy.eu/published/workflow?id=31bc2a44fb89aa89, https://usegalaxy.eu/published/workflow?id=327beb1edda305ff, https://usegalaxy.eu/published/workflow?id=3d604dad4f1f5b3d, https://usegalaxy.eu/published/workflow?id=4077e77b6af919c4, https://usegalaxy.eu/published/workflow?id=6550f72070b24082, https://usegalaxy.eu/published/workflow?id=8976041ef89e4548, https://usegalaxy.org.au/published/workflow?id=dd9272826e6dd98d, https://usegalaxy.org.au/published/workflow?id=e263fcb6c98e231b, https://usegalaxy.org.au/published/workflow?id=ec7efff3d3249177, https://usegalaxy.org/published/workflow?id=4605309321799e9b, https://usegalaxy.org/published/workflow?id=80a7124c40438cc9, https://usegalaxy.org/published/workflow?id=f4111a169255535c, https://workflowhub.eu/workflows/1096?version=2, https://workflowhub.eu/workflows/1255?version=3, https://workflowhub.eu/workflows/1500?version=1, https://workflowhub.eu/workflows/1746?version=1, https://workflowhub.eu/workflows/2029?version=1</td>
      <td>genome-annotation/braker3, genome-annotation/comparison-braker-helixer-annotation, genome-annotation/helixer</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>openms</td>
      <td>AccurateMassSearch, AdditiveSeries, BaselineFilter, CVInspector, CompNovo, CompNovoCID, ConsensusID, ConsensusMapNormalizer, ConvertTSVToTraML, ConvertTraMLToTSV, DTAExtractor, DeMeanderize, Decharger, DecoyDatabase, Digestor, DigestorMotif, EICExtractor, ERPairFinder, ExternalCalibration, FFEval, FalseDiscoveryRate, FeatureFinderCentroided, FeatureFinderIdentification, FeatureFinderIsotopeWavelet, FeatureFinderMRM, FeatureFinderMetabo, FeatureFinderMultiplex, FeatureFinderSuperHirn, FeatureLinkerLabeled, FeatureLinkerUnlabeled, FeatureLinkerUnlabeledQT, FidoAdapter, FileConverter, FileFilter, FileInfo, FileMerger, FuzzyDiff, HighResPrecursorMassCorrector, IDConflictResolver, IDDecoyProbability, IDExtractor, IDFileConverter, IDFilter, IDMapper, IDMassAccuracy, IDMerger, IDPosteriorErrorProbability, IDRTCalibration, IDRipper, IDScoreSwitcher, IDSplitter, ITRAQAnalyzer, InclusionExclusionListCreator, InspectAdapter, InternalCalibration, IsobaricAnalyzer, LabeledEval, LowMemPeakPickerHiRes, LowMemPeakPickerHiRes_RandomAccess, LuciphorAdapter, MRMMapper, MRMPairFinder, MRMTransitionGroupPicker, MSGFPlusAdapter, MSSimulator, MapAlignmentEvaluation, MapNormalizer, MapRTTransformer, MapStatistics, MascotAdapter, MascotAdapterOnline, MassCalculator, MassTraceExtractor, MetaProSIP, MetaboliteSpectralMatcher, MultiplexResolver, MzMLSplitter, MzTabExporter, NoiseFilterGaussian, NoiseFilterSGolay, OpenSwathAnalyzer, OpenSwathAssayGenerator, OpenSwathChromatogramExtractor, OpenSwathConfidenceScoring, OpenSwathDIAPreScoring, OpenSwathDecoyGenerator, OpenSwathFeatureXMLToTSV, OpenSwathFileSplitter, OpenSwathMzMLFileCacher, OpenSwathRTNormalizer, OpenSwathRewriteToFeatureXML, OpenSwathWorkflow, PTModel, PTPredict, PeakPickerHiRes, PeakPickerIterative, PeakPickerWavelet, PepNovoAdapter, PeptideIndexer, PhosphoScoring, PrecursorIonSelector, PrecursorMassCorrector, ProteinInference, ProteinQuantifier, ProteinResolver, QCCalculator, QCEmbedder, QCExporter, QCExtractor, QCImporter, QCMerger, QCShrinker, RNPxl, RNPxlXICFilter, RTEvaluation, RTModel, RTPredict, SemanticValidator, SequenceCoverageCalculator, SimpleSearchEngine, SpecLibCreator, SpectraFilterBernNorm, SpectraFilterMarkerMower, SpectraFilterNLargest, SpectraFilterNormalizer, SpectraFilterParentPeakMower, SpectraFilterScaler, SpectraFilterSqrtMower, SpectraFilterThresholdMower, SpectraFilterWindowMower, SpectraMerger, SvmTheoreticalSpectrumGeneratorTrainer, TICCalculator, TMTAnalyzer, TOFCalibration, TextExporter, TopPerc, TransformationEvaluation, XMLValidator, XTandemAdapter</td>
      <td>OpenMS in version 2.1.</td>
      <td>2015-02-28</td>
      <td></td>
      <td>2.1.0</td>
      <td>openms</td>
      <td>3.5.0</td>
      <td>To update</td>
      <td>Proteomics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/openms</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/openms</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>7</td>
      <td>34</td>
      <td>135</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>27</td>
      <td>135</td>
      <td>134</td>
      <td>0</td>
      <td>583</td>
      <td>698</td>
      <td>30109</td>
      <td>114383</td>
      <td>10</td>
      <td>10</td>
      <td>417</td>
      <td>417</td>
      <td>33</td>
      <td>34</td>
      <td>1321</td>
      <td>1325</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>116125</td>
      <td>31847</td>
      <td>742</td>
      <td>626</td>
      <td>https://usegalaxy.eu/published/workflow?id=41d096891bada138, https://usegalaxy.eu/published/workflow?id=68edbb8eec38ad00, https://usegalaxy.eu/published/workflow?id=7463b26aab6a965d, https://usegalaxy.eu/published/workflow?id=86d50bbaaa3936dc, https://usegalaxy.eu/published/workflow?id=a2cdadcd549c9c3c, https://usegalaxy.eu/published/workflow?id=afb138fc03ca61a8, https://usegalaxy.eu/published/workflow?id=d55df785d4c71876, https://usegalaxy.eu/published/workflow?id=da10c475c83c5469, https://usegalaxy.eu/published/workflow?id=e9e41c23261d4ac5, https://usegalaxy.eu/published/workflow?id=eaf4d8f8fb3e683d, https://usegalaxy.org.au/published/workflow?id=03efeb2948494130, https://usegalaxy.org.au/published/workflow?id=049c52f685984dec, https://usegalaxy.org.au/published/workflow?id=236e0f306916149b, https://usegalaxy.org.au/published/workflow?id=33922db405a97cbb, https://usegalaxy.org.au/published/workflow?id=59c9a6efaf57ce35, https://usegalaxy.org.au/published/workflow?id=65a659982f955743, https://usegalaxy.org.au/published/workflow?id=7cddb345877f08bf, https://usegalaxy.org.au/published/workflow?id=88fa3e250db2d70e, https://usegalaxy.org.au/published/workflow?id=8c874ea77827448c, https://usegalaxy.org.au/published/workflow?id=c2f449e69d06ccf5, https://usegalaxy.org.au/published/workflow?id=da79aa2dfd6b34bc, https://usegalaxy.org/published/workflow?id=0775ff66d1bc04b5, https://usegalaxy.org/published/workflow?id=9865fee6c68fc201, https://usegalaxy.org/published/workflow?id=f9048d9678e21f36, https://workflowhub.eu/workflows/1394?version=1, https://workflowhub.eu/workflows/1399?version=1, https://workflowhub.eu/workflows/1421?version=1, https://workflowhub.eu/workflows/1424?version=1, https://workflowhub.eu/workflows/1434?version=1, https://workflowhub.eu/workflows/1438?version=1, https://workflowhub.eu/workflows/1446?version=1, https://workflowhub.eu/workflows/1449?version=1, https://workflowhub.eu/workflows/1452?version=1, https://workflowhub.eu/workflows/1463?version=1, https://workflowhub.eu/workflows/613?version=2</td>
      <td>proteomics/DIA_Analysis_OSW, proteomics/DIA_lib_OSW, proteomics/database-handling, proteomics/ntails, proteomics/protein-id-oms, proteomics/protein-id-sg-ps, proteomics/protein-quant-sil</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>orfipy</td>
      <td>orfipy</td>
      <td>Galaxy wrapper for ORFIPY</td>
      <td>2022-04-07</td>
      <td>https://github.com/urmi-21/orfipy</td>
      <td>0.0.4</td>
      <td>orfipy</td>
      <td>0.0.4</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Coding region prediction, Database search, Transcriptome assembly, De-novo assembly</td>
      <td>Coding region prediction, Database search, Transcriptome assembly, De-novo assembly</td>
      <td>Computer science, RNA-Seq, Transcriptomics, Small molecules</td>
      <td>Computer science, RNA-Seq, Transcriptomics, Small molecules</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/orfipy</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/orfipy</td>
      <td>orfipy</td>
      <td>orfipy</td>
      <td>A fast and flexible tool for extracting ORFs.orfipy is a tool written in python/cython to extract ORFs in extremely an fast and flexible manner. Other popular ORF searching tools are OrfM and getorf. Compared to OrfM and getorf, orfipy provides the most options to fine tune ORF searches. orfipy uses multiple CPU cores and is particularly faster for data containing multiple smaller fasta sequences such as de-novo transcriptome assemblies. Please read the preprint here.</td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>172</td>
      <td>172</td>
      <td>1509</td>
      <td>1509</td>
      <td>343</td>
      <td>343</td>
      <td>2776</td>
      <td>2776</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4285</td>
      <td>4285</td>
      <td>515</td>
      <td>515</td>
      <td>https://usegalaxy.eu/published/workflow?id=30dc526dcfa500df</td>
      <td>genome-annotation/gene-centric</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>orthofinder</td>
      <td>orthofinder_onlygroups</td>
      <td>Accurate inference of orthologous gene groups made easy</td>
      <td>2017-05-22</td>
      <td>https://github.com/davidemms/OrthoFinder</td>
      <td>2.5.5</td>
      <td>orthofinder</td>
      <td>3.1.2</td>
      <td>To update</td>
      <td>Phylogenetics, Sequence Analysis</td>
      <td>Genome comparison, Phylogenetic tree generation (from molecular sequences), Phylogenetic tree analysis, Genome alignment</td>
      <td>Genome comparison, Genome alignment</td>
      <td>Phylogenetics, Phylogenomics, Bioinformatics, Comparative genomics, Sequence analysis</td>
      <td>Phylogenetics, Phylogenomics, Bioinformatics, Comparative genomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/orthofinder</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/orthofinder</td>
      <td>OrthoFinder</td>
      <td>OrthoFinder</td>
      <td>OrthoFinder is a fast, accurate and comprehensive platform for comparative genomics. It finds orthogroups and orthologs, infers rooted gene trees for all orthogroups and identifies all of the gene duplcation events in those gene trees. It also infers a rooted species tree for the species being analysed and maps the gene duplication events from the gene trees to branches in the species tree. OrthoFinder also provides comprehensive statistics for comparative genomic analyses.</td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>801</td>
      <td>801</td>
      <td>3870</td>
      <td>3870</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>133</td>
      <td>133</td>
      <td>1450</td>
      <td>1450</td>
      <td>36</td>
      <td>36</td>
      <td>221</td>
      <td>221</td>
      <td>5541</td>
      <td>5541</td>
      <td>970</td>
      <td>970</td>
      <td>https://usegalaxy.eu/published/workflow?id=d8492b49aed84d4f, https://usegalaxy.eu/published/workflow?id=e5becc6d66b2568c</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>pacu</td>
      <td>pacu_map, pacu_snp</td>
      <td>PACU is a workflow for whole genome sequencing based phylogeny of Illumina and ONT R9/R10 data.</td>
      <td>2024-08-13</td>
      <td>https://github.com/BioinformaticsPlatformWIV-ISP/PACU</td>
      <td>0.0.5</td>
      <td>pacu_snp</td>
      <td>1.0.0</td>
      <td>To update</td>
      <td>Sequence Analysis, Phylogenetics</td>
      <td>Clustering</td>
      <td>Clustering</td>
      <td>Phylogenetics, Sequence analysis</td>
      <td>Phylogenetics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pacu</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pacu</td>
      <td>pacu</td>
      <td>PACU</td>
      <td>PACU is a workflow for whole genome sequencing based phylogeny of Illumina and ONT R9/R10 data.PACU stands for the Prokaryotic Awesome variant Calling Utility and is named after an omnivorous fish (that eats both Illumina and ONT reads).</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>19</td>
      <td>19</td>
      <td>223</td>
      <td>223</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>223</td>
      <td>223</td>
      <td>19</td>
      <td>19</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>pal2nal</td>
      <td>pal2nal</td>
      <td>Wraps PAL2NAL to convert protein alignments and nucleotide FASTA into codon-aware alignments.</td>
      <td>2025-11-09</td>
      <td>http://www.bork.embl.de/pal2nal</td>
      <td></td>
      <td>pal2nal</td>
      <td>14.1</td>
      <td>To update</td>
      <td>Sequence Analysis, Phylogenetics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/georgehe23/tools-iuc/tree/main/tools/pal2nal</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pal2nal</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>pampa</td>
      <td>pampa_communitymetrics, pampa_presabs, pampa_glmcomm, pampa_glmsp, pampa_plotglm</td>
      <td>Tools to compute and analyse biodiversity metrics</td>
      <td>2020-06-06</td>
      <td></td>
      <td>0.0.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>Ecology</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/ColineRoyaux/PAMPA-Galaxy</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/PAMPA</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>4</td>
      <td>0</td>
      <td>50</td>
      <td>53</td>
      <td>1997</td>
      <td>2045</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>26</td>
      <td>26</td>
      <td>1141</td>
      <td>1141</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>3187</td>
      <td>3139</td>
      <td>80</td>
      <td>77</td>
      <td>https://usegalaxy.eu/published/workflow?id=428be4c7d1fd828f, https://usegalaxy.eu/published/workflow?id=7c45c90fffe40aca, https://usegalaxy.eu/published/workflow?id=d33ab07e4cf44f8c, https://workflowhub.eu/workflows/1696?version=1</td>
      <td>ecology/PAMPA-toolsuite-tutorial</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>panaroo</td>
      <td>panaroo</td>
      <td>A Bacterial Pangenome Analysis Pipeline</td>
      <td>2025-02-28</td>
      <td>https://github.com/gtonkinhill/panaroo/releases</td>
      <td>1.6.0</td>
      <td>panaroo</td>
      <td>1.6.0</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>De-novo assembly, Genome assembly, Clustering</td>
      <td>De-novo assembly, Genome assembly, Clustering</td>
      <td>Workflows, Sequence assembly, Gene and protein families, Plant biology, Sequencing</td>
      <td>Workflows, Sequence assembly, Gene and protein families, Plant biology, Sequencing</td>
      <td>galaxy-australia</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/panaroo</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/panaroo</td>
      <td>panaroo</td>
      <td>Panaroo</td>
      <td>Producing Polished Prokaryotic Pangenomes with the Panaroo Pipeline.</td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>67</td>
      <td>67</td>
      <td>297</td>
      <td>297</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>39</td>
      <td>39</td>
      <td>312</td>
      <td>312</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>609</td>
      <td>609</td>
      <td>106</td>
      <td>106</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>panta</td>
      <td>panta</td>
      <td>Efficient inference of large prokaryotic pangenomes with PanTA.</td>
      <td>2025-09-15</td>
      <td>https://github.com/amromics/panta</td>
      <td>1.0.1</td>
      <td>panta</td>
      <td>1.0.1</td>
      <td>Up-to-date</td>
      <td>Genome annotation, Sequence Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/panta</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/panta</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>peptideshaker</td>
      <td>fasta_cli, ident_params, peptide_shaker, search_gui</td>
      <td>PeptideShaker and SearchGUI</td>
      <td>2015-01-31</td>
      <td>http://compomics.github.io</td>
      <td></td>
      <td>searchgui</td>
      <td>4.3.15</td>
      <td>To update</td>
      <td>Proteomics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/peptideshaker</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/peptideshaker</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>2</td>
      <td>0</td>
      <td>541</td>
      <td>635</td>
      <td>10278</td>
      <td>19384</td>
      <td>236</td>
      <td>236</td>
      <td>1790</td>
      <td>1790</td>
      <td>54</td>
      <td>54</td>
      <td>3790</td>
      <td>3855</td>
      <td>2</td>
      <td>3</td>
      <td>15</td>
      <td>21</td>
      <td>25050</td>
      <td>15873</td>
      <td>928</td>
      <td>833</td>
      <td>https://usegalaxy.eu/published/workflow?id=03e52c8c71710cae, https://usegalaxy.eu/published/workflow?id=0aee7ffa5f6384bd, https://usegalaxy.eu/published/workflow?id=0e0be913c3e27022, https://usegalaxy.eu/published/workflow?id=1706bfa819bbfa89, https://usegalaxy.eu/published/workflow?id=1809fcaec264eac9, https://usegalaxy.eu/published/workflow?id=229a955d29bd4064, https://usegalaxy.eu/published/workflow?id=249c7569b61fb349, https://usegalaxy.eu/published/workflow?id=2a7e63d1cc583002, https://usegalaxy.eu/published/workflow?id=3e01a2c6d954ae6e, https://usegalaxy.eu/published/workflow?id=41d096891bada138, https://usegalaxy.eu/published/workflow?id=41df8c7821b5f2fc, https://usegalaxy.eu/published/workflow?id=41e9882300fa2386, https://usegalaxy.eu/published/workflow?id=4c81726bde25c36a, https://usegalaxy.eu/published/workflow?id=5e4774036aa20b80, https://usegalaxy.eu/published/workflow?id=607ca3a9d45bd080, https://usegalaxy.eu/published/workflow?id=612dc5bdf388585f, https://usegalaxy.eu/published/workflow?id=6306656ec10bd18f, https://usegalaxy.eu/published/workflow?id=6766bed39f614b7c, https://usegalaxy.eu/published/workflow?id=68967c922de149ae, https://usegalaxy.eu/published/workflow?id=6aca8a72626c521b, https://usegalaxy.eu/published/workflow?id=6b7b86756060659b, https://usegalaxy.eu/published/workflow?id=6bd8311554412c38, https://usegalaxy.eu/published/workflow?id=6cb85fe1ad0c9f3b, https://usegalaxy.eu/published/workflow?id=6e5f0d05c91876c2, https://usegalaxy.eu/published/workflow?id=7463b26aab6a965d, https://usegalaxy.eu/published/workflow?id=78e1769f1dcb797b, https://usegalaxy.eu/published/workflow?id=7dd7908f4c8d722a, https://usegalaxy.eu/published/workflow?id=82b6871682f1659a, https://usegalaxy.eu/published/workflow?id=8a85663218676b59, https://usegalaxy.eu/published/workflow?id=8af41219411062ad, https://usegalaxy.eu/published/workflow?id=8fc1ea9e94bc723d, https://usegalaxy.eu/published/workflow?id=912b81c444e35045, https://usegalaxy.eu/published/workflow?id=9a8b5e99ca9500e3, https://usegalaxy.eu/published/workflow?id=9c0132dd81297afe, https://usegalaxy.eu/published/workflow?id=9ef76d764d9961b2, https://usegalaxy.eu/published/workflow?id=a4d6b9dc6b396933, https://usegalaxy.eu/published/workflow?id=a657b412b37b971b, https://usegalaxy.eu/published/workflow?id=b73fa5f8c0403454, https://usegalaxy.eu/published/workflow?id=b89cf3cb6a25c999, https://usegalaxy.eu/published/workflow?id=ba3a5d59b3879764, https://usegalaxy.eu/published/workflow?id=c4119ea7ffd29094, https://usegalaxy.eu/published/workflow?id=c6b6ba35e5b6500a, https://usegalaxy.eu/published/workflow?id=c750484145c9903b, https://usegalaxy.eu/published/workflow?id=cd675ab32d3e7833, https://usegalaxy.eu/published/workflow?id=cefc49c13ff73231, https://usegalaxy.eu/published/workflow?id=d0e062aee910086a, https://usegalaxy.eu/published/workflow?id=d1d5a6bd6571d47f, https://usegalaxy.eu/published/workflow?id=d2a9d4eb5b3c72a2, https://usegalaxy.eu/published/workflow?id=d5756e3ebfa285ef, https://usegalaxy.eu/published/workflow?id=d9e7a940b2d410bc, https://usegalaxy.eu/published/workflow?id=df926c5ab8d27192, https://usegalaxy.eu/published/workflow?id=eb54512829d8509c, https://usegalaxy.eu/published/workflow?id=edea08c943ff9c5e, https://usegalaxy.eu/published/workflow?id=f0bec337f9998289, https://usegalaxy.eu/published/workflow?id=f34c76e2d8045725, https://usegalaxy.eu/published/workflow?id=fc42cc9467c6ee8d, https://usegalaxy.org.au/published/workflow?id=1b9bfe7ce484fdc2, https://usegalaxy.org.au/published/workflow?id=33922db405a97cbb, https://usegalaxy.org.au/published/workflow?id=ba217a34d7e1b5ee, https://usegalaxy.org.au/published/workflow?id=c2f449e69d06ccf5, https://usegalaxy.org.au/published/workflow?id=f9f685d6150e48c5, https://usegalaxy.org/published/workflow?id=e97da2da9e0290a7, https://workflowhub.eu/workflows/1225?version=1, https://workflowhub.eu/workflows/1394?version=1, https://workflowhub.eu/workflows/1396?version=1, https://workflowhub.eu/workflows/1399?version=1, https://workflowhub.eu/workflows/1429?version=1, https://workflowhub.eu/workflows/1435?version=1, https://workflowhub.eu/workflows/1443?version=1, https://workflowhub.eu/workflows/1450?version=1</td>
      <td>microbiome/clinical-mp-2-discovery, proteomics/clinical-mp-2-discovery, proteomics/metaproteomics, proteomics/metaquantome-data-creation, proteomics/protein-id-sg-ps, proteomics/proteogenomics-dbsearch</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>pfamscan</td>
      <td>pfamscan</td>
      <td>Search a FASTA sequence against a library of Pfam HMM.</td>
      <td>2023-02-02</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/Pfam/Tools/</td>
      <td>1.6</td>
      <td>pfam_scan</td>
      <td>1.6</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Protein sequence analysis</td>
      <td>Protein sequence analysis</td>
      <td>Sequence analysis</td>
      <td>Sequence analysis</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/pfamscan</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/pfamscan</td>
      <td>pfamscan</td>
      <td>PfamScan</td>
      <td>This tool is used to search a FASTA sequence against a library of Pfam HMM.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>146</td>
      <td>146</td>
      <td>7522</td>
      <td>7522</td>
      <td>143</td>
      <td>143</td>
      <td>881</td>
      <td>881</td>
      <td>33</td>
      <td>33</td>
      <td>545</td>
      <td>545</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>8949</td>
      <td>8949</td>
      <td>323</td>
      <td>323</td>
      <td>https://usegalaxy.eu/published/workflow?id=11eba1b28ec87963, https://usegalaxy.eu/published/workflow?id=1f9a9a7998ef5ab3, https://usegalaxy.eu/published/workflow?id=47a0407867a90f95, https://usegalaxy.eu/published/workflow?id=87231a6a7938cb95, https://usegalaxy.eu/published/workflow?id=9b025479fade745d, https://usegalaxy.eu/published/workflow?id=b7a3cbcd32734e8b, https://usegalaxy.org.au/published/workflow?id=6e1e24f9e0326930, https://workflowhub.eu/workflows/1688?version=1</td>
      <td>transcriptomics/differential-isoform-expression</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>phabox</td>
      <td>phabox_cherry, phabox_contamination, phabox_end_to_end, phabox_phagcn, phabox_phamer, phabox_phatyp, phabox_phavip, phabox_tree, phabox_votu</td>
      <td>Identify and analyze phage contigs in metagenomic data</td>
      <td>2024-11-12</td>
      <td>https://github.com/KennthShang/PhaBOX</td>
      <td>2.1.11</td>
      <td>phabox</td>
      <td>2.1.13</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Sequence assembly, Taxonomic classification, Visualisation</td>
      <td>Sequence assembly, Taxonomic classification, Visualisation</td>
      <td>Metagenomics, Microbial ecology, Sequence assembly, Taxonomy</td>
      <td>Metagenomics, Microbial ecology, Sequence assembly, Taxonomy</td>
      <td>ufz</td>
      <td>https://github.com/Helmholtz-UFZ/ufz-galaxy-tools/blob/main/tools/phabox</td>
      <td>https://github.com/Helmholtz-UFZ/galaxy-tools/tree/main/tools/phabox</td>
      <td>phabox</td>
      <td>PhaBOX</td>
      <td>Web server for identifying and characterizing phage contigs in metagenomic data.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>pharokka</td>
      <td>pharokka</td>
      <td>rapid standardised annotation tool for bacteriophage genomes and metagenomes</td>
      <td>2023-02-14</td>
      <td>https://github.com/gbouras13/pharokka</td>
      <td>1.3.2</td>
      <td>\n                pharokka\n</td>
      <td></td>
      <td>To update</td>
      <td>Genome annotation</td>
      <td>Genome annotation, Antimicrobial resistance prediction, tRNA gene prediction, Formatting, Sequence assembly</td>
      <td>Genome annotation, Antimicrobial resistance prediction, tRNA gene prediction, Formatting, Sequence assembly</td>
      <td>Metagenomics, Sequence sites, features and motifs, Workflows, Functional, regulatory and non-coding RNA</td>
      <td>Metagenomics, Sequence sites, features and motifs, Workflows, Functional, regulatory and non-coding RNA</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pharokka</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pharokka</td>
      <td>pharokka</td>
      <td>Pharokka</td>
      <td>Pharokka is a rapid standardised annotation tool for bacteriophage genomes and metagenomes.</td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>766</td>
      <td>766</td>
      <td>16933</td>
      <td>16933</td>
      <td>1</td>
      <td>1</td>
      <td>21</td>
      <td>21</td>
      <td>117</td>
      <td>117</td>
      <td>1202</td>
      <td>1202</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>18156</td>
      <td>18156</td>
      <td>884</td>
      <td>884</td>
      <td>https://usegalaxy.eu/published/workflow?id=62392daea25a3980</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>phi_toolkit_report</td>
      <td>phi_toolkit_report</td>
      <td>Phage host interaction toolkit report generator</td>
      <td>2025-06-04</td>
      <td>https://git.ufz.de/borimcor/phage-host-analysis</td>
      <td>0.2.0</td>
      <td>bioconductor-biostrings</td>
      <td>2.74.0</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ufz</td>
      <td>https://github.com/Helmholtz-UFZ/ufz-galaxy-tools/blob/main/tools/phi-toolkit</td>
      <td>https://github.com/Helmholtz-UFZ/galaxy-tools/tree/main/tools/phi-toolkit</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>https://usegalaxy.eu/published/workflow?id=c62d65832377e376</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>phyloseq</td>
      <td>phyloseq_add_rank_names, phyloseq_from_biom, phyloseq_from_dada2, phyloseq_plot_bar, phyloseq_plot_ordination, phyloseq_plot_richness, phyloseq_tax_glom</td>
      <td>Handling and analysis of high-throughput microbiome census data</td>
      <td>2022-03-03</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/phyloseq.html</td>
      <td>1.50.0</td>
      <td>bioconductor-phyloseq</td>
      <td>1.50.0</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Deposition, Analysis, Visualisation</td>
      <td>Deposition, Analysis, Visualisation</td>
      <td>Microbiology, Sequence analysis, Metagenomics</td>
      <td>Microbiology, Sequence analysis, Metagenomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/phyloseq</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/phyloseq</td>
      <td>phyloseq</td>
      <td>phyloseq</td>
      <td>Provides a set of classes and tools to facilitate the import, storage, analysis, and graphical display of microbiome census data.</td>
      <td></td>
      <td>4</td>
      <td>1</td>
      <td>7</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>251</td>
      <td>251</td>
      <td>3161</td>
      <td>3161</td>
      <td>156</td>
      <td>156</td>
      <td>1022</td>
      <td>1022</td>
      <td>35</td>
      <td>35</td>
      <td>190</td>
      <td>190</td>
      <td>14</td>
      <td>14</td>
      <td>163</td>
      <td>163</td>
      <td>4536</td>
      <td>4536</td>
      <td>456</td>
      <td>456</td>
      <td>https://usegalaxy.eu/published/workflow?id=466bdd8ba7b67264, https://usegalaxy.eu/published/workflow?id=e3918808605cf8cd, https://usegalaxy.org/published/workflow?id=4faff660f6defa10, https://workflowhub.eu/workflows/1395?version=2</td>
      <td>microbiome/dada-16S</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>phyml</td>
      <td>phyml</td>
      <td>PhyML is a phylogeny software based on the maximum-likelihood principle.</td>
      <td>2019-05-27</td>
      <td>http://www.atgc-montpellier.fr/phyml/</td>
      <td>3.3.20220408</td>
      <td>phyml</td>
      <td>3.3.20220408</td>
      <td>Up-to-date</td>
      <td>Phylogenetics</td>
      <td>Phylogenetic tree generation (maximum likelihood and Bayesian methods)</td>
      <td></td>
      <td>Phylogenetics, Bioinformatics, Phylogenetics</td>
      <td>Phylogenetics, Bioinformatics, Phylogenetics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/phyml</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/phyml</td>
      <td>phyml</td>
      <td>PhyML</td>
      <td>Phylogenetic estimation software using Maximum Likelihood</td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>286</td>
      <td>335</td>
      <td>2452</td>
      <td>2654</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>215</td>
      <td>240</td>
      <td>1475</td>
      <td>1564</td>
      <td>79</td>
      <td>79</td>
      <td>334</td>
      <td>334</td>
      <td>4552</td>
      <td>4261</td>
      <td>654</td>
      <td>580</td>
      <td>https://usegalaxy.fr/published/workflow?id=63b49f322ec366b5, https://usegalaxy.fr/published/workflow?id=894143d81af8dff0</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>picrust</td>
      <td>picrust_categorize, picrust_compare_biom, picrust_format_tree_and_trait_table, picrust_metagenome_contributions, picrust_normalize_by_copy_number, picrust_predict_metagenomes</td>
      <td>PICRUSt wrappers</td>
      <td>2016-11-08</td>
      <td>https://picrust.github.io/picrust/</td>
      <td>1.1.1</td>
      <td>picrust</td>
      <td>1.1.4</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Phylogenetic reconstruction, Expression analysis, Genome annotation, DNA barcoding</td>
      <td>Phylogenetic reconstruction, Expression analysis, Genome annotation, DNA barcoding</td>
      <td>Metagenomics, Microbial ecology, Functional, regulatory and non-coding RNA, Metagenomic sequencing</td>
      <td>Metagenomics, Microbial ecology, Functional, regulatory and non-coding RNA, Metagenomic sequencing</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/picrust</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/picrust</td>
      <td>picrust</td>
      <td>PICRUSt</td>
      <td>PICRUSt (Phylogenetic Investigation of Communities by Reconstruction of Unobserved States) is a bioinformatics software package designed to predict metagenome functional content from marker gene (e.g., 16S rRNA) surveys and full genomes.</td>
      <td></td>
      <td>0</td>
      <td>6</td>
      <td>5</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>44</td>
      <td>63</td>
      <td>562</td>
      <td>1142</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>38</td>
      <td>41</td>
      <td>2348</td>
      <td>2626</td>
      <td>20</td>
      <td>20</td>
      <td>442</td>
      <td>442</td>
      <td>4210</td>
      <td>3352</td>
      <td>124</td>
      <td>102</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>picrust2</td>
      <td>picrust2_add_descriptions, picrust2_hsp, picrust2_metagenome_pipeline, picrust2_pathway_pipeline, picrust2_pipeline, picrust2_place_seqs, picrust2_shuffle_predictions</td>
      <td>PICRUSt2: Phylogenetic Investigation of Communities by Reconstruction of Unobserved States</td>
      <td>2021-07-09</td>
      <td>https://github.com/picrust/picrust2/wiki</td>
      <td>2.5.3</td>
      <td>picrust2</td>
      <td>2.6.3</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Phylogenetic reconstruction, Expression analysis, Rarefaction, Pathway analysis</td>
      <td>Phylogenetic reconstruction, Expression analysis, Rarefaction, Pathway analysis</td>
      <td>Metagenomics, Microbiology, Phylogenetics, Metagenomic sequencing</td>
      <td>Metagenomics, Microbiology, Phylogenetics, Metagenomic sequencing</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/picrust2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/picrust2</td>
      <td>picrust2</td>
      <td>PICRUSt2</td>
      <td>PICRUSt2 (Phylogenetic Investigation of Communities by Reconstruction of Unobserved States) is a software for predicting functional abundances based only on marker gene sequences.</td>
      <td></td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>327</td>
      <td>327</td>
      <td>6432</td>
      <td>6432</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>64</td>
      <td>64</td>
      <td>2888</td>
      <td>2888</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9321</td>
      <td>9321</td>
      <td>392</td>
      <td>392</td>
      <td>https://usegalaxy.eu/published/workflow?id=be9ff4159593cc34</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>pimento</td>
      <td>pimento_are_there_primers, pimento_auto, pimento_choose_primer_cutoff, pimento_find_cutoffs, pimento_gen_bcv, pimento_std</td>
      <td>A PrIMEr infereNce TOolkit to facilitate large-scale calling of metabarcoding amplicon sequence variants.</td>
      <td>2025-08-04</td>
      <td>https://github.com/EBI-Metagenomics/PIMENTO</td>
      <td>1.0.2</td>
      <td>mi-pimento</td>
      <td>1.0.2</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pimento</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pimento</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>plasclass</td>
      <td>plasclass</td>
      <td>PlasClass is a tool to classify sequences of plasmid or chromosomal origin</td>
      <td>2025-07-31</td>
      <td>https://github.com/Shamir-Lab/PlasClass</td>
      <td>0.1.1</td>
      <td>plasclass</td>
      <td>0.1.1</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Sequence classification, Sequence assembly, k-mer counting</td>
      <td>Sequence classification, Sequence assembly, k-mer counting</td>
      <td>Metagenomics, Sequence assembly, DNA, Metagenomic sequencing</td>
      <td>Metagenomics, Sequence assembly, DNA, Metagenomic sequencing</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/plasclass</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/plasclass</td>
      <td>PlasClass</td>
      <td>PlasClass</td>
      <td>PlasClass is a tool for easy classification of sequences as either plasmid or chromosomal. For example, it can be used to classify the contigs in a (metagenomic) assembly.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>plasflow</td>
      <td>PlasFlow</td>
      <td>PlasFlow - Prediction of plasmid sequences in metagenomic contigs.</td>
      <td>2018-09-05</td>
      <td>https://github.com/smaegol/PlasFlow</td>
      <td>1.1.0</td>
      <td>plasflow</td>
      <td>1.1.0</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Sequence analysis</td>
      <td>Sequence analysis</td>
      <td>Metagenomics, Sequence assembly, Mobile genetic elements, Machine learning</td>
      <td>Metagenomics, Sequence assembly, Mobile genetic elements, Machine learning</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/plasflow</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/plasflow</td>
      <td>plasflow</td>
      <td>PlasFlow</td>
      <td>PlasFlow is a set of scripts used for prediction of plasmid sequences in metagenomic contigs.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>876</td>
      <td>1004</td>
      <td>55069</td>
      <td>56639</td>
      <td>757</td>
      <td>797</td>
      <td>7929</td>
      <td>8195</td>
      <td>87</td>
      <td>87</td>
      <td>3496</td>
      <td>3496</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>68330</td>
      <td>66494</td>
      <td>1888</td>
      <td>1720</td>
      <td>https://usegalaxy.eu/published/workflow?id=152223c8aed27357, https://usegalaxy.eu/published/workflow?id=249c115413f17f28, https://usegalaxy.eu/published/workflow?id=3081569659a45b9e, https://usegalaxy.eu/published/workflow?id=32d2f5340098bcbd, https://usegalaxy.eu/published/workflow?id=39837ca09374598b, https://usegalaxy.eu/published/workflow?id=44e53ea319e07daa, https://usegalaxy.eu/published/workflow?id=9219557de92e0203, https://usegalaxy.eu/published/workflow?id=b4de76c0a4d0e23a, https://usegalaxy.eu/published/workflow?id=c48807045e258dd8, https://usegalaxy.eu/published/workflow?id=dc7107896a3f2964, https://usegalaxy.org.au/published/workflow?id=d9571b3dfef34e5e, https://usegalaxy.org/published/workflow?id=33d90e718ce500ef, https://workflowhub.eu/workflows/1477?version=1, https://workflowhub.eu/workflows/406?version=1</td>
      <td>microbiome/plasmid-metagenomics-nanopore</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>plasmidfinder</td>
      <td>plasmidfinder</td>
      <td>"PlasmidFinder provides the detection of replicons in the WGSand assigns the plasmids under study to lineages that trace backthe information to the existing knowledge on Inc groups and suggestspossible reference plasmids for each lineage"</td>
      <td>2022-09-19</td>
      <td>https://bitbucket.org/genomicepidemiology/plasmidfinder/src/master/</td>
      <td>2.1.6</td>
      <td>plasmidfinder</td>
      <td>2.1.6</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Genome assembly, Scaffolding, Multilocus sequence typing</td>
      <td>Genome assembly, Scaffolding, Multilocus sequence typing</td>
      <td>Whole genome sequencing, Sequence assembly, Mapping, Probes and primers</td>
      <td>Whole genome sequencing, Sequence assembly, Mapping, Probes and primers</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/plasmidfinder</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/plasmidfinder</td>
      <td>PlasmidFinder</td>
      <td>PlasmidFinder</td>
      <td>PlasmidFinder is a tool for the identification and typing of Plasmid Replicons in Whole-Genome Sequencing (WGS).</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1015</td>
      <td>1015</td>
      <td>35994</td>
      <td>35994</td>
      <td>623</td>
      <td>623</td>
      <td>11939</td>
      <td>11939</td>
      <td>211</td>
      <td>211</td>
      <td>3278</td>
      <td>3278</td>
      <td>48</td>
      <td>48</td>
      <td>10730</td>
      <td>10730</td>
      <td>61941</td>
      <td>61941</td>
      <td>1897</td>
      <td>1897</td>
      <td>https://usegalaxy.eu/published/workflow?id=1306ed083d91a852, https://usegalaxy.eu/published/workflow?id=2c1a6da84d891cf2, https://usegalaxy.eu/published/workflow?id=36ea584e0f60893e, https://usegalaxy.eu/published/workflow?id=3f5fe28793ed0c6c, https://usegalaxy.eu/published/workflow?id=4a218fb93d78ba67, https://usegalaxy.eu/published/workflow?id=61948fcaefa1418a, https://usegalaxy.eu/published/workflow?id=6b69ab27c565257b, https://usegalaxy.eu/published/workflow?id=8792eaecb69fbc26, https://usegalaxy.eu/published/workflow?id=a18aed4b03ffd27e, https://usegalaxy.eu/published/workflow?id=b1afc8d1e81fc904, https://usegalaxy.eu/published/workflow?id=bcd50e655809cfcb, https://usegalaxy.eu/published/workflow?id=d8db0b2e30370b84, https://usegalaxy.eu/published/workflow?id=dfb8e0f3c534000e, https://usegalaxy.eu/published/workflow?id=fd63b9884fda75b6, https://usegalaxy.fr/published/workflow?id=7964ddc68853458f, https://usegalaxy.org.au/published/workflow?id=05f3abecd4ddd4a1, https://usegalaxy.org/published/workflow?id=04c5ea133de3204c, https://usegalaxy.org/published/workflow?id=3940186d02245b1d, https://usegalaxy.org/published/workflow?id=49da012e2839c046, https://usegalaxy.org/published/workflow?id=4f8bdd61c524b901, https://usegalaxy.org/published/workflow?id=649cdc14757e5755, https://usegalaxy.org/published/workflow?id=67ca17ab34b10772, https://usegalaxy.org/published/workflow?id=8cb8a588a0495cfa, https://usegalaxy.org/published/workflow?id=993534c11310a207, https://usegalaxy.org/published/workflow?id=d2a5f6a5181aada5, https://workflowhub.eu/workflows/1050?version=13, https://workflowhub.eu/workflows/1513?version=1</td>
      <td>genome-annotation/bacterial-genome-annotation</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>plasmidspades</td>
      <td>plasmidspades</td>
      <td>Genome assembler for assemblying plasmid</td>
      <td>2016-06-06</td>
      <td></td>
      <td>1.1</td>
      <td>spades</td>
      <td>4.2.0</td>
      <td>To update</td>
      <td>Assembly</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>nml</td>
      <td></td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/plasmidspades</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>polypolish</td>
      <td>polypolish</td>
      <td>"Polypolish is a tool for polishing genome assemblies with short reads.Polypolish uses SAM files where each read has been aligned to all possible locations (not just a single best location).This allows it to repair errors in repeat regions that other alignment-based polishers cannot fix."</td>
      <td>2022-09-22</td>
      <td>https://github.com/rrwick/Polypolish</td>
      <td>0.6.1</td>
      <td>polypolish</td>
      <td>0.6.1</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Genome assembly, Read mapping, Mapping assembly, Sequencing error detection</td>
      <td>Genome assembly, Read mapping, Mapping assembly, Sequencing error detection</td>
      <td>Sequence assembly, Sequence composition, complexity and repeats, Mapping</td>
      <td>Sequence assembly, Sequence composition, complexity and repeats, Mapping</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/polypolish</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/polypolish</td>
      <td>Polypolish</td>
      <td>Polypolish</td>
      <td>Polypolish is a tool for polishing genome assemblies with short reads. Unlike other tools in this category, Polypolish uses SAM files where each read has been aligned to all possible locations (not just a single best location). This allows it to repair errors in repeat regions that other alignment-based polishers cannot fix.</td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>147</td>
      <td>147</td>
      <td>1066</td>
      <td>1066</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>26</td>
      <td>26</td>
      <td>615</td>
      <td>615</td>
      <td>10</td>
      <td>10</td>
      <td>79</td>
      <td>79</td>
      <td>1760</td>
      <td>1760</td>
      <td>183</td>
      <td>183</td>
      <td>https://workflowhub.eu/workflows/1613?version=2</td>
      <td>assembly/mrsa-nanopore</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>ppanggolin</td>
      <td>ppanggolin_all, ppanggolin_msa, ppanggolin_projection, ppanggolin_rarefaction</td>
      <td>Microbial Partitioned PanGenome</td>
      <td>2025-01-22</td>
      <td>https://github.com/labgem/PPanGGOLiN</td>
      <td>2.2.6</td>
      <td>ppanggolin</td>
      <td>2.2.6</td>
      <td>Up-to-date</td>
      <td>Genome annotation</td>
      <td>Rarefaction, Essential dynamics, Sequence assembly, Genome annotation</td>
      <td>Rarefaction, Essential dynamics, Sequence assembly, Genome annotation</td>
      <td>Metagenomics, Gene and protein families, Model organisms, Public health and epidemiology, Mapping</td>
      <td>Metagenomics, Gene and protein families, Model organisms, Public health and epidemiology, Mapping</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ppanggolin</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ppanggolin</td>
      <td>ppanggolin</td>
      <td>PPanGGOLiN</td>
      <td>PPanGGOLiN is a software suite used to create and manipulate prokaryotic pangenomes from a set of either genomic DNA sequences or provided genome annotations. It is designed to scale up to tens of thousands of genomes. It has the specificity to partition the pangenome using a statistical approach rather than using fixed thresholds which gives it the ability to work with low-quality data such as Metagenomic Assembled Genomes (MAGs) or Single-cell Amplified Genomes (SAGs) thus taking advantage of large scale environmental studies and letting users study the pangenome of uncultivable species</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>55</td>
      <td>55</td>
      <td>249</td>
      <td>249</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>16</td>
      <td>16</td>
      <td>67</td>
      <td>67</td>
      <td>316</td>
      <td>316</td>
      <td>71</td>
      <td>71</td>
      <td></td>
      <td>genome-annotation/bacterial-pangenomics</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>prodigal</td>
      <td>prodigal</td>
      <td>A protein-coding gene prediction software tool for bacterial and archaeal genomes</td>
      <td>2024-03-14</td>
      <td>https://github.com/hyattpd/Prodigal</td>
      <td>2.6.3</td>
      <td>prodigal</td>
      <td>2.6.3</td>
      <td>Up-to-date</td>
      <td>Genome annotation</td>
      <td>Genome annotation</td>
      <td>Genome annotation</td>
      <td>Genomics, Sequence analysis</td>
      <td>Genomics, Sequence analysis</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/prodigal</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/prodigal</td>
      <td>prodigal</td>
      <td>Prodigal</td>
      <td>Fast, reliable protein-coding gene prediction for prokaryotic genomes.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>611</td>
      <td>611</td>
      <td>6456</td>
      <td>6456</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>70</td>
      <td>70</td>
      <td>1515</td>
      <td>1515</td>
      <td>19</td>
      <td>19</td>
      <td>1487</td>
      <td>1487</td>
      <td>9458</td>
      <td>9458</td>
      <td>700</td>
      <td>700</td>
      <td>https://usegalaxy.eu/published/workflow?id=2ee0eb74e839b476, https://usegalaxy.eu/published/workflow?id=62392daea25a3980, https://usegalaxy.eu/published/workflow?id=b9c938d1af08124b, https://workflowhub.eu/workflows/1663?version=1, https://workflowhub.eu/workflows/2024?version=2</td>
      <td>ecology/marine_omics_bgc</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>prokka</td>
      <td>prokka</td>
      <td>Rapid annotation of prokaryotic genomes</td>
      <td>2016-10-07</td>
      <td>http://github.com/tseemann/prokka</td>
      <td>1.14.6</td>
      <td>prokka</td>
      <td>1.15.6</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Gene prediction, Coding region prediction, Genome annotation</td>
      <td>Coding region prediction, Genome annotation</td>
      <td>Genomics, Model organisms, Virology</td>
      <td>Genomics, Model organisms, Virology</td>
      <td>crs4</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/prokka</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/prokka</td>
      <td>prokka</td>
      <td>Prokka</td>
      <td>Software tool to annotate bacterial, archaeal and viral genomes quickly and produce standards-compliant output files.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>10459</td>
      <td>11098</td>
      <td>819587</td>
      <td>847581</td>
      <td>14547</td>
      <td>16587</td>
      <td>462648</td>
      <td>506432</td>
      <td>5094</td>
      <td>6142</td>
      <td>199279</td>
      <td>229418</td>
      <td>422</td>
      <td>424</td>
      <td>7463</td>
      <td>7481</td>
      <td>1590912</td>
      <td>1488977</td>
      <td>34251</td>
      <td>30522</td>
      <td>https://usegalaxy.eu/published/workflow?id=007ce6405d196df3, https://usegalaxy.eu/published/workflow?id=12653a1e65a20304, https://usegalaxy.eu/published/workflow?id=152223c8aed27357, https://usegalaxy.eu/published/workflow?id=165089132248a0b0, https://usegalaxy.eu/published/workflow?id=2e0075791f7fa384, https://usegalaxy.eu/published/workflow?id=36de05d50c703164, https://usegalaxy.eu/published/workflow?id=374797e58377c40f, https://usegalaxy.eu/published/workflow?id=37da7b42cb063a3a, https://usegalaxy.eu/published/workflow?id=42c0da2e86318b37, https://usegalaxy.eu/published/workflow?id=4ddbffe4b3fef275, https://usegalaxy.eu/published/workflow?id=5826e573b6f93762, https://usegalaxy.eu/published/workflow?id=628062a8cf93261f, https://usegalaxy.eu/published/workflow?id=6984990d0ac2e1e6, https://usegalaxy.eu/published/workflow?id=6a5c39c1d38a10c2, https://usegalaxy.eu/published/workflow?id=6d87481be47902e1, https://usegalaxy.eu/published/workflow?id=7aa32aba3fde0708, https://usegalaxy.eu/published/workflow?id=87231a6a7938cb95, https://usegalaxy.eu/published/workflow?id=8f3ce55d44d9f369, https://usegalaxy.eu/published/workflow?id=9219557de92e0203, https://usegalaxy.eu/published/workflow?id=94c3d329c31c712f, https://usegalaxy.eu/published/workflow?id=98bfad0c684011e9, https://usegalaxy.eu/published/workflow?id=a337bfb2eb1e583f, https://usegalaxy.eu/published/workflow?id=a448400f69594135, https://usegalaxy.eu/published/workflow?id=a847773b563b2328, https://usegalaxy.eu/published/workflow?id=a84bac86ec183d46, https://usegalaxy.eu/published/workflow?id=b1a799497c366eef, https://usegalaxy.eu/published/workflow?id=bf48b0eb15e8ff75, https://usegalaxy.eu/published/workflow?id=c9ef2d2ce92e2dfb, https://usegalaxy.eu/published/workflow?id=ce8bf03d3df24806, https://usegalaxy.eu/published/workflow?id=cf8591391a17a914, https://usegalaxy.eu/published/workflow?id=d92f9c33dded693f, https://usegalaxy.eu/published/workflow?id=e16e524c557d7928, https://usegalaxy.eu/published/workflow?id=e1894c36f1ab4ab4, https://usegalaxy.eu/published/workflow?id=e8eb80ee09159bac, https://usegalaxy.eu/published/workflow?id=ea485dbabc76488d, https://usegalaxy.eu/published/workflow?id=ecf48fa50af560a0, https://usegalaxy.eu/published/workflow?id=f141db5179ea8422, https://usegalaxy.eu/published/workflow?id=f97e6fe1fcbc53a7, https://usegalaxy.eu/published/workflow?id=fe43cf59e48bf556, https://usegalaxy.org.au/published/workflow?id=05f3abecd4ddd4a1, https://usegalaxy.org.au/published/workflow?id=2e29b21cdec7d0cc, https://usegalaxy.org.au/published/workflow?id=33401b29d277b447, https://usegalaxy.org.au/published/workflow?id=35f39173f0ac9179, https://usegalaxy.org.au/published/workflow?id=39c724282ee008e3, https://usegalaxy.org.au/published/workflow?id=59618ed15510f531, https://usegalaxy.org.au/published/workflow?id=6c67e22d18566af3, https://usegalaxy.org.au/published/workflow?id=7c343e5b55474d19, https://usegalaxy.org.au/published/workflow?id=87cdeb05b1ebeab0, https://usegalaxy.org.au/published/workflow?id=8f7d5ebead46a6d0, https://usegalaxy.org.au/published/workflow?id=914330feb75ea9ab, https://usegalaxy.org.au/published/workflow?id=b5403e4fff2961f9, https://usegalaxy.org.au/published/workflow?id=c997a34b55425155, https://usegalaxy.org.au/published/workflow?id=e20172eb0f30d6cd, https://usegalaxy.org.au/published/workflow?id=f01be6a7685ac476, https://usegalaxy.org.au/published/workflow?id=fa76b66785ce38b8, https://usegalaxy.org.au/published/workflow?id=fc664a96d8099652, https://usegalaxy.org/published/workflow?id=0795dd1d83662bae, https://usegalaxy.org/published/workflow?id=109092d8dbda4398, https://usegalaxy.org/published/workflow?id=1c7af480caf150bd, https://usegalaxy.org/published/workflow?id=33d90e718ce500ef, https://usegalaxy.org/published/workflow?id=3cf46c167ccb374f, https://usegalaxy.org/published/workflow?id=547f86db467bf620, https://usegalaxy.org/published/workflow?id=78d8b738f5715a8a, https://usegalaxy.org/published/workflow?id=7fc8b2398b6f093f, https://usegalaxy.org/published/workflow?id=80565301ada0d3eb, https://usegalaxy.org/published/workflow?id=8111bb9bb26bb665, https://usegalaxy.org/published/workflow?id=87b3305ea433b4e2, https://usegalaxy.org/published/workflow?id=88bc9aa7090875c1, https://usegalaxy.org/published/workflow?id=8d229c0f58a07ebe, https://usegalaxy.org/published/workflow?id=b43b72dcf5f7694d, https://usegalaxy.org/published/workflow?id=c451e4dca9981ed0, https://usegalaxy.org/published/workflow?id=c65bec6c332bf740, https://usegalaxy.org/published/workflow?id=d4f0a7ccb09f1e55, https://usegalaxy.org/published/workflow?id=e2ab3d3219a648a7, https://usegalaxy.org/published/workflow?id=e75bd7d392e65209, https://usegalaxy.org/published/workflow?id=fb01e1a49793c8be, https://workflowhub.eu/workflows/1497?version=1, https://workflowhub.eu/workflows/1598?version=1, https://workflowhub.eu/workflows/1617?version=1, https://workflowhub.eu/workflows/1767?version=1</td>
      <td>assembly/chloroplast-assembly, assembly/unicycler-assembly, genome-annotation/annotation-with-prokka</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>promer</td>
      <td>promer4_substitutions</td>
      <td>Aligns two sets of contigs and reports amino acid substitutions between them</td>
      <td>2019-12-14</td>
      <td>https://github.com/phac-nml/promer</td>
      <td>1.2</td>
      <td>python</td>
      <td></td>
      <td>To update</td>
      <td>Assembly</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>nml</td>
      <td>https://github.com/phac-nml/promer</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/promer</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>proteinortho</td>
      <td>proteinortho, proteinortho_clustering, proteinortho_grab_proteins, proteinortho_summary</td>
      <td>Proteinortho is a tool to detect orthologous proteins/genes within different species.</td>
      <td>2020-02-05</td>
      <td>https://gitlab.com/paulklemm_PHD/proteinortho</td>
      <td>6.3.6</td>
      <td>proteinortho</td>
      <td>6.3.6</td>
      <td>Up-to-date</td>
      <td>Proteomics</td>
      <td>Sequence clustering, Sequence analysis</td>
      <td>Sequence clustering, Sequence analysis</td>
      <td>Comparative genomics</td>
      <td>Comparative genomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/proteinortho</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/proteinortho</td>
      <td>proteinortho</td>
      <td>Proteinortho</td>
      <td>Proteinortho is a tool to detect orthologous genes within different species</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>432</td>
      <td>463</td>
      <td>5028</td>
      <td>5485</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>23</td>
      <td>23</td>
      <td>5508</td>
      <td>5051</td>
      <td>467</td>
      <td>436</td>
      <td>https://workflowhub.eu/workflows/1656?version=1, https://workflowhub.eu/workflows/358?version=1</td>
      <td>ecology/phylogeny-data-prep</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>pycoqc</td>
      <td>pycoqc</td>
      <td>QC metrics for ONT Basecalling</td>
      <td>2021-03-02</td>
      <td>https://github.com/tleonardi/pycoQC</td>
      <td>2.5.2</td>
      <td>pycoqc</td>
      <td>2.5.2</td>
      <td>Up-to-date</td>
      <td>Nanopore</td>
      <td>Sequencing quality control, Statistical calculation</td>
      <td>Sequencing quality control, Statistical calculation</td>
      <td>Sequence analysis, Data quality management, Sequencing</td>
      <td>Sequence analysis, Data quality management, Sequencing</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pycoqc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pycoqc</td>
      <td>pycoqc</td>
      <td>pycoQC</td>
      <td>PycoQC computes metrics and generates interactive QC plots for Oxford Nanopore technologies sequencing data.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>832</td>
      <td>832</td>
      <td>23549</td>
      <td>23549</td>
      <td>1182</td>
      <td>1182</td>
      <td>4440</td>
      <td>4440</td>
      <td>606</td>
      <td>606</td>
      <td>1143</td>
      <td>1143</td>
      <td>158</td>
      <td>158</td>
      <td>322</td>
      <td>322</td>
      <td>29454</td>
      <td>29454</td>
      <td>2778</td>
      <td>2778</td>
      <td>https://usegalaxy.org.au/published/workflow?id=01393562b82f1884, https://usegalaxy.org.au/published/workflow?id=0d1d4392c3042fbd, https://usegalaxy.org.au/published/workflow?id=31dbe7344d72b903, https://usegalaxy.org.au/published/workflow?id=364025bd13970867, https://usegalaxy.org.au/published/workflow?id=419e09e4a2fa6d38, https://usegalaxy.org.au/published/workflow?id=5f4e164c18810cca, https://usegalaxy.org.au/published/workflow?id=866b69c5ee6601fe, https://usegalaxy.org.au/published/workflow?id=8fe84de499e96485, https://usegalaxy.org.au/published/workflow?id=98f285d6f0253b81, https://usegalaxy.org.au/published/workflow?id=c87b645aedfb857d, https://usegalaxy.org.au/published/workflow?id=da3d4b39b0d655e7, https://usegalaxy.org.au/published/workflow?id=ed43af416c6a7d3b, https://usegalaxy.org.au/published/workflow?id=fdf32312e8567093</td>
      <td>sequence-analysis/quality-control</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>pygenometracks</td>
      <td>pygenomeTracks</td>
      <td>pyGenomeTracks: Standalone program and library to plot beautiful genome browser tracks.</td>
      <td>2018-06-08</td>
      <td>https://github.com/deeptools/pyGenomeTracks</td>
      <td>3.9</td>
      <td>pygenometracks</td>
      <td>3.9</td>
      <td>Up-to-date</td>
      <td>Visualization</td>
      <td>Visualisation, Formatting</td>
      <td>Visualisation, Formatting</td>
      <td>Model organisms, Imaging, Workflows</td>
      <td>Model organisms, Imaging, Workflows</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pygenometracks</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pygenometracks</td>
      <td>pygenometracks</td>
      <td>pyGenomeTracks</td>
      <td>reproducible plots for multivariate genomic data sets.Standalone program and library to plot beautiful genome browser tracks.pyGenomeTracks aims to produce high-quality genome browser tracks that are highly customizable. Currently, it is possible to plot:.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1301</td>
      <td>1446</td>
      <td>13377</td>
      <td>16146</td>
      <td>1427</td>
      <td>1472</td>
      <td>11997</td>
      <td>12438</td>
      <td>210</td>
      <td>210</td>
      <td>5014</td>
      <td>5054</td>
      <td>44</td>
      <td>44</td>
      <td>81</td>
      <td>81</td>
      <td>33719</td>
      <td>30469</td>
      <td>3172</td>
      <td>2982</td>
      <td>https://usegalaxy.eu/published/workflow?id=02655d0e4af0ee0d, https://usegalaxy.eu/published/workflow?id=2a9c812f59b7b36c, https://usegalaxy.eu/published/workflow?id=4464d3b0fc782a1a, https://usegalaxy.eu/published/workflow?id=47a0407867a90f95, https://usegalaxy.eu/published/workflow?id=506a451679fe25b4, https://usegalaxy.eu/published/workflow?id=72201ffaf488f278, https://usegalaxy.eu/published/workflow?id=9843ae66c3b367b7, https://usegalaxy.eu/published/workflow?id=985044b4b2c51425, https://usegalaxy.eu/published/workflow?id=9f8a93cfffb31cd6, https://usegalaxy.eu/published/workflow?id=9fc8b89954c04ee1, https://usegalaxy.eu/published/workflow?id=a28c207730e7e13d, https://usegalaxy.eu/published/workflow?id=f394d1b12779ba41, https://usegalaxy.org.au/published/workflow?id=13ce18408ee7acc9, https://usegalaxy.org.au/published/workflow?id=99821674db7682bb, https://usegalaxy.org/published/workflow?id=06eaa5b141fe7a08, https://usegalaxy.org/published/workflow?id=0bc1b8888d8b20c4, https://usegalaxy.org/published/workflow?id=1dc312e3fba9f810, https://usegalaxy.org/published/workflow?id=4998f171ec922c7d, https://usegalaxy.org/published/workflow?id=582f3de3971fc38c, https://usegalaxy.org/published/workflow?id=7353afb9301780e5, https://usegalaxy.org/published/workflow?id=b31ab9e76409a9b7, https://usegalaxy.org/published/workflow?id=e33f6a058c5f57e5, https://usegalaxy.org/published/workflow?id=f13432eef7ac28e9, https://usegalaxy.org/published/workflow?id=f80b80aa7fb196ba, https://usegalaxy.org/published/workflow?id=fb789920b057fe8d, https://workflowhub.eu/workflows/1568?version=2, https://workflowhub.eu/workflows/1688?version=1, https://workflowhub.eu/workflows/1847?version=1, https://workflowhub.eu/workflows/420?version=3</td>
      <td>epigenetics/atac-seq, epigenetics/hicexplorer, transcriptomics/differential-isoform-expression, transcriptomics/ref-based</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>qiime2_dbotu_q2</td>
      <td>qiime2_dbotu_q2</td>
      <td>q2-dbotu is a Qiime 2 plugin for performing OTU clustering</td>
      <td>2025-03-20</td>
      <td>https://github.com/cduvallet/q2-dbotu/tree/master</td>
      <td>2022.11.1+galaxy0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Demultiplexing, Visualisation, Taxonomic classification</td>
      <td>Demultiplexing, Visualisation, Taxonomic classification</td>
      <td>Microbial ecology, Metatranscriptomics, Metagenomics</td>
      <td>Microbial ecology, Metatranscriptomics, Metagenomics</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/qiime2_dbotu_q2/</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/qiime2_dbotu_q2</td>
      <td>qiime2</td>
      <td>QIIME 2</td>
      <td>QIIME 2 is an AI-ready microbiome multi-omics data science platform that is trusted, free, open source, extensible, and community developed and supported bioinformatics.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>13</td>
      <td>13</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>13</td>
      <td>13</td>
      <td>7</td>
      <td>7</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>qualimap</td>
      <td>qualimap_bamqc, qualimap_counts, qualimap_multi_bamqc, qualimap_rnaseq</td>
      <td>Qualimap 2 is a platform-independent application written in Java andR that facilitates the quality control of alignment sequencing data and itsderivatives like feature counts.</td>
      <td>2019-10-10</td>
      <td>http://qualimap.bioinfo.cipf.es/</td>
      <td>2.3</td>
      <td>qualimap</td>
      <td>2.3</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis, Transcriptomics, SAM</td>
      <td>Sequencing quality control</td>
      <td>Sequencing quality control</td>
      <td>Data quality management</td>
      <td>Data quality management</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/qualimap</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/qualimap</td>
      <td>qualimap</td>
      <td>QualiMap</td>
      <td>Platform-independent application written in Java and R that provides both a Graphical User Inteface (GUI) and a command-line interface to facilitate the quality control of alignment sequencing data.</td>
      <td></td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>2949</td>
      <td>3172</td>
      <td>735774</td>
      <td>740034</td>
      <td>3586</td>
      <td>3810</td>
      <td>117198</td>
      <td>120922</td>
      <td>847</td>
      <td>882</td>
      <td>36259</td>
      <td>41650</td>
      <td>70</td>
      <td>70</td>
      <td>6924</td>
      <td>6924</td>
      <td>909530</td>
      <td>896155</td>
      <td>7934</td>
      <td>7452</td>
      <td>https://usegalaxy.eu/published/workflow?id=0189bab68f828d86, https://usegalaxy.eu/published/workflow?id=0be293debca8fb9b, https://usegalaxy.eu/published/workflow?id=0f6a7bd5362e4a2b, https://usegalaxy.eu/published/workflow?id=220961d803697f54, https://usegalaxy.eu/published/workflow?id=2573209308aaad23, https://usegalaxy.eu/published/workflow?id=2c3d05023c02113e, https://usegalaxy.eu/published/workflow?id=2c79eee5ca5a3b09, https://usegalaxy.eu/published/workflow?id=2d9d4445a1871230, https://usegalaxy.eu/published/workflow?id=3f046b416b10b871, https://usegalaxy.eu/published/workflow?id=3fb550d7c2bc390a, https://usegalaxy.eu/published/workflow?id=40d2c7ac42bd5302, https://usegalaxy.eu/published/workflow?id=429ae28a178cd07c, https://usegalaxy.eu/published/workflow?id=45fc2dada84d251e, https://usegalaxy.eu/published/workflow?id=476db04368d5e90c, https://usegalaxy.eu/published/workflow?id=4b4bfc79205d83f1, https://usegalaxy.eu/published/workflow?id=4c75ab0be3c56494, https://usegalaxy.eu/published/workflow?id=4d97068a725c975c, https://usegalaxy.eu/published/workflow?id=529f30e894beacc2, https://usegalaxy.eu/published/workflow?id=52bdf400f97c5770, https://usegalaxy.eu/published/workflow?id=53de0e742573e588, https://usegalaxy.eu/published/workflow?id=579dd1e95e76402f, https://usegalaxy.eu/published/workflow?id=5c15f251642bfa1f, https://usegalaxy.eu/published/workflow?id=5ed38d8114bcd6f7, https://usegalaxy.eu/published/workflow?id=629e57958afed44c, https://usegalaxy.eu/published/workflow?id=6dbac47941a804da, https://usegalaxy.eu/published/workflow?id=6e1fe7372e93fae0, https://usegalaxy.eu/published/workflow?id=6ed1016dfdf9f928, https://usegalaxy.eu/published/workflow?id=7117dbcbd5d12de5, https://usegalaxy.eu/published/workflow?id=7136ce9f18c74853, https://usegalaxy.eu/published/workflow?id=74f19aff1b9cdd6b, https://usegalaxy.eu/published/workflow?id=7bdd9ad918edab0b, https://usegalaxy.eu/published/workflow?id=7c986f4d14c6a539, https://usegalaxy.eu/published/workflow?id=98bfad0c684011e9, https://usegalaxy.eu/published/workflow?id=9e9c9dbed98f62ff, https://usegalaxy.eu/published/workflow?id=9feebfc6c34da287, https://usegalaxy.eu/published/workflow?id=a15dabcadb63b46d, https://usegalaxy.eu/published/workflow?id=a31100610fba17a1, https://usegalaxy.eu/published/workflow?id=a6f4371ef9bc1aab, https://usegalaxy.eu/published/workflow?id=ada52ba09b462567, https://usegalaxy.eu/published/workflow?id=af30010868c97316, https://usegalaxy.eu/published/workflow?id=b77e3b08092fa73e, https://usegalaxy.eu/published/workflow?id=b97532c398b821a2, https://usegalaxy.eu/published/workflow?id=c789ab71876c6ead, https://usegalaxy.eu/published/workflow?id=c9ef2d2ce92e2dfb, https://usegalaxy.eu/published/workflow?id=cd25f70f71683ab6, https://usegalaxy.eu/published/workflow?id=d88145e7475e0b7b, https://usegalaxy.eu/published/workflow?id=d967194ca51ff11e, https://usegalaxy.eu/published/workflow?id=ddd604341afd9cf7, https://usegalaxy.eu/published/workflow?id=eaf01ccb13aad3ec, https://usegalaxy.eu/published/workflow?id=ed20bd120064bab0, https://usegalaxy.eu/published/workflow?id=ee6dc47660540ba6, https://usegalaxy.eu/published/workflow?id=f97e6fe1fcbc53a7, https://usegalaxy.eu/published/workflow?id=fc216dfb0ec9ed41, https://usegalaxy.eu/published/workflow?id=fcf745185d6e0ea4, https://usegalaxy.fr/published/workflow?id=2345fad7dac13daa, https://usegalaxy.fr/published/workflow?id=591c57173e259ca6, https://usegalaxy.fr/published/workflow?id=61235264d5d2f666, https://usegalaxy.fr/published/workflow?id=727cfdb7c79287d1, https://usegalaxy.fr/published/workflow?id=7d27dd394b921846, https://usegalaxy.fr/published/workflow?id=98529cff0ef36b91, https://usegalaxy.fr/published/workflow?id=d0d12c0b80c286f9, https://usegalaxy.fr/published/workflow?id=e0b225d4b2556365, https://usegalaxy.fr/published/workflow?id=e3c04a30258421aa, https://usegalaxy.org.au/published/workflow?id=094a545d44ce28cc, https://usegalaxy.org.au/published/workflow?id=0c6f7bfc826433c4, https://usegalaxy.org.au/published/workflow?id=203f9e69152fbb2a, https://usegalaxy.org.au/published/workflow?id=2b2ce7e1206d177d, https://usegalaxy.org.au/published/workflow?id=3223ceaa2b58d940, https://usegalaxy.org.au/published/workflow?id=3c657dd766d02f6a, https://usegalaxy.org.au/published/workflow?id=47ce6a2d30887637, https://usegalaxy.org.au/published/workflow?id=48a60965b6b3ff54, https://usegalaxy.org.au/published/workflow?id=49b5c35b9723fa62, https://usegalaxy.org.au/published/workflow?id=4bc8ac41ecff7daf, https://usegalaxy.org.au/published/workflow?id=5cf5334877cc1297, https://usegalaxy.org.au/published/workflow?id=6b1b0a945d367b4c, https://usegalaxy.org.au/published/workflow?id=7563b3ef697a69fd, https://usegalaxy.org.au/published/workflow?id=7fd58f5fc93f414e, https://usegalaxy.org.au/published/workflow?id=810242a3dc4f6170, https://usegalaxy.org.au/published/workflow?id=888eb87b62c4e6dd, https://usegalaxy.org.au/published/workflow?id=888f8377eb6eabf2, https://usegalaxy.org.au/published/workflow?id=932bad88f8dc8d33, https://usegalaxy.org.au/published/workflow?id=af12a3ec85859563, https://usegalaxy.org.au/published/workflow?id=b432a1312859edd2, https://usegalaxy.org.au/published/workflow?id=ba7a9d4c857d26b7, https://usegalaxy.org.au/published/workflow?id=baa763dbdf6afb71, https://usegalaxy.org.au/published/workflow?id=bcf230e526632064, https://usegalaxy.org.au/published/workflow?id=be9213411c520096, https://usegalaxy.org.au/published/workflow?id=c1ce3f2ae7c81c3e, https://usegalaxy.org.au/published/workflow?id=c9156ca3df2b595c, https://usegalaxy.org.au/published/workflow?id=ccf936c5b2f2c2f3, https://usegalaxy.org.au/published/workflow?id=d35d05be576e8b13, https://usegalaxy.org.au/published/workflow?id=e439a0e17a90d24c, https://usegalaxy.org.au/published/workflow?id=fca05a51c9c91df4, https://usegalaxy.org.au/published/workflow?id=fe5542c99cdc57ce, https://usegalaxy.org/published/workflow?id=00596e862edb23f2, https://usegalaxy.org/published/workflow?id=0a57740c68e85e21, https://usegalaxy.org/published/workflow?id=0fa2ce12c2874b1d, https://usegalaxy.org/published/workflow?id=1518c8c50747a1c1, https://usegalaxy.org/published/workflow?id=17d9b233640aefec, https://usegalaxy.org/published/workflow?id=1f48b44fa22ac5fc, https://usegalaxy.org/published/workflow?id=426e3951dac0704e, https://usegalaxy.org/published/workflow?id=4357180d40560019, https://usegalaxy.org/published/workflow?id=4b41a8247f52c463, https://usegalaxy.org/published/workflow?id=4cf8cc42e97b2da6, https://usegalaxy.org/published/workflow?id=562bcaa9b4d2e8a4, https://usegalaxy.org/published/workflow?id=5641d1cbe795203f, https://usegalaxy.org/published/workflow?id=69d8863d40524120, https://usegalaxy.org/published/workflow?id=6b11d738ee57f245, https://usegalaxy.org/published/workflow?id=7dd6e4150bb81ad1, https://usegalaxy.org/published/workflow?id=7dde0721a382ea3e, https://usegalaxy.org/published/workflow?id=7e2651e5ed17e3db, https://usegalaxy.org/published/workflow?id=80ac6adead9f97d8, https://usegalaxy.org/published/workflow?id=88d0b64011c3148c, https://usegalaxy.org/published/workflow?id=8b81b9a3b49bbf98, https://usegalaxy.org/published/workflow?id=8e1c6ad6ff623015, https://usegalaxy.org/published/workflow?id=953d9cbe05e632a8, https://usegalaxy.org/published/workflow?id=97263f3308077ff0, https://usegalaxy.org/published/workflow?id=98b2ac8fc75781f8, https://usegalaxy.org/published/workflow?id=9adf8691809326e5, https://usegalaxy.org/published/workflow?id=9fc9f1db50019929, https://usegalaxy.org/published/workflow?id=b1b75a9323d25085, https://usegalaxy.org/published/workflow?id=b34ba9b2e4d8f050, https://usegalaxy.org/published/workflow?id=bd355e8ce58a139e, https://usegalaxy.org/published/workflow?id=bf3ccae8d5b6ef58, https://usegalaxy.org/published/workflow?id=c12e6b7896f8e114, https://usegalaxy.org/published/workflow?id=c7b2d2ccc4b0dec5, https://usegalaxy.org/published/workflow?id=cd56ed91856fcc6f, https://usegalaxy.org/published/workflow?id=cdf5dffbed938d49, https://usegalaxy.org/published/workflow?id=d3428dd172a78102, https://usegalaxy.org/published/workflow?id=d4ea6cdd40522eb1, https://usegalaxy.org/published/workflow?id=d96bd5c9fb553f2c, https://usegalaxy.org/published/workflow?id=e282f505ba9f729f, https://usegalaxy.org/published/workflow?id=e7ce46762faccaac, https://usegalaxy.org/published/workflow?id=eb4f7dbca0114270, https://workflowhub.eu/workflows/1035?version=1, https://workflowhub.eu/workflows/110?version=11, https://workflowhub.eu/workflows/111?version=5, https://workflowhub.eu/workflows/1260?version=2, https://workflowhub.eu/workflows/155?version=6, https://workflowhub.eu/workflows/1647?version=1, https://workflowhub.eu/workflows/1876?version=2, https://workflowhub.eu/workflows/2027?version=2, https://workflowhub.eu/workflows/439?version=4, https://workflowhub.eu/workflows/519?version=1, https://workflowhub.eu/workflows/628?version=1</td>
      <td>variant-analysis/aiv-analysis, variant-analysis/pox-tiled-amplicon, variant-analysis/tb-variant-analysis</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>quast</td>
      <td>quast</td>
      <td>Quast (Quality ASsessment Tool) evaluates genome assemblies.</td>
      <td>2016-09-07</td>
      <td>http://quast.bioinf.spbau.ru/</td>
      <td>5.3.0</td>
      <td>quast</td>
      <td>5.3.0</td>
      <td>Up-to-date</td>
      <td>Assembly</td>
      <td>Visualisation, Sequence assembly validation</td>
      <td>Visualisation, Sequence assembly validation</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/quast</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/quast</td>
      <td>quast</td>
      <td>QUAST</td>
      <td>QUAST stands for QUality ASsessment Tool.  It evaluates a quality of genome assemblies by computing various metrics and providing nice reports.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>11028</td>
      <td>11672</td>
      <td>120605</td>
      <td>126603</td>
      <td>13078</td>
      <td>14773</td>
      <td>78457</td>
      <td>86986</td>
      <td>4849</td>
      <td>5353</td>
      <td>46644</td>
      <td>50987</td>
      <td>613</td>
      <td>619</td>
      <td>14902</td>
      <td>14963</td>
      <td>279539</td>
      <td>260608</td>
      <td>32417</td>
      <td>29568</td>
      <td>https://usegalaxy.eu/published/workflow?id=007ce6405d196df3, https://usegalaxy.eu/published/workflow?id=01dc0619e050dc91, https://usegalaxy.eu/published/workflow?id=0404013d5c27210d, https://usegalaxy.eu/published/workflow?id=0563f58718be932d, https://usegalaxy.eu/published/workflow?id=085c227c9a56fcae, https://usegalaxy.eu/published/workflow?id=0ebfab0e957b7267, https://usegalaxy.eu/published/workflow?id=1524b189069b42a5, https://usegalaxy.eu/published/workflow?id=165089132248a0b0, https://usegalaxy.eu/published/workflow?id=1860e02bcb894a64, https://usegalaxy.eu/published/workflow?id=1e5f1d9a51032e44, https://usegalaxy.eu/published/workflow?id=1ea8bfbe6cb04531, https://usegalaxy.eu/published/workflow?id=2e0075791f7fa384, https://usegalaxy.eu/published/workflow?id=309fc4fc2e0da89c, https://usegalaxy.eu/published/workflow?id=30b514cfb9cce524, https://usegalaxy.eu/published/workflow?id=3303a04887a47675, https://usegalaxy.eu/published/workflow?id=34a07bf6407510ef, https://usegalaxy.eu/published/workflow?id=368d1ac0150ea0b6, https://usegalaxy.eu/published/workflow?id=374797e58377c40f, https://usegalaxy.eu/published/workflow?id=37da7b42cb063a3a, https://usegalaxy.eu/published/workflow?id=3d937648809ca4e7, https://usegalaxy.eu/published/workflow?id=4216424f032c055f, https://usegalaxy.eu/published/workflow?id=45c0933b5c522628, https://usegalaxy.eu/published/workflow?id=4ddbffe4b3fef275, https://usegalaxy.eu/published/workflow?id=56edd22d14392691, https://usegalaxy.eu/published/workflow?id=579ff83177c5ee09, https://usegalaxy.eu/published/workflow?id=5826e573b6f93762, https://usegalaxy.eu/published/workflow?id=5b940b283d91d99d, https://usegalaxy.eu/published/workflow?id=6158c3b5fc12044e, https://usegalaxy.eu/published/workflow?id=628062a8cf93261f, https://usegalaxy.eu/published/workflow?id=62c47e3c9d4f3aa3, https://usegalaxy.eu/published/workflow?id=667bac8d7453e5f1, https://usegalaxy.eu/published/workflow?id=69be0fb8276753ec, https://usegalaxy.eu/published/workflow?id=6d87481be47902e1, https://usegalaxy.eu/published/workflow?id=723e718f37e9ab91, https://usegalaxy.eu/published/workflow?id=7aa32aba3fde0708, https://usegalaxy.eu/published/workflow?id=7e48134082dab0a3, https://usegalaxy.eu/published/workflow?id=7ecd92cb382d8168, https://usegalaxy.eu/published/workflow?id=842e0ea7b1eb56af, https://usegalaxy.eu/published/workflow?id=8b2f53745e720d0f, https://usegalaxy.eu/published/workflow?id=8b8e89b9b8801f2f, https://usegalaxy.eu/published/workflow?id=8deb08d4f228e198, https://usegalaxy.eu/published/workflow?id=8f3ce55d44d9f369, https://usegalaxy.eu/published/workflow?id=9219557de92e0203, https://usegalaxy.eu/published/workflow?id=94c3d329c31c712f, https://usegalaxy.eu/published/workflow?id=968f0ac2617aa524, https://usegalaxy.eu/published/workflow?id=97312d273b6e8bd9, https://usegalaxy.eu/published/workflow?id=98bfad0c684011e9, https://usegalaxy.eu/published/workflow?id=9b9e1e28f3683f92, https://usegalaxy.eu/published/workflow?id=9ddb703e1db13504, https://usegalaxy.eu/published/workflow?id=a337bfb2eb1e583f, https://usegalaxy.eu/published/workflow?id=a847773b563b2328, https://usegalaxy.eu/published/workflow?id=a84bac86ec183d46, https://usegalaxy.eu/published/workflow?id=b1d9e56aef03672b, https://usegalaxy.eu/published/workflow?id=b2b46e71046b1999, https://usegalaxy.eu/published/workflow?id=b627aee461e50756, https://usegalaxy.eu/published/workflow?id=b7c2440621968526, https://usegalaxy.eu/published/workflow?id=b93333efdab316d8, https://usegalaxy.eu/published/workflow?id=ba1fda2318046543, https://usegalaxy.eu/published/workflow?id=c9ef2d2ce92e2dfb, https://usegalaxy.eu/published/workflow?id=cf8591391a17a914, https://usegalaxy.eu/published/workflow?id=d57d41e306241396, https://usegalaxy.eu/published/workflow?id=d67299c34e63f3d9, https://usegalaxy.eu/published/workflow?id=e532d98f4b368666, https://usegalaxy.eu/published/workflow?id=e8eb80ee09159bac, https://usegalaxy.eu/published/workflow?id=ea485dbabc76488d, https://usegalaxy.eu/published/workflow?id=eb5ab8bc82569eee, https://usegalaxy.eu/published/workflow?id=ecf48fa50af560a0, https://usegalaxy.eu/published/workflow?id=fba1671d835b6dd3, https://usegalaxy.eu/published/workflow?id=fca458178b3c794a, https://usegalaxy.eu/published/workflow?id=fde21b5f5c169114, https://usegalaxy.eu/published/workflow?id=fe43cf59e48bf556, https://usegalaxy.org.au/published/workflow?id=02c74095a7128c9d, https://usegalaxy.org.au/published/workflow?id=1ad4c47cee851dda, https://usegalaxy.org.au/published/workflow?id=243c67fa4312855e, https://usegalaxy.org.au/published/workflow?id=2e29b21cdec7d0cc, https://usegalaxy.org.au/published/workflow?id=30430b75d05f54eb, https://usegalaxy.org.au/published/workflow?id=39c724282ee008e3, https://usegalaxy.org.au/published/workflow?id=4023bca282ca9315, https://usegalaxy.org.au/published/workflow?id=434127690544d0e9, https://usegalaxy.org.au/published/workflow?id=478c79365fc9677f, https://usegalaxy.org.au/published/workflow?id=533dbb3242ee4255, https://usegalaxy.org.au/published/workflow?id=5b32e2f0d390a4ba, https://usegalaxy.org.au/published/workflow?id=5cd804644a34c8fd, https://usegalaxy.org.au/published/workflow?id=68f5c371adbe5490, https://usegalaxy.org.au/published/workflow?id=7fa13ccbf044edc5, https://usegalaxy.org.au/published/workflow?id=83e3ceceb1a92493, https://usegalaxy.org.au/published/workflow?id=874492eb776ba813, https://usegalaxy.org.au/published/workflow?id=8a40059343cf4b87, https://usegalaxy.org.au/published/workflow?id=8b6739a000780d99, https://usegalaxy.org.au/published/workflow?id=8d47811c6154d347, https://usegalaxy.org.au/published/workflow?id=8f7d5ebead46a6d0, https://usegalaxy.org.au/published/workflow?id=92fcdd490831c6c5, https://usegalaxy.org.au/published/workflow?id=9b75924c483b5f6b, https://usegalaxy.org.au/published/workflow?id=aee5b5645c51cac3, https://usegalaxy.org.au/published/workflow?id=b6bad1f167257248, https://usegalaxy.org.au/published/workflow?id=b7aa22291322e505, https://usegalaxy.org.au/published/workflow?id=d06665e4bc7fff6d, https://usegalaxy.org.au/published/workflow?id=d832905fbaf2ca31, https://usegalaxy.org.au/published/workflow?id=d957bfeb3cd03db0, https://usegalaxy.org.au/published/workflow?id=df11f09771e73e04, https://usegalaxy.org.au/published/workflow?id=ee7a1615f187391d, https://usegalaxy.org.au/published/workflow?id=f01be6a7685ac476, https://usegalaxy.org.au/published/workflow?id=f25a05070bf97ded, https://usegalaxy.org.au/published/workflow?id=fb0986a952c35e0a, https://usegalaxy.org.au/published/workflow?id=fc664a96d8099652, https://usegalaxy.org/published/workflow?id=00596e862edb23f2, https://usegalaxy.org/published/workflow?id=0602c14704dbe1ff, https://usegalaxy.org/published/workflow?id=0e30324072570abb, https://usegalaxy.org/published/workflow?id=109092d8dbda4398, https://usegalaxy.org/published/workflow?id=1c7af480caf150bd, https://usegalaxy.org/published/workflow?id=3cf46c167ccb374f, https://usegalaxy.org/published/workflow?id=4029a0c8fa23d375, https://usegalaxy.org/published/workflow?id=4161cfd67ed94b5d, https://usegalaxy.org/published/workflow?id=4d1279007d0137fa, https://usegalaxy.org/published/workflow?id=4e1ae78405034ad3, https://usegalaxy.org/published/workflow?id=51c485af11d90700, https://usegalaxy.org/published/workflow?id=524602abe59b18f7, https://usegalaxy.org/published/workflow?id=547f86db467bf620, https://usegalaxy.org/published/workflow?id=562bcaa9b4d2e8a4, https://usegalaxy.org/published/workflow?id=5cd56232e873beab, https://usegalaxy.org/published/workflow?id=69d8863d40524120, https://usegalaxy.org/published/workflow?id=7a33bc2d0970b502, https://usegalaxy.org/published/workflow?id=7dd6e4150bb81ad1, https://usegalaxy.org/published/workflow?id=80565301ada0d3eb, https://usegalaxy.org/published/workflow?id=80ac6adead9f97d8, https://usegalaxy.org/published/workflow?id=8111bb9bb26bb665, https://usegalaxy.org/published/workflow?id=88bc9aa7090875c1, https://usegalaxy.org/published/workflow?id=9ac1e4efe310b3b3, https://usegalaxy.org/published/workflow?id=9c0fbd9f51a9857f, https://usegalaxy.org/published/workflow?id=9e5c5ddb60a24889, https://usegalaxy.org/published/workflow?id=a67afbea4b2eb26b, https://usegalaxy.org/published/workflow?id=b76a9ce44ea019f1, https://usegalaxy.org/published/workflow?id=c30f6ca171896908, https://usegalaxy.org/published/workflow?id=c454c930ccfb0d3f, https://usegalaxy.org/published/workflow?id=c65bec6c332bf740, https://usegalaxy.org/published/workflow?id=c8d74054c1820087, https://usegalaxy.org/published/workflow?id=cdcf91a8ddb5dcd1, https://usegalaxy.org/published/workflow?id=db08887d7d6dc147, https://usegalaxy.org/published/workflow?id=e2ab3d3219a648a7, https://usegalaxy.org/published/workflow?id=e745c45651d829b8, https://usegalaxy.org/published/workflow?id=f4d623a19b32370e, https://usegalaxy.org/published/workflow?id=f926e70bee242785, https://usegalaxy.org/published/workflow?id=fb01e1a49793c8be, https://workflowhub.eu/workflows/1352?version=4, https://workflowhub.eu/workflows/1390?version=2, https://workflowhub.eu/workflows/1580?version=1, https://workflowhub.eu/workflows/1595?version=1, https://workflowhub.eu/workflows/1598?version=1, https://workflowhub.eu/workflows/1602?version=2, https://workflowhub.eu/workflows/1607?version=1, https://workflowhub.eu/workflows/1613?version=2, https://workflowhub.eu/workflows/1621?version=1, https://workflowhub.eu/workflows/1622?version=1, https://workflowhub.eu/workflows/1634?version=1, https://workflowhub.eu/workflows/1646?version=1, https://workflowhub.eu/workflows/1882?version=3, https://workflowhub.eu/workflows/2011?version=1, https://workflowhub.eu/workflows/2024?version=2, https://workflowhub.eu/workflows/225?version=1, https://workflowhub.eu/workflows/229?version=1, https://workflowhub.eu/workflows/403?version=9, https://workflowhub.eu/workflows/750?version=3, https://workflowhub.eu/workflows/794?version=2</td>
      <td>assembly/assembly-quality-control, assembly/general-introduction, assembly/hifi-assembly, assembly/hybrid_denovo_assembly, assembly/largegenome, assembly/metagenomics-assembly, assembly/mrsa-illumina, assembly/mrsa-nanopore, assembly/unicycler-assembly, assembly/vgp_genome_assembly, assembly/vgp_workflow_training, microbiome/metagenomics-assembly</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>quickmerge</td>
      <td>quickmerge</td>
      <td>Merge long-read and hybrid assemblies to increase contiguity</td>
      <td>2022-07-08</td>
      <td>https://github.com/mahulchak/quickmerge</td>
      <td>0.3</td>
      <td>quickmerge</td>
      <td>0.3</td>
      <td>Up-to-date</td>
      <td>Assembly</td>
      <td>Genome assembly, Scaffolding, De-novo assembly, Genotyping</td>
      <td>Genome assembly, Scaffolding, De-novo assembly, Genotyping</td>
      <td>Structural variation, Sequence assembly, DNA polymorphism, Whole genome sequencing, Genotype and phenotype</td>
      <td>Structural variation, Sequence assembly, DNA polymorphism, Whole genome sequencing, Genotype and phenotype</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/quickmerge</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/quickmerge</td>
      <td>quickmerge</td>
      <td>quickmerge</td>
      <td>Quickmerge is a program that uses complementary information from genomes assembled with long reads in order to improve contiguity, and works with assemblies derived from both Pacific Biosciences or Oxford Nanopore. Quickmerge will even work with hybrid assemblies made by combining long reads and Illumina short reads.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>quicktree</td>
      <td>quicktree</td>
      <td>neighbour-joining phylogenetic inference</td>
      <td>2024-11-21</td>
      <td>https://github.com/khowe/quicktree</td>
      <td>2.5</td>
      <td>quicktree</td>
      <td>2.5</td>
      <td>Up-to-date</td>
      <td>Phylogenetics</td>
      <td>Phylogenetic inference (from molecular sequences)</td>
      <td>Phylogenetic inference (from molecular sequences)</td>
      <td>Phylogenetics</td>
      <td>Phylogenetics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/quicktree</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/quicktree</td>
      <td>quicktree</td>
      <td>QuickTree</td>
      <td>QuickTree is an efficient implementation of the Neighbor-Joining algorithm (PMID: 3447015), capable of reconstructing phylogenies from huge alignments in time less than the age of the universe.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>69</td>
      <td>69</td>
      <td>466</td>
      <td>466</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>466</td>
      <td>466</td>
      <td>69</td>
      <td>69</td>
      <td>https://workflowhub.eu/workflows/1650?version=1</td>
      <td>sequence-analysis/tapscan-streptophyte-algae</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>racon</td>
      <td>racon</td>
      <td>Consensus module for raw de novo DNA assembly of long uncorrected reads.</td>
      <td>2018-06-11</td>
      <td>https://github.com/isovic/racon</td>
      <td>1.5.0</td>
      <td>racon</td>
      <td>1.5.0</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Genome assembly, Mapping assembly</td>
      <td>Genome assembly, Mapping assembly</td>
      <td>Whole genome sequencing, Sequence assembly</td>
      <td>Whole genome sequencing, Sequence assembly</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/racon</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/racon</td>
      <td>Racon</td>
      <td>Racon</td>
      <td>Consensus module for raw de novo DNA assembly of long uncorrected readsRacon is intended as a standalone consensus module to correct raw contigs generated by rapid assembly methods which do not include a consensus step. The goal of Racon is to generate genomic consensus which is of similar or better quality compared to the output generated by assembly methods which employ both error correction and consensus steps, while providing a speedup of several times compared to those methods. It supports data produced by both Pacific Biosciences and Oxford Nanopore Technologies.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1046</td>
      <td>1151</td>
      <td>32503</td>
      <td>37525</td>
      <td>683</td>
      <td>683</td>
      <td>23074</td>
      <td>23074</td>
      <td>263</td>
      <td>268</td>
      <td>9144</td>
      <td>9214</td>
      <td>31</td>
      <td>31</td>
      <td>320</td>
      <td>320</td>
      <td>70133</td>
      <td>65041</td>
      <td>2133</td>
      <td>2023</td>
      <td>https://usegalaxy.eu/published/workflow?id=249c115413f17f28, https://usegalaxy.eu/published/workflow?id=32d2f5340098bcbd, https://usegalaxy.eu/published/workflow?id=39837ca09374598b, https://usegalaxy.eu/published/workflow?id=44e53ea319e07daa, https://usegalaxy.eu/published/workflow?id=4eb9dfe91fc2dc67, https://usegalaxy.eu/published/workflow?id=5500de2961f7470c, https://usegalaxy.eu/published/workflow?id=5826e573b6f93762, https://usegalaxy.eu/published/workflow?id=8b2f53745e720d0f, https://usegalaxy.eu/published/workflow?id=a337bfb2eb1e583f, https://usegalaxy.eu/published/workflow?id=a4f50449afde4a05, https://usegalaxy.eu/published/workflow?id=a84bac86ec183d46, https://usegalaxy.eu/published/workflow?id=b4de76c0a4d0e23a, https://usegalaxy.eu/published/workflow?id=c48807045e258dd8, https://usegalaxy.eu/published/workflow?id=dc7107896a3f2964, https://usegalaxy.org.au/published/workflow?id=0c421197e022f323, https://usegalaxy.org.au/published/workflow?id=1ff6aabb7e3954ac, https://usegalaxy.org.au/published/workflow?id=45e9910f46a210d8, https://usegalaxy.org.au/published/workflow?id=d9571b3dfef34e5e, https://usegalaxy.org/published/workflow?id=bcdc5ce2ea8700fc, https://workflowhub.eu/workflows/1477?version=1, https://workflowhub.eu/workflows/1599?version=1, https://workflowhub.eu/workflows/1604?version=1, https://workflowhub.eu/workflows/227?version=1, https://workflowhub.eu/workflows/228?version=1, https://workflowhub.eu/workflows/406?version=1, https://workflowhub.eu/workflows/51?version=1, https://workflowhub.eu/workflows/563?version=1</td>
      <td>assembly/largegenome, microbiome/plasmid-metagenomics-nanopore</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>rasusa</td>
      <td>rasusa</td>
      <td>Randomly subsample sequencing reads to a specified coverage</td>
      <td>2024-02-16</td>
      <td>https://github.com/mbhall88/rasusa</td>
      <td>2.0.0</td>
      <td>rasusa</td>
      <td>2.2.2</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Sequence assembly validation, Sequencing quality control</td>
      <td>Sequence assembly validation</td>
      <td>Genomics, Sequence analysis, Sequence assembly, Sequencing, RNA-Seq</td>
      <td>Genomics, Sequence analysis, Sequence assembly, RNA-Seq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/rasusa</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/rasusa</td>
      <td>rasusa</td>
      <td>rasusa</td>
      <td>Rasusa: Randomly subsample sequencing reads to aspecified coverageProduces an unbiased subsample of your reads</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>24</td>
      <td>24</td>
      <td>104</td>
      <td>104</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>104</td>
      <td>104</td>
      <td>24</td>
      <td>24</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>raxml</td>
      <td>raxml</td>
      <td>RAxML - A Maximum Likelihood based phylogenetic inference</td>
      <td>2015-11-05</td>
      <td>http://www.exelixis-lab.org/web/software/raxml/</td>
      <td>8.2.12</td>
      <td>raxml</td>
      <td>8.2.13</td>
      <td>To update</td>
      <td>Phylogenetics</td>
      <td>Sequence analysis, Phylogenetic tree analysis</td>
      <td>Sequence analysis</td>
      <td>Phylogenetics, Sequence analysis</td>
      <td>Phylogenetics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/raxml</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/raxml</td>
      <td>raxml</td>
      <td>RAxML</td>
      <td>A tool for Phylogenetic Analysis and Post-Analysis of Large Phylogenies.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1422</td>
      <td>1464</td>
      <td>15492</td>
      <td>16986</td>
      <td>1144</td>
      <td>1144</td>
      <td>19599</td>
      <td>19633</td>
      <td>731</td>
      <td>1034</td>
      <td>5870</td>
      <td>7945</td>
      <td>21</td>
      <td>21</td>
      <td>305</td>
      <td>305</td>
      <td>44869</td>
      <td>41266</td>
      <td>3663</td>
      <td>3318</td>
      <td>https://usegalaxy.eu/published/workflow?id=470656ef317c9dd5, https://usegalaxy.eu/published/workflow?id=875b3ca170ac12a1, https://usegalaxy.org/published/workflow?id=7251c6e0dad8f927, https://usegalaxy.org/published/workflow?id=d4f0a7ccb09f1e55, https://workflowhub.eu/workflows/1577?version=1</td>
      <td>evolution/mtb_phylogeny</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>raxmlng</td>
      <td>raxmlng</td>
      <td>RAxML-NG - A Maximum Likelihood based phylogenetic tree inference</td>
      <td>2025-12-20</td>
      <td>http://www.exelixis-lab.org/web/software/raxml-ng/</td>
      <td>1.2.2</td>
      <td>raxml-ng</td>
      <td>1.2.2</td>
      <td>Up-to-date</td>
      <td>Phylogenetics</td>
      <td>Phylogenetic tree generation (maximum likelihood and Bayesian methods), Phylogenetic tree bootstrapping</td>
      <td>Phylogenetic tree bootstrapping</td>
      <td>Phylogenetics, Agricultural science, Phylogenomics</td>
      <td>Phylogenetics, Agricultural science, Phylogenomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/raxml-ng</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/raxmlng</td>
      <td>RAxML-NG</td>
      <td>RAxML-NG</td>
      <td>Phylogenetic tree inference tool which uses maximum-likelihood (ML) optimality criterion.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>read2tree</td>
      <td>read2tree</td>
      <td>Infer a species tree from sequencing reads.</td>
      <td>2025-11-23</td>
      <td>https://github.com/DessimozLab/read2tree</td>
      <td>2.0.1</td>
      <td>read2tree</td>
      <td>2.0.1</td>
      <td>Up-to-date</td>
      <td>Phylogenetics</td>
      <td></td>
      <td></td>
      <td>Phylogenomics</td>
      <td>Phylogenomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/read2tree</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/read2tree</td>
      <td>read2tree</td>
      <td>read2tree</td>
      <td>read2tree is a software tool for generating alignment matrices and performing tree inference directly from sequencing reads. It leverages the OMA database and a set of input reads, bypassing many of the standard steps typically required in phylogenomic analysis. In particular, it avoids for read filtering, assembly, gene prediction, gene annotation, all-vs-all comparison, orthology prediction, alignment, and concatenation.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>read_it_and_keep</td>
      <td>read_it_and_keep</td>
      <td>Rapid decontamination of SARS-CoV-2 sequencing reads</td>
      <td>2022-01-28</td>
      <td>https://github.com/GenomePathogenAnalysisService/read-it-and-keep</td>
      <td>0.2.2</td>
      <td>read-it-and-keep</td>
      <td>0.3.0</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Filtering, Genome alignment</td>
      <td>Filtering, Genome alignment</td>
      <td>Pathology, Genomics</td>
      <td>Pathology, Genomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/read_it_and_keep</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/read_it_and_keep</td>
      <td>read_it_and_keep</td>
      <td>read_it_and_keep</td>
      <td>Read-it-and-keep is a bioinformatics tool designed to filter and extract sequencing reads according to user-defined criteria, such as the presence of specific motifs or coverage of regions of interest. This type of filtering can be particularly useful for targeted analyses, where only relevant sequences are retained for further study, optimising computing and storage resources.</td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>81</td>
      <td>81</td>
      <td>3781</td>
      <td>3781</td>
      <td>8</td>
      <td>8</td>
      <td>9</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3790</td>
      <td>3790</td>
      <td>89</td>
      <td>89</td>
      <td>https://usegalaxy.eu/published/workflow?id=3fb550d7c2bc390a, https://usegalaxy.eu/published/workflow?id=4a764275d4bf9f31, https://usegalaxy.eu/published/workflow?id=629e57958afed44c, https://usegalaxy.eu/published/workflow?id=7c986f4d14c6a539, https://usegalaxy.eu/published/workflow?id=d88145e7475e0b7b, https://usegalaxy.eu/published/workflow?id=d967194ca51ff11e, https://workflowhub.eu/workflows/519?version=1, https://workflowhub.eu/workflows/521?version=1</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>reago</td>
      <td>reago</td>
      <td>Reago is tool to assembly 16S ribosomal RNA recovery from metagenomic data.</td>
      <td>2015-12-09</td>
      <td>https://github.com/chengyuan/reago-1.1</td>
      <td>1.1</td>
      <td>reago</td>
      <td>1.1</td>
      <td>Up-to-date</td>
      <td>Metagenomics, RNA</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly, RNA, Metagenomics, Microbiology</td>
      <td>Sequence assembly, RNA, Metagenomics, Microbiology</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/reago</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/reago</td>
      <td>reago</td>
      <td>REAGO</td>
      <td>This is an assembly tool for 16S ribosomal RNA recovery from metagenomic data.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>18</td>
      <td>18</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>18</td>
      <td>18</td>
      <td>7</td>
      <td>7</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>recentrifuge</td>
      <td>recentrifuge</td>
      <td>"With Recentrifuge, researchers can analyze results from taxonomic classifiers using interactive charts with emphasis on the confidence level of the classifications.In addition to contamination-subtracted samples.Recentrifuge provides shared and exclusive taxa per sample,thus enabling robust contamination removal and comparative analysis in environmental and clinical metagenomics."</td>
      <td>2022-05-04</td>
      <td>https://github.com/khyox/recentrifuge</td>
      <td>1.16.1</td>
      <td>recentrifuge</td>
      <td>2.0.0</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Taxonomic classification, Expression analysis, Cross-assembly</td>
      <td>Taxonomic classification, Expression analysis, Cross-assembly</td>
      <td>Metagenomics, Microbial ecology, Metagenomic sequencing</td>
      <td>Metagenomics, Microbial ecology, Metagenomic sequencing</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/recentrifuge</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/recentrifuge</td>
      <td>Recentrifuge</td>
      <td>Recentrifuge</td>
      <td>Robust comparative analysis and contamination removal for metagenomics.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>306</td>
      <td>306</td>
      <td>2546</td>
      <td>2546</td>
      <td>160</td>
      <td>160</td>
      <td>593</td>
      <td>593</td>
      <td>37</td>
      <td>37</td>
      <td>105</td>
      <td>105</td>
      <td>27</td>
      <td>27</td>
      <td>2397</td>
      <td>2397</td>
      <td>5641</td>
      <td>5641</td>
      <td>530</td>
      <td>530</td>
      <td>https://usegalaxy.eu/published/workflow?id=0a82eb8b44d9d502, https://usegalaxy.eu/published/workflow?id=0abd32dbc5207b88, https://usegalaxy.eu/published/workflow?id=0c47571d20535426, https://usegalaxy.eu/published/workflow?id=2fa2f67603772413, https://usegalaxy.eu/published/workflow?id=47e14222073e7192, https://usegalaxy.eu/published/workflow?id=884a4415b669690c, https://usegalaxy.eu/published/workflow?id=aafcfcb585c5ca75, https://usegalaxy.eu/published/workflow?id=bb63b5bb983fd49c, https://usegalaxy.eu/published/workflow?id=bdc7cfaac0db2eab, https://usegalaxy.eu/published/workflow?id=c09159d7aad0f264, https://usegalaxy.eu/published/workflow?id=c14d2e8b3c191bc4, https://usegalaxy.eu/published/workflow?id=c930a54a30dcc7e3, https://usegalaxy.eu/published/workflow?id=e69b555faa6bb962, https://usegalaxy.eu/published/workflow?id=ebb131f2a6172081, https://usegalaxy.fr/published/workflow?id=3d3b40398d1cd1ae, https://usegalaxy.org/published/workflow?id=1be5c982c2d36a6c, https://usegalaxy.org/published/workflow?id=2a782ad24271f570, https://usegalaxy.org/published/workflow?id=323c651ca583c9e5, https://usegalaxy.org/published/workflow?id=4f698eaeef30e364, https://usegalaxy.org/published/workflow?id=623fc80c7f91a33a, https://usegalaxy.org/published/workflow?id=9f68718f025bed30, https://usegalaxy.org/published/workflow?id=b6a795834009ce06, https://usegalaxy.org/published/workflow?id=de9bf4c1024f8d53, https://usegalaxy.org/published/workflow?id=dfd73857c0eb2a21, https://usegalaxy.org/published/workflow?id=f378960ee55f567e, https://workflowhub.eu/workflows/1052?version=11, https://workflowhub.eu/workflows/1644?version=1, https://workflowhub.eu/workflows/1674?version=2, https://workflowhub.eu/workflows/1881?version=2, https://workflowhub.eu/workflows/1882?version=3, https://workflowhub.eu/workflows/2043?version=1</td>
      <td>ecology/bacterial-isolate-species-contamination-checking, sequence-analysis/quality-contamination-control</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>remove_terminal_stop_codons</td>
      <td>remove_terminal_stop_codons</td>
      <td>Remove terminal stop codons from coding sequences</td>
      <td>2025-12-04</td>
      <td>https://github.com/veg/CAPHEINE</td>
      <td>1.0.0</td>
      <td>biopython</td>
      <td>1.70</td>
      <td>To update</td>
      <td>Phylogenetics, Sequence Analysis, Fasta Manipulation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/remove_terminal_stop_codons</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/remove_terminal_stop_codons</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>repeatexplorer2</td>
      <td>repeatexplorer_clustering</td>
      <td>Tool for annotation of repeats from unassembled shotgun reads.</td>
      <td>2023-11-01</td>
      <td>https://github.com/repeatexplorer/repex_tarean</td>
      <td>2.3.8</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>Genome annotation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>gga</td>
      <td>https://github.com/galaxy-genome-annotation/galaxy-tools/tree/master/tools/repeatexplorer2</td>
      <td>https://github.com/galaxy-genome-annotation/galaxy-tools/tree/master/tools/repeatexplorer2</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>83</td>
      <td>83</td>
      <td>369</td>
      <td>369</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>369</td>
      <td>369</td>
      <td>83</td>
      <td>83</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>roary</td>
      <td>roary</td>
      <td>Roary the pangenome pipeline</td>
      <td>2017-06-21</td>
      <td>https://sanger-pathogens.github.io/Roary/</td>
      <td>3.13.0</td>
      <td>roary</td>
      <td>3.13.0</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Genome assembly</td>
      <td>Genome assembly</td>
      <td>DNA, Genomics, Mapping</td>
      <td>DNA, Genomics, Mapping</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/roary</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/roary</td>
      <td>roary</td>
      <td>Roary</td>
      <td>A high speed stand alone pan genome pipeline, which takes annotated assemblies in GFF3 format (produced by Prokka (Seemann, 2014)) and calculates the pan genome.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2251</td>
      <td>2377</td>
      <td>21218</td>
      <td>23887</td>
      <td>2319</td>
      <td>2454</td>
      <td>17744</td>
      <td>18851</td>
      <td>653</td>
      <td>749</td>
      <td>5251</td>
      <td>7169</td>
      <td>47</td>
      <td>47</td>
      <td>187</td>
      <td>187</td>
      <td>50094</td>
      <td>44400</td>
      <td>5627</td>
      <td>5270</td>
      <td>https://usegalaxy.eu/published/workflow?id=628062a8cf93261f, https://usegalaxy.eu/published/workflow?id=94c3d329c31c712f, https://usegalaxy.eu/published/workflow?id=b1a799497c366eef, https://usegalaxy.eu/published/workflow?id=e16e524c557d7928, https://usegalaxy.org.au/published/workflow?id=05f3abecd4ddd4a1, https://usegalaxy.org.au/published/workflow?id=3fd2195b6928d5e0, https://usegalaxy.org/published/workflow?id=78d8b738f5715a8a, https://usegalaxy.org/published/workflow?id=80565301ada0d3eb</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>rseqc</td>
      <td>rseqc_FPKM_count, rseqc_RNA_fragment_size, rseqc_RPKM_saturation, rseqc_bam2wig, rseqc_bam_stat, rseqc_clipping_profile, rseqc_deletion_profile, rseqc_geneBody_coverage, rseqc_geneBody_coverage2, rseqc_infer_experiment, rseqc_inner_distance, rseqc_insertion_profile, rseqc_junction_annotation, rseqc_junction_saturation, rseqc_mismatch_profile, rseqc_read_GC, rseqc_read_NVC, rseqc_read_distribution, rseqc_read_duplication, rseqc_read_hexamer, rseqc_read_quality, rseqc_tin</td>
      <td>an RNA-seq quality control package</td>
      <td>2017-02-27</td>
      <td>https://code.google.com/p/rseqc/</td>
      <td>5.0.3</td>
      <td>rseqc</td>
      <td>5.0.4</td>
      <td>To update</td>
      <td>Convert Formats, Sequence Analysis, RNA, Transcriptomics, Visualization</td>
      <td>Data handling</td>
      <td>Data handling</td>
      <td>Sequencing</td>
      <td>Sequencing</td>
      <td>nilesh</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/rseqc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/rseqc</td>
      <td>rseqc</td>
      <td>RSeQC</td>
      <td>Provides a number of useful modules that can comprehensively evaluate high throughput sequence data especially RNA-seq data. Some basic modules quickly inspect sequence quality, nucleotide composition bias, PCR bias and GC bias, while RNA-seq specific modules evaluate sequencing saturation, mapped reads distribution, coverage uniformity, strand specificity, transcript level RNA integrity etc.</td>
      <td></td>
      <td>22</td>
      <td>22</td>
      <td>22</td>
      <td>22</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>22</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>22</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>22</td>
      <td>22</td>
      <td>22</td>
      <td>0</td>
      <td>4113</td>
      <td>5223</td>
      <td>155237</td>
      <td>181534</td>
      <td>4079</td>
      <td>5016</td>
      <td>146353</td>
      <td>197501</td>
      <td>629</td>
      <td>697</td>
      <td>28108</td>
      <td>32069</td>
      <td>809</td>
      <td>810</td>
      <td>78294</td>
      <td>78366</td>
      <td>489470</td>
      <td>407992</td>
      <td>11746</td>
      <td>9630</td>
      <td>https://usegalaxy.eu/published/workflow?id=1ee43d90b69829e9, https://usegalaxy.eu/published/workflow?id=1f9a9a7998ef5ab3, https://usegalaxy.eu/published/workflow?id=20ada53472782c9b, https://usegalaxy.eu/published/workflow?id=47a0407867a90f95, https://usegalaxy.eu/published/workflow?id=5e4135eeb4972709, https://usegalaxy.eu/published/workflow?id=93d1035a307fe63d, https://usegalaxy.eu/published/workflow?id=985044b4b2c51425, https://usegalaxy.eu/published/workflow?id=9b025479fade745d, https://usegalaxy.eu/published/workflow?id=9c7a218993788493, https://usegalaxy.eu/published/workflow?id=a28c207730e7e13d, https://usegalaxy.eu/published/workflow?id=a52c5dd688def9ba, https://usegalaxy.eu/published/workflow?id=b7a3cbcd32734e8b, https://usegalaxy.eu/published/workflow?id=cc37667c2d1f9056, https://usegalaxy.eu/published/workflow?id=ccef778cfd9024d1, https://usegalaxy.eu/published/workflow?id=f33b5135ee99e0dc, https://usegalaxy.fr/published/workflow?id=334189184455d9a2, https://usegalaxy.fr/published/workflow?id=5d83e0886f2a7465, https://usegalaxy.fr/published/workflow?id=74b4d11c07f7f0a4, https://usegalaxy.fr/published/workflow?id=826cffb2458e4067, https://usegalaxy.org.au/published/workflow?id=6e1e24f9e0326930, https://usegalaxy.org.au/published/workflow?id=75113a1b41904d2a, https://usegalaxy.org.au/published/workflow?id=85aede91db0122e4, https://usegalaxy.org.au/published/workflow?id=a763e53653b10ce0, https://usegalaxy.org.au/published/workflow?id=af7b95f46485308c, https://usegalaxy.org.au/published/workflow?id=b08342866a118a5e, https://usegalaxy.org.au/published/workflow?id=bb0feff5d764df60, https://usegalaxy.org.au/published/workflow?id=ce7f6a1c824920e2, https://usegalaxy.org.au/published/workflow?id=e1fd89e7248b6661, https://usegalaxy.org/published/workflow?id=044c5ce43f2e1829, https://usegalaxy.org/published/workflow?id=582f3de3971fc38c, https://usegalaxy.org/published/workflow?id=5944c5effd56b0d6, https://usegalaxy.org/published/workflow?id=682b1c7e2ad9d443, https://usegalaxy.org/published/workflow?id=7022c4d9af725e5c, https://usegalaxy.org/published/workflow?id=73f5b1a6fd3c703f, https://usegalaxy.org/published/workflow?id=76b8ed1c68678b62, https://usegalaxy.org/published/workflow?id=7e9269e78e989480, https://usegalaxy.org/published/workflow?id=a743746151101352, https://usegalaxy.org/published/workflow?id=b11796364c22235e, https://usegalaxy.org/published/workflow?id=b1a98a2d5ebda1c5, https://usegalaxy.org/published/workflow?id=e035be74e3c85bf5, https://usegalaxy.org/published/workflow?id=e33f6a058c5f57e5, https://usegalaxy.org/published/workflow?id=eb4b32fcf4a9f64a, https://usegalaxy.org/published/workflow?id=fb789920b057fe8d, https://workflowhub.eu/workflows/1103?version=3, https://workflowhub.eu/workflows/1104?version=2, https://workflowhub.eu/workflows/1675?version=1, https://workflowhub.eu/workflows/1688?version=1, https://workflowhub.eu/workflows/1692?version=1, https://workflowhub.eu/workflows/1695?version=1, https://workflowhub.eu/workflows/688?version=1</td>
      <td>transcriptomics/differential-isoform-expression, transcriptomics/minerva-pathways, transcriptomics/ref-based, transcriptomics/rna-seq-reads-to-counts</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>scoary</td>
      <td>scoary</td>
      <td>Scoary calculates the assocations between all genes in the accessory genome and the traits.</td>
      <td>2021-03-18</td>
      <td>https://github.com/AdmiralenOla/Scoary</td>
      <td>1.6.16</td>
      <td>scoary</td>
      <td>1.6.16</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Analysis</td>
      <td>Analysis</td>
      <td>Genotype and phenotype, Model organisms, GWAS study, Functional genomics</td>
      <td>Genotype and phenotype, Model organisms, GWAS study, Functional genomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/scoary</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/scoary</td>
      <td>scoary</td>
      <td>Scoary</td>
      <td>Pan-genome wide association studies and  is designed to take the gene_presence_absence.csv file from Roary as well as a traits file created by the user and calculate the assocations between all genes in the accessory genome (all genes that are present in i genomes where 1 &lt; i &lt; N) and the traits. It reports a list of genes sorted by strength of association per trait.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>113</td>
      <td>113</td>
      <td>1366</td>
      <td>1366</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1366</td>
      <td>1366</td>
      <td>113</td>
      <td>113</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>semibin</td>
      <td>semibin_bin, semibin_concatenate_fasta, semibin_generate_cannot_links, semibin_generate_sequence_features, semibin, semibin_train</td>
      <td>SemiBin: Semi-supervised Metagenomic Binning Using Siamese Neural Networks</td>
      <td>2022-10-14</td>
      <td>https://semibin.readthedocs.io/en/latest/</td>
      <td>2.1.0</td>
      <td>semibin</td>
      <td>2.2.1</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Sequence assembly, Read binning</td>
      <td>Sequence assembly, Read binning</td>
      <td>Metagenomics, Machine learning, Microbial ecology, Sequence assembly</td>
      <td>Metagenomics, Machine learning, Microbial ecology, Sequence assembly</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/semibin</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/semibin</td>
      <td>semibin</td>
      <td>SemiBin</td>
      <td>Command tool for metagenomic binning with semi-supervised deep learning using information from reference genomes.</td>
      <td></td>
      <td>1</td>
      <td>6</td>
      <td>6</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>53</td>
      <td>53</td>
      <td>2779</td>
      <td>2779</td>
      <td>9</td>
      <td>9</td>
      <td>18</td>
      <td>18</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>75</td>
      <td>75</td>
      <td>2872</td>
      <td>2872</td>
      <td>64</td>
      <td>64</td>
      <td>https://usegalaxy.eu/published/workflow?id=00f15824c7a9b196, https://usegalaxy.eu/published/workflow?id=0563f58718be932d, https://usegalaxy.eu/published/workflow?id=1e5f1d9a51032e44, https://usegalaxy.eu/published/workflow?id=43098b8afe78b071, https://usegalaxy.eu/published/workflow?id=7ecd92cb382d8168, https://usegalaxy.eu/published/workflow?id=97312d273b6e8bd9, https://usegalaxy.eu/published/workflow?id=ce5c9287a67e6085, https://workflowhub.eu/workflows/1352?version=4, https://workflowhub.eu/workflows/2028?version=1</td>
      <td>microbiome/metagenomics-binning</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>seqkit</td>
      <td>seqkit_fx2tab, seqkit_grep, seqkit_head, seqkit_locate, seqkit_sort, seqkit_split2, seqkit_stats, seqkit_translate</td>
      <td>A cross-platform and ultrafast toolkit for FASTA/Q file manipulation</td>
      <td>2022-06-01</td>
      <td>https://bioinf.shenwei.me/seqkit/</td>
      <td>2.12.0</td>
      <td>seqkit</td>
      <td>2.12.0</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>DNA transcription, Sequence trimming, DNA translation, Sequence conversion</td>
      <td>DNA transcription, Sequence trimming, DNA translation</td>
      <td>Database management, Sequence analysis</td>
      <td>Database management, Sequence analysis</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/seqkit</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/seqkit</td>
      <td>seqkit</td>
      <td>seqkit</td>
      <td>FASTA and FASTQ are basic and ubiquitous formats for storing nucleotide and protein sequences. Common manipulations of FASTA/Q file include converting, searching, filtering, deduplication, splitting, shuffling, and sampling. Existing tools only implement some of these manipulations, and not particularly efficiently, and some are only available for certain operating systems. Furthermore, the complicated installation process of required packages and running environments can render these programs less user friendly. SeqKit demonstrates competitive performance in execution time and memory usage compared to similar tools. The efficiency and usability of SeqKit enable researchers to rapidly accomplish common FASTA/Q file manipulations.</td>
      <td></td>
      <td>3</td>
      <td>6</td>
      <td>7</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>396</td>
      <td>396</td>
      <td>13926</td>
      <td>13926</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>120</td>
      <td>120</td>
      <td>2190</td>
      <td>2190</td>
      <td>69</td>
      <td>69</td>
      <td>648</td>
      <td>648</td>
      <td>16764</td>
      <td>16764</td>
      <td>585</td>
      <td>585</td>
      <td>https://usegalaxy.org.au/published/workflow?id=7559224fc32dce3a, https://usegalaxy.org.au/published/workflow?id=7a0c4804e4d4c905, https://usegalaxy.org.au/published/workflow?id=ed3e48a73b5aaa56, https://workflowhub.eu/workflows/2024?version=2, https://workflowhub.eu/workflows/2036?version=1, https://workflowhub.eu/workflows/2038?version=1, https://workflowhub.eu/workflows/601?version=2, https://workflowhub.eu/workflows/602?version=2, https://workflowhub.eu/workflows/697?version=1</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>seqprep</td>
      <td>seqprep</td>
      <td>Tool for merging paired-end Illumina reads and trimming adapters.</td>
      <td>2024-01-15</td>
      <td>https://github.com/jstjohn/SeqPrep</td>
      <td>1.3.2</td>
      <td>seqprep</td>
      <td>1.3.2</td>
      <td>Up-to-date</td>
      <td>Fastq Manipulation, Sequence Analysis</td>
      <td>Nucleic acid design</td>
      <td>Nucleic acid design</td>
      <td>Genomics, Sequence assembly, Sequencing, Probes and primers</td>
      <td>Genomics, Sequence assembly, Sequencing, Probes and primers</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/seqprep</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/seqprep</td>
      <td>seqprep</td>
      <td>SeqPrep</td>
      <td>Strips adapters and optionally merges overlapping paired-end (or paired-end contamination in mate-pair libraries) illumina style reads.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>227</td>
      <td>227</td>
      <td>3602</td>
      <td>3602</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3602</td>
      <td>3602</td>
      <td>227</td>
      <td>227</td>
      <td></td>
      <td>microbiome/mgnify-amplicon</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>seqsero2</td>
      <td>seqsero2</td>
      <td>Salmonella serotype prediction from genome sequencing data</td>
      <td>2023-11-07</td>
      <td>https://github.com/denglab/SeqSero2</td>
      <td>1.3.2</td>
      <td>seqsero2</td>
      <td>1.3.2</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Genome indexing, Antimicrobial resistance prediction, Genome alignment</td>
      <td>Genome indexing, Antimicrobial resistance prediction, Genome alignment</td>
      <td>Whole genome sequencing, Sequence assembly, Genomics</td>
      <td>Whole genome sequencing, Sequence assembly, Genomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/seqsero2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/seqsero2</td>
      <td>seqsero2</td>
      <td>SeqSero2</td>
      <td>rapid and improved Salmonella serotype determination using whole genome sequencing data | SeqSero-Salmonella Serotyping by Whole Genome Sequencing | Salmonella Serotyping by Whole Genome Sequencing | Online version: http://www.denglab.info/SeqSero2 | Salmonella serotype prediction from genome sequencing data | Citation: SeqSero, Zhang et al. 2015; SeqSero2, Zhang et al. 2019 | Salmonella serotype derterminants databases | Upon executing the command, a directory named 'SeqSero_result_Time_your_run' will be created. Your result will be stored in 'SeqSero_result.txt' in that directory. And the assembled alleles can also be found in the directory if using "-m a" (allele mode)</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>116</td>
      <td>116</td>
      <td>14011</td>
      <td>14011</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>14013</td>
      <td>14013</td>
      <td>118</td>
      <td>118</td>
      <td>https://usegalaxy.eu/published/workflow?id=fde21b5f5c169114, https://workflowhub.eu/workflows/407?version=1</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>shovill</td>
      <td>shovill</td>
      <td>Faster de novo assembly pipeline based around Spades</td>
      <td>2017-10-24</td>
      <td>https://github.com/tseemann/shovill</td>
      <td>1.4.2</td>
      <td>shovill</td>
      <td>1.4.2</td>
      <td>Up-to-date</td>
      <td>Assembly</td>
      <td>Genome assembly</td>
      <td>Genome assembly</td>
      <td>Genomics, Microbiology, Sequence assembly</td>
      <td>Genomics, Microbiology, Sequence assembly</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/shovill</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/shovill</td>
      <td>shovill</td>
      <td>shovill</td>
      <td>Shovill is a pipeline for assembly of bacterial isolate genomes from Illumina paired-end reads.  Shovill uses SPAdes at its core, but alters the steps before and after the primary assembly step to get similar results in less time. Shovill also supports other assemblers like SKESA, Velvet and Megahit, so you can take advantage of the pre- and post-processing the Shovill provides with those too.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>3269</td>
      <td>3491</td>
      <td>70801</td>
      <td>76107</td>
      <td>4933</td>
      <td>5212</td>
      <td>51724</td>
      <td>55190</td>
      <td>1425</td>
      <td>1669</td>
      <td>30312</td>
      <td>35794</td>
      <td>83</td>
      <td>83</td>
      <td>9136</td>
      <td>9136</td>
      <td>176227</td>
      <td>161973</td>
      <td>10455</td>
      <td>9710</td>
      <td>https://usegalaxy.eu/published/workflow?id=0404013d5c27210d, https://usegalaxy.eu/published/workflow?id=1ea8bfbe6cb04531, https://usegalaxy.eu/published/workflow?id=309fc4fc2e0da89c, https://usegalaxy.eu/published/workflow?id=3303a04887a47675, https://usegalaxy.eu/published/workflow?id=3d937648809ca4e7, https://usegalaxy.eu/published/workflow?id=4216424f032c055f, https://usegalaxy.eu/published/workflow?id=42c0da2e86318b37, https://usegalaxy.eu/published/workflow?id=4f245ac304ab76d9, https://usegalaxy.eu/published/workflow?id=5b893b69a1b6748f, https://usegalaxy.eu/published/workflow?id=5b940b283d91d99d, https://usegalaxy.eu/published/workflow?id=628062a8cf93261f, https://usegalaxy.eu/published/workflow?id=723e718f37e9ab91, https://usegalaxy.eu/published/workflow?id=7aa32aba3fde0708, https://usegalaxy.eu/published/workflow?id=7e48134082dab0a3, https://usegalaxy.eu/published/workflow?id=94c3d329c31c712f, https://usegalaxy.eu/published/workflow?id=a727e22dc0431614, https://usegalaxy.eu/published/workflow?id=b5ffba506152b1a1, https://usegalaxy.eu/published/workflow?id=b90585626a793502, https://usegalaxy.eu/published/workflow?id=b93333efdab316d8, https://usegalaxy.eu/published/workflow?id=ba1fda2318046543, https://usegalaxy.eu/published/workflow?id=cbe3769f1abb9fa5, https://usegalaxy.eu/published/workflow?id=e16e524c557d7928, https://usegalaxy.eu/published/workflow?id=e532d98f4b368666, https://usegalaxy.eu/published/workflow?id=fca458178b3c794a, https://usegalaxy.eu/published/workflow?id=fde21b5f5c169114, https://usegalaxy.org.au/published/workflow?id=52065a4100b11184, https://usegalaxy.org.au/published/workflow?id=7a0c4804e4d4c905, https://usegalaxy.org.au/published/workflow?id=92fcdd490831c6c5, https://usegalaxy.org.au/published/workflow?id=d78f885b4b86635f, https://usegalaxy.org.au/published/workflow?id=f2541030f6d70b59, https://usegalaxy.org.au/published/workflow?id=fa76b66785ce38b8, https://usegalaxy.org/published/workflow?id=0e30324072570abb, https://usegalaxy.org/published/workflow?id=109092d8dbda4398, https://usegalaxy.org/published/workflow?id=1c7af480caf150bd, https://usegalaxy.org/published/workflow?id=3cf46c167ccb374f, https://usegalaxy.org/published/workflow?id=4161cfd67ed94b5d, https://usegalaxy.org/published/workflow?id=4d1279007d0137fa, https://usegalaxy.org/published/workflow?id=524602abe59b18f7, https://usegalaxy.org/published/workflow?id=5cd56232e873beab, https://usegalaxy.org/published/workflow?id=8ebc2b1ae24ed884, https://usegalaxy.org/published/workflow?id=9ac1e4efe310b3b3, https://usegalaxy.org/published/workflow?id=9e5c5ddb60a24889, https://usegalaxy.org/published/workflow?id=b76a9ce44ea019f1, https://usegalaxy.org/published/workflow?id=c454c930ccfb0d3f, https://usegalaxy.org/published/workflow?id=d4c790a45978bb9e, https://usegalaxy.org/published/workflow?id=d4f0a7ccb09f1e55, https://usegalaxy.org/published/workflow?id=db08887d7d6dc147, https://usegalaxy.org/published/workflow?id=e2ab3d3219a648a7, https://usegalaxy.org/published/workflow?id=e75bd7d392e65209, https://usegalaxy.org/published/workflow?id=f4d623a19b32370e, https://workflowhub.eu/workflows/100?version=1, https://workflowhub.eu/workflows/102?version=1, https://workflowhub.eu/workflows/103?version=1, https://workflowhub.eu/workflows/1043?version=12, https://workflowhub.eu/workflows/1189?version=2, https://workflowhub.eu/workflows/1602?version=2, https://workflowhub.eu/workflows/1767?version=1, https://workflowhub.eu/workflows/407?version=1, https://workflowhub.eu/workflows/470?version=1</td>
      <td>assembly/mrsa-illumina</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>sistr_cmd</td>
      <td>sistr_cmd</td>
      <td>SISTR in silico serotyping tool</td>
      <td>2017-02-20</td>
      <td>https://github.com/phac-nml/sistr_cmd</td>
      <td>1.1.3</td>
      <td>sistr_cmd</td>
      <td>1.1.3</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Genome assembly, Visualisation, Multilocus sequence typing</td>
      <td>Genome assembly, Visualisation, Multilocus sequence typing</td>
      <td>Whole genome sequencing, Immunoproteins and antigens, Sequence assembly, Zoology, Public health and epidemiology</td>
      <td>Whole genome sequencing, Immunoproteins and antigens, Sequence assembly, Zoology, Public health and epidemiology</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/sistr_cmd</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/sistr_cmd</td>
      <td>SISTR</td>
      <td>SISTR</td>
      <td>The Salmonella In Silico Typing Resource (SISTR) is an open-source and freely available web application for rapid in silico typing and serovar prediction from Salmonella genome assemblies using cgMLST and O and H antigen gene searching.</td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>395</td>
      <td>397</td>
      <td>5345</td>
      <td>5466</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>91</td>
      <td>95</td>
      <td>907</td>
      <td>1010</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6476</td>
      <td>6252</td>
      <td>492</td>
      <td>486</td>
      <td>https://usegalaxy.eu/published/workflow?id=628062a8cf93261f, https://usegalaxy.eu/published/workflow?id=94c3d329c31c712f, https://usegalaxy.eu/published/workflow?id=cbe3769f1abb9fa5, https://usegalaxy.eu/published/workflow?id=fde21b5f5c169114, https://usegalaxy.org.au/published/workflow?id=92fcdd490831c6c5, https://workflowhub.eu/workflows/407?version=1</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>smallgenomeutilities</td>
      <td>smgu_frameshift_deletions_checks</td>
      <td>Set of utilities for manipulating small viral genome data.</td>
      <td>2023-05-30</td>
      <td>https://github.com/cbg-ethz/smallgenomeutilities</td>
      <td>0.5.2</td>
      <td>smallgenomeutilities</td>
      <td>0.5.2</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Read pre-processing, Sequence alignment, Genetic variation analysis</td>
      <td>Read pre-processing, Sequence alignment, Genetic variation analysis</td>
      <td>Genomics, Population genetics, Workflows, Virology, Sequencing</td>
      <td>Genomics, Population genetics, Workflows, Virology, Sequencing</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/smallgenomeutilities</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/smallgenomeutilities</td>
      <td>v-pipe</td>
      <td>V-pipe</td>
      <td>Bioinformatics pipeline for the analysis of next-generation sequencing data derived from intra-host viral populations.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>15</td>
      <td>15</td>
      <td>168</td>
      <td>168</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>168</td>
      <td>168</td>
      <td>15</td>
      <td>15</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>smalt</td>
      <td>smalt</td>
      <td>SMALT aligns DNA sequencing reads with a reference genome.</td>
      <td>2017-09-19</td>
      <td>http://www.sanger.ac.uk/science/tools/smalt-0</td>
      <td>0.7.6</td>
      <td>smalt</td>
      <td>0.7.6</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>nml</td>
      <td>https://sourceforge.net/projects/smalt/</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/smalt</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>snap</td>
      <td>snap, snap_training</td>
      <td>SNAP is a general purpose gene finding program suitable for both eukaryotic and prokaryotic genomes.</td>
      <td>2017-10-12</td>
      <td>https://github.com/KorfLab/SNAP</td>
      <td>2013_11_29</td>
      <td>snap</td>
      <td>2013_11_29</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Gene prediction</td>
      <td>Gene prediction</td>
      <td>DNA, DNA polymorphism, Genetics</td>
      <td>DNA polymorphism, Genetics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/snap</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/snap</td>
      <td>snap</td>
      <td>SNAP</td>
      <td>The Semi-HMM-based Nucleic Acid Parser is a gene prediction tool.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>348</td>
      <td>410</td>
      <td>1443</td>
      <td>1821</td>
      <td>258</td>
      <td>292</td>
      <td>813</td>
      <td>939</td>
      <td>100</td>
      <td>101</td>
      <td>578</td>
      <td>594</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3354</td>
      <td>2834</td>
      <td>803</td>
      <td>706</td>
      <td>https://usegalaxy.eu/published/workflow?id=26ad77bc07a7e512, https://usegalaxy.eu/published/workflow?id=6ca20e710f4f3ffb, https://usegalaxy.org.au/published/workflow?id=7e40485d1ed45b7c, https://workflowhub.eu/workflows/1504?version=1</td>
      <td>genome-annotation/annotation-with-maker</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>snippy</td>
      <td>snippy_core, snippy, snippy_clean_full_aln</td>
      <td>Contains the snippy tool for characterising microbial snps</td>
      <td>2017-07-13</td>
      <td>https://github.com/tseemann/snippy</td>
      <td></td>
      <td>snippy</td>
      <td>4.6.0</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Phylogenetic tree visualisation, Phylogenetic tree generation, Variant calling</td>
      <td>Phylogenetic tree visualisation, Variant calling</td>
      <td>Genomics, Model organisms, DNA polymorphism, Phylogenetics</td>
      <td>Genomics, Model organisms, DNA polymorphism, Phylogenetics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/snippy</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/snippy</td>
      <td>snippy</td>
      <td>snippy</td>
      <td>Rapid haploid variant calling and core SNP phylogeny generation.</td>
      <td></td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>3483</td>
      <td>3838</td>
      <td>188403</td>
      <td>199916</td>
      <td>4057</td>
      <td>4306</td>
      <td>138187</td>
      <td>142349</td>
      <td>2223</td>
      <td>2852</td>
      <td>67168</td>
      <td>78979</td>
      <td>60</td>
      <td>60</td>
      <td>8769</td>
      <td>8769</td>
      <td>430013</td>
      <td>402527</td>
      <td>11056</td>
      <td>9823</td>
      <td>https://usegalaxy.eu/published/workflow?id=2b25eaebd3987612, https://usegalaxy.eu/published/workflow?id=2c79eee5ca5a3b09, https://usegalaxy.eu/published/workflow?id=309fc4fc2e0da89c, https://usegalaxy.eu/published/workflow?id=3a8fde248a4d0383, https://usegalaxy.eu/published/workflow?id=3ff4dbcc7a602e6b, https://usegalaxy.eu/published/workflow?id=461d71caea871db8, https://usegalaxy.eu/published/workflow?id=464b642261ed7452, https://usegalaxy.eu/published/workflow?id=4f9f7065975b3f97, https://usegalaxy.eu/published/workflow?id=515ef1601bc24a6c, https://usegalaxy.eu/published/workflow?id=628062a8cf93261f, https://usegalaxy.eu/published/workflow?id=65249196143d7930, https://usegalaxy.eu/published/workflow?id=6d87481be47902e1, https://usegalaxy.eu/published/workflow?id=920aab45ff028ed4, https://usegalaxy.eu/published/workflow?id=94c3d329c31c712f, https://usegalaxy.eu/published/workflow?id=968f0ac2617aa524, https://usegalaxy.eu/published/workflow?id=a183da6b56cd1d60, https://usegalaxy.eu/published/workflow?id=b556b0a67ad979da, https://usegalaxy.eu/published/workflow?id=b681f7b677b23168, https://usegalaxy.eu/published/workflow?id=c38e3ac59d8b86d6, https://usegalaxy.eu/published/workflow?id=c9ef2d2ce92e2dfb, https://usegalaxy.eu/published/workflow?id=d92f9c33dded693f, https://usegalaxy.eu/published/workflow?id=fde21b5f5c169114, https://usegalaxy.org.au/published/workflow?id=1719e0eb54b41f68, https://usegalaxy.org.au/published/workflow?id=5693811cd933d2a2, https://usegalaxy.org.au/published/workflow?id=a61d4bda54de7cd1, https://usegalaxy.org.au/published/workflow?id=fa76b66785ce38b8, https://usegalaxy.org/published/workflow?id=00596e862edb23f2, https://usegalaxy.org/published/workflow?id=562bcaa9b4d2e8a4, https://usegalaxy.org/published/workflow?id=5674956f5c18b85b, https://usegalaxy.org/published/workflow?id=69d8863d40524120, https://usegalaxy.org/published/workflow?id=7dd6e4150bb81ad1, https://usegalaxy.org/published/workflow?id=80ac6adead9f97d8, https://usegalaxy.org/published/workflow?id=d4f0a7ccb09f1e55, https://usegalaxy.org/published/workflow?id=e7ce46762faccaac, https://usegalaxy.org/published/workflow?id=ee4679b3e9e830b7, https://workflowhub.eu/workflows/1035?version=1, https://workflowhub.eu/workflows/1573?version=1, https://workflowhub.eu/workflows/1639?version=1, https://workflowhub.eu/workflows/1647?version=1</td>
      <td>evolution/mtb_transmission, variant-analysis/microbial-variants, variant-analysis/tb-variant-analysis</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>sonneityping</td>
      <td>sonneityping</td>
      <td>Scripts for parsing Mykrobe predict results for Shigella sonnei.</td>
      <td>2021-09-16</td>
      <td>https://github.com/katholt/sonneityping</td>
      <td>20210201</td>
      <td>sonneityping</td>
      <td>20210201</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Antimicrobial resistance prediction, Variant calling, Genotyping</td>
      <td>Antimicrobial resistance prediction, Variant calling, Genotyping</td>
      <td>Whole genome sequencing, Genotype and phenotype, Genetic variation, Metagenomics</td>
      <td>Whole genome sequencing, Genotype and phenotype, Genetic variation, Metagenomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/sonneityping</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/sonneityping</td>
      <td>sonneityping</td>
      <td>sonneityping</td>
      <td>Scripts for parsing Mykrobe predict results for Shigella sonnei.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>sortmerna</td>
      <td>bg_sortmerna</td>
      <td>SortMeRNA is a software designed to rapidly filter ribosomal RNA fragments from metatransriptomic data produced by next-generation sequencers.</td>
      <td>2015-02-28</td>
      <td>http://bioinfo.lifl.fr/RNA/sortmerna/</td>
      <td>4.3.6</td>
      <td>sortmerna</td>
      <td>4.3.7</td>
      <td>To update</td>
      <td>RNA</td>
      <td>Sequence similarity search, Sequence comparison, Sequence alignment analysis</td>
      <td>Sequence similarity search, Sequence alignment analysis</td>
      <td>Metatranscriptomics, Metagenomics</td>
      <td>Metatranscriptomics, Metagenomics</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/sortmerna</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/sortmerna</td>
      <td>sortmerna</td>
      <td>SortMeRNA</td>
      <td>Sequence analysis tool for filtering, mapping and OTU-picking NGS reads.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1135</td>
      <td>1352</td>
      <td>23920</td>
      <td>26406</td>
      <td>581</td>
      <td>581</td>
      <td>6479</td>
      <td>6479</td>
      <td>219</td>
      <td>225</td>
      <td>3668</td>
      <td>3882</td>
      <td>116</td>
      <td>116</td>
      <td>648</td>
      <td>651</td>
      <td>37418</td>
      <td>34715</td>
      <td>2274</td>
      <td>2051</td>
      <td>https://usegalaxy.eu/published/workflow?id=096b75501c8e0888, https://usegalaxy.eu/published/workflow?id=2cf09adb1ed6c3ec, https://usegalaxy.eu/published/workflow?id=385cc7df70d7916b, https://usegalaxy.eu/published/workflow?id=401746888e8b3aa8, https://usegalaxy.eu/published/workflow?id=4d32b1b3ac0a7161, https://usegalaxy.eu/published/workflow?id=5f0faac42b0a5062, https://usegalaxy.eu/published/workflow?id=72e3e5fdc766e24a, https://usegalaxy.eu/published/workflow?id=7ab70660e6235cf0, https://usegalaxy.eu/published/workflow?id=87cc0cd143a12c41, https://usegalaxy.eu/published/workflow?id=9382f9bbbdddb5ad, https://usegalaxy.eu/published/workflow?id=9472be79da012999, https://usegalaxy.eu/published/workflow?id=994e5ff1df7d6c9e, https://usegalaxy.eu/published/workflow?id=9cbba5de2cbeeb0f, https://usegalaxy.eu/published/workflow?id=a8417c63e9e22afb, https://usegalaxy.eu/published/workflow?id=a8f829deb69b30ab, https://usegalaxy.eu/published/workflow?id=c3aa516e78ca4191, https://usegalaxy.eu/published/workflow?id=d05c6f0f3205fbd2, https://usegalaxy.eu/published/workflow?id=dfc2c53ac3103eff, https://usegalaxy.eu/published/workflow?id=e423c80058601e78, https://usegalaxy.eu/published/workflow?id=f6e8a63283262518, https://usegalaxy.eu/published/workflow?id=fd90652d475ed739, https://usegalaxy.eu/published/workflow?id=fdca6c9d394986bf, https://usegalaxy.fr/published/workflow?id=c4f6062e51d0f45e, https://usegalaxy.fr/published/workflow?id=e6991d64e851b09e, https://usegalaxy.org.au/published/workflow?id=408a31694e9c85e6, https://workflowhub.eu/workflows/1444?version=2, https://workflowhub.eu/workflows/1466?version=2</td>
      <td>microbiome/metatranscriptomics, microbiome/metatranscriptomics-short</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>spades</td>
      <td>spades_biosyntheticspades, spades_coronaspades, spades_metaplasmidspades, metaspades, spades_metaviralspades, spades_plasmidspades, rnaspades, spades_rnaviralspades, spades</td>
      <td>SPAdes is an assembly toolkit containing various assembly pipelines. It implements the following 4 stages: assembly graph construction, k-bimer adjustment, construction of paired assembly graph and contig construction.</td>
      <td>2016-12-21</td>
      <td>https://github.com/ablab/spades</td>
      <td>4.2.0</td>
      <td>spades</td>
      <td>4.2.0</td>
      <td>Up-to-date</td>
      <td>Assembly, RNA, Metagenomics</td>
      <td>Genome assembly</td>
      <td>Genome assembly</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/spades</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/spades</td>
      <td>spades</td>
      <td>SPAdes</td>
      <td>St. Petersburg genome assembler  is intended for both standard isolates and single-cell MDA bacteria assemblies. SPAdes 3.9 works with Illumina or IonTorrent reads and is capable of providing hybrid assemblies using PacBio, Oxford Nanopore and Sanger reads. Additional contigs can be provided and can be used as long reads.</td>
      <td></td>
      <td>9</td>
      <td>9</td>
      <td>9</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>9</td>
      <td>3</td>
      <td>0</td>
      <td>8250</td>
      <td>8882</td>
      <td>104429</td>
      <td>116176</td>
      <td>14023</td>
      <td>14731</td>
      <td>147425</td>
      <td>154112</td>
      <td>5211</td>
      <td>6722</td>
      <td>62902</td>
      <td>76811</td>
      <td>373</td>
      <td>380</td>
      <td>2677</td>
      <td>2762</td>
      <td>349861</td>
      <td>317433</td>
      <td>30715</td>
      <td>27857</td>
      <td>https://usegalaxy.eu/published/workflow?id=007ce6405d196df3, https://usegalaxy.eu/published/workflow?id=02631612a43a4d9b, https://usegalaxy.eu/published/workflow?id=1860e02bcb894a64, https://usegalaxy.eu/published/workflow?id=37da7b42cb063a3a, https://usegalaxy.eu/published/workflow?id=49c46d7653d6c349, https://usegalaxy.eu/published/workflow?id=579ff83177c5ee09, https://usegalaxy.eu/published/workflow?id=62c47e3c9d4f3aa3, https://usegalaxy.eu/published/workflow?id=7b755497d168cb13, https://usegalaxy.eu/published/workflow?id=7ecd92cb382d8168, https://usegalaxy.eu/published/workflow?id=87231a6a7938cb95, https://usegalaxy.eu/published/workflow?id=8c7958cb05fba814, https://usegalaxy.eu/published/workflow?id=9382f9bbbdddb5ad, https://usegalaxy.eu/published/workflow?id=93d1035a307fe63d, https://usegalaxy.eu/published/workflow?id=98bfad0c684011e9, https://usegalaxy.eu/published/workflow?id=9a97c6eb4fe015e7, https://usegalaxy.eu/published/workflow?id=a8417c63e9e22afb, https://usegalaxy.eu/published/workflow?id=a847773b563b2328, https://usegalaxy.eu/published/workflow?id=bebcbd6f0ffa55c5, https://usegalaxy.eu/published/workflow?id=c702c76cadebac30, https://usegalaxy.eu/published/workflow?id=d2f75f70c33394ee, https://usegalaxy.eu/published/workflow?id=d4a95588e6e50893, https://usegalaxy.eu/published/workflow?id=db10c092fad70d69, https://usegalaxy.eu/published/workflow?id=ef83f7fd2176a244, https://usegalaxy.fr/published/workflow?id=5bb882218afda132, https://usegalaxy.org.au/published/workflow?id=05f3abecd4ddd4a1, https://usegalaxy.org.au/published/workflow?id=38d992baf9a2a2bc, https://usegalaxy.org.au/published/workflow?id=39c724282ee008e3, https://usegalaxy.org.au/published/workflow?id=813a4938397cf6ee, https://usegalaxy.org.au/published/workflow?id=87cdeb05b1ebeab0, https://usegalaxy.org.au/published/workflow?id=8abcfe166f1a6cd8, https://usegalaxy.org.au/published/workflow?id=8f7d5ebead46a6d0, https://usegalaxy.org.au/published/workflow?id=914330feb75ea9ab, https://usegalaxy.org.au/published/workflow?id=fa76b66785ce38b8, https://usegalaxy.org/published/workflow?id=15cca0e249f40aea, https://usegalaxy.org/published/workflow?id=18899840e49d89c3, https://usegalaxy.org/published/workflow?id=33d90e718ce500ef, https://usegalaxy.org/published/workflow?id=5039ae2c81d0aba8, https://usegalaxy.org/published/workflow?id=71f8a4090877f171, https://usegalaxy.org/published/workflow?id=7251c6e0dad8f927, https://usegalaxy.org/published/workflow?id=87b3305ea433b4e2, https://usegalaxy.org/published/workflow?id=8a350e0d96e937a5, https://usegalaxy.org/published/workflow?id=8d229c0f58a07ebe, https://usegalaxy.org/published/workflow?id=9ba6da7dae465d3e, https://usegalaxy.org/published/workflow?id=9c0fbd9f51a9857f, https://usegalaxy.org/published/workflow?id=ae81cc0e8263d03e, https://usegalaxy.org/published/workflow?id=e0c33057ac2670ba, https://usegalaxy.org/published/workflow?id=fb01e1a49793c8be, https://workflowhub.eu/workflows/1390?version=2, https://workflowhub.eu/workflows/1626?version=1, https://workflowhub.eu/workflows/1634?version=1, https://workflowhub.eu/workflows/434?version=1, https://workflowhub.eu/workflows/52?version=1, https://workflowhub.eu/workflows/68?version=1</td>
      <td>assembly/debruijn-graph-assembly, assembly/metagenomics-assembly, microbiome/metagenomics-assembly</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>spotyping</td>
      <td>spotyping</td>
      <td>SpoTyping allows fast and accurate in silico Mycobacterium spoligotyping from sequence reads</td>
      <td>2018-05-07</td>
      <td>https://github.com/xiaeryu/SpoTyping-v2.0</td>
      <td>2.1</td>
      <td>spotyping</td>
      <td>2.1</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Variant pattern analysis</td>
      <td>Variant pattern analysis</td>
      <td>Microbiology, Sequencing, Sequence composition, complexity and repeats, Genetic variation</td>
      <td>Microbiology, Sequencing, Sequence composition, complexity and repeats, Genetic variation</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/spotyping</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/spotyping</td>
      <td>spotyping</td>
      <td>SpoTyping</td>
      <td>Fast and accurate in silico Mycobacterium spoligotyping from sequence reads.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>34</td>
      <td>34</td>
      <td>2042</td>
      <td>2042</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2042</td>
      <td>2042</td>
      <td>34</td>
      <td>34</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>squirrel</td>
      <td>squirrel_phylo, squirrel_qc</td>
      <td>QC and Phylogenetic analysis of MPXV</td>
      <td>2025-01-16</td>
      <td>https://github.com/aineniamh/squirrel</td>
      <td>1.0.13</td>
      <td>squirrel</td>
      <td>1.3.1</td>
      <td>To update</td>
      <td>Phylogenetics</td>
      <td>Phylogenetic analysis, Sequencing quality control</td>
      <td>Phylogenetic analysis, Sequencing quality control</td>
      <td>Virology</td>
      <td>Virology</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/squirrel</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/squirrel</td>
      <td>squirrel</td>
      <td>squirrel</td>
      <td>Some QUIck Reconstruction to Resolve Evolutionary LinksSquirrel provides a rapid way of producing reliable alignments for MPXV and also enable maximum-likelihood phylogenetics pipeline tree estimation.</td>
      <td></td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>41</td>
      <td>41</td>
      <td>193</td>
      <td>193</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>200</td>
      <td>200</td>
      <td>43</td>
      <td>43</td>
      <td>https://usegalaxy.org.au/published/workflow?id=b9fd488b0938d323</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>sr_bowtie</td>
      <td>bowtieForSmallRNA</td>
      <td>bowtie wrapper tool to align small RNA sequencing reads</td>
      <td>2017-09-02</td>
      <td>http://artbio.fr</td>
      <td>2.3.1</td>
      <td>bowtie</td>
      <td>1.3.1</td>
      <td>To update</td>
      <td>RNA, Next Gen Mappers</td>
      <td></td>
      <td></td>
      <td>Microbiology, Biology</td>
      <td>Microbiology</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/sr_bowtie</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/sr_bowtie</td>
      <td>sr_bowtie</td>
      <td>sr_bowtie</td>
      <td>Bowtie_ is a short read aligner designed to be ultrafast and memory-efficient. It is developed by Ben Langmead and Cole Trapnell. Please cite: Langmead B, Trapnell C, Pop M, Salzberg SL. Ultrafast and memory-efficient alignment of short DNA sequences to the human genome. Genome Biology 10:R25... _Bowtie: http://bowtie-bio.sourceforge.net/index.shtmlA generic "Map with Bowtie for Illumina" Galaxy tool is available in the main Galaxy distribution.However, this Bowtie wrapper tool only takes FASTQ files as inputs.The sRbowtie wrapper works with short (-v bowtie mode) reads inputs, in fasta or fastq format, and proposes a simplified set of configurations suited to small RNA analysis.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>98</td>
      <td>98</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>98</td>
      <td>98</td>
      <td>2</td>
      <td>2</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>srst2</td>
      <td>srst2</td>
      <td>SRST2 Short Read Sequence Typing for Bacterial Pathogens</td>
      <td>2022-08-22</td>
      <td>http://katholt.github.io/srst2/</td>
      <td>0.2.0</td>
      <td>samtools</td>
      <td>1.23</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Multilocus sequence typing</td>
      <td>Multilocus sequence typing</td>
      <td>Whole genome sequencing, Public health and epidemiology, Comparative genomics</td>
      <td>Whole genome sequencing, Public health and epidemiology, Comparative genomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/srst2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/srst2</td>
      <td>srst2</td>
      <td>srst2</td>
      <td>Short Read Sequence Typing for Bacterial Pathogens</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>68</td>
      <td>68</td>
      <td>407</td>
      <td>407</td>
      <td>1</td>
      <td>1</td>
      <td>71</td>
      <td>71</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>478</td>
      <td>478</td>
      <td>69</td>
      <td>69</td>
      <td>https://workflowhub.eu/workflows/407?version=1</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>srst2</td>
      <td>srst2</td>
      <td>Short Read Sequence Typing for Bacterial Pathogens</td>
      <td>2015-12-02</td>
      <td></td>
      <td>0.3.7</td>
      <td>srst2</td>
      <td>0.2.0</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>nml</td>
      <td></td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/srst2</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>68</td>
      <td>68</td>
      <td>407</td>
      <td>407</td>
      <td>1</td>
      <td>1</td>
      <td>71</td>
      <td>71</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>478</td>
      <td>478</td>
      <td>69</td>
      <td>69</td>
      <td>https://workflowhub.eu/workflows/407?version=1</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>staramr</td>
      <td>staramr_search</td>
      <td>Scan genome contigs against the ResFinder, PlasmidFinder, and PointFinder antimicrobial resistance databases.</td>
      <td>2024-09-19</td>
      <td>https://github.com/phac-nml/staramr</td>
      <td>0.11.1</td>
      <td>staramr</td>
      <td>0.11.1</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Antimicrobial resistance prediction</td>
      <td>Antimicrobial resistance prediction</td>
      <td>Microbiology, Public health and epidemiology, Infectious disease</td>
      <td>Microbiology, Public health and epidemiology, Infectious disease</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/staramr</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/staramr</td>
      <td>staramr</td>
      <td>staramr</td>
      <td>staramr (*AMR) scans bacterial genome contigs against the ResFinder, PointFinder, and PlasmidFinder databases (used by the ResFinder webservice and other webservices offered by the Center for Genomic Epidemiology) and compiles a summary report of detected antimicrobial resistance genes. The star|* in staramr indicates that it can handle all of the ResFinder, PointFinder, and PlasmidFinder databases.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2921</td>
      <td>3072</td>
      <td>31443</td>
      <td>33124</td>
      <td>2130</td>
      <td>2130</td>
      <td>14763</td>
      <td>14763</td>
      <td>1173</td>
      <td>1182</td>
      <td>8989</td>
      <td>9237</td>
      <td>97</td>
      <td>97</td>
      <td>9838</td>
      <td>9838</td>
      <td>66962</td>
      <td>65033</td>
      <td>6481</td>
      <td>6321</td>
      <td>https://usegalaxy.eu/published/workflow?id=007ce6405d196df3, https://usegalaxy.eu/published/workflow?id=152223c8aed27357, https://usegalaxy.eu/published/workflow?id=1ce523ef51f47134, https://usegalaxy.eu/published/workflow?id=249c115413f17f28, https://usegalaxy.eu/published/workflow?id=2e0075791f7fa384, https://usegalaxy.eu/published/workflow?id=32d2f5340098bcbd, https://usegalaxy.eu/published/workflow?id=32de10b9de8a999a, https://usegalaxy.eu/published/workflow?id=34df701aa23a9fb3, https://usegalaxy.eu/published/workflow?id=374797e58377c40f, https://usegalaxy.eu/published/workflow?id=39837ca09374598b, https://usegalaxy.eu/published/workflow?id=41d207d06a32ba05, https://usegalaxy.eu/published/workflow?id=44e53ea319e07daa, https://usegalaxy.eu/published/workflow?id=46b6812817c57a01, https://usegalaxy.eu/published/workflow?id=4ddbffe4b3fef275, https://usegalaxy.eu/published/workflow?id=628062a8cf93261f, https://usegalaxy.eu/published/workflow?id=6d87481be47902e1, https://usegalaxy.eu/published/workflow?id=7aa32aba3fde0708, https://usegalaxy.eu/published/workflow?id=9219557de92e0203, https://usegalaxy.eu/published/workflow?id=94c3d329c31c712f, https://usegalaxy.eu/published/workflow?id=a06b1ed5cd25922f, https://usegalaxy.eu/published/workflow?id=abe85758e77c78e6, https://usegalaxy.eu/published/workflow?id=b203dec6ee6da2fd, https://usegalaxy.eu/published/workflow?id=b2b46e71046b1999, https://usegalaxy.eu/published/workflow?id=b4de76c0a4d0e23a, https://usegalaxy.eu/published/workflow?id=c48807045e258dd8, https://usegalaxy.eu/published/workflow?id=dc7107896a3f2964, https://usegalaxy.eu/published/workflow?id=e37cc0b64774561b, https://usegalaxy.eu/published/workflow?id=ea485dbabc76488d, https://usegalaxy.eu/published/workflow?id=eb5ab8bc82569eee, https://usegalaxy.eu/published/workflow?id=ecf48fa50af560a0, https://usegalaxy.eu/published/workflow?id=f30f8e7dcd50cd0d, https://usegalaxy.org.au/published/workflow?id=05f3abecd4ddd4a1, https://usegalaxy.org.au/published/workflow?id=2e29b21cdec7d0cc, https://usegalaxy.org.au/published/workflow?id=35f39173f0ac9179, https://usegalaxy.org.au/published/workflow?id=92fcdd490831c6c5, https://usegalaxy.org.au/published/workflow?id=c9b7de574fb2f2d5, https://usegalaxy.org.au/published/workflow?id=d9571b3dfef34e5e, https://usegalaxy.org/published/workflow?id=2078a54de1f2628d, https://usegalaxy.org/published/workflow?id=2b4b1fe82258f303, https://usegalaxy.org/published/workflow?id=3c177fdf38311118, https://usegalaxy.org/published/workflow?id=547f86db467bf620, https://usegalaxy.org/published/workflow?id=aaf8e740b3f8e2c0, https://usegalaxy.org/published/workflow?id=d3dc805cd43b2737, https://usegalaxy.org/published/workflow?id=d4f0a7ccb09f1e55, https://usegalaxy.org/published/workflow?id=ef187f85b103d858, https://usegalaxy.org/published/workflow?id=f8fab0cd6fc30d92, https://usegalaxy.org/published/workflow?id=fde774a049875466, https://workflowhub.eu/workflows/1049?version=9, https://workflowhub.eu/workflows/1189?version=2, https://workflowhub.eu/workflows/1477?version=1, https://workflowhub.eu/workflows/1488?version=2, https://workflowhub.eu/workflows/2024?version=2, https://workflowhub.eu/workflows/406?version=1, https://workflowhub.eu/workflows/470?version=1</td>
      <td>genome-annotation/amr-gene-detection, microbiome/plasmid-metagenomics-nanopore</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>stringmlst</td>
      <td>stringmlst</td>
      <td>Rapid and accurate identification of the sequence type (ST)</td>
      <td>2016-10-19</td>
      <td></td>
      <td>1.1.0</td>
      <td>stringMLST</td>
      <td>0.6.3</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>nml</td>
      <td></td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/stringmlst</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>structure</td>
      <td>structure</td>
      <td>for using multi-locus genotype data to investigate population structure.</td>
      <td>2017-09-22</td>
      <td>https://web.stanford.edu/group/pritchardlab/structure.html</td>
      <td>2.3.4</td>
      <td>structure</td>
      <td>2.3.4</td>
      <td>Up-to-date</td>
      <td>Phylogenetics, Variant Analysis</td>
      <td>Genetic variation analysis</td>
      <td>Genetic variation analysis</td>
      <td>Population genetics</td>
      <td>Population genetics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/structure</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/structure</td>
      <td>structure</td>
      <td>Structure</td>
      <td>The program structureis a free software package for using multi-locus genotype data to investigate population structure. Its uses include inferring the presence of distinct populations, assigning individuals to populations, studying hybrid zones, identifying migrants and admixed individuals, and estimating population allele frequencies in situations where many individuals are migrants or admixed.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>138</td>
      <td>155</td>
      <td>3903</td>
      <td>4354</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>330</td>
      <td>330</td>
      <td>4684</td>
      <td>4233</td>
      <td>158</td>
      <td>141</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2_core</td>
      <td></td>
      <td></td>
      <td>2022-08-29</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>Sequence Analysis, Metagenomics, Statistics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>q2d2</td>
      <td></td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tool_collections/suite_qiime2_core</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>sylph</td>
      <td>sylph_profile, sylph_query</td>
      <td>sylph - fast and precise species-level metagenomic profiling with ANIs</td>
      <td>2024-10-08</td>
      <td>https://github.com/bluenote-1577/sylph</td>
      <td>0.8.1</td>
      <td>sylph</td>
      <td>0.9.0</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Taxonomic classification</td>
      <td>Taxonomic classification</td>
      <td>Metagenomics</td>
      <td>Metagenomics</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/main/tools/sylph</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/sylph</td>
      <td>sylph</td>
      <td>sylph</td>
      <td>fast and precise species-level metagenomic profiling with ANIs</td>
      <td></td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>9</td>
      <td>257</td>
      <td>257</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>12</td>
      <td>12</td>
      <td>270</td>
      <td>270</td>
      <td>11</td>
      <td>11</td>
      <td>https://usegalaxy.eu/published/workflow?id=7491883694fff308, https://usegalaxy.eu/published/workflow?id=cb2df493f5e334f7</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>syri</td>
      <td>syri</td>
      <td>Synteny and Rearrangement Identifier</td>
      <td>2025-04-16</td>
      <td>https://github.com/schneebergerlab/syri</td>
      <td>1.7.1</td>
      <td>syri</td>
      <td>1.7.1</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Haplotype mapping, Variant calling, Genotyping, Sequence assembly, Read mapping</td>
      <td>Haplotype mapping, Variant calling, Genotyping, Sequence assembly, Read mapping</td>
      <td>DNA polymorphism, Sequence assembly, DNA structural variation, Mapping, Sequencing</td>
      <td>DNA polymorphism, Sequence assembly, Mapping, Sequencing</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/syri</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/syri</td>
      <td>SyRI</td>
      <td>SyRI</td>
      <td>SyRI is tool for finding genomic rearrangements and local sequence differences from whole-genome assemblies.Genomic differences range from single nucleotide differences to complex structural variations. Current methods typically annotate sequence differences ranging from SNPs to large indels accurately but do not unravel the full complexity of structural rearrangements, including inversions, translocations, and duplications, where highly similar sequence changes in location, orientation, or copy number. Here, we present SyRI, a pairwise whole-genome comparison tool for chromosome-level assemblies. SyRI starts by finding rearranged regions and then searches for differences in the sequences, which are distinguished for residing in syntenic or rearranged regions. This distinction is important as rearranged regions are inherited differently compared to syntenic regions.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>22</td>
      <td>22</td>
      <td>84</td>
      <td>84</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>76</td>
      <td>76</td>
      <td>160</td>
      <td>160</td>
      <td>27</td>
      <td>27</td>
      <td>https://usegalaxy.fr/published/workflow?id=356dced9687596b7, https://usegalaxy.fr/published/workflow?id=ce4acd38084ee9ef</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>t2ps</td>
      <td>Draw_phylogram</td>
      <td>Draw phylogeny</td>
      <td>2014-01-27</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>1.0.0</td>
      <td>taxonomy</td>
      <td>0.10.2</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Phylogenetic tree visualisation</td>
      <td>Phylogenetic tree visualisation</td>
      <td>Phylogenomics</td>
      <td>Phylogenomics</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/taxonomy/t2ps</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/taxonomy/t2ps</td>
      <td>t2ps</td>
      <td>Draw phylogeny</td>
      <td>"Given taxonomy representation (produced by *Fetch taxonomic representation* tool) this utility produces a graphical representations of phylogenetic tree in PDF format." - Galaxy tool wrapper</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>64</td>
      <td>93</td>
      <td>405</td>
      <td>479</td>
      <td>13</td>
      <td>1240</td>
      <td>102</td>
      <td>7952</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8431</td>
      <td>507</td>
      <td>1333</td>
      <td>77</td>
      <td>https://usegalaxy.org/published/workflow?id=19c9c8cc3c813bda, https://usegalaxy.org/published/workflow?id=2f7638035fb889ac, https://usegalaxy.org/published/workflow?id=55aa743125c2465d, https://usegalaxy.org/published/workflow?id=59347cc210c4cdbc, https://usegalaxy.org/published/workflow?id=7294f75ad01ca7ed, https://usegalaxy.org/published/workflow?id=7b67ffcf1b7c2232, https://usegalaxy.org/published/workflow?id=85d825931edf4a64, https://usegalaxy.org/published/workflow?id=896d29573ea3e574, https://usegalaxy.org/published/workflow?id=d851f8436497dd1c</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>t2t_report</td>
      <td>t2t_report</td>
      <td>Summarize taxonomy</td>
      <td>2014-01-27</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>1.0.0</td>
      <td>taxonomy</td>
      <td>0.10.2</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/taxonomy/t2t_report</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/taxonomy/t2t_report</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>55</td>
      <td>82</td>
      <td>745</td>
      <td>1071</td>
      <td>16</td>
      <td>1143</td>
      <td>118</td>
      <td>7194</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8265</td>
      <td>863</td>
      <td>1225</td>
      <td>71</td>
      <td>https://usegalaxy.org/published/workflow?id=19c9c8cc3c813bda, https://usegalaxy.org/published/workflow?id=2f7638035fb889ac, https://usegalaxy.org/published/workflow?id=55aa743125c2465d, https://usegalaxy.org/published/workflow?id=59347cc210c4cdbc, https://usegalaxy.org/published/workflow?id=7b67ffcf1b7c2232, https://usegalaxy.org/published/workflow?id=85d825931edf4a64, https://usegalaxy.org/published/workflow?id=896d29573ea3e574, https://usegalaxy.org/published/workflow?id=d851f8436497dd1c</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>t_coffee</td>
      <td>t_coffee</td>
      <td>T-Coffee</td>
      <td>2016-12-13</td>
      <td>https://tcoffee.org/</td>
      <td>13.45.0.4846264</td>
      <td>t-coffee</td>
      <td>13.46.0.919e8c6b</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>earlhaminst</td>
      <td>https://github.com/EarlhamInst/earlham-galaxytools/tree/main/tools/t_coffee</td>
      <td>https://github.com/EarlhamInst/earlham-galaxytools/tree/main/tools/t_coffee</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>274</td>
      <td>311</td>
      <td>3958</td>
      <td>9664</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9665</td>
      <td>3959</td>
      <td>312</td>
      <td>275</td>
      <td>https://usegalaxy.eu/published/workflow?id=2cec6a39cd496f09</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>taxonkit</td>
      <td>name2taxid, profile2cami</td>
      <td>TaxonKit - A Practical and Efficient NCBI Taxonomy Toolkit</td>
      <td>2024-07-26</td>
      <td>https://bioinf.shenwei.me/taxonkit/</td>
      <td>0.20.0</td>
      <td>taxonkit</td>
      <td>0.20.0</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Formatting, Data retrieval</td>
      <td>Formatting, Data retrieval</td>
      <td>Taxonomy, Biotechnology, Ecology</td>
      <td>Taxonomy, Biotechnology, Ecology</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/taxonkit</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/taxonkit</td>
      <td>taxonkit</td>
      <td>TaxonKit</td>
      <td>TaxonKit is a practical and efficient NCBI taxonomy toolkit.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>12</td>
      <td>12</td>
      <td>1094</td>
      <td>1094</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1094</td>
      <td>1094</td>
      <td>12</td>
      <td>12</td>
      <td>https://usegalaxy.eu/published/workflow?id=cf0159c14418525e, https://usegalaxy.eu/published/workflow?id=e4dabaf57668dbd1</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>taxonomy_krona_chart</td>
      <td>taxonomy_krona_chart</td>
      <td>Krona pie chart from taxonomic profile</td>
      <td>2015-08-06</td>
      <td>http://sourceforge.net/projects/krona/</td>
      <td>2.7.1+galaxy0</td>
      <td>krona</td>
      <td>2.8.1</td>
      <td>To update</td>
      <td>Assembly</td>
      <td>Visualisation</td>
      <td>Visualisation</td>
      <td>Metagenomics</td>
      <td>Metagenomics</td>
      <td>crs4</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/taxonomy_krona_chart</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/taxonomy_krona_chart</td>
      <td>krona</td>
      <td>Krona</td>
      <td>Krona creates interactive HTML5 charts of hierarchical data (such as taxonomic abundance in a metagenome).</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>7406</td>
      <td>7874</td>
      <td>48617</td>
      <td>51653</td>
      <td>6438</td>
      <td>7659</td>
      <td>33268</td>
      <td>41191</td>
      <td>2005</td>
      <td>2042</td>
      <td>11003</td>
      <td>11192</td>
      <td>162</td>
      <td>163</td>
      <td>565</td>
      <td>586</td>
      <td>104622</td>
      <td>93453</td>
      <td>17738</td>
      <td>16011</td>
      <td>https://usegalaxy.eu/published/workflow?id=007ce6405d196df3, https://usegalaxy.eu/published/workflow?id=0590554db9794d13, https://usegalaxy.eu/published/workflow?id=05f5f040337786bf, https://usegalaxy.eu/published/workflow?id=08e25b733cd796e6, https://usegalaxy.eu/published/workflow?id=08eb5e55109c4d9c, https://usegalaxy.eu/published/workflow?id=096b75501c8e0888, https://usegalaxy.eu/published/workflow?id=0bafeb17c21ebc9c, https://usegalaxy.eu/published/workflow?id=10101558b211a782, https://usegalaxy.eu/published/workflow?id=14714876dda7186d, https://usegalaxy.eu/published/workflow?id=181e751be8dd9abc, https://usegalaxy.eu/published/workflow?id=19de5cbae7f48103, https://usegalaxy.eu/published/workflow?id=2cf09adb1ed6c3ec, https://usegalaxy.eu/published/workflow?id=309fc4fc2e0da89c, https://usegalaxy.eu/published/workflow?id=39d36a7ce0ef268d, https://usegalaxy.eu/published/workflow?id=437704898229dfb6, https://usegalaxy.eu/published/workflow?id=44e16f0782f54111, https://usegalaxy.eu/published/workflow?id=4a764275d4bf9f31, https://usegalaxy.eu/published/workflow?id=4d32b1b3ac0a7161, https://usegalaxy.eu/published/workflow?id=4f0daf8e9dd54127, https://usegalaxy.eu/published/workflow?id=4f21205e71cabd37, https://usegalaxy.eu/published/workflow?id=57cfe5062c8e01bf, https://usegalaxy.eu/published/workflow?id=5f0faac42b0a5062, https://usegalaxy.eu/published/workflow?id=628062a8cf93261f, https://usegalaxy.eu/published/workflow?id=7491883694fff308, https://usegalaxy.eu/published/workflow?id=79b4ae4403772098, https://usegalaxy.eu/published/workflow?id=7ab70660e6235cf0, https://usegalaxy.eu/published/workflow?id=86465932556bc96c, https://usegalaxy.eu/published/workflow?id=87cc0cd143a12c41, https://usegalaxy.eu/published/workflow?id=8deb08d4f228e198, https://usegalaxy.eu/published/workflow?id=8ee3b635c1c380cd, https://usegalaxy.eu/published/workflow?id=9472be79da012999, https://usegalaxy.eu/published/workflow?id=94c3d329c31c712f, https://usegalaxy.eu/published/workflow?id=994e5ff1df7d6c9e, https://usegalaxy.eu/published/workflow?id=9cbba5de2cbeeb0f, https://usegalaxy.eu/published/workflow?id=a337bfb2eb1e583f, https://usegalaxy.eu/published/workflow?id=a403c661bf1bc4fa, https://usegalaxy.eu/published/workflow?id=a459c311f461adce, https://usegalaxy.eu/published/workflow?id=a84bac86ec183d46, https://usegalaxy.eu/published/workflow?id=a8f829deb69b30ab, https://usegalaxy.eu/published/workflow?id=ad8b6d73c9654305, https://usegalaxy.eu/published/workflow?id=b14845359b702444, https://usegalaxy.eu/published/workflow?id=b2b46e71046b1999, https://usegalaxy.eu/published/workflow?id=b37800d990cceb83, https://usegalaxy.eu/published/workflow?id=b66e1d8171e6dac2, https://usegalaxy.eu/published/workflow?id=b86f24473a282ef9, https://usegalaxy.eu/published/workflow?id=b91a380ceaa0342b, https://usegalaxy.eu/published/workflow?id=bd4e3cdc7f29049b, https://usegalaxy.eu/published/workflow?id=c01ef8354b585a1f, https://usegalaxy.eu/published/workflow?id=c1e26abe61c69688, https://usegalaxy.eu/published/workflow?id=c2b7ec5aacfc6107, https://usegalaxy.eu/published/workflow?id=c3aa516e78ca4191, https://usegalaxy.eu/published/workflow?id=c67d2e75eeb4fd19, https://usegalaxy.eu/published/workflow?id=ca80039c4933aa0a, https://usegalaxy.eu/published/workflow?id=cb2df493f5e334f7, https://usegalaxy.eu/published/workflow?id=ce21961260fe4351, https://usegalaxy.eu/published/workflow?id=cfd37e9e307e5d3d, https://usegalaxy.eu/published/workflow?id=d05c6f0f3205fbd2, https://usegalaxy.eu/published/workflow?id=d0ae255151450dad, https://usegalaxy.eu/published/workflow?id=d88145e7475e0b7b, https://usegalaxy.eu/published/workflow?id=dadca1020f586338, https://usegalaxy.eu/published/workflow?id=dfc2c53ac3103eff, https://usegalaxy.eu/published/workflow?id=ed0ad728cf21702b, https://usegalaxy.eu/published/workflow?id=ee0005a5487653ba, https://usegalaxy.eu/published/workflow?id=ef6556d65ce5d72c, https://usegalaxy.eu/published/workflow?id=f21d6a40a10a8ae8, https://usegalaxy.eu/published/workflow?id=fb806b88850f3427, https://usegalaxy.eu/published/workflow?id=fdca6c9d394986bf, https://usegalaxy.eu/published/workflow?id=fde21b5f5c169114, https://usegalaxy.fr/published/workflow?id=09e8293456ee6d99, https://usegalaxy.fr/published/workflow?id=c4f6062e51d0f45e, https://usegalaxy.org.au/published/workflow?id=131636a795bac485, https://usegalaxy.org.au/published/workflow?id=2717e90438c7fb8c, https://usegalaxy.org.au/published/workflow?id=4c7e0703aace6aa3, https://usegalaxy.org.au/published/workflow?id=6b7e565258a9da2a, https://usegalaxy.org.au/published/workflow?id=a335b9c9103b57ad, https://usegalaxy.org.au/published/workflow?id=d9ba165e6ae55417, https://usegalaxy.org.au/published/workflow?id=ed3e48a73b5aaa56, https://usegalaxy.org/published/workflow?id=21161075002d6778, https://usegalaxy.org/published/workflow?id=3011f5dbf884c9bc, https://usegalaxy.org/published/workflow?id=45f959d5bf505b15, https://usegalaxy.org/published/workflow?id=4b5f6b5d4f36e38b, https://usegalaxy.org/published/workflow?id=5b74b780d3955eb3, https://usegalaxy.org/published/workflow?id=60f0d0d5d6e93941, https://usegalaxy.org/published/workflow?id=6f090b2c259847d2, https://usegalaxy.org/published/workflow?id=7321c6ad5425d347, https://usegalaxy.org/published/workflow?id=775335812a6b7a61, https://usegalaxy.org/published/workflow?id=8151be18aab3a3c6, https://usegalaxy.org/published/workflow?id=8f5904693b5f74f4, https://usegalaxy.org/published/workflow?id=9ba6da7dae465d3e, https://usegalaxy.org/published/workflow?id=a63927dd19759714, https://usegalaxy.org/published/workflow?id=b68ec42db8953569, https://usegalaxy.org/published/workflow?id=b7817a12dbee583c, https://usegalaxy.org/published/workflow?id=bc01b8b203ecc754, https://usegalaxy.org/published/workflow?id=c3ea8fa3bb202dc6, https://usegalaxy.org/published/workflow?id=cade0c5c11978323, https://usegalaxy.org/published/workflow?id=d5436cee11a34de9, https://usegalaxy.org/published/workflow?id=d653eaefcf6b4697, https://usegalaxy.org/published/workflow?id=e12324f5e3b930fd, https://workflowhub.eu/workflows/100?version=1, https://workflowhub.eu/workflows/101?version=1, https://workflowhub.eu/workflows/1059?version=1, https://workflowhub.eu/workflows/1199?version=2, https://workflowhub.eu/workflows/124?version=1, https://workflowhub.eu/workflows/1270?version=2, https://workflowhub.eu/workflows/1273?version=2, https://workflowhub.eu/workflows/1439?version=1, https://workflowhub.eu/workflows/1451?version=2, https://workflowhub.eu/workflows/1465?version=1, https://workflowhub.eu/workflows/1466?version=2, https://workflowhub.eu/workflows/1470?version=2, https://workflowhub.eu/workflows/1472?version=1, https://workflowhub.eu/workflows/1473?version=1, https://workflowhub.eu/workflows/1483?version=1, https://workflowhub.eu/workflows/1842?version=1, https://workflowhub.eu/workflows/1856?version=1, https://workflowhub.eu/workflows/2024?version=2, https://workflowhub.eu/workflows/53?version=1, https://workflowhub.eu/workflows/624?version=1</td>
      <td>microbiome/beer-data-analysis, microbiome/general-tutorial, microbiome/metatranscriptomics, microbiome/metatranscriptomics-short, microbiome/mgnify-amplicon, microbiome/mothur-miseq-sop, microbiome/nanopore-16S-metagenomics, microbiome/pathogen-detection-from-nanopore-foodborne-data, microbiome/taxonomic-profiling</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>taxpasta</td>
      <td>taxpasta</td>
      <td>standardise taxonomic profiles</td>
      <td>2023-08-30</td>
      <td>https://taxpasta.readthedocs.io/en/latest/</td>
      <td>0.7.0</td>
      <td>taxpasta</td>
      <td>0.7.0</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Standardisation and normalisation, Aggregation, Formatting, Conversion</td>
      <td>Standardisation and normalisation, Aggregation, Formatting, Conversion</td>
      <td>Taxonomy, Metagenomics</td>
      <td>Taxonomy, Metagenomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/taxpasta</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/taxpasta</td>
      <td>taxpasta</td>
      <td>taxpasta</td>
      <td>TAXonomic Profile Aggregation and STAndardisationThe main purpose of taxpasta is to standardise taxonomic profiles created by a range of bioinformatics tools. We call those tools taxonomic profilers. They each come with their own particular tabular output format. Across the profilers, relative abundances can be reported in read counts, fractions, or percentages, as well as any number of additional columns with extra information. We therefore decided to take the lessons learnt to heart and provide our own solution to deal with this pasticcio. With taxpasta you can ingest all of those formats and, at a minimum, output taxonomy identifiers and their integer counts. Taxpasta can not only standardise profiles but also merge them across samples for the same profiler into a single table.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>51</td>
      <td>51</td>
      <td>323</td>
      <td>323</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>18</td>
      <td>18</td>
      <td>341</td>
      <td>341</td>
      <td>52</td>
      <td>52</td>
      <td>https://usegalaxy.eu/published/workflow?id=7491883694fff308, https://usegalaxy.eu/published/workflow?id=b14845359b702444, https://usegalaxy.eu/published/workflow?id=cb2df493f5e334f7</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>tb-profiler</td>
      <td>tb_profiler_collate, tb_profiler_profile</td>
      <td>Processes M. tuberculosis sequence data to infer strain type and identify known drug resistance markers.</td>
      <td>2019-04-02</td>
      <td>https://github.com/jodyphelan/TBProfiler</td>
      <td>6.6.5</td>
      <td>tb-profiler</td>
      <td>6.6.6</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Antimicrobial resistance prediction</td>
      <td>Antimicrobial resistance prediction</td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/tb-profiler</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/tb-profiler</td>
      <td>tb-profiler</td>
      <td>tb-profiler</td>
      <td>A tool for drug resistance prediction from _M. tuberculosis_ genomic data (sequencing reads, alignments or variants).</td>
      <td></td>
      <td>1</td>
      <td>2</td>
      <td>2</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>491</td>
      <td>535</td>
      <td>15947</td>
      <td>16201</td>
      <td>490</td>
      <td>506</td>
      <td>11483</td>
      <td>11641</td>
      <td>199</td>
      <td>201</td>
      <td>10939</td>
      <td>10981</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>38823</td>
      <td>38369</td>
      <td>1242</td>
      <td>1180</td>
      <td>https://usegalaxy.eu/published/workflow?id=2b25eaebd3987612, https://usegalaxy.eu/published/workflow?id=2c79eee5ca5a3b09, https://usegalaxy.eu/published/workflow?id=3ff4dbcc7a602e6b, https://usegalaxy.eu/published/workflow?id=461d71caea871db8, https://usegalaxy.eu/published/workflow?id=89ed570bc0703472, https://usegalaxy.eu/published/workflow?id=c38e3ac59d8b86d6, https://usegalaxy.eu/published/workflow?id=d0d6dae0d1eecc21, https://usegalaxy.org.au/published/workflow?id=5693811cd933d2a2, https://usegalaxy.org.au/published/workflow?id=a61d4bda54de7cd1, https://usegalaxy.org/published/workflow?id=e7ce46762faccaac, https://workflowhub.eu/workflows/1035?version=1, https://workflowhub.eu/workflows/1564?version=1, https://workflowhub.eu/workflows/1647?version=1</td>
      <td>evolution/mtb_transmission, variant-analysis/tb-variant-analysis</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>tiara</td>
      <td>tiara</td>
      <td>Tool for identification of eukaryotic sequences in the metagenomic datasets.</td>
      <td>2024-05-23</td>
      <td>https://github.com/ibe-uw/tiara</td>
      <td>1.0.3</td>
      <td>tiara</td>
      <td></td>
      <td>To update</td>
      <td>Metagenomics, Sequence Analysis</td>
      <td>Genome comparison, Data retrieval, Variant calling, Genome visualisation, Structural variation detection</td>
      <td>Genome comparison, Data retrieval, Variant calling, Genome visualisation, Structural variation detection</td>
      <td>Sequencing, Genomics, DNA polymorphism, DNA structural variation</td>
      <td>Sequencing, Genomics, DNA polymorphism</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/tiara</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/tiara</td>
      <td>Tiara</td>
      <td>TIARA</td>
      <td>Total Integrated Archive of Short-Read and Array (TIARA) database, contains personal genomic information obtained from next generation sequencing (NGS) techniques and ultra-high-resolution comparative genomic hybridization (CGH) arrays. This database improves the accuracy of detecting personal genomic variations, such as SNPs, short indels and structural variants (SVs).</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>33</td>
      <td>33</td>
      <td>365</td>
      <td>365</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>365</td>
      <td>365</td>
      <td>33</td>
      <td>33</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>tooldistillator</td>
      <td>tooldistillator, tooldistillator_summarize</td>
      <td>ToolDistillator extract and aggregate information from different tool outputs to JSON parsable files</td>
      <td>2024-04-30</td>
      <td>https://gitlab.com/ifb-elixirfr/abromics/tooldistillator</td>
      <td>1.0.4</td>
      <td>tooldistillator</td>
      <td>1.0.4</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Data parsing</td>
      <td></td>
      <td>Microbiology, Bioinformatics, Sequence analysis</td>
      <td>Microbiology, Bioinformatics, Sequence analysis</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/tooldistillator</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/tooldistillator</td>
      <td>tooldistillator</td>
      <td>ToolDistillator</td>
      <td>ToolDistillator is a tool to extract information from output files of specific tools, expose it as JSON files, and aggregate over several tools.It can produce both a single file to each tool or a summarized file from a set of reports.</td>
      <td></td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>121</td>
      <td>121</td>
      <td>5042</td>
      <td>5042</td>
      <td>162</td>
      <td>162</td>
      <td>1550</td>
      <td>1550</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>9</td>
      <td>19253</td>
      <td>19253</td>
      <td>25845</td>
      <td>25845</td>
      <td>292</td>
      <td>292</td>
      <td>https://usegalaxy.eu/published/workflow?id=0404013d5c27210d, https://usegalaxy.eu/published/workflow?id=0a82eb8b44d9d502, https://usegalaxy.eu/published/workflow?id=0abd32dbc5207b88, https://usegalaxy.eu/published/workflow?id=0c47571d20535426, https://usegalaxy.eu/published/workflow?id=1306ed083d91a852, https://usegalaxy.eu/published/workflow?id=1ce523ef51f47134, https://usegalaxy.eu/published/workflow?id=2c1a6da84d891cf2, https://usegalaxy.eu/published/workflow?id=2fa2f67603772413, https://usegalaxy.eu/published/workflow?id=32de10b9de8a999a, https://usegalaxy.eu/published/workflow?id=3303a04887a47675, https://usegalaxy.eu/published/workflow?id=34df701aa23a9fb3, https://usegalaxy.eu/published/workflow?id=36ea584e0f60893e, https://usegalaxy.eu/published/workflow?id=3d937648809ca4e7, https://usegalaxy.eu/published/workflow?id=41d207d06a32ba05, https://usegalaxy.eu/published/workflow?id=4216424f032c055f, https://usegalaxy.eu/published/workflow?id=46b6812817c57a01, https://usegalaxy.eu/published/workflow?id=47e14222073e7192, https://usegalaxy.eu/published/workflow?id=4a218fb93d78ba67, https://usegalaxy.eu/published/workflow?id=5b893b69a1b6748f, https://usegalaxy.eu/published/workflow?id=61948fcaefa1418a, https://usegalaxy.eu/published/workflow?id=6b69ab27c565257b, https://usegalaxy.eu/published/workflow?id=723e718f37e9ab91, https://usegalaxy.eu/published/workflow?id=7e48134082dab0a3, https://usegalaxy.eu/published/workflow?id=8792eaecb69fbc26, https://usegalaxy.eu/published/workflow?id=884a4415b669690c, https://usegalaxy.eu/published/workflow?id=a18aed4b03ffd27e, https://usegalaxy.eu/published/workflow?id=a727e22dc0431614, https://usegalaxy.eu/published/workflow?id=aafcfcb585c5ca75, https://usegalaxy.eu/published/workflow?id=abe85758e77c78e6, https://usegalaxy.eu/published/workflow?id=b1afc8d1e81fc904, https://usegalaxy.eu/published/workflow?id=b203dec6ee6da2fd, https://usegalaxy.eu/published/workflow?id=b93333efdab316d8, https://usegalaxy.eu/published/workflow?id=ba1fda2318046543, https://usegalaxy.eu/published/workflow?id=bb63b5bb983fd49c, https://usegalaxy.eu/published/workflow?id=bcd50e655809cfcb, https://usegalaxy.eu/published/workflow?id=c14d2e8b3c191bc4, https://usegalaxy.eu/published/workflow?id=c930a54a30dcc7e3, https://usegalaxy.eu/published/workflow?id=d8db0b2e30370b84, https://usegalaxy.eu/published/workflow?id=dfb8e0f3c534000e, https://usegalaxy.eu/published/workflow?id=e37cc0b64774561b, https://usegalaxy.eu/published/workflow?id=e532d98f4b368666, https://usegalaxy.eu/published/workflow?id=ebb131f2a6172081, https://usegalaxy.eu/published/workflow?id=f30f8e7dcd50cd0d, https://usegalaxy.eu/published/workflow?id=fca458178b3c794a, https://usegalaxy.eu/published/workflow?id=fd63b9884fda75b6, https://usegalaxy.org/published/workflow?id=04c5ea133de3204c, https://usegalaxy.org/published/workflow?id=0e30324072570abb, https://usegalaxy.org/published/workflow?id=1be5c982c2d36a6c, https://usegalaxy.org/published/workflow?id=2078a54de1f2628d, https://usegalaxy.org/published/workflow?id=2a782ad24271f570, https://usegalaxy.org/published/workflow?id=2b4b1fe82258f303, https://usegalaxy.org/published/workflow?id=323c651ca583c9e5, https://usegalaxy.org/published/workflow?id=3940186d02245b1d, https://usegalaxy.org/published/workflow?id=3c177fdf38311118, https://usegalaxy.org/published/workflow?id=4161cfd67ed94b5d, https://usegalaxy.org/published/workflow?id=49da012e2839c046, https://usegalaxy.org/published/workflow?id=4d1279007d0137fa, https://usegalaxy.org/published/workflow?id=4f698eaeef30e364, https://usegalaxy.org/published/workflow?id=4f8bdd61c524b901, https://usegalaxy.org/published/workflow?id=524602abe59b18f7, https://usegalaxy.org/published/workflow?id=5cd56232e873beab, https://usegalaxy.org/published/workflow?id=623fc80c7f91a33a, https://usegalaxy.org/published/workflow?id=649cdc14757e5755, https://usegalaxy.org/published/workflow?id=67ca17ab34b10772, https://usegalaxy.org/published/workflow?id=8cb8a588a0495cfa, https://usegalaxy.org/published/workflow?id=8ebc2b1ae24ed884, https://usegalaxy.org/published/workflow?id=993534c11310a207, https://usegalaxy.org/published/workflow?id=9ac1e4efe310b3b3, https://usegalaxy.org/published/workflow?id=9e5c5ddb60a24889, https://usegalaxy.org/published/workflow?id=9f68718f025bed30, https://usegalaxy.org/published/workflow?id=aaf8e740b3f8e2c0, https://usegalaxy.org/published/workflow?id=b6a795834009ce06, https://usegalaxy.org/published/workflow?id=b76a9ce44ea019f1, https://usegalaxy.org/published/workflow?id=c454c930ccfb0d3f, https://usegalaxy.org/published/workflow?id=d2a5f6a5181aada5, https://usegalaxy.org/published/workflow?id=d3dc805cd43b2737, https://usegalaxy.org/published/workflow?id=d4c790a45978bb9e, https://usegalaxy.org/published/workflow?id=de9bf4c1024f8d53, https://usegalaxy.org/published/workflow?id=dfd73857c0eb2a21, https://usegalaxy.org/published/workflow?id=ef187f85b103d858, https://usegalaxy.org/published/workflow?id=f378960ee55f567e, https://usegalaxy.org/published/workflow?id=f4d623a19b32370e, https://usegalaxy.org/published/workflow?id=f8fab0cd6fc30d92, https://usegalaxy.org/published/workflow?id=fde774a049875466, https://workflowhub.eu/workflows/1043?version=12, https://workflowhub.eu/workflows/1049?version=9, https://workflowhub.eu/workflows/1050?version=13, https://workflowhub.eu/workflows/1052?version=11, https://workflowhub.eu/workflows/1881?version=2, https://workflowhub.eu/workflows/1882?version=3, https://workflowhub.eu/workflows/2024?version=2</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>transtermhp</td>
      <td>transtermhp</td>
      <td>Finds rho-independent transcription terminators in bacterial genomes</td>
      <td>2015-10-08</td>
      <td>https://transterm.cbcb.umd.edu</td>
      <td></td>
      <td>transtermhp</td>
      <td>2.09</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>Transcriptional regulatory element prediction</td>
      <td>Transcriptional regulatory element prediction</td>
      <td>Transcription factors and regulatory sites</td>
      <td>Transcription factors and regulatory sites</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/transtermhp</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/transtermhp</td>
      <td>transtermhp</td>
      <td>TransTermHP</td>
      <td>TransTermHP finds rho-independent transcription terminators in bacterial genomes. Each terminator found by the program is assigned a confidence value that estimates its probability of being a true terminator</td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>59</td>
      <td>62</td>
      <td>467</td>
      <td>502</td>
      <td>49</td>
      <td>55</td>
      <td>170</td>
      <td>216</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>718</td>
      <td>637</td>
      <td>117</td>
      <td>108</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>treebest_best</td>
      <td>treebest_best</td>
      <td>TreeBeST best</td>
      <td>2015-08-06</td>
      <td>https://treesoft.sourceforge.net/treebest.shtml</td>
      <td>1.9.2.post0</td>
      <td>treebest</td>
      <td>1.9.2.post1</td>
      <td>To update</td>
      <td>Phylogenetics</td>
      <td>Phylogenetic tree visualisation, Phylogenetic analysis, Phylogenetic inference (from molecular sequences)</td>
      <td>Phylogenetic tree visualisation, Phylogenetic inference (from molecular sequences)</td>
      <td>Phylogenetics</td>
      <td>Phylogenetics</td>
      <td>earlhaminst</td>
      <td>https://github.com/EarlhamInst/earlham-galaxytools/tree/main/tools/TreeBest</td>
      <td>https://github.com/EarlhamInst/earlham-galaxytools/tree/main/tools/TreeBest</td>
      <td>treebest</td>
      <td>TreeBeST</td>
      <td>TreeBeST, which stands for (gene) Tree Building guided by Species Tree, is a versatile program that builds, manipulates and displays phylogenetic trees. It is particularly designed for building gene trees with a known species tree and is highly efficient and accurate.TreeBeST is previously known as NJTREE. It has been largely used in the TreeFam database, Ensembl Compara and OPTIC database of Chris Ponting group.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>45</td>
      <td>58</td>
      <td>966</td>
      <td>1033</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1034</td>
      <td>967</td>
      <td>59</td>
      <td>46</td>
      <td>https://usegalaxy.eu/published/workflow?id=2cec6a39cd496f09</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>trim_galore</td>
      <td>trim_galore</td>
      <td>Trim Galore adaptive quality and adapter trimmer</td>
      <td>2015-02-28</td>
      <td>http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/</td>
      <td>0.6.10</td>
      <td>trim-galore</td>
      <td>0.6.10</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis, Fastq Manipulation</td>
      <td>Sequence trimming, Primer removal, Read pre-processing</td>
      <td>Sequence trimming, Primer removal, Read pre-processing</td>
      <td>Sequence analysis</td>
      <td>Sequence analysis</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/trim_galore</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/trim_galore</td>
      <td>trim_galore</td>
      <td>Trim Galore</td>
      <td>A wrapper tool around Cutadapt and FastQC to consistently apply quality and adapter trimming to FastQ files, with some extra functionality for MspI-digested RRBS-type (Reduced Representation Bisufite-Seq) libraries.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>6966</td>
      <td>8378</td>
      <td>297289</td>
      <td>379413</td>
      <td>13276</td>
      <td>20041</td>
      <td>284172</td>
      <td>418931</td>
      <td>1722</td>
      <td>2020</td>
      <td>36616</td>
      <td>41727</td>
      <td>303</td>
      <td>305</td>
      <td>3334</td>
      <td>3366</td>
      <td>843437</td>
      <td>621411</td>
      <td>30744</td>
      <td>22267</td>
      <td>https://usegalaxy.eu/published/workflow?id=02631612a43a4d9b, https://usegalaxy.eu/published/workflow?id=02a212cc9f134570, https://usegalaxy.eu/published/workflow?id=1524b189069b42a5, https://usegalaxy.eu/published/workflow?id=1590085d315aa80e, https://usegalaxy.eu/published/workflow?id=1d2e17a4f71f2455, https://usegalaxy.eu/published/workflow?id=1ee43d90b69829e9, https://usegalaxy.eu/published/workflow?id=21dd4bd16579fc12, https://usegalaxy.eu/published/workflow?id=2a73d658446747de, https://usegalaxy.eu/published/workflow?id=2cf09adb1ed6c3ec, https://usegalaxy.eu/published/workflow?id=2d2d0c2231babc50, https://usegalaxy.eu/published/workflow?id=2f37c35ade97aa8d, https://usegalaxy.eu/published/workflow?id=346b46e6316c6950, https://usegalaxy.eu/published/workflow?id=385cc7df70d7916b, https://usegalaxy.eu/published/workflow?id=401746888e8b3aa8, https://usegalaxy.eu/published/workflow?id=40a621934a07c7ee, https://usegalaxy.eu/published/workflow?id=41b5f0d519404df7, https://usegalaxy.eu/published/workflow?id=443ea80b36d862f5, https://usegalaxy.eu/published/workflow?id=48fe9c4de045d414, https://usegalaxy.eu/published/workflow?id=4d80a87510b2a54e, https://usegalaxy.eu/published/workflow?id=579ff83177c5ee09, https://usegalaxy.eu/published/workflow?id=59b274da8df01ead, https://usegalaxy.eu/published/workflow?id=5ec0d6d108e02e1a, https://usegalaxy.eu/published/workflow?id=60a45de522b8b695, https://usegalaxy.eu/published/workflow?id=60f2a8721e203057, https://usegalaxy.eu/published/workflow?id=6158c3b5fc12044e, https://usegalaxy.eu/published/workflow?id=6223333100a9e73f, https://usegalaxy.eu/published/workflow?id=667bac8d7453e5f1, https://usegalaxy.eu/published/workflow?id=67e20e3082d24fbd, https://usegalaxy.eu/published/workflow?id=6a5c39c1d38a10c2, https://usegalaxy.eu/published/workflow?id=6fc2b9bea5b75a49, https://usegalaxy.eu/published/workflow?id=7136ce9f18c74853, https://usegalaxy.eu/published/workflow?id=7734928ebc0a2654, https://usegalaxy.eu/published/workflow?id=7d84b0419296857d, https://usegalaxy.eu/published/workflow?id=8601a62dbca3082f, https://usegalaxy.eu/published/workflow?id=8bc1c3b9e665ea37, https://usegalaxy.eu/published/workflow?id=8deb08d4f228e198, https://usegalaxy.eu/published/workflow?id=9382f9bbbdddb5ad, https://usegalaxy.eu/published/workflow?id=9472be79da012999, https://usegalaxy.eu/published/workflow?id=9a97c6eb4fe015e7, https://usegalaxy.eu/published/workflow?id=9cbba5de2cbeeb0f, https://usegalaxy.eu/published/workflow?id=9e9c9dbed98f62ff, https://usegalaxy.eu/published/workflow?id=a08279c2a305afff, https://usegalaxy.eu/published/workflow?id=a108b575b16e6cb9, https://usegalaxy.eu/published/workflow?id=a8417c63e9e22afb, https://usegalaxy.eu/published/workflow?id=ad4b347f9b68e127, https://usegalaxy.eu/published/workflow?id=b66e1d8171e6dac2, https://usegalaxy.eu/published/workflow?id=b6eb5fe7e37f680a, https://usegalaxy.eu/published/workflow?id=b91a380ceaa0342b, https://usegalaxy.eu/published/workflow?id=bbe2a1615c8faaae, https://usegalaxy.eu/published/workflow?id=bc79a9f797334b01, https://usegalaxy.eu/published/workflow?id=c3aa516e78ca4191, https://usegalaxy.eu/published/workflow?id=d4603d244cf8ac5b, https://usegalaxy.eu/published/workflow?id=d57d41e306241396, https://usegalaxy.eu/published/workflow?id=d72edb9f8c529f47, https://usegalaxy.eu/published/workflow?id=d9f8c5d575da02b1, https://usegalaxy.eu/published/workflow?id=e7363b9f450c56b4, https://usegalaxy.eu/published/workflow?id=eb8168bc01249596, https://usegalaxy.eu/published/workflow?id=fd90652d475ed739, https://usegalaxy.eu/published/workflow?id=fe64d430d50faf0b, https://usegalaxy.org.au/published/workflow?id=0a477899db53d098, https://usegalaxy.org.au/published/workflow?id=25f65a29ce3c3ee0, https://usegalaxy.org.au/published/workflow?id=5926a7669eabb1c6, https://usegalaxy.org.au/published/workflow?id=7a63fff9a122e60c, https://usegalaxy.org.au/published/workflow?id=7ad277dd448e57b3, https://usegalaxy.org.au/published/workflow?id=c456492ce4f963d0, https://usegalaxy.org.au/published/workflow?id=daff6854b5dfc6f1, https://usegalaxy.org/published/workflow?id=07fdeadce7a6d5cb, https://usegalaxy.org/published/workflow?id=092be3cf6e40282c, https://usegalaxy.org/published/workflow?id=0f1659b3290c3492, https://usegalaxy.org/published/workflow?id=0f3f91de5f022b6e, https://usegalaxy.org/published/workflow?id=0f43ad9dfd0f6d16, https://usegalaxy.org/published/workflow?id=10ac77144ced73ad, https://usegalaxy.org/published/workflow?id=11ba260b93522c22, https://usegalaxy.org/published/workflow?id=173ecc0c88090d0e, https://usegalaxy.org/published/workflow?id=18c6e87a884fb5ed, https://usegalaxy.org/published/workflow?id=1ba0239e55ac0d07, https://usegalaxy.org/published/workflow?id=204d0da8a023d7da, https://usegalaxy.org/published/workflow?id=2219c4aa7eb8cfb8, https://usegalaxy.org/published/workflow?id=228ae7f537a60226, https://usegalaxy.org/published/workflow?id=2f21358621337cf2, https://usegalaxy.org/published/workflow?id=357d644e99be5e13, https://usegalaxy.org/published/workflow?id=3ffb59671b58d5ce, https://usegalaxy.org/published/workflow?id=41efc922cf146e0c, https://usegalaxy.org/published/workflow?id=427fce51fd9508eb, https://usegalaxy.org/published/workflow?id=43c39bc86399188f, https://usegalaxy.org/published/workflow?id=44b5161868a34561, https://usegalaxy.org/published/workflow?id=48adcc44fe62a99c, https://usegalaxy.org/published/workflow?id=5203bc68df3cf819, https://usegalaxy.org/published/workflow?id=526b05cbffecbc85, https://usegalaxy.org/published/workflow?id=53834a5fb46ca0f6, https://usegalaxy.org/published/workflow?id=5efe288b18c7d0ab, https://usegalaxy.org/published/workflow?id=5fc0c7c65edc7457, https://usegalaxy.org/published/workflow?id=623dcf47f7963700, https://usegalaxy.org/published/workflow?id=69af7dd51aff958a, https://usegalaxy.org/published/workflow?id=6d0a3f36d2736087, https://usegalaxy.org/published/workflow?id=6dada297572d77bd, https://usegalaxy.org/published/workflow?id=734e08331f8f34b7, https://usegalaxy.org/published/workflow?id=73ef57dfa532af3d, https://usegalaxy.org/published/workflow?id=7ee7412cdd82f1ee, https://usegalaxy.org/published/workflow?id=82681bd5f059e39a, https://usegalaxy.org/published/workflow?id=84dfa09b472ed3f2, https://usegalaxy.org/published/workflow?id=8b4572ac274176c0, https://usegalaxy.org/published/workflow?id=8e32b0a5a141543e, https://usegalaxy.org/published/workflow?id=93650de42fb8c015, https://usegalaxy.org/published/workflow?id=93bfbd7db42db543, https://usegalaxy.org/published/workflow?id=958f78270fc8f55c, https://usegalaxy.org/published/workflow?id=9e187245d39c7864, https://usegalaxy.org/published/workflow?id=a46febfebe3bcfe3, https://usegalaxy.org/published/workflow?id=a89def2c2462674a, https://usegalaxy.org/published/workflow?id=aac2ae0419f7d515, https://usegalaxy.org/published/workflow?id=b094b6e1e7d48416, https://usegalaxy.org/published/workflow?id=b3026f28a45626a5, https://usegalaxy.org/published/workflow?id=b3823d276917baf0, https://usegalaxy.org/published/workflow?id=bb9e69ae66c3d916, https://usegalaxy.org/published/workflow?id=c4cde7e8d897cd25, https://usegalaxy.org/published/workflow?id=cd56ed91856fcc6f, https://usegalaxy.org/published/workflow?id=d5514e327878b531, https://usegalaxy.org/published/workflow?id=ddb737f2ad0589e8, https://usegalaxy.org/published/workflow?id=dec0cf06fa059e65, https://usegalaxy.org/published/workflow?id=e04f443a8ceca0d2, https://usegalaxy.org/published/workflow?id=e215018e24b84197, https://usegalaxy.org/published/workflow?id=e26fabceece3b858, https://usegalaxy.org/published/workflow?id=eb33df15225b8aa0, https://usegalaxy.org/published/workflow?id=eb4b32fcf4a9f64a, https://usegalaxy.org/published/workflow?id=ecb279a9114140ab, https://usegalaxy.org/published/workflow?id=eed1aadf2fb83f6c, https://usegalaxy.org/published/workflow?id=eedbd4e9caca6195, https://usegalaxy.org/published/workflow?id=efb62e8178ecdcaf, https://usegalaxy.org/published/workflow?id=f1a9667c0b0999f4, https://usegalaxy.org/published/workflow?id=fcd98ca3abcf6c61, https://workflowhub.eu/workflows/1562?version=1, https://workflowhub.eu/workflows/1615?version=2, https://workflowhub.eu/workflows/1619?version=1, https://workflowhub.eu/workflows/1628?version=1, https://workflowhub.eu/workflows/1657?version=1, https://workflowhub.eu/workflows/1661?version=2, https://workflowhub.eu/workflows/1673?version=1, https://workflowhub.eu/workflows/1717?version=2, https://workflowhub.eu/workflows/99?version=1</td>
      <td>epigenetics/cut_and_run, epigenetics/estrogen-receptor-binding-site-identification, epigenetics/formation_of_super-structures_on_xi, sequence-analysis/mapping, transcriptomics/mirna-target-finder, transcriptomics/srna, variant-analysis/baculovirus-isolate-variation</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>trycycler</td>
      <td>trycycler_cluster, trycycler_consensus, trycycler_partition, trycycler_reconcile_msa, trycycler_subsample</td>
      <td>Trycycler toolkit wrappers</td>
      <td>2021-02-11</td>
      <td>https://github.com/rrwick/Trycycler</td>
      <td>0.5.5</td>
      <td>trycycler</td>
      <td>0.5.6</td>
      <td>To update</td>
      <td>Assembly</td>
      <td>Genome assembly, Sequence trimming, Scaffold gap completion, Sequence assembly validation</td>
      <td>Genome assembly, Sequence trimming, Scaffold gap completion, Sequence assembly validation</td>
      <td>Sequence assembly, DNA, Human biology, Whole genome sequencing, Genomics</td>
      <td>Sequence assembly, DNA, Human biology, Whole genome sequencing, Genomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/trycycler</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/trycycler</td>
      <td>trycycler</td>
      <td>Trycycler</td>
      <td>Trycycler: consensus long-read assemblies for bacterial genomes</td>
      <td></td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>122</td>
      <td>122</td>
      <td>15743</td>
      <td>15743</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>57</td>
      <td>57</td>
      <td>6233</td>
      <td>6233</td>
      <td>3</td>
      <td>3</td>
      <td>27</td>
      <td>27</td>
      <td>22003</td>
      <td>22003</td>
      <td>182</td>
      <td>182</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>tsebra</td>
      <td>tsebra</td>
      <td>This tool has been developed to combine BRAKER predictions.</td>
      <td>2023-10-19</td>
      <td>https://github.com/Gaius-Augustus/TSEBRA</td>
      <td>1.1.2.5</td>
      <td>tsebra</td>
      <td>1.1.2.5</td>
      <td>Up-to-date</td>
      <td>Genome annotation</td>
      <td>Homology-based gene prediction, Alternative splicing prediction</td>
      <td>Homology-based gene prediction, Alternative splicing prediction</td>
      <td>Gene expression, RNA-Seq, Gene transcripts, Model organisms</td>
      <td>Gene expression, RNA-Seq, Gene transcripts, Model organisms</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/tsebra</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/tsebra</td>
      <td>tsebra</td>
      <td>TSEBRA</td>
      <td>TSEBRA is a combiner tool that selects transcripts from gene predictions based on the support by extrisic evidence in form of introns and start/stop codons. It was developed to combine BRAKER1 and BRAKER2 predicitons to increase their accuracies.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>16</td>
      <td>16</td>
      <td>51</td>
      <td>51</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>51</td>
      <td>51</td>
      <td>16</td>
      <td>16</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>unicycler</td>
      <td>unicycler</td>
      <td>Unicycler is a hybrid assembly pipeline for bacterial genomes.</td>
      <td>2017-05-08</td>
      <td>https://github.com/rrwick/Unicycler</td>
      <td>0.5.1</td>
      <td>unicycler</td>
      <td>0.5.1</td>
      <td>Up-to-date</td>
      <td>Assembly</td>
      <td>Genome assembly, Aggregation</td>
      <td>Genome assembly, Aggregation</td>
      <td>Microbiology, Genomics, Sequencing, Sequence assembly</td>
      <td>Microbiology, Genomics, Sequencing, Sequence assembly</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/unicycler</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/unicycler</td>
      <td>unicycler</td>
      <td>Unicycler</td>
      <td>A tool for assembling bacterial genomes from a combination of short (2nd generation) and long (3rd generation) sequencing reads.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>4697</td>
      <td>5202</td>
      <td>83521</td>
      <td>96874</td>
      <td>5770</td>
      <td>8769</td>
      <td>65067</td>
      <td>104850</td>
      <td>2392</td>
      <td>2613</td>
      <td>22717</td>
      <td>26831</td>
      <td>272</td>
      <td>276</td>
      <td>2160</td>
      <td>2183</td>
      <td>230738</td>
      <td>173465</td>
      <td>16860</td>
      <td>13131</td>
      <td>https://usegalaxy.eu/published/workflow?id=165089132248a0b0, https://usegalaxy.eu/published/workflow?id=1ea8bfbe6cb04531, https://usegalaxy.eu/published/workflow?id=249c115413f17f28, https://usegalaxy.eu/published/workflow?id=24eaa682897379c8, https://usegalaxy.eu/published/workflow?id=2e0075791f7fa384, https://usegalaxy.eu/published/workflow?id=346b46e6316c6950, https://usegalaxy.eu/published/workflow?id=374797e58377c40f, https://usegalaxy.eu/published/workflow?id=39837ca09374598b, https://usegalaxy.eu/published/workflow?id=44e53ea319e07daa, https://usegalaxy.eu/published/workflow?id=69be516a48761365, https://usegalaxy.eu/published/workflow?id=7b755497d168cb13, https://usegalaxy.eu/published/workflow?id=842e0ea7b1eb56af, https://usegalaxy.eu/published/workflow?id=8b8e89b9b8801f2f, https://usegalaxy.eu/published/workflow?id=8f3ce55d44d9f369, https://usegalaxy.eu/published/workflow?id=9219557de92e0203, https://usegalaxy.eu/published/workflow?id=b2b46e71046b1999, https://usegalaxy.eu/published/workflow?id=b2dfa74f55a73409, https://usegalaxy.eu/published/workflow?id=c48807045e258dd8, https://usegalaxy.eu/published/workflow?id=c702c76cadebac30, https://usegalaxy.eu/published/workflow?id=dc7107896a3f2964, https://usegalaxy.eu/published/workflow?id=e8eb80ee09159bac, https://usegalaxy.eu/published/workflow?id=ea485dbabc76488d, https://usegalaxy.eu/published/workflow?id=ecf48fa50af560a0, https://usegalaxy.eu/published/workflow?id=fe43cf59e48bf556, https://usegalaxy.fr/published/workflow?id=5bb882218afda132, https://usegalaxy.org.au/published/workflow?id=38d992baf9a2a2bc, https://usegalaxy.org.au/published/workflow?id=8a40059343cf4b87, https://usegalaxy.org.au/published/workflow?id=8f7d5ebead46a6d0, https://usegalaxy.org.au/published/workflow?id=9c60b83feaa22415, https://usegalaxy.org.au/published/workflow?id=b59fada651043735, https://usegalaxy.org.au/published/workflow?id=b7aa22291322e505, https://usegalaxy.org.au/published/workflow?id=c9b7de574fb2f2d5, https://usegalaxy.org.au/published/workflow?id=d9571b3dfef34e5e, https://usegalaxy.org.au/published/workflow?id=f01be6a7685ac476, https://usegalaxy.org.au/published/workflow?id=fc3c83b11f527977, https://usegalaxy.org/published/workflow?id=5039ae2c81d0aba8, https://usegalaxy.org/published/workflow?id=69514a75e4c9af08, https://usegalaxy.org/published/workflow?id=8111bb9bb26bb665, https://usegalaxy.org/published/workflow?id=a67afbea4b2eb26b, https://usegalaxy.org/published/workflow?id=b43b72dcf5f7694d, https://usegalaxy.org/published/workflow?id=bc72dcbca89ca35d, https://usegalaxy.org/published/workflow?id=c65bec6c332bf740, https://usegalaxy.org/published/workflow?id=db0671862a57968d, https://workflowhub.eu/workflows/1477?version=1, https://workflowhub.eu/workflows/1598?version=1, https://workflowhub.eu/workflows/1612?version=1, https://workflowhub.eu/workflows/1616?version=1, https://workflowhub.eu/workflows/2011?version=1, https://workflowhub.eu/workflows/38?version=1, https://workflowhub.eu/workflows/406?version=1, https://workflowhub.eu/workflows/52?version=1, https://workflowhub.eu/workflows/68?version=1</td>
      <td>assembly/assembly-with-preprocessing, assembly/hybrid_denovo_assembly, assembly/unicycler-assembly, microbiome/plasmid-metagenomics-nanopore</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>unipept</td>
      <td>unipept</td>
      <td>Unipept retrieves metaproteomics information</td>
      <td>2015-04-03</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp</td>
      <td>6.2.4+galaxy1</td>
      <td>python</td>
      <td></td>
      <td>To update</td>
      <td>Proteomics</td>
      <td>Prediction and recognition, Visualisation</td>
      <td>Prediction and recognition, Visualisation</td>
      <td>Proteomics, Proteogenomics, Biodiversity, Workflows</td>
      <td>Proteomics, Proteogenomics, Biodiversity, Workflows</td>
      <td>galaxyp</td>
      <td>https://unipept.ugent.be/apidocs</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/unipept</td>
      <td>unipept</td>
      <td>Unipept</td>
      <td>Metaproteomics data analysis with a focus on interactive data visualizations.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>282</td>
      <td>314</td>
      <td>4723</td>
      <td>6047</td>
      <td>102</td>
      <td>102</td>
      <td>384</td>
      <td>384</td>
      <td>33</td>
      <td>33</td>
      <td>1383</td>
      <td>1383</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>7815</td>
      <td>6491</td>
      <td>450</td>
      <td>418</td>
      <td>https://usegalaxy.eu/published/workflow?id=00e246ea459b0991, https://usegalaxy.eu/published/workflow?id=02700454f57eb556, https://usegalaxy.eu/published/workflow?id=0e0be913c3e27022, https://usegalaxy.eu/published/workflow?id=1706bfa819bbfa89, https://usegalaxy.eu/published/workflow?id=17a1433d0b681d4a, https://usegalaxy.eu/published/workflow?id=229a955d29bd4064, https://usegalaxy.eu/published/workflow?id=28af61ead68e0976, https://usegalaxy.eu/published/workflow?id=3e01a2c6d954ae6e, https://usegalaxy.eu/published/workflow?id=41df8c7821b5f2fc, https://usegalaxy.eu/published/workflow?id=50d85103f2275d38, https://usegalaxy.eu/published/workflow?id=58db3e1996a8f093, https://usegalaxy.eu/published/workflow?id=5e4774036aa20b80, https://usegalaxy.eu/published/workflow?id=612dc5bdf388585f, https://usegalaxy.eu/published/workflow?id=6306656ec10bd18f, https://usegalaxy.eu/published/workflow?id=6766bed39f614b7c, https://usegalaxy.eu/published/workflow?id=68967c922de149ae, https://usegalaxy.eu/published/workflow?id=6cb85fe1ad0c9f3b, https://usegalaxy.eu/published/workflow?id=6da171871d4f3fd5, https://usegalaxy.eu/published/workflow?id=755ede37d94dc456, https://usegalaxy.eu/published/workflow?id=78e1769f1dcb797b, https://usegalaxy.eu/published/workflow?id=82b6871682f1659a, https://usegalaxy.eu/published/workflow?id=8a85663218676b59, https://usegalaxy.eu/published/workflow?id=8f3cafa3bc59ff82, https://usegalaxy.eu/published/workflow?id=8fc1ea9e94bc723d, https://usegalaxy.eu/published/workflow?id=9ef76d764d9961b2, https://usegalaxy.eu/published/workflow?id=a4d6b9dc6b396933, https://usegalaxy.eu/published/workflow?id=a657b412b37b971b, https://usegalaxy.eu/published/workflow?id=ba3a5d59b3879764, https://usegalaxy.eu/published/workflow?id=bfe26679d670bb5e, https://usegalaxy.eu/published/workflow?id=c6b6ba35e5b6500a, https://usegalaxy.eu/published/workflow?id=cd675ab32d3e7833, https://usegalaxy.eu/published/workflow?id=cefc49c13ff73231, https://usegalaxy.eu/published/workflow?id=d0e062aee910086a, https://usegalaxy.eu/published/workflow?id=d148459a4d4b91f1, https://usegalaxy.eu/published/workflow?id=d9e7a940b2d410bc, https://usegalaxy.eu/published/workflow?id=df926c5ab8d27192, https://usegalaxy.eu/published/workflow?id=f0bec337f9998289, https://usegalaxy.eu/published/workflow?id=fc42cc9467c6ee8d, https://usegalaxy.org.au/published/workflow?id=050a66ff6d4c2b53, https://usegalaxy.org.au/published/workflow?id=f9f685d6150e48c5, https://usegalaxy.org/published/workflow?id=3f797bfb67130883, https://workflowhub.eu/workflows/1219?version=1, https://workflowhub.eu/workflows/1433?version=1, https://workflowhub.eu/workflows/1443?version=1, https://workflowhub.eu/workflows/1450?version=1, https://workflowhub.eu/workflows/1482?version=1</td>
      <td>microbiome/clinical-mp-5-data-interpretation, proteomics/clinical-mp-5-data-interpretation, proteomics/metaproteomics, proteomics/metaquantome-data-creation</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>uniprotxml_downloader</td>
      <td>uniprotxml_downloader</td>
      <td>Download UniProt proteome in XML or fasta format</td>
      <td>2016-03-08</td>
      <td>https://www.uniprot.org/help/programmatic_access</td>
      <td>2.5.0</td>
      <td>requests</td>
      <td></td>
      <td>To update</td>
      <td>Proteomics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/uniprotxml_downloader</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/uniprotxml_downloader</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>239</td>
      <td>313</td>
      <td>1805</td>
      <td>2133</td>
      <td>73</td>
      <td>73</td>
      <td>143</td>
      <td>143</td>
      <td>41</td>
      <td>41</td>
      <td>1139</td>
      <td>1139</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3415</td>
      <td>3087</td>
      <td>427</td>
      <td>353</td>
      <td>https://usegalaxy.eu/published/workflow?id=0ab3e47ebd3b520e, https://usegalaxy.eu/published/workflow?id=229a955d29bd4064, https://usegalaxy.eu/published/workflow?id=24eaa682897379c8, https://usegalaxy.eu/published/workflow?id=3e01a2c6d954ae6e, https://usegalaxy.eu/published/workflow?id=41df8c7821b5f2fc, https://usegalaxy.eu/published/workflow?id=5374daf356516a40, https://usegalaxy.eu/published/workflow?id=5be158931abca947, https://usegalaxy.eu/published/workflow?id=5e4774036aa20b80, https://usegalaxy.eu/published/workflow?id=612dc5bdf388585f, https://usegalaxy.eu/published/workflow?id=6306656ec10bd18f, https://usegalaxy.eu/published/workflow?id=6cb85fe1ad0c9f3b, https://usegalaxy.eu/published/workflow?id=71816adecac46e74, https://usegalaxy.eu/published/workflow?id=73dc2384535bf6af, https://usegalaxy.eu/published/workflow?id=82b6871682f1659a, https://usegalaxy.eu/published/workflow?id=9a60ecf98d4f8134, https://usegalaxy.eu/published/workflow?id=9ef76d764d9961b2, https://usegalaxy.eu/published/workflow?id=c6aecbdbeb27585f, https://usegalaxy.eu/published/workflow?id=cb9a26ea53ce7a27, https://usegalaxy.eu/published/workflow?id=d87fc6685e47d27d, https://usegalaxy.eu/published/workflow?id=eeb6cff210cac091, https://usegalaxy.eu/published/workflow?id=f0bec337f9998289, https://usegalaxy.org.au/published/workflow?id=9281258aefd17c0c, https://usegalaxy.org/published/workflow?id=42331381df05b956, https://workflowhub.eu/workflows/1218?version=1, https://workflowhub.eu/workflows/1425?version=1, https://workflowhub.eu/workflows/1464?version=1, https://workflowhub.eu/workflows/37?version=1, https://workflowhub.eu/workflows/38?version=1, https://workflowhub.eu/workflows/39?version=1, https://workflowhub.eu/workflows/40?version=1</td>
      <td>microbiome/clinical-mp-1-database-generation, microbiome/clinical-mp-3-verification, proteomics/clinical-mp-1-database-generation, proteomics/clinical-mp-3-verification</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>usher</td>
      <td>usher_matutils, usher</td>
      <td>UShER toolkit wrappers</td>
      <td>2021-05-11</td>
      <td>https://github.com/yatisht/usher</td>
      <td>0.2.1</td>
      <td>usher</td>
      <td>0.6.6</td>
      <td>To update</td>
      <td>Phylogenetics</td>
      <td>Classification, Phylogenetic tree visualisation, Phylogenetic inference (from molecular sequences)</td>
      <td>Classification, Phylogenetic tree visualisation, Phylogenetic inference (from molecular sequences)</td>
      <td>Phylogeny, Evolutionary biology, Cladistics, Genotype and phenotype, Phylogenomics</td>
      <td>Cladistics, Genotype and phenotype, Phylogenomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/usher</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/usher</td>
      <td>usher</td>
      <td>usher</td>
      <td>The UShER toolkit includes a set of tools for for rapid, accurate placement of samples to existing phylogenies. While not restricted to SARS-CoV-2 phylogenetic analyses, it has enabled real-time phylogenetic analyses and genomic contact tracing in that its placement is orders of magnitude faster and more memory-efficient than previous methods.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>1060</td>
      <td>1060</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1060</td>
      <td>1060</td>
      <td>5</td>
      <td>5</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>valet</td>
      <td>valet</td>
      <td>A pipeline for detecting mis-assemblies in metagenomic assemblies.</td>
      <td>2017-11-16</td>
      <td>https://github.com/marbl/VALET</td>
      <td></td>
      <td>valet</td>
      <td>1.0</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Sequence assembly, Sequence assembly visualisation</td>
      <td>Sequence assembly, Sequence assembly visualisation</td>
      <td>Metagenomics, Sequence assembly</td>
      <td>Metagenomics, Sequence assembly</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/valet</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/valet</td>
      <td>valet</td>
      <td>VALET</td>
      <td>VALET is a pipeline for detecting mis-assemblies in metagenomic assemblies.</td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>74</td>
      <td>101</td>
      <td>449</td>
      <td>711</td>
      <td>67</td>
      <td>76</td>
      <td>160</td>
      <td>249</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>960</td>
      <td>609</td>
      <td>177</td>
      <td>141</td>
      <td>https://usegalaxy.eu/published/workflow?id=1524b189069b42a5, https://usegalaxy.eu/published/workflow?id=579ff83177c5ee09</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>vapor</td>
      <td>vapor</td>
      <td>Classify Influenza samples from raw short read sequence data</td>
      <td>2022-08-24</td>
      <td>https://github.com/wm75/vaporup</td>
      <td>1.0.3</td>
      <td>vapor</td>
      <td>1.0.3</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>Data retrieval, De-novo assembly, Read mapping</td>
      <td>Data retrieval, De-novo assembly, Read mapping</td>
      <td>Whole genome sequencing, Mapping, Sequence assembly</td>
      <td>Whole genome sequencing, Mapping, Sequence assembly</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/vapor</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/vapor</td>
      <td>vapor</td>
      <td>VAPOR</td>
      <td>VAPOR is a tool for classification of Influenza samples from raw short read sequence data for downstream bioinformatics analysis. VAPOR is provided with a fasta file of full-length sequences (&gt; 20,000) for a given segment, a set of reads, and attempts to retrieve a reference that is closest to the sample strain.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>188</td>
      <td>188</td>
      <td>197466</td>
      <td>197466</td>
      <td>127</td>
      <td>127</td>
      <td>1849</td>
      <td>1849</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>199315</td>
      <td>199315</td>
      <td>315</td>
      <td>315</td>
      <td>https://usegalaxy.eu/published/workflow?id=629e57958afed44c, https://usegalaxy.eu/published/workflow?id=7c986f4d14c6a539, https://usegalaxy.eu/published/workflow?id=a15dabcadb63b46d, https://usegalaxy.eu/published/workflow?id=c789ab71876c6ead, https://usegalaxy.eu/published/workflow?id=cd25f70f71683ab6, https://usegalaxy.org/published/workflow?id=1518c8c50747a1c1, https://usegalaxy.org/published/workflow?id=97263f3308077ff0, https://workflowhub.eu/workflows/1260?version=2</td>
      <td>variant-analysis/aiv-analysis</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>varvamp</td>
      <td>varvamp</td>
      <td>Variable VirusAMPlicons (varVAMP) is a tool to design primers for highly diverse viruses</td>
      <td>2024-01-27</td>
      <td>https://github.com/jonas-fuchs/varVAMP/</td>
      <td>1.2.2</td>
      <td>varvamp</td>
      <td>1.2.2</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis</td>
      <td>PCR primer design</td>
      <td>PCR primer design</td>
      <td>Virology, Probes and primers</td>
      <td>Virology, Probes and primers</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/varvamp</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/varvamp</td>
      <td>varvamp</td>
      <td>varVAMP</td>
      <td>variable VirusAMPlicons (varVAMP) is a tool to design primers for highly diverse viruses. The input is an alignment of your viral (full-genome) sequences.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>55</td>
      <td>55</td>
      <td>912</td>
      <td>912</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>912</td>
      <td>912</td>
      <td>55</td>
      <td>55</td>
      <td>https://workflowhub.eu/workflows/1627?version=1</td>
      <td>sequence-analysis/viral_primer_design</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>vegan</td>
      <td>vegan_diversity, vegan_fisher_alpha, vegan_rarefaction</td>
      <td>an R package fo community ecologist</td>
      <td>2015-08-27</td>
      <td>https://cran.r-project.org/package=vegan</td>
      <td>2.4-3</td>
      <td>r-vegan</td>
      <td>2.3_4</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Standardisation and normalisation, Analysis</td>
      <td>Standardisation and normalisation, Analysis</td>
      <td>Ecology, Phylogenetics, Environmental science</td>
      <td>Ecology, Phylogenetics, Environmental science</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/vegan</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/vegan</td>
      <td>vegan</td>
      <td>vegan</td>
      <td>Ordination methods, diversity analysis and other functions for community and vegetation ecologists</td>
      <td></td>
      <td>3</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>101</td>
      <td>116</td>
      <td>2314</td>
      <td>2547</td>
      <td>167</td>
      <td>318</td>
      <td>3493</td>
      <td>5788</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8335</td>
      <td>5807</td>
      <td>434</td>
      <td>268</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>velvet</td>
      <td>velvetg, velveth</td>
      <td>de novo genomic assembler specially designed for short read sequencing technologies</td>
      <td>2020-06-09</td>
      <td>https://www.ebi.ac.uk/~zerbino/velvet/</td>
      <td></td>
      <td>velvet</td>
      <td>1.2.10</td>
      <td>To update</td>
      <td>Assembly</td>
      <td>Formatting, De-novo assembly</td>
      <td>Formatting, De-novo assembly</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/velvet</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/velvet</td>
      <td>velvet</td>
      <td>Velvet</td>
      <td>A de novo genomic assembler specially designed for short read sequencing technologies, such as Solexa or 454 or SOLiD.</td>
      <td></td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>2567</td>
      <td>2888</td>
      <td>27621</td>
      <td>32158</td>
      <td>4951</td>
      <td>5263</td>
      <td>52855</td>
      <td>57273</td>
      <td>2097</td>
      <td>2221</td>
      <td>30746</td>
      <td>33121</td>
      <td>109</td>
      <td>109</td>
      <td>1097</td>
      <td>1097</td>
      <td>123649</td>
      <td>112319</td>
      <td>10481</td>
      <td>9724</td>
      <td>https://usegalaxy.eu/published/workflow?id=01dc0619e050dc91, https://usegalaxy.eu/published/workflow?id=56edd22d14392691, https://usegalaxy.eu/published/workflow?id=6d87481be47902e1, https://usegalaxy.eu/published/workflow?id=842e0ea7b1eb56af, https://usegalaxy.eu/published/workflow?id=b7c2440621968526, https://usegalaxy.eu/published/workflow?id=ef83f7fd2176a244, https://usegalaxy.org.au/published/workflow?id=30430b75d05f54eb, https://usegalaxy.org.au/published/workflow?id=533dbb3242ee4255, https://usegalaxy.org.au/published/workflow?id=7fa13ccbf044edc5, https://usegalaxy.org.au/published/workflow?id=83e3ceceb1a92493, https://usegalaxy.org.au/published/workflow?id=874492eb776ba813, https://usegalaxy.org.au/published/workflow?id=8b6739a000780d99, https://usegalaxy.org.au/published/workflow?id=94c7fdcbbc7b428a, https://usegalaxy.org.au/published/workflow?id=9b75924c483b5f6b, https://usegalaxy.org.au/published/workflow?id=aee5b5645c51cac3, https://usegalaxy.org.au/published/workflow?id=dccc8fb49d61ac33, https://usegalaxy.org.au/published/workflow?id=ee7a1615f187391d, https://usegalaxy.org.au/published/workflow?id=f25a05070bf97ded, https://usegalaxy.org.au/published/workflow?id=fb0986a952c35e0a, https://usegalaxy.org/published/workflow?id=109092d8dbda4398, https://usegalaxy.org/published/workflow?id=1c7af480caf150bd, https://usegalaxy.org/published/workflow?id=3cf46c167ccb374f, https://usegalaxy.org/published/workflow?id=4e1ae78405034ad3, https://usegalaxy.org/published/workflow?id=51c485af11d90700, https://usegalaxy.org/published/workflow?id=7a33bc2d0970b502, https://usegalaxy.org/published/workflow?id=c30f6ca171896908, https://usegalaxy.org/published/workflow?id=e2ab3d3219a648a7, https://usegalaxy.org/published/workflow?id=e745c45651d829b8, https://usegalaxy.org/published/workflow?id=f6bd33dedc7c6c12, https://usegalaxy.org/published/workflow?id=f926e70bee242785, https://workflowhub.eu/workflows/1622?version=1</td>
      <td>assembly/debruijn-graph-assembly, assembly/general-introduction</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>vibrant</td>
      <td>vibrant</td>
      <td>Virus Identification By iteRative ANnoTation</td>
      <td>2024-09-11</td>
      <td>https://github.com/AnantharamanLab/VIBRANT</td>
      <td>1.2.1</td>
      <td>vibrant</td>
      <td>1.2.1</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Sequence trimming, Scaffolding, Genome annotation</td>
      <td>Sequence trimming, Scaffolding, Genome annotation</td>
      <td>Metagenomics, Microbial ecology, Sequence assembly, Molecular interactions, pathways and networks, Endocrinology and metabolism</td>
      <td>Metagenomics, Microbial ecology, Sequence assembly, Molecular interactions, pathways and networks, Endocrinology and metabolism</td>
      <td>ufz</td>
      <td>https://github.com/Helmholtz-UFZ/ufz-galaxy-tools/blob/main/tools/vibrant</td>
      <td>https://github.com/Helmholtz-UFZ/galaxy-tools/tree/main/tools/vibrant</td>
      <td>VIBRANT</td>
      <td>VIBRANT</td>
      <td>Automated recovery, annotation and curation of microbial viruses, and evaluation of virome function from genomic sequences.Virus Identification By iteRative ANnoTation.Discovery Environment Applications List.Created by upendra_35, last modified on Nov 04, 2019.If you find VIBRANT useful please consider citing our preprint on bioRxiv:.Kieft, K., Zhou, Z., and Anantharaman, K. (2019). VIBRANT: Automated recovery, annotation and curation of microbial viruses, and evaluation of virome function from genomic sequences. BioRxiv 855387.VIBRANT is a tool for automated recovery and annotation of bacterial and archaeal viruses, determination of genome completeness, and characterization of virome function from metagenomic assemblies.The QuickStart tutorial provides an introduction to basic DE functionality and navigation.Please work through the tutorial and use the intercom button on the bottom right of this page if you have any questions</td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>65</td>
      <td>65</td>
      <td>1586</td>
      <td>1586</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1586</td>
      <td>1586</td>
      <td>65</td>
      <td>65</td>
      <td>https://usegalaxy.eu/published/workflow?id=87231a6a7938cb95, https://usegalaxy.eu/published/workflow?id=c62d65832377e376</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>viralverify</td>
      <td>viralverify</td>
      <td>Contigs prediction as viral, chromosomal, plasmidic or uncertain.</td>
      <td>2025-07-04</td>
      <td>https://github.com/ablab/viralVerify</td>
      <td>1.1</td>
      <td>viralverify</td>
      <td>1.1</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis, Metagenomics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/viralverify</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/viralverify</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>virannot</td>
      <td>virannot_blast2tsv, virannot_otu, virAnnot_rps2tsv</td>
      <td>virAnnot wrappers</td>
      <td>2025-08-04</td>
      <td>https://github.com/marieBvr/virAnnot</td>
      <td>1.2.0</td>
      <td>biopython</td>
      <td>1.70</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Sequence annotation, Sequence clustering, Sequence cluster visualisation</td>
      <td>Sequence annotation, Sequence clustering, Sequence cluster visualisation</td>
      <td>Metagenomics, Virology, Microbial ecology</td>
      <td>Metagenomics, Virology, Microbial ecology</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/virannot</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/virannot</td>
      <td>virannot</td>
      <td>virAnnot</td>
      <td>VirAnnot was build to ease the assembly, blast search, taxonomic annotation and OTUs assignation of viral metagenomic HTS data. It is used in Virologie team of UMR1332 BFP laboratory at INRAE. VirAnnot also takes part of the Euphresco project "Plant Health Bioinformatics Network". See more.It is designed to identify viruses in plant metagenomic data but it can be used to assemble and annotate any sequences with the NCBI taxonomy.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>16</td>
      <td>16</td>
      <td>336</td>
      <td>336</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>14</td>
      <td>14</td>
      <td>350</td>
      <td>350</td>
      <td>18</td>
      <td>18</td>
      <td>https://usegalaxy.eu/published/workflow?id=17f73aba442315e6, https://usegalaxy.eu/published/workflow?id=86465932556bc96c</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>virsorter</td>
      <td>virsorter</td>
      <td>VirSorter2 applies a multi-classifier, expert-guided approach to detect diverse DNA and RNA virus genomes.</td>
      <td>2024-06-20</td>
      <td>https://github.com/jiarong/VirSorter2/</td>
      <td>2.2.4</td>
      <td>virsorter</td>
      <td>2.2.4</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Taxonomic classification</td>
      <td>Taxonomic classification</td>
      <td>Metagenomics</td>
      <td>Metagenomics</td>
      <td>ufz</td>
      <td>https://github.com/Helmholtz-UFZ/ufz-galaxy-tools/blob/main/tools/virsorter</td>
      <td>https://github.com/Helmholtz-UFZ/galaxy-tools/tree/main/tools/virsorter</td>
      <td>virsorter</td>
      <td>virsorter</td>
      <td>Identify DNA and RNA virus sequences.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>144</td>
      <td>144</td>
      <td>2379</td>
      <td>2379</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2379</td>
      <td>2379</td>
      <td>144</td>
      <td>144</td>
      <td>https://usegalaxy.eu/published/workflow?id=62392daea25a3980, https://usegalaxy.eu/published/workflow?id=87231a6a7938cb95</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>vrhyme</td>
      <td>vrhyme</td>
      <td>Tool for dereplication and binning virus genomes from metagenomes.</td>
      <td>2025-05-14</td>
      <td>https://github.com/AnantharamanLab/vRhyme</td>
      <td>1.1.0</td>
      <td>vrhyme</td>
      <td>1.1.0</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Scaffolding, Read binning, Read mapping, Sequence trimming</td>
      <td>Scaffolding, Read binning, Read mapping, Sequence trimming</td>
      <td>Metagenomics, Microbial ecology, Model organisms, Machine learning, Sequence assembly</td>
      <td>Metagenomics, Microbial ecology, Model organisms, Machine learning, Sequence assembly</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/vrhyme</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/vrhyme</td>
      <td>vRhyme</td>
      <td>vRhyme</td>
      <td>vRhyme enables binning of viral genomes from metagenomes.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>47</td>
      <td>47</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>47</td>
      <td>47</td>
      <td>6</td>
      <td>6</td>
      <td>https://usegalaxy.eu/published/workflow?id=62392daea25a3980</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>vsearch</td>
      <td>vsearch_alignment, vsearch_chimera_detection, vsearch_clustering, vsearch_dereplication, vsearch_masking, vsearch_search, vsearch_shuffling, vsearch_sorting</td>
      <td>VSEARCH including searching, clustering, chimera detection, dereplication, sorting, masking and shuffling of sequences.</td>
      <td>2014-11-30</td>
      <td>https://github.com/torognes/vsearch</td>
      <td>2.8.3</td>
      <td>vsearch</td>
      <td>2.30.4</td>
      <td>To update</td>
      <td>Sequence Analysis</td>
      <td>DNA mapping, Chimera detection</td>
      <td>DNA mapping, Chimera detection</td>
      <td>Metagenomics, Sequence analysis</td>
      <td>Metagenomics, Sequence analysis</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/vsearch</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/vsearch</td>
      <td>vsearch</td>
      <td>VSEARCH</td>
      <td>High-throughput search and clustering sequence analysis tool. It supports de novo and reference based chimera detection, clustering, full-length and prefix dereplication, reverse complementation, masking, all-vs-all pairwise global alignment, exact and global alignment searching, shuffling, subsampling and sorting. It also supports FASTQ file analysis, filtering and conversion.</td>
      <td></td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>226</td>
      <td>259</td>
      <td>13600</td>
      <td>15795</td>
      <td>323</td>
      <td>642</td>
      <td>14828</td>
      <td>59476</td>
      <td>82</td>
      <td>97</td>
      <td>6641</td>
      <td>7811</td>
      <td>11</td>
      <td>12</td>
      <td>285</td>
      <td>286</td>
      <td>83368</td>
      <td>35354</td>
      <td>1010</td>
      <td>642</td>
      <td>https://usegalaxy.eu/published/workflow?id=401746888e8b3aa8, https://usegalaxy.eu/published/workflow?id=4eb9dfe91fc2dc67, https://usegalaxy.eu/published/workflow?id=9382f9bbbdddb5ad, https://usegalaxy.eu/published/workflow?id=9cbba5de2cbeeb0f, https://usegalaxy.eu/published/workflow?id=a8417c63e9e22afb, https://usegalaxy.eu/published/workflow?id=c3aa516e78ca4191, https://usegalaxy.org.au/published/workflow?id=14e80b7f9f68ec2a, https://usegalaxy.org.au/published/workflow?id=869440d35a618836, https://workflowhub.eu/workflows/232?version=1, https://workflowhub.eu/workflows/233?version=1</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
  </tbody>
</table>

</div>