<script src="https://code.jquery.com/jquery-3.7.0.js"></script>
<script src="https://cdn.datatables.net/1.13.6/js/jquery.dataTables.min.js"></script>
<script src="https://cdn.datatables.net/searchbuilder/1.6.0/js/dataTables.searchBuilder.min.js"></script>
<script src="https://cdn.datatables.net/datetime/1.5.1/js/dataTables.dateTime.min.js"></script>

<script src="https://cdn.datatables.net/responsive/2.5.0/js/dataTables.responsive.min.js"></script>

<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/responsive/2.2.3/css/responsive.dataTables.min.css">

<link rel="stylesheet" href="https://cdn.datatables.net/1.13.6/css/jquery.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/searchbuilder/1.6.0/css/searchBuilder.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/datetime/1.5.1/css/dataTables.dateTime.min.css">

<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" />
<script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>

<style type="text/css">
  .tooltip {
    font-family: Georgia;
    font-size: 18px;
  }
  .tooltip .tooltip-inner {
    background-color: #00FFFF;
    color: #000000;
    min-width: 400px;
  }
  </style>

<script>
    $(document).ready(function() {
    $('#dataframe').DataTable( {
        "sScrollX": "100%",
        "dom": "Qlfrtip",

        responsive: {
            details: {
                type: 'column'
            }
        },
        "bAutoWidth": false,
        columnDefs: [ {
            className: 'control',
            orderable: false,
            targets:   [ 0 ],
            width: 100,
        },
        { type: 'natural', targets: 1 } // allow normal sorting
        ],

    });
});
</script>

<div class="display nowrap" style="width:80%" width="100%">

<table class="dataframe display nowrap" id="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th>Expand</th>
      <th>Topic</th>
      <th>Title</th>
      <th>Link</th>
      <th>EDAM topic</th>
      <th>EDAM operation</th>
      <th>Creation</th>
      <th>Last modification</th>
      <th>Version</th>
      <th>Tutorial</th>
      <th>Slides</th>
      <th>Video</th>
      <th>Workflows</th>
      <th>Tools</th>
      <th>Servers with precise tool versions</th>
      <th>Servers with tool but different versions</th>
      <th>Feedback number</th>
      <th>Feedback mean note</th>
      <th>Visitors</th>
      <th>Page views</th>
      <th>Visit duration</th>
      <th>Video views</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Genome Assembly of a bacterial genome (MRSA) sequenced using Illumina MiSeq Data</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/mrsa-illumina/tutorial.html</td>
      <td>Sequence assembly, Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology</td>
      <td>Sequence assembly validation, Statistical calculation, Sequence assembly visualisation, Visualisation, Genome assembly, Sequencing quality control, Sequence composition calculation, Sequence contamination filtering</td>
      <td>2021-03-24</td>
      <td>2024-09-18</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>fastp, bandage_image, upload1, bandage_info, quast, fastqc, shovill</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>Galaxy@AuBi, GalaxyTrakr, UseGalaxy.be, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>6735</td>
      <td>10837</td>
      <td>3.783333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Genome Assembly of MRSA from Oxford Nanopore MinION data (and optionally Illumina data)</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/mrsa-nanopore/tutorial.html</td>
      <td>Sequence assembly, Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology, Antimicrobial Resistance</td>
      <td>Scatter plot plotting, Sequencing error detection, Sequence alignment, Visualisation, Cross-assembly, Read mapping, Sequencing quality control, Genome assembly, De-novo assembly, Box-Whisker plot plotting, Sequence contamination filtering, Sequence assembly validation, Mapping assembly, Statistical calculation, Sequence assembly visualisation, Filtering, Sequence composition calculation</td>
      <td>2021-03-24</td>
      <td>2024-03-13</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>CONVERTER_bz2_to_uncompressed, fastp, filtlong, bandage_image, porechop, upload1, quast, flye, nanoplot, bwa_mem2, polypolish, fastqc</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>7147</td>
      <td>10976</td>
      <td>3.400000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Assembly of metagenomic sequencing data</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/metagenomics-assembly/tutorial.html</td>
      <td>Metagenomics, Sequence assembly</td>
      <td>Data handling, Visualisation, Sequence trimming, Read mapping, Sequencing quality control, Genome assembly, Sequence alignment analysis, Sequence contamination filtering, Sequence assembly validation, Variant calling, Statistical calculation, Sequence assembly visualisation, Sequence file editing, Local alignment, Sequence composition calculation, Read pre-processing, Primer removal, Formatting</td>
      <td>2024-10-02</td>
      <td>2024-10-02</td>
      <td>0</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>ngsutils_bam_filter, tp_cat, bamtools, bandage_image, filter_tabular, megahit_contig2fastg, bandage_info, bg_uniq, quast, seqtk_subseq, cutadapt, random_lines1, collection_column_join, megahit, coverm_contig, fastqc, metaspades, bowtie2</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td>0</td>
      <td></td>
      <td>6</td>
      <td>7</td>
      <td>0.383333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Unicycler Assembly</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/unicycler-assembly/tutorial.html</td>
      <td>Sequence assembly, Genomics, Microbiology</td>
      <td>Genome annotation, Visualisation, Genome assembly, Sequencing quality control, Sequence assembly validation, Gene prediction, Statistical calculation, Aggregation, Validation, Sequence composition calculation, Coding region prediction</td>
      <td>2017-10-11</td>
      <td>2024-03-13</td>
      <td>23</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>prokka, unicycler, quast, multiqc, fastqc</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>GalaxyTrakr, HyPhy HIV NGS Tools, MISSISSIPPI, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2144791</td>
      <td>5917098</td>
      <td>4.566667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Making sense of a newly assembled genome</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/ecoli_comparison/tutorial.html</td>
      <td>Sequence assembly, Genomics, Microbiology</td>
      <td>Sequence alignment, Sequence visualisation, Read mapping, Genome visualisation, Mapping</td>
      <td>2018-06-14</td>
      <td>2024-06-24</td>
      <td>29</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>join1, tp_sed_tool, lastz_wrapper_2, upload1, fasta_filter_by_length, bedtools_intersectbed, datamash_ops, circos_interval_to_tiles, jbrowse, tp_cat, mergeCols1, Filter1, bedtools_sortbed, tp_sort_header_tool, Grep1, Cut1, collapse_dataset, cat1, tp_replace_in_column, addValue, circos_aln_to_links, fasta_compute_length, circos, tp_grep_tool, tp_replace_in_line, random_lines1, bedtools_complementbed</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>6026</td>
      <td>12166</td>
      <td>3.016667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Checking expected species and contamination in bacterial isolate</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/bacterial-isolate-species-contamination-checking/tutorial.html</td>
      <td>Whole genome sequencing, Genomics, Microbiology, Microbial ecology</td>
      <td>Statistical calculation, Taxonomic classification, Cross-assembly, Expression analysis</td>
      <td>2024-03-04</td>
      <td>2024-06-27</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>recentrifuge, kraken2, upload1, est_abundance</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td>Galaxy@AuBi, GalaxyTrakr, UseGalaxy.cz, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>260</td>
      <td>343</td>
      <td>3.533333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Tree thinking for tuberculosis evolution and epidemiology</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/mtb_phylogeny/tutorial.html</td>
      <td>Genomics, Microbiology, Phylogeny, Infectious disease</td>
      <td>Phylogenetic tree analysis, Sequence analysis</td>
      <td>2022-03-16</td>
      <td>2024-03-12</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>upload1, interactive_tool_rstudio, raxml</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>1373</td>
      <td>2071</td>
      <td>2.233333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Identifying tuberculosis transmission links: from SNPs to transmission clusters</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/mtb_transmission/tutorial.html</td>
      <td>Genomics, Microbiology, Infectious disease, DNA polymorphism</td>
      <td>Phylogenetic tree visualisation, Variant calling, Antimicrobial resistance prediction, Phylogenetic tree generation</td>
      <td>2022-03-16</td>
      <td>2024-07-12</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tp_sed_tool, tp_easyjoin_tool, tp_cat, bcftools_consensus, samtools_view, upload1, snippy, tp_grep_tool, tp_replace_in_line, snp_dists, Grep1, tb_variant_filter, tb_profiler_profile, collapse_dataset, snp_sites, trimmomatic, __MERGE_COLLECTION__, addName</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>1605</td>
      <td>2499</td>
      <td>3.050000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Genome annotation with Prokka</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/annotation-with-prokka/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis</td>
      <td>Genome visualisation, Gene prediction, Genome annotation, Coding region prediction</td>
      <td>2018-03-06</td>
      <td>2024-08-08</td>
      <td>25</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>jbrowse, prokka</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>Galaxy@AuBi, GalaxyTrakr, MISSISSIPPI, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>2149523</td>
      <td>5925124</td>
      <td>4.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Identification of AMR genes in an assembled bacterial genome</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/amr-gene-detection/tutorial.html</td>
      <td>Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology, Sequence analysis, Infectious disease, Antimicrobial Resistance</td>
      <td>Genome visualisation, Genome annotation, Read mapping</td>
      <td>2024-01-23</td>
      <td>2024-09-27</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>jbrowse, tbl2gff3, upload1, Grep1, staramr_search, bakta, bowtie2</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr</td>
      <td>Galaxy@AuBi, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>1138</td>
      <td>1576</td>
      <td>2.750000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Refining Genome Annotations with Apollo (prokaryotes)</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/apollo/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis</td>
      <td>Genome visualisation</td>
      <td>2021-06-04</td>
      <td>2024-08-08</td>
      <td>20</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>jbrowse, create_or_update, iframe, list_organism, create_account</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>1906178</td>
      <td>5119567</td>
      <td>4.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Bacterial Genome Annotation</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/bacterial-genome-annotation/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis, Whole genome sequencing, Functional genomics, Mobile genetic elements</td>
      <td>Genome annotation, Nucleic acid feature detection, Multilocus sequence typing, Genome visualisation, Genome assembly, Sequence motif recognition, Structural variation detection, Scaffolding, Protein feature detection</td>
      <td>2024-02-01</td>
      <td>2024-09-08</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>jbrowse, tbl2gff3, plasmidfinder, integron_finder, upload1, isescan, tp_tail_tool, bakta, tp_replace_in_column, Grouping1</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td>Galaxy@AuBi, UseGalaxy.cz, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>2243</td>
      <td>2969</td>
      <td>2.333333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Comparative gene analysis in unannotated genomes</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/gene-centric/tutorial.html</td>
      <td>Genomics, Gene and protein families, Sequence analysis, Phylogeny</td>
      <td>Database search, Transcriptome assembly, Phylogenetic tree generation, De-novo assembly, Sequence alignment analysis, Multiple sequence alignment, Coding region prediction</td>
      <td>2022-09-08</td>
      <td>2024-03-18</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>orfipy, rapidnj, join1, bg_diamond_makedb, Filter1, tp_split_on_column, tab2fasta, collapse_dataset, rbc_mafft, bg_diamond_view, Cut1, Add_a_column1, bg_diamond, regexColumn1, gops_intersect_1, cat1</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>1045</td>
      <td>1602</td>
      <td>3.583333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Essential genes detection with Transposon insertion sequencing</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/tnseq/tutorial.html</td>
      <td>Genomics, Microbiology, Sequence analysis, Mobile genetic elements</td>
      <td>Sequence trimming, Transposon prediction, Read pre-processing, Primer removal</td>
      <td>2019-07-02</td>
      <td>2024-03-18</td>
      <td>12</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>__EXTRACT_DATASET__, bowtie_wrapper, gff_to_prot, tp_easyjoin_tool, Filter1, transit_gumbel, bg_find_subsequences, tp_sort_header_tool, cutadapt, Cut1, Add_a_column1, deeptools_bam_coverage</td>
      <td>UseGalaxy.be, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td>0</td>
      <td></td>
      <td>2140389</td>
      <td>5910080</td>
      <td>4.266667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Genome annotation with Prokka</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/annotation-with-prokka/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis</td>
      <td>Genome visualisation, Gene prediction, Genome annotation, Coding region prediction</td>
      <td>2018-03-06</td>
      <td>2024-08-08</td>
      <td>25</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>jbrowse, prokka</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>Galaxy@AuBi, GalaxyTrakr, MISSISSIPPI, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>2149523</td>
      <td>5925127</td>
      <td>4.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Identification of AMR genes in an assembled bacterial genome</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/amr-gene-detection/tutorial.html</td>
      <td>Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology, Sequence analysis, Infectious disease, Antimicrobial Resistance</td>
      <td>Genome visualisation, Genome annotation, Read mapping</td>
      <td>2024-01-23</td>
      <td>2024-09-27</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>jbrowse, tbl2gff3, upload1, Grep1, staramr_search, bakta, bowtie2</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr</td>
      <td>Galaxy@AuBi, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>1138</td>
      <td>1576</td>
      <td>2.750000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Refining Genome Annotations with Apollo (prokaryotes)</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/apollo/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis</td>
      <td>Genome visualisation</td>
      <td>2021-06-04</td>
      <td>2024-08-08</td>
      <td>20</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>jbrowse, create_or_update, iframe, list_organism, create_account</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>1906178</td>
      <td>5119570</td>
      <td>4.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Bacterial Genome Annotation</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/bacterial-genome-annotation/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis, Whole genome sequencing, Functional genomics, Mobile genetic elements</td>
      <td>Genome annotation, Nucleic acid feature detection, Multilocus sequence typing, Genome visualisation, Genome assembly, Sequence motif recognition, Structural variation detection, Scaffolding, Protein feature detection</td>
      <td>2024-02-01</td>
      <td>2024-09-08</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>jbrowse, tbl2gff3, plasmidfinder, integron_finder, upload1, isescan, tp_tail_tool, bakta, tp_replace_in_column, Grouping1</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td>Galaxy@AuBi, UseGalaxy.cz, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>2243</td>
      <td>2969</td>
      <td>2.333333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Microbial Variant Calling</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/microbial-variants/tutorial.html</td>
      <td>Genomics, Sequence assembly, DNA polymorphism, Microbiology, Sequence analysis, Genetic variation</td>
      <td>Genome visualisation, Variant calling, Phylogenetic tree visualisation, Phylogenetic tree generation</td>
      <td>2018-02-26</td>
      <td>2024-03-15</td>
      <td>23</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>jbrowse, snippy</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>GalaxyTrakr, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>12475</td>
      <td>20692</td>
      <td>1.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Genome Assembly of MRSA from Oxford Nanopore MinION data (and optionally Illumina data)</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/mrsa-nanopore/tutorial.html</td>
      <td>Sequence assembly, Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology, Antimicrobial Resistance</td>
      <td>Scatter plot plotting, Sequencing error detection, Sequence alignment, Visualisation, Cross-assembly, Read mapping, Sequencing quality control, Genome assembly, De-novo assembly, Box-Whisker plot plotting, Sequence contamination filtering, Sequence assembly validation, Mapping assembly, Statistical calculation, Sequence assembly visualisation, Filtering, Sequence composition calculation</td>
      <td>2021-03-24</td>
      <td>2024-03-13</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>CONVERTER_bz2_to_uncompressed, fastp, filtlong, bandage_image, porechop, upload1, quast, flye, nanoplot, bwa_mem2, polypolish, fastqc</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>7148</td>
      <td>10977</td>
      <td>3.400000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Making sense of a newly assembled genome</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/ecoli_comparison/tutorial.html</td>
      <td>Sequence assembly, Genomics, Microbiology</td>
      <td>Sequence alignment, Sequence visualisation, Read mapping, Genome visualisation, Mapping</td>
      <td>2018-06-14</td>
      <td>2024-06-24</td>
      <td>29</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>join1, tp_sed_tool, lastz_wrapper_2, upload1, fasta_filter_by_length, bedtools_intersectbed, datamash_ops, circos_interval_to_tiles, jbrowse, tp_cat, mergeCols1, Filter1, bedtools_sortbed, tp_sort_header_tool, Grep1, Cut1, collapse_dataset, cat1, tp_replace_in_column, addValue, circos_aln_to_links, fasta_compute_length, circos, tp_grep_tool, tp_replace_in_line, random_lines1, bedtools_complementbed</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>6026</td>
      <td>12166</td>
      <td>3.016667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Analyses of metagenomics data - The global picture</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/general-tutorial/tutorial.html</td>
      <td>Metagenomics, Microbial ecology, Taxonomy, Sequence analysis</td>
      <td>DNA barcoding, Visualisation, Phylogenetic analysis, Sequencing quality control, Taxonomic classification, Sequence read processing, Sequence clustering</td>
      <td>2017-06-22</td>
      <td>2024-08-09</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>humann2_regroup_table, mothur_summary_seqs, humann2_renorm_table, mothur_make_biom, mothur_count_seqs, mothur_classify_seqs, mothur_make_group, mothur_make_shared, mothur_filter_seqs, mothur_merge_files, mothur_classify_otu, mothur_align_seqs, humann2, mothur_cluster_split, mothur_screen_seqs, metaphlan2, mothur_pre_cluster, mothur_unique_seqs, taxonomy_krona_chart, krona-text, metaphlan2krona</td>
      <td>UseGalaxy.be</td>
      <td>UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>3586</td>
      <td>4852</td>
      <td>2.783333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Identification of the micro-organisms in a beer using Nanopore sequencing</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/beer-data-analysis/tutorial.html</td>
      <td>Metagenomics, Microbial ecology, Taxonomy, Sequence analysis</td>
      <td>Taxonomic classification, Statistical calculation, Aggregation, Visualisation, Sequencing quality control, Sequence composition calculation, Sequence contamination filtering</td>
      <td>2022-09-29</td>
      <td>2024-06-14</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>porechop, fastp, Filter1, taxonomy_krona_chart, kraken2, krakentools_kreport2krona, fastqc</td>
      <td>GalaxyTrakr, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>1239</td>
      <td>1942</td>
      <td>2.966667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Pathogen detection from (direct Nanopore) sequencing data using Galaxy - Foodborne Edition</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.html</td>
      <td>Metagenomics, Public health and epidemiology, Taxonomy, Sequence assembly, Pathology, Sequence analysis</td>
      <td>Scatter plot plotting, Visualisation, Cross-assembly, Phylogenetic tree generation, Genome assembly, Sequencing quality control, Box-Whisker plot plotting, Multiple sequence alignment, Mapping assembly, Base-calling, Aggregation, Statistical calculation, Pairwise sequence alignment, Validation, Taxonomic classification, Sequence composition calculation, Mapping, Antimicrobial resistance prediction, Phylogenetic tree analysis, Data handling, Multilocus sequence typing, De-novo assembly, Sequence alignment analysis, Phylogenetic tree reconstruction, Sequence contamination filtering, Sequence assembly, Variant calling, Sequence assembly visualisation, Phylogenetic tree generation (from molecular sequences), Phylogenetic tree generation (maximum likelihood and Bayesian methods)</td>
      <td>2023-01-26</td>
      <td>2024-09-27</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>__FILTER_FAILED_DATASETS__, Count1, minimap2, snpSift_filter, compose_text_param, tab2fasta, tp_find_and_replace, tp_head_tool, tp_multijoin_tool, param_value_from_file, collection_column_join, collection_element_identifiers, bcftools_norm, krakentools_kreport2krona, krakentools_extract_kraken_reads, ggplot2_heatmap, bamtools_split_mapped, fasta_merge_files_and_filter_unique_sequences, barchart_gnuplot, samtools_fastx, flye, medaka_consensus_pipeline, tp_cut_tool, fastqc, snpSift_extractFields, regex1, Grouping1, tp_sorted_uniq, porechop, table_compute, bcftools_consensus, split_file_to_collection, bandage_image, abricate, Paste1, add_line_to_file, tp_split_on_column, multiqc, Grep1, Cut1, collapse_dataset, Remove beginning1, clustalw, bedtools_getfastabed, tp_replace_in_column, fasttree, newick_display, fastp, samtools_coverage, samtools_depth, clair3, taxonomy_krona_chart, __FILTER_EMPTY_DATASETS__, kraken2, __BUILD_LIST__, Add_a_column1, mlst, nanoplot, regexColumn1, CONVERTER_gz_to_uncompressed, fasta2tab</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>1964</td>
      <td>2918</td>
      <td>3.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Metatranscriptomics analysis using microbiome RNA-seq data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metatranscriptomics/tutorial.html</td>
      <td>Metatranscriptomics, Microbial ecology, Taxonomy, Function analysis, Sequence analysis</td>
      <td>Visualisation, Sequencing quality control, Phylogenetic inference, Phylogenetic tree editing, Statistical calculation, Taxonomic classification, Validation, Sequence composition calculation, Sequence comparison, Sequence similarity search, Phylogenetic tree analysis, Sequence trimming, Phylogenetic analysis, Phylogenetic tree visualisation, Sequence alignment analysis, Species frequency estimation, Conversion, Nucleic acid sequence analysis, Read pre-processing, Primer removal</td>
      <td>2019-11-21</td>
      <td>2024-06-14</td>
      <td>4</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>fastq_paired_end_interlacer, tp_find_and_replace, metaphlan, fastqc, humann, graphlan_annotate, graphlan, humann_split_stratified_table, humann_renorm_table, humann_regroup_table, humann_rename_table, cutadapt, combine_metaphlan2_humann2, multiqc, bg_sortmerna, Grep1, Cut1, export2graphlan, humann_unpack_pathways, taxonomy_krona_chart</td>
      <td>UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2136358</td>
      <td>5901051</td>
      <td>4.266667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Taxonomic Profiling and Visualization of Metagenomic Data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/taxonomic-profiling/tutorial.html</td>
      <td>Metagenomics, Microbial ecology, Taxonomy, Sequence analysis</td>
      <td>Phylogenetic tree analysis, Statistical calculation, Taxonomic classification, Aggregation, Visualisation, Nucleic acid sequence analysis</td>
      <td>2023-05-03</td>
      <td>2024-06-14</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>interactive_tool_phinch, est_abundance, taxonomy_krona_chart, interactive_tool_pavian, kraken2, kraken_biom, __UNZIP_COLLECTION__, krakentools_kreport2krona, metaphlan</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.fr, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>4443</td>
      <td>6096</td>
      <td>1.966667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Metatranscriptomics analysis using microbiome RNA-seq data (short)</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metatranscriptomics-short/tutorial.html</td>
      <td>Metatranscriptomics, Microbial ecology, Taxonomy, Function analysis, Sequence analysis</td>
      <td>Visualisation, Sequencing quality control, Phylogenetic inference, Phylogenetic tree editing, Statistical calculation, Taxonomic classification, Validation, Sequence composition calculation, Sequence comparison, Sequence similarity search, Phylogenetic tree analysis, Sequence trimming, Phylogenetic analysis, Phylogenetic tree visualisation, Sequence alignment analysis, Species frequency estimation, Conversion, Nucleic acid sequence analysis, Read pre-processing, Primer removal</td>
      <td>2020-02-13</td>
      <td>2024-06-14</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>fastq_paired_end_interlacer, tp_find_and_replace, metaphlan, fastqc, humann, graphlan_annotate, graphlan, humann_split_stratified_table, humann_renorm_table, humann_regroup_table, humann_rename_table, cutadapt, combine_metaphlan2_humann2, multiqc, bg_sortmerna, Grep1, Cut1, export2graphlan, humann_unpack_pathways, taxonomy_krona_chart</td>
      <td>UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>274</td>
      <td>351</td>
      <td>1.433333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Antibiotic resistance detection</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/plasmid-metagenomics-nanopore/tutorial.html</td>
      <td>Metagenomic sequencing, Public health and epidemiology, Metagenomics, Microbiology, Infectious disease, Sequence analysis, Antimicrobial Resistance</td>
      <td>Scatter plot plotting, Genome assembly, Box-Whisker plot plotting, De-novo assembly, Sequence analysis, Mapping assembly, Aggregation, Sequence assembly visualisation, Pairwise sequence alignment</td>
      <td>2019-06-25</td>
      <td>2024-06-14</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>unicycler, minimap2, bandage_image, PlasFlow, racon, nanoplot, gfa_to_fa, staramr_search, miniasm</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>1281</td>
      <td>1950</td>
      <td>4.033333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>16S Microbial analysis with Nanopore data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/nanopore-16S-metagenomics/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis</td>
      <td>Taxonomic classification, Statistical calculation, Visualisation, Validation, Sequencing quality control, Sequence composition calculation, Sequence contamination filtering</td>
      <td>2020-11-24</td>
      <td>2024-07-31</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>porechop, fastp, tp_replace_in_line, taxonomy_krona_chart, multiqc, kraken2, datamash_reverse, fastqc, Remove beginning1</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>GalaxyTrakr, UseGalaxy.be, UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>2758</td>
      <td>3844</td>
      <td>2.716667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Building an amplicon sequence variant (ASV) table from 16S data using DADA2</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/dada-16S/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis</td>
      <td>Variant calling, DNA barcoding, Visualisation, Analysis, Deposition</td>
      <td>2024-06-05</td>
      <td>2024-09-26</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tp_head_tool, collection_element_identifiers, __UNZIP_COLLECTION__, interactive_tool_phyloseq, dada2_mergePairs, dada2_removeBimeraDenovo, dada2_assignTaxonomyAddspecies, Grouping1, dada2_plotQualityProfile, dada2_filterAndTrim, cat1, tp_replace_in_column, dada2_seqCounts, dada2_makeSequenceTable, tp_replace_in_line, Add_a_column1, dada2_dada, __SORTLIST__, dada2_learnErrors, phyloseq_from_dada2</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1089</td>
      <td>1460</td>
      <td>1.833333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Assembly of metagenomic sequencing data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metagenomics-assembly/tutorial.html</td>
      <td>Metagenomics, Sequence assembly</td>
      <td>Data handling, Visualisation, Sequence trimming, Read mapping, Sequencing quality control, Genome assembly, Sequence alignment analysis, Sequence contamination filtering, Sequence assembly validation, Variant calling, Statistical calculation, Sequence assembly visualisation, Sequence file editing, Local alignment, Sequence composition calculation, Read pre-processing, Primer removal, Formatting</td>
      <td>2022-12-05</td>
      <td>2024-06-14</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>ngsutils_bam_filter, tp_cat, bamtools, bandage_image, filter_tabular, megahit_contig2fastg, bandage_info, bg_uniq, quast, seqtk_subseq, cutadapt, random_lines1, collection_column_join, megahit, coverm_contig, fastqc, metaspades, bowtie2</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td>0</td>
      <td></td>
      <td>1613</td>
      <td>2258</td>
      <td>2.950000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>QIIME 2 Moving Pictures</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/qiime2-moving-pictures/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis</td>
      <td></td>
      <td>2024-03-14</td>
      <td>2024-03-14</td>
      <td>1</td>
      <td>external</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>131</td>
      <td>195</td>
      <td>3.383333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Binning of metagenomic sequencing data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metagenomics-binning/tutorial.html</td>
      <td>Metagenomics, Sequence assembly</td>
      <td>Sequence assembly validation, Statistical calculation, Sequencing quality control, Validation, Genome assembly, Sequence composition calculation</td>
      <td>2023-12-05</td>
      <td>2024-09-12</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>megahit, checkm_lineage_wf</td>
      <td>GalaxyTrakr, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr</td>
      <td>UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>2076</td>
      <td>2537</td>
      <td>1.416667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>QIIME 2 Cancer Microbiome Intervention</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/qiime2-cancer-microbiome-intervention/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis</td>
      <td></td>
      <td>2024-02-12</td>
      <td>2024-03-14</td>
      <td>3</td>
      <td>external</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>153</td>
      <td>281</td>
      <td>3.483333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>16S Microbial Analysis with mothur (extended)</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/mothur-miseq-sop/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis</td>
      <td>Phylogenetic tree analysis, DNA barcoding, Visualisation, Phylogenetic analysis, Sequencing quality control, Phylogenetic tree generation, Phylogenetic tree reconstruction, Taxonomic classification, Sequence read processing, Sequence clustering</td>
      <td>2017-02-12</td>
      <td>2024-03-14</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>mothur_dist_shared, mothur_seq_error, mothur_summary_seqs, mothur_remove_lineage, mothur_make_biom, mothur_count_seqs, mothur_get_groups, mothur_classify_seqs, mothur_remove_groups, mothur_make_shared, mothur_filter_seqs, mothur_dist_seqs, mothur_remove_seqs, mothur_rarefaction_single, mothur_venn, mothur_classify_otu, mothur_count_groups, mothur_align_seqs, mothur_taxonomy_to_krona, mothur_heatmap_sim, mothur_chimera_vsearch, mothur_cluster_split, mothur_screen_seqs, newick_display, mothur_pre_cluster, mothur_make_contigs, mothur_unique_seqs, XY_Plot_1, taxonomy_krona_chart, mothur_cluster, mothur_tree_shared, mothur_summary_single, mothur_sub_sample</td>
      <td>UseGalaxy.be, UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>5448</td>
      <td>8271</td>
      <td>2.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>16S Microbial Analysis with mothur (short)</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/mothur-miseq-sop-short/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis</td>
      <td>Phylogenetic tree analysis, DNA barcoding, Visualisation, Phylogenetic analysis, Sequencing quality control, Phylogenetic tree generation, Phylogenetic tree reconstruction, Taxonomic classification, Sequence read processing, Sequence clustering</td>
      <td>2019-05-13</td>
      <td>2024-06-14</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>mothur_dist_shared, mothur_summary_seqs, mothur_remove_lineage, mothur_count_seqs, mothur_classify_seqs, mothur_remove_groups, mothur_make_shared, mothur_filter_seqs, mothur_dist_seqs, mothur_remove_seqs, mothur_rarefaction_single, mothur_venn, mothur_classify_otu, mothur_count_groups, mothur_heatmap_sim, mothur_chimera_vsearch, mothur_cluster_split, mothur_screen_seqs, collapse_dataset, newick_display, mothur_pre_cluster, mothur_unique_seqs, XY_Plot_1, mothur_cluster, mothur_tree_shared, mothur_summary_single, mothur_sub_sample</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1985</td>
      <td>3572</td>
      <td>5.616667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Tree thinking for tuberculosis evolution and epidemiology</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/mtb_phylogeny/tutorial.html</td>
      <td>Genomics, Microbiology, Phylogeny, Infectious disease</td>
      <td>Phylogenetic tree analysis, Sequence analysis</td>
      <td>2022-03-16</td>
      <td>2024-03-12</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>upload1, interactive_tool_rstudio, raxml</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>1374</td>
      <td>2072</td>
      <td>2.216667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Identifying tuberculosis transmission links: from SNPs to transmission clusters</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/mtb_transmission/tutorial.html</td>
      <td>Genomics, Microbiology, Infectious disease, DNA polymorphism</td>
      <td>Phylogenetic tree visualisation, Variant calling, Antimicrobial resistance prediction, Phylogenetic tree generation</td>
      <td>2022-03-16</td>
      <td>2024-07-12</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tp_sed_tool, tp_easyjoin_tool, tp_cat, bcftools_consensus, samtools_view, upload1, snippy, tp_grep_tool, tp_replace_in_line, snp_dists, Grep1, tb_variant_filter, tb_profiler_profile, collapse_dataset, snp_sites, trimmomatic, __MERGE_COLLECTION__, addName</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>1605</td>
      <td>2499</td>
      <td>3.050000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Identification of AMR genes in an assembled bacterial genome</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/amr-gene-detection/tutorial.html</td>
      <td>Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology, Sequence analysis, Infectious disease, Antimicrobial Resistance</td>
      <td>Genome visualisation, Genome annotation, Read mapping</td>
      <td>2024-01-23</td>
      <td>2024-09-27</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>jbrowse, tbl2gff3, upload1, Grep1, staramr_search, bakta, bowtie2</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr</td>
      <td>Galaxy@AuBi, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>1138</td>
      <td>1576</td>
      <td>2.750000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>M. tuberculosis Variant Analysis</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/tb-variant-analysis/tutorial.html</td>
      <td>Genomics, Microbiology, Sequence assembly, Genetic variation, Public health and epidemiology, Infectious disease</td>
      <td>Sequence alignment, Phylogenetic tree generation, Phylogenetic tree visualisation, Sequencing quality control, Sequence alignment analysis, Genome visualisation, Sequence analysis, Sequence contamination filtering, Variant calling, Global alignment, Statistical calculation, Taxonomic classification, Validation, Local alignment, Sequence composition calculation, Antimicrobial resistance prediction</td>
      <td>2020-07-25</td>
      <td>2024-07-26</td>
      <td>30</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tp_sed_tool, upload1, snippy, qualimap_bamqc, tb_variant_filter, EMBOSS: seqret84, tb_profiler_profile, mosdepth, jvarkit_wgscoverageplotter, tbvcfreport, tp_awk_tool, fastqc, jbrowse, samtools_stats, bcftools_consensus, multiqc, fastp, kraken2, EMBOSS:%20seqret84, __FLATTEN__</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>5788</td>
      <td>9172</td>
      <td>2.950000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>16S Microbial analysis with Nanopore data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/nanopore-16S-metagenomics/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis</td>
      <td>Taxonomic classification, Statistical calculation, Visualisation, Validation, Sequencing quality control, Sequence composition calculation, Sequence contamination filtering</td>
      <td>2020-11-24</td>
      <td>2024-07-31</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>porechop, fastp, tp_replace_in_line, taxonomy_krona_chart, multiqc, kraken2, datamash_reverse, fastqc, Remove beginning1</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>GalaxyTrakr, UseGalaxy.be, UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>2758</td>
      <td>3844</td>
      <td>2.716667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Metaproteomics tutorial</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaproteomics/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Biodiversity, Taxonomy</td>
      <td>Prediction and recognition, Visualisation</td>
      <td>2017-06-28</td>
      <td>2024-06-14</td>
      <td>41</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>query_tabular, peptide_shaker, search_gui, unipept, sqlite_to_tabular</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>2310</td>
      <td>3828</td>
      <td>2.633333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>metaQuantome 3: Taxonomy</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaquantome-taxonomy/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Metatranscriptomics, Microbial ecology, Metagenomics, Taxonomy</td>
      <td>Visualisation, Indexing, Differential protein expression analysis, Functional clustering, Statistical inference, Principal component visualisation, Query and retrieval, Heat map generation, Quantification, Filtering</td>
      <td>2020-10-29</td>
      <td>2024-03-15</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>metaquantome_filter, metaquantome_sample, metaquantome_stat, metaquantome_expand, metaquantome_viz, metaquantome_db</td>
      <td>UseGalaxy.be, UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>377</td>
      <td>509</td>
      <td>2.733333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>metaQuantome 2: Function</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaquantome-function/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Metatranscriptomics, Microbial ecology, Metagenomics</td>
      <td>Visualisation, Indexing, Differential protein expression analysis, Functional clustering, Statistical inference, Principal component visualisation, Query and retrieval, Heat map generation, Quantification, Filtering</td>
      <td>2020-10-29</td>
      <td>2024-03-15</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>metaquantome_filter, metaquantome_sample, metaquantome_stat, metaquantome_expand, metaquantome_viz, metaquantome_db</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>352</td>
      <td>508</td>
      <td>3.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>metaQuantome 1: Data creation</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaquantome-data-creation/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Biodiversity, Taxonomy</td>
      <td>Prediction and recognition, Visualisation, Filtering, Label-free quantification, Formatting</td>
      <td>2020-10-16</td>
      <td>2024-03-14</td>
      <td>20</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>query_tabular, flashlfq, Filter1, peptide_shaker, regex1, tp_replace_in_line, msconvert, search_gui, unipept, Grep1, Cut1, Remove beginning1, tp_replace_in_column</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>781</td>
      <td>1353</td>
      <td>3.816667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Sequence analysis</td>
      <td>Quality and contamination control in bacterial isolate using Illumina MiSeq Data</td>
      <td>https://training.galaxyproject.org//topics/sequence-analysis/tutorials/quality-contamination-control/tutorial.html</td>
      <td>Whole genome sequencing, Genomics, Microbiology, Microbial ecology</td>
      <td>Cross-assembly, Sequencing quality control, Sequence contamination filtering, Statistical calculation, Taxonomic classification, Expression analysis, Sequence composition calculation</td>
      <td>2024-07-15</td>
      <td>2024-09-02</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>fastp, est_abundance, recentrifuge, upload1, kraken2, fastqc</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td>Galaxy@AuBi, GalaxyTrakr, UseGalaxy.cz, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>185</td>
      <td>225</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Calling variants in non-diploid systems</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/non-dip/tutorial.html</td>
      <td>Genomics, Sequence assembly, DNA polymorphism, Microbiology, Sequence analysis, Genetic variation</td>
      <td>Data handling, Sequence alignment, Read mapping, Sequencing quality control, Sequence alignment analysis, Generation, Statistical calculation, Variant calling, Sequence composition calculation, Formatting, Genome indexing</td>
      <td>2017-02-16</td>
      <td>2024-03-15</td>
      <td>23</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>bwa_mem, picard_MarkDuplicates, vcffilter2, bamFilter, picard_MergeSamFiles, Cut1, bamleftalign, fastqc, vcf2tsv, freebayes</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>7149</td>
      <td>13577</td>
      <td>2.050000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>M. tuberculosis Variant Analysis</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/tb-variant-analysis/tutorial.html</td>
      <td>Genomics, Microbiology, Sequence assembly, Genetic variation, Public health and epidemiology, Infectious disease</td>
      <td>Sequence alignment, Phylogenetic tree generation, Phylogenetic tree visualisation, Sequencing quality control, Sequence alignment analysis, Genome visualisation, Sequence analysis, Sequence contamination filtering, Variant calling, Global alignment, Statistical calculation, Taxonomic classification, Validation, Local alignment, Sequence composition calculation, Antimicrobial resistance prediction</td>
      <td>2020-07-25</td>
      <td>2024-07-26</td>
      <td>30</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tp_sed_tool, upload1, snippy, qualimap_bamqc, tb_variant_filter, EMBOSS: seqret84, tb_profiler_profile, mosdepth, jvarkit_wgscoverageplotter, tbvcfreport, tp_awk_tool, fastqc, jbrowse, samtools_stats, bcftools_consensus, multiqc, fastp, kraken2, EMBOSS:%20seqret84, __FLATTEN__</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>5788</td>
      <td>9172</td>
      <td>2.950000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Microbial Variant Calling</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/microbial-variants/tutorial.html</td>
      <td>Genomics, Sequence assembly, DNA polymorphism, Microbiology, Sequence analysis, Genetic variation</td>
      <td>Genome visualisation, Variant calling, Phylogenetic tree visualisation, Phylogenetic tree generation</td>
      <td>2018-02-26</td>
      <td>2024-03-15</td>
      <td>23</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>jbrowse, snippy</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>GalaxyTrakr, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>12476</td>
      <td>20693</td>
      <td>1.983333</td>
      <td>0</td>
    </tr>
  </tbody>
</table>

</div>