<script src="https://code.jquery.com/jquery-3.7.0.js"></script>
<script src="https://cdn.datatables.net/1.13.6/js/jquery.dataTables.min.js"></script>
<script src="https://cdn.datatables.net/searchbuilder/1.6.0/js/dataTables.searchBuilder.min.js"></script>
<script src="https://cdn.datatables.net/datetime/1.5.1/js/dataTables.dateTime.min.js"></script>

<script src="https://cdn.datatables.net/responsive/2.5.0/js/dataTables.responsive.min.js"></script>

<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/responsive/2.2.3/css/responsive.dataTables.min.css">

<link rel="stylesheet" href="https://cdn.datatables.net/1.13.6/css/jquery.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/searchbuilder/1.6.0/css/searchBuilder.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/datetime/1.5.1/css/dataTables.dateTime.min.css">

<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" />
<script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>

<style type="text/css">
  .tooltip {
    font-family: Georgia;
    font-size: 18px;
  }
  .tooltip .tooltip-inner {
    background-color: #00FFFF;
    color: #000000;
    min-width: 400px;
  }
  </style>

<script>
    $(document).ready(function() {
    $('#dataframe').DataTable( {
        "sScrollX": "100%",
        "dom": "Qlfrtip",

        responsive: {
            details: {
                type: 'column'
            }
        },
        "bAutoWidth": false,
        columnDefs: [ {
            className: 'control',
            orderable: false,
            targets:   [ 0 ],
            width: 100,
        },
        { type: 'natural', targets: 1 } // allow normal sorting
        ],

    });
});
</script>

<div class="display nowrap" style="width:80%" width="100%">

<table class="dataframe display nowrap" id="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th>Expand</th>
      <th>Name</th>
      <th>Source</th>
      <th>ID</th>
      <th>Link</th>
      <th>Creators</th>
      <th>Tags</th>
      <th>Creation time</th>
      <th>Update time</th>
      <th>Latest version</th>
      <th>Versions</th>
      <th>Number of steps</th>
      <th>Tools</th>
      <th>EDAM operations</th>
      <th>EDAM topics</th>
      <th>License</th>
      <th>DOI</th>
      <th>Projects</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td></td>
      <td>Taxonomy classification using Kraken2 and Bracken</td>
      <td>WorkflowHub</td>
      <td>1199</td>
      <td>https://workflowhub.eu/workflows/1199?version=2</td>
      <td>Valentine Murigneux, Mike Thang</td>
      <td>gucfg2galaxy, metagenomics, name:collection, shotgun</td>
      <td>2024-12-10</td>
      <td>2024-12-10</td>
      <td>2</td>
      <td>2</td>
      <td>29.0</td>
      <td>Prepare alpha diversity summary file (paste sample identifiers and Simpson results)\nPaste1, kraken2, krakentools_beta_diversity, collection_element_identifiers, regex1, add_line_to_file, Prepare alpha diversity summary file (paste Simpson and Fisher results)\nPaste1, Prepare alpha diversity summary file\nPaste1, est_abundance, cat_multiple, Fisher results contains a header line we want to exclude "Fisher's alpha...loading"\nShow tail1, taxonomy_krona_chart, collection_column_join, krakentools_alpha_diversity, __RELABEL_FROM_FILE__, Extract column name and fraction_total_reads from Bracken report\nCut1, krakentools_kreport2krona, krakentools_combine_kreports</td>
      <td>Aggregation, Visualisation, Statistical calculation, Taxonomic classification</td>
      <td>Metagenomics, Taxonomy</td>
      <td>CC-BY-4.0</td>
      <td>10.48546/workflowhub.workflow.1199.2</td>
      <td>QCIF Bioinformatics</td>
    </tr>
    <tr>
      <td></td>
      <td>AMR-Pathfinder</td>
      <td>WorkflowHub</td>
      <td>1189</td>
      <td>https://workflowhub.eu/workflows/1189?version=2</td>
      <td>Helena Rasche, Dennis Dollée, Birgit Rijvers</td>
      <td>amr, amr-detection, benchamrking</td>
      <td>2024-11-14</td>
      <td>2024-11-14</td>
      <td>2</td>
      <td>2</td>
      <td>36.0</td>
      <td>, staramr_search, hamronize_tool, abricate, tp_find_and_replace, __MERGE_COLLECTION__, datamash_ops, tp_text_file_with_recurring_lines, hamronize_summarize, tp_split_on_column, collapse_dataset, cat_multi_datasets, addValue, __APPLY_RULES__, cat1, Cut1, cast, shovill, Grep1</td>
      <td>Data handling, Antimicrobial resistance prediction, Genome assembly, Parsing</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>Seq4AMR, ErasmusMC Clinical Bioinformatics</td>
    </tr>
    <tr>
      <td></td>
      <td>bacterial-genome-assembly/main</td>
      <td>WorkflowHub</td>
      <td>1043</td>
      <td>https://workflowhub.eu/workflows/1043?version=7</td>
      <td>Abromics None, Pierre Marin, Clea Siguret</td>
      <td>abromics, assembly, fastq, genomics, bacterial-genomics, paired-end, quality</td>
      <td>2024-11-26</td>
      <td>2024-11-26</td>
      <td>7</td>
      <td>7</td>
      <td>7.0</td>
      <td>refseq_masher_matches, tooldistillator_summarize, bandage_image, quast, shovill, tooldistillator, bandage_info</td>
      <td>Data handling, Sequence assembly validation, Parsing, Visualisation, Genome assembly, Sequence assembly visualisation</td>
      <td></td>
      <td>GPL-3.0-or-later</td>
      <td></td>
      <td>Intergalactic Workflow Commission (IWC)</td>
    </tr>
    <tr>
      <td></td>
      <td>quality-and-contamination-control/main</td>
      <td>WorkflowHub</td>
      <td>1052</td>
      <td>https://workflowhub.eu/workflows/1052?version=8</td>
      <td>ABRomics None, Pierre Marin, Clea Siguret</td>
      <td>abromics, fastq, genomics, bacterial-genomics, paired-end, quality, taxonomy-assignment, trimming</td>
      <td>2024-11-23</td>
      <td>2024-11-23</td>
      <td>8</td>
      <td>8</td>
      <td>6.0</td>
      <td>tooldistillator_summarize, kraken2, est_abundance, fastp, tooldistillator, recentrifuge</td>
      <td>Data handling, Expression analysis, Sequencing quality control, Statistical calculation, Sequence contamination filtering, Taxonomic classification, Cross-assembly, Parsing</td>
      <td></td>
      <td>GPL-3.0-or-later</td>
      <td></td>
      <td>Intergalactic Workflow Commission (IWC)</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow 4: Staramr</td>
      <td>WorkflowHub</td>
      <td>470</td>
      <td>https://workflowhub.eu/workflows/470?version=1</td>
      <td></td>
      <td>10.3390/microorganisms10020292, amr, amr-detection, bioinformatics, antimicrobial resistance</td>
      <td>2023-05-11</td>
      <td>2024-09-09</td>
      <td>1</td>
      <td>1</td>
      <td>10.0</td>
      <td>hamronize_tool, abricate, collapse_dataset, tp_find_and_replace, shovill, hamronize_summarize, staramr_search</td>
      <td>Data handling, Antimicrobial resistance prediction, Genome assembly, Parsing</td>
      <td></td>
      <td>GPL-3.0</td>
      <td></td>
      <td>Seq4AMR</td>
    </tr>
    <tr>
      <td></td>
      <td>bacterial_genome_annotation/main</td>
      <td>WorkflowHub</td>
      <td>1050</td>
      <td>https://workflowhub.eu/workflows/1050?version=9</td>
      <td>ABRomics None, Pierre Marin, Clea Siguret</td>
      <td>abromics, annotation, genomics, bacterial-genomics, fasta, genome-annotation</td>
      <td>2024-10-26</td>
      <td>2024-10-26</td>
      <td>9</td>
      <td>9</td>
      <td>6.0</td>
      <td>tooldistillator_summarize, bakta, integron_finder, plasmidfinder, isescan, tooldistillator</td>
      <td>Protein feature detection, Data handling, Nucleic acid feature detection, Multilocus sequence typing, Genome annotation, Sequence motif recognition, Scaffolding, Structural variation detection, Genome assembly, Parsing</td>
      <td></td>
      <td>GPL-3.0-or-later</td>
      <td></td>
      <td>Intergalactic Workflow Commission (IWC)</td>
    </tr>
    <tr>
      <td></td>
      <td>amr_gene_detection/main</td>
      <td>WorkflowHub</td>
      <td>1049</td>
      <td>https://workflowhub.eu/workflows/1049?version=7</td>
      <td>ABRomics None, Pierre Marin, Clea Siguret</td>
      <td>abromics, amr, amr-detection, genomics, antibiotic-resistance, antimicrobial resistance, antimicrobial-resistance-genes, bacterial-genomics, fasta</td>
      <td>2024-10-26</td>
      <td>2024-10-27</td>
      <td>7</td>
      <td>7</td>
      <td>5.0</td>
      <td>abricate, tooldistillator_summarize, staramr_search, amrfinderplus, tooldistillator</td>
      <td>Data handling, Antimicrobial resistance prediction, Parsing</td>
      <td></td>
      <td>GPL-3.0-or-later</td>
      <td></td>
      <td>Intergalactic Workflow Commission (IWC)</td>
    </tr>
    <tr>
      <td></td>
      <td>Analyses of shotgun metagenomics data with MetaPhlAn2</td>
      <td>WorkflowHub</td>
      <td>624</td>
      <td>https://workflowhub.eu/workflows/624?version=1</td>
      <td>Valentine Murigneux, Mike Thang, Saskia Hiltemann, Bérénice Batut</td>
      <td>gucfg2galaxy, metagenomics, shotgun</td>
      <td>2023-10-26</td>
      <td>2024-04-05</td>
      <td>1</td>
      <td>1</td>
      <td>17.0</td>
      <td>, metaphlan2krona, metaphlan2, Cut1, taxonomy_krona_chart, humann2_renorm_table, humann2, humann2_regroup_table, merge_metaphlan_tables</td>
      <td>Phylogenetic tree analysis, Nucleic acid sequence analysis, Visualisation</td>
      <td>Metagenomic sequencing, Metagenomics</td>
      <td>CC-BY-4.0</td>
      <td>10.48546/workflowhub.workflow.624.1</td>
      <td>QCIF Bioinformatics</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow 7 : Beta Diversity [16S Microbial Analysis With Mothur]</td>
      <td>WorkflowHub</td>
      <td>653</td>
      <td>https://workflowhub.eu/workflows/653?version=1</td>
      <td>Saskia Hiltemann, Bérénice Batut, Dave Clements, Ahmed Mehdi</td>
      <td>metagenomics</td>
      <td>2023-11-09</td>
      <td>2023-11-09</td>
      <td>1</td>
      <td>1</td>
      <td>4.0</td>
      <td>mothur_tree_shared, mothur_dist_shared, mothur_heatmap_sim, newick_display</td>
      <td>Taxonomic classification, Phylogenetic tree analysis, Sequencing quality control, Phylogenetic analysis, Phylogenetic tree generation, Visualisation, Phylogenetic tree reconstruction, Sequence clustering, Sequence read processing, DNA barcoding</td>
      <td></td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>QCIF Bioinformatics</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow 6: Alpha Diversity [16S Microbial Analysis With Mothur]</td>
      <td>WorkflowHub</td>
      <td>652</td>
      <td>https://workflowhub.eu/workflows/652?version=1</td>
      <td>Saskia Hiltemann, Bérénice Batut, Dave Clements, Ahmed Mehdi</td>
      <td>metagenomics</td>
      <td>2023-11-09</td>
      <td>2023-11-09</td>
      <td>1</td>
      <td>1</td>
      <td>3.0</td>
      <td>mothur_rarefaction_single, XY_Plot_1, mothur_summary_single</td>
      <td>Taxonomic classification, Sequencing quality control, Phylogenetic analysis, Visualisation, Sequence clustering, Sequence read processing, DNA barcoding</td>
      <td></td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>QCIF Bioinformatics</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow 5: OTU Clustering [16S Microbial Analysis With Mothur]</td>
      <td>WorkflowHub</td>
      <td>651</td>
      <td>https://workflowhub.eu/workflows/651?version=1</td>
      <td>Saskia Hiltemann, Bérénice Batut, Dave Clements, Ahmed Mehdi</td>
      <td>metagenomics</td>
      <td>2023-11-09</td>
      <td>2023-11-09</td>
      <td>1</td>
      <td>1</td>
      <td>5.0</td>
      <td>mothur_make_shared, mothur_cluster_split, mothur_sub_sample, mothur_classify_otu, mothur_count_groups</td>
      <td>Taxonomic classification, Sequencing quality control, Phylogenetic analysis, Visualisation, Sequence clustering, Sequence read processing, DNA barcoding</td>
      <td></td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>QCIF Bioinformatics</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow 3: Classification [Galaxy Training: 16S Microbial Analysis With Mothur]</td>
      <td>WorkflowHub</td>
      <td>650</td>
      <td>https://workflowhub.eu/workflows/650?version=1</td>
      <td>Saskia Hiltemann, Bérénice Batut, Dave Clements, Ahmed Mehdi</td>
      <td>metagenomics</td>
      <td>2023-11-09</td>
      <td>2023-11-09</td>
      <td>1</td>
      <td>1</td>
      <td>2.0</td>
      <td>mothur_remove_lineage, mothur_classify_seqs</td>
      <td>Taxonomic classification, Sequencing quality control, Phylogenetic analysis, Visualisation, Sequence clustering, Sequence read processing, DNA barcoding</td>
      <td></td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>QCIF Bioinformatics</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow 1: Further Quality Control [16S Microbial Analysis With Mothur]</td>
      <td>WorkflowHub</td>
      <td>648</td>
      <td>https://workflowhub.eu/workflows/648?version=1</td>
      <td>Saskia  Hiltemann, Bérénice Batut, Dave Clements, Ahmed Mehdi</td>
      <td>metagenomics</td>
      <td>2023-11-09</td>
      <td>2023-11-09</td>
      <td>1</td>
      <td>1</td>
      <td>5.0</td>
      <td>mothur_summary_seqs, mothur_count_seqs, mothur_unique_seqs, mothur_screen_seqs</td>
      <td>Taxonomic classification, Sequencing quality control, Phylogenetic analysis, Visualisation, Sequence clustering, Sequence read processing, DNA barcoding</td>
      <td></td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>QCIF Bioinformatics</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow 2: Data Cleaning And Chimera Removal [16S Microbial Analysis With Mothur]</td>
      <td>WorkflowHub</td>
      <td>649</td>
      <td>https://workflowhub.eu/workflows/649?version=1</td>
      <td>Saskia Hiltemann, Bérénice Batut, Dave Clements, Ahmed Mehdi</td>
      <td>metagenomics</td>
      <td>2023-11-09</td>
      <td>2023-11-09</td>
      <td>1</td>
      <td>1</td>
      <td>9.0</td>
      <td>mothur_pre_cluster, mothur_screen_seqs, mothur_remove_seqs, mothur_unique_seqs, mothur_filter_seqs, mothur_chimera_vsearch, mothur_summary_seqs</td>
      <td>Taxonomic classification, Sequencing quality control, Phylogenetic analysis, Visualisation, Sequence clustering, Sequence read processing, DNA barcoding</td>
      <td></td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>QCIF Bioinformatics</td>
    </tr>
    <tr>
      <td></td>
      <td>pox-virus-amplicon/main</td>
      <td>WorkflowHub</td>
      <td>439</td>
      <td>https://workflowhub.eu/workflows/439?version=2</td>
      <td>Viktoria Isabel Schwarz, Wolfgang Maier</td>
      <td>virology, pox</td>
      <td>2024-06-22</td>
      <td>2024-06-22</td>
      <td>2</td>
      <td>2</td>
      <td>40.0</td>
      <td>\n Cut1, samtools_view, fastp, \n __SORTLIST__, samtools_merge, ivar_consensus, split_file_to_collection, \n __ZIP_COLLECTION__, \n __APPLY_RULES__, \n Grep1, tp_sed_tool, EMBOSS: maskseq51, datamash_ops, samtools_stats, \n param_value_from_file, \n __FILTER_FAILED_DATASETS__, compose_text_param, qualimap_bamqc, fasta_compute_length, ivar_trim, bwa_mem, \n __FLATTEN__, collection_element_identifiers, multiqc, tp_cat</td>
      <td>Sequence alignment analysis, Read mapping, Sequencing quality control, Local alignment, Sequence alignment, Genome indexing, Sequence contamination filtering, Global alignment, Sequence analysis, Generation, Validation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>Intergalactic Workflow Commission (IWC)</td>
    </tr>
    <tr>
      <td></td>
      <td>3: Plant virus exploration</td>
      <td>WorkflowHub</td>
      <td>103</td>
      <td>https://workflowhub.eu/workflows/103?version=1</td>
      <td></td>
      <td>de_novo, virology, exploration</td>
      <td>2021-02-04</td>
      <td>2023-02-13</td>
      <td>1</td>
      <td>1</td>
      <td>2.0</td>
      <td>fastp, shovill</td>
      <td>Sequencing quality control, Sequence contamination filtering, Genome assembly</td>
      <td></td>
      <td>Apache-2.0</td>
      <td></td>
      <td>Integrated and Urban Plant Pathology Laboratory</td>
    </tr>
    <tr>
      <td></td>
      <td>2: Plant virus confirmation</td>
      <td>WorkflowHub</td>
      <td>102</td>
      <td>https://workflowhub.eu/workflows/102?version=1</td>
      <td></td>
      <td>assembly, virology, blast, mapping, reads_selection</td>
      <td>2021-02-04</td>
      <td>2023-02-13</td>
      <td>1</td>
      <td>1</td>
      <td>7.0</td>
      <td>fasta_merge_files_and_filter_unique_sequences, minimap2, samtools_stats, ngsutils_bam_filter, shovill, picard_SamToFastq, ncbi_blastn_wrapper</td>
      <td>Variant calling, Sequence contamination filtering, Read pre-processing, Genome assembly, Sequencing quality control, Pairwise sequence alignment, Formatting</td>
      <td></td>
      <td>Apache-2.0</td>
      <td></td>
      <td>Integrated and Urban Plant Pathology Laboratory</td>
    </tr>
    <tr>
      <td></td>
      <td>GTN Training - Antibiotic Resistance Detection</td>
      <td>WorkflowHub</td>
      <td>406</td>
      <td>https://workflowhub.eu/workflows/406?version=1</td>
      <td>Saskia Hiltemann, Willem de Koning</td>
      <td>metagenomics</td>
      <td>2022-11-24</td>
      <td>2023-02-13</td>
      <td>1</td>
      <td>1</td>
      <td>12.0</td>
      <td>gfa_to_fa, bandage_image, nanoplot, staramr_search, minimap2, PlasFlow, miniasm, racon, unicycler</td>
      <td>Aggregation, Scatter plot plotting, De-novo assembly, Pairwise sequence alignment, Box-Whisker plot plotting, Sequence assembly visualisation, Sequence analysis, Genome assembly, Mapping assembly</td>
      <td>Microbiology</td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>Galaxy Training Network</td>
    </tr>
    <tr>
      <td></td>
      <td>1: Plant virus detection with kraken2 (SE)</td>
      <td>WorkflowHub</td>
      <td>124</td>
      <td>https://workflowhub.eu/workflows/124?version=1</td>
      <td></td>
      <td>virology, kraken</td>
      <td>2021-06-17</td>
      <td>2023-02-13</td>
      <td>1</td>
      <td>1</td>
      <td>3.0</td>
      <td>taxonomy_krona_chart, Kraken2Tax, kraken2</td>
      <td>Visualisation, Taxonomic classification</td>
      <td></td>
      <td>Apache-2.0</td>
      <td></td>
      <td>Integrated and Urban Plant Pathology Laboratory</td>
    </tr>
    <tr>
      <td></td>
      <td>1: Plant virus detection with kraken2 (PE)</td>
      <td>WorkflowHub</td>
      <td>101</td>
      <td>https://workflowhub.eu/workflows/101?version=1</td>
      <td></td>
      <td>virology, kraken</td>
      <td>2021-02-04</td>
      <td>2023-02-13</td>
      <td>1</td>
      <td>1</td>
      <td>3.0</td>
      <td>taxonomy_krona_chart, Kraken2Tax, kraken2</td>
      <td>Visualisation, Taxonomic classification</td>
      <td></td>
      <td>Apache-2.0</td>
      <td></td>
      <td>Integrated and Urban Plant Pathology Laboratory</td>
    </tr>
    <tr>
      <td></td>
      <td>Halophiles workup of Comparative gene analysis</td>
      <td>https://usegalaxy.eu</td>
      <td>a2c46deea34d9d80</td>
      <td>https://usegalaxy.eu/published/workflow?id=a2c46deea34d9d80</td>
      <td>Anton Nekrutenko</td>
      <td>genome-annotation, name:microgalaxy</td>
      <td>2024-12-03</td>
      <td>2024-12-05</td>
      <td>2</td>
      <td>2</td>
      <td></td>
      <td>cat1, tp_split_on_column, Add_a_column1, regexColumn1, collapse_dataset, join1, Cut1, tab2fasta, rapidnj, rbc_mafft, bg_diamond, bg_diamond_view, Filter1, gops_intersect_1</td>
      <td>Multiple sequence alignment, Sequence alignment analysis, Phylogenetic tree generation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>pAllori Blood RNA</td>
      <td>https://usegalaxy.eu</td>
      <td>ca9d3233b0912765</td>
      <td>https://usegalaxy.eu/published/workflow?id=ca9d3233b0912765</td>
      <td>Engy Nasr, Paul Zierep</td>
      <td>name:microgalaxy, name:pallori, name:bloodrna, name:umi, name:barcode, name:transcriptomics</td>
      <td>2023-07-02</td>
      <td>2024-11-24</td>
      <td>43</td>
      <td>43</td>
      <td></td>
      <td>fastqc, __FILTER_FAILED_DATASETS__, rna_star, featurecounts, multiqc, collection_column_join, umi_tools_extract</td>
      <td>Read summarisation, Statistical calculation, Sequence composition calculation, Sequencing quality control, Sequence alignment, Validation, RNA-Seq quantification</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Metaproteomics workflow</td>
      <td>https://usegalaxy.eu</td>
      <td>cefc49c13ff73231</td>
      <td>https://usegalaxy.eu/published/workflow?id=cefc49c13ff73231</td>
      <td>Timothy Griffin, Pratik Jagtap, James Johnson, Clemens Blank, Subina Mehta</td>
      <td>name:metaproteomics, name:galaxyp, name:massspectrometry, name:microgalaxy</td>
      <td>2024-11-21</td>
      <td>2024-11-21</td>
      <td>40</td>
      <td>40</td>
      <td></td>
      <td>search_gui, peptide_shaker, unipept, query_tabular, sqlite_to_tabular</td>
      <td>Visualisation, Prediction and recognition</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Feature-Count Table Normalization</td>
      <td>https://usegalaxy.eu</td>
      <td>6239178d7cc4ac68</td>
      <td>https://usegalaxy.eu/published/workflow?id=6239178d7cc4ac68</td>
      <td>Johannes Effenberger</td>
      <td>name:microgalaxy, normalization, softmax, relative-abundance, sigmoid, css, limma-tmm, limma-rle</td>
      <td>2024-11-21</td>
      <td>2024-11-21</td>
      <td>5</td>
      <td>5</td>
      <td></td>
      <td>__EXTRACT_DATASET__, __FILTER_EMPTY_DATASETS__, map_param_value, __BUILD_LIST__, Grep1</td>
      <td></td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>metaQuantome_datacreation_workflow</td>
      <td>https://usegalaxy.eu</td>
      <td>cd675ab32d3e7833</td>
      <td>https://usegalaxy.eu/published/workflow?id=cd675ab32d3e7833</td>
      <td>Subina Metha, Timothy J. Griffin, Pratik Jagtap, Emma Leith, Marie Crane</td>
      <td>proteomics, name:microgalaxy</td>
      <td>2024-11-21</td>
      <td>2024-11-21</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>msconvert, search_gui, peptide_shaker, Cut1, regex1, unipept, tp_replace_in_line, Remove beginning1, query_tabular, flashlfq, Filter1, tp_replace_in_column</td>
      <td>Filtering, Label-free quantification, Visualisation, Prediction and recognition, Formatting</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>metaquantome-function-worklow</td>
      <td>https://usegalaxy.eu</td>
      <td>e5a89ef7b5f1c1d9</td>
      <td>https://usegalaxy.eu/published/workflow?id=e5a89ef7b5f1c1d9</td>
      <td>Subina Mehta, Timothy J. Griffin, Pratik Jagtap, Emma Leith, Marie Crane, Praveen Kumar</td>
      <td>proteomics, name:microgalaxy</td>
      <td>2024-11-21</td>
      <td>2024-11-21</td>
      <td>3</td>
      <td>3</td>
      <td></td>
      <td>metaquantome_viz, metaquantome_stat, metaquantome_expand, metaquantome_sample, metaquantome_db, metaquantome_filter</td>
      <td>Heat map generation, Statistical inference, Principal component visualisation, Visualisation, Query and retrieval, Differential protein expression analysis, Filtering, Quantification, Indexing, Functional clustering</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>pAllori Amplicon 16S Pre-Processing and Taxonomy Classification</td>
      <td>https://usegalaxy.eu</td>
      <td>466bdd8ba7b67264</td>
      <td>https://usegalaxy.eu/published/workflow?id=466bdd8ba7b67264</td>
      <td>Engy Nasr, Paul Zierep</td>
      <td>name:microgalaxy, name:16s, name:metagenomics, name:pallori</td>
      <td>2023-06-30</td>
      <td>2024-11-21</td>
      <td>70</td>
      <td>70</td>
      <td></td>
      <td>ampvis2_load, trimmomatic, kraken2, tp_cut_tool, ampvis2_ordinate, unzip, taxonomy_krona_chart, collection_column_join, multiqc, lotus2, fastqc, __FLATTEN__, krakentools_kreport2krona, cutadapt, ampvis2_heatmap</td>
      <td>Aggregation, Validation, Sequence composition calculation, Sequencing quality control, Visualisation, Analysis, Statistical calculation, Primer removal, Taxonomic classification, Read pre-processing, Sequence trimming, Sequence feature detection, DNA barcoding</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Cloud Aerosol MT-MG Contamination Filtering</td>
      <td>https://usegalaxy.eu</td>
      <td>c09159d7aad0f264</td>
      <td>https://usegalaxy.eu/published/workflow?id=c09159d7aad0f264</td>
      <td>Engy Nasr , Bérénice Batut, Raphaëlle Péguilhan, Pierre Amato</td>
      <td>metagenomics, metatranscriptomics, name:microgalaxy</td>
      <td>2024-11-21</td>
      <td>2024-11-21</td>
      <td>2</td>
      <td>2</td>
      <td></td>
      <td>seq_filter_by_id, kraken2, fastq_to_tabular, filter_tabular, bowtie2, recentrifuge</td>
      <td>Taxonomic classification, Read mapping, Expression analysis, Cross-assembly</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Metatranscriptomics data analysis</td>
      <td>https://usegalaxy.eu</td>
      <td>fd90652d475ed739</td>
      <td>https://usegalaxy.eu/published/workflow?id=fd90652d475ed739</td>
      <td>Valerie C. Schiml, Magnus Ø. Arntzen, Francesco Delogu, Praveen Kumar, Benoit Kunath, Bérénice Batut,  Subina Mehta, James E. Johnson,  Björn Grüning, Phillip B. Pope, Pratik D. Jagtap, Timothy J. Griffin</td>
      <td>name:microgalaxy, metatranscriptomics, microbiome</td>
      <td>2024-11-21</td>
      <td>2024-11-21</td>
      <td>5</td>
      <td>5</td>
      <td></td>
      <td>bg_sortmerna, trim_galore, __UNZIP_COLLECTION__, kallisto_quant, collection_column_join, fastqc</td>
      <td>Sequence alignment analysis, Sequence similarity search, Sequence composition calculation, Gene expression profiling, Sequencing quality control, Sequence comparison, Primer removal, Read pre-processing, Sequence trimming, Statistical calculation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Cloud-Aerosole MT-MG Functional Profiling</td>
      <td>https://usegalaxy.eu</td>
      <td>63478edcea3f449a</td>
      <td>https://usegalaxy.eu/published/workflow?id=63478edcea3f449a</td>
      <td>Engy Nasr, Bérénice Batut, Raphaëlle Péguilhan, Pierre Amato</td>
      <td>metagenomics, metatranscriptomics, name:microgalaxy</td>
      <td>2024-11-21</td>
      <td>2024-11-21</td>
      <td>4</td>
      <td>4</td>
      <td></td>
      <td>multiqc, bowtie2, featurecounts</td>
      <td>Read summarisation, Read mapping, Sequencing quality control, Validation, RNA-Seq quantification</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Cloud-Aerosole MT-MG Pre-Processing</td>
      <td>https://usegalaxy.eu</td>
      <td>1ef76b7b86e15792</td>
      <td>https://usegalaxy.eu/published/workflow?id=1ef76b7b86e15792</td>
      <td>Engy Nasr, Bérénice Batut, Raphaëlle Péguilhan, Pierre Amato</td>
      <td>metagenomics, metatranscriptomics, name:microgalaxy</td>
      <td>2024-11-21</td>
      <td>2024-11-21</td>
      <td>6</td>
      <td>6</td>
      <td></td>
      <td>fastqc, multiqc, trimmomatic</td>
      <td>Statistical calculation, Sequence composition calculation, Sequencing quality control, Validation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Allele-based Pathogen Identification (release v0.1.2)</td>
      <td>https://usegalaxy.eu</td>
      <td>64abcddeb516a712</td>
      <td>https://usegalaxy.eu/published/workflow?id=64abcddeb516a712</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-10-31</td>
      <td>2024-10-31</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>bcftools_norm, samtools_depth, table_compute, collapse_dataset, tp_cut_tool, Cut1, clair3, snpSift_extractFields, bcftools_consensus, Count1, regexColumn1, minimap2, Paste1, Remove beginning1, tp_head_tool, samtools_coverage, CONVERTER_gz_to_uncompressed, snpSift_filter</td>
      <td>Pairwise sequence alignment, Variant calling</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Taxonomy Profiling and Visualization with Kron (release v0.1)</td>
      <td>https://usegalaxy.eu</td>
      <td>ad8b6d73c9654305</td>
      <td>https://usegalaxy.eu/published/workflow?id=ad8b6d73c9654305</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-09-19</td>
      <td>2024-09-19</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>taxonomy_krona_chart, krakentools_kreport2krona, kraken2</td>
      <td>Aggregation, Visualisation, Taxonomic classification</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Pathogen Detection PathoGFAIR Samples Aggregation and Visualisation (release v0.1)</td>
      <td>https://usegalaxy.eu</td>
      <td>f6a763951d815944</td>
      <td>https://usegalaxy.eu/published/workflow?id=f6a763951d815944</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-09-19</td>
      <td>2024-09-19</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>tp_multijoin_tool, bedtools_getfastabed, ggplot2_heatmap, regex1, Grouping1, clustalw, tp_split_on_column, fasttree, __FILTER_FAILED_DATASETS__, collapse_dataset, tab2fasta, newick_display, tp_replace_in_column, fasta2tab, fasta_merge_files_and_filter_unique_sequences, regexColumn1, Count1, Cut1, __FILTER_EMPTY_DATASETS__, tp_sorted_uniq, Remove beginning1, collection_column_join</td>
      <td>Phylogenetic tree generation (maximum likelihood and Bayesian methods), Visualisation, Phylogenetic tree generation (from molecular sequences), Phylogenetic tree reconstruction, Mapping, Multiple sequence alignment, Phylogenetic tree analysis, Phylogenetic tree generation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Nanopore Preprocessing (release v0.1)</td>
      <td>https://usegalaxy.eu</td>
      <td>344cd3a3c2ce5302</td>
      <td>https://usegalaxy.eu/published/workflow?id=344cd3a3c2ce5302</td>
      <td>Bérénice Batut, Engy Nasr, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:nanopore, name:iwc</td>
      <td>2024-09-19</td>
      <td>2024-09-19</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>Add_a_column1, porechop, collapse_dataset, __FILTER_FAILED_DATASETS__, samtools_fastx, kraken2, Cut1, nanoplot, krakentools_extract_kraken_reads, regexColumn1, minimap2, fastp, multiqc, collection_column_join, fastqc, Grep1, bamtools_split_mapped</td>
      <td>Aggregation, Data handling, Scatter plot plotting, Sequence alignment analysis, Sequence composition calculation, Sequencing quality control, Pairwise sequence alignment, Box-Whisker plot plotting, Visualisation, Sequence contamination filtering, Statistical calculation, Taxonomic classification, Validation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Gene-based Pathogen Identification (release v0.1)</td>
      <td>https://usegalaxy.eu</td>
      <td>77e5bbd317750915</td>
      <td>https://usegalaxy.eu/published/workflow?id=77e5bbd317750915</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:pathogfair, name:iwc, name:microgalaxy</td>
      <td>2024-09-19</td>
      <td>2024-09-19</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>fasta2tab, abricate, flye, collection_element_identifiers, tp_find_and_replace, bandage_image, param_value_from_file, tab2fasta, medaka_consensus_pipeline, compose_text_param, split_file_to_collection, __BUILD_LIST__</td>
      <td>Base-calling, De-novo assembly, Sequence assembly, Variant calling, Sequence assembly visualisation, Antimicrobial resistance prediction, Genome assembly, Mapping assembly, Cross-assembly</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Allele-based Pathogen Identification (release v0.1)</td>
      <td>https://usegalaxy.eu</td>
      <td>d215a2e554fd83bc</td>
      <td>https://usegalaxy.eu/published/workflow?id=d215a2e554fd83bc</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-09-19</td>
      <td>2024-09-19</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>bcftools_norm, samtools_depth, table_compute, collapse_dataset, tp_cut_tool, Cut1, clair3, snpSift_extractFields, bcftools_consensus, Count1, regexColumn1, minimap2, Paste1, Remove beginning1, tp_head_tool, samtools_coverage, CONVERTER_gz_to_uncompressed, snpSift_filter</td>
      <td>Pairwise sequence alignment, Variant calling</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Allele-based Pathogen Identification (release v0.1.1)</td>
      <td>https://usegalaxy.eu</td>
      <td>cdf2f02428f5f83f</td>
      <td>https://usegalaxy.eu/published/workflow?id=cdf2f02428f5f83f</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-09-19</td>
      <td>2024-09-19</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>bcftools_norm, samtools_depth, table_compute, collapse_dataset, tp_cut_tool, Cut1, clair3, snpSift_extractFields, bcftools_consensus, Count1, regexColumn1, minimap2, Paste1, Remove beginning1, tp_head_tool, samtools_coverage, CONVERTER_gz_to_uncompressed, snpSift_filter</td>
      <td>Pairwise sequence alignment, Variant calling</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>pAllori WGS</td>
      <td>https://usegalaxy.eu</td>
      <td>4f245ac304ab76d9</td>
      <td>https://usegalaxy.eu/published/workflow?id=4f245ac304ab76d9</td>
      <td>Engy Nasr, Paul Zierep</td>
      <td>name:pallori, name:microgalaxy, name:wgs</td>
      <td>2024-07-16</td>
      <td>2024-07-28</td>
      <td>16</td>
      <td>16</td>
      <td></td>
      <td>bcftools_norm, __FLATTEN__, abricate, trimmomatic, regexColumn1, tp_awk_tool, ggplot2_heatmap, freebayes, Remove beginning1, multiqc, Grouping1, shovill, collection_column_join, fastqc, bowtie2, vcf2tsv, bcftools_view, snpSift_filter</td>
      <td>Sequence composition calculation, Read mapping, Sequencing quality control, Variant calling, Visualisation, Antimicrobial resistance prediction, Statistical calculation, Genome assembly, Validation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>MAGs workflow</td>
      <td>https://usegalaxy.eu</td>
      <td>97312d273b6e8bd9</td>
      <td>https://usegalaxy.eu/published/workflow?id=97312d273b6e8bd9</td>
      <td></td>
      <td>name:microgalaxy</td>
      <td>2023-11-15</td>
      <td>2024-07-11</td>
      <td>14</td>
      <td>14</td>
      <td></td>
      <td>concoct_extract_fasta_bins, maxbin2, Fasta_to_Contig2Bin, fastq_paired_end_interlacer, samtools_sort, __UNZIP_COLLECTION__, megahit, coverm_genome, concoct_merge_cut_up_clustering, drep_dereplicate, concoct, metabat2, semibin, quast, metabat2_jgi_summarize_bam_contig_depths, checkm_lineage_wf, das_tool, concoct_coverage_table, concoct_cut_up_fasta, bowtie2</td>
      <td>Sequence assembly, Genome comparison, Sequence composition calculation, Read mapping, Sequencing quality control, Genome assembly, Local alignment, Read binning, Sequence assembly validation, Genome annotation, Visualisation, Statistical calculation, Sequence clustering, Validation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>PathoGFAIR</td>
      <td>https://usegalaxy.eu</td>
      <td>0dce37adb369492c</td>
      <td>https://usegalaxy.eu/published/workflow?id=0dce37adb369492c</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:microgalaxy, name:iwc, name:pathogfair, name:nanopore, name:collection</td>
      <td>2024-05-31</td>
      <td>2024-07-03</td>
      <td>93</td>
      <td>93</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Pathogen Detection PathoGFAIR Samples Aggregation and Visualisation</td>
      <td>https://usegalaxy.eu</td>
      <td>376119528377a3ae</td>
      <td>https://usegalaxy.eu/published/workflow?id=376119528377a3ae</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-04-03</td>
      <td>2024-06-24</td>
      <td>58</td>
      <td>58</td>
      <td></td>
      <td>tp_multijoin_tool, bedtools_getfastabed, ggplot2_heatmap, regex1, Grouping1, clustalw, tp_split_on_column, fasttree, __FILTER_FAILED_DATASETS__, collapse_dataset, tab2fasta, newick_display, tp_replace_in_column, fasta2tab, fasta_merge_files_and_filter_unique_sequences, regexColumn1, Count1, Cut1, __FILTER_EMPTY_DATASETS__, tp_sorted_uniq, Remove beginning1, collection_column_join</td>
      <td>Phylogenetic tree generation (maximum likelihood and Bayesian methods), Visualisation, Phylogenetic tree generation (from molecular sequences), Phylogenetic tree reconstruction, Mapping, Multiple sequence alignment, Phylogenetic tree analysis, Phylogenetic tree generation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Allele-based Pathogen Identification</td>
      <td>https://usegalaxy.eu</td>
      <td>09c7069ae409c362</td>
      <td>https://usegalaxy.eu/published/workflow?id=09c7069ae409c362</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2023-04-12</td>
      <td>2024-06-24</td>
      <td>72</td>
      <td>72</td>
      <td></td>
      <td>bcftools_norm, samtools_depth, table_compute, collapse_dataset, tp_cut_tool, Cut1, clair3, snpSift_extractFields, bcftools_consensus, Count1, regexColumn1, minimap2, Paste1, Remove beginning1, tp_head_tool, samtools_coverage, CONVERTER_gz_to_uncompressed, snpSift_filter</td>
      <td>Pairwise sequence alignment, Variant calling</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Nanopore Preprocessing</td>
      <td>https://usegalaxy.eu</td>
      <td>a705370bc2c13d5c</td>
      <td>https://usegalaxy.eu/published/workflow?id=a705370bc2c13d5c</td>
      <td>Bérénice Batut, Engy Nasr, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:nanopore, name:iwc</td>
      <td>2023-04-12</td>
      <td>2024-06-24</td>
      <td>190</td>
      <td>190</td>
      <td></td>
      <td>Add_a_column1, porechop, collapse_dataset, __FILTER_FAILED_DATASETS__, samtools_fastx, kraken2, Cut1, nanoplot, krakentools_extract_kraken_reads, regexColumn1, minimap2, fastp, multiqc, collection_column_join, fastqc, Grep1, bamtools_split_mapped</td>
      <td>Aggregation, Data handling, Scatter plot plotting, Sequence alignment analysis, Sequence composition calculation, Sequencing quality control, Pairwise sequence alignment, Box-Whisker plot plotting, Visualisation, Sequence contamination filtering, Statistical calculation, Taxonomic classification, Validation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Taxonomy Profiling and Visualization with Phinch</td>
      <td>https://usegalaxy.eu</td>
      <td>b67b5ecd3305f830</td>
      <td>https://usegalaxy.eu/published/workflow?id=b67b5ecd3305f830</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:gtn</td>
      <td>2023-04-12</td>
      <td>2024-06-19</td>
      <td>8</td>
      <td>8</td>
      <td></td>
      <td>kraken_biom, kraken2, interactive_tool_phinch</td>
      <td>Taxonomic classification</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>metaQuantome_datacreation_workflow</td>
      <td>https://usegalaxy.eu</td>
      <td>fc42cc9467c6ee8d</td>
      <td>https://usegalaxy.eu/published/workflow?id=fc42cc9467c6ee8d</td>
      <td></td>
      <td>proteomics, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2024-06-19</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>msconvert, search_gui, peptide_shaker, Cut1, regex1, unipept, tp_replace_in_line, Remove beginning1, query_tabular, flashlfq, Filter1, tp_replace_in_column</td>
      <td>Filtering, Label-free quantification, Visualisation, Prediction and recognition, Formatting</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Gene-based Pathogen Identification</td>
      <td>https://usegalaxy.eu</td>
      <td>585c21b7b1d864fc</td>
      <td>https://usegalaxy.eu/published/workflow?id=585c21b7b1d864fc</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:pathogfair, name:iwc, name:microgalaxy</td>
      <td>2023-04-13</td>
      <td>2024-06-05</td>
      <td>142</td>
      <td>142</td>
      <td></td>
      <td>fasta2tab, abricate, flye, collection_element_identifiers, tp_find_and_replace, bandage_image, param_value_from_file, tab2fasta, medaka_consensus_pipeline, compose_text_param, split_file_to_collection, __BUILD_LIST__</td>
      <td>Base-calling, De-novo assembly, Sequence assembly, Variant calling, Sequence assembly visualisation, Antimicrobial resistance prediction, Genome assembly, Mapping assembly, Cross-assembly</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Taxonomy Profiling and Visualization with Krona</td>
      <td>https://usegalaxy.eu</td>
      <td>10101558b211a782</td>
      <td>https://usegalaxy.eu/published/workflow?id=10101558b211a782</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2023-05-11</td>
      <td>2024-06-05</td>
      <td>58</td>
      <td>58</td>
      <td></td>
      <td>taxonomy_krona_chart, krakentools_kreport2krona, kraken2</td>
      <td>Aggregation, Visualisation, Taxonomic classification</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Pathogen-Detection-Nanopore-All-Samples-Analysis-Juyptool</td>
      <td>https://usegalaxy.eu</td>
      <td>c8bc4af6dd3b5d02</td>
      <td>https://usegalaxy.eu/published/workflow?id=c8bc4af6dd3b5d02</td>
      <td>Engy Nasr, Bérénice Batut</td>
      <td>microbiome, nanopore, iwc, name:collection, name:microgalaxy</td>
      <td>2022-11-29</td>
      <td>2024-04-04</td>
      <td>205</td>
      <td>205</td>
      <td></td>
      <td>tp_multijoin_tool, bedtools_getfastabed, ggplot2_heatmap, regex1, Grouping1, clustalw, tp_split_on_column, fasttree, __FILTER_FAILED_DATASETS__, collapse_dataset, tab2fasta, newick_display, interactive_tool_jupyter_notebook, tp_replace_in_column, fasta2tab, fasta_merge_files_and_filter_unique_sequences, regexColumn1, Count1, Cut1, __FILTER_EMPTY_DATASETS__, tp_sorted_uniq, Remove beginning1, collection_column_join</td>
      <td>Phylogenetic tree generation (maximum likelihood and Bayesian methods), Visualisation, Phylogenetic tree generation (from molecular sequences), Phylogenetic tree reconstruction, Mapping, Multiple sequence alignment, Phylogenetic tree analysis, Phylogenetic tree generation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Mgnify ML</td>
      <td>https://usegalaxy.eu</td>
      <td>011f701c5a55bfdf</td>
      <td>https://usegalaxy.eu/published/workflow?id=011f701c5a55bfdf</td>
      <td></td>
      <td>name:microgalaxy</td>
      <td>2023-10-10</td>
      <td>2024-03-15</td>
      <td>128</td>
      <td>128</td>
      <td></td>
      <td>interactive_tool_jupyter_notebook, __BUILD_LIST__</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Normalization</td>
      <td>https://usegalaxy.eu</td>
      <td>0603f0898211f806</td>
      <td>https://usegalaxy.eu/published/workflow?id=0603f0898211f806</td>
      <td></td>
      <td>name:microgalaxy</td>
      <td>2023-09-25</td>
      <td>2024-03-13</td>
      <td>80</td>
      <td>80</td>
      <td></td>
      <td>__EXTRACT_DATASET__, __FILTER_EMPTY_DATASETS__, map_param_value, __BUILD_LIST__, Grep1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>WF1_Database_Generation_Workflow</td>
      <td>https://usegalaxy.eu</td>
      <td>7cfbf4b7bd3f30d3</td>
      <td>https://usegalaxy.eu/published/workflow?id=7cfbf4b7bd3f30d3</td>
      <td></td>
      <td>name:clinicalmp, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>metanovo, fasta_merge_files_and_filter_unique_sequences</td>
      <td>Protein identification, Expression analysis, Target-Decoy, de Novo sequencing, Tag-based peptide identification</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>WF2_Discovery-Workflow</td>
      <td>https://usegalaxy.eu</td>
      <td>8af41219411062ad</td>
      <td>https://usegalaxy.eu/published/workflow?id=8af41219411062ad</td>
      <td></td>
      <td>name:microgalaxy, name:clinicalmp</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>msconvert, fasta2tab, fasta_merge_files_and_filter_unique_sequences, search_gui, peptide_shaker, fasta_cli, Cut1, ident_params, Remove beginning1, query_tabular, Grouping1, dbbuilder, tp_cat, filter_tabular, Filter1, Grep1, maxquant</td>
      <td>Filtering, Clustering, Heat map generation, Protein quantification, Formatting, Imputation, Standardisation and normalisation, Visualisation, Statistical calculation, Principal component plotting</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>WF3_VERIFICATION_WORKFLOW</td>
      <td>https://usegalaxy.eu</td>
      <td>d87fc6685e47d27d</td>
      <td>https://usegalaxy.eu/published/workflow?id=d87fc6685e47d27d</td>
      <td></td>
      <td>name:microgalaxy, name:clinicalmp</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>pepquery2, fasta_merge_files_and_filter_unique_sequences, collapse_dataset, Cut1, Remove beginning1, query_tabular, Grouping1, dbbuilder, Filter1, uniprotxml_downloader, tp_cat</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>WF4_Quantitation_Workflow</td>
      <td>https://usegalaxy.eu</td>
      <td>a49275eb8b2b2ecb</td>
      <td>https://usegalaxy.eu/published/workflow?id=a49275eb8b2b2ecb</td>
      <td></td>
      <td>name:clinicalmp, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>Cut1, Grep1, Grouping1, maxquant</td>
      <td>Clustering, Heat map generation, Protein quantification, Imputation, Standardisation and normalisation, Visualisation, Statistical calculation, Principal component plotting</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>WF5_Data_Interpretation_Worklow</td>
      <td>https://usegalaxy.eu</td>
      <td>755ede37d94dc456</td>
      <td>https://usegalaxy.eu/published/workflow?id=755ede37d94dc456</td>
      <td></td>
      <td>name:microgalaxy, name:clinicalmp</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>unipept, Grep1, msstatstmt</td>
      <td>Visualisation, Prediction and recognition</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>MetaG_extended</td>
      <td>https://usegalaxy.eu</td>
      <td>d57d41e306241396</td>
      <td>https://usegalaxy.eu/published/workflow?id=d57d41e306241396</td>
      <td></td>
      <td>name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>interproscan, fasplit, tp_awk_tool, maxbin2, trim_galore, quast, kofamscan, tp_grep_tool, fraggenescan, hmmer_hmmscan, sort1, fastqc, coverm_genome, checkm_lineage_wf, cat_bins, coverm_contig, tp_cat</td>
      <td>Conversion, Sequence generation, Probabilistic sequence generation, Sequence assembly, Coding region prediction, Sequencing quality control, Formatting, Visualisation, Data retrieval, Read pre-processing, Sequence trimming, Database search, Gene functional annotation, Protein feature detection, Sequence profile generation, Format validation, Sequence composition calculation, Sequence database search, Local alignment, Sequence assembly validation, Gene prediction, Sequence motif recognition, Primer removal, Statistical calculation, Taxonomic classification, Multiple sequence alignment, Sequence analysis, Validation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>MetaP</td>
      <td>https://usegalaxy.eu</td>
      <td>01ac8bf9832789e2</td>
      <td>https://usegalaxy.eu/published/workflow?id=01ac8bf9832789e2</td>
      <td></td>
      <td>name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>maxquant</td>
      <td>Clustering, Heat map generation, Protein quantification, Imputation, Standardisation and normalisation, Visualisation, Statistical calculation, Principal component plotting</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>MetaT</td>
      <td>https://usegalaxy.eu</td>
      <td>3733bcdf5b458be7</td>
      <td>https://usegalaxy.eu/published/workflow?id=3733bcdf5b458be7</td>
      <td></td>
      <td>name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>bg_sortmerna, trim_galore, __UNZIP_COLLECTION__, kallisto_quant, collection_column_join, fastqc</td>
      <td>Sequence alignment analysis, Sequence similarity search, Sequence composition calculation, Gene expression profiling, Sequencing quality control, Sequence comparison, Primer removal, Read pre-processing, Sequence trimming, Statistical calculation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>metaquantome-taxonomy-workflow</td>
      <td>https://usegalaxy.eu</td>
      <td>695f7defb516d80d</td>
      <td>https://usegalaxy.eu/published/workflow?id=695f7defb516d80d</td>
      <td></td>
      <td>proteomics, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>metaquantome_viz, metaquantome_stat, metaquantome_expand, metaquantome_sample, metaquantome_db, metaquantome_filter</td>
      <td>Heat map generation, Statistical inference, Principal component visualisation, Visualisation, Query and retrieval, Differential protein expression analysis, Filtering, Quantification, Indexing, Functional clustering</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>metaquantome-function-worklow</td>
      <td>https://usegalaxy.eu</td>
      <td>4c364c6be27981ba</td>
      <td>https://usegalaxy.eu/published/workflow?id=4c364c6be27981ba</td>
      <td></td>
      <td>proteomics, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>metaquantome_viz, metaquantome_stat, metaquantome_expand, metaquantome_sample, metaquantome_db, metaquantome_filter</td>
      <td>Heat map generation, Statistical inference, Principal component visualisation, Visualisation, Query and retrieval, Differential protein expression analysis, Filtering, Quantification, Indexing, Functional clustering</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Metaproteomics_GTN</td>
      <td>https://usegalaxy.eu</td>
      <td>68967c922de149ae</td>
      <td>https://usegalaxy.eu/published/workflow?id=68967c922de149ae</td>
      <td></td>
      <td>proteomics, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>search_gui, peptide_shaker, unipept, query_tabular, sqlite_to_tabular</td>
      <td>Visualisation, Prediction and recognition</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Ecoli Comparison</td>
      <td>https://usegalaxy.eu</td>
      <td>0e492839192d26e0</td>
      <td>https://usegalaxy.eu/published/workflow?id=0e492839192d26e0</td>
      <td></td>
      <td>assembly, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>lastz_wrapper_2, tp_grep_tool, fasta_filter_by_length, tp_sed_tool, datamash_ops, mergeCols1, tp_sort_header_tool, bedtools_complementbed, collapse_dataset, join1, addValue, random_lines1, fasta_compute_length, cat1, Cut1, bedtools_sortbed, Filter1, Grep1, bedtools_intersectbed, tp_cat</td>
      <td>Read mapping, Mapping, Sequence alignment</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Training: 16S rRNA Analysis with Nanopore Sequencing Reads</td>
      <td>https://usegalaxy.eu</td>
      <td>08eb5e55109c4d9c</td>
      <td>https://usegalaxy.eu/published/workflow?id=08eb5e55109c4d9c</td>
      <td></td>
      <td>metagenomics, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>kraken2, porechop, tp_replace_in_line, taxonomy_krona_chart, fastp, multiqc, Remove beginning1, fastqc, datamash_reverse</td>
      <td>Visualisation, Sequence contamination filtering, Statistical calculation, Taxonomic classification, Sequence composition calculation, Sequencing quality control, Validation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Main Metatranscriptomics Analysis</td>
      <td>https://usegalaxy.eu</td>
      <td>7a08ecdf23e0c7a3</td>
      <td>https://usegalaxy.eu/published/workflow?id=7a08ecdf23e0c7a3</td>
      <td>Bérénice Batut, Pratik Jagtap, Subina Mehta, Ray Sajulga, Emma Leith, Praveen Kumar, Saskia Hiltemann, Paul Zierep</td>
      <td>name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Copy Of GTN Training - Antibiotic Resistance Detection</td>
      <td>https://usegalaxy.eu</td>
      <td>44e53ea319e07daa</td>
      <td>https://usegalaxy.eu/published/workflow?id=44e53ea319e07daa</td>
      <td></td>
      <td>metagenomics, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>gfa_to_fa, bandage_image, nanoplot, racon, minimap2, PlasFlow, miniasm, staramr_search, unicycler</td>
      <td>Aggregation, Scatter plot plotting, Mapping assembly, De-novo assembly, Pairwise sequence alignment, Box-Whisker plot plotting, Sequence assembly visualisation, Genome assembly, Sequence analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Identification of the micro-organisms in a beer using Nanopore sequencing</td>
      <td>https://usegalaxy.eu</td>
      <td>0590554db9794d13</td>
      <td>https://usegalaxy.eu/published/workflow?id=0590554db9794d13</td>
      <td>Bérénice Batut, Teresa Müller, Polina Polunina</td>
      <td>metagenomics, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>kraken2, porechop, taxonomy_krona_chart, fastp, fastqc, Filter1, krakentools_kreport2krona</td>
      <td>Aggregation, Visualisation, Sequence contamination filtering, Statistical calculation, Taxonomic classification, Sequence composition calculation, Sequencing quality control</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>workflow-generate-dataset-for-assembly-tutorial</td>
      <td>https://usegalaxy.eu</td>
      <td>ce5c16ac23c4fbb1</td>
      <td>https://usegalaxy.eu/published/workflow?id=ce5c16ac23c4fbb1</td>
      <td></td>
      <td>name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>fastqc, seqtk_subseq, bamtools, filter_tabular, bg_uniq, ngsutils_bam_filter, megahit, random_lines1, bowtie2, cutadapt, tp_cat</td>
      <td>Data handling, Sequence alignment analysis, Sequence composition calculation, Read mapping, Sequencing quality control, Formatting, Variant calling, Genome assembly, Sequence trimming, Statistical calculation, Sequence contamination filtering, Read pre-processing, Primer removal, Sequence file editing</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Metagenomics assembly tutorial workflow</td>
      <td>https://usegalaxy.eu</td>
      <td>62c47e3c9d4f3aa3</td>
      <td>https://usegalaxy.eu/published/workflow?id=62c47e3c9d4f3aa3</td>
      <td></td>
      <td>name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>megahit_contig2fastg, bandage_image, quast, collection_column_join, metaspades, megahit, bowtie2, bandage_info</td>
      <td>Sequence assembly validation, Visualisation, Genome assembly, Read mapping, Sequence assembly visualisation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>WGS Part In "Analyses Of Metagenomics Data - The Global Picture"</td>
      <td>https://usegalaxy.eu</td>
      <td>c2b7ec5aacfc6107</td>
      <td>https://usegalaxy.eu/published/workflow?id=c2b7ec5aacfc6107</td>
      <td></td>
      <td>metagenomics, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>metaphlan2krona, metaphlan2, humann2_renorm_table, taxonomy_krona_chart, humann2, humann2_regroup_table</td>
      <td>Visualisation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Taxonomic Profiling and Visualization of Metagenomic Data</td>
      <td>https://usegalaxy.eu</td>
      <td>4f0daf8e9dd54127</td>
      <td>https://usegalaxy.eu/published/workflow?id=4f0daf8e9dd54127</td>
      <td>Bérénice Batut</td>
      <td>metagenomics, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>metaphlan, kraken2, interactive_tool_phinch, kraken_biom, est_abundance, taxonomy_krona_chart, __UNZIP_COLLECTION__, interactive_tool_pavian, krakentools_kreport2krona</td>
      <td>Aggregation, Visualisation, Statistical calculation, Taxonomic classification, Phylogenetic tree analysis, Nucleic acid sequence analysis</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Amplicon Tutorial</td>
      <td>https://usegalaxy.eu</td>
      <td>565c989730f68362</td>
      <td>https://usegalaxy.eu/published/workflow?id=565c989730f68362</td>
      <td></td>
      <td>metagenomics, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>mothur_make_shared, mothur_cluster_split, krona-text, mothur_pre_cluster, mothur_screen_seqs, mothur_classify_seqs, mothur_count_seqs, mothur_unique_seqs, mothur_filter_seqs, mothur_make_group, mothur_classify_otu, mothur_summary_seqs, mothur_merge_files, mothur_make_biom, mothur_align_seqs</td>
      <td>Taxonomic classification, Sequencing quality control, Phylogenetic analysis, Visualisation, Sequence clustering, Sequence read processing, DNA barcoding</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Microbial Variant Calling</td>
      <td>https://usegalaxy.eu</td>
      <td>515ef1601bc24a6c</td>
      <td>https://usegalaxy.eu/published/workflow?id=515ef1601bc24a6c</td>
      <td></td>
      <td>variant-analysis, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>snippy, jbrowse</td>
      <td>Phylogenetic tree visualisation, Variant calling, Phylogenetic tree generation, Genome visualisation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Calling variants in non-diploid systems</td>
      <td>https://usegalaxy.eu</td>
      <td>291c36e1b18e9deb</td>
      <td>https://usegalaxy.eu/published/workflow?id=291c36e1b18e9deb</td>
      <td></td>
      <td>variant-analysis, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>bwa_mem, bamleftalign, picard_MarkDuplicates, picard_MergeSamFiles, vcffilter2, Cut1, freebayes, fastqc, bamFilter, vcf2tsv</td>
      <td>Data handling, Sequence alignment analysis, Sequence composition calculation, Read mapping, Sequencing quality control, Formatting, Variant calling, Sequence alignment, Statistical calculation, Genome indexing, Generation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Training: 16S rRNA Sequencing With Mothur: Main Tutorial</td>
      <td>https://usegalaxy.eu</td>
      <td>c67d2e75eeb4fd19</td>
      <td>https://usegalaxy.eu/published/workflow?id=c67d2e75eeb4fd19</td>
      <td></td>
      <td>metagenomics, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>mothur_cluster_split, mothur_screen_seqs, mothur_count_seqs, mothur_dist_shared, mothur_classify_otu, taxonomy_krona_chart, mothur_count_groups, mothur_make_contigs, mothur_make_biom, mothur_make_shared, mothur_pre_cluster, mothur_summary_single, mothur_taxonomy_to_krona, mothur_remove_groups, mothur_venn, mothur_sub_sample, mothur_classify_seqs, mothur_filter_seqs, mothur_chimera_vsearch, mothur_summary_seqs, newick_display, mothur_rarefaction_single, mothur_get_groups, mothur_align_seqs, mothur_heatmap_sim, mothur_remove_lineage, mothur_tree_shared, mothur_remove_seqs, mothur_cluster, mothur_unique_seqs, mothur_dist_seqs, XY_Plot_1, mothur_seq_error</td>
      <td>Taxonomic classification, Phylogenetic tree analysis, Sequencing quality control, Phylogenetic analysis, Phylogenetic tree generation, Visualisation, Phylogenetic tree reconstruction, Sequence clustering, Sequence read processing, DNA barcoding</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Comparative gene analysis</td>
      <td>https://usegalaxy.eu</td>
      <td>86900be4282d4b1d</td>
      <td>https://usegalaxy.eu/published/workflow?id=86900be4282d4b1d</td>
      <td>Anton Nekrutenko</td>
      <td>vgp, genome-annotation, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>cat1, tp_split_on_column, Add_a_column1, regexColumn1, collapse_dataset, join1, Cut1, tab2fasta, rapidnj, rbc_mafft, bg_diamond, bg_diamond_view, Filter1, gops_intersect_1</td>
      <td>Multiple sequence alignment, Sequence alignment analysis, Phylogenetic tree generation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Unicycler training</td>
      <td>https://usegalaxy.eu</td>
      <td>8f3ce55d44d9f369</td>
      <td>https://usegalaxy.eu/published/workflow?id=8f3ce55d44d9f369</td>
      <td></td>
      <td>assembly, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>quast, multiqc, prokka, fastqc, unicycler</td>
      <td>Aggregation, Sequence composition calculation, Sequencing quality control, Sequence assembly validation, Visualisation, Gene prediction, Genome annotation, Statistical calculation, Genome assembly, Coding region prediction, Validation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Apollo Load Test</td>
      <td>https://usegalaxy.eu</td>
      <td>3c233cc56d4df8b0</td>
      <td>https://usegalaxy.eu/published/workflow?id=3c233cc56d4df8b0</td>
      <td></td>
      <td>genome-annotation, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>create_or_update, list_organism, iframe, jbrowse, create_account</td>
      <td>Genome visualisation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation with Prokka [Feb 2020]</td>
      <td>https://usegalaxy.eu</td>
      <td>e1894c36f1ab4ab4</td>
      <td>https://usegalaxy.eu/published/workflow?id=e1894c36f1ab4ab4</td>
      <td></td>
      <td>genome-annotation, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>jbrowse, prokka</td>
      <td>Genome visualisation, Gene prediction, Coding region prediction, Genome annotation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Flye-AMR workflow EMC/WMDI - v3.5  (imported from URL)</td>
      <td>https://usegalaxy.eu</td>
      <td>152223c8aed27357</td>
      <td>https://usegalaxy.eu/published/workflow?id=152223c8aed27357</td>
      <td></td>
      <td>wmdi, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>tp_easyjoin_tool, bg_column_arrange_by_header, bandage_image, minimap2, jbrowse, prokka, tbl2gff3, circos, staramr_search, circos_interval_to_tile, tp_cut_tool, regex1, tp_text_file_with_recurring_lines, ncbi_blastn_wrapper, tp_split_on_column, Add_a_column1, tp_sort_header_tool, gff2bed1, join1, tab2fasta, regex_replace, fasta_compute_length, deeptools_bam_coverage, circos_gc_skew, fasta2tab, trimmomatic, flye, Cut1, nanoplot, medaka_consensus_pipeline, PlasFlow, circos_wiggle_to_scatter, tp_head_tool, Filter1, Grep1, tp_cat</td>
      <td>Scatter plot plotting, Base-calling, De-novo assembly, Sequence assembly, Pairwise sequence alignment, Genome assembly, Box-Whisker plot plotting, Sequence assembly visualisation, Variant calling, Genome visualisation, Gene prediction, Genome annotation, Sequence analysis, Sequence visualisation, Coding region prediction, Mapping assembly, Cross-assembly</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Mtb phylogeny</td>
      <td>https://usegalaxy.eu</td>
      <td>875b3ca170ac12a1</td>
      <td>https://usegalaxy.eu/published/workflow?id=875b3ca170ac12a1</td>
      <td></td>
      <td>evolution, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>interactive_tool_rstudio, raxml</td>
      <td>Phylogenetic tree analysis, Sequence analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>From VCFs to SNP distance matrix</td>
      <td>https://usegalaxy.eu</td>
      <td>6d333ed886aab471</td>
      <td>https://usegalaxy.eu/published/workflow?id=6d333ed886aab471</td>
      <td></td>
      <td>evolution, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>bcftools_consensus, tb_variant_filter, snp_dists, snp_sites, tp_cat</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>From Fastqs to VCFs and BAMs</td>
      <td>https://usegalaxy.eu</td>
      <td>3a8fde248a4d0383</td>
      <td>https://usegalaxy.eu/published/workflow?id=3a8fde248a4d0383</td>
      <td></td>
      <td>evolution, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>__MERGE_COLLECTION__, tb_variant_filter, snippy, trimmomatic</td>
      <td>Phylogenetic tree visualisation, Variant calling, Phylogenetic tree generation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>From BAMs to drug resistance prediction with TB-profiler</td>
      <td>https://usegalaxy.eu</td>
      <td>d0d6dae0d1eecc21</td>
      <td>https://usegalaxy.eu/published/workflow?id=d0d6dae0d1eecc21</td>
      <td></td>
      <td>evolution, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>tb_profiler_profile, samtools_view, tp_grep_tool, tp_sed_tool, __MERGE_COLLECTION__, tp_replace_in_line, addName, tp_cat</td>
      <td>Antimicrobial resistance prediction</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>M. tuberculosis Variant Analysis tutorial</td>
      <td>https://usegalaxy.eu</td>
      <td>3ff4dbcc7a602e6b</td>
      <td>https://usegalaxy.eu/published/workflow?id=3ff4dbcc7a602e6b</td>
      <td>Peter van Heusden</td>
      <td>variant-analysis, prokaryote, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>tb_profiler_profile, tbvcfreport, kraken2, trimmomatic, tb_variant_filter, snippy, multiqc, jbrowse, fastqc</td>
      <td>Sequence composition calculation, Sequencing quality control, Variant calling, Phylogenetic tree generation, Genome visualisation, Antimicrobial resistance prediction, Statistical calculation, Taxonomic classification, Phylogenetic tree visualisation, Validation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Compute Allel Frequency Table from paired reads</td>
      <td>https://usegalaxy.eu</td>
      <td>21a39ac4c1becb31</td>
      <td>https://usegalaxy.eu/published/workflow?id=21a39ac4c1becb31</td>
      <td></td>
      <td>name:microgalaxy, hpylori</td>
      <td>2023-12-13</td>
      <td>2023-12-13</td>
      <td>2</td>
      <td>2</td>
      <td></td>
      <td>bcftools_norm, tp_awk_tool, freebayes, bowtie2, vcf2tsv, bcftools_view, snpSift_filter</td>
      <td>Read mapping, Variant calling, Statistical calculation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Reference genome choice (indels, coverage, depth) for short reads</td>
      <td>https://usegalaxy.eu</td>
      <td>61fe08fb3d5e83f8</td>
      <td>https://usegalaxy.eu/published/workflow?id=61fe08fb3d5e83f8</td>
      <td></td>
      <td>name:microgalaxy, hpylori</td>
      <td>2023-10-27</td>
      <td>2023-12-13</td>
      <td>13</td>
      <td>13</td>
      <td></td>
      <td>bcftools_norm, samtools_depth, collapse_dataset, collection_element_identifiers, vcf2tsv, freebayes, bowtie2, samtools_coverage, tp_cat</td>
      <td>Read mapping, Variant calling, Statistical calculation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>MGnify amplicon rRNA-prediction + ITS</td>
      <td>https://usegalaxy.eu</td>
      <td>437704898229dfb6</td>
      <td>https://usegalaxy.eu/published/workflow?id=437704898229dfb6</td>
      <td></td>
      <td>mgnify_amplicon_showcase, name:microgalaxy</td>
      <td>2023-12-12</td>
      <td>2023-12-13</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>infernal_cmsearch, gops_concat_1, tp_awk_tool, bedtools_maskfastabed, mapseq, bedtools_getfastabed, biom_convert, taxonomy_krona_chart, query_tabular, cmsearch_deoverlap</td>
      <td>Visualisation, Comparison, Alignment, Nucleic acid feature detection, Mapping, Formatting, k-mer counting</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>MGnify amplicon QC</td>
      <td>https://usegalaxy.eu</td>
      <td>8f415b0c409f56f7</td>
      <td>https://usegalaxy.eu/published/workflow?id=8f415b0c409f56f7</td>
      <td></td>
      <td>mgnify_amplicon_showcase, name:microgalaxy</td>
      <td>2023-12-13</td>
      <td>2023-12-13</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>prinseq, trimmomatic, tp_find_and_replace, fastq_dump, fastp, __UNZIP_COLLECTION__, cshl_fasta_formatter, multiqc, iuc_pear, fastqc, fastq_to_fasta_python</td>
      <td>Data handling, Sequence merging, Sequence composition calculation, Sequencing quality control, Sequence contamination filtering, Read pre-processing, Sequence trimming, Statistical calculation, Validation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Cloud-Aerosole MT-MG Functional Profiling</td>
      <td>https://usegalaxy.eu</td>
      <td>4566262dd0acc8ab</td>
      <td>https://usegalaxy.eu/published/workflow?id=4566262dd0acc8ab</td>
      <td></td>
      <td>metagenomics, metatranscriptomics, name:microgalaxy</td>
      <td>2021-11-10</td>
      <td>2023-12-07</td>
      <td>2</td>
      <td>2</td>
      <td></td>
      <td>multiqc, bowtie2, featurecounts</td>
      <td>Read summarisation, Read mapping, Sequencing quality control, Validation, RNA-Seq quantification</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Cloud-Aerosole MT-MG Reference Gene Catalog Building</td>
      <td>https://usegalaxy.eu</td>
      <td>9b9e1e28f3683f92</td>
      <td>https://usegalaxy.eu/published/workflow?id=9b9e1e28f3683f92</td>
      <td></td>
      <td>metagenomics, metatranscriptomics, name:microgalaxy</td>
      <td>2021-11-01</td>
      <td>2023-12-07</td>
      <td>72</td>
      <td>72</td>
      <td></td>
      <td>cat1, metagene_annotator, bg_diamond_makedb, metaeuk_easy_predict, translate_nucleotides, quast, kofamscan, bedtools_getfastabed, cd_hit, eggnog_mapper, multiqc, bg_diamond, dbbuilder, megahit, fastqc, bowtie2</td>
      <td>Sequence alignment analysis, Sequence annotation, Sequencing quality control, Fold recognition, Visualisation, Homology-based gene prediction, Query and retrieval, Genome assembly, Sequence clustering, Gene functional annotation, Sequence composition calculation, Read mapping, Sequence assembly validation, Genome annotation, Statistical calculation, Mapping, Sequence analysis, Information extraction, Validation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Cloud-Aerosole MT-MG Pre-Processing</td>
      <td>https://usegalaxy.eu</td>
      <td>bb604ff5e010e7ab</td>
      <td>https://usegalaxy.eu/published/workflow?id=bb604ff5e010e7ab</td>
      <td></td>
      <td>metagenomics, metatranscriptomics, name:microgalaxy</td>
      <td>2021-11-01</td>
      <td>2023-12-07</td>
      <td>51</td>
      <td>51</td>
      <td></td>
      <td>fastqc, multiqc, trimmomatic</td>
      <td>Statistical calculation, Sequence composition calculation, Sequencing quality control, Validation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Cloud Aerosol MT-MG Contamination Filtering</td>
      <td>https://usegalaxy.eu</td>
      <td>bdc7cfaac0db2eab</td>
      <td>https://usegalaxy.eu/published/workflow?id=bdc7cfaac0db2eab</td>
      <td></td>
      <td>metagenomics, metatranscriptomics, name:microgalaxy</td>
      <td>2022-06-29</td>
      <td>2023-12-07</td>
      <td>28</td>
      <td>28</td>
      <td></td>
      <td>seq_filter_by_id, kraken2, fastq_to_tabular, filter_tabular, bowtie2, recentrifuge</td>
      <td>Taxonomic classification, Read mapping, Expression analysis, Cross-assembly</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Cloud-Aerosole MT-MG Taxonomic Profiling</td>
      <td>https://usegalaxy.eu</td>
      <td>e69b555faa6bb962</td>
      <td>https://usegalaxy.eu/published/workflow?id=e69b555faa6bb962</td>
      <td></td>
      <td>metagenomics, metatranscriptomics, name:microgalaxy</td>
      <td>2022-09-03</td>
      <td>2023-12-07</td>
      <td>18</td>
      <td>18</td>
      <td></td>
      <td>maaslin2, kraken2, interactive_tool_phinch, kraken_biom, recentrifuge</td>
      <td>Filtering, Visualisation, Expression analysis, Statistical calculation, Taxonomic classification, Cross-assembly, Standardisation and normalisation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>pAllori ASiam Functional Information MT</td>
      <td>https://usegalaxy.eu</td>
      <td>a7a6d8ecc2795f58</td>
      <td>https://usegalaxy.eu/published/workflow?id=a7a6d8ecc2795f58</td>
      <td>Bérénice Batut, Pratik Jagtap, Subina Mehta, Ray Sajulga, Emma Leith, Praveen Kumar, Saskia Hiltemann, Paul Zierep, Engy Nasr</td>
      <td>name:microgalaxy, name:metatranscriptomics, name:adaptedasiam, name:functionalprofiling</td>
      <td>2023-07-07</td>
      <td>2023-07-07</td>
      <td>3</td>
      <td>3</td>
      <td></td>
      <td>combine_metaphlan2_humann2, Cut1, tp_find_and_replace, humann_renorm_table, humann, humann_regroup_table, humann_unpack_pathways, humann_split_stratified_table, Grep1, humann_rename_table</td>
      <td>Species frequency estimation, Taxonomic classification, Phylogenetic analysis</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>pAllori ASiam Community Profile MT</td>
      <td>https://usegalaxy.eu</td>
      <td>05f5f040337786bf</td>
      <td>https://usegalaxy.eu/published/workflow?id=05f5f040337786bf</td>
      <td>Engy Nasr, Bérénice Batut</td>
      <td>name:microgalaxy, name:metatranscriptomics, name:adaptedasiam, name:communityprofiling</td>
      <td>2023-06-30</td>
      <td>2023-07-06</td>
      <td>3</td>
      <td>3</td>
      <td></td>
      <td>metaphlan, graphlan_annotate, Cut1, graphlan, taxonomy_krona_chart, export2graphlan</td>
      <td>Conversion, Phylogenetic tree analysis, Visualisation, Phylogenetic inference, Taxonomic classification, Phylogenetic tree visualisation, Nucleic acid sequence analysis, Phylogenetic tree editing</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>pAllori ASiam Preprocessing MT</td>
      <td>https://usegalaxy.eu</td>
      <td>72e3e5fdc766e24a</td>
      <td>https://usegalaxy.eu/published/workflow?id=72e3e5fdc766e24a</td>
      <td>Bérénice Batut, Pratik Jagtap, Subina Mehta, Ray Sajulga, Emma Leith, Praveen Kumar, Saskia Hiltemann, Paul Zierep, Engy Nasr</td>
      <td>name:microgalaxy, name:adaptedasiam, name:metatranscriptomics, name:preprocessing</td>
      <td>2023-06-30</td>
      <td>2023-07-06</td>
      <td>2</td>
      <td>2</td>
      <td></td>
      <td>bg_sortmerna, fastq_paired_end_interlacer, multiqc, fastqc, __FLATTEN__, cutadapt</td>
      <td>Sequence alignment analysis, Sequence similarity search, Sequence composition calculation, Sequencing quality control, Sequence comparison, Statistical calculation, Primer removal, Read pre-processing, Sequence trimming, Validation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>MetaP</td>
      <td>https://usegalaxy.eu</td>
      <td>96d198ed953449fd</td>
      <td>https://usegalaxy.eu/published/workflow?id=96d198ed953449fd</td>
      <td></td>
      <td>name:microgalaxy</td>
      <td>2020-01-30</td>
      <td>2023-03-16</td>
      <td>2</td>
      <td>2</td>
      <td></td>
      <td>maxquant</td>
      <td>Clustering, Heat map generation, Protein quantification, Imputation, Standardisation and normalisation, Visualisation, Statistical calculation, Principal component plotting</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>MetaG_extended</td>
      <td>https://usegalaxy.eu</td>
      <td>6158c3b5fc12044e</td>
      <td>https://usegalaxy.eu/published/workflow?id=6158c3b5fc12044e</td>
      <td></td>
      <td>name:microgalaxy</td>
      <td>2023-02-10</td>
      <td>2023-03-16</td>
      <td>22</td>
      <td>22</td>
      <td></td>
      <td>interproscan, fasplit, tp_awk_tool, maxbin2, trim_galore, quast, kofamscan, tp_grep_tool, fraggenescan, hmmer_hmmscan, sort1, fastqc, coverm_genome, checkm_lineage_wf, cat_bins, coverm_contig, tp_cat</td>
      <td>Conversion, Sequence generation, Probabilistic sequence generation, Sequence assembly, Coding region prediction, Sequencing quality control, Formatting, Visualisation, Data retrieval, Read pre-processing, Sequence trimming, Database search, Gene functional annotation, Protein feature detection, Sequence profile generation, Format validation, Sequence composition calculation, Sequence database search, Local alignment, Sequence assembly validation, Gene prediction, Sequence motif recognition, Primer removal, Statistical calculation, Taxonomic classification, Multiple sequence alignment, Sequence analysis, Validation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>MetaT</td>
      <td>https://usegalaxy.eu</td>
      <td>385cc7df70d7916b</td>
      <td>https://usegalaxy.eu/published/workflow?id=385cc7df70d7916b</td>
      <td></td>
      <td>name:microgalaxy</td>
      <td>2020-01-27</td>
      <td>2023-02-27</td>
      <td>16</td>
      <td>16</td>
      <td></td>
      <td>bg_sortmerna, trim_galore, __UNZIP_COLLECTION__, kallisto_quant, collection_column_join, fastqc</td>
      <td>Sequence alignment analysis, Sequence similarity search, Sequence composition calculation, Gene expression profiling, Sequencing quality control, Sequence comparison, Primer removal, Read pre-processing, Sequence trimming, Statistical calculation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Allele-based Pathogen Identification (release v0.1.1)</td>
      <td>https://usegalaxy.org</td>
      <td>29ba77708cdafe0f</td>
      <td>https://usegalaxy.org/published/workflow?id=29ba77708cdafe0f</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-11-28</td>
      <td>2024-11-28</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>bcftools_norm, samtools_depth, table_compute, collapse_dataset, tp_cut_tool, Cut1, clair3, snpSift_extractFields, bcftools_consensus, Count1, regexColumn1, minimap2, Paste1, Remove beginning1, tp_head_tool, samtools_coverage, CONVERTER_gz_to_uncompressed, snpSift_filter</td>
      <td>Pairwise sequence alignment, Variant calling</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Allele-based Pathogen Identification (release v0.1.2)</td>
      <td>https://usegalaxy.org</td>
      <td>6d778eeb176d563c</td>
      <td>https://usegalaxy.org/published/workflow?id=6d778eeb176d563c</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-10-31</td>
      <td>2024-10-31</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>bcftools_norm, samtools_depth, table_compute, collapse_dataset, tp_cut_tool, Cut1, clair3, snpSift_extractFields, bcftools_consensus, Count1, regexColumn1, minimap2, Paste1, Remove beginning1, tp_head_tool, samtools_coverage, CONVERTER_gz_to_uncompressed, snpSift_filter</td>
      <td>Pairwise sequence alignment, Variant calling</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Taxonomy Profiling and Visualization with Kron (release v0.1)</td>
      <td>https://usegalaxy.org</td>
      <td>4b5f6b5d4f36e38b</td>
      <td>https://usegalaxy.org/published/workflow?id=4b5f6b5d4f36e38b</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-09-19</td>
      <td>2024-09-19</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>taxonomy_krona_chart, krakentools_kreport2krona, kraken2</td>
      <td>Aggregation, Visualisation, Taxonomic classification</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Pathogen Detection PathoGFAIR Samples Aggregation and Visualisation (release v0.1)</td>
      <td>https://usegalaxy.org</td>
      <td>3ef08553982f6ffc</td>
      <td>https://usegalaxy.org/published/workflow?id=3ef08553982f6ffc</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-09-19</td>
      <td>2024-09-19</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>tp_multijoin_tool, bedtools_getfastabed, ggplot2_heatmap, regex1, Grouping1, clustalw, tp_split_on_column, fasttree, __FILTER_FAILED_DATASETS__, collapse_dataset, tab2fasta, newick_display, tp_replace_in_column, fasta2tab, fasta_merge_files_and_filter_unique_sequences, regexColumn1, Count1, Cut1, __FILTER_EMPTY_DATASETS__, tp_sorted_uniq, Remove beginning1, collection_column_join</td>
      <td>Phylogenetic tree generation (maximum likelihood and Bayesian methods), Visualisation, Phylogenetic tree generation (from molecular sequences), Phylogenetic tree reconstruction, Mapping, Multiple sequence alignment, Phylogenetic tree analysis, Phylogenetic tree generation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Gene-based Pathogen Identification (release v0.1)</td>
      <td>https://usegalaxy.org</td>
      <td>3f252e077e0bcce5</td>
      <td>https://usegalaxy.org/published/workflow?id=3f252e077e0bcce5</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:pathogfair, name:iwc, name:microgalaxy</td>
      <td>2024-09-19</td>
      <td>2024-09-19</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>fasta2tab, abricate, flye, collection_element_identifiers, tp_find_and_replace, bandage_image, param_value_from_file, tab2fasta, medaka_consensus_pipeline, compose_text_param, split_file_to_collection, __BUILD_LIST__</td>
      <td>Base-calling, De-novo assembly, Sequence assembly, Variant calling, Sequence assembly visualisation, Antimicrobial resistance prediction, Genome assembly, Mapping assembly, Cross-assembly</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Pathogen Detection PathoGFAIR Samples Aggregation and Visualisation</td>
      <td>https://usegalaxy.org</td>
      <td>2d3063882d8239ff</td>
      <td>https://usegalaxy.org/published/workflow?id=2d3063882d8239ff</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-07-03</td>
      <td>2024-07-03</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>tp_multijoin_tool, bedtools_getfastabed, ggplot2_heatmap, regex1, Grouping1, clustalw, tp_split_on_column, fasttree, __FILTER_FAILED_DATASETS__, collapse_dataset, tab2fasta, newick_display, tp_replace_in_column, fasta2tab, fasta_merge_files_and_filter_unique_sequences, regexColumn1, Count1, Cut1, __FILTER_EMPTY_DATASETS__, tp_sorted_uniq, Remove beginning1, collection_column_join</td>
      <td>Phylogenetic tree generation (maximum likelihood and Bayesian methods), Visualisation, Phylogenetic tree generation (from molecular sequences), Phylogenetic tree reconstruction, Mapping, Multiple sequence alignment, Phylogenetic tree analysis, Phylogenetic tree generation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>PathoGFAIR (imported from uploaded file)</td>
      <td>https://usegalaxy.org</td>
      <td>e55593af91337a05</td>
      <td>https://usegalaxy.org/published/workflow?id=e55593af91337a05</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:microgalaxy, name:iwc, name:pathogfair, name:nanopore, name:collection</td>
      <td>2024-07-03</td>
      <td>2024-07-03</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Taxonomy Profiling and Visualization with Krona</td>
      <td>https://usegalaxy.org</td>
      <td>8f5904693b5f74f4</td>
      <td>https://usegalaxy.org/published/workflow?id=8f5904693b5f74f4</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-06-25</td>
      <td>2024-06-25</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>taxonomy_krona_chart, krakentools_kreport2krona, kraken2</td>
      <td>Aggregation, Visualisation, Taxonomic classification</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Gene-based Pathogen Identification</td>
      <td>https://usegalaxy.org</td>
      <td>cce88bc57b180d09</td>
      <td>https://usegalaxy.org/published/workflow?id=cce88bc57b180d09</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:pathogfair, name:iwc, name:microgalaxy</td>
      <td>2024-06-25</td>
      <td>2024-06-25</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>fasta2tab, abricate, flye, collection_element_identifiers, tp_find_and_replace, bandage_image, param_value_from_file, tab2fasta, medaka_consensus_pipeline, compose_text_param, split_file_to_collection, __BUILD_LIST__</td>
      <td>Base-calling, De-novo assembly, Sequence assembly, Variant calling, Sequence assembly visualisation, Antimicrobial resistance prediction, Genome assembly, Mapping assembly, Cross-assembly</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Allele-based Pathogen Identification</td>
      <td>https://usegalaxy.org</td>
      <td>38911ba6f66d80f6</td>
      <td>https://usegalaxy.org/published/workflow?id=38911ba6f66d80f6</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-06-25</td>
      <td>2024-06-25</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>bcftools_norm, samtools_depth, table_compute, collapse_dataset, tp_cut_tool, Cut1, clair3, snpSift_extractFields, bcftools_consensus, Count1, regexColumn1, minimap2, Paste1, Remove beginning1, tp_head_tool, samtools_coverage, CONVERTER_gz_to_uncompressed, snpSift_filter</td>
      <td>Pairwise sequence alignment, Variant calling</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Nanopore Preprocessing</td>
      <td>https://usegalaxy.org</td>
      <td>574e42683dc3961b</td>
      <td>https://usegalaxy.org/published/workflow?id=574e42683dc3961b</td>
      <td>Bérénice Batut, Engy Nasr, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:nanopore, name:iwc</td>
      <td>2024-06-25</td>
      <td>2024-06-25</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>Add_a_column1, porechop, collapse_dataset, __FILTER_FAILED_DATASETS__, samtools_fastx, kraken2, Cut1, nanoplot, krakentools_extract_kraken_reads, regexColumn1, minimap2, fastp, multiqc, collection_column_join, fastqc, Grep1, bamtools_split_mapped</td>
      <td>Aggregation, Data handling, Scatter plot plotting, Sequence alignment analysis, Sequence composition calculation, Sequencing quality control, Pairwise sequence alignment, Box-Whisker plot plotting, Visualisation, Sequence contamination filtering, Statistical calculation, Taxonomic classification, Validation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Allele-based Pathogen Identification (release v0.1.2)</td>
      <td>https://usegalaxy.org.au</td>
      <td>857e482acfcb2dba</td>
      <td>https://usegalaxy.org.au/published/workflow?id=857e482acfcb2dba</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-10-31</td>
      <td>2024-10-31</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>bcftools_norm, samtools_depth, table_compute, collapse_dataset, tp_cut_tool, Cut1, clair3, snpSift_extractFields, bcftools_consensus, Count1, regexColumn1, minimap2, Paste1, Remove beginning1, tp_head_tool, samtools_coverage, CONVERTER_gz_to_uncompressed, snpSift_filter</td>
      <td>Pairwise sequence alignment, Variant calling</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Taxonomy Profiling and Visualization with Kron (release v0.1)</td>
      <td>https://usegalaxy.org.au</td>
      <td>131636a795bac485</td>
      <td>https://usegalaxy.org.au/published/workflow?id=131636a795bac485</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-09-19</td>
      <td>2024-09-19</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>taxonomy_krona_chart, krakentools_kreport2krona, kraken2</td>
      <td>Aggregation, Visualisation, Taxonomic classification</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Pathogen Detection PathoGFAIR Samples Aggregation and Visualisation (release v0.1)</td>
      <td>https://usegalaxy.org.au</td>
      <td>b60922a253df6654</td>
      <td>https://usegalaxy.org.au/published/workflow?id=b60922a253df6654</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-09-19</td>
      <td>2024-09-19</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>tp_multijoin_tool, bedtools_getfastabed, ggplot2_heatmap, regex1, Grouping1, clustalw, tp_split_on_column, fasttree, __FILTER_FAILED_DATASETS__, collapse_dataset, tab2fasta, newick_display, tp_replace_in_column, fasta2tab, fasta_merge_files_and_filter_unique_sequences, regexColumn1, Count1, Cut1, __FILTER_EMPTY_DATASETS__, tp_sorted_uniq, Remove beginning1, collection_column_join</td>
      <td>Phylogenetic tree generation (maximum likelihood and Bayesian methods), Visualisation, Phylogenetic tree generation (from molecular sequences), Phylogenetic tree reconstruction, Mapping, Multiple sequence alignment, Phylogenetic tree analysis, Phylogenetic tree generation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Nanopore Preprocessing (release v0.1)</td>
      <td>https://usegalaxy.org.au</td>
      <td>69d91340fc7effa2</td>
      <td>https://usegalaxy.org.au/published/workflow?id=69d91340fc7effa2</td>
      <td>Bérénice Batut, Engy Nasr, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:nanopore, name:iwc</td>
      <td>2024-09-19</td>
      <td>2024-09-19</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>Add_a_column1, porechop, collapse_dataset, __FILTER_FAILED_DATASETS__, samtools_fastx, kraken2, Cut1, nanoplot, krakentools_extract_kraken_reads, regexColumn1, minimap2, fastp, multiqc, collection_column_join, fastqc, Grep1, bamtools_split_mapped</td>
      <td>Aggregation, Data handling, Scatter plot plotting, Sequence alignment analysis, Sequence composition calculation, Sequencing quality control, Pairwise sequence alignment, Box-Whisker plot plotting, Visualisation, Sequence contamination filtering, Statistical calculation, Taxonomic classification, Validation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Gene-based Pathogen Identification (release v0.1)</td>
      <td>https://usegalaxy.org.au</td>
      <td>084bb76cf47d7060</td>
      <td>https://usegalaxy.org.au/published/workflow?id=084bb76cf47d7060</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:pathogfair, name:iwc, name:microgalaxy</td>
      <td>2024-09-19</td>
      <td>2024-09-19</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>fasta2tab, abricate, flye, collection_element_identifiers, tp_find_and_replace, bandage_image, param_value_from_file, tab2fasta, medaka_consensus_pipeline, compose_text_param, split_file_to_collection, __BUILD_LIST__</td>
      <td>Base-calling, De-novo assembly, Sequence assembly, Variant calling, Sequence assembly visualisation, Antimicrobial resistance prediction, Genome assembly, Mapping assembly, Cross-assembly</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>PathoGFAIR (imported from uploaded file)</td>
      <td>https://usegalaxy.org.au</td>
      <td>f5f9808fb50b6f2c</td>
      <td>https://usegalaxy.org.au/published/workflow?id=f5f9808fb50b6f2c</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:microgalaxy, name:iwc, name:pathogfair, name:nanopore, name:collection</td>
      <td>2024-07-03</td>
      <td>2024-07-03</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Pathogen Detection PathoGFAIR Samples Aggregation and Visualisation</td>
      <td>https://usegalaxy.org.au</td>
      <td>eda40b58616a0fe4</td>
      <td>https://usegalaxy.org.au/published/workflow?id=eda40b58616a0fe4</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-07-03</td>
      <td>2024-07-03</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>tp_multijoin_tool, bedtools_getfastabed, ggplot2_heatmap, regex1, Grouping1, clustalw, tp_split_on_column, fasttree, __FILTER_FAILED_DATASETS__, collapse_dataset, tab2fasta, newick_display, tp_replace_in_column, fasta2tab, fasta_merge_files_and_filter_unique_sequences, regexColumn1, Count1, Cut1, __FILTER_EMPTY_DATASETS__, tp_sorted_uniq, Remove beginning1, collection_column_join</td>
      <td>Phylogenetic tree generation (maximum likelihood and Bayesian methods), Visualisation, Phylogenetic tree generation (from molecular sequences), Phylogenetic tree reconstruction, Mapping, Multiple sequence alignment, Phylogenetic tree analysis, Phylogenetic tree generation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Allele-based Pathogen Identification</td>
      <td>https://usegalaxy.org.au</td>
      <td>244ea5e94237ebad</td>
      <td>https://usegalaxy.org.au/published/workflow?id=244ea5e94237ebad</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-06-25</td>
      <td>2024-06-25</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>bcftools_norm, samtools_depth, table_compute, collapse_dataset, tp_cut_tool, Cut1, clair3, snpSift_extractFields, bcftools_consensus, Count1, regexColumn1, minimap2, Paste1, Remove beginning1, tp_head_tool, samtools_coverage, CONVERTER_gz_to_uncompressed, snpSift_filter</td>
      <td>Pairwise sequence alignment, Variant calling</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Gene-based Pathogen Identification</td>
      <td>https://usegalaxy.org.au</td>
      <td>ef8c22c2525063a2</td>
      <td>https://usegalaxy.org.au/published/workflow?id=ef8c22c2525063a2</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:pathogfair, name:iwc, name:microgalaxy</td>
      <td>2024-06-25</td>
      <td>2024-06-25</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>fasta2tab, abricate, flye, collection_element_identifiers, tp_find_and_replace, bandage_image, param_value_from_file, tab2fasta, medaka_consensus_pipeline, compose_text_param, split_file_to_collection, __BUILD_LIST__</td>
      <td>Base-calling, De-novo assembly, Sequence assembly, Variant calling, Sequence assembly visualisation, Antimicrobial resistance prediction, Genome assembly, Mapping assembly, Cross-assembly</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Taxonomy Profiling and Visualization with Krona</td>
      <td>https://usegalaxy.org.au</td>
      <td>d9ba165e6ae55417</td>
      <td>https://usegalaxy.org.au/published/workflow?id=d9ba165e6ae55417</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-06-25</td>
      <td>2024-06-25</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>taxonomy_krona_chart, krakentools_kreport2krona, kraken2</td>
      <td>Aggregation, Visualisation, Taxonomic classification</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Nanopore Preprocessing</td>
      <td>https://usegalaxy.org.au</td>
      <td>25d52afddaa3451b</td>
      <td>https://usegalaxy.org.au/published/workflow?id=25d52afddaa3451b</td>
      <td>Bérénice Batut, Engy Nasr, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:nanopore, name:iwc</td>
      <td>2024-06-25</td>
      <td>2024-06-25</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>Add_a_column1, porechop, collapse_dataset, __FILTER_FAILED_DATASETS__, samtools_fastx, kraken2, Cut1, nanoplot, krakentools_extract_kraken_reads, regexColumn1, minimap2, fastp, multiqc, collection_column_join, fastqc, Grep1, bamtools_split_mapped</td>
      <td>Aggregation, Data handling, Scatter plot plotting, Sequence alignment analysis, Sequence composition calculation, Sequencing quality control, Pairwise sequence alignment, Box-Whisker plot plotting, Visualisation, Sequence contamination filtering, Statistical calculation, Taxonomic classification, Validation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
  </tbody>
</table>

</div>