<script src="https://code.jquery.com/jquery-3.7.0.js"></script>
<script src="https://cdn.datatables.net/1.13.6/js/jquery.dataTables.min.js"></script>
<script src="https://cdn.datatables.net/searchbuilder/1.6.0/js/dataTables.searchBuilder.min.js"></script>
<script src="https://cdn.datatables.net/datetime/1.5.1/js/dataTables.dateTime.min.js"></script>

<script src="https://cdn.datatables.net/responsive/2.5.0/js/dataTables.responsive.min.js"></script>

<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/responsive/2.2.3/css/responsive.dataTables.min.css">

<link rel="stylesheet" href="https://cdn.datatables.net/1.13.6/css/jquery.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/searchbuilder/1.6.0/css/searchBuilder.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/datetime/1.5.1/css/dataTables.dateTime.min.css">

<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" />
<script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>

<style type="text/css">
  .tooltip {
    font-family: Georgia;
    font-size: 18px;
  }
  .tooltip .tooltip-inner {
    background-color: #00FFFF;
    color: #000000;
    min-width: 400px;
  }
  </style>

<script>
    $(document).ready(function() {
    $('#dataframe').DataTable( {
        "sScrollX": "100%",
        "dom": "Qlfrtip",

        responsive: {
            details: {
                type: 'column'
            }
        },
        "bAutoWidth": false,
        columnDefs: [ {
            className: 'control',
            orderable: false,
            targets:   [ 0 ],
            width: 100,
        },
        { type: 'natural', targets: 1 } // allow normal sorting
        ],

    });
});
</script>

<div class="display nowrap" style="width:80%" width="100%">

<table class="dataframe display nowrap" id="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th>Expand</th>
      <th>Name</th>
      <th>Source</th>
      <th>ID</th>
      <th>Link</th>
      <th>Creators</th>
      <th>Tags</th>
      <th>Creation time</th>
      <th>Update time</th>
      <th>Latest version</th>
      <th>Versions</th>
      <th>Number of steps</th>
      <th>Tools</th>
      <th>EDAM operations</th>
      <th>EDAM topics</th>
      <th>License</th>
      <th>DOI</th>
      <th>Projects</th>
      <th>To keep</th>
      <th>Deprecated</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td></td>
      <td>cgmlst-bacterial-genome/main</td>
      <td>WorkflowHub</td>
      <td>2055</td>
      <td>https://workflowhub.eu/workflows/2055?version=1</td>
      <td>ABRomics None, Clea Siguret, Hugo Lefeuvre</td>
      <td></td>
      <td>2026-01-21</td>
      <td>2026-01-21</td>
      <td>1</td>
      <td>1</td>
      <td>3.0</td>
      <td>coreprofiler_allele_calling, tooldistillator_summarize, tooldistillator</td>
      <td>Multilocus sequence typing, Data parsing</td>
      <td></td>
      <td>GPL-3.0-or-later</td>
      <td></td>
      <td>Intergalactic Workflow Commission (IWC)</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>bacterial-quality-and-contamination-control-post-assembly/main</td>
      <td>WorkflowHub</td>
      <td>1882</td>
      <td>https://workflowhub.eu/workflows/1882?version=3</td>
      <td>ABRomics None, Pierre Marin, Clea Siguret</td>
      <td></td>
      <td>2025-12-04</td>
      <td>2025-12-04</td>
      <td>3</td>
      <td>3</td>
      <td>14.0</td>
      <td>recentrifuge, collapse_dataset, est_abundance, tooldistillator, kraken2, pick_value, checkm2, quast, tooldistillator_summarize</td>
      <td>Visualisation, Taxonomic classification, Sequence composition calculation, Sequencing quality control, Cross-assembly, Statistical calculation, Validation, Data parsing, Expression analysis, Sequence assembly validation</td>
      <td></td>
      <td>GPL-3.0-or-later</td>
      <td></td>
      <td>Intergalactic Workflow Commission (IWC)</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>sars-cov-2-pe-illumina-artic-ivar-analysis/SARS-COV-2-ILLUMINA-AMPLICON-IVAR-PANGOLIN-NEXTCLADE</td>
      <td>WorkflowHub</td>
      <td>155</td>
      <td>https://workflowhub.eu/workflows/155?version=6</td>
      <td>Peter van Heusden</td>
      <td>virology</td>
      <td>2025-07-31</td>
      <td>2026-01-21</td>
      <td>6</td>
      <td>6</td>
      <td>16.0</td>
      <td>qualimap_bamqc, tp_sed_tool, snpeff_sars_cov_2, tp_cat, __FLATTEN__, samtools_view, nextclade, ivar_consensus, ivar_trim, ivar_variants, bwa_mem, samtools_stats, fastp, multiqc, pangolin</td>
      <td>Variant calling, Variant classification, Read mapping, Sequencing quality control, Primer removal, Generation, Sequence contamination filtering, Validation, SNP detection, Genome indexing, Methylation analysis, Sequence alignment, Tree-based sequence alignment</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>Intergalactic Workflow Commission (IWC)</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>sars-cov-2-pe-illumina-wgs-variant-calling/COVID-19-PE-WGS-ILLUMINA</td>
      <td>WorkflowHub</td>
      <td>113</td>
      <td>https://workflowhub.eu/workflows/113?version=4</td>
      <td>Wolfgang Maier</td>
      <td>virology, covid-19, covid19.galaxyproject.org, emergen_validated</td>
      <td>2021-12-21</td>
      <td>2025-12-12</td>
      <td>4</td>
      <td>4</td>
      <td>11.0</td>
      <td>lofreq_viterbi, picard_MarkDuplicates, snpeff_sars_cov_2, lofreq_filter, lofreq_call, samtools_view, bwa_mem, samtools_stats, fastp, multiqc, lofreq_indelqual</td>
      <td>Formatting, Read mapping, Sequencing quality control, Generation, Sequence contamination filtering, Validation, SNP detection, Genome indexing, Sequence alignment</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>Intergalactic Workflow Commission (IWC)</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>generic-non-segmented-viral-variant-calling/main</td>
      <td>WorkflowHub</td>
      <td>1876</td>
      <td>https://workflowhub.eu/workflows/1876?version=2</td>
      <td>Peter van Heusden, Wolfgang Maier</td>
      <td>virology</td>
      <td>2025-12-05</td>
      <td>2025-12-05</td>
      <td>2</td>
      <td>2</td>
      <td>23.0</td>
      <td>snpEff, pick_value, ivar_trim, samtools_stats, fastp, lofreq_viterbi, tp_sed_tool, tp_cat, ivar_consensus, bwa_mem, ivar_variants, __FILTER_FAILED_DATASETS__, snpSift_extractFields, samtools_view, map_param_value, qualimap_bamqc, calculate_numeric_param, __FLATTEN__, collapse_dataset, snpEff_build_gb, tp_find_and_replace, multiqc</td>
      <td>Read mapping, Generation, Sequencing quality control, Primer removal, Sequence contamination filtering, Validation, Genome indexing, Sequence alignment</td>
      <td></td>
      <td>AGPL-3.0-or-later</td>
      <td></td>
      <td>Intergalactic Workflow Commission (IWC)</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>influenza-isolates-consensus-and-subtyping/main</td>
      <td>WorkflowHub</td>
      <td>1260</td>
      <td>https://workflowhub.eu/workflows/1260?version=2</td>
      <td>Viktoria Isabel Schwarz, Wolfgang Maier, Aaron Kolbecher, Saim Momin</td>
      <td>virology</td>
      <td>2025-05-14</td>
      <td>2025-12-12</td>
      <td>2</td>
      <td>2</td>
      <td>68.0</td>
      <td>Filter1, Grep1, wc_gnu, rbc_mafft, fastp, snipit, __FILTER_FROM_FILE__, collection_element_identifiers, tp_cat, __DUPLICATE_FILE_TO_COLLECTION__, __UNZIP_COLLECTION__, tp_text_file_with_recurring_lines, ivar_consensus, bwa_mem, __MERGE_COLLECTION__, __FILTER_FAILED_DATASETS__, iqtree, __HARMONIZELISTS__, tp_easyjoin_tool, Paste1, samtools_view, Cut1, __RELABEL_FROM_FILE__, bamtools_split_ref, __APPLY_RULES__, seqtk_subseq, This seemingly no-op transformation will remove samples with no remaining segment results (cases where vapor failed for  every segment).\n__APPLY_RULES__, vapor, qualimap_bamqc, param_value_from_file, __FLATTEN__, collapse_dataset, Show beginning1, tp_find_and_replace, bwa mem will fail if no reference for any segment got suggested by vapor, i.e. the reference genome is empty. In that case, we need to skip further analysis of the sample.\n__FILTER_FAILED_DATASETS__</td>
      <td>De-novo assembly, Read mapping, Sequencing quality control, Primer removal, Generation, Sequence contamination filtering, Base position variability plotting, Data handling, Multiple sequence alignment, Genome indexing, Sequence analysis, Sequence alignment analysis, Sequence alignment, Phylogenetic analysis, Sequence file editing, Data retrieval</td>
      <td></td>
      <td>AGPL-3.0-or-later</td>
      <td></td>
      <td>Intergalactic Workflow Commission (IWC)</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>pox-virus-amplicon/main</td>
      <td>WorkflowHub</td>
      <td>439</td>
      <td>https://workflowhub.eu/workflows/439?version=4</td>
      <td>Viktoria Isabel Schwarz, Wolfgang Maier</td>
      <td>virology, pox</td>
      <td>2025-10-11</td>
      <td>2025-11-27</td>
      <td>4</td>
      <td>4</td>
      <td>40.0</td>
      <td>Grep1, __ZIP_COLLECTION__, samtools_merge, ivar_trim, samtools_stats, fastp, collection_element_identifiers, tp_sed_tool, tp_cat, compose_text_param, EMBOSS: maskseq51, ivar_consensus, bwa_mem, __FILTER_FAILED_DATASETS__, split_file_to_collection, Cut1, fasta_compute_length, samtools_view, datamash_ops, __APPLY_RULES__, __SORTLIST__, qualimap_bamqc, param_value_from_file, __FLATTEN__, multiqc</td>
      <td>Local alignment, Read mapping, Sequencing quality control, Primer removal, Generation, Sequence contamination filtering, Global alignment, Validation, Genome indexing, Sequence analysis, Sequence alignment analysis, Sequence alignment</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>Intergalactic Workflow Commission (IWC)</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>allele-based-pathogen-identification/main</td>
      <td>WorkflowHub</td>
      <td>1063</td>
      <td>https://workflowhub.eu/workflows/1063?version=5</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td></td>
      <td>2025-03-26</td>
      <td>2025-12-12</td>
      <td>5</td>
      <td>5</td>
      <td>23.0</td>
      <td>table_compute, bcftools_norm, clair3, samtools_depth, tp_cut_tool, snpSift_filter, collapse_dataset, Cut1, snpSift_extractFields, tp_head_tool, Paste1, regexColumn1, Count1, samtools_coverage, CONVERTER_gz_to_uncompressed, Remove beginning1, bcftools_consensus, minimap2</td>
      <td>Variant calling, Pairwise sequence alignment</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>Intergalactic Workflow Commission (IWC)</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>mgnify-amplicon-pipeline-v5-complete/main</td>
      <td>WorkflowHub</td>
      <td>1274</td>
      <td>https://workflowhub.eu/workflows/1274?version=2</td>
      <td>Rand Zoabi, Paul Zierep</td>
      <td></td>
      <td>2025-03-26</td>
      <td>2025-12-12</td>
      <td>2</td>
      <td>2</td>
      <td>20.0</td>
      <td>, tp_awk_tool, CONVERTER_uncompressed_to_gz, CONVERTER_gz_to_uncompressed, fastq_dl, __MERGE_COLLECTION__</td>
      <td></td>
      <td></td>
      <td>Apache-2.0</td>
      <td></td>
      <td>Intergalactic Workflow Commission (IWC)</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>mgnify-amplicon-pipeline-v5-its/main</td>
      <td>WorkflowHub</td>
      <td>1273</td>
      <td>https://workflowhub.eu/workflows/1273?version=2</td>
      <td>Rand Zoabi, Paul Zierep</td>
      <td></td>
      <td>2025-03-26</td>
      <td>2025-12-12</td>
      <td>2</td>
      <td>2</td>
      <td>30.0</td>
      <td>collection_element_identifiers, , biom_convert, __FILTER_EMPTY_DATASETS__, tp_awk_tool, mapseq, taxonomy_krona_chart, bedtools_maskfastabed, __FILTER_FROM_FILE__</td>
      <td>Visualisation, Formatting, k-mer counting, Mapping</td>
      <td></td>
      <td>Apache-2.0</td>
      <td></td>
      <td>Intergalactic Workflow Commission (IWC)</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>mgnify-amplicon-pipeline-v5-quality-control-paired-end/main</td>
      <td>WorkflowHub</td>
      <td>1272</td>
      <td>https://workflowhub.eu/workflows/1272?version=2</td>
      <td>Rand Zoabi, Paul Zierep</td>
      <td></td>
      <td>2025-03-26</td>
      <td>2025-12-12</td>
      <td>2</td>
      <td>2</td>
      <td>17.0</td>
      <td>trimmomatic, fastp, __UNZIP_COLLECTION__, fastq_to_fasta_python, fastq_filter, tp_find_and_replace, prinseq, cshl_fasta_formatter, fastqc, multiqc, mgnify_seqprep</td>
      <td>Sequence composition calculation, Read pre-processing, Sequence trimming, Nucleic acid design, Sequencing quality control, Statistical calculation, Sequence contamination filtering, Validation</td>
      <td></td>
      <td>Apache-2.0</td>
      <td></td>
      <td>Intergalactic Workflow Commission (IWC)</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>mgnify-amplicon-pipeline-v5-quality-control-single-end/main</td>
      <td>WorkflowHub</td>
      <td>1271</td>
      <td>https://workflowhub.eu/workflows/1271?version=3</td>
      <td>Rand Zoabi, Paul Zierep</td>
      <td></td>
      <td>2025-03-26</td>
      <td>2025-12-12</td>
      <td>3</td>
      <td>3</td>
      <td>14.0</td>
      <td>trimmomatic, fastq_to_fasta_python, tp_find_and_replace, fastq_filter, prinseq, cshl_fasta_formatter, fastqc, multiqc</td>
      <td>Sequence composition calculation, Read pre-processing, Sequence trimming, Sequencing quality control, Statistical calculation, Sequence contamination filtering, Validation</td>
      <td></td>
      <td>Apache-2.0</td>
      <td></td>
      <td>Intergalactic Workflow Commission (IWC)</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>mgnify-amplicon-pipeline-v5-rrna-prediction/main</td>
      <td>WorkflowHub</td>
      <td>1270</td>
      <td>https://workflowhub.eu/workflows/1270?version=2</td>
      <td>Rand Zoabi, Paul Zierep</td>
      <td></td>
      <td>2025-03-26</td>
      <td>2025-12-12</td>
      <td>2</td>
      <td>2</td>
      <td>47.0</td>
      <td>collection_element_identifiers, , gops_concat_1, biom_convert, __FILTER_EMPTY_DATASETS__, query_tabular, bedtools_getfastabed, tp_awk_tool, infernal_cmsearch, mapseq, cshl_fasta_formatter, cmsearch_deoverlap, taxonomy_krona_chart, __FILTER_FROM_FILE__</td>
      <td>Formatting, Comparison, k-mer counting, Mapping, Visualisation, Nucleic acid feature detection, Alignment</td>
      <td></td>
      <td>Apache-2.0</td>
      <td></td>
      <td>Intergalactic Workflow Commission (IWC)</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>mgnify-amplicon-taxonomic-summary-tables/main</td>
      <td>WorkflowHub</td>
      <td>1269</td>
      <td>https://workflowhub.eu/workflows/1269?version=2</td>
      <td>Rand Zoabi</td>
      <td></td>
      <td>2025-03-26</td>
      <td>2025-12-12</td>
      <td>2</td>
      <td>2</td>
      <td>10.0</td>
      <td>query_tabular, collection_column_join, tp_awk_tool, filter_tabular, Grouping1</td>
      <td></td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>Intergalactic Workflow Commission (IWC)</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>clinicalmp-discovery/main</td>
      <td>WorkflowHub</td>
      <td>1225</td>
      <td>https://workflowhub.eu/workflows/1225?version=1</td>
      <td>Subina Mehta</td>
      <td></td>
      <td>2024-12-10</td>
      <td>2025-11-27</td>
      <td>1</td>
      <td>1</td>
      <td>24.0</td>
      <td>fasta2tab, tp_cat, maxquant, Filter1, Grep1, dbbuilder, Cut1, query_tabular, msconvert, filter_tabular, ident_params, fasta_merge_files_and_filter_unique_sequences, search_gui, peptide_shaker, fasta_cli, Remove beginning1, Grouping1</td>
      <td>Clustering, Formatting, Imputation, Heat map generation, Filtering, Protein quantification, Principal component plotting, Statistical calculation, Standardisation and normalisation, Visualisation</td>
      <td></td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>Intergalactic Workflow Commission (IWC)</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>clinicalmp-data-interpretation/main</td>
      <td>WorkflowHub</td>
      <td>1219</td>
      <td>https://workflowhub.eu/workflows/1219?version=1</td>
      <td>GalaxyP</td>
      <td></td>
      <td>2024-11-26</td>
      <td>2025-11-27</td>
      <td>1</td>
      <td>1</td>
      <td>6.0</td>
      <td>Grep1, unipept, msstatstmt</td>
      <td>Visualisation, Prediction and recognition</td>
      <td></td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>Intergalactic Workflow Commission (IWC)</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>clinicalmp-verification/main</td>
      <td>WorkflowHub</td>
      <td>1218</td>
      <td>https://workflowhub.eu/workflows/1218?version=1</td>
      <td>Pratik Jagtap</td>
      <td></td>
      <td>2024-11-26</td>
      <td>2025-11-27</td>
      <td>1</td>
      <td>1</td>
      <td>19.0</td>
      <td>tp_cat, uniprotxml_downloader, Filter1, dbbuilder, Cut1, collapse_dataset, query_tabular, fasta_merge_files_and_filter_unique_sequences, pepquery2, Remove beginning1, Grouping1</td>
      <td></td>
      <td></td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>Intergalactic Workflow Commission (IWC)</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>clinicalmp-database-generation/main</td>
      <td>WorkflowHub</td>
      <td>1216</td>
      <td>https://workflowhub.eu/workflows/1216?version=1</td>
      <td>Subina Mehta</td>
      <td></td>
      <td>2024-11-23</td>
      <td>2025-11-27</td>
      <td>1</td>
      <td>1</td>
      <td>3.0</td>
      <td>fasta_merge_files_and_filter_unique_sequences, metanovo</td>
      <td>Tag-based peptide identification, de Novo sequencing, Target-Decoy, Protein identification, Expression analysis</td>
      <td></td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>Intergalactic Workflow Commission (IWC)</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>clinicalmp-quantitation/main</td>
      <td>WorkflowHub</td>
      <td>1177</td>
      <td>https://workflowhub.eu/workflows/1177?version=1</td>
      <td>GalaxyP</td>
      <td></td>
      <td>2024-10-07</td>
      <td>2025-12-12</td>
      <td>1</td>
      <td>1</td>
      <td>7.0</td>
      <td>maxquant, Quantified-Proteins\nGrouping1, Quantified-Peptides\nGrouping1, extracting microbial Proteins\nGrep1, extract proteins\nCut1, extract peptides\nCut1, extracting microbial Peptides\nGrep1</td>
      <td>Clustering, Imputation, Heat map generation, Protein quantification, Principal component plotting, Statistical calculation, Standardisation and normalisation, Visualisation</td>
      <td></td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>Intergalactic Workflow Commission (IWC)</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>bacterial-genome-assembly/main</td>
      <td>WorkflowHub</td>
      <td>1043</td>
      <td>https://workflowhub.eu/workflows/1043?version=12</td>
      <td>Abromics None, Pierre Marin, Clea Siguret</td>
      <td>abromics, assembly, fastq, genomics, bacterial-genomics, paired-end, quality</td>
      <td>2025-12-04</td>
      <td>2025-12-04</td>
      <td>12</td>
      <td>12</td>
      <td>5.0</td>
      <td>bandage_image, tooldistillator, bandage_info, shovill, tooldistillator_summarize</td>
      <td>Genome assembly, Sequence assembly visualisation, Data parsing</td>
      <td></td>
      <td>GPL-3.0-or-later</td>
      <td></td>
      <td>Intergalactic Workflow Commission (IWC)</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>quality-and-contamination-control/main</td>
      <td>WorkflowHub</td>
      <td>1052</td>
      <td>https://workflowhub.eu/workflows/1052?version=11</td>
      <td>ABRomics None, Pierre Marin, Clea Siguret</td>
      <td>abromics, fastq, genomics, bacterial-genomics, paired-end, quality, taxonomy-assignment, trimming</td>
      <td>2025-06-19</td>
      <td>2025-12-12</td>
      <td>11</td>
      <td>11</td>
      <td>8.0</td>
      <td>recentrifuge, est_abundance, __ZIP_COLLECTION__, __UNZIP_COLLECTION__, kraken2, tooldistillator, tooldistillator_summarize, fastp</td>
      <td>Taxonomic classification, Cross-assembly, Data parsing, Sequencing quality control, Statistical calculation, Sequence contamination filtering, Expression analysis</td>
      <td></td>
      <td>GPL-3.0-or-later</td>
      <td></td>
      <td>Intergalactic Workflow Commission (IWC)</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>qiime2-III-VI-downsteam/QIIME2-III-V-Phylogeny-Rarefaction-Taxonomic-Analysis</td>
      <td>WorkflowHub</td>
      <td>1090</td>
      <td>https://workflowhub.eu/workflows/1090?version=2</td>
      <td>Debjyoti Ghosh</td>
      <td></td>
      <td>2024-11-26</td>
      <td>2025-12-12</td>
      <td>2</td>
      <td>2</td>
      <td>3.0</td>
      <td>Phylogenetic tree for diversity analysis, Taxonomic analysis, Rarefaction</td>
      <td></td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>Intergalactic Workflow Commission (IWC)</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>gene-based-pathogen-identification/main</td>
      <td>WorkflowHub</td>
      <td>1062</td>
      <td>https://workflowhub.eu/workflows/1062?version=1</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td></td>
      <td>2024-06-26</td>
      <td>2025-11-27</td>
      <td>1</td>
      <td>1</td>
      <td>15.0</td>
      <td>collection_element_identifiers, fasta2tab, param_value_from_file, split_file_to_collection, __BUILD_LIST__, compose_text_param, tab2fasta, medaka_consensus_pipeline, tp_find_and_replace, abricate, bandage_image, flye</td>
      <td>Variant calling, De-novo assembly, Sequence assembly, Cross-assembly, Sequence assembly visualisation, Antimicrobial resistance prediction, Genome assembly, Base-calling, Mapping assembly</td>
      <td></td>
      <td>MIT</td>
      <td>10.48546/workflowhub.workflow.1062.1</td>
      <td>Intergalactic Workflow Commission (IWC)</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>pathogen-detection-pathogfair-samples-aggregation-and-visualisation/main</td>
      <td>WorkflowHub</td>
      <td>1060</td>
      <td>https://workflowhub.eu/workflows/1060?version=1</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td></td>
      <td>2024-06-26</td>
      <td>2025-11-27</td>
      <td>1</td>
      <td>1</td>
      <td>60.0</td>
      <td>tp_sorted_uniq, tab2fasta, collection_column_join, collapse_dataset, tab2fasta, regex1, fasttree, __FILTER_FAILED_DATASETS__, ggplot2_heatmap, fasta2tab, tp_multijoin_tool, regex1, __FILTER_FAILED_DATASETS__, Cut1, Count1, ggplot2_heatmap, tp_split_on_column, Grouping1, fasta2tab, regexColumn1, __FILTER_EMPTY_DATASETS__, collapse_dataset, tp_replace_in_column, bedtools_getfastabed, newick_display, fasta_merge_files_and_filter_unique_sequences, Remove beginning1, clustalw, fasttree</td>
      <td>Multiple sequence alignment, Phylogenetic tree generation (maximum likelihood and Bayesian methods), Phylogenetic tree reconstruction, Mapping, Phylogenetic tree generation (from molecular sequences), Visualisation, Phylogenetic tree generation, Phylogenetic tree analysis</td>
      <td></td>
      <td>MIT</td>
      <td>10.48546/workflowhub.workflow.1060.1</td>
      <td>Intergalactic Workflow Commission (IWC)</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>taxonomy-profiling-and-visualization-with-krona/main</td>
      <td>WorkflowHub</td>
      <td>1059</td>
      <td>https://workflowhub.eu/workflows/1059?version=1</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td></td>
      <td>2024-06-26</td>
      <td>2025-11-27</td>
      <td>1</td>
      <td>1</td>
      <td>3.0</td>
      <td>krakentools_kreport2krona, kraken2, taxonomy_krona_chart</td>
      <td>Visualisation, Taxonomic classification, Aggregation</td>
      <td></td>
      <td>MIT</td>
      <td>10.48546/workflowhub.workflow.1059.1</td>
      <td>Intergalactic Workflow Commission (IWC)</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>bacterial_genome_annotation/main</td>
      <td>WorkflowHub</td>
      <td>1050</td>
      <td>https://workflowhub.eu/workflows/1050?version=13</td>
      <td>ABRomics None, Pierre Marin, Clea Siguret</td>
      <td>abromics, annotation, genomics, bacterial-genomics, fasta, genome-annotation</td>
      <td>2025-12-06</td>
      <td>2025-12-06</td>
      <td>13</td>
      <td>13</td>
      <td>6.0</td>
      <td>tooldistillator, isescan, bakta, plasmidfinder, tooldistillator_summarize, integron_finder</td>
      <td>Protein feature detection, Nucleic acid feature detection, Genome assembly, Data parsing, Sequence motif recognition, Genome annotation, Multilocus sequence typing, Scaffolding, Structural variation detection</td>
      <td></td>
      <td>GPL-3.0-or-later</td>
      <td></td>
      <td>Intergalactic Workflow Commission (IWC)</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>amr_gene_detection/main</td>
      <td>WorkflowHub</td>
      <td>1049</td>
      <td>https://workflowhub.eu/workflows/1049?version=9</td>
      <td>ABRomics None, Pierre Marin, Clea Siguret</td>
      <td>abromics, amr, amr-detection, genomics, antibiotic-resistance, antimicrobial resistance, antimicrobial-resistance-genes, bacterial-genomics, fasta</td>
      <td>2025-12-04</td>
      <td>2025-12-04</td>
      <td>9</td>
      <td>9</td>
      <td>5.0</td>
      <td>tooldistillator, staramr_search, amrfinderplus, abricate, tooldistillator_summarize</td>
      <td>Antimicrobial resistance prediction, Data parsing</td>
      <td></td>
      <td>GPL-3.0-or-later</td>
      <td></td>
      <td>Intergalactic Workflow Commission (IWC)</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>dada2/main</td>
      <td>WorkflowHub</td>
      <td>790</td>
      <td>https://workflowhub.eu/workflows/790?version=3</td>
      <td>Matthias Bernt</td>
      <td>name:amplicon</td>
      <td>2025-03-26</td>
      <td>2025-12-12</td>
      <td>3</td>
      <td>3</td>
      <td>14.0</td>
      <td>dada2_mergePairs, dada2_removeBimeraDenovo, dada2_plotQualityProfile, dada2_filterAndTrim, dada2_makeSequenceTable, dada2_seqCounts, __UNZIP_COLLECTION__, dada2_assignTaxonomyAddspecies, __APPLY_RULES__, dada2_learnErrors, dada2_dada</td>
      <td>Variant calling, DNA barcoding</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>Intergalactic Workflow Commission (IWC)</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Refining Genome Annotations with Apollo (prokaryotes)</td>
      <td>WorkflowHub</td>
      <td>749</td>
      <td>https://workflowhub.eu/workflows/749?version=1</td>
      <td>Anthony Bretaudeau</td>
      <td>genome-annotation</td>
      <td>2024-02-14</td>
      <td>2025-11-04</td>
      <td>1</td>
      <td>1</td>
      <td>5.0</td>
      <td>iframe, create_or_update, create_account, jbrowse, list_organism</td>
      <td>Genome visualisation</td>
      <td></td>
      <td>CC-BY-4.0</td>
      <td>10.48546/workflowhub.workflow.749.1</td>
      <td>EuroScienceGateway</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>TB Variant Analysis v1.0</td>
      <td>WorkflowHub</td>
      <td>1647</td>
      <td>https://workflowhub.eu/workflows/1647?version=2</td>
      <td></td>
      <td>gtn, galaxy, pathogen, tuberculosis</td>
      <td>2026-01-12</td>
      <td>2026-01-12</td>
      <td>2</td>
      <td>2</td>
      <td>15.0</td>
      <td>qualimap_bamqc, bcftools_consensus, tp_sed_tool, mosdepth, __FLATTEN__, tb_variant_filter, snippy, EMBOSS: seqret84, kraken2, tp_awk_tool, tbvcfreport, tb_profiler_profile, fastp, multiqc</td>
      <td>Variant calling, Local alignment, Taxonomic classification, Sequencing quality control, Phylogenetic tree generation, Sequence contamination filtering, Global alignment, Antimicrobial resistance prediction, Validation, Sequence analysis, Sequence alignment analysis, Sequence alignment, Phylogenetic tree visualisation</td>
      <td></td>
      <td>AGPL-3.0-or-later</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Quality and contamination control in bacterial isolate using Illumina MiSeq Data</td>
      <td>WorkflowHub</td>
      <td>2043</td>
      <td>https://workflowhub.eu/workflows/2043?version=1</td>
      <td></td>
      <td>gtn, galaxy, sequence-analysis</td>
      <td>2025-12-22</td>
      <td>2025-12-22</td>
      <td>1</td>
      <td>1</td>
      <td>5.0</td>
      <td>recentrifuge, est_abundance, kraken2, falco, fastp</td>
      <td>Taxonomic classification, Read mapping, Sequencing quality control, Cross-assembly, Statistical calculation, Sequence contamination filtering, Expression analysis, Visualisation</td>
      <td></td>
      <td>GPL-3.0-or-later</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Taxonomic Profiling and Visualization of Metagenomic Data</td>
      <td>WorkflowHub</td>
      <td>1470</td>
      <td>https://workflowhub.eu/workflows/1470?version=2</td>
      <td></td>
      <td>gtn, galaxy, microbiome</td>
      <td>2025-12-22</td>
      <td>2025-12-22</td>
      <td>2</td>
      <td>2</td>
      <td>10.0</td>
      <td>interactive_tool_phinch, interactive_tool_pavian, est_abundance, __UNZIP_COLLECTION__, krakentools_kreport2krona, metaphlan, interactive_tool_phinch, kraken_biom, kraken2, kraken_biom, metaphlan, taxonomy_krona_chart</td>
      <td>Taxonomic classification, Genome annotation, Visualisation, Statistical calculation, Aggregation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Checking expected species and contamination in bacterial isolate</td>
      <td>WorkflowHub</td>
      <td>1674</td>
      <td>https://workflowhub.eu/workflows/1674?version=2</td>
      <td></td>
      <td>ecology, gtn, galaxy</td>
      <td>2025-12-22</td>
      <td>2025-12-22</td>
      <td>2</td>
      <td>2</td>
      <td>3.0</td>
      <td>est_abundance, kraken2, recentrifuge</td>
      <td>Taxonomic classification, Cross-assembly, Statistical calculation, Expression analysis</td>
      <td></td>
      <td>GPL-3.0-or-later</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Metagenomic Binning</td>
      <td>WorkflowHub</td>
      <td>2028</td>
      <td>https://workflowhub.eu/workflows/2028?version=1</td>
      <td></td>
      <td>gtn, galaxy, microgalaxy, binning, microbiome</td>
      <td>2025-12-08</td>
      <td>2025-12-08</td>
      <td>1</td>
      <td>1</td>
      <td>17.0</td>
      <td>metabat2_jgi_summarize_bam_contig_depths, semibin, concoct_cut_up_fasta, concoct_extract_fasta_bins, bowtie2, __BUILD_LIST__, concoct_coverage_table, samtools_sort, maxbin2, Fasta_to_Contig2Bin, concoct, concoct_merge_cut_up_clustering, metabat2, binette</td>
      <td>Read binning, Genome annotation, Sequence assembly, Read mapping, Sequence clustering</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>MGnify's amplicon pipeline v5.0 - ITS</td>
      <td>WorkflowHub</td>
      <td>1856</td>
      <td>https://workflowhub.eu/workflows/1856?version=1</td>
      <td></td>
      <td>amplicon, gtn, galaxy, metagenomics, mgnify_amplicon, name:microgalaxy</td>
      <td>2025-08-11</td>
      <td>2025-08-11</td>
      <td>1</td>
      <td>1</td>
      <td>30.0</td>
      <td>collection_element_identifiers, , biom_convert, __FILTER_EMPTY_DATASETS__, tp_awk_tool, mapseq, taxonomy_krona_chart, bedtools_maskfastabed, __FILTER_FROM_FILE__</td>
      <td>Visualisation, Formatting, k-mer counting, Mapping</td>
      <td></td>
      <td>Apache-2.0</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>MAPseq to ampvis2</td>
      <td>WorkflowHub</td>
      <td>1855</td>
      <td>https://workflowhub.eu/workflows/1855?version=1</td>
      <td></td>
      <td>gtn, galaxy, microbiome</td>
      <td>2025-08-11</td>
      <td>2025-08-11</td>
      <td>1</td>
      <td>1</td>
      <td>9.0</td>
      <td>ampvis2_load, query_tabular, collapse_dataset, tp_awk_tool, collection_column_join</td>
      <td>Visualisation, Analysis</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>MGnify's amplicon pipeline v5.0 - Quality control PE</td>
      <td>WorkflowHub</td>
      <td>1854</td>
      <td>https://workflowhub.eu/workflows/1854?version=1</td>
      <td></td>
      <td>amplicon, gtn, galaxy, metagenomics, mgnify_amplicon, name:microgalaxy</td>
      <td>2025-08-11</td>
      <td>2025-08-11</td>
      <td>1</td>
      <td>1</td>
      <td>17.0</td>
      <td>trimmomatic, fastp, __UNZIP_COLLECTION__, fastq_to_fasta_python, fastq_filter, tp_find_and_replace, prinseq, cshl_fasta_formatter, fastqc, multiqc, mgnify_seqprep</td>
      <td>Sequence composition calculation, Read pre-processing, Sequence trimming, Nucleic acid design, Sequencing quality control, Statistical calculation, Sequence contamination filtering, Validation</td>
      <td></td>
      <td>Apache-2.0</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>MGnify's amplicon pipeline v5.0</td>
      <td>WorkflowHub</td>
      <td>1853</td>
      <td>https://workflowhub.eu/workflows/1853?version=1</td>
      <td></td>
      <td>amplicon, gtn, galaxy, metagenomics, mgnify_amplicon, name:microgalaxy</td>
      <td>2025-08-11</td>
      <td>2025-08-11</td>
      <td>1</td>
      <td>1</td>
      <td>20.0</td>
      <td>, tp_awk_tool, CONVERTER_uncompressed_to_gz, CONVERTER_gz_to_uncompressed, fastq_dl, __MERGE_COLLECTION__</td>
      <td></td>
      <td></td>
      <td>Apache-2.0</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>MGnify amplicon summary tables</td>
      <td>WorkflowHub</td>
      <td>1851</td>
      <td>https://workflowhub.eu/workflows/1851?version=1</td>
      <td></td>
      <td>amplicon, gtn, galaxy, metagenomics, mgnify_amplicon, name:microgalaxy</td>
      <td>2025-08-11</td>
      <td>2025-08-11</td>
      <td>1</td>
      <td>1</td>
      <td>10.0</td>
      <td>query_tabular, collection_column_join, tp_awk_tool, filter_tabular, Grouping1</td>
      <td></td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>MGnify's amplicon pipeline v5.0 - Quality control SE</td>
      <td>WorkflowHub</td>
      <td>1850</td>
      <td>https://workflowhub.eu/workflows/1850?version=1</td>
      <td></td>
      <td>amplicon, gtn, galaxy, metagenomics, mgnify_amplicon, name:microgalaxy</td>
      <td>2025-08-11</td>
      <td>2025-08-11</td>
      <td>1</td>
      <td>1</td>
      <td>14.0</td>
      <td>trimmomatic, fastq_to_fasta_python, tp_find_and_replace, fastq_filter, prinseq, cshl_fasta_formatter, fastqc, multiqc</td>
      <td>Sequence composition calculation, Read pre-processing, Sequence trimming, Sequencing quality control, Statistical calculation, Sequence contamination filtering, Validation</td>
      <td></td>
      <td>Apache-2.0</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>MGnify's amplicon pipeline v5.0 - rRNA prediction</td>
      <td>WorkflowHub</td>
      <td>1842</td>
      <td>https://workflowhub.eu/workflows/1842?version=1</td>
      <td></td>
      <td>amplicon, gtn, galaxy, metagenomics, mgnify_amplicon, name:microgalaxy</td>
      <td>2025-08-04</td>
      <td>2025-08-11</td>
      <td>1</td>
      <td>1</td>
      <td>47.0</td>
      <td>collection_element_identifiers, , gops_concat_1, biom_convert, __FILTER_EMPTY_DATASETS__, query_tabular, bedtools_getfastabed, tp_awk_tool, infernal_cmsearch, mapseq, cshl_fasta_formatter, cmsearch_deoverlap, taxonomy_krona_chart, __FILTER_FROM_FILE__</td>
      <td>Formatting, Comparison, k-mer counting, Mapping, Visualisation, Nucleic acid feature detection, Alignment</td>
      <td></td>
      <td>Apache-2.0</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow 3: Functional Information (quick)</td>
      <td>WorkflowHub</td>
      <td>1447</td>
      <td>https://workflowhub.eu/workflows/1447?version=2</td>
      <td></td>
      <td>asaim, gtn, galaxy, metagenomics</td>
      <td>2025-07-14</td>
      <td>2025-07-14</td>
      <td>2</td>
      <td>2</td>
      <td>12.0</td>
      <td>humann_renorm_table, Grep1, Cut1, humann_unpack_pathways, humann_split_stratified_table, tp_find_and_replace, combine_metaphlan2_humann2, humann_regroup_table, humann_rename_table</td>
      <td>Taxonomic classification, Phylogenetic analysis, Species frequency estimation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Building an amplicon sequence variant (ASV) table from 16S data using DADA2</td>
      <td>WorkflowHub</td>
      <td>1395</td>
      <td>https://workflowhub.eu/workflows/1395?version=2</td>
      <td></td>
      <td>gtn, galaxy, microbiome</td>
      <td>2025-07-07</td>
      <td>2025-07-07</td>
      <td>2</td>
      <td>2</td>
      <td>21.0</td>
      <td>collection_element_identifiers, dada2_mergePairs, dada2_removeBimeraDenovo, tp_replace_in_line, dada2_plotQualityProfile, dada2_filterAndTrim, dada2_makeSequenceTable, dada2_seqCounts, tp_replace_in_column, __UNZIP_COLLECTION__, cat1, dada2_assignTaxonomyAddspecies, Add_a_column1, dada2_learnErrors, phyloseq_from_dada2, dada2_dada, __SORTLIST__, tp_head_tool</td>
      <td>Variant calling, Visualisation, DNA barcoding, Analysis, Variant calling, Visualisation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Metatranscriptomics analysis using microbiome RNA-seq data</td>
      <td>WorkflowHub</td>
      <td>1466</td>
      <td>https://workflowhub.eu/workflows/1466?version=2</td>
      <td></td>
      <td>gtn, galaxy, microbiome</td>
      <td>2025-06-23</td>
      <td>2025-06-23</td>
      <td>2</td>
      <td>2</td>
      <td>33.0</td>
      <td>Grep1, humann_unpack_pathways, cutadapt, fastqc, tp_sort_header_tool, humann, graphlan, humann_regroup_table, export2graphlan, combine_metaphlan_humann, fastq_paired_end_interlacer, graphlan_annotate, Cut1, humann_split_stratified_table, Grouping1, humann_renorm_table, bg_sortmerna, tp_find_and_replace, metaphlan, humann_rename_table, taxonomy_krona_chart, multiqc</td>
      <td>Taxonomic classification, Sequence composition calculation, Sequence trimming, Read pre-processing, Phylogenetic inference, Sequence alignment analysis, Visualisation, Sequence similarity search, Sequencing quality control, Primer removal, Species frequency estimation, Statistical calculation, Phylogenetic tree visualisation, Validation, Phylogenetic tree editing, Phylogenetic analysis, Conversion, Sequence comparison, Aggregation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Metatranscriptomics analysis using microbiome RNA-seq data - Workflow 2: Community profile</td>
      <td>WorkflowHub</td>
      <td>1451</td>
      <td>https://workflowhub.eu/workflows/1451?version=2</td>
      <td></td>
      <td>gtn, galaxy, microbiome</td>
      <td>2025-06-23</td>
      <td>2025-06-23</td>
      <td>2</td>
      <td>2</td>
      <td>6.0</td>
      <td>graphlan_annotate, Cut1, export2graphlan, graphlan, metaphlan, taxonomy_krona_chart</td>
      <td>Taxonomic classification, Conversion, Phylogenetic inference, Phylogenetic tree editing, Visualisation, Phylogenetic tree editing, Conversion</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Metatranscriptomics analysis using microbiome RNA-seq data - Workflow 1: Preprocessing</td>
      <td>WorkflowHub</td>
      <td>1444</td>
      <td>https://workflowhub.eu/workflows/1444?version=2</td>
      <td></td>
      <td>gtn, galaxy, microbiome</td>
      <td>2025-06-23</td>
      <td>2025-06-23</td>
      <td>2</td>
      <td>2</td>
      <td>6.0</td>
      <td>bg_sortmerna, cutadapt, fastqc, cutadapt, multiqc, fastq_paired_end_interlacer</td>
      <td>Sequence similarity search, Sequence composition calculation, Sequence trimming, Sequencing quality control, Primer removal, Statistical calculation, Read pre-processing, Validation, Sequence alignment analysis, Sequence comparison</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Metatranscriptomics analysis using microbiome RNA-seq data - Workflow 3: Functional Information</td>
      <td>WorkflowHub</td>
      <td>1456</td>
      <td>https://workflowhub.eu/workflows/1456?version=2</td>
      <td></td>
      <td>gtn, galaxy, microbiome</td>
      <td>2025-06-23</td>
      <td>2025-06-23</td>
      <td>2</td>
      <td>2</td>
      <td>18.0</td>
      <td>tp_sort_header_tool, humann_renorm_table, humann, Grep1, humann_unpack_pathways, humann_split_stratified_table, tp_find_and_replace, humann_regroup_table, humann_rename_table</td>
      <td>Taxonomic classification, Phylogenetic analysis, Species frequency estimation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly of metagenomic sequencing data</td>
      <td>WorkflowHub</td>
      <td>1390</td>
      <td>https://workflowhub.eu/workflows/1390?version=2</td>
      <td></td>
      <td>gtn, galaxy, microbiome</td>
      <td>2025-06-16</td>
      <td>2025-06-16</td>
      <td>2</td>
      <td>2</td>
      <td>9.0</td>
      <td>megahit, coverm_contig, bowtie2, megahit_contig2fastg, bandage_info, collection_column_join, quast, metaspades, bandage_image</td>
      <td>Visualisation, Sequence assembly validation, Local alignment, Read mapping, Sequence assembly visualisation, Genome assembly</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Taxonomic Analysis of eDNA</td>
      <td>WorkflowHub</td>
      <td>1723</td>
      <td>https://workflowhub.eu/workflows/1723?version=1</td>
      <td></td>
      <td>ecology, gtn, galaxy</td>
      <td>2025-06-02</td>
      <td>2025-06-02</td>
      <td>1</td>
      <td>1</td>
      <td>8.0</td>
      <td>Count1, cshl_fastq_to_fasta, fastp, ncbi_blastn_wrapper</td>
      <td>Sequencing quality control, Sequence contamination filtering</td>
      <td></td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Quality and contamination control in bacterial isolate using Illumina MiSeq Data</td>
      <td>WorkflowHub</td>
      <td>1644</td>
      <td>https://workflowhub.eu/workflows/1644?version=1</td>
      <td></td>
      <td>gtn, galaxy, sequence-analysis</td>
      <td>2025-06-02</td>
      <td>2025-06-02</td>
      <td>1</td>
      <td>1</td>
      <td>6.0</td>
      <td>recentrifuge, est_abundance, kraken2, falco, fastp</td>
      <td>Taxonomic classification, Read mapping, Sequencing quality control, Cross-assembly, Statistical calculation, Sequence contamination filtering, Expression analysis, Visualisation</td>
      <td></td>
      <td>GPL-3.0-or-later</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>pox-virus-tiled-amplicon-ref-masking</td>
      <td>WorkflowHub</td>
      <td>1632</td>
      <td>https://workflowhub.eu/workflows/1632?version=1</td>
      <td></td>
      <td>gtn, galaxy, variant-analysis</td>
      <td>2025-06-02</td>
      <td>2025-06-02</td>
      <td>1</td>
      <td>1</td>
      <td>14.0</td>
      <td>param_value_from_file, Grep1, Cut1, fasta_compute_length, compose_text_param, datamash_ops, EMBOSS: maskseq51, Add_a_column1</td>
      <td>Local alignment, Sequence analysis, Sequence alignment analysis, Sequence alignment, Global alignment</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Copy Of GTN Training - Antibiotic Resistance Detection</td>
      <td>WorkflowHub</td>
      <td>1477</td>
      <td>https://workflowhub.eu/workflows/1477?version=1</td>
      <td></td>
      <td>gtn, galaxy, microbiome</td>
      <td>2025-06-02</td>
      <td>2025-06-02</td>
      <td>1</td>
      <td>1</td>
      <td>12.0</td>
      <td>unicycler, staramr_search, nanoplot, PlasFlow, gfa_to_fa, miniasm, bandage_image, minimap2, racon</td>
      <td>De-novo assembly, Pairwise sequence alignment, Box-Whisker plot plotting, Scatter plot plotting, Sequence assembly visualisation, Antimicrobial resistance prediction, Genome assembly, Pairwise sequence alignment, Sequence analysis, Mapping assembly, Aggregation</td>
      <td></td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Amplicon Tutorial</td>
      <td>WorkflowHub</td>
      <td>1476</td>
      <td>https://workflowhub.eu/workflows/1476?version=1</td>
      <td></td>
      <td>gtn, galaxy, microbiome</td>
      <td>2025-06-02</td>
      <td>2025-06-02</td>
      <td>1</td>
      <td>1</td>
      <td>17.0</td>
      <td>mothur_pre_cluster, mothur_unique_seqs, mothur_cluster_split, krona-text, mothur_summary_seqs, mothur_merge_files, mothur_count_seqs, mothur_screen_seqs, mothur_make_shared, mothur_classify_otu, mothur_classify_seqs, mothur_filter_seqs, mothur_align_seqs, mothur_make_group, mothur_make_biom</td>
      <td>Taxonomic classification, Phylogenetic analysis, Sequencing quality control, Sequence read processing, Visualisation, DNA barcoding, Sequence clustering</td>
      <td></td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Training: 16S rRNA Analysis with Nanopore Sequencing Reads</td>
      <td>WorkflowHub</td>
      <td>1473</td>
      <td>https://workflowhub.eu/workflows/1473?version=1</td>
      <td></td>
      <td>gtn, galaxy, microbiome</td>
      <td>2025-06-02</td>
      <td>2025-06-02</td>
      <td>1</td>
      <td>1</td>
      <td>11.0</td>
      <td>fastp, fastqc, tp_replace_in_line, kraken2, datamash_reverse, porechop, Remove beginning1, fastp, taxonomy_krona_chart, multiqc</td>
      <td>Taxonomic classification, Sequence composition calculation, Visualisation, Sequencing quality control, Statistical calculation, Sequence contamination filtering, Validation</td>
      <td></td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>WGS Part In "Analyses Of Metagenomics Data - The Global Picture"</td>
      <td>WorkflowHub</td>
      <td>1472</td>
      <td>https://workflowhub.eu/workflows/1472?version=1</td>
      <td></td>
      <td>gtn, galaxy, microbiome</td>
      <td>2025-06-02</td>
      <td>2025-06-02</td>
      <td>1</td>
      <td>1</td>
      <td>7.0</td>
      <td>metaphlan2krona, humann2, humann2_renorm_table, humann2_regroup_table, metaphlan2, taxonomy_krona_chart</td>
      <td>Visualisation</td>
      <td></td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Query a metaplasmidome database to identify and annotate plasmids in metagenomes</td>
      <td>WorkflowHub</td>
      <td>1469</td>
      <td>https://workflowhub.eu/workflows/1469?version=1</td>
      <td></td>
      <td>gtn, galaxy, metagenomics, metaplasmidome, name:microgalaxy</td>
      <td>2025-06-02</td>
      <td>2025-06-02</td>
      <td>1</td>
      <td>1</td>
      <td>47.0</td>
      <td>ggplot2_histogram, count_gff_features, histogram_rpy, Filter1, Grouping1, Cut1, tp_replace_in_column, cat1, tp_sorted_uniq, join1, tab2fasta, sort1, tp_tail_tool, add_column_headers, Add_a_column1, CONVERTER_fasta_to_tabular, MQoutputfilter, minimap2</td>
      <td>Visualisation, Pairwise sequence alignment</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Training: 16S rRNA Sequencing With Mothur: Main Tutorial</td>
      <td>WorkflowHub</td>
      <td>1465</td>
      <td>https://workflowhub.eu/workflows/1465?version=1</td>
      <td></td>
      <td>gtn, galaxy, microbiome</td>
      <td>2025-06-02</td>
      <td>2025-06-02</td>
      <td>1</td>
      <td>1</td>
      <td>38.0</td>
      <td>mothur_summary_seqs, mothur_make_contigs, mothur_make_shared, mothur_classify_seqs, mothur_filter_seqs, mothur_chimera_vsearch, mothur_align_seqs, mothur_sub_sample, mothur_make_biom, mothur_venn, mothur_seq_error, mothur_summary_single, mothur_heatmap_sim, mothur_count_seqs, mothur_classify_otu, mothur_count_groups, mothur_taxonomy_to_krona, XY_Plot_1, mothur_remove_groups, mothur_pre_cluster, mothur_unique_seqs, mothur_remove_seqs, mothur_cluster_split, mothur_get_groups, mothur_rarefaction_single, mothur_dist_shared, mothur_tree_shared, mothur_screen_seqs, mothur_remove_lineage, newick_display, mothur_cluster, mothur_dist_seqs, taxonomy_krona_chart</td>
      <td>Taxonomic classification, Phylogenetic analysis, Phylogenetic tree reconstruction, Sequencing quality control, Phylogenetic tree generation, Sequence read processing, Phylogenetic tree analysis, Visualisation, DNA barcoding, Sequence clustering</td>
      <td></td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow for Identifying MF from ITS2 sequencing using LotuS2 - tutorial example run'</td>
      <td>WorkflowHub</td>
      <td>1460</td>
      <td>https://workflowhub.eu/workflows/1460?version=1</td>
      <td></td>
      <td>ecology, fungi, gtn, galaxy, lotus2, metagenomics</td>
      <td>2025-06-02</td>
      <td>2025-06-02</td>
      <td>1</td>
      <td>1</td>
      <td>1.0</td>
      <td>lotus2</td>
      <td>Sequence feature detection, DNA barcoding</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Identification of the micro-organisms in a beer using Nanopore sequencing</td>
      <td>WorkflowHub</td>
      <td>1439</td>
      <td>https://workflowhub.eu/workflows/1439?version=1</td>
      <td></td>
      <td>gtn, galaxy, microbiome</td>
      <td>2025-06-02</td>
      <td>2025-06-02</td>
      <td>1</td>
      <td>1</td>
      <td>8.0</td>
      <td>fastp, fastqc, Filter1, kraken2, krakentools_kreport2krona, porechop, fastp, taxonomy_krona_chart</td>
      <td>Taxonomic classification, Sequence composition calculation, Visualisation, Sequencing quality control, Statistical calculation, Sequence contamination filtering, Aggregation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Calculating diversity from microbiome taxonomic data</td>
      <td>WorkflowHub</td>
      <td>1431</td>
      <td>https://workflowhub.eu/workflows/1431?version=1</td>
      <td></td>
      <td>gtn, galaxy, name:gtn</td>
      <td>2025-06-02</td>
      <td>2025-06-02</td>
      <td>1</td>
      <td>1</td>
      <td>6.0</td>
      <td>krakentools_beta_diversity, krakentools_alpha_diversity</td>
      <td>Visualisation, Aggregation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow 3: Classification [Galaxy Training: 16S Microbial Analysis With Mothur]</td>
      <td>WorkflowHub</td>
      <td>1428</td>
      <td>https://workflowhub.eu/workflows/1428?version=1</td>
      <td></td>
      <td>gtn, galaxy, microbiome</td>
      <td>2025-06-02</td>
      <td>2025-06-02</td>
      <td>1</td>
      <td>1</td>
      <td>3.0</td>
      <td>mothur_summary_seqs, mothur_remove_lineage, mothur_classify_seqs</td>
      <td>Taxonomic classification, Phylogenetic analysis, Sequencing quality control, Sequence read processing, Visualisation, DNA barcoding, Sequence clustering</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow 1: Quality Control [Galaxy Training: 16S Microbial Analysis With Mothur]</td>
      <td>WorkflowHub</td>
      <td>1422</td>
      <td>https://workflowhub.eu/workflows/1422?version=1</td>
      <td></td>
      <td>gtn, galaxy, microbiome</td>
      <td>2025-06-02</td>
      <td>2025-06-02</td>
      <td>1</td>
      <td>1</td>
      <td>5.0</td>
      <td>mothur_count_seqs, mothur_unique_seqs, mothur_screen_seqs, mothur_summary_seqs</td>
      <td>Taxonomic classification, Phylogenetic analysis, Sequencing quality control, Sequence read processing, Visualisation, DNA barcoding, Sequence clustering</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow7: Beta Diversity [Galaxy Training: 16S Microbial Analysis With Mothur]</td>
      <td>WorkflowHub</td>
      <td>1418</td>
      <td>https://workflowhub.eu/workflows/1418?version=1</td>
      <td></td>
      <td>gtn, galaxy, microbiome</td>
      <td>2025-06-02</td>
      <td>2025-06-02</td>
      <td>1</td>
      <td>1</td>
      <td>6.0</td>
      <td>mothur_venn, mothur_dist_shared, mothur_heatmap_sim, mothur_tree_shared, collapse_dataset, newick_display</td>
      <td>Taxonomic classification, Phylogenetic analysis, Phylogenetic tree reconstruction, Sequencing quality control, Phylogenetic tree generation, Sequence read processing, Phylogenetic tree analysis, Visualisation, DNA barcoding, Sequence clustering</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow 6: Alpha Diversity [Galaxy Training: 16S Microbial Analysis With Mothur]</td>
      <td>WorkflowHub</td>
      <td>1412</td>
      <td>https://workflowhub.eu/workflows/1412?version=1</td>
      <td></td>
      <td>gtn, galaxy, microbiome</td>
      <td>2025-06-02</td>
      <td>2025-06-02</td>
      <td>1</td>
      <td>1</td>
      <td>3.0</td>
      <td>mothur_rarefaction_single, XY_Plot_1, mothur_summary_single</td>
      <td>Taxonomic classification, Phylogenetic analysis, Sequencing quality control, Sequence read processing, Visualisation, DNA barcoding, Sequence clustering</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow 4: Mock OTU Clustering [Galaxy Training: 16S Microbial Analysis With Mothur]</td>
      <td>WorkflowHub</td>
      <td>1408</td>
      <td>https://workflowhub.eu/workflows/1408?version=1</td>
      <td></td>
      <td>gtn, galaxy, microbiome</td>
      <td>2025-06-02</td>
      <td>2025-06-02</td>
      <td>1</td>
      <td>1</td>
      <td>4.0</td>
      <td>mothur_dist_seqs, mothur_rarefaction_single, mothur_make_shared, mothur_cluster</td>
      <td>Taxonomic classification, Phylogenetic analysis, Sequencing quality control, Sequence read processing, Visualisation, DNA barcoding, Sequence clustering</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow 5: OTU Clustering [Galaxy Training: 16S Microbial Analysis With Mothur]</td>
      <td>WorkflowHub</td>
      <td>1404</td>
      <td>https://workflowhub.eu/workflows/1404?version=1</td>
      <td></td>
      <td>gtn, galaxy, microbiome</td>
      <td>2025-06-02</td>
      <td>2025-06-02</td>
      <td>1</td>
      <td>1</td>
      <td>6.0</td>
      <td>mothur_cluster_split, mothur_classify_otu, mothur_count_groups, mothur_make_shared, mothur_remove_groups, mothur_sub_sample</td>
      <td>Taxonomic classification, Phylogenetic analysis, Sequencing quality control, Sequence read processing, Visualisation, DNA barcoding, Sequence clustering</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow 2: Data Cleaning And Chimera Removal [Galaxy Training: 16S Microbial Analysis With Mothur]</td>
      <td>WorkflowHub</td>
      <td>1400</td>
      <td>https://workflowhub.eu/workflows/1400?version=1</td>
      <td></td>
      <td>gtn, galaxy, microbiome</td>
      <td>2025-06-02</td>
      <td>2025-06-02</td>
      <td>1</td>
      <td>1</td>
      <td>9.0</td>
      <td>mothur_pre_cluster, mothur_unique_seqs, mothur_remove_seqs, mothur_summary_seqs, mothur_screen_seqs, mothur_filter_seqs, mothur_chimera_vsearch</td>
      <td>Taxonomic classification, Phylogenetic analysis, Sequencing quality control, Sequence read processing, Visualisation, DNA barcoding, Sequence clustering</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>GTN Training - Antibiotic Resistance Detection</td>
      <td>WorkflowHub</td>
      <td>406</td>
      <td>https://workflowhub.eu/workflows/406?version=1</td>
      <td>Saskia Hiltemann, Willem de Koning</td>
      <td>metagenomics</td>
      <td>2022-11-24</td>
      <td>2023-02-13</td>
      <td>1</td>
      <td>1</td>
      <td>12.0</td>
      <td>unicycler, staramr_search, nanoplot, PlasFlow, gfa_to_fa, miniasm, bandage_image, minimap2, racon</td>
      <td>De-novo assembly, Pairwise sequence alignment, Box-Whisker plot plotting, Scatter plot plotting, Sequence assembly visualisation, Antimicrobial resistance prediction, Genome assembly, Pairwise sequence alignment, Sequence analysis, Mapping assembly, Aggregation</td>
      <td>Microbiology</td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>3: Plant virus exploration</td>
      <td>WorkflowHub</td>
      <td>103</td>
      <td>https://workflowhub.eu/workflows/103?version=1</td>
      <td></td>
      <td>de_novo, virology, exploration</td>
      <td>2021-02-04</td>
      <td>2023-02-13</td>
      <td>1</td>
      <td>1</td>
      <td>2.0</td>
      <td>fastp, shovill</td>
      <td>Sequencing quality control, Sequence contamination filtering, Genome assembly</td>
      <td></td>
      <td>Apache-2.0</td>
      <td></td>
      <td>Integrated and Urban Plant Pathology Laboratory</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>2: Plant virus confirmation</td>
      <td>WorkflowHub</td>
      <td>102</td>
      <td>https://workflowhub.eu/workflows/102?version=1</td>
      <td></td>
      <td>assembly, virology, blast, mapping, reads_selection</td>
      <td>2021-02-04</td>
      <td>2023-02-13</td>
      <td>1</td>
      <td>1</td>
      <td>7.0</td>
      <td>ngsutils_bam_filter, ncbi_blastn_wrapper, fasta_merge_files_and_filter_unique_sequences, shovill, samtools_stats, picard_SamToFastq, minimap2</td>
      <td>Variant calling, Formatting, Pairwise sequence alignment, Read pre-processing, Sequencing quality control, Sequence contamination filtering, Genome assembly</td>
      <td></td>
      <td>Apache-2.0</td>
      <td></td>
      <td>Integrated and Urban Plant Pathology Laboratory</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>1: Plant virus detection with kraken2 (SE)</td>
      <td>WorkflowHub</td>
      <td>124</td>
      <td>https://workflowhub.eu/workflows/124?version=1</td>
      <td></td>
      <td>virology, kraken</td>
      <td>2021-06-17</td>
      <td>2023-02-13</td>
      <td>1</td>
      <td>1</td>
      <td>3.0</td>
      <td>Kraken2Tax, taxonomy_krona_chart, kraken2</td>
      <td>Visualisation, Taxonomic classification</td>
      <td></td>
      <td>Apache-2.0</td>
      <td></td>
      <td>Integrated and Urban Plant Pathology Laboratory</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>1: Plant virus detection with kraken2 (PE)</td>
      <td>WorkflowHub</td>
      <td>101</td>
      <td>https://workflowhub.eu/workflows/101?version=1</td>
      <td></td>
      <td>virology, kraken</td>
      <td>2021-02-04</td>
      <td>2023-02-13</td>
      <td>1</td>
      <td>1</td>
      <td>3.0</td>
      <td>Kraken2Tax, taxonomy_krona_chart, kraken2</td>
      <td>Visualisation, Taxonomic classification</td>
      <td></td>
      <td>Apache-2.0</td>
      <td></td>
      <td>Integrated and Urban Plant Pathology Laboratory</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Taxonomy classification using Kraken2 and Bracken</td>
      <td>WorkflowHub</td>
      <td>1199</td>
      <td>https://workflowhub.eu/workflows/1199?version=2</td>
      <td>Valentine Murigneux, Mike Thang</td>
      <td>gucfg2galaxy, metagenomics, name:collection, shotgun</td>
      <td>2024-12-10</td>
      <td>2024-12-10</td>
      <td>2</td>
      <td>2</td>
      <td>29.0</td>
      <td>collection_element_identifiers, Extract column name and fraction_total_reads from Bracken report\nCut1, Fisher results contains a header line we want to exclude "Fisher's alpha...loading"\nShow tail1, Prepare alpha diversity summary file (paste sample identifiers and Simpson results)\nPaste1, krakentools_beta_diversity, est_abundance, cat_multiple, add_line_to_file, kraken2, krakentools_kreport2krona, collection_column_join, __RELABEL_FROM_FILE__, Prepare alpha diversity summary file\nPaste1, Prepare alpha diversity summary file (paste Simpson and Fisher results)\nPaste1, regex1, krakentools_combine_kreports, taxonomy_krona_chart, krakentools_alpha_diversity</td>
      <td>Taxonomic classification, Visualisation, Statistical calculation, Aggregation</td>
      <td>Metagenomics, Taxonomy</td>
      <td>CC-BY-4.0</td>
      <td>10.48546/workflowhub.workflow.1199.2</td>
      <td>QCIF Bioinformatics</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Analyses of shotgun metagenomics data with MetaPhlAn2</td>
      <td>WorkflowHub</td>
      <td>624</td>
      <td>https://workflowhub.eu/workflows/624?version=1</td>
      <td>Valentine Murigneux, Mike Thang, Saskia Hiltemann, Bérénice Batut</td>
      <td>gucfg2galaxy, metagenomics, shotgun</td>
      <td>2023-10-26</td>
      <td>2024-04-05</td>
      <td>1</td>
      <td>1</td>
      <td>17.0</td>
      <td>Cut1, , merge_metaphlan_tables, metaphlan2krona, humann2, Cut1, humann2_renorm_table, humann2_regroup_table, metaphlan2, taxonomy_krona_chart</td>
      <td>Visualisation, Taxonomic classification</td>
      <td>Metagenomic sequencing, Metagenomics</td>
      <td>CC-BY-4.0</td>
      <td>10.48546/workflowhub.workflow.624.1</td>
      <td>QCIF Bioinformatics</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow 7 : Beta Diversity [16S Microbial Analysis With Mothur]</td>
      <td>WorkflowHub</td>
      <td>653</td>
      <td>https://workflowhub.eu/workflows/653?version=1</td>
      <td>Saskia Hiltemann, Bérénice Batut, Dave Clements, Ahmed Mehdi</td>
      <td>metagenomics</td>
      <td>2023-11-09</td>
      <td>2023-11-09</td>
      <td>1</td>
      <td>1</td>
      <td>4.0</td>
      <td>mothur_tree_shared, mothur_dist_shared, mothur_heatmap_sim, newick_display</td>
      <td>Taxonomic classification, Phylogenetic analysis, Phylogenetic tree reconstruction, Sequencing quality control, Phylogenetic tree generation, Sequence read processing, Phylogenetic tree analysis, Visualisation, DNA barcoding, Sequence clustering</td>
      <td></td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>QCIF Bioinformatics</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow 6: Alpha Diversity [16S Microbial Analysis With Mothur]</td>
      <td>WorkflowHub</td>
      <td>652</td>
      <td>https://workflowhub.eu/workflows/652?version=1</td>
      <td>Saskia Hiltemann, Bérénice Batut, Dave Clements, Ahmed Mehdi</td>
      <td>metagenomics</td>
      <td>2023-11-09</td>
      <td>2023-11-09</td>
      <td>1</td>
      <td>1</td>
      <td>3.0</td>
      <td>mothur_rarefaction_single, XY_Plot_1, mothur_summary_single</td>
      <td>Taxonomic classification, Phylogenetic analysis, Sequencing quality control, Sequence read processing, Visualisation, DNA barcoding, Sequence clustering</td>
      <td></td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>QCIF Bioinformatics</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow 5: OTU Clustering [16S Microbial Analysis With Mothur]</td>
      <td>WorkflowHub</td>
      <td>651</td>
      <td>https://workflowhub.eu/workflows/651?version=1</td>
      <td>Saskia Hiltemann, Bérénice Batut, Dave Clements, Ahmed Mehdi</td>
      <td>metagenomics</td>
      <td>2023-11-09</td>
      <td>2023-11-09</td>
      <td>1</td>
      <td>1</td>
      <td>5.0</td>
      <td>mothur_cluster_split, mothur_classify_otu, mothur_make_shared, mothur_count_groups, mothur_sub_sample</td>
      <td>Taxonomic classification, Phylogenetic analysis, Sequencing quality control, Sequence read processing, Visualisation, DNA barcoding, Sequence clustering</td>
      <td></td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>QCIF Bioinformatics</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow 3: Classification [Galaxy Training: 16S Microbial Analysis With Mothur]</td>
      <td>WorkflowHub</td>
      <td>650</td>
      <td>https://workflowhub.eu/workflows/650?version=1</td>
      <td>Saskia Hiltemann, Bérénice Batut, Dave Clements, Ahmed Mehdi</td>
      <td>metagenomics</td>
      <td>2023-11-09</td>
      <td>2023-11-09</td>
      <td>1</td>
      <td>1</td>
      <td>2.0</td>
      <td>mothur_remove_lineage, mothur_classify_seqs</td>
      <td>Taxonomic classification, Phylogenetic analysis, Sequencing quality control, Sequence read processing, Visualisation, DNA barcoding, Sequence clustering</td>
      <td></td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>QCIF Bioinformatics</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow 1: Further Quality Control [16S Microbial Analysis With Mothur]</td>
      <td>WorkflowHub</td>
      <td>648</td>
      <td>https://workflowhub.eu/workflows/648?version=1</td>
      <td>Saskia  Hiltemann, Bérénice Batut, Dave Clements, Ahmed Mehdi</td>
      <td>metagenomics</td>
      <td>2023-11-09</td>
      <td>2023-11-09</td>
      <td>1</td>
      <td>1</td>
      <td>5.0</td>
      <td>mothur_count_seqs, mothur_unique_seqs, mothur_screen_seqs, mothur_summary_seqs</td>
      <td>Taxonomic classification, Phylogenetic analysis, Sequencing quality control, Sequence read processing, Visualisation, DNA barcoding, Sequence clustering</td>
      <td></td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>QCIF Bioinformatics</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow 2: Data Cleaning And Chimera Removal [16S Microbial Analysis With Mothur]</td>
      <td>WorkflowHub</td>
      <td>649</td>
      <td>https://workflowhub.eu/workflows/649?version=1</td>
      <td>Saskia Hiltemann, Bérénice Batut, Dave Clements, Ahmed Mehdi</td>
      <td>metagenomics</td>
      <td>2023-11-09</td>
      <td>2023-11-09</td>
      <td>1</td>
      <td>1</td>
      <td>9.0</td>
      <td>mothur_pre_cluster, mothur_unique_seqs, mothur_remove_seqs, mothur_summary_seqs, mothur_screen_seqs, mothur_filter_seqs, mothur_chimera_vsearch</td>
      <td>Taxonomic classification, Phylogenetic analysis, Sequencing quality control, Sequence read processing, Visualisation, DNA barcoding, Sequence clustering</td>
      <td></td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>QCIF Bioinformatics</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>TB Variant Analysis v1.0</td>
      <td>WorkflowHub</td>
      <td>1035</td>
      <td>https://workflowhub.eu/workflows/1035?version=1</td>
      <td>Peter van Heusden</td>
      <td>pathogen, tuberculosis</td>
      <td>2024-06-07</td>
      <td>2024-06-12</td>
      <td>1</td>
      <td>1</td>
      <td>15.0</td>
      <td>qualimap_bamqc, bcftools_consensus, tp_sed_tool, mosdepth, __FLATTEN__, tb_variant_filter, snippy, EMBOSS: seqret84, kraken2, tp_awk_tool, tbvcfreport, tb_profiler_profile, fastp, multiqc</td>
      <td>Variant calling, Local alignment, Taxonomic classification, Sequencing quality control, Phylogenetic tree generation, Sequence contamination filtering, Global alignment, Antimicrobial resistance prediction, Validation, Sequence analysis, Sequence alignment analysis, Sequence alignment, Phylogenetic tree visualisation</td>
      <td>Genetic variation, Infectious disease, Microbiology, Public health and epidemiology, Sequence assembly</td>
      <td>AGPL-3.0-or-later</td>
      <td></td>
      <td>SANBI Pathogen Bioinformatics</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow 3: AMR - SeqSero2/SISTR</td>
      <td>WorkflowHub</td>
      <td>407</td>
      <td>https://workflowhub.eu/workflows/407?version=1</td>
      <td></td>
      <td>bioinformatics, antimicrobial resistance</td>
      <td>2022-11-24</td>
      <td>2024-09-09</td>
      <td>1</td>
      <td>1</td>
      <td>14.0</td>
      <td>bbtools_bbduk, srst2, kma_map, bbtools_tadpole, mob_recon, __UNZIP_COLLECTION__, shovill, sistr_cmd, hamronize_tool, seqsero2, hamronize_summarize</td>
      <td>Clustering, Read binning, Sequence trimming, Genome assembly, Genome indexing, Visualisation, RNA-Seq analysis, Read mapping, Sequence contamination filtering, Antimicrobial resistance prediction, Data handling, Parsing, Sequence analysis, Sequence alignment, Multilocus sequence typing, Genome alignment</td>
      <td></td>
      <td>GPL-3.0</td>
      <td></td>
      <td>Seq4AMR</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow 2: Sciensano</td>
      <td>WorkflowHub</td>
      <td>644</td>
      <td>https://workflowhub.eu/workflows/644?version=1</td>
      <td></td>
      <td></td>
      <td>2023-11-07</td>
      <td>2024-09-09</td>
      <td>1</td>
      <td>1</td>
      <td>1.0</td>
      <td>pipeline_stec_1.0</td>
      <td></td>
      <td></td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>Seq4AMR</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow 4: Staramr</td>
      <td>WorkflowHub</td>
      <td>470</td>
      <td>https://workflowhub.eu/workflows/470?version=1</td>
      <td></td>
      <td>10.3390/microorganisms10020292, amr, amr-detection, bioinformatics, antimicrobial resistance</td>
      <td>2023-05-11</td>
      <td>2024-09-09</td>
      <td>1</td>
      <td>1</td>
      <td>10.0</td>
      <td>collapse_dataset, staramr_search, tp_find_and_replace, shovill, abricate, hamronize_tool, hamronize_summarize</td>
      <td>Data handling, Parsing, Antimicrobial resistance prediction, Genome assembly</td>
      <td></td>
      <td>GPL-3.0</td>
      <td></td>
      <td>Seq4AMR</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow 1: AbritAMR</td>
      <td>WorkflowHub</td>
      <td>634</td>
      <td>https://workflowhub.eu/workflows/634?version=1</td>
      <td></td>
      <td></td>
      <td>2023-10-31</td>
      <td>2024-09-09</td>
      <td>1</td>
      <td>1</td>
      <td>1.0</td>
      <td>abritamr</td>
      <td>Antimicrobial resistance prediction</td>
      <td></td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>Seq4AMR</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>AMR-Pathfinder</td>
      <td>WorkflowHub</td>
      <td>1189</td>
      <td>https://workflowhub.eu/workflows/1189?version=2</td>
      <td>Helena Rasche, Dennis Dollée, Birgit Rijvers</td>
      <td>amr, amr-detection, benchamrking</td>
      <td>2024-11-14</td>
      <td>2024-12-24</td>
      <td>2</td>
      <td>2</td>
      <td>36.0</td>
      <td>, addValue, cat_multi_datasets, Grep1, hamronize_summarize, tp_text_file_with_recurring_lines, cat1, hamronize_tool, __MERGE_COLLECTION__, cast, Cut1, datamash_ops, shovill, __APPLY_RULES__, tp_split_on_column, collapse_dataset, staramr_search, tp_find_and_replace, abricate</td>
      <td>Data handling, Parsing, Antimicrobial resistance prediction, Genome assembly</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>Seq4AMR, ErasmusMC Clinical Bioinformatics</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Metagenomic Taxonomy Analysis</td>
      <td>https://usegalaxy.eu</td>
      <td>7491883694fff308</td>
      <td>https://usegalaxy.eu/published/workflow?id=7491883694fff308</td>
      <td>Bérénice Batut, Géraldine Piot, Mina Hojat Ansari</td>
      <td>name:metagenomics, microbiome, diversity, name:microgalaxy</td>
      <td>2025-11-28</td>
      <td>2025-12-17</td>
      <td>3</td>
      <td>3</td>
      <td></td>
      <td>collection_element_identifiers, sylph_profile, datamash_transpose, metaphlan2krona, est_abundance, add_line_to_file, taxpasta, kraken2, krakentools_kreport2krona, cat1, Remove beginning1, metaphlan, taxonomy_krona_chart</td>
      <td>Formatting, Taxonomic classification, Standardisation and normalisation, Visualisation, Conversion, Statistical calculation, Aggregation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Quality control of bins generated by MetaBAT2, MaxBin2, SemiBin, and CONCOCT</td>
      <td>https://usegalaxy.eu</td>
      <td>7371b6918e895e0c</td>
      <td>https://usegalaxy.eu/published/workflow?id=7371b6918e895e0c</td>
      <td>Bérénice Batut</td>
      <td>microbiome, name:microgalaxy</td>
      <td>2025-12-01</td>
      <td>2025-12-08</td>
      <td>16</td>
      <td>16</td>
      <td></td>
      <td>collection_element_identifiers, __FLATTEN__, multiqc, __RELABEL_FROM_FILE__, checkm2, regex1, __MERGE_COLLECTION__</td>
      <td>Sequence composition calculation, Sequence assembly validation, Sequencing quality control, Statistical calculation, Validation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>pAllori Amplicon 16S Pre-Processing and Taxonomy Classification</td>
      <td>https://usegalaxy.eu</td>
      <td>466bdd8ba7b67264</td>
      <td>https://usegalaxy.eu/published/workflow?id=466bdd8ba7b67264</td>
      <td>Engy Nasr, Paul Zierep</td>
      <td>name:microgalaxy, name:16s, name:metagenomics, name:pallori</td>
      <td>2023-06-30</td>
      <td>2025-11-18</td>
      <td>193</td>
      <td>193</td>
      <td></td>
      <td>biom_convert, ampvis2_heatmap, lotus2, fastqc, mmuphin, unzip, datamash_transpose, decontam, ampvis2_export_otu, tp_cut_tool, maaslin2, compose_text_param, volcanoplot, ampvis2_ordinate, phyloseq_tax_glom, regex1, tp_easyjoin_tool, tp_awk_tool, phyloseq_add_rank_names, ampvis2_subset_samples, bg_column_arrange_by_header, param_value_from_file, calculate_numeric_param, __FLATTEN__, phyloseq_from_biom, ampvis2_load, Remove beginning1, multiqc</td>
      <td>Formatting, Sequence composition calculation, Filtering, Classification, Sequencing quality control, Statistical calculation, Analysis, Sequence feature detection, Deposition, Validation, Standardisation and normalisation, Visualisation, DNA barcoding</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Metagenomic Taxonomy and Functional Analysis</td>
      <td>https://usegalaxy.eu</td>
      <td>cb2df493f5e334f7</td>
      <td>https://usegalaxy.eu/published/workflow?id=cb2df493f5e334f7</td>
      <td>Bérénice Batut, Géraldine Piot</td>
      <td>name:metagenomics, microbiome, diversity, name:microgalaxy</td>
      <td>2024-12-20</td>
      <td>2025-11-13</td>
      <td>18</td>
      <td>18</td>
      <td></td>
      <td>humann_unpack_pathways, sort1, collection_element_identifiers, humann, tp_sed_tool, datamash_transpose, est_abundance, add_line_to_file, __UNZIP_COLLECTION__, tp_text_file_with_recurring_lines, krakentools_kreport2krona, cat1, collection_column_join, humann_regroup_table, fastq_paired_end_interlacer, sylph_profile, tp_replace_in_line, tp_easyjoin_tool, tp_awk_tool, csv_to_tabular, krakentools_alpha_diversity, humann_renorm_table, collapse_dataset, taxpasta, kraken2, Remove beginning1, metaphlan, humann_rename_table, taxonomy_krona_chart</td>
      <td>Formatting, Taxonomic classification, Phylogenetic analysis, Statistical calculation, Species frequency estimation, Standardisation and normalisation, Visualisation, Conversion, Aggregation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>pAllori Blood RNA</td>
      <td>https://usegalaxy.eu</td>
      <td>ca9d3233b0912765</td>
      <td>https://usegalaxy.eu/published/workflow?id=ca9d3233b0912765</td>
      <td>Engy Nasr, Paul Zierep</td>
      <td>name:microgalaxy, name:pallori, name:bloodrna, name:umi, name:barcode, name:transcriptomics</td>
      <td>2023-07-02</td>
      <td>2025-06-26</td>
      <td>48</td>
      <td>48</td>
      <td></td>
      <td>umi_tools_dedup, collection_column_join, __FILTER_FAILED_DATASETS__, rna_star, featurecounts, fastqc, __FILTER_FAILED_DATASETS__, multiqc, umi_tools_extract</td>
      <td>Statistical calculation, Sequence composition calculation, Read summarisation, Sequence alignment, Sequencing quality control, RNA-Seq quantification, Validation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>MetaG: Preparation, Building and Annotation of Metagenomics Assembled Genomes</td>
      <td>https://usegalaxy.eu</td>
      <td>667bac8d7453e5f1</td>
      <td>https://usegalaxy.eu/published/workflow?id=667bac8d7453e5f1</td>
      <td>Valerie C. Schiml, Magnus Ø. Arntzen, Bérénice Batut</td>
      <td>name:microgalaxy, name:microbiome, name:mags</td>
      <td>2024-09-30</td>
      <td>2024-12-21</td>
      <td>14</td>
      <td>14</td>
      <td></td>
      <td>tp_cat, checkm_lineage_wf, hmmer_hmmscan, cat_bins, tp_grep_tool, trim_galore, kofamscan, interproscan, maxbin2, tp_awk_tool, coverm_genome, fasplit, sort1, quast, coverm_contig, fraggenescan, fastqc</td>
      <td>Taxonomic classification, Sequence composition calculation, Sequence trimming, Sequence assembly, Sequence generation, Read pre-processing, Gene functional annotation, Probabilistic sequence generation, Sequence database search, Coding region prediction, Visualisation, Data retrieval, Formatting, Protein feature detection, Format validation, Sequencing quality control, Statistical calculation, Primer removal, Validation, Sequence motif recognition, Multiple sequence alignment, Gene prediction, Sequence analysis, Database search, Local alignment, Sequence assembly validation, Conversion, Sequence profile generation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>ASaiM-MT: Metatranscriptomics Analysis of Microbes</td>
      <td>https://usegalaxy.eu</td>
      <td>096b75501c8e0888</td>
      <td>https://usegalaxy.eu/published/workflow?id=096b75501c8e0888</td>
      <td>Subina Mehta, Marie Crane, Emma Leith, Bérénice Batut, Saskia Hiltemann, Magnus Ø Arntzen, Benoit J. Kunath, Phillip B. Pope, Francesco Delogu, Ray Sajulga, Praveen Kumar, James E. Johnson, Timothy J. Griffin, Pratik D. Jagtap</td>
      <td>asaim, metatranscriptomics, rna-seq, microbiome, name:microgalaxy</td>
      <td>2024-12-20</td>
      <td>2024-12-20</td>
      <td>3</td>
      <td>3</td>
      <td></td>
      <td>metaphlan2krona, humann2, Grep1, humann2_renorm_table, combine_metaphlan2_humann2, humann2_genefamilies_genus_level, metaphlan2, cutadapt, fastqc, tp_sort_header_tool, graphlan, export2graphlan, fastq_paired_end_interlacer, graphlan_annotate, humann2_unpack_pathways, group_humann2_uniref_abundances_to_go, format_metaphlan2_output, Grouping1, bg_sortmerna, taxonomy_krona_chart, multiqc</td>
      <td>Sequence similarity search, Formatting, Taxonomic classification, Sequence composition calculation, Sequence trimming, Sequencing quality control, Statistical calculation, Primer removal, Read pre-processing, Validation, Conversion, Phylogenetic inference, Sequence alignment analysis, Phylogenetic tree editing, Visualisation, Phylogenetic tree visualisation, Sequence comparison</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>MetaT: Metatranscriptomics data analysis</td>
      <td>https://usegalaxy.eu</td>
      <td>fd90652d475ed739</td>
      <td>https://usegalaxy.eu/published/workflow?id=fd90652d475ed739</td>
      <td>Valerie C. Schiml, Magnus Ø. Arntzen, Francesco Delogu, Praveen Kumar, Benoit Kunath, Bérénice Batut,  Subina Mehta, James E. Johnson,  Björn Grüning, Phillip B. Pope, Pratik D. Jagtap, Timothy J. Griffin</td>
      <td>name:microgalaxy, metatranscriptomics, microbiome</td>
      <td>2024-11-21</td>
      <td>2024-12-20</td>
      <td>6</td>
      <td>6</td>
      <td></td>
      <td>trim_galore, bg_sortmerna, __UNZIP_COLLECTION__, collection_column_join, kallisto_quant, fastqc</td>
      <td>Sequence similarity search, Gene expression profiling, Sequence composition calculation, Sequence trimming, Sequencing quality control, Primer removal, Statistical calculation, Read pre-processing, Sequence alignment analysis, Sequence comparison</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Halophiles workup of Comparative gene analysis</td>
      <td>https://usegalaxy.eu</td>
      <td>a2c46deea34d9d80</td>
      <td>https://usegalaxy.eu/published/workflow?id=a2c46deea34d9d80</td>
      <td>Anton Nekrutenko</td>
      <td>genome-annotation, name:microgalaxy</td>
      <td>2024-12-03</td>
      <td>2024-12-05</td>
      <td>2</td>
      <td>2</td>
      <td></td>
      <td>bg_diamond, regexColumn1, Filter1, collapse_dataset, Cut1, tp_split_on_column, rbc_mafft, rapidnj, cat1, tab2fasta, bg_diamond_view, join1, gops_intersect_1, Add_a_column1</td>
      <td>Multiple sequence alignment, Phylogenetic tree generation, Sequence alignment analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Metaproteomics workflow</td>
      <td>https://usegalaxy.eu</td>
      <td>cefc49c13ff73231</td>
      <td>https://usegalaxy.eu/published/workflow?id=cefc49c13ff73231</td>
      <td>Timothy Griffin, Pratik Jagtap, James Johnson, Clemens Blank, Subina Mehta</td>
      <td>name:metaproteomics, name:galaxyp, name:massspectrometry, name:microgalaxy</td>
      <td>2024-11-21</td>
      <td>2024-11-21</td>
      <td>40</td>
      <td>40</td>
      <td></td>
      <td>sqlite_to_tabular, query_tabular, unipept, search_gui, peptide_shaker</td>
      <td>Visualisation, Prediction and recognition</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Feature-Count Table Normalization</td>
      <td>https://usegalaxy.eu</td>
      <td>6239178d7cc4ac68</td>
      <td>https://usegalaxy.eu/published/workflow?id=6239178d7cc4ac68</td>
      <td>Johannes Effenberger</td>
      <td>name:microgalaxy, normalization, softmax, relative-abundance, sigmoid, css, limma-tmm, limma-rle</td>
      <td>2024-11-21</td>
      <td>2024-11-21</td>
      <td>5</td>
      <td>5</td>
      <td></td>
      <td>__FILTER_EMPTY_DATASETS__, Grep1, __BUILD_LIST__, __EXTRACT_DATASET__, map_param_value</td>
      <td></td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>metaQuantome_datacreation_workflow</td>
      <td>https://usegalaxy.eu</td>
      <td>cd675ab32d3e7833</td>
      <td>https://usegalaxy.eu/published/workflow?id=cd675ab32d3e7833</td>
      <td>Subina Metha, Timothy J. Griffin, Pratik Jagtap, Emma Leith, Marie Crane</td>
      <td>proteomics, name:microgalaxy</td>
      <td>2024-11-21</td>
      <td>2024-11-21</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>tp_replace_in_line, Filter1, query_tabular, tp_replace_in_column, Cut1, msconvert, unipept, search_gui, peptide_shaker, Remove beginning1, flashlfq, regex1</td>
      <td>Formatting, Filtering, Prediction and recognition, Visualisation, Label-free quantification</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>metaquantome-function-worklow</td>
      <td>https://usegalaxy.eu</td>
      <td>e5a89ef7b5f1c1d9</td>
      <td>https://usegalaxy.eu/published/workflow?id=e5a89ef7b5f1c1d9</td>
      <td>Subina Mehta, Timothy J. Griffin, Pratik Jagtap, Emma Leith, Marie Crane, Praveen Kumar</td>
      <td>proteomics, name:microgalaxy</td>
      <td>2024-11-21</td>
      <td>2024-11-21</td>
      <td>3</td>
      <td>3</td>
      <td></td>
      <td>metaquantome_sample, metaquantome_db, metaquantome_stat, metaquantome_viz, metaquantome_filter, metaquantome_expand</td>
      <td>Heat map generation, Filtering, Differential protein expression analysis, Visualisation, Principal component visualisation, Query and retrieval, Indexing, Statistical inference, Quantification, Functional clustering</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Cloud Aerosol MT-MG Contamination Filtering</td>
      <td>https://usegalaxy.eu</td>
      <td>c09159d7aad0f264</td>
      <td>https://usegalaxy.eu/published/workflow?id=c09159d7aad0f264</td>
      <td>Engy Nasr , Bérénice Batut, Raphaëlle Péguilhan, Pierre Amato</td>
      <td>metagenomics, metatranscriptomics, name:microgalaxy</td>
      <td>2024-11-21</td>
      <td>2024-11-21</td>
      <td>2</td>
      <td>2</td>
      <td></td>
      <td>recentrifuge, seq_filter_by_id, bowtie2, kraken2, filter_tabular, fastq_to_tabular</td>
      <td>Taxonomic classification, Read mapping, Cross-assembly, Expression analysis</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Cloud-Aerosole MT-MG Functional Profiling</td>
      <td>https://usegalaxy.eu</td>
      <td>63478edcea3f449a</td>
      <td>https://usegalaxy.eu/published/workflow?id=63478edcea3f449a</td>
      <td>Engy Nasr, Bérénice Batut, Raphaëlle Péguilhan, Pierre Amato</td>
      <td>metagenomics, metatranscriptomics, name:microgalaxy</td>
      <td>2024-11-21</td>
      <td>2024-11-21</td>
      <td>4</td>
      <td>4</td>
      <td></td>
      <td>bowtie2, featurecounts, multiqc</td>
      <td>Read summarisation, Read mapping, Sequencing quality control, RNA-Seq quantification, Validation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Cloud-Aerosole MT-MG Pre-Processing</td>
      <td>https://usegalaxy.eu</td>
      <td>1ef76b7b86e15792</td>
      <td>https://usegalaxy.eu/published/workflow?id=1ef76b7b86e15792</td>
      <td>Engy Nasr, Bérénice Batut, Raphaëlle Péguilhan, Pierre Amato</td>
      <td>metagenomics, metatranscriptomics, name:microgalaxy</td>
      <td>2024-11-21</td>
      <td>2024-11-21</td>
      <td>6</td>
      <td>6</td>
      <td></td>
      <td>fastqc, trimmomatic, multiqc</td>
      <td>Sequence composition calculation, Sequencing quality control, Statistical calculation, Validation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Mgnify ML</td>
      <td>https://usegalaxy.eu</td>
      <td>011f701c5a55bfdf</td>
      <td>https://usegalaxy.eu/published/workflow?id=011f701c5a55bfdf</td>
      <td></td>
      <td>name:microgalaxy</td>
      <td>2023-10-10</td>
      <td>2024-03-15</td>
      <td>128</td>
      <td>128</td>
      <td></td>
      <td>__BUILD_LIST__, interactive_tool_jupyter_notebook</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecoli Comparison</td>
      <td>https://usegalaxy.eu</td>
      <td>0e492839192d26e0</td>
      <td>https://usegalaxy.eu/published/workflow?id=0e492839192d26e0</td>
      <td></td>
      <td>assembly, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>mergeCols1, tp_grep_tool, addValue, Filter1, Grep1, lastz_wrapper_2, tp_sort_header_tool, tp_sed_tool, tp_cat, cat1, fasta_filter_by_length, bedtools_intersectbed, bedtools_complementbed, Cut1, fasta_compute_length, datamash_ops, join1, bedtools_sortbed, collapse_dataset, random_lines1</td>
      <td>Sequence alignment, Read mapping, Mapping</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Comparative gene analysis</td>
      <td>https://usegalaxy.eu</td>
      <td>86900be4282d4b1d</td>
      <td>https://usegalaxy.eu/published/workflow?id=86900be4282d4b1d</td>
      <td>Anton Nekrutenko</td>
      <td>vgp, genome-annotation, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>bg_diamond, regexColumn1, Filter1, collapse_dataset, Cut1, tp_split_on_column, rbc_mafft, rapidnj, cat1, tab2fasta, bg_diamond_view, join1, gops_intersect_1, Add_a_column1</td>
      <td>Multiple sequence alignment, Phylogenetic tree generation, Sequence alignment analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Apollo Load Test</td>
      <td>https://usegalaxy.eu</td>
      <td>3c233cc56d4df8b0</td>
      <td>https://usegalaxy.eu/published/workflow?id=3c233cc56d4df8b0</td>
      <td></td>
      <td>genome-annotation, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>iframe, create_or_update, create_account, jbrowse, list_organism</td>
      <td>Genome visualisation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Flye-AMR workflow EMC/WMDI - v3.5  (imported from URL)</td>
      <td>https://usegalaxy.eu</td>
      <td>152223c8aed27357</td>
      <td>https://usegalaxy.eu/published/workflow?id=152223c8aed27357</td>
      <td></td>
      <td>wmdi, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>ncbi_blastn_wrapper, deeptools_bam_coverage, Grep1, Filter1, circos, circos_wiggle_to_scatter, tab2fasta, prokka, circos_gc_skew, bandage_image, flye, tp_sort_header_tool, tp_cat, tp_cut_tool, tbl2gff3, tp_text_file_with_recurring_lines, medaka_consensus_pipeline, PlasFlow, regex1, minimap2, circos_interval_to_tile, trimmomatic, tp_easyjoin_tool, Cut1, fasta_compute_length, nanoplot, join1, jbrowse, tp_split_on_column, Add_a_column1, bg_column_arrange_by_header, fasta2tab, regex_replace, staramr_search, gff2bed1, tp_head_tool</td>
      <td>De-novo assembly, Variant calling, Pairwise sequence alignment, Box-Whisker plot plotting, Scatter plot plotting, Antimicrobial resistance prediction, Genome annotation, Pairwise sequence alignment, Cross-assembly, Genome visualisation, Variant calling, Sequence analysis, Base-calling, Box-Whisker plot plotting, Coding region prediction, Sequence assembly, Genome annotation, Cross-assembly, Sequence assembly visualisation, Antimicrobial resistance prediction, Genome assembly, Genome visualisation, Gene prediction, Base-calling, Sequence analysis, Sequence visualisation, Coding region prediction, Mapping assembly</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>From VCFs to SNP distance matrix</td>
      <td>https://usegalaxy.eu</td>
      <td>6d333ed886aab471</td>
      <td>https://usegalaxy.eu/published/workflow?id=6d333ed886aab471</td>
      <td></td>
      <td>evolution, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>tp_cat, tb_variant_filter, snp_dists, bcftools_consensus, snp_sites</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>From Fastqs to VCFs and BAMs</td>
      <td>https://usegalaxy.eu</td>
      <td>3a8fde248a4d0383</td>
      <td>https://usegalaxy.eu/published/workflow?id=3a8fde248a4d0383</td>
      <td></td>
      <td>evolution, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>__MERGE_COLLECTION__, tb_variant_filter, snippy, trimmomatic, __MERGE_COLLECTION__</td>
      <td>Variant calling, Phylogenetic tree visualisation, Phylogenetic tree generation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>From BAMs to drug resistance prediction with TB-profiler</td>
      <td>https://usegalaxy.eu</td>
      <td>d0d6dae0d1eecc21</td>
      <td>https://usegalaxy.eu/published/workflow?id=d0d6dae0d1eecc21</td>
      <td></td>
      <td>evolution, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>tp_sed_tool, tp_grep_tool, tp_cat, tp_replace_in_line, samtools_view, addName, tb_profiler_profile, __MERGE_COLLECTION__</td>
      <td>Antimicrobial resistance prediction</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Compute Allel Frequency Table from paired reads</td>
      <td>https://usegalaxy.eu</td>
      <td>21a39ac4c1becb31</td>
      <td>https://usegalaxy.eu/published/workflow?id=21a39ac4c1becb31</td>
      <td></td>
      <td>name:microgalaxy, hpylori</td>
      <td>2023-12-13</td>
      <td>2023-12-13</td>
      <td>2</td>
      <td>2</td>
      <td></td>
      <td>bcftools_norm, freebayes, snpSift_filter, bowtie2, vcf2tsv, tp_awk_tool, bcftools_view</td>
      <td>Variant calling, Read mapping, Statistical calculation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Reference genome choice (indels, coverage, depth) for short reads</td>
      <td>https://usegalaxy.eu</td>
      <td>61fe08fb3d5e83f8</td>
      <td>https://usegalaxy.eu/published/workflow?id=61fe08fb3d5e83f8</td>
      <td></td>
      <td>name:microgalaxy, hpylori</td>
      <td>2023-10-27</td>
      <td>2023-12-13</td>
      <td>13</td>
      <td>13</td>
      <td></td>
      <td>collection_element_identifiers, tp_cat, bcftools_norm, freebayes, samtools_depth, collapse_dataset, bowtie2, vcf2tsv, samtools_coverage</td>
      <td>Variant calling, Read mapping, Statistical calculation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Quality control of bins generated by MetaBAT2, MaxBin2, SemiBin, and CONCOCT</td>
      <td>https://usegalaxy.fr</td>
      <td>f0776f7a890b523a</td>
      <td>https://usegalaxy.fr/published/workflow?id=f0776f7a890b523a</td>
      <td>Bérénice Batut</td>
      <td>microbiome, name:microgalaxy</td>
      <td>2025-12-08</td>
      <td>2025-12-08</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>collection_element_identifiers, __FLATTEN__, multiqc, __RELABEL_FROM_FILE__, checkm2, regex1, __MERGE_COLLECTION__</td>
      <td>Sequence composition calculation, Sequence assembly validation, Sequencing quality control, Statistical calculation, Validation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Quality control of bins generated by MetaBAT2, MaxBin2, SemiBin, and CONCOCT</td>
      <td>https://usegalaxy.org</td>
      <td>a8aee61c2cbaf6ea</td>
      <td>https://usegalaxy.org/published/workflow?id=a8aee61c2cbaf6ea</td>
      <td>Bérénice Batut</td>
      <td>microbiome, name:microgalaxy</td>
      <td>2025-12-08</td>
      <td>2025-12-08</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>collection_element_identifiers, __FLATTEN__, multiqc, __RELABEL_FROM_FILE__, checkm2, regex1, __MERGE_COLLECTION__</td>
      <td>Sequence composition calculation, Sequence assembly validation, Sequencing quality control, Statistical calculation, Validation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>MGnify's amplicon pipeline v5.0 (release v0.1)</td>
      <td>https://usegalaxy.org</td>
      <td>8e863bbef37cc36b</td>
      <td>https://usegalaxy.org/published/workflow?id=8e863bbef37cc36b</td>
      <td>EMBL's European Bioinformatics Institute, Rand Zoabi, Paul Zierep</td>
      <td>mgnify_amplicon, metagenomics, amplicon, name:microgalaxy</td>
      <td>2025-02-06</td>
      <td>2025-04-25</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>tp_awk_tool, CONVERTER_uncompressed_to_gz, CONVERTER_gz_to_uncompressed, fastq_dl, __MERGE_COLLECTION__</td>
      <td></td>
      <td></td>
      <td>Apache-2.0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>MGnify's amplicon pipeline v5.0 - ITS (release v0.1)</td>
      <td>https://usegalaxy.org</td>
      <td>45f959d5bf505b15</td>
      <td>https://usegalaxy.org/published/workflow?id=45f959d5bf505b15</td>
      <td>EMBL's European Bioinformatics Institute, Rand Zoabi, Paul Zierep</td>
      <td>mgnify_amplicon, amplicon, metagenomics, name:microgalaxy</td>
      <td>2025-02-06</td>
      <td>2025-04-25</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>collection_element_identifiers, bedtools_maskfastabed, biom_convert, __FILTER_EMPTY_DATASETS__, tp_awk_tool, mapseq, taxonomy_krona_chart, bedtools_maskfastabed, __FILTER_FROM_FILE__</td>
      <td>Visualisation, Formatting, k-mer counting, Mapping</td>
      <td></td>
      <td>Apache-2.0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>MGnify's amplicon pipeline v5.0 - Quality control PE (release v0.1)</td>
      <td>https://usegalaxy.org</td>
      <td>a71fcbc60caae8b6</td>
      <td>https://usegalaxy.org/published/workflow?id=a71fcbc60caae8b6</td>
      <td>EMBL's European Bioinformatics Institute, Rand Zoabi, Paul Zierep</td>
      <td>mgnify_amplicon, amplicon, metagenomics, name:microgalaxy</td>
      <td>2025-02-06</td>
      <td>2025-04-25</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>trimmomatic, fastp, __UNZIP_COLLECTION__, fastq_to_fasta_python, fastq_filter, tp_find_and_replace, prinseq, cshl_fasta_formatter, fastqc, multiqc, mgnify_seqprep</td>
      <td>Sequence composition calculation, Read pre-processing, Sequence trimming, Nucleic acid design, Sequencing quality control, Statistical calculation, Sequence contamination filtering, Validation</td>
      <td></td>
      <td>Apache-2.0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>MGnify's amplicon pipeline v5.0 - Quality control SE (release v0.1)</td>
      <td>https://usegalaxy.org</td>
      <td>5f878c4cc3bff68c</td>
      <td>https://usegalaxy.org/published/workflow?id=5f878c4cc3bff68c</td>
      <td>MGnify - EMBL, Rand Zoabi, Paul Zierep</td>
      <td>mgnify_amplicon, amplicon, metagenomics, name:microgalaxy</td>
      <td>2025-02-06</td>
      <td>2025-04-25</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>trimmomatic, fastq_to_fasta_python, tp_find_and_replace, fastq_filter, prinseq, cshl_fasta_formatter, fastqc, multiqc</td>
      <td>Sequence composition calculation, Read pre-processing, Sequence trimming, Sequencing quality control, Statistical calculation, Sequence contamination filtering, Validation</td>
      <td></td>
      <td>Apache-2.0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Allele-based Pathogen Identification (release v0.1.3)</td>
      <td>https://usegalaxy.org</td>
      <td>3e703abaeb870f6f</td>
      <td>https://usegalaxy.org/published/workflow?id=3e703abaeb870f6f</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2025-03-06</td>
      <td>2025-04-25</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>table_compute, bcftools_norm, clair3, samtools_depth, tp_cut_tool, snpSift_filter, collapse_dataset, Cut1, snpSift_extractFields, tp_head_tool, Paste1, regexColumn1, Count1, samtools_coverage, CONVERTER_gz_to_uncompressed, Remove beginning1, bcftools_consensus, minimap2</td>
      <td>Variant calling, Pairwise sequence alignment</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>MGnify's amplicon pipeline v5.0 - Quality control SE (release v0.2)</td>
      <td>https://usegalaxy.org</td>
      <td>afb67b4b9952f57d</td>
      <td>https://usegalaxy.org/published/workflow?id=afb67b4b9952f57d</td>
      <td>MGnify - EMBL, Rand Zoabi, Paul Zierep</td>
      <td>mgnify_amplicon, amplicon, metagenomics, name:microgalaxy</td>
      <td>2025-03-06</td>
      <td>2025-04-25</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>trimmomatic, fastq_to_fasta_python, tp_find_and_replace, fastq_filter, prinseq, cshl_fasta_formatter, fastqc, multiqc</td>
      <td>Sequence composition calculation, Read pre-processing, Sequence trimming, Sequencing quality control, Statistical calculation, Sequence contamination filtering, Validation</td>
      <td></td>
      <td>Apache-2.0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Allele-based Pathogen Identification (release v0.1.4)</td>
      <td>https://usegalaxy.org</td>
      <td>cb810cdaa4915f4b</td>
      <td>https://usegalaxy.org/published/workflow?id=cb810cdaa4915f4b</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2025-03-13</td>
      <td>2025-04-25</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>table_compute, bcftools_norm, clair3, samtools_depth, tp_cut_tool, snpSift_filter, collapse_dataset, Cut1, snpSift_extractFields, tp_head_tool, Paste1, regexColumn1, Count1, samtools_coverage, CONVERTER_gz_to_uncompressed, Remove beginning1, bcftools_consensus, minimap2</td>
      <td>Variant calling, Pairwise sequence alignment</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>MGnify's amplicon pipeline v5.0 (release v0.2)</td>
      <td>https://usegalaxy.org</td>
      <td>77de7101effbff3d</td>
      <td>https://usegalaxy.org/published/workflow?id=77de7101effbff3d</td>
      <td>EMBL's European Bioinformatics Institute, Rand Zoabi, Paul Zierep</td>
      <td>mgnify_amplicon, metagenomics, amplicon, name:microgalaxy</td>
      <td>2025-03-13</td>
      <td>2025-04-25</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>tp_awk_tool, CONVERTER_uncompressed_to_gz, CONVERTER_gz_to_uncompressed, fastq_dl, __MERGE_COLLECTION__</td>
      <td></td>
      <td></td>
      <td>Apache-2.0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>MGnify's amplicon pipeline v5.0 - ITS (release v0.2)</td>
      <td>https://usegalaxy.org</td>
      <td>8151be18aab3a3c6</td>
      <td>https://usegalaxy.org/published/workflow?id=8151be18aab3a3c6</td>
      <td>EMBL's European Bioinformatics Institute, Rand Zoabi, Paul Zierep</td>
      <td>mgnify_amplicon, amplicon, metagenomics, name:microgalaxy</td>
      <td>2025-03-13</td>
      <td>2025-04-25</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>collection_element_identifiers, bedtools_maskfastabed, biom_convert, __FILTER_EMPTY_DATASETS__, tp_awk_tool, mapseq, taxonomy_krona_chart, bedtools_maskfastabed, __FILTER_FROM_FILE__</td>
      <td>Visualisation, Formatting, k-mer counting, Mapping</td>
      <td></td>
      <td>Apache-2.0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>MGnify's amplicon pipeline v5.0 - Quality control PE (release v0.2)</td>
      <td>https://usegalaxy.org</td>
      <td>20b7ae4b517a80fd</td>
      <td>https://usegalaxy.org/published/workflow?id=20b7ae4b517a80fd</td>
      <td>EMBL's European Bioinformatics Institute, Rand Zoabi, Paul Zierep</td>
      <td>mgnify_amplicon, amplicon, metagenomics, name:microgalaxy</td>
      <td>2025-03-13</td>
      <td>2025-04-25</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>trimmomatic, fastp, __UNZIP_COLLECTION__, fastq_to_fasta_python, fastq_filter, tp_find_and_replace, prinseq, cshl_fasta_formatter, fastqc, multiqc, mgnify_seqprep</td>
      <td>Sequence composition calculation, Read pre-processing, Sequence trimming, Nucleic acid design, Sequencing quality control, Statistical calculation, Sequence contamination filtering, Validation</td>
      <td></td>
      <td>Apache-2.0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>MGnify's amplicon pipeline v5.0 - Quality control SE (release v0.3)</td>
      <td>https://usegalaxy.org</td>
      <td>46f184a0e95f3c1c</td>
      <td>https://usegalaxy.org/published/workflow?id=46f184a0e95f3c1c</td>
      <td>MGnify - EMBL, Rand Zoabi, Paul Zierep</td>
      <td>mgnify_amplicon, amplicon, metagenomics, name:microgalaxy</td>
      <td>2025-03-13</td>
      <td>2025-04-25</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>trimmomatic, fastq_to_fasta_python, tp_find_and_replace, fastq_filter, prinseq, cshl_fasta_formatter, fastqc, multiqc</td>
      <td>Sequence composition calculation, Read pre-processing, Sequence trimming, Sequencing quality control, Statistical calculation, Sequence contamination filtering, Validation</td>
      <td></td>
      <td>Apache-2.0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Nanopore Preprocessing (release v0.1)</td>
      <td>https://usegalaxy.org</td>
      <td>a8b9252fad5fe06a</td>
      <td>https://usegalaxy.org/published/workflow?id=a8b9252fad5fe06a</td>
      <td>Bérénice Batut, Engy Nasr, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:nanopore, name:iwc</td>
      <td>2025-03-20</td>
      <td>2025-04-25</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>samtools_fastx, krakentools_extract_kraken_reads, regexColumn1, Grep1, collapse_dataset, Cut1, fastp, kraken2, collection_column_join, nanoplot, porechop, bamtools_split_mapped, Add_a_column1, fastqc, multiqc, minimap2, __FILTER_FAILED_DATASETS__</td>
      <td>Pairwise sequence alignment, Taxonomic classification, Box-Whisker plot plotting, Sequence composition calculation, Scatter plot plotting, Sequencing quality control, Statistical calculation, Sequence contamination filtering, Validation, Data handling, Sequence alignment analysis, Visualisation, Aggregation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Quality control of bins generated by MetaBAT2, MaxBin2, SemiBin, and CONCOCT</td>
      <td>https://usegalaxy.org.au</td>
      <td>f05ed1e5c5dfffa9</td>
      <td>https://usegalaxy.org.au/published/workflow?id=f05ed1e5c5dfffa9</td>
      <td>Bérénice Batut</td>
      <td>microbiome, name:microgalaxy</td>
      <td>2025-12-08</td>
      <td>2025-12-08</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>collection_element_identifiers, __FLATTEN__, multiqc, __RELABEL_FROM_FILE__, checkm2, regex1, __MERGE_COLLECTION__</td>
      <td>Sequence composition calculation, Sequence assembly validation, Sequencing quality control, Statistical calculation, Validation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Allele-based Pathogen Identification (release v0.1.3)</td>
      <td>https://usegalaxy.org.au</td>
      <td>0e11619979c2830c</td>
      <td>https://usegalaxy.org.au/published/workflow?id=0e11619979c2830c</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2025-03-06</td>
      <td>2025-04-25</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>table_compute, bcftools_norm, clair3, samtools_depth, tp_cut_tool, snpSift_filter, collapse_dataset, Cut1, snpSift_extractFields, tp_head_tool, Paste1, regexColumn1, Count1, samtools_coverage, CONVERTER_gz_to_uncompressed, Remove beginning1, bcftools_consensus, minimap2</td>
      <td>Variant calling, Pairwise sequence alignment</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Allele-based Pathogen Identification (release v0.1.4)</td>
      <td>https://usegalaxy.org.au</td>
      <td>76a0a3af8de11622</td>
      <td>https://usegalaxy.org.au/published/workflow?id=76a0a3af8de11622</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2025-03-13</td>
      <td>2025-04-25</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>table_compute, bcftools_norm, clair3, samtools_depth, tp_cut_tool, snpSift_filter, collapse_dataset, Cut1, snpSift_extractFields, tp_head_tool, Paste1, regexColumn1, Count1, samtools_coverage, CONVERTER_gz_to_uncompressed, Remove beginning1, bcftools_consensus, minimap2</td>
      <td>Variant calling, Pairwise sequence alignment</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
  </tbody>
</table>

</div>