<script src="https://code.jquery.com/jquery-3.7.0.js"></script>
<script src="https://cdn.datatables.net/1.13.6/js/jquery.dataTables.min.js"></script>
<script src="https://cdn.datatables.net/searchbuilder/1.6.0/js/dataTables.searchBuilder.min.js"></script>
<script src="https://cdn.datatables.net/datetime/1.5.1/js/dataTables.dateTime.min.js"></script>

<script src="https://cdn.datatables.net/responsive/2.5.0/js/dataTables.responsive.min.js"></script>

<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/responsive/2.2.3/css/responsive.dataTables.min.css">

<link rel="stylesheet" href="https://cdn.datatables.net/1.13.6/css/jquery.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/searchbuilder/1.6.0/css/searchBuilder.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/datetime/1.5.1/css/dataTables.dateTime.min.css">

<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" />
<script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>

<style type="text/css">
  .tooltip {
    font-family: Georgia;
    font-size: 18px;
  }
  .tooltip .tooltip-inner {
    background-color: #00FFFF;
    color: #000000;
    min-width: 400px;
  }
  </style>

<script>
    $(document).ready(function() {
    $('#dataframe').DataTable( {
        "sScrollX": "100%",
        "dom": "Qlfrtip",

        responsive: {
            details: {
                type: 'column'
            }
        },
        "bAutoWidth": false,
        columnDefs: [ {
            className: 'control',
            orderable: false,
            targets:   [ 0 ],
            width: 100,
        },
        { type: 'natural', targets: 1 } // allow normal sorting
        ],

    });
});
</script>

<div class="display nowrap" style="width:80%" width="100%">

<table class="dataframe display nowrap" id="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th>Expand</th>
      <th>Galaxy wrapper id</th>
      <th>Galaxy tool ids</th>
      <th>Description</th>
      <th>bio.tool id</th>
      <th>bio.tool ids</th>
      <th>biii</th>
      <th>bio.tool name</th>
      <th>bio.tool description</th>
      <th>EDAM operation</th>
      <th>EDAM topic</th>
      <th>Status</th>
      <th>Source</th>
      <th>ToolShed categories</th>
      <th>ToolShed id</th>
      <th>Date of first commit of the suite</th>
      <th>Galaxy wrapper owner</th>
      <th>Galaxy wrapper source</th>
      <th>Galaxy wrapper parsed folder</th>
      <th>Galaxy wrapper version</th>
      <th>Conda id</th>
      <th>Conda version</th>
      <th>EDAM operation (no superclasses)</th>
      <th>EDAM topic (no superclasses)</th>
      <th>Available on UseGalaxy.org (Main)</th>
      <th>Available on UseGalaxy.org.au</th>
      <th>Available on UseGalaxy.eu</th>
      <th>Available on UseGalaxy.fr</th>
      <th>Tools available on UseGalaxy.org (Main)</th>
      <th>Tools available on UseGalaxy.org.au</th>
      <th>Tools available on UseGalaxy.eu</th>
      <th>Tools available on UseGalaxy.fr</th>
      <th>Tools available on APOSTL</th>
      <th>Tools available on ARGs-OAP</th>
      <th>Tools available on CIRM-CFBP</th>
      <th>Tools available on ChemFlow</th>
      <th>Tools available on Coloc-stats</th>
      <th>Tools available on CropGalaxy</th>
      <th>Tools available on Dintor</th>
      <th>Tools available on GASLINI</th>
      <th>Tools available on Galaxy@AuBi</th>
      <th>Tools available on Galaxy@Pasteur</th>
      <th>Tools available on GalaxyTrakr</th>
      <th>Tools available on Genomic Hyperbrowser</th>
      <th>Tools available on GigaGalaxy</th>
      <th>Tools available on HyPhy HIV NGS Tools</th>
      <th>Tools available on IPK Galaxy Blast Suite</th>
      <th>Tools available on ImmPort Galaxy</th>
      <th>Tools available on InteractoMIX</th>
      <th>Tools available on MISSISSIPPI</th>
      <th>Tools available on Mandoiu Lab</th>
      <th>Tools available on MiModD NacreousMap</th>
      <th>Tools available on Oqtans</th>
      <th>Tools available on Palfinder</th>
      <th>Tools available on PepSimili</th>
      <th>Tools available on PhagePromotor</th>
      <th>Tools available on UseGalaxy.be</th>
      <th>Tools available on UseGalaxy.cz</th>
      <th>Tools available on UseGalaxy.no</th>
      <th>Tools available on Viral Variant Visualizer (VVV)</th>
      <th>No. of tool users (5 years) (usegalaxy.eu)</th>
      <th>No. of tool users (all time) (usegalaxy.eu)</th>
      <th>Tool usage (5 years) (usegalaxy.eu)</th>
      <th>Tool usage (all time) (usegalaxy.eu)</th>
      <th>No. of tool users (5 years) (usegalaxy.org)</th>
      <th>No. of tool users (all time) (usegalaxy.org)</th>
      <th>Tool usage (5 years) (usegalaxy.org)</th>
      <th>Tool usage (all time) (usegalaxy.org)</th>
      <th>No. of tool users (5 years) (usegalaxy.org.au)</th>
      <th>No. of tool users (all time) (usegalaxy.org.au)</th>
      <th>Tool usage (5 years) (usegalaxy.org.au)</th>
      <th>Tool usage (all time) (usegalaxy.org.au)</th>
      <th>No. of tool users (5 years) - all main servers</th>
      <th>No. of tool users (all time) - all main servers</th>
      <th>Tool usage (5 years) - all main servers</th>
      <th>Tool usage (all time) - all main servers</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td></td>
      <td>2d_auto_threshold</td>
      <td>ip_threshold</td>
      <td>Automatic thresholding</td>
      <td>scikit-image</td>
      <td>scikit-image</td>
      <td>scikit-image</td>
      <td>scikit-image</td>
      <td>Scikit-image contains image processing algorithms for SciPy, including IO, morphology, filtering, warping, color manipulation, object detection, etc.</td>
      <td>Image analysis, Image annotation, Visualisation, Data handling</td>
      <td>Imaging, Software engineering, Literature and language</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>2d_auto_threshold</td>
      <td>2019-01-22</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/2d_auto_threshold/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/2d_auto_threshold</td>
      <td>0.18.1</td>
      <td>scikit-image</td>
      <td></td>
      <td>Image analysis, Image annotation, Visualisation, Data handling</td>
      <td>Imaging, Software engineering, Literature and language</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1434</td>
      <td>6746</td>
      <td>120</td>
      <td>122</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>305</td>
      <td>305</td>
      <td>11</td>
      <td>11</td>
      <td>1739</td>
      <td>7051</td>
      <td>131</td>
      <td>133</td>
    </tr>
    <tr>
      <td></td>
      <td>2d_feature_extraction</td>
      <td>ip_2d_feature_extraction</td>
      <td>2D feature extraction</td>
      <td>scikit-image</td>
      <td>scikit-image</td>
      <td>scikit-image</td>
      <td>scikit-image</td>
      <td>Scikit-image contains image processing algorithms for SciPy, including IO, morphology, filtering, warping, color manipulation, object detection, etc.</td>
      <td>Image analysis, Image annotation, Visualisation, Data handling</td>
      <td>Imaging, Software engineering, Literature and language</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>2d_feature_extraction</td>
      <td>2019-01-08</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/2d_feature_extraction/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/2d_feature_extraction</td>
      <td>0.18.1</td>
      <td>scikit-image</td>
      <td></td>
      <td>Image analysis, Image annotation, Visualisation, Data handling</td>
      <td>Imaging, Software engineering, Literature and language</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1156</td>
      <td>13842</td>
      <td>40</td>
      <td>42</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>121</td>
      <td>121</td>
      <td>4</td>
      <td>4</td>
      <td>1277</td>
      <td>13963</td>
      <td>44</td>
      <td>46</td>
    </tr>
    <tr>
      <td></td>
      <td>2d_filter_segmentation_by_features</td>
      <td>ip_2d_filter_segmentation_by_features</td>
      <td>filter segmentation by rules</td>
      <td>galaxy_image_analysis</td>
      <td>galaxy_image_analysis</td>
      <td></td>
      <td>Galaxy Image Analysis</td>
      <td>Galaxy Image Analysis is mainly developed by the Biomedical Computer Vision (BMCV) Group at Heidelberg University. Its dedication is to provide tools for image analysis and image processing for the Galaxy platform.</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>2d_filter_segmentation_by_features</td>
      <td>2019-07-19</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/2d_filter_segmentation_by_features/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/2d_filter_segmentation_by_features</td>
      <td>0.0.1-4</td>
      <td>scikit-image</td>
      <td></td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>591</td>
      <td>6863</td>
      <td>38</td>
      <td>40</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>119</td>
      <td>119</td>
      <td>3</td>
      <td>3</td>
      <td>710</td>
      <td>6982</td>
      <td>41</td>
      <td>43</td>
    </tr>
    <tr>
      <td></td>
      <td>2d_histogram_equalization</td>
      <td>ip_histogram_equalization</td>
      <td>2d histogram equalization</td>
      <td>scikit-image</td>
      <td>scikit-image</td>
      <td>scikit-image</td>
      <td>scikit-image</td>
      <td>Scikit-image contains image processing algorithms for SciPy, including IO, morphology, filtering, warping, color manipulation, object detection, etc.</td>
      <td>Image analysis, Image annotation, Visualisation, Data handling</td>
      <td>Imaging, Software engineering, Literature and language</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>2d_histogram_equalization</td>
      <td>2019-07-23</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/2d_histogram_equalization/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/2d_histogram_equalization</td>
      <td>0.18.1</td>
      <td>scikit-image</td>
      <td></td>
      <td>Image analysis, Image annotation, Visualisation, Data handling</td>
      <td>Imaging, Software engineering, Literature and language</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>762</td>
      <td>782</td>
      <td>100</td>
      <td>101</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>230</td>
      <td>230</td>
      <td>14</td>
      <td>14</td>
      <td>992</td>
      <td>1012</td>
      <td>114</td>
      <td>115</td>
    </tr>
    <tr>
      <td></td>
      <td>2d_simple_filter</td>
      <td>ip_filter_standard</td>
      <td>2d simple filter</td>
      <td></td>
      <td></td>
      <td>scipy</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>2d_simple_filter</td>
      <td>2019-01-22</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/2d_simple_filter/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/2d_simple_filter</td>
      <td>1.12.0</td>
      <td>scipy</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>948</td>
      <td>6261</td>
      <td>98</td>
      <td>100</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>153</td>
      <td>153</td>
      <td>13</td>
      <td>13</td>
      <td>1101</td>
      <td>6414</td>
      <td>111</td>
      <td>113</td>
    </tr>
    <tr>
      <td></td>
      <td>Ensembl-REST</td>
      <td>get_feature_info, get_genetree, get_sequences</td>
      <td>A suite of Galaxy tools designed to work with Ensembl REST API.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://rest.ensembl.org</td>
      <td>Data Source</td>
      <td></td>
      <td>2015-11-17</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/Ensembl-REST</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/Ensembl-REST</td>
      <td>0.1.2</td>
      <td>requests</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>474</td>
      <td>2914</td>
      <td>132</td>
      <td>163</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>474</td>
      <td>2914</td>
      <td>132</td>
      <td>163</td>
    </tr>
    <tr>
      <td></td>
      <td>GAFA</td>
      <td>gafa</td>
      <td>Gene Align and Family Aggregator</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://aequatus.tgac.ac.uk</td>
      <td>Visualization</td>
      <td>gafa</td>
      <td>2016-12-15</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/GAFA/</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/GAFA</td>
      <td>0.3.1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>40</td>
      <td>47</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>40</td>
      <td>47</td>
      <td>6</td>
      <td>6</td>
    </tr>
    <tr>
      <td></td>
      <td>TreeBest</td>
      <td>treebest_best</td>
      <td>TreeBeST best</td>
      <td>treebest</td>
      <td>treebest</td>
      <td></td>
      <td>TreeBeST</td>
      <td>TreeBeST, which stands for (gene) Tree Building guided by Species Tree, is a versatile program that builds, manipulates and displays phylogenetic trees. It is particularly designed for building gene trees with a known species tree and is highly efficient and accurate.TreeBeST is previously known as NJTREE. It has been largely used in the TreeFam database, Ensembl Compara and OPTIC database of Chris Ponting group.</td>
      <td>Phylogenetic tree visualisation, Phylogenetic analysis, Phylogenetic inference (from molecular sequences)</td>
      <td>Phylogenetics</td>
      <td>To update</td>
      <td>http://treesoft.sourceforge.net/treebest.shtml</td>
      <td>Phylogenetics</td>
      <td>treebest_best</td>
      <td>2015-08-06</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/TreeBest</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/TreeBest</td>
      <td>1.9.2.post0</td>
      <td>treebest</td>
      <td>1.9.2.post1</td>
      <td>Phylogenetic tree visualisation, Phylogenetic inference (from molecular sequences)</td>
      <td>Phylogenetics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1003</td>
      <td>1022</td>
      <td>48</td>
      <td>52</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1003</td>
      <td>1022</td>
      <td>48</td>
      <td>52</td>
    </tr>
    <tr>
      <td></td>
      <td>abacas</td>
      <td>abacas</td>
      <td>Order and Orientate Contigs</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/abacas</td>
      <td>Assembly</td>
      <td>abacas</td>
      <td>2019-11-20</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/abacas</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/abacas</td>
      <td>1.1</td>
      <td>mummer</td>
      <td>3.23</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>anisotropic_diffusion</td>
      <td>ip_anisotropic_diffusion</td>
      <td>Anisotropic image diffusion</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>anisotropic_diffusion</td>
      <td>2018-12-04</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/anisotropic-diffusion/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/anisotropic_diffusion</td>
      <td>0.4.0</td>
      <td>medpy</td>
      <td>0.4.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>36</td>
      <td>41</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>36</td>
      <td>41</td>
      <td>2</td>
      <td>2</td>
    </tr>
    <tr>
      <td></td>
      <td>apoc</td>
      <td>apoc</td>
      <td>Large-scale structural comparison of protein pockets</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://cssb.biology.gatech.edu/APoc</td>
      <td>Computational chemistry</td>
      <td>apoc</td>
      <td>2017-07-04</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/apoc/</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/apoc</td>
      <td>1.0+galaxy1</td>
      <td>apoc</td>
      <td>1b16</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
    </tr>
    <tr>
      <td></td>
      <td>apollo</td>
      <td>create_account, feat_from_gff3, create_or_update, delete_features, delete_organism, export, fetch_jbrowse, iframe, list_organism</td>
      <td>Access an Apollo instance from Galaxy</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxy-genome-annotation/python-apollo</td>
      <td>Web Services</td>
      <td></td>
      <td>2020-06-29</td>
      <td>gga</td>
      <td>https://github.com/galaxy-genome-annotation/galaxy-tools/tree/master/tools/apollo</td>
      <td>https://github.com/galaxy-genome-annotation/galaxy-tools/tree/master/tools/apollo</td>
      <td></td>
      <td>apollo</td>
      <td>4.2.13</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>9</td>
      <td>0</td>
      <td>3600</td>
      <td>3760</td>
      <td>883</td>
      <td>911</td>
      <td>6</td>
      <td>6</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3606</td>
      <td>3766</td>
      <td>887</td>
      <td>915</td>
    </tr>
    <tr>
      <td></td>
      <td>askomics</td>
      <td>askomics_integrate</td>
      <td>Galaxy tools allowing to interact with a remote AskOmics server.AskOmics is a visual SPARQL query builder for RDF database.https://github.com/askomics/</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/askomics/</td>
      <td>Web Services</td>
      <td></td>
      <td>2017-08-11</td>
      <td>gga</td>
      <td>https://github.com/galaxy-genome-annotation/galaxy-tools/tree/master/tools/askomics</td>
      <td>https://github.com/galaxy-genome-annotation/galaxy-tools/tree/master/tools/askomics</td>
      <td></td>
      <td>askocli</td>
      <td>0.5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>askor</td>
      <td>askor_de</td>
      <td>AskoR links EdgeR and AskOmics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/askomics/askoR</td>
      <td>Transcriptomics</td>
      <td>askor_de</td>
      <td>2018-04-09</td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/askor</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/askor</td>
      <td>0.2</td>
      <td>bioconductor-limma</td>
      <td>3.58.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>assemblystats</td>
      <td>assemblystats</td>
      <td>Summarise an assembly (e.g. N50 metrics)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>Assembly</td>
      <td>assemblystats</td>
      <td>2017-10-13</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/assemblystats</td>
      <td>1.1.0</td>
      <td>perl-bioperl</td>
      <td>1.7.8</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>background_removal</td>
      <td>background_removal</td>
      <td>Background removal filters using scikit-image</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>background_removal</td>
      <td>2024-07-12</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tools/background_removal</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/background_removal</td>
      <td>0.24.0</td>
      <td>scikit-image</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>1</td>
      <td>1</td>
    </tr>
    <tr>
      <td></td>
      <td>bam2mappingstats</td>
      <td>bam2mappingstats</td>
      <td>Generates mapping stats from a bam file.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>Assembly</td>
      <td>bam2mappingstats</td>
      <td>2017-10-13</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/bam2mappingstats</td>
      <td>1.1.0</td>
      <td>perl</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>bamclipper</td>
      <td>bamclipper</td>
      <td>Soft-clip gene-specific primers from BAM alignment file based on genomic coordinates of primer pairs in BEDPE format.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/tommyau/bamclipper</td>
      <td>Sequence Analysis</td>
      <td>bamclipper</td>
      <td>2020-04-28</td>
      <td>nml</td>
      <td>https://github.com/tommyau/bamclipper</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/bamclipper</td>
      <td>1.0.0</td>
      <td>bamclipper</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>baric_archive</td>
      <td>baric_archive_rennes, baric_archive_toulouse</td>
      <td>A data source tool to fetch data from a BARIC Archive server.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.cesgo.org/catibaric/</td>
      <td>Data Source</td>
      <td></td>
      <td>2022-03-03</td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/baric_archive</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/baric_archive</td>
      <td>1.1.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>batched_lastz</td>
      <td>batched_lastz</td>
      <td>Galaxy wrapper for the batching Lastz runs</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproject/KegAlign</td>
      <td>Next Gen Mappers</td>
      <td>batched_lastz</td>
      <td>2024-04-30</td>
      <td>richard-burhans</td>
      <td>https://github.com/richard-burhans/galaxytools/tree/main/tools/batched_lastz</td>
      <td>https://github.com/richard-burhans/galaxytools/tree/main/tools/batched_lastz</td>
      <td>1.04.22</td>
      <td>lastz</td>
      <td>1.04.22</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>123</td>
      <td>123</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>123</td>
      <td>123</td>
      <td>5</td>
      <td>5</td>
    </tr>
    <tr>
      <td></td>
      <td>bfconvert</td>
      <td>ip_convertimage</td>
      <td>Convert image</td>
      <td></td>
      <td></td>
      <td>python-bioformats</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging, Convert Formats</td>
      <td>bfconvert</td>
      <td>2019-07-22</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/bfconvert/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/bfconvert</td>
      <td>6.7.0</td>
      <td>bftools</td>
      <td>6.7.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1635</td>
      <td>1671</td>
      <td>123</td>
      <td>124</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>334</td>
      <td>334</td>
      <td>21</td>
      <td>21</td>
      <td>1969</td>
      <td>2005</td>
      <td>144</td>
      <td>145</td>
    </tr>
    <tr>
      <td></td>
      <td>binary2labelimage</td>
      <td>ip_binary_to_labelimage</td>
      <td>Binary 2 label image</td>
      <td>galaxy_image_analysis</td>
      <td>galaxy_image_analysis</td>
      <td></td>
      <td>Galaxy Image Analysis</td>
      <td>Galaxy Image Analysis is mainly developed by the Biomedical Computer Vision (BMCV) Group at Heidelberg University. Its dedication is to provide tools for image analysis and image processing for the Galaxy platform.</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>binary2labelimage</td>
      <td>2017-02-11</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/binary2labelimage/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/binary2labelimage</td>
      <td>0.5</td>
      <td>scikit-image</td>
      <td></td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>401</td>
      <td>1385</td>
      <td>87</td>
      <td>88</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>188</td>
      <td>188</td>
      <td>9</td>
      <td>9</td>
      <td>589</td>
      <td>1573</td>
      <td>96</td>
      <td>97</td>
    </tr>
    <tr>
      <td></td>
      <td>binaryimage2points</td>
      <td>ip_binaryimage_to_points</td>
      <td>Binary Image to Points</td>
      <td>galaxy_image_analysis</td>
      <td>galaxy_image_analysis</td>
      <td></td>
      <td>Galaxy Image Analysis</td>
      <td>Galaxy Image Analysis is mainly developed by the Biomedical Computer Vision (BMCV) Group at Heidelberg University. Its dedication is to provide tools for image analysis and image processing for the Galaxy platform.</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>binaryimage2points</td>
      <td>2017-10-30</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/binaryimage2points/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/binaryimage2points</td>
      <td>0.1-2</td>
      <td>numpy</td>
      <td></td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>24</td>
      <td>29</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>24</td>
      <td>29</td>
      <td>1</td>
      <td>1</td>
    </tr>
    <tr>
      <td></td>
      <td>bioformats2raw</td>
      <td>bf2raw</td>
      <td>Convert image to OME-Zarr</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/Euro-BioImaging</td>
      <td>Imaging, Convert Formats</td>
      <td>bioformats2raw</td>
      <td>2023-11-02</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/bioformats2raw</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/bioformats2raw</td>
      <td>0.7.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>61</td>
      <td>61</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>61</td>
      <td>61</td>
      <td>4</td>
      <td>4</td>
    </tr>
    <tr>
      <td></td>
      <td>biohansel</td>
      <td>biohansel</td>
      <td>Heidelberg and Enteritidis SNP Elucidation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/biohansel</td>
      <td>Sequence Analysis</td>
      <td>biohansel</td>
      <td>2018-08-01</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/biohansel</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/biohansel</td>
      <td>2.4.0</td>
      <td>bio_hansel</td>
      <td>2.6.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>biohansel_bionumeric_converter</td>
      <td>bionumeric_convert</td>
      <td>Convert BioHansel output data to a Bionumerics friendly form</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>Text Manipulation</td>
      <td>biohansel_bionumeric_converter</td>
      <td>2019-03-15</td>
      <td>nml</td>
      <td></td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/biohansel_bionumeric_converter</td>
      <td>0.2.0</td>
      <td>pandas</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>blast_parser</td>
      <td>blast_parser</td>
      <td>Convert 12- or 24-column BLAST output into 3-column hcluster_sg input</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/TGAC/earlham-galaxytools/</td>
      <td>Phylogenetics</td>
      <td>blast_parser</td>
      <td>2015-08-06</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/blast_parser</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/blast_parser</td>
      <td>0.1.2</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>417</td>
      <td>438</td>
      <td>99</td>
      <td>104</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>417</td>
      <td>438</td>
      <td>99</td>
      <td>104</td>
    </tr>
    <tr>
      <td></td>
      <td>braker</td>
      <td>braker</td>
      <td>BRAKER is a pipeline for fully automated prediction of protein coding gene structures with GeneMark-ES/ET and AUGUSTUS in novel eukaryotic genomes .</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/Gaius-Augustus/BRAKER</td>
      <td>Genome annotation</td>
      <td>braker</td>
      <td>2021-10-05</td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/braker</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/braker</td>
      <td>2.1.6</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>165</td>
      <td>165</td>
      <td>48</td>
      <td>48</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>165</td>
      <td>165</td>
      <td>48</td>
      <td>48</td>
    </tr>
    <tr>
      <td></td>
      <td>braker3</td>
      <td>braker3</td>
      <td>BRAKER3 is a pipeline for fully automated prediction of protein coding gene structures with GeneMark-ES/ET and AUGUSTUS in novel eukaryotic genomes .</td>
      <td>braker3</td>
      <td>braker3</td>
      <td></td>
      <td>BRAKER3</td>
      <td>BRAKER3 is a pipeline for fully automated prediction of protein coding gene structures with GeneMark-ES/ET and AUGUSTUS in novel eukaryotic genomes</td>
      <td>Genome annotation, Gene prediction</td>
      <td>RNA-Seq, Genomics, Structure prediction, Sequence analysis</td>
      <td>To update</td>
      <td>https://github.com/Gaius-Augustus/BRAKER</td>
      <td>Genome annotation</td>
      <td>braker3</td>
      <td>2023-07-14</td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/braker</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/braker3</td>
      <td>3.0.8</td>
      <td></td>
      <td></td>
      <td>Genome annotation, Gene prediction</td>
      <td>RNA-Seq, Genomics, Structure prediction, Sequence analysis</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1938</td>
      <td>1938</td>
      <td>294</td>
      <td>294</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>412</td>
      <td>412</td>
      <td>71</td>
      <td>71</td>
      <td>2350</td>
      <td>2350</td>
      <td>365</td>
      <td>365</td>
    </tr>
    <tr>
      <td></td>
      <td>bundle_collections</td>
      <td>bundle_collection</td>
      <td>Tool to bundle up list collection into a single zip to be download</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>bundle_collections</td>
      <td>2015-11-20</td>
      <td>nml</td>
      <td></td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/bundle_collections</td>
      <td>1.3.0</td>
      <td>perl-getopt-long</td>
      <td>2.58</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>1</td>
      <td>1</td>
      <td>1518</td>
      <td>1526</td>
      <td>73</td>
      <td>76</td>
      <td>1520</td>
      <td>1528</td>
      <td>74</td>
      <td>77</td>
    </tr>
    <tr>
      <td></td>
      <td>camera</td>
      <td>abims_CAMERA_annotateDiffreport, abims_CAMERA_combinexsAnnos</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Metabolomics</td>
      <td>camera</td>
      <td>2023-05-25</td>
      <td>workflow4metabolomics</td>
      <td></td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/camera</td>
      <td>1.48.0</td>
      <td>r-snow</td>
      <td>0.4_1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>1</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>677</td>
      <td>787</td>
      <td>114</td>
      <td>120</td>
      <td>60</td>
      <td>60</td>
      <td>30</td>
      <td>30</td>
      <td>734</td>
      <td>746</td>
      <td>26</td>
      <td>26</td>
      <td>1471</td>
      <td>1593</td>
      <td>170</td>
      <td>176</td>
    </tr>
    <tr>
      <td></td>
      <td>cfm</td>
      <td>cfmid</td>
      <td>Competitive Fragmentation Modeling (CFM)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://sourceforge.net/p/cfm-id/</td>
      <td>Metabolomics</td>
      <td>cfmid</td>
      <td>2019-03-07</td>
      <td>computational-metabolomics</td>
      <td>https://github.com/computational-metabolomics/cfm-galaxy/tree/master/tools/cfm</td>
      <td>https://github.com/computational-metabolomics/cfm-galaxy/tree/master/tools/cfm</td>
      <td>33</td>
      <td>cfm</td>
      <td>33</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>chado</td>
      <td>analysis_add_analysis, analysis_delete_analyses, analysis_get_analyses, export_export_fasta, export_export_gbk, export_export_gff3, expression_add_biomaterial, expression_add_expression, expression_delete_all_biomaterials, expression_delete_biomaterials, expression_get_biomaterials, feature_delete_features, feature_get_features, feature_load_fasta, feature_load_featureprops, feature_load_gff, feature_load_go, load_blast, load_interpro, organism_add_organism, organism_delete_all_organisms, organism_delete_organisms, organism_get_organisms, phylogeny_gene_families, phylogeny_gene_order, phylogeny_load_tree</td>
      <td>Galaxy tools allowing to load data into a remote Chado database.Chado is a member of the GMOD family of tools.https://github.com/galaxy-genome-annotation/python-chado</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxy-genome-annotation/python-chado</td>
      <td>Web Services</td>
      <td></td>
      <td>2018-11-05</td>
      <td>gga</td>
      <td>https://github.com/galaxy-genome-annotation/galaxy-tools/tree/master/tools/chado</td>
      <td>https://github.com/galaxy-genome-annotation/galaxy-tools/tree/master/tools/chado</td>
      <td></td>
      <td>python-chado</td>
      <td>2.3.9</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>collapse_collection</td>
      <td>collapse_dataset</td>
      <td>Collection tool that collapses a list of files into a single datasset in order of appears in collection</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>collapse_collections</td>
      <td>2015-12-10</td>
      <td>nml</td>
      <td></td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/collapse_collection</td>
      <td>5.1.0</td>
      <td>gawk</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>39518</td>
      <td>40471</td>
      <td>3555</td>
      <td>3629</td>
      <td>22007</td>
      <td>24841</td>
      <td>3536</td>
      <td>4045</td>
      <td>3969</td>
      <td>4101</td>
      <td>814</td>
      <td>835</td>
      <td>65494</td>
      <td>69413</td>
      <td>7905</td>
      <td>8509</td>
    </tr>
    <tr>
      <td></td>
      <td>color_deconvolution</td>
      <td>ip_color_deconvolution</td>
      <td>Color-deconvolution methods</td>
      <td>galaxy_image_analysis</td>
      <td>galaxy_image_analysis</td>
      <td></td>
      <td>Galaxy Image Analysis</td>
      <td>Galaxy Image Analysis is mainly developed by the Biomedical Computer Vision (BMCV) Group at Heidelberg University. Its dedication is to provide tools for image analysis and image processing for the Galaxy platform.</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>color_deconvolution</td>
      <td>2024-09-30</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/color_deconvolution/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/color_deconvolution</td>
      <td>0.8</td>
      <td>scikit-image</td>
      <td></td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>15</td>
      <td>16</td>
      <td>2</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>15</td>
      <td>16</td>
      <td>2</td>
      <td>3</td>
    </tr>
    <tr>
      <td></td>
      <td>colorize_labels</td>
      <td>colorize_labels</td>
      <td>Colorize label map</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>colorize_labels</td>
      <td>2024-03-12</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/colorize_labels</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/colorize_labels</td>
      <td>3.2.1</td>
      <td>networkx</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>40</td>
      <td>40</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>40</td>
      <td>40</td>
      <td>2</td>
      <td>2</td>
    </tr>
    <tr>
      <td></td>
      <td>combineJSON</td>
      <td>combine_json</td>
      <td>JSON collection tool that takes multiple JSON data arrays and combines them into a single JSON array.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>combine_json</td>
      <td>2018-03-02</td>
      <td>nml</td>
      <td></td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/combineJSON</td>
      <td>0.1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>combine_assembly_stats</td>
      <td>combine_stats</td>
      <td>Combine multiple Assemblystats datasets into a single tabular report</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>Assembly</td>
      <td>combine_assemblystats</td>
      <td>2017-11-08</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/combine_assembly_stats</td>
      <td>1.0</td>
      <td>perl-getopt-long</td>
      <td>2.58</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>combine_tabular_collection</td>
      <td>combine</td>
      <td>Combine Tabular Collection into a single file</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>combine_tabular_collection</td>
      <td>2017-02-06</td>
      <td>nml</td>
      <td></td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/combine_tabular_collection</td>
      <td>0.1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>concat_channels</td>
      <td>ip_concat_channels</td>
      <td>Concatenate images</td>
      <td>galaxy_image_analysis</td>
      <td>galaxy_image_analysis</td>
      <td></td>
      <td>Galaxy Image Analysis</td>
      <td>Galaxy Image Analysis is mainly developed by the Biomedical Computer Vision (BMCV) Group at Heidelberg University. Its dedication is to provide tools for image analysis and image processing for the Galaxy platform.</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>concat_channels</td>
      <td>2017-10-30</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/concat_channels/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/concat_channels</td>
      <td>0.3-1</td>
      <td>scikit-image</td>
      <td></td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>76</td>
      <td>82</td>
      <td>11</td>
      <td>12</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>76</td>
      <td>82</td>
      <td>11</td>
      <td>12</td>
    </tr>
    <tr>
      <td></td>
      <td>concat_paired</td>
      <td>concat_fastqs</td>
      <td>Concatenate paired datasets</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/concat</td>
      <td>Text Manipulation</td>
      <td>concat_paired</td>
      <td>2019-12-16</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/concat</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/concat_paired</td>
      <td>0.2</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>coordinates_of_roi</td>
      <td>ip_coordinates_of_roi</td>
      <td>Coordinates of ROI</td>
      <td>galaxy_image_analysis</td>
      <td>galaxy_image_analysis</td>
      <td></td>
      <td>Galaxy Image Analysis</td>
      <td>Galaxy Image Analysis is mainly developed by the Biomedical Computer Vision (BMCV) Group at Heidelberg University. Its dedication is to provide tools for image analysis and image processing for the Galaxy platform.</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>coordinates_of_roi</td>
      <td>2019-01-08</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/coordinates_of_roi/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/coordinates_of_roi</td>
      <td>0.0.4-2</td>
      <td>scikit-image</td>
      <td></td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>16077</td>
      <td>16547</td>
      <td>11</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>16077</td>
      <td>16547</td>
      <td>11</td>
      <td>11</td>
    </tr>
    <tr>
      <td></td>
      <td>correlation_analysis</td>
      <td>correlation_analysis</td>
      <td>[Metabolomics][W4M] Metabolites Correlation Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://workflow4metabolomics.org</td>
      <td>Metabolomics</td>
      <td>correlation_analysis</td>
      <td>2020-12-08</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/blob/master/tools/correlation_analysis/</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/correlation_analysis</td>
      <td>1.0.1+galaxy0</td>
      <td>r-batch</td>
      <td>1.1_4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>count_objects</td>
      <td>ip_count_objects</td>
      <td>Count Objects</td>
      <td>galaxy_image_analysis</td>
      <td>galaxy_image_analysis</td>
      <td></td>
      <td>Galaxy Image Analysis</td>
      <td>Galaxy Image Analysis is mainly developed by the Biomedical Computer Vision (BMCV) Group at Heidelberg University. Its dedication is to provide tools for image analysis and image processing for the Galaxy platform.</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>count_objects</td>
      <td>2019-01-08</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/count_objects/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/count_objects</td>
      <td>0.0.5-2</td>
      <td>scikit-image</td>
      <td></td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>314</td>
      <td>326</td>
      <td>73</td>
      <td>75</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>129</td>
      <td>129</td>
      <td>6</td>
      <td>6</td>
      <td>443</td>
      <td>455</td>
      <td>79</td>
      <td>81</td>
    </tr>
    <tr>
      <td></td>
      <td>cryptogenotyper</td>
      <td>CryptoGenotyper</td>
      <td>CryptoGenotyper is a standalone tool to *in-silico*  determine species and subtype based on SSU rRNA and gp60 markers.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/phac-nml/CryptoGenotyper</td>
      <td>Sequence Analysis</td>
      <td>cryptogenotyper</td>
      <td>2020-10-14</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/CryptoGenotyper</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/cryptogenotyper</td>
      <td>1.0</td>
      <td>cryptogenotyper</td>
      <td>1.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>9942</td>
      <td>9942</td>
      <td>26</td>
      <td>26</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9942</td>
      <td>9942</td>
      <td>26</td>
      <td>26</td>
    </tr>
    <tr>
      <td></td>
      <td>csvtk</td>
      <td>csvtk_awklike_filter, csvtk_awklike_mutate, csvtk_collapse, csvtk_concat, csvtk_convert, csvtk_correlation, csvtk_cut, csvtk_filter, csvtk_freq, csvtk_gather, csvtk_join, csvtk_mutate, csvtk_plot, csvtk_replace, csvtk_sample, csvtk_separate, csvtk_sort, csvtk_split, csvtk_summary, csvtk_uniq</td>
      <td>Rapid data investigation and manipulation of csv/tsv files</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bioinf.shenwei.me/csvtk/</td>
      <td>Text Manipulation</td>
      <td>csvtk</td>
      <td>2019-08-27</td>
      <td>nml</td>
      <td>https://github.com/shenwei356/csvtk</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/csvtk</td>
      <td>0.20.0</td>
      <td>csvtk</td>
      <td>0.30.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>20</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>curve_fitting</td>
      <td>ip_curve_fitting</td>
      <td>Polynomial curve fitting to data points</td>
      <td>galaxy_image_analysis</td>
      <td>galaxy_image_analysis</td>
      <td></td>
      <td>Galaxy Image Analysis</td>
      <td>Galaxy Image Analysis is mainly developed by the Biomedical Computer Vision (BMCV) Group at Heidelberg University. Its dedication is to provide tools for image analysis and image processing for the Galaxy platform.</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>To update</td>
      <td>https://github.com/BMCV/galaxy-image-analysis</td>
      <td>Imaging</td>
      <td>curve_fitting</td>
      <td>2021-07-22</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/curve_fitting/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/curve_fitting</td>
      <td>0.0.3-2</td>
      <td>numpy</td>
      <td></td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>12</td>
      <td>12</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>12</td>
      <td>12</td>
      <td>2</td>
      <td>2</td>
    </tr>
    <tr>
      <td></td>
      <td>detection_viz</td>
      <td>ip_detection_viz</td>
      <td>Detection Visualization</td>
      <td>galaxy_image_analysis</td>
      <td>galaxy_image_analysis</td>
      <td></td>
      <td>Galaxy Image Analysis</td>
      <td>Galaxy Image Analysis is mainly developed by the Biomedical Computer Vision (BMCV) Group at Heidelberg University. Its dedication is to provide tools for image analysis and image processing for the Galaxy platform.</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>detection_viz</td>
      <td>2017-10-30</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/detection_viz/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/detection_viz</td>
      <td>0.3-2</td>
      <td>scikit-image</td>
      <td></td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>43</td>
      <td>48</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>43</td>
      <td>48</td>
      <td>8</td>
      <td>8</td>
    </tr>
    <tr>
      <td></td>
      <td>ectyper</td>
      <td>ectyper</td>
      <td>EC-Typer - in silico serotyping of Escherichia coli species</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/phac-nml/ecoli_serotyping</td>
      <td>Sequence Analysis</td>
      <td>ectyper</td>
      <td>2018-12-21</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/ecoli_serotyping</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/ectyper</td>
      <td>1.0.0</td>
      <td>ectyper</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>12996</td>
      <td>12996</td>
      <td>137</td>
      <td>137</td>
      <td>10</td>
      <td>10</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>13006</td>
      <td>13006</td>
      <td>138</td>
      <td>138</td>
    </tr>
    <tr>
      <td></td>
      <td>ete</td>
      <td>ete_gene_csv_finder, ete_genetree_splitter, ete_homology_classifier, ete_init_taxdb, ete_lineage_generator, ete3_mod, ete_species_tree_generator</td>
      <td>Analyse phylogenetic trees using the ETE Toolkit</td>
      <td>ete</td>
      <td>ete</td>
      <td></td>
      <td>ete</td>
      <td>The Environment for Tree Exploration (ETE) is a computational framework that simplifies the reconstruction, analysis, and visualization of phylogenetic trees and multiple sequence alignments. Here, we present ETE v3, featuring numerous improvements in the underlying library of methods, and providing a novel set of standalone tools to perform common tasks in comparative genomics and phylogenetics. The new features include (i) building gene-based and supermatrix-based phylogenies using a single command, (ii) testing and visualizing evolutionary models, (iii) calculating distances between trees of different size or including duplications, and (iv) providing seamless integration with the NCBI taxonomy database. ETE is freely available at http://etetoolkit.org</td>
      <td>Phylogenetic analysis, Phylogenetic tree editing</td>
      <td>Phylogenetics</td>
      <td>To update</td>
      <td>http://etetoolkit.org/</td>
      <td>Phylogenetics</td>
      <td>ete</td>
      <td>2018-10-11</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/ete</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/ete</td>
      <td>3.1.2</td>
      <td>ete3</td>
      <td>3.1.1</td>
      <td>Phylogenetic tree editing</td>
      <td>Phylogenetics</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>820</td>
      <td>966</td>
      <td>89</td>
      <td>106</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>820</td>
      <td>966</td>
      <td>89</td>
      <td>106</td>
    </tr>
    <tr>
      <td></td>
      <td>export_to_cluster</td>
      <td>export_to_cluster</td>
      <td>Export datasets to cluster</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/TGAC/earlham-galaxytools/</td>
      <td>Data Export</td>
      <td>export_to_cluster</td>
      <td>2016-01-18</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/export_to_cluster/</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/export_to_cluster</td>
      <td>0.0.2</td>
      <td>EXPORT_DIR_PREFIX</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>fasta2bed</td>
      <td>fasta2bed</td>
      <td>Convert multiple fasta file into tabular bed file format</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>Sequence Analysis</td>
      <td>fasta2bed</td>
      <td>2017-10-11</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/fasta2bed</td>
      <td>1.0.0</td>
      <td>perl-bioperl</td>
      <td>1.7.8</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>fasta_extract</td>
      <td>fa-extract-sequence</td>
      <td>extract single fasta from multiple fasta file</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://toolshed.g2.bx.psu.edu/view/nml/fasta_extract</td>
      <td>Sequence Analysis</td>
      <td>fasta_extract</td>
      <td>2016-10-26</td>
      <td>nml</td>
      <td>https://toolshed.g2.bx.psu.edu/view/nml/fasta_extract</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/fasta_extract</td>
      <td>1.1.0</td>
      <td>perl-bioperl</td>
      <td>1.7.8</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>10</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1286</td>
      <td>1827</td>
      <td>144</td>
      <td>181</td>
      <td>1296</td>
      <td>1837</td>
      <td>145</td>
      <td>182</td>
    </tr>
    <tr>
      <td></td>
      <td>fastqc_stats</td>
      <td>FastQC_Summary</td>
      <td>Summary multiple FastQC into a single tabular line report</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>Sequence Analysis</td>
      <td>fastqc_stats</td>
      <td>2017-10-11</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/fastqc_stats</td>
      <td>1.2</td>
      <td>perl-bioperl</td>
      <td>1.7.8</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>feelnc2asko</td>
      <td>feelnc2asko</td>
      <td>Convert FeelNC GTF to GFF3</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/tderrien/FEELnc</td>
      <td>Convert Formats</td>
      <td>feelnc2asko</td>
      <td>2018-04-11</td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/feelnc2asko</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/feelnc2asko</td>
      <td>0.1</td>
      <td>perl-bioperl</td>
      <td>1.7.8</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>feht</td>
      <td>feht</td>
      <td>Automatically identify makers predictive of groups.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>Sequence Analysis</td>
      <td>feht</td>
      <td>2017-10-11</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/feht</td>
      <td>0.1.0</td>
      <td>feht</td>
      <td>1.1.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>filter_spades_repeats</td>
      <td>filter_spades_repeat</td>
      <td>Remove short and repeat contigs/scaffolds</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/galaxy_tools/</td>
      <td>Assembly</td>
      <td>filter_spades_repeats</td>
      <td>2017-10-12</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools/</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/filter_spades_repeats</td>
      <td>1.0.1</td>
      <td>perl-bioperl</td>
      <td>1.7.8</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>gblocks</td>
      <td>gblocks</td>
      <td>Gblocks</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://molevol.cmima.csic.es/castresana/Gblocks.html</td>
      <td>Sequence Analysis</td>
      <td>gblocks</td>
      <td>2018-04-11</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/gblocks</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/gblocks</td>
      <td>0.91b</td>
      <td>gblocks</td>
      <td>0.91b</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>466</td>
      <td>466</td>
      <td>62</td>
      <td>62</td>
      <td>466</td>
      <td>466</td>
      <td>62</td>
      <td>62</td>
    </tr>
    <tr>
      <td></td>
      <td>gcms2isocor</td>
      <td>gcms2isocor</td>
      <td>Corrective method dedicated to Isocor for calculating carbon isotopologue distribution from GCMS runs</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/p2m2/p2m2tools</td>
      <td>Metabolomics</td>
      <td>gcms2isocor</td>
      <td>2023-03-15</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/blob/master/tools/gcms2isocor/</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/gcms2isocor</td>
      <td>0.2.1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>gcms2isocor</td>
      <td>gcms2isocor</td>
      <td>Conversion from GCMS PostRun Analysis to Isocor</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Metabolomics</td>
      <td>gcms2isocor</td>
      <td>2021-09-16</td>
      <td>genouest</td>
      <td></td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/gcms2isocor</td>
      <td>0.1.0</td>
      <td>openjdk</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>genenotebook</td>
      <td>genenotebook_build</td>
      <td>Galaxy tools allowing to load data into a GeneNoteBook database.https://genenotebook.github.io</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://genenotebook.github.io</td>
      <td>Web Services</td>
      <td></td>
      <td>2023-04-07</td>
      <td>gga</td>
      <td>https://github.com/galaxy-genome-annotation/galaxy-tools/tree/master/tools/genenotebook</td>
      <td>https://github.com/galaxy-genome-annotation/galaxy-tools/tree/master/tools/genenotebook</td>
      <td>0.4.16</td>
      <td>genoboo</td>
      <td>0.4.16</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
    </tr>
    <tr>
      <td></td>
      <td>genform</td>
      <td>genform</td>
      <td>genform: generation of molecular formulas by high-resolution MS and MS/MS data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://sourceforge.net/projects/genform/</td>
      <td>Metabolomics</td>
      <td>genform</td>
      <td>2019-09-12</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/blob/master/tools/genform/</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/genform</td>
      <td></td>
      <td>genform</td>
      <td>r8</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>68</td>
      <td>68</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>68</td>
      <td>68</td>
      <td>4</td>
      <td>4</td>
    </tr>
    <tr>
      <td></td>
      <td>get_pairs</td>
      <td>get_pairs</td>
      <td>Separate paired and unpaired reads from two fastq files</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Fastq Manipulation</td>
      <td>get_pairs</td>
      <td>2018-09-10</td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/get_pairs</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/get_pairs</td>
      <td>0.3</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>getmlst</td>
      <td>getmlst</td>
      <td>Download MLST datasets by species from pubmlst.org</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>getmlst</td>
      <td>2015-12-03</td>
      <td>nml</td>
      <td></td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/getmlst</td>
      <td>0.1.4.1</td>
      <td>srst2</td>
      <td>0.2.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>gnali</td>
      <td>gnali</td>
      <td>A tool to find nonessential, loss-of-function gene variants</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/phac-nml/gnali/</td>
      <td>Variant Analysis</td>
      <td>gnali</td>
      <td>2020-03-30</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/gnali/</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/gnali</td>
      <td>1.1.0</td>
      <td>gnali</td>
      <td>1.1.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>gstf_preparation</td>
      <td>gstf_preparation</td>
      <td>GeneSeqToFamily preparation converts data for the workflow</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/TGAC/earlham-galaxytools/</td>
      <td>Convert Formats</td>
      <td>gstf_preparation</td>
      <td>2018-04-24</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/gstf_preparation</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/gstf_preparation</td>
      <td>0.4.3</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>377</td>
      <td>400</td>
      <td>17</td>
      <td>17</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>377</td>
      <td>400</td>
      <td>17</td>
      <td>17</td>
    </tr>
    <tr>
      <td></td>
      <td>hcluster_sg</td>
      <td>hcluster_sg</td>
      <td>Hierarchically clustering on a sparse graph</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/douglasgscofield/hcluster</td>
      <td>Phylogenetics</td>
      <td>hcluster_sg</td>
      <td>2015-08-05</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/hcluster_sg</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/hcluster_sg</td>
      <td>0.5.1.1</td>
      <td>hcluster_sg</td>
      <td>0.5.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>243</td>
      <td>263</td>
      <td>54</td>
      <td>57</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>243</td>
      <td>263</td>
      <td>54</td>
      <td>57</td>
    </tr>
    <tr>
      <td></td>
      <td>hcluster_sg_parser</td>
      <td>hcluster_sg_parser</td>
      <td>Converts hcluster_sg 3-column output into lists of ids</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/TGAC/earlham-galaxytools/</td>
      <td>Phylogenetics</td>
      <td>hcluster_sg_parser</td>
      <td>2015-08-13</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/hcluster_sg_parser</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/hcluster_sg_parser</td>
      <td>0.2.1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>266</td>
      <td>293</td>
      <td>22</td>
      <td>25</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>266</td>
      <td>293</td>
      <td>22</td>
      <td>25</td>
    </tr>
    <tr>
      <td></td>
      <td>helixer</td>
      <td>helixer</td>
      <td>Gene calling with Deep Neural Networks</td>
      <td>helixer</td>
      <td>helixer</td>
      <td></td>
      <td>Helixer</td>
      <td>Deep Learning to predict gene annotations</td>
      <td>Gene prediction, Genome annotation</td>
      <td>Sequence analysis, Gene transcripts</td>
      <td>To update</td>
      <td>https://github.com/weberlab-hhu/Helixer</td>
      <td>Genome annotation</td>
      <td>helixer</td>
      <td>2023-06-14</td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/helixer</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/helixer</td>
      <td>0.3.3</td>
      <td></td>
      <td></td>
      <td>Gene prediction, Genome annotation</td>
      <td>Sequence analysis, Gene transcripts</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>267</td>
      <td>267</td>
      <td>39</td>
      <td>39</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>80</td>
      <td>80</td>
      <td>7</td>
      <td>7</td>
      <td>347</td>
      <td>347</td>
      <td>46</td>
      <td>46</td>
    </tr>
    <tr>
      <td></td>
      <td>hivtrace</td>
      <td>hivtrace</td>
      <td>An application that identifies potential transmission clusters within a supplied FASTA file with an option to find potential links against the Los Alamos HIV Sequence Database.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>hivtrace</td>
      <td>2015-12-10</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/tools/hivtrace</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/hivtrace</td>
      <td>1.0.1</td>
      <td>hivtrace</td>
      <td>1.5.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>image_info</td>
      <td>ip_imageinfo</td>
      <td>Extracts image metadata</td>
      <td></td>
      <td></td>
      <td>python-bioformats</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>image_info</td>
      <td>2019-07-19</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/image_info/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/image_info</td>
      <td>5.7.1</td>
      <td>bftools</td>
      <td>6.7.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>273</td>
      <td>659</td>
      <td>101</td>
      <td>102</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>153</td>
      <td>153</td>
      <td>15</td>
      <td>15</td>
      <td>426</td>
      <td>812</td>
      <td>116</td>
      <td>117</td>
    </tr>
    <tr>
      <td></td>
      <td>image_math</td>
      <td>image_math</td>
      <td>Process images using arithmetic expressions</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>image_math</td>
      <td>2024-03-09</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/image_math</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/image_math</td>
      <td>1.26.4</td>
      <td>numpy</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>58</td>
      <td>58</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>58</td>
      <td>58</td>
      <td>2</td>
      <td>2</td>
    </tr>
    <tr>
      <td></td>
      <td>image_registration_affine</td>
      <td>ip_image_registration</td>
      <td>Intensity-based Image Registration</td>
      <td>galaxy_image_analysis</td>
      <td>galaxy_image_analysis</td>
      <td></td>
      <td>Galaxy Image Analysis</td>
      <td>Galaxy Image Analysis is mainly developed by the Biomedical Computer Vision (BMCV) Group at Heidelberg University. Its dedication is to provide tools for image analysis and image processing for the Galaxy platform.</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>image_registration_affine</td>
      <td>2020-12-29</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/image_registration_affine/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/image_registration_affine</td>
      <td>0.0.3-2</td>
      <td>scikit-image</td>
      <td></td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>16</td>
      <td>16</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>16</td>
      <td>16</td>
      <td>5</td>
      <td>5</td>
    </tr>
    <tr>
      <td></td>
      <td>imagecoordinates_flipaxis</td>
      <td>imagecoordinates_flipaxis</td>
      <td>Flip coordinate axes</td>
      <td>galaxy_image_analysis</td>
      <td>galaxy_image_analysis</td>
      <td></td>
      <td>Galaxy Image Analysis</td>
      <td>Galaxy Image Analysis is mainly developed by the Biomedical Computer Vision (BMCV) Group at Heidelberg University. Its dedication is to provide tools for image analysis and image processing for the Galaxy platform.</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>imagecoordinates_flipaxis</td>
      <td>2019-03-08</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/imagecoordinates_flipaxis/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/imagecoordinates_flipaxis</td>
      <td>0.1-2</td>
      <td>pandas</td>
      <td></td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>21409</td>
      <td>21849</td>
      <td>10</td>
      <td>12</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>21409</td>
      <td>21849</td>
      <td>10</td>
      <td>12</td>
    </tr>
    <tr>
      <td></td>
      <td>influx_data_manager</td>
      <td>influx_data_manager</td>
      <td>Handling influx_si data inputs in Galaxy workflows</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/MetaboHUB-MetaToul-FluxoMet/tools_w4m/</td>
      <td>Metabolomics</td>
      <td>influx_si_data_manager</td>
      <td>2024-02-09</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/MetaboHUB-MetaToul-FluxoMet/tools_w4m/tree/main/tools/influx_data_manager</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/influx_data_manager</td>
      <td>1.0.2</td>
      <td>influx-si-data-manager</td>
      <td>1.0.3</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>influx_si</td>
      <td>influx_si</td>
      <td>metabolic flux estimation based on [in]stationary labeling</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/sgsokol/influx</td>
      <td>Metabolomics</td>
      <td>influx_si</td>
      <td>2020-04-17</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/blob/master/tools/influx_si/</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/influx_si</td>
      <td>7.0.1</td>
      <td>influx_si</td>
      <td>7.1.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>ipo</td>
      <td>ipo4retgroup, ipo4xcmsSet</td>
      <td>[W4M][LC-MS] IPO</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/rietho/IPO</td>
      <td>Metabolomics</td>
      <td>ipo</td>
      <td>2019-10-03</td>
      <td>lecorguille</td>
      <td>https://github.com/rietho/IPO</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/ipo</td>
      <td>1.10.0</td>
      <td>bioconductor-ipo</td>
      <td>1.28.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>isoplot</td>
      <td>isoplot</td>
      <td>Isoplot is a software for the visualisation of MS data from C13 labelling experiments</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Metabolomics, Visualization</td>
      <td>isoplot</td>
      <td>2021-07-12</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/llegregam/Isoplot/tree/main</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/isoplot</td>
      <td>1.3.0+galaxy1</td>
      <td>isoplot</td>
      <td>1.3.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>3</td>
      <td>3</td>
    </tr>
    <tr>
      <td></td>
      <td>jbrowse</td>
      <td>jbrowse_to_container</td>
      <td>A tool allowing to export a JBrowse dataset into a JBrowse docker container</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://jbrowse.org</td>
      <td>Web Services</td>
      <td>jbrowse_to_container</td>
      <td>2016-11-08</td>
      <td>gga</td>
      <td>https://github.com/galaxy-genome-annotation/galaxy-tools/tree/master/tools/jbrowse</td>
      <td>https://github.com/galaxy-genome-annotation/galaxy-tools/tree/master/tools/jbrowse</td>
      <td></td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>kaptive</td>
      <td>kaptive</td>
      <td>Kaptive reports information about capsular (K) loci found in genome assemblies.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>kaptive</td>
      <td>2017-04-10</td>
      <td>nml</td>
      <td></td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/kaptive</td>
      <td>0.3.0</td>
      <td>kaptive</td>
      <td>3.0.0b5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>kat_filter</td>
      <td>kat_@EXECUTABLE@</td>
      <td>Filtering kmers or reads from a database of kmers hashes</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>kat_filter</td>
      <td>2017-03-13</td>
      <td>nml</td>
      <td></td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/kat_filter</td>
      <td>2.3</td>
      <td>kat</td>
      <td>2.4.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>kat_sect</td>
      <td>kat_@EXECUTABLE@</td>
      <td>SEquence Coverage estimator Tool. Estimates the coverage of each sequence in a file using K-mers from another sequence file.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td>kat_sect</td>
      <td>2017-02-06</td>
      <td>nml</td>
      <td></td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/kat_sect</td>
      <td>2.3</td>
      <td>kat</td>
      <td>2.4.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>kegalign</td>
      <td>kegalign</td>
      <td>A Scalable GPU System for Pairwise Whole Genome Alignments based on LASTZ's seed-filter-extend paradigm.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproject/KegAlign</td>
      <td>Next Gen Mappers</td>
      <td>kegalign</td>
      <td>2024-08-29</td>
      <td>richard-burhans</td>
      <td>https://github.com/richard-burhans/galaxytools/tree/main/tools/kegalign</td>
      <td>https://github.com/richard-burhans/galaxytools/tree/main/tools/kegalign</td>
      <td>0.1.2.7</td>
      <td>kegalign-full</td>
      <td>0.1.2.7</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>1</td>
      <td>1</td>
    </tr>
    <tr>
      <td></td>
      <td>kmd_hmdb_data_plot</td>
      <td>kmd_hmdb_data_plot</td>
      <td>retrieves data from KMD HMDB API and produce plot and csv file</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics</td>
      <td>Metabolomics</td>
      <td>kmd_hmdb_data_plot</td>
      <td>2023-07-10</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/blob/master/tools/kmd_hmdb_data_plot/</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/kmd_hmdb_data_plot</td>
      <td>1.0.0</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>label_to_binary</td>
      <td>label_to_binary</td>
      <td>Convert label map to binary image</td>
      <td>galaxy_image_analysis</td>
      <td>galaxy_image_analysis</td>
      <td></td>
      <td>Galaxy Image Analysis</td>
      <td>Galaxy Image Analysis is mainly developed by the Biomedical Computer Vision (BMCV) Group at Heidelberg University. Its dedication is to provide tools for image analysis and image processing for the Galaxy platform.</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>label_to_binary</td>
      <td>2024-03-13</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/label_to_binary/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/label_to_binary</td>
      <td>1.26.4</td>
      <td>numpy</td>
      <td></td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>labelimage2points</td>
      <td>ip_labelimage_to_points</td>
      <td>Label Image to Points</td>
      <td>galaxy_image_analysis</td>
      <td>galaxy_image_analysis</td>
      <td></td>
      <td>Galaxy Image Analysis</td>
      <td>Galaxy Image Analysis is mainly developed by the Biomedical Computer Vision (BMCV) Group at Heidelberg University. Its dedication is to provide tools for image analysis and image processing for the Galaxy platform.</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>labelimage2points</td>
      <td>2017-10-30</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/labelimage2points/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/labelimage2points</td>
      <td>0.2-2</td>
      <td>scikit-image</td>
      <td></td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>34</td>
      <td>38</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>34</td>
      <td>38</td>
      <td>2</td>
      <td>2</td>
    </tr>
    <tr>
      <td></td>
      <td>landmark_registration</td>
      <td>ip_landmark_registration</td>
      <td>Landmark Registration</td>
      <td>galaxy_image_analysis</td>
      <td>galaxy_image_analysis</td>
      <td></td>
      <td>Galaxy Image Analysis</td>
      <td>Galaxy Image Analysis is mainly developed by the Biomedical Computer Vision (BMCV) Group at Heidelberg University. Its dedication is to provide tools for image analysis and image processing for the Galaxy platform.</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>landmark_registration</td>
      <td>2019-02-04</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/landmark_registration/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/landmark_registration</td>
      <td>0.1.0-2</td>
      <td>scikit-image</td>
      <td></td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>887</td>
      <td>927</td>
      <td>13</td>
      <td>15</td>
      <td>4</td>
      <td>4</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>891</td>
      <td>931</td>
      <td>16</td>
      <td>18</td>
    </tr>
    <tr>
      <td></td>
      <td>logol</td>
      <td>logol_wrapper</td>
      <td>Logol is a pattern matching grammar language and a set of tools to search a pattern in a sequence</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://logol.genouest.org/web/app.php/logol</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>2018-06-06</td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/logol</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/logol</td>
      <td>1.7.8</td>
      <td>logol</td>
      <td>1.7.8</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>lotus2</td>
      <td>lotus2</td>
      <td>LotuS2 OTU processing pipeline</td>
      <td>lotus2</td>
      <td>lotus2</td>
      <td></td>
      <td>lotus2</td>
      <td>LotuS2 is a lightweight and user-friendly pipeline that is fast, precise, and streamlined, using extensive pre- and post-ASV/OTU clustering steps to further increase data quality. High data usage rates and reliability enable high-throughput microbiome analysis in minutes.</td>
      <td>Sequence feature detection, DNA barcoding</td>
      <td>Metagenomics, Taxonomy, Microbial ecology</td>
      <td>To update</td>
      <td>http://lotus2.earlham.ac.uk/</td>
      <td>Metagenomics</td>
      <td>lotus2</td>
      <td>2021-06-08</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/lotus2</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/lotus2</td>
      <td>2.32</td>
      <td>lotus2</td>
      <td>2.34.1</td>
      <td>Sequence feature detection, DNA barcoding</td>
      <td>Metagenomics, Taxonomy, Microbial ecology</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1219</td>
      <td>1219</td>
      <td>182</td>
      <td>182</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>37</td>
      <td>37</td>
      <td>4</td>
      <td>4</td>
      <td>1256</td>
      <td>1256</td>
      <td>186</td>
      <td>186</td>
    </tr>
    <tr>
      <td></td>
      <td>mahotas_features</td>
      <td>ip_mahotas_features</td>
      <td>Compute image features using mahotas.</td>
      <td></td>
      <td></td>
      <td>mahotas-feature-computation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/luispedro/mahotas</td>
      <td>Imaging</td>
      <td>mahotas_features</td>
      <td>2024-09-30</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/mahotas-features/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/mahotas_features</td>
      <td>0.7-2</td>
      <td>mahotas</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>39</td>
      <td>44</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>39</td>
      <td>44</td>
      <td>6</td>
      <td>6</td>
    </tr>
    <tr>
      <td></td>
      <td>mauve_contig_mover</td>
      <td>mauve_contig_mover</td>
      <td>Order a draft genome relative to a related reference genome</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/mauve_contig_mover</td>
      <td>Sequence Analysis</td>
      <td>mauve_contig_mover</td>
      <td>2019-11-19</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/mauve_contig_mover</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/mauve_contig_mover</td>
      <td>1.0.10</td>
      <td>mauve</td>
      <td>2.4.0.r4736</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>meneco</td>
      <td>meneco</td>
      <td>Meneco computes minimal completions to your draft network with reactions from a repair network</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://bioasp.github.io/meneco/</td>
      <td>Systems Biology</td>
      <td>meneco</td>
      <td>2018-06-11</td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/meneco</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/meneco</td>
      <td>1.5.2</td>
      <td>meneco</td>
      <td>1.5.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>mergeneighboursinlabelimage</td>
      <td>ip_merge_neighbours_in_label</td>
      <td>Merge Neighbours in Label Image</td>
      <td>galaxy_image_analysis</td>
      <td>galaxy_image_analysis</td>
      <td></td>
      <td>Galaxy Image Analysis</td>
      <td>Galaxy Image Analysis is mainly developed by the Biomedical Computer Vision (BMCV) Group at Heidelberg University. Its dedication is to provide tools for image analysis and image processing for the Galaxy platform.</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>mergeneighboursinlabelimage</td>
      <td>2019-01-08</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/\mergeneighboursinlabelimage</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/mergeneighboursinlabelimage</td>
      <td>0.3-2</td>
      <td>scikit-image</td>
      <td></td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>18</td>
      <td>25</td>
      <td>2</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>18</td>
      <td>25</td>
      <td>2</td>
      <td>3</td>
    </tr>
    <tr>
      <td></td>
      <td>metfrag</td>
      <td>metfrag</td>
      <td>[Metabolomics][LC-MS][MS/MS] MetFrag - Perform metfrag analysis (can work with the output from msPurity)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/computational-metabolomics/metfrag-galaxy</td>
      <td>Metabolomics</td>
      <td></td>
      <td>2019-08-01</td>
      <td>computational-metabolomics</td>
      <td>https://github.com/computational-metabolomics/metfrag-galaxy</td>
      <td>https://github.com/computational-metabolomics/metfrag-galaxy/tree/master/tools/metfrag</td>
      <td>2.4.5+galaxy3</td>
      <td>metfrag</td>
      <td>2.4.5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>192</td>
      <td>192</td>
      <td>20</td>
      <td>20</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>192</td>
      <td>192</td>
      <td>20</td>
      <td>20</td>
    </tr>
    <tr>
      <td></td>
      <td>metfrag-vis</td>
      <td>metfrag_vis</td>
      <td>[Metabolomics][LC-MS][MS/MS] This tool summarises the results generated by MetFrag</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/computational-metabolomics/metfrag-galaxy</td>
      <td>Metabolomics</td>
      <td></td>
      <td>2020-06-30</td>
      <td>computational-metabolomics</td>
      <td>https://github.com/computational-metabolomics/metfrag-galaxy</td>
      <td>https://github.com/computational-metabolomics/metfrag-galaxy/tree/master/tools/metfrag-vis</td>
      <td>2.4.5+galaxy0</td>
      <td>metfrag</td>
      <td>2.4.5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>50</td>
      <td>50</td>
      <td>14</td>
      <td>14</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>50</td>
      <td>50</td>
      <td>14</td>
      <td>14</td>
    </tr>
    <tr>
      <td></td>
      <td>miranda</td>
      <td>miranda</td>
      <td>Finds potential target sites for miRNAs in genomic sequences</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.microrna.org/</td>
      <td>RNA</td>
      <td>miranda</td>
      <td>2015-10-30</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/miranda</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/miranda</td>
      <td>3.3a+galaxy1</td>
      <td>miranda</td>
      <td>3.3a</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6408</td>
      <td>6514</td>
      <td>94</td>
      <td>99</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6408</td>
      <td>6514</td>
      <td>94</td>
      <td>99</td>
    </tr>
    <tr>
      <td></td>
      <td>miranda2asko</td>
      <td>miranda2asko</td>
      <td>Converts miRanda output into AskOmics format</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Convert Formats</td>
      <td>miranda2asko</td>
      <td>2018-04-12</td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/miranda2asko</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/miranda2asko</td>
      <td>0.2</td>
      <td>perl</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>mixmodel4repeated_measures</td>
      <td>mixmodel4repeated_measures</td>
      <td>[Metabolomics][W4M][Statistics] Mixed models - Analysis of variance for repeated measures using mixed model</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://workflow4metabolomics.org</td>
      <td>Metabolomics</td>
      <td>mixmodel4repeated_measures</td>
      <td>2022-05-16</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/mixmodel4repeated_measures</td>
      <td>3.1.0</td>
      <td>r-lme4</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>mob_suite</td>
      <td>mob_recon, mob_typer</td>
      <td>MOB-suite is a set of software tools for clustering, reconstruction and typing of plasmids from draft assemblies</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/mob-suite</td>
      <td>Sequence Analysis</td>
      <td>mob_suite</td>
      <td>2018-08-20</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/mob-suite</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/mob_suite</td>
      <td>3.0.3</td>
      <td>mob_suite</td>
      <td>3.1.9</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>119024</td>
      <td>119024</td>
      <td>628</td>
      <td>628</td>
      <td>42</td>
      <td>42</td>
      <td>2</td>
      <td>2</td>
      <td>36310</td>
      <td>36310</td>
      <td>191</td>
      <td>191</td>
      <td>155376</td>
      <td>155376</td>
      <td>821</td>
      <td>821</td>
    </tr>
    <tr>
      <td></td>
      <td>morphological_operations</td>
      <td>morphological_operations</td>
      <td>Apply morphological operations to images</td>
      <td></td>
      <td></td>
      <td>scipy</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>morphological_operations</td>
      <td>2024-03-08</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/morphological_operations</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/morphological_operations</td>
      <td>1.12.0</td>
      <td>scipy</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>mrbayes</td>
      <td>mrbayes</td>
      <td>A program for the Bayesian estimation of phylogeny.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>mrbayes</td>
      <td>2015-12-04</td>
      <td>nml</td>
      <td></td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/mrbayes</td>
      <td>1.0.2</td>
      <td>mrbayes</td>
      <td>3.2.7</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>ms2snoop</td>
      <td>ms2snoop</td>
      <td>[W4M][Utils] Extraction et nettoyage des spectre MS1/2 post-traitement MSPurity.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://workflow4metabolomics.org</td>
      <td>Metabolomics</td>
      <td>ms2snoop</td>
      <td>2022-03-15</td>
      <td>workflow4metabolomics</td>
      <td></td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/ms2snoop</td>
      <td>2.2.1</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>msPurity</td>
      <td>mspurity_averagefragspectra, mspurity_combineannotations, mspurity_createdatabase, mspurity_createmsp, mspurity_dimspredictpuritysingle, mspurity_filterfragspectra, mspurity_flagremove, mspurity_frag4feature, mspurity_puritya, mspurity_purityx, mspurity_spectralmatching</td>
      <td>[Metabolomics][LC-MS][DIMS][MS/MS] msPurity -  R package for precursor ion purity assessments, data processing and spectral matching for LC-MS(/MS) and DI-MS(/MS) data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/computational-metabolomics/mspurity-galaxy</td>
      <td>Metabolomics</td>
      <td></td>
      <td>2019-05-24</td>
      <td>computational-metabolomics</td>
      <td>https://github.com/computational-metabolomics/mspurity-galaxy</td>
      <td>https://github.com/computational-metabolomics/mspurity-galaxy/tree/master/tools/msPurity</td>
      <td>1.28.0</td>
      <td>bioconductor-mspurity</td>
      <td>1.28.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>848</td>
      <td>848</td>
      <td>91</td>
      <td>91</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>848</td>
      <td>848</td>
      <td>91</td>
      <td>91</td>
    </tr>
    <tr>
      <td></td>
      <td>msp_split</td>
      <td>msp_split</td>
      <td>[Metabolomics][LC-MS][DIMS][MS/MS] msPurity -  R package for precursor ion purity assessments, data processing and spectral matching for LC-MS(/MS) and DI-MS(/MS) data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/computational-metabolomics/mspurity-galaxy</td>
      <td>Metabolomics</td>
      <td></td>
      <td>2019-09-24</td>
      <td>tomnl</td>
      <td>https://github.com/computational-metabolomics/mspurity-galaxy</td>
      <td>https://github.com/computational-metabolomics/mspurity-galaxy/tree/master/tools/utils/msp_split</td>
      <td>0.0.2</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>mykrobe_parser</td>
      <td>mykrobe_parseR</td>
      <td>RScript to parse the results of mykrobe predictor.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/mykrobe-parser</td>
      <td>Sequence Analysis</td>
      <td>mykrobe_parser</td>
      <td>2018-09-28</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/mykrobe-parser</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/mykrobe_parser</td>
      <td>0.1.4.1</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>mzxmlshaper</td>
      <td>mzxmlshaper</td>
      <td>Convert mzML, mzXML or netCDF files to mzML or mzXML</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://rdrr.io/bioc/mzR/</td>
      <td>Metabolomics</td>
      <td>mzxmlshaper</td>
      <td>2024-09-20</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/mzxmlshaper</td>
      <td>1.0.0+galaxy0</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>ncbi_egapx</td>
      <td>ncbi_egapx</td>
      <td>Eukaryotic Genome Annotation Pipeline - External (EGAPx)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ncbi/egapx</td>
      <td>Genome annotation</td>
      <td>ncbi_egapx</td>
      <td>2024-08-19</td>
      <td>richard-burhans</td>
      <td>https://github.com/richard-burhans/galaxytools/tree/main/tools/ncbi_egapx</td>
      <td>https://github.com/richard-burhans/galaxytools/tree/main/tools/ncbi_egapx</td>
      <td>0.2-alpha</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>15</td>
      <td>15</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>15</td>
      <td>15</td>
      <td>6</td>
      <td>6</td>
    </tr>
    <tr>
      <td></td>
      <td>ncbi_fcs_adaptor</td>
      <td>ncbi_fcs_adaptor</td>
      <td>FCS-adaptor detects adaptor and vector contamination in genome sequences.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ncbi/fcs</td>
      <td>Sequence Analysis</td>
      <td>ncbi_fcs_adaptor</td>
      <td>2024-02-23</td>
      <td>richard-burhans</td>
      <td>https://github.com/richard-burhans/galaxytools/tree/main/tools/ncbi_fcs_adaptor</td>
      <td>https://github.com/richard-burhans/galaxytools/tree/main/tools/ncbi_fcs_adaptor</td>
      <td>0.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>842</td>
      <td>842</td>
      <td>93</td>
      <td>93</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>842</td>
      <td>842</td>
      <td>93</td>
      <td>93</td>
    </tr>
    <tr>
      <td></td>
      <td>nmr_annotation</td>
      <td>NmrAnnotation</td>
      <td>[Metabolomics][W4M][NMR] NMR Annotation - Annotation of complex mixture NMR spectra and metabolite proportion estimation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://workflow4metabolomics.org</td>
      <td>Metabolomics</td>
      <td>nmr_annotation</td>
      <td>2019-07-30</td>
      <td>marie-tremblay-metatoul</td>
      <td>https://github.com/workflow4metabolomics/nmr_annotation</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/nmr_annotation</td>
      <td>3.0.0</td>
      <td>r-batch</td>
      <td>1.1_4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>49</td>
      <td>49</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>49</td>
      <td>49</td>
      <td>5</td>
      <td>5</td>
    </tr>
    <tr>
      <td></td>
      <td>nmr_annotation2d</td>
      <td>2DNmrAnnotation</td>
      <td>[Metabolomics][W4M][NMR] NMR Annotation2D - Automatic annotation of bi-dimensional NMR spectra</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://workflow4metabolomics.org</td>
      <td>Metabolomics</td>
      <td>2dnmrannotation</td>
      <td>2021-01-21</td>
      <td>marie-tremblay-metatoul</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/nmr_annotation2d</td>
      <td>2.0.0</td>
      <td>r-batch</td>
      <td>1.1_4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>nmr_preprocessing</td>
      <td>NMR_Preprocessing, NMR_Read</td>
      <td>[Metabolomics][W4M][NMR] NMR Preprocessing - Preprocessing of 1D NMR spectra from FID to baseline correction</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://workflow4metabolomics.org</td>
      <td>Metabolomics</td>
      <td>nmr_preprocessing</td>
      <td>2019-07-29</td>
      <td>marie-tremblay-metatoul</td>
      <td>https://github.com/workflow4metabolomics/nmr_preprocessing</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/nmr_preprocessing</td>
      <td></td>
      <td>r-batch</td>
      <td>1.1_4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>142</td>
      <td>142</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>335</td>
      <td>341</td>
      <td>9</td>
      <td>9</td>
      <td>477</td>
      <td>483</td>
      <td>17</td>
      <td>17</td>
    </tr>
    <tr>
      <td></td>
      <td>normalization</td>
      <td>normalization</td>
      <td>[Metabolomics][W4M][ALL] Normalization (operation applied on each individual spectrum) of preprocessed data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://workflow4metabolomics.org</td>
      <td>Metabolomics</td>
      <td>normalization</td>
      <td>2019-07-30</td>
      <td>marie-tremblay-metatoul</td>
      <td>https://github.com/workflow4metabolomics/normalization</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/normalization</td>
      <td>1.0.7</td>
      <td>r-batch</td>
      <td>1.1_4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>152</td>
      <td>162</td>
      <td>39</td>
      <td>40</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>233</td>
      <td>236</td>
      <td>12</td>
      <td>12</td>
      <td>385</td>
      <td>398</td>
      <td>51</td>
      <td>52</td>
    </tr>
    <tr>
      <td></td>
      <td>openlabcds2csv</td>
      <td>openlabcds2csv</td>
      <td>Creates a summary of several "Internal Standard Report" OpenLabCDS results.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Metabolomics</td>
      <td>openlabcds2csv</td>
      <td>2021-10-28</td>
      <td>genouest</td>
      <td></td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/openlabcds2csv</td>
      <td>0.1.0</td>
      <td>openjdk</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>orientationpy</td>
      <td>orientationpy</td>
      <td>Compute image orientation</td>
      <td></td>
      <td></td>
      <td>orientationj</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>orientationpy</td>
      <td>2024-03-12</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/orientationpy</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/orientationpy</td>
      <td>0.2.0.4</td>
      <td>orientationpy</td>
      <td>0.2.0.4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>overlay_images</td>
      <td>ip_overlay_images</td>
      <td>Overlay two images</td>
      <td>galaxy_image_analysis</td>
      <td>galaxy_image_analysis</td>
      <td></td>
      <td>Galaxy Image Analysis</td>
      <td>Galaxy Image Analysis is mainly developed by the Biomedical Computer Vision (BMCV) Group at Heidelberg University. Its dedication is to provide tools for image analysis and image processing for the Galaxy platform.</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>To update</td>
      <td>https://github.com/BMCV/galaxy-image-analysis</td>
      <td>Imaging</td>
      <td>overlay_images</td>
      <td>2022-02-26</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/overlay_images/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/overlay_images</td>
      <td>0.0.4</td>
      <td>scikit-image</td>
      <td></td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>149</td>
      <td>149</td>
      <td>31</td>
      <td>31</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>18</td>
      <td>18</td>
      <td>1</td>
      <td>1</td>
      <td>167</td>
      <td>167</td>
      <td>32</td>
      <td>32</td>
    </tr>
    <tr>
      <td></td>
      <td>pangolin</td>
      <td>pangolin</td>
      <td>Phylogenetic Assignment of Named Global Outbreak LINeages</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/hCoV-2019/pangolin</td>
      <td>Sequence Analysis</td>
      <td>pangolin</td>
      <td>2020-05-01</td>
      <td>nml</td>
      <td>https://github.com/hCoV-2019/pangolin</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/pangolin</td>
      <td>1.1.14</td>
      <td>pangolin</td>
      <td>4.3</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>7457</td>
      <td>7457</td>
      <td>508</td>
      <td>508</td>
      <td>8518</td>
      <td>8518</td>
      <td>377</td>
      <td>377</td>
      <td>2760</td>
      <td>2760</td>
      <td>259</td>
      <td>259</td>
      <td>18735</td>
      <td>18735</td>
      <td>1144</td>
      <td>1144</td>
    </tr>
    <tr>
      <td></td>
      <td>patrist</td>
      <td>patrist</td>
      <td>Extract Patristic Distance From a Tree</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://gist.github.com/ArtPoon/7330231e74201ded54b87142a1d6cd02</td>
      <td>Phylogenetics</td>
      <td>patrist</td>
      <td>2019-12-14</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/patrist</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/patrist</td>
      <td>0.1.2</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>peptimapper</td>
      <td>peptimapper_clustqualify, peptimapper_clust_to_gff, peptimapper_pep_match, peptimapper_pep_novo_tag</td>
      <td>Proteogenomics workflow for the expert annotation of eukaryotic genomes</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-019-5431-9</td>
      <td>Proteomics</td>
      <td></td>
      <td>2021-11-03</td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/peptimapper</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/peptimapper</td>
      <td>2.1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>permutate_axis</td>
      <td>ip_permutate_axis</td>
      <td>Permutates axes</td>
      <td>galaxy_image_analysis</td>
      <td>galaxy_image_analysis</td>
      <td></td>
      <td>Galaxy Image Analysis</td>
      <td>Galaxy Image Analysis is mainly developed by the Biomedical Computer Vision (BMCV) Group at Heidelberg University. Its dedication is to provide tools for image analysis and image processing for the Galaxy platform.</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>permutate_axis</td>
      <td>2017-10-30</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/permutate_axis/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/permutate_axis</td>
      <td>0.2-2</td>
      <td>scikit-image</td>
      <td></td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>36</td>
      <td>42</td>
      <td>1</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>36</td>
      <td>42</td>
      <td>1</td>
      <td>2</td>
    </tr>
    <tr>
      <td></td>
      <td>physiofit</td>
      <td>physiofit</td>
      <td>PhysioFit is a scientific tool designed to quantify cell growth parameters and uptake &amp; production fluxes</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>physiofit.readthedocs.io</td>
      <td>Metabolomics</td>
      <td>physiofit</td>
      <td>2022-09-28</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/MetaSys-LISBP/PhysioFit</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/physiofit</td>
      <td>3.4.0</td>
      <td>physiofit</td>
      <td>3.4.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>physiofit_manager</td>
      <td>physiofit_data_manager</td>
      <td>Handling of physiofit input files</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>physiofit.readthedocs.io</td>
      <td>Metabolomics</td>
      <td>physiofit_manager</td>
      <td>2022-11-03</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/MetaboHUB-MetaToul-FluxoMet/PhysioFit_Data_Manager</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/physiofit_manager</td>
      <td>1.0.1</td>
      <td>physiofit_data_manager</td>
      <td>1.0.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>plantseg</td>
      <td>plantseg</td>
      <td>Tool for cell instance aware segmentation in densely packed 3D volumetric images</td>
      <td>plantseg</td>
      <td>plantseg</td>
      <td></td>
      <td>PlantSeg</td>
      <td>Accurate and Versatile 3D Segmentation of Plant Tissues at Cellular Resolution.PlantSeg is a tool for cell instance aware segmentation in densely packed 3D volumetric images. The pipeline uses a two stages segmentation strategy (Neural Network + Segmentation). The pipeline is tuned for plant cell tissue acquired with confocal and light sheet microscopy. Pre-trained models are provided.</td>
      <td>Network analysis, Quantification, Microscope image visualisation</td>
      <td>Plant biology, Bioimaging, Light microscopy, Machine learning, Electron microscopy</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>plantseg</td>
      <td>2024-06-19</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/plantseg/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/plantseg</td>
      <td>1.8.1</td>
      <td>plant-seg</td>
      <td></td>
      <td>Network analysis, Quantification, Microscope image visualisation</td>
      <td>Plant biology, Bioimaging, Light microscopy, Machine learning, Electron microscopy</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>plasmid_profiler</td>
      <td>plasmid_profiler</td>
      <td>Explores plasmid content in WGS data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td>plasmid_profiler</td>
      <td>2016-11-07</td>
      <td>nml</td>
      <td></td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/plasmid_profiler</td>
      <td>0.1.6</td>
      <td>r</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>plasmid_profiler_suite</td>
      <td></td>
      <td>Plasmid Profiler suite defining all dependencies for Plasmid Profiler</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>suite_plasmid_profiler</td>
      <td>2016-11-07</td>
      <td>nml</td>
      <td></td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/plasmid_profiler_suite</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>plasmidspades</td>
      <td>plasmidspades</td>
      <td>Genome assembler for assemblying plasmid</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Assembly</td>
      <td>plasmidspades</td>
      <td>2016-06-06</td>
      <td>nml</td>
      <td></td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/plasmidspades</td>
      <td>1.1</td>
      <td>spades</td>
      <td>4.0.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>plotheatmap</td>
      <td>plotheatmap</td>
      <td>This tool can be used to plot heatmap of gene expression data. The genes are chosen based on p-value, FDR, log FC and log CPM from edgeR output.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Computational chemistry</td>
      <td>plotheatmap</td>
      <td>2017-07-04</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/plotheatmap</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/plotheatmap</td>
      <td>1.0</td>
      <td>bioconductor-preprocesscore</td>
      <td>1.64.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>pneumocat</td>
      <td>pneumocat</td>
      <td>Pneumococcal Capsular Typing of illumina fastq reads</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/phe-bioinformatics/PneumoCaT</td>
      <td>Variant Analysis</td>
      <td>pneumocat</td>
      <td>2020-03-24</td>
      <td>nml</td>
      <td>https://github.com/phe-bioinformatics/PneumoCaT</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/pneumocat</td>
      <td>1.2.1</td>
      <td>pneumocat</td>
      <td>1.2.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>points2labelimage</td>
      <td>ip_points_to_label</td>
      <td>Points to label image</td>
      <td>galaxy_image_analysis</td>
      <td>galaxy_image_analysis</td>
      <td></td>
      <td>Galaxy Image Analysis</td>
      <td>Galaxy Image Analysis is mainly developed by the Biomedical Computer Vision (BMCV) Group at Heidelberg University. Its dedication is to provide tools for image analysis and image processing for the Galaxy platform.</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>points2labelimage</td>
      <td>2018-11-20</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/points2labelimage/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/points2labelimage</td>
      <td>0.4</td>
      <td>scikit-image</td>
      <td></td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>33</td>
      <td>33</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>33</td>
      <td>33</td>
      <td>2</td>
      <td>2</td>
    </tr>
    <tr>
      <td></td>
      <td>points_association_nn</td>
      <td>ip_points_association_nn</td>
      <td>Association of points in consecutive frames</td>
      <td>galaxy_image_analysis</td>
      <td>galaxy_image_analysis</td>
      <td></td>
      <td>Galaxy Image Analysis</td>
      <td>Galaxy Image Analysis is mainly developed by the Biomedical Computer Vision (BMCV) Group at Heidelberg University. Its dedication is to provide tools for image analysis and image processing for the Galaxy platform.</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>To update</td>
      <td>https://github.com/BMCV/galaxy-image-analysis</td>
      <td>Imaging</td>
      <td>points_association_nn</td>
      <td>2021-07-22</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/points_association_nn/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/points_association_nn</td>
      <td>0.0.3-2</td>
      <td>numpy</td>
      <td></td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>29</td>
      <td>29</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>29</td>
      <td>29</td>
      <td>4</td>
      <td>4</td>
    </tr>
    <tr>
      <td></td>
      <td>projective_transformation</td>
      <td>ip_projective_transformation</td>
      <td>Projective transformation</td>
      <td>galaxy_image_analysis</td>
      <td>galaxy_image_analysis</td>
      <td></td>
      <td>Galaxy Image Analysis</td>
      <td>Galaxy Image Analysis is mainly developed by the Biomedical Computer Vision (BMCV) Group at Heidelberg University. Its dedication is to provide tools for image analysis and image processing for the Galaxy platform.</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>projective_transformation</td>
      <td>2018-12-07</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/projective_transformation/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/projective_transformation</td>
      <td>0.1.2</td>
      <td>scikit-image</td>
      <td></td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>40</td>
      <td>86</td>
      <td>4</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>40</td>
      <td>86</td>
      <td>4</td>
      <td>6</td>
    </tr>
    <tr>
      <td></td>
      <td>projective_transformation_points</td>
      <td>ip_projective_transformation_points</td>
      <td>Projective transformation of ROIs defined by pixel (point) coordinates</td>
      <td>galaxy_image_analysis</td>
      <td>galaxy_image_analysis</td>
      <td></td>
      <td>Galaxy Image Analysis</td>
      <td>Galaxy Image Analysis is mainly developed by the Biomedical Computer Vision (BMCV) Group at Heidelberg University. Its dedication is to provide tools for image analysis and image processing for the Galaxy platform.</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>projective_transformation_points</td>
      <td>2018-12-07</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/projective_transformation_points/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/projective_transformation_points</td>
      <td>0.1.1</td>
      <td>scikit-image</td>
      <td></td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>20858</td>
      <td>21444</td>
      <td>14</td>
      <td>18</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>20858</td>
      <td>21444</td>
      <td>14</td>
      <td>18</td>
    </tr>
    <tr>
      <td></td>
      <td>promer</td>
      <td>promer4_substitutions</td>
      <td>Aligns two sets of contigs and reports amino acid substitutions between them</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/promer</td>
      <td>Assembly</td>
      <td>promer</td>
      <td>2019-12-14</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/promer</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/promer</td>
      <td>1.2</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>pseudogenome</td>
      <td>pseudogenome</td>
      <td>Create a pseudogenome from a multiple fasta file either with a JCVI linker or custom length and characters.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>Sequence Analysis</td>
      <td>pseudogenome</td>
      <td>2018-04-06</td>
      <td>nml</td>
      <td></td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/pseudogenome</td>
      <td>1.0.0</td>
      <td>perl-bioperl</td>
      <td>1.7.8</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>quasitools</td>
      <td>aacoverage, aavariants, callcodonvar, callntvar, complexity_bam, complexity_fasta, consensus, distance, dnds, drmutations, hydra, quality</td>
      <td>A collection of tools for analysing Viral Quasispecies</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/phac-nml/quasitools</td>
      <td>Sequence Analysis</td>
      <td>quasitools</td>
      <td>2018-07-06</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/quasitools</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/quasitools</td>
      <td>0.7.0</td>
      <td>quasitools</td>
      <td>0.7.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>12</td>
      <td>0</td>
      <td>12</td>
      <td>0</td>
      <td>12</td>
      <td>0</td>
      <td>12</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>12</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>13483</td>
      <td>13483</td>
      <td>434</td>
      <td>434</td>
      <td>13483</td>
      <td>13483</td>
      <td>434</td>
      <td>434</td>
    </tr>
    <tr>
      <td></td>
      <td>rdock</td>
      <td>rdock</td>
      <td>Docking ligands to proteins and nucleic acids</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://rdock.sourceforge.net/</td>
      <td>Computational chemistry</td>
      <td>rdock</td>
      <td>2016-12-14</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/rdock/</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/rdock</td>
      <td>1.0</td>
      <td>rDock</td>
      <td>2013.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>refseq_masher</td>
      <td>refseq_masher_contains, refseq_masher_matches</td>
      <td>Find what genomes match or are contained within your sequence data using Mash_ and a Mash sketch database.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/phac-nml/refseq_masher</td>
      <td>Sequence Analysis</td>
      <td>refseq_masher</td>
      <td>2018-01-31</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/refseq_masher</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/refseq_masher</td>
      <td>0.1.2</td>
      <td>refseq_masher</td>
      <td>0.1.2</td>
      <td></td>
      <td></td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>428</td>
      <td>428</td>
      <td>35</td>
      <td>35</td>
      <td>141</td>
      <td>141</td>
      <td>42</td>
      <td>42</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>569</td>
      <td>569</td>
      <td>77</td>
      <td>77</td>
    </tr>
    <tr>
      <td></td>
      <td>repeat_channels</td>
      <td>repeat_channels</td>
      <td>Convert single-channel to multi-channel image</td>
      <td>galaxy_image_analysis</td>
      <td>galaxy_image_analysis</td>
      <td></td>
      <td>Galaxy Image Analysis</td>
      <td>Galaxy Image Analysis is mainly developed by the Biomedical Computer Vision (BMCV) Group at Heidelberg University. Its dedication is to provide tools for image analysis and image processing for the Galaxy platform.</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>repeat_channels</td>
      <td>2024-04-25</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/repeat_channels/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/repeat_channels</td>
      <td>1.26.4</td>
      <td>numpy</td>
      <td></td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>1</td>
      <td>1</td>
    </tr>
    <tr>
      <td></td>
      <td>repeatexplorer2</td>
      <td>repeatexplorer_clustering</td>
      <td>Tool for annotation of repeats from unassembled shotgun reads.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/repeatexplorer/repex_tarean</td>
      <td>Genome annotation</td>
      <td>repeatexplorer2</td>
      <td>2023-11-01</td>
      <td>gga</td>
      <td>https://github.com/galaxy-genome-annotation/galaxy-tools/tree/master/tools/repeatexplorer2</td>
      <td>https://github.com/galaxy-genome-annotation/galaxy-tools/tree/master/tools/repeatexplorer2</td>
      <td>2.3.8</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>95</td>
      <td>95</td>
      <td>38</td>
      <td>38</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>95</td>
      <td>95</td>
      <td>38</td>
      <td>38</td>
    </tr>
    <tr>
      <td></td>
      <td>replace_chromosome_names</td>
      <td>replace_chromosome_names</td>
      <td>Replace chromosome names</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>replace_chromosome_names</td>
      <td>2017-05-18</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/replace_chromosome_names/</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/replace_chromosome_names</td>
      <td>0.1</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>514</td>
      <td>573</td>
      <td>69</td>
      <td>83</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>514</td>
      <td>573</td>
      <td>69</td>
      <td>83</td>
    </tr>
    <tr>
      <td></td>
      <td>rfove</td>
      <td>rfove</td>
      <td>Perform segmentation region-based fitting of overlapping ellipses</td>
      <td>rfove</td>
      <td>rfove</td>
      <td></td>
      <td>RFOVE</td>
      <td>RFOVE (Region-based Fitting of Overlapping Ellipses and its Application to Cells Segmentation) is a MATLAB script for performing image segmentation on cells.</td>
      <td>Image analysis</td>
      <td>Cell biology, Biomedical science, Imaging</td>
      <td>To update</td>
      <td>https://sites.google.com/site/costaspanagiotakis/research/cs</td>
      <td>Imaging</td>
      <td>rfove</td>
      <td>2023-11-12</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/rfove/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/rfove</td>
      <td>2023.11.12</td>
      <td></td>
      <td></td>
      <td>Image analysis</td>
      <td>Cell biology, Biomedical science, Imaging</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>15</td>
      <td>15</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>15</td>
      <td>15</td>
      <td>7</td>
      <td>7</td>
    </tr>
    <tr>
      <td></td>
      <td>rsat_filter_snps</td>
      <td>rsat_filter_snps</td>
      <td>Filter SNPs in RSAT Matrix Scan output</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/TGAC/earlham-galaxytools/</td>
      <td>ChIP-seq, Systems Biology</td>
      <td>rsat_filter_snps</td>
      <td>2016-01-29</td>
      <td>earlham</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/rsat_filter_snps</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/rsat_filter_snps</td>
      <td>0.1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>scale_image</td>
      <td>ip_scale_image</td>
      <td>Scale image</td>
      <td>scikit-image</td>
      <td>scikit-image</td>
      <td>scikit-image</td>
      <td>scikit-image</td>
      <td>Scikit-image contains image processing algorithms for SciPy, including IO, morphology, filtering, warping, color manipulation, object detection, etc.</td>
      <td>Image analysis, Image annotation, Visualisation, Data handling</td>
      <td>Imaging, Software engineering, Literature and language</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>scale_image</td>
      <td>2018-12-07</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/scale_image/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/scale_image</td>
      <td>0.18.3</td>
      <td>scikit-image</td>
      <td></td>
      <td>Image analysis, Image annotation, Visualisation, Data handling</td>
      <td>Imaging, Software engineering, Literature and language</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>477</td>
      <td>520</td>
      <td>13</td>
      <td>16</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>477</td>
      <td>520</td>
      <td>13</td>
      <td>16</td>
    </tr>
    <tr>
      <td></td>
      <td>segalign</td>
      <td>segalign</td>
      <td>A Scalable GPU System for Pairwise Whole Genome Alignments based on LASTZ's seed-filter-extend paradigm.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/gsneha26/SegAlign</td>
      <td>Next Gen Mappers</td>
      <td>segalign</td>
      <td>2024-04-18</td>
      <td>richard-burhans</td>
      <td>https://github.com/richard-burhans/galaxytools/tree/main/tools/segalign</td>
      <td>https://github.com/richard-burhans/galaxytools/tree/main/tools/segalign</td>
      <td>0.1.2.7</td>
      <td>segalign-galaxy</td>
      <td>0.1.2.7</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>252</td>
      <td>252</td>
      <td>47</td>
      <td>47</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>252</td>
      <td>252</td>
      <td>47</td>
      <td>47</td>
    </tr>
    <tr>
      <td></td>
      <td>segmetrics</td>
      <td>ip_segmetrics</td>
      <td>Image segmentation and object detection performance measures</td>
      <td>segmetrics</td>
      <td>segmetrics</td>
      <td>segmetrics</td>
      <td>SegMetrics</td>
      <td>Image segmentation and object detection performance measures</td>
      <td>Image analysis</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>segmetrics</td>
      <td>2022-10-07</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/segmetrics/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/segmetrics</td>
      <td>1.4</td>
      <td>segmetrics</td>
      <td>1.5</td>
      <td>Image analysis</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>2</td>
      <td>2</td>
    </tr>
    <tr>
      <td></td>
      <td>seqtk_nml</td>
      <td>seqtk_nml_sample</td>
      <td>Tool to downsample fastq reads</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/lh3/seqtk</td>
      <td>Sequence Analysis</td>
      <td>seqtk_nml</td>
      <td>2017-09-19</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/seqtk_nml</td>
      <td>1.0.1</td>
      <td>seqtk</td>
      <td>1.4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>sistr_cmd</td>
      <td>sistr_cmd</td>
      <td>SISTR in silico serotyping tool</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/sistr_cmd</td>
      <td>Sequence Analysis</td>
      <td>sistr_cmd</td>
      <td>2017-02-20</td>
      <td>nml</td>
      <td></td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/sistr_cmd</td>
      <td>1.1.1</td>
      <td>sistr_cmd</td>
      <td>1.1.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>3457</td>
      <td>3457</td>
      <td>301</td>
      <td>301</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>556</td>
      <td>556</td>
      <td>65</td>
      <td>65</td>
      <td>4013</td>
      <td>4013</td>
      <td>366</td>
      <td>366</td>
    </tr>
    <tr>
      <td></td>
      <td>skyline2isocor</td>
      <td>skyline2isocor</td>
      <td>Converting skyline output to IsoCor input</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/MetaboHUB-MetaToul-FluxoMet/Skyline2IsoCor</td>
      <td>Metabolomics</td>
      <td>skyline2isocor</td>
      <td>2024-02-07</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/MetaboHUB-MetaToul-FluxoMet/tools_w4m</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/skyline2isocor</td>
      <td>1.0.0</td>
      <td>skyline2isocor</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>slice_image</td>
      <td>ip_slice_image</td>
      <td>Slice image</td>
      <td>galaxy_image_analysis</td>
      <td>galaxy_image_analysis</td>
      <td></td>
      <td>Galaxy Image Analysis</td>
      <td>Galaxy Image Analysis is mainly developed by the Biomedical Computer Vision (BMCV) Group at Heidelberg University. Its dedication is to provide tools for image analysis and image processing for the Galaxy platform.</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>slice_image</td>
      <td>2017-10-30</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/slice_image/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/slice_image</td>
      <td>0.3-4</td>
      <td>scikit-image</td>
      <td></td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>96</td>
      <td>103</td>
      <td>4</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>1</td>
      <td>1</td>
      <td>100</td>
      <td>107</td>
      <td>5</td>
      <td>6</td>
    </tr>
    <tr>
      <td></td>
      <td>smalt</td>
      <td>smalt</td>
      <td>SMALT aligns DNA sequencing reads with a reference genome.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://www.sanger.ac.uk/science/tools/smalt-0</td>
      <td>Sequence Analysis</td>
      <td>smalt</td>
      <td>2017-09-19</td>
      <td>nml</td>
      <td>https://sourceforge.net/projects/smalt/</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/smalt</td>
      <td>0.7.6</td>
      <td>smalt</td>
      <td>0.7.6</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>smart_domains</td>
      <td>smart_domains</td>
      <td>SMART domains</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://smart.embl.de/</td>
      <td>Sequence Analysis</td>
      <td>smart_domains</td>
      <td>2017-05-12</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/smart_domains</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/smart_domains</td>
      <td>0.1.0</td>
      <td>perl-bioperl</td>
      <td>1.7.8</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>smina</td>
      <td>smina</td>
      <td>smina is a fork of Autodock Vina (http://vina.scripps.edu/) that focuses on improving scoring and minimization</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://sourceforge.net/projects/smina/</td>
      <td>Computational chemistry</td>
      <td>smina</td>
      <td>2017-07-07</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/smina/</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/smina</td>
      <td>1.0</td>
      <td>smina</td>
      <td>2017.11.9</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>499</td>
      <td>499</td>
      <td>16</td>
      <td>16</td>
      <td>44</td>
      <td>44</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>543</td>
      <td>543</td>
      <td>26</td>
      <td>26</td>
    </tr>
    <tr>
      <td></td>
      <td>spades_header_fixer</td>
      <td>spades_header_fixer</td>
      <td>Fixes Spades Fasta ids</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>Fasta Manipulation</td>
      <td>spades_fasta_header_fixer</td>
      <td>2019-12-18</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/spades_header_fixer</td>
      <td>1.1.2+galaxy1</td>
      <td>sed</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>spatyper</td>
      <td>spatyper</td>
      <td>Determines SPA type based on repeats in a submitted staphylococcal protein A fasta file.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/HCGB-IGTP/spaTyper</td>
      <td>Sequence Analysis</td>
      <td>spatyper</td>
      <td>2023-03-16</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/spatyper</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/spatyper</td>
      <td>0.3.3</td>
      <td>spatyper</td>
      <td>0.3.3</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>split_labelmap</td>
      <td>ip_split_labelmap</td>
      <td>Split Labelmaps</td>
      <td>galaxy_image_analysis</td>
      <td>galaxy_image_analysis</td>
      <td></td>
      <td>Galaxy Image Analysis</td>
      <td>Galaxy Image Analysis is mainly developed by the Biomedical Computer Vision (BMCV) Group at Heidelberg University. Its dedication is to provide tools for image analysis and image processing for the Galaxy platform.</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>split_labelmap</td>
      <td>2018-12-11</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/split_labelmaps/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/split_labelmap</td>
      <td>0.2-3</td>
      <td>scikit-image</td>
      <td></td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>44</td>
      <td>51</td>
      <td>2</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>44</td>
      <td>51</td>
      <td>2</td>
      <td>3</td>
    </tr>
    <tr>
      <td></td>
      <td>spolpred</td>
      <td>spolpred</td>
      <td>A program for predicting the spoligotype from raw sequence reads</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>spolpred</td>
      <td>2015-12-14</td>
      <td>nml</td>
      <td></td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/spolpred</td>
      <td>1.0.1</td>
      <td>spolpred</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>spot_detection_2d</td>
      <td>ip_spot_detection_2d</td>
      <td>Spot detection in 2D image sequence</td>
      <td>galaxy_image_analysis</td>
      <td>galaxy_image_analysis</td>
      <td></td>
      <td>Galaxy Image Analysis</td>
      <td>Galaxy Image Analysis is mainly developed by the Biomedical Computer Vision (BMCV) Group at Heidelberg University. Its dedication is to provide tools for image analysis and image processing for the Galaxy platform.</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>To update</td>
      <td>https://github.com/BMCV/galaxy-image-analysis</td>
      <td>Imaging</td>
      <td>spot_detection_2d</td>
      <td>2021-07-20</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/spot_detection_2d/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/spot_detection_2d</td>
      <td>0.1</td>
      <td>giatools</td>
      <td>0.3.1</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>16</td>
      <td>16</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>16</td>
      <td>16</td>
      <td>4</td>
      <td>4</td>
    </tr>
    <tr>
      <td></td>
      <td>srst2</td>
      <td>srst2</td>
      <td>Short Read Sequence Typing for Bacterial Pathogens</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>srst2</td>
      <td>2015-12-02</td>
      <td>nml</td>
      <td></td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/srst2</td>
      <td>0.3.7</td>
      <td>srst2</td>
      <td>0.2.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>297</td>
      <td>297</td>
      <td>40</td>
      <td>40</td>
      <td>71</td>
      <td>71</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>368</td>
      <td>368</td>
      <td>41</td>
      <td>41</td>
    </tr>
    <tr>
      <td></td>
      <td>staramr</td>
      <td>staramr_search</td>
      <td>Scan genome contigs against the ResFinder, PlasmidFinder, and PointFinder antimicrobial resistance databases.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/phac-nml/staramr</td>
      <td>Sequence Analysis</td>
      <td>staramr</td>
      <td>2022-06-10</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/staramr</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/staramr</td>
      <td>0.10.0</td>
      <td>staramr</td>
      <td>0.10.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>20778</td>
      <td>20891</td>
      <td>2030</td>
      <td>2045</td>
      <td>9520</td>
      <td>9520</td>
      <td>1424</td>
      <td>1424</td>
      <td>6753</td>
      <td>6753</td>
      <td>864</td>
      <td>864</td>
      <td>37051</td>
      <td>37164</td>
      <td>4318</td>
      <td>4333</td>
    </tr>
    <tr>
      <td></td>
      <td>stringmlst</td>
      <td>stringmlst</td>
      <td>Rapid and accurate identification of the sequence type (ST)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>stringmlst</td>
      <td>2016-10-19</td>
      <td>nml</td>
      <td></td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/stringmlst</td>
      <td>1.1.0</td>
      <td>stringMLST</td>
      <td>0.6.3</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__alignment</td>
      <td>qiime2__alignment__mafft, qiime2__alignment__mafft_add, qiime2__alignment__mask</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/qiime2/q2-alignment</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__alignment</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__alignment</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>249</td>
      <td>249</td>
      <td>32</td>
      <td>32</td>
      <td>16</td>
      <td>16</td>
      <td>10</td>
      <td>10</td>
      <td>102</td>
      <td>102</td>
      <td>16</td>
      <td>16</td>
      <td>367</td>
      <td>367</td>
      <td>58</td>
      <td>58</td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__composition</td>
      <td>qiime2__composition__add_pseudocount, qiime2__composition__ancom, qiime2__composition__ancombc, qiime2__composition__da_barplot, qiime2__composition__tabulate</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/qiime2/q2-composition</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__composition</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__composition</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>2</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>117</td>
      <td>117</td>
      <td>41</td>
      <td>41</td>
      <td>84</td>
      <td>84</td>
      <td>24</td>
      <td>24</td>
      <td>161</td>
      <td>161</td>
      <td>13</td>
      <td>13</td>
      <td>362</td>
      <td>362</td>
      <td>78</td>
      <td>78</td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__cutadapt</td>
      <td>qiime2__cutadapt__demux_paired, qiime2__cutadapt__demux_single, qiime2__cutadapt__trim_paired, qiime2__cutadapt__trim_single</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/qiime2/q2-cutadapt</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__cutadapt</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__cutadapt</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>217</td>
      <td>217</td>
      <td>47</td>
      <td>47</td>
      <td>42</td>
      <td>42</td>
      <td>20</td>
      <td>20</td>
      <td>96</td>
      <td>96</td>
      <td>12</td>
      <td>12</td>
      <td>355</td>
      <td>355</td>
      <td>79</td>
      <td>79</td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__dada2</td>
      <td>qiime2__dada2__denoise_ccs, qiime2__dada2__denoise_paired, qiime2__dada2__denoise_pyro, qiime2__dada2__denoise_single</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://benjjneb.github.io/dada2/</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__dada2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__dada2</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>721</td>
      <td>721</td>
      <td>160</td>
      <td>160</td>
      <td>350</td>
      <td>350</td>
      <td>91</td>
      <td>91</td>
      <td>214</td>
      <td>214</td>
      <td>35</td>
      <td>35</td>
      <td>1285</td>
      <td>1285</td>
      <td>286</td>
      <td>286</td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__deblur</td>
      <td>qiime2__deblur__denoise_16S, qiime2__deblur__denoise_other, qiime2__deblur__visualize_stats</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/biocore/deblur</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__deblur</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__deblur</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>125</td>
      <td>125</td>
      <td>30</td>
      <td>30</td>
      <td>13</td>
      <td>13</td>
      <td>9</td>
      <td>9</td>
      <td>83</td>
      <td>83</td>
      <td>8</td>
      <td>8</td>
      <td>221</td>
      <td>221</td>
      <td>47</td>
      <td>47</td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__demux</td>
      <td>qiime2__demux__emp_paired, qiime2__demux__emp_single, qiime2__demux__filter_samples, qiime2__demux__partition_samples_paired, qiime2__demux__partition_samples_single, qiime2__demux__subsample_paired, qiime2__demux__subsample_single, qiime2__demux__summarize, qiime2__demux__tabulate_read_counts</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/qiime2/q2-demux</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__demux</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__demux</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>396</td>
      <td>396</td>
      <td>134</td>
      <td>134</td>
      <td>121</td>
      <td>121</td>
      <td>57</td>
      <td>57</td>
      <td>144</td>
      <td>144</td>
      <td>39</td>
      <td>39</td>
      <td>661</td>
      <td>661</td>
      <td>230</td>
      <td>230</td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__diversity</td>
      <td>qiime2__diversity__adonis, qiime2__diversity__alpha, qiime2__diversity__alpha_correlation, qiime2__diversity__alpha_group_significance, qiime2__diversity__alpha_phylogenetic, qiime2__diversity__alpha_rarefaction, qiime2__diversity__beta, qiime2__diversity__beta_correlation, qiime2__diversity__beta_group_significance, qiime2__diversity__beta_phylogenetic, qiime2__diversity__beta_rarefaction, qiime2__diversity__bioenv, qiime2__diversity__core_metrics, qiime2__diversity__core_metrics_phylogenetic, qiime2__diversity__filter_alpha_diversity, qiime2__diversity__filter_distance_matrix, qiime2__diversity__mantel, qiime2__diversity__partial_procrustes, qiime2__diversity__pcoa, qiime2__diversity__pcoa_biplot, qiime2__diversity__procrustes_analysis, qiime2__diversity__tsne, qiime2__diversity__umap</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/qiime2/q2-diversity</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__diversity</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__diversity</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>21</td>
      <td>21</td>
      <td>22</td>
      <td>21</td>
      <td>21</td>
      <td>21</td>
      <td>22</td>
      <td>21</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>21</td>
      <td>0</td>
      <td>0</td>
      <td>964</td>
      <td>964</td>
      <td>318</td>
      <td>318</td>
      <td>249</td>
      <td>249</td>
      <td>118</td>
      <td>118</td>
      <td>475</td>
      <td>475</td>
      <td>108</td>
      <td>108</td>
      <td>1688</td>
      <td>1688</td>
      <td>544</td>
      <td>544</td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__diversity_lib</td>
      <td>qiime2__diversity_lib__alpha_passthrough, qiime2__diversity_lib__beta_passthrough, qiime2__diversity_lib__beta_phylogenetic_meta_passthrough, qiime2__diversity_lib__beta_phylogenetic_passthrough, qiime2__diversity_lib__bray_curtis, qiime2__diversity_lib__faith_pd, qiime2__diversity_lib__jaccard, qiime2__diversity_lib__observed_features, qiime2__diversity_lib__pielou_evenness, qiime2__diversity_lib__shannon_entropy, qiime2__diversity_lib__unweighted_unifrac, qiime2__diversity_lib__weighted_unifrac</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/qiime2/q2-diversity-lib</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__diversity_lib</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__diversity_lib</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>12</td>
      <td>12</td>
      <td>12</td>
      <td>12</td>
      <td>12</td>
      <td>12</td>
      <td>12</td>
      <td>12</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>12</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>13</td>
      <td>13</td>
      <td>11</td>
      <td>11</td>
      <td>1302</td>
      <td>1302</td>
      <td>19</td>
      <td>19</td>
      <td>1323</td>
      <td>1323</td>
      <td>38</td>
      <td>38</td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__emperor</td>
      <td>qiime2__emperor__biplot, qiime2__emperor__plot, qiime2__emperor__procrustes_plot</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://emperor.microbio.me</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__emperor</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__emperor</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>58</td>
      <td>58</td>
      <td>23</td>
      <td>23</td>
      <td>24</td>
      <td>24</td>
      <td>11</td>
      <td>11</td>
      <td>92</td>
      <td>92</td>
      <td>11</td>
      <td>11</td>
      <td>174</td>
      <td>174</td>
      <td>45</td>
      <td>45</td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__feature_classifier</td>
      <td>qiime2__feature_classifier__blast, qiime2__feature_classifier__classify_consensus_blast, qiime2__feature_classifier__classify_consensus_vsearch, qiime2__feature_classifier__classify_hybrid_vsearch_sklearn, qiime2__feature_classifier__classify_sklearn, qiime2__feature_classifier__extract_reads, qiime2__feature_classifier__find_consensus_annotation, qiime2__feature_classifier__fit_classifier_naive_bayes, qiime2__feature_classifier__fit_classifier_sklearn, qiime2__feature_classifier__makeblastdb, qiime2__feature_classifier__vsearch_global</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/qiime2/q2-feature-classifier</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__feature_classifier</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__feature_classifier</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>10</td>
      <td>10</td>
      <td>10</td>
      <td>10</td>
      <td>10</td>
      <td>10</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>863</td>
      <td>863</td>
      <td>220</td>
      <td>220</td>
      <td>237</td>
      <td>237</td>
      <td>98</td>
      <td>98</td>
      <td>181</td>
      <td>181</td>
      <td>41</td>
      <td>41</td>
      <td>1281</td>
      <td>1281</td>
      <td>359</td>
      <td>359</td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__feature_table</td>
      <td>qiime2__feature_table__core_features, qiime2__feature_table__filter_features, qiime2__feature_table__filter_features_conditionally, qiime2__feature_table__filter_samples, qiime2__feature_table__filter_seqs, qiime2__feature_table__group, qiime2__feature_table__heatmap, qiime2__feature_table__merge, qiime2__feature_table__merge_seqs, qiime2__feature_table__merge_taxa, qiime2__feature_table__presence_absence, qiime2__feature_table__rarefy, qiime2__feature_table__relative_frequency, qiime2__feature_table__rename_ids, qiime2__feature_table__split, qiime2__feature_table__subsample_ids, qiime2__feature_table__summarize, qiime2__feature_table__summarize_plus, qiime2__feature_table__tabulate_feature_frequencies, qiime2__feature_table__tabulate_sample_frequencies, qiime2__feature_table__tabulate_seqs, qiime2__feature_table__transpose</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/qiime2/q2-feature-table</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__feature_table</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__feature_table</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>17</td>
      <td>17</td>
      <td>17</td>
      <td>17</td>
      <td>17</td>
      <td>17</td>
      <td>17</td>
      <td>17</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>17</td>
      <td>0</td>
      <td>0</td>
      <td>983</td>
      <td>983</td>
      <td>287</td>
      <td>287</td>
      <td>375</td>
      <td>375</td>
      <td>144</td>
      <td>144</td>
      <td>907</td>
      <td>907</td>
      <td>106</td>
      <td>106</td>
      <td>2265</td>
      <td>2265</td>
      <td>537</td>
      <td>537</td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__fragment_insertion</td>
      <td>qiime2__fragment_insertion__classify_otus_experimental, qiime2__fragment_insertion__filter_features, qiime2__fragment_insertion__sepp</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/qiime2/q2-fragment-insertion</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__fragment_insertion</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__fragment_insertion</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>46</td>
      <td>46</td>
      <td>13</td>
      <td>13</td>
      <td>9</td>
      <td>9</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>55</td>
      <td>55</td>
      <td>18</td>
      <td>18</td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__longitudinal</td>
      <td>qiime2__longitudinal__anova, qiime2__longitudinal__feature_volatility, qiime2__longitudinal__first_differences, qiime2__longitudinal__first_distances, qiime2__longitudinal__linear_mixed_effects, qiime2__longitudinal__maturity_index, qiime2__longitudinal__nmit, qiime2__longitudinal__pairwise_differences, qiime2__longitudinal__pairwise_distances, qiime2__longitudinal__plot_feature_volatility, qiime2__longitudinal__volatility</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/qiime2/q2-longitudinal</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__longitudinal</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__longitudinal</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>11</td>
      <td>11</td>
      <td>11</td>
      <td>11</td>
      <td>11</td>
      <td>11</td>
      <td>11</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>37</td>
      <td>37</td>
      <td>16</td>
      <td>16</td>
      <td>22</td>
      <td>22</td>
      <td>7</td>
      <td>7</td>
      <td>89</td>
      <td>89</td>
      <td>14</td>
      <td>14</td>
      <td>148</td>
      <td>148</td>
      <td>37</td>
      <td>37</td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__metadata</td>
      <td>qiime2__metadata__distance_matrix, qiime2__metadata__merge, qiime2__metadata__shuffle_groups, qiime2__metadata__tabulate</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/qiime2/q2-metadata</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__metadata</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__metadata</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>615</td>
      <td>615</td>
      <td>143</td>
      <td>143</td>
      <td>246</td>
      <td>246</td>
      <td>75</td>
      <td>75</td>
      <td>115</td>
      <td>115</td>
      <td>27</td>
      <td>27</td>
      <td>976</td>
      <td>976</td>
      <td>245</td>
      <td>245</td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__phylogeny</td>
      <td>qiime2__phylogeny__align_to_tree_mafft_fasttree, qiime2__phylogeny__align_to_tree_mafft_iqtree, qiime2__phylogeny__align_to_tree_mafft_raxml, qiime2__phylogeny__fasttree, qiime2__phylogeny__filter_table, qiime2__phylogeny__filter_tree, qiime2__phylogeny__iqtree, qiime2__phylogeny__iqtree_ultrafast_bootstrap, qiime2__phylogeny__midpoint_root, qiime2__phylogeny__raxml, qiime2__phylogeny__raxml_rapid_bootstrap, qiime2__phylogeny__robinson_foulds</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/qiime2/q2-phylogeny</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__phylogeny</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__phylogeny</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>12</td>
      <td>12</td>
      <td>12</td>
      <td>12</td>
      <td>12</td>
      <td>12</td>
      <td>12</td>
      <td>12</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>12</td>
      <td>0</td>
      <td>0</td>
      <td>330</td>
      <td>330</td>
      <td>83</td>
      <td>83</td>
      <td>64</td>
      <td>64</td>
      <td>40</td>
      <td>40</td>
      <td>195</td>
      <td>195</td>
      <td>49</td>
      <td>49</td>
      <td>589</td>
      <td>589</td>
      <td>172</td>
      <td>172</td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__quality_control</td>
      <td>qiime2__quality_control__bowtie2_build, qiime2__quality_control__decontam_identify, qiime2__quality_control__decontam_identify_batches, qiime2__quality_control__decontam_remove, qiime2__quality_control__decontam_score_viz, qiime2__quality_control__evaluate_composition, qiime2__quality_control__evaluate_seqs, qiime2__quality_control__evaluate_taxonomy, qiime2__quality_control__exclude_seqs, qiime2__quality_control__filter_reads</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/qiime2/q2-quality-control</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__quality_control</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__quality_control</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>10</td>
      <td>10</td>
      <td>10</td>
      <td>10</td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__quality_filter</td>
      <td>qiime2__quality_filter__q_score</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/qiime2/q2-quality-filter</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__quality_filter</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__quality_filter</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>29</td>
      <td>29</td>
      <td>14</td>
      <td>14</td>
      <td>16</td>
      <td>16</td>
      <td>6</td>
      <td>6</td>
      <td>37</td>
      <td>37</td>
      <td>2</td>
      <td>2</td>
      <td>82</td>
      <td>82</td>
      <td>22</td>
      <td>22</td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__rescript</td>
      <td>qiime2__rescript__cull_seqs, qiime2__rescript__degap_seqs, qiime2__rescript__dereplicate, qiime2__rescript__edit_taxonomy, qiime2__rescript__evaluate_classifications, qiime2__rescript__evaluate_cross_validate, qiime2__rescript__evaluate_fit_classifier, qiime2__rescript__evaluate_seqs, qiime2__rescript__evaluate_taxonomy, qiime2__rescript__extract_seq_segments, qiime2__rescript__filter_seqs_length, qiime2__rescript__filter_seqs_length_by_taxon, qiime2__rescript__filter_taxa, qiime2__rescript__get_gtdb_data, qiime2__rescript__get_ncbi_data, qiime2__rescript__get_ncbi_data_protein, qiime2__rescript__get_ncbi_genomes, qiime2__rescript__get_silva_data, qiime2__rescript__get_unite_data, qiime2__rescript__merge_taxa, qiime2__rescript__orient_seqs, qiime2__rescript__parse_silva_taxonomy, qiime2__rescript__reverse_transcribe, qiime2__rescript__subsample_fasta, qiime2__rescript__trim_alignment</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/nbokulich/RESCRIPt</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2024-04-25</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__rescript</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__rescript</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__sample_classifier</td>
      <td>qiime2__sample_classifier__classify_samples, qiime2__sample_classifier__classify_samples_from_dist, qiime2__sample_classifier__classify_samples_ncv, qiime2__sample_classifier__confusion_matrix, qiime2__sample_classifier__fit_classifier, qiime2__sample_classifier__fit_regressor, qiime2__sample_classifier__heatmap, qiime2__sample_classifier__metatable, qiime2__sample_classifier__predict_classification, qiime2__sample_classifier__predict_regression, qiime2__sample_classifier__regress_samples, qiime2__sample_classifier__regress_samples_ncv, qiime2__sample_classifier__scatterplot, qiime2__sample_classifier__split_table, qiime2__sample_classifier__summarize</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/qiime2/q2-sample-classifier</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__sample_classifier</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__sample_classifier</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>15</td>
      <td>15</td>
      <td>15</td>
      <td>15</td>
      <td>15</td>
      <td>15</td>
      <td>15</td>
      <td>15</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>15</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>10</td>
      <td>3</td>
      <td>3</td>
      <td>14</td>
      <td>14</td>
      <td>10</td>
      <td>10</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>29</td>
      <td>29</td>
      <td>18</td>
      <td>18</td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__taxa</td>
      <td>qiime2__taxa__barplot, qiime2__taxa__collapse, qiime2__taxa__filter_seqs, qiime2__taxa__filter_table</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/qiime2/q2-taxa</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__taxa</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__taxa</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>664</td>
      <td>664</td>
      <td>145</td>
      <td>145</td>
      <td>113</td>
      <td>113</td>
      <td>43</td>
      <td>43</td>
      <td>255</td>
      <td>255</td>
      <td>50</td>
      <td>50</td>
      <td>1032</td>
      <td>1032</td>
      <td>238</td>
      <td>238</td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2__vsearch</td>
      <td>qiime2__vsearch__cluster_features_closed_reference, qiime2__vsearch__cluster_features_de_novo, qiime2__vsearch__cluster_features_open_reference, qiime2__vsearch__dereplicate_sequences, qiime2__vsearch__fastq_stats, qiime2__vsearch__merge_pairs, qiime2__vsearch__uchime_denovo, qiime2__vsearch__uchime_ref</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/qiime2/q2-vsearch</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__vsearch</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2__vsearch</td>
      <td>2024.5.0+q2galaxy.2024.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>7</td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>84</td>
      <td>84</td>
      <td>52</td>
      <td>52</td>
      <td>25</td>
      <td>25</td>
      <td>16</td>
      <td>16</td>
      <td>9</td>
      <td>9</td>
      <td>5</td>
      <td>5</td>
      <td>118</td>
      <td>118</td>
      <td>73</td>
      <td>73</td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2_core</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics, Metagenomics, Sequence Analysis</td>
      <td></td>
      <td>2022-08-29</td>
      <td>q2d2</td>
      <td></td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tool_collections/suite_qiime2_core</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>suite_qiime2_core__tools</td>
      <td>qiime2_core__tools__export, qiime2_core__tools__import, qiime2_core__tools__import_fastq</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://qiime2.org</td>
      <td>Metagenomics, Sequence Analysis, Statistics</td>
      <td></td>
      <td>2022-08-26</td>
      <td>q2d2</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2_core__tools</td>
      <td>https://github.com/qiime2/galaxy-tools/tree/main/tools/suite_qiime2_core__tools</td>
      <td>2024.5.0+dist.he540b0b0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>14941</td>
      <td>14941</td>
      <td>383</td>
      <td>383</td>
      <td>25941</td>
      <td>25941</td>
      <td>255</td>
      <td>255</td>
      <td>2856</td>
      <td>2856</td>
      <td>112</td>
      <td>112</td>
      <td>43738</td>
      <td>43738</td>
      <td>750</td>
      <td>750</td>
    </tr>
    <tr>
      <td></td>
      <td>superdsm</td>
      <td>ip_superdsm</td>
      <td>Globally optimal segmentation method based on superadditivity and deformable shape models for cell nuclei in fluorescence microscopy images</td>
      <td>superdsm</td>
      <td>superdsm</td>
      <td>superdsm</td>
      <td>SuperDSM</td>
      <td>SuperDSM is a globally optimal segmentation method based on superadditivity and deformable shape models for cell nuclei in fluorescence microscopy images and beyond.</td>
      <td>Image analysis</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>superdsm</td>
      <td>2023-07-05</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/superdsm/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/superdsm</td>
      <td>0.2.0</td>
      <td>superdsm</td>
      <td>0.4.0</td>
      <td>Image analysis</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>26</td>
      <td>26</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>26</td>
      <td>26</td>
      <td>5</td>
      <td>5</td>
    </tr>
    <tr>
      <td></td>
      <td>t_coffee</td>
      <td>t_coffee</td>
      <td>T-Coffee</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.tcoffee.org/</td>
      <td>Sequence Analysis</td>
      <td>t_coffee</td>
      <td>2016-12-13</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/t_coffee</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/t_coffee</td>
      <td>13.45.0.4846264</td>
      <td>t-coffee</td>
      <td>13.46.0.919e8c6b</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>3783</td>
      <td>9125</td>
      <td>211</td>
      <td>229</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3784</td>
      <td>9126</td>
      <td>212</td>
      <td>230</td>
    </tr>
    <tr>
      <td></td>
      <td>tree_relabeler</td>
      <td>tree_relabel</td>
      <td>Relabels the tips of a newick formatted tree.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>tree_relabeler</td>
      <td>2019-04-15</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools/blob/master/tools/tree_relabeler</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/tree_relabeler</td>
      <td>1.1.0</td>
      <td>perl-bioperl</td>
      <td>1.7.8</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>tripal</td>
      <td>analysis_add_analysis, analysis_get_analyses, analysis_load_blast, analysis_load_fasta, analysis_load_gff3, analysis_load_go, analysis_load_interpro, analysis_sync, db_index, db_populate_mviews, entity_publish, expression_add_biomaterial, expression_add_expression, expression_delete_biomaterials, expression_get_biomaterials, expression_sync_biomaterials, feature_delete_orphans, feature_sync, organism_add_organism, organism_get_organisms, organism_sync, phylogeny_sync</td>
      <td>Galaxy tools allowing to load data into a remote Tripal server.Tripal is a toolkit for construction of online biological (genetics, genomics, breeding, etc), community database,and is a member of the GMOD family of tools. Tripal provides by default integration with the GMOD Chado database schema and Drupal, a popular Content Management Systems (CMS).https://github.com/galaxy-genome-annotation/python-tripal</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxy-genome-annotation/python-tripal</td>
      <td>Web Services</td>
      <td></td>
      <td>2018-06-20</td>
      <td>gga</td>
      <td>https://github.com/galaxy-genome-annotation/galaxy-tools/tree/master/tools/tripal</td>
      <td>https://github.com/galaxy-genome-annotation/galaxy-tools/tree/master/tools/tripal</td>
      <td></td>
      <td>python-tripal</td>
      <td>3.2.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>unzip</td>
      <td>unzip</td>
      <td>Unzip file</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Convert Formats</td>
      <td>unzip</td>
      <td>2019-08-01</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/unzip/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/unzip</td>
      <td>6.0</td>
      <td>unzip</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>6942</td>
      <td>7024</td>
      <td>1345</td>
      <td>1356</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>2461</td>
      <td>2461</td>
      <td>266</td>
      <td>266</td>
      <td>9404</td>
      <td>9486</td>
      <td>1612</td>
      <td>1623</td>
    </tr>
    <tr>
      <td></td>
      <td>visceral_evaluatesegmentation</td>
      <td>ip_visceral_evaluatesegmentation</td>
      <td>Visceral Project - Evaluate Segmentation Tool</td>
      <td></td>
      <td></td>
      <td>evaluatesegmentation-tool</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>visceral_evaluatesegmentation</td>
      <td>2024-09-30</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/visceral-evaluatesegmentation</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/visceral_evaluatesegmentation</td>
      <td>0.5-2</td>
      <td>visceral-evaluatesegmentation</td>
      <td>2015.07.03</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>20</td>
      <td>25</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>20</td>
      <td>25</td>
      <td>1</td>
      <td>1</td>
    </tr>
    <tr>
      <td></td>
      <td>voronoi_tessellation</td>
      <td>voronoi_tessellation</td>
      <td>Compute Voronoi tesselation</td>
      <td>scikit-image</td>
      <td>scikit-image</td>
      <td>scikit-image</td>
      <td>scikit-image</td>
      <td>Scikit-image contains image processing algorithms for SciPy, including IO, morphology, filtering, warping, color manipulation, object detection, etc.</td>
      <td>Image analysis, Image annotation, Visualisation, Data handling</td>
      <td>Imaging, Software engineering, Literature and language</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>voronoi_tesselation</td>
      <td>2024-03-09</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/voronoi_tesselation</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/voronoi_tessellation</td>
      <td>0.22.0</td>
      <td>scikit-image</td>
      <td></td>
      <td>Image analysis, Image annotation, Visualisation, Data handling</td>
      <td>Imaging, Software engineering, Literature and language</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>43</td>
      <td>43</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>43</td>
      <td>43</td>
      <td>3</td>
      <td>3</td>
    </tr>
    <tr>
      <td></td>
      <td>w4mclassfilter</td>
      <td>w4mclassfilter</td>
      <td>Filter W4M data by values or metadata</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/HegemanLab/w4mclassfilter_galaxy_wrapper</td>
      <td>Metabolomics</td>
      <td>w4mclassfilter</td>
      <td>2019-09-26</td>
      <td>eschen42</td>
      <td>https://github.com/HegemanLab/w4mclassfilter_galaxy_wrapper/tree/master</td>
      <td>https://github.com/HegemanLab/w4mclassfilter_galaxy_wrapper/tree/master/tools/w4mclassfilter</td>
      <td>0.98.19</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2370</td>
      <td>2836</td>
      <td>9</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1482</td>
      <td>1509</td>
      <td>4</td>
      <td>4</td>
      <td>3852</td>
      <td>4345</td>
      <td>13</td>
      <td>14</td>
    </tr>
    <tr>
      <td></td>
      <td>w4mconcatenate</td>
      <td>W4Mconcatenate</td>
      <td>[W4M][Utils] concatenate two Metadata tables</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://workflow4metabolomics.org</td>
      <td>Metabolomics</td>
      <td>w4mconcatenate</td>
      <td>2024-07-10</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/w4mconcatenate</td>
      <td>1.0.0+galaxy0</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>w4mcorcov</td>
      <td>w4mcorcov</td>
      <td>OPLS-DA Contrasts of Univariate Results</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/HegemanLab/w4mcorcov_galaxy_wrapper</td>
      <td>Metabolomics</td>
      <td>w4mcorcov</td>
      <td>2018-09-05</td>
      <td>eschen42</td>
      <td>https://github.com/HegemanLab/w4mcorcov_galaxy_wrapper/tree/master</td>
      <td>https://github.com/HegemanLab/w4mcorcov_galaxy_wrapper/tree/master/tools/w4mcorcov</td>
      <td>0.98.18</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>238</td>
      <td>375</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>10</td>
      <td>2</td>
      <td>2</td>
      <td>248</td>
      <td>385</td>
      <td>12</td>
      <td>12</td>
    </tr>
    <tr>
      <td></td>
      <td>w4mjoinpn</td>
      <td>w4mjoinpn</td>
      <td>Join positive- and negative-mode W4M datasets</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/HegemanLab/w4mjoinpn_galaxy_wrapper</td>
      <td>Metabolomics</td>
      <td>w4mjoinpn</td>
      <td>2018-08-06</td>
      <td>eschen42</td>
      <td>https://github.com/HegemanLab/w4mjoinpn_galaxy_wrapper/tree/master</td>
      <td>https://github.com/HegemanLab/w4mjoinpn_galaxy_wrapper/tree/master/tools/w4mjoinpn</td>
      <td>0.98.2</td>
      <td>coreutils</td>
      <td>8.25</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>150</td>
      <td>171</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>2</td>
      <td>2</td>
      <td>157</td>
      <td>178</td>
      <td>9</td>
      <td>9</td>
    </tr>
    <tr>
      <td></td>
      <td>wade</td>
      <td>wade</td>
      <td>identify regions of interest</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/wade</td>
      <td>Sequence Analysis</td>
      <td>wade</td>
      <td>2019-09-20</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/wade</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/wade</td>
      <td>0.2.5+galaxy1</td>
      <td>wade</td>
      <td>0.2.6</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>wsi_extract_top_view</td>
      <td>ip_wsi_extract_top_view</td>
      <td>WSI Extract Top View</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>wsi_extract_top_view</td>
      <td>2017-10-30</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/wsi_extract_top_view/</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/wsi_extract_top_view</td>
      <td>0.2-2</td>
      <td>scikit-image</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>32</td>
      <td>38</td>
      <td>2</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>32</td>
      <td>38</td>
      <td>2</td>
      <td>3</td>
    </tr>
    <tr>
      <td></td>
      <td>xcms</td>
      <td>abims_xcms_fillPeaks, abims_xcms_group, abims_xcms_refine, abims_xcms_retcor, abims_xcms_summary, abims_xcms_xcmsSet, msnbase_readmsdata, xcms_export_samplemetadata, xcms_merge, xcms_plot_chromatogram, xcms_plot_eic, xcms_plot_raw</td>
      <td></td>
      <td>XCMS</td>
      <td>XCMS</td>
      <td></td>
      <td>xcms</td>
      <td>Framework for processing and visualization of chromatographically separated and single-spectra mass spectral data.  The packages enables imports from AIA/ANDI NetCDF, mzXML, mzData and mzML files and preprocesses data for high-throughput, untargeted analyte profiling.</td>
      <td>Mass spectrum visualisation, Label-free quantification, Validation, Filtering, Chromatographic alignment, Peak detection, Chromatogram visualisation</td>
      <td>Biological imaging, Data visualisation, Metabolomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/sneumann/xcms</td>
      <td>Metabolomics</td>
      <td>xcms</td>
      <td>2023-05-24</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/tree/master/tools/xcms</td>
      <td>4.0.0</td>
      <td>bioconductor-xcms</td>
      <td>4.0.0</td>
      <td>Mass spectrum visualisation, Label-free quantification, Validation, Filtering, Chromatographic alignment, Peak detection, Chromatogram visualisation</td>
      <td>Data visualisation, Metabolomics</td>
      <td>9</td>
      <td>11</td>
      <td>12</td>
      <td>12</td>
      <td>9</td>
      <td>11</td>
      <td>12</td>
      <td>12</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>10</td>
      <td>9</td>
      <td>0</td>
      <td>31909</td>
      <td>35790</td>
      <td>1357</td>
      <td>1384</td>
      <td>7099</td>
      <td>7099</td>
      <td>413</td>
      <td>413</td>
      <td>5333</td>
      <td>5357</td>
      <td>330</td>
      <td>330</td>
      <td>44341</td>
      <td>48246</td>
      <td>2100</td>
      <td>2127</td>
    </tr>
    <tr>
      <td></td>
      <td>10x_bamtofastq</td>
      <td>10x_bamtofastq</td>
      <td>Converts 10x Genomics BAM to FASTQ</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/10XGenomics/bamtofastq</td>
      <td>Convert Formats</td>
      <td>10x_bamtofastq</td>
      <td>2022-05-10</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/10x_bamtofastq</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/10x_bamtofastq</td>
      <td>1.4.1</td>
      <td>10x_bamtofastq</td>
      <td>1.4.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>307</td>
      <td>307</td>
      <td>80</td>
      <td>80</td>
      <td>8</td>
      <td>8</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>1</td>
      <td>1</td>
      <td>318</td>
      <td>318</td>
      <td>84</td>
      <td>84</td>
    </tr>
    <tr>
      <td></td>
      <td>AggregateAlignments</td>
      <td>graphclust_aggregate_alignments</td>
      <td>Aggregate and filter alignment metrics of individual clusters, like the output of graphclust_align_cluster.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td></td>
      <td>RNA</td>
      <td>graphclust_aggregate_alignments</td>
      <td>2018-10-24</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tools/GraphClust/AggregateAlignments</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust/AggregateAlignments</td>
      <td>0.6.0</td>
      <td>graphclust-wrappers</td>
      <td>0.6.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>45</td>
      <td>119</td>
      <td>14</td>
      <td>17</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>45</td>
      <td>119</td>
      <td>14</td>
      <td>17</td>
    </tr>
    <tr>
      <td></td>
      <td>AlignCluster</td>
      <td>graphclust_align_cluster</td>
      <td>Align predicted clusters of glob_report_no_align step with locarna and conservation analysis and visualizations.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>graphclust_align_cluster</td>
      <td>2018-10-22</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tools/GraphClust/AlignCluster</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust/AlignCluster</td>
      <td>0.1</td>
      <td>graphclust-wrappers</td>
      <td>0.6.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>189</td>
      <td>3094</td>
      <td>23</td>
      <td>25</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>189</td>
      <td>3094</td>
      <td>23</td>
      <td>25</td>
    </tr>
    <tr>
      <td></td>
      <td>CMFinder</td>
      <td>cmFinder</td>
      <td>Determines consensus motives for sequences.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>graphclust_cmfinder</td>
      <td>2016-12-15</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust/CMFinder</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust/CMFinder</td>
      <td>0.4</td>
      <td>graphclust-wrappers</td>
      <td>0.6.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>25532</td>
      <td>50045</td>
      <td>45</td>
      <td>57</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>25532</td>
      <td>50045</td>
      <td>45</td>
      <td>57</td>
    </tr>
    <tr>
      <td></td>
      <td>CollectResults</td>
      <td>glob_report</td>
      <td>Post-processing. Redundant clusters are merged and instances that belong to multiple clusters are assigned unambiguously. For every pair of clusters, the relative overlap (i.e. the fraction of instances that occur in both clusters) is computed and clusters are merged if the overlap exceeds 50%. instances that occur in both clusters) is computed and clusters are merged if the overlap exceeds 50%.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>graphclust_postprocessing</td>
      <td>2017-01-21</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust/CollectResults</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust/CollectResults</td>
      <td>0.5</td>
      <td>graphclust-wrappers</td>
      <td>0.6.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1119</td>
      <td>2115</td>
      <td>50</td>
      <td>62</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1119</td>
      <td>2115</td>
      <td>50</td>
      <td>62</td>
    </tr>
    <tr>
      <td></td>
      <td>CollectResultsNoAlign</td>
      <td>graphclust_glob_report_no_align</td>
      <td>Redundant GraphClust clusters are merged and instances that belong to multiple clusters are assigned unambiguously.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>graphclust_postprocessing_no_align</td>
      <td>2018-10-25</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust/CollectResultsNoAlign</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust/CollectResultsNoAlign</td>
      <td>0.5</td>
      <td>graphclust-wrappers</td>
      <td>0.6.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>37</td>
      <td>108</td>
      <td>1</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>37</td>
      <td>108</td>
      <td>1</td>
      <td>3</td>
    </tr>
    <tr>
      <td></td>
      <td>GSPAN</td>
      <td>gspan</td>
      <td>Second step of GraphClust</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>graphclust_fasta_to_gspan</td>
      <td>2016-12-15</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust/GSPAN</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust/GSPAN</td>
      <td>0.4</td>
      <td>graphclust-wrappers</td>
      <td>0.6.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>80</td>
      <td>150</td>
      <td>18</td>
      <td>23</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>80</td>
      <td>150</td>
      <td>18</td>
      <td>23</td>
    </tr>
    <tr>
      <td></td>
      <td>LocARNAGraphClust</td>
      <td>locarna_best_subtree</td>
      <td>MLocARNA computes a multiple sequence-structure alignment of RNA sequences.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>graphclust_mlocarna</td>
      <td>2016-12-15</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust/LocARNAGraphClust</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust/LocARNAGraphClust</td>
      <td>0.4</td>
      <td>graphclust-wrappers</td>
      <td>0.6.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>25572</td>
      <td>50177</td>
      <td>45</td>
      <td>57</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>25572</td>
      <td>50177</td>
      <td>45</td>
      <td>57</td>
    </tr>
    <tr>
      <td></td>
      <td>NSPDK</td>
      <td>NSPDK_candidateClust, nspdk_sparse</td>
      <td>Produces an explicit sparse feature encoding and copmutes global feature index and returns top dense sets.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>graphclust_nspdk</td>
      <td>2016-12-15</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust/NSPDK</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust/NSPDK</td>
      <td>9.2.3.1</td>
      <td>graphclust-wrappers</td>
      <td>0.6.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>3210</td>
      <td>62854</td>
      <td>96</td>
      <td>124</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3210</td>
      <td>62854</td>
      <td>96</td>
      <td>124</td>
    </tr>
    <tr>
      <td></td>
      <td>Plotting</td>
      <td>motifFinderPlot</td>
      <td>Plotting results for GraphClust</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>graphclust_motif_finder_plot</td>
      <td>2017-02-22</td>
      <td>rnateam</td>
      <td>https://github.com/eteriSokhoyan/galaxytools/tree/master/tools/GraphClust/Plotting</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust/Plotting</td>
      <td>0.4</td>
      <td>seaborn</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>45</td>
      <td>71</td>
      <td>9</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>45</td>
      <td>71</td>
      <td>9</td>
      <td>10</td>
    </tr>
    <tr>
      <td></td>
      <td>PrepareForMlocarna</td>
      <td>preMloc</td>
      <td>This tool prepares files for locarna step.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>graphclust_prepocessing_for_mlocarna</td>
      <td>2016-12-15</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust/PrepareForMlocarna</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust/PrepareForMlocarna</td>
      <td>0.4</td>
      <td>graphclust-wrappers</td>
      <td>0.6.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1202</td>
      <td>2121</td>
      <td>48</td>
      <td>62</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1202</td>
      <td>2121</td>
      <td>48</td>
      <td>62</td>
    </tr>
    <tr>
      <td></td>
      <td>Preprocessing</td>
      <td>preproc</td>
      <td>Preprocessing input for GraphClust</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>graphclust_preprocessing</td>
      <td>2017-01-17</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust/Preprocessing</td>
      <td>0.5</td>
      <td>graphclust-wrappers</td>
      <td>0.6.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1116</td>
      <td>1897</td>
      <td>57</td>
      <td>71</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1116</td>
      <td>1897</td>
      <td>57</td>
      <td>71</td>
    </tr>
    <tr>
      <td></td>
      <td>Structure_GSPAN</td>
      <td>structure_to_gspan</td>
      <td>Convert RNA structure to GSPAN graphs</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>structure_to_gspan</td>
      <td>2017-05-18</td>
      <td>rnateam</td>
      <td>https://github.com/mmiladi/galaxytools/blob/graphclust-gspan/tools/GraphClust/Structure_GSPAN</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust/Structure_GSPAN</td>
      <td>0.4</td>
      <td>graphclust-wrappers</td>
      <td>0.6.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>991</td>
      <td>3060</td>
      <td>50</td>
      <td>62</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>991</td>
      <td>3060</td>
      <td>50</td>
      <td>62</td>
    </tr>
    <tr>
      <td></td>
      <td>add_line_to_file</td>
      <td>add_line_to_file</td>
      <td>Adds a text line to the beginning or end of a file.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>add_line_to_file</td>
      <td>2020-11-01</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/text_processing/add_line_to_file</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/text_processing/add_line_to_file</td>
      <td>0.1.0</td>
      <td>coreutils</td>
      <td>8.25</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>15933</td>
      <td>15933</td>
      <td>378</td>
      <td>378</td>
      <td>1736</td>
      <td>1736</td>
      <td>214</td>
      <td>214</td>
      <td>380</td>
      <td>380</td>
      <td>28</td>
      <td>28</td>
      <td>18049</td>
      <td>18049</td>
      <td>620</td>
      <td>620</td>
    </tr>
    <tr>
      <td></td>
      <td>add_value</td>
      <td>addValue</td>
      <td>Add a value as a new column.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>add_value</td>
      <td>2012-12-04</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/add_value</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/add_value</td>
      <td>1.0.1</td>
      <td>perl</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>37860</td>
      <td>374587</td>
      <td>1293</td>
      <td>1610</td>
      <td>17588</td>
      <td>293713</td>
      <td>1243</td>
      <td>6197</td>
      <td>3873</td>
      <td>4397</td>
      <td>202</td>
      <td>229</td>
      <td>59321</td>
      <td>672697</td>
      <td>2738</td>
      <td>8036</td>
    </tr>
    <tr>
      <td></td>
      <td>agat</td>
      <td>agat</td>
      <td>GTF/GFF analysis toolkit</td>
      <td>agat</td>
      <td>agat</td>
      <td></td>
      <td>AGAT</td>
      <td>Another Gff Analysis Toolkit (AGAT)Suite of tools to handle gene annotations in any GTF/GFF format.</td>
      <td>Data handling, Genome annotation</td>
      <td>Genomics</td>
      <td>To update</td>
      <td>https://github.com/NBISweden/AGAT</td>
      <td>Convert Formats, Statistics, Fasta Manipulation</td>
      <td>agat</td>
      <td>2023-05-22</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/agat</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/agat</td>
      <td>1.4.0</td>
      <td>agat</td>
      <td>1.4.1</td>
      <td>Data handling, Genome annotation</td>
      <td>Genomics</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1436</td>
      <td>1436</td>
      <td>244</td>
      <td>244</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>119</td>
      <td>119</td>
      <td>10</td>
      <td>10</td>
      <td>1555</td>
      <td>1555</td>
      <td>254</td>
      <td>254</td>
    </tr>
    <tr>
      <td></td>
      <td>align_back_trans</td>
      <td>align_back_trans</td>
      <td>Thread nucleotides onto a protein alignment (back-translation)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/align_back_trans</td>
      <td>Fasta Manipulation, Sequence Analysis</td>
      <td>align_back_trans</td>
      <td>2015-04-28</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/align_back_trans</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/align_back_trans</td>
      <td>0.0.10</td>
      <td>biopython</td>
      <td>1.70</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>511</td>
      <td>541</td>
      <td>32</td>
      <td>35</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>511</td>
      <td>541</td>
      <td>32</td>
      <td>35</td>
    </tr>
    <tr>
      <td></td>
      <td>antarna</td>
      <td>antarna</td>
      <td>antaRNA uses ant colony optimization to solve the inverse folding problem in RNA research .</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>antarna</td>
      <td>2015-04-30</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/antarna/</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/antarna</td>
      <td>1.1</td>
      <td>antarna</td>
      <td>2.0.1.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>292</td>
      <td>306</td>
      <td>5</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>292</td>
      <td>306</td>
      <td>5</td>
      <td>7</td>
    </tr>
    <tr>
      <td></td>
      <td>antismash</td>
      <td>antismash</td>
      <td>Antismash allows the genome-wide identification, annotation and analysis of secondary metabolite biosynthesis gene clusters</td>
      <td>antismash</td>
      <td>antismash</td>
      <td></td>
      <td>antiSMASH</td>
      <td>Rapid genome-wide identification, annotation and analysis of secondary metabolite biosynthesis gene clusters in bacterial and fungal genomes. It integrates and cross-links with a large number of in silico secondary metabolite analysis tools that have been published earlier.</td>
      <td>Sequence clustering, Gene prediction, Differential gene expression analysis</td>
      <td>Molecular interactions, pathways and networks, Gene and protein families</td>
      <td>To update</td>
      <td>https://antismash.secondarymetabolites.org</td>
      <td>Sequence Analysis</td>
      <td>antismash</td>
      <td>2015-02-28</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/antismash</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/antismash</td>
      <td>6.1.1</td>
      <td>antismash</td>
      <td>7.1.0</td>
      <td>Sequence clustering, Gene prediction, Differential gene expression analysis</td>
      <td>Molecular interactions, pathways and networks, Gene and protein families</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>15949</td>
      <td>16490</td>
      <td>726</td>
      <td>811</td>
      <td>5113</td>
      <td>5113</td>
      <td>649</td>
      <td>649</td>
      <td>6043</td>
      <td>6043</td>
      <td>169</td>
      <td>169</td>
      <td>27105</td>
      <td>27646</td>
      <td>1544</td>
      <td>1629</td>
    </tr>
    <tr>
      <td></td>
      <td>aresite2</td>
      <td>AREsite2_REST</td>
      <td>AREsite2 REST Interface</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://rna.tbi.univie.ac.at/AREsite</td>
      <td>RNA, Data Source, Sequence Analysis</td>
      <td>aresite2</td>
      <td>2017-02-01</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/aresite2</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/aresite2</td>
      <td>0.1.2</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>59</td>
      <td>74</td>
      <td>10</td>
      <td>12</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>59</td>
      <td>74</td>
      <td>10</td>
      <td>12</td>
    </tr>
    <tr>
      <td></td>
      <td>atactk_trim_adapters</td>
      <td>atactk_trim_adapters</td>
      <td>Trim adapters from paired-end HTS reads.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ParkerLab/atactk/</td>
      <td>Fastq Manipulation</td>
      <td>atactk_trim_adapters</td>
      <td>2018-06-20</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/blob/master/tools/trim_adapters</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/atactk_trim_adapters</td>
      <td>0.1.6</td>
      <td>atactk</td>
      <td>0.1.9</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>232</td>
      <td>279</td>
      <td>62</td>
      <td>72</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>232</td>
      <td>279</td>
      <td>62</td>
      <td>72</td>
    </tr>
    <tr>
      <td></td>
      <td>augustus</td>
      <td>augustus</td>
      <td>AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://bioinf.uni-greifswald.de/augustus/</td>
      <td>Sequence Analysis</td>
      <td>augustus</td>
      <td>2015-02-28</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/augustus</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/augustus</td>
      <td>3.1.0</td>
      <td>augustus</td>
      <td>3.5.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>10109</td>
      <td>11052</td>
      <td>1378</td>
      <td>1509</td>
      <td>12573</td>
      <td>12573</td>
      <td>2206</td>
      <td>2206</td>
      <td>5274</td>
      <td>5274</td>
      <td>369</td>
      <td>369</td>
      <td>27956</td>
      <td>28899</td>
      <td>3953</td>
      <td>4084</td>
    </tr>
    <tr>
      <td></td>
      <td>bamhash</td>
      <td>bamhash</td>
      <td>Hash BAM and FASTQ files to verify data integrity</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/DecodeGenetics/BamHash</td>
      <td>Sequence Analysis</td>
      <td>bamhash</td>
      <td>2015-09-17</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/bamhash</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/bamhash</td>
      <td>1.1</td>
      <td>bamhash</td>
      <td>1.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>158</td>
      <td>181</td>
      <td>40</td>
      <td>44</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>158</td>
      <td>181</td>
      <td>40</td>
      <td>44</td>
    </tr>
    <tr>
      <td></td>
      <td>barcode_collapse</td>
      <td>barcode_collapse</td>
      <td>Paired End randomer aware duplicate removal algorithm</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/YeoLab/gscripts</td>
      <td>RNA, Sequence Analysis</td>
      <td>barcode_collapse</td>
      <td>2016-06-11</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/barcode_collapse</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/barcode_collapse</td>
      <td>0.1.0</td>
      <td>pysam</td>
      <td>0.22.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>basecoverage</td>
      <td>gops_basecoverage_1</td>
      <td>Base Coverage of all intervals</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/gops</td>
      <td>Genomic Interval Operations</td>
      <td>basecoverage</td>
      <td>2014-04-01</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/gops/basecoverage</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/gops/basecoverage</td>
      <td>1.0.0</td>
      <td>bx-python</td>
      <td>0.13.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>226</td>
      <td>308</td>
      <td>27</td>
      <td>35</td>
      <td>2615</td>
      <td>45557</td>
      <td>302</td>
      <td>2284</td>
      <td>482</td>
      <td>522</td>
      <td>32</td>
      <td>36</td>
      <td>3323</td>
      <td>46387</td>
      <td>361</td>
      <td>2355</td>
    </tr>
    <tr>
      <td></td>
      <td>best_regression_subsets</td>
      <td>BestSubsetsRegression1</td>
      <td>Perform Best-subsets Regression</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>best_regression_subsets</td>
      <td>2014-04-01</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/best_regression_subsets</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/best_regression_subsets</td>
      <td>1.0.0</td>
      <td>numpy</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>3</td>
      <td>4</td>
      <td>3</td>
      <td>4</td>
      <td>14</td>
      <td>253</td>
      <td>7</td>
      <td>72</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>17</td>
      <td>257</td>
      <td>10</td>
      <td>76</td>
    </tr>
    <tr>
      <td></td>
      <td>bia-ftplinks</td>
      <td>bia_download</td>
      <td>Tool to query ftp links for study from bioimage archive</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Imaging</td>
      <td>bia_download</td>
      <td>2023-12-06</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/image_processing/bia-ftplinks</td>
      <td>0.1.0</td>
      <td>wget</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>2</td>
      <td>2</td>
    </tr>
    <tr>
      <td></td>
      <td>bigwig_to_bedgraph</td>
      <td>bigwig_to_bedgraph</td>
      <td>Convert from bigWig to bedGraph format</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Convert Formats</td>
      <td>bigwig_to_bedgraph</td>
      <td>2015-06-04</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/bigwig_to_bedgraph</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/bigwig_to_bedgraph</td>
      <td>0.1.0</td>
      <td>ucsc_tools</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>5769</td>
      <td>6802</td>
      <td>567</td>
      <td>650</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>1</td>
      <td>5769</td>
      <td>6805</td>
      <td>567</td>
      <td>651</td>
    </tr>
    <tr>
      <td></td>
      <td>bioimaging</td>
      <td>bioimage_inference</td>
      <td>Load model from BioImage.IO and make inferences</td>
      <td>pytorch</td>
      <td>pytorch</td>
      <td>pytorch</td>
      <td>PyTorch</td>
      <td>PyTorch is an optimized tensor library for deep learning using GPUs and CPUs.</td>
      <td></td>
      <td>Machine learning, Computer science</td>
      <td>To update</td>
      <td>https://github.com/bgruening/galaxytools</td>
      <td>Imaging</td>
      <td>bioimage_inference</td>
      <td>2024-08-02</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/recommendation_training/tools/bioimaging</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/bioimaging</td>
      <td>2.4.1</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td>Machine learning</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>1</td>
      <td>1</td>
    </tr>
    <tr>
      <td></td>
      <td>biomodelsML</td>
      <td>biomodels_biomd0000001066, biomodels_biomd0000001076</td>
      <td>Wrappers for tools to bring BioModels AI models into Galaxy.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.ebi.ac.uk/biomodels/</td>
      <td>Machine Learning</td>
      <td>biomodels</td>
      <td>2023-11-07</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/biomodelsML</td>
      <td>1.1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>25</td>
      <td>25</td>
      <td>11</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>25</td>
      <td>25</td>
      <td>11</td>
      <td>11</td>
    </tr>
    <tr>
      <td></td>
      <td>bionano</td>
      <td>bionano_scaffold</td>
      <td>Bionano Solve is a set of tools for analyzing Bionano data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bionanogenomics.com/</td>
      <td>Assembly</td>
      <td>bionano</td>
      <td>2021-05-17</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/bionano</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/bionano</td>
      <td>3.7.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>547</td>
      <td>547</td>
      <td>151</td>
      <td>151</td>
      <td>143</td>
      <td>143</td>
      <td>43</td>
      <td>43</td>
      <td>596</td>
      <td>596</td>
      <td>22</td>
      <td>22</td>
      <td>1286</td>
      <td>1286</td>
      <td>216</td>
      <td>216</td>
    </tr>
    <tr>
      <td></td>
      <td>bismark</td>
      <td>bismark_pretty_report, bismark_bowtie2, bismark_deduplicate, bismark_methylation_extractor</td>
      <td>A tool to map bisulfite converted sequence reads and determine cytosine methylation states</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.bioinformatics.babraham.ac.uk/projects/bismark/</td>
      <td>Sequence Analysis, Next Gen Mappers</td>
      <td>bismark</td>
      <td>2018-08-22</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/bismark</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/bismark</td>
      <td>0.22.1</td>
      <td>bismark</td>
      <td>0.24.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>13715</td>
      <td>14997</td>
      <td>992</td>
      <td>1099</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>197</td>
      <td>197</td>
      <td>22</td>
      <td>22</td>
      <td>13912</td>
      <td>15194</td>
      <td>1014</td>
      <td>1121</td>
    </tr>
    <tr>
      <td></td>
      <td>blat_coverage_report</td>
      <td>generate_coverage_report</td>
      <td>Polymorphism of the Reads</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Next Gen Mappers, Sequence Analysis</td>
      <td>blat_coverage_report</td>
      <td>2014-05-19</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/blat_coverage_report</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/blat_coverage_report</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>blat_mapping</td>
      <td>blat2wig</td>
      <td>Coverage of the Reads in wiggle format</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Next Gen Mappers, Sequence Analysis</td>
      <td>blat_mapping</td>
      <td>2014-05-19</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/blat_mapping</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/blat_mapping</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>blobtoolkit</td>
      <td>blobtoolkit</td>
      <td>Identification and isolation non-target data in draft and publicly available genome assemblies.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://blobtoolkit.genomehubs.org/</td>
      <td>Sequence Analysis, Assembly</td>
      <td>blobtoolkit</td>
      <td>2022-11-10</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/blobtoolkit</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/blobtoolkit</td>
      <td>4.0.7</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1527</td>
      <td>1527</td>
      <td>79</td>
      <td>79</td>
      <td>6</td>
      <td>6</td>
      <td>1</td>
      <td>1</td>
      <td>317</td>
      <td>317</td>
      <td>26</td>
      <td>26</td>
      <td>1850</td>
      <td>1850</td>
      <td>106</td>
      <td>106</td>
    </tr>
    <tr>
      <td></td>
      <td>blockbuster</td>
      <td>blockbuster</td>
      <td>Blockbuster detects blocks of overlapping reads using a gaussian-distribution approach.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://hoffmann.bioinf.uni-leipzig.de/LIFE/blockbuster.html</td>
      <td>RNA, Sequence Analysis</td>
      <td>blockbuster</td>
      <td>2015-02-28</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/blockbuster</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/blockbuster</td>
      <td>0.1.2</td>
      <td>blockbuster</td>
      <td>0.0.1.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>768</td>
      <td>3043</td>
      <td>97</td>
      <td>119</td>
      <td>14</td>
      <td>14</td>
      <td>11</td>
      <td>11</td>
      <td>276</td>
      <td>276</td>
      <td>9</td>
      <td>9</td>
      <td>1058</td>
      <td>3333</td>
      <td>117</td>
      <td>139</td>
    </tr>
    <tr>
      <td></td>
      <td>blockclust</td>
      <td>blockclust</td>
      <td>BlockClust detects transcripts with similar processing patterns.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/workflows/blockclust</td>
      <td>RNA</td>
      <td>blockclust</td>
      <td>2015-02-28</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/blockclust</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/blockclust</td>
      <td>1.1.1</td>
      <td>blockclust</td>
      <td>1.1.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>811</td>
      <td>1501</td>
      <td>54</td>
      <td>79</td>
      <td>64</td>
      <td>64</td>
      <td>29</td>
      <td>29</td>
      <td>561</td>
      <td>561</td>
      <td>15</td>
      <td>15</td>
      <td>1436</td>
      <td>2126</td>
      <td>98</td>
      <td>123</td>
    </tr>
    <tr>
      <td></td>
      <td>bowtie_wrappers</td>
      <td>bowtie_wrapper</td>
      <td>Galaxy wrappers for the Bowtie short read mapping tools.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://bowtie-bio.sourceforge.net/</td>
      <td>Next Gen Mappers</td>
      <td>bowtie_wrappers</td>
      <td>2012-11-26</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/bowtie_wrappers</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/bowtie_wrappers</td>
      <td>1.2.0</td>
      <td>bowtie</td>
      <td>1.3.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>24164</td>
      <td>25149</td>
      <td>1320</td>
      <td>1480</td>
      <td>17537</td>
      <td>203397</td>
      <td>1330</td>
      <td>17491</td>
      <td>15654</td>
      <td>15992</td>
      <td>291</td>
      <td>351</td>
      <td>57355</td>
      <td>244538</td>
      <td>2941</td>
      <td>19322</td>
    </tr>
    <tr>
      <td></td>
      <td>canonical_correlation_analysis</td>
      <td>cca1</td>
      <td>Canonical Correlation Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>canonical_correlation_analysis</td>
      <td>2014-05-19</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/canonical_correlation_analysis</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/canonical_correlation_analysis</td>
      <td>1.0.0</td>
      <td>R</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>66</td>
      <td>75</td>
      <td>16</td>
      <td>18</td>
      <td>141</td>
      <td>505</td>
      <td>31</td>
      <td>147</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>207</td>
      <td>580</td>
      <td>47</td>
      <td>165</td>
    </tr>
    <tr>
      <td></td>
      <td>canu</td>
      <td>canu</td>
      <td>Canu is a hierarchical assembly pipeline designed for high-noise single-molecule sequencing (such as the PacBio RS II/Sequel or Oxford Nanopore MinION).</td>
      <td>canu</td>
      <td>canu</td>
      <td></td>
      <td>CANU</td>
      <td>De-novo assembly tool for long read chemistry like Nanopore data and PacBio data.</td>
      <td>De-novo assembly</td>
      <td>Genomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/marbl/canu</td>
      <td></td>
      <td>canu</td>
      <td>2018-06-07</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/canu</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/canu</td>
      <td>2.2</td>
      <td>canu</td>
      <td>2.2</td>
      <td>De-novo assembly</td>
      <td>Genomics</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>13360</td>
      <td>14618</td>
      <td>1383</td>
      <td>1444</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3542</td>
      <td>3542</td>
      <td>385</td>
      <td>385</td>
      <td>16902</td>
      <td>18160</td>
      <td>1768</td>
      <td>1829</td>
    </tr>
    <tr>
      <td></td>
      <td>categorize_elements_satisfying_criteria</td>
      <td>categorize_elements_satisfying_criteria</td>
      <td>Categorize Elements satisfying criteria.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>categorize_elements_satisfying_criteria</td>
      <td>2013-09-25</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/categorize_elements_satisfying_criteria</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/categorize_elements_satisfying_criteria</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>37</td>
      <td>41</td>
      <td>4</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>168</td>
      <td>186</td>
      <td>12</td>
      <td>14</td>
      <td>205</td>
      <td>227</td>
      <td>16</td>
      <td>19</td>
    </tr>
    <tr>
      <td></td>
      <td>ccat</td>
      <td>peakcalling_ccat</td>
      <td>Control-based ChIP-seq Analysis Tool</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>ChIP-seq</td>
      <td>ccat</td>
      <td>2014-01-27</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/ccat</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/ccat</td>
      <td>0.0.2</td>
      <td>ccat</td>
      <td>3.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>88</td>
      <td>175</td>
      <td>28</td>
      <td>39</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>88</td>
      <td>175</td>
      <td>28</td>
      <td>39</td>
    </tr>
    <tr>
      <td></td>
      <td>cd_hit_dup</td>
      <td>cd_hit_dup</td>
      <td>simple tool for removing duplicates from sequencing reads</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Metagenomics, Sequence Analysis</td>
      <td>cd_hit_dup</td>
      <td>2015-04-28</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/cd_hit_dup</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/cd_hit_dup</td>
      <td>0.0.1</td>
      <td>cd-hit-auxtools</td>
      <td>4.8.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5946</td>
      <td>7379</td>
      <td>407</td>
      <td>628</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5946</td>
      <td>7379</td>
      <td>407</td>
      <td>628</td>
    </tr>
    <tr>
      <td></td>
      <td>cellpose</td>
      <td>cellpose</td>
      <td>Cellpose is an anatomical segmentation algorithm</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/MouseLand/cellpose</td>
      <td>Imaging</td>
      <td>cellpose</td>
      <td>2024-02-29</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/cellpose</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/cellpose</td>
      <td>3.0.10</td>
      <td>cellpose</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>14</td>
      <td>14</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>14</td>
      <td>14</td>
      <td>4</td>
      <td>4</td>
    </tr>
    <tr>
      <td></td>
      <td>cellprofiler</td>
      <td>cp_cellprofiler, cp_color_to_gray, cp_convert_objects_to_image, cp_display_data_on_image, cp_enhance_or_suppress_features, cp_export_to_spreadsheet, cp_gray_to_color, cp_identify_primary_objects, cp_image_math, cp_mask_image, cp_measure_granularity, cp_measure_image_area_occupied, cp_measure_image_intensity, cp_measure_image_quality, cp_measure_object_intensity, cp_measure_object_size_shape, cp_measure_texture, cp_overlay_outlines, cp_relate_objects, cp_save_images, cp_common, cp_tile, cp_track_objects</td>
      <td>cellProfiler wrapper</td>
      <td>CellProfiler</td>
      <td>CellProfiler</td>
      <td>cellprofiler</td>
      <td>CellProfiler</td>
      <td>Tool for quantifying data from biological images, particularly in high-throughput experiments.</td>
      <td>Quantification, Image analysis, Parsing</td>
      <td>Imaging, Microarray experiment, Genotype and phenotype</td>
      <td>To update</td>
      <td></td>
      <td>Imaging</td>
      <td>cellprofiler</td>
      <td>2020-05-05</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/cellprofiler</td>
      <td></td>
      <td></td>
      <td></td>
      <td>Quantification, Image analysis, Parsing</td>
      <td>Imaging, Microarray experiment, Genotype and phenotype</td>
      <td>0</td>
      <td>23</td>
      <td>23</td>
      <td>19</td>
      <td>0</td>
      <td>23</td>
      <td>23</td>
      <td>19</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>19</td>
      <td>23</td>
      <td>19</td>
      <td>0</td>
      <td>4773</td>
      <td>4773</td>
      <td>749</td>
      <td>749</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5688</td>
      <td>5688</td>
      <td>94</td>
      <td>94</td>
      <td>10461</td>
      <td>10461</td>
      <td>843</td>
      <td>843</td>
    </tr>
    <tr>
      <td></td>
      <td>cellprofiler_v4</td>
      <td>cp_cellprofiler4</td>
      <td>cellProfiler4 wrapper</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/CellProfiler/CellProfiler</td>
      <td>Imaging</td>
      <td>cellprofiler4</td>
      <td>2021-09-15</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/cellprofiler_v4</td>
      <td>4.2.7</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>55</td>
      <td>55</td>
      <td>11</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>55</td>
      <td>55</td>
      <td>11</td>
      <td>11</td>
    </tr>
    <tr>
      <td></td>
      <td>change_case</td>
      <td>ChangeCase</td>
      <td>Convert column case.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>change_case</td>
      <td>2012-12-04</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/change_case</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/change_case</td>
      <td>1.0.1</td>
      <td>perl</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>10606</td>
      <td>12531</td>
      <td>2204</td>
      <td>2547</td>
      <td>9044</td>
      <td>17167</td>
      <td>1748</td>
      <td>2394</td>
      <td>923</td>
      <td>938</td>
      <td>240</td>
      <td>245</td>
      <td>20573</td>
      <td>30636</td>
      <td>4192</td>
      <td>5186</td>
    </tr>
    <tr>
      <td></td>
      <td>chatgpt</td>
      <td>chatgpt_openai_api</td>
      <td>Using the OpenAI GPT models to generate text based on user input.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/chatgpt</td>
      <td>Machine Learning</td>
      <td>chatgpt_openai_api</td>
      <td>2024-08-12</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/chatgpt</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/chatgpt</td>
      <td>2024</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>62</td>
      <td>62</td>
      <td>12</td>
      <td>12</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>62</td>
      <td>62</td>
      <td>12</td>
      <td>12</td>
    </tr>
    <tr>
      <td></td>
      <td>chipseeker</td>
      <td>chipseeker</td>
      <td>A tool for ChIP peak annotation and visualization</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/ChIPseeker.html</td>
      <td>ChIP-seq, Genome annotation</td>
      <td>chipseeker</td>
      <td>2018-05-28</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/chipseeker</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/chipseeker</td>
      <td>1.32.0</td>
      <td>bioconductor-chipseeker</td>
      <td>1.38.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>16468</td>
      <td>18344</td>
      <td>956</td>
      <td>991</td>
      <td>33692</td>
      <td>34453</td>
      <td>2559</td>
      <td>2625</td>
      <td>2948</td>
      <td>2948</td>
      <td>104</td>
      <td>104</td>
      <td>53108</td>
      <td>55745</td>
      <td>3619</td>
      <td>3720</td>
    </tr>
    <tr>
      <td></td>
      <td>chromosome_diagram</td>
      <td>chromosome_diagram</td>
      <td>Chromosome Diagrams using Biopython</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Graphics, Sequence Analysis, Visualization</td>
      <td>chromosome_diagram</td>
      <td>2014-11-19</td>
      <td>peterjc</td>
      <td></td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/chromosome_diagram</td>
      <td>0.0.3</td>
      <td>biopython</td>
      <td>1.70</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>circexplorer</td>
      <td>circexplorer</td>
      <td>A combined strategy to identify circular RNAs (circRNAs and ciRNAs)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/YangLab/CIRCexplorer</td>
      <td>Sequence Analysis, RNA</td>
      <td>circexplorer</td>
      <td>2016-06-06</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/circexplorer</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/circexplorer</td>
      <td>1.1.9.0</td>
      <td>circexplorer</td>
      <td>1.1.10</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>211</td>
      <td>263</td>
      <td>32</td>
      <td>40</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>209</td>
      <td>209</td>
      <td>6</td>
      <td>6</td>
      <td>420</td>
      <td>472</td>
      <td>38</td>
      <td>46</td>
    </tr>
    <tr>
      <td></td>
      <td>clc_assembly_cell</td>
      <td>clc_assembler, clc_mapper</td>
      <td>Galaxy wrapper for the CLC Assembly Cell suite from CLCBio</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/clc_assembly_cell</td>
      <td>Assembly, Next Gen Mappers, SAM</td>
      <td>clc_assembly_cell</td>
      <td>2013-11-05</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/clc_assembly_cell</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/clc_assembly_cell</td>
      <td>0.0.7</td>
      <td>samtools</td>
      <td>1.21</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>clinod</td>
      <td>clinod</td>
      <td>NoD: a Nucleolar localization sequence detector for eukaryotic and viral proteins</td>
      <td>clinod</td>
      <td>clinod</td>
      <td></td>
      <td>clinod</td>
      <td>The command line NoD predictor (clinod) can be run from the command line to predict Nucleolar localization sequences (NoLSs) that are short targeting sequences responsible for the localization of proteins to the nucleolus.The predictor accepts a list of FASTA formatted sequences as an input and outputs the NOLS predictions as a result.Please note that currently, JPred secondary structure predictions are not supported by clinod. However, we are working on it.</td>
      <td>Nucleic acid sequence analysis</td>
      <td>Sequence analysis</td>
      <td>To update</td>
      <td>http://www.compbio.dundee.ac.uk/www-nod/</td>
      <td>Sequence Analysis</td>
      <td>clinod</td>
      <td>2014-02-18</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/clinod</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/clinod</td>
      <td>0.1.0</td>
      <td>clinod</td>
      <td>1.3</td>
      <td>Nucleic acid sequence analysis</td>
      <td>Sequence analysis</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>548</td>
      <td>548</td>
      <td>18</td>
      <td>18</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>548</td>
      <td>548</td>
      <td>18</td>
      <td>18</td>
    </tr>
    <tr>
      <td></td>
      <td>cluster</td>
      <td>gops_cluster_1</td>
      <td>Cluster the intervals of a dataset</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/gops</td>
      <td>Genomic Interval Operations</td>
      <td>cluster</td>
      <td>2014-04-01</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/gops/cluster</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/gops/cluster</td>
      <td>1.0.0</td>
      <td>bx-python</td>
      <td>0.13.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>513</td>
      <td>813</td>
      <td>18</td>
      <td>35</td>
      <td>2406</td>
      <td>30579</td>
      <td>156</td>
      <td>1547</td>
      <td>1353</td>
      <td>1393</td>
      <td>17</td>
      <td>20</td>
      <td>4272</td>
      <td>32785</td>
      <td>191</td>
      <td>1602</td>
    </tr>
    <tr>
      <td></td>
      <td>cmsearch_deoverlap</td>
      <td>cmsearch_deoverlap</td>
      <td>removes lower scoring overlaps from cmsearch results.</td>
      <td>cmsearch-deoverlap</td>
      <td>cmsearch-deoverlap</td>
      <td></td>
      <td>cmsearch-deoverlap</td>
      <td>Removes lower scoring overlaps from cmsearch results.</td>
      <td>Comparison, Alignment</td>
      <td>Biology, Medicine</td>
      <td>To update</td>
      <td>https://github.com/EBI-Metagenomics/pipeline-v5/blob/master/tools/RNA_prediction/cmsearch-deoverlap/cmsearch-deoverlap.pl</td>
      <td>RNA</td>
      <td>cmsearch_deoverlap</td>
      <td>2023-08-19</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/cmsearch_deoverlap</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/cmsearch_deoverlap</td>
      <td>0.08+galaxy2</td>
      <td>perl</td>
      <td></td>
      <td>Comparison, Alignment</td>
      <td>Biology, Medicine</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>2609</td>
      <td>2609</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2609</td>
      <td>2609</td>
      <td>3</td>
      <td>3</td>
    </tr>
    <tr>
      <td></td>
      <td>cmv</td>
      <td>cmcv, cmv, hmmcv, hmmv</td>
      <td>cmv is a collection of tools for the visualisation of Hidden Markov Models and RNA-family models.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/eggzilla/cmv</td>
      <td>RNA</td>
      <td>cmv</td>
      <td>2017-09-26</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/cmv</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/cmv</td>
      <td>1.0.8</td>
      <td>cmv</td>
      <td>1.0.8</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>90</td>
      <td>112</td>
      <td>11</td>
      <td>14</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>90</td>
      <td>112</td>
      <td>11</td>
      <td>14</td>
    </tr>
    <tr>
      <td></td>
      <td>cofold</td>
      <td>cofold</td>
      <td>Cofold predicts RNA secondary structures that takes co-transcriptional folding into account.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.e-rna.org/cofold/</td>
      <td>RNA</td>
      <td>cofold</td>
      <td>2015-02-28</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/cofold</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/cofold</td>
      <td>2.0.4.0</td>
      <td>cofold</td>
      <td>2.0.4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>316</td>
      <td>355</td>
      <td>25</td>
      <td>29</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>316</td>
      <td>355</td>
      <td>25</td>
      <td>29</td>
    </tr>
    <tr>
      <td></td>
      <td>column_arrange_by_header</td>
      <td>bg_column_arrange_by_header</td>
      <td>Column arrange by header name</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>column_arrange_by_header</td>
      <td>2015-03-02</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/text_processing/column_arrange_by_header</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/text_processing/column_arrange_by_header</td>
      <td>0.2</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>5356</td>
      <td>5636</td>
      <td>446</td>
      <td>484</td>
      <td>2026</td>
      <td>2026</td>
      <td>375</td>
      <td>375</td>
      <td>1400</td>
      <td>1531</td>
      <td>117</td>
      <td>138</td>
      <td>8782</td>
      <td>9193</td>
      <td>938</td>
      <td>997</td>
    </tr>
    <tr>
      <td></td>
      <td>combine_metaphlan_humann</td>
      <td>combine_metaphlan_humann</td>
      <td>Combine MetaPhlAn2 and HUMAnN2 outputs to relate genus/species abundances and gene families/pathways abundances</td>
      <td>combine_metaphlan_and_humann</td>
      <td>combine_metaphlan_and_humann</td>
      <td></td>
      <td>Combine Metaphlan and HUMAnN</td>
      <td>This tool combine MetaPhlAn outputs and HUMANnN outputs</td>
      <td>Aggregation</td>
      <td>Metagenomics, Molecular interactions, pathways and networks</td>
      <td>To update</td>
      <td></td>
      <td>Metagenomics</td>
      <td>combine_metaphlan2_humann2</td>
      <td>2023-07-20</td>
      <td>bebatut</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/combine_metaphlan2_humann2</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/combine_metaphlan_humann</td>
      <td>0.3.0</td>
      <td>python</td>
      <td></td>
      <td>Aggregation</td>
      <td>Metagenomics, Molecular interactions, pathways and networks</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>530</td>
      <td>530</td>
      <td>65</td>
      <td>65</td>
      <td>8</td>
      <td>8</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>538</td>
      <td>538</td>
      <td>71</td>
      <td>71</td>
    </tr>
    <tr>
      <td></td>
      <td>compalignp</td>
      <td>compalignp</td>
      <td>Compute fractional identity between trusted alignment and test alignment</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td></td>
      <td>RNA, Sequence Analysis</td>
      <td>compalignp</td>
      <td>2015-06-03</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/compalignp/</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/compalignp</td>
      <td>1.0</td>
      <td>compalignp</td>
      <td>1.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>39</td>
      <td>221</td>
      <td>5</td>
      <td>28</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>39</td>
      <td>221</td>
      <td>5</td>
      <td>28</td>
    </tr>
    <tr>
      <td></td>
      <td>compare_humann2_output</td>
      <td>compare_humann2_output</td>
      <td>Compare outputs of HUMAnN2 for several samples and extract similar and specific information</td>
      <td>compare_humann2_outputs</td>
      <td>compare_humann2_outputs</td>
      <td></td>
      <td>Compare HUMAnN2 outputs</td>
      <td>This tool compare HUMANnN2 outputs with gene families or pathways and their relative abundances between several samples</td>
      <td>Comparison</td>
      <td>Metagenomics, Gene and protein families</td>
      <td>To update</td>
      <td></td>
      <td>Metagenomics</td>
      <td>compare_humann2_output</td>
      <td>2022-10-19</td>
      <td>bebatut</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/compare_humann2_output</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/compare_humann2_output</td>
      <td>0.2.0</td>
      <td></td>
      <td></td>
      <td>Comparison</td>
      <td>Metagenomics, Gene and protein families</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>306</td>
      <td>332</td>
      <td>28</td>
      <td>32</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>306</td>
      <td>332</td>
      <td>28</td>
      <td>32</td>
    </tr>
    <tr>
      <td></td>
      <td>complement</td>
      <td>gops_complement_1</td>
      <td>Complement intervals of a dataset</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/gops</td>
      <td>Genomic Interval Operations</td>
      <td>complement</td>
      <td>2014-04-01</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/gops/complement</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/gops/complement</td>
      <td>1.0.0</td>
      <td>bx-python</td>
      <td>0.13.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>187</td>
      <td>233</td>
      <td>15</td>
      <td>20</td>
      <td>495</td>
      <td>4810</td>
      <td>165</td>
      <td>836</td>
      <td>405</td>
      <td>451</td>
      <td>7</td>
      <td>7</td>
      <td>1087</td>
      <td>5494</td>
      <td>187</td>
      <td>863</td>
    </tr>
    <tr>
      <td></td>
      <td>compute_motif_frequencies_for_all_motifs</td>
      <td>compute_motif_frequencies_for_all_motifs</td>
      <td>Compute Motif Frequencies For All Motifs, motif by motif.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Statistics</td>
      <td>compute_motif_frequencies_for_all_motifs</td>
      <td>2013-09-25</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/compute_motif_frequencies_for_all_motifs</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/compute_motif_frequencies_for_all_motifs</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>85</td>
      <td>96</td>
      <td>9</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>316</td>
      <td>334</td>
      <td>8</td>
      <td>8</td>
      <td>401</td>
      <td>430</td>
      <td>17</td>
      <td>18</td>
    </tr>
    <tr>
      <td></td>
      <td>compute_motifs_frequency</td>
      <td>compute_motifs_frequency</td>
      <td>Compute Motif Frequencies in indel flanking regions.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Statistics</td>
      <td>compute_motifs_frequency</td>
      <td>2013-09-25</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/compute_motifs_frequency</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/compute_motifs_frequency</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>55</td>
      <td>65</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>317</td>
      <td>335</td>
      <td>5</td>
      <td>5</td>
      <td>372</td>
      <td>400</td>
      <td>7</td>
      <td>7</td>
    </tr>
    <tr>
      <td></td>
      <td>compute_q_values</td>
      <td>compute_q_values</td>
      <td>Compute q-values based on multiple simultaneous tests p-values</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>compute_q_values</td>
      <td>2014-04-01</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/compute_q_values</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/compute_q_values</td>
      <td>1.0.1</td>
      <td>R</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>35</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>35</td>
      <td>0</td>
      <td>3</td>
    </tr>
    <tr>
      <td></td>
      <td>concat</td>
      <td>gops_concat_1</td>
      <td>Concatenate two bed files</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/gops</td>
      <td>Genomic Interval Operations</td>
      <td>concat</td>
      <td>2014-04-01</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/gops/concat</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/gops/concat</td>
      <td>1.0.1</td>
      <td>bx-python</td>
      <td>0.13.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>575128</td>
      <td>575441</td>
      <td>694</td>
      <td>783</td>
      <td>43420</td>
      <td>85345</td>
      <td>629</td>
      <td>3508</td>
      <td>19330</td>
      <td>19418</td>
      <td>104</td>
      <td>117</td>
      <td>637878</td>
      <td>680204</td>
      <td>1427</td>
      <td>4408</td>
    </tr>
    <tr>
      <td></td>
      <td>condense_characters</td>
      <td>Condense characters1</td>
      <td>Condense consecutive characters.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>condense_characters</td>
      <td>2012-12-04</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/condense_characters</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/condense_characters</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>47</td>
      <td>2173</td>
      <td>28</td>
      <td>368</td>
      <td>195</td>
      <td>205</td>
      <td>8</td>
      <td>8</td>
      <td>242</td>
      <td>2378</td>
      <td>36</td>
      <td>376</td>
    </tr>
    <tr>
      <td></td>
      <td>convert_characters</td>
      <td>Convert characters1</td>
      <td>Convert delimiters to tab.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>convert_characters</td>
      <td>2012-12-04</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/convert_characters</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/convert_characters</td>
      <td>1.0.1</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>23589</td>
      <td>28986</td>
      <td>1061</td>
      <td>1200</td>
      <td>23507</td>
      <td>160693</td>
      <td>2302</td>
      <td>8699</td>
      <td>5413</td>
      <td>6446</td>
      <td>251</td>
      <td>329</td>
      <td>52509</td>
      <td>196125</td>
      <td>3614</td>
      <td>10228</td>
    </tr>
    <tr>
      <td></td>
      <td>convert_solid_color2nuc</td>
      <td>color2nuc</td>
      <td>Convert Color Space to Nucleotides</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Fasta Manipulation</td>
      <td>convert_solid_color2nuc</td>
      <td>2014-05-19</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/convert_solid_color2nuc</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/convert_solid_color2nuc</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>coprarna</td>
      <td>coprarna</td>
      <td>Target prediction for prokaryotic trans-acting small RNAs</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/PatrickRWright/CopraRNA</td>
      <td>RNA, Sequence Analysis</td>
      <td>coprarna</td>
      <td>2017-11-01</td>
      <td>rnateam</td>
      <td>https://github.com/PatrickRWright/CopraRNA</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/coprarna</td>
      <td>2.1.1</td>
      <td>coprarna</td>
      <td>2.1.4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>correlation</td>
      <td>cor2</td>
      <td>Correlation for numeric columns</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>correlation</td>
      <td>2014-07-28</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/correlation</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/correlation</td>
      <td>1.0.0</td>
      <td>rpy</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>398</td>
      <td>876</td>
      <td>106</td>
      <td>134</td>
      <td>412</td>
      <td>5568</td>
      <td>105</td>
      <td>755</td>
      <td>192</td>
      <td>239</td>
      <td>21</td>
      <td>27</td>
      <td>1002</td>
      <td>6683</td>
      <td>232</td>
      <td>916</td>
    </tr>
    <tr>
      <td></td>
      <td>count_gff_features</td>
      <td>count_gff_features</td>
      <td>Count GFF Features</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>count_gff_features</td>
      <td>2014-07-28</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/count_gff_features</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/count_gff_features</td>
      <td>0.2</td>
      <td>galaxy-ops</td>
      <td>1.1.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>280</td>
      <td>307</td>
      <td>110</td>
      <td>124</td>
      <td>712</td>
      <td>1277</td>
      <td>332</td>
      <td>628</td>
      <td>476</td>
      <td>541</td>
      <td>55</td>
      <td>79</td>
      <td>1468</td>
      <td>2125</td>
      <td>497</td>
      <td>831</td>
    </tr>
    <tr>
      <td></td>
      <td>count_roi_variants</td>
      <td>count_roi_variants</td>
      <td>Count sequence variants in region of interest in BAM file</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/count_roi_variants</td>
      <td>Assembly, SAM</td>
      <td>count_roi_variants</td>
      <td>2016-02-05</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/count_roi_variants</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/count_roi_variants</td>
      <td>0.0.6</td>
      <td>samtools</td>
      <td>1.21</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>coverage</td>
      <td>gops_coverage_1</td>
      <td>Coverage of a set of intervals on second set of intervals</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/gops</td>
      <td>Genomic Interval Operations</td>
      <td>coverage</td>
      <td>2014-04-01</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/gops/coverage</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/gops/coverage</td>
      <td>1.0.0</td>
      <td>bx-python</td>
      <td>0.13.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1560</td>
      <td>1725</td>
      <td>43</td>
      <td>58</td>
      <td>4328</td>
      <td>44209</td>
      <td>193</td>
      <td>2113</td>
      <td>283</td>
      <td>316</td>
      <td>16</td>
      <td>18</td>
      <td>6171</td>
      <td>46250</td>
      <td>252</td>
      <td>2189</td>
    </tr>
    <tr>
      <td></td>
      <td>coverage_stats</td>
      <td>coverage_stats</td>
      <td>BAM coverage statistics using samtools idxstats and depth</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/coverage_stats</td>
      <td>Assembly, SAM</td>
      <td>coverage_stats</td>
      <td>2015-07-29</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/coverage_stats</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/coverage_stats</td>
      <td>0.1.0</td>
      <td>samtools</td>
      <td>1.21</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>cpat</td>
      <td>cpat</td>
      <td>Coding-potential assessment tool using an alignment-free logistic regression model.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/liguowang/cpat</td>
      <td>Transcriptomics</td>
      <td>cpat</td>
      <td>2023-02-01</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/cpat</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/cpat</td>
      <td>3.0.5</td>
      <td>cpat</td>
      <td>3.0.5</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>204</td>
      <td>204</td>
      <td>47</td>
      <td>47</td>
      <td>165</td>
      <td>165</td>
      <td>40</td>
      <td>40</td>
      <td>125</td>
      <td>125</td>
      <td>7</td>
      <td>7</td>
      <td>494</td>
      <td>494</td>
      <td>94</td>
      <td>94</td>
    </tr>
    <tr>
      <td></td>
      <td>crt</td>
      <td>crispr_recognition_tool</td>
      <td>CRISPR Recognition Tool</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>crispr_recognition_tool</td>
      <td>2015-02-28</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/crt</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/crt</td>
      <td>1.2.0</td>
      <td>crisper_recognition_tool</td>
      <td>1.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>4066</td>
      <td>4139</td>
      <td>166</td>
      <td>180</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4066</td>
      <td>4139</td>
      <td>166</td>
      <td>180</td>
    </tr>
    <tr>
      <td></td>
      <td>ctd_batch</td>
      <td>ctdBatch_1</td>
      <td>CTD analysis of chemicals, diseases, or genes</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>ctd_batch</td>
      <td>2013-09-25</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/ctd_batch</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/ctd_batch</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>196</td>
      <td>211</td>
      <td>41</td>
      <td>48</td>
      <td>1</td>
      <td>447</td>
      <td>1</td>
      <td>161</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>197</td>
      <td>658</td>
      <td>42</td>
      <td>209</td>
    </tr>
    <tr>
      <td></td>
      <td>cummerbund</td>
      <td>cummeRbund</td>
      <td>Wrapper for the Bioconductor cummeRbund library</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/cummeRbund.html</td>
      <td>RNA, Visualization</td>
      <td>cummerbund</td>
      <td>2015-04-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/cummerbund</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/cummerbund</td>
      <td>2.16.0</td>
      <td>fonts-conda-ecosystem</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1251</td>
      <td>1803</td>
      <td>98</td>
      <td>166</td>
      <td>3461</td>
      <td>15159</td>
      <td>486</td>
      <td>2044</td>
      <td>2425</td>
      <td>3338</td>
      <td>57</td>
      <td>159</td>
      <td>7137</td>
      <td>20300</td>
      <td>641</td>
      <td>2369</td>
    </tr>
    <tr>
      <td></td>
      <td>cummerbund_to_tabular</td>
      <td>cummerbund_to_cuffdiff</td>
      <td>Regenerate the tabular files generated by cuffdiff from a cummeRbund SQLite database.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Convert Formats, Next Gen Mappers</td>
      <td>cummerbund_to_tabular</td>
      <td>2014-12-22</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/cummerbund_to_tabular</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/cummerbund_to_tabular</td>
      <td>1.0.1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>888</td>
      <td>1208</td>
      <td>43</td>
      <td>97</td>
      <td>1334</td>
      <td>2683</td>
      <td>101</td>
      <td>479</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2222</td>
      <td>3891</td>
      <td>144</td>
      <td>576</td>
    </tr>
    <tr>
      <td></td>
      <td>cut_columns</td>
      <td>Cut1</td>
      <td>Select columns from a dataset.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>cut_columns</td>
      <td>2012-12-04</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/cut_columns</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/cut_columns</td>
      <td>1.0.2</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1973168</td>
      <td>2034965</td>
      <td>11939</td>
      <td>12833</td>
      <td>352364</td>
      <td>803522</td>
      <td>12967</td>
      <td>30199</td>
      <td>128777</td>
      <td>133584</td>
      <td>2341</td>
      <td>2572</td>
      <td>2454309</td>
      <td>2972071</td>
      <td>27247</td>
      <td>45604</td>
    </tr>
    <tr>
      <td></td>
      <td>delete_overlapping_indels</td>
      <td>delete_overlapping_indels</td>
      <td>Delete Overlapping Indels from a chromosome indels file</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>delete_overlapping_indels</td>
      <td>2013-09-25</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/delete_overlapping_indels</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/delete_overlapping_indels</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>33</td>
      <td>40</td>
      <td>6</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>33</td>
      <td>40</td>
      <td>6</td>
      <td>7</td>
    </tr>
    <tr>
      <td></td>
      <td>dewseq</td>
      <td>dewseq</td>
      <td>DEWSeq is a sliding window based peak caller for eCLIP/iCLIP data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/EMBL-Hentze-group/DEWSeq_analysis_helpers</td>
      <td>Sequence Analysis, RNA, CLIP-seq</td>
      <td>dewseq</td>
      <td>2022-10-20</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/dewseq</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/dewseq</td>
      <td>0.1.0+galaxy0</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>89</td>
      <td>89</td>
      <td>18</td>
      <td>18</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>89</td>
      <td>89</td>
      <td>18</td>
      <td>18</td>
    </tr>
    <tr>
      <td></td>
      <td>dgidb_annotator</td>
      <td>dgidb_annotate</td>
      <td>Annotates a tabular file with information from the Drug-Gene Interaction Database (http://dgidb.genome.wustl.edu/)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Systems Biology, Variant Analysis</td>
      <td>dgidb_annotator</td>
      <td>2013-11-28</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/dgidb_annotator</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/dgidb_annotator</td>
      <td>0.1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>59</td>
      <td>69</td>
      <td>26</td>
      <td>30</td>
      <td>0</td>
      <td>304</td>
      <td>0</td>
      <td>133</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>59</td>
      <td>373</td>
      <td>26</td>
      <td>163</td>
    </tr>
    <tr>
      <td></td>
      <td>diff</td>
      <td>diff</td>
      <td>GNU diff tool that calculates the differences between two files.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.gnu.org/software/diffutils/</td>
      <td>Text Manipulation</td>
      <td>diff</td>
      <td>2020-03-29</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/diff</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/diff</td>
      <td>3.7</td>
      <td>diffutils</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>362</td>
      <td>362</td>
      <td>99</td>
      <td>99</td>
      <td>23</td>
      <td>23</td>
      <td>17</td>
      <td>17</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>385</td>
      <td>385</td>
      <td>116</td>
      <td>116</td>
    </tr>
    <tr>
      <td></td>
      <td>diffbind</td>
      <td></td>
      <td>Diffbind provides functions for  processing  ChIP-Seq  data.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://bioconductor.org/packages/release/bioc/html/DiffBind.html</td>
      <td>ChIP-seq</td>
      <td>diffbind</td>
      <td>2015-02-28</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/diffbind</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/diffbind</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>divide_pg_snp</td>
      <td>dividePgSnp</td>
      <td>Separate pgSnp alleles into columns</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>divide_pg_snp</td>
      <td>2013-09-25</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/divide_pg_snp</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/divide_pg_snp</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>26</td>
      <td>32</td>
      <td>8</td>
      <td>9</td>
      <td>86</td>
      <td>172</td>
      <td>20</td>
      <td>53</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>112</td>
      <td>204</td>
      <td>28</td>
      <td>62</td>
    </tr>
    <tr>
      <td></td>
      <td>dorina</td>
      <td>dorina_search</td>
      <td>data source for RNA interactions in post-transcriptional regulation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA, Data Source</td>
      <td>dorina</td>
      <td>2015-05-20</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/dorina/</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/dorina</td>
      <td>1.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>921</td>
      <td>1626</td>
      <td>4</td>
      <td>31</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>921</td>
      <td>1626</td>
      <td>4</td>
      <td>31</td>
    </tr>
    <tr>
      <td></td>
      <td>dot2ct</td>
      <td>rnastructure_dot2ct</td>
      <td>Dot-Bracket to Connect Table (CT)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, RNA</td>
      <td>dot2ct</td>
      <td>2015-06-10</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/dot2ct</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/dot2ct</td>
      <td>5.7.a</td>
      <td>rnastructure</td>
      <td>6.4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>dotknot</td>
      <td>dotknot</td>
      <td>DotKnot is a heuristic method for pseudoknot prediction in a given RNA sequence</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://dotknot.csse.uwa.edu.au/</td>
      <td>RNA, Proteomics</td>
      <td>dotknot</td>
      <td>2015-09-17</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna/dotknot</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/dotknot</td>
      <td>1.3.1</td>
      <td>vienna_rna</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>74</td>
      <td>88</td>
      <td>12</td>
      <td>15</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>74</td>
      <td>88</td>
      <td>12</td>
      <td>15</td>
    </tr>
    <tr>
      <td></td>
      <td>draw_stacked_barplots</td>
      <td>draw_stacked_barplots</td>
      <td>Draw Stacked Bar Plots for different categories and different criteria</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Graphics, Statistics</td>
      <td>draw_stacked_barplots</td>
      <td>2013-09-25</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/draw_stacked_barplots</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/draw_stacked_barplots</td>
      <td>1.0.0</td>
      <td>R</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>189</td>
      <td>216</td>
      <td>47</td>
      <td>52</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>183</td>
      <td>213</td>
      <td>22</td>
      <td>31</td>
      <td>372</td>
      <td>429</td>
      <td>69</td>
      <td>83</td>
    </tr>
    <tr>
      <td></td>
      <td>dwt_cor_ava_perclass</td>
      <td>compute_p-values_correlation_coefficients_feature_occurrences_between_two_datasets_using_discrete_wavelet_transfom</td>
      <td>Compute P-values and Correlation Coefficients for Feature Occurrences</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>dwt_cor_ava_perclass</td>
      <td>2014-01-27</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/dwt_cor_ava_perclass</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/dwt_cor_ava_perclass</td>
      <td>1.0.1</td>
      <td>r-waveslim</td>
      <td>1.7.5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>136</td>
      <td>141</td>
      <td>8</td>
      <td>11</td>
      <td>136</td>
      <td>141</td>
      <td>8</td>
      <td>11</td>
    </tr>
    <tr>
      <td></td>
      <td>dwt_cor_avb_all</td>
      <td>compute_p-values_correlation_coefficients_featureA_featureB_occurrences_between_two_datasets_using_discrete_wavelet_transfom</td>
      <td>Compute P-values and Correlation Coefficients for Occurrences of Two Set of Features</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>dwt_cor_avb_all</td>
      <td>2014-01-27</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/dwt_cor_avb_all</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/dwt_cor_avb_all</td>
      <td>1.0.1</td>
      <td>r-waveslim</td>
      <td>1.7.5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>124</td>
      <td>124</td>
      <td>7</td>
      <td>7</td>
      <td>124</td>
      <td>124</td>
      <td>7</td>
      <td>7</td>
    </tr>
    <tr>
      <td></td>
      <td>dwt_ivc_all</td>
      <td>compute_p-values_second_moments_feature_occurrences_between_two_datasets_using_discrete_wavelet_transfom</td>
      <td>Compute P-values and Second Moments for Feature Occurrences</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>dwt_ivc_all</td>
      <td>2014-01-27</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/dwt_ivc_all</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/dwt_ivc_all</td>
      <td>1.0.1</td>
      <td>r-waveslim</td>
      <td>1.7.5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>121</td>
      <td>123</td>
      <td>5</td>
      <td>7</td>
      <td>121</td>
      <td>123</td>
      <td>5</td>
      <td>7</td>
    </tr>
    <tr>
      <td></td>
      <td>dwt_var_perclass</td>
      <td>compute_p-values_max_variances_feature_occurrences_in_one_dataset_using_discrete_wavelet_transfom</td>
      <td>Compute P-values and Max Variances for Feature Occurrences</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>dwt_var_perclass</td>
      <td>2014-01-27</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/dwt_var_perclass</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/dwt_var_perclass</td>
      <td>1.0.1</td>
      <td>r-waveslim</td>
      <td>1.7.5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>131</td>
      <td>132</td>
      <td>7</td>
      <td>8</td>
      <td>131</td>
      <td>132</td>
      <td>7</td>
      <td>8</td>
    </tr>
    <tr>
      <td></td>
      <td>dwt_var_perfeature</td>
      <td>dwt_var1</td>
      <td>Wavelet variance using Discrete Wavelet Transfoms</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>dwt_var_perfeature</td>
      <td>2014-01-27</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/dwt_var_perfeature</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/dwt_var_perfeature</td>
      <td>1.0.2</td>
      <td>r-bitops</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>30</td>
      <td>36</td>
      <td>2</td>
      <td>4</td>
      <td>11</td>
      <td>11</td>
      <td>4</td>
      <td>4</td>
      <td>206</td>
      <td>220</td>
      <td>6</td>
      <td>7</td>
      <td>247</td>
      <td>267</td>
      <td>12</td>
      <td>15</td>
    </tr>
    <tr>
      <td></td>
      <td>ear</td>
      <td>make_ear</td>
      <td>A tool to compile assembly reports and stastics from assembly pipeline</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ERGA-consortium/EARs/tree/main</td>
      <td>Sequence Analysis, Assembly</td>
      <td>erga_ear</td>
      <td>2024-06-07</td>
      <td>bgruening</td>
      <td>https://github.com/ERGA-consortium/EARs/tree/main</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/ear</td>
      <td>24.10.15</td>
      <td>reportlab</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>28</td>
      <td>28</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>28</td>
      <td>28</td>
      <td>2</td>
      <td>2</td>
    </tr>
    <tr>
      <td></td>
      <td>edta</td>
      <td>edta</td>
      <td>The EDTA package was designed to filter out false discoveries in raw TE candidates and generate a high-quality non-redundant TE library for whole-genome TE annotations.</td>
      <td>edta</td>
      <td>edta</td>
      <td></td>
      <td>The Extensive de novo TE Annotator (EDTA)</td>
      <td>The EDTA package was designed to filter out false discoveries in raw TE candidates and generate a high-quality non-redundant TE library for whole-genome TE annotations. Selection of initial search programs were based on benckmarkings on the annotation performance using a manually curated TE library in the rice genome.</td>
      <td>De-novo assembly, Deisotoping, Genome annotation</td>
      <td>Workflows, Mobile genetic elements, Plant biology, Transcription factors and regulatory sites, Model organisms</td>
      <td>To update</td>
      <td>https://github.com/oushujun/EDTA</td>
      <td>Variant Analysis</td>
      <td>edta</td>
      <td>2022-10-16</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/edta</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/edta</td>
      <td></td>
      <td>edta</td>
      <td>2.2.0</td>
      <td>De-novo assembly, Deisotoping, Genome annotation</td>
      <td>Workflows, Mobile genetic elements, Plant biology, Transcription factors and regulatory sites, Model organisms</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>4495</td>
      <td>4495</td>
      <td>111</td>
      <td>111</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4495</td>
      <td>4495</td>
      <td>111</td>
      <td>111</td>
    </tr>
    <tr>
      <td></td>
      <td>effectiveT3</td>
      <td>effectiveT3</td>
      <td>Find bacterial type III effectors in protein sequences</td>
      <td>effectivet3</td>
      <td>effectivet3</td>
      <td></td>
      <td>EffectiveT3</td>
      <td>Prediction of putative Type-III secreted proteins.</td>
      <td>Sequence classification</td>
      <td>Sequence analysis</td>
      <td>To update</td>
      <td>http://effectors.org</td>
      <td>Sequence Analysis</td>
      <td>effectivet3</td>
      <td>2015-09-21</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/effectiveT3</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/effectiveT3</td>
      <td>0.0.21</td>
      <td>effectiveT3</td>
      <td>1.0.1</td>
      <td>Sequence classification</td>
      <td>Sequence analysis</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>epicseg</td>
      <td>epicseg_segment</td>
      <td>EpiCSeg is a tool for conducting chromatin segmentation.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/lamortenera/epicseg</td>
      <td>Epigenetics</td>
      <td>epicseg</td>
      <td>2018-03-02</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/epicseg</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/epicseg</td>
      <td>@VERSION_STRING@</td>
      <td>epicseg</td>
      <td>1.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>160</td>
      <td>252</td>
      <td>17</td>
      <td>21</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>160</td>
      <td>252</td>
      <td>17</td>
      <td>21</td>
    </tr>
    <tr>
      <td></td>
      <td>exparna</td>
      <td>exparna</td>
      <td>ExpaRNA is a fast, motif-based comparison and alignment tool for RNA molecules.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://rna.informatik.uni-freiburg.de/ExpaRNA/Input.jsp</td>
      <td>RNA</td>
      <td>exparna</td>
      <td>2015-02-28</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/exparna</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/exparna</td>
      <td>1.0.1</td>
      <td>exparna</td>
      <td>1.0.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>express</td>
      <td>express</td>
      <td>Quantify the abundances of a set of target sequences from sampled subsequences</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>express</td>
      <td>2014-01-27</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/express</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/express</td>
      <td>1.1.1</td>
      <td>eXpress</td>
      <td>1.5.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>278</td>
      <td>337</td>
      <td>34</td>
      <td>45</td>
      <td>0</td>
      <td>1184</td>
      <td>0</td>
      <td>117</td>
      <td>2402</td>
      <td>2844</td>
      <td>23</td>
      <td>60</td>
      <td>2680</td>
      <td>4365</td>
      <td>57</td>
      <td>222</td>
    </tr>
    <tr>
      <td></td>
      <td>fasta_compute_length</td>
      <td>fasta_compute_length</td>
      <td>Compute sequence length</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/fasta_compute_length</td>
      <td>Fasta Manipulation</td>
      <td>fasta_compute_length</td>
      <td>2014-05-19</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/fasta_compute_length</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/fasta_compute_length</td>
      <td>1.0.4</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>8547</td>
      <td>9136</td>
      <td>978</td>
      <td>1129</td>
      <td>17748</td>
      <td>51150</td>
      <td>1404</td>
      <td>5670</td>
      <td>4239</td>
      <td>4404</td>
      <td>162</td>
      <td>190</td>
      <td>30534</td>
      <td>64690</td>
      <td>2544</td>
      <td>6989</td>
    </tr>
    <tr>
      <td></td>
      <td>fasta_concatenate_by_species</td>
      <td>fasta_concatenate0</td>
      <td>Concatenate FASTA alignment by species</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Fasta Manipulation</td>
      <td>fasta_concatenate_by_species</td>
      <td>2014-05-19</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/fasta_concatenate_by_species</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/fasta_concatenate_by_species</td>
      <td>0.0.1</td>
      <td>bx-python</td>
      <td>0.13.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>39695</td>
      <td>39742</td>
      <td>242</td>
      <td>255</td>
      <td>13660</td>
      <td>19007</td>
      <td>600</td>
      <td>1398</td>
      <td>1521</td>
      <td>1603</td>
      <td>92</td>
      <td>112</td>
      <td>54876</td>
      <td>60352</td>
      <td>934</td>
      <td>1765</td>
    </tr>
    <tr>
      <td></td>
      <td>fasta_filter_by_id</td>
      <td>fasta_filter_by_id</td>
      <td>Filter FASTA sequences by ID (DEPRECATED)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/fasta_filter_by_id</td>
      <td>Fasta Manipulation, Sequence Analysis, Text Manipulation</td>
      <td>fasta_filter_by_id</td>
      <td>2017-02-02</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/fasta_filter_by_id</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/fasta_filter_by_id</td>
      <td>0.0.7</td>
      <td>galaxy_sequence_utils</td>
      <td>1.2.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>fasta_filter_by_length</td>
      <td>fasta_filter_by_length</td>
      <td>Filter sequences by length</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Fasta Manipulation</td>
      <td>fasta_filter_by_length</td>
      <td>2014-05-19</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/fasta_filter_by_length</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/fasta_filter_by_length</td>
      <td>1.2</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>47494</td>
      <td>51924</td>
      <td>889</td>
      <td>997</td>
      <td>50663</td>
      <td>101663</td>
      <td>1488</td>
      <td>3742</td>
      <td>3919</td>
      <td>5601</td>
      <td>241</td>
      <td>276</td>
      <td>102076</td>
      <td>159188</td>
      <td>2618</td>
      <td>5015</td>
    </tr>
    <tr>
      <td></td>
      <td>fasta_to_tabular</td>
      <td>fasta2tab</td>
      <td>FASTA-to-Tabular converter</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Fasta Manipulation</td>
      <td>fasta_to_tabular</td>
      <td>2014-05-19</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/fasta_to_tabular</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/fasta_to_tabular</td>
      <td>1.1.1</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>108876</td>
      <td>113638</td>
      <td>1572</td>
      <td>1793</td>
      <td>82527</td>
      <td>192112</td>
      <td>2209</td>
      <td>8565</td>
      <td>5711</td>
      <td>7649</td>
      <td>275</td>
      <td>334</td>
      <td>197114</td>
      <td>313399</td>
      <td>4056</td>
      <td>10692</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_filter_by_id</td>
      <td>fastq_filter_by_id</td>
      <td>Filter FASTQ sequences by ID (DEPRECATED)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/fastq_filter_by_id</td>
      <td>Fastq Manipulation, Sequence Analysis, Text Manipulation</td>
      <td>fastq_filter_by_id</td>
      <td>2017-02-02</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/fastq_filter_by_id</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/fastq_filter_by_id</td>
      <td>0.0.7</td>
      <td>galaxy_sequence_utils</td>
      <td>1.2.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_info</td>
      <td>fastq_info</td>
      <td>FASTQ info allows to validate single or paired fastq files</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/nunofonseca/fastq_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_info</td>
      <td>2021-09-12</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/fastq_info</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/fastq_info</td>
      <td>0.25.1</td>
      <td>fastq_utils</td>
      <td>0.25.2</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>7560</td>
      <td>7560</td>
      <td>1266</td>
      <td>1266</td>
      <td>6539</td>
      <td>6539</td>
      <td>1581</td>
      <td>1581</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>14099</td>
      <td>14099</td>
      <td>2847</td>
      <td>2847</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_pair_names</td>
      <td>fastq_pair_names</td>
      <td>Extract FASTQ paired read names</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/fastq_pair_names</td>
      <td>Sequence Analysis</td>
      <td>fastq_pair_names</td>
      <td>2014-05-05</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/fastq_pair_names</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/fastq_pair_names</td>
      <td>0.0.5</td>
      <td>galaxy_sequence_utils</td>
      <td>1.2.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_paired_unpaired</td>
      <td>fastq_paired_unpaired</td>
      <td>Divide FASTQ file into paired and unpaired reads</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/fastq_paired_unpaired</td>
      <td>Sequence Analysis, Text Manipulation</td>
      <td>fastq_paired_unpaired</td>
      <td>2015-04-28</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/fastq_paired_unpaired</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/fastq_paired_unpaired</td>
      <td>0.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>1.2.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_trimmer_by_quality</td>
      <td>fastq_quality_trimmer</td>
      <td>FASTQ Quality Trimmer by sliding window</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Fastq Manipulation</td>
      <td>fastq_trimmer_by_quality</td>
      <td>2014-01-27</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/fastq_trimmer_by_quality</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/fastq_trimmer_by_quality</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>1.2.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>4881</td>
      <td>6165</td>
      <td>859</td>
      <td>967</td>
      <td>29458</td>
      <td>99179</td>
      <td>3455</td>
      <td>11369</td>
      <td>2171</td>
      <td>2678</td>
      <td>259</td>
      <td>338</td>
      <td>36510</td>
      <td>108022</td>
      <td>4573</td>
      <td>12674</td>
    </tr>
    <tr>
      <td></td>
      <td>fastqsolexa_to_fasta_qual</td>
      <td>fastqsolexa_to_fasta_qual</td>
      <td>FASTQSOLEXA-to-FASTA-QUAL extracts sequences and quality scores from FASTQSOLEXA data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Convert Formats, Fastq Manipulation</td>
      <td>fastqsolexa_to_fasta_qual</td>
      <td>2014-05-19</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/fastqsolexa_to_fasta_qual</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/fastqsolexa_to_fasta_qual</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>0</td>
      <td>3</td>
    </tr>
    <tr>
      <td></td>
      <td>featurecounter</td>
      <td>featureCoverage1</td>
      <td>Feature coverage</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>featurecounter</td>
      <td>2014-04-01</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/featurecounter</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/featurecounter</td>
      <td>2.0.0</td>
      <td>bx-python</td>
      <td>0.13.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>22308</td>
      <td>22388</td>
      <td>16</td>
      <td>19</td>
      <td>350</td>
      <td>13711</td>
      <td>54</td>
      <td>349</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>22658</td>
      <td>36099</td>
      <td>70</td>
      <td>368</td>
    </tr>
    <tr>
      <td></td>
      <td>file_manipulation</td>
      <td>bg_uniq</td>
      <td>This tool returns all unique lines from a tab-separated file.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/file_manipulation</td>
      <td>Text Manipulation</td>
      <td>unique</td>
      <td>2015-02-28</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/file_manipulation</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/file_manipulation</td>
      <td>0.4</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>6690</td>
      <td>8818</td>
      <td>554</td>
      <td>612</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6690</td>
      <td>8818</td>
      <td>554</td>
      <td>612</td>
    </tr>
    <tr>
      <td></td>
      <td>filter_transcripts_via_tracking</td>
      <td>filter_combined_via_tracking</td>
      <td>Filter Combined Transcripts</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>filter_transcripts_via_tracking</td>
      <td>2014-01-27</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/filter_transcripts_via_tracking</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/filter_transcripts_via_tracking</td>
      <td>0.1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>23</td>
      <td>24</td>
      <td>9</td>
      <td>10</td>
      <td>62</td>
      <td>635</td>
      <td>26</td>
      <td>241</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>87</td>
      <td>661</td>
      <td>37</td>
      <td>253</td>
    </tr>
    <tr>
      <td></td>
      <td>find_diag_hits</td>
      <td>find_diag_hits</td>
      <td>Find diagnostic hits</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>Metagenomics</td>
      <td>find_diag_hits</td>
      <td>2014-01-27</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/taxonomy/find_diag_hits</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/taxonomy/find_diag_hits</td>
      <td>1.0.0</td>
      <td>taxonomy</td>
      <td>0.10.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>62</td>
      <td>70</td>
      <td>10</td>
      <td>12</td>
      <td>3</td>
      <td>1787</td>
      <td>1</td>
      <td>204</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>65</td>
      <td>1857</td>
      <td>11</td>
      <td>216</td>
    </tr>
    <tr>
      <td></td>
      <td>find_subsequences</td>
      <td>bg_find_subsequences</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td>find_subsequences</td>
      <td>2015-03-19</td>
      <td>bgruening</td>
      <td></td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/find_subsequences</td>
      <td>0.3</td>
      <td>biopython</td>
      <td>1.70</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>554</td>
      <td>784</td>
      <td>69</td>
      <td>79</td>
      <td>2820</td>
      <td>3047</td>
      <td>323</td>
      <td>361</td>
      <td>20</td>
      <td>20</td>
      <td>1</td>
      <td>1</td>
      <td>3394</td>
      <td>3851</td>
      <td>393</td>
      <td>441</td>
    </tr>
    <tr>
      <td></td>
      <td>flanking_features</td>
      <td>flanking_features_1</td>
      <td>Fetch closest non-overlapping feature for every interval</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/gops</td>
      <td>Genomic Interval Operations</td>
      <td>flanking_features</td>
      <td>2014-04-01</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/gops/flanking_features</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/gops/flanking_features</td>
      <td>4.0.1</td>
      <td>bx-python</td>
      <td>0.13.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>589</td>
      <td>731</td>
      <td>51</td>
      <td>63</td>
      <td>2445</td>
      <td>32847</td>
      <td>304</td>
      <td>2549</td>
      <td>1214</td>
      <td>1294</td>
      <td>15</td>
      <td>22</td>
      <td>4248</td>
      <td>34872</td>
      <td>370</td>
      <td>2634</td>
    </tr>
    <tr>
      <td></td>
      <td>flexynesis</td>
      <td>flexynesis</td>
      <td>This is a deep-learning based multi-omics bulk sequencing data integration suite with a focus on (pre-)clinical endpoint prediction.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/BIMSBbioinfo/flexynesis/tree/main</td>
      <td>Machine Learning, Statistics, Systems Biology</td>
      <td>flexynesis</td>
      <td>2024-08-12</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/flexynesis</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/flexynesis</td>
      <td>0.2.10</td>
      <td>flexynesis</td>
      <td>0.2.10</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>9</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>9</td>
      <td>2</td>
      <td>2</td>
    </tr>
    <tr>
      <td></td>
      <td>flux</td>
      <td>black_forest_labs_flux</td>
      <td>Using the black forest labs FLUX.1 models to generate image based on user input.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/cflux</td>
      <td>Machine Learning</td>
      <td>black_forest_labs_flux</td>
      <td>2024-10-14</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/flux</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/flux</td>
      <td>2024</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>flye</td>
      <td>flye</td>
      <td>Assembly of long and error-prone reads.</td>
      <td>Flye</td>
      <td>Flye</td>
      <td></td>
      <td>Flye</td>
      <td>Flye is a de novo assembler for single molecule sequencing reads, such as those produced by PacBio and Oxford Nanopore Technologies. It is designed for a wide range of datasets, from small bacterial projects to large mammalian-scale assemblies. The package represents a complete pipeline: it takes raw PB / ONT reads as input and outputs polished contigs.</td>
      <td>Genome assembly, De-novo assembly, Mapping assembly, Cross-assembly</td>
      <td>Sequence assembly, Metagenomics, Whole genome sequencing, Genomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/fenderglass/Flye/</td>
      <td>Assembly</td>
      <td>flye</td>
      <td>2018-09-24</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/flye</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/flye</td>
      <td>2.9.5</td>
      <td>flye</td>
      <td>2.9.5</td>
      <td>Genome assembly, De-novo assembly, Mapping assembly, Cross-assembly</td>
      <td>Sequence assembly, Metagenomics, Whole genome sequencing</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>32514</td>
      <td>32675</td>
      <td>3379</td>
      <td>3422</td>
      <td>19434</td>
      <td>19434</td>
      <td>2537</td>
      <td>2537</td>
      <td>9843</td>
      <td>9843</td>
      <td>1624</td>
      <td>1624</td>
      <td>61791</td>
      <td>61952</td>
      <td>7540</td>
      <td>7583</td>
    </tr>
    <tr>
      <td></td>
      <td>footprint</td>
      <td>footprint</td>
      <td>Find transcription factor footprints</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://ohlerlab.mdc-berlin.de/software/Reproducible_footprinting_139/</td>
      <td>Epigenetics</td>
      <td>footprint</td>
      <td>2017-04-18</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/footprint</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/footprint</td>
      <td>1.0.0</td>
      <td>footprint</td>
      <td>1.0.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>57</td>
      <td>57</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>57</td>
      <td>57</td>
      <td>5</td>
      <td>5</td>
    </tr>
    <tr>
      <td></td>
      <td>format_cd_hit_output</td>
      <td>format_cd_hit_output</td>
      <td>Format CD-hit output to rename representative sequences with cluster name and/or extract distribution inside clusters given a mapping file</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Fasta Manipulation</td>
      <td>format_cd_hit_output</td>
      <td>2022-10-19</td>
      <td>bebatut</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/format_cd_hit_output/</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/format_cd_hit_output</td>
      <td>1.0.0+galaxy1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>189</td>
      <td>203</td>
      <td>57</td>
      <td>60</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>189</td>
      <td>203</td>
      <td>57</td>
      <td>60</td>
    </tr>
    <tr>
      <td></td>
      <td>format_metaphlan2_output</td>
      <td>format_metaphlan2_output</td>
      <td>Format MetaPhlAn2 output to extract abundance at different taxonomic levels</td>
      <td>format_metaphlan2_output</td>
      <td>format_metaphlan2_output</td>
      <td></td>
      <td>Format metaphlan2 output</td>
      <td>This tool format output file of MetaPhlan2 containing community content (abundance) at all taxonomic levels (from kingdom to strains).</td>
      <td>Formatting</td>
      <td>Taxonomy, Metagenomics</td>
      <td>To update</td>
      <td></td>
      <td>Metagenomics</td>
      <td>format_metaphlan2_output</td>
      <td>2022-10-19</td>
      <td>bebatut</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/format_metaphlan2_output/</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/format_metaphlan2_output</td>
      <td>0.2.0</td>
      <td></td>
      <td></td>
      <td>Formatting</td>
      <td>Taxonomy, Metagenomics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>5940</td>
      <td>6474</td>
      <td>558</td>
      <td>608</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5940</td>
      <td>6474</td>
      <td>558</td>
      <td>608</td>
    </tr>
    <tr>
      <td></td>
      <td>generate_pc_lda_matrix</td>
      <td>generate_matrix_for_pca_and_lda1</td>
      <td>Generate a Matrix for using PC and LDA</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>generate_pc_lda_matrix</td>
      <td>2014-07-28</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/generate_pc_lda_matrix</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/generate_pc_lda_matrix</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>115</td>
      <td>124</td>
      <td>29</td>
      <td>30</td>
      <td>518</td>
      <td>1029</td>
      <td>102</td>
      <td>261</td>
      <td>184</td>
      <td>198</td>
      <td>14</td>
      <td>17</td>
      <td>817</td>
      <td>1351</td>
      <td>145</td>
      <td>308</td>
    </tr>
    <tr>
      <td></td>
      <td>get_flanks</td>
      <td>get_flanks1</td>
      <td>Get flanks returns flanking region/s for every gene</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/gops</td>
      <td>Genomic Interval Operations</td>
      <td>get_flanks</td>
      <td>2014-04-01</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/gops/get_flanks</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/gops/get_flanks</td>
      <td>1.0.0</td>
      <td>bx-python</td>
      <td>0.13.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>3768</td>
      <td>325011</td>
      <td>1462</td>
      <td>2044</td>
      <td>9350</td>
      <td>242043</td>
      <td>2201</td>
      <td>6583</td>
      <td>787</td>
      <td>1070</td>
      <td>119</td>
      <td>146</td>
      <td>13905</td>
      <td>568124</td>
      <td>3782</td>
      <td>8773</td>
    </tr>
    <tr>
      <td></td>
      <td>get_orfs_or_cdss</td>
      <td>get_orfs_or_cdss</td>
      <td>Search nucleotide sequences for open reading frames (ORFs), or coding sequences (CDSs)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/get_orfs_or_cdss</td>
      <td>Sequence Analysis</td>
      <td>get_orfs_or_cdss</td>
      <td>2015-04-29</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/get_orfs_or_cdss</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/get_orfs_or_cdss</td>
      <td>0.2.3</td>
      <td>biopython</td>
      <td>1.70</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>746</td>
      <td>746</td>
      <td>136</td>
      <td>136</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1504</td>
      <td>1504</td>
      <td>244</td>
      <td>244</td>
      <td>2250</td>
      <td>2250</td>
      <td>380</td>
      <td>380</td>
    </tr>
    <tr>
      <td></td>
      <td>getindelrates_3way</td>
      <td>indelRates_3way</td>
      <td>Estimate Indel Rates for 3-way alignments</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>getindelrates_3way</td>
      <td>2014-04-01</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/getindelrates_3way</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/getindelrates_3way</td>
      <td>1.0.0</td>
      <td>bx-python</td>
      <td>0.13.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>70</td>
      <td>290</td>
      <td>20</td>
      <td>86</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>70</td>
      <td>290</td>
      <td>20</td>
      <td>86</td>
    </tr>
    <tr>
      <td></td>
      <td>getindels_2way</td>
      <td>getIndels_2way</td>
      <td>Fetch Indels from pairwise alignments</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>getindels_2way</td>
      <td>2014-04-01</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/getindels_2way</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/getindels_2way</td>
      <td>1.0.0</td>
      <td>numpy</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>74</td>
      <td>237</td>
      <td>24</td>
      <td>61</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>74</td>
      <td>237</td>
      <td>24</td>
      <td>61</td>
    </tr>
    <tr>
      <td></td>
      <td>gfastats</td>
      <td>gfastats</td>
      <td>Tool for generating sequence statistics and simultaneous genome assembly file manipulation.</td>
      <td>gfastats</td>
      <td>gfastats</td>
      <td></td>
      <td>gfastats</td>
      <td>gfastats is a single fast and exhaustive tool for summary statistics and simultaneous genome assembly file manipulation. gfastats also allows seamless fasta/fastq/gfa conversion.</td>
      <td>Data handling</td>
      <td>Computational biology</td>
      <td>Up-to-date</td>
      <td>https://github.com/vgl-hub/gfastats</td>
      <td>Sequence Analysis</td>
      <td>gfastats</td>
      <td>2022-03-08</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/gfastats</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/gfastats</td>
      <td>1.3.8</td>
      <td>gfastats</td>
      <td>1.3.8</td>
      <td>Data handling</td>
      <td>Computational biology</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>11748</td>
      <td>11748</td>
      <td>767</td>
      <td>767</td>
      <td>6560</td>
      <td>6560</td>
      <td>397</td>
      <td>397</td>
      <td>2556</td>
      <td>2556</td>
      <td>106</td>
      <td>106</td>
      <td>20864</td>
      <td>20864</td>
      <td>1270</td>
      <td>1270</td>
    </tr>
    <tr>
      <td></td>
      <td>gi2taxonomy</td>
      <td>Fetch Taxonomic Ranks</td>
      <td>Fetch taxonomic representation</td>
      <td>gi2taxonomy</td>
      <td>gi2taxonomy</td>
      <td></td>
      <td>gi2taxonomy</td>
      <td>The tool fetches taxonomic information for a list of sequence identifiers (i.e. GI numbers, as used by the National Center for Biotechnology Information (NCBI).</td>
      <td>Database search, ID mapping</td>
      <td>Taxonomy</td>
      <td>To update</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>Metagenomics</td>
      <td>gi2taxonomy</td>
      <td>2014-01-27</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/taxonomy/gi2taxonomy</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/taxonomy/gi2taxonomy</td>
      <td>1.1.1</td>
      <td>taxonomy</td>
      <td>0.10.0</td>
      <td>Database search, ID mapping</td>
      <td>Taxonomy</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>490</td>
      <td>682</td>
      <td>74</td>
      <td>109</td>
      <td>127</td>
      <td>14058</td>
      <td>20</td>
      <td>1798</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>617</td>
      <td>14740</td>
      <td>94</td>
      <td>1907</td>
    </tr>
    <tr>
      <td></td>
      <td>glimmer_hmm</td>
      <td></td>
      <td>GlimmerHMM is a new gene finder based on a Generalized Hidden Markov Model (GHMM)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://ccb.jhu.edu/software/glimmerhmm/</td>
      <td>Sequence Analysis</td>
      <td>glimmer_hmm</td>
      <td>2015-02-28</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/glimmer_hmm</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/glimmer_hmm</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>gmaj</td>
      <td>gmaj_1</td>
      <td>GMAJ Multiple Alignment Viewer</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Visualization</td>
      <td>gmaj</td>
      <td>2014-01-27</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/gmaj</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/gmaj</td>
      <td>2.0.1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>11</td>
      <td>11</td>
      <td>6</td>
      <td>6</td>
      <td>85</td>
      <td>984</td>
      <td>29</td>
      <td>230</td>
      <td>10</td>
      <td>11</td>
      <td>7</td>
      <td>8</td>
      <td>106</td>
      <td>1006</td>
      <td>42</td>
      <td>244</td>
    </tr>
    <tr>
      <td></td>
      <td>gotohscan</td>
      <td>rbc_gotohscan</td>
      <td>Find subsequences in db</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>gotohscan</td>
      <td>2015-07-02</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/gotohscan</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/gotohscan</td>
      <td>1.3.0</td>
      <td>gotohscan</td>
      <td>1.3</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>34</td>
      <td>72</td>
      <td>5</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>34</td>
      <td>72</td>
      <td>5</td>
      <td>7</td>
    </tr>
    <tr>
      <td></td>
      <td>graph_converter</td>
      <td>graph_converter</td>
      <td>Convert between different graph formats</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Convert Formats</td>
      <td>graph_converter</td>
      <td>2015-02-28</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/graph_converter</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/graph_converter</td>
      <td>0.1.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>graphclust</td>
      <td>graphclust</td>
      <td>GraphClust can be used for structural clustering of RNA sequences.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.bioinf.uni-freiburg.de/Software/GraphClust/</td>
      <td>RNA</td>
      <td>graphclust</td>
      <td>2015-02-28</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/graphclust</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/graphclust</td>
      <td>0.1</td>
      <td>GraphClust</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>0</td>
      <td>1</td>
    </tr>
    <tr>
      <td></td>
      <td>graphicsmagick</td>
      <td>graphicsmagick_image_compare, graphicsmagick_image_convert, graphicsmagick_image_montage</td>
      <td>Contains tools based on GraphicsMagick</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.graphicsmagick.org</td>
      <td>Imaging</td>
      <td>graphicsmagick</td>
      <td>2016-12-26</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/new/gm/tools/image_processing/image_processing/</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/image_processing/graphicsmagick</td>
      <td>1.3.45</td>
      <td>graphicsmagick</td>
      <td>1.3.26</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>1725</td>
      <td>2708</td>
      <td>97</td>
      <td>99</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1727</td>
      <td>2710</td>
      <td>99</td>
      <td>101</td>
    </tr>
    <tr>
      <td></td>
      <td>graphmap</td>
      <td>graphmap_align, graphmap_overlap</td>
      <td>Mapper for long, error-prone reads.</td>
      <td>graphmap</td>
      <td>graphmap</td>
      <td></td>
      <td>graphmap</td>
      <td>Splice-aware RNA-seq mapper for long reads | GraphMap - A highly sensitive and accurate mapper for long, error-prone reads http://www.nature.com/ncomms/2016/160415/ncomms11307/full/ncomms11307.html https://www.biorxiv.org/content/10.1101/720458v1</td>
      <td>Sequence trimming, EST assembly, Read mapping</td>
      <td>Gene transcripts, RNA-Seq, RNA splicing</td>
      <td>To update</td>
      <td>https://github.com/isovic/graphmap/</td>
      <td>Assembly</td>
      <td>graphmap</td>
      <td>2018-09-27</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/graphmap</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/graphmap</td>
      <td>0.5.2</td>
      <td>graphmap</td>
      <td>0.6.4</td>
      <td>Sequence trimming, EST assembly, Read mapping</td>
      <td>Gene transcripts, RNA-Seq, RNA splicing</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>7012</td>
      <td>7344</td>
      <td>323</td>
      <td>338</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7012</td>
      <td>7344</td>
      <td>323</td>
      <td>338</td>
    </tr>
    <tr>
      <td></td>
      <td>graphprot</td>
      <td>graphprot_predict_profile</td>
      <td>GraphProt models binding preferences of RNA-binding proteins.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/dmaticzka/GraphProt</td>
      <td>Sequence Analysis, RNA, CLIP-seq</td>
      <td>graphprot</td>
      <td>2018-05-25</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/graphprot</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/graphprot</td>
      <td>1.1.7+galaxy2</td>
      <td>graphprot</td>
      <td>1.1.7</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>463</td>
      <td>484</td>
      <td>18</td>
      <td>19</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>463</td>
      <td>484</td>
      <td>18</td>
      <td>19</td>
    </tr>
    <tr>
      <td></td>
      <td>hclust2</td>
      <td>hclust2</td>
      <td>Plots heatmaps</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/nsegata/hclust2/</td>
      <td>Data Visualization</td>
      <td>hclust2</td>
      <td>2018-07-22</td>
      <td>rnateam</td>
      <td>https://github.com/yuanbit/galaxytools/tree/hclust2/tools/hclust2</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/hclust2</td>
      <td>0.99</td>
      <td>hclust2</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>hgv_fundo</td>
      <td>hgv_funDo</td>
      <td>FunDO human genes associated with disease terms</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>hgv_fundo</td>
      <td>2013-09-25</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/hgv/hgv_fundo</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/hgv/hgv_fundo</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>145</td>
      <td>1794</td>
      <td>33</td>
      <td>397</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>145</td>
      <td>1794</td>
      <td>33</td>
      <td>397</td>
    </tr>
    <tr>
      <td></td>
      <td>hgv_hilbertvis</td>
      <td>hgv_hilbertvis</td>
      <td>HVIS visualization of genomic data with the Hilbert curve</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.ebi.ac.uk/huber-srv/hilbert/</td>
      <td>Graphics, Visualization</td>
      <td>hgv_hilbertvis</td>
      <td>2013-09-25</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/hgv/hgv_hilbertvis</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/hgv/hgv_hilbertvis</td>
      <td>1.0.0</td>
      <td>R</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>119</td>
      <td>265</td>
      <td>16</td>
      <td>60</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>119</td>
      <td>265</td>
      <td>16</td>
      <td>60</td>
    </tr>
    <tr>
      <td></td>
      <td>hictk</td>
      <td>hictk</td>
      <td>Convert cooler to juicebox_hic</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/paulsengroup/hictk</td>
      <td>Convert Formats, Epigenetics</td>
      <td>hictk</td>
      <td>2024-02-03</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/hictk</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/hictk</td>
      <td>1.0.0</td>
      <td>hictk</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>17</td>
      <td>17</td>
      <td>9</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>17</td>
      <td>17</td>
      <td>9</td>
      <td>9</td>
    </tr>
    <tr>
      <td></td>
      <td>hicup</td>
      <td>hicup2juicer, hicup_deduplicator, hicup_digester, hicup_filter, hicup_hicup, hicup_mapper, hicup_truncater</td>
      <td>The HiCUP-Pipeline from the Bioinformatics Babraham Institute.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.bioinformatics.babraham.ac.uk/projects/hicup/read_the_docs/html/index.html</td>
      <td>Epigenetics</td>
      <td>hicup</td>
      <td>2017-03-09</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/hicup</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/hicup</td>
      <td>0.9.2</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>7</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>6</td>
      <td>0</td>
      <td>901</td>
      <td>1020</td>
      <td>160</td>
      <td>184</td>
      <td>440</td>
      <td>440</td>
      <td>111</td>
      <td>111</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1341</td>
      <td>1460</td>
      <td>271</td>
      <td>295</td>
    </tr>
    <tr>
      <td></td>
      <td>hisat</td>
      <td>hisat</td>
      <td>HISAT is a fast and sensitive spliced alignment program.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://ccb.jhu.edu/software/hisat/index.shtml</td>
      <td>Assembly</td>
      <td>hisat</td>
      <td>2015-05-13</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/hisat</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/hisat</td>
      <td>1.0.3</td>
      <td>hisat</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>228</td>
      <td>0</td>
      <td>12</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>228</td>
      <td>0</td>
      <td>12</td>
    </tr>
    <tr>
      <td></td>
      <td>histogram</td>
      <td>histogram_rpy</td>
      <td>Histogram of a numeric column</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Graphics, Statistics</td>
      <td>histogram</td>
      <td>2014-07-28</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/histogram</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/histogram</td>
      <td>1.0.4</td>
      <td>rpy2</td>
      <td>2.7.8</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>3748</td>
      <td>4661</td>
      <td>866</td>
      <td>1002</td>
      <td>5993</td>
      <td>48272</td>
      <td>1507</td>
      <td>7101</td>
      <td>549</td>
      <td>746</td>
      <td>152</td>
      <td>253</td>
      <td>10290</td>
      <td>53679</td>
      <td>2525</td>
      <td>8356</td>
    </tr>
    <tr>
      <td></td>
      <td>homer</td>
      <td></td>
      <td>Software for motif discovery and next generation sequencing analysis.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://homer.salk.edu/homer/</td>
      <td>Sequence Analysis</td>
      <td>homer</td>
      <td>2015-02-28</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/homer</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/homer</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>htseq-clip</td>
      <td>htseq_clip</td>
      <td>htseq-clip is a toolset for the analysis of eCLIP/iCLIP datasets</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/EMBL-Hentze-group/htseq-clip</td>
      <td>Sequence Analysis, RNA, CLIP-seq</td>
      <td>htseq_clip</td>
      <td>2022-10-11</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/htseq-clip</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/htseq-clip</td>
      <td>0.1.0+galaxy0</td>
      <td>htseq-clip</td>
      <td>2.19.0b0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>98</td>
      <td>98</td>
      <td>19</td>
      <td>19</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>98</td>
      <td>98</td>
      <td>19</td>
      <td>19</td>
    </tr>
    <tr>
      <td></td>
      <td>illumina_methylation_analyser</td>
      <td>illumina_methylation_analyser</td>
      <td>Methylation analyzer for Illumina 450k DNA emthylation microarrays</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/illumina_methylation_analyser</td>
      <td>Sequence Analysis</td>
      <td>illumina_methylation_analyser</td>
      <td>2015-02-28</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/illumina_methylation_analyser</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/illumina_methylation_analyser</td>
      <td>0.1</td>
      <td>Rscript</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>imagej2</td>
      <td>imagej2_adjust_threshold_binary, imagej2_analyze_particles_binary, imagej2_analyze_skeleton, imagej2_binary_to_edm, imagej2_bunwarpj_adapt_transform, imagej2_bunwarpj_align, imagej2_bunwarpj_compare_elastic, imagej2_bunwarpj_compare_elastic_raw, imagej2_bunwarpj_compare_raw, imagej2_bunwarpj_compose_elastic, imagej2_bunwarpj_compose_raw, imagej2_bunwarpj_compose_raw_elastic, imagej2_bunwarpj_convert_to_raw, imagej2_bunwarpj_elastic_transform, imagej2_bunwarpj_raw_transform, imagej2_create_image, imagej2_crop, imagej2_enhance_contrast, imagej2_filter, imagej2_find_edges, imagej2_find_maxima, imagej2_make_binary, imagej2_math, imagej2_noise, imagej2_shadows, imagej2_sharpen, imagej2_skeletonize3d, imagej2_smooth, imagej2_watershed_binary</td>
      <td>ImageJ2 is a new version of ImageJ for the next generation of multidimensionalimage data, with a focus on scientific imaging.</td>
      <td>imagej</td>
      <td>imagej</td>
      <td>imagej2</td>
      <td>ImageJ2</td>
      <td>It is a public domain Java image processing program, which was designed with an open architecture. Custom acquisition, analysis and processing plugins can be developed using ImageJs built-in editor and a Java compiler. User-written plugins make it possible to solve many image processing and analysis problems, from three-dimensional live-cell imaging, to radiological image processing, multiple imaging system data comparisons to automated hematology systems.</td>
      <td>Image analysis, Image annotation, Visualisation</td>
      <td>Imaging</td>
      <td>To update</td>
      <td>http://fiji.sc</td>
      <td>Imaging</td>
      <td>imagej2</td>
      <td>2020-09-09</td>
      <td>imgteam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/image_processing/imagej2</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/image_processing/imagej2</td>
      <td>20240614</td>
      <td></td>
      <td></td>
      <td>Image analysis, Image annotation, Visualisation</td>
      <td>Imaging</td>
      <td>0</td>
      <td>0</td>
      <td>27</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>27</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>27</td>
      <td>27</td>
      <td>0</td>
      <td>1731</td>
      <td>1731</td>
      <td>109</td>
      <td>109</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1731</td>
      <td>1731</td>
      <td>109</td>
      <td>109</td>
    </tr>
    <tr>
      <td></td>
      <td>improviser</td>
      <td>proteomics_improviser</td>
      <td>Visualisation of PepXML files</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.improviser.uni-freiburg.de/</td>
      <td>Proteomics</td>
      <td>proteomics_improviser</td>
      <td>2015-02-28</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/proteomics/improviser</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/proteomics/improviser</td>
      <td>1.1.0.1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>indels_3way</td>
      <td>indels_3way</td>
      <td>Fetch Indels from 3-way alignments</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>indels_3way</td>
      <td>2013-09-25</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/indels_3way</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/indels_3way</td>
      <td>1.0.3</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>18</td>
      <td>22</td>
      <td>1</td>
      <td>1</td>
      <td>44</td>
      <td>292</td>
      <td>3</td>
      <td>21</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>62</td>
      <td>314</td>
      <td>4</td>
      <td>22</td>
    </tr>
    <tr>
      <td></td>
      <td>infernal</td>
      <td>infernal_cmalign, infernal_cmbuild, infernal_cmpress, infernal_cmscan, infernal_cmsearch, infernal_cmstat</td>
      <td>Infernal ("INFERence of RNA ALignment") is for searching DNA sequence databases for RNA structure and sequence similarities.</td>
      <td>infernal</td>
      <td>infernal</td>
      <td></td>
      <td>Infernal</td>
      <td>Infernal ("INFERence of RNA ALignment") is for searching DNA sequence databases for RNA structure and sequence similarities. It is an implementation of a special case of profile stochastic context-free grammars called covariance models (CMs). A CM is like a sequence profile, but it scores a combination of sequence consensus and RNA secondary structure consensus, so in many cases, it is more capable of identifying RNA homologs that conserve their secondary structure more than their primary sequence.</td>
      <td>Nucleic acid feature detection</td>
      <td>Sequence sites, features and motifs, Structural genomics</td>
      <td>To update</td>
      <td>http://infernal.janelia.org/</td>
      <td>RNA</td>
      <td>infernal</td>
      <td>2015-05-07</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/infernal</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/infernal</td>
      <td>1.1.4</td>
      <td>infernal</td>
      <td>1.1.5</td>
      <td>Nucleic acid feature detection</td>
      <td>Sequence sites, features and motifs, Structural genomics</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>59700</td>
      <td>112582</td>
      <td>223</td>
      <td>282</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1157</td>
      <td>1221</td>
      <td>34</td>
      <td>35</td>
      <td>60857</td>
      <td>113803</td>
      <td>257</td>
      <td>317</td>
    </tr>
    <tr>
      <td></td>
      <td>inforna</td>
      <td></td>
      <td>INFO-RNA is a service for the design of RNA sequences that fold into a given pseudo-knot free RNA secondary structure.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://rna.informatik.uni-freiburg.de/INFORNA/Input.jsp</td>
      <td>RNA</td>
      <td>inforna</td>
      <td>2015-02-28</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/inforna</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/inforna</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>instagraal</td>
      <td>instagraal</td>
      <td>Large genome reassembly based on Hi-C data</td>
      <td>instagraal</td>
      <td>instagraal</td>
      <td></td>
      <td>instaGRAAL</td>
      <td>Chromosome-level quality scaffolding of brown algal genomes using InstaGRAAL.Large genome reassembly based on Hi-C data, continuation of GRAAL.Large genome reassembly based on Hi-C data (continuation and partial rewrite of GRAAL) and post-scaffolding polishing libraries.This work is under continuous development/improvement - see GRAAL for information about the basic principles.sudo pip3 install -e git+https://github.com/koszullab/instagraal.git@master#egg=instagraal.Note to OS X users: There is currently no CUDA support on Mojave (10.14) and it is unclear when it is going to be added, if it is to be added at all. This means instaGRAAL (or indeed any CUDA-based application) will not work on Mojave. If you wish to run it on OS X, the only solution for now is to downgrade to High Sierra (10.13)</td>
      <td>Genome assembly, Mapping assembly, Genetic mapping, Scaffolding</td>
      <td>Sequence assembly, Mapping, Metagenomics, Statistics and probability, DNA binding sites</td>
      <td>To update</td>
      <td>https://github.com/koszullab/instaGRAAL</td>
      <td>Assembly</td>
      <td>instagraal</td>
      <td>2022-11-10</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/instagraal</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/instagraal</td>
      <td>0.1.6</td>
      <td></td>
      <td></td>
      <td>Genome assembly, Mapping assembly, Genetic mapping, Scaffolding</td>
      <td>Sequence assembly, Mapping, Metagenomics, Statistics and probability, DNA binding sites</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>157</td>
      <td>157</td>
      <td>14</td>
      <td>14</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>157</td>
      <td>157</td>
      <td>14</td>
      <td>14</td>
    </tr>
    <tr>
      <td></td>
      <td>intarna</td>
      <td>intarna</td>
      <td>Efficient RNA-RNA interaction prediction incorporating accessibility and seeding of interaction sites.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/BackofenLab/IntaRNA</td>
      <td>RNA</td>
      <td>intarna</td>
      <td>2019-01-10</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/intarna</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/intarna</td>
      <td>3.4.1</td>
      <td>intarna</td>
      <td>3.4.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>3071</td>
      <td>7652</td>
      <td>41</td>
      <td>47</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3071</td>
      <td>7652</td>
      <td>41</td>
      <td>47</td>
    </tr>
    <tr>
      <td></td>
      <td>intersect</td>
      <td>gops_intersect_1</td>
      <td>Intersect the intervals of two datasets</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/gops</td>
      <td>Genomic Interval Operations</td>
      <td>intersect</td>
      <td>2014-04-01</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/gops/intersect</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/gops/intersect</td>
      <td>1.0.0</td>
      <td>bx-python</td>
      <td>0.13.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>20472</td>
      <td>349984</td>
      <td>2351</td>
      <td>3007</td>
      <td>45587</td>
      <td>778988</td>
      <td>4359</td>
      <td>14116</td>
      <td>6087</td>
      <td>7417</td>
      <td>484</td>
      <td>561</td>
      <td>72146</td>
      <td>1136389</td>
      <td>7194</td>
      <td>17684</td>
    </tr>
    <tr>
      <td></td>
      <td>iprscan5</td>
      <td></td>
      <td>Interproscan queries the interpro database and provides annotations.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.ebi.ac.uk/Tools/pfa/iprscan5/</td>
      <td>Sequence Analysis</td>
      <td>iprscan5</td>
      <td>2015-02-28</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/iprscan5</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/iprscan5</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>itsx</td>
      <td>itsx</td>
      <td>ITSx is an open source software utility to extract the highly variable ITS1 and ITS2 subregions from ITS sequences.</td>
      <td>ITSx</td>
      <td>ITSx</td>
      <td></td>
      <td>ITSx</td>
      <td>TSx is an open source software utility to extract the highly variable ITS1 and ITS2 subregions from ITS sequences, which is commonly used as a molecular barcode for e.g. fungi. As the inclusion of parts of the neighbouring, very conserved, ribosomal genes (SSU, 5S and LSU rRNA sequences) in the sequence identification process can lead to severely misleading results, ITSx identifies and extracts only the ITS regions themselves.</td>
      <td>Sequence feature detection</td>
      <td>Functional, regulatory and non-coding RNA, Microbiology</td>
      <td>Up-to-date</td>
      <td>https://microbiology.se/software/itsx/</td>
      <td>Metagenomics</td>
      <td>itsx</td>
      <td>2022-05-02</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/itsx</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/itsx</td>
      <td>1.1.3</td>
      <td>itsx</td>
      <td>1.1.3</td>
      <td>Sequence feature detection</td>
      <td>Functional, regulatory and non-coding RNA, Microbiology</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1426</td>
      <td>1426</td>
      <td>71</td>
      <td>71</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1426</td>
      <td>1426</td>
      <td>71</td>
      <td>71</td>
    </tr>
    <tr>
      <td></td>
      <td>jbrowse2</td>
      <td>jbrowse2</td>
      <td>JBrowse2 Genome Browser integrated as a Galaxy Tool</td>
      <td>jbrowse_2</td>
      <td>jbrowse_2</td>
      <td></td>
      <td>JBrowse 2</td>
      <td>Modular genome browser with views of synteny and structural variation.</td>
      <td>Genome visualisation, Structure visualisation, Pathway visualisation</td>
      <td>Mapping, Structural variation, Genomics, Gene structure, Sequence assembly</td>
      <td>Up-to-date</td>
      <td>https://jbrowse.org</td>
      <td>Sequence Analysis</td>
      <td>jbrowse2</td>
      <td>2024-06-04</td>
      <td>fubar</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/jbrowse2</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/jbrowse2</td>
      <td>2.15.4</td>
      <td>jbrowse2</td>
      <td>2.15.4</td>
      <td>Genome visualisation, Structure visualisation, Pathway visualisation</td>
      <td>Mapping, Structural variation, Genomics, Gene structure, Sequence assembly</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1312</td>
      <td>1312</td>
      <td>219</td>
      <td>219</td>
      <td>116</td>
      <td>116</td>
      <td>58</td>
      <td>58</td>
      <td>37</td>
      <td>37</td>
      <td>6</td>
      <td>6</td>
      <td>1465</td>
      <td>1465</td>
      <td>283</td>
      <td>283</td>
    </tr>
    <tr>
      <td></td>
      <td>join</td>
      <td>gops_join_1</td>
      <td>Join the intervals of two datasets side-by-side</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/gops</td>
      <td>Genomic Interval Operations</td>
      <td>join</td>
      <td>2014-04-01</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/gops/join</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/gops/join</td>
      <td>1.0.0</td>
      <td>bx-python</td>
      <td>0.13.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>15624</td>
      <td>330820</td>
      <td>1864</td>
      <td>2438</td>
      <td>52317</td>
      <td>513688</td>
      <td>10198</td>
      <td>39421</td>
      <td>1329</td>
      <td>1645</td>
      <td>213</td>
      <td>289</td>
      <td>69270</td>
      <td>846153</td>
      <td>12275</td>
      <td>42148</td>
    </tr>
    <tr>
      <td></td>
      <td>join_files_on_column_fuzzy</td>
      <td>join_files_on_column_fuzzy</td>
      <td>Join two files on a common column, allowing a certain difference.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>join_files_on_column_fuzzy</td>
      <td>2017-11-26</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/text_processing/join_files_on_column_fuzzy</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/text_processing/join_files_on_column_fuzzy</td>
      <td>1.0.1</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1980</td>
      <td>2697</td>
      <td>345</td>
      <td>410</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1081</td>
      <td>1081</td>
      <td>44</td>
      <td>44</td>
      <td>3061</td>
      <td>3778</td>
      <td>389</td>
      <td>454</td>
    </tr>
    <tr>
      <td></td>
      <td>jupyter_job</td>
      <td>run_jupyter_job</td>
      <td>Run jupyter notebook script in Galaxy</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Machine Learning</td>
      <td>run_jupyter_job</td>
      <td>2021-12-11</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/jupyter_job</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/jupyter_job</td>
      <td>0.0.1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>30</td>
      <td>30</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>30</td>
      <td>30</td>
      <td>2</td>
      <td>2</td>
    </tr>
    <tr>
      <td></td>
      <td>kernel_canonical_correlation_analysis</td>
      <td>kcca1</td>
      <td>Kernel Canonical Correlation Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>kernel_canonical_correlation_analysis</td>
      <td>2014-05-19</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/kernel_canonical_correlation_analysis</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/kernel_canonical_correlation_analysis</td>
      <td>1.0.0</td>
      <td>rpy</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>62</td>
      <td>70</td>
      <td>4</td>
      <td>4</td>
      <td>207</td>
      <td>334</td>
      <td>27</td>
      <td>71</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>269</td>
      <td>404</td>
      <td>31</td>
      <td>75</td>
    </tr>
    <tr>
      <td></td>
      <td>kernel_principal_component_analysis</td>
      <td>kpca1</td>
      <td>Kernel Principal Component Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>kernel_principal_component_analysis</td>
      <td>2014-05-19</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/kernel_principal_component_analysis</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/kernel_principal_component_analysis</td>
      <td>1.0.0</td>
      <td>rpy</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>144</td>
      <td>157</td>
      <td>26</td>
      <td>30</td>
      <td>268</td>
      <td>707</td>
      <td>83</td>
      <td>248</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>412</td>
      <td>864</td>
      <td>109</td>
      <td>278</td>
    </tr>
    <tr>
      <td></td>
      <td>kinwalker</td>
      <td></td>
      <td>Kinwalker splits the folding process into a series of events where each event can either be a folding event or a transcription event.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.bioinf.uni-leipzig.de/Software/Kinwalker/</td>
      <td>RNA</td>
      <td>kinwalker</td>
      <td>2015-02-28</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/kinwalker</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/kinwalker</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>kraken2tax</td>
      <td>Kraken2Tax</td>
      <td>Convert Kraken output to Galaxy taxonomy data.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>Metagenomics</td>
      <td>kraken2tax</td>
      <td>2015-08-05</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/blob/master/tool_collections/taxonomy/kraken2tax/</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/taxonomy/kraken2tax</td>
      <td>1.2+galaxy0</td>
      <td>gawk</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>23008</td>
      <td>23014</td>
      <td>1060</td>
      <td>1061</td>
      <td>16194</td>
      <td>24490</td>
      <td>1719</td>
      <td>2168</td>
      <td>14898</td>
      <td>14898</td>
      <td>436</td>
      <td>436</td>
      <td>54100</td>
      <td>62402</td>
      <td>3215</td>
      <td>3665</td>
    </tr>
    <tr>
      <td></td>
      <td>labels</td>
      <td>bg_labels</td>
      <td>remaps and annotates alignments</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/labels</td>
      <td>Sequence Analysis</td>
      <td>labels</td>
      <td>2015-02-28</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/labels</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/labels</td>
      <td>1.0.5.0</td>
      <td>labels</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>lastz_paired_reads</td>
      <td>lastz_paired_reads_wrapper</td>
      <td>Galaxy wrapper for the Lastz alignment tool on paired reads</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Next Gen Mappers</td>
      <td>lastz_paired_reads</td>
      <td>2012-11-26</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/lastz_paired_reads</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/lastz_paired_reads</td>
      <td>1.1.1</td>
      <td>lastz</td>
      <td>1.04.22</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>7</td>
      <td>9</td>
      <td>3</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>10</td>
      <td>5</td>
      <td>5</td>
      <td>15</td>
      <td>19</td>
      <td>8</td>
      <td>9</td>
    </tr>
    <tr>
      <td></td>
      <td>lca_wrapper</td>
      <td>lca1</td>
      <td>Find lowest diagnostic rank</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>Metagenomics</td>
      <td>lca_wrapper</td>
      <td>2014-01-27</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/taxonomy/lca_wrapper</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/taxonomy/lca_wrapper</td>
      <td>1.0.1</td>
      <td>taxonomy</td>
      <td>0.10.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>89</td>
      <td>137</td>
      <td>4</td>
      <td>20</td>
      <td>118</td>
      <td>6136</td>
      <td>15</td>
      <td>1112</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>207</td>
      <td>6273</td>
      <td>19</td>
      <td>1132</td>
    </tr>
    <tr>
      <td></td>
      <td>lda_analysis</td>
      <td>lda_analy1</td>
      <td>Perform Linear Discriminant Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Graphics, Statistics</td>
      <td>lda_analysis</td>
      <td>2014-07-28</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/lda_analysis</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/lda_analysis</td>
      <td>1.0.1</td>
      <td>R</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>147</td>
      <td>159</td>
      <td>46</td>
      <td>48</td>
      <td>793</td>
      <td>1538</td>
      <td>101</td>
      <td>314</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>3</td>
      <td>940</td>
      <td>1701</td>
      <td>147</td>
      <td>365</td>
    </tr>
    <tr>
      <td></td>
      <td>lighter</td>
      <td>lighter</td>
      <td>Lighter is a kmer-based error correction method for whole genome sequencing data</td>
      <td>lighter</td>
      <td>lighter</td>
      <td></td>
      <td>Lighter</td>
      <td>Kmer-based error correction method for whole genome sequencing data. Lighter uses sampling (rather than counting) to obtain a set of kmers that are likely from the genome. Using this information, Lighter can correct the reads containing sequence errors.</td>
      <td>k-mer counting, Sequence read processing, Sequencing quality control, Sequencing error detection</td>
      <td>Sequencing, Whole genome sequencing, DNA, Genomics</td>
      <td>To update</td>
      <td>https://github.com/mourisl/Lighter</td>
      <td>Sequence Analysis, Fasta Manipulation</td>
      <td>lighter</td>
      <td>2016-06-04</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/lighter</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/lighter</td>
      <td>1.0</td>
      <td>lighter</td>
      <td>1.1.3</td>
      <td>k-mer counting, Sequence read processing, Sequencing quality control, Sequencing error detection</td>
      <td>Whole genome sequencing, DNA, Genomics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>149</td>
      <td>171</td>
      <td>27</td>
      <td>30</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>149</td>
      <td>171</td>
      <td>27</td>
      <td>30</td>
    </tr>
    <tr>
      <td></td>
      <td>linear_regression</td>
      <td>LinearRegression1</td>
      <td>Perform Linear Regression</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>linear_regression</td>
      <td>2014-04-01</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/linear_regression</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/linear_regression</td>
      <td>1.0.1</td>
      <td>R</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>146</td>
      <td>1103</td>
      <td>31</td>
      <td>271</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>146</td>
      <td>1103</td>
      <td>31</td>
      <td>271</td>
    </tr>
    <tr>
      <td></td>
      <td>locarna</td>
      <td>locarna_exparnap, locarna_multiple, locarna_pairwise, locarna_pairwise_p, locarna_reliability_profile</td>
      <td>LocARNA - A suite for multiple alignment and folding of RNAs</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://www.bioinf.uni-freiburg.de/Software/LocARNA/</td>
      <td>RNA</td>
      <td>locarna</td>
      <td>2015-02-28</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/locarna</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/locarna</td>
      <td>2.0.1</td>
      <td>locarna</td>
      <td>2.0.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>314</td>
      <td>350</td>
      <td>36</td>
      <td>46</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>314</td>
      <td>350</td>
      <td>36</td>
      <td>46</td>
    </tr>
    <tr>
      <td></td>
      <td>logistic_regression_vif</td>
      <td>LogisticRegression</td>
      <td>Perform Logistic Regression with vif</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis, Statistics</td>
      <td>logistic_regression_vif</td>
      <td>2014-04-01</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/logistic_regression_vif</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/logistic_regression_vif</td>
      <td>1.0.1</td>
      <td>numpy</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>64</td>
      <td>173</td>
      <td>12</td>
      <td>48</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>64</td>
      <td>173</td>
      <td>12</td>
      <td>48</td>
    </tr>
    <tr>
      <td></td>
      <td>maf_cpg_filter</td>
      <td>cpgFilter</td>
      <td>Mask CpG/non-CpG sites from MAF file</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>maf_cpg_filter</td>
      <td>2014-04-01</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/maf_cpg_filter</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/maf_cpg_filter</td>
      <td>1.0.0</td>
      <td>bx-python</td>
      <td>0.13.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>85</td>
      <td>356</td>
      <td>6</td>
      <td>28</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>85</td>
      <td>356</td>
      <td>6</td>
      <td>28</td>
    </tr>
    <tr>
      <td></td>
      <td>mafft</td>
      <td>rbc_mafft_add, rbc_mafft</td>
      <td>Multiple alignment program for amino acid or nucleotide sequences</td>
      <td>MAFFT</td>
      <td>MAFFT</td>
      <td></td>
      <td>MAFFT</td>
      <td>MAFFT (Multiple Alignment using Fast Fourier Transform) is a high speed multiple sequence alignment program.</td>
      <td>Multiple sequence alignment</td>
      <td>Sequence analysis</td>
      <td>To update</td>
      <td>https://mafft.cbrc.jp/alignment/software/</td>
      <td>RNA</td>
      <td>mafft</td>
      <td>2023-11-06</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/mafft</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/mafft</td>
      <td>7.526</td>
      <td>mafft</td>
      <td>7.525</td>
      <td>Multiple sequence alignment</td>
      <td>Sequence analysis</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>172433</td>
      <td>174456</td>
      <td>1865</td>
      <td>1951</td>
      <td>108640</td>
      <td>122790</td>
      <td>4266</td>
      <td>5521</td>
      <td>20027</td>
      <td>20027</td>
      <td>1172</td>
      <td>1172</td>
      <td>301100</td>
      <td>317273</td>
      <td>7303</td>
      <td>8644</td>
    </tr>
    <tr>
      <td></td>
      <td>mapping_to_ucsc</td>
      <td>mapToUCSC</td>
      <td>Format mapping data as UCSC custom track</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Visualization, Convert Formats, Next Gen Mappers</td>
      <td>mapping_to_ucsc</td>
      <td>2014-05-19</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/mapping_to_ucsc</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/mapping_to_ucsc</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>mavedb</td>
      <td>mavedb_importer</td>
      <td>data source for MaveDB</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Data Source</td>
      <td>mavedb_importer</td>
      <td>2023-12-13</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/mave_tools/mavedb/</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/mave_tools/mavedb</td>
      <td>0.9</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>2</td>
      <td>2</td>
    </tr>
    <tr>
      <td></td>
      <td>mea</td>
      <td>mea</td>
      <td>Maximum expected accuracy prediction</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.bioinf.uni-leipzig.de/Software/mea</td>
      <td>RNA</td>
      <td>mea</td>
      <td>2015-07-29</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/mea</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/mea</td>
      <td>0.6.4.1</td>
      <td>mea</td>
      <td>0.6.4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>73</td>
      <td>86</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>73</td>
      <td>86</td>
      <td>5</td>
      <td>5</td>
    </tr>
    <tr>
      <td></td>
      <td>megablast_xml_parser</td>
      <td>megablast_xml_parser</td>
      <td>Parse blast XML output</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Next Gen Mappers, Convert Formats</td>
      <td>megablast_xml_parser</td>
      <td>2014-05-19</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/megablast_xml_parser</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/megablast_xml_parser</td>
      <td>1.0.1</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>743</td>
      <td>816</td>
      <td>103</td>
      <td>141</td>
      <td>292</td>
      <td>2868</td>
      <td>82</td>
      <td>459</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1035</td>
      <td>3684</td>
      <td>185</td>
      <td>600</td>
    </tr>
    <tr>
      <td></td>
      <td>merge</td>
      <td>gops_merge_1</td>
      <td>Merge the overlapping intervals of a dataset</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/gops</td>
      <td>Genomic Interval Operations</td>
      <td>merge</td>
      <td>2014-04-01</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/gops/merge</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/gops/merge</td>
      <td>1.0.0</td>
      <td>bx-python</td>
      <td>0.13.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>565922</td>
      <td>566630</td>
      <td>418</td>
      <td>441</td>
      <td>45844</td>
      <td>107523</td>
      <td>671</td>
      <td>4353</td>
      <td>19828</td>
      <td>19994</td>
      <td>100</td>
      <td>137</td>
      <td>631594</td>
      <td>694147</td>
      <td>1189</td>
      <td>4931</td>
    </tr>
    <tr>
      <td></td>
      <td>merge_cols</td>
      <td>mergeCols1</td>
      <td>Merge columns together.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>merge_cols</td>
      <td>2012-12-04</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/merge_cols</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/merge_cols</td>
      <td>1.0.3</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>20905</td>
      <td>31051</td>
      <td>1906</td>
      <td>2128</td>
      <td>21780</td>
      <td>132833</td>
      <td>885</td>
      <td>3172</td>
      <td>1524</td>
      <td>1925</td>
      <td>122</td>
      <td>152</td>
      <td>44209</td>
      <td>165809</td>
      <td>2913</td>
      <td>5452</td>
    </tr>
    <tr>
      <td></td>
      <td>methtools</td>
      <td>methtools_calling, r_correlation_matrix, methtools_destrand, methtools_dmr, methtools_filter, methtools_plot, smooth_running_window, methtools_tiling</td>
      <td>tools for methylation analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/methtools</td>
      <td>Sequence Analysis</td>
      <td>methtools</td>
      <td>2015-02-28</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/methtools</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/methtools</td>
      <td>0.1.1</td>
      <td>methtools</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>632</td>
      <td>5461</td>
      <td>155</td>
      <td>453</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>632</td>
      <td>5461</td>
      <td>155</td>
      <td>453</td>
    </tr>
    <tr>
      <td></td>
      <td>methyldackel</td>
      <td>pileometh</td>
      <td>A tool for processing bisulfite sequencing alignments</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/dpryan79/MethylDackel</td>
      <td>Sequence Analysis</td>
      <td>pileometh</td>
      <td>2017-02-14</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/methyldackel</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/methyldackel</td>
      <td>0.5.2</td>
      <td>methyldackel</td>
      <td>0.6.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>14907</td>
      <td>18971</td>
      <td>638</td>
      <td>894</td>
      <td>2677</td>
      <td>2677</td>
      <td>230</td>
      <td>230</td>
      <td>2313</td>
      <td>2313</td>
      <td>83</td>
      <td>83</td>
      <td>19897</td>
      <td>23961</td>
      <td>951</td>
      <td>1207</td>
    </tr>
    <tr>
      <td></td>
      <td>methylkit</td>
      <td>methylkit</td>
      <td>A method for DNA methylation analysis and annotation from high-throughput bisulfite sequencing.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://bioconductor.org/packages/release/bioc/html/methylKit.html</td>
      <td>Epigenetics</td>
      <td>methylkit</td>
      <td>2016-12-20</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/methylkit</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/methylkit</td>
      <td>0.99.2</td>
      <td>bioconductor-methylkit</td>
      <td>1.28.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>metilene</td>
      <td>metilene</td>
      <td>Differential DNA methylation calling</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA, Statistics</td>
      <td>metilene</td>
      <td>2015-12-14</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/metilene</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/metilene</td>
      <td>0.2.6.1</td>
      <td>metilene</td>
      <td>0.2.8</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>3209</td>
      <td>4572</td>
      <td>312</td>
      <td>442</td>
      <td>479</td>
      <td>479</td>
      <td>79</td>
      <td>79</td>
      <td>315</td>
      <td>315</td>
      <td>51</td>
      <td>51</td>
      <td>4003</td>
      <td>5366</td>
      <td>442</td>
      <td>572</td>
    </tr>
    <tr>
      <td></td>
      <td>mgnify_seqprep</td>
      <td>mgnify_seqprep</td>
      <td>A modifiied vesion of SeqPrep. Made for use with the MGnify pipelines.</td>
      <td>seqprep</td>
      <td>seqprep</td>
      <td></td>
      <td>SeqPrep</td>
      <td>Strips adapters and optionally merges overlapping paired-end (or paired-end contamination in mate-pair libraries) illumina style reads.</td>
      <td>Nucleic acid design</td>
      <td>Genomics, Sequence assembly, Sequencing, Probes and primers</td>
      <td>To update</td>
      <td>https://github.com/jstjohn/SeqPrep</td>
      <td>Fastq Manipulation, Sequence Analysis</td>
      <td>mgnify_seqprep</td>
      <td>2024-05-14</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/mgnify_seqprep</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/mgnify_seqprep</td>
      <td>1.2</td>
      <td></td>
      <td></td>
      <td>Nucleic acid design</td>
      <td>Genomics, Sequence assembly, Sequencing, Probes and primers</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>573</td>
      <td>573</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>573</td>
      <td>573</td>
      <td>3</td>
      <td>3</td>
    </tr>
    <tr>
      <td></td>
      <td>miclip</td>
      <td>mi_clip</td>
      <td>Identification of binding sites in CLIP-Seq data.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://cran.r-project.org/src/contrib/Archive/MiClip/</td>
      <td>Sequence Analysis</td>
      <td>miclip</td>
      <td>2015-02-28</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/miclip</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/miclip</td>
      <td>1.2.0</td>
      <td>Rscript</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>microsatellite_birthdeath</td>
      <td>microsatellite_birthdeath</td>
      <td>Identify microsatellite births and deaths</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>microsatellite_birthdeath</td>
      <td>2013-09-25</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/microsatellite_birthdeath</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/microsatellite_birthdeath</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>microsats_alignment_level</td>
      <td>microsats_align1</td>
      <td>Extract Orthologous Microsatellites from pair-wise alignments</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>microsats_alignment_level</td>
      <td>2014-04-01</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/microsats_alignment_level</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/microsats_alignment_level</td>
      <td>1.0.0</td>
      <td>sputnik</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>135</td>
      <td>973</td>
      <td>50</td>
      <td>140</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>135</td>
      <td>973</td>
      <td>50</td>
      <td>140</td>
    </tr>
    <tr>
      <td></td>
      <td>microsats_mutability</td>
      <td>microsats_mutability1</td>
      <td>Estimate microsatellite mutability by specified attributes</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>microsats_mutability</td>
      <td>2014-04-01</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/microsats_mutability</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/microsats_mutability</td>
      <td>1.1.0</td>
      <td>bx-python</td>
      <td>0.13.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>34</td>
      <td>312</td>
      <td>22</td>
      <td>87</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>34</td>
      <td>312</td>
      <td>22</td>
      <td>87</td>
    </tr>
    <tr>
      <td></td>
      <td>minced</td>
      <td>minced</td>
      <td>MinCED - Mining CRISPRs in Environmental Datasets</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://bioweb2.pasteur.fr/docs/modules/minced/0.1.5/_README</td>
      <td>Sequence Analysis</td>
      <td>minced</td>
      <td>2015-02-28</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/minced</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/minced</td>
      <td>0.2.0</td>
      <td>minced</td>
      <td>0.4.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1213</td>
      <td>1309</td>
      <td>115</td>
      <td>124</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1213</td>
      <td>1309</td>
      <td>115</td>
      <td>124</td>
    </tr>
    <tr>
      <td></td>
      <td>mine</td>
      <td>maximal_information_based_nonparametric_exploration</td>
      <td>Maximal Information-based Nonparametric Exploration</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Variant Analysis</td>
      <td>mine</td>
      <td>2014-07-28</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/mine</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/mine</td>
      <td>0.0.1</td>
      <td>MINE</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>9</td>
      <td>11</td>
      <td>8</td>
      <td>8</td>
      <td>17</td>
      <td>63</td>
      <td>6</td>
      <td>19</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>26</td>
      <td>74</td>
      <td>14</td>
      <td>27</td>
    </tr>
    <tr>
      <td></td>
      <td>minipolish</td>
      <td>minipolish</td>
      <td>Polishing miniasm assemblies</td>
      <td>minipolish</td>
      <td>minipolish</td>
      <td></td>
      <td>minipolish</td>
      <td>A tool that bridges the output of miniasm (long-read assembly) and racon (assembly polishing) together to polish a draft assembly. It also provides read depth information in contigs.</td>
      <td>Localised reassembly, Read depth analysis</td>
      <td>Sequence assembly, Sequencing</td>
      <td>Up-to-date</td>
      <td>https://github.com/rrwick/Minipolish</td>
      <td>Sequence Analysis</td>
      <td>minipolish</td>
      <td>2022-10-19</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/minipolish</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/minipolish</td>
      <td>0.1.3</td>
      <td>minipolish</td>
      <td>0.1.3</td>
      <td>Localised reassembly, Read depth analysis</td>
      <td>Sequence assembly, Sequencing</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>229</td>
      <td>229</td>
      <td>32</td>
      <td>32</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>229</td>
      <td>229</td>
      <td>32</td>
      <td>32</td>
    </tr>
    <tr>
      <td></td>
      <td>mitohifi</td>
      <td>mitohifi</td>
      <td>Assembly mitogenomes from Pacbio HiFi read.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/marcelauliano/MitoHiFi/tree/mitohifi_v2</td>
      <td>Assembly</td>
      <td>mitohifi</td>
      <td>2021-05-29</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/mitohifi</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/mitohifi</td>
      <td>3</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1020</td>
      <td>1020</td>
      <td>176</td>
      <td>176</td>
      <td>414</td>
      <td>414</td>
      <td>73</td>
      <td>73</td>
      <td>277</td>
      <td>277</td>
      <td>20</td>
      <td>20</td>
      <td>1711</td>
      <td>1711</td>
      <td>269</td>
      <td>269</td>
    </tr>
    <tr>
      <td></td>
      <td>molecule2gspan</td>
      <td>bg_mol2gspan</td>
      <td>converter</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/molecule2gspan</td>
      <td>Convert Formats</td>
      <td>molecule_to_gspan</td>
      <td>2015-02-28</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/molecule2gspan</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/molecule2gspan</td>
      <td>0.2</td>
      <td>openbabel</td>
      <td>2.3.90dev7d621d9</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>motus</td>
      <td>mereg_mOTUs_tables, motus_profiler</td>
      <td>Tool for profiling the abundance of microbial taxa.</td>
      <td>mOTUs</td>
      <td>mOTUs</td>
      <td></td>
      <td>Metagenomic operational taxonomic units (mOTUs)</td>
      <td>Metagenomic operational taxonomic units (mOTUs) enable high-accuracy taxonomic profiling of known (sequenced) and unknown microorganisms at species-level resolution from shotgun metagenomic or metatranscriptomic data. The method clusters single-copy phylogenetic marker gene sequences from metagenomes and reference genomes into mOTUs to quantify their abundances in meta-omics data with very high precision and recall.</td>
      <td>Taxonomic classification</td>
      <td>Metagenomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/motu-tool/mOTUs</td>
      <td>Metagenomics</td>
      <td>motus</td>
      <td>2024-09-06</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/mOTUs</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/motus</td>
      <td>3.1.0</td>
      <td>motus</td>
      <td>3.1.0</td>
      <td>Taxonomic classification</td>
      <td>Metagenomics</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>mqc</td>
      <td>mqc</td>
      <td>Ribosome profiling mapping quality control tool</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/Biobix/mQC</td>
      <td>Sequence Analysis</td>
      <td>mqc</td>
      <td>2017-08-11</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/mqc/</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/mqc</td>
      <td>1.9</td>
      <td>mqc</td>
      <td>1.10</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>71</td>
      <td>76</td>
      <td>9</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>71</td>
      <td>76</td>
      <td>9</td>
      <td>9</td>
    </tr>
    <tr>
      <td></td>
      <td>multispecies_orthologous_microsats</td>
      <td>multispecies_orthologous_microsats</td>
      <td>Extract orthologous microsatellites</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>multispecies_orthologous_microsats</td>
      <td>2013-09-25</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/multispecies_orthologous_microsats</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/multispecies_orthologous_microsats</td>
      <td>1.0.0</td>
      <td>bx-sputnik</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>mummer</td>
      <td>mummerplot_wrapper</td>
      <td>Draw dotplots using mummer, mucmer, or promer with mummerplot</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://mummer.sourceforge.net/</td>
      <td>Graphics, Sequence Analysis, Visualization</td>
      <td>mummer</td>
      <td>2014-12-02</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/mummer</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/mummer</td>
      <td>0.0.8</td>
      <td>ghostscript</td>
      <td>9.18</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>883</td>
      <td>921</td>
      <td>199</td>
      <td>211</td>
      <td>730</td>
      <td>730</td>
      <td>199</td>
      <td>199</td>
      <td>136</td>
      <td>202</td>
      <td>34</td>
      <td>49</td>
      <td>1749</td>
      <td>1853</td>
      <td>432</td>
      <td>459</td>
    </tr>
    <tr>
      <td></td>
      <td>music_deconvolution</td>
      <td>music_construct_eset, music_inspect_eset, music_manipulate_eset, music_compare, music_deconvolution</td>
      <td>Multi-subject Single Cell deconvolution (MuSiC)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/xuranw/MuSiC</td>
      <td>Transcriptomics</td>
      <td>music</td>
      <td>2021-09-12</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/music/</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/music_deconvolution</td>
      <td>0.1.1</td>
      <td>music-deconvolution</td>
      <td>0.1.1</td>
      <td></td>
      <td></td>
      <td>5</td>
      <td>5</td>
      <td>4</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>2801</td>
      <td>2801</td>
      <td>289</td>
      <td>289</td>
      <td>364</td>
      <td>364</td>
      <td>80</td>
      <td>80</td>
      <td>1357</td>
      <td>1357</td>
      <td>21</td>
      <td>21</td>
      <td>4522</td>
      <td>4522</td>
      <td>390</td>
      <td>390</td>
    </tr>
    <tr>
      <td></td>
      <td>mutate_snp_codon</td>
      <td>mutate_snp_codon_1</td>
      <td>Mutate Codons with SNPs</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Variant Analysis</td>
      <td>mutate_snp_codon</td>
      <td>2013-09-25</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/mutate_snp_codon</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/mutate_snp_codon</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>44</td>
      <td>49</td>
      <td>5</td>
      <td>5</td>
      <td>81</td>
      <td>417</td>
      <td>23</td>
      <td>118</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>125</td>
      <td>466</td>
      <td>28</td>
      <td>123</td>
    </tr>
    <tr>
      <td></td>
      <td>nanopolish</td>
      <td>nanopolish_eventalign, nanopolish_methylation, nanopolish_polya, nanopolish_variants</td>
      <td>Nanopolish software package for signal-level analysis of Oxford Nanopore sequencing data.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/jts/nanopolish</td>
      <td></td>
      <td>nanopolish</td>
      <td>2018-06-05</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/nanopolish</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/nanopolish</td>
      <td>0.14.0</td>
      <td>nanopolish</td>
      <td>0.14.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>9284</td>
      <td>9694</td>
      <td>215</td>
      <td>229</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2358</td>
      <td>2358</td>
      <td>49</td>
      <td>49</td>
      <td>11642</td>
      <td>12052</td>
      <td>264</td>
      <td>278</td>
    </tr>
    <tr>
      <td></td>
      <td>nastiseq</td>
      <td>nastiseq</td>
      <td>A method to identify cis-NATs using ssRNA-seq</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://ohlerlab.mdc-berlin.de/software/NASTIseq_104/</td>
      <td>RNA</td>
      <td>nastiseq</td>
      <td>2017-02-21</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/nastiseq</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/nastiseq</td>
      <td>1.0</td>
      <td>r-nastiseq</td>
      <td>1.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>36</td>
      <td>40</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>36</td>
      <td>40</td>
      <td>1</td>
      <td>1</td>
    </tr>
    <tr>
      <td></td>
      <td>netboxr</td>
      <td>netboxr</td>
      <td>netboxr enables automated discovery of biological process modules by network analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Systems Biology</td>
      <td>netboxr</td>
      <td>2022-08-24</td>
      <td>bgruening</td>
      <td></td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/netboxr</td>
      <td>1.6.0</td>
      <td>bioconductor-netboxr</td>
      <td>1.9.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>12</td>
      <td>12</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>12</td>
      <td>12</td>
      <td>6</td>
      <td>6</td>
    </tr>
    <tr>
      <td></td>
      <td>nextdenovo</td>
      <td>nextdenovo</td>
      <td>String graph-based de novo assembler for long reads</td>
      <td>nextdenovo</td>
      <td>nextdenovo</td>
      <td></td>
      <td>NextDenovo</td>
      <td>NextDenovo is a string graph-based de novo assembler for long reads (CLR, HiFi and ONT). It uses a "correct-then-assemble" strategy similar to canu (no correction step for PacBio Hifi reads), but requires significantly less computing resources and storages.</td>
      <td>De-novo assembly, Genome assembly</td>
      <td>Sequencing, Sequence assembly</td>
      <td>To update</td>
      <td>https://github.com/Nextomics/NextDenovo</td>
      <td>Assembly</td>
      <td>nextdenovo</td>
      <td>2023-02-09</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/nextdenovo</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/nextdenovo</td>
      <td>2.5.0</td>
      <td>nextdenovo</td>
      <td>2.5.2</td>
      <td>De-novo assembly, Genome assembly</td>
      <td>Sequencing, Sequence assembly</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>391</td>
      <td>391</td>
      <td>158</td>
      <td>158</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>391</td>
      <td>391</td>
      <td>158</td>
      <td>158</td>
    </tr>
    <tr>
      <td></td>
      <td>nlstradamus</td>
      <td>nlstradamus</td>
      <td>Find nuclear localization signals (NLSs) in protein sequences</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.moseslab.csb.utoronto.ca/NLStradamus</td>
      <td>Fasta Manipulation, Sequence Analysis</td>
      <td>nlstradamus</td>
      <td>2013-09-10</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/nlstradamus</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/nlstradamus</td>
      <td>0.0.11</td>
      <td>NLStradamus</td>
      <td>1.8</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>nucleosome_prediction</td>
      <td>Nucleosome</td>
      <td>Prediction of Nucleosomes Positions on the Genome</td>
      <td>nucleosome_prediction</td>
      <td>nucleosome_prediction</td>
      <td></td>
      <td>nucleosome_prediction</td>
      <td>Prediction of Nucleosomes Positions on the Genome</td>
      <td>Prediction and recognition, Nucleosome position prediction, Sequence analysis</td>
      <td>Structural genomics, Nucleic acid sites, features and motifs</td>
      <td>Up-to-date</td>
      <td>https://genie.weizmann.ac.il/software/nucleo_exe.html</td>
      <td>Sequence Analysis</td>
      <td>nucleosome_prediction</td>
      <td>2016-09-27</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/nucleosome_prediction</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/nucleosome_prediction</td>
      <td>3.0</td>
      <td>nucleosome_prediction</td>
      <td>3.0</td>
      <td>Prediction and recognition, Nucleosome position prediction, Sequence analysis</td>
      <td>Structural genomics, Nucleic acid sites, features and motifs</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>118</td>
      <td>863</td>
      <td>19</td>
      <td>26</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>118</td>
      <td>863</td>
      <td>19</td>
      <td>26</td>
    </tr>
    <tr>
      <td></td>
      <td>numeric_clustering</td>
      <td>numeric_clustering</td>
      <td>Clustering tool for numberic values</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://scikit-learn.org/stable/index.html</td>
      <td>Statistics</td>
      <td>numeric_clustering</td>
      <td>2015-12-19</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/numeric_clustering</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/numeric_clustering</td>
      <td>0.9</td>
      <td>anaconda</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>391</td>
      <td>486</td>
      <td>5</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>391</td>
      <td>486</td>
      <td>5</td>
      <td>11</td>
    </tr>
    <tr>
      <td></td>
      <td>openms</td>
      <td>AccurateMassSearch, AdditiveSeries, BaselineFilter, CVInspector, CompNovo, CompNovoCID, ConsensusID, ConsensusMapNormalizer, ConvertTSVToTraML, ConvertTraMLToTSV, DTAExtractor, DeMeanderize, Decharger, DecoyDatabase, Digestor, DigestorMotif, EICExtractor, ERPairFinder, ExternalCalibration, FFEval, FalseDiscoveryRate, FeatureFinderCentroided, FeatureFinderIdentification, FeatureFinderIsotopeWavelet, FeatureFinderMRM, FeatureFinderMetabo, FeatureFinderMultiplex, FeatureFinderSuperHirn, FeatureLinkerLabeled, FeatureLinkerUnlabeled, FeatureLinkerUnlabeledQT, FidoAdapter, FileConverter, FileFilter, FileInfo, FileMerger, FuzzyDiff, HighResPrecursorMassCorrector, IDConflictResolver, IDDecoyProbability, IDExtractor, IDFileConverter, IDFilter, IDMapper, IDMassAccuracy, IDMerger, IDPosteriorErrorProbability, IDRTCalibration, IDRipper, IDScoreSwitcher, IDSplitter, ITRAQAnalyzer, InclusionExclusionListCreator, InspectAdapter, InternalCalibration, IsobaricAnalyzer, LabeledEval, LowMemPeakPickerHiRes, LowMemPeakPickerHiRes_RandomAccess, LuciphorAdapter, MRMMapper, MRMPairFinder, MRMTransitionGroupPicker, MSGFPlusAdapter, MSSimulator, MapAlignmentEvaluation, MapNormalizer, MapRTTransformer, MapStatistics, MascotAdapter, MascotAdapterOnline, MassCalculator, MassTraceExtractor, MetaProSIP, MetaboliteSpectralMatcher, MultiplexResolver, MzMLSplitter, MzTabExporter, NoiseFilterGaussian, NoiseFilterSGolay, OpenSwathAnalyzer, OpenSwathAssayGenerator, OpenSwathChromatogramExtractor, OpenSwathConfidenceScoring, OpenSwathDIAPreScoring, OpenSwathDecoyGenerator, OpenSwathFeatureXMLToTSV, OpenSwathFileSplitter, OpenSwathMzMLFileCacher, OpenSwathRTNormalizer, OpenSwathRewriteToFeatureXML, OpenSwathWorkflow, PTModel, PTPredict, PeakPickerHiRes, PeakPickerIterative, PeakPickerWavelet, PepNovoAdapter, PeptideIndexer, PhosphoScoring, PrecursorIonSelector, PrecursorMassCorrector, ProteinInference, ProteinQuantifier, ProteinResolver, QCCalculator, QCEmbedder, QCExporter, QCExtractor, QCImporter, QCMerger, QCShrinker, RNPxl, RNPxlXICFilter, RTEvaluation, RTModel, RTPredict, SemanticValidator, SequenceCoverageCalculator, SimpleSearchEngine, SpecLibCreator, SpectraFilterBernNorm, SpectraFilterMarkerMower, SpectraFilterNLargest, SpectraFilterNormalizer, SpectraFilterParentPeakMower, SpectraFilterScaler, SpectraFilterSqrtMower, SpectraFilterThresholdMower, SpectraFilterWindowMower, SpectraMerger, SvmTheoreticalSpectrumGeneratorTrainer, TICCalculator, TMTAnalyzer, TOFCalibration, TextExporter, TopPerc, TransformationEvaluation, XMLValidator, XTandemAdapter</td>
      <td>OpenMS in version 2.1.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>openms</td>
      <td>2015-10-09</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/openms</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/openms</td>
      <td>2.1.0</td>
      <td>openms</td>
      <td>3.2.0</td>
      <td></td>
      <td></td>
      <td>7</td>
      <td>34</td>
      <td>135</td>
      <td>0</td>
      <td>7</td>
      <td>34</td>
      <td>135</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>19</td>
      <td>135</td>
      <td>134</td>
      <td>0</td>
      <td>31121</td>
      <td>108907</td>
      <td>3369</td>
      <td>5029</td>
      <td>50</td>
      <td>50</td>
      <td>15</td>
      <td>15</td>
      <td>933</td>
      <td>933</td>
      <td>242</td>
      <td>242</td>
      <td>32104</td>
      <td>109890</td>
      <td>3626</td>
      <td>5286</td>
    </tr>
    <tr>
      <td></td>
      <td>pandas_rolling_window</td>
      <td>pandas_rolling_window</td>
      <td>Rolling window calculations</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.rolling.html</td>
      <td>Statistics</td>
      <td>pandas_rolling_window</td>
      <td>2019-05-20</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/pandas_rolling_window</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/pandas_rolling_window</td>
      <td>0.1</td>
      <td>numpy</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>261</td>
      <td>275</td>
      <td>12</td>
      <td>12</td>
      <td>63</td>
      <td>63</td>
      <td>22</td>
      <td>22</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>324</td>
      <td>338</td>
      <td>34</td>
      <td>34</td>
    </tr>
    <tr>
      <td></td>
      <td>paralyzer</td>
      <td>paralyzer</td>
      <td>A method to generate a high resolution map of interaction sites between RNA-binding proteins and their targets.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://ohlerlab.mdc-berlin.de/software/PARalyzer_85/</td>
      <td>RNA</td>
      <td>paralyzer</td>
      <td>2016-12-02</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/paralyzer</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/paralyzer</td>
      <td>1.5</td>
      <td>paralyzer</td>
      <td>1.5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>322</td>
      <td>349</td>
      <td>24</td>
      <td>29</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>322</td>
      <td>349</td>
      <td>24</td>
      <td>29</td>
    </tr>
    <tr>
      <td></td>
      <td>partialr_square</td>
      <td>partialRsq</td>
      <td>Compute partial R square</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>partialr_square</td>
      <td>2014-04-01</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/partialr_square</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/partialr_square</td>
      <td>1.0.0</td>
      <td>R</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>55</td>
      <td>142</td>
      <td>8</td>
      <td>42</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>55</td>
      <td>142</td>
      <td>8</td>
      <td>42</td>
    </tr>
    <tr>
      <td></td>
      <td>peakachu</td>
      <td>peakachu</td>
      <td>PEAKachu is a peak-caller for CLIP- and RIP-Seq data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, RNA</td>
      <td>peakachu</td>
      <td>2018-02-14</td>
      <td>rnateam</td>
      <td>https://github.com/tbischler/PEAKachu</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/peakachu</td>
      <td>0.2.0+galaxy1</td>
      <td>peakachu</td>
      <td>0.2.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2807</td>
      <td>3337</td>
      <td>178</td>
      <td>196</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2807</td>
      <td>3337</td>
      <td>178</td>
      <td>196</td>
    </tr>
    <tr>
      <td></td>
      <td>pearson_correlation</td>
      <td>Pearson_and_apos_Correlation1</td>
      <td>Pearson and apos Correlation between any two numeric columns</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>pearson_correlation</td>
      <td>2014-07-28</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/pearson_correlation</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/pearson_correlation</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
    </tr>
    <tr>
      <td></td>
      <td>perf</td>
      <td>stats_perf_tool</td>
      <td>suitable for boolean classification problems</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://osmot.cs.cornell.edu/kddcup/software.html</td>
      <td></td>
      <td>stats_perf_tool</td>
      <td>2015-02-28</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/perf</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/perf</td>
      <td>5.11.0</td>
      <td>perf</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>pfamscan</td>
      <td>pfamscan</td>
      <td>Search a FASTA sequence against a library of Pfam HMM.</td>
      <td>pfamscan</td>
      <td>pfamscan</td>
      <td></td>
      <td>PfamScan</td>
      <td>This tool is used to search a FASTA sequence against a library of Pfam HMM.</td>
      <td>Protein sequence analysis</td>
      <td>Sequence analysis</td>
      <td>Up-to-date</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/Pfam/Tools/</td>
      <td>Sequence Analysis</td>
      <td>pfamscan</td>
      <td>2023-02-02</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/pfamscan</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/pfamscan</td>
      <td>1.6</td>
      <td>pfam_scan</td>
      <td>1.6</td>
      <td>Protein sequence analysis</td>
      <td>Sequence analysis</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>773</td>
      <td>773</td>
      <td>60</td>
      <td>60</td>
      <td>495</td>
      <td>495</td>
      <td>72</td>
      <td>72</td>
      <td>246</td>
      <td>246</td>
      <td>17</td>
      <td>17</td>
      <td>1514</td>
      <td>1514</td>
      <td>149</td>
      <td>149</td>
    </tr>
    <tr>
      <td></td>
      <td>pg_tools</td>
      <td>pg_dump, pg_import, pg_query</td>
      <td>tool suite for dealing with Postgresql databases from Galaxy's history</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.postgresql.org</td>
      <td>Data Export, Data Source</td>
      <td>pgtools</td>
      <td>2017-09-28</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tools/pgtools</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/pg_tools</td>
      <td></td>
      <td>postgresql</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>pgsnp2gd_snp</td>
      <td>pgSnp2gd_snp</td>
      <td>Convert from pgSnp to gd_snp</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Variant Analysis</td>
      <td>pgsnp2gd_snp</td>
      <td>2014-07-28</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/pgsnp2gd_snp</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/pgsnp2gd_snp</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>pharmcat</td>
      <td>pharmcat</td>
      <td>Pharmacogenomics Clinical Annotation Tool</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://pharmcat.org/</td>
      <td>Variant Analysis</td>
      <td>pharmcat</td>
      <td>2022-01-28</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/pharmcat</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/pharmcat</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>109</td>
      <td>109</td>
      <td>29</td>
      <td>29</td>
      <td>250</td>
      <td>250</td>
      <td>84</td>
      <td>84</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>359</td>
      <td>359</td>
      <td>113</td>
      <td>113</td>
    </tr>
    <tr>
      <td></td>
      <td>pileup_interval</td>
      <td>pileup_interval</td>
      <td>Pileup-to-Interval condenses pileup format into ranges of bases</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>SAM</td>
      <td>pileup_interval</td>
      <td>2015-03-06</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/pileup_interval</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/pileup_interval</td>
      <td>1.0.3</td>
      <td>bx-python</td>
      <td>0.13.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>613</td>
      <td>640</td>
      <td>48</td>
      <td>54</td>
      <td>1381</td>
      <td>7804</td>
      <td>201</td>
      <td>1552</td>
      <td>756</td>
      <td>815</td>
      <td>22</td>
      <td>30</td>
      <td>2750</td>
      <td>9259</td>
      <td>271</td>
      <td>1636</td>
    </tr>
    <tr>
      <td></td>
      <td>pileup_parser</td>
      <td>pileup_parser</td>
      <td>Filter pileup on coverage and SNPs</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-devteam/</td>
      <td>SAM</td>
      <td>pileup_parser</td>
      <td>2013-08-26</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/pileup_parser</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/pileup_parser</td>
      <td>1.0.2</td>
      <td>perl</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>3639</td>
      <td>3774</td>
      <td>414</td>
      <td>461</td>
      <td>9754</td>
      <td>52522</td>
      <td>1247</td>
      <td>4531</td>
      <td>1893</td>
      <td>2433</td>
      <td>191</td>
      <td>318</td>
      <td>15286</td>
      <td>58729</td>
      <td>1852</td>
      <td>5310</td>
    </tr>
    <tr>
      <td></td>
      <td>pipmir</td>
      <td>pipmir</td>
      <td>A method to identify novel plant miRNA.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://ohlerlab.mdc-berlin.de/software/Pipeline_for_the_Identification_of_Plant_miRNAs_84/</td>
      <td>RNA</td>
      <td>pipmir</td>
      <td>2016-11-25</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/pipmir</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/pipmir</td>
      <td>0.1.0</td>
      <td>pipmir</td>
      <td>1.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>238</td>
      <td>283</td>
      <td>42</td>
      <td>45</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>238</td>
      <td>283</td>
      <td>42</td>
      <td>45</td>
    </tr>
    <tr>
      <td></td>
      <td>piranha</td>
      <td>piranha</td>
      <td>Piranha is a peak-caller for CLIP- and RIP-Seq data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, RNA</td>
      <td>piranha</td>
      <td>2015-07-02</td>
      <td>rnateam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/piranha</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/piranha</td>
      <td>1.2.1.0</td>
      <td>piranha</td>
      <td>1.2.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1408</td>
      <td>2068</td>
      <td>94</td>
      <td>119</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1408</td>
      <td>2068</td>
      <td>94</td>
      <td>119</td>
    </tr>
    <tr>
      <td></td>
      <td>platypus</td>
      <td>bg_platypus</td>
      <td>efficient and accurate variant-detection in high-throughput sequencing data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.well.ox.ac.uk/platypus</td>
      <td>Sequence Analysis</td>
      <td>platypus</td>
      <td>2015-02-28</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/platypus</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/platypus</td>
      <td>0.0.11</td>
      <td>platypus</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>plot_from_lda</td>
      <td>plot_for_lda_output1</td>
      <td>Draw ROC plot on "Perform LDA" output</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Graphics, Statistics</td>
      <td>plot_from_lda</td>
      <td>2014-07-28</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/plot_from_lda</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/plot_from_lda</td>
      <td>1.0.1</td>
      <td>R</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>65</td>
      <td>77</td>
      <td>19</td>
      <td>25</td>
      <td>153</td>
      <td>886</td>
      <td>38</td>
      <td>343</td>
      <td>142</td>
      <td>155</td>
      <td>8</td>
      <td>10</td>
      <td>360</td>
      <td>1118</td>
      <td>65</td>
      <td>378</td>
    </tr>
    <tr>
      <td></td>
      <td>plotly_ml_performance_plots</td>
      <td>plotly_ml_performance_plots</td>
      <td>performance plots for machine learning problems</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://scikit-learn.org/stable/modules/classes.html#module-sklearn.metrics</td>
      <td>Visualization</td>
      <td>plotly_ml_performance_plots</td>
      <td>2018-10-11</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/plotly_ml_performance_plots</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/plotly_ml_performance_plots</td>
      <td>0.4</td>
      <td>galaxy-ml</td>
      <td>0.10.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1342</td>
      <td>1472</td>
      <td>184</td>
      <td>197</td>
      <td>902</td>
      <td>902</td>
      <td>156</td>
      <td>156</td>
      <td>630</td>
      <td>630</td>
      <td>24</td>
      <td>24</td>
      <td>2874</td>
      <td>3004</td>
      <td>364</td>
      <td>377</td>
    </tr>
    <tr>
      <td></td>
      <td>plotly_parallel_coordinates_plot</td>
      <td>plotly_parallel_coordinates_plot</td>
      <td>parallel coordinates plot produced with plotly</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://plot.ly/python/parallel-coordinates-plot/</td>
      <td>Visualization</td>
      <td>plotly_parallel_coordinates_plot</td>
      <td>2018-09-23</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/plotly_parallel_coordinates_plot</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/plotly_parallel_coordinates_plot</td>
      <td>0.2</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>408</td>
      <td>664</td>
      <td>91</td>
      <td>122</td>
      <td>627</td>
      <td>627</td>
      <td>118</td>
      <td>118</td>
      <td>605</td>
      <td>605</td>
      <td>26</td>
      <td>26</td>
      <td>1640</td>
      <td>1896</td>
      <td>235</td>
      <td>266</td>
    </tr>
    <tr>
      <td></td>
      <td>plotly_regression_performance_plots</td>
      <td>plotly_regression_performance_plots</td>
      <td>performance plots for regression problems</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://scikit-learn.org/stable/supervised_learning.html#supervised-learning</td>
      <td>Visualization</td>
      <td>plotly_regression_performance_plots</td>
      <td>2018-10-30</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/plotly_regression_performance_plots</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/plotly_regression_performance_plots</td>
      <td>0.1</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>764</td>
      <td>879</td>
      <td>213</td>
      <td>237</td>
      <td>304</td>
      <td>304</td>
      <td>95</td>
      <td>95</td>
      <td>193</td>
      <td>193</td>
      <td>26</td>
      <td>26</td>
      <td>1261</td>
      <td>1376</td>
      <td>334</td>
      <td>358</td>
    </tr>
    <tr>
      <td></td>
      <td>poisson2test</td>
      <td>poisson2test</td>
      <td>Poisson two-sample test</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>Statistics, Metagenomics</td>
      <td>poisson2test</td>
      <td>2014-01-27</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/taxonomy/poisson2test</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/taxonomy/poisson2test</td>
      <td>1.0.0</td>
      <td>taxonomy</td>
      <td>0.10.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>112</td>
      <td>120</td>
      <td>16</td>
      <td>16</td>
      <td>4</td>
      <td>562</td>
      <td>1</td>
      <td>140</td>
      <td>320</td>
      <td>347</td>
      <td>7</td>
      <td>8</td>
      <td>436</td>
      <td>1029</td>
      <td>24</td>
      <td>164</td>
    </tr>
    <tr>
      <td></td>
      <td>predictnls</td>
      <td>predictnls</td>
      <td>Python reimplementation of predictNLS for Galaxy</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/predictnls</td>
      <td>Sequence Analysis</td>
      <td>predictnls</td>
      <td>2014-11-19</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/predictnls</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/predictnls</td>
      <td>0.0.10</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>principal_component_analysis</td>
      <td>pca1</td>
      <td>Principal Component Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>principal_component_analysis</td>
      <td>2014-05-19</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/principal_component_analysis</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/principal_component_analysis</td>
      <td>1.0.2</td>
      <td>rpy</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>9788</td>
      <td>10032</td>
      <td>207</td>
      <td>244</td>
      <td>3072</td>
      <td>8054</td>
      <td>607</td>
      <td>1488</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>12860</td>
      <td>18086</td>
      <td>814</td>
      <td>1732</td>
    </tr>
    <tr>
      <td></td>
      <td>protease_prediction</td>
      <td>eden_protease_prediction</td>
      <td>This tool can learn the cleavage specificity of a given class of proteases.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/fabriziocosta/eden</td>
      <td>Sequence Analysis, Proteomics</td>
      <td>protease_prediction</td>
      <td>2016-03-13</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/protease_prediction</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/protease_prediction</td>
      <td>0.9</td>
      <td>eden</td>
      <td>2.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>129</td>
      <td>155</td>
      <td>15</td>
      <td>21</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>129</td>
      <td>155</td>
      <td>15</td>
      <td>21</td>
    </tr>
    <tr>
      <td></td>
      <td>protein_analysis</td>
      <td>promoter2, Psortb, rxlr_motifs, signalp3, tmhmm2, wolf_psort</td>
      <td>TMHMM, SignalP, Promoter, RXLR motifs, WoLF PSORT and PSORTb</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/protein_analysis</td>
      <td>Sequence Analysis</td>
      <td>tmhmm_and_signalp</td>
      <td>2017-08-30</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/protein_analysis</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/protein_analysis</td>
      <td>0.0.13</td>
      <td>promoter</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>11743</td>
      <td>12752</td>
      <td>1137</td>
      <td>1278</td>
      <td>2</td>
      <td>2</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>11745</td>
      <td>12754</td>
      <td>1138</td>
      <td>1279</td>
    </tr>
    <tr>
      <td></td>
      <td>protein_properties</td>
      <td>bg_protein_properties</td>
      <td>Calculation of various properties from given protein sequences</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>protein_properties</td>
      <td>2015-06-07</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/protein_properties</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/protein_properties</td>
      <td>0.2.0</td>
      <td>biopython</td>
      <td>1.70</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>465</td>
      <td>638</td>
      <td>81</td>
      <td>100</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>465</td>
      <td>638</td>
      <td>81</td>
      <td>100</td>
    </tr>
    <tr>
      <td></td>
      <td>quality_filter</td>
      <td>qualityFilter</td>
      <td>Filter nucleotides based on quality scores</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>quality_filter</td>
      <td>2014-04-01</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/quality_filter</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/quality_filter</td>
      <td>1.0.1</td>
      <td>bx-python</td>
      <td>0.13.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>98</td>
      <td>724</td>
      <td>35</td>
      <td>62</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>98</td>
      <td>724</td>
      <td>35</td>
      <td>62</td>
    </tr>
    <tr>
      <td></td>
      <td>rRNA</td>
      <td>meta_rna</td>
      <td>Identification of ribosomal RNA genes in metagenomic fragments.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://weizhong-lab.ucsd.edu/meta_rna/</td>
      <td>RNA</td>
      <td>rrna</td>
      <td>2015-02-28</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rRNA</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/rRNA</td>
      <td>0.1</td>
      <td>hmmsearch3.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>racon</td>
      <td>racon</td>
      <td>Consensus module for raw de novo DNA assembly of long uncorrected reads.</td>
      <td>Racon</td>
      <td>Racon</td>
      <td></td>
      <td>Racon</td>
      <td>Consensus module for raw de novo DNA assembly of long uncorrected readsRacon is intended as a standalone consensus module to correct raw contigs generated by rapid assembly methods which do not include a consensus step. The goal of Racon is to generate genomic consensus which is of similar or better quality compared to the output generated by assembly methods which employ both error correction and consensus steps, while providing a speedup of several times compared to those methods. It supports data produced by both Pacific Biosciences and Oxford Nanopore Technologies.</td>
      <td>Genome assembly, Mapping assembly</td>
      <td>Whole genome sequencing, Sequence assembly</td>
      <td>Up-to-date</td>
      <td>https://github.com/isovic/racon</td>
      <td>Sequence Analysis</td>
      <td>racon</td>
      <td>2018-06-11</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/racon</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/racon</td>
      <td>1.5.0</td>
      <td>racon</td>
      <td>1.5.0</td>
      <td>Genome assembly, Mapping assembly</td>
      <td>Whole genome sequencing, Sequence assembly</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>27113</td>
      <td>27315</td>
      <td>839</td>
      <td>850</td>
      <td>9089</td>
      <td>9089</td>
      <td>475</td>
      <td>475</td>
      <td>5452</td>
      <td>5452</td>
      <td>171</td>
      <td>171</td>
      <td>41654</td>
      <td>41856</td>
      <td>1485</td>
      <td>1496</td>
    </tr>
    <tr>
      <td></td>
      <td>rbpbench</td>
      <td>rbpbench</td>
      <td>Evaluate CLIP-seq and other genomic region data using a comprehensive collection of RBP binding motifs</td>
      <td>rbpbench</td>
      <td>rbpbench</td>
      <td></td>
      <td>RBPBench</td>
      <td>Evaluate CLIP-seq and other genomic region data using a comprehensive collection of RBP binding motifs</td>
      <td></td>
      <td>RNA, Protein interactions, RNA immunoprecipitation, Bioinformatics, Sequence analysis</td>
      <td>To update</td>
      <td>https://github.com/michauhl/RBPBench</td>
      <td>Sequence Analysis, RNA, CLIP-seq</td>
      <td>rbpbench</td>
      <td>2023-12-03</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/rbpbench</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/rbpbench</td>
      <td>0.8.1</td>
      <td>rbpbench</td>
      <td>1.0.1</td>
      <td></td>
      <td>RNA, Protein interactions, RNA immunoprecipitation, Bioinformatics, Sequence analysis</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>90</td>
      <td>90</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>90</td>
      <td>90</td>
      <td>6</td>
      <td>6</td>
    </tr>
    <tr>
      <td></td>
      <td>rcas</td>
      <td>rcas</td>
      <td>RCAS (RNA Centric Annotation System) for functional analysis of transcriptome-wide regions detected by high-throughput experiments</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/BIMSBbioinfo/RCAS</td>
      <td>RNA</td>
      <td>rcas</td>
      <td>2017-04-13</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/rcas/</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/rcas</td>
      <td>1.5.4</td>
      <td>bioconductor-rcas</td>
      <td>1.28.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>860</td>
      <td>1257</td>
      <td>88</td>
      <td>98</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>860</td>
      <td>1257</td>
      <td>88</td>
      <td>98</td>
    </tr>
    <tr>
      <td></td>
      <td>rcve</td>
      <td>rcve1</td>
      <td>Compute RCVE</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>rcve</td>
      <td>2014-04-01</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/rcve</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/rcve</td>
      <td>1.0.0</td>
      <td>R</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>49</td>
      <td>191</td>
      <td>5</td>
      <td>46</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>49</td>
      <td>191</td>
      <td>5</td>
      <td>46</td>
    </tr>
    <tr>
      <td></td>
      <td>reago</td>
      <td>reago</td>
      <td>Reago is tool to assembly 16S ribosomal RNA recovery from metagenomic data.</td>
      <td>reago</td>
      <td>reago</td>
      <td></td>
      <td>REAGO</td>
      <td>This is an assembly tool for 16S ribosomal RNA recovery from metagenomic data.</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly, RNA, Metagenomics, Microbiology</td>
      <td>Up-to-date</td>
      <td>https://github.com/chengyuan/reago-1.1</td>
      <td>Metagenomics, RNA</td>
      <td>reago</td>
      <td>2015-12-09</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/reago</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/reago</td>
      <td>1.1</td>
      <td>reago</td>
      <td>1.1</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly, RNA, Metagenomics, Microbiology</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>remove_beginning</td>
      <td>Remove beginning1</td>
      <td>Remove lines from the beginning of a file.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>remove_beginning</td>
      <td>2012-12-04</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/remove_beginning</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/remove_beginning</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>148572</td>
      <td>169964</td>
      <td>3525</td>
      <td>3682</td>
      <td>36694</td>
      <td>113826</td>
      <td>3910</td>
      <td>8163</td>
      <td>9713</td>
      <td>10577</td>
      <td>1475</td>
      <td>1598</td>
      <td>194979</td>
      <td>294367</td>
      <td>8910</td>
      <td>13443</td>
    </tr>
    <tr>
      <td></td>
      <td>remurna</td>
      <td>remurna</td>
      <td>remuRNA - Measurement of Single Nucleotide Polymorphism induced Changes of RNA Conformation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.ncbi.nlm.nih.gov/CBBresearch/Przytycka/index.cgi#remurna</td>
      <td>RNA</td>
      <td>remurna</td>
      <td>2016-11-24</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/remurna</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/remurna</td>
      <td>1.0.0</td>
      <td>remurna</td>
      <td>1.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>39</td>
      <td>44</td>
      <td>6</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>39</td>
      <td>44</td>
      <td>6</td>
      <td>7</td>
    </tr>
    <tr>
      <td></td>
      <td>repeat_masker</td>
      <td>repeatmasker_wrapper</td>
      <td>RepeatMasker is a program that screens DNA sequences for interspersed repeats and low complexity DNA sequences.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.repeatmasker.org/</td>
      <td>Sequence Analysis</td>
      <td>repeat_masker</td>
      <td>2015-02-28</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/repeat_masker</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/repeat_masker</td>
      <td>0.1.2</td>
      <td>RepeatMasker</td>
      <td>4.0.9_p2</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>5998</td>
      <td>5998</td>
      <td>810</td>
      <td>810</td>
      <td>7618</td>
      <td>7618</td>
      <td>803</td>
      <td>803</td>
      <td>3135</td>
      <td>3135</td>
      <td>292</td>
      <td>292</td>
      <td>16751</td>
      <td>16751</td>
      <td>1905</td>
      <td>1905</td>
    </tr>
    <tr>
      <td></td>
      <td>replaceColumn</td>
      <td>replace_column_with_key_value_file</td>
      <td>A tool to replace all column entries of a file given by values of a key-value file.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>replace_column_by_key_value_file</td>
      <td>2017-02-24</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/replaceColumn/tools/replaceColumn</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/replaceColumn</td>
      <td>0.2</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>577184</td>
      <td>577312</td>
      <td>463</td>
      <td>492</td>
      <td>43900</td>
      <td>44233</td>
      <td>283</td>
      <td>373</td>
      <td>2190</td>
      <td>2190</td>
      <td>112</td>
      <td>112</td>
      <td>623274</td>
      <td>623735</td>
      <td>858</td>
      <td>977</td>
    </tr>
    <tr>
      <td></td>
      <td>rest_tool</td>
      <td>pubchem_rest_tool</td>
      <td>This tool fetches data from pubchem via the PubChem REST API.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://pubchem.ncbi.nlm.nih.gov/pug_rest/PUG_REST.html</td>
      <td>Data Source</td>
      <td>pubchem_rest_tool</td>
      <td>2015-02-28</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rest_tool</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rest_tool</td>
      <td>0.1.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>ribotaper</td>
      <td>ribotaper_create_annotation, ribotaper_create_metaplots, ribotaper_ribosome_profiling</td>
      <td>A method for defining traslated ORFs using Ribosome Profiling data.</td>
      <td>ribotaper</td>
      <td>ribotaper</td>
      <td></td>
      <td>RiboTaper</td>
      <td>New analysis pipeline for Ribosome Profiling (Ribo-seq) experiments, which exploits the triplet periodicity of ribosomal footprints to call translated regions.</td>
      <td>Gene expression profiling</td>
      <td>Functional genomics</td>
      <td>To update</td>
      <td>https://ohlerlab.mdc-berlin.de/software/RiboTaper_126/</td>
      <td>RNA</td>
      <td>ribotaper</td>
      <td>2016-03-23</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/ribotaper/</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/ribotaper</td>
      <td>1.3.1a</td>
      <td>ribotaper</td>
      <td>1.3.1</td>
      <td>Gene expression profiling</td>
      <td>Functional genomics</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>661</td>
      <td>681</td>
      <td>78</td>
      <td>78</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>661</td>
      <td>681</td>
      <td>78</td>
      <td>78</td>
    </tr>
    <tr>
      <td></td>
      <td>rmap</td>
      <td>rmap_wrapper</td>
      <td>RMAP for Solexa Short Reads Alignment</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Next Gen Mappers</td>
      <td>rmap</td>
      <td>2014-05-19</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/rmap</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/rmap</td>
      <td>1.0.0</td>
      <td>rmap</td>
      <td>2.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>rmapq</td>
      <td>rmapq_wrapper</td>
      <td>RMAPQ for Solexa Short Reads Alignment with Quality Scores</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Next Gen Mappers</td>
      <td>rmapq</td>
      <td>2014-05-19</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/rmapq</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/rmapq</td>
      <td>1.0.0</td>
      <td>rmap</td>
      <td>2.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>rna_shapes</td>
      <td>RNAshapes</td>
      <td>Compute secondary structures of RNA</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>rnashapes</td>
      <td>2015-02-28</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/rna_shapes</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/rna_shapes</td>
      <td>3.3.0</td>
      <td>@EXECUTABLE@</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>137</td>
      <td>173</td>
      <td>40</td>
      <td>49</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>137</td>
      <td>173</td>
      <td>40</td>
      <td>49</td>
    </tr>
    <tr>
      <td></td>
      <td>rnabob</td>
      <td>rbc_rnabob</td>
      <td>Fast pattern searching for RNA structural motifs</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://eddylab.org/software.html</td>
      <td>RNA</td>
      <td>rnabob</td>
      <td>2015-02-28</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/rnabob</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/rnabob</td>
      <td>2.2.1.0</td>
      <td>rnabob</td>
      <td>2.2.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>90</td>
      <td>196</td>
      <td>9</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>90</td>
      <td>196</td>
      <td>9</td>
      <td>11</td>
    </tr>
    <tr>
      <td></td>
      <td>rnacode</td>
      <td>rbc_rnacode</td>
      <td>Analyze the protein coding potential in MSA</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>rnacode</td>
      <td>2015-06-16</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/rnacode</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/rnacode</td>
      <td>0.3.2</td>
      <td>rnacode</td>
      <td>0.3</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>878</td>
      <td>1363</td>
      <td>29</td>
      <td>35</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>878</td>
      <td>1363</td>
      <td>29</td>
      <td>35</td>
    </tr>
    <tr>
      <td></td>
      <td>rnacommender</td>
      <td>rbc_rnacommender</td>
      <td>RNAcommender is a tool for genome-wide recommendation of RNA-protein interactions.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/gianlucacorrado/RNAcommender</td>
      <td>RNA</td>
      <td>rnacommender</td>
      <td>2016-05-31</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/rna_commander/tools/rna_tools/rna_commender</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/rnacommender</td>
      <td>0.1.1</td>
      <td>sam</td>
      <td>3.5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1024</td>
      <td>1077</td>
      <td>18</td>
      <td>21</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1024</td>
      <td>1077</td>
      <td>18</td>
      <td>21</td>
    </tr>
    <tr>
      <td></td>
      <td>rnaformer</td>
      <td>infer_rnaformer</td>
      <td>RNAformer: RNA secondary structure prediction</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/automl/RNAformer</td>
      <td>RNA</td>
      <td>rnaformer</td>
      <td>2024-07-11</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/rnaformer</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/rnaformer</td>
      <td>1.0.0</td>
      <td>rnaformer</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>26</td>
      <td>26</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>26</td>
      <td>26</td>
      <td>6</td>
      <td>6</td>
    </tr>
    <tr>
      <td></td>
      <td>rnalien</td>
      <td>RNAlien</td>
      <td>RNAlien unsupervized RNA family model construction</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://rna.tbi.univie.ac.at/rnalien/</td>
      <td>RNA, Sequence Analysis</td>
      <td>rnalien</td>
      <td>2017-03-07</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/rnalien</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/rnalien</td>
      <td>1.3.6</td>
      <td>rnalien</td>
      <td>1.8.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>23</td>
      <td>33</td>
      <td>10</td>
      <td>14</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>23</td>
      <td>33</td>
      <td>10</td>
      <td>14</td>
    </tr>
    <tr>
      <td></td>
      <td>rnasnp</td>
      <td>rnasnp</td>
      <td>RNAsnp Efficient detection of local RNA secondary structure changes induced by SNPs</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://rth.dk/resources/rnasnp/</td>
      <td>RNA</td>
      <td>rnasnp</td>
      <td>2016-11-24</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rnasnp</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/rnasnp</td>
      <td>1.2.0</td>
      <td>rnasnp</td>
      <td>1.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>65</td>
      <td>89</td>
      <td>14</td>
      <td>18</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>65</td>
      <td>89</td>
      <td>14</td>
      <td>18</td>
    </tr>
    <tr>
      <td></td>
      <td>rnaz</td>
      <td>rnaz, rnaz_annotate, rnaz_cluster, rnaz_randomize_aln, rnaz_select_seqs, rnaz_window</td>
      <td>RNAz is a program for predicting structurally conserved and thermodynamically stable RNA secondary structures in multiple sequence alignments.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://www.tbi.univie.ac.at/~wash/RNAz/</td>
      <td>RNA</td>
      <td>rnaz</td>
      <td>2016-10-22</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_team/rnaz</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/rnaz</td>
      <td>2.1.1</td>
      <td>rnaz</td>
      <td>2.1.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>1156</td>
      <td>43309</td>
      <td>93</td>
      <td>121</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1156</td>
      <td>43309</td>
      <td>93</td>
      <td>121</td>
    </tr>
    <tr>
      <td></td>
      <td>sailfish</td>
      <td>sailfish</td>
      <td>Sailfish is a tool for transcript quantification from RNA-seq data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.cs.cmu.edu/~ckingsf/software/sailfish/</td>
      <td>Sequence Analysis, RNA</td>
      <td>sailfish</td>
      <td>2015-09-05</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/sailfish</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/sailfish</td>
      <td>0.10.1.1</td>
      <td>bzip2</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>3628</td>
      <td>4168</td>
      <td>165</td>
      <td>199</td>
      <td>8399</td>
      <td>14473</td>
      <td>573</td>
      <td>1147</td>
      <td>2710</td>
      <td>3000</td>
      <td>151</td>
      <td>175</td>
      <td>14737</td>
      <td>21641</td>
      <td>889</td>
      <td>1521</td>
    </tr>
    <tr>
      <td></td>
      <td>salmon</td>
      <td>alevin, salmon, salmonquantmerge</td>
      <td>Salmon is a wicked-fast program to produce a highly-accurate, transcript-level quantification estimates from RNA-seq and single-cell data.</td>
      <td>salmon</td>
      <td>salmon</td>
      <td></td>
      <td>Salmon</td>
      <td>A tool for transcript expression quantification from RNA-seq data</td>
      <td>Sequence composition calculation, RNA-Seq quantification, Gene expression analysis</td>
      <td>RNA-Seq, Gene expression, Transcriptomics</td>
      <td>To update</td>
      <td>https://github.com/COMBINE-lab/salmon</td>
      <td>Sequence Analysis, RNA, Transcriptomics</td>
      <td></td>
      <td>2019-09-18</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/salmon</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/salmon</td>
      <td>1.10.1</td>
      <td>salmon</td>
      <td>1.10.3</td>
      <td>Sequence composition calculation, RNA-Seq quantification, Gene expression analysis</td>
      <td>RNA-Seq, Transcriptomics</td>
      <td>2</td>
      <td>2</td>
      <td>3</td>
      <td>1</td>
      <td>2</td>
      <td>2</td>
      <td>3</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>73085</td>
      <td>77081</td>
      <td>2243</td>
      <td>2319</td>
      <td>90634</td>
      <td>122835</td>
      <td>3402</td>
      <td>4726</td>
      <td>13371</td>
      <td>15818</td>
      <td>513</td>
      <td>586</td>
      <td>177090</td>
      <td>215734</td>
      <td>6158</td>
      <td>7631</td>
    </tr>
    <tr>
      <td></td>
      <td>sam2interval</td>
      <td>sam2interval</td>
      <td>Convert SAM to interval.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>SAM</td>
      <td>sam2interval</td>
      <td>2013-08-26</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/sam2interval</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/sam2interval</td>
      <td>1.0.2</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1155</td>
      <td>1251</td>
      <td>57</td>
      <td>63</td>
      <td>2264</td>
      <td>18187</td>
      <td>189</td>
      <td>1748</td>
      <td>525</td>
      <td>590</td>
      <td>10</td>
      <td>17</td>
      <td>3944</td>
      <td>20028</td>
      <td>256</td>
      <td>1828</td>
    </tr>
    <tr>
      <td></td>
      <td>sam_bitwise_flag_filter</td>
      <td>sam_bw_filter</td>
      <td>Filter SAM on bitwise flag values</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>SAM</td>
      <td>sam_bitwise_flag_filter</td>
      <td>2013-08-26</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/sam_bitwise_flag_filter</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/sam_bitwise_flag_filter</td>
      <td>1.0.1</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>96</td>
      <td>278</td>
      <td>12</td>
      <td>26</td>
      <td>3554</td>
      <td>76029</td>
      <td>450</td>
      <td>6346</td>
      <td>229</td>
      <td>345</td>
      <td>16</td>
      <td>29</td>
      <td>3879</td>
      <td>76652</td>
      <td>478</td>
      <td>6401</td>
    </tr>
    <tr>
      <td></td>
      <td>sambamba</td>
      <td>sambamba_flagstat, sambamba_markdup, sambamba_merge, sambamba_sort</td>
      <td>Sambamba: process your BAM data faster!</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/biod/sambamba</td>
      <td>SAM</td>
      <td>sambamba</td>
      <td>2016-06-23</td>
      <td>bgruening</td>
      <td>https://github.com/biod/sambamba</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/sambamba</td>
      <td>1.0.1</td>
      <td>sambamba</td>
      <td>1.0.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>297</td>
      <td>297</td>
      <td>51</td>
      <td>51</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>297</td>
      <td>297</td>
      <td>51</td>
      <td>51</td>
    </tr>
    <tr>
      <td></td>
      <td>sample_seqs</td>
      <td>sample_seqs</td>
      <td>Sub-sample sequences files (e.g. to reduce coverage)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/sample_seqs</td>
      <td>Assembly, Fasta Manipulation, Fastq Manipulation, Sequence Analysis</td>
      <td>sample_seqs</td>
      <td>2015-08-26</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/sample_seqs</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/sample_seqs</td>
      <td>0.2.6</td>
      <td>biopython</td>
      <td>1.70</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>3421</td>
      <td>3859</td>
      <td>340</td>
      <td>382</td>
      <td>883</td>
      <td>883</td>
      <td>109</td>
      <td>109</td>
      <td>1138</td>
      <td>1138</td>
      <td>15</td>
      <td>15</td>
      <td>5442</td>
      <td>5880</td>
      <td>464</td>
      <td>506</td>
    </tr>
    <tr>
      <td></td>
      <td>samtools_depad</td>
      <td>samtools_depad</td>
      <td>Re-align a SAM/BAM file with a padded reference (using samtools depad)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.htslib.org/</td>
      <td>Assembly, SAM, Sequence Analysis</td>
      <td>samtools_depad</td>
      <td>2014-04-14</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/samtools_depad</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/samtools_depad</td>
      <td>0.0.5</td>
      <td>samtools</td>
      <td>1.21</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>samtools_depth</td>
      <td>samtools_depth</td>
      <td>Coverage depth via samtools</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.htslib.org/</td>
      <td>Assembly, Sequence Analysis, SAM</td>
      <td>samtools_depth</td>
      <td>2014-11-19</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/samtools_depth</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/samtools_depth</td>
      <td>0.0.3</td>
      <td>samtools</td>
      <td>1.21</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>6485</td>
      <td>6549</td>
      <td>720</td>
      <td>726</td>
      <td>10127</td>
      <td>10127</td>
      <td>1518</td>
      <td>1518</td>
      <td>238</td>
      <td>238</td>
      <td>37</td>
      <td>37</td>
      <td>16850</td>
      <td>16914</td>
      <td>2275</td>
      <td>2281</td>
    </tr>
    <tr>
      <td></td>
      <td>samtools_idxstats</td>
      <td>samtools_idxstats</td>
      <td>BAM mapping statistics (using samtools idxstats)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.htslib.org/</td>
      <td>Assembly, Next Gen Mappers, SAM</td>
      <td>samtools_idxstats</td>
      <td>2013-11-11</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/samtools_idxstats</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/samtools_idxstats</td>
      <td>0.0.6</td>
      <td>samtools</td>
      <td>1.21</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>52099</td>
      <td>55110</td>
      <td>4053</td>
      <td>4497</td>
      <td>73194</td>
      <td>108684</td>
      <td>6930</td>
      <td>9936</td>
      <td>14771</td>
      <td>15714</td>
      <td>1222</td>
      <td>1419</td>
      <td>140064</td>
      <td>179508</td>
      <td>12205</td>
      <td>15852</td>
    </tr>
    <tr>
      <td></td>
      <td>scatterplot</td>
      <td>scatterplot_rpy</td>
      <td>Scatterplot of two numeric columns</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Graphics, Statistics</td>
      <td>scatterplot</td>
      <td>2014-07-28</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/scatterplot</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/scatterplot</td>
      <td>1.0.3</td>
      <td>numpy</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1234</td>
      <td>1752</td>
      <td>179</td>
      <td>261</td>
      <td>680</td>
      <td>9223</td>
      <td>231</td>
      <td>1927</td>
      <td>0</td>
      <td>49</td>
      <td>0</td>
      <td>18</td>
      <td>1914</td>
      <td>11024</td>
      <td>410</td>
      <td>2206</td>
    </tr>
    <tr>
      <td></td>
      <td>sed</td>
      <td>sed_stream_editor</td>
      <td>Manipulate your data with the sed command line tool.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/sed</td>
      <td>Text Manipulation</td>
      <td>sed_wrapper</td>
      <td>2015-02-28</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/sed</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/sed</td>
      <td>0.0.1</td>
      <td>sed</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>281</td>
      <td>2284</td>
      <td>10</td>
      <td>53</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>281</td>
      <td>2284</td>
      <td>10</td>
      <td>53</td>
    </tr>
    <tr>
      <td></td>
      <td>segemehl</td>
      <td>segemehl</td>
      <td>segemehl - short read mapping with gaps</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.bioinf.uni-leipzig.de/Software/segemehl/</td>
      <td>Next Gen Mappers</td>
      <td>segemehl</td>
      <td>2015-02-28</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/segemehl</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/segemehl</td>
      <td>0.2.0.4</td>
      <td>segemehl</td>
      <td>0.3.4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1095</td>
      <td>1306</td>
      <td>98</td>
      <td>113</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1095</td>
      <td>1306</td>
      <td>98</td>
      <td>113</td>
    </tr>
    <tr>
      <td></td>
      <td>selectsequencesfrommsa</td>
      <td>selectsequencesfrommsa</td>
      <td>SelectSequences - selects representative entries from a multiple sequence alignment in clustal format</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/eggzilla/SelectSequences</td>
      <td>RNA, Sequence Analysis</td>
      <td>selectsequencesfrommsa</td>
      <td>2017-03-11</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/selectsequencesfrommsa</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/selectsequencesfrommsa</td>
      <td>1.0.5</td>
      <td>selectsequencesfrommsa</td>
      <td>1.0.5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>305</td>
      <td>463</td>
      <td>43</td>
      <td>50</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>305</td>
      <td>463</td>
      <td>43</td>
      <td>50</td>
    </tr>
    <tr>
      <td></td>
      <td>seq_composition</td>
      <td>seq_composition</td>
      <td>Sequence composition</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_composition</td>
      <td>Sequence Analysis</td>
      <td>seq_composition</td>
      <td>2014-08-12</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_composition</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_composition</td>
      <td>0.0.5</td>
      <td>biopython</td>
      <td>1.70</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1053</td>
      <td>1186</td>
      <td>137</td>
      <td>195</td>
      <td>680</td>
      <td>680</td>
      <td>206</td>
      <td>206</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1733</td>
      <td>1866</td>
      <td>343</td>
      <td>401</td>
    </tr>
    <tr>
      <td></td>
      <td>seq_filter_by_id</td>
      <td>seq_filter_by_id</td>
      <td>Filter sequences by ID</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_filter_by_id</td>
      <td>Fasta Manipulation, Sequence Analysis, Text Manipulation</td>
      <td>seq_filter_by_id</td>
      <td>2016-05-17</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_filter_by_id</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_filter_by_id</td>
      <td>0.2.9</td>
      <td>biopython</td>
      <td>1.70</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>30300</td>
      <td>32553</td>
      <td>732</td>
      <td>799</td>
      <td>21726</td>
      <td>31891</td>
      <td>619</td>
      <td>621</td>
      <td>2886</td>
      <td>2886</td>
      <td>144</td>
      <td>144</td>
      <td>54912</td>
      <td>67330</td>
      <td>1495</td>
      <td>1564</td>
    </tr>
    <tr>
      <td></td>
      <td>seq_filter_by_mapping</td>
      <td>seq_filter_by_mapping</td>
      <td>Filter sequencing reads using SAM/BAM mapping files</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_filter_by_mapping</td>
      <td>Assembly, Fasta Manipulation, Fastq Manipulation, SAM, Sequence Analysis</td>
      <td>seq_filter_by_mapping</td>
      <td>2015-04-29</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_filter_by_mapping</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_filter_by_mapping</td>
      <td>0.0.8</td>
      <td>biopython</td>
      <td>1.70</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>4322</td>
      <td>4714</td>
      <td>180</td>
      <td>205</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4322</td>
      <td>4714</td>
      <td>180</td>
      <td>205</td>
    </tr>
    <tr>
      <td></td>
      <td>seq_length</td>
      <td>seq_length</td>
      <td>Compute sequence length (from FASTA, QUAL, FASTQ, SFF, etc)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_length</td>
      <td>Fasta Manipulation, Fastq Manipulation, Sequence Analysis</td>
      <td>seq_length</td>
      <td>2018-05-08</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_length</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_length</td>
      <td>0.0.5</td>
      <td>biopython</td>
      <td>1.70</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>seq_primer_clip</td>
      <td>seq_primer_clip</td>
      <td>Trim off 5' or 3' primers</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_primer_clip</td>
      <td>Assembly, Fasta Manipulation, Text Manipulation</td>
      <td>seq_primer_clip</td>
      <td>2015-04-28</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_primer_clip</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_primer_clip</td>
      <td>0.0.18</td>
      <td>galaxy_sequence_utils</td>
      <td>1.2.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>seq_rename</td>
      <td>seq_rename</td>
      <td>Rename sequences with ID mapping from a tabular file</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_rename</td>
      <td>Fasta Manipulation, Sequence Analysis, Text Manipulation</td>
      <td>seq_rename</td>
      <td>2014-12-02</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_rename</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_rename</td>
      <td>0.0.10</td>
      <td>galaxy_sequence_utils</td>
      <td>1.2.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>seq_select_by_id</td>
      <td>seq_select_by_id</td>
      <td>Select sequences by ID</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_select_by_id</td>
      <td>Fasta Manipulation, Sequence Analysis, Text Manipulation</td>
      <td>seq_select_by_id</td>
      <td>2015-04-24</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_select_by_id</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_select_by_id</td>
      <td>0.0.15</td>
      <td>biopython</td>
      <td>1.70</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2196</td>
      <td>3529</td>
      <td>260</td>
      <td>289</td>
      <td>2196</td>
      <td>3529</td>
      <td>260</td>
      <td>289</td>
    </tr>
    <tr>
      <td></td>
      <td>short_reads_figure_high_quality_length</td>
      <td>hist_high_quality_score</td>
      <td>Histogram of high quality score reads</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Graphics</td>
      <td>short_reads_figure_high_quality_length</td>
      <td>2014-05-19</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/short_reads_figure_high_quality_length</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/short_reads_figure_high_quality_length</td>
      <td>1.0.0</td>
      <td>rpy</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>short_reads_figure_score</td>
      <td>quality_score_distribution</td>
      <td>Build base quality distribution</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Graphics</td>
      <td>short_reads_figure_score</td>
      <td>2014-05-19</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/short_reads_figure_score</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/short_reads_figure_score</td>
      <td>1.0.2</td>
      <td>fontconfig</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>145</td>
      <td>163</td>
      <td>25</td>
      <td>30</td>
      <td>196</td>
      <td>3360</td>
      <td>23</td>
      <td>711</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>341</td>
      <td>3523</td>
      <td>48</td>
      <td>741</td>
    </tr>
    <tr>
      <td></td>
      <td>short_reads_trim_seq</td>
      <td>trim_reads</td>
      <td>Select high quality segments</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Fastq Manipulation</td>
      <td>short_reads_trim_seq</td>
      <td>2014-05-19</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/short_reads_trim_seq</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/short_reads_trim_seq</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>175</td>
      <td>220</td>
      <td>51</td>
      <td>60</td>
      <td>350</td>
      <td>6657</td>
      <td>85</td>
      <td>1602</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>525</td>
      <td>6877</td>
      <td>136</td>
      <td>1662</td>
    </tr>
    <tr>
      <td></td>
      <td>show_beginning</td>
      <td>Show beginning1</td>
      <td>Select lines from the beginning of a file.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>show_beginning</td>
      <td>2012-12-04</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/show_beginning</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/show_beginning</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>27232</td>
      <td>36366</td>
      <td>2184</td>
      <td>2789</td>
      <td>16648</td>
      <td>84460</td>
      <td>4191</td>
      <td>18767</td>
      <td>2785</td>
      <td>3276</td>
      <td>373</td>
      <td>452</td>
      <td>46665</td>
      <td>124102</td>
      <td>6748</td>
      <td>22008</td>
    </tr>
    <tr>
      <td></td>
      <td>show_tail</td>
      <td>Show tail1</td>
      <td>Select lines from the end of a file.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>show_tail</td>
      <td>2012-12-04</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/show_tail</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/show_tail</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1620</td>
      <td>2660</td>
      <td>378</td>
      <td>589</td>
      <td>10551</td>
      <td>24579</td>
      <td>682</td>
      <td>1851</td>
      <td>284</td>
      <td>398</td>
      <td>67</td>
      <td>83</td>
      <td>12455</td>
      <td>27637</td>
      <td>1127</td>
      <td>2523</td>
    </tr>
    <tr>
      <td></td>
      <td>sicer</td>
      <td>peakcalling_sicer</td>
      <td>Statistical approach for the Identification of ChIP-Enriched Regions</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://home.gwu.edu/~wpeng/Software.htm</td>
      <td>ChIP-seq</td>
      <td>sicer</td>
      <td>2014-01-27</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/sicer</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/sicer</td>
      <td>1.1</td>
      <td>SICER</td>
      <td>1.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>276</td>
      <td>379</td>
      <td>30</td>
      <td>42</td>
      <td>972</td>
      <td>9053</td>
      <td>146</td>
      <td>1008</td>
      <td>644</td>
      <td>689</td>
      <td>5</td>
      <td>7</td>
      <td>1892</td>
      <td>10121</td>
      <td>181</td>
      <td>1057</td>
    </tr>
    <tr>
      <td></td>
      <td>sklearn</td>
      <td>sklearn_mlxtend_association_rules, sklearn_clf_metrics, sklearn_discriminant_classifier, sklearn_ensemble, sklearn_estimator_attributes, sklearn_feature_selection, sklearn_fitted_model_eval, sklearn_generalized_linear, keras_batch_models, keras_model_builder, keras_model_config, keras_train_and_eval, sklearn_label_encoder, sklearn_lightgbm, ml_visualization_ex, model_prediction, sklearn_model_validation, sklearn_nn_classifier, sklearn_numeric_clustering, sklearn_pairwise_metrics, sklearn_pca, sklearn_build_pipeline, sklearn_data_preprocess, sklearn_regression_metrics, sklearn_sample_generator, sklearn_searchcv, sklearn_model_fit, scipy_sparse, stacking_ensemble_models, sklearn_svm_classifier, sklearn_to_categorical, sklearn_train_test_eval, sklearn_train_test_split</td>
      <td>Machine Learning tool suite from Scikit-learn</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://scikit-learn.org</td>
      <td>Machine Learning, Statistics</td>
      <td>sklearn</td>
      <td>2021-04-12</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/sklearn</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/sklearn</td>
      <td>1.0.11.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>30</td>
      <td>14</td>
      <td>31</td>
      <td>16</td>
      <td>30</td>
      <td>14</td>
      <td>31</td>
      <td>16</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>29</td>
      <td>31</td>
      <td>27</td>
      <td>0</td>
      <td>82169</td>
      <td>116810</td>
      <td>2402</td>
      <td>2666</td>
      <td>18101</td>
      <td>18101</td>
      <td>2375</td>
      <td>2375</td>
      <td>17549</td>
      <td>17549</td>
      <td>321</td>
      <td>321</td>
      <td>117819</td>
      <td>152460</td>
      <td>5098</td>
      <td>5362</td>
    </tr>
    <tr>
      <td></td>
      <td>snpfreq</td>
      <td>hgv_snpFreq</td>
      <td>snpFreq significant SNPs in case-control data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Variant Analysis, Statistics</td>
      <td>snpfreq</td>
      <td>2014-07-28</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/hgv/snpfreq</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/hgv/snpfreq</td>
      <td>1.0.1</td>
      <td>R</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>35</td>
      <td>44</td>
      <td>10</td>
      <td>12</td>
      <td>307</td>
      <td>649</td>
      <td>68</td>
      <td>181</td>
      <td>227</td>
      <td>260</td>
      <td>16</td>
      <td>23</td>
      <td>569</td>
      <td>953</td>
      <td>94</td>
      <td>216</td>
    </tr>
    <tr>
      <td></td>
      <td>sortmerna</td>
      <td>bg_sortmerna</td>
      <td>SortMeRNA is a software designed to rapidly filter ribosomal RNA fragments from metatransriptomic data produced by next-generation sequencers.</td>
      <td>sortmerna</td>
      <td>sortmerna</td>
      <td></td>
      <td>SortMeRNA</td>
      <td>Sequence analysis tool for filtering, mapping and OTU-picking NGS reads.</td>
      <td>Sequence similarity search, Sequence comparison, Sequence alignment analysis</td>
      <td>Metatranscriptomics, Metagenomics</td>
      <td>To update</td>
      <td>http://bioinfo.lifl.fr/RNA/sortmerna/</td>
      <td>RNA</td>
      <td>sortmerna</td>
      <td>2016-03-17</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/sortmerna</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/sortmerna</td>
      <td>4.3.6</td>
      <td>sortmerna</td>
      <td>4.3.7</td>
      <td>Sequence similarity search, Sequence alignment analysis</td>
      <td>Metatranscriptomics, Metagenomics</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>19865</td>
      <td>21034</td>
      <td>1047</td>
      <td>1126</td>
      <td>3141</td>
      <td>3141</td>
      <td>328</td>
      <td>328</td>
      <td>3304</td>
      <td>3304</td>
      <td>189</td>
      <td>189</td>
      <td>26310</td>
      <td>27479</td>
      <td>1564</td>
      <td>1643</td>
    </tr>
    <tr>
      <td></td>
      <td>split_file_on_column</td>
      <td>tp_split_on_column</td>
      <td>Split a file on a specific column.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>split_file_on_column</td>
      <td>2015-02-28</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/text_processing/split_file_on_column</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/text_processing/split_file_on_column</td>
      <td>0.6</td>
      <td>gawk</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>5995</td>
      <td>6261</td>
      <td>408</td>
      <td>453</td>
      <td>2150</td>
      <td>2150</td>
      <td>311</td>
      <td>311</td>
      <td>26</td>
      <td>26</td>
      <td>2</td>
      <td>2</td>
      <td>8171</td>
      <td>8437</td>
      <td>721</td>
      <td>766</td>
    </tr>
    <tr>
      <td></td>
      <td>split_file_to_collection</td>
      <td>split_file_to_collection</td>
      <td>Split tabular, MGF, FASTA, or FASTQ files to a dataset collection.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/text_processing/split_file_to_collection</td>
      <td>Text Manipulation</td>
      <td>split_file_to_collection</td>
      <td>2019-08-28</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/text_processing/split_file_to_collection</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/text_processing/split_file_to_collection</td>
      <td>0.5.2</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>16415</td>
      <td>16623</td>
      <td>968</td>
      <td>984</td>
      <td>9133</td>
      <td>9133</td>
      <td>672</td>
      <td>672</td>
      <td>4140</td>
      <td>4140</td>
      <td>148</td>
      <td>148</td>
      <td>29688</td>
      <td>29896</td>
      <td>1788</td>
      <td>1804</td>
    </tr>
    <tr>
      <td></td>
      <td>split_paired_reads</td>
      <td>split_paired_reads</td>
      <td>Split paired end reads</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Fastq Manipulation</td>
      <td>split_paired_reads</td>
      <td>2013-09-25</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/split_paired_reads</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/split_paired_reads</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>13</td>
      <td>0</td>
      <td>3</td>
      <td>200</td>
      <td>384</td>
      <td>31</td>
      <td>66</td>
      <td>200</td>
      <td>397</td>
      <td>31</td>
      <td>69</td>
    </tr>
    <tr>
      <td></td>
      <td>splitfasta</td>
      <td>rbc_splitfasta</td>
      <td>Split a multi-sequence fasta file into files containing single sequences</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>splitfasta</td>
      <td>2015-10-16</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/splitfasta</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/splitfasta</td>
      <td>0.4.0</td>
      <td>biopython</td>
      <td>1.70</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1772</td>
      <td>1811</td>
      <td>278</td>
      <td>294</td>
      <td>3012</td>
      <td>3012</td>
      <td>564</td>
      <td>564</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4784</td>
      <td>4823</td>
      <td>842</td>
      <td>858</td>
    </tr>
    <tr>
      <td></td>
      <td>sshmm</td>
      <td>sshmm</td>
      <td>ssHMM is an RNA sequence-structure motif finder for RNA-binding protein data, such as CLIP-Seq data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.molgen.mpg.de/heller/ssHMM</td>
      <td>Sequence Analysis, RNA</td>
      <td>sshmm</td>
      <td>2018-07-05</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/sshmm/</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/sshmm</td>
      <td>1.0.7</td>
      <td>sshmm</td>
      <td>1.0.7</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>214</td>
      <td>225</td>
      <td>17</td>
      <td>18</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>214</td>
      <td>225</td>
      <td>17</td>
      <td>18</td>
    </tr>
    <tr>
      <td></td>
      <td>statistics</td>
      <td>bg_statistical_hypothesis_testing</td>
      <td>Tool for computing statistical tests.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/statistics</td>
      <td>Statistics</td>
      <td>bg_statistical_hypothesis_testing</td>
      <td>2015-02-28</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/statistics</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/statistics</td>
      <td>0.3</td>
      <td>numpy</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>169</td>
      <td>346</td>
      <td>46</td>
      <td>61</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>169</td>
      <td>346</td>
      <td>46</td>
      <td>61</td>
    </tr>
    <tr>
      <td></td>
      <td>stress_ng</td>
      <td>stress_ng</td>
      <td>stress test a computer system in various selectable ways</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Web Services</td>
      <td>stress_ng</td>
      <td>2021-03-04</td>
      <td>bgruening-util</td>
      <td>https://github.com/ColinIanKing/stress-ng</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/stress_ng</td>
      <td>0.12.04</td>
      <td>stress-ng</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>62</td>
      <td>62</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>62</td>
      <td>62</td>
      <td>3</td>
      <td>3</td>
    </tr>
    <tr>
      <td></td>
      <td>substitution_rates</td>
      <td>subRate1</td>
      <td>Estimate substitution rates for non-coding regions</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>substitution_rates</td>
      <td>2014-04-01</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/substitution_rates</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/substitution_rates</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>121</td>
      <td>3901</td>
      <td>34</td>
      <td>747</td>
      <td>105</td>
      <td>114</td>
      <td>4</td>
      <td>4</td>
      <td>226</td>
      <td>4015</td>
      <td>38</td>
      <td>751</td>
    </tr>
    <tr>
      <td></td>
      <td>substitutions</td>
      <td>substitutions1</td>
      <td>Fetch substitutions from pairwise alignments</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>substitutions</td>
      <td>2014-04-01</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/substitutions</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/substitutions</td>
      <td>1.0.1</td>
      <td>bx-python</td>
      <td>0.13.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>93</td>
      <td>901</td>
      <td>13</td>
      <td>84</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>93</td>
      <td>901</td>
      <td>13</td>
      <td>84</td>
    </tr>
    <tr>
      <td></td>
      <td>subtract</td>
      <td>gops_subtract_1</td>
      <td>Subtract the intervals of two datasets</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/gops</td>
      <td>Genomic Interval Operations</td>
      <td>subtract</td>
      <td>2014-04-01</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/gops/subtract</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/gops/subtract</td>
      <td>1.0.0</td>
      <td>bx-python</td>
      <td>0.13.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>566512</td>
      <td>567199</td>
      <td>410</td>
      <td>470</td>
      <td>49985</td>
      <td>184650</td>
      <td>1473</td>
      <td>7631</td>
      <td>20648</td>
      <td>20897</td>
      <td>155</td>
      <td>188</td>
      <td>637145</td>
      <td>772746</td>
      <td>2038</td>
      <td>8289</td>
    </tr>
    <tr>
      <td></td>
      <td>subtract_query</td>
      <td>subtract_query1</td>
      <td>Subtract Whole Dataset from another dataset</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/gops</td>
      <td>Genomic Interval Operations</td>
      <td>subtract_query</td>
      <td>2014-04-01</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/gops/subtract_query</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/gops/subtract_query</td>
      <td>0.1</td>
      <td>bx-python</td>
      <td>0.13.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1178</td>
      <td>1450</td>
      <td>124</td>
      <td>141</td>
      <td>3169</td>
      <td>40539</td>
      <td>636</td>
      <td>3435</td>
      <td>799</td>
      <td>950</td>
      <td>26</td>
      <td>38</td>
      <td>5146</td>
      <td>42939</td>
      <td>786</td>
      <td>3614</td>
    </tr>
    <tr>
      <td></td>
      <td>t2ps</td>
      <td>Draw_phylogram</td>
      <td>Draw phylogeny</td>
      <td>t2ps</td>
      <td>t2ps</td>
      <td></td>
      <td>Draw phylogeny</td>
      <td>"Given taxonomy representation (produced by *Fetch taxonomic representation* tool) this utility produces a graphical representations of phylogenetic tree in PDF format." - Galaxy tool wrapper</td>
      <td>Phylogenetic tree visualisation</td>
      <td>Phylogenomics</td>
      <td>To update</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>Metagenomics</td>
      <td>t2ps</td>
      <td>2014-01-27</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/taxonomy/t2ps</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/taxonomy/t2ps</td>
      <td>1.0.0</td>
      <td>taxonomy</td>
      <td>0.10.0</td>
      <td>Phylogenetic tree visualisation</td>
      <td>Phylogenomics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>411</td>
      <td>472</td>
      <td>68</td>
      <td>90</td>
      <td>102</td>
      <td>7952</td>
      <td>13</td>
      <td>1240</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>513</td>
      <td>8424</td>
      <td>81</td>
      <td>1330</td>
    </tr>
    <tr>
      <td></td>
      <td>t2t_report</td>
      <td>t2t_report</td>
      <td>Summarize taxonomy</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>Metagenomics</td>
      <td>t2t_report</td>
      <td>2014-01-27</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/taxonomy/t2t_report</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/taxonomy/t2t_report</td>
      <td>1.0.0</td>
      <td>taxonomy</td>
      <td>0.10.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>927</td>
      <td>959</td>
      <td>65</td>
      <td>78</td>
      <td>130</td>
      <td>7194</td>
      <td>18</td>
      <td>1143</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1057</td>
      <td>8153</td>
      <td>83</td>
      <td>1221</td>
    </tr>
    <tr>
      <td></td>
      <td>t_test_two_samples</td>
      <td>t_test_two_samples</td>
      <td>T Test for Two Samples</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>t_test_two_samples</td>
      <td>2013-09-25</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/t_test_two_samples</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/t_test_two_samples</td>
      <td>1.0.1</td>
      <td>R</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>141</td>
      <td>215</td>
      <td>20</td>
      <td>31</td>
      <td>520</td>
      <td>1166</td>
      <td>79</td>
      <td>274</td>
      <td>1025</td>
      <td>1118</td>
      <td>11</td>
      <td>12</td>
      <td>1686</td>
      <td>2499</td>
      <td>110</td>
      <td>317</td>
    </tr>
    <tr>
      <td></td>
      <td>table_annovar</td>
      <td>table_annovar</td>
      <td>Annotate a VCF file using ANNOVAR annotations to produce a tabular file that can be filtered</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Variant Analysis</td>
      <td>table_annovar</td>
      <td>2013-12-02</td>
      <td>devteam</td>
      <td>Nonehttps://github.com/galaxyproject/tools-devteam/tree/master/tools/table_annovar</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/table_annovar</td>
      <td>0.2</td>
      <td>annovar</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>16526</td>
      <td>64515</td>
      <td>2803</td>
      <td>7456</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>16526</td>
      <td>64515</td>
      <td>2803</td>
      <td>7456</td>
    </tr>
    <tr>
      <td></td>
      <td>tables_arithmetic_operations</td>
      <td>tables_arithmetic_operations</td>
      <td>Arithmetic Operations on tables</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/gops</td>
      <td>Genomic Interval Operations</td>
      <td>tables_arithmetic_operations</td>
      <td>2014-04-01</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/gops/tables_arithmetic_operations</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/gops/tables_arithmetic_operations</td>
      <td>1.0.0</td>
      <td>perl</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2188</td>
      <td>2220</td>
      <td>33</td>
      <td>38</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2188</td>
      <td>2220</td>
      <td>33</td>
      <td>38</td>
    </tr>
    <tr>
      <td></td>
      <td>tabular_to_fasta</td>
      <td>tab2fasta</td>
      <td>Tabular-to-FASTA</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Convert Formats</td>
      <td>tabular_to_fasta</td>
      <td>2014-05-19</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/tabular_to_fasta</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/tabular_to_fasta</td>
      <td>1.1.1</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>329032</td>
      <td>333057</td>
      <td>1489</td>
      <td>1702</td>
      <td>79033</td>
      <td>215770</td>
      <td>2441</td>
      <td>8587</td>
      <td>27876</td>
      <td>33727</td>
      <td>296</td>
      <td>360</td>
      <td>435941</td>
      <td>582554</td>
      <td>4226</td>
      <td>10649</td>
    </tr>
    <tr>
      <td></td>
      <td>tapscan</td>
      <td>tapscan_classify</td>
      <td>Search for transcription associated proteins (TAPs)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://plantcode.cup.uni-freiburg.de/tapscan/</td>
      <td>Proteomics</td>
      <td>tapscan</td>
      <td>2024-02-14</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/tapscan</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/tapscan</td>
      <td>4.76+galaxy0</td>
      <td>hmmer</td>
      <td>3.4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>19</td>
      <td>19</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>19</td>
      <td>19</td>
      <td>6</td>
      <td>6</td>
    </tr>
    <tr>
      <td></td>
      <td>targetfinder</td>
      <td>targetfinder</td>
      <td>Plant small RNA target prediction tool</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/carringtonlab/TargetFinder.git</td>
      <td>RNA</td>
      <td>targetfinder</td>
      <td>2015-10-23</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/targetfinder/</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/targetfinder</td>
      <td>1.7</td>
      <td>targetfinder</td>
      <td>1.7</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>827</td>
      <td>838</td>
      <td>89</td>
      <td>92</td>
      <td>114</td>
      <td>114</td>
      <td>41</td>
      <td>41</td>
      <td>900</td>
      <td>900</td>
      <td>29</td>
      <td>29</td>
      <td>1841</td>
      <td>1852</td>
      <td>159</td>
      <td>162</td>
    </tr>
    <tr>
      <td></td>
      <td>text_processing</td>
      <td>tp_awk_tool, tp_cat, tp_cut_tool, tp_easyjoin_tool, tp_find_and_replace, tp_grep_tool, tp_head_tool, tp_multijoin_tool, nl, tp_text_file_with_recurring_lines, tp_replace_in_column, tp_replace_in_line, tp_sed_tool, tp_sort_header_tool, tp_sort_rows, tp_uniq_tool, tp_tac, tp_tail_tool, tp_unfold_column_tool, tp_sorted_uniq</td>
      <td>High performance text processing tools using the GNU coreutils, sed, awk and friends.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.gnu.org/software/</td>
      <td>Text Manipulation</td>
      <td>text_processing</td>
      <td>2019-01-12</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/text_processing/text_processing</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/text_processing/text_processing</td>
      <td>9.3</td>
      <td>coreutils</td>
      <td>8.25</td>
      <td></td>
      <td></td>
      <td>20</td>
      <td>20</td>
      <td>20</td>
      <td>20</td>
      <td>20</td>
      <td>20</td>
      <td>20</td>
      <td>20</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>19</td>
      <td>1</td>
      <td>19</td>
      <td>0</td>
      <td>0</td>
      <td>19</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>20</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>19</td>
      <td>19</td>
      <td>19</td>
      <td>0</td>
      <td>3898014</td>
      <td>4015573</td>
      <td>37698</td>
      <td>42095</td>
      <td>648268</td>
      <td>915282</td>
      <td>47636</td>
      <td>69828</td>
      <td>103365</td>
      <td>105214</td>
      <td>7778</td>
      <td>7953</td>
      <td>4649647</td>
      <td>5036069</td>
      <td>93112</td>
      <td>119876</td>
    </tr>
    <tr>
      <td></td>
      <td>tgsgapcloser</td>
      <td>tgsgapcloser</td>
      <td>TGS-GapCloser uses error-prone long reads or preassembled contigs to fill N-gap in the genome assembly.</td>
      <td>TGS-GapCloser</td>
      <td>TGS-GapCloser</td>
      <td></td>
      <td>TGS-GapCloser</td>
      <td>TGS-GapCloser is a fast and accurately passing through the Bermuda in large genome using error-prone third-generation long reads.</td>
      <td>Genome assembly, Read mapping, Scaffolding, Localised reassembly</td>
      <td>Sequencing, Sequence assembly, Phylogeny, Transcription factors and regulatory sites, Mapping</td>
      <td>To update</td>
      <td>https://github.com/BGI-Qingdao/TGS-GapCloser</td>
      <td>Assembly</td>
      <td>tgsgapcloser</td>
      <td>2021-11-14</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/tgsgapcloser</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/tgsgapcloser</td>
      <td>1.0.3</td>
      <td>tgsgapcloser</td>
      <td>1.2.1</td>
      <td>Genome assembly, Read mapping, Scaffolding, Localised reassembly</td>
      <td>Sequencing, Sequence assembly, Phylogeny, Transcription factors and regulatory sites, Mapping</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>547</td>
      <td>547</td>
      <td>51</td>
      <td>51</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>547</td>
      <td>547</td>
      <td>51</td>
      <td>51</td>
    </tr>
    <tr>
      <td></td>
      <td>tiara</td>
      <td>tiara</td>
      <td>Tool for identification of eukaryotic sequences in the metagenomic datasets.</td>
      <td>Tiara</td>
      <td>Tiara</td>
      <td></td>
      <td>TIARA</td>
      <td>Total Integrated Archive of Short-Read and Array (TIARA) database, contains personal genomic information obtained from next generation sequencing (NGS) techniques and ultra-high-resolution comparative genomic hybridization (CGH) arrays. This database improves the accuracy of detecting personal genomic variations, such as SNPs, short indels and structural variants (SVs).</td>
      <td>Genome comparison, Data retrieval, Variant calling, Genome visualisation, Structural variation detection</td>
      <td>Sequencing, Genomics, DNA polymorphism, DNA structural variation</td>
      <td>To update</td>
      <td>https://github.com/ibe-uw/tiara</td>
      <td>Metagenomics, Sequence Analysis</td>
      <td>tiara</td>
      <td>2024-05-23</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/tiara</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/tiara</td>
      <td>1.0.3</td>
      <td>tiara</td>
      <td></td>
      <td>Genome comparison, Data retrieval, Variant calling, Genome visualisation, Structural variation detection</td>
      <td>Sequencing, Genomics, DNA polymorphism</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>15</td>
      <td>15</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>15</td>
      <td>15</td>
      <td>8</td>
      <td>8</td>
    </tr>
    <tr>
      <td></td>
      <td>tool_recommendation_model</td>
      <td>create_tool_recommendation_model</td>
      <td>Create model to recommend tools</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bgruening/galaxytools</td>
      <td>Machine Learning</td>
      <td>create_tool_recommendation_model</td>
      <td>2019-08-23</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/recommendation_training/tools/tool_recommendation_model</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/tool_recommendation_model</td>
      <td>0.0.5</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>445</td>
      <td>445</td>
      <td>25</td>
      <td>25</td>
      <td>71</td>
      <td>71</td>
      <td>32</td>
      <td>32</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>516</td>
      <td>516</td>
      <td>57</td>
      <td>57</td>
    </tr>
    <tr>
      <td></td>
      <td>tophat</td>
      <td>tophat</td>
      <td>Tophat for Illumina</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA, Next Gen Mappers</td>
      <td>tophat</td>
      <td>2014-01-27</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/tophat</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/tophat</td>
      <td>1.5.0</td>
      <td>samtools</td>
      <td>1.21</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>158</td>
      <td>66524</td>
      <td>28</td>
      <td>7228</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>158</td>
      <td>66525</td>
      <td>28</td>
      <td>7229</td>
    </tr>
    <tr>
      <td></td>
      <td>tophat2</td>
      <td>tophat2</td>
      <td>Tophat - fast splice junction mapper for RNA-Seq reads</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://ccb.jhu.edu/software/tophat/index.shtml</td>
      <td>RNA, Next Gen Mappers</td>
      <td>tophat2</td>
      <td>2014-10-15</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/tophat2</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/tophat2</td>
      <td>2.1.1</td>
      <td>bowtie2</td>
      <td>2.5.4</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>15496</td>
      <td>26623</td>
      <td>1076</td>
      <td>1664</td>
      <td>83063</td>
      <td>343017</td>
      <td>8372</td>
      <td>27829</td>
      <td>5388</td>
      <td>14057</td>
      <td>508</td>
      <td>1012</td>
      <td>103947</td>
      <td>383697</td>
      <td>9956</td>
      <td>30505</td>
    </tr>
    <tr>
      <td></td>
      <td>tophat_fusion_post</td>
      <td>tophat_fusion_post</td>
      <td>Wrapper for Tophat-Fusion post step</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics</td>
      <td>tophat_fusion_post</td>
      <td>2014-01-27</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/tophat_fusion_post</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/tophat_fusion_post</td>
      <td>0.1</td>
      <td>blast+</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>220</td>
      <td>242</td>
      <td>41</td>
      <td>46</td>
      <td>6</td>
      <td>814</td>
      <td>2</td>
      <td>191</td>
      <td>100</td>
      <td>120</td>
      <td>8</td>
      <td>12</td>
      <td>326</td>
      <td>1176</td>
      <td>51</td>
      <td>249</td>
    </tr>
    <tr>
      <td></td>
      <td>trim_galore</td>
      <td>trim_galore</td>
      <td>Trim Galore adaptive quality and adapter trimmer</td>
      <td>trim_galore</td>
      <td>trim_galore</td>
      <td></td>
      <td>Trim Galore</td>
      <td>A wrapper tool around Cutadapt and FastQC to consistently apply quality and adapter trimming to FastQ files, with some extra functionality for MspI-digested RRBS-type (Reduced Representation Bisufite-Seq) libraries.</td>
      <td>Sequence trimming, Primer removal, Read pre-processing</td>
      <td>Sequence analysis</td>
      <td>To update</td>
      <td>http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/</td>
      <td>Sequence Analysis, Fastq Manipulation</td>
      <td>trim_galore</td>
      <td>2015-04-22</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/trim_galore</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/trim_galore</td>
      <td>0.6.7</td>
      <td>trim-galore</td>
      <td>0.6.10</td>
      <td>Sequence trimming, Primer removal, Read pre-processing</td>
      <td>Sequence analysis</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>229489</td>
      <td>275008</td>
      <td>5733</td>
      <td>6500</td>
      <td>228708</td>
      <td>324716</td>
      <td>11708</td>
      <td>16638</td>
      <td>22644</td>
      <td>23869</td>
      <td>1439</td>
      <td>1527</td>
      <td>480841</td>
      <td>623593</td>
      <td>18880</td>
      <td>24665</td>
    </tr>
    <tr>
      <td></td>
      <td>trimmer</td>
      <td>trimmer</td>
      <td>Trim leading or trailing characters.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>trimmer</td>
      <td>2012-12-04</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/trimmer</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/trimmer</td>
      <td>0.0.1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>5304</td>
      <td>7438</td>
      <td>447</td>
      <td>512</td>
      <td>15698</td>
      <td>108672</td>
      <td>658</td>
      <td>2946</td>
      <td>266</td>
      <td>822</td>
      <td>53</td>
      <td>81</td>
      <td>21268</td>
      <td>116932</td>
      <td>1158</td>
      <td>3539</td>
    </tr>
    <tr>
      <td></td>
      <td>trna_prediction</td>
      <td>aragorn_trna, trnascan</td>
      <td>Aragorn predicts tRNA and tmRNA in nucleotide sequences.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://mbioserv2.mbioekol.lu.se/ARAGORN/</td>
      <td>RNA</td>
      <td>trna_prediction</td>
      <td>2015-05-08</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/trna_prediction</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/trna_prediction</td>
      <td>0.6</td>
      <td>aragorn</td>
      <td>1.2.41</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>4361</td>
      <td>4601</td>
      <td>741</td>
      <td>807</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2469</td>
      <td>2469</td>
      <td>190</td>
      <td>190</td>
      <td>6830</td>
      <td>7070</td>
      <td>931</td>
      <td>997</td>
    </tr>
    <tr>
      <td></td>
      <td>ucsc_custom_track</td>
      <td>build_ucsc_custom_track_1</td>
      <td>Build custom track for UCSC genome browser</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>ucsc_custom_track</td>
      <td>2014-07-28</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/ucsc_custom_track</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/ucsc_custom_track</td>
      <td>1.0.1</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>373</td>
      <td>463</td>
      <td>134</td>
      <td>151</td>
      <td>2249</td>
      <td>18611</td>
      <td>638</td>
      <td>4080</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2622</td>
      <td>19074</td>
      <td>772</td>
      <td>4231</td>
    </tr>
    <tr>
      <td></td>
      <td>uniprot_rest_interface</td>
      <td>uniprot</td>
      <td>UniProt ID mapping and sequence retrieval</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/jdrudolph/uniprot</td>
      <td>Proteomics, Sequence Analysis</td>
      <td>uniprot_rest_interface</td>
      <td>2015-10-09</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/uniprot_rest_interface</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/uniprot_rest_interface</td>
      <td>0.6</td>
      <td>requests</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2117</td>
      <td>2725</td>
      <td>341</td>
      <td>412</td>
      <td>3111</td>
      <td>5779</td>
      <td>594</td>
      <td>1011</td>
      <td>806</td>
      <td>806</td>
      <td>50</td>
      <td>50</td>
      <td>6034</td>
      <td>9310</td>
      <td>985</td>
      <td>1473</td>
    </tr>
    <tr>
      <td></td>
      <td>varscan_version_2</td>
      <td>varscan</td>
      <td>VarScan wrapper</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://dkoboldt.github.io/varscan/</td>
      <td>Variant Analysis</td>
      <td>varscan_version_2</td>
      <td>2013-11-17</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/varscan_version_2</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/varscan_version_2</td>
      <td>2.4.2</td>
      <td>varscan</td>
      <td>2.4.6</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>5058</td>
      <td>6139</td>
      <td>606</td>
      <td>699</td>
      <td>17232</td>
      <td>53268</td>
      <td>2965</td>
      <td>5686</td>
      <td>2298</td>
      <td>2656</td>
      <td>424</td>
      <td>482</td>
      <td>24588</td>
      <td>62063</td>
      <td>3995</td>
      <td>6867</td>
    </tr>
    <tr>
      <td></td>
      <td>vcf2pgsnp</td>
      <td>vcf2pgSnp</td>
      <td>VCF to pgSnp</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Variant Analysis</td>
      <td>vcf2pgsnp</td>
      <td>2014-07-28</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/vcf2pgsnp</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/vcf2pgsnp</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>191</td>
      <td>221</td>
      <td>57</td>
      <td>65</td>
      <td>1399</td>
      <td>2700</td>
      <td>209</td>
      <td>584</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1590</td>
      <td>2921</td>
      <td>266</td>
      <td>649</td>
    </tr>
    <tr>
      <td></td>
      <td>vcf_annotate</td>
      <td>vcf_annotate</td>
      <td>Annotate a VCF file (dbSNP, hapmap)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Variant Analysis</td>
      <td>vcf_annotate</td>
      <td>2014-01-27</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/vcf_annotate</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/vcf_annotate</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>263</td>
      <td>340</td>
      <td>31</td>
      <td>54</td>
      <td>263</td>
      <td>342</td>
      <td>31</td>
      <td>56</td>
    </tr>
    <tr>
      <td></td>
      <td>vcf_extract</td>
      <td>vcf_extract</td>
      <td>Extract reads from a specified region</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Variant Analysis</td>
      <td>vcf_extract</td>
      <td>2014-01-27</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/vcf_extract</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/vcf_extract</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>513</td>
      <td>569</td>
      <td>6</td>
      <td>9</td>
      <td>513</td>
      <td>569</td>
      <td>6</td>
      <td>9</td>
    </tr>
    <tr>
      <td></td>
      <td>vcf_filter</td>
      <td>vcf_filter</td>
      <td>Filter a VCF file</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Variant Analysis</td>
      <td>vcf_filter</td>
      <td>2014-01-27</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/vcf_filter</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/vcf_filter</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>400</td>
      <td>455</td>
      <td>129</td>
      <td>140</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>238</td>
      <td>606</td>
      <td>19</td>
      <td>56</td>
      <td>638</td>
      <td>1061</td>
      <td>148</td>
      <td>196</td>
    </tr>
    <tr>
      <td></td>
      <td>vcf_intersect</td>
      <td>vcf_intersect</td>
      <td>Generate the intersection of two VCF files</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Variant Analysis</td>
      <td>vcf_intersect</td>
      <td>2014-01-27</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/vcf_intersect</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/vcf_intersect</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>19</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>335</td>
      <td>463</td>
      <td>19</td>
      <td>28</td>
      <td>335</td>
      <td>482</td>
      <td>19</td>
      <td>31</td>
    </tr>
    <tr>
      <td></td>
      <td>vcftools_annotate</td>
      <td>vcftools_annotate</td>
      <td>Annotate VCF using custom/user-defined annotations</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://vcftools.github.io/</td>
      <td>Variant Analysis</td>
      <td>vcftools_annotate</td>
      <td>2013-11-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/vcftools/vcftools_annotate</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/vcftools/vcftools_annotate</td>
      <td>0.1</td>
      <td>echo</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>155</td>
      <td>192</td>
      <td>81</td>
      <td>90</td>
      <td>1853</td>
      <td>3943</td>
      <td>581</td>
      <td>1280</td>
      <td>144</td>
      <td>156</td>
      <td>13</td>
      <td>16</td>
      <td>2152</td>
      <td>4291</td>
      <td>675</td>
      <td>1386</td>
    </tr>
    <tr>
      <td></td>
      <td>vcftools_compare</td>
      <td>vcftools_compare</td>
      <td>Compare VCF files to get overlap and uniqueness statistics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://vcftools.github.io/</td>
      <td>Variant Analysis</td>
      <td>vcftools_compare</td>
      <td>2013-11-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/vcftools/vcftools_compare</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/vcftools/vcftools_compare</td>
      <td>0.1</td>
      <td>tabix</td>
      <td>1.11</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>440</td>
      <td>594</td>
      <td>127</td>
      <td>142</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>16</td>
      <td>0</td>
      <td>12</td>
      <td>440</td>
      <td>610</td>
      <td>127</td>
      <td>154</td>
    </tr>
    <tr>
      <td></td>
      <td>vcftools_consensus</td>
      <td>vcftools_consensus</td>
      <td>Apply VCF variants to a fasta file to create consensus sequence</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://vcftools.github.io/</td>
      <td>Variant Analysis</td>
      <td>vcftools_consensus</td>
      <td>2016-10-16</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/vcftools/vcftools_consensus</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/vcftools/vcftools_consensus</td>
      <td>0.1.11</td>
      <td>samtools</td>
      <td>1.21</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>vcftools_isec</td>
      <td>vcftools_isec</td>
      <td>Intersect multiple VCF datasets</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://vcftools.github.io/</td>
      <td>Variant Analysis</td>
      <td>vcftools_isec</td>
      <td>2013-11-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/vcftools/vcftools_isec</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/vcftools/vcftools_isec</td>
      <td>0.1.1</td>
      <td>tabix</td>
      <td>1.11</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>379</td>
      <td>537</td>
      <td>104</td>
      <td>118</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>39</td>
      <td>496</td>
      <td>28</td>
      <td>48</td>
      <td>418</td>
      <td>1034</td>
      <td>132</td>
      <td>167</td>
    </tr>
    <tr>
      <td></td>
      <td>vcftools_merge</td>
      <td>vcftools_merge</td>
      <td>Merge multiple VCF datasets into a single dataset</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://vcftools.github.io/</td>
      <td>Variant Analysis</td>
      <td>vcftools_merge</td>
      <td>2013-11-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/vcftools/vcftools_merge</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/vcftools/vcftools_merge</td>
      <td>0.1.11</td>
      <td>tabix</td>
      <td>1.11</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>817</td>
      <td>1084</td>
      <td>244</td>
      <td>273</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>93</td>
      <td>448</td>
      <td>44</td>
      <td>60</td>
      <td>910</td>
      <td>1532</td>
      <td>288</td>
      <td>333</td>
    </tr>
    <tr>
      <td></td>
      <td>vcftools_slice</td>
      <td>vcftools_slice</td>
      <td>Subset VCF dataset by genomic regions</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://vcftools.github.io/</td>
      <td>Variant Analysis</td>
      <td>vcftools_slice</td>
      <td>2013-11-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/vcftools/vcftools_slice</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/vcftools/vcftools_slice</td>
      <td>0.1</td>
      <td>echo</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>410</td>
      <td>423</td>
      <td>56</td>
      <td>60</td>
      <td>6261</td>
      <td>8115</td>
      <td>133</td>
      <td>362</td>
      <td>127</td>
      <td>157</td>
      <td>10</td>
      <td>19</td>
      <td>6798</td>
      <td>8695</td>
      <td>199</td>
      <td>441</td>
    </tr>
    <tr>
      <td></td>
      <td>vcftools_subset</td>
      <td>vcftools_subset</td>
      <td>Select samples from a VCF dataset</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://vcftools.github.io/</td>
      <td>Variant Analysis</td>
      <td>vcftools_subset</td>
      <td>2013-11-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/vcftools/vcftools_subset</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/vcftools/vcftools_subset</td>
      <td>0.1</td>
      <td>tabix</td>
      <td>1.11</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>134</td>
      <td>142</td>
      <td>46</td>
      <td>49</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>120</td>
      <td>142</td>
      <td>12</td>
      <td>19</td>
      <td>254</td>
      <td>284</td>
      <td>58</td>
      <td>68</td>
    </tr>
    <tr>
      <td></td>
      <td>venn_list</td>
      <td>venn_list</td>
      <td>Draw Venn Diagram (PDF) from lists, FASTA files, etc</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/venn_list</td>
      <td>Graphics, Sequence Analysis, Visualization</td>
      <td>venn_list</td>
      <td>2015-12-08</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/venn_list</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/venn_list</td>
      <td>0.1.2</td>
      <td>galaxy_sequence_utils</td>
      <td>1.2.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>4519</td>
      <td>5476</td>
      <td>667</td>
      <td>730</td>
      <td>9858</td>
      <td>18420</td>
      <td>986</td>
      <td>2010</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>14377</td>
      <td>23896</td>
      <td>1653</td>
      <td>2740</td>
    </tr>
    <tr>
      <td></td>
      <td>vienna_rna</td>
      <td>viennarna_kinfold, viennarna_kinwalker, viennarna_rna2dfold, viennarna_rnaaliduplex, viennarna_rnaalifold, viennarna_rnacofold, viennarna_rnadistance, viennarna_rnaduplex, viennarna_rnaeval, viennarna_rnafold, viennarna_rnaheat, viennarna_rnainverse, viennarna_rnalalifold, viennarna_rnalfold, viennarna_rnapaln, viennarna_rnadpdist, viennarna_rnapkplex, viennarna_rnaplex, viennarna_rnaplfold, viennarna_rnaplot, viennarna_rnasnoop, viennarna_rnasubopt, viennarna_rnaup</td>
      <td>ViennaRNA - Prediction and comparison of RNA secondary structures</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.tbi.univie.ac.at/RNA/</td>
      <td>RNA</td>
      <td>viennarna</td>
      <td>2017-08-19</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/vienna_rna</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/vienna_rna</td>
      <td>2.2.10</td>
      <td>viennarna</td>
      <td>2.6.4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>21</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>21</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>21</td>
      <td>21</td>
      <td>0</td>
      <td>3197</td>
      <td>8107</td>
      <td>344</td>
      <td>415</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3197</td>
      <td>8107</td>
      <td>344</td>
      <td>415</td>
    </tr>
    <tr>
      <td></td>
      <td>vt</td>
      <td>vt_@BINARY@, vt_@BINARY@</td>
      <td>A tool set for short variant discovery in genetic sequence data.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>vt</td>
      <td>2015-02-28</td>
      <td>bgruening</td>
      <td>https://github.com/atks/vt</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/vt</td>
      <td>0.2</td>
      <td>vt</td>
      <td>2015.11.10</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>weightedaverage</td>
      <td>wtavg</td>
      <td>Assign weighted-average of the values of features overlapping an interval</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>weightedaverage</td>
      <td>2014-04-01</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/weightedaverage</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/weightedaverage</td>
      <td>1.0.1</td>
      <td>galaxy-ops</td>
      <td>1.1.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>227</td>
      <td>2307</td>
      <td>17</td>
      <td>75</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>227</td>
      <td>2307</td>
      <td>17</td>
      <td>75</td>
    </tr>
    <tr>
      <td></td>
      <td>whisper</td>
      <td>whisper</td>
      <td>Transcribe audio or video files to text using the OpenAI Whisper.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/whisper</td>
      <td>Machine Learning</td>
      <td>whisper</td>
      <td>2024-04-23</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/whisper</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/whisper</td>
      <td>20231117</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>96</td>
      <td>96</td>
      <td>13</td>
      <td>13</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>96</td>
      <td>96</td>
      <td>13</td>
      <td>13</td>
    </tr>
    <tr>
      <td></td>
      <td>windowsplitter</td>
      <td>winSplitter</td>
      <td>Make windows</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>windowsplitter</td>
      <td>2014-04-01</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/windowsplitter</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/windowsplitter</td>
      <td>1.0.1</td>
      <td>bx-python</td>
      <td>0.13.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>11</td>
      <td>0</td>
      <td>2</td>
      <td>171</td>
      <td>3961</td>
      <td>34</td>
      <td>328</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>171</td>
      <td>3972</td>
      <td>34</td>
      <td>330</td>
    </tr>
    <tr>
      <td></td>
      <td>woundhealing</td>
      <td>woundhealing_scratch_assay</td>
      <td>Tool to automate quantification of wound healing in high-throughput microscopy scratch assays</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://git.embl.de/grp-cba/wound-healing-htm-screen</td>
      <td>Imaging</td>
      <td>Wound healing scratch assay image analysis</td>
      <td>2024-02-29</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/image_processing/woundhealing</td>
      <td>1.6.1</td>
      <td>fiji-morpholibj</td>
      <td>1.6.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>31</td>
      <td>31</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>31</td>
      <td>31</td>
      <td>1</td>
      <td>1</td>
    </tr>
    <tr>
      <td></td>
      <td>wtdbg</td>
      <td>wtdbg</td>
      <td>WTDBG is a fuzzy Bruijn graph (FBG) approach to long noisy reads assembly.</td>
      <td>wtdbg2</td>
      <td>wtdbg2</td>
      <td></td>
      <td>wtdbg2</td>
      <td>Wtdbg2 is a de novo sequence assembler for long noisy reads produced by PacBio or Oxford Nanopore Technologies (ONT). It assembles raw reads without error correction and then builds the consensus from intermediate assembly output. Wtdbg2 is able to assemble the human and even the 32Gb Axolotl genome at a speed tens of times faster than CANU and FALCON while producing contigs of comparable base accuracy.</td>
      <td>Genome assembly, De-novo assembly</td>
      <td>Sequence assembly, Sequencing</td>
      <td>Up-to-date</td>
      <td>https://github.com/ruanjue/wtdbg2</td>
      <td>Assembly</td>
      <td>wtdbg</td>
      <td>2018-06-12</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/wtdbg</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/wtdbg</td>
      <td>2.5</td>
      <td>wtdbg</td>
      <td>2.5</td>
      <td>Genome assembly, De-novo assembly</td>
      <td>Sequence assembly, Sequencing</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1619</td>
      <td>1715</td>
      <td>337</td>
      <td>376</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1619</td>
      <td>1715</td>
      <td>337</td>
      <td>376</td>
    </tr>
    <tr>
      <td></td>
      <td>xy_plot</td>
      <td>XY_Plot_1</td>
      <td>Plotting tool for multiple series and graph types</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Graphics, Statistics</td>
      <td>xy_plot</td>
      <td>2013-09-25</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/xy_plot</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/xy_plot</td>
      <td>1.0.2</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>4490</td>
      <td>6169</td>
      <td>768</td>
      <td>889</td>
      <td>6120</td>
      <td>15516</td>
      <td>1921</td>
      <td>3653</td>
      <td>2033</td>
      <td>2212</td>
      <td>556</td>
      <td>590</td>
      <td>12643</td>
      <td>23897</td>
      <td>3245</td>
      <td>5132</td>
    </tr>
    <tr>
      <td></td>
      <td>TrimNs</td>
      <td>trimns</td>
      <td>TrimNs is used to trim and remove fake cut sites from bionano hybrid scaffold data in the VGP pipeline</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/VGP/vgp-assembly/tree/master/pipeline/bionano/trimNs</td>
      <td>Assembly</td>
      <td>trimns</td>
      <td>2021-01-29</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/trimN</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/TrimNs</td>
      <td>0.1.0</td>
      <td>trimns_vgp</td>
      <td>1.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>43</td>
      <td>43</td>
      <td>20</td>
      <td>20</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>43</td>
      <td>43</td>
      <td>20</td>
      <td>20</td>
    </tr>
    <tr>
      <td></td>
      <td>abricate</td>
      <td>abricate, abricate_list, abricate_summary</td>
      <td>Mass screening of contigs for antiobiotic resistance genes</td>
      <td>ABRicate</td>
      <td>ABRicate</td>
      <td></td>
      <td>ABRicate</td>
      <td>Mass screening of contigs for antimicrobial resistance or virulence genes.</td>
      <td>Antimicrobial resistance prediction</td>
      <td>Genomics, Microbiology</td>
      <td>Up-to-date</td>
      <td>https://github.com/tseemann/abricate</td>
      <td>Sequence Analysis</td>
      <td>abricate</td>
      <td>2017-07-10</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/abricate/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/abricate</td>
      <td>1.0.1</td>
      <td>abricate</td>
      <td>1.0.1</td>
      <td>Antimicrobial resistance prediction</td>
      <td>Genomics, Microbiology</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>619817</td>
      <td>622353</td>
      <td>4097</td>
      <td>4130</td>
      <td>320454</td>
      <td>320454</td>
      <td>2915</td>
      <td>2915</td>
      <td>496156</td>
      <td>503743</td>
      <td>2012</td>
      <td>2227</td>
      <td>1436427</td>
      <td>1446550</td>
      <td>9024</td>
      <td>9272</td>
    </tr>
    <tr>
      <td></td>
      <td>abritamr</td>
      <td>abritamr</td>
      <td>A pipeline for running AMRfinderPlus and collating results into functional classes</td>
      <td>abritamr</td>
      <td>abritamr</td>
      <td></td>
      <td>abriTAMR</td>
      <td>an AMR gene detection pipeline that runs AMRFinderPlus on a single (or list ) of given isolates and collates the results into a table, separating genes identified into functionally relevant groups.</td>
      <td>Antimicrobial resistance prediction</td>
      <td>Microbiology, Public health and epidemiology, Infectious disease</td>
      <td>Up-to-date</td>
      <td>https://zenodo.org/record/7370628</td>
      <td>Sequence Analysis</td>
      <td>abritamr</td>
      <td>2023-04-03</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/abritamr</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/abritamr</td>
      <td>1.0.19</td>
      <td>abritamr</td>
      <td>1.0.19</td>
      <td>Antimicrobial resistance prediction</td>
      <td>Microbiology, Public health and epidemiology, Infectious disease</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1139</td>
      <td>1139</td>
      <td>109</td>
      <td>109</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1139</td>
      <td>1139</td>
      <td>109</td>
      <td>109</td>
    </tr>
    <tr>
      <td></td>
      <td>abyss</td>
      <td>abyss-pe</td>
      <td>Assembly By Short Sequences - a de novo, parallel, paired-end sequence assembler</td>
      <td>abyss</td>
      <td>abyss</td>
      <td></td>
      <td>ABySS</td>
      <td>De novo genome sequence assembler using short reads.</td>
      <td>Genome assembly, De-novo assembly, Scaffolding</td>
      <td>Sequence assembly</td>
      <td>Up-to-date</td>
      <td>http://www.bcgsc.ca/platform/bioinfo/software/abyss</td>
      <td>Assembly</td>
      <td>abyss</td>
      <td>2015-04-14</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/abyss</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/abyss</td>
      <td>2.3.9</td>
      <td>abyss</td>
      <td>2.3.9</td>
      <td>Genome assembly, De-novo assembly, Scaffolding</td>
      <td>Sequence assembly</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>5543</td>
      <td>5543</td>
      <td>1042</td>
      <td>1042</td>
      <td>1301</td>
      <td>1305</td>
      <td>413</td>
      <td>414</td>
      <td>2030</td>
      <td>2030</td>
      <td>299</td>
      <td>299</td>
      <td>8874</td>
      <td>8878</td>
      <td>1754</td>
      <td>1755</td>
    </tr>
    <tr>
      <td></td>
      <td>adapter_removal</td>
      <td>adapter_removal</td>
      <td>Removes residual adapter sequences from single-end (SE) or paired-end (PE) FASTQ reads.</td>
      <td>adapterremoval</td>
      <td>adapterremoval</td>
      <td></td>
      <td>AdapterRemoval</td>
      <td>AdapterRemoval searches for and removes adapter sequences from High-Throughput Sequencing (HTS) data and (optionally) trims low quality bases from the 3' end of reads following adapter removal. AdapterRemoval can analyze both single end and paired end data, and can be used to merge overlapping paired-ended reads into (longer) consensus sequences. Additionally, AdapterRemoval can construct a consensus adapter sequence for paired-ended reads, if which this information is not available.</td>
      <td>Sequence trimming, Sequence merging, Primer removal</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/MikkelSchubert/adapterremoval</td>
      <td>Fasta Manipulation, Sequence Analysis</td>
      <td>adapter_removal</td>
      <td>2022-05-04</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/adapter_removal/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/adapter_removal</td>
      <td>2.3.4</td>
      <td>adapterremoval</td>
      <td>2.3.4</td>
      <td>Sequence trimming, Sequence merging, Primer removal</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>348</td>
      <td>348</td>
      <td>62</td>
      <td>62</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>348</td>
      <td>348</td>
      <td>62</td>
      <td>62</td>
    </tr>
    <tr>
      <td></td>
      <td>add_input_name_as_column</td>
      <td>addName</td>
      <td>Add input name as column on an existing tabular file</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/add_input_name_as_column</td>
      <td>Text Manipulation</td>
      <td>add_input_name_as_column</td>
      <td>2020-03-24</td>
      <td>mvdbeek</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/add_input_name_as_column</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/add_input_name_as_column</td>
      <td>0.2.0</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>76834</td>
      <td>86002</td>
      <td>227</td>
      <td>246</td>
      <td>20174</td>
      <td>20953</td>
      <td>312</td>
      <td>395</td>
      <td>11</td>
      <td>11</td>
      <td>2</td>
      <td>2</td>
      <td>97019</td>
      <td>106966</td>
      <td>541</td>
      <td>643</td>
    </tr>
    <tr>
      <td></td>
      <td>aegean</td>
      <td>aegean_canongff3, aegean_gaeval, aegean_locuspocus, aegean_parseval</td>
      <td>AEGeAn toolkit wrappers</td>
      <td>gaeval</td>
      <td>gaeval</td>
      <td></td>
      <td>GAEVAL</td>
      <td>Gene Annotation EVAluation.</td>
      <td>Sequence annotation</td>
      <td>Sequence analysis, Gene structure</td>
      <td>Up-to-date</td>
      <td>https://github.com/BrendelGroup/AEGeAn</td>
      <td>Transcriptomics, Sequence Analysis</td>
      <td>aegean</td>
      <td>2021-01-03</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/aegean</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/aegean</td>
      <td>0.16.0</td>
      <td>aegean</td>
      <td>0.16.0</td>
      <td>Sequence annotation</td>
      <td>Sequence analysis, Gene structure</td>
      <td>1</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>1</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>200</td>
      <td>200</td>
      <td>77</td>
      <td>77</td>
      <td>83</td>
      <td>83</td>
      <td>39</td>
      <td>39</td>
      <td>1747</td>
      <td>1747</td>
      <td>44</td>
      <td>44</td>
      <td>2030</td>
      <td>2030</td>
      <td>160</td>
      <td>160</td>
    </tr>
    <tr>
      <td></td>
      <td>aldex2</td>
      <td>aldex2</td>
      <td>Performs analysis Of differential abundance taking sample variation into account</td>
      <td>aldex2</td>
      <td>aldex2</td>
      <td></td>
      <td>ALDEx2</td>
      <td>A differential abundance analysis for the comparison of two or more conditions. It uses a Dirichlet-multinomial model to infer abundance from counts, that has been optimized for three or more experimental replicates. Infers sampling variation and calculates the expected FDR given the biological and sampling variation using the Wilcox rank test and Welches t-test, or the glm and Kruskal Wallis tests. Reports both P and fdr values calculated by the Benjamini Hochberg correction.</td>
      <td>Statistical inference</td>
      <td>Gene expression, Statistics and probability</td>
      <td>To update</td>
      <td>https://github.com/ggloor/ALDEx_bioc</td>
      <td>Metagenomics</td>
      <td>aldex2</td>
      <td>2022-06-29</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/aldex2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/aldex2</td>
      <td>1.26.0</td>
      <td>bioconductor-aldex2</td>
      <td>1.34.0</td>
      <td>Statistical inference</td>
      <td>Gene expression, Statistics and probability</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>262</td>
      <td>262</td>
      <td>36</td>
      <td>36</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>262</td>
      <td>262</td>
      <td>36</td>
      <td>36</td>
    </tr>
    <tr>
      <td></td>
      <td>allegro</td>
      <td>allegro</td>
      <td>Linkage and haplotype analysis from deCODE</td>
      <td>allegro</td>
      <td>allegro</td>
      <td></td>
      <td>Allegro</td>
      <td>It does simultaneous discovery of cis-regulatory motifs and their associated expression profiles. Its input are DNA sequences (typically, promoters or 3 UTRs) and genome-wide expression profiles. Its output is the set of motifs found, and for each motif the set of genes it regulates (its transcriptional module). It is highly efficient and can analyze expression profiles of thousands of genes, measured across dozens of experimental conditions, along with all regulatory sequences in the genome.</td>
      <td>Sequence motif discovery</td>
      <td>Sequence analysis, Transcription factors and regulatory sites, DNA</td>
      <td>To update</td>
      <td>http://www.decode.com/software/</td>
      <td>Variant Analysis</td>
      <td>allegro</td>
      <td>2017-11-19</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/allegro/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/allegro</td>
      <td>@VER@.0</td>
      <td>allegro</td>
      <td>3</td>
      <td>Sequence motif discovery</td>
      <td>Sequence analysis, Transcription factors and regulatory sites, DNA</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>amplican</td>
      <td>amplican</td>
      <td>AmpliCan is an analysis tool for genome editing.</td>
      <td>amplican</td>
      <td>amplican</td>
      <td></td>
      <td>amplican</td>
      <td>It performs alignment of the amplicon reads, normalizes gathered data, calculates multiple statistics (e.g. cut rates, frameshifts) and presents results in form of aggregated reports. Data and statistics can be broken down by experiments, barcodes, user defined groups, guides and amplicons allowing for quick identification of potential problems.</td>
      <td>Alignment, Standardisation and normalisation</td>
      <td>PCR experiment, Statistics and probability</td>
      <td>To update</td>
      <td>https://github.com/valenlab/amplican</td>
      <td>Sequence Analysis</td>
      <td>amplican</td>
      <td>2021-09-22</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/amplican</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/amplican</td>
      <td>1.14.0</td>
      <td>bioconductor-amplican</td>
      <td>1.24.0</td>
      <td>Alignment, Standardisation and normalisation</td>
      <td>PCR experiment, Statistics and probability</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>81</td>
      <td>81</td>
      <td>17</td>
      <td>17</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>81</td>
      <td>81</td>
      <td>17</td>
      <td>17</td>
    </tr>
    <tr>
      <td></td>
      <td>ampvis2</td>
      <td>ampvis2_alpha_diversity, ampvis2_boxplot, ampvis2_core, ampvis2_export_fasta, ampvis2_frequency, ampvis2_heatmap, ampvis2_load, ampvis2_merge_ampvis2, ampvis2_mergereplicates, ampvis2_octave, ampvis2_ordinate, ampvis2_otu_network, ampvis2_rankabundance, ampvis2_rarecurve, ampvis2_setmetadata, ampvis2_subset_samples, ampvis2_subset_taxa, ampvis2_timeseries, ampvis2_venn</td>
      <td>ampvis2</td>
      <td>ampvis</td>
      <td>ampvis</td>
      <td></td>
      <td>ampvis</td>
      <td>ampvis2 is an R-package to conveniently visualise and analyse 16S rRNA amplicon data in different ways.</td>
      <td>Analysis, Visualisation</td>
      <td>Biodiversity</td>
      <td>To update</td>
      <td>https://github.com/MadsAlbertsen/ampvis2/</td>
      <td>Metagenomics</td>
      <td>ampvis2</td>
      <td>2022-04-04</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ampvis2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ampvis2</td>
      <td>2.8.9</td>
      <td></td>
      <td></td>
      <td>Analysis, Visualisation</td>
      <td>Biodiversity</td>
      <td>0</td>
      <td>0</td>
      <td>19</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>19</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>19</td>
      <td>0</td>
      <td>0</td>
      <td>1213</td>
      <td>1213</td>
      <td>155</td>
      <td>155</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1213</td>
      <td>1213</td>
      <td>155</td>
      <td>155</td>
    </tr>
    <tr>
      <td></td>
      <td>amrfinderplus</td>
      <td>amrfinderplus</td>
      <td>"AMRFinderPlus is designed to find acquired antimicrobial resistance genes and point mutations in protein and/or assembled nucleotide sequences.It can also search "plus", stress, heat, and biocide resistance and virulence factors for some organisms.</td>
      <td>amrfinderplus</td>
      <td>amrfinderplus</td>
      <td></td>
      <td>AMRFinderPlus</td>
      <td>AMRFinderPlus is designed to find acquired antimicrobial resistance genes and point mutations in protein and/or assembled nucleotide sequences.It can also search "plus", stress, heat, and biocide resistance and virulence factors for some organisms</td>
      <td>Antimicrobial resistance prediction</td>
      <td>Microbiology, Public health and epidemiology, Infectious disease</td>
      <td>Up-to-date</td>
      <td>https://github.com/ncbi/amr</td>
      <td>Sequence Analysis</td>
      <td>AMRFinderPlus</td>
      <td>2023-05-12</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/amrfinderplus</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/amrfinderplus</td>
      <td>3.12.8</td>
      <td>ncbi-amrfinderplus</td>
      <td>3.12.8</td>
      <td>Antimicrobial resistance prediction</td>
      <td>Microbiology, Public health and epidemiology, Infectious disease</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5734</td>
      <td>5734</td>
      <td>334</td>
      <td>334</td>
      <td>1035</td>
      <td>1035</td>
      <td>106</td>
      <td>106</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6769</td>
      <td>6769</td>
      <td>440</td>
      <td>440</td>
    </tr>
    <tr>
      <td></td>
      <td>ancombc</td>
      <td>ancombc</td>
      <td>Performs analysis of compositions of microbiomes with bias correction.</td>
      <td>ancombc</td>
      <td>ancombc</td>
      <td></td>
      <td>ANCOMBC</td>
      <td>Determine taxa whose absolute abundances, per unit volume, of the ecosystem (e.g. gut) are significantly different with changes in the covariate of interest (e.g. group). The current version of ancombc function implements Analysis of Compositions of Microbiomes with Bias Correction (ANCOM-BC) in cross-sectional data while allowing for covariate adjustment.</td>
      <td>DNA barcoding</td>
      <td>Microbial ecology, Metagenomics, Taxonomy</td>
      <td>To update</td>
      <td>https://github.com/FrederickHuangLin/ANCOMBC</td>
      <td>Metagenomics</td>
      <td>ancombc</td>
      <td>2022-07-16</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ancombc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ancombc</td>
      <td>1.4.0</td>
      <td>bioconductor-ancombc</td>
      <td>2.4.0</td>
      <td>DNA barcoding</td>
      <td>Microbial ecology, Metagenomics, Taxonomy</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>26</td>
      <td>26</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>26</td>
      <td>26</td>
      <td>7</td>
      <td>7</td>
    </tr>
    <tr>
      <td></td>
      <td>anndata</td>
      <td>anndata_export, anndata_import, anndata_inspect, anndata_manipulate, modify_loom</td>
      <td>Import, Export, Inspect and Manipulate Anndata and Loom objects</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://anndata.readthedocs.io</td>
      <td>Single Cell, Spatial Omics</td>
      <td>anndata</td>
      <td>2020-10-09</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/anndata/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/anndata</td>
      <td>0.10.9</td>
      <td>anndata</td>
      <td>0.6.22.post1</td>
      <td></td>
      <td></td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>43047</td>
      <td>43139</td>
      <td>1826</td>
      <td>1827</td>
      <td>17240</td>
      <td>17240</td>
      <td>878</td>
      <td>878</td>
      <td>9309</td>
      <td>9309</td>
      <td>298</td>
      <td>298</td>
      <td>69596</td>
      <td>69688</td>
      <td>3002</td>
      <td>3003</td>
    </tr>
    <tr>
      <td></td>
      <td>annotatemyids</td>
      <td>annotatemyids</td>
      <td>annotateMyIDs: get annotation for a set of IDs using the Bioconductor annotation packages</td>
      <td>annotatemyids</td>
      <td>annotatemyids</td>
      <td></td>
      <td>annotatemyids</td>
      <td>This tool can get annotation for a generic set of IDs, using the Bioconductor annotation data packages. Supported organisms are human, mouse, rat, fruit fly and zebrafish. The org.db packages that are used here are primarily based on mapping using Entrez Gene identifiers. More information on the annotation packages can be found at the Bioconductor website, for example, information on the human annotation package (org.Hs.eg.db) can be found here.</td>
      <td>Annotation</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/annotatemyids</td>
      <td>Genome annotation</td>
      <td>annotatemyids</td>
      <td>2018-03-11</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/annotatemyids</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/annotatemyids</td>
      <td>3.18.0</td>
      <td>bioconductor-org.hs.eg.db</td>
      <td>3.18.0</td>
      <td>Annotation</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>29266</td>
      <td>30655</td>
      <td>2849</td>
      <td>2973</td>
      <td>59299</td>
      <td>66180</td>
      <td>5917</td>
      <td>6355</td>
      <td>5644</td>
      <td>5644</td>
      <td>1049</td>
      <td>1049</td>
      <td>94209</td>
      <td>102479</td>
      <td>9815</td>
      <td>10377</td>
    </tr>
    <tr>
      <td></td>
      <td>argnorm</td>
      <td>argnorm</td>
      <td>argNorm is a tool to normalize antibiotic resistance genes (ARGs) by mapping them to the antibiotic resistance ontology (ARO) created by the CARD database</td>
      <td>argnorm</td>
      <td>argnorm</td>
      <td></td>
      <td>argNorm</td>
      <td>argNorm is a tool to normalize antibiotic resistance genes (ARGs) by mapping them to the antibiotic resistance ontology (ARO) created by the CARD database.argNorm also enhances antibiotic resistance gene annotations by providing drug categorization of the drugs that antibiotic resistance genes confer resistance to.</td>
      <td>Gene functional annotation</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/BigDataBiology/argNorm</td>
      <td>Genome annotation</td>
      <td>argnorm</td>
      <td>2024-08-28</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/argnorm</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/argnorm</td>
      <td>0.6.0</td>
      <td>argnorm</td>
      <td>0.6.0</td>
      <td>Gene functional annotation</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>arriba</td>
      <td>arriba, arriba_draw_fusions, arriba_get_filters</td>
      <td>Arriba detects fusion genes in RNA-Seq data after running RNA-STAR</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/suhrig/arriba</td>
      <td>Sequence Analysis, Transcriptomics</td>
      <td>arriba</td>
      <td>2022-07-27</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/arriba</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/arriba</td>
      <td>2.4.0</td>
      <td>arriba</td>
      <td>2.4.0</td>
      <td></td>
      <td></td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>3670</td>
      <td>3670</td>
      <td>71</td>
      <td>71</td>
      <td>3</td>
      <td>3</td>
      <td>2</td>
      <td>2</td>
      <td>234</td>
      <td>234</td>
      <td>10</td>
      <td>10</td>
      <td>3907</td>
      <td>3907</td>
      <td>83</td>
      <td>83</td>
    </tr>
    <tr>
      <td></td>
      <td>art</td>
      <td>art_454, art_illumina, art_solid</td>
      <td>Simulator for Illumina, 454, and SOLiD sequencing data</td>
      <td>art</td>
      <td>art</td>
      <td></td>
      <td>ART</td>
      <td>ART is a set of simulation tools to generate synthetic next-generation sequencing reads. ART simulates sequencing reads by mimicking real sequencing process with empirical error models or quality profiles summarized from large recalibrated sequencing data. ART can also simulate reads using user own read error model or quality profiles. ART supports simulation of single-end, paired-end/mate-pair reads of three major commercial next-generation sequencing platforms. Illuminas Solexa, Roches 454 and Applied Biosystems SOLiD</td>
      <td>Conversion</td>
      <td>Bioinformatics</td>
      <td>To update</td>
      <td>http://www.niehs.nih.gov/research/resources/software/biostatistics/art/</td>
      <td>Sequence Analysis, Data Source</td>
      <td>art</td>
      <td>2015-04-08</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/art</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/art</td>
      <td>2014.11.03.0</td>
      <td>art</td>
      <td>2016.06.05</td>
      <td>Conversion</td>
      <td>Bioinformatics</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>140</td>
      <td>140</td>
      <td>9</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>140</td>
      <td>140</td>
      <td>9</td>
      <td>9</td>
    </tr>
    <tr>
      <td></td>
      <td>artic</td>
      <td>artic_guppyplex, artic_minion</td>
      <td>The artic pipeline is designed to help run the artic bioinformatics protocols;for example the nCoV-2019 novel coronavirus protocol.Features include: read filtering, primer trimming, amplicon coverage normalisation,variant calling and consensus building</td>
      <td>artic</td>
      <td>artic</td>
      <td></td>
      <td>ARTIC</td>
      <td>A bioinformatics pipeline for working with virus sequencing data sequenced with nanopore</td>
      <td>Sequence alignment</td>
      <td>Genomics</td>
      <td>To update</td>
      <td>https://github.com/artic-network/fieldbioinformatics</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>2020-04-22</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/artic</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/artic</td>
      <td></td>
      <td>artic</td>
      <td>1.2.4</td>
      <td>Sequence alignment</td>
      <td>Genomics</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>2</td>
      <td>1</td>
      <td>0</td>
      <td>9021</td>
      <td>9021</td>
      <td>238</td>
      <td>238</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9021</td>
      <td>9021</td>
      <td>238</td>
      <td>238</td>
    </tr>
    <tr>
      <td></td>
      <td>assembly-stats</td>
      <td>assembly_stats</td>
      <td>Assembly metric visualisations to facilitate rapid assessment and comparison of assembly quality.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/rjchallis/assembly-stats</td>
      <td>Assembly</td>
      <td>assembly_stats</td>
      <td>2023-06-21</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/assembly-stats</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/assembly-stats</td>
      <td>17.02</td>
      <td>rjchallis-assembly-stats</td>
      <td>17.02</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>astral</td>
      <td>astral</td>
      <td>Tool for estimating an unrooted species tree given a set of unrooted gene trees</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/smirarab/ASTRAL</td>
      <td>Phylogenetics</td>
      <td>astral</td>
      <td>2024-08-28</td>
      <td>iuc</td>
      <td>https://github.com/usegalaxy-be/galaxytools/tree/main/astral</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/astral</td>
      <td>5.7.8</td>
      <td>astral-tree</td>
      <td>5.7.8</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>151</td>
      <td>151</td>
      <td>5</td>
      <td>5</td>
      <td>151</td>
      <td>151</td>
      <td>5</td>
      <td>5</td>
    </tr>
    <tr>
      <td></td>
      <td>augustus</td>
      <td>augustus, augustus_training</td>
      <td>AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences.</td>
      <td>augustus</td>
      <td>augustus</td>
      <td></td>
      <td>AUGUSTUS</td>
      <td>AUGUSTUS is a eukaryotic gene prediction tool. It can integrate evidence, e.g. from RNA-Seq, ESTs, proteomics, but can also predict genes ab initio. The PPX extension to AUGUSTUS can take a protein sequence multiple sequence alignment as input to find new members of the family in a genome. It can be run through a web interface (see https://bio.tools/webaugustus), or downloaded and run locally.</td>
      <td>Gene prediction, Ab-initio gene prediction, Homology-based gene prediction, Homology-based gene prediction, Operation</td>
      <td>Gene transcripts, Gene and protein families</td>
      <td>To update</td>
      <td>http://bioinf.uni-greifswald.de/augustus/</td>
      <td>Sequence Analysis</td>
      <td>augustus</td>
      <td>2017-10-12</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/augustus</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/augustus</td>
      <td>3.4.0</td>
      <td>augustus</td>
      <td>3.5.0</td>
      <td>Ab-initio gene prediction, Homology-based gene prediction, Homology-based gene prediction, Operation</td>
      <td>Gene transcripts, Gene and protein families</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>12890</td>
      <td>13988</td>
      <td>1826</td>
      <td>1988</td>
      <td>14901</td>
      <td>14901</td>
      <td>2633</td>
      <td>2633</td>
      <td>6187</td>
      <td>6187</td>
      <td>511</td>
      <td>511</td>
      <td>33978</td>
      <td>35076</td>
      <td>4970</td>
      <td>5132</td>
    </tr>
    <tr>
      <td></td>
      <td>b2btools</td>
      <td>b2btools_single_sequence</td>
      <td>This software suite provides structural predictions for protein sequences made by Bio2Byte group.About Bio2Byte: We investigate how the dynamics, conformational states, and available experimental data of proteins relate to their amino acid sequence.Underlying physical and chemical principles are computationally unraveled through data integration, analysis, and machine learning, so connecting themto biological events and improving our understanding of the way proteins work.</td>
      <td>b2btools</td>
      <td>b2btools</td>
      <td></td>
      <td>b2bTools</td>
      <td>The bio2byte tools server (b2btools) offers the following single protein sequence based predictions:- Backbone and sidechain dynamics (DynaMine)- Helix, sheet, coil and polyproline-II propensity- Early folding propensity (EFoldMine)- Disorder (DisoMine)- Beta-sheet aggregation (Agmata)In addition, multiple sequence alignments (MSAs) can be uploaded to scan the 'biophysical limits' of a protein family as defined in the MSA</td>
      <td>Protein disorder prediction, Protein secondary structure prediction, Protein feature detection</td>
      <td></td>
      <td>To update</td>
      <td>https://bio2byte.be</td>
      <td>Computational chemistry, Molecular Dynamics, Proteomics, Sequence Analysis, Synthetic Biology</td>
      <td></td>
      <td>2022-08-09</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ucsc-tools/b2tools</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/b2btools</td>
      <td>3.0.5+galaxy0</td>
      <td>b2btools</td>
      <td>3.0.7</td>
      <td>Protein disorder prediction, Protein secondary structure prediction</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>369</td>
      <td>369</td>
      <td>19</td>
      <td>19</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>369</td>
      <td>369</td>
      <td>19</td>
      <td>19</td>
    </tr>
    <tr>
      <td></td>
      <td>bakta</td>
      <td>bakta</td>
      <td>"Bakta is a tool for the rapid &amp; standardized annotation of bacterial genomes and plasmids from both isolates and MAGs.It provides dbxref-rich and sORF-including annotations in machine-readable JSON &amp; bioinformatics standard file formats for automatic downstream analysis."</td>
      <td>bakta</td>
      <td>bakta</td>
      <td></td>
      <td>Bakta</td>
      <td>Rapid &amp; standardized annotation of bacterial genomes, MAGs &amp; plasmids</td>
      <td>Genome annotation</td>
      <td>Genomics, Data submission, annotation and curation, Sequence analysis</td>
      <td>Up-to-date</td>
      <td>https://github.com/oschwengers/bakta</td>
      <td>Sequence Analysis</td>
      <td>bakta</td>
      <td>2022-09-01</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/bakta</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bakta</td>
      <td>1.9.4</td>
      <td>bakta</td>
      <td>1.9.4</td>
      <td>Genome annotation</td>
      <td>Genomics, Data submission, annotation and curation, Sequence analysis</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>11117</td>
      <td>11117</td>
      <td>534</td>
      <td>534</td>
      <td>19</td>
      <td>19</td>
      <td>1</td>
      <td>1</td>
      <td>9601</td>
      <td>9601</td>
      <td>167</td>
      <td>167</td>
      <td>20737</td>
      <td>20737</td>
      <td>702</td>
      <td>702</td>
    </tr>
    <tr>
      <td></td>
      <td>bam_to_scidx</td>
      <td>bam_to_scidx</td>
      <td>Contains a tool that converts a BAM file to an ScIdx file.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/seqcode/cegr-tools/tree/master/src/org/seqcode/cegrtools/bamtoscidx</td>
      <td>Convert Formats</td>
      <td>bam_to_scidx</td>
      <td>2015-12-07</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bam_to_scidx</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bam_to_scidx</td>
      <td>1.0.1</td>
      <td>openjdk</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>83</td>
      <td>131</td>
      <td>15</td>
      <td>20</td>
      <td>329</td>
      <td>785</td>
      <td>50</td>
      <td>210</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>412</td>
      <td>916</td>
      <td>65</td>
      <td>230</td>
    </tr>
    <tr>
      <td></td>
      <td>bamtools</td>
      <td>bamtools</td>
      <td>Operate on and transform BAM datasets in various ways using bamtools</td>
      <td>bamtools</td>
      <td>bamtools</td>
      <td></td>
      <td>BamTools</td>
      <td>BamTools provides a fast, flexible C++ API &amp; toolkit for reading, writing, and managing BAM files.</td>
      <td>Data handling, Sequence alignment analysis</td>
      <td>Sequencing, Data management, Sequence analysis</td>
      <td>Up-to-date</td>
      <td>https://github.com/pezmaster31/bamtools</td>
      <td>Sequence Analysis, SAM</td>
      <td>bamtools</td>
      <td>2017-06-09</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/bamtools/bamtools</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/bamtools/bamtools</td>
      <td>2.5.2</td>
      <td>bamtools</td>
      <td>2.5.2</td>
      <td>Data handling, Sequence alignment analysis</td>
      <td>Sequencing, Sequence analysis</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>14693</td>
      <td>15620</td>
      <td>653</td>
      <td>721</td>
      <td>61872</td>
      <td>101400</td>
      <td>1656</td>
      <td>5678</td>
      <td>39</td>
      <td>39</td>
      <td>3</td>
      <td>3</td>
      <td>76604</td>
      <td>117059</td>
      <td>2312</td>
      <td>6402</td>
    </tr>
    <tr>
      <td></td>
      <td>bamtools_filter</td>
      <td>bamFilter</td>
      <td>Filter BAM datasets on various attributes using bamtools filter</td>
      <td>bamtools</td>
      <td>bamtools</td>
      <td></td>
      <td>BamTools</td>
      <td>BamTools provides a fast, flexible C++ API &amp; toolkit for reading, writing, and managing BAM files.</td>
      <td>Data handling, Sequence alignment analysis</td>
      <td>Sequencing, Data management, Sequence analysis</td>
      <td>Up-to-date</td>
      <td>https://github.com/pezmaster31/bamtools</td>
      <td>Sequence Analysis, SAM</td>
      <td>bamtools_filter</td>
      <td>2017-06-09</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/bamtools/bamtools_filter</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/bamtools/bamtools_filter</td>
      <td>2.5.2</td>
      <td>bamtools</td>
      <td>2.5.2</td>
      <td>Data handling, Sequence alignment analysis</td>
      <td>Sequencing, Sequence analysis</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>115954</td>
      <td>124795</td>
      <td>2994</td>
      <td>3180</td>
      <td>95433</td>
      <td>142254</td>
      <td>6712</td>
      <td>10660</td>
      <td>5793</td>
      <td>5793</td>
      <td>497</td>
      <td>497</td>
      <td>217180</td>
      <td>272842</td>
      <td>10203</td>
      <td>14337</td>
    </tr>
    <tr>
      <td></td>
      <td>bamtools_split</td>
      <td>bamtools_split_mapped, bamtools_split_paired, bamtools_split_ref, bamtools_split_tag</td>
      <td>Utility for filtering BAM files. It is based on the BAMtools suiteof tools by Derek Barnett.</td>
      <td>bamtools</td>
      <td>bamtools</td>
      <td></td>
      <td>BamTools</td>
      <td>BamTools provides a fast, flexible C++ API &amp; toolkit for reading, writing, and managing BAM files.</td>
      <td>Data handling, Sequence alignment analysis</td>
      <td>Sequencing, Data management, Sequence analysis</td>
      <td>Up-to-date</td>
      <td>https://github.com/pezmaster31/bamtools</td>
      <td>Sequence Analysis, SAM</td>
      <td></td>
      <td>2017-06-09</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/bamtools/bamtools_split</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/bamtools/bamtools_split</td>
      <td>2.5.2</td>
      <td>bamtools</td>
      <td>2.5.2</td>
      <td>Data handling, Sequence alignment analysis</td>
      <td>Sequencing, Sequence analysis</td>
      <td>4</td>
      <td>3</td>
      <td>4</td>
      <td>0</td>
      <td>4</td>
      <td>3</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>9073</td>
      <td>9073</td>
      <td>401</td>
      <td>401</td>
      <td>2746</td>
      <td>2746</td>
      <td>224</td>
      <td>224</td>
      <td>103</td>
      <td>103</td>
      <td>10</td>
      <td>10</td>
      <td>11922</td>
      <td>11922</td>
      <td>635</td>
      <td>635</td>
    </tr>
    <tr>
      <td></td>
      <td>bamutil</td>
      <td>bamutil_clip_overlap, bamutil_diff</td>
      <td>bamUtil is a repository that contains several programs that perform operations on SAM/BAM files.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/statgen/bamUtil</td>
      <td>Sequence Analysis</td>
      <td>bamutil</td>
      <td>2021-03-29</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bamutil</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bamutil</td>
      <td></td>
      <td>bamutil</td>
      <td>1.0.15</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>45</td>
      <td>45</td>
      <td>20</td>
      <td>20</td>
      <td>69</td>
      <td>69</td>
      <td>21</td>
      <td>21</td>
      <td>258</td>
      <td>258</td>
      <td>1</td>
      <td>1</td>
      <td>372</td>
      <td>372</td>
      <td>42</td>
      <td>42</td>
    </tr>
    <tr>
      <td></td>
      <td>bandage</td>
      <td>bandage_image, bandage_info</td>
      <td>Bandage - A Bioinformatics Application for Navigating De novo Assembly Graphs Easily</td>
      <td>bandage</td>
      <td>bandage</td>
      <td></td>
      <td>Bandage</td>
      <td>GUI program that allows users to interact with the assembly graphs made by de novo assemblers such as Velvet, SPAdes, MEGAHIT and others. It visualises assembly graphs, with connections, using graph layout algorithms.</td>
      <td>Sequence assembly visualisation</td>
      <td>Genomics, Sequence assembly</td>
      <td>Up-to-date</td>
      <td>https://github.com/rrwick/Bandage</td>
      <td>Visualization</td>
      <td>bandage</td>
      <td>2018-09-14</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bandage</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bandage</td>
      <td>2022.09</td>
      <td>bandage_ng</td>
      <td>2022.09</td>
      <td>Sequence assembly visualisation</td>
      <td>Genomics, Sequence assembly</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>55392</td>
      <td>56095</td>
      <td>4951</td>
      <td>5035</td>
      <td>47603</td>
      <td>47603</td>
      <td>4916</td>
      <td>4916</td>
      <td>18089</td>
      <td>18089</td>
      <td>3121</td>
      <td>3121</td>
      <td>121084</td>
      <td>121787</td>
      <td>12988</td>
      <td>13072</td>
    </tr>
    <tr>
      <td></td>
      <td>barcode_splitter</td>
      <td>barcode_splitter</td>
      <td>A utility to split sequence files using multiple sets of barcodes</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/princeton_genomics/barcode_splitter/</td>
      <td>Fastq Manipulation</td>
      <td>barcode_splitter</td>
      <td>2019-02-21</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/barcode_splitter</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/barcode_splitter</td>
      <td>0.18.4.0</td>
      <td>barcode_splitter</td>
      <td>0.18.6</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>baredsc</td>
      <td>baredsc_1d, baredsc_2d, baredsc_combine_1d, baredsc_combine_2d</td>
      <td>baredSC is a tool that uses a Monte-Carlo Markov Chain to estimate a confidence interval on the probability density function (PDF) of expression of one or two genes from single-cell RNA-seq data.</td>
      <td>baredsc</td>
      <td>baredsc</td>
      <td></td>
      <td>baredSC</td>
      <td>The baredSC (Bayesian Approach to Retreive Expression Distribution of Single Cell) is a tool that uses a Monte-Carlo Markov Chain to estimate a confidence interval on the probability density function (PDF) of expression of one or two genes from single-cell RNA-seq data.</td>
      <td>Data retrieval, Expression correlation analysis, Differential gene expression profiling</td>
      <td>RNA-Seq, Cytometry, Transcriptomics, Gene transcripts, Statistics and probability</td>
      <td>Up-to-date</td>
      <td>https://github.com/lldelisle/baredSC</td>
      <td>Single Cell, Transcriptomics, Visualization</td>
      <td>baredsc</td>
      <td>2023-09-06</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/baredsc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/baredsc</td>
      <td>1.1.3</td>
      <td>baredsc</td>
      <td>1.1.3</td>
      <td>Data retrieval, Expression correlation analysis, Differential gene expression profiling</td>
      <td>RNA-Seq, Cytometry, Transcriptomics, Gene transcripts, Statistics and probability</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>111</td>
      <td>111</td>
      <td>8</td>
      <td>8</td>
      <td>29</td>
      <td>29</td>
      <td>10</td>
      <td>10</td>
      <td>21</td>
      <td>21</td>
      <td>4</td>
      <td>4</td>
      <td>161</td>
      <td>161</td>
      <td>22</td>
      <td>22</td>
    </tr>
    <tr>
      <td></td>
      <td>barrnap</td>
      <td>barrnap</td>
      <td>Contains the Barrnap tool for finding ribosomal RNAs in FASTA sequences.</td>
      <td>barrnap</td>
      <td>barrnap</td>
      <td></td>
      <td>Barrnap</td>
      <td>Predict the location of ribosomal RNA genes in genomes. It supports bacteria (5S,23S,16S), archaea (5S,5.8S,23S,16S), mitochondria (12S,16S) and eukaryotes (5S,5.8S,28S,18S).</td>
      <td>Gene prediction</td>
      <td>Genomics, Model organisms, Model organisms</td>
      <td>To update</td>
      <td>https://github.com/tseemann/barrnap</td>
      <td>Sequence Analysis</td>
      <td>barrnap</td>
      <td>2017-08-25</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ucsc-tools/barrnap</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/barrnap</td>
      <td>1.2.2</td>
      <td>barrnap</td>
      <td>0.9</td>
      <td>Gene prediction</td>
      <td>Genomics, Model organisms, Model organisms</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>6041</td>
      <td>6041</td>
      <td>316</td>
      <td>316</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6723</td>
      <td>7273</td>
      <td>240</td>
      <td>322</td>
      <td>12764</td>
      <td>13314</td>
      <td>556</td>
      <td>638</td>
    </tr>
    <tr>
      <td></td>
      <td>basil</td>
      <td>basil</td>
      <td>Breakpoint detection, including large insertions</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/seqan/anise_basil</td>
      <td>Variant Analysis</td>
      <td>basil</td>
      <td>2019-05-22</td>
      <td>iuc</td>
      <td></td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/basil</td>
      <td>1.2.0</td>
      <td>anise_basil</td>
      <td>1.2.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>306</td>
      <td>311</td>
      <td>82</td>
      <td>83</td>
      <td>467</td>
      <td>467</td>
      <td>99</td>
      <td>99</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>773</td>
      <td>778</td>
      <td>181</td>
      <td>182</td>
    </tr>
    <tr>
      <td></td>
      <td>bax2bam</td>
      <td>bax2bam</td>
      <td>BAX to BAM converter</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/pacificbiosciences/bax2bam/</td>
      <td>Convert Formats, Sequence Analysis</td>
      <td>bax2bam</td>
      <td>2019-10-12</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pax2bam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bax2bam</td>
      <td>0.0.11</td>
      <td>bax2bam</td>
      <td>0.0.11</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>208</td>
      <td>208</td>
      <td>26</td>
      <td>26</td>
      <td>210</td>
      <td>210</td>
      <td>30</td>
      <td>30</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>418</td>
      <td>418</td>
      <td>56</td>
      <td>56</td>
    </tr>
    <tr>
      <td></td>
      <td>bayescan</td>
      <td>BayeScan</td>
      <td>Detecting natural selection from population-based genetic data</td>
      <td>bayescan</td>
      <td>bayescan</td>
      <td></td>
      <td>BayeScan</td>
      <td>BAYEsian genome SCAN for outliers, aims at identifying candidate loci under natural selection from genetic data, using differences in allele frequencies between populations. It is based on the multinomial-Dirichlet model.</td>
      <td>Statistical inference</td>
      <td>Genetics, Evolutionary biology, Statistics and probability, DNA polymorphism</td>
      <td>To update</td>
      <td>http://cmpg.unibe.ch/software/BayeScan/index.html</td>
      <td>Sequence Analysis</td>
      <td>bayescan</td>
      <td>2017-03-09</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bayescan/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bayescan</td>
      <td>2.1</td>
      <td>bayescan</td>
      <td>2.0.1</td>
      <td>Statistical inference</td>
      <td>Genetics, Evolutionary biology, Statistics and probability, DNA polymorphism</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>90</td>
      <td>90</td>
      <td>28</td>
      <td>28</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>90</td>
      <td>90</td>
      <td>28</td>
      <td>28</td>
    </tr>
    <tr>
      <td></td>
      <td>bbgbigwig</td>
      <td>bbgtobigwig</td>
      <td>Make a coverage bigwig from bam, bed or gff, optionally with a chromosome length file.</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td></td>
      <td>UCSC Genome Browser Utilities</td>
      <td>Utilities for handling sequences and assemblies from the UCSC Genome Browser project.</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>To update</td>
      <td>https://www.encodeproject.org/software/bedgraphtobigwig/</td>
      <td>Convert Formats</td>
      <td>bbgbigwig</td>
      <td>2024-06-14</td>
      <td>iuc</td>
      <td>https://www.encodeproject.org/software/bedgraphtobigwig/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bbgbigwig</td>
      <td>0.1</td>
      <td>ucsc-bedgraphtobigwig</td>
      <td>469</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>42</td>
      <td>42</td>
      <td>7</td>
      <td>7</td>
      <td>175</td>
      <td>175</td>
      <td>26</td>
      <td>26</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>217</td>
      <td>217</td>
      <td>33</td>
      <td>33</td>
    </tr>
    <tr>
      <td></td>
      <td>bbtools</td>
      <td>bbtools_bbduk, bbtools_bbmap, bbtools_bbmerge, bbtools_bbnorm, bbtools_callvariants, bbtools_tadpole</td>
      <td>BBTools is a suite of fast, multithreaded bioinformatics tools designed for analysis of DNA and RNA sequence data.BBTools can handle common sequencing file formats such as fastq, fasta, sam, scarf, fasta+qual, compressed or raw,with autodetection of quality encoding and interleaving. It is written in Java and works on any platform supportingJava, including Linux, MacOS, and Microsoft Windows and Linux; there are no dependencies other than Java (version7 or higher). Program descriptions and options are shown when running the shell scripts with no parameters.</td>
      <td>bbmap</td>
      <td>bbmap, bbtools</td>
      <td></td>
      <td>BBMap</td>
      <td>BBMap is a fast splice-aware aligner for RNA and DNA.  It is faster than almost all short-read aligners, yet retains unrivaled sensitivity and specificity, particularly for reads with many errors and indels.</td>
      <td>RNA-Seq analysis, Sequence trimming, Read mapping, Sequence contamination filtering, Read binning, Sequence alignment</td>
      <td>Sequencing, RNA splicing, Whole genome sequencing, Phylogenetics, Metagenomics, RNA-Seq</td>
      <td>To update</td>
      <td>https://jgi.doe.gov/data-and-tools/bbtools/</td>
      <td>Sequence Analysis</td>
      <td>bbtools</td>
      <td>2021-10-04</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bbtools</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bbtools</td>
      <td>39.08</td>
      <td>bbmap</td>
      <td>39.10</td>
      <td>RNA-Seq analysis, Sequence trimming, Read mapping, Sequence contamination filtering, Read binning, Sequence alignment</td>
      <td>RNA splicing, Whole genome sequencing, Phylogenetics, Metagenomics, RNA-Seq</td>
      <td>6</td>
      <td>3</td>
      <td>5</td>
      <td>0</td>
      <td>6</td>
      <td>3</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>11989</td>
      <td>11989</td>
      <td>768</td>
      <td>768</td>
      <td>4458</td>
      <td>4458</td>
      <td>652</td>
      <td>652</td>
      <td>2507</td>
      <td>2507</td>
      <td>80</td>
      <td>80</td>
      <td>18954</td>
      <td>18954</td>
      <td>1500</td>
      <td>1500</td>
    </tr>
    <tr>
      <td></td>
      <td>bcftools</td>
      <td>bcftools_@EXECUTABLE@, bcftools_@EXECUTABLE@, bcftools_@EXECUTABLE@, bcftools_@EXECUTABLE@, bcftools_@EXECUTABLE@, bcftools_@EXECUTABLE@_from_vcf, bcftools_@EXECUTABLE@_to_vcf, bcftools_@EXECUTABLE@, bcftools_@EXECUTABLE@, bcftools_@EXECUTABLE@, bcftools_@EXECUTABLE@, bcftools_@EXECUTABLE@, bcftools_@EXECUTABLE@, bcftools_@EXECUTABLE@, bcftools_plugin_@PLUGIN_ID@, bcftools_plugin_@PLUGIN_ID@, bcftools_plugin_@PLUGIN_ID@, bcftools_plugin_@PLUGIN_ID@, bcftools_plugin_@PLUGIN_ID@, bcftools_plugin_@PLUGIN_ID@, bcftools_plugin_@PLUGIN_ID@, bcftools_plugin_@PLUGIN_ID@, bcftools_plugin_@PLUGIN_ID@, bcftools_plugin_@PLUGIN_ID@, bcftools_plugin_@PLUGIN_ID@, bcftools_plugin_@PLUGIN_ID@, bcftools_plugin_@PLUGIN_ID@, bcftools_@EXECUTABLE@, bcftools_@EXECUTABLE@_list_samples, bcftools_@EXECUTABLE@, bcftools_@EXECUTABLE@, bcftools_@EXECUTABLE@, bcftools_@EXECUTABLE@</td>
      <td>BCFtools toolkit wrappers</td>
      <td>bcftools</td>
      <td>bcftools</td>
      <td></td>
      <td>BCFtools</td>
      <td>BCFtools is a set of utilities that manipulate variant calls in the Variant Call Format (VCF) and its binary counterpart BCF. All commands work transparently with both VCFs and BCFs, both uncompressed and BGZF-compressed.</td>
      <td>Data handling, Variant calling</td>
      <td>Genetic variation, DNA polymorphism, GWAS study, Genotyping experiment</td>
      <td>To update</td>
      <td>https://samtools.github.io/bcftools/</td>
      <td>Variant Analysis</td>
      <td></td>
      <td>2021-01-03</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bcftools</td>
      <td>1.15.1</td>
      <td>bcftools</td>
      <td>1.21</td>
      <td>Data handling, Variant calling</td>
      <td>DNA polymorphism, GWAS study, Genotyping experiment</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>bctools</td>
      <td>bctools_convert_to_binary_barcode, bctools_extract_crosslinked_nucleotides, bctools_extract_alignment_ends, bctools_extract_barcodes, bctools_merge_pcr_duplicates, bctools_remove_tail, bctools_remove_spurious_events</td>
      <td>bctools is a set of tools for handling barcodes and UMIs in NGS data.bctools can be used to merge PCR duplicates according to unique molecular barcodes (UMIs),to extract barcodes from arbitrary positions relative to the read starts,to clean up readthroughs into UMIs with paired-end sequencing andhandles binary barcodes as used with uvCLAP and FLASH.License: Apache License 2.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/dmaticzka/bctools</td>
      <td>Sequence Analysis, Transcriptomics</td>
      <td></td>
      <td>2017-10-09</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bedtools</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bctools</td>
      <td>0.2.2</td>
      <td>bctools</td>
      <td>0.2.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>2352</td>
      <td>3092</td>
      <td>234</td>
      <td>265</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2352</td>
      <td>3092</td>
      <td>234</td>
      <td>265</td>
    </tr>
    <tr>
      <td></td>
      <td>beacon2</td>
      <td>beacon2_csv2xlsx, beacon2_pxf2bff, beacon2_vcf2bff</td>
      <td>beacon2-ri-tools are part of the ELIXIR-CRG Beacon v2 Reference Implementation (B2RI).</td>
      <td>ga4gh_beacon</td>
      <td>ga4gh_beacon</td>
      <td></td>
      <td>GA4GH Beacon</td>
      <td>A global search engine for genetic mutations.</td>
      <td>Service discovery, Database search, Genetic variation analysis</td>
      <td>Genetic variation, Population genetics, Data security, Rare diseases</td>
      <td>Up-to-date</td>
      <td>https://github.com/EGA-archive/beacon2-ri-tools/tree/main</td>
      <td>Variant Analysis</td>
      <td>beacon2</td>
      <td>2023-08-17</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/beacon2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/beacon2</td>
      <td>2.0.0</td>
      <td>beacon2-ri-tools</td>
      <td>2.0.0</td>
      <td>Service discovery, Database search, Genetic variation analysis</td>
      <td>Genetic variation, Population genetics, Data security, Rare diseases</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>33</td>
      <td>33</td>
      <td>9</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>33</td>
      <td>33</td>
      <td>9</td>
      <td>9</td>
    </tr>
    <tr>
      <td></td>
      <td>beacon2-import</td>
      <td>beacon2_analyses, beacon2_biosamples, beacon2_bracket, beacon2_cnv, beacon2_cohorts, beacon2_datasets, beacon2_gene, beacon2_import, beacon2_individuals, beacon2_range, beacon2_runs, beacon2_sequence</td>
      <td>Beacon Import uploads local genetic data to the server, while Beacon Query searches for genetic information such as genes, sequences, and variants.</td>
      <td>ga4gh_beacon</td>
      <td>ga4gh_beacon</td>
      <td></td>
      <td>GA4GH Beacon</td>
      <td>A global search engine for genetic mutations.</td>
      <td>Service discovery, Database search, Genetic variation analysis</td>
      <td>Genetic variation, Population genetics, Data security, Rare diseases</td>
      <td>Up-to-date</td>
      <td>https://pypi.org/project/beacon2-import/</td>
      <td>Variant Analysis</td>
      <td>Beacon2_Import</td>
      <td>2024-07-22</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/beacon2-import</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/beacon2-import</td>
      <td>2.2.4</td>
      <td>beacon2-import</td>
      <td>2.2.4</td>
      <td>Service discovery, Database search, Genetic variation analysis</td>
      <td>Genetic variation, Population genetics, Data security, Rare diseases</td>
      <td>0</td>
      <td>0</td>
      <td>12</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>12</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>22</td>
      <td>22</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>22</td>
      <td>22</td>
      <td>7</td>
      <td>7</td>
    </tr>
    <tr>
      <td></td>
      <td>beagle</td>
      <td>beagle</td>
      <td>Beagle is a program for phasing and imputing missing genotypes.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://faculty.washington.edu/browning/beagle/beagle.html</td>
      <td>Variant Analysis</td>
      <td>beagle</td>
      <td>2021-07-03</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/beagle</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/beagle</td>
      <td>5.2_21Apr21.304</td>
      <td>beagle</td>
      <td>5.4_22Jul22.46e</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>88</td>
      <td>88</td>
      <td>28</td>
      <td>28</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>88</td>
      <td>88</td>
      <td>28</td>
      <td>28</td>
    </tr>
    <tr>
      <td></td>
      <td>bedops</td>
      <td>bedops-sort-bed</td>
      <td>BEDOPS: high-performance genomic feature operations</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://bedops.readthedocs.io/en/latest/</td>
      <td>Genomic Interval Operations</td>
      <td>bedops_sortbed</td>
      <td>2023-08-08</td>
      <td>iuc</td>
      <td>https://bedops.readthedocs.io/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bedops</td>
      <td>2.4.41</td>
      <td>bedops</td>
      <td>2.4.41</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>93</td>
      <td>93</td>
      <td>36</td>
      <td>36</td>
      <td>181</td>
      <td>181</td>
      <td>62</td>
      <td>62</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>274</td>
      <td>274</td>
      <td>98</td>
      <td>98</td>
    </tr>
    <tr>
      <td></td>
      <td>bedtools</td>
      <td>bedtools_annotatebed, bedtools_bamtobed, bedtools_bed12tobed6, bedtools_bedtobam, bedtools_bedtoigv, bedtools_bedpetobam, bedtools_closestbed, bedtools_clusterbed, bedtools_complementbed, bedtools_coveragebed, bedtools_expandbed, bedtools_fisher, bedtools_flankbed, bedtools_genomecoveragebed, bedtools_getfastabed, bedtools_groupbybed, bedtools_intersectbed, bedtools_jaccard, bedtools_links, bedtools_makewindowsbed, bedtools_map, bedtools_maskfastabed, bedtools_mergebed, bedtools_multicovtbed, bedtools_multiintersectbed, bedtools_nucbed, bedtools_overlapbed, bedtools_randombed, bedtools_reldistbed, bedtools_shufflebed, bedtools_slopbed, bedtools_sortbed, bedtools_spacingbed, bedtools_subtractbed, bedtools_tagbed, bedtools_unionbedgraph, bedtools_windowbed</td>
      <td>bedtools is a powerful toolset for genome arithmetic</td>
      <td>bedtools</td>
      <td>bedtools</td>
      <td></td>
      <td>BEDTools</td>
      <td>BEDTools is an extensive suite of utilities for comparing genomic features in BED format.</td>
      <td>Mapping</td>
      <td>Genomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/arq5x/bedtools2</td>
      <td>Genomic Interval Operations, Text Manipulation</td>
      <td>bedtools</td>
      <td>2019-09-05</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bedtools</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bedtools</td>
      <td>2.31.1</td>
      <td>bedtools</td>
      <td>2.31.1</td>
      <td>Mapping</td>
      <td>Genomics</td>
      <td>37</td>
      <td>37</td>
      <td>37</td>
      <td>37</td>
      <td>37</td>
      <td>37</td>
      <td>37</td>
      <td>37</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>37</td>
      <td>6</td>
      <td>37</td>
      <td>0</td>
      <td>0</td>
      <td>37</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>37</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>37</td>
      <td>37</td>
      <td>37</td>
      <td>0</td>
      <td>916111</td>
      <td>962278</td>
      <td>13235</td>
      <td>14525</td>
      <td>407944</td>
      <td>647621</td>
      <td>25849</td>
      <td>41751</td>
      <td>84008</td>
      <td>89233</td>
      <td>2486</td>
      <td>2832</td>
      <td>1408063</td>
      <td>1699132</td>
      <td>41570</td>
      <td>59108</td>
    </tr>
    <tr>
      <td></td>
      <td>bellerophon</td>
      <td>bellerophon</td>
      <td>Filter mapped reads where the mapping spans a junction, retaining the 5-prime read.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/davebx/bellerophon</td>
      <td>Sequence Analysis</td>
      <td>bellerophon</td>
      <td>2021-05-28</td>
      <td>iuc</td>
      <td>https://github.com/davebx/bellerophon</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bellerophon</td>
      <td>1.0</td>
      <td>bellerophon</td>
      <td>1.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1611</td>
      <td>1611</td>
      <td>262</td>
      <td>262</td>
      <td>763</td>
      <td>763</td>
      <td>67</td>
      <td>67</td>
      <td>543</td>
      <td>543</td>
      <td>30</td>
      <td>30</td>
      <td>2917</td>
      <td>2917</td>
      <td>359</td>
      <td>359</td>
    </tr>
    <tr>
      <td></td>
      <td>berokka</td>
      <td>berokka</td>
      <td>Berokka is used to trim, circularise, orient &amp; filter long read bacterial genome assemblies.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/tseemann/berokka</td>
      <td>Fasta Manipulation</td>
      <td>berokka</td>
      <td>2019-03-08</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/berokka</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/berokka</td>
      <td>0.2.3</td>
      <td>berokka</td>
      <td>0.2.3</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>617</td>
      <td>617</td>
      <td>63</td>
      <td>63</td>
      <td>617</td>
      <td>617</td>
      <td>63</td>
      <td>63</td>
    </tr>
    <tr>
      <td></td>
      <td>bigscape</td>
      <td>bigscape</td>
      <td>Construct sequence similarity networks of BGCs and groups them into GCF</td>
      <td>BiG-SCAPE</td>
      <td>BiG-SCAPE</td>
      <td></td>
      <td>BiG-SCAPE</td>
      <td>A computational framework to explore large-scale biosynthetic diversity.BiG-SCAPE (Biosynthetic Gene Similarity Clustering and Prospecting Engine) is a software package, written in Python, that constructs sequence similarity networks of Biosynthetic Gene Clusters (BGCs) and groups them into Gene Cluster Families (GCFs). BiG-SCAPE does this by rapidly calculating a distance matrix between gene clusters based on a comparison of their protein domain content, order, copy number and sequence identity.It defines a distance metric between Gene Clusters using a combination of three indices (Jaccard Index of domain types, Domain Sequence Similarity the Adjacency...BiG-SCAPE and CORASON provide a set of tools to explore the diversity of biosynthetic gene clusters (BGCs) across large numbers of genomes, by constructing BGC sequence similarity networks, grouping BGCs into gene cluster families, and exploring gene cluster diversity linked to enzyme phylogenies.</td>
      <td>Clustering, Global alignment, Fold recognition</td>
      <td>Phylogeny, Microbial ecology, Mapping, Metabolomics, Bioinformatics, Gene and protein families</td>
      <td>Up-to-date</td>
      <td>https://github.com/medema-group/BiG-SCAPE</td>
      <td>Metagenomics</td>
      <td>bigscape</td>
      <td>2024-02-18</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bigscape/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bigscape</td>
      <td>1.1.9</td>
      <td>bigscape</td>
      <td>1.1.9</td>
      <td>Clustering, Global alignment, Fold recognition</td>
      <td>Phylogeny, Microbial ecology, Mapping, Metabolomics, Bioinformatics, Gene and protein families</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>544</td>
      <td>544</td>
      <td>20</td>
      <td>20</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>544</td>
      <td>544</td>
      <td>20</td>
      <td>20</td>
    </tr>
    <tr>
      <td></td>
      <td>bigwig_outlier_bed</td>
      <td>bigwig_outlier_bed</td>
      <td>pybigtools and numpy code to find continuous runs above a high or below a low quantile cutpoint in bigwig files</td>
      <td>bigtools</td>
      <td>bigtools</td>
      <td></td>
      <td>bigtools</td>
      <td>Bigtools is a library and associated tools for reading and writing bigwig and bigbed files. Rust.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bigwig_outlier_bed</td>
      <td>Sequence Analysis</td>
      <td>bigwig_outlier_bed</td>
      <td>2024-07-05</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bigwig_outlier_bed</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bigwig_outlier_bed</td>
      <td>0.2.0</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>218</td>
      <td>218</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>218</td>
      <td>218</td>
      <td>5</td>
      <td>5</td>
    </tr>
    <tr>
      <td></td>
      <td>binning_refiner</td>
      <td>bin_refiner</td>
      <td>Reconciles the outputs of different binning programs with the aim to improve the quality of genome bins,especially with respect to contamination levels.</td>
      <td>binning_refiner</td>
      <td>binning_refiner</td>
      <td></td>
      <td>Binning_refiner</td>
      <td>Improving genome bins through the combination of different binning programs</td>
      <td>Read binning, Sequence clustering</td>
      <td>Metagenomics, Sequence assembly, Microbial ecology</td>
      <td>Up-to-date</td>
      <td>https://github.com/songweizhi/Binning_refiner</td>
      <td>Metagenomics</td>
      <td>binning_refiner</td>
      <td>2022-02-18</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/binning_refiner/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/binning_refiner</td>
      <td>1.4.3</td>
      <td>binning_refiner</td>
      <td>1.4.3</td>
      <td>Read binning, Sequence clustering</td>
      <td>Metagenomics, Sequence assembly, Microbial ecology</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>126</td>
      <td>126</td>
      <td>42</td>
      <td>42</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>126</td>
      <td>126</td>
      <td>42</td>
      <td>42</td>
    </tr>
    <tr>
      <td></td>
      <td>bioext</td>
      <td>bioext_bam2msa, bioext_bealign</td>
      <td>A suite of Galaxy tools designed around the BioExt extension to BioPython. Align sequences, merge duplicate sequences into one, and more!</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://pypi.python.org/pypi/biopython-extensions/</td>
      <td>Next Gen Mappers</td>
      <td></td>
      <td>2018-04-23</td>
      <td>iuc</td>
      <td>https://github.com/davebx/bioext-gx/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bioext</td>
      <td>0.21.9</td>
      <td>python-bioext</td>
      <td>0.21.9</td>
      <td></td>
      <td></td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>17413</td>
      <td>17413</td>
      <td>822</td>
      <td>822</td>
      <td>15349</td>
      <td>15349</td>
      <td>2020</td>
      <td>2020</td>
      <td>2</td>
      <td>2</td>
      <td>1</td>
      <td>1</td>
      <td>32764</td>
      <td>32764</td>
      <td>2843</td>
      <td>2843</td>
    </tr>
    <tr>
      <td></td>
      <td>bioinformatics_cafe</td>
      <td>fasta_regex_finder</td>
      <td>Miscellanea of scripts for bioinformatics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/dariober/bioinformatics-cafe/</td>
      <td>Sequence Analysis</td>
      <td>bioinformatics_cafe</td>
      <td>2023-01-25</td>
      <td>mbernt</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bioinformatics-cafe</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bioinformatics_cafe</td>
      <td>0.1.0</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1137</td>
      <td>1137</td>
      <td>97</td>
      <td>97</td>
      <td>1029</td>
      <td>1029</td>
      <td>71</td>
      <td>71</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2166</td>
      <td>2166</td>
      <td>168</td>
      <td>168</td>
    </tr>
    <tr>
      <td></td>
      <td>biom_format</td>
      <td>biom_add_metadata, biom_convert, biom_from_uc, biom_normalize_table, biom_subset_table, biom_summarize_table</td>
      <td>The biom-format package provides a command line interface and Python API for working with BIOM files.</td>
      <td>biomformat</td>
      <td>biomformat</td>
      <td></td>
      <td>biomformat</td>
      <td>This package includes basic tools for reading biom-format files, accessing and subsetting data tables from a biom object, as well as limited support for writing a biom-object back to a biom-format file. The design of this API is intended to match the python API and other tools included with the biom-format project, but with a decidedly "R flavor" that should be familiar to R users. This includes S4 classes and methods, as well as extensions of common core functions/methods.</td>
      <td>Formatting</td>
      <td>Laboratory information management, Sequence analysis</td>
      <td>To update</td>
      <td>https://github.com/biocore/biom-format</td>
      <td>Metagenomics</td>
      <td></td>
      <td>2016-06-07</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/biom_format</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/biom_format</td>
      <td>2.1.15</td>
      <td>biom-format</td>
      <td>2.1.7</td>
      <td>Formatting</td>
      <td>Laboratory information management, Sequence analysis</td>
      <td>2</td>
      <td>2</td>
      <td>6</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>12725</td>
      <td>12725</td>
      <td>297</td>
      <td>297</td>
      <td>2851</td>
      <td>5049</td>
      <td>450</td>
      <td>638</td>
      <td>2171</td>
      <td>2371</td>
      <td>214</td>
      <td>239</td>
      <td>17747</td>
      <td>20145</td>
      <td>961</td>
      <td>1174</td>
    </tr>
    <tr>
      <td></td>
      <td>bioperl</td>
      <td>bp_genbank2gff3</td>
      <td>Converts GenBank format files to GFF3</td>
      <td>bioperl</td>
      <td>bioperl</td>
      <td></td>
      <td>BioPerl</td>
      <td>A collection of Perl modules that facilitate the development of Perl scripts for bioinformatics applications.  It provides software modules for many of the typical tasks of bioinformatics programming.</td>
      <td>Data handling, Service invocation</td>
      <td>Genomics, Software engineering, Data management</td>
      <td>To update</td>
      <td>https://bioperl.org/</td>
      <td>Sequence Analysis</td>
      <td>bp_genbank2gff3</td>
      <td>2015-08-14</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bioperl</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bioperl</td>
      <td>1.1</td>
      <td>perl-bioperl</td>
      <td>1.7.8</td>
      <td>Data handling, Service invocation</td>
      <td>Genomics, Software engineering</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>7738</td>
      <td>7931</td>
      <td>478</td>
      <td>508</td>
      <td>8301</td>
      <td>8301</td>
      <td>849</td>
      <td>849</td>
      <td>1369</td>
      <td>1369</td>
      <td>154</td>
      <td>154</td>
      <td>17408</td>
      <td>17601</td>
      <td>1481</td>
      <td>1511</td>
    </tr>
    <tr>
      <td></td>
      <td>biotradis</td>
      <td>bacteria_tradis, tradis_essentiality, tradis_gene_insert_sites</td>
      <td>Bio-Tradis is a tool suite dedicated to essentiality analyses with TraDis data.</td>
      <td>biotradis</td>
      <td>biotradis</td>
      <td></td>
      <td>biotradis</td>
      <td>The Bio::TraDIS pipeline provides software utilities for the processing, mapping, and analysis of transposon insertion sequencing data. The pipeline was designed with the data from the TraDIS sequencing protocol in mind, but should work with a variety of transposon insertion sequencing protocols as long as they produce data in the expected format.</td>
      <td>Sequence analysis</td>
      <td>Mobile genetic elements, Workflows</td>
      <td>Up-to-date</td>
      <td>https://www.sanger.ac.uk/science/tools/bio-tradis</td>
      <td>Genome annotation</td>
      <td>biotradis</td>
      <td>2020-01-17</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/biotradis</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/biotradis</td>
      <td>1.4.5</td>
      <td>biotradis</td>
      <td>1.4.5</td>
      <td>Sequence analysis</td>
      <td>Mobile genetic elements, Workflows</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6243</td>
      <td>6243</td>
      <td>375</td>
      <td>375</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6243</td>
      <td>6243</td>
      <td>375</td>
      <td>375</td>
    </tr>
    <tr>
      <td></td>
      <td>biscot</td>
      <td>biscot</td>
      <td>Bionano scaffolding correction tool</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/institut-de-genomique/biscot</td>
      <td>Assembly</td>
      <td>biscot</td>
      <td>2023-01-06</td>
      <td>iuc</td>
      <td>https://github.com/bgruening/iuc/tree/master/tools/biscot</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/biscot</td>
      <td>2.3.3</td>
      <td>biscot</td>
      <td>2.3.3</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>1</td>
      <td>1</td>
    </tr>
    <tr>
      <td></td>
      <td>blast</td>
      <td>magicblast</td>
      <td>Maps large next-generation RNA or DNA sequencing runs against a whole genome or transcriptome</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://ncbi.github.io/magicblast/</td>
      <td>Next Gen Mappers</td>
      <td>magicblast</td>
      <td>2022-04-05</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/blast</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/blast</td>
      <td>1.7.0</td>
      <td>magicblast</td>
      <td>1.7.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>738</td>
      <td>738</td>
      <td>151</td>
      <td>151</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>1</td>
      <td>1</td>
      <td>740</td>
      <td>740</td>
      <td>152</td>
      <td>152</td>
    </tr>
    <tr>
      <td></td>
      <td>blastxml_to_gapped_gff3</td>
      <td>blastxml_to_gapped_gff3</td>
      <td>BlastXML to gapped GFF3</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/blastxml_to_gapped_gff3</td>
      <td>Convert Formats, Sequence Analysis</td>
      <td>blastxml_to_gapped_gff3</td>
      <td>2015-05-25</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/blastxml_to_gapped_gff3</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/blastxml_to_gapped_gff3</td>
      <td>1.1</td>
      <td>bcbiogff</td>
      <td>0.6.6</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>585</td>
      <td>585</td>
      <td>102</td>
      <td>102</td>
      <td>172</td>
      <td>172</td>
      <td>63</td>
      <td>63</td>
      <td>135</td>
      <td>135</td>
      <td>16</td>
      <td>16</td>
      <td>892</td>
      <td>892</td>
      <td>181</td>
      <td>181</td>
    </tr>
    <tr>
      <td></td>
      <td>bowtie2</td>
      <td>bowtie2</td>
      <td>Bowtie2: Fast and sensitive read alignment</td>
      <td>bowtie2</td>
      <td>bowtie2</td>
      <td></td>
      <td>Bowtie 2</td>
      <td>Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes. Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome, its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes.</td>
      <td>Read mapping</td>
      <td>Mapping, Genomics, Mapping</td>
      <td>To update</td>
      <td>http://bowtie-bio.sourceforge.net/bowtie2</td>
      <td>Next Gen Mappers</td>
      <td>bowtie2</td>
      <td>2018-04-10</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bowtie2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bowtie2</td>
      <td>2.5.3</td>
      <td>bowtie2</td>
      <td>2.5.4</td>
      <td>Read mapping</td>
      <td>Mapping, Genomics, Mapping</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>397893</td>
      <td>427447</td>
      <td>12133</td>
      <td>13176</td>
      <td>577209</td>
      <td>903193</td>
      <td>33086</td>
      <td>52020</td>
      <td>60105</td>
      <td>67495</td>
      <td>3779</td>
      <td>4271</td>
      <td>1035207</td>
      <td>1398135</td>
      <td>48998</td>
      <td>69467</td>
    </tr>
    <tr>
      <td></td>
      <td>bracken</td>
      <td>est_abundance</td>
      <td>Bayesian Reestimation of Abundance with KrakEN</td>
      <td>bracken</td>
      <td>bracken</td>
      <td></td>
      <td>Bracken</td>
      <td>Statistical method that computes the abundance of species in DNA sequences from a metagenomics sample.</td>
      <td>Statistical calculation</td>
      <td>Metagenomics, Microbial ecology</td>
      <td>Up-to-date</td>
      <td>https://ccb.jhu.edu/software/bracken/</td>
      <td>Sequence Analysis, Metagenomics</td>
      <td>bracken</td>
      <td>2019-10-15</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bracken</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bracken</td>
      <td>3.0</td>
      <td>bracken</td>
      <td>3.0</td>
      <td>Statistical calculation</td>
      <td>Metagenomics, Microbial ecology</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>51378</td>
      <td>51378</td>
      <td>775</td>
      <td>775</td>
      <td>3298</td>
      <td>3298</td>
      <td>160</td>
      <td>160</td>
      <td>1660</td>
      <td>1660</td>
      <td>63</td>
      <td>63</td>
      <td>56336</td>
      <td>56336</td>
      <td>998</td>
      <td>998</td>
    </tr>
    <tr>
      <td></td>
      <td>breseq</td>
      <td>breseq</td>
      <td>Predicts mutations in microbial genomes</td>
      <td>breseq</td>
      <td>breseq</td>
      <td></td>
      <td>breseq</td>
      <td>Runs Breseq software on a set of fastq files.</td>
      <td>Polymorphism detection</td>
      <td>Sequencing, Sequence analysis, DNA mutation</td>
      <td>To update</td>
      <td>https://github.com/barricklab/breseq</td>
      <td>Variant Analysis</td>
      <td>breseq</td>
      <td>2019-10-21</td>
      <td>iuc</td>
      <td></td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/breseq</td>
      <td>0.35.5</td>
      <td>breseq</td>
      <td>0.39.0</td>
      <td>Polymorphism detection</td>
      <td>Sequencing, Sequence analysis, DNA mutation</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2976</td>
      <td>2976</td>
      <td>98</td>
      <td>98</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>884</td>
      <td>884</td>
      <td>70</td>
      <td>70</td>
      <td>3860</td>
      <td>3860</td>
      <td>168</td>
      <td>168</td>
    </tr>
    <tr>
      <td></td>
      <td>brew3r_r</td>
      <td>brew3r_r</td>
      <td>Extend 3' end of a GTF using another GTF as a template</td>
      <td>brew3r.r</td>
      <td>brew3r.r</td>
      <td></td>
      <td>BREW3R.r</td>
      <td>This R package provide functions that are used in the BREW3R workflow. This mainly contains a function that extend a gtf as GRanges using information from another gtf (also as GRanges). The process allows to extend gene annotation without increasing the overlap between gene ids.</td>
      <td>Genome annotation</td>
      <td>Transcriptomics, Genomics</td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/BREW3R.r.html</td>
      <td>Transcriptomics, RNA</td>
      <td>brew3r_r</td>
      <td>2024-06-11</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/brew3r_r</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/brew3r_r</td>
      <td>1.0.2</td>
      <td></td>
      <td></td>
      <td>Genome annotation</td>
      <td>Transcriptomics</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>6</td>
      <td>6</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>3</td>
      <td>3</td>
    </tr>
    <tr>
      <td></td>
      <td>busco</td>
      <td>busco</td>
      <td>BUSCO assess genome and annotation completeness</td>
      <td>busco</td>
      <td>busco</td>
      <td></td>
      <td>BUSCO</td>
      <td>Provides measures for quantitative assessment of genome assembly, gene set, and transcriptome completeness based on evolutionarily informed expectations of gene content from near-universal single-copy orthologs.</td>
      <td>Sequence assembly validation, Scaffolding, Genome assembly, Transcriptome assembly</td>
      <td>Sequence assembly, Genomics, Transcriptomics, Sequence analysis</td>
      <td>To update</td>
      <td>https://gitlab.com/ezlab/busco/-/releases</td>
      <td>Sequence Analysis</td>
      <td>busco</td>
      <td>2019-12-01</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/busco/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/busco</td>
      <td>5.7.1</td>
      <td>busco</td>
      <td>5.8.0</td>
      <td>Sequence assembly validation, Scaffolding, Genome assembly, Transcriptome assembly</td>
      <td>Sequence assembly, Transcriptomics, Sequence analysis</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>102717</td>
      <td>103051</td>
      <td>3877</td>
      <td>3926</td>
      <td>34251</td>
      <td>34251</td>
      <td>3506</td>
      <td>3506</td>
      <td>33192</td>
      <td>34369</td>
      <td>2439</td>
      <td>2525</td>
      <td>170160</td>
      <td>171671</td>
      <td>9822</td>
      <td>9957</td>
    </tr>
    <tr>
      <td></td>
      <td>bwa</td>
      <td>bwa_mem, bwa</td>
      <td>Wrapper for bwa mem, aln, sampe, and samse</td>
      <td>bwa</td>
      <td>bwa</td>
      <td></td>
      <td>BWA</td>
      <td>Fast, accurate, memory-efficient aligner for short and long sequencing reads</td>
      <td>Genome indexing, Sequence alignment, Read mapping, Sequence alignment, Generation, Sequence alignment, Generation, Sequence alignment, Sequence alignment</td>
      <td>Mapping</td>
      <td>Up-to-date</td>
      <td>http://bio-bwa.sourceforge.net/</td>
      <td>Next Gen Mappers</td>
      <td>bwa</td>
      <td>2017-11-21</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bwa</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bwa</td>
      <td>0.7.18</td>
      <td>bwa</td>
      <td>0.7.18</td>
      <td>Genome indexing, Sequence alignment, Read mapping, Sequence alignment, Generation, Sequence alignment, Generation, Sequence alignment, Sequence alignment</td>
      <td>Mapping</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>976935</td>
      <td>1017671</td>
      <td>13526</td>
      <td>14401</td>
      <td>474702</td>
      <td>681286</td>
      <td>32627</td>
      <td>47624</td>
      <td>79802</td>
      <td>87191</td>
      <td>4219</td>
      <td>4954</td>
      <td>1531439</td>
      <td>1786148</td>
      <td>50372</td>
      <td>66979</td>
    </tr>
    <tr>
      <td></td>
      <td>bwa_mem2</td>
      <td>bwa_mem2</td>
      <td>Bwa-mem2 is the next version of the bwa-mem algorithm in bwa.</td>
      <td>bwa-mem2</td>
      <td>bwa-mem2</td>
      <td></td>
      <td>Bwa-mem2</td>
      <td>Bwa-mem2 is the next version of the bwa-mem algorithm in bwa. It produces alignment identical to bwa and is ~1.3-3.1x faster depending on the use-case,  dataset and the running machine.</td>
      <td>Sequence alignment</td>
      <td>Mapping</td>
      <td>Up-to-date</td>
      <td>https://github.com/bwa-mem2/bwa-mem2</td>
      <td>Next Gen Mappers</td>
      <td>bwa_mem2</td>
      <td>2021-10-08</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bwa_mem2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bwa_mem2</td>
      <td>2.2.1</td>
      <td>bwa-mem2</td>
      <td>2.2.1</td>
      <td>Sequence alignment</td>
      <td>Mapping</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>56619</td>
      <td>56619</td>
      <td>2324</td>
      <td>2324</td>
      <td>36631</td>
      <td>36631</td>
      <td>2808</td>
      <td>2808</td>
      <td>18121</td>
      <td>18121</td>
      <td>713</td>
      <td>713</td>
      <td>111371</td>
      <td>111371</td>
      <td>5845</td>
      <td>5845</td>
    </tr>
    <tr>
      <td></td>
      <td>bwameth</td>
      <td>bwameth</td>
      <td>Fast and accurate alignment of BS-seq reads</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/brentp/bwa-meth</td>
      <td>Sequence Analysis, Next Gen Mappers</td>
      <td>bwameth</td>
      <td>2016-09-14</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bwameth</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bwameth</td>
      <td>0.2.7</td>
      <td>bwameth</td>
      <td>0.2.7</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>7970</td>
      <td>11724</td>
      <td>643</td>
      <td>826</td>
      <td>5524</td>
      <td>5524</td>
      <td>611</td>
      <td>611</td>
      <td>1242</td>
      <td>1242</td>
      <td>114</td>
      <td>114</td>
      <td>14736</td>
      <td>18490</td>
      <td>1368</td>
      <td>1551</td>
    </tr>
    <tr>
      <td></td>
      <td>cactus</td>
      <td>cactus_cactus, cactus_export</td>
      <td>Cactus is a reference-free whole-genome multiple alignment program</td>
      <td>cactus</td>
      <td>cactus</td>
      <td></td>
      <td>Cactus</td>
      <td>Cactus is a reference-free whole-genome multiple alignment program.</td>
      <td>Multiple sequence alignment, Genome alignment</td>
      <td>Genomics, Sequence analysis, Phylogeny, Sequence assembly, Mapping, Phylogenetics</td>
      <td>To update</td>
      <td>https://github.com/ComparativeGenomicsToolkit/cactus</td>
      <td>Sequence Analysis</td>
      <td>cactus</td>
      <td>2023-01-11</td>
      <td>galaxy-australia</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cactus</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cactus</td>
      <td>2.7.1</td>
      <td></td>
      <td></td>
      <td>Multiple sequence alignment, Genome alignment</td>
      <td>Genomics, Sequence assembly, Mapping, Phylogenetics</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>1</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>299</td>
      <td>299</td>
      <td>61</td>
      <td>61</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1339</td>
      <td>1339</td>
      <td>95</td>
      <td>95</td>
      <td>1638</td>
      <td>1638</td>
      <td>156</td>
      <td>156</td>
    </tr>
    <tr>
      <td></td>
      <td>calculate_contrast_threshold</td>
      <td>calculate_contrast_threshold</td>
      <td>Calculates a contrast threshold from the CDT file generated by ``tag_pileup_frequency``. The calculated values are then used to set a uniform contrast for all the heatmaps generated downstream.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/CEGRcode/ChIP-QC-tools/tree/master/calculate_contrast_threshold</td>
      <td>Visualization, Genomic Interval Operations, SAM</td>
      <td>calculate_contrast_threshold</td>
      <td>2019-08-26</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/calculate_contrast_threshold</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/calculate_contrast_threshold</td>
      <td>1.0.0</td>
      <td>numpy</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>calculate_numeric_param</td>
      <td>calculate_numeric_param</td>
      <td>Calculate a numeric parameter value using integer and float values.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>calculate_numeric_param</td>
      <td>2021-05-04</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/calculate_numeric_param</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/calculate_numeric_param</td>
      <td>0.1.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1393</td>
      <td>1393</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1393</td>
      <td>1393</td>
      <td>10</td>
      <td>10</td>
    </tr>
    <tr>
      <td></td>
      <td>cami_amber</td>
      <td>biobox_add_taxid, cami_amber, cami_amber_add, cami_amber_convert</td>
      <td>Evaluation package for the comparative assessment of genome reconstructions and taxonomic assignments</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/CAMI-challenge/AMBER</td>
      <td>Metagenomics</td>
      <td>cami_amber</td>
      <td>2024-05-20</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/amber/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cami_amber</td>
      <td>2.0.7</td>
      <td>cami-amber</td>
      <td>2.0.7</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>195</td>
      <td>195</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>195</td>
      <td>195</td>
      <td>4</td>
      <td>4</td>
    </tr>
    <tr>
      <td></td>
      <td>cat</td>
      <td>cat_add_names, cat_bins, cat_contigs, cat_prepare, cat_summarise</td>
      <td>Contig Annotation Tool (CAT)</td>
      <td>cat_bins</td>
      <td>cat_bins</td>
      <td></td>
      <td>CAT and BAT</td>
      <td>Contig Annotation Tool (CAT) and Bin Annotation Tool (BAT) are pipelines for the taxonomic classification of long DNA sequences and metagenome assembled genomes (MAGs/bins) of both known and (highly) unknown microorganisms, as generated by contemporary metagenomics studies. The core algorithm of both programs involves gene calling, mapping of predicted ORFs against the nr protein database, and voting-based classification of the entire contig / MAG based on classification of the individual ORFs.</td>
      <td>Taxonomic classification, Sequence assembly, Coding region prediction</td>
      <td>Metagenomics, Metagenomic sequencing, Taxonomy, Sequence assembly</td>
      <td>To update</td>
      <td>https://github.com/dutilh/CAT</td>
      <td>Metagenomics</td>
      <td>contig_annotation_tool</td>
      <td>2019-11-27</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/cat</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cat</td>
      <td>5.2.3</td>
      <td>cat</td>
      <td>5.3</td>
      <td>Taxonomic classification, Sequence assembly, Coding region prediction</td>
      <td>Metagenomics, Metagenomic sequencing, Taxonomy, Sequence assembly</td>
      <td>5</td>
      <td>2</td>
      <td>5</td>
      <td>0</td>
      <td>5</td>
      <td>2</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>3835</td>
      <td>3835</td>
      <td>597</td>
      <td>597</td>
      <td>1359</td>
      <td>1359</td>
      <td>467</td>
      <td>467</td>
      <td>646</td>
      <td>646</td>
      <td>98</td>
      <td>98</td>
      <td>5840</td>
      <td>5840</td>
      <td>1162</td>
      <td>1162</td>
    </tr>
    <tr>
      <td></td>
      <td>cdhit</td>
      <td>cd_hit</td>
      <td>Cluster or compare biological sequence datasets</td>
      <td>cd-hit</td>
      <td>cd-hit</td>
      <td></td>
      <td>cd-hit</td>
      <td>Cluster a nucleotide dataset into representative sequences.</td>
      <td>Sequence clustering</td>
      <td>Sequencing</td>
      <td>Up-to-date</td>
      <td>http://weizhongli-lab.org/cd-hit/</td>
      <td>Sequence Analysis, Fasta Manipulation</td>
      <td>cd_hit</td>
      <td>2018-02-26</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cdhit</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cdhit</td>
      <td>4.8.1</td>
      <td>cd-hit</td>
      <td>4.8.1</td>
      <td>Sequence clustering</td>
      <td>Sequencing</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>10414</td>
      <td>10414</td>
      <td>510</td>
      <td>510</td>
      <td>2</td>
      <td>2</td>
      <td>1</td>
      <td>1</td>
      <td>2</td>
      <td>2</td>
      <td>1</td>
      <td>1</td>
      <td>10418</td>
      <td>10418</td>
      <td>512</td>
      <td>512</td>
    </tr>
    <tr>
      <td></td>
      <td>cemitool</td>
      <td>cemitool</td>
      <td>Gene co-expression network analysis tool</td>
      <td>cemitool</td>
      <td>cemitool</td>
      <td></td>
      <td>CEMiTool</td>
      <td>It unifies the discovery and the analysis of coexpression gene modules in a fully automatic manner, while providing a user-friendly html report with high quality graphs. Our tool evaluates if modules contain genes that are over-represented by specific pathways or that are altered in a specific sample group. Additionally, CEMiTool is able to integrate transcriptomic data with interactome information, identifying the potential hubs on each network.</td>
      <td>Enrichment analysis, Pathway or network analysis</td>
      <td>Gene expression, Transcriptomics, Microarray experiment</td>
      <td>Up-to-date</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/CEMiTool.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>cemitool</td>
      <td>2022-10-10</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cemitool</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cemitool</td>
      <td>1.26.0</td>
      <td>bioconductor-cemitool</td>
      <td>1.26.0</td>
      <td>Enrichment analysis, Pathway or network analysis</td>
      <td>Transcriptomics, Microarray experiment</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>225</td>
      <td>225</td>
      <td>32</td>
      <td>32</td>
      <td>271</td>
      <td>271</td>
      <td>61</td>
      <td>61</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>496</td>
      <td>496</td>
      <td>93</td>
      <td>93</td>
    </tr>
    <tr>
      <td></td>
      <td>charts</td>
      <td>charts</td>
      <td>Enables advanced visualization options in Galaxy Charts</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/charts/</td>
      <td>Visualization</td>
      <td>charts</td>
      <td>2018-01-30</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/charts/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/charts</td>
      <td>1.0.1</td>
      <td>r-getopt</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3383</td>
      <td>4001</td>
      <td>1576</td>
      <td>1822</td>
      <td>7870</td>
      <td>13727</td>
      <td>3823</td>
      <td>6880</td>
      <td>249</td>
      <td>350</td>
      <td>108</td>
      <td>145</td>
      <td>11502</td>
      <td>18078</td>
      <td>5507</td>
      <td>8847</td>
    </tr>
    <tr>
      <td></td>
      <td>checkm</td>
      <td>checkm_analyze, checkm_lineage_set, checkm_lineage_wf, checkm_plot, checkm_qa, checkm_taxon_set, checkm_taxonomy_wf, checkm_tetra, checkm_tree, checkm_tree_qa</td>
      <td>Assess the quality of microbial genomes recovered from isolates, single cells, and metagenomes</td>
      <td>checkm</td>
      <td>checkm</td>
      <td></td>
      <td>CheckM</td>
      <td>CheckM provides a set of tools for assessing the quality of genomes recovered from isolates, single cells, or metagenomes.</td>
      <td>Sequence assembly validation, Validation, Sequence composition calculation, Sequencing quality control, Statistical calculation</td>
      <td>Genomics, Phylogenomics, Phylogenetics, Taxonomy, Metagenomics, Data quality management</td>
      <td>Up-to-date</td>
      <td>https://github.com/Ecogenomics/CheckM</td>
      <td>Metagenomics</td>
      <td>checkm</td>
      <td>2022-07-29</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/checkm</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/checkm</td>
      <td>1.2.3</td>
      <td>checkm-genome</td>
      <td>1.2.3</td>
      <td>Sequence assembly validation, Sequence composition calculation, Statistical calculation</td>
      <td>Phylogenomics, Phylogenetics, Taxonomy, Metagenomics, Data quality management</td>
      <td>0</td>
      <td>10</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>10</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>7500</td>
      <td>7500</td>
      <td>930</td>
      <td>930</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7500</td>
      <td>7500</td>
      <td>930</td>
      <td>930</td>
    </tr>
    <tr>
      <td></td>
      <td>cherri</td>
      <td>cherri_eval, cherri_train</td>
      <td>Computational Help Evaluating RNA-RNA interactions</td>
      <td>cherri</td>
      <td>cherri</td>
      <td></td>
      <td>cherri</td>
      <td>CheRRI detects functional RNA-RNA interaction (RRI) sites, by evaluating if an interaction site most likely occurs in nature. It helps to filter interaction sites generated either experimentally or by an RRI prediction algorithm by removing false positive interactions.</td>
      <td></td>
      <td>Molecular interactions, pathways and networks, Structure analysis, Machine learning</td>
      <td>To update</td>
      <td>https://github.com/BackofenLab/Cherri</td>
      <td>Transcriptomics, RNA</td>
      <td></td>
      <td>2022-12-09</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cherri</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cherri</td>
      <td>0.7</td>
      <td>cherri</td>
      <td>0.8</td>
      <td></td>
      <td>Molecular interactions, pathways and networks, Structure analysis, Machine learning</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>207</td>
      <td>207</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>207</td>
      <td>207</td>
      <td>1</td>
      <td>1</td>
    </tr>
    <tr>
      <td></td>
      <td>chewbbaca</td>
      <td>chewbbaca_allelecall, chewbbaca_allelecallevaluator, chewbbaca_createschema, chewbbaca_downloadschema, chewbbaca_extractcgmlst, chewbbaca_joinprofiles, chewbbaca_nsstats, chewbbaca_prepexternalschema</td>
      <td>BSR-Based Allele Calling Algorithm</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/B-UMMI/chewBBACA/tree/master</td>
      <td>Variant Analysis</td>
      <td>chewbbaca</td>
      <td>2024-04-13</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/chewbbaca</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/chewbbaca</td>
      <td></td>
      <td>chewbbaca</td>
      <td>3.3.10</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>727</td>
      <td>727</td>
      <td>76</td>
      <td>76</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>727</td>
      <td>727</td>
      <td>76</td>
      <td>76</td>
    </tr>
    <tr>
      <td></td>
      <td>chira</td>
      <td>chira_collapse, chira_extract, chira_map, chira_merge, chira_quantify</td>
      <td>Chimeric Read Annotator for RNA-RNA interactome data</td>
      <td>chira</td>
      <td>chira</td>
      <td></td>
      <td>ChiRA</td>
      <td>ChiRA is a tool suite to analyze RNA-RNA interactome experimental data such as CLASH, CLEAR-CLIP, PARIS, SPLASH, etc.</td>
      <td></td>
      <td>RNA, Molecular interactions, pathways and networks, Functional, regulatory and non-coding RNA</td>
      <td>Up-to-date</td>
      <td>https://github.com/pavanvidem/chira</td>
      <td>RNA, Transcriptomics, Sequence Analysis</td>
      <td>chira</td>
      <td>2020-01-19</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/chira</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/chira</td>
      <td>1.4.3</td>
      <td>chira</td>
      <td>1.4.3</td>
      <td></td>
      <td>Molecular interactions, pathways and networks, Functional, regulatory and non-coding RNA</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>16542</td>
      <td>16542</td>
      <td>287</td>
      <td>287</td>
      <td>1061</td>
      <td>1061</td>
      <td>102</td>
      <td>102</td>
      <td>27</td>
      <td>27</td>
      <td>5</td>
      <td>5</td>
      <td>17630</td>
      <td>17630</td>
      <td>394</td>
      <td>394</td>
    </tr>
    <tr>
      <td></td>
      <td>chopin2</td>
      <td>chopin2</td>
      <td>Domain-Agnostic Supervised Learning with Hyperdimensional Computing</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/cumbof/chopin2</td>
      <td>Machine Learning</td>
      <td>chopin2</td>
      <td>2023-01-18</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/chopin2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/chopin2</td>
      <td>1.0.9.post1</td>
      <td>chopin2</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>14</td>
      <td>14</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>14</td>
      <td>14</td>
      <td>6</td>
      <td>6</td>
    </tr>
    <tr>
      <td></td>
      <td>chromeister</td>
      <td>chromeister</td>
      <td>ultra-fast pairwise genome comparisons</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/estebanpw/chromeister</td>
      <td>Sequence Analysis</td>
      <td>chromeister</td>
      <td>2020-09-30</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/chromeister</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/chromeister</td>
      <td>1.5.a</td>
      <td>chromeister</td>
      <td>1.5.a</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>2696</td>
      <td>2696</td>
      <td>398</td>
      <td>398</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1235</td>
      <td>1235</td>
      <td>186</td>
      <td>186</td>
      <td>3931</td>
      <td>3931</td>
      <td>584</td>
      <td>584</td>
    </tr>
    <tr>
      <td></td>
      <td>circexplorer2</td>
      <td>circexplorer2</td>
      <td>Comprehensive and integrative circular RNA analysis toolset.</td>
      <td>circexplorer2</td>
      <td>circexplorer2</td>
      <td></td>
      <td>CIRCexplorer2</td>
      <td>Genome-wide annotation of circRNAs and their alternative back-splicing/splicing.</td>
      <td></td>
      <td>RNA splicing, Gene transcripts, Literature and language</td>
      <td>Up-to-date</td>
      <td>https://github.com/YangLab/CIRCexplorer2</td>
      <td>RNA, Assembly</td>
      <td>circexplorer2</td>
      <td>2022-08-26</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/circexplorer2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/circexplorer2</td>
      <td>2.3.8</td>
      <td>circexplorer2</td>
      <td>2.3.8</td>
      <td></td>
      <td>RNA splicing, Gene transcripts, Literature and language</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>524</td>
      <td>524</td>
      <td>32</td>
      <td>32</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>524</td>
      <td>524</td>
      <td>32</td>
      <td>32</td>
    </tr>
    <tr>
      <td></td>
      <td>circos</td>
      <td>circos_aln_to_links, circos_binlinks, circos_bundlelinks, circos, circos_gc_skew, circos_resample, circos_wiggle_to_scatter, circos_wiggle_to_stacked, circos_tableviewer, circos_interval_to_text, circos_interval_to_tile</td>
      <td>Build Circos Plots in Galaxy</td>
      <td>galactic_circos</td>
      <td>galactic_circos</td>
      <td></td>
      <td>Galactic Circos</td>
      <td>Galactic Circos is a Galaxy wrapper providing a GUI for the Circos tool. Circos allows visualizing data in a circular format.</td>
      <td>Sequence visualisation</td>
      <td></td>
      <td>To update</td>
      <td>http://circos.ca/</td>
      <td>Graphics</td>
      <td>circos</td>
      <td>2020-05-25</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/circos</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/circos</td>
      <td>0.69.8</td>
      <td>circos</td>
      <td>0.69.9</td>
      <td>Sequence visualisation</td>
      <td></td>
      <td>11</td>
      <td>11</td>
      <td>11</td>
      <td>11</td>
      <td>11</td>
      <td>11</td>
      <td>11</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>11</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>11</td>
      <td>11</td>
      <td>11</td>
      <td>0</td>
      <td>26265</td>
      <td>26265</td>
      <td>2483</td>
      <td>2483</td>
      <td>31332</td>
      <td>31332</td>
      <td>3535</td>
      <td>3535</td>
      <td>26359</td>
      <td>26359</td>
      <td>999</td>
      <td>999</td>
      <td>83956</td>
      <td>83956</td>
      <td>7017</td>
      <td>7017</td>
    </tr>
    <tr>
      <td></td>
      <td>cite_seq_count</td>
      <td>cite_seq_count</td>
      <td>Count CMO/HTO</td>
      <td>CITE-seq-Count</td>
      <td>CITE-seq-Count</td>
      <td></td>
      <td>CITE-seq-Count</td>
      <td>Tool for counting antibody TAGS from a CITE-seq and/or cell hashing experiment.</td>
      <td>RNA-Seq quantification</td>
      <td>Transcriptomics, Immunoproteins and antigens</td>
      <td>To update</td>
      <td>https://github.com/Hoohm/CITE-seq-Count</td>
      <td>Single Cell, Transcriptomics, Proteomics</td>
      <td>cite_seq_count</td>
      <td>2023-01-18</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cite_seq_count</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cite_seq_count</td>
      <td>1.4.4</td>
      <td>cite-seq-count</td>
      <td>1.4.5</td>
      <td>RNA-Seq quantification</td>
      <td>Transcriptomics, Immunoproteins and antigens</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>21</td>
      <td>21</td>
      <td>4</td>
      <td>4</td>
      <td>9</td>
      <td>9</td>
      <td>5</td>
      <td>5</td>
      <td>18</td>
      <td>18</td>
      <td>1</td>
      <td>1</td>
      <td>48</td>
      <td>48</td>
      <td>10</td>
      <td>10</td>
    </tr>
    <tr>
      <td></td>
      <td>clair3</td>
      <td>clair3</td>
      <td>Symphonizing pileup and full-alignment for high-performance long-read variant calling</td>
      <td>clair3</td>
      <td>clair3</td>
      <td></td>
      <td>Clair3</td>
      <td>Clair3 is a germline small variant caller for long-reads. Clair3 makes the best of two major method categories: pileup calling handles most variant candidates with speed, and full-alignment tackles complicated candidates to maximize precision and recall. Clair3 runs fast and has superior performance, especially at lower coverage. Clair3 is simple and modular for easy deployment and integration.</td>
      <td>Variant calling</td>
      <td>Molecular genetics</td>
      <td>To update</td>
      <td>https://github.com/HKU-BAL/Clair3</td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>clair3</td>
      <td>2022-06-15</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/clair3</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/clair3</td>
      <td>1.0.8</td>
      <td>clair3</td>
      <td>1.0.10</td>
      <td>Variant calling</td>
      <td>Molecular genetics</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>3388</td>
      <td>3388</td>
      <td>187</td>
      <td>187</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>3390</td>
      <td>3390</td>
      <td>189</td>
      <td>189</td>
    </tr>
    <tr>
      <td></td>
      <td>clustalw</td>
      <td>clustalw</td>
      <td>ClustalW multiple sequence alignment program for DNA or proteins</td>
      <td>clustal2</td>
      <td>clustal2</td>
      <td></td>
      <td>Clustal 2 (Clustal W, Clustal X)</td>
      <td>Multiple sequence alignment program with a command-line interface (Clustal W) and a graphical user interface (Clustal X). The display colours allow conserved features to be highlighted for easy viewing in the alignment. It is available for several platforms, including Windows, Macintosh PowerMac, Linux and Solaris.Names occassionally spelled also as Clustal W2, ClustalW2, ClustalW, ClustalX, Clustal2.</td>
      <td>Multiple sequence alignment</td>
      <td>Phylogeny, Sequence analysis</td>
      <td>Up-to-date</td>
      <td>http://www.clustal.org/clustal2/</td>
      <td>Phylogenetics, Sequence Analysis</td>
      <td>clustalw</td>
      <td>2022-10-02</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/clustalw</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/clustalw</td>
      <td>2.1</td>
      <td>clustalw</td>
      <td>2.1</td>
      <td>Multiple sequence alignment</td>
      <td>Phylogeny, Sequence analysis</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>79867</td>
      <td>80817</td>
      <td>1675</td>
      <td>1785</td>
      <td>49493</td>
      <td>87884</td>
      <td>3703</td>
      <td>7380</td>
      <td>8797</td>
      <td>9713</td>
      <td>670</td>
      <td>728</td>
      <td>138157</td>
      <td>178414</td>
      <td>6048</td>
      <td>9893</td>
    </tr>
    <tr>
      <td></td>
      <td>clustering_from_distmat</td>
      <td>clustering_from_distmat</td>
      <td>Distance matrix-based hierarchical clustering using SciPy</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/clustering_from_distmat/</td>
      <td>Statistics</td>
      <td>clustering_from_distmat</td>
      <td>2024-08-08</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/clustering_from_distmat/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/clustering_from_distmat</td>
      <td>1.1.1</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>12</td>
      <td>12</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>12</td>
      <td>12</td>
      <td>5</td>
      <td>5</td>
    </tr>
    <tr>
      <td></td>
      <td>cnv-phenopacket</td>
      <td>cnv_phenopacket</td>
      <td>cnv-phenopacket Converts TSV metadata file to JSON.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://pypi.org/project/cnv-phenopacket/</td>
      <td>Variant Analysis</td>
      <td>cnv_phenopacket</td>
      <td>2024-02-27</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cnv-phenopacket</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cnv-phenopacket</td>
      <td>1.0.2</td>
      <td>cnv-phenopacket</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
    </tr>
    <tr>
      <td></td>
      <td>cnv-vcf2json</td>
      <td>cnv_vcf2json</td>
      <td>cnv-vcf2json Converts structural variants VCF file to JSON.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://pypi.org/project/cnv-phenopacket/</td>
      <td>Variant Analysis</td>
      <td>cnv-vcf2json</td>
      <td>2024-02-27</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cnv-vcf2json</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cnv-vcf2json</td>
      <td>1.1.0</td>
      <td>cnv-vcf2json</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>9</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>9</td>
      <td>5</td>
      <td>5</td>
    </tr>
    <tr>
      <td></td>
      <td>cnvkit</td>
      <td>cnvkit_access, cnvkit_antitarget, cnvkit_autobin, cnvkit_batch, cnvkit_breaks, cnvkit_call, cnvkit_coverage, cnvkit_diagram, cnvkit_fix, cnvkit_genemetrics, cnvkit_heatmap, cnvkit_reference, cnvkit_scatter, cnvkit_segment, cnvkit_segmetrics, cnvkit_sex, cnvkit_target</td>
      <td>detecting copy number variants and alterations genome-wide from high-throughput sequencing</td>
      <td>cnvkit</td>
      <td>cnvkit</td>
      <td></td>
      <td>CNVkit</td>
      <td>CNVkit is a software toolkit to infer and visualize copy number from targeted DNA sequencing data.</td>
      <td>Variant calling</td>
      <td>DNA structural variation</td>
      <td>Up-to-date</td>
      <td>https://github.com/etal/cnvkit</td>
      <td>Variant Analysis</td>
      <td>cnvkit</td>
      <td>2023-05-14</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cnvkit</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cnvkit</td>
      <td>0.9.11</td>
      <td>cnvkit</td>
      <td>0.9.11</td>
      <td>Variant calling</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>17</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>17</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>17</td>
      <td>0</td>
      <td>0</td>
      <td>618</td>
      <td>618</td>
      <td>177</td>
      <td>177</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>618</td>
      <td>618</td>
      <td>177</td>
      <td>177</td>
    </tr>
    <tr>
      <td></td>
      <td>codeml</td>
      <td>codeml</td>
      <td>Detects positive selection</td>
      <td>paml</td>
      <td>paml</td>
      <td></td>
      <td>PAML</td>
      <td>Package of programs for phylogenetic analyses of DNA or protein sequences using maximum likelihood.</td>
      <td>Probabilistic sequence generation, Phylogenetic tree generation (maximum likelihood and Bayesian methods), Phylogenetic tree analysis</td>
      <td>Phylogenetics, Sequence analysis</td>
      <td>To update</td>
      <td>http://abacus.gene.ucl.ac.uk/software/paml.html</td>
      <td>Phylogenetics</td>
      <td>codeml</td>
      <td>2017-07-19</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/codeml</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/codeml</td>
      <td>4.9</td>
      <td>paml</td>
      <td>4.10.7</td>
      <td>Probabilistic sequence generation</td>
      <td>Phylogenetics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>61197</td>
      <td>61220</td>
      <td>64</td>
      <td>65</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>61197</td>
      <td>61220</td>
      <td>64</td>
      <td>65</td>
    </tr>
    <tr>
      <td></td>
      <td>cojac</td>
      <td>cooc_mutbamscan, cooc_pubmut, cooc_tabmut</td>
      <td>co-occurrence of mutations on amplicons</td>
      <td>cojac</td>
      <td>cojac</td>
      <td></td>
      <td>COJAC</td>
      <td>CoOccurrence adJusted Analysis and Calling - The cojac package comprises a set of command-line tools to analyse co-occurrence of mutations on amplicons. It is useful, for example, for early detection of viral variants of concern (e.g. Alpha, Delta, Omicron) in environmental samples, and has been designed to scan for multiple SARS-CoV-2 variants in wastewater samples, as analyzed jointly by ETH Zurich, EPFL and Eawag.</td>
      <td></td>
      <td>Genetic variation</td>
      <td>Up-to-date</td>
      <td>https://github.com/cbg-ethz/cojac</td>
      <td>Metagenomics, Sequence Analysis</td>
      <td>cojac</td>
      <td>2022-08-11</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cojac</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cojac</td>
      <td>0.9.2</td>
      <td>cojac</td>
      <td>0.9.2</td>
      <td></td>
      <td>Genetic variation</td>
      <td>2</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>2758</td>
      <td>2758</td>
      <td>22</td>
      <td>22</td>
      <td>20</td>
      <td>20</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2778</td>
      <td>2778</td>
      <td>32</td>
      <td>32</td>
    </tr>
    <tr>
      <td></td>
      <td>colabfold</td>
      <td>colabfold_alphafold, colabfold_msa</td>
      <td>Protein prediction based on AlphaFold2</td>
      <td>Colabfold</td>
      <td>Colabfold</td>
      <td></td>
      <td>ColabFold</td>
      <td>ColabFold databases are MMseqs2 expandable profile databases to generate diverse multiple sequence alignments to predict protein structures.</td>
      <td>Database search, Protein structure prediction, Fold recognition</td>
      <td>Protein folds and structural domains, Protein folding, stability and design, Structure prediction, Sequence sites, features and motifs, Metagenomics</td>
      <td>To update</td>
      <td>https://github.com/sokrypton/ColabFold</td>
      <td>Proteomics, Graphics</td>
      <td>colabfold</td>
      <td>2024-03-24</td>
      <td>iuc</td>
      <td>https://github.com/sokrypton/ColabFold</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/colabfold</td>
      <td>1.5.5</td>
      <td></td>
      <td></td>
      <td>Database search, Protein structure prediction, Fold recognition</td>
      <td>Protein folds and structural domains, Protein folding, stability and design, Structure prediction, Sequence sites, features and motifs, Metagenomics</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6771</td>
      <td>6771</td>
      <td>50</td>
      <td>50</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6771</td>
      <td>6771</td>
      <td>50</td>
      <td>50</td>
    </tr>
    <tr>
      <td></td>
      <td>colibread</td>
      <td>commet, discosnp_rad, discosnp_pp, kissplice, lordec, mapsembler2, takeabreak</td>
      <td>Colib'read tools are all dedicated to the analysis of NGS datasets without the need of any reference genome</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://colibread.inria.fr/</td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>colibread</td>
      <td>2017-10-06</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/colibread</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/colibread</td>
      <td>24.7.14</td>
      <td>commet</td>
      <td>24.7.14</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>273</td>
      <td>273</td>
      <td>40</td>
      <td>40</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>273</td>
      <td>273</td>
      <td>40</td>
      <td>40</td>
    </tr>
    <tr>
      <td></td>
      <td>collection_column_join</td>
      <td>collection_column_join</td>
      <td>Column Join on Collections</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/collection_column_join</td>
      <td>Text Manipulation</td>
      <td>collection_column_join</td>
      <td>2016-05-26</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/collection_column_join</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/collection_column_join</td>
      <td>0.0.3</td>
      <td>coreutils</td>
      <td>8.25</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>23979</td>
      <td>25337</td>
      <td>3827</td>
      <td>4011</td>
      <td>21671</td>
      <td>25163</td>
      <td>4645</td>
      <td>5045</td>
      <td>2939</td>
      <td>3036</td>
      <td>688</td>
      <td>701</td>
      <td>48589</td>
      <td>53536</td>
      <td>9160</td>
      <td>9757</td>
    </tr>
    <tr>
      <td></td>
      <td>collection_element_identifiers</td>
      <td>collection_element_identifiers</td>
      <td>Extract element identifiers of a collection</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/collection_element_identifiers</td>
      <td>Text Manipulation</td>
      <td>collection_element_identifiers</td>
      <td>2018-06-13</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/collection_element_identifiers</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/collection_element_identifiers</td>
      <td>0.0.2</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>6905</td>
      <td>7102</td>
      <td>1620</td>
      <td>1630</td>
      <td>7761</td>
      <td>7761</td>
      <td>1897</td>
      <td>1897</td>
      <td>1157</td>
      <td>1157</td>
      <td>243</td>
      <td>243</td>
      <td>15823</td>
      <td>16020</td>
      <td>3760</td>
      <td>3770</td>
    </tr>
    <tr>
      <td></td>
      <td>column_maker</td>
      <td>Add_a_column1</td>
      <td>Compute an expression on every row</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/column_maker</td>
      <td>Text Manipulation</td>
      <td>column_maker</td>
      <td>2019-06-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/column_maker</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/column_maker</td>
      <td>2.1</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2940752</td>
      <td>3907183</td>
      <td>5422</td>
      <td>6121</td>
      <td>263254</td>
      <td>737503</td>
      <td>6095</td>
      <td>13728</td>
      <td>96478</td>
      <td>97667</td>
      <td>884</td>
      <td>974</td>
      <td>3300484</td>
      <td>4742353</td>
      <td>12401</td>
      <td>20823</td>
    </tr>
    <tr>
      <td></td>
      <td>column_order_header_sort</td>
      <td>column_order_header_sort</td>
      <td>Sort Column Order by heading</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/column_order_header_sort</td>
      <td>Text Manipulation</td>
      <td>column_order_header_sort</td>
      <td>2017-04-04</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/column_order_header_sort</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/column_order_header_sort</td>
      <td>0.0.1</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>3735</td>
      <td>3891</td>
      <td>268</td>
      <td>297</td>
      <td>1923</td>
      <td>1923</td>
      <td>381</td>
      <td>381</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5658</td>
      <td>5814</td>
      <td>649</td>
      <td>678</td>
    </tr>
    <tr>
      <td></td>
      <td>column_remove_by_header</td>
      <td>column_remove_by_header</td>
      <td>Remove columns by header</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/column_remove_by_header</td>
      <td>Text Manipulation</td>
      <td>column_remove_by_header</td>
      <td>2017-04-04</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/column_remove_by_header</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/column_remove_by_header</td>
      <td>1.0</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>8331</td>
      <td>8917</td>
      <td>613</td>
      <td>662</td>
      <td>5206</td>
      <td>5206</td>
      <td>716</td>
      <td>716</td>
      <td>1976</td>
      <td>1976</td>
      <td>27</td>
      <td>27</td>
      <td>15513</td>
      <td>16099</td>
      <td>1356</td>
      <td>1405</td>
    </tr>
    <tr>
      <td></td>
      <td>compleasm</td>
      <td>compleasm</td>
      <td>Compleasm: a faster and more accurate reimplementation of BUSCO</td>
      <td>compleasm</td>
      <td>compleasm</td>
      <td></td>
      <td>compleasm</td>
      <td>"Compleasm: a faster and more accurate reimplementation of BUSCO"</td>
      <td>Sequence assembly validation, Sequence analysis, Scaffolding, Transcriptome assembly</td>
      <td>Sequence assembly, Genomics, Transcriptomics, Sequence analysis</td>
      <td>Up-to-date</td>
      <td>https://github.com/huangnengCSU/compleasm</td>
      <td>Sequence Analysis</td>
      <td>compleasm</td>
      <td>2023-12-04</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/compleasm/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/compleasm</td>
      <td>0.2.6</td>
      <td>compleasm</td>
      <td>0.2.6</td>
      <td>Sequence assembly validation, Sequence analysis, Scaffolding, Transcriptome assembly</td>
      <td>Sequence assembly, Transcriptomics, Sequence analysis</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>266</td>
      <td>266</td>
      <td>39</td>
      <td>39</td>
      <td>28</td>
      <td>28</td>
      <td>15</td>
      <td>15</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>294</td>
      <td>294</td>
      <td>54</td>
      <td>54</td>
    </tr>
    <tr>
      <td></td>
      <td>compose_text_param</td>
      <td>compose_text_param</td>
      <td>Compose a text parameter value using text, integer and float values</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>compose_text_param</td>
      <td>2019-05-09</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/compose_text_param</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/compose_text_param</td>
      <td>0.1.1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>45827</td>
      <td>45831</td>
      <td>689</td>
      <td>691</td>
      <td>7721</td>
      <td>7721</td>
      <td>303</td>
      <td>303</td>
      <td>1481</td>
      <td>1481</td>
      <td>121</td>
      <td>121</td>
      <td>55029</td>
      <td>55033</td>
      <td>1113</td>
      <td>1115</td>
    </tr>
    <tr>
      <td></td>
      <td>compress_file</td>
      <td>compress_file</td>
      <td>Compress files.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/compress_file</td>
      <td>Text Manipulation</td>
      <td>compress_file</td>
      <td>2022-02-04</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/compress_file</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/compress_file</td>
      <td>0.1.0</td>
      <td>gzip</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>6343</td>
      <td>6343</td>
      <td>290</td>
      <td>290</td>
      <td>3674</td>
      <td>3674</td>
      <td>286</td>
      <td>286</td>
      <td>11832</td>
      <td>11832</td>
      <td>59</td>
      <td>59</td>
      <td>21849</td>
      <td>21849</td>
      <td>635</td>
      <td>635</td>
    </tr>
    <tr>
      <td></td>
      <td>concoct</td>
      <td>concoct, concoct_coverage_table, concoct_cut_up_fasta, concoct_extract_fasta_bins, concoct_merge_cut_up_clustering</td>
      <td>CONCOCT (Clustering cONtigs with COverage and ComposiTion) does unsupervised binning of metagenomic contigs byusing nucleotide composition - kmer frequencies - and coverage data for multiple samples. CONCOCT can accurately(up to species level) bin metagenomic contigs.</td>
      <td>concoct</td>
      <td>concoct</td>
      <td></td>
      <td>CONCOCT</td>
      <td>A program for unsupervised binning of metagenomic contigs by using nucleotide composition, coverage data in multiple samples and linkage data from paired end reads.</td>
      <td>Sequence clustering, Read binning</td>
      <td>Metagenomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/BinPro/CONCOCT</td>
      <td>Metagenomics</td>
      <td>concoct</td>
      <td>2022-02-18</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/concoct</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/concoct</td>
      <td>1.1.0</td>
      <td>concoct</td>
      <td>1.1.0</td>
      <td>Sequence clustering, Read binning</td>
      <td>Metagenomics</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>2105</td>
      <td>2105</td>
      <td>155</td>
      <td>155</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2105</td>
      <td>2105</td>
      <td>155</td>
      <td>155</td>
    </tr>
    <tr>
      <td></td>
      <td>cosg</td>
      <td>cosg</td>
      <td>Marker gene identification for single-cell sequencing data using COSG.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/genecell/COSG</td>
      <td>Transcriptomics, Sequence Analysis, Single Cell</td>
      <td>cosg</td>
      <td>2024-05-29</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cosg/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cosg</td>
      <td>1.0.1</td>
      <td>cosg</td>
      <td>1.0.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>coverage_report</td>
      <td>CoverageReport2</td>
      <td>Generate Detailed Coverage Report from BAM file</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc</td>
      <td>Sequence Analysis</td>
      <td>coverage_report</td>
      <td>2017-10-16</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/coverage_report</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/coverage_report</td>
      <td>0.0.4</td>
      <td>perl-number-format</td>
      <td>1.76</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>coverm</td>
      <td>coverm_contig, coverm_genome</td>
      <td>CoverM genome and contig wrappers</td>
      <td>coverm</td>
      <td>coverm</td>
      <td></td>
      <td>CoverM</td>
      <td>Read coverage calculator for metagenomics</td>
      <td>Local alignment</td>
      <td>Bioinformatics</td>
      <td>Up-to-date</td>
      <td>https://github.com/wwood/CoverM</td>
      <td>Sequence Analysis</td>
      <td>coverm</td>
      <td>2022-04-26</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tools/coverm</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/coverm</td>
      <td>0.7.0</td>
      <td>coverm</td>
      <td>0.7.0</td>
      <td>Local alignment</td>
      <td>Bioinformatics</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>1081</td>
      <td>1081</td>
      <td>151</td>
      <td>151</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1081</td>
      <td>1081</td>
      <td>151</td>
      <td>151</td>
    </tr>
    <tr>
      <td></td>
      <td>crispr_studio</td>
      <td>crispr_studio</td>
      <td>CRISPR Studio is a program developed to facilitate and accelerate CRISPR array visualization.</td>
      <td>crisprstudio</td>
      <td>crisprstudio</td>
      <td></td>
      <td>CRISPRStudio</td>
      <td>CRISPRStudio is a program developed to facilitate and accelerate CRISPR array visualization. It works by first comparing spacers sequence homology in a dataset, then assigning a two-color-code to each cluster of spacers and finally writing an svg file, which can be opened in graphics vector editor.</td>
      <td>Visualisation</td>
      <td>Sequence analysis, Genomics, Data visualisation</td>
      <td>To update</td>
      <td>https://github.com/moineaulab/CRISPRStudio</td>
      <td>Sequence Analysis</td>
      <td>crispr_studio</td>
      <td>2019-08-13</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/crispr_studio/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/crispr_studio</td>
      <td>1+galaxy0</td>
      <td>crispr_studio</td>
      <td>1</td>
      <td>Visualisation</td>
      <td>Sequence analysis, Genomics, Data visualisation</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1052</td>
      <td>1053</td>
      <td>75</td>
      <td>76</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1052</td>
      <td>1053</td>
      <td>75</td>
      <td>76</td>
    </tr>
    <tr>
      <td></td>
      <td>crosscontamination_barcode_filter</td>
      <td>crosscontamination_barcode_filter</td>
      <td>Barcode contamination discovery tool</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, Visualization</td>
      <td>crosscontamination_barcode_filter</td>
      <td>2018-07-27</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/crosscontamination_barcode_filter</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/crosscontamination_barcode_filter</td>
      <td>0.3</td>
      <td>r-ggplot2</td>
      <td>2.2.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>276</td>
      <td>390</td>
      <td>61</td>
      <td>76</td>
      <td>131</td>
      <td>131</td>
      <td>42</td>
      <td>42</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>407</td>
      <td>521</td>
      <td>103</td>
      <td>118</td>
    </tr>
    <tr>
      <td></td>
      <td>crossmap</td>
      <td>crossmap_bam, crossmap_bed, crossmap_bw, crossmap_gff, crossmap_region, crossmap_vcf, crossmap_wig</td>
      <td>CrossMap converts genome coordinates or annotation files between genome assemblies</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://crossmap.sourceforge.net/</td>
      <td>Convert Formats, Genomic Interval Operations</td>
      <td>crossmap</td>
      <td>2017-07-02</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/crossmap</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/crossmap</td>
      <td>0.7.0</td>
      <td>crossmap</td>
      <td>0.7.0</td>
      <td></td>
      <td></td>
      <td>5</td>
      <td>0</td>
      <td>6</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>6</td>
      <td>5</td>
      <td>0</td>
      <td>2182</td>
      <td>2182</td>
      <td>331</td>
      <td>331</td>
      <td>5251</td>
      <td>5251</td>
      <td>903</td>
      <td>903</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7433</td>
      <td>7433</td>
      <td>1234</td>
      <td>1234</td>
    </tr>
    <tr>
      <td></td>
      <td>cuffcompare</td>
      <td>cuffcompare</td>
      <td>Galaxy wrappers for the Cuffcompare tool.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://cole-trapnell-lab.github.io/cufflinks/</td>
      <td>Transcriptomics</td>
      <td>cuffcompare</td>
      <td>2020-06-08</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/cufflinks/cuffcompare</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/cufflinks/cuffcompare</td>
      <td>2.2.1</td>
      <td>cufflinks</td>
      <td>2.2.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>768</td>
      <td>1199</td>
      <td>171</td>
      <td>300</td>
      <td>2205</td>
      <td>24981</td>
      <td>754</td>
      <td>5750</td>
      <td>251</td>
      <td>493</td>
      <td>46</td>
      <td>98</td>
      <td>3224</td>
      <td>26673</td>
      <td>971</td>
      <td>6148</td>
    </tr>
    <tr>
      <td></td>
      <td>cuffdiff</td>
      <td>cuffdiff</td>
      <td>Galaxy wrappers for the Cuffdiff tool.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://cole-trapnell-lab.github.io/cufflinks/</td>
      <td>Transcriptomics</td>
      <td>cuffdiff</td>
      <td>2020-06-08</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/cufflinks/cuffdiff</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/cufflinks/cuffdiff</td>
      <td>2.2.1</td>
      <td>cufflinks</td>
      <td>2.2.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>4176</td>
      <td>6065</td>
      <td>564</td>
      <td>725</td>
      <td>24284</td>
      <td>117007</td>
      <td>4952</td>
      <td>20249</td>
      <td>971</td>
      <td>4231</td>
      <td>193</td>
      <td>616</td>
      <td>29431</td>
      <td>127303</td>
      <td>5709</td>
      <td>21590</td>
    </tr>
    <tr>
      <td></td>
      <td>cufflinks</td>
      <td>cufflinks</td>
      <td>Galaxy wrappers for the Cufflinks tool.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://cole-trapnell-lab.github.io/cufflinks/</td>
      <td>Transcriptomics</td>
      <td>cufflinks</td>
      <td>2020-06-08</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/cufflinks/cufflinks</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/cufflinks/cufflinks</td>
      <td>2.2.1</td>
      <td>cufflinks</td>
      <td>2.2.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>26122</td>
      <td>33320</td>
      <td>870</td>
      <td>1213</td>
      <td>70653</td>
      <td>349091</td>
      <td>6984</td>
      <td>27076</td>
      <td>3963</td>
      <td>10315</td>
      <td>440</td>
      <td>836</td>
      <td>100738</td>
      <td>392726</td>
      <td>8294</td>
      <td>29125</td>
    </tr>
    <tr>
      <td></td>
      <td>cuffmerge</td>
      <td>cuffmerge</td>
      <td>Galaxy wrappers for the Cuffmerge tool.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://cole-trapnell-lab.github.io/cufflinks/</td>
      <td>Transcriptomics</td>
      <td>cuffmerge</td>
      <td>2020-06-08</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/cufflinks/cuffmerge</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/cufflinks/cuffmerge</td>
      <td>2.2.1</td>
      <td>cufflinks</td>
      <td>2.2.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2406</td>
      <td>3448</td>
      <td>351</td>
      <td>637</td>
      <td>20978</td>
      <td>67769</td>
      <td>3641</td>
      <td>13764</td>
      <td>1209</td>
      <td>3057</td>
      <td>227</td>
      <td>573</td>
      <td>24593</td>
      <td>74274</td>
      <td>4219</td>
      <td>14974</td>
    </tr>
    <tr>
      <td></td>
      <td>cuffnorm</td>
      <td>cuffnorm</td>
      <td>The Cuffnorm tool</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://cole-trapnell-lab.github.io/cufflinks/</td>
      <td>Transcriptomics</td>
      <td>cuffnorm</td>
      <td>2020-06-08</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/cufflinks/cuffnorm</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/cufflinks/cuffnorm</td>
      <td>2.2.1</td>
      <td>cufflinks</td>
      <td>2.2.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>374</td>
      <td>701</td>
      <td>81</td>
      <td>114</td>
      <td>1032</td>
      <td>13499</td>
      <td>273</td>
      <td>1166</td>
      <td>262</td>
      <td>322</td>
      <td>53</td>
      <td>70</td>
      <td>1668</td>
      <td>14522</td>
      <td>407</td>
      <td>1350</td>
    </tr>
    <tr>
      <td></td>
      <td>cuffquant</td>
      <td>cuffquant</td>
      <td>The Cuffquant tool</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://cole-trapnell-lab.github.io/cufflinks/</td>
      <td>Transcriptomics</td>
      <td>cuffquant</td>
      <td>2020-06-08</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/cufflinks/cuffquant</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/cufflinks/cuffquant</td>
      <td>2.2.1</td>
      <td>cufflinks</td>
      <td>2.2.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>408</td>
      <td>732</td>
      <td>54</td>
      <td>71</td>
      <td>1964</td>
      <td>15883</td>
      <td>267</td>
      <td>1310</td>
      <td>308</td>
      <td>422</td>
      <td>26</td>
      <td>46</td>
      <td>2680</td>
      <td>17037</td>
      <td>347</td>
      <td>1427</td>
    </tr>
    <tr>
      <td></td>
      <td>cutadapt</td>
      <td>cutadapt</td>
      <td>Flexible tool to remove adapter sequences (and quality trim) high throughput sequencing reads (fasta/fastq).</td>
      <td>cutadapt</td>
      <td>cutadapt</td>
      <td></td>
      <td>Cutadapt</td>
      <td>Find and remove adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.</td>
      <td>Sequence trimming, Primer removal, Read pre-processing</td>
      <td>Genomics, Probes and primers, Sequencing</td>
      <td>Up-to-date</td>
      <td>https://cutadapt.readthedocs.org/en/stable/</td>
      <td>Fasta Manipulation, Fastq Manipulation, Sequence Analysis</td>
      <td>cutadapt</td>
      <td>2023-11-03</td>
      <td>lparsons</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cutadapt</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cutadapt</td>
      <td>4.9</td>
      <td>cutadapt</td>
      <td>4.9</td>
      <td>Sequence trimming, Primer removal, Read pre-processing</td>
      <td>Genomics, Probes and primers, Sequencing</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>259754</td>
      <td>272370</td>
      <td>11716</td>
      <td>12601</td>
      <td>334550</td>
      <td>362356</td>
      <td>18618</td>
      <td>20162</td>
      <td>64377</td>
      <td>65986</td>
      <td>3507</td>
      <td>3627</td>
      <td>658681</td>
      <td>700712</td>
      <td>33841</td>
      <td>36390</td>
    </tr>
    <tr>
      <td></td>
      <td>cutesv</td>
      <td>cutesv</td>
      <td>Long-read sequencing enables the comprehensive discovery of structural variations (SVs). However, it is still non-trivial to achieve high sensitivity and performance simultaneously due to the complex SV characteristics implied by noisy long reads. Therefore, we propose cuteSV, a sensitive, fast and scalable long-read-based SV detection approach. cuteSV uses tailored methods to collect the signatures of various types of SVs and employs a clustering-and-refinement method to analyze the signatures to implement sensitive SV detection. Benchmarks on real Pacific Biosciences (PacBio) and Oxford Nanopore Technology (ONT) datasets demonstrate that cuteSV has better yields and scalability than state-of-the-art tools.</td>
      <td>cuteSV</td>
      <td>cuteSV</td>
      <td></td>
      <td>cuteSV</td>
      <td>Long Read based Human Genomic Structural Variation Detection with cuteSV | Long-read sequencing technologies enable to comprehensively discover structural variations (SVs). However, it is still non-trivial for state-of-the-art approaches to detect SVs with high sensitivity or high performance or both. Herein, we propose cuteSV, a sensitive, fast and lightweight SV detection approach. cuteSV uses tailored methods to comprehensively collect various types of SV signatures, and a clustering-and-refinement method to implement a stepwise SV detection, which enables to achieve high sensitivity without loss of accuracy. Benchmark results demonstrate that cuteSV has better yields on real datasets. Further, its speed and scalability are outstanding and promising to large-scale data analysis</td>
      <td>Split read mapping, Genotyping, Structural variation detection</td>
      <td>DNA structural variation, Sequencing, Computer science</td>
      <td>Up-to-date</td>
      <td>https://github.com/tjiangHIT/cuteSV</td>
      <td>Variant Analysis</td>
      <td>cutesv</td>
      <td>2020-09-23</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cutesv</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cutesv</td>
      <td>2.1.1</td>
      <td>cutesv</td>
      <td>2.1.1</td>
      <td>Split read mapping, Genotyping, Structural variation detection</td>
      <td>Sequencing, Computer science</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>260</td>
      <td>260</td>
      <td>43</td>
      <td>43</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>119</td>
      <td>119</td>
      <td>8</td>
      <td>8</td>
      <td>379</td>
      <td>379</td>
      <td>51</td>
      <td>51</td>
    </tr>
    <tr>
      <td></td>
      <td>cwpair2</td>
      <td>cwpair2</td>
      <td>Contains a tool that takes a list of called peaks on both strands and produces a list of matched pairsand a list of unmatched orphans.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>ChIP-seq</td>
      <td>cwpair2</td>
      <td>2015-12-18</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cwpair2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cwpair2</td>
      <td>1.1.1</td>
      <td>matplotlib</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>173</td>
      <td>280</td>
      <td>41</td>
      <td>83</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>173</td>
      <td>280</td>
      <td>41</td>
      <td>83</td>
    </tr>
    <tr>
      <td></td>
      <td>dada2</td>
      <td>dada2_assignTaxonomyAddspecies, dada2_dada, dada2_filterAndTrim, dada2_learnErrors, dada2_makeSequenceTable, dada2_mergePairs, dada2_plotComplexity, dada2_plotQualityProfile, dada2_removeBimeraDenovo, dada2_seqCounts</td>
      <td>DADA2 wrappers</td>
      <td>dada2</td>
      <td>dada2</td>
      <td></td>
      <td>dada2</td>
      <td>This package infers exact sequence variants (SVs) from amplicon data, replacing the commonly used and coarser OTU clustering approach. This pipeline inputs demultiplexed fastq files, and outputs the sequence variants and their sample-wise abundances after removing substitution and chimera errors. Taxonomic classification is available via a native implementation of the RDP naive Bayesian classifier.</td>
      <td>Variant calling, DNA barcoding</td>
      <td>Sequencing, Genetic variation, Microbial ecology, Metagenomics</td>
      <td>To update</td>
      <td>https://benjjneb.github.io/dada2/index.html</td>
      <td>Metagenomics</td>
      <td>dada2</td>
      <td>2019-07-05</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/dada2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/dada2</td>
      <td></td>
      <td>bioconductor-dada2</td>
      <td>1.30.0</td>
      <td>Variant calling, DNA barcoding</td>
      <td>Sequencing, Genetic variation, Microbial ecology, Metagenomics</td>
      <td>10</td>
      <td>10</td>
      <td>10</td>
      <td>10</td>
      <td>10</td>
      <td>10</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>0</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>92240</td>
      <td>92240</td>
      <td>2716</td>
      <td>2716</td>
      <td>179176</td>
      <td>179176</td>
      <td>5317</td>
      <td>5317</td>
      <td>12478</td>
      <td>12478</td>
      <td>394</td>
      <td>394</td>
      <td>283894</td>
      <td>283894</td>
      <td>8427</td>
      <td>8427</td>
    </tr>
    <tr>
      <td></td>
      <td>das_tool</td>
      <td>Fasta_to_Contig2Bin, das_tool</td>
      <td>DAS Tool for genome resolved metagenomics</td>
      <td>dastool</td>
      <td>dastool</td>
      <td></td>
      <td>dastool</td>
      <td>DAS Tool is an automated method that integrates the results of a flexible number of binning algorithms to calculate an optimized, non-redundant set of bins from a single assembly.</td>
      <td>Read binning</td>
      <td>Metagenomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/cmks/DAS_Tool</td>
      <td>Metagenomics</td>
      <td>das_tool</td>
      <td>2022-06-23</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/das_tool</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/das_tool</td>
      <td>1.1.7</td>
      <td>das_tool</td>
      <td>1.1.7</td>
      <td>Read binning</td>
      <td>Metagenomics</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>1323</td>
      <td>1323</td>
      <td>47</td>
      <td>47</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1323</td>
      <td>1323</td>
      <td>47</td>
      <td>47</td>
    </tr>
    <tr>
      <td></td>
      <td>data_source_iris_tcga</td>
      <td>data_source_iris_tcga</td>
      <td>IRIS-TCGA Data source tool</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Data Source</td>
      <td>data_source_iris_tcga</td>
      <td>2016-02-18</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/data_source_iris_tcga</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/data_source_iris_tcga</td>
      <td>1.0.0</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>datamash</td>
      <td>datamash_ops, datamash_reverse, datamash_transpose</td>
      <td>GNU Datamash is a command-line program which performs basicnumeric,textual and statistical operations on input textual data files.It is designed to be portable and reliable, and aid researchersto easily automate analysis pipelines, without writing code or even short scripts.License: GPL Version 3 (or later).These tool wrappers were originally writen by Assaf Gordon.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.gnu.org/software/datamash/</td>
      <td>Text Manipulation</td>
      <td></td>
      <td>2015-08-03</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/datamash</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/datamash</td>
      <td>1.8</td>
      <td>datamash</td>
      <td>1.1.0</td>
      <td></td>
      <td></td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>748586</td>
      <td>760943</td>
      <td>7066</td>
      <td>7307</td>
      <td>98089</td>
      <td>112831</td>
      <td>5306</td>
      <td>6131</td>
      <td>16235</td>
      <td>16317</td>
      <td>2161</td>
      <td>2171</td>
      <td>862910</td>
      <td>890091</td>
      <td>14533</td>
      <td>15609</td>
    </tr>
    <tr>
      <td></td>
      <td>decontam</td>
      <td>decontam</td>
      <td>Removes decontamination features (ASVs/OTUs) using control samples</td>
      <td>decontam</td>
      <td>decontam</td>
      <td></td>
      <td>decontam</td>
      <td>Simple statistical identification of contaminating sequence features in marker-gene or metagenomics data. Works on any kind of feature derived from environmental sequencing data. Requires DNA quantitation data or sequenced negative control samples.</td>
      <td>Classification</td>
      <td>Metagenomics, Microbial ecology, Sequencing</td>
      <td>To update</td>
      <td>https://github.com/benjjneb/decontam</td>
      <td>Metagenomics</td>
      <td>decontam</td>
      <td>2024-09-13</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/decontam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/decontam</td>
      <td>1.22</td>
      <td>bioconductor-decontam</td>
      <td>1.22.0</td>
      <td>Classification</td>
      <td>Metagenomics, Microbial ecology, Sequencing</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>decontaminator</td>
      <td>decontaminator</td>
      <td>Deep Learning method for novel virus detection in sequencing data</td>
      <td>decontaminator</td>
      <td>decontaminator</td>
      <td></td>
      <td>decontaminator</td>
      <td>Decontaminator is a deep learning helping tool that filters out phage or fungi contigs from plant virome RNAseq assemblies.</td>
      <td>Filtering</td>
      <td>Metagenomics</td>
      <td>To update</td>
      <td>https://github.com/cbib/decontaminator</td>
      <td>Machine Learning</td>
      <td>decontaminator</td>
      <td>2023-01-09</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/decontaminator</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/decontaminator</td>
      <td>1.0.0</td>
      <td>numpy</td>
      <td></td>
      <td>Filtering</td>
      <td>Metagenomics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>121</td>
      <td>121</td>
      <td>28</td>
      <td>28</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>121</td>
      <td>121</td>
      <td>28</td>
      <td>28</td>
    </tr>
    <tr>
      <td></td>
      <td>deepmicro</td>
      <td>deepmicro</td>
      <td>Representation learning and classification framework</td>
      <td>DeepMicro</td>
      <td>DeepMicro</td>
      <td></td>
      <td>DeepMicro</td>
      <td>Deep representation learning for disease prediction based on microbiome data.DeepMicro is a deep representation learning framework exploiting various autoencoders to learn robust low-dimensional representations from high-dimensional data and training classification models based on the learned representation.</td>
      <td>Essential dynamics, Splitting, Community profiling</td>
      <td>Microbial ecology, Machine learning, Pathology, Public health and epidemiology, Allergy, clinical immunology and immunotherapeutics</td>
      <td>Up-to-date</td>
      <td>https://github.com/paulzierep/DeepMicro</td>
      <td>Machine Learning</td>
      <td>deepmicro</td>
      <td>2023-03-16</td>
      <td>iuc</td>
      <td>https://github.com/paulzierep/DeepMicro</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/deepmicro</td>
      <td>1.4</td>
      <td>deepmicro</td>
      <td>1.4</td>
      <td>Essential dynamics, Splitting</td>
      <td>Microbial ecology, Machine learning, Pathology, Public health and epidemiology, Allergy, clinical immunology and immunotherapeutics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>558</td>
      <td>558</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>558</td>
      <td>558</td>
      <td>7</td>
      <td>7</td>
    </tr>
    <tr>
      <td></td>
      <td>deepsig</td>
      <td>deepsig</td>
      <td>Predictor of signal peptides in proteins based on deep learning</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/BolognaBiocomp/deepsig</td>
      <td>Genome annotation</td>
      <td>deepsig</td>
      <td>2023-04-28</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/deepsig</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/deepsig</td>
      <td>1.2.5</td>
      <td>deepsig</td>
      <td>1.2.5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>65</td>
      <td>65</td>
      <td>37</td>
      <td>37</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>65</td>
      <td>65</td>
      <td>37</td>
      <td>37</td>
    </tr>
    <tr>
      <td></td>
      <td>deepvariant</td>
      <td>deepvariant</td>
      <td>DeepVariant is a deep learning-based variant caller</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/google/deepvariant</td>
      <td>Variant Analysis</td>
      <td>deepvariant</td>
      <td>2021-09-06</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/deepvariant</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/deepvariant</td>
      <td>1.5.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>2561</td>
      <td>2561</td>
      <td>325</td>
      <td>325</td>
      <td>2916</td>
      <td>2916</td>
      <td>460</td>
      <td>460</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5477</td>
      <td>5477</td>
      <td>785</td>
      <td>785</td>
    </tr>
    <tr>
      <td></td>
      <td>deg_annotate</td>
      <td>deg_annotate</td>
      <td>Annotate DESeq2/DEXSeq output tables</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/deg_annotate</td>
      <td>Transcriptomics</td>
      <td>deg_annotate</td>
      <td>2018-11-22</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/deg_annotate</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/deg_annotate</td>
      <td>1.1.0</td>
      <td>bedtools</td>
      <td>2.31.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>22774</td>
      <td>23146</td>
      <td>4084</td>
      <td>4155</td>
      <td>33972</td>
      <td>33972</td>
      <td>5372</td>
      <td>5372</td>
      <td>4236</td>
      <td>4236</td>
      <td>846</td>
      <td>846</td>
      <td>60982</td>
      <td>61354</td>
      <td>10302</td>
      <td>10373</td>
    </tr>
    <tr>
      <td></td>
      <td>delly</td>
      <td>delly_call, delly_classify, delly_cnv, delly_filter, delly_lr, delly_merge</td>
      <td>Delly is an integrated structural variant (SV) prediction method that can discover, genotype and visualize deletions, tandem duplications, inversions and translocations at single-nucleotide resolution in short-read massively parallel sequencing data. It uses paired-ends, split-reads and read-depth to sensitively and accurately delineate genomic rearrangements throughout the genome.</td>
      <td>delly2</td>
      <td>delly2</td>
      <td></td>
      <td>Delly2</td>
      <td>Integrated structural variant prediction method that can discover, genotype and visualize deletions, tandem duplications, inversions and translocations at single-nucleotide resolution in short-read massively parallel sequencing data. It uses paired-ends and split-reads to sensitively and accurately delineate genomic rearrangements throughout the genome. Structural variants can be visualized using Delly-maze and Delly-suave.</td>
      <td>Indel detection, Structural variation detection, Variant calling, Genotyping, Genetic variation analysis</td>
      <td>DNA structural variation, Sequencing, Pathology, Genomics, Genetic variation, Bioinformatics, Population genomics, Rare diseases</td>
      <td>To update</td>
      <td>https://github.com/dellytools/delly</td>
      <td>Variant Analysis</td>
      <td>delly</td>
      <td>2020-10-23</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/delly</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/delly</td>
      <td>0.9.1</td>
      <td>delly</td>
      <td>1.3.1</td>
      <td>Indel detection, Structural variation detection, Genotyping</td>
      <td>Sequencing, Genetic variation, Bioinformatics, Population genomics, Rare diseases</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>1451</td>
      <td>1451</td>
      <td>288</td>
      <td>288</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1451</td>
      <td>1451</td>
      <td>288</td>
      <td>288</td>
    </tr>
    <tr>
      <td></td>
      <td>deseq2</td>
      <td>deseq2</td>
      <td>Differential gene expression analysis based on the negative binomial distribution</td>
      <td>DESeq2</td>
      <td>DESeq2</td>
      <td></td>
      <td>DESeq2</td>
      <td>R/Bioconductor package for differential gene expression analysis based on the negative binomial distribution. Estimate variance-mean dependence in count data from high-throughput sequencing assays and test for differential expression based on a model using the negative binomial distribution.</td>
      <td>Differential gene expression analysis, RNA-Seq analysis</td>
      <td>RNA-Seq</td>
      <td>To update</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/DESeq2.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>deseq2</td>
      <td>2018-11-15</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/deseq2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/deseq2</td>
      <td>2.11.40.8</td>
      <td>bioconductor-deseq2</td>
      <td>1.42.0</td>
      <td>Differential gene expression analysis, RNA-Seq analysis</td>
      <td>RNA-Seq</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>97434</td>
      <td>109474</td>
      <td>11355</td>
      <td>12514</td>
      <td>183905</td>
      <td>246475</td>
      <td>24951</td>
      <td>31957</td>
      <td>18551</td>
      <td>20166</td>
      <td>2647</td>
      <td>2905</td>
      <td>299890</td>
      <td>376115</td>
      <td>38953</td>
      <td>47376</td>
    </tr>
    <tr>
      <td></td>
      <td>dexseq</td>
      <td>dexseq, dexseq_count, plotdexseq</td>
      <td>Inference of differential exon usage in RNA-Seq</td>
      <td>dexseq</td>
      <td>dexseq</td>
      <td></td>
      <td>DEXSeq</td>
      <td>The package is focused on finding differential exon usage using RNA-seq exon counts between samples with different experimental designs. It provides functions that allows the user to make the necessary statistical tests based on a model that uses the negative binomial distribution to estimate the variance between biological replicates and generalized linear models for testing. The package also provides functions for the visualization and exploration of the results.</td>
      <td>Enrichment analysis, Exonic splicing enhancer prediction</td>
      <td>RNA-Seq</td>
      <td>Up-to-date</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/DEXSeq.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>dexseq</td>
      <td>2018-05-02</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/dexseq</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/dexseq</td>
      <td>1.48.0</td>
      <td>bioconductor-dexseq</td>
      <td>1.48.0</td>
      <td>Enrichment analysis, Exonic splicing enhancer prediction</td>
      <td>RNA-Seq</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>14874</td>
      <td>16927</td>
      <td>533</td>
      <td>837</td>
      <td>18475</td>
      <td>28854</td>
      <td>1403</td>
      <td>2568</td>
      <td>3854</td>
      <td>4271</td>
      <td>141</td>
      <td>163</td>
      <td>37203</td>
      <td>50052</td>
      <td>2077</td>
      <td>3568</td>
    </tr>
    <tr>
      <td></td>
      <td>diamond</td>
      <td>bg_diamond, bg_diamond_makedb, bg_diamond_view</td>
      <td>DIAMOND is a new alignment tool for aligning short DNA sequencing reads to a protein reference database such as NCBI-NR.</td>
      <td>diamond</td>
      <td>diamond</td>
      <td></td>
      <td>Diamond</td>
      <td>Sequence aligner for protein and translated DNA searches and functions as a drop-in replacement for the NCBI BLAST software tools. It is suitable for protein-protein search as well as DNA-protein search on short reads and longer sequences including contigs and assemblies, providing a speedup of BLAST ranging up to x20,000.</td>
      <td>Sequence alignment analysis</td>
      <td>Sequence analysis, Proteins</td>
      <td>To update</td>
      <td>https://github.com/bbuchfink/diamond</td>
      <td>Sequence Analysis</td>
      <td>diamond</td>
      <td>2021-03-21</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/diamond</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/diamond</td>
      <td>2.0.15</td>
      <td>diamond</td>
      <td>2.1.10</td>
      <td>Sequence alignment analysis</td>
      <td>Sequence analysis, Proteins</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>61376</td>
      <td>61746</td>
      <td>1847</td>
      <td>1895</td>
      <td>23164</td>
      <td>28729</td>
      <td>1140</td>
      <td>1556</td>
      <td>63034</td>
      <td>63034</td>
      <td>522</td>
      <td>522</td>
      <td>147574</td>
      <td>153509</td>
      <td>3509</td>
      <td>3973</td>
    </tr>
    <tr>
      <td></td>
      <td>diffbind</td>
      <td>diffbind</td>
      <td>Diffbind provides functions for  processing  ChIP-Seq  data.</td>
      <td>diffbind</td>
      <td>diffbind</td>
      <td></td>
      <td>DiffBind</td>
      <td>Compute differentially bound sites from multiple ChIP-seq experiments using affinity (quantitative) data. Also enables occupancy (overlap) analysis and plotting functions.</td>
      <td>Differential binding analysis</td>
      <td>ChIP-seq</td>
      <td>Up-to-date</td>
      <td>http://bioconductor.org/packages/release/bioc/html/DiffBind.html</td>
      <td>ChIP-seq</td>
      <td>diffbind</td>
      <td>2018-04-07</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/diffbind</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/diffbind</td>
      <td>3.12.0</td>
      <td>bioconductor-diffbind</td>
      <td>3.12.0</td>
      <td>Differential binding analysis</td>
      <td>ChIP-seq</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>7092</td>
      <td>7663</td>
      <td>606</td>
      <td>652</td>
      <td>14367</td>
      <td>18624</td>
      <td>1360</td>
      <td>1916</td>
      <td>1477</td>
      <td>1640</td>
      <td>69</td>
      <td>84</td>
      <td>22936</td>
      <td>27927</td>
      <td>2035</td>
      <td>2652</td>
    </tr>
    <tr>
      <td></td>
      <td>dimet</td>
      <td>dimet_@EXECUTABLE@, dimet_@EXECUTABLE@, dimet_@EXECUTABLE@, dimet_@EXECUTABLE@, dimet_@EXECUTABLE@, dimet_@EXECUTABLE@, dimet_@EXECUTABLE@, dimet_@EXECUTABLE@, dimet_@EXECUTABLE@, dimet_@EXECUTABLE@</td>
      <td>DIMet is a bioinformatics pipeline for differential analysis of isotopic targeted labeling data.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/cbib/DIMet</td>
      <td>Metabolomics</td>
      <td></td>
      <td>2023-10-09</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/DIMet</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/dimet</td>
      <td>0.2.4</td>
      <td>dimet</td>
      <td>0.2.4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>disco</td>
      <td>disco</td>
      <td>DISCO is a overlap-layout-consensus (OLC) metagenome assembler</td>
      <td>disco</td>
      <td>disco</td>
      <td></td>
      <td>DISCO</td>
      <td>DISCO is software to perform structure determination of protein homo-oligomers with cyclic symmetry.DISCO computes oligomeric protein structures using geometric constraints derived from RDCs and intermolecular distance restraints such as NOEs or disulfide bonds. When a reliable subunit structure can be calculated from intramolecular restraints, DISCO guarantees that all satisfying oligomer structures will be discovered, yet can run in minutes to hours on only a single desktop-class computer.</td>
      <td>Protein sequence analysis</td>
      <td>Structure determination</td>
      <td>To update</td>
      <td>http://disco.omicsbio.org/</td>
      <td>Metagenomics, Assembly</td>
      <td>disco</td>
      <td>2017-10-26</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/disco/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/disco</td>
      <td></td>
      <td>disco</td>
      <td>1.2</td>
      <td>Protein sequence analysis</td>
      <td>Structure determination</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>358</td>
      <td>406</td>
      <td>124</td>
      <td>139</td>
      <td>1361</td>
      <td>1361</td>
      <td>384</td>
      <td>384</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1719</td>
      <td>1767</td>
      <td>508</td>
      <td>523</td>
    </tr>
    <tr>
      <td></td>
      <td>dnabot</td>
      <td>dnabot</td>
      <td>DNA assembly using BASIC on OpenTrons</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/BASIC-DNA-ASSEMBLY/DNA-BOT</td>
      <td>Synthetic Biology</td>
      <td>dnabot</td>
      <td>2022-04-17</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/dnabot</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/dnabot</td>
      <td>3.1.0</td>
      <td>dnabot</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>dnaweaver</td>
      <td>dnaweaver</td>
      <td>Given a SBOL input, calculate assembly parts for Gibson or Golden Gate.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/Edinburgh-Genome-Foundry/DnaWeaver</td>
      <td>Synthetic Biology</td>
      <td>dnaweaver</td>
      <td>2022-12-08</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/dnaweaver</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/dnaweaver</td>
      <td>1.0.2</td>
      <td>dnaweaver_synbiocad</td>
      <td>1.0.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>dram</td>
      <td>dram_annotate, dram_distill, dram_merge_annotations, dram_neighborhoods, dram_strainer</td>
      <td>DRAM for distilling microbial metabolism to automate the curation of microbiome function</td>
      <td>dram</td>
      <td>dram</td>
      <td></td>
      <td>DRAM</td>
      <td>Distilled and Refined Annotation of Metabolism: A tool for the annotation and curation of function for microbial and viral genomes</td>
      <td>Gene functional annotation</td>
      <td>Metagenomics, Biological databases, Molecular genetics</td>
      <td>Up-to-date</td>
      <td>https://github.com/WrightonLabCSU/DRAM</td>
      <td>Metagenomics</td>
      <td>dram</td>
      <td>2022-09-01</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/dram</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/dram</td>
      <td>1.5.0</td>
      <td>dram</td>
      <td>1.5.0</td>
      <td>Gene functional annotation</td>
      <td>Metagenomics, Biological databases, Molecular genetics</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>7358</td>
      <td>7358</td>
      <td>88</td>
      <td>88</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7358</td>
      <td>7358</td>
      <td>88</td>
      <td>88</td>
    </tr>
    <tr>
      <td></td>
      <td>drep</td>
      <td>drep_compare, drep_dereplicate</td>
      <td>dRep compares and dereplicates genome sets</td>
      <td>drep</td>
      <td>drep</td>
      <td></td>
      <td>dRep</td>
      <td>Fast and accurate genomic comparisons that enables improved genome recovery from metagenomes through de-replication.</td>
      <td>Genome comparison</td>
      <td>Metagenomics, Genomics, Sequence analysis</td>
      <td>Up-to-date</td>
      <td>https://github.com/MrOlm/drep</td>
      <td>Metagenomics</td>
      <td>drep</td>
      <td>2020-01-06</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/drep</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/drep</td>
      <td>3.5.0</td>
      <td>drep</td>
      <td>3.5.0</td>
      <td>Genome comparison</td>
      <td>Metagenomics, Sequence analysis</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>916</td>
      <td>916</td>
      <td>113</td>
      <td>113</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>916</td>
      <td>916</td>
      <td>113</td>
      <td>113</td>
    </tr>
    <tr>
      <td></td>
      <td>dropletutils</td>
      <td>dropletutils</td>
      <td>DropletUtils - Utilities for handling droplet-based single-cell RNA-seq data</td>
      <td>dropletutils</td>
      <td>dropletutils</td>
      <td></td>
      <td>DropletUtils</td>
      <td>Provides a number of utility functions for handling single-cell (RNA-seq) data from droplet technologies such as 10X Genomics. This includes data loading, identification of cells from empty droplets, removal of barcode-swapped pseudo-cells, and downsampling of the count matrix.</td>
      <td>Loading, Community profiling</td>
      <td>Gene expression, RNA-seq, Sequencing, Transcriptomics</td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/devel/bioc/html/DropletUtils.html</td>
      <td>Single Cell, Sequence Analysis</td>
      <td>dropletutils</td>
      <td>2019-09-02</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/dropletutils/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/dropletutils</td>
      <td>1.10.0</td>
      <td>bioconductor-dropletutils</td>
      <td>1.22.0</td>
      <td></td>
      <td>Sequencing, Transcriptomics</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>4413</td>
      <td>4442</td>
      <td>430</td>
      <td>431</td>
      <td>4403</td>
      <td>4403</td>
      <td>255</td>
      <td>255</td>
      <td>1816</td>
      <td>1816</td>
      <td>33</td>
      <td>33</td>
      <td>10632</td>
      <td>10661</td>
      <td>718</td>
      <td>719</td>
    </tr>
    <tr>
      <td></td>
      <td>ebi_tools</td>
      <td>ebi_search_rest_results</td>
      <td>Tools to query and download data from several EMBL-EBI databases</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.ebi.ac.uk/services/all</td>
      <td>Web Services, Data Source</td>
      <td>ebi_tools</td>
      <td>2016-11-29</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ebi_tools</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ebi_tools</td>
      <td>0.1.1</td>
      <td>six</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>276</td>
      <td>406</td>
      <td>133</td>
      <td>163</td>
      <td>1421</td>
      <td>1421</td>
      <td>547</td>
      <td>547</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1697</td>
      <td>1827</td>
      <td>680</td>
      <td>710</td>
    </tr>
    <tr>
      <td></td>
      <td>edger</td>
      <td>edger</td>
      <td>Perform RNA-Seq differential expression analysis using edgeR pipeline</td>
      <td>edger</td>
      <td>edger</td>
      <td></td>
      <td>edgeR</td>
      <td>Differential expression analysis of RNA-seq expression profiles with biological replication. Implements a range of statistical methodology based on the negative binomial distributions, including empirical Bayes estimation, exact tests, generalized linear models and quasi-likelihood tests. As well as RNA-seq, it be applied to differential signal analysis of other types of genomic data that produce counts, including ChIP-seq, SAGE and CAGE.</td>
      <td>Differential gene expression analysis</td>
      <td>Genetics, RNA-Seq, ChIP-seq</td>
      <td>To update</td>
      <td>http://bioconductor.org/packages/release/bioc/html/edgeR.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>edger</td>
      <td>2019-02-08</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/edger</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/edger</td>
      <td>3.36.0</td>
      <td>bioconductor-edger</td>
      <td>4.0.16</td>
      <td>Differential gene expression analysis</td>
      <td>Genetics, RNA-Seq, ChIP-seq</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>19294</td>
      <td>20905</td>
      <td>2423</td>
      <td>2618</td>
      <td>48876</td>
      <td>58723</td>
      <td>6341</td>
      <td>7527</td>
      <td>8010</td>
      <td>8705</td>
      <td>846</td>
      <td>892</td>
      <td>76180</td>
      <td>88333</td>
      <td>9610</td>
      <td>11037</td>
    </tr>
    <tr>
      <td></td>
      <td>egsea</td>
      <td>egsea</td>
      <td>This tool implements the Ensemble of Gene Set Enrichment Analyses (EGSEA) method for gene set testing</td>
      <td>egsea</td>
      <td>egsea</td>
      <td></td>
      <td>EGSEA</td>
      <td>This package implements the Ensemble of Gene Set Enrichment Analyses method for gene set testing.</td>
      <td>Gene set testing</td>
      <td>Systems biology</td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/EGSEA.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>egsea</td>
      <td>2018-01-01</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/egsea</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/egsea</td>
      <td>1.20.0</td>
      <td>bioconductor-egsea</td>
      <td>1.28.0</td>
      <td>Gene set testing</td>
      <td>Systems biology</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2809</td>
      <td>3060</td>
      <td>371</td>
      <td>402</td>
      <td>5179</td>
      <td>5179</td>
      <td>759</td>
      <td>759</td>
      <td>1931</td>
      <td>1931</td>
      <td>226</td>
      <td>226</td>
      <td>9919</td>
      <td>10170</td>
      <td>1356</td>
      <td>1387</td>
    </tr>
    <tr>
      <td></td>
      <td>emboss_5</td>
      <td>EMBOSS: antigenic1, EMBOSS: backtranseq2, EMBOSS: banana3, EMBOSS: biosed4, EMBOSS: btwisted5, EMBOSS: cai6, EMBOSS: cai_custom6, EMBOSS: chaos7, EMBOSS: charge8, EMBOSS: checktrans9, EMBOSS: chips10, EMBOSS: cirdna11, EMBOSS: codcmp12, EMBOSS: coderet13, EMBOSS: compseq14, EMBOSS: cpgplot15, EMBOSS: cpgreport16, EMBOSS: cusp17, EMBOSS: cutseq18, EMBOSS: dan19, EMBOSS: degapseq20, EMBOSS: descseq21, EMBOSS: diffseq22, EMBOSS: digest23, EMBOSS: dotmatcher24, EMBOSS: dotpath25, EMBOSS: dottup26, EMBOSS: dreg27, EMBOSS: einverted28, EMBOSS: epestfind29, EMBOSS: equicktandem31, EMBOSS: est2genome32, EMBOSS: etandem33, EMBOSS: extractfeat34, EMBOSS: extractseq35, EMBOSS: freak36, EMBOSS: fuzznuc37, EMBOSS: fuzzpro38, EMBOSS: fuzztran39, EMBOSS: garnier40, EMBOSS: geecee41, EMBOSS: getorf42, EMBOSS: helixturnhelix43, EMBOSS: hmoment44, EMBOSS: iep45, EMBOSS: infoseq46, EMBOSS: isochore47, EMBOSS: lindna48, EMBOSS: marscan49, EMBOSS: maskfeat50, EMBOSS: maskseq51, EMBOSS: matcher52, EMBOSS: megamerger53, EMBOSS: merger54, EMBOSS: msbar55, EMBOSS: needle56, EMBOSS: newcpgreport57, EMBOSS: newcpgseek58, EMBOSS: newseq59, EMBOSS: noreturn60, EMBOSS: notseq61, EMBOSS: nthseq62, EMBOSS: octanol63, EMBOSS: oddcomp64, EMBOSS: palindrome65, EMBOSS: pasteseq66, EMBOSS: patmatdb67, EMBOSS: pepcoil68, EMBOSS: pepinfo69, EMBOSS: pepnet70, EMBOSS: pepstats71, EMBOSS: pepwheel72, EMBOSS: pepwindow73, EMBOSS: pepwindowall74, EMBOSS: plotcon75, EMBOSS: plotorf76, EMBOSS: polydot77, EMBOSS: preg78, EMBOSS: prettyplot79, EMBOSS: prettyseq80, EMBOSS: primersearch81, EMBOSS: revseq82, EMBOSS: seqmatchall83, EMBOSS: seqret84, EMBOSS: showfeat85, EMBOSS: shuffleseq87, EMBOSS: sigcleave88, EMBOSS: sirna89, EMBOSS: sixpack90, EMBOSS: skipseq91, EMBOSS: splitter92, EMBOSS: supermatcher95, EMBOSS: syco96, EMBOSS: tcode97, EMBOSS: textsearch98, EMBOSS: tmap99, EMBOSS: tranalign100, EMBOSS: transeq101, EMBOSS: trimest102, EMBOSS: trimseq103, EMBOSS: twofeat104, EMBOSS: union105, EMBOSS: vectorstrip106, EMBOSS: water107, EMBOSS: wobble108, EMBOSS: wordcount109, EMBOSS: wordmatch110</td>
      <td>Galaxy wrappers for EMBOSS version 5.0.0 tools</td>
      <td>emboss</td>
      <td>emboss</td>
      <td></td>
      <td>EMBOSS</td>
      <td>Diverse suite of tools for sequence analysis; many programs analagous to GCG; context-sensitive help for each tool.</td>
      <td>Sequence analysis, Local alignment, Sequence alignment analysis, Global alignment, Sequence alignment</td>
      <td>Molecular biology, Sequence analysis, Biology</td>
      <td>To update</td>
      <td>http://emboss.open-bio.org/</td>
      <td>Sequence Analysis, Fasta Manipulation</td>
      <td>emboss_5</td>
      <td>2017-01-11</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/emboss_5</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/emboss_5</td>
      <td>5.0.0</td>
      <td>emboss</td>
      <td>6.6.0</td>
      <td>Sequence analysis, Local alignment, Sequence alignment analysis, Global alignment, Sequence alignment</td>
      <td>Molecular biology, Sequence analysis</td>
      <td>107</td>
      <td>107</td>
      <td>107</td>
      <td>107</td>
      <td>107</td>
      <td>107</td>
      <td>107</td>
      <td>107</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>107</td>
      <td>0</td>
      <td>0</td>
      <td>107</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>107</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>107</td>
      <td>107</td>
      <td>107</td>
      <td>0</td>
      <td>143762</td>
      <td>148864</td>
      <td>4481</td>
      <td>5039</td>
      <td>151989</td>
      <td>296028</td>
      <td>13224</td>
      <td>31140</td>
      <td>24955</td>
      <td>27001</td>
      <td>2397</td>
      <td>2705</td>
      <td>320706</td>
      <td>471893</td>
      <td>20102</td>
      <td>38884</td>
    </tr>
    <tr>
      <td></td>
      <td>ena_upload</td>
      <td>ena_upload</td>
      <td>Submits experimental data and respective metadata to the European Nucleotide Archive (ENA).</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/usegalaxy-eu/ena-upload-cli</td>
      <td>Data Export</td>
      <td>ena_upload</td>
      <td>2020-11-20</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ena_upload</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ena_upload</td>
      <td>0.7.3</td>
      <td>ena-upload-cli</td>
      <td>0.7.4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>366</td>
      <td>366</td>
      <td>41</td>
      <td>41</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1302</td>
      <td>1302</td>
      <td>14</td>
      <td>14</td>
      <td>1668</td>
      <td>1668</td>
      <td>55</td>
      <td>55</td>
    </tr>
    <tr>
      <td></td>
      <td>enasearch</td>
      <td>enasearch_retrieve_analysis_report, enasearch_retrieve_data, enasearch_retrieve_run_report, enasearch_retrieve_taxons, enasearch_search_data</td>
      <td>A Python library for interacting with ENA's API</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bebatut/enasearch</td>
      <td>Data Source</td>
      <td>enasearch</td>
      <td>2017-08-05</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/enasearch</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/enasearch</td>
      <td></td>
      <td>enasearch</td>
      <td>0.2.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>1989</td>
      <td>2302</td>
      <td>367</td>
      <td>453</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1989</td>
      <td>2302</td>
      <td>367</td>
      <td>453</td>
    </tr>
    <tr>
      <td></td>
      <td>ensembl_vep</td>
      <td>ensembl_vep</td>
      <td>Ensembl VEP: Annotate VCFs with variant effect predictions</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/Ensembl/ensembl-vep</td>
      <td>Variant Analysis</td>
      <td>ensembl_vep</td>
      <td>2022-05-11</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ensembl_vep</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ensembl_vep</td>
      <td>112.0</td>
      <td>ensembl-vep</td>
      <td>113.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>3051</td>
      <td>3051</td>
      <td>171</td>
      <td>171</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3051</td>
      <td>3051</td>
      <td>171</td>
      <td>171</td>
    </tr>
    <tr>
      <td></td>
      <td>episcanpy</td>
      <td>episcanpy_build_matrix, episcanpy_cluster_embed, episcanpy_preprocess</td>
      <td>EpiScanpy  Epigenomics single cell analysis in python</td>
      <td>episcanpy</td>
      <td>episcanpy</td>
      <td></td>
      <td>epiScanpy</td>
      <td>Epigenomics Single Cell Analysis in Python.</td>
      <td>Enrichment analysis, Imputation</td>
      <td>Epigenomics, Cell biology, DNA</td>
      <td>To update</td>
      <td>https://github.com/colomemaria/epiScanpy</td>
      <td>Single Cell, Epigenetics</td>
      <td>episcanpy</td>
      <td>2023-03-22</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/episcanpy/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/episcanpy</td>
      <td>0.3.2</td>
      <td>episcanpy</td>
      <td>0.4.0</td>
      <td>Enrichment analysis, Imputation</td>
      <td>Epigenomics, Cell biology, DNA</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>880</td>
      <td>880</td>
      <td>80</td>
      <td>80</td>
      <td>221</td>
      <td>221</td>
      <td>37</td>
      <td>37</td>
      <td>49</td>
      <td>49</td>
      <td>3</td>
      <td>3</td>
      <td>1150</td>
      <td>1150</td>
      <td>120</td>
      <td>120</td>
    </tr>
    <tr>
      <td></td>
      <td>evidencemodeler</td>
      <td>evidencemodeler</td>
      <td>EVidenceModeler (EVM) combines ab intio genetic predictions with protein and transcript alignments in weighted consensus genetic structures.</td>
      <td>EvidenceModeler</td>
      <td>EvidenceModeler</td>
      <td></td>
      <td>EvidenceModeler</td>
      <td>The EVidenceModeler (aka EVM) software combines ab intio gene predictions and protein and transcript alignments into weighted consensus gene structures. EVM provides a flexible and intuitive framework for combining diverse evidence types into a single automated gene structure annotation system.</td>
      <td>Gene prediction</td>
      <td>Gene expression, Gene structure</td>
      <td>Up-to-date</td>
      <td>https://github.com/EVidenceModeler/EVidenceModeler?tab=readme-ov-file</td>
      <td>Genome annotation</td>
      <td>evidencemodeler</td>
      <td>2024-07-19</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/evidencemodeler</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/evidencemodeler</td>
      <td>2.1.0</td>
      <td>evidencemodeler</td>
      <td>2.1.0</td>
      <td>Gene prediction</td>
      <td>Gene expression, Gene structure</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>exomedepth</td>
      <td>exomedepth</td>
      <td>ExomeDepth: Calls copy number variants (CNVs) from targeted sequence data</td>
      <td>exomedepth</td>
      <td>exomedepth</td>
      <td></td>
      <td>ExomeDepth</td>
      <td>Copy number variant (CNV) calling algorithm designed to control technical variability between samples. It calls CNVs from targeted sequence data, typically exome sequencing experiments designed to identify the genetic basis of Mendelian disorders.</td>
      <td>Sequence analysis, Variant calling, Genotyping, Copy number estimation</td>
      <td>Exome sequencing, Gene transcripts, Mapping, Sequencing, Genetic variation, Rare diseases</td>
      <td>To update</td>
      <td>https://cran.r-project.org/package=ExomeDepth</td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>exomedepth</td>
      <td>2019-11-08</td>
      <td>crs4</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/exomedepth</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/exomedepth</td>
      <td>1.1.0</td>
      <td>r-exomedepth</td>
      <td>1.1.16</td>
      <td>Sequence analysis, Variant calling, Genotyping, Copy number estimation</td>
      <td>Exome sequencing, Gene transcripts, Mapping, Genetic variation, Rare diseases</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>517</td>
      <td>517</td>
      <td>61</td>
      <td>61</td>
      <td>2995</td>
      <td>2995</td>
      <td>184</td>
      <td>184</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3512</td>
      <td>3512</td>
      <td>245</td>
      <td>245</td>
    </tr>
    <tr>
      <td></td>
      <td>exonerate</td>
      <td>exonerate</td>
      <td>Exonerate is a generic tool for pairwise sequence comparison.</td>
      <td>exonerate</td>
      <td>exonerate</td>
      <td></td>
      <td>Exonerate</td>
      <td>A tool for pairwise sequence alignment. It enables alignment for DNA-DNA and DNA-protein pairs and also gapped and ungapped alignment.</td>
      <td>Pairwise sequence alignment, Protein threading, Genome alignment</td>
      <td>Sequence analysis, Sequence sites, features and motifs, Molecular interactions, pathways and networks</td>
      <td>Up-to-date</td>
      <td>https://www.ebi.ac.uk/about/vertebrate-genomics/software/exonerate</td>
      <td>Sequence Analysis</td>
      <td>exonerate</td>
      <td>2018-08-02</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/exonerate</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/exonerate</td>
      <td>2.4.0</td>
      <td>exonerate</td>
      <td>2.4.0</td>
      <td>Pairwise sequence alignment, Protein threading, Genome alignment</td>
      <td>Sequence analysis, Sequence sites, features and motifs, Molecular interactions, pathways and networks</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1030</td>
      <td>1072</td>
      <td>185</td>
      <td>188</td>
      <td>1058</td>
      <td>1058</td>
      <td>189</td>
      <td>189</td>
      <td>922</td>
      <td>922</td>
      <td>35</td>
      <td>35</td>
      <td>3010</td>
      <td>3052</td>
      <td>409</td>
      <td>412</td>
    </tr>
    <tr>
      <td></td>
      <td>export2graphlan</td>
      <td>export2graphlan</td>
      <td>export2graphlan is a conversion software tool for producing both annotation and tree file for GraPhlAn</td>
      <td>export2graphlan</td>
      <td>export2graphlan</td>
      <td></td>
      <td>export2graphlan</td>
      <td>export2graphlan is a conversion software tool for producing both annotation and tree file for GraPhlAn. In particular, the annotation file tries to highlight specific sub-trees deriving automatically from input file what nodes are important.</td>
      <td>Conversion</td>
      <td>Taxonomy, Metabolomics, Biomarkers</td>
      <td>To update</td>
      <td>https://bitbucket.org/CibioCM/export2graphlan/overview</td>
      <td>Metagenomics</td>
      <td>export2graphlan</td>
      <td>2017-03-03</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/export2graphlan/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/export2graphlan</td>
      <td>0.20</td>
      <td>export2graphlan</td>
      <td>0.22</td>
      <td>Conversion</td>
      <td>Taxonomy, Metabolomics, Biomarkers</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>5396</td>
      <td>5938</td>
      <td>580</td>
      <td>607</td>
      <td>628</td>
      <td>628</td>
      <td>168</td>
      <td>168</td>
      <td>519</td>
      <td>519</td>
      <td>107</td>
      <td>107</td>
      <td>6543</td>
      <td>7085</td>
      <td>855</td>
      <td>882</td>
    </tr>
    <tr>
      <td></td>
      <td>extract_genomic_dna</td>
      <td>Extract genomic DNA 1</td>
      <td>Contains a tool that extracts genomic DNA using coordinates from ASSEMBLED genomes and UNassembled genomes.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Genomic Interval Operations</td>
      <td>extract_genomic_dna</td>
      <td>2016-01-15</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/extract_genomic_dna</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/extract_genomic_dna</td>
      <td>3.0.3+galaxy2</td>
      <td>bx-python</td>
      <td>0.13.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>9409</td>
      <td>12735</td>
      <td>813</td>
      <td>923</td>
      <td>20180</td>
      <td>259099</td>
      <td>2072</td>
      <td>11791</td>
      <td>168</td>
      <td>611</td>
      <td>20</td>
      <td>63</td>
      <td>29757</td>
      <td>272445</td>
      <td>2905</td>
      <td>12777</td>
    </tr>
    <tr>
      <td></td>
      <td>falco</td>
      <td>falco</td>
      <td>A high throughput sequence QC analysis tool</td>
      <td>falco</td>
      <td>falco</td>
      <td></td>
      <td>Falco</td>
      <td>A high-speed FastQC emulation for quality control of sequencing data.</td>
      <td>Sequencing quality control, Visualisation, Read mapping</td>
      <td>Workflows, Mapping, Imaging</td>
      <td>Up-to-date</td>
      <td>https://github.com/smithlabcode/falco/</td>
      <td>Sequence Analysis</td>
      <td>falco</td>
      <td>2024-04-17</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/falco</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/falco</td>
      <td>1.2.4</td>
      <td>falco</td>
      <td>1.2.4</td>
      <td>Sequencing quality control, Visualisation, Read mapping</td>
      <td>Workflows, Mapping, Imaging</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4298</td>
      <td>4298</td>
      <td>34</td>
      <td>34</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4298</td>
      <td>4298</td>
      <td>34</td>
      <td>34</td>
    </tr>
    <tr>
      <td></td>
      <td>fargene</td>
      <td>fargene</td>
      <td>fARGene (Fragmented Antibiotic Resistance Gene iENntifiEr )</td>
      <td>fargene</td>
      <td>fargene</td>
      <td></td>
      <td>fARGene</td>
      <td>fARGene (Fragmented Antibiotic Resistance Gene iENntifiEr ) is a tool that takes either fragmented metagenomic data or longer sequences as input and predicts and delivers full-length antiobiotic resistance genes as output.</td>
      <td>Antimicrobial resistance prediction</td>
      <td>Metagenomics, Microbiology, Public health and epidemiology</td>
      <td>Up-to-date</td>
      <td>https://github.com/fannyhb/fargene</td>
      <td>Sequence Analysis</td>
      <td>fargene</td>
      <td>2019-08-30</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fargene</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fargene</td>
      <td>0.1</td>
      <td>fargene</td>
      <td>0.1</td>
      <td>Antimicrobial resistance prediction</td>
      <td>Metagenomics, Microbiology, Public health and epidemiology</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>601</td>
      <td>601</td>
      <td>128</td>
      <td>128</td>
      <td>1165</td>
      <td>1165</td>
      <td>232</td>
      <td>232</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1766</td>
      <td>1766</td>
      <td>360</td>
      <td>360</td>
    </tr>
    <tr>
      <td></td>
      <td>fasplit</td>
      <td>fasplit</td>
      <td>faSplit is a tool to split a single FASTA file into several files</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td></td>
      <td>UCSC Genome Browser Utilities</td>
      <td>Utilities for handling sequences and assemblies from the UCSC Genome Browser project.</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>Up-to-date</td>
      <td>http://hgdownload.cse.ucsc.edu/admin/exe/</td>
      <td>Fasta Manipulation</td>
      <td>ucsc_fasplit</td>
      <td>2017-09-08</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ucsc_tools/fasplit</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ucsc_tools/fasplit</td>
      <td>469</td>
      <td>ucsc-fasplit</td>
      <td>469</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>3278</td>
      <td>3361</td>
      <td>460</td>
      <td>478</td>
      <td>6144</td>
      <td>6144</td>
      <td>595</td>
      <td>595</td>
      <td>1139</td>
      <td>1139</td>
      <td>183</td>
      <td>183</td>
      <td>10561</td>
      <td>10644</td>
      <td>1238</td>
      <td>1256</td>
    </tr>
    <tr>
      <td></td>
      <td>fasta_clipping_histogram</td>
      <td>cshl_fasta_clipping_histogram</td>
      <td>Length Distribution chart</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hannonlab.cshl.edu/fastx_toolkit/</td>
      <td>Fasta Manipulation, Graphics, Statistics</td>
      <td>fasta_clipping_histogram</td>
      <td>2017-07-10</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/fastx_toolkit/fasta_clipping_histogram</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/fastx_toolkit/fasta_clipping_histogram</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>0.0.14</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5191</td>
      <td>5367</td>
      <td>104</td>
      <td>130</td>
      <td>5191</td>
      <td>5367</td>
      <td>104</td>
      <td>130</td>
    </tr>
    <tr>
      <td></td>
      <td>fasta_formatter</td>
      <td>cshl_fasta_formatter</td>
      <td>FASTA Width formatter</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hannonlab.cshl.edu/fastx_toolkit/</td>
      <td>Fasta Manipulation</td>
      <td>fasta_formatter</td>
      <td>2018-04-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/fastx_toolkit/fasta_formatter</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/fastx_toolkit/fasta_formatter</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>0.0.14</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>5235</td>
      <td>5486</td>
      <td>237</td>
      <td>288</td>
      <td>4060</td>
      <td>25609</td>
      <td>480</td>
      <td>2039</td>
      <td>816</td>
      <td>1093</td>
      <td>73</td>
      <td>89</td>
      <td>10111</td>
      <td>32188</td>
      <td>790</td>
      <td>2416</td>
    </tr>
    <tr>
      <td></td>
      <td>fasta_nucleotide_changer</td>
      <td>cshl_fasta_nucleotides_changer</td>
      <td>RNA/DNA converter.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hannonlab.cshl.edu/fastx_toolkit/</td>
      <td>Fasta Manipulation</td>
      <td>fasta_nucleotide_changer</td>
      <td>2018-04-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/fastx_toolkit/fasta_nucleotide_changer</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/fastx_toolkit/fasta_nucleotide_changer</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>0.0.14</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>497</td>
      <td>808</td>
      <td>71</td>
      <td>108</td>
      <td>2249</td>
      <td>5479</td>
      <td>216</td>
      <td>703</td>
      <td>415</td>
      <td>437</td>
      <td>18</td>
      <td>20</td>
      <td>3161</td>
      <td>6724</td>
      <td>305</td>
      <td>831</td>
    </tr>
    <tr>
      <td></td>
      <td>fasta_nucleotide_color_plot</td>
      <td>fasta_nucleotide_color_plot</td>
      <td>Contains a tool that produces a graphical representation of FASTA data with each nucleotide represented by a selected color.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/seqcode/cegr-tools/tree/master/src/org/seqcode/cegrtools/fourcolorplot</td>
      <td>Visualization</td>
      <td>fasta_nucleotide_color_plot</td>
      <td>2016-03-06</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fasta_nucleotide_color_plot</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fasta_nucleotide_color_plot</td>
      <td>1.0.1</td>
      <td>openjdk</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>164</td>
      <td>179</td>
      <td>42</td>
      <td>47</td>
      <td>758</td>
      <td>905</td>
      <td>128</td>
      <td>170</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>922</td>
      <td>1084</td>
      <td>170</td>
      <td>217</td>
    </tr>
    <tr>
      <td></td>
      <td>fasta_stats</td>
      <td>fasta-stats</td>
      <td>Display summary statistics for a fasta file.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fasta_stats/</td>
      <td>Sequence Analysis</td>
      <td>fasta_stats</td>
      <td>2018-11-21</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fasta_stats/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fasta_stats</td>
      <td>2.0</td>
      <td>numpy</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>43726</td>
      <td>44512</td>
      <td>3104</td>
      <td>3259</td>
      <td>19941</td>
      <td>19941</td>
      <td>3693</td>
      <td>3693</td>
      <td>16788</td>
      <td>18589</td>
      <td>2467</td>
      <td>2650</td>
      <td>80455</td>
      <td>83042</td>
      <td>9264</td>
      <td>9602</td>
    </tr>
    <tr>
      <td></td>
      <td>fastani</td>
      <td>fastani</td>
      <td>Fast alignment-free computation of whole-genome Average Nucleotide Identity</td>
      <td>fastani</td>
      <td>fastani</td>
      <td></td>
      <td>FastANI</td>
      <td>FastANI is developed for fast alignment-free computation of whole-genome Average Nucleotide Identity (ANI). ANI is defined as mean nucleotide identity of orthologous gene pairs shared between two microbial genomes. FastANI supports pairwise comparison of both complete and draft genome assemblies.</td>
      <td>Genome alignment, Sequence similarity search</td>
      <td>Microbiology, Genetic variation</td>
      <td>To update</td>
      <td>https://github.com/ParBLiSS/FastANI</td>
      <td>Sequence Analysis</td>
      <td>fastani</td>
      <td>2020-02-17</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fastani</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fastani</td>
      <td>1.3</td>
      <td>fastani</td>
      <td>1.34</td>
      <td>Genome alignment, Sequence similarity search</td>
      <td>Microbiology, Genetic variation</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>6685</td>
      <td>6685</td>
      <td>603</td>
      <td>603</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6685</td>
      <td>6685</td>
      <td>603</td>
      <td>603</td>
    </tr>
    <tr>
      <td></td>
      <td>fastk</td>
      <td>fastk_fastk</td>
      <td>FastK: A K-mer counter (for HQ assembly data sets)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/thegenemyers/FASTK</td>
      <td>Assembly</td>
      <td>fastk</td>
      <td>2024-05-03</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fastk</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fastk</td>
      <td>1.0.0</td>
      <td>fastk</td>
      <td>1.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>136</td>
      <td>136</td>
      <td>11</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>136</td>
      <td>136</td>
      <td>11</td>
      <td>11</td>
    </tr>
    <tr>
      <td></td>
      <td>fastp</td>
      <td>fastp</td>
      <td>Fast all-in-one preprocessing for FASTQ files</td>
      <td>fastp</td>
      <td>fastp</td>
      <td></td>
      <td>fastp</td>
      <td>A tool designed to provide fast all-in-one preprocessing for FastQ files. This tool is developed in C++ with multithreading supported to afford high performance.</td>
      <td>Sequencing quality control, Sequence contamination filtering</td>
      <td>Sequence analysis, Probes and primers</td>
      <td>Up-to-date</td>
      <td>https://github.com/OpenGene/fastp</td>
      <td>Sequence Analysis</td>
      <td>fastp</td>
      <td>2018-03-18</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fastp</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fastp</td>
      <td>0.23.4</td>
      <td>fastp</td>
      <td>0.23.4</td>
      <td>Sequence contamination filtering</td>
      <td>Probes and primers</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1126086</td>
      <td>1127111</td>
      <td>6847</td>
      <td>6909</td>
      <td>422259</td>
      <td>422259</td>
      <td>10722</td>
      <td>10722</td>
      <td>76462</td>
      <td>76462</td>
      <td>2242</td>
      <td>2242</td>
      <td>1624807</td>
      <td>1625832</td>
      <td>19811</td>
      <td>19873</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_combiner</td>
      <td>fastq_combiner</td>
      <td>Combine FASTA and QUAL into FASTQ</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation, Fasta Manipulation</td>
      <td>fastq_combiner</td>
      <td>2019-06-21</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_combiner</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/galaxy_sequence_utils/fastq_combiner</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>1.2.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>8604</td>
      <td>8799</td>
      <td>215</td>
      <td>240</td>
      <td>54200</td>
      <td>75329</td>
      <td>919</td>
      <td>2957</td>
      <td>1562</td>
      <td>1680</td>
      <td>76</td>
      <td>85</td>
      <td>64366</td>
      <td>85808</td>
      <td>1210</td>
      <td>3282</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_filter</td>
      <td>fastq_filter</td>
      <td>Filter FASTQ reads by quality score and length</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_filter</td>
      <td>2019-06-21</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_filter</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/galaxy_sequence_utils/fastq_filter</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>1.2.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>15582</td>
      <td>18350</td>
      <td>690</td>
      <td>771</td>
      <td>40489</td>
      <td>121585</td>
      <td>2715</td>
      <td>9239</td>
      <td>4657</td>
      <td>5793</td>
      <td>214</td>
      <td>287</td>
      <td>60728</td>
      <td>145728</td>
      <td>3619</td>
      <td>10297</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_groomer</td>
      <td>fastq_groomer</td>
      <td>Convert between various FASTQ quality formats.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_groomer</td>
      <td>2019-06-21</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_groomer</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/galaxy_sequence_utils/fastq_groomer</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>1.2.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>92594</td>
      <td>102535</td>
      <td>2576</td>
      <td>2995</td>
      <td>179461</td>
      <td>897457</td>
      <td>9437</td>
      <td>44701</td>
      <td>15434</td>
      <td>25533</td>
      <td>664</td>
      <td>1073</td>
      <td>287489</td>
      <td>1025525</td>
      <td>12677</td>
      <td>48769</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_manipulation</td>
      <td>fastq_manipulation</td>
      <td>Manipulate FASTQ reads on various attributes.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_manipulation</td>
      <td>2019-06-21</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_manipulation</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/galaxy_sequence_utils/fastq_manipulation</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>1.2.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2488</td>
      <td>3132</td>
      <td>215</td>
      <td>267</td>
      <td>6241</td>
      <td>14508</td>
      <td>834</td>
      <td>2121</td>
      <td>4121</td>
      <td>4592</td>
      <td>131</td>
      <td>158</td>
      <td>12850</td>
      <td>22232</td>
      <td>1180</td>
      <td>2546</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_masker_by_quality</td>
      <td>fastq_masker_by_quality</td>
      <td>FASTQ Masker by quality score</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_masker_by_quality</td>
      <td>2019-06-21</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_masker_by_quality</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/galaxy_sequence_utils/fastq_masker_by_quality</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>1.2.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>234</td>
      <td>249</td>
      <td>43</td>
      <td>44</td>
      <td>2648</td>
      <td>8092</td>
      <td>288</td>
      <td>983</td>
      <td>442</td>
      <td>466</td>
      <td>12</td>
      <td>17</td>
      <td>3324</td>
      <td>8807</td>
      <td>343</td>
      <td>1044</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_paired_end_deinterlacer</td>
      <td>fastq_paired_end_deinterlacer</td>
      <td>FASTQ de-interlacer on paired end reads.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_paired_end_deinterlacer</td>
      <td>2019-06-21</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_paired_end_deinterlacer</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/galaxy_sequence_utils/fastq_paired_end_deinterlacer</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>1.2.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>5873</td>
      <td>8936</td>
      <td>532</td>
      <td>657</td>
      <td>14738</td>
      <td>20229</td>
      <td>1889</td>
      <td>2366</td>
      <td>2316</td>
      <td>2711</td>
      <td>225</td>
      <td>261</td>
      <td>22927</td>
      <td>31876</td>
      <td>2646</td>
      <td>3284</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_paired_end_interlacer</td>
      <td>fastq_paired_end_interlacer</td>
      <td>FASTQ interlacer on paired end reads</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_paired_end_interlacer</td>
      <td>2019-06-21</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_paired_end_interlacer</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/galaxy_sequence_utils/fastq_paired_end_interlacer</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>1.2.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>18216</td>
      <td>19573</td>
      <td>2348</td>
      <td>2492</td>
      <td>23484</td>
      <td>25936</td>
      <td>3482</td>
      <td>3851</td>
      <td>6207</td>
      <td>6551</td>
      <td>1402</td>
      <td>1442</td>
      <td>47907</td>
      <td>52060</td>
      <td>7232</td>
      <td>7785</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_paired_end_joiner</td>
      <td>fastq_paired_end_joiner</td>
      <td>FASTQ joiner on paired end reads</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_paired_end_joiner</td>
      <td>2019-06-21</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_paired_end_joiner</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/galaxy_sequence_utils/fastq_paired_end_joiner</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>1.2.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>15628</td>
      <td>17130</td>
      <td>536</td>
      <td>679</td>
      <td>24157</td>
      <td>58368</td>
      <td>1954</td>
      <td>5716</td>
      <td>1941</td>
      <td>2266</td>
      <td>172</td>
      <td>214</td>
      <td>41726</td>
      <td>77764</td>
      <td>2662</td>
      <td>6609</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_paired_end_splitter</td>
      <td>fastq_paired_end_splitter</td>
      <td>FASTQ splitter on joined paired end reads</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_paired_end_splitter</td>
      <td>2019-06-21</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_paired_end_splitter</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/galaxy_sequence_utils/fastq_paired_end_splitter</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>1.2.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>3169</td>
      <td>3521</td>
      <td>459</td>
      <td>535</td>
      <td>19309</td>
      <td>30747</td>
      <td>1410</td>
      <td>3441</td>
      <td>807</td>
      <td>1023</td>
      <td>219</td>
      <td>247</td>
      <td>23285</td>
      <td>35291</td>
      <td>2088</td>
      <td>4223</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_quality_boxplot</td>
      <td>cshl_fastq_quality_boxplot</td>
      <td>Draw quality score boxplot</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hannonlab.cshl.edu/fastx_toolkit/</td>
      <td>Fastq Manipulation, Graphics, Statistics</td>
      <td>fastq_quality_boxplot</td>
      <td>2017-07-10</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/fastx_toolkit/fastq_quality_boxplot</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/fastx_toolkit/fastq_quality_boxplot</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>0.0.14</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1271</td>
      <td>1393</td>
      <td>131</td>
      <td>176</td>
      <td>12763</td>
      <td>58819</td>
      <td>2073</td>
      <td>8850</td>
      <td>430</td>
      <td>479</td>
      <td>32</td>
      <td>51</td>
      <td>14464</td>
      <td>60691</td>
      <td>2236</td>
      <td>9077</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_quality_converter</td>
      <td>cshl_fastq_quality_converter</td>
      <td>Quality format converter (ASCII-Numeric)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hannonlab.cshl.edu/fastx_toolkit/</td>
      <td>Fastq Manipulation</td>
      <td>fastq_quality_converter</td>
      <td>2018-04-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/fastx_toolkit/fastq_quality_converter</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/fastx_toolkit/fastq_quality_converter</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>0.0.14</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>335</td>
      <td>352</td>
      <td>44</td>
      <td>47</td>
      <td>1583</td>
      <td>9696</td>
      <td>269</td>
      <td>2229</td>
      <td>1131</td>
      <td>1254</td>
      <td>15</td>
      <td>27</td>
      <td>3049</td>
      <td>11302</td>
      <td>328</td>
      <td>2303</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_quality_filter</td>
      <td>cshl_fastq_quality_filter</td>
      <td>Filter by quality</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hannonlab.cshl.edu/fastx_toolkit/</td>
      <td>Fastq Manipulation</td>
      <td>fastq_quality_filter</td>
      <td>2017-07-10</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/fastx_toolkit/fastq_quality_filter</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/fastx_toolkit/fastq_quality_filter</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>0.0.14</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>17279</td>
      <td>18386</td>
      <td>3826</td>
      <td>4062</td>
      <td>55268</td>
      <td>136888</td>
      <td>6055</td>
      <td>12590</td>
      <td>7685</td>
      <td>9356</td>
      <td>1854</td>
      <td>2043</td>
      <td>80232</td>
      <td>164630</td>
      <td>11735</td>
      <td>18695</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_stats</td>
      <td>fastq_stats</td>
      <td>FASTQ Summary Statistics by column</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_stats</td>
      <td>2019-06-21</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_stats</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/galaxy_sequence_utils/fastq_stats</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>1.2.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>3858</td>
      <td>4595</td>
      <td>680</td>
      <td>815</td>
      <td>15213</td>
      <td>81513</td>
      <td>3159</td>
      <td>14109</td>
      <td>3855</td>
      <td>4163</td>
      <td>184</td>
      <td>256</td>
      <td>22926</td>
      <td>90271</td>
      <td>4023</td>
      <td>15180</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_to_fasta</td>
      <td>cshl_fastq_to_fasta</td>
      <td>FASTQ to FASTA converter</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hannonlab.cshl.edu/fastx_toolkit/</td>
      <td>Fasta Manipulation, Convert Formats</td>
      <td>fastq_to_fasta</td>
      <td>2016-09-17</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/fastx_toolkit/fastq_to_fasta</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/fastx_toolkit/fastq_to_fasta</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>0.0.14</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>36069</td>
      <td>37805</td>
      <td>1151</td>
      <td>1234</td>
      <td>45213</td>
      <td>99852</td>
      <td>2691</td>
      <td>6370</td>
      <td>8457</td>
      <td>13335</td>
      <td>408</td>
      <td>609</td>
      <td>89739</td>
      <td>150992</td>
      <td>4250</td>
      <td>8213</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_to_tabular</td>
      <td>fastq_to_tabular</td>
      <td>FASTQ to Tabular converter</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_to_tabular</td>
      <td>2019-06-21</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_to_tabular</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/galaxy_sequence_utils/fastq_to_tabular</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>1.2.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>12895</td>
      <td>14110</td>
      <td>269</td>
      <td>297</td>
      <td>13315</td>
      <td>30673</td>
      <td>612</td>
      <td>1810</td>
      <td>5125</td>
      <td>5373</td>
      <td>64</td>
      <td>87</td>
      <td>31335</td>
      <td>50156</td>
      <td>945</td>
      <td>2194</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_trimmer</td>
      <td>fastq_trimmer</td>
      <td>FASTQ Trimmer by quality</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_trimmer</td>
      <td>2019-06-21</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_trimmer</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/galaxy_sequence_utils/fastq_trimmer</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>1.2.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>10114</td>
      <td>11383</td>
      <td>582</td>
      <td>665</td>
      <td>23906</td>
      <td>92351</td>
      <td>1986</td>
      <td>8818</td>
      <td>1588</td>
      <td>2607</td>
      <td>108</td>
      <td>212</td>
      <td>35608</td>
      <td>106341</td>
      <td>2676</td>
      <td>9695</td>
    </tr>
    <tr>
      <td></td>
      <td>fastqc</td>
      <td>fastqc</td>
      <td>Read QC reports using FastQC</td>
      <td>fastqc</td>
      <td>fastqc</td>
      <td></td>
      <td>FastQC</td>
      <td>This tool aims to provide a QC report which can spot problems or biases which originate either in the sequencer or in the starting library material. It can be run in one of two modes. It can either run as a stand alone interactive application for the immediate analysis of small numbers of FastQ files, or it can be run in a non-interactive mode where it would be suitable for integrating into a larger analysis pipeline for the systematic processing of large numbers of files.</td>
      <td>Sequence composition calculation, Sequencing quality control, Statistical calculation</td>
      <td>Sequencing, Data quality management, Sequence analysis</td>
      <td>To update</td>
      <td>http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</td>
      <td>Fastq Manipulation</td>
      <td>fastqc</td>
      <td>2017-12-23</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fastqc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fastqc</td>
      <td>0.74+galaxy1</td>
      <td>fastqc</td>
      <td>0.12.1</td>
      <td>Sequence composition calculation, Sequencing quality control, Statistical calculation</td>
      <td>Sequencing, Data quality management, Sequence analysis</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1784056</td>
      <td>1913280</td>
      <td>41344</td>
      <td>43940</td>
      <td>2269843</td>
      <td>3195869</td>
      <td>90828</td>
      <td>140008</td>
      <td>410004</td>
      <td>445555</td>
      <td>14195</td>
      <td>16026</td>
      <td>4463903</td>
      <td>5554704</td>
      <td>146367</td>
      <td>199974</td>
    </tr>
    <tr>
      <td></td>
      <td>fastqe</td>
      <td>fastqe</td>
      <td>FASTQE</td>
      <td>fastqe</td>
      <td>fastqe</td>
      <td></td>
      <td>FASTQE</td>
      <td>Compute quality stats for FASTQ files and print those stats as emoji... for some reason.</td>
      <td>Sequencing quality control</td>
      <td>Sequence analysis, Sequencing</td>
      <td>To update</td>
      <td>https://fastqe.com/</td>
      <td>Sequence Analysis</td>
      <td>fastqe</td>
      <td>2020-07-13</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fastqe</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fastqe</td>
      <td>0.3.1+galaxy0</td>
      <td>fastqe</td>
      <td>0.3.1</td>
      <td>Sequencing quality control</td>
      <td>Sequence analysis, Sequencing</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>5162</td>
      <td>5162</td>
      <td>2350</td>
      <td>2350</td>
      <td>10176</td>
      <td>10176</td>
      <td>4714</td>
      <td>4714</td>
      <td>2610</td>
      <td>2610</td>
      <td>991</td>
      <td>991</td>
      <td>17948</td>
      <td>17948</td>
      <td>8055</td>
      <td>8055</td>
    </tr>
    <tr>
      <td></td>
      <td>fastqtofasta</td>
      <td>fastq_to_fasta_python</td>
      <td>FASTQ to FASTA converter</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastqtofasta</td>
      <td>2019-06-21</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastqtofasta</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/galaxy_sequence_utils/fastqtofasta</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>1.2.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>208349</td>
      <td>217568</td>
      <td>3421</td>
      <td>3732</td>
      <td>131388</td>
      <td>245005</td>
      <td>5108</td>
      <td>13227</td>
      <td>18655</td>
      <td>18655</td>
      <td>679</td>
      <td>679</td>
      <td>358392</td>
      <td>481228</td>
      <td>9208</td>
      <td>17638</td>
    </tr>
    <tr>
      <td></td>
      <td>fasttree</td>
      <td>fasttree</td>
      <td>FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide or protein sequences - GVL</td>
      <td>fasttree</td>
      <td>fasttree</td>
      <td></td>
      <td>FastTree</td>
      <td>Infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide or protein sequences.</td>
      <td>Phylogenetic tree generation (from molecular sequences), Phylogenetic tree generation (maximum likelihood and Bayesian methods)</td>
      <td>Phylogenetics, Sequence analysis</td>
      <td>To update</td>
      <td>http://www.microbesonline.org/fasttree/</td>
      <td>Phylogenetics</td>
      <td>fasttree</td>
      <td>2018-02-02</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fasttree</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fasttree</td>
      <td>2.1.10</td>
      <td>fasttree</td>
      <td>2.1.11</td>
      <td></td>
      <td>Phylogenetics</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>65542</td>
      <td>65631</td>
      <td>1049</td>
      <td>1071</td>
      <td>25992</td>
      <td>25992</td>
      <td>2147</td>
      <td>2147</td>
      <td>6733</td>
      <td>7339</td>
      <td>649</td>
      <td>745</td>
      <td>98267</td>
      <td>98962</td>
      <td>3845</td>
      <td>3963</td>
    </tr>
    <tr>
      <td></td>
      <td>fastx_artifacts_filter</td>
      <td>cshl_fastx_artifacts_filter</td>
      <td>Remove sequencing artifacts</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hannonlab.cshl.edu/fastx_toolkit/</td>
      <td>Fasta Manipulation, Fastq Manipulation</td>
      <td>fastx_artifacts_filter</td>
      <td>2017-07-10</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/fastx_toolkit/fastx_artifacts_filter</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/fastx_toolkit/fastx_artifacts_filter</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>0.0.14</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1769</td>
      <td>2020</td>
      <td>88</td>
      <td>101</td>
      <td>6948</td>
      <td>24960</td>
      <td>358</td>
      <td>2424</td>
      <td>623</td>
      <td>998</td>
      <td>32</td>
      <td>50</td>
      <td>9340</td>
      <td>27978</td>
      <td>478</td>
      <td>2575</td>
    </tr>
    <tr>
      <td></td>
      <td>fastx_barcode_splitter</td>
      <td>cshl_fastx_barcode_splitter</td>
      <td>Barcode Splitter</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hannonlab.cshl.edu/fastx_toolkit/</td>
      <td>Fasta Manipulation, Fastq Manipulation</td>
      <td>fastx_barcode_splitter</td>
      <td>2017-07-10</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/fastx_toolkit/fastx_barcode_splitter</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/fastx_toolkit/fastx_barcode_splitter</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>0.0.14</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>16957</td>
      <td>18124</td>
      <td>225</td>
      <td>260</td>
      <td>37001</td>
      <td>89126</td>
      <td>919</td>
      <td>2720</td>
      <td>863</td>
      <td>989</td>
      <td>56</td>
      <td>68</td>
      <td>54821</td>
      <td>108239</td>
      <td>1200</td>
      <td>3048</td>
    </tr>
    <tr>
      <td></td>
      <td>fastx_clipper</td>
      <td>cshl_fastx_clipper</td>
      <td>Clip adapter sequences</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hannonlab.cshl.edu/fastx_toolkit/</td>
      <td>Fasta Manipulation, Fastq Manipulation</td>
      <td>fastx_clipper</td>
      <td>2017-07-10</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/fastx_toolkit/fastx_clipper</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/fastx_toolkit/fastx_clipper</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>0.0.14</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>3140</td>
      <td>3520</td>
      <td>178</td>
      <td>212</td>
      <td>13975</td>
      <td>117900</td>
      <td>1198</td>
      <td>6006</td>
      <td>409</td>
      <td>712</td>
      <td>36</td>
      <td>55</td>
      <td>17524</td>
      <td>122132</td>
      <td>1412</td>
      <td>6273</td>
    </tr>
    <tr>
      <td></td>
      <td>fastx_collapser</td>
      <td>cshl_fastx_collapser</td>
      <td>Collapse sequences</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hannonlab.cshl.edu/fastx_toolkit/</td>
      <td>Fasta Manipulation</td>
      <td>fastx_collapser</td>
      <td>2017-07-10</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/fastx_toolkit/fastx_collapser</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/fastx_toolkit/fastx_collapser</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>0.0.14</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>64187</td>
      <td>67981</td>
      <td>326</td>
      <td>349</td>
      <td>87477</td>
      <td>185555</td>
      <td>781</td>
      <td>3067</td>
      <td>16563</td>
      <td>17125</td>
      <td>58</td>
      <td>78</td>
      <td>168227</td>
      <td>270661</td>
      <td>1165</td>
      <td>3494</td>
    </tr>
    <tr>
      <td></td>
      <td>fastx_nucleotides_distribution</td>
      <td>cshl_fastx_nucleotides_distribution</td>
      <td>Draw nucleotides distribution chart</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hannonlab.cshl.edu/fastx_toolkit/</td>
      <td>Fastq Manipulation, Graphics</td>
      <td>fastx_nucleotides_distribution</td>
      <td>2017-07-10</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/fastx_toolkit/fastx_nucleotides_distribution</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/fastx_toolkit/fastx_nucleotides_distribution</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>0.0.14</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>174</td>
      <td>272</td>
      <td>61</td>
      <td>92</td>
      <td>4295</td>
      <td>25068</td>
      <td>1056</td>
      <td>4180</td>
      <td>1127</td>
      <td>1171</td>
      <td>70</td>
      <td>89</td>
      <td>5596</td>
      <td>26511</td>
      <td>1187</td>
      <td>4361</td>
    </tr>
    <tr>
      <td></td>
      <td>fastx_quality_statistics</td>
      <td>cshl_fastx_quality_statistics</td>
      <td>Compute quality statistics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hannonlab.cshl.edu/fastx_toolkit/</td>
      <td>Fastq Manipulation, Statistics</td>
      <td>fastx_quality_statistics</td>
      <td>2018-04-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/fastx_toolkit/fastx_quality_statistics</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/fastx_toolkit/fastx_quality_statistics</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>0.0.14</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2218</td>
      <td>2344</td>
      <td>168</td>
      <td>206</td>
      <td>20551</td>
      <td>60032</td>
      <td>3784</td>
      <td>9866</td>
      <td>2347</td>
      <td>2460</td>
      <td>234</td>
      <td>256</td>
      <td>25116</td>
      <td>64836</td>
      <td>4186</td>
      <td>10328</td>
    </tr>
    <tr>
      <td></td>
      <td>fastx_renamer</td>
      <td>cshl_fastx_renamer</td>
      <td>Rename sequences</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hannonlab.cshl.edu/fastx_toolkit/</td>
      <td>Fasta Manipulation, Fastq Manipulation</td>
      <td>fastx_renamer</td>
      <td>2017-07-10</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/fastx_toolkit/fastx_renamer</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/fastx_toolkit/fastx_renamer</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>0.0.14</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2364</td>
      <td>2526</td>
      <td>180</td>
      <td>214</td>
      <td>6572</td>
      <td>11745</td>
      <td>303</td>
      <td>1024</td>
      <td>462</td>
      <td>696</td>
      <td>49</td>
      <td>64</td>
      <td>9398</td>
      <td>14967</td>
      <td>532</td>
      <td>1302</td>
    </tr>
    <tr>
      <td></td>
      <td>fastx_reverse_complement</td>
      <td>cshl_fastx_reverse_complement</td>
      <td>Reverse-Complement</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hannonlab.cshl.edu/fastx_toolkit/</td>
      <td>Fastq Manipulation, Fasta Manipulation</td>
      <td>fastx_reverse_complement</td>
      <td>2017-07-10</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/fastx_toolkit/fastx_reverse_complement</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/fastx_toolkit/fastx_reverse_complement</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>0.0.14</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>10707</td>
      <td>12597</td>
      <td>356</td>
      <td>415</td>
      <td>21568</td>
      <td>67304</td>
      <td>574</td>
      <td>1527</td>
      <td>719</td>
      <td>861</td>
      <td>61</td>
      <td>78</td>
      <td>32994</td>
      <td>80762</td>
      <td>991</td>
      <td>2020</td>
    </tr>
    <tr>
      <td></td>
      <td>fastx_trimmer</td>
      <td>cshl_fastx_trimmer</td>
      <td>Trim sequences</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hannonlab.cshl.edu/fastx_toolkit/</td>
      <td>Fasta Manipulation, Fastq Manipulation</td>
      <td>fastx_trimmer</td>
      <td>2017-07-10</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/fastx_toolkit/fastx_trimmer</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/fastx_toolkit/fastx_trimmer</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>0.0.14</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>13584</td>
      <td>14557</td>
      <td>550</td>
      <td>641</td>
      <td>123676</td>
      <td>206733</td>
      <td>2034</td>
      <td>8390</td>
      <td>2819</td>
      <td>4757</td>
      <td>229</td>
      <td>267</td>
      <td>140079</td>
      <td>226047</td>
      <td>2813</td>
      <td>9298</td>
    </tr>
    <tr>
      <td></td>
      <td>fatovcf</td>
      <td>fatovcf</td>
      <td>Convert a FASTA alignment file to Variant Call Format (VCF) single-nucleotide diffs</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td></td>
      <td>UCSC Genome Browser Utilities</td>
      <td>Utilities for handling sequences and assemblies from the UCSC Genome Browser project.</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>Up-to-date</td>
      <td>http://hgdownload.cse.ucsc.edu/admin/exe/</td>
      <td>Convert Formats</td>
      <td>ucsc_fatovcf</td>
      <td>2023-01-11</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ucsc_tools/fatovcf</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ucsc_tools/fatovcf</td>
      <td>448</td>
      <td>ucsc-fatovcf</td>
      <td>448</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>385</td>
      <td>385</td>
      <td>46</td>
      <td>46</td>
      <td>587</td>
      <td>587</td>
      <td>107</td>
      <td>107</td>
      <td>58</td>
      <td>58</td>
      <td>5</td>
      <td>5</td>
      <td>1030</td>
      <td>1030</td>
      <td>158</td>
      <td>158</td>
    </tr>
    <tr>
      <td></td>
      <td>featurecounts</td>
      <td>featurecounts</td>
      <td>featureCounts counts the number of reads aligned to defined masked regions in a reference genome</td>
      <td>featurecounts</td>
      <td>featurecounts</td>
      <td></td>
      <td>FeatureCounts</td>
      <td>featureCounts is a very efficient read quantifier. It can be used to summarize RNA-seq reads and gDNA-seq reads to a variety of genomic features such as genes, exons, promoters, gene bodies and genomic bins. It is included in the Bioconductor Rsubread package and also in the SourceForge Subread package.</td>
      <td>Read summarisation, RNA-Seq quantification</td>
      <td>RNA-Seq</td>
      <td>To update</td>
      <td>http://bioinf.wehi.edu.au/featureCounts</td>
      <td>RNA, Transcriptomics, Sequence Analysis</td>
      <td>featurecounts</td>
      <td>2019-05-28</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/featurecounts</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/featurecounts</td>
      <td>2.0.3</td>
      <td>subread</td>
      <td>2.0.6</td>
      <td>Read summarisation, RNA-Seq quantification</td>
      <td>RNA-Seq</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>740719</td>
      <td>770524</td>
      <td>11699</td>
      <td>12547</td>
      <td>643892</td>
      <td>784396</td>
      <td>23272</td>
      <td>27598</td>
      <td>66108</td>
      <td>68867</td>
      <td>2348</td>
      <td>2463</td>
      <td>1450719</td>
      <td>1623787</td>
      <td>37319</td>
      <td>42608</td>
    </tr>
    <tr>
      <td></td>
      <td>feelnc</td>
      <td>feelnc</td>
      <td>Galaxy wrapper for FEELnc</td>
      <td>feelnc</td>
      <td>feelnc</td>
      <td></td>
      <td>FEELnc</td>
      <td>A tool to annotate long non-coding RNAs from RNA-seq assembled transcripts.</td>
      <td>Annotation, Classification</td>
      <td>RNA-seq, Functional, regulatory and non-coding RNA</td>
      <td>To update</td>
      <td>https://github.com/tderrien/FEELnc</td>
      <td>Sequence Analysis</td>
      <td>feelnc</td>
      <td>2018-03-27</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/feelnc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/feelnc</td>
      <td>0.2.1</td>
      <td>feelnc</td>
      <td>0.2</td>
      <td>Annotation, Classification</td>
      <td>Functional, regulatory and non-coding RNA</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1375</td>
      <td>1500</td>
      <td>95</td>
      <td>99</td>
      <td>797</td>
      <td>797</td>
      <td>116</td>
      <td>116</td>
      <td>59</td>
      <td>59</td>
      <td>3</td>
      <td>3</td>
      <td>2231</td>
      <td>2356</td>
      <td>214</td>
      <td>218</td>
    </tr>
    <tr>
      <td></td>
      <td>fermikit</td>
      <td>fermi2, fermikit_variants</td>
      <td>FermiKit is a de novo assembly based variant calling pipeline for deep Illumina resequencing data.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/lh3/fermikit</td>
      <td>Assembly, Variant Analysis</td>
      <td>fermikit</td>
      <td>2017-01-05</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fermikit</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fermikit</td>
      <td>r193</td>
      <td>fermi2</td>
      <td>r193</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>fgsea</td>
      <td>fgsea</td>
      <td>Perform gene set testing using fgsea</td>
      <td>fgsea</td>
      <td>fgsea</td>
      <td></td>
      <td>fgsea</td>
      <td>The package implements an algorithm for fast gene set enrichment analysis. Using the fast algorithm allows to make more permutations and get more fine grained p-values, which allows to use accurate stantard approaches to multiple hypothesis correction.</td>
      <td>Gene-set enrichment analysis</td>
      <td>Genetics</td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/fgsea.html</td>
      <td>Visualization, Transcriptomics, Statistics</td>
      <td>fgsea</td>
      <td>2018-10-19</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fgsea</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fgsea</td>
      <td>1.8.0+galaxy1</td>
      <td>bioconductor-fgsea</td>
      <td>1.28.0</td>
      <td>Gene-set enrichment analysis</td>
      <td>Genetics</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>5618</td>
      <td>5820</td>
      <td>582</td>
      <td>607</td>
      <td>21673</td>
      <td>21673</td>
      <td>938</td>
      <td>938</td>
      <td>869</td>
      <td>869</td>
      <td>110</td>
      <td>110</td>
      <td>28160</td>
      <td>28362</td>
      <td>1630</td>
      <td>1655</td>
    </tr>
    <tr>
      <td></td>
      <td>filtlong</td>
      <td>filtlong</td>
      <td>Filtlong - Filtering long reads by quality</td>
      <td>filtlong</td>
      <td>filtlong</td>
      <td></td>
      <td>Filtlong</td>
      <td>Filtlong is a tool for filtering long reads by quality. It can take a set of long reads and produce a smaller, better subset. It uses both read length (longer is better) and read identity (higher is better) when choosing which reads pass the filter.</td>
      <td>Filtering, Sequencing quality control</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/rrwick/Filtlong</td>
      <td>Fastq Manipulation, Sequence Analysis</td>
      <td>filtlong</td>
      <td>2018-09-14</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/filtlong</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/filtlong</td>
      <td>0.2.1</td>
      <td>filtlong</td>
      <td>0.2.1</td>
      <td>Filtering, Sequencing quality control</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>53310</td>
      <td>53944</td>
      <td>1499</td>
      <td>1575</td>
      <td>25043</td>
      <td>25043</td>
      <td>726</td>
      <td>726</td>
      <td>10824</td>
      <td>10824</td>
      <td>593</td>
      <td>593</td>
      <td>89177</td>
      <td>89811</td>
      <td>2818</td>
      <td>2894</td>
    </tr>
    <tr>
      <td></td>
      <td>flair</td>
      <td>flair_collapse, flair_correct</td>
      <td>FLAIR (Full-Length Alternative Isoform analysis of RNA) for the correction, isoform definition, and alternative splicing analysis of noisy reads.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/BrooksLabUCSC/flair</td>
      <td>Nanopore</td>
      <td>flair</td>
      <td>2020-11-19</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/flair</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/flair</td>
      <td>1.5</td>
      <td>flair</td>
      <td>2.0.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>1985</td>
      <td>1985</td>
      <td>41</td>
      <td>41</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1985</td>
      <td>1985</td>
      <td>41</td>
      <td>41</td>
    </tr>
    <tr>
      <td></td>
      <td>flash</td>
      <td>flash</td>
      <td>Fast Length Adjustment of SHort reads</td>
      <td>flash</td>
      <td>flash</td>
      <td></td>
      <td>FLASH</td>
      <td>Identifies paired-end reads which overlap in the middle, converting them to single long reads</td>
      <td>Read pre-processing, Sequence merging, Sequence assembly</td>
      <td>Sequencing, Sequence assembly</td>
      <td>Up-to-date</td>
      <td>https://ccb.jhu.edu/software/FLASH/</td>
      <td>Assembly, Fastq Manipulation</td>
      <td>flash</td>
      <td>2017-09-25</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/flash</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/flash</td>
      <td>1.2.11</td>
      <td>flash</td>
      <td>1.2.11</td>
      <td>Read pre-processing, Sequence merging, Sequence assembly</td>
      <td>Sequencing, Sequence assembly</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>22017</td>
      <td>22304</td>
      <td>239</td>
      <td>255</td>
      <td>9399</td>
      <td>14886</td>
      <td>462</td>
      <td>709</td>
      <td>4</td>
      <td>4</td>
      <td>1</td>
      <td>1</td>
      <td>31420</td>
      <td>37194</td>
      <td>702</td>
      <td>965</td>
    </tr>
    <tr>
      <td></td>
      <td>fraggenescan</td>
      <td>fraggenescan</td>
      <td>Tool for finding (fragmented) genes in short read</td>
      <td>fraggenescan</td>
      <td>fraggenescan</td>
      <td></td>
      <td>FragGeneScan</td>
      <td>Application for finding (fragmented) genes in short reads</td>
      <td>Gene prediction</td>
      <td>Genetics, Sequence analysis</td>
      <td>To update</td>
      <td>https://sourceforge.net/projects/fraggenescan/</td>
      <td>Sequence Analysis</td>
      <td>fraggenescan</td>
      <td>2017-09-04</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fraggenescan/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fraggenescan</td>
      <td></td>
      <td>fraggenescan</td>
      <td>1.31</td>
      <td>Gene prediction</td>
      <td>Genetics, Sequence analysis</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1257</td>
      <td>1450</td>
      <td>150</td>
      <td>162</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1052</td>
      <td>1052</td>
      <td>45</td>
      <td>45</td>
      <td>2309</td>
      <td>2502</td>
      <td>195</td>
      <td>207</td>
    </tr>
    <tr>
      <td></td>
      <td>freebayes</td>
      <td>freebayes, bamleftalign</td>
      <td>Galaxy Freebayes Bayesian genetic variant detector tool</td>
      <td>freebayes</td>
      <td>freebayes</td>
      <td></td>
      <td>FreeBayes</td>
      <td>Bayesian genetic variant detector designed to find small polymorphisms, specifically SNPs, indels, multi-nucleotide polymorphisms, and complex events (composite insertion and substitution events) smaller than the length of a short-read sequencing alignment.</td>
      <td>Variant calling, Statistical calculation</td>
      <td>Genomics, Genetic variation, Rare diseases</td>
      <td>Up-to-date</td>
      <td>https://github.com/ekg/freebayes</td>
      <td>Variant Analysis</td>
      <td>freebayes</td>
      <td>2017-01-23</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/freebayes</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/freebayes</td>
      <td>1.3.8</td>
      <td>freebayes</td>
      <td>1.3.8</td>
      <td>Variant calling, Statistical calculation</td>
      <td>Genomics, Genetic variation, Rare diseases</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>149715</td>
      <td>171093</td>
      <td>8158</td>
      <td>8794</td>
      <td>144708</td>
      <td>256315</td>
      <td>15996</td>
      <td>25107</td>
      <td>19156</td>
      <td>21936</td>
      <td>2251</td>
      <td>2571</td>
      <td>313579</td>
      <td>449344</td>
      <td>26405</td>
      <td>36472</td>
    </tr>
    <tr>
      <td></td>
      <td>freec</td>
      <td>control_freec</td>
      <td>Control-FREEC is a tool for detection of copy-number changes and allelic imbalances (including LOH) using deep-sequencing data originally developed by the Bioinformatics Laboratory of Institut Curie (Paris). It automatically computes, normalizes, segments copy number and beta allele frequency (BAF) profiles, then calls copy number alterations and LOH.</td>
      <td>freec</td>
      <td>freec</td>
      <td></td>
      <td>FREEC</td>
      <td>A tool for control-free copy number alteration (CNA) and allelic imbalances (LOH) detection using deep-sequencing data, particularly useful for cancer studies.</td>
      <td>Copy number estimation, Variant calling, Genome alignment</td>
      <td>DNA structural variation, Oncology, Human genetics, Data mining</td>
      <td>To update</td>
      <td>http://boevalab.inf.ethz.ch/FREEC/</td>
      <td>Variant Analysis</td>
      <td>control_freec</td>
      <td>2020-08-13</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/freec</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/freec</td>
      <td>11.6</td>
      <td>gawk</td>
      <td></td>
      <td>Copy number estimation, Variant calling, Genome alignment</td>
      <td>Oncology, Human genetics, Data mining</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>748</td>
      <td>748</td>
      <td>91</td>
      <td>91</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>12</td>
      <td>12</td>
      <td>1</td>
      <td>1</td>
      <td>760</td>
      <td>760</td>
      <td>92</td>
      <td>92</td>
    </tr>
    <tr>
      <td></td>
      <td>freyja</td>
      <td>freyja_aggregate_plot, freyja_boot, freyja_demix, freyja_variants</td>
      <td>lineage abundances estimation</td>
      <td>freyja</td>
      <td>freyja</td>
      <td></td>
      <td>Freyja</td>
      <td>Recover relative lineage abundances from mixed SARS-CoV-2 samples from a sequencing dataset</td>
      <td>RNA-Seq quantification</td>
      <td>Metagenomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/andersen-lab/Freyja</td>
      <td>Metagenomics, Sequence Analysis</td>
      <td>freyja</td>
      <td>2022-07-05</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/freyja</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/freyja</td>
      <td>1.5.1</td>
      <td>freyja</td>
      <td>1.5.1</td>
      <td>RNA-Seq quantification</td>
      <td>Metagenomics</td>
      <td>2</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>21757</td>
      <td>21757</td>
      <td>101</td>
      <td>101</td>
      <td>217</td>
      <td>217</td>
      <td>21</td>
      <td>21</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>21974</td>
      <td>21974</td>
      <td>122</td>
      <td>122</td>
    </tr>
    <tr>
      <td></td>
      <td>fsd</td>
      <td>fsd, fsd_beforevsafter, fsd_regions, td</td>
      <td>Tool that plots a histogram of sizes of read families</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Graphics</td>
      <td>duplex_family_size_distribution</td>
      <td>2019-10-24</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fsd</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fsd</td>
      <td>1.0.2</td>
      <td>matplotlib</td>
      <td></td>
      <td></td>
      <td></td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>578</td>
      <td>578</td>
      <td>34</td>
      <td>34</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>578</td>
      <td>578</td>
      <td>34</td>
      <td>34</td>
    </tr>
    <tr>
      <td></td>
      <td>funannotate</td>
      <td>funannotate_annotate, funannotate_clean, funannotate_compare, funannotate_predict, funannotate_sort</td>
      <td>Funannotate is a genome prediction, annotation, and comparison software package.</td>
      <td>funannotate</td>
      <td>funannotate</td>
      <td></td>
      <td>funannotate</td>
      <td>funannotate is a pipeline for genome annotation (built specifically for fungi, but will also work with higher eukaryotes).</td>
      <td>Genome annotation</td>
      <td>Genomics</td>
      <td>To update</td>
      <td>https://funannotate.readthedocs.io</td>
      <td>Genome annotation</td>
      <td></td>
      <td>2021-08-26</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/funannotate</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/funannotate</td>
      <td>1.8.15</td>
      <td></td>
      <td></td>
      <td>Genome annotation</td>
      <td>Genomics</td>
      <td>3</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>3</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>8648</td>
      <td>8648</td>
      <td>1408</td>
      <td>1408</td>
      <td>32</td>
      <td>32</td>
      <td>6</td>
      <td>6</td>
      <td>6406</td>
      <td>6406</td>
      <td>597</td>
      <td>597</td>
      <td>15086</td>
      <td>15086</td>
      <td>2011</td>
      <td>2011</td>
    </tr>
    <tr>
      <td></td>
      <td>gatk4</td>
      <td>gatk4_mutect2</td>
      <td>A Galaxy wrapper for Mutect2 from GATK</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://software.broadinstitute.org/gatk/gatk4</td>
      <td>Variant Analysis</td>
      <td>gatk4_mutect2</td>
      <td>2019-10-28</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gatk4</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/gatk4</td>
      <td>4.1.7.0</td>
      <td>gatk4</td>
      <td>4.5.0.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>3463</td>
      <td>3463</td>
      <td>452</td>
      <td>452</td>
      <td>9040</td>
      <td>9040</td>
      <td>1639</td>
      <td>1639</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>12503</td>
      <td>12503</td>
      <td>2091</td>
      <td>2091</td>
    </tr>
    <tr>
      <td></td>
      <td>gdcwebapp</td>
      <td>data_source_gdcwebapp</td>
      <td>GDCWebApp automatically filter, extract, and convert genomic data from the Genomic Data Commons portal to BED format</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://bioinf.iasi.cnr.it/gdcwebapp/</td>
      <td>Data Source, Convert Formats</td>
      <td>gdcwebapp</td>
      <td>2017-06-23</td>
      <td>iuc</td>
      <td>https://github.com/fabio-cumbo/GDCWebApp4Galaxy</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/gdcwebapp</td>
      <td>1.0.0</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>gecko</td>
      <td>gecko</td>
      <td>Ungapped genome comparison</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/otorreno/gecko</td>
      <td>Sequence Analysis</td>
      <td>gecko</td>
      <td>2020-11-25</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gecko</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/gecko</td>
      <td>1.2</td>
      <td>gecko</td>
      <td>1.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>723</td>
      <td>723</td>
      <td>235</td>
      <td>235</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>786</td>
      <td>786</td>
      <td>95</td>
      <td>95</td>
      <td>1509</td>
      <td>1509</td>
      <td>330</td>
      <td>330</td>
    </tr>
    <tr>
      <td></td>
      <td>gemini</td>
      <td>gemini_@BINARY@, gemini_@BINARY@, gemini_@BINARY@, gemini_@BINARY@, gemini_db_info, gemini_@BINARY@, gemini_@BINARY@, gemini_inheritance, gemini_@BINARY@, gemini_@BINARY@, gemini_@BINARY@, gemini_@BINARY@, gemini_@BINARY@, gemini_@BINARY@, gemini_@BINARY@, gemini_@BINARY@, gemini_@BINARY@, gemini_@BINARY@</td>
      <td>GEMINI: a flexible framework for exploring genome variation</td>
      <td>gemini</td>
      <td>gemini</td>
      <td></td>
      <td>GEMINI</td>
      <td>GEMINI (GEnome MINIng) is a flexible framework for exploring genetic variation in the context of the wealth of genome annotations available for the human genome. By placing genetic variants, sample phenotypes and genotypes, as well as genome annotations into an integrated database framework, GEMINI provides a simple, flexible, and powerful system for exploring genetic variation for disease and population genetics.</td>
      <td>Sequence analysis, Genetic variation analysis</td>
      <td>Sequence analysis</td>
      <td>To update</td>
      <td>https://github.com/arq5x/gemini</td>
      <td>Sequence Analysis, Next Gen Mappers</td>
      <td>gemini</td>
      <td>2019-01-05</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gemini</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/gemini</td>
      <td>0.20.1</td>
      <td>gemini</td>
      <td>0.30.2</td>
      <td>Sequence analysis, Genetic variation analysis</td>
      <td>Sequence analysis</td>
      <td>1</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>1</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>4967</td>
      <td>5475</td>
      <td>1863</td>
      <td>1992</td>
      <td>992</td>
      <td>1118</td>
      <td>279</td>
      <td>370</td>
      <td>3756</td>
      <td>3811</td>
      <td>423</td>
      <td>425</td>
      <td>9715</td>
      <td>10404</td>
      <td>2565</td>
      <td>2787</td>
    </tr>
    <tr>
      <td></td>
      <td>genebed_maf_to_fasta</td>
      <td>GeneBed_Maf_Fasta2</td>
      <td>Stitch gene blocks given a set of coding exon intervals</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/genebed_maf_to_fasta/</td>
      <td>Genomic Interval Operations</td>
      <td>genebed_maf_to_fasta</td>
      <td>2020-08-05</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/genebed_maf_to_fasta/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/genebed_maf_to_fasta</td>
      <td>1.0.1+galaxy0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>2</td>
      <td>2</td>
      <td>221</td>
      <td>2905</td>
      <td>23</td>
      <td>279</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>225</td>
      <td>2909</td>
      <td>25</td>
      <td>281</td>
    </tr>
    <tr>
      <td></td>
      <td>genehunter_modscore</td>
      <td>genehunter_modscore</td>
      <td>Maximised LOD score pedigree analysis utility</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.unimedizin-mainz.de/imbei/biometricsgenomic-statistics-and-bioinformatics/software/genehunter-modscore-40.html?L=1</td>
      <td>Variant Analysis</td>
      <td>genehunter_modscore</td>
      <td>2017-11-27</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/genehunter_modscore/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/genehunter_modscore</td>
      <td>3.0.0</td>
      <td>ghm</td>
      <td>3.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>geneiobio</td>
      <td>gene_iobio_display_generation_iframe</td>
      <td>Gene.iobio is an interactive tool for variant and trio analysis.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/iobio/gene.iobio</td>
      <td>Sequence Analysis</td>
      <td>geneiobio</td>
      <td>2023-06-20</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/geneiobio</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/geneiobio</td>
      <td>4.7.1+galaxy1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>101</td>
      <td>101</td>
      <td>25</td>
      <td>25</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>59</td>
      <td>59</td>
      <td>6</td>
      <td>6</td>
      <td>160</td>
      <td>160</td>
      <td>31</td>
      <td>31</td>
    </tr>
    <tr>
      <td></td>
      <td>genetrack</td>
      <td>genetrack</td>
      <td>Contains a tool that separately identifies peaks on the forward "+ (W) and reverse - (C) strand.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>ChIP-seq</td>
      <td>genetrack</td>
      <td>2015-12-18</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/genetrack</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/genetrack</td>
      <td></td>
      <td>numpy</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>445</td>
      <td>660</td>
      <td>104</td>
      <td>197</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>445</td>
      <td>660</td>
      <td>104</td>
      <td>197</td>
    </tr>
    <tr>
      <td></td>
      <td>genomescope</td>
      <td>genomescope</td>
      <td>Analyze unassembled short reads</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/tbenavi1/genomescope2.0</td>
      <td>Statistics</td>
      <td>genomescope</td>
      <td>2021-04-30</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/genomescope</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/genomescope</td>
      <td>2.0.1</td>
      <td>genomescope2</td>
      <td>2.0.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>3181</td>
      <td>3181</td>
      <td>665</td>
      <td>665</td>
      <td>2112</td>
      <td>2112</td>
      <td>435</td>
      <td>435</td>
      <td>1622</td>
      <td>1622</td>
      <td>148</td>
      <td>148</td>
      <td>6915</td>
      <td>6915</td>
      <td>1248</td>
      <td>1248</td>
    </tr>
    <tr>
      <td></td>
      <td>genomic_super_signature</td>
      <td>genomic_super_signature</td>
      <td>Interpretation of RNAseq experiments through robust, efficient comparison to public databases</td>
      <td>genomicsupersignature</td>
      <td>genomicsupersignature</td>
      <td></td>
      <td>GenomicSuperSignature</td>
      <td>GenomicSuperSignature is a package for the interpretation of RNA-seq experiments through robust, efficient comparison to public databases.</td>
      <td>Gene-set enrichment analysis, Essential dynamics, Deposition, Principal component visualisation, Dimensionality reduction</td>
      <td>RNA-Seq, Transcriptomics, Microbial ecology, Genotype and phenotype, Microarray experiment</td>
      <td>To update</td>
      <td>https://github.com/shbrief/GenomicSuperSignature</td>
      <td>Sequence Analysis, RNA, Transcriptomics</td>
      <td>genomic_super_signature</td>
      <td>2022-01-12</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/genomic_super_signature</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/genomic_super_signature</td>
      <td>1.2.0</td>
      <td>bioconductor-genomicsupersignature</td>
      <td>1.10.0</td>
      <td>Gene-set enrichment analysis, Essential dynamics, Deposition, Principal component visualisation, Dimensionality reduction</td>
      <td>RNA-Seq, Transcriptomics, Microbial ecology, Genotype and phenotype, Microarray experiment</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>47</td>
      <td>47</td>
      <td>13</td>
      <td>13</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>47</td>
      <td>47</td>
      <td>13</td>
      <td>13</td>
    </tr>
    <tr>
      <td></td>
      <td>genrich</td>
      <td>genrich</td>
      <td>Genrich is a peak-caller for genomic enrichment assays (e.g. ChIP-seq, ATAC-seq).</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/jsh58/Genrich</td>
      <td>ChIP-seq</td>
      <td>genrich</td>
      <td>2019-07-05</td>
      <td>iuc</td>
      <td>https://github.com/jsh58/Genrich</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/genrich</td>
      <td>0.5+galaxy2</td>
      <td>genrich</td>
      <td>0.6.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>3534</td>
      <td>3577</td>
      <td>402</td>
      <td>405</td>
      <td>2840</td>
      <td>2840</td>
      <td>419</td>
      <td>419</td>
      <td>571</td>
      <td>571</td>
      <td>31</td>
      <td>31</td>
      <td>6945</td>
      <td>6988</td>
      <td>852</td>
      <td>855</td>
    </tr>
    <tr>
      <td></td>
      <td>get_hrun</td>
      <td>get_hrun</td>
      <td>Annotate indel variants with homopolymer context</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/get_hrun</td>
      <td>Variant Analysis</td>
      <td>get_hrun</td>
      <td>2021-03-02</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/get_hrun</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/get_hrun</td>
      <td>0.5.9.2</td>
      <td>pyfaidx</td>
      <td>0.8.1.3</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>9</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>9</td>
      <td>5</td>
      <td>5</td>
    </tr>
    <tr>
      <td></td>
      <td>getorganelle</td>
      <td>get_annotated_regions_from_gb, get_organelle_from_reads</td>
      <td>GetOrganelle - This toolkit assembles organelle genomes from genomic skimming data.</td>
      <td>getorganelle</td>
      <td>getorganelle</td>
      <td></td>
      <td>GetOrganelle</td>
      <td>A fast and versatile toolkit for accurate de novo assembly of organelle genomes.This toolkit assemblies organelle genome from genomic skimming data.</td>
      <td>De-novo assembly, Genome assembly, Mapping assembly, Mapping, Sequence trimming</td>
      <td>Cell biology, Sequence assembly, Whole genome sequencing, Plant biology, Model organisms</td>
      <td>Up-to-date</td>
      <td>https://github.com/Kinggerm/GetOrganelle</td>
      <td>Assembly</td>
      <td>getorganelle</td>
      <td>2022-05-04</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/getorganelle</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/getorganelle</td>
      <td>1.7.7.1</td>
      <td>getorganelle</td>
      <td>1.7.7.1</td>
      <td>De-novo assembly, Genome assembly, Mapping assembly, Mapping, Sequence trimming</td>
      <td>Cell biology, Sequence assembly, Whole genome sequencing, Plant biology, Model organisms</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>1542</td>
      <td>1542</td>
      <td>159</td>
      <td>159</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>587</td>
      <td>587</td>
      <td>64</td>
      <td>64</td>
      <td>2129</td>
      <td>2129</td>
      <td>223</td>
      <td>223</td>
    </tr>
    <tr>
      <td></td>
      <td>gfa_to_fa</td>
      <td>gfa_to_fa</td>
      <td>gfa_to_fa - Converting GFA format to Fasta format</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://gfa-spec.github.io/GFA-spec/</td>
      <td>Convert Formats</td>
      <td>gfa_to_fa</td>
      <td>2019-06-18</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gfa_to_fa</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/gfa_to_fa</td>
      <td>0.1.2</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>9664</td>
      <td>9775</td>
      <td>763</td>
      <td>765</td>
      <td>5623</td>
      <td>5623</td>
      <td>493</td>
      <td>493</td>
      <td>1000</td>
      <td>1000</td>
      <td>135</td>
      <td>135</td>
      <td>16287</td>
      <td>16398</td>
      <td>1391</td>
      <td>1393</td>
    </tr>
    <tr>
      <td></td>
      <td>gff3_rebase</td>
      <td>gff3.rebase</td>
      <td>Rebase a GFF against a parent GFF (e.g. an original genome)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gff3_rebase</td>
      <td>Sequence Analysis</td>
      <td>gff3_rebase</td>
      <td>2015-05-25</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gff3_rebase</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/gff3_rebase</td>
      <td>1.2</td>
      <td>bcbiogff</td>
      <td>0.6.6</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>806</td>
      <td>812</td>
      <td>68</td>
      <td>68</td>
      <td>197</td>
      <td>197</td>
      <td>37</td>
      <td>37</td>
      <td>132</td>
      <td>132</td>
      <td>12</td>
      <td>12</td>
      <td>1135</td>
      <td>1141</td>
      <td>117</td>
      <td>117</td>
    </tr>
    <tr>
      <td></td>
      <td>gffcompare</td>
      <td>gffcompare</td>
      <td>Galaxy wrappers for Geo Pertea's GffCompare package.</td>
      <td>gffcompare</td>
      <td>gffcompare</td>
      <td></td>
      <td>gffcompare</td>
      <td>Program for comparing, annotating, merging and tracking transcripts in GFF files.</td>
      <td>Sequence annotation</td>
      <td>Nucleic acids, Sequence analysis</td>
      <td>Up-to-date</td>
      <td>https://github.com/gpertea/gffcompare/</td>
      <td>Transcriptomics</td>
      <td>gffcompare</td>
      <td>2018-10-29</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gffcompare</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/gffcompare</td>
      <td>0.12.6</td>
      <td>gffcompare</td>
      <td>0.12.6</td>
      <td>Sequence annotation</td>
      <td>Nucleic acids, Sequence analysis</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>3560</td>
      <td>3980</td>
      <td>916</td>
      <td>999</td>
      <td>5281</td>
      <td>8000</td>
      <td>1713</td>
      <td>2521</td>
      <td>1468</td>
      <td>1642</td>
      <td>208</td>
      <td>225</td>
      <td>10309</td>
      <td>13622</td>
      <td>2837</td>
      <td>3745</td>
    </tr>
    <tr>
      <td></td>
      <td>gffread</td>
      <td>gffread</td>
      <td>gffread filters and/or converts GFF3/GTF2 records</td>
      <td>gffread</td>
      <td>gffread</td>
      <td></td>
      <td>gffread</td>
      <td>program for filtering, converting and manipulating GFF files</td>
      <td>Sequence annotation</td>
      <td>Nucleic acids, Sequence analysis</td>
      <td>Up-to-date</td>
      <td>http://ccb.jhu.edu/software/stringtie/gff.shtml#gffread/</td>
      <td>Sequence Analysis</td>
      <td>gffread</td>
      <td>2019-10-29</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gffread</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/gffread</td>
      <td>0.12.7</td>
      <td>gffread</td>
      <td>0.12.7</td>
      <td>Sequence annotation</td>
      <td>Nucleic acids, Sequence analysis</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>12828</td>
      <td>13716</td>
      <td>1740</td>
      <td>1829</td>
      <td>15217</td>
      <td>23818</td>
      <td>2588</td>
      <td>3486</td>
      <td>4423</td>
      <td>4423</td>
      <td>732</td>
      <td>732</td>
      <td>32468</td>
      <td>41957</td>
      <td>5060</td>
      <td>6047</td>
    </tr>
    <tr>
      <td></td>
      <td>ggplot2</td>
      <td>ggplot2_heatmap, ggplot2_pca, ggplot2_histogram, ggplot2_point, ggplot2_violin</td>
      <td>ggplot2 is a system for declaratively creating graphics, based on The Grammar of Graphics.You provide the data, tell ggplot2 how to map variables to aesthetics, what graphical primitives to use,and it takes care of the details.</td>
      <td>ggplot2</td>
      <td>ggplot2</td>
      <td></td>
      <td>ggplot2</td>
      <td>Plotting system for R, based on the grammar of graphics.</td>
      <td>Visualisation</td>
      <td>Data visualisation</td>
      <td>To update</td>
      <td>https://github.com/tidyverse/ggplot2</td>
      <td>Visualization</td>
      <td></td>
      <td>2018-04-26</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ggplot2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ggplot2</td>
      <td>3.4.0</td>
      <td>r-base</td>
      <td></td>
      <td>Visualisation</td>
      <td>Data visualisation</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>24606</td>
      <td>25662</td>
      <td>3463</td>
      <td>3630</td>
      <td>26805</td>
      <td>26808</td>
      <td>5143</td>
      <td>5144</td>
      <td>8174</td>
      <td>8237</td>
      <td>1259</td>
      <td>1279</td>
      <td>59585</td>
      <td>60707</td>
      <td>9865</td>
      <td>10053</td>
    </tr>
    <tr>
      <td></td>
      <td>ggupset</td>
      <td>emc-ggupset</td>
      <td>Create Upset Plots with ggupset</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/const-ae/ggupset</td>
      <td>Graphics</td>
      <td>ggupset</td>
      <td>2022-12-17</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ggupset</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ggupset</td>
      <td>1.0</td>
      <td>r-ggupset</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>glimmer</td>
      <td>glimmer_acgt_content, glimmer_build_icm, glimmer_extract, glimmer_gbk_to_orf, glimmer_glimmer_to_gff, glimmer_long_orfs, glimmer_knowledge_based, glimmer_not_knowledge_based</td>
      <td>Glimmer makes gene predictions.</td>
      <td>gemini</td>
      <td>gemini</td>
      <td></td>
      <td>GEMINI</td>
      <td>GEMINI (GEnome MINIng) is a flexible framework for exploring genetic variation in the context of the wealth of genome annotations available for the human genome. By placing genetic variants, sample phenotypes and genotypes, as well as genome annotations into an integrated database framework, GEMINI provides a simple, flexible, and powerful system for exploring genetic variation for disease and population genetics.</td>
      <td>Sequence analysis, Genetic variation analysis</td>
      <td>Sequence analysis</td>
      <td>To update</td>
      <td>https://ccb.jhu.edu/software/glimmer/</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>2017-11-28</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/glimmer</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/glimmer</td>
      <td></td>
      <td>glimmer</td>
      <td>3.02</td>
      <td>Sequence analysis, Genetic variation analysis</td>
      <td>Sequence analysis</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>4343</td>
      <td>4343</td>
      <td>407</td>
      <td>407</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4343</td>
      <td>4343</td>
      <td>407</td>
      <td>407</td>
    </tr>
    <tr>
      <td></td>
      <td>goenrichment</td>
      <td>goenrichment, goslimmer</td>
      <td>Performs GO Enrichment analysis.</td>
      <td>goenrichment</td>
      <td>goenrichment</td>
      <td></td>
      <td>GOEnrichment</td>
      <td>GOEnrichment is a tool for performing GO enrichment analysis of gene sets, such as those obtained from RNA-seq or Microarray experiments, to help characterize them at the functional level. It is available in Galaxy Europe and as a stand-alone tool.GOEnrichment is flexible in that it allows the user to use any version of the Gene Ontology and any GO annotation file they desire. To enable the use of GO slims, it is accompanied by a sister tool GOSlimmer, which can convert annotation files from full GO to any specified GO slim.The tool features an optional graph clustering algorithm to reduce the redundancy in the set of enriched GO terms and simplify its output.It was developed by the BioData.pt / ELIXIR-PT team at the Instituto Gulbenkian de Cincia.</td>
      <td>Gene-set enrichment analysis</td>
      <td>Transcriptomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/DanFaria/GOEnrichment</td>
      <td>Genome annotation</td>
      <td></td>
      <td>2018-12-28</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/goenrichment</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/goenrichment</td>
      <td>2.0.1</td>
      <td>goenrichment</td>
      <td>2.0.1</td>
      <td>Gene-set enrichment analysis</td>
      <td>Transcriptomics</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>5800</td>
      <td>6212</td>
      <td>826</td>
      <td>871</td>
      <td>7756</td>
      <td>7756</td>
      <td>1048</td>
      <td>1048</td>
      <td>1149</td>
      <td>1149</td>
      <td>215</td>
      <td>215</td>
      <td>14705</td>
      <td>15117</td>
      <td>2089</td>
      <td>2134</td>
    </tr>
    <tr>
      <td></td>
      <td>goseq</td>
      <td>goseq</td>
      <td>goseq does selection-unbiased testing for category enrichment amongst differentially expressed (DE) genes for RNA-seq data</td>
      <td>goseq</td>
      <td>goseq</td>
      <td></td>
      <td>GOseq</td>
      <td>Detect Gene Ontology and/or other user defined categories which are over/under represented in RNA-seq data.</td>
      <td>Gene functional annotation</td>
      <td>RNA-Seq</td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/goseq.html</td>
      <td>Statistics, RNA, Micro-array Analysis</td>
      <td>goseq</td>
      <td>2018-09-30</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/goseq</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/goseq</td>
      <td>1.50.0</td>
      <td>bioconductor-goseq</td>
      <td>1.54.0</td>
      <td>Gene functional annotation</td>
      <td>RNA-Seq</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>20207</td>
      <td>21960</td>
      <td>2807</td>
      <td>3139</td>
      <td>24488</td>
      <td>26198</td>
      <td>3066</td>
      <td>3321</td>
      <td>4155</td>
      <td>4155</td>
      <td>545</td>
      <td>545</td>
      <td>48850</td>
      <td>52313</td>
      <td>6418</td>
      <td>7005</td>
    </tr>
    <tr>
      <td></td>
      <td>gprofiler</td>
      <td>gprofiler_convert, gprofiler_gost, gprofiler_orth, gprofiler_random, gprofiler_snpense</td>
      <td>functional enrichment analysis of gene lists, convertion between various types of namespaces, translation gene identifiers between organisms and more</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://biit.cs.ut.ee/gprofiler</td>
      <td>Statistics, Web Services</td>
      <td>gprofiler</td>
      <td>2019-10-30</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gprofiler/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/gprofiler</td>
      <td>@TOOL_VERSION@+galaxy11</td>
      <td>r-gprofiler2</td>
      <td></td>
      <td></td>
      <td></td>
      <td>5</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>4001</td>
      <td>4001</td>
      <td>614</td>
      <td>614</td>
      <td>7882</td>
      <td>7882</td>
      <td>1238</td>
      <td>1238</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>11883</td>
      <td>11883</td>
      <td>1852</td>
      <td>1852</td>
    </tr>
    <tr>
      <td></td>
      <td>graphembed</td>
      <td>graphembed</td>
      <td>Compute a 2D embedding of a data matrix given supervised class information</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/fabriziocosta/GraphEmbed</td>
      <td>Statistics, Graphics</td>
      <td>graphembed</td>
      <td>2019-12-10</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/graphembed/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/graphembed</td>
      <td>2.4</td>
      <td>graph_embed</td>
      <td>2.4</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>78</td>
      <td>78</td>
      <td>6</td>
      <td>6</td>
      <td>120</td>
      <td>120</td>
      <td>43</td>
      <td>43</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>198</td>
      <td>198</td>
      <td>49</td>
      <td>49</td>
    </tr>
    <tr>
      <td></td>
      <td>graphlan</td>
      <td>graphlan, graphlan_annotate</td>
      <td>GraPhlAn is a software tool for producing high-quality circular representations of taxonomic and phylogenetic trees</td>
      <td>graphlan</td>
      <td>graphlan</td>
      <td></td>
      <td>GraPhlAn</td>
      <td>GraPhlAn is a software tool for producing high-quality circular representations of taxonomic and phylogenetic trees. GraPhlAn focuses on concise, integrative, informative, and publication-ready representations of phylogenetically- and taxonomically-driven investigation.</td>
      <td>Phylogenetic inference, Phylogenetic tree visualisation, Phylogenetic tree editing, Taxonomic classification</td>
      <td>Metagenomics, Phylogenetics, Phylogenomics, Cladistics</td>
      <td>To update</td>
      <td>https://github.com/biobakery/graphlan</td>
      <td>Metagenomics, Graphics, Phylogenetics</td>
      <td>graphlan</td>
      <td>2017-03-03</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/humann2/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/graphlan</td>
      <td></td>
      <td>graphlan</td>
      <td>1.1.3</td>
      <td>Phylogenetic inference, Phylogenetic tree visualisation, Phylogenetic tree editing, Taxonomic classification</td>
      <td>Metagenomics, Phylogenetics, Phylogenomics, Cladistics</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>9816</td>
      <td>10927</td>
      <td>1285</td>
      <td>1350</td>
      <td>1273</td>
      <td>1273</td>
      <td>368</td>
      <td>368</td>
      <td>1104</td>
      <td>1104</td>
      <td>242</td>
      <td>242</td>
      <td>12193</td>
      <td>13304</td>
      <td>1895</td>
      <td>1960</td>
    </tr>
    <tr>
      <td></td>
      <td>groot</td>
      <td>groot</td>
      <td>GROOT is a tool to type Antibiotic Resistance Genes (ARGs) in metagenomic samples</td>
      <td>groot</td>
      <td>groot</td>
      <td></td>
      <td>GROOT</td>
      <td>Indexed variation graphs for efficient and accurate resistome profiling.</td>
      <td>Antimicrobial resistance prediction</td>
      <td>Metagenomics, Microbiology, Genomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/will-rowe/groot</td>
      <td>Metagenomics</td>
      <td>groot</td>
      <td>2024-05-14</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/groot</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/groot</td>
      <td>1.1.2</td>
      <td>groot</td>
      <td>1.1.2</td>
      <td>Antimicrobial resistance prediction</td>
      <td>Metagenomics, Microbiology</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>gtdb_to_taxdump</td>
      <td>gtdb_to_taxdump</td>
      <td>Convert GTDB taxonomy to NCBI taxdump format</td>
      <td>gtdb_to_taxdump</td>
      <td>gtdb_to_taxdump</td>
      <td></td>
      <td>gtdb_to_taxdump</td>
      <td>Tool with multiple functions. Main functions are to create a DIAMOND database from the GTDB taxonomy data or create a NCBI taxdump format out of this data. This tool can also create a mapping between the taxonomy classification between GTDB and NCBI.</td>
      <td>Data handling, Mapping, Generation</td>
      <td>Computational biology</td>
      <td>Up-to-date</td>
      <td>https://github.com/nick-youngblut/gtdb_to_taxdump</td>
      <td>Metagenomics</td>
      <td>gtdb_to_taxdump</td>
      <td>2024-08-25</td>
      <td>iuc</td>
      <td>https://github.com/nick-youngblut/gtdb_to_taxdump</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/gtdb_to_taxdump</td>
      <td>0.1.9</td>
      <td>gtdb_to_taxdump</td>
      <td>0.1.9</td>
      <td>Data handling, Mapping, Generation</td>
      <td>Computational biology</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>gtdbtk</td>
      <td>gtdbtk_classify_wf</td>
      <td>GTDB-Tk is a software tool kit for assigning objective taxonomic classifications to bacterial and archaeal genomesbased on the Genome Database Taxonomy GTDB. It is designed to work with recent advances that allow hundreds orthousands of metagenome-assembled genomes (MAGs) to be obtained directly from environmental samples. It can alsobe applied to isolate and single-cell genomes.</td>
      <td>GTDB-Tk</td>
      <td>GTDB-Tk</td>
      <td></td>
      <td>GTDB-Tk</td>
      <td>a toolkit to classify genomes with the Genome Taxonomy Database.GTDB-Tk: a toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes.GTDB-Tk is a software toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes based on the Genome Database Taxonomy GTDB. It is designed to work with recent advances that allow hundreds or thousands of metagenome-assembled genomes (MAGs) to be obtained directly from environmental samples. It can also be applied to isolate and single-cell genomes. The GTDB-Tk is open source and released under the GNU General Public License (Version 3).</td>
      <td>Genome alignment, Taxonomic classification, Sequence assembly, Query and retrieval</td>
      <td>Metagenomics, Taxonomy, Phylogenetics, Database management, Proteins</td>
      <td>Up-to-date</td>
      <td>https://github.com/Ecogenomics/GTDBTk</td>
      <td>Metagenomics</td>
      <td>gtdbtk</td>
      <td>2022-12-13</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gtdbtk</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/gtdbtk</td>
      <td>2.4.0</td>
      <td>gtdbtk</td>
      <td>2.4.0</td>
      <td>Genome alignment, Taxonomic classification, Sequence assembly, Query and retrieval</td>
      <td>Metagenomics, Taxonomy, Phylogenetics, Database management, Proteins</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>476</td>
      <td>476</td>
      <td>87</td>
      <td>87</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>588</td>
      <td>588</td>
      <td>74</td>
      <td>74</td>
      <td>1064</td>
      <td>1064</td>
      <td>161</td>
      <td>161</td>
    </tr>
    <tr>
      <td></td>
      <td>gtfToBed12</td>
      <td>gtftobed12</td>
      <td>Convert GTF files to BED12 format</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td></td>
      <td>UCSC Genome Browser Utilities</td>
      <td>Utilities for handling sequences and assemblies from the UCSC Genome Browser project.</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>To update</td>
      <td>http://genome-source.cse.ucsc.edu/gitweb/?p=kent.git;a=blob;f=src/userApps/README</td>
      <td>Convert Formats</td>
      <td>gtftobed12</td>
      <td>2018-05-17</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gtfToBed12</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/gtfToBed12</td>
      <td>357</td>
      <td>ucsc-gtftogenepred</td>
      <td>469</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>8329</td>
      <td>9363</td>
      <td>3336</td>
      <td>3828</td>
      <td>8009</td>
      <td>8009</td>
      <td>3130</td>
      <td>3130</td>
      <td>1577</td>
      <td>1577</td>
      <td>344</td>
      <td>344</td>
      <td>17915</td>
      <td>18949</td>
      <td>6810</td>
      <td>7302</td>
    </tr>
    <tr>
      <td></td>
      <td>gubbins</td>
      <td>gubbins</td>
      <td>Gubbins - bacterial recombination detection</td>
      <td>gubbins</td>
      <td>gubbins</td>
      <td></td>
      <td>Gubbins</td>
      <td>Gubbins is a tool for rapid phylogenetic analysis of large samples of recombinant bacterial whole genome sequences.</td>
      <td>Genotyping, Phylogenetic inference, Ancestral reconstruction</td>
      <td>Phylogeny, Genotype and phenotype, Whole genome sequencing</td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>gubbins</td>
      <td>2017-06-22</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gubbins</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/gubbins</td>
      <td>3.2.1</td>
      <td>gubbins</td>
      <td>3.3.5</td>
      <td>Genotyping, Phylogenetic inference, Ancestral reconstruction</td>
      <td>Phylogeny, Genotype and phenotype, Whole genome sequencing</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>3690</td>
      <td>3811</td>
      <td>326</td>
      <td>336</td>
      <td>3323</td>
      <td>3323</td>
      <td>235</td>
      <td>235</td>
      <td>1895</td>
      <td>2429</td>
      <td>234</td>
      <td>258</td>
      <td>8908</td>
      <td>9563</td>
      <td>795</td>
      <td>829</td>
    </tr>
    <tr>
      <td></td>
      <td>gvcftools</td>
      <td>gvcftools_extract_variants</td>
      <td>gvcftools is a set of utilities to help create and analyze Genome VCF (gVCF) files.gVCF are VCF 4.1 files which follow a set of conventions for representing all sitesin the genome, further described at https://sites.google.com/site/gvcftools/home/about-gvcf.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/sequencing/gvcftools</td>
      <td>Variant Analysis</td>
      <td></td>
      <td>2016-12-29</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gvcftools</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/gvcftools</td>
      <td>0.1</td>
      <td>gvcftools</td>
      <td>0.17.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>gwastools</td>
      <td>gwastools_manhattan_plot</td>
      <td></td>
      <td>gwastools</td>
      <td>gwastools</td>
      <td></td>
      <td>GWASTools</td>
      <td>Classes for storing very large GWAS data sets and annotation, and functions for GWAS data cleaning and analysis.</td>
      <td>Deposition, Analysis, Annotation</td>
      <td>GWAS study</td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/GWASTools.html</td>
      <td>Visualization, Variant Analysis</td>
      <td></td>
      <td>2019-10-07</td>
      <td>iuc</td>
      <td></td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/gwastools</td>
      <td>0.1.0</td>
      <td>bioconductor-gwastools</td>
      <td>1.48.0</td>
      <td>Deposition, Analysis, Annotation</td>
      <td>GWAS study</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>hamronization</td>
      <td>hamronize_summarize, hamronize_tool</td>
      <td>Convert AMR gene detection tool output to hAMRonization specification format.</td>
      <td>hamronization</td>
      <td>hamronization</td>
      <td></td>
      <td>hAMRonization</td>
      <td>Parse multiple Antimicrobial Resistance Analysis Reports into a common data structure</td>
      <td>Data handling, Antimicrobial resistance prediction, Parsing</td>
      <td>Public health and epidemiology, Microbiology, Bioinformatics</td>
      <td>Up-to-date</td>
      <td>https://github.com/pha4ge/hAMRonization</td>
      <td>Sequence Analysis</td>
      <td>hamronization</td>
      <td>2021-02-02</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hamronization</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hamronization</td>
      <td>1.1.4</td>
      <td>hamronization</td>
      <td>1.1.4</td>
      <td>Antimicrobial resistance prediction, Parsing</td>
      <td>Public health and epidemiology, Microbiology, Bioinformatics</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>7303</td>
      <td>7303</td>
      <td>222</td>
      <td>222</td>
      <td>20</td>
      <td>20</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7323</td>
      <td>7323</td>
      <td>224</td>
      <td>224</td>
    </tr>
    <tr>
      <td></td>
      <td>hansel</td>
      <td>bio_hansel</td>
      <td>Heidelberg and Enteritidis SNP Elucidation</td>
      <td>Biohansel</td>
      <td>Biohansel</td>
      <td></td>
      <td>BioHansel</td>
      <td>BioHansel is a tool for performing high-resolution genotyping of bacterial isolates by identifying phylogenetically informative single nucleotide polymorphisms (SNPs), also known as canonical SNPs, in whole genome sequencing (WGS) data. The application uses a fast k-mer matching algorithm to map pathogen WGS data to canonical SNPs contained in hierarchically structured schemas and assigns genotypes based on the detected SNP profile.</td>
      <td>Genotyping, SNP detection, Genome assembly</td>
      <td>Whole genome sequencing, DNA polymorphism, Genotype and phenotype, Infectious disease, Agricultural science</td>
      <td>Up-to-date</td>
      <td>https://github.com/phac-nml/bio_hansel</td>
      <td>Sequence Analysis</td>
      <td>bio_hansel</td>
      <td>2017-09-29</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hansel</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hansel</td>
      <td>2.6.1</td>
      <td>bio_hansel</td>
      <td>2.6.1</td>
      <td>Genotyping, SNP detection, Genome assembly</td>
      <td>Whole genome sequencing, DNA polymorphism, Genotype and phenotype, Infectious disease, Agricultural science</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>349</td>
      <td>359</td>
      <td>32</td>
      <td>33</td>
      <td>364</td>
      <td>364</td>
      <td>53</td>
      <td>53</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>713</td>
      <td>723</td>
      <td>85</td>
      <td>86</td>
    </tr>
    <tr>
      <td></td>
      <td>hapcut2</td>
      <td>hapcut2</td>
      <td>Robust and accurate haplotype assembly for diverse sequencing technologies</td>
      <td>hapcut2</td>
      <td>hapcut2</td>
      <td></td>
      <td>HapCUT2</td>
      <td>HapCUT2 is a maximum-likelihood-based tool for assembling haplotypes from DNA sequence reads, designed to "just work" with excellent speed and accuracy across a range of long- and short-read sequencing technologies.The output is in Haplotype block format described here: https://github.com/vibansal/HapCUT2/blob/master/outputformat.md</td>
      <td>Haplotype mapping, Variant classification</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/vibansal/HapCUT2</td>
      <td>Assembly</td>
      <td>hapcut2</td>
      <td>2022-10-19</td>
      <td>galaxy-australia</td>
      <td>https://github.com/galaxyproject/tools-iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hapcut2</td>
      <td>1.3.4</td>
      <td>hapcut2</td>
      <td>1.3.4</td>
      <td>Haplotype mapping, Variant classification</td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>103</td>
      <td>103</td>
      <td>4</td>
      <td>4</td>
      <td>103</td>
      <td>103</td>
      <td>4</td>
      <td>4</td>
    </tr>
    <tr>
      <td></td>
      <td>hapog</td>
      <td>hapog</td>
      <td>Hapo-G - Haplotype-Aware Polishing of Genomes</td>
      <td>hapog</td>
      <td>hapog</td>
      <td></td>
      <td>Hapo-G</td>
      <td>Hapo-G is a tool that aims to improve the quality of genome assemblies by polishing the consensus with accurate reads. It capable of incorporating phasing information from high-quality reads (short or long-reads) to polish genome assemblies and in particular assemblies of diploid and heterozygous genomes.</td>
      <td>Genome assembly, Optimisation and refinement</td>
      <td>Sequence assembly, Genomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/institut-de-genomique/HAPO-G</td>
      <td>Assembly</td>
      <td>hapog</td>
      <td>2022-09-16</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hapog</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hapog</td>
      <td>1.3.8</td>
      <td>hapog</td>
      <td>1.3.8</td>
      <td>Genome assembly, Optimisation and refinement</td>
      <td>Sequence assembly, Genomics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>502</td>
      <td>502</td>
      <td>66</td>
      <td>66</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>502</td>
      <td>502</td>
      <td>66</td>
      <td>66</td>
    </tr>
    <tr>
      <td></td>
      <td>happy</td>
      <td>som.py</td>
      <td>A tool to perform comparisons only based on chromosome, position, and allele identity for comparison of somatic callsets.</td>
      <td>hap.py</td>
      <td>hap.py</td>
      <td></td>
      <td>hap.py</td>
      <td>This is a set of programs based on htslib to benchmark variant calls against gold standard truth datasets.To compare a VCF against a gold standard dataset, use the following commmand line to perform genotype-level haplotype comparison.</td>
      <td>Variant calling, Sequence analysis, Genotyping</td>
      <td>Genomics, DNA polymorphism</td>
      <td>To update</td>
      <td>https://github.com/Illumina/hap.py</td>
      <td>Variant Analysis</td>
      <td>happy</td>
      <td>2022-01-25</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/happy</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/happy</td>
      <td>0.3.14</td>
      <td>hap.py</td>
      <td>0.3.15</td>
      <td>Variant calling, Sequence analysis, Genotyping</td>
      <td>Genomics, DNA polymorphism</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>heatmap2</td>
      <td>ggplot2_heatmap2</td>
      <td>heatmap.2 function from the R gplots package</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/cran/gplots</td>
      <td>Visualization</td>
      <td>ggplot2_heatmap2</td>
      <td>2019-02-10</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/heatmap2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/heatmap2</td>
      <td>3.2.0</td>
      <td>r-gplots</td>
      <td>2.17.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>46700</td>
      <td>50808</td>
      <td>4645</td>
      <td>5114</td>
      <td>64214</td>
      <td>70429</td>
      <td>5799</td>
      <td>6830</td>
      <td>7783</td>
      <td>7912</td>
      <td>704</td>
      <td>724</td>
      <td>118697</td>
      <td>129149</td>
      <td>11148</td>
      <td>12668</td>
    </tr>
    <tr>
      <td></td>
      <td>heinz</td>
      <td>heinz_bum, heinz, heinz_scoring, heinz_visualization</td>
      <td>An algorithm for identification of the optimal scoring subnetwork.</td>
      <td>heinz</td>
      <td>heinz, bionet</td>
      <td></td>
      <td>Heinz</td>
      <td>Tool for single-species active module discovery.</td>
      <td>Pathway or network analysis</td>
      <td>Genetics, Gene expression, Molecular interactions, pathways and networks</td>
      <td>To update</td>
      <td>https://github.com/ls-cwi/heinz</td>
      <td>Transcriptomics, Visualization, Statistics</td>
      <td>heinz</td>
      <td>2018-06-11</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/heinz</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/heinz</td>
      <td>1.0</td>
      <td>bioconductor-bionet</td>
      <td>1.62.0</td>
      <td>Pathway or network analysis</td>
      <td>Genetics, Gene expression, Molecular interactions, pathways and networks</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>1281</td>
      <td>1434</td>
      <td>367</td>
      <td>442</td>
      <td>1559</td>
      <td>1559</td>
      <td>636</td>
      <td>636</td>
      <td>678</td>
      <td>678</td>
      <td>52</td>
      <td>52</td>
      <td>3518</td>
      <td>3671</td>
      <td>1055</td>
      <td>1130</td>
    </tr>
    <tr>
      <td></td>
      <td>hgvsparser</td>
      <td>hgvsparser</td>
      <td>Parsing and building variant descriptor strings compliant with the HGVS standard</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/VariantEffect/hgvsParseR/tree/master</td>
      <td>Variant Analysis</td>
      <td>hgvsparser</td>
      <td>2024-06-07</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hgvsparser/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hgvsparser</td>
      <td>0.1.0</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6346</td>
      <td>6346</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6346</td>
      <td>6346</td>
      <td>2</td>
      <td>2</td>
    </tr>
    <tr>
      <td></td>
      <td>hicexplorer</td>
      <td>hicexplorer_chicaggregatestatistic, hicexplorer_chicdifferentialtest, hicexplorer_chicexportdata, hicexplorer_chicplotviewpoint, hicexplorer_chicqualitycontrol, hicexplorer_chicsignificantinteractions, hicexplorer_chicviewpoint, hicexplorer_chicviewpointbackgroundmodel, hicexplorer_hicadjustmatrix, hicexplorer_hicaggregatecontacts, hicexplorer_hicaverageregions, hicexplorer_hicbuildmatrix, hicexplorer_hiccomparematrices, hicexplorer_hiccompartmentspolarization, hicexplorer_hicconvertformat, hicexplorer_hiccorrectmatrix, hicexplorer_hiccorrelate, hicexplorer_hicdetectloops, hicexplorer_hicdifferentialtad, hicexplorer_hicfindrestrictionsites, hicexplorer_hicfindtads, hicexplorer_hichyperoptDetectLoops, hicexplorer_hicinfo, hicexplorer_hicinterintratad, hicexplorer_hicmergedomains, hicexplorer_hicmergeloops, hicexplorer_hicmergematrixbins, hicexplorer_hicnormalize, hicexplorer_hicpca, hicexplorer_hicplotaverageregions, hicexplorer_hicplotdistvscounts, hicexplorer_hicplotmatrix, hicexplorer_hicplotsvl, hicexplorer_hicplotviewpoint, hicexplorer_hicquickqc, hicexplorer_hicsummatrices, hicexplorer_hictransform, hicexplorer_hicvalidatelocations</td>
      <td>HiCExplorer: Set of programs to process, analyze and visualize Hi-C data.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/deeptools/HiCExplorer</td>
      <td>Sequence Analysis, Visualization</td>
      <td>hicexplorer</td>
      <td>2020-03-12</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hicexplorer</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hicexplorer</td>
      <td>3.7.5</td>
      <td>hicexplorer</td>
      <td>3.7.5</td>
      <td></td>
      <td></td>
      <td>38</td>
      <td>38</td>
      <td>38</td>
      <td>4</td>
      <td>38</td>
      <td>38</td>
      <td>38</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>13</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>33</td>
      <td>36</td>
      <td>33</td>
      <td>0</td>
      <td>44449</td>
      <td>68602</td>
      <td>3236</td>
      <td>4177</td>
      <td>46</td>
      <td>46</td>
      <td>21</td>
      <td>21</td>
      <td>1328</td>
      <td>1328</td>
      <td>67</td>
      <td>67</td>
      <td>45823</td>
      <td>69976</td>
      <td>3324</td>
      <td>4265</td>
    </tr>
    <tr>
      <td></td>
      <td>hicstuff</td>
      <td>hicstuff_pipeline</td>
      <td>A toolkit to generate and manipulate Hi-C matrices</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/koszullab/hicstuff</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>2022-11-25</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hicstuff</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hicstuff</td>
      <td>3.2.3</td>
      <td>hicstuff</td>
      <td>3.2.3</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>hifiasm</td>
      <td>hifiasm</td>
      <td>A fast haplotype-resolved de novo assembler</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/chhylp123/hifiasm</td>
      <td>Assembly</td>
      <td>hifiasm</td>
      <td>2024-07-11</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/hifiasm</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hifiasm</td>
      <td>0.19.9</td>
      <td>hifiasm</td>
      <td>0.20.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1819</td>
      <td>1819</td>
      <td>539</td>
      <td>539</td>
      <td>1334</td>
      <td>1334</td>
      <td>389</td>
      <td>389</td>
      <td>2472</td>
      <td>2472</td>
      <td>137</td>
      <td>137</td>
      <td>5625</td>
      <td>5625</td>
      <td>1065</td>
      <td>1065</td>
    </tr>
    <tr>
      <td></td>
      <td>hifiasm_meta</td>
      <td>hifiasm_meta</td>
      <td>A hifiasm fork for metagenome assembly using Hifi reads.</td>
      <td>hifiasm-meta</td>
      <td>hifiasm-meta</td>
      <td></td>
      <td>Hifiasm-meta</td>
      <td>Hifiasm_meta - de novo metagenome assembler, based on hifiasm, a haplotype-resolved de novo assembler for PacBio Hifi reads.</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly, Metagenomics</td>
      <td>To update</td>
      <td>https://github.com/xfengnefx/hifiasm-meta</td>
      <td>Metagenomics</td>
      <td>hifiasm_meta</td>
      <td>2023-01-18</td>
      <td>galaxy-australia</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hifiasm_meta</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hifiasm_meta</td>
      <td>0.3.1</td>
      <td>hifiasm_meta</td>
      <td>hamtv0.3.1</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly, Metagenomics</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>156</td>
      <td>156</td>
      <td>20</td>
      <td>20</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>220</td>
      <td>220</td>
      <td>11</td>
      <td>11</td>
      <td>376</td>
      <td>376</td>
      <td>31</td>
      <td>31</td>
    </tr>
    <tr>
      <td></td>
      <td>hisat2</td>
      <td>hisat2</td>
      <td>HISAT2 is a fast and sensitive spliced alignment program.</td>
      <td>hisat2</td>
      <td>hisat2</td>
      <td></td>
      <td>HISAT2</td>
      <td>Alignment program for mapping next-generation sequencing reads (both DNA and RNA) to a population of human genomes (as well as to a single reference genome).</td>
      <td>Sequence alignment</td>
      <td>RNA-seq</td>
      <td>Up-to-date</td>
      <td>http://ccb.jhu.edu/software/hisat2/</td>
      <td>Assembly</td>
      <td>hisat2</td>
      <td>2020-11-24</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hisat2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hisat2</td>
      <td>2.2.1</td>
      <td>hisat2</td>
      <td>2.2.1</td>
      <td>Sequence alignment</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>337151</td>
      <td>353499</td>
      <td>10046</td>
      <td>10719</td>
      <td>699868</td>
      <td>896164</td>
      <td>26656</td>
      <td>34018</td>
      <td>87016</td>
      <td>94859</td>
      <td>5302</td>
      <td>5695</td>
      <td>1124035</td>
      <td>1344522</td>
      <td>42004</td>
      <td>50432</td>
    </tr>
    <tr>
      <td></td>
      <td>hivclustering</td>
      <td>hivclustering</td>
      <td>Infers transmission networks from pairwise distances inferred by tn93</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://pypi.org/project/hivclustering/</td>
      <td>Next Gen Mappers</td>
      <td>hivclustering</td>
      <td>2018-06-06</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hivclustering/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hivclustering</td>
      <td>1.3.1</td>
      <td>python-hivclustering</td>
      <td>1.8.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>hmmer3</td>
      <td>hmmer_alimask, hmmer_hmmalign, hmmer_hmmbuild, hmmer_hmmconvert, hmmer_hmmemit, hmmer_hmmfetch, hmmer_hmmscan, hmmer_hmmsearch, hmmer_jackhmmer, hmmer_nhmmer, hmmer_nhmmscan, hmmer_phmmer</td>
      <td>HMMER is used for searching sequence databases for homologs of proteinsequences, and for making protein sequence alignments. It implementsmethods using probabilistic models called profile hidden Markov models(profile HMMs).</td>
      <td>hmmer3</td>
      <td>hmmer3</td>
      <td></td>
      <td>HMMER3</td>
      <td>This tool is used for searching sequence databases for homologs of protein sequences, and for making protein sequence alignments. It implements methods using probabilistic models called profile hidden Markov models. The new HMMER3 project, HMMER is now as fast as BLAST for protein search.</td>
      <td>Formatting, Multiple sequence alignment, Sequence profile generation, Format validation, Conversion, Sequence generation, Data retrieval, Statistical calculation, Database search, Formatting, Database search, Database search, Probabilistic sequence generation, Statistical calculation, Statistical calculation, Sequence database search, Formatting, Sequence database search, Database search, Sequence database search</td>
      <td>Sequence analysis, Sequence sites, features and motifs, Gene and protein families</td>
      <td>Up-to-date</td>
      <td>http://hmmer.org/</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>2016-11-14</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hmmer3</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hmmer3</td>
      <td>3.4</td>
      <td>hmmer</td>
      <td>3.4</td>
      <td>Formatting, Multiple sequence alignment, Sequence profile generation, Format validation, Conversion, Data retrieval, Statistical calculation, Formatting, Probabilistic sequence generation, Statistical calculation, Statistical calculation, Sequence database search, Formatting, Sequence database search, Sequence database search</td>
      <td>Sequence analysis, Sequence sites, features and motifs, Gene and protein families</td>
      <td>0</td>
      <td>12</td>
      <td>12</td>
      <td>12</td>
      <td>0</td>
      <td>12</td>
      <td>12</td>
      <td>12</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>12</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>12</td>
      <td>12</td>
      <td>12</td>
      <td>0</td>
      <td>35919</td>
      <td>36952</td>
      <td>1283</td>
      <td>1321</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3023</td>
      <td>3023</td>
      <td>180</td>
      <td>180</td>
      <td>38942</td>
      <td>39975</td>
      <td>1463</td>
      <td>1501</td>
    </tr>
    <tr>
      <td></td>
      <td>homer</td>
      <td>homer_annotatePeaks, homer_findMotifs, homer_findMotifsGenome, homer_gtf_to_annotations, homer_scanMotifGenomeWide</td>
      <td>HOMER (Hypergeometric Optimization of Motif EnRichment) is a suite of tools for Motif Discovery and next-gen sequencing analysis.</td>
      <td>homer</td>
      <td>homer</td>
      <td></td>
      <td>homer</td>
      <td>HOMER contains a novel motif discovery algorithm that was designed for regulatory element analysis in genomics applications (DNA only, no protein).  It is a differential motif discovery algorithm, which means that it takes two sets of sequences and tries to identify the regulatory elements that are specifically enriched in on set relative to the other.  It uses ZOOPS scoring (zero or one occurrence per sequence) coupled with the hypergeometric enrichment calculations (or binomial) to determine motif enrichment.  HOMER also tries its best to account for sequenced bias in the dataset.  It was designed with ChIP-Seq and promoter analysis in mind, but can be applied to pretty much any nucleic acids motif finding problem.</td>
      <td>Sequence motif discovery</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://homer.ucsd.edu/homer/index.html</td>
      <td>Sequence Analysis</td>
      <td>data_manager_homer_preparse</td>
      <td>2021-08-08</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/homer</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/homer</td>
      <td>4.11</td>
      <td>homer</td>
      <td>4.11</td>
      <td>Sequence motif discovery</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>8531</td>
      <td>8531</td>
      <td>633</td>
      <td>633</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8531</td>
      <td>8531</td>
      <td>633</td>
      <td>633</td>
    </tr>
    <tr>
      <td></td>
      <td>htseq_count</td>
      <td>htseq_count</td>
      <td>Count aligned reads (SAM/BAM) that overlap genomic features (GFF)</td>
      <td>htseq</td>
      <td>htseq</td>
      <td></td>
      <td>HTSeq</td>
      <td>Python framework to process and analyse high-throughput sequencing (HTS) data</td>
      <td>Nucleic acid sequence analysis</td>
      <td>Sequence analysis</td>
      <td>Up-to-date</td>
      <td>https://readthedocs.org/projects/htseq/</td>
      <td>Genomic Interval Operations, SAM, Sequence Analysis, RNA</td>
      <td>htseq_count</td>
      <td>2017-01-06</td>
      <td>lparsons</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/htseq_count</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/htseq_count</td>
      <td>2.0.5</td>
      <td>htseq</td>
      <td>2.0.5</td>
      <td>Nucleic acid sequence analysis</td>
      <td>Sequence analysis</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>144907</td>
      <td>168824</td>
      <td>3703</td>
      <td>4366</td>
      <td>213407</td>
      <td>346792</td>
      <td>10798</td>
      <td>16485</td>
      <td>27209</td>
      <td>33124</td>
      <td>2613</td>
      <td>2854</td>
      <td>385523</td>
      <td>548740</td>
      <td>17114</td>
      <td>23705</td>
    </tr>
    <tr>
      <td></td>
      <td>humann</td>
      <td>humann, humann_barplot, humann_join_tables, humann_reduce_table, humann_regroup_table, humann_rename_table, humann_renorm_table, humann_split_stratified_table, humann_split_table, humann_unpack_pathways</td>
      <td>HUMAnN for functionally profiling metagenomes and metatranscriptomes at species-level resolution</td>
      <td>humann</td>
      <td>humann</td>
      <td></td>
      <td>humann</td>
      <td>HUMAnN is a pipeline for efficiently and accurately profiling the presence/absence and abundance of microbial pathways in a community from metagenomic or metatranscriptomic sequencing data (typically millions of short DNA/RNA reads). This process, referred to as functional profiling, aims to describe the metabolic potential of a microbial community and its members. More generally, functional profiling answers the question What are the microbes in my community-of-interest doing (or are capable of doing)?</td>
      <td>Species frequency estimation, Taxonomic classification, Phylogenetic analysis</td>
      <td>Metagenomics, Phylogenomics</td>
      <td>Up-to-date</td>
      <td>http://huttenhower.sph.harvard.edu/humann</td>
      <td>Metagenomics</td>
      <td>humann</td>
      <td>2021-05-12</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/humann</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/humann</td>
      <td>3.9</td>
      <td>humann</td>
      <td>3.9</td>
      <td>Species frequency estimation, Taxonomic classification, Phylogenetic analysis</td>
      <td>Metagenomics, Phylogenomics</td>
      <td>6</td>
      <td>10</td>
      <td>10</td>
      <td>10</td>
      <td>6</td>
      <td>10</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>26659</td>
      <td>26659</td>
      <td>2067</td>
      <td>2067</td>
      <td>865</td>
      <td>865</td>
      <td>174</td>
      <td>174</td>
      <td>3238</td>
      <td>3238</td>
      <td>351</td>
      <td>351</td>
      <td>30762</td>
      <td>30762</td>
      <td>2592</td>
      <td>2592</td>
    </tr>
    <tr>
      <td></td>
      <td>hybpiper</td>
      <td>hybpiper</td>
      <td>Analyse targeted sequence capture data</td>
      <td>HybPiper</td>
      <td>HybPiper</td>
      <td></td>
      <td>HybPiper</td>
      <td>Paralogs and off-target sequences improve phylogenetic resolution in a densely-sampled study of the breadfruit genus (Artocarpus, Moraceae).Recovering genes from targeted sequence capture data.Current version: 1.3.1 (August 2018).-- Read our article in Applications in Plant Sciences (Open Access).HybPiper was designed for targeted sequence capture, in which DNA sequencing libraries are enriched for gene regions of interest, especially for phylogenetics. HybPiper is a suite of Python scripts that wrap and connect bioinformatics tools in order to extract target sequences from high-throughput DNA sequencing reads.</td>
      <td>Sequence trimming, Sequence assembly, Read mapping</td>
      <td>Phylogenetics, Plant biology, Gene transcripts, Sequence assembly, Phylogenomics</td>
      <td>To update</td>
      <td>https://github.com/mossmatters/HybPiper</td>
      <td>Sequence Analysis, Phylogenetics</td>
      <td>hybpiper</td>
      <td>2023-09-04</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hybpiper</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hybpiper</td>
      <td>2.1.6</td>
      <td>hybpiper</td>
      <td>2.3.0</td>
      <td>Sequence trimming, Sequence assembly, Read mapping</td>
      <td>Phylogenetics, Plant biology, Gene transcripts, Sequence assembly, Phylogenomics</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1571</td>
      <td>1571</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>144</td>
      <td>144</td>
      <td>4</td>
      <td>4</td>
      <td>1715</td>
      <td>1715</td>
      <td>7</td>
      <td>7</td>
    </tr>
    <tr>
      <td></td>
      <td>hyphy</td>
      <td>hyphy_absrel, hyphy_annotate, hyphy_bgm, hyphy_busted, hyphy_cfel, hyphy_conv, hyphy_fade, hyphy_fel, hyphy_fubar, hyphy_gard, hyphy_meme, hyphy_prime, hyphy_relax, hyphy_slac, hyphy_sm19, hyphy_strike_ambigs, hyphy_summary</td>
      <td>Hypothesis Testing using Phylogenies</td>
      <td>HyPhy</td>
      <td>HyPhy</td>
      <td></td>
      <td>HyPhy</td>
      <td>Software package for the analysis of genetic sequences using techniques in phylogenetics, molecular evolution, and machine learning.</td>
      <td>Statistical calculation</td>
      <td>Phylogeny, Small molecules, Molecular interactions, pathways and networks</td>
      <td>To update</td>
      <td>http://www.hyphy.org</td>
      <td>Phylogenetics</td>
      <td></td>
      <td>2021-04-19</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hyphy/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hyphy</td>
      <td>2.5.47</td>
      <td>hyphy</td>
      <td>2.5.62</td>
      <td>Statistical calculation</td>
      <td>Phylogeny, Small molecules, Molecular interactions, pathways and networks</td>
      <td>17</td>
      <td>2</td>
      <td>17</td>
      <td>2</td>
      <td>17</td>
      <td>2</td>
      <td>17</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>12</td>
      <td>17</td>
      <td>12</td>
      <td>0</td>
      <td>11429</td>
      <td>11429</td>
      <td>428</td>
      <td>428</td>
      <td>7751</td>
      <td>7751</td>
      <td>676</td>
      <td>676</td>
      <td>715</td>
      <td>715</td>
      <td>58</td>
      <td>58</td>
      <td>19895</td>
      <td>19895</td>
      <td>1162</td>
      <td>1162</td>
    </tr>
    <tr>
      <td></td>
      <td>hypo</td>
      <td>hypo</td>
      <td>Super Fast &amp; Accurate Polisher for Long Read Genome Assemblies</td>
      <td>HyPo</td>
      <td>HyPo</td>
      <td></td>
      <td>HyPo</td>
      <td>HyPo, a Hybrid Polisher, utilizes short as well as long reads within a single run to polish a long reads assembly of small and large genomes.</td>
      <td>Optimisation and refinement, Genome assembly</td>
      <td>Sequence assembly, Genomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/kensung-lab/hypo</td>
      <td>Assembly</td>
      <td>hypo</td>
      <td>2021-11-15</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hypo</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hypo</td>
      <td>1.0.3</td>
      <td>hypo</td>
      <td>1.0.3</td>
      <td>Optimisation and refinement, Genome assembly</td>
      <td>Sequence assembly, Genomics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>473</td>
      <td>473</td>
      <td>93</td>
      <td>93</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>473</td>
      <td>473</td>
      <td>93</td>
      <td>93</td>
    </tr>
    <tr>
      <td></td>
      <td>icescreen</td>
      <td>icescreen</td>
      <td>ICEscreen identifies Integrative Conjugative Elements (ICEs) and Integrative Mobilizable Elements (IMEs) in Bacillota genomes.</td>
      <td>icescreen</td>
      <td>icescreen</td>
      <td></td>
      <td>ICEscreen</td>
      <td>A tool to detect Firmicute ICEs and IMEs, isolated or enclosed in composite structures.</td>
      <td>Database search, Protein feature detection</td>
      <td>Mobile genetic elements, Sequence sites, features and motifs, Genomics, Molecular interactions, pathways and networks, Structural variation</td>
      <td>To update</td>
      <td>https://icescreen.migale.inrae.fr/</td>
      <td>Genome annotation</td>
      <td>icescreen</td>
      <td>2022-01-04</td>
      <td>iuc</td>
      <td>https://forgemia.inra.fr/ices_imes_analysis/icescreen</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/icescreen</td>
      <td>1.3.1</td>
      <td>icescreen</td>
      <td>1.3.2</td>
      <td>Database search, Protein feature detection</td>
      <td>Mobile genetic elements, Sequence sites, features and motifs, Genomics, Molecular interactions, pathways and networks, Structural variation</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>idba_ud</td>
      <td>idba_hybrid, idba_tran, idba_ud</td>
      <td>Wrappers for the idba assembler variants.</td>
      <td>idba</td>
      <td>idba</td>
      <td></td>
      <td>IDBA</td>
      <td>A short read assembler based on iterative De Bruijn graph. It is developed under 64-bit Linux, but should be suitable for all unix-like system.</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly</td>
      <td>To update</td>
      <td>https://i.cs.hku.hk/~alse/hkubrg/projects/index.html</td>
      <td>Assembly</td>
      <td>idba</td>
      <td>2018-04-26</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/idba_ud</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/idba_ud</td>
      <td></td>
      <td>idba</td>
      <td>1.1.3</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly</td>
      <td>3</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>1200</td>
      <td>1220</td>
      <td>187</td>
      <td>195</td>
      <td>1588</td>
      <td>1588</td>
      <td>267</td>
      <td>267</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2788</td>
      <td>2808</td>
      <td>454</td>
      <td>462</td>
    </tr>
    <tr>
      <td></td>
      <td>idr</td>
      <td>idr</td>
      <td>Galaxy wrappers for the IDR package from Nathan Boleu</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/nboley/idr</td>
      <td>Sequence Analysis</td>
      <td>idr</td>
      <td>2017-08-21</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/idr/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/idr</td>
      <td>2.0.3</td>
      <td>idr</td>
      <td>2.0.4.2</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2953</td>
      <td>3027</td>
      <td>109</td>
      <td>114</td>
      <td>1710</td>
      <td>1710</td>
      <td>199</td>
      <td>199</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4663</td>
      <td>4737</td>
      <td>308</td>
      <td>313</td>
    </tr>
    <tr>
      <td></td>
      <td>idr_download</td>
      <td>idr_download_by_ids</td>
      <td>Image Data Resource downloading tool</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://idr.openmicroscopy.org</td>
      <td>Data Source</td>
      <td>idr_download_by_ids</td>
      <td>2020-05-15</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/idr_download</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/idr_download</td>
      <td>0.45</td>
      <td>omero-py</td>
      <td>5.11.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>404</td>
      <td>404</td>
      <td>37</td>
      <td>37</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>24</td>
      <td>24</td>
      <td>1</td>
      <td>1</td>
      <td>428</td>
      <td>428</td>
      <td>38</td>
      <td>38</td>
    </tr>
    <tr>
      <td></td>
      <td>iedb_api</td>
      <td>iedb_api</td>
      <td>Get epitope binding predictions from IEDB-API</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://tools.immuneepitope.org/main/tools-api/</td>
      <td>Data Source, Sequence Analysis</td>
      <td>iedb_api</td>
      <td>2020-02-26</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/iedb_api</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/iedb_api</td>
      <td>2.15.2</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1539</td>
      <td>1539</td>
      <td>35</td>
      <td>35</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1539</td>
      <td>1539</td>
      <td>35</td>
      <td>35</td>
    </tr>
    <tr>
      <td></td>
      <td>infercnv</td>
      <td>infercnv</td>
      <td>Infer Copy Number Variation from Single-Cell RNA-Seq Data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/broadinstitute/infercnv</td>
      <td>Transcriptomics, Variant Analysis</td>
      <td>infercnv</td>
      <td>2024-07-23</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/infercnv</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/infercnv</td>
      <td>1.20.0</td>
      <td>bioconductor-infercnv</td>
      <td>1.20.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>10</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>10</td>
      <td>2</td>
      <td>2</td>
    </tr>
    <tr>
      <td></td>
      <td>instrain</td>
      <td>instrain_compare, instrain_profile</td>
      <td>InStrain is a tool for analysis of co-occurring genome populations from metagenomes</td>
      <td>instrain</td>
      <td>instrain</td>
      <td></td>
      <td>InStrain</td>
      <td>InStrain is a tool for analysis of co-occurring genome populations from metagenomes that allows highly accurate genome comparisons, analysis of coverage, microdiversity, and linkage, and sensitive SNP detection with gene localization and synonymous non-synonymous identification</td>
      <td>SNP detection, Genome comparison</td>
      <td>Mapping, Metagenomics</td>
      <td>To update</td>
      <td>https://instrain.readthedocs.io/en/latest/#</td>
      <td>Metagenomics</td>
      <td>instrain</td>
      <td>2021-08-11</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/instrain</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/instrain</td>
      <td>1.5.3</td>
      <td>instrain</td>
      <td>1.9.0</td>
      <td>SNP detection, Genome comparison</td>
      <td>Mapping, Metagenomics</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>102</td>
      <td>102</td>
      <td>16</td>
      <td>16</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>102</td>
      <td>102</td>
      <td>16</td>
      <td>16</td>
    </tr>
    <tr>
      <td></td>
      <td>integron_finder</td>
      <td>integron_finder</td>
      <td>"IntegronFinder identify integrons with high accuracy and sensitivity.It searches for attC sites using covariance models, for integron-integrases using HMM profiles, and for other features (promoters, attI site) using pattern matching"</td>
      <td>integron_finder</td>
      <td>integron_finder</td>
      <td></td>
      <td>Integron Finder</td>
      <td>A tool to detect Integron in DNA sequences.</td>
      <td>Nucleic acid feature detection, Sequence motif recognition, Protein feature detection, Genome annotation</td>
      <td>Functional genomics, Mobile genetic elements, Molecular biology, Sequence analysis</td>
      <td>Up-to-date</td>
      <td>https://github.com/gem-pasteur/Integron_Finder</td>
      <td>Sequence Analysis</td>
      <td>integronfinder</td>
      <td>2022-09-22</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/integron_finder</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/integron_finder</td>
      <td>2.0.5</td>
      <td>integron_finder</td>
      <td>2.0.5</td>
      <td>Nucleic acid feature detection, Sequence motif recognition, Protein feature detection, Genome annotation</td>
      <td>Functional genomics, Mobile genetic elements, Molecular biology, Sequence analysis</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>55149</td>
      <td>55149</td>
      <td>161</td>
      <td>161</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6085</td>
      <td>6085</td>
      <td>73</td>
      <td>73</td>
      <td>61234</td>
      <td>61234</td>
      <td>234</td>
      <td>234</td>
    </tr>
    <tr>
      <td></td>
      <td>intermine_galaxy_exchange</td>
      <td>galaxy_intermine_exchange</td>
      <td>InterMine Exporter</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/intermine</td>
      <td>Convert Formats</td>
      <td>intermine_galaxy_exchange</td>
      <td>2018-07-01</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/intermine_galaxy_exchange</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/intermine_galaxy_exchange</td>
      <td>0.0.1</td>
      <td>coreutils</td>
      <td>8.25</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>37</td>
      <td>46</td>
      <td>6</td>
      <td>12</td>
      <td>72</td>
      <td>87</td>
      <td>13</td>
      <td>15</td>
      <td>108</td>
      <td>108</td>
      <td>1</td>
      <td>1</td>
      <td>217</td>
      <td>241</td>
      <td>20</td>
      <td>28</td>
    </tr>
    <tr>
      <td></td>
      <td>interproscan</td>
      <td>interproscan</td>
      <td>Interproscan queries the interpro database and provides annotations.</td>
      <td>interproscan_ebi</td>
      <td>interproscan_ebi</td>
      <td></td>
      <td>InterProScan (EBI)</td>
      <td>Scan sequences against the InterPro protein signature databases.</td>
      <td>Sequence motif recognition, Protein feature detection</td>
      <td>Gene and protein families, Sequence analysis</td>
      <td>To update</td>
      <td>http://www.ebi.ac.uk/Tools/pfa/iprscan5/</td>
      <td>Sequence Analysis</td>
      <td>interproscan</td>
      <td>2021-11-15</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/interproscan</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/interproscan</td>
      <td>5.59-91.0</td>
      <td>interproscan</td>
      <td>5.59_91.0</td>
      <td>Sequence motif recognition, Protein feature detection</td>
      <td>Gene and protein families, Sequence analysis</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>34689</td>
      <td>55189</td>
      <td>1634</td>
      <td>1713</td>
      <td>65</td>
      <td>65</td>
      <td>15</td>
      <td>15</td>
      <td>10897</td>
      <td>10897</td>
      <td>607</td>
      <td>607</td>
      <td>45651</td>
      <td>66151</td>
      <td>2256</td>
      <td>2335</td>
    </tr>
    <tr>
      <td></td>
      <td>interval2maf</td>
      <td>Interval2Maf1</td>
      <td>Extract MAF blocks given a set of intervals</td>
      <td>bx-python</td>
      <td>bx-python</td>
      <td></td>
      <td>bx-python</td>
      <td>Tools for manipulating biological data, particularly multiple sequence alignments.</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/interval2maf/</td>
      <td>Genomic Interval Operations</td>
      <td>interval2maf</td>
      <td>2020-07-14</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/interval2maf/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/interval2maf</td>
      <td>1.0.1+galaxy1</td>
      <td>bx-python</td>
      <td>0.13.0</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>32</td>
      <td>107</td>
      <td>7</td>
      <td>10</td>
      <td>758</td>
      <td>19538</td>
      <td>87</td>
      <td>1246</td>
      <td>16</td>
      <td>24</td>
      <td>2</td>
      <td>2</td>
      <td>806</td>
      <td>19669</td>
      <td>96</td>
      <td>1258</td>
    </tr>
    <tr>
      <td></td>
      <td>intervene</td>
      <td>intervene_pairwise, intervene_upset</td>
      <td>Create pairwise and upset plots</td>
      <td>intervene</td>
      <td>intervene</td>
      <td></td>
      <td>Intervene</td>
      <td>Tool for intersection and visualization of multiple gene or genomic region sets. Intervene contains three modules: venn to generate Venn diagrams of up to six sets, upset to generate UpSet plots of multiple sets, and pairwise to compute and visualize intersections of multiple sets as clustered heat maps.</td>
      <td>Sequence comparison, Sequence visualisation</td>
      <td>Computational biology</td>
      <td>Up-to-date</td>
      <td>https://intervene.readthedocs.io</td>
      <td>Statistics</td>
      <td>intervene</td>
      <td>2018-09-11</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/intervene</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/intervene</td>
      <td>0.6.5</td>
      <td>intervene</td>
      <td>0.6.5</td>
      <td>Sequence comparison, Sequence visualisation</td>
      <td>Computational biology</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>1557</td>
      <td>1607</td>
      <td>278</td>
      <td>292</td>
      <td>4458</td>
      <td>4458</td>
      <td>458</td>
      <td>458</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6015</td>
      <td>6065</td>
      <td>736</td>
      <td>750</td>
    </tr>
    <tr>
      <td></td>
      <td>iqtree</td>
      <td>iqtree</td>
      <td>Efficient phylogenomic software by maximum likelihood</td>
      <td>iqtree</td>
      <td>iqtree</td>
      <td></td>
      <td>iqtree</td>
      <td>A fast and effective stochastic algorithm to infer phylogenetic trees by maximum likelihood. IQ-TREE compares favorably to RAxML and PhyML in terms of likelihoods with similar computing time</td>
      <td>Phylogenetic analysis, Sequence analysis</td>
      <td>Phylogenetics</td>
      <td>Up-to-date</td>
      <td>http://www.iqtree.org/</td>
      <td>Phylogenetics</td>
      <td>iqtree</td>
      <td>2017-11-15</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/iqtree/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/iqtree</td>
      <td>2.3.6</td>
      <td>iqtree</td>
      <td>2.3.6</td>
      <td>Phylogenetic analysis, Sequence analysis</td>
      <td>Phylogenetics</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>28279</td>
      <td>28481</td>
      <td>1623</td>
      <td>1646</td>
      <td>24662</td>
      <td>24662</td>
      <td>2093</td>
      <td>2093</td>
      <td>9780</td>
      <td>10086</td>
      <td>979</td>
      <td>992</td>
      <td>62721</td>
      <td>63229</td>
      <td>4695</td>
      <td>4731</td>
    </tr>
    <tr>
      <td></td>
      <td>irissv</td>
      <td>irissv</td>
      <td>Refine insertion sequences</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/mkirsche/Iris</td>
      <td>Variant Analysis</td>
      <td>irissv</td>
      <td>2021-01-17</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tools/irissv/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/irissv</td>
      <td>1.0.5</td>
      <td>samtools</td>
      <td>1.21</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>29</td>
      <td>29</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>29</td>
      <td>29</td>
      <td>8</td>
      <td>8</td>
    </tr>
    <tr>
      <td></td>
      <td>isescan</td>
      <td>isescan</td>
      <td>"ISEScan is a pipeline to identify IS (Insertion Sequence) elements in genome and metagenomebased on profile hidden Markov models constructed from manually curated IS elements."</td>
      <td>ISEScan</td>
      <td>ISEScan</td>
      <td></td>
      <td>ISEScan</td>
      <td>Automated identification of insertion sequence elements in prokaryotic genomes.</td>
      <td>Structural variation detection</td>
      <td>Genomics, DNA structural variation, Sequence analysis, Genetic variation</td>
      <td>To update</td>
      <td>https://github.com/xiezhq/ISEScan</td>
      <td>Sequence Analysis</td>
      <td>ISEScan</td>
      <td>2022-09-01</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/isescan</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/isescan</td>
      <td>1.7.2.3</td>
      <td>isescan</td>
      <td>1.7.2.1</td>
      <td>Structural variation detection</td>
      <td>Genomics, Sequence analysis, Genetic variation</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>59386</td>
      <td>59386</td>
      <td>151</td>
      <td>151</td>
      <td>56</td>
      <td>56</td>
      <td>1</td>
      <td>1</td>
      <td>4741</td>
      <td>4741</td>
      <td>59</td>
      <td>59</td>
      <td>64183</td>
      <td>64183</td>
      <td>211</td>
      <td>211</td>
    </tr>
    <tr>
      <td></td>
      <td>isoformswitchanalyzer</td>
      <td>isoformswitchanalyzer</td>
      <td>Statistical identification of isoform switching from RNA-seq derived quantification of novel and/or annotated full-length isoforms.</td>
      <td>IsoformSwitchAnalyzeR</td>
      <td>IsoformSwitchAnalyzeR</td>
      <td></td>
      <td>IsoformSwitchAnalyzeR</td>
      <td>Enables identification of isoform switches with predicted functional consequences from RNA-seq data. Consequences can be chosen from a long list but includes protein domains gain/loss changes in NMD sensitivity etc. It directly supports import of data from Cufflinks/Cuffdiff, Kallisto, Salmon and RSEM but other transcript qunatification tools are easy to import as well.</td>
      <td>Sequence comparison, Sequence analysis</td>
      <td>Computational biology, Gene transcripts</td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/devel/bioc/html/IsoformSwitchAnalyzeR.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>isoformswitchanalyzer</td>
      <td>2023-05-24</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/isoformswitchanalyzer</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/isoformswitchanalyzer</td>
      <td>1.20.0</td>
      <td>bioconductor-isoformswitchanalyzer</td>
      <td>2.2.0</td>
      <td>Sequence comparison, Sequence analysis</td>
      <td>Computational biology, Gene transcripts</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1301</td>
      <td>1301</td>
      <td>59</td>
      <td>59</td>
      <td>1331</td>
      <td>1331</td>
      <td>60</td>
      <td>60</td>
      <td>507</td>
      <td>507</td>
      <td>9</td>
      <td>9</td>
      <td>3139</td>
      <td>3139</td>
      <td>128</td>
      <td>128</td>
    </tr>
    <tr>
      <td></td>
      <td>ivar</td>
      <td>ivar_consensus, ivar_filtervariants, ivar_removereads, ivar_trim, ivar_variants</td>
      <td>iVar is a computational package that contains functions broadly useful for viral amplicon-based sequencing</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/andersen-lab/ivar</td>
      <td>Sequence Analysis</td>
      <td>ivar</td>
      <td>2020-06-05</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ivar/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ivar</td>
      <td>1.4.3</td>
      <td>ivar</td>
      <td>1.4.3</td>
      <td></td>
      <td></td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>1252220</td>
      <td>1252220</td>
      <td>2366</td>
      <td>2366</td>
      <td>126027</td>
      <td>126027</td>
      <td>2496</td>
      <td>2496</td>
      <td>41901</td>
      <td>41901</td>
      <td>993</td>
      <td>993</td>
      <td>1420148</td>
      <td>1420148</td>
      <td>5855</td>
      <td>5855</td>
    </tr>
    <tr>
      <td></td>
      <td>iwtomics</td>
      <td>iwtomics_loadandplot, iwtomics_plotwithscale, iwtomics_testandplot</td>
      <td>Interval-Wise Testing for Omics Data</td>
      <td>iwtomics</td>
      <td>iwtomics</td>
      <td></td>
      <td>IWTomics</td>
      <td>Implementation of the Interval-Wise Testing (IWT) for omics data. This inferential procedure tests for differences in "Omics" data between two groups of genomic regions (or between a group of genomic regions and a reference center of symmetry), and does not require fixing location and scale at the outset.</td>
      <td>Differential gene expression analysis, Differentially-methylated region identification, Peak calling, Genome annotation, Comparison</td>
      <td>Statistics and probability</td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/IWTomics.html</td>
      <td>Statistics</td>
      <td>iwtomics</td>
      <td>2017-06-08</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/iwtomics</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/iwtomics</td>
      <td>1.0.0</td>
      <td>bioconductor-iwtomics</td>
      <td>1.26.0</td>
      <td>Differential gene expression analysis, Peak calling, Genome annotation, Comparison</td>
      <td>Statistics and probability</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>675</td>
      <td>690</td>
      <td>27</td>
      <td>28</td>
      <td>159</td>
      <td>251</td>
      <td>29</td>
      <td>49</td>
      <td>383</td>
      <td>383</td>
      <td>12</td>
      <td>12</td>
      <td>1217</td>
      <td>1324</td>
      <td>68</td>
      <td>89</td>
    </tr>
    <tr>
      <td></td>
      <td>jasminesv</td>
      <td>jasminesv</td>
      <td>Merge structural variants across samples</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/mkirsche/Jasmine/</td>
      <td>Variant Analysis</td>
      <td>jasminesv</td>
      <td>2021-01-17</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/jasminesv/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/jasminesv</td>
      <td>1.0.11</td>
      <td>jasminesv</td>
      <td>1.1.5</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>32</td>
      <td>32</td>
      <td>9</td>
      <td>9</td>
      <td>53</td>
      <td>53</td>
      <td>19</td>
      <td>19</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>85</td>
      <td>85</td>
      <td>28</td>
      <td>28</td>
    </tr>
    <tr>
      <td></td>
      <td>jbrowse</td>
      <td>jbrowse_to_standalone, jbrowse</td>
      <td>JBrowse Genome Browser integrated as a Galaxy Tool</td>
      <td>jbrowse</td>
      <td>jbrowse</td>
      <td></td>
      <td>JBrowse</td>
      <td>Slick, speedy genome browser with a responsive and dynamic AJAX interface for visualization of genome data. Being developed by the GMOD project as a successor to GBrowse.</td>
      <td>Genome visualisation</td>
      <td>Genomics</td>
      <td>Up-to-date</td>
      <td>https://jbrowse.org</td>
      <td>Sequence Analysis</td>
      <td>jbrowse</td>
      <td>2019-08-29</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/jbrowse</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/jbrowse</td>
      <td>1.16.11</td>
      <td>jbrowse</td>
      <td>1.16.11</td>
      <td>Genome visualisation</td>
      <td>Genomics</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>21153</td>
      <td>22174</td>
      <td>5514</td>
      <td>5779</td>
      <td>17428</td>
      <td>17428</td>
      <td>6458</td>
      <td>6458</td>
      <td>15673</td>
      <td>17267</td>
      <td>3248</td>
      <td>3786</td>
      <td>54254</td>
      <td>56869</td>
      <td>15220</td>
      <td>16023</td>
    </tr>
    <tr>
      <td></td>
      <td>jcvi_gff_stats</td>
      <td>jcvi_gff_stats</td>
      <td>Compute statistics from a genome annotation in GFF3 format (using JCVI Python utilities)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/tanghaibao/jcvi</td>
      <td>Sequence Analysis</td>
      <td>jcvi_gff_stats</td>
      <td>2018-07-31</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/jcvi_gff_stats</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/jcvi_gff_stats</td>
      <td>0.8.4</td>
      <td>jcvi</td>
      <td>1.4.16</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2747</td>
      <td>2926</td>
      <td>596</td>
      <td>637</td>
      <td>2311</td>
      <td>2311</td>
      <td>741</td>
      <td>741</td>
      <td>1145</td>
      <td>1145</td>
      <td>469</td>
      <td>469</td>
      <td>6203</td>
      <td>6382</td>
      <td>1806</td>
      <td>1847</td>
    </tr>
    <tr>
      <td></td>
      <td>jellyfish</td>
      <td>jellyfish</td>
      <td>Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA</td>
      <td>Jellyfish</td>
      <td>Jellyfish</td>
      <td></td>
      <td>Jellyfish</td>
      <td>A command-line algorithm for counting k-mers in DNA sequence.</td>
      <td>k-mer counting</td>
      <td>Sequence analysis, Genomics</td>
      <td>To update</td>
      <td>https://github.com/gmarcais/Jellyfish</td>
      <td>Assembly</td>
      <td>jellyfish</td>
      <td>2021-04-07</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/jellyfish</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/jellyfish</td>
      <td></td>
      <td>kmer-jellyfish</td>
      <td>2.3.1</td>
      <td>k-mer counting</td>
      <td>Sequence analysis, Genomics</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1825</td>
      <td>1825</td>
      <td>237</td>
      <td>237</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>971</td>
      <td>971</td>
      <td>56</td>
      <td>56</td>
      <td>2796</td>
      <td>2796</td>
      <td>293</td>
      <td>293</td>
    </tr>
    <tr>
      <td></td>
      <td>join_files_by_id</td>
      <td>join_files_by_id</td>
      <td>This tool will join datasets according to a column with identifier</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/join_files_by_id</td>
      <td>Text Manipulation</td>
      <td>join_files_by_id</td>
      <td>2017-05-31</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/join_files_by_id</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/join_files_by_id</td>
      <td>1.0</td>
      <td>r-data.table</td>
      <td>1.11.6</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>jq</td>
      <td>jq</td>
      <td>JQ is a lightweight and flexible command-line JSON processor</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://stedolan.github.io/jq/</td>
      <td>Text Manipulation</td>
      <td>jq</td>
      <td>2017-06-22</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/jq</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/jq</td>
      <td>1.0</td>
      <td>jq</td>
      <td>1.5</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>660</td>
      <td>2374</td>
      <td>61</td>
      <td>63</td>
      <td>336</td>
      <td>336</td>
      <td>82</td>
      <td>82</td>
      <td>281</td>
      <td>281</td>
      <td>11</td>
      <td>11</td>
      <td>1277</td>
      <td>2991</td>
      <td>154</td>
      <td>156</td>
    </tr>
    <tr>
      <td></td>
      <td>jvarkit</td>
      <td>jvarkit_wgscoverageplotter</td>
      <td>Jvarkit : Java utilities for Bioinformatics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://lindenb.github.io/jvarkit/</td>
      <td>SAM</td>
      <td>jvarkit</td>
      <td>2021-02-11</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/iuc/tree/master/tools/jvarkit</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/jvarkit</td>
      <td>20201223</td>
      <td>jvarkit-wgscoverageplotter</td>
      <td>20201223</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>7179</td>
      <td>7179</td>
      <td>509</td>
      <td>509</td>
      <td>11138</td>
      <td>11138</td>
      <td>1145</td>
      <td>1145</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>18317</td>
      <td>18317</td>
      <td>1654</td>
      <td>1654</td>
    </tr>
    <tr>
      <td></td>
      <td>kallisto</td>
      <td>kallisto_pseudo, kallisto_quant</td>
      <td>kallisto is a program for quantifying abundances of transcripts from RNA-Seqdata, or more generally of target sequences using high-throughput sequencingreads. It is based on the novel idea of pseudoalignment for rapidlydetermining the compatibility of reads with targets, without the need foralignment.</td>
      <td>kallisto</td>
      <td>kallisto</td>
      <td></td>
      <td>kallisto</td>
      <td>A program for quantifying abundances of transcripts from RNA-Seq data, or more generally of target sequences using high-throughput sequencing reads. It is based on the novel idea of pseudoalignment for rapidly determining the compatibility of reads with targets, without the need for alignment.</td>
      <td>Gene expression profiling</td>
      <td>Transcriptomics, RNA-seq, Gene expression</td>
      <td>To update</td>
      <td>https://pachterlab.github.io/kallisto/</td>
      <td>Transcriptomics</td>
      <td></td>
      <td>2017-08-09</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/kallisto/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/kallisto</td>
      <td>0.48.0</td>
      <td>kallisto</td>
      <td>0.51.1</td>
      <td>Gene expression profiling</td>
      <td>Transcriptomics</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>28398</td>
      <td>29031</td>
      <td>932</td>
      <td>965</td>
      <td>73705</td>
      <td>80087</td>
      <td>2603</td>
      <td>3025</td>
      <td>5333</td>
      <td>5567</td>
      <td>218</td>
      <td>230</td>
      <td>107436</td>
      <td>114685</td>
      <td>3753</td>
      <td>4220</td>
    </tr>
    <tr>
      <td></td>
      <td>kc-align</td>
      <td>kc-align</td>
      <td>Kc-Align custom tool</td>
      <td>kc-align</td>
      <td>kc-align</td>
      <td></td>
      <td>kc-align</td>
      <td>A fast and accurate tool for performing codon-aware multiple sequence alignments</td>
      <td>Multiple sequence alignment</td>
      <td>Mapping</td>
      <td>Up-to-date</td>
      <td>https://github.com/davebx/kc-align</td>
      <td>Sequence Analysis</td>
      <td>kc_align</td>
      <td>2020-02-20</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/kc-align</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/kc-align</td>
      <td>1.0.2</td>
      <td>kcalign</td>
      <td>1.0.2</td>
      <td>Multiple sequence alignment</td>
      <td>Mapping</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>379</td>
      <td>379</td>
      <td>53</td>
      <td>53</td>
      <td>8259</td>
      <td>8259</td>
      <td>271</td>
      <td>271</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8638</td>
      <td>8638</td>
      <td>324</td>
      <td>324</td>
    </tr>
    <tr>
      <td></td>
      <td>khmer</td>
      <td>khmer_abundance_distribution_single, khmer_abundance_distribution, khmer_count_median, khmer_partition, khmer_extract_partitions, khmer_filter_abundance, khmer_filter_below_abundance_cutoff, khmer_normalize_by_median</td>
      <td>In-memory nucleotide sequence k-mer counting, filtering, graph traversal and more</td>
      <td>khmer</td>
      <td>khmer</td>
      <td></td>
      <td>khmer</td>
      <td>khmer is a set of command-line tools for working with DNA shotgun sequencing data from genomes, transcriptomes, metagenomes, and single cells. khmer can make de novo assemblies faster, and sometimes better. khmer can also identify (and fix) problems with shotgun data.</td>
      <td>Standardisation and normalisation, De-novo assembly</td>
      <td>Sequence assembly</td>
      <td>Up-to-date</td>
      <td>https://khmer.readthedocs.org/</td>
      <td>Assembly, Next Gen Mappers</td>
      <td>khmer</td>
      <td>2015-11-08</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/khmer</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/khmer</td>
      <td>3.0.0a3</td>
      <td>khmer</td>
      <td>3.0.0a3</td>
      <td>Standardisation and normalisation, De-novo assembly</td>
      <td>Sequence assembly</td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>1995</td>
      <td>2091</td>
      <td>174</td>
      <td>193</td>
      <td>1430</td>
      <td>1430</td>
      <td>389</td>
      <td>389</td>
      <td>1580</td>
      <td>1580</td>
      <td>56</td>
      <td>56</td>
      <td>5005</td>
      <td>5101</td>
      <td>619</td>
      <td>638</td>
    </tr>
    <tr>
      <td></td>
      <td>king</td>
      <td>king</td>
      <td>Kinship-based INference for Gwas</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://people.virginia.edu/~wc9c/KING/</td>
      <td>Variant Analysis</td>
      <td>king</td>
      <td>2021-11-16</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/king/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/king</td>
      <td>2.2.7</td>
      <td>king</td>
      <td>2.2.7</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>4</td>
      <td>4</td>
    </tr>
    <tr>
      <td></td>
      <td>kleborate</td>
      <td>kleborate</td>
      <td>Screen genome assemblies of Klebsiella pneumoniae and the Klebsiella pneumoniae species complex (KpSC)</td>
      <td>kleborate</td>
      <td>kleborate</td>
      <td></td>
      <td>Kleborate</td>
      <td>Genomic surveillance framework and global population structure for Klebsiella pneumoniae.Kleborate is a tool to screen genome assemblies of Klebsiella pneumoniae and the Klebsiella pneumoniae species complex (KpSC) for:.A manuscript describing the Kleborate software in full is currently in preparation. In the meantime, if you use Kleborate, please cite the preprint: Lam, MMC. et al. Genomic surveillance framework and global population structure for Klebsiella pneumoniae. bioRxiv (2020).</td>
      <td>Multilocus sequence typing, Genome assembly, Virulence prediction</td>
      <td>Public health and epidemiology, Metagenomics, Population genomics, Sequence assembly, Whole genome sequencing</td>
      <td>To update</td>
      <td>https://github.com/katholt/Kleborate/wiki</td>
      <td>Metagenomics</td>
      <td>kleborate</td>
      <td>2022-09-09</td>
      <td>iuc</td>
      <td>https://github.com/katholt/Kleborate</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/kleborate</td>
      <td>2.3.2</td>
      <td>kleborate</td>
      <td>3.1.0</td>
      <td>Multilocus sequence typing, Genome assembly, Virulence prediction</td>
      <td>Public health and epidemiology, Metagenomics, Population genomics, Sequence assembly, Whole genome sequencing</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>503</td>
      <td>503</td>
      <td>58</td>
      <td>58</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>503</td>
      <td>503</td>
      <td>58</td>
      <td>58</td>
    </tr>
    <tr>
      <td></td>
      <td>kma</td>
      <td>kma_map</td>
      <td>Map with KMA</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/genomicepidemiology/kma</td>
      <td>Next Gen Mappers</td>
      <td>kma</td>
      <td>2019-10-30</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/kma</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/kma</td>
      <td>1.4.14</td>
      <td>kma</td>
      <td>1.4.15</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
    </tr>
    <tr>
      <td></td>
      <td>kofamscan</td>
      <td>kofamscan</td>
      <td>Gene function annotation tool based on KEGG Orthology and hidden Markov model</td>
      <td>kofamscan</td>
      <td>kofamscan</td>
      <td></td>
      <td>kofamscan</td>
      <td>KofamScan is a gene function annotation tool based on KEGG Orthology and hidden Markov model. You need KOfam database to use this tool.</td>
      <td>Sequence analysis, Gene functional annotation</td>
      <td>Genomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/takaram/kofam_scan</td>
      <td>Sequence Analysis</td>
      <td>kofamscan</td>
      <td>2020-11-12</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/kofamscan</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/kofamscan</td>
      <td>1.3.0</td>
      <td>kofamscan</td>
      <td>1.3.0</td>
      <td>Sequence analysis, Gene functional annotation</td>
      <td>Genomics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1342</td>
      <td>1342</td>
      <td>63</td>
      <td>63</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1342</td>
      <td>1342</td>
      <td>63</td>
      <td>63</td>
    </tr>
    <tr>
      <td></td>
      <td>kraken</td>
      <td>kraken-filter, kraken-mpa-report, kraken-report, kraken-translate, kraken</td>
      <td>Kraken is a system for assigning taxonomic labels to short DNAsequences, usually obtained through metagenomic studies. Previous attempts by otherbioinformatics software to accomplish this task have often used sequence alignmentor machine learning techniques that were quite slow, leading to the developmentof less sensitive but much faster abundance estimation programs. Kraken aims toachieve high sensitivity and high speed by utilizing exact alignments of k-mersand a novel classification algorithm.</td>
      <td>kraken</td>
      <td>kraken</td>
      <td></td>
      <td>Kraken</td>
      <td>System for assigning taxonomic labels to short DNA sequences, usually obtained through metagenomic studies. Previous attempts by other bioinformatics software to accomplish this task have often used sequence alignment or machine learning techniques that were quite slow, leading to the development of less sensitive but much faster abundance estimation programs. It aims to achieve high sensitivity and high speed by utilizing exact alignments of k-mers and a novel classification algorithm.</td>
      <td>Taxonomic classification</td>
      <td>Taxonomy, Metagenomics</td>
      <td>To update</td>
      <td>http://ccb.jhu.edu/software/kraken/</td>
      <td>Metagenomics</td>
      <td>kraken</td>
      <td>2017-01-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tool_collections/kraken/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/kraken</td>
      <td></td>
      <td>kraken</td>
      <td>1.1.1</td>
      <td>Taxonomic classification</td>
      <td>Taxonomy, Metagenomics</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>48136</td>
      <td>48164</td>
      <td>2754</td>
      <td>2760</td>
      <td>59952</td>
      <td>130398</td>
      <td>5869</td>
      <td>8654</td>
      <td>29008</td>
      <td>31148</td>
      <td>1936</td>
      <td>2200</td>
      <td>137096</td>
      <td>209710</td>
      <td>10559</td>
      <td>13614</td>
    </tr>
    <tr>
      <td></td>
      <td>kraken2</td>
      <td>kraken2</td>
      <td>Kraken2 for taxonomic designation.</td>
      <td>kraken2</td>
      <td>kraken2</td>
      <td></td>
      <td>kraken2</td>
      <td>Kraken 2 is the newest version of Kraken, a taxonomic classification system using exact k-mer matches to achieve high accuracy and fast classification speeds. This classifier matches each k-mer within a query sequence to the lowest common ancestor (LCA) of all genomes containing the given k-mer. The k-mer assignments inform the classification algorithm.</td>
      <td>Taxonomic classification</td>
      <td>Taxonomy, Metagenomics</td>
      <td>Up-to-date</td>
      <td>http://ccb.jhu.edu/software/kraken/</td>
      <td>Metagenomics</td>
      <td>kraken2</td>
      <td>2019-03-06</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tool_collections/kraken2/kraken2/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/kraken2/kraken2</td>
      <td>2.1.3</td>
      <td>kraken2</td>
      <td>2.1.3</td>
      <td>Taxonomic classification</td>
      <td>Taxonomy, Metagenomics</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>271367</td>
      <td>271786</td>
      <td>6384</td>
      <td>6409</td>
      <td>101752</td>
      <td>101752</td>
      <td>4662</td>
      <td>4662</td>
      <td>53562</td>
      <td>53562</td>
      <td>1663</td>
      <td>1663</td>
      <td>426681</td>
      <td>427100</td>
      <td>12709</td>
      <td>12734</td>
    </tr>
    <tr>
      <td></td>
      <td>kraken_biom</td>
      <td>kraken_biom</td>
      <td>Create BIOM-format tables (http://biom-format.org) from Kraken output (http://ccb.jhu.edu/software/kraken/)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/smdabdoub/kraken-biom</td>
      <td>Metagenomics</td>
      <td>kraken_biom</td>
      <td>2022-09-03</td>
      <td>iuc</td>
      <td>https://github.com/smdabdoub/kraken-biom</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/kraken_biom</td>
      <td>1.2.0</td>
      <td>kraken-biom</td>
      <td>1.2.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>3185</td>
      <td>3185</td>
      <td>495</td>
      <td>495</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>385</td>
      <td>385</td>
      <td>48</td>
      <td>48</td>
      <td>3570</td>
      <td>3570</td>
      <td>543</td>
      <td>543</td>
    </tr>
    <tr>
      <td></td>
      <td>kraken_taxonomy_report</td>
      <td>kraken_taxonomy_report</td>
      <td>Kraken taxonomy report</td>
      <td>Kraken-Taxonomy-Report</td>
      <td>Kraken-Taxonomy-Report</td>
      <td></td>
      <td>Kraken-Taxonomy-Report</td>
      <td>view report of classification for multiple samples</td>
      <td>Visualisation, Classification</td>
      <td>Metagenomics, Taxonomy</td>
      <td>To update</td>
      <td>https://github.com/blankenberg/Kraken-Taxonomy-Report</td>
      <td>Metagenomics</td>
      <td>kraken_taxonomy_report</td>
      <td>2016-06-01</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/kraken_taxonomy_report</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/kraken_taxonomy_report</td>
      <td>0.0.3</td>
      <td>biopython</td>
      <td>1.70</td>
      <td>Visualisation, Classification</td>
      <td>Metagenomics, Taxonomy</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>3981</td>
      <td>3986</td>
      <td>769</td>
      <td>771</td>
      <td>4087</td>
      <td>5610</td>
      <td>991</td>
      <td>1284</td>
      <td>88</td>
      <td>88</td>
      <td>41</td>
      <td>41</td>
      <td>8156</td>
      <td>9684</td>
      <td>1801</td>
      <td>2096</td>
    </tr>
    <tr>
      <td></td>
      <td>krakentools</td>
      <td>krakentools_alpha_diversity, krakentools_beta_diversity, krakentools_combine_kreports, krakentools_extract_kraken_reads, krakentools_kreport2krona, krakentools_kreport2mpa</td>
      <td>KrakenTools is a suite of scripts to be used alongside the Kraken</td>
      <td>krakentools</td>
      <td>krakentools</td>
      <td></td>
      <td>KrakenTools</td>
      <td>KrakenTools provides individual scripts to analyze Kraken/Kraken2/Bracken/KrakenUniq output files</td>
      <td>Visualisation, Aggregation</td>
      <td>Taxonomy, Metagenomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/jenniferlu717/KrakenTools</td>
      <td>Metagenomics</td>
      <td>krakentools</td>
      <td>2023-01-13</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/krakentools</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/krakentools</td>
      <td>1.2</td>
      <td>krakentools</td>
      <td>1.2</td>
      <td>Visualisation, Aggregation</td>
      <td>Taxonomy, Metagenomics</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>48516</td>
      <td>48516</td>
      <td>2373</td>
      <td>2373</td>
      <td>8787</td>
      <td>8787</td>
      <td>943</td>
      <td>943</td>
      <td>4369</td>
      <td>4369</td>
      <td>393</td>
      <td>393</td>
      <td>61672</td>
      <td>61672</td>
      <td>3709</td>
      <td>3709</td>
    </tr>
    <tr>
      <td></td>
      <td>krocus</td>
      <td>krocus</td>
      <td>Predict MLST directly from uncorrected long reads</td>
      <td>krocus</td>
      <td>krocus</td>
      <td></td>
      <td>krocus</td>
      <td>Predict MLST directly from uncorrected long reads</td>
      <td>Multilocus sequence typing, k-mer counting</td>
      <td>Public health and epidemiology</td>
      <td>To update</td>
      <td>https://github.com/quadram-institute-bioscience/krocus</td>
      <td>Sequence Analysis</td>
      <td>krocus</td>
      <td>2019-09-03</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/krocus</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/krocus</td>
      <td>1.0.1</td>
      <td>krocus</td>
      <td>1.0.3</td>
      <td>Multilocus sequence typing, k-mer counting</td>
      <td>Public health and epidemiology</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>last</td>
      <td>last_al, last_db, last_split, last_train, last_maf_convert</td>
      <td>LAST finds similar regions between sequences.</td>
      <td>last</td>
      <td>last</td>
      <td></td>
      <td>LAST</td>
      <td>Short read alignment program incorporating quality scores</td>
      <td>Sequence alignment</td>
      <td>Genomics, Comparative genomics</td>
      <td>To update</td>
      <td>http://last.cbrc.jp/</td>
      <td>Sequence Analysis</td>
      <td>last</td>
      <td>2020-06-17</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/last</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/last</td>
      <td>1205</td>
      <td>last</td>
      <td>1584</td>
      <td>Sequence alignment</td>
      <td>Comparative genomics</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>329</td>
      <td>329</td>
      <td>94</td>
      <td>94</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>329</td>
      <td>329</td>
      <td>94</td>
      <td>94</td>
    </tr>
    <tr>
      <td></td>
      <td>lastz</td>
      <td>lastz_wrapper_2, lastz_d_wrapper</td>
      <td>Galaxy wrappers for the Lastz and Lastz_d</td>
      <td>lastz</td>
      <td>lastz</td>
      <td></td>
      <td>LASTZ</td>
      <td>A tool for (1) aligning two DNA sequences, and (2) inferring appropriate scoring parameters automatically.</td>
      <td>Sequence alignment, Read mapping</td>
      <td>Genomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/lastz/lastz</td>
      <td>Next Gen Mappers</td>
      <td>lastz</td>
      <td>2018-02-19</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/lastz</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/lastz</td>
      <td>1.04.22</td>
      <td>lastz</td>
      <td>1.04.22</td>
      <td>Sequence alignment, Read mapping</td>
      <td>Genomics</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>77409</td>
      <td>84356</td>
      <td>447</td>
      <td>475</td>
      <td>156658</td>
      <td>193590</td>
      <td>870</td>
      <td>3511</td>
      <td>38579</td>
      <td>38780</td>
      <td>173</td>
      <td>203</td>
      <td>272646</td>
      <td>316726</td>
      <td>1490</td>
      <td>4189</td>
    </tr>
    <tr>
      <td></td>
      <td>lcrgenie</td>
      <td>lcrgenie</td>
      <td>Ligase Chain Reaction Genie</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/neilswainston/LCRGenie</td>
      <td>Synthetic Biology</td>
      <td>lcrgenie</td>
      <td>2022-12-02</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/lcrgenie</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/lcrgenie</td>
      <td>1.0.2</td>
      <td>lcr_genie</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>legsta</td>
      <td>legsta</td>
      <td>Performs in silico Legionella pneumophila sequence based typing.</td>
      <td>legsta</td>
      <td>legsta</td>
      <td></td>
      <td>legsta</td>
      <td>Performs in silico Legionella pneumophila sequence based typing</td>
      <td>Sequence analysis</td>
      <td>Public health and epidemiology</td>
      <td>Up-to-date</td>
      <td>https://github.com/tseemann/legsta</td>
      <td>Sequence Analysis</td>
      <td>legsta</td>
      <td>2022-02-21</td>
      <td>iuc</td>
      <td>https://github.com/tseemann/legsta</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/legsta</td>
      <td>0.5.1</td>
      <td>legsta</td>
      <td>0.5.1</td>
      <td>Sequence analysis</td>
      <td>Public health and epidemiology</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>92</td>
      <td>92</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>92</td>
      <td>92</td>
      <td>8</td>
      <td>8</td>
    </tr>
    <tr>
      <td></td>
      <td>length_and_gc_content</td>
      <td>length_and_gc_content</td>
      <td>Gets gene length and gc content from a fasta and a GTF file</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/length_and_gc_content</td>
      <td>Fasta Manipulation, Statistics, RNA, Micro-array Analysis</td>
      <td>length_and_gc_content</td>
      <td>2016-11-13</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/length_and_gc_content</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/length_and_gc_content</td>
      <td>0.1.2</td>
      <td>r-optparse</td>
      <td>1.3.2</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>4349</td>
      <td>4466</td>
      <td>666</td>
      <td>698</td>
      <td>2093</td>
      <td>2286</td>
      <td>928</td>
      <td>998</td>
      <td>201</td>
      <td>201</td>
      <td>77</td>
      <td>77</td>
      <td>6643</td>
      <td>6953</td>
      <td>1671</td>
      <td>1773</td>
    </tr>
    <tr>
      <td></td>
      <td>limma_voom</td>
      <td>limma_voom</td>
      <td>Perform RNA-Seq differential expression analysis using limma voom pipeline</td>
      <td>limma</td>
      <td>limma</td>
      <td></td>
      <td>limma</td>
      <td>Data analysis, linear models and differential expression for microarray data.</td>
      <td>RNA-Seq analysis</td>
      <td>Molecular biology, Genetics</td>
      <td>Up-to-date</td>
      <td>http://bioconductor.org/packages/release/bioc/html/limma.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>limma_voom</td>
      <td>2019-02-17</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/limma_voom</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/limma_voom</td>
      <td>3.58.1</td>
      <td>bioconductor-limma</td>
      <td>3.58.1</td>
      <td>RNA-Seq analysis</td>
      <td>Molecular biology, Genetics</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>23442</td>
      <td>24466</td>
      <td>2379</td>
      <td>2475</td>
      <td>45734</td>
      <td>49342</td>
      <td>4565</td>
      <td>5032</td>
      <td>9327</td>
      <td>9517</td>
      <td>835</td>
      <td>842</td>
      <td>78503</td>
      <td>83325</td>
      <td>7779</td>
      <td>8349</td>
    </tr>
    <tr>
      <td></td>
      <td>lineagespot</td>
      <td>lineagespot</td>
      <td>Identification of SARS-CoV-2 related metagenomic mutations based on a single (or a list of) variant(s) file(s)</td>
      <td>lineagespot</td>
      <td>lineagespot</td>
      <td></td>
      <td>lineagespot</td>
      <td>Lineagespot is a framework written in R, and aims to identify and assign different SARS-CoV-2 lineages based on a single variant file (i.e., variant calling format).</td>
      <td>Variant calling</td>
      <td>Metagenomics, Gene transcripts, Evolutionary biology, Sequencing, Genetic variation</td>
      <td>To update</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/lineagespot.html</td>
      <td>Metagenomics, Sequence Analysis</td>
      <td>lineagespot</td>
      <td>2023-07-05</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/lineagespot</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/lineagespot</td>
      <td>1.6.0</td>
      <td>r-base</td>
      <td></td>
      <td>Variant calling</td>
      <td>Metagenomics, Gene transcripts, Evolutionary biology, Sequencing, Genetic variation</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>39</td>
      <td>39</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>39</td>
      <td>39</td>
      <td>4</td>
      <td>4</td>
    </tr>
    <tr>
      <td></td>
      <td>links</td>
      <td>links</td>
      <td>Scaffold genome assemblies with long reads.</td>
      <td>links</td>
      <td>links</td>
      <td></td>
      <td>LINKS</td>
      <td>LINKS (Long Interval Nucleotide K-mer Scaffolder) is a genomics application for scaffolding genome assemblies with long reads, such as those produced by Oxford Nanopore Technologies Ltd. It can be used to scaffold high-quality draft genome assemblies with any long sequences (eg. ONT reads, PacBio reads, other draft genomes, etc). It is also used to scaffold contig pairs linked by ARCS/ARKS.</td>
      <td>Scaffolding, Genome assembly, Read mapping, Read pre-processing, Sequence trimming</td>
      <td>Sequence assembly, Mapping, Sequencing</td>
      <td>Up-to-date</td>
      <td>https://github.com/bcgsc/LINKS</td>
      <td>Assembly</td>
      <td>links</td>
      <td>2022-02-23</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/links</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/links</td>
      <td>2.0.1</td>
      <td>links</td>
      <td>2.0.1</td>
      <td>Scaffolding, Genome assembly, Read mapping, Read pre-processing, Sequence trimming</td>
      <td>Sequence assembly, Mapping, Sequencing</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>512</td>
      <td>512</td>
      <td>126</td>
      <td>126</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>480</td>
      <td>480</td>
      <td>65</td>
      <td>65</td>
      <td>992</td>
      <td>992</td>
      <td>191</td>
      <td>191</td>
    </tr>
    <tr>
      <td></td>
      <td>lofreq</td>
      <td>lofreq_alnqual, lofreq_call, lofreq_filter, lofreq_indelqual, lofreq_viterbi</td>
      <td>LoFreq is a fast and sensitive variant-caller for inferring SNVs and indelsfrom next-generation sequencing data. It makes full use of base-call qualitiesand other sources of errors inherent in sequencing (e.g. mapping or base/indelalignment uncertainty), which are usually ignored by other methods or onlyused for filtering.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://csb5.github.io/lofreq/</td>
      <td>Variant Analysis</td>
      <td></td>
      <td>2019-12-17</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/lofreq</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/lofreq</td>
      <td>2.1.5</td>
      <td>lofreq</td>
      <td>2.1.5</td>
      <td></td>
      <td></td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>4</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>4176367</td>
      <td>4176367</td>
      <td>4301</td>
      <td>4301</td>
      <td>509615</td>
      <td>509615</td>
      <td>8213</td>
      <td>8213</td>
      <td>106342</td>
      <td>106342</td>
      <td>1815</td>
      <td>1815</td>
      <td>4792324</td>
      <td>4792324</td>
      <td>14329</td>
      <td>14329</td>
    </tr>
    <tr>
      <td></td>
      <td>lorikeet</td>
      <td>lorikeet_spoligotype</td>
      <td>Tools for M. tuberculosis DNA fingerprinting (spoligotyping)</td>
      <td>lorikeet</td>
      <td>lorikeet</td>
      <td></td>
      <td>lorikeet</td>
      <td>Tools for M. tuberculosis DNA fingerprinting (spoligotyping)</td>
      <td>Sequence analysis, Genotyping</td>
      <td>Genotype and phenotype</td>
      <td>Up-to-date</td>
      <td>https://github.com/AbeelLab/lorikeet</td>
      <td>Sequence Analysis</td>
      <td>lorikeet_spoligotype</td>
      <td>2018-05-07</td>
      <td>iuc</td>
      <td>https://github.com/AbeelLab/lorikeet</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/lorikeet</td>
      <td>20</td>
      <td>lorikeet</td>
      <td>20</td>
      <td>Sequence analysis, Genotyping</td>
      <td>Genotype and phenotype</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>822</td>
      <td>822</td>
      <td>14</td>
      <td>14</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>822</td>
      <td>822</td>
      <td>14</td>
      <td>14</td>
    </tr>
    <tr>
      <td></td>
      <td>lumpy_sv</td>
      <td>lumpy_prep, lumpy_sv</td>
      <td>LUMPY - a general probabilistic framework for structural variant discovery</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://layerlab.org/software/</td>
      <td>Variant Analysis</td>
      <td>lumpy_sv</td>
      <td>2020-11-12</td>
      <td>iuc</td>
      <td>https://github.com/arq5x/lumpy-sv</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/lumpy_sv</td>
      <td>0.3.1</td>
      <td>lumpy-sv</td>
      <td>0.3.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>1534</td>
      <td>1534</td>
      <td>199</td>
      <td>199</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1534</td>
      <td>1534</td>
      <td>199</td>
      <td>199</td>
    </tr>
    <tr>
      <td></td>
      <td>m6anet</td>
      <td>m6anet</td>
      <td>m6anet to detect m6A RNA modifications from nanopore data</td>
      <td>m6Anet</td>
      <td>m6Anet</td>
      <td></td>
      <td>m6Anet</td>
      <td>Detection of m6A from direct RNA sequencing using a Multiple Instance Learning framework.</td>
      <td>Quantification, Imputation, Gene expression profiling</td>
      <td>RNA-Seq, Transcriptomics, RNA, Machine learning</td>
      <td>Up-to-date</td>
      <td>https://m6anet.readthedocs.io/en/latest</td>
      <td>Sequence Analysis</td>
      <td>m6anet</td>
      <td>2023-10-25</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/m6anet</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/m6anet</td>
      <td>2.1.0</td>
      <td>m6anet</td>
      <td>2.1.0</td>
      <td>Quantification, Imputation, Gene expression profiling</td>
      <td>RNA-Seq, Transcriptomics, RNA, Machine learning</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>44</td>
      <td>44</td>
      <td>11</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>44</td>
      <td>44</td>
      <td>11</td>
      <td>11</td>
    </tr>
    <tr>
      <td></td>
      <td>maaslin2</td>
      <td>maaslin2</td>
      <td>MaAsLin2 is comprehensive R package for efficiently determining multivariable association between microbial meta'omic features and clinical metadata.</td>
      <td>maaslin2</td>
      <td>maaslin2</td>
      <td></td>
      <td>MaAsLin2</td>
      <td>MaAsLin2 is comprehensive R package for efficiently determining multivariable association between phenotypes, environments, exposures, covariates and microbial metaomic features. MaAsLin2 relies on general linear models to accommodate most modern epidemiological study designs, including cross-sectional and longitudinal, and offers a variety of data exploration, normalization, and transformation methods.</td>
      <td>Filtering, Statistical calculation, Standardisation and normalisation, Visualisation</td>
      <td>Metagenomics, Statistics and probability</td>
      <td>Up-to-date</td>
      <td>http://huttenhower.sph.harvard.edu/maaslin</td>
      <td>Metagenomics</td>
      <td>maaslin2</td>
      <td>2021-11-05</td>
      <td>iuc</td>
      <td>https://github.com/biobakery/Maaslin2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/maaslin2</td>
      <td>1.18.0</td>
      <td>bioconductor-maaslin2</td>
      <td>1.18.0</td>
      <td>Filtering, Standardisation and normalisation, Visualisation</td>
      <td>Metagenomics, Statistics and probability</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>2835</td>
      <td>2835</td>
      <td>59</td>
      <td>59</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2835</td>
      <td>2835</td>
      <td>59</td>
      <td>59</td>
    </tr>
    <tr>
      <td></td>
      <td>macs2</td>
      <td>macs2_bdgbroadcall, macs2_bdgcmp, macs2_bdgdiff, macs2_bdgpeakcall, macs2_callpeak, macs2_filterdup, macs2_predictd, macs2_randsample, macs2_refinepeak</td>
      <td>MACS - Model-based Analysis of ChIP-Seq</td>
      <td>macs</td>
      <td>macs</td>
      <td></td>
      <td>MACS</td>
      <td>Model-based Analysis of ChIP-seq data.</td>
      <td>Peak calling, Enrichment analysis, Gene regulatory network analysis</td>
      <td>ChIP-seq, Molecular interactions, pathways and networks, Transcription factors and regulatory sites</td>
      <td>Up-to-date</td>
      <td>https://github.com/taoliu/MACS</td>
      <td>Sequence Analysis, Statistics</td>
      <td>macs2</td>
      <td>2018-01-21</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/macs2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/macs2</td>
      <td>2.2.9.1</td>
      <td>macs2</td>
      <td>2.2.9.1</td>
      <td>Peak calling, Enrichment analysis, Gene regulatory network analysis</td>
      <td>ChIP-seq, Molecular interactions, pathways and networks, Transcription factors and regulatory sites</td>
      <td>9</td>
      <td>9</td>
      <td>9</td>
      <td>9</td>
      <td>9</td>
      <td>9</td>
      <td>9</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>9</td>
      <td>9</td>
      <td>0</td>
      <td>87810</td>
      <td>96069</td>
      <td>4980</td>
      <td>5629</td>
      <td>153500</td>
      <td>227569</td>
      <td>12946</td>
      <td>18825</td>
      <td>10988</td>
      <td>12301</td>
      <td>692</td>
      <td>807</td>
      <td>252298</td>
      <td>335939</td>
      <td>18618</td>
      <td>25261</td>
    </tr>
    <tr>
      <td></td>
      <td>maf_stats</td>
      <td>maf_stats1</td>
      <td>MAF Coverage statistics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/interval2maf/</td>
      <td>Genomic Interval Operations</td>
      <td>maf_stats</td>
      <td>2020-08-12</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/interval2maf/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/maf_stats</td>
      <td>1.0.2+galaxy0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>15</td>
      <td>21</td>
      <td>7</td>
      <td>9</td>
      <td>227</td>
      <td>2470</td>
      <td>52</td>
      <td>464</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>243</td>
      <td>2492</td>
      <td>60</td>
      <td>474</td>
    </tr>
    <tr>
      <td></td>
      <td>maftoaxt</td>
      <td>maftoaxt</td>
      <td>Convert dataset from MAF to axt format</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td></td>
      <td>UCSC Genome Browser Utilities</td>
      <td>Utilities for handling sequences and assemblies from the UCSC Genome Browser project.</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>Up-to-date</td>
      <td>http://hgdownload.cse.ucsc.edu/admin/exe/</td>
      <td>Convert Formats</td>
      <td>ucsc_maftoaxt</td>
      <td>2024-07-30</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ucsc_tools/maftoaxt</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ucsc_tools/maftoaxt</td>
      <td>469</td>
      <td>ucsc-maftoaxt</td>
      <td>469</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>mageck</td>
      <td>mageck_count, mageck_gsea, mageck_mle, mageck_pathway, mageck_test</td>
      <td>Model-based Analysis of Genome-wide CRISPR-Cas9 Knockout (MAGeCK) is a computational tool to identifyimportant genes from the recent genome-scale CRISPR-Cas9 knockout screens technology.</td>
      <td>mageck</td>
      <td>mageck</td>
      <td></td>
      <td>MAGeCK</td>
      <td>Computational tool to identify important genes from the recent genome-scale CRISPR-Cas9 knockout screens technology.</td>
      <td>Genetic variation analysis</td>
      <td>Genetics, Genetic variation, Genomics</td>
      <td>To update</td>
      <td>https://sourceforge.net/projects/mageck/</td>
      <td>Genome editing</td>
      <td></td>
      <td>2018-03-25</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mageck</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mageck</td>
      <td>0.5.9.2</td>
      <td>mageck</td>
      <td>0.5.9.5</td>
      <td>Genetic variation analysis</td>
      <td>Genetics, Genetic variation, Genomics</td>
      <td>3</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>3</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>11761</td>
      <td>12093</td>
      <td>873</td>
      <td>889</td>
      <td>5762</td>
      <td>5762</td>
      <td>473</td>
      <td>473</td>
      <td>5610</td>
      <td>5610</td>
      <td>343</td>
      <td>343</td>
      <td>23133</td>
      <td>23465</td>
      <td>1689</td>
      <td>1705</td>
    </tr>
    <tr>
      <td></td>
      <td>maker</td>
      <td>maker, maker_map_ids</td>
      <td>MAKER is a portable and easily configurable genome annotation pipeline.Its purpose is to allow smaller eukaryotic and prokaryotic genome projects to independently annotate their genomes and to create genome databases.</td>
      <td>maker</td>
      <td>maker</td>
      <td></td>
      <td>MAKER</td>
      <td>Portable and easily configurable genome annotation pipeline. Its purpose is to allow smaller eukaryotic and prokaryotic genome projects to independently annotate their genomes and to create genome databases.</td>
      <td>Genome annotation</td>
      <td>Genomics, DNA, Sequence analysis</td>
      <td>To update</td>
      <td>http://www.yandell-lab.org/software/maker.html</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>2017-10-13</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/maker</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/maker</td>
      <td>2.31.11</td>
      <td>maker</td>
      <td>3.01.03</td>
      <td>Genome annotation</td>
      <td>Genomics, DNA, Sequence analysis</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>6417</td>
      <td>6861</td>
      <td>1315</td>
      <td>1366</td>
      <td>7235</td>
      <td>7235</td>
      <td>1858</td>
      <td>1858</td>
      <td>4112</td>
      <td>4112</td>
      <td>902</td>
      <td>902</td>
      <td>17764</td>
      <td>18208</td>
      <td>4075</td>
      <td>4126</td>
    </tr>
    <tr>
      <td></td>
      <td>malt</td>
      <td>malt_run</td>
      <td>Aligns an input sequence (DNA or proteins) against an index representing a collection of reference DNA or protein sequences.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/husonlab/malt</td>
      <td>Next Gen Mappers</td>
      <td>malt_run</td>
      <td>2021-11-12</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/malt</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/malt</td>
      <td>0.5.3</td>
      <td>malt</td>
      <td>0.62</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>24</td>
      <td>24</td>
      <td>9</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>24</td>
      <td>24</td>
      <td>9</td>
      <td>9</td>
    </tr>
    <tr>
      <td></td>
      <td>map_param_value</td>
      <td>map_param_value</td>
      <td>Map a parameter value to another value</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>map_param_value</td>
      <td>2022-10-13</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/map_param_value</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/map_param_value</td>
      <td>0.2.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3780</td>
      <td>3780</td>
      <td>67</td>
      <td>67</td>
      <td>1012</td>
      <td>1012</td>
      <td>57</td>
      <td>57</td>
      <td>105</td>
      <td>105</td>
      <td>3</td>
      <td>3</td>
      <td>4897</td>
      <td>4897</td>
      <td>127</td>
      <td>127</td>
    </tr>
    <tr>
      <td></td>
      <td>mapseq</td>
      <td>mapseq</td>
      <td>fast and accurate sequence read classification tool designed to assign taxonomy and OTU classifications to ribosomal RNA sequences.</td>
      <td>mapseq</td>
      <td>mapseq</td>
      <td></td>
      <td>MAPseq</td>
      <td>Highly efficient k-mer search with confidence estimates, for rRNA sequence analysis .</td>
      <td>k-mer counting</td>
      <td>Functional, regulatory and non-coding RNA, Sequence analysis, Sequence sites, features and motifs</td>
      <td>To update</td>
      <td>https://github.com/jfmrod/MAPseq</td>
      <td>Metagenomics</td>
      <td>mapseq</td>
      <td>2023-08-02</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mapseq</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mapseq</td>
      <td>2.1.1b</td>
      <td>perl</td>
      <td></td>
      <td>k-mer counting</td>
      <td>Functional, regulatory and non-coding RNA, Sequence analysis, Sequence sites, features and motifs</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>7871</td>
      <td>7871</td>
      <td>32</td>
      <td>32</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7871</td>
      <td>7871</td>
      <td>32</td>
      <td>32</td>
    </tr>
    <tr>
      <td></td>
      <td>mash</td>
      <td>mash_screen, mash_sketch</td>
      <td>Fast genome and metagenome distance estimation using MinHash</td>
      <td>mash</td>
      <td>mash</td>
      <td></td>
      <td>Mash</td>
      <td>Fast genome and metagenome distance estimation using MinHash.</td>
      <td>Sequence distance matrix generation</td>
      <td>Genomics, Metagenomics, Statistics and probability, Sequence analysis, DNA mutation</td>
      <td>Up-to-date</td>
      <td>https://github.com/marbl/Mash</td>
      <td>Metagenomics</td>
      <td></td>
      <td>2019-01-23</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mash</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mash</td>
      <td>2.3</td>
      <td>mash</td>
      <td>2.3</td>
      <td>Sequence distance matrix generation</td>
      <td>Metagenomics, Statistics and probability, Sequence analysis, DNA mutation</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>10119</td>
      <td>10119</td>
      <td>61</td>
      <td>61</td>
      <td>19</td>
      <td>19</td>
      <td>9</td>
      <td>9</td>
      <td>647</td>
      <td>647</td>
      <td>7</td>
      <td>7</td>
      <td>10785</td>
      <td>10785</td>
      <td>77</td>
      <td>77</td>
    </tr>
    <tr>
      <td></td>
      <td>mashmap</td>
      <td>mashmap</td>
      <td>Fast local alignment boundaries</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mashmap</td>
      <td>Sequence Analysis</td>
      <td>mashmap</td>
      <td>2024-02-26</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mashmap</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mashmap</td>
      <td>3.1.3</td>
      <td>mashmap</td>
      <td>3.1.3</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>894</td>
      <td>894</td>
      <td>12</td>
      <td>12</td>
      <td>81</td>
      <td>81</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>975</td>
      <td>975</td>
      <td>17</td>
      <td>17</td>
    </tr>
    <tr>
      <td></td>
      <td>masigpro</td>
      <td>masigpro</td>
      <td>Identify significantly differential expression profiles in time-course microarray experiments</td>
      <td>masigpro</td>
      <td>masigpro</td>
      <td></td>
      <td>maSigPro</td>
      <td>Regression based approach to find genes for which there are significant gene expression profile differences between experimental groups in time course microarray and RNA-Seq experiments.</td>
      <td>Regression analysis</td>
      <td>Gene expression, Molecular genetics, Microarray experiment, RNA-Seq</td>
      <td>To update</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/maSigPro.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>masigpro</td>
      <td>2017-05-09</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/masigpro</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/masigpro</td>
      <td>1.49.3</td>
      <td>coreutils</td>
      <td>8.25</td>
      <td>Regression analysis</td>
      <td>Gene expression, Microarray experiment, RNA-Seq</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>471</td>
      <td>582</td>
      <td>35</td>
      <td>40</td>
      <td>589</td>
      <td>589</td>
      <td>80</td>
      <td>80</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1060</td>
      <td>1171</td>
      <td>115</td>
      <td>120</td>
    </tr>
    <tr>
      <td></td>
      <td>maxbin2</td>
      <td>maxbin2</td>
      <td>clusters metagenomic contigs into bins</td>
      <td>maxbin</td>
      <td>maxbin</td>
      <td></td>
      <td>MaxBin</td>
      <td>Software for binning assembled metagenomic sequences based on an Expectation-Maximization algorithm.</td>
      <td>Sequence assembly</td>
      <td>Metagenomics, Sequence assembly, Microbiology</td>
      <td>To update</td>
      <td>https://downloads.jbei.org/data/microbial_communities/MaxBin/MaxBin.html</td>
      <td>Metagenomics</td>
      <td>maxbin2</td>
      <td>2019-10-24</td>
      <td>mbernt</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/maxbin2/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/maxbin2</td>
      <td></td>
      <td>maxbin2</td>
      <td>2.2.7</td>
      <td>Sequence assembly</td>
      <td>Metagenomics, Sequence assembly, Microbiology</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2798</td>
      <td>2798</td>
      <td>314</td>
      <td>314</td>
      <td>2452</td>
      <td>2452</td>
      <td>508</td>
      <td>508</td>
      <td>914</td>
      <td>914</td>
      <td>89</td>
      <td>89</td>
      <td>6164</td>
      <td>6164</td>
      <td>911</td>
      <td>911</td>
    </tr>
    <tr>
      <td></td>
      <td>mcl</td>
      <td>mcl</td>
      <td>The Markov Cluster Algorithm, a cluster algorithm for graphs</td>
      <td>mcl</td>
      <td>mcl</td>
      <td></td>
      <td>MCL</td>
      <td>MCL is a clustering algorithm widely used in bioinformatics and gaining traction in other fields.</td>
      <td>Clustering, Network analysis, Gene regulatory network analysis</td>
      <td>Molecular interactions, pathways and networks</td>
      <td>Up-to-date</td>
      <td>https://micans.org/mcl/man/mcl.html</td>
      <td>Sequence Analysis</td>
      <td>mcl</td>
      <td>2022-05-12</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/mcl</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mcl</td>
      <td>22.282</td>
      <td>mcl</td>
      <td>22.282</td>
      <td>Clustering, Gene regulatory network analysis</td>
      <td>Molecular interactions, pathways and networks</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>35</td>
      <td>35</td>
      <td>13</td>
      <td>13</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>35</td>
      <td>35</td>
      <td>13</td>
      <td>13</td>
    </tr>
    <tr>
      <td></td>
      <td>medaka</td>
      <td>medaka_consensus, medaka_consensus_pipeline, medaka_snp, medaka_variant</td>
      <td>Sequence correction provided by ONT Research</td>
      <td>medaka</td>
      <td>medaka</td>
      <td></td>
      <td>Medaka</td>
      <td>medaka is a tool to create consensus sequences and variant calls from nanopore sequencing data. This task is performed using neural networks applied a pileup of individual sequencing reads against a draft assembly.</td>
      <td>Base-calling, Variant calling, Sequence assembly</td>
      <td>Sequence assembly, Machine learning</td>
      <td>To update</td>
      <td>https://github.com/nanoporetech/medaka</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>2020-08-24</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/medaka</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/medaka</td>
      <td>1.7.2</td>
      <td>medaka</td>
      <td>2.0.1</td>
      <td>Base-calling, Variant calling, Sequence assembly</td>
      <td>Sequence assembly, Machine learning</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>117975</td>
      <td>117975</td>
      <td>1693</td>
      <td>1693</td>
      <td>55063</td>
      <td>55063</td>
      <td>796</td>
      <td>796</td>
      <td>9325</td>
      <td>9325</td>
      <td>639</td>
      <td>639</td>
      <td>182363</td>
      <td>182363</td>
      <td>3128</td>
      <td>3128</td>
    </tr>
    <tr>
      <td></td>
      <td>megahit</td>
      <td>megahit</td>
      <td>An ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph.</td>
      <td>megahit</td>
      <td>megahit</td>
      <td></td>
      <td>MEGAHIT</td>
      <td>Single node assembler for large and complex metagenomics NGS reads, such as soil. It makes use of succinct de Bruijn graph to achieve low memory usage, whereas its goal is not to make memory usage as low as possible.</td>
      <td>Genome assembly</td>
      <td>Metagenomics, Sequencing, Ecology, Sequence assembly</td>
      <td>Up-to-date</td>
      <td>https://github.com/voutcn/megahit</td>
      <td>Sequence Analysis, Assembly, Metagenomics</td>
      <td>megahit</td>
      <td>2017-09-22</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/megahit</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/megahit</td>
      <td>1.2.9</td>
      <td>megahit</td>
      <td>1.2.9</td>
      <td>Genome assembly</td>
      <td>Metagenomics, Sequencing, Sequence assembly</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>12692</td>
      <td>13292</td>
      <td>1349</td>
      <td>1400</td>
      <td>16645</td>
      <td>16645</td>
      <td>2140</td>
      <td>2140</td>
      <td>6214</td>
      <td>6214</td>
      <td>431</td>
      <td>431</td>
      <td>35551</td>
      <td>36151</td>
      <td>3920</td>
      <td>3971</td>
    </tr>
    <tr>
      <td></td>
      <td>megahit_contig2fastg</td>
      <td>megahit_contig2fastg</td>
      <td>A subprogram within the Megahit toolkit for converting contigs to assembly graphs (fastg)</td>
      <td>megahit</td>
      <td>megahit</td>
      <td></td>
      <td>MEGAHIT</td>
      <td>Single node assembler for large and complex metagenomics NGS reads, such as soil. It makes use of succinct de Bruijn graph to achieve low memory usage, whereas its goal is not to make memory usage as low as possible.</td>
      <td>Genome assembly</td>
      <td>Metagenomics, Sequencing, Ecology, Sequence assembly</td>
      <td>To update</td>
      <td>https://github.com/voutcn/megahit/blob/master/tools/toolkit.cpp</td>
      <td>Sequence Analysis, Assembly, Metagenomics</td>
      <td>megahit_contig2fastg</td>
      <td>2018-11-09</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/megahit_contig2fastg</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/megahit_contig2fastg</td>
      <td>1.1.3</td>
      <td>megahit</td>
      <td>1.2.9</td>
      <td>Genome assembly</td>
      <td>Metagenomics, Sequencing, Sequence assembly</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>513</td>
      <td>557</td>
      <td>152</td>
      <td>155</td>
      <td>74</td>
      <td>74</td>
      <td>35</td>
      <td>35</td>
      <td>10</td>
      <td>10</td>
      <td>4</td>
      <td>4</td>
      <td>597</td>
      <td>641</td>
      <td>191</td>
      <td>194</td>
    </tr>
    <tr>
      <td></td>
      <td>megan</td>
      <td>megan_blast2lca, megan_blast2rma, megan_daa2info, megan_daa2rma, megan_daa_meganizer, megan_read_extractor, megan_sam2rma</td>
      <td>MEGAN Community Edition - Interactive exploration and analysis of large-scale microbiome sequencing data.  MEGAN is a tool for studying the taxonomic content of a set of DNA reads, typically collected in a metagenomics project.In a preprocessing step, a sequence alignment of all reads against a suitable database of reference DNA or proteinsequences must be performed to produce an input file for the program. MEGAN is suitable for DNA reads (metagenomedata), RNA reads (metatranscriptome data), peptide sequences (metaproteomics data) and, using a suitable synonymsfile that maps SILVA ids to taxon ids, for 16S rRNA data (amplicon sequencing).</td>
      <td>megan</td>
      <td>megan</td>
      <td></td>
      <td>MEGAN</td>
      <td>Metagenome Analysis Software - MEGAN (MEtaGenome ANalyzer) is a new computer program that allows laptop analysis of large metagenomic datasets. In a preprocessing step, the set of DNA reads (or contigs) is compared against databases of known sequences using BLAST or another comparison tool. MEGAN can then be used to compute and interactively explore the taxonomical content of the dataset, employing the NCBI taxonomy to summarize and order the results.</td>
      <td>Sequence analysis, Taxonomic classification</td>
      <td>Sequence analysis</td>
      <td>To update</td>
      <td>https://github.com/husonlab/megan-ce</td>
      <td>Sequence Analysis</td>
      <td>megan</td>
      <td>2021-11-24</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/megan</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/megan</td>
      <td>6.21.7</td>
      <td>megan</td>
      <td>6.25.9</td>
      <td>Sequence analysis, Taxonomic classification</td>
      <td>Sequence analysis</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>4130</td>
      <td>4130</td>
      <td>447</td>
      <td>447</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4130</td>
      <td>4130</td>
      <td>447</td>
      <td>447</td>
    </tr>
    <tr>
      <td></td>
      <td>meme</td>
      <td>meme_dreme, meme_fimo, meme_meme, meme_psp_gen</td>
      <td>The MEME Suite allows the biologist to discover novel motifs in collections of unaligned nucleotideor protein sequences, and to perform a wide variety of other motif-based analyses.</td>
      <td>meme_meme</td>
      <td>meme_fimo, meme_meme</td>
      <td></td>
      <td>meme_meme</td>
      <td>An algorithm that discovers one or more motifs in a collection of DNA or protein sequences by using the technique of expectation maximization to fit a two-component finite mixture model to the set of sequences.</td>
      <td>Nucleic acid feature detection, Protein feature detection, Statistical calculation</td>
      <td>Data mining, Sequence analysis, Genetic variation, Statistics and probability</td>
      <td>To update</td>
      <td>http://meme-suite.org/</td>
      <td>ChIP-seq</td>
      <td>meme</td>
      <td>2018-04-25</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/meme</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/meme</td>
      <td>5.5.6</td>
      <td>meme</td>
      <td>5.5.7</td>
      <td>Nucleic acid feature detection, Protein feature detection, Statistical calculation</td>
      <td>Data mining, Sequence analysis, Genetic variation, Statistics and probability</td>
      <td>3</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>20990</td>
      <td>22979</td>
      <td>762</td>
      <td>892</td>
      <td>10975</td>
      <td>25929</td>
      <td>1551</td>
      <td>3249</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>31965</td>
      <td>48908</td>
      <td>2313</td>
      <td>4141</td>
    </tr>
    <tr>
      <td></td>
      <td>meme_chip</td>
      <td>meme_chip</td>
      <td>Performs motif discovery, motif enrichment analysis and clustering on large nucleotide datasets.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://meme-suite.org/</td>
      <td>ChIP-seq</td>
      <td>meme_chip</td>
      <td>2018-03-15</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/meme_chip</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/meme_chip</td>
      <td>4.11.2</td>
      <td>graphicsmagick</td>
      <td>1.3.26</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>6413</td>
      <td>7000</td>
      <td>705</td>
      <td>743</td>
      <td>7260</td>
      <td>7260</td>
      <td>1171</td>
      <td>1171</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>13673</td>
      <td>14260</td>
      <td>1876</td>
      <td>1914</td>
    </tr>
    <tr>
      <td></td>
      <td>meningotype</td>
      <td>meningotype</td>
      <td>Assign sequence type to N. meningitidis genome assemblies</td>
      <td>meningotype</td>
      <td>meningotype</td>
      <td></td>
      <td>meningotype</td>
      <td>In silico typing of Neisseria meningitidis contigs.</td>
      <td>Genotyping, Multilocus sequence typing</td>
      <td>Microbiology, Genotype and phenotype</td>
      <td>Up-to-date</td>
      <td>https://github.com/MDU-PHL/meningotype</td>
      <td>Sequence Analysis</td>
      <td>meningotype</td>
      <td>2023-06-06</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/meningotype</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/meningotype</td>
      <td>0.8.5</td>
      <td>meningotype</td>
      <td>0.8.5</td>
      <td>Multilocus sequence typing</td>
      <td>Microbiology, Genotype and phenotype</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>merlin</td>
      <td>merlin</td>
      <td>Pedigree Analysis package</td>
      <td>merlin</td>
      <td>merlin</td>
      <td></td>
      <td>Merlin</td>
      <td>Can be used for parametric and non-parametric linkage analysis, regression-based linkage analysis or association analysis for quantitative traits, ibd and kinship estimation, haplotyping, error detection and simulation</td>
      <td>Haplotype mapping, Genetic mapping</td>
      <td>GWAS study, Mapping</td>
      <td>Up-to-date</td>
      <td>http://csg.sph.umich.edu/abecasis/Merlin/</td>
      <td>Variant Analysis</td>
      <td>merlin</td>
      <td>2020-04-15</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/merlin/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/merlin</td>
      <td>1.1.2</td>
      <td>merlin</td>
      <td>1.1.2</td>
      <td>Haplotype mapping</td>
      <td>GWAS study, Mapping</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>merqury</td>
      <td>merqury, merquryplot</td>
      <td>Merqury is a tool for evaluating genomes assemblies based of k-mer operations.</td>
      <td>merqury</td>
      <td>merqury</td>
      <td></td>
      <td>Merqury</td>
      <td>Reference-free quality, completeness, and phasing assessment for genome assemblies.Evaluate genome assemblies with k-mers and more.Often, genome assembly projects have illumina whole genome sequencing reads available for the assembled individual.Merqury provides a set of tools for this purpose.</td>
      <td>Genome assembly, k-mer counting, Scaffolding, Phasing, De-novo assembly</td>
      <td>Sequence assembly, Whole genome sequencing, Plant biology</td>
      <td>Up-to-date</td>
      <td>https://github.com/marbl/merqury</td>
      <td>Assembly</td>
      <td>merqury</td>
      <td>2021-04-18</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/merqury</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/merqury</td>
      <td>1.3</td>
      <td>merqury</td>
      <td>1.3</td>
      <td>Genome assembly, k-mer counting, Scaffolding, Phasing, De-novo assembly</td>
      <td>Sequence assembly, Whole genome sequencing, Plant biology</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>3150</td>
      <td>3150</td>
      <td>477</td>
      <td>477</td>
      <td>730</td>
      <td>730</td>
      <td>196</td>
      <td>196</td>
      <td>1183</td>
      <td>1183</td>
      <td>151</td>
      <td>151</td>
      <td>5063</td>
      <td>5063</td>
      <td>824</td>
      <td>824</td>
    </tr>
    <tr>
      <td></td>
      <td>meryl</td>
      <td>meryl_arithmetic_kmers, meryl_count_kmers, meryl_filter_kmers, meryl_groups_kmers, meryl_histogram_kmers, meryl_print, meryl_trio_mode</td>
      <td>Meryl a k-mer counter.</td>
      <td>meryl</td>
      <td>meryl</td>
      <td></td>
      <td>Meryl</td>
      <td>Meryl is a tool for counting and working with sets of k-mers that was originally developed for use in the Celera Assembler and has since been migrated and maintained as part of Canu.</td>
      <td>k-mer counting</td>
      <td>Whole genome sequencing, Genomics, Sequence analysis, Sequencing</td>
      <td>Up-to-date</td>
      <td>https://github.com/marbl/meryl</td>
      <td>Assembly</td>
      <td>meryl</td>
      <td>2021-04-05</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/meryl</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/meryl</td>
      <td>1.3</td>
      <td>merqury</td>
      <td>1.3</td>
      <td>k-mer counting</td>
      <td>Whole genome sequencing, Genomics, Sequence analysis</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>metabat2</td>
      <td>metabat2_jgi_summarize_bam_contig_depths, metabat2</td>
      <td>MetaBAT2 (Metagenome Binning based on Abundance and Tetranucleotide frequency) is an automated metagenome binningsoftware that integrates empirical probabilistic distances of genome abundance and tetranucleotide frequency.</td>
      <td>MetaBAT_2</td>
      <td>MetaBAT_2</td>
      <td></td>
      <td>MetaBAT 2</td>
      <td>an adaptive binning algorithm for robust and efficient genome reconstruction from metagenome assemblies | MetaBAT2 clusters metagenomic contigs into different "bins", each of which should correspond to a putative genome | MetaBAT2 uses nucleotide composition information and source strain abundance (measured by depth-of-coverage by aligning the reads to the contigs) to perform binning</td>
      <td>Read binning, Sequence assembly, Genome annotation</td>
      <td>Metagenomics, Sequence assembly, Metagenomic sequencing</td>
      <td>To update</td>
      <td>https://bitbucket.org/berkeleylab/metabat/src/master/</td>
      <td>Metagenomics</td>
      <td>metabat2</td>
      <td>2022-01-28</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/metabat2/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/metabat2</td>
      <td>2.15</td>
      <td>metabat2</td>
      <td>2.17</td>
      <td>Read binning, Sequence assembly, Genome annotation</td>
      <td>Metagenomics, Sequence assembly, Metagenomic sequencing</td>
      <td>2</td>
      <td>1</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>1</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>6135</td>
      <td>6135</td>
      <td>382</td>
      <td>382</td>
      <td>881</td>
      <td>881</td>
      <td>151</td>
      <td>151</td>
      <td>1094</td>
      <td>1094</td>
      <td>104</td>
      <td>104</td>
      <td>8110</td>
      <td>8110</td>
      <td>637</td>
      <td>637</td>
    </tr>
    <tr>
      <td></td>
      <td>metabuli</td>
      <td>metabuli_classify</td>
      <td>Classifying metagenomes by jointly analysing both DNA and amino acid (AA) sequences</td>
      <td>metabuli</td>
      <td>metabuli</td>
      <td></td>
      <td>metabuli</td>
      <td>Metabuli: specific and sensitive metagenomic classification via joint analysis of DNA and amino acid</td>
      <td>Taxonomic classification</td>
      <td>Taxonomy</td>
      <td>To update</td>
      <td>https://github.com/steineggerlab/Metabuli</td>
      <td>Sequence Analysis, Metagenomics</td>
      <td>metabuli</td>
      <td>2024-06-04</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/metabuli</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/metabuli</td>
      <td>1.0.5</td>
      <td>metabuli</td>
      <td>1.0.8</td>
      <td>Taxonomic classification</td>
      <td>Taxonomy</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>metaeuk</td>
      <td>metaeuk_easy_predict</td>
      <td>MetaEuk is a modular toolkit designed for large-scale gene discovery andannotation in eukaryotic metagenomic contigs. Metaeuk combines the fast andsensitive homology search capabilities of MMseqs2 with a dynamic programmingprocedure to recover optimal exons sets. It reduces redundancies in multiplediscoveries of the same gene and resolves conflicting gene predictions onthe same strand.</td>
      <td>MetaEuk</td>
      <td>MetaEuk</td>
      <td></td>
      <td>MetaEuk</td>
      <td>MetaEuk - sensitive, high-throughput gene discovery and annotation for large-scale eukaryotic metagenomics</td>
      <td>Homology-based gene prediction</td>
      <td>Metagenomics, Gene and protein families</td>
      <td>To update</td>
      <td>https://github.com/soedinglab/metaeuk</td>
      <td>Sequence Analysis, Genome annotation</td>
      <td></td>
      <td>2020-08-04</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/metaeuk</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/metaeuk</td>
      <td>7.bba0d80</td>
      <td>metaeuk</td>
      <td>6.a5d39d9</td>
      <td>Homology-based gene prediction</td>
      <td>Metagenomics, Gene and protein families</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>312</td>
      <td>312</td>
      <td>40</td>
      <td>40</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>312</td>
      <td>312</td>
      <td>40</td>
      <td>40</td>
    </tr>
    <tr>
      <td></td>
      <td>metagenomeseq</td>
      <td>metagenomeseq_normalizaton</td>
      <td>metagenomeSeq Normalization</td>
      <td>metagenomeseq</td>
      <td>metagenomeseq</td>
      <td></td>
      <td>metagenomeSeq</td>
      <td>Designed to determine features (be it Operational Taxanomic Unit (OTU), species, etc.) that are differentially abundant between two or more groups of multiple samples. It is designed to address the effects of both normalization and under-sampling of microbial communities on disease association detection and the testing of feature correlations.</td>
      <td>Sequence visualisation, Statistical calculation</td>
      <td>Metagenomics, Sequencing</td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/3.18/bioc/html/metagenomeSeq.html</td>
      <td>Metagenomics</td>
      <td>metagenomeseq_normalization</td>
      <td>2017-03-27</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/metagenomeseq</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/metagenomeseq</td>
      <td>1.16.0-0.0.1</td>
      <td>bioconductor-metagenomeseq</td>
      <td>1.43.0</td>
      <td>Sequence visualisation, Statistical calculation</td>
      <td>Metagenomics, Sequencing</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>556</td>
      <td>756</td>
      <td>17</td>
      <td>21</td>
      <td>310</td>
      <td>514</td>
      <td>35</td>
      <td>55</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>866</td>
      <td>1270</td>
      <td>52</td>
      <td>76</td>
    </tr>
    <tr>
      <td></td>
      <td>metaphlan</td>
      <td>customize_metaphlan_database, extract_metaphlan_database, merge_metaphlan_tables, metaphlan</td>
      <td>MetaPhlAn for Metagenomic Phylogenetic Analysis</td>
      <td>metaphlan</td>
      <td>metaphlan</td>
      <td></td>
      <td>MetaPhlAn</td>
      <td>Computational tool for profiling the composition of microbial communities from metagenomic shotgun sequencing data.</td>
      <td>Nucleic acid sequence analysis, Phylogenetic tree analysis</td>
      <td>Metagenomics, Phylogenomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/biobakery/MetaPhlAn</td>
      <td>Metagenomics</td>
      <td>metaphlan</td>
      <td>2021-04-13</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/metaphlan/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/metaphlan</td>
      <td>4.1.1</td>
      <td>metaphlan</td>
      <td>4.1.1</td>
      <td>Nucleic acid sequence analysis</td>
      <td>Metagenomics, Phylogenomics</td>
      <td>1</td>
      <td>2</td>
      <td>4</td>
      <td>4</td>
      <td>1</td>
      <td>2</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>16629</td>
      <td>16648</td>
      <td>1183</td>
      <td>1187</td>
      <td>3983</td>
      <td>3983</td>
      <td>572</td>
      <td>572</td>
      <td>4747</td>
      <td>4747</td>
      <td>278</td>
      <td>278</td>
      <td>25359</td>
      <td>25378</td>
      <td>2033</td>
      <td>2037</td>
    </tr>
    <tr>
      <td></td>
      <td>metawrapmg</td>
      <td>metawrapmg_binning</td>
      <td>A flexible pipeline for genome-resolved metagenomic data analysis</td>
      <td>metawrap</td>
      <td>metawrap</td>
      <td></td>
      <td>MetaWRAP</td>
      <td>MetaWRAP aims to be an easy-to-use metagenomic wrapper suite that accomplishes the core tasks of metagenomic analysis from start to finish: read quality control, assembly, visualization, taxonomic profiling, extracting draft genomes (binning), and functional annotation.</td>
      <td>Read binning, Sequence assembly, Genome annotation, Sequence trimming, Demultiplexing</td>
      <td>Whole genome sequencing, Metagenomic sequencing, Metagenomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/bxlab/metaWRAP</td>
      <td>Metagenomics</td>
      <td>metawrapmg_binning</td>
      <td>2024-04-11</td>
      <td>galaxy-australia</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/metawrapmg</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/metawrapmg</td>
      <td>1.3.0</td>
      <td>metawrap-mg</td>
      <td>1.3.0</td>
      <td>Read binning, Sequence assembly, Genome annotation, Sequence trimming, Demultiplexing</td>
      <td>Whole genome sequencing, Metagenomic sequencing, Metagenomics</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>142</td>
      <td>142</td>
      <td>32</td>
      <td>32</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>379</td>
      <td>379</td>
      <td>47</td>
      <td>47</td>
      <td>521</td>
      <td>521</td>
      <td>79</td>
      <td>79</td>
    </tr>
    <tr>
      <td></td>
      <td>mg_toolkit</td>
      <td>mg_toolkit_bulk_download, mg_toolkit_original_metadata</td>
      <td>Metagenomics toolkit enables scientists to download all of the sample metadata for a given study or sequence to a single csv file.</td>
      <td>mg-toolkit</td>
      <td>mg-toolkit</td>
      <td></td>
      <td>mg-toolkit</td>
      <td>Metagenomics toolkit enables scientists to download all of the sample metadata for a given study or sequence to a single csv file.</td>
      <td>Data retrieval</td>
      <td>Metagenomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/EBI-Metagenomics/emg-toolkit</td>
      <td>Metagenomics</td>
      <td>mg_toolkit</td>
      <td>2024-07-12</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mg_toolkit</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mg_toolkit</td>
      <td>0.10.4</td>
      <td>mg-toolkit</td>
      <td>0.10.4</td>
      <td>Data retrieval</td>
      <td>Metagenomics</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
    </tr>
    <tr>
      <td></td>
      <td>microsatbed</td>
      <td>microsatbed</td>
      <td>Select microsatellites for a bed file</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/lmdu/pytrf</td>
      <td>Sequence Analysis</td>
      <td>microsatbed</td>
      <td>2024-07-21</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/microsatbed</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/microsatbed</td>
      <td>1.3.3</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>228</td>
      <td>228</td>
      <td>4</td>
      <td>4</td>
      <td>18</td>
      <td>18</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>246</td>
      <td>246</td>
      <td>5</td>
      <td>5</td>
    </tr>
    <tr>
      <td></td>
      <td>migmap</td>
      <td>migmap</td>
      <td>mapper for full-length T- and B-cell repertoire sequencing</td>
      <td>MiGMAP</td>
      <td>MiGMAP</td>
      <td></td>
      <td>MiGMAP</td>
      <td>Mapper for full-length T- and B-cell repertoire sequencing.</td>
      <td>Sequence analysis, Read mapping</td>
      <td>Immunoproteins, genes and antigens, Sequence analysis</td>
      <td>Up-to-date</td>
      <td>https://github.com/mikessh/migmap</td>
      <td>RNA, Sequence Analysis</td>
      <td>migmap</td>
      <td>2017-09-29</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/migmap</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/migmap</td>
      <td>1.0.3</td>
      <td>migmap</td>
      <td>1.0.3</td>
      <td>Sequence analysis, Read mapping</td>
      <td>Sequence analysis</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1082</td>
      <td>1229</td>
      <td>25</td>
      <td>33</td>
      <td>1153</td>
      <td>1376</td>
      <td>70</td>
      <td>89</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2235</td>
      <td>2605</td>
      <td>95</td>
      <td>122</td>
    </tr>
    <tr>
      <td></td>
      <td>minia</td>
      <td>minia</td>
      <td>Short-read assembler based on a de Bruijn graph</td>
      <td>minia</td>
      <td>minia</td>
      <td></td>
      <td>Minia</td>
      <td>Short-read assembler based on a de Bruijn graph, capable of assembling a human genome on a desktop computer in a day.</td>
      <td>Genome assembly</td>
      <td>Sequence assembly</td>
      <td>Up-to-date</td>
      <td>https://gatb.inria.fr/software/minia/</td>
      <td>Assembly</td>
      <td>minia</td>
      <td>2020-04-08</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/minia</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/minia</td>
      <td>3.2.6</td>
      <td>minia</td>
      <td>3.2.6</td>
      <td>Genome assembly</td>
      <td>Sequence assembly</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>7116</td>
      <td>7116</td>
      <td>256</td>
      <td>256</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>278</td>
      <td>278</td>
      <td>60</td>
      <td>60</td>
      <td>7394</td>
      <td>7394</td>
      <td>316</td>
      <td>316</td>
    </tr>
    <tr>
      <td></td>
      <td>miniasm</td>
      <td>miniasm</td>
      <td>Miniasm - Ultrafast de novo assembly for long noisy reads (though having no consensus step)</td>
      <td>miniasm</td>
      <td>miniasm</td>
      <td></td>
      <td>miniasm</td>
      <td>Miniasm is a very fast OLC-based de novo assembler for noisy long reads. It takes all-vs-all read self-mappings (typically by minimap) as input and outputs an assembly graph in the GFA format.</td>
      <td>De-novo assembly</td>
      <td>Genomics, Sequence assembly</td>
      <td>To update</td>
      <td>https://github.com/lh3/miniasm</td>
      <td>Assembly</td>
      <td>miniasm</td>
      <td>2019-06-18</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/miniasm</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/miniasm</td>
      <td>0.3_r179</td>
      <td>miniasm</td>
      <td>0.3</td>
      <td>De-novo assembly</td>
      <td>Genomics, Sequence assembly</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>13357</td>
      <td>13479</td>
      <td>554</td>
      <td>562</td>
      <td>8416</td>
      <td>8416</td>
      <td>448</td>
      <td>448</td>
      <td>898</td>
      <td>898</td>
      <td>85</td>
      <td>85</td>
      <td>22671</td>
      <td>22793</td>
      <td>1087</td>
      <td>1095</td>
    </tr>
    <tr>
      <td></td>
      <td>minimap2</td>
      <td>minimap2</td>
      <td>A fast pairwise aligner for genomic and spliced nucleotide sequences</td>
      <td>minimap2</td>
      <td>minimap2</td>
      <td></td>
      <td>Minimap2</td>
      <td>Pairwise aligner for genomic and spliced nucleotide sequences</td>
      <td>Pairwise sequence alignment</td>
      <td>Mapping</td>
      <td>Up-to-date</td>
      <td>https://github.com/lh3/minimap2</td>
      <td>Next Gen Mappers</td>
      <td>minimap2</td>
      <td>2018-08-31</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/minimap2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/minimap2</td>
      <td>2.28</td>
      <td>minimap2</td>
      <td>2.28</td>
      <td>Pairwise sequence alignment</td>
      <td>Mapping</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>346530</td>
      <td>349075</td>
      <td>4323</td>
      <td>4419</td>
      <td>108535</td>
      <td>108535</td>
      <td>4075</td>
      <td>4075</td>
      <td>22331</td>
      <td>22331</td>
      <td>1166</td>
      <td>1166</td>
      <td>477396</td>
      <td>479941</td>
      <td>9564</td>
      <td>9660</td>
    </tr>
    <tr>
      <td></td>
      <td>miniprot</td>
      <td>miniprot, miniprot_index</td>
      <td>Align a protein sequence against a genome with affine gap penalty, splicing and frameshift.</td>
      <td>miniprot</td>
      <td>miniprot</td>
      <td></td>
      <td>miniprot</td>
      <td>Miniprot aligns a protein sequence against a genome with affine gap penalty, splicing and frameshift. It is primarily intended for annotating protein-coding genes in a new species using known genes from other species.</td>
      <td>Sequence alignment, Protein sequence analysis</td>
      <td>Sequence sites, features and motifs, Sequence analysis</td>
      <td>Up-to-date</td>
      <td>https://github.com/lh3/miniprot</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>2023-03-23</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/miniprot</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/miniprot</td>
      <td>0.13</td>
      <td>miniprot</td>
      <td>0.13</td>
      <td>Sequence alignment, Protein sequence analysis</td>
      <td>Sequence sites, features and motifs, Sequence analysis</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>1244</td>
      <td>1244</td>
      <td>86</td>
      <td>86</td>
      <td>24</td>
      <td>24</td>
      <td>15</td>
      <td>15</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1268</td>
      <td>1268</td>
      <td>101</td>
      <td>101</td>
    </tr>
    <tr>
      <td></td>
      <td>mirmachine</td>
      <td>mirmachine</td>
      <td>Tool to detect miRNA in genome sequences</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/sinanugur/MirMachine</td>
      <td>Sequence Analysis</td>
      <td>mirmachine</td>
      <td>2024-03-15</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mirmachine</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mirmachine</td>
      <td>0.2.13</td>
      <td>mirmachine</td>
      <td>0.2.13</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
    </tr>
    <tr>
      <td></td>
      <td>mirnature</td>
      <td>mirnature</td>
      <td>Computational detection of canonical microRNAs</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/Bierinformatik/miRNAture</td>
      <td>RNA, Sequence Analysis</td>
      <td>mirnature</td>
      <td>2022-12-11</td>
      <td>iuc</td>
      <td>https://github.com/Bierinformatik/miRNAture</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mirnature</td>
      <td>1.1</td>
      <td>mirnature</td>
      <td>1.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>18</td>
      <td>18</td>
      <td>9</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>18</td>
      <td>18</td>
      <td>9</td>
      <td>9</td>
    </tr>
    <tr>
      <td></td>
      <td>mitobim</td>
      <td>mitobim</td>
      <td>assemble mitochondrial genomes</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/chrishah/MITObim</td>
      <td>Assembly</td>
      <td>mitobim</td>
      <td>2020-12-29</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mitobim</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mitobim</td>
      <td>1.9.1</td>
      <td>mitobim</td>
      <td>1.9.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1117</td>
      <td>1117</td>
      <td>116</td>
      <td>116</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>295</td>
      <td>295</td>
      <td>25</td>
      <td>25</td>
      <td>1412</td>
      <td>1412</td>
      <td>141</td>
      <td>141</td>
    </tr>
    <tr>
      <td></td>
      <td>mitos</td>
      <td>mitos, mitos2</td>
      <td>de-novo annotation of metazoan mitochondrial genomes</td>
      <td>mitos</td>
      <td>mitos</td>
      <td></td>
      <td>MITOS</td>
      <td>De novo metazoan mitochondrial genome annotation.</td>
      <td>Genome annotation</td>
      <td>Zoology, Whole genome sequencing</td>
      <td>To update</td>
      <td>http://mitos.bioinf.uni-leipzig.de/</td>
      <td>Sequence Analysis</td>
      <td>mitos</td>
      <td>2020-02-18</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mitos</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mitos</td>
      <td>1.1.7</td>
      <td>mitos</td>
      <td>2.1.9</td>
      <td>Genome annotation</td>
      <td>Zoology, Whole genome sequencing</td>
      <td>1</td>
      <td>1</td>
      <td>2</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>1</td>
      <td>0</td>
      <td>191295</td>
      <td>191295</td>
      <td>1111</td>
      <td>1111</td>
      <td>42265</td>
      <td>42265</td>
      <td>320</td>
      <td>320</td>
      <td>10904</td>
      <td>10904</td>
      <td>83</td>
      <td>83</td>
      <td>244464</td>
      <td>244464</td>
      <td>1514</td>
      <td>1514</td>
    </tr>
    <tr>
      <td></td>
      <td>mlst</td>
      <td>mlst, mlst_list</td>
      <td>Scan contig files against PubMLST typing schemes</td>
      <td>mlst</td>
      <td>mlst</td>
      <td></td>
      <td>MLST</td>
      <td>Multi Locus Sequence Typing from an assembled genome or from a set of reads.</td>
      <td>Multilocus sequence typing</td>
      <td>Immunoproteins and antigens</td>
      <td>To update</td>
      <td>https://github.com/tseemann/mlst</td>
      <td>Sequence Analysis</td>
      <td>mlst</td>
      <td>2016-12-12</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mlst</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mlst</td>
      <td>2.22.0</td>
      <td>mlst</td>
      <td>2.23.0</td>
      <td>Multilocus sequence typing</td>
      <td>Immunoproteins and antigens</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>13300</td>
      <td>13471</td>
      <td>1480</td>
      <td>1501</td>
      <td>6532</td>
      <td>6532</td>
      <td>1167</td>
      <td>1167</td>
      <td>10334</td>
      <td>11841</td>
      <td>1083</td>
      <td>1300</td>
      <td>30166</td>
      <td>31844</td>
      <td>3730</td>
      <td>3968</td>
    </tr>
    <tr>
      <td></td>
      <td>moabs</td>
      <td>moabs</td>
      <td>MOABS for differential methylation analysis on Bisulfite sequencing data.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/sunnyisgalaxy/moabs</td>
      <td>Epigenetics</td>
      <td>moabs</td>
      <td>2019-07-28</td>
      <td>iuc</td>
      <td>https://github.com/sunnyisgalaxy/moabs</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/moabs</td>
      <td>1.3.4.6</td>
      <td>moabs</td>
      <td>1.3.9.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>339</td>
      <td>339</td>
      <td>56</td>
      <td>56</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>339</td>
      <td>339</td>
      <td>56</td>
      <td>56</td>
    </tr>
    <tr>
      <td></td>
      <td>mosdepth</td>
      <td>mosdepth</td>
      <td>fast and flexible BAM/CRAM depth calculation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/brentp/mosdepth</td>
      <td>SAM</td>
      <td>mosdepth</td>
      <td>2022-12-04</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mosdepth</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mosdepth</td>
      <td>0.3.8</td>
      <td>mosdepth</td>
      <td>0.3.8</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1278</td>
      <td>1278</td>
      <td>87</td>
      <td>87</td>
      <td>168</td>
      <td>168</td>
      <td>24</td>
      <td>24</td>
      <td>102</td>
      <td>102</td>
      <td>5</td>
      <td>5</td>
      <td>1548</td>
      <td>1548</td>
      <td>116</td>
      <td>116</td>
    </tr>
    <tr>
      <td></td>
      <td>mothur</td>
      <td>mothur_align_check, mothur_align_seqs, mothur_amova, mothur_anosim, mothur_bin_seqs, mothur_biom_info, mothur_chimera_bellerophon, mothur_chimera_ccode, mothur_chimera_check, mothur_chimera_perseus, mothur_chimera_pintail, mothur_chimera_slayer, mothur_chimera_uchime, mothur_chimera_vsearch, mothur_chop_seqs, mothur_classify_otu, mothur_classify_seqs, mothur_classify_tree, mothur_clearcut, mothur_cluster_classic, mothur_cluster_fragments, mothur_cluster_split, mothur_cluster, mothur_collect_shared, mothur_collect_single, mothur_consensus_seqs, mothur_cooccurrence, mothur_corr_axes, mothur_count_groups, mothur_count_seqs, mothur_create_database, mothur_degap_seqs, mothur_deunique_seqs, mothur_deunique_tree, mothur_dist_seqs, mothur_dist_shared, mothur_fastq_info, mothur_filter_seqs, mothur_filter_shared, mothur_get_communitytype, mothur_get_coremicrobiome, mothur_get_dists, mothur_get_group, mothur_get_groups, mothur_get_label, mothur_get_lineage, mothur_get_mimarkspackage, mothur_get_otulabels, mothur_get_otulist, mothur_get_oturep, mothur_get_otus, mothur_get_rabund, mothur_get_relabund, mothur_get_sabund, mothur_get_seqs, mothur_get_sharedseqs, mothur_heatmap_bin, mothur_heatmap_sim, mothur_homova, mothur_indicator, mothur_lefse, mothur_libshuff, mothur_list_otulabels, mothur_list_seqs, mothur_make_biom, mothur_make_contigs, mothur_make_design, mothur_make_fastq, mothur_make_group, mothur_make_lefse, mothur_make_lookup, mothur_make_shared, mothur_make_sra, mothur_mantel, mothur_merge_count, mothur_merge_files, mothur_merge_groups, mothur_merge_sfffiles, mothur_merge_taxsummary, mothur_metastats, mothur_mimarks_attributes, mothur_nmds, mothur_normalize_shared, mothur_otu_association, mothur_otu_hierarchy, mothur_pairwise_seqs, mothur_parse_list, mothur_parsimony, mothur_pca, mothur_pcoa, mothur_pcr_seqs, mothur_phylo_diversity, mothur_phylotype, mothur_pre_cluster, mothur_primer_design, mothur_rarefaction_shared, mothur_rarefaction_single, mothur_remove_dists, mothur_remove_groups, mothur_remove_lineage, mothur_remove_otulabels, mothur_remove_otus, mothur_remove_rare, mothur_remove_seqs, mothur_rename_seqs, mothur_reverse_seqs, mothur_screen_seqs, mothur_sens_spec, mothur_seq_error, mothur_sffinfo, mothur_shhh_flows, mothur_shhh_seqs, mothur_sort_seqs, mothur_split_abund, mothur_split_groups, mothur_sub_sample, mothur_summary_qual, mothur_summary_seqs, mothur_summary_shared, mothur_summary_single, mothur_summary_tax, mothur_taxonomy_to_krona, mothur_tree_shared, mothur_trim_flows, mothur_trim_seqs, mothur_unifrac_unweighted, mothur_unifrac_weighted, mothur_unique_seqs, mothur_venn</td>
      <td>Mothur wrappers</td>
      <td>mothur</td>
      <td>mothur</td>
      <td></td>
      <td>mothur</td>
      <td>Open-source, platform-independent, community-supported software for describing and comparing microbial communities</td>
      <td>DNA barcoding, Sequencing quality control, Sequence clustering, Taxonomic classification, Visualisation, Sequence read processing, Phylogenetic analysis</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Phylogeny</td>
      <td>To update</td>
      <td>https://www.mothur.org</td>
      <td>Metagenomics</td>
      <td>mothur</td>
      <td>2018-08-03</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mothur</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mothur</td>
      <td>1.0</td>
      <td>mothur</td>
      <td>1.48.0</td>
      <td>DNA barcoding, Sequencing quality control, Sequence clustering, Taxonomic classification, Visualisation, Sequence read processing, Phylogenetic analysis</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Phylogeny</td>
      <td>129</td>
      <td>129</td>
      <td>129</td>
      <td>129</td>
      <td>129</td>
      <td>129</td>
      <td>129</td>
      <td>129</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>128</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>126</td>
      <td>129</td>
      <td>129</td>
      <td>0</td>
      <td>297546</td>
      <td>326056</td>
      <td>37506</td>
      <td>41763</td>
      <td>565218</td>
      <td>682083</td>
      <td>72163</td>
      <td>89926</td>
      <td>167400</td>
      <td>182490</td>
      <td>26963</td>
      <td>28404</td>
      <td>1030164</td>
      <td>1190629</td>
      <td>136632</td>
      <td>160093</td>
    </tr>
    <tr>
      <td></td>
      <td>msaboot</td>
      <td>msaboot</td>
      <td>A multiple sequences alignment bootstrapping tool.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/phac-nml/msaboot</td>
      <td>Fasta Manipulation</td>
      <td>msaboot</td>
      <td>2018-02-12</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/msaboot</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/msaboot</td>
      <td>0.1.2</td>
      <td>msaboot</td>
      <td>0.1.2</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>864</td>
      <td>929</td>
      <td>91</td>
      <td>101</td>
      <td>1100</td>
      <td>1100</td>
      <td>175</td>
      <td>175</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1964</td>
      <td>2029</td>
      <td>266</td>
      <td>276</td>
    </tr>
    <tr>
      <td></td>
      <td>multigps</td>
      <td>multigps</td>
      <td>Analyzes collections of multi-condition ChIP-seq data.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://mahonylab.org/software/multigps/</td>
      <td>ChIP-seq</td>
      <td>multigps</td>
      <td>2017-03-06</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/multigps</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/multigps</td>
      <td>0.74.0</td>
      <td>fonts-conda-ecosystem</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>485</td>
      <td>871</td>
      <td>159</td>
      <td>298</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>485</td>
      <td>871</td>
      <td>159</td>
      <td>298</td>
    </tr>
    <tr>
      <td></td>
      <td>multigsea</td>
      <td>multigsea</td>
      <td>GSEA-based pathway enrichment analysis for multi-omics data</td>
      <td>multiGSEA</td>
      <td>multiGSEA</td>
      <td></td>
      <td>multiGSEA</td>
      <td>A GSEA-based pathway enrichment analysis for multi-omics data.multiGSEA: a GSEA-based pathway enrichment analysis for multi-omics data, BMC Bioinformatics 21, 561 (2020).Combining GSEA-based pathway enrichment with multi omics data integration.</td>
      <td>Gene-set enrichment analysis, Aggregation, Pathway analysis</td>
      <td>Metabolomics, Molecular interactions, pathways and networks, Proteomics, Transcriptomics, Small molecules</td>
      <td>Up-to-date</td>
      <td>https://bioconductor.org/packages/devel/bioc/html/multiGSEA.html</td>
      <td>Transcriptomics, Proteomics, Statistics</td>
      <td>multigsea</td>
      <td>2023-06-07</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/multigsea</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/multigsea</td>
      <td>1.12.0</td>
      <td>bioconductor-multigsea</td>
      <td>1.12.0</td>
      <td>Gene-set enrichment analysis, Aggregation, Pathway analysis</td>
      <td>Metabolomics, Molecular interactions, pathways and networks, Proteomics, Transcriptomics, Small molecules</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>106</td>
      <td>106</td>
      <td>25</td>
      <td>25</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>106</td>
      <td>106</td>
      <td>25</td>
      <td>25</td>
    </tr>
    <tr>
      <td></td>
      <td>multiqc</td>
      <td>multiqc</td>
      <td>MultiQC aggregates results from bioinformatics analyses across many samples into a single report</td>
      <td>multiqc</td>
      <td>multiqc</td>
      <td></td>
      <td>MultiQC</td>
      <td>MultiQC aggregates results from multiple bioinformatics analyses across many samples into a single report. It searches a given directory for analysis logs and compiles a HTML report. It's a general use tool, perfect for summarising the output from numerous bioinformatics tools.</td>
      <td>Validation, Sequencing quality control</td>
      <td>Sequencing, Bioinformatics, Sequence analysis, Genomics</td>
      <td>To update</td>
      <td>http://multiqc.info/</td>
      <td>Fastq Manipulation, Statistics, Visualization</td>
      <td>multiqc</td>
      <td>2020-04-28</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/multiqc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/multiqc</td>
      <td>1.24.1</td>
      <td>multiqc</td>
      <td>1.25.1</td>
      <td>Sequencing quality control</td>
      <td>Sequencing, Bioinformatics, Sequence analysis, Genomics</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>182221</td>
      <td>192389</td>
      <td>19343</td>
      <td>20494</td>
      <td>163961</td>
      <td>182413</td>
      <td>26449</td>
      <td>29758</td>
      <td>40663</td>
      <td>41855</td>
      <td>5974</td>
      <td>6157</td>
      <td>386845</td>
      <td>416657</td>
      <td>51766</td>
      <td>56409</td>
    </tr>
    <tr>
      <td></td>
      <td>mummer4</td>
      <td>mummer_delta_filter, mummer_dnadiff, mummer_mummer, mummer_mummerplot, mummer_nucmer, mummer_show_coords</td>
      <td>Mummer4 Tools</td>
      <td>mummer4</td>
      <td>mummer4</td>
      <td></td>
      <td>MUMmer4</td>
      <td>System for rapidly aligning large DNA sequences to one another.</td>
      <td>Multiple sequence alignment</td>
      <td>Sequence analysis, Human genetics</td>
      <td>Up-to-date</td>
      <td>https://github.com/mummer4/mummer</td>
      <td>Sequence Analysis</td>
      <td>mummer4</td>
      <td>2018-06-30</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mummer4</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mummer4</td>
      <td>4.0.0rc1</td>
      <td>mummer4</td>
      <td>4.0.0rc1</td>
      <td>Multiple sequence alignment</td>
      <td>Sequence analysis, Human genetics</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>21587</td>
      <td>22119</td>
      <td>1940</td>
      <td>2033</td>
      <td>29284</td>
      <td>29284</td>
      <td>3626</td>
      <td>3626</td>
      <td>8194</td>
      <td>8194</td>
      <td>545</td>
      <td>545</td>
      <td>59065</td>
      <td>59597</td>
      <td>6111</td>
      <td>6204</td>
    </tr>
    <tr>
      <td></td>
      <td>mykrobe</td>
      <td>mykrobe_predict</td>
      <td>Antibiotic resistance predictions</td>
      <td>Mykrobe</td>
      <td>Mykrobe</td>
      <td></td>
      <td>Mykrobe</td>
      <td>Antibiotic resistance prediction for Mycobacterium tuberculosis from genome sequence data with Mykrobe.Antibiotic resistance prediction in minutes.Table of Contents generated with DocToc.AMR prediction (Mykrobe predictor).Before attempting to install with bioconda, please ensure you have your channels set up as specified in the documentation. If you don't, you may run into issues with an older version of mykrobe being installed</td>
      <td>Antimicrobial resistance prediction, Variant calling, Genotyping, Sequence trimming</td>
      <td>Whole genome sequencing, Genotype and phenotype, Probes and primers, Genetic variation, Metagenomics</td>
      <td>To update</td>
      <td>https://github.com/Mykrobe-tools/mykrobe</td>
      <td>Sequence Analysis</td>
      <td>mykrobe</td>
      <td>2017-12-18</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mykrobe</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mykrobe</td>
      <td>0.10.0</td>
      <td>mykrobe</td>
      <td>0.13.0</td>
      <td>Antimicrobial resistance prediction, Variant calling, Genotyping, Sequence trimming</td>
      <td>Whole genome sequencing, Genotype and phenotype, Probes and primers, Genetic variation, Metagenomics</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>mzmine</td>
      <td>mzmine_batch</td>
      <td>mass-spectrometry data processing, with the main focus on LC-MS data</td>
      <td>mzmine</td>
      <td>mzmine</td>
      <td></td>
      <td>MZmine</td>
      <td>Toolbox for visualization and analysis of LC-MS data in netCDF or mzXML.</td>
      <td>Natural product identification, Standardisation and normalisation, Peptide database search, Deisotoping, Clustering, Filtering, Chromatographic alignment, Peak detection, Peptide identification, Chromatogram visualisation, Mass spectrum visualisation, Structure visualisation, Plotting, Heat map generation</td>
      <td>Proteomics, Metabolomics, Proteomics experiment, Small molecules</td>
      <td>Up-to-date</td>
      <td>http://mzmine.github.io/</td>
      <td>Metabolomics</td>
      <td>mzmine_batch</td>
      <td>2023-10-21</td>
      <td>iuc</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/blob/master/tools/mzmine/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mzmine</td>
      <td>3.9.0</td>
      <td>mzmine</td>
      <td>3.9.0</td>
      <td>Natural product identification, Standardisation and normalisation, Peptide database search, Deisotoping, Clustering, Filtering, Chromatographic alignment, Peak detection, Peptide identification, Chromatogram visualisation, Mass spectrum visualisation, Structure visualisation, Plotting, Heat map generation</td>
      <td>Proteomics, Metabolomics, Proteomics experiment, Small molecules</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>9</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>9</td>
      <td>5</td>
      <td>5</td>
    </tr>
    <tr>
      <td></td>
      <td>naltorfs</td>
      <td>bicodon_counts_from_fasta, codon_freq_from_bicodons, find_nested_alt_orfs</td>
      <td>nAlt-ORFs: Nested Alternate Open Reading Frames (nAltORFs)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/BlankenbergLab/nAltORFs</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>2022-04-11</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/naltorfs/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/naltorfs</td>
      <td>0.1.2</td>
      <td>naltorfs</td>
      <td>0.1.2</td>
      <td></td>
      <td></td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>204</td>
      <td>204</td>
      <td>75</td>
      <td>75</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>204</td>
      <td>204</td>
      <td>75</td>
      <td>75</td>
    </tr>
    <tr>
      <td></td>
      <td>nanocompore</td>
      <td>nanocompore_db, nanocompore_sampcomp</td>
      <td>Nanocompore compares 2 ONT nanopore direct RNA sequencing datasets from different experimental conditions expected to have a significant impact on RNA modifications. It is recommended to have at least 2 replicates per condition. For example one can use a control condition with a significantly reduced number of modifications such as a cell line for which a modification writing enzyme was knocked-down or knocked-out. Alternatively, on a smaller scale transcripts of interests could be synthesized in-vitro.</td>
      <td>Nanocompore</td>
      <td>Nanocompore</td>
      <td></td>
      <td>Nanocompore</td>
      <td>RNA modifications detection by comparative Nanopore direct RNA sequencing.RNA modifications detection from Nanopore dRNA-Seq data.Nanocompore identifies differences in ONT nanopore sequencing raw signal corresponding to RNA modifications by comparing 2 samples.Analyses performed for the nanocompore paper.Nanocompore compares 2 ONT nanopore direct RNA sequencing datasets from different experimental conditions expected to have a significant impact on RNA modifications. It is recommended to have at least 2 replicates per condition. For example one can use a control condition with a significantly reduced number of modifications such as a cell line for which a modification writing enzyme was knocked-down or knocked-out. Alternatively, on a smaller scale transcripts of interests could be synthesized in-vitro</td>
      <td>Post-translation modification site prediction, PolyA signal detection, Genotyping, k-mer counting</td>
      <td>Functional, regulatory and non-coding RNA, RNA-Seq, Gene transcripts, Transcriptomics, Transcription factors and regulatory sites</td>
      <td>To update</td>
      <td>https://nanocompore.rna.rocks/</td>
      <td>Sequence Analysis</td>
      <td>nanocompore</td>
      <td>2020-05-01</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/nanocompore</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/nanocompore</td>
      <td>1.0.0rc3.post2</td>
      <td>nanocompore</td>
      <td>1.0.4</td>
      <td>PolyA signal detection, Genotyping, k-mer counting</td>
      <td>Functional, regulatory and non-coding RNA, RNA-Seq, Gene transcripts, Transcriptomics, Transcription factors and regulatory sites</td>
      <td>0</td>
      <td>1</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>839</td>
      <td>839</td>
      <td>12</td>
      <td>12</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>269</td>
      <td>269</td>
      <td>5</td>
      <td>5</td>
      <td>1108</td>
      <td>1108</td>
      <td>17</td>
      <td>17</td>
    </tr>
    <tr>
      <td></td>
      <td>nanoplot</td>
      <td>nanoplot</td>
      <td>Plotting tool for long read sequencing data and alignments</td>
      <td>nanoplot</td>
      <td>nanoplot</td>
      <td></td>
      <td>NanoPlot</td>
      <td>NanoPlot is a tool with various visualizations of sequencing data in bam, cram, fastq, fasta or platform-specific TSV summaries, mainly intended for long-read sequencing from Oxford Nanopore Technologies and Pacific Biosciences</td>
      <td>Scatter plot plotting, Box-Whisker plot plotting</td>
      <td>Genomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/wdecoster/NanoPlot</td>
      <td>Visualization</td>
      <td>nanoplot</td>
      <td>2018-09-28</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/nanoplot/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/nanoplot</td>
      <td>1.43.0</td>
      <td>nanoplot</td>
      <td>1.43.0</td>
      <td>Scatter plot plotting, Box-Whisker plot plotting</td>
      <td>Genomics</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>99052</td>
      <td>100009</td>
      <td>5671</td>
      <td>5755</td>
      <td>40386</td>
      <td>40386</td>
      <td>3371</td>
      <td>3371</td>
      <td>17919</td>
      <td>17919</td>
      <td>2188</td>
      <td>2188</td>
      <td>157357</td>
      <td>158314</td>
      <td>11230</td>
      <td>11314</td>
    </tr>
    <tr>
      <td></td>
      <td>nanopolishcomp</td>
      <td>nanopolishcomp_eventaligncollapse, nanopolishcomp_freqmethcalculate</td>
      <td>NanopolishComp contains 2 modules. Eventalign_collapse collapses the raw file generated by nanopolish eventalign by kmers rather than by event. Freq_meth_calculate methylation frequency at genomic CpG sites from the output of nanopolish call-methylation.</td>
      <td>nanopolishcomp</td>
      <td>nanopolishcomp</td>
      <td></td>
      <td>NanopolishComp</td>
      <td>NanopolishComp is a Python3 package for downstream analyses of Nanopolish output files.It is a companion package for Nanopolish.</td>
      <td>Methylation analysis, Collapsing methods</td>
      <td>Sequence analysis, Sequencing, Genetic variation</td>
      <td>To update</td>
      <td>https://a-slide.github.io/NanopolishComp</td>
      <td>Sequence Analysis</td>
      <td>nanopolishcomp</td>
      <td>2020-04-27</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/nanopolishcomp</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/nanopolishcomp</td>
      <td>0.6.11</td>
      <td>nanopolishcomp</td>
      <td>0.6.12</td>
      <td>Methylation analysis, Collapsing methods</td>
      <td>Sequence analysis, Sequencing, Genetic variation</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>1091</td>
      <td>1091</td>
      <td>44</td>
      <td>44</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1091</td>
      <td>1091</td>
      <td>44</td>
      <td>44</td>
    </tr>
    <tr>
      <td></td>
      <td>ncbi_acc_download</td>
      <td>ncbi_acc_download</td>
      <td>Download sequences from GenBank/RefSeq by accession through the NCBI ENTREZ API</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/kblin/ncbi-acc-download</td>
      <td>Data Source</td>
      <td>ncbi_acc_download</td>
      <td>2019-11-07</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ncbi_acc_download</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ncbi_acc_download</td>
      <td>0.2.8</td>
      <td>ncbi-acc-download</td>
      <td>0.2.8</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>6873</td>
      <td>6873</td>
      <td>1664</td>
      <td>1664</td>
      <td>17651</td>
      <td>17651</td>
      <td>4513</td>
      <td>4513</td>
      <td>6228</td>
      <td>6228</td>
      <td>972</td>
      <td>972</td>
      <td>30752</td>
      <td>30752</td>
      <td>7149</td>
      <td>7149</td>
    </tr>
    <tr>
      <td></td>
      <td>ncbi_datasets</td>
      <td>datasets_download_gene, datasets_download_genome</td>
      <td>NCBI datasets downloads biological sequence data across all domains of life from NCBI.</td>
      <td>ncbi_datasets</td>
      <td>ncbi_datasets</td>
      <td></td>
      <td>NCBI Datasets</td>
      <td>NCBI Datasets is a new resource that lets you easily gather data from across NCBI databases. Find and download sequence, annotation, and metadata for genes and genomes using our command-line tools or web interface.</td>
      <td>Data handling, Sequence database search, Data retrieval</td>
      <td>Biological databases</td>
      <td>To update</td>
      <td>https://github.com/ncbi/datasets</td>
      <td>Data Source</td>
      <td>ncbi_datasets</td>
      <td>2022-06-23</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ncbi_datasets</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ncbi_datasets</td>
      <td>16.20.0</td>
      <td>ncbi-datasets-cli</td>
      <td></td>
      <td>Data handling, Sequence database search, Data retrieval</td>
      <td>Biological databases</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>4417</td>
      <td>4417</td>
      <td>1024</td>
      <td>1024</td>
      <td>6940</td>
      <td>6940</td>
      <td>2275</td>
      <td>2275</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>11358</td>
      <td>11358</td>
      <td>3300</td>
      <td>3300</td>
    </tr>
    <tr>
      <td></td>
      <td>ncbi_entrez_direct</td>
      <td>ncbi_entrez_direct_efetch, ncbi_entrez_direct_einfo, ncbi_entrez_direct_esearch</td>
      <td>NCBI Entrez Direct allow fetching data from NCBI Databases</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://www.ncbi.nlm.nih.gov/books/NBK179288/</td>
      <td>Data Source</td>
      <td>ncbi_entrez_direct</td>
      <td>2022-03-22</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ncbi_entrez_direct</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ncbi_entrez_direct</td>
      <td>22.4</td>
      <td>entrez-direct</td>
      <td>22.4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>2</td>
    </tr>
    <tr>
      <td></td>
      <td>ncbi_entrez_eutils</td>
      <td>ncbi_eutils_ecitmatch, ncbi_eutils_efetch, ncbi_eutils_einfo, ncbi_eutils_elink, ncbi_eutils_epost, ncbi_eutils_esearch, ncbi_eutils_esummary</td>
      <td>NCBI Entrez E-Utilties allow fetching data from NCBI Databases</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.ncbi.nlm.nih.gov/books/NBK25501/</td>
      <td>Data Source</td>
      <td>ncbi_entrez_eutils</td>
      <td>2020-08-07</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ncbi_entrez_eutils</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ncbi_entrez_eutils</td>
      <td>1.70</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>2794</td>
      <td>3191</td>
      <td>533</td>
      <td>664</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2794</td>
      <td>3191</td>
      <td>533</td>
      <td>664</td>
    </tr>
    <tr>
      <td></td>
      <td>ncbi_fcs_gx</td>
      <td>ncbi_fcs_gx</td>
      <td>FCS-GX detects contamination from foreign organisms in genome sequences using the genome cross-species aligner (GX).</td>
      <td>ncbi_fcs</td>
      <td>ncbi_fcs</td>
      <td></td>
      <td>NCBI fcs</td>
      <td>The NCBI Foreign Contamination Screen (FCS) is a tool suite for identifying and removing contaminant sequences in genome assemblies. Contaminants are defined as sequences in a dataset that do not originate from the biological source organism and can arise from a variety of environmental and laboratory sources. FCS will help you remove contaminants from genomes before submission to GenBank.</td>
      <td>Sequence assembly validation, Sequence trimming, Sequence contamination filtering</td>
      <td>Sequence analysis, Sequence assembly</td>
      <td>Up-to-date</td>
      <td>https://github.com/ncbi/fcs-gx</td>
      <td>Sequence Analysis</td>
      <td>ncbi_fcs_gx</td>
      <td>2023-11-14</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ncbi_fcs_gx</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ncbi_fcs_gx</td>
      <td>0.5.4</td>
      <td>ncbi-fcs-gx</td>
      <td>0.5.4</td>
      <td>Sequence assembly validation, Sequence trimming, Sequence contamination filtering</td>
      <td>Sequence analysis, Sequence assembly</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>125</td>
      <td>125</td>
      <td>26</td>
      <td>26</td>
      <td>2489</td>
      <td>2489</td>
      <td>139</td>
      <td>139</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2614</td>
      <td>2614</td>
      <td>165</td>
      <td>165</td>
    </tr>
    <tr>
      <td></td>
      <td>necat</td>
      <td>necat</td>
      <td>Error correction and de-novo assembly for ONT Nanopore reads</td>
      <td>necat</td>
      <td>necat</td>
      <td></td>
      <td>NECAT</td>
      <td>NECAT is an error correction and de-novo assembly tool for Nanopore long noisy reads.</td>
      <td>De-novo assembly</td>
      <td>Sequence assembly</td>
      <td>Up-to-date</td>
      <td>https://github.com/xiaochuanle/NECAT</td>
      <td>Assembly</td>
      <td>necat</td>
      <td>2021-11-29</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/necat</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/necat</td>
      <td>0.0.1_update20200803</td>
      <td>necat</td>
      <td>0.0.1_update20200803</td>
      <td>De-novo assembly</td>
      <td>Sequence assembly</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>790</td>
      <td>790</td>
      <td>160</td>
      <td>160</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>790</td>
      <td>790</td>
      <td>160</td>
      <td>160</td>
    </tr>
    <tr>
      <td></td>
      <td>newick_utils</td>
      <td>newick_display</td>
      <td>Perform operations on Newick trees</td>
      <td>newick_utilities</td>
      <td>newick_utilities</td>
      <td></td>
      <td>Newick Utilities</td>
      <td>The Newick Utilities are a set of command-line tools for processing phylogenetic trees. They can process arbitrarily large amounts of data and do not require user interaction, which makes them suitable for automating phylogeny processing tasks.</td>
      <td>Phylogenetic tree generation, Phylogenetic tree analysis, Phylogenetic tree reconstruction</td>
      <td>Phylogeny, Genomics, Computer science</td>
      <td>To update</td>
      <td>http://cegg.unige.ch/newick_utils</td>
      <td>Visualization, Metagenomics</td>
      <td>newick_utils</td>
      <td>2018-10-01</td>
      <td>iuc</td>
      <td>https://github.com/tjunier/newick_utils</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/newick_utils</td>
      <td>1.6+galaxy1</td>
      <td>newick_utils</td>
      <td>1.6</td>
      <td></td>
      <td>Phylogeny, Genomics, Computer science</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>33637</td>
      <td>34195</td>
      <td>1113</td>
      <td>1172</td>
      <td>16257</td>
      <td>16257</td>
      <td>1816</td>
      <td>1816</td>
      <td>7582</td>
      <td>7582</td>
      <td>899</td>
      <td>899</td>
      <td>57476</td>
      <td>58034</td>
      <td>3828</td>
      <td>3887</td>
    </tr>
    <tr>
      <td></td>
      <td>nextclade</td>
      <td>nextalign, nextclade</td>
      <td>Identify differences between your sequences and a reference sequence used by Nextstrain</td>
      <td>nextclade</td>
      <td>nextclade</td>
      <td></td>
      <td>Nextclade</td>
      <td>Nextclade is an open-source project for viral genome alignment, mutation calling, clade assignment, quality checks and phylogenetic placement.</td>
      <td>Methylation analysis, Variant calling</td>
      <td>Genomics, Sequence analysis, Cladistics</td>
      <td>To update</td>
      <td>https://github.com/nextstrain/nextclade</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>2021-04-26</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/nextclade</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/nextclade</td>
      <td>2.7.0</td>
      <td>nextalign</td>
      <td>2.14.0</td>
      <td>Methylation analysis, Variant calling</td>
      <td>Genomics, Cladistics</td>
      <td>1</td>
      <td>1</td>
      <td>2</td>
      <td>2</td>
      <td>1</td>
      <td>1</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>4411</td>
      <td>4411</td>
      <td>396</td>
      <td>396</td>
      <td>9708</td>
      <td>9708</td>
      <td>279</td>
      <td>279</td>
      <td>2099</td>
      <td>2099</td>
      <td>212</td>
      <td>212</td>
      <td>16218</td>
      <td>16218</td>
      <td>887</td>
      <td>887</td>
    </tr>
    <tr>
      <td></td>
      <td>ngmlr</td>
      <td>ngmlr</td>
      <td>CoNvex Gap-cost alignMents for Long Reads</td>
      <td>ngmlr</td>
      <td>ngmlr</td>
      <td></td>
      <td>NGMLR</td>
      <td>An algorithm to map third generation long-read sequencing data (PacBio and Oxford Nanopore) to a reference genome with a focus on reads that span structural variation.</td>
      <td>DNA mapping, Sequence alignment, Genetic variation analysis</td>
      <td>Sequencing, Mapping, DNA structural variation</td>
      <td>Up-to-date</td>
      <td>https://github.com/philres/ngmlr</td>
      <td>Next Gen Mappers</td>
      <td>ngmlr</td>
      <td>2020-10-27</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ngmlr</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ngmlr</td>
      <td>0.2.7</td>
      <td>ngmlr</td>
      <td>0.2.7</td>
      <td>DNA mapping, Sequence alignment, Genetic variation analysis</td>
      <td>Sequencing, Mapping</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1403</td>
      <td>1403</td>
      <td>49</td>
      <td>49</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1403</td>
      <td>1403</td>
      <td>49</td>
      <td>49</td>
    </tr>
    <tr>
      <td></td>
      <td>ngsutils</td>
      <td>ngsutils_bam_filter</td>
      <td>NGSUtils is a suite of software tools for working with next-generation sequencing datasets.Starting in 2009, we (Liu Lab @ Indiana University School of Medicine) starting working withnext-generation sequencing data. We initially started doing custom coding for each project in a one-off manner.It quickly became apparent that this was an inefficient manner to work, so we started assembling smallerutilities that could be adapted into larger, more complicated, workflows. We have used them for Illumia,SOLiD, 454, Ion Torrent, and Pac Bio sequencing data. We have used them for DNA and RNA resequcing,ChIP-Seq, CLIP-Seq, and targeted resequencing (Agilent exome capture and PCR targeting).These tools are also used heavily in our in-house DNA and RNA mapping pipelines.NGSUtils is made up of 50+ programs, mainly written in Python.These are separated into modules based on the type of file that is to be analyzed.</td>
      <td>ngsutils</td>
      <td>ngsutils</td>
      <td></td>
      <td>NGSUtils</td>
      <td>NGSUtils is a suite of software tools for working with next-generation sequencing datasets</td>
      <td>Read pre-processing, Sequencing quality control, Variant calling, Formatting, Sequence contamination filtering</td>
      <td>Genomics, Transcriptomics</td>
      <td>To update</td>
      <td>https://github.com/ngsutils/ngsutils</td>
      <td>SAM</td>
      <td></td>
      <td>2015-11-10</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ngsutils</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ngsutils</td>
      <td></td>
      <td>ngsutils</td>
      <td>0.5.9</td>
      <td>Read pre-processing, Variant calling, Formatting, Sequence contamination filtering</td>
      <td>Transcriptomics</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>24611</td>
      <td>29662</td>
      <td>633</td>
      <td>716</td>
      <td>15523</td>
      <td>21395</td>
      <td>740</td>
      <td>1128</td>
      <td>7751</td>
      <td>8972</td>
      <td>125</td>
      <td>145</td>
      <td>47885</td>
      <td>60029</td>
      <td>1498</td>
      <td>1989</td>
    </tr>
    <tr>
      <td></td>
      <td>nonpareil</td>
      <td>nonpareil</td>
      <td>Estimate average coverage in metagenomic datasets</td>
      <td>nonpareil</td>
      <td>nonpareil</td>
      <td></td>
      <td>nonpareil</td>
      <td>Estimate metagenomic coverage and sequence diversity</td>
      <td>Operation</td>
      <td></td>
      <td>To update</td>
      <td>http://nonpareil.readthedocs.io</td>
      <td>Metagenomics</td>
      <td>nonpareil</td>
      <td>2017-11-03</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/nonpareil</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/nonpareil</td>
      <td>3.1.1</td>
      <td>nonpareil</td>
      <td>3.5.5</td>
      <td>Operation</td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>107</td>
      <td>143</td>
      <td>13</td>
      <td>15</td>
      <td>148</td>
      <td>148</td>
      <td>20</td>
      <td>20</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>255</td>
      <td>291</td>
      <td>33</td>
      <td>35</td>
    </tr>
    <tr>
      <td></td>
      <td>novoplasty</td>
      <td>novoplasty</td>
      <td>NOVOPlasty is a de novo assembler and heteroplasmy/variance caller for short circular genomes.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ndierckx/NOVOPlasty</td>
      <td>Assembly</td>
      <td>novoplasty</td>
      <td>2020-05-18</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/novoplasty</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/novoplasty</td>
      <td>4.3.1</td>
      <td>novoplasty</td>
      <td>4.3.5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>7780</td>
      <td>7780</td>
      <td>337</td>
      <td>337</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>2323</td>
      <td>2323</td>
      <td>99</td>
      <td>99</td>
      <td>10104</td>
      <td>10104</td>
      <td>437</td>
      <td>437</td>
    </tr>
    <tr>
      <td></td>
      <td>nugen_nudup</td>
      <td>nugen_nudup</td>
      <td>Marks/removes PCR introduced duplicate molecules based on the molecular tagging technology used in NuGEN products.</td>
      <td>nudup</td>
      <td>nudup</td>
      <td></td>
      <td>NuDup</td>
      <td>Marks/removes duplicate molecules based on the molecular tagging technology used in Tecan products.</td>
      <td>Duplication detection</td>
      <td>Sequencing</td>
      <td>Up-to-date</td>
      <td>https://github.com/tecangenomics/nudup</td>
      <td>SAM, Metagenomics, Sequence Analysis, Transcriptomics</td>
      <td>nugen_nudup</td>
      <td>2016-11-18</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/nugen_nudup</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/nugen_nudup</td>
      <td>2.3.3</td>
      <td>nudup</td>
      <td>2.3.3</td>
      <td>Duplication detection</td>
      <td>Sequencing</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>obitools</td>
      <td>obi_illumina_pairend, obi_ngsfilter, obi_annotate, obi_clean, obi_convert, obi_grep, obi_sort, obi_stat, obi_tab, obi_uniq</td>
      <td>OBITools is a set of programs developed to simplify the manipulation of sequence files</td>
      <td>obitools</td>
      <td>obitools</td>
      <td></td>
      <td>OBITools</td>
      <td>Set of python programs developed to simplify the manipulation of sequence files. They were mainly designed to help us for analyzing Next Generation Sequencer outputs (454 or Illumina) in the context of DNA Metabarcoding.</td>
      <td>Sequence analysis, Sequence analysis</td>
      <td>Sequence analysis, DNA, Sequencing</td>
      <td>Up-to-date</td>
      <td>http://metabarcoding.org/obitools</td>
      <td>Sequence Analysis</td>
      <td>obitools</td>
      <td>2017-03-23</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/obitools</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/obitools</td>
      <td>1.2.13</td>
      <td>obitools</td>
      <td>1.2.13</td>
      <td>Sequence analysis, Sequence analysis</td>
      <td>Sequence analysis, DNA, Sequencing</td>
      <td>0</td>
      <td>10</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>10</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>37598</td>
      <td>37598</td>
      <td>1277</td>
      <td>1277</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3862</td>
      <td>3862</td>
      <td>280</td>
      <td>280</td>
      <td>41460</td>
      <td>41460</td>
      <td>1557</td>
      <td>1557</td>
    </tr>
    <tr>
      <td></td>
      <td>ococo</td>
      <td>ococo</td>
      <td>Variant detection of SNVs</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/karel-brinda/ococo</td>
      <td>Variant Analysis</td>
      <td>ococo</td>
      <td>2017-12-07</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ococo</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ococo</td>
      <td>0.1.2.6</td>
      <td>ococo</td>
      <td>0.1.2.7</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1850</td>
      <td>1963</td>
      <td>79</td>
      <td>93</td>
      <td>1370</td>
      <td>1370</td>
      <td>194</td>
      <td>194</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3220</td>
      <td>3333</td>
      <td>273</td>
      <td>287</td>
    </tr>
    <tr>
      <td></td>
      <td>odgi</td>
      <td>odgi_build, odgi_viz</td>
      <td>Representing large genomic variation graphs with minimal memory overhead requires a careful encoding of the graph entities. odgi follows the dynamic GBWT in developing a byte-packed version of the graph and paths through it.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/vgteam/odgi</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>2020-04-06</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/odgi/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/odgi</td>
      <td>0.3</td>
      <td>odgi</td>
      <td>0.9.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>26</td>
      <td>26</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>26</td>
      <td>26</td>
      <td>6</td>
      <td>6</td>
    </tr>
    <tr>
      <td></td>
      <td>omark</td>
      <td>omark</td>
      <td>Proteome quality assessment software</td>
      <td>omark</td>
      <td>omark</td>
      <td></td>
      <td>OMArk</td>
      <td>Proteome quality assessment software</td>
      <td>Sequence assembly validation, Differential protein expression profiling</td>
      <td>Proteomics, Sequence analysis, Statistics and probability</td>
      <td>To update</td>
      <td>https://github.com/DessimozLab/OMArk</td>
      <td>Sequence Analysis</td>
      <td>omark</td>
      <td>2023-11-15</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/omark/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/omark</td>
      <td>0.3.0</td>
      <td></td>
      <td></td>
      <td>Sequence assembly validation, Differential protein expression profiling</td>
      <td>Proteomics, Sequence analysis, Statistics and probability</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>158</td>
      <td>158</td>
      <td>12</td>
      <td>12</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>158</td>
      <td>158</td>
      <td>12</td>
      <td>12</td>
    </tr>
    <tr>
      <td></td>
      <td>ont_fast5_api</td>
      <td>ont_fast5_api_compress_fast5, ont_fast5_api_fast5_subset, ont_fast5_api_multi_to_single_fast5, ont_fast5_api_single_to_multi_fast5</td>
      <td>ont_fast5_api is a simple interface to HDF5 files of the Oxford Nanopore FAST5 file format.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/nanoporetech/ont_fast5_api/</td>
      <td>Nanopore</td>
      <td>ont_fast5_api</td>
      <td>2020-06-08</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ont_fast5_api</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ont_fast5_api</td>
      <td>3.1.3</td>
      <td>ont-fast5-api</td>
      <td>4.1.3</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>3051</td>
      <td>3051</td>
      <td>87</td>
      <td>87</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3051</td>
      <td>3051</td>
      <td>87</td>
      <td>87</td>
    </tr>
    <tr>
      <td></td>
      <td>onto_toolkit</td>
      <td>onto_tk_get_ancestor_terms, onto_tk_get_child_terms, onto_tk_get_descendent_terms, onto_tk_get_parent_terms, onto_tk_get_parent_terms_by_relationship_type, onto_tk_get_relationship_id_vs_relationship_def, onto_tk_get_relationship_id_vs_relationship_name, onto_tk_get_relationship_id_vs_relationship_namespace, onto_tk_get_relationship_types, onto_tk_get_root_terms, onto_tk_get_subontology_from, onto_tk_term_id_vs_term_def, onto_tk_term_id_vs_term_name, onto_tk_get_term_synonyms, onto_tk_get_terms, onto_tk_get_terms_by_relationship_type, onto_tk_obo2owl, onto_tk_obo2rdf, onto_tk_term_id_vs_term_def</td>
      <td>ONTO-Toolkit is a collection of tools for managing ontologies.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://search.cpan.org/~easr/ONTO-PERL-1.45/</td>
      <td>Ontology Manipulation</td>
      <td>onto_toolkit</td>
      <td>2017-11-04</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/onto-toolkit</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/onto_toolkit</td>
      <td>1.45</td>
      <td>perl-onto-perl</td>
      <td>1.45</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>17</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>17</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>17</td>
      <td>17</td>
      <td>0</td>
      <td>582</td>
      <td>663</td>
      <td>68</td>
      <td>69</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>582</td>
      <td>663</td>
      <td>68</td>
      <td>69</td>
    </tr>
    <tr>
      <td></td>
      <td>optdoe</td>
      <td>optdoe</td>
      <td>Optimal Design Of Experiment</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/pablocarb/doebase</td>
      <td>Synthetic Biology</td>
      <td>optdoe</td>
      <td>2022-10-27</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/optdoe</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/optdoe</td>
      <td>v2.0.2</td>
      <td>doebase</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>optitype</td>
      <td>optitype</td>
      <td>Precision HLA typing from NGS data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/FRED-2/OptiType</td>
      <td>Sequence Analysis</td>
      <td>optitype</td>
      <td>2021-02-17</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/optitype1</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/optitype</td>
      <td>1.3.5</td>
      <td>optitype</td>
      <td>1.3.5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>500</td>
      <td>500</td>
      <td>45</td>
      <td>45</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>500</td>
      <td>500</td>
      <td>45</td>
      <td>45</td>
    </tr>
    <tr>
      <td></td>
      <td>orfipy</td>
      <td>orfipy</td>
      <td>Galaxy wrapper for ORFIPY</td>
      <td>orfipy</td>
      <td>orfipy</td>
      <td></td>
      <td>orfipy</td>
      <td>A fast and flexible tool for extracting ORFs.orfipy is a tool written in python/cython to extract ORFs in extremely an fast and flexible manner. Other popular ORF searching tools are OrfM and getorf. Compared to OrfM and getorf, orfipy provides the most options to fine tune ORF searches. orfipy uses multiple CPU cores and is particularly faster for data containing multiple smaller fasta sequences such as de-novo transcriptome assemblies. Please read the preprint here.</td>
      <td>Coding region prediction, Database search, Transcriptome assembly, De-novo assembly</td>
      <td>Computer science, RNA-Seq, Transcriptomics, Small molecules</td>
      <td>Up-to-date</td>
      <td>https://github.com/urmi-21/orfipy</td>
      <td>Sequence Analysis</td>
      <td>orfipy</td>
      <td>2022-04-07</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/orfipy</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/orfipy</td>
      <td>0.0.4</td>
      <td>orfipy</td>
      <td>0.0.4</td>
      <td>Coding region prediction, Database search, Transcriptome assembly, De-novo assembly</td>
      <td>Computer science, RNA-Seq, Transcriptomics, Small molecules</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>897</td>
      <td>897</td>
      <td>99</td>
      <td>99</td>
      <td>1792</td>
      <td>1792</td>
      <td>239</td>
      <td>239</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2689</td>
      <td>2689</td>
      <td>338</td>
      <td>338</td>
    </tr>
    <tr>
      <td></td>
      <td>orthofinder</td>
      <td>orthofinder_onlygroups</td>
      <td>Accurate inference of orthologous gene groups made easy</td>
      <td>OrthoFinder</td>
      <td>OrthoFinder</td>
      <td></td>
      <td>OrthoFinder</td>
      <td>OrthoFinder is a fast, accurate and comprehensive platform for comparative genomics. It finds orthogroups and orthologs, infers rooted gene trees for all orthogroups and identifies all of the gene duplcation events in those gene trees. It also infers a rooted species tree for the species being analysed and maps the gene duplication events from the gene trees to branches in the species tree. OrthoFinder also provides comprehensive statistics for comparative genomic analyses.</td>
      <td>Genome comparison, Phylogenetic tree generation (from molecular sequences), Phylogenetic tree analysis, Genome alignment</td>
      <td>Phylogenetics, Phylogenomics, Bioinformatics, Comparative genomics, Sequence analysis</td>
      <td>To update</td>
      <td>https://github.com/davidemms/OrthoFinder</td>
      <td>Phylogenetics, Sequence Analysis</td>
      <td>orthofinder</td>
      <td>2017-08-30</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/orthofinder</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/orthofinder</td>
      <td>2.5.5</td>
      <td>orthofinder</td>
      <td>3.0.1b1</td>
      <td>Genome comparison, Genome alignment</td>
      <td>Phylogenetics, Phylogenomics, Bioinformatics, Comparative genomics</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>2018</td>
      <td>2018</td>
      <td>419</td>
      <td>419</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>438</td>
      <td>438</td>
      <td>58</td>
      <td>58</td>
      <td>2456</td>
      <td>2456</td>
      <td>477</td>
      <td>477</td>
    </tr>
    <tr>
      <td></td>
      <td>packaged_annotation_loader</td>
      <td>packaged_annotation_loader</td>
      <td>Tool to make cached genome annotation data available as a list of datasets collection</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/packaged_annotation_loader</td>
      <td>Data Source</td>
      <td>packaged_annotation_loader</td>
      <td>2022-01-04</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/packaged_annotation_loader</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/packaged_annotation_loader</td>
      <td>0.1</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>pacu</td>
      <td>pacu_map, pacu_snp</td>
      <td>PACU is a workflow for whole genome sequencing based phylogeny of Illumina and ONT R9/R10 data.</td>
      <td>pacu</td>
      <td>pacu</td>
      <td></td>
      <td>PACU</td>
      <td>PACU is a workflow for whole genome sequencing based phylogeny of Illumina and ONT R9/R10 data.PACU stands for the Prokaryotic Awesome variant Calling Utility and is named after an omnivorous fish (that eats both Illumina and ONT reads).</td>
      <td>Clustering</td>
      <td>Phylogenetics, Sequence analysis</td>
      <td>Up-to-date</td>
      <td>https://github.com/BioinformaticsPlatformWIV-ISP/PACU</td>
      <td>Sequence Analysis, Phylogenetics</td>
      <td>pacu</td>
      <td>2024-08-13</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pacu</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pacu</td>
      <td>0.0.5</td>
      <td>pacu_snp</td>
      <td>0.0.5</td>
      <td>Clustering</td>
      <td>Phylogenetics</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>pairtools</td>
      <td>pairtools_dedup, pairtools_parse, pairtools_sort, pairtools_split, pairtools_stats</td>
      <td>Flexible tools for Hi-C data processing</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://pairtools.readthedocs.io</td>
      <td>Sequence Analysis</td>
      <td>pairtools</td>
      <td>2024-03-26</td>
      <td>iuc</td>
      <td>https://github.com/open2c/pairtools</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pairtools</td>
      <td>1.1.0</td>
      <td>pairtools</td>
      <td>1.1.0</td>
      <td></td>
      <td></td>
      <td>5</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>196</td>
      <td>196</td>
      <td>16</td>
      <td>16</td>
      <td>19</td>
      <td>19</td>
      <td>14</td>
      <td>14</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>215</td>
      <td>215</td>
      <td>30</td>
      <td>30</td>
    </tr>
    <tr>
      <td></td>
      <td>pangolin</td>
      <td>pangolin</td>
      <td>Pangolin assigns SARS-CoV-2 genome sequences their most likely lineages under the Pango nomenclature system.</td>
      <td>pangolin_cov-lineages</td>
      <td>pangolin_cov-lineages</td>
      <td></td>
      <td>pangolin</td>
      <td>Phylogenetic Assignment of Named Global Outbreak LINeages - software package for assigning SARS-CoV-2 genome sequences to global lineages</td>
      <td>Tree-based sequence alignment, Variant classification</td>
      <td>Virology</td>
      <td>Up-to-date</td>
      <td>https://github.com/cov-lineages/pangolin</td>
      <td>Sequence Analysis</td>
      <td>pangolin</td>
      <td>2021-04-25</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pangolin</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pangolin</td>
      <td>4.3</td>
      <td>pangolin</td>
      <td>4.3</td>
      <td>Tree-based sequence alignment, Variant classification</td>
      <td>Virology</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>7457</td>
      <td>7457</td>
      <td>508</td>
      <td>508</td>
      <td>8518</td>
      <td>8518</td>
      <td>377</td>
      <td>377</td>
      <td>2760</td>
      <td>2760</td>
      <td>259</td>
      <td>259</td>
      <td>18735</td>
      <td>18735</td>
      <td>1144</td>
      <td>1144</td>
    </tr>
    <tr>
      <td></td>
      <td>parse_mito_blast</td>
      <td>parse_mito_blast</td>
      <td>Filtering blast out from querying assembly against mitochondrial database.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://raw.githubusercontent.com/VGP/vgp-assembly/master/galaxy_tools/parse_mito_blast/parse_mito_blast.py</td>
      <td>Sequence Analysis</td>
      <td>parse_mito_blast</td>
      <td>2022-05-12</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/parse_mito_blast</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/parse_mito_blast</td>
      <td>1.0.2</td>
      <td>parse_mito_blast</td>
      <td>1.0.2</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>203</td>
      <td>203</td>
      <td>65</td>
      <td>65</td>
      <td>288</td>
      <td>288</td>
      <td>62</td>
      <td>62</td>
      <td>110</td>
      <td>110</td>
      <td>20</td>
      <td>20</td>
      <td>601</td>
      <td>601</td>
      <td>147</td>
      <td>147</td>
    </tr>
    <tr>
      <td></td>
      <td>pathview</td>
      <td>pathview</td>
      <td>Pathview is a tool set for pathway based data integration and visualization.</td>
      <td>pathview</td>
      <td>pathview</td>
      <td></td>
      <td>pathview</td>
      <td>Tool set for pathway based data integration and visualization that maps and renders a wide variety of biological data on relevant pathway graphs. It downloads the pathway graph data, parses the data file, maps user data to the pathway, and render pathway graph with the mapped data. In addition, it integrates with pathway and gene set (enrichment) analysis tools for large-scale and fully automated analysis.</td>
      <td>Pathway or network analysis, Pathway or network visualisation</td>
      <td>Molecular interactions, pathways and networks, Systems biology, Data visualisation</td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/pathview.html</td>
      <td>Statistics, RNA, Micro-array Analysis</td>
      <td>pathview</td>
      <td>2019-08-26</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pathview</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pathview</td>
      <td>1.34.0</td>
      <td>bioconductor-pathview</td>
      <td>1.42.0</td>
      <td>Pathway or network analysis, Pathway or network visualisation</td>
      <td>Molecular interactions, pathways and networks, Systems biology, Data visualisation</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>6204</td>
      <td>6211</td>
      <td>1261</td>
      <td>1261</td>
      <td>6667</td>
      <td>6667</td>
      <td>1143</td>
      <td>1143</td>
      <td>1160</td>
      <td>1160</td>
      <td>189</td>
      <td>189</td>
      <td>14031</td>
      <td>14038</td>
      <td>2593</td>
      <td>2593</td>
    </tr>
    <tr>
      <td></td>
      <td>pbgcpp</td>
      <td>pbgcpp</td>
      <td>Compute genomic consensus and call variants using PacBio reads mapped to a reference</td>
      <td>genomicconsensus</td>
      <td>genomicconsensus</td>
      <td></td>
      <td>GenomicConsensus</td>
      <td>The GenomicConsensus package provides the variantCaller tool, which allows you to apply the Quiver or Arrow algorithm to mapped PacBio reads to get consensus and variant calls.</td>
      <td>Variant calling</td>
      <td>Mapping</td>
      <td>Up-to-date</td>
      <td>https://github.com/PacificBiosciences/gcpp</td>
      <td>Variant Analysis</td>
      <td>pbgcpp</td>
      <td>2022-03-14</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pbgcpp</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pbgcpp</td>
      <td>2.0.2</td>
      <td>pbgcpp</td>
      <td>2.0.2</td>
      <td>Variant calling</td>
      <td>Mapping</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>pbmm2</td>
      <td>pbmm2</td>
      <td>A minimap2 SMRT wrapper for PacBio data.</td>
      <td>pbmm2</td>
      <td>pbmm2</td>
      <td></td>
      <td>pbmm2</td>
      <td>pbmm2 is a SMRT C++ wrapper for minimap2's C API. Its purpose is to support native PacBio in- and output, provide sets of recommended parameters, generate sorted output on-the-fly, and postprocess alignments. Sorted output can be used directly for polishing using GenomicConsensus, if BAM has been used as input to pbmm2. Benchmarks show that pbmm2 outperforms BLASR in sequence identity, number of mapped bases, and especially runtime. pbmm2 is the official replacement for BLASR.</td>
      <td>Pairwise sequence alignment, Sorting</td>
      <td>Mapping</td>
      <td>Up-to-date</td>
      <td>https://github.com/PacificBiosciences/pbmm2</td>
      <td>Next Gen Mappers</td>
      <td>pbmm2</td>
      <td>2022-03-23</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pbmm2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pbmm2</td>
      <td>1.14.99</td>
      <td>pbmm2</td>
      <td>1.14.99</td>
      <td>Pairwise sequence alignment, Sorting</td>
      <td>Mapping</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>70</td>
      <td>70</td>
      <td>17</td>
      <td>17</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>70</td>
      <td>70</td>
      <td>17</td>
      <td>17</td>
    </tr>
    <tr>
      <td></td>
      <td>pbtk</td>
      <td>bam2fastx</td>
      <td>Convert PacBio Bam File to fasta or fastq file</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/PacificBiosciences/pbtk</td>
      <td>Convert Formats, Fasta Manipulation, Fastq Manipulation</td>
      <td>bam2fastx</td>
      <td>2024-03-11</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pbtk</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pbtk</td>
      <td>3.1.1</td>
      <td>pbtk</td>
      <td>3.1.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>58</td>
      <td>58</td>
      <td>22</td>
      <td>22</td>
      <td>109</td>
      <td>109</td>
      <td>19</td>
      <td>19</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>167</td>
      <td>167</td>
      <td>41</td>
      <td>41</td>
    </tr>
    <tr>
      <td></td>
      <td>pe_histogram</td>
      <td>pe_histogram</td>
      <td>Contains a tool that produces an insert size histogram for a paired-end BAM file.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/seqcode/cegr-tools/tree/master/src/org/seqcode/cegrtools/pehistogram</td>
      <td>Graphics</td>
      <td>pe_histogram</td>
      <td>2016-06-16</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pe_histogram</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pe_histogram</td>
      <td>1.0.2</td>
      <td>openjdk</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>5505</td>
      <td>5505</td>
      <td>510</td>
      <td>510</td>
      <td>7932</td>
      <td>8582</td>
      <td>811</td>
      <td>909</td>
      <td>736</td>
      <td>736</td>
      <td>119</td>
      <td>119</td>
      <td>14173</td>
      <td>14823</td>
      <td>1440</td>
      <td>1538</td>
    </tr>
    <tr>
      <td></td>
      <td>peakzilla</td>
      <td>peakzilla</td>
      <td>Peakzilla identifies sites of enrichment and transcription factor binding sites from ChIP-seq and ChIP-exo experiments.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/steinmann/peakzilla</td>
      <td>ChIP-seq</td>
      <td>peakzilla</td>
      <td>2024-02-25</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/peakzilla</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/peakzilla</td>
      <td>1.0</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>18</td>
      <td>18</td>
      <td>9</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>18</td>
      <td>18</td>
      <td>9</td>
      <td>9</td>
    </tr>
    <tr>
      <td></td>
      <td>pear</td>
      <td>iuc_pear</td>
      <td>PEAR evaluates all possible paired-end read overlaps</td>
      <td>pear</td>
      <td>pear</td>
      <td></td>
      <td>PEAR</td>
      <td>Paired-end read merger. PEAR evaluates all possible paired-end read overlaps without requiring the target fragment size as input. In addition, it implements a statistical test for minimizing false-positive results.</td>
      <td>Sequence merging</td>
      <td>Sequence assembly</td>
      <td>Up-to-date</td>
      <td>https://cme.h-its.org/exelixis/web/software/pear/</td>
      <td>Fastq Manipulation</td>
      <td>pear</td>
      <td>2015-02-11</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pear</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pear</td>
      <td>0.9.6</td>
      <td>pear</td>
      <td>0.9.6</td>
      <td>Sequence merging</td>
      <td>Sequence assembly</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>6269</td>
      <td>6866</td>
      <td>260</td>
      <td>292</td>
      <td>5530</td>
      <td>20068</td>
      <td>467</td>
      <td>1664</td>
      <td>5647</td>
      <td>8471</td>
      <td>198</td>
      <td>300</td>
      <td>17446</td>
      <td>35405</td>
      <td>925</td>
      <td>2256</td>
    </tr>
    <tr>
      <td></td>
      <td>pharokka</td>
      <td>pharokka</td>
      <td>rapid standardised annotation tool for bacteriophage genomes and metagenomes</td>
      <td>pharokka</td>
      <td>pharokka</td>
      <td></td>
      <td>Pharokka</td>
      <td>Pharokka is a rapid standardised annotation tool for bacteriophage genomes and metagenomes.</td>
      <td>Genome annotation, Antimicrobial resistance prediction, tRNA gene prediction, Formatting, Sequence assembly</td>
      <td>Metagenomics, Sequence sites, features and motifs, Workflows, Functional, regulatory and non-coding RNA</td>
      <td>To update</td>
      <td>https://github.com/gbouras13/pharokka</td>
      <td>Genome annotation</td>
      <td>pharokka</td>
      <td>2023-02-14</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pharokka</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pharokka</td>
      <td>1.3.2</td>
      <td>\n                pharokka\n</td>
      <td></td>
      <td>Genome annotation, Antimicrobial resistance prediction, tRNA gene prediction, Formatting, Sequence assembly</td>
      <td>Metagenomics, Sequence sites, features and motifs, Workflows, Functional, regulatory and non-coding RNA</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>5462</td>
      <td>5462</td>
      <td>334</td>
      <td>334</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>309</td>
      <td>309</td>
      <td>52</td>
      <td>52</td>
      <td>5771</td>
      <td>5771</td>
      <td>386</td>
      <td>386</td>
    </tr>
    <tr>
      <td></td>
      <td>phyloseq</td>
      <td>phyloseq_from_biom, phyloseq_from_dada2, phyloseq_plot_ordination, phyloseq_plot_richness</td>
      <td>Handling and analysis of high-throughput microbiome census data</td>
      <td>phyloseq</td>
      <td>phyloseq</td>
      <td></td>
      <td>phyloseq</td>
      <td>Provides a set of classes and tools to facilitate the import, storage, analysis, and graphical display of microbiome census data.</td>
      <td>Deposition, Analysis, Visualisation</td>
      <td>Microbiology, Sequence analysis, Metagenomics</td>
      <td>Up-to-date</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/phyloseq.html</td>
      <td>Metagenomics</td>
      <td>phyloseq</td>
      <td>2022-03-03</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/phyloseq</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/phyloseq</td>
      <td>1.46.0</td>
      <td>bioconductor-phyloseq</td>
      <td>1.46.0</td>
      <td>Deposition, Analysis, Visualisation</td>
      <td>Microbiology, Sequence analysis, Metagenomics</td>
      <td>4</td>
      <td>1</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>1</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>1199</td>
      <td>1199</td>
      <td>193</td>
      <td>193</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>1</td>
      <td>1</td>
      <td>1205</td>
      <td>1205</td>
      <td>194</td>
      <td>194</td>
    </tr>
    <tr>
      <td></td>
      <td>phyml</td>
      <td>phyml</td>
      <td>PhyML is a phylogeny software based on the maximum-likelihood principle.</td>
      <td>phyml</td>
      <td>phyml</td>
      <td></td>
      <td>PhyML</td>
      <td>Phylogenetic estimation software using Maximum Likelihood</td>
      <td>Phylogenetic tree generation (maximum likelihood and Bayesian methods)</td>
      <td>Phylogenetics, Bioinformatics, Phylogenetics</td>
      <td>Up-to-date</td>
      <td>http://www.atgc-montpellier.fr/phyml/</td>
      <td>Phylogenetics</td>
      <td>phyml</td>
      <td>2019-05-27</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/phyml</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/phyml</td>
      <td>3.3.20220408</td>
      <td>phyml</td>
      <td>3.3.20220408</td>
      <td></td>
      <td>Phylogenetics, Bioinformatics, Phylogenetics</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1975</td>
      <td>2034</td>
      <td>246</td>
      <td>258</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>553</td>
      <td>564</td>
      <td>140</td>
      <td>144</td>
      <td>2528</td>
      <td>2598</td>
      <td>386</td>
      <td>402</td>
    </tr>
    <tr>
      <td></td>
      <td>picard</td>
      <td>picard_AddCommentsToBam, picard_AddOrReplaceReadGroups, picard_BedToIntervalList, picard_CleanSam, picard_CASM, picard_CollectBaseDistributionByCycle, picard_CollectGcBiasMetrics, picard_CollectHsMetrics, picard_CollectInsertSizeMetrics, picard_CollectRnaSeqMetrics, picard_artifact_metrics, picard_CollectWgsMetrics, picard_DownsampleSam, picard_EstimateLibraryComplexity, picard_FastqToSam, picard_FilterSamReads, picard_FixMateInformation, picard_MarkDuplicates, picard_MarkDuplicatesWithMateCigar, picard_MeanQualityByCycle, picard_MergeBamAlignment, picard_MergeSamFiles, picard_NormalizeFasta, picard_QualityScoreDistribution, picard_ReorderSam, picard_ReplaceSamHeader, picard_RevertOriginalBaseQualitiesAndAddMateCigar, picard_RevertSam, picard_SamToFastq, picard_SortSam, picard_ValidateSamFile</td>
      <td>Picard SAM/BAM manipulation tools.</td>
      <td>picard_samtofastq</td>
      <td>picard_samtofastq, picard_replacesamheader, picard_fastqtosam, picard_reordersam</td>
      <td></td>
      <td>picard_samtofastq</td>
      <td>Create a FASTQ file.</td>
      <td>Formatting</td>
      <td>Sequencing</td>
      <td>To update</td>
      <td>http://broadinstitute.github.io/picard/</td>
      <td>SAM</td>
      <td>picard</td>
      <td>2019-05-30</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/picard</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/picard</td>
      <td>3.1.1</td>
      <td>picard</td>
      <td>3.3.0</td>
      <td>Formatting</td>
      <td>Sequencing</td>
      <td>31</td>
      <td>31</td>
      <td>31</td>
      <td>31</td>
      <td>31</td>
      <td>31</td>
      <td>31</td>
      <td>31</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>31</td>
      <td>4</td>
      <td>31</td>
      <td>0</td>
      <td>0</td>
      <td>31</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>31</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>31</td>
      <td>31</td>
      <td>31</td>
      <td>0</td>
      <td>386934</td>
      <td>399039</td>
      <td>11024</td>
      <td>11956</td>
      <td>366646</td>
      <td>589980</td>
      <td>32000</td>
      <td>57111</td>
      <td>56622</td>
      <td>62258</td>
      <td>2845</td>
      <td>3398</td>
      <td>810202</td>
      <td>1051277</td>
      <td>45869</td>
      <td>72465</td>
    </tr>
    <tr>
      <td></td>
      <td>pick_value</td>
      <td>pick_value</td>
      <td>Compose a text parameter value using text, integer and float values</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>pick_value</td>
      <td>2023-01-23</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pick_value</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pick_value</td>
      <td>0.2.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1607</td>
      <td>1607</td>
      <td>41</td>
      <td>41</td>
      <td>490</td>
      <td>490</td>
      <td>53</td>
      <td>53</td>
      <td>127</td>
      <td>127</td>
      <td>4</td>
      <td>4</td>
      <td>2224</td>
      <td>2224</td>
      <td>98</td>
      <td>98</td>
    </tr>
    <tr>
      <td></td>
      <td>picrust</td>
      <td>picrust_categorize, picrust_compare_biom, picrust_format_tree_and_trait_table, picrust_metagenome_contributions, picrust_normalize_by_copy_number, picrust_predict_metagenomes</td>
      <td>PICRUSt wrappers</td>
      <td>picrust</td>
      <td>picrust</td>
      <td></td>
      <td>PICRUSt</td>
      <td>PICRUSt (Phylogenetic Investigation of Communities by Reconstruction of Unobserved States) is a bioinformatics software package designed to predict metagenome functional content from marker gene (e.g., 16S rRNA) surveys and full genomes.</td>
      <td>Phylogenetic reconstruction, Expression analysis, Genome annotation, DNA barcoding</td>
      <td>Metagenomics, Microbial ecology, Functional, regulatory and non-coding RNA, Metagenomic sequencing</td>
      <td>To update</td>
      <td>https://picrust.github.io/picrust/</td>
      <td>Metagenomics</td>
      <td>picrust</td>
      <td>2017-07-31</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/picrust</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/picrust</td>
      <td>1.1.1</td>
      <td>picrust</td>
      <td>1.1.4</td>
      <td>Phylogenetic reconstruction, Expression analysis, Genome annotation, DNA barcoding</td>
      <td>Metagenomics, Microbial ecology, Functional, regulatory and non-coding RNA, Metagenomic sequencing</td>
      <td>0</td>
      <td>6</td>
      <td>5</td>
      <td>6</td>
      <td>0</td>
      <td>6</td>
      <td>5</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>867</td>
      <td>1128</td>
      <td>144</td>
      <td>165</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2106</td>
      <td>2106</td>
      <td>121</td>
      <td>121</td>
      <td>2973</td>
      <td>3234</td>
      <td>265</td>
      <td>286</td>
    </tr>
    <tr>
      <td></td>
      <td>picrust2</td>
      <td>picrust2_add_descriptions, picrust2_hsp, picrust2_metagenome_pipeline, picrust2_pathway_pipeline, picrust2_pipeline, picrust2_place_seqs, picrust2_shuffle_predictions</td>
      <td>PICRUSt2: Phylogenetic Investigation of Communities by Reconstruction of Unobserved States</td>
      <td>picrust2</td>
      <td>picrust2</td>
      <td></td>
      <td>PICRUSt2</td>
      <td>PICRUSt2 (Phylogenetic Investigation of Communities by Reconstruction of Unobserved States) is a software for predicting functional abundances based only on marker gene sequences.</td>
      <td>Phylogenetic reconstruction, Expression analysis, Rarefaction, Pathway analysis</td>
      <td>Metagenomics, Microbiology, Phylogenetics, Metagenomic sequencing</td>
      <td>Up-to-date</td>
      <td>https://github.com/picrust/picrust2/wiki</td>
      <td>Metagenomics</td>
      <td>picrust2</td>
      <td>2021-11-07</td>
      <td>iuc</td>
      <td>https://github.com/picrust/picrust2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/picrust2</td>
      <td>2.5.3</td>
      <td>picrust2</td>
      <td>2.5.3</td>
      <td>Phylogenetic reconstruction, Expression analysis, Rarefaction, Pathway analysis</td>
      <td>Metagenomics, Microbiology, Phylogenetics, Metagenomic sequencing</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>2532</td>
      <td>2532</td>
      <td>382</td>
      <td>382</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1311</td>
      <td>1311</td>
      <td>83</td>
      <td>83</td>
      <td>3844</td>
      <td>3844</td>
      <td>466</td>
      <td>466</td>
    </tr>
    <tr>
      <td></td>
      <td>pilon</td>
      <td>pilon</td>
      <td>pilon is a tool for assembly improvement and variant analysis in bacteria</td>
      <td>pilon</td>
      <td>pilon</td>
      <td></td>
      <td>pilon</td>
      <td>Read alignment analysis to diagnose, report, and automatically improve de novo genome assemblies.</td>
      <td>Sequence assembly, Analysis, Read alignment</td>
      <td>Assembly</td>
      <td>To update</td>
      <td>https://github.com/broadinstitute/pilon/wiki</td>
      <td>Variant Analysis</td>
      <td>pilon</td>
      <td>2016-08-11</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pilon</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pilon</td>
      <td>1.20.1</td>
      <td>pilon</td>
      <td>1.24</td>
      <td>Sequence assembly, Analysis</td>
      <td>Assembly</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>9045</td>
      <td>9458</td>
      <td>1115</td>
      <td>1144</td>
      <td>7989</td>
      <td>7989</td>
      <td>1070</td>
      <td>1070</td>
      <td>3945</td>
      <td>4137</td>
      <td>726</td>
      <td>741</td>
      <td>20979</td>
      <td>21584</td>
      <td>2911</td>
      <td>2955</td>
    </tr>
    <tr>
      <td></td>
      <td>pipelign</td>
      <td>pipelign</td>
      <td>Multipe sequence alignment</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/asmmhossain/pipelign/</td>
      <td>Next Gen Mappers</td>
      <td>pipelign</td>
      <td>2019-08-20</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pipelign</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pipelign</td>
      <td>0.2</td>
      <td>pipelign</td>
      <td>0.2</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1245</td>
      <td>1245</td>
      <td>231</td>
      <td>231</td>
      <td>9046</td>
      <td>9046</td>
      <td>630</td>
      <td>630</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10291</td>
      <td>10291</td>
      <td>861</td>
      <td>861</td>
    </tr>
    <tr>
      <td></td>
      <td>pizzly</td>
      <td>pizzly</td>
      <td>Pizzly is a program for detecting gene fusions from RNA-Seq data of cancer samples.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/pmelsted/pizzly/</td>
      <td>Transcriptomics</td>
      <td></td>
      <td>2017-09-03</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pizzly/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pizzly</td>
      <td>0.37.3.1</td>
      <td>pizzly</td>
      <td>0.37.3</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>201</td>
      <td>210</td>
      <td>23</td>
      <td>24</td>
      <td>998</td>
      <td>998</td>
      <td>76</td>
      <td>76</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1199</td>
      <td>1208</td>
      <td>99</td>
      <td>100</td>
    </tr>
    <tr>
      <td></td>
      <td>plasflow</td>
      <td>PlasFlow</td>
      <td>PlasFlow - Prediction of plasmid sequences in metagenomic contigs.</td>
      <td>plasflow</td>
      <td>plasflow</td>
      <td></td>
      <td>PlasFlow</td>
      <td>PlasFlow is a set of scripts used for prediction of plasmid sequences in metagenomic contigs.</td>
      <td>Sequence analysis</td>
      <td>Metagenomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/smaegol/PlasFlow</td>
      <td>Sequence Analysis</td>
      <td>plasflow</td>
      <td>2018-09-05</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/plasflow</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/plasflow</td>
      <td>1.1.0</td>
      <td>plasflow</td>
      <td>1.1.0</td>
      <td>Sequence analysis</td>
      <td>Metagenomics</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>37086</td>
      <td>37272</td>
      <td>711</td>
      <td>734</td>
      <td>6346</td>
      <td>6346</td>
      <td>583</td>
      <td>583</td>
      <td>3065</td>
      <td>3065</td>
      <td>61</td>
      <td>61</td>
      <td>46497</td>
      <td>46683</td>
      <td>1355</td>
      <td>1378</td>
    </tr>
    <tr>
      <td></td>
      <td>plasmidfinder</td>
      <td>plasmidfinder</td>
      <td>"PlasmidFinder provides the detection of replicons in the WGSand assigns the plasmids under study to lineages that trace backthe information to the existing knowledge on Inc groups and suggestspossible reference plasmids for each lineage"</td>
      <td>PlasmidFinder</td>
      <td>PlasmidFinder</td>
      <td></td>
      <td>PlasmidFinder</td>
      <td>PlasmidFinder is a tool for the identification and typing of Plasmid Replicons in Whole-Genome Sequencing (WGS).</td>
      <td>Genome assembly, Scaffolding, Multilocus sequence typing</td>
      <td>Whole genome sequencing, Sequence assembly, Mapping, Probes and primers</td>
      <td>Up-to-date</td>
      <td>https://bitbucket.org/genomicepidemiology/plasmidfinder/src/master/</td>
      <td>Sequence Analysis</td>
      <td>plasmidfinder</td>
      <td>2022-09-19</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/plasmidfinder</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/plasmidfinder</td>
      <td>2.1.6</td>
      <td>plasmidfinder</td>
      <td>2.1.6</td>
      <td>Genome assembly, Scaffolding, Multilocus sequence typing</td>
      <td>Whole genome sequencing, Sequence assembly, Mapping, Probes and primers</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>14652</td>
      <td>14652</td>
      <td>251</td>
      <td>251</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>101</td>
      <td>101</td>
      <td>18</td>
      <td>18</td>
      <td>14753</td>
      <td>14753</td>
      <td>269</td>
      <td>269</td>
    </tr>
    <tr>
      <td></td>
      <td>plink</td>
      <td>plink</td>
      <td>Whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner.</td>
      <td>plink</td>
      <td>plink</td>
      <td></td>
      <td>PLINK</td>
      <td>Free, open-source whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner.</td>
      <td>Genetic variation analysis</td>
      <td>GWAS study</td>
      <td>Up-to-date</td>
      <td>https://www.cog-genomics.org/plink</td>
      <td>Genome-Wide Association Study</td>
      <td>plink</td>
      <td>2020-09-14</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/plink</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/plink</td>
      <td>1.90b6.21</td>
      <td>plink</td>
      <td>1.90b6.21</td>
      <td>Genetic variation analysis</td>
      <td>GWAS study</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>851</td>
      <td>851</td>
      <td>127</td>
      <td>127</td>
      <td>249</td>
      <td>249</td>
      <td>43</td>
      <td>43</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1100</td>
      <td>1100</td>
      <td>170</td>
      <td>170</td>
    </tr>
    <tr>
      <td></td>
      <td>polypolish</td>
      <td>polypolish</td>
      <td>"Polypolish is a tool for polishing genome assemblies with short reads.Polypolish uses SAM files where each read has been aligned to all possible locations (not just a single best location).This allows it to repair errors in repeat regions that other alignment-based polishers cannot fix."</td>
      <td>Polypolish</td>
      <td>Polypolish</td>
      <td></td>
      <td>Polypolish</td>
      <td>Polypolish is a tool for polishing genome assemblies with short reads. Unlike other tools in this category, Polypolish uses SAM files where each read has been aligned to all possible locations (not just a single best location). This allows it to repair errors in repeat regions that other alignment-based polishers cannot fix.</td>
      <td>Genome assembly, Read mapping, Mapping assembly, Sequencing error detection</td>
      <td>Sequence assembly, Sequence composition, complexity and repeats, Mapping</td>
      <td>To update</td>
      <td>https://github.com/rrwick/Polypolish</td>
      <td>Sequence Analysis</td>
      <td>polypolish</td>
      <td>2022-09-22</td>
      <td>iuc</td>
      <td>https://github.com/mesocentre-clermont-auvergne/galaxy-tools/tree/master/tools/polypolish</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/polypolish</td>
      <td>0.5.0</td>
      <td>polypolish</td>
      <td>0.6.0</td>
      <td>Genome assembly, Read mapping, Mapping assembly, Sequencing error detection</td>
      <td>Sequence assembly, Sequence composition, complexity and repeats, Mapping</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>731</td>
      <td>731</td>
      <td>86</td>
      <td>86</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>42</td>
      <td>42</td>
      <td>4</td>
      <td>4</td>
      <td>773</td>
      <td>773</td>
      <td>90</td>
      <td>90</td>
    </tr>
    <tr>
      <td></td>
      <td>porechop</td>
      <td>porechop</td>
      <td>Porechop - Finding and removing adapters from Oxford Nanopore reads</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/rrwick/Porechop</td>
      <td>Fasta Manipulation, Fastq Manipulation</td>
      <td>porechop</td>
      <td>2018-09-13</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/porechop</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/porechop</td>
      <td></td>
      <td>porechop</td>
      <td>0.2.4</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>262015</td>
      <td>262852</td>
      <td>2627</td>
      <td>2711</td>
      <td>83623</td>
      <td>83623</td>
      <td>1918</td>
      <td>1918</td>
      <td>52558</td>
      <td>52558</td>
      <td>1037</td>
      <td>1037</td>
      <td>398196</td>
      <td>399033</td>
      <td>5582</td>
      <td>5666</td>
    </tr>
    <tr>
      <td></td>
      <td>poretools</td>
      <td>poretools_events, poretools_extract, poretools_hist, poretools_nucdist, poretools_occupancy, poretools_qualdist, poretools_qualpos, poretools_squiggle, poretools_stats, poretools_tabular, poretools_times, poretools_winner, poretools_yield_plot</td>
      <td>A flexible toolkit for exploring datasets generated by nanopore sequencing devices from MinION for the purposes of quality control and downstream analysis.</td>
      <td>poretools</td>
      <td>poretools</td>
      <td></td>
      <td>Poretools</td>
      <td>Flexible toolkit for exploring datasets generated by nanopore sequencing devices from MinION for the purposes of quality control and downstream analysis.</td>
      <td>Nucleic acid sequence analysis</td>
      <td>DNA, Sequencing</td>
      <td>Up-to-date</td>
      <td>https://poretools.readthedocs.io/en/latest/</td>
      <td>Fasta Manipulation, Fastq Manipulation</td>
      <td></td>
      <td>2017-12-17</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/poretools</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/poretools</td>
      <td>0.6.1a1</td>
      <td>poretools</td>
      <td>0.6.1a1</td>
      <td>Nucleic acid sequence analysis</td>
      <td>DNA, Sequencing</td>
      <td>13</td>
      <td>13</td>
      <td>13</td>
      <td>0</td>
      <td>13</td>
      <td>13</td>
      <td>13</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>13</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>13</td>
      <td>0</td>
      <td>19942</td>
      <td>20587</td>
      <td>1055</td>
      <td>1245</td>
      <td>33619</td>
      <td>40808</td>
      <td>2183</td>
      <td>2594</td>
      <td>9261</td>
      <td>9261</td>
      <td>365</td>
      <td>365</td>
      <td>62822</td>
      <td>70656</td>
      <td>3603</td>
      <td>4204</td>
    </tr>
    <tr>
      <td></td>
      <td>presto</td>
      <td>presto_alignsets, presto_assemblepairs, presto_buildconsensus, presto_collapseseq, presto_filterseq, presto_maskprimers, presto_pairseq, presto_parseheaders, presto_parselog, presto_partition, prestor_abseq3</td>
      <td>pRESTO toolkit for immune repertoire analysis.</td>
      <td>presto</td>
      <td>presto</td>
      <td></td>
      <td>pRESTO</td>
      <td>Integrated collection of platform-independent Python modules for processing raw reads from high-throughput (next-generation) sequencing of lymphocyte repertoires.</td>
      <td>Nucleic acid sequence analysis</td>
      <td>Sequencing, DNA, Immunology</td>
      <td>To update</td>
      <td>https://presto.readthedocs.io/</td>
      <td>Sequence Analysis</td>
      <td>presto</td>
      <td>2017-08-30</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/presto</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/presto</td>
      <td>0.6.2</td>
      <td>presto</td>
      <td>0.7.2</td>
      <td>Nucleic acid sequence analysis</td>
      <td>Sequencing, DNA, Immunology</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>63244</td>
      <td>65177</td>
      <td>1392</td>
      <td>1599</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>63244</td>
      <td>65177</td>
      <td>1392</td>
      <td>1599</td>
    </tr>
    <tr>
      <td></td>
      <td>pretext</td>
      <td>pretext_graph, pretext_map, pretext_snapshot</td>
      <td>Process genome contacts maps processing images.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/wtsi-hpag/PretextSnapshot</td>
      <td>Sequence Analysis</td>
      <td>suite_pretext</td>
      <td>2021-04-05</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pretext</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pretext</td>
      <td>0.0.6</td>
      <td>pretextgraph</td>
      <td>0.0.6</td>
      <td></td>
      <td></td>
      <td>3</td>
      <td>2</td>
      <td>3</td>
      <td>0</td>
      <td>3</td>
      <td>2</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>3463</td>
      <td>3463</td>
      <td>538</td>
      <td>538</td>
      <td>1853</td>
      <td>1853</td>
      <td>256</td>
      <td>256</td>
      <td>1097</td>
      <td>1097</td>
      <td>70</td>
      <td>70</td>
      <td>6413</td>
      <td>6413</td>
      <td>864</td>
      <td>864</td>
    </tr>
    <tr>
      <td></td>
      <td>prinseq</td>
      <td>prinseq</td>
      <td>PRINSEQ is a tool for easy and rapid quality control and data processing of metagenomic and metatranscriptomic datasets</td>
      <td>prinseq</td>
      <td>prinseq</td>
      <td></td>
      <td>PRINSEQ</td>
      <td>PRINSEQ is a sequence processing tool that can be used to filter, reformat and trim genomic and metagenomic sequence data. It generates summary statistics of the input in graphical and tabular formats that can be used for quality control steps. PRINSEQ is available as both standalone and web-based versions.</td>
      <td>Read pre-processing, Sequence trimming, Sequence contamination filtering</td>
      <td>Transcriptomics, Metagenomics, Genomics</td>
      <td>To update</td>
      <td>http://prinseq.sourceforge.net/manual.html</td>
      <td>Fastq Manipulation, Metagenomics</td>
      <td>prinseq</td>
      <td>2016-05-30</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/prinseq/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/prinseq</td>
      <td>@TOOL_VERSION+galaxy2</td>
      <td>prinseq</td>
      <td>0.20.4</td>
      <td>Read pre-processing, Sequence trimming, Sequence contamination filtering</td>
      <td>Transcriptomics, Metagenomics</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>10152</td>
      <td>10626</td>
      <td>253</td>
      <td>281</td>
      <td>10581</td>
      <td>10581</td>
      <td>1240</td>
      <td>1240</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>20733</td>
      <td>21207</td>
      <td>1493</td>
      <td>1521</td>
    </tr>
    <tr>
      <td></td>
      <td>prodigal</td>
      <td>prodigal</td>
      <td>A protein-coding gene prediction software tool for bacterial and archaeal genomes</td>
      <td>prodigal</td>
      <td>prodigal</td>
      <td></td>
      <td>Prodigal</td>
      <td>Fast, reliable protein-coding gene prediction for prokaryotic genomes.</td>
      <td>Genome annotation</td>
      <td>Genomics, Sequence analysis</td>
      <td>Up-to-date</td>
      <td>https://github.com/hyattpd/Prodigal</td>
      <td>Genome annotation</td>
      <td>prodigal</td>
      <td>2024-03-14</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/prodigal</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/prodigal</td>
      <td>2.6.3</td>
      <td>prodigal</td>
      <td>2.6.3</td>
      <td>Genome annotation</td>
      <td>Genomics, Sequence analysis</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1902</td>
      <td>1902</td>
      <td>167</td>
      <td>167</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1902</td>
      <td>1902</td>
      <td>167</td>
      <td>167</td>
    </tr>
    <tr>
      <td></td>
      <td>progressivemauve</td>
      <td>progressivemauve, xmfa2gff3</td>
      <td>Mauve/ProgressiveMauve Multiple Sequence Aligner</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://darlinglab.org/mauve/user-guide/progressivemauve.html</td>
      <td>Sequence Analysis</td>
      <td>progressivemauve</td>
      <td>2015-05-18</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/progressivemauve</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/progressivemauve</td>
      <td></td>
      <td>progressivemauve</td>
      <td>snapshot_2015_02_13</td>
      <td></td>
      <td></td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>2064</td>
      <td>2173</td>
      <td>678</td>
      <td>695</td>
      <td>2900</td>
      <td>2900</td>
      <td>1034</td>
      <td>1034</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4964</td>
      <td>5073</td>
      <td>1712</td>
      <td>1729</td>
    </tr>
    <tr>
      <td></td>
      <td>prokka</td>
      <td>prokka</td>
      <td>Rapid annotation of prokaryotic genomes</td>
      <td>prokka</td>
      <td>prokka</td>
      <td></td>
      <td>Prokka</td>
      <td>Software tool to annotate bacterial, archaeal and viral genomes quickly and produce standards-compliant output files.</td>
      <td>Gene prediction, Coding region prediction, Genome annotation</td>
      <td>Genomics, Model organisms, Virology</td>
      <td>Up-to-date</td>
      <td>http://github.com/tseemann/prokka</td>
      <td>Sequence Analysis</td>
      <td>prokka</td>
      <td>2016-10-10</td>
      <td>crs4</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/prokka/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/prokka</td>
      <td>1.14.6</td>
      <td>prokka</td>
      <td>1.14.6</td>
      <td>Coding region prediction, Genome annotation</td>
      <td>Genomics, Model organisms, Virology</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>464359</td>
      <td>476554</td>
      <td>7475</td>
      <td>7721</td>
      <td>313285</td>
      <td>331256</td>
      <td>11243</td>
      <td>12384</td>
      <td>131211</td>
      <td>141875</td>
      <td>4358</td>
      <td>4947</td>
      <td>908855</td>
      <td>949685</td>
      <td>23076</td>
      <td>25052</td>
    </tr>
    <tr>
      <td></td>
      <td>prot-scriber</td>
      <td>prot_scriber</td>
      <td>Protein annotation of short human readable descriptions</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/usadellab/prot-scriber</td>
      <td>Proteomics</td>
      <td>prot_scriber</td>
      <td>2022-05-10</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/prot-scriber</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/prot-scriber</td>
      <td>0.1.6</td>
      <td>prot-scriber</td>
      <td>0.1.6</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>10</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>10</td>
      <td>7</td>
      <td>7</td>
    </tr>
    <tr>
      <td></td>
      <td>proteinortho</td>
      <td>proteinortho, proteinortho_grab_proteins, proteinortho_summary</td>
      <td>Proteinortho is a tool to detect orthologous proteins/genes within different species.</td>
      <td>proteinortho</td>
      <td>proteinortho</td>
      <td></td>
      <td>Proteinortho</td>
      <td>Proteinortho is a tool to detect orthologous genes within different species</td>
      <td>Sequence clustering, Sequence analysis</td>
      <td>Comparative genomics</td>
      <td>To update</td>
      <td>https://gitlab.com/paulklemm_PHD/proteinortho</td>
      <td>Proteomics</td>
      <td>proteinortho</td>
      <td>2020-10-02</td>
      <td>iuc</td>
      <td>https://gitlab.com/paulklemm_PHD/proteinortho</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/proteinortho</td>
      <td>6.3.1</td>
      <td>proteinortho</td>
      <td>6.3.2</td>
      <td>Sequence clustering, Sequence analysis</td>
      <td>Comparative genomics</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>3962</td>
      <td>3962</td>
      <td>501</td>
      <td>501</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3962</td>
      <td>3962</td>
      <td>501</td>
      <td>501</td>
    </tr>
    <tr>
      <td></td>
      <td>psiclass</td>
      <td>psiclass</td>
      <td>PsiCLASS is a reference-based transcriptome assembler for single or multiple RNA-seq samples.</td>
      <td>psiclass</td>
      <td>psiclass</td>
      <td></td>
      <td>PsiCLASS</td>
      <td>Reference-based transcriptome assembler for single or multiple RNA-seq samples</td>
      <td>Transcriptome assembly</td>
      <td>Sequence assembly</td>
      <td>Up-to-date</td>
      <td>https://github.com/splicebox/PsiCLASS</td>
      <td>Transcriptomics</td>
      <td>psiclass</td>
      <td>2023-09-06</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/psiclass</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/psiclass</td>
      <td>1.0.3</td>
      <td>psiclass</td>
      <td>1.0.3</td>
      <td>Transcriptome assembly</td>
      <td>Sequence assembly</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>24</td>
      <td>24</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>24</td>
      <td>24</td>
      <td>5</td>
      <td>5</td>
    </tr>
    <tr>
      <td></td>
      <td>pureclip</td>
      <td>pureclip</td>
      <td>PureCLIP is an HMM based peak caller specifically designed for eCLIP/iCLIP data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/skrakau/PureCLIP</td>
      <td>Sequence Analysis, RNA, CLIP-seq</td>
      <td>pureclip</td>
      <td>2018-04-26</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pureclip</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pureclip</td>
      <td>1.0.4</td>
      <td>pureclip</td>
      <td>1.3.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1319</td>
      <td>1507</td>
      <td>81</td>
      <td>92</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1319</td>
      <td>1507</td>
      <td>81</td>
      <td>92</td>
    </tr>
    <tr>
      <td></td>
      <td>purge_dups</td>
      <td>purge_dups</td>
      <td>Purge haplotigs and overlaps in an assembly based on read depth</td>
      <td>purge_dups</td>
      <td>purge_dups</td>
      <td></td>
      <td>purge_dups</td>
      <td>Identifying and removing haplotypic duplication in primary genome assemblies | haplotypic duplication identification tool | scripts/pd_config.py: script to generate a configuration file used by run_purge_dups.py | purge haplotigs and overlaps in an assembly based on read depth | Given a primary assembly pri_asm and an alternative assembly hap_asm (optional, if you have one), follow the steps shown below to build your own purge_dups pipeline, steps with same number can be run simultaneously. Among all the steps, although step 4 is optional, we highly recommend our users to do so, because assemblers may produce overrepresented seqeuences. In such a case, The final step 4 can be applied to remove those seqeuences</td>
      <td>Genome assembly, Read binning, Scaffolding</td>
      <td>Sequence assembly</td>
      <td>Up-to-date</td>
      <td>https://github.com/dfguan/purge_dups</td>
      <td>Assembly</td>
      <td>purge_dups</td>
      <td>2021-02-03</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/purge_dups</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/purge_dups</td>
      <td>1.2.6</td>
      <td>purge_dups</td>
      <td>1.2.6</td>
      <td>Genome assembly, Read binning, Scaffolding</td>
      <td>Sequence assembly</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>17782</td>
      <td>17782</td>
      <td>287</td>
      <td>287</td>
      <td>853</td>
      <td>853</td>
      <td>65</td>
      <td>65</td>
      <td>2224</td>
      <td>2224</td>
      <td>65</td>
      <td>65</td>
      <td>20859</td>
      <td>20859</td>
      <td>417</td>
      <td>417</td>
    </tr>
    <tr>
      <td></td>
      <td>pycoqc</td>
      <td>pycoqc</td>
      <td>QC metrics for ONT Basecalling</td>
      <td>pycoqc</td>
      <td>pycoqc</td>
      <td></td>
      <td>pycoQC</td>
      <td>PycoQC computes metrics and generates interactive QC plots for Oxford Nanopore technologies sequencing data.</td>
      <td>Sequencing quality control, Statistical calculation</td>
      <td>Sequence analysis, Data quality management, Sequencing</td>
      <td>Up-to-date</td>
      <td>https://github.com/tleonardi/pycoQC</td>
      <td>Nanopore</td>
      <td>pycoqc</td>
      <td>2021-03-02</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pycoqc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pycoqc</td>
      <td>2.5.2</td>
      <td>pycoqc</td>
      <td>2.5.2</td>
      <td>Sequencing quality control, Statistical calculation</td>
      <td>Sequence analysis, Data quality management, Sequencing</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>22742</td>
      <td>22742</td>
      <td>557</td>
      <td>557</td>
      <td>2159</td>
      <td>2159</td>
      <td>716</td>
      <td>716</td>
      <td>776</td>
      <td>776</td>
      <td>448</td>
      <td>448</td>
      <td>25677</td>
      <td>25677</td>
      <td>1721</td>
      <td>1721</td>
    </tr>
    <tr>
      <td></td>
      <td>pyega3</td>
      <td>pyega3</td>
      <td>EGA python client uses the EGA REST API to download authorized datasets and files.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/EGA-archive/ega-download-client</td>
      <td>Data Source</td>
      <td>ega_download_client</td>
      <td>2022-10-13</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pyega3</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pyega3</td>
      <td>5.0.2</td>
      <td>pyega3</td>
      <td>5.2.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2716</td>
      <td>2716</td>
      <td>104</td>
      <td>104</td>
      <td>134</td>
      <td>134</td>
      <td>43</td>
      <td>43</td>
      <td>40</td>
      <td>40</td>
      <td>1</td>
      <td>1</td>
      <td>2890</td>
      <td>2890</td>
      <td>148</td>
      <td>148</td>
    </tr>
    <tr>
      <td></td>
      <td>pygenometracks</td>
      <td>pygenomeTracks</td>
      <td>pyGenomeTracks: Standalone program and library to plot beautiful genome browser tracks.</td>
      <td>pygenometracks</td>
      <td>pygenometracks</td>
      <td></td>
      <td>pyGenomeTracks</td>
      <td>reproducible plots for multivariate genomic data sets.Standalone program and library to plot beautiful genome browser tracks.pyGenomeTracks aims to produce high-quality genome browser tracks that are highly customizable. Currently, it is possible to plot:.</td>
      <td>Visualisation, Formatting</td>
      <td>Model organisms, Imaging, Workflows</td>
      <td>To update</td>
      <td>https://github.com/deeptools/pyGenomeTracks</td>
      <td>Visualization</td>
      <td>pygenometracks</td>
      <td>2020-03-27</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pygenometracks</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pygenometracks</td>
      <td>3.8</td>
      <td>pygenometracks</td>
      <td>3.9</td>
      <td>Visualisation, Formatting</td>
      <td>Model organisms, Imaging, Workflows</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>12858</td>
      <td>13232</td>
      <td>986</td>
      <td>996</td>
      <td>8092</td>
      <td>8092</td>
      <td>943</td>
      <td>943</td>
      <td>3238</td>
      <td>3238</td>
      <td>124</td>
      <td>124</td>
      <td>24188</td>
      <td>24562</td>
      <td>2053</td>
      <td>2063</td>
    </tr>
    <tr>
      <td></td>
      <td>pysradb</td>
      <td>pysradb_search</td>
      <td>pysradb allows to retrieve metadata, such as run accession numbers, from SRA and ENA based on multiple criteria.</td>
      <td>pysradb</td>
      <td>pysradb</td>
      <td></td>
      <td>pysradb</td>
      <td>Python package to query next-generation sequencing metadata and data from NCBI Sequence Read Archive.</td>
      <td>Deposition, Data retrieval</td>
      <td>Sequencing, Gene transcripts, Bioinformatics</td>
      <td>To update</td>
      <td>https://github.com/saketkc/pysradb</td>
      <td>Sequence Analysis</td>
      <td>pysradb_search</td>
      <td>2022-11-10</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pysradb</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pysradb</td>
      <td>1.4.2</td>
      <td>pysradb</td>
      <td>2.2.2</td>
      <td>Deposition, Data retrieval</td>
      <td>Sequencing, Gene transcripts, Bioinformatics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>163</td>
      <td>163</td>
      <td>35</td>
      <td>35</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>163</td>
      <td>163</td>
      <td>35</td>
      <td>35</td>
    </tr>
    <tr>
      <td></td>
      <td>qfilt</td>
      <td>qfilt</td>
      <td>Filter sequencing data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/veg/qfilt</td>
      <td>Fastq Manipulation</td>
      <td>qfilt</td>
      <td>2018-06-06</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/qfilt</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/qfilt</td>
      <td>1.0.0+galaxy1</td>
      <td>qfilt</td>
      <td>0.0.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>qiime_add_on</td>
      <td>qiime_collapse_samples, qiime_make_otu_table</td>
      <td>QIIME to perform microbial community analysis</td>
      <td>qiime_add_on</td>
      <td>qiime_core, qiime_add_on</td>
      <td></td>
      <td>qiime_add_on</td>
      <td>QIIME 2 is a next-generation microbiome bioinformatics platform that is extensible, free, open source, and community developed.</td>
      <td>Demultiplexing, Visualisation, Taxonomic classification, Phylogenetic analysis, Sequencing quality control</td>
      <td>Microbial ecology, Phylogeny, Metagenomics, Metatranscriptomics</td>
      <td>To update</td>
      <td>http://www.qiime.org</td>
      <td>Metagenomics</td>
      <td>qiime</td>
      <td>2017-01-25</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/qiime/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/qiime/qiime_add_on</td>
      <td></td>
      <td>qiime</td>
      <td>1.9.1</td>
      <td>Demultiplexing, Visualisation, Taxonomic classification, Phylogenetic analysis, Sequencing quality control</td>
      <td>Microbial ecology, Phylogeny, Metagenomics, Metatranscriptomics</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>2092</td>
      <td>2342</td>
      <td>209</td>
      <td>238</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>1</td>
      <td>1</td>
      <td>2094</td>
      <td>2344</td>
      <td>210</td>
      <td>239</td>
    </tr>
    <tr>
      <td></td>
      <td>qiime_core</td>
      <td>qiime_align_seqs, qiime_alpha_diversity, qiime_alpha_rarefaction, qiime_assign_taxonomy, qiime_beta_diversity, qiime_beta_diversity_through_plots, qiime_compare_categories, qiime_core_diversity, qiime_count_seqs, qiime_extract_barcodes, qiime_filter_alignment, qiime_filter_fasta, qiime_filter_otus_from_otu_table, qiime_filter_samples_from_otu_table, qiime_filter_taxa_from_otu_table, qiime_jackknifed_beta_diversity, qiime_make_emperor, qiime_make_otu_heatmap, qiime_make_phylogeny, qiime_multiple_join_paired_ends, qiime_multiple_split_libraries_fastq, qiime_pick_closed_reference_otus, qiime_pick_open_reference_otus, qiime_pick_otus, qiime_pick_rep_set, qiime_plot_taxa_summary, qiime_split_libraries, qiime_split_libraries_fastq, qiime_summarize_taxa, qiime_summarize_taxa_through_plots, qiime_upgma_cluster, qiime_validate_mapping_file</td>
      <td>QIIME to perform microbial community analysis</td>
      <td>qiime_core</td>
      <td>qiime_core</td>
      <td></td>
      <td>qiime_core</td>
      <td>QIIME 2 is a next-generation microbiome bioinformatics platform that is extensible, free, open source, and community developed.</td>
      <td>Demultiplexing, Visualisation, Taxonomic classification, Phylogenetic analysis, Sequencing quality control</td>
      <td>Microbial ecology, Phylogeny, Metagenomics, Metatranscriptomics</td>
      <td>To update</td>
      <td>http://www.qiime.org</td>
      <td>Metagenomics</td>
      <td>qiime</td>
      <td>2017-06-14</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/qiime/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/qiime/qiime_core</td>
      <td></td>
      <td>qiime</td>
      <td>1.9.1</td>
      <td>Demultiplexing, Visualisation, Taxonomic classification, Phylogenetic analysis, Sequencing quality control</td>
      <td>Microbial ecology, Phylogeny, Metagenomics, Metatranscriptomics</td>
      <td>0</td>
      <td>0</td>
      <td>32</td>
      <td>32</td>
      <td>0</td>
      <td>0</td>
      <td>32</td>
      <td>32</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>31</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>31</td>
      <td>31</td>
      <td>0</td>
      <td>31825</td>
      <td>34677</td>
      <td>3375</td>
      <td>3785</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>14</td>
      <td>14</td>
      <td>4</td>
      <td>4</td>
      <td>31840</td>
      <td>34692</td>
      <td>3380</td>
      <td>3790</td>
    </tr>
    <tr>
      <td></td>
      <td>qiime_extract_viz</td>
      <td>qiime_extract_viz</td>
      <td>Extract vizualization from QIIME artifacts</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.qiime.org</td>
      <td>Metagenomics</td>
      <td>qiime_extract_viz</td>
      <td>2022-10-06</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/qiimme_extract_viz</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/qiime_extract_viz</td>
      <td>0.1.0</td>
      <td>unzip</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>17</td>
      <td>17</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>17</td>
      <td>17</td>
      <td>8</td>
      <td>8</td>
    </tr>
    <tr>
      <td></td>
      <td>qq_tools</td>
      <td>qq_manhattan</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://CRAN.R-project.org/package=qqman</td>
      <td>Visualization, Variant Analysis</td>
      <td></td>
      <td>2020-10-07</td>
      <td>iuc</td>
      <td></td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/qq_tools</td>
      <td>0.1.0</td>
      <td>r-qqman</td>
      <td>0.1.4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>qualimap</td>
      <td>qualimap_bamqc, qualimap_counts, qualimap_multi_bamqc, qualimap_rnaseq</td>
      <td>Qualimap 2 is a platform-independent application written in Java andR that facilitates the quality control of alignment sequencing data and itsderivatives like feature counts.</td>
      <td>qualimap</td>
      <td>qualimap</td>
      <td></td>
      <td>QualiMap</td>
      <td>Platform-independent application written in Java and R that provides both a Graphical User Inteface (GUI) and a command-line interface to facilitate the quality control of alignment sequencing data.</td>
      <td>Sequencing quality control</td>
      <td>Data quality management</td>
      <td>Up-to-date</td>
      <td>http://qualimap.bioinfo.cipf.es/</td>
      <td>Sequence Analysis, Transcriptomics, SAM</td>
      <td>qualimap</td>
      <td>2019-10-10</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/qualimap</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/qualimap</td>
      <td>2.3</td>
      <td>qualimap</td>
      <td>2.3</td>
      <td>Sequencing quality control</td>
      <td>Data quality management</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>1</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>684820</td>
      <td>684820</td>
      <td>3239</td>
      <td>3239</td>
      <td>90503</td>
      <td>90503</td>
      <td>3881</td>
      <td>3881</td>
      <td>29018</td>
      <td>29018</td>
      <td>826</td>
      <td>826</td>
      <td>804341</td>
      <td>804341</td>
      <td>7946</td>
      <td>7946</td>
    </tr>
    <tr>
      <td></td>
      <td>quast</td>
      <td>quast</td>
      <td>Quast (Quality ASsessment Tool) evaluates genome assemblies.</td>
      <td>quast</td>
      <td>quast</td>
      <td></td>
      <td>QUAST</td>
      <td>QUAST stands for QUality ASsessment Tool.  It evaluates a quality of genome assemblies by computing various metrics and providing nice reports.</td>
      <td>Visualisation, Sequence assembly validation</td>
      <td>Sequence assembly</td>
      <td>Up-to-date</td>
      <td>http://quast.bioinf.spbau.ru/</td>
      <td>Assembly</td>
      <td>quast</td>
      <td>2018-02-10</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/quast</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/quast</td>
      <td>5.2.0</td>
      <td>quast</td>
      <td>5.2.0</td>
      <td>Visualisation, Sequence assembly validation</td>
      <td>Sequence assembly</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>67737</td>
      <td>69738</td>
      <td>7787</td>
      <td>7971</td>
      <td>51925</td>
      <td>56602</td>
      <td>9692</td>
      <td>10646</td>
      <td>32003</td>
      <td>34145</td>
      <td>3890</td>
      <td>4127</td>
      <td>151665</td>
      <td>160485</td>
      <td>21369</td>
      <td>22744</td>
    </tr>
    <tr>
      <td></td>
      <td>query_impc</td>
      <td>query_impc</td>
      <td>Contains a tool to query the IMPC database.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/INFRAFRONTIERDIB/tools-iuc/tree/query_impc/tools/query_impc</td>
      <td>Convert Formats, Web Services</td>
      <td>query_impc</td>
      <td>2023-10-11</td>
      <td>iuc</td>
      <td>https://github.com/INFRAFRONTIERDIB/tools-iuc/tree/query_impc/tools/query_impc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/query_impc</td>
      <td>0.9.0</td>
      <td>requests</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>2</td>
      <td>2</td>
    </tr>
    <tr>
      <td></td>
      <td>query_tabular</td>
      <td>filter_tabular, query_tabular, sqlite_to_tabular</td>
      <td>Loads tabular files into a SQLite DB to perform a SQL query producing a tabular output</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/query_tabular</td>
      <td>Text Manipulation</td>
      <td></td>
      <td>2017-07-18</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/query_tabular</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/query_tabular</td>
      <td>3.3.1</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>1</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>130124</td>
      <td>135361</td>
      <td>1265</td>
      <td>1305</td>
      <td>11135</td>
      <td>11135</td>
      <td>922</td>
      <td>922</td>
      <td>5939</td>
      <td>5939</td>
      <td>153</td>
      <td>153</td>
      <td>147198</td>
      <td>152435</td>
      <td>2340</td>
      <td>2380</td>
    </tr>
    <tr>
      <td></td>
      <td>quickmerge</td>
      <td>quickmerge</td>
      <td>Merge long-read and hybrid assemblies to increase contiguity</td>
      <td>quickmerge</td>
      <td>quickmerge</td>
      <td></td>
      <td>quickmerge</td>
      <td>Quickmerge is a program that uses complementary information from genomes assembled with long reads in order to improve contiguity, and works with assemblies derived from both Pacific Biosciences or Oxford Nanopore. Quickmerge will even work with hybrid assemblies made by combining long reads and Illumina short reads.</td>
      <td>Genome assembly, Scaffolding, De-novo assembly, Genotyping</td>
      <td>Structural variation, Sequence assembly, DNA polymorphism, Whole genome sequencing, Genotype and phenotype</td>
      <td>Up-to-date</td>
      <td>https://github.com/mahulchak/quickmerge</td>
      <td>Assembly</td>
      <td>quickmerge</td>
      <td>2022-07-08</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/quickmerge</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/quickmerge</td>
      <td>0.3</td>
      <td>quickmerge</td>
      <td>0.3</td>
      <td>Genome assembly, Scaffolding, De-novo assembly, Genotyping</td>
      <td>Structural variation, Sequence assembly, DNA polymorphism, Whole genome sequencing, Genotype and phenotype</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>raceid</td>
      <td>raceid_clustering, raceid_filtnormconf, raceid_inspectclusters, raceid_inspecttrajectory, raceid_trajectory</td>
      <td>RaceID3, StemID2, FateID - scRNA analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/dgrun/RaceID3_StemID2_package/</td>
      <td>Single Cell, Transcriptomics</td>
      <td></td>
      <td>2021-04-14</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/raceid3</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/raceid</td>
      <td>0.2.3</td>
      <td>r-raceid</td>
      <td>0.1.3</td>
      <td></td>
      <td></td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>1</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>3077</td>
      <td>4581</td>
      <td>395</td>
      <td>518</td>
      <td>2856</td>
      <td>2856</td>
      <td>1093</td>
      <td>1093</td>
      <td>1977</td>
      <td>1977</td>
      <td>19</td>
      <td>19</td>
      <td>7910</td>
      <td>9414</td>
      <td>1507</td>
      <td>1630</td>
    </tr>
    <tr>
      <td></td>
      <td>ragtag</td>
      <td>ragtag</td>
      <td>Reference-guided scaffolding of draft genomes tool.</td>
      <td>ragtag</td>
      <td>ragtag</td>
      <td></td>
      <td>ragtag</td>
      <td>RagTag is a collection of software tools for scaffolding and improving modern genome assemblies.</td>
      <td>Genome assembly</td>
      <td>Sequence assembly</td>
      <td>Up-to-date</td>
      <td>https://github.com/malonge/RagTag</td>
      <td>Assembly</td>
      <td>ragtag</td>
      <td>2021-11-10</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ragtag</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ragtag</td>
      <td>2.1.0</td>
      <td>ragtag</td>
      <td>2.1.0</td>
      <td>Genome assembly</td>
      <td>Sequence assembly</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>4844</td>
      <td>4844</td>
      <td>480</td>
      <td>480</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>3</td>
      <td>3</td>
      <td>4851</td>
      <td>4851</td>
      <td>483</td>
      <td>483</td>
    </tr>
    <tr>
      <td></td>
      <td>rapidnj</td>
      <td>rapidnj</td>
      <td>Galaxy wrapper for the RapidNJ tool</td>
      <td>rapidnj</td>
      <td>rapidnj</td>
      <td></td>
      <td>RapidNJ</td>
      <td>A tool for fast canonical neighbor-joining tree construction.</td>
      <td>Phylogenetic tree generation</td>
      <td>Phylogeny</td>
      <td>Up-to-date</td>
      <td>https://birc.au.dk/software/rapidnj/</td>
      <td>Phylogenetics</td>
      <td>rapidnj</td>
      <td>2020-05-11</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/rapidnj</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/rapidnj</td>
      <td>2.3.2</td>
      <td>rapidnj</td>
      <td>2.3.2</td>
      <td></td>
      <td>Phylogeny</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>339</td>
      <td>339</td>
      <td>30</td>
      <td>30</td>
      <td>1737</td>
      <td>1737</td>
      <td>256</td>
      <td>256</td>
      <td>6</td>
      <td>6</td>
      <td>1</td>
      <td>1</td>
      <td>2082</td>
      <td>2082</td>
      <td>287</td>
      <td>287</td>
    </tr>
    <tr>
      <td></td>
      <td>rasusa</td>
      <td>rasusa</td>
      <td>Randomly subsample sequencing reads to a specified coverage</td>
      <td>rasusa</td>
      <td>rasusa</td>
      <td></td>
      <td>rasusa</td>
      <td>Produces an unbiased subsample of your reads</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/mbhall88/rasusa</td>
      <td>Sequence Analysis</td>
      <td>rasusa</td>
      <td>2024-02-16</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/rasusa</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/rasusa</td>
      <td>2.0.0</td>
      <td>rasusa</td>
      <td>2.1.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>42</td>
      <td>42</td>
      <td>9</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>42</td>
      <td>42</td>
      <td>9</td>
      <td>9</td>
    </tr>
    <tr>
      <td></td>
      <td>raven</td>
      <td>raven</td>
      <td>Raven is a de novo genome assembler for long uncorrected reads.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/lbcb-sci/raven</td>
      <td>Assembly</td>
      <td></td>
      <td>2020-11-04</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/raven</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/raven</td>
      <td>1.8.3</td>
      <td>raven-assembler</td>
      <td>1.8.3</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>9477</td>
      <td>9477</td>
      <td>438</td>
      <td>438</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10585</td>
      <td>10585</td>
      <td>182</td>
      <td>182</td>
      <td>20062</td>
      <td>20062</td>
      <td>620</td>
      <td>620</td>
    </tr>
    <tr>
      <td></td>
      <td>raxml</td>
      <td>raxml</td>
      <td>RAxML - A Maximum Likelihood based phylogenetic inference</td>
      <td>raxml</td>
      <td>raxml</td>
      <td></td>
      <td>RAxML</td>
      <td>A tool for Phylogenetic Analysis and Post-Analysis of Large Phylogenies.</td>
      <td>Sequence analysis, Phylogenetic tree analysis</td>
      <td>Phylogenetics, Sequence analysis</td>
      <td>To update</td>
      <td>http://www.exelixis-lab.org/web/software/raxml/</td>
      <td>Phylogenetics</td>
      <td>raxml</td>
      <td>2015-11-05</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/raxml</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/raxml</td>
      <td>8.2.12</td>
      <td>raxml</td>
      <td>8.2.13</td>
      <td>Sequence analysis</td>
      <td>Phylogenetics</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>10905</td>
      <td>10905</td>
      <td>966</td>
      <td>966</td>
      <td>5100</td>
      <td>5114</td>
      <td>587</td>
      <td>587</td>
      <td>4955</td>
      <td>6094</td>
      <td>765</td>
      <td>892</td>
      <td>20960</td>
      <td>22113</td>
      <td>2318</td>
      <td>2445</td>
    </tr>
    <tr>
      <td></td>
      <td>rcorrector</td>
      <td>rcorrector</td>
      <td>Rcorrector (RNA-seq error CORRECTOR) is a kmer-based error correction method for RNA-seq data.</td>
      <td>rcorrector</td>
      <td>rcorrector</td>
      <td></td>
      <td>Rcorrector</td>
      <td>This is a kmer-based error correction method for RNA-seq data. It can also be applied to other types of sequencing data where the read coverage is non-uniform, such as single-cell sequencing.</td>
      <td>Sequencing error detection</td>
      <td>RNA, RNA-Seq, Sequencing</td>
      <td>To update</td>
      <td>https://github.com/mourisl/Rcorrector</td>
      <td>Fastq Manipulation</td>
      <td>rcorrector</td>
      <td>2018-05-02</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/rcorrector</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/rcorrector</td>
      <td>1.0.3+galaxy1</td>
      <td>rcorrector</td>
      <td>1.0.7</td>
      <td>Sequencing error detection</td>
      <td>RNA, RNA-Seq</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>655</td>
      <td>655</td>
      <td>57</td>
      <td>57</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>655</td>
      <td>655</td>
      <td>57</td>
      <td>57</td>
    </tr>
    <tr>
      <td></td>
      <td>read_it_and_keep</td>
      <td>read_it_and_keep</td>
      <td>Rapid decontamination of SARS-CoV-2 sequencing reads</td>
      <td>read_it_and_keep</td>
      <td>read_it_and_keep</td>
      <td></td>
      <td>read_it_and_keep</td>
      <td>Read contamination removal</td>
      <td>Filtering, Genome alignment</td>
      <td>Pathology, Genomics</td>
      <td>To update</td>
      <td>https://github.com/GenomePathogenAnalysisService/read-it-and-keep</td>
      <td>Sequence Analysis</td>
      <td>read_it_and_keep</td>
      <td>2022-01-28</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/read-it-and-keep</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/read_it_and_keep</td>
      <td>0.2.2</td>
      <td>read-it-and-keep</td>
      <td>0.3.0</td>
      <td>Filtering, Genome alignment</td>
      <td>Pathology, Genomics</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>3710</td>
      <td>3710</td>
      <td>80</td>
      <td>80</td>
      <td>7</td>
      <td>7</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3717</td>
      <td>3717</td>
      <td>86</td>
      <td>86</td>
    </tr>
    <tr>
      <td></td>
      <td>recentrifuge</td>
      <td>recentrifuge</td>
      <td>"With Recentrifuge, researchers can analyze results from taxonomic classifiers using interactive charts with emphasis on the confidence level of the classifications.In addition to contamination-subtracted samples.Recentrifuge provides shared and exclusive taxa per sample,thus enabling robust contamination removal and comparative analysis in environmental and clinical metagenomics."</td>
      <td>Recentrifuge</td>
      <td>Recentrifuge</td>
      <td></td>
      <td>Recentrifuge</td>
      <td>Robust comparative analysis and contamination removal for metagenomics.</td>
      <td>Taxonomic classification, Expression analysis, Cross-assembly</td>
      <td>Metagenomics, Microbial ecology, Metagenomic sequencing</td>
      <td>Up-to-date</td>
      <td>https://github.com/khyox/recentrifuge</td>
      <td>Metagenomics</td>
      <td>recentrifuge</td>
      <td>2022-05-04</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/recentrifuge</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/recentrifuge</td>
      <td>1.15.0</td>
      <td>recentrifuge</td>
      <td>1.15.0</td>
      <td>Taxonomic classification, Expression analysis, Cross-assembly</td>
      <td>Metagenomics, Microbial ecology, Metagenomic sequencing</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>472</td>
      <td>472</td>
      <td>92</td>
      <td>92</td>
      <td>32</td>
      <td>32</td>
      <td>13</td>
      <td>13</td>
      <td>6</td>
      <td>6</td>
      <td>4</td>
      <td>4</td>
      <td>510</td>
      <td>510</td>
      <td>109</td>
      <td>109</td>
    </tr>
    <tr>
      <td></td>
      <td>red</td>
      <td>red</td>
      <td>Red (REpeat Detector)</td>
      <td>red</td>
      <td>red</td>
      <td></td>
      <td>RED</td>
      <td>This is a program to detect and visualize RNA editing events at genomic scale using next-generation sequencing data.</td>
      <td>RNA-Seq analysis, Editing</td>
      <td>RNA, Sequencing, Data visualisation</td>
      <td>Up-to-date</td>
      <td>https://github.com/BioinformaticsToolsmith/Red</td>
      <td>Sequence Analysis</td>
      <td>red</td>
      <td>2022-09-14</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/red</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/red</td>
      <td>2018.09.10</td>
      <td>red</td>
      <td>2018.09.10</td>
      <td>RNA-Seq analysis, Editing</td>
      <td>RNA, Sequencing, Data visualisation</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>859</td>
      <td>859</td>
      <td>203</td>
      <td>203</td>
      <td>305</td>
      <td>305</td>
      <td>142</td>
      <td>142</td>
      <td>227</td>
      <td>227</td>
      <td>58</td>
      <td>58</td>
      <td>1391</td>
      <td>1391</td>
      <td>403</td>
      <td>403</td>
    </tr>
    <tr>
      <td></td>
      <td>repeatmasker</td>
      <td>repeatmasker_wrapper</td>
      <td>RepeatMasker is a program that screens DNA sequences for interspersed repeats and low complexity DNA sequences.</td>
      <td>repeatmasker</td>
      <td>repeatmasker</td>
      <td></td>
      <td>RepeatMasker</td>
      <td>A program that screens DNA sequences for interspersed repeats and low complexity DNA sequences. The output of the program is a detailed annotation of the repeats that are present in the query sequence as well as a modified version of the query sequence in which all the annotated repeats have been masked (default: replaced by Ns).</td>
      <td>Genome annotation</td>
      <td>Sequence analysis, Sequence composition, complexity and repeats</td>
      <td>To update</td>
      <td>http://www.repeatmasker.org/</td>
      <td>Sequence Analysis</td>
      <td>repeat_masker</td>
      <td>2018-04-30</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/repeatmasker</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/repeatmasker</td>
      <td>4.1.5</td>
      <td>repeatmasker</td>
      <td>4.0.9_p2</td>
      <td>Genome annotation</td>
      <td>Sequence composition, complexity and repeats</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>5998</td>
      <td>5998</td>
      <td>810</td>
      <td>810</td>
      <td>7618</td>
      <td>7618</td>
      <td>803</td>
      <td>803</td>
      <td>3135</td>
      <td>3135</td>
      <td>292</td>
      <td>292</td>
      <td>16751</td>
      <td>16751</td>
      <td>1905</td>
      <td>1905</td>
    </tr>
    <tr>
      <td></td>
      <td>repeatmodeler</td>
      <td>repeatmodeler</td>
      <td>RepeatModeler - Model repetitive DNA</td>
      <td>repeatmodeler</td>
      <td>repeatmodeler</td>
      <td></td>
      <td>RepeatModeler</td>
      <td>De-novo repeat family identification and modeling package. At the heart of RepeatModeler are two de-novo repeat finding programs ( RECON and RepeatScout ) which employ complementary computational methods for identifying repeat element boundaries and family relationships from sequence data. RepeatModeler assists in automating the runs of RECON and RepeatScout given a genomic database and uses the output to build, refine and classify consensus models of putative interspersed repeats.</td>
      <td>Repeat sequence detection</td>
      <td>Sequence composition, complexity and repeats, Sequence composition, complexity and repeats</td>
      <td>To update</td>
      <td>https://www.repeatmasker.org/RepeatModeler/</td>
      <td>Genome annotation</td>
      <td>repeatmodeler</td>
      <td>2021-08-26</td>
      <td>csbl</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/repeatmodeler</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/repeatmodeler</td>
      <td>2.0.5</td>
      <td></td>
      <td></td>
      <td>Repeat sequence detection</td>
      <td>Sequence composition, complexity and repeats, Sequence composition, complexity and repeats</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>7963</td>
      <td>7963</td>
      <td>413</td>
      <td>413</td>
      <td>1485</td>
      <td>1485</td>
      <td>220</td>
      <td>220</td>
      <td>8402</td>
      <td>8402</td>
      <td>168</td>
      <td>168</td>
      <td>17850</td>
      <td>17850</td>
      <td>801</td>
      <td>801</td>
    </tr>
    <tr>
      <td></td>
      <td>repmatch_gff3</td>
      <td>repmatch_gff3</td>
      <td>Contains a tool that matches corresponding peak-pair midpoints from separate datasets based onuser-defined criteria.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>ChIP-seq</td>
      <td>repmatch_gff3</td>
      <td>2015-12-18</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/repmatch_gff3</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/repmatch_gff3</td>
      <td></td>
      <td>matplotlib</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>415</td>
      <td>721</td>
      <td>128</td>
      <td>219</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>415</td>
      <td>721</td>
      <td>128</td>
      <td>219</td>
    </tr>
    <tr>
      <td></td>
      <td>reshape2</td>
      <td>cast, melt</td>
      <td>Flexibly restructure and aggregate data using just the two functions melt and dcast</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://cran.r-project.org/web/packages/reshape2/index.html</td>
      <td>Text Manipulation</td>
      <td></td>
      <td>2017-05-26</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/reshape2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/reshape2</td>
      <td>1.4.2</td>
      <td>r-reshape2</td>
      <td></td>
      <td></td>
      <td></td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>3543</td>
      <td>3619</td>
      <td>108</td>
      <td>111</td>
      <td>536</td>
      <td>536</td>
      <td>157</td>
      <td>157</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4079</td>
      <td>4155</td>
      <td>265</td>
      <td>268</td>
    </tr>
    <tr>
      <td></td>
      <td>resize_coordinate_window</td>
      <td>resize_coordinate_window</td>
      <td>Contains a tool that modifies the start and stop coordinates of GFF data, expanding the coordinate windowby a specified size.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Genomic Interval Operations</td>
      <td>resize_coordinate_window</td>
      <td>2016-01-13</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/resize_coordinate_window</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/resize_coordinate_window</td>
      <td>1.0.2</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>75</td>
      <td>86</td>
      <td>8</td>
      <td>9</td>
      <td>138</td>
      <td>216</td>
      <td>30</td>
      <td>57</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>213</td>
      <td>302</td>
      <td>38</td>
      <td>66</td>
    </tr>
    <tr>
      <td></td>
      <td>revoluzer</td>
      <td>revoluzer_crex, revoluzer_distmat</td>
      <td>revoluzer wrappers</td>
      <td>revoluzer</td>
      <td>revoluzer</td>
      <td></td>
      <td>revoluzer</td>
      <td>Various tools for genome rearrangement analysis. CREx, TreeREx, etc</td>
      <td>Structural variation detection</td>
      <td>Molecular evolution, Phylogeny</td>
      <td>Up-to-date</td>
      <td>https://gitlab.com/Bernt/revoluzer/</td>
      <td>Phylogenetics</td>
      <td>revoluzer</td>
      <td>2024-03-19</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/revoluzer</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/revoluzer</td>
      <td>0.1.6</td>
      <td>revoluzer</td>
      <td>0.1.6</td>
      <td>Structural variation detection</td>
      <td>Molecular evolution, Phylogeny</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2444</td>
      <td>2444</td>
      <td>52</td>
      <td>52</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2444</td>
      <td>2444</td>
      <td>52</td>
      <td>52</td>
    </tr>
    <tr>
      <td></td>
      <td>rgcca</td>
      <td>rgcca</td>
      <td>multi-block analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/rgcca-factory/RGCCA</td>
      <td>Statistics</td>
      <td>rgcca</td>
      <td>2020-09-02</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/rgcca</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/rgcca</td>
      <td>3.0.2</td>
      <td>rgccacmd</td>
      <td>3.0.3</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>44</td>
      <td>44</td>
      <td>6</td>
      <td>6</td>
      <td>77</td>
      <td>77</td>
      <td>35</td>
      <td>35</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>121</td>
      <td>121</td>
      <td>41</td>
      <td>41</td>
    </tr>
    <tr>
      <td></td>
      <td>rgrnastar</td>
      <td>rna_star, rna_starsolo</td>
      <td>RNA STAR is an ultra fast universal RNA and scRNA-seq aligner and mapper</td>
      <td>star</td>
      <td>star</td>
      <td></td>
      <td>STAR</td>
      <td>Ultrafast universal RNA-seq data aligner</td>
      <td>Sequence alignment</td>
      <td>RNA-Seq, Transcriptomics</td>
      <td>To update</td>
      <td>https://github.com/alexdobin/STAR</td>
      <td>Next Gen Mappers, Transcriptomics</td>
      <td>rgrnastar</td>
      <td>2023-02-27</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/rgrnastar</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/rgrnastar</td>
      <td>2.7.11a</td>
      <td>star</td>
      <td>2.7.11b</td>
      <td>Sequence alignment</td>
      <td>RNA-Seq, Transcriptomics</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>438078</td>
      <td>473957</td>
      <td>10142</td>
      <td>10900</td>
      <td>325474</td>
      <td>403925</td>
      <td>15284</td>
      <td>18791</td>
      <td>49662</td>
      <td>53138</td>
      <td>1359</td>
      <td>1449</td>
      <td>813214</td>
      <td>931020</td>
      <td>26785</td>
      <td>31140</td>
    </tr>
    <tr>
      <td></td>
      <td>ribowaltz</td>
      <td>ribowaltz_process, ribowaltz_plot</td>
      <td>Calculation of optimal P-site offsets, diagnostic analysis and visual inspection of ribosome profiling data</td>
      <td>riboWaltz</td>
      <td>riboWaltz</td>
      <td></td>
      <td>riboWaltz</td>
      <td>riboWaltz is an R package for calculation of optimal P-site offsets, diagnostic analysis and visual inspection of ribosome profiling data.</td>
      <td></td>
      <td>Computational biology</td>
      <td>To update</td>
      <td>https://github.com/LabTranslationalArchitectomics/riboWaltz</td>
      <td>Transcriptomics, RNA</td>
      <td></td>
      <td>2022-09-22</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ribowaltz</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ribowaltz</td>
      <td>1.2.0</td>
      <td>ribowaltz</td>
      <td>2.0</td>
      <td></td>
      <td>Computational biology</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>171</td>
      <td>171</td>
      <td>20</td>
      <td>20</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>171</td>
      <td>171</td>
      <td>20</td>
      <td>20</td>
    </tr>
    <tr>
      <td></td>
      <td>rnaquast</td>
      <td>rna_quast</td>
      <td>rnaQuast (RNA Quality Assessment Tool) evaluates genome assemblies.</td>
      <td>rnaQUAST</td>
      <td>rnaQUAST</td>
      <td></td>
      <td>rnaQUAST</td>
      <td>Quality assessment tool for de novo transcriptome assemblies.</td>
      <td>De-novo assembly, Transcriptome assembly, Sequence assembly validation</td>
      <td>Sequence assembly, Transcriptomics, RNA-seq</td>
      <td>Up-to-date</td>
      <td>https://github.com/ablab/rnaquast</td>
      <td>Assembly, RNA</td>
      <td>rnaquast</td>
      <td>2023-06-03</td>
      <td>iuc</td>
      <td>https://git.ufz.de/lehmanju/rnaquast</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/rnaquast</td>
      <td>2.3.0</td>
      <td>rnaquast</td>
      <td>2.3.0</td>
      <td>De-novo assembly, Transcriptome assembly, Sequence assembly validation</td>
      <td>Sequence assembly, Transcriptomics</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1522</td>
      <td>1522</td>
      <td>254</td>
      <td>254</td>
      <td>372</td>
      <td>372</td>
      <td>75</td>
      <td>75</td>
      <td>5</td>
      <td>5</td>
      <td>1</td>
      <td>1</td>
      <td>1899</td>
      <td>1899</td>
      <td>330</td>
      <td>330</td>
    </tr>
    <tr>
      <td></td>
      <td>roary</td>
      <td>roary</td>
      <td>Roary the pangenome pipeline</td>
      <td>roary</td>
      <td>roary</td>
      <td></td>
      <td>Roary</td>
      <td>A high speed stand alone pan genome pipeline, which takes annotated assemblies in GFF3 format (produced by Prokka (Seemann, 2014)) and calculates the pan genome.</td>
      <td>Genome assembly</td>
      <td>DNA, Genomics, Mapping</td>
      <td>Up-to-date</td>
      <td>https://sanger-pathogens.github.io/Roary/</td>
      <td>Sequence Analysis</td>
      <td>roary</td>
      <td>2017-06-21</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/roary</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/roary</td>
      <td>3.13.0</td>
      <td>roary</td>
      <td>3.13.0</td>
      <td>Genome assembly</td>
      <td>DNA, Genomics, Mapping</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>15196</td>
      <td>15685</td>
      <td>1459</td>
      <td>1501</td>
      <td>12188</td>
      <td>12188</td>
      <td>1768</td>
      <td>1768</td>
      <td>4881</td>
      <td>5539</td>
      <td>518</td>
      <td>555</td>
      <td>32265</td>
      <td>33412</td>
      <td>3745</td>
      <td>3824</td>
    </tr>
    <tr>
      <td></td>
      <td>rp2biosensor</td>
      <td>rp2biosensor</td>
      <td>Build Sensing-Enabling Metabolic Pathways from RetroPath2.0 output</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/brsynth/rp2biosensor</td>
      <td>Synthetic Biology</td>
      <td>rp2biosensor</td>
      <td>2023-01-05</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/rp2biosensor</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/rp2biosensor</td>
      <td>3.2.1</td>
      <td>rp2biosensor</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>rp2paths</td>
      <td>rp2paths</td>
      <td>Enumerate and seperate the different pathways generated by RetroPath2.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/brsynth/rp2paths</td>
      <td>Synthetic Biology</td>
      <td>rp2paths</td>
      <td>2022-10-17</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/rp2paths</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/rp2paths</td>
      <td>1.5.1</td>
      <td>rp2paths</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>rpbasicdesign</td>
      <td>rpbasicdesign</td>
      <td>Extracting enzyme IDs from rpSBML files</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/brsynth/rpbasicdesign</td>
      <td>Synthetic Biology</td>
      <td>rpbasicdesign</td>
      <td>2022-03-30</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/rpbasicdesign</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/rpbasicdesign</td>
      <td>1.2.2</td>
      <td>rpbasicdesign</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>rpfba</td>
      <td>rpfba</td>
      <td>Perform FBA for the RetroPath2.0 heterologous pathways</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/brsynth/rptools/releases</td>
      <td>Synthetic Biology</td>
      <td>rpfba</td>
      <td>2022-01-17</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/rpfba</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/rpfba</td>
      <td>5.12.3</td>
      <td>rptools</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>rptools</td>
      <td>rptools_rpextractsink, rptools_rpfba, rptools_rpranker, rptools_rpreport, rptools_rpviz</td>
      <td>Suite of tools that work on rpSBML format</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/brsynth/rptools</td>
      <td>Synthetic Biology</td>
      <td>rptools</td>
      <td>2022-11-16</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/rptools</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/rptools</td>
      <td>5.13.1</td>
      <td>rptools</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>rrparser</td>
      <td>rrparser</td>
      <td>Reaction Rules Parser</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/brsynth/RRParser</td>
      <td>Synthetic Biology</td>
      <td>rrparser</td>
      <td>2022-06-27</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/rrparser</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/rrparser</td>
      <td>2.5.2</td>
      <td>rrparser</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>rseqc</td>
      <td>rseqc_FPKM_count, rseqc_RNA_fragment_size, rseqc_RPKM_saturation, rseqc_bam2wig, rseqc_bam_stat, rseqc_clipping_profile, rseqc_deletion_profile, rseqc_geneBody_coverage, rseqc_geneBody_coverage2, rseqc_infer_experiment, rseqc_inner_distance, rseqc_insertion_profile, rseqc_junction_annotation, rseqc_junction_saturation, rseqc_mismatch_profile, rseqc_read_GC, rseqc_read_NVC, rseqc_read_distribution, rseqc_read_duplication, rseqc_read_hexamer, rseqc_read_quality, rseqc_tin</td>
      <td>an RNA-seq quality control package</td>
      <td>rseqc</td>
      <td>rseqc</td>
      <td></td>
      <td>RSeQC</td>
      <td>Provides a number of useful modules that can comprehensively evaluate high throughput sequence data especially RNA-seq data. Some basic modules quickly inspect sequence quality, nucleotide composition bias, PCR bias and GC bias, while RNA-seq specific modules evaluate sequencing saturation, mapped reads distribution, coverage uniformity, strand specificity, transcript level RNA integrity etc.</td>
      <td>Data handling</td>
      <td>Sequencing</td>
      <td>To update</td>
      <td>https://code.google.com/p/rseqc/</td>
      <td>Convert Formats, Sequence Analysis, RNA, Transcriptomics, Visualization</td>
      <td>rseqc</td>
      <td>2017-02-27</td>
      <td>nilesh</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/rseqc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/rseqc</td>
      <td>5.0.3</td>
      <td>rseqc</td>
      <td>5.0.4</td>
      <td>Data handling</td>
      <td>Sequencing</td>
      <td>22</td>
      <td>22</td>
      <td>22</td>
      <td>22</td>
      <td>22</td>
      <td>22</td>
      <td>22</td>
      <td>22</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>22</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>22</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>22</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>22</td>
      <td>22</td>
      <td>22</td>
      <td>0</td>
      <td>136315</td>
      <td>147595</td>
      <td>8341</td>
      <td>9385</td>
      <td>138707</td>
      <td>161046</td>
      <td>10696</td>
      <td>12789</td>
      <td>22718</td>
      <td>23767</td>
      <td>1593</td>
      <td>1695</td>
      <td>297740</td>
      <td>332408</td>
      <td>20630</td>
      <td>23869</td>
    </tr>
    <tr>
      <td></td>
      <td>ruvseq</td>
      <td>ruvseq</td>
      <td>Remove Unwanted Variation from RNA-Seq Data</td>
      <td>ruvseq</td>
      <td>ruvseq</td>
      <td></td>
      <td>RUVSeq</td>
      <td>This package implements the remove unwanted variation (RUV) methods for the normalization of RNA-Seq read counts between samples.</td>
      <td>Differential gene expression analysis</td>
      <td>Gene expression, RNA-seq</td>
      <td>To update</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/DESeq2.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>ruvseq</td>
      <td>2018-08-30</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ruvseq</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ruvseq</td>
      <td>1.26.0</td>
      <td>bioconductor-ruvseq</td>
      <td>1.36.0</td>
      <td>Differential gene expression analysis</td>
      <td>Gene expression</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1286</td>
      <td>1364</td>
      <td>178</td>
      <td>193</td>
      <td>2220</td>
      <td>2220</td>
      <td>299</td>
      <td>299</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3506</td>
      <td>3584</td>
      <td>477</td>
      <td>492</td>
    </tr>
    <tr>
      <td></td>
      <td>salsa2</td>
      <td>salsa</td>
      <td>A tool to scaffold long read assemblies with Hi-C</td>
      <td>SALSA</td>
      <td>SALSA</td>
      <td></td>
      <td>SALSA</td>
      <td>&gt; VERY_LOW CONFIDENCE! | &gt; CORRECT NAME OF TOOL COULD ALSO BE 'chromosome-scale', 'reference-quality', 'Hi-C', 'scaffolder' | Integrating Hi-C links with assembly graphs for chromosome-scale assembly | SALSA: A tool to scaffold long read assemblies with Hi-C data | SALSA: A tool to scaffold long read assemblies with Hi-C | This code is used to scaffold your assemblies using Hi-C data. This version implements some improvements in the original SALSA algorithm. If you want to use the old version, it can be found in the old_salsa branch</td>
      <td>Genome assembly, De-novo assembly, Scaffolding</td>
      <td>Sequence assembly, DNA binding sites, Mapping</td>
      <td>Up-to-date</td>
      <td>https://github.com/marbl/SALSA</td>
      <td>Assembly</td>
      <td>salsa</td>
      <td>2021-01-14</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/salsa2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/salsa2</td>
      <td>2.3</td>
      <td>salsa2</td>
      <td>2.3</td>
      <td>Genome assembly, De-novo assembly, Scaffolding</td>
      <td>Sequence assembly, DNA binding sites, Mapping</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>463</td>
      <td>463</td>
      <td>119</td>
      <td>119</td>
      <td>1038</td>
      <td>1038</td>
      <td>87</td>
      <td>87</td>
      <td>410</td>
      <td>410</td>
      <td>24</td>
      <td>24</td>
      <td>1911</td>
      <td>1911</td>
      <td>230</td>
      <td>230</td>
    </tr>
    <tr>
      <td></td>
      <td>samblaster</td>
      <td>samblaster</td>
      <td>samblaster marks duplicates and can output split and discordant alignments from SAM/BAM files</td>
      <td>samblaster</td>
      <td>samblaster</td>
      <td></td>
      <td>SAMBLASTER</td>
      <td>A tool to mark duplicates and extract discordant and split reads from SAM files.</td>
      <td>Split read mapping</td>
      <td>DNA, Sequencing, Mapping</td>
      <td>Up-to-date</td>
      <td>https://github.com/GregoryFaust/samblaster</td>
      <td>SAM, Fastq Manipulation, Variant Analysis</td>
      <td>samblaster</td>
      <td>2016-12-18</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/samblaster</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/samblaster</td>
      <td>0.1.26</td>
      <td>samblaster</td>
      <td>0.1.26</td>
      <td>Split read mapping</td>
      <td>DNA, Sequencing, Mapping</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>samtools</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/samtools/samtools</td>
      <td>SAM</td>
      <td></td>
      <td>2021-01-04</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/samtools</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/samtools</td>
      <td>1.20</td>
      <td>samtools</td>
      <td>1.21</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>sansa</td>
      <td>sansa_annotate</td>
      <td>Sansa is a tool for structural variant annotation.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/dellytools/sansa</td>
      <td>Variant Analysis</td>
      <td>sansa</td>
      <td>2020-12-05</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/sansa</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/sansa</td>
      <td>0.2.2</td>
      <td>sansa</td>
      <td>0.2.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>138</td>
      <td>138</td>
      <td>36</td>
      <td>36</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>138</td>
      <td>138</td>
      <td>36</td>
      <td>36</td>
    </tr>
    <tr>
      <td></td>
      <td>sarscov2formatter</td>
      <td>sarscov2formatter</td>
      <td>sarscov2formatter custom script</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/nickeener/sarscov2formatter</td>
      <td>Sequence Analysis</td>
      <td>sarscov2formatter</td>
      <td>2020-05-01</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/sarscov2formatter</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/sarscov2formatter</td>
      <td>1.0</td>
      <td>sarscov2formatter</td>
      <td>1.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>174</td>
      <td>174</td>
      <td>16</td>
      <td>16</td>
      <td>399</td>
      <td>399</td>
      <td>29</td>
      <td>29</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>573</td>
      <td>573</td>
      <td>45</td>
      <td>45</td>
    </tr>
    <tr>
      <td></td>
      <td>sarscov2summary</td>
      <td>sarscov2summary</td>
      <td>sarscov2summary custom script</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/nickeener/sarscov2summary</td>
      <td>Sequence Analysis</td>
      <td>sarscov2summary</td>
      <td>2020-05-01</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/sarscov2summary</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/sarscov2summary</td>
      <td>0.1</td>
      <td>sarscov2summary</td>
      <td>0.5</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>140</td>
      <td>140</td>
      <td>5</td>
      <td>5</td>
      <td>253</td>
      <td>253</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>393</td>
      <td>393</td>
      <td>11</td>
      <td>11</td>
    </tr>
    <tr>
      <td></td>
      <td>sbml2sbol</td>
      <td>sbml2sbol</td>
      <td>Convert SBML to SBOL format</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/neilswainston/SbmlToSbol</td>
      <td>Synthetic Biology</td>
      <td>sbml2sbol</td>
      <td>2022-10-15</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/sbml2sbol</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/sbml2sbol</td>
      <td>0.1.13</td>
      <td>sbml2sbol</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>scanpy</td>
      <td>scanpy_cluster_reduce_dimension, scanpy_filter, scanpy_inspect, scanpy_normalize, scanpy_plot, scanpy_remove_confounders</td>
      <td>Scanpy  Single-Cell Analysis in Python</td>
      <td>scanpy</td>
      <td>scanpy</td>
      <td></td>
      <td>SCANPY</td>
      <td>Scalable toolkit for analyzing single-cell gene expression data. It includes preprocessing, visualization, clustering, pseudotime and trajectory inference and differential expression testing. The Python-based implementation efficiently deals with datasets of more than one million cells.</td>
      <td>Differential gene expression analysis</td>
      <td>Gene expression, Cell biology, Genetics</td>
      <td>To update</td>
      <td>https://scanpy.readthedocs.io</td>
      <td>Single Cell, Spatial Omics, Transcriptomics</td>
      <td>scanpy</td>
      <td>2024-09-03</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/scanpy/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/scanpy</td>
      <td>1.10.2</td>
      <td>scanpy</td>
      <td>1.7.2</td>
      <td>Differential gene expression analysis</td>
      <td>Gene expression, Cell biology, Genetics</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>63789</td>
      <td>64034</td>
      <td>2543</td>
      <td>2547</td>
      <td>29962</td>
      <td>29962</td>
      <td>1185</td>
      <td>1185</td>
      <td>18028</td>
      <td>18028</td>
      <td>379</td>
      <td>379</td>
      <td>111779</td>
      <td>112024</td>
      <td>4107</td>
      <td>4111</td>
    </tr>
    <tr>
      <td></td>
      <td>scater</td>
      <td>scater_create_qcmetric_ready_sce, scater_filter, scater_plot_dist_scatter, scater_plot_pca, scater_plot_tsne</td>
      <td>Scater (Single-Cell Analysis Toolkit for gene Expression data in R) is acollection of tools for doing various analyses of single-cell RNA-seq geneexpression data, with a focus on quality control and visualization.</td>
      <td>scater</td>
      <td>scater</td>
      <td></td>
      <td>scater</td>
      <td>Pre-processing, quality control, normalization and visualization of single-cell RNA-seq data.</td>
      <td>Read pre-processing, Sequencing quality control, Sequence visualisation</td>
      <td>RNA-seq, Quality affairs, Molecular genetics</td>
      <td>To update</td>
      <td>http://bioconductor.org/packages/scater/</td>
      <td>Single Cell, Transcriptomics, Visualization</td>
      <td></td>
      <td>2019-07-18</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/scater</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/scater</td>
      <td>1.22.0</td>
      <td>bioconductor-scater</td>
      <td>1.30.1</td>
      <td>Read pre-processing, Sequence visualisation</td>
      <td>Quality affairs, Molecular genetics</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>1345</td>
      <td>1371</td>
      <td>350</td>
      <td>351</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>903</td>
      <td>903</td>
      <td>29</td>
      <td>29</td>
      <td>2248</td>
      <td>2274</td>
      <td>379</td>
      <td>380</td>
    </tr>
    <tr>
      <td></td>
      <td>sceasy</td>
      <td>sceasy_convert</td>
      <td>Converter between difference single-cell formats</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/cellgeni/sceasy/</td>
      <td>Single Cell, Spatial Omics, Convert Formats</td>
      <td>sceasy_convert</td>
      <td>2023-11-10</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/sceasy/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/sceasy</td>
      <td>0.0.7</td>
      <td>r-sceasy</td>
      <td>0.0.7</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1059</td>
      <td>1059</td>
      <td>222</td>
      <td>222</td>
      <td>426</td>
      <td>426</td>
      <td>71</td>
      <td>71</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1486</td>
      <td>1486</td>
      <td>294</td>
      <td>294</td>
    </tr>
    <tr>
      <td></td>
      <td>schicexplorer</td>
      <td>schicexplorer_schicadjustmatrix, schicexplorer_schiccluster, schicexplorer_schicclustercompartments, schicexplorer_schicclusterminhash, schicexplorer_schicclustersvl, schicexplorer_schicconsensusmatrices, schicexplorer_schiccorrectmatrices, schicexplorer_schiccreatebulkmatrix, schicexplorer_schicdemultiplex, schicexplorer_schicinfo, schicexplorer_schicmergematrixbins, schicexplorer_schicmergetoscool, schicexplorer_schicnormalize, schicexplorer_schicplotclusterprofiles, schicexplorer_schicplotconsensusmatrices, schicexplorer_schicqualitycontrol</td>
      <td>scHiCExplorer: Set of programs to process, analyze and visualize scHi-C data.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/joachimwolff/schicexplorer</td>
      <td>Sequence Analysis, Transcriptomics, Visualization, Single Cell</td>
      <td>schicexplorer</td>
      <td>2019-12-16</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/schicexplorer</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/schicexplorer</td>
      <td>4</td>
      <td>schicexplorer</td>
      <td>7</td>
      <td></td>
      <td></td>
      <td>16</td>
      <td>16</td>
      <td>16</td>
      <td>0</td>
      <td>16</td>
      <td>16</td>
      <td>16</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>16</td>
      <td>16</td>
      <td>16</td>
      <td>0</td>
      <td>809</td>
      <td>809</td>
      <td>78</td>
      <td>78</td>
      <td>3</td>
      <td>3</td>
      <td>2</td>
      <td>2</td>
      <td>105</td>
      <td>105</td>
      <td>16</td>
      <td>16</td>
      <td>917</td>
      <td>917</td>
      <td>96</td>
      <td>96</td>
    </tr>
    <tr>
      <td></td>
      <td>scikit-bio</td>
      <td>scikit_bio_diversity_beta_diversity</td>
      <td>scikit-bio: an open-source, BSD-licensed, python package providing data structures, algorithms, and educational resources for bioinformatics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://scikit-bio.org/</td>
      <td>Sequence Analysis</td>
      <td>scikit_bio</td>
      <td>2016-05-31</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/scikit_bio</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/scikit-bio</td>
      <td>0.4.2</td>
      <td>scikit-bio</td>
      <td>0.4.2</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1503</td>
      <td>1955</td>
      <td>283</td>
      <td>402</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1504</td>
      <td>1956</td>
      <td>284</td>
      <td>403</td>
    </tr>
    <tr>
      <td></td>
      <td>scoary</td>
      <td>scoary</td>
      <td>Scoary calculates the assocations between all genes in the accessory genome and the traits.</td>
      <td>scoary</td>
      <td>scoary</td>
      <td></td>
      <td>Scoary</td>
      <td>Pan-genome wide association studies and  is designed to take the gene_presence_absence.csv file from Roary as well as a traits file created by the user and calculate the assocations between all genes in the accessory genome (all genes that are present in i genomes where 1 &lt; i &lt; N) and the traits. It reports a list of genes sorted by strength of association per trait.</td>
      <td>Analysis</td>
      <td>Genotype and phenotype, Model organisms, GWAS study, Functional genomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/AdmiralenOla/Scoary</td>
      <td>Metagenomics</td>
      <td>scoary</td>
      <td>2021-03-18</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/scoary</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/scoary</td>
      <td>1.6.16</td>
      <td>scoary</td>
      <td>1.6.16</td>
      <td>Analysis</td>
      <td>Genotype and phenotype, Model organisms, GWAS study, Functional genomics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>901</td>
      <td>901</td>
      <td>90</td>
      <td>90</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>901</td>
      <td>901</td>
      <td>90</td>
      <td>90</td>
    </tr>
    <tr>
      <td></td>
      <td>scpipe</td>
      <td>scpipe</td>
      <td>A flexible preprocessing pipeline for single-cell RNA-sequencing data</td>
      <td>scpipe</td>
      <td>scpipe</td>
      <td></td>
      <td>scPipe</td>
      <td>A preprocessing pipeline for single cell RNA-seq data that starts from the fastq files and produces a gene count matrix with associated quality control information. It can process fastq data generated by CEL-seq, MARS-seq, Drop-seq, Chromium 10x and SMART-seq protocols.</td>
      <td>Genome annotation, Validation, Alignment, Visualisation</td>
      <td>Gene expression, RNA-Seq, Sequencing</td>
      <td>To update</td>
      <td>http://bioconductor.org/packages/release/bioc/html/scPipe.html</td>
      <td>Transcriptomics, Single Cell</td>
      <td>scpipe</td>
      <td>2018-08-13</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/scpipe</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/scpipe</td>
      <td>1.0.0+galaxy2</td>
      <td>bioconductor-scpipe</td>
      <td>2.2.0</td>
      <td>Genome annotation, Validation, Alignment, Visualisation</td>
      <td>Gene expression, RNA-Seq</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>284</td>
      <td>632</td>
      <td>34</td>
      <td>38</td>
      <td>786</td>
      <td>786</td>
      <td>61</td>
      <td>61</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1071</td>
      <td>1419</td>
      <td>96</td>
      <td>100</td>
    </tr>
    <tr>
      <td></td>
      <td>seacr</td>
      <td>seacr</td>
      <td>SEACR is intended to call peaks and enriched regions from sparse CUT&amp;RUN or chromatin profiling data in which background is dominated by zeroes.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/FredHutch/SEACR</td>
      <td>Epigenetics, ChIP-seq</td>
      <td>seacr</td>
      <td>2020-02-05</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/seacr</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/seacr</td>
      <td>1.3</td>
      <td>seacr</td>
      <td>1.3</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>5106</td>
      <td>5106</td>
      <td>116</td>
      <td>116</td>
      <td>6419</td>
      <td>6419</td>
      <td>241</td>
      <td>241</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>11525</td>
      <td>11525</td>
      <td>357</td>
      <td>357</td>
    </tr>
    <tr>
      <td></td>
      <td>selenzy_wrapper</td>
      <td>selenzy_wrapper</td>
      <td>Performs enzyme selection from a reaction query.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/brsynth/selenzy-wrapper</td>
      <td>Synthetic Biology</td>
      <td>selenzy_wrapper</td>
      <td>2022-06-29</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/selenzy_wrapper</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/selenzy_wrapper</td>
      <td>0.3.0</td>
      <td>selenzy_wrapper</td>
      <td>0.3.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>semibin</td>
      <td>semibin_bin, semibin_concatenate_fasta, semibin_generate_cannot_links, semibin_generate_sequence_features, semibin, semibin_train</td>
      <td>SemiBin: Semi-supervised Metagenomic Binning Using Siamese Neural Networks</td>
      <td>semibin</td>
      <td>semibin</td>
      <td></td>
      <td>SemiBin</td>
      <td>Command tool for metagenomic binning with semi-supervised deep learning using information from reference genomes.</td>
      <td>Sequence assembly, Read binning</td>
      <td>Metagenomics, Machine learning, Microbial ecology, Sequence assembly</td>
      <td>To update</td>
      <td>https://semibin.readthedocs.io/en/latest/</td>
      <td>Metagenomics</td>
      <td>semibin</td>
      <td>2022-10-14</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/semibin</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/semibin</td>
      <td>2.0.2</td>
      <td>semibin</td>
      <td>2.1.0</td>
      <td>Sequence assembly, Read binning</td>
      <td>Metagenomics, Machine learning, Microbial ecology, Sequence assembly</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>931</td>
      <td>931</td>
      <td>93</td>
      <td>93</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>931</td>
      <td>931</td>
      <td>93</td>
      <td>93</td>
    </tr>
    <tr>
      <td></td>
      <td>seq2hla</td>
      <td>seq2hla</td>
      <td>Precision HLA typing and expression from RNAseq data</td>
      <td>seq2hla</td>
      <td>seq2hla</td>
      <td></td>
      <td>Seq2HLA</td>
      <td>seq2HLA is a computational tool to determine Human Leukocyte Antigen (HLA) directly from existing and future short RNA-Seq reads. It takes standard RNA-Seq sequence reads in fastq format as input, uses a bowtie index comprising known HLA alleles and outputs the most likely HLA class I and class II types, a p-value for each call, and the expression of each class.</td>
      <td>Read mapping, Genetic variation analysis</td>
      <td>Transcriptomics, Mapping</td>
      <td>Up-to-date</td>
      <td>https://github.com/TRON-Bioinformatics/seq2HLA</td>
      <td>Sequence Analysis</td>
      <td>seq2hla</td>
      <td>2020-02-20</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/seq2hla</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/seq2hla</td>
      <td>2.3</td>
      <td>seq2hla</td>
      <td>2.3</td>
      <td>Read mapping, Genetic variation analysis</td>
      <td>Transcriptomics, Mapping</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>342</td>
      <td>342</td>
      <td>40</td>
      <td>40</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>342</td>
      <td>342</td>
      <td>40</td>
      <td>40</td>
    </tr>
    <tr>
      <td></td>
      <td>seqcomplexity</td>
      <td>seqcomplexity</td>
      <td>Sequence complexity for raw reads</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/stevenweaver/seqcomplexity</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>2022-06-07</td>
      <td>iuc</td>
      <td>https://github.com/stephenshank/tools-iuc/tree/seqcomplexity/tools/seqcomplexity/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/seqcomplexity</td>
      <td>0.1.2</td>
      <td>seqcomplexity</td>
      <td>0.1.2</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>70</td>
      <td>70</td>
      <td>19</td>
      <td>19</td>
      <td>109</td>
      <td>109</td>
      <td>47</td>
      <td>47</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>179</td>
      <td>179</td>
      <td>66</td>
      <td>66</td>
    </tr>
    <tr>
      <td></td>
      <td>seqkit</td>
      <td>seqkit_fx2tab, seqkit_locate, seqkit_sort, seqkit_stats, seqkit_translate</td>
      <td>A cross-platform and ultrafast toolkit for FASTA/Q file manipulation</td>
      <td>seqkit</td>
      <td>seqkit</td>
      <td></td>
      <td>seqkit</td>
      <td>FASTA and FASTQ are basic and ubiquitous formats for storing nucleotide and protein sequences. Common manipulations of FASTA/Q file include converting, searching, filtering, deduplication, splitting, shuffling, and sampling. Existing tools only implement some of these manipulations, and not particularly efficiently, and some are only available for certain operating systems. Furthermore, the complicated installation process of required packages and running environments can render these programs less user friendly. SeqKit demonstrates competitive performance in execution time and memory usage compared to similar tools. The efficiency and usability of SeqKit enable researchers to rapidly accomplish common FASTA/Q file manipulations.</td>
      <td>DNA transcription, Sequence trimming, DNA translation, Sequence conversion</td>
      <td>Database management, Sequence analysis</td>
      <td>Up-to-date</td>
      <td>https://bioinf.shenwei.me/seqkit/</td>
      <td>Sequence Analysis</td>
      <td>seqkit</td>
      <td>2022-06-01</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/seqkit</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/seqkit</td>
      <td>2.8.2</td>
      <td>seqkit</td>
      <td>2.8.2</td>
      <td>DNA transcription, Sequence trimming, DNA translation</td>
      <td>Database management, Sequence analysis</td>
      <td>0</td>
      <td>2</td>
      <td>5</td>
      <td>3</td>
      <td>0</td>
      <td>2</td>
      <td>5</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>3364</td>
      <td>3364</td>
      <td>247</td>
      <td>247</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>244</td>
      <td>244</td>
      <td>33</td>
      <td>33</td>
      <td>3608</td>
      <td>3608</td>
      <td>280</td>
      <td>280</td>
    </tr>
    <tr>
      <td></td>
      <td>seqprep</td>
      <td>seqprep</td>
      <td>Tool for merging paired-end Illumina reads and trimming adapters.</td>
      <td>seqprep</td>
      <td>seqprep</td>
      <td></td>
      <td>SeqPrep</td>
      <td>Strips adapters and optionally merges overlapping paired-end (or paired-end contamination in mate-pair libraries) illumina style reads.</td>
      <td>Nucleic acid design</td>
      <td>Genomics, Sequence assembly, Sequencing, Probes and primers</td>
      <td>Up-to-date</td>
      <td>https://github.com/jstjohn/SeqPrep</td>
      <td>Fastq Manipulation, Sequence Analysis</td>
      <td>seqprep</td>
      <td>2024-01-15</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/seqprep</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/seqprep</td>
      <td>1.3.2</td>
      <td>seqprep</td>
      <td>1.3.2</td>
      <td>Nucleic acid design</td>
      <td>Genomics, Sequence assembly, Sequencing, Probes and primers</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2424</td>
      <td>2424</td>
      <td>102</td>
      <td>102</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2424</td>
      <td>2424</td>
      <td>102</td>
      <td>102</td>
    </tr>
    <tr>
      <td></td>
      <td>seqsero2</td>
      <td>seqsero2</td>
      <td>Salmonella serotype prediction from genome sequencing data</td>
      <td>seqsero2</td>
      <td>seqsero2</td>
      <td></td>
      <td>SeqSero2</td>
      <td>rapid and improved Salmonella serotype determination using whole genome sequencing data | SeqSero-Salmonella Serotyping by Whole Genome Sequencing | Salmonella Serotyping by Whole Genome Sequencing | Online version: http://www.denglab.info/SeqSero2 | Salmonella serotype prediction from genome sequencing data | Citation: SeqSero, Zhang et al. 2015; SeqSero2, Zhang et al. 2019 | Salmonella serotype derterminants databases | Upon executing the command, a directory named 'SeqSero_result_Time_your_run' will be created. Your result will be stored in 'SeqSero_result.txt' in that directory. And the assembled alleles can also be found in the directory if using "-m a" (allele mode)</td>
      <td>Genome indexing, Antimicrobial resistance prediction, Genome alignment</td>
      <td>Whole genome sequencing, Sequence assembly, Genomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/denglab/SeqSero2</td>
      <td>Sequence Analysis</td>
      <td>seqsero2</td>
      <td>2023-11-07</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/seqsero2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/seqsero2</td>
      <td>1.3.1</td>
      <td>seqsero2</td>
      <td>1.3.1</td>
      <td>Genome indexing, Antimicrobial resistance prediction, Genome alignment</td>
      <td>Whole genome sequencing, Sequence assembly, Genomics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4182</td>
      <td>4182</td>
      <td>42</td>
      <td>42</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4182</td>
      <td>4182</td>
      <td>42</td>
      <td>42</td>
    </tr>
    <tr>
      <td></td>
      <td>seqtk</td>
      <td>seqtk_comp, seqtk_cutN, seqtk_dropse, seqtk_fqchk, seqtk_hety, seqtk_listhet, seqtk_mergefa, seqtk_mergepe, seqtk_mutfa, seqtk_randbase, seqtk_sample, seqtk_seq, seqtk_subseq, seqtk_telo, seqtk_trimfq</td>
      <td>Toolkit for processing sequences in FASTA/Q formats</td>
      <td>seqtk</td>
      <td>seqtk</td>
      <td></td>
      <td>seqtk</td>
      <td>A tool for processing sequences in the FASTA or FASTQ format. It parses both FASTA and FASTQ files which can also be optionally compressed by gzip.</td>
      <td>Data handling, Sequence file editing</td>
      <td>Data management</td>
      <td>Up-to-date</td>
      <td>https://github.com/lh3/seqtk</td>
      <td>Sequence Analysis</td>
      <td>seqtk</td>
      <td>2017-01-20</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/seqtk</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/seqtk</td>
      <td>1.4</td>
      <td>seqtk</td>
      <td>1.4</td>
      <td>Data handling, Sequence file editing</td>
      <td></td>
      <td>15</td>
      <td>15</td>
      <td>15</td>
      <td>15</td>
      <td>15</td>
      <td>15</td>
      <td>15</td>
      <td>15</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>14</td>
      <td>0</td>
      <td>14</td>
      <td>0</td>
      <td>0</td>
      <td>14</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>15</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>14</td>
      <td>14</td>
      <td>14</td>
      <td>14</td>
      <td>72816</td>
      <td>73514</td>
      <td>2210</td>
      <td>2300</td>
      <td>102777</td>
      <td>107831</td>
      <td>2313</td>
      <td>2669</td>
      <td>154613</td>
      <td>154613</td>
      <td>682</td>
      <td>682</td>
      <td>330206</td>
      <td>335958</td>
      <td>5205</td>
      <td>5651</td>
    </tr>
    <tr>
      <td></td>
      <td>seqwish</td>
      <td>seqwish</td>
      <td>Alignment to variation graph inducer</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/seqwish</td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td></td>
      <td>2020-04-06</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/seqwish/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/seqwish</td>
      <td>0.7.5</td>
      <td>seqwish</td>
      <td>0.7.11</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>271</td>
      <td>271</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>271</td>
      <td>271</td>
      <td>2</td>
      <td>2</td>
    </tr>
    <tr>
      <td></td>
      <td>seurat</td>
      <td>seurat_create, seurat_data, seurat_integrate, seurat_clustering, seurat_preprocessing, seurat_plot, seurat_reduce_dimension</td>
      <td>Seurat - R toolkit for single cell genomics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://satijalab.org/seurat/</td>
      <td>Single Cell, Transcriptomics, Sequence Analysis</td>
      <td>seurat</td>
      <td>2020-07-22</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/seurat_v5</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/seurat</td>
      <td>5.0</td>
      <td>r-seurat</td>
      <td>3.0.2</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>7</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>7</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>121</td>
      <td>121</td>
      <td>11</td>
      <td>11</td>
      <td>32</td>
      <td>32</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>153</td>
      <td>153</td>
      <td>16</td>
      <td>16</td>
    </tr>
    <tr>
      <td></td>
      <td>shasta</td>
      <td>shasta</td>
      <td>Fast de novo assembly of long read sequencing data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/chanzuckerberg/shasta</td>
      <td>Assembly, Nanopore</td>
      <td>shasta</td>
      <td>2020-11-11</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/shasta</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/shasta</td>
      <td>0.6.0</td>
      <td>shasta</td>
      <td>0.13.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>913</td>
      <td>913</td>
      <td>308</td>
      <td>308</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>568</td>
      <td>568</td>
      <td>73</td>
      <td>73</td>
      <td>1481</td>
      <td>1481</td>
      <td>381</td>
      <td>381</td>
    </tr>
    <tr>
      <td></td>
      <td>shorah</td>
      <td>shorah_amplicon</td>
      <td>Reconstruct haplotypes using ShoRAH in amplicon mode</td>
      <td>shorah</td>
      <td>shorah</td>
      <td></td>
      <td>ShoRAH</td>
      <td>Inference of a population from a set of short reads. The package contains programs that support mapping of reads to a reference genome, correcting sequencing errors by locally clustering reads in small windows of the alignment, reconstructing a minimal set of global haplotypes that explain the reads, and estimating the frequencies of the inferred haplotypes.</td>
      <td>Haplotype mapping, Variant calling</td>
      <td>Metagenomics, Sequencing, Genetics</td>
      <td>To update</td>
      <td>https://github.com/cbg-ethz/shorah/blob/master/README.md</td>
      <td>Sequence Analysis</td>
      <td>shorah_amplicon</td>
      <td>2018-11-27</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/shorah</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/shorah</td>
      <td>1.1.3</td>
      <td>shorah</td>
      <td>1.99.2</td>
      <td>Haplotype mapping, Variant calling</td>
      <td>Metagenomics, Sequencing, Genetics</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>shovill</td>
      <td>shovill</td>
      <td>Faster de novo assembly pipeline based around Spades</td>
      <td>shovill</td>
      <td>shovill</td>
      <td></td>
      <td>shovill</td>
      <td>Shovill is a pipeline for assembly of bacterial isolate genomes from Illumina paired-end reads.  Shovill uses SPAdes at its core, but alters the steps before and after the primary assembly step to get similar results in less time. Shovill also supports other assemblers like SKESA, Velvet and Megahit, so you can take advantage of the pre- and post-processing the Shovill provides with those too.</td>
      <td>Genome assembly</td>
      <td>Genomics, Microbiology, Sequence assembly</td>
      <td>Up-to-date</td>
      <td>https://github.com/tseemann/shovill</td>
      <td>Assembly</td>
      <td>shovill</td>
      <td>2017-10-24</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/shovill</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/shovill</td>
      <td>1.1.0</td>
      <td>shovill</td>
      <td>1.1.0</td>
      <td>Genome assembly</td>
      <td>Genomics, Microbiology, Sequence assembly</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>46125</td>
      <td>47376</td>
      <td>2273</td>
      <td>2356</td>
      <td>40577</td>
      <td>40577</td>
      <td>3971</td>
      <td>3971</td>
      <td>19960</td>
      <td>21130</td>
      <td>1085</td>
      <td>1199</td>
      <td>106662</td>
      <td>109083</td>
      <td>7329</td>
      <td>7526</td>
    </tr>
    <tr>
      <td></td>
      <td>sickle</td>
      <td>sickle</td>
      <td>A windowed adaptive trimming tool for FASTQ files using quality</td>
      <td>sickle</td>
      <td>sickle</td>
      <td></td>
      <td>sickle</td>
      <td>A  tool that uses sliding windows along with quality and length thresholds to determine when quality is sufficiently low to trim the 3'-end of reads and also determines when the quality is sufficiently high enough to trim the 5'-end of reads.</td>
      <td>Sequence trimming</td>
      <td>Data quality management</td>
      <td>To update</td>
      <td>https://github.com/najoshi/sickle</td>
      <td>Fastq Manipulation, Sequence Analysis</td>
      <td>sickle</td>
      <td>2015-11-03</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/sickle</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/sickle</td>
      <td>1.33.2</td>
      <td>sickle-trim</td>
      <td>1.33</td>
      <td>Sequence trimming</td>
      <td>Data quality management</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>18184</td>
      <td>18937</td>
      <td>555</td>
      <td>608</td>
      <td>10514</td>
      <td>10514</td>
      <td>1063</td>
      <td>1063</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>28698</td>
      <td>29451</td>
      <td>1618</td>
      <td>1671</td>
    </tr>
    <tr>
      <td></td>
      <td>simtext</td>
      <td>abstracts_by_pmids, pmids_to_pubtator_matrix, pubmed_by_queries, text_to_wordmatrix</td>
      <td>A text mining framework for interactive analysis and visualization of similarities among biomedical entities.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/dlal-group/simtext</td>
      <td>Text Manipulation</td>
      <td>simtext</td>
      <td>2021-02-28</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tools/simtext</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/simtext</td>
      <td>0.0.2</td>
      <td>r-argparse</td>
      <td>1.0.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>2</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>220</td>
      <td>220</td>
      <td>68</td>
      <td>68</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>416</td>
      <td>416</td>
      <td>10</td>
      <td>10</td>
      <td>636</td>
      <td>636</td>
      <td>78</td>
      <td>78</td>
    </tr>
    <tr>
      <td></td>
      <td>sina</td>
      <td>sina</td>
      <td>SINA reference based multiple sequence alignment</td>
      <td>sina</td>
      <td>sina</td>
      <td></td>
      <td>SINA</td>
      <td>Aligns and optionally taxonomically classifies your rRNA gene sequences.Reference based multiple sequence alignment</td>
      <td>Sequence alignment analysis, Multiple sequence alignment, Taxonomic classification, Structure-based sequence alignment</td>
      <td>Sequencing, RNA, Nucleic acid structure analysis, Taxonomy, Sequence analysis, Taxonomy</td>
      <td>Up-to-date</td>
      <td>https://sina.readthedocs.io/en/latest/</td>
      <td>Sequence Analysis</td>
      <td>sina</td>
      <td>2019-10-11</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/sina</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/sina</td>
      <td>1.7.2</td>
      <td>sina</td>
      <td>1.7.2</td>
      <td>Sequence alignment analysis, Multiple sequence alignment, Taxonomic classification, Structure-based sequence alignment</td>
      <td>Sequencing, RNA, Nucleic acid structure analysis, Taxonomy, Sequence analysis, Taxonomy</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1369</td>
      <td>1369</td>
      <td>125</td>
      <td>125</td>
      <td>2398</td>
      <td>2398</td>
      <td>420</td>
      <td>420</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3767</td>
      <td>3767</td>
      <td>545</td>
      <td>545</td>
    </tr>
    <tr>
      <td></td>
      <td>sinto</td>
      <td>sinto_barcode, sinto_fragments</td>
      <td>Sinto single-cell analysis tools</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/timoast/sinto</td>
      <td>Single Cell, Epigenetics</td>
      <td></td>
      <td>2023-04-12</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/sinto</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/sinto</td>
      <td>0.10.1</td>
      <td>sinto</td>
      <td>0.10.1</td>
      <td></td>
      <td></td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>284</td>
      <td>284</td>
      <td>70</td>
      <td>70</td>
      <td>165</td>
      <td>165</td>
      <td>38</td>
      <td>38</td>
      <td>14</td>
      <td>14</td>
      <td>2</td>
      <td>2</td>
      <td>463</td>
      <td>463</td>
      <td>110</td>
      <td>110</td>
    </tr>
    <tr>
      <td></td>
      <td>slamdunk</td>
      <td>alleyoop, slamdunk</td>
      <td>Slamdunk maps and quantifies SLAMseq reads</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://t-neumann.github.io/slamdunk</td>
      <td>RNA, Transcriptomics, Sequence Analysis, Next Gen Mappers</td>
      <td>slamdunk</td>
      <td>2019-01-20</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/slamdunk</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/slamdunk</td>
      <td>0.4.3</td>
      <td>slamdunk</td>
      <td>0.4.3</td>
      <td></td>
      <td></td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>346</td>
      <td>362</td>
      <td>16</td>
      <td>16</td>
      <td>872</td>
      <td>872</td>
      <td>67</td>
      <td>67</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1218</td>
      <td>1234</td>
      <td>83</td>
      <td>83</td>
    </tr>
    <tr>
      <td></td>
      <td>sleuth</td>
      <td>sleuth</td>
      <td>Sleuth is a program for differential analysis of RNA-Seq data.</td>
      <td>sleuth</td>
      <td>sleuth</td>
      <td></td>
      <td>sleuth</td>
      <td>A statistical model and software application for RNA-seq differential expression analysis.</td>
      <td>Expression data visualisation, Differential gene expression analysis, Gene expression profiling, Statistical calculation</td>
      <td>RNA-seq, Gene expression, Statistics and probability</td>
      <td>Up-to-date</td>
      <td>https://github.com/pachterlab/sleuth</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>sleuth</td>
      <td>2023-05-31</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/sleuth</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/sleuth</td>
      <td>0.30.1</td>
      <td>r-sleuth</td>
      <td>0.30.1</td>
      <td>Expression data visualisation, Differential gene expression analysis, Gene expression profiling, Statistical calculation</td>
      <td>Gene expression, Statistics and probability</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>141</td>
      <td>141</td>
      <td>26</td>
      <td>26</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>141</td>
      <td>141</td>
      <td>26</td>
      <td>26</td>
    </tr>
    <tr>
      <td></td>
      <td>smallgenomeutilities</td>
      <td>smgu_frameshift_deletions_checks</td>
      <td>Set of utilities for manipulating small viral genome data.</td>
      <td>v-pipe</td>
      <td>v-pipe</td>
      <td></td>
      <td>V-pipe</td>
      <td>Bioinformatics pipeline for the analysis of next-generation sequencing data derived from intra-host viral populations.</td>
      <td>Read pre-processing, Sequence alignment, Genetic variation analysis</td>
      <td>Genomics, Population genetics, Workflows, Virology, Sequencing</td>
      <td>Up-to-date</td>
      <td>https://github.com/cbg-ethz/smallgenomeutilities</td>
      <td>Sequence Analysis</td>
      <td>smallgenomeutilities</td>
      <td>2023-05-30</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/smallgenomeutilities</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/smallgenomeutilities</td>
      <td>0.4.1</td>
      <td>smallgenomeutilities</td>
      <td>0.4.1</td>
      <td>Read pre-processing, Sequence alignment, Genetic variation analysis</td>
      <td>Genomics, Population genetics, Workflows, Virology, Sequencing</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>158</td>
      <td>158</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>158</td>
      <td>158</td>
      <td>8</td>
      <td>8</td>
    </tr>
    <tr>
      <td></td>
      <td>smudgeplot</td>
      <td>smudgeplot</td>
      <td>Inference of ploidy and heterozygosity structure using whole genome sequencing</td>
      <td>smudgeplots</td>
      <td>smudgeplots</td>
      <td></td>
      <td>Smudgeplots</td>
      <td>Reference-free profiling of polyploid genomes | Inference of ploidy and heterozygosity structure using whole genome sequencing data | Smudgeplots are computed from raw or even better from trimmed reads and show the haplotype structure using heterozygous kmer pairs. For example: | This tool extracts heterozygous kmer pairs from kmer dump files and performs gymnastics with them. We are able to disentangle genome structure by comparing the sum of kmer pair coverages (CovA + CovB) to their relative coverage (CovA / (CovA + CovB)). Such an approach also allows us to analyze obscure genomes with duplications, various ploidy levels, etc | GenomeScope 2.0 and Smudgeplots: Reference-free profiling of polyploid genomes Timothy Rhyker Ranallo-Benavidez, Kamil S. Jaron, Michael C. Schatz bioRxiv 747568; doi: https://doi.org/10.1101/747568</td>
      <td>Sequence trimming, Genotyping, k-mer counting</td>
      <td>Sequence assembly, Genetic variation, Mathematics</td>
      <td>To update</td>
      <td>https://github.com/KamilSJaron/smudgeplot</td>
      <td>Assembly</td>
      <td>smudgeplot</td>
      <td>2022-06-30</td>
      <td>galaxy-australia</td>
      <td>https://github.com/galaxyproject/tools-iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/smudgeplot</td>
      <td>0.2.5</td>
      <td>smudgeplot</td>
      <td>0.3.0</td>
      <td>Sequence trimming, Genotyping, k-mer counting</td>
      <td>Sequence assembly, Genetic variation, Mathematics</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>325</td>
      <td>325</td>
      <td>61</td>
      <td>61</td>
      <td>137</td>
      <td>137</td>
      <td>54</td>
      <td>54</td>
      <td>843</td>
      <td>843</td>
      <td>63</td>
      <td>63</td>
      <td>1305</td>
      <td>1305</td>
      <td>178</td>
      <td>178</td>
    </tr>
    <tr>
      <td></td>
      <td>snap</td>
      <td>snap, snap_training</td>
      <td>SNAP is a general purpose gene finding program suitable for both eukaryotic and prokaryotic genomes.</td>
      <td>snap</td>
      <td>snap</td>
      <td></td>
      <td>SNAP</td>
      <td>The Semi-HMM-based Nucleic Acid Parser is a gene prediction tool.</td>
      <td>Gene prediction</td>
      <td>DNA, DNA polymorphism, Genetics</td>
      <td>Up-to-date</td>
      <td>https://github.com/KorfLab/SNAP</td>
      <td>Sequence Analysis</td>
      <td>snap</td>
      <td>2017-10-12</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/snap</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/snap</td>
      <td>2013_11_29</td>
      <td>snap</td>
      <td>2013_11_29</td>
      <td>Gene prediction</td>
      <td>DNA polymorphism, Genetics</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1499</td>
      <td>1638</td>
      <td>331</td>
      <td>358</td>
      <td>779</td>
      <td>779</td>
      <td>244</td>
      <td>244</td>
      <td>467</td>
      <td>467</td>
      <td>83</td>
      <td>83</td>
      <td>2745</td>
      <td>2884</td>
      <td>658</td>
      <td>685</td>
    </tr>
    <tr>
      <td></td>
      <td>snapatac2</td>
      <td>snapatac2_clustering, snapatac2_peaks_and_motif, snapatac2_plotting, snapatac2_preprocessing</td>
      <td>SnapATAC2  A Python/Rust package for single-cell epigenomics analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://kzhang.org/SnapATAC2/</td>
      <td>Single Cell, Epigenetics</td>
      <td>snapatac2</td>
      <td>2024-05-16</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/snapatac2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/snapatac2</td>
      <td>2.6.4</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1817</td>
      <td>1817</td>
      <td>14</td>
      <td>14</td>
      <td>4</td>
      <td>4</td>
      <td>2</td>
      <td>2</td>
      <td>136</td>
      <td>136</td>
      <td>4</td>
      <td>4</td>
      <td>1957</td>
      <td>1957</td>
      <td>20</td>
      <td>20</td>
    </tr>
    <tr>
      <td></td>
      <td>sniffles</td>
      <td>sniffles</td>
      <td>Galaxy wrapper for sniffles</td>
      <td>sniffles</td>
      <td>sniffles</td>
      <td></td>
      <td>Sniffles</td>
      <td>An algorithm for structural variation detection from third generation sequencing alignment.</td>
      <td>Sequence analysis, Structural variation detection</td>
      <td>DNA structural variation, Sequencing</td>
      <td>Up-to-date</td>
      <td>https://github.com/fritzsedlazeck/Sniffles</td>
      <td>Sequence Analysis</td>
      <td>sniffles</td>
      <td>2020-08-29</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/sniffles</td>
      <td>2.4</td>
      <td>sniffles</td>
      <td>2.4</td>
      <td>Sequence analysis, Structural variation detection</td>
      <td>Sequencing</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1141</td>
      <td>1141</td>
      <td>134</td>
      <td>134</td>
      <td>606</td>
      <td>606</td>
      <td>156</td>
      <td>156</td>
      <td>2267</td>
      <td>2267</td>
      <td>80</td>
      <td>80</td>
      <td>4014</td>
      <td>4014</td>
      <td>370</td>
      <td>370</td>
    </tr>
    <tr>
      <td></td>
      <td>snipit</td>
      <td>snipit</td>
      <td>Summarise snps relative to a reference sequence</td>
      <td>snipit</td>
      <td>snipit</td>
      <td></td>
      <td>snipit</td>
      <td>Summarise snps relative to a reference sequence</td>
      <td>Base position variability plotting</td>
      <td>Virology</td>
      <td>Up-to-date</td>
      <td>https://github.com/aineniamh/snipit</td>
      <td>Variant Analysis, Sequence Analysis</td>
      <td>snipit</td>
      <td>2022-07-17</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/snipit</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/snipit</td>
      <td>1.2</td>
      <td>snipit</td>
      <td>1.2</td>
      <td>Base position variability plotting</td>
      <td>Virology</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>905</td>
      <td>905</td>
      <td>44</td>
      <td>44</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>905</td>
      <td>905</td>
      <td>44</td>
      <td>44</td>
    </tr>
    <tr>
      <td></td>
      <td>snippy</td>
      <td>snippy_core, snippy, snippy_clean_full_aln</td>
      <td>Contains the snippy tool for characterising microbial snps</td>
      <td>snippy</td>
      <td>snippy</td>
      <td></td>
      <td>snippy</td>
      <td>Rapid haploid variant calling and core SNP phylogeny generation.</td>
      <td>Phylogenetic tree visualisation, Phylogenetic tree generation, Variant calling</td>
      <td>Genomics, Model organisms, DNA polymorphism, Phylogenetics</td>
      <td>To update</td>
      <td>https://github.com/tseemann/snippy</td>
      <td>Sequence Analysis</td>
      <td>snippy</td>
      <td>2019-04-02</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/snippy</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/snippy</td>
      <td></td>
      <td>snippy</td>
      <td>4.6.0</td>
      <td>Phylogenetic tree visualisation, Variant calling</td>
      <td>Genomics, Model organisms, DNA polymorphism, Phylogenetics</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>2</td>
      <td>0</td>
      <td>121363</td>
      <td>124773</td>
      <td>3448</td>
      <td>3602</td>
      <td>92504</td>
      <td>92504</td>
      <td>3945</td>
      <td>3945</td>
      <td>48068</td>
      <td>52034</td>
      <td>2705</td>
      <td>3073</td>
      <td>261935</td>
      <td>269311</td>
      <td>10098</td>
      <td>10620</td>
    </tr>
    <tr>
      <td></td>
      <td>snp-dists</td>
      <td>snp_dists</td>
      <td>Compute pairwise SNP distance matrix from a FASTA sequence alignment</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/tseemann/snp-dists</td>
      <td>Variant Analysis</td>
      <td>snp_dists</td>
      <td>2019-10-18</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/snp-dists</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/snp-dists</td>
      <td>0.8.2</td>
      <td>snp-dists</td>
      <td>0.8.2</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>6555</td>
      <td>6555</td>
      <td>807</td>
      <td>807</td>
      <td>5336</td>
      <td>5336</td>
      <td>171</td>
      <td>171</td>
      <td>1143</td>
      <td>1143</td>
      <td>182</td>
      <td>182</td>
      <td>13034</td>
      <td>13034</td>
      <td>1160</td>
      <td>1160</td>
    </tr>
    <tr>
      <td></td>
      <td>snp-sites</td>
      <td>snp_sites</td>
      <td>Finds SNP sites from a multi-FASTA alignment file</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/sanger-pathogens/snp-sites</td>
      <td>Variant Analysis</td>
      <td>snp_sites</td>
      <td>2017-06-28</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/snp-sites</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/snp-sites</td>
      <td>2.5.1</td>
      <td>snp-sites</td>
      <td>2.5.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>6597</td>
      <td>6597</td>
      <td>663</td>
      <td>663</td>
      <td>1467</td>
      <td>1467</td>
      <td>256</td>
      <td>256</td>
      <td>1279</td>
      <td>1279</td>
      <td>114</td>
      <td>114</td>
      <td>9343</td>
      <td>9343</td>
      <td>1033</td>
      <td>1033</td>
    </tr>
    <tr>
      <td></td>
      <td>snpeff</td>
      <td>snpEff, snpEff_build_gb, snpEff_databases, snpEff_download, snpEff_get_chr_names</td>
      <td>SnpEff is a genetic variant annotation and effect prediction toolbox</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://snpeff.sourceforge.net/</td>
      <td>Genome-Wide Association Study, Variant Analysis</td>
      <td>snpeff</td>
      <td>2018-07-06</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/snpeff</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/snpeff</td>
      <td></td>
      <td>biopython</td>
      <td>1.70</td>
      <td></td>
      <td></td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>103826</td>
      <td>108296</td>
      <td>7181</td>
      <td>7908</td>
      <td>115727</td>
      <td>178110</td>
      <td>15871</td>
      <td>22769</td>
      <td>11817</td>
      <td>13155</td>
      <td>2148</td>
      <td>2321</td>
      <td>231370</td>
      <td>299561</td>
      <td>25200</td>
      <td>32998</td>
    </tr>
    <tr>
      <td></td>
      <td>snpeff-covid19</td>
      <td>snpeff_sars_cov_2</td>
      <td>SnpEff, the COVID-19 version, is a genetic variant annotation and effect prediction toolbox</td>
      <td>snpeff</td>
      <td>snpeff</td>
      <td></td>
      <td>snpEff</td>
      <td>Variant annotation and effect prediction tool. It annotates and predicts the effects of variants on genes and proteins (such as amino acid changes).</td>
      <td>SNP detection</td>
      <td>DNA polymorphism, Genetic variation, Nucleic acid sites, features and motifs</td>
      <td>To update</td>
      <td>http://snpeff.sourceforge.net/</td>
      <td>Genome-Wide Association Study, Variant Analysis</td>
      <td>snpeff_sars_cov_2</td>
      <td>2020-06-17</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/snpeff</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/snpeff-covid19</td>
      <td></td>
      <td></td>
      <td></td>
      <td>SNP detection</td>
      <td>DNA polymorphism, Nucleic acid sites, features and motifs</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>764895</td>
      <td>764895</td>
      <td>1276</td>
      <td>1276</td>
      <td>94572</td>
      <td>94572</td>
      <td>1636</td>
      <td>1636</td>
      <td>26990</td>
      <td>26990</td>
      <td>561</td>
      <td>561</td>
      <td>886457</td>
      <td>886457</td>
      <td>3473</td>
      <td>3473</td>
    </tr>
    <tr>
      <td></td>
      <td>snpfreqplot</td>
      <td>snpfreqplot</td>
      <td>A plotting app to visualise the SNPs across a region</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/BrownfieldPlantLab/SNPFreqPlot</td>
      <td>Variant Analysis</td>
      <td>snpfreqplot</td>
      <td>2020-12-02</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/snpfreqplot/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/snpfreqplot</td>
      <td>1.0</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>3683</td>
      <td>3683</td>
      <td>338</td>
      <td>338</td>
      <td>1401</td>
      <td>1401</td>
      <td>433</td>
      <td>433</td>
      <td>1029</td>
      <td>1029</td>
      <td>95</td>
      <td>95</td>
      <td>6113</td>
      <td>6113</td>
      <td>866</td>
      <td>866</td>
    </tr>
    <tr>
      <td></td>
      <td>snpsift</td>
      <td>snpSift_annotate, snpSift_caseControl, snpSift_extractFields, snpSift_filter, snpSift_int, snpSift_rmInfo, snpsift_vartype, snpSift_vcfCheck</td>
      <td>snpEff SnpSift tools from Pablo Cingolani</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://snpeff.sourceforge.net/SnpSift.html</td>
      <td>Variant Analysis</td>
      <td>snpsift</td>
      <td>2015-04-01</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/snpsift/snpsift</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/snpsift/snpsift</td>
      <td></td>
      <td>snpsift</td>
      <td>5.2</td>
      <td></td>
      <td></td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>5142904</td>
      <td>5146618</td>
      <td>3602</td>
      <td>4283</td>
      <td>382391</td>
      <td>421190</td>
      <td>7013</td>
      <td>9678</td>
      <td>104030</td>
      <td>105024</td>
      <td>960</td>
      <td>1184</td>
      <td>5629325</td>
      <td>5672832</td>
      <td>11575</td>
      <td>15145</td>
    </tr>
    <tr>
      <td></td>
      <td>snpsift_dbnsfp</td>
      <td>snpSift_dbnsfp</td>
      <td>snpEff SnpSift dbnsfp tool from Pablo Cingolani</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://snpeff.sourceforge.net/SnpSift.html#dbNSFP</td>
      <td>Variant Analysis</td>
      <td>snpsift_dbnsfp</td>
      <td>2014-11-11</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/snpsift/snpsift_dbnsfp</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/snpsift/snpsift_dbnsfp</td>
      <td></td>
      <td>snpsift</td>
      <td>5.2</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>52</td>
      <td>55</td>
      <td>23</td>
      <td>24</td>
      <td>421</td>
      <td>421</td>
      <td>75</td>
      <td>75</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>473</td>
      <td>476</td>
      <td>98</td>
      <td>99</td>
    </tr>
    <tr>
      <td></td>
      <td>snpsift_genesets</td>
      <td>snpSift_geneSets</td>
      <td>Annotate SnpEff vcf with genesets such as Gene Ontology (GO), KEGG, Reactome</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://snpeff.sourceforge.net/SnpSift.html#geneSets</td>
      <td>Variant Analysis</td>
      <td>snpsift_genesets</td>
      <td>2014-11-11</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/snpsift/snpsift_genesets/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/snpsift/snpsift_genesets</td>
      <td></td>
      <td>snpsift</td>
      <td>5.2</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>509</td>
      <td>554</td>
      <td>107</td>
      <td>128</td>
      <td>474</td>
      <td>474</td>
      <td>190</td>
      <td>190</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>983</td>
      <td>1028</td>
      <td>297</td>
      <td>318</td>
    </tr>
    <tr>
      <td></td>
      <td>socru</td>
      <td>socru</td>
      <td>Order and orientation of complete bacterial genomes</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/quadram-institute-bioscience/socru</td>
      <td>Sequence Analysis</td>
      <td>socru</td>
      <td>2019-09-03</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/socru</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/socru</td>
      <td>2.1.7</td>
      <td>socru</td>
      <td>2.2.4</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>659</td>
      <td>659</td>
      <td>42</td>
      <td>42</td>
      <td>296</td>
      <td>296</td>
      <td>92</td>
      <td>92</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>955</td>
      <td>955</td>
      <td>134</td>
      <td>134</td>
    </tr>
    <tr>
      <td></td>
      <td>sonneityping</td>
      <td>sonneityping</td>
      <td>Scripts for parsing Mykrobe predict results for Shigella sonnei.</td>
      <td>sonneityping</td>
      <td>sonneityping</td>
      <td></td>
      <td>sonneityping</td>
      <td>Scripts for parsing Mykrobe predict results for Shigella sonnei.</td>
      <td>Antimicrobial resistance prediction, Variant calling, Genotyping</td>
      <td>Whole genome sequencing, Genotype and phenotype, Genetic variation, Metagenomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/katholt/sonneityping</td>
      <td>Sequence Analysis</td>
      <td>sonneityping</td>
      <td>2021-09-16</td>
      <td>iuc</td>
      <td>https://github.com/katholt/sonneityping</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/sonneityping</td>
      <td>20210201</td>
      <td>sonneityping</td>
      <td>20210201</td>
      <td>Antimicrobial resistance prediction, Variant calling, Genotyping</td>
      <td>Whole genome sequencing, Genotype and phenotype, Genetic variation, Metagenomics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
    </tr>
    <tr>
      <td></td>
      <td>spades</td>
      <td>spades_biosyntheticspades, spades_coronaspades, spades_metaplasmidspades, metaspades, spades_metaviralspades, spades_plasmidspades, rnaspades, spades_rnaviralspades, spades</td>
      <td>SPAdes is an assembly toolkit containing various assembly pipelines. It implements the following 4 stages: assembly graph construction, k-bimer adjustment, construction of paired assembly graph and contig construction.</td>
      <td>spades</td>
      <td>plasmidspades, metaplasmidspades, rnaspades, metaspades, spades, coronaspades, metaviralspades, biosyntheticspades, rnaviralspades</td>
      <td></td>
      <td>SPAdes</td>
      <td>St. Petersburg genome assembler  is intended for both standard isolates and single-cell MDA bacteria assemblies. SPAdes 3.9 works with Illumina or IonTorrent reads and is capable of providing hybrid assemblies using PacBio, Oxford Nanopore and Sanger reads. Additional contigs can be provided and can be used as long reads.</td>
      <td>Genome assembly</td>
      <td>Sequence assembly</td>
      <td>To update</td>
      <td>https://github.com/ablab/spades</td>
      <td>Assembly, RNA, Metagenomics</td>
      <td>spades</td>
      <td>2017-06-30</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/spades</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/spades</td>
      <td>3.15.5</td>
      <td>spades</td>
      <td>4.0.0</td>
      <td>Genome assembly</td>
      <td>Sequence assembly</td>
      <td>9</td>
      <td>9</td>
      <td>9</td>
      <td>9</td>
      <td>9</td>
      <td>9</td>
      <td>9</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>3</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>9</td>
      <td>3</td>
      <td>0</td>
      <td>82716</td>
      <td>87113</td>
      <td>8209</td>
      <td>8526</td>
      <td>120471</td>
      <td>120475</td>
      <td>14787</td>
      <td>14790</td>
      <td>54067</td>
      <td>61541</td>
      <td>5817</td>
      <td>6653</td>
      <td>257254</td>
      <td>269129</td>
      <td>28813</td>
      <td>29969</td>
    </tr>
    <tr>
      <td></td>
      <td>spaln</td>
      <td>list_spaln_tables, spaln</td>
      <td>Spaln (space-efficient spliced alignment) maps and aligns a set of cDNA or protein sequences onto a whole genomic sequence.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.genome.ist.i.kyoto-u.ac.jp/~aln_user/spaln/</td>
      <td>Sequence Analysis, Genome annotation</td>
      <td>spaln</td>
      <td>2019-01-11</td>
      <td>iuc</td>
      <td>https://github.com/ogotoh/spaln</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/spaln</td>
      <td>2.4.9</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>450</td>
      <td>491</td>
      <td>91</td>
      <td>100</td>
      <td>4270</td>
      <td>4270</td>
      <td>288</td>
      <td>288</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4720</td>
      <td>4761</td>
      <td>379</td>
      <td>388</td>
    </tr>
    <tr>
      <td></td>
      <td>spapros</td>
      <td>spapros_evaluation, spapros_selection</td>
      <td>Select and evaluate probe sets for targeted spatial transcriptomics.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/theislab/spapros</td>
      <td>Single Cell, Transcriptomics, Sequence Analysis</td>
      <td>spapros</td>
      <td>2024-08-25</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/spapros/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/spapros</td>
      <td>0.1.5</td>
      <td>spapros</td>
      <td>0.1.5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>spotyping</td>
      <td>spotyping</td>
      <td>SpoTyping allows fast and accurate in silico Mycobacterium spoligotyping from sequence reads</td>
      <td>spotyping</td>
      <td>spotyping</td>
      <td></td>
      <td>SpoTyping</td>
      <td>Fast and accurate in silico Mycobacterium spoligotyping from sequence reads.</td>
      <td>Variant pattern analysis</td>
      <td>Microbiology, Sequencing, Sequence composition, complexity and repeats, Genetic variation</td>
      <td>Up-to-date</td>
      <td>https://github.com/xiaeryu/SpoTyping-v2.0</td>
      <td>Sequence Analysis</td>
      <td>spotyping</td>
      <td>2018-05-07</td>
      <td>iuc</td>
      <td>https://github.com/xiaeryu/SpoTyping-v2.0/tree/master/SpoTyping-v2.0-commandLine</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/spotyping</td>
      <td>2.1</td>
      <td>spotyping</td>
      <td>2.1</td>
      <td>Variant pattern analysis</td>
      <td>Microbiology, Sequencing, Sequence composition, complexity and repeats, Genetic variation</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1436</td>
      <td>1436</td>
      <td>23</td>
      <td>23</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1436</td>
      <td>1436</td>
      <td>23</td>
      <td>23</td>
    </tr>
    <tr>
      <td></td>
      <td>spyboat</td>
      <td>spyboat</td>
      <td>Wavelet analysis for 3d-image stacks</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://github.com/tensionhead/spyboat</td>
      <td>Imaging, Graphics</td>
      <td>spyboat</td>
      <td>2020-11-28</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/packages/spyboat</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/spyboat</td>
      <td>0.1.2</td>
      <td>spyboat</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>142</td>
      <td>142</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>142</td>
      <td>142</td>
      <td>5</td>
      <td>5</td>
    </tr>
    <tr>
      <td></td>
      <td>sra-tools</td>
      <td>fasterq_dump, fastq_dump, sam_dump</td>
      <td>NCBI Sequence Read Archive toolkit utilities</td>
      <td>sra-tools</td>
      <td>sra-tools</td>
      <td></td>
      <td>SRA Software Toolkit</td>
      <td>The SRA Toolkit and SDK from NCBI is a collection of tools and libraries for using data in the INSDC Sequence Read Archives.</td>
      <td>Data handling</td>
      <td>DNA, Genomics, Sequencing</td>
      <td>Up-to-date</td>
      <td>https://github.com/ncbi/sra-tools</td>
      <td>Data Source, Fastq Manipulation</td>
      <td>sra_tools</td>
      <td>2020-06-23</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/sra-tools</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/sra-tools</td>
      <td>3.1.1</td>
      <td>sra-tools</td>
      <td>3.1.1</td>
      <td>Data handling</td>
      <td>DNA, Genomics, Sequencing</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>364871</td>
      <td>376580</td>
      <td>16710</td>
      <td>17342</td>
      <td>583675</td>
      <td>678448</td>
      <td>53271</td>
      <td>59638</td>
      <td>48131</td>
      <td>50455</td>
      <td>5910</td>
      <td>6181</td>
      <td>996677</td>
      <td>1105483</td>
      <td>75891</td>
      <td>83161</td>
    </tr>
    <tr>
      <td></td>
      <td>srst2</td>
      <td>srst2</td>
      <td>SRST2 Short Read Sequence Typing for Bacterial Pathogens</td>
      <td>srst2</td>
      <td>srst2</td>
      <td></td>
      <td>srst2</td>
      <td>Short Read Sequence Typing for Bacterial Pathogens</td>
      <td>Multilocus sequence typing</td>
      <td>Whole genome sequencing, Public health and epidemiology</td>
      <td>To update</td>
      <td>http://katholt.github.io/srst2/</td>
      <td>Metagenomics</td>
      <td>srst2</td>
      <td>2022-08-22</td>
      <td>iuc</td>
      <td>https://github.com/katholt/srst2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/srst2</td>
      <td>0.2.0</td>
      <td>samtools</td>
      <td>1.21</td>
      <td>Multilocus sequence typing</td>
      <td>Whole genome sequencing, Public health and epidemiology</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>297</td>
      <td>297</td>
      <td>40</td>
      <td>40</td>
      <td>71</td>
      <td>71</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>368</td>
      <td>368</td>
      <td>41</td>
      <td>41</td>
    </tr>
    <tr>
      <td></td>
      <td>stacks</td>
      <td>stacks_assembleperead, stacks_clonefilter, stacks_cstacks, stacks_denovomap, stacks_genotypes, stacks_populations, stacks_procrad, stacks_pstacks, stacks_refmap, stacks_rxstacks, stacks_sstacks, stacks_stats, stacks_ustacks</td>
      <td>Stacks is a software pipeline for building loci from short-read sequences, such as RAD-seq</td>
      <td>stacks</td>
      <td>stacks</td>
      <td></td>
      <td>Stacks</td>
      <td>Developed to work with restriction enzyme based sequence data, such as RADseq, for building genetic maps and conducting population genomics and phylogeography analysis.</td>
      <td>Data handling</td>
      <td>Mapping, Population genetics</td>
      <td>To update</td>
      <td>http://catchenlab.life.illinois.edu/stacks/</td>
      <td>Sequence Analysis</td>
      <td>stacks</td>
      <td>2017-04-13</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/stacks</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/stacks</td>
      <td></td>
      <td>stacks</td>
      <td>2.65</td>
      <td>Data handling</td>
      <td>Mapping, Population genetics</td>
      <td>0</td>
      <td>13</td>
      <td>13</td>
      <td>12</td>
      <td>0</td>
      <td>13</td>
      <td>13</td>
      <td>12</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>11</td>
      <td>13</td>
      <td>13</td>
      <td>0</td>
      <td>6440</td>
      <td>11998</td>
      <td>465</td>
      <td>593</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4070</td>
      <td>4070</td>
      <td>76</td>
      <td>76</td>
      <td>10510</td>
      <td>16068</td>
      <td>541</td>
      <td>669</td>
    </tr>
    <tr>
      <td></td>
      <td>stacks2</td>
      <td>stacks2_clonefilter, stacks2_cstacks, stacks2_denovomap, stacks2_gstacks, stacks2_kmerfilter, stacks2_populations, stacks2_procrad, stacks2_refmap, stacks2_shortreads, stacks2_sstacks, stacks2_tsv2bam, stacks2_ustacks</td>
      <td>Stacks is a software pipeline for building loci from short-read sequences, such as RAD-seq</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://catchenlab.life.illinois.edu/stacks/</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>2019-09-30</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/stacks2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/stacks2</td>
      <td>2.55</td>
      <td>stacks</td>
      <td>2.65</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>12</td>
      <td>12</td>
      <td>12</td>
      <td>0</td>
      <td>12</td>
      <td>12</td>
      <td>12</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>12</td>
      <td>12</td>
      <td>0</td>
      <td>8437</td>
      <td>8592</td>
      <td>626</td>
      <td>646</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>18016</td>
      <td>18016</td>
      <td>270</td>
      <td>270</td>
      <td>26453</td>
      <td>26608</td>
      <td>896</td>
      <td>916</td>
    </tr>
    <tr>
      <td></td>
      <td>star_fusion</td>
      <td>star_fusion</td>
      <td>STAR Fusion detects fusion genes in RNA-Seq data after running RNA-STAR</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/STAR-Fusion/STAR-Fusion</td>
      <td>Sequence Analysis, Transcriptomics</td>
      <td>star_fusion</td>
      <td>2015-10-15</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/star_fusion</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/star_fusion</td>
      <td>0.5.4-3+galaxy1</td>
      <td>star-fusion</td>
      <td>1.13.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1312</td>
      <td>1383</td>
      <td>82</td>
      <td>95</td>
      <td>2198</td>
      <td>3076</td>
      <td>234</td>
      <td>303</td>
      <td>173</td>
      <td>173</td>
      <td>3</td>
      <td>3</td>
      <td>3683</td>
      <td>4632</td>
      <td>319</td>
      <td>401</td>
    </tr>
    <tr>
      <td></td>
      <td>straindesign</td>
      <td>straindesign_analyzing_model, straindesign_reduce_model, straindesign_simulate_deletion</td>
      <td>Toolbox to optimize biological model</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/brsynth/straindesign</td>
      <td>Systems Biology, Synthetic Biology</td>
      <td>straindesign</td>
      <td>2022-10-02</td>
      <td>iuc</td>
      <td>https://github.com/brsynth/straindesign</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/straindesign</td>
      <td>3.2.2</td>
      <td>straindesign</td>
      <td>3.2.3</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>strelka</td>
      <td>strelka_germline, strelka_somatic</td>
      <td>Strelka2 is a fast and accurate small variant caller optimizedfor analysis of germline variation in small cohorts and somaticvariation in tumor/normal sample pairs. The germline caller employs anefficient tiered haplotype model to improve accuracy and provide read-backedphasing, adaptively selecting between assembly and a faster alignment-basedhaplotyping approach at each variant locus. The germline caller also analyzesinput sequencing data using a mixture-model indel error estimation method toimprove robustness to indel noise. The somatic calling model improves on theoriginal Strelka method for liquid and late-stage tumor analysis by accountingfor possible tumor cell contamination in the normal sample. A final empiricalvariant re-scoring step using random forest models trained on various callquality features has been added to both callers to further improve precision.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/Illumina/strelka/</td>
      <td>Variant Analysis</td>
      <td>strelka</td>
      <td>2021-01-27</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/strelka</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/strelka</td>
      <td>2.9.10</td>
      <td>strelka</td>
      <td>2.9.10</td>
      <td></td>
      <td></td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>669</td>
      <td>669</td>
      <td>112</td>
      <td>112</td>
      <td>1261</td>
      <td>1261</td>
      <td>417</td>
      <td>417</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1930</td>
      <td>1930</td>
      <td>529</td>
      <td>529</td>
    </tr>
    <tr>
      <td></td>
      <td>stringtie</td>
      <td>stringtie, stringtie_merge</td>
      <td>StringTie is a fast and highly efficient assembler of RNA-Seq alignments into potential transcripts.</td>
      <td>stringtie</td>
      <td>stringtie</td>
      <td></td>
      <td>StringTie</td>
      <td>Fast and highly efficient assembler of RNA-Seq alignments into potential transcripts. It uses a novel network flow algorithm as well as an optional de novo assembly step to assemble and quantitate full-length transcripts representing multiple splice variants for each gene locus.</td>
      <td>Transcriptome assembly, RNA-Seq analysis</td>
      <td>Transcriptomics, RNA-seq</td>
      <td>Up-to-date</td>
      <td>http://ccb.jhu.edu/software/stringtie/</td>
      <td>Transcriptomics</td>
      <td>stringtie</td>
      <td>2019-09-05</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/stringtie</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/stringtie</td>
      <td>2.2.3</td>
      <td>stringtie</td>
      <td>2.2.3</td>
      <td>Transcriptome assembly, RNA-Seq analysis</td>
      <td>Transcriptomics</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>178677</td>
      <td>187728</td>
      <td>4972</td>
      <td>5347</td>
      <td>320780</td>
      <td>407379</td>
      <td>10825</td>
      <td>15171</td>
      <td>26375</td>
      <td>28921</td>
      <td>1032</td>
      <td>1167</td>
      <td>525832</td>
      <td>624028</td>
      <td>16829</td>
      <td>21685</td>
    </tr>
    <tr>
      <td></td>
      <td>structure</td>
      <td>structure</td>
      <td>for using multi-locus genotype data to investigate population structure.</td>
      <td>structure</td>
      <td>structure</td>
      <td></td>
      <td>Structure</td>
      <td>The program structureis a free software package for using multi-locus genotype data to investigate population structure. Its uses include inferring the presence of distinct populations, assigning individuals to populations, studying hybrid zones, identifying migrants and admixed individuals, and estimating population allele frequencies in situations where many individuals are migrants or admixed.</td>
      <td>Genetic variation analysis</td>
      <td>Population genetics</td>
      <td>Up-to-date</td>
      <td>https://web.stanford.edu/group/pritchardlab/structure.html</td>
      <td>Phylogenetics, Variant Analysis</td>
      <td>structure</td>
      <td>2017-09-22</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/structure</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/structure</td>
      <td>2.3.4</td>
      <td>structure</td>
      <td>2.3.4</td>
      <td>Genetic variation analysis</td>
      <td>Population genetics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2858</td>
      <td>3001</td>
      <td>118</td>
      <td>123</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2858</td>
      <td>3001</td>
      <td>118</td>
      <td>123</td>
    </tr>
    <tr>
      <td></td>
      <td>structureharvester</td>
      <td>structureharvester</td>
      <td>for parsing STRUCTURE outputs and for performing the Evanno method</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://alumni.soe.ucsc.edu/~dearl/software/structureHarvester/</td>
      <td>Phylogenetics, Variant Analysis</td>
      <td>structureharvester</td>
      <td>2017-12-06</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/structureharvester</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/structureharvester</td>
      <td>0.6.94</td>
      <td>structureharvester</td>
      <td>0.6.94</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>swift</td>
      <td>swiftlink</td>
      <td>Parallel MCMC Linkage Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/ajm/swiftlink</td>
      <td>Variant Analysis</td>
      <td>swiftlink</td>
      <td>2017-11-21</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/swiftlink/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/swift</td>
      <td>1.0</td>
      <td>swiftlink</td>
      <td>1.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>syndiva</td>
      <td>syndiva</td>
      <td>SynDivA was developed to analyze the diversity of synthetic libraries of a Fibronectin domain.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>syndiva</td>
      <td>2022-06-23</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/SynDivA</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/syndiva</td>
      <td>1.0</td>
      <td>clustalo</td>
      <td>1.2.4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>30</td>
      <td>30</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>30</td>
      <td>30</td>
      <td>2</td>
      <td>2</td>
    </tr>
    <tr>
      <td></td>
      <td>table_compute</td>
      <td>table_compute</td>
      <td>Perform general-purpose table operations</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/table_compute</td>
      <td>Text Manipulation</td>
      <td>table_compute</td>
      <td>2019-08-28</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/table_compute</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/table_compute</td>
      <td>1.2.4</td>
      <td>pandas</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>216072</td>
      <td>216115</td>
      <td>1811</td>
      <td>1814</td>
      <td>12326</td>
      <td>12326</td>
      <td>1558</td>
      <td>1558</td>
      <td>16136</td>
      <td>16136</td>
      <td>256</td>
      <td>256</td>
      <td>244534</td>
      <td>244577</td>
      <td>3625</td>
      <td>3628</td>
    </tr>
    <tr>
      <td></td>
      <td>tabular_to_fastq</td>
      <td>tabular_to_fastq</td>
      <td>Tabular to FASTQ converter</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>tabular_to_fastq</td>
      <td>2019-06-21</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/tabular_to_fastq</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/galaxy_sequence_utils/tabular_to_fastq</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>1.2.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>7011</td>
      <td>8135</td>
      <td>449</td>
      <td>482</td>
      <td>9473</td>
      <td>28494</td>
      <td>626</td>
      <td>2183</td>
      <td>850</td>
      <td>1090</td>
      <td>80</td>
      <td>104</td>
      <td>17334</td>
      <td>37719</td>
      <td>1155</td>
      <td>2769</td>
    </tr>
    <tr>
      <td></td>
      <td>tag_pileup_frequency</td>
      <td>tag_pileup_frequency</td>
      <td>Contains a tool that generates a frequency pileup of the 5' ends of aligned reads in a BAM filerelative to reference points in a BED file.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/seqcode/cegr-tools/tree/master/src/org/seqcode/cegrtools/tagpileup</td>
      <td>Statistics, SAM, Genomic Interval Operations</td>
      <td>tag_pileup_frequency</td>
      <td>2016-05-05</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/tag_pileup_frequency</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/tag_pileup_frequency</td>
      <td>1.0.2</td>
      <td>openjdk</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>158</td>
      <td>164</td>
      <td>15</td>
      <td>16</td>
      <td>722</td>
      <td>865</td>
      <td>40</td>
      <td>73</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>880</td>
      <td>1029</td>
      <td>55</td>
      <td>89</td>
    </tr>
    <tr>
      <td></td>
      <td>tasmanian_mismatch</td>
      <td>tasmanian_mismatch</td>
      <td>Analysis of positional mismatches</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/nebiolabs/tasmanian-mismatch</td>
      <td>Sequence Analysis</td>
      <td>tasmanian_mismatch</td>
      <td>2020-04-30</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/tasmanian_mismatch</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/tasmanian_mismatch</td>
      <td>1.0.7</td>
      <td>tasmanian-mismatch</td>
      <td>1.0.7</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>61</td>
      <td>61</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>65</td>
      <td>65</td>
      <td>11</td>
      <td>11</td>
    </tr>
    <tr>
      <td></td>
      <td>taxonkit</td>
      <td>name2taxid, profile2cami</td>
      <td>TaxonKit - A Practical and Efficient NCBI Taxonomy Toolkit</td>
      <td>taxonkit</td>
      <td>taxonkit</td>
      <td></td>
      <td>TaxonKit</td>
      <td>TaxonKit is a practical and efficient NCBI taxonomy toolkit.</td>
      <td>Formatting, Data retrieval</td>
      <td>Taxonomy, Biotechnology, Ecology</td>
      <td>Up-to-date</td>
      <td>https://bioinf.shenwei.me/taxonkit/</td>
      <td>Metagenomics</td>
      <td>taxonkit</td>
      <td>2024-07-26</td>
      <td>iuc</td>
      <td>https://github.com/shenwei356/taxonkit</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/taxonkit</td>
      <td>0.17.0</td>
      <td>taxonkit</td>
      <td>0.17.0</td>
      <td>Formatting, Data retrieval</td>
      <td>Taxonomy, Biotechnology, Ecology</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>taxonomy_filter_refseq</td>
      <td>taxonomy_filter_refseq</td>
      <td>Filter RefSeq by taxonomy</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/pvanheus/ncbitaxonomy</td>
      <td>Sequence Analysis, Genome annotation</td>
      <td>taxonomy_filter_refseq</td>
      <td>2019-01-13</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/taxonomy_filter_refseq</td>
      <td>0.3.0</td>
      <td>rust-ncbitaxonomy</td>
      <td>1.0.7</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>taxonomy_krona_chart</td>
      <td>taxonomy_krona_chart</td>
      <td>Krona pie chart from taxonomic profile</td>
      <td>krona</td>
      <td>krona</td>
      <td></td>
      <td>Krona</td>
      <td>Krona creates interactive HTML5 charts of hierarchical data (such as taxonomic abundance in a metagenome).</td>
      <td>Visualisation</td>
      <td>Metagenomics</td>
      <td>To update</td>
      <td>http://sourceforge.net/projects/krona/</td>
      <td>Assembly</td>
      <td>taxonomy_krona_chart</td>
      <td>2015-08-12</td>
      <td>crs4</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/taxonomy_krona_chart</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/taxonomy_krona_chart</td>
      <td>2.7.1+galaxy0</td>
      <td>krona</td>
      <td>2.8.1</td>
      <td>Visualisation</td>
      <td>Metagenomics</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>36069</td>
      <td>36922</td>
      <td>5257</td>
      <td>5431</td>
      <td>25255</td>
      <td>30822</td>
      <td>5066</td>
      <td>5826</td>
      <td>7949</td>
      <td>7949</td>
      <td>1584</td>
      <td>1584</td>
      <td>69273</td>
      <td>75693</td>
      <td>11907</td>
      <td>12841</td>
    </tr>
    <tr>
      <td></td>
      <td>taxpasta</td>
      <td>taxpasta</td>
      <td>standardise taxonomic profiles</td>
      <td>taxpasta</td>
      <td>taxpasta</td>
      <td></td>
      <td>taxpasta</td>
      <td>TAXonomic Profile Aggregation and STAndardisationThe main purpose of taxpasta is to standardise taxonomic profiles created by a range of bioinformatics tools. We call those tools taxonomic profilers. They each come with their own particular tabular output format. Across the profilers, relative abundances can be reported in read counts, fractions, or percentages, as well as any number of additional columns with extra information. We therefore decided to take the lessons learnt to heart and provide our own solution to deal with this pasticcio. With taxpasta you can ingest all of those formats and, at a minimum, output taxonomy identifiers and their integer counts. Taxpasta can not only standardise profiles but also merge them across samples for the same profiler into a single table.</td>
      <td>Standardisation and normalisation, Aggregation, Formatting, Conversion</td>
      <td>Taxonomy, Metagenomics</td>
      <td>Up-to-date</td>
      <td>https://taxpasta.readthedocs.io/en/latest/</td>
      <td>Sequence Analysis</td>
      <td>taxpasta</td>
      <td>2023-08-30</td>
      <td>iuc</td>
      <td>https://github.com/taxprofiler/taxpasta</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/taxpasta</td>
      <td>0.7.0</td>
      <td>taxpasta</td>
      <td>0.7.0</td>
      <td>Standardisation and normalisation, Aggregation, Formatting, Conversion</td>
      <td>Taxonomy, Metagenomics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>34</td>
      <td>34</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>34</td>
      <td>34</td>
      <td>5</td>
      <td>5</td>
    </tr>
    <tr>
      <td></td>
      <td>tb-profiler</td>
      <td>tb_profiler_profile</td>
      <td>Processes M. tuberculosis sequence data to infer strain type and identify known drug resistance markers.</td>
      <td>tb-profiler</td>
      <td>tb-profiler</td>
      <td></td>
      <td>tb-profiler</td>
      <td>A tool for drug resistance prediction from _M. tuberculosis_ genomic data (sequencing reads, alignments or variants).</td>
      <td>Antimicrobial resistance prediction</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/jodyphelan/TBProfiler</td>
      <td>Sequence Analysis</td>
      <td>tbprofiler</td>
      <td>2019-08-16</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/tb-profiler</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/tb-profiler</td>
      <td>6.2.1</td>
      <td>tb-profiler</td>
      <td>6.3.0</td>
      <td>Antimicrobial resistance prediction</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>6372</td>
      <td>6395</td>
      <td>372</td>
      <td>379</td>
      <td>7339</td>
      <td>7339</td>
      <td>378</td>
      <td>378</td>
      <td>6966</td>
      <td>6966</td>
      <td>154</td>
      <td>154</td>
      <td>20677</td>
      <td>20700</td>
      <td>904</td>
      <td>911</td>
    </tr>
    <tr>
      <td></td>
      <td>tb_variant_filter</td>
      <td>tb_variant_filter</td>
      <td>M. tuberculosis H37Rv VCF filter</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/COMBAT-TB/tb_variant_filter</td>
      <td>Variant Analysis</td>
      <td>tb_variant_filter</td>
      <td>2019-10-03</td>
      <td>iuc</td>
      <td>https://github.com/COMBAT-TB/tb_variant_filter</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/tb_variant_filter</td>
      <td>0.4.0</td>
      <td>tb_variant_filter</td>
      <td>0.4.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>22977</td>
      <td>22977</td>
      <td>461</td>
      <td>461</td>
      <td>3888</td>
      <td>3888</td>
      <td>274</td>
      <td>274</td>
      <td>3611</td>
      <td>3611</td>
      <td>80</td>
      <td>80</td>
      <td>30476</td>
      <td>30476</td>
      <td>815</td>
      <td>815</td>
    </tr>
    <tr>
      <td></td>
      <td>tbl2gff3</td>
      <td>tbl2gff3</td>
      <td>Table to GFF3</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/tbl2gff3</td>
      <td>Convert Formats, Sequence Analysis</td>
      <td>tbl2gff3</td>
      <td>2020-07-07</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/tbl2gff3</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/tbl2gff3</td>
      <td>1.2</td>
      <td>bcbiogff</td>
      <td>0.6.6</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1941</td>
      <td>1941</td>
      <td>466</td>
      <td>466</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>463</td>
      <td>463</td>
      <td>112</td>
      <td>112</td>
      <td>2404</td>
      <td>2404</td>
      <td>578</td>
      <td>578</td>
    </tr>
    <tr>
      <td></td>
      <td>tbvcfreport</td>
      <td>tbvcfreport</td>
      <td>Generate HTML report from SnpEff M.tuberculosis VCF(s)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/COMBAT-TB/tbvcfreport</td>
      <td>Variant Analysis</td>
      <td>tbvcfreport</td>
      <td>2019-08-29</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/tbvcfreport</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/tbvcfreport</td>
      <td>1.0.1</td>
      <td>tbvcfreport</td>
      <td>1.0.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2955</td>
      <td>2955</td>
      <td>335</td>
      <td>335</td>
      <td>6090</td>
      <td>6090</td>
      <td>348</td>
      <td>348</td>
      <td>2347</td>
      <td>2347</td>
      <td>72</td>
      <td>72</td>
      <td>11392</td>
      <td>11392</td>
      <td>755</td>
      <td>755</td>
    </tr>
    <tr>
      <td></td>
      <td>te_finder</td>
      <td>te_finder</td>
      <td>Transposable element insertions finder</td>
      <td>tefinder</td>
      <td>tefinder</td>
      <td></td>
      <td>TEfinder</td>
      <td>A Bioinformatics Pipeline for Detecting New Transposable Element Insertion Events in Next-Generation Sequencing Data.A bioinformatics tool for detecting novel transposable element insertions.TEfinder uses discordant reads to detect novel transposable element insertion events in paired-end sample sequencing data.</td>
      <td>Genome indexing, Variant calling, PCR primer design</td>
      <td>Sequencing, Mobile genetic elements, Workflows, Evolutionary biology, Genetic variation</td>
      <td>To update</td>
      <td>https://github.com/VistaSohrab/TEfinder</td>
      <td>Sequence Analysis</td>
      <td>te_finder</td>
      <td>2022-08-08</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/te_finder/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/te_finder</td>
      <td>1.0.1</td>
      <td>samtools</td>
      <td>1.21</td>
      <td>Genome indexing, Variant calling, PCR primer design</td>
      <td>Sequencing, Mobile genetic elements, Workflows, Evolutionary biology, Genetic variation</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>203</td>
      <td>203</td>
      <td>20</td>
      <td>20</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>203</td>
      <td>203</td>
      <td>20</td>
      <td>20</td>
    </tr>
    <tr>
      <td></td>
      <td>telescope</td>
      <td>telescope_assign</td>
      <td>Single locus resolution of Transposable ELEment expression.</td>
      <td>Telescope-expression</td>
      <td>Telescope-expression</td>
      <td></td>
      <td>Telescope</td>
      <td>Telescope is a tool for the characterization of the retrotranscriptome by accurate estimation of transposable element expression and the quantification of transposable element expression using RNA-seq.It can be used for Statistical Performance of TE Quantification Methods.All scripts needed to examine the sensitivity and biases of computational approaches for quantifying TE expression: 1) unique counts, 2) best counts, 3) RepEnrich, 4) TEtranscripts, 5) RSEM, 6) SalmonTE, and 7) Telescope.</td>
      <td>Essential dynamics, Sequence trimming, RNA-Seq quantification, Expression analysis, Read mapping</td>
      <td>RNA-Seq, Transcriptomics, Mapping, Gene transcripts, Sequence assembly</td>
      <td>Up-to-date</td>
      <td>https://github.com/mlbendall/telescope/</td>
      <td>Genome annotation</td>
      <td>telescope_assign</td>
      <td>2019-09-03</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/telescope</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/telescope</td>
      <td>1.0.3</td>
      <td>telescope</td>
      <td>1.0.3</td>
      <td>Essential dynamics, Sequence trimming, RNA-Seq quantification, Expression analysis, Read mapping</td>
      <td>RNA-Seq, Transcriptomics, Mapping, Gene transcripts, Sequence assembly</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>tetoolkit</td>
      <td>tetoolkit_tetranscripts</td>
      <td>The TEToolkit suite improves the bioinformatic analysis of repetitive sequences, particularly transposable elements, in order to elucidate novel (and previously ignored) biological insights of their functions in development and diseases.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hammelllab.labsites.cshl.edu/software/</td>
      <td>Sequence Analysis</td>
      <td>tetoolkit</td>
      <td>2020-04-10</td>
      <td>iuc</td>
      <td>https://github.com/mhammell-laboratory/TEtranscripts</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/tetoolkit</td>
      <td>2.2.3</td>
      <td>tetranscripts</td>
      <td>2.2.3</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1013</td>
      <td>1013</td>
      <td>74</td>
      <td>74</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>70</td>
      <td>70</td>
      <td>4</td>
      <td>4</td>
      <td>1083</td>
      <td>1083</td>
      <td>78</td>
      <td>78</td>
    </tr>
    <tr>
      <td></td>
      <td>tetyper</td>
      <td>tetyper</td>
      <td>Type a specific transposable element (TE) of interest from paired-end sequencing data.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/aesheppard/TETyper</td>
      <td>Sequence Analysis</td>
      <td>tetyper</td>
      <td>2019-11-23</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/tetyper</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/tetyper</td>
      <td>1.1</td>
      <td>tetyper</td>
      <td>1.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>106</td>
      <td>106</td>
      <td>16</td>
      <td>16</td>
      <td>155</td>
      <td>155</td>
      <td>47</td>
      <td>47</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>261</td>
      <td>261</td>
      <td>63</td>
      <td>63</td>
    </tr>
    <tr>
      <td></td>
      <td>tn93</td>
      <td>tn93_readreduce, tn93, tn93_cluster, tn93_filter</td>
      <td>Compute distances between sequences</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/veg/tn93/</td>
      <td>Sequence Analysis</td>
      <td>tn93</td>
      <td>2018-06-06</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/tn93/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/tn93</td>
      <td>1.0.14</td>
      <td>tn93</td>
      <td>1.0.14</td>
      <td></td>
      <td></td>
      <td>4</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>1915</td>
      <td>1915</td>
      <td>42</td>
      <td>42</td>
      <td>1972</td>
      <td>1972</td>
      <td>225</td>
      <td>225</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3887</td>
      <td>3887</td>
      <td>267</td>
      <td>267</td>
    </tr>
    <tr>
      <td></td>
      <td>tooldistillator</td>
      <td>tooldistillator, tooldistillator_summarize</td>
      <td>ToolDistillator extract and aggregate information from different tool outputs to JSON parsable files</td>
      <td>tooldistillator</td>
      <td>tooldistillator</td>
      <td></td>
      <td>ToolDistillator</td>
      <td>ToolDistillator is a tool to extract information from output files of specific tools, expose it as JSON files, and aggregate over several tools.It can produce both a single file to each tool or a summarized file from a set of reports.</td>
      <td>Data handling, Parsing</td>
      <td>Microbiology, Bioinformatics, Sequence analysis</td>
      <td>Up-to-date</td>
      <td>https://gitlab.com/ifb-elixirfr/abromics/tooldistillator</td>
      <td>Sequence Analysis</td>
      <td>tooldistillator</td>
      <td>2024-04-30</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/tooldistillator</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/tooldistillator</td>
      <td>0.9.1</td>
      <td>tooldistillator</td>
      <td>0.9.1</td>
      <td>Parsing</td>
      <td>Microbiology, Bioinformatics, Sequence analysis</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>65</td>
      <td>65</td>
      <td>9</td>
      <td>9</td>
      <td>6</td>
      <td>6</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>71</td>
      <td>71</td>
      <td>14</td>
      <td>14</td>
    </tr>
    <tr>
      <td></td>
      <td>tracegroomer</td>
      <td>tracegroomer</td>
      <td>TraceGroomer is a solution for formatting and normalising Tracer metabolomics given file(s), to produce the .csv files which are ready for DIMet tool.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/cbib/TraceGroomer</td>
      <td>Metabolomics</td>
      <td></td>
      <td>2024-03-28</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/Tracegroomer</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/tracegroomer</td>
      <td>0.1.4</td>
      <td>tracegroomer</td>
      <td>0.1.4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>1</td>
      <td>1</td>
    </tr>
    <tr>
      <td></td>
      <td>tracy</td>
      <td>tracy_align, tracy_assemble, tracy_basecall, tracy_decompose</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td>2021-10-12</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/tracy</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/tracy</td>
      <td>0.6.1</td>
      <td>tracy</td>
      <td>0.7.8</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>3314</td>
      <td>3314</td>
      <td>124</td>
      <td>124</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3314</td>
      <td>3314</td>
      <td>124</td>
      <td>124</td>
    </tr>
    <tr>
      <td></td>
      <td>transdecoder</td>
      <td>transdecoder</td>
      <td>TransDecoder finds coding regions within transcripts</td>
      <td>TransDecoder</td>
      <td>TransDecoder</td>
      <td></td>
      <td>TransDecoder</td>
      <td>TransDecoder identifies candidate coding regions within transcript sequences, such as those generated by de novo RNA-Seq transcript assembly using Trinity, or constructed based on RNA-Seq alignments to the genome using Tophat and Cufflinks.</td>
      <td>Coding region prediction, de Novo sequencing, De-novo assembly</td>
      <td>Genomics, Gene transcripts, RNA-Seq, Gene expression, Sequence assembly, Whole genome sequencing</td>
      <td>To update</td>
      <td>https://transdecoder.github.io/</td>
      <td>Transcriptomics, RNA</td>
      <td>transdecoder</td>
      <td>2015-11-09</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/transdecoder</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/transdecoder</td>
      <td>5.5.0</td>
      <td>transdecoder</td>
      <td>5.7.1</td>
      <td>Coding region prediction, de Novo sequencing, De-novo assembly</td>
      <td>Genomics, Gene transcripts, RNA-Seq, Gene expression, Sequence assembly, Whole genome sequencing</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>5964</td>
      <td>6418</td>
      <td>944</td>
      <td>970</td>
      <td>5450</td>
      <td>5450</td>
      <td>1066</td>
      <td>1066</td>
      <td>2610</td>
      <td>2796</td>
      <td>317</td>
      <td>344</td>
      <td>14024</td>
      <td>14664</td>
      <td>2327</td>
      <td>2380</td>
    </tr>
    <tr>
      <td></td>
      <td>transit</td>
      <td>gff_to_prot, transit_gumbel, transit_hmm, transit_resampling, transit_tn5gaps</td>
      <td>TRANSIT</td>
      <td>transit</td>
      <td>transit</td>
      <td></td>
      <td>TRANSIT</td>
      <td>A tool for the analysis of Tn-Seq data. It provides an easy to use graphical interface and access to three different analysis methods that allow the user to determine essentiality in a single condition as well as between conditions.</td>
      <td>Transposon prediction</td>
      <td>DNA, Sequencing, Mobile genetic elements</td>
      <td>To update</td>
      <td>https://github.com/mad-lab/transit/</td>
      <td>Genome annotation</td>
      <td></td>
      <td>2019-03-01</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/transit/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/transit</td>
      <td>3.0.2</td>
      <td>transit</td>
      <td>3.2.3</td>
      <td>Transposon prediction</td>
      <td>DNA, Sequencing, Mobile genetic elements</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>1039</td>
      <td>1039</td>
      <td>189</td>
      <td>189</td>
      <td>2672</td>
      <td>3349</td>
      <td>467</td>
      <td>485</td>
      <td>358</td>
      <td>358</td>
      <td>47</td>
      <td>47</td>
      <td>4069</td>
      <td>4746</td>
      <td>703</td>
      <td>721</td>
    </tr>
    <tr>
      <td></td>
      <td>transtermhp</td>
      <td>transtermhp</td>
      <td>Finds rho-independent transcription terminators in bacterial genomes</td>
      <td>transtermhp</td>
      <td>transtermhp</td>
      <td></td>
      <td>TransTermHP</td>
      <td>TransTermHP finds rho-independent transcription terminators in bacterial genomes. Each terminator found by the program is assigned a confidence value that estimates its probability of being a true terminator</td>
      <td>Transcriptional regulatory element prediction</td>
      <td>Transcription factors and regulatory sites</td>
      <td>To update</td>
      <td>https://transterm.cbcb.umd.edu</td>
      <td>Sequence Analysis</td>
      <td>transtermhp</td>
      <td>2015-10-08</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/transtermhp</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/transtermhp</td>
      <td></td>
      <td>transtermhp</td>
      <td>2.09</td>
      <td>Transcriptional regulatory element prediction</td>
      <td>Transcription factors and regulatory sites</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>250</td>
      <td>254</td>
      <td>31</td>
      <td>31</td>
      <td>174</td>
      <td>174</td>
      <td>45</td>
      <td>45</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>424</td>
      <td>428</td>
      <td>76</td>
      <td>76</td>
    </tr>
    <tr>
      <td></td>
      <td>trimmomatic</td>
      <td>trimmomatic</td>
      <td>A flexible read trimming tool for Illumina NGS data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://www.usadellab.org/cms/?page=trimmomatic</td>
      <td>Fastq Manipulation</td>
      <td>trimmomatic</td>
      <td>2024-01-03</td>
      <td>pjbriggs</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/packages/trimmomatic</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/trimmomatic</td>
      <td>0.39</td>
      <td>trimmomatic</td>
      <td>0.39</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>352353</td>
      <td>383014</td>
      <td>14547</td>
      <td>15408</td>
      <td>632385</td>
      <td>839058</td>
      <td>39323</td>
      <td>49850</td>
      <td>125340</td>
      <td>139097</td>
      <td>5268</td>
      <td>5975</td>
      <td>1110078</td>
      <td>1361169</td>
      <td>59138</td>
      <td>71233</td>
    </tr>
    <tr>
      <td></td>
      <td>trinity</td>
      <td>trinity_abundance_estimates_to_matrix, trinity_align_and_estimate_abundance, trinity_analyze_diff_expr, trinity_contig_exn50_statistic, trinity_define_clusters_by_cutting_tree, describe_samples, trinity_filter_low_expr_transcripts, trinity_gene_to_trans_map, trinity_run_de_analysis, trinity_samples_qccheck, trinity_super_transcripts, trinity, trinity_stats</td>
      <td>Trinity represents a method for the efficient and robust de novo reconstruction of transcriptomes from RNA-seq datahttps://github.com/trinityrnaseq/trinityrnaseq</td>
      <td>trinity</td>
      <td>trinity</td>
      <td></td>
      <td>Trinity</td>
      <td>Trinity is a transcriptome assembler which relies on three different tools, inchworm an assembler, chrysalis which pools contigs and butterfly which amongst others compacts a graph resulting from butterfly with reads.</td>
      <td>Transcriptome assembly</td>
      <td>Transcriptomics, Gene expression, Gene transcripts</td>
      <td>To update</td>
      <td>https://github.com/trinityrnaseq/trinityrnaseq</td>
      <td>Transcriptomics, RNA</td>
      <td></td>
      <td>2019-11-13</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/trinity</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/trinity</td>
      <td>2.15.1</td>
      <td>trinity</td>
      <td>2.15.2</td>
      <td>Transcriptome assembly</td>
      <td>Transcriptomics, Gene transcripts</td>
      <td>9</td>
      <td>13</td>
      <td>13</td>
      <td>13</td>
      <td>9</td>
      <td>13</td>
      <td>13</td>
      <td>13</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>2</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>11</td>
      <td>13</td>
      <td>12</td>
      <td>0</td>
      <td>35698</td>
      <td>39132</td>
      <td>4376</td>
      <td>4632</td>
      <td>44509</td>
      <td>44509</td>
      <td>6714</td>
      <td>6714</td>
      <td>21883</td>
      <td>23889</td>
      <td>1924</td>
      <td>2070</td>
      <td>102090</td>
      <td>107530</td>
      <td>13014</td>
      <td>13416</td>
    </tr>
    <tr>
      <td></td>
      <td>trinotate</td>
      <td>trinotate</td>
      <td>Trinotate is a comprehensive annotation suite designed for automatic functional annotation of de novo transcriptomes.</td>
      <td>trinotate</td>
      <td>trinotate</td>
      <td></td>
      <td>Trinotate</td>
      <td>Comprehensive annotation suite designed for automatic functional annotation of transcriptomes, particularly de novo assembled transcriptomes, from model or non-model organisms.</td>
      <td>Gene functional annotation</td>
      <td>Gene expression, Transcriptomics</td>
      <td>To update</td>
      <td>https://trinotate.github.io/</td>
      <td>Transcriptomics, RNA</td>
      <td>trinotate</td>
      <td>2016-11-15</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/trinotate</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/trinotate</td>
      <td>3.2.2</td>
      <td>trinotate</td>
      <td>4.0.2</td>
      <td>Gene functional annotation</td>
      <td>Transcriptomics</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1968</td>
      <td>2080</td>
      <td>374</td>
      <td>390</td>
      <td>1825</td>
      <td>1825</td>
      <td>527</td>
      <td>527</td>
      <td>862</td>
      <td>1060</td>
      <td>161</td>
      <td>195</td>
      <td>4655</td>
      <td>4965</td>
      <td>1062</td>
      <td>1112</td>
    </tr>
    <tr>
      <td></td>
      <td>trycycler</td>
      <td>trycycler_cluster, trycycler_consensus, trycycler_partition, trycycler_reconcile_msa, trycycler_subsample</td>
      <td>Trycycler toolkit wrappers</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/rrwick/Trycycler</td>
      <td>Assembly</td>
      <td>trycycler</td>
      <td>2021-02-11</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/trycycler</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/trycycler</td>
      <td>0.5.5</td>
      <td>trycycler</td>
      <td>0.5.5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>14681</td>
      <td>14681</td>
      <td>265</td>
      <td>265</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4488</td>
      <td>4488</td>
      <td>147</td>
      <td>147</td>
      <td>19169</td>
      <td>19169</td>
      <td>412</td>
      <td>412</td>
    </tr>
    <tr>
      <td></td>
      <td>tsebra</td>
      <td>tsebra</td>
      <td>This tool has been developed to combine BRAKER predictions.</td>
      <td>tsebra</td>
      <td>tsebra</td>
      <td></td>
      <td>TSEBRA</td>
      <td>TSEBRA is a combiner tool that selects transcripts from gene predictions based on the support by extrisic evidence in form of introns and start/stop codons. It was developed to combine BRAKER1 and BRAKER2 predicitons to increase their accuracies.</td>
      <td>Homology-based gene prediction, Alternative splicing prediction</td>
      <td>Gene expression, RNA-Seq, Gene transcripts, Model organisms</td>
      <td>Up-to-date</td>
      <td>https://github.com/Gaius-Augustus/TSEBRA</td>
      <td>Genome annotation</td>
      <td></td>
      <td>2023-10-19</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/tsebra</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/tsebra</td>
      <td>1.1.2.5</td>
      <td>tsebra</td>
      <td>1.1.2.5</td>
      <td>Homology-based gene prediction, Alternative splicing prediction</td>
      <td>Gene expression, RNA-Seq, Gene transcripts, Model organisms</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>17</td>
      <td>17</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>17</td>
      <td>17</td>
      <td>7</td>
      <td>7</td>
    </tr>
    <tr>
      <td></td>
      <td>tsne</td>
      <td>tsne</td>
      <td>T-Distributed Stochastic Neighbor Embedding using a Barnes-Hut Implementation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://cran.r-project.org/web/packages/Rtsne/</td>
      <td>Text Manipulation</td>
      <td>tsne</td>
      <td>2017-05-26</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/tsne</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/tsne</td>
      <td>0.0.2</td>
      <td>r-rtsne</td>
      <td>0.13</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>255</td>
      <td>379</td>
      <td>31</td>
      <td>37</td>
      <td>281</td>
      <td>281</td>
      <td>64</td>
      <td>64</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>536</td>
      <td>660</td>
      <td>95</td>
      <td>101</td>
    </tr>
    <tr>
      <td></td>
      <td>twobittofa</td>
      <td>ucsc-twobittofa</td>
      <td>twoBitToFa is a tool to convert all or part of .2bit file to FASTA</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td></td>
      <td>UCSC Genome Browser Utilities</td>
      <td>Utilities for handling sequences and assemblies from the UCSC Genome Browser project.</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>Up-to-date</td>
      <td>https://genome.ucsc.edu/goldenpath/help/twoBit.html</td>
      <td>Convert Formats</td>
      <td>ucsc_twobittofa</td>
      <td>2016-08-19</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ucsc_tools/twobittofa</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ucsc_tools/twobittofa</td>
      <td>469</td>
      <td>ucsc-twobittofa</td>
      <td>469</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>413</td>
      <td>511</td>
      <td>84</td>
      <td>118</td>
      <td>441</td>
      <td>441</td>
      <td>123</td>
      <td>123</td>
      <td>21</td>
      <td>21</td>
      <td>2</td>
      <td>2</td>
      <td>875</td>
      <td>973</td>
      <td>209</td>
      <td>243</td>
    </tr>
    <tr>
      <td></td>
      <td>tximport</td>
      <td>tximport</td>
      <td>Wrapper for the Bioconductor package tximport</td>
      <td>tximport</td>
      <td>tximport</td>
      <td></td>
      <td>tximport</td>
      <td>An R/Bioconductor package that imports transcript-level abundance, estimated counts and transcript lengths, and summarizes into matrices for use with downstream gene-level analysis packages.</td>
      <td>Pathway or network analysis, Formatting, RNA-Seq analysis</td>
      <td>Transcriptomics, Gene transcripts, Workflows</td>
      <td>Up-to-date</td>
      <td>http://bioconductor.org/packages/tximport/</td>
      <td>Transcriptomics</td>
      <td>tximport</td>
      <td>2019-11-27</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/tximport</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/tximport</td>
      <td>1.30.0</td>
      <td>bioconductor-tximport</td>
      <td>1.30.0</td>
      <td>Pathway or network analysis, Formatting, RNA-Seq analysis</td>
      <td>Transcriptomics, Gene transcripts, Workflows</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1802</td>
      <td>1802</td>
      <td>225</td>
      <td>225</td>
      <td>6963</td>
      <td>6963</td>
      <td>904</td>
      <td>904</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8765</td>
      <td>8765</td>
      <td>1129</td>
      <td>1129</td>
    </tr>
    <tr>
      <td></td>
      <td>ucsc_axtchain</td>
      <td>ucsc_axtchain</td>
      <td>Chain together genome alignments</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td></td>
      <td>UCSC Genome Browser Utilities</td>
      <td>Utilities for handling sequences and assemblies from the UCSC Genome Browser project.</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>Up-to-date</td>
      <td>http://hgdownload.cse.ucsc.edu/admin/exe/</td>
      <td>Sequence Analysis</td>
      <td>ucsc_axtchain</td>
      <td>2024-08-26</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ucsc_tools/axtchain</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ucsc_tools/ucsc_axtchain</td>
      <td>469</td>
      <td>ucsc-axtchain</td>
      <td>469</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>ucsc_axttomaf</td>
      <td>ucsc_axtomaf</td>
      <td>Convert dataset from axt to MAF format.</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td></td>
      <td>UCSC Genome Browser Utilities</td>
      <td>Utilities for handling sequences and assemblies from the UCSC Genome Browser project.</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>Up-to-date</td>
      <td>http://hgdownload.cse.ucsc.edu/admin/exe/</td>
      <td>Convert Formats</td>
      <td>ucsc_axttomaf</td>
      <td>2024-08-27</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ucsc_tools/axttomaf</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ucsc_tools/ucsc_axttomaf</td>
      <td>469</td>
      <td>ucsc-axttomaf</td>
      <td>469</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>ucsc_blat</td>
      <td>ucsc_blat</td>
      <td>Standalone blat sequence search command line tool</td>
      <td>blat</td>
      <td>blat</td>
      <td></td>
      <td>BLAT</td>
      <td>Fast, accurate spliced alignment of DNA sequences.</td>
      <td>Sequence alignment</td>
      <td>Sequence analysis</td>
      <td>Up-to-date</td>
      <td>http://genome.ucsc.edu/goldenPath/help/blatSpec.html</td>
      <td>Sequence Analysis</td>
      <td>ucsc_blat</td>
      <td>2017-05-17</td>
      <td>yating-l</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ucsc_blat/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ucsc_blat</td>
      <td>469</td>
      <td>ucsc-blat</td>
      <td>469</td>
      <td>Sequence alignment</td>
      <td>Sequence analysis</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>ucsc_chainnet</td>
      <td>ucsc_chainnet</td>
      <td>Make alignment nets out of chains</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td></td>
      <td>UCSC Genome Browser Utilities</td>
      <td>Utilities for handling sequences and assemblies from the UCSC Genome Browser project.</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>Up-to-date</td>
      <td>http://hgdownload.cse.ucsc.edu/admin/exe/</td>
      <td>Sequence Analysis</td>
      <td>ucsc_chainnet</td>
      <td>2024-08-30</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ucsc_tools/ucsc_chainnet</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ucsc_tools/ucsc_chainnet</td>
      <td>469</td>
      <td>ucsc-chainnet</td>
      <td>469</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>ucsc_chainprenet</td>
      <td>ucsc_chainprenet</td>
      <td>Remove chains that don't have a chance of being netted</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td></td>
      <td>UCSC Genome Browser Utilities</td>
      <td>Utilities for handling sequences and assemblies from the UCSC Genome Browser project.</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>Up-to-date</td>
      <td>http://hgdownload.cse.ucsc.edu/admin/exe/</td>
      <td>Sequence Analysis</td>
      <td>ucsc_chainprenet</td>
      <td>2024-08-27</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ucsc_tools/chainprenet</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ucsc_tools/ucsc_chainprenet</td>
      <td>469</td>
      <td>ucsc-chainprenet</td>
      <td>469</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>ucsc_chainsort</td>
      <td>ucsc_chainsort</td>
      <td>Sort chains. By default sorts by score.</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td></td>
      <td>UCSC Genome Browser Utilities</td>
      <td>Utilities for handling sequences and assemblies from the UCSC Genome Browser project.</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>Up-to-date</td>
      <td>http://hgdownload.cse.ucsc.edu/admin/exe/</td>
      <td>Sequence Analysis</td>
      <td>ucsc_chainsort</td>
      <td>2024-08-26</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ucsc_tools/chainsort</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ucsc_tools/ucsc_chainsort</td>
      <td>469</td>
      <td>ucsc-chainsort</td>
      <td>469</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>ucsc_netfilter</td>
      <td>ucsc_netfilter</td>
      <td>Filter out parts of net</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td></td>
      <td>UCSC Genome Browser Utilities</td>
      <td>Utilities for handling sequences and assemblies from the UCSC Genome Browser project.</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>Up-to-date</td>
      <td>http://hgdownload.cse.ucsc.edu/admin/exe/</td>
      <td>Sequence Analysis</td>
      <td>ucsc_netfilter</td>
      <td>2024-09-18</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ucsc_tools/ucsc_netfilter</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ucsc_tools/ucsc_netfilter</td>
      <td>469</td>
      <td>ucsc-netfilter</td>
      <td>469</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>ucsc_netsyntenic</td>
      <td>ucsc_netsyntenic</td>
      <td>Add synteny info to a net dataset</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td></td>
      <td>UCSC Genome Browser Utilities</td>
      <td>Utilities for handling sequences and assemblies from the UCSC Genome Browser project.</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>Up-to-date</td>
      <td>http://hgdownload.cse.ucsc.edu/admin/exe/</td>
      <td>Sequence Analysis</td>
      <td>ucsc_netsyntenic</td>
      <td>2024-09-04</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ucsc_tools/ucsc_netsyntenic</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ucsc_tools/ucsc_netsyntenic</td>
      <td>469</td>
      <td>ucsc-netsyntenic</td>
      <td>469</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>ucsc_nettoaxt</td>
      <td>ucsc_nettoaxt</td>
      <td>Convert net (and chain) to axt format.</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td></td>
      <td>UCSC Genome Browser Utilities</td>
      <td>Utilities for handling sequences and assemblies from the UCSC Genome Browser project.</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>Up-to-date</td>
      <td>http://hgdownload.cse.ucsc.edu/admin/exe/</td>
      <td>Convert Formats</td>
      <td>ucsc_nettoaxt</td>
      <td>2024-09-19</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ucsc_tools/ucsc_nettoaxt</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ucsc_tools/ucsc_nettoaxt</td>
      <td>469</td>
      <td>ucsc-nettoaxt</td>
      <td>469</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>umi_tools</td>
      <td>umi_tools_count, umi_tools_dedup, umi_tools_extract, umi_tools_group, umi_tools_whitelist</td>
      <td>UMI-tools extract - Extract UMIs from fastq</td>
      <td>umi-tools</td>
      <td>umi-tools</td>
      <td></td>
      <td>UMI-tools</td>
      <td>Tools for handling Unique Molecular Identifiers in NGS data sets.</td>
      <td>Sequencing quality control</td>
      <td>NGS, Sequence sites, features and motifs, Quality affairs</td>
      <td>Up-to-date</td>
      <td>https://github.com/CGATOxford/UMI-tools</td>
      <td>Sequence Analysis, Transcriptomics, Single Cell</td>
      <td></td>
      <td>2021-07-15</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/umi_tools</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/umi_tools</td>
      <td>1.1.5</td>
      <td>umi_tools</td>
      <td>1.1.5</td>
      <td>Sequencing quality control</td>
      <td>Sequence sites, features and motifs, Quality affairs</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>55050</td>
      <td>68123</td>
      <td>1090</td>
      <td>1223</td>
      <td>9001</td>
      <td>9001</td>
      <td>906</td>
      <td>906</td>
      <td>12444</td>
      <td>12444</td>
      <td>187</td>
      <td>187</td>
      <td>76495</td>
      <td>89568</td>
      <td>2183</td>
      <td>2316</td>
    </tr>
    <tr>
      <td></td>
      <td>unicycler</td>
      <td>unicycler</td>
      <td>Unicycler is a hybrid assembly pipeline for bacterial genomes.</td>
      <td>unicycler</td>
      <td>unicycler</td>
      <td></td>
      <td>Unicycler</td>
      <td>A tool for assembling bacterial genomes from a combination of short (2nd generation) and long (3rd generation) sequencing reads.</td>
      <td>Genome assembly, Aggregation</td>
      <td>Microbiology, Genomics, Sequencing, Sequence assembly</td>
      <td>Up-to-date</td>
      <td>https://github.com/rrwick/Unicycler</td>
      <td>Assembly</td>
      <td>unicycler</td>
      <td>2018-08-28</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/unicycler</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/unicycler</td>
      <td>0.5.1</td>
      <td>unicycler</td>
      <td>0.5.1</td>
      <td>Genome assembly, Aggregation</td>
      <td>Microbiology, Genomics, Sequencing, Sequence assembly</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>74120</td>
      <td>77648</td>
      <td>3798</td>
      <td>3963</td>
      <td>64946</td>
      <td>90075</td>
      <td>5451</td>
      <td>7380</td>
      <td>19516</td>
      <td>20422</td>
      <td>2010</td>
      <td>2080</td>
      <td>158582</td>
      <td>188145</td>
      <td>11259</td>
      <td>13423</td>
    </tr>
    <tr>
      <td></td>
      <td>usher</td>
      <td>usher_matutils, usher</td>
      <td>UShER toolkit wrappers</td>
      <td>usher</td>
      <td>usher</td>
      <td></td>
      <td>usher</td>
      <td>The UShER toolkit includes a set of tools for for rapid, accurate placement of samples to existing phylogenies. While not restricted to SARS-CoV-2 phylogenetic analyses, it has enabled real-time phylogenetic analyses and genomic contact tracing in that its placement is orders of magnitude faster and more memory-efficient than previous methods.</td>
      <td>Classification, Phylogenetic tree visualisation, Phylogenetic inference (from molecular sequences)</td>
      <td>Phylogeny, Evolutionary biology, Cladistics, Genotype and phenotype, Phylogenomics</td>
      <td>To update</td>
      <td>https://github.com/yatisht/usher</td>
      <td>Phylogenetics</td>
      <td>usher</td>
      <td>2021-05-11</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/usher</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/usher</td>
      <td>0.2.1</td>
      <td>usher</td>
      <td>0.6.3</td>
      <td>Classification, Phylogenetic tree visualisation, Phylogenetic inference (from molecular sequences)</td>
      <td>Cladistics, Genotype and phenotype, Phylogenomics</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>1060</td>
      <td>1060</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1060</td>
      <td>1060</td>
      <td>5</td>
      <td>5</td>
    </tr>
    <tr>
      <td></td>
      <td>valet</td>
      <td>valet</td>
      <td>A pipeline for detecting mis-assemblies in metagenomic assemblies.</td>
      <td>valet</td>
      <td>valet</td>
      <td></td>
      <td>VALET</td>
      <td>VALET is a pipeline for detecting mis-assemblies in metagenomic assemblies.</td>
      <td>Sequence assembly, Sequence assembly visualisation</td>
      <td>Metagenomics, Sequence assembly</td>
      <td>To update</td>
      <td>https://github.com/marbl/VALET</td>
      <td>Metagenomics</td>
      <td>valet</td>
      <td>2017-11-16</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/valet</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/valet</td>
      <td></td>
      <td>valet</td>
      <td>1.0</td>
      <td>Sequence assembly, Sequence assembly visualisation</td>
      <td>Metagenomics, Sequence assembly</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>518</td>
      <td>677</td>
      <td>75</td>
      <td>87</td>
      <td>234</td>
      <td>234</td>
      <td>65</td>
      <td>65</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>752</td>
      <td>911</td>
      <td>140</td>
      <td>152</td>
    </tr>
    <tr>
      <td></td>
      <td>vapor</td>
      <td>vapor</td>
      <td>Classify Influenza samples from raw short read sequence data</td>
      <td>vapor</td>
      <td>vapor</td>
      <td></td>
      <td>VAPOR</td>
      <td>VAPOR is a tool for classification of Influenza samples from raw short read sequence data for downstream bioinformatics analysis. VAPOR is provided with a fasta file of full-length sequences (&gt; 20,000) for a given segment, a set of reads, and attempts to retrieve a reference that is closest to the sample strain.</td>
      <td>Data retrieval, De-novo assembly, Read mapping</td>
      <td>Whole genome sequencing, Mapping, Sequence assembly</td>
      <td>Up-to-date</td>
      <td>https://github.com/connor-lab/vapor</td>
      <td>Sequence Analysis</td>
      <td>vapor</td>
      <td>2022-08-24</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/vapor</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/vapor</td>
      <td>1.0.2</td>
      <td>vapor</td>
      <td>1.0.2</td>
      <td>Data retrieval, De-novo assembly, Read mapping</td>
      <td>Whole genome sequencing, Mapping, Sequence assembly</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>5936</td>
      <td>5936</td>
      <td>147</td>
      <td>147</td>
      <td>523</td>
      <td>523</td>
      <td>15</td>
      <td>15</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6459</td>
      <td>6459</td>
      <td>162</td>
      <td>162</td>
    </tr>
    <tr>
      <td></td>
      <td>vardict</td>
      <td>vardict_java</td>
      <td>VarDict - calls SNVs and indels for tumour-normal pairs</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/AstraZeneca-NGS/VarDictJava</td>
      <td>Variant Analysis</td>
      <td>vardict_java</td>
      <td>2020-08-23</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/vardict</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/vardict</td>
      <td>1.8.3</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>381</td>
      <td>381</td>
      <td>37</td>
      <td>37</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>381</td>
      <td>381</td>
      <td>37</td>
      <td>37</td>
    </tr>
    <tr>
      <td></td>
      <td>variant_analyzer</td>
      <td>mut2read, mut2sscs, read2mut</td>
      <td>Collection of tools for analyzing variants in duplex consensus sequencing (DCS) data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Variant Analysis</td>
      <td>variant_analyzer</td>
      <td>2019-11-20</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/variant_analyzer</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/variant_analyzer</td>
      <td>2.0.0</td>
      <td>matplotlib</td>
      <td></td>
      <td></td>
      <td></td>
      <td>3</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>95</td>
      <td>95</td>
      <td>14</td>
      <td>14</td>
      <td>516</td>
      <td>516</td>
      <td>67</td>
      <td>67</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>611</td>
      <td>611</td>
      <td>81</td>
      <td>81</td>
    </tr>
    <tr>
      <td></td>
      <td>varscan</td>
      <td>varscan_copynumber, varscan_mpileup, varscan_somatic</td>
      <td>VarScan is a variant caller for high-throughput sequencing data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://dkoboldt.github.io/varscan/</td>
      <td>Variant Analysis</td>
      <td>varscan</td>
      <td>2018-11-29</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/iuc/tree/master/tools/varscan</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/varscan</td>
      <td>2.4.3</td>
      <td>varscan</td>
      <td>2.4.6</td>
      <td></td>
      <td></td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>10711</td>
      <td>11502</td>
      <td>1442</td>
      <td>1560</td>
      <td>16705</td>
      <td>16705</td>
      <td>3135</td>
      <td>3135</td>
      <td>1036</td>
      <td>1036</td>
      <td>100</td>
      <td>100</td>
      <td>28452</td>
      <td>29243</td>
      <td>4677</td>
      <td>4795</td>
    </tr>
    <tr>
      <td></td>
      <td>varvamp</td>
      <td>varvamp</td>
      <td>Variable VirusAMPlicons (varVAMP) is a tool to design primers for highly diverse viruses</td>
      <td>varvamp</td>
      <td>varvamp</td>
      <td></td>
      <td>varVAMP</td>
      <td>variable VirusAMPlicons (varVAMP) is a tool to design primers for highly diverse viruses. The input is an alignment of your viral (full-genome) sequences.</td>
      <td>PCR primer design</td>
      <td>Virology, Probes and primers</td>
      <td>To update</td>
      <td>https://github.com/jonas-fuchs/varVAMP/</td>
      <td>Sequence Analysis</td>
      <td>varvamp</td>
      <td>2024-01-27</td>
      <td>iuc</td>
      <td>https://github.com/jonas-fuchs/varVAMP</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/varvamp</td>
      <td>1.2.0</td>
      <td>varvamp</td>
      <td>1.2.1</td>
      <td>PCR primer design</td>
      <td>Virology, Probes and primers</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>273</td>
      <td>273</td>
      <td>12</td>
      <td>12</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>273</td>
      <td>273</td>
      <td>12</td>
      <td>12</td>
    </tr>
    <tr>
      <td></td>
      <td>vcf2maf</td>
      <td>vcf2maf</td>
      <td>vcf2maf: Convert VCF into MAF</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/mskcc/vcf2maf</td>
      <td>Convert Formats</td>
      <td>vcf2maf</td>
      <td>2022-06-28</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/vcf2maf</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/vcf2maf</td>
      <td>1.6.21</td>
      <td>vcf2maf</td>
      <td>1.6.22</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1097</td>
      <td>1097</td>
      <td>46</td>
      <td>46</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>65</td>
      <td>65</td>
      <td>5</td>
      <td>5</td>
      <td>1162</td>
      <td>1162</td>
      <td>51</td>
      <td>51</td>
    </tr>
    <tr>
      <td></td>
      <td>vcf2tsv</td>
      <td>vcf2tsv</td>
      <td>Converts VCF files into tab-delimited format</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis, Convert Formats</td>
      <td>vcf2tsv</td>
      <td>2017-01-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcf2tsv</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/vcflib/vcf2tsv</td>
      <td></td>
      <td>vcflib</td>
      <td>1.0.10</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>169090</td>
      <td>169679</td>
      <td>823</td>
      <td>874</td>
      <td>16304</td>
      <td>42215</td>
      <td>1141</td>
      <td>2594</td>
      <td>2485</td>
      <td>2511</td>
      <td>169</td>
      <td>172</td>
      <td>187879</td>
      <td>214405</td>
      <td>2133</td>
      <td>3640</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfaddinfo</td>
      <td>vcfaddinfo</td>
      <td>Adds info fields from the second dataset which are not present in the first dataset.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfaddinfo</td>
      <td>2017-01-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfaddinfo</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/vcflib/vcfaddinfo</td>
      <td></td>
      <td>vcflib</td>
      <td>1.0.10</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>882</td>
      <td>888</td>
      <td>30</td>
      <td>32</td>
      <td>6118</td>
      <td>8261</td>
      <td>36</td>
      <td>114</td>
      <td>175</td>
      <td>184</td>
      <td>6</td>
      <td>6</td>
      <td>7175</td>
      <td>9333</td>
      <td>72</td>
      <td>152</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfallelicprimitives</td>
      <td>vcfallelicprimitives</td>
      <td>Splits alleleic primitives (gaps or mismatches) into multiple VCF lines</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfallelicprimitives</td>
      <td>2017-01-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfallelicprimitives</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/vcflib/vcfallelicprimitives</td>
      <td></td>
      <td>vcflib</td>
      <td>1.0.10</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>6879</td>
      <td>17157</td>
      <td>897</td>
      <td>1219</td>
      <td>7585</td>
      <td>14509</td>
      <td>1245</td>
      <td>1925</td>
      <td>1648</td>
      <td>1681</td>
      <td>225</td>
      <td>229</td>
      <td>16112</td>
      <td>33347</td>
      <td>2367</td>
      <td>3373</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfanno</td>
      <td>vcfanno</td>
      <td>Annotate VCF files</td>
      <td>vcfanno</td>
      <td>vcfanno</td>
      <td></td>
      <td>vcfanno</td>
      <td>Fast, flexible annotation of genetic variants.</td>
      <td>SNP annotation</td>
      <td>Genetic variation, Data submission, annotation and curation</td>
      <td>Up-to-date</td>
      <td>https://github.com/brentp/vcfanno</td>
      <td>Variant Analysis</td>
      <td>vcfanno</td>
      <td>2021-01-17</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/vcfanno/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/vcfanno</td>
      <td>0.3.5</td>
      <td>vcfanno</td>
      <td>0.3.5</td>
      <td>SNP annotation</td>
      <td>Genetic variation, Data submission, annotation and curation</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1139</td>
      <td>1139</td>
      <td>174</td>
      <td>174</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1139</td>
      <td>1139</td>
      <td>174</td>
      <td>174</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfannotate</td>
      <td>vcfannotate</td>
      <td>Intersect VCF records with BED annotations</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfannotate</td>
      <td>2017-01-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfannotate</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/vcflib/vcfannotate</td>
      <td></td>
      <td>vcflib</td>
      <td>1.0.10</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2797</td>
      <td>2850</td>
      <td>157</td>
      <td>181</td>
      <td>3668</td>
      <td>14532</td>
      <td>652</td>
      <td>1392</td>
      <td>494</td>
      <td>507</td>
      <td>61</td>
      <td>63</td>
      <td>6959</td>
      <td>17889</td>
      <td>870</td>
      <td>1636</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfannotategenotypes</td>
      <td>vcfannotategenotypes</td>
      <td>Annotate genotypes in a VCF dataset using genotypes from another VCF dataset.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfannotategenotypes</td>
      <td>2017-01-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfannotategenotypes</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/vcflib/vcfannotategenotypes</td>
      <td></td>
      <td>vcflib</td>
      <td>1.0.10</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>189</td>
      <td>203</td>
      <td>62</td>
      <td>66</td>
      <td>497</td>
      <td>1828</td>
      <td>155</td>
      <td>387</td>
      <td>184</td>
      <td>197</td>
      <td>11</td>
      <td>11</td>
      <td>870</td>
      <td>2228</td>
      <td>228</td>
      <td>464</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfbedintersect</td>
      <td>vcfbedintersect</td>
      <td>Intersect VCF and BED datasets</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfbedintersect</td>
      <td>2017-01-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfbedintersect</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/vcflib/vcfbedintersect</td>
      <td></td>
      <td>vcflib</td>
      <td>1.0.10</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>3017</td>
      <td>4049</td>
      <td>185</td>
      <td>210</td>
      <td>4709</td>
      <td>20250</td>
      <td>406</td>
      <td>1147</td>
      <td>507</td>
      <td>527</td>
      <td>40</td>
      <td>42</td>
      <td>8233</td>
      <td>24826</td>
      <td>631</td>
      <td>1399</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfbreakcreatemulti</td>
      <td>vcfbreakcreatemulti</td>
      <td>Break multiple alleles into multiple records, or combine overallpoing alleles into a single record</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfbreakcreatemulti</td>
      <td>2017-01-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfbreakcreatemulti</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/vcflib/vcfbreakcreatemulti</td>
      <td></td>
      <td>vcflib</td>
      <td>1.0.10</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>416</td>
      <td>478</td>
      <td>35</td>
      <td>40</td>
      <td>3969</td>
      <td>10457</td>
      <td>82</td>
      <td>309</td>
      <td>362</td>
      <td>378</td>
      <td>8</td>
      <td>8</td>
      <td>4747</td>
      <td>11313</td>
      <td>125</td>
      <td>357</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfcheck</td>
      <td>vcfcheck</td>
      <td>Verify that the reference allele matches the reference genome</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfcheck</td>
      <td>2017-01-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfcheck</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/vcflib/vcfcheck</td>
      <td></td>
      <td>vcflib</td>
      <td>1.0.10</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>745</td>
      <td>767</td>
      <td>96</td>
      <td>106</td>
      <td>803</td>
      <td>1895</td>
      <td>222</td>
      <td>627</td>
      <td>241</td>
      <td>256</td>
      <td>16</td>
      <td>17</td>
      <td>1789</td>
      <td>2918</td>
      <td>334</td>
      <td>750</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfcombine</td>
      <td>vcfcombine</td>
      <td>Combine multiple VCF datasets</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfcombine</td>
      <td>2017-01-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfcombine</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/vcflib/vcfcombine</td>
      <td></td>
      <td>vcflib</td>
      <td>1.0.10</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1422</td>
      <td>2020</td>
      <td>284</td>
      <td>505</td>
      <td>4951</td>
      <td>19530</td>
      <td>1021</td>
      <td>2669</td>
      <td>493</td>
      <td>516</td>
      <td>60</td>
      <td>62</td>
      <td>6866</td>
      <td>22066</td>
      <td>1365</td>
      <td>3236</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfcommonsamples</td>
      <td>vcfcommonsamples</td>
      <td>Output records belonging to samples commong between two datasets.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfcommonsamples</td>
      <td>2017-01-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfcommonsamples</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/vcflib/vcfcommonsamples</td>
      <td></td>
      <td>vcflib</td>
      <td>1.0.10</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>233</td>
      <td>253</td>
      <td>37</td>
      <td>42</td>
      <td>234</td>
      <td>1839</td>
      <td>79</td>
      <td>287</td>
      <td>199</td>
      <td>214</td>
      <td>11</td>
      <td>12</td>
      <td>666</td>
      <td>2306</td>
      <td>127</td>
      <td>341</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfdistance</td>
      <td>vcfdistance</td>
      <td>Calculate distance to the nearest variant.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfdistance</td>
      <td>2017-01-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfdistance</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/vcflib/vcfdistance</td>
      <td></td>
      <td>vcflib</td>
      <td>1.0.10</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>572</td>
      <td>585</td>
      <td>90</td>
      <td>92</td>
      <td>2063</td>
      <td>2427</td>
      <td>139</td>
      <td>296</td>
      <td>272</td>
      <td>290</td>
      <td>19</td>
      <td>22</td>
      <td>2907</td>
      <td>3302</td>
      <td>248</td>
      <td>410</td>
    </tr>
    <tr>
      <td></td>
      <td>vcffilter</td>
      <td>vcffilter2</td>
      <td>Tool for filtering VCF files</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcffilter</td>
      <td>2017-01-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcffilter</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/vcflib/vcffilter</td>
      <td></td>
      <td>vcflib</td>
      <td>1.0.10</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>37489</td>
      <td>47142</td>
      <td>2113</td>
      <td>2218</td>
      <td>75128</td>
      <td>162296</td>
      <td>5730</td>
      <td>10533</td>
      <td>2626</td>
      <td>2736</td>
      <td>320</td>
      <td>331</td>
      <td>115243</td>
      <td>212174</td>
      <td>8163</td>
      <td>13082</td>
    </tr>
    <tr>
      <td></td>
      <td>vcffixup</td>
      <td>vcffixup</td>
      <td>Count the allele frequencies across alleles present in each record in the VCF file.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcffixup</td>
      <td>2017-01-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcffixup</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/vcflib/vcffixup</td>
      <td></td>
      <td>vcflib</td>
      <td>1.0.10</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>926</td>
      <td>957</td>
      <td>155</td>
      <td>173</td>
      <td>2291</td>
      <td>5323</td>
      <td>313</td>
      <td>985</td>
      <td>251</td>
      <td>265</td>
      <td>28</td>
      <td>31</td>
      <td>3468</td>
      <td>6545</td>
      <td>496</td>
      <td>1189</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfflatten</td>
      <td>vcfflatten2</td>
      <td>Removes multi-allelic sites by picking the most common alternate</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfflatten</td>
      <td>2017-01-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfflatten</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/vcflib/vcfflatten</td>
      <td></td>
      <td>vcflib</td>
      <td>1.0.10</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>172</td>
      <td>183</td>
      <td>21</td>
      <td>25</td>
      <td>351</td>
      <td>879</td>
      <td>67</td>
      <td>201</td>
      <td>312</td>
      <td>342</td>
      <td>15</td>
      <td>16</td>
      <td>835</td>
      <td>1404</td>
      <td>103</td>
      <td>242</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfgeno2haplo</td>
      <td>vcfgeno2haplo</td>
      <td>Convert genotype-based phased alleles into haplotype alleles</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfgeno2haplo</td>
      <td>2017-01-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfgeno2haplo</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/vcflib/vcfgeno2haplo</td>
      <td></td>
      <td>vcflib</td>
      <td>1.0.10</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>778</td>
      <td>794</td>
      <td>107</td>
      <td>112</td>
      <td>2414</td>
      <td>3721</td>
      <td>179</td>
      <td>508</td>
      <td>209</td>
      <td>225</td>
      <td>20</td>
      <td>21</td>
      <td>3401</td>
      <td>4740</td>
      <td>306</td>
      <td>641</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfgenotypes</td>
      <td>vcfgenotypes</td>
      <td>Convert numerical representation of genotypes to allelic.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfgenotypes</td>
      <td>2017-01-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfgenotypes</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/vcflib/vcfgenotypes</td>
      <td></td>
      <td>vcflib</td>
      <td>1.0.10</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>502</td>
      <td>555</td>
      <td>134</td>
      <td>143</td>
      <td>3428</td>
      <td>6338</td>
      <td>266</td>
      <td>858</td>
      <td>220</td>
      <td>235</td>
      <td>23</td>
      <td>24</td>
      <td>4150</td>
      <td>7128</td>
      <td>423</td>
      <td>1025</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfhethom</td>
      <td>vcfhethom</td>
      <td>Count the number of heterozygotes and alleles, compute het/hom ratio.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfhethom</td>
      <td>2017-01-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfhethom</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/vcflib/vcfhethom</td>
      <td></td>
      <td>vcflib</td>
      <td>1.0.10</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1964</td>
      <td>1991</td>
      <td>182</td>
      <td>186</td>
      <td>1937</td>
      <td>3846</td>
      <td>188</td>
      <td>644</td>
      <td>567</td>
      <td>601</td>
      <td>26</td>
      <td>27</td>
      <td>4468</td>
      <td>6438</td>
      <td>396</td>
      <td>857</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfleftalign</td>
      <td>vcfleftalign</td>
      <td>Left-align indels and complex variants in VCF dataset</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfleftalign</td>
      <td>2017-01-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfleftalign</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/vcflib/vcfleftalign</td>
      <td></td>
      <td>vcflib</td>
      <td>1.0.10</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>269</td>
      <td>310</td>
      <td>38</td>
      <td>42</td>
      <td>358</td>
      <td>1071</td>
      <td>89</td>
      <td>287</td>
      <td>179</td>
      <td>213</td>
      <td>12</td>
      <td>14</td>
      <td>806</td>
      <td>1594</td>
      <td>139</td>
      <td>343</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfprimers</td>
      <td>vcfprimers</td>
      <td>Extract flanking sequences for each VCF record</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfprimers</td>
      <td>2017-01-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfprimers</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/vcflib/vcfprimers</td>
      <td></td>
      <td>vcflib</td>
      <td>1.0.10</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>590</td>
      <td>711</td>
      <td>28</td>
      <td>33</td>
      <td>403</td>
      <td>831</td>
      <td>75</td>
      <td>200</td>
      <td>190</td>
      <td>214</td>
      <td>10</td>
      <td>12</td>
      <td>1183</td>
      <td>1756</td>
      <td>113</td>
      <td>245</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfrandomsample</td>
      <td>vcfrandomsample</td>
      <td>Randomly sample sites from VCF dataset</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfrandomsample</td>
      <td>2017-01-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfrandomsample</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/vcflib/vcfrandomsample</td>
      <td></td>
      <td>vcflib</td>
      <td>1.0.10</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>161</td>
      <td>168</td>
      <td>13</td>
      <td>15</td>
      <td>153</td>
      <td>350</td>
      <td>27</td>
      <td>75</td>
      <td>184</td>
      <td>192</td>
      <td>6</td>
      <td>6</td>
      <td>498</td>
      <td>710</td>
      <td>46</td>
      <td>96</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfselectsamples</td>
      <td>vcfselectsamples</td>
      <td>Select samples from a VCF file</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfselectsamples</td>
      <td>2017-01-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfselectsamples</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/vcflib/vcfselectsamples</td>
      <td></td>
      <td>vcflib</td>
      <td>1.0.10</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>830</td>
      <td>900</td>
      <td>69</td>
      <td>88</td>
      <td>1315</td>
      <td>6257</td>
      <td>237</td>
      <td>959</td>
      <td>344</td>
      <td>363</td>
      <td>20</td>
      <td>21</td>
      <td>2489</td>
      <td>7520</td>
      <td>326</td>
      <td>1068</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfsort</td>
      <td>vcfsort</td>
      <td>Sort VCF dataset by coordinate</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfsort</td>
      <td>2017-01-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfsort</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/vcflib/vcfsort</td>
      <td></td>
      <td>vcflib</td>
      <td>1.0.10</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1868</td>
      <td>1904</td>
      <td>174</td>
      <td>185</td>
      <td>18378</td>
      <td>24175</td>
      <td>422</td>
      <td>1279</td>
      <td>537</td>
      <td>674</td>
      <td>70</td>
      <td>73</td>
      <td>20783</td>
      <td>26753</td>
      <td>666</td>
      <td>1537</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfvcfintersect</td>
      <td>vcfvcfintersect</td>
      <td>Intersect two VCF datasets</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfvcfintersect</td>
      <td>2017-01-24</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfvcfintersect</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/vcflib/vcfvcfintersect</td>
      <td></td>
      <td>vcflib</td>
      <td>1.0.10</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>976544</td>
      <td>977015</td>
      <td>703</td>
      <td>748</td>
      <td>64155</td>
      <td>81830</td>
      <td>981</td>
      <td>2103</td>
      <td>11044</td>
      <td>11069</td>
      <td>225</td>
      <td>229</td>
      <td>1051743</td>
      <td>1069914</td>
      <td>1909</td>
      <td>3080</td>
    </tr>
    <tr>
      <td></td>
      <td>vegan</td>
      <td>vegan_diversity, vegan_fisher_alpha, vegan_rarefaction</td>
      <td>an R package fo community ecologist</td>
      <td>vegan</td>
      <td>vegan</td>
      <td></td>
      <td>vegan</td>
      <td>Ordination methods, diversity analysis and other functions for community and vegetation ecologists</td>
      <td>Standardisation and normalisation, Analysis</td>
      <td>Ecology, Phylogenetics, Environmental science</td>
      <td>To update</td>
      <td>https://cran.r-project.org/package=vegan</td>
      <td>Metagenomics</td>
      <td></td>
      <td>2015-08-27</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/vegan/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/vegan</td>
      <td>2.4-3</td>
      <td>r-vegan</td>
      <td>2.3_4</td>
      <td>Standardisation and normalisation, Analysis</td>
      <td>Ecology, Phylogenetics, Environmental science</td>
      <td>3</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>2157</td>
      <td>2287</td>
      <td>198</td>
      <td>209</td>
      <td>3930</td>
      <td>5591</td>
      <td>446</td>
      <td>744</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6087</td>
      <td>7878</td>
      <td>644</td>
      <td>953</td>
    </tr>
    <tr>
      <td></td>
      <td>velocyto</td>
      <td>velocyto_cli</td>
      <td>Velocyto is a library for the analysis of RNA velocity.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://velocyto.org/</td>
      <td>Transcriptomics, Single Cell</td>
      <td>velocyto</td>
      <td>2021-04-30</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/velocyto</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/velocyto</td>
      <td>0.17.17</td>
      <td>velocyto.py</td>
      <td>0.17.17</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>4</td>
      <td>4</td>
      <td>22</td>
      <td>22</td>
      <td>6</td>
      <td>6</td>
      <td>16</td>
      <td>16</td>
      <td>1</td>
      <td>1</td>
      <td>45</td>
      <td>45</td>
      <td>11</td>
      <td>11</td>
    </tr>
    <tr>
      <td></td>
      <td>velvet</td>
      <td>velvetg, velveth</td>
      <td>de novo genomic assembler specially designed for short read sequencing technologies</td>
      <td>velvet</td>
      <td>velvet</td>
      <td></td>
      <td>Velvet</td>
      <td>A de novo genomic assembler specially designed for short read sequencing technologies, such as Solexa or 454 or SOLiD.</td>
      <td>Formatting, De-novo assembly</td>
      <td>Sequence assembly</td>
      <td>To update</td>
      <td>https://www.ebi.ac.uk/~zerbino/velvet/</td>
      <td>Assembly</td>
      <td>velvet</td>
      <td>2020-06-09</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/velvet</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/velvet</td>
      <td></td>
      <td>velvet</td>
      <td>1.2.10</td>
      <td>Formatting, De-novo assembly</td>
      <td>Sequence assembly</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>25171</td>
      <td>27943</td>
      <td>3899</td>
      <td>4270</td>
      <td>51187</td>
      <td>51187</td>
      <td>7557</td>
      <td>7557</td>
      <td>28546</td>
      <td>28619</td>
      <td>3585</td>
      <td>3613</td>
      <td>104904</td>
      <td>107749</td>
      <td>15041</td>
      <td>15440</td>
    </tr>
    <tr>
      <td></td>
      <td>velvet_optimiser</td>
      <td>velvetoptimiser</td>
      <td>Automatically optimize Velvet assemblies</td>
      <td>velvetoptimiser</td>
      <td>velvetoptimiser</td>
      <td></td>
      <td>VelvetOptimiser</td>
      <td>This tool is designed to run as a wrapper script for the Velvet assembler (Daniel Zerbino, EBI UK) and to assist with optimising the assembly.</td>
      <td>Optimisation and refinement, Sequence assembly</td>
      <td>Genomics, Sequence assembly</td>
      <td>To update</td>
      <td></td>
      <td>Assembly</td>
      <td>velvetoptimiser</td>
      <td>2017-12-19</td>
      <td>simon-gladman</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/velvetoptimiser</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/velvet_optimiser</td>
      <td>2.2.6+galaxy2</td>
      <td>velvet</td>
      <td>1.2.10</td>
      <td>Optimisation and refinement, Sequence assembly</td>
      <td>Genomics, Sequence assembly</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>4715</td>
      <td>5343</td>
      <td>888</td>
      <td>965</td>
      <td>9575</td>
      <td>9576</td>
      <td>2117</td>
      <td>2117</td>
      <td>3271</td>
      <td>5524</td>
      <td>1050</td>
      <td>1397</td>
      <td>17561</td>
      <td>20443</td>
      <td>4055</td>
      <td>4479</td>
    </tr>
    <tr>
      <td></td>
      <td>verkko</td>
      <td>verkko</td>
      <td>Telomere-to-telomere assembly pipeline</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/marbl/verkko</td>
      <td>Assembly</td>
      <td>verkko</td>
      <td>2023-01-24</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/verkko</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/verkko</td>
      <td>1.3.1</td>
      <td>verkko</td>
      <td>2.2.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>48</td>
      <td>48</td>
      <td>25</td>
      <td>25</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>48</td>
      <td>48</td>
      <td>25</td>
      <td>25</td>
    </tr>
    <tr>
      <td></td>
      <td>vg</td>
      <td>vg_convert, vg_deconstruct, vg_view</td>
      <td>Variation graph data structures, interchange formats, alignment, genotyping, and variant calling methods</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/vgteam/vg</td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td></td>
      <td>2020-04-08</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/vg</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/vg</td>
      <td>1.23.0</td>
      <td>vg</td>
      <td>1.59.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>478</td>
      <td>478</td>
      <td>17</td>
      <td>17</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>478</td>
      <td>478</td>
      <td>17</td>
      <td>17</td>
    </tr>
    <tr>
      <td></td>
      <td>virAnnot</td>
      <td>virannot_blast2tsv, virannot_otu, virAnnot_rps2tsv</td>
      <td>virAnnot wrappers</td>
      <td>virannot</td>
      <td>virannot</td>
      <td></td>
      <td>virAnnot</td>
      <td>VirAnnot was build to ease the assembly, blast search, taxonomic annotation and OTUs assignation of viral metagenomic HTS data. It is used in Virologie team of UMR1332 BFP laboratory at INRAE. VirAnnot also takes part of the Euphresco project "Plant Health Bioinformatics Network". See more.It is designed to identify viruses in plant metagenomic data but it can be used to assemble and annotate any sequences with the NCBI taxonomy.</td>
      <td>Sequence annotation, Sequence clustering, Sequence cluster visualisation</td>
      <td>Metagenomics, Virology, Microbial ecology</td>
      <td>To update</td>
      <td>https://github.com/marieBvr/virAnnot</td>
      <td>Metagenomics</td>
      <td>virannot</td>
      <td>2024-03-04</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/virAnnot</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/virAnnot</td>
      <td>1.1.0+galaxy0</td>
      <td>biopython</td>
      <td>1.70</td>
      <td>Sequence annotation, Sequence clustering, Sequence cluster visualisation</td>
      <td>Metagenomics, Virology, Microbial ecology</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>94</td>
      <td>94</td>
      <td>16</td>
      <td>16</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>94</td>
      <td>94</td>
      <td>16</td>
      <td>16</td>
    </tr>
    <tr>
      <td></td>
      <td>virheat</td>
      <td>virheat</td>
      <td>generates a heatmap of allele frequencies from vcf files</td>
      <td>virheat</td>
      <td>virheat</td>
      <td></td>
      <td>virHEAT</td>
      <td>VirHEAT tool generates multi-sample variant-frequency plots from SnpEff-annotated viral variant lists.  The tool provides a condensed look at variant frequencies after mapping  raw reads to a viral/bacterial reference genome and compares multiple vcf files at the same time.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/jonas-fuchs/virHEAT</td>
      <td>Visualization, Variant Analysis</td>
      <td>virheat</td>
      <td>2024-05-15</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/virheat</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/virheat</td>
      <td>0.7.1</td>
      <td>virheat</td>
      <td>0.7.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>12</td>
      <td>12</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>12</td>
      <td>12</td>
      <td>2</td>
      <td>2</td>
    </tr>
    <tr>
      <td></td>
      <td>virhunter</td>
      <td>virhunter</td>
      <td>Deep Learning method for novel virus detection in sequencing data</td>
      <td>virhunter</td>
      <td>virhunter</td>
      <td></td>
      <td>VirHunter</td>
      <td>VirHunter is a deep learning method that uses Convolutional Neural Networks (CNNs) and a Random Forest Classifier to identify viruses in sequencing datasets. More precisely, VirHunter classifies previously assembled contigs as viral, host, and bacterial (contamination).</td>
      <td>Sequence classification</td>
      <td>Virology</td>
      <td>To update</td>
      <td>https://github.com/cbib/virhunter</td>
      <td>Machine Learning</td>
      <td>virhunter</td>
      <td>2022-09-13</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/VirHunter</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/virhunter</td>
      <td>1.0.0</td>
      <td>numpy</td>
      <td></td>
      <td>Sequence classification</td>
      <td>Virology</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>574</td>
      <td>574</td>
      <td>76</td>
      <td>76</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>574</td>
      <td>574</td>
      <td>76</td>
      <td>76</td>
    </tr>
    <tr>
      <td></td>
      <td>volcanoplot</td>
      <td>volcanoplot</td>
      <td>Tool to create a Volcano Plot</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://ggplot2.tidyverse.org/</td>
      <td>Visualization, Transcriptomics, Statistics</td>
      <td>volcanoplot</td>
      <td>2018-10-14</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/volcanoplot</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/volcanoplot</td>
      <td>0.0.6</td>
      <td>r-ggplot2</td>
      <td>2.2.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>34917</td>
      <td>36852</td>
      <td>3670</td>
      <td>3786</td>
      <td>54931</td>
      <td>54931</td>
      <td>5597</td>
      <td>5597</td>
      <td>7183</td>
      <td>7183</td>
      <td>905</td>
      <td>905</td>
      <td>97031</td>
      <td>98966</td>
      <td>10172</td>
      <td>10288</td>
    </tr>
    <tr>
      <td></td>
      <td>vsearch</td>
      <td>vsearch_alignment, vsearch_chimera_detection, vsearch_clustering, vsearch_dereplication, vsearch_masking, vsearch_search, vsearch_shuffling, vsearch_sorting</td>
      <td>VSEARCH including searching, clustering, chimera detection, dereplication, sorting, masking and shuffling of sequences.</td>
      <td>vsearch</td>
      <td>vsearch</td>
      <td></td>
      <td>VSEARCH</td>
      <td>High-throughput search and clustering sequence analysis tool. It supports de novo and reference based chimera detection, clustering, full-length and prefix dereplication, reverse complementation, masking, all-vs-all pairwise global alignment, exact and global alignment searching, shuffling, subsampling and sorting. It also supports FASTQ file analysis, filtering and conversion.</td>
      <td>DNA mapping, Chimera detection</td>
      <td>Metagenomics, Sequence analysis</td>
      <td>To update</td>
      <td>https://github.com/torognes/vsearch</td>
      <td>Sequence Analysis</td>
      <td>vsearch</td>
      <td>2015-07-01</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/vsearch</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/vsearch</td>
      <td>2.8.3</td>
      <td>vsearch</td>
      <td>2.29.0</td>
      <td>DNA mapping, Chimera detection</td>
      <td>Metagenomics, Sequence analysis</td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>9694</td>
      <td>10338</td>
      <td>559</td>
      <td>628</td>
      <td>12282</td>
      <td>53546</td>
      <td>1031</td>
      <td>2007</td>
      <td>5334</td>
      <td>5672</td>
      <td>246</td>
      <td>277</td>
      <td>27310</td>
      <td>69556</td>
      <td>1836</td>
      <td>2912</td>
    </tr>
    <tr>
      <td></td>
      <td>vsnp</td>
      <td>vsnp_add_zero_coverage, vsnp_build_tables, vsnp_determine_ref_from_data, vsnp_get_snps, vsnp_statistics</td>
      <td>The vSNP tools are critical components of several workflows that validate SNPs and produce annotatedSNP tables and corresponding phylogenetic trees.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/USDA-VS/vSNP</td>
      <td>Sequence Analysis</td>
      <td>vsnp</td>
      <td>2020-04-27</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/vsnp</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/vsnp</td>
      <td>3.0.6</td>
      <td>pysam</td>
      <td>0.22.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>weather_app</td>
      <td>simple_weather</td>
      <td>provides simple weather in text format</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://wttr.in/</td>
      <td>Visualization, Web Services</td>
      <td>simpleweather</td>
      <td>2016-07-19</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/weather_app</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/weather_app</td>
      <td>0.1.2</td>
      <td>curl</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>weblogo3</td>
      <td>rgweblogo3</td>
      <td>Sequence Logo generator for fasta</td>
      <td>weblogo</td>
      <td>weblogo</td>
      <td></td>
      <td>WebLogo</td>
      <td>Web-based application designed to make generate sequence logos.</td>
      <td>Sequence cluster visualisation, Sequence visualisation, Sequence motif recognition</td>
      <td>Nucleic acid sites, features and motifs, Sequence analysis</td>
      <td>To update</td>
      <td></td>
      <td>Graphics</td>
      <td>weblogo3</td>
      <td>2017-11-17</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/weblogo3</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/weblogo3</td>
      <td>3.5.0</td>
      <td>weblogo</td>
      <td>3.7.9</td>
      <td>Sequence cluster visualisation, Sequence visualisation, Sequence motif recognition</td>
      <td>Nucleic acid sites, features and motifs, Sequence analysis</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2720</td>
      <td>2818</td>
      <td>206</td>
      <td>228</td>
      <td>10299</td>
      <td>30578</td>
      <td>1354</td>
      <td>3657</td>
      <td>1226</td>
      <td>1285</td>
      <td>50</td>
      <td>59</td>
      <td>14245</td>
      <td>34681</td>
      <td>1610</td>
      <td>3944</td>
    </tr>
    <tr>
      <td></td>
      <td>wigtobigwig</td>
      <td>ucsc_wigtobigwig</td>
      <td>converts bedGraph (wig) files into binary bigwig</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td></td>
      <td>UCSC Genome Browser Utilities</td>
      <td>Utilities for handling sequences and assemblies from the UCSC Genome Browser project.</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>Up-to-date</td>
      <td>https://genome.ucsc.edu/goldenPath/help/bigWig.html</td>
      <td>Convert Formats</td>
      <td>ucsc_wigtobigwig</td>
      <td>2024-01-14</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/wigtobigwig</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ucsc_tools/wigtobigwig</td>
      <td>469</td>
      <td>ucsc-wigtobigwig</td>
      <td>469</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1601</td>
      <td>1601</td>
      <td>106</td>
      <td>106</td>
      <td>1108</td>
      <td>1108</td>
      <td>172</td>
      <td>172</td>
      <td>200</td>
      <td>200</td>
      <td>15</td>
      <td>15</td>
      <td>2909</td>
      <td>2909</td>
      <td>293</td>
      <td>293</td>
    </tr>
    <tr>
      <td></td>
      <td>windowmasker</td>
      <td>windowmasker_mkcounts, windowmasker_ustat</td>
      <td>Identify repetitive regions using WindowMasker</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.ncbi.nlm.nih.gov/IEB/ToolBox/CPP_DOC/lxr/source/src/app/winmasker/</td>
      <td>Sequence Analysis</td>
      <td>windowmasker</td>
      <td>2023-12-14</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/windowmasker/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/windowmasker</td>
      <td>1.0</td>
      <td>blast</td>
      <td>2.16.0</td>
      <td></td>
      <td></td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>537</td>
      <td>537</td>
      <td>19</td>
      <td>19</td>
      <td>12</td>
      <td>12</td>
      <td>7</td>
      <td>7</td>
      <td>297</td>
      <td>297</td>
      <td>2</td>
      <td>2</td>
      <td>846</td>
      <td>846</td>
      <td>28</td>
      <td>28</td>
    </tr>
    <tr>
      <td></td>
      <td>winnowmap</td>
      <td>winnowmap</td>
      <td>A long-read mapping tool optimized for mapping ONT and PacBio reads to repetitive reference sequences.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/marbl/Winnowmap</td>
      <td>Next Gen Mappers</td>
      <td>winnowmap</td>
      <td>2021-04-29</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/winnowmap</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/winnowmap</td>
      <td>2.03</td>
      <td>winnowmap</td>
      <td>2.03</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>279</td>
      <td>279</td>
      <td>46</td>
      <td>46</td>
      <td>12</td>
      <td>12</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>291</td>
      <td>291</td>
      <td>50</td>
      <td>50</td>
    </tr>
    <tr>
      <td></td>
      <td>xpath</td>
      <td>xpath</td>
      <td>XPath XML querying tool</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://search.cpan.org/dist/XML-XPath/</td>
      <td>Text Manipulation</td>
      <td>xpath</td>
      <td>2015-04-14</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/xpath</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/xpath</td>
      <td></td>
      <td>perl-xml-xpath</td>
      <td>1.47</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>128</td>
      <td>235</td>
      <td>7</td>
      <td>8</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>128</td>
      <td>236</td>
      <td>7</td>
      <td>9</td>
    </tr>
    <tr>
      <td></td>
      <td>yahs</td>
      <td>yahs</td>
      <td>Yet Another Hi-C scaffolding tool</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/c-zhou/yahs</td>
      <td>Assembly</td>
      <td>yahs</td>
      <td>2022-06-28</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/yahs</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/yahs</td>
      <td>1.2a.2</td>
      <td>yahs</td>
      <td>1.2.2</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>618</td>
      <td>618</td>
      <td>176</td>
      <td>176</td>
      <td>336</td>
      <td>336</td>
      <td>62</td>
      <td>62</td>
      <td>588</td>
      <td>588</td>
      <td>25</td>
      <td>25</td>
      <td>1542</td>
      <td>1542</td>
      <td>263</td>
      <td>263</td>
    </tr>
    <tr>
      <td></td>
      <td>zerone</td>
      <td>zerone</td>
      <td>ChIP-seq discretization and quality control</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/nanakiksc/zerone</td>
      <td>ChIP-seq</td>
      <td>zerone</td>
      <td>2018-09-05</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/zerone</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/zerone</td>
      <td>1.0</td>
      <td>zerone</td>
      <td>1.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>83</td>
      <td>96</td>
      <td>9</td>
      <td>12</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>83</td>
      <td>96</td>
      <td>9</td>
      <td>12</td>
    </tr>
    <tr>
      <td></td>
      <td>EMLassemblyline</td>
      <td>annotations_template, eal_table_template, eal_templates, eml2eal, eml_validate, entities_template, geo_cov_template, makeeml, raster_template, taxo_cov_template, vector_template</td>
      <td>Tools using EML Assembly Line R package to generate EML metadata from template metadata files and vice versa</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/EDIorg/EMLassemblyline</td>
      <td>Ecology</td>
      <td>emlassemblyline</td>
      <td>2023-12-23</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/EMLassemblyline</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/EMLassemblyline</td>
      <td>0.1.0+galaxy0</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>11</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>11</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>516</td>
      <td>516</td>
      <td>128</td>
      <td>128</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>516</td>
      <td>516</td>
      <td>128</td>
      <td>128</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecoregionalization_workflow</td>
      <td>ecoregion_brt_analysis, ecoregion_GeoNearestNeighbor, ecoregion_cluster_estimate, ecoregion_clara_cluster, ecoregion_eco_map, ecoregion_taxa_seeker</td>
      <td>Tools to compute ecoregionalization with BRT model predictions and clustering.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/PaulineSGN/Workflow_Galaxy</td>
      <td>Ecology</td>
      <td>ecoregionalization</td>
      <td>2023-10-17</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/Ecoregionalization_workflow</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/Ecoregionalization_workflow</td>
      <td>0.1.0+galaxy0</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>352</td>
      <td>352</td>
      <td>36</td>
      <td>36</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>352</td>
      <td>352</td>
      <td>36</td>
      <td>36</td>
    </tr>
    <tr>
      <td></td>
      <td>Geom_mean_workflow</td>
      <td>Map_shp, Mean_geom, bar_plot</td>
      <td>Tools to compute The evolution of the total volume of very large trees, standing dead wood and dead wood on the ground on an area and the rate of devolution of the volume of wood favorable to biodiversity by large ecological regions (France).</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/PaulineSGN/Galaxy_tool_moyenne_geom</td>
      <td>Ecology</td>
      <td>Geometric means (Dead wood)</td>
      <td>2023-11-10</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/Geom_mean_workflow</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/Geom_mean_workflow</td>
      <td>0.1.0+galaxy0</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>53</td>
      <td>53</td>
      <td>18</td>
      <td>18</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>53</td>
      <td>53</td>
      <td>18</td>
      <td>18</td>
    </tr>
    <tr>
      <td></td>
      <td>PAMPA</td>
      <td>pampa_communitymetrics, pampa_presabs, pampa_glmcomm, pampa_glmsp, pampa_plotglm</td>
      <td>Tools to compute and analyse biodiversity metrics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Ecology</td>
      <td>pampa</td>
      <td>2020-11-13</td>
      <td>ecology</td>
      <td>https://github.com/ColineRoyaux/PAMPA-Galaxy</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/PAMPA</td>
      <td>0.0.2</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>4</td>
      <td>0</td>
      <td>1038</td>
      <td>1038</td>
      <td>146</td>
      <td>146</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>849</td>
      <td>849</td>
      <td>38</td>
      <td>38</td>
      <td>1887</td>
      <td>1887</td>
      <td>184</td>
      <td>184</td>
    </tr>
    <tr>
      <td></td>
      <td>ThermoRawFileParser</td>
      <td>thermo_raw_file_converter</td>
      <td>Thermo RAW file converter</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/compomics/ThermoRawFileParser</td>
      <td>Proteomics</td>
      <td>thermo_raw_file_converter</td>
      <td>2019-08-06</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/ThermoRawFileParser</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/ThermoRawFileParser</td>
      <td>1.3.4</td>
      <td>thermorawfileparser</td>
      <td>1.4.5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2467</td>
      <td>2959</td>
      <td>88</td>
      <td>124</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1097</td>
      <td>1097</td>
      <td>18</td>
      <td>18</td>
      <td>3564</td>
      <td>4056</td>
      <td>106</td>
      <td>142</td>
    </tr>
    <tr>
      <td></td>
      <td>ab1_fastq</td>
      <td>ab1_fastq_converter</td>
      <td>Tool to convert ab1 files into FASTQ files</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Convert Formats</td>
      <td>ab1fastq</td>
      <td>2021-10-08</td>
      <td>ecology</td>
      <td>https://github.com/ColineRoyaux/Galaxy_tool_projects/tree/main/ab1_fastq</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/ab1_fastq</td>
      <td>1.20.0</td>
      <td>bioconductor-sangerseqr</td>
      <td>1.38.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>49465</td>
      <td>49465</td>
      <td>281</td>
      <td>281</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>49465</td>
      <td>49465</td>
      <td>281</td>
      <td>281</td>
    </tr>
    <tr>
      <td></td>
      <td>ambertools</td>
      <td>ambertools_acpype, acpype_Amber2Gromacs, ambertools_antechamber, mmpbsa_mmgbsa, ambertools_parmchk2, parmconv, tleap</td>
      <td>Ambertools is a set of packages for preparing systems for molecular dynamics (MD) simulations and analyzing trajectories.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://ambermd.org/AmberTools.php</td>
      <td>Molecular Dynamics, Computational chemistry</td>
      <td>ambertools</td>
      <td>2020-04-03</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/buildtools/ambertools</td>
      <td>21.10</td>
      <td>ambertools</td>
      <td></td>
      <td></td>
      <td></td>
      <td>7</td>
      <td>2</td>
      <td>7</td>
      <td>0</td>
      <td>7</td>
      <td>2</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>7</td>
      <td>5</td>
      <td>0</td>
      <td>41793</td>
      <td>41881</td>
      <td>510</td>
      <td>511</td>
      <td>1126</td>
      <td>1126</td>
      <td>159</td>
      <td>159</td>
      <td>1552</td>
      <td>1552</td>
      <td>85</td>
      <td>85</td>
      <td>44471</td>
      <td>44559</td>
      <td>754</td>
      <td>755</td>
    </tr>
    <tr>
      <td></td>
      <td>appendfdr</td>
      <td>append_fdr</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td>appendfdr</td>
      <td>2015-01-24</td>
      <td>galaxyp</td>
      <td></td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/appendfdr</td>
      <td>0.2.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>aquainfra_importer</td>
      <td>aquainfra_importer</td>
      <td>A data source tool for downloading datasets via the AquaINFRA Interaction Platform.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/AquaINFRA/galaxy</td>
      <td>Ecology</td>
      <td></td>
      <td>2024-05-14</td>
      <td>ecology</td>
      <td>https://github.com/AquaINFRA/tools-ecology/tree/aquainfra_importer</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/aquainfra_importer</td>
      <td>1.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>75</td>
      <td>75</td>
      <td>9</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>75</td>
      <td>75</td>
      <td>9</td>
      <td>9</td>
    </tr>
    <tr>
      <td></td>
      <td>aquainfra_ogc_api_processes</td>
      <td>aquainfra_ogc_api_processes</td>
      <td>Wrapper for OGC API Processes developed in the AquaINFRA project.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/AquaINFRA/galaxy</td>
      <td>Ecology</td>
      <td></td>
      <td>2024-10-10</td>
      <td>ecology</td>
      <td>https://github.com/AquaINFRA/tools-ecology/tree/master</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/aquainfra_ogc_api_processes</td>
      <td>0.1.0</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>artbio_bam_cleaning</td>
      <td>artbio_bam_cleaning</td>
      <td>filter bam files before somatic-varscan or lumpy-smoove analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>SAM, Variant Analysis</td>
      <td>artbio_bam_cleaning</td>
      <td>2020-10-02</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/artbio_bam_cleaning</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/artbio_bam_cleaning</td>
      <td>1.10+galaxy0</td>
      <td>samtools</td>
      <td>1.21</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>astropytools</td>
      <td>astropy_fits2bitmap, astropy_fits2csv, astropy_fitsinfo</td>
      <td>AstropyTools library contains Galaxy tools for elementary Astrophysical operations</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/astropy/astropy</td>
      <td>Astronomy</td>
      <td>astropytools</td>
      <td>2023-04-11</td>
      <td>volodymyrss</td>
      <td>https://github.com/esg-epfl-apc/tools-astro/tree/main/tools/astropytools</td>
      <td>https://github.com/esg-epfl-apc/tools-astro/tree/main/tools/astropytools</td>
      <td>0.1.0+galaxy0</td>
      <td>astropy</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>13</td>
      <td>13</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>13</td>
      <td>13</td>
      <td>5</td>
      <td>5</td>
    </tr>
    <tr>
      <td></td>
      <td>bamparse</td>
      <td>bamparse</td>
      <td>Generates hit count lists from bam alignments.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, Transcriptomics</td>
      <td>bamparse</td>
      <td>2017-10-12</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/bamparse</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/bamparse</td>
      <td>4.1.1</td>
      <td>pysam</td>
      <td>0.22.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>baseline_calculator</td>
      <td>tt_baseline</td>
      <td>Toxicity prediction using QSAR models</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bernt-matthias/mb-galaxy-tools</td>
      <td>Ecology, Text Manipulation</td>
      <td>baseline_toxicity_calculator</td>
      <td>2024-03-25</td>
      <td>mbernt</td>
      <td>https://github.com/bernt-matthias/mb-galaxy-tools/tools/tox_tools/baseline_calculator</td>
      <td>https://github.com/Helmholtz-UFZ/galaxy-tools/tree/main/tools/tox_tools/baseline_calculator</td>
      <td>0.1.0+galaxy0</td>
      <td>pandas</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>bed_to_protein_map</td>
      <td>bed_to_protein_map</td>
      <td>Converts a BED file to a tabular list of exon locations</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>bed_to_protein_map</td>
      <td>2018-01-04</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/bed_to_protein_map</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/bed_to_protein_map</td>
      <td>0.2.0</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>410</td>
      <td>419</td>
      <td>134</td>
      <td>137</td>
      <td>363</td>
      <td>363</td>
      <td>99</td>
      <td>99</td>
      <td>158</td>
      <td>158</td>
      <td>13</td>
      <td>13</td>
      <td>931</td>
      <td>940</td>
      <td>246</td>
      <td>249</td>
    </tr>
    <tr>
      <td></td>
      <td>bigwig_to_bedgraph</td>
      <td>bigwig_to_bedgraph</td>
      <td>Converts a bigWig file to bedGraph format</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Convert Formats</td>
      <td>bigwig_to_bedgraph</td>
      <td>2021-10-06</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/bigwig_to_bedgraph</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/bigwig_to_bedgraph</td>
      <td>377+galaxy1</td>
      <td>ucsc-bigwigtobedgraph</td>
      <td>469</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>5769</td>
      <td>6802</td>
      <td>567</td>
      <td>650</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>1</td>
      <td>5769</td>
      <td>6805</td>
      <td>567</td>
      <td>651</td>
    </tr>
    <tr>
      <td></td>
      <td>bigwig_to_wig</td>
      <td>bigwig_to_wig</td>
      <td>Converts a bigWig file to Wiggle (WIG) format</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://artbio.fr</td>
      <td>Convert Formats</td>
      <td>bigwig_to_wig</td>
      <td>2018-09-25</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/bigwig_to_wig</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/bigwig_to_wig</td>
      <td>3+galaxy0</td>
      <td>ucsc-bigwiginfo</td>
      <td>469</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>bio3d</td>
      <td>bio3d_dccm, bio3d_pca, bio3d_rmsd, bio3d_rmsf, bio3d_pca_visualize</td>
      <td>Bio3d is a program that can be used to analyse molecular dynamics trajectories.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://thegrantlab.org/bio3d/index.php</td>
      <td>Computational chemistry</td>
      <td>bio3d</td>
      <td>2018-10-03</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/bio3d</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/bio3d</td>
      <td>2.4_1</td>
      <td>r-bio3d</td>
      <td>2.3_3</td>
      <td></td>
      <td></td>
      <td>5</td>
      <td>4</td>
      <td>5</td>
      <td>0</td>
      <td>5</td>
      <td>4</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>10600</td>
      <td>10729</td>
      <td>1150</td>
      <td>1159</td>
      <td>1042</td>
      <td>1042</td>
      <td>255</td>
      <td>255</td>
      <td>2756</td>
      <td>2756</td>
      <td>215</td>
      <td>215</td>
      <td>14398</td>
      <td>14527</td>
      <td>1620</td>
      <td>1629</td>
    </tr>
    <tr>
      <td></td>
      <td>biomoldyn</td>
      <td>biomd_neqgamma, fastpca, biomd_extract_clusters, biomd_rmsd_clustering</td>
      <td>Tools for MD analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/moldyn/</td>
      <td>Molecular Dynamics, Computational chemistry</td>
      <td>biomoldyn</td>
      <td>2020-01-30</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/biomoldyn</td>
      <td>1.5.2</td>
      <td>scipy</td>
      <td></td>
      <td></td>
      <td></td>
      <td>4</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>3</td>
      <td>0</td>
      <td>493</td>
      <td>493</td>
      <td>119</td>
      <td>119</td>
      <td>441</td>
      <td>441</td>
      <td>148</td>
      <td>148</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>934</td>
      <td>934</td>
      <td>267</td>
      <td>267</td>
    </tr>
    <tr>
      <td></td>
      <td>biotransformer</td>
      <td>biotransformer</td>
      <td>BioTransformer is a tool for prediction of small molecule metabolism in mammals.</td>
      <td>biotransformer</td>
      <td>biotransformer</td>
      <td></td>
      <td>BioTransformer</td>
      <td>BioTransformer is a freely available web server that supports accurate, rapid and comprehensive in silico metabolism prediction.</td>
      <td>Metabolic pathway prediction, PTM site prediction, Natural product identification</td>
      <td>Small molecules, Endocrinology and metabolism, Metabolomics, Carbohydrates, NMR</td>
      <td>Up-to-date</td>
      <td>https://bitbucket.org/djoumbou/biotransformerjar/src/master/</td>
      <td>Metabolomics</td>
      <td>biotransformer</td>
      <td>2020-12-11</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/biotransformer</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/biotransformer</td>
      <td>3.0.20230403</td>
      <td>biotransformer</td>
      <td>3.0.20230403</td>
      <td>Metabolic pathway prediction, PTM site prediction, Natural product identification</td>
      <td>Small molecules, Endocrinology and metabolism, Metabolomics, Carbohydrates, NMR</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>48</td>
      <td>48</td>
      <td>1</td>
      <td>1</td>
      <td>48</td>
      <td>48</td>
      <td>1</td>
      <td>1</td>
    </tr>
    <tr>
      <td></td>
      <td>blast2go</td>
      <td>blast2go</td>
      <td>Maps BLAST results to GO annotation terms</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/blast2go</td>
      <td>Ontology Manipulation, Sequence Analysis</td>
      <td>blast2go</td>
      <td>2015-12-08</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/blast2go</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/blast2go</td>
      <td>0.0.11</td>
      <td>b2g4pipe</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1063</td>
      <td>1237</td>
      <td>257</td>
      <td>300</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1063</td>
      <td>1237</td>
      <td>257</td>
      <td>300</td>
    </tr>
    <tr>
      <td></td>
      <td>blast_plus_remote_blastp</td>
      <td>blast_plus_remote_blastp</td>
      <td>NCBI BLAST+ with -remote option</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://blast.ncbi.nlm.nih.gov/</td>
      <td>Sequence Analysis</td>
      <td>blast_plus_remote_blastp</td>
      <td>2015-01-24</td>
      <td>galaxyp</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/ncbi_blast_plus</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/blast_plus_remote_blastp</td>
      <td>2.6.0</td>
      <td>blast</td>
      <td>2.16.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>blast_rbh</td>
      <td>blast_reciprocal_best_hits</td>
      <td>BLAST Reciprocal Best Hits (RBH) from two FASTA files</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/blast_rbh</td>
      <td>Fasta Manipulation, Sequence Analysis</td>
      <td>blast_rbh</td>
      <td>2016-12-05</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/blast_rbh</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/blast_rbh</td>
      <td>0.3.0</td>
      <td>biopython</td>
      <td>1.70</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>24916</td>
      <td>27120</td>
      <td>283</td>
      <td>347</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>24916</td>
      <td>27120</td>
      <td>283</td>
      <td>347</td>
    </tr>
    <tr>
      <td></td>
      <td>blast_to_scaffold</td>
      <td>blast2scaffold</td>
      <td>Generate DNA scaffold from blastn or tblastx alignments of Contigs</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, Sequence Analysis, Assembly</td>
      <td>blast_to_scaffold</td>
      <td>2016-01-05</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/blast_to_scaffold</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/blast_to_scaffold</td>
      <td>1.1.0</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>1</td>
      <td>1</td>
    </tr>
    <tr>
      <td></td>
      <td>blast_unmatched</td>
      <td>blast_unmatched</td>
      <td>Extract unmatched query sequences from blast</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Fasta Manipulation</td>
      <td>blast_unmatched</td>
      <td>2017-10-03</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/blast_unmatched</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/blast_unmatched</td>
      <td>1.0.1</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>blastparser_and_hits</td>
      <td>BlastParser_and_hits</td>
      <td>Parse blast outputs and compile hits</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Assembly, RNA</td>
      <td>blastparser_and_hits</td>
      <td>2017-10-15</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/blastparser_and_hits</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/blastparser_and_hits</td>
      <td>2.7.1</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>1</td>
      <td>1</td>
    </tr>
    <tr>
      <td></td>
      <td>blastx_to_scaffold</td>
      <td>blastx2scaffold</td>
      <td>Generate DNA scaffold from blastx alignment of Contigs</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, Sequence Analysis, Assembly</td>
      <td>blastx_to_scaffold</td>
      <td>2015-06-17</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/blastx_to_scaffold</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/blastx_to_scaffold</td>
      <td>1.1.1</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>blastxml_to_top_descr</td>
      <td>blastxml_to_top_descr</td>
      <td>Make table of top BLAST match descriptions</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/blastxml_to_top_descr</td>
      <td>Convert Formats, Sequence Analysis, Text Manipulation</td>
      <td>blastxml_to_top_descr</td>
      <td>2015-05-21</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/blastxml_to_top_descr</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/blastxml_to_top_descr</td>
      <td>0.1.2</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>264392</td>
      <td>265217</td>
      <td>483</td>
      <td>552</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>264392</td>
      <td>265217</td>
      <td>483</td>
      <td>552</td>
    </tr>
    <tr>
      <td></td>
      <td>bumbershoot</td>
      <td>idpqonvertEmbedder, idpassemble, idpqonvert, idpquery, myrimatch</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://proteowizard.sourceforge.net/</td>
      <td>Proteomics</td>
      <td></td>
      <td>2017-06-08</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tools/bumbershoot</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/bumbershoot</td>
      <td>3.0.21142</td>
      <td>bumbershoot</td>
      <td>3_0_21142_0e4f4a4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>1186</td>
      <td>1356</td>
      <td>21</td>
      <td>32</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1186</td>
      <td>1356</td>
      <td>21</td>
      <td>32</td>
    </tr>
    <tr>
      <td></td>
      <td>c3s</td>
      <td>c3s</td>
      <td>Copernicus Climate Change Service (C3S)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://cds.climate.copernicus.eu/cdsapp#!/search?type=dataset</td>
      <td>Climate Analysis</td>
      <td>c3s</td>
      <td>2021-04-13</td>
      <td>climate</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/c3s</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/c3s</td>
      <td>0.3.0</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>234</td>
      <td>234</td>
      <td>11</td>
      <td>11</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>92</td>
      <td>92</td>
      <td>3</td>
      <td>3</td>
      <td>330</td>
      <td>330</td>
      <td>18</td>
      <td>18</td>
    </tr>
    <tr>
      <td></td>
      <td>cads</td>
      <td>cads</td>
      <td>Copernicus Atmosphere Data Store (ADS)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://ads.atmosphere.copernicus.eu/#!/home</td>
      <td>Climate Analysis</td>
      <td>cads</td>
      <td>2021-06-19</td>
      <td>climate</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/cads</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/cads</td>
      <td>0.1.0</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>52</td>
      <td>52</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>52</td>
      <td>52</td>
      <td>1</td>
      <td>1</td>
    </tr>
    <tr>
      <td></td>
      <td>calisp</td>
      <td>calisp</td>
      <td>Calgary approach to isotopes in proteomics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/kinestetika/Calisp/</td>
      <td>Proteomics</td>
      <td>calisp</td>
      <td>2023-06-01</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tools/calisp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/calisp</td>
      <td>3.0.13</td>
      <td>calisp</td>
      <td>3.0.13</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
    </tr>
    <tr>
      <td></td>
      <td>cap3</td>
      <td>cap3</td>
      <td>cap3 wrapper</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Assembly</td>
      <td>cap3</td>
      <td>2017-09-02</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/cap3</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/cap3</td>
      <td>2.0.1</td>
      <td>cap3</td>
      <td>10.2011</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>9869</td>
      <td>9869</td>
      <td>158</td>
      <td>158</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6886</td>
      <td>6886</td>
      <td>146</td>
      <td>146</td>
      <td>16755</td>
      <td>16755</td>
      <td>304</td>
      <td>304</td>
    </tr>
    <tr>
      <td></td>
      <td>cardinal</td>
      <td>cardinal_classification, cardinal_colocalization, cardinal_combine, cardinal_data_exporter, cardinal_filtering, cardinal_mz_images, cardinal_preprocessing, cardinal_quality_report, cardinal_segmentations, cardinal_single_ion_segmentation, cardinal_spectra_plots</td>
      <td>Statistical and computational tools for analyzing mass spectrometry imaging datasets</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://cardinalmsi.org</td>
      <td>Proteomics, Metabolomics</td>
      <td></td>
      <td>2020-12-23</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/cardinal</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/cardinal</td>
      <td>3.4.3</td>
      <td>bioconductor-cardinal</td>
      <td>3.4.3</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>9</td>
      <td>11</td>
      <td>11</td>
      <td>0</td>
      <td>9</td>
      <td>11</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>11</td>
      <td>9</td>
      <td>0</td>
      <td>41875</td>
      <td>50702</td>
      <td>663</td>
      <td>713</td>
      <td>12</td>
      <td>12</td>
      <td>1</td>
      <td>1</td>
      <td>7512</td>
      <td>7512</td>
      <td>231</td>
      <td>231</td>
      <td>49399</td>
      <td>58226</td>
      <td>895</td>
      <td>945</td>
    </tr>
    <tr>
      <td></td>
      <td>cdo</td>
      <td>cdo_info, cdo_operations</td>
      <td>CDO (Climate Data Operators) is a collection of command line Operators to manipulate and analyse Climate and NWP model Data.Supported data formats are GRIB 1/2, netCDF 3/4, SERVICE, EXTRA and IEG. There are more than 600 operators available.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://code.mpimet.mpg.de/projects/cdo/</td>
      <td>Climate Analysis</td>
      <td></td>
      <td>2021-09-01</td>
      <td>climate</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/cdo</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/cdo</td>
      <td>2.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>218</td>
      <td>218</td>
      <td>17</td>
      <td>17</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>218</td>
      <td>218</td>
      <td>17</td>
      <td>17</td>
    </tr>
    <tr>
      <td></td>
      <td>cell-types-analysis</td>
      <td>ct_build_cell_ontology_dict, ct_check_labels, ct_combine_tool_outputs, ct_downsample_cells, ct_get_consensus_outputs, ct_get_empirical_dist, ct_get_tool_perf_table, ct_get_tool_pvals</td>
      <td>Tools for analysis of predictions from scRNAseq cell type classification tools, see https://github.com/ebi-gene-expression-group/cell-types-analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>suite_cell_types_analysis</td>
      <td>2020-04-07</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/cell-types-analysis</td>
      <td>1.1.1</td>
      <td>cell-types-analysis</td>
      <td>0.1.11</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>4</td>
      <td>4</td>
    </tr>
    <tr>
      <td></td>
      <td>cesm</td>
      <td>cesm</td>
      <td>Community Earth System Model (CESM)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://www.cesm.ucar.edu/</td>
      <td>Climate Analysis</td>
      <td>cesm</td>
      <td>2021-06-15</td>
      <td>climate</td>
      <td>https://github.com/ESCOMP/CESM</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/cesm</td>
      <td>2.1.3</td>
      <td>cesm</td>
      <td>2.1.3</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>14</td>
      <td>14</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>14</td>
      <td>14</td>
      <td>2</td>
      <td>2</td>
    </tr>
    <tr>
      <td></td>
      <td>champ_blocs</td>
      <td>cb_dissim, cb_ivr, cb_div</td>
      <td>Compute indicators for turnover boulders fields</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Ecology</td>
      <td></td>
      <td>2023-03-06</td>
      <td>ecology</td>
      <td>https://github.com/Marie59/champ_blocs</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/champ_blocs</td>
      <td>0.0.0</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>183</td>
      <td>183</td>
      <td>20</td>
      <td>20</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>183</td>
      <td>183</td>
      <td>20</td>
      <td>20</td>
    </tr>
    <tr>
      <td></td>
      <td>checkv</td>
      <td>checkv_end_to_end</td>
      <td>Assess quality of single-contig viral genomes</td>
      <td>checkv</td>
      <td>checkv</td>
      <td></td>
      <td>CheckV</td>
      <td>CheckV is a fully automated command-line pipeline for assessing the quality of single-contig viral genomes, including identification of host contamination for integrated proviruses, estimating completeness for genome fragments, and identification of closed genomes.</td>
      <td>Sequence assembly, Validation, Read mapping</td>
      <td>Metagenomics, Sequence assembly, Metagenomic sequencing, Mapping</td>
      <td>Up-to-date</td>
      <td>https://bitbucket.org/berkeleylab/checkv/</td>
      <td>Metagenomics</td>
      <td>checkv</td>
      <td>2024-09-13</td>
      <td>ufz</td>
      <td>https://github.com/Helmholtz-UFZ/ufz-galaxy-tools/blob/main/tools/checkv/</td>
      <td>https://github.com/Helmholtz-UFZ/galaxy-tools/tree/main/tools/checkv</td>
      <td>1.0.3</td>
      <td>checkv</td>
      <td>1.0.3</td>
      <td>Sequence assembly, Validation, Read mapping</td>
      <td>Metagenomics, Sequence assembly, Metagenomic sequencing, Mapping</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>cherry_pick_fasta</td>
      <td>cherry_pick_fasta</td>
      <td>Pick fasta sequence with specific header content</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Fasta Manipulation</td>
      <td>cherry_pick_fasta</td>
      <td>2015-06-17</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/cherry_pick_fasta</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/cherry_pick_fasta</td>
      <td>4.1</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>climate-stripes</td>
      <td>climate_stripes</td>
      <td>Create climate stripes from a tabular input file</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.climate-lab-book.ac.uk/2018/warming-stripes/</td>
      <td>Climate Analysis, Visualization</td>
      <td>climate_stripes</td>
      <td>2019-10-05</td>
      <td>climate</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/climate-stripes</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/climate-stripes</td>
      <td>1.0.2</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>239</td>
      <td>239</td>
      <td>35</td>
      <td>35</td>
      <td>16</td>
      <td>16</td>
      <td>2</td>
      <td>2</td>
      <td>182</td>
      <td>182</td>
      <td>3</td>
      <td>3</td>
      <td>437</td>
      <td>437</td>
      <td>40</td>
      <td>40</td>
    </tr>
    <tr>
      <td></td>
      <td>concat_multi_datasets</td>
      <td>cat_multi_datasets</td>
      <td>Concatenate multiple datasets tail-to-head, including collection datasets.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Text Manipulation</td>
      <td>concatenate_multiple_datasets</td>
      <td>2019-04-15</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/concat_multi_datasets</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/concat_multi_datasets</td>
      <td>1.4.3</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>4822</td>
      <td>4824</td>
      <td>613</td>
      <td>614</td>
      <td>1317</td>
      <td>1317</td>
      <td>204</td>
      <td>204</td>
      <td>3088</td>
      <td>3795</td>
      <td>252</td>
      <td>289</td>
      <td>9227</td>
      <td>9936</td>
      <td>1069</td>
      <td>1107</td>
    </tr>
    <tr>
      <td></td>
      <td>consensus_from_alignments</td>
      <td>aligned_to_consensus</td>
      <td>Tool to compute a consensus sequence from several aligned fasta sequences</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>consalign</td>
      <td>2023-04-11</td>
      <td>ecology</td>
      <td>https://github.com/ColineRoyaux/Galaxy_tool_projects/tree/main/consensus_from_alignments</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/consensus_from_alignments</td>
      <td>1.0.0</td>
      <td>r-bioseq</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1184</td>
      <td>1184</td>
      <td>154</td>
      <td>154</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1184</td>
      <td>1184</td>
      <td>154</td>
      <td>154</td>
    </tr>
    <tr>
      <td></td>
      <td>consolidate_vcfs</td>
      <td>consolidate_vcfs</td>
      <td>Combines freebayes and mpileup files for use by vcf2snvalignment</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://snvphyl.readthedocs.io/en/latest/</td>
      <td>Sequence Analysis</td>
      <td>consolidate_vcfs</td>
      <td>2015-11-26</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy/tree/development/tools/snvphyl-tools/consolidate_vcfs</td>
      <td>1.8.2</td>
      <td>snvphyl-tools</td>
      <td>1.8.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>cooler</td>
      <td>cooler_balance, cooler_cload_tabix, cooler_csort_tabix, cooler_makebins, cooler_zoomify</td>
      <td>cooler different tools to process Hi-C from mirnylab</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/open2c/cooler</td>
      <td>Epigenetics</td>
      <td>cooler</td>
      <td>2018-12-07</td>
      <td>lldelisle</td>
      <td>https://github.com/lldelisle/tools-lldelisle/blob/master/tools/cooler/.shed.yml</td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/cooler</td>
      <td>0.9.3</td>
      <td>htslib</td>
      <td>1.21</td>
      <td></td>
      <td></td>
      <td>4</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>170</td>
      <td>170</td>
      <td>26</td>
      <td>26</td>
      <td>195</td>
      <td>195</td>
      <td>39</td>
      <td>39</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>365</td>
      <td>365</td>
      <td>65</td>
      <td>65</td>
    </tr>
    <tr>
      <td></td>
      <td>cpm_tpm_rpk</td>
      <td>cpm_tpm_rpk</td>
      <td>Generate CPM,TPM or RPK from raw counts</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Transcriptomics</td>
      <td>cpm_tpm_rpk</td>
      <td>2018-07-13</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/cpm_tpm_rpk</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/cpm_tpm_rpk</td>
      <td>0.6.0</td>
      <td>r-optparse</td>
      <td>1.3.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>cta</td>
      <td>cta_astro_tool</td>
      <td>Basic simulation of CTA telescope observations using gammapy package</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Astronomy</td>
      <td>cta_astro_tool</td>
      <td>2024-04-19</td>
      <td>astroteam</td>
      <td>https://github.com/esg-epfl-apc/tools-astro/tree/main/tools</td>
      <td>https://github.com/esg-epfl-apc/tools-astro/tree/main/tools/cta</td>
      <td>0.0.1+galaxy0</td>
      <td>unzip</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>custom_pro_db</td>
      <td>custom_pro_db</td>
      <td>CustomProDB</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/customProDB.html</td>
      <td>Proteomics</td>
      <td>custom_pro_db</td>
      <td>2017-10-06</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tools/bumbershoot/custom_pro_db</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/probam_suite/custom_pro_db</td>
      <td>1.22.0</td>
      <td>bioconductor-rgalaxy</td>
      <td>1.37.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1729</td>
      <td>1744</td>
      <td>144</td>
      <td>146</td>
      <td>450</td>
      <td>450</td>
      <td>72</td>
      <td>72</td>
      <td>361</td>
      <td>361</td>
      <td>12</td>
      <td>12</td>
      <td>2540</td>
      <td>2555</td>
      <td>228</td>
      <td>230</td>
    </tr>
    <tr>
      <td></td>
      <td>custom_pro_db_annotation_data_manager</td>
      <td></td>
      <td>CustomProDB Annotation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/customProDB.html</td>
      <td>Proteomics</td>
      <td>custom_pro_db_annotation_data_manager</td>
      <td>2017-10-06</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tools/bumbershoot/custom_pro_db</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/probam_suite/custom_pro_db_annotation_data_manager</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>data-hca</td>
      <td>hca_matrix_downloader</td>
      <td>Tools for interacting with the Human Cell Atlas resource https://prod.data.humancellatlas.org/explore/projects</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, Sequence Analysis</td>
      <td>suite_human_cell_atlas_tools</td>
      <td>2019-07-02</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/data-hca</td>
      <td>v0.0.4+galaxy0</td>
      <td>hca-matrix-downloader</td>
      <td>0.0.4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>441</td>
      <td>451</td>
      <td>83</td>
      <td>87</td>
      <td>3</td>
      <td>3</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>444</td>
      <td>454</td>
      <td>85</td>
      <td>89</td>
    </tr>
    <tr>
      <td></td>
      <td>data-scxa</td>
      <td>retrieve_scxa</td>
      <td>Tools for interacting with the EMBL-EBI Expression Atlas resource https://www.ebi.ac.uk/gxa/home https://www.ebi.ac.uk/gxa/sc/home</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, Sequence Analysis</td>
      <td>suite_ebi_expression_atlas</td>
      <td>2019-07-02</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/data-scxa</td>
      <td>v0.0.2+galaxy2</td>
      <td>wget</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>867</td>
      <td>877</td>
      <td>213</td>
      <td>216</td>
      <td>339</td>
      <td>339</td>
      <td>94</td>
      <td>94</td>
      <td>64</td>
      <td>64</td>
      <td>5</td>
      <td>5</td>
      <td>1270</td>
      <td>1280</td>
      <td>312</td>
      <td>315</td>
    </tr>
    <tr>
      <td></td>
      <td>data_exploration</td>
      <td>tool_anonymization, ecology_homogeneity_normality, ecology_beta_diversity, ecology_link_between_var, ecology_presence_abs_abund, ecology_stat_presence_abs</td>
      <td>Explore data through multiple statistical tools</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Ecology</td>
      <td></td>
      <td>2021-07-27</td>
      <td>ecology</td>
      <td>https://github.com/Marie59/Data_explo_tools</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/data_exploration</td>
      <td>0.0.0</td>
      <td>r-tangles</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>967</td>
      <td>967</td>
      <td>308</td>
      <td>308</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>967</td>
      <td>967</td>
      <td>308</td>
      <td>308</td>
    </tr>
    <tr>
      <td></td>
      <td>data_manager_eggnog_mapper</td>
      <td></td>
      <td>downloads eggnog data for eggnog-mapper</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>data_manager_eggnog_mapper</td>
      <td>2019-11-11</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/eggnog_mapper/eggnog_mapper_data_manager</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/eggnog_mapper/data_manager_eggnog_mapper</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>data_manager_eggnog_mapper_abspath</td>
      <td></td>
      <td>download eggnog data for eggnog-mapper</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>data_manager_eggnog_mapper_abspath</td>
      <td>2019-11-11</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/eggnog_mapper/data_manager_eggnog_mapper_abspath</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/eggnog_mapper/data_manager_eggnog_mapper_abspath</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>dbbuilder</td>
      <td>dbbuilder</td>
      <td>Protein Database Downloader</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/dbbuilder</td>
      <td>Proteomics</td>
      <td>dbbuilder</td>
      <td>2015-01-24</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/dbbuilder</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/dbbuilder</td>
      <td>0.3.4</td>
      <td>wget</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>4813</td>
      <td>5555</td>
      <td>687</td>
      <td>805</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>961</td>
      <td>961</td>
      <td>87</td>
      <td>87</td>
      <td>5774</td>
      <td>6516</td>
      <td>774</td>
      <td>892</td>
    </tr>
    <tr>
      <td></td>
      <td>decoupler</td>
      <td>score_genes_aucell, decoupler_pathway_inference, decoupler_pseudobulk</td>
      <td>decoupler - Ensemble of methods to infer biological activities</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://decoupler-py.readthedocs.io/en/latest/</td>
      <td>Transcriptomics</td>
      <td>suite_decoupler</td>
      <td>2023-09-24</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/decoupler</td>
      <td>1.4.0+galaxy3</td>
      <td>decoupler</td>
      <td>1.5.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>54</td>
      <td>54</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>54</td>
      <td>54</td>
      <td>6</td>
      <td>6</td>
    </tr>
    <tr>
      <td></td>
      <td>decoyfasta</td>
      <td></td>
      <td>Galaxy tool wrapper for the transproteomic pipeline decoyFASTA tool.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>decoyfasta</td>
      <td>2015-01-24</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/decoyfasta</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/decoyfasta</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>deseq2_normalization</td>
      <td>deseq2_normalization</td>
      <td>Normalizes gene hitlists</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, Transcriptomics, Sequence Analysis, Statistics</td>
      <td>deseq2_normalization</td>
      <td>2018-01-05</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/deseq2_normalization</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/deseq2_normalization</td>
      <td>1.40.2+galaxy0</td>
      <td>bioconductor-deseq2</td>
      <td>1.42.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>dia_umpire</td>
      <td>dia_umpire_se</td>
      <td>DIA-Umpire analysis for data independent acquisition (DIA) mass spectrometry-based proteomics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://diaumpire.sourceforge.net/</td>
      <td>Proteomics</td>
      <td>dia_umpire</td>
      <td>2015-08-26</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/dia_umpire</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/dia_umpire</td>
      <td>2.1.3</td>
      <td>dia_umpire</td>
      <td>2.1.6</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>28</td>
      <td>33</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>134</td>
      <td>134</td>
      <td>4</td>
      <td>4</td>
      <td>162</td>
      <td>167</td>
      <td>8</td>
      <td>8</td>
    </tr>
    <tr>
      <td></td>
      <td>dialignr</td>
      <td>dialignr</td>
      <td>DIAlignR is an R package for retention time alignment of targeted mass spectrometric data, including DIA and SWATH-MS data. This tool works with MS2 chromatograms directly and uses dynamic programming for alignment of raw chromatographic traces. DIAlignR uses a hybrid approach of global (feature-based) and local (raw data-based) alignment to establish correspondence between peaks.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/shubham1637/DIAlignR</td>
      <td>Proteomics</td>
      <td>dialignr</td>
      <td>2020-12-23</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/dialignr</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/dialignr</td>
      <td>1.2.0</td>
      <td>bioconductor-dialignr</td>
      <td>2.10.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>40</td>
      <td>40</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>40</td>
      <td>40</td>
      <td>2</td>
      <td>2</td>
    </tr>
    <tr>
      <td></td>
      <td>diann</td>
      <td>diann</td>
      <td>DiaNN (DIA-based Neural Networks) is a software for DIA/SWATH data processing.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/vdemichev/DiaNN</td>
      <td>Proteomics</td>
      <td>diann</td>
      <td>2023-06-26</td>
      <td>galaxyp</td>
      <td>https://github.com/vdemichev/DiaNN</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/diann</td>
      <td>1.8.1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>72</td>
      <td>72</td>
      <td>14</td>
      <td>14</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>171</td>
      <td>171</td>
      <td>14</td>
      <td>14</td>
      <td>243</td>
      <td>243</td>
      <td>28</td>
      <td>28</td>
    </tr>
    <tr>
      <td></td>
      <td>diapysef</td>
      <td>diapysef</td>
      <td>diapysef is a convenience package for working with DIA-PASEF data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://pypi.org/project/diapysef/</td>
      <td>Proteomics</td>
      <td>diapysef</td>
      <td>2020-02-27</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/diapysef</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/diapysef</td>
      <td>0.3.5.0</td>
      <td>diapysef</td>
      <td>1.0.10</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>262</td>
      <td>262</td>
      <td>38</td>
      <td>38</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>256</td>
      <td>256</td>
      <td>6</td>
      <td>6</td>
      <td>518</td>
      <td>518</td>
      <td>44</td>
      <td>44</td>
    </tr>
    <tr>
      <td></td>
      <td>diffacto</td>
      <td>diffacto</td>
      <td>Diffacto comparative protein abundance estimation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/statisticalbiotechnology/diffacto</td>
      <td>Proteomics</td>
      <td>diffacto</td>
      <td>2021-06-20</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/diffacto</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/diffacto</td>
      <td>1.0.6</td>
      <td>diffacto</td>
      <td>1.0.7</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>11</td>
      <td>11</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>11</td>
      <td>11</td>
      <td>7</td>
      <td>7</td>
    </tr>
    <tr>
      <td></td>
      <td>digestdb</td>
      <td>digestdb</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td>digestdb</td>
      <td>2015-01-24</td>
      <td>galaxyp</td>
      <td></td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/digestdb</td>
      <td>0.1.0</td>
      <td>trans_proteomic_pipeline</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>directag_and_tagrecon</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td>directag_and_tagrecon</td>
      <td>2015-01-24</td>
      <td>galaxyp</td>
      <td></td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/directag_and_tagrecon</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>dose_responses</td>
      <td>dr_curve</td>
      <td>A  tool for analyzing and visualizing the relationship between various doses and their corresponding biological responses</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bernt-matthias/mb-galaxy-tools</td>
      <td>Ecology</td>
      <td>dose_response_analysis_tool</td>
      <td>2024-04-06</td>
      <td>ufz</td>
      <td>https://github.com/bernt-matthias/mb-galaxy-tools/tools/tox_tools/baseline_calculator</td>
      <td>https://github.com/Helmholtz-UFZ/galaxy-tools/tree/main/tools/tox_tools/dose_responses</td>
      <td>3.0.1</td>
      <td>r-drc</td>
      <td>3.0_1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>droplet-barcode-plot</td>
      <td>_dropletBarcodePlot</td>
      <td>Make a cell barcode plot for droplet single-cell RNA-seq QC</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary</td>
      <td>Sequence Analysis</td>
      <td>droplet_barcode_plot</td>
      <td>2019-11-07</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/develop/tools/droplet-rank-plot/.shed.yml</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/droplet-barcode-plot</td>
      <td>1.6.1+galaxy2</td>
      <td>scxa-plots</td>
      <td>0.0.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1151</td>
      <td>1151</td>
      <td>184</td>
      <td>184</td>
      <td>356</td>
      <td>356</td>
      <td>77</td>
      <td>77</td>
      <td>3</td>
      <td>3</td>
      <td>1</td>
      <td>1</td>
      <td>1510</td>
      <td>1510</td>
      <td>262</td>
      <td>262</td>
    </tr>
    <tr>
      <td></td>
      <td>dropletutils</td>
      <td>dropletutils_empty_drops, dropletutils_read_10x</td>
      <td>De-composed DropletUtils functionality tools, based on https://github.com/ebi-gene-expression-group/dropletutils-scripts and DropletUtils 1.0.3</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_dropletutils</td>
      <td>2019-01-22</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/dropletutils</td>
      <td>1.0.4</td>
      <td>dropletutils-scripts</td>
      <td>0.0.5</td>
      <td></td>
      <td></td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>1900</td>
      <td>1901</td>
      <td>412</td>
      <td>413</td>
      <td>609</td>
      <td>609</td>
      <td>131</td>
      <td>131</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2509</td>
      <td>2510</td>
      <td>543</td>
      <td>544</td>
    </tr>
    <tr>
      <td></td>
      <td>eggnog_mapper</td>
      <td>eggnog_mapper, eggnog_mapper_annotate, eggnog_mapper_search</td>
      <td>eggnog-mapper fast functional annotation of novel sequences</td>
      <td>eggnog-mapper-v2</td>
      <td>eggnog-mapper-v2</td>
      <td></td>
      <td>eggNOG-mapper v2</td>
      <td>EggNOG-mapper is a tool for fast functional annotation of novel sequences. It uses precomputed orthologous groups and phylogenies from the eggNOG database (http://eggnog5.embl.de) to transfer functional information from fine-grained orthologs only.</td>
      <td>Homology-based gene prediction, Genome annotation, Fold recognition, Information extraction, Query and retrieval</td>
      <td>Metagenomics, Phylogeny, Transcriptomics, Workflows, Sequence analysis</td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>eggnog_mapper</td>
      <td>2019-11-11</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/eggnog_mapper/eggnog_mapper</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/eggnog_mapper/eggnog_mapper</td>
      <td>2.1.8</td>
      <td>eggnog-mapper</td>
      <td>2.1.12</td>
      <td>Homology-based gene prediction, Genome annotation, Fold recognition, Information extraction, Query and retrieval</td>
      <td>Metagenomics, Phylogeny, Transcriptomics, Workflows, Sequence analysis</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>1</td>
      <td>3</td>
      <td>1</td>
      <td>0</td>
      <td>34432</td>
      <td>34437</td>
      <td>1149</td>
      <td>1152</td>
      <td>42</td>
      <td>42</td>
      <td>2</td>
      <td>2</td>
      <td>21027</td>
      <td>21027</td>
      <td>413</td>
      <td>413</td>
      <td>55501</td>
      <td>55506</td>
      <td>1564</td>
      <td>1567</td>
    </tr>
    <tr>
      <td></td>
      <td>embl2fa</td>
      <td>embl2fa</td>
      <td>Converts EMBL flat format to fasta format</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Text Manipulation</td>
      <td>embl2fa</td>
      <td>2022-11-12</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/embl2fa</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/embl2fa</td>
      <td>0.2</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>encyclopedia</td>
      <td>encyclopedia_encyclopedia, encyclopedia_fasta_to_prosit_csv, encyclopedia_library_to_blib, encyclopedia_prosit_csv_to_library, encyclopedia_quantify, encyclopedia_searchtolib, encyclopedia_walnut</td>
      <td>Mass Spec Data-Independent Acquisition (DIA) MS/MS analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/searleb/encyclopedia/wiki/Home</td>
      <td>Proteomics</td>
      <td>encyclopedia</td>
      <td>2020-09-11</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/encyclopedia/tools/encyclopedia</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/encyclopedia</td>
      <td>1.12.34</td>
      <td>encyclopedia</td>
      <td>2.12.30</td>
      <td></td>
      <td></td>
      <td>2</td>
      <td>4</td>
      <td>7</td>
      <td>0</td>
      <td>2</td>
      <td>4</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>738</td>
      <td>738</td>
      <td>98</td>
      <td>98</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>42</td>
      <td>42</td>
      <td>4</td>
      <td>4</td>
      <td>781</td>
      <td>781</td>
      <td>103</td>
      <td>103</td>
    </tr>
    <tr>
      <td></td>
      <td>eodie</td>
      <td>eodie</td>
      <td>Earth Observation Data Information Extractor</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://eodie.readthedocs.io/</td>
      <td>Climate Analysis</td>
      <td>eodie</td>
      <td>2021-12-30</td>
      <td>climate</td>
      <td>https://gitlab.com/eetun-tiimi/EODIE</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/eodie</td>
      <td>1.0.2</td>
      <td>eodie</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>45</td>
      <td>45</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>45</td>
      <td>45</td>
      <td>4</td>
      <td>4</td>
    </tr>
    <tr>
      <td></td>
      <td>essential_climate_variables</td>
      <td>cds_essential_variability</td>
      <td>Get Copernicus Essential Climate Variables for assessing climate variability</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://cds.climate.copernicus.eu/cdsapp#!/dataset/ecv-for-climate-change?tab=overview</td>
      <td>Climate Analysis, Data Source</td>
      <td>cds_essential_variability</td>
      <td>2019-05-03</td>
      <td>climate</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/essential_climate_variables</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/essential_climate_variables</td>
      <td>0.2.0</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>232</td>
      <td>336</td>
      <td>39</td>
      <td>40</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>232</td>
      <td>336</td>
      <td>39</td>
      <td>40</td>
    </tr>
    <tr>
      <td></td>
      <td>ez_histograms</td>
      <td>ez_histograms</td>
      <td>ggplot2 histograms and density plots</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/tidyverse/ggplot2</td>
      <td>Visualization, Statistics</td>
      <td>ez_histograms</td>
      <td>2024-02-07</td>
      <td>artbio</td>
      <td>https://github.com/artbio/tools-artbio/tree/main/tools/ez_histograms</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/ez_histograms</td>
      <td>3.4.4</td>
      <td>r-ggplot2</td>
      <td>2.2.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>fasta_merge_files_and_filter_unique_sequences</td>
      <td>fasta_merge_files_and_filter_unique_sequences</td>
      <td>Concatenate FASTA database files together</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/</td>
      <td>Fasta Manipulation</td>
      <td>fasta_merge_files_and_filter_unique_sequences</td>
      <td>2015-01-24</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/fasta_merge_files_and_filter_unique_sequences</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/fasta_merge_files_and_filter_unique_sequences</td>
      <td>1.2.0</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>30917</td>
      <td>32191</td>
      <td>1142</td>
      <td>1257</td>
      <td>36869</td>
      <td>36869</td>
      <td>1346</td>
      <td>1346</td>
      <td>5236</td>
      <td>5236</td>
      <td>230</td>
      <td>230</td>
      <td>73022</td>
      <td>74296</td>
      <td>2718</td>
      <td>2833</td>
    </tr>
    <tr>
      <td></td>
      <td>fastg2protlib</td>
      <td>fastg2protlib-peptides, fastg2protlib-validate</td>
      <td>Generate FASTA from FASTG</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproteomics/fastg2protlib.git</td>
      <td>Proteomics</td>
      <td>fastg2protlib</td>
      <td>2020-07-23</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/fastg2protlib</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/fastg2protlib</td>
      <td>1.0.2</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>29</td>
      <td>29</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>29</td>
      <td>29</td>
      <td>8</td>
      <td>8</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_pair</td>
      <td>fastq_pair</td>
      <td>Paired-end fastq pairer</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/linsalrob/fastq-pair</td>
      <td>Fastq Manipulation</td>
      <td>fastq_pair</td>
      <td>2022-03-17</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/develop/tools/qc/fastq_pair</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/qc/fastq_pair</td>
      <td>1.0+galaxy0</td>
      <td>fastq-pair</td>
      <td>1.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_provider</td>
      <td>fastq_provider</td>
      <td>Retrieval and download of FASTQ files from ENA and other repositories such as HCA.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ebi-gene-expression-group/atlas-fastq-provider</td>
      <td>Data Source, RNA, Transcriptomics</td>
      <td>atlas_fastq_provider</td>
      <td>2022-03-09</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/develop/tools/fastq_provider</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/fastq_provider</td>
      <td>0.4.4</td>
      <td>atlas-fastq-provider</td>
      <td>0.4.7</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_quality_trimmer</td>
      <td>cshl_fastq_quality_trimmer</td>
      <td>FASTQ trimmer based on quality</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/agordon/fastx_toolkit</td>
      <td>Fastq Manipulation</td>
      <td>fastq_quality_trimmer</td>
      <td>2022-03-11</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/develop/tools/qc/fastq_quality_trimmer</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/qc/fastq_quality_trimmer</td>
      <td>0.0.14+galaxy0</td>
      <td>fastx_toolkit</td>
      <td>0.0.14</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_utils</td>
      <td>fastq_filter_n, fastq_trim_poly_at</td>
      <td>Set of tools for handling fastq files</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/nunofonseca/fastq_utils</td>
      <td>Transcriptomics, RNA</td>
      <td>fastq_utils</td>
      <td>2022-02-15</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/develop/tools/qc/fastq_utils</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/qc/fastq_utils</td>
      <td>0.25.1+galaxy0</td>
      <td>fastq_utils</td>
      <td>0.25.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>fates-emerald</td>
      <td>ctsm_fates</td>
      <td>EMERALD version of the Functionally Assembled Terrestrial Ecosystem Simulator (FATES) with Community Terrestrial Systems Model as host model</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/NordicESMhub/ctsm/blob/fates_emerald_api/README_fates_emerald_api</td>
      <td>Climate Analysis</td>
      <td>ctsm_fates</td>
      <td>2020-10-21</td>
      <td>climate</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/fates-emerald</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/fates-emerald</td>
      <td>2.0</td>
      <td>fates-emerald</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>264</td>
      <td>264</td>
      <td>69</td>
      <td>69</td>
      <td>34</td>
      <td>34</td>
      <td>11</td>
      <td>11</td>
      <td>103</td>
      <td>103</td>
      <td>1</td>
      <td>1</td>
      <td>401</td>
      <td>401</td>
      <td>81</td>
      <td>81</td>
    </tr>
    <tr>
      <td></td>
      <td>feature_alignment</td>
      <td>feature_alignment</td>
      <td>TRIC integrates information from all available runs via a graph-based alignment strategy</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td></td>
      <td>Proteomics</td>
      <td>feature_alignment</td>
      <td>2021-11-22</td>
      <td>galaxyp</td>
      <td>https://github.com/msproteomicstools/msproteomicstools/blob/master/TRIC-README.md</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/feature_alignment</td>
      <td>0.11.0</td>
      <td>msproteomicstools</td>
      <td>0.11.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>18</td>
      <td>18</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>18</td>
      <td>18</td>
      <td>1</td>
      <td>1</td>
    </tr>
    <tr>
      <td></td>
      <td>fetch_fasta_from_ncbi</td>
      <td>retrieve_fasta_from_NCBI</td>
      <td>Fetch fasta sequences from NCBI using eutils wrappers</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Fasta Manipulation, Data Source</td>
      <td>fetch_fasta_from_ncbi</td>
      <td>2017-11-21</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/fetch_fasta_from_ncbi</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/fetch_fasta_from_ncbi</td>
      <td>3.1.0</td>
      <td>urllib3</td>
      <td>1.12</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
    </tr>
    <tr>
      <td></td>
      <td>filter_by_fasta_ids</td>
      <td>filter_by_fasta_ids</td>
      <td>Filter FASTA on the headers and/or the sequences</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Fasta Manipulation, Proteomics</td>
      <td>filter_by_fasta_ids</td>
      <td>2015-01-24</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/filter_by_fasta_ids</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/filter_by_fasta_ids</td>
      <td>2.3</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>26717</td>
      <td>29932</td>
      <td>1026</td>
      <td>1072</td>
      <td>42990</td>
      <td>42990</td>
      <td>1176</td>
      <td>1176</td>
      <td>9619</td>
      <td>9619</td>
      <td>230</td>
      <td>230</td>
      <td>79326</td>
      <td>82541</td>
      <td>2432</td>
      <td>2478</td>
    </tr>
    <tr>
      <td></td>
      <td>filter_compounds</td>
      <td>filter_orgmet_anorg</td>
      <td>Tool for filtering organometallics/anorganic compounds from a list of compounds.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/RECETOX/galaxytools/</td>
      <td>Metabolomics</td>
      <td>filter_compounds</td>
      <td>2021-01-06</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/filter_compounds</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/filter_compounds</td>
      <td>3.1.1</td>
      <td>openbabel</td>
      <td>2.3.90dev7d621d9</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>filter_density</td>
      <td>filterdensity</td>
      <td>Filter out position based on distance between SNVs</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://snvphyl.readthedocs.io/en/latest/</td>
      <td>Sequence Analysis</td>
      <td>filter_density</td>
      <td>2015-11-23</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy/tree/development/tools/snvphyl-tools/filter_density</td>
      <td>1.8.2</td>
      <td>snvphyl-tools</td>
      <td>1.8.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>filter_stats</td>
      <td>filterstat</td>
      <td>SNVPhyl filter_stats</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://snvphyl.readthedocs.io/en/latest/</td>
      <td>Sequence Analysis</td>
      <td>filter_stats</td>
      <td>2015-08-14</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy/tree/development/tools/snvphyl-tools/filter_stats</td>
      <td>1.8.2</td>
      <td>snvphyl-tools</td>
      <td>1.8.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>filter_vcf</td>
      <td>filtervcf</td>
      <td>SNVPhyl filter_vcf</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://snvphyl.readthedocs.io/en/latest/</td>
      <td>Sequence Analysis</td>
      <td>filter_vcf</td>
      <td>2015-08-04</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy/tree/development/tools/snvphyl-tools/filter_vcf</td>
      <td>1.8.2</td>
      <td>snvphyl-tools</td>
      <td>1.8.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>find_repeats</td>
      <td>findrepeat</td>
      <td>Find repetitive regions on a reference genome using MUMMer</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://snvphyl.readthedocs.io/en/latest/</td>
      <td>Sequence Analysis</td>
      <td>find_repeats</td>
      <td>2015-08-14</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy/tree/development/tools/snvphyl-tools/find_repeats</td>
      <td>1.8.2</td>
      <td>snvphyl-tools</td>
      <td>1.8.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>fisher_test</td>
      <td>fishertest</td>
      <td>Fisher's exact test on two-column hit lists.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, Statistics</td>
      <td>fishertest</td>
      <td>2015-06-17</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/fisher_test</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/fisher_test</td>
      <td>2.32.0+galaxy0</td>
      <td>bioconductor-qvalue</td>
      <td>2.34.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>flashlfq</td>
      <td>flashlfq</td>
      <td>FlashLFQ mass-spectrometry proteomics label-free quantification</td>
      <td>flashlfq</td>
      <td>flashlfq</td>
      <td></td>
      <td>FlashLFQ</td>
      <td>FlashLFQ is an ultrafast label-free quantification algorithm for mass-spectrometry proteomics.</td>
      <td>Label-free quantification</td>
      <td>Proteomics experiment, Proteomics</td>
      <td>To update</td>
      <td>https://github.com/smith-chem-wisc/FlashLFQ</td>
      <td>Proteomics</td>
      <td>flashlfq</td>
      <td>2017-12-06</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/flashlfq</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/flashlfq</td>
      <td>1.0.3.1</td>
      <td>flashlfq</td>
      <td>1.2.6</td>
      <td>Label-free quantification</td>
      <td>Proteomics experiment, Proteomics</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>661</td>
      <td>715</td>
      <td>52</td>
      <td>54</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>167</td>
      <td>167</td>
      <td>15</td>
      <td>15</td>
      <td>828</td>
      <td>882</td>
      <td>67</td>
      <td>69</td>
    </tr>
    <tr>
      <td></td>
      <td>fragpipe</td>
      <td>fragpipe</td>
      <td>Data analysis for mass spectrometry-based proteomics.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://fragpipe.nesvilab.org/</td>
      <td>Proteomics</td>
      <td>fragpipe</td>
      <td>2023-05-10</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/fragpipe</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/fragpipe</td>
      <td>20.0</td>
      <td>fragpipe</td>
      <td>20.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>64</td>
      <td>64</td>
      <td>9</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>64</td>
      <td>64</td>
      <td>9</td>
      <td>9</td>
    </tr>
    <tr>
      <td></td>
      <td>free_energy</td>
      <td></td>
      <td>Free energy tools of BRIDGE.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Molecular Dynamics, Computational chemistry</td>
      <td>freeenergy</td>
      <td>2019-11-11</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tools/free_energy</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/free_energy</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>frogs</td>
      <td>FROGS_affiliation_filters, FROGS_affiliation_postprocess, FROGS_affiliation_stats, FROGS_biom_to_stdBiom, FROGS_biom_to_tsv, FROGS_cluster_filters, FROGS_cluster_stats, FROGS_clustering, FROGS_demultiplex, FROGSSTAT_DESeq2_Preprocess, FROGSSTAT_DESeq2_Visualisation, FROGSFUNC_step2_functions, FROGSFUNC_step3_pathways, FROGSFUNC_step1_placeseqs, FROGS_itsx, FROGS_normalisation, FROGSSTAT_Phyloseq_Alpha_Diversity, FROGSSTAT_Phyloseq_Beta_Diversity, FROGSSTAT_Phyloseq_Sample_Clustering, FROGSSTAT_Phyloseq_Composition_Visualisation, FROGSSTAT_Phyloseq_Import_Data, FROGSSTAT_Phyloseq_Multivariate_Analysis_Of_Variance, FROGSSTAT_Phyloseq_Structure_Visualisation, FROGS_preprocess, FROGS_remove_chimera, FROGS_taxonomic_affiliation, FROGS_Tree, FROGS_tsv_to_biom</td>
      <td>Suite for metabarcoding analysis</td>
      <td>frogs</td>
      <td>frogs</td>
      <td></td>
      <td>FROGS</td>
      <td>The user-friendly and Galaxy-supported pipeline FROGS analyses large sets of DNA amplicons sequences accurately and rapidly, essential for microbe community studies.</td>
      <td>Taxonomic classification</td>
      <td>Metagenomics, Microbial ecology, Taxonomy, Evolutionary biology, Sequencing</td>
      <td>To update</td>
      <td>http://frogs.toulouse.inrae.fr/</td>
      <td>Metagenomics</td>
      <td>frogs</td>
      <td>2023-04-26</td>
      <td>frogs</td>
      <td>https://github.com/geraldinepascal/FROGS-wrappers/</td>
      <td>https://github.com/geraldinepascal/FROGS-wrappers/tree/master/tools/frogs</td>
      <td>4.1.0</td>
      <td>frogs</td>
      <td>5.0.0</td>
      <td>Taxonomic classification</td>
      <td>Metagenomics, Microbial ecology, Taxonomy, Evolutionary biology, Sequencing</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>28</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>28</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>18</td>
      <td>18</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>fromHicupToJuicebox</td>
      <td>fromHicupToJuicebox</td>
      <td>Convert the output of hicup (as sam or bam) to the input of juicebox.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Epigenetics</td>
      <td>from_hicup_to_juicebox</td>
      <td>2018-06-05</td>
      <td>lldelisle</td>
      <td></td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/fromHicupToJuicebox</td>
      <td>0.0.2</td>
      <td>pysam</td>
      <td>0.22.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>fromgtfTobed12</td>
      <td>fromgtfTobed12</td>
      <td>Convert GTF files to BED12 format</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://pythonhosted.org/gffutils/contents.html</td>
      <td>Convert Formats</td>
      <td>fromgtftobed12</td>
      <td>2019-07-31</td>
      <td>lldelisle</td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/fromgtfTobed12</td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/fromgtfTobed12</td>
      <td>0.11.1+galaxy1</td>
      <td>gffutils</td>
      <td>0.13</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>garnett</td>
      <td>garnett_check_markers, garnett_classify_cells, garnett_get_feature_genes, garnett_get_std_output, garnett_train_classifier, garnett_transform_markers, update_marker_file</td>
      <td>De-composed Garnett functionality tools, see https://github.com/ebi-gene-expression-group/garnett-cli and r-garnett 0.2.8</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_garnett</td>
      <td>2020-04-03</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/garnett</td>
      <td>0.2.8</td>
      <td>garnett-cli</td>
      <td>0.0.5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>27</td>
      <td>27</td>
      <td>11</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>27</td>
      <td>27</td>
      <td>11</td>
      <td>11</td>
    </tr>
    <tr>
      <td></td>
      <td>gatk4</td>
      <td>filtermutectcalls, mergemutectstats, mutect2</td>
      <td>Find somatic variations</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Variant Analysis</td>
      <td>gatk4</td>
      <td>2021-12-29</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gatk4</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gatk4</td>
      <td>4.1.7.0</td>
      <td>gatk4</td>
      <td>4.5.0.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>gc_derivatization</td>
      <td>gc_derivatization</td>
      <td>In silico derivatization for GC.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/RECETOX/gc-meox-tms</td>
      <td>Metabolomics</td>
      <td>gc_derivatization</td>
      <td>2024-03-15</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/gc_derivatization</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/gc_derivatization</td>
      <td>1.0.1</td>
      <td>gc-meox-tms</td>
      <td>1.0.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>gdal</td>
      <td>gdal_gdal_merge, gdal_gdal_translate, gdal_gdaladdo, gdal_gdalbuildvrt, gdal_gdalinfo, gdal_gdalwarp, gdal_ogr2ogr, gdal_ogrinfo</td>
      <td>Geospatial Data Abstraction Library tools are all dedicated to manipulate raster and vector geospatial data formats.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.gdal.org</td>
      <td>Ecology</td>
      <td>gdal</td>
      <td>2019-02-25</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/gdal</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/gdal</td>
      <td>3.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>1409</td>
      <td>1498</td>
      <td>39</td>
      <td>42</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1409</td>
      <td>1498</td>
      <td>39</td>
      <td>42</td>
    </tr>
    <tr>
      <td></td>
      <td>genomad</td>
      <td>genomad_end_to_end</td>
      <td>Identify virus and plasmid genomes from nucleotide sequences</td>
      <td>genomad</td>
      <td>genomad</td>
      <td></td>
      <td>geNomad</td>
      <td>geNomad is a tool that identifies virus and plasmid genomes from nucleotide sequences. It provides state-of-the-art classification performance and can be used to quickly find mobile genetic elements from genomes, metagenomes, or metatranscriptomes.</td>
      <td>Sequence annotation, Taxonomic classification</td>
      <td>Sequence analysis</td>
      <td>To update</td>
      <td>https://github.com/apcamargo/genomad/</td>
      <td>Metagenomics</td>
      <td>genomad</td>
      <td>2024-06-17</td>
      <td>ufz</td>
      <td>https://github.com/Helmholtz-UFZ/ufz-galaxy-tools/blob/main/tools/genomad/</td>
      <td>https://github.com/Helmholtz-UFZ/galaxy-tools/tree/main/tools/genomad</td>
      <td>1.8.0</td>
      <td>genomad</td>
      <td>1.8.1</td>
      <td>Sequence annotation, Taxonomic classification</td>
      <td>Sequence analysis</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>getTn5ExtendedCoverage</td>
      <td>getTn5ExtendedCoverage</td>
      <td>Take an input bam from ATAC-seq and generate a bedgraph using the center of the Tn5 insertion with an extension</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Epigenetics</td>
      <td>gettn5extendedcoverage</td>
      <td>2018-06-13</td>
      <td>lldelisle</td>
      <td></td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/getTn5ExtendedCoverage</td>
      <td>0.0.2</td>
      <td>pysam</td>
      <td>0.22.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>get_reference_fasta</td>
      <td>get_fasta_reference</td>
      <td>Obtain reference genome sequence.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Data Source, Fasta Manipulation</td>
      <td>get_reference_fasta</td>
      <td>2015-06-17</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/get_reference_fasta</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/get_reference_fasta</td>
      <td>0.3.2</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>gffcompare_to_bed</td>
      <td>gffcompare_to_bed</td>
      <td>Filter and convert a gffCompare GTF to BED</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/gpertea/gffcompare/</td>
      <td>Convert Formats</td>
      <td>gffcompare_to_bed</td>
      <td>2018-01-10</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/gffcompare_to_bed</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/gffcompare_to_bed</td>
      <td>0.2.1</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1375</td>
      <td>2230</td>
      <td>329</td>
      <td>347</td>
      <td>1071</td>
      <td>1071</td>
      <td>362</td>
      <td>362</td>
      <td>492</td>
      <td>492</td>
      <td>64</td>
      <td>64</td>
      <td>2938</td>
      <td>3793</td>
      <td>755</td>
      <td>773</td>
    </tr>
    <tr>
      <td></td>
      <td>gromacs</td>
      <td>gmx_check, gmx_editconf, gmx_energy, gmx_get_builtin_file, gmx_rg, gmx_makendx, gmx_merge_topology_files, gmx_em, gmx_restraints, gmx_rmsd, gmx_rmsf, gmx_setup, gmx_sim, gmx_solvate, gmx_trj</td>
      <td>GROMACS is a package for performing molecular dynamics, primarily designed for biochemical molecules such as proteins, lipids and nucleic acids.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/gromacs</td>
      <td>Molecular Dynamics, Computational chemistry</td>
      <td>gromacs</td>
      <td>2022-03-22</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/gromacs</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/gromacs</td>
      <td>2022</td>
      <td>gromacs</td>
      <td>2021.3</td>
      <td></td>
      <td></td>
      <td>14</td>
      <td>8</td>
      <td>15</td>
      <td>0</td>
      <td>14</td>
      <td>8</td>
      <td>15</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>13</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>15</td>
      <td>9</td>
      <td>0</td>
      <td>184067</td>
      <td>184203</td>
      <td>3441</td>
      <td>3451</td>
      <td>9382</td>
      <td>9382</td>
      <td>1438</td>
      <td>1438</td>
      <td>10325</td>
      <td>10325</td>
      <td>648</td>
      <td>648</td>
      <td>203774</td>
      <td>203910</td>
      <td>5527</td>
      <td>5537</td>
    </tr>
    <tr>
      <td></td>
      <td>gsc_center_scale</td>
      <td>center_scale</td>
      <td>Center or scale (standardize) data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Statistics</td>
      <td>gsc_center_scale</td>
      <td>2019-07-09</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_center_scale</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_center_scale</td>
      <td>4.3.1+galaxy0</td>
      <td>r-optparse</td>
      <td>1.3.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>gsc_filter_cells</td>
      <td>filter_cells</td>
      <td>Filter single cell RNAseq data on libray depth and number of detected genes</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Transcriptomics</td>
      <td>gsc_filter_cells</td>
      <td>2019-06-23</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_filter_cells</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_filter_cells</td>
      <td>4.3.1+galaxy0</td>
      <td>r-optparse</td>
      <td>1.3.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>gsc_filter_genes</td>
      <td>filter_genes</td>
      <td>Filter genes that are detected in less than a fraction of libraries in single cell RNAseq data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Transcriptomics</td>
      <td>gsc_filter_genes</td>
      <td>2019-06-23</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_filter_genes</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_filter_genes</td>
      <td>4.3.1+galaxy0</td>
      <td>r-optparse</td>
      <td>1.3.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>gsc_gene_expression_correlations</td>
      <td>single_cell_gene_expression_correlations</td>
      <td>Compute single-cell paire-wise gene expressions correlations</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Transcriptomics</td>
      <td>gsc_gene_expression_correlations</td>
      <td>2019-06-24</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/gsc_gene_expression_correlations</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_gene_expression_correlations</td>
      <td>4.3.1+galaxy0</td>
      <td>r-optparse</td>
      <td>1.3.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>gsc_high_dimensions_visualisation</td>
      <td>high_dimensions_visualisation</td>
      <td>Generates PCA, t-SNE and HCPC visualisation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Transcriptomics, Visualization</td>
      <td>gsc_high_dimensions_visualisation</td>
      <td>2019-07-09</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_high_dimension_visualization</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_high_dimensions_visualisation</td>
      <td>4.3+galaxy0</td>
      <td>r-optparse</td>
      <td>1.3.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>gsc_mannwhitney_de</td>
      <td>mannwhitney_de</td>
      <td>Perform a mann-whitney differential testing between two sets of gene expression data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Transcriptomics</td>
      <td>gsc_mannwhitney_de</td>
      <td>2019-06-23</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_mannwhitney_de</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_mannwhitney_de</td>
      <td>4.1.3+galaxy0</td>
      <td>r-optparse</td>
      <td>1.3.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>gsc_scran_normalize</td>
      <td>scran_normalize</td>
      <td>Normalize raw counts using scran</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Transcriptomics</td>
      <td>gsc_scran_normalize</td>
      <td>2019-09-23</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_scran_normalize</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_scran_normalize</td>
      <td>1.28.1+galaxy0</td>
      <td>bioconductor-scran</td>
      <td>1.30.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>89</td>
      <td>89</td>
      <td>24</td>
      <td>24</td>
      <td>3</td>
      <td>3</td>
      <td>2</td>
      <td>2</td>
      <td>10</td>
      <td>10</td>
      <td>2</td>
      <td>2</td>
      <td>102</td>
      <td>102</td>
      <td>28</td>
      <td>28</td>
    </tr>
    <tr>
      <td></td>
      <td>gsc_signature_score</td>
      <td>signature_score</td>
      <td>Compute signature scores from single cell RNAseq data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Transcriptomics</td>
      <td>gsc_signature_score</td>
      <td>2019-06-23</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_signature_score</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_signature_score</td>
      <td>2.3.9+galaxy0</td>
      <td>r-optparse</td>
      <td>1.3.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>gtf-2-gene-list</td>
      <td>_ensembl_gtf2gene_list</td>
      <td>Utility to extract annotations from Ensembl GTF files.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary</td>
      <td>Sequence Analysis</td>
      <td>gtf2gene_list</td>
      <td>2019-11-07</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/develop/tools/gtf-2-gene-list/.shed.yml</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/gtf-2-gene-list</td>
      <td>1.52.0+galaxy0</td>
      <td>atlas-gene-annotation-manipulation</td>
      <td>1.1.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1982</td>
      <td>1983</td>
      <td>323</td>
      <td>324</td>
      <td>1040</td>
      <td>1040</td>
      <td>343</td>
      <td>343</td>
      <td>205</td>
      <td>205</td>
      <td>38</td>
      <td>38</td>
      <td>3227</td>
      <td>3228</td>
      <td>704</td>
      <td>705</td>
    </tr>
    <tr>
      <td></td>
      <td>guppy</td>
      <td>guppy-basecaller</td>
      <td>A wrapper for the guppy basecaller tool from Oxford Nanopore Technologies</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Nanopore</td>
      <td>guppy_basecaller</td>
      <td>2020-11-18</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/guppy</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/guppy</td>
      <td>6.5.7+galaxy0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>1</td>
      <td>1</td>
      <td>7</td>
      <td>7</td>
      <td>1</td>
      <td>1</td>
    </tr>
    <tr>
      <td></td>
      <td>hardklor</td>
      <td>hardklor, kronik</td>
      <td>Hardklr</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>hardklor</td>
      <td>2016-04-26</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tools/hardklor</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/hardklor</td>
      <td>2.30.1+galaxy1</td>
      <td>hardklor</td>
      <td>2.3.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>98</td>
      <td>111</td>
      <td>5</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>98</td>
      <td>111</td>
      <td>5</td>
      <td>8</td>
    </tr>
    <tr>
      <td></td>
      <td>hess</td>
      <td>hess_astro_tool</td>
      <td>Basic analysis of Data Level 3 public data sample of HESS gamma-ray telescope</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Astronomy</td>
      <td>hess_astro_tool</td>
      <td>2024-02-19</td>
      <td>astroteam</td>
      <td>https://github.com/esg-epfl-apc/tools-astro/tree/main/tools</td>
      <td>https://github.com/esg-epfl-apc/tools-astro/tree/main/tools/hess</td>
      <td>0.0.2+galaxy0</td>
      <td>ipython</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
    </tr>
    <tr>
      <td></td>
      <td>high_dim_heatmap</td>
      <td>high_dim_heatmap</td>
      <td>gplot heatmap.2 function adapted for plotting large heatmaps</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/cran/gplots</td>
      <td>Visualization</td>
      <td>high_dim_heatmap</td>
      <td>2019-07-20</td>
      <td>artbio</td>
      <td>https://github.com/artbio/tools-artbio/tree/main/tools/high_dim_heatmap</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/high_dim_heatmap</td>
      <td>3.1.3+galaxy0</td>
      <td>r-gplots</td>
      <td>2.17.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>hirondelle_crim_ogc_api_processes</td>
      <td>hirondelle_crim</td>
      <td>This tool is a wrapper for OGC API Processes coming from https://osf.io/gfbws/.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/AquaINFRA/galaxy</td>
      <td>Ecology</td>
      <td></td>
      <td>2024-06-07</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/ogc_api_processes_wrapper</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/hirondelle_crim_ogc_api_processes</td>
      <td>0.2.0</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>18</td>
      <td>18</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>18</td>
      <td>18</td>
      <td>2</td>
      <td>2</td>
    </tr>
    <tr>
      <td></td>
      <td>hyperstack_to_bleach_corrected_movie</td>
      <td>hyperstack_to_bleach_corrected_movie</td>
      <td>Generate blach corrected movie from hyperstack</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Imaging</td>
      <td>hyperstack_to_bleach_corrected_movie</td>
      <td>2023-02-27</td>
      <td>lldelisle</td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/hyperstack_to_bleach_corrected_movie</td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/hyperstack_to_bleach_corrected_movie</td>
      <td>20230328</td>
      <td>Fiji</td>
      <td>20240614</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>idconvert</td>
      <td>idconvert</td>
      <td>Convert mass spectrometry identification files on linux or MacOSX</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>idconvert</td>
      <td>2019-02-22</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msconvert</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/idconvert</td>
      <td></td>
      <td>proteowizard</td>
      <td>3_0_9992</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>241</td>
      <td>276</td>
      <td>24</td>
      <td>33</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>241</td>
      <td>276</td>
      <td>24</td>
      <td>33</td>
    </tr>
    <tr>
      <td></td>
      <td>incucyte_stack_and_upload_omero</td>
      <td>incucyte_stack_and_upload_omero</td>
      <td>Combine images to stack and upload to the omero server</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Imaging</td>
      <td>incucyte_stack_and_upload_omero</td>
      <td>2023-02-17</td>
      <td>lldelisle</td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/incucyte_stack_and_upload_omero</td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/incucyte_stack_and_upload_omero</td>
      <td>20231221</td>
      <td>Fiji</td>
      <td>20240614</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>interpolation</td>
      <td>interpolation_run_idw_interpolation</td>
      <td>Run IDW interpolation based on a .csv and .geojson file</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/AquaINFRA/galaxy</td>
      <td>Ecology</td>
      <td></td>
      <td>2024-01-05</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/interpolation</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/interpolation</td>
      <td>1.0</td>
      <td>r-getopt</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>70</td>
      <td>70</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>70</td>
      <td>70</td>
      <td>7</td>
      <td>7</td>
    </tr>
    <tr>
      <td></td>
      <td>iphop</td>
      <td>iphop_predict</td>
      <td>Integrated Phage Host Prediction</td>
      <td>iphop</td>
      <td>iphop</td>
      <td></td>
      <td>iPHoP</td>
      <td>An automated command-line pipeline for predicting host genus of novel bacteriophages and archaeoviruses based on their genome sequences.</td>
      <td>Sample comparison</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://bitbucket.org/srouxjgi/iphop/</td>
      <td>Metagenomics</td>
      <td>iphop</td>
      <td>2024-10-14</td>
      <td>ufz</td>
      <td>https://github.com/Helmholtz-UFZ/ufz-galaxy-tools/blob/main/tools/iphop/</td>
      <td>https://github.com/Helmholtz-UFZ/galaxy-tools/tree/main/tools/iphop</td>
      <td>1.3.3</td>
      <td>iphop</td>
      <td>1.3.3</td>
      <td>Sample comparison</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>isolib</td>
      <td>isolib</td>
      <td>Create an isotopic pattern library for given compounds and adducts.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/RECETOX/galaxytools/</td>
      <td>Metabolomics</td>
      <td>isolib</td>
      <td>2023-10-26</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/isolib</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/isolib</td>
      <td>1.0.1+galaxy0</td>
      <td>bioconductor-metabocoreutils</td>
      <td>1.10.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>justdiff</td>
      <td>justdiff</td>
      <td>Unix diff</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Text Manipulation</td>
      <td>justdiff</td>
      <td>2017-09-27</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/justdiff</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/justdiff</td>
      <td>3.10+galaxy0</td>
      <td>diffutils</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>justgzip</td>
      <td>justgzip</td>
      <td>Compress fastq sequence files</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Convert Formats</td>
      <td>justgzip</td>
      <td>2018-05-16</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/justgzip</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/justgzip</td>
      <td>2.8+galaxy0</td>
      <td>pigz</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>lfq_protein_quant</td>
      <td>lfq_protein_quant</td>
      <td>Enable protein summarisation and quantitation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/compomics/LFQ_galaxy_p</td>
      <td>Proteomics</td>
      <td>lfq_protein_quant</td>
      <td>2018-10-02</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/lfq_protein_quant</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/lfq_protein_quant</td>
      <td>1.0</td>
      <td>bioconductor-msnbase</td>
      <td>2.28.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>78</td>
      <td>116</td>
      <td>12</td>
      <td>19</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>78</td>
      <td>116</td>
      <td>12</td>
      <td>19</td>
    </tr>
    <tr>
      <td></td>
      <td>longorf</td>
      <td>longORF</td>
      <td>obtain longest ORF in six-frame translations</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>longorf</td>
      <td>2018-06-01</td>
      <td>mbernt</td>
      <td>https://github.com/bernt-matthias/mb-galaxy-tools/blob/master/tools/longorf/</td>
      <td>https://github.com/Helmholtz-UFZ/galaxy-tools/tree/main/tools/longorf</td>
      <td>0.3.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>ltq_iquant_cli</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td>ltq_iquant_cli</td>
      <td>2015-01-24</td>
      <td>galaxyp</td>
      <td></td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/ltq_iquant_cli</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>lumpy_smoove</td>
      <td>lumpy_smoove, vcf2hrdetect</td>
      <td>Galaxy wrapper of the lumpy-using smoove workflow</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Variant Analysis</td>
      <td>lumpy_smoove</td>
      <td>2020-07-12</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/lumpy_smoove</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/lumpy_smoove</td>
      <td>0.2.8+galaxy1</td>
      <td>svtyper</td>
      <td>0.7.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>lumpy_sv</td>
      <td>lumpy</td>
      <td>Find structural variations</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Variant Analysis</td>
      <td>lumpy_sv</td>
      <td>2017-07-24</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/lumpy_sv</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/lumpy_sv</td>
      <td>1.3</td>
      <td>lumpy-sv</td>
      <td>0.3.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>make_data_paper_sketches</td>
      <td>data_paper_from_EML</td>
      <td>This tool derived from the R Shiny App MetaShRIMPS &lt;https://github.com/TanguyGen/metaCure&gt; is made to produce draft of data paper from Ecological Metadata Language (EML) based metadata documents.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/TanguyGen/emldown</td>
      <td>Ecology</td>
      <td>Make data paper sketches from EML</td>
      <td>2024-10-15</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/main/tools-ecology/tools/make_data_paper_sketches</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/make_data_paper_sketches</td>
      <td>0.1.0+galaxy0</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>make_nr</td>
      <td>make_nr</td>
      <td>Make a FASTA file non-redundant</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/make_nr</td>
      <td>Fasta Manipulation, Sequence Analysis</td>
      <td>make_nr</td>
      <td>2018-11-06</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/make_nr</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/make_nr</td>
      <td>0.0.3</td>
      <td>biopython</td>
      <td>1.70</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>maldiquant</td>
      <td>maldi_quant_peak_detection, maldi_quant_preprocessing</td>
      <td>MALDIquant provides a complete analysis pipeline for MALDI-TOF and other 2D mass spectrometry data.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://strimmerlab.org/software/maldiquant/</td>
      <td>Proteomics</td>
      <td>MALDIquant</td>
      <td>2018-08-22</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/MALDIquant</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/maldiquant</td>
      <td>1.22.0</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>1452</td>
      <td>3822</td>
      <td>61</td>
      <td>68</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1065</td>
      <td>1065</td>
      <td>39</td>
      <td>39</td>
      <td>2517</td>
      <td>4887</td>
      <td>100</td>
      <td>107</td>
    </tr>
    <tr>
      <td></td>
      <td>manta</td>
      <td>manta</td>
      <td>Structural variant and indel caller for mapped sequencing data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Variant Analysis</td>
      <td>manta</td>
      <td>2023-06-07</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/manta</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/manta</td>
      <td>1.6</td>
      <td>manta</td>
      <td>1.6.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>218</td>
      <td>218</td>
      <td>34</td>
      <td>34</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>218</td>
      <td>218</td>
      <td>34</td>
      <td>34</td>
    </tr>
    <tr>
      <td></td>
      <td>map_peptides_to_bed</td>
      <td>map_peptides_to_bed</td>
      <td>Map peptides to a reference genome for display by a genome browser</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>map_peptides_to_bed</td>
      <td>2016-01-13</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/map_peptides_to_bed</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/map_peptides_to_bed</td>
      <td>0.2</td>
      <td>biopython</td>
      <td>1.70</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>38</td>
      <td>44</td>
      <td>6</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>38</td>
      <td>44</td>
      <td>6</td>
      <td>7</td>
    </tr>
    <tr>
      <td></td>
      <td>mapping_quality_stats</td>
      <td>mapqstatistics</td>
      <td>Collects and shows the distribution of MAPQ values in a BAM alignment file</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Sequence Analysis, Statistics</td>
      <td>mapping_quality_stats</td>
      <td>2022-06-15</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/mapping_quality_stats</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/mapping_quality_stats</td>
      <td>0.22.0</td>
      <td>r-optparse</td>
      <td>1.3.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>marine_omics</td>
      <td>sanntis_marine</td>
      <td>The Sanntis tool identify biosynthetic gene clusters (BGCs) in genomic &amp; metagenomic data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/Finn-Lab/SanntiS</td>
      <td>Ecology</td>
      <td></td>
      <td>2024-07-26</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/marine_omics</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/marine_omics</td>
      <td>0.9.3.5</td>
      <td>sanntis</td>
      <td>0.9.3.5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>51</td>
      <td>51</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>51</td>
      <td>51</td>
      <td>4</td>
      <td>4</td>
    </tr>
    <tr>
      <td></td>
      <td>matchms</td>
      <td>matchms_add_key, matchms_convert, matchms_filtering, matchms_fingerprint_similarity, matchms_formatter, matchms_metadata_export, matchms_metadata_match, matchms_metadata_merge, matchms_networking, matchms_remove_key, matchms_remove_spectra, matchms_spectral_similarity, matchms_split, matchms_subsetting</td>
      <td>Searching, filtering and converting mass spectral libraries.</td>
      <td>matchms</td>
      <td>matchms</td>
      <td></td>
      <td>Matchms</td>
      <td>Tool to import, process, clean, and compare mass spectrometry data.</td>
      <td>Spectral library search, Format validation, Filtering</td>
      <td>Metabolomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/matchms/matchms</td>
      <td>Metabolomics</td>
      <td>matchms</td>
      <td>2023-12-12</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/matchms</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/matchms</td>
      <td>0.27.0</td>
      <td>matchms</td>
      <td>0.27.0</td>
      <td>Spectral library search, Format validation, Filtering</td>
      <td>Metabolomics</td>
      <td>2</td>
      <td>14</td>
      <td>14</td>
      <td>11</td>
      <td>2</td>
      <td>14</td>
      <td>14</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>1620</td>
      <td>1620</td>
      <td>70</td>
      <td>70</td>
      <td>27</td>
      <td>27</td>
      <td>8</td>
      <td>8</td>
      <td>1685</td>
      <td>1685</td>
      <td>12</td>
      <td>12</td>
      <td>3332</td>
      <td>3332</td>
      <td>90</td>
      <td>90</td>
    </tr>
    <tr>
      <td></td>
      <td>maxquant</td>
      <td>maxquant, maxquant_mqpar</td>
      <td>wrapper for MaxQuant</td>
      <td>maxquant</td>
      <td>maxquant</td>
      <td></td>
      <td>MaxQuant</td>
      <td>Quantitative proteomics software package designed for analyzing large mass-spectrometric data sets. It is specifically aimed at high-resolution MS data.</td>
      <td>Imputation, Visualisation, Protein quantification, Statistical calculation, Standardisation and normalisation, Heat map generation, Clustering, Principal component plotting</td>
      <td>Proteomics experiment, Proteomics, Statistics and probability</td>
      <td>Up-to-date</td>
      <td>https://www.maxquant.org/</td>
      <td>Proteomics</td>
      <td>maxquant</td>
      <td>2021-08-05</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/maxquant</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/maxquant</td>
      <td>2.0.3.0</td>
      <td>maxquant</td>
      <td>2.0.3.0</td>
      <td>Imputation, Visualisation, Protein quantification, Standardisation and normalisation, Heat map generation, Clustering</td>
      <td>Proteomics experiment, Proteomics, Statistics and probability</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>6863</td>
      <td>7021</td>
      <td>1251</td>
      <td>1264</td>
      <td>2798</td>
      <td>2798</td>
      <td>543</td>
      <td>543</td>
      <td>3189</td>
      <td>3189</td>
      <td>352</td>
      <td>352</td>
      <td>12850</td>
      <td>13008</td>
      <td>2146</td>
      <td>2159</td>
    </tr>
    <tr>
      <td></td>
      <td>mdanalysis</td>
      <td>mdanalysis_angle, mdanalysis_dihedral, mdanalysis_distance, mdanalysis_endtoend, mdanalysis_extract_rmsd, mdanalysis_hbonds, mdanalysis_cosine_analysis, mdanalysis_ramachandran_protein, mdanalysis_ramachandran_plot, mdanalysis_rdf</td>
      <td>MDAnalysis is a package for analyzing trajectories from molecular dynamics (MD) simulations</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/MDAnalysis/mdanalysis</td>
      <td>Computational chemistry</td>
      <td>mdanalysis</td>
      <td>2018-10-08</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/mdanalysis</td>
      <td>1.0.0</td>
      <td>mdanalysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td>10</td>
      <td>5</td>
      <td>10</td>
      <td>0</td>
      <td>10</td>
      <td>5</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>3209</td>
      <td>3268</td>
      <td>318</td>
      <td>326</td>
      <td>182</td>
      <td>182</td>
      <td>51</td>
      <td>51</td>
      <td>397</td>
      <td>397</td>
      <td>39</td>
      <td>39</td>
      <td>3788</td>
      <td>3847</td>
      <td>408</td>
      <td>416</td>
    </tr>
    <tr>
      <td></td>
      <td>mdfileconverter</td>
      <td>md_converter</td>
      <td>A tool for interconverting between different MD structure and trajectory file formats.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Molecular Dynamics, Computational chemistry</td>
      <td>md_converter</td>
      <td>2018-10-13</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tools/mdfileconverter</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/mdfileconverter</td>
      <td>1.9.7</td>
      <td>mdtraj</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>26720</td>
      <td>26762</td>
      <td>251</td>
      <td>253</td>
      <td>784</td>
      <td>784</td>
      <td>54</td>
      <td>54</td>
      <td>977</td>
      <td>977</td>
      <td>53</td>
      <td>53</td>
      <td>28481</td>
      <td>28523</td>
      <td>358</td>
      <td>360</td>
    </tr>
    <tr>
      <td></td>
      <td>mdslicer</td>
      <td>md_slicer</td>
      <td>A tool for slicing trajectory files using MDTraj.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Molecular Dynamics, Computational chemistry</td>
      <td>md_converter</td>
      <td>2019-10-07</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tools/mdslicer</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/mdslicer</td>
      <td>1.9.9</td>
      <td>mdtraj</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>174</td>
      <td>174</td>
      <td>19</td>
      <td>19</td>
      <td>6</td>
      <td>6</td>
      <td>4</td>
      <td>4</td>
      <td>441</td>
      <td>441</td>
      <td>7</td>
      <td>7</td>
      <td>621</td>
      <td>621</td>
      <td>30</td>
      <td>30</td>
    </tr>
    <tr>
      <td></td>
      <td>mdtraj</td>
      <td>traj_selections_and_merge</td>
      <td>MDTraj is a python library that allows users to manipulate molecular dynamics (MD) trajectories</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/mdtraj/mdtraj</td>
      <td>Computational chemistry</td>
      <td>mdtraj</td>
      <td>2020-06-24</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/mdtraj</td>
      <td>1.9.7</td>
      <td>mdtraj</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>36</td>
      <td>36</td>
      <td>8</td>
      <td>8</td>
      <td>63</td>
      <td>63</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>99</td>
      <td>99</td>
      <td>12</td>
      <td>12</td>
    </tr>
    <tr>
      <td></td>
      <td>mean-per-zone</td>
      <td>mean_per_zone</td>
      <td>Creates a png image showing statistic over areas as defined in the vector file</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/blob/master/tools/mean-per-zone/</td>
      <td>Visualization, GIS, Climate Analysis</td>
      <td>mean_per_zone</td>
      <td>2019-03-15</td>
      <td>climate</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/mean-per-zone</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/mean-per-zone</td>
      <td>0.2.0</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>20</td>
      <td>25</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>20</td>
      <td>25</td>
      <td>3</td>
      <td>3</td>
    </tr>
    <tr>
      <td></td>
      <td>measure_gastruloids</td>
      <td>measureGastruloids</td>
      <td>Get the ROI coordinates around the gastruloids as well as measurements like Area, elongation index</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Imaging</td>
      <td>measure_gastruloids</td>
      <td>2022-01-18</td>
      <td>lldelisle</td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/measure_gastruloids</td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/measure_gastruloids</td>
      <td>20221216</td>
      <td>fiji</td>
      <td>20240614</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>medenv</td>
      <td>iabiodiv_smartbiodiv_med_environ</td>
      <td>Retrieve environmental data from etopo, cmems and woa</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/jeremyfix/medenv</td>
      <td>Ecology, Data Source</td>
      <td></td>
      <td>2023-12-12</td>
      <td>ecology</td>
      <td>https://github.com/jeremyfix/medenv</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/medenv</td>
      <td>0.1.0</td>
      <td>pandas</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>27</td>
      <td>27</td>
      <td>9</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>27</td>
      <td>27</td>
      <td>9</td>
      <td>9</td>
    </tr>
    <tr>
      <td></td>
      <td>meta_proteome_analyzer</td>
      <td>meta_proteome_analyzer</td>
      <td>MetaProteomeAnalyzer</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/compomics/meta-proteome-analyzer/</td>
      <td>Proteomics</td>
      <td>meta_proteome_analyzer</td>
      <td>2017-03-03</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/meta_proteome_analyzer</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/meta_proteome_analyzer</td>
      <td>2.0.0</td>
      <td>mpa-portable</td>
      <td>2.0.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>126</td>
      <td>126</td>
      <td>20</td>
      <td>20</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>126</td>
      <td>126</td>
      <td>20</td>
      <td>20</td>
    </tr>
    <tr>
      <td></td>
      <td>metagene_annotator</td>
      <td>metagene_annotator</td>
      <td>MetaGeneAnnotator gene-finding program for prokaryote and phage</td>
      <td>metageneannotator</td>
      <td>metageneannotator</td>
      <td></td>
      <td>MetaGeneAnnotator</td>
      <td>Prokaryotic gene finding program from environmental genome shotgun sequences or metagenomic sequences.</td>
      <td>Sequence annotation</td>
      <td>Genomics, Model organisms, Data submission, annotation and curation</td>
      <td>Up-to-date</td>
      <td>http://metagene.nig.ac.jp/</td>
      <td>Sequence Analysis</td>
      <td>metagene_annotator</td>
      <td>2018-03-21</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/metagene_annotator</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/metagene_annotator</td>
      <td>1.0</td>
      <td>metagene_annotator</td>
      <td>1.0</td>
      <td>Sequence annotation</td>
      <td>Genomics, Model organisms, Data submission, annotation and curation</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>980</td>
      <td>980</td>
      <td>323</td>
      <td>323</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>980</td>
      <td>980</td>
      <td>323</td>
      <td>323</td>
    </tr>
    <tr>
      <td></td>
      <td>metanovo</td>
      <td>metanovo</td>
      <td>Produce targeted databases for mass spectrometry analysis.</td>
      <td>metanovo</td>
      <td>metanovo</td>
      <td></td>
      <td>MetaNovo</td>
      <td>An open-source pipeline for probabilistic peptide discovery in complex metaproteomic datasets.</td>
      <td>Target-Decoy, de Novo sequencing, Tag-based peptide identification, Protein identification, Expression analysis</td>
      <td>Proteomics, Microbial ecology, Metagenomics, Proteomics experiment, Small molecules</td>
      <td>Up-to-date</td>
      <td>https://github.com/uct-cbio/proteomics-pipelines</td>
      <td>Proteomics</td>
      <td>metanovo</td>
      <td>2022-03-29</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/metanovo</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/metanovo</td>
      <td>1.9.4</td>
      <td>metanovo</td>
      <td>1.9.4</td>
      <td>Target-Decoy, de Novo sequencing, Tag-based peptide identification, Protein identification, Expression analysis</td>
      <td>Proteomics, Microbial ecology, Metagenomics, Proteomics experiment, Small molecules</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>4471</td>
      <td>4471</td>
      <td>29</td>
      <td>29</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4471</td>
      <td>4471</td>
      <td>29</td>
      <td>29</td>
    </tr>
    <tr>
      <td></td>
      <td>metaquantome</td>
      <td>metaquantome_db, metaquantome_expand, metaquantome_filter, metaquantome_sample, metaquantome_stat, metaquantome_viz</td>
      <td>quantitative analysis of microbiome taxonomy and function</td>
      <td>metaQuantome</td>
      <td>metaQuantome</td>
      <td></td>
      <td>metaQuantome</td>
      <td>metaQuantome software suite analyzes the state of a microbiome by leveraging complex taxonomic and functional hierarchies to summarize peptide-level quantitative information. metaQuantome offers differential abundance analysis, principal components analysis, and clustered heat map visualizations, as well as exploratory analysis for a single sample or experimental condition.</td>
      <td>Principal component visualisation, Visualisation, Functional clustering, Query and retrieval, Differential protein expression analysis, Heat map generation, Quantification, Indexing, Filtering, Statistical inference</td>
      <td>Proteomics, Metatranscriptomics, Microbial ecology, Proteomics experiment, Metagenomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproteomics/metaquantome/</td>
      <td>Proteomics</td>
      <td>metaquantome</td>
      <td>2019-04-05</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/metaquantome</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/metaquantome</td>
      <td>2.0.2</td>
      <td>metaquantome</td>
      <td>2.0.2</td>
      <td>Principal component visualisation, Functional clustering, Query and retrieval, Heat map generation, Quantification, Indexing, Filtering, Statistical inference</td>
      <td>Proteomics, Metatranscriptomics, Microbial ecology, Proteomics experiment, Metagenomics</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>2730</td>
      <td>2831</td>
      <td>260</td>
      <td>262</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1540</td>
      <td>1540</td>
      <td>71</td>
      <td>71</td>
      <td>4270</td>
      <td>4371</td>
      <td>331</td>
      <td>333</td>
    </tr>
    <tr>
      <td></td>
      <td>mfassignr</td>
      <td>mfassignr_findRecalSeries, mfassignr_histnoise, mfassignr_isofiltr, mfassignr_kmdnoise, mfassignr_mfassign, mfassignr_mfassignCHO, mfassignr_recal, mfassignr_recallist, mfassignr_snplot</td>
      <td>The MFAssignR package was designed for multi-element molecular formula (MF) assignment of ultrahigh resolution mass spectrometry measurements</td>
      <td>mfassignr</td>
      <td>mfassignr</td>
      <td></td>
      <td>MFAssignR</td>
      <td>Molecular formula assignment software for ultrahigh resolution mass spectrometry analysis of environmental complex mixtures.Ultrahigh resolution mass spectrometry is widely used for nontargeted analysis of complex environmental and biological mixtures, such as dissolved organic matter, due to its unparalleled ability to provide accurate mass measurements.</td>
      <td>Visualisation</td>
      <td>Proteomics experiment, Molecular interactions, pathways and networks, Workflows</td>
      <td>Up-to-date</td>
      <td>https://github.com/RECETOX/MFAssignR</td>
      <td>Metabolomics</td>
      <td>MFAssignR</td>
      <td>2024-08-08</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/mfassignr</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/mfassignr</td>
      <td>1.1.1</td>
      <td>r-mfassignr</td>
      <td>1.1.1</td>
      <td>Visualisation</td>
      <td>Proteomics experiment, Molecular interactions, pathways and networks, Workflows</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>10</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>10</td>
      <td>8</td>
      <td>8</td>
    </tr>
    <tr>
      <td></td>
      <td>mgf_formatter</td>
      <td>mgf_formatter</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td>mgf_formatter</td>
      <td>2014-09-26</td>
      <td>galaxyp</td>
      <td></td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/mgf_formatter</td>
      <td>1.0.0</td>
      <td>mgf-formatter</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>mircounts</td>
      <td>mircounts</td>
      <td>Generates miRNA count lists from read alignments to mirBase.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, Transcriptomics</td>
      <td>mircounts</td>
      <td>2017-07-24</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/mircounts</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/mircounts</td>
      <td>1.6</td>
      <td>tar</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1458</td>
      <td>1458</td>
      <td>36</td>
      <td>36</td>
      <td>1458</td>
      <td>1458</td>
      <td>36</td>
      <td>36</td>
    </tr>
    <tr>
      <td></td>
      <td>misc</td>
      <td>target_screen, use_theoretical_mz_annotations</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/RECETOX/galaxytools</td>
      <td>Metabolomics</td>
      <td></td>
      <td>2024-02-16</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/misc</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/misc</td>
      <td>1.0.0</td>
      <td>pandas</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>2</td>
      <td>2</td>
    </tr>
    <tr>
      <td></td>
      <td>moFF</td>
      <td>proteomics_moff</td>
      <td>moFF (a modest Feature Finder) extracts MS1 intensities from RAW and mzML spectrum files.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/compomics/moFF</td>
      <td>Proteomics</td>
      <td>proteomics_moff</td>
      <td>2019-01-28</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/moFF</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/moFF</td>
      <td>2.0.3</td>
      <td>moff</td>
      <td>2.0.3</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>331</td>
      <td>762</td>
      <td>20</td>
      <td>34</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>137</td>
      <td>137</td>
      <td>4</td>
      <td>4</td>
      <td>468</td>
      <td>899</td>
      <td>24</td>
      <td>38</td>
    </tr>
    <tr>
      <td></td>
      <td>monocle3</td>
      <td>monocle3_create, monocle3_diffExp, monocle3_learnGraph, monocle3_orderCells, monocle3_partition, monocle3_plotCells, monocle3_preprocess, monocle3_reduceDim, monocle3_topmarkers</td>
      <td>De-composed monocle3 functionality tools, based on https://github.com/ebi-gene-expression-group/monocle-scripts and monocle3 0.1.2.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_monocle3</td>
      <td>2019-09-09</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/monocle3</td>
      <td>0.1.4</td>
      <td>monocle3-cli</td>
      <td>0.0.9</td>
      <td></td>
      <td></td>
      <td>9</td>
      <td>0</td>
      <td>9</td>
      <td>0</td>
      <td>9</td>
      <td>0</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3610</td>
      <td>3610</td>
      <td>559</td>
      <td>559</td>
      <td>1728</td>
      <td>1728</td>
      <td>434</td>
      <td>434</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5338</td>
      <td>5338</td>
      <td>993</td>
      <td>993</td>
    </tr>
    <tr>
      <td></td>
      <td>morpheus</td>
      <td>morpheus</td>
      <td>Morpheus MS Search Application</td>
      <td>morpheus</td>
      <td>morpheus</td>
      <td></td>
      <td>Morpheus</td>
      <td>A proteomics search algorithm specifically designed for high-resolution tandem mass spectra.</td>
      <td>Peptide database search</td>
      <td>Proteomics</td>
      <td>To update</td>
      <td>https://cwenger.github.io/Morpheus</td>
      <td>Proteomics</td>
      <td>morpheus</td>
      <td>2015-10-29</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/morpheus</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/morpheus</td>
      <td>288</td>
      <td>morpheus</td>
      <td>290</td>
      <td>Peptide database search</td>
      <td>Proteomics</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>105</td>
      <td>153</td>
      <td>12</td>
      <td>19</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>833</td>
      <td>833</td>
      <td>93</td>
      <td>93</td>
      <td>938</td>
      <td>986</td>
      <td>105</td>
      <td>112</td>
    </tr>
    <tr>
      <td></td>
      <td>mqppep</td>
      <td>mqppep_anova, mqppep_preproc</td>
      <td>MaxQuant Phosphoproteomic Enrichment Pipeline - Preprocessing and ANOVA</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/</td>
      <td>Proteomics</td>
      <td>mqppep</td>
      <td>2022-03-31</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/mqppep</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/mqppep</td>
      <td>0.1.19</td>
      <td>bioconductor-preprocesscore</td>
      <td>1.64.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>13</td>
      <td>13</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>13</td>
      <td>13</td>
      <td>3</td>
      <td>3</td>
    </tr>
    <tr>
      <td></td>
      <td>ms2deepscore</td>
      <td>ms2deepscore_config_generator, ms2deepscore_similarity, ms2deepscore_training</td>
      <td>Mass spectra similarity scoring using a trained ms2deepscore model.</td>
      <td>ms2deepscore</td>
      <td>ms2deepscore</td>
      <td></td>
      <td>MS2DeepScore</td>
      <td>MS2DeepScore is a deep learning similarity measure for mass fragmentation spectrum comparisons. MS2DeepScore provides a Siamese neural network that is trained to predict molecular structural similarities (Tanimoto scores) from pairs of mass spectrometry spectra.</td>
      <td>Spectrum calculation, Spectral library search, Network analysis</td>
      <td>Proteomics experiment, Machine learning, Metabolomics, Small molecules, Compound libraries and screening</td>
      <td>To update</td>
      <td>https://github.com/matchms/ms2deepscore</td>
      <td>Metabolomics</td>
      <td>ms2deepscore</td>
      <td>2024-08-15</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/ms2deepscore</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/ms2deepscore</td>
      <td>2.0.0</td>
      <td>ms2deepscore</td>
      <td>2.2.0</td>
      <td>Spectrum calculation, Spectral library search, Network analysis</td>
      <td>Proteomics experiment, Machine learning, Metabolomics, Small molecules, Compound libraries and screening</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>msconvert</td>
      <td>msconvert</td>
      <td>msconvert Convert and/or filter mass spectrometry files (including vendor formats) using the official Docker container</td>
      <td>msconvert</td>
      <td>msconvert</td>
      <td></td>
      <td>msConvert</td>
      <td>msConvert is a command-line utility for converting between various mass spectrometry data formats, including from raw data from several commercial companies (with vendor libraries, Windows-only). For Windows users, there is also a GUI, msConvertGUI.</td>
      <td>Filtering, Formatting</td>
      <td>Proteomics, Proteomics experiment</td>
      <td>To update</td>
      <td>http://proteowizard.sourceforge.net/tools.shtml</td>
      <td>Proteomics</td>
      <td>msconvert</td>
      <td>2019-02-14</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msconvert</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msconvert</td>
      <td>3.0.20287</td>
      <td></td>
      <td></td>
      <td>Filtering, Formatting</td>
      <td>Proteomics, Proteomics experiment</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>20749</td>
      <td>22184</td>
      <td>455</td>
      <td>469</td>
      <td>1239</td>
      <td>1239</td>
      <td>71</td>
      <td>71</td>
      <td>2666</td>
      <td>2666</td>
      <td>53</td>
      <td>53</td>
      <td>24654</td>
      <td>26089</td>
      <td>579</td>
      <td>593</td>
    </tr>
    <tr>
      <td></td>
      <td>msgfplus</td>
      <td>msgfplus</td>
      <td>MSGF+</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>msgfplus</td>
      <td>2017-01-12</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msgfplus</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msgfplus</td>
      <td>0.5</td>
      <td>msgf_plus</td>
      <td>2024.03.26</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>473</td>
      <td>529</td>
      <td>33</td>
      <td>43</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>473</td>
      <td>529</td>
      <td>33</td>
      <td>43</td>
    </tr>
    <tr>
      <td></td>
      <td>msmetaenhancer</td>
      <td>msmetaenhancer</td>
      <td></td>
      <td>msmetaenhancer</td>
      <td>msmetaenhancer</td>
      <td></td>
      <td>MSMetaEnhancer</td>
      <td>Tool for mass spectra metadata annotation.</td>
      <td>Annotation, Standardisation and normalisation</td>
      <td>Metabolomics, Compound libraries and screening, Data submission, annotation and curation</td>
      <td>Up-to-date</td>
      <td>https://github.com/RECETOX/MSMetaEnhancer</td>
      <td>Metabolomics</td>
      <td></td>
      <td>2022-04-22</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/msmetaenhancer</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/msmetaenhancer</td>
      <td>0.4.0</td>
      <td>msmetaenhancer</td>
      <td>0.4.0</td>
      <td>Annotation, Standardisation and normalisation</td>
      <td>Metabolomics, Compound libraries and screening, Data submission, annotation and curation</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>203</td>
      <td>203</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>48</td>
      <td>48</td>
      <td>1</td>
      <td>1</td>
      <td>251</td>
      <td>251</td>
      <td>11</td>
      <td>11</td>
    </tr>
    <tr>
      <td></td>
      <td>msms_extractor</td>
      <td>msms_extractor</td>
      <td>Extract MS/MS scans from the mzML file(s) based on PSM report.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>msms_extractor</td>
      <td>2019-10-23</td>
      <td>galaxyp</td>
      <td></td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msms_extractor</td>
      <td>1.0.0</td>
      <td>proteowizard</td>
      <td>3_0_9992</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>110</td>
      <td>110</td>
      <td>11</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>110</td>
      <td>110</td>
      <td>11</td>
      <td>11</td>
    </tr>
    <tr>
      <td></td>
      <td>msp_merge</td>
      <td>msp_merge</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/RECETOX/galaxytools</td>
      <td>Metabolomics</td>
      <td></td>
      <td>2022-05-05</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/msp_merge</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/msp_merge</td>
      <td>0.1.0</td>
      <td>matchms</td>
      <td>0.27.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>msstats</td>
      <td>msstats</td>
      <td>MSstats tool for analyzing mass spectrometry proteomic datasets</td>
      <td>msstatstmt</td>
      <td>msstatstmt</td>
      <td></td>
      <td>MSstatsTMT</td>
      <td>Tools for detecting differentially abundant peptides and proteins in shotgun mass spectrometry-based proteomic experiments with tandem mass tag (TMT) labeling</td>
      <td>Spectrum calculation, Tag-based peptide identification, Differential protein expression profiling</td>
      <td>Proteomics, Proteomics experiment, Protein expression</td>
      <td>To update</td>
      <td>https://github.com/MeenaChoi/MSstats</td>
      <td>Proteomics</td>
      <td></td>
      <td>2020-07-25</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msstats</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msstats</td>
      <td>4.0.0</td>
      <td>bioconductor-msstats</td>
      <td>4.10.0</td>
      <td>Spectrum calculation, Tag-based peptide identification, Differential protein expression profiling</td>
      <td>Proteomics, Proteomics experiment, Protein expression</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2353</td>
      <td>2353</td>
      <td>559</td>
      <td>559</td>
      <td>1122</td>
      <td>1122</td>
      <td>211</td>
      <td>211</td>
      <td>1091</td>
      <td>1091</td>
      <td>35</td>
      <td>35</td>
      <td>4566</td>
      <td>4566</td>
      <td>805</td>
      <td>805</td>
    </tr>
    <tr>
      <td></td>
      <td>msstatstmt</td>
      <td>msstatstmt</td>
      <td>MSstatsTMT protein significance analysis in shotgun mass spectrometry-based proteomic experiments with tandem mass tag (TMT) labeling</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://msstats.org/msstatstmt/</td>
      <td>Proteomics</td>
      <td>msstatstmt</td>
      <td>2021-02-26</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msstatstmt</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msstatstmt</td>
      <td>2.0.0</td>
      <td>bioconductor-msstatstmt</td>
      <td>2.10.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>795</td>
      <td>795</td>
      <td>104</td>
      <td>104</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>461</td>
      <td>461</td>
      <td>12</td>
      <td>12</td>
      <td>1256</td>
      <td>1256</td>
      <td>116</td>
      <td>116</td>
    </tr>
    <tr>
      <td></td>
      <td>mt2mq</td>
      <td>mt2mq</td>
      <td>Tool to prepare metatranscriptomic outputs from ASaiM for Metaquantome</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>mt2mq</td>
      <td>2020-06-23</td>
      <td>galaxyp</td>
      <td></td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/mt2mq</td>
      <td>1.1.0</td>
      <td>r-tidyverse</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>275</td>
      <td>275</td>
      <td>38</td>
      <td>38</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>275</td>
      <td>275</td>
      <td>38</td>
      <td>38</td>
    </tr>
    <tr>
      <td></td>
      <td>mutational_patterns</td>
      <td>mutational_patterns</td>
      <td>Mutational patterns and signatures in base substitution catalogs</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://artbio.fr</td>
      <td>Variant Analysis</td>
      <td>mutational_patterns</td>
      <td>2020-10-19</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/mutational_patterns</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/mutational_patterns</td>
      <td>3.12.0</td>
      <td>bioconductor-mutationalpatterns</td>
      <td>3.12.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>mz_to_sqlite</td>
      <td>mz_to_sqlite</td>
      <td>Creates a SQLite database for proteomics data</td>
      <td>mztosqlite</td>
      <td>mztosqlite</td>
      <td></td>
      <td>mzToSQLite</td>
      <td>Convert proteomics data files into a SQLite database</td>
      <td>Conversion, Peptide database search</td>
      <td>Proteomics, Biological databases</td>
      <td>To update</td>
      <td>https://github.com/galaxyproteomics/mzToSQLite</td>
      <td>Proteomics</td>
      <td>mz_to_sqlite</td>
      <td>2015-06-01</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/mz_to_sqlite</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/mz_to_sqlite</td>
      <td>2.1.1+galaxy0</td>
      <td>mztosqlite</td>
      <td>2.1.1</td>
      <td>Conversion, Peptide database search</td>
      <td>Proteomics, Biological databases</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>854</td>
      <td>860</td>
      <td>110</td>
      <td>111</td>
      <td>79</td>
      <td>79</td>
      <td>15</td>
      <td>15</td>
      <td>274</td>
      <td>274</td>
      <td>13</td>
      <td>13</td>
      <td>1207</td>
      <td>1213</td>
      <td>138</td>
      <td>139</td>
    </tr>
    <tr>
      <td></td>
      <td>mzml_validator</td>
      <td>mzml_validator</td>
      <td>mzML Validator checks if mzML file validates against XML Schema Definition of HUPO Proteomics Standard Initiative.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/RECETOX/galaxytools</td>
      <td>Metabolomics, Proteomics</td>
      <td></td>
      <td>2023-01-12</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/mzml_validator</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/mzml_validator</td>
      <td>0.1.0+galaxy2</td>
      <td>lxml</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>ncbi_blast_plus</td>
      <td>blastxml_to_tabular, get_species_taxids, ncbi_blastdbcmd_info, ncbi_blastdbcmd_wrapper, ncbi_blastn_wrapper, ncbi_blastp_wrapper, ncbi_blastx_wrapper, ncbi_convert2blastmask_wrapper, ncbi_deltablast_wrapper, ncbi_dustmasker_wrapper, ncbi_makeblastdb, ncbi_makeprofiledb, ncbi_psiblast_wrapper, ncbi_rpsblast_wrapper, ncbi_rpstblastn_wrapper, ncbi_segmasker_wrapper, ncbi_tblastn_wrapper, ncbi_tblastx_wrapper</td>
      <td>NCBI BLAST+</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://blast.ncbi.nlm.nih.gov/</td>
      <td>Sequence Analysis</td>
      <td>ncbi_blast_plus</td>
      <td>2020-09-08</td>
      <td>devteam</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/ncbi_blast_plus</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/ncbi_blast_plus</td>
      <td>2.14.1</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>16</td>
      <td>16</td>
      <td>16</td>
      <td>16</td>
      <td>16</td>
      <td>16</td>
      <td>16</td>
      <td>16</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>16</td>
      <td>15</td>
      <td>16</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>16</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>16</td>
      <td>15</td>
      <td>16</td>
      <td>16</td>
      <td>15</td>
      <td>0</td>
      <td>391141</td>
      <td>443752</td>
      <td>9760</td>
      <td>10546</td>
      <td>306361</td>
      <td>350178</td>
      <td>11767</td>
      <td>13684</td>
      <td>227235</td>
      <td>244814</td>
      <td>3192</td>
      <td>3617</td>
      <td>924737</td>
      <td>1038744</td>
      <td>24719</td>
      <td>27847</td>
    </tr>
    <tr>
      <td></td>
      <td>oases</td>
      <td>oasesoptimiserv</td>
      <td>Short read assembler</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Assembly, RNA</td>
      <td>oases</td>
      <td>2017-10-15</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/oases</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/oases</td>
      <td>1.4.0</td>
      <td>oases</td>
      <td>0.2.09</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>obisindicators</td>
      <td>obisindicators, obis_data</td>
      <td>Compute biodiveristy indicators for marine data from obis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/Marie59/obisindicators</td>
      <td>Ecology</td>
      <td></td>
      <td>2022-11-04</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/obisindicators</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/obisindicators</td>
      <td>0.0.2</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>2</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>352</td>
      <td>352</td>
      <td>42</td>
      <td>42</td>
      <td>8</td>
      <td>8</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>360</td>
      <td>360</td>
      <td>48</td>
      <td>48</td>
    </tr>
    <tr>
      <td></td>
      <td>ocean</td>
      <td>argo_getdata, divand_full_analysis</td>
      <td>Access, process, visualise oceanographic data for the Earth System</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/Marie59/FE-ft-ESG/tree/main/argo</td>
      <td>Ecology</td>
      <td></td>
      <td>2023-11-17</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/ocean</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/ocean</td>
      <td>0.1.15</td>
      <td>julia</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>54</td>
      <td>54</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>54</td>
      <td>54</td>
      <td>6</td>
      <td>6</td>
    </tr>
    <tr>
      <td></td>
      <td>ogcProcess_otb_bandmath</td>
      <td>otb_band_math</td>
      <td>Outputs a monoband image which is the result of a mathematical operation on several multi-band images.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/AquaINFRA/galaxy</td>
      <td>Ecology</td>
      <td></td>
      <td>2024-03-12</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/OtbBandMath</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/ogcProcess_otb_bandmath</td>
      <td>1.0</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>98</td>
      <td>98</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>98</td>
      <td>98</td>
      <td>6</td>
      <td>6</td>
    </tr>
    <tr>
      <td></td>
      <td>ogcProcess_otb_meanShiftSmoothing</td>
      <td>otb_mean_shift_smoothing</td>
      <td>This application smooths an image using the MeanShift algorithm.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/AquaINFRA/galaxy</td>
      <td>Ecology</td>
      <td></td>
      <td>2024-03-12</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/interpolation</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/ogcProcess_otb_meanShiftSmoothing</td>
      <td>1.0</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>57</td>
      <td>57</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>57</td>
      <td>57</td>
      <td>4</td>
      <td>4</td>
    </tr>
    <tr>
      <td></td>
      <td>omero</td>
      <td>omero_import, omero_metadata_import, omero_roi_import</td>
      <td>Import images, region of interest, metadata into an OMERO.server using omero-py</td>
      <td>omero</td>
      <td>omero</td>
      <td></td>
      <td>OMERO</td>
      <td>Client-server Java software for visualisation, management and analysis of biological microscope images.</td>
      <td>Image analysis</td>
      <td>Imaging, Data visualisation</td>
      <td>To update</td>
      <td>https://github.com/ome/omero-py/</td>
      <td>Imaging</td>
      <td>omero_upload</td>
      <td>2024-07-29</td>
      <td>ufz</td>
      <td>https://github.com/Helmholtz-UFZ/galaxy-tools/tree/main/tools/omero</td>
      <td>https://github.com/Helmholtz-UFZ/galaxy-tools/tree/main/tools/omero</td>
      <td>5.18.0</td>
      <td>omero-py</td>
      <td>5.11.1</td>
      <td>Image analysis</td>
      <td>Imaging, Data visualisation</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>omero_clean_rois_tables</td>
      <td>omero_clean_rois_tables</td>
      <td>Remove all ROIs and all tables on OMERO associated to an omero object and recursively up and down</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Imaging</td>
      <td>omero_clean_rois_tables</td>
      <td>2023-06-23</td>
      <td>lldelisle</td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/omero_clean_rois_tables</td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/omero_clean_rois_tables</td>
      <td>20230623</td>
      <td>fiji</td>
      <td>20240614</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>omero_get_children_ids</td>
      <td>omero_get_children_ids</td>
      <td>Get omero id of children of an omero object id</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Imaging</td>
      <td>omero_get_children_ids</td>
      <td>2023-12-22</td>
      <td>lldelisle</td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/omero_get_children_ids</td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/omero_get_children_ids</td>
      <td>0.2.0</td>
      <td>omero-py</td>
      <td>5.11.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>omero_get_full_images</td>
      <td>omero_get_full_images</td>
      <td>Get full images from omero</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Imaging</td>
      <td>omero_get_full_images</td>
      <td>2024-05-21</td>
      <td>lldelisle</td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/omero_get_full_images</td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/omero_get_full_images</td>
      <td>20240521</td>
      <td>fiji</td>
      <td>20240614</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>omero_hyperstack_to_fluo_measurements_on_gastruloid</td>
      <td>omero_hyperstack_to_fluo_measurements_on_gastruloid</td>
      <td>Analyse Hyperstack on OMERO server to measure fluorescence levels</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Imaging</td>
      <td>omero_hyperstack_to_fluo_measurements_on_gastruloid</td>
      <td>2023-03-24</td>
      <td>lldelisle</td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/omero_hyperstack_to_fluo_measurements_on_gastruloid</td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/omero_hyperstack_to_fluo_measurements_on_gastruloid</td>
      <td>20230809</td>
      <td>fiji</td>
      <td>20240614</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>omero_hyperstack_to_gastruloid_measurements</td>
      <td>omero_hyperstack_to_gastruloid_measurements</td>
      <td>Analyse Hyperstack on OMERO server to segment gastruloid and compute measurements</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Imaging</td>
      <td>omero_hyperstack_to_gastruloid_measurements</td>
      <td>2023-03-24</td>
      <td>lldelisle</td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/omero_hyperstack_to_gastruloid_measurements</td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/omero_hyperstack_to_gastruloid_measurements</td>
      <td>20240214</td>
      <td>fiji</td>
      <td>20240614</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>openmm</td>
      <td>pdbfixer</td>
      <td>OpenMM is a toolkit for molecular simulation using high performance GPU code.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/openmm</td>
      <td>Molecular Dynamics, Computational chemistry</td>
      <td>openmm</td>
      <td>2022-03-25</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/openmm</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/openmm</td>
      <td>1.8.1</td>
      <td>pdbfixer</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>491</td>
      <td>491</td>
      <td>52</td>
      <td>52</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>491</td>
      <td>491</td>
      <td>52</td>
      <td>52</td>
    </tr>
    <tr>
      <td></td>
      <td>openms</td>
      <td>AccurateMassSearch, AssayGeneratorMetabo, BaselineFilter, CVInspector, ClusterMassTraces, ClusterMassTracesByPrecursor, CometAdapter, ConsensusID, ConsensusMapNormalizer, DTAExtractor, DatabaseFilter, DatabaseSuitability, DeMeanderize, DecoyDatabase, Digestor, DigestorMotif, EICExtractor, ERPairFinder, Epifany, ExternalCalibration, FalseDiscoveryRate, FeatureFinderCentroided, FeatureFinderIdentification, FeatureFinderIsotopeWavelet, FeatureFinderMRM, FeatureFinderMetabo, FeatureFinderMetaboIdent, FeatureFinderMultiplex, FeatureLinkerLabeled, FeatureLinkerUnlabeled, FeatureLinkerUnlabeledKD, FeatureLinkerUnlabeledQT, FileConverter, FileFilter, FileInfo, FileMerger, FuzzyDiff, GNPSExport, HighResPrecursorMassCorrector, IDConflictResolver, IDExtractor, IDFileConverter, IDFilter, IDMapper, IDMassAccuracy, IDMerger, IDPosteriorErrorProbability, IDRTCalibration, IDRipper, IDScoreSwitcher, IDSplitter, InternalCalibration, IsobaricAnalyzer, JSONExporter, LuciphorAdapter, MRMMapper, MRMPairFinder, MRMTransitionGroupPicker, MSFraggerAdapter, MSGFPlusAdapter, MSstatsConverter, MapAlignerIdentification, MapAlignerPoseClustering, MapAlignerSpectrum, MapAlignerTreeGuided, MapNormalizer, MapRTTransformer, MapStatistics, MascotAdapter, MascotAdapterOnline, MassCalculator, MassTraceExtractor, MetaProSIP, MetaboliteSpectralMatcher, MultiplexResolver, MzMLSplitter, MzTabExporter, NoiseFilterGaussian, NoiseFilterSGolay, NovorAdapter, NucleicAcidSearchEngine, OpenMSDatabasesInfo, OpenMSInfo, OpenPepXL, OpenPepXLLF, OpenSwathAnalyzer, OpenSwathAssayGenerator, OpenSwathChromatogramExtractor, OpenSwathConfidenceScoring, OpenSwathDIAPreScoring, OpenSwathDecoyGenerator, OpenSwathFeatureXMLToTSV, OpenSwathFileSplitter, OpenSwathMzMLFileCacher, OpenSwathRTNormalizer, OpenSwathRewriteToFeatureXML, OpenSwathWorkflow, PSMFeatureExtractor, PeakPickerHiRes, PeakPickerIterative, PeakPickerWavelet, PeptideIndexer, PercolatorAdapter, PhosphoScoring, PrecursorMassCorrector, ProteinInference, ProteinQuantifier, ProteinResolver, QCCalculator, QCEmbedder, QCExporter, QCExtractor, QCImporter, QCMerger, QCShrinker, QualityControl, RNADigestor, RNAMassCalculator, RNPxlSearch, RNPxlXICFilter, SageAdapter, SeedListGenerator, SemanticValidator, SequenceCoverageCalculator, SimpleSearchEngine, SiriusAdapter, SpecLibCreator, SpecLibSearcher, SpectraFilterBernNorm, SpectraFilterMarkerMower, SpectraFilterNLargest, SpectraFilterNormalizer, SpectraFilterParentPeakMower, SpectraFilterScaler, SpectraFilterSqrtMower, SpectraFilterThresholdMower, SpectraFilterWindowMower, SpectraMerger, SpectraSTSearchAdapter, StaticModification, TICCalculator, TOFCalibration, TargetedFileConverter, TextExporter, TriqlerConverter, XFDR, XMLValidator, XTandemAdapter</td>
      <td>OpenMS Suite for LC/MS data management and analyses</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.openms.de/</td>
      <td>Proteomics</td>
      <td>openms</td>
      <td>2024-04-09</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/openms</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/openms</td>
      <td>3.1</td>
      <td>openms</td>
      <td>3.2.0</td>
      <td></td>
      <td></td>
      <td>8</td>
      <td>34</td>
      <td>144</td>
      <td>0</td>
      <td>8</td>
      <td>34</td>
      <td>144</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>18</td>
      <td>141</td>
      <td>124</td>
      <td>0</td>
      <td>32169</td>
      <td>108654</td>
      <td>3316</td>
      <td>4889</td>
      <td>50</td>
      <td>50</td>
      <td>15</td>
      <td>15</td>
      <td>940</td>
      <td>940</td>
      <td>245</td>
      <td>245</td>
      <td>33159</td>
      <td>109644</td>
      <td>3576</td>
      <td>5149</td>
    </tr>
    <tr>
      <td></td>
      <td>packmol</td>
      <td>packmol</td>
      <td>PACKMOL is a package for creating starting structures for Molecular Dynamics simulations</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://m3g.iqm.unicamp.br/packmol/home.shtml</td>
      <td>Molecular Dynamics, Computational chemistry</td>
      <td>packmol</td>
      <td>2018-10-04</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/buildtools/packmol</td>
      <td>18.169.1</td>
      <td>packmol</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>282</td>
      <td>295</td>
      <td>32</td>
      <td>33</td>
      <td>420</td>
      <td>420</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>702</td>
      <td>715</td>
      <td>40</td>
      <td>41</td>
    </tr>
    <tr>
      <td></td>
      <td>pathifier</td>
      <td>pathifier</td>
      <td>pathifier</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://</td>
      <td>Transcriptomics, Statistics</td>
      <td>pathifier</td>
      <td>2019-11-08</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/pathifier</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/pathifier</td>
      <td>1.40.0</td>
      <td>bioconductor-pathifier</td>
      <td>1.40.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>245</td>
      <td>245</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>245</td>
      <td>245</td>
      <td>10</td>
      <td>10</td>
    </tr>
    <tr>
      <td></td>
      <td>pathwaymatcher</td>
      <td>reactome_pathwaymatcher</td>
      <td>Reactome Pathway Matcher</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/LuisFranciscoHS/PathwayMatcher</td>
      <td>Proteomics</td>
      <td>reactome_pathwaymatcher</td>
      <td>2018-06-08</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pathwaymatcher</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pathwaymatcher</td>
      <td></td>
      <td>pathwaymatcher</td>
      <td>1.9.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>174</td>
      <td>244</td>
      <td>65</td>
      <td>76</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>174</td>
      <td>244</td>
      <td>65</td>
      <td>76</td>
    </tr>
    <tr>
      <td></td>
      <td>pep_pointer</td>
      <td>pep_pointer</td>
      <td>PepPointer categorizes peptides by their genomic coordinates.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Genomic Interval Operations, Proteomics</td>
      <td>pep_pointer</td>
      <td>2017-12-19</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pep_pointer</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pep_pointer</td>
      <td>0.1.3+galaxy1</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>506</td>
      <td>511</td>
      <td>54</td>
      <td>55</td>
      <td>21</td>
      <td>21</td>
      <td>12</td>
      <td>12</td>
      <td>163</td>
      <td>163</td>
      <td>8</td>
      <td>8</td>
      <td>690</td>
      <td>695</td>
      <td>74</td>
      <td>75</td>
    </tr>
    <tr>
      <td></td>
      <td>pepquery</td>
      <td>pepquery</td>
      <td>A peptide-centric MS search engine for novel peptide identification and validation.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://pepquery.org</td>
      <td>Proteomics</td>
      <td>pepquery</td>
      <td>2020-01-22</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pepquery</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pepquery</td>
      <td>1.6.2</td>
      <td>pepquery</td>
      <td>2.0.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>7047</td>
      <td>7047</td>
      <td>76</td>
      <td>76</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7047</td>
      <td>7047</td>
      <td>76</td>
      <td>76</td>
    </tr>
    <tr>
      <td></td>
      <td>pepquery2</td>
      <td>pepquery2, pepquery2_index, pepquery2_show_sets</td>
      <td>PepQuery2 peptide-centric MS search for peptide identification and validation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://pepquery.org</td>
      <td>Proteomics</td>
      <td>pepquery2</td>
      <td>2022-10-02</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pepquery2</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pepquery2</td>
      <td>2.0.2</td>
      <td>pepquery</td>
      <td>2.0.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>1222</td>
      <td>1222</td>
      <td>28</td>
      <td>28</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1222</td>
      <td>1222</td>
      <td>28</td>
      <td>28</td>
    </tr>
    <tr>
      <td></td>
      <td>peptide_genomic_coordinate</td>
      <td>peptide_genomic_coordinate</td>
      <td>Gets genomic coordinate of peptides based on the information in mzsqlite and genomic mapping sqlite files</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>peptide_genomic_coordinate</td>
      <td>2019-03-29</td>
      <td>galaxyp</td>
      <td></td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/peptide_genomic_coordinate</td>
      <td>1.0.0</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>472</td>
      <td>472</td>
      <td>56</td>
      <td>56</td>
      <td>16</td>
      <td>16</td>
      <td>10</td>
      <td>10</td>
      <td>149</td>
      <td>149</td>
      <td>5</td>
      <td>5</td>
      <td>637</td>
      <td>637</td>
      <td>71</td>
      <td>71</td>
    </tr>
    <tr>
      <td></td>
      <td>peptideshaker</td>
      <td>fasta_cli, ident_params, peptide_shaker, search_gui</td>
      <td>PeptideShaker and SearchGUI</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://compomics.github.io</td>
      <td>Proteomics</td>
      <td>peptideshaker</td>
      <td>2021-04-02</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/peptideshaker</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/peptideshaker</td>
      <td></td>
      <td>searchgui</td>
      <td>4.3.11</td>
      <td></td>
      <td></td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>4</td>
      <td>2</td>
      <td>0</td>
      <td>11986</td>
      <td>17898</td>
      <td>1046</td>
      <td>1153</td>
      <td>1314</td>
      <td>1314</td>
      <td>389</td>
      <td>389</td>
      <td>3034</td>
      <td>3034</td>
      <td>118</td>
      <td>118</td>
      <td>16334</td>
      <td>22246</td>
      <td>1553</td>
      <td>1660</td>
    </tr>
    <tr>
      <td></td>
      <td>pepxml_to_xls</td>
      <td></td>
      <td>Convert PepXML to Tabular</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>pepxml_to_xls</td>
      <td>2015-01-24</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pepxml_to_xls</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pepxml_to_xls</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>percolator</td>
      <td>batched_set_list_creator, percolator, percolator_input_converters, pout2mzid</td>
      <td>Percolator</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>percolator</td>
      <td>2017-03-03</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tools/percolator</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/percolator</td>
      <td>3.5</td>
      <td>percolator</td>
      <td>3.7.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>338</td>
      <td>378</td>
      <td>15</td>
      <td>16</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1059</td>
      <td>1059</td>
      <td>13</td>
      <td>13</td>
      <td>1397</td>
      <td>1437</td>
      <td>28</td>
      <td>29</td>
    </tr>
    <tr>
      <td></td>
      <td>pi_db_tools</td>
      <td>calc_delta_pi, pi_db_split, pi_dbspec_align</td>
      <td>HiRIEF tools</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>hirieftools</td>
      <td>2017-04-11</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tools/pi_db_tools</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pi_db_tools</td>
      <td>1.3</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>pindel</td>
      <td>pindel</td>
      <td>Pindel detects genome-wide structural variation.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://artbio.fr</td>
      <td>Variant Analysis</td>
      <td>pindel</td>
      <td>2021-09-29</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/pindel</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/pindel</td>
      <td>0.2.5b9</td>
      <td>pindel</td>
      <td>0.2.5b9</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>pmd_fdr</td>
      <td>pmd_fdr</td>
      <td>Calculate Precursor Mass Discrepancy (PMD) for MS/MS</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/slhubler/PMD-FDR-for-Galaxy-P</td>
      <td>Proteomics</td>
      <td>pmd_fdr</td>
      <td>2019-10-01</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pmd_fdr</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pmd_fdr</td>
      <td>1.4.0</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>positions2snv_alignment</td>
      <td>positions2snv_alignment</td>
      <td>Generate alignment of SNVs from SNVPhyl variant table.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://snvphyl.readthedocs.io/en/latest/</td>
      <td>Variant Analysis</td>
      <td>positions2snv_alignment</td>
      <td>2016-11-04</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy/tree/development/tools/snvphyl-tools/positions2snv_alignment</td>
      <td>1.8.2</td>
      <td>snvphyl-tools</td>
      <td>1.8.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>positions2snv_invariant_alignment</td>
      <td>positions2snv_invariant_alignment</td>
      <td>Generate alignment of SNVs and non-variant positions from SNVPhyl variant table.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://snvphyl.readthedocs.io/en/latest/</td>
      <td>Variant Analysis</td>
      <td>positions2snv_invariant_alignment</td>
      <td>2016-11-07</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy/tree/development/tools/snvphyl-tools/positions2snv_invariant_alignment</td>
      <td>1.8.2</td>
      <td>snvphyl-tools</td>
      <td>1.8.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>probecoverage</td>
      <td>probecoverage</td>
      <td>computes and plots read coverage of genomic regions by sequencing datasets</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Sequence Analysis, Genomic Interval Operations, Graphics, Statistics</td>
      <td>probecoverage</td>
      <td>2017-09-22</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/probecoverage</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/probecoverage</td>
      <td>0.22.0</td>
      <td>pysam</td>
      <td>0.22.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>proteinpilot</td>
      <td>convert_windows_newlines, proteinpilot, proteinpilot_group_extractor, proteinpilot_tabular, proteinpilot_xml</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td>proteinpilot</td>
      <td>2015-01-24</td>
      <td>galaxyp</td>
      <td></td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteinpilot</td>
      <td>0.1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_joinquantpepionswithproteins</td>
      <td>proteomiqon_joinquantpepionswithproteins</td>
      <td>The tool JoinQuantPepIonsWithProteins combines results from ProteinInference and PSMBasedQuantification.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/JoinQuantPepIonsWithProteins.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_joinquantpepionswithproteins</td>
      <td>2021-09-16</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_joinquantpepionswithproteins</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_joinquantpepionswithproteins</td>
      <td>0.0.1</td>
      <td>proteomiqon-joinquantpepionswithproteins</td>
      <td>0.0.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>366</td>
      <td>366</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>366</td>
      <td>366</td>
      <td>4</td>
      <td>4</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_labeledproteinquantification</td>
      <td>proteomiqon_labeledproteinquantification</td>
      <td>The tool LabeledProteinQuantification estimates protein abundances using quantified peptide ions.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/LabeledProteinQuantification.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_labeledproteinquantification</td>
      <td>2021-09-16</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_labeledproteinquantification</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_labeledproteinquantification</td>
      <td>0.0.1</td>
      <td>proteomiqon-labeledproteinquantification</td>
      <td>0.0.3</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>16</td>
      <td>16</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>16</td>
      <td>16</td>
      <td>8</td>
      <td>8</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_labelfreeproteinquantification</td>
      <td>proteomiqon_labelfreeproteinquantification</td>
      <td>The tool LabelFreeProteinQuantification estimates protein abundances using quantified peptide ions.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/LabelfreeProteinQuantification.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_labelfreeproteinquantification</td>
      <td>2021-10-13</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_labelfreeproteinquantification</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_labelfreeproteinquantification</td>
      <td>0.0.1</td>
      <td>proteomiqon-labelfreeproteinquantification</td>
      <td>0.0.3</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>4</td>
      <td>4</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_mzmltomzlite</td>
      <td>proteomiqon_mzmltomzlite</td>
      <td>The tool MzMLToMzLite allows to convert mzML files to mzLite files.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/MzMLToMzLite.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_mzmltomzlite</td>
      <td>2021-06-30</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomIQon_MzMLToMzLite</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_mzmltomzlite</td>
      <td>0.0.8</td>
      <td>proteomiqon-mzmltomzlite</td>
      <td>0.0.8</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>721</td>
      <td>721</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>721</td>
      <td>721</td>
      <td>8</td>
      <td>8</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_peptidedb</td>
      <td>proteomiqon_peptidedb</td>
      <td>The tool ProteomIQon PeptideDB creates a peptide database in the SQLite format.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/PeptideDB.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_peptidedb</td>
      <td>2021-07-04</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_peptidedb</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_peptidedb</td>
      <td>0.0.7</td>
      <td>proteomiqon-peptidedb</td>
      <td>0.0.7</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>96</td>
      <td>96</td>
      <td>12</td>
      <td>12</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>96</td>
      <td>96</td>
      <td>12</td>
      <td>12</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_peptidespectrummatching</td>
      <td>proteomiqon_peptidespectrummatching</td>
      <td>Given raw an MS run in the mzLite format, this tool iterates across all MS/MS scans, determines precursor charge states and possible peptide spectrum matches using reimplementations of SEQUEST,Andromeda and XTandem.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/PeptideSpectrumMatching.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_peptidespectrummatching</td>
      <td>2021-07-15</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_peptidespectrummatching</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_peptidespectrummatching</td>
      <td>0.0.7</td>
      <td>proteomiqon-peptidespectrummatching</td>
      <td>0.0.7</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>686</td>
      <td>686</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>686</td>
      <td>686</td>
      <td>5</td>
      <td>5</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_proteininference</td>
      <td>proteomiqon_proteininference</td>
      <td>MS-based shotgun proteomics estimates protein abundances using a proxy: peptides. The process of 'Protein Inference' is concerned with the mapping of identified peptides to the proteins they putatively originated from.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/ProteinInference.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_proteininference</td>
      <td>2021-07-09</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_proteininference</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_proteininference</td>
      <td>0.0.7</td>
      <td>proteomiqon-proteininference</td>
      <td>0.0.7</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>89</td>
      <td>89</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>89</td>
      <td>89</td>
      <td>6</td>
      <td>6</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_psmbasedquantification</td>
      <td>proteomiqon_psmbasedquantification</td>
      <td>The PSMBasedQuantification tool was designed to allow label-free quantification as well as quantification of full metabolic labeled samples.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/PSMBasedQuantification.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_psmbasedquantification</td>
      <td>2021-07-09</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_psmbasedquantification</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_psmbasedquantification</td>
      <td>0.0.8</td>
      <td>proteomiqon-psmbasedquantification</td>
      <td>0.0.9</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>604</td>
      <td>604</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>604</td>
      <td>604</td>
      <td>4</td>
      <td>4</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_psmstatistics</td>
      <td>proteomiqon_psmstatistics</td>
      <td>The PSMStatistics tool utilizes semi supervised machine learning techniques to integrate search engine scores as well as the mentioned quality scores into one single consensus score.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/PSMStatistics.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_psmstatistics</td>
      <td>2021-07-09</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_psmstatistics</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_psmstatistics</td>
      <td>0.0.8</td>
      <td>proteomiqon-psmstatistics</td>
      <td>0.0.8</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>694</td>
      <td>694</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>694</td>
      <td>694</td>
      <td>6</td>
      <td>6</td>
    </tr>
    <tr>
      <td></td>
      <td>proteore_venn_diagram</td>
      <td>proteore_venn_diagram</td>
      <td>ProteoRE JVenn Diagram</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>proteore_venn_diagram</td>
      <td>2021-05-17</td>
      <td>galaxyp</td>
      <td></td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteore_venn_diagram</td>
      <td>2021.06.08</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>15</td>
      <td>15</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>15</td>
      <td>15</td>
      <td>1</td>
      <td>1</td>
    </tr>
    <tr>
      <td></td>
      <td>protxml_to_xls</td>
      <td>protxml_to_xls</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td>protxml_to_xls</td>
      <td>2015-01-24</td>
      <td>galaxyp</td>
      <td></td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/protxml_to_xls</td>
      <td>0.1.0</td>
      <td>trans_proteomic_pipeline</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>psm2sam</td>
      <td>PSMtoSAM</td>
      <td>PSM to SAM</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/proBAMr.html</td>
      <td>Proteomics</td>
      <td>psm_to_sam</td>
      <td>2017-10-06</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tools/bumbershoot/psm2sam</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/probam_suite/psm2sam</td>
      <td>1.3.2.1</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>psm_eval</td>
      <td>psm_eval</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td>psm_eval</td>
      <td>2015-01-24</td>
      <td>galaxyp</td>
      <td></td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/psm_eval</td>
      <td>0.1.0</td>
      <td>binaries_for_psm_eval</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>psm_validation</td>
      <td>psmvalidator</td>
      <td>Validate PSM from Ion Fragmentation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproteomics/psm_fragments.git</td>
      <td>Proteomics</td>
      <td>psm_validation</td>
      <td>2020-10-13</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/psm_validation</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/psm_validation</td>
      <td>1.0.3</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>20</td>
      <td>20</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>20</td>
      <td>20</td>
      <td>4</td>
      <td>4</td>
    </tr>
    <tr>
      <td></td>
      <td>psy-maps</td>
      <td>psy_maps</td>
      <td>Visualization of regular geographical data on a map with psyplot</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/Chilipp/psy-maps</td>
      <td>Visualization, Climate Analysis</td>
      <td>psy_maps</td>
      <td>2019-10-04</td>
      <td>climate</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/psy-maps</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/psy-maps</td>
      <td>1.3.1</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>333</td>
      <td>502</td>
      <td>62</td>
      <td>62</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>333</td>
      <td>502</td>
      <td>62</td>
      <td>62</td>
    </tr>
    <tr>
      <td></td>
      <td>pyprophet</td>
      <td>pyprophet_export, pyprophet_merge, pyprophet_peptide, pyprophet_protein, pyprophet_score, pyprophet_subsample</td>
      <td>Semi-supervised learning and scoring of OpenSWATH results.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/PyProphet/pyprophet</td>
      <td>Proteomics</td>
      <td></td>
      <td>2015-02-06</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pyprophet</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pyprophet</td>
      <td>2.1.4</td>
      <td>pyprophet</td>
      <td>2.2.5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>5</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>2626</td>
      <td>2626</td>
      <td>150</td>
      <td>150</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>609</td>
      <td>609</td>
      <td>9</td>
      <td>9</td>
      <td>3235</td>
      <td>3235</td>
      <td>159</td>
      <td>159</td>
    </tr>
    <tr>
      <td></td>
      <td>pyscenic</td>
      <td>pyscenic_aucell, pyscenic_binarize, pyscenic_ctx, pyscenic_grn</td>
      <td>PySCENIC scripts based on usage at https://pyscenic.readthedocs.io/</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Sequence Analysis</td>
      <td>suite_pyscenic</td>
      <td>2024-08-20</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/pyscenic</td>
      <td>0.12.1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>pyteomics</td>
      <td>mztab2tsv</td>
      <td>Tools using the pyteomics library</td>
      <td>pyteomics</td>
      <td>pyteomics</td>
      <td></td>
      <td>Pyteomics</td>
      <td>Framework for proteomics data analysis, supporting mzML, MGF, pepXML and more.</td>
      <td>Protein identification</td>
      <td>Proteomics, Proteomics experiment</td>
      <td>To update</td>
      <td>https://pyteomics.readthedocs.io/en/latest/</td>
      <td>Proteomics, Metabolomics</td>
      <td>pyteomics</td>
      <td>2021-01-15</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pyteomics</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pyteomics</td>
      <td>4.4.1</td>
      <td>pyteomics</td>
      <td>4.7.5</td>
      <td>Protein identification</td>
      <td>Proteomics, Proteomics experiment</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>92</td>
      <td>92</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>92</td>
      <td>92</td>
      <td>4</td>
      <td>4</td>
    </tr>
    <tr>
      <td></td>
      <td>pyvo_integration</td>
      <td>astronomical_archives</td>
      <td>Astronomical archives tools contains tools for querying and fetching resources from astronomical archives into Galaxy</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Data Source</td>
      <td>astronomicalarchivestool</td>
      <td>2023-08-26</td>
      <td>astroteam</td>
      <td>https://github.com/esg-epfl-apc/tools-astro/tree/main/tools/</td>
      <td>https://github.com/esg-epfl-apc/tools-astro/tree/main/tools/archives/pyvo_integration</td>
      <td>0.10.0</td>
      <td>astropy</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>53</td>
      <td>53</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>53</td>
      <td>53</td>
      <td>6</td>
      <td>6</td>
    </tr>
    <tr>
      <td></td>
      <td>qcxms</td>
      <td>qcxms_getres, qcxms_neutral_run, qcxms_production_run</td>
      <td>QCxMS is a quantum chemical (QC) based program that enables users to calculate mass spectra (MS) using Born-Oppenheimer Molecular Dynamics (MD).</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/grimme-lab/QCxMS</td>
      <td>Computational chemistry, Molecular Dynamics</td>
      <td>QCxMS</td>
      <td>2023-10-21</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/qcxms</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/qcxms</td>
      <td>5.2.1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>11246</td>
      <td>11246</td>
      <td>13</td>
      <td>13</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>11246</td>
      <td>11246</td>
      <td>13</td>
      <td>13</td>
    </tr>
    <tr>
      <td></td>
      <td>quantp</td>
      <td>quantp</td>
      <td>Correlation between protein and transcript abundance</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>quantp</td>
      <td>2018-09-14</td>
      <td>galaxyp</td>
      <td></td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/quantp</td>
      <td>1.1.2</td>
      <td>r-data.table</td>
      <td>1.11.6</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>197</td>
      <td>241</td>
      <td>17</td>
      <td>22</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>197</td>
      <td>241</td>
      <td>17</td>
      <td>22</td>
    </tr>
    <tr>
      <td></td>
      <td>quantwiz_iq</td>
      <td>quantwiz_iq</td>
      <td>Isobaric Quantitation using QuantWiz-IQ</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://sourceforge.net/projects/quantwiz/</td>
      <td>Proteomics</td>
      <td>quantwiz_iq</td>
      <td>2020-01-21</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/quantwiz_iq</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/quantwiz_iq</td>
      <td>2.0</td>
      <td>quantwiz-iq</td>
      <td>2.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>32</td>
      <td>32</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>32</td>
      <td>32</td>
      <td>3</td>
      <td>3</td>
    </tr>
    <tr>
      <td></td>
      <td>query</td>
      <td>query</td>
      <td>Execute an SQL statement on a set of tables</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>query</td>
      <td>2020-12-09</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/query</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/query</td>
      <td>0.2</td>
      <td>click</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>qupath_roi_splitter</td>
      <td>qupath_roi_splitter</td>
      <td>Split ROI coordinates of QuPath TMA annotation by cell type</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/npinter/ROIsplitter</td>
      <td>Imaging</td>
      <td>qupath_roi_splitter</td>
      <td>2023-04-12</td>
      <td>galaxyp</td>
      <td>hhttps://github.com/npinter/ROIsplitter</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/qupath_roi_splitter</td>
      <td>0.3.2</td>
      <td>geojson</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>172</td>
      <td>172</td>
      <td>9</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>172</td>
      <td>172</td>
      <td>9</td>
      <td>9</td>
    </tr>
    <tr>
      <td></td>
      <td>ramclustr</td>
      <td>ramclustr, ramclustr_define_experiment</td>
      <td></td>
      <td>ramclustr</td>
      <td>ramclustr</td>
      <td></td>
      <td>RAMClustR</td>
      <td>A feature clustering algorithm for non-targeted mass spectrometric metabolomics data.</td>
      <td>Imputation, Standardisation and normalisation, Clustering, Correlation</td>
      <td>Metabolomics</td>
      <td>To update</td>
      <td>https://rdrr.io/cran/RAMClustR/</td>
      <td>Metabolomics</td>
      <td></td>
      <td>2023-05-31</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/ramclustr</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/ramclustr</td>
      <td>1.3.0</td>
      <td>r-ramclustr</td>
      <td>1.3.1</td>
      <td>Imputation, Standardisation and normalisation, Clustering, Correlation</td>
      <td>Metabolomics</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>58</td>
      <td>58</td>
      <td>22</td>
      <td>22</td>
      <td>29</td>
      <td>29</td>
      <td>14</td>
      <td>14</td>
      <td>344</td>
      <td>344</td>
      <td>4</td>
      <td>4</td>
      <td>431</td>
      <td>431</td>
      <td>40</td>
      <td>40</td>
    </tr>
    <tr>
      <td></td>
      <td>rawtools</td>
      <td>rawtools</td>
      <td>Raw Tools</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/kevinkovalchik/RawTools</td>
      <td>Proteomics</td>
      <td>rawtools</td>
      <td>2019-02-28</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/rawtools</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/rawtools</td>
      <td></td>
      <td>rawtools</td>
      <td>2.0.4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>166</td>
      <td>179</td>
      <td>32</td>
      <td>33</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>414</td>
      <td>414</td>
      <td>13</td>
      <td>13</td>
      <td>580</td>
      <td>593</td>
      <td>45</td>
      <td>46</td>
    </tr>
    <tr>
      <td></td>
      <td>rcx-tk</td>
      <td>rcx-tk</td>
      <td>Tools to process metadata or alkane files.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/RECETOX/rcx-tk</td>
      <td>Metabolomics</td>
      <td>rcx_tk</td>
      <td>2024-08-01</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/rcx-tk/</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/rcx-tk</td>
      <td>0.1.0+galaxy0</td>
      <td>rcx-tk</td>
      <td>0.1.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>recetox_aplcms</td>
      <td>recetox_aplcms_align_features, recetox_aplcms_compute_clusters, recetox_aplcms_compute_template, recetox_aplcms_correct_time, recetox_aplcms_generate_feature_table, recetox_aplcms_merge_known_table, recetox_aplcms_recover_weaker_signals, recetox_aplcms_remove_noise</td>
      <td>Peak detection tool for HRMS profile data.</td>
      <td>recetox-aplcms</td>
      <td>recetox-aplcms</td>
      <td></td>
      <td>recetox-aplcms</td>
      <td>recetox-aplcms is a tool for peak detection in mass spectrometry data. The tool performs (1) noise removal, (2) peak detection, (3) retention time drift correction, (4) peak alignment and (5) weaker signal recovery as well as (6) suspect screening.</td>
      <td>Chromatographic alignment, Quantification, Peak detection, Feature extraction, Alignment</td>
      <td>Metabolomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/RECETOX/recetox-aplcms</td>
      <td>Metabolomics</td>
      <td>recetox-aplcms</td>
      <td>2023-05-18</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/recetox_aplcms</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/recetox_aplcms</td>
      <td>0.13.3</td>
      <td>r-recetox-aplcms</td>
      <td>0.13.3</td>
      <td>Chromatographic alignment, Quantification, Peak detection, Feature extraction, Alignment</td>
      <td>Metabolomics</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>862</td>
      <td>862</td>
      <td>12</td>
      <td>12</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>48</td>
      <td>48</td>
      <td>1</td>
      <td>1</td>
      <td>910</td>
      <td>910</td>
      <td>13</td>
      <td>13</td>
    </tr>
    <tr>
      <td></td>
      <td>recetox_msfinder</td>
      <td>recetox_msfinder</td>
      <td></td>
      <td>recetox-msfinder</td>
      <td>recetox-msfinder</td>
      <td></td>
      <td>recetox-msfinder</td>
      <td>This is a modified copy of MS-FINDER with source code modifications to make the tool accessible in Galaxy.MS-FINDER - software for structure elucidation of unknown spectra with hydrogen rearrangement (HR) rulesThe program supports molecular formula prediction, metabolie class prediction, and structure elucidation for EI-MS and MS/MS spectra, and the assembly is licensed under the CC-BY 4.0.</td>
      <td>Annotation</td>
      <td>Metabolomics</td>
      <td>To update</td>
      <td>https://github.com/RECETOX/recetox-msfinder</td>
      <td>Metabolomics</td>
      <td></td>
      <td>2022-12-02</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/recetox_msfinder</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/recetox_msfinder</td>
      <td>v3.5.2</td>
      <td></td>
      <td></td>
      <td>Annotation</td>
      <td>Metabolomics</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>47</td>
      <td>47</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>47</td>
      <td>47</td>
      <td>4</td>
      <td>4</td>
    </tr>
    <tr>
      <td></td>
      <td>recetox_xmsannotator</td>
      <td>recetox_xmsannotator_advanced</td>
      <td></td>
      <td>recetox-xmsannotator</td>
      <td>recetox-xmsannotator</td>
      <td></td>
      <td>recetox-xMSannotator</td>
      <td>Annotation tool for untargeted LCMS1 data. Uses a database and adduct list for compound annotation and intensity networks, isotopic patterns and pathways for annotation scoring.</td>
      <td>Expression profile pathway mapping, Structure comparison, Isotopic distributions calculation, Annotation</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/RECETOX/recetox-xMSannotator</td>
      <td>Metabolomics</td>
      <td></td>
      <td>2021-07-21</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/recetox-xmsannotator</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/recetox_xmsannotator</td>
      <td>0.10.0</td>
      <td>r-recetox-xmsannotator</td>
      <td>0.10.0</td>
      <td>Expression profile pathway mapping, Structure comparison, Isotopic distributions calculation, Annotation</td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>93</td>
      <td>93</td>
      <td>2</td>
      <td>2</td>
      <td>93</td>
      <td>93</td>
      <td>2</td>
      <td>2</td>
    </tr>
    <tr>
      <td></td>
      <td>regex_find_replace</td>
      <td>regex1, regexColumn1</td>
      <td>Use python regular expressions to find and replace text</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>regex_find_replace</td>
      <td>2017-01-13</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/regex_find_replace</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/regex_find_replace</td>
      <td>1.0.3</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>78648</td>
      <td>78834</td>
      <td>1144</td>
      <td>1156</td>
      <td>18063</td>
      <td>18063</td>
      <td>870</td>
      <td>870</td>
      <td>4455</td>
      <td>6051</td>
      <td>194</td>
      <td>289</td>
      <td>101166</td>
      <td>102948</td>
      <td>2208</td>
      <td>2315</td>
    </tr>
    <tr>
      <td></td>
      <td>regionalgam</td>
      <td>regionalgam_ab_index, regionalgam_autocor_acf, regionalgam_flight_curve, regionalgam_glmmpql, regionalgam_gls_adjusted, regionalgam_gls, regionalgam_plot_trend</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/RetoSchmucki/regionalGAM</td>
      <td>Ecology</td>
      <td></td>
      <td>2019-06-18</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/regionalgam</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/regionalgam</td>
      <td>1.5</td>
      <td>r-mgcv</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>438</td>
      <td>542</td>
      <td>128</td>
      <td>157</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>438</td>
      <td>542</td>
      <td>128</td>
      <td>157</td>
    </tr>
    <tr>
      <td></td>
      <td>rem_complex</td>
      <td>rem_complex</td>
      <td>Removes molecular coordination complexes.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/RECETOX/galaxytools</td>
      <td>Metabolomics</td>
      <td>rem_complex</td>
      <td>2023-11-21</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/rem_complex</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/rem_complex</td>
      <td>1.0.0</td>
      <td>pandas</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>18</td>
      <td>18</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>18</td>
      <td>18</td>
      <td>2</td>
      <td>2</td>
    </tr>
    <tr>
      <td></td>
      <td>rename_annotated_feature</td>
      <td>rename_annotated_feature</td>
      <td>Update column names in an abundance table using a annotation table with spectral matching results</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/RECETOX/galaxytools/</td>
      <td>Metabolomics</td>
      <td>rename_annotated_feature</td>
      <td>2024-05-13</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/rename_annotated_feature</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/rename_annotated_feature</td>
      <td>1.0.0</td>
      <td>pandas</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>repenrich</td>
      <td>edger-repenrich, repenrich</td>
      <td>Repeat element profiling</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/repenrich</td>
      <td>Transcriptomics</td>
      <td>repenrich</td>
      <td>2017-05-31</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/repenrich</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/repenrich</td>
      <td>2.31.1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1725</td>
      <td>1790</td>
      <td>56</td>
      <td>57</td>
      <td>1725</td>
      <td>1790</td>
      <td>56</td>
      <td>57</td>
    </tr>
    <tr>
      <td></td>
      <td>repenrich2</td>
      <td>edger-repenrich2, repenrich2</td>
      <td>Repeat element profiling using bowtie2 aligner</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/repenrich2</td>
      <td>Transcriptomics</td>
      <td>repenrich2</td>
      <td>2024-04-20</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/repenrich2</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/repenrich2</td>
      <td>2.31.1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>retip</td>
      <td>retip_apply, retip_descriptors, retip_filter_rt, retip_train</td>
      <td></td>
      <td>retip</td>
      <td>retip</td>
      <td></td>
      <td>Retip</td>
      <td>Retention Time Prediction for Compound Annotation in Untargeted Metabolomics.Retip is an R package for predicting Retention Time (RT) for small molecules in a high pressure liquid chromatography (HPLC) Mass Spectrometry analysis.Retip - Retention Time prediction for Metabolomics.Retip: Retention Time Prediction for Compound Annotation in Untargeted Metabolomics Paolo Bonini, Tobias Kind, Hiroshi Tsugawa, Dinesh Kumar Barupal, and Oliver Fiehn Analytical Chemistry 2020 92 (11), 7515-7522 DOI: 10.1021/acs.analchem.9b05765.</td>
      <td>Retention time prediction, Spectrum calculation, Deisotoping, Formatting, Deposition</td>
      <td>Metabolomics, Proteomics experiment, Machine learning, Cheminformatics, Chemistry</td>
      <td>To update</td>
      <td>https://github.com/PaoloBnn/Retip</td>
      <td>Metabolomics</td>
      <td></td>
      <td>2020-09-07</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/retip</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/retip</td>
      <td>0.5.4</td>
      <td></td>
      <td></td>
      <td>Retention time prediction, Spectrum calculation, Deisotoping, Formatting, Deposition</td>
      <td>Metabolomics, Proteomics experiment, Machine learning, Cheminformatics, Chemistry</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>retrieve_bold</td>
      <td>retrieve_bold</td>
      <td>Search a list of sequences in BOLD (Barcode of Life Data System) from specified taxa list and markers</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://anaconda.org/conda-forge/r-bold</td>
      <td>Ecology</td>
      <td>retrieve_bold</td>
      <td>2024-06-21</td>
      <td>ecology</td>
      <td>https://github.com/wpearman1996/MARES_database_pipeline/tree/master</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/retrieve_bold</td>
      <td>1.3.0</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
    </tr>
    <tr>
      <td></td>
      <td>retrieve_ensembl_bed</td>
      <td>retrieve_ensembl_bed</td>
      <td>Retrieve cDNA features from Ensembl REST API in BED format</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://rest.ensembl.org/</td>
      <td>Data Source</td>
      <td>retrieve_ensembl_bed</td>
      <td>2018-01-14</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteogenomics/retrieve_ensembl_bed</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteogenomics/retrieve_ensembl_bed</td>
      <td>0.1.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>revertR2orientationInBam</td>
      <td>revertR2orientationInBam</td>
      <td>Revert the mapped orientation of R2 mates in a bam.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>SAM</td>
      <td>revertr2orientationinbam</td>
      <td>2019-04-17</td>
      <td>lldelisle</td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/revertR2orientationInBam</td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/revertR2orientationInBam</td>
      <td>0.0.2</td>
      <td>samtools</td>
      <td>1.21</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>40</td>
      <td>40</td>
      <td>11</td>
      <td>11</td>
      <td>520</td>
      <td>520</td>
      <td>31</td>
      <td>31</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>560</td>
      <td>560</td>
      <td>42</td>
      <td>42</td>
    </tr>
    <tr>
      <td></td>
      <td>riassigner</td>
      <td>riassigner, riassigner_from_comment</td>
      <td></td>
      <td>riassigner</td>
      <td>riassigner</td>
      <td></td>
      <td>RIAssigner</td>
      <td>RIAssigner is a python tool for retention index (RI) computation for GC-MS data.</td>
      <td>Standardisation and normalisation</td>
      <td>Metabolomics, Compound libraries and screening, Data submission, annotation and curation</td>
      <td>Up-to-date</td>
      <td>https://github.com/RECETOX/RIAssigner</td>
      <td>Metabolomics</td>
      <td></td>
      <td>2021-08-19</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/riassigner</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/riassigner</td>
      <td>0.4.1</td>
      <td>riassigner</td>
      <td>0.4.1</td>
      <td>Standardisation and normalisation</td>
      <td>Metabolomics, Compound libraries and screening, Data submission, annotation and curation</td>
      <td>1</td>
      <td>2</td>
      <td>2</td>
      <td>1</td>
      <td>1</td>
      <td>2</td>
      <td>2</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>46</td>
      <td>46</td>
      <td>19</td>
      <td>19</td>
      <td>15</td>
      <td>15</td>
      <td>4</td>
      <td>4</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>62</td>
      <td>62</td>
      <td>24</td>
      <td>24</td>
    </tr>
    <tr>
      <td></td>
      <td>rmassbank</td>
      <td>rmassbank</td>
      <td>RMassBank is an R package for processing tandem MS files and building of MassBank records.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/MassBank/RMassBank</td>
      <td>Metabolomics</td>
      <td>rmassbank</td>
      <td>2021-02-25</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/rmassbank</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/rmassbank</td>
      <td>3.0.0</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>48</td>
      <td>48</td>
      <td>1</td>
      <td>1</td>
      <td>48</td>
      <td>48</td>
      <td>1</td>
      <td>1</td>
    </tr>
    <tr>
      <td></td>
      <td>rsem</td>
      <td>extract_transcript_to_gene_map_from_trinity, purgegtffrommultichromgenes, rsembowtie2, rsembowtie</td>
      <td>transcript quantification from RNA-Seq data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/deweylab/RSEM</td>
      <td>Transcriptomics, RNA</td>
      <td>rsem</td>
      <td>2018-03-28</td>
      <td>artbio</td>
      <td>https://github.com/artbio/tools-artbio/tree/master/tools/rsem</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/rsem</td>
      <td></td>
      <td>rsem</td>
      <td>1.3.3</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>363</td>
      <td>430</td>
      <td>158</td>
      <td>183</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>67</td>
      <td>146</td>
      <td>13</td>
      <td>39</td>
      <td>430</td>
      <td>576</td>
      <td>171</td>
      <td>222</td>
    </tr>
    <tr>
      <td></td>
      <td>salmon-kallisto-mtx-to-10x</td>
      <td>_salmon_kallisto_mtx_to_10x</td>
      <td>Transforms .mtx matrix and associated labels into a format compatible with tools expecting old-style 10X data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary</td>
      <td>Sequence Analysis</td>
      <td>salmon_kallisto_mtx_to_10x</td>
      <td>2019-11-07</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/develop/tools/salmon-kallisto-mtx-to-10x/.shed.yml</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/salmon-kallisto-mtx-to-10x</td>
      <td>0.0.1+galaxy6</td>
      <td>scipy</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>682</td>
      <td>682</td>
      <td>172</td>
      <td>172</td>
      <td>322</td>
      <td>322</td>
      <td>74</td>
      <td>74</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>1007</td>
      <td>1007</td>
      <td>249</td>
      <td>249</td>
    </tr>
    <tr>
      <td></td>
      <td>sambamba</td>
      <td>sambamba_sample_or_filter</td>
      <td>filter BAM/SAM on flags, fields, tags, and region, or down-sample, or slice BAM/SAM</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>SAM</td>
      <td>sambamba</td>
      <td>2020-05-19</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/sambamba</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/sambamba</td>
      <td>0.7.1+galaxy1</td>
      <td>sambamba</td>
      <td>1.0.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>684</td>
      <td>684</td>
      <td>17</td>
      <td>17</td>
      <td>684</td>
      <td>684</td>
      <td>17</td>
      <td>17</td>
    </tr>
    <tr>
      <td></td>
      <td>sashimi_plot</td>
      <td>sashimi_plot</td>
      <td>Generates a sashimi plot from bam files.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, Transcriptomics, Graphics, Visualization</td>
      <td>sashimi_plot</td>
      <td>2019-08-21</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/sashimi_plot</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/sashimi_plot</td>
      <td>0.1.1</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>sc3</td>
      <td>sc3_calc_biology, sc3_calc_consens, sc3_calc_dists, sc3_calc_transfs, sc3_estimate_k, sc3_kmeans, sc3_prepare</td>
      <td>De-composed SC3 functionality tools, based on https://github.com/ebi-gene-expression-group/bioconductor-sc3-scripts and SC3 1.8.0.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_sc3</td>
      <td>2018-11-05</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/sc3</td>
      <td>1.8.0</td>
      <td>sc3-scripts</td>
      <td>0.0.6</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>13</td>
      <td>13</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>13</td>
      <td>13</td>
      <td>10</td>
      <td>10</td>
    </tr>
    <tr>
      <td></td>
      <td>scaffold</td>
      <td>scaffold, scaffold_export</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td>scaffold</td>
      <td>2015-01-24</td>
      <td>galaxyp</td>
      <td></td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/scaffold</td>
      <td>0.1.0</td>
      <td>scaffold</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>scanpy</td>
      <td>anndata_ops, scanpy_filter_cells, scanpy_filter_genes, scanpy_find_cluster, scanpy_find_markers, scanpy_find_variable_genes, scanpy_integrate_bbknn, scanpy_integrate_combat, scanpy_integrate_harmony, scanpy_integrate_mnn, scanpy_plot_scrublet, scanpy_multiplet_scrublet, scanpy_compute_graph, scanpy_normalise_data, scanpy_parameter_iterator, scanpy_plot_embed, scanpy_plot_trajectory, scanpy_read_10x, scanpy_regress_variable, scanpy_run_diffmap, scanpy_run_dpt, scanpy_run_fdg, scanpy_run_paga, scanpy_run_pca, scanpy_run_tsne, scanpy_run_umap, scanpy_scale_data</td>
      <td>scanpy-scripts, command-line wrapper scripts around Scanpy.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://scanpy.readthedocs.io</td>
      <td>Transcriptomics, Sequence Analysis, RNA</td>
      <td>scanpy_scripts</td>
      <td>2020-09-29</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/develop/tools/tertiary-analysis/scanpy</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/scanpy</td>
      <td>1.9.3</td>
      <td>scanpy-scripts</td>
      <td>1.9.301</td>
      <td></td>
      <td></td>
      <td>27</td>
      <td>27</td>
      <td>27</td>
      <td>0</td>
      <td>27</td>
      <td>27</td>
      <td>27</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>14</td>
      <td>13</td>
      <td>0</td>
      <td>45501</td>
      <td>45666</td>
      <td>4832</td>
      <td>4868</td>
      <td>7304</td>
      <td>7304</td>
      <td>1846</td>
      <td>1846</td>
      <td>3106</td>
      <td>3106</td>
      <td>457</td>
      <td>457</td>
      <td>55911</td>
      <td>56076</td>
      <td>7135</td>
      <td>7171</td>
    </tr>
    <tr>
      <td></td>
      <td>scater</td>
      <td>scater_calculate_cpm, scater_calculate_qc_metrics, scater_filter, scater_is_outlier, scater_normalize, scater_read_10x_results</td>
      <td>De-composed Scater functionality tools, based on https://github.com/ebi-gene-expression-group/bioconductor-scater-scripts and Scater 1.8.4.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_scater</td>
      <td>2018-10-11</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/scater</td>
      <td>1.10.0</td>
      <td>scater-scripts</td>
      <td>0.0.5</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>2</td>
      <td>6</td>
      <td>0</td>
      <td>1</td>
      <td>2</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>381</td>
      <td>388</td>
      <td>99</td>
      <td>100</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>297</td>
      <td>297</td>
      <td>9</td>
      <td>9</td>
      <td>678</td>
      <td>685</td>
      <td>108</td>
      <td>109</td>
    </tr>
    <tr>
      <td></td>
      <td>sccaf</td>
      <td>run_sccaf, sccaf_asses, sccaf_asses_merger, sccaf_regress_out</td>
      <td>SCCAF: Single Cell Clustering Assessment Framework.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/sccaf/sccaf</td>
      <td>Transcriptomics</td>
      <td>SCCAF</td>
      <td>2019-10-20</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/develop/tools/tertiary-analysis/sccaf</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/sccaf</td>
      <td>0.0.9</td>
      <td>sccaf</td>
      <td>0.0.10</td>
      <td></td>
      <td></td>
      <td>4</td>
      <td>3</td>
      <td>4</td>
      <td>0</td>
      <td>4</td>
      <td>3</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>110</td>
      <td>110</td>
      <td>19</td>
      <td>19</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>110</td>
      <td>110</td>
      <td>19</td>
      <td>19</td>
    </tr>
    <tr>
      <td></td>
      <td>sceasy</td>
      <td>sceasy_convert</td>
      <td>Convert scRNA data object between popular formats</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics</td>
      <td>sceasy</td>
      <td>2019-10-14</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/sceasy</td>
      <td>0.0.5</td>
      <td>r-sceasy</td>
      <td>0.0.7</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1059</td>
      <td>1059</td>
      <td>222</td>
      <td>222</td>
      <td>426</td>
      <td>426</td>
      <td>71</td>
      <td>71</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1486</td>
      <td>1486</td>
      <td>294</td>
      <td>294</td>
    </tr>
    <tr>
      <td></td>
      <td>scmap</td>
      <td>scmap_get_std_output, scmap_index_cell, scmap_index_cluster, scmap_preprocess_sce, scmap_scmap_cell, scmap_scmap_cluster, scmap_select_features</td>
      <td>De-composed scmap functionality tools, based on https://github.com/ebi-gene-expression-group/scmap-cli and scmap 1.6.0.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_scmap</td>
      <td>2019-08-30</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/scmap</td>
      <td>1.6.4</td>
      <td>scmap-cli</td>
      <td>0.1.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>97</td>
      <td>97</td>
      <td>41</td>
      <td>41</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>97</td>
      <td>97</td>
      <td>41</td>
      <td>41</td>
    </tr>
    <tr>
      <td></td>
      <td>scpred</td>
      <td>scpred_get_feature_space, scpred_get_std_output, scpred_predict_labels, scpred_train_model</td>
      <td>De-composed scPred functionality tools, see https://github.com/ebi-gene-expression-group/scpred-cli and r-scPred 1.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_scpred</td>
      <td>2020-04-07</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/scpred</td>
      <td>1.0.2</td>
      <td>scpred-cli</td>
      <td>0.1.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>25</td>
      <td>25</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>25</td>
      <td>25</td>
      <td>5</td>
      <td>5</td>
    </tr>
    <tr>
      <td></td>
      <td>scripting</td>
      <td>singularity_scriptrunner</td>
      <td>suite_scripting</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bernt-matthias/mb-galaxy-tools/tree/master/tools/scripting/</td>
      <td>Text Manipulation</td>
      <td>scripting tools</td>
      <td>2023-07-29</td>
      <td>mbernt</td>
      <td>https://github.com/bernt-matthias/mb-galaxy-tools/tree/master/tools/scripting/</td>
      <td>https://github.com/Helmholtz-UFZ/galaxy-tools/tree/main/tools/scripting</td>
      <td>0.1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>sdmpredictors</td>
      <td>sdmpredictors_list_layers</td>
      <td>Terrestrial and marine predictors for species distribution modelling.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://cran.r-project.org/web/packages/sdmpredictors/index.html</td>
      <td>Ecology</td>
      <td>sdmpredictors</td>
      <td>2024-03-29</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/sdmpredictors</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/sdmpredictors</td>
      <td>0.2.15</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
    </tr>
    <tr>
      <td></td>
      <td>sequence_format_converter</td>
      <td>sequence_format_converter</td>
      <td>various fasta to tabular conversions</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Convert Formats, Fasta Manipulation</td>
      <td>sequence_format_converter</td>
      <td>2017-09-04</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/sequence_format_converter</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/sequence_format_converter</td>
      <td>2.2.0</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>seurat</td>
      <td>seurat_convert, seurat_dim_plot, seurat_export_cellbrowser, seurat_filter_cells, seurat_find_clusters, seurat_find_markers, seurat_find_neighbours, seurat_find_variable_genes, seurat_hover_locator, seurat_integration, seurat_map_query, seurat_normalise_data, seurat_plot, seurat_read10x, seurat_run_pca, seurat_run_tsne, seurat_run_umap, seurat_scale_data, seurat_select_integration_features</td>
      <td>De-composed Seurat functionality tools, based on https://github.com/ebi-gene-expression-group/r-seurat-scripts and Seurat 2.3.1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_seurat</td>
      <td>2018-07-09</td>
      <td>ebi-gxa</td>
      <td></td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/seurat</td>
      <td>4.0.0</td>
      <td>seurat-scripts</td>
      <td>4.4.0</td>
      <td></td>
      <td></td>
      <td>14</td>
      <td>14</td>
      <td>14</td>
      <td>11</td>
      <td>14</td>
      <td>14</td>
      <td>14</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>19</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2296</td>
      <td>2328</td>
      <td>673</td>
      <td>676</td>
      <td>141</td>
      <td>141</td>
      <td>39</td>
      <td>39</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2437</td>
      <td>2469</td>
      <td>712</td>
      <td>715</td>
    </tr>
    <tr>
      <td></td>
      <td>sgwb</td>
      <td>sgwb_astro_tool</td>
      <td>SGWB</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Astronomy</td>
      <td>sgwb_astro_tool</td>
      <td>2024-04-17</td>
      <td>astroteam</td>
      <td>https://github.com/esg-epfl-apc/tools-astro/tree/main/tools</td>
      <td>https://github.com/esg-epfl-apc/tools-astro/tree/main/tools/sgwb</td>
      <td>0.0.1+galaxy0</td>
      <td>ipython</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>2</td>
      <td>2</td>
    </tr>
    <tr>
      <td></td>
      <td>shift-longitudes</td>
      <td>shyft_longitudes</td>
      <td>Shift longitudes ranging from 0. and 360 degrees to -180. and 180. degrees</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/blob/master/tools/shift-longitudes/</td>
      <td>Climate Analysis</td>
      <td>shift_longitudes</td>
      <td>2019-03-15</td>
      <td>climate</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/shift-longitudes</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/shift-longitudes</td>
      <td>0.1.0</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>54</td>
      <td>64</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>54</td>
      <td>64</td>
      <td>4</td>
      <td>4</td>
    </tr>
    <tr>
      <td></td>
      <td>sixgill</td>
      <td>sixgill_build, sixgill_filter, sixgill_makefasta, sixgill_merge</td>
      <td>Six-frame Genome-Inferred Libraries for LC-MS/MS</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td></td>
      <td>Proteomics, MetaProteomics</td>
      <td>sixgill</td>
      <td>2016-10-11</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/sixgill</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/sixgill</td>
      <td>0.2.4</td>
      <td>sixgill</td>
      <td>0.2.4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>312</td>
      <td>312</td>
      <td>71</td>
      <td>71</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>312</td>
      <td>312</td>
      <td>71</td>
      <td>71</td>
    </tr>
    <tr>
      <td></td>
      <td>small_rna_clusters</td>
      <td>small_rna_clusters</td>
      <td>clusters small rna reads in alignment BAM files</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, SAM, Graphics, Next Gen Mappers</td>
      <td>small_rna_clusters</td>
      <td>2019-10-06</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/small_rna_clusters</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/small_rna_clusters</td>
      <td>1.3.0</td>
      <td>pysam</td>
      <td>0.22.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>small_rna_maps</td>
      <td>small_rna_maps</td>
      <td>Generates small read maps from alignment BAM files</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, SAM, Graphics, Next Gen Mappers</td>
      <td>small_rna_maps</td>
      <td>2018-11-25</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/small_rna_maps</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/small_rna_maps</td>
      <td>3.1.1</td>
      <td>numpy</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>small_rna_signatures</td>
      <td>overlapping_reads, signature</td>
      <td>Computes the tendency of small RNAs to overlap with each other.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA</td>
      <td>small_rna_signatures</td>
      <td>2017-09-06</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/small_rna_signatures</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/small_rna_signatures</td>
      <td>3.4.2</td>
      <td>pysam</td>
      <td>0.22.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>smithsonian-volcanoes</td>
      <td>smithsonian_volcanoes</td>
      <td>Retrieve data from Volcanoes of the World (VOTW) Database</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://volcano.si.edu/gvp_votw.cfm</td>
      <td>Retrieve Data</td>
      <td>smithsonian_volcanoes</td>
      <td>2019-04-15</td>
      <td>climate</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/smithsonian-volcanoes</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/smithsonian-volcanoes</td>
      <td>0.1.0</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>snv_matrix</td>
      <td>snvmatrix</td>
      <td>Generate matrix of SNV distances</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://snvphyl.readthedocs.io/en/latest/</td>
      <td>Sequence Analysis</td>
      <td>snv_matrix</td>
      <td>2016-01-21</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy/tree/development/tools/snvphyl-tools/snv_matrix</td>
      <td>1.8.2</td>
      <td>snvphyl-tools</td>
      <td>1.8.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>snvtocnv</td>
      <td>sequenzaindex, snvtocnv</td>
      <td>infer copy number variations from a vcf file with SNVs using R sequenza</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Variant Analysis</td>
      <td>snvtocnv</td>
      <td>2021-03-07</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/snvtocnv</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/snvtocnv</td>
      <td>3.0.0+galaxy1</td>
      <td>sequenza-utils</td>
      <td>3.0.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>spec2vec</td>
      <td>spec2vec_similarity, spec2vec_training</td>
      <td>Mass spectra similarity scoring using a trained Spec2Vec model.</td>
      <td>spec2vec</td>
      <td>spec2vec</td>
      <td></td>
      <td>Spec2Vec</td>
      <td>Improved mass spectral similarity scoring through learning of structural relationships.Spec2vec is a novel spectral similarity score inspired by a natural language processing algorithm -- Word2Vec. Where Word2Vec learns relationships between words in sentences, spec2vec does so for mass fragments and neutral losses in MS/MS spectra. The spectral similarity score is based on spectral embeddings learnt from the fragmental relationships within a large set of spectral data.Analysis and benchmarking of mass spectra similarity measures using gnps data set.</td>
      <td>Spectrum calculation, Spectral library search, Database search, Natural product identification</td>
      <td>Proteomics experiment, Metabolomics, Natural language processing, Proteomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/iomega/spec2vec</td>
      <td>Metabolomics</td>
      <td>spec2vec</td>
      <td>2022-12-07</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/spec2vec</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/spec2vec</td>
      <td>0.8.0</td>
      <td>spec2vec</td>
      <td>0.8.0</td>
      <td>Spectrum calculation, Spectral library search, Database search, Natural product identification</td>
      <td>Proteomics experiment, Metabolomics, Natural language processing, Proteomics</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
    </tr>
    <tr>
      <td></td>
      <td>spectrast2spectrast_irt</td>
      <td>gp_spectrast2spectrast_irt</td>
      <td>Filter from spectraST files to swath input files</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>spectrast2spectrast_irt</td>
      <td>2015-04-13</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/spectrast2spectrast_irt</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/spectrast2spectrast_irt</td>
      <td>0.1.0</td>
      <td>msproteomicstools</td>
      <td>0.11.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>spectrast2tsv</td>
      <td>gp_spectrast2tsv</td>
      <td>Filter from spectraST files to swath input files</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>spectrast2tsv</td>
      <td>2015-02-06</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/spectrast2tsv</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/spectrast2tsv</td>
      <td>0.1.0</td>
      <td>msproteomicstools</td>
      <td>0.11.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>spocc</td>
      <td>spocc_occ</td>
      <td>Get species occurences data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://cran.r-project.org/web/packages/spocc/index.html</td>
      <td>Ecology</td>
      <td>spocc_occ</td>
      <td>2019-05-23</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/spocc</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/spocc</td>
      <td>1.2.2</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>789</td>
      <td>840</td>
      <td>130</td>
      <td>134</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>789</td>
      <td>840</td>
      <td>130</td>
      <td>134</td>
    </tr>
    <tr>
      <td></td>
      <td>sr_bowtie</td>
      <td>bowtieForSmallRNA</td>
      <td>bowtie wrapper tool to align small RNA sequencing reads</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, Next Gen Mappers</td>
      <td>sr_bowtie</td>
      <td>2017-09-02</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/sr_bowtie</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/sr_bowtie</td>
      <td>2.3.0</td>
      <td>bowtie</td>
      <td>1.3.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>80</td>
      <td>80</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>80</td>
      <td>80</td>
      <td>1</td>
      <td>1</td>
    </tr>
    <tr>
      <td></td>
      <td>sr_bowtie_dataset_annotation</td>
      <td>sr_bowtie_dataset_annotation</td>
      <td>Maps iteratively small RNA sequencing datasets to reference sequences.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA</td>
      <td>sr_bowtie_dataset_annotation</td>
      <td>2017-09-11</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/sr_bowtie_dataset_annotation</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/sr_bowtie_dataset_annotation</td>
      <td>2.8</td>
      <td>bowtie</td>
      <td>1.3.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>srs_tools</td>
      <td>srs_diversity_maps, srs_global_indices, srs_process_data, srs_spectral_indices, srs_pca, srs_preprocess_s2, srs_metadata</td>
      <td>Compute biodiversity indicators for remote sensing data from Sentinel 2</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Ecology</td>
      <td></td>
      <td>2022-12-30</td>
      <td>ecology</td>
      <td>https://github.com/Marie59/Sentinel_2A/srs_tools</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/srs_tools</td>
      <td>0.0.1</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>4</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>4</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>239</td>
      <td>239</td>
      <td>34</td>
      <td>34</td>
      <td>44</td>
      <td>44</td>
      <td>13</td>
      <td>13</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>283</td>
      <td>283</td>
      <td>47</td>
      <td>47</td>
    </tr>
    <tr>
      <td></td>
      <td>stoc</td>
      <td>stoceps_filteringsp, stoceps_glm, stoceps_glm_group, stoceps_maketablecarrer, stoceps_trend_indic</td>
      <td>Tools to analyse STOC data.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Ecology</td>
      <td>stoceps</td>
      <td>2020-03-27</td>
      <td>ecology</td>
      <td>https://github.com/Alanamosse/Galaxy-E/tree/stoctool/tools/stoc</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/stoc</td>
      <td>0.0.2</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>328</td>
      <td>328</td>
      <td>38</td>
      <td>38</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>328</td>
      <td>328</td>
      <td>38</td>
      <td>38</td>
    </tr>
    <tr>
      <td></td>
      <td>suite_snvphyl</td>
      <td></td>
      <td>SNVPhyl suite defining all dependencies for SNVPhyl</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>suite_snvphyl_1_2_3</td>
      <td>2015-09-10</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy/tree/development/tools/suite_snvphyl</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>tarfast5</td>
      <td>tarfast5</td>
      <td>produces a tar.gz archive of fast5 sequence files</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Nanopore</td>
      <td>tarfast5</td>
      <td>2021-05-05</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/tarfast5</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/tarfast5</td>
      <td>0.6.1</td>
      <td>pigz</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>topologyeditors</td>
      <td>gromacs_modify_topology, gromacs_extract_topology</td>
      <td>Set of python scripts and associated tool files that can be used to modify topology files.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/buildtools/topologyeditors</td>
      <td>Molecular Dynamics, Computational chemistry</td>
      <td>topologyeditors</td>
      <td>2021-12-23</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/buildtools/topologyeditors</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/buildtools/topologyeditors</td>
      <td>0</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>57</td>
      <td>57</td>
      <td>10</td>
      <td>10</td>
      <td>20</td>
      <td>20</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>77</td>
      <td>77</td>
      <td>18</td>
      <td>18</td>
    </tr>
    <tr>
      <td></td>
      <td>translate_bed</td>
      <td>translate_bed</td>
      <td>Translate BED transcript CDS or cDNA in 3 frames</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://rest.ensembl.org/</td>
      <td>Proteomics</td>
      <td>translate_bed</td>
      <td>2018-01-14</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteogenomics/translate_bed</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteogenomics/translate_bed</td>
      <td>0.1.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>649</td>
      <td>685</td>
      <td>138</td>
      <td>142</td>
      <td>473</td>
      <td>473</td>
      <td>105</td>
      <td>105</td>
      <td>761</td>
      <td>761</td>
      <td>18</td>
      <td>18</td>
      <td>1883</td>
      <td>1919</td>
      <td>261</td>
      <td>265</td>
    </tr>
    <tr>
      <td></td>
      <td>translate_bed_sequences</td>
      <td>translate_bed_sequences</td>
      <td>Perform 3 frame translation of BED file augmented with a sequence column</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>translate_bed_sequences</td>
      <td>2016-01-13</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/translate_bed_sequences</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/translate_bed_sequences</td>
      <td>0.2.0</td>
      <td>biopython</td>
      <td>1.70</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>57</td>
      <td>69</td>
      <td>16</td>
      <td>21</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>57</td>
      <td>69</td>
      <td>16</td>
      <td>21</td>
    </tr>
    <tr>
      <td></td>
      <td>ucsc-cell-browser</td>
      <td>ucsc_cell_browser</td>
      <td>Python pipeline and Javascript scatter plot library for single-cell datasets</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://cells.ucsc.edu/</td>
      <td>Transcriptomics</td>
      <td>ucsc_cell_browser</td>
      <td>2018-09-11</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/develop/tools/tertiary-analysis/ucsc-cell-browser/.shed.yml</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/ucsc-cell-browser</td>
      <td>1.0.0+galaxy1</td>
      <td>ucsc-cell-browser</td>
      <td>1.2.8</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>965</td>
      <td>975</td>
      <td>178</td>
      <td>181</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>965</td>
      <td>975</td>
      <td>178</td>
      <td>181</td>
    </tr>
    <tr>
      <td></td>
      <td>unipept</td>
      <td>unipept</td>
      <td>Unipept retrieves metaproteomics information</td>
      <td>unipept</td>
      <td>unipept</td>
      <td></td>
      <td>Unipept</td>
      <td>Metaproteomics data analysis with a focus on interactive data visualizations.</td>
      <td>Prediction and recognition, Visualisation</td>
      <td>Proteomics, Proteogenomics, Biodiversity, Workflows</td>
      <td>To update</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp</td>
      <td>Proteomics</td>
      <td>unipept</td>
      <td>2015-04-03</td>
      <td>galaxyp</td>
      <td>https://unipept.ugent.be/apidocs</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/unipept</td>
      <td>4.5.1</td>
      <td>python</td>
      <td></td>
      <td>Prediction and recognition, Visualisation</td>
      <td>Proteomics, Proteogenomics, Biodiversity, Workflows</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>4900</td>
      <td>5146</td>
      <td>257</td>
      <td>270</td>
      <td>251</td>
      <td>251</td>
      <td>81</td>
      <td>81</td>
      <td>1069</td>
      <td>1069</td>
      <td>29</td>
      <td>29</td>
      <td>6220</td>
      <td>6466</td>
      <td>367</td>
      <td>380</td>
    </tr>
    <tr>
      <td></td>
      <td>uniprotxml_downloader</td>
      <td>uniprotxml_downloader</td>
      <td>Download UniProt proteome in XML or fasta format</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>uniprotxml_downloader</td>
      <td>2016-03-08</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/uniprotxml_downloader</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/uniprotxml_downloader</td>
      <td>2.4.0</td>
      <td>requests</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1488</td>
      <td>1596</td>
      <td>230</td>
      <td>266</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>743</td>
      <td>743</td>
      <td>35</td>
      <td>35</td>
      <td>2231</td>
      <td>2339</td>
      <td>265</td>
      <td>301</td>
    </tr>
    <tr>
      <td></td>
      <td>upload_roi_and_measures_to_omero</td>
      <td>uploadROIandMeasuresToOMERO</td>
      <td>Upload the ROI coordinates and the measurements to the omero server</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Imaging</td>
      <td>upload_roi_and_measures_to_omero</td>
      <td>2022-01-18</td>
      <td>lldelisle</td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/upload_roi_and_measures_to_omero</td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/upload_roi_and_measures_to_omero</td>
      <td>0.0.5</td>
      <td>omero-py</td>
      <td>5.11.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>validate_fasta_database</td>
      <td>validate_fasta_database</td>
      <td>runs Compomics database identification tool on any FASTA database, and separates valid and invalid entries based on a series of checks.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Fasta Manipulation, Proteomics</td>
      <td>validate_fasta_database</td>
      <td>2017-09-14</td>
      <td>galaxyp</td>
      <td></td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/validate_fasta_database</td>
      <td>0.1.5</td>
      <td>validate-fasta-database</td>
      <td>1.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>112</td>
      <td>112</td>
      <td>38</td>
      <td>38</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>112</td>
      <td>112</td>
      <td>38</td>
      <td>38</td>
    </tr>
    <tr>
      <td></td>
      <td>varscan_vaf</td>
      <td>varscan_vaf</td>
      <td>Compute variant allele frequency in vcf files generated by varscan.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Variant Analysis</td>
      <td>varscan_vaf</td>
      <td>2022-11-28</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/varscan_vaf</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/varscan_vaf</td>
      <td>0.1</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>vcf2snvalignment</td>
      <td>vcf2snvalignment</td>
      <td>Generates multiple alignment of variant calls</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://snvphyl.readthedocs.io/en/latest/</td>
      <td>Sequence Analysis</td>
      <td>vcf2snvalignment</td>
      <td>2016-01-20</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy/tree/development/tools/snvphyl-tools/vcf2snvalignment</td>
      <td>1.8.2</td>
      <td>snvphyl-tools</td>
      <td>1.8.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>verify_map</td>
      <td>verify_map</td>
      <td>Checks the mapping quality of all BAM(s)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://snvphyl.readthedocs.io/en/latest/</td>
      <td>Sequence Analysis</td>
      <td>verify_map</td>
      <td>2015-10-28</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy/tree/development/tools/snvphyl-tools/verify_map</td>
      <td>1.8.2</td>
      <td>snvphyl-tools</td>
      <td>1.8.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>vibrant</td>
      <td>vibrant</td>
      <td>Virus Identification By iteRative ANnoTation</td>
      <td>VIBRANT</td>
      <td>VIBRANT</td>
      <td></td>
      <td>VIBRANT</td>
      <td>Automated recovery, annotation and curation of microbial viruses, and evaluation of virome function from genomic sequences.Virus Identification By iteRative ANnoTation.Discovery Environment Applications List.Created by upendra_35, last modified on Nov 04, 2019.If you find VIBRANT useful please consider citing our preprint on bioRxiv:.Kieft, K., Zhou, Z., and Anantharaman, K. (2019). VIBRANT: Automated recovery, annotation and curation of microbial viruses, and evaluation of virome function from genomic sequences. BioRxiv 855387.VIBRANT is a tool for automated recovery and annotation of bacterial and archaeal viruses, determination of genome completeness, and characterization of virome function from metagenomic assemblies.The QuickStart tutorial provides an introduction to basic DE functionality and navigation.Please work through the tutorial and use the intercom button on the bottom right of this page if you have any questions</td>
      <td>Sequence trimming, Scaffolding, Genome annotation</td>
      <td>Metagenomics, Microbial ecology, Sequence assembly, Molecular interactions, pathways and networks, Endocrinology and metabolism</td>
      <td>Up-to-date</td>
      <td>https://github.com/AnantharamanLab/VIBRANT</td>
      <td>Metagenomics</td>
      <td>vibrant</td>
      <td>2024-09-11</td>
      <td>ufz</td>
      <td>https://github.com/Helmholtz-UFZ/ufz-galaxy-tools/blob/main/tools/vibrant</td>
      <td>https://github.com/Helmholtz-UFZ/galaxy-tools/tree/main/tools/vibrant</td>
      <td>1.2.1</td>
      <td>vibrant</td>
      <td>1.2.1</td>
      <td>Sequence trimming, Scaffolding, Genome annotation</td>
      <td>Metagenomics, Microbial ecology, Sequence assembly, Molecular interactions, pathways and networks, Endocrinology and metabolism</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>vigiechiro</td>
      <td>vigiechiro_bilanenrichipf, vigiechiro_bilanenrichirp, vigiechiro_idcorrect_2ndlayer, vigiechiro_idvalid</td>
      <td>Tools created by the vigiechiro team to analyses and identify chiro sounds files.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.vigienature-ecole.fr/les-observatoires/le-protocole-vigie-chiro</td>
      <td>Ecology</td>
      <td>vigiechiro</td>
      <td>2019-03-13</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/vigiechiro</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/vigiechiro</td>
      <td>0.1.1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>14412</td>
      <td>14590</td>
      <td>722</td>
      <td>732</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>14412</td>
      <td>14590</td>
      <td>722</td>
      <td>732</td>
    </tr>
    <tr>
      <td></td>
      <td>virsorter</td>
      <td>virsorter</td>
      <td>VirSorter2 applies a multi-classifier, expert-guided approach to detect diverse DNA and RNA virus genomes.</td>
      <td>virsorter</td>
      <td>virsorter</td>
      <td></td>
      <td>virsorter</td>
      <td>Identify DNA and RNA virus sequences.</td>
      <td>Taxonomic classification</td>
      <td>Metagenomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/jiarong/VirSorter2/</td>
      <td>Metagenomics</td>
      <td>virsorter</td>
      <td>2024-06-20</td>
      <td>ufz</td>
      <td>https://github.com/Helmholtz-UFZ/ufz-galaxy-tools/blob/main/tools/virsorter</td>
      <td>https://github.com/Helmholtz-UFZ/galaxy-tools/tree/main/tools/virsorter</td>
      <td>2.2.4</td>
      <td>virsorter</td>
      <td>2.2.4</td>
      <td>Taxonomic classification</td>
      <td>Metagenomics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>vmd</td>
      <td></td>
      <td>vmd is a package for  visualizing and analyzing trajectories from molecular dynamics (MD) simulations</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.ks.uiuc.edu/Research/vmd/</td>
      <td>Computational chemistry</td>
      <td>vmd</td>
      <td>2019-10-24</td>
      <td>chemteam</td>
      <td>https://github.com/thatchristoph/vmd-cvs-github/tree/master/vmd</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/vmd</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>waveica</td>
      <td>waveica</td>
      <td>Removal of batch effects for large-scale untargeted metabolomics data based on wavelet analysis.</td>
      <td>waveica</td>
      <td>waveica</td>
      <td></td>
      <td>WaveICA</td>
      <td>Removal of batch effects for large-scale untargeted metabolomics data based on wavelet transform.</td>
      <td>Standardisation and normalisation</td>
      <td>Metabolomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/RECETOX/WaveICA</td>
      <td>Metabolomics</td>
      <td></td>
      <td>2022-09-19</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/waveica</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/waveica</td>
      <td>0.2.0</td>
      <td>r-recetox-waveica</td>
      <td>0.2.0</td>
      <td>Standardisation and normalisation</td>
      <td>Metabolomics</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>39</td>
      <td>39</td>
      <td>6</td>
      <td>6</td>
      <td>4</td>
      <td>4</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>43</td>
      <td>43</td>
      <td>9</td>
      <td>9</td>
    </tr>
    <tr>
      <td></td>
      <td>xarray</td>
      <td>timeseries_extraction, xarray_coords_info, xarray_mapplot, xarray_metadata_info, xarray_netcdf2netcdf, xarray_select</td>
      <td>xarray (formerly xray) is an open source project and Python package that makes working withlabelled multi-dimensional arrays simple, efficient, and fun!xarray integrates with Dask to support parallel computations and streaming computation on datasetsthat dont fit into memory.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://xarray.pydata.org</td>
      <td>Ecology</td>
      <td></td>
      <td>2022-07-06</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/data_manipulation/xarray/</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/data_manipulation/xarray</td>
      <td>2022.3.0</td>
      <td>xarray</td>
      <td></td>
      <td></td>
      <td></td>
      <td>5</td>
      <td>2</td>
      <td>6</td>
      <td>5</td>
      <td>5</td>
      <td>2</td>
      <td>6</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>2</td>
      <td>0</td>
      <td>5995</td>
      <td>5995</td>
      <td>275</td>
      <td>275</td>
      <td>66</td>
      <td>66</td>
      <td>23</td>
      <td>23</td>
      <td>295</td>
      <td>295</td>
      <td>3</td>
      <td>3</td>
      <td>6356</td>
      <td>6356</td>
      <td>301</td>
      <td>301</td>
    </tr>
    <tr>
      <td></td>
      <td>xmlstarlet</td>
      <td>xmlstarlet</td>
      <td>Tool to convert a xml file from one metadata standard to another</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Convert Formats</td>
      <td>xmlstarlet</td>
      <td>2021-10-18</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/main/tools-ecology/tools/xmlstarlet</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/xmlstarlet</td>
      <td>1.6.1</td>
      <td>xmlstarlet</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>59</td>
      <td>59</td>
      <td>17</td>
      <td>17</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>59</td>
      <td>59</td>
      <td>17</td>
      <td>17</td>
    </tr>
    <tr>
      <td></td>
      <td>xpore</td>
      <td>xpore_dataprep, xpore_diffmod</td>
      <td>Identification and quantification of differential RNA modifications from direct RNA sequencing</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/GoekeLab/xpore</td>
      <td>Nanopore</td>
      <td>xpore</td>
      <td>2021-05-26</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/xpore</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/xpore</td>
      <td>2.1+galaxy0</td>
      <td>xpore</td>
      <td>2.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>xtb</td>
      <td>xtb_molecular_optimization</td>
      <td>Performs semiempirical molecular optimization.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/grimme-lab/xtb</td>
      <td>Metabolomics</td>
      <td>xtb_molecular_optimization</td>
      <td>2023-09-06</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/xtb</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/xtb</td>
      <td>6.6.1</td>
      <td>xtb</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>143</td>
      <td>143</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>143</td>
      <td>143</td>
      <td>4</td>
      <td>4</td>
    </tr>
    <tr>
      <td></td>
      <td>yac_clipper</td>
      <td>yac</td>
      <td>Clips 3' adapters for small RNA sequencing reads.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, Fastq Manipulation</td>
      <td>yac_clipper</td>
      <td>2015-06-17</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/yac_clipper</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/yac_clipper</td>
      <td>2.5.1</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>zeiss_lmd_converter</td>
      <td>ZeissLMDconverter</td>
      <td>Converts coordinates from a tabular file into a formatted text file readable by Zeiss laser-capture microdissection systems</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/zeiss_lmd_converter</td>
      <td>Proteomics</td>
      <td></td>
      <td>2024-06-29</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/zeiss_lmd_converter</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/zeiss_lmd_converter</td>
      <td>2.0.4</td>
      <td>shapely</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>1</td>
      <td>1</td>
    </tr>
    <tr>
      <td></td>
      <td>zoo_project_ogc_api_processes</td>
      <td>zoo_project_ogc_api_processes</td>
      <td>This tool is a wrapper for OGC API Processes (OTB) coming from the Zoo Project (https://zoo-project.github.io/docs/intro.html) and was created using the OGC-API-Process2Galaxy tool (https://github.com/AquaINFRA/OGC-API-Process2Galaxy). Check the README in the repository for more information.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/AquaINFRA/galaxy</td>
      <td>Ecology</td>
      <td></td>
      <td>2024-05-03</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/ogc_api_processes_wrapper</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/zoo_project_ogc_api_processes</td>
      <td>0.1.0</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>26</td>
      <td>26</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>26</td>
      <td>26</td>
      <td>4</td>
      <td>4</td>
    </tr>
  </tbody>
</table>

</div>