<script src="https://code.jquery.com/jquery-3.7.0.js"></script>
<script src="https://cdn.datatables.net/1.13.6/js/jquery.dataTables.min.js"></script>
<script src="https://cdn.datatables.net/searchbuilder/1.6.0/js/dataTables.searchBuilder.min.js"></script>
<script src="https://cdn.datatables.net/datetime/1.5.1/js/dataTables.dateTime.min.js"></script>

<script src="https://cdn.datatables.net/responsive/2.5.0/js/dataTables.responsive.min.js"></script>

<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/responsive/2.2.3/css/responsive.dataTables.min.css">

<link rel="stylesheet" href="https://cdn.datatables.net/1.13.6/css/jquery.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/searchbuilder/1.6.0/css/searchBuilder.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/datetime/1.5.1/css/dataTables.dateTime.min.css">

<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" />
<script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>

<style type="text/css">
  .tooltip {
    font-family: Georgia;
    font-size: 18px;
  }
  .tooltip .tooltip-inner {
    background-color: #00FFFF;
    color: #000000;
    min-width: 400px;
  }
  </style>

<script>
    $(document).ready(function() {
    $('#dataframe').DataTable( {
        "sScrollX": "100%",
        "dom": "Qlfrtip",

        responsive: {
            details: {
                type: 'column'
            }
        },
        "bAutoWidth": false,
        columnDefs: [ {
            className: 'control',
            orderable: false,
            targets:   [ 0 ],
            width: 100,
        },
        { type: 'natural', targets: 1 } // allow normal sorting
        ],

    });
});
</script>

<div class="display nowrap" style="width:80%" width="100%">

<table class="dataframe display nowrap" id="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th>Expand</th>
      <th>Topic</th>
      <th>Title</th>
      <th>Link</th>
      <th>EDAM topic</th>
      <th>EDAM operation</th>
      <th>Creation</th>
      <th>Last modification</th>
      <th>Version</th>
      <th>Tutorial</th>
      <th>Slides</th>
      <th>Video</th>
      <th>Workflows</th>
      <th>Tools</th>
      <th>Servers with precise tool versions</th>
      <th>Servers with tool but different versions</th>
      <th>Feedback number</th>
      <th>Feedback mean note</th>
      <th>Visitors</th>
      <th>Page views</th>
      <th>Visit duration</th>
      <th>Video views</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Galaxy on the Cloud</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/cloudbursting/slides.html</td>
      <td></td>
      <td></td>
      <td>2019-01-31</td>
      <td>2024-12-19</td>
      <td>7</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3115529</td>
      <td>7964527</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Galaxy Tool Management with Ephemeris</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/tool-management/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-01-27</td>
      <td>2025-04-03</td>
      <td>39</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3117347</td>
      <td>7967117</td>
      <td>5.683333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Galaxy Installation with Ansible</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/ansible-galaxy/tutorial.html</td>
      <td>Database management, Informatics, Data management</td>
      <td></td>
      <td>2019-01-28</td>
      <td>2025-11-28</td>
      <td>108</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3141186</td>
      <td>8012172</td>
      <td>4.050000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Connecting Galaxy to a compute cluster</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/connect-to-compute-cluster/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-01-07</td>
      <td>2025-11-24</td>
      <td>65</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3119633</td>
      <td>7971578</td>
      <td>5.566667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Galaxy Monitoring with gxadmin</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/gxadmin/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-01-28</td>
      <td>2025-11-13</td>
      <td>27</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3116437</td>
      <td>7965942</td>
      <td>5.066667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Reference Data with CVMFS</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/cvmfs/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-01-27</td>
      <td>2025-11-24</td>
      <td>54</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3117710</td>
      <td>7967740</td>
      <td>5.466667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Galaxy Monitoring with Telegraf and Grafana</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/monitoring/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-01-31</td>
      <td>2025-11-24</td>
      <td>54</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3118434</td>
      <td>7969894</td>
      <td>3.183333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>User, Role, Group, Quota, and Authentication managment</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/users-groups-quotas/slides.html</td>
      <td></td>
      <td></td>
      <td>2018-01-07</td>
      <td>2024-08-08</td>
      <td>19</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3115529</td>
      <td>7964530</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Distributed Object Storage</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/object-store/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-01-28</td>
      <td>2025-03-24</td>
      <td>28</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3116895</td>
      <td>7966665</td>
      <td>3.883333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Galaxy Interactive Tools</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/interactive-tools/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-02-27</td>
      <td>2025-11-27</td>
      <td>28</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3117870</td>
      <td>7968038</td>
      <td>4.950000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Gearing towards production</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/production/slides.html</td>
      <td></td>
      <td></td>
      <td>2019-01-28</td>
      <td>2025-11-20</td>
      <td>11</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3115529</td>
      <td>7964530</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Data Libraries</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/data-library/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-01-14</td>
      <td>2025-11-24</td>
      <td>35</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3099454</td>
      <td>7901505</td>
      <td>4.333333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Server Maintenance: Cleanup, Backup, and Restoration</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/backup-cleanup/tutorial.html</td>
      <td>Database management, Informatics, Data management</td>
      <td></td>
      <td>2019-01-31</td>
      <td>2025-11-24</td>
      <td>10</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3116132</td>
      <td>7965331</td>
      <td>10.850000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Running Jobs on Remote Resources with Pulsar</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/pulsar/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-01-28</td>
      <td>2025-11-25</td>
      <td>58</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3118120</td>
      <td>7968778</td>
      <td>5.150000</td>
      <td>81</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Ansible</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/ansible/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-07-11</td>
      <td>2024-08-08</td>
      <td>48</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3122028</td>
      <td>7976114</td>
      <td>4.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Docker and Galaxy</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/galaxy-docker/slides.html</td>
      <td></td>
      <td></td>
      <td>2016-09-29</td>
      <td>2022-10-18</td>
      <td>16</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3115529</td>
      <td>7964530</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>uWSGI</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/uwsgi/slides.html</td>
      <td></td>
      <td></td>
      <td>2019-01-28</td>
      <td>2022-03-01</td>
      <td>9</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3115529</td>
      <td>7964530</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Galaxy from an administrator's point of view</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/introduction/slides.html</td>
      <td></td>
      <td></td>
      <td>2017-06-12</td>
      <td>2024-06-25</td>
      <td>6</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3115529</td>
      <td>7964530</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Galaxy Troubleshooting</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/troubleshooting/slides.html</td>
      <td></td>
      <td></td>
      <td>2019-01-28</td>
      <td>2024-06-25</td>
      <td>15</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3115529</td>
      <td>7964530</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>External Authentication</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/external-auth/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-01-28</td>
      <td>2022-10-18</td>
      <td>14</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3116241</td>
      <td>7965644</td>
      <td>3.100000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Galactic Database</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/database/slides.html</td>
      <td></td>
      <td></td>
      <td>2019-01-28</td>
      <td>2025-11-20</td>
      <td>6</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3115529</td>
      <td>7964530</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Controlling Galaxy with systemd or Supervisor</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/systemd-supervisor/slides.html</td>
      <td></td>
      <td></td>
      <td>2019-01-28</td>
      <td>2023-04-16</td>
      <td>5</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3115529</td>
      <td>7964530</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Galaxy Monitoring</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/general-monitoring/slides.html</td>
      <td></td>
      <td></td>
      <td>2019-04-26</td>
      <td>2022-03-01</td>
      <td>5</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3115529</td>
      <td>7964530</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Galaxy Administrator Time Burden and Technology Usage</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/poll-ssa/slides.html</td>
      <td></td>
      <td></td>
      <td>2024-07-10</td>
      <td>2024-07-10</td>
      <td>1</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1107243</td>
      <td>2368615</td>
      <td>1.933333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Server: Other</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/stuff/slides.html</td>
      <td></td>
      <td></td>
      <td>2018-01-07</td>
      <td>2023-02-17</td>
      <td>6</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3115529</td>
      <td>7964530</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Reference Data with Data Managers</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/reference-genomes/tutorial.html</td>
      <td></td>
      <td>Genome indexing, Read mapping, Sequence alignment, Generation</td>
      <td>2019-01-28</td>
      <td>2023-04-21</td>
      <td>7</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>data_manager_fetch_genome_all_fasta_dbkey, bwa, bwa_mem_index_builder_data_manager</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3116116</td>
      <td>7965292</td>
      <td>6.483333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Advanced customisation of a Galaxy instance</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/advanced-galaxy-customisation/slides.html</td>
      <td></td>
      <td></td>
      <td>2017-06-27</td>
      <td>2023-04-17</td>
      <td>18</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3115529</td>
      <td>7964530</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Setting up Celery Workers for Galaxy</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/celery/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-07-16</td>
      <td>2025-11-24</td>
      <td>19</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2260330</td>
      <td>5144047</td>
      <td>3.750000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Empathy</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/empathy/slides.html</td>
      <td></td>
      <td></td>
      <td>2019-01-31</td>
      <td>2021-04-06</td>
      <td>4</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3115529</td>
      <td>7964530</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Deploying a Beacon v1 in Galaxy</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/beacon/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-04-16</td>
      <td>2025-11-22</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>156</td>
      <td>174</td>
      <td>1.866667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Configuring the Onedata connectors (remotes, Object Store, BYOS, BYOD)</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/onedata-configuration/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-03-24</td>
      <td>2025-04-08</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>58</td>
      <td>64</td>
      <td>3.100000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Enable upload via FTP</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/ftp/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-06-18</td>
      <td>2025-11-24</td>
      <td>28</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1265</td>
      <td>1727</td>
      <td>1.633333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Upgrading Galaxy</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/upgrading/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-06-08</td>
      <td>2024-11-25</td>
      <td>10</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>728</td>
      <td>1108</td>
      <td>2.866667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Create a subdomain for your community on UseGalaxy.eu</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/subdomain/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-11-27</td>
      <td>2024-12-13</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>46</td>
      <td>52</td>
      <td>2.566667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Performant Uploads with TUS</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/tus/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-03-01</td>
      <td>2025-11-22</td>
      <td>22</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>890</td>
      <td>1299</td>
      <td>5.483333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Monitoring Galaxy and Pulsar with Sentry</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/sentry/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-04-19</td>
      <td>2025-11-25</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>236</td>
      <td>284</td>
      <td>1.400000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Galaxy Monitoring with Reports</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/reports/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-01-31</td>
      <td>2025-11-24</td>
      <td>47</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>507</td>
      <td>674</td>
      <td>2.700000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>How I learned to stop worrying and love the systemd</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/stop-worrying-love-systemd/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-07-06</td>
      <td>2023-09-28</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>690</td>
      <td>793</td>
      <td>0.600000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Training Infrastructure as a Service (TIaaS)</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/tiaas/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-02-12</td>
      <td>2025-11-24</td>
      <td>45</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>560</td>
      <td>748</td>
      <td>3.183333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Mapping Jobs to Destinations using TPV</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/job-destinations/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-01-17</td>
      <td>2025-11-24</td>
      <td>43</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1712</td>
      <td>2669</td>
      <td>3.350000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Customizing the look of Galaxy (Manual)</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/customization-manual/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-04-16</td>
      <td>2025-04-08</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>208</td>
      <td>247</td>
      <td>1.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Customizing the look of Galaxy</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/customization/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-04-16</td>
      <td>2025-11-22</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>604</td>
      <td>738</td>
      <td>4.350000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Alternative Celery Deployment for Galaxy</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/celeryless/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-11-07</td>
      <td>2024-11-07</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>70</td>
      <td>78</td>
      <td>2.483333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Use Apptainer containers for running Galaxy jobs</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/apptainer/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-01-08</td>
      <td>2025-11-24</td>
      <td>19</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1103</td>
      <td>1404</td>
      <td>2.583333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Galaxy Database schema</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/database-schema/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2016-07-20</td>
      <td>2025-04-08</td>
      <td>23</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3049</td>
      <td>4977</td>
      <td>1.066667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Adding file-sources to Galaxy</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/file_sources/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-01-20</td>
      <td>2025-01-20</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>61</td>
      <td>79</td>
      <td>2.550000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Automation with Jenkins</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/jenkins/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-08-06</td>
      <td>2022-10-18</td>
      <td>10</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>8325</td>
      <td>16474</td>
      <td>0.716667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Reference Data with CVMFS without Ansible</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/cvmfs-manual/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-06-17</td>
      <td>2023-11-23</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>406</td>
      <td>543</td>
      <td>2.383333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Galaxy Installation on Kubernetes</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/k8s-deploying-galaxy/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-07-02</td>
      <td>2024-06-30</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1626</td>
      <td>2677</td>
      <td>2.266667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Managing Galaxy on Kubernetes</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/k8s-managing-galaxy/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-07-02</td>
      <td>2024-12-19</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>678</td>
      <td>988</td>
      <td>2.100000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Galaxy usage on SURF Research Cloud</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/surf-research-cloud-galaxy/tutorial.html</td>
      <td>Informatics, Data management</td>
      <td></td>
      <td>2024-12-18</td>
      <td>2025-08-05</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>83</td>
      <td>102</td>
      <td>1.566667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Pulsar usage on SURF Research Cloud</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/surf-research-cloud-pulsar/tutorial.html</td>
      <td>Informatics, Data management</td>
      <td></td>
      <td>2024-12-18</td>
      <td>2024-12-19</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>81</td>
      <td>89</td>
      <td>0.450000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Deploying Wireguard for private mesh networking</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/wireguard/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-09-21</td>
      <td>2024-12-19</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1113</td>
      <td>1267</td>
      <td>0.333333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Deploying Tailscale/Headscale for private mesh networking</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/wireguard-headscale/tutorial.html</td>
      <td>Data security, Software engineering</td>
      <td></td>
      <td>2022-09-21</td>
      <td>2024-12-19</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2382</td>
      <td>3065</td>
      <td>0.433333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Deploying a compute cluster in OpenStack via Terraform</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/terraform/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-10-27</td>
      <td>2024-12-19</td>
      <td>15</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3131595</td>
      <td>7994732</td>
      <td>3.266667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Image analysis using Deep Learning</td>
      <td>Contribute to the BioImage.IO models</td>
      <td>https://training.galaxyproject.org//topics/ai4life/tutorials/contribute-materials/slides.html</td>
      <td></td>
      <td></td>
      <td>2023-05-18</td>
      <td>2023-05-24</td>
      <td>3</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1753015</td>
      <td>3856386</td>
      <td>2.100000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Image analysis using Deep Learning</td>
      <td>Welcome to the BioImage Model Zoo</td>
      <td>https://training.galaxyproject.org//topics/ai4life/tutorials/welcome-bioimageio/slides.html</td>
      <td></td>
      <td></td>
      <td>2023-05-11</td>
      <td>2023-05-18</td>
      <td>3</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1767305</td>
      <td>3888762</td>
      <td>2.100000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Image analysis using Deep Learning</td>
      <td>BioImage Model Zoo: Advanced models in one-click</td>
      <td>https://training.galaxyproject.org//topics/ai4life/tutorials/getting-started/slides.html</td>
      <td></td>
      <td></td>
      <td>2023-04-18</td>
      <td>2023-05-24</td>
      <td>4</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1814215</td>
      <td>3992158</td>
      <td>2.100000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>An Introduction to Genome Assembly</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/general-introduction/tutorial.html</td>
      <td></td>
      <td>Sequence assembly validation, Visualisation, Sequencing quality control, De-novo assembly, Formatting, Validation, Statistical calculation, Sequence composition calculation</td>
      <td>2017-05-23</td>
      <td>2024-09-12</td>
      <td>29</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>velvetg, quast, velveth, fastqc, multiqc, fastq_paired_end_interlacer</td>
      <td>Galaxy@AuBi, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>3151676</td>
      <td>8023644</td>
      <td>5.166667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Genome Assembly Quality Control</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/assembly-quality-control/tutorial.html</td>
      <td>Biodiversity, Sequence assembly, Data quality management</td>
      <td>Sequence assembly validation, Visualisation, Phasing, Scaffolding, Genome assembly, De-novo assembly, k-mer counting</td>
      <td>2022-08-26</td>
      <td>2025-06-04</td>
      <td>6</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>busco, chromeister, meryl, merqury, quast</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>2214986</td>
      <td>5010914</td>
      <td>4.366667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Genome Assembly of MRSA from Oxford Nanopore MinION data (and optionally Illumina data)</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/mrsa-nanopore/tutorial.html</td>
      <td>Sequence assembly, Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology, Antimicrobial resistance</td>
      <td>Sequence assembly validation, Visualisation, Read mapping, Mapping assembly, Sequencing quality control, De-novo assembly, Sequence contamination filtering, Box-Whisker plot plotting, Sequence assembly visualisation, Cross-assembly, Sequence alignment, Scatter plot plotting, Genome assembly, Filtering, Statistical calculation, Sequence composition calculation, Sequencing error detection</td>
      <td>2021-03-24</td>
      <td>2024-11-29</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>upload1, bwa_mem2, bandage_image, polypolish, porechop, filtlong, fastqc, flye, fastp, quast, CONVERTER_bz2_to_uncompressed, nanoplot</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>10600</td>
      <td>15866</td>
      <td>3.400000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Large genome assembly and polishing</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/largegenome/tutorial.html</td>
      <td>Genomics, Sequence assembly, Biodiversity</td>
      <td>Sequence assembly, Sequence contamination filtering, Validation, Sequence assembly visualisation, Cross-assembly, Scatter plot plotting, Genome assembly, Statistical calculation, Sequence composition calculation, Variant calling, Sequence assembly validation, Visualisation, Mapping assembly, Sequencing quality control, De-novo assembly, Box-Whisker plot plotting, Pairwise sequence alignment, Base-calling</td>
      <td>2023-03-06</td>
      <td>2025-06-04</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>busco, barchart_gnuplot, bandage_image, meryl, minimap2, medaka_consensus_pipeline, fasta-stats, quast, fastqc, genomescope, flye, fastp, racon, nanoplot, multiqc</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>7592</td>
      <td>9844</td>
      <td>1.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Hybrid genome assembly - Nanopore and Illumina</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/hybrid_denovo_assembly/tutorial.html</td>
      <td></td>
      <td>Sequence assembly validation, Visualisation, Read alignment, Analysis, Read mapping, Sequence assembly, Generation, Mapping assembly, Sequencing quality control, De-novo assembly, Genome indexing, Validation, Cross-assembly, Sequence alignment, Aggregation, Genome assembly</td>
      <td>2025-11-03</td>
      <td>2025-11-06</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>busco, unicycler, quast, pilon, flye, multiqc, bwa_mem</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>GalaxyTrakr, UseGalaxy.be, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>248</td>
      <td>320</td>
      <td>2.066667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>ERGA post-assembly QC</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/ERGA-post-assembly-QC/tutorial.html</td>
      <td>Genomics, Sequence assembly, Biodiversity</td>
      <td>Sequence assembly validation, Visualisation, Phasing, Data deposition, Sequence trimming, De-novo assembly, Sequence alignment analysis, Sequence assembly visualisation, Genotyping, Data handling, Scaffolding, Genome assembly, Data parsing, Sequence alignment, Pairwise sequence alignment, k-mer counting</td>
      <td>2023-06-23</td>
      <td>2025-06-04</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>busco, bg_diamond, meryl, pretext_map, samtools_stats, pretext_snapshot, smudgeplot, gfastats, __EXTRACT_DATASET__, interactive_tool_blobtoolkit, merqury, bellerophon, collapse_dataset, bandage_image, blobtoolkit, bwa_mem2, CONVERTER_gz_to_uncompressed, minimap2, genomescope, tp_find_and_replace</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>2041</td>
      <td>2644</td>
      <td>2.033333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Chloroplast genome assembly</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/chloroplast-assembly/tutorial.html</td>
      <td></td>
      <td>Read alignment, Analysis, Read mapping, Generation, Sequence assembly, Genome annotation, Coding region prediction, Mapping assembly, Genome indexing, De-novo assembly, Box-Whisker plot plotting, Sequence assembly visualisation, Cross-assembly, Genome visualisation, Scatter plot plotting, Genome assembly, Sequence alignment, Gene prediction</td>
      <td>2020-12-04</td>
      <td>2025-05-15</td>
      <td>21</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>bandage_info, upload1, jbrowse, bandage_image, fasta-stats, pilon, nanoplot, flye, bwa_mem, prokka</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>8143</td>
      <td>12654</td>
      <td>2.783333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Assembly of the mitochondrial genome from PacBio HiFi reads</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/mitochondrion-assembly/tutorial.html</td>
      <td>Genomics, Sequence assembly, Biodiversity, Mitochondria</td>
      <td></td>
      <td>2024-09-03</td>
      <td>2025-06-04</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>mitohifi, compress_file</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>922</td>
      <td>1116</td>
      <td>1.816667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Decontamination of a genome assembly</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/assembly-decontamination/tutorial.html</td>
      <td>Genomics, Sequence assembly, Biodiversity</td>
      <td>Taxonomic classification, Data handling</td>
      <td>2024-09-04</td>
      <td>2025-06-04</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>ncbi_dustmasker_wrapper, parse_mito_blast, ncbi_blastn_wrapper, tp_sed_tool, Cut1, Filter1, tp_cat, kraken2, gfastats</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>1725</td>
      <td>2128</td>
      <td>2.433333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Genome Assembly of a bacterial genome (MRSA) sequenced using Illumina MiSeq Data</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/mrsa-illumina/tutorial.html</td>
      <td>Sequence assembly, Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology</td>
      <td>Sequence assembly validation, Visualisation, Read mapping, Sequencing quality control, Sequence contamination filtering, Sequence assembly visualisation, Genome assembly, Statistical calculation, Sequence composition calculation</td>
      <td>2021-03-24</td>
      <td>2025-11-18</td>
      <td>17</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>bandage_info, falco, shovill, upload1, bandage_image, fastqc, fastp, quast</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>10363</td>
      <td>15886</td>
      <td>3.566667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Assembly of metagenomic sequencing data</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/metagenomics-assembly/tutorial.html</td>
      <td>Metagenomics, Sequence assembly</td>
      <td>Read mapping, Local alignment, Sequence contamination filtering, Sequence assembly visualisation, Data handling, Genome assembly, Statistical calculation, Read pre-processing, Sequence composition calculation, Variant calling, Sequence assembly validation, Visualisation, Sequence trimming, Sequencing quality control, Formatting, Sequence alignment analysis, Primer removal, Sequence file editing</td>
      <td>2025-12-16</td>
      <td>2025-12-16</td>
      <td>0</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>bandage_info, metaspades, bg_uniq, megahit, megahit_contig2fastg, bandage_image, seqtk_subseq, coverm_contig, filter_tabular, tp_cat, fastqc, random_lines1, cutadapt, bamtools, quast, ngsutils_bam_filter, bowtie2, collection_column_join</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td>0</td>
      <td></td>
      <td>6</td>
      <td>10</td>
      <td>3.016667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>De Bruijn Graph Assembly</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/debruijn-graph-assembly/tutorial.html</td>
      <td></td>
      <td>Optimisation and refinement, Genome assembly, Sequence assembly</td>
      <td>2017-05-24</td>
      <td>2024-02-01</td>
      <td>25</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>fasta-stats, velvetoptimiser, spades</td>
      <td>Galaxy@AuBi, UseGalaxy.be, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>3122159</td>
      <td>7975048</td>
      <td>5.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Unicycler Assembly</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/unicycler-assembly/tutorial.html</td>
      <td>Sequence assembly, Genomics, Microbiology</td>
      <td>Sequence assembly validation, Visualisation, Genome annotation, Coding region prediction, Sequencing quality control, Validation, Aggregation, Genome assembly, Gene prediction, Statistical calculation, Sequence composition calculation</td>
      <td>2017-10-11</td>
      <td>2025-05-15</td>
      <td>24</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>unicycler, quast, fastqc, multiqc, prokka</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>Galaxy@AuBi, GalaxyTrakr, MISSISSIPPI, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>3128570</td>
      <td>7986804</td>
      <td>4.383333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Unicycler assembly of SARS-CoV-2 genome with preprocessing to remove human genome reads</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/assembly-with-preprocessing/tutorial.html</td>
      <td></td>
      <td>Read mapping, Sequencing quality control, Sequence contamination filtering, Validation, Box-Whisker plot plotting, Sequence assembly visualisation, Scatter plot plotting, Data handling, Aggregation, Genome assembly, Pairwise sequence alignment, Sequence file editing</td>
      <td>2020-04-21</td>
      <td>2025-05-15</td>
      <td>14</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>bandage_info, fasta_filter_by_length, seqtk_sample, __ZIP_COLLECTION__, fasterq_dump, collapse_dataset, bandage_image, unicycler, samtools_stats, minimap2, samtools_fastx, samtools_view, fastp, multiqc, nanoplot, bowtie2</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td>Galaxy@AuBi, MISSISSIPPI, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>3121575</td>
      <td>7974601</td>
      <td>5.500000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Vertebrate genome assembly using HiFi, Bionano and Hi-C data - Step by Step</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/vgp_genome_assembly/tutorial.html</td>
      <td>Genomics, Sequence assembly, Biodiversity</td>
      <td>Sequence assembly validation, Visualisation, Phasing, Read binning, Sequence trimming, De-novo assembly, Scaffolding, Data handling, Genome assembly, Primer removal, Sequence alignment, Pairwise sequence alignment, Read pre-processing, Mapping, k-mer counting</td>
      <td>2021-06-04</td>
      <td>2025-06-04</td>
      <td>57</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>busco, meryl, tp_grep_tool, Convert characters1, pretext_map, salsa, pretext_snapshot, gfastats, merqury, yahs, bellerophon, collapse_dataset, hifiasm, gfa_to_fa, bionano_scaffold, sort1, purge_dups, cutadapt, Add_a_column1, bwa_mem2, cat1, minimap2, tp_cut_tool, genomescope, tp_find_and_replace, quast, bedtools_bamtobed, collection_column_join</td>
      <td></td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>16177</td>
      <td>24332</td>
      <td>1.966667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Genome assembly using PacBio data</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/hifi-assembly/tutorial.html</td>
      <td>Genomics, Sequence assembly, Biodiversity</td>
      <td>Sequence assembly validation, Visualisation, Genome assembly, Mapping assembly, De-novo assembly, Cross-assembly</td>
      <td>2021-11-29</td>
      <td>2025-06-04</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>busco, hifiasm, gfa_to_fa, fasta-stats, flye, quast</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>915</td>
      <td>1076</td>
      <td>1.550000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Using the VGP workflows to assemble a vertebrate genome with HiFi and Hi-C data</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/vgp_workflow_training/tutorial.html</td>
      <td>Sequence assembly, Chromosome conformation capture</td>
      <td>Sequence assembly validation, Visualisation</td>
      <td>2022-04-06</td>
      <td>2025-09-09</td>
      <td>26</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>quast</td>
      <td>Galaxy@AuBi, GalaxyTrakr, MISSISSIPPI, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3673</td>
      <td>5082</td>
      <td>2.616667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Making sense of a newly assembled genome</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/ecoli_comparison/tutorial.html</td>
      <td>Sequence assembly, Genomics, Microbiology</td>
      <td>Read mapping, Sequence alignment, Genome visualisation, Sequence visualisation, Mapping</td>
      <td>2018-06-14</td>
      <td>2024-06-24</td>
      <td>29</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>fasta_filter_by_length, Grep1, tp_sort_header_tool, circos, tp_grep_tool, random_lines1, bedtools_sortbed, bedtools_complementbed, lastz_wrapper_2, tp_replace_in_column, tp_sed_tool, datamash_ops, fasta_compute_length, circos_aln_to_links, addValue, upload1, collapse_dataset, bedtools_intersectbed, Filter1, circos_interval_to_tiles, tp_cat, jbrowse, cat1, Cut1, join1, mergeCols1, tp_replace_in_line</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>6780</td>
      <td>13032</td>
      <td>2.916667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Genome assembly and assembly QC - Introduction short version</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/assembly_and_qc_short/slides.html</td>
      <td></td>
      <td></td>
      <td>2025-04-22</td>
      <td>2025-04-23</td>
      <td>2</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>653846</td>
      <td>1255559</td>
      <td>1.466667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Deeper look into Genome Assembly algorithms</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/algorithms-introduction/slides.html</td>
      <td></td>
      <td></td>
      <td>2021-01-12</td>
      <td>2021-01-12</td>
      <td>1</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>3094799</td>
      <td>7896003</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>An introduction to get started in genome assembly and annotation</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/get-started-genome-assembly/slides.html</td>
      <td></td>
      <td></td>
      <td>2021-11-30</td>
      <td>2023-10-23</td>
      <td>7</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>2634009</td>
      <td>6339485</td>
      <td>2.250000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Climate</td>
      <td>Sentinel 5P data visualisation</td>
      <td>https://training.galaxyproject.org//topics/climate/tutorials/sentinel5_data/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-01-26</td>
      <td>2025-05-10</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>interactive_tool_copernicus, interactive_tool_copernicus_notebook, interactive_tool_panoply</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>319</td>
      <td>366</td>
      <td>0.500000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Climate</td>
      <td>Pangeo Notebook in Galaxy - Introduction to Xarray</td>
      <td>https://training.galaxyproject.org//topics/climate/tutorials/pangeo-notebook/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-02-18</td>
      <td>2025-05-10</td>
      <td>11</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>interactive_tool_pangeo_notebook</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2508931</td>
      <td>5940157</td>
      <td>4.133333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Climate</td>
      <td>Analyse Argo data</td>
      <td>https://training.galaxyproject.org//topics/climate/tutorials/argo_pangeo/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-03-24</td>
      <td>2025-05-10</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>argo_getdata, xarray_coords_info, timeseries_extraction, xarray_metadata_info, interactive_tool_odv</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>320</td>
      <td>382</td>
      <td>1.066667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Climate</td>
      <td>Pangeo ecosystem 101 for everyone - Introduction to Xarray Galaxy Tools</td>
      <td>https://training.galaxyproject.org//topics/climate/tutorials/pangeo/tutorial.html</td>
      <td></td>
      <td>Visualisation</td>
      <td>2022-02-18</td>
      <td>2025-05-10</td>
      <td>6</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>graphicsmagick_image_montage, ggplot2_point, xarray_coords_info, regexColumn1, climate_stripes, xarray_metadata_info, __MERGE_COLLECTION__, __EXTRACT_DATASET__, xarray_netcdf2netcdf, xarray_select, xarray_mapplot</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>2508696</td>
      <td>5939628</td>
      <td>5.016667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Climate</td>
      <td>Ocean's variables study</td>
      <td>https://training.galaxyproject.org//topics/climate/tutorials/ocean-variables/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-11-29</td>
      <td>2025-05-10</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>interactive_tool_divand, interactive_tool_odv, divand_full_analysis</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>237</td>
      <td>284</td>
      <td>1.616667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Climate</td>
      <td>Nitrate DMQC for autonomous platforms such as Argo floats</td>
      <td>https://training.galaxyproject.org//topics/climate/tutorials/ocean_qcv_analysis/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-08-08</td>
      <td>2025-09-16</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tool_odv, tool_biogeochemical_calibration, harmonize_insitu_to_netcdf, interactive_tool_odv, tool_odv_history</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>49</td>
      <td>60</td>
      <td>1.683333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Climate</td>
      <td>Functionally Assembled Terrestrial Ecosystem Simulator (FATES)</td>
      <td>https://training.galaxyproject.org//topics/climate/tutorials/fates/tutorial.html</td>
      <td></td>
      <td>Visualisation</td>
      <td>2020-10-25</td>
      <td>2025-05-10</td>
      <td>18</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>ggplot2_point, xarray_select, xarray_metadata_info, interactive_tool_panoply, tp_find_and_replace, __EXTRACT_DATASET__, ctsm_fates</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3117023</td>
      <td>7966889</td>
      <td>4.100000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Climate</td>
      <td>Getting your hands-on climate data</td>
      <td>https://training.galaxyproject.org//topics/climate/tutorials/climate-101/tutorial.html</td>
      <td></td>
      <td>Visualisation</td>
      <td>2020-04-30</td>
      <td>2025-05-10</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>psy_maps, Grep1, ggplot2_point, datamash_ops, climate_stripes, interactive_tool_panoply, tp_awk_tool, cds_essential_variability</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1091</td>
      <td>1500</td>
      <td>1.266667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Climate</td>
      <td>Introduction to climate data</td>
      <td>https://training.galaxyproject.org//topics/climate/tutorials/introduction/slides.html</td>
      <td></td>
      <td></td>
      <td>2020-10-11</td>
      <td>2025-05-10</td>
      <td>2</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>3115529</td>
      <td>7964542</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Climate</td>
      <td>Ocean Data View (ODV)</td>
      <td>https://training.galaxyproject.org//topics/climate/tutorials/ocean-data-view/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-11-27</td>
      <td>2025-05-10</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>interactive_tool_odv</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>427</td>
      <td>514</td>
      <td>1.166667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Climate</td>
      <td>Functionally Assembled Terrestrial Ecosystem Simulator (FATES) with Galaxy Climate JupyterLab</td>
      <td>https://training.galaxyproject.org//topics/climate/tutorials/fates-jupyterlab/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-10-25</td>
      <td>2025-05-10</td>
      <td>16</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>interactive_tool_jupyter_notebook</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>562</td>
      <td>969</td>
      <td>1.733333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Climate</td>
      <td>Getting your hands-on earth data</td>
      <td>https://training.galaxyproject.org//topics/climate/tutorials/earth_system/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-10-04</td>
      <td>2025-05-10</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>c3s, argo_getdata, spocc_occ, obis_data</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>115</td>
      <td>149</td>
      <td>1.916667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Climate</td>
      <td>Collaboration with JupyterGIS</td>
      <td>https://training.galaxyproject.org//topics/climate/tutorials/jupytergis_collaboration/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-04-03</td>
      <td>2025-05-10</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>190</td>
      <td>236</td>
      <td>0.866667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Climate</td>
      <td>Visualize Climate data with Panoply netCDF viewer</td>
      <td>https://training.galaxyproject.org//topics/climate/tutorials/panoply/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-03-05</td>
      <td>2025-05-10</td>
      <td>18</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>interactive_tool_panoply</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>6378</td>
      <td>10355</td>
      <td>0.800000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Community Building</td>
      <td>Creation of the labs in the different Galaxy instances for your community</td>
      <td>https://training.galaxyproject.org//topics/community/tutorials/community-lab/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-10-04</td>
      <td>2025-11-04</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>44</td>
      <td>47</td>
      <td>0.300000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Community Building</td>
      <td>Creating community content</td>
      <td>https://training.galaxyproject.org//topics/community/tutorials/community_content/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-12-20</td>
      <td>2024-12-20</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>64</td>
      <td>78</td>
      <td>0.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Community Building</td>
      <td>Creation of resources listing Galaxy workflow for your community</td>
      <td>https://training.galaxyproject.org//topics/community/tutorials/community-workflow-table/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-09-17</td>
      <td>2025-11-27</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>18</td>
      <td>19</td>
      <td>0.016667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Community Building</td>
      <td>Make your tools available on your subdomain</td>
      <td>https://training.galaxyproject.org//topics/community/tutorials/tools_subdomains/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-12-03</td>
      <td>2024-12-20</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>47</td>
      <td>62</td>
      <td>7.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Community Building</td>
      <td>Creation of resources listing all the tools and their metadata relevant to your community</td>
      <td>https://training.galaxyproject.org//topics/community/tutorials/community-tool-table/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-03-13</td>
      <td>2025-10-28</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>105</td>
      <td>129</td>
      <td>1.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Community Building</td>
      <td>Creation of a Galaxy tutorial table for your community</td>
      <td>https://training.galaxyproject.org//topics/community/tutorials/community-tutorial-table/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-09-17</td>
      <td>2025-10-28</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>24</td>
      <td>27</td>
      <td>1.733333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Community Building</td>
      <td>What's a Special Interest Group?</td>
      <td>https://training.galaxyproject.org//topics/community/tutorials/sig_define/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-07-08</td>
      <td>2024-12-20</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>108</td>
      <td>142</td>
      <td>2.550000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Community Building</td>
      <td>Creating a Special Interest Group</td>
      <td>https://training.galaxyproject.org//topics/community/tutorials/sig_create/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-07-08</td>
      <td>2024-12-20</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>136</td>
      <td>149</td>
      <td>1.716667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Computational chemistry</td>
      <td>Protein target prediction of a bioactive ligand with Align-it and ePharmaLib</td>
      <td>https://training.galaxyproject.org//topics/computational-chemistry/tutorials/zauberkugel/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-02-15</td>
      <td>2024-06-14</td>
      <td>10</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>split_file_to_collection, rdconf, openbabel_addh, ctb_alignit, tp_cat, sort1</td>
      <td>UseGalaxy.cz</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1281</td>
      <td>1842</td>
      <td>2.366667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Computational chemistry</td>
      <td>Data management in Medicinal Chemistry</td>
      <td>https://training.galaxyproject.org//topics/computational-chemistry/tutorials/med-chem-data/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-01-08</td>
      <td>2024-09-13</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>ctb_silicos_qed, ctb_im_rxn_maker, chembl, get_pdb, md_converter, openbabel_compound_convert, openbabel_svg_depiction</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>480</td>
      <td>575</td>
      <td>1.616667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Computational chemistry</td>
      <td>Protein-ligand docking</td>
      <td>https://training.galaxyproject.org//topics/computational-chemistry/tutorials/cheminformatics/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-10-19</td>
      <td>2025-12-04</td>
      <td>17</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>chembl, ctb_compound_convert, tp_grep_tool, prepare_box, prepare_receptor, get_pdb, ctb_chemfp_mol2fps, ctb_online_data_fetch, openbabel_compound_convert, openbabel_svg_depiction, ctb_chemfp_butina_clustering, ctb_change_title, docking, ctb_remIons, collapse_dataset, ctb_obgrep, tp_cat, ctb_np-likeness-calculator, ctb_remDuplicates, sdf_to_tab, fpocket, ctb_filter, tp_find_and_replace, ctb_chemfp_nxn_clustering</td>
      <td></td>
      <td>UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>19980</td>
      <td>33990</td>
      <td>1.616667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Computational chemistry</td>
      <td>Analysis of molecular dynamics simulations</td>
      <td>https://training.galaxyproject.org//topics/computational-chemistry/tutorials/analysis-md-simulations/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-06-03</td>
      <td>2023-11-09</td>
      <td>13</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>mdanalysis_cosine_analysis, mdanalysis_ramachandran_plot, mdanalysis_distance, mdanalysis_rdf, bio3d_rmsf, mdanalysis_dihedral, bio3d_pca, bio3d_rmsd</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>Galaxy@AuBi, UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>32598</td>
      <td>57691</td>
      <td>1.250000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Computational chemistry</td>
      <td>Virtual screening of the SARS-CoV-2 main protease with rxDock and pose scoring</td>
      <td>https://training.galaxyproject.org//topics/computational-chemistry/tutorials/covid19-docking/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-03-27</td>
      <td>2023-11-09</td>
      <td>10</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>xchem_transfs_scoring, collapse_dataset, enumerate_charges, split_file_to_collection, rdock_rbcavity, sucos_max_score, ctb_frankenstein_ligand, rxdock_rbcavity, rdock_rbdock, openbabel_compound_convert, xchem_pose_scoring, rxdock_rbdock</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3206</td>
      <td>5260</td>
      <td>3.016667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Computational chemistry</td>
      <td>Running molecular dynamics simulations using GROMACS</td>
      <td>https://training.galaxyproject.org//topics/computational-chemistry/tutorials/md-simulation-gromacs/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-06-03</td>
      <td>2023-11-09</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>gmx_setup, gmx_sim, get_pdb, gmx_editconf, gmx_em, gmx_solvate, tp_grep_tool</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>Galaxy@AuBi, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>27033</td>
      <td>48201</td>
      <td>1.433333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Computational chemistry</td>
      <td>High Throughput Molecular Dynamics and Analysis</td>
      <td>https://training.galaxyproject.org//topics/computational-chemistry/tutorials/htmd-analysis/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-05-20</td>
      <td>2025-04-08</td>
      <td>19</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>mdanalysis_cosine_analysis, gmx_merge_topology_files, gmx_setup, gmx_sim, ambertools_acpype, get_pdb, bio3d_rmsf, gmx_editconf, gmx_em, gmx_solvate, bio3d_pca, tp_grep_tool, md_converter, gmx_energy, bio3d_pca_visualize, openbabel_compound_convert, vmd_hbonds, bio3d_rmsd</td>
      <td>UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>10980</td>
      <td>19201</td>
      <td>2.416667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Computational chemistry</td>
      <td>Running molecular dynamics simulations using NAMD</td>
      <td>https://training.galaxyproject.org//topics/computational-chemistry/tutorials/md-simulation-namd/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-06-03</td>
      <td>2023-11-09</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>namd_npt, setup, minimizer, namd_nvt</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>4269</td>
      <td>7123</td>
      <td>1.216667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Computational chemistry</td>
      <td>Setting up molecular systems</td>
      <td>https://training.galaxyproject.org//topics/computational-chemistry/tutorials/setting-up-molecular-systems/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-06-03</td>
      <td>2023-11-09</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1823</td>
      <td>2452</td>
      <td>1.616667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Contributing to the Galaxy Training Network with GitHub</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/github-contribution/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2017-06-12</td>
      <td>2025-03-14</td>
      <td>2</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3115642</td>
      <td>7964671</td>
      <td>3.783333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Overview of the Galaxy Training Material</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/introduction/slides.html</td>
      <td></td>
      <td></td>
      <td>2017-06-25</td>
      <td>2025-03-14</td>
      <td>2</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3115529</td>
      <td>7964543</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Principles of learning and how they apply to training and teaching</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/learning-principles/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-09-23</td>
      <td>2025-03-11</td>
      <td>17</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>5449</td>
      <td>6487</td>
      <td>0.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Teaching Python</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/python/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-10-19</td>
      <td>2023-11-23</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>329</td>
      <td>635</td>
      <td>0.933333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Adding auto-generated video to your slides</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/slides-with-video/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-10-20</td>
      <td>2023-11-09</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>280</td>
      <td>362</td>
      <td>3.466667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Contributing with GitHub via its interface</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/github-interface-contribution/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-06-12</td>
      <td>2023-11-09</td>
      <td>17</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>688</td>
      <td>1039</td>
      <td>2.383333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Generating PDF artefacts of the website</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/generating-pdf/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-06-04</td>
      <td>2023-11-09</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>80</td>
      <td>97</td>
      <td>2.300000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Preview the GTN website as you edit your training material</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/rendering_gtn/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-03-14</td>
      <td>2025-07-02</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>422</td>
      <td>568</td>
      <td>4.333333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Creating Interactive Galaxy Tours</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/create-new-tutorial-tours/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2017-06-25</td>
      <td>2025-04-08</td>
      <td>13</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>302</td>
      <td>460</td>
      <td>2.366667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>GTN Metadata</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/schemas/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-06-02</td>
      <td>2024-10-15</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>272</td>
      <td>359</td>
      <td>2.716667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Updating diffs in admin training</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/admin-knitting/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-06-07</td>
      <td>2023-11-23</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>68</td>
      <td>110</td>
      <td>0.666667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Tools, Data, and Workflows for tutorials</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/create-new-tutorial-technical/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2017-06-25</td>
      <td>2025-04-08</td>
      <td>26</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>562</td>
      <td>753</td>
      <td>1.650000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Including a new topic</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/create-new-topic/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-06-04</td>
      <td>2025-12-01</td>
      <td>33</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1255</td>
      <td>1713</td>
      <td>0.550000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Design and plan session, course, materials</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/design/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-09-23</td>
      <td>2025-03-11</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>614</td>
      <td>806</td>
      <td>2.633333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>FAIR-by-Design methodology</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/fair-by-design/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-10-17</td>
      <td>2024-10-17</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>47</td>
      <td>52</td>
      <td>0.466667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Adding Quizzes to your Tutorial</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/create-new-tutorial-quiz/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-05-19</td>
      <td>2022-10-18</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>114</td>
      <td>128</td>
      <td>0.283333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Creating a new tutorial</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/create-new-tutorial/tutorial.html</td>
      <td></td>
      <td>Read mapping, Data handling, Primer removal, Sequence trimming, Read pre-processing, Sequence file editing</td>
      <td>2018-06-20</td>
      <td>2025-11-20</td>
      <td>33</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>tp_sorted_uniq, c3s, seqtk_subseq, bam_to_sam, tp_sed_tool, Cut1, Filter1, tp_cat, tp_cut_tool, random_lines1, cutadapt, fastq_to_tabular, bowtie2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1826</td>
      <td>2688</td>
      <td>4.933333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>FAIR Galaxy Training Material</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/fair-gtn/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-12-16</td>
      <td>2025-12-16</td>
      <td>0</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Single Cell Publication - Data Analysis</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/meta-analysis-data/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-11-07</td>
      <td>2024-11-07</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>50</td>
      <td>52</td>
      <td>0.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Single Cell Publication - Data Plotting</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/meta-analysis-plot/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-11-07</td>
      <td>2024-11-07</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>45</td>
      <td>45</td>
      <td>0.083333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Creating content in Markdown</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/create-new-tutorial-content/tutorial.html</td>
      <td></td>
      <td>Sequencing quality control, Validation</td>
      <td>2017-06-25</td>
      <td>2025-04-08</td>
      <td>85</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>upload1, multiqc</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3767</td>
      <td>9783</td>
      <td>2.100000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Creating Slides</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/create-new-tutorial-slides/slides.html</td>
      <td></td>
      <td></td>
      <td>2018-06-13</td>
      <td>2024-11-08</td>
      <td>17</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3115529</td>
      <td>7964544</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Updating tool versions in a tutorial</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/updating_tutorial/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-11-29</td>
      <td>2024-11-29</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>109</td>
      <td>127</td>
      <td>3.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Automating Galaxy workflows using the command line</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/workflow-automation/tutorial.html</td>
      <td></td>
      <td>Visualisation, Read mapping, Sequence contamination filtering, Tree-based sequence alignment, Sequencing quality control, Validation, Variant classification</td>
      <td>2021-06-08</td>
      <td>2025-11-21</td>
      <td>10</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>falco, snpSift_filter, collapse_dataset, pangolin, compose_text_param, fastp, multiqc, bcftools_consensus</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>2915</td>
      <td>3528</td>
      <td>2.016667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Submitting sequence data to ENA</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/upload-data-to-ena/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-08-10</td>
      <td>2024-12-11</td>
      <td>10</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>ena_consensus_submit, ena_upload</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2794708</td>
      <td>6874080</td>
      <td>3.616667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Mutation calling, viral genome reconstruction and lineage/clade assignment from SARS-CoV-2 sequencing data</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/sars-cov-2-variant-discovery/tutorial.html</td>
      <td></td>
      <td>Tree-based sequence alignment, Methylation analysis, Variant classification, Variant calling</td>
      <td>2021-06-30</td>
      <td>2025-03-28</td>
      <td>18</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>upload1, pangolin, fasta_regex_finder, datamash_ops, Filter1, nextclade</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>7424</td>
      <td>13261</td>
      <td>3.416667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>From NCBI's Sequence Read Archive (SRA) to Galaxy: SARS-CoV-2 variant analysis</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/sars-cov-2/tutorial.html</td>
      <td></td>
      <td>Generation, Read mapping, SNP detection, Sequence contamination filtering, Sequencing quality control, Formatting, Validation, Genome indexing, Sequence alignment, Data handling</td>
      <td>2020-06-24</td>
      <td>2024-06-14</td>
      <td>16</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>Grep1, snpSift_extractFields, fasterq_dump, picard_MarkDuplicates, lofreq_viterbi, Filter1, samtools_stats, tp_cut_tool, snpeff_sars_cov_2, lofreq_call, fastp, multiqc, bwa_mem, lofreq_indelqual</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>MISSISSIPPI, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>13769</td>
      <td>20680</td>
      <td>2.866667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Sequence analysis</td>
      <td>Removal of human reads from SARS-CoV-2 sequencing data</td>
      <td>https://training.galaxyproject.org//topics/sequence-analysis/tutorials/human-reads-removal/tutorial.html</td>
      <td></td>
      <td>Generation, Read mapping, Sequence alignment, Data handling, Genome indexing, Sequence file editing</td>
      <td>2021-08-04</td>
      <td>2023-11-09</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>__ZIP_COLLECTION__, Grep1, tp_replace_in_line, seqtk_subseq, trimmomatic, samtools_fastx, __UNZIP_COLLECTION__, bwa_mem</td>
      <td>UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>Galaxy@AuBi, GalaxyTrakr, MISSISSIPPI, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>4104</td>
      <td>5812</td>
      <td>2.400000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>SRA Aligned Read Format to Speed Up SARS-CoV-2 data Analysis</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/ncbi-sarf/tutorial.html</td>
      <td></td>
      <td>SNP detection, Data handling</td>
      <td>2021-05-31</td>
      <td>2024-06-14</td>
      <td>12</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>fasterq_dump, fastq_dump, query_tabular, snpeff_sars_cov_2, tp_cat, jq, fastq_to_fasta_python</td>
      <td>UseGalaxy.be, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2896216</td>
      <td>7210027</td>
      <td>3.950000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Sequence analysis</td>
      <td>SARS-CoV-2 Viral Sample Alignment and Variant Visualization</td>
      <td>https://training.galaxyproject.org//topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.html</td>
      <td></td>
      <td>Genome indexing, Generation, Read mapping, Sequence alignment, Genome visualisation, Sequencing quality control, Statistical calculation, Sequence composition calculation</td>
      <td>2023-07-10</td>
      <td>2024-02-29</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>fastqc, bwa_mem, jbrowse</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>Galaxy@AuBi, GalaxyTrakr, MISSISSIPPI, UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>1289</td>
      <td>1669</td>
      <td>3.966667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Data Manipulation Olympics</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/data-manipulation-olympics/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-12-16</td>
      <td>2025-12-16</td>
      <td>0</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tp_sorted_uniq, Remove beginning1, tp_sort_header_tool, {{version_cut_columns}}, Show beginning1, regexColumn1, {{version_cat}}, {{version_datamash}}, {{version_paste}}, {{version_wc}}, {{version_cut_advanced}}, {{version_replace_text_column}}, datamash_ops, tp_split_on_column, {{version_compute}}, {{version_select_first}}, upload1, Count1, {{version_remove_beginning}}, {{version_sort}}, {{version_split}}, Filter1, tabular_to_csv, tp_cat, Grouping1, {{version_join}}, {{version_replace_text_line}}, {{version_unique}}, wc_gnu, Add_a_column1, cat1, {{version_filter}}, Cut1, join1, {{version_count}}, {{version_remove_columns_by_header}}</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>0.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>A (very) brief history of genomics</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/gnmx-lecture1/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-01-09</td>
      <td>2024-02-20</td>
      <td>2</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1395073</td>
      <td>3063521</td>
      <td>2.550000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Learning about one gene across biological resources and formats</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/online-resources-gene/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-09-07</td>
      <td>2023-11-09</td>
      <td>3</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>ncbi_blastp_wrapper</td>
      <td>Galaxy@AuBi, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>GalaxyTrakr, MISSISSIPPI, PepSimili, PhagePromotor, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>1550767</td>
      <td>3413894</td>
      <td>4.550000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Version Control with Git</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/bash-git/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-07-06</td>
      <td>2023-10-23</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>interactive_tool_rstudio</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>306</td>
      <td>537</td>
      <td>0.566667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Python - Warm-up for statistics and machine learning</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-warmup-stat-ml/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-03-11</td>
      <td>2025-03-11</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>144</td>
      <td>166</td>
      <td>0.833333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>SQL with Python</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/sql-python/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-10-11</td>
      <td>2023-11-16</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>513</td>
      <td>695</td>
      <td>0.466667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>SQL Educational Game - Murder Mystery</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/sql-game/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-10-11</td>
      <td>2023-11-23</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>688</td>
      <td>744</td>
      <td>0.366667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Variant Calling Workflow</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/bash-variant-calling/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-07-07</td>
      <td>2023-10-23</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>9099</td>
      <td>11872</td>
      <td>2.033333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Advanced CLI in Galaxy</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/cli-advanced/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-09-30</td>
      <td>2024-10-15</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>960</td>
      <td>1219</td>
      <td>2.333333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Introduction to SQL</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/sql-basic/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-10-11</td>
      <td>2024-12-13</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>592</td>
      <td>721</td>
      <td>1.366667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Introduction to Python</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-basics/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-11-08</td>
      <td>2024-10-15</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1405</td>
      <td>1830</td>
      <td>1.866667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>CLI basics</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/cli-basics/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-09-30</td>
      <td>2024-10-15</td>
      <td>10</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1156</td>
      <td>1627</td>
      <td>2.800000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>One protein along the UniProt page</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/online-resources-protein/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-09-07</td>
      <td>2024-12-05</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>474</td>
      <td>504</td>
      <td>0.433333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Python - Math</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-math/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-04-25</td>
      <td>2023-02-13</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>983</td>
      <td>1220</td>
      <td>1.283333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Make &amp; Snakemake</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/snakemake/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-11-18</td>
      <td>2025-07-09</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>3401</td>
      <td>4382</td>
      <td>0.933333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>CLI Educational Game - Bashcrawl</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/cli-bashcrawl/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-09-30</td>
      <td>2024-10-15</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>2098</td>
      <td>2381</td>
      <td>0.550000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Basics of using Git from the Command Line</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/git-cli/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-06-16</td>
      <td>2023-11-16</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>154</td>
      <td>399</td>
      <td>0.266667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>R basics in Galaxy</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/r-basics/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-10-08</td>
      <td>2024-06-14</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>interactive_tool_rstudio</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2789</td>
      <td>3771</td>
      <td>2.133333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Advanced SQL</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/sql-advanced/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-10-11</td>
      <td>2024-12-13</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1429</td>
      <td>1794</td>
      <td>0.550000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>SQL with R</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/sql-r/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-10-11</td>
      <td>2023-11-16</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>625</td>
      <td>786</td>
      <td>0.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Data visualisation Olympics - Visualization in R</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/data-manipulation-olympics-viz-r/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-06-16</td>
      <td>2024-05-07</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>642</td>
      <td>727</td>
      <td>0.966667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Advanced R in Galaxy</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/r-advanced/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-12-09</td>
      <td>2024-10-15</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>interactive_tool_rstudio</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1628</td>
      <td>2542</td>
      <td>2.783333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Python - Functions</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-functions/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-04-25</td>
      <td>2023-02-13</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>984</td>
      <td>1155</td>
      <td>0.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Introduction to sequencing with Python (part one)</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/gnmx-lecture2/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-01-23</td>
      <td>2024-02-20</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>257</td>
      <td>318</td>
      <td>1.900000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Advanced Python</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-advanced-np-pd/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-11-08</td>
      <td>2024-10-15</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>539</td>
      <td>718</td>
      <td>1.216667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Python - Basic Types &amp; Type Conversion</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-types/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-04-25</td>
      <td>2023-02-13</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>396</td>
      <td>674</td>
      <td>5.950000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Introduction to sequencing with Python (part two)</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/gnmx-lecture3/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-01-23</td>
      <td>2024-02-20</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>135</td>
      <td>141</td>
      <td>0.416667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>dplyr &amp; tidyverse for data processing</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/r-dplyr/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-10-20</td>
      <td>2024-10-15</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>631</td>
      <td>820</td>
      <td>0.816667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Plotting in Python</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-plotting/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-11-08</td>
      <td>2024-10-15</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>584</td>
      <td>706</td>
      <td>1.500000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Data Manipulation Olympics - SQL</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/data-manipulation-olympics-sql/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-06-16</td>
      <td>2024-05-07</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>823</td>
      <td>918</td>
      <td>0.533333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Data Manipulation Olympics - JQ</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/data-manipulation-olympics-jq/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-06-16</td>
      <td>2024-05-07</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>369</td>
      <td>420</td>
      <td>0.833333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Introduction to sequencing with Python (part three)</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/gnmx-lecture4/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-02-06</td>
      <td>2024-02-20</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>141</td>
      <td>151</td>
      <td>0.383333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Python - Lists &amp; Strings &amp; Dictionaries</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-iterables/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-04-25</td>
      <td>2024-06-25</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>2141</td>
      <td>2523</td>
      <td>0.550000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Introduction to sequencing with Python (part four)</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/gnmx-lecture5/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-02-13</td>
      <td>2024-02-20</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>148</td>
      <td>161</td>
      <td>0.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Python - Flow Control</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-flow/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-04-25</td>
      <td>2023-02-13</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>276</td>
      <td>327</td>
      <td>0.750000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Python - Loops</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-loops/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-04-25</td>
      <td>2023-02-13</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>388</td>
      <td>463</td>
      <td>1.633333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Data manipulation with Pandas</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/gnmx-lecture6/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-02-20</td>
      <td>2025-02-27</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>58</td>
      <td>64</td>
      <td>0.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Versioning your code and data with git</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/gnmx-lecture7/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-03-12</td>
      <td>2024-03-12</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>12</td>
      <td>12</td>
      <td>0.066667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Python - Files &amp; CSV</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-files/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-04-25</td>
      <td>2023-02-13</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>553</td>
      <td>642</td>
      <td>0.633333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Python - Try &amp; Except</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-exceptions/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-04-25</td>
      <td>2023-02-13</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>597</td>
      <td>683</td>
      <td>0.366667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Python - Introductory Graduation</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-basics-recap/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-04-25</td>
      <td>2023-02-13</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>238</td>
      <td>282</td>
      <td>1.583333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Conda Environments For Software Development</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-conda/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-10-19</td>
      <td>2023-05-18</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>697</td>
      <td>750</td>
      <td>0.533333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Python - Argparse</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-argparse/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-04-25</td>
      <td>2023-01-24</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1333</td>
      <td>1449</td>
      <td>0.400000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Virtual Environments For Software Development</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-venv/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-10-19</td>
      <td>2023-05-18</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>207</td>
      <td>229</td>
      <td>1.166667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Python - Type annotations</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-typing/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-10-19</td>
      <td>2023-02-13</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>628</td>
      <td>665</td>
      <td>0.183333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Python - Testing</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-testing/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-10-19</td>
      <td>2023-02-13</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>228</td>
      <td>251</td>
      <td>0.366667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Python - Globbing</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-glob/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-04-25</td>
      <td>2023-02-13</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>537</td>
      <td>579</td>
      <td>0.433333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Python - Subprocess</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-subprocess/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-04-25</td>
      <td>2023-02-13</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1164</td>
      <td>1349</td>
      <td>0.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Python - Coding Style</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-linting/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-11-03</td>
      <td>2023-05-18</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>188</td>
      <td>201</td>
      <td>0.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Python - Multiprocessing</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-multiprocessing/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-10-19</td>
      <td>2023-11-16</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>871</td>
      <td>936</td>
      <td>0.283333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Galaxy Interactive Environments</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/interactive-environments/slides.html</td>
      <td></td>
      <td></td>
      <td>2017-02-05</td>
      <td>2023-02-17</td>
      <td>16</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3115529</td>
      <td>7964552</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Galaxy Code Architecture</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/architecture/slides.html</td>
      <td></td>
      <td></td>
      <td>2017-06-20</td>
      <td>2024-06-14</td>
      <td>34</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3115529</td>
      <td>7964552</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Scripting Galaxy using the API and BioBlend</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/bioblend-api/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2017-01-17</td>
      <td>2023-11-16</td>
      <td>22</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3116272</td>
      <td>7965518</td>
      <td>4.466667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>JavaScript plugins</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/visualization-charts/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2017-06-22</td>
      <td>2025-04-08</td>
      <td>20</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3116250</td>
      <td>7965410</td>
      <td>3.266667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Tool Shed: sharing Galaxy tools</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/toolshed/slides.html</td>
      <td></td>
      <td></td>
      <td>2017-06-12</td>
      <td>2022-10-18</td>
      <td>15</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3115529</td>
      <td>7964552</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Tool Dependencies and Containers</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/containers/slides.html</td>
      <td></td>
      <td></td>
      <td>2017-06-20</td>
      <td>2022-10-18</td>
      <td>14</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3115529</td>
      <td>7964552</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Prerequisites for building software/conda packages</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/conda_sys/slides.html</td>
      <td></td>
      <td></td>
      <td>2018-02-15</td>
      <td>2021-05-28</td>
      <td>6</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3115529</td>
      <td>7964552</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Generic plugins</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/visualization-generic/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2016-09-30</td>
      <td>2023-11-09</td>
      <td>21</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3115885</td>
      <td>7965060</td>
      <td>4.100000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>ToolFactory: Generating Tools From Simple Scripts</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/tool-generators/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-05-26</td>
      <td>2024-06-14</td>
      <td>10</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2903930</td>
      <td>7235341</td>
      <td>4.733333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Galaxy Webhooks</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/webhooks/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2017-04-03</td>
      <td>2025-04-08</td>
      <td>16</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3115725</td>
      <td>7964813</td>
      <td>3.616667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Tool Dependencies and Conda</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/conda/slides.html</td>
      <td></td>
      <td></td>
      <td>2017-06-20</td>
      <td>2023-04-14</td>
      <td>33</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3115529</td>
      <td>7964553</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Tool development and integration into Galaxy</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/tool-integration/slides.html</td>
      <td></td>
      <td></td>
      <td>2016-09-30</td>
      <td>2025-04-08</td>
      <td>30</td>
      <td>external</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3115532</td>
      <td>7964557</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Galaxy Interactive Tours</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/interactive-tours/slides.html</td>
      <td></td>
      <td></td>
      <td>2017-01-12</td>
      <td>2021-07-27</td>
      <td>14</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3115529</td>
      <td>7964553</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>nb2workflow: Generating Galaxy Tools From Jupyter Notebooks</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/tool-from-notebook/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-12-04</td>
      <td>2025-12-04</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>5</td>
      <td>6</td>
      <td>3.366667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>ToolFactory: Generating Tools From More Complex Scripts</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/tool-generators-advanced/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-05-26</td>
      <td>2024-09-27</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>434</td>
      <td>518</td>
      <td>1.500000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Data source integration</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/data-source-integration/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2016-09-30</td>
      <td>2022-10-18</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>233</td>
      <td>342</td>
      <td>2.833333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Galaxy Interactive Tools</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/interactive-tools/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-03-02</td>
      <td>2024-12-03</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1236</td>
      <td>1638</td>
      <td>2.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Contributing to BioBlend as a developer</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/bioblend-dev/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-05-25</td>
      <td>2023-11-09</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>249</td>
      <td>339</td>
      <td>2.933333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Creating Galaxy tools from Conda Through Deployment</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/tool-from-scratch/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-06-15</td>
      <td>2024-10-08</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2771</td>
      <td>4461</td>
      <td>2.533333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Debugging Galaxy</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/debugging/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-06-15</td>
      <td>2025-11-05</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>603</td>
      <td>1207</td>
      <td>2.783333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Adding and updating best practice metadata for Galaxy tools using the bio.tools registry</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/tool-annotation/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-03-06</td>
      <td>2024-11-06</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>281</td>
      <td>336</td>
      <td>2.283333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Contributing a New Feature to Galaxy Core</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/core-contributing/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-06-08</td>
      <td>2025-11-05</td>
      <td>10</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>739</td>
      <td>1103</td>
      <td>2.633333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Writing Automated Tests for Galaxy</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/writing_tests/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-07-15</td>
      <td>2025-11-05</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>284</td>
      <td>712</td>
      <td>1.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Setting up a dev Onedata instance</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/onedata-dev-instance/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-03-24</td>
      <td>2025-04-08</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>31</td>
      <td>34</td>
      <td>1.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Digital Humanities</td>
      <td>Introduction to Digital Humanities in Galaxy</td>
      <td>https://training.galaxyproject.org//topics/digital-humanities/tutorials/introduction_to_dh/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-09-11</td>
      <td>2025-11-25</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>wordcloud, Remove beginning1, diff, tp_grep_tool, tp_find_and_replace, tp_replace_in_line, wc_gnu</td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td>UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>79</td>
      <td>95</td>
      <td>1.933333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Digital Humanities</td>
      <td>Text-Mining Differences in Chinese Newspaper Articles</td>
      <td>https://training.galaxyproject.org//topics/digital-humanities/tutorials/text_mining_chinese/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-02-28</td>
      <td>2025-09-09</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Add_a_column1, wordcloud, diff, Cut1, Filter1, datamash_ops, sort1, tp_replace_in_line</td>
      <td></td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>162</td>
      <td>189</td>
      <td>1.466667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Digital Humanities</td>
      <td>OpenRefine Tutorial for researching cultural data</td>
      <td>https://training.galaxyproject.org//topics/digital-humanities/tutorials/open-refine-tutorial/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-09-24</td>
      <td>2025-10-21</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>wordcloud, Remove beginning1, tp_sort_header_tool, barchart_gnuplot, filter_tabular, interactive_tool_openrefine, Cut1, param_value_from_file, regexColumn1, tp_head_tool, tp_grep_tool, datamash_ops, compose_text_param</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>40</td>
      <td>44</td>
      <td>0.016667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Creating metadata using Ecological Metadata Language (EML) standard with EML Assembly Line functionalities</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/MetaShARK_tutorial/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-03-04</td>
      <td>2024-09-10</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>interactive_tool_metashark</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>212</td>
      <td>257</td>
      <td>1.783333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Phylodiversity analysis quick tutorial</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/phylodiversity_workflow/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-06-06</td>
      <td>2025-06-06</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>crsconverter, phylogenetic_occupancy_matcher, phylo_index, estimate_endem</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>94</td>
      <td>119</td>
      <td>0.816667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Sentinel 2 biodiversity</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/remote-sensing/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-04-13</td>
      <td>2025-12-08</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>srs_pca, srs_preprocess_s2, srs_global_indices, srs_process_data, srs_spectral_indices, srs_diversity_maps</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>634</td>
      <td>810</td>
      <td>1.583333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Compute and analyze biodiversity metrics with PAMPA toolsuite</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/PAMPA-toolsuite-tutorial/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-11-19</td>
      <td>2025-09-03</td>
      <td>23</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>regex1, pampa_glmcomm, tp_sort_header_tool, pampa_communitymetrics, Count1, Filter1, regexColumn1, tp_cat, tp_cut_tool, mergeCols1, pampa_plotglm, tp_easyjoin_tool, datamash_transpose, pampa_glmsp, pampa_presabs</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>1120</td>
      <td>1487</td>
      <td>1.916667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>RAD-Seq to construct genetic maps</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/genetic-map-rad-seq/tutorial.html</td>
      <td></td>
      <td>Data handling</td>
      <td>2017-02-14</td>
      <td>2025-06-03</td>
      <td>13</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>stacks_denovomap, stacks_genotypes</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>906</td>
      <td>1268</td>
      <td>1.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Champs blocs indicators</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/champs-blocs/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-01-04</td>
      <td>2025-12-02</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>cb_eco, cb_ivr, cb_qecb, cb_dissim, cb_div</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>181</td>
      <td>331</td>
      <td>1.816667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Species distribution modeling</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/species-distribution-modeling/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-07-04</td>
      <td>2025-06-03</td>
      <td>18</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>spocc_occ, Filter1, tabular_to_csv, tp_cut_tool, tp_find_and_replace, interactive_tool_wallace</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3221</td>
      <td>4856</td>
      <td>1.150000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Visualization of Climate Data using  NetCDF xarray Map Plotting</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/x-array-map-plot/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-10-26</td>
      <td>2024-06-27</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>graphicsmagick_image_montage, xarray_coords_info, xarray_metadata_info, cdo_operations, xarray_mapplot</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>2402</td>
      <td>2777</td>
      <td>0.450000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>RAD-Seq de-novo data analysis</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/de-novo-rad-seq/tutorial.html</td>
      <td></td>
      <td>Data handling, Sequencing quality control, Statistical calculation, Sequence composition calculation</td>
      <td>2017-02-14</td>
      <td>2025-06-03</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>stacks_populations, stacks_denovomap, fastqc, stacks_procrad</td>
      <td>UseGalaxy.be, UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>2718</td>
      <td>4315</td>
      <td>2.083333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Regional GAM</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/regionalGAM/tutorial.html</td>
      <td>Ecology</td>
      <td>Visualisation</td>
      <td>2019-07-04</td>
      <td>2025-06-03</td>
      <td>18</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tab2csv_R, regionalgam_gls, Remove beginning1, regionalgam_ab_index, regionalgam_glmmpql, Count1, regionalgam_autocor_acf, regionalgam_plot_trend, regionalgam_flight_curve, ggplot2_point, Filter1, regexColumn1, flight-curve, Paste1, mergeCols1, tp_awk_tool, csv2tab_R</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>351</td>
      <td>469</td>
      <td>3.400000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Metabarcoding/eDNA through Obitools</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/Obitools-metabarcoding/tutorial.html</td>
      <td></td>
      <td>Sequence analysis, Sequencing quality control, Statistical calculation, Sequence composition calculation</td>
      <td>2022-02-28</td>
      <td>2025-06-03</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>obi_ngsfilter, obi_tab, obi_uniq, ncbi_blastn_wrapper, obi_grep, obi_illumina_pairend, fastq_groomer, Cut1, Filter1, obi_annotate, obi_stat, seq_filter_by_id, fastqc, join1, unzip, obi_clean, wc_gnu</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>4572</td>
      <td>6351</td>
      <td>1.933333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Obis marine indicators</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/obisindicators/tutorial.html</td>
      <td>Biodiversity, Ecology, Data mining</td>
      <td></td>
      <td>2023-04-11</td>
      <td>2025-06-03</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tp_cut_tool, obisindicators, obis_data</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>320</td>
      <td>400</td>
      <td>2.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Preparing genomic data for phylogeny reconstruction</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/phylogeny-data-prep/tutorial.html</td>
      <td></td>
      <td>Sequence assembly validation, Genome annotation, Sequence clustering, Sequence analysis, Multiple sequence alignment, Genetic variation analysis</td>
      <td>2022-10-21</td>
      <td>2025-06-03</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>regex1, busco, collapse_dataset, clipkit, Filter1, funannotate_predict, glimmer_gbk_to_orf, proteinortho, clustalw, proteinortho_grab_proteins, phykit_alignment_based, tp_replace_in_line, repeatmasker_wrapper</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2267</td>
      <td>3123</td>
      <td>2.816667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Ecoregionalization workflow tutorial</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/Ecoregionalization_tutorial/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-09-16</td>
      <td>2025-06-03</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>ecoregion_GeoNearestNeighbor, ecoregion_taxa_seeker, filter_tabular, regexColumn1, ecoregion_eco_map, ecoregion_brt_analysis, tp_cut_tool, mergeCols1, ecoregion_clara_cluster, interactive_tool_jupyter_notebook, unzip, ecoregion_cluster_estimate</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>293</td>
      <td>387</td>
      <td>2.566667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Life Traits Ecoregionalization workflow</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/ecoregiolifetraits/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-06-02</td>
      <td>2025-06-10</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>ecoregion_GeoNearestNeighbor, ecoregion_taxa_seeker, WormsMeasurements, column_remove_by_header, Filter1, regexColumn1, ecoregion_eco_map, ecoregion_brt_analysis, tp_cut_tool, ClaraGuess, unzip</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>46</td>
      <td>49</td>
      <td>2.100000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Checking expected species and contamination in bacterial isolate</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/bacterial-isolate-species-contamination-checking/tutorial.html</td>
      <td>Whole genome sequencing, Genomics, Microbiology, Microbial ecology</td>
      <td>Expression analysis, Statistical calculation, Taxonomic classification, Cross-assembly</td>
      <td>2024-03-04</td>
      <td>2024-06-27</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>upload1, kraken2, est_abundance, recentrifuge</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td>Galaxy@AuBi, GalaxyTrakr, UseGalaxy.cz, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>899</td>
      <td>1194</td>
      <td>3.300000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>RAD-Seq Reference-based data analysis</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/ref-based-rad-seq/tutorial.html</td>
      <td></td>
      <td>Generation, Read mapping, Sequencing quality control, Genome indexing, Validation, Sequence alignment, Data handling, Statistical calculation, Sequence composition calculation</td>
      <td>2017-02-14</td>
      <td>2025-11-06</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>addValue, regex_replace, Grep1, tp_sort_header_tool, Count1, cat1, Cut1, Filter1, bwa_wrapper, stacks_populations, stacks_procrad, Convert characters1, fastqc, stacks_refmap, Summary_Statistics1, multiqc, tp_replace_in_line, bwa</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3180</td>
      <td>4945</td>
      <td>1.816667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Creating FAIR Quality assessment reports and draft of Data Papers from EML metadata with MetaShRIMPS</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/Metashrimps_tutorial/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-01-22</td>
      <td>2025-06-03</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>interactive_tool_metashrimps</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>115</td>
      <td>125</td>
      <td>1.450000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Taxonomic Analysis of eDNA</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/eDNA-taxonomic-analysis/tutorial.html</td>
      <td></td>
      <td>Sequencing quality control, Sequence contamination filtering</td>
      <td>2025-06-02</td>
      <td>2025-06-03</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Count1, fastp, ncbi_blastn_wrapper, cshl_fastq_to_fasta</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>Galaxy@AuBi, GalaxyTrakr, MISSISSIPPI, UseGalaxy.be, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>488</td>
      <td>613</td>
      <td>0.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Cleaning GBIF data for the use in Ecology</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/gbif_cleaning/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-10-28</td>
      <td>2025-06-03</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>spocc_occ, Count1, Filter1, Summary_Statistics1, gdal_ogr2ogr</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.no</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>654</td>
      <td>809</td>
      <td>0.916667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>From NDVI data with OpenEO to time series visualisation with Holoviews</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/ndvi_openeo/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-01-24</td>
      <td>2024-06-29</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>interactive_tool_holoviz, interactive_tool_copernicus_notebook</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>218</td>
      <td>257</td>
      <td>0.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Marine Omics identifying biosynthetic gene clusters</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/marine_omics_bgc/tutorial.html</td>
      <td></td>
      <td>Sequence motif recognition, Genome annotation, Protein feature detection</td>
      <td>2024-08-19</td>
      <td>2025-06-03</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>sanntis_marine, interproscan, regex1, prodigal, SanntiS_marine</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>195</td>
      <td>229</td>
      <td>2.283333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Biodiversity data exploration</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/biodiversity-data-exploration/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-03-01</td>
      <td>2025-06-03</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>Remove beginning1, regexColumn1, ecology_beta_diversity, ecology_homogeneity_normality, tp_cut_tool, ecology_link_between_var, ecology_stat_presence_abs, ecology_presence_abs_abund, tool_anonymization</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1643</td>
      <td>2296</td>
      <td>2.700000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Data submission using ENA upload Tool</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/ENA_Biodiv_submission/tutorial.html</td>
      <td></td>
      <td>Visualisation, Read mapping, Primer removal, Sequence trimming, Sequencing quality control, Validation, Read pre-processing</td>
      <td>2024-12-13</td>
      <td>2025-03-20</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>regex1, falco, __RELABEL_FROM_FILE__, bg_uniq, ncbi_blastn_wrapper, fastq_groomer, ab1_fastq_converter, Paste1, collection_element_identifiers, cutadapt, multiqc, __FILTER_EMPTY_DATASETS__, fastq_filter, fastq_to_fasta_python</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>262</td>
      <td>278</td>
      <td>0.633333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Visualize EBV cube data with Panoply netCDF viewer</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/panoply_ebv/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-10-28</td>
      <td>2024-06-27</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>interactive_tool_panoply</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>245</td>
      <td>308</td>
      <td>1.216667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Audio data annotation with NEAL (Nature + Energy Audio labeler)</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/NEAL_tutorial/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-10-14</td>
      <td>2025-11-06</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>22</td>
      <td>25</td>
      <td>0.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Cleaning GBIF data using OpenRefine</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/openrefine_gbif/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-01-21</td>
      <td>2025-06-03</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>regex1, interactive_tool_openrefine</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>168</td>
      <td>205</td>
      <td>1.083333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>QGIS Web Feature Services</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/QGIS_Web_Feature_Services/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-10-11</td>
      <td>2025-06-03</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>interactive_tool_qgis</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>289</td>
      <td>332</td>
      <td>1.616667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Foundational Aspects of Machine Learning using Python</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/intro-to-ml-with-python/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-03-11</td>
      <td>2025-05-19</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>666</td>
      <td>834</td>
      <td>1.383333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Pretraining a Large Language Model (LLM) from Scratch on DNA Sequences</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/genomic-llm-pretraining/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-04-17</td>
      <td>2025-05-22</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1023</td>
      <td>1222</td>
      <td>1.500000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Teaching and Hosting Galaxy training</td>
      <td>Motivation and Demotivation</td>
      <td>https://training.galaxyproject.org//topics/teaching/tutorials/motivation_demotivation/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-09-23</td>
      <td>2025-03-11</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1940</td>
      <td>2299</td>
      <td>0.666667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Teaching and Hosting Galaxy training</td>
      <td>Train-the-Trainer: putting it all together</td>
      <td>https://training.galaxyproject.org//topics/teaching/tutorials/workshop-final-exercise/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-06-12</td>
      <td>2025-03-11</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>99</td>
      <td>114</td>
      <td>1.166667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Teaching and Hosting Galaxy training</td>
      <td>Training techniques to enhance learner participation and engagement</td>
      <td>https://training.galaxyproject.org//topics/teaching/tutorials/learner_participation_engagement/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-09-23</td>
      <td>2025-03-11</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1632</td>
      <td>1918</td>
      <td>0.733333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Teaching and Hosting Galaxy training</td>
      <td>Assessment and feedback in training and teachings</td>
      <td>https://training.galaxyproject.org//topics/teaching/tutorials/assessment/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-09-23</td>
      <td>2025-03-11</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1281</td>
      <td>1481</td>
      <td>0.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Fine-tuning a LLM for DNA Sequence Classification</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/genomic-llm-finetuning/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-04-17</td>
      <td>2025-05-22</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>591</td>
      <td>690</td>
      <td>1.266667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Neural networks using Python</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/neural-networks-with-python/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-03-11</td>
      <td>2025-03-11</td>
      <td>1</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>732104</td>
      <td>1447683</td>
      <td>2.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Predicting Mutation Impact with Zero-shot Learning using a pretrained DNA LLM</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/genomic-llm-zeroshot-prediction/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-04-17</td>
      <td>2025-05-22</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>194</td>
      <td>219</td>
      <td>1.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Deep Learning (without Generative Artificial Intelligence) using Python</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/deep-learning-without-gai-with-python/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-03-11</td>
      <td>2025-03-11</td>
      <td>1</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>732105</td>
      <td>1447684</td>
      <td>1.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Generating Artificial Yeast DNA Sequences using a DNA LLM</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/genomic-llm-sequence-generation/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-04-17</td>
      <td>2025-05-22</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>159</td>
      <td>179</td>
      <td>0.750000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Optimizing DNA Sequences for Biological Functions using a DNA LLM</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/genomic-llm-sequence-optimization/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-04-17</td>
      <td>2025-05-22</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>39</td>
      <td>41</td>
      <td>0.850000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Regulations/standards for AI using DOME</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/dome/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-03-11</td>
      <td>2025-05-19</td>
      <td>3</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>732143</td>
      <td>1447736</td>
      <td>2.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Epigenetics</td>
      <td>Formation of the Super-Structures on the Inactive X</td>
      <td>https://training.galaxyproject.org//topics/epigenetics/tutorials/formation_of_super-structures_on_xi/tutorial.html</td>
      <td></td>
      <td>Pathway visualisation, Read mapping, Sequence trimming, Sequencing quality control, Sequence composition calculation, Peak calling, Gene regulatory network analysis, Structure visualisation, Genome visualisation, Primer removal, Mapping, Statistical calculation, Read pre-processing, Enrichment analysis</td>
      <td>2018-09-17</td>
      <td>2025-04-08</td>
      <td>14</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>samtools_idxstats, deeptools_multi_bam_summary, bedtools_mergebed, deeptools_compute_matrix, deeptools_plot_correlation, bbgtobigwig, deeptools_plot_fingerprint, cat1, jbrowse2, deeptools_plot_heatmap, Filter1, trim_galore, fastqc, macs2_callpeak, deeptools_bam_coverage, bedtools_sortbed, deeptools_bam_compare, bowtie2</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3121560</td>
      <td>7976416</td>
      <td>5.933333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Epigenetics</td>
      <td>CUT&amp;RUN data analysis</td>
      <td>https://training.galaxyproject.org//topics/epigenetics/tutorials/cut_and_run/tutorial.html</td>
      <td></td>
      <td>Visualisation, Read mapping, Sequence trimming, Sequencing quality control, Peak calling, Formatting, Validation, Sequence alignment analysis, Gene regulatory network analysis, Data handling, Primer removal, Mapping, Read pre-processing, Enrichment analysis</td>
      <td>2024-03-06</td>
      <td>2025-03-19</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>falco, deeptools_compute_matrix, Grep1, idr, wig_to_bigWig, picard_MarkDuplicates, deeptools_plot_heatmap, bamFilter, __EXTRACT_DATASET__, multiqc, bedtools_intersectbed, meme_chip, __FLATTEN__, trim_galore, macs2_callpeak, bowtie2, pe_histogram, deeptools_plot_fingerprint, datamash_transpose, bedtools_bamtobed, Extract genomic DNA 1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>6375</td>
      <td>8118</td>
      <td>1.366667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Epigenetics</td>
      <td>ATAC-Seq data analysis</td>
      <td>https://training.galaxyproject.org//topics/epigenetics/tutorials/atac-seq/tutorial.html</td>
      <td></td>
      <td>Visualisation, Read mapping, Peak calling, Sequencing quality control, Gene regulatory network analysis, Formatting, Sequence alignment analysis, Sequence trimming, Data handling, Primer removal, Mapping, Read pre-processing, Statistical calculation, Sequence composition calculation, Enrichment analysis</td>
      <td>2019-09-02</td>
      <td>2024-08-08</td>
      <td>23</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>samtools_idxstats, deeptools_compute_matrix, wig_to_bigWig, Grep1, tp_sort_header_tool, bedtools_intersectbed, pe_histogram, picard_MarkDuplicates, deeptools_plot_heatmap, Cut1, Filter1, bedtools_bamtobed, fastqc, macs2_callpeak, bamFilter, cutadapt, pygenomeTracks, bowtie2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3180236</td>
      <td>8080573</td>
      <td>4.100000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Epigenetics</td>
      <td>Hi-C analysis of Drosophila melanogaster cells using HiCExplorer</td>
      <td>https://training.galaxyproject.org//topics/epigenetics/tutorials/hicexplorer/tutorial.html</td>
      <td></td>
      <td>Visualisation, Read mapping, Formatting</td>
      <td>2018-02-23</td>
      <td>2025-08-11</td>
      <td>30</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>bowtie2, hicexplorer_hicinfo, hicexplorer_hicmergematrixbins, hicexplorer_hicpca, pygenomeTracks, hicexplorer_hiccorrectmatrix, hicexplorer_hicfindrestrictionsites, hicexplorer_hicfindtads, hicexplorer_hicdetectloops, hicexplorer_hicbuildmatrix, hicexplorer_hicplotmatrix</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>7572</td>
      <td>12655</td>
      <td>1.483333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Epigenetics</td>
      <td>Identification of the binding sites of the Estrogen receptor</td>
      <td>https://training.galaxyproject.org//topics/epigenetics/tutorials/estrogen-receptor-binding-site-identification/tutorial.html</td>
      <td></td>
      <td>Read mapping, Sequence trimming, Sequencing quality control, Peak calling, Gene regulatory network analysis, Primer removal, Read pre-processing, Statistical calculation, Sequence composition calculation, Enrichment analysis</td>
      <td>2017-09-18</td>
      <td>2025-04-08</td>
      <td>10</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>deeptools_compute_gc_bias, deeptools_multi_bam_summary, samtools_idxstats, deeptools_compute_matrix, deeptools_plot_correlation, bam_to_sam, deeptools_plot_fingerprint, deeptools_plot_heatmap, trim_galore, fastqc, macs2_callpeak, deeptools_bam_coverage, deeptools_bam_compare, bowtie2</td>
      <td>UseGalaxy.be</td>
      <td>Galaxy@AuBi, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>1939</td>
      <td>2853</td>
      <td>2.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Epigenetics</td>
      <td>Identification of the binding sites of the T-cell acute lymphocytic leukemia protein 1 (TAL1)</td>
      <td>https://training.galaxyproject.org//topics/epigenetics/tutorials/tal1-binding-site-identification/tutorial.html</td>
      <td>ChIP-seq</td>
      <td>Generation, Read mapping, Peak calling, Sequencing quality control, Gene regulatory network analysis, Genome indexing, Sequence alignment, Mapping, Statistical calculation, Sequence composition calculation, Enrichment analysis</td>
      <td>2016-12-20</td>
      <td>2025-04-08</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>deeptools_compute_gc_bias, deeptools_multi_bam_summary, samtools_idxstats, deeptools_compute_matrix, deeptools_plot_correlation, bedtools_intersectbed, deeptools_plot_fingerprint, deeptools_plot_heatmap, deeptools_correct_gc_bias, trimmomatic, fastqc, macs2_callpeak, bwa, deeptools_bam_compare</td>
      <td>UseGalaxy.be, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>Galaxy@AuBi, MISSISSIPPI, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>7445</td>
      <td>13741</td>
      <td>3.350000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Epigenetics</td>
      <td>Infinium Human Methylation BeadChip</td>
      <td>https://training.galaxyproject.org//topics/epigenetics/tutorials/ewas-suite/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-08-27</td>
      <td>2024-10-15</td>
      <td>21</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>clusterprofiler_go, Remove beginning1, clusterprofiler_bitr, Cut1, chipeakanno_annopeaks, minfi_analysis</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3117619</td>
      <td>7967827</td>
      <td>4.183333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Epigenetics</td>
      <td>DNA Methylation data analysis</td>
      <td>https://training.galaxyproject.org//topics/epigenetics/tutorials/methylation-seq/tutorial.html</td>
      <td>Epigenomics</td>
      <td>Visualisation, Read mapping, Sequencing quality control, Statistical calculation, Sequence composition calculation</td>
      <td>2017-02-16</td>
      <td>2025-04-03</td>
      <td>24</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>falco, deeptools_compute_matrix, wig_to_bigWig, pileometh, deeptools_plot_profile, replace_column_with_key_value_file, tp_tail_tool, metilene, bwameth, fastqc, tp_awk_tool</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>15138</td>
      <td>25109</td>
      <td>2.033333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Epigenetics</td>
      <td>Introduction to ChIP-Seq data analysis</td>
      <td>https://training.galaxyproject.org//topics/epigenetics/tutorials/introduction-chip-seq/slides.html</td>
      <td></td>
      <td></td>
      <td>2017-06-12</td>
      <td>2024-03-05</td>
      <td>2</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>3115529</td>
      <td>7964558</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Epigenetics</td>
      <td>Introduction to DNA Methylation data analysis</td>
      <td>https://training.galaxyproject.org//topics/epigenetics/tutorials/introduction-dna-methylation/slides.html</td>
      <td></td>
      <td></td>
      <td>2016-10-05</td>
      <td>2024-03-05</td>
      <td>2</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>3115529</td>
      <td>7964558</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Phylogenetics - Back to basics</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/abc_intro_phylo/tutorial.html</td>
      <td></td>
      <td>Phylogenetic tree analysis, Phylogenetic analysis, Phylogenetic tree reconstruction, Sequence analysis, Phylogenetic tree generation (maximum likelihood and Bayesian methods), Phylogenetic tree generation, Multiple sequence alignment, Phylogenetic tree generation (from molecular sequences)</td>
      <td>2024-05-10</td>
      <td>2024-05-10</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>iqtree, rbc_mafft, newick_display, fasttree</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>GalaxyTrakr, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>5198</td>
      <td>6709</td>
      <td>1.850000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Identifying tuberculosis transmission links: from SNPs to transmission clusters</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/mtb_transmission/tutorial.html</td>
      <td>Genomics, Microbiology, Infectious disease, DNA polymorphism</td>
      <td>Phylogenetic tree visualisation, Antimicrobial resistance prediction, Phylogenetic tree generation, Variant calling</td>
      <td>2022-03-16</td>
      <td>2025-05-26</td>
      <td>13</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Grep1, snp_sites, tp_grep_tool, tb_variant_filter, snippy, snp_dists, tp_sed_tool, tp_easyjoin_tool, upload1, collapse_dataset, trimmomatic, __MERGE_COLLECTION__, tp_cat, tb_profiler_profile, bcftools_consensus, samtools_view, clustering_from_distmat, addName, tp_replace_in_line</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2979</td>
      <td>4276</td>
      <td>2.650000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Tree thinking for tuberculosis evolution and epidemiology</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/mtb_phylogeny/tutorial.html</td>
      <td>Genomics, Microbiology, Phylogeny, Infectious disease</td>
      <td>Sequence analysis, Phylogenetic tree analysis</td>
      <td>2022-03-16</td>
      <td>2025-06-04</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>raxml, upload1, interactive_tool_rstudio</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2286</td>
      <td>3297</td>
      <td>2.500000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Phylogenetics - Back to Basics - Introduction</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/abc_intro_phylo-intro/slides.html</td>
      <td></td>
      <td></td>
      <td>2024-05-10</td>
      <td>2024-05-10</td>
      <td>1</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1194575</td>
      <td>2585861</td>
      <td>1.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Phylogenetic analysis for bacterial comparative genomics</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/bacterial-comparative-genomics/tutorial.html</td>
      <td>Genomics, Microbiology, Phylogeny</td>
      <td>Phylogenetic tree generation (maximum likelihood and Bayesian methods), Phylogenetic tree generation (from molecular sequences)</td>
      <td>2025-03-14</td>
      <td>2025-03-14</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>upload1, fasttree</td>
      <td>Galaxy@AuBi, MISSISSIPPI, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>GalaxyTrakr</td>
      <td>0</td>
      <td></td>
      <td>108</td>
      <td>130</td>
      <td>2.266667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Phylogenetics - Back to Basics - Terminology</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/abc_intro_phylo-terminology/slides.html</td>
      <td></td>
      <td></td>
      <td>2024-05-10</td>
      <td>2024-05-10</td>
      <td>1</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1194575</td>
      <td>2585861</td>
      <td>1.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Phylogenetics - Back to Basics - Multiple Sequence Alignment</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/abc_intro_phylo-msa/slides.html</td>
      <td></td>
      <td></td>
      <td>2024-05-10</td>
      <td>2024-05-10</td>
      <td>1</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1194575</td>
      <td>2585862</td>
      <td>1.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Phylogenetics - Back to Basics - Building Trees</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/abc_intro_phylo-trees/slides.html</td>
      <td></td>
      <td></td>
      <td>2024-05-10</td>
      <td>2024-05-10</td>
      <td>1</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1194575</td>
      <td>2585862</td>
      <td>1.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Phylogenetics - Back to Basics - Estimating trees from alignments</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/abc_intro_phylo-mle-trees/slides.html</td>
      <td></td>
      <td></td>
      <td>2024-05-10</td>
      <td>2024-05-10</td>
      <td>1</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1194575</td>
      <td>2585862</td>
      <td>1.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Phylogenetics - Back to Basics - Phylogenetic Networks</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/abc_intro_phylo-networks/slides.html</td>
      <td></td>
      <td></td>
      <td>2024-05-10</td>
      <td>2024-05-10</td>
      <td>1</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1194575</td>
      <td>2585862</td>
      <td>1.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>Data management in Medicinal Chemistry</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/med-chem-data/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-12-16</td>
      <td>2025-12-16</td>
      <td>0</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>ctb_silicos_qed, ctb_im_rxn_maker, chembl, get_pdb, md_converter, openbabel_compound_convert, openbabel_svg_depiction</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>An earth-system RO-Crate</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/earth_system_rocrate/slides.html</td>
      <td></td>
      <td></td>
      <td>2024-12-02</td>
      <td>2024-12-02</td>
      <td>1</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>881546</td>
      <td>1816165</td>
      <td>1.750000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>Uploading Data to Zenodo from Galaxy</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/zenodo-in-galaxy/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-11-06</td>
      <td>2025-11-06</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>17</td>
      <td>19</td>
      <td>0.750000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>Integrating InvenioRDM-compatible Repositories with Galaxy</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/inveniordm-integration/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-11-06</td>
      <td>2025-11-06</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>17</td>
      <td>18</td>
      <td>1.466667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>RO-Crate - Introduction</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/ro-crate-intro/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-05-23</td>
      <td>2025-04-08</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>617</td>
      <td>808</td>
      <td>1.700000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>FAIR in a nutshell</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/fair-intro/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-05-30</td>
      <td>2024-04-30</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>880</td>
      <td>1128</td>
      <td>1.683333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>Sequence data submission to ENA</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/fair-ena/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-11-01</td>
      <td>2024-06-25</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>602</td>
      <td>704</td>
      <td>1.016667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>Introduction to Data Management Plan (DMP) for Peatland Research and PeatDataHub</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/data-management-peatlands/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-09-13</td>
      <td>2024-09-13</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>109</td>
      <td>140</td>
      <td>2.033333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>Workflow Run RO-Crate Introduction</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/ro-crate-workflow-run-ro-crate/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-05-11</td>
      <td>2023-05-11</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>351</td>
      <td>445</td>
      <td>1.350000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>FAIR and its Origins</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/fair-origin/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-03-26</td>
      <td>2024-03-27</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>155</td>
      <td>171</td>
      <td>0.483333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>Making clinical datasets FAIR</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/fair-clinical/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-10-24</td>
      <td>2024-09-13</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>344</td>
      <td>448</td>
      <td>1.616667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>DataPLANT ARCs</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/dataplant-arcs/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-10-22</td>
      <td>2025-09-17</td>
      <td>4</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>942289</td>
      <td>1960567</td>
      <td>3.100000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>FAIR data management solutions</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/data-management/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-05-30</td>
      <td>2025-01-16</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>382</td>
      <td>471</td>
      <td>2.550000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>RO-Crate in Python</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/ro-crate-in-python/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-05-11</td>
      <td>2025-04-08</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>244</td>
      <td>271</td>
      <td>1.333333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>Exporting Workflow Run RO-Crates from Galaxy</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/ro-crate-in-galaxy/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-05-11</td>
      <td>2024-08-26</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>281</td>
      <td>341</td>
      <td>2.300000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>Metadata</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/fair-metadata/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-03-26</td>
      <td>2024-03-27</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>190</td>
      <td>226</td>
      <td>1.350000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>FAIR Galaxy Training Material</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/fair-gtn/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-05-30</td>
      <td>2024-03-27</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>157</td>
      <td>191</td>
      <td>2.233333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>Data Registration</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/fair-data-registration/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-03-26</td>
      <td>2024-09-27</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>88</td>
      <td>92</td>
      <td>1.783333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>Best practices for workflows in GitHub repositories</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/ro-crate-galaxy-best-practices/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-05-11</td>
      <td>2025-04-08</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>375</td>
      <td>431</td>
      <td>1.150000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>FAIR Bioimage Metadata</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/bioimage-metadata/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-09-13</td>
      <td>2025-11-27</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>296</td>
      <td>354</td>
      <td>2.383333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>Access</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/fair-access/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-03-26</td>
      <td>2024-03-27</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>65</td>
      <td>75</td>
      <td>2.150000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>Submitting workflows to LifeMonitor</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/ro-crate-submitting-life-monitor/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-05-11</td>
      <td>2024-09-26</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>51</td>
      <td>60</td>
      <td>3.750000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>REMBI - Recommended Metadata for Biological Images  metadata guidelines for bioimaging data</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/bioimage-REMBI/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-09-13</td>
      <td>2025-11-27</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>289</td>
      <td>330</td>
      <td>2.183333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>Persistent Identifiers</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/fair-persistent-identifiers/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-03-26</td>
      <td>2024-03-27</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>259</td>
      <td>324</td>
      <td>1.416667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>FAIRification of an RNAseq dataset</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/fair-rna/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-03-27</td>
      <td>2024-03-27</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>267</td>
      <td>325</td>
      <td>1.233333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Workflow Reports</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/workflow-reports/tutorial.html</td>
      <td></td>
      <td>Visualisation</td>
      <td>2022-06-02</td>
      <td>2024-06-14</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>tp_sorted_uniq, upload1, ggplot2_point, Remove beginning1, Cut1, csv_to_tabular, datamash_ops, Grouping1</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>Galaxy@AuBi, MISSISSIPPI, UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>885</td>
      <td>1223</td>
      <td>2.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Creating, Editing and Importing Galaxy Workflows</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/workflow-editor/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-07-17</td>
      <td>2023-11-09</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Paste1, tp_tac, Show beginning1</td>
      <td>Galaxy@AuBi, GalaxyTrakr, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>MISSISSIPPI</td>
      <td>0</td>
      <td></td>
      <td>6040</td>
      <td>7516</td>
      <td>2.950000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>InterMine integration with Galaxy</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/intermine/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-12-09</td>
      <td>2024-02-29</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>intermine, galaxy_intermine_exchange</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>188</td>
      <td>245</td>
      <td>2.800000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Using Workflow Parameters</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/workflow-parameters/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-06-27</td>
      <td>2024-06-14</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>tp_head_tool</td>
      <td>Galaxy@AuBi, GalaxyTrakr, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>MISSISSIPPI</td>
      <td>0</td>
      <td></td>
      <td>1995</td>
      <td>2542</td>
      <td>2.266667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Getting data into Galaxy</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/get-data/slides.html</td>
      <td></td>
      <td></td>
      <td>2017-05-24</td>
      <td>2022-10-18</td>
      <td>4</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>3115529</td>
      <td>7964560</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Galaxy workflows in Dockstore</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/galaxy-on-dockstore/slides.html</td>
      <td></td>
      <td></td>
      <td>2022-01-14</td>
      <td>2022-10-27</td>
      <td>4</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>2567726</td>
      <td>6128580</td>
      <td>2.233333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Rule Based Uploader</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/upload-rules/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-07-04</td>
      <td>2024-06-14</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>4849</td>
      <td>6377</td>
      <td>2.966667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Group tags for complex experimental designs</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/group-tags/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-03-06</td>
      <td>2023-11-09</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>2759</td>
      <td>3724</td>
      <td>3.283333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>RStudio in Galaxy</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/rstudio/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-10-08</td>
      <td>2025-07-04</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>interactive_tool_rstudio</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>5180</td>
      <td>8019</td>
      <td>2.216667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Importing (uploading) data from Onedata</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/onedata-remote-import/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-03-24</td>
      <td>2025-04-08</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>212</td>
      <td>241</td>
      <td>1.283333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Extracting Workflows from Histories</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/history-to-workflow/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2017-10-18</td>
      <td>2024-12-02</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>3057</td>
      <td>3952</td>
      <td>3.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Use Jupyter notebooks in Galaxy</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/galaxy-intro-jupyter/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-07-02</td>
      <td>2025-05-11</td>
      <td>10</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>interactive_tool_jupyter_notebook</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2478</td>
      <td>3275</td>
      <td>2.366667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Downloading and Deleting Data in Galaxy</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/download-delete-data/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-10-17</td>
      <td>2023-11-09</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>7471</td>
      <td>9888</td>
      <td>1.616667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Rule Based Uploader: Advanced</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/upload-rules-advanced/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-05-25</td>
      <td>2024-06-14</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>801</td>
      <td>1062</td>
      <td>4.583333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Using dataset collections</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/collections/tutorial.html</td>
      <td></td>
      <td>Genome indexing, Read mapping, Sequence alignment, Generation</td>
      <td>2016-09-30</td>
      <td>2024-12-05</td>
      <td>20</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>lofreq_call, collapse_dataset, bwa_mem, snpSift_extractFields</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>MISSISSIPPI, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>16149</td>
      <td>22043</td>
      <td>2.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Searching Your History</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/search/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-07-15</td>
      <td>2023-11-16</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>411</td>
      <td>617</td>
      <td>2.200000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Annotate, prepare tests and publish Galaxy workflows in workflow registries</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/workflow-fairification/tutorial.html</td>
      <td></td>
      <td>Visualisation, Read mapping, Sequencing quality control, Validation</td>
      <td>2025-05-26</td>
      <td>2025-05-26</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>falco, multiqc</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>GalaxyTrakr</td>
      <td>0</td>
      <td></td>
      <td>239</td>
      <td>278</td>
      <td>2.050000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>JupyterLab in Galaxy</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/jupyterlab/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-03-05</td>
      <td>2025-04-25</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>interactive_tool_jupyter_notebook</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2148</td>
      <td>3428</td>
      <td>2.783333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Onedata user-owned storage</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/onedata-byos/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-03-24</td>
      <td>2025-04-08</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>91</td>
      <td>106</td>
      <td>3.083333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Getting started with Onedata distributed storage</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/onedata-getting-started/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-03-24</td>
      <td>2025-04-08</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>150</td>
      <td>191</td>
      <td>2.916667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Exporting to Onedata remote</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/onedata-remote-export/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-03-24</td>
      <td>2025-04-08</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>42</td>
      <td>44</td>
      <td>0.083333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Name tags for following complex histories</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/name-tags/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-04-08</td>
      <td>2023-11-09</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1692</td>
      <td>2262</td>
      <td>2.783333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Understanding Galaxy history system</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/history/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2017-02-20</td>
      <td>2025-04-28</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>5558</td>
      <td>7158</td>
      <td>2.850000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Creating high resolution images of Galaxy Workflows</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/workflow-posters/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-02-05</td>
      <td>2025-04-08</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>graphicsmagick_image_convert</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>275</td>
      <td>333</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Genome annotation with Prokka</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/annotation-with-prokka/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis</td>
      <td>Genome annotation, Genome visualisation, Gene prediction, Coding region prediction</td>
      <td>2018-03-06</td>
      <td>2025-03-19</td>
      <td>26</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>jbrowse, prokka</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>Galaxy@AuBi, GalaxyTrakr, MISSISSIPPI, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>3134434</td>
      <td>7996470</td>
      <td>4.250000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>From small to large-scale genome comparison</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/hpc-for-lsgc/tutorial.html</td>
      <td></td>
      <td>Multiple sequence alignment</td>
      <td>2021-02-08</td>
      <td>2024-07-31</td>
      <td>8</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>gecko, tp_awk_tool, clustalw, chromeister</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>3070956</td>
      <td>7804228</td>
      <td>4.533333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Identification of AMR genes in an assembled bacterial genome</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/amr-gene-detection/tutorial.html</td>
      <td>Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology, Sequence analysis, Infectious disease, Antimicrobial resistance</td>
      <td>Antimicrobial resistance prediction, Genome annotation, Genome visualisation, Read mapping</td>
      <td>2024-01-23</td>
      <td>2025-06-10</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>upload1, tbl2gff3, Grep1, jbrowse, staramr_search, bakta, bowtie2</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr</td>
      <td>Galaxy@AuBi, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>3566</td>
      <td>4921</td>
      <td>3.033333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Comparison of two annotation tools - Helixer and Braker3</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/comparison-braker-helixer-annotation/tutorial.html</td>
      <td>Sequence assembly, Genomics</td>
      <td>Differential protein expression profiling, Sequence assembly validation, Genome annotation, Genome visualisation, Sequence alignment, Gene prediction, Sequence annotation</td>
      <td>2025-06-20</td>
      <td>2025-06-25</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>busco, jbrowse, rna_star, helixer, braker3, omark, gffread</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>394</td>
      <td>469</td>
      <td>1.600000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Masking repeats with RepeatMasker</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/repeatmasker/tutorial.html</td>
      <td>Sequence composition, complexity and repeats, Genomics</td>
      <td>RNA-Seq analysis, Genome annotation, Mapping, Editing, Repeat sequence detection</td>
      <td>2021-11-29</td>
      <td>2025-06-03</td>
      <td>22</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>repeatmodeler, red, bedtools_maskfastabed, repeatmasker_wrapper</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>10219</td>
      <td>13000</td>
      <td>1.366667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Secondary metabolite discovery</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/secondary-metabolite-discovery/tutorial.html</td>
      <td></td>
      <td>Sequence clustering, Data handling, Gene prediction, Service invocation, Differential gene expression analysis</td>
      <td>2024-02-21</td>
      <td>2024-09-27</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>ctb_silicos_qed, bp_genbank2gff3, ctb_simsearch, openbabel_remDuplicates, collapse_dataset, Remove beginning1, ctb_chemfp_mol2fps, ncbi_acc_download, interactive_tool_jupyter_notebook, edu.tamu.cpt.gff3.customGbkToGff, antismash, tp_awk_tool, ctb_np-likeness-calculator</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>698</td>
      <td>881</td>
      <td>1.300000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Refining Genome Annotations with Apollo (prokaryotes)</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/apollo/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis</td>
      <td>Genome visualisation</td>
      <td>2021-06-04</td>
      <td>2025-05-15</td>
      <td>22</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>list_organism, jbrowse, iframe, create_account, create_or_update</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2890530</td>
      <td>7188062</td>
      <td>5.100000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Genome annotation with Braker3</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/braker3/tutorial.html</td>
      <td>Sequence assembly, Genomics, Biodiversity, Zoology, Plant biology</td>
      <td>Differential protein expression profiling, Sequence assembly validation, Genome annotation, Genome visualisation, Sequence alignment, Gene prediction, Sequence annotation</td>
      <td>2025-03-17</td>
      <td>2025-06-20</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>busco, jbrowse, fasta-stats, rna_star, braker3, omark, gffread</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1436</td>
      <td>1766</td>
      <td>1.816667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Genome annotation with Helixer</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/helixer/tutorial.html</td>
      <td>Biodiversity, Genomics, Zoology, Plant biology</td>
      <td>Differential protein expression profiling, Sequence assembly validation, Genome annotation, Genome visualisation, Gene prediction, Sequence annotation</td>
      <td>2024-08-21</td>
      <td>2025-06-04</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>busco, jbrowse, helixer, omark, jcvi_gff_stats, gffread</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1261</td>
      <td>1498</td>
      <td>1.650000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Bacterial Genome Annotation</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/bacterial-genome-annotation/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis, Whole genome sequencing, Functional genomics, Mobile genetic elements</td>
      <td>Sequence motif recognition, Genome annotation, Protein feature detection, Multilocus sequence typing, Genome visualisation, Scaffolding, Structural variation detection, Genome assembly, Nucleic acid feature detection</td>
      <td>2024-02-01</td>
      <td>2024-10-15</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>upload1, tbl2gff3, tp_replace_in_column, jbrowse, tp_tail_tool, isescan, plasmidfinder, Grouping1, bakta, integron_finder</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td>Galaxy@AuBi, UseGalaxy.cz, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>6694</td>
      <td>9127</td>
      <td>2.583333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Introduction to Genome Annotation</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/introduction/slides.html</td>
      <td></td>
      <td></td>
      <td>2019-05-13</td>
      <td>2025-04-23</td>
      <td>5</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>3115529</td>
      <td>7964562</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Genome Annotation</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/genome-annotation/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2015-06-29</td>
      <td>2025-06-10</td>
      <td>19</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>35158</td>
      <td>63998</td>
      <td>1.450000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Creating an Official Gene Set</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/official-gene-set/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-11-02</td>
      <td>2024-02-29</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>44</td>
      <td>50</td>
      <td>1.450000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Dataset construction for bacterial comparative genomics</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/bacterial-comparative-genomics-dataset-construction/tutorial.html</td>
      <td>Genomics, Microbiology</td>
      <td>Sequence database search, Data retrieval, Data handling</td>
      <td>2025-03-14</td>
      <td>2025-03-14</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>upload1, bg_uniq, Remove beginning1, cat1, Cut1, datasets_download_genome, sort1</td>
      <td>UseGalaxy.fr</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>84</td>
      <td>102</td>
      <td>3.033333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Bacterial genome quality control</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/bacterial-genome-quality-control/tutorial.html</td>
      <td>Genomics, Microbiology</td>
      <td>Sequence assembly validation, Sequence composition calculation, Genome comparison, Sequencing quality control, Statistical calculation, Validation</td>
      <td>2025-03-14</td>
      <td>2025-03-14</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>upload1, drep_dereplicate, checkm2</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>GalaxyTrakr, UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>97</td>
      <td>107</td>
      <td>2.083333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Bacterial pangenomics</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/bacterial-pangenomics/tutorial.html</td>
      <td>Genomics, Microbiology</td>
      <td>Rarefaction, Genome annotation, Sequence assembly, Essential dynamics</td>
      <td>2025-03-14</td>
      <td>2025-03-14</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>upload1, ppanggolin_all, ppanggolin_msa</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>95</td>
      <td>109</td>
      <td>0.883333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Genome annotation with Funannotate</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/funannotate/tutorial.html</td>
      <td>Gene and protein families</td>
      <td>Sequence motif recognition, Sequence assembly validation, Genome annotation, Query and retrieval, Information extraction, Formatting, Protein feature detection, Genome visualisation, Homology-based gene prediction, Sequence alignment, Sequence annotation, Fold recognition</td>
      <td>2021-11-29</td>
      <td>2025-06-10</td>
      <td>26</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>busco, interproscan, jbrowse, picard_MergeSamFiles, eggnog_mapper, funannotate_compare, funannotate_predict, aegean_parseval, funannotate_annotate, rna_star</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>10825</td>
      <td>15308</td>
      <td>1.950000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Comparative gene analysis in unannotated genomes</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/gene-centric/tutorial.html</td>
      <td>Genomics, Gene and protein families, Sequence analysis, Phylogeny, Comparative genomics</td>
      <td>Database search, Sequence alignment analysis, Coding region prediction, Multiple sequence alignment, Transcriptome assembly, De-novo assembly, Phylogenetic tree generation</td>
      <td>2022-09-08</td>
      <td>2025-06-03</td>
      <td>10</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Add_a_column1, rapidnj, bg_diamond_view, orfipy, collapse_dataset, bg_diamond, tab2fasta, cat1, rbc_mafft, Cut1, Filter1, bg_diamond_makedb, tp_split_on_column, regexColumn1, join1, gops_intersect_1</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>1877</td>
      <td>2628</td>
      <td>3.016667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Genome annotation with Maker (short)</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/annotation-with-maker-short/tutorial.html</td>
      <td></td>
      <td>Sequence assembly validation, Sequence annotation, Genome annotation, Genome visualisation</td>
      <td>2021-01-12</td>
      <td>2024-02-29</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>busco, maker_map_ids, jbrowse, fasta-stats, maker, jcvi_gff_stats, gffread</td>
      <td></td>
      <td>UseGalaxy.be</td>
      <td>0</td>
      <td></td>
      <td>4020</td>
      <td>5638</td>
      <td>2.900000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Essential genes detection with Transposon insertion sequencing</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/tnseq/tutorial.html</td>
      <td>Genomics, Microbiology, Sequence analysis, Mobile genetic elements</td>
      <td>Sequence trimming, Transposon prediction, Read pre-processing, Primer removal</td>
      <td>2019-07-02</td>
      <td>2024-03-18</td>
      <td>12</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>Add_a_column1, bowtie_wrapper, bg_find_subsequences, tp_sort_header_tool, deeptools_bam_coverage, transit_gumbel, Cut1, Filter1, tp_easyjoin_tool, gff_to_prot, __EXTRACT_DATASET__, cutadapt</td>
      <td>UseGalaxy.no, UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td>0</td>
      <td></td>
      <td>3123896</td>
      <td>7979709</td>
      <td>4.066667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Genome annotation with Maker</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/annotation-with-maker/tutorial.html</td>
      <td></td>
      <td>Ab-initio gene prediction, Sequence assembly validation, Genome annotation, Genome visualisation, Homology-based gene prediction, Gene prediction, Sequence annotation, Operation</td>
      <td>2018-09-24</td>
      <td>2024-02-29</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>busco, maker_map_ids, jbrowse, augustus_training, fasta-stats, snap_training, maker, jcvi_gff_stats, gffread</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>13873</td>
      <td>24491</td>
      <td>1.850000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Functional annotation of protein sequences</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/functional/tutorial.html</td>
      <td>Function analysis, Protein sites, features and motifs</td>
      <td>Sequence motif recognition, Genome annotation, Query and retrieval, Homology-based gene prediction, Sequence alignment analysis, Information extraction, Protein feature detection, Fold recognition</td>
      <td>2022-07-20</td>
      <td>2025-06-03</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>eggnog_mapper, ncbi_blastp_wrapper, interproscan, bg_diamond</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>4355</td>
      <td>5823</td>
      <td>1.666667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Refining Genome Annotations with Apollo (eukaryotes)</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/apollo-euk/tutorial.html</td>
      <td>Genomics</td>
      <td>Genome visualisation</td>
      <td>2022-08-22</td>
      <td>2025-06-03</td>
      <td>16</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>iframe, create_or_update, jbrowse</td>
      <td>UseGalaxy.no</td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td>0</td>
      <td></td>
      <td>1469</td>
      <td>2235</td>
      <td>2.550000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Long non-coding RNAs (lncRNAs) annotation with FEELnc</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/lncrna/tutorial.html</td>
      <td></td>
      <td>RNA-Seq analysis, Sequence annotation, Classification, Transcriptome assembly, Annotation</td>
      <td>2022-09-23</td>
      <td>2024-09-18</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>feelnc, Extract_features1, stringtie, tp_cat, gffread</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>1919</td>
      <td>2562</td>
      <td>2.133333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>CRISPR screen analysis</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/crispr-screen/tutorial.html</td>
      <td></td>
      <td>Sequence trimming, Sequencing quality control, Validation, Primer removal, Read pre-processing, Statistical calculation, Sequence composition calculation, Genetic variation analysis</td>
      <td>2021-10-15</td>
      <td>2024-08-08</td>
      <td>12</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>cutadapt, volcanoplot, mageck_count, fastqc, mageck_test, multiqc, tp_awk_tool, mageck_mle</td>
      <td>UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2723645</td>
      <td>6600652</td>
      <td>4.150000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Microbial Variant Calling</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/microbial-variants/tutorial.html</td>
      <td>Genomics, Sequence assembly, DNA polymorphism, Microbiology, Sequence analysis, Genetic variation</td>
      <td>Phylogenetic tree visualisation, Genome visualisation, Phylogenetic tree generation, Variant calling</td>
      <td>2018-02-26</td>
      <td>2025-03-19</td>
      <td>24</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>jbrowse, snippy</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>GalaxyTrakr, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>15243</td>
      <td>24323</td>
      <td>2.050000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Visualisation</td>
      <td>Genomic Data Visualisation with JBrowse</td>
      <td>https://training.galaxyproject.org//topics/visualisation/tutorials/jbrowse/tutorial.html</td>
      <td></td>
      <td>Genome visualisation</td>
      <td>2020-01-10</td>
      <td>2024-10-15</td>
      <td>17</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>jbrowse</td>
      <td>Galaxy@AuBi, UseGalaxy.be, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>GalaxyTrakr, MISSISSIPPI, UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>3120722</td>
      <td>7971598</td>
      <td>4.616667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Imaging</td>
      <td>Using BioImage.IO models for image analysis in Galaxy</td>
      <td>https://training.galaxyproject.org//topics/imaging/tutorials/process-image-bioimageio/tutorial.html</td>
      <td></td>
      <td>Visualisation, Data handling, Image annotation, Image analysis</td>
      <td>2025-04-09</td>
      <td>2025-11-26</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>bioimage_inference, ip_binary_to_labelimage, ip_threshold, image_math, ip_split_image, ip_overlay_images, __EXTRACT_DATASET__</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>192</td>
      <td>229</td>
      <td>3.016667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Imaging</td>
      <td>Overview of the Galaxy OMERO-suite - Upload images and metadata in OMERO using Galaxy</td>
      <td>https://training.galaxyproject.org//topics/imaging/tutorials/omero-suite/tutorial.html</td>
      <td></td>
      <td>Image analysis</td>
      <td>2024-12-04</td>
      <td>2025-11-26</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>omero_get_value, omero_get_id, omero_metadata_import, omero_import, omero_roi_import, omero_filter</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>200</td>
      <td>252</td>
      <td>2.716667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Imaging</td>
      <td>Parameter tuning and optimization - Evaluating nuclei segmentation with Galaxy</td>
      <td>https://training.galaxyproject.org//topics/imaging/tutorials/parameter-tuning/tutorial.html</td>
      <td></td>
      <td>Visualisation, Data handling, Image annotation, Image analysis</td>
      <td>2025-04-28</td>
      <td>2025-11-26</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>collapse_dataset, ip_segmetrics, ip_threshold, param_value_from_file, tp_split_on_column, ip_histogram_equalization, ip_filter_standard</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>97</td>
      <td>110</td>
      <td>1.716667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Imaging</td>
      <td>Quantification of single-molecule RNA fluorescence in situ hybridization (smFISH) in yeast cell lines</td>
      <td>https://training.galaxyproject.org//topics/imaging/tutorials/2D-spot-detection/tutorial.html</td>
      <td></td>
      <td>Visualisation, Data handling, Image annotation, Image analysis</td>
      <td>2025-03-20</td>
      <td>2025-11-26</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>idr_download_by_ids, ip_binary_to_labelimage, ip_spot_detection_2d, ip_histogram_equalization, ip_overlay_images, __EXTRACT_DATASET__, unzip, ip_points_to_label</td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>193</td>
      <td>238</td>
      <td>1.250000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Imaging</td>
      <td>Object tracking using CellProfiler</td>
      <td>https://training.galaxyproject.org//topics/imaging/tutorials/object-tracking-using-cell-profiler/tutorial.html</td>
      <td></td>
      <td>Quantification, Parsing, Image analysis</td>
      <td>2021-03-17</td>
      <td>2025-11-26</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>cp_measure_object_size_shape, cp_identify_primary_objects, cp_export_to_spreadsheet, cp_overlay_outlines, cp_measure_object_intensity, cp_cellprofiler, cp_track_objects, cp_color_to_gray, cp_save_images, cp_common, unzip, cp_tile</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>2207</td>
      <td>2747</td>
      <td>1.050000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Imaging</td>
      <td>Quantification of electrophoresis gel bands using QuPath and Galaxy imaging tools</td>
      <td>https://training.galaxyproject.org//topics/imaging/tutorials/electrophoresis-gel-bands-image-analysis/tutorial.html</td>
      <td></td>
      <td>Visualisation, Data handling, Image annotation, Image analysis</td>
      <td>2025-08-11</td>
      <td>2025-11-26</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>ip_imageinfo, ip_split_image, ip_2d_feature_extraction, __EXTRACT_DATASET__, interactive_tool_qupath, graphicsmagick_image_convert, ip_points_to_label</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>278</td>
      <td>335</td>
      <td>1.050000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Imaging</td>
      <td>Nucleoli segmentation and feature extraction using CellProfiler</td>
      <td>https://training.galaxyproject.org//topics/imaging/tutorials/tutorial-CP/tutorial.html</td>
      <td></td>
      <td>Quantification, Parsing, Image analysis</td>
      <td>2020-07-01</td>
      <td>2025-11-26</td>
      <td>12</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>cp_gray_to_color, cp_measure_texture, cp_identify_primary_objects, cp_export_to_spreadsheet, cp_measure_image_intensity, cp_common, cp_relate_objects, idr_download_by_ids, cp_measure_object_size_shape, cp_convert_objects_to_image, cp_mask_image, cp_measure_image_area_occupied, cp_image_math, cp_measure_object_intensity, cp_cellprofiler, cp_save_images, cp_measure_granularity, cp_measure_image_quality, cp_display_data_on_image, cp_enhance_or_suppress_features</td>
      <td>UseGalaxy.be, UseGalaxy.eu</td>
      <td>Galaxy@AuBi, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>3119495</td>
      <td>7970511</td>
      <td>3.533333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Imaging</td>
      <td>Voronoi segmentation</td>
      <td>https://training.galaxyproject.org//topics/imaging/tutorials/voronoi-segmentation/tutorial.html</td>
      <td></td>
      <td>Visualisation, Data handling, Image annotation, Image analysis</td>
      <td>2025-05-13</td>
      <td>2025-11-26</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>voronoi_tessellation, ip_binary_to_labelimage, ip_threshold, ip_count_objects, image_math, ip_overlay_images, ip_2d_feature_extraction, ip_filter_standard, repeat_channels, ip_imageinfo, unzip, colorize_labels, ip_convertimage</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>294</td>
      <td>435</td>
      <td>3.450000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Imaging</td>
      <td>Tracking of mitochondria and capturing mitoflashes</td>
      <td>https://training.galaxyproject.org//topics/imaging/tutorials/detection-of-mitoflashes/tutorial.html</td>
      <td></td>
      <td>Image analysis</td>
      <td>2024-11-20</td>
      <td>2025-11-26</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>ip_spot_detection_2d, ip_curve_fitting, ip_points_association_nn</td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>185</td>
      <td>215</td>
      <td>1.033333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Imaging</td>
      <td>Introduction to Image Analysis using Galaxy</td>
      <td>https://training.galaxyproject.org//topics/imaging/tutorials/imaging-introduction/tutorial.html</td>
      <td></td>
      <td>Visualisation, Data handling, Image annotation, Image analysis</td>
      <td>2019-08-13</td>
      <td>2025-11-26</td>
      <td>17</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>ip_binary_to_labelimage, ip_threshold, ip_count_objects, ip_histogram_equalization, ip_overlay_images, ip_filter_standard, ip_imageinfo, unzip</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2690</td>
      <td>3752</td>
      <td>1.716667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Imaging</td>
      <td>Training Custom YOLO Models for Segmentation of Bioimages</td>
      <td>https://training.galaxyproject.org//topics/imaging/tutorials/yolo-segmentation-training/tutorial.html</td>
      <td></td>
      <td>Image annotation, Data handling, Visualisation, Image analysis</td>
      <td>2025-07-25</td>
      <td>2025-11-26</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>yolo_training, interactive_tool_anylabeling, json2yolosegment, ip_histogram_equalization, graphicsmagick_image_convert, unzip</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>243</td>
      <td>302</td>
      <td>1.100000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Imaging</td>
      <td>End-to-End Tissue Microarray Image Analysis with Galaxy-ME</td>
      <td>https://training.galaxyproject.org//topics/imaging/tutorials/multiplex-tissue-imaging-TMA/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-02-14</td>
      <td>2025-11-26</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>scanpy_compute_graph, squidpy_spatial, celesta, gate_finder, scimap_mcmicro_to_anndata, rename_tiff_channels, quantification, basic_illumination, scimap_phenotyping, mesmer, unet_coreograph, ashlar, vitessce_spatial, ip_convertimage, palom</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1456</td>
      <td>1960</td>
      <td>1.800000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Imaging</td>
      <td>Analyse HeLa fluorescence siRNA screen</td>
      <td>https://training.galaxyproject.org//topics/imaging/tutorials/hela-screen-analysis/tutorial.html</td>
      <td></td>
      <td>Visualisation, Data handling, Image annotation, Image analysis</td>
      <td>2019-08-13</td>
      <td>2025-11-26</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>collapse_dataset, ip_binary_to_labelimage, ip_2d_filter_segmentation_by_features, ip_threshold, ip_2d_feature_extraction, ip_filter_standard, unzip</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>799</td>
      <td>1037</td>
      <td>2.033333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Introduction to Galaxy Analyses</td>
      <td>From peaks to genes</td>
      <td>https://training.galaxyproject.org//topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2016-03-29</td>
      <td>2025-11-05</td>
      <td>66</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>Add_a_column1, upload1, CONVERTER_interval_to_bed_0, tp_sort_header_tool, tp_replace_in_column, tp_tail_tool, Cut1, get_flanks1, Grouping1, gops_intersect_1</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>20547</td>
      <td>33763</td>
      <td>3.300000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Introduction to Galaxy Analyses</td>
      <td>NGS data logistics</td>
      <td>https://training.galaxyproject.org//topics/introduction/tutorials/galaxy-intro-ngs-data-managment/tutorial.html</td>
      <td></td>
      <td>Generation, Read mapping, SNP detection, Genome indexing, Sequencing quality control, Formatting, Sequence contamination filtering, Validation, Sequence alignment, Data handling</td>
      <td>2017-02-22</td>
      <td>2025-07-24</td>
      <td>38</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>samtools_view, fasterq_dump, bwa_mem2, snpSift_extractFields, collapse_dataset, picard_MarkDuplicates, snpEff, lofreq_viterbi, Filter1, samtools_stats, snpeff_sars_cov_2, snpEff_build_gb, bwa, fastp, multiqc, bwa_mem, lofreq_call, lofreq_indelqual</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>MISSISSIPPI, UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>20419</td>
      <td>34830</td>
      <td>3.400000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Introduction to Galaxy Analyses</td>
      <td>How to reproduce published Galaxy analyses</td>
      <td>https://training.galaxyproject.org//topics/introduction/tutorials/galaxy-reproduce/tutorial.html</td>
      <td></td>
      <td>Visualisation</td>
      <td>2021-08-25</td>
      <td>2024-10-15</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tp_sorted_uniq, upload1, ggplot2_point, Remove beginning1, Cut1, csv_to_tabular, datamash_ops, Grouping1</td>
      <td>Galaxy@AuBi, UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>MISSISSIPPI, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2209</td>
      <td>2939</td>
      <td>4.033333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Introduction to Galaxy Analyses</td>
      <td>Data Manipulation Olympics</td>
      <td>https://training.galaxyproject.org//topics/introduction/tutorials/data-manipulation-olympics/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-07-20</td>
      <td>2025-05-16</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tp_sorted_uniq, Remove beginning1, tp_sort_header_tool, {{version_cut_columns}}, Show beginning1, regexColumn1, {{version_cat}}, {{version_datamash}}, {{version_paste}}, {{version_wc}}, {{version_cut_advanced}}, {{version_replace_text_column}}, datamash_ops, tp_split_on_column, {{version_compute}}, {{version_select_first}}, upload1, Count1, {{version_remove_beginning}}, {{version_sort}}, {{version_split}}, Filter1, tabular_to_csv, tp_cat, Grouping1, {{version_join}}, {{version_replace_text_line}}, {{version_unique}}, wc_gnu, Add_a_column1, cat1, {{version_filter}}, Cut1, join1, {{version_count}}, {{version_remove_columns_by_header}}</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>6099</td>
      <td>8848</td>
      <td>3.416667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Introduction to Galaxy Analyses</td>
      <td>Introduction to Genomics and Galaxy</td>
      <td>https://training.galaxyproject.org//topics/introduction/tutorials/galaxy-intro-strands/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2017-10-18</td>
      <td>2024-10-15</td>
      <td>41</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>cat1, gene2exon1, Filter1, tp_cat, gops_intersect_1</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>MISSISSIPPI</td>
      <td>0</td>
      <td></td>
      <td>20312</td>
      <td>32018</td>
      <td>3.566667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Introduction to Galaxy Analyses</td>
      <td>Introduction to Galaxy</td>
      <td>https://training.galaxyproject.org//topics/introduction/tutorials/introduction/slides.html</td>
      <td></td>
      <td></td>
      <td>2016-09-30</td>
      <td>2025-09-23</td>
      <td>6</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>3115664</td>
      <td>7964911</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Introduction to Galaxy Analyses</td>
      <td>Options for using Galaxy</td>
      <td>https://training.galaxyproject.org//topics/introduction/tutorials/options-for-using-galaxy/slides.html</td>
      <td></td>
      <td></td>
      <td>2017-05-24</td>
      <td>2022-09-23</td>
      <td>14</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>3115664</td>
      <td>7964911</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Introduction to Galaxy Analyses</td>
      <td>IGV Introduction</td>
      <td>https://training.galaxyproject.org//topics/introduction/tutorials/igv-introduction/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2016-03-30</td>
      <td>2023-02-17</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>8169</td>
      <td>11490</td>
      <td>1.466667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Introduction to Galaxy Analyses</td>
      <td>Best Practices for Citing Galaxy</td>
      <td>https://training.galaxyproject.org//topics/introduction/tutorials/galaxy-cite/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-11-06</td>
      <td>2025-11-06</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>50</td>
      <td>55</td>
      <td>0.700000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Introduction to Galaxy Analyses</td>
      <td>Upload data to Galaxy</td>
      <td>https://training.galaxyproject.org//topics/introduction/tutorials/dummy/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-09-23</td>
      <td>2023-11-09</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>450</td>
      <td>511</td>
      <td>2.633333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Introduction to Galaxy Analyses</td>
      <td>A short introduction to Galaxy</td>
      <td>https://training.galaxyproject.org//topics/introduction/tutorials/galaxy-intro-short/tutorial.html</td>
      <td></td>
      <td>Sequencing quality control, Statistical calculation, Sequence composition calculation</td>
      <td>2018-08-27</td>
      <td>2025-11-17</td>
      <td>43</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>cshl_fastq_quality_filter, fastqc</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>GalaxyTrakr, MISSISSIPPI, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>3170569</td>
      <td>8050417</td>
      <td>7.033333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Introduction to Galaxy Analyses</td>
      <td>Galaxy Basics for everyone</td>
      <td>https://training.galaxyproject.org//topics/introduction/tutorials/galaxy-intro-101-everyone/tutorial.html</td>
      <td></td>
      <td>Visualisation</td>
      <td>2019-11-14</td>
      <td>2025-11-06</td>
      <td>35</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>tp_sorted_uniq, upload1, ggplot2_point, Remove beginning1, Cut1, csv_to_tabular, datamash_ops, Grouping1</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>Galaxy@AuBi, MISSISSIPPI, UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>23723</td>
      <td>37625</td>
      <td>3.933333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Introduction to Galaxy Analyses</td>
      <td>Galaxy Basics for genomics</td>
      <td>https://training.galaxyproject.org//topics/introduction/tutorials/galaxy-intro-101/tutorial.html</td>
      <td></td>
      <td>Mapping</td>
      <td>2016-12-19</td>
      <td>2025-04-22</td>
      <td>68</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>upload1, comp1, bedtools_intersectbed, tp_sort_header_tool, datamash_ops, tp_head_tool</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>Galaxy@AuBi, MISSISSIPPI, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>49141</td>
      <td>79240</td>
      <td>3.433333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Introduction to Galaxy Analyses</td>
      <td>Very Short Introductions: QC</td>
      <td>https://training.galaxyproject.org//topics/introduction/tutorials/vsi_qc/tutorial.html</td>
      <td></td>
      <td>Sequencing quality control, Validation, Sequence contamination filtering</td>
      <td>2024-03-26</td>
      <td>2024-06-25</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>fastp, multiqc</td>
      <td>Galaxy@AuBi, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>GalaxyTrakr, MISSISSIPPI, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>336</td>
      <td>403</td>
      <td>2.750000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Materials Science</td>
      <td>Finding the muon stopping site with pymuon-suite in Galaxy</td>
      <td>https://training.galaxyproject.org//topics/materials-science/tutorials/muon-stopping-sites-muairss-uep/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-01-22</td>
      <td>2024-06-14</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>pm_uep_opt, pm_muairss_read</td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>277</td>
      <td>358</td>
      <td>1.850000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Metabolomics</td>
      <td>Mass spectrometry: LC-MS analysis</td>
      <td>https://training.galaxyproject.org//topics/metabolomics/tutorials/lcms/tutorial.html</td>
      <td></td>
      <td>Chromatographic alignment, Label-free quantification, Validation, Chromatogram visualisation, Mass spectrum visualisation, Peak detection, Data filtering</td>
      <td>2019-07-02</td>
      <td>2025-09-21</td>
      <td>13</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>xcms_merge, msnbase_readmsdata, xcms_plot_chromatogram, abims_xcms_fillPeaks, generic_filter, quality_metrics, abims_xcms_group, abims_xcms_xcmsSet, Batch_correction, abims_xcms_retcor, xcms_export_samplemetadata, wsdl_hmdb, Univariate, abims_CAMERA_annotateDiffreport</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>12848</td>
      <td>20945</td>
      <td>1.600000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Metabolomics</td>
      <td>Mass spectrometry: GC-MS data processing (with XCMS, RAMClustR, RIAssigner, and matchms)</td>
      <td>https://training.galaxyproject.org//topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.html</td>
      <td></td>
      <td>Clustering, Chromatographic alignment, Label-free quantification, Validation, Chromatogram visualisation, Imputation, Filtering, Data filtering, Format validation, Correlation, Standardisation and normalisation, Formatting, Mass spectrum visualisation, Peak detection, Spectral library search</td>
      <td>2023-05-08</td>
      <td>2023-11-09</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>xcms_merge, msnbase_readmsdata, matchms_similarity, ramclustr, abims_xcms_fillPeaks, msconvert, matchms_filtering, matchms_formatter, abims_xcms_xcmsSet, riassigner, ramclustr_define_experiment, abims_xcms_retcor, waveica, abims_xcms_group</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>2763</td>
      <td>3538</td>
      <td>1.500000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Metabolomics</td>
      <td>Mass spectrometry imaging: Examining the spatial distribution of analytes</td>
      <td>https://training.galaxyproject.org//topics/metabolomics/tutorials/msi-analyte-distribution/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-02-21</td>
      <td>2024-10-15</td>
      <td>18</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>cardinal_quality_report, cardinal_mz_images, cardinal_filtering, Filter1, cardinal_spectra_plots, cardinal_data_exporter</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>3041</td>
      <td>4315</td>
      <td>1.733333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Metabolomics</td>
      <td>Predicting EI+ mass spectra with QCxMS</td>
      <td>https://training.galaxyproject.org//topics/metabolomics/tutorials/qcxms-predictions/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-10-01</td>
      <td>2024-10-01</td>
      <td>1</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>qcxms_getres, qcxms_production_run, split_file_to_collection, __FILTER_FAILED_DATASETS__, param_value_from_file, tp_cat, tp_cut_tool, qcxms_neutral_run, xtb_molecular_optimization, openbabel_compound_convert, ctb_im_conformers</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>980118</td>
      <td>2076030</td>
      <td>2.833333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Metabolomics</td>
      <td>Molecular formula assignment and mass recalibration with MFAssignR package</td>
      <td>https://training.galaxyproject.org//topics/metabolomics/tutorials/mfassignr/tutorial.html</td>
      <td></td>
      <td>Visualisation, Annotation, Mass spectra calibration, Deisotoping</td>
      <td>2025-05-19</td>
      <td>2025-05-19</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>mfassignr_recallist, mfassignr_mfassign, mfassignr_mfassignCHO, mfassignr_kmdnoise, mfassignr_findRecalSeries, mfassignr_isofiltr, mfassignr_recal, mfassignr_histnoise, mfassignr_snplot</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>137</td>
      <td>159</td>
      <td>1.700000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Metabolomics</td>
      <td>Mass spectrometry imaging: Finding differential analytes</td>
      <td>https://training.galaxyproject.org//topics/metabolomics/tutorials/msi-finding-nglycans/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-04-13</td>
      <td>2023-11-09</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>maldi_quant_peak_detection, cardinal_classification, join_files_on_column_fuzzy, cardinal_combine, Grep1, cardinal_quality_report, cardinal_mz_images, cardinal_preprocessing, Filter1, maldi_quant_preprocessing, cardinal_segmentations, Summary_Statistics1</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>1993</td>
      <td>2849</td>
      <td>1.816667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Metabolomics</td>
      <td>Introduction to Metabolomics</td>
      <td>https://training.galaxyproject.org//topics/metabolomics/tutorials/introduction/slides.html</td>
      <td></td>
      <td></td>
      <td>2018-11-20</td>
      <td>2023-01-12</td>
      <td>1</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>3115664</td>
      <td>7964915</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Metabolomics</td>
      <td>Mass spectrometry: LC-MS preprocessing with XCMS</td>
      <td>https://training.galaxyproject.org//topics/metabolomics/tutorials/lcms-preprocessing/tutorial.html</td>
      <td></td>
      <td>Chromatographic alignment, Label-free quantification, Validation, Chromatogram visualisation, Mass spectrum visualisation, Peak detection, Data filtering</td>
      <td>2020-02-04</td>
      <td>2024-10-15</td>
      <td>14</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>xcms_merge, msnbase_readmsdata, abims_CAMERA_annotateDiffreport, xcms_plot_chromatogram, abims_xcms_fillPeaks, abims_xcms_summary, abims_xcms_xcmsSet, abims_xcms_retcor, xcms_export_samplemetadata, intens_check, idchoice, abims_xcms_group</td>
      <td>UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>3121827</td>
      <td>7974575</td>
      <td>4.583333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Metabolomics</td>
      <td>Mass spectrometry: GC-MS analysis with the metaMS package</td>
      <td>https://training.galaxyproject.org//topics/metabolomics/tutorials/gcms/tutorial.html</td>
      <td></td>
      <td>Chromatographic alignment, Label-free quantification, Validation, Chromatogram visualisation, Mass spectrum visualisation, Peak detection, Data filtering</td>
      <td>2021-09-22</td>
      <td>2025-10-01</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>xcms_merge, msnbase_readmsdata, xcms_plot_chromatogram, abims_xcms_fillPeaks, metams_runGC, abims_xcms_xcmsSet, xcms_export_samplemetadata, abims_xcms_group</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>1643</td>
      <td>2588</td>
      <td>1.650000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Metabolomics</td>
      <td>Mass spectrometry: LC-MS data processing</td>
      <td>https://training.galaxyproject.org//topics/metabolomics/tutorials/lcms-dataprocessing/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-12-14</td>
      <td>2024-10-15</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>quality_metrics, Multivariate, Batch_correction, generic_filter, normalization, intens_check</td>
      <td>UseGalaxy.fr</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2917</td>
      <td>3917</td>
      <td>1.866667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Introduction to Microbiome Analysis</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/introduction/slides.html</td>
      <td></td>
      <td></td>
      <td>2017-06-22</td>
      <td>2024-08-08</td>
      <td>5</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>3115664</td>
      <td>7964916</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Analyses of metagenomics data - The global picture</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/general-tutorial/tutorial.html</td>
      <td>Metagenomics, Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td>Visualisation, Sequence clustering, Phylogenetic analysis, Sequencing quality control, DNA barcoding, Sequence read processing, Taxonomic classification</td>
      <td>2017-06-22</td>
      <td>2024-08-09</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>mothur_make_shared, mothur_classify_otu, mothur_filter_seqs, mothur_merge_files, mothur_pre_cluster, humann2_regroup_table, mothur_unique_seqs, mothur_classify_seqs, krona-text, mothur_align_seqs, mothur_summary_seqs, mothur_screen_seqs, humann2, mothur_cluster_split, metaphlan2, taxonomy_krona_chart, mothur_make_biom, mothur_count_seqs, humann2_renorm_table, metaphlan2krona, mothur_make_group</td>
      <td></td>
      <td>UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>9004</td>
      <td>12309</td>
      <td>2.666667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Antibiotic resistance detection</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/plasmid-metagenomics-nanopore/tutorial.html</td>
      <td>Metagenomic sequencing, Public health and epidemiology, Metagenomics, Microbiology, Infectious disease, Sequence analysis, Antimicrobial resistance</td>
      <td>Antimicrobial resistance prediction, Mapping assembly, De-novo assembly, Box-Whisker plot plotting, Sequence assembly visualisation, Scatter plot plotting, Aggregation, Genome assembly, Pairwise sequence alignment, Sequence analysis</td>
      <td>2019-06-25</td>
      <td>2024-06-14</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>miniasm, PlasFlow, bandage_image, gfa_to_fa, unicycler, minimap2, staramr_search, racon, nanoplot</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>3283</td>
      <td>4587</td>
      <td>2.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>MGnify v5.0 Amplicon Pipeline</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/mgnify-amplicon/tutorial.html</td>
      <td>Metagenomics, Taxonomy, Sequence analysis</td>
      <td>Analysis, Sequence contamination filtering, Validation, Alignment, Nucleic acid design, Sequence feature detection, Mapping, Statistical calculation, Read pre-processing, Sequence composition calculation, Visualisation, Sequence trimming, Sequencing quality control, Formatting, Comparison, DNA barcoding, k-mer counting, Nucleic acid feature detection</td>
      <td>2025-07-30</td>
      <td>2025-11-18</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>mgnify_seqprep, param_value_from_file, lotus2, bedtools_maskfastabed, fastp, fastq_dl, ampvis2_load, prinseq, cmsearch_deoverlap, cshl_fasta_formatter, filter_tabular, mapseq, CONVERTER_uncompressed_to_gz, bedtools_getfastabed, gops_concat_1, ampvis2_heatmap, __FILTER_FROM_FILE__, multiqc, fastq_to_fasta_python, biom_convert, collapse_dataset, query_tabular, seqprep, trimmomatic, __MERGE_COLLECTION__, infernal_cmsearch, Grouping1, taxonomy_krona_chart, tp_awk_tool, fastq_filter, wc_gnu, Add_a_column1, CONVERTER_gz_to_uncompressed, fastqc, collection_element_identifiers, tp_find_and_replace, __UNZIP_COLLECTION__, __FILTER_EMPTY_DATASETS__, collection_column_join</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>194</td>
      <td>258</td>
      <td>4.433333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Pathogen detection from (direct Nanopore) sequencing data using Galaxy - Foodborne Edition</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.html</td>
      <td>Metagenomics, Public health and epidemiology, Taxonomy, Sequence assembly, Pathology, Sequence analysis</td>
      <td>Sequence assembly, Sequence contamination filtering, Phylogenetic tree generation, Validation, Cross-assembly, Sequence assembly visualisation, Multilocus sequence typing, Scatter plot plotting, Data handling, Genome assembly, Mapping, Taxonomic classification, Statistical calculation, Variant calling, Sequence composition calculation, Antimicrobial resistance prediction, Visualisation, Mapping assembly, Sequencing quality control, De-novo assembly, Multiple sequence alignment, Sequence alignment analysis, Phylogenetic tree reconstruction, Phylogenetic tree generation (from molecular sequences), Box-Whisker plot plotting, Phylogenetic tree analysis, Aggregation, Pairwise sequence alignment, Phylogenetic tree generation (maximum likelihood and Bayesian methods), Base-calling</td>
      <td>2023-01-26</td>
      <td>2025-11-18</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>clair3, tp_sorted_uniq, Grep1, Remove beginning1, snpSift_extractFields, barchart_gnuplot, ggplot2_heatmap, param_value_from_file, regexColumn1, bcftools_norm, kraken2, clustalw, krakentools_kreport2krona, samtools_fastx, compose_text_param, fastp, table_compute, tp_replace_in_column, fasta2tab, tab2fasta, samtools_depth, __FILTER_FAILED_DATASETS__, bamtools_split_mapped, samtools_coverage, tp_split_on_column, medaka_consensus_pipeline, Paste1, bedtools_getfastabed, flye, newick_display, fasta_merge_files_and_filter_unique_sequences, multiqc, krakentools_extract_kraken_reads, snpSift_filter, collapse_dataset, fasttree, bandage_image, Count1, porechop, split_file_to_collection, mlst, Grouping1, taxonomy_krona_chart, bcftools_consensus, nanoplot, Add_a_column1, tp_multijoin_tool, regex1, CONVERTER_gz_to_uncompressed, Cut1, minimap2, tp_head_tool, tp_cut_tool, __BUILD_LIST__, add_line_to_file, fastqc, collection_element_identifiers, tp_find_and_replace, __FILTER_EMPTY_DATASETS__, abricate, collection_column_join</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>4876</td>
      <td>7284</td>
      <td>3.366667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Clinical Metaproteomics 2: Discovery</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/clinical-mp-2-discovery/tutorial.html</td>
      <td></td>
      <td>Clustering, Visualisation, Heat map generation, Standardisation and normalisation, Formatting, Imputation, Principal component plotting, Protein quantification, Filtering, Statistical calculation</td>
      <td>2025-12-16</td>
      <td>2025-12-16</td>
      <td>0</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>maxquant, Grep1, Remove beginning1, fasta2tab, fasta_cli, dbbuilder, filter_tabular, Cut1, Filter1, msconvert, query_tabular, tp_cat, Grouping1, peptide_shaker, fasta_merge_files_and_filter_unique_sequences, search_gui, ident_params</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Clinical Metaproteomics 4: Quantitation</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/clinical-mp-4-quantitation/tutorial.html</td>
      <td></td>
      <td>Clustering, Visualisation, Heat map generation, Standardisation and normalisation, Imputation, Principal component plotting, Protein quantification, Statistical calculation</td>
      <td>2025-12-16</td>
      <td>2025-12-16</td>
      <td>0</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>Grep1, Cut1, Grouping1, maxquant</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>0.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Calculating  and  diversity from microbiome taxonomic data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/diversity/tutorial.html</td>
      <td></td>
      <td>Visualisation, Aggregation</td>
      <td>2024-08-01</td>
      <td>2025-06-17</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>krakentools_alpha_diversity, krakentools_beta_diversity</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>Galaxy@AuBi, GalaxyTrakr</td>
      <td>0</td>
      <td></td>
      <td>6909</td>
      <td>8307</td>
      <td>1.216667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Taxonomic Profiling and Visualization of Metagenomic Data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/taxonomic-profiling/tutorial.html</td>
      <td>Metagenomics, Microbial ecology, Taxonomy, Sequence analysis</td>
      <td>Visualisation, Genome annotation, Aggregation, Statistical calculation, Taxonomic classification</td>
      <td>2023-05-03</td>
      <td>2025-04-24</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>est_abundance, taxonomy_krona_chart, kraken2, krakentools_kreport2krona, kraken_biom, metaphlan, __UNZIP_COLLECTION__, interactive_tool_phinch, interactive_tool_pavian</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>12409</td>
      <td>16575</td>
      <td>1.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Assembly of metagenomic sequencing data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metagenomics-assembly/tutorial.html</td>
      <td>Metagenomics, Sequence assembly</td>
      <td>Read mapping, Local alignment, Sequence contamination filtering, Sequence assembly visualisation, Data handling, Genome assembly, Statistical calculation, Read pre-processing, Sequence composition calculation, Variant calling, Sequence assembly validation, Visualisation, Sequence trimming, Sequencing quality control, Formatting, Sequence alignment analysis, Primer removal, Sequence file editing</td>
      <td>2022-12-05</td>
      <td>2025-12-08</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>bandage_info, metaspades, bg_uniq, megahit, megahit_contig2fastg, bandage_image, seqtk_subseq, coverm_contig, filter_tabular, tp_cat, fastqc, random_lines1, cutadapt, bamtools, quast, ngsutils_bam_filter, bowtie2, collection_column_join</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td>0</td>
      <td></td>
      <td>4454</td>
      <td>6194</td>
      <td>2.966667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Building an amplicon sequence variant (ASV) table from 16S data using DADA2</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/dada-16S/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td>Visualisation, Analysis, Deposition, DNA barcoding, Variant calling</td>
      <td>2024-06-05</td>
      <td>2025-07-03</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>dada2_assignTaxonomyAddspecies, dada2_mergePairs, dada2_removeBimeraDenovo, tp_replace_in_column, dada2_makeSequenceTable, phyloseq_from_dada2, dada2_plotQualityProfile, dada2_seqCounts, __SORTLIST__, dada2_filterAndTrim, Grouping1, Add_a_column1, dada2_learnErrors, cat1, tp_head_tool, dada2_dada, collection_element_identifiers, interactive_tool_phyloseq, __UNZIP_COLLECTION__, tp_replace_in_line</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>7173</td>
      <td>10151</td>
      <td>2.383333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Binning of metagenomic sequencing data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metagenomics-binning/tutorial.html</td>
      <td>Metagenomics, Sequence assembly</td>
      <td>Sequence assembly validation, Sequence clustering, Genome annotation, Read mapping, Sequence assembly, Read binning, Sequencing quality control, Validation, Genome comparison, Statistical calculation, Sequence composition calculation</td>
      <td>2023-12-05</td>
      <td>2025-12-02</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>samtools_sort, concoct_merge_cut_up_clustering, metabat2_jgi_summarize_bam_contig_depths, concoct_cut_up_fasta, concoct, concoct_extract_fasta_bins, drep_dereplicate, metabat2, semibin, checkm_lineage_wf, __BUILD_LIST__, Fasta_to_Contig2Bin, das_tool, maxbin2, concoct_coverage_table, binette, bowtie2</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr</td>
      <td>UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>5672</td>
      <td>6924</td>
      <td>1.466667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Identifying Mycorrhizal Fungi from ITS2 sequencing using LotuS2</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/lotus2-identifying-fungi/tutorial.html</td>
      <td></td>
      <td>DNA barcoding, Sequence feature detection</td>
      <td>2024-09-30</td>
      <td>2024-09-30</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>lotus2</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>743</td>
      <td>941</td>
      <td>1.666667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Clinical Metaproteomics 5: Data Interpretation</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/clinical-mp-5-data-interpretation/tutorial.html</td>
      <td></td>
      <td>Prediction and recognition, Visualisation</td>
      <td>2025-12-16</td>
      <td>2025-12-16</td>
      <td>0</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>unipept, msstatstmt, Grep1</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>16S Microbial analysis with Nanopore data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/nanopore-16S-metagenomics/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td>Visualisation, Sequence contamination filtering, Validation, Statistical calculation, Sequencing quality control, Taxonomic classification, Sequence composition calculation</td>
      <td>2020-11-24</td>
      <td>2024-07-31</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>fastp, Remove beginning1, porechop, kraken2, fastqc, taxonomy_krona_chart, multiqc, datamash_reverse, tp_replace_in_line</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>GalaxyTrakr, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>8336</td>
      <td>11507</td>
      <td>2.616667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Clinical Metaproteomics 3: Verification</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/clinical-mp-3-verification/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-12-16</td>
      <td>2025-12-16</td>
      <td>0</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>collapse_dataset, Remove beginning1, dbbuilder, query_tabular, Cut1, Filter1, uniprotxml_downloader, tp_cat, Grouping1, fasta_merge_files_and_filter_unique_sequences, pepquery2</td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td>UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Identification of the micro-organisms in a beer using Nanopore sequencing</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/beer-data-analysis/tutorial.html</td>
      <td>Metagenomics, Microbial ecology, Taxonomy, Sequence analysis</td>
      <td>Visualisation, Sequence contamination filtering, Aggregation, Statistical calculation, Sequencing quality control, Taxonomic classification, Sequence composition calculation</td>
      <td>2022-09-29</td>
      <td>2024-11-29</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>fastp, porechop, Filter1, kraken2, fastqc, krakentools_kreport2krona, taxonomy_krona_chart</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>GalaxyTrakr, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2741</td>
      <td>3989</td>
      <td>2.833333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Query an annotated mobile genetic element database to identify and annotate genetic elements (e.g. plasmids) in metagenomics data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metaplasmidome_query/tutorial.html</td>
      <td>Genomics, Microbiology, Sequence analysis, Mobile genetic elements</td>
      <td>Visualisation, Pairwise sequence alignment</td>
      <td>2025-01-07</td>
      <td>2025-01-07</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Add_a_column1, tp_sorted_uniq, tp_replace_in_column, tab2fasta, count_gff_features, tp_tail_tool, cat1, MQoutputfilter, Cut1, Filter1, minimap2, histogram_rpy, join1, Grouping1, add_column_headers, sort1, ggplot2_histogram, CONVERTER_fasta_to_tabular</td>
      <td>UseGalaxy.fr</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>306</td>
      <td>350</td>
      <td>0.816667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Metatranscriptomics analysis using microbiome RNA-seq data (short)</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metatranscriptomics-short/tutorial.html</td>
      <td>Metatranscriptomics, Microbial ecology, Taxonomy, Function analysis, Sequence analysis</td>
      <td>Species frequency estimation, Phylogenetic analysis, Validation, Sequence comparison, Phylogenetic inference, Taxonomic classification, Read pre-processing, Statistical calculation, Sequence composition calculation, Visualisation, Conversion, Phylogenetic tree visualisation, Sequencing quality control, Sequence trimming, Sequence alignment analysis, Sequence similarity search, Primer removal, Phylogenetic tree editing</td>
      <td>2020-02-13</td>
      <td>2025-06-16</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>export2graphlan, Grep1, tp_sort_header_tool, graphlan, tp_find_and_replace, metaphlan, humann_regroup_table, humann_renorm_table, multiqc, graphlan_annotate, taxonomy_krona_chart, cutadapt, fastq_paired_end_interlacer, humann, humann_rename_table, Cut1, bg_sortmerna, fastqc, humann_unpack_pathways, combine_metaphlan2_humann2, humann_split_stratified_table</td>
      <td>UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>804</td>
      <td>1052</td>
      <td>2.450000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Metatranscriptomics analysis using microbiome RNA-seq data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metatranscriptomics/tutorial.html</td>
      <td>Metatranscriptomics, Microbial ecology, Taxonomy, Function analysis, Sequence analysis</td>
      <td>Species frequency estimation, Phylogenetic analysis, Validation, Sequence comparison, Phylogenetic inference, Taxonomic classification, Read pre-processing, Statistical calculation, Sequence composition calculation, Visualisation, Conversion, Phylogenetic tree visualisation, Sequencing quality control, Sequence trimming, Sequence alignment analysis, Aggregation, Sequence similarity search, Primer removal, Phylogenetic tree editing</td>
      <td>2019-11-21</td>
      <td>2025-06-16</td>
      <td>6</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>export2graphlan, Grep1, tp_sort_header_tool, graphlan, tp_find_and_replace, metaphlan, humann_renorm_table, multiqc, graphlan_annotate, Grouping1, taxonomy_krona_chart, cutadapt, fastq_paired_end_interlacer, humann, humann_rename_table, Cut1, bg_sortmerna, fastqc, humann_unpack_pathways, humann_regroup_table, humann_split_stratified_table, combine_metaphlan_humann</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.eu</td>
      <td>0</td>
      <td></td>
      <td>3122425</td>
      <td>7973733</td>
      <td>4.266667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Clinical Metaproteomics 1: Database-Generation</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/clinical-mp-1-database-generation/tutorial.html</td>
      <td></td>
      <td>Tag-based peptide identification, Expression analysis, Target-Decoy, Protein identification, de Novo sequencing</td>
      <td>2025-12-16</td>
      <td>2025-12-16</td>
      <td>0</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>metanovo, dbbuilder, fasta_merge_files_and_filter_unique_sequences, uniprotxml_downloader</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org.au</td>
      <td>UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Indexing and profiling microbes with MetaSBT</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metasbt/tutorial.html</td>
      <td>Microbial ecology, Taxonomy</td>
      <td></td>
      <td>2025-08-25</td>
      <td>2025-08-25</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>53</td>
      <td>61</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>QIIME 2 Cancer Microbiome Intervention</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/qiime2-cancer-microbiome-intervention/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td></td>
      <td>2024-02-12</td>
      <td>2024-03-14</td>
      <td>3</td>
      <td>external</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>416</td>
      <td>671</td>
      <td>3.133333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>QIIME 2 Moving Pictures</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/qiime2-moving-pictures/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td></td>
      <td>2024-03-14</td>
      <td>2024-03-14</td>
      <td>1</td>
      <td>external</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>593</td>
      <td>912</td>
      <td>2.700000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>16S Microbial Analysis with mothur (short)</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/mothur-miseq-sop-short/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td>Visualisation, Sequence clustering, Phylogenetic analysis, Sequencing quality control, Phylogenetic tree generation, Phylogenetic tree reconstruction, Phylogenetic tree analysis, DNA barcoding, Sequence read processing, Taxonomic classification</td>
      <td>2019-05-13</td>
      <td>2024-06-14</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>mothur_make_shared, mothur_venn, mothur_classify_otu, mothur_heatmap_sim, mothur_tree_shared, mothur_dist_shared, mothur_filter_seqs, mothur_pre_cluster, mothur_rarefaction_single, mothur_unique_seqs, mothur_classify_seqs, mothur_summary_single, newick_display, mothur_summary_seqs, mothur_screen_seqs, mothur_cluster_split, collapse_dataset, mothur_dist_seqs, mothur_remove_lineage, mothur_sub_sample, mothur_chimera_vsearch, mothur_count_seqs, mothur_remove_groups, XY_Plot_1, mothur_cluster, mothur_count_groups, mothur_remove_seqs</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>5559</td>
      <td>9374</td>
      <td>4.583333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>16S Microbial Analysis with mothur (extended)</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/mothur-miseq-sop/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td>Visualisation, Sequence clustering, Phylogenetic analysis, Sequencing quality control, Phylogenetic tree generation, Phylogenetic tree reconstruction, Phylogenetic tree analysis, DNA barcoding, Sequence read processing, Taxonomic classification</td>
      <td>2017-02-12</td>
      <td>2024-03-14</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>mothur_make_shared, mothur_venn, mothur_classify_otu, mothur_heatmap_sim, mothur_tree_shared, mothur_dist_shared, mothur_filter_seqs, mothur_pre_cluster, mothur_rarefaction_single, mothur_make_contigs, mothur_unique_seqs, mothur_classify_seqs, mothur_summary_single, mothur_align_seqs, mothur_seq_error, mothur_taxonomy_to_krona, newick_display, mothur_summary_seqs, mothur_screen_seqs, mothur_cluster_split, mothur_dist_seqs, mothur_remove_lineage, mothur_get_groups, mothur_sub_sample, taxonomy_krona_chart, mothur_chimera_vsearch, mothur_make_biom, mothur_count_seqs, mothur_remove_groups, XY_Plot_1, mothur_cluster, mothur_count_groups, mothur_remove_seqs</td>
      <td>UseGalaxy.be, UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>12289</td>
      <td>18057</td>
      <td>2.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Pox virus genome analysis from tiled-amplicon sequencing data</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/pox-tiled-amplicon/tutorial.html</td>
      <td></td>
      <td>Generation, Read mapping, Local alignment, Genome indexing, Sequence contamination filtering, Sequencing quality control, Multiple sequence alignment, Sequence alignment analysis, Sequence alignment, Primer removal, Sequence analysis, Global alignment</td>
      <td>2023-05-15</td>
      <td>2024-03-20</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>ivar_trim, __ZIP_COLLECTION__, Grep1, rbc_mafft, param_value_from_file, compose_text_param, fastp, datamash_ops, fatovcf, fasta_compute_length, ivar_consensus, upload1, samtools_merge, bwa_mem, samtools_view, __APPLY_RULES__, Add_a_column1, fasta_regex_finder, Cut1, qualimap_bamqc, mimodd_info, EMBOSS: maskseq51</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>1056</td>
      <td>1251</td>
      <td>2.300000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>M. tuberculosis Variant Analysis</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/tb-variant-analysis/tutorial.html</td>
      <td>Genomics, Microbiology, Sequence assembly, Genetic variation, Public health and epidemiology, Infectious disease</td>
      <td>Antimicrobial resistance prediction, Local alignment, Phylogenetic tree visualisation, Sequence contamination filtering, Sequencing quality control, Validation, Phylogenetic tree generation, Sequence alignment analysis, Global alignment, Genome visualisation, Sequence alignment, Statistical calculation, Sequence analysis, Taxonomic classification, Sequence composition calculation, Variant calling</td>
      <td>2020-07-25</td>
      <td>2025-11-18</td>
      <td>31</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>kraken2, tb_variant_filter, tbvcfreport, fastp, mosdepth, snippy, tp_sed_tool, samtools_stats, jvarkit_wgscoverageplotter, multiqc, upload1, __FLATTEN__, tb_profiler_profile, bcftools_consensus, tp_awk_tool, jbrowse, EMBOSS: seqret84, EMBOSS:%20seqret84, qualimap_bamqc, fastqc</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>7558</td>
      <td>11661</td>
      <td>3.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Avian influenza viral strain analysis from gene segment sequencing data</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/aiv-analysis/tutorial.html</td>
      <td></td>
      <td>Generation, Read mapping, Data retrieval, Phylogenetic analysis, Sequencing quality control, Sequence contamination filtering, Genome indexing, Multiple sequence alignment, Sequence alignment analysis, De-novo assembly, Sequence alignment, Data handling, Primer removal, Sequence analysis, Sequence file editing</td>
      <td>2022-11-21</td>
      <td>2024-10-04</td>
      <td>10</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>iqtree, upload1, __RELABEL_FROM_FILE__, Grep1, seqtk_subseq, collapse_dataset, bamtools_split_ref, rbc_mafft, Show beginning1, qualimap_bamqc, ivar_consensus, tp_find_and_replace, fastp, bwa_mem, samtools_view, vapor</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>2075</td>
      <td>3011</td>
      <td>2.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>Whole transcriptome analysis of Arabidopsis thaliana</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/mirna-target-finder/tutorial.html</td>
      <td></td>
      <td>Visualisation, Read mapping, Sequence trimming, Sequencing quality control, Differential gene expression analysis, Sequence composition calculation, Validation, RNA-Seq quantification, RNA-Seq analysis, Primer removal, Read pre-processing, Gene expression analysis</td>
      <td>2021-04-08</td>
      <td>2025-08-10</td>
      <td>18</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>falco, targetfinder, cat1, Cut1, Filter1, deseq2, trim_galore, __MERGE_COLLECTION__, sample_seqs, sort1, rbc_mirdeep2_quantifier, filter_by_fasta_ids, multiqc, rbc_mirdeep2_mapper, salmon</td>
      <td>UseGalaxy.cz</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2976450</td>
      <td>7462024</td>
      <td>4.866667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Sequence analysis</td>
      <td>Identification and Evolutionary Analysis of Transcription-Associated Proteins in Streptophyte algae and Land plants</td>
      <td>https://training.galaxyproject.org//topics/sequence-analysis/tutorials/tapscan-streptophyte-algae/tutorial.html</td>
      <td></td>
      <td>Phylogenetic inference (from molecular sequences), Phylogenetic analysis, Sequence analysis, Phylogenetic tree editing, Multiple sequence alignment</td>
      <td>2025-03-26</td>
      <td>2025-04-27</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>iqtree, quicktree, Remove beginning1, rbc_mafft, Cut1, Filter1, ete_treeviewer, trimal, filter_by_fasta_ids, tapscan_classify</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>258</td>
      <td>316</td>
      <td>1.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Sequence analysis</td>
      <td>NCBI BLAST+ against the MAdLand</td>
      <td>https://training.galaxyproject.org//topics/sequence-analysis/tutorials/ncbi-blast-against-the-madland/tutorial.html</td>
      <td></td>
      <td>Sequence alignment analysis</td>
      <td>2023-01-16</td>
      <td>2024-07-31</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>ncbi_blastp_wrapper, bg_diamond, ncbi_blastx_wrapper</td>
      <td>Galaxy@AuBi, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>1692</td>
      <td>2139</td>
      <td>3.033333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Analysis of plant scRNA-Seq Data with Scanpy</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-plant/tutorial.html</td>
      <td></td>
      <td>Expression profile clustering, Principal component visualisation, Differential gene expression analysis</td>
      <td>2021-04-08</td>
      <td>2025-02-13</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>scanpy_inspect, scanpy_cluster_reduce_dimension, scanpy_plot, anndata_import, scanpy_remove_confounders, anndata_manipulate, scanpy_normalize, scanpy_filter</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2786</td>
      <td>3476</td>
      <td>1.600000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Statistical analysis of DIA data</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/DIA_Analysis_MSstats/tutorial.html</td>
      <td></td>
      <td>Differential protein expression profiling, Tag-based peptide identification, Spectrum calculation</td>
      <td>2020-12-02</td>
      <td>2023-11-09</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>histogram_rpy, msstats, Grep1, Filter1</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>1350</td>
      <td>1885</td>
      <td>2.566667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Neoantigen 5b: IEDB binding Validated Neopeptides</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/neoantigen-hla-binding-novel-peptides/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-01-14</td>
      <td>2025-12-03</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>table_compute, query_tabular, Filter1, fasta_to_tabular, iedb_api</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>66</td>
      <td>70</td>
      <td>0.500000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Proteogenomics 2: Database Search</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/proteogenomics-dbsearch/tutorial.html</td>
      <td></td>
      <td>Conversion, Peptide database search</td>
      <td>2018-11-20</td>
      <td>2025-05-08</td>
      <td>24</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>tab2fasta, mz_to_sqlite, query_tabular, peptide_shaker, search_gui, ident_params</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>1558</td>
      <td>2310</td>
      <td>4.750000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Library Generation for DIA Analysis</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/DIA_lib_OSW/tutorial.html</td>
      <td></td>
      <td>Clustering, Visualisation, Heat map generation, Standardisation and normalisation, Imputation, Principal component plotting, Protein quantification, Statistical calculation</td>
      <td>2020-12-02</td>
      <td>2025-06-02</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>maxquant, TargetedFileConverter, OpenSwathDecoyGenerator, Filter1, diapysef, OpenSwathAssayGenerator</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>4619</td>
      <td>6040</td>
      <td>1.366667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Proteogenomics 3: Novel peptide analysis</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/proteogenomics-novel-peptide-analysis/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-11-20</td>
      <td>2025-05-15</td>
      <td>20</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>pep_pointer, ncbi_blastp_wrapper, peptide_genomic_coordinate, query_tabular</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>1700</td>
      <td>2448</td>
      <td>2.600000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>metaQuantome 3: Taxonomy</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaquantome-taxonomy/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Metatranscriptomics, Microbial ecology, Metagenomics, Taxonomy</td>
      <td>Query and retrieval, Heat map generation, Filtering, Principal component visualisation, Functional clustering, Quantification, Visualisation, Indexing, Statistical inference, Differential protein expression analysis</td>
      <td>2020-10-29</td>
      <td>2024-03-15</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>metaquantome_viz, metaquantome_stat, metaquantome_sample, metaquantome_db, metaquantome_filter, metaquantome_expand</td>
      <td>UseGalaxy.be, UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>448</td>
      <td>586</td>
      <td>2.250000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Neoantigen 1a: Fusion-Database-Generation</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/neoantigen-fusion-database-generation/tutorial.html</td>
      <td></td>
      <td>Sequence alignment</td>
      <td>2025-01-14</td>
      <td>2025-12-03</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>regex1, tab2fasta, arriba, query_tabular, CONVERTER_gz_to_uncompressed, arriba_get_filters, tp_awk_tool, rna_star</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>163</td>
      <td>194</td>
      <td>1.116667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Neoantigen 2: Database merge and FragPipe discovery</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/neoantigen-fragpipe-discovery/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-01-14</td>
      <td>2025-12-03</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>validate_fasta_database, fragpipe_manifest_generator, Grep1, Remove beginning1, collapse_dataset, query_tabular, fragpipe, fasta_merge_files_and_filter_unique_sequences</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>207</td>
      <td>241</td>
      <td>1.116667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Clinical Metaproteomics 2: Discovery</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/clinical-mp-2-discovery/tutorial.html</td>
      <td></td>
      <td>Clustering, Visualisation, Heat map generation, Standardisation and normalisation, Formatting, Imputation, Principal component plotting, Protein quantification, Filtering, Statistical calculation</td>
      <td>2024-02-06</td>
      <td>2025-08-10</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>maxquant, Grep1, Remove beginning1, fasta2tab, fasta_cli, dbbuilder, filter_tabular, Cut1, Filter1, msconvert, query_tabular, tp_cat, Grouping1, peptide_shaker, fasta_merge_files_and_filter_unique_sequences, search_gui, ident_params</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>351</td>
      <td>465</td>
      <td>6.216667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Detection and quantitation of N-termini (degradomics) via N-TAILS</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/ntails/tutorial.html</td>
      <td>Proteomics</td>
      <td></td>
      <td>2017-02-14</td>
      <td>2025-04-08</td>
      <td>16</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>IDMerger, FeatureFinderMultiplex, HighResPrecursorMassCorrector, MzTabExporter, IDConflictResolver, FileFilter, FileMerger, MSGFPlusAdapter, FalseDiscoveryRate, IDMapper, PeptideIndexer, ConsensusID, IDFilter</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>873</td>
      <td>1198</td>
      <td>1.400000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Proteogenomics 1: Database Creation</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/proteogenomics-dbcreation/tutorial.html</td>
      <td></td>
      <td>RNA-Seq analysis, Sequence annotation, Sequence alignment, Transcriptome assembly, Statistical calculation, Variant calling</td>
      <td>2018-11-20</td>
      <td>2025-05-08</td>
      <td>29</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>bed_to_protein_map, hisat2, sqlite_to_tabular, fasta2tab, tab2fasta, tp_replace_in_column, gffcompare_to_bed, stringtie, filter_tabular, query_tabular, freebayes, regexColumn1, cat_multi_datasets, gffcompare, custom_pro_db, translate_bed, fasta_merge_files_and_filter_unique_sequences</td>
      <td>UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>5924</td>
      <td>9660</td>
      <td>2.583333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Neoantigen 5a: Predicting HLA Binding</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/neoantigen-predicting-hla-binding/tutorial.html</td>
      <td></td>
      <td>Read mapping, Genetic variation analysis</td>
      <td>2025-01-14</td>
      <td>2025-12-03</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>query_tabular, tp_awk_tool, seq2hla, optitype</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>71</td>
      <td>76</td>
      <td>0.033333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Clinical Metaproteomics 4: Quantitation</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/clinical-mp-4-quantitation/tutorial.html</td>
      <td></td>
      <td>Clustering, Visualisation, Heat map generation, Standardisation and normalisation, Imputation, Principal component plotting, Protein quantification, Statistical calculation</td>
      <td>2024-02-06</td>
      <td>2025-08-10</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>Grep1, Cut1, Grouping1, maxquant</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>183</td>
      <td>249</td>
      <td>2.683333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Mass spectrometry imaging: Loading and exploring MSI data</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/mass-spectrometry-imaging-loading-exploring-data/tutorial.html</td>
      <td>Proteomics</td>
      <td></td>
      <td>2018-11-27</td>
      <td>2025-04-08</td>
      <td>18</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>cardinal_quality_report, Grep1, tp_sort_header_tool, cardinal_data_exporter</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>2784</td>
      <td>4284</td>
      <td>1.666667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Protein FASTA Database Handling</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/database-handling/tutorial.html</td>
      <td>Proteomics</td>
      <td></td>
      <td>2017-02-14</td>
      <td>2025-04-08</td>
      <td>30</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>addValue, fasta2tab, tab2fasta, DecoyDatabase, dbbuilder, fasta_merge_files_and_filter_unique_sequences</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>4822</td>
      <td>7650</td>
      <td>2.166667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Label-free data analysis using MaxQuant</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/maxquant-label-free/tutorial.html</td>
      <td></td>
      <td>Differential protein expression profiling, Clustering, Visualisation, Tag-based peptide identification, Heat map generation, Standardisation and normalisation, Spectrum calculation, Imputation, Principal component plotting, Protein quantification, Statistical calculation</td>
      <td>2020-04-29</td>
      <td>2025-05-21</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Add_a_column1, maxquant, msstats, Grep1, tp_sort_header_tool, Cut1, Filter1, histogram_rpy</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>20066</td>
      <td>29284</td>
      <td>1.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Machine Learning Modeling of Anticancer Peptides</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/ml-modeling-of-anti-cancer-peptides/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-01-22</td>
      <td>2023-11-09</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>pdaug_sequence_property_based_descriptors, pdaug_ml_models, pdaug_basic_plots, pdaug_merge_dataframes, pdaug_addclasslabel</td>
      <td>UseGalaxy.be</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1033</td>
      <td>1535</td>
      <td>2.183333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Neoantigen 4: Variant Annotation</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/neoantigen-variant-annotation/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-01-14</td>
      <td>2025-12-03</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>pep_pointer, cut, tab2fasta, query_tabular, regexColumn1, select, Convert characters1, column_regex_find_and_replace</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>82</td>
      <td>97</td>
      <td>1.166667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Biomarker candidate identification</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/biomarker_selection/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-06-13</td>
      <td>2023-11-09</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Jvenn, retrieve_from_hpa, MQoutputfilter, tp_cut_tool, prot_features, IDconverter, retr_pepatlas1, rna_abbased_data</td>
      <td></td>
      <td>UseGalaxy.be, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1707</td>
      <td>2315</td>
      <td>2.483333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>MaxQuant and MSstats for the analysis of label-free data</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/maxquant-msstats-dda-lfq/tutorial.html</td>
      <td></td>
      <td>Differential protein expression profiling, Clustering, Visualisation, Tag-based peptide identification, Heat map generation, Standardisation and normalisation, Spectrum calculation, Imputation, Principal component plotting, Protein quantification, Statistical calculation</td>
      <td>2021-02-17</td>
      <td>2025-09-22</td>
      <td>19</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>ggplot2_heatmap2, maxquant, msstats, Grep1, tp_replace_in_column, fasta2tab, uniprot, Cut1, Filter1, datamash_ops, tp_easyjoin_tool, Summary_Statistics1</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>8548</td>
      <td>13743</td>
      <td>2.650000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Neoantigen 3: PepQuery2 Verification</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/neoantigen-peptide-verification/tutorial.html</td>
      <td></td>
      <td>Formatting, Filtering</td>
      <td>2025-01-14</td>
      <td>2025-12-03</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>ncbi_blastp_wrapper, tab2fasta, query_tabular, msconvert, pepquery2</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>66</td>
      <td>74</td>
      <td>1.733333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Annotating a protein list identified by LC-MS/MS experiments</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/proteome_annotation/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-09-12</td>
      <td>2024-06-14</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>Jvenn, retrieve_from_hpa, MQoutputfilter, IDconverter, rna_abbased_data, reactome_analysis, cluter_profiler</td>
      <td></td>
      <td>UseGalaxy.be, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>3098</td>
      <td>4492</td>
      <td>2.750000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>DIA Analysis using OpenSwathWorkflow</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/DIA_Analysis_OSW/tutorial.html</td>
      <td></td>
      <td>Formatting, Filtering</td>
      <td>2020-12-02</td>
      <td>2025-05-20</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>pyprophet_score, OpenSwathWorkflow, Grep1, pyprophet_merge, msconvert, pyprophet_peptide, pyprophet_protein, pyprophet_export</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>3243</td>
      <td>4611</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>EncyclopeDIA</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/encyclopedia/tutorial.html</td>
      <td></td>
      <td>Formatting, Filtering</td>
      <td>2021-06-21</td>
      <td>2025-05-19</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>encyclopedia_searchtolib, encyclopedia_quantify, msconvert</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>2007</td>
      <td>2729</td>
      <td>1.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Peptide and Protein ID using SearchGUI and PeptideShaker</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/protein-id-sg-ps/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2017-06-12</td>
      <td>2025-04-08</td>
      <td>28</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Grep1, peptide_shaker, search_gui, PeakPickerHiRes, FileConverter</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3179</td>
      <td>5060</td>
      <td>1.866667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Peptide and Protein ID using OpenMS tools</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/protein-id-oms/tutorial.html</td>
      <td></td>
      <td>Formatting, Filtering</td>
      <td>2017-10-16</td>
      <td>2025-04-08</td>
      <td>32</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>IDMerger, Grep1, IDScoreSwitcher, IDPosteriorErrorProbability, MSGFPlusAdapter, TextExporter, msconvert, FileInfo, FalseDiscoveryRate, PeptideIndexer, XTandemAdapter, PeakPickerHiRes, ConsensusID, FidoAdapter, IDFilter</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>3557</td>
      <td>5965</td>
      <td>2.866667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Clinical Metaproteomics 5: Data Interpretation</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/clinical-mp-5-data-interpretation/tutorial.html</td>
      <td></td>
      <td>Prediction and recognition, Visualisation</td>
      <td>2024-02-06</td>
      <td>2025-08-10</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>unipept, msstatstmt, Grep1</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>296</td>
      <td>386</td>
      <td>2.366667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Neoantigen 1b: Non-Reference-Database-Generation</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/neoantigen-non-reference-database-generation/tutorial.html</td>
      <td></td>
      <td>RNA-Seq analysis, Sequence annotation, Sequence alignment, Transcriptome assembly, Statistical calculation, Variant calling</td>
      <td>2025-01-14</td>
      <td>2025-12-03</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>concatenate, bed_to_protein_map, sqlite_to_tabular, hisat2, fasta2tab, tab2fasta, gffcompare_to_bed, stringtie, CONVERTER_gz_to_uncompressed, freebayes, regexColumn1, gffcompare, custom_pro_db, translate_bed, fasta_merge_files_and_filter_unique_sequences, column_regex_find_and_replace</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>93</td>
      <td>105</td>
      <td>0.583333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Multiomics data analysis using MultiGSEA</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/multiGSEA-tutorial/tutorial.html</td>
      <td></td>
      <td>Pathway analysis, Gene-set enrichment analysis, Aggregation</td>
      <td>2025-03-10</td>
      <td>2025-03-11</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>multigsea, tp_grep_tool, Filter1</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>305</td>
      <td>335</td>
      <td>1.483333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Metaproteomics tutorial</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaproteomics/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Biodiversity, Taxonomy</td>
      <td>Prediction and recognition, Visualisation</td>
      <td>2017-06-28</td>
      <td>2024-06-14</td>
      <td>41</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>sqlite_to_tabular, query_tabular, unipept, peptide_shaker, search_gui</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>2919</td>
      <td>4637</td>
      <td>2.716667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Clinical Metaproteomics 3: Verification</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/clinical-mp-3-verification/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-02-06</td>
      <td>2025-08-10</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>collapse_dataset, Remove beginning1, dbbuilder, query_tabular, Cut1, Filter1, uniprotxml_downloader, tp_cat, Grouping1, fasta_merge_files_and_filter_unique_sequences, pepquery2</td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td>UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>207</td>
      <td>273</td>
      <td>3.700000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Peptide and Protein Quantification via Stable Isotope Labelling (SIL)</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/protein-quant-sil/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2017-03-17</td>
      <td>2025-04-08</td>
      <td>32</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>FeatureFinderMultiplex, Grep1, IDScoreSwitcher, tp_tail_tool, ProteinQuantifier, IDConflictResolver, IDPosteriorErrorProbability, MultiplexResolver, TextExporter, FileInfo, histogram_rpy, FalseDiscoveryRate, IDMapper, PeptideIndexer, XTandemAdapter, Summary_Statistics1, FidoAdapter</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>1370</td>
      <td>1909</td>
      <td>1.533333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Peptide Library Data Analysis</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/peptide-library-data-analysis/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-01-22</td>
      <td>2023-11-09</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>pdaug_sequence_property_based_descriptors, pdaug_peptide_data_access, pdaug_tsvtofasta, pdaug_peptide_sequence_analysis, pdaug_basic_plots, pdaug_merge_dataframes, pdaug_fishers_plot, pdaug_addclasslabel</td>
      <td>UseGalaxy.be</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>750</td>
      <td>1015</td>
      <td>2.300000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>MaxQuant and MSstats for the analysis of TMT data</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/maxquant-msstats-tmt/tutorial.html</td>
      <td></td>
      <td>Clustering, Visualisation, Heat map generation, Standardisation and normalisation, Imputation, Principal component plotting, Protein quantification, Statistical calculation</td>
      <td>2021-08-13</td>
      <td>2025-01-08</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>msstatstmt, maxquant</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>4295</td>
      <td>5734</td>
      <td>1.633333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Secretome Prediction</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/secretome-prediction/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2017-04-13</td>
      <td>2025-04-08</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>get_subontology_from, tp_sorted_uniq, comp1, Grep1, bg_uniq, uniprot, Filter1, tp_cut_tool, Convert characters1, wolf_psort, tp_easyjoin_tool, term_id_vs_term_name</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>652</td>
      <td>971</td>
      <td>1.466667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>metaQuantome 2: Function</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaquantome-function/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Metatranscriptomics, Microbial ecology, Metagenomics</td>
      <td>Query and retrieval, Heat map generation, Filtering, Principal component visualisation, Functional clustering, Quantification, Visualisation, Indexing, Statistical inference, Differential protein expression analysis</td>
      <td>2020-10-29</td>
      <td>2024-03-15</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>metaquantome_viz, metaquantome_stat, metaquantome_sample, metaquantome_db, metaquantome_filter, metaquantome_expand</td>
      <td>UseGalaxy.be, UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>422</td>
      <td>593</td>
      <td>4.133333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>metaQuantome 1: Data creation</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaquantome-data-creation/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Biodiversity, Taxonomy</td>
      <td>Prediction and recognition, Visualisation, Filtering, Label-free quantification, Formatting</td>
      <td>2020-10-16</td>
      <td>2024-03-14</td>
      <td>20</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>regex1, search_gui, tp_replace_in_column, Grep1, Remove beginning1, flashlfq, query_tabular, Cut1, Filter1, msconvert, unipept, peptide_shaker, tp_replace_in_line</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>892</td>
      <td>1506</td>
      <td>3.733333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Single Cell Proteomics data analysis with bioconductor-scp</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/bioconductor-scp/tutorial.html</td>
      <td></td>
      <td>Visualisation, Expression analysis</td>
      <td>2025-02-07</td>
      <td>2025-02-10</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>bioconductor_scp, Grouping1</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>243</td>
      <td>303</td>
      <td>1.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Clinical Metaproteomics 1: Database-Generation</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/clinical-mp-1-database-generation/tutorial.html</td>
      <td></td>
      <td>Tag-based peptide identification, Expression analysis, Target-Decoy, Protein identification, de Novo sequencing</td>
      <td>2024-02-06</td>
      <td>2025-07-30</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>metanovo, dbbuilder, fasta_merge_files_and_filter_unique_sequences, uniprotxml_downloader</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org.au</td>
      <td>UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>599</td>
      <td>899</td>
      <td>3.833333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Introduction to proteomics, protein identification, quantification and statistical modelling</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/introduction/slides.html</td>
      <td></td>
      <td></td>
      <td>2017-12-01</td>
      <td>2024-08-08</td>
      <td>3</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>3115664</td>
      <td>7964932</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Label-free versus Labelled - How to Choose Your Quantitation Method</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/labelfree-vs-labelled/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2017-03-17</td>
      <td>2025-05-27</td>
      <td>18</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>7698</td>
      <td>12414</td>
      <td>0.900000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Sequence analysis</td>
      <td>Mapping</td>
      <td>https://training.galaxyproject.org//topics/sequence-analysis/tutorials/mapping/tutorial.html</td>
      <td>Mapping</td>
      <td>Read mapping, Sequence trimming, Sequencing quality control, Sequence alignment analysis, Validation, Genome visualisation, Data handling, Primer removal, Read pre-processing, Statistical calculation, Sequence composition calculation</td>
      <td>2016-10-04</td>
      <td>2025-03-31</td>
      <td>43</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>jbrowse, samtools_stats, trim_galore, fastqc, bamFilter, multiqc, bowtie2</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>Galaxy@AuBi, GalaxyTrakr, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>3181750</td>
      <td>8075423</td>
      <td>5.066667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Sequence analysis</td>
      <td>Clean and manage Sanger sequences from raw files to aligned consensus</td>
      <td>https://training.galaxyproject.org//topics/sequence-analysis/tutorials/Manage_AB1_Sanger/tutorial.html</td>
      <td></td>
      <td>Visualisation, Sequence clustering, Phylogenetic analysis, Demultiplexing, Sequencing quality control, Data handling, DNA barcoding, Sequence read processing, Taxonomic classification, Sequence file editing</td>
      <td>2024-01-08</td>
      <td>2024-03-05</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>ncbi_blastn_wrapper, mothur_merge_files, qiime_align_seqs, mothur_degap_seqs, unzip, filter_by_fasta_ids, tab2fasta, fastq_groomer, seqtk_trimfq, __FILTER_FROM_FILE__, fastq_to_tabular, cshl_fastx_reverse_complement, seqtk_mergepe, cshl_fastq_to_fasta, __SORTLIST__, regex1, fastq_masker_by_quality, ab1_fastq_converter, aligned_to_consensus, collection_element_identifiers</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3409</td>
      <td>4264</td>
      <td>1.650000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Sequence analysis</td>
      <td>Primer and primer scheme design for pan-specific detection and sequencing of viral pathogens across genotypes</td>
      <td>https://training.galaxyproject.org//topics/sequence-analysis/tutorials/viral_primer_design/tutorial.html</td>
      <td></td>
      <td>Sequence alignment, Local alignment, Sequence alignment analysis, Sequence analysis, PCR primer design, Global alignment, Multiple sequence alignment</td>
      <td>2025-03-20</td>
      <td>2025-11-20</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>rbc_mafft, split_file_to_collection, fasta_compute_length, varvamp, ncbi_makeblastdb, EMBOSS: degapseq20</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>631</td>
      <td>729</td>
      <td>0.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Sequence analysis</td>
      <td>Quality Control</td>
      <td>https://training.galaxyproject.org//topics/sequence-analysis/tutorials/quality-control/tutorial.html</td>
      <td></td>
      <td>Sequence trimming, Sequencing quality control, Sequence composition calculation, Validation, Box-Whisker plot plotting, Scatter plot plotting, Primer removal, Statistical calculation, Read pre-processing</td>
      <td>2016-10-04</td>
      <td>2025-03-29</td>
      <td>48</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>pycoqc, fastqc, fastqe, cutadapt, multiqc, nanoplot</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>Galaxy@AuBi, UseGalaxy.cz, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>3223472</td>
      <td>8160689</td>
      <td>5.333333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Sequence analysis</td>
      <td>Screening assembled genomes for contamination using NCBI FCS</td>
      <td>https://training.galaxyproject.org//topics/sequence-analysis/tutorials/ncbi-fcs/tutorial.html</td>
      <td></td>
      <td>Sequence assembly validation, Sequence contamination filtering, Sequence trimming</td>
      <td>2024-04-16</td>
      <td>2024-04-16</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>ncbi_fcs_gx, ncbi_fcs_adaptor</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>1889</td>
      <td>2556</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Sequence analysis</td>
      <td>Quality and contamination control in bacterial isolate using Illumina MiSeq Data</td>
      <td>https://training.galaxyproject.org//topics/sequence-analysis/tutorials/quality-contamination-control/tutorial.html</td>
      <td>Whole genome sequencing, Genomics, Microbiology, Microbial ecology</td>
      <td>Visualisation, Read mapping, Sequencing quality control, Sequence contamination filtering, Cross-assembly, Expression analysis, Statistical calculation, Taxonomic classification</td>
      <td>2024-07-15</td>
      <td>2025-03-20</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>falco, upload1, est_abundance, recentrifuge, kraken2, fastp</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td>GalaxyTrakr, UseGalaxy.cz, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>1699</td>
      <td>2307</td>
      <td>2.783333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Bulk RNA Deconvolution with MuSiC</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/bulk-music/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-02-11</td>
      <td>2025-02-14</td>
      <td>10</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>music_inspect_eset, music_construct_eset, music_deconvolution</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>4017</td>
      <td>5060</td>
      <td>1.583333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Generating a single cell matrix using Alevin</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-case_alevin/tutorial.html</td>
      <td></td>
      <td>Gene expression analysis, Sequence composition calculation, RNA-Seq quantification</td>
      <td>2021-03-03</td>
      <td>2025-11-18</td>
      <td>25</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>dropletutils_empty_drops, _salmon_kallisto_mtx_to_10x, alevin, Cut1, _ensembl_gtf2gene_list, _dropletBarcodePlot, anndata_manipulate, join1, sceasy_convert, dropletutils_read_10x, anndata_inspect</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3501</td>
      <td>5306</td>
      <td>3.433333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Converting between common single cell data formats</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-data-ingest/tutorial.html</td>
      <td></td>
      <td>Community profiling, Loading</td>
      <td>2024-02-13</td>
      <td>2025-03-11</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>monocle3_create, dropletutils, anndata_inspect, tp_tail_tool, tabular_to_fastq, Cut1, Filter1, regexColumn1, seurat_read10x, tp_cut_tool, join1, anndata_ops, scanpy_filter_cells, sceasy_convert, dropletutils_read_10x, datamash_transpose, fastq_to_tabular</td>
      <td></td>
      <td>UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>2623</td>
      <td>3108</td>
      <td>1.283333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Pre-processing of 10X Single-Cell RNA Datasets</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-preprocessing-tenx/tutorial.html</td>
      <td></td>
      <td>Community profiling, Sequence alignment, Loading, Sequencing quality control, Validation</td>
      <td>2019-09-11</td>
      <td>2025-04-29</td>
      <td>13</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>multiqc, dropletutils, rna_starsolo</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.fr, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>12701</td>
      <td>16423</td>
      <td>1.866667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Pre-processing of Single-Cell RNA Data</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-preprocessing/tutorial.html</td>
      <td></td>
      <td>Sequence alignment, Data handling, Sequence alignment analysis, Read summarisation, Sequencing quality control, Validation, RNA-Seq quantification</td>
      <td>2019-02-22</td>
      <td>2025-04-08</td>
      <td>5</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>umi_tools_count, umi_tools_extract, tp_sed_tool, __FLATTEN__, samtools_stats, featurecounts, bamFilter, multiqc, rna_star, collection_column_join</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>3122114</td>
      <td>7973328</td>
      <td>4.583333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Pseudobulk Analysis with Decoupler and EdgeR</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/pseudobulk-analysis/tutorial.html</td>
      <td></td>
      <td>Principal component visualisation, Expression profile clustering, Differential gene expression analysis</td>
      <td>2025-02-12</td>
      <td>2025-05-09</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>tp_replace_in_column, anndata_inspect, column_remove_by_header, anndata_manipulate, edger, decoupler_pseudobulk, volcanoplot, __EXTRACT_DATASET__, tp_awk_tool, tp_replace_in_line, scanpy_filter</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>1622</td>
      <td>1863</td>
      <td>0.650000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Single-cell quality control with scater</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-scater-qc/tutorial.html</td>
      <td></td>
      <td>Sequence visualisation, Sequencing quality control, Read pre-processing</td>
      <td>2019-10-23</td>
      <td>2025-02-14</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>scater_plot_pca, scater_filter, scater_create_qcmetric_ready_sce, scater_plot_dist_scatter</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>714</td>
      <td>805</td>
      <td>0.716667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>GO Enrichment Analysis on Single-Cell RNA-Seq Data</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/GO-enrichment/tutorial.html</td>
      <td></td>
      <td>Gene-set enrichment analysis</td>
      <td>2024-09-17</td>
      <td>2025-04-08</td>
      <td>8</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>gprofiler_gost, goenrichment, Cut1, tp_split_on_column</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1005488</td>
      <td>2138917</td>
      <td>2.716667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Pre-processing of 10X Single-Cell ATAC-seq Datasets</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scatac-preprocessing-tenx/tutorial.html</td>
      <td></td>
      <td>Visualisation, Read mapping, Expression profile clustering, Generation, Peak calling, Sequencing quality control, Differential gene expression analysis, Gene regulatory network analysis, Genome indexing, Imputation, Sequence alignment, Principal component visualisation, Mapping, Statistical calculation, Sequence composition calculation, Enrichment analysis</td>
      <td>2023-04-24</td>
      <td>2025-10-16</td>
      <td>10</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tp_sorted_uniq, falco, episcanpy_build_matrix, scanpy_plot, episcanpy_preprocess, fastqc, sinto_barcode, sinto_fragments, macs2_callpeak, bedtools_sortbed, anndata_inspect, bwa_mem</td>
      <td>UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>6052</td>
      <td>7113</td>
      <td>1.083333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>An introduction to scRNA-seq data analysis</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-intro/slides.html</td>
      <td></td>
      <td></td>
      <td>2021-01-29</td>
      <td>2025-02-13</td>
      <td>8</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>3083239</td>
      <td>7850895</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Generating a single cell matrix using Alevin and combining datasets (bash + R)</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/alevin-commandline/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-12-08</td>
      <td>2025-04-22</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>625</td>
      <td>861</td>
      <td>3.583333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Removing the effects of the cell cycle</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-case_cell-cycle/tutorial.html</td>
      <td></td>
      <td>Expression profile clustering, Principal component visualisation, Differential gene expression analysis</td>
      <td>2023-01-25</td>
      <td>2025-03-11</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>scanpy_inspect, addValue, scanpy_regress_variable, scanpy_cluster_reduce_dimension, table_compute, scanpy_plot, cat1, anndata_manipulate, tp_easyjoin_tool, sort1, anndata_inspect</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2122</td>
      <td>2495</td>
      <td>0.750000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Importing files from public atlases</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/EBI-retrieval/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-11-14</td>
      <td>2025-03-11</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>retrieve_scxa, sceasy_convert, regexColumn1, seurat_read10x, tp_grep_tool, anndata_ops, tp_find_and_replace, scanpy_read_10x, hca_matrix_downloader</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>585</td>
      <td>770</td>
      <td>3.600000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Single-cell ATAC-seq standard processing with SnapATAC2</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scatac-standard-processing-snapatac2/tutorial.html</td>
      <td></td>
      <td>Expression profile clustering, Read mapping, Generation, Gene expression profiling, Genome indexing, Differential gene expression analysis, Dimensionality reduction, Essential dynamics, Sequence alignment, Principal component visualisation</td>
      <td>2024-07-11</td>
      <td>2025-07-02</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>scanpy_inspect, snapatac2_peaks_and_motif, scanpy_plot, replace_column_with_key_value_file, Cut1, snapatac2_preprocessing, snapatac2_clustering, fasta_compute_length, anndata_manipulate, anndata_ops, scanpy_normalize, snapatac2_plotting, anndata_inspect, bwa_mem, scanpy_filter</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1601</td>
      <td>1932</td>
      <td>1.100000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Clustering 3K PBMCs with Scanpy</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-scanpy-pbmc3k/tutorial.html</td>
      <td></td>
      <td>Expression profile clustering, Principal component visualisation, Differential gene expression analysis</td>
      <td>2019-12-19</td>
      <td>2025-04-29</td>
      <td>16</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>scanpy_inspect, scanpy_cluster_reduce_dimension, scanpy_plot, anndata_import, datamash_ops, scanpy_remove_confounders, anndata_manipulate, scanpy_normalize, anndata_inspect, scanpy_filter</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.fr, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>3123801</td>
      <td>7976518</td>
      <td>4.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Combining single cell datasets after pre-processing</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-case_alevin-combine-datasets/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-09-08</td>
      <td>2025-11-18</td>
      <td>23</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>tp_replace_in_column, Cut1, anndata_manipulate, Paste1, anndata_ops, anndata_inspect</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>2540</td>
      <td>3559</td>
      <td>2.533333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Filter, plot, and explore single cell RNA-seq data with Seurat (R)</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-case_FilterPlotandExploreRStudio/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-10-02</td>
      <td>2025-03-11</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>3326</td>
      <td>3788</td>
      <td>0.966667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Understanding Barcodes</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-umis/tutorial.html</td>
      <td></td>
      <td>Sequencing quality control, Statistical calculation, Sequence composition calculation</td>
      <td>2019-02-20</td>
      <td>2025-02-14</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>seq_filter_by_id, fastqc, umi_tools_extract</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>Galaxy@AuBi, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>6418</td>
      <td>7984</td>
      <td>1.233333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Downstream Single-cell RNA analysis with RaceID</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-raceid/tutorial.html</td>
      <td></td>
      <td>Expression profile clustering, Molecular dynamics, Expression analysis</td>
      <td>2019-03-25</td>
      <td>2025-02-14</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tp_sed_tool, raceid_inspecttrajectory, raceid_filtnormconf, datamash_ops, raceid_trajectory, tp_head_tool, datamash_transpose, raceid_inspectclusters, raceid_clustering</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>1094</td>
      <td>1290</td>
      <td>1.250000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Comparing inferred cell compositions using MuSiC deconvolution</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/bulk-music-4-compare/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-01-20</td>
      <td>2025-02-13</td>
      <td>10</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>music_compare</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>290</td>
      <td>433</td>
      <td>1.350000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Filter, plot and explore single-cell RNA-seq data with Scanpy (Python)</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-case-jupyter_basic-pipeline/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-08-25</td>
      <td>2025-03-11</td>
      <td>13</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>3375</td>
      <td>3999</td>
      <td>1.483333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Filter, plot and explore single-cell RNA-seq data with Scanpy</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-case_basic-pipeline/tutorial.html</td>
      <td></td>
      <td>Expression profile clustering, Principal component visualisation, Differential gene expression analysis</td>
      <td>2021-03-24</td>
      <td>2025-04-23</td>
      <td>24</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>retrieve_scxa, scanpy_run_pca, scanpy_find_cluster, scanpy_filter_genes, interactive_tool_cellxgene_vip, scanpy_filter_cells, scanpy_filter, scanpy_plot_embed, scanpy_inspect, scanpy_run_umap, anndata_ops, scanpy_normalize, anndata_inspect, scanpy_compute_graph, scanpy_normalise_data, scanpy_cluster_reduce_dimension, scanpy_find_variable_genes, scanpy_plot, Cut1, anndata_manipulate, join1, scanpy_scale_data, scanpy_run_tsne, scanpy_find_markers</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>5513</td>
      <td>8668</td>
      <td>2.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Clustering 3K PBMCs with Seurat</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-seurat-pbmc3k/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-01-23</td>
      <td>2025-05-09</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>seurat_clustering, seurat_create, seurat_preprocessing, Cut1, csv_to_tabular, Filter1, seurat_data, seurat_plot, seurat_reduce_dimension</td>
      <td>MISSISSIPPI, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1367</td>
      <td>1646</td>
      <td>1.150000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Filter, plot, and explore single cell RNA-seq data with Seurat</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-case_FilterPlotandExplore_SeuratTools/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-04-09</td>
      <td>2025-03-11</td>
      <td>10</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>seurat_scale_data, seurat_normalise_data, retrieve_scxa, seurat_run_pca, seurat_find_markers, seurat_run_umap, seurat_plot, seurat_read10x, seurat_find_variable_genes, seurat_find_clusters, seurat_find_neighbours, seurat_filter_cells</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2040</td>
      <td>2479</td>
      <td>1.400000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Scanpy Parameter Iterator</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scanpy_parameter_iterator/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-07-19</td>
      <td>2025-02-13</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>scanpy_compute_graph, scanpy_plot_embed, scanpy_run_umap, scanpy_find_cluster, scanpy_parameter_iterator, scanpy_run_tsne</td>
      <td>UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>293</td>
      <td>327</td>
      <td>1.033333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Converting NCBI Data to the AnnData Format</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-ncbi-anndata/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-12-13</td>
      <td>2025-03-11</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>tp_replace_in_column, anndata_import, Cut1, anndata_manipulate, Paste1, anndata_ops, scanpy_filter_cells, unzip, anndata_inspect</td>
      <td>UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>465</td>
      <td>549</td>
      <td>1.583333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Evaluating Reference Data for Bulk RNA Deconvolution</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/bulk-deconvolution-evaluate/tutorial.html</td>
      <td></td>
      <td>Visualisation, Annotation</td>
      <td>2025-02-02</td>
      <td>2025-03-11</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>ggplot2_point, Remove beginning1, table_compute, tp_sed_tool, Paste1, tp_easyjoin_tool, music_construct_eset, collapse_dataset, annotatemyids, __DUPLICATE_FILE_TO_COLLECTION__, Count1, music_deconvolution, tp_awk_tool, Add_a_column1, tp_multijoin_tool, Cut1, bg_column_arrange_by_header, tp_cut_tool, join1, datamash_transpose, ggplot2_violin, collection_column_join</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>115</td>
      <td>138</td>
      <td>0.950000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Multi-sample batch correction with Harmony and SnapATAC2</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scatac-batch-correction-snapatac2/tutorial.html</td>
      <td></td>
      <td>Essential dynamics, Gene expression profiling, Dimensionality reduction</td>
      <td>2025-02-28</td>
      <td>2025-07-02</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>snapatac2_peaks_and_motif, snapatac2_preprocessing, Show beginning1, snapatac2_clustering, fasta_compute_length, anndata_manipulate, collection_element_identifiers, snapatac2_plotting, __EXTRACT_DATASET__, __FILTER_FROM_FILE__, anndata_inspect</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>510</td>
      <td>568</td>
      <td>0.833333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Plates, Batches, and Barcodes</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-plates-batches-barcodes/slides.html</td>
      <td></td>
      <td></td>
      <td>2019-02-16</td>
      <td>2025-02-13</td>
      <td>4</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>3115664</td>
      <td>7964938</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Inferring single cell trajectories with Scanpy (Python)</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-case_JUPYTER-trajectories/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-04-07</td>
      <td>2025-03-11</td>
      <td>20</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>4619</td>
      <td>5446</td>
      <td>1.266667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Inferring single cell trajectories with Scanpy</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-case_trajectories/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-12-08</td>
      <td>2025-03-11</td>
      <td>13</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>scanpy_compute_graph, scanpy_plot_embed, scanpy_plot_trajectory, scanpy_run_dpt, scanpy_find_cluster, anndata_manipulate, scanpy_run_paga, scanpy_run_fdg, scanpy_run_diffmap</td>
      <td></td>
      <td>UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>1320</td>
      <td>1659</td>
      <td>2.100000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Matrix Exchange Format to ESet | Creating a single-cell RNA-seq reference dataset for deconvolution</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/bulk-music-2-preparescref/tutorial.html</td>
      <td></td>
      <td>Annotation</td>
      <td>2023-01-20</td>
      <td>2024-02-13</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>scanpy_read_10x, regex1, table_compute, music_construct_eset, retrieve_scxa, annotatemyids, anndata_inspect, tp_sed_tool, music_manipulate_eset, Cut1, tp_cut_tool, add_line_to_file, join1, datamash_transpose, tp_easyjoin_tool, sort1, tp_awk_tool</td>
      <td></td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>744</td>
      <td>1242</td>
      <td>2.150000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Single-cell Formats and Resources</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-data-formats/slides.html</td>
      <td></td>
      <td></td>
      <td>2024-07-26</td>
      <td>2025-02-13</td>
      <td>4</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1082395</td>
      <td>2309710</td>
      <td>1.916667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Inferring single cell trajectories with Monocle3 (R)</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-case_monocle3-rstudio/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-04-12</td>
      <td>2025-03-11</td>
      <td>13</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>6530</td>
      <td>7410</td>
      <td>0.783333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Inferring single cell trajectories with Monocle3</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-case_monocle3-trajectories/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-09-30</td>
      <td>2025-04-23</td>
      <td>24</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>monocle3_create, monocle3_diffExp, monocle3_preprocess, monocle3_reduceDim, monocle3_partition, monocle3_topmarkers, Cut1, Filter1, regexColumn1, tp_cut_tool, join1, datamash_transpose, anndata_inspect, monocle3_learnGraph, monocle3_plotCells, monocle3_orderCells</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3960</td>
      <td>4900</td>
      <td>1.483333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Bulk matrix to ESet | Creating the bulk RNA-seq dataset for deconvolution</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/bulk-music-3-preparebulk/tutorial.html</td>
      <td></td>
      <td>Annotation</td>
      <td>2023-01-20</td>
      <td>2024-02-13</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>regex1, table_compute, music_construct_eset, annotatemyids, tp_sed_tool, music_manipulate_eset, column_remove_by_header, tp_cut_tool, tp_easyjoin_tool, sort1, tp_awk_tool</td>
      <td></td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>746</td>
      <td>1156</td>
      <td>1.783333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Trajectory analysis</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-trajectories/slides.html</td>
      <td></td>
      <td></td>
      <td>2022-09-30</td>
      <td>2025-02-13</td>
      <td>4</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>2160369</td>
      <td>4878623</td>
      <td>2.150000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Automated Cell Annotation</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-case-cell-annotation/slides.html</td>
      <td></td>
      <td></td>
      <td>2023-09-04</td>
      <td>2025-02-27</td>
      <td>7</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1557882</td>
      <td>3430615</td>
      <td>2.083333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Machine learning: classification and regression</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/classification_regression/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-03-07</td>
      <td>2025-04-08</td>
      <td>13</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>plotly_regression_performance_plots, plotly_ml_performance_plots, sklearn_ensemble, sklearn_svm_classifier</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>5709</td>
      <td>10248</td>
      <td>0.783333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Clustering in Machine Learning</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/clustering_machinelearning/tutorial.html</td>
      <td></td>
      <td>Visualisation</td>
      <td>2020-05-08</td>
      <td>2025-04-08</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>sklearn_numeric_clustering, ggplot2_point, csv_to_tabular</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>8535</td>
      <td>14111</td>
      <td>0.933333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Deep Learning (Part 3) - Convolutional neural networks (CNN)</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/CNN/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-04-19</td>
      <td>2025-05-09</td>
      <td>19</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>ml_visualization_ex, model_prediction, keras_train_and_eval, keras_model_builder, keras_model_config, sklearn_to_categorical</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2973796</td>
      <td>7433350</td>
      <td>3.200000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Basics of machine learning</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/machinelearning/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-11-05</td>
      <td>2025-05-13</td>
      <td>15</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>sklearn_svm_classifier</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.fr, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>3118473</td>
      <td>7968724</td>
      <td>5.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Prepare Data from CbioPortal for Flexynesis Integration</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/flexynesis_cbio_import/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-08-10</td>
      <td>2025-08-13</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>table_compute, tp_sort_header_tool, flexynesis_cbioportal_import, tp_cut_tool, flexynesis_utils, __EXTRACT_DATASET__</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>50</td>
      <td>51</td>
      <td>0.600000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Train and Test a Deep learning image classifier with Galaxy-Ludwig</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/galaxy-ludwig/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-10-28</td>
      <td>2024-12-11</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>ludwig_experiment</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>232</td>
      <td>294</td>
      <td>2.116667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>PAPAA PI3K_OG: PanCancer Aberrant Pathway Activity Analysis</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/aberrant_pi3k_pathway_analysis/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-05-06</td>
      <td>2023-11-09</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>pancancer_targene_cell_line_predictions, pancancer_targene_pharmacology, pancancer_compare_within_models, pancancer_targene_summary_figures, pancancer_map_mutation_class, pancancer_visualize_decisions, pancancer_pathway_count_heatmaps, pancancer_classifier, pancancer_apply_weights, pancancer_external_sample_status_prediction, pancancer_alternative_genes_pathwaymapper, pancancer_within_disease_analysis</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>801</td>
      <td>1242</td>
      <td>3.033333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Text-mining with the SimText toolset</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/text-mining_simtext/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-04-05</td>
      <td>2025-05-07</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>pmids_to_pubtator_matrix, interactive_tool_simtext_app, pubmed_by_queries</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>532</td>
      <td>729</td>
      <td>2.616667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Age prediction using machine learning</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/age-prediction-with-ml/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-01-25</td>
      <td>2025-04-08</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>plotly_regression_performance_plots, sklearn_searchcv, sklearn_build_pipeline, plotly_parallel_coordinates_plot, sklearn_ensemble</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>13251</td>
      <td>21358</td>
      <td>0.683333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Modeling Breast Cancer Subtypes with Flexynesis</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/flexynesis_classification/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-08-13</td>
      <td>2025-09-02</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>table_compute, flexynesis, tp_sort_header_tool, Show beginning1, tabpfn, tp_cut_tool, join1, datamash_transpose, flexynesis_plot, tp_easyjoin_tool, __EXTRACT_DATASET__, pick_value</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>85</td>
      <td>120</td>
      <td>1.900000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Regression in Machine Learning</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/regression_machinelearning/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-01-25</td>
      <td>2025-05-14</td>
      <td>19</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>plotly_regression_performance_plots, sklearn_generalized_linear, sklearn_searchcv, sklearn_build_pipeline, Remove beginning1, sklearn_ensemble</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>3118929</td>
      <td>7969063</td>
      <td>3.600000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Building the LORIS LLR6 PanCancer Model Using PyCaret</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/loris_model/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-12-13</td>
      <td>2025-05-08</td>
      <td>2</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>PyCaret_Model_Comparison</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>863156</td>
      <td>1769273</td>
      <td>3.666667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Deep Learning (Part 1) - Feedforward neural networks (FNN)</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/FNN/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-04-28</td>
      <td>2024-06-14</td>
      <td>12</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>plotly_regression_performance_plots, model_prediction, keras_train_and_eval, keras_model_builder, keras_model_config</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2978431</td>
      <td>7422463</td>
      <td>3.050000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Image classification in Galaxy with fruit 360 dataset</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/fruit_360/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-12-01</td>
      <td>2024-07-09</td>
      <td>8</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>ml_visualization_ex, model_prediction, keras_train_and_eval, keras_model_builder, tp_cut_tool, keras_model_config, sklearn_to_categorical</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2634428</td>
      <td>6336119</td>
      <td>2.950000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Introduction to deep learning</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/intro_deep_learning/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-03-26</td>
      <td>2025-04-08</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>ml_visualization_ex, model_prediction, keras_train_and_eval, keras_model_builder, keras_model_config</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2500</td>
      <td>3187</td>
      <td>1.633333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Identifing Survival Markers of Brain tumor with Flexynesis</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/flexynesis_survival/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-08-13</td>
      <td>2025-08-13</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Add_a_column1, flexynesis, tp_replace_in_column, datamash_ops, flexynesis_plot, __EXTRACT_DATASET__</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>72</td>
      <td>85</td>
      <td>0.583333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Classification in Machine Learning</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/classification_machinelearning/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-04-30</td>
      <td>2025-05-14</td>
      <td>18</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>sklearn_generalized_linear, sklearn_searchcv, sklearn_build_pipeline, Remove beginning1, sklearn_svm_classifier, sklearn_nn_classifier, plotly_ml_performance_plots, sklearn_ensemble</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>3118532</td>
      <td>7969106</td>
      <td>4.483333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Deep Learning (Part 2) - Recurrent neural networks (RNN)</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/RNN/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-02-23</td>
      <td>2025-05-10</td>
      <td>15</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>ml_visualization_ex, model_prediction, keras_train_and_eval, keras_model_builder, keras_model_config</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>3048820</td>
      <td>7714682</td>
      <td>3.116667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Unsupervised Analysis of Bone Marrow Cells with Flexynesis</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/flexynesis_unsupervised/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-08-10</td>
      <td>2025-08-13</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>__EXTRACT_DATASET__, flexynesis, flexynesis_utils, flexynesis_plot</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>39</td>
      <td>44</td>
      <td>3.166667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>A Docker-based interactive Jupyterlab powered by GPU for artificial intelligence in Galaxy</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/gpu_jupyter_lab/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-04-06</td>
      <td>2025-04-08</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>interactive_tool_ml_jupyter_notebook, Cut1, Filter1</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1219</td>
      <td>1541</td>
      <td>1.450000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Interval-Wise Testing for omics data</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/iwtomics/tutorial.html</td>
      <td></td>
      <td>Differentially-methylated region identification, Genome annotation, Peak calling, Differential gene expression analysis, Comparison</td>
      <td>2018-06-14</td>
      <td>2023-11-09</td>
      <td>17</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>iwtomics_loadandplot, iwtomics_plotwithscale, iwtomics_testandplot</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>375</td>
      <td>497</td>
      <td>2.850000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Fine tune large protein model (ProtTrans) using HuggingFace</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/fine_tuning_protTrans/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-06-17</td>
      <td>2025-04-08</td>
      <td>4</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>interactive_tool_ml_jupyter_notebook</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1140884</td>
      <td>2451609</td>
      <td>3.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Supervised Learning with Hyperdimensional Computing</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/hyperdimensional_computing/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-04-28</td>
      <td>2023-11-09</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>chopin2</td>
      <td>UseGalaxy.org (Main)</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>448</td>
      <td>555</td>
      <td>0.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Introduction to Machine Learning using R</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/intro-to-ml-with-r/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-05-21</td>
      <td>2024-10-15</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>interactive_tool_rstudio</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>4172</td>
      <td>6196</td>
      <td>1.083333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Synthetic Biology</td>
      <td>Evaluating and ranking a set of pathways based on multiple metrics</td>
      <td>https://training.galaxyproject.org//topics/synthetic-biology/tutorials/pathway_analysis/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-09-26</td>
      <td>2024-06-14</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>rpthermo, rpscore, rpfba, rpranker</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>655</td>
      <td>968</td>
      <td>2.933333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Synthetic Biology</td>
      <td>Designing plasmids encoding predicted pathways by using the BASIC assembly method</td>
      <td>https://training.galaxyproject.org//topics/synthetic-biology/tutorials/basic_assembly_analysis/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-05-12</td>
      <td>2024-06-14</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>dnabot, selenzy-wrapper, rpbasicdesign</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1046</td>
      <td>1612</td>
      <td>2.616667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Synthetic Biology</td>
      <td>Generating theoretical possible pathways for the production of Lycopene in E.Coli using Retrosynthesis tools</td>
      <td>https://training.galaxyproject.org//topics/synthetic-biology/tutorials/retrosynthesis_analysis/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-06-23</td>
      <td>2024-06-14</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>rrparser, retropath2, rpextractsink, get_sbml_model, rpcompletion, rp2paths</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>598</td>
      <td>866</td>
      <td>2.416667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Teaching and Hosting Galaxy training</td>
      <td>Overview of the Galaxy Training Material for Instructors</td>
      <td>https://training.galaxyproject.org//topics/teaching/tutorials/gtn-for-instructors/slides.html</td>
      <td></td>
      <td></td>
      <td>2022-09-23</td>
      <td>2024-08-08</td>
      <td>3</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2170938</td>
      <td>4905665</td>
      <td>2.150000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Teaching and Hosting Galaxy training</td>
      <td>Workshop Kickoff</td>
      <td>https://training.galaxyproject.org//topics/teaching/tutorials/workshop-intro/slides.html</td>
      <td></td>
      <td></td>
      <td>2019-06-04</td>
      <td>2024-02-20</td>
      <td>5</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3115664</td>
      <td>7964949</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Teaching and Hosting Galaxy training</td>
      <td>Teaching experiences</td>
      <td>https://training.galaxyproject.org//topics/teaching/tutorials/experiences/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-09-23</td>
      <td>2022-09-23</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>296</td>
      <td>519</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Teaching and Hosting Galaxy training</td>
      <td>Teaching online</td>
      <td>https://training.galaxyproject.org//topics/teaching/tutorials/teaching-online/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-09-23</td>
      <td>2023-10-23</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>25</td>
      <td>25</td>
      <td>0.083333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Teaching and Hosting Galaxy training</td>
      <td>Galaxy Admin Training</td>
      <td>https://training.galaxyproject.org//topics/teaching/tutorials/galaxy-admin-training/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-11-04</td>
      <td>2024-09-17</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>162</td>
      <td>175</td>
      <td>2.416667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Teaching and Hosting Galaxy training</td>
      <td>Hybrid training</td>
      <td>https://training.galaxyproject.org//topics/teaching/tutorials/hybrid-training/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-09-23</td>
      <td>2024-06-25</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>29</td>
      <td>31</td>
      <td>0.016667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Teaching and Hosting Galaxy training</td>
      <td>Organizing a workshop</td>
      <td>https://training.galaxyproject.org//topics/teaching/tutorials/organize-workshop/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-11-15</td>
      <td>2023-11-09</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>271</td>
      <td>294</td>
      <td>0.933333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Teaching and Hosting Galaxy training</td>
      <td>Asynchronous training</td>
      <td>https://training.galaxyproject.org//topics/teaching/tutorials/gallantries-async/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-11-02</td>
      <td>2023-11-02</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>13</td>
      <td>13</td>
      <td>0.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Teaching and Hosting Galaxy training</td>
      <td>Training Infrastructure as a Service</td>
      <td>https://training.galaxyproject.org//topics/teaching/tutorials/setup-tiaas-for-training/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-11-15</td>
      <td>2023-12-20</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>232</td>
      <td>424</td>
      <td>5.883333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Teaching and Hosting Galaxy training</td>
      <td>Running a workshop as instructor</td>
      <td>https://training.galaxyproject.org//topics/teaching/tutorials/running-workshop/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-11-15</td>
      <td>2025-04-08</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>171</td>
      <td>300</td>
      <td>1.066667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Teaching and Hosting Galaxy training</td>
      <td>Set up a Galaxy for Training</td>
      <td>https://training.galaxyproject.org//topics/teaching/tutorials/setup-galaxy-for-training/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-06-04</td>
      <td>2023-11-09</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>329</td>
      <td>376</td>
      <td>3.966667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Teaching and Hosting Galaxy training</td>
      <td>Course Builder</td>
      <td>https://training.galaxyproject.org//topics/teaching/tutorials/gallantries-coursebuilder/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-11-02</td>
      <td>2023-11-02</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>147</td>
      <td>170</td>
      <td>1.266667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Teaching and Hosting Galaxy training</td>
      <td>Live Coding is a Skill</td>
      <td>https://training.galaxyproject.org//topics/teaching/tutorials/live-coding/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-09-23</td>
      <td>2023-11-09</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>89</td>
      <td>93</td>
      <td>0.266667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>RNA-RNA interactome data analysis</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/rna-interactome/tutorial.html</td>
      <td></td>
      <td>Read pre-processing, Primer removal, Sequence trimming, Sequencing quality control, Statistical calculation, Sequence composition calculation</td>
      <td>2020-03-23</td>
      <td>2023-11-09</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>chira_extract, chira_merge, chira_map, query_tabular, chira_quantify, fastqc, chira_collapse, cutadapt</td>
      <td>UseGalaxy.be, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>1502</td>
      <td>2259</td>
      <td>2.750000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>CLIP-Seq data analysis from pre-processing to motif detection</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/clipseq/tutorial.html</td>
      <td></td>
      <td>Read pre-processing, Sequence trimming, Sequence alignment, Primer removal, Mapping, Sequencing quality control, Statistical calculation, Sequence composition calculation</td>
      <td>2018-08-17</td>
      <td>2023-11-09</td>
      <td>23</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>deeptools_multi_bam_summary, rcas, peakachu, bctools_extract_alignment_ends, bedtools_slopbed, wig_to_bigWig, umi_tools_extract, meme_chip, bedtools_genomecoveragebed_bedgraph, __SORTLIST__, __MERGE_COLLECTION__, sort1, umi_tools_dedup, tp_awk_tool, rna_star, deeptools_plot_correlation, deeptools_plot_fingerprint, fastqc, cutadapt, Extract genomic DNA 1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>8375</td>
      <td>16408</td>
      <td>1.866667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>Genome-wide alternative splicing analysis</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.html</td>
      <td></td>
      <td>Sequence contamination filtering, Validation, RNA-Seq analysis, Sequence comparison, Sequence alignment, Data handling, Sequence analysis, Sequence annotation, Statistical calculation, Sequence composition calculation, Protein sequence analysis, Sequence assembly validation, Visualisation, Sequencing quality control, De-novo assembly, Formatting, Transcriptome assembly</td>
      <td>2023-05-17</td>
      <td>2025-03-28</td>
      <td>11</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>Remove beginning1, isoformswitchanalyzer, gene2exon1, tp_grep_tool, cpat, fastp, tp_uniq_tool, rseqc_geneBody_coverage, rna_quast, rseqc_junction_annotation, rseqc_infer_experiment, rseqc_junction_saturation, pfamscan, __FILTER_FROM_FILE__, multiqc, pygenomeTracks, gffread, gtftobed12, addValue, rseqc_inner_distance, __FLATTEN__, Filter1, tp_cat, gffcompare, sort1, tp_awk_tool, rna_star, Add_a_column1, stringtie, rseqc_read_distribution, cat1, Cut1, fastqc, collection_element_identifiers, stringtie_merge</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1762020</td>
      <td>3871154</td>
      <td>3.950000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>3: RNA-seq genes to pathways</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/rna-seq-genes-to-pathways/tutorial.html</td>
      <td></td>
      <td>Gene set testing, Gene-set enrichment analysis, Gene functional annotation</td>
      <td>2018-12-31</td>
      <td>2023-11-09</td>
      <td>13</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Add_a_column1, fgsea, tp_sort_header_tool, goseq, Cut1, tp_cut_tool, join1, egsea</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>26622</td>
      <td>42599</td>
      <td>1.950000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>De novo transcriptome assembly, annotation, and differential expression analysis</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/full-de-novo/tutorial.html</td>
      <td></td>
      <td>de Novo sequencing, Coding region prediction, Sequencing quality control, De-novo assembly, Validation, Gene functional annotation, Transcriptome assembly, Statistical calculation, Sequence composition calculation</td>
      <td>2020-02-13</td>
      <td>2023-11-09</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>trinity_contig_exn50_statistic, trinity_align_and_estimate_abundance, trinity_filter_low_expr_transcripts, trinity_samples_qccheck, trinity_run_de_analysis, trimmomatic, trinity, fastqc, trinotate, multiqc, trinity_define_clusters_by_cutting_tree, transdecoder, trinity_analyze_diff_expr, trinity_abundance_estimates_to_matrix, describe_samples</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>14798</td>
      <td>26663</td>
      <td>2.433333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>Visualization of RNA-Seq results with heatmap2</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/rna-seq-viz-with-heatmap2/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-12-31</td>
      <td>2025-11-24</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>ggplot2_heatmap2, Show tail1, tp_sort_header_tool, cat1, Cut1, Filter1, Show beginning1, join1, datamash_transpose</td>
      <td>GalaxyTrakr, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>78178</td>
      <td>128401</td>
      <td>1.366667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>RNA-Seq data analysis, clustering and visualisation tutorial</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/rna-seq-analysis-clustering-viz/tutorial.html</td>
      <td></td>
      <td>RNA-Seq analysis, Differential gene expression analysis, Gene functional annotation</td>
      <td>2025-05-28</td>
      <td>2025-06-16</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Add_a_column1, ggplot2_heatmap2, length_and_gc_content, table_compute, tp_sort_header_tool, cat1, goseq, Cut1, Filter1, Show beginning1, deg_annotate, deseq2, volcanoplot, Multivariate, Grouping1, tp_easyjoin_tool</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>1137</td>
      <td>1470</td>
      <td>1.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>1: RNA-Seq reads to counts</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/rna-seq-reads-to-counts/tutorial.html</td>
      <td></td>
      <td>Read summarisation, Sequencing quality control, Sequence trimming, Formatting, RNA-Seq quantification, Validation, Sequence alignment, Data handling, Primer removal, Read pre-processing, Statistical calculation, Sequence composition calculation</td>
      <td>2018-09-23</td>
      <td>2024-08-09</td>
      <td>33</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>samtools_idxstats, hisat2, rseqc_geneBody_coverage, rseqc_read_distribution, picard_MarkDuplicates, rseqc_infer_experiment, fastqc, featurecounts, cutadapt, multiqc, collection_column_join</td>
      <td>UseGalaxy.eu</td>
      <td>MISSISSIPPI, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>79541</td>
      <td>143917</td>
      <td>2.600000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>Small Non-coding RNA Clustering using BlockClust</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/small_ncrna_clustering/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-12-14</td>
      <td>2023-11-09</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>samtools_sort, blockbuster, blockclust, tp_sort_header_tool</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>1046</td>
      <td>1419</td>
      <td>2.916667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>Differential abundance testing of small RNAs</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/srna/tutorial.html</td>
      <td></td>
      <td>Sequence trimming, Sequencing quality control, Differential gene expression analysis, Sequence composition calculation, RNA-Seq quantification, RNA-Seq analysis, Sequence alignment, Primer removal, Statistical calculation, Read pre-processing, Gene expression analysis</td>
      <td>2017-06-29</td>
      <td>2023-11-09</td>
      <td>26</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>hisat2, bedtools_bamtofastq, deseq2, Filter1, trim_galore, fastqc, samtool_filter2, fastq_manipulation, salmon</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>7016</td>
      <td>12254</td>
      <td>2.083333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>Network analysis with Heinz</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/network-analysis-with-heinz/tutorial.html</td>
      <td></td>
      <td>RNA-Seq analysis, Differential gene expression analysis, Pathway or network analysis</td>
      <td>2018-10-11</td>
      <td>2024-01-15</td>
      <td>16</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>heinz_scoring, heinz, heinz_bum, heinz_visualization, Cut1, deseq2</td>
      <td>UseGalaxy.be, UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>3117720</td>
      <td>7968143</td>
      <td>5.550000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>2: RNA-seq counts to genes</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/rna-seq-counts-to-genes/tutorial.html</td>
      <td></td>
      <td>RNA-Seq analysis, Annotation</td>
      <td>2018-12-31</td>
      <td>2024-04-10</td>
      <td>21</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>annotatemyids, limma_voom, tp_cut_tool, mergeCols1, tp_replace_in_line</td>
      <td>UseGalaxy.be, UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>44358</td>
      <td>74799</td>
      <td>2.283333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>GO Enrichment Analysis</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/goenrichment/tutorial.html</td>
      <td></td>
      <td>Gene-set enrichment analysis</td>
      <td>2019-01-23</td>
      <td>2024-03-05</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>goenrichment, goslimmer, Filter1</td>
      <td>Galaxy@AuBi, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>23326</td>
      <td>37765</td>
      <td>1.333333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>Visualization of RNA-Seq results with Volcano Plot</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/rna-seq-viz-with-volcanoplot/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-12-31</td>
      <td>2025-04-17</td>
      <td>16</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>volcanoplot</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>Galaxy@AuBi, GalaxyTrakr, MISSISSIPPI, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>154969</td>
      <td>253154</td>
      <td>1.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>Introduction to Transcriptomics</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/introduction/slides.html</td>
      <td></td>
      <td></td>
      <td>2016-10-05</td>
      <td>2025-04-08</td>
      <td>3</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>3115664</td>
      <td>7964953</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>Visualization of RNA-Seq results with CummeRbund</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/rna-seq-viz-with-cummerbund/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2017-10-16</td>
      <td>2023-11-09</td>
      <td>16</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>3118271</td>
      <td>7968862</td>
      <td>4.066667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>RNA-Seq analysis with AskOmics Interactive Tool</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/rna-seq-analysis-with-askomics-it/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-07-09</td>
      <td>2023-11-09</td>
      <td>13</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>3116697</td>
      <td>7966266</td>
      <td>5.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>RNA Seq Counts to Viz in R</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/rna-seq-counts-to-viz-in-r/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-10-08</td>
      <td>2024-10-15</td>
      <td>21</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>interactive_tool_rstudio</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>4294</td>
      <td>5498</td>
      <td>1.950000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>Reference-based RNAseq data analysis (long)</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/rb-rnaseq/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2017-02-22</td>
      <td>2023-02-17</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>22146</td>
      <td>34652</td>
      <td>2.183333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>RNA-seq Alignment with STAR</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/rna-seq-bash-star-align/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-05-15</td>
      <td>2023-11-09</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>10968</td>
      <td>13429</td>
      <td>1.450000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>Visualization of RNA-Seq results with Volcano Plot in R</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/rna-seq-viz-with-volcanoplot-r/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-06-14</td>
      <td>2024-06-14</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>volcanoplot, interactive_tool_rstudio</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>31087</td>
      <td>42419</td>
      <td>1.683333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>Reference-based RNA-Seq data analysis</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/ref-based/tutorial.html</td>
      <td></td>
      <td>Read mapping, Pathway visualisation, Pathway or network visualisation, Differential gene expression analysis, RNA-Seq quantification, Gene functional annotation, Validation, RNA-Seq analysis, Structure visualisation, Sequence alignment, Data handling, Read pre-processing, Enrichment analysis, Visualisation, Read summarisation, Sequencing quality control, Exonic splicing enhancer prediction, Formatting, Sequence trimming, Pathway or network analysis, Genome visualisation, Primer removal</td>
      <td>2016-10-05</td>
      <td>2025-12-11</td>
      <td>107</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>falco, Remove beginning1, tp_sort_header_tool, pathview, deseq2, ChangeCase, tp_grep_tool, featurecounts, samtools_idxstats, length_and_gc_content, table_compute, rseqc_geneBody_coverage, picard_MarkDuplicates, dexseq, goseq, rseqc_infer_experiment, __EXTRACT_DATASET__, __FILTER_FROM_FILE__, multiqc, pygenomeTracks, gtftobed12, __FLATTEN__, Filter1, __MERGE_COLLECTION__, tp_cat, Grouping1, rna_star, samtools_view, Add_a_column1, ggplot2_heatmap2, rseqc_read_distribution, cat1, Cut1, deg_annotate, __TAG_FROM_FILE__, dexseq_count, join1, collection_element_identifiers, cutadapt, jbrowse2, tp_replace_in_line</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>190049</td>
      <td>354459</td>
      <td>2.783333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>Pathway analysis with the MINERVA Platform</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/minerva-pathways/tutorial.html</td>
      <td></td>
      <td>Read summarisation, Sequencing quality control, Sequence trimming, Validation, RNA-Seq quantification, Gene functional annotation, Annotation, RNA-Seq analysis, Sequence alignment, Data handling, Primer removal, Read pre-processing, Statistical calculation, Sequence composition calculation</td>
      <td>2024-03-28</td>
      <td>2024-07-26</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tp_sorted_uniq, hisat2, Grep1, tp_sort_header_tool, featurecounts, rseqc_geneBody_coverage, limma_voom, goseq, __EXTRACT_DATASET__, multiqc, annotatemyids, Add_a_column1, fasterq_dump, rseqc_read_distribution, Cut1, join1, fastqc, cutadapt, tp_replace_in_line, collection_column_join</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>652</td>
      <td>769</td>
      <td>1.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>De novo transcriptome reconstruction with RNA-Seq</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/de-novo/tutorial.html</td>
      <td></td>
      <td>Sequence annotation, Read summarisation, Sequencing quality control, Differential gene expression analysis, RNA-Seq quantification, RNA-Seq analysis, Sequence alignment, Transcriptome assembly, Statistical calculation, Sequence composition calculation</td>
      <td>2017-02-19</td>
      <td>2024-06-14</td>
      <td>36</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>hisat2, stringtie, deseq2, Filter1, trimmomatic, featurecounts, fastqc, gffcompare, deeptools_bam_coverage, stringtie_merge</td>
      <td>Galaxy@AuBi, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>16935</td>
      <td>29053</td>
      <td>2.650000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Introduction to Variant analysis</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/introduction/slides.html</td>
      <td></td>
      <td></td>
      <td>2016-08-19</td>
      <td>2023-03-05</td>
      <td>2</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>3115664</td>
      <td>7964954</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Calling very rare variants</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/dunovo/tutorial.html</td>
      <td></td>
      <td>Genome indexing, Generation, Read mapping, Sequence alignment, Sequencing quality control, Statistical calculation, Sequence composition calculation, Variant calling</td>
      <td>2017-02-22</td>
      <td>2024-12-09</td>
      <td>21</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>bamleftalign, sequence_content_trimmer, correct_barcodes, naive_variant_caller, align_families, Filter1, fastqc, allele_counts_1, dunovo, make_families, bwa_mem</td>
      <td>UseGalaxy.org (Main)</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2055</td>
      <td>3172</td>
      <td>1.733333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Mapping and molecular identification of phenotype-causing mutations</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/mapping-by-sequencing/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-03-07</td>
      <td>2023-11-09</td>
      <td>16</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>mimodd_vcf_filter, mimodd_varcall, mimodd_varextract, tp_find_and_replace, mimodd_map, mimodd_varreport, snpEff_download, snpEff</td>
      <td>UseGalaxy.be, UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>4445</td>
      <td>6510</td>
      <td>2.683333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Trio Analysis using Synthetic Datasets from RD-Connect GPAP</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/trio-analysis/tutorial.html</td>
      <td></td>
      <td>Sequence analysis, Genetic variation analysis</td>
      <td>2022-05-19</td>
      <td>2024-09-17</td>
      <td>13</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>CONVERTER_gz_to_uncompressed, Filter1, regexColumn1, gemini_load, bcftools_norm, tp_grep_tool, gemini_inheritance, CONVERTER_uncompressed_to_gz, bcftools_merge, pyega3, gene_iobio_display_generation_iframe, snpEff</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>1978</td>
      <td>3472</td>
      <td>3.550000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Identification of somatic and germline variants from tumor and normal sample pairs</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/somatic-variants/tutorial.html</td>
      <td></td>
      <td>Visualisation, Read mapping, Generation, Sequencing quality control, Genome indexing, Sequence alignment analysis, Validation, Sequence alignment, Data handling, Statistical calculation, Sequence composition calculation, Variant calling</td>
      <td>2019-03-13</td>
      <td>2025-03-20</td>
      <td>16</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>bamleftalign, falco, samtools_calmd, gemini_query, gemini_annotate, varscan_somatic, gemini_load, trimmomatic, bg_column_arrange_by_header, fastqc, tp_easyjoin_tool, bamFilter, samtools_rmdup, multiqc, bwa_mem, snpEff</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>14639</td>
      <td>26270</td>
      <td>2.500000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Calling variants in diploid systems</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/dip/tutorial.html</td>
      <td></td>
      <td>Sequence analysis, Statistical calculation, Variant calling, Genetic variation analysis</td>
      <td>2017-02-20</td>
      <td>2025-04-28</td>
      <td>17</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>vcfallelicprimitives, gemini_db_info, freebayes, gemini_load, gemini_query, snpEff</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr</td>
      <td>UseGalaxy.be, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>15772</td>
      <td>27050</td>
      <td>3.233333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Somatic Variant Discovery from WES Data Using Control-FREEC</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/somatic-variant-discovery/tutorial.html</td>
      <td></td>
      <td>Visualisation, Read mapping, Generation, Sequencing quality control, Genome indexing, Validation, Genome alignment, Sequence alignment, Copy number estimation, Sequence visualisation, Statistical calculation, Sequence composition calculation, Variant calling</td>
      <td>2022-10-05</td>
      <td>2025-03-20</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>bamleftalign, falco, __RELABEL_FROM_FILE__, Grep1, samtools_calmd, circos, bwa_mem, trimmomatic, __BUILD_LIST__, fastqc, __EXTRACT_DATASET__, samtools_rmdup, control_freec, multiqc, tp_replace_in_line, samtools_view, tp_text_file_with_recurring_lines</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>1901</td>
      <td>2428</td>
      <td>2.050000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Calling variants in non-diploid systems</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/non-dip/tutorial.html</td>
      <td>Genomics, Sequence assembly, DNA polymorphism, Microbiology, Sequence analysis, Genetic variation</td>
      <td>Generation, Read mapping, Sequencing quality control, Genome indexing, Formatting, Sequence alignment analysis, Sequence alignment, Data handling, Statistical calculation, Sequence composition calculation, Variant calling</td>
      <td>2017-02-16</td>
      <td>2024-03-15</td>
      <td>23</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>bamleftalign, picard_MergeSamFiles, picard_MarkDuplicates, Cut1, freebayes, vcffilter2, fastqc, bamFilter, vcf2tsv, bwa_mem</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>8801</td>
      <td>15695</td>
      <td>2.100000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Exome sequencing data analysis for diagnosing a genetic disease</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/exome-seq/tutorial.html</td>
      <td></td>
      <td>Generation, Read mapping, Sequencing quality control, Genome indexing, Validation, Sequence alignment, Sequence analysis, Statistical calculation, Sequence composition calculation, Variant calling, Genetic variation analysis</td>
      <td>2016-08-19</td>
      <td>2025-05-11</td>
      <td>29</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>freebayes, gemini_load, trimmomatic, bcftools_norm, gemini_inheritance, fastqc, samtools_rmdup, multiqc, snpEff_download, bwa_mem, samtools_view, snpEff</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.be, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>35973</td>
      <td>64954</td>
      <td>2.716667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Deciphering Virus Populations - Single Nucleotide Variants (SNVs) and Specificities in Baculovirus Isolates</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/baculovirus-isolate-variation/tutorial.html</td>
      <td></td>
      <td>Visualisation, Read mapping, Generation, Sequence trimming, Genome indexing, Sequence alignment, Data handling, Primer removal, Read pre-processing</td>
      <td>2025-02-24</td>
      <td>2025-02-24</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>collapse_dataset, tp_replace_in_column, bcftools_mpileup, ggplot2_point, bcftools_call, Filter1, ncbi_acc_download, trim_galore, bwa_mem, vcf2tsv, fastq_dump, tp_awk_tool, tp_replace_in_line</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>278</td>
      <td>376</td>
      <td>1.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Querying the University of Bradford GDC Beacon Database for Copy Number Variants (CNVs)</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/beacon_cnv_query/tutorial.html</td>
      <td></td>
      <td>Database search, Service discovery, Genetic variation analysis</td>
      <td>2024-08-06</td>
      <td>2024-09-27</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>beacon2_cnv</td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>198</td>
      <td>235</td>
      <td>2.150000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Working with Beacon V2: A Comprehensive Guide to Creating, Uploading, and Searching for Variants with Beacons</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/beaconise_1000hg/tutorial.html</td>
      <td></td>
      <td>Database search, Service discovery, Genetic variation analysis</td>
      <td>2024-08-06</td>
      <td>2024-09-27</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>beacon2_import, cnv_vcf2json, beacon2_cnv, cnv_phenopacket</td>
      <td></td>
      <td>UseGalaxy.eu</td>
      <td>0</td>
      <td></td>
      <td>340</td>
      <td>380</td>
      <td>0.616667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Visualisation</td>
      <td>Visualisation with Circos</td>
      <td>https://training.galaxyproject.org//topics/visualisation/tutorials/circos/tutorial.html</td>
      <td></td>
      <td>Sequence visualisation</td>
      <td>2020-01-10</td>
      <td>2024-10-15</td>
      <td>22</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>upload1, Grep1, Remove beginning1, circos, Cut1, random_lines1</td>
      <td>Galaxy@AuBi, MISSISSIPPI, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>GalaxyTrakr, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>3146246</td>
      <td>8013637</td>
      <td>3.966667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Visualisation</td>
      <td>Ploting a Microbial Genome with Circos</td>
      <td>https://training.galaxyproject.org//topics/visualisation/tutorials/circos-microbial/tutorial.html</td>
      <td></td>
      <td>Sequence visualisation</td>
      <td>2023-11-08</td>
      <td>2024-09-17</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>gff2bed1, circos_wiggle_to_scatter, circos_interval_to_text, Cut1, circos, deeptools_bam_coverage, circos_gc_skew, circos_interval_to_tile</td>
      <td>UseGalaxy.cz, UseGalaxy.fr</td>
      <td>Galaxy@AuBi, MISSISSIPPI, UseGalaxy.be, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>1480</td>
      <td>1868</td>
      <td>1.883333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Visualisation</td>
      <td>Friends Don't Let Friends Make Bad Graphs</td>
      <td>https://training.galaxyproject.org//topics/visualisation/tutorials/bad-graphs/slides.html</td>
      <td></td>
      <td></td>
      <td>2023-11-20</td>
      <td>2023-11-20</td>
      <td>2</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1473684</td>
      <td>3246655</td>
      <td>2.066667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Visualisation</td>
      <td>Visualisations in Galaxy</td>
      <td>https://training.galaxyproject.org//topics/visualisation/tutorials/introduction/slides.html</td>
      <td></td>
      <td></td>
      <td>2020-01-10</td>
      <td>2023-01-12</td>
      <td>1</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>3115664</td>
      <td>7964957</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
  </tbody>
</table>

</div>