<script src="https://code.jquery.com/jquery-3.7.0.js"></script>
<script src="https://cdn.datatables.net/1.13.6/js/jquery.dataTables.min.js"></script>
<script src="https://cdn.datatables.net/searchbuilder/1.6.0/js/dataTables.searchBuilder.min.js"></script>
<script src="https://cdn.datatables.net/datetime/1.5.1/js/dataTables.dateTime.min.js"></script>

<script src="https://cdn.datatables.net/responsive/2.5.0/js/dataTables.responsive.min.js"></script>

<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/responsive/2.2.3/css/responsive.dataTables.min.css">

<link rel="stylesheet" href="https://cdn.datatables.net/1.13.6/css/jquery.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/searchbuilder/1.6.0/css/searchBuilder.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/datetime/1.5.1/css/dataTables.dateTime.min.css">

<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" />
<script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>

<style type="text/css">
  .tooltip {
    font-family: Georgia;
    font-size: 18px;
  }
  .tooltip .tooltip-inner {
    background-color: #00FFFF;
    color: #000000;
    min-width: 400px;
  }
  </style>

<script>
    $(document).ready(function() {
    $('#dataframe').DataTable( {
        "sScrollX": "100%",
        "dom": "Qlfrtip",

        responsive: {
            details: {
                type: 'column'
            }
        },
        "bAutoWidth": false,
        columnDefs: [ {
            className: 'control',
            orderable: false,
            targets:   [ 0 ],
            width: 100,
        },
        { type: 'natural', targets: 1 } // allow normal sorting
        ],

    });
});
</script>

<div class="display nowrap" style="width:80%" width="100%">

<table class="dataframe display nowrap" id="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th>Expand</th>
      <th>Galaxy wrapper id</th>
      <th>Galaxy tool ids</th>
      <th>Description</th>
      <th>bio.tool id</th>
      <th>bio.tool ids</th>
      <th>biii</th>
      <th>bio.tool name</th>
      <th>bio.tool description</th>
      <th>EDAM operation</th>
      <th>EDAM topic</th>
      <th>Status</th>
      <th>Source</th>
      <th>ToolShed categories</th>
      <th>ToolShed id</th>
      <th>Galaxy wrapper owner</th>
      <th>Galaxy wrapper source</th>
      <th>Galaxy wrapper parsed folder</th>
      <th>Galaxy wrapper version</th>
      <th>Conda id</th>
      <th>Conda version</th>
      <th>EDAM operation (no superclasses)</th>
      <th>EDAM topic (no superclasses)</th>
      <th>Available on UseGalaxy.org (Main)</th>
      <th>Available on UseGalaxy.org.au</th>
      <th>Available on UseGalaxy.eu</th>
      <th>Available on UseGalaxy.fr</th>
      <th>Tools available on UseGalaxy.org (Main)</th>
      <th>Tools available on UseGalaxy.org.au</th>
      <th>Tools available on UseGalaxy.eu</th>
      <th>Tools available on UseGalaxy.fr</th>
      <th>Tools available on ANASTASIA</th>
      <th>Tools available on APOSTL</th>
      <th>Tools available on ARGs-OAP</th>
      <th>Tools available on BF2I-MAP</th>
      <th>Tools available on BioBix</th>
      <th>Tools available on CIRM-CFBP</th>
      <th>Tools available on Center for Phage Technology (CPT)</th>
      <th>Tools available on ChemFlow</th>
      <th>Tools available on Coloc-stats</th>
      <th>Tools available on CorGAT</th>
      <th>Tools available on CropGalaxy</th>
      <th>Tools available on Dintor</th>
      <th>Tools available on FreeBioinfo</th>
      <th>Tools available on GASLINI</th>
      <th>Tools available on Galaxy@AuBi</th>
      <th>Tools available on Galaxy@Pasteur</th>
      <th>Tools available on GalaxyTrakr</th>
      <th>Tools available on Genomic Hyperbrowser</th>
      <th>Tools available on GigaGalaxy</th>
      <th>Tools available on HyPhy HIV NGS Tools</th>
      <th>Tools available on IPK Galaxy Blast Suite</th>
      <th>Tools available on ImmPort Galaxy</th>
      <th>Tools available on InteractoMIX</th>
      <th>Tools available on MISSISSIPPI</th>
      <th>Tools available on Mandoiu Lab</th>
      <th>Tools available on MiModD NacreousMap</th>
      <th>Tools available on Oqtans</th>
      <th>Tools available on Palfinder</th>
      <th>Tools available on PepSimili</th>
      <th>Tools available on PhagePromotor</th>
      <th>Tools available on Protologger</th>
      <th>Tools available on UseGalaxy.be</th>
      <th>Tools available on UseGalaxy.cz</th>
      <th>Tools available on UseGalaxy.no</th>
      <th>No. of tool users (2022-2023) (usegalaxy.eu)</th>
      <th>Total tool usage (usegalaxy.eu)</th>
      <th>Deprecated</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td></td>
      <td>aegean</td>
      <td>aegean_canongff3, aegean_gaeval, aegean_locuspocus, aegean_parseval</td>
      <td>AEGeAn toolkit wrappers</td>
      <td>gaeval</td>
      <td>gaeval</td>
      <td></td>
      <td>GAEVAL</td>
      <td>Gene Annotation EVAluation.</td>
      <td>Sequence annotation</td>
      <td>Sequence analysis, Gene structure</td>
      <td>Up-to-date</td>
      <td>https://github.com/BrendelGroup/AEGeAn</td>
      <td>Transcriptomics, Sequence Analysis</td>
      <td>aegean</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/aegean</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/aegean</td>
      <td>0.16.0</td>
      <td>aegean</td>
      <td>0.16.0</td>
      <td>Sequence annotation</td>
      <td>Sequence analysis, Gene structure</td>
      <td>1</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>1</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>47</td>
      <td>168</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>anndata</td>
      <td>anndata_export, anndata_import, anndata_inspect, anndata_manipulate, modify_loom</td>
      <td>Import, Export, Inspect and Manipulate Anndata and Loom objects</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://anndata.readthedocs.io</td>
      <td>Transcriptomics, Sequence Analysis</td>
      <td>anndata</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/anndata/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/anndata</td>
      <td>0.10.3</td>
      <td>anndata</td>
      <td>0.6.22.post1</td>
      <td></td>
      <td></td>
      <td>5</td>
      <td>4</td>
      <td>5</td>
      <td>0</td>
      <td>5</td>
      <td>4</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>625</td>
      <td>35620</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>arriba</td>
      <td>arriba, arriba_draw_fusions, arriba_get_filters</td>
      <td>Arriba detects fusion genes in RNA-Seq data after running RNA-STAR</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/suhrig/arriba</td>
      <td>Sequence Analysis, Transcriptomics</td>
      <td>arriba</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/arriba</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/arriba</td>
      <td>2.4.0</td>
      <td>arriba</td>
      <td>2.4.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>38</td>
      <td>3528</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>askor</td>
      <td>askor_de</td>
      <td>AskoR links EdgeR and AskOmics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/askomics/askoR</td>
      <td>Transcriptomics</td>
      <td>askor_de</td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/askor</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/askor</td>
      <td>0.2</td>
      <td>bioconductor-limma</td>
      <td>3.58.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>bamparse</td>
      <td>bamparse</td>
      <td>Generates hit count lists from bam alignments.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, Transcriptomics</td>
      <td>bamparse</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/bamparse</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/bamparse</td>
      <td>4.1.1</td>
      <td>pysam</td>
      <td>0.22.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>baredsc</td>
      <td>baredsc_1d, baredsc_2d, baredsc_combine_1d, baredsc_combine_2d</td>
      <td>baredSC is a tool that uses a Monte-Carlo Markov Chain to estimate a confidence interval on the probability density function (PDF) of expression of one or two genes from single-cell RNA-seq data.</td>
      <td>baredsc</td>
      <td>baredsc</td>
      <td></td>
      <td>baredSC</td>
      <td>The baredSC (Bayesian Approach to Retreive Expression Distribution of Single Cell) is a tool that uses a Monte-Carlo Markov Chain to estimate a confidence interval on the probability density function (PDF) of expression of one or two genes from single-cell RNA-seq data.</td>
      <td>Data retrieval, Expression correlation analysis, Differential gene expression profiling</td>
      <td>RNA-Seq, Cytometry, Transcriptomics, Gene transcripts, Statistics and probability</td>
      <td>Up-to-date</td>
      <td>https://github.com/lldelisle/baredSC</td>
      <td>Transcriptomics, Visualization</td>
      <td>baredsc</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/baredsc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/baredsc</td>
      <td>1.1.3</td>
      <td>baredsc</td>
      <td>1.1.3</td>
      <td>Data retrieval, Expression correlation analysis, Differential gene expression profiling</td>
      <td>RNA-Seq, Cytometry, Transcriptomics, Gene transcripts, Statistics and probability</td>
      <td>4</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>71</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>bctools</td>
      <td>bctools_convert_to_binary_barcode, bctools_extract_crosslinked_nucleotides, bctools_extract_alignment_ends, bctools_extract_barcodes, bctools_merge_pcr_duplicates, bctools_remove_tail, bctools_remove_spurious_events</td>
      <td>bctools is a set of tools for handling barcodes and UMIs in NGS data.bctools can be used to merge PCR duplicates according to unique molecular barcodes (UMIs),to extract barcodes from arbitrary positions relative to the read starts,to clean up readthroughs into UMIs with paired-end sequencing andhandles binary barcodes as used with uvCLAP and FLASH.License: Apache License 2.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/dmaticzka/bctools</td>
      <td>Sequence Analysis, Transcriptomics</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bedtools</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bctools</td>
      <td>0.2.2</td>
      <td>bctools</td>
      <td>0.2.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>87</td>
      <td>2895</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>brew3r_r</td>
      <td>brew3r_r</td>
      <td>Extend 3' end of a GTF using another GTF as a template</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/BREW3R.r.html</td>
      <td>Transcriptomics, RNA</td>
      <td>brew3r_r</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/brew3r_r</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/brew3r_r</td>
      <td>1.0.2</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>cell-types-analysis</td>
      <td>ct_build_cell_ontology_dict, ct_check_labels, ct_combine_tool_outputs, ct_downsample_cells, ct_get_consensus_outputs, ct_get_empirical_dist, ct_get_tool_perf_table, ct_get_tool_pvals</td>
      <td>Tools for analysis of predictions from scRNAseq cell type classification tools, see https://github.com/ebi-gene-expression-group/cell-types-analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>suite_cell_types_analysis</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/cell-types-analysis</td>
      <td>1.1.1</td>
      <td>cell-types-analysis</td>
      <td>0.1.11</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>6</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>cemitool</td>
      <td>cemitool</td>
      <td>Gene co-expression network analysis tool</td>
      <td>cemitool</td>
      <td>cemitool</td>
      <td></td>
      <td>CEMiTool</td>
      <td>It unifies the discovery and the analysis of coexpression gene modules in a fully automatic manner, while providing a user-friendly html report with high quality graphs. Our tool evaluates if modules contain genes that are over-represented by specific pathways or that are altered in a specific sample group. Additionally, CEMiTool is able to integrate transcriptomic data with interactome information, identifying the potential hubs on each network.</td>
      <td>Enrichment analysis, Pathway or network analysis</td>
      <td>Gene expression, Transcriptomics, Microarray experiment</td>
      <td>To update</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/CEMiTool.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>cemitool</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cemitool</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cemitool</td>
      <td>1.18.1</td>
      <td>bioconductor-cemitool</td>
      <td>1.26.0</td>
      <td>Enrichment analysis, Pathway or network analysis</td>
      <td>Transcriptomics, Microarray experiment</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>9</td>
      <td>98</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>cherri</td>
      <td>cherri_eval, cherri_train</td>
      <td>Computational Help Evaluating RNA-RNA interactions</td>
      <td>cherri</td>
      <td>cherri</td>
      <td></td>
      <td>cherri</td>
      <td>CheRRI detects functional RNA-RNA interaction (RRI) sites, by evaluating if an interaction site most likely occurs in nature. It helps to filter interaction sites generated either experimentally or by an RRI prediction algorithm by removing false positive interactions.</td>
      <td></td>
      <td>Molecular interactions, pathways and networks, Structure analysis, Machine learning</td>
      <td>To update</td>
      <td>https://github.com/BackofenLab/Cherri</td>
      <td>Transcriptomics, RNA</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cherri</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cherri</td>
      <td>0.7</td>
      <td>cherri</td>
      <td>0.8</td>
      <td></td>
      <td>Molecular interactions, pathways and networks, Structure analysis, Machine learning</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>1</td>
      <td>207</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>chira</td>
      <td>chira_collapse, chira_extract, chira_map, chira_merge, chira_quantify</td>
      <td>Chimeric Read Annotator for RNA-RNA interactome data</td>
      <td>chira</td>
      <td>chira</td>
      <td></td>
      <td>ChiRA</td>
      <td>ChiRA is a tool suite to analyze RNA-RNA interactome experimental data such as CLASH, CLEAR-CLIP, PARIS, SPLASH, etc.</td>
      <td></td>
      <td>RNA, Molecular interactions, pathways and networks, Functional, regulatory and non-coding RNA</td>
      <td>Up-to-date</td>
      <td>https://github.com/pavanvidem/chira</td>
      <td>RNA, Transcriptomics, Sequence Analysis</td>
      <td>chira</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/chira</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/chira</td>
      <td>1.4.3</td>
      <td>chira</td>
      <td>1.4.3</td>
      <td></td>
      <td>Molecular interactions, pathways and networks, Functional, regulatory and non-coding RNA</td>
      <td>5</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>97</td>
      <td>6418</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>cite_seq_count</td>
      <td>cite_seq_count</td>
      <td>Count CMO/HTO</td>
      <td>CITE-seq-Count</td>
      <td>CITE-seq-Count</td>
      <td></td>
      <td>CITE-seq-Count</td>
      <td>Tool for counting antibody TAGS from a CITE-seq and/or cell hashing experiment.</td>
      <td>RNA-Seq quantification</td>
      <td>Transcriptomics, Immunoproteins and antigens</td>
      <td>Up-to-date</td>
      <td>https://github.com/Hoohm/CITE-seq-Count</td>
      <td>Transcriptomics</td>
      <td>cite_seq_count</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cite_seq_count</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cite_seq_count</td>
      <td>1.4.4</td>
      <td>cite-seq-count</td>
      <td>1.4.4</td>
      <td>RNA-Seq quantification</td>
      <td>Transcriptomics, Immunoproteins and antigens</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>18</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>cosg</td>
      <td>cosg</td>
      <td>Marker gene identification for single-cell sequencing data using COSG.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/genecell/COSG</td>
      <td>Transcriptomics, Sequence Analysis</td>
      <td>cosg</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cosg/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cosg</td>
      <td>1.0.1</td>
      <td>cosg</td>
      <td>1.0.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>cpat</td>
      <td>cpat</td>
      <td>Coding-potential assessment tool using an alignment-free logistic regression model.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/liguowang/cpat</td>
      <td>Transcriptomics</td>
      <td>cpat</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/cpat</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/cpat</td>
      <td>3.0.5</td>
      <td>cpat</td>
      <td>3.0.5</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>21</td>
      <td>134</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>cpm_tpm_rpk</td>
      <td>cpm_tpm_rpk</td>
      <td>Generate CPM,TPM or RPK from raw counts</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Transcriptomics</td>
      <td>cpm_tpm_rpk</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/cpm_tpm_rpk</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/cpm_tpm_rpk</td>
      <td>0.5.2</td>
      <td>r-optparse</td>
      <td>1.3.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>crosscontamination_barcode_filter</td>
      <td>crosscontamination_barcode_filter</td>
      <td>Barcode contamination discovery tool</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, Visualization</td>
      <td>crosscontamination_barcode_filter</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/crosscontamination_barcode_filter</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/crosscontamination_barcode_filter</td>
      <td>0.3</td>
      <td>r-ggplot2</td>
      <td>2.2.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>17</td>
      <td>347</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>cuffcompare</td>
      <td>cuffcompare</td>
      <td>Galaxy wrappers for the Cuffcompare tool.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://cole-trapnell-lab.github.io/cufflinks/</td>
      <td>Transcriptomics</td>
      <td>cuffcompare</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/cufflinks/cuffcompare</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/cufflinks/cuffcompare</td>
      <td>2.2.1</td>
      <td>cufflinks</td>
      <td>2.2.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>65</td>
      <td>1130</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>cuffdiff</td>
      <td>cuffdiff</td>
      <td>Galaxy wrappers for the Cuffdiff tool.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://cole-trapnell-lab.github.io/cufflinks/</td>
      <td>Transcriptomics</td>
      <td>cuffdiff</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/cufflinks/cuffdiff</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/cufflinks/cuffdiff</td>
      <td>2.2.1</td>
      <td>cufflinks</td>
      <td>2.2.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>228</td>
      <td>5831</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>cufflinks</td>
      <td>cufflinks</td>
      <td>Galaxy wrappers for the Cufflinks tool.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://cole-trapnell-lab.github.io/cufflinks/</td>
      <td>Transcriptomics</td>
      <td>cufflinks</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/cufflinks/cufflinks</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/cufflinks/cufflinks</td>
      <td>2.2.1</td>
      <td>cufflinks</td>
      <td>2.2.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>319</td>
      <td>32218</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>cuffmerge</td>
      <td>cuffmerge</td>
      <td>Galaxy wrappers for the Cuffmerge tool.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://cole-trapnell-lab.github.io/cufflinks/</td>
      <td>Transcriptomics</td>
      <td>cuffmerge</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/cufflinks/cuffmerge</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/cufflinks/cuffmerge</td>
      <td>2.2.1</td>
      <td>cufflinks</td>
      <td>2.2.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>122</td>
      <td>3292</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>cuffnorm</td>
      <td>cuffnorm</td>
      <td>The Cuffnorm tool</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://cole-trapnell-lab.github.io/cufflinks/</td>
      <td>Transcriptomics</td>
      <td>cuffnorm</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/cufflinks/cuffnorm</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/cufflinks/cuffnorm</td>
      <td>2.2.1</td>
      <td>cufflinks</td>
      <td>2.2.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>27</td>
      <td>660</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>cuffquant</td>
      <td>cuffquant</td>
      <td>The Cuffquant tool</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://cole-trapnell-lab.github.io/cufflinks/</td>
      <td>Transcriptomics</td>
      <td>cuffquant</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/cufflinks/cuffquant</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/cufflinks/cuffquant</td>
      <td>2.2.1</td>
      <td>cufflinks</td>
      <td>2.2.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>18</td>
      <td>688</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>data-hca</td>
      <td>hca_matrix_downloader</td>
      <td>Tools for interacting with the Human Cell Atlas resource https://prod.data.humancellatlas.org/explore/projects</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, Sequence Analysis</td>
      <td>suite_human_cell_atlas_tools</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/data-hca</td>
      <td>v0.0.4+galaxy0</td>
      <td>hca-matrix-downloader</td>
      <td>0.0.4</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>34</td>
      <td>439</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>data-scxa</td>
      <td>retrieve_scxa</td>
      <td>Tools for interacting with the EMBL-EBI Expression Atlas resource https://www.ebi.ac.uk/gxa/home https://www.ebi.ac.uk/gxa/sc/home</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, Sequence Analysis</td>
      <td>suite_ebi_expression_atlas</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/data-scxa</td>
      <td>v0.0.2+galaxy2</td>
      <td>wget</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>99</td>
      <td>799</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>decoupler</td>
      <td>score_genes_aucell, decoupler_pathway_inference, decoupler_pseudobulk</td>
      <td>decoupler - Ensemble of methods to infer biological activities</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://decoupler-py.readthedocs.io/en/latest/</td>
      <td>Transcriptomics</td>
      <td>suite_decoupler</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/decoupler</td>
      <td>1.4.0+galaxy3</td>
      <td>decoupler</td>
      <td>1.5.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>deg_annotate</td>
      <td>deg_annotate</td>
      <td>Annotate DESeq2/DEXSeq output tables</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics</td>
      <td>deg_annotate</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/deg_annotate</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/deg_annotate</td>
      <td>1.1.0</td>
      <td>bedtools</td>
      <td>2.31.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1774</td>
      <td>19910</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>deseq2</td>
      <td>deseq2</td>
      <td>Differential gene expression analysis based on the negative binomial distribution</td>
      <td>DESeq2</td>
      <td>DESeq2</td>
      <td></td>
      <td>DESeq2</td>
      <td>R/Bioconductor package for differential gene expression analysis based on the negative binomial distribution. Estimate variance-mean dependence in count data from high-throughput sequencing assays and test for differential expression based on a model using the negative binomial distribution.</td>
      <td>Differential gene expression analysis, RNA-Seq analysis</td>
      <td>RNA-Seq</td>
      <td>To update</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/DESeq2.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>deseq2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/deseq2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/deseq2</td>
      <td>2.11.40.8</td>
      <td>bioconductor-deseq2</td>
      <td>1.42.0</td>
      <td>Differential gene expression analysis, RNA-Seq analysis</td>
      <td>RNA-Seq</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>4990</td>
      <td>95752</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>deseq2_normalization</td>
      <td>deseq2_normalization</td>
      <td>Normalizes gene hitlists</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, Transcriptomics, Sequence Analysis, Statistics</td>
      <td>deseq2_normalization</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/deseq2_normalization</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/deseq2_normalization</td>
      <td>1.40.2+galaxy0</td>
      <td>bioconductor-deseq2</td>
      <td>1.42.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>dexseq</td>
      <td>dexseq, dexseq_count, plotdexseq</td>
      <td>Inference of differential exon usage in RNA-Seq</td>
      <td>dexseq</td>
      <td>dexseq</td>
      <td></td>
      <td>DEXSeq</td>
      <td>The package is focused on finding differential exon usage using RNA-seq exon counts between samples with different experimental designs. It provides functions that allows the user to make the necessary statistical tests based on a model that uses the negative binomial distribution to estimate the variance between biological replicates and generalized linear models for testing. The package also provides functions for the visualization and exploration of the results.</td>
      <td>Enrichment analysis, Exonic splicing enhancer prediction</td>
      <td>RNA-Seq</td>
      <td>Up-to-date</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/DEXSeq.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>dexseq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/dexseq</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/dexseq</td>
      <td>1.48.0</td>
      <td>bioconductor-dexseq</td>
      <td>1.48.0</td>
      <td>Enrichment analysis, Exonic splicing enhancer prediction</td>
      <td>RNA-Seq</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>218</td>
      <td>16064</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>dropletutils</td>
      <td>dropletutils_empty_drops, dropletutils_read_10x</td>
      <td>De-composed DropletUtils functionality tools, based on https://github.com/ebi-gene-expression-group/dropletutils-scripts and DropletUtils 1.0.3</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_dropletutils</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/dropletutils</td>
      <td>1.0.4</td>
      <td>dropletutils-scripts</td>
      <td>0.0.5</td>
      <td></td>
      <td></td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>201</td>
      <td>1599</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>dropletutils</td>
      <td>dropletutils</td>
      <td>DropletUtils - Utilities for handling droplet-based single-cell RNA-seq data</td>
      <td>dropletutils</td>
      <td>dropletutils</td>
      <td></td>
      <td>DropletUtils</td>
      <td>Provides a number of utility functions for handling single-cell (RNA-seq) data from droplet technologies such as 10X Genomics. This includes data loading, identification of cells from empty droplets, removal of barcode-swapped pseudo-cells, and downsampling of the count matrix.</td>
      <td>Loading, Community profiling</td>
      <td>Gene expression, RNA-seq, Sequencing, Transcriptomics</td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/devel/bioc/html/DropletUtils.html</td>
      <td>Transcriptomics, Sequence Analysis</td>
      <td>dropletutils</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/dropletutils/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/dropletutils</td>
      <td>1.10.0</td>
      <td>bioconductor-dropletutils</td>
      <td>1.22.0</td>
      <td></td>
      <td>Sequencing, Transcriptomics</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>126</td>
      <td>3934</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>edger</td>
      <td>edger</td>
      <td>Perform RNA-Seq differential expression analysis using edgeR pipeline</td>
      <td>edger</td>
      <td>edger</td>
      <td></td>
      <td>edgeR</td>
      <td>Differential expression analysis of RNA-seq expression profiles with biological replication. Implements a range of statistical methodology based on the negative binomial distributions, including empirical Bayes estimation, exact tests, generalized linear models and quasi-likelihood tests. As well as RNA-seq, it be applied to differential signal analysis of other types of genomic data that produce counts, including ChIP-seq, SAGE and CAGE.</td>
      <td>Differential gene expression analysis</td>
      <td>Genetics, RNA-Seq, ChIP-seq</td>
      <td>To update</td>
      <td>http://bioconductor.org/packages/release/bioc/html/edgeR.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>edger</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/edger</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/edger</td>
      <td>3.36.0</td>
      <td>bioconductor-edger</td>
      <td>4.0.16</td>
      <td>Differential gene expression analysis</td>
      <td>Genetics, RNA-Seq, ChIP-seq</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>945</td>
      <td>18117</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>egsea</td>
      <td>egsea</td>
      <td>This tool implements the Ensemble of Gene Set Enrichment Analyses (EGSEA) method for gene set testing</td>
      <td>egsea</td>
      <td>egsea</td>
      <td></td>
      <td>EGSEA</td>
      <td>This package implements the Ensemble of Gene Set Enrichment Analyses method for gene set testing.</td>
      <td>Gene set testing</td>
      <td>Systems biology</td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/EGSEA.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>egsea</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/egsea</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/egsea</td>
      <td>1.20.0</td>
      <td>bioconductor-egsea</td>
      <td>1.28.0</td>
      <td>Gene set testing</td>
      <td>Systems biology</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>177</td>
      <td>2524</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_provider</td>
      <td>fastq_provider</td>
      <td>Retrieval and download of FASTQ files from ENA and other repositories such as HCA.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ebi-gene-expression-group/atlas-fastq-provider</td>
      <td>Data Source, RNA, Transcriptomics</td>
      <td>atlas_fastq_provider</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/develop/tools/fastq_provider</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/fastq_provider</td>
      <td>0.4.4</td>
      <td>atlas-fastq-provider</td>
      <td>0.4.7</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_utils</td>
      <td>fastq_filter_n, fastq_trim_poly_at</td>
      <td>Set of tools for handling fastq files</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/nunofonseca/fastq_utils</td>
      <td>Transcriptomics, RNA</td>
      <td>fastq_utils</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/develop/tools/qc/fastq_utils</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/qc/fastq_utils</td>
      <td>0.25.1+galaxy0</td>
      <td>fastq_utils</td>
      <td>0.25.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>featurecounts</td>
      <td>featurecounts</td>
      <td>featureCounts counts the number of reads aligned to defined masked regions in a reference genome</td>
      <td>featurecounts</td>
      <td>featurecounts</td>
      <td></td>
      <td>FeatureCounts</td>
      <td>featureCounts is a very efficient read quantifier. It can be used to summarize RNA-seq reads and gDNA-seq reads to a variety of genomic features such as genes, exons, promoters, gene bodies and genomic bins. It is included in the Bioconductor Rsubread package and also in the SourceForge Subread package.</td>
      <td>Read summarisation, RNA-Seq quantification</td>
      <td>RNA-Seq</td>
      <td>To update</td>
      <td>http://bioinf.wehi.edu.au/featureCounts</td>
      <td>RNA, Transcriptomics, Sequence Analysis</td>
      <td>featurecounts</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/featurecounts</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/featurecounts</td>
      <td>2.0.3</td>
      <td>subread</td>
      <td>2.0.6</td>
      <td>Read summarisation, RNA-Seq quantification</td>
      <td>RNA-Seq</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>4679</td>
      <td>696399</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>fgsea</td>
      <td>fgsea</td>
      <td>Perform gene set testing using fgsea</td>
      <td>fgsea</td>
      <td>fgsea</td>
      <td></td>
      <td>fgsea</td>
      <td>The package implements an algorithm for fast gene set enrichment analysis. Using the fast algorithm allows to make more permutations and get more fine grained p-values, which allows to use accurate stantard approaches to multiple hypothesis correction.</td>
      <td>Gene-set enrichment analysis</td>
      <td>Genetics</td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/fgsea.html</td>
      <td>Visualization, Transcriptomics, Statistics</td>
      <td>fgsea</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fgsea</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fgsea</td>
      <td>1.8.0+galaxy1</td>
      <td>bioconductor-fgsea</td>
      <td>1.28.0</td>
      <td>Gene-set enrichment analysis</td>
      <td>Genetics</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>307</td>
      <td>5240</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>garnett</td>
      <td>garnett_check_markers, garnett_classify_cells, garnett_get_feature_genes, garnett_get_std_output, garnett_train_classifier, garnett_transform_markers, update_marker_file</td>
      <td>De-composed Garnett functionality tools, see https://github.com/ebi-gene-expression-group/garnett-cli and r-garnett 0.2.8</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_garnett</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/garnett</td>
      <td>0.2.8</td>
      <td>garnett-cli</td>
      <td>0.0.5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>27</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>genomic_super_signature</td>
      <td>genomic_super_signature</td>
      <td>Interpretation of RNAseq experiments through robust, efficient comparison to public databases</td>
      <td>genomicsupersignature</td>
      <td>genomicsupersignature</td>
      <td></td>
      <td>GenomicSuperSignature</td>
      <td>GenomicSuperSignature is a package for the interpretation of RNA-seq experiments through robust, efficient comparison to public databases.</td>
      <td>Gene-set enrichment analysis, Essential dynamics, Deposition, Principal component visualisation, Dimensionality reduction</td>
      <td>RNA-Seq, Transcriptomics, Microbial ecology, Genotype and phenotype, Microarray experiment</td>
      <td>To update</td>
      <td>https://github.com/shbrief/GenomicSuperSignature</td>
      <td>Sequence Analysis, RNA, Transcriptomics</td>
      <td>genomic_super_signature</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/genomic_super_signature</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/genomic_super_signature</td>
      <td>1.2.0</td>
      <td>bioconductor-genomicsupersignature</td>
      <td>1.10.0</td>
      <td>Gene-set enrichment analysis, Essential dynamics, Deposition, Principal component visualisation, Dimensionality reduction</td>
      <td>RNA-Seq, Transcriptomics, Microbial ecology, Genotype and phenotype, Microarray experiment</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>11</td>
      <td>46</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>gffcompare</td>
      <td>gffcompare</td>
      <td>Galaxy wrappers for Geo Pertea's GffCompare package.</td>
      <td>gffcompare</td>
      <td>gffcompare</td>
      <td></td>
      <td>gffcompare</td>
      <td>Program for comparing, annotating, merging and tracking transcripts in GFF files.</td>
      <td>Sequence annotation</td>
      <td>Nucleic acids, Sequence analysis</td>
      <td>Up-to-date</td>
      <td>https://github.com/gpertea/gffcompare/</td>
      <td>Transcriptomics</td>
      <td>gffcompare</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gffcompare</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/gffcompare</td>
      <td>0.12.6</td>
      <td>gffcompare</td>
      <td>0.12.6</td>
      <td>Sequence annotation</td>
      <td>Nucleic acids, Sequence analysis</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>381</td>
      <td>3477</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>gsc_filter_cells</td>
      <td>filter_cells</td>
      <td>Filter single cell RNAseq data on libray depth and number of detected genes</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Transcriptomics</td>
      <td>gsc_filter_cells</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_filter_cells</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_filter_cells</td>
      <td>4.3.1+galaxy0</td>
      <td>r-optparse</td>
      <td>1.3.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>gsc_filter_genes</td>
      <td>filter_genes</td>
      <td>Filter genes that are detected in less than a fraction of libraries in single cell RNAseq data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Transcriptomics</td>
      <td>gsc_filter_genes</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_filter_genes</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_filter_genes</td>
      <td>4.3.1+galaxy0</td>
      <td>r-optparse</td>
      <td>1.3.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>gsc_gene_expression_correlations</td>
      <td>single_cell_gene_expression_correlations</td>
      <td>Compute single-cell paire-wise gene expressions correlations</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Transcriptomics</td>
      <td>gsc_gene_expression_correlations</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/gsc_gene_expression_correlations</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_gene_expression_correlations</td>
      <td>4.3.1+galaxy0</td>
      <td>r-optparse</td>
      <td>1.3.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>gsc_high_dimensions_visualisation</td>
      <td>high_dimensions_visualisation</td>
      <td>Generates PCA, t-SNE and HCPC visualisation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Transcriptomics, Visualization</td>
      <td>gsc_high_dimensions_visualisation</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_high_dimension_visualization</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_high_dimensions_visualisation</td>
      <td>4.3+galaxy0</td>
      <td>r-optparse</td>
      <td>1.3.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>gsc_mannwhitney_de</td>
      <td>mannwhitney_de</td>
      <td>Perform a mann-whitney differential testing between two sets of gene expression data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Transcriptomics</td>
      <td>gsc_mannwhitney_de</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_mannwhitney_de</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_mannwhitney_de</td>
      <td>4.1.3+galaxy0</td>
      <td>r-optparse</td>
      <td>1.3.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>gsc_scran_normalize</td>
      <td>scran_normalize</td>
      <td>Normalize raw counts using scran</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Transcriptomics</td>
      <td>gsc_scran_normalize</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_scran_normalize</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_scran_normalize</td>
      <td>1.28.1+galaxy0</td>
      <td>bioconductor-scran</td>
      <td>1.30.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>14</td>
      <td>78</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>gsc_signature_score</td>
      <td>signature_score</td>
      <td>Compute signature scores from single cell RNAseq data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Transcriptomics</td>
      <td>gsc_signature_score</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_signature_score</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_signature_score</td>
      <td>2.3.9+galaxy0</td>
      <td>r-optparse</td>
      <td>1.3.2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>heinz</td>
      <td>heinz_bum, heinz, heinz_scoring, heinz_visualization</td>
      <td>An algorithm for identification of the optimal scoring subnetwork.</td>
      <td>heinz</td>
      <td>heinz, bionet</td>
      <td></td>
      <td>Heinz</td>
      <td>Tool for single-species active module discovery.</td>
      <td>Pathway or network analysis</td>
      <td>Genetics, Gene expression, Molecular interactions, pathways and networks</td>
      <td>To update</td>
      <td>https://github.com/ls-cwi/heinz</td>
      <td>Transcriptomics, Visualization, Statistics</td>
      <td>heinz</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/heinz</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/heinz</td>
      <td>1.0</td>
      <td>bioconductor-bionet</td>
      <td>1.62.0</td>
      <td>Pathway or network analysis</td>
      <td>Genetics, Gene expression, Molecular interactions, pathways and networks</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>242</td>
      <td>1186</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>isoformswitchanalyzer</td>
      <td>isoformswitchanalyzer</td>
      <td>Statistical identification of isoform switching from RNA-seq derived quantification of novel and/or annotated full-length isoforms.</td>
      <td>IsoformSwitchAnalyzeR</td>
      <td>IsoformSwitchAnalyzeR</td>
      <td></td>
      <td>IsoformSwitchAnalyzeR</td>
      <td>Enables identification of isoform switches with predicted functional consequences from RNA-seq data. Consequences can be chosen from a long list but includes protein domains gain/loss changes in NMD sensitivity etc. It directly supports import of data from Cufflinks/Cuffdiff, Kallisto, Salmon and RSEM but other transcript qunatification tools are easy to import as well.</td>
      <td>Sequence comparison, Sequence analysis</td>
      <td>Computational biology, Gene transcripts</td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/devel/bioc/html/IsoformSwitchAnalyzeR.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>isoformswitchanalyzer</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/isoformswitchanalyzer</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/isoformswitchanalyzer</td>
      <td>1.20.0</td>
      <td>bioconductor-isoformswitchanalyzer</td>
      <td>2.2.0</td>
      <td>Sequence comparison, Sequence analysis</td>
      <td>Computational biology, Gene transcripts</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>29</td>
      <td>822</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>kallisto</td>
      <td>kallisto_pseudo, kallisto_quant</td>
      <td>kallisto is a program for quantifying abundances of transcripts from RNA-Seqdata, or more generally of target sequences using high-throughput sequencingreads. It is based on the novel idea of pseudoalignment for rapidlydetermining the compatibility of reads with targets, without the need foralignment.</td>
      <td>kallisto</td>
      <td>kallisto</td>
      <td></td>
      <td>kallisto</td>
      <td>A program for quantifying abundances of transcripts from RNA-Seq data, or more generally of target sequences using high-throughput sequencing reads. It is based on the novel idea of pseudoalignment for rapidly determining the compatibility of reads with targets, without the need for alignment.</td>
      <td>Gene expression profiling</td>
      <td>Transcriptomics, RNA-seq, Gene expression</td>
      <td>To update</td>
      <td>https://pachterlab.github.io/kallisto/</td>
      <td>Transcriptomics</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/kallisto/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/kallisto</td>
      <td>0.48.0</td>
      <td>kallisto</td>
      <td>0.50.1</td>
      <td>Gene expression profiling</td>
      <td>Transcriptomics</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>299</td>
      <td>24155</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>limma_voom</td>
      <td>limma_voom</td>
      <td>Perform RNA-Seq differential expression analysis using limma voom pipeline</td>
      <td>limma</td>
      <td>limma</td>
      <td></td>
      <td>limma</td>
      <td>Data analysis, linear models and differential expression for microarray data.</td>
      <td>RNA-Seq analysis</td>
      <td>Molecular biology, Genetics</td>
      <td>Up-to-date</td>
      <td>http://bioconductor.org/packages/release/bioc/html/limma.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>limma_voom</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/limma_voom</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/limma_voom</td>
      <td>3.58.1</td>
      <td>bioconductor-limma</td>
      <td>3.58.1</td>
      <td>RNA-Seq analysis</td>
      <td>Molecular biology, Genetics</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1012</td>
      <td>20344</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>masigpro</td>
      <td>masigpro</td>
      <td>Identify significantly differential expression profiles in time-course microarray experiments</td>
      <td>masigpro</td>
      <td>masigpro</td>
      <td></td>
      <td>maSigPro</td>
      <td>Regression based approach to find genes for which there are significant gene expression profile differences between experimental groups in time course microarray and RNA-Seq experiments.</td>
      <td>Regression analysis</td>
      <td>Gene expression, Molecular genetics, Microarray experiment, RNA-Seq</td>
      <td>To update</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/maSigPro.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>masigpro</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/masigpro</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/masigpro</td>
      <td>1.49.3</td>
      <td>coreutils</td>
      <td>8.25</td>
      <td>Regression analysis</td>
      <td>Gene expression, Microarray experiment, RNA-Seq</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>13</td>
      <td>576</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>mircounts</td>
      <td>mircounts</td>
      <td>Generates miRNA count lists from read alignments to mirBase.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, Transcriptomics</td>
      <td>mircounts</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/mircounts</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/mircounts</td>
      <td>1.6</td>
      <td>tar</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>monocle3</td>
      <td>monocle3_create, monocle3_diffExp, monocle3_learnGraph, monocle3_orderCells, monocle3_partition, monocle3_plotCells, monocle3_preprocess, monocle3_reduceDim, monocle3_topmarkers</td>
      <td>De-composed monocle3 functionality tools, based on https://github.com/ebi-gene-expression-group/monocle-scripts and monocle3 0.1.2.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_monocle3</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/monocle3</td>
      <td>0.1.4</td>
      <td>monocle3-cli</td>
      <td>0.0.9</td>
      <td></td>
      <td></td>
      <td>9</td>
      <td>0</td>
      <td>9</td>
      <td>0</td>
      <td>9</td>
      <td>0</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>264</td>
      <td>3266</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>multigsea</td>
      <td>multigsea</td>
      <td>GSEA-based pathway enrichment analysis for multi-omics data</td>
      <td>multiGSEA</td>
      <td>multiGSEA</td>
      <td></td>
      <td>multiGSEA</td>
      <td>A GSEA-based pathway enrichment analysis for multi-omics data.multiGSEA: a GSEA-based pathway enrichment analysis for multi-omics data, BMC Bioinformatics 21, 561 (2020).Combining GSEA-based pathway enrichment with multi omics data integration.</td>
      <td>Gene-set enrichment analysis, Aggregation, Pathway analysis</td>
      <td>Metabolomics, Molecular interactions, pathways and networks, Proteomics, Transcriptomics, Small molecules</td>
      <td>Up-to-date</td>
      <td>https://bioconductor.org/packages/devel/bioc/html/multiGSEA.html</td>
      <td>Transcriptomics, Proteomics, Statistics</td>
      <td>multigsea</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/multigsea</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/multigsea</td>
      <td>1.12.0</td>
      <td>bioconductor-multigsea</td>
      <td>1.12.0</td>
      <td>Gene-set enrichment analysis, Aggregation, Pathway analysis</td>
      <td>Metabolomics, Molecular interactions, pathways and networks, Proteomics, Transcriptomics, Small molecules</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>2</td>
      <td>53</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>music_deconvolution</td>
      <td>music_construct_eset, music_inspect_eset, music_manipulate_eset, music_compare, music_deconvolution</td>
      <td>Multi-subject Single Cell deconvolution (MuSiC)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/xuranw/MuSiC</td>
      <td>Transcriptomics</td>
      <td>music</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/music/</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/music_deconvolution</td>
      <td>0.1.1</td>
      <td>music-deconvolution</td>
      <td>0.1.1</td>
      <td></td>
      <td></td>
      <td>5</td>
      <td>5</td>
      <td>4</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>240</td>
      <td>1872</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>nugen_nudup</td>
      <td>nugen_nudup</td>
      <td>Marks/removes PCR introduced duplicate molecules based on the molecular tagging technology used in NuGEN products.</td>
      <td>nudup</td>
      <td>nudup</td>
      <td></td>
      <td>NuDup</td>
      <td>Marks/removes duplicate molecules based on the molecular tagging technology used in Tecan products.</td>
      <td>Duplication detection</td>
      <td>Sequencing</td>
      <td>Up-to-date</td>
      <td>https://github.com/tecangenomics/nudup</td>
      <td>SAM, Metagenomics, Sequence Analysis, Transcriptomics</td>
      <td>nugen_nudup</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/nugen_nudup</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/nugen_nudup</td>
      <td>2.3.3</td>
      <td>nudup</td>
      <td>2.3.3</td>
      <td>Duplication detection</td>
      <td>Sequencing</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>pathifier</td>
      <td>pathifier</td>
      <td>pathifier</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://</td>
      <td>Transcriptomics, Statistics</td>
      <td>pathifier</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/pathifier</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/pathifier</td>
      <td>1.40.0</td>
      <td>bioconductor-pathifier</td>
      <td>1.40.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>10</td>
      <td>228</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>pizzly</td>
      <td>pizzly</td>
      <td>Pizzly is a program for detecting gene fusions from RNA-Seq data of cancer samples.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/pmelsted/pizzly/</td>
      <td>Transcriptomics</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pizzly/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pizzly</td>
      <td>0.37.3.1</td>
      <td>pizzly</td>
      <td>0.37.3</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>16</td>
      <td>208</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>psiclass</td>
      <td>psiclass</td>
      <td>PsiCLASS is a reference-based transcriptome assembler for single or multiple RNA-seq samples.</td>
      <td>psiclass</td>
      <td>psiclass</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/splicebox/PsiCLASS</td>
      <td>Transcriptomics</td>
      <td>psiclass</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/psiclass</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/psiclass</td>
      <td>1.0.3</td>
      <td>psiclass</td>
      <td>1.0.3</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>15</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>qualimap</td>
      <td>qualimap_bamqc, qualimap_counts, qualimap_multi_bamqc, qualimap_rnaseq</td>
      <td></td>
      <td>qualimap</td>
      <td>qualimap</td>
      <td></td>
      <td>QualiMap</td>
      <td>Platform-independent application written in Java and R that provides both a Graphical User Inteface (GUI) and a command-line interface to facilitate the quality control of alignment sequencing data.</td>
      <td>Sequencing quality control</td>
      <td>Data quality management</td>
      <td>To update</td>
      <td>http://qualimap.bioinfo.cipf.es/</td>
      <td>Sequence Analysis, Transcriptomics, SAM</td>
      <td>qualimap</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/qualimap</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/qualimap</td>
      <td>2.2.2d</td>
      <td>qualimap</td>
      <td>2.3</td>
      <td>Sequencing quality control</td>
      <td>Data quality management</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>1</td>
      <td>4</td>
      <td>4</td>
      <td>4</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>1328</td>
      <td>664470</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>raceid</td>
      <td>raceid_clustering, raceid_filtnormconf, raceid_inspectclusters, raceid_inspecttrajectory, raceid_trajectory</td>
      <td>RaceID3, StemID2, FateID - scRNA analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/dgrun/RaceID3_StemID2_package/</td>
      <td>Transcriptomics</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/raceid3</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/raceid</td>
      <td>0.2.3</td>
      <td>r-raceid</td>
      <td>0.1.3</td>
      <td></td>
      <td></td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>1</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>82</td>
      <td>4569</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>repenrich</td>
      <td>edger-repenrich, repenrich</td>
      <td>Repeat element profiling</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/repenrich</td>
      <td>Transcriptomics</td>
      <td>repenrich</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/repenrich</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/repenrich</td>
      <td>2.31.1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>repenrich2</td>
      <td>edger-repenrich2, repenrich2</td>
      <td>Repeat element profiling using bowtie2 aligner</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/repenrich2</td>
      <td>Transcriptomics</td>
      <td>repenrich2</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/repenrich2</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/repenrich2</td>
      <td>2.31.1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>rgrnastar</td>
      <td>rna_star, rna_starsolo</td>
      <td>RNA STAR is an ultra fast universal RNA and scRNA-seq aligner and mapper</td>
      <td>star</td>
      <td>star</td>
      <td></td>
      <td>STAR</td>
      <td>Ultrafast universal RNA-seq data aligner</td>
      <td>Sequence alignment</td>
      <td>RNA-Seq, Transcriptomics</td>
      <td>To update</td>
      <td>https://github.com/alexdobin/STAR</td>
      <td>Next Gen Mappers, Transcriptomics</td>
      <td>rgrnastar</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/rgrnastar</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/rgrnastar</td>
      <td>2.7.11a</td>
      <td>star</td>
      <td>2.7.11b</td>
      <td>Sequence alignment</td>
      <td>RNA-Seq, Transcriptomics</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>4015</td>
      <td>425658</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>ribowaltz</td>
      <td>ribowaltz_process, ribowaltz_plot</td>
      <td>Calculation of optimal P-site offsets, diagnostic analysis and visual inspection of ribosome profiling data</td>
      <td>riboWaltz</td>
      <td>riboWaltz</td>
      <td></td>
      <td>riboWaltz</td>
      <td>riboWaltz is an R package for calculation of optimal P-site offsets, diagnostic analysis and visual inspection of ribosome profiling data.</td>
      <td></td>
      <td>Computational biology</td>
      <td>To update</td>
      <td>https://github.com/LabTranslationalArchitectomics/riboWaltz</td>
      <td>Transcriptomics, RNA</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ribowaltz</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ribowaltz</td>
      <td>1.2.0</td>
      <td>ribowaltz</td>
      <td>2.0</td>
      <td></td>
      <td>Computational biology</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>14</td>
      <td>161</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>rsem</td>
      <td>extract_transcript_to_gene_map_from_trinity, purgegtffrommultichromgenes, rsembowtie2, rsembowtie</td>
      <td>transcript quantification from RNA-Seq data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/deweylab/RSEM</td>
      <td>Transcriptomics, RNA</td>
      <td>rsem</td>
      <td>artbio</td>
      <td>https://github.com/artbio/tools-artbio/tree/master/tools/rsem</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/rsem</td>
      <td></td>
      <td>rsem</td>
      <td>1.3.3</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>67</td>
      <td>377</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>rseqc</td>
      <td>rseqc_FPKM_count, rseqc_RNA_fragment_size, rseqc_RPKM_saturation, rseqc_bam2wig, rseqc_bam_stat, rseqc_clipping_profile, rseqc_deletion_profile, rseqc_geneBody_coverage, rseqc_geneBody_coverage2, rseqc_infer_experiment, rseqc_inner_distance, rseqc_insertion_profile, rseqc_junction_annotation, rseqc_junction_saturation, rseqc_mismatch_profile, rseqc_read_GC, rseqc_read_NVC, rseqc_read_distribution, rseqc_read_duplication, rseqc_read_hexamer, rseqc_read_quality, rseqc_tin</td>
      <td>an RNA-seq quality control package</td>
      <td>rseqc</td>
      <td>rseqc</td>
      <td></td>
      <td>RSeQC</td>
      <td>Provides a number of useful modules that can comprehensively evaluate high throughput sequence data especially RNA-seq data. Some basic modules quickly inspect sequence quality, nucleotide composition bias, PCR bias and GC bias, while RNA-seq specific modules evaluate sequencing saturation, mapped reads distribution, coverage uniformity, strand specificity, transcript level RNA integrity etc.</td>
      <td>Data handling</td>
      <td>Sequencing</td>
      <td>Up-to-date</td>
      <td>https://code.google.com/p/rseqc/</td>
      <td>Convert Formats, Sequence Analysis, RNA, Transcriptomics, Visualization</td>
      <td>rseqc</td>
      <td>nilesh</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/rseqc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/rseqc</td>
      <td>5.0.3</td>
      <td>rseqc</td>
      <td>5.0.3</td>
      <td>Data handling</td>
      <td>Sequencing</td>
      <td>22</td>
      <td>22</td>
      <td>22</td>
      <td>22</td>
      <td>22</td>
      <td>22</td>
      <td>22</td>
      <td>22</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>22</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>22</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>22</td>
      <td>22</td>
      <td>3206</td>
      <td>133498</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>ruvseq</td>
      <td>ruvseq</td>
      <td>Remove Unwanted Variation from RNA-Seq Data</td>
      <td>ruvseq</td>
      <td>ruvseq</td>
      <td></td>
      <td>RUVSeq</td>
      <td>This package implements the remove unwanted variation (RUV) methods for the normalization of RNA-Seq read counts between samples.</td>
      <td>Differential gene expression analysis</td>
      <td>Gene expression, RNA-seq</td>
      <td>To update</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/DESeq2.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>ruvseq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ruvseq</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ruvseq</td>
      <td>1.26.0</td>
      <td>bioconductor-ruvseq</td>
      <td>1.36.0</td>
      <td>Differential gene expression analysis</td>
      <td>Gene expression</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>76</td>
      <td>1236</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>salmon</td>
      <td>alevin, salmon, salmonquantmerge</td>
      <td>Salmon is a wicked-fast program to produce a highly-accurate, transcript-level quantification estimates from RNA-seq and single-cell data.</td>
      <td>salmon</td>
      <td>salmon</td>
      <td></td>
      <td>Salmon</td>
      <td>A tool for transcript expression quantification from RNA-seq data</td>
      <td>Sequence composition calculation, RNA-Seq quantification, Gene expression analysis</td>
      <td>RNA-Seq, Gene expression, Transcriptomics</td>
      <td>To update</td>
      <td>https://github.com/COMBINE-lab/salmon</td>
      <td>Sequence Analysis, RNA, Transcriptomics</td>
      <td></td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/salmon</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/salmon</td>
      <td>1.10.1</td>
      <td>salmon</td>
      <td>1.10.3</td>
      <td>Sequence composition calculation, RNA-Seq quantification, Gene expression analysis</td>
      <td>RNA-Seq, Transcriptomics</td>
      <td>2</td>
      <td>1</td>
      <td>3</td>
      <td>1</td>
      <td>2</td>
      <td>1</td>
      <td>3</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>969</td>
      <td>61937</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>sashimi_plot</td>
      <td>sashimi_plot</td>
      <td>Generates a sashimi plot from bam files.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, Transcriptomics, Graphics, Visualization</td>
      <td>sashimi_plot</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/sashimi_plot</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/sashimi_plot</td>
      <td>0.1.1</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>sc3</td>
      <td>sc3_calc_biology, sc3_calc_consens, sc3_calc_dists, sc3_calc_transfs, sc3_estimate_k, sc3_kmeans, sc3_prepare</td>
      <td>De-composed SC3 functionality tools, based on https://github.com/ebi-gene-expression-group/bioconductor-sc3-scripts and SC3 1.8.0.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_sc3</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/sc3</td>
      <td>1.8.0</td>
      <td>sc3-scripts</td>
      <td>0.0.6</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>9</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>scanpy</td>
      <td>scanpy_cluster_reduce_dimension, scanpy_filter, scanpy_inspect, scanpy_normalize, scanpy_plot, scanpy_remove_confounders</td>
      <td>Scanpy  Single-Cell Analysis in Python</td>
      <td>scanpy</td>
      <td>scanpy</td>
      <td></td>
      <td>SCANPY</td>
      <td>Scalable toolkit for analyzing single-cell gene expression data. It includes preprocessing, visualization, clustering, pseudotime and trajectory inference and differential expression testing. The Python-based implementation efficiently deals with datasets of more than one million cells.</td>
      <td>Differential gene expression analysis</td>
      <td>Gene expression, Cell biology, Genetics</td>
      <td>To update</td>
      <td>https://scanpy.readthedocs.io</td>
      <td>Transcriptomics, Sequence Analysis</td>
      <td>scanpy</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/scanpy/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/scanpy</td>
      <td>1.9.6</td>
      <td>scanpy</td>
      <td>1.7.2</td>
      <td>Differential gene expression analysis</td>
      <td>Gene expression, Cell biology, Genetics</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>766</td>
      <td>51978</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>scanpy</td>
      <td>anndata_ops, scanpy_filter_cells, scanpy_filter_genes, scanpy_find_cluster, scanpy_find_markers, scanpy_find_variable_genes, scanpy_integrate_bbknn, scanpy_integrate_combat, scanpy_integrate_harmony, scanpy_integrate_mnn, scanpy_plot_scrublet, scanpy_multiplet_scrublet, scanpy_compute_graph, scanpy_normalise_data, scanpy_parameter_iterator, scanpy_plot_embed, scanpy_plot_trajectory, scanpy_read_10x, scanpy_regress_variable, scanpy_run_diffmap, scanpy_run_dpt, scanpy_run_fdg, scanpy_run_paga, scanpy_run_pca, scanpy_run_tsne, scanpy_run_umap, scanpy_scale_data</td>
      <td>scanpy-scripts, command-line wrapper scripts around Scanpy.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://scanpy.readthedocs.io</td>
      <td>Transcriptomics, Sequence Analysis, RNA</td>
      <td>scanpy_scripts</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/develop/tools/tertiary-analysis/scanpy</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/scanpy</td>
      <td>1.9.3</td>
      <td>scanpy-scripts</td>
      <td>1.9.301</td>
      <td></td>
      <td></td>
      <td>22</td>
      <td>27</td>
      <td>27</td>
      <td>0</td>
      <td>22</td>
      <td>27</td>
      <td>27</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>14</td>
      <td>13</td>
      <td>2185</td>
      <td>39356</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>scater</td>
      <td>scater_calculate_cpm, scater_calculate_qc_metrics, scater_filter, scater_is_outlier, scater_normalize, scater_read_10x_results</td>
      <td>De-composed Scater functionality tools, based on https://github.com/ebi-gene-expression-group/bioconductor-scater-scripts and Scater 1.8.4.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_scater</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/scater</td>
      <td>1.10.0</td>
      <td>scater-scripts</td>
      <td>0.0.5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>16</td>
      <td>387</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>scater</td>
      <td>scater_create_qcmetric_ready_sce, scater_filter, scater_plot_dist_scatter, scater_plot_pca, scater_plot_tsne</td>
      <td>Scater (Single-Cell Analysis Toolkit for gene Expression data in R) is acollection of tools for doing various analyses of single-cell RNA-seq geneexpression data, with a focus on quality control and visualization.</td>
      <td>scater</td>
      <td>scater</td>
      <td></td>
      <td>scater</td>
      <td>Pre-processing, quality control, normalization and visualization of single-cell RNA-seq data.</td>
      <td>Read pre-processing, Sequencing quality control, Sequence visualisation</td>
      <td>RNA-seq, Quality affairs, Molecular genetics</td>
      <td>To update</td>
      <td>http://bioconductor.org/packages/scater/</td>
      <td>Transcriptomics, RNA, Visualization</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/scater</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/scater</td>
      <td>1.22.0</td>
      <td>bioconductor-scater</td>
      <td>1.30.1</td>
      <td>Read pre-processing, Sequence visualisation</td>
      <td>Quality affairs, Molecular genetics</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>77</td>
      <td>1365</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>sccaf</td>
      <td>run_sccaf, sccaf_asses, sccaf_asses_merger, sccaf_regress_out</td>
      <td>SCCAF: Single Cell Clustering Assessment Framework.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/sccaf/sccaf</td>
      <td>Transcriptomics</td>
      <td>SCCAF</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/develop/tools/tertiary-analysis/sccaf</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/sccaf</td>
      <td>0.0.9</td>
      <td>sccaf</td>
      <td>0.0.10</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>110</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>sceasy</td>
      <td>sceasy_convert</td>
      <td>Converter between difference single-cell formats</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/cellgeni/sceasy/</td>
      <td>Transcriptomics</td>
      <td>sceasy_convert</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/sceasy/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/sceasy</td>
      <td>0.0.7</td>
      <td>r-sceasy</td>
      <td>0.0.7</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>90</td>
      <td>721</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>sceasy</td>
      <td>sceasy_convert</td>
      <td>Convert scRNA data object between popular formats</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics</td>
      <td>sceasy</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/sceasy</td>
      <td>0.0.5</td>
      <td>r-sceasy</td>
      <td>0.0.7</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>90</td>
      <td>721</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>schicexplorer</td>
      <td>schicexplorer_schicadjustmatrix, schicexplorer_schiccluster, schicexplorer_schicclustercompartments, schicexplorer_schicclusterminhash, schicexplorer_schicclustersvl, schicexplorer_schicconsensusmatrices, schicexplorer_schiccorrectmatrices, schicexplorer_schiccreatebulkmatrix, schicexplorer_schicdemultiplex, schicexplorer_schicinfo, schicexplorer_schicmergematrixbins, schicexplorer_schicmergetoscool, schicexplorer_schicnormalize, schicexplorer_schicplotclusterprofiles, schicexplorer_schicplotconsensusmatrices, schicexplorer_schicqualitycontrol</td>
      <td>scHiCExplorer: Set of programs to process, analyze and visualize scHi-C data.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/joachimwolff/schicexplorer</td>
      <td>Sequence Analysis, Transcriptomics, Visualization</td>
      <td>schicexplorer</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/schicexplorer</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/schicexplorer</td>
      <td>4</td>
      <td>schicexplorer</td>
      <td>7</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>16</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>16</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>16</td>
      <td>16</td>
      <td>23</td>
      <td>779</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>scmap</td>
      <td>scmap_get_std_output, scmap_index_cell, scmap_index_cluster, scmap_preprocess_sce, scmap_scmap_cell, scmap_scmap_cluster, scmap_select_features</td>
      <td>De-composed scmap functionality tools, based on https://github.com/ebi-gene-expression-group/scmap-cli and scmap 1.6.0.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_scmap</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/scmap</td>
      <td>1.6.4</td>
      <td>scmap-cli</td>
      <td>0.1.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>13</td>
      <td>83</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>scpipe</td>
      <td>scpipe</td>
      <td>A flexible preprocessing pipeline for single-cell RNA-sequencing data</td>
      <td>scpipe</td>
      <td>scpipe</td>
      <td></td>
      <td>scPipe</td>
      <td>A preprocessing pipeline for single cell RNA-seq data that starts from the fastq files and produces a gene count matrix with associated quality control information. It can process fastq data generated by CEL-seq, MARS-seq, Drop-seq, Chromium 10x and SMART-seq protocols.</td>
      <td>Genome annotation, Validation, Alignment, Visualisation</td>
      <td>Gene expression, RNA-Seq, Sequencing</td>
      <td>To update</td>
      <td>http://bioconductor.org/packages/release/bioc/html/scPipe.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>scpipe</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/scpipe</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/scpipe</td>
      <td>1.0.0+galaxy2</td>
      <td>bioconductor-scpipe</td>
      <td>2.2.0</td>
      <td>Genome annotation, Validation, Alignment, Visualisation</td>
      <td>Gene expression, RNA-Seq</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>11</td>
      <td>628</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>scpred</td>
      <td>scpred_get_feature_space, scpred_get_std_output, scpred_predict_labels, scpred_train_model</td>
      <td>De-composed scPred functionality tools, see https://github.com/ebi-gene-expression-group/scpred-cli and r-scPred 1.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_scpred</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/scpred</td>
      <td>1.0.2</td>
      <td>scpred-cli</td>
      <td>0.1.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>25</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>seurat</td>
      <td>seurat</td>
      <td>A toolkit for quality control, analysis, and exploration of single cell RNA sequencing data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/satijalab/seurat</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>seurat</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/seurat</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/seurat</td>
      <td>4.3.0.1</td>
      <td>r-seurat</td>
      <td>3.0.2</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>66</td>
      <td>1543</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>seurat</td>
      <td>seurat_convert, seurat_dim_plot, seurat_export_cellbrowser, seurat_filter_cells, seurat_find_clusters, seurat_find_markers, seurat_find_neighbours, seurat_find_variable_genes, seurat_hover_locator, seurat_integration, seurat_map_query, seurat_normalise_data, seurat_plot, seurat_read10x, seurat_run_pca, seurat_run_tsne, seurat_run_umap, seurat_scale_data, seurat_select_integration_features</td>
      <td>De-composed Seurat functionality tools, based on https://github.com/ebi-gene-expression-group/r-seurat-scripts and Seurat 2.3.1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_seurat</td>
      <td>ebi-gxa</td>
      <td></td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/seurat</td>
      <td>4.0.0</td>
      <td>seurat-scripts</td>
      <td>4.0.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>14</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>14</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>19</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>181</td>
      <td>1966</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>slamdunk</td>
      <td>alleyoop, slamdunk</td>
      <td>Slamdunk maps and quantifies SLAMseq reads</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://t-neumann.github.io/slamdunk</td>
      <td>RNA, Transcriptomics, Sequence Analysis, Next Gen Mappers</td>
      <td>slamdunk</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/slamdunk</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/slamdunk</td>
      <td>0.4.3</td>
      <td>slamdunk</td>
      <td>0.4.3</td>
      <td></td>
      <td></td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>361</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>sleuth</td>
      <td>sleuth</td>
      <td>Sleuth is a program for differential analysis of RNA-Seq data.</td>
      <td>sleuth</td>
      <td>sleuth</td>
      <td></td>
      <td>sleuth</td>
      <td>A statistical model and software application for RNA-seq differential expression analysis.</td>
      <td>Expression data visualisation, Differential gene expression analysis, Gene expression profiling, Statistical calculation</td>
      <td>RNA-seq, Gene expression, Statistics and probability</td>
      <td>Up-to-date</td>
      <td>https://github.com/pachterlab/sleuth</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>sleuth</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/sleuth</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/sleuth</td>
      <td>0.30.1</td>
      <td>r-sleuth</td>
      <td>0.30.1</td>
      <td>Expression data visualisation, Differential gene expression analysis, Gene expression profiling, Statistical calculation</td>
      <td>Gene expression, Statistics and probability</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>8</td>
      <td>64</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>star_fusion</td>
      <td>star_fusion</td>
      <td>STAR Fusion detects fusion genes in RNA-Seq data after running RNA-STAR</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Transcriptomics</td>
      <td>star_fusion</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/star_fusion</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/star_fusion</td>
      <td>0.5.4-3+galaxy1</td>
      <td>star-fusion</td>
      <td>1.13.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>35</td>
      <td>1212</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>stringtie</td>
      <td>stringtie, stringtie_merge</td>
      <td>StringTie is a fast and highly efficient assembler of RNA-Seq alignments into potential transcripts.</td>
      <td>stringtie</td>
      <td>stringtie</td>
      <td></td>
      <td>StringTie</td>
      <td>Fast and highly efficient assembler of RNA-Seq alignments into potential transcripts. It uses a novel network flow algorithm as well as an optional de novo assembly step to assemble and quantitate full-length transcripts representing multiple splice variants for each gene locus.</td>
      <td>Transcriptome assembly, RNA-Seq analysis</td>
      <td>Transcriptomics, RNA-seq</td>
      <td>Up-to-date</td>
      <td>http://ccb.jhu.edu/software/stringtie/</td>
      <td>Transcriptomics</td>
      <td>stringtie</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/stringtie</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/stringtie</td>
      <td>2.2.3</td>
      <td>stringtie</td>
      <td>2.2.3</td>
      <td>Transcriptome assembly, RNA-Seq analysis</td>
      <td>Transcriptomics</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2165</td>
      <td>167659</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>tophat_fusion_post</td>
      <td>tophat_fusion_post</td>
      <td>Wrapper for Tophat-Fusion post step</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics</td>
      <td>tophat_fusion_post</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/tophat_fusion_post</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/tophat_fusion_post</td>
      <td>0.1</td>
      <td>blast+</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>15</td>
      <td>216</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>transdecoder</td>
      <td>transdecoder</td>
      <td>TransDecoder finds coding regions within transcripts</td>
      <td>TransDecoder</td>
      <td>TransDecoder</td>
      <td></td>
      <td>TransDecoder</td>
      <td>TransDecoder identifies candidate coding regions within transcript sequences, such as those generated by de novo RNA-Seq transcript assembly using Trinity, or constructed based on RNA-Seq alignments to the genome using Tophat and Cufflinks.</td>
      <td>Coding region prediction, de Novo sequencing, De-novo assembly</td>
      <td>Genomics, Gene transcripts, RNA-Seq, Gene expression, Sequence assembly, Whole genome sequencing</td>
      <td>To update</td>
      <td>https://transdecoder.github.io/</td>
      <td>Transcriptomics, RNA</td>
      <td>transdecoder</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/transdecoder</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/transdecoder</td>
      <td>5.5.0</td>
      <td>transdecoder</td>
      <td>5.7.1</td>
      <td>Coding region prediction, de Novo sequencing, De-novo assembly</td>
      <td>Genomics, Gene transcripts, RNA-Seq, Gene expression, Sequence assembly, Whole genome sequencing</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>348</td>
      <td>5468</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>trinity</td>
      <td>trinity_abundance_estimates_to_matrix, trinity_align_and_estimate_abundance, trinity_analyze_diff_expr, trinity_contig_exn50_statistic, trinity_define_clusters_by_cutting_tree, describe_samples, trinity_filter_low_expr_transcripts, trinity_gene_to_trans_map, trinity_run_de_analysis, trinity_samples_qccheck, trinity_super_transcripts, trinity, trinity_stats</td>
      <td>Trinity represents a method for the efficient and robust de novo reconstruction of transcriptomes from RNA-seq datahttps://github.com/trinityrnaseq/trinityrnaseq</td>
      <td>trinity</td>
      <td>trinity</td>
      <td></td>
      <td>Trinity</td>
      <td>Trinity is a transcriptome assembler which relies on three different tools, inchworm an assembler, chrysalis which pools contigs and butterfly which amongst others compacts a graph resulting from butterfly with reads.</td>
      <td>Transcriptome assembly</td>
      <td>Transcriptomics, Gene expression, Gene transcripts</td>
      <td>Up-to-date</td>
      <td>https://github.com/trinityrnaseq/trinityrnaseq</td>
      <td>Transcriptomics, RNA</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/trinity</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/trinity</td>
      <td>2.15.1</td>
      <td>trinity</td>
      <td>2.15.1</td>
      <td>Transcriptome assembly</td>
      <td>Transcriptomics, Gene transcripts</td>
      <td>9</td>
      <td>13</td>
      <td>13</td>
      <td>13</td>
      <td>9</td>
      <td>13</td>
      <td>13</td>
      <td>13</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>2</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>13</td>
      <td>12</td>
      <td>1762</td>
      <td>31951</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>trinotate</td>
      <td>trinotate</td>
      <td>Trinotate is a comprehensive annotation suite designed for automatic functional annotation of de novo transcriptomes.</td>
      <td>trinotate</td>
      <td>trinotate</td>
      <td></td>
      <td>Trinotate</td>
      <td>Comprehensive annotation suite designed for automatic functional annotation of transcriptomes, particularly de novo assembled transcriptomes, from model or non-model organisms.</td>
      <td>Gene functional annotation</td>
      <td>Gene expression, Transcriptomics</td>
      <td>To update</td>
      <td>https://trinotate.github.io/</td>
      <td>Transcriptomics, RNA</td>
      <td>trinotate</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/trinotate</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/trinotate</td>
      <td>3.2.2</td>
      <td>trinotate</td>
      <td>4.0.2</td>
      <td>Gene functional annotation</td>
      <td>Transcriptomics</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>151</td>
      <td>1796</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>tximport</td>
      <td>tximport</td>
      <td>Wrapper for the Bioconductor package tximport</td>
      <td>tximport</td>
      <td>tximport</td>
      <td></td>
      <td>tximport</td>
      <td>An R/Bioconductor package that imports transcript-level abundance, estimated counts and transcript lengths, and summarizes into matrices for use with downstream gene-level analysis packages.</td>
      <td>Pathway or network analysis, Formatting, RNA-Seq analysis</td>
      <td>Transcriptomics, Gene transcripts, Workflows</td>
      <td>To update</td>
      <td>http://bioconductor.org/packages/tximport/</td>
      <td>Transcriptomics</td>
      <td>tximport</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/tximport</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/tximport</td>
      <td>1.22.0</td>
      <td>bioconductor-tximport</td>
      <td>1.30.0</td>
      <td>Pathway or network analysis, Formatting, RNA-Seq analysis</td>
      <td>Transcriptomics, Gene transcripts, Workflows</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>88</td>
      <td>1408</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>ucsc-cell-browser</td>
      <td>ucsc_cell_browser</td>
      <td>Python pipeline and Javascript scatter plot library for single-cell datasets</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://cells.ucsc.edu/</td>
      <td>Transcriptomics</td>
      <td>ucsc_cell_browser</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/develop/tools/tertiary-analysis/ucsc-cell-browser/.shed.yml</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/main/tools/tertiary-analysis/ucsc-cell-browser</td>
      <td>1.0.0+galaxy1</td>
      <td>ucsc-cell-browser</td>
      <td>1.2.6</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>78</td>
      <td>941</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>umi_tools</td>
      <td>umi_tools_count, umi_tools_dedup, umi_tools_extract, umi_tools_group, umi_tools_whitelist</td>
      <td>UMI-tools extract - Extract UMIs from fastq</td>
      <td>umi-tools</td>
      <td>umi-tools</td>
      <td></td>
      <td>UMI-tools</td>
      <td>Tools for handling Unique Molecular Identifiers in NGS data sets.</td>
      <td>Sequencing quality control</td>
      <td>NGS, Sequence sites, features and motifs, Quality affairs</td>
      <td>To update</td>
      <td>https://github.com/CGATOxford/UMI-tools</td>
      <td>Sequence Analysis, Transcriptomics</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/umi_tools</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/umi_tools</td>
      <td>1.1.2</td>
      <td>umi_tools</td>
      <td>1.1.5</td>
      <td>Sequencing quality control</td>
      <td>Sequence sites, features and motifs, Quality affairs</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>379</td>
      <td>61618</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>velocyto</td>
      <td>velocyto_cli</td>
      <td>Velocyto is a library for the analysis of RNA velocity.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://velocyto.org/</td>
      <td>Transcriptomics</td>
      <td>velocyto</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/velocyto</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/velocyto</td>
      <td>0.17.17</td>
      <td>velocyto.py</td>
      <td>0.17.17</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>volcanoplot</td>
      <td>volcanoplot</td>
      <td>Tool to create a Volcano Plot</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://ggplot2.tidyverse.org/</td>
      <td>Visualization, Transcriptomics, Statistics</td>
      <td>volcanoplot</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/volcanoplot</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/volcanoplot</td>
      <td>0.0.6</td>
      <td>r-ggplot2</td>
      <td>2.2.1</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1749</td>
      <td>30946</td>
      <td></td>
    </tr>
  </tbody>
</table>

</div>