<script src="https://code.jquery.com/jquery-3.7.0.js"></script>
<script src="https://cdn.datatables.net/1.13.6/js/jquery.dataTables.min.js"></script>
<script src="https://cdn.datatables.net/searchbuilder/1.6.0/js/dataTables.searchBuilder.min.js"></script>
<script src="https://cdn.datatables.net/datetime/1.5.1/js/dataTables.dateTime.min.js"></script>

<script src="https://cdn.datatables.net/responsive/2.5.0/js/dataTables.responsive.min.js"></script>

<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/responsive/2.2.3/css/responsive.dataTables.min.css">

<link rel="stylesheet" href="https://cdn.datatables.net/1.13.6/css/jquery.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/searchbuilder/1.6.0/css/searchBuilder.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/datetime/1.5.1/css/dataTables.dateTime.min.css">

<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" />
<script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>

<style type="text/css">
  .tooltip {
    font-family: Georgia;
    font-size: 18px;
  }
  .tooltip .tooltip-inner {
    background-color: #00FFFF;
    color: #000000;
    min-width: 400px;
  }
  </style>

<script>
    $(document).ready(function() {
    $('#dataframe').DataTable( {
        "sScrollX": "100%",
        "dom": "Qlfrtip",

        responsive: {
            details: {
                type: 'column'
            }
        },
        "bAutoWidth": false,
        columnDefs: [ {
            className: 'control',
            orderable: false,
            targets:   [ 0 ],
            width: 100,
        },
        { type: 'natural', targets: 1 } // allow normal sorting
        ],

    });
});
</script>

<div class="display nowrap" style="width:80%" width="100%">

<table class="dataframe display nowrap" id="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th>Expand</th>
      <th>Galaxy wrapper id</th>
      <th>Galaxy wrapper version</th>
      <th>Conda version</th>
      <th>Conda id</th>
      <th>Status</th>
      <th>bio.tool id</th>
      <th>bio.tool name</th>
      <th>EDAM operation</th>
      <th>EDAM topic</th>
      <th>Description</th>
      <th>bio.tool description</th>
      <th>biii</th>
      <th>Status</th>
      <th>Source</th>
      <th>ToolShed categories</th>
      <th>ToolShed id</th>
      <th>Galaxy wrapper owner</th>
      <th>Galaxy wrapper source</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td></td>
      <td>askor</td>
      <td>0.2</td>
      <td>3.58.1</td>
      <td>bioconductor-limma</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>AskoR links EdgeR and AskOmics</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/askomics/askoR</td>
      <td>Transcriptomics</td>
      <td>askor_de</td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/askor</td>
    </tr>
    <tr>
      <td></td>
      <td>baric_archive</td>
      <td>1.1.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A data source tool to fetch data from a BARIC Archive server.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.cesgo.org/catibaric/</td>
      <td>Data Source</td>
      <td></td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/baric_archive</td>
    </tr>
    <tr>
      <td></td>
      <td>braker</td>
      <td>2.1.6</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>BRAKER is a pipeline for fully automated prediction of protein coding gene structures with GeneMark-ES/ET and AUGUSTUS in novel eukaryotic genomes .</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/Gaius-Augustus/BRAKER</td>
      <td>Genome annotation</td>
      <td>braker</td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/braker</td>
    </tr>
    <tr>
      <td></td>
      <td>braker3</td>
      <td>3.0.8</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>BRAKER3 is a pipeline for fully automated prediction of protein coding gene structures with GeneMark-ES/ET and AUGUSTUS in novel eukaryotic genomes .</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/Gaius-Augustus/BRAKER</td>
      <td>Genome annotation</td>
      <td>braker3</td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/braker</td>
    </tr>
    <tr>
      <td></td>
      <td>feelnc2asko</td>
      <td>0.1</td>
      <td>1.7.8</td>
      <td>perl-bioperl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert FeelNC GTF to GFF3</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/tderrien/FEELnc</td>
      <td>Convert Formats</td>
      <td>feelnc2asko</td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/feelnc2asko</td>
    </tr>
    <tr>
      <td></td>
      <td>gcms2isocor</td>
      <td>0.1.0</td>
      <td></td>
      <td>openjdk</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Conversion from GCMS PostRun Analysis to Isocor</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Metabolomics</td>
      <td>gcms2isocor</td>
      <td>genouest</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>get_pairs</td>
      <td>0.3</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Separate paired and unpaired reads from two fastq files</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Fastq Manipulation</td>
      <td>get_pairs</td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/get_pairs</td>
    </tr>
    <tr>
      <td></td>
      <td>helixer</td>
      <td>0.3.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Gene calling with Deep Neural Networks</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/weberlab-hhu/Helixer</td>
      <td>Genome annotation</td>
      <td>helixer</td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/helixer</td>
    </tr>
    <tr>
      <td></td>
      <td>logol</td>
      <td>1.7.8</td>
      <td>1.7.8</td>
      <td>logol</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Logol is a pattern matching grammar language and a set of tools to search a pattern in a sequence</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://logol.genouest.org/web/app.php/logol</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/logol</td>
    </tr>
    <tr>
      <td></td>
      <td>meneco</td>
      <td>1.5.2</td>
      <td>1.5.2</td>
      <td>meneco</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Meneco computes minimal completions to your draft network with reactions from a repair network</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://bioasp.github.io/meneco/</td>
      <td>Systems Biology</td>
      <td>meneco</td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/meneco</td>
    </tr>
    <tr>
      <td></td>
      <td>miranda2asko</td>
      <td>0.2</td>
      <td></td>
      <td>perl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Converts miRanda output into AskOmics format</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Convert Formats</td>
      <td>miranda2asko</td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/miranda2asko</td>
    </tr>
    <tr>
      <td></td>
      <td>openlabcds2csv</td>
      <td>0.1.0</td>
      <td></td>
      <td>openjdk</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Creates a summary of several "Internal Standard Report" OpenLabCDS results.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Metabolomics</td>
      <td>openlabcds2csv</td>
      <td>genouest</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>peptimapper</td>
      <td>2.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Proteogenomics workflow for the expert annotation of eukaryotic genomes</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-019-5431-9</td>
      <td>Proteomics</td>
      <td></td>
      <td>genouest</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Ensembl-REST</td>
      <td>0.1.2</td>
      <td></td>
      <td>requests</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A suite of Galaxy tools designed to work with Ensembl REST API.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://rest.ensembl.org</td>
      <td>Data Source</td>
      <td></td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/Ensembl-REST</td>
    </tr>
    <tr>
      <td></td>
      <td>GAFA</td>
      <td>0.3.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Gene Align and Family Aggregator</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://aequatus.tgac.ac.uk</td>
      <td>Visualization</td>
      <td>gafa</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/GAFA/</td>
    </tr>
    <tr>
      <td></td>
      <td>TreeBest</td>
      <td>1.9.2.post0</td>
      <td>1.9.2.post1</td>
      <td>treebest</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>TreeBeST best</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://treesoft.sourceforge.net/treebest.shtml</td>
      <td>Phylogenetics</td>
      <td>treebest_best</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/TreeBest</td>
    </tr>
    <tr>
      <td></td>
      <td>apoc</td>
      <td>1.0+galaxy1</td>
      <td>1b16</td>
      <td>apoc</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Large-scale structural comparison of protein pockets</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://cssb.biology.gatech.edu/APoc</td>
      <td>Computational chemistry</td>
      <td>apoc</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/apoc/</td>
    </tr>
    <tr>
      <td></td>
      <td>blast_parser</td>
      <td>0.1.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert 12- or 24-column BLAST output into 3-column hcluster_sg input</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/TGAC/earlham-galaxytools/</td>
      <td>Phylogenetics</td>
      <td>blast_parser</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/blast_parser</td>
    </tr>
    <tr>
      <td></td>
      <td>ete</td>
      <td>3.1.2</td>
      <td>3.1.1</td>
      <td>ete3</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Analyse phylogenetic trees using the ETE Toolkit</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://etetoolkit.org/</td>
      <td>Phylogenetics</td>
      <td>ete</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/ete</td>
    </tr>
    <tr>
      <td></td>
      <td>export_to_cluster</td>
      <td>0.0.2</td>
      <td></td>
      <td>EXPORT_DIR_PREFIX</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Export datasets to cluster</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/TGAC/earlham-galaxytools/</td>
      <td>Data Export</td>
      <td>export_to_cluster</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/export_to_cluster/</td>
    </tr>
    <tr>
      <td></td>
      <td>gblocks</td>
      <td>0.91b</td>
      <td>0.91b</td>
      <td>gblocks</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Gblocks</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://molevol.cmima.csic.es/castresana/Gblocks.html</td>
      <td>Sequence Analysis</td>
      <td>gblocks</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/gblocks</td>
    </tr>
    <tr>
      <td></td>
      <td>gstf_preparation</td>
      <td>0.4.3</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>GeneSeqToFamily preparation converts data for the workflow</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/TGAC/earlham-galaxytools/</td>
      <td>Convert Formats</td>
      <td>gstf_preparation</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/gstf_preparation</td>
    </tr>
    <tr>
      <td></td>
      <td>hcluster_sg</td>
      <td>0.5.1.1</td>
      <td>0.5.1</td>
      <td>hcluster_sg</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Hierarchically clustering on a sparse graph</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/douglasgscofield/hcluster</td>
      <td>Phylogenetics</td>
      <td>hcluster_sg</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/hcluster_sg</td>
    </tr>
    <tr>
      <td></td>
      <td>hcluster_sg_parser</td>
      <td>0.2.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Converts hcluster_sg 3-column output into lists of ids</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/TGAC/earlham-galaxytools/</td>
      <td>Phylogenetics</td>
      <td>hcluster_sg_parser</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/hcluster_sg_parser</td>
    </tr>
    <tr>
      <td></td>
      <td>lotus2</td>
      <td>2.32</td>
      <td>2.32</td>
      <td>lotus2</td>
      <td>Up-to-date</td>
      <td>lotus2</td>
      <td>lotus2</td>
      <td>Sequence feature detection</td>
      <td>Metagenomics</td>
      <td>LotuS2 OTU processing pipeline</td>
      <td>LotuS2 is a lightweight and user-friendly pipeline that is fast, precise, and streamlined, using extensive pre- and post-ASV/OTU clustering steps to further increase data quality. High data usage rates and reliability enable high-throughput microbiome analysis in minutes.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://lotus2.earlham.ac.uk/</td>
      <td>Metagenomics</td>
      <td>lotus2</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/lotus2</td>
    </tr>
    <tr>
      <td></td>
      <td>miranda</td>
      <td>3.3a+galaxy1</td>
      <td>3.3a</td>
      <td>miranda</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Finds potential target sites for miRNAs in genomic sequences</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.microrna.org/</td>
      <td>RNA</td>
      <td>miranda</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/miranda</td>
    </tr>
    <tr>
      <td></td>
      <td>plotheatmap</td>
      <td>1.0</td>
      <td>1.64.0</td>
      <td>bioconductor-preprocesscore</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>This tool can be used to plot heatmap of gene expression data. The genes are chosen based on p-value, FDR, log FC and log CPM from edgeR output.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Computational chemistry</td>
      <td>plotheatmap</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/plotheatmap</td>
    </tr>
    <tr>
      <td></td>
      <td>rdock</td>
      <td>1.0</td>
      <td>2013.1</td>
      <td>rDock</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Docking ligands to proteins and nucleic acids</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://rdock.sourceforge.net/</td>
      <td>Computational chemistry</td>
      <td>rdock</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/rdock/</td>
    </tr>
    <tr>
      <td></td>
      <td>replace_chromosome_names</td>
      <td>0.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Replace chromosome names</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>replace_chromosome_names</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/replace_chromosome_names/</td>
    </tr>
    <tr>
      <td></td>
      <td>rsat_filter_snps</td>
      <td>0.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter SNPs in RSAT Matrix Scan output</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/TGAC/earlham-galaxytools/</td>
      <td>ChIP-seq, Systems Biology</td>
      <td>rsat_filter_snps</td>
      <td>earlham</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/rsat_filter_snps</td>
    </tr>
    <tr>
      <td></td>
      <td>smart_domains</td>
      <td>0.1.0</td>
      <td>1.7.8</td>
      <td>perl-bioperl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SMART domains</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://smart.embl.de/</td>
      <td>Sequence Analysis</td>
      <td>smart_domains</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/smart_domains</td>
    </tr>
    <tr>
      <td></td>
      <td>smina</td>
      <td>1.0</td>
      <td>2017.11.9</td>
      <td>smina</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>smina is a fork of Autodock Vina (http://vina.scripps.edu/) that focuses on improving scoring and minimization</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://sourceforge.net/projects/smina/</td>
      <td>Computational chemistry</td>
      <td>smina</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/smina/</td>
    </tr>
    <tr>
      <td></td>
      <td>t_coffee</td>
      <td>13.45.0.4846264</td>
      <td>13.46.0.919e8c6b</td>
      <td>t-coffee</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>T-Coffee</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.tcoffee.org/</td>
      <td>Sequence Analysis</td>
      <td>t_coffee</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/t_coffee</td>
    </tr>
    <tr>
      <td></td>
      <td>abacas</td>
      <td>1.1</td>
      <td>3.23</td>
      <td>mummer</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Order and Orientate Contigs</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/abacas</td>
      <td>Assembly</td>
      <td>abacas</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/abacas</td>
    </tr>
    <tr>
      <td></td>
      <td>assemblystats</td>
      <td>1.1.0</td>
      <td>1.7.8</td>
      <td>perl-bioperl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Summarise an assembly (e.g. N50 metrics)</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>Assembly</td>
      <td>assemblystats</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
    </tr>
    <tr>
      <td></td>
      <td>bam2mappingstats</td>
      <td>1.1.0</td>
      <td></td>
      <td>perl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Generates mapping stats from a bam file.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>Assembly</td>
      <td>bam2mappingstats</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
    </tr>
    <tr>
      <td></td>
      <td>bamclipper</td>
      <td>1.0.0</td>
      <td>1.0.0</td>
      <td>bamclipper</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Soft-clip gene-specific primers from BAM alignment file based on genomic coordinates of primer pairs in BEDPE format.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/tommyau/bamclipper</td>
      <td>Sequence Analysis</td>
      <td>bamclipper</td>
      <td>nml</td>
      <td>https://github.com/tommyau/bamclipper</td>
    </tr>
    <tr>
      <td></td>
      <td>biohansel</td>
      <td>2.4.0</td>
      <td>2.6.1</td>
      <td>bio_hansel</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Heidelberg and Enteritidis SNP Elucidation</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/biohansel</td>
      <td>Sequence Analysis</td>
      <td>biohansel</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/biohansel</td>
    </tr>
    <tr>
      <td></td>
      <td>biohansel_bionumeric_converter</td>
      <td>0.2.0</td>
      <td></td>
      <td>pandas</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert BioHansel output data to a Bionumerics friendly form</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>Text Manipulation</td>
      <td>biohansel_bionumeric_converter</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>bundle_collections</td>
      <td>1.3.0</td>
      <td>2.54</td>
      <td>perl-getopt-long</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tool to bundle up list collection into a single zip to be download</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>bundle_collections</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>collapse_collection</td>
      <td>5.1.0</td>
      <td></td>
      <td>gawk</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Collection tool that collapses a list of files into a single datasset in order of appears in collection</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>collapse_collections</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>combineJSON</td>
      <td>0.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>JSON collection tool that takes multiple JSON data arrays and combines them into a single JSON array.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>combine_json</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>combine_assembly_stats</td>
      <td>1.0</td>
      <td>2.54</td>
      <td>perl-getopt-long</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Combine multiple Assemblystats datasets into a single tabular report</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>Assembly</td>
      <td>combine_assemblystats</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
    </tr>
    <tr>
      <td></td>
      <td>combine_tabular_collection</td>
      <td>0.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Combine Tabular Collection into a single file</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>combine_tabular_collection</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>concat_paired</td>
      <td>0.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Concatenate paired datasets</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/concat</td>
      <td>Text Manipulation</td>
      <td>concat_paired</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/concat</td>
    </tr>
    <tr>
      <td></td>
      <td>cryptogenotyper</td>
      <td>1.0</td>
      <td>1.0</td>
      <td>cryptogenotyper</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>CryptoGenotyper is a standalone tool to *in-silico*  determine species and subtype based on SSU rRNA and gp60 markers.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/phac-nml/CryptoGenotyper</td>
      <td>Sequence Analysis</td>
      <td>cryptogenotyper</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/CryptoGenotyper</td>
    </tr>
    <tr>
      <td></td>
      <td>csvtk</td>
      <td>0.20.0</td>
      <td>0.29.0</td>
      <td>csvtk</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Rapid data investigation and manipulation of csv/tsv files</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bioinf.shenwei.me/csvtk/</td>
      <td>Text Manipulation</td>
      <td>csvtk</td>
      <td>nml</td>
      <td>https://github.com/shenwei356/csvtk</td>
    </tr>
    <tr>
      <td></td>
      <td>ectyper</td>
      <td>1.0.0</td>
      <td>1.0.0</td>
      <td>ectyper</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>EC-Typer - in silico serotyping of Escherichia coli species</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/phac-nml/ecoli_serotyping</td>
      <td>Sequence Analysis</td>
      <td>ectyper</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/ecoli_serotyping</td>
    </tr>
    <tr>
      <td></td>
      <td>fasta2bed</td>
      <td>1.0.0</td>
      <td>1.7.8</td>
      <td>perl-bioperl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert multiple fasta file into tabular bed file format</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>Sequence Analysis</td>
      <td>fasta2bed</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
    </tr>
    <tr>
      <td></td>
      <td>fasta_extract</td>
      <td>1.1.0</td>
      <td>1.7.8</td>
      <td>perl-bioperl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>extract single fasta from multiple fasta file</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://toolshed.g2.bx.psu.edu/view/nml/fasta_extract</td>
      <td>Sequence Analysis</td>
      <td>fasta_extract</td>
      <td>nml</td>
      <td>https://toolshed.g2.bx.psu.edu/view/nml/fasta_extract</td>
    </tr>
    <tr>
      <td></td>
      <td>fastqc_stats</td>
      <td>1.2</td>
      <td>1.7.8</td>
      <td>perl-bioperl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Summary multiple FastQC into a single tabular line report</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>Sequence Analysis</td>
      <td>fastqc_stats</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
    </tr>
    <tr>
      <td></td>
      <td>feht</td>
      <td>0.1.0</td>
      <td>1.1.0</td>
      <td>feht</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Automatically identify makers predictive of groups.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>Sequence Analysis</td>
      <td>feht</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
    </tr>
    <tr>
      <td></td>
      <td>filter_spades_repeats</td>
      <td>1.0.1</td>
      <td>1.7.8</td>
      <td>perl-bioperl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Remove short and repeat contigs/scaffolds</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/galaxy_tools/</td>
      <td>Assembly</td>
      <td>filter_spades_repeats</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools/</td>
    </tr>
    <tr>
      <td></td>
      <td>getmlst</td>
      <td>0.1.4.1</td>
      <td>0.2.0</td>
      <td>srst2</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Download MLST datasets by species from pubmlst.org</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>getmlst</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>gnali</td>
      <td>1.1.0</td>
      <td>1.1.0</td>
      <td>gnali</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A tool to find nonessential, loss-of-function gene variants</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/phac-nml/gnali/</td>
      <td>Variant Analysis</td>
      <td>gnali</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/gnali/</td>
    </tr>
    <tr>
      <td></td>
      <td>hivtrace</td>
      <td>1.0.1</td>
      <td>1.5.0</td>
      <td>hivtrace</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>An application that identifies potential transmission clusters within a supplied FASTA file with an option to find potential links against the Los Alamos HIV Sequence Database.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>hivtrace</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/tools/hivtrace</td>
    </tr>
    <tr>
      <td></td>
      <td>kaptive</td>
      <td>0.3.0</td>
      <td>2.0.6</td>
      <td>kaptive</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Kaptive reports information about capsular (K) loci found in genome assemblies.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>kaptive</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>kat_filter</td>
      <td>2.3</td>
      <td>2.4.2</td>
      <td>kat</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filtering kmers or reads from a database of kmers hashes</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>kat_filter</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>kat_sect</td>
      <td>2.3</td>
      <td>2.4.2</td>
      <td>kat</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SEquence Coverage estimator Tool. Estimates the coverage of each sequence in a file using K-mers from another sequence file.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td>kat_sect</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>mauve_contig_mover</td>
      <td>1.0.10</td>
      <td>2.4.0.r4736</td>
      <td>mauve</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Order a draft genome relative to a related reference genome</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/mauve_contig_mover</td>
      <td>Sequence Analysis</td>
      <td>mauve_contig_mover</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/mauve_contig_mover</td>
    </tr>
    <tr>
      <td></td>
      <td>mob_suite</td>
      <td>3.0.3</td>
      <td>3.1.8</td>
      <td>mob_suite</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MOB-suite is a set of software tools for clustering, reconstruction and typing of plasmids from draft assemblies</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/mob-suite</td>
      <td>Sequence Analysis</td>
      <td>mob_suite</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/mob-suite</td>
    </tr>
    <tr>
      <td></td>
      <td>mrbayes</td>
      <td>1.0.2</td>
      <td>3.2.7</td>
      <td>mrbayes</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A program for the Bayesian estimation of phylogeny.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>mrbayes</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>mykrobe_parser</td>
      <td>0.1.4.1</td>
      <td></td>
      <td>r-base</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>RScript to parse the results of mykrobe predictor.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/mykrobe-parser</td>
      <td>Sequence Analysis</td>
      <td>mykrobe_parser</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/mykrobe-parser</td>
    </tr>
    <tr>
      <td></td>
      <td>pangolin</td>
      <td>1.1.14</td>
      <td>4.3</td>
      <td>pangolin</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Phylogenetic Assignment of Named Global Outbreak LINeages</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/hCoV-2019/pangolin</td>
      <td>Sequence Analysis</td>
      <td>pangolin</td>
      <td>nml</td>
      <td>https://github.com/hCoV-2019/pangolin</td>
    </tr>
    <tr>
      <td></td>
      <td>patrist</td>
      <td>0.1.2</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Extract Patristic Distance From a Tree</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://gist.github.com/ArtPoon/7330231e74201ded54b87142a1d6cd02</td>
      <td>Phylogenetics</td>
      <td>patrist</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/patrist</td>
    </tr>
    <tr>
      <td></td>
      <td>plasmid_profiler</td>
      <td>0.1.6</td>
      <td></td>
      <td>r</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Explores plasmid content in WGS data</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td>plasmid_profiler</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>plasmid_profiler_suite</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Plasmid Profiler suite defining all dependencies for Plasmid Profiler</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>suite_plasmid_profiler</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>plasmidspades</td>
      <td>1.1</td>
      <td>3.15.5</td>
      <td>spades</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Genome assembler for assemblying plasmid</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Assembly</td>
      <td>plasmidspades</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>pneumocat</td>
      <td>1.2.1</td>
      <td>1.2.1</td>
      <td>pneumocat</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Pneumococcal Capsular Typing of illumina fastq reads</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/phe-bioinformatics/PneumoCaT</td>
      <td>Variant Analysis</td>
      <td>pneumocat</td>
      <td>nml</td>
      <td>https://github.com/phe-bioinformatics/PneumoCaT</td>
    </tr>
    <tr>
      <td></td>
      <td>promer</td>
      <td>1.2</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Aligns two sets of contigs and reports amino acid substitutions between them</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/promer</td>
      <td>Assembly</td>
      <td>promer</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/promer</td>
    </tr>
    <tr>
      <td></td>
      <td>pseudogenome</td>
      <td>1.0.0</td>
      <td>1.7.8</td>
      <td>perl-bioperl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Create a pseudogenome from a multiple fasta file either with a JCVI linker or custom length and characters.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>Sequence Analysis</td>
      <td>pseudogenome</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>quasitools</td>
      <td>0.7.0</td>
      <td>0.7.0</td>
      <td>quasitools</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A collection of tools for analysing Viral Quasispecies</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/phac-nml/quasitools</td>
      <td>Sequence Analysis</td>
      <td>quasitools</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/quasitools</td>
    </tr>
    <tr>
      <td></td>
      <td>refseq_masher</td>
      <td>0.1.2</td>
      <td>0.1.2</td>
      <td>refseq_masher</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Find what genomes match or are contained within your sequence data using Mash_ and a Mash sketch database.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/phac-nml/refseq_masher</td>
      <td>Sequence Analysis</td>
      <td>refseq_masher</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/refseq_masher</td>
    </tr>
    <tr>
      <td></td>
      <td>seqtk_nml</td>
      <td>1.0.1</td>
      <td>1.4</td>
      <td>seqtk</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tool to downsample fastq reads</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/lh3/seqtk</td>
      <td>Sequence Analysis</td>
      <td>seqtk_nml</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy</td>
    </tr>
    <tr>
      <td></td>
      <td>sistr_cmd</td>
      <td>1.1.1</td>
      <td>1.1.1</td>
      <td>sistr_cmd</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SISTR in silico serotyping tool</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/phac-nml/sistr_cmd</td>
      <td>Sequence Analysis</td>
      <td>sistr_cmd</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>smalt</td>
      <td>0.7.6</td>
      <td>0.7.6</td>
      <td>smalt</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SMALT aligns DNA sequencing reads with a reference genome.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://www.sanger.ac.uk/science/tools/smalt-0</td>
      <td>Sequence Analysis</td>
      <td>smalt</td>
      <td>nml</td>
      <td>https://sourceforge.net/projects/smalt/</td>
    </tr>
    <tr>
      <td></td>
      <td>spades_header_fixer</td>
      <td>1.1.2+galaxy1</td>
      <td></td>
      <td>sed</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Fixes Spades Fasta ids</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>Fasta Manipulation</td>
      <td>spades_fasta_header_fixer</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
    </tr>
    <tr>
      <td></td>
      <td>spatyper</td>
      <td>0.3.3</td>
      <td>0.3.3</td>
      <td>spatyper</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Determines SPA type based on repeats in a submitted staphylococcal protein A fasta file.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/HCGB-IGTP/spaTyper</td>
      <td>Sequence Analysis</td>
      <td>spatyper</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/spatyper</td>
    </tr>
    <tr>
      <td></td>
      <td>spolpred</td>
      <td>1.0.1</td>
      <td></td>
      <td>spolpred</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A program for predicting the spoligotype from raw sequence reads</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>spolpred</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>srst2</td>
      <td>0.3.7</td>
      <td>0.2.0</td>
      <td>srst2</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Short Read Sequence Typing for Bacterial Pathogens</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>srst2</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>staramr</td>
      <td>0.10.0</td>
      <td>0.10.0</td>
      <td>staramr</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Scan genome contigs against the ResFinder, PlasmidFinder, and PointFinder antimicrobial resistance databases.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/phac-nml/staramr</td>
      <td>Sequence Analysis</td>
      <td>staramr</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/staramr</td>
    </tr>
    <tr>
      <td></td>
      <td>stringmlst</td>
      <td>1.1.0</td>
      <td>0.6.3</td>
      <td>stringMLST</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Rapid and accurate identification of the sequence type (ST)</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>stringmlst</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>tree_relabeler</td>
      <td>1.1.0</td>
      <td>1.7.8</td>
      <td>perl-bioperl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Relabels the tips of a newick formatted tree.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>tree_relabeler</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools/blob/master/tools/tree_relabeler</td>
    </tr>
    <tr>
      <td></td>
      <td>wade</td>
      <td>0.2.5+galaxy1</td>
      <td>0.2.6</td>
      <td>wade</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>identify regions of interest</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/wade</td>
      <td>Sequence Analysis</td>
      <td>wade</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/wade</td>
    </tr>
    <tr>
      <td></td>
      <td>camera</td>
      <td>1.48.0</td>
      <td>0.4_1</td>
      <td>r-snow</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Metabolomics</td>
      <td>camera</td>
      <td>workflow4metabolomics</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>correlation_analysis</td>
      <td>1.0.1+galaxy0</td>
      <td>1.1_4</td>
      <td>r-batch</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>[Metabolomics][W4M] Metabolites Correlation Analysis</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://workflow4metabolomics.org</td>
      <td>Metabolomics</td>
      <td>correlation_analysis</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/blob/master/tools/correlation_analysis/</td>
    </tr>
    <tr>
      <td></td>
      <td>genform</td>
      <td></td>
      <td>r8</td>
      <td>genform</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>genform: generation of molecular formulas by high-resolution MS and MS/MS data</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://sourceforge.net/projects/genform/</td>
      <td>Metabolomics</td>
      <td>genform</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/blob/master/tools/genform/</td>
    </tr>
    <tr>
      <td></td>
      <td>influx_data_manager</td>
      <td>1.0.0</td>
      <td>1.0.0</td>
      <td>influx-si-data-manager</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Handling influx_si data inputs in Galaxy workflows</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/MetaboHUB-MetaToul-FluxoMet/tools_w4m/</td>
      <td>Metabolomics</td>
      <td>influx_si_data_manager</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/MetaboHUB-MetaToul-FluxoMet/tools_w4m/tree/main/tools/influx_data_manager</td>
    </tr>
    <tr>
      <td></td>
      <td>influx_si</td>
      <td>7.0.1</td>
      <td>7.0.2</td>
      <td>influx_si</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>metabolic flux estimation based on [in]stationary labeling</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/sgsokol/influx</td>
      <td>Metabolomics</td>
      <td>influx_si</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/blob/master/tools/influx_si/</td>
    </tr>
    <tr>
      <td></td>
      <td>ipo</td>
      <td>1.10.0</td>
      <td>1.28.0</td>
      <td>bioconductor-ipo</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>[W4M][LC-MS] IPO</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/rietho/IPO</td>
      <td>Metabolomics</td>
      <td>ipo</td>
      <td>lecorguille</td>
      <td>https://github.com/rietho/IPO</td>
    </tr>
    <tr>
      <td></td>
      <td>isoplot</td>
      <td>1.3.0+galaxy1</td>
      <td>1.3.1</td>
      <td>isoplot</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Isoplot is a software for the visualisation of MS data from C13 labelling experiments</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Metabolomics, Visualization</td>
      <td>isoplot</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/llegregam/Isoplot/tree/main</td>
    </tr>
    <tr>
      <td></td>
      <td>kmd_hmdb_data_plot</td>
      <td>1.0.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>retrieves data from KMD HMDB API and produce plot and csv file</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics</td>
      <td>Metabolomics</td>
      <td>kmd_hmdb_data_plot</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/blob/master/tools/kmd_hmdb_data_plot/</td>
    </tr>
    <tr>
      <td></td>
      <td>mixmodel4repeated_measures</td>
      <td>3.1.0</td>
      <td></td>
      <td>r-lme4</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>[Metabolomics][W4M][Statistics] Mixed models - Analysis of variance for repeated measures using mixed model</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://workflow4metabolomics.org</td>
      <td>Metabolomics</td>
      <td>mixmodel4repeated_measures</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics</td>
    </tr>
    <tr>
      <td></td>
      <td>ms2snoop</td>
      <td>2.2.1</td>
      <td></td>
      <td>r-base</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>[W4M][Utils] Extraction et nettoyage des spectre MS1/2 post-traitement MSPurity.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://workflow4metabolomics.org</td>
      <td>Metabolomics</td>
      <td>ms2snoop</td>
      <td>workflow4metabolomics</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>nmr_annotation</td>
      <td>3.0.0</td>
      <td>1.1_4</td>
      <td>r-batch</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>[Metabolomics][W4M][NMR] NMR Annotation - Annotation of complex mixture NMR spectra and metabolite proportion estimation</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://workflow4metabolomics.org</td>
      <td>Metabolomics</td>
      <td>nmr_annotation</td>
      <td>marie-tremblay-metatoul</td>
      <td>https://github.com/workflow4metabolomics/nmr_annotation</td>
    </tr>
    <tr>
      <td></td>
      <td>nmr_annotation2d</td>
      <td>2.0.0</td>
      <td>1.1_4</td>
      <td>r-batch</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>[Metabolomics][W4M][NMR] NMR Annotation2D - Automatic annotation of bi-dimensional NMR spectra</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://workflow4metabolomics.org</td>
      <td>Metabolomics</td>
      <td>2dnmrannotation</td>
      <td>marie-tremblay-metatoul</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics</td>
    </tr>
    <tr>
      <td></td>
      <td>nmr_preprocessing</td>
      <td></td>
      <td>1.1_4</td>
      <td>r-batch</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>[Metabolomics][W4M][NMR] NMR Preprocessing - Preprocessing of 1D NMR spectra from FID to baseline correction</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://workflow4metabolomics.org</td>
      <td>Metabolomics</td>
      <td>nmr_preprocessing</td>
      <td>marie-tremblay-metatoul</td>
      <td>https://github.com/workflow4metabolomics/nmr_preprocessing</td>
    </tr>
    <tr>
      <td></td>
      <td>normalization</td>
      <td>1.0.7</td>
      <td>1.1_4</td>
      <td>r-batch</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>[Metabolomics][W4M][ALL] Normalization (operation applied on each individual spectrum) of preprocessed data</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://workflow4metabolomics.org</td>
      <td>Metabolomics</td>
      <td>normalization</td>
      <td>marie-tremblay-metatoul</td>
      <td>https://github.com/workflow4metabolomics/normalization</td>
    </tr>
    <tr>
      <td></td>
      <td>physiofit</td>
      <td>3.3.2</td>
      <td>3.3.2</td>
      <td>physiofit</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>PhysioFit is a scientific tool designed to quantify cell growth parameters and uptake &amp; production fluxes</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>physiofit.readthedocs.io</td>
      <td>Metabolomics</td>
      <td>physiofit</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/MetaSys-LISBP/PhysioFit</td>
    </tr>
    <tr>
      <td></td>
      <td>physiofit_manager</td>
      <td>1.0.1</td>
      <td>1.0.1</td>
      <td>physiofit_data_manager</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Handling of physiofit input files</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/MetaboHUB-MetaToul-FluxoMet/PhysioFit_Data_Manager</td>
      <td>Metabolomics</td>
      <td>physiofit_manager</td>
      <td>workflow4metabolomics</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>xcms</td>
      <td>3.12.0</td>
      <td>4.0.0</td>
      <td>bioconductor-xcms</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/sneumann/xcms</td>
      <td>Metabolomics</td>
      <td>xcms</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/</td>
    </tr>
    <tr>
      <td></td>
      <td>apollo</td>
      <td></td>
      <td>4.2.13</td>
      <td>apollo</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Access an Apollo instance from Galaxy</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxy-genome-annotation/python-apollo</td>
      <td>Web Services</td>
      <td></td>
      <td>gga</td>
      <td>https://github.com/galaxy-genome-annotation/galaxy-tools/tree/master/tools/apollo</td>
    </tr>
    <tr>
      <td></td>
      <td>askomics</td>
      <td></td>
      <td>0.5</td>
      <td>askocli</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Galaxy tools allowing to interact with a remote AskOmics server.AskOmics is a visual SPARQL query builder for RDF database.https://github.com/askomics/</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/askomics/</td>
      <td>Web Services</td>
      <td></td>
      <td>gga</td>
      <td>https://github.com/galaxy-genome-annotation/galaxy-tools/tree/master/tools/askomics</td>
    </tr>
    <tr>
      <td></td>
      <td>chado</td>
      <td></td>
      <td>2.3.9</td>
      <td>python-chado</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Galaxy tools allowing to load data into a remote Chado database.Chado is a member of the GMOD family of tools.https://github.com/galaxy-genome-annotation/python-chado</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxy-genome-annotation/python-chado</td>
      <td>Web Services</td>
      <td></td>
      <td>gga</td>
      <td>https://github.com/galaxy-genome-annotation/galaxy-tools/tree/master/tools/chado</td>
    </tr>
    <tr>
      <td></td>
      <td>genenotebook</td>
      <td>0.4.13</td>
      <td>0.4.13</td>
      <td>genoboo</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Galaxy tools allowing to load data into a GeneNoteBook database.https://genenotebook.github.io</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://genenotebook.github.io</td>
      <td>Web Services</td>
      <td></td>
      <td>gga</td>
      <td>https://github.com/galaxy-genome-annotation/galaxy-tools/tree/master/tools/genenotebook</td>
    </tr>
    <tr>
      <td></td>
      <td>jbrowse</td>
      <td></td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A tool allowing to export a JBrowse dataset into a JBrowse docker container</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://jbrowse.org</td>
      <td>Web Services</td>
      <td>jbrowse_to_container</td>
      <td>gga</td>
      <td>https://github.com/galaxy-genome-annotation/galaxy-tools/tree/master/tools/jbrowse</td>
    </tr>
    <tr>
      <td></td>
      <td>repeatexplorer2</td>
      <td>2.3.8</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tool for annotation of repeats from unassembled shotgun reads.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/repeatexplorer/repex_tarean</td>
      <td>Genome annotation</td>
      <td>repeatexplorer2</td>
      <td>gga</td>
      <td>https://github.com/galaxy-genome-annotation/galaxy-tools/tree/master/tools/repeatexplorer2</td>
    </tr>
    <tr>
      <td></td>
      <td>tripal</td>
      <td></td>
      <td>3.2.1</td>
      <td>python-tripal</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Galaxy tools allowing to load data into a remote Tripal server.Tripal is a toolkit for construction of online biological (genetics, genomics, breeding, etc), community database,and is a member of the GMOD family of tools. Tripal provides by default integration with the GMOD Chado database schema and Drupal, a popular Content Management Systems (CMS).https://github.com/galaxy-genome-annotation/python-tripal</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxy-genome-annotation/python-tripal</td>
      <td>Web Services</td>
      <td></td>
      <td>gga</td>
      <td>https://github.com/galaxy-genome-annotation/galaxy-tools/tree/master/tools/tripal</td>
    </tr>
    <tr>
      <td></td>
      <td>w4mcorcov</td>
      <td>0.98.18</td>
      <td></td>
      <td>r-base</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>OPLS-DA Contrasts of Univariate Results</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/HegemanLab/w4mcorcov_galaxy_wrapper</td>
      <td>Metabolomics</td>
      <td>w4mcorcov</td>
      <td>eschen42</td>
      <td>https://github.com/HegemanLab/w4mcorcov_galaxy_wrapper/tree/master</td>
    </tr>
    <tr>
      <td></td>
      <td>w4mclassfilter</td>
      <td>0.98.19</td>
      <td></td>
      <td>r-base</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter W4M data by values or metadata</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/HegemanLab/w4mclassfilter_galaxy_wrapper</td>
      <td>Metabolomics</td>
      <td>w4mclassfilter</td>
      <td>eschen42</td>
      <td>https://github.com/HegemanLab/w4mclassfilter_galaxy_wrapper/tree/master</td>
    </tr>
    <tr>
      <td></td>
      <td>w4mjoinpn</td>
      <td>0.98.2</td>
      <td>8.25</td>
      <td>coreutils</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Join positive- and negative-mode W4M datasets</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/HegemanLab/w4mjoinpn_galaxy_wrapper</td>
      <td>Metabolomics</td>
      <td>w4mjoinpn</td>
      <td>eschen42</td>
      <td>https://github.com/HegemanLab/w4mjoinpn_galaxy_wrapper/tree/master</td>
    </tr>
    <tr>
      <td></td>
      <td>2d_auto_threshold</td>
      <td>0.0.5-2</td>
      <td></td>
      <td>scikit-image</td>
      <td>To update</td>
      <td>scikit-image</td>
      <td>scikit-image</td>
      <td>Image analysis, Image annotation, Visualisation, Data handling</td>
      <td>Imaging, Software engineering, Literature and language</td>
      <td>Automatic thresholding</td>
      <td>Scikit-image contains image processing algorithms for SciPy, including IO, morphology, filtering, warping, color manipulation, object detection, etc.</td>
      <td>scikit-image</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>2d_auto_threshold</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/2d_auto_threshold/</td>
    </tr>
    <tr>
      <td></td>
      <td>2d_feature_extraction</td>
      <td>0.1.1-2</td>
      <td></td>
      <td>pandas</td>
      <td>To update</td>
      <td>scikit-image</td>
      <td>scikit-image</td>
      <td>Image analysis, Image annotation, Visualisation, Data handling</td>
      <td>Imaging, Software engineering, Literature and language</td>
      <td>2D feature extraction</td>
      <td>Scikit-image contains image processing algorithms for SciPy, including IO, morphology, filtering, warping, color manipulation, object detection, etc.</td>
      <td>scikit-image</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>2d_feature_extraction</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/2d_feature_extraction/</td>
    </tr>
    <tr>
      <td></td>
      <td>2d_filter_segmentation_by_features</td>
      <td>0.0.1-2</td>
      <td></td>
      <td>scikit-image</td>
      <td>To update</td>
      <td>galaxy_image_analysis</td>
      <td>Galaxy Image Analysis</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>filter segmentation by rules</td>
      <td>Developed within the Biomedical Computer Vision (BMCV) Group Heidelberg.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>2d_filter_segmentation_by_features</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/2d_filter_segmentation_by_features/</td>
    </tr>
    <tr>
      <td></td>
      <td>2d_histogram_equalization</td>
      <td>0.0.1-2</td>
      <td></td>
      <td>scikit-image</td>
      <td>To update</td>
      <td>galaxy_image_analysis</td>
      <td>Galaxy Image Analysis</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>2d histogram equalization</td>
      <td>Developed within the Biomedical Computer Vision (BMCV) Group Heidelberg.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>2d_histogram_equalization</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/2d_histogram_equalization/</td>
    </tr>
    <tr>
      <td></td>
      <td>2d_simple_filter</td>
      <td>0.0.3-3</td>
      <td></td>
      <td>scikit-image</td>
      <td>To update</td>
      <td>scikit-image</td>
      <td>scikit-image</td>
      <td>Image analysis, Image annotation, Visualisation, Data handling</td>
      <td>Imaging, Software engineering, Literature and language</td>
      <td>2d simple filter</td>
      <td>Scikit-image contains image processing algorithms for SciPy, including IO, morphology, filtering, warping, color manipulation, object detection, etc.</td>
      <td>scikit-image</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>2d_simple_filter</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/2d_simple_filter/</td>
    </tr>
    <tr>
      <td></td>
      <td>3d_tensor_feature_dimension_reduction</td>
      <td>0.0.1</td>
      <td></td>
      <td>numpy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Dimensionality reduction for features (channels) of 3D tensor data using UMAP</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/BMCV/galaxy-image-analysis</td>
      <td>Imaging</td>
      <td>3d_tensor_feature_dimension_reduction</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/3d_tensor_feature_dimension_reduction/</td>
    </tr>
    <tr>
      <td></td>
      <td>anisotropic_diffusion</td>
      <td>0.2-2</td>
      <td></td>
      <td>scikit-image</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Anisotropic image diffusion</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>anisotropic_diffusion</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/anisotropic-diffusion/</td>
    </tr>
    <tr>
      <td></td>
      <td>bfconvert</td>
      <td>6.7.0+galaxy2</td>
      <td>6.7.0</td>
      <td>bftools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert image</td>
      <td></td>
      <td>python-bioformats</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging, Convert Formats</td>
      <td>bfconvert</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/bfconvert/</td>
    </tr>
    <tr>
      <td></td>
      <td>binary2labelimage</td>
      <td>0.5</td>
      <td></td>
      <td>scikit-image</td>
      <td>To update</td>
      <td>galaxy_image_analysis</td>
      <td>Galaxy Image Analysis</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>Binary 2 label image</td>
      <td>Developed within the Biomedical Computer Vision (BMCV) Group Heidelberg.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>binary2labelimage</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/binary2labelimage/</td>
    </tr>
    <tr>
      <td></td>
      <td>binaryimage2points</td>
      <td>0.1-2</td>
      <td></td>
      <td>numpy</td>
      <td>To update</td>
      <td>galaxy_image_analysis</td>
      <td>Galaxy Image Analysis</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>Binary Image to Points</td>
      <td>Developed within the Biomedical Computer Vision (BMCV) Group Heidelberg.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>binaryimage2points</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/binaryimage2points/</td>
    </tr>
    <tr>
      <td></td>
      <td>bioformats2raw</td>
      <td>0.7.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert image to OME-Zarr</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/Euro-BioImaging</td>
      <td>Imaging, Convert Formats</td>
      <td>bioformats2raw</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/bioformats2raw</td>
    </tr>
    <tr>
      <td></td>
      <td>color-deconvolution</td>
      <td>0.8-2</td>
      <td></td>
      <td>scikit-image</td>
      <td>To update</td>
      <td>galaxy_image_analysis</td>
      <td>Galaxy Image Analysis</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>Color-deconvolution methods</td>
      <td>Developed within the Biomedical Computer Vision (BMCV) Group Heidelberg.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>color_deconvolution</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/color-deconvolution/</td>
    </tr>
    <tr>
      <td></td>
      <td>concat_channels</td>
      <td>0.2-2</td>
      <td></td>
      <td>scikit-image</td>
      <td>To update</td>
      <td>galaxy_image_analysis</td>
      <td>Galaxy Image Analysis</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>Concatenate images</td>
      <td>Developed within the Biomedical Computer Vision (BMCV) Group Heidelberg.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>concat_channels</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/concat_channels/</td>
    </tr>
    <tr>
      <td></td>
      <td>coordinates_of_roi</td>
      <td>0.0.4-2</td>
      <td></td>
      <td>scikit-image</td>
      <td>To update</td>
      <td>galaxy_image_analysis</td>
      <td>Galaxy Image Analysis</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>Coordinates of ROI</td>
      <td>Developed within the Biomedical Computer Vision (BMCV) Group Heidelberg.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>coordinates_of_roi</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/coordinates_of_roi/</td>
    </tr>
    <tr>
      <td></td>
      <td>count_objects</td>
      <td>0.0.5-2</td>
      <td></td>
      <td>scikit-image</td>
      <td>To update</td>
      <td>galaxy_image_analysis</td>
      <td>Galaxy Image Analysis</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>Count Objects</td>
      <td>Developed within the Biomedical Computer Vision (BMCV) Group Heidelberg.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>count_objects</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/count_objects/</td>
    </tr>
    <tr>
      <td></td>
      <td>curl_post</td>
      <td>0.0.2</td>
      <td></td>
      <td>curl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Send file via cURL POST</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Data Export, Web Services</td>
      <td>curl_post</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/curl_post</td>
    </tr>
    <tr>
      <td></td>
      <td>curve_fitting</td>
      <td>0.0.3-2</td>
      <td></td>
      <td>numpy</td>
      <td>To update</td>
      <td>galaxy_image_analysis</td>
      <td>Galaxy Image Analysis</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>Polynomial curve fitting to data points</td>
      <td>Developed within the Biomedical Computer Vision (BMCV) Group Heidelberg.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/BMCV/galaxy-image-analysis</td>
      <td>Imaging</td>
      <td>curve_fitting</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/curve_fitting/</td>
    </tr>
    <tr>
      <td></td>
      <td>detection_viz</td>
      <td>0.3-2</td>
      <td></td>
      <td>scikit-image</td>
      <td>To update</td>
      <td>galaxy_image_analysis</td>
      <td>Galaxy Image Analysis</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>Detection Visualization</td>
      <td>Developed within the Biomedical Computer Vision (BMCV) Group Heidelberg.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>detection_viz</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/detection_viz/</td>
    </tr>
    <tr>
      <td></td>
      <td>image_info</td>
      <td>5.7.1</td>
      <td>6.7.0</td>
      <td>bftools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Extracts image metadata</td>
      <td></td>
      <td>python-bioformats</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>image_info</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/image_info/</td>
    </tr>
    <tr>
      <td></td>
      <td>image_registration_affine</td>
      <td>0.0.3-2</td>
      <td></td>
      <td>scikit-image</td>
      <td>To update</td>
      <td>galaxy_image_analysis</td>
      <td>Galaxy Image Analysis</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>Intensity-based Image Registration</td>
      <td>Developed within the Biomedical Computer Vision (BMCV) Group Heidelberg.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>image_registration_affine</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/image_registration_affine/</td>
    </tr>
    <tr>
      <td></td>
      <td>imagecoordinates_flipaxis</td>
      <td>0.1-2</td>
      <td></td>
      <td>pandas</td>
      <td>To update</td>
      <td>galaxy_image_analysis</td>
      <td>Galaxy Image Analysis</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>Flip coordinate axes</td>
      <td>Developed within the Biomedical Computer Vision (BMCV) Group Heidelberg.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>imagecoordinates_flipaxis</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/imagecoordinates_flipaxis/</td>
    </tr>
    <tr>
      <td></td>
      <td>labelimage2points</td>
      <td>0.2-2</td>
      <td></td>
      <td>scikit-image</td>
      <td>To update</td>
      <td>galaxy_image_analysis</td>
      <td>Galaxy Image Analysis</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>Label Image to Points</td>
      <td>Developed within the Biomedical Computer Vision (BMCV) Group Heidelberg.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>labelimage2points</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/labelimage2points/</td>
    </tr>
    <tr>
      <td></td>
      <td>landmark_registration</td>
      <td>0.1.0-2</td>
      <td></td>
      <td>scikit-image</td>
      <td>To update</td>
      <td>galaxy_image_analysis</td>
      <td>Galaxy Image Analysis</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>Landmark Registration</td>
      <td>Developed within the Biomedical Computer Vision (BMCV) Group Heidelberg.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>landmark_registration</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/landmark_registration/</td>
    </tr>
    <tr>
      <td></td>
      <td>mahotas-features</td>
      <td>0.7-2</td>
      <td></td>
      <td>mahotas</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute image features using mahotas.</td>
      <td></td>
      <td>mahotas-feature-computation</td>
      <td>To update</td>
      <td>https://github.com/luispedro/mahotas</td>
      <td>Imaging</td>
      <td>mahotas_features</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/mahotas-features/</td>
    </tr>
    <tr>
      <td></td>
      <td>mergeneighboursinlabelimage</td>
      <td>0.3-2</td>
      <td></td>
      <td>scikit-image</td>
      <td>To update</td>
      <td>galaxy_image_analysis</td>
      <td>Galaxy Image Analysis</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>Merge Neighbours in Label Image</td>
      <td>Developed within the Biomedical Computer Vision (BMCV) Group Heidelberg.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>mergeneighboursinlabelimage</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/\mergeneighboursinlabelimage</td>
    </tr>
    <tr>
      <td></td>
      <td>overlay_images</td>
      <td>0.0.4</td>
      <td></td>
      <td>scikit-image</td>
      <td>To update</td>
      <td>galaxy_image_analysis</td>
      <td>Galaxy Image Analysis</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>Overlay two images</td>
      <td>Developed within the Biomedical Computer Vision (BMCV) Group Heidelberg.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/BMCV/galaxy-image-analysis</td>
      <td>Imaging</td>
      <td>overlay_images</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/overlay_images/</td>
    </tr>
    <tr>
      <td></td>
      <td>permutate_axis</td>
      <td>0.2-2</td>
      <td></td>
      <td>scikit-image</td>
      <td>To update</td>
      <td>galaxy_image_analysis</td>
      <td>Galaxy Image Analysis</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>Permutates axes</td>
      <td>Developed within the Biomedical Computer Vision (BMCV) Group Heidelberg.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>permutate_axis</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/permutate_axis/</td>
    </tr>
    <tr>
      <td></td>
      <td>points2binaryimage</td>
      <td>0.2-1</td>
      <td></td>
      <td>scikit-image</td>
      <td>To update</td>
      <td>galaxy_image_analysis</td>
      <td>Galaxy Image Analysis</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>Points to Binary Image</td>
      <td>Developed within the Biomedical Computer Vision (BMCV) Group Heidelberg.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>points2binaryimage</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/points2binaryimage/</td>
    </tr>
    <tr>
      <td></td>
      <td>points2labelimage</td>
      <td>0.3-2</td>
      <td></td>
      <td>numpy</td>
      <td>To update</td>
      <td>galaxy_image_analysis</td>
      <td>Galaxy Image Analysis</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>Points to label image</td>
      <td>Developed within the Biomedical Computer Vision (BMCV) Group Heidelberg.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>points2labelimage</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/points2labelimage/</td>
    </tr>
    <tr>
      <td></td>
      <td>points_association_nn</td>
      <td>0.0.3-2</td>
      <td></td>
      <td>numpy</td>
      <td>To update</td>
      <td>galaxy_image_analysis</td>
      <td>Galaxy Image Analysis</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>Association of points in consecutive frames</td>
      <td>Developed within the Biomedical Computer Vision (BMCV) Group Heidelberg.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/BMCV/galaxy-image-analysis</td>
      <td>Imaging</td>
      <td>points_association_nn</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/points_association_nn/</td>
    </tr>
    <tr>
      <td></td>
      <td>projective_transformation</td>
      <td>0.1.2-2</td>
      <td></td>
      <td>scikit-image</td>
      <td>To update</td>
      <td>galaxy_image_analysis</td>
      <td>Galaxy Image Analysis</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>Projective transformation</td>
      <td>Developed within the Biomedical Computer Vision (BMCV) Group Heidelberg.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>projective_transformation</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/projective_transformation/</td>
    </tr>
    <tr>
      <td></td>
      <td>projective_transformation_points</td>
      <td>0.1.1-2</td>
      <td></td>
      <td>scikit-image</td>
      <td>To update</td>
      <td>galaxy_image_analysis</td>
      <td>Galaxy Image Analysis</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>Projective transformation of ROIs defined by pixel (point) coordinates</td>
      <td>Developed within the Biomedical Computer Vision (BMCV) Group Heidelberg.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>projective_transformation_points</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/projective_transformation_points/</td>
    </tr>
    <tr>
      <td></td>
      <td>rfove</td>
      <td>2023.11.12</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>rfove</td>
      <td>RFOVE</td>
      <td>Image analysis</td>
      <td>Cell biology, Biomedical science, Imaging</td>
      <td>Perform segmentation region-based fitting of overlapping ellipses</td>
      <td>RFOVE (Region-based Fitting of Overlapping Ellipses and its Application to Cells Segmentation) is a MATLAB script for performing image segmentation on cells.</td>
      <td></td>
      <td>To update</td>
      <td>https://sites.google.com/site/costaspanagiotakis/research/cs</td>
      <td>Imaging</td>
      <td>rfove</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/rfove/</td>
    </tr>
    <tr>
      <td></td>
      <td>scale_image</td>
      <td>0.4-2</td>
      <td></td>
      <td>pillow</td>
      <td>To update</td>
      <td>scikit-image</td>
      <td>scikit-image</td>
      <td>Image analysis, Image annotation, Visualisation, Data handling</td>
      <td>Imaging, Software engineering, Literature and language</td>
      <td>Scale image</td>
      <td>Scikit-image contains image processing algorithms for SciPy, including IO, morphology, filtering, warping, color manipulation, object detection, etc.</td>
      <td>scikit-image</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>scale_image</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/scale_image/</td>
    </tr>
    <tr>
      <td></td>
      <td>segmetrics</td>
      <td>1.4</td>
      <td>1.4.1</td>
      <td>segmetrics</td>
      <td>To update</td>
      <td>segmetrics</td>
      <td>SegMetrics</td>
      <td>Image analysis</td>
      <td></td>
      <td>Image segmentation and object detection performance measures</td>
      <td>Image segmentation and object detection performance measures</td>
      <td>segmetrics</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>segmetrics</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/segmetrics/</td>
    </tr>
    <tr>
      <td></td>
      <td>slice_image</td>
      <td>0.3-2</td>
      <td></td>
      <td>scikit-image</td>
      <td>To update</td>
      <td>galaxy_image_analysis</td>
      <td>Galaxy Image Analysis</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>Slice image</td>
      <td>Developed within the Biomedical Computer Vision (BMCV) Group Heidelberg.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>slice_image</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/slice_image/</td>
    </tr>
    <tr>
      <td></td>
      <td>split_labelmap</td>
      <td>0.2-2</td>
      <td></td>
      <td>scikit-image</td>
      <td>To update</td>
      <td>galaxy_image_analysis</td>
      <td>Galaxy Image Analysis</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>Split Labelmaps</td>
      <td>Developed within the Biomedical Computer Vision (BMCV) Group Heidelberg.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>split_labelmap</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/split_labelmaps/</td>
    </tr>
    <tr>
      <td></td>
      <td>spot_detection_2d</td>
      <td>0.0.3-2</td>
      <td></td>
      <td>imageio</td>
      <td>To update</td>
      <td>galaxy_image_analysis</td>
      <td>Galaxy Image Analysis</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>Spot detection in 2D image sequence</td>
      <td>Developed within the Biomedical Computer Vision (BMCV) Group Heidelberg.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/BMCV/galaxy-image-analysis</td>
      <td>Imaging</td>
      <td>spot_detection_2d</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/spot_detection_2d/</td>
    </tr>
    <tr>
      <td></td>
      <td>superdsm</td>
      <td>0.2.0</td>
      <td>0.2.0</td>
      <td>superdsm</td>
      <td>Up-to-date</td>
      <td>superdsm</td>
      <td>SuperDSM</td>
      <td>Image analysis</td>
      <td></td>
      <td>Globally optimal segmentation method based on superadditivity and deformable shape models for cell nuclei in fluorescence microscopy images</td>
      <td>SuperDSM is a globally optimal segmentation method based on superadditivity and deformable shape models for cell nuclei in fluorescence microscopy images and beyond.</td>
      <td>superdsm</td>
      <td>Up-to-date</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>superdsm</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/superdsm/</td>
    </tr>
    <tr>
      <td></td>
      <td>unzip</td>
      <td>6.0</td>
      <td></td>
      <td>unzip</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Unzip file</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Convert Formats</td>
      <td>unzip</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/unzip/</td>
    </tr>
    <tr>
      <td></td>
      <td>visceral-evaluatesegmentation</td>
      <td>0.5-2</td>
      <td>2015.07.03</td>
      <td>visceral-evaluatesegmentation</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Visceral Project - Evaluate Segmentation Tool</td>
      <td></td>
      <td>evaluatesegmentation-tool</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>visceral_evaluatesegmentation</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/visceral-evaluatesegmentation</td>
    </tr>
    <tr>
      <td></td>
      <td>wsi_extract_top_view</td>
      <td>0.2-2</td>
      <td></td>
      <td>scikit-image</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>WSI Extract Top View</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>wsi_extract_top_view</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/wsi_extract_top_view/</td>
    </tr>
    <tr>
      <td></td>
      <td>add_value</td>
      <td>1.0.1</td>
      <td></td>
      <td>perl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Add a value as a new column.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>add_value</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/add_value</td>
    </tr>
    <tr>
      <td></td>
      <td>annotation_profiler</td>
      <td>1.0.0</td>
      <td>0.11.0</td>
      <td>bx-python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Profile Annotations for a set of genomic intervals</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Genomic Interval Operations</td>
      <td>annotation_profiler</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/annotation_profiler</td>
    </tr>
    <tr>
      <td></td>
      <td>best_regression_subsets</td>
      <td>1.0.0</td>
      <td></td>
      <td>numpy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Perform Best-subsets Regression</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>best_regression_subsets</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/best_regression_subsets</td>
    </tr>
    <tr>
      <td></td>
      <td>blat_coverage_report</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Polymorphism of the Reads</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Next Gen Mappers, Sequence Analysis</td>
      <td>blat_coverage_report</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/blat_coverage_report</td>
    </tr>
    <tr>
      <td></td>
      <td>blat_mapping</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Coverage of the Reads in wiggle format</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Next Gen Mappers, Sequence Analysis</td>
      <td>blat_mapping</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/blat_mapping</td>
    </tr>
    <tr>
      <td></td>
      <td>bowtie_wrappers</td>
      <td>1.2.0</td>
      <td>1.3.1</td>
      <td>bowtie</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Galaxy wrappers for the Bowtie short read mapping tools.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://bowtie-bio.sourceforge.net/</td>
      <td>Next Gen Mappers</td>
      <td>bowtie_wrappers</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/bowtie_wrappers</td>
    </tr>
    <tr>
      <td></td>
      <td>canonical_correlation_analysis</td>
      <td>1.0.0</td>
      <td></td>
      <td>R</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Canonical Correlation Analysis</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>canonical_correlation_analysis</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/canonical_correlation_analysis</td>
    </tr>
    <tr>
      <td></td>
      <td>categorize_elements_satisfying_criteria</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Categorize Elements satisfying criteria.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>categorize_elements_satisfying_criteria</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/categorize_elements_satisfying_criteria</td>
    </tr>
    <tr>
      <td></td>
      <td>ccat</td>
      <td>0.0.2</td>
      <td>3.0</td>
      <td>ccat</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Control-based ChIP-seq Analysis Tool</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>ChIP-seq</td>
      <td>ccat</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/ccat</td>
    </tr>
    <tr>
      <td></td>
      <td>cd_hit_dup</td>
      <td>0.0.1</td>
      <td>4.8.1</td>
      <td>cd-hit-auxtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>simple tool for removing duplicates from sequencing reads</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Metagenomics, Sequence Analysis</td>
      <td>cd_hit_dup</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/cd_hit_dup</td>
    </tr>
    <tr>
      <td></td>
      <td>change_case</td>
      <td>1.0.1</td>
      <td></td>
      <td>perl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert column case.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>change_case</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/change_case</td>
    </tr>
    <tr>
      <td></td>
      <td>compute_motif_frequencies_for_all_motifs</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute Motif Frequencies For All Motifs, motif by motif.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Statistics</td>
      <td>compute_motif_frequencies_for_all_motifs</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/compute_motif_frequencies_for_all_motifs</td>
    </tr>
    <tr>
      <td></td>
      <td>compute_motifs_frequency</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute Motif Frequencies in indel flanking regions.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Statistics</td>
      <td>compute_motifs_frequency</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/compute_motifs_frequency</td>
    </tr>
    <tr>
      <td></td>
      <td>compute_q_values</td>
      <td>1.0.1</td>
      <td></td>
      <td>R</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute q-values based on multiple simultaneous tests p-values</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>compute_q_values</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/compute_q_values</td>
    </tr>
    <tr>
      <td></td>
      <td>condense_characters</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Condense consecutive characters.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>condense_characters</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/condense_characters</td>
    </tr>
    <tr>
      <td></td>
      <td>convert_characters</td>
      <td>1.0.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert delimiters to tab.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>convert_characters</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/convert_characters</td>
    </tr>
    <tr>
      <td></td>
      <td>convert_solid_color2nuc</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert Color Space to Nucleotides</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Fasta Manipulation</td>
      <td>convert_solid_color2nuc</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/convert_solid_color2nuc</td>
    </tr>
    <tr>
      <td></td>
      <td>correlation</td>
      <td>1.0.0</td>
      <td></td>
      <td>rpy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Correlation for numeric columns</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>correlation</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/correlation</td>
    </tr>
    <tr>
      <td></td>
      <td>count_gff_features</td>
      <td>0.2</td>
      <td>1.1.0</td>
      <td>galaxy-ops</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Count GFF Features</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>count_gff_features</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/count_gff_features</td>
    </tr>
    <tr>
      <td></td>
      <td>ctd_batch</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>CTD analysis of chemicals, diseases, or genes</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>ctd_batch</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/ctd_batch</td>
    </tr>
    <tr>
      <td></td>
      <td>cummerbund</td>
      <td>2.16.0</td>
      <td></td>
      <td>fonts-conda-ecosystem</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Wrapper for the Bioconductor cummeRbund library</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/cummeRbund.html</td>
      <td>RNA, Visualization</td>
      <td>cummerbund</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/cummerbund</td>
    </tr>
    <tr>
      <td></td>
      <td>cummerbund_to_tabular</td>
      <td>1.0.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Regenerate the tabular files generated by cuffdiff from a cummeRbund SQLite database.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Convert Formats, Next Gen Mappers</td>
      <td>cummerbund_to_tabular</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/cummerbund_to_tabular</td>
    </tr>
    <tr>
      <td></td>
      <td>cut_columns</td>
      <td>1.0.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Select columns from a dataset.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>cut_columns</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/cut_columns</td>
    </tr>
    <tr>
      <td></td>
      <td>delete_overlapping_indels</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Delete Overlapping Indels from a chromosome indels file</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>delete_overlapping_indels</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/delete_overlapping_indels</td>
    </tr>
    <tr>
      <td></td>
      <td>dgidb_annotator</td>
      <td>0.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Annotates a tabular file with information from the Drug-Gene Interaction Database (http://dgidb.genome.wustl.edu/)</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Systems Biology, Variant Analysis</td>
      <td>dgidb_annotator</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/dgidb_annotator</td>
    </tr>
    <tr>
      <td></td>
      <td>divide_pg_snp</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Separate pgSnp alleles into columns</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>divide_pg_snp</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/divide_pg_snp</td>
    </tr>
    <tr>
      <td></td>
      <td>draw_stacked_barplots</td>
      <td>1.0.0</td>
      <td></td>
      <td>R</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Draw Stacked Bar Plots for different categories and different criteria</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Graphics, Statistics</td>
      <td>draw_stacked_barplots</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/draw_stacked_barplots</td>
    </tr>
    <tr>
      <td></td>
      <td>dwt_cor_ava_perclass</td>
      <td>1.0.1</td>
      <td>1.7.5</td>
      <td>r-waveslim</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute P-values and Correlation Coefficients for Feature Occurrences</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>dwt_cor_ava_perclass</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/dwt_cor_ava_perclass</td>
    </tr>
    <tr>
      <td></td>
      <td>dwt_cor_avb_all</td>
      <td>1.0.1</td>
      <td>1.7.5</td>
      <td>r-waveslim</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute P-values and Correlation Coefficients for Occurrences of Two Set of Features</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>dwt_cor_avb_all</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/dwt_cor_avb_all</td>
    </tr>
    <tr>
      <td></td>
      <td>dwt_ivc_all</td>
      <td>1.0.1</td>
      <td>1.7.5</td>
      <td>r-waveslim</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute P-values and Second Moments for Feature Occurrences</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>dwt_ivc_all</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/dwt_ivc_all</td>
    </tr>
    <tr>
      <td></td>
      <td>dwt_var_perclass</td>
      <td>1.0.1</td>
      <td>1.7.5</td>
      <td>r-waveslim</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute P-values and Max Variances for Feature Occurrences</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>dwt_var_perclass</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/dwt_var_perclass</td>
    </tr>
    <tr>
      <td></td>
      <td>dwt_var_perfeature</td>
      <td>1.0.2</td>
      <td></td>
      <td>r-bitops</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Wavelet variance using Discrete Wavelet Transfoms</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>dwt_var_perfeature</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/dwt_var_perfeature</td>
    </tr>
    <tr>
      <td></td>
      <td>express</td>
      <td>1.1.1</td>
      <td>1.5.1</td>
      <td>eXpress</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Quantify the abundances of a set of target sequences from sampled subsequences</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>express</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/express</td>
    </tr>
    <tr>
      <td></td>
      <td>fasta_compute_length</td>
      <td>1.0.3</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute sequence length</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Fasta Manipulation</td>
      <td>fasta_compute_length</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/fasta_compute_length</td>
    </tr>
    <tr>
      <td></td>
      <td>fasta_concatenate_by_species</td>
      <td>0.0.1</td>
      <td>0.11.0</td>
      <td>bx-python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Concatenate FASTA alignment by species</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Fasta Manipulation</td>
      <td>fasta_concatenate_by_species</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/fasta_concatenate_by_species</td>
    </tr>
    <tr>
      <td></td>
      <td>fasta_filter_by_length</td>
      <td>1.2</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter sequences by length</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Fasta Manipulation</td>
      <td>fasta_filter_by_length</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/fasta_filter_by_length</td>
    </tr>
    <tr>
      <td></td>
      <td>fasta_to_tabular</td>
      <td>1.1.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FASTA-to-Tabular converter</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Fasta Manipulation</td>
      <td>fasta_to_tabular</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/fasta_to_tabular</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_trimmer_by_quality</td>
      <td>1.1.5</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FASTQ Quality Trimmer by sliding window</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td></td>
      <td>Fastq Manipulation</td>
      <td>fastq_trimmer_by_quality</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/fastq_trimmer_by_quality</td>
    </tr>
    <tr>
      <td></td>
      <td>fastqsolexa_to_fasta_qual</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FASTQSOLEXA-to-FASTA-QUAL extracts sequences and quality scores from FASTQSOLEXA data</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Convert Formats, Fastq Manipulation</td>
      <td>fastqsolexa_to_fasta_qual</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/fastqsolexa_to_fasta_qual</td>
    </tr>
    <tr>
      <td></td>
      <td>featurecounter</td>
      <td>2.0.0</td>
      <td>0.11.0</td>
      <td>bx-python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Feature coverage</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>featurecounter</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/featurecounter</td>
    </tr>
    <tr>
      <td></td>
      <td>filter_transcripts_via_tracking</td>
      <td>0.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter Combined Transcripts</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>filter_transcripts_via_tracking</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/filter_transcripts_via_tracking</td>
    </tr>
    <tr>
      <td></td>
      <td>generate_pc_lda_matrix</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Generate a Matrix for using PC and LDA</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>generate_pc_lda_matrix</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/generate_pc_lda_matrix</td>
    </tr>
    <tr>
      <td></td>
      <td>getindelrates_3way</td>
      <td>1.0.0</td>
      <td>0.11.0</td>
      <td>bx-python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Estimate Indel Rates for 3-way alignments</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>getindelrates_3way</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/getindelrates_3way</td>
    </tr>
    <tr>
      <td></td>
      <td>getindels_2way</td>
      <td>1.0.0</td>
      <td></td>
      <td>numpy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Fetch Indels from pairwise alignments</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>getindels_2way</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/getindels_2way</td>
    </tr>
    <tr>
      <td></td>
      <td>gmaj</td>
      <td>2.0.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>GMAJ Multiple Alignment Viewer</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Visualization</td>
      <td>gmaj</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/gmaj</td>
    </tr>
    <tr>
      <td></td>
      <td>hisat</td>
      <td>1.0.3</td>
      <td></td>
      <td>hisat</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>HISAT is a fast and sensitive spliced alignment program.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://ccb.jhu.edu/software/hisat/index.shtml</td>
      <td>Assembly</td>
      <td>hisat</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/hisat</td>
    </tr>
    <tr>
      <td></td>
      <td>histogram</td>
      <td>1.0.4</td>
      <td>2.7.8</td>
      <td>rpy2</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Histogram of a numeric column</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Graphics, Statistics</td>
      <td>histogram</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/histogram</td>
    </tr>
    <tr>
      <td></td>
      <td>indels_3way</td>
      <td>1.0.3</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Fetch Indels from 3-way alignments</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>indels_3way</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/indels_3way</td>
    </tr>
    <tr>
      <td></td>
      <td>kernel_canonical_correlation_analysis</td>
      <td>1.0.0</td>
      <td></td>
      <td>rpy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Kernel Canonical Correlation Analysis</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>kernel_canonical_correlation_analysis</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/kernel_canonical_correlation_analysis</td>
    </tr>
    <tr>
      <td></td>
      <td>kernel_principal_component_analysis</td>
      <td>1.0.0</td>
      <td></td>
      <td>rpy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Kernel Principal Component Analysis</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>kernel_principal_component_analysis</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/kernel_principal_component_analysis</td>
    </tr>
    <tr>
      <td></td>
      <td>lastz_paired_reads</td>
      <td>1.1.1</td>
      <td>1.04.22</td>
      <td>lastz</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Galaxy wrapper for the Lastz alignment tool on paired reads</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Next Gen Mappers</td>
      <td>lastz_paired_reads</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/lastz_paired_reads</td>
    </tr>
    <tr>
      <td></td>
      <td>lda_analysis</td>
      <td>1.0.1</td>
      <td></td>
      <td>R</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Perform Linear Discriminant Analysis</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Graphics, Statistics</td>
      <td>lda_analysis</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/lda_analysis</td>
    </tr>
    <tr>
      <td></td>
      <td>linear_regression</td>
      <td>1.0.1</td>
      <td></td>
      <td>R</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Perform Linear Regression</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>linear_regression</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/linear_regression</td>
    </tr>
    <tr>
      <td></td>
      <td>logistic_regression_vif</td>
      <td>1.0.1</td>
      <td></td>
      <td>numpy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Perform Logistic Regression with vif</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis, Statistics</td>
      <td>logistic_regression_vif</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/logistic_regression_vif</td>
    </tr>
    <tr>
      <td></td>
      <td>maf_cpg_filter</td>
      <td>1.0.0</td>
      <td>0.11.0</td>
      <td>bx-python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Mask CpG/non-CpG sites from MAF file</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>maf_cpg_filter</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/maf_cpg_filter</td>
    </tr>
    <tr>
      <td></td>
      <td>mapping_to_ucsc</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Format mapping data as UCSC custom track</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Visualization, Convert Formats, Next Gen Mappers</td>
      <td>mapping_to_ucsc</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/mapping_to_ucsc</td>
    </tr>
    <tr>
      <td></td>
      <td>megablast_xml_parser</td>
      <td>1.0.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Parse blast XML output</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Next Gen Mappers, Convert Formats</td>
      <td>megablast_xml_parser</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/megablast_xml_parser</td>
    </tr>
    <tr>
      <td></td>
      <td>merge_cols</td>
      <td>1.0.3</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Merge columns together.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>merge_cols</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/merge_cols</td>
    </tr>
    <tr>
      <td></td>
      <td>microsatellite_birthdeath</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Identify microsatellite births and deaths</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>microsatellite_birthdeath</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/microsatellite_birthdeath</td>
    </tr>
    <tr>
      <td></td>
      <td>microsats_alignment_level</td>
      <td>1.0.0</td>
      <td></td>
      <td>sputnik</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Extract Orthologous Microsatellites from pair-wise alignments</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>microsats_alignment_level</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/microsats_alignment_level</td>
    </tr>
    <tr>
      <td></td>
      <td>microsats_mutability</td>
      <td>1.1.0</td>
      <td>0.11.0</td>
      <td>bx-python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Estimate microsatellite mutability by specified attributes</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>microsats_mutability</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/microsats_mutability</td>
    </tr>
    <tr>
      <td></td>
      <td>mine</td>
      <td>0.0.1</td>
      <td></td>
      <td>MINE</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Maximal Information-based Nonparametric Exploration</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Variant Analysis</td>
      <td>mine</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/mine</td>
    </tr>
    <tr>
      <td></td>
      <td>multispecies_orthologous_microsats</td>
      <td>1.0.0</td>
      <td></td>
      <td>bx-sputnik</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Extract orthologous microsatellites</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>multispecies_orthologous_microsats</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/multispecies_orthologous_microsats</td>
    </tr>
    <tr>
      <td></td>
      <td>mutate_snp_codon</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Mutate Codons with SNPs</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Variant Analysis</td>
      <td>mutate_snp_codon</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/mutate_snp_codon</td>
    </tr>
    <tr>
      <td></td>
      <td>partialr_square</td>
      <td>1.0.0</td>
      <td></td>
      <td>R</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute partial R square</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>partialr_square</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/partialr_square</td>
    </tr>
    <tr>
      <td></td>
      <td>pearson_correlation</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Pearson and apos Correlation between any two numeric columns</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>pearson_correlation</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/pearson_correlation</td>
    </tr>
    <tr>
      <td></td>
      <td>pgsnp2gd_snp</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert from pgSnp to gd_snp</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Variant Analysis</td>
      <td>pgsnp2gd_snp</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/pgsnp2gd_snp</td>
    </tr>
    <tr>
      <td></td>
      <td>pileup_interval</td>
      <td>1.0.3</td>
      <td>0.11.0</td>
      <td>bx-python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Pileup-to-Interval condenses pileup format into ranges of bases</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>SAM</td>
      <td>pileup_interval</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/pileup_interval</td>
    </tr>
    <tr>
      <td></td>
      <td>pileup_parser</td>
      <td>1.0.2</td>
      <td></td>
      <td>perl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter pileup on coverage and SNPs</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-devteam/</td>
      <td>SAM</td>
      <td>pileup_parser</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/pileup_parser</td>
    </tr>
    <tr>
      <td></td>
      <td>plot_from_lda</td>
      <td>1.0.1</td>
      <td></td>
      <td>R</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Draw ROC plot on "Perform LDA" output</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Graphics, Statistics</td>
      <td>plot_from_lda</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/plot_from_lda</td>
    </tr>
    <tr>
      <td></td>
      <td>principal_component_analysis</td>
      <td>1.0.2</td>
      <td></td>
      <td>rpy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Principal Component Analysis</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>principal_component_analysis</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/principal_component_analysis</td>
    </tr>
    <tr>
      <td></td>
      <td>quality_filter</td>
      <td>1.0.1</td>
      <td>0.11.0</td>
      <td>bx-python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter nucleotides based on quality scores</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>quality_filter</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/quality_filter</td>
    </tr>
    <tr>
      <td></td>
      <td>rcve</td>
      <td>1.0.0</td>
      <td></td>
      <td>R</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute RCVE</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>rcve</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/rcve</td>
    </tr>
    <tr>
      <td></td>
      <td>remove_beginning</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Remove lines from the beginning of a file.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>remove_beginning</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/remove_beginning</td>
    </tr>
    <tr>
      <td></td>
      <td>rmap</td>
      <td>1.0.0</td>
      <td>2.1</td>
      <td>rmap</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>RMAP for Solexa Short Reads Alignment</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Next Gen Mappers</td>
      <td>rmap</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/rmap</td>
    </tr>
    <tr>
      <td></td>
      <td>rmapq</td>
      <td>1.0.0</td>
      <td>2.1</td>
      <td>rmap</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>RMAPQ for Solexa Short Reads Alignment with Quality Scores</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Next Gen Mappers</td>
      <td>rmapq</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/rmapq</td>
    </tr>
    <tr>
      <td></td>
      <td>sam2interval</td>
      <td>1.0.2</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert SAM to interval.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>SAM</td>
      <td>sam2interval</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/sam2interval</td>
    </tr>
    <tr>
      <td></td>
      <td>sam_bitwise_flag_filter</td>
      <td>1.0.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter SAM on bitwise flag values</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>SAM</td>
      <td>sam_bitwise_flag_filter</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/sam_bitwise_flag_filter</td>
    </tr>
    <tr>
      <td></td>
      <td>scatterplot</td>
      <td>1.0.3</td>
      <td></td>
      <td>numpy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Scatterplot of two numeric columns</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Graphics, Statistics</td>
      <td>scatterplot</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/scatterplot</td>
    </tr>
    <tr>
      <td></td>
      <td>short_reads_figure_high_quality_length</td>
      <td>1.0.0</td>
      <td></td>
      <td>rpy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Histogram of high quality score reads</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Graphics</td>
      <td>short_reads_figure_high_quality_length</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/short_reads_figure_high_quality_length</td>
    </tr>
    <tr>
      <td></td>
      <td>short_reads_figure_score</td>
      <td>1.0.2</td>
      <td></td>
      <td>fontconfig</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Build base quality distribution</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Graphics</td>
      <td>short_reads_figure_score</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/short_reads_figure_score</td>
    </tr>
    <tr>
      <td></td>
      <td>short_reads_trim_seq</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Select high quality segments</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Fastq Manipulation</td>
      <td>short_reads_trim_seq</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/short_reads_trim_seq</td>
    </tr>
    <tr>
      <td></td>
      <td>show_beginning</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Select lines from the beginning of a file.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>show_beginning</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/show_beginning</td>
    </tr>
    <tr>
      <td></td>
      <td>show_tail</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Select lines from the end of a file.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>show_tail</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/show_tail</td>
    </tr>
    <tr>
      <td></td>
      <td>sicer</td>
      <td>1.1</td>
      <td>1.1</td>
      <td>SICER</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Statistical approach for the Identification of ChIP-Enriched Regions</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://home.gwu.edu/~wpeng/Software.htm</td>
      <td>ChIP-seq</td>
      <td>sicer</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/sicer</td>
    </tr>
    <tr>
      <td></td>
      <td>split_paired_reads</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Split paired end reads</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Fastq Manipulation</td>
      <td>split_paired_reads</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/split_paired_reads</td>
    </tr>
    <tr>
      <td></td>
      <td>substitution_rates</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Estimate substitution rates for non-coding regions</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>substitution_rates</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/substitution_rates</td>
    </tr>
    <tr>
      <td></td>
      <td>substitutions</td>
      <td>1.0.1</td>
      <td>0.11.0</td>
      <td>bx-python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Fetch substitutions from pairwise alignments</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>substitutions</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/substitutions</td>
    </tr>
    <tr>
      <td></td>
      <td>t_test_two_samples</td>
      <td>1.0.1</td>
      <td></td>
      <td>R</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>T Test for Two Samples</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>t_test_two_samples</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/t_test_two_samples</td>
    </tr>
    <tr>
      <td></td>
      <td>table_annovar</td>
      <td>0.2</td>
      <td></td>
      <td>annovar</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Annotate a VCF file using ANNOVAR annotations to produce a tabular file that can be filtered</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Variant Analysis</td>
      <td>table_annovar</td>
      <td>devteam</td>
      <td>Nonehttps://github.com/galaxyproject/tools-devteam/tree/master/tools/table_annovar</td>
    </tr>
    <tr>
      <td></td>
      <td>tabular_to_fasta</td>
      <td>1.1.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tabular-to-FASTA</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Convert Formats</td>
      <td>tabular_to_fasta</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/tabular_to_fasta</td>
    </tr>
    <tr>
      <td></td>
      <td>tophat</td>
      <td>1.5.0</td>
      <td>1.19.2</td>
      <td>samtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tophat for Illumina</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA, Next Gen Mappers</td>
      <td>tophat</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/tophat</td>
    </tr>
    <tr>
      <td></td>
      <td>tophat2</td>
      <td>2.1.1</td>
      <td>2.5.3</td>
      <td>bowtie2</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tophat - fast splice junction mapper for RNA-Seq reads</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://ccb.jhu.edu/software/tophat/index.shtml</td>
      <td>RNA, Next Gen Mappers</td>
      <td>tophat2</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/tophat2</td>
    </tr>
    <tr>
      <td></td>
      <td>tophat_fusion_post</td>
      <td>0.1</td>
      <td></td>
      <td>blast+</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Wrapper for Tophat-Fusion post step</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics</td>
      <td>tophat_fusion_post</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/tophat_fusion_post</td>
    </tr>
    <tr>
      <td></td>
      <td>trimmer</td>
      <td>0.0.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Trim leading or trailing characters.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>trimmer</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/trimmer</td>
    </tr>
    <tr>
      <td></td>
      <td>ucsc_custom_track</td>
      <td>1.0.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Build custom track for UCSC genome browser</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>ucsc_custom_track</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/ucsc_custom_track</td>
    </tr>
    <tr>
      <td></td>
      <td>varscan_version_2</td>
      <td>2.4.2</td>
      <td>2.4.6</td>
      <td>varscan</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>VarScan wrapper</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://dkoboldt.github.io/varscan/</td>
      <td>Variant Analysis</td>
      <td>varscan_version_2</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/varscan_version_2</td>
    </tr>
    <tr>
      <td></td>
      <td>vcf2pgsnp</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>VCF to pgSnp</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Variant Analysis</td>
      <td>vcf2pgsnp</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/vcf2pgsnp</td>
    </tr>
    <tr>
      <td></td>
      <td>vcf_annotate</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Annotate a VCF file (dbSNP, hapmap)</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Variant Analysis</td>
      <td>vcf_annotate</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/vcf_annotate</td>
    </tr>
    <tr>
      <td></td>
      <td>vcf_extract</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Extract reads from a specified region</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Variant Analysis</td>
      <td>vcf_extract</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/vcf_extract</td>
    </tr>
    <tr>
      <td></td>
      <td>vcf_filter</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter a VCF file</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Variant Analysis</td>
      <td>vcf_filter</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/vcf_filter</td>
    </tr>
    <tr>
      <td></td>
      <td>vcf_intersect</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Generate the intersection of two VCF files</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Variant Analysis</td>
      <td>vcf_intersect</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/vcf_intersect</td>
    </tr>
    <tr>
      <td></td>
      <td>weightedaverage</td>
      <td>1.0.1</td>
      <td>1.1.0</td>
      <td>galaxy-ops</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Assign weighted-average of the values of features overlapping an interval</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>weightedaverage</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/weightedaverage</td>
    </tr>
    <tr>
      <td></td>
      <td>windowsplitter</td>
      <td>1.0.1</td>
      <td>0.11.0</td>
      <td>bx-python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Make windows</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>windowsplitter</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/windowsplitter</td>
    </tr>
    <tr>
      <td></td>
      <td>xy_plot</td>
      <td>1.0.2</td>
      <td></td>
      <td>r-base</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Plotting tool for multiple series and graph types</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Graphics, Statistics</td>
      <td>xy_plot</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/xy_plot</td>
    </tr>
    <tr>
      <td></td>
      <td>basecoverage</td>
      <td>1.0.0</td>
      <td>0.11.0</td>
      <td>bx-python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Base Coverage of all intervals</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/gops</td>
      <td>Genomic Interval Operations</td>
      <td>basecoverage</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/gops/basecoverage</td>
    </tr>
    <tr>
      <td></td>
      <td>cluster</td>
      <td>1.0.0</td>
      <td>0.11.0</td>
      <td>bx-python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Cluster the intervals of a dataset</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/gops</td>
      <td>Genomic Interval Operations</td>
      <td>cluster</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/gops/cluster</td>
    </tr>
    <tr>
      <td></td>
      <td>complement</td>
      <td>1.0.0</td>
      <td>0.11.0</td>
      <td>bx-python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Complement intervals of a dataset</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/gops</td>
      <td>Genomic Interval Operations</td>
      <td>complement</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/gops/complement</td>
    </tr>
    <tr>
      <td></td>
      <td>concat</td>
      <td>1.0.1</td>
      <td>0.11.0</td>
      <td>bx-python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Concatenate two bed files</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/gops</td>
      <td>Genomic Interval Operations</td>
      <td>concat</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/gops/concat</td>
    </tr>
    <tr>
      <td></td>
      <td>coverage</td>
      <td>1.0.0</td>
      <td>0.11.0</td>
      <td>bx-python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Coverage of a set of intervals on second set of intervals</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/gops</td>
      <td>Genomic Interval Operations</td>
      <td>coverage</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/gops/coverage</td>
    </tr>
    <tr>
      <td></td>
      <td>flanking_features</td>
      <td>4.0.1</td>
      <td>0.11.0</td>
      <td>bx-python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Fetch closest non-overlapping feature for every interval</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/gops</td>
      <td>Genomic Interval Operations</td>
      <td>flanking_features</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/gops/flanking_features</td>
    </tr>
    <tr>
      <td></td>
      <td>get_flanks</td>
      <td>1.0.0</td>
      <td>0.11.0</td>
      <td>bx-python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Get flanks returns flanking region/s for every gene</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/gops</td>
      <td>Genomic Interval Operations</td>
      <td>get_flanks</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/gops/get_flanks</td>
    </tr>
    <tr>
      <td></td>
      <td>intersect</td>
      <td>1.0.0</td>
      <td>0.11.0</td>
      <td>bx-python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Intersect the intervals of two datasets</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/gops</td>
      <td>Genomic Interval Operations</td>
      <td>intersect</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/gops/intersect</td>
    </tr>
    <tr>
      <td></td>
      <td>join</td>
      <td>1.0.0</td>
      <td>0.11.0</td>
      <td>bx-python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Join the intervals of two datasets side-by-side</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/gops</td>
      <td>Genomic Interval Operations</td>
      <td>join</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/gops/join</td>
    </tr>
    <tr>
      <td></td>
      <td>merge</td>
      <td>1.0.0</td>
      <td>0.11.0</td>
      <td>bx-python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Merge the overlapping intervals of a dataset</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/gops</td>
      <td>Genomic Interval Operations</td>
      <td>merge</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/gops/merge</td>
    </tr>
    <tr>
      <td></td>
      <td>subtract</td>
      <td>1.0.0</td>
      <td>0.11.0</td>
      <td>bx-python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Subtract the intervals of two datasets</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/gops</td>
      <td>Genomic Interval Operations</td>
      <td>subtract</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/gops/subtract</td>
    </tr>
    <tr>
      <td></td>
      <td>subtract_query</td>
      <td>0.1</td>
      <td>0.11.0</td>
      <td>bx-python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Subtract Whole Dataset from another dataset</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/gops</td>
      <td>Genomic Interval Operations</td>
      <td>subtract_query</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/gops/subtract_query</td>
    </tr>
    <tr>
      <td></td>
      <td>tables_arithmetic_operations</td>
      <td>1.0.0</td>
      <td></td>
      <td>perl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Arithmetic Operations on tables</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/gops</td>
      <td>Genomic Interval Operations</td>
      <td>tables_arithmetic_operations</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/gops/tables_arithmetic_operations</td>
    </tr>
    <tr>
      <td></td>
      <td>hgv_fundo</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FunDO human genes associated with disease terms</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>hgv_fundo</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/hgv/hgv_fundo</td>
    </tr>
    <tr>
      <td></td>
      <td>hgv_hilbertvis</td>
      <td>1.0.0</td>
      <td></td>
      <td>R</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>HVIS visualization of genomic data with the Hilbert curve</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.ebi.ac.uk/huber-srv/hilbert/</td>
      <td>Graphics, Visualization</td>
      <td>hgv_hilbertvis</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/hgv/hgv_hilbertvis</td>
    </tr>
    <tr>
      <td></td>
      <td>snpfreq</td>
      <td>1.0.1</td>
      <td></td>
      <td>R</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>snpFreq significant SNPs in case-control data</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Variant Analysis, Statistics</td>
      <td>snpfreq</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/hgv/snpfreq</td>
    </tr>
    <tr>
      <td></td>
      <td>find_diag_hits</td>
      <td>1.0.0</td>
      <td>0.10.0</td>
      <td>taxonomy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Find diagnostic hits</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>Metagenomics</td>
      <td>find_diag_hits</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/taxonomy/find_diag_hits</td>
    </tr>
    <tr>
      <td></td>
      <td>gi2taxonomy</td>
      <td>1.1.1</td>
      <td>0.10.0</td>
      <td>taxonomy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Fetch taxonomic representation</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>Metagenomics</td>
      <td>gi2taxonomy</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/taxonomy/gi2taxonomy</td>
    </tr>
    <tr>
      <td></td>
      <td>kraken2tax</td>
      <td>1.2+galaxy0</td>
      <td></td>
      <td>gawk</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert Kraken output to Galaxy taxonomy data.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>Metagenomics</td>
      <td>kraken2tax</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/blob/master/tool_collections/taxonomy/kraken2tax/</td>
    </tr>
    <tr>
      <td></td>
      <td>lca_wrapper</td>
      <td>1.0.1</td>
      <td>0.10.0</td>
      <td>taxonomy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Find lowest diagnostic rank</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>Metagenomics</td>
      <td>lca_wrapper</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/taxonomy/lca_wrapper</td>
    </tr>
    <tr>
      <td></td>
      <td>poisson2test</td>
      <td>1.0.0</td>
      <td>0.10.0</td>
      <td>taxonomy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Poisson two-sample test</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>Statistics, Metagenomics</td>
      <td>poisson2test</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/taxonomy/poisson2test</td>
    </tr>
    <tr>
      <td></td>
      <td>t2ps</td>
      <td>1.0.0</td>
      <td>0.10.0</td>
      <td>taxonomy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Draw phylogeny</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>Metagenomics</td>
      <td>t2ps</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/taxonomy/t2ps</td>
    </tr>
    <tr>
      <td></td>
      <td>t2t_report</td>
      <td>1.0.0</td>
      <td>0.10.0</td>
      <td>taxonomy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Summarize taxonomy</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>Metagenomics</td>
      <td>t2t_report</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/taxonomy/t2t_report</td>
    </tr>
    <tr>
      <td></td>
      <td>vcftools_annotate</td>
      <td>0.1</td>
      <td></td>
      <td>echo</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Annotate VCF using custom/user-defined annotations</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://vcftools.github.io/</td>
      <td>Variant Analysis</td>
      <td>vcftools_annotate</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/vcftools/vcftools_annotate</td>
    </tr>
    <tr>
      <td></td>
      <td>vcftools_compare</td>
      <td>0.1</td>
      <td>1.11</td>
      <td>tabix</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compare VCF files to get overlap and uniqueness statistics</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://vcftools.github.io/</td>
      <td>Variant Analysis</td>
      <td>vcftools_compare</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/vcftools/vcftools_compare</td>
    </tr>
    <tr>
      <td></td>
      <td>vcftools_consensus</td>
      <td>0.1.11</td>
      <td>1.19.2</td>
      <td>samtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Apply VCF variants to a fasta file to create consensus sequence</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://vcftools.github.io/</td>
      <td>Variant Analysis</td>
      <td>vcftools_consensus</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/vcftools/vcftools_consensus</td>
    </tr>
    <tr>
      <td></td>
      <td>vcftools_isec</td>
      <td>0.1.1</td>
      <td>1.11</td>
      <td>tabix</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Intersect multiple VCF datasets</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://vcftools.github.io/</td>
      <td>Variant Analysis</td>
      <td>vcftools_isec</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/vcftools/vcftools_isec</td>
    </tr>
    <tr>
      <td></td>
      <td>vcftools_merge</td>
      <td>0.1.11</td>
      <td>1.11</td>
      <td>tabix</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Merge multiple VCF datasets into a single dataset</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://vcftools.github.io/</td>
      <td>Variant Analysis</td>
      <td>vcftools_merge</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/vcftools/vcftools_merge</td>
    </tr>
    <tr>
      <td></td>
      <td>vcftools_slice</td>
      <td>0.1</td>
      <td></td>
      <td>echo</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Subset VCF dataset by genomic regions</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://vcftools.github.io/</td>
      <td>Variant Analysis</td>
      <td>vcftools_slice</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/vcftools/vcftools_slice</td>
    </tr>
    <tr>
      <td></td>
      <td>vcftools_subset</td>
      <td>0.1</td>
      <td>1.11</td>
      <td>tabix</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Select samples from a VCF dataset</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://vcftools.github.io/</td>
      <td>Variant Analysis</td>
      <td>vcftools_subset</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/vcftools/vcftools_subset</td>
    </tr>
    <tr>
      <td></td>
      <td>10x_bamtofastq</td>
      <td>1.4.1</td>
      <td>1.4.1</td>
      <td>10x_bamtofastq</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Converts 10x Genomics BAM to FASTQ</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/10XGenomics/bamtofastq</td>
      <td>Convert Formats</td>
      <td>10x_bamtofastq</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/10x_bamtofastq</td>
    </tr>
    <tr>
      <td></td>
      <td>AggregateAlignments</td>
      <td>0.6.0</td>
      <td>0.6.0</td>
      <td>graphclust-wrappers</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Aggregate and filter alignment metrics of individual clusters, like the output of graphclust_align_cluster.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td></td>
      <td>RNA</td>
      <td>graphclust_aggregate_alignments</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tools/GraphClust/AggregateAlignments</td>
    </tr>
    <tr>
      <td></td>
      <td>AlignCluster</td>
      <td>0.1</td>
      <td>0.6.0</td>
      <td>graphclust-wrappers</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Align predicted clusters of glob_report_no_align step with locarna and conservation analysis and visualizations.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>graphclust_align_cluster</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tools/GraphClust/AlignCluster</td>
    </tr>
    <tr>
      <td></td>
      <td>CMFinder</td>
      <td>0.4</td>
      <td>0.6.0</td>
      <td>graphclust-wrappers</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Determines consensus motives for sequences.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>graphclust_cmfinder</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust/CMFinder</td>
    </tr>
    <tr>
      <td></td>
      <td>CollectResults</td>
      <td>0.5</td>
      <td>0.6.0</td>
      <td>graphclust-wrappers</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Post-processing. Redundant clusters are merged and instances that belong to multiple clusters are assigned unambiguously. For every pair of clusters, the relative overlap (i.e. the fraction of instances that occur in both clusters) is computed and clusters are merged if the overlap exceeds 50%. instances that occur in both clusters) is computed and clusters are merged if the overlap exceeds 50%.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>graphclust_postprocessing</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust/CollectResults</td>
    </tr>
    <tr>
      <td></td>
      <td>CollectResultsNoAlign</td>
      <td>0.5</td>
      <td>0.6.0</td>
      <td>graphclust-wrappers</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Redundant GraphClust clusters are merged and instances that belong to multiple clusters are assigned unambiguously.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>graphclust_postprocessing_no_align</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust/CollectResultsNoAlign</td>
    </tr>
    <tr>
      <td></td>
      <td>GSPAN</td>
      <td>0.4</td>
      <td>0.6.0</td>
      <td>graphclust-wrappers</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Second step of GraphClust</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>graphclust_fasta_to_gspan</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust/GSPAN</td>
    </tr>
    <tr>
      <td></td>
      <td>LocARNAGraphClust</td>
      <td>0.4</td>
      <td>0.6.0</td>
      <td>graphclust-wrappers</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MLocARNA computes a multiple sequence-structure alignment of RNA sequences.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>graphclust_mlocarna</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust/LocARNAGraphClust</td>
    </tr>
    <tr>
      <td></td>
      <td>NSPDK</td>
      <td>9.2.3.1</td>
      <td>0.6.0</td>
      <td>graphclust-wrappers</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Produces an explicit sparse feature encoding and copmutes global feature index and returns top dense sets.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>graphclust_nspdk</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust/NSPDK</td>
    </tr>
    <tr>
      <td></td>
      <td>Plotting</td>
      <td>0.4</td>
      <td></td>
      <td>seaborn</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Plotting results for GraphClust</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>graphclust_motif_finder_plot</td>
      <td>rnateam</td>
      <td>https://github.com/eteriSokhoyan/galaxytools/tree/master/tools/GraphClust/Plotting</td>
    </tr>
    <tr>
      <td></td>
      <td>PrepareForMlocarna</td>
      <td>0.4</td>
      <td>0.6.0</td>
      <td>graphclust-wrappers</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>This tool prepares files for locarna step.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>graphclust_prepocessing_for_mlocarna</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust/PrepareForMlocarna</td>
    </tr>
    <tr>
      <td></td>
      <td>Preprocessing</td>
      <td>0.5</td>
      <td>0.6.0</td>
      <td>graphclust-wrappers</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Preprocessing input for GraphClust</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>graphclust_preprocessing</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust</td>
    </tr>
    <tr>
      <td></td>
      <td>Structure_GSPAN</td>
      <td>0.4</td>
      <td>0.6.0</td>
      <td>graphclust-wrappers</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert RNA structure to GSPAN graphs</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>structure_to_gspan</td>
      <td>rnateam</td>
      <td>https://github.com/mmiladi/galaxytools/blob/graphclust-gspan/tools/GraphClust/Structure_GSPAN</td>
    </tr>
    <tr>
      <td></td>
      <td>agat</td>
      <td>1.2.0</td>
      <td>1.3.0</td>
      <td>agat</td>
      <td>To update</td>
      <td>agat</td>
      <td>AGAT</td>
      <td>Data handling, Genome annotation</td>
      <td>Genomics</td>
      <td>GTF/GFF analysis toolkit</td>
      <td>Another Gff Analysis Toolkit (AGAT)Suite of tools to handle gene annotations in any GTF/GFF format.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/NBISweden/AGAT</td>
      <td>Convert Formats, Statistics, Fasta Manipulation</td>
      <td>agat</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/agat</td>
    </tr>
    <tr>
      <td></td>
      <td>antismash</td>
      <td>6.1.1</td>
      <td>7.1.0</td>
      <td>antismash</td>
      <td>To update</td>
      <td>antismash</td>
      <td>antiSMASH</td>
      <td>Sequence clustering, Gene prediction, Differential gene expression analysis</td>
      <td>Molecular interactions, pathways and networks, Gene and protein families</td>
      <td>Antismash allows the genome-wide identification, annotation and analysis of secondary metabolite biosynthesis gene clusters</td>
      <td>Rapid genome-wide identification, annotation and analysis of secondary metabolite biosynthesis gene clusters in bacterial and fungal genomes. It integrates and cross-links with a large number of in silico secondary metabolite analysis tools that have been published earlier.</td>
      <td></td>
      <td>To update</td>
      <td>https://antismash.secondarymetabolites.org</td>
      <td>Sequence Analysis</td>
      <td>antismash</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/antismash</td>
    </tr>
    <tr>
      <td></td>
      <td>atactk_trim_adapters</td>
      <td>0.1.6</td>
      <td>0.1.9</td>
      <td>atactk</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Trim adapters from paired-end HTS reads.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ParkerLab/atactk/</td>
      <td>Fastq Manipulation</td>
      <td>atactk_trim_adapters</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/blob/master/tools/trim_adapters</td>
    </tr>
    <tr>
      <td></td>
      <td>augustus</td>
      <td>3.1.0</td>
      <td>3.5.0</td>
      <td>augustus</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://bioinf.uni-greifswald.de/augustus/</td>
      <td>Sequence Analysis</td>
      <td>augustus</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/augustus</td>
    </tr>
    <tr>
      <td></td>
      <td>bamhash</td>
      <td>1.1</td>
      <td>1.1</td>
      <td>bamhash</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Hash BAM and FASTQ files to verify data integrity</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/DecodeGenetics/BamHash</td>
      <td>Sequence Analysis</td>
      <td>bamhash</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/bamhash</td>
    </tr>
    <tr>
      <td></td>
      <td>barcode_collapse</td>
      <td>0.1.0</td>
      <td>0.22.0</td>
      <td>pysam</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Paired End randomer aware duplicate removal algorithm</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/YeoLab/gscripts</td>
      <td>RNA, Sequence Analysis</td>
      <td>barcode_collapse</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/barcode_collapse</td>
    </tr>
    <tr>
      <td></td>
      <td>bigwig_to_bedgraph</td>
      <td>0.1.0</td>
      <td></td>
      <td>ucsc_tools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert from bigWig to bedGraph format</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Convert Formats</td>
      <td>bigwig_to_bedgraph</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/bigwig_to_bedgraph</td>
    </tr>
    <tr>
      <td></td>
      <td>biomodelsML</td>
      <td>1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Random Forest model to predict efficacy of immune checkpoint blockade across multiple cancer patient cohorts</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.ebi.ac.uk/biomodels/BIOMD0000001066</td>
      <td>Machine Learning</td>
      <td>biomodels_biomd0000001066</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/biomodelsML</td>
    </tr>
    <tr>
      <td></td>
      <td>bionano</td>
      <td>3.7.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Bionano Solve is a set of tools for analyzing Bionano data</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bionanogenomics.com/</td>
      <td>Assembly</td>
      <td>bionano</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/bionano</td>
    </tr>
    <tr>
      <td></td>
      <td>bismark</td>
      <td>0.22.1</td>
      <td>0.24.2</td>
      <td>bismark</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A tool to map bisulfite converted sequence reads and determine cytosine methylation states</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.bioinformatics.babraham.ac.uk/projects/bismark/</td>
      <td>Sequence Analysis, Next Gen Mappers</td>
      <td>bismark</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/bismark</td>
    </tr>
    <tr>
      <td></td>
      <td>blobtoolkit</td>
      <td>4.0.7</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Identification and isolation non-target data in draft and publicly available genome assemblies.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://blobtoolkit.genomehubs.org/</td>
      <td>Sequence Analysis, Assembly</td>
      <td>blobtoolkit</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/blobtoolkit</td>
    </tr>
    <tr>
      <td></td>
      <td>blockbuster</td>
      <td>0.1.2</td>
      <td>0.0.1.1</td>
      <td>blockbuster</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Blockbuster detects blocks of overlapping reads using a gaussian-distribution approach.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://hoffmann.bioinf.uni-leipzig.de/LIFE/blockbuster.html</td>
      <td>RNA, Sequence Analysis</td>
      <td>blockbuster</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/blockbuster</td>
    </tr>
    <tr>
      <td></td>
      <td>canu</td>
      <td>2.2</td>
      <td>2.2</td>
      <td>canu</td>
      <td>Up-to-date</td>
      <td>canu</td>
      <td>CANU</td>
      <td>De-novo assembly</td>
      <td>Genomics</td>
      <td>Canu is a hierarchical assembly pipeline designed for high-noise single-molecule sequencing (such as the PacBio RS II/Sequel or Oxford Nanopore MinION).</td>
      <td>De-novo assembly tool for long read chemistry like Nanopore data and PacBio data.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/marbl/canu</td>
      <td></td>
      <td>canu</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/canu</td>
    </tr>
    <tr>
      <td></td>
      <td>cellpose</td>
      <td>3.0.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Cellpose is an anatomical segmentation algorithm</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/MouseLand/cellpose</td>
      <td>Imaging</td>
      <td>cellpose</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/cellpose</td>
    </tr>
    <tr>
      <td></td>
      <td>cellprofiler</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>CellProfiler</td>
      <td>CellProfiler</td>
      <td>Quantification, Image analysis, Parsing</td>
      <td>Imaging, Microarray experiment, Genotype and phenotype</td>
      <td>cellProfiler wrapper</td>
      <td>Tool for quantifying data from biological images, particularly in high-throughput experiments.</td>
      <td>cellprofiler</td>
      <td>To update</td>
      <td></td>
      <td>Imaging</td>
      <td>cellprofiler</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools</td>
    </tr>
    <tr>
      <td></td>
      <td>cellprofiler_v4</td>
      <td>4.2.6</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>cellProfiler4 wrapper</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Imaging</td>
      <td>cellprofiler4</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools</td>
    </tr>
    <tr>
      <td></td>
      <td>chipseeker</td>
      <td>1.32.0</td>
      <td>1.38.0</td>
      <td>bioconductor-chipseeker</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A tool for ChIP peak annotation and visualization</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/ChIPseeker.html</td>
      <td>ChIP-seq, Genome annotation</td>
      <td>chipseeker</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/chipseeker</td>
    </tr>
    <tr>
      <td></td>
      <td>circexplorer</td>
      <td>1.1.9.0</td>
      <td>1.1.10</td>
      <td>circexplorer</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A combined strategy to identify circular RNAs (circRNAs and ciRNAs)</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/YangLab/CIRCexplorer</td>
      <td>Sequence Analysis, RNA</td>
      <td>circexplorer</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/circexplorer</td>
    </tr>
    <tr>
      <td></td>
      <td>combine_metaphlan_humann</td>
      <td>0.3.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td>combine_metaphlan_and_humann</td>
      <td>Combine Metaphlan and HUMAnN</td>
      <td>Aggregation</td>
      <td>Metagenomics, Molecular interactions, pathways and networks</td>
      <td>Combine MetaPhlAn2 and HUMAnN2 outputs to relate genus/species abundances and gene families/pathways abundances</td>
      <td>"This tool combine MetaPhlAn outputs and HUMANnN outputs." - Galaxy tool wrapper</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Metagenomics</td>
      <td>combine_metaphlan2_humann2</td>
      <td>bebatut</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/combine_metaphlan2_humann2</td>
    </tr>
    <tr>
      <td></td>
      <td>compare_humann2_output</td>
      <td>0.2.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>compare_humann2_outputs</td>
      <td>Compare HUMAnN2 outputs</td>
      <td>Comparison</td>
      <td>Metagenomics, Gene and protein families</td>
      <td>Compare outputs of HUMAnN2 for several samples and extract similar and specific information</td>
      <td>"This tool compare HUMANnN2 outputs with gene families or pathways and their relative abundances between several samples." - Galaxy tool wrapper</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Metagenomics</td>
      <td>compare_humann2_output</td>
      <td>bebatut</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/compare_humann2_output</td>
    </tr>
    <tr>
      <td></td>
      <td>cpat</td>
      <td>3.0.5</td>
      <td>3.0.5</td>
      <td>cpat</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Coding-potential assessment tool using an alignment-free logistic regression model.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/liguowang/cpat</td>
      <td>Transcriptomics</td>
      <td>cpat</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/cpat</td>
    </tr>
    <tr>
      <td></td>
      <td>crt</td>
      <td>1.2.0</td>
      <td>1.2</td>
      <td>crisper_recognition_tool</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>CRISPR Recognition Tool</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>crispr_recognition_tool</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/crt</td>
    </tr>
    <tr>
      <td></td>
      <td>diff</td>
      <td>3.7</td>
      <td></td>
      <td>diffutils</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>GNU diff tool that calculates the differences between two files.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.gnu.org/software/diffutils/</td>
      <td>Text Manipulation</td>
      <td>diff</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/diff</td>
    </tr>
    <tr>
      <td></td>
      <td>diffbind</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Diffbind provides functions for  processing  ChIP-Seq  data.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://bioconductor.org/packages/release/bioc/html/DiffBind.html</td>
      <td>ChIP-seq</td>
      <td>diffbind</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/diffbind</td>
    </tr>
    <tr>
      <td></td>
      <td>edta</td>
      <td></td>
      <td>2.2.0</td>
      <td>edta</td>
      <td>To update</td>
      <td>edta</td>
      <td>The Extensive de novo TE Annotator (EDTA)</td>
      <td>De-novo assembly, Deisotoping, Genome annotation</td>
      <td>Workflows, Mobile genetic elements, Plant biology, Transcription factors and regulatory sites, Model organisms</td>
      <td>The EDTA package was designed to filter out false discoveries in raw TE candidates and generate a high-quality non-redundant TE library for whole-genome TE annotations.</td>
      <td>The EDTA package was designed to filter out false discoveries in raw TE candidates and generate a high-quality non-redundant TE library for whole-genome TE annotations. Selection of initial search programs were based on benckmarkings on the annotation performance using a manually curated TE library in the rice genome.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/oushujun/EDTA</td>
      <td>Variant Analysis</td>
      <td>edta</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/edta</td>
    </tr>
    <tr>
      <td></td>
      <td>epicseg</td>
      <td>@VERSION_STRING@</td>
      <td>1.0</td>
      <td>epicseg</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>EpiCSeg is a tool for conducting chromatin segmentation.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/lamortenera/epicseg</td>
      <td>Epigenetics</td>
      <td>epicseg</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/epicseg</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_info</td>
      <td>0.25.1</td>
      <td>0.25.2</td>
      <td>fastq_utils</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FASTQ info allows to validate single or paired fastq files</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/nunofonseca/fastq_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_info</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/fastq_info</td>
    </tr>
    <tr>
      <td></td>
      <td>file_manipulation</td>
      <td>0.4</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>This tool returns all unique lines from a tab-separated file.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/file_manipulation</td>
      <td>Text Manipulation</td>
      <td>unique</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/file_manipulation</td>
    </tr>
    <tr>
      <td></td>
      <td>find_subsequences</td>
      <td>0.3</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td>find_subsequences</td>
      <td>bgruening</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>flye</td>
      <td>2.9.3</td>
      <td>2.9.3</td>
      <td>flye</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Assembly of long and error-prone reads.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/fenderglass/Flye/</td>
      <td>Assembly</td>
      <td>flye</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/flye</td>
    </tr>
    <tr>
      <td></td>
      <td>footprint</td>
      <td>1.0.0</td>
      <td>1.0.1</td>
      <td>footprint</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Find transcription factor footprints</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://ohlerlab.mdc-berlin.de/software/Reproducible_footprinting_139/</td>
      <td>Epigenetics</td>
      <td>footprint</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/footprint</td>
    </tr>
    <tr>
      <td></td>
      <td>format_cd_hit_output</td>
      <td>1.0.0+galaxy1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Format CD-hit output to rename representative sequences with cluster name and/or extract distribution inside clusters given a mapping file</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Fasta Manipulation</td>
      <td>format_cd_hit_output</td>
      <td>bebatut</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/format_cd_hit_output/</td>
    </tr>
    <tr>
      <td></td>
      <td>format_metaphlan2_output</td>
      <td>0.2.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>format_metaphlan2_output</td>
      <td>Format metaphlan2 output</td>
      <td>Formatting</td>
      <td>Taxonomy, Metagenomics</td>
      <td>Format MetaPhlAn2 output to extract abundance at different taxonomic levels</td>
      <td>"This tool format output file of MetaPhlan2 containing community content (abundance) at all taxonomic levels (from kingdom to strains)." - Galaxy tool wrapper</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Metagenomics</td>
      <td>format_metaphlan2_output</td>
      <td>bebatut</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/format_metaphlan2_output/</td>
    </tr>
    <tr>
      <td></td>
      <td>gfastats</td>
      <td>1.3.6</td>
      <td>1.3.6</td>
      <td>gfastats</td>
      <td>Up-to-date</td>
      <td>gfastats</td>
      <td>gfastats</td>
      <td>Data handling</td>
      <td>Computational biology</td>
      <td>Tool for generating sequence statistics and simultaneous genome assembly file manipulation.</td>
      <td>gfastats is a single fast and exhaustive tool for summary statistics and simultaneous genome assembly file manipulation. gfastats also allows seamless fasta/fastq/gfa conversion.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/vgl-hub/gfastats</td>
      <td>Sequence Analysis</td>
      <td>gfastats</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/gfastats</td>
    </tr>
    <tr>
      <td></td>
      <td>glimmer_hmm</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>GlimmerHMM is a new gene finder based on a Generalized Hidden Markov Model (GHMM)</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://ccb.jhu.edu/software/glimmerhmm/</td>
      <td>Sequence Analysis</td>
      <td>glimmer_hmm</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/glimmer_hmm</td>
    </tr>
    <tr>
      <td></td>
      <td>gotohscan</td>
      <td>1.3.0</td>
      <td>1.3</td>
      <td>gotohscan</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Find subsequences in db</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>gotohscan</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/gotohscan</td>
    </tr>
    <tr>
      <td></td>
      <td>graph_converter</td>
      <td>0.1.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert between different graph formats</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Convert Formats</td>
      <td>graph_converter</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/graph_converter</td>
    </tr>
    <tr>
      <td></td>
      <td>graphclust</td>
      <td>0.1</td>
      <td></td>
      <td>GraphClust</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>GraphClust can be used for structural clustering of RNA sequences.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.bioinf.uni-freiburg.de/Software/GraphClust/</td>
      <td>RNA</td>
      <td>graphclust</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/graphclust</td>
    </tr>
    <tr>
      <td></td>
      <td>graphmap</td>
      <td>0.5.2</td>
      <td>0.6.3</td>
      <td>graphmap</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Mapper for long, error-prone reads.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/isovic/graphmap/</td>
      <td>Assembly</td>
      <td>graphmap</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/graphmap</td>
    </tr>
    <tr>
      <td></td>
      <td>hclust2</td>
      <td>0.99</td>
      <td>1.0.0</td>
      <td>hclust2</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Plots heatmaps</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/nsegata/hclust2/</td>
      <td>Data Visualization</td>
      <td>hclust2</td>
      <td>rnateam</td>
      <td>https://github.com/yuanbit/galaxytools/tree/hclust2/tools/hclust2</td>
    </tr>
    <tr>
      <td></td>
      <td>hictk</td>
      <td>0.0.8</td>
      <td>0.0.9</td>
      <td>hictk</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>hictk, a blazing-fast toolkit to work with .hic and .cool files</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/paulsengroup/hictk</td>
      <td></td>
      <td>hictk</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/hictk</td>
    </tr>
    <tr>
      <td></td>
      <td>hicup</td>
      <td>0.9.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>The HiCUP-Pipeline from the Bioinformatics Babraham Institute.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.bioinformatics.babraham.ac.uk/projects/hicup/read_the_docs/html/index.html</td>
      <td>Epigenetics</td>
      <td>hicup</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/hicup</td>
    </tr>
    <tr>
      <td></td>
      <td>hifiasm</td>
      <td>0.19.8</td>
      <td>0.19.8</td>
      <td>hifiasm</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A fast haplotype-resolved de novo assembler</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/chhylp123/hifiasm</td>
      <td>Assembly</td>
      <td>hifiasm</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/hifiasm</td>
    </tr>
    <tr>
      <td></td>
      <td>homer</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Software for motif discovery and next generation sequencing analysis.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://homer.salk.edu/homer/</td>
      <td>Sequence Analysis</td>
      <td>homer</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/homer</td>
    </tr>
    <tr>
      <td></td>
      <td>illumina_methylation_analyser</td>
      <td>0.1</td>
      <td></td>
      <td>Rscript</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Methylation analyzer for Illumina 450k DNA emthylation microarrays</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/illumina_methylation_analyser</td>
      <td>Sequence Analysis</td>
      <td>illumina_methylation_analyser</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/illumina_methylation_analyser</td>
    </tr>
    <tr>
      <td></td>
      <td>bia-ftplinks</td>
      <td>0.1.0</td>
      <td></td>
      <td>wget</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tool to query ftp links for study from bioimage archive</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Imaging</td>
      <td>bia_download</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools</td>
    </tr>
    <tr>
      <td></td>
      <td>graphicsmagick</td>
      <td>1.3.42</td>
      <td>1.3.26</td>
      <td>graphicsmagick</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Contains tools based on GraphicsMagick</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.graphicsmagick.org</td>
      <td>Imaging</td>
      <td>graphicsmagick</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/new/gm/tools/image_processing/image_processing/</td>
    </tr>
    <tr>
      <td></td>
      <td>imagej2</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>imagej</td>
      <td>ImageJ2</td>
      <td>Image analysis, Image annotation, Visualisation</td>
      <td>Imaging</td>
      <td>ImageJ2 is a new version of ImageJ for the next generation of multidimensionalimage data, with a focus on scientific imaging.</td>
      <td>It is a public domain Java image processing program, which was designed with an open architecture. Custom acquisition, analysis and processing plugins can be developed using ImageJs built-in editor and a Java compiler. User-written plugins make it possible to solve many image processing and analysis problems, from three-dimensional live-cell imaging, to radiological image processing, multiple imaging system data comparisons to automated hematology systems.</td>
      <td>imagej2</td>
      <td>To update</td>
      <td>http://fiji.sc</td>
      <td>Imaging</td>
      <td>imagej2</td>
      <td>imgteam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/image_processing/imagej2</td>
    </tr>
    <tr>
      <td></td>
      <td>woundhealing</td>
      <td>1.6.1</td>
      <td>1.6.1</td>
      <td>fiji-morpholibj</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tool to automate quantification of wound healing in high-throughput microscopy scratch assays</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://git.embl.de/grp-cba/wound-healing-htm-screen</td>
      <td>Imaging</td>
      <td>Wound healing scratch assay image analysis</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools</td>
    </tr>
    <tr>
      <td></td>
      <td>instagraal</td>
      <td>0.1.6</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>instagraal</td>
      <td>instaGRAAL</td>
      <td>Genome assembly, Mapping assembly, Genetic mapping, Scaffolding</td>
      <td>Sequence assembly, Mapping, Metagenomics, Statistics and probability, DNA binding sites</td>
      <td>Large genome reassembly based on Hi-C data</td>
      <td>Chromosome-level quality scaffolding of brown algal genomes using InstaGRAAL.Large genome reassembly based on Hi-C data, continuation of GRAAL.Large genome reassembly based on Hi-C data (continuation and partial rewrite of GRAAL) and post-scaffolding polishing libraries.This work is under continuous development/improvement - see GRAAL for information about the basic principles.sudo pip3 install -e git+https://github.com/koszullab/instagraal.git@master#egg=instagraal.Note to OS X users: There is currently no CUDA support on Mojave (10.14) and it is unclear when it is going to be added, if it is to be added at all. This means instaGRAAL (or indeed any CUDA-based application) will not work on Mojave. If you wish to run it on OS X, the only solution for now is to downgrade to High Sierra (10.13)</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/koszullab/instaGRAAL</td>
      <td>Assembly</td>
      <td>instagraal</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/instagraal</td>
    </tr>
    <tr>
      <td></td>
      <td>iprscan5</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Interproscan queries the interpro database and provides annotations.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.ebi.ac.uk/Tools/pfa/iprscan5/</td>
      <td>Sequence Analysis</td>
      <td>iprscan5</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/iprscan5</td>
    </tr>
    <tr>
      <td></td>
      <td>itsx</td>
      <td>1.1.3</td>
      <td>1.1.3</td>
      <td>itsx</td>
      <td>Up-to-date</td>
      <td>ITSx</td>
      <td>ITSx</td>
      <td>Sequence feature detection</td>
      <td>Functional, regulatory and non-coding RNA, Microbiology</td>
      <td>ITSx is an open source software utility to extract the highly variable ITS1 and ITS2 subregions from ITS sequences.</td>
      <td>TSx is an open source software utility to extract the highly variable ITS1 and ITS2 subregions from ITS sequences, which is commonly used as a molecular barcode for e.g. fungi. As the inclusion of parts of the neighbouring, very conserved, ribosomal genes (SSU, 5S and LSU rRNA sequences) in the sequence identification process can lead to severely misleading results, ITSx identifies and extracts only the ITS regions themselves.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://microbiology.se/software/itsx/</td>
      <td>Metagenomics</td>
      <td>itsx</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/itsx</td>
    </tr>
    <tr>
      <td></td>
      <td>jupyter_job</td>
      <td>0.0.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Run jupyter notebook script in Galaxy</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Machine Learning</td>
      <td>run_jupyter_job</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/jupyter_job</td>
    </tr>
    <tr>
      <td></td>
      <td>labels</td>
      <td>1.0.5.0</td>
      <td></td>
      <td>labels</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>remaps and annotates alignments</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/labels</td>
      <td>Sequence Analysis</td>
      <td>labels</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/labels</td>
    </tr>
    <tr>
      <td></td>
      <td>lighter</td>
      <td>1.0</td>
      <td>1.1.2</td>
      <td>lighter</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Lighter is a kmer-based error correction method for whole genome sequencing data</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Fasta Manipulation</td>
      <td>lighter</td>
      <td>bgruening</td>
      <td>https://github.com/mourisl/Lighter</td>
    </tr>
    <tr>
      <td></td>
      <td>mafft</td>
      <td>7.508</td>
      <td>7.520</td>
      <td>mafft</td>
      <td>To update</td>
      <td>MAFFT</td>
      <td>MAFFT</td>
      <td>Multiple sequence alignment</td>
      <td>Sequence analysis</td>
      <td>Multiple alignment program for amino acid or nucleotide sequences</td>
      <td>MAFFT (Multiple Alignment using Fast Fourier Transform) is a high speed multiple sequence alignment program.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>mafft</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/mafft</td>
    </tr>
    <tr>
      <td></td>
      <td>mavedb</td>
      <td>0.9</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>data source for MaveDB</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Data Source</td>
      <td>mavedb_importer</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/mave_tools/mavedb/</td>
    </tr>
    <tr>
      <td></td>
      <td>mcl</td>
      <td>14.137</td>
      <td>22.282</td>
      <td>mcl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Markov Cluster Algorithm</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://micans.org/mcl/</td>
      <td>Sequence Analysis, Metagenomics</td>
      <td>mcl</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/mcl</td>
    </tr>
    <tr>
      <td></td>
      <td>methtools</td>
      <td>0.1.1</td>
      <td></td>
      <td>methtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>tools for methylation analysis</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/methtools</td>
      <td>Sequence Analysis</td>
      <td>methtools</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/methtools</td>
    </tr>
    <tr>
      <td></td>
      <td>methyldackel</td>
      <td>0.5.2</td>
      <td>0.6.1</td>
      <td>methyldackel</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A tool for processing bisulfite sequencing alignments</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/dpryan79/MethylDackel</td>
      <td>Sequence Analysis</td>
      <td>pileometh</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/methyldackel</td>
    </tr>
    <tr>
      <td></td>
      <td>methylkit</td>
      <td>0.99.2</td>
      <td>1.28.0</td>
      <td>bioconductor-methylkit</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A method for DNA methylation analysis and annotation from high-throughput bisulfite sequencing.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://bioconductor.org/packages/release/bioc/html/methylKit.html</td>
      <td>Epigenetics</td>
      <td>methylkit</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/methylkit</td>
    </tr>
    <tr>
      <td></td>
      <td>metilene</td>
      <td>0.2.6.1</td>
      <td>0.2.8</td>
      <td>metilene</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Differential DNA methylation calling</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA, Statistics</td>
      <td>metilene</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/metilene</td>
    </tr>
    <tr>
      <td></td>
      <td>miclip</td>
      <td>1.2.0</td>
      <td></td>
      <td>Rscript</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Identification of binding sites in CLIP-Seq data.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://cran.r-project.org/src/contrib/Archive/MiClip/</td>
      <td>Sequence Analysis</td>
      <td>miclip</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/miclip</td>
    </tr>
    <tr>
      <td></td>
      <td>minced</td>
      <td>0.2.0</td>
      <td>0.4.2</td>
      <td>minced</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MinCED - Mining CRISPRs in Environmental Datasets</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://bioweb2.pasteur.fr/docs/modules/minced/0.1.5/_README</td>
      <td>Sequence Analysis</td>
      <td>minced</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/minced</td>
    </tr>
    <tr>
      <td></td>
      <td>minipolish</td>
      <td>0.1.3</td>
      <td>0.1.3</td>
      <td>minipolish</td>
      <td>Up-to-date</td>
      <td>minipolish</td>
      <td>minipolish</td>
      <td>Localised reassembly, Read depth analysis</td>
      <td>Sequence assembly, Sequencing</td>
      <td>Polishing miniasm assemblies</td>
      <td>A tool that bridges the output of miniasm (long-read assembly) and racon (assembly polishing) together to polish a draft assembly. It also provides read depth information in contigs.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/rrwick/Minipolish</td>
      <td>Sequence Analysis</td>
      <td>minipolish</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/minipolish</td>
    </tr>
    <tr>
      <td></td>
      <td>mitohifi</td>
      <td>3</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Assembly mitogenomes from Pacbio HiFi read.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/marcelauliano/MitoHiFi/tree/mitohifi_v2</td>
      <td>Assembly</td>
      <td>mitohifi</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/mitohifi</td>
    </tr>
    <tr>
      <td></td>
      <td>molecule2gspan</td>
      <td>0.2</td>
      <td>2.3.90dev7d621d9</td>
      <td>openbabel</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>converter</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/molecule2gspan</td>
      <td>Convert Formats</td>
      <td>molecule_to_gspan</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/molecule2gspan</td>
    </tr>
    <tr>
      <td></td>
      <td>music_deconvolution</td>
      <td>0.1.1</td>
      <td>0.1.1</td>
      <td>music-deconvolution</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Multi-subject Single Cell deconvolution (MuSiC)</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/xuranw/MuSiC</td>
      <td>Transcriptomics</td>
      <td>music</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/music/</td>
    </tr>
    <tr>
      <td></td>
      <td>nanopolish</td>
      <td>0.14.0</td>
      <td>0.14.0</td>
      <td>nanopolish</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Nanopolish software package for signal-level analysis of Oxford Nanopore sequencing data.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/jts/nanopolish</td>
      <td></td>
      <td>nanopolish</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/nanopolish</td>
    </tr>
    <tr>
      <td></td>
      <td>netboxr</td>
      <td>1.6.0</td>
      <td>1.9.0</td>
      <td>bioconductor-netboxr</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>netboxr enables automated discovery of biological process modules by network analysis</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Systems Biology</td>
      <td>netboxr</td>
      <td>bgruening</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>nextdenovo</td>
      <td>2.5.0</td>
      <td>2.5.2</td>
      <td>nextdenovo</td>
      <td>To update</td>
      <td>nextdenovo</td>
      <td>NextDenovo</td>
      <td>De-novo assembly, Genome assembly</td>
      <td>Sequencing, Sequence assembly</td>
      <td>String graph-based de novo assembler for long reads</td>
      <td>NextDenovo is a string graph-based de novo assembler for long reads (CLR, HiFi and ONT). It uses a "correct-then-assemble" strategy similar to canu (no correction step for PacBio Hifi reads), but requires significantly less computing resources and storages.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/Nextomics/NextDenovo</td>
      <td>Assembly</td>
      <td>nextdenovo</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/nextdenovo</td>
    </tr>
    <tr>
      <td></td>
      <td>nucleosome_prediction</td>
      <td>3.0</td>
      <td>3.0</td>
      <td>nucleosome_prediction</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Prediction of Nucleosomes Positions on the Genome</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://genie.weizmann.ac.il/software/nucleo_exe.html</td>
      <td>Sequence Analysis</td>
      <td>nucleosome_prediction</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/nucleosome_prediction</td>
    </tr>
    <tr>
      <td></td>
      <td>numeric_clustering</td>
      <td>0.9</td>
      <td></td>
      <td>anaconda</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Clustering tool for numberic values</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://scikit-learn.org/stable/index.html</td>
      <td>Statistics</td>
      <td>numeric_clustering</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/numeric_clustering</td>
    </tr>
    <tr>
      <td></td>
      <td>openms</td>
      <td>2.1.0</td>
      <td>3.1.0</td>
      <td>openms</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>OpenMS in version 2.1.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>openms</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/openms</td>
    </tr>
    <tr>
      <td></td>
      <td>pandas_rolling_window</td>
      <td>0.1</td>
      <td></td>
      <td>numpy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Rolling window calculations</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.rolling.html</td>
      <td>Statistics</td>
      <td>pandas_rolling_window</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/pandas_rolling_window</td>
    </tr>
    <tr>
      <td></td>
      <td>peakachu</td>
      <td>0.2.0+galaxy1</td>
      <td>0.2.0</td>
      <td>peakachu</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>PEAKachu is a peak-caller for CLIP- and RIP-Seq data</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, RNA</td>
      <td>peakachu</td>
      <td>rnateam</td>
      <td>https://github.com/tbischler/PEAKachu</td>
    </tr>
    <tr>
      <td></td>
      <td>perf</td>
      <td>5.11.0</td>
      <td></td>
      <td>perf</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>suitable for boolean classification problems</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://osmot.cs.cornell.edu/kddcup/software.html</td>
      <td></td>
      <td>stats_perf_tool</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/perf</td>
    </tr>
    <tr>
      <td></td>
      <td>pfamscan</td>
      <td>1.6</td>
      <td>1.6</td>
      <td>pfam_scan</td>
      <td>Up-to-date</td>
      <td>pfamscan</td>
      <td>PfamScan</td>
      <td>Protein sequence analysis</td>
      <td>Sequence analysis</td>
      <td>Search a FASTA sequence against a library of Pfam HMM.</td>
      <td>This tool is used to search a FASTA sequence against a library of Pfam HMM.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/Pfam/Tools/</td>
      <td>Sequence Analysis</td>
      <td>pfamscan</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/pfamscan</td>
    </tr>
    <tr>
      <td></td>
      <td>pg_tools</td>
      <td></td>
      <td></td>
      <td>postgresql</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>tool suite for dealing with Postgresql databases from Galaxy's history</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.postgresql.org</td>
      <td>Data Export, Data Source</td>
      <td>pgtools</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tools/pgtools</td>
    </tr>
    <tr>
      <td></td>
      <td>pharmcat</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Pharmacogenomics Clinical Annotation Tool</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://pharmcat.org/</td>
      <td>Variant Analysis</td>
      <td>pharmcat</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/pharmcat</td>
    </tr>
    <tr>
      <td></td>
      <td>piranha</td>
      <td>1.2.1.0</td>
      <td>1.2.1</td>
      <td>piranha</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Piranha is a peak-caller for CLIP- and RIP-Seq data</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, RNA</td>
      <td>piranha</td>
      <td>rnateam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/piranha</td>
    </tr>
    <tr>
      <td></td>
      <td>platypus</td>
      <td>0.0.11</td>
      <td></td>
      <td>platypus</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>efficient and accurate variant-detection in high-throughput sequencing data</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.well.ox.ac.uk/platypus</td>
      <td>Sequence Analysis</td>
      <td>platypus</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/platypus</td>
    </tr>
    <tr>
      <td></td>
      <td>plotly_ml_performance_plots</td>
      <td>0.2</td>
      <td></td>
      <td>pandas</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>performance plots for machine learning problems</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://scikit-learn.org/stable/modules/classes.html#module-sklearn.metrics</td>
      <td>Visualization</td>
      <td>plotly_ml_performance_plots</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/plotly_ml_performance_plots</td>
    </tr>
    <tr>
      <td></td>
      <td>plotly_parallel_coordinates_plot</td>
      <td>0.2</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>parallel coordinates plot produced with plotly</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://plot.ly/python/parallel-coordinates-plot/</td>
      <td>Visualization</td>
      <td>plotly_parallel_coordinates_plot</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/plotly_parallel_coordinates_plot</td>
    </tr>
    <tr>
      <td></td>
      <td>plotly_regression_performance_plots</td>
      <td>0.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>performance plots for regression problems</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://scikit-learn.org/stable/supervised_learning.html#supervised-learning</td>
      <td>Visualization</td>
      <td>plotly_regression_performance_plots</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/plotly_regression_performance_plots</td>
    </tr>
    <tr>
      <td></td>
      <td>protease_prediction</td>
      <td>0.9</td>
      <td>2.0</td>
      <td>eden</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>This tool can learn the cleavage specificity of a given class of proteases.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/fabriziocosta/eden</td>
      <td>Sequence Analysis, Proteomics</td>
      <td>protease_prediction</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/protease_prediction</td>
    </tr>
    <tr>
      <td></td>
      <td>protein_properties</td>
      <td>0.2.0</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Calculation of various properties from given protein sequences</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>protein_properties</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/protein_properties</td>
    </tr>
    <tr>
      <td></td>
      <td>improviser</td>
      <td>1.1.0.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Visualisation of PepXML files</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.improviser.uni-freiburg.de/</td>
      <td>Proteomics</td>
      <td>proteomics_improviser</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/proteomics/improviser</td>
    </tr>
    <tr>
      <td></td>
      <td>racon</td>
      <td>1.5.0</td>
      <td>1.5.0</td>
      <td>racon</td>
      <td>Up-to-date</td>
      <td>Racon</td>
      <td>Racon</td>
      <td>Genome assembly, Mapping assembly, Sequence trimming</td>
      <td>Whole genome sequencing, Sequence assembly, Plant biology</td>
      <td>Consensus module for raw de novo DNA assembly of long uncorrected reads.</td>
      <td>The Possibility to Use Oxford Nanopore Technology | Ultrafast consensus module for raw de novo genome assembly of long uncorrected reads. http://genome.cshlp.org/content/early/2017/01/18/gr.214270.116 Note: This was the original repository which will no longer be officially maintained. Please use the new official repository here: https://github.com/isovic/racon| Racon is intended as a standalone consensus module to correct raw contigs generated by rapid assembly methods which do not include a consensus step | Consensus module for raw de novo DNA assembly of long uncorrected reads</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/isovic/racon</td>
      <td>Sequence Analysis</td>
      <td>racon</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/racon</td>
    </tr>
    <tr>
      <td></td>
      <td>repeat_masker</td>
      <td>0.1.2</td>
      <td>4.1.5</td>
      <td>RepeatMasker</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>RepeatMasker is a program that screens DNA sequences for interspersed repeats and low complexity DNA sequences.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.repeatmasker.org/</td>
      <td>Sequence Analysis</td>
      <td>repeat_masker</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/repeat_masker</td>
    </tr>
    <tr>
      <td></td>
      <td>replaceColumn</td>
      <td>0.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A tool to replace all column entries of a file given by values of a key-value file.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>replace_column_by_key_value_file</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/replaceColumn/tools/replaceColumn</td>
    </tr>
    <tr>
      <td></td>
      <td>rest_tool</td>
      <td>0.1.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>This tool fetches data from pubchem via the PubChem REST API.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://pubchem.ncbi.nlm.nih.gov/pug_rest/PUG_REST.html</td>
      <td>Data Source</td>
      <td>pubchem_rest_tool</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rest_tool</td>
    </tr>
    <tr>
      <td></td>
      <td>antarna</td>
      <td>1.1</td>
      <td>2.0.1.2</td>
      <td>antarna</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>antaRNA uses ant colony optimization to solve the inverse folding problem in RNA research .</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>antarna</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/antarna/</td>
    </tr>
    <tr>
      <td></td>
      <td>aresite2</td>
      <td>0.1.2</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>AREsite2 REST Interface</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://rna.tbi.univie.ac.at/AREsite</td>
      <td>RNA, Data Source, Sequence Analysis</td>
      <td>aresite2</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/aresite2</td>
    </tr>
    <tr>
      <td></td>
      <td>blockclust</td>
      <td>1.1.1</td>
      <td>1.1.1</td>
      <td>blockclust</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>BlockClust detects transcripts with similar processing patterns.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/workflows/blockclust</td>
      <td>RNA</td>
      <td>blockclust</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/blockclust</td>
    </tr>
    <tr>
      <td></td>
      <td>cmsearch_deoverlap</td>
      <td>0.08+galaxy0</td>
      <td></td>
      <td>perl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>removes lower scoring overlaps from cmsearch results.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/EBI-Metagenomics/pipeline-v5/blob/master/tools/RNA_prediction/cmsearch-deoverlap/cmsearch-deoverlap.pl</td>
      <td>RNA</td>
      <td>cmsearch_deoverlap</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/cmsearch_deoverlap</td>
    </tr>
    <tr>
      <td></td>
      <td>cmv</td>
      <td>1.0.8</td>
      <td>1.0.8</td>
      <td>cmv</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>cmv is a collection of tools for the visualisation of Hidden Markov Models and RNA-family models.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/eggzilla/cmv</td>
      <td>RNA</td>
      <td>cmv</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/cmv</td>
    </tr>
    <tr>
      <td></td>
      <td>cofold</td>
      <td>2.0.4.0</td>
      <td>2.0.4</td>
      <td>cofold</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Cofold predicts RNA secondary structures that takes co-transcriptional folding into account.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.e-rna.org/cofold/</td>
      <td>RNA</td>
      <td>cofold</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/cofold</td>
    </tr>
    <tr>
      <td></td>
      <td>compalignp</td>
      <td>1.0</td>
      <td>1.0</td>
      <td>compalignp</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute fractional identity between trusted alignment and test alignment</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td></td>
      <td>RNA, Sequence Analysis</td>
      <td>compalignp</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/compalignp/</td>
    </tr>
    <tr>
      <td></td>
      <td>coprarna</td>
      <td>2.1.1</td>
      <td>2.1.4</td>
      <td>coprarna</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Target prediction for prokaryotic trans-acting small RNAs</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/PatrickRWright/CopraRNA</td>
      <td>RNA, Sequence Analysis</td>
      <td>coprarna</td>
      <td>rnateam</td>
      <td>https://github.com/PatrickRWright/CopraRNA</td>
    </tr>
    <tr>
      <td></td>
      <td>dewseq</td>
      <td>0.1.0+galaxy0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>DEWSeq is a sliding window based peak caller for eCLIP/iCLIP data</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/EMBL-Hentze-group/DEWSeq_analysis_helpers</td>
      <td>Sequence Analysis, RNA, CLIP-seq</td>
      <td>dewseq</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/dewseq</td>
    </tr>
    <tr>
      <td></td>
      <td>dorina</td>
      <td>1.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>data source for RNA interactions in post-transcriptional regulation</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA, Data Source</td>
      <td>dorina</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/dorina/</td>
    </tr>
    <tr>
      <td></td>
      <td>dot2ct</td>
      <td>5.7.a</td>
      <td>6.4</td>
      <td>rnastructure</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Dot-Bracket to Connect Table (CT)</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, RNA</td>
      <td>dot2ct</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/dot2ct</td>
    </tr>
    <tr>
      <td></td>
      <td>dotknot</td>
      <td>1.3.1</td>
      <td></td>
      <td>vienna_rna</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>DotKnot is a heuristic method for pseudoknot prediction in a given RNA sequence</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://dotknot.csse.uwa.edu.au/</td>
      <td>RNA, Proteomics</td>
      <td>dotknot</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna/dotknot</td>
    </tr>
    <tr>
      <td></td>
      <td>exparna</td>
      <td>1.0.1</td>
      <td>1.0.1</td>
      <td>exparna</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ExpaRNA is a fast, motif-based comparison and alignment tool for RNA molecules.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://rna.informatik.uni-freiburg.de/ExpaRNA/Input.jsp</td>
      <td>RNA</td>
      <td>exparna</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/exparna</td>
    </tr>
    <tr>
      <td></td>
      <td>graphprot</td>
      <td>1.1.7+galaxy1</td>
      <td>1.1.7</td>
      <td>graphprot</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>GraphProt models binding preferences of RNA-binding proteins.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/dmaticzka/GraphProt</td>
      <td>Sequence Analysis, RNA, CLIP-seq</td>
      <td>graphprot</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/graphprot</td>
    </tr>
    <tr>
      <td></td>
      <td>htseq-clip</td>
      <td>0.1.0+galaxy0</td>
      <td>2.19.0b0</td>
      <td>htseq-clip</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>htseq-clip is a toolset for the analysis of eCLIP/iCLIP datasets</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/EMBL-Hentze-group/htseq-clip</td>
      <td>Sequence Analysis, RNA, CLIP-seq</td>
      <td>htseq_clip</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/htseq-clip</td>
    </tr>
    <tr>
      <td></td>
      <td>infernal</td>
      <td>1.1.4</td>
      <td>1.1.5</td>
      <td>infernal</td>
      <td>To update</td>
      <td>infernal</td>
      <td>Infernal</td>
      <td>Nucleic acid feature detection</td>
      <td>Sequence sites, features and motifs</td>
      <td>Infernal ("INFERence of RNA ALignment") is for searching DNA sequence databases for RNA structure and sequence similarities.</td>
      <td>Infernal ("INFERence of RNA ALignment") is for searching DNA sequence databases for RNA structure and sequence similarities. It is an implementation of a special case of profile stochastic context-free grammars called covariance models (CMs). A CM is like a sequence profile, but it scores a combination of sequence consensus and RNA secondary structure consensus, so in many cases, it is more capable of identifying RNA homologs that conserve their secondary structure more than their primary sequence.</td>
      <td></td>
      <td>To update</td>
      <td>http://infernal.janelia.org/</td>
      <td>RNA</td>
      <td>infernal</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/infernal</td>
    </tr>
    <tr>
      <td></td>
      <td>inforna</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>INFO-RNA is a service for the design of RNA sequences that fold into a given pseudo-knot free RNA secondary structure.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://rna.informatik.uni-freiburg.de/INFORNA/Input.jsp</td>
      <td>RNA</td>
      <td>inforna</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/inforna</td>
    </tr>
    <tr>
      <td></td>
      <td>intarna</td>
      <td>3.4.0</td>
      <td>3.4.0</td>
      <td>intarna</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Efficient RNA-RNA interaction prediction incorporating accessibility and seeding of interaction sites.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/BackofenLab/IntaRNA</td>
      <td>RNA</td>
      <td>intarna</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/intarna</td>
    </tr>
    <tr>
      <td></td>
      <td>kinwalker</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Kinwalker splits the folding process into a series of events where each event can either be a folding event or a transcription event.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.bioinf.uni-leipzig.de/Software/Kinwalker/</td>
      <td>RNA</td>
      <td>kinwalker</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/kinwalker</td>
    </tr>
    <tr>
      <td></td>
      <td>locarna</td>
      <td>1.9.2.3</td>
      <td>2.0.0</td>
      <td>locarna</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>LocARNA - A suite for multiple alignment and folding of RNAs</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.bioinf.uni-freiburg.de/Software/LocARNA/</td>
      <td>RNA</td>
      <td>locarna</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/locarna</td>
    </tr>
    <tr>
      <td></td>
      <td>mea</td>
      <td>0.6.4.1</td>
      <td>0.6.4</td>
      <td>mea</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Maximum expected accuracy prediction</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.bioinf.uni-leipzig.de/Software/mea</td>
      <td>RNA</td>
      <td>mea</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/mea</td>
    </tr>
    <tr>
      <td></td>
      <td>mqc</td>
      <td>1.9</td>
      <td>1.10</td>
      <td>mqc</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Ribosome profiling mapping quality control tool</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/Biobix/mQC</td>
      <td>Sequence Analysis</td>
      <td>mqc</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/mqc/</td>
    </tr>
    <tr>
      <td></td>
      <td>nastiseq</td>
      <td>1.0</td>
      <td>1.0</td>
      <td>r-nastiseq</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A method to identify cis-NATs using ssRNA-seq</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://ohlerlab.mdc-berlin.de/software/NASTIseq_104/</td>
      <td>RNA</td>
      <td>nastiseq</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/nastiseq</td>
    </tr>
    <tr>
      <td></td>
      <td>paralyzer</td>
      <td>1.5</td>
      <td>1.5</td>
      <td>paralyzer</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A method to generate a high resolution map of interaction sites between RNA-binding proteins and their targets.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://ohlerlab.mdc-berlin.de/software/PARalyzer_85/</td>
      <td>RNA</td>
      <td>paralyzer</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/paralyzer</td>
    </tr>
    <tr>
      <td></td>
      <td>pipmir</td>
      <td>0.1.0</td>
      <td>1.1</td>
      <td>pipmir</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A method to identify novel plant miRNA.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://ohlerlab.mdc-berlin.de/software/Pipeline_for_the_Identification_of_Plant_miRNAs_84/</td>
      <td>RNA</td>
      <td>pipmir</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/pipmir</td>
    </tr>
    <tr>
      <td></td>
      <td>rRNA</td>
      <td>0.1</td>
      <td></td>
      <td>hmmsearch3.0</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Identification of ribosomal RNA genes in metagenomic fragments.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://weizhong-lab.ucsd.edu/meta_rna/</td>
      <td>RNA</td>
      <td>rrna</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rRNA</td>
    </tr>
    <tr>
      <td></td>
      <td>rbpbench</td>
      <td>0.8.1</td>
      <td>0.8.1</td>
      <td>rbpbench</td>
      <td>Up-to-date</td>
      <td>rbpbench</td>
      <td>RBPBench</td>
      <td></td>
      <td>RNA, Protein interactions, RNA immunoprecipitation, Bioinformatics, Sequence analysis</td>
      <td>Evaluate CLIP-seq and other genomic region data using a comprehensive collection of RBP binding motifs</td>
      <td>Evaluate CLIP-seq and other genomic region data using a comprehensive collection of RBP binding motifs</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/michauhl/RBPBench</td>
      <td>Sequence Analysis, RNA, CLIP-seq</td>
      <td>rbpbench</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/rbpbench</td>
    </tr>
    <tr>
      <td></td>
      <td>rcas</td>
      <td>1.5.4</td>
      <td>1.28.2</td>
      <td>bioconductor-rcas</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>RCAS (RNA Centric Annotation System) for functional analysis of transcriptome-wide regions detected by high-throughput experiments</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/BIMSBbioinfo/RCAS</td>
      <td>RNA</td>
      <td>rcas</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/rcas/</td>
    </tr>
    <tr>
      <td></td>
      <td>reago</td>
      <td>1.1</td>
      <td>1.1</td>
      <td>reago</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Reago is tool to assembly 16S ribosomal RNA recovery from metagenomic data.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/chengyuan/reago-1.1</td>
      <td>Metagenomics, RNA</td>
      <td>reago</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/reago</td>
    </tr>
    <tr>
      <td></td>
      <td>remurna</td>
      <td>1.0.0</td>
      <td>1.0</td>
      <td>remurna</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>remuRNA - Measurement of Single Nucleotide Polymorphism induced Changes of RNA Conformation</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.ncbi.nlm.nih.gov/CBBresearch/Przytycka/index.cgi#remurna</td>
      <td>RNA</td>
      <td>remurna</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/remurna</td>
    </tr>
    <tr>
      <td></td>
      <td>ribotaper</td>
      <td>1.3.1a</td>
      <td>1.3.1</td>
      <td>ribotaper</td>
      <td>To update</td>
      <td>ribotaper</td>
      <td>RiboTaper</td>
      <td>Gene expression profiling</td>
      <td>Functional genomics</td>
      <td>A method for defining traslated ORFs using Ribosome Profiling data.</td>
      <td>New analysis pipeline for Ribosome Profiling (Ribo-seq) experiments, which exploits the triplet periodicity of ribosomal footprints to call translated regions.</td>
      <td></td>
      <td>To update</td>
      <td>https://ohlerlab.mdc-berlin.de/software/RiboTaper_126/</td>
      <td>RNA</td>
      <td>ribotaper</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/ribotaper/</td>
    </tr>
    <tr>
      <td></td>
      <td>rna_shapes</td>
      <td>3.3.0</td>
      <td></td>
      <td>@EXECUTABLE@</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute secondary structures of RNA</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>rnashapes</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/rna_shapes</td>
    </tr>
    <tr>
      <td></td>
      <td>rnabob</td>
      <td>2.2.1.0</td>
      <td>2.2.1</td>
      <td>rnabob</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Fast pattern searching for RNA structural motifs</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://eddylab.org/software.html</td>
      <td>RNA</td>
      <td>rnabob</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/rnabob</td>
    </tr>
    <tr>
      <td></td>
      <td>rnacode</td>
      <td>0.3.2</td>
      <td>0.3</td>
      <td>rnacode</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Analyze the protein coding potential in MSA</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>rnacode</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/rnacode</td>
    </tr>
    <tr>
      <td></td>
      <td>rnacommender</td>
      <td>0.1.1</td>
      <td>3.5</td>
      <td>sam</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>RNAcommender is a tool for genome-wide recommendation of RNA-protein interactions.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/gianlucacorrado/RNAcommender</td>
      <td>RNA</td>
      <td>rnacommender</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/rna_commander/tools/rna_tools/rna_commender</td>
    </tr>
    <tr>
      <td></td>
      <td>rnalien</td>
      <td>1.3.6</td>
      <td>1.8.0</td>
      <td>rnalien</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>RNAlien unsupervized RNA family model construction</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://rna.tbi.univie.ac.at/rnalien/</td>
      <td>RNA, Sequence Analysis</td>
      <td>rnalien</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/rnalien</td>
    </tr>
    <tr>
      <td></td>
      <td>rnasnp</td>
      <td>1.2.0</td>
      <td>1.2</td>
      <td>rnasnp</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>RNAsnp Efficient detection of local RNA secondary structure changes induced by SNPs</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://rth.dk/resources/rnasnp/</td>
      <td>RNA</td>
      <td>rnasnp</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rnasnp</td>
    </tr>
    <tr>
      <td></td>
      <td>rnaz</td>
      <td>2.1.1</td>
      <td>2.1.1</td>
      <td>rnaz</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>RNAz is a program for predicting structurally conserved and thermodynamically stable RNA secondary structures in multiple sequence alignments.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://www.tbi.univie.ac.at/~wash/RNAz/</td>
      <td>RNA</td>
      <td>rnaz</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_team/rnaz</td>
    </tr>
    <tr>
      <td></td>
      <td>selectsequencesfrommsa</td>
      <td>1.0.5</td>
      <td>1.0.5</td>
      <td>selectsequencesfrommsa</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SelectSequences - selects representative entries from a multiple sequence alignment in clustal format</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/eggzilla/SelectSequences</td>
      <td>RNA, Sequence Analysis</td>
      <td>selectsequencesfrommsa</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/selectsequencesfrommsa</td>
    </tr>
    <tr>
      <td></td>
      <td>sortmerna</td>
      <td>4.3.6</td>
      <td>4.3.6</td>
      <td>sortmerna</td>
      <td>Up-to-date</td>
      <td>sortmerna</td>
      <td>SortMeRNA</td>
      <td>Sequence similarity search, Sequence comparison, Sequence alignment analysis</td>
      <td>Metatranscriptomics, Metagenomics</td>
      <td>SortMeRNA is a software designed to rapidly filter ribosomal RNA fragments from metatransriptomic data produced by next-generation sequencers.</td>
      <td>Sequence analysis tool for filtering, mapping and OTU-picking NGS reads.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://bioinfo.lifl.fr/RNA/sortmerna/</td>
      <td>RNA</td>
      <td>sortmerna</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/sortmerna</td>
    </tr>
    <tr>
      <td></td>
      <td>sshmm</td>
      <td>1.0.7</td>
      <td>1.0.7</td>
      <td>sshmm</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ssHMM is an RNA sequence-structure motif finder for RNA-binding protein data, such as CLIP-Seq data</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.molgen.mpg.de/heller/ssHMM</td>
      <td>Sequence Analysis, RNA</td>
      <td>sshmm</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/sshmm/</td>
    </tr>
    <tr>
      <td></td>
      <td>targetfinder</td>
      <td>1.7</td>
      <td>1.7</td>
      <td>targetfinder</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Plant small RNA target prediction tool</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/carringtonlab/TargetFinder.git</td>
      <td>RNA</td>
      <td>targetfinder</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/targetfinder/</td>
    </tr>
    <tr>
      <td></td>
      <td>trna_prediction</td>
      <td>0.6</td>
      <td>1.2.41</td>
      <td>aragorn</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Aragorn predicts tRNA and tmRNA in nucleotide sequences.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://mbioserv2.mbioekol.lu.se/ARAGORN/</td>
      <td>RNA</td>
      <td>trna_prediction</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/trna_prediction</td>
    </tr>
    <tr>
      <td></td>
      <td>vienna_rna</td>
      <td>2.2.10</td>
      <td>2.6.4</td>
      <td>viennarna</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ViennaRNA - Prediction and comparison of RNA secondary structures</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.tbi.univie.ac.at/RNA/</td>
      <td>RNA</td>
      <td>viennarna</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/vienna_rna</td>
    </tr>
    <tr>
      <td></td>
      <td>sailfish</td>
      <td>0.10.1.1</td>
      <td></td>
      <td>bzip2</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Sailfish is a tool for transcript quantification from RNA-seq data</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.cs.cmu.edu/~ckingsf/software/sailfish/</td>
      <td>Sequence Analysis, RNA</td>
      <td>sailfish</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/sailfish</td>
    </tr>
    <tr>
      <td></td>
      <td>salmon</td>
      <td>1.10.1</td>
      <td>1.10.2</td>
      <td>salmon</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Salmon is a wicked-fast program to produce a highly-accurate, transcript-level quantification estimates from RNA-seq and single-cell data.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/COMBINE-lab/salmon</td>
      <td>Sequence Analysis, RNA, Transcriptomics</td>
      <td></td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/salmon</td>
    </tr>
    <tr>
      <td></td>
      <td>sambamba</td>
      <td></td>
      <td>1.0</td>
      <td>sambamba</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SAMBAMBA: process your BAM data faster!</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/lomereiter/sambamba</td>
      <td>SAM</td>
      <td>sambamba</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/sambamba</td>
    </tr>
    <tr>
      <td></td>
      <td>sed</td>
      <td>0.0.1</td>
      <td></td>
      <td>sed</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Manipulate your data with the sed command line tool.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/sed</td>
      <td>Text Manipulation</td>
      <td>sed_wrapper</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/sed</td>
    </tr>
    <tr>
      <td></td>
      <td>segemehl</td>
      <td>0.2.0.4</td>
      <td>0.3.4</td>
      <td>segemehl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>segemehl - short read mapping with gaps</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.bioinf.uni-leipzig.de/Software/segemehl/</td>
      <td>Next Gen Mappers</td>
      <td>segemehl</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/segemehl</td>
    </tr>
    <tr>
      <td></td>
      <td>sklearn</td>
      <td>1.0.11.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Machine Learning tool suite from Scikit-learn</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://scikit-learn.org</td>
      <td>Machine Learning, Statistics</td>
      <td>sklearn</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/sklearn</td>
    </tr>
    <tr>
      <td></td>
      <td>splitfasta</td>
      <td>0.4.0</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Split a multi-sequence fasta file into files containing single sequences</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>splitfasta</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/splitfasta</td>
    </tr>
    <tr>
      <td></td>
      <td>statistics</td>
      <td>0.3</td>
      <td></td>
      <td>numpy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tool for computing statistical tests.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/statistics</td>
      <td>Statistics</td>
      <td>bg_statistical_hypothesis_testing</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/statistics</td>
    </tr>
    <tr>
      <td></td>
      <td>stress_ng</td>
      <td>0.12.04</td>
      <td></td>
      <td>stress-ng</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>stress test a computer system in various selectable ways</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Web Services</td>
      <td>stress_ng</td>
      <td>bgruening-util</td>
      <td>https://github.com/ColinIanKing/stress-ng</td>
    </tr>
    <tr>
      <td></td>
      <td>tapscan</td>
      <td>4.76+galaxy0</td>
      <td>3.4</td>
      <td>hmmer</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Search for transcription associated proteins (TAPs)</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://plantcode.cup.uni-freiburg.de/tapscan/</td>
      <td>Proteomics</td>
      <td>tapscan</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/tapscan</td>
    </tr>
    <tr>
      <td></td>
      <td>add_line_to_file</td>
      <td>0.1.0</td>
      <td>8.25</td>
      <td>coreutils</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Adds a text line to the beginning or end of a file.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>add_line_to_file</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/text_processing/add_line_to_file</td>
    </tr>
    <tr>
      <td></td>
      <td>column_arrange_by_header</td>
      <td>0.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Column arrange by header name</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>column_arrange_by_header</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/text_processing/column_arrange_by_header</td>
    </tr>
    <tr>
      <td></td>
      <td>join_files_on_column_fuzzy</td>
      <td>1.0.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Join two files on a common column, allowing a certain difference.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>join_files_on_column_fuzzy</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/text_processing/join_files_on_column_fuzzy</td>
    </tr>
    <tr>
      <td></td>
      <td>split_file_on_column</td>
      <td>0.6</td>
      <td></td>
      <td>gawk</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Split a file on a specific column.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>split_file_on_column</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/text_processing/split_file_on_column</td>
    </tr>
    <tr>
      <td></td>
      <td>split_file_to_collection</td>
      <td>0.5.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Split tabular, MGF, FASTA, or FASTQ files to a dataset collection.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>split_file_to_collection</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/text_processing/split_file_to_collection</td>
    </tr>
    <tr>
      <td></td>
      <td>text_processing</td>
      <td>9.3</td>
      <td>8.25</td>
      <td>coreutils</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>High performance text processing tools using the GNU coreutils, sed, awk and friends.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.gnu.org/software/</td>
      <td>Text Manipulation</td>
      <td>text_processing</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/text_processing/text_processing</td>
    </tr>
    <tr>
      <td></td>
      <td>tgsgapcloser</td>
      <td>1.0.3</td>
      <td>1.2.1</td>
      <td>tgsgapcloser</td>
      <td>To update</td>
      <td>TGS-GapCloser</td>
      <td>TGS-GapCloser</td>
      <td>Genome assembly, Read mapping, Scaffolding, Localised reassembly</td>
      <td>Sequencing, Sequence assembly, Phylogeny, Transcription factors and regulatory sites, Mapping</td>
      <td>TGS-GapCloser uses error-prone long reads or preassembled contigs to fill N-gap in the genome assembly.</td>
      <td>TGS-GapCloser is a fast and accurately passing through the Bermuda in large genome using error-prone third-generation long reads.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/BGI-Qingdao/TGS-GapCloser</td>
      <td>Assembly</td>
      <td>tgsgapcloser</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/tgsgapcloser</td>
    </tr>
    <tr>
      <td></td>
      <td>tool_recommendation_model</td>
      <td>0.0.5</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Create model to recommend tools</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bgruening/galaxytools</td>
      <td>Machine Learning</td>
      <td>create_tool_recommendation_model</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/recommendation_training/tools/tool_recommendation_model</td>
    </tr>
    <tr>
      <td></td>
      <td>trim_galore</td>
      <td>0.6.7</td>
      <td>0.6.10</td>
      <td>trim-galore</td>
      <td>To update</td>
      <td>trim_galore</td>
      <td>Trim Galore</td>
      <td>Sequence trimming</td>
      <td>Sequence analysis</td>
      <td>Trim Galore adaptive quality and adapter trimmer</td>
      <td>A wrapper tool around Cutadapt and FastQC to consistently apply quality and adapter trimming to FastQ files, with some extra functionality for MspI-digested RRBS-type (Reduced Representation Bisufite-Seq) libraries.</td>
      <td></td>
      <td>To update</td>
      <td>http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/</td>
      <td>Sequence Analysis, Fastq Manipulation</td>
      <td>trim_galore</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/trim_galore</td>
    </tr>
    <tr>
      <td></td>
      <td>uniprot_rest_interface</td>
      <td>0.4</td>
      <td></td>
      <td>requests</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>UniProt ID mapping and sequence retrieval</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/jdrudolph/uniprot</td>
      <td>Proteomics, Sequence Analysis</td>
      <td>uniprot_rest_interface</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/uniprot_rest_interface</td>
    </tr>
    <tr>
      <td></td>
      <td>vt</td>
      <td>0.2</td>
      <td>2015.11.10</td>
      <td>vt</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A tool set for short variant discovery in genetic sequence data.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>vt</td>
      <td>bgruening</td>
      <td>https://github.com/atks/vt</td>
    </tr>
    <tr>
      <td></td>
      <td>wtdbg</td>
      <td>2.5</td>
      <td>2.5</td>
      <td>wtdbg</td>
      <td>Up-to-date</td>
      <td>wtdbg2</td>
      <td>wtdbg2</td>
      <td>Genome assembly, De-novo assembly</td>
      <td>Sequence assembly, Sequencing</td>
      <td>WTDBG is a fuzzy Bruijn graph (FBG) approach to long noisy reads assembly.</td>
      <td>Wtdbg2 is a de novo sequence assembler for long noisy reads produced by PacBio or Oxford Nanopore Technologies (ONT). It assembles raw reads without error correction and then builds the consensus from intermediate assembly output. Wtdbg2 is able to assemble the human and even the 32Gb Axolotl genome at a speed tens of times faster than CANU and FALCON while producing contigs of comparable base accuracy.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/ruanjue/wtdbg2</td>
      <td>Assembly</td>
      <td>wtdbg</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/wtdbg</td>
    </tr>
    <tr>
      <td></td>
      <td>align_back_trans</td>
      <td>0.0.10</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Thread nucleotides onto a protein alignment (back-translation)</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/align_back_trans</td>
      <td>Fasta Manipulation, Sequence Analysis</td>
      <td>align_back_trans</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/align_back_trans</td>
    </tr>
    <tr>
      <td></td>
      <td>chromosome_diagram</td>
      <td>0.0.3</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Chromosome Diagrams using Biopython</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Graphics, Sequence Analysis, Visualization</td>
      <td>chromosome_diagram</td>
      <td>peterjc</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>clc_assembly_cell</td>
      <td>0.0.7</td>
      <td>1.19.2</td>
      <td>samtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Galaxy wrapper for the CLC Assembly Cell suite from CLCBio</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/clc_assembly_cell</td>
      <td>Assembly, Next Gen Mappers, SAM</td>
      <td>clc_assembly_cell</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/clc_assembly_cell</td>
    </tr>
    <tr>
      <td></td>
      <td>clinod</td>
      <td>0.1.0</td>
      <td>1.3</td>
      <td>clinod</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>NoD: a Nucleolar localization sequence detector for eukaryotic and viral proteins</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.compbio.dundee.ac.uk/www-nod/</td>
      <td>Sequence Analysis</td>
      <td>clinod</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/clinod</td>
    </tr>
    <tr>
      <td></td>
      <td>count_roi_variants</td>
      <td>0.0.6</td>
      <td>1.19.2</td>
      <td>samtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Count sequence variants in region of interest in BAM file</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/count_roi_variants</td>
      <td>Assembly, SAM</td>
      <td>count_roi_variants</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/count_roi_variants</td>
    </tr>
    <tr>
      <td></td>
      <td>coverage_stats</td>
      <td>0.1.0</td>
      <td>1.19.2</td>
      <td>samtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>BAM coverage statistics using samtools idxstats and depth</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/coverage_stats</td>
      <td>Assembly, SAM</td>
      <td>coverage_stats</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/coverage_stats</td>
    </tr>
    <tr>
      <td></td>
      <td>effectiveT3</td>
      <td>0.0.20</td>
      <td>1.0.1</td>
      <td>effectiveT3</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Find bacterial type III effectors in protein sequences</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://effectors.org</td>
      <td>Sequence Analysis</td>
      <td>effectivet3</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/effectiveT3</td>
    </tr>
    <tr>
      <td></td>
      <td>fasta_filter_by_id</td>
      <td>0.0.7</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter FASTA sequences by ID (DEPRECATED)</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/fasta_filter_by_id</td>
      <td>Fasta Manipulation, Sequence Analysis, Text Manipulation</td>
      <td>fasta_filter_by_id</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/fasta_filter_by_id</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_filter_by_id</td>
      <td>0.0.7</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter FASTQ sequences by ID (DEPRECATED)</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/fastq_filter_by_id</td>
      <td>Fastq Manipulation, Sequence Analysis, Text Manipulation</td>
      <td>fastq_filter_by_id</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/fastq_filter_by_id</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_pair_names</td>
      <td>0.0.5</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Extract FASTQ paired read names</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/fastq_pair_names</td>
      <td>Sequence Analysis</td>
      <td>fastq_pair_names</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/fastq_pair_names</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_paired_unpaired</td>
      <td>0.1.5</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Divide FASTQ file into paired and unpaired reads</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/fastq_paired_unpaired</td>
      <td>Sequence Analysis, Text Manipulation</td>
      <td>fastq_paired_unpaired</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/fastq_paired_unpaired</td>
    </tr>
    <tr>
      <td></td>
      <td>get_orfs_or_cdss</td>
      <td>0.2.3</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Search nucleotide sequences for open reading frames (ORFs), or coding sequences (CDSs)</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/get_orfs_or_cdss</td>
      <td>Sequence Analysis</td>
      <td>get_orfs_or_cdss</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/get_orfs_or_cdss</td>
    </tr>
    <tr>
      <td></td>
      <td>mummer</td>
      <td>0.0.8</td>
      <td>9.18</td>
      <td>ghostscript</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Draw dotplots using mummer, mucmer, or promer with mummerplot</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://mummer.sourceforge.net/</td>
      <td>Graphics, Sequence Analysis, Visualization</td>
      <td>mummer</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/mummer</td>
    </tr>
    <tr>
      <td></td>
      <td>nlstradamus</td>
      <td>0.0.11</td>
      <td>1.8</td>
      <td>NLStradamus</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Find nuclear localization signals (NLSs) in protein sequences</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.moseslab.csb.utoronto.ca/NLStradamus</td>
      <td>Fasta Manipulation, Sequence Analysis</td>
      <td>nlstradamus</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/nlstradamus</td>
    </tr>
    <tr>
      <td></td>
      <td>predictnls</td>
      <td>0.0.10</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Python reimplementation of predictNLS for Galaxy</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/predictnls</td>
      <td>Sequence Analysis</td>
      <td>predictnls</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/predictnls</td>
    </tr>
    <tr>
      <td></td>
      <td>protein_analysis</td>
      <td>0.0.13</td>
      <td></td>
      <td>promoter</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>TMHMM, SignalP, Promoter, RXLR motifs, WoLF PSORT and PSORTb</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/protein_analysis</td>
      <td>Sequence Analysis</td>
      <td>tmhmm_and_signalp</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/protein_analysis</td>
    </tr>
    <tr>
      <td></td>
      <td>sample_seqs</td>
      <td>0.2.6</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Sub-sample sequences files (e.g. to reduce coverage)</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/sample_seqs</td>
      <td>Assembly, Fasta Manipulation, Fastq Manipulation, Sequence Analysis</td>
      <td>sample_seqs</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/sample_seqs</td>
    </tr>
    <tr>
      <td></td>
      <td>samtools_depad</td>
      <td>0.0.5</td>
      <td>1.19.2</td>
      <td>samtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Re-align a SAM/BAM file with a padded reference (using samtools depad)</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.htslib.org/</td>
      <td>Assembly, SAM, Sequence Analysis</td>
      <td>samtools_depad</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/samtools_depad</td>
    </tr>
    <tr>
      <td></td>
      <td>samtools_depth</td>
      <td>0.0.3</td>
      <td>1.19.2</td>
      <td>samtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Coverage depth via samtools</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.htslib.org/</td>
      <td>Assembly, Sequence Analysis, SAM</td>
      <td>samtools_depth</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/samtools_depth</td>
    </tr>
    <tr>
      <td></td>
      <td>samtools_idxstats</td>
      <td>0.0.6</td>
      <td>1.19.2</td>
      <td>samtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>BAM mapping statistics (using samtools idxstats)</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.htslib.org/</td>
      <td>Assembly, Next Gen Mappers, SAM</td>
      <td>samtools_idxstats</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/samtools_idxstats</td>
    </tr>
    <tr>
      <td></td>
      <td>seq_composition</td>
      <td>0.0.5</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Sequence composition</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_composition</td>
      <td>Sequence Analysis</td>
      <td>seq_composition</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_composition</td>
    </tr>
    <tr>
      <td></td>
      <td>seq_filter_by_id</td>
      <td>0.2.9</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter sequences by ID</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_filter_by_id</td>
      <td>Fasta Manipulation, Sequence Analysis, Text Manipulation</td>
      <td>seq_filter_by_id</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_filter_by_id</td>
    </tr>
    <tr>
      <td></td>
      <td>seq_filter_by_mapping</td>
      <td>0.0.8</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter sequencing reads using SAM/BAM mapping files</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_filter_by_mapping</td>
      <td>Assembly, Fasta Manipulation, Fastq Manipulation, SAM, Sequence Analysis</td>
      <td>seq_filter_by_mapping</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_filter_by_mapping</td>
    </tr>
    <tr>
      <td></td>
      <td>seq_length</td>
      <td>0.0.5</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute sequence length (from FASTA, QUAL, FASTQ, SFF, etc)</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_length</td>
      <td>Fasta Manipulation, Fastq Manipulation, Sequence Analysis</td>
      <td>seq_length</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_length</td>
    </tr>
    <tr>
      <td></td>
      <td>seq_primer_clip</td>
      <td>0.0.18</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Trim off 5' or 3' primers</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_primer_clip</td>
      <td>Assembly, Fasta Manipulation, Text Manipulation</td>
      <td>seq_primer_clip</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_primer_clip</td>
    </tr>
    <tr>
      <td></td>
      <td>seq_rename</td>
      <td>0.0.10</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Rename sequences with ID mapping from a tabular file</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_rename</td>
      <td>Fasta Manipulation, Sequence Analysis, Text Manipulation</td>
      <td>seq_rename</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_rename</td>
    </tr>
    <tr>
      <td></td>
      <td>seq_select_by_id</td>
      <td>0.0.15</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Select sequences by ID</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_select_by_id</td>
      <td>Fasta Manipulation, Sequence Analysis, Text Manipulation</td>
      <td>seq_select_by_id</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_select_by_id</td>
    </tr>
    <tr>
      <td></td>
      <td>venn_list</td>
      <td>0.1.2</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Draw Venn Diagram (PDF) from lists, FASTA files, etc</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/venn_list</td>
      <td>Graphics, Sequence Analysis, Visualization</td>
      <td>venn_list</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/venn_list</td>
    </tr>
    <tr>
      <td></td>
      <td>TrimNs</td>
      <td>0.1.0</td>
      <td>1.0</td>
      <td>trimns_vgp</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>TrimNs is used to trim and remove fake cut sites from bionano hybrid scaffold data in the VGP pipeline</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/VGP/vgp-assembly/tree/master/pipeline/bionano/trimNs</td>
      <td>Assembly</td>
      <td>trimns</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/trimN</td>
    </tr>
    <tr>
      <td></td>
      <td>abricate</td>
      <td>1.0.1</td>
      <td>1.0.1</td>
      <td>abricate</td>
      <td>Up-to-date</td>
      <td>ABRicate</td>
      <td>ABRicate</td>
      <td>Antimicrobial resistance prediction</td>
      <td>Genomics, Microbiology</td>
      <td>Mass screening of contigs for antiobiotic resistance genes</td>
      <td>Mass screening of contigs for antimicrobial resistance or virulence genes.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/tseemann/abricate</td>
      <td>Sequence Analysis</td>
      <td>abricate</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/abricate/</td>
    </tr>
    <tr>
      <td></td>
      <td>abritamr</td>
      <td>1.0.14</td>
      <td>1.0.14</td>
      <td>abritamr</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A pipeline for running AMRfinderPlus and collating results into functional classes</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://zenodo.org/record/7370628</td>
      <td>Sequence Analysis</td>
      <td>abritamr</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/abritamr</td>
    </tr>
    <tr>
      <td></td>
      <td>abyss</td>
      <td>2.3.7</td>
      <td>2.3.7</td>
      <td>abyss</td>
      <td>Up-to-date</td>
      <td>abyss</td>
      <td>ABySS</td>
      <td>Genome assembly, De-novo assembly, Scaffolding</td>
      <td>Sequence assembly</td>
      <td>Assembly By Short Sequences - a de novo, parallel, paired-end sequence assembler</td>
      <td>De novo genome sequence assembler using short reads.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://www.bcgsc.ca/platform/bioinfo/software/abyss</td>
      <td>Assembly</td>
      <td>abyss</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/abyss</td>
    </tr>
    <tr>
      <td></td>
      <td>adapter_removal</td>
      <td>2.3.3</td>
      <td>2.3.3</td>
      <td>adapterremoval</td>
      <td>Up-to-date</td>
      <td>adapterremoval</td>
      <td>AdapterRemoval</td>
      <td>Sequence trimming, Sequence merging, Primer removal</td>
      <td></td>
      <td>Removes residual adapter sequences from single-end (SE) or paired-end (PE) FASTQ reads.</td>
      <td>AdapterRemoval searches for and removes adapter sequences from High-Throughput Sequencing (HTS) data and (optionally) trims low quality bases from the 3' end of reads following adapter removal. AdapterRemoval can analyze both single end and paired end data, and can be used to merge overlapping paired-ended reads into (longer) consensus sequences. Additionally, AdapterRemoval can construct a consensus adapter sequence for paired-ended reads, if which this information is not available.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/MikkelSchubert/adapterremoval</td>
      <td>Fasta Manipulation, Sequence Analysis</td>
      <td>adapter_removal</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/adapter_removal/</td>
    </tr>
    <tr>
      <td></td>
      <td>add_input_name_as_column</td>
      <td>0.2.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Add input name as column on an existing tabular file</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/add_input_name_as_column</td>
      <td>Text Manipulation</td>
      <td>add_input_name_as_column</td>
      <td>mvdbeek</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/add_input_name_as_column</td>
    </tr>
    <tr>
      <td></td>
      <td>aegean</td>
      <td>0.16.0</td>
      <td>0.16.0</td>
      <td>aegean</td>
      <td>Up-to-date</td>
      <td>gaeval</td>
      <td>GAEVAL</td>
      <td>Sequence annotation</td>
      <td>Sequence analysis, Gene structure</td>
      <td>AEGeAn toolkit wrappers</td>
      <td>Gene Annotation EVAluation.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/BrendelGroup/AEGeAn</td>
      <td>Transcriptomics, Sequence Analysis</td>
      <td>aegean</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/aegean</td>
    </tr>
    <tr>
      <td></td>
      <td>aldex2</td>
      <td>1.26.0</td>
      <td>1.34.0</td>
      <td>bioconductor-aldex2</td>
      <td>To update</td>
      <td>aldex2</td>
      <td>ALDEx2</td>
      <td>Statistical inference</td>
      <td>Gene expression, Statistics and probability</td>
      <td>Performs analysis Of differential abundance taking sample variation into account</td>
      <td>A differential abundance analysis for the comparison of two or more conditions. It uses a Dirichlet-multinomial model to infer abundance from counts, that has been optimized for three or more experimental replicates. Infers sampling variation and calculates the expected FDR given the biological and sampling variation using the Wilcox rank test and Welches t-test, or the glm and Kruskal Wallis tests. Reports both P and fdr values calculated by the Benjamini Hochberg correction.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ggloor/ALDEx_bioc</td>
      <td>Metagenomics</td>
      <td>aldex2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/aldex2</td>
    </tr>
    <tr>
      <td></td>
      <td>allegro</td>
      <td>@VER@.0</td>
      <td>3</td>
      <td>allegro</td>
      <td>To update</td>
      <td>allegro</td>
      <td>Allegro</td>
      <td>Sequence motif discovery</td>
      <td>Sequence analysis, Transcription factors and regulatory sites, DNA</td>
      <td>Linkage and haplotype analysis from deCODE</td>
      <td>It does simultaneous discovery of cis-regulatory motifs and their associated expression profiles. Its input are DNA sequences (typically, promoters or 3 UTRs) and genome-wide expression profiles. Its output is the set of motifs found, and for each motif the set of genes it regulates (its transcriptional module). It is highly efficient and can analyze expression profiles of thousands of genes, measured across dozens of experimental conditions, along with all regulatory sequences in the genome.</td>
      <td></td>
      <td>To update</td>
      <td>http://www.decode.com/software/</td>
      <td>Variant Analysis</td>
      <td>allegro</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/allegro/</td>
    </tr>
    <tr>
      <td></td>
      <td>amplican</td>
      <td>1.14.0</td>
      <td>1.24.0</td>
      <td>bioconductor-amplican</td>
      <td>To update</td>
      <td>amplican</td>
      <td>amplican</td>
      <td>Alignment, Standardisation and normalisation</td>
      <td>PCR experiment, Statistics and probability</td>
      <td>AmpliCan is an analysis tool for genome editing.</td>
      <td>It performs alignment of the amplicon reads, normalizes gathered data, calculates multiple statistics (e.g. cut rates, frameshifts) and presents results in form of aggregated reports. Data and statistics can be broken down by experiments, barcodes, user defined groups, guides and amplicons allowing for quick identification of potential problems.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/valenlab/amplican</td>
      <td>Sequence Analysis</td>
      <td>amplican</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/amplican</td>
    </tr>
    <tr>
      <td></td>
      <td>ampvis2</td>
      <td>2.8.6</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>ampvis</td>
      <td>ampvis</td>
      <td>Analysis, Visualisation</td>
      <td>Biodiversity</td>
      <td>ampvis2</td>
      <td>ampvis2 is an R-package to conveniently visualise and analyse 16S rRNA amplicon data in different ways.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/MadsAlbertsen/ampvis2/</td>
      <td>Metagenomics</td>
      <td>ampvis2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ampvis2</td>
    </tr>
    <tr>
      <td></td>
      <td>amrfinderplus</td>
      <td>3.11.26</td>
      <td>3.12.8</td>
      <td>ncbi-amrfinderplus</td>
      <td>To update</td>
      <td>amrfinderplus</td>
      <td>AMRFinderPlus</td>
      <td>Antimicrobial resistance prediction</td>
      <td>Microbiology, Public health and epidemiology, Infectious disease</td>
      <td>"AMRFinderPlus is designed to find acquired antimicrobial resistance genes and point mutations in protein and/or assembled nucleotide sequences.It can also search "plus", stress, heat, and biocide resistance and virulence factors for some organisms.</td>
      <td>AMRFinderPlus is designed to find acquired antimicrobial resistance genes and point mutations in protein and/or assembled nucleotide sequences.It can also search "plus", stress, heat, and biocide resistance and virulence factors for some organisms</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ncbi/amr</td>
      <td>Sequence Analysis</td>
      <td>AMRFinderPlus</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/amrfinderplus</td>
    </tr>
    <tr>
      <td></td>
      <td>ancombc</td>
      <td>1.4.0</td>
      <td>2.4.0</td>
      <td>bioconductor-ancombc</td>
      <td>To update</td>
      <td>ancombc</td>
      <td>ANCOMBC</td>
      <td>DNA barcoding</td>
      <td>Microbial ecology, Metagenomics</td>
      <td>Performs analysis of compositions of microbiomes with bias correction.</td>
      <td>Determine taxa whose absolute abundances, per unit volume, of the ecosystem (e.g. gut) are significantly different with changes in the covariate of interest (e.g. group). The current version of ancombc function implements Analysis of Compositions of Microbiomes with Bias Correction (ANCOM-BC) in cross-sectional data while allowing for covariate adjustment.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/FrederickHuangLin/ANCOMBC</td>
      <td>Metagenomics</td>
      <td>ancombc</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ancombc</td>
    </tr>
    <tr>
      <td></td>
      <td>anndata</td>
      <td>0.10.3</td>
      <td>0.6.22.post1</td>
      <td>anndata</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Import, Export, Inspect and Manipulate Anndata and Loom objects</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://anndata.readthedocs.io</td>
      <td>Transcriptomics, Sequence Analysis</td>
      <td>anndata</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/anndata/</td>
    </tr>
    <tr>
      <td></td>
      <td>annotatemyids</td>
      <td>3.18.0</td>
      <td>3.18.0</td>
      <td>bioconductor-org.hs.eg.db</td>
      <td>Up-to-date</td>
      <td>annotatemyids</td>
      <td>annotatemyids</td>
      <td>Annotation</td>
      <td></td>
      <td>annotateMyIDs: get annotation for a set of IDs using the Bioconductor annotation packages</td>
      <td>This tool can get annotation for a generic set of IDs, using the Bioconductor annotation data packages. Supported organisms are human, mouse, rat, fruit fly and zebrafish. The org.db packages that are used here are primarily based on mapping using Entrez Gene identifiers. More information on the annotation packages can be found at the Bioconductor website, for example, information on the human annotation package (org.Hs.eg.db) can be found here.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/annotatemyids</td>
      <td>Genome annotation</td>
      <td>annotatemyids</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/annotatemyids</td>
    </tr>
    <tr>
      <td></td>
      <td>arriba</td>
      <td>2.4.0</td>
      <td>2.4.0</td>
      <td>arriba</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Arriba detects fusion genes in RNA-Seq data after running RNA-STAR</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/suhrig/arriba</td>
      <td>Sequence Analysis, Transcriptomics</td>
      <td>arriba</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/arriba</td>
    </tr>
    <tr>
      <td></td>
      <td>art</td>
      <td>2014.11.03.0</td>
      <td>2016.06.05</td>
      <td>art</td>
      <td>To update</td>
      <td>art</td>
      <td>ART</td>
      <td>Conversion</td>
      <td>Bioinformatics</td>
      <td>Simulator for Illumina, 454, and SOLiD sequencing data</td>
      <td>ART is a set of simulation tools to generate synthetic next-generation sequencing reads. ART simulates sequencing reads by mimicking real sequencing process with empirical error models or quality profiles summarized from large recalibrated sequencing data. ART can also simulate reads using user own read error model or quality profiles. ART supports simulation of single-end, paired-end/mate-pair reads of three major commercial next-generation sequencing platforms. Illuminas Solexa, Roches 454 and Applied Biosystems SOLiD</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Data Source</td>
      <td>art</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/art</td>
    </tr>
    <tr>
      <td></td>
      <td>artic</td>
      <td></td>
      <td>1.2.4</td>
      <td>artic</td>
      <td>To update</td>
      <td>artic</td>
      <td>ARTIC</td>
      <td>Sequence alignment</td>
      <td>Genomics</td>
      <td>The artic pipeline is designed to help run the artic bioinformatics protocols;for example the nCoV-2019 novel coronavirus protocol.Features include: read filtering, primer trimming, amplicon coverage normalisation,variant calling and consensus building</td>
      <td>A bioinformatics pipeline for working with virus sequencing data sequenced with nanopore</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/artic-network/fieldbioinformatics</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/artic</td>
    </tr>
    <tr>
      <td></td>
      <td>assembly-stats</td>
      <td>17.02</td>
      <td>17.02</td>
      <td>rjchallis-assembly-stats</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Assembly metric visualisations to facilitate rapid assessment and comparison of assembly quality.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/rjchallis/assembly-stats</td>
      <td>Assembly</td>
      <td>assembly_stats</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/assembly-stats</td>
    </tr>
    <tr>
      <td></td>
      <td>augustus</td>
      <td>3.4.0</td>
      <td>3.5.0</td>
      <td>augustus</td>
      <td>To update</td>
      <td>augustus</td>
      <td>AUGUSTUS</td>
      <td>Gene prediction, Ab-initio gene prediction, Homology-based gene prediction, Homology-based gene prediction, Operation</td>
      <td>Gene transcripts, Gene and protein families</td>
      <td>AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences.</td>
      <td>AUGUSTUS is a eukaryotic gene prediction tool. It can integrate evidence, e.g. from RNA-Seq, ESTs, proteomics, but can also predict genes ab initio. The PPX extension to AUGUSTUS can take a protein sequence multiple sequence alignment as input to find new members of the family in a genome. It can be run through a web interface (see https://bio.tools/webaugustus), or downloaded and run locally.</td>
      <td></td>
      <td>To update</td>
      <td>http://bioinf.uni-greifswald.de/augustus/</td>
      <td>Sequence Analysis</td>
      <td>augustus</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/augustus</td>
    </tr>
    <tr>
      <td></td>
      <td>b2btools</td>
      <td>3.0.5+galaxy0</td>
      <td>3.0.6</td>
      <td>b2btools</td>
      <td>To update</td>
      <td>b2btools</td>
      <td>b2bTools</td>
      <td>Protein disorder prediction, Protein secondary structure prediction, Protein feature detection</td>
      <td></td>
      <td>This software suite provides structural predictions for protein sequences made by Bio2Byte group.About Bio2Byte: We investigate how the dynamics, conformational states, and available experimental data of proteins relate to their amino acid sequence.Underlying physical and chemical principles are computationally unraveled through data integration, analysis, and machine learning, so connecting themto biological events and improving our understanding of the way proteins work.</td>
      <td>The bio2byte tools server (b2btools) offers the following single protein sequence based predictions:- Backbone and sidechain dynamics (DynaMine)- Helix, sheet, coil and polyproline-II propensity- Early folding propensity (EFoldMine)- Disorder (DisoMine)- Beta-sheet aggregation (Agmata)In addition, multiple sequence alignments (MSAs) can be uploaded to scan the 'biophysical limits' of a protein family as defined in the MSA</td>
      <td></td>
      <td>To update</td>
      <td>https://bio2byte.be</td>
      <td>Computational chemistry, Molecular Dynamics, Proteomics, Sequence Analysis, Synthetic Biology</td>
      <td></td>
      <td>iuc</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>bakta</td>
      <td>1.9.2</td>
      <td>1.9.2</td>
      <td>bakta</td>
      <td>Up-to-date</td>
      <td>Bakta</td>
      <td>Bakta</td>
      <td>Genome annotation</td>
      <td>Genomics, Data submission, annotation and curation, Sequence analysis</td>
      <td>"Bakta is a tool for the rapid &amp; standardized annotation of bacterial genomes and plasmids from both isolates and MAGs.It provides dbxref-rich and sORF-including annotations in machine-readable JSON &amp; bioinformatics standard file formats for automatic downstream analysis."</td>
      <td>Rapid &amp; standardized annotation of bacterial genomes, MAGs &amp; plasmids</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/oschwengers/bakta</td>
      <td>Sequence Analysis</td>
      <td>bakta</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/bakta</td>
    </tr>
    <tr>
      <td></td>
      <td>bam_to_scidx</td>
      <td>1.0.1</td>
      <td></td>
      <td>openjdk</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Contains a tool that converts a BAM file to an ScIdx file.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/seqcode/cegr-tools/tree/master/src/org/seqcode/cegrtools/bamtoscidx</td>
      <td>Convert Formats</td>
      <td>bam_to_scidx</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bam_to_scidx</td>
    </tr>
    <tr>
      <td></td>
      <td>bamutil</td>
      <td></td>
      <td>1.0.15</td>
      <td>bamutil</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>bamUtil is a repository that contains several programs that perform operations on SAM/BAM files.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/statgen/bamUtil</td>
      <td>Sequence Analysis</td>
      <td>bamutil</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bamutil</td>
    </tr>
    <tr>
      <td></td>
      <td>bandage</td>
      <td>2022.09</td>
      <td>2022.09</td>
      <td>bandage_ng</td>
      <td>Up-to-date</td>
      <td>bandage</td>
      <td>Bandage</td>
      <td>Sequence assembly visualisation</td>
      <td>Genomics, Sequence assembly</td>
      <td>Bandage - A Bioinformatics Application for Navigating De novo Assembly Graphs Easily</td>
      <td>GUI program that allows users to interact with the assembly graphs made by de novo assemblers such as Velvet, SPAdes, MEGAHIT and others. It visualises assembly graphs, with connections, using graph layout algorithms.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/rrwick/Bandage</td>
      <td>Visualization</td>
      <td>bandage</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bandage</td>
    </tr>
    <tr>
      <td></td>
      <td>barcode_splitter</td>
      <td>0.18.4.0</td>
      <td>0.18.6</td>
      <td>barcode_splitter</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A utility to split sequence files using multiple sets of barcodes</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/princeton_genomics/barcode_splitter/</td>
      <td>Fastq Manipulation</td>
      <td>barcode_splitter</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/barcode_splitter</td>
    </tr>
    <tr>
      <td></td>
      <td>baredsc</td>
      <td>1.1.2</td>
      <td>1.1.2</td>
      <td>baredsc</td>
      <td>Up-to-date</td>
      <td>baredsc</td>
      <td>baredSC</td>
      <td>Data retrieval, Expression correlation analysis, Differential gene expression profiling</td>
      <td>RNA-Seq, Cytometry, Transcriptomics, Gene transcripts, Statistics and probability</td>
      <td>baredSC is a tool that uses a Monte-Carlo Markov Chain to estimate a confidence interval on the probability density function (PDF) of expression of one or two genes from single-cell RNA-seq data.</td>
      <td>The baredSC (Bayesian Approach to Retreive Expression Distribution of Single Cell) is a tool that uses a Monte-Carlo Markov Chain to estimate a confidence interval on the probability density function (PDF) of expression of one or two genes from single-cell RNA-seq data.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/lldelisle/baredSC</td>
      <td>Transcriptomics, Visualization</td>
      <td>baredsc</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/baredsc</td>
    </tr>
    <tr>
      <td></td>
      <td>barrnap</td>
      <td>1.2.2</td>
      <td>0.9</td>
      <td>barrnap</td>
      <td>To update</td>
      <td>barrnap</td>
      <td>Barrnap</td>
      <td>Gene prediction</td>
      <td>Genomics, Model organisms, Model organisms</td>
      <td>Contains the Barrnap tool for finding ribosomal RNAs in FASTA sequences.</td>
      <td>Predict the location of ribosomal RNA genes in genomes. It supports bacteria (5S,23S,16S), archaea (5S,5.8S,23S,16S), mitochondria (12S,16S) and eukaryotes (5S,5.8S,28S,18S).</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>barrnap</td>
      <td>iuc</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>basil</td>
      <td>1.2.0</td>
      <td>1.2.0</td>
      <td>anise_basil</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Breakpoint detection, including large insertions</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/seqan/anise_basil</td>
      <td>Variant Analysis</td>
      <td>basil</td>
      <td>iuc</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>bax2bam</td>
      <td>0.0.11</td>
      <td>0.0.11</td>
      <td>bax2bam</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>BAX to BAM converter</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/pacificbiosciences/bax2bam/</td>
      <td>Convert Formats, Sequence Analysis</td>
      <td>bax2bam</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pax2bam</td>
    </tr>
    <tr>
      <td></td>
      <td>bayescan</td>
      <td>2.1</td>
      <td>2.0.1</td>
      <td>bayescan</td>
      <td>To update</td>
      <td>bayescan</td>
      <td>BayeScan</td>
      <td>Statistical inference</td>
      <td>Genetics, Evolutionary biology, Statistics and probability, DNA polymorphism</td>
      <td>Detecting natural selection from population-based genetic data</td>
      <td>BAYEsian genome SCAN for outliers, aims at identifying candidate loci under natural selection from genetic data, using differences in allele frequencies between populations. It is based on the multinomial-Dirichlet model.</td>
      <td></td>
      <td>To update</td>
      <td>http://cmpg.unibe.ch/software/BayeScan/index.html</td>
      <td>Sequence Analysis</td>
      <td>bayescan</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bayescan/</td>
    </tr>
    <tr>
      <td></td>
      <td>bbtools</td>
      <td>39.06</td>
      <td>39.06</td>
      <td>bbmap</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>BBTools is a suite of fast, multithreaded bioinformatics tools designed for analysis of DNA and RNA sequence data.BBTools can handle common sequencing file formats such as fastq, fasta, sam, scarf, fasta+qual, compressed or raw,with autodetection of quality encoding and interleaving. It is written in Java and works on any platform supportingJava, including Linux, MacOS, and Microsoft Windows and Linux; there are no dependencies other than Java (version7 or higher). Program descriptions and options are shown when running the shell scripts with no parameters.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://jgi.doe.gov/data-and-tools/bbtools/</td>
      <td>Sequence Analysis</td>
      <td>bbtools</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bbtools</td>
    </tr>
    <tr>
      <td></td>
      <td>bcftools</td>
      <td>1.15.1</td>
      <td>1.19</td>
      <td>bcftools</td>
      <td>To update</td>
      <td>bcftools</td>
      <td>BCFtools</td>
      <td>Data handling, Variant calling</td>
      <td>Genetic variation, DNA polymorphism, GWAS study, Genotyping experiment</td>
      <td>BCFtools toolkit wrappers</td>
      <td>BCFtools is a set of utilities that manipulate variant calls in the Variant Call Format (VCF) and its binary counterpart BCF. All commands work transparently with both VCFs and BCFs, both uncompressed and BGZF-compressed.</td>
      <td></td>
      <td>To update</td>
      <td>https://samtools.github.io/bcftools/</td>
      <td>Variant Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bcftools</td>
    </tr>
    <tr>
      <td></td>
      <td>bctools</td>
      <td>0.2.2</td>
      <td>0.2.2</td>
      <td>bctools</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>bctools is a set of tools for handling barcodes and UMIs in NGS data.bctools can be used to merge PCR duplicates according to unique molecular barcodes (UMIs),to extract barcodes from arbitrary positions relative to the read starts,to clean up readthroughs into UMIs with paired-end sequencing andhandles binary barcodes as used with uvCLAP and FLASH.License: Apache License 2.0</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/dmaticzka/bctools</td>
      <td>Sequence Analysis, Transcriptomics</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bedtools</td>
    </tr>
    <tr>
      <td></td>
      <td>beacon2</td>
      <td>2.0.0</td>
      <td>2.0.0</td>
      <td>beacon2-ri-tools</td>
      <td>Up-to-date</td>
      <td>GA4GH Beacon</td>
      <td></td>
      <td></td>
      <td></td>
      <td>beacon2-ri-tools are part of the ELIXIR-CRG Beacon v2 Reference Implementation (B2RI).</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/EGA-archive/beacon2-ri-tools/tree/main</td>
      <td>Variant Analysis</td>
      <td>beacon2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/beacon2</td>
    </tr>
    <tr>
      <td></td>
      <td>beagle</td>
      <td>5.2_21Apr21.304</td>
      <td>5.4_22Jul22.46e</td>
      <td>beagle</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Beagle is a program for phasing and imputing missing genotypes.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://faculty.washington.edu/browning/beagle/beagle.html</td>
      <td>Variant Analysis</td>
      <td>beagle</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/beagle</td>
    </tr>
    <tr>
      <td></td>
      <td>bedops</td>
      <td>2.4.41</td>
      <td>2.4.41</td>
      <td>bedops</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>BEDOPS: high-performance genomic feature operations</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://bedops.readthedocs.io/en/latest/</td>
      <td>Genomic Interval Operations</td>
      <td>bedops_sortbed</td>
      <td>iuc</td>
      <td>https://bedops.readthedocs.io/</td>
    </tr>
    <tr>
      <td></td>
      <td>bedtools</td>
      <td>2.30.0</td>
      <td>2.31.1</td>
      <td>bedtools</td>
      <td>To update</td>
      <td>bedtools</td>
      <td>BEDTools</td>
      <td>Mapping</td>
      <td>Genomics</td>
      <td>bedtools is a powerful toolset for genome arithmetic</td>
      <td>BEDTools is an extensive suite of utilities for comparing genomic features in BED format.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/arq5x/bedtools2</td>
      <td>Genomic Interval Operations, Text Manipulation</td>
      <td>bedtools</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bedtools</td>
    </tr>
    <tr>
      <td></td>
      <td>bellerophon</td>
      <td>1.0</td>
      <td>1.0</td>
      <td>bellerophon</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter mapped reads where the mapping spans a junction, retaining the 5-prime read.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/davebx/bellerophon</td>
      <td>Sequence Analysis</td>
      <td>bellerophon</td>
      <td>iuc</td>
      <td>https://github.com/davebx/bellerophon</td>
    </tr>
    <tr>
      <td></td>
      <td>berokka</td>
      <td>0.2.3</td>
      <td>0.2.3</td>
      <td>berokka</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Berokka is used to trim, circularise, orient &amp; filter long read bacterial genome assemblies.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/tseemann/berokka</td>
      <td>Fasta Manipulation</td>
      <td>berokka</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/berokka</td>
    </tr>
    <tr>
      <td></td>
      <td>bigscape</td>
      <td>1.1.9</td>
      <td>1.1.9</td>
      <td>bigscape</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Construct sequence similarity networks of BGCs and groups them into GCF</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/medema-group/BiG-SCAPE</td>
      <td>Metagenomics</td>
      <td>bigscape</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bigscape/</td>
    </tr>
    <tr>
      <td></td>
      <td>binning_refiner</td>
      <td>1.4.3</td>
      <td>1.4.3</td>
      <td>binning_refiner</td>
      <td>Up-to-date</td>
      <td>binning_refiner</td>
      <td>Binning_refiner</td>
      <td>Read binning, Sequence clustering</td>
      <td>Metagenomics</td>
      <td>Reconciles the outputs of different binning programs with the aim to improve the quality of genome bins,especially with respect to contamination levels.</td>
      <td>Improving genome bins through the combination of different binning programs</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/songweizhi/Binning_refiner</td>
      <td>Metagenomics</td>
      <td>binning_refiner</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/binning_refiner/</td>
    </tr>
    <tr>
      <td></td>
      <td>bioext</td>
      <td>0.21.2</td>
      <td>0.21.2</td>
      <td>python-bioext</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A suite of Galaxy tools designed around the BioExt extension to BioPython. Align sequences, merge duplicate sequences into one, and more!</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://pypi.python.org/pypi/biopython-extensions/</td>
      <td>Next Gen Mappers</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/davebx/bioext-gx/</td>
    </tr>
    <tr>
      <td></td>
      <td>bioinformatics_cafe</td>
      <td>0.1.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Miscellanea of scripts for bioinformatics</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/dariober/bioinformatics-cafe/</td>
      <td>Sequence Analysis</td>
      <td>bioinformatics_cafe</td>
      <td>mbernt</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bioinformatics-cafe</td>
    </tr>
    <tr>
      <td></td>
      <td>biom_format</td>
      <td>2.1.15</td>
      <td>2.1.7</td>
      <td>biom-format</td>
      <td>To update</td>
      <td>biomformat</td>
      <td>biomformat</td>
      <td>Formatting</td>
      <td>Laboratory information management, Sequence analysis</td>
      <td>The biom-format package provides a command line interface and Python API for working with BIOM files.</td>
      <td>This package includes basic tools for reading biom-format files, accessing and subsetting data tables from a biom object, as well as limited support for writing a biom-object back to a biom-format file. The design of this API is intended to match the python API and other tools included with the biom-format project, but with a decidedly "R flavor" that should be familiar to R users. This includes S4 classes and methods, as well as extensions of common core functions/methods.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/biocore/biom-format</td>
      <td>Metagenomics</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/biom_format</td>
    </tr>
    <tr>
      <td></td>
      <td>bioperl</td>
      <td>1.1</td>
      <td>1.7.8</td>
      <td>perl-bioperl</td>
      <td>To update</td>
      <td>bioperl</td>
      <td>BioPerl</td>
      <td>Data handling, Service invocation</td>
      <td>Genomics, Software engineering, Data management</td>
      <td>Converts GenBank format files to GFF3</td>
      <td>A collection of Perl modules that facilitate the development of Perl scripts for bioinformatics applications.  It provides software modules for many of the typical tasks of bioinformatics programming.</td>
      <td></td>
      <td>To update</td>
      <td>https://bioperl.org/</td>
      <td>Sequence Analysis</td>
      <td>bp_genbank2gff3</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bioperl</td>
    </tr>
    <tr>
      <td></td>
      <td>biscot</td>
      <td>2.3.3</td>
      <td>2.3.3</td>
      <td>biscot</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Bionano scaffolding correction tool</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/institut-de-genomique/biscot</td>
      <td>Assembly</td>
      <td>biscot</td>
      <td>iuc</td>
      <td>https://github.com/bgruening/iuc/tree/master/tools/biscot</td>
    </tr>
    <tr>
      <td></td>
      <td>blast</td>
      <td>1.7.0</td>
      <td>1.7.0</td>
      <td>magicblast</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Maps large next-generation RNA or DNA sequencing runs against a whole genome or transcriptome</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://ncbi.github.io/magicblast/</td>
      <td>Next Gen Mappers</td>
      <td>magicblast</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/blast</td>
    </tr>
    <tr>
      <td></td>
      <td>blastxml_to_gapped_gff3</td>
      <td>1.1</td>
      <td>0.6.6</td>
      <td>bcbiogff</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>BlastXML to gapped GFF3</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Convert Formats, Sequence Analysis</td>
      <td>blastxml_to_gapped_gff3</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/blastxml_to_gapped_gff3</td>
    </tr>
    <tr>
      <td></td>
      <td>bowtie2</td>
      <td>2.5.3</td>
      <td>2.5.3</td>
      <td>bowtie2</td>
      <td>Up-to-date</td>
      <td>bowtie2</td>
      <td>Bowtie 2</td>
      <td>Read mapping</td>
      <td>Mapping, Genomics, Mapping</td>
      <td>Bowtie2: Fast and sensitive read alignment</td>
      <td>Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes. Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome, its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://bowtie-bio.sourceforge.net/bowtie2</td>
      <td>Next Gen Mappers</td>
      <td>bowtie2</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bowtie2</td>
    </tr>
    <tr>
      <td></td>
      <td>bracken</td>
      <td>2.9</td>
      <td>2.9</td>
      <td>bracken</td>
      <td>Up-to-date</td>
      <td>bracken</td>
      <td>Bracken</td>
      <td>Statistical calculation</td>
      <td>Metagenomics, Microbial ecology</td>
      <td>Bayesian Reestimation of Abundance with KrakEN</td>
      <td>Statistical method that computes the abundance of species in DNA sequences from a metagenomics sample.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://ccb.jhu.edu/software/bracken/</td>
      <td>Sequence Analysis, Metagenomics</td>
      <td>bracken</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bracken</td>
    </tr>
    <tr>
      <td></td>
      <td>breseq</td>
      <td>0.35.5</td>
      <td>0.38.3</td>
      <td>breseq</td>
      <td>To update</td>
      <td>breseq</td>
      <td>breseq</td>
      <td>Polymorphism detection</td>
      <td>Sequencing, Sequence analysis, DNA mutation</td>
      <td>Predicts mutations in microbial genomes</td>
      <td>Runs Breseq software on a set of fastq files.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/barricklab/breseq</td>
      <td>Variant Analysis</td>
      <td>breseq</td>
      <td>iuc</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>busco</td>
      <td>5.5.0</td>
      <td>5.6.1</td>
      <td>busco</td>
      <td>To update</td>
      <td>busco</td>
      <td>BUSCO</td>
      <td>Sequence assembly validation, Scaffolding, Genome assembly, Transcriptome assembly</td>
      <td>Sequence assembly, Genomics, Transcriptomics, Sequence analysis</td>
      <td>BUSCO assess genome and annotation completeness</td>
      <td>Provides measures for quantitative assessment of genome assembly, gene set, and transcriptome completeness based on evolutionarily informed expectations of gene content from near-universal single-copy orthologs.</td>
      <td></td>
      <td>To update</td>
      <td>https://gitlab.com/ezlab/busco/-/releases</td>
      <td>Sequence Analysis</td>
      <td>busco</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/busco/</td>
    </tr>
    <tr>
      <td></td>
      <td>bwa</td>
      <td>0.7.17</td>
      <td>0.7.17</td>
      <td>bwa</td>
      <td>Up-to-date</td>
      <td>bwa</td>
      <td>BWA</td>
      <td>Genome indexing, Sequence alignment, Read mapping, Sequence alignment, Generation, Sequence alignment, Generation, Sequence alignment, Sequence alignment</td>
      <td>Mapping</td>
      <td>Wrapper for bwa mem, aln, sampe, and samse</td>
      <td>Fast, accurate, memory-efficient aligner for short and long sequencing reads</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://bio-bwa.sourceforge.net/</td>
      <td>Next Gen Mappers</td>
      <td>bwa</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bwa</td>
    </tr>
    <tr>
      <td></td>
      <td>bwa_mem2</td>
      <td>2.2.1</td>
      <td>2.2.1</td>
      <td>bwa-mem2</td>
      <td>Up-to-date</td>
      <td>bwa-mem2</td>
      <td>Bwa-mem2</td>
      <td>Sequence alignment</td>
      <td>Mapping</td>
      <td>Bwa-mem2 is the next version of the bwa-mem algorithm in bwa.</td>
      <td>Bwa-mem2 is the next version of the bwa-mem algorithm in bwa. It produces alignment identical to bwa and is ~1.3-3.1x faster depending on the use-case,  dataset and the running machine.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/bwa-mem2/bwa-mem2</td>
      <td>Next Gen Mappers</td>
      <td>bwa_mem2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bwa_mem2</td>
    </tr>
    <tr>
      <td></td>
      <td>bwameth</td>
      <td>0.2.6</td>
      <td>0.2.7</td>
      <td>bwameth</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Fast and accurate alignment of BS-seq reads</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/brentp/bwa-meth</td>
      <td>Sequence Analysis, Next Gen Mappers</td>
      <td>bwameth</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bwameth</td>
    </tr>
    <tr>
      <td></td>
      <td>cactus</td>
      <td>2.7.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>cactus</td>
      <td>Cactus</td>
      <td>Multiple sequence alignment, Genome alignment</td>
      <td>Genomics, Sequence analysis, Phylogeny, Sequence assembly, Mapping, Phylogenetics</td>
      <td>Cactus is a reference-free whole-genome multiple alignment program</td>
      <td>Cactus is a reference-free whole-genome multiple alignment program.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ComparativeGenomicsToolkit/cactus</td>
      <td>Sequence Analysis</td>
      <td>cactus</td>
      <td>galaxy-australia</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cactus</td>
    </tr>
    <tr>
      <td></td>
      <td>calculate_contrast_threshold</td>
      <td>1.0.0</td>
      <td></td>
      <td>numpy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Calculates a contrast threshold from the CDT file generated by ``tag_pileup_frequency``. The calculated values are then used to set a uniform contrast for all the heatmaps generated downstream.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/CEGRcode/ChIP-QC-tools/tree/master/calculate_contrast_threshold</td>
      <td>Visualization, Genomic Interval Operations, SAM</td>
      <td>calculate_contrast_threshold</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/calculate_contrast_threshold</td>
    </tr>
    <tr>
      <td></td>
      <td>calculate_numeric_param</td>
      <td>0.1.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Calculate a numeric parameter value using integer and float values.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>calculate_numeric_param</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/calculate_numeric_param</td>
    </tr>
    <tr>
      <td></td>
      <td>cat</td>
      <td>5.2.3</td>
      <td>5.3</td>
      <td>cat</td>
      <td>To update</td>
      <td>cat_bins</td>
      <td>CAT and BAT</td>
      <td>Taxonomic classification, Sequence assembly, Coding region prediction</td>
      <td>Metagenomics, Metagenomic sequencing, Taxonomy, Sequence assembly</td>
      <td>Contig Annotation Tool (CAT)</td>
      <td>Contig Annotation Tool (CAT) and Bin Annotation Tool (BAT) are pipelines for the taxonomic classification of long DNA sequences and metagenome assembled genomes (MAGs/bins) of both known and (highly) unknown microorganisms, as generated by contemporary metagenomics studies. The core algorithm of both programs involves gene calling, mapping of predicted ORFs against the nr protein database, and voting-based classification of the entire contig / MAG based on classification of the individual ORFs.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/dutilh/CAT</td>
      <td>Metagenomics</td>
      <td>contig_annotation_tool</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/cat</td>
    </tr>
    <tr>
      <td></td>
      <td>cdhit</td>
      <td>4.8.1</td>
      <td>4.8.1</td>
      <td>cd-hit</td>
      <td>Up-to-date</td>
      <td>cd-hit</td>
      <td>cd-hit</td>
      <td>Sequence clustering</td>
      <td>Sequencing</td>
      <td>Cluster or compare biological sequence datasets</td>
      <td>Cluster a nucleotide dataset into representative sequences.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://weizhongli-lab.org/cd-hit/</td>
      <td>Sequence Analysis, Fasta Manipulation</td>
      <td>cd_hit</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cdhit</td>
    </tr>
    <tr>
      <td></td>
      <td>cemitool</td>
      <td>1.18.1</td>
      <td>1.26.0</td>
      <td>bioconductor-cemitool</td>
      <td>To update</td>
      <td>cemitool</td>
      <td>CEMiTool</td>
      <td>Enrichment analysis, Pathway or network analysis</td>
      <td>Gene expression, Transcriptomics, Microarray experiment</td>
      <td>Gene co-expression network analysis tool</td>
      <td>It unifies the discovery and the analysis of coexpression gene modules in a fully automatic manner, while providing a user-friendly html report with high quality graphs. Our tool evaluates if modules contain genes that are over-represented by specific pathways or that are altered in a specific sample group. Additionally, CEMiTool is able to integrate transcriptomic data with interactome information, identifying the potential hubs on each network.</td>
      <td></td>
      <td>To update</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/CEMiTool.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>cemitool</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cemitool</td>
    </tr>
    <tr>
      <td></td>
      <td>charts</td>
      <td>1.0.1</td>
      <td></td>
      <td>r-getopt</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Enables advanced visualization options in Galaxy Charts</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Visualization</td>
      <td>charts</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/charts/</td>
    </tr>
    <tr>
      <td></td>
      <td>checkm</td>
      <td>1.2.0</td>
      <td>1.2.2</td>
      <td>checkm-genome</td>
      <td>To update</td>
      <td>checkm</td>
      <td>CheckM</td>
      <td>Operation</td>
      <td>Genomics, Phylogenomics, Phylogenetics, Taxonomy, Metagenomics</td>
      <td>Assess the quality of microbial genomes recovered from isolates, single cells, and metagenomes</td>
      <td>CheckM provides a set of tools for assessing the quality of genomes recovered from isolates, single cells, or metagenomes.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/Ecogenomics/CheckM</td>
      <td>Metagenomics</td>
      <td>checkm</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/checkm</td>
    </tr>
    <tr>
      <td></td>
      <td>cherri</td>
      <td>0.7</td>
      <td>0.8</td>
      <td>cherri</td>
      <td>To update</td>
      <td>cherri</td>
      <td>cherri</td>
      <td></td>
      <td>Molecular interactions, pathways and networks, Structure analysis, Machine learning</td>
      <td>Computational Help Evaluating RNA-RNA interactions</td>
      <td>CheRRI detects functional RNA-RNA interaction (RRI) sites, by evaluating if an interaction site most likely occurs in nature. It helps to filter interaction sites generated either experimentally or by an RRI prediction algorithm by removing false positive interactions.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/BackofenLab/Cherri</td>
      <td>Transcriptomics, RNA</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cherri</td>
    </tr>
    <tr>
      <td></td>
      <td>chira</td>
      <td>1.4.3</td>
      <td>1.4.3</td>
      <td>chira</td>
      <td>Up-to-date</td>
      <td>chira</td>
      <td>ChiRA</td>
      <td></td>
      <td>RNA, Molecular interactions, pathways and networks, Functional, regulatory and non-coding RNA</td>
      <td>Chimeric Read Annotator for RNA-RNA interactome data</td>
      <td>ChiRA is a tool suite to analyze RNA-RNA interactome experimental data such as CLASH, CLEAR-CLIP, PARIS, SPLASH, etc.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/pavanvidem/chira</td>
      <td>RNA, Transcriptomics, Sequence Analysis</td>
      <td>chira</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/chira</td>
    </tr>
    <tr>
      <td></td>
      <td>chopin2</td>
      <td>1.0.7</td>
      <td></td>
      <td>chopin2</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Domain-Agnostic Supervised Learning with Hyperdimensional Computing</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/cumbof/chopin2</td>
      <td>Machine Learning</td>
      <td>chopin2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/chopin2</td>
    </tr>
    <tr>
      <td></td>
      <td>chromeister</td>
      <td>1.5.a</td>
      <td>1.5.a</td>
      <td>chromeister</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ultra-fast pairwise genome comparisons</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/estebanpw/chromeister</td>
      <td>Sequence Analysis</td>
      <td>chromeister</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/chromeister</td>
    </tr>
    <tr>
      <td></td>
      <td>circexplorer2</td>
      <td>2.3.8</td>
      <td>2.3.8</td>
      <td>circexplorer2</td>
      <td>Up-to-date</td>
      <td>circexplorer2</td>
      <td>CIRCexplorer2</td>
      <td></td>
      <td>RNA splicing, Gene transcripts, Literature and language</td>
      <td>Comprehensive and integrative circular RNA analysis toolset.</td>
      <td>Genome-wide annotation of circRNAs and their alternative back-splicing/splicing.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/YangLab/CIRCexplorer2</td>
      <td>RNA, Assembly</td>
      <td>circexplorer2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/circexplorer2</td>
    </tr>
    <tr>
      <td></td>
      <td>circos</td>
      <td>0.69.8</td>
      <td>0.69.9</td>
      <td>circos</td>
      <td>To update</td>
      <td>galactic_circos</td>
      <td>Galactic Circos</td>
      <td>Sequence visualisation</td>
      <td></td>
      <td>Build Circos Plots in Galaxy</td>
      <td>Galactic Circos is a Galaxy wrapper providing a GUI for the Circos tool. Circos allows visualizing data in a circular format.</td>
      <td></td>
      <td>To update</td>
      <td>http://circos.ca/</td>
      <td>Graphics</td>
      <td>circos</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/circos</td>
    </tr>
    <tr>
      <td></td>
      <td>cite_seq_count</td>
      <td>1.4.4</td>
      <td>1.4.4</td>
      <td>cite-seq-count</td>
      <td>Up-to-date</td>
      <td>CITE-seq-Count</td>
      <td>CITE-seq-Count</td>
      <td>RNA-Seq quantification</td>
      <td>Transcriptomics, Immunoproteins and antigens</td>
      <td>Count CMO/HTO</td>
      <td>Tool for counting antibody TAGS from a CITE-seq and/or cell hashing experiment.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/Hoohm/CITE-seq-Count</td>
      <td>Transcriptomics</td>
      <td>cite_seq_count</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cite_seq_count</td>
    </tr>
    <tr>
      <td></td>
      <td>clair3</td>
      <td>0.1.12</td>
      <td>1.0.5</td>
      <td>clair3</td>
      <td>To update</td>
      <td>clair3</td>
      <td>Clair3</td>
      <td>Variant calling</td>
      <td>Molecular genetics</td>
      <td>Symphonizing pileup and full-alignment for high-performance long-read variant calling</td>
      <td>Clair3 is a germline small variant caller for long-reads. Clair3 makes the best of two major method categories: pileup calling handles most variant candidates with speed, and full-alignment tackles complicated candidates to maximize precision and recall. Clair3 runs fast and has superior performance, especially at lower coverage. Clair3 is simple and modular for easy deployment and integration.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/HKU-BAL/Clair3</td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>clair3</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/clair3</td>
    </tr>
    <tr>
      <td></td>
      <td>clustalw</td>
      <td>2.1</td>
      <td>2.1</td>
      <td>clustalw</td>
      <td>Up-to-date</td>
      <td>\n                clustal2\n</td>
      <td></td>
      <td></td>
      <td></td>
      <td>ClustalW multiple sequence alignment program for DNA or proteins</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://www.clustal.org/clustal2/</td>
      <td>Phylogenetics, Sequence Analysis</td>
      <td>clustalw</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/clustalw</td>
    </tr>
    <tr>
      <td></td>
      <td>cnv-phenopacket</td>
      <td>1.0.2</td>
      <td></td>
      <td>cnv-phenopacket</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>cnv-phenopacket Converts TSV metadata file to JSON.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://pypi.org/project/cnv-phenopacket/</td>
      <td>Variant Analysis</td>
      <td>cnv_phenopacket</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cnv-phenopacket</td>
    </tr>
    <tr>
      <td></td>
      <td>cnv-vcf2json</td>
      <td>1.0.4</td>
      <td></td>
      <td>cnv-vcf2json</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>cnv-vcf2json Converts structural variants VCF file to JSON.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://pypi.org/project/cnv-phenopacket/</td>
      <td>Variant Analysis</td>
      <td>cnv-vcf2json</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cnv-vcf2json</td>
    </tr>
    <tr>
      <td></td>
      <td>cnvkit</td>
      <td>0.9.10</td>
      <td>0.9.10</td>
      <td>cnvkit</td>
      <td>Up-to-date</td>
      <td>cnvkit</td>
      <td>CNVkit</td>
      <td>Variant calling</td>
      <td>DNA structural variation</td>
      <td>detecting copy number variants and alterations genome-wide from high-throughput sequencing</td>
      <td>CNVkit is a software toolkit to infer and visualize copy number from targeted DNA sequencing data.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/etal/cnvkit</td>
      <td>Variant Analysis</td>
      <td>cnvkit</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cnvkit</td>
    </tr>
    <tr>
      <td></td>
      <td>codeml</td>
      <td>4.9</td>
      <td>4.10.7</td>
      <td>paml</td>
      <td>To update</td>
      <td>paml</td>
      <td>PAML</td>
      <td>Probabilistic sequence generation, Phylogenetic tree generation (maximum likelihood and Bayesian methods), Phylogenetic tree analysis</td>
      <td>Phylogenetics, Sequence analysis</td>
      <td>Detects positive selection</td>
      <td>Package of programs for phylogenetic analyses of DNA or protein sequences using maximum likelihood.</td>
      <td></td>
      <td>To update</td>
      <td>http://abacus.gene.ucl.ac.uk/software/paml.html</td>
      <td>Phylogenetics</td>
      <td>codeml</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/codeml</td>
    </tr>
    <tr>
      <td></td>
      <td>cojac</td>
      <td>0.9.1</td>
      <td>0.9.1</td>
      <td>cojac</td>
      <td>Up-to-date</td>
      <td>cojac</td>
      <td>COJAC</td>
      <td></td>
      <td>Genetic variation</td>
      <td>co-occurrence of mutations on amplicons</td>
      <td>CoOccurrence adJusted Analysis and Calling - The cojac package comprises a set of command-line tools to analyse co-occurrence of mutations on amplicons. It is useful, for example, for early detection of viral variants of concern (e.g. Alpha, Delta, Omicron) in environmental samples, and has been designed to scan for multiple SARS-CoV-2 variants in wastewater samples, as analyzed jointly by ETH Zurich, EPFL and Eawag.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/cbg-ethz/cojac</td>
      <td>Metagenomics, Sequence Analysis</td>
      <td>cojac</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cojac</td>
    </tr>
    <tr>
      <td></td>
      <td>colibread</td>
      <td>24.7.14+galaxy0</td>
      <td>24.7.14</td>
      <td>commet</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Colib'read tools are all dedicated to the analysis of NGS datasets without the need of any reference genome</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://colibread.inria.fr/</td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>colibread</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/colibread</td>
    </tr>
    <tr>
      <td></td>
      <td>collection_column_join</td>
      <td>0.0.3</td>
      <td>8.25</td>
      <td>coreutils</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Column Join on Collections</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>collection_column_join</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/collection_column_join</td>
    </tr>
    <tr>
      <td></td>
      <td>collection_element_identifiers</td>
      <td>0.0.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Extract element identifiers of a collection</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>collection_element_identifiers</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/collection_element_identifiers</td>
    </tr>
    <tr>
      <td></td>
      <td>column_maker</td>
      <td>2.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute an expression on every row</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>column_maker</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/column_maker</td>
    </tr>
    <tr>
      <td></td>
      <td>column_order_header_sort</td>
      <td>0.0.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Sort Column Order by heading</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>column_order_header_sort</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/column_order_header_sort</td>
    </tr>
    <tr>
      <td></td>
      <td>column_remove_by_header</td>
      <td>1.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Remove columns by header</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>column_remove_by_header</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/column_remove_by_header</td>
    </tr>
    <tr>
      <td></td>
      <td>compleasm</td>
      <td>0.2.5</td>
      <td>0.2.5</td>
      <td>compleasm</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compleasm: a faster and more accurate reimplementation of BUSCO</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/huangnengCSU/compleasm</td>
      <td>Sequence Analysis</td>
      <td>compleasm</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/compleasm/</td>
    </tr>
    <tr>
      <td></td>
      <td>compose_text_param</td>
      <td>0.1.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compose a text parameter value using text, integer and float values</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>compose_text_param</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/compose_text_param</td>
    </tr>
    <tr>
      <td></td>
      <td>compress_file</td>
      <td>0.1.0</td>
      <td></td>
      <td>gzip</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compress files.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>compress_file</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/compress_file</td>
    </tr>
    <tr>
      <td></td>
      <td>concoct</td>
      <td>1.1.0</td>
      <td>1.1.0</td>
      <td>concoct</td>
      <td>Up-to-date</td>
      <td>concoct</td>
      <td>CONCOCT</td>
      <td>Sequence clustering, Read binning</td>
      <td>Metagenomics</td>
      <td>CONCOCT (Clustering cONtigs with COverage and ComposiTion) does unsupervised binning of metagenomic contigs byusing nucleotide composition - kmer frequencies - and coverage data for multiple samples. CONCOCT can accurately(up to species level) bin metagenomic contigs.</td>
      <td>A program for unsupervised binning of metagenomic contigs by using nucleotide composition, coverage data in multiple samples and linkage data from paired end reads.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/BinPro/CONCOCT</td>
      <td>Metagenomics</td>
      <td>concoct</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/concoct</td>
    </tr>
    <tr>
      <td></td>
      <td>coverage_report</td>
      <td>0.0.4</td>
      <td>1.76</td>
      <td>perl-number-format</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Generate Detailed Coverage Report from BAM file</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc</td>
      <td>Sequence Analysis</td>
      <td>coverage_report</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/coverage_report</td>
    </tr>
    <tr>
      <td></td>
      <td>coverm</td>
      <td>0.7.0</td>
      <td>0.7.0</td>
      <td>coverm</td>
      <td>Up-to-date</td>
      <td>coverm</td>
      <td>CoverM</td>
      <td>Local alignment</td>
      <td>Bioinformatics</td>
      <td>CoverM genome and contig wrappers</td>
      <td>Read coverage calculator for metagenomics</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/wwood/CoverM</td>
      <td>Sequence Analysis</td>
      <td>coverm</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tools/coverm</td>
    </tr>
    <tr>
      <td></td>
      <td>crispr_studio</td>
      <td>1+galaxy0</td>
      <td>1</td>
      <td>crispr_studio</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>CRISPR Studio is a program developed to facilitate and accelerate CRISPR array visualization.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/moineaulab/CRISPRStudio</td>
      <td>Sequence Analysis</td>
      <td>crispr_studio</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/crispr_studio/</td>
    </tr>
    <tr>
      <td></td>
      <td>crosscontamination_barcode_filter</td>
      <td>0.3</td>
      <td>2.2.1</td>
      <td>r-ggplot2</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Barcode contamination discovery tool</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, Visualization</td>
      <td>crosscontamination_barcode_filter</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/crosscontamination_barcode_filter</td>
    </tr>
    <tr>
      <td></td>
      <td>crossmap</td>
      <td>0.6.1</td>
      <td>0.7.0</td>
      <td>crossmap</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>CrossMap converts genome coordinates or annotation files between genome assemblies</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://crossmap.sourceforge.net/</td>
      <td>Convert Formats, Genomic Interval Operations</td>
      <td>crossmap</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/crossmap</td>
    </tr>
    <tr>
      <td></td>
      <td>cutadapt</td>
      <td>4.6</td>
      <td>4.6</td>
      <td>cutadapt</td>
      <td>Up-to-date</td>
      <td>cutadapt</td>
      <td>Cutadapt</td>
      <td>Sequence trimming</td>
      <td>Genomics, Probes and primers, Sequencing</td>
      <td>Flexible tool to remove adapter sequences (and quality trim) high throughput sequencing reads (fasta/fastq).</td>
      <td>Find and remove adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://cutadapt.readthedocs.org/en/stable/</td>
      <td>Fasta Manipulation, Fastq Manipulation, Sequence Analysis</td>
      <td>cutadapt</td>
      <td>lparsons</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cutadapt</td>
    </tr>
    <tr>
      <td></td>
      <td>cutesv</td>
      <td>1.0.8</td>
      <td>2.1.0</td>
      <td>cutesv</td>
      <td>To update</td>
      <td>cuteSV</td>
      <td>cuteSV</td>
      <td>Split read mapping, Genotyping, Structural variation detection</td>
      <td>DNA structural variation, Sequencing, Computer science</td>
      <td>Long-read sequencing enables the comprehensive discovery of structural variations (SVs). However, it is still non-trivial to achieve high sensitivity and performance simultaneously due to the complex SV characteristics implied by noisy long reads. Therefore, we propose cuteSV, a sensitive, fast and scalable long-read-based SV detection approach. cuteSV uses tailored methods to collect the signatures of various types of SVs and employs a clustering-and-refinement method to analyze the signatures to implement sensitive SV detection. Benchmarks on real Pacific Biosciences (PacBio) and Oxford Nanopore Technology (ONT) datasets demonstrate that cuteSV has better yields and scalability than state-of-the-art tools.</td>
      <td>Long Read based Human Genomic Structural Variation Detection with cuteSV | Long-read sequencing technologies enable to comprehensively discover structural variations (SVs). However, it is still non-trivial for state-of-the-art approaches to detect SVs with high sensitivity or high performance or both. Herein, we propose cuteSV, a sensitive, fast and lightweight SV detection approach. cuteSV uses tailored methods to comprehensively collect various types of SV signatures, and a clustering-and-refinement method to implement a stepwise SV detection, which enables to achieve high sensitivity without loss of accuracy. Benchmark results demonstrate that cuteSV has better yields on real datasets. Further, its speed and scalability are outstanding and promising to large-scale data analysis</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/tjiangHIT/cuteSV</td>
      <td>Variant Analysis</td>
      <td>cutesv</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cutesv</td>
    </tr>
    <tr>
      <td></td>
      <td>cwpair2</td>
      <td>1.1.1</td>
      <td></td>
      <td>matplotlib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Contains a tool that takes a list of called peaks on both strands and produces a list of matched pairsand a list of unmatched orphans.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>ChIP-seq</td>
      <td>cwpair2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cwpair2</td>
    </tr>
    <tr>
      <td></td>
      <td>dada2</td>
      <td></td>
      <td>1.30.0</td>
      <td>bioconductor-dada2</td>
      <td>To update</td>
      <td>dada2</td>
      <td>dada2</td>
      <td>Variant calling, DNA barcoding</td>
      <td>Sequencing, Genetic variation, Microbial ecology, Metagenomics</td>
      <td>DADA2 wrappers</td>
      <td>This package infers exact sequence variants (SVs) from amplicon data, replacing the commonly used and coarser OTU clustering approach. This pipeline inputs demultiplexed fastq files, and outputs the sequence variants and their sample-wise abundances after removing substitution and chimera errors. Taxonomic classification is available via a native implementation of the RDP naive Bayesian classifier.</td>
      <td></td>
      <td>To update</td>
      <td>https://benjjneb.github.io/dada2/index.html</td>
      <td>Metagenomics</td>
      <td>dada2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/dada2</td>
    </tr>
    <tr>
      <td></td>
      <td>das_tool</td>
      <td>1.1.7</td>
      <td>1.1.7</td>
      <td>das_tool</td>
      <td>Up-to-date</td>
      <td>dastool</td>
      <td>dastool</td>
      <td>Read binning</td>
      <td>Metagenomics</td>
      <td>DAS Tool for genome resolved metagenomics</td>
      <td>DAS Tool is an automated method that integrates the results of a flexible number of binning algorithms to calculate an optimized, non-redundant set of bins from a single assembly.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/cmks/DAS_Tool</td>
      <td>Metagenomics</td>
      <td>das_tool</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/das_tool</td>
    </tr>
    <tr>
      <td></td>
      <td>data_source_iris_tcga</td>
      <td>1.0.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>IRIS-TCGA Data source tool</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Data Source</td>
      <td>data_source_iris_tcga</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/data_source_iris_tcga</td>
    </tr>
    <tr>
      <td></td>
      <td>datamash</td>
      <td>1.8</td>
      <td>1.1.0</td>
      <td>datamash</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>GNU Datamash is a command-line program which performs basicnumeric,textual and statistical operations on input textual data files.It is designed to be portable and reliable, and aid researchersto easily automate analysis pipelines, without writing code or even short scripts.License: GPL Version 3 (or later).These tool wrappers were originally writen by Assaf Gordon.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.gnu.org/software/datamash/</td>
      <td>Text Manipulation</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/datamash</td>
    </tr>
    <tr>
      <td></td>
      <td>decontaminator</td>
      <td>1.0.0</td>
      <td></td>
      <td>numpy</td>
      <td>To update</td>
      <td>decontaminator</td>
      <td></td>
      <td></td>
      <td></td>
      <td>Deep Learning method for novel virus detection in sequencing data</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/cbib/decontaminator</td>
      <td>Machine Learning</td>
      <td>decontaminator</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/decontaminator</td>
    </tr>
    <tr>
      <td></td>
      <td>deepmicro</td>
      <td>1.4</td>
      <td>1.4</td>
      <td>deepmicro</td>
      <td>Up-to-date</td>
      <td>\n                DeepMicro\n</td>
      <td></td>
      <td></td>
      <td></td>
      <td>Representation learning and classification framework</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/paulzierep/DeepMicro</td>
      <td>Machine Learning</td>
      <td>deepmicro</td>
      <td>iuc</td>
      <td>https://github.com/paulzierep/DeepMicro</td>
    </tr>
    <tr>
      <td></td>
      <td>deepsig</td>
      <td>1.2.5</td>
      <td>1.2.5</td>
      <td>deepsig</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Predictor of signal peptides in proteins based on deep learning</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/BolognaBiocomp/deepsig</td>
      <td>Genome annotation</td>
      <td>deepsig</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/deepsig</td>
    </tr>
    <tr>
      <td></td>
      <td>deepvariant</td>
      <td>1.5.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>DeepVariant is a deep learning-based variant caller</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/google/deepvariant</td>
      <td>Variant Analysis</td>
      <td>deepvariant</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/deepvariant</td>
    </tr>
    <tr>
      <td></td>
      <td>deg_annotate</td>
      <td>1.1.0</td>
      <td>2.31.1</td>
      <td>bedtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Annotate DESeq2/DEXSeq output tables</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics</td>
      <td>deg_annotate</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/deg_annotate</td>
    </tr>
    <tr>
      <td></td>
      <td>delly</td>
      <td>0.9.1</td>
      <td>1.2.6</td>
      <td>delly</td>
      <td>To update</td>
      <td>delly2</td>
      <td>Delly2</td>
      <td>Indel detection, Structural variation detection, Variant calling, Genotyping, Genetic variation analysis</td>
      <td>DNA structural variation, Sequencing, Pathology, Genomics, Genetic variation, Bioinformatics, Population genomics, Rare diseases</td>
      <td>Delly is an integrated structural variant (SV) prediction method that can discover, genotype and visualize deletions, tandem duplications, inversions and translocations at single-nucleotide resolution in short-read massively parallel sequencing data. It uses paired-ends, split-reads and read-depth to sensitively and accurately delineate genomic rearrangements throughout the genome.</td>
      <td>Integrated structural variant prediction method that can discover, genotype and visualize deletions, tandem duplications, inversions and translocations at single-nucleotide resolution in short-read massively parallel sequencing data. It uses paired-ends and split-reads to sensitively and accurately delineate genomic rearrangements throughout the genome. Structural variants can be visualized using Delly-maze and Delly-suave.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/dellytools/delly</td>
      <td>Variant Analysis</td>
      <td>delly</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/delly</td>
    </tr>
    <tr>
      <td></td>
      <td>deseq2</td>
      <td>2.11.40.8</td>
      <td>1.42.0</td>
      <td>bioconductor-deseq2</td>
      <td>To update</td>
      <td>DESeq2</td>
      <td>DESeq2</td>
      <td>Differential gene expression analysis</td>
      <td>Transcriptomics</td>
      <td>Differential gene expression analysis based on the negative binomial distribution</td>
      <td>R/Bioconductor package for differential gene expression analysis based on the negative binomial distribution. Estimate variance-mean dependence in count data from high-throughput sequencing assays and test for differential expression based on a model using the negative binomial distribution.</td>
      <td></td>
      <td>To update</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/DESeq2.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>deseq2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/deseq2</td>
    </tr>
    <tr>
      <td></td>
      <td>dexseq</td>
      <td>1.44</td>
      <td>1.48.0</td>
      <td>bioconductor-dexseq</td>
      <td>To update</td>
      <td>dexseq</td>
      <td>DEXSeq</td>
      <td>Enrichment analysis, Exonic splicing enhancer prediction</td>
      <td>RNA-Seq</td>
      <td>Inference of differential exon usage in RNA-Seq</td>
      <td>The package is focused on finding differential exon usage using RNA-seq exon counts between samples with different experimental designs. It provides functions that allows the user to make the necessary statistical tests based on a model that uses the negative binomial distribution to estimate the variance between biological replicates and generalized linear models for testing. The package also provides functions for the visualization and exploration of the results.</td>
      <td></td>
      <td>To update</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/DEXSeq.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>dexseq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/dexseq</td>
    </tr>
    <tr>
      <td></td>
      <td>diamond</td>
      <td>2.0.15</td>
      <td>2.1.9</td>
      <td>diamond</td>
      <td>To update</td>
      <td>diamond</td>
      <td>Diamond</td>
      <td>Sequence alignment analysis</td>
      <td>Sequence analysis, Proteins</td>
      <td>DIAMOND is a new alignment tool for aligning short DNA sequencing reads to a protein reference database such as NCBI-NR.</td>
      <td>Sequence aligner for protein and translated DNA searches and functions as a drop-in replacement for the NCBI BLAST software tools. It is suitable for protein-protein search as well as DNA-protein search on short reads and longer sequences including contigs and assemblies, providing a speedup of BLAST ranging up to x20,000.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bbuchfink/diamond</td>
      <td>Sequence Analysis</td>
      <td>diamond</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/diamond</td>
    </tr>
    <tr>
      <td></td>
      <td>diffbind</td>
      <td>2.10.0</td>
      <td>3.12.0</td>
      <td>bioconductor-diffbind</td>
      <td>To update</td>
      <td>diffbind</td>
      <td>DiffBind</td>
      <td>Differential binding analysis</td>
      <td>ChIP-seq</td>
      <td>Diffbind provides functions for  processing  ChIP-Seq  data.</td>
      <td>Compute differentially bound sites from multiple ChIP-seq experiments using affinity (quantitative) data. Also enables occupancy (overlap) analysis and plotting functions.</td>
      <td></td>
      <td>To update</td>
      <td>http://bioconductor.org/packages/release/bioc/html/DiffBind.html</td>
      <td>ChIP-seq</td>
      <td>diffbind</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/diffbind</td>
    </tr>
    <tr>
      <td></td>
      <td>dimet</td>
      <td>0.2.1</td>
      <td>0.2.2</td>
      <td>dimet</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>DIMet is a bioinformatics pipeline for differential analysis of isotopic targeted labeling data.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/cbib/DIMet</td>
      <td>Metabolomics</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/DIMet</td>
    </tr>
    <tr>
      <td></td>
      <td>disco</td>
      <td></td>
      <td>1.2</td>
      <td>disco</td>
      <td>To update</td>
      <td>disco</td>
      <td>DISCO</td>
      <td>Protein sequence analysis</td>
      <td>Structure determination</td>
      <td>DISCO is a overlap-layout-consensus (OLC) metagenome assembler</td>
      <td>DISCO is software to perform structure determination of protein homo-oligomers with cyclic symmetry.DISCO computes oligomeric protein structures using geometric constraints derived from RDCs and intermolecular distance restraints such as NOEs or disulfide bonds. When a reliable subunit structure can be calculated from intramolecular restraints, DISCO guarantees that all satisfying oligomer structures will be discovered, yet can run in minutes to hours on only a single desktop-class computer.</td>
      <td></td>
      <td>To update</td>
      <td>http://disco.omicsbio.org/</td>
      <td>Metagenomics, Assembly</td>
      <td>disco</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/disco/</td>
    </tr>
    <tr>
      <td></td>
      <td>dnabot</td>
      <td>3.1.0</td>
      <td></td>
      <td>dnabot</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>DNA assembly using BASIC on OpenTrons</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/BASIC-DNA-ASSEMBLY/DNA-BOT</td>
      <td>Synthetic Biology</td>
      <td>dnabot</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/dnabot</td>
    </tr>
    <tr>
      <td></td>
      <td>dnaweaver</td>
      <td>1.0.2</td>
      <td>1.0.2</td>
      <td>dnaweaver_synbiocad</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Given a SBOL input, calculate assembly parts for Gibson or Golden Gate.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/Edinburgh-Genome-Foundry/DnaWeaver</td>
      <td>Synthetic Biology</td>
      <td>dnaweaver</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/dnaweaver</td>
    </tr>
    <tr>
      <td></td>
      <td>dram</td>
      <td>1.3.5</td>
      <td>1.4.6</td>
      <td>dram</td>
      <td>To update</td>
      <td>dram</td>
      <td>DRAM</td>
      <td>Gene functional annotation</td>
      <td>Metagenomics, Biological databases, Molecular genetics</td>
      <td>DRAM for distilling microbial metabolism to automate the curation of microbiome function</td>
      <td>Distilled and Refined Annotation of Metabolism: A tool for the annotation and curation of function for microbial and viral genomes</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/WrightonLabCSU/DRAM</td>
      <td>Metagenomics</td>
      <td>dram</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/dram</td>
    </tr>
    <tr>
      <td></td>
      <td>drep</td>
      <td>3.4.5</td>
      <td>3.5.0</td>
      <td>drep</td>
      <td>To update</td>
      <td>drep</td>
      <td>dRep</td>
      <td>Genome comparison</td>
      <td>Metagenomics, Genomics, Sequence analysis</td>
      <td>dRep compares and dereplicates genome sets</td>
      <td>Fast and accurate genomic comparisons that enables improved genome recovery from metagenomes through de-replication.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/MrOlm/drep</td>
      <td>Metagenomics</td>
      <td>drep</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/drep</td>
    </tr>
    <tr>
      <td></td>
      <td>dropletutils</td>
      <td>1.10.0</td>
      <td>1.22.0</td>
      <td>bioconductor-dropletutils</td>
      <td>To update</td>
      <td>dropletutils</td>
      <td>DropletUtils</td>
      <td>Loading, Community profiling</td>
      <td>Gene expression, RNA-seq, Sequencing, Transcriptomics</td>
      <td>DropletUtils - Utilities for handling droplet-based single-cell RNA-seq data</td>
      <td>Provides a number of utility functions for handling single-cell (RNA-seq) data from droplet technologies such as 10X Genomics. This includes data loading, identification of cells from empty droplets, removal of barcode-swapped pseudo-cells, and downsampling of the count matrix.</td>
      <td></td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/devel/bioc/html/DropletUtils.html</td>
      <td>Transcriptomics, Sequence Analysis</td>
      <td>dropletutils</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/dropletutils/</td>
    </tr>
    <tr>
      <td></td>
      <td>ebi_tools</td>
      <td>0.1.0</td>
      <td></td>
      <td>six</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tools to query and download data from several EMBL-EBI databases</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.ebi.ac.uk/services/all</td>
      <td>Web Services, Data Source</td>
      <td>ebi_tools</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ebi_tools</td>
    </tr>
    <tr>
      <td></td>
      <td>edger</td>
      <td>3.36.0</td>
      <td>4.0.2</td>
      <td>bioconductor-edger</td>
      <td>To update</td>
      <td>edger</td>
      <td>edgeR</td>
      <td>Differential gene expression analysis</td>
      <td>Genetics, RNA-Seq, ChIP-seq</td>
      <td>Perform RNA-Seq differential expression analysis using edgeR pipeline</td>
      <td>Differential expression analysis of RNA-seq expression profiles with biological replication. Implements a range of statistical methodology based on the negative binomial distributions, including empirical Bayes estimation, exact tests, generalized linear models and quasi-likelihood tests. As well as RNA-seq, it be applied to differential signal analysis of other types of genomic data that produce counts, including ChIP-seq, SAGE and CAGE.</td>
      <td></td>
      <td>To update</td>
      <td>http://bioconductor.org/packages/release/bioc/html/edgeR.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>edger</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/edger</td>
    </tr>
    <tr>
      <td></td>
      <td>egsea</td>
      <td>1.20.0</td>
      <td>1.28.0</td>
      <td>bioconductor-egsea</td>
      <td>To update</td>
      <td>egsea</td>
      <td>EGSEA</td>
      <td>Gene set testing</td>
      <td>Systems biology</td>
      <td>This tool implements the Ensemble of Gene Set Enrichment Analyses (EGSEA) method for gene set testing</td>
      <td>This package implements the Ensemble of Gene Set Enrichment Analyses method for gene set testing.</td>
      <td></td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/EGSEA.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>egsea</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/egsea</td>
    </tr>
    <tr>
      <td></td>
      <td>emboss_5</td>
      <td>5.0.0</td>
      <td>6.6.0</td>
      <td>emboss</td>
      <td>To update</td>
      <td>emboss</td>
      <td>EMBOSS</td>
      <td>Sequence analysis, Local alignment, Sequence alignment analysis, Global alignment, Sequence alignment</td>
      <td>Molecular biology, Sequence analysis, Biology</td>
      <td>Galaxy wrappers for EMBOSS version 5.0.0 tools</td>
      <td>Diverse suite of tools for sequence analysis; many programs analagous to GCG; context-sensitive help for each tool.</td>
      <td></td>
      <td>To update</td>
      <td>http://emboss.open-bio.org/</td>
      <td>Sequence Analysis, Fasta Manipulation</td>
      <td>emboss_5</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/emboss_5</td>
    </tr>
    <tr>
      <td></td>
      <td>ena_upload</td>
      <td>0.6.3</td>
      <td>0.7.1</td>
      <td>ena-upload-cli</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Submits experimental data and respective metadata to the European Nucleotide Archive (ENA).</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/usegalaxy-eu/ena-upload-cli</td>
      <td>Data Export</td>
      <td>ena_upload</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ena_upload</td>
    </tr>
    <tr>
      <td></td>
      <td>enasearch</td>
      <td></td>
      <td>0.2.2</td>
      <td>enasearch</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A Python library for interacting with ENA's API</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bebatut/enasearch</td>
      <td>Data Source</td>
      <td>enasearch</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/enasearch</td>
    </tr>
    <tr>
      <td></td>
      <td>ensembl_vep</td>
      <td>110.1</td>
      <td>111.0</td>
      <td>ensembl-vep</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Ensembl VEP: Annotate VCFs with variant effect predictions</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/Ensembl/ensembl-vep</td>
      <td>Variant Analysis</td>
      <td>ensembl_vep</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ensembl_vep</td>
    </tr>
    <tr>
      <td></td>
      <td>episcanpy</td>
      <td>0.3.2</td>
      <td>0.4.0</td>
      <td>episcanpy</td>
      <td>To update</td>
      <td>episcanpy</td>
      <td>epiScanpy</td>
      <td>Enrichment analysis, Imputation</td>
      <td>Epigenomics, Cell biology, DNA</td>
      <td>EpiScanpy  Epigenomics single cell analysis in python</td>
      <td>Epigenomics Single Cell Analysis in Python.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/colomemaria/epiScanpy</td>
      <td>Epigenetics</td>
      <td>episcanpy</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/episcanpy/</td>
    </tr>
    <tr>
      <td></td>
      <td>exomedepth</td>
      <td>1.1.0</td>
      <td>1.1.16</td>
      <td>r-exomedepth</td>
      <td>To update</td>
      <td>exomedepth</td>
      <td>ExomeDepth</td>
      <td>Sequence analysis, Variant calling, Genotyping, Copy number estimation</td>
      <td>Exome sequencing, Gene transcripts, Mapping, Sequencing, Genetic variation, Rare diseases</td>
      <td>ExomeDepth: Calls copy number variants (CNVs) from targeted sequence data</td>
      <td>Copy number variant (CNV) calling algorithm designed to control technical variability between samples. It calls CNVs from targeted sequence data, typically exome sequencing experiments designed to identify the genetic basis of Mendelian disorders.</td>
      <td></td>
      <td>To update</td>
      <td>https://cran.r-project.org/package=ExomeDepth</td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>exomedepth</td>
      <td>crs4</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/exomedepth</td>
    </tr>
    <tr>
      <td></td>
      <td>exonerate</td>
      <td>2.4.0</td>
      <td>2.4.0</td>
      <td>exonerate</td>
      <td>Up-to-date</td>
      <td>exonerate</td>
      <td>Exonerate</td>
      <td>Pairwise sequence alignment, Protein threading, Genome alignment</td>
      <td>Sequence analysis, Sequence sites, features and motifs, Molecular interactions, pathways and networks</td>
      <td>Exonerate is a generic tool for pairwise sequence comparison.</td>
      <td>A tool for pairwise sequence alignment. It enables alignment for DNA-DNA and DNA-protein pairs and also gapped and ungapped alignment.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://www.ebi.ac.uk/about/vertebrate-genomics/software/exonerate</td>
      <td>Sequence Analysis</td>
      <td>exonerate</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/exonerate</td>
    </tr>
    <tr>
      <td></td>
      <td>export2graphlan</td>
      <td>0.20</td>
      <td>0.22</td>
      <td>export2graphlan</td>
      <td>To update</td>
      <td>export2graphlan</td>
      <td>export2graphlan</td>
      <td>Conversion</td>
      <td>Taxonomy, Metabolomics, Biomarkers</td>
      <td>export2graphlan is a conversion software tool for producing both annotation and tree file for GraPhlAn</td>
      <td>export2graphlan is a conversion software tool for producing both annotation and tree file for GraPhlAn. In particular, the annotation file tries to highlight specific sub-trees deriving automatically from input file what nodes are important.</td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/CibioCM/export2graphlan/overview</td>
      <td>Metagenomics</td>
      <td>export2graphlan</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/export2graphlan/</td>
    </tr>
    <tr>
      <td></td>
      <td>extract_genomic_dna</td>
      <td>3.0.3+galaxy2</td>
      <td>0.11.0</td>
      <td>bx-python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Contains a tool that extracts genomic DNA using coordinates from ASSEMBLED genomes and UNassembled genomes.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Genomic Interval Operations</td>
      <td>extract_genomic_dna</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/extract_genomic_dna</td>
    </tr>
    <tr>
      <td></td>
      <td>fargene</td>
      <td>0.1</td>
      <td>0.1</td>
      <td>fargene</td>
      <td>Up-to-date</td>
      <td>fargene</td>
      <td>fARGene</td>
      <td>Antimicrobial resistance prediction</td>
      <td>Metagenomics, Microbiology, Public health and epidemiology</td>
      <td>fARGene (Fragmented Antibiotic Resistance Gene iENntifiEr )</td>
      <td>fARGene (Fragmented Antibiotic Resistance Gene iENntifiEr ) is a tool that takes either fragmented metagenomic data or longer sequences as input and predicts and delivers full-length antiobiotic resistance genes as output.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/fannyhb/fargene</td>
      <td>Sequence Analysis</td>
      <td>fargene</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fargene</td>
    </tr>
    <tr>
      <td></td>
      <td>fasta_nucleotide_color_plot</td>
      <td>1.0.1</td>
      <td></td>
      <td>openjdk</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Contains a tool that produces a graphical representation of FASTA data with each nucleotide represented by a selected color.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/seqcode/cegr-tools/tree/master/src/org/seqcode/cegrtools/fourcolorplot</td>
      <td>Visualization</td>
      <td>fasta_nucleotide_color_plot</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fasta_nucleotide_color_plot</td>
    </tr>
    <tr>
      <td></td>
      <td>fasta_stats</td>
      <td>2.0</td>
      <td></td>
      <td>numpy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Display summary statistics for a fasta file.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fasta_stats/</td>
      <td>Sequence Analysis</td>
      <td>fasta_stats</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fasta_stats/</td>
    </tr>
    <tr>
      <td></td>
      <td>fastani</td>
      <td>1.3</td>
      <td>1.34</td>
      <td>fastani</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Fast alignment-free computation of whole-genome Average Nucleotide Identity</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ParBLiSS/FastANI</td>
      <td>Sequence Analysis</td>
      <td>fastani</td>
      <td>iuc</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>fastp</td>
      <td></td>
      <td>0.23.4</td>
      <td>fastp</td>
      <td>To update</td>
      <td>\n                fastp\n</td>
      <td></td>
      <td></td>
      <td></td>
      <td>Fast all-in-one preprocessing for FASTQ files</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/OpenGene/fastp</td>
      <td>Sequence Analysis</td>
      <td>fastp</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fastp</td>
    </tr>
    <tr>
      <td></td>
      <td>fastqc</td>
      <td>0.74+galaxy0</td>
      <td>0.12.1</td>
      <td>fastqc</td>
      <td>To update</td>
      <td>fastqc</td>
      <td>FastQC</td>
      <td>Sequence composition calculation, Sequencing quality control, Statistical calculation</td>
      <td>Sequencing, Data quality management, Sequence analysis</td>
      <td>Read QC reports using FastQC</td>
      <td>This tool aims to provide a QC report which can spot problems or biases which originate either in the sequencer or in the starting library material. It can be run in one of two modes. It can either run as a stand alone interactive application for the immediate analysis of small numbers of FastQ files, or it can be run in a non-interactive mode where it would be suitable for integrating into a larger analysis pipeline for the systematic processing of large numbers of files.</td>
      <td></td>
      <td>To update</td>
      <td>http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</td>
      <td>Fastq Manipulation</td>
      <td>fastqc</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fastqc</td>
    </tr>
    <tr>
      <td></td>
      <td>fastqe</td>
      <td>0.3.1+galaxy0</td>
      <td>0.3.1</td>
      <td>fastqe</td>
      <td>To update</td>
      <td>fastqe</td>
      <td>FASTQE</td>
      <td>Sequencing quality control</td>
      <td>Sequence analysis, Sequencing</td>
      <td>FASTQE</td>
      <td>Compute quality stats for FASTQ files and print those stats as emoji... for some reason.</td>
      <td></td>
      <td>To update</td>
      <td>https://fastqe.com/</td>
      <td>Sequence Analysis</td>
      <td>fastqe</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fastqe</td>
    </tr>
    <tr>
      <td></td>
      <td>fasttree</td>
      <td>2.1.10</td>
      <td>2.1.11</td>
      <td>fasttree</td>
      <td>To update</td>
      <td>fasttree</td>
      <td>FastTree</td>
      <td>Phylogenetic tree generation (from molecular sequences), Phylogenetic tree generation (maximum likelihood and Bayesian methods)</td>
      <td>Phylogenetics, Sequence analysis</td>
      <td>FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide or protein sequences - GVL</td>
      <td>Infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide or protein sequences.</td>
      <td></td>
      <td>To update</td>
      <td>http://www.microbesonline.org/fasttree/</td>
      <td>Phylogenetics</td>
      <td>fasttree</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fasttree</td>
    </tr>
    <tr>
      <td></td>
      <td>featurecounts</td>
      <td>2.0.3</td>
      <td>2.0.6</td>
      <td>subread</td>
      <td>To update</td>
      <td>featurecounts</td>
      <td>FeatureCounts</td>
      <td>Read summarisation</td>
      <td>Sequencing</td>
      <td>featureCounts counts the number of reads aligned to defined masked regions in a reference genome</td>
      <td>featureCounts is a very efficient read quantifier. It can be used to summarize RNA-seq reads and gDNA-seq reads to a variety of genomic features such as genes, exons, promoters, gene bodies and genomic bins. It is included in the Bioconductor Rsubread package and also in the SourceForge Subread package.</td>
      <td></td>
      <td>To update</td>
      <td>http://bioinf.wehi.edu.au/featureCounts</td>
      <td>RNA, Transcriptomics, Sequence Analysis</td>
      <td>featurecounts</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/featurecounts</td>
    </tr>
    <tr>
      <td></td>
      <td>feelnc</td>
      <td>0.2.1</td>
      <td>0.2</td>
      <td>feelnc</td>
      <td>To update</td>
      <td>feelnc</td>
      <td>FEELnc</td>
      <td>Annotation, Classification</td>
      <td>RNA-seq, Functional, regulatory and non-coding RNA</td>
      <td>Galaxy wrapper for FEELnc</td>
      <td>A tool to annotate long non-coding RNAs from RNA-seq assembled transcripts.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/tderrien/FEELnc</td>
      <td>Sequence Analysis</td>
      <td>feelnc</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/feelnc</td>
    </tr>
    <tr>
      <td></td>
      <td>fermikit</td>
      <td>r193</td>
      <td>r193</td>
      <td>fermi2</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FermiKit is a de novo assembly based variant calling pipeline for deep Illumina resequencing data.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/lh3/fermikit</td>
      <td>Assembly, Variant Analysis</td>
      <td>fermikit</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fermikit</td>
    </tr>
    <tr>
      <td></td>
      <td>fgsea</td>
      <td>1.8.0+galaxy1</td>
      <td>1.28.0</td>
      <td>bioconductor-fgsea</td>
      <td>To update</td>
      <td>fgsea</td>
      <td>fgsea</td>
      <td>Gene-set enrichment analysis</td>
      <td>Genetics</td>
      <td>Perform gene set testing using fgsea</td>
      <td>The package implements an algorithm for fast gene set enrichment analysis. Using the fast algorithm allows to make more permutations and get more fine grained p-values, which allows to use accurate stantard approaches to multiple hypothesis correction.</td>
      <td></td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/fgsea.html</td>
      <td>Visualization, Transcriptomics, Statistics</td>
      <td>fgsea</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fgsea</td>
    </tr>
    <tr>
      <td></td>
      <td>filtlong</td>
      <td>0.2.1</td>
      <td>0.2.1</td>
      <td>filtlong</td>
      <td>Up-to-date</td>
      <td>\n                filtlong\n</td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filtlong - Filtering long reads by quality</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/rrwick/Filtlong</td>
      <td>Fastq Manipulation, Sequence Analysis</td>
      <td>filtlong</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/filtlong</td>
    </tr>
    <tr>
      <td></td>
      <td>flair</td>
      <td>1.5</td>
      <td>2.0.0</td>
      <td>flair</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FLAIR (Full-Length Alternative Isoform analysis of RNA) for the correction, isoform definition, and alternative splicing analysis of noisy reads.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/BrooksLabUCSC/flair</td>
      <td>Nanopore</td>
      <td>flair</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/flair</td>
    </tr>
    <tr>
      <td></td>
      <td>flash</td>
      <td>1.2.11</td>
      <td>1.2.11</td>
      <td>flash</td>
      <td>Up-to-date</td>
      <td>flash</td>
      <td>FLASH</td>
      <td>Read pre-processing, Sequence merging, Sequence assembly</td>
      <td>Sequencing, Sequence assembly</td>
      <td>Fast Length Adjustment of SHort reads</td>
      <td>Identifies paired-end reads which overlap in the middle, converting them to single long reads</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://ccb.jhu.edu/software/FLASH/</td>
      <td>Assembly, Fastq Manipulation</td>
      <td>flash</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/flash</td>
    </tr>
    <tr>
      <td></td>
      <td>fraggenescan</td>
      <td></td>
      <td>1.31</td>
      <td>fraggenescan</td>
      <td>To update</td>
      <td>fraggenescan</td>
      <td>FragGeneScan</td>
      <td>Gene prediction</td>
      <td>Genetics, Sequence analysis</td>
      <td>Tool for finding (fragmented) genes in short read</td>
      <td>Application for finding (fragmented) genes in short reads</td>
      <td></td>
      <td>To update</td>
      <td>https://sourceforge.net/projects/fraggenescan/</td>
      <td>Sequence Analysis</td>
      <td>fraggenescan</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fraggenescan/</td>
    </tr>
    <tr>
      <td></td>
      <td>freebayes</td>
      <td>1.3.6</td>
      <td>1.3.7</td>
      <td>freebayes</td>
      <td>To update</td>
      <td>freebayes</td>
      <td>FreeBayes</td>
      <td>Variant calling, Statistical calculation</td>
      <td>Genomics, Genetic variation, Rare diseases</td>
      <td>Galaxy Freebayes Bayesian genetic variant detector tool</td>
      <td>Bayesian genetic variant detector designed to find small polymorphisms, specifically SNPs, indels, multi-nucleotide polymorphisms, and complex events (composite insertion and substitution events) smaller than the length of a short-read sequencing alignment.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/freebayes</td>
      <td>Variant Analysis</td>
      <td>freebayes</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/freebayes</td>
    </tr>
    <tr>
      <td></td>
      <td>freec</td>
      <td>11.6</td>
      <td></td>
      <td>gawk</td>
      <td>To update</td>
      <td>freec</td>
      <td>FREEC</td>
      <td>Copy number estimation, Variant calling, Genome alignment</td>
      <td>DNA structural variation, Oncology, Human genetics, Data mining</td>
      <td>Control-FREEC is a tool for detection of copy-number changes and allelic imbalances (including LOH) using deep-sequencing data originally developed by the Bioinformatics Laboratory of Institut Curie (Paris). It automatically computes, normalizes, segments copy number and beta allele frequency (BAF) profiles, then calls copy number alterations and LOH.</td>
      <td>A tool for control-free copy number alteration (CNA) and allelic imbalances (LOH) detection using deep-sequencing data, particularly useful for cancer studies.</td>
      <td></td>
      <td>To update</td>
      <td>http://boevalab.inf.ethz.ch/FREEC/</td>
      <td>Variant Analysis</td>
      <td>control_freec</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/freec</td>
    </tr>
    <tr>
      <td></td>
      <td>freyja</td>
      <td>1.4.4</td>
      <td>1.4.9</td>
      <td>freyja</td>
      <td>To update</td>
      <td>freyja</td>
      <td></td>
      <td></td>
      <td></td>
      <td>lineage abundances estimation</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/andersen-lab/Freyja</td>
      <td>Metagenomics, Sequence Analysis</td>
      <td>freyja</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/freyja</td>
    </tr>
    <tr>
      <td></td>
      <td>fsd</td>
      <td>1.0.2</td>
      <td></td>
      <td>matplotlib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tool that plots a histogram of sizes of read families</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Graphics</td>
      <td>duplex_family_size_distribution</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fsd</td>
    </tr>
    <tr>
      <td></td>
      <td>funannotate</td>
      <td>1.8.15</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>\n                funannotate\n</td>
      <td></td>
      <td></td>
      <td></td>
      <td>Funannotate is a genome prediction, annotation, and comparison software package.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://funannotate.readthedocs.io</td>
      <td>Genome annotation</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/funannotate</td>
    </tr>
    <tr>
      <td></td>
      <td>gatk4</td>
      <td>4.1.7.0</td>
      <td>4.5.0.0</td>
      <td>gatk4</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A Galaxy wrapper for Mutect2 from GATK</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://software.broadinstitute.org/gatk/gatk4</td>
      <td>Variant Analysis</td>
      <td>gatk4_mutect2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gatk4</td>
    </tr>
    <tr>
      <td></td>
      <td>gdcwebapp</td>
      <td>1.0.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>GDCWebApp automatically filter, extract, and convert genomic data from the Genomic Data Commons portal to BED format</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://bioinf.iasi.cnr.it/gdcwebapp/</td>
      <td>Data Source, Convert Formats</td>
      <td>gdcwebapp</td>
      <td>iuc</td>
      <td>https://github.com/fabio-cumbo/GDCWebApp4Galaxy</td>
    </tr>
    <tr>
      <td></td>
      <td>gecko</td>
      <td>1.2</td>
      <td>1.2</td>
      <td>gecko</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Ungapped genome comparison</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/otorreno/gecko</td>
      <td>Sequence Analysis</td>
      <td>gecko</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gecko</td>
    </tr>
    <tr>
      <td></td>
      <td>gemini</td>
      <td>0.20.1</td>
      <td>0.30.2</td>
      <td>gemini</td>
      <td>To update</td>
      <td>gemini</td>
      <td>GEMINI</td>
      <td>Sequence analysis, Genetic variation analysis</td>
      <td>Sequence analysis</td>
      <td>GEMINI: a flexible framework for exploring genome variation</td>
      <td>GEMINI (GEnome MINIng) is a flexible framework for exploring genetic variation in the context of the wealth of genome annotations available for the human genome. By placing genetic variants, sample phenotypes and genotypes, as well as genome annotations into an integrated database framework, GEMINI provides a simple, flexible, and powerful system for exploring genetic variation for disease and population genetics.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/arq5x/gemini</td>
      <td>Sequence Analysis, Next Gen Mappers</td>
      <td>gemini</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gemini</td>
    </tr>
    <tr>
      <td></td>
      <td>genebed_maf_to_fasta</td>
      <td>1.0.1+galaxy0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Stitch gene blocks given a set of coding exon intervals</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/genebed_maf_to_fasta/</td>
      <td>Genomic Interval Operations</td>
      <td>genebed_maf_to_fasta</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/genebed_maf_to_fasta/</td>
    </tr>
    <tr>
      <td></td>
      <td>genehunter_modscore</td>
      <td>3.0.0</td>
      <td>3.1</td>
      <td>ghm</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Maximised LOD score pedigree analysis utility</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.helmholtz-muenchen.de/en/ige/service/software-download/genehunter-modscore/index.html</td>
      <td>Variant Analysis</td>
      <td>genehunter_modscore</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/genehunter_modscore/</td>
    </tr>
    <tr>
      <td></td>
      <td>geneiobio</td>
      <td>4.7.1+galaxy1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Gene.iobio is an interactive tool for variant and trio analysis.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/iobio/gene.iobio</td>
      <td>Sequence Analysis</td>
      <td>geneiobio</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/geneiobio</td>
    </tr>
    <tr>
      <td></td>
      <td>genetrack</td>
      <td></td>
      <td></td>
      <td>numpy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Contains a tool that separately identifies peaks on the forward "+ (W) and reverse - (C) strand.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>ChIP-seq</td>
      <td>genetrack</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/genetrack</td>
    </tr>
    <tr>
      <td></td>
      <td>genomescope</td>
      <td>2.0</td>
      <td>2.0</td>
      <td>genomescope2</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Analyze unassembled short reads</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/tbenavi1/genomescope2.0</td>
      <td>Statistics</td>
      <td>genomescope</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/genomescope</td>
    </tr>
    <tr>
      <td></td>
      <td>genomic_super_signature</td>
      <td>1.2.0</td>
      <td>1.10.0</td>
      <td>bioconductor-genomicsupersignature</td>
      <td>To update</td>
      <td>genomicsupersignature</td>
      <td>GenomicSuperSignature</td>
      <td>Gene-set enrichment analysis, Essential dynamics, Deposition, Principal component visualisation, Dimensionality reduction</td>
      <td>RNA-Seq, Transcriptomics, Microbial ecology, Genotype and phenotype, Microarray experiment</td>
      <td>Interpretation of RNAseq experiments through robust, efficient comparison to public databases</td>
      <td>GenomicSuperSignature is a package for the interpretation of RNA-seq experiments through robust, efficient comparison to public databases.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/shbrief/GenomicSuperSignature</td>
      <td>Sequence Analysis, RNA, Transcriptomics</td>
      <td>genomic_super_signature</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/genomic_super_signature</td>
    </tr>
    <tr>
      <td></td>
      <td>genrich</td>
      <td>0.5+galaxy2</td>
      <td>0.6.1</td>
      <td>genrich</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Genrich is a peak-caller for genomic enrichment assays (e.g. ChIP-seq, ATAC-seq).</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/jsh58/Genrich</td>
      <td>ChIP-seq</td>
      <td>genrich</td>
      <td>iuc</td>
      <td>https://github.com/jsh58/Genrich</td>
    </tr>
    <tr>
      <td></td>
      <td>get_hrun</td>
      <td>0.5.9.2</td>
      <td>0.8.1.1</td>
      <td>pyfaidx</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Annotate indel variants with homopolymer context</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/get_hrun</td>
      <td>Variant Analysis</td>
      <td>get_hrun</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/get_hrun</td>
    </tr>
    <tr>
      <td></td>
      <td>getorganelle</td>
      <td>1.7.7.0</td>
      <td>1.7.7.0</td>
      <td>getorganelle</td>
      <td>Up-to-date</td>
      <td>getorganelle</td>
      <td>GetOrganelle</td>
      <td>De-novo assembly, Genome assembly, Mapping assembly, Mapping, Sequence trimming</td>
      <td>Cell biology, Sequence assembly, Whole genome sequencing, Plant biology, Model organisms</td>
      <td>GetOrganelle - This toolkit assembles organelle genomes from genomic skimming data.</td>
      <td>A fast and versatile toolkit for accurate de novo assembly of organelle genomes.This toolkit assemblies organelle genome from genomic skimming data.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/Kinggerm/GetOrganelle</td>
      <td>Assembly</td>
      <td>getorganelle</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/getorganelle</td>
    </tr>
    <tr>
      <td></td>
      <td>gfa_to_fa</td>
      <td>0.1.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>gfa_to_fa - Converting GFA format to Fasta format</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://gfa-spec.github.io/GFA-spec/</td>
      <td>Convert Formats</td>
      <td>gfa_to_fa</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gfa_to_fa</td>
    </tr>
    <tr>
      <td></td>
      <td>gff3_rebase</td>
      <td>1.2</td>
      <td>0.6.6</td>
      <td>bcbiogff</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Rebase a GFF against a parent GFF (e.g. an original genome)</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gff3_rebase</td>
      <td>Sequence Analysis</td>
      <td>gff3_rebase</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gff3_rebase</td>
    </tr>
    <tr>
      <td></td>
      <td>gffcompare</td>
      <td>0.12.6</td>
      <td>0.12.6</td>
      <td>gffcompare</td>
      <td>Up-to-date</td>
      <td>gffcompare</td>
      <td>gffcompare</td>
      <td>Sequence annotation</td>
      <td>Nucleic acids, Sequence analysis</td>
      <td>Galaxy wrappers for Geo Pertea's GffCompare package.</td>
      <td>Program for comparing, annotating, merging and tracking transcripts in GFF files.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/gpertea/gffcompare/</td>
      <td>Transcriptomics</td>
      <td>gffcompare</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gffcompare</td>
    </tr>
    <tr>
      <td></td>
      <td>gffread</td>
      <td>0.12.7</td>
      <td>0.12.7</td>
      <td>gffread</td>
      <td>Up-to-date</td>
      <td>gffread</td>
      <td>gffread</td>
      <td>Sequence annotation</td>
      <td>Nucleic acids, Sequence analysis</td>
      <td>gffread filters and/or converts GFF3/GTF2 records</td>
      <td>program for filtering, converting and manipulating GFF files</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://ccb.jhu.edu/software/stringtie/gff.shtml#gffread/</td>
      <td>Sequence Analysis</td>
      <td>gffread</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gffread</td>
    </tr>
    <tr>
      <td></td>
      <td>ggplot2</td>
      <td>3.4.0</td>
      <td></td>
      <td>r-base</td>
      <td>To update</td>
      <td>ggplot2</td>
      <td>ggplot2</td>
      <td>Visualisation</td>
      <td>Data visualisation</td>
      <td>ggplot2 is a system for declaratively creating graphics, based on The Grammar of Graphics.You provide the data, tell ggplot2 how to map variables to aesthetics, what graphical primitives to use,and it takes care of the details.</td>
      <td>Plotting system for R, based on the grammar of graphics.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/tidyverse/ggplot2</td>
      <td>Visualization</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ggplot2</td>
    </tr>
    <tr>
      <td></td>
      <td>ggupset</td>
      <td>1.0</td>
      <td></td>
      <td>r-ggupset</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Create Upset Plots with ggupset</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/const-ae/ggupset</td>
      <td>Graphics</td>
      <td>ggupset</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ggupset</td>
    </tr>
    <tr>
      <td></td>
      <td>glimmer</td>
      <td></td>
      <td>3.02</td>
      <td>glimmer</td>
      <td>To update</td>
      <td>gemini</td>
      <td>GEMINI</td>
      <td>Sequence analysis, Genetic variation analysis</td>
      <td>Sequence analysis</td>
      <td>Glimmer makes gene predictions.</td>
      <td>GEMINI (GEnome MINIng) is a flexible framework for exploring genetic variation in the context of the wealth of genome annotations available for the human genome. By placing genetic variants, sample phenotypes and genotypes, as well as genome annotations into an integrated database framework, GEMINI provides a simple, flexible, and powerful system for exploring genetic variation for disease and population genetics.</td>
      <td></td>
      <td>To update</td>
      <td>https://ccb.jhu.edu/software/glimmer/</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/glimmer</td>
    </tr>
    <tr>
      <td></td>
      <td>goenrichment</td>
      <td>2.0.1</td>
      <td>2.0.1</td>
      <td>goenrichment</td>
      <td>Up-to-date</td>
      <td>goenrichment</td>
      <td>GOEnrichment</td>
      <td>Gene-set enrichment analysis</td>
      <td>Transcriptomics</td>
      <td>Performs GO Enrichment analysis.</td>
      <td>GOEnrichment is a tool for performing GO enrichment analysis of gene sets, such as those obtained from RNA-seq or Microarray experiments, to help characterize them at the functional level. It is available in Galaxy Europe and as a stand-alone tool.GOEnrichment is flexible in that it allows the user to use any version of the Gene Ontology and any GO annotation file they desire. To enable the use of GO slims, it is accompanied by a sister tool GOSlimmer, which can convert annotation files from full GO to any specified GO slim.The tool features an optional graph clustering algorithm to reduce the redundancy in the set of enriched GO terms and simplify its output.It was developed by the BioData.pt / ELIXIR-PT team at the Instituto Gulbenkian de Cincia.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/DanFaria/GOEnrichment</td>
      <td>Genome annotation</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/goenrichment</td>
    </tr>
    <tr>
      <td></td>
      <td>goseq</td>
      <td>1.50.0</td>
      <td>1.54.0</td>
      <td>bioconductor-goseq</td>
      <td>To update</td>
      <td>goseq</td>
      <td>GOseq</td>
      <td>Gene functional annotation</td>
      <td>RNA-Seq</td>
      <td>goseq does selection-unbiased testing for category enrichment amongst differentially expressed (DE) genes for RNA-seq data</td>
      <td>Detect Gene Ontology and/or other user defined categories which are over/under represented in RNA-seq data.</td>
      <td></td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/goseq.html</td>
      <td>Statistics, RNA, Micro-array Analysis</td>
      <td>goseq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/goseq</td>
    </tr>
    <tr>
      <td></td>
      <td>gprofiler</td>
      <td>@TOOL_VERSION@+galaxy11</td>
      <td></td>
      <td>r-gprofiler2</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>functional enrichment analysis of gene lists, convertion between various types of namespaces, translation gene identifiers between organisms and more</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://biit.cs.ut.ee/gprofiler</td>
      <td>Statistics, Web Services</td>
      <td>gprofiler</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gprofiler/</td>
    </tr>
    <tr>
      <td></td>
      <td>graphembed</td>
      <td>2.4</td>
      <td>2.4</td>
      <td>graph_embed</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute a 2D embedding of a data matrix given supervised class information</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/fabriziocosta/GraphEmbed</td>
      <td>Statistics, Graphics</td>
      <td>graphembed</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/graphembed/</td>
    </tr>
    <tr>
      <td></td>
      <td>graphlan</td>
      <td></td>
      <td>1.1.3</td>
      <td>graphlan</td>
      <td>To update</td>
      <td>graphlan</td>
      <td>GraPhlAn</td>
      <td>Phylogenetic inference, Phylogenetic tree visualisation, Phylogenetic tree editing, Taxonomic classification</td>
      <td>Metagenomics, Phylogenetics, Phylogenomics, Cladistics</td>
      <td>GraPhlAn is a software tool for producing high-quality circular representations of taxonomic and phylogenetic trees</td>
      <td>GraPhlAn is a software tool for producing high-quality circular representations of taxonomic and phylogenetic trees. GraPhlAn focuses on concise, integrative, informative, and publication-ready representations of phylogenetically- and taxonomically-driven investigation.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/biobakery/graphlan</td>
      <td>Metagenomics, Graphics, Phylogenetics</td>
      <td>graphlan</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/humann2/</td>
    </tr>
    <tr>
      <td></td>
      <td>gtdbtk</td>
      <td>2.3.2</td>
      <td>2.3.2</td>
      <td>gtdbtk</td>
      <td>Up-to-date</td>
      <td>GTDB-Tk</td>
      <td>GTDB-Tk</td>
      <td>Genome alignment, Taxonomic classification, Sequence assembly, Query and retrieval</td>
      <td>Metagenomics, Taxonomy, Phylogenetics, Database management, Proteins</td>
      <td>GTDB-Tk is a software tool kit for assigning objective taxonomic classifications to bacterial and archaeal genomesbased on the Genome Database Taxonomy GTDB. It is designed to work with recent advances that allow hundreds orthousands of metagenome-assembled genomes (MAGs) to be obtained directly from environmental samples. It can alsobe applied to isolate and single-cell genomes.</td>
      <td>a toolkit to classify genomes with the Genome Taxonomy Database.GTDB-Tk: a toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes.GTDB-Tk is a software toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes based on the Genome Database Taxonomy GTDB. It is designed to work with recent advances that allow hundreds or thousands of metagenome-assembled genomes (MAGs) to be obtained directly from environmental samples. It can also be applied to isolate and single-cell genomes. The GTDB-Tk is open source and released under the GNU General Public License (Version 3).</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/Ecogenomics/GTDBTk</td>
      <td>Metagenomics</td>
      <td>gtdbtk</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gtdbtk</td>
    </tr>
    <tr>
      <td></td>
      <td>gtfToBed12</td>
      <td>357</td>
      <td>447</td>
      <td>ucsc-gtftogenepred</td>
      <td>To update</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td>UCSC Genome Browser Utilities</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>Convert GTF files to BED12 format</td>
      <td>Utilities for handling sequences and assemblies from the UCSC Genome Browser project.</td>
      <td></td>
      <td>To update</td>
      <td>http://genome-source.cse.ucsc.edu/gitweb/?p=kent.git;a=blob;f=src/userApps/README</td>
      <td>Convert Formats</td>
      <td>gtftobed12</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gtfToBed12</td>
    </tr>
    <tr>
      <td></td>
      <td>gubbins</td>
      <td>3.2.1</td>
      <td>3.3.3</td>
      <td>gubbins</td>
      <td>To update</td>
      <td>gubbins</td>
      <td>Gubbins</td>
      <td>Genotyping, Phylogenetic inference, Ancestral reconstruction</td>
      <td>Phylogeny, Genotype and phenotype, Whole genome sequencing</td>
      <td>Gubbins - bacterial recombination detection</td>
      <td>Gubbins is a tool for rapid phylogenetic analysis of large samples of recombinant bacterial whole genome sequences.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>gubbins</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gubbins</td>
    </tr>
    <tr>
      <td></td>
      <td>gvcftools</td>
      <td>0.1</td>
      <td>0.17.0</td>
      <td>gvcftools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/sequencing/gvcftools</td>
      <td>Variant Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gvcftools</td>
    </tr>
    <tr>
      <td></td>
      <td>gwastools</td>
      <td>0.1.0</td>
      <td>1.48.0</td>
      <td>bioconductor-gwastools</td>
      <td>To update</td>
      <td>gwastools</td>
      <td>GWASTools</td>
      <td>Deposition, Analysis, Annotation</td>
      <td>GWAS study</td>
      <td></td>
      <td>Classes for storing very large GWAS data sets and annotation, and functions for GWAS data cleaning and analysis.</td>
      <td></td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/GWASTools.html</td>
      <td>Visualization, Variant Analysis</td>
      <td></td>
      <td>iuc</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>hamronization</td>
      <td>1.0.3</td>
      <td>1.1.4</td>
      <td>hamronization</td>
      <td>To update</td>
      <td>hamronization</td>
      <td>hAMRonization</td>
      <td>Data handling, Antimicrobial resistance prediction, Parsing</td>
      <td>Public health and epidemiology, Microbiology, Bioinformatics</td>
      <td>Convert AMR gene detection tool output to hAMRonization specification format.</td>
      <td>Parse multiple Antimicrobial Resistance Analysis Reports into a common data structure</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/pha4ge/hAMRonization</td>
      <td>Sequence Analysis</td>
      <td>hamronization</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hamronization</td>
    </tr>
    <tr>
      <td></td>
      <td>hansel</td>
      <td>2.6.1</td>
      <td>2.6.1</td>
      <td>bio_hansel</td>
      <td>Up-to-date</td>
      <td>Biohansel</td>
      <td>BioHansel</td>
      <td>Genotyping, SNP detection, Genome assembly</td>
      <td>Whole genome sequencing, DNA polymorphism, Genotype and phenotype, Infectious disease, Agricultural science</td>
      <td>Heidelberg and Enteritidis SNP Elucidation</td>
      <td>BioHansel is a tool for performing high-resolution genotyping of bacterial isolates by identifying phylogenetically informative single nucleotide polymorphisms (SNPs), also known as canonical SNPs, in whole genome sequencing (WGS) data. The application uses a fast k-mer matching algorithm to map pathogen WGS data to canonical SNPs contained in hierarchically structured schemas and assigns genotypes based on the detected SNP profile.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/phac-nml/bio_hansel</td>
      <td>Sequence Analysis</td>
      <td>bio_hansel</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hansel</td>
    </tr>
    <tr>
      <td></td>
      <td>hapcut2</td>
      <td>1.3.3</td>
      <td>1.3.3</td>
      <td>hapcut2</td>
      <td>Up-to-date</td>
      <td>hapcut2</td>
      <td>HapCUT2</td>
      <td>Haplotype mapping, Variant classification</td>
      <td></td>
      <td>Robust and accurate haplotype assembly for diverse sequencing technologies</td>
      <td>HapCUT2 is a maximum-likelihood-based tool for assembling haplotypes from DNA sequence reads, designed to "just work" with excellent speed and accuracy across a range of long- and short-read sequencing technologies.The output is in Haplotype block format described here: https://github.com/vibansal/HapCUT2/blob/master/outputformat.md</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/vibansal/HapCUT2</td>
      <td>Assembly</td>
      <td>hapcut2</td>
      <td>galaxy-australia</td>
      <td>https://github.com/galaxyproject/tools-iuc</td>
    </tr>
    <tr>
      <td></td>
      <td>hapog</td>
      <td>1.3.7</td>
      <td>1.3.7</td>
      <td>hapog</td>
      <td>Up-to-date</td>
      <td>hapog</td>
      <td>Hapo-G</td>
      <td>Genome assembly, Optimisation and refinement</td>
      <td>Sequence assembly, Genomics</td>
      <td>Hapo-G - Haplotype-Aware Polishing of Genomes</td>
      <td>Hapo-G is a tool that aims to improve the quality of genome assemblies by polishing the consensus with accurate reads. It capable of incorporating phasing information from high-quality reads (short or long-reads) to polish genome assemblies and in particular assemblies of diploid and heterozygous genomes.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/institut-de-genomique/HAPO-G</td>
      <td>Assembly</td>
      <td>hapog</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hapog</td>
    </tr>
    <tr>
      <td></td>
      <td>happy</td>
      <td>0.3.14</td>
      <td>0.3.15</td>
      <td>hap.py</td>
      <td>To update</td>
      <td>hap.py</td>
      <td>hap.py</td>
      <td>Variant calling, Sequence analysis, Genotyping</td>
      <td>Genomics, DNA polymorphism</td>
      <td>A tool to perform comparisons only based on chromosome, position, and allele identity for comparison of somatic callsets.</td>
      <td>This is a set of programs based on htslib to benchmark variant calls against gold standard truth datasets.To compare a VCF against a gold standard dataset, use the following commmand line to perform genotype-level haplotype comparison.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/Illumina/hap.py</td>
      <td>Variant Analysis</td>
      <td>happy</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/happy</td>
    </tr>
    <tr>
      <td></td>
      <td>heatmap2</td>
      <td>3.1.3.1</td>
      <td>2.17.0</td>
      <td>r-gplots</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>heatmap.2 function from the R gplots package</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/cran/gplots</td>
      <td>Visualization</td>
      <td>ggplot2_heatmap2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/heatmap2</td>
    </tr>
    <tr>
      <td></td>
      <td>heinz</td>
      <td>1.0</td>
      <td>1.62.0</td>
      <td>bioconductor-bionet</td>
      <td>To update</td>
      <td>bionet</td>
      <td>BioNet</td>
      <td>Protein interaction analysis</td>
      <td>Molecular interactions, pathways and networks, Protein interactions</td>
      <td>An algorithm for identification of the optimal scoring subnetwork.</td>
      <td>This package provides functions for the integrated analysis of protein-protein interaction networks and the detection of functional modules. Different datasets can be integrated into the network by assigning p-values of statistical tests to the nodes of the network. By fitting a beta-uniform mixture model and calculating scores from these p-values, overall scores of network regions can be calculated and an integer linear programming algorithm identifies the maximum scoring subnetwork.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ls-cwi/heinz</td>
      <td>Transcriptomics, Visualization, Statistics</td>
      <td>heinz</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/heinz</td>
    </tr>
    <tr>
      <td></td>
      <td>hicexplorer</td>
      <td>3.7.2</td>
      <td>3.7.3</td>
      <td>hicexplorer</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>HiCExplorer: Set of programs to process, analyze and visualize Hi-C data.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/deeptools/HiCExplorer</td>
      <td>Sequence Analysis, Visualization</td>
      <td>hicexplorer</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hicexplorer</td>
    </tr>
    <tr>
      <td></td>
      <td>hicstuff</td>
      <td>3.1.5</td>
      <td>3.2.2</td>
      <td>hicstuff</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/koszullab/hicstuff</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hicstuff</td>
    </tr>
    <tr>
      <td></td>
      <td>hifiasm_meta</td>
      <td>0.3.1</td>
      <td>hamtv0.3.1</td>
      <td>hifiasm_meta</td>
      <td>To update</td>
      <td>hifiasm-meta</td>
      <td>Hifiasm-meta</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly, Metagenomics</td>
      <td>A hifiasm fork for metagenome assembly using Hifi reads.</td>
      <td>Hifiasm_meta - de novo metagenome assembler, based on hifiasm, a haplotype-resolved de novo assembler for PacBio Hifi reads.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/xfengnefx/hifiasm-meta</td>
      <td>Metagenomics</td>
      <td>hifiasm_meta</td>
      <td>galaxy-australia</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hifiasm_meta</td>
    </tr>
    <tr>
      <td></td>
      <td>hisat2</td>
      <td>2.2.1</td>
      <td>2.2.1</td>
      <td>hisat2</td>
      <td>Up-to-date</td>
      <td>hisat2</td>
      <td>HISAT2</td>
      <td>Sequence alignment</td>
      <td>RNA-seq</td>
      <td>HISAT2 is a fast and sensitive spliced alignment program.</td>
      <td>Alignment program for mapping next-generation sequencing reads (both DNA and RNA) to a population of human genomes (as well as to a single reference genome).</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://ccb.jhu.edu/software/hisat2/</td>
      <td>Assembly</td>
      <td>hisat2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hisat2</td>
    </tr>
    <tr>
      <td></td>
      <td>hivclustering</td>
      <td>1.3.1</td>
      <td>1.6.8</td>
      <td>python-hivclustering</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Infers transmission networks from pairwise distances inferred by tn93</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://pypi.org/project/hivclustering/</td>
      <td>Next Gen Mappers</td>
      <td>hivclustering</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hivclustering/</td>
    </tr>
    <tr>
      <td></td>
      <td>hmmer3</td>
      <td>3.4</td>
      <td>3.4</td>
      <td>hmmer</td>
      <td>Up-to-date</td>
      <td>hmmer3</td>
      <td>HMMER3</td>
      <td>Formatting, Multiple sequence alignment, Sequence profile generation, Format validation, Conversion, Sequence generation, Data retrieval, Statistical calculation, Database search, Formatting, Database search, Database search, Probabilistic sequence generation, Statistical calculation, Statistical calculation, Sequence database search, Formatting, Sequence database search, Database search, Sequence database search</td>
      <td>Sequence analysis, Sequence sites, features and motifs, Gene and protein families</td>
      <td>HMMER is used for searching sequence databases for homologs of proteinsequences, and for making protein sequence alignments. It implementsmethods using probabilistic models called profile hidden Markov models(profile HMMs).</td>
      <td>This tool is used for searching sequence databases for homologs of protein sequences, and for making protein sequence alignments. It implements methods using probabilistic models called profile hidden Markov models. The new HMMER3 project, HMMER is now as fast as BLAST for protein search.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hmmer.org/</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hmmer3</td>
    </tr>
    <tr>
      <td></td>
      <td>homer</td>
      <td>4.11</td>
      <td>4.11</td>
      <td>homer</td>
      <td>Up-to-date</td>
      <td>homer</td>
      <td>homer</td>
      <td>Sequence motif discovery</td>
      <td></td>
      <td>HOMER (Hypergeometric Optimization of Motif EnRichment) is a suite of tools for Motif Discovery and next-gen sequencing analysis.</td>
      <td>HOMER contains a novel motif discovery algorithm that was designed for regulatory element analysis in genomics applications (DNA only, no protein).  It is a differential motif discovery algorithm, which means that it takes two sets of sequences and tries to identify the regulatory elements that are specifically enriched in on set relative to the other.  It uses ZOOPS scoring (zero or one occurrence per sequence) coupled with the hypergeometric enrichment calculations (or binomial) to determine motif enrichment.  HOMER also tries its best to account for sequenced bias in the dataset.  It was designed with ChIP-Seq and promoter analysis in mind, but can be applied to pretty much any nucleic acids motif finding problem.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://homer.ucsd.edu/homer/index.html</td>
      <td>Sequence Analysis</td>
      <td>data_manager_homer_preparse</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/homer</td>
    </tr>
    <tr>
      <td></td>
      <td>htseq_count</td>
      <td>2.0.5</td>
      <td>2.0.5</td>
      <td>htseq</td>
      <td>Up-to-date</td>
      <td>htseq</td>
      <td>HTSeq</td>
      <td>Nucleic acid sequence analysis</td>
      <td>Sequence analysis</td>
      <td>Count aligned reads (SAM/BAM) that overlap genomic features (GFF)</td>
      <td>Python framework to process and analyse high-throughput sequencing (HTS) data</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://readthedocs.org/projects/htseq/</td>
      <td>Genomic Interval Operations, SAM, Sequence Analysis, RNA</td>
      <td>htseq_count</td>
      <td>lparsons</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/htseq_count</td>
    </tr>
    <tr>
      <td></td>
      <td>humann</td>
      <td>3.8</td>
      <td>3.8</td>
      <td>humann</td>
      <td>Up-to-date</td>
      <td>humann</td>
      <td>humann</td>
      <td>Species frequency estimation, Taxonomic classification, Phylogenetic analysis</td>
      <td>Metagenomics, Phylogenomics</td>
      <td>HUMAnN for functionally profiling metagenomes and metatranscriptomes at species-level resolution</td>
      <td>HUMAnN is a pipeline for efficiently and accurately profiling the presence/absence and abundance of microbial pathways in a community from metagenomic or metatranscriptomic sequencing data (typically millions of short DNA/RNA reads). This process, referred to as functional profiling, aims to describe the metabolic potential of a microbial community and its members. More generally, functional profiling answers the question What are the microbes in my community-of-interest doing (or are capable of doing)?</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://huttenhower.sph.harvard.edu/humann</td>
      <td>Metagenomics</td>
      <td>humann</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/humann</td>
    </tr>
    <tr>
      <td></td>
      <td>hybpiper</td>
      <td>2.1.6</td>
      <td>2.1.6</td>
      <td>hybpiper</td>
      <td>Up-to-date</td>
      <td>HybPiper</td>
      <td>HybPiper</td>
      <td>Sequence trimming, Sequence assembly, Read mapping</td>
      <td>Phylogenetics, Plant biology, Gene transcripts, Sequence assembly, Phylogenomics</td>
      <td>Analyse targeted sequence capture data</td>
      <td>Paralogs and off-target sequences improve phylogenetic resolution in a densely-sampled study of the breadfruit genus (Artocarpus, Moraceae).Recovering genes from targeted sequence capture data.Current version: 1.3.1 (August 2018).-- Read our article in Applications in Plant Sciences (Open Access).HybPiper was designed for targeted sequence capture, in which DNA sequencing libraries are enriched for gene regions of interest, especially for phylogenetics. HybPiper is a suite of Python scripts that wrap and connect bioinformatics tools in order to extract target sequences from high-throughput DNA sequencing reads.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/mossmatters/HybPiper</td>
      <td>Sequence Analysis, Phylogenetics</td>
      <td>hybpiper</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hybpiper</td>
    </tr>
    <tr>
      <td></td>
      <td>hyphy</td>
      <td>2.5.47</td>
      <td>2.5.59</td>
      <td>hyphy</td>
      <td>To update</td>
      <td>HyPhy</td>
      <td>HyPhy</td>
      <td>Statistical calculation</td>
      <td>Phylogeny, Small molecules, Molecular interactions, pathways and networks</td>
      <td>Hypothesis Testing using Phylogenies</td>
      <td>Software package for the analysis of genetic sequences using techniques in phylogenetics, molecular evolution, and machine learning.</td>
      <td></td>
      <td>To update</td>
      <td>http://www.hyphy.org</td>
      <td>Phylogenetics</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hyphy/</td>
    </tr>
    <tr>
      <td></td>
      <td>hypo</td>
      <td>1.0.3</td>
      <td>1.0.3</td>
      <td>hypo</td>
      <td>Up-to-date</td>
      <td>HyPo</td>
      <td>HyPo</td>
      <td>Optimisation and refinement, Genome assembly</td>
      <td>Sequence assembly, Genomics</td>
      <td>Super Fast &amp; Accurate Polisher for Long Read Genome Assemblies</td>
      <td>HyPo, a Hybrid Polisher, utilizes short as well as long reads within a single run to polish a long reads assembly of small and large genomes.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/kensung-lab/hypo</td>
      <td>Assembly</td>
      <td>hypo</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hypo</td>
    </tr>
    <tr>
      <td></td>
      <td>icescreen</td>
      <td>1.3.1</td>
      <td>1.3.2</td>
      <td>icescreen</td>
      <td>To update</td>
      <td>icescreen</td>
      <td>ICEscreen</td>
      <td>Database search, Protein feature detection</td>
      <td>Mobile genetic elements, Sequence sites, features and motifs, Genomics, Molecular interactions, pathways and networks, Structural variation</td>
      <td>ICEscreen identifies Integrative Conjugative Elements (ICEs) and Integrative Mobilizable Elements (IMEs) in Bacillota genomes.</td>
      <td>A tool to detect Firmicute ICEs and IMEs, isolated or enclosed in composite structures.</td>
      <td></td>
      <td>To update</td>
      <td>https://icescreen.migale.inrae.fr/</td>
      <td>Genome annotation</td>
      <td>icescreen</td>
      <td>iuc</td>
      <td>https://forgemia.inra.fr/ices_imes_analysis/icescreen</td>
    </tr>
    <tr>
      <td></td>
      <td>idba_ud</td>
      <td></td>
      <td>1.1.3</td>
      <td>idba</td>
      <td>To update</td>
      <td>idba</td>
      <td>IDBA</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly</td>
      <td>Wrappers for the idba assembler variants.</td>
      <td>A short read assembler based on iterative De Bruijn graph. It is developed under 64-bit Linux, but should be suitable for all unix-like system.</td>
      <td></td>
      <td>To update</td>
      <td>https://i.cs.hku.hk/~alse/hkubrg/projects/index.html</td>
      <td>Assembly</td>
      <td>idba</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/idba_ud</td>
    </tr>
    <tr>
      <td></td>
      <td>idr</td>
      <td>2.0.3</td>
      <td>2.0.4.2</td>
      <td>idr</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Galaxy wrappers for the IDR package from Nathan Boleu</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/nboley/idr</td>
      <td>Sequence Analysis</td>
      <td>idr</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/idr/</td>
    </tr>
    <tr>
      <td></td>
      <td>idr_download</td>
      <td>0.44.1</td>
      <td>5.11.1</td>
      <td>omero-py</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Image Data Resource downloading tool</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://idr.openmicroscopy.org</td>
      <td>Data Source</td>
      <td>idr_download_by_ids</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/idr_download</td>
    </tr>
    <tr>
      <td></td>
      <td>iedb_api</td>
      <td>2.15.2</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Get epitope binding predictions from IEDB-API</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://tools.immuneepitope.org/main/tools-api/</td>
      <td>Data Source, Sequence Analysis</td>
      <td>iedb_api</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/iedb_api</td>
    </tr>
    <tr>
      <td></td>
      <td>instrain</td>
      <td>1.5.3</td>
      <td>1.8.0</td>
      <td>instrain</td>
      <td>To update</td>
      <td>instrain</td>
      <td>InStrain</td>
      <td>SNP detection, Genome comparison</td>
      <td>Mapping, Metagenomics</td>
      <td>InStrain is a tool for analysis of co-occurring genome populations from metagenomes</td>
      <td>InStrain is a tool for analysis of co-occurring genome populations from metagenomes that allows highly accurate genome comparisons, analysis of coverage, microdiversity, and linkage, and sensitive SNP detection with gene localization and synonymous non-synonymous identification</td>
      <td></td>
      <td>To update</td>
      <td>https://instrain.readthedocs.io/en/latest/#</td>
      <td>Metagenomics</td>
      <td>instrain</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/instrain</td>
    </tr>
    <tr>
      <td></td>
      <td>integron_finder</td>
      <td>2.0.3</td>
      <td>2.0.3</td>
      <td>integron_finder</td>
      <td>Up-to-date</td>
      <td>integron_finder</td>
      <td>Integron Finder</td>
      <td>Nucleic acid feature detection, Sequence motif recognition, Protein feature detection, Genome annotation</td>
      <td>Functional genomics, Mobile genetic elements, Molecular biology, Sequence analysis</td>
      <td>"IntegronFinder identify integrons with high accuracy and sensitivity.It searches for attC sites using covariance models, for integron-integrases using HMM profiles, and for other features (promoters, attI site) using pattern matching"</td>
      <td>A tool to detect Integron in DNA sequences.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/gem-pasteur/Integron_Finder</td>
      <td>Sequence Analysis</td>
      <td>integronfinder</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/integron_finder</td>
    </tr>
    <tr>
      <td></td>
      <td>intermine_galaxy_exchange</td>
      <td>0.0.1</td>
      <td>8.25</td>
      <td>coreutils</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>InterMine Exporter</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Convert Formats</td>
      <td>intermine_galaxy_exchange</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/intermine_galaxy_exchange</td>
    </tr>
    <tr>
      <td></td>
      <td>interproscan</td>
      <td>5.59-91.0</td>
      <td></td>
      <td>interproscan</td>
      <td>To update</td>
      <td>interproscan_ebi</td>
      <td>InterProScan (EBI)</td>
      <td>Sequence motif recognition, Protein feature detection</td>
      <td>Gene and protein families, Sequence analysis</td>
      <td>Interproscan queries the interpro database and provides annotations.</td>
      <td>Scan sequences against the InterPro protein signature databases.</td>
      <td></td>
      <td>To update</td>
      <td>http://www.ebi.ac.uk/Tools/pfa/iprscan5/</td>
      <td>Sequence Analysis</td>
      <td>interproscan</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/interproscan</td>
    </tr>
    <tr>
      <td></td>
      <td>interval2maf</td>
      <td>1.0.1+galaxy1</td>
      <td>0.11.0</td>
      <td>bx-python</td>
      <td>To update</td>
      <td>bx-python</td>
      <td>bx-python</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>Extract MAF blocks given a set of intervals</td>
      <td>Tools for manipulating biological data, particularly multiple sequence alignments.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/interval2maf/</td>
      <td>Genomic Interval Operations</td>
      <td>interval2maf</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/interval2maf/</td>
    </tr>
    <tr>
      <td></td>
      <td>intervene</td>
      <td>0.6.5</td>
      <td>0.6.5</td>
      <td>intervene</td>
      <td>Up-to-date</td>
      <td>intervene</td>
      <td>Intervene</td>
      <td>Sequence comparison, Sequence visualisation</td>
      <td>Computational biology</td>
      <td>Create pairwise and upset plots</td>
      <td>Tool for intersection and visualization of multiple gene or genomic region sets. Intervene contains three modules: venn to generate Venn diagrams of up to six sets, upset to generate UpSet plots of multiple sets, and pairwise to compute and visualize intersections of multiple sets as clustered heat maps.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://intervene.readthedocs.io</td>
      <td>Statistics</td>
      <td>intervene</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/intervene</td>
    </tr>
    <tr>
      <td></td>
      <td>iqtree</td>
      <td>2.1.2</td>
      <td>2.2.6</td>
      <td>iqtree</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Efficient phylogenomic software by maximum likelihood</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.iqtree.org/</td>
      <td>Phylogenetics</td>
      <td>iqtree</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/iqtree/</td>
    </tr>
    <tr>
      <td></td>
      <td>irissv</td>
      <td>1.0.4</td>
      <td>1.19.2</td>
      <td>samtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Refine insertion sequences</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/mkirsche/Iris</td>
      <td>Variant Analysis</td>
      <td>irissv</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tools/irissv/</td>
    </tr>
    <tr>
      <td></td>
      <td>isescan</td>
      <td>1.7.2.3</td>
      <td>1.7.2.1</td>
      <td>isescan</td>
      <td>To update</td>
      <td>ISEScan</td>
      <td>ISEScan</td>
      <td>Structural variation detection</td>
      <td>Genomics, DNA structural variation, Sequence analysis, Genetic variation</td>
      <td>"ISEScan is a pipeline to identify IS (Insertion Sequence) elements in genome and metagenomebased on profile hidden Markov models constructed from manually curated IS elements."</td>
      <td>Automated identification of insertion sequence elements in prokaryotic genomes.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/xiezhq/ISEScan</td>
      <td>Sequence Analysis</td>
      <td>ISEScan</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/isescan</td>
    </tr>
    <tr>
      <td></td>
      <td>isoformswitchanalyzer</td>
      <td>1.20.0</td>
      <td>2.2.0</td>
      <td>bioconductor-isoformswitchanalyzer</td>
      <td>To update</td>
      <td>IsoformSwitchAnalyzeR</td>
      <td>IsoformSwitchAnalyzeR</td>
      <td>Sequence comparison, Sequence analysis</td>
      <td>Computational biology, Gene transcripts</td>
      <td>Statistical identification of isoform switching from RNA-seq derived quantification of novel and/or annotated full-length isoforms.</td>
      <td>Enables identification of isoform switches with predicted functional consequences from RNA-seq data. Consequences can be chosen from a long list but includes protein domains gain/loss changes in NMD sensitivity etc. It directly supports import of data from Cufflinks/Cuffdiff, Kallisto, Salmon and RSEM but other transcript qunatification tools are easy to import as well.</td>
      <td></td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/devel/bioc/html/IsoformSwitchAnalyzeR.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>isoformswitchanalyzer</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/isoformswitchanalyzer</td>
    </tr>
    <tr>
      <td></td>
      <td>ivar</td>
      <td>1.4.2</td>
      <td>1.4.2</td>
      <td>ivar</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iVar is a computational package that contains functions broadly useful for viral amplicon-based sequencing</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/andersen-lab/ivar</td>
      <td>Sequence Analysis</td>
      <td>ivar</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ivar/</td>
    </tr>
    <tr>
      <td></td>
      <td>iwtomics</td>
      <td>1.0.0</td>
      <td>1.26.0</td>
      <td>bioconductor-iwtomics</td>
      <td>To update</td>
      <td>iwtomics</td>
      <td>IWTomics</td>
      <td>Differential gene expression analysis, Differentially-methylated region identification, Peak calling, Genome annotation, Comparison</td>
      <td>Statistics and probability</td>
      <td>Interval-Wise Testing for Omics Data</td>
      <td>Implementation of the Interval-Wise Testing (IWT) for omics data. This inferential procedure tests for differences in "Omics" data between two groups of genomic regions (or between a group of genomic regions and a reference center of symmetry), and does not require fixing location and scale at the outset.</td>
      <td></td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/IWTomics.html</td>
      <td>Statistics</td>
      <td>iwtomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/iwtomics</td>
    </tr>
    <tr>
      <td></td>
      <td>jasminesv</td>
      <td>1.0.11</td>
      <td>1.1.5</td>
      <td>jasminesv</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Merge structural variants across samples</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/mkirsche/Jasmine/</td>
      <td>Variant Analysis</td>
      <td>jasminesv</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/jasminesv/</td>
    </tr>
    <tr>
      <td></td>
      <td>jbrowse</td>
      <td>1.16.11</td>
      <td>1.16.11</td>
      <td>jbrowse</td>
      <td>Up-to-date</td>
      <td>jbrowse</td>
      <td>JBrowse</td>
      <td>Genome visualisation</td>
      <td>Genomics</td>
      <td>JBrowse Genome Browser integrated as a Galaxy Tool</td>
      <td>Slick, speedy genome browser with a responsive and dynamic AJAX interface for visualization of genome data. Being developed by the GMOD project as a successor to GBrowse.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://jbrowse.org</td>
      <td>Sequence Analysis</td>
      <td>jbrowse</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/jbrowse</td>
    </tr>
    <tr>
      <td></td>
      <td>jcvi_gff_stats</td>
      <td>0.8.4</td>
      <td>1.3.9</td>
      <td>jcvi</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute statistics from a genome annotation in GFF3 format (using JCVI Python utilities)</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/tanghaibao/jcvi</td>
      <td>Sequence Analysis</td>
      <td>jcvi_gff_stats</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/jcvi_gff_stats</td>
    </tr>
    <tr>
      <td></td>
      <td>jellyfish</td>
      <td></td>
      <td>2.3.1</td>
      <td>kmer-jellyfish</td>
      <td>To update</td>
      <td>Jellyfish</td>
      <td>Jellyfish</td>
      <td>k-mer counting</td>
      <td>Sequence analysis, Genomics</td>
      <td>Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA</td>
      <td>A command-line algorithm for counting k-mers in DNA sequence.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/gmarcais/Jellyfish</td>
      <td>Assembly</td>
      <td>jellyfish</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/jellyfish</td>
    </tr>
    <tr>
      <td></td>
      <td>join_files_by_id</td>
      <td>1.0</td>
      <td>1.11.6</td>
      <td>r-data.table</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>This tool will join datasets according to a column with identifier</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>join_files_by_id</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/join_files_by_id</td>
    </tr>
    <tr>
      <td></td>
      <td>jq</td>
      <td>1.0</td>
      <td>1.5</td>
      <td>jq</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>JQ is a lightweight and flexible command-line JSON processor</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://stedolan.github.io/jq/</td>
      <td>Text Manipulation</td>
      <td>jq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/jq</td>
    </tr>
    <tr>
      <td></td>
      <td>jvarkit</td>
      <td>20201223</td>
      <td>20201223</td>
      <td>jvarkit-wgscoverageplotter</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Jvarkit : Java utilities for Bioinformatics</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://lindenb.github.io/jvarkit/</td>
      <td>SAM</td>
      <td>jvarkit</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/iuc/tree/master/tools/jvarkit</td>
    </tr>
    <tr>
      <td></td>
      <td>kallisto</td>
      <td>0.48.0</td>
      <td>0.50.1</td>
      <td>kallisto</td>
      <td>To update</td>
      <td>kallisto</td>
      <td>kallisto</td>
      <td>Gene expression profiling</td>
      <td>Transcriptomics, RNA-seq, Gene expression</td>
      <td>kallisto is a program for quantifying abundances of transcripts from RNA-Seqdata, or more generally of target sequences using high-throughput sequencingreads. It is based on the novel idea of pseudoalignment for rapidlydetermining the compatibility of reads with targets, without the need foralignment.</td>
      <td>A program for quantifying abundances of transcripts from RNA-Seq data, or more generally of target sequences using high-throughput sequencing reads. It is based on the novel idea of pseudoalignment for rapidly determining the compatibility of reads with targets, without the need for alignment.</td>
      <td></td>
      <td>To update</td>
      <td>https://pachterlab.github.io/kallisto/</td>
      <td>Transcriptomics</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/kallisto/</td>
    </tr>
    <tr>
      <td></td>
      <td>kc-align</td>
      <td>1.0.2</td>
      <td>1.0.2</td>
      <td>kcalign</td>
      <td>Up-to-date</td>
      <td>kc-align</td>
      <td>kc-align</td>
      <td>Multiple sequence alignment</td>
      <td>Mapping</td>
      <td>Kc-Align custom tool</td>
      <td>A fast and accurate tool for performing codon-aware multiple sequence alignments</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/davebx/kc-align</td>
      <td>Sequence Analysis</td>
      <td>kc_align</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/kc-align</td>
    </tr>
    <tr>
      <td></td>
      <td>khmer</td>
      <td>3.0.0a3</td>
      <td>3.0.0a3</td>
      <td>khmer</td>
      <td>Up-to-date</td>
      <td>khmer</td>
      <td>khmer</td>
      <td>Standardisation and normalisation, De-novo assembly</td>
      <td>Sequence assembly</td>
      <td>In-memory nucleotide sequence k-mer counting, filtering, graph traversal and more</td>
      <td>khmer is a set of command-line tools for working with DNA shotgun sequencing data from genomes, transcriptomes, metagenomes, and single cells. khmer can make de novo assemblies faster, and sometimes better. khmer can also identify (and fix) problems with shotgun data.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://khmer.readthedocs.org/</td>
      <td>Assembly, Next Gen Mappers</td>
      <td>khmer</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/khmer</td>
    </tr>
    <tr>
      <td></td>
      <td>king</td>
      <td>2.2.7</td>
      <td>2.2.7</td>
      <td>king</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Kinship-based INference for Gwas</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://people.virginia.edu/~wc9c/KING/</td>
      <td>Variant Analysis</td>
      <td>king</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/king/</td>
    </tr>
    <tr>
      <td></td>
      <td>kleborate</td>
      <td>2.3.2</td>
      <td>2.3.2</td>
      <td>kleborate</td>
      <td>Up-to-date</td>
      <td>kleborate</td>
      <td>Kleborate</td>
      <td>Multilocus sequence typing, Genome assembly, Virulence prediction</td>
      <td>Public health and epidemiology, Metagenomics, Population genomics, Sequence assembly, Whole genome sequencing</td>
      <td>Screen genome assemblies of Klebsiella pneumoniae and the Klebsiella pneumoniae species complex (KpSC)</td>
      <td>Genomic surveillance framework and global population structure for Klebsiella pneumoniae.Kleborate is a tool to screen genome assemblies of Klebsiella pneumoniae and the Klebsiella pneumoniae species complex (KpSC) for:.A manuscript describing the Kleborate software in full is currently in preparation. In the meantime, if you use Kleborate, please cite the preprint: Lam, MMC. et al. Genomic surveillance framework and global population structure for Klebsiella pneumoniae. bioRxiv (2020).</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/katholt/Kleborate/wiki</td>
      <td>Metagenomics</td>
      <td>kleborate</td>
      <td>iuc</td>
      <td>https://github.com/katholt/Kleborate</td>
    </tr>
    <tr>
      <td></td>
      <td>kma</td>
      <td>1.2.21</td>
      <td>1.4.14</td>
      <td>kma</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Map with KMA</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/genomicepidemiology/kma</td>
      <td>Next Gen Mappers</td>
      <td>kma</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/kma</td>
    </tr>
    <tr>
      <td></td>
      <td>kofamscan</td>
      <td>1.3.0</td>
      <td>1.3.0</td>
      <td>kofamscan</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Gene function annotation tool based on KEGG Orthology and hidden Markov model</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/takaram/kofam_scan</td>
      <td>Sequence Analysis</td>
      <td>kofamscan</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/kofamscan</td>
    </tr>
    <tr>
      <td></td>
      <td>kraken_biom</td>
      <td>1.2.0</td>
      <td>1.2.0</td>
      <td>kraken-biom</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Create BIOM-format tables (http://biom-format.org) from Kraken output (http://ccb.jhu.edu/software/kraken/)</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/smdabdoub/kraken-biom</td>
      <td>Metagenomics</td>
      <td>kraken_biom</td>
      <td>iuc</td>
      <td>https://github.com/smdabdoub/kraken-biom</td>
    </tr>
    <tr>
      <td></td>
      <td>kraken_taxonomy_report</td>
      <td>0.0.3</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Kraken taxonomy report</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/blankenberg/Kraken-Taxonomy-Report</td>
      <td>Metagenomics</td>
      <td>kraken_taxonomy_report</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/kraken_taxonomy_report</td>
    </tr>
    <tr>
      <td></td>
      <td>krakentools</td>
      <td>1.2</td>
      <td>1.2</td>
      <td>krakentools</td>
      <td>Up-to-date</td>
      <td>krakentools</td>
      <td>KrakenTools</td>
      <td>Visualisation, Aggregation</td>
      <td>Taxonomy, Metagenomics</td>
      <td>KrakenTools is a suite of scripts to be used alongside the Kraken</td>
      <td>KrakenTools provides individual scripts to analyze Kraken/Kraken2/Bracken/KrakenUniq output files</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/jenniferlu717/KrakenTools</td>
      <td>Metagenomics</td>
      <td>krakentools</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/krakentools</td>
    </tr>
    <tr>
      <td></td>
      <td>krocus</td>
      <td>1.0.1</td>
      <td>1.0.3</td>
      <td>krocus</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Predict MLST directly from uncorrected long reads</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/quadram-institute-bioscience/krocus</td>
      <td>Sequence Analysis</td>
      <td>krocus</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/krocus</td>
    </tr>
    <tr>
      <td></td>
      <td>last</td>
      <td>1205</td>
      <td>1542</td>
      <td>last</td>
      <td>To update</td>
      <td>last</td>
      <td>LAST</td>
      <td>Sequence alignment</td>
      <td>Genomics, Comparative genomics</td>
      <td>LAST finds similar regions between sequences.</td>
      <td>Short read alignment program incorporating quality scores</td>
      <td></td>
      <td>To update</td>
      <td>http://last.cbrc.jp/</td>
      <td>Sequence Analysis</td>
      <td>last</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/last</td>
    </tr>
    <tr>
      <td></td>
      <td>lastz</td>
      <td>1.04.22</td>
      <td>1.04.22</td>
      <td>lastz</td>
      <td>Up-to-date</td>
      <td>lastz</td>
      <td>LASTZ</td>
      <td>Sequence alignment, Read mapping</td>
      <td>Genomics</td>
      <td>Galaxy wrappers for the Lastz and Lastz_d</td>
      <td>A tool for (1) aligning two DNA sequences, and (2) inferring appropriate scoring parameters automatically.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/lastz/lastz</td>
      <td>Next Gen Mappers</td>
      <td>lastz</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/lastz</td>
    </tr>
    <tr>
      <td></td>
      <td>lcrgenie</td>
      <td>1.0.2</td>
      <td></td>
      <td>lcr_genie</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Ligase Chain Reaction Genie</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/neilswainston/LCRGenie</td>
      <td>Synthetic Biology</td>
      <td>lcrgenie</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/lcrgenie</td>
    </tr>
    <tr>
      <td></td>
      <td>legsta</td>
      <td>0.5.1</td>
      <td>0.5.1</td>
      <td>legsta</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Performs in silico Legionella pneumophila sequence based typing.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/tseemann/legsta</td>
      <td>Sequence Analysis</td>
      <td>legsta</td>
      <td>iuc</td>
      <td>https://github.com/tseemann/legsta</td>
    </tr>
    <tr>
      <td></td>
      <td>length_and_gc_content</td>
      <td>0.1.2</td>
      <td>1.3.2</td>
      <td>r-optparse</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Gets gene length and gc content from a fasta and a GTF file</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/length_and_gc_content</td>
      <td>Fasta Manipulation, Statistics, RNA, Micro-array Analysis</td>
      <td>length_and_gc_content</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/length_and_gc_content</td>
    </tr>
    <tr>
      <td></td>
      <td>limma_voom</td>
      <td>3.58.1</td>
      <td>3.58.1</td>
      <td>bioconductor-limma</td>
      <td>Up-to-date</td>
      <td>limma</td>
      <td>limma</td>
      <td>RNA-Seq analysis</td>
      <td>Molecular biology, Genetics</td>
      <td>Perform RNA-Seq differential expression analysis using limma voom pipeline</td>
      <td>Data analysis, linear models and differential expression for microarray data.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://bioconductor.org/packages/release/bioc/html/limma.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>limma_voom</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/limma_voom</td>
    </tr>
    <tr>
      <td></td>
      <td>lineagespot</td>
      <td>1.6.0</td>
      <td></td>
      <td>r-base</td>
      <td>To update</td>
      <td>lineagespot</td>
      <td>lineagespot</td>
      <td>Variant calling</td>
      <td>Metagenomics, Gene transcripts, Evolutionary biology, Sequencing, Genetic variation</td>
      <td>Identification of SARS-CoV-2 related metagenomic mutations based on a single (or a list of) variant(s) file(s)</td>
      <td>Lineagespot is a framework written in R, and aims to identify and assign different SARS-CoV-2 lineages based on a single variant file (i.e., variant calling format).</td>
      <td></td>
      <td>To update</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/lineagespot.html</td>
      <td>Metagenomics, Sequence Analysis</td>
      <td>lineagespot</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/lineagespot</td>
    </tr>
    <tr>
      <td></td>
      <td>links</td>
      <td>2.0.1</td>
      <td>2.0.1</td>
      <td>links</td>
      <td>Up-to-date</td>
      <td>links</td>
      <td>LINKS</td>
      <td>Scaffolding, Genome assembly, Read mapping, Read pre-processing, Sequence trimming</td>
      <td>Sequence assembly, Mapping, Sequencing</td>
      <td>Scaffold genome assemblies with long reads.</td>
      <td>LINKS (Long Interval Nucleotide K-mer Scaffolder) is a genomics application for scaffolding genome assemblies with long reads, such as those produced by Oxford Nanopore Technologies Ltd. It can be used to scaffold high-quality draft genome assemblies with any long sequences (eg. ONT reads, PacBio reads, other draft genomes, etc). It is also used to scaffold contig pairs linked by ARCS/ARKS.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/bcgsc/LINKS</td>
      <td>Assembly</td>
      <td>links</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/links</td>
    </tr>
    <tr>
      <td></td>
      <td>lofreq</td>
      <td>2.1.5</td>
      <td>2.1.5</td>
      <td>lofreq</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>LoFreq is a fast and sensitive variant-caller for inferring SNVs and indelsfrom next-generation sequencing data. It makes full use of base-call qualitiesand other sources of errors inherent in sequencing (e.g. mapping or base/indelalignment uncertainty), which are usually ignored by other methods or onlyused for filtering.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://csb5.github.io/lofreq/</td>
      <td>Variant Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/lofreq</td>
    </tr>
    <tr>
      <td></td>
      <td>lorikeet</td>
      <td>20</td>
      <td>20</td>
      <td>lorikeet</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tools for M. tuberculosis DNA fingerprinting (spoligotyping)</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/AbeelLab/lorikeet</td>
      <td>Sequence Analysis</td>
      <td>lorikeet_spoligotype</td>
      <td>iuc</td>
      <td>https://github.com/AbeelLab/lorikeet</td>
    </tr>
    <tr>
      <td></td>
      <td>lumpy_sv</td>
      <td>0.3.1</td>
      <td>0.3.1</td>
      <td>lumpy-sv</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>LUMPY - a general probabilistic framework for structural variant discovery</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://layerlab.org/software/</td>
      <td>Variant Analysis</td>
      <td>lumpy_sv</td>
      <td>iuc</td>
      <td>https://github.com/arq5x/lumpy-sv</td>
    </tr>
    <tr>
      <td></td>
      <td>m6anet</td>
      <td>2.1.0</td>
      <td>2.1.0</td>
      <td>m6anet</td>
      <td>Up-to-date</td>
      <td>m6Anet</td>
      <td>m6Anet</td>
      <td>Quantification, Imputation, Gene expression profiling</td>
      <td>RNA-Seq, Transcriptomics, RNA, Machine learning</td>
      <td>m6anet to detect m6A RNA modifications from nanopore data</td>
      <td>Detection of m6A from direct RNA sequencing using a Multiple Instance Learning framework.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://m6anet.readthedocs.io/en/latest</td>
      <td>Sequence Analysis</td>
      <td>m6anet</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/m6anet</td>
    </tr>
    <tr>
      <td></td>
      <td>maaslin2</td>
      <td>0.99.12</td>
      <td>0.99.12</td>
      <td>maaslin2</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MaAsLin2 is comprehensive R package for efficiently determining multivariable association between microbial meta'omic features and clinical metadata.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://huttenhower.sph.harvard.edu/maaslin</td>
      <td>Metagenomics</td>
      <td>maaslin2</td>
      <td>iuc</td>
      <td>https://github.com/biobakery/Maaslin2</td>
    </tr>
    <tr>
      <td></td>
      <td>macs2</td>
      <td>2.2.9.1</td>
      <td>2.2.9.1</td>
      <td>macs2</td>
      <td>Up-to-date</td>
      <td>macs</td>
      <td>MACS</td>
      <td>Peak calling, Enrichment analysis, Gene regulatory network analysis</td>
      <td>ChIP-seq, Molecular interactions, pathways and networks, Transcription factors and regulatory sites</td>
      <td>MACS - Model-based Analysis of ChIP-Seq</td>
      <td>Model-based Analysis of ChIP-seq data.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/taoliu/MACS</td>
      <td>Sequence Analysis, Statistics</td>
      <td>macs2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/macs2</td>
    </tr>
    <tr>
      <td></td>
      <td>maf_stats</td>
      <td>1.0.2+galaxy0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MAF Coverage statistics</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/interval2maf/</td>
      <td>Genomic Interval Operations</td>
      <td>maf_stats</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/interval2maf/</td>
    </tr>
    <tr>
      <td></td>
      <td>mageck</td>
      <td>0.5.9.2</td>
      <td>0.5.9.5</td>
      <td>mageck</td>
      <td>To update</td>
      <td>mageck</td>
      <td>MAGeCK</td>
      <td>Genetic variation analysis</td>
      <td>Genetics, Genetic variation, Genomics</td>
      <td>Model-based Analysis of Genome-wide CRISPR-Cas9 Knockout (MAGeCK) is a computational tool to identifyimportant genes from the recent genome-scale CRISPR-Cas9 knockout screens technology.</td>
      <td>Computational tool to identify important genes from the recent genome-scale CRISPR-Cas9 knockout screens technology.</td>
      <td></td>
      <td>To update</td>
      <td>https://sourceforge.net/projects/mageck/</td>
      <td>Genome editing</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mageck</td>
    </tr>
    <tr>
      <td></td>
      <td>maker</td>
      <td>2.31.11</td>
      <td>3.01.03</td>
      <td>maker</td>
      <td>To update</td>
      <td>maker</td>
      <td>MAKER</td>
      <td>Genome annotation</td>
      <td>Genomics, DNA, Sequence analysis</td>
      <td>MAKER is a portable and easily configurable genome annotation pipeline.Its purpose is to allow smaller eukaryotic and prokaryotic genome projects to independently annotate their genomes and to create genome databases.</td>
      <td>Portable and easily configurable genome annotation pipeline. Its purpose is to allow smaller eukaryotic and prokaryotic genome projects to independently annotate their genomes and to create genome databases.</td>
      <td></td>
      <td>To update</td>
      <td>http://www.yandell-lab.org/software/maker.html</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/maker</td>
    </tr>
    <tr>
      <td></td>
      <td>malt</td>
      <td>0.5.3</td>
      <td>0.62</td>
      <td>malt</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Aligns an input sequence (DNA or proteins) against an index representing a collection of reference DNA or protein sequences.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/husonlab/malt</td>
      <td>Next Gen Mappers</td>
      <td>malt_run</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/malt</td>
    </tr>
    <tr>
      <td></td>
      <td>map_param_value</td>
      <td>0.2.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Map a parameter value to another value</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>map_param_value</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/map_param_value</td>
    </tr>
    <tr>
      <td></td>
      <td>mapseq</td>
      <td>2.1.1</td>
      <td></td>
      <td>perl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>fast and accurate sequence read classification tool designed to assign taxonomy and OTU classifications to ribosomal RNA sequences.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/jfmrod/MAPseq</td>
      <td>Metagenomics</td>
      <td>mapseq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mapseq</td>
    </tr>
    <tr>
      <td></td>
      <td>mash</td>
      <td>2.3</td>
      <td>2.3</td>
      <td>mash</td>
      <td>Up-to-date</td>
      <td>mash</td>
      <td>Mash</td>
      <td>Sequence distance matrix generation</td>
      <td>Genomics, Metagenomics, Statistics and probability, Sequence analysis, DNA mutation</td>
      <td>Fast genome and metagenome distance estimation using MinHash</td>
      <td>Fast genome and metagenome distance estimation using MinHash.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/marbl/Mash</td>
      <td>Metagenomics</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mash</td>
    </tr>
    <tr>
      <td></td>
      <td>mashmap</td>
      <td>3.1.3</td>
      <td>3.1.3</td>
      <td>mashmap</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Fast local alignment boundaries</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mashmap</td>
      <td>Sequence Analysis</td>
      <td>mashmap</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mashmap</td>
    </tr>
    <tr>
      <td></td>
      <td>masigpro</td>
      <td>1.49.3</td>
      <td>8.25</td>
      <td>coreutils</td>
      <td>To update</td>
      <td>masigpro</td>
      <td>maSigPro</td>
      <td>Regression analysis</td>
      <td>Gene expression, Molecular genetics, Microarray experiment, RNA-Seq</td>
      <td>Identify significantly differential expression profiles in time-course microarray experiments</td>
      <td>Regression based approach to find genes for which there are significant gene expression profile differences between experimental groups in time course microarray and RNA-Seq experiments.</td>
      <td></td>
      <td>To update</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/maSigPro.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>masigpro</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/masigpro</td>
    </tr>
    <tr>
      <td></td>
      <td>maxbin2</td>
      <td></td>
      <td>2.2.7</td>
      <td>maxbin2</td>
      <td>To update</td>
      <td>maxbin</td>
      <td>MaxBin</td>
      <td>Sequence assembly</td>
      <td>Metagenomics, Sequence assembly, Microbiology</td>
      <td>clusters metagenomic contigs into bins</td>
      <td>Software for binning assembled metagenomic sequences based on an Expectation-Maximization algorithm.</td>
      <td></td>
      <td>To update</td>
      <td>https://downloads.jbei.org/data/microbial_communities/MaxBin/MaxBin.html</td>
      <td>Metagenomics</td>
      <td>maxbin2</td>
      <td>mbernt</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/maxbin2/</td>
    </tr>
    <tr>
      <td></td>
      <td>mcl</td>
      <td>22.282</td>
      <td>22.282</td>
      <td>mcl</td>
      <td>Up-to-date</td>
      <td>mcl</td>
      <td>MCL</td>
      <td>Clustering, Network analysis, Gene regulatory network analysis</td>
      <td>Molecular interactions, pathways and networks</td>
      <td>The Markov Cluster Algorithm, a cluster algorithm for graphs</td>
      <td>MCL is a clustering algorithm widely used in bioinformatics and gaining traction in other fields.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://micans.org/mcl/man/mcl.html</td>
      <td>Sequence Analysis</td>
      <td>mcl</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/mcl</td>
    </tr>
    <tr>
      <td></td>
      <td>medaka</td>
      <td>1.7.2</td>
      <td>1.11.3</td>
      <td>medaka</td>
      <td>To update</td>
      <td>medaka</td>
      <td>Medaka</td>
      <td>Base-calling, Variant calling, Sequence assembly</td>
      <td>Sequence assembly, Machine learning</td>
      <td>Sequence correction provided by ONT Research</td>
      <td>medaka is a tool to create consensus sequences and variant calls from nanopore sequencing data. This task is performed using neural networks applied a pileup of individual sequencing reads against a draft assembly.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/nanoporetech/medaka</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/medaka</td>
    </tr>
    <tr>
      <td></td>
      <td>megahit</td>
      <td>1.2.9</td>
      <td>1.2.9</td>
      <td>megahit</td>
      <td>Up-to-date</td>
      <td>megahit</td>
      <td>MEGAHIT</td>
      <td>Genome assembly</td>
      <td>Metagenomics, Sequencing, Ecology, Sequence assembly</td>
      <td>An ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph.</td>
      <td>Single node assembler for large and complex metagenomics NGS reads, such as soil. It makes use of succinct de Bruijn graph to achieve low memory usage, whereas its goal is not to make memory usage as low as possible.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/voutcn/megahit</td>
      <td>Sequence Analysis, Assembly, Metagenomics</td>
      <td>megahit</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/megahit</td>
    </tr>
    <tr>
      <td></td>
      <td>megahit_contig2fastg</td>
      <td>1.1.3</td>
      <td>1.2.9</td>
      <td>megahit</td>
      <td>To update</td>
      <td>megahit</td>
      <td>MEGAHIT</td>
      <td>Genome assembly</td>
      <td>Metagenomics, Sequencing, Ecology, Sequence assembly</td>
      <td>A subprogram within the Megahit toolkit for converting contigs to assembly graphs (fastg)</td>
      <td>Single node assembler for large and complex metagenomics NGS reads, such as soil. It makes use of succinct de Bruijn graph to achieve low memory usage, whereas its goal is not to make memory usage as low as possible.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/voutcn/megahit/blob/master/tools/toolkit.cpp</td>
      <td>Sequence Analysis, Assembly, Metagenomics</td>
      <td>megahit_contig2fastg</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/megahit_contig2fastg</td>
    </tr>
    <tr>
      <td></td>
      <td>megan</td>
      <td>6.21.7</td>
      <td>6.25.9</td>
      <td>megan</td>
      <td>To update</td>
      <td>megan</td>
      <td>MEGAN</td>
      <td>Sequence analysis, Taxonomic classification</td>
      <td>Sequence analysis</td>
      <td>MEGAN Community Edition - Interactive exploration and analysis of large-scale microbiome sequencing data.  MEGAN is a tool for studying the taxonomic content of a set of DNA reads, typically collected in a metagenomics project.In a preprocessing step, a sequence alignment of all reads against a suitable database of reference DNA or proteinsequences must be performed to produce an input file for the program. MEGAN is suitable for DNA reads (metagenomedata), RNA reads (metatranscriptome data), peptide sequences (metaproteomics data) and, using a suitable synonymsfile that maps SILVA ids to taxon ids, for 16S rRNA data (amplicon sequencing).</td>
      <td>Metagenome Analysis Software - MEGAN (MEtaGenome ANalyzer) is a new computer program that allows laptop analysis of large metagenomic datasets. In a preprocessing step, the set of DNA reads (or contigs) is compared against databases of known sequences using BLAST or another comparison tool. MEGAN can then be used to compute and interactively explore the taxonomical content of the dataset, employing the NCBI taxonomy to summarize and order the results.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/husonlab/megan-ce</td>
      <td>Sequence Analysis</td>
      <td>megan</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/megan</td>
    </tr>
    <tr>
      <td></td>
      <td>meme</td>
      <td>5.4.1</td>
      <td>5.5.5</td>
      <td>meme</td>
      <td>To update</td>
      <td>meme_fimo</td>
      <td>meme_fimo</td>
      <td>Sequence motif discovery, Nucleic acid feature detection, Protein feature detection, Statistical calculation</td>
      <td>Sequence analysis, Genetic variation, Statistics and probability, Data mining</td>
      <td>The MEME Suite allows the biologist to discover novel motifs in collections of unaligned nucleotideor protein sequences, and to perform a wide variety of other motif-based analyses.</td>
      <td>A software tool for scanning DNA or protein sequences with motifs described as position-specific scoring matrices.</td>
      <td></td>
      <td>To update</td>
      <td>http://meme-suite.org/</td>
      <td>ChIP-seq</td>
      <td>meme</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/meme</td>
    </tr>
    <tr>
      <td></td>
      <td>meme_chip</td>
      <td>4.11.2</td>
      <td>1.3.26</td>
      <td>graphicsmagick</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Performs motif discovery, motif enrichment analysis and clustering on large nucleotide datasets.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://meme-suite.org/</td>
      <td>ChIP-seq</td>
      <td>meme_chip</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/meme_chip</td>
    </tr>
    <tr>
      <td></td>
      <td>meningotype</td>
      <td>0.8.5</td>
      <td>0.8.5</td>
      <td>meningotype</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Assign sequence type to N. meningitidis genome assemblies</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/MDU-PHL/meningotype</td>
      <td>Sequence Analysis</td>
      <td>meningotype</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/meningotype</td>
    </tr>
    <tr>
      <td></td>
      <td>merlin</td>
      <td>1.1.2</td>
      <td>1.1.2</td>
      <td>merlin</td>
      <td>Up-to-date</td>
      <td>merlin</td>
      <td>Merlin</td>
      <td>Haplotype mapping, Genetic mapping</td>
      <td>GWAS study, Mapping</td>
      <td>Pedigree Analysis package</td>
      <td>Can be used for parametric and non-parametric linkage analysis, regression-based linkage analysis or association analysis for quantitative traits, ibd and kinship estimation, haplotyping, error detection and simulation</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://csg.sph.umich.edu/abecasis/Merlin/</td>
      <td>Variant Analysis</td>
      <td>merlin</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/merlin/</td>
    </tr>
    <tr>
      <td></td>
      <td>merqury</td>
      <td>1.3</td>
      <td>1.3</td>
      <td>merqury</td>
      <td>Up-to-date</td>
      <td>merqury</td>
      <td>Merqury</td>
      <td>Genome assembly, k-mer counting, Scaffolding, Phasing, De-novo assembly</td>
      <td>Sequence assembly, Whole genome sequencing, Plant biology</td>
      <td>Merqury is a tool for evaluating genomes assemblies based of k-mer operations.</td>
      <td>Reference-free quality, completeness, and phasing assessment for genome assemblies.Evaluate genome assemblies with k-mers and more.Often, genome assembly projects have illumina whole genome sequencing reads available for the assembled individual.Merqury provides a set of tools for this purpose.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/marbl/merqury</td>
      <td>Assembly</td>
      <td>merqury</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/merqury</td>
    </tr>
    <tr>
      <td></td>
      <td>meryl</td>
      <td>1.3</td>
      <td>1.3</td>
      <td>merqury</td>
      <td>Up-to-date</td>
      <td>meryl</td>
      <td>Meryl</td>
      <td>k-mer counting</td>
      <td>Whole genome sequencing, Genomics, Sequence analysis, Sequencing</td>
      <td>Meryl a k-mer counter.</td>
      <td>Meryl is a tool for counting and working with sets of k-mers that was originally developed for use in the Celera Assembler and has since been migrated and maintained as part of Canu.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/marbl/meryl</td>
      <td>Assembly</td>
      <td>meryl</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/meryl</td>
    </tr>
    <tr>
      <td></td>
      <td>metabat2</td>
      <td>2.15</td>
      <td>2.15</td>
      <td>metabat2</td>
      <td>Up-to-date</td>
      <td>MetaBAT_2</td>
      <td>MetaBAT 2</td>
      <td>Read binning, Sequence assembly, Genome annotation</td>
      <td>Metagenomics, Sequence assembly, Metagenomic sequencing</td>
      <td>MetaBAT2 (Metagenome Binning based on Abundance and Tetranucleotide frequency) is an automated metagenome binningsoftware that integrates empirical probabilistic distances of genome abundance and tetranucleotide frequency.</td>
      <td>an adaptive binning algorithm for robust and efficient genome reconstruction from metagenome assemblies | MetaBAT2 clusters metagenomic contigs into different "bins", each of which should correspond to a putative genome | MetaBAT2 uses nucleotide composition information and source strain abundance (measured by depth-of-coverage by aligning the reads to the contigs) to perform binning</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://bitbucket.org/berkeleylab/metabat/src/master/</td>
      <td>Metagenomics</td>
      <td>metabat2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/metabat2/</td>
    </tr>
    <tr>
      <td></td>
      <td>metaeuk</td>
      <td>5.34c21f2</td>
      <td>6.a5d39d9</td>
      <td>metaeuk</td>
      <td>To update</td>
      <td>MetaEuk</td>
      <td>MetaEuk</td>
      <td>Homology-based gene prediction</td>
      <td>Metagenomics, Gene and protein families</td>
      <td>MetaEuk is a modular toolkit designed for large-scale gene discovery andannotation in eukaryotic metagenomic contigs. Metaeuk combines the fast andsensitive homology search capabilities of MMseqs2 with a dynamic programmingprocedure to recover optimal exons sets. It reduces redundancies in multiplediscoveries of the same gene and resolves conflicting gene predictions onthe same strand.</td>
      <td>MetaEuk - sensitive, high-throughput gene discovery and annotation for large-scale eukaryotic metagenomics</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/soedinglab/metaeuk</td>
      <td>Sequence Analysis, Genome annotation</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/metaeuk</td>
    </tr>
    <tr>
      <td></td>
      <td>metagenomeseq</td>
      <td>1.16.0-0.0.1</td>
      <td>1.43.0</td>
      <td>bioconductor-metagenomeseq</td>
      <td>To update</td>
      <td>metagenomeseq</td>
      <td>metagenomeSeq</td>
      <td>Sequence visualisation, Statistical calculation</td>
      <td>Metagenomics, Sequencing</td>
      <td>metagenomeSeq Normalization</td>
      <td>Designed to determine features (be it Operational Taxanomic Unit (OTU), species, etc.) that are differentially abundant between two or more groups of multiple samples. It is designed to address the effects of both normalization and under-sampling of microbial communities on disease association detection and the testing of feature correlations.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Metagenomics</td>
      <td>metagenomeseq_normalization</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/metagenomeseq</td>
    </tr>
    <tr>
      <td></td>
      <td>metaphlan</td>
      <td>4.0.6</td>
      <td>4.1.0</td>
      <td>metaphlan</td>
      <td>To update</td>
      <td>metaphlan</td>
      <td>MetaPhlAn</td>
      <td>Nucleic acid sequence analysis, Phylogenetic tree analysis</td>
      <td>Metagenomics, Phylogenomics</td>
      <td>MetaPhlAn for Metagenomic Phylogenetic Analysis</td>
      <td>Computational tool for profiling the composition of microbial communities from metagenomic shotgun sequencing data.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/biobakery/MetaPhlAn</td>
      <td>Metagenomics</td>
      <td>metaphlan</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/metaphlan/</td>
    </tr>
    <tr>
      <td></td>
      <td>migmap</td>
      <td>1.0.3</td>
      <td>1.0.3</td>
      <td>migmap</td>
      <td>Up-to-date</td>
      <td>MiGMAP</td>
      <td>MiGMAP</td>
      <td>Sequence analysis, Read mapping</td>
      <td>Immunoproteins, genes and antigens, Sequence analysis</td>
      <td>mapper for full-length T- and B-cell repertoire sequencing</td>
      <td>Mapper for full-length T- and B-cell repertoire sequencing.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/mikessh/migmap</td>
      <td>RNA, Sequence Analysis</td>
      <td>migmap</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/migmap</td>
    </tr>
    <tr>
      <td></td>
      <td>minia</td>
      <td>3.2.6</td>
      <td>3.2.6</td>
      <td>minia</td>
      <td>Up-to-date</td>
      <td>minia</td>
      <td>Minia</td>
      <td>Genome assembly</td>
      <td>Sequence assembly</td>
      <td>Short-read assembler based on a de Bruijn graph</td>
      <td>Short-read assembler based on a de Bruijn graph, capable of assembling a human genome on a desktop computer in a day.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://gatb.inria.fr/software/minia/</td>
      <td>Assembly</td>
      <td>minia</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/minia</td>
    </tr>
    <tr>
      <td></td>
      <td>miniasm</td>
      <td>0.3_r179</td>
      <td>0.3</td>
      <td>miniasm</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Miniasm - Ultrafast de novo assembly for long noisy reads (though having no consensus step)</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/lh3/miniasm</td>
      <td>Assembly</td>
      <td>miniasm</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/miniasm</td>
    </tr>
    <tr>
      <td></td>
      <td>minimap2</td>
      <td>2.26</td>
      <td>2.26</td>
      <td>minimap2</td>
      <td>Up-to-date</td>
      <td>minimap2</td>
      <td>Minimap2</td>
      <td>Pairwise sequence alignment</td>
      <td>Mapping</td>
      <td>A fast pairwise aligner for genomic and spliced nucleotide sequences</td>
      <td>Pairwise aligner for genomic and spliced nucleotide sequences</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/lh3/minimap2</td>
      <td>Next Gen Mappers</td>
      <td>minimap2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/minimap2</td>
    </tr>
    <tr>
      <td></td>
      <td>miniprot</td>
      <td>0.12</td>
      <td>0.12</td>
      <td>miniprot</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Align a protein sequence against a genome with affine gap penalty, splicing and frameshift.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/lh3/miniprot</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/miniprot</td>
    </tr>
    <tr>
      <td></td>
      <td>mirnature</td>
      <td>1.1</td>
      <td>1.1</td>
      <td>mirnature</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Computational detection of canonical microRNAs</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/Bierinformatik/miRNAture</td>
      <td>RNA, Sequence Analysis</td>
      <td>mirnature</td>
      <td>iuc</td>
      <td>https://github.com/Bierinformatik/miRNAture</td>
    </tr>
    <tr>
      <td></td>
      <td>mitobim</td>
      <td>1.9.1</td>
      <td>1.9.1</td>
      <td>mitobim</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>assemble mitochondrial genomes</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/chrishah/MITObim</td>
      <td>Assembly</td>
      <td>mitobim</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mitobim</td>
    </tr>
    <tr>
      <td></td>
      <td>mitos</td>
      <td>1.1.5</td>
      <td>2.1.7</td>
      <td>mitos</td>
      <td>To update</td>
      <td>mitos</td>
      <td>MITOS</td>
      <td>Genome annotation</td>
      <td>Zoology, Whole genome sequencing</td>
      <td>de-novo annotation of metazoan mitochondrial genomes</td>
      <td>De novo metazoan mitochondrial genome annotation.</td>
      <td></td>
      <td>To update</td>
      <td>http://mitos.bioinf.uni-leipzig.de/</td>
      <td>Sequence Analysis</td>
      <td>mitos</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mitos</td>
    </tr>
    <tr>
      <td></td>
      <td>mlst</td>
      <td>2.22.0</td>
      <td>2.23.0</td>
      <td>mlst</td>
      <td>To update</td>
      <td>mlst</td>
      <td>MLST</td>
      <td>Taxonomic classification</td>
      <td>Immunoproteins, genes and antigens</td>
      <td>Scan contig files against PubMLST typing schemes</td>
      <td>Multi Locus Sequence Typing from an assembled genome or from a set of reads.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/tseemann/mlst</td>
      <td>Sequence Analysis</td>
      <td>mlst</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mlst</td>
    </tr>
    <tr>
      <td></td>
      <td>moabs</td>
      <td>1.3.4.6</td>
      <td>1.3.9.0</td>
      <td>moabs</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MOABS for differential methylation analysis on Bisulfite sequencing data.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/sunnyisgalaxy/moabs</td>
      <td>Epigenetics</td>
      <td>moabs</td>
      <td>iuc</td>
      <td>https://github.com/sunnyisgalaxy/moabs</td>
    </tr>
    <tr>
      <td></td>
      <td>mosdepth</td>
      <td>0.3.6</td>
      <td>0.3.6</td>
      <td>mosdepth</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>fast and flexible BAM/CRAM depth calculation</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/brentp/mosdepth</td>
      <td>SAM</td>
      <td>mosdepth</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mosdepth</td>
    </tr>
    <tr>
      <td></td>
      <td>mothur</td>
      <td>1.0</td>
      <td>1.48.0</td>
      <td>mothur</td>
      <td>To update</td>
      <td>mothur</td>
      <td>mothur</td>
      <td>DNA barcoding, Sequencing quality control, Sequence clustering, Taxonomic classification, Visualisation, Sequence read processing, Phylogenetic analysis</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Phylogeny</td>
      <td>Mothur wrappers</td>
      <td>Open-source, platform-independent, community-supported software for describing and comparing microbial communities</td>
      <td></td>
      <td>To update</td>
      <td>https://www.mothur.org</td>
      <td>Metagenomics</td>
      <td>mothur</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mothur</td>
    </tr>
    <tr>
      <td></td>
      <td>msaboot</td>
      <td>0.1.2</td>
      <td>0.1.2</td>
      <td>msaboot</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A multiple sequences alignment bootstrapping tool.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/phac-nml/msaboot</td>
      <td>Fasta Manipulation</td>
      <td>msaboot</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/msaboot</td>
    </tr>
    <tr>
      <td></td>
      <td>multigps</td>
      <td>0.74.0</td>
      <td></td>
      <td>fonts-conda-ecosystem</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Analyzes collections of multi-condition ChIP-seq data.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://mahonylab.org/software/multigps/</td>
      <td>ChIP-seq</td>
      <td>multigps</td>
      <td>iuc</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>multigsea</td>
      <td>1.12.0</td>
      <td>1.12.0</td>
      <td>bioconductor-multigsea</td>
      <td>Up-to-date</td>
      <td>multiGSEA</td>
      <td>multiGSEA</td>
      <td>Gene-set enrichment analysis, Aggregation, Pathway analysis</td>
      <td>Metabolomics, Molecular interactions, pathways and networks, Proteomics, Transcriptomics, Small molecules</td>
      <td>GSEA-based pathway enrichment analysis for multi-omics data</td>
      <td>A GSEA-based pathway enrichment analysis for multi-omics data.multiGSEA: a GSEA-based pathway enrichment analysis for multi-omics data, BMC Bioinformatics 21, 561 (2020).Combining GSEA-based pathway enrichment with multi omics data integration.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://bioconductor.org/packages/devel/bioc/html/multiGSEA.html</td>
      <td>Transcriptomics, Proteomics, Statistics</td>
      <td>multigsea</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/multigsea</td>
    </tr>
    <tr>
      <td></td>
      <td>multiqc</td>
      <td>1.11</td>
      <td>1.21</td>
      <td>multiqc</td>
      <td>To update</td>
      <td>multiqc</td>
      <td>MultiQC</td>
      <td>Validation</td>
      <td>Sequencing, Bioinformatics</td>
      <td>MultiQC aggregates results from bioinformatics analyses across many samples into a single report</td>
      <td>MultiQC aggregates results from multiple bioinformatics analyses across many samples into a single report. It searches a given directory for analysis logs and compiles a HTML report. It's a general use tool, perfect for summarising the output from numerous bioinformatics tools.</td>
      <td></td>
      <td>To update</td>
      <td>http://multiqc.info/</td>
      <td>Fastq Manipulation, Statistics, Visualization</td>
      <td>multiqc</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/multiqc</td>
    </tr>
    <tr>
      <td></td>
      <td>mummer4</td>
      <td>4.0.0rc1</td>
      <td>4.0.0rc1</td>
      <td>mummer4</td>
      <td>Up-to-date</td>
      <td>mumer4</td>
      <td>MUMmer4</td>
      <td>Multiple sequence alignment</td>
      <td>Sequence analysis, Human genetics</td>
      <td>Mummer4 Tools</td>
      <td>System for rapidly aligning large DNA sequences to one another.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/mummer4/mummer</td>
      <td>Sequence Analysis</td>
      <td>mummer4</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mummer4</td>
    </tr>
    <tr>
      <td></td>
      <td>mykrobe</td>
      <td>0.10.0</td>
      <td>0.13.0</td>
      <td>mykrobe</td>
      <td>To update</td>
      <td>Mykrobe</td>
      <td>Mykrobe</td>
      <td>Antimicrobial resistance prediction, Variant calling, Genotyping, Sequence trimming</td>
      <td>Whole genome sequencing, Genotype and phenotype, Probes and primers, Genetic variation, Metagenomics</td>
      <td>Antibiotic resistance predictions</td>
      <td>Antibiotic resistance prediction for Mycobacterium tuberculosis from genome sequence data with Mykrobe.Antibiotic resistance prediction in minutes.Table of Contents generated with DocToc.AMR prediction (Mykrobe predictor).Before attempting to install with bioconda, please ensure you have your channels set up as specified in the documentation. If you don't, you may run into issues with an older version of mykrobe being installed</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/Mykrobe-tools/mykrobe</td>
      <td>Sequence Analysis</td>
      <td>mykrobe</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mykrobe</td>
    </tr>
    <tr>
      <td></td>
      <td>mzmine</td>
      <td>3.9.0</td>
      <td>3.9.0</td>
      <td>mzmine</td>
      <td>Up-to-date</td>
      <td>mzmine</td>
      <td>MZmine</td>
      <td>Natural product identification, Standardisation and normalisation, Peptide database search, Deisotoping, Clustering, Filtering, Chromatographic alignment, Peak detection, Peptide identification, Chromatogram visualisation, Mass spectrum visualisation, Structure visualisation, Plotting, Heat map generation</td>
      <td>Proteomics, Metabolomics, Proteomics experiment, Small molecules</td>
      <td>mass-spectrometry data processing, with the main focus on LC-MS data</td>
      <td>Toolbox for visualization and analysis of LC-MS data in netCDF or mzXML.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://mzmine.github.io/</td>
      <td>Metabolomics</td>
      <td>mzmine_batch</td>
      <td>iuc</td>
      <td>https://github.com/workflow4metabolomics/tools-metabolomics/blob/master/tools/mzmine/</td>
    </tr>
    <tr>
      <td></td>
      <td>naltorfs</td>
      <td>0.1.2</td>
      <td>0.1.2</td>
      <td>naltorfs</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>nAlt-ORFs: Nested Alternate Open Reading Frames (nAltORFs)</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/BlankenbergLab/nAltORFs</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/naltorfs/</td>
    </tr>
    <tr>
      <td></td>
      <td>nanocompore</td>
      <td>1.0.0rc3.post2</td>
      <td>1.0.4</td>
      <td>nanocompore</td>
      <td>To update</td>
      <td>Nanocompore</td>
      <td>Nanocompore</td>
      <td>Post-translation modification site prediction, PolyA signal detection, Genotyping, k-mer counting</td>
      <td>Functional, regulatory and non-coding RNA, RNA-Seq, Gene transcripts, Transcriptomics, Transcription factors and regulatory sites</td>
      <td>Nanocompore compares 2 ONT nanopore direct RNA sequencing datasets from different experimental conditions expected to have a significant impact on RNA modifications. It is recommended to have at least 2 replicates per condition. For example one can use a control condition with a significantly reduced number of modifications such as a cell line for which a modification writing enzyme was knocked-down or knocked-out. Alternatively, on a smaller scale transcripts of interests could be synthesized in-vitro.</td>
      <td>RNA modifications detection by comparative Nanopore direct RNA sequencing.RNA modifications detection from Nanopore dRNA-Seq data.Nanocompore identifies differences in ONT nanopore sequencing raw signal corresponding to RNA modifications by comparing 2 samples.Analyses performed for the nanocompore paper.Nanocompore compares 2 ONT nanopore direct RNA sequencing datasets from different experimental conditions expected to have a significant impact on RNA modifications. It is recommended to have at least 2 replicates per condition. For example one can use a control condition with a significantly reduced number of modifications such as a cell line for which a modification writing enzyme was knocked-down or knocked-out. Alternatively, on a smaller scale transcripts of interests could be synthesized in-vitro</td>
      <td></td>
      <td>To update</td>
      <td>https://nanocompore.rna.rocks/</td>
      <td>Sequence Analysis</td>
      <td>nanocompore</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/nanocompore</td>
    </tr>
    <tr>
      <td></td>
      <td>nanoplot</td>
      <td>1.41.0</td>
      <td>1.42.0</td>
      <td>nanoplot</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Plotting tool for long read sequencing data and alignments</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/wdecoster/NanoPlot</td>
      <td>Visualization</td>
      <td>nanoplot</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/nanoplot/</td>
    </tr>
    <tr>
      <td></td>
      <td>nanopolishcomp</td>
      <td>0.6.11</td>
      <td>0.6.12</td>
      <td>nanopolishcomp</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>NanopolishComp contains 2 modules. Eventalign_collapse collapses the raw file generated by nanopolish eventalign by kmers rather than by event. Freq_meth_calculate methylation frequency at genomic CpG sites from the output of nanopolish call-methylation.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://a-slide.github.io/NanopolishComp</td>
      <td>Sequence Analysis</td>
      <td>nanopolishcomp</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/nanopolishcomp</td>
    </tr>
    <tr>
      <td></td>
      <td>ncbi_acc_download</td>
      <td>0.2.8</td>
      <td>0.2.8</td>
      <td>ncbi-acc-download</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Download sequences from GenBank/RefSeq by accession through the NCBI ENTREZ API</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/kblin/ncbi-acc-download</td>
      <td>Data Source</td>
      <td>ncbi_acc_download</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ncbi_acc_download</td>
    </tr>
    <tr>
      <td></td>
      <td>ncbi_datasets</td>
      <td>15.19.1</td>
      <td></td>
      <td>ncbi-datasets-cli</td>
      <td>To update</td>
      <td>ncbi_datasets</td>
      <td>NCBI Datasets</td>
      <td>Data handling, Sequence database search, Data retrieval</td>
      <td>Biological databases</td>
      <td>NCBI datasets downloads biological sequence data across all domains of life from NCBI.</td>
      <td>NCBI Datasets is a new resource that lets you easily gather data from across NCBI databases. Find and download sequence, annotation, and metadata for genes and genomes using our command-line tools or web interface.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ncbi/datasets</td>
      <td>Data Source</td>
      <td>ncbi_datasets</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ncbi_datasets</td>
    </tr>
    <tr>
      <td></td>
      <td>ncbi_entrez_direct</td>
      <td>16.2</td>
      <td>16.2</td>
      <td>entrez-direct</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>NCBI Entrez Direct allow fetching data from NCBI Databases</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://www.ncbi.nlm.nih.gov/books/NBK179288/</td>
      <td>Data Source</td>
      <td>ncbi_entrez_direct</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ncbi_entrez_direct</td>
    </tr>
    <tr>
      <td></td>
      <td>ncbi_entrez_eutils</td>
      <td>1.70</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>NCBI Entrez E-Utilties allow fetching data from NCBI Databases</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.ncbi.nlm.nih.gov/books/NBK25501/</td>
      <td>Data Source</td>
      <td>ncbi_entrez_eutils</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ncbi_entrez_eutils</td>
    </tr>
    <tr>
      <td></td>
      <td>ncbi_fcs_gx</td>
      <td>0.5.0</td>
      <td>0.5.0</td>
      <td>ncbi-fcs-gx</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FCS-GX detects contamination from foreign organisms in genome sequences using the genome cross-species aligner (GX).</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/ncbi/fcs-gx</td>
      <td>Sequence Analysis</td>
      <td>ncbi_fcs_gx</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ncbi_fcs_gx</td>
    </tr>
    <tr>
      <td></td>
      <td>necat</td>
      <td>0.0.1_update20200803</td>
      <td>0.0.1_update20200803</td>
      <td>necat</td>
      <td>Up-to-date</td>
      <td>necat</td>
      <td>NECAT</td>
      <td>De-novo assembly</td>
      <td>Sequence assembly</td>
      <td>Error correction and de-novo assembly for ONT Nanopore reads</td>
      <td>NECAT is an error correction and de-novo assembly tool for Nanopore long noisy reads.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/xiaochuanle/NECAT</td>
      <td>Assembly</td>
      <td>necat</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/necat</td>
    </tr>
    <tr>
      <td></td>
      <td>newick_utils</td>
      <td>1.6+galaxy1</td>
      <td>1.6</td>
      <td>newick_utils</td>
      <td>To update</td>
      <td>newick_utilities</td>
      <td>Newick Utilities</td>
      <td>Phylogenetic tree generation, Phylogenetic tree analysis, Phylogenetic tree reconstruction</td>
      <td>Phylogeny, Genomics, Computer science</td>
      <td>Perform operations on Newick trees</td>
      <td>The Newick Utilities are a set of command-line tools for processing phylogenetic trees. They can process arbitrarily large amounts of data and do not require user interaction, which makes them suitable for automating phylogeny processing tasks.</td>
      <td></td>
      <td>To update</td>
      <td>http://cegg.unige.ch/newick_utils</td>
      <td>Visualization, Metagenomics</td>
      <td>newick_utils</td>
      <td>iuc</td>
      <td>https://github.com/tjunier/newick_utils</td>
    </tr>
    <tr>
      <td></td>
      <td>nextclade</td>
      <td>2.7.0</td>
      <td>2.14.0</td>
      <td>nextalign</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Identify differences between your sequences and a reference sequence used by Nextstrain</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/nextstrain/nextclade</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/nextclade</td>
    </tr>
    <tr>
      <td></td>
      <td>ngmlr</td>
      <td>0.2.7</td>
      <td>0.2.7</td>
      <td>ngmlr</td>
      <td>Up-to-date</td>
      <td>ngmlr</td>
      <td>NGMLR</td>
      <td>DNA mapping, Sequence alignment, Genetic variation analysis</td>
      <td>Sequencing, Mapping, DNA structural variation</td>
      <td>CoNvex Gap-cost alignMents for Long Reads</td>
      <td>An algorithm to map third generation long-read sequencing data (PacBio and Oxford Nanopore) to a reference genome with a focus on reads that span structural variation.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/philres/ngmlr</td>
      <td>Next Gen Mappers</td>
      <td>ngmlr</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ngmlr</td>
    </tr>
    <tr>
      <td></td>
      <td>ngsutils</td>
      <td></td>
      <td>0.5.9</td>
      <td>ngsutils</td>
      <td>To update</td>
      <td>ngsutils</td>
      <td>NGSUtils</td>
      <td>Read pre-processing, Sequencing quality control, Variant calling, Formatting, Sequence contamination filtering</td>
      <td>Genomics, Transcriptomics</td>
      <td>NGSUtils is a suite of software tools for working with next-generation sequencing datasets.Starting in 2009, we (Liu Lab @ Indiana University School of Medicine) starting working withnext-generation sequencing data. We initially started doing custom coding for each project in a one-off manner.It quickly became apparent that this was an inefficient manner to work, so we started assembling smallerutilities that could be adapted into larger, more complicated, workflows. We have used them for Illumia,SOLiD, 454, Ion Torrent, and Pac Bio sequencing data. We have used them for DNA and RNA resequcing,ChIP-Seq, CLIP-Seq, and targeted resequencing (Agilent exome capture and PCR targeting).These tools are also used heavily in our in-house DNA and RNA mapping pipelines.NGSUtils is made up of 50+ programs, mainly written in Python.These are separated into modules based on the type of file that is to be analyzed.</td>
      <td>NGSUtils is a suite of software tools for working with next-generation sequencing datasets</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ngsutils/ngsutils</td>
      <td>SAM</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ngsutils</td>
    </tr>
    <tr>
      <td></td>
      <td>nonpareil</td>
      <td>3.1.1</td>
      <td>3.4.1</td>
      <td>nonpareil</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Estimate average coverage in metagenomic datasets</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://nonpareil.readthedocs.io</td>
      <td>Metagenomics</td>
      <td>nonpareil</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/nonpareil</td>
    </tr>
    <tr>
      <td></td>
      <td>novoplasty</td>
      <td>4.3.1</td>
      <td>4.3.5</td>
      <td>novoplasty</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>NOVOPlasty is a de novo assembler and heteroplasmy/variance caller for short circular genomes.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ndierckx/NOVOPlasty</td>
      <td>Assembly</td>
      <td>novoplasty</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/novoplasty</td>
    </tr>
    <tr>
      <td></td>
      <td>nugen_nudup</td>
      <td>2.3.3</td>
      <td>2.3.3</td>
      <td>nudup</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Marks/removes PCR introduced duplicate molecules based on the molecular tagging technology used in NuGEN products.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://nugentechnologies.github.io/nudup/</td>
      <td>SAM, Metagenomics, Sequence Analysis, Transcriptomics</td>
      <td>nugen_nudup</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/nugen_nudup</td>
    </tr>
    <tr>
      <td></td>
      <td>obitools</td>
      <td>1.2.13</td>
      <td>1.2.13</td>
      <td>obitools</td>
      <td>Up-to-date</td>
      <td>obitools</td>
      <td>OBITools</td>
      <td>Sequence analysis, Sequence analysis</td>
      <td>Sequence analysis, DNA, Sequencing</td>
      <td>OBITools is a set of programs developed to simplify the manipulation of sequence files</td>
      <td>Set of python programs developed to simplify the manipulation of sequence files. They were mainly designed to help us for analyzing Next Generation Sequencer outputs (454 or Illumina) in the context of DNA Metabarcoding.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://metabarcoding.org/obitools</td>
      <td>Sequence Analysis</td>
      <td>obitools</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/obitools</td>
    </tr>
    <tr>
      <td></td>
      <td>ococo</td>
      <td>0.1.2.6</td>
      <td>0.1.2.6</td>
      <td>ococo</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Variant detection of SNVs</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/karel-brinda/ococo</td>
      <td>Variant Analysis</td>
      <td>ococo</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ococo</td>
    </tr>
    <tr>
      <td></td>
      <td>odgi</td>
      <td>0.3</td>
      <td>0.8.4</td>
      <td>odgi</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Representing large genomic variation graphs with minimal memory overhead requires a careful encoding of the graph entities. odgi follows the dynamic GBWT in developing a byte-packed version of the graph and paths through it.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/vgteam/odgi</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/odgi/</td>
    </tr>
    <tr>
      <td></td>
      <td>omark</td>
      <td>0.3.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Proteome quality assessment software</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/DessimozLab/OMArk</td>
      <td>Sequence Analysis</td>
      <td>omark</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/omark/</td>
    </tr>
    <tr>
      <td></td>
      <td>ont_fast5_api</td>
      <td>3.1.3</td>
      <td>4.1.3</td>
      <td>ont-fast5-api</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ont_fast5_api is a simple interface to HDF5 files of the Oxford Nanopore FAST5 file format.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/nanoporetech/ont_fast5_api/</td>
      <td>Nanopore</td>
      <td>ont_fast5_api</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ont_fast5_api</td>
    </tr>
    <tr>
      <td></td>
      <td>onto_toolkit</td>
      <td>1.45</td>
      <td>1.45</td>
      <td>perl-onto-perl</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ONTO-Toolkit is a collection of tools for managing ontologies.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://search.cpan.org/~easr/ONTO-PERL-1.45/</td>
      <td>Ontology Manipulation</td>
      <td>onto_toolkit</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/onto-toolkit</td>
    </tr>
    <tr>
      <td></td>
      <td>optdoe</td>
      <td>v2.0.2</td>
      <td></td>
      <td>doebase</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Optimal Design Of Experiment</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/pablocarb/doebase</td>
      <td>Synthetic Biology</td>
      <td>optdoe</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/optdoe</td>
    </tr>
    <tr>
      <td></td>
      <td>optitype</td>
      <td>1.3.5</td>
      <td>1.3.5</td>
      <td>optitype</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Precision HLA typing from NGS data</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/FRED-2/OptiType</td>
      <td>Sequence Analysis</td>
      <td>optitype</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/optitype1</td>
    </tr>
    <tr>
      <td></td>
      <td>orfipy</td>
      <td>0.0.4</td>
      <td>0.0.4</td>
      <td>orfipy</td>
      <td>Up-to-date</td>
      <td>orfipy</td>
      <td>orfipy</td>
      <td>Coding region prediction, Database search, Transcriptome assembly, De-novo assembly</td>
      <td>Computer science, RNA-Seq, Transcriptomics, Small molecules</td>
      <td>Galaxy wrapper for ORFIPY</td>
      <td>A fast and flexible tool for extracting ORFs.orfipy is a tool written in python/cython to extract ORFs in extremely an fast and flexible manner. Other popular ORF searching tools are OrfM and getorf. Compared to OrfM and getorf, orfipy provides the most options to fine tune ORF searches. orfipy uses multiple CPU cores and is particularly faster for data containing multiple smaller fasta sequences such as de-novo transcriptome assemblies. Please read the preprint here.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/urmi-21/orfipy</td>
      <td>Sequence Analysis</td>
      <td>orfipy</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/orfipy</td>
    </tr>
    <tr>
      <td></td>
      <td>orthofinder</td>
      <td>2.5.5</td>
      <td>2.5.5</td>
      <td>orthofinder</td>
      <td>Up-to-date</td>
      <td>OrthoFinder</td>
      <td>OrthoFinder</td>
      <td>Genome comparison, Phylogenetic tree generation (from molecular sequences), Phylogenetic tree analysis, Genome alignment</td>
      <td>Phylogenetics, Phylogenomics, Bioinformatics, Comparative genomics, Sequence analysis</td>
      <td>Accurate inference of orthologous gene groups made easy</td>
      <td>OrthoFinder is a fast, accurate and comprehensive platform for comparative genomics. It finds orthogroups and orthologs, infers rooted gene trees for all orthogroups and identifies all of the gene duplcation events in those gene trees. It also infers a rooted species tree for the species being analysed and maps the gene duplication events from the gene trees to branches in the species tree. OrthoFinder also provides comprehensive statistics for comparative genomic analyses.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/davidemms/OrthoFinder</td>
      <td>Phylogenetics, Sequence Analysis</td>
      <td>orthofinder</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/orthofinder</td>
    </tr>
    <tr>
      <td></td>
      <td>packaged_annotation_loader</td>
      <td>0.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tool to make cached genome annotation data available as a list of datasets collection</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Data Source</td>
      <td>packaged_annotation_loader</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/packaged_annotation_loader</td>
    </tr>
    <tr>
      <td></td>
      <td>pangolin</td>
      <td>4.3</td>
      <td>4.3</td>
      <td>pangolin</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Pangolin assigns SARS-CoV-2 genome sequences their most likely lineages under the Pango nomenclature system.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/cov-lineages/pangolin</td>
      <td>Sequence Analysis</td>
      <td>pangolin</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pangolin</td>
    </tr>
    <tr>
      <td></td>
      <td>parse_mito_blast</td>
      <td>1.0.2</td>
      <td>1.0.2</td>
      <td>parse_mito_blast</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filtering blast out from querying assembly against mitochondrial database.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://raw.githubusercontent.com/VGP/vgp-assembly/master/galaxy_tools/parse_mito_blast/parse_mito_blast.py</td>
      <td>Sequence Analysis</td>
      <td>parse_mito_blast</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/parse_mito_blast</td>
    </tr>
    <tr>
      <td></td>
      <td>pathview</td>
      <td>1.34.0</td>
      <td>1.42.0</td>
      <td>bioconductor-pathview</td>
      <td>To update</td>
      <td>pathview</td>
      <td>pathview</td>
      <td>Pathway or network analysis, Pathway or network visualisation</td>
      <td>Molecular interactions, pathways and networks, Systems biology, Data visualisation</td>
      <td>Pathview is a tool set for pathway based data integration and visualization.</td>
      <td>Tool set for pathway based data integration and visualization that maps and renders a wide variety of biological data on relevant pathway graphs. It downloads the pathway graph data, parses the data file, maps user data to the pathway, and render pathway graph with the mapped data. In addition, it integrates with pathway and gene set (enrichment) analysis tools for large-scale and fully automated analysis.</td>
      <td></td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/pathview.html</td>
      <td>Statistics, RNA, Micro-array Analysis</td>
      <td>pathview</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pathview</td>
    </tr>
    <tr>
      <td></td>
      <td>pbgcpp</td>
      <td>2.0.2</td>
      <td>2.0.2</td>
      <td>pbgcpp</td>
      <td>Up-to-date</td>
      <td>genomicconsensus</td>
      <td>GenomicConsensus</td>
      <td>Variant calling</td>
      <td>Mapping</td>
      <td>Compute genomic consensus and call variants using PacBio reads mapped to a reference</td>
      <td>The GenomicConsensus package provides the variantCaller tool, which allows you to apply the Quiver or Arrow algorithm to mapped PacBio reads to get consensus and variant calls.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/PacificBiosciences/gcpp</td>
      <td>Variant Analysis</td>
      <td>pbgcpp</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pbgcpp</td>
    </tr>
    <tr>
      <td></td>
      <td>pbmm2</td>
      <td>1.13.1</td>
      <td>1.13.1</td>
      <td>pbmm2</td>
      <td>Up-to-date</td>
      <td>pbmm2</td>
      <td>pbmm2</td>
      <td>Pairwise sequence alignment, Sorting</td>
      <td>Mapping</td>
      <td>A minimap2 SMRT wrapper for PacBio data.</td>
      <td>pbmm2 is a SMRT C++ wrapper for minimap2's C API. Its purpose is to support native PacBio in- and output, provide sets of recommended parameters, generate sorted output on-the-fly, and postprocess alignments. Sorted output can be used directly for polishing using GenomicConsensus, if BAM has been used as input to pbmm2. Benchmarks show that pbmm2 outperforms BLASR in sequence identity, number of mapped bases, and especially runtime. pbmm2 is the official replacement for BLASR.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/PacificBiosciences/pbmm2</td>
      <td>Next Gen Mappers</td>
      <td>pbmm2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pbmm2</td>
    </tr>
    <tr>
      <td></td>
      <td>pe_histogram</td>
      <td>1.0.1</td>
      <td></td>
      <td>openjdk</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Contains a tool that produces an insert size histogram for a paired-end BAM file.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/seqcode/cegr-tools/tree/master/src/org/seqcode/cegrtools/pehistogram</td>
      <td>Graphics</td>
      <td>pe_histogram</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pe_histogram</td>
    </tr>
    <tr>
      <td></td>
      <td>peakzilla</td>
      <td>1.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Peakzilla identifies sites of enrichment and transcription factor binding sites from ChIP-seq and ChIP-exo experiments.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/steinmann/peakzilla</td>
      <td>ChIP-seq</td>
      <td>peakzilla</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/peakzilla</td>
    </tr>
    <tr>
      <td></td>
      <td>pear</td>
      <td>0.9.6</td>
      <td>0.9.6</td>
      <td>pear</td>
      <td>Up-to-date</td>
      <td>pear</td>
      <td>PEAR</td>
      <td>Sequence merging</td>
      <td>Sequence assembly</td>
      <td>PEAR evaluates all possible paired-end read overlaps</td>
      <td>Paired-end read merger. PEAR evaluates all possible paired-end read overlaps without requiring the target fragment size as input. In addition, it implements a statistical test for minimizing false-positive results.</td>
      <td></td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td>pear</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pear</td>
    </tr>
    <tr>
      <td></td>
      <td>pharokka</td>
      <td>1.3.2</td>
      <td></td>
      <td>\n                pharokka\n</td>
      <td>To update</td>
      <td>\n                pharokka\n</td>
      <td></td>
      <td></td>
      <td></td>
      <td>rapid standardised annotation tool for bacteriophage genomes and metagenomes</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/gbouras13/pharokka</td>
      <td>Genome annotation</td>
      <td>pharokka</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pharokka</td>
    </tr>
    <tr>
      <td></td>
      <td>phyloseq</td>
      <td>1.38.0</td>
      <td>1.46.0</td>
      <td>bioconductor-phyloseq</td>
      <td>To update</td>
      <td>phyloseq</td>
      <td>phyloseq</td>
      <td>Deposition, Analysis, Visualisation</td>
      <td>Microbiology, Sequence analysis, Metagenomics</td>
      <td>Handling and analysis of high-throughput microbiome census data</td>
      <td>Provides a set of classes and tools to facilitate the import, storage, analysis, and graphical display of microbiome census data.</td>
      <td></td>
      <td>To update</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/phyloseq.html</td>
      <td>Metagenomics</td>
      <td>phyloseq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/phyloseq</td>
    </tr>
    <tr>
      <td></td>
      <td>phyml</td>
      <td>3.3.20220408</td>
      <td>3.3.20220408</td>
      <td>phyml</td>
      <td>Up-to-date</td>
      <td>phyml</td>
      <td>PhyML</td>
      <td>Phylogenetic tree generation (maximum likelihood and Bayesian methods)</td>
      <td>Phylogenetics, Bioinformatics, Phylogenetics</td>
      <td>PhyML is a phylogeny software based on the maximum-likelihood principle.</td>
      <td>Phylogenetic estimation software using Maximum Likelihood</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://www.atgc-montpellier.fr/phyml/</td>
      <td>Phylogenetics</td>
      <td>phyml</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/phyml</td>
    </tr>
    <tr>
      <td></td>
      <td>picard</td>
      <td>3.1.1</td>
      <td>3.1.1</td>
      <td>picard</td>
      <td>Up-to-date</td>
      <td>picard_fastqtosam</td>
      <td>picard_fastqtosam</td>
      <td>Formatting</td>
      <td>Sequencing</td>
      <td>Picard SAM/BAM manipulation tools.</td>
      <td>Create an unaligned BAM file.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://broadinstitute.github.io/picard/</td>
      <td>SAM</td>
      <td>picard</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/picard</td>
    </tr>
    <tr>
      <td></td>
      <td>pick_value</td>
      <td>0.2.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compose a text parameter value using text, integer and float values</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>pick_value</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pick_value</td>
    </tr>
    <tr>
      <td></td>
      <td>picrust</td>
      <td>1.1.1</td>
      <td>1.1.4</td>
      <td>picrust</td>
      <td>To update</td>
      <td>picrust</td>
      <td>PICRUSt</td>
      <td>Phylogenetic reconstruction, Expression analysis, Genome annotation, DNA barcoding</td>
      <td>Metagenomics, Microbial ecology, Functional, regulatory and non-coding RNA, Metagenomic sequencing</td>
      <td>PICRUSt wrappers</td>
      <td>PICRUSt (Phylogenetic Investigation of Communities by Reconstruction of Unobserved States) is a bioinformatics software package designed to predict metagenome functional content from marker gene (e.g., 16S rRNA) surveys and full genomes.</td>
      <td></td>
      <td>To update</td>
      <td>https://picrust.github.io/picrust/</td>
      <td>Metagenomics</td>
      <td>picrust</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/picrust</td>
    </tr>
    <tr>
      <td></td>
      <td>picrust2</td>
      <td>2.5.1</td>
      <td>2.5.2</td>
      <td>picrust2</td>
      <td>To update</td>
      <td>picrust2</td>
      <td>PICRUSt2</td>
      <td>Phylogenetic reconstruction, Expression analysis, Rarefaction, Pathway analysis</td>
      <td>Metagenomics, Microbiology, Phylogenetics, Metagenomic sequencing</td>
      <td>PICRUSt2: Phylogenetic Investigation of Communities by Reconstruction of Unobserved States</td>
      <td>PICRUSt2 (Phylogenetic Investigation of Communities by Reconstruction of Unobserved States) is a software for predicting functional abundances based only on marker gene sequences.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/picrust/picrust2/wiki</td>
      <td>Metagenomics</td>
      <td>picrust2</td>
      <td>iuc</td>
      <td>https://github.com/picrust/picrust2</td>
    </tr>
    <tr>
      <td></td>
      <td>pilon</td>
      <td>1.20.1</td>
      <td>1.24</td>
      <td>pilon</td>
      <td>To update</td>
      <td>pilon</td>
      <td>pilon</td>
      <td>Sequence assembly, Analysis, Read alignment</td>
      <td>Assembly</td>
      <td>pilon is a tool for assembly improvement and variant analysis in bacteria</td>
      <td>Read alignment analysis to diagnose, report, and automatically improve de novo genome assemblies.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Variant Analysis</td>
      <td>pilon</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pilon</td>
    </tr>
    <tr>
      <td></td>
      <td>pipelign</td>
      <td>0.2</td>
      <td>0.2</td>
      <td>pipelign</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Multipe sequence alignment</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/asmmhossain/pipelign/</td>
      <td>Next Gen Mappers</td>
      <td>pipelign</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pipelign</td>
    </tr>
    <tr>
      <td></td>
      <td>pizzly</td>
      <td>0.37.3.1</td>
      <td>0.37.3</td>
      <td>pizzly</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Pizzly is a program for detecting gene fusions from RNA-Seq data of cancer samples.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/pmelsted/pizzly/</td>
      <td>Transcriptomics</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pizzly/</td>
    </tr>
    <tr>
      <td></td>
      <td>plasflow</td>
      <td>1.1.0</td>
      <td>1.1.0</td>
      <td>plasflow</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>PlasFlow - Prediction of plasmid sequences in metagenomic contigs.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/smaegol/PlasFlow</td>
      <td>Sequence Analysis</td>
      <td>plasflow</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/plasflow</td>
    </tr>
    <tr>
      <td></td>
      <td>plasmidfinder</td>
      <td>2.1.6</td>
      <td>2.1.6</td>
      <td>plasmidfinder</td>
      <td>Up-to-date</td>
      <td>PlasmidFinder</td>
      <td>PlasmidFinder</td>
      <td>Genome assembly, Scaffolding, Multilocus sequence typing</td>
      <td>Whole genome sequencing, Sequence assembly, Mapping, Probes and primers</td>
      <td>"PlasmidFinder provides the detection of replicons in the WGSand assigns the plasmids under study to lineages that trace backthe information to the existing knowledge on Inc groups and suggestspossible reference plasmids for each lineage"</td>
      <td>PlasmidFinder is a tool for the identification and typing of Plasmid Replicons in Whole-Genome Sequencing (WGS).</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://bitbucket.org/genomicepidemiology/plasmidfinder/src/master/</td>
      <td>Sequence Analysis</td>
      <td>plasmidfinder</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/plasmidfinder</td>
    </tr>
    <tr>
      <td></td>
      <td>plink</td>
      <td>1.90b6.21</td>
      <td>1.90b6.21</td>
      <td>plink</td>
      <td>Up-to-date</td>
      <td>plink</td>
      <td>PLINK</td>
      <td>Genetic variation analysis</td>
      <td>GWAS study</td>
      <td>PLINK is a free, open-source whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner.</td>
      <td>Free, open-source whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://www.cog-genomics.org/plink</td>
      <td>Genome-Wide Association Study</td>
      <td>plink</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/plink</td>
    </tr>
    <tr>
      <td></td>
      <td>polypolish</td>
      <td>0.5.0</td>
      <td>0.6.0</td>
      <td>polypolish</td>
      <td>To update</td>
      <td>Polypolish</td>
      <td>Polypolish</td>
      <td>Genome assembly, Read mapping, Mapping assembly, Sequencing error detection</td>
      <td>Sequence assembly, Sequence composition, complexity and repeats, Mapping</td>
      <td>"Polypolish is a tool for polishing genome assemblies with short reads.Polypolish uses SAM files where each read has been aligned to all possible locations (not just a single best location).This allows it to repair errors in repeat regions that other alignment-based polishers cannot fix."</td>
      <td>Polypolish is a tool for polishing genome assemblies with short reads. Unlike other tools in this category, Polypolish uses SAM files where each read has been aligned to all possible locations (not just a single best location). This allows it to repair errors in repeat regions that other alignment-based polishers cannot fix.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/rrwick/Polypolish</td>
      <td>Sequence Analysis</td>
      <td>polypolish</td>
      <td>iuc</td>
      <td>https://github.com/mesocentre-clermont-auvergne/galaxy-tools/tree/master/tools/polypolish</td>
    </tr>
    <tr>
      <td></td>
      <td>porechop</td>
      <td></td>
      <td>0.2.4</td>
      <td>porechop</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Porechop - Finding and removing adapters from Oxford Nanopore reads</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/rrwick/Porechop</td>
      <td>Fasta Manipulation, Fastq Manipulation</td>
      <td>porechop</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/porechop</td>
    </tr>
    <tr>
      <td></td>
      <td>poretools</td>
      <td>0.6.1a1</td>
      <td>0.6.1a1</td>
      <td>poretools</td>
      <td>Up-to-date</td>
      <td>poretools</td>
      <td>Poretools</td>
      <td>Nucleic acid sequence analysis</td>
      <td>DNA, Sequencing</td>
      <td>A flexible toolkit for exploring datasets generated by nanopore sequencing devices from MinION for the purposes of quality control and downstream analysis.</td>
      <td>Flexible toolkit for exploring datasets generated by nanopore sequencing devices from MinION for the purposes of quality control and downstream analysis.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://poretools.readthedocs.io/en/latest/</td>
      <td>Fasta Manipulation, Fastq Manipulation</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/poretools</td>
    </tr>
    <tr>
      <td></td>
      <td>presto</td>
      <td>0.6.2</td>
      <td>0.7.2</td>
      <td>presto</td>
      <td>To update</td>
      <td>presto</td>
      <td>pRESTO</td>
      <td>Nucleic acid sequence analysis</td>
      <td>Sequencing, DNA, Immunology</td>
      <td>pRESTO toolkit for immune repertoire analysis.</td>
      <td>Integrated collection of platform-independent Python modules for processing raw reads from high-throughput (next-generation) sequencing of lymphocyte repertoires.</td>
      <td></td>
      <td>To update</td>
      <td>https://presto.readthedocs.io/</td>
      <td>Sequence Analysis</td>
      <td>presto</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/presto</td>
    </tr>
    <tr>
      <td></td>
      <td>pretext</td>
      <td>0.0.6</td>
      <td>0.0.6</td>
      <td>pretextgraph</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Process genome contacts maps processing images.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/wtsi-hpag/PretextSnapshot</td>
      <td>Sequence Analysis</td>
      <td>suite_pretext</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pretext</td>
    </tr>
    <tr>
      <td></td>
      <td>prinseq</td>
      <td>@TOOL_VERSION+galaxy2</td>
      <td>0.20.4</td>
      <td>prinseq</td>
      <td>To update</td>
      <td>prinseq</td>
      <td>PRINSEQ</td>
      <td>Read pre-processing, Sequence trimming, Sequence contamination filtering</td>
      <td>Transcriptomics, Metagenomics, Genomics</td>
      <td>PRINSEQ is a tool for easy and rapid quality control and data processing of metagenomic and metatranscriptomic datasets</td>
      <td>PRINSEQ is a sequence processing tool that can be used to filter, reformat and trim genomic and metagenomic sequence data. It generates summary statistics of the input in graphical and tabular formats that can be used for quality control steps. PRINSEQ is available as both standalone and web-based versions.</td>
      <td></td>
      <td>To update</td>
      <td>http://prinseq.sourceforge.net/manual.html</td>
      <td>Fastq Manipulation, Metagenomics</td>
      <td>prinseq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/prinseq/</td>
    </tr>
    <tr>
      <td></td>
      <td>progressivemauve</td>
      <td></td>
      <td>snapshot_2015_02_13</td>
      <td>progressivemauve</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Mauve/ProgressiveMauve Multiple Sequence Aligner</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>progressivemauve</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/progressivemauve</td>
    </tr>
    <tr>
      <td></td>
      <td>prokka</td>
      <td>1.14.6</td>
      <td>1.14.6</td>
      <td>prokka</td>
      <td>Up-to-date</td>
      <td>prokka</td>
      <td>Prokka</td>
      <td>Gene prediction, Coding region prediction, Genome annotation</td>
      <td>Genomics, Model organisms, Virology</td>
      <td>Rapid annotation of prokaryotic genomes</td>
      <td>Software tool to annotate bacterial, archaeal and viral genomes quickly and produce standards-compliant output files.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://github.com/tseemann/prokka</td>
      <td>Sequence Analysis</td>
      <td>prokka</td>
      <td>crs4</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/prokka/</td>
    </tr>
    <tr>
      <td></td>
      <td>prot-scriber</td>
      <td>0.1.4</td>
      <td>0.1.5</td>
      <td>prot-scriber</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Protein annotation of short human readable descriptions</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/usadellab/prot-scriber</td>
      <td>Proteomics</td>
      <td>prot_scriber</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/prot-scriber</td>
    </tr>
    <tr>
      <td></td>
      <td>proteinortho</td>
      <td>6.3.1</td>
      <td>6.3.1</td>
      <td>proteinortho</td>
      <td>Up-to-date</td>
      <td>proteinortho</td>
      <td>Proteinortho</td>
      <td>Homology-based gene prediction</td>
      <td>Phylogeny</td>
      <td>Proteinortho is a tool to detect orthologous proteins/genes within different species.</td>
      <td>Proteinortho is a tool to detect orthologous genes within different species</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://gitlab.com/paulklemm_PHD/proteinortho</td>
      <td>Proteomics</td>
      <td>proteinortho</td>
      <td>iuc</td>
      <td>https://gitlab.com/paulklemm_PHD/proteinortho</td>
    </tr>
    <tr>
      <td></td>
      <td>psiclass</td>
      <td>1.0.3</td>
      <td>1.0.3</td>
      <td>psiclass</td>
      <td>Up-to-date</td>
      <td>psiclass</td>
      <td></td>
      <td></td>
      <td></td>
      <td>PsiCLASS is a reference-based transcriptome assembler for single or multiple RNA-seq samples.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/splicebox/PsiCLASS</td>
      <td>Transcriptomics</td>
      <td>psiclass</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/psiclass</td>
    </tr>
    <tr>
      <td></td>
      <td>pureclip</td>
      <td>1.0.4</td>
      <td>1.3.1</td>
      <td>pureclip</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>PureCLIP is an HMM based peak caller specifically designed for eCLIP/iCLIP data</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/skrakau/PureCLIP</td>
      <td>Sequence Analysis, RNA, CLIP-seq</td>
      <td>pureclip</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pureclip</td>
    </tr>
    <tr>
      <td></td>
      <td>purge_dups</td>
      <td>1.2.6</td>
      <td>1.2.6</td>
      <td>purge_dups</td>
      <td>Up-to-date</td>
      <td>purge_dups</td>
      <td>purge_dups</td>
      <td>Genome assembly, Read binning, Scaffolding</td>
      <td>Sequence assembly</td>
      <td>Purge haplotigs and overlaps in an assembly based on read depth</td>
      <td>Identifying and removing haplotypic duplication in primary genome assemblies | haplotypic duplication identification tool | scripts/pd_config.py: script to generate a configuration file used by run_purge_dups.py | purge haplotigs and overlaps in an assembly based on read depth | Given a primary assembly pri_asm and an alternative assembly hap_asm (optional, if you have one), follow the steps shown below to build your own purge_dups pipeline, steps with same number can be run simultaneously. Among all the steps, although step 4 is optional, we highly recommend our users to do so, because assemblers may produce overrepresented seqeuences. In such a case, The final step 4 can be applied to remove those seqeuences</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/dfguan/purge_dups</td>
      <td>Assembly</td>
      <td>purge_dups</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/purge_dups</td>
    </tr>
    <tr>
      <td></td>
      <td>pycoqc</td>
      <td>2.5.2</td>
      <td>2.5.2</td>
      <td>pycoqc</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>QC metrics for ONT Basecalling</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/tleonardi/pycoQC</td>
      <td>Nanopore</td>
      <td>pycoqc</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pycoqc</td>
    </tr>
    <tr>
      <td></td>
      <td>pyega3</td>
      <td>5.0.2</td>
      <td>5.1.0</td>
      <td>pyega3</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>EGA python client uses the EGA REST API to download authorized datasets and files.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/EGA-archive/ega-download-client</td>
      <td>Data Source</td>
      <td>ega_download_client</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pyega3</td>
    </tr>
    <tr>
      <td></td>
      <td>pygenometracks</td>
      <td>3.8</td>
      <td>3.8</td>
      <td>pygenometracks</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>pyGenomeTracks: Standalone program and library to plot beautiful genome browser tracks.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/deeptools/pyGenomeTracks</td>
      <td>Visualization</td>
      <td>pygenometracks</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pygenometracks</td>
    </tr>
    <tr>
      <td></td>
      <td>pysradb</td>
      <td>1.4.2</td>
      <td>2.2.0</td>
      <td>pysradb</td>
      <td>To update</td>
      <td>pysradb</td>
      <td>pysradb</td>
      <td>Deposition, Data retrieval</td>
      <td>Sequencing, Gene transcripts, Bioinformatics</td>
      <td>pysradb allows to retrieve metadata, such as run accession numbers, from SRA and ENA based on multiple criteria.</td>
      <td>Python package to query next-generation sequencing metadata and data from NCBI Sequence Read Archive.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/saketkc/pysradb</td>
      <td>Sequence Analysis</td>
      <td>pysradb_search</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pysradb</td>
    </tr>
    <tr>
      <td></td>
      <td>qfilt</td>
      <td>1.0.0+galaxy1</td>
      <td>0.0.1</td>
      <td>qfilt</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter sequencing data</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/veg/qfilt</td>
      <td>Fastq Manipulation</td>
      <td>qfilt</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/qfilt</td>
    </tr>
    <tr>
      <td></td>
      <td>qiime_add_on</td>
      <td></td>
      <td>1.9.1</td>
      <td>qiime</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>QIIME to perform microbial community analysis</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.qiime.org</td>
      <td>Metagenomics</td>
      <td>qiime</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/qiime/</td>
    </tr>
    <tr>
      <td></td>
      <td>qiime_core</td>
      <td></td>
      <td>1.9.1</td>
      <td>qiime</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>QIIME to perform microbial community analysis</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.qiime.org</td>
      <td>Metagenomics</td>
      <td>qiime</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/qiime/</td>
    </tr>
    <tr>
      <td></td>
      <td>qq_tools</td>
      <td>0.1.0</td>
      <td>0.1.4</td>
      <td>r-qqman</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://CRAN.R-project.org/package=qqman</td>
      <td>Visualization, Variant Analysis</td>
      <td></td>
      <td>iuc</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>qualimap</td>
      <td>2.2.2d</td>
      <td>2.3</td>
      <td>qualimap</td>
      <td>To update</td>
      <td>qualimap</td>
      <td>QualiMap</td>
      <td>Sequencing quality control</td>
      <td>Data quality management</td>
      <td></td>
      <td>Platform-independent application written in Java and R that provides both a Graphical User Inteface (GUI) and a command-line interface to facilitate the quality control of alignment sequencing data.</td>
      <td></td>
      <td>To update</td>
      <td>http://qualimap.bioinfo.cipf.es/</td>
      <td>Sequence Analysis, Transcriptomics, SAM</td>
      <td>qualimap</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/qualimap</td>
    </tr>
    <tr>
      <td></td>
      <td>quast</td>
      <td>5.2.0</td>
      <td>5.2.0</td>
      <td>quast</td>
      <td>Up-to-date</td>
      <td>quast</td>
      <td>QUAST</td>
      <td>Visualisation, Sequence assembly validation</td>
      <td>Sequence assembly</td>
      <td>Quast (Quality ASsessment Tool) evaluates genome assemblies.</td>
      <td>QUAST stands for QUality ASsessment Tool.  It evaluates a quality of genome assemblies by computing various metrics and providing nice reports.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://quast.bioinf.spbau.ru/</td>
      <td>Assembly</td>
      <td>quast</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/quast</td>
    </tr>
    <tr>
      <td></td>
      <td>query_impc</td>
      <td>0.9.0</td>
      <td></td>
      <td>requests</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Contains a tool to query the IMPC database.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/INFRAFRONTIERDIB/tools-iuc/tree/query_impc/tools/query_impc</td>
      <td>Convert Formats, Web Services</td>
      <td>query_impc</td>
      <td>iuc</td>
      <td>https://github.com/INFRAFRONTIERDIB/tools-iuc/tree/query_impc/tools/query_impc</td>
    </tr>
    <tr>
      <td></td>
      <td>query_tabular</td>
      <td>3.3.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Loads tabular files into a SQLite DB to perform a SQL query producing a tabular output</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/query_tabular</td>
    </tr>
    <tr>
      <td></td>
      <td>quickmerge</td>
      <td>0.3</td>
      <td>0.3</td>
      <td>quickmerge</td>
      <td>Up-to-date</td>
      <td>quickmerge</td>
      <td>quickmerge</td>
      <td>Genome assembly, Scaffolding, De-novo assembly, Genotyping</td>
      <td>Structural variation, Sequence assembly, DNA polymorphism, Whole genome sequencing, Genotype and phenotype</td>
      <td>Merge long-read and hybrid assemblies to increase contiguity</td>
      <td>Quickmerge is a program that uses complementary information from genomes assembled with long reads in order to improve contiguity, and works with assemblies derived from both Pacific Biosciences or Oxford Nanopore. Quickmerge will even work with hybrid assemblies made by combining long reads and Illumina short reads.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/mahulchak/quickmerge</td>
      <td>Assembly</td>
      <td>quickmerge</td>
      <td>galaxy-australia</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/quickmerge</td>
    </tr>
    <tr>
      <td></td>
      <td>raceid</td>
      <td>0.2.3</td>
      <td>0.1.3</td>
      <td>r-raceid</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>RaceID3, StemID2, FateID - scRNA analysis</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/dgrun/RaceID3_StemID2_package/</td>
      <td>Transcriptomics</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/raceid3</td>
    </tr>
    <tr>
      <td></td>
      <td>ragtag</td>
      <td>2.1.0</td>
      <td>2.1.0</td>
      <td>ragtag</td>
      <td>Up-to-date</td>
      <td>ragtag</td>
      <td>ragtag</td>
      <td>Genome assembly</td>
      <td>Sequence assembly</td>
      <td>Reference-guided scaffolding of draft genomes tool.</td>
      <td>RagTag is a collection of software tools for scaffolding and improving modern genome assemblies.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/malonge/RagTag</td>
      <td>Assembly</td>
      <td>ragtag</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ragtag</td>
    </tr>
    <tr>
      <td></td>
      <td>rapidnj</td>
      <td>2.3.2</td>
      <td>2.3.2</td>
      <td>rapidnj</td>
      <td>Up-to-date</td>
      <td>rapidnj</td>
      <td>RapidNJ</td>
      <td>Phylogenetic tree generation</td>
      <td>Phylogeny</td>
      <td>Galaxy wrapper for the RapidNJ tool</td>
      <td>A tool for fast canonical neighbor-joining tree construction.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://birc.au.dk/software/rapidnj/</td>
      <td>Phylogenetics</td>
      <td>rapidnj</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/rapidnj</td>
    </tr>
    <tr>
      <td></td>
      <td>rasusa</td>
      <td>0.8.0</td>
      <td>0.8.0</td>
      <td>rasusa</td>
      <td>Up-to-date</td>
      <td>rasusa</td>
      <td>rasusa</td>
      <td></td>
      <td></td>
      <td>Randomly subsample sequencing reads to a specified coverage</td>
      <td>Produces an unbiased subsample of your reads</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/mbhall88/rasusa</td>
      <td>Sequence Analysis</td>
      <td>rasusa</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/rasusa</td>
    </tr>
    <tr>
      <td></td>
      <td>raven</td>
      <td>1.8.0</td>
      <td>1.8.3</td>
      <td>raven-assembler</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Raven is a de novo genome assembler for long uncorrected reads.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/lbcb-sci/raven</td>
      <td>Assembly</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/raven</td>
    </tr>
    <tr>
      <td></td>
      <td>raxml</td>
      <td>8.2.12</td>
      <td>8.2.13</td>
      <td>raxml</td>
      <td>To update</td>
      <td>raxml</td>
      <td>RAxML</td>
      <td>Sequence analysis, Phylogenetic tree analysis</td>
      <td>Phylogenetics, Sequence analysis</td>
      <td>RAxML - A Maximum Likelihood based phylogenetic inference</td>
      <td>A tool for Phylogenetic Analysis and Post-Analysis of Large Phylogenies.</td>
      <td></td>
      <td>To update</td>
      <td>http://www.exelixis-lab.org/web/software/raxml/</td>
      <td>Phylogenetics</td>
      <td>raxml</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/raxml</td>
    </tr>
    <tr>
      <td></td>
      <td>rcorrector</td>
      <td>1.0.3+galaxy1</td>
      <td>1.0.6</td>
      <td>rcorrector</td>
      <td>To update</td>
      <td>rcorrector</td>
      <td>Rcorrector</td>
      <td>Sequencing error detection</td>
      <td>RNA, RNA-Seq, Sequencing</td>
      <td>Rcorrector (RNA-seq error CORRECTOR) is a kmer-based error correction method for RNA-seq data.</td>
      <td>This is a kmer-based error correction method for RNA-seq data. It can also be applied to other types of sequencing data where the read coverage is non-uniform, such as single-cell sequencing.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/mourisl/Rcorrector</td>
      <td>Fastq Manipulation</td>
      <td>rcorrector</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/rcorrector</td>
    </tr>
    <tr>
      <td></td>
      <td>read_it_and_keep</td>
      <td>0.2.2</td>
      <td>0.3.0</td>
      <td>read-it-and-keep</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Rapid decontamination of SARS-CoV-2 sequencing reads</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/GenomePathogenAnalysisService/read-it-and-keep</td>
      <td>Sequence Analysis</td>
      <td>read_it_and_keep</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/read-it-and-keep</td>
    </tr>
    <tr>
      <td></td>
      <td>recentrifuge</td>
      <td>1.14.0</td>
      <td>1.14.0</td>
      <td>recentrifuge</td>
      <td>Up-to-date</td>
      <td>Recentrifuge</td>
      <td>Recentrifuge</td>
      <td>Taxonomic classification, Expression analysis, Cross-assembly</td>
      <td>Metagenomics, Microbial ecology, Metagenomic sequencing</td>
      <td>"With Recentrifuge, researchers can analyze results from taxonomic classifiers using interactive charts with emphasis on the confidence level of the classifications.In addition to contamination-subtracted samples.Recentrifuge provides shared and exclusive taxa per sample,thus enabling robust contamination removal and comparative analysis in environmental and clinical metagenomics."</td>
      <td>Robust comparative analysis and contamination removal for metagenomics.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/khyox/recentrifuge</td>
      <td>Metagenomics</td>
      <td>recentrifuge</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/recentrifuge</td>
    </tr>
    <tr>
      <td></td>
      <td>red</td>
      <td>2018.09.10</td>
      <td>2018.09.10</td>
      <td>red</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Red (REpeat Detector)</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/BioinformaticsToolsmith/Red</td>
      <td>Sequence Analysis</td>
      <td>red</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/red</td>
    </tr>
    <tr>
      <td></td>
      <td>repeatmasker</td>
      <td>4.1.5</td>
      <td>4.1.5</td>
      <td>repeatmasker</td>
      <td>Up-to-date</td>
      <td>repeatmasker</td>
      <td>RepeatMasker</td>
      <td>Genome annotation</td>
      <td>Sequence analysis, Sequence composition, complexity and repeats</td>
      <td>RepeatMasker is a program that screens DNA sequences for interspersed repeats and low complexity DNA sequences.</td>
      <td>A program that screens DNA sequences for interspersed repeats and low complexity DNA sequences. The output of the program is a detailed annotation of the repeats that are present in the query sequence as well as a modified version of the query sequence in which all the annotated repeats have been masked (default: replaced by Ns).</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://www.repeatmasker.org/</td>
      <td>Sequence Analysis</td>
      <td>repeat_masker</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/repeatmasker</td>
    </tr>
    <tr>
      <td></td>
      <td>repeatmodeler</td>
      <td>2.0.5</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>RepeatModeler - Model repetitive DNA</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.repeatmasker.org/RepeatModeler/</td>
      <td>Genome annotation</td>
      <td>repeatmodeler</td>
      <td>csbl</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/repeatmodeler</td>
    </tr>
    <tr>
      <td></td>
      <td>repmatch_gff3</td>
      <td></td>
      <td></td>
      <td>matplotlib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Contains a tool that matches corresponding peak-pair midpoints from separate datasets based onuser-defined criteria.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>ChIP-seq</td>
      <td>repmatch_gff3</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/repmatch_gff3</td>
    </tr>
    <tr>
      <td></td>
      <td>reshape2</td>
      <td>1.4.2</td>
      <td></td>
      <td>r-reshape2</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Flexibly restructure and aggregate data using just the two functions melt and dcast</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://cran.r-project.org/web/packages/reshape2/index.html</td>
      <td>Text Manipulation</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/reshape2</td>
    </tr>
    <tr>
      <td></td>
      <td>resize_coordinate_window</td>
      <td>1.0.2</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Contains a tool that modifies the start and stop coordinates of GFF data, expanding the coordinate windowby a specified size.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Genomic Interval Operations</td>
      <td>resize_coordinate_window</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/resize_coordinate_window</td>
    </tr>
    <tr>
      <td></td>
      <td>rgcca</td>
      <td>3.0.2</td>
      <td>3.0.3</td>
      <td>rgccacmd</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>multi-block analysis</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/rgcca-factory/RGCCA</td>
      <td>Statistics</td>
      <td>rgcca</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/rgcca</td>
    </tr>
    <tr>
      <td></td>
      <td>rgrnastar</td>
      <td>2.7.11a</td>
      <td>2.7.11b</td>
      <td>star</td>
      <td>To update</td>
      <td>star</td>
      <td>STAR</td>
      <td>Sequence alignment</td>
      <td>RNA-Seq, Transcriptomics</td>
      <td>RNA STAR is an ultra fast universal RNA and scRNA-seq aligner and mapper</td>
      <td>Ultrafast universal RNA-seq data aligner</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/alexdobin/STAR</td>
      <td>Next Gen Mappers, Transcriptomics</td>
      <td>rgrnastar</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/rgrnastar</td>
    </tr>
    <tr>
      <td></td>
      <td>ribowaltz</td>
      <td>1.2.0</td>
      <td>2.0</td>
      <td>ribowaltz</td>
      <td>To update</td>
      <td>riboWaltz</td>
      <td>riboWaltz</td>
      <td></td>
      <td>Computational biology</td>
      <td>Calculation of optimal P-site offsets, diagnostic analysis and visual inspection of ribosome profiling data</td>
      <td>riboWaltz is an R package for calculation of optimal P-site offsets, diagnostic analysis and visual inspection of ribosome profiling data.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/LabTranslationalArchitectomics/riboWaltz</td>
      <td>Transcriptomics, RNA</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ribowaltz</td>
    </tr>
    <tr>
      <td></td>
      <td>rnaquast</td>
      <td>2.2.3</td>
      <td>2.2.3</td>
      <td>rnaquast</td>
      <td>Up-to-date</td>
      <td>rnaQUAST</td>
      <td>rnaQUAST</td>
      <td>De-novo assembly, Transcriptome assembly, Sequence assembly validation</td>
      <td>Sequence assembly, Transcriptomics, RNA-seq</td>
      <td>rnaQuast (RNA Quality Assessment Tool) evaluates genome assemblies.</td>
      <td>Quality assessment tool for de novo transcriptome assemblies.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/ablab/rnaquast</td>
      <td>Assembly, RNA</td>
      <td>rnaquast</td>
      <td>iuc</td>
      <td>https://git.ufz.de/lehmanju/rnaquast</td>
    </tr>
    <tr>
      <td></td>
      <td>roary</td>
      <td>3.13.0</td>
      <td>3.13.0</td>
      <td>roary</td>
      <td>Up-to-date</td>
      <td>roary</td>
      <td>Roary</td>
      <td>Genome assembly</td>
      <td>DNA, Genomics, Mapping</td>
      <td>Roary the pangenome pipeline</td>
      <td>A high speed stand alone pan genome pipeline, which takes annotated assemblies in GFF3 format (produced by Prokka (Seemann, 2014)) and calculates the pan genome.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://sanger-pathogens.github.io/Roary/</td>
      <td>Sequence Analysis</td>
      <td>roary</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/roary</td>
    </tr>
    <tr>
      <td></td>
      <td>rp2biosensor</td>
      <td>3.2.1</td>
      <td></td>
      <td>rp2biosensor</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Build Sensing-Enabling Metabolic Pathways from RetroPath2.0 output</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/brsynth/rp2biosensor</td>
      <td>Synthetic Biology</td>
      <td>rp2biosensor</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/rp2biosensor</td>
    </tr>
    <tr>
      <td></td>
      <td>rp2paths</td>
      <td>1.5.1</td>
      <td></td>
      <td>rp2paths</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Enumerate and seperate the different pathways generated by RetroPath2.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/brsynth/rp2paths</td>
      <td>Synthetic Biology</td>
      <td>rp2paths</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/rp2paths</td>
    </tr>
    <tr>
      <td></td>
      <td>rpbasicdesign</td>
      <td>1.2.2</td>
      <td></td>
      <td>rpbasicdesign</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Extracting enzyme IDs from rpSBML files</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/brsynth/rpbasicdesign</td>
      <td>Synthetic Biology</td>
      <td>rpbasicdesign</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/rpbasicdesign</td>
    </tr>
    <tr>
      <td></td>
      <td>rpfba</td>
      <td>5.12.3</td>
      <td></td>
      <td>rptools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Perform FBA for the RetroPath2.0 heterologous pathways</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/brsynth/rptools/releases</td>
      <td>Synthetic Biology</td>
      <td>rpfba</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/rpfba</td>
    </tr>
    <tr>
      <td></td>
      <td>rptools</td>
      <td>5.13.1</td>
      <td></td>
      <td>rptools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Suite of tools that work on rpSBML format</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/brsynth/rptools</td>
      <td>Synthetic Biology</td>
      <td>rptools</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/rptools</td>
    </tr>
    <tr>
      <td></td>
      <td>rrparser</td>
      <td>2.5.2</td>
      <td></td>
      <td>rrparser</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Reaction Rules Parser</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/brsynth/RRParser</td>
      <td>Synthetic Biology</td>
      <td>rrparser</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/rrparser</td>
    </tr>
    <tr>
      <td></td>
      <td>rseqc</td>
      <td>5.0.1</td>
      <td>5.0.3</td>
      <td>rseqc</td>
      <td>To update</td>
      <td>rseqc</td>
      <td>RSeQC</td>
      <td>Data handling</td>
      <td>Sequencing</td>
      <td>an RNA-seq quality control package</td>
      <td>Provides a number of useful modules that can comprehensively evaluate high throughput sequence data especially RNA-seq data. Some basic modules quickly inspect sequence quality, nucleotide composition bias, PCR bias and GC bias, while RNA-seq specific modules evaluate sequencing saturation, mapped reads distribution, coverage uniformity, strand specificity, transcript level RNA integrity etc.</td>
      <td></td>
      <td>To update</td>
      <td>https://code.google.com/p/rseqc/</td>
      <td>Convert Formats, Sequence Analysis, RNA, Transcriptomics, Visualization</td>
      <td>rseqc</td>
      <td>nilesh</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/rseqc</td>
    </tr>
    <tr>
      <td></td>
      <td>ruvseq</td>
      <td>1.26.0</td>
      <td>1.36.0</td>
      <td>bioconductor-ruvseq</td>
      <td>To update</td>
      <td>ruvseq</td>
      <td>RUVSeq</td>
      <td>Differential gene expression analysis</td>
      <td>Gene expression, RNA-seq</td>
      <td>Remove Unwanted Variation from RNA-Seq Data</td>
      <td>This package implements the remove unwanted variation (RUV) methods for the normalization of RNA-Seq read counts between samples.</td>
      <td></td>
      <td>To update</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/DESeq2.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>ruvseq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ruvseq</td>
    </tr>
    <tr>
      <td></td>
      <td>salsa2</td>
      <td>2.3</td>
      <td>2.3</td>
      <td>salsa2</td>
      <td>Up-to-date</td>
      <td>SALSA</td>
      <td>SALSA</td>
      <td>Genome assembly, De-novo assembly, Scaffolding</td>
      <td>Sequence assembly, DNA binding sites, Mapping</td>
      <td>A tool to scaffold long read assemblies with Hi-C</td>
      <td>&gt; VERY_LOW CONFIDENCE! | &gt; CORRECT NAME OF TOOL COULD ALSO BE 'chromosome-scale', 'reference-quality', 'Hi-C', 'scaffolder' | Integrating Hi-C links with assembly graphs for chromosome-scale assembly | SALSA: A tool to scaffold long read assemblies with Hi-C data | SALSA: A tool to scaffold long read assemblies with Hi-C | This code is used to scaffold your assemblies using Hi-C data. This version implements some improvements in the original SALSA algorithm. If you want to use the old version, it can be found in the old_salsa branch</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/marbl/SALSA</td>
      <td>Assembly</td>
      <td>salsa</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/salsa2</td>
    </tr>
    <tr>
      <td></td>
      <td>samblaster</td>
      <td>0.1.24</td>
      <td>0.1.26</td>
      <td>samblaster</td>
      <td>To update</td>
      <td>samblaster</td>
      <td>SAMBLASTER</td>
      <td>Split read mapping</td>
      <td>DNA, Sequencing, Mapping</td>
      <td>samblaster marks duplicates and can output split and discordant alignments from SAM/BAM files</td>
      <td>A tool to mark duplicates and extract discordant and split reads from SAM files.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/GregoryFaust/samblaster</td>
      <td>SAM, Fastq Manipulation, Variant Analysis</td>
      <td>samblaster</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/samblaster</td>
    </tr>
    <tr>
      <td></td>
      <td>sansa</td>
      <td>0.0.8</td>
      <td>0.1.1</td>
      <td>sansa</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Sansa is a tool for structural variant annotation.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/dellytools/sansa</td>
      <td>Variant Analysis</td>
      <td>sansa</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/sansa</td>
    </tr>
    <tr>
      <td></td>
      <td>sarscov2formatter</td>
      <td>1.0</td>
      <td>1.0</td>
      <td>sarscov2formatter</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>sarscov2formatter custom script</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/nickeener/sarscov2formatter</td>
      <td>Sequence Analysis</td>
      <td>sarscov2formatter</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/sarscov2formatter</td>
    </tr>
    <tr>
      <td></td>
      <td>sarscov2summary</td>
      <td>0.1</td>
      <td>0.5</td>
      <td>sarscov2summary</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>sarscov2summary custom script</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/nickeener/sarscov2summary</td>
      <td>Sequence Analysis</td>
      <td>sarscov2summary</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/sarscov2summary</td>
    </tr>
    <tr>
      <td></td>
      <td>sbml2sbol</td>
      <td>0.1.13</td>
      <td></td>
      <td>sbml2sbol</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert SBML to SBOL format</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/neilswainston/SbmlToSbol</td>
      <td>Synthetic Biology</td>
      <td>sbml2sbol</td>
      <td>iuc</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>scanpy</td>
      <td>1.9.6</td>
      <td>1.7.2</td>
      <td>scanpy</td>
      <td>To update</td>
      <td>scanpy</td>
      <td>SCANPY</td>
      <td>Differential gene expression analysis</td>
      <td>Gene expression, Cell biology, Genetics</td>
      <td>Scanpy  Single-Cell Analysis in Python</td>
      <td>Scalable toolkit for analyzing single-cell gene expression data. It includes preprocessing, visualization, clustering, pseudotime and trajectory inference and differential expression testing. The Python-based implementation efficiently deals with datasets of more than one million cells.</td>
      <td></td>
      <td>To update</td>
      <td>https://scanpy.readthedocs.io</td>
      <td>Transcriptomics, Sequence Analysis</td>
      <td>scanpy</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/scanpy/</td>
    </tr>
    <tr>
      <td></td>
      <td>scater</td>
      <td>1.22.0</td>
      <td>1.30.1</td>
      <td>bioconductor-scater</td>
      <td>To update</td>
      <td>scater</td>
      <td>scater</td>
      <td>Read pre-processing, Sequencing quality control, Sequence visualisation</td>
      <td>RNA-seq, Quality affairs, Molecular genetics</td>
      <td>Scater (Single-Cell Analysis Toolkit for gene Expression data in R) is acollection of tools for doing various analyses of single-cell RNA-seq geneexpression data, with a focus on quality control and visualization.</td>
      <td>Pre-processing, quality control, normalization and visualization of single-cell RNA-seq data.</td>
      <td></td>
      <td>To update</td>
      <td>http://bioconductor.org/packages/scater/</td>
      <td>Transcriptomics, RNA, Visualization</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/scater</td>
    </tr>
    <tr>
      <td></td>
      <td>sceasy</td>
      <td>0.0.7</td>
      <td>0.0.7</td>
      <td>r-sceasy</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Converter between difference single-cell formats</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/cellgeni/sceasy/</td>
      <td>Transcriptomics</td>
      <td>sceasy_convert</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/sceasy/</td>
    </tr>
    <tr>
      <td></td>
      <td>schicexplorer</td>
      <td>4</td>
      <td>7</td>
      <td>schicexplorer</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>scHiCExplorer: Set of programs to process, analyze and visualize scHi-C data.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/joachimwolff/schicexplorer</td>
      <td>Sequence Analysis, Transcriptomics, Visualization</td>
      <td>schicexplorer</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/schicexplorer</td>
    </tr>
    <tr>
      <td></td>
      <td>scikit-bio</td>
      <td>0.4.2</td>
      <td>0.4.2</td>
      <td>scikit-bio</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>scikit-bio: an open-source, BSD-licensed, python package providing data structures, algorithms, and educational resources for bioinformatics</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://scikit-bio.org/</td>
      <td>Sequence Analysis</td>
      <td>scikit_bio</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/scikit_bio</td>
    </tr>
    <tr>
      <td></td>
      <td>scoary</td>
      <td>1.6.16</td>
      <td>1.6.16</td>
      <td>scoary</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Scoary calculates the assocations between all genes in the accessory genome and the traits.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/AdmiralenOla/Scoary</td>
      <td>Metagenomics</td>
      <td>scoary</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/scoary</td>
    </tr>
    <tr>
      <td></td>
      <td>scpipe</td>
      <td>1.0.0+galaxy2</td>
      <td>2.2.0</td>
      <td>bioconductor-scpipe</td>
      <td>To update</td>
      <td>scpipe</td>
      <td>scPipe</td>
      <td>Genome annotation, Validation, Alignment, Visualisation</td>
      <td>Gene expression, RNA-seq, Sequencing</td>
      <td>A flexible preprocessing pipeline for single-cell RNA-sequencing data</td>
      <td>A preprocessing pipeline for single cell RNA-seq data that starts from the fastq files and produces a gene count matrix with associated quality control information. It can process fastq data generated by CEL-seq, MARS-seq, Drop-seq, Chromium 10x and SMART-seq protocols.</td>
      <td></td>
      <td>To update</td>
      <td>http://bioconductor.org/packages/release/bioc/html/scPipe.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>scpipe</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/scpipe</td>
    </tr>
    <tr>
      <td></td>
      <td>seacr</td>
      <td>1.3</td>
      <td>1.3</td>
      <td>seacr</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SEACR is intended to call peaks and enriched regions from sparse CUT&amp;RUN or chromatin profiling data in which background is dominated by zeroes.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/FredHutch/SEACR</td>
      <td>Epigenetics, ChIP-seq</td>
      <td>seacr</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/seacr</td>
    </tr>
    <tr>
      <td></td>
      <td>selenzy_wrapper</td>
      <td>0.3.0</td>
      <td>0.3.0</td>
      <td>selenzy_wrapper</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Performs enzyme selection from a reaction query.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/brsynth/selenzy-wrapper</td>
      <td>Synthetic Biology</td>
      <td>selenzy_wrapper</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/selenzy_wrapper</td>
    </tr>
    <tr>
      <td></td>
      <td>semibin</td>
      <td>2.0.2</td>
      <td>2.0.2</td>
      <td>semibin</td>
      <td>Up-to-date</td>
      <td>semibin</td>
      <td>SemiBin</td>
      <td>Sequence assembly, Read binning</td>
      <td>Metagenomics, Machine learning, Microbial ecology, Sequence assembly</td>
      <td>SemiBin: Semi-supervised Metagenomic Binning Using Siamese Neural Networks</td>
      <td>Command tool for metagenomic binning with semi-supervised deep learning using information from reference genomes.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://semibin.readthedocs.io/en/latest/</td>
      <td>Metagenomics</td>
      <td>semibin</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/semibin</td>
    </tr>
    <tr>
      <td></td>
      <td>seq2hla</td>
      <td>2.3</td>
      <td>2.3</td>
      <td>seq2hla</td>
      <td>Up-to-date</td>
      <td>seq2hla</td>
      <td>Seq2HLA</td>
      <td>Read mapping, Genetic variation analysis</td>
      <td>Transcriptomics, Mapping</td>
      <td>Precision HLA typing and expression from RNAseq data</td>
      <td>seq2HLA is a computational tool to determine Human Leukocyte Antigen (HLA) directly from existing and future short RNA-Seq reads. It takes standard RNA-Seq sequence reads in fastq format as input, uses a bowtie index comprising known HLA alleles and outputs the most likely HLA class I and class II types, a p-value for each call, and the expression of each class.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/TRON-Bioinformatics/seq2HLA</td>
      <td>Sequence Analysis</td>
      <td>seq2hla</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/seq2hla</td>
    </tr>
    <tr>
      <td></td>
      <td>seqcomplexity</td>
      <td>0.1.2</td>
      <td>0.1.2</td>
      <td>seqcomplexity</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Sequence complexity for raw reads</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/stevenweaver/seqcomplexity</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/stephenshank/tools-iuc/tree/seqcomplexity/tools/seqcomplexity/</td>
    </tr>
    <tr>
      <td></td>
      <td>seqkit</td>
      <td>2.3.1</td>
      <td>2.7.0</td>
      <td>seqkit</td>
      <td>To update</td>
      <td>seqkit</td>
      <td>seqkit</td>
      <td>DNA transcription, Sequence trimming, DNA translation, Sequence conversion</td>
      <td>Database management, Sequence analysis</td>
      <td>A cross-platform and ultrafast toolkit for FASTA/Q file manipulation</td>
      <td>FASTA and FASTQ are basic and ubiquitous formats for storing nucleotide and protein sequences. Common manipulations of FASTA/Q file include converting, searching, filtering, deduplication, splitting, shuffling, and sampling. Existing tools only implement some of these manipulations, and not particularly efficiently, and some are only available for certain operating systems. Furthermore, the complicated installation process of required packages and running environments can render these programs less user friendly. SeqKit demonstrates competitive performance in execution time and memory usage compared to similar tools. The efficiency and usability of SeqKit enable researchers to rapidly accomplish common FASTA/Q file manipulations.</td>
      <td></td>
      <td>To update</td>
      <td>https://bioinf.shenwei.me/seqkit/</td>
      <td>Sequence Analysis</td>
      <td>seqkit</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/seqkit</td>
    </tr>
    <tr>
      <td></td>
      <td>seqprep</td>
      <td>1.3.2</td>
      <td>1.3.2</td>
      <td>seqprep</td>
      <td>Up-to-date</td>
      <td>seqprep</td>
      <td>SeqPrep</td>
      <td>Nucleic acid design</td>
      <td>Genomics, Sequence assembly, Sequencing, Probes and primers</td>
      <td>Tool for merging paired-end Illumina reads and trimming adapters.</td>
      <td>Strips adapters and optionally merges overlapping paired-end (or paired-end contamination in mate-pair libraries) illumina style reads.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/jstjohn/SeqPrep</td>
      <td>Fastq Manipulation, Sequence Analysis</td>
      <td>seqprep</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/seqprep</td>
    </tr>
    <tr>
      <td></td>
      <td>seqsero2</td>
      <td>1.2.1</td>
      <td>1.2.1</td>
      <td>seqsero2</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Salmonella serotype prediction from genome sequencing data</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/denglab/SeqSero2</td>
      <td>Sequence Analysis</td>
      <td>seqsero2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/seqsero2</td>
    </tr>
    <tr>
      <td></td>
      <td>seqtk</td>
      <td>1.4</td>
      <td>1.4</td>
      <td>seqtk</td>
      <td>Up-to-date</td>
      <td>seqtk</td>
      <td>seqtk</td>
      <td>Data handling, Sequence file editing</td>
      <td>Data management</td>
      <td>Toolkit for processing sequences in FASTA/Q formats</td>
      <td>A tool for processing sequences in the FASTA or FASTQ format. It parses both FASTA and FASTQ files which can also be optionally compressed by gzip.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/lh3/seqtk</td>
      <td>Sequence Analysis</td>
      <td>seqtk</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/seqtk</td>
    </tr>
    <tr>
      <td></td>
      <td>seqwish</td>
      <td>0.7.5</td>
      <td>0.7.9</td>
      <td>seqwish</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Alignment to variation graph inducer</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/seqwish</td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/seqwish/</td>
    </tr>
    <tr>
      <td></td>
      <td>seurat</td>
      <td>4.3.0.1</td>
      <td>3.0.2</td>
      <td>r-seurat</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A toolkit for quality control, analysis, and exploration of single cell RNA sequencing data</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/satijalab/seurat</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>seurat</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/seurat</td>
    </tr>
    <tr>
      <td></td>
      <td>shasta</td>
      <td>0.6.0</td>
      <td>0.11.1</td>
      <td>shasta</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Fast de novo assembly of long read sequencing data</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/chanzuckerberg/shasta</td>
      <td>Assembly, Nanopore</td>
      <td>shasta</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/shasta</td>
    </tr>
    <tr>
      <td></td>
      <td>shorah</td>
      <td>1.1.3</td>
      <td>1.99.2</td>
      <td>shorah</td>
      <td>To update</td>
      <td>shorah</td>
      <td>ShoRAH</td>
      <td>Haplotype mapping, Variant calling</td>
      <td>Metagenomics, Sequencing, Genetics</td>
      <td>Reconstruct haplotypes using ShoRAH in amplicon mode</td>
      <td>Inference of a population from a set of short reads. The package contains programs that support mapping of reads to a reference genome, correcting sequencing errors by locally clustering reads in small windows of the alignment, reconstructing a minimal set of global haplotypes that explain the reads, and estimating the frequencies of the inferred haplotypes.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/cbg-ethz/shorah/blob/master/README.md</td>
      <td>Sequence Analysis</td>
      <td>shorah_amplicon</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/shorah</td>
    </tr>
    <tr>
      <td></td>
      <td>shovill</td>
      <td>1.1.0</td>
      <td>1.1.0</td>
      <td>shovill</td>
      <td>Up-to-date</td>
      <td>shovill</td>
      <td>shovill</td>
      <td>Genome assembly</td>
      <td>Genomics, Microbiology, Sequence assembly</td>
      <td>Faster de novo assembly pipeline based around Spades</td>
      <td>Shovill is a pipeline for assembly of bacterial isolate genomes from Illumina paired-end reads.  Shovill uses SPAdes at its core, but alters the steps before and after the primary assembly step to get similar results in less time. Shovill also supports other assemblers like SKESA, Velvet and Megahit, so you can take advantage of the pre- and post-processing the Shovill provides with those too.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/tseemann/shovill</td>
      <td>Assembly</td>
      <td>shovill</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/shovill</td>
    </tr>
    <tr>
      <td></td>
      <td>sickle</td>
      <td>1.33.2</td>
      <td>1.33</td>
      <td>sickle-trim</td>
      <td>To update</td>
      <td>sickle</td>
      <td>sickle</td>
      <td>Sequence trimming</td>
      <td>Data quality management</td>
      <td>A windowed adaptive trimming tool for FASTQ files using quality</td>
      <td>A  tool that uses sliding windows along with quality and length thresholds to determine when quality is sufficiently low to trim the 3'-end of reads and also determines when the quality is sufficiently high enough to trim the 5'-end of reads.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/najoshi/sickle</td>
      <td>Fastq Manipulation, Sequence Analysis</td>
      <td>sickle</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/sickle</td>
    </tr>
    <tr>
      <td></td>
      <td>simtext</td>
      <td>0.0.2</td>
      <td>1.0.1</td>
      <td>r-argparse</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A text mining framework for interactive analysis and visualization of similarities among biomedical entities.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/dlal-group/simtext</td>
      <td>Text Manipulation</td>
      <td>simtext</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tools/simtext</td>
    </tr>
    <tr>
      <td></td>
      <td>sina</td>
      <td>1.7.2</td>
      <td>1.7.2</td>
      <td>sina</td>
      <td>Up-to-date</td>
      <td>sina</td>
      <td>SINA</td>
      <td>Sequence alignment analysis, Multiple sequence alignment, Taxonomic classification, Structure-based sequence alignment</td>
      <td>Sequencing, RNA, Nucleic acid structure analysis, Taxonomy, Sequence analysis, Taxonomy</td>
      <td>SINA reference based multiple sequence alignment</td>
      <td>Aligns and optionally taxonomically classifies your rRNA gene sequences.Reference based multiple sequence alignment</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://sina.readthedocs.io/en/latest/</td>
      <td>Sequence Analysis</td>
      <td>sina</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/sina</td>
    </tr>
    <tr>
      <td></td>
      <td>sinto</td>
      <td>0.9.0</td>
      <td>0.10.0</td>
      <td>sinto</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Sinto single-cell analysis tools</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/timoast/sinto</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/sinto</td>
    </tr>
    <tr>
      <td></td>
      <td>slamdunk</td>
      <td>0.4.3</td>
      <td>0.4.3</td>
      <td>slamdunk</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Slamdunk maps and quantifies SLAMseq reads</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://t-neumann.github.io/slamdunk</td>
      <td>RNA, Transcriptomics, Sequence Analysis, Next Gen Mappers</td>
      <td>slamdunk</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/slamdunk</td>
    </tr>
    <tr>
      <td></td>
      <td>sleuth</td>
      <td>0.30.1</td>
      <td>0.30.1</td>
      <td>r-sleuth</td>
      <td>Up-to-date</td>
      <td>sleuth</td>
      <td>sleuth</td>
      <td>Expression data visualisation, Differential gene expression analysis, Gene expression profiling, Statistical calculation</td>
      <td>RNA-seq, Gene expression, Statistics and probability</td>
      <td>Sleuth is a program for differential analysis of RNA-Seq data.</td>
      <td>A statistical model and software application for RNA-seq differential expression analysis.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/pachterlab/sleuth</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>sleuth</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/sleuth</td>
    </tr>
    <tr>
      <td></td>
      <td>smallgenomeutilities</td>
      <td>0.4.0</td>
      <td>0.4.0</td>
      <td>smallgenomeutilities</td>
      <td>Up-to-date</td>
      <td>v-pipe</td>
      <td>V-pipe</td>
      <td>Read pre-processing, Sequence alignment, Genetic variation analysis</td>
      <td>Genomics, Population genetics, Workflows, Virology, Sequencing</td>
      <td>Set of utilities for manipulating small viral genome data.</td>
      <td>Bioinformatics pipeline for the analysis of next-generation sequencing data derived from intra-host viral populations.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/cbg-ethz/smallgenomeutilities</td>
      <td>Sequence Analysis</td>
      <td>smallgenomeutilities</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/smallgenomeutilities</td>
    </tr>
    <tr>
      <td></td>
      <td>smudgeplot</td>
      <td>0.2.5</td>
      <td>0.2.5</td>
      <td>smudgeplot</td>
      <td>Up-to-date</td>
      <td>smudgeplots</td>
      <td>Smudgeplots</td>
      <td>Sequence trimming, Genotyping, k-mer counting</td>
      <td>Sequence assembly, Genetic variation, Mathematics</td>
      <td>Inference of ploidy and heterozygosity structure using whole genome sequencing</td>
      <td>Reference-free profiling of polyploid genomes | Inference of ploidy and heterozygosity structure using whole genome sequencing data | Smudgeplots are computed from raw or even better from trimmed reads and show the haplotype structure using heterozygous kmer pairs. For example: | This tool extracts heterozygous kmer pairs from kmer dump files and performs gymnastics with them. We are able to disentangle genome structure by comparing the sum of kmer pair coverages (CovA + CovB) to their relative coverage (CovA / (CovA + CovB)). Such an approach also allows us to analyze obscure genomes with duplications, various ploidy levels, etc | GenomeScope 2.0 and Smudgeplots: Reference-free profiling of polyploid genomes Timothy Rhyker Ranallo-Benavidez, Kamil S. Jaron, Michael C. Schatz bioRxiv 747568; doi: https://doi.org/10.1101/747568</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/KamilSJaron/smudgeplot</td>
      <td>Assembly</td>
      <td>smudgeplot</td>
      <td>galaxy-australia</td>
      <td>https://github.com/galaxyproject/tools-iuc</td>
    </tr>
    <tr>
      <td></td>
      <td>snap</td>
      <td>2013_11_29</td>
      <td>2013_11_29</td>
      <td>snap</td>
      <td>Up-to-date</td>
      <td>snap</td>
      <td>SNAP</td>
      <td>Gene prediction</td>
      <td>DNA, DNA polymorphism, Genetics</td>
      <td>SNAP is a general purpose gene finding program suitable for both eukaryotic and prokaryotic genomes.</td>
      <td>The Semi-HMM-based Nucleic Acid Parser is a gene prediction tool.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/KorfLab/SNAP</td>
      <td>Sequence Analysis</td>
      <td>snap</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/snap</td>
    </tr>
    <tr>
      <td></td>
      <td>sniffles</td>
      <td>1.0.12</td>
      <td>2.2</td>
      <td>sniffles</td>
      <td>To update</td>
      <td>sniffles</td>
      <td>Sniffles</td>
      <td>Sequence analysis, Structural variation detection</td>
      <td>DNA structural variation, Sequencing</td>
      <td>Galaxy wrapper for sniffles</td>
      <td>An algorithm for structural variation detection from third generation sequencing alignment.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/fritzsedlazeck/Sniffles</td>
      <td>Sequence Analysis</td>
      <td>sniffles</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc</td>
    </tr>
    <tr>
      <td></td>
      <td>snipit</td>
      <td>1.2</td>
      <td>1.2</td>
      <td>snipit</td>
      <td>Up-to-date</td>
      <td>snipit</td>
      <td>snipit</td>
      <td>Base position variability plotting</td>
      <td>Virology</td>
      <td>Summarise snps relative to a reference sequence</td>
      <td>Summarise snps relative to a reference sequence</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/aineniamh/snipit</td>
      <td>Variant Analysis, Sequence Analysis</td>
      <td>snipit</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/snipit</td>
    </tr>
    <tr>
      <td></td>
      <td>snippy</td>
      <td></td>
      <td>4.6.0</td>
      <td>snippy</td>
      <td>To update</td>
      <td>snippy</td>
      <td>snippy</td>
      <td>Phylogenetic tree visualisation, Phylogenetic tree generation, Variant calling</td>
      <td>Genomics, Model organisms, DNA polymorphism, Phylogenetics</td>
      <td>Contains the snippy tool for characterising microbial snps</td>
      <td>Rapid haploid variant calling and core SNP phylogeny generation.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/tseemann/snippy</td>
      <td>Sequence Analysis</td>
      <td>snippy</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/snippy</td>
    </tr>
    <tr>
      <td></td>
      <td>snp-dists</td>
      <td>0.8.2</td>
      <td>0.8.2</td>
      <td>snp-dists</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute pairwise SNP distance matrix from a FASTA sequence alignment</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/tseemann/snp-dists</td>
      <td>Variant Analysis</td>
      <td>snp_dists</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/snp-dists</td>
    </tr>
    <tr>
      <td></td>
      <td>snp-sites</td>
      <td>2.5.1</td>
      <td>2.5.1</td>
      <td>snp-sites</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Finds SNP sites from a multi-FASTA alignment file</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/sanger-pathogens/snp-sites</td>
      <td>Variant Analysis</td>
      <td>snp_sites</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/snp-sites</td>
    </tr>
    <tr>
      <td></td>
      <td>snpeff-covid19</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>snpeff</td>
      <td>snpEff</td>
      <td>SNP detection</td>
      <td>DNA polymorphism, Genetic variation, Nucleic acid sites, features and motifs</td>
      <td>SnpEff, the COVID-19 version, is a genetic variant annotation and effect prediction toolbox</td>
      <td>Variant annotation and effect prediction tool. It annotates and predicts the effects of variants on genes and proteins (such as amino acid changes).</td>
      <td></td>
      <td>To update</td>
      <td>http://snpeff.sourceforge.net/</td>
      <td>Genome-Wide Association Study, Variant Analysis</td>
      <td>snpeff_sars_cov_2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/snpeff</td>
    </tr>
    <tr>
      <td></td>
      <td>snpfreqplot</td>
      <td>1.0</td>
      <td></td>
      <td>r-base</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Generates a heatmap of allele frequencies grouped by variant type for SARS-CoV-2 data</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/snpfreqplot/</td>
      <td>Variant Analysis</td>
      <td>snpfreqplot</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/snpfreqplot/</td>
    </tr>
    <tr>
      <td></td>
      <td>socru</td>
      <td>2.1.7</td>
      <td>2.2.4</td>
      <td>socru</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Order and orientation of complete bacterial genomes</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/quadram-institute-bioscience/socru</td>
      <td>Sequence Analysis</td>
      <td>socru</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/socru</td>
    </tr>
    <tr>
      <td></td>
      <td>sonneityping</td>
      <td>20210201</td>
      <td>20210201</td>
      <td>sonneityping</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Scripts for parsing Mykrobe predict results for Shigella sonnei.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/katholt/sonneityping</td>
      <td>Sequence Analysis</td>
      <td>sonneityping</td>
      <td>iuc</td>
      <td>https://github.com/katholt/sonneityping</td>
    </tr>
    <tr>
      <td></td>
      <td>spades</td>
      <td>3.15.5</td>
      <td>3.15.5</td>
      <td>spades</td>
      <td>Up-to-date</td>
      <td>biosyntheticspades</td>
      <td>biosyntheticSPAdes</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly, Sequence sites, features and motifs, Genetic engineering</td>
      <td>SPAdes is an assembly toolkit containing various assembly pipelines. It implements the following 4 stages: assembly graph construction, k-bimer adjustment, construction of paired assembly graph and contig construction.</td>
      <td>biosyntheticSPAdes - biosynthetic gene cluster assembly with paired-end reads</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/ablab/spades</td>
      <td>Assembly, RNA, Metagenomics</td>
      <td>spades</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/spades</td>
    </tr>
    <tr>
      <td></td>
      <td>spaln</td>
      <td>2.4.9</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Spaln (space-efficient spliced alignment) maps and aligns a set of cDNA or protein sequences onto a whole genomic sequence.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.genome.ist.i.kyoto-u.ac.jp/~aln_user/spaln/</td>
      <td>Sequence Analysis, Genome annotation</td>
      <td>spaln</td>
      <td>iuc</td>
      <td>https://github.com/ogotoh/spaln</td>
    </tr>
    <tr>
      <td></td>
      <td>spotyping</td>
      <td>2.1</td>
      <td>2.1</td>
      <td>spotyping</td>
      <td>Up-to-date</td>
      <td>spotyping</td>
      <td>SpoTyping</td>
      <td>Variant pattern analysis</td>
      <td>Microbiology, Sequencing, Sequence composition, complexity and repeats, Genetic variation</td>
      <td>SpoTyping allows fast and accurate in silico Mycobacterium spoligotyping from sequence reads</td>
      <td>Fast and accurate in silico Mycobacterium spoligotyping from sequence reads.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/xiaeryu/SpoTyping-v2.0</td>
      <td>Sequence Analysis</td>
      <td>spotyping</td>
      <td>iuc</td>
      <td>https://github.com/xiaeryu/SpoTyping-v2.0/tree/master/SpoTyping-v2.0-commandLine</td>
    </tr>
    <tr>
      <td></td>
      <td>spyboat</td>
      <td>0.1.2</td>
      <td></td>
      <td>spyboat</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Wavelet analysis for 3d-image stacks</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://github.com/tensionhead/spyboat</td>
      <td>Imaging, Graphics</td>
      <td>spyboat</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/packages/spyboat</td>
    </tr>
    <tr>
      <td></td>
      <td>sra-tools</td>
      <td>3.0.8</td>
      <td>3.0.10</td>
      <td>sra-tools</td>
      <td>To update</td>
      <td>sra-tools</td>
      <td>SRA Software Toolkit</td>
      <td>Data handling</td>
      <td>DNA, Genomics, Sequencing</td>
      <td>NCBI Sequence Read Archive toolkit utilities</td>
      <td>The SRA Toolkit and SDK from NCBI is a collection of tools and libraries for using data in the INSDC Sequence Read Archives.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ncbi/sra-tools</td>
      <td>Data Source, Fastq Manipulation</td>
      <td>sra_tools</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/sra-tools</td>
    </tr>
    <tr>
      <td></td>
      <td>srst2</td>
      <td>0.2.0</td>
      <td>1.19.2</td>
      <td>samtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SRST2 Short Read Sequence Typing for Bacterial Pathogens</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://katholt.github.io/srst2/</td>
      <td>Metagenomics</td>
      <td>srst2</td>
      <td>iuc</td>
      <td>https://github.com/katholt/srst2</td>
    </tr>
    <tr>
      <td></td>
      <td>stacks</td>
      <td></td>
      <td>2.65</td>
      <td>stacks</td>
      <td>To update</td>
      <td>stacks</td>
      <td>Stacks</td>
      <td>Data handling</td>
      <td>Mapping, Population genetics</td>
      <td>Stacks is a software pipeline for building loci from short-read sequences, such as RAD-seq</td>
      <td>Developed to work with restriction enzyme based sequence data, such as RADseq, for building genetic maps and conducting population genomics and phylogeography analysis.</td>
      <td></td>
      <td>To update</td>
      <td>http://catchenlab.life.illinois.edu/stacks/</td>
      <td>Sequence Analysis</td>
      <td>stacks</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/stacks</td>
    </tr>
    <tr>
      <td></td>
      <td>stacks2</td>
      <td>2.55</td>
      <td>2.65</td>
      <td>stacks</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Stacks is a software pipeline for building loci from short-read sequences, such as RAD-seq</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://catchenlab.life.illinois.edu/stacks/</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/stacks2</td>
    </tr>
    <tr>
      <td></td>
      <td>star_fusion</td>
      <td>0.5.4-3+galaxy1</td>
      <td>1.12.0</td>
      <td>star-fusion</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>STAR Fusion detects fusion genes in RNA-Seq data after running RNA-STAR</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Transcriptomics</td>
      <td>star_fusion</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/star_fusion</td>
    </tr>
    <tr>
      <td></td>
      <td>straindesign</td>
      <td>3.2.2</td>
      <td>3.2.2</td>
      <td>straindesign</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Toolbox to optimize biological model</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/brsynth/straindesign</td>
      <td>Systems Biology, Synthetic Biology</td>
      <td>straindesign</td>
      <td>iuc</td>
      <td>https://github.com/brsynth/straindesign</td>
    </tr>
    <tr>
      <td></td>
      <td>strelka</td>
      <td>2.9.10</td>
      <td>2.9.10</td>
      <td>strelka</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/Illumina/strelka/</td>
      <td>Variant Analysis</td>
      <td>strelka</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/strelka</td>
    </tr>
    <tr>
      <td></td>
      <td>stringtie</td>
      <td>2.2.1</td>
      <td>2.2.1</td>
      <td>stringtie</td>
      <td>Up-to-date</td>
      <td>stringtie</td>
      <td>StringTie</td>
      <td>Transcriptome assembly, RNA-Seq analysis</td>
      <td>Transcriptomics, RNA-seq</td>
      <td>StringTie is a fast and highly efficient assembler of RNA-Seq alignments into potential transcripts.</td>
      <td>Fast and highly efficient assembler of RNA-Seq alignments into potential transcripts. It uses a novel network flow algorithm as well as an optional de novo assembly step to assemble and quantitate full-length transcripts representing multiple splice variants for each gene locus.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://ccb.jhu.edu/software/stringtie/</td>
      <td>Transcriptomics</td>
      <td>stringtie</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/stringtie</td>
    </tr>
    <tr>
      <td></td>
      <td>structure</td>
      <td>2.3.4</td>
      <td>2.3.4</td>
      <td>structure</td>
      <td>Up-to-date</td>
      <td>structure</td>
      <td>Structure</td>
      <td>Genetic variation analysis</td>
      <td>Population genetics</td>
      <td>for using multi-locus genotype data to investigate population structure.</td>
      <td>The program structureis a free software package for using multi-locus genotype data to investigate population structure. Its uses include inferring the presence of distinct populations, assigning individuals to populations, studying hybrid zones, identifying migrants and admixed individuals, and estimating population allele frequencies in situations where many individuals are migrants or admixed.</td>
      <td></td>
      <td>Up-to-date</td>
      <td></td>
      <td>Phylogenetics, Variant Analysis</td>
      <td>structure</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/structure</td>
    </tr>
    <tr>
      <td></td>
      <td>structureharvester</td>
      <td>0.6.94</td>
      <td>0.6.94</td>
      <td>structureharvester</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>for parsing STRUCTURE outputs and for performing the Evanno method</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td></td>
      <td>Phylogenetics, Variant Analysis</td>
      <td>structureharvester</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/structureharvester</td>
    </tr>
    <tr>
      <td></td>
      <td>swift</td>
      <td>1.0</td>
      <td>1.0</td>
      <td>swiftlink</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Parallel MCMC Linkage Analysis</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/ajm/swiftlink</td>
      <td>Variant Analysis</td>
      <td>swiftlink</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/swiftlink/</td>
    </tr>
    <tr>
      <td></td>
      <td>syndiva</td>
      <td>1.0</td>
      <td></td>
      <td>clustalo</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SynDivA was developed to analyze the diversity of synthetic libraries of a Fibronectin domain.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>syndiva</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/SynDivA</td>
    </tr>
    <tr>
      <td></td>
      <td>table_compute</td>
      <td>1.2.4</td>
      <td></td>
      <td>pandas</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Perform general-purpose table operations</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>table_compute</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/table_compute</td>
    </tr>
    <tr>
      <td></td>
      <td>tag_pileup_frequency</td>
      <td>1.0.2</td>
      <td></td>
      <td>openjdk</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Contains a tool that generates a frequency pileup of the 5' ends of aligned reads in a BAM filerelative to reference points in a BED file.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/seqcode/cegr-tools/tree/master/src/org/seqcode/cegrtools/tagpileup</td>
      <td>Statistics, SAM, Genomic Interval Operations</td>
      <td>tag_pileup_frequency</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/tag_pileup_frequency</td>
    </tr>
    <tr>
      <td></td>
      <td>tasmanian_mismatch</td>
      <td>1.0.7</td>
      <td>1.0.7</td>
      <td>tasmanian-mismatch</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Analysis of positional mismatches</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>tasmanian_mismatch</td>
      <td>iuc</td>
      <td>https://github.com/nebiolabs/tasmanian-mismatch</td>
    </tr>
    <tr>
      <td></td>
      <td>taxonomy_filter_refseq</td>
      <td>0.3.0</td>
      <td>1.0.7</td>
      <td>rust-ncbitaxonomy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter RefSeq by taxonomy</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/pvanheus/ncbitaxonomy</td>
      <td>Sequence Analysis, Genome annotation</td>
      <td>taxonomy_filter_refseq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc</td>
    </tr>
    <tr>
      <td></td>
      <td>taxonomy_krona_chart</td>
      <td>2.7.1+galaxy0</td>
      <td>2.8.1</td>
      <td>krona</td>
      <td>To update</td>
      <td>krona</td>
      <td>Krona</td>
      <td>Visualisation</td>
      <td>Metagenomics</td>
      <td>Krona pie chart from taxonomic profile</td>
      <td>Krona creates interactive HTML5 charts of hierarchical data (such as taxonomic abundance in a metagenome).</td>
      <td></td>
      <td>To update</td>
      <td>http://sourceforge.net/projects/krona/</td>
      <td>Assembly</td>
      <td>taxonomy_krona_chart</td>
      <td>crs4</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/taxonomy_krona_chart</td>
    </tr>
    <tr>
      <td></td>
      <td>tb-profiler</td>
      <td>4.4.1</td>
      <td>5.0.1</td>
      <td>tb-profiler</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Processes M. tuberculosis sequence data to infer strain type and identify known drug resistance markers.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/jodyphelan/TBProfiler</td>
      <td>Sequence Analysis</td>
      <td>tbprofiler</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/tb-profiler</td>
    </tr>
    <tr>
      <td></td>
      <td>tb_variant_filter</td>
      <td>0.4.0</td>
      <td>0.4.0</td>
      <td>tb_variant_filter</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>M. tuberculosis H37Rv VCF filter</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/COMBAT-TB/tb_variant_filter</td>
      <td>Variant Analysis</td>
      <td>tb_variant_filter</td>
      <td>iuc</td>
      <td>https://github.com/COMBAT-TB/tb_variant_filter</td>
    </tr>
    <tr>
      <td></td>
      <td>tbl2gff3</td>
      <td>1.2</td>
      <td>0.6.6</td>
      <td>bcbiogff</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Table to GFF3</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/tbl2gff3</td>
      <td>Convert Formats, Sequence Analysis</td>
      <td>tbl2gff3</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/tbl2gff3</td>
    </tr>
    <tr>
      <td></td>
      <td>tbvcfreport</td>
      <td>0.1.10</td>
      <td>0.1.10</td>
      <td>tbvcfreport</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Generate HTML report from SnpEff M.tuberculosis VCF(s)</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/COMBAT-TB/tbvcfreport</td>
      <td>Variant Analysis</td>
      <td>tbvcfreport</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/tbvcfreport</td>
    </tr>
    <tr>
      <td></td>
      <td>te_finder</td>
      <td>1.0.1</td>
      <td>1.19.2</td>
      <td>samtools</td>
      <td>To update</td>
      <td>tefinder</td>
      <td>TEfinder</td>
      <td>Genome indexing, Variant calling, PCR primer design</td>
      <td>Sequencing, Mobile genetic elements, Workflows, Evolutionary biology, Genetic variation</td>
      <td>Transposable element insertions finder</td>
      <td>A Bioinformatics Pipeline for Detecting New Transposable Element Insertion Events in Next-Generation Sequencing Data.A bioinformatics tool for detecting novel transposable element insertions.TEfinder uses discordant reads to detect novel transposable element insertion events in paired-end sample sequencing data.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/VistaSohrab/TEfinder</td>
      <td>Sequence Analysis</td>
      <td>te_finder</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/te_finder/</td>
    </tr>
    <tr>
      <td></td>
      <td>telescope</td>
      <td>1.0.3</td>
      <td>1.0.3</td>
      <td>telescope</td>
      <td>Up-to-date</td>
      <td>Telescope-expression</td>
      <td>Telescope</td>
      <td>Essential dynamics, Sequence trimming, RNA-Seq quantification, Expression analysis, Read mapping</td>
      <td>RNA-Seq, Transcriptomics, Mapping, Gene transcripts, Sequence assembly</td>
      <td>Single locus resolution of Transposable ELEment expression.</td>
      <td>Telescope is a tool for the characterization of the retrotranscriptome by accurate estimation of transposable element expression and the quantification of transposable element expression using RNA-seq.It can be used for Statistical Performance of TE Quantification Methods.All scripts needed to examine the sensitivity and biases of computational approaches for quantifying TE expression: 1) unique counts, 2) best counts, 3) RepEnrich, 4) TEtranscripts, 5) RSEM, 6) SalmonTE, and 7) Telescope.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/mlbendall/telescope/</td>
      <td>Genome annotation</td>
      <td>telescope_assign</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/telescope</td>
    </tr>
    <tr>
      <td></td>
      <td>tetoolkit</td>
      <td>2.2.3</td>
      <td>2.2.3</td>
      <td>tetranscripts</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>The TEToolkit suite improves the bioinformatic analysis of repetitive sequences, particularly transposable elements, in order to elucidate novel (and previously ignored) biological insights of their functions in development and diseases.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hammelllab.labsites.cshl.edu/software/</td>
      <td>Sequence Analysis</td>
      <td>tetoolkit</td>
      <td>iuc</td>
      <td>https://github.com/mhammell-laboratory/TEtranscripts</td>
    </tr>
    <tr>
      <td></td>
      <td>tetyper</td>
      <td>1.1</td>
      <td>1.1</td>
      <td>tetyper</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Type a specific transposable element (TE) of interest from paired-end sequencing data.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/aesheppard/TETyper</td>
      <td>Sequence Analysis</td>
      <td>tetyper</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/tetyper</td>
    </tr>
    <tr>
      <td></td>
      <td>tn93</td>
      <td>1.0.6</td>
      <td>1.0.13</td>
      <td>tn93</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute distances between sequences</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/veg/tn93/</td>
      <td>Sequence Analysis</td>
      <td>tn93</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/tn93/</td>
    </tr>
    <tr>
      <td></td>
      <td>tracy</td>
      <td>0.6.1</td>
      <td>0.7.6</td>
      <td>tracy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/tracy</td>
    </tr>
    <tr>
      <td></td>
      <td>transdecoder</td>
      <td>5.5.0</td>
      <td>5.7.1</td>
      <td>transdecoder</td>
      <td>To update</td>
      <td>TransDecoder</td>
      <td>TransDecoder</td>
      <td>Coding region prediction, de Novo sequencing, De-novo assembly</td>
      <td>Genomics, Gene transcripts, RNA-Seq, Gene expression, Sequence assembly, Whole genome sequencing</td>
      <td>TransDecoder finds coding regions within transcripts</td>
      <td>TransDecoder identifies candidate coding regions within transcript sequences, such as those generated by de novo RNA-Seq transcript assembly using Trinity, or constructed based on RNA-Seq alignments to the genome using Tophat and Cufflinks.</td>
      <td></td>
      <td>To update</td>
      <td>https://transdecoder.github.io/</td>
      <td>Transcriptomics, RNA</td>
      <td>transdecoder</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/transdecoder</td>
    </tr>
    <tr>
      <td></td>
      <td>transit</td>
      <td>3.0.2</td>
      <td>3.2.3</td>
      <td>transit</td>
      <td>To update</td>
      <td>transit</td>
      <td>TRANSIT</td>
      <td>Transposon prediction</td>
      <td>DNA, Sequencing, Mobile genetic elements</td>
      <td>TRANSIT</td>
      <td>A tool for the analysis of Tn-Seq data. It provides an easy to use graphical interface and access to three different analysis methods that allow the user to determine essentiality in a single condition as well as between conditions.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/mad-lab/transit/</td>
      <td>Genome annotation</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/transit/</td>
    </tr>
    <tr>
      <td></td>
      <td>transtermhp</td>
      <td></td>
      <td>2.09</td>
      <td>transtermhp</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Finds rho-independent transcription terminators in bacterial genomes</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>transtermhp</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/transtermhp</td>
    </tr>
    <tr>
      <td></td>
      <td>trimmomatic</td>
      <td>0.39</td>
      <td>0.39</td>
      <td>trimmomatic</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A flexible read trimming tool for Illumina NGS data</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://www.usadellab.org/cms/?page=trimmomatic</td>
      <td>Fastq Manipulation</td>
      <td>trimmomatic</td>
      <td>pjbriggs</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/packages/trimmomatic</td>
    </tr>
    <tr>
      <td></td>
      <td>trinity</td>
      <td>2.15.1</td>
      <td>2.15.1</td>
      <td>trinity</td>
      <td>Up-to-date</td>
      <td>trinity</td>
      <td>Trinity</td>
      <td>Transcriptome assembly</td>
      <td>Transcriptomics, Gene expression, Gene transcripts</td>
      <td>Trinity represents a method for the efficient and robust de novo reconstruction of transcriptomes from RNA-seq datahttps://github.com/trinityrnaseq/trinityrnaseq</td>
      <td>Trinity is a transcriptome assembler which relies on three different tools, inchworm an assembler, chrysalis which pools contigs and butterfly which amongst others compacts a graph resulting from butterfly with reads.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/trinityrnaseq/trinityrnaseq</td>
      <td>Transcriptomics, RNA</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/trinity</td>
    </tr>
    <tr>
      <td></td>
      <td>trinotate</td>
      <td>3.2.2</td>
      <td>4.0.2</td>
      <td>trinotate</td>
      <td>To update</td>
      <td>trinotate</td>
      <td>Trinotate</td>
      <td>Gene functional annotation</td>
      <td>Gene expression, Transcriptomics</td>
      <td>Trinotate is a comprehensive annotation suite designed for automatic functional annotation of de novo transcriptomes.</td>
      <td>Comprehensive annotation suite designed for automatic functional annotation of transcriptomes, particularly de novo assembled transcriptomes, from model or non-model organisms.</td>
      <td></td>
      <td>To update</td>
      <td>https://trinotate.github.io/</td>
      <td>Transcriptomics, RNA</td>
      <td>trinotate</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/trinotate</td>
    </tr>
    <tr>
      <td></td>
      <td>trycycler</td>
      <td>0.5.4</td>
      <td>0.5.4</td>
      <td>trycycler</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Trycycler toolkit wrappers</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/rrwick/Trycycler</td>
      <td>Assembly</td>
      <td>trycycler</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/trycycler</td>
    </tr>
    <tr>
      <td></td>
      <td>tsebra</td>
      <td>1.1.2.3</td>
      <td>1.1.2.3</td>
      <td>tsebra</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>This tool has been developed to combine BRAKER predictions.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/Gaius-Augustus/TSEBRA</td>
      <td>Genome annotation</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/tsebra</td>
    </tr>
    <tr>
      <td></td>
      <td>tsne</td>
      <td>0.0.2</td>
      <td>0.13</td>
      <td>r-rtsne</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>T-Distributed Stochastic Neighbor Embedding using a Barnes-Hut Implementation</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://cran.r-project.org/web/packages/Rtsne/</td>
      <td>Text Manipulation</td>
      <td>tsne</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/tsne</td>
    </tr>
    <tr>
      <td></td>
      <td>tximport</td>
      <td>1.22.0</td>
      <td>1.30.0</td>
      <td>bioconductor-tximport</td>
      <td>To update</td>
      <td>tximport</td>
      <td>tximport</td>
      <td>Pathway or network analysis, Formatting, RNA-Seq analysis</td>
      <td>Transcriptomics, Gene transcripts, Workflows</td>
      <td>Wrapper for the Bioconductor package tximport</td>
      <td>An R/Bioconductor package that imports transcript-level abundance, estimated counts and transcript lengths, and summarizes into matrices for use with downstream gene-level analysis packages.</td>
      <td></td>
      <td>To update</td>
      <td>http://bioconductor.org/packages/tximport/</td>
      <td>Transcriptomics</td>
      <td>tximport</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/tximport</td>
    </tr>
    <tr>
      <td></td>
      <td>ucsc_blat</td>
      <td>377</td>
      <td>445</td>
      <td>ucsc-blat</td>
      <td>To update</td>
      <td>blat</td>
      <td>BLAT</td>
      <td>Sequence alignment</td>
      <td>Sequence analysis</td>
      <td>Standalone blat sequence search command line tool</td>
      <td>Fast, accurate spliced alignment of DNA sequences.</td>
      <td></td>
      <td>To update</td>
      <td>http://genome.ucsc.edu/goldenPath/help/blatSpec.html</td>
      <td>Sequence Analysis</td>
      <td>ucsc_blat</td>
      <td>yating-l</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>fasplit</td>
      <td>377</td>
      <td>377</td>
      <td>ucsc-fasplit</td>
      <td>Up-to-date</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td>UCSC Genome Browser Utilities</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>faSplit is a tool to split a single FASTA file into several files</td>
      <td>Utilities for handling sequences and assemblies from the UCSC Genome Browser project.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hgdownload.cse.ucsc.edu/admin/exe/</td>
      <td>Fasta Manipulation</td>
      <td>ucsc_fasplit</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ucsc-tools/fasplit</td>
    </tr>
    <tr>
      <td></td>
      <td>fatovcf</td>
      <td>448</td>
      <td>448</td>
      <td>ucsc-fatovcf</td>
      <td>Up-to-date</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td>UCSC Genome Browser Utilities</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>Convert a FASTA alignment file to Variant Call Format (VCF) single-nucleotide diffs</td>
      <td>Utilities for handling sequences and assemblies from the UCSC Genome Browser project.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hgdownload.cse.ucsc.edu/admin/exe/</td>
      <td>Convert Formats</td>
      <td>ucsc_fatovcf</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ucsc-tools/fatovcf</td>
    </tr>
    <tr>
      <td></td>
      <td>twobittofa</td>
      <td>377</td>
      <td>455</td>
      <td>ucsc-twobittofa</td>
      <td>To update</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td>UCSC Genome Browser Utilities</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>twoBitToFa is a tool to convert all or part of .2bit file to FASTA</td>
      <td>Utilities for handling sequences and assemblies from the UCSC Genome Browser project.</td>
      <td></td>
      <td>To update</td>
      <td>https://genome.ucsc.edu/goldenpath/help/twoBit.html</td>
      <td>Convert Formats</td>
      <td>ucsc_twobittofa</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ucsc-tools/twobittofa</td>
    </tr>
    <tr>
      <td></td>
      <td>wigtobigwig</td>
      <td>447</td>
      <td>447</td>
      <td>ucsc-wigtobigwig</td>
      <td>Up-to-date</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td>UCSC Genome Browser Utilities</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>converts bedGraph (wig) files into binary bigwig</td>
      <td>Utilities for handling sequences and assemblies from the UCSC Genome Browser project.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://genome.ucsc.edu/goldenPath/help/bigWig.html</td>
      <td>Convert Formats</td>
      <td>ucsc_wigtobigwig</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/wigtobigwig</td>
    </tr>
    <tr>
      <td></td>
      <td>umi_tools</td>
      <td>1.1.2</td>
      <td>1.1.5</td>
      <td>umi_tools</td>
      <td>To update</td>
      <td>umi-tools</td>
      <td>UMI-tools</td>
      <td>Sequencing quality control</td>
      <td>NGS, Sequence sites, features and motifs, Quality affairs</td>
      <td>UMI-tools extract - Extract UMIs from fastq</td>
      <td>Tools for handling Unique Molecular Identifiers in NGS data sets.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/CGATOxford/UMI-tools</td>
      <td>Sequence Analysis, Transcriptomics</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/umi_tools</td>
    </tr>
    <tr>
      <td></td>
      <td>unicycler</td>
      <td>0.5.0</td>
      <td>0.5.0</td>
      <td>unicycler</td>
      <td>Up-to-date</td>
      <td>unicycler</td>
      <td>Unicycler</td>
      <td>Genome assembly, Aggregation</td>
      <td>Microbiology, Genomics, Sequencing, Sequence assembly</td>
      <td>Unicycler is a hybrid assembly pipeline for bacterial genomes.</td>
      <td>A tool for assembling bacterial genomes from a combination of short (2nd generation) and long (3rd generation) sequencing reads.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/rrwick/Unicycler</td>
      <td>Assembly</td>
      <td>unicycler</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/unicycler</td>
    </tr>
    <tr>
      <td></td>
      <td>usher</td>
      <td>0.2.1</td>
      <td>0.6.3</td>
      <td>usher</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>UShER toolkit wrappers</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/yatisht/usher</td>
      <td>Phylogenetics</td>
      <td>usher</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/usher</td>
    </tr>
    <tr>
      <td></td>
      <td>valet</td>
      <td></td>
      <td>1.0</td>
      <td>valet</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A pipeline for detecting mis-assemblies in metagenomic assemblies.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/marbl/VALET</td>
      <td>Metagenomics</td>
      <td>valet</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/valet</td>
    </tr>
    <tr>
      <td></td>
      <td>vapor</td>
      <td>1.0.2</td>
      <td>1.0.2</td>
      <td>vapor</td>
      <td>Up-to-date</td>
      <td>vapor</td>
      <td>VAPOR</td>
      <td>Data retrieval, De-novo assembly, Read mapping</td>
      <td>Whole genome sequencing, Mapping, Sequence assembly</td>
      <td>Classify Influenza samples from raw short read sequence data</td>
      <td>VAPOR is a tool for classification of Influenza samples from raw short read sequence data for downstream bioinformatics analysis. VAPOR is provided with a fasta file of full-length sequences (&gt; 20,000) for a given segment, a set of reads, and attempts to retrieve a reference that is closest to the sample strain.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/connor-lab/vapor</td>
      <td>Sequence Analysis</td>
      <td>vapor</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/vapor</td>
    </tr>
    <tr>
      <td></td>
      <td>vardict</td>
      <td>1.8.3</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>VarDict - calls SNVs and indels for tumour-normal pairs</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/AstraZeneca-NGS/VarDictJava</td>
      <td>Variant Analysis</td>
      <td>vardict_java</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/vardict</td>
    </tr>
    <tr>
      <td></td>
      <td>variant_analyzer</td>
      <td>2.0.0</td>
      <td></td>
      <td>matplotlib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Collection of tools for analyzing variants in duplex consensus sequencing (DCS) data</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Variant Analysis</td>
      <td>variant_analyzer</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/variant_analyzer</td>
    </tr>
    <tr>
      <td></td>
      <td>varscan</td>
      <td>2.4.3</td>
      <td>2.4.6</td>
      <td>varscan</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>VarScan is a variant caller for high-throughput sequencing data</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://dkoboldt.github.io/varscan/</td>
      <td>Variant Analysis</td>
      <td>varscan</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/iuc/tree/master/tools/varscan</td>
    </tr>
    <tr>
      <td></td>
      <td>varvamp</td>
      <td>1.1.1</td>
      <td>1.1.1</td>
      <td>varvamp</td>
      <td>Up-to-date</td>
      <td>varvamp</td>
      <td>varVAMP</td>
      <td>PCR primer design</td>
      <td>Virology</td>
      <td>Variable VirusAMPlicons (varVAMP) is a tool to design primers for highly diverse viruses</td>
      <td>variable VirusAMPlicons (varVAMP) is a tool to design primers for highly diverse viruses. The input is an alignment of your viral (full-genome) sequences.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/jonas-fuchs/varVAMP/</td>
      <td>Sequence Analysis</td>
      <td>varvamp</td>
      <td>iuc</td>
      <td>https://github.com/jonas-fuchs/varVAMP</td>
    </tr>
    <tr>
      <td></td>
      <td>vcf2maf</td>
      <td>1.6.21</td>
      <td>1.6.21</td>
      <td>vcf2maf</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>vcf2maf: Convert VCF into MAF</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td></td>
      <td>Convert Formats</td>
      <td>vcf2maf</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/vcf2maf</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfanno</td>
      <td>0.3.5</td>
      <td>0.3.5</td>
      <td>vcfanno</td>
      <td>Up-to-date</td>
      <td>vcfanno</td>
      <td>vcfanno</td>
      <td>SNP annotation</td>
      <td>Genetic variation, Data submission, annotation and curation</td>
      <td>Annotate VCF files</td>
      <td>Fast, flexible annotation of genetic variants.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/brentp/vcfanno</td>
      <td>Variant Analysis</td>
      <td>vcfanno</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/vcfanno/</td>
    </tr>
    <tr>
      <td></td>
      <td>vegan</td>
      <td>2.4-3</td>
      <td>2.3_4</td>
      <td>r-vegan</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://cran.r-project.org/package=vegan</td>
      <td>Metagenomics</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/vegan/</td>
    </tr>
    <tr>
      <td></td>
      <td>velocyto</td>
      <td>0.17.17</td>
      <td>0.17.17</td>
      <td>velocyto.py</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Velocyto is a library for the analysis of RNA velocity.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://velocyto.org/</td>
      <td>Transcriptomics</td>
      <td>velocyto</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/velocyto</td>
    </tr>
    <tr>
      <td></td>
      <td>velvet</td>
      <td></td>
      <td>1.2.10</td>
      <td>velvet</td>
      <td>To update</td>
      <td>velvet</td>
      <td>Velvet</td>
      <td>Formatting, De-novo assembly</td>
      <td>Sequence assembly</td>
      <td>de novo genomic assembler specially designed for short read sequencing technologies</td>
      <td>A de novo genomic assembler specially designed for short read sequencing technologies, such as Solexa or 454 or SOLiD.</td>
      <td></td>
      <td>To update</td>
      <td>https://www.ebi.ac.uk/~zerbino/velvet/</td>
      <td>Assembly</td>
      <td>velvet</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/velvet</td>
    </tr>
    <tr>
      <td></td>
      <td>velvet_optimiser</td>
      <td>2.2.6+galaxy2</td>
      <td>1.2.10</td>
      <td>velvet</td>
      <td>To update</td>
      <td>velvetoptimiser</td>
      <td>VelvetOptimiser</td>
      <td>Optimisation and refinement, Sequence assembly</td>
      <td>Genomics, Sequence assembly</td>
      <td>Automatically optimize Velvet assemblies</td>
      <td>This tool is designed to run as a wrapper script for the Velvet assembler (Daniel Zerbino, EBI UK) and to assist with optimising the assembly.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Assembly</td>
      <td>velvetoptimiser</td>
      <td>simon-gladman</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/velvetoptimiser</td>
    </tr>
    <tr>
      <td></td>
      <td>verkko</td>
      <td>1.3.1</td>
      <td>2.0</td>
      <td>verkko</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Telomere-to-telomere assembly pipeline</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/marbl/verkko</td>
      <td>Assembly</td>
      <td>verkko</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/verkko</td>
    </tr>
    <tr>
      <td></td>
      <td>vg</td>
      <td>1.23.0</td>
      <td>1.55.0</td>
      <td>vg</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Variation graph data structures, interchange formats, alignment, genotyping, and variant calling methods</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/vgteam/vg</td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/vg</td>
    </tr>
    <tr>
      <td></td>
      <td>virhunter</td>
      <td>1.0.0</td>
      <td></td>
      <td>numpy</td>
      <td>To update</td>
      <td>virhunter</td>
      <td>VirHunter</td>
      <td>Sequence classification</td>
      <td>Virology</td>
      <td>Deep Learning method for novel virus detection in sequencing data</td>
      <td>VirHunter is a deep learning method that uses Convolutional Neural Networks (CNNs) and a Random Forest Classifier to identify viruses in sequencing datasets. More precisely, VirHunter classifies previously assembled contigs as viral, host, and bacterial (contamination).</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/cbib/virhunter</td>
      <td>Machine Learning</td>
      <td>virhunter</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/VirHunter</td>
    </tr>
    <tr>
      <td></td>
      <td>volcanoplot</td>
      <td>0.0.5</td>
      <td>2.2.1</td>
      <td>r-ggplot2</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tool to create a Volcano Plot</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Visualization, Transcriptomics, Statistics</td>
      <td>volcanoplot</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/volcanoplot</td>
    </tr>
    <tr>
      <td></td>
      <td>vsearch</td>
      <td>2.8.3</td>
      <td>2.27.0</td>
      <td>vsearch</td>
      <td>To update</td>
      <td>vsearch</td>
      <td>VSEARCH</td>
      <td>DNA mapping, Chimera detection</td>
      <td>Metagenomics, Sequence analysis</td>
      <td>VSEARCH including searching, clustering, chimera detection, dereplication, sorting, masking and shuffling of sequences.</td>
      <td>High-throughput search and clustering sequence analysis tool. It supports de novo and reference based chimera detection, clustering, full-length and prefix dereplication, reverse complementation, masking, all-vs-all pairwise global alignment, exact and global alignment searching, shuffling, subsampling and sorting. It also supports FASTQ file analysis, filtering and conversion.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/torognes/vsearch</td>
      <td>Sequence Analysis</td>
      <td>vsearch</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/vsearch</td>
    </tr>
    <tr>
      <td></td>
      <td>vsnp</td>
      <td>3.0.6</td>
      <td>0.22.0</td>
      <td>pysam</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>The vSNP tools are critical components of several workflows that validate SNPs and produce annotatedSNP tables and corresponding phylogenetic trees.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/USDA-VS/vSNP</td>
      <td>Sequence Analysis</td>
      <td>vsnp</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/vsnp</td>
    </tr>
    <tr>
      <td></td>
      <td>weather_app</td>
      <td>0.1.2</td>
      <td></td>
      <td>curl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>provides simple weather in text format</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://wttr.in/</td>
      <td>Visualization, Web Services</td>
      <td>simpleweather</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/weather_app</td>
    </tr>
    <tr>
      <td></td>
      <td>weblogo3</td>
      <td>3.5.0</td>
      <td>3.7.9</td>
      <td>weblogo</td>
      <td>To update</td>
      <td>weblogo</td>
      <td>WebLogo</td>
      <td>Sequence cluster visualisation, Sequence visualisation, Sequence motif recognition</td>
      <td>Nucleic acid sites, features and motifs, Sequence analysis</td>
      <td>Sequence Logo generator for fasta</td>
      <td>Web-based application designed to make generate sequence logos.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Graphics</td>
      <td>weblogo3</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/weblogo3</td>
    </tr>
    <tr>
      <td></td>
      <td>windowmasker</td>
      <td>1.0</td>
      <td>2.15.0</td>
      <td>blast</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Identify repetitive regions using WindowMasker</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.ncbi.nlm.nih.gov/IEB/ToolBox/CPP_DOC/lxr/source/src/app/winmasker/</td>
      <td>Sequence Analysis</td>
      <td>windowmasker</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/windowmasker/</td>
    </tr>
    <tr>
      <td></td>
      <td>winnowmap</td>
      <td>2.03</td>
      <td>2.03</td>
      <td>winnowmap</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A long-read mapping tool optimized for mapping ONT and PacBio reads to repetitive reference sequences.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/marbl/Winnowmap</td>
      <td>Next Gen Mappers</td>
      <td>winnowmap</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/winnowmap</td>
    </tr>
    <tr>
      <td></td>
      <td>xpath</td>
      <td></td>
      <td>1.47</td>
      <td>perl-xml-xpath</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>XPath XML querying tool</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://search.cpan.org/dist/XML-XPath/</td>
      <td>Text Manipulation</td>
      <td>xpath</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/xpath</td>
    </tr>
    <tr>
      <td></td>
      <td>yahs</td>
      <td>1.2a.2</td>
      <td>1.2a.2</td>
      <td>yahs</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Yet Another Hi-C scaffolding tool</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/c-zhou/yahs</td>
      <td>Assembly</td>
      <td>yahs</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/yahs</td>
    </tr>
    <tr>
      <td></td>
      <td>zerone</td>
      <td>1.0</td>
      <td>1.0</td>
      <td>zerone</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ChIP-seq discretization and quality control</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/nanakiksc/zerone</td>
      <td>ChIP-seq</td>
      <td>zerone</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/zerone</td>
    </tr>
    <tr>
      <td></td>
      <td>bamtools</td>
      <td>2.5.2</td>
      <td>2.5.2</td>
      <td>bamtools</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Operate on and transform BAM datasets in various ways using bamtools</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/pezmaster31/bamtools</td>
      <td>Sequence Analysis, SAM</td>
      <td>bamtools</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/bamtools/bamtools</td>
    </tr>
    <tr>
      <td></td>
      <td>bamtools_filter</td>
      <td>2.5.2</td>
      <td>2.5.2</td>
      <td>bamtools</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter BAM datasets on various attributes using bamtools filter</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/pezmaster31/bamtools</td>
      <td>Sequence Analysis, SAM</td>
      <td>bamtools_filter</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/bamtools/bamtools_filter</td>
    </tr>
    <tr>
      <td></td>
      <td>bamtools_split</td>
      <td>2.5.2</td>
      <td>2.5.2</td>
      <td>bamtools</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/pezmaster31/bamtools</td>
      <td>Sequence Analysis, SAM</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/bamtools/bamtools_split</td>
    </tr>
    <tr>
      <td></td>
      <td>biotradis</td>
      <td>1.4.5</td>
      <td>1.4.5</td>
      <td>biotradis</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Bio-Tradis is a tool suite dedicated to essentiality analyses with TraDis data.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://www.sanger.ac.uk/science/tools/bio-tradis</td>
      <td>Genome annotation</td>
      <td>biotradis</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/biotradis</td>
    </tr>
    <tr>
      <td></td>
      <td>cuffcompare</td>
      <td>2.2.1</td>
      <td>2.2.1</td>
      <td>cufflinks</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Galaxy wrappers for the Cuffcompare tool.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://cole-trapnell-lab.github.io/cufflinks/</td>
      <td>Transcriptomics</td>
      <td>cuffcompare</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/cufflinks/cuffcompare</td>
    </tr>
    <tr>
      <td></td>
      <td>cuffdiff</td>
      <td>2.2.1</td>
      <td>2.2.1</td>
      <td>cufflinks</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Galaxy wrappers for the Cuffdiff tool.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://cole-trapnell-lab.github.io/cufflinks/</td>
      <td>Transcriptomics</td>
      <td>cuffdiff</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/cufflinks/cuffdiff</td>
    </tr>
    <tr>
      <td></td>
      <td>cufflinks</td>
      <td>2.2.1</td>
      <td>2.2.1</td>
      <td>cufflinks</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Galaxy wrappers for the Cufflinks tool.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://cole-trapnell-lab.github.io/cufflinks/</td>
      <td>Transcriptomics</td>
      <td>cufflinks</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/cufflinks/cufflinks</td>
    </tr>
    <tr>
      <td></td>
      <td>cuffmerge</td>
      <td>2.2.1</td>
      <td>2.2.1</td>
      <td>cufflinks</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Galaxy wrappers for the Cuffmerge tool.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://cole-trapnell-lab.github.io/cufflinks/</td>
      <td>Transcriptomics</td>
      <td>cuffmerge</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/cufflinks/cuffmerge</td>
    </tr>
    <tr>
      <td></td>
      <td>cuffnorm</td>
      <td>2.2.1</td>
      <td>2.2.1</td>
      <td>cufflinks</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>The Cuffnorm tool</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://cole-trapnell-lab.github.io/cufflinks/</td>
      <td>Transcriptomics</td>
      <td>cuffnorm</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/cufflinks/cuffnorm</td>
    </tr>
    <tr>
      <td></td>
      <td>cuffquant</td>
      <td>2.2.1</td>
      <td>2.2.1</td>
      <td>cufflinks</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>The Cuffquant tool</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://cole-trapnell-lab.github.io/cufflinks/</td>
      <td>Transcriptomics</td>
      <td>cuffquant</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/cufflinks/cuffquant</td>
    </tr>
    <tr>
      <td></td>
      <td>fasta_clipping_histogram</td>
      <td>0.0.14</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Length Distribution chart</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hannonlab.cshl.edu/fastx_toolkit/</td>
      <td>Fasta Manipulation, Graphics, Statistics</td>
      <td>fasta_clipping_histogram</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/fastx_toolkit/fasta_clipping_histogram</td>
    </tr>
    <tr>
      <td></td>
      <td>fasta_formatter</td>
      <td>0.0.14</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FASTA Width formatter</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hannonlab.cshl.edu/fastx_toolkit/</td>
      <td>Fasta Manipulation</td>
      <td>fasta_formatter</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/fastx_toolkit/fasta_formatter</td>
    </tr>
    <tr>
      <td></td>
      <td>fasta_nucleotide_changer</td>
      <td>0.0.14</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>RNA/DNA converter.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hannonlab.cshl.edu/fastx_toolkit/</td>
      <td>Fasta Manipulation</td>
      <td>fasta_nucleotide_changer</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/fastx_toolkit/fasta_nucleotide_changer</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_quality_boxplot</td>
      <td>0.0.14</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Draw quality score boxplot</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hannonlab.cshl.edu/fastx_toolkit/</td>
      <td>Fastq Manipulation, Graphics, Statistics</td>
      <td>fastq_quality_boxplot</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/fastx_toolkit/fastq_quality_boxplot</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_quality_converter</td>
      <td>0.0.14</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Quality format converter (ASCII-Numeric)</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hannonlab.cshl.edu/fastx_toolkit/</td>
      <td>Fastq Manipulation</td>
      <td>fastq_quality_converter</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/fastx_toolkit/fastq_quality_converter</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_quality_filter</td>
      <td>0.0.14</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter by quality</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hannonlab.cshl.edu/fastx_toolkit/</td>
      <td>Fastq Manipulation</td>
      <td>fastq_quality_filter</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/fastx_toolkit/fastq_quality_filter</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_to_fasta</td>
      <td>0.0.14</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FASTQ to FASTA converter</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hannonlab.cshl.edu/fastx_toolkit/</td>
      <td>Fasta Manipulation, Convert Formats</td>
      <td>fastq_to_fasta</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/fastx_toolkit/fastq_to_fasta</td>
    </tr>
    <tr>
      <td></td>
      <td>fastx_artifacts_filter</td>
      <td>0.0.14</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Remove sequencing artifacts</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hannonlab.cshl.edu/fastx_toolkit/</td>
      <td>Fasta Manipulation, Fastq Manipulation</td>
      <td>fastx_artifacts_filter</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/fastx_toolkit/fastx_artifacts_filter</td>
    </tr>
    <tr>
      <td></td>
      <td>fastx_barcode_splitter</td>
      <td>0.0.14</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Barcode Splitter</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hannonlab.cshl.edu/fastx_toolkit/</td>
      <td>Fasta Manipulation, Fastq Manipulation</td>
      <td>fastx_barcode_splitter</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/fastx_toolkit/fastx_barcode_splitter</td>
    </tr>
    <tr>
      <td></td>
      <td>fastx_clipper</td>
      <td>0.0.14</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Clip adapter sequences</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hannonlab.cshl.edu/fastx_toolkit/</td>
      <td>Fasta Manipulation, Fastq Manipulation</td>
      <td>fastx_clipper</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/fastx_toolkit/fastx_clipper</td>
    </tr>
    <tr>
      <td></td>
      <td>fastx_collapser</td>
      <td>0.0.14</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Collapse sequences</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hannonlab.cshl.edu/fastx_toolkit/</td>
      <td>Fasta Manipulation</td>
      <td>fastx_collapser</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/fastx_toolkit/fastx_collapser</td>
    </tr>
    <tr>
      <td></td>
      <td>fastx_nucleotides_distribution</td>
      <td>0.0.14</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Draw nucleotides distribution chart</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hannonlab.cshl.edu/fastx_toolkit/</td>
      <td>Fastq Manipulation, Graphics</td>
      <td>fastx_nucleotides_distribution</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/fastx_toolkit/fastx_nucleotides_distribution</td>
    </tr>
    <tr>
      <td></td>
      <td>fastx_quality_statistics</td>
      <td>0.0.14</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute quality statistics</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hannonlab.cshl.edu/fastx_toolkit/</td>
      <td>Fastq Manipulation, Statistics</td>
      <td>fastx_quality_statistics</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/fastx_toolkit/fastx_quality_statistics</td>
    </tr>
    <tr>
      <td></td>
      <td>fastx_renamer</td>
      <td>0.0.14</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Rename sequences</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hannonlab.cshl.edu/fastx_toolkit/</td>
      <td>Fasta Manipulation, Fastq Manipulation</td>
      <td>fastx_renamer</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/fastx_toolkit/fastx_renamer</td>
    </tr>
    <tr>
      <td></td>
      <td>fastx_reverse_complement</td>
      <td>0.0.14</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Reverse-Complement</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hannonlab.cshl.edu/fastx_toolkit/</td>
      <td>Fastq Manipulation, Fasta Manipulation</td>
      <td>fastx_reverse_complement</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/fastx_toolkit/fastx_reverse_complement</td>
    </tr>
    <tr>
      <td></td>
      <td>fastx_trimmer</td>
      <td>0.0.14</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Trim sequences</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hannonlab.cshl.edu/fastx_toolkit/</td>
      <td>Fasta Manipulation, Fastq Manipulation</td>
      <td>fastx_trimmer</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/fastx_toolkit/fastx_trimmer</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_combiner</td>
      <td>1.1.5</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Combine FASTA and QUAL into FASTQ</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation, Fasta Manipulation</td>
      <td>fastq_combiner</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_combiner</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_filter</td>
      <td>1.1.5</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter FASTQ reads by quality score and length</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_filter</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_filter</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_groomer</td>
      <td>1.1.5</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert between various FASTQ quality formats.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_groomer</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_groomer</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_manipulation</td>
      <td>1.1.5</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Manipulate FASTQ reads on various attributes.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_manipulation</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_manipulation</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_masker_by_quality</td>
      <td>1.1.5</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FASTQ Masker by quality score</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_masker_by_quality</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_masker_by_quality</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_paired_end_deinterlacer</td>
      <td>1.1.5</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FASTQ de-interlacer on paired end reads.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_paired_end_deinterlacer</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_paired_end_deinterlacer</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_paired_end_interlacer</td>
      <td>1.1.5</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FASTQ interlacer on paired end reads</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_paired_end_interlacer</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_paired_end_interlacer</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_paired_end_joiner</td>
      <td>1.1.5</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FASTQ joiner on paired end reads</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_paired_end_joiner</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_paired_end_joiner</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_paired_end_splitter</td>
      <td>1.1.5</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FASTQ splitter on joined paired end reads</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_paired_end_splitter</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_paired_end_splitter</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_stats</td>
      <td>1.1.5</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FASTQ Summary Statistics by column</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_stats</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_stats</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_to_tabular</td>
      <td>1.1.5</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FASTQ to Tabular converter</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_to_tabular</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_to_tabular</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_trimmer</td>
      <td>1.1.5</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FASTQ Trimmer by quality</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_trimmer</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_trimmer</td>
    </tr>
    <tr>
      <td></td>
      <td>fastqtofasta</td>
      <td>1.1.5</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FASTQ to FASTA converter</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastqtofasta</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastqtofasta</td>
    </tr>
    <tr>
      <td></td>
      <td>tabular_to_fastq</td>
      <td>1.1.5</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tabular to FASTQ converter</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>tabular_to_fastq</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/tabular_to_fastq</td>
    </tr>
    <tr>
      <td></td>
      <td>kraken</td>
      <td></td>
      <td>1.1.1</td>
      <td>kraken</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Kraken is a system for assigning taxonomic labels to short DNAsequences, usually obtained through metagenomic studies. Previous attempts by otherbioinformatics software to accomplish this task have often used sequence alignmentor machine learning techniques that were quite slow, leading to the developmentof less sensitive but much faster abundance estimation programs. Kraken aims toachieve high sensitivity and high speed by utilizing exact alignments of k-mersand a novel classification algorithm.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://ccb.jhu.edu/software/kraken/</td>
      <td>Metagenomics</td>
      <td>kraken</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tool_collections/kraken/</td>
    </tr>
    <tr>
      <td></td>
      <td>kraken2</td>
      <td>2.1.1</td>
      <td>2.1.3</td>
      <td>kraken2</td>
      <td>To update</td>
      <td>kraken2</td>
      <td>kraken2</td>
      <td>Taxonomic classification</td>
      <td>Taxonomy, Metagenomics</td>
      <td>Kraken2 for taxonomic designation.</td>
      <td>Kraken 2 is the newest version of Kraken, a taxonomic classification system using exact k-mer matches to achieve high accuracy and fast classification speeds. This classifier matches each k-mer within a query sequence to the lowest common ancestor (LCA) of all genomes containing the given k-mer. The k-mer assignments inform the classification algorithm.</td>
      <td></td>
      <td>To update</td>
      <td>http://ccb.jhu.edu/software/kraken/</td>
      <td>Metagenomics</td>
      <td>kraken2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tool_collections/kraken2/kraken2/</td>
    </tr>
    <tr>
      <td></td>
      <td>samtools</td>
      <td>1.15.1</td>
      <td>1.19.2</td>
      <td>samtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/samtools/samtools</td>
      <td>SAM</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/samtools</td>
    </tr>
    <tr>
      <td></td>
      <td>snpeff</td>
      <td></td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SnpEff is a genetic variant annotation and effect prediction toolbox</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://snpeff.sourceforge.net/</td>
      <td>Genome-Wide Association Study, Variant Analysis</td>
      <td>snpeff</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/snpeff</td>
    </tr>
    <tr>
      <td></td>
      <td>snpsift</td>
      <td></td>
      <td>5.2</td>
      <td>snpsift</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>snpEff SnpSift tools from Pablo Cingolani</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://snpeff.sourceforge.net/SnpSift.html</td>
      <td>Variant Analysis</td>
      <td>snpsift</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/snpsift/snpsift</td>
    </tr>
    <tr>
      <td></td>
      <td>snpsift_dbnsfp</td>
      <td></td>
      <td>5.2</td>
      <td>snpsift</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>snpEff SnpSift dbnsfp tool from Pablo Cingolani</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://snpeff.sourceforge.net/SnpSift.html#dbNSFP</td>
      <td>Variant Analysis</td>
      <td>snpsift_dbnsfp</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/snpsift/snpsift_dbnsfp</td>
    </tr>
    <tr>
      <td></td>
      <td>snpsift_genesets</td>
      <td></td>
      <td>5.2</td>
      <td>snpsift</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Annotate SnpEff vcf with genesets such as Gene Ontology (GO), KEGG, Reactome</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://snpeff.sourceforge.net/SnpSift.html#geneSets</td>
      <td>Variant Analysis</td>
      <td>snpsift_genesets</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/snpsift/snpsift_genesets/</td>
    </tr>
    <tr>
      <td></td>
      <td>vcf2tsv</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Converts VCF files into tab-delimited format</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis, Convert Formats</td>
      <td>vcf2tsv</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcf2tsv</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfaddinfo</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Adds info fields from the second dataset which are not present in the first dataset.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfaddinfo</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfaddinfo</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfallelicprimitives</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Splits alleleic primitives (gaps or mismatches) into multiple VCF lines</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfallelicprimitives</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfallelicprimitives</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfannotate</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Intersect VCF records with BED annotations</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfannotate</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfannotate</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfannotategenotypes</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Annotate genotypes in a VCF dataset using genotypes from another VCF dataset.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfannotategenotypes</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfannotategenotypes</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfbedintersect</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Intersect VCF and BED datasets</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfbedintersect</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfbedintersect</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfbreakcreatemulti</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Break multiple alleles into multiple records, or combine overallpoing alleles into a single record</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfbreakcreatemulti</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfbreakcreatemulti</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfcheck</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Verify that the reference allele matches the reference genome</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfcheck</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfcheck</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfcombine</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Combine multiple VCF datasets</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfcombine</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfcombine</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfcommonsamples</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Output records belonging to samples commong between two datasets.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfcommonsamples</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfcommonsamples</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfdistance</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Calculate distance to the nearest variant.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfdistance</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfdistance</td>
    </tr>
    <tr>
      <td></td>
      <td>vcffilter</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tool for filtering VCF files</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcffilter</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcffilter</td>
    </tr>
    <tr>
      <td></td>
      <td>vcffixup</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Count the allele frequencies across alleles present in each record in the VCF file.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcffixup</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcffixup</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfflatten</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Removes multi-allelic sites by picking the most common alternate</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfflatten</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfflatten</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfgeno2haplo</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert genotype-based phased alleles into haplotype alleles</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfgeno2haplo</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfgeno2haplo</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfgenotypes</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert numerical representation of genotypes to allelic.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfgenotypes</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfgenotypes</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfhethom</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Count the number of heterozygotes and alleles, compute het/hom ratio.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfhethom</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfhethom</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfleftalign</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Left-align indels and complex variants in VCF dataset</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfleftalign</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfleftalign</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfprimers</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Extract flanking sequences for each VCF record</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfprimers</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfprimers</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfrandomsample</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Randomly sample sites from VCF dataset</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfrandomsample</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfrandomsample</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfselectsamples</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Select samples from a VCF file</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfselectsamples</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfselectsamples</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfsort</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Sort VCF dataset by coordinate</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfsort</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfsort</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfvcfintersect</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Intersect two VCF datasets</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfvcfintersect</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfvcfintersect</td>
    </tr>
    <tr>
      <td></td>
      <td>ThermoRawFileParser</td>
      <td>1.3.4</td>
      <td>1.4.3</td>
      <td>thermorawfileparser</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Thermo RAW file converter</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/compomics/ThermoRawFileParser</td>
      <td>Proteomics</td>
      <td>thermo_raw_file_converter</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/ThermoRawFileParser</td>
    </tr>
    <tr>
      <td></td>
      <td>appendfdr</td>
      <td>0.2.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td>appendfdr</td>
      <td>galaxyp</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>bed_to_protein_map</td>
      <td>0.2.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Converts a BED file to a tabular list of exon locations</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>bed_to_protein_map</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/bed_to_protein_map</td>
    </tr>
    <tr>
      <td></td>
      <td>blast_plus_remote_blastp</td>
      <td>2.6.0</td>
      <td>2.15.0</td>
      <td>blast</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>NCBI BLAST+ with -remote option</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://blast.ncbi.nlm.nih.gov/</td>
      <td>Sequence Analysis</td>
      <td>blast_plus_remote_blastp</td>
      <td>galaxyp</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/ncbi_blast_plus</td>
    </tr>
    <tr>
      <td></td>
      <td>bumbershoot</td>
      <td>3.0.21142</td>
      <td>3_0_21142_0e4f4a4</td>
      <td>bumbershoot</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://proteowizard.sourceforge.net/</td>
      <td>Proteomics</td>
      <td></td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tools/bumbershoot</td>
    </tr>
    <tr>
      <td></td>
      <td>calisp</td>
      <td>3.0.13</td>
      <td>3.0.13</td>
      <td>calisp</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Calgary approach to isotopes in proteomics</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/kinestetika/Calisp/</td>
      <td>Proteomics</td>
      <td>calisp</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tools/calisp</td>
    </tr>
    <tr>
      <td></td>
      <td>cardinal</td>
      <td>2.10.0</td>
      <td>3.4.3</td>
      <td>bioconductor-cardinal</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Statistical and computational tools for analyzing mass spectrometry imaging datasets</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://cardinalmsi.org</td>
      <td>Proteomics, Metabolomics</td>
      <td></td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/cardinal</td>
    </tr>
    <tr>
      <td></td>
      <td>dbbuilder</td>
      <td>0.3.4</td>
      <td></td>
      <td>wget</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Protein Database Downloader</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/dbbuilder</td>
      <td>Proteomics</td>
      <td>dbbuilder</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/dbbuilder</td>
    </tr>
    <tr>
      <td></td>
      <td>decoyfasta</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Galaxy tool wrapper for the transproteomic pipeline decoyFASTA tool.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>decoyfasta</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/decoyfasta</td>
    </tr>
    <tr>
      <td></td>
      <td>dia_umpire</td>
      <td>2.1.3</td>
      <td>2.1.6</td>
      <td>dia_umpire</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>DIA-Umpire analysis for data independent acquisition (DIA) mass spectrometry-based proteomics</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://diaumpire.sourceforge.net/</td>
      <td>Proteomics</td>
      <td>dia_umpire</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/dia_umpire</td>
    </tr>
    <tr>
      <td></td>
      <td>dialignr</td>
      <td>1.2.0</td>
      <td>2.10.0</td>
      <td>bioconductor-dialignr</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>DIAlignR is an R package for retention time alignment of targeted mass spectrometric data, including DIA and SWATH-MS data. This tool works with MS2 chromatograms directly and uses dynamic programming for alignment of raw chromatographic traces. DIAlignR uses a hybrid approach of global (feature-based) and local (raw data-based) alignment to establish correspondence between peaks.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/shubham1637/DIAlignR</td>
      <td>Proteomics</td>
      <td>dialignr</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/dialignr</td>
    </tr>
    <tr>
      <td></td>
      <td>diann</td>
      <td>1.8.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>DiaNN (DIA-based Neural Networks) is a software for DIA/SWATH data processing.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/vdemichev/DiaNN</td>
      <td>Proteomics</td>
      <td>diann</td>
      <td>galaxyp</td>
      <td>https://github.com/vdemichev/DiaNN</td>
    </tr>
    <tr>
      <td></td>
      <td>diapysef</td>
      <td>0.3.5.0</td>
      <td>1.0.10</td>
      <td>diapysef</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>diapysef is a convenience package for working with DIA-PASEF data</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://pypi.org/project/diapysef/</td>
      <td>Proteomics</td>
      <td>diapysef</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/diapysef</td>
    </tr>
    <tr>
      <td></td>
      <td>diffacto</td>
      <td>1.0.6</td>
      <td>1.0.6</td>
      <td>diffacto</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Diffacto comparative protein abundance estimation</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/statisticalbiotechnology/diffacto</td>
      <td>Proteomics</td>
      <td>diffacto</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/diffacto</td>
    </tr>
    <tr>
      <td></td>
      <td>digestdb</td>
      <td>0.1.0</td>
      <td></td>
      <td>trans_proteomic_pipeline</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td>digestdb</td>
      <td>galaxyp</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>directag_and_tagrecon</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td>directag_and_tagrecon</td>
      <td>galaxyp</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>data_manager_eggnog_mapper</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>downloads eggnog data for eggnog-mapper</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>data_manager_eggnog_mapper</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/eggnog_mapper/eggnog_mapper_data_manager</td>
    </tr>
    <tr>
      <td></td>
      <td>data_manager_eggnog_mapper_abspath</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>download eggnog data for eggnog-mapper</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>data_manager_eggnog_mapper_abspath</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/eggnog_mapper/data_manager_eggnog_mapper_abspath</td>
    </tr>
    <tr>
      <td></td>
      <td>eggnog_mapper</td>
      <td>2.1.8</td>
      <td>2.1.12</td>
      <td>eggnog-mapper</td>
      <td>To update</td>
      <td>eggnog-mapper-v2</td>
      <td>eggNOG-mapper v2</td>
      <td>Homology-based gene prediction, Genome annotation, Fold recognition, Information extraction, Query and retrieval</td>
      <td>Metagenomics, Phylogeny, Transcriptomics, Workflows, Sequence analysis</td>
      <td>eggnog-mapper fast functional annotation of novel sequences</td>
      <td>EggNOG-mapper is a tool for fast functional annotation of novel sequences. It uses precomputed orthologous groups and phylogenies from the eggNOG database (http://eggnog5.embl.de) to transfer functional information from fine-grained orthologs only.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>eggnog_mapper</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/eggnog_mapper/eggnog_mapper</td>
    </tr>
    <tr>
      <td></td>
      <td>encyclopedia</td>
      <td>1.12.34</td>
      <td>2.12.30</td>
      <td>encyclopedia</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Mass Spec Data-Independent Acquisition (DIA) MS/MS analysis</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/searleb/encyclopedia/wiki/Home</td>
      <td>Proteomics</td>
      <td>encyclopedia</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/encyclopedia/tools/encyclopedia</td>
    </tr>
    <tr>
      <td></td>
      <td>fasta_merge_files_and_filter_unique_sequences</td>
      <td>1.2.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Concatenate FASTA database files together</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/</td>
      <td>Fasta Manipulation</td>
      <td>fasta_merge_files_and_filter_unique_sequences</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/fasta_merge_files_and_filter_unique_sequences</td>
    </tr>
    <tr>
      <td></td>
      <td>fastg2protlib</td>
      <td>1.0.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Generate FASTA from FASTG</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproteomics/fastg2protlib.git</td>
      <td>Proteomics</td>
      <td>fastg2protlib</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/fastg2protlib</td>
    </tr>
    <tr>
      <td></td>
      <td>feature_alignment</td>
      <td>0.11.0</td>
      <td>0.11.0</td>
      <td>msproteomicstools</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>TRIC integrates information from all available runs via a graph-based alignment strategy</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td></td>
      <td>Proteomics</td>
      <td>feature_alignment</td>
      <td>galaxyp</td>
      <td>https://github.com/msproteomicstools/msproteomicstools/blob/master/TRIC-README.md</td>
    </tr>
    <tr>
      <td></td>
      <td>filter_by_fasta_ids</td>
      <td>2.3</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter FASTA on the headers and/or the sequences</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Fasta Manipulation, Proteomics</td>
      <td>filter_by_fasta_ids</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/filter_by_fasta_ids</td>
    </tr>
    <tr>
      <td></td>
      <td>flashlfq</td>
      <td>1.0.3.1</td>
      <td>1.2.6</td>
      <td>flashlfq</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FlashLFQ mass-spectrometry proteomics label-free quantification</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/smith-chem-wisc/FlashLFQ</td>
      <td>Proteomics</td>
      <td>flashlfq</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/flashlfq</td>
    </tr>
    <tr>
      <td></td>
      <td>gffcompare_to_bed</td>
      <td>0.2.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter and convert a gffCompare GTF to BED</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/gpertea/gffcompare/</td>
      <td>Convert Formats</td>
      <td>gffcompare_to_bed</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/gffcompare_to_bed</td>
    </tr>
    <tr>
      <td></td>
      <td>hardklor</td>
      <td>2.30.1+galaxy1</td>
      <td>2.3.2</td>
      <td>hardklor</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Hardklr</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>hardklor</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tools/hardklor</td>
    </tr>
    <tr>
      <td></td>
      <td>idconvert</td>
      <td></td>
      <td>3_0_9992</td>
      <td>proteowizard</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert mass spectrometry identification files on linux or MacOSX</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>idconvert</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msconvert</td>
    </tr>
    <tr>
      <td></td>
      <td>lfq_protein_quant</td>
      <td>1.0</td>
      <td>2.28.1</td>
      <td>bioconductor-msnbase</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Enable protein summarisation and quantitation</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/compomics/LFQ_galaxy_p</td>
      <td>Proteomics</td>
      <td>lfq_protein_quant</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/lfq_protein_quant</td>
    </tr>
    <tr>
      <td></td>
      <td>ltq_iquant_cli</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td>ltq_iquant_cli</td>
      <td>galaxyp</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>maldiquant</td>
      <td>1.22.0</td>
      <td></td>
      <td>r-base</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MALDIquant provides a complete analysis pipeline for MALDI-TOF and other 2D mass spectrometry data.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://strimmerlab.org/software/maldiquant/</td>
      <td>Proteomics</td>
      <td>MALDIquant</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/MALDIquant</td>
    </tr>
    <tr>
      <td></td>
      <td>map_peptides_to_bed</td>
      <td>0.2</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Map peptides to a reference genome for display by a genome browser</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>map_peptides_to_bed</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/map_peptides_to_bed</td>
    </tr>
    <tr>
      <td></td>
      <td>maxquant</td>
      <td>2.0.3.0</td>
      <td>2.0.3.0</td>
      <td>maxquant</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>wrapper for MaxQuant</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/maxquant</td>
      <td>Proteomics</td>
      <td>maxquant</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/maxquant</td>
    </tr>
    <tr>
      <td></td>
      <td>meta_proteome_analyzer</td>
      <td>2.0.0</td>
      <td>2.0.0</td>
      <td>mpa-portable</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MetaProteomeAnalyzer</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/compomics/meta-proteome-analyzer/</td>
      <td>Proteomics</td>
      <td>meta_proteome_analyzer</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/meta_proteome_analyzer</td>
    </tr>
    <tr>
      <td></td>
      <td>metagene_annotator</td>
      <td>1.0.0</td>
      <td>1.0</td>
      <td>metagene_annotator</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MetaGeneAnnotator gene-finding program for prokaryote and phage</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>metagene_annotator</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/metagene_annotator</td>
    </tr>
    <tr>
      <td></td>
      <td>metanovo</td>
      <td>1.9.4</td>
      <td>1.9.4</td>
      <td>metanovo</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Produce targeted databases for mass spectrometry analysis.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/uct-cbio/proteomics-pipelines</td>
      <td>Proteomics</td>
      <td>metanovo</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/metanovo</td>
    </tr>
    <tr>
      <td></td>
      <td>metaquantome</td>
      <td>2.0.2</td>
      <td>2.0.2</td>
      <td>metaquantome</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>quantitative analysis of microbiome taxonomy and function</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproteomics/metaquantome/</td>
      <td>Proteomics</td>
      <td>metaquantome</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/metaquantome</td>
    </tr>
    <tr>
      <td></td>
      <td>mgf_formatter</td>
      <td>1.0.0</td>
      <td>1.0.0</td>
      <td>mgf-formatter</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td>mgf_formatter</td>
      <td>galaxyp</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>moFF</td>
      <td>2.0.3</td>
      <td>2.0.3</td>
      <td>moff</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>moFF (a modest Feature Finder) extracts MS1 intensities from RAW and mzML spectrum files.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/compomics/moFF</td>
      <td>Proteomics</td>
      <td>proteomics_moff</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/moFF</td>
    </tr>
    <tr>
      <td></td>
      <td>morpheus</td>
      <td>2.255.0</td>
      <td>287</td>
      <td>morpheus</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Morpheus MS Search Application</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>morpheus</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/morpheus</td>
    </tr>
    <tr>
      <td></td>
      <td>mqppep</td>
      <td>0.1.19</td>
      <td>1.64.0</td>
      <td>bioconductor-preprocesscore</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MaxQuant Phosphoproteomic Enrichment Pipeline - Preprocessing and ANOVA</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/</td>
      <td>Proteomics</td>
      <td>mqppep</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/mqppep</td>
    </tr>
    <tr>
      <td></td>
      <td>msconvert</td>
      <td>3.0.20287</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>msconvert Convert and/or filter mass spectrometry files (including vendor formats) using the official Docker container</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>msconvert</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msconvert</td>
    </tr>
    <tr>
      <td></td>
      <td>msgfplus</td>
      <td>0.5</td>
      <td>2023.01.1202</td>
      <td>msgf_plus</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MSGF+</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>msgfplus</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msgfplus</td>
    </tr>
    <tr>
      <td></td>
      <td>msms_extractor</td>
      <td>1.0.0</td>
      <td>3_0_9992</td>
      <td>proteowizard</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Extract MS/MS scans from the mzML file(s) based on PSM report.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>msms_extractor</td>
      <td>galaxyp</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>msstats</td>
      <td>4.0.0</td>
      <td>4.10.0</td>
      <td>bioconductor-msstats</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MSstats tool for analyzing mass spectrometry proteomic datasets</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/MeenaChoi/MSstats</td>
      <td>Proteomics</td>
      <td></td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msstats</td>
    </tr>
    <tr>
      <td></td>
      <td>msstatstmt</td>
      <td>2.0.0</td>
      <td>2.10.0</td>
      <td>bioconductor-msstatstmt</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MSstatsTMT protein significance analysis in shotgun mass spectrometry-based proteomic experiments with tandem mass tag (TMT) labeling</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://msstats.org/msstatstmt/</td>
      <td>Proteomics</td>
      <td>msstatstmt</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msstatstmt</td>
    </tr>
    <tr>
      <td></td>
      <td>mt2mq</td>
      <td>1.1.0</td>
      <td></td>
      <td>r-tidyverse</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tool to prepare metatranscriptomic outputs from ASaiM for Metaquantome</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>mt2mq</td>
      <td>galaxyp</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>mz_to_sqlite</td>
      <td>2.1.1+galaxy0</td>
      <td>2.1.1</td>
      <td>mztosqlite</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Creates a SQLite database for proteomics data</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>mz_to_sqlite</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/mz_to_sqlite</td>
    </tr>
    <tr>
      <td></td>
      <td>openms</td>
      <td>2.8</td>
      <td>3.1.0</td>
      <td>openms</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>OpenMS Suite for LC/MS data management and analyses</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.openms.de/</td>
      <td>Proteomics</td>
      <td>openms</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/openms</td>
    </tr>
    <tr>
      <td></td>
      <td>pathwaymatcher</td>
      <td></td>
      <td>1.9.1</td>
      <td>pathwaymatcher</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Reactome Pathway Matcher</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/LuisFranciscoHS/PathwayMatcher</td>
      <td>Proteomics</td>
      <td>reactome_pathwaymatcher</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pathwaymatcher</td>
    </tr>
    <tr>
      <td></td>
      <td>pep_pointer</td>
      <td>0.1.3+galaxy1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>PepPointer categorizes peptides by their genomic coordinates.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Genomic Interval Operations, Proteomics</td>
      <td>pep_pointer</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pep_pointer</td>
    </tr>
    <tr>
      <td></td>
      <td>pepquery</td>
      <td>1.6.2</td>
      <td>2.0.2</td>
      <td>pepquery</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A peptide-centric MS search engine for novel peptide identification and validation.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://pepquery.org</td>
      <td>Proteomics</td>
      <td>pepquery</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pepquery</td>
    </tr>
    <tr>
      <td></td>
      <td>pepquery2</td>
      <td>2.0.2</td>
      <td>2.0.2</td>
      <td>pepquery</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>PepQuery2 peptide-centric MS search for peptide identification and validation</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://pepquery.org</td>
      <td>Proteomics</td>
      <td>pepquery2</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pepquery2</td>
    </tr>
    <tr>
      <td></td>
      <td>peptide_genomic_coordinate</td>
      <td>1.0.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Gets genomic coordinate of peptides based on the information in mzsqlite and genomic mapping sqlite files</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>peptide_genomic_coordinate</td>
      <td>galaxyp</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>peptideshaker</td>
      <td></td>
      <td>4.3.6</td>
      <td>searchgui</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>PeptideShaker and SearchGUI</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://compomics.github.io</td>
      <td>Proteomics</td>
      <td>peptideshaker</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/peptideshaker</td>
    </tr>
    <tr>
      <td></td>
      <td>pepxml_to_xls</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert PepXML to Tabular</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>pepxml_to_xls</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pepxml_to_xls</td>
    </tr>
    <tr>
      <td></td>
      <td>percolator</td>
      <td>3.5</td>
      <td>3.5</td>
      <td>percolator</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Percolator</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td></td>
      <td>Proteomics</td>
      <td>percolator</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tools/percolator</td>
    </tr>
    <tr>
      <td></td>
      <td>pi_db_tools</td>
      <td>1.3</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>HiRIEF tools</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>hirieftools</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tools/pi_db_tools</td>
    </tr>
    <tr>
      <td></td>
      <td>pmd_fdr</td>
      <td>1.4.0</td>
      <td></td>
      <td>r-base</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Calculate Precursor Mass Discrepancy (PMD) for MS/MS</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/slhubler/PMD-FDR-for-Galaxy-P</td>
      <td>Proteomics</td>
      <td>pmd_fdr</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pmd_fdr</td>
    </tr>
    <tr>
      <td></td>
      <td>custom_pro_db</td>
      <td>1.22.0</td>
      <td>1.37.1</td>
      <td>bioconductor-rgalaxy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>CustomProDB</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/customProDB.html</td>
      <td>Proteomics</td>
      <td>custom_pro_db</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tools/bumbershoot/custom_pro_db</td>
    </tr>
    <tr>
      <td></td>
      <td>custom_pro_db_annotation_data_manager</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>CustomProDB Annotation</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/customProDB.html</td>
      <td>Proteomics</td>
      <td>custom_pro_db_annotation_data_manager</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tools/bumbershoot/custom_pro_db</td>
    </tr>
    <tr>
      <td></td>
      <td>psm2sam</td>
      <td>1.3.2.1</td>
      <td></td>
      <td>r-base</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>PSM to SAM</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/proBAMr.html</td>
      <td>Proteomics</td>
      <td>psm_to_sam</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tools/bumbershoot/psm2sam</td>
    </tr>
    <tr>
      <td></td>
      <td>proteinpilot</td>
      <td>0.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td>proteinpilot</td>
      <td>galaxyp</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>retrieve_ensembl_bed</td>
      <td>0.1.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Retrieve cDNA features from Ensembl REST API in BED format</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://rest.ensembl.org/</td>
      <td>Data Source</td>
      <td>retrieve_ensembl_bed</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteogenomics/retrieve_ensembl_bed</td>
    </tr>
    <tr>
      <td></td>
      <td>translate_bed</td>
      <td>0.1.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Translate BED transcript CDS or cDNA in 3 frames</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://rest.ensembl.org/</td>
      <td>Proteomics</td>
      <td>translate_bed</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteogenomics/translate_bed</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_joinquantpepionswithproteins</td>
      <td>0.0.1</td>
      <td>0.0.2</td>
      <td>proteomiqon-joinquantpepionswithproteins</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>The tool JoinQuantPepIonsWithProteins combines results from ProteinInference and PSMBasedQuantification.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/JoinQuantPepIonsWithProteins.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_joinquantpepionswithproteins</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_joinquantpepionswithproteins</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_labeledproteinquantification</td>
      <td>0.0.1</td>
      <td>0.0.3</td>
      <td>proteomiqon-labeledproteinquantification</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>The tool LabeledProteinQuantification estimates protein abundances using quantified peptide ions.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/LabeledProteinQuantification.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_labeledproteinquantification</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_labeledproteinquantification</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_labelfreeproteinquantification</td>
      <td>0.0.1</td>
      <td>0.0.3</td>
      <td>proteomiqon-labelfreeproteinquantification</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>The tool LabelFreeProteinQuantification estimates protein abundances using quantified peptide ions.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/LabelfreeProteinQuantification.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_labelfreeproteinquantification</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_labelfreeproteinquantification</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_mzmltomzlite</td>
      <td>0.0.8</td>
      <td>0.0.8</td>
      <td>proteomiqon-mzmltomzlite</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>The tool MzMLToMzLite allows to convert mzML files to mzLite files.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/MzMLToMzLite.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_mzmltomzlite</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomIQon_MzMLToMzLite</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_peptidedb</td>
      <td>0.0.7</td>
      <td>0.0.7</td>
      <td>proteomiqon-peptidedb</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>The tool ProteomIQon PeptideDB creates a peptide database in the SQLite format.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/PeptideDB.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_peptidedb</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_peptidedb</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_peptidespectrummatching</td>
      <td>0.0.7</td>
      <td>0.0.7</td>
      <td>proteomiqon-peptidespectrummatching</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Given raw an MS run in the mzLite format, this tool iterates across all MS/MS scans, determines precursor charge states and possible peptide spectrum matches using reimplementations of SEQUEST,Andromeda and XTandem.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/PeptideSpectrumMatching.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_peptidespectrummatching</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_peptidespectrummatching</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_proteininference</td>
      <td>0.0.7</td>
      <td>0.0.7</td>
      <td>proteomiqon-proteininference</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MS-based shotgun proteomics estimates protein abundances using a proxy: peptides. The process of 'Protein Inference' is concerned with the mapping of identified peptides to the proteins they putatively originated from.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/ProteinInference.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_proteininference</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_proteininference</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_psmbasedquantification</td>
      <td>0.0.8</td>
      <td>0.0.9</td>
      <td>proteomiqon-psmbasedquantification</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>The PSMBasedQuantification tool was designed to allow label-free quantification as well as quantification of full metabolic labeled samples.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/PSMBasedQuantification.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_psmbasedquantification</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_psmbasedquantification</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_psmstatistics</td>
      <td>0.0.8</td>
      <td>0.0.8</td>
      <td>proteomiqon-psmstatistics</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>The PSMStatistics tool utilizes semi supervised machine learning techniques to integrate search engine scores as well as the mentioned quality scores into one single consensus score.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/PSMStatistics.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_psmstatistics</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_psmstatistics</td>
    </tr>
    <tr>
      <td></td>
      <td>proteore_venn_diagram</td>
      <td>2021.06.08</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ProteoRE JVenn Diagram</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>proteore_venn_diagram</td>
      <td>galaxyp</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>protxml_to_xls</td>
      <td>0.1.0</td>
      <td></td>
      <td>trans_proteomic_pipeline</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td>protxml_to_xls</td>
      <td>galaxyp</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>psm_eval</td>
      <td>0.1.0</td>
      <td></td>
      <td>binaries_for_psm_eval</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td>psm_eval</td>
      <td>galaxyp</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>psm_validation</td>
      <td>1.0.3</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Validate PSM from Ion Fragmentation</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproteomics/psm_fragments.git</td>
      <td>Proteomics</td>
      <td>psm_validation</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/psm_validation</td>
    </tr>
    <tr>
      <td></td>
      <td>pyprophet</td>
      <td>2.1.4</td>
      <td>2.2.5</td>
      <td>pyprophet</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Semi-supervised learning and scoring of OpenSWATH results.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/PyProphet/pyprophet</td>
      <td>Proteomics</td>
      <td></td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pyprophet</td>
    </tr>
    <tr>
      <td></td>
      <td>pyteomics</td>
      <td>4.4.1</td>
      <td>4.7.1</td>
      <td>pyteomics</td>
      <td>To update</td>
      <td>pyteomics</td>
      <td>Pyteomics</td>
      <td>Protein identification</td>
      <td>Proteomics, Proteomics experiment</td>
      <td>Tools using the pyteomics library</td>
      <td>Framework for proteomics data analysis, supporting mzML, MGF, pepXML and more.</td>
      <td></td>
      <td>To update</td>
      <td>https://pyteomics.readthedocs.io/en/latest/</td>
      <td>Proteomics, Metabolomics</td>
      <td>pyteomics</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pyteomics</td>
    </tr>
    <tr>
      <td></td>
      <td>quantp</td>
      <td>1.1.2</td>
      <td>1.11.6</td>
      <td>r-data.table</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Correlation between protein and transcript abundance</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>quantp</td>
      <td>galaxyp</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>quantwiz_iq</td>
      <td>2.0</td>
      <td>2.0</td>
      <td>quantwiz-iq</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Isobaric Quantitation using QuantWiz-IQ</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://sourceforge.net/projects/quantwiz/</td>
      <td>Proteomics</td>
      <td>quantwiz_iq</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/quantwiz_iq</td>
    </tr>
    <tr>
      <td></td>
      <td>qupath_roi_splitter</td>
      <td>0.1.0+galaxy1</td>
      <td></td>
      <td>geojson</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Split ROI coordinates of QuPath TMA annotation by cell type</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/npinter/ROIsplitter</td>
      <td>Imaging</td>
      <td>qupath_roi_splitter</td>
      <td>galaxyp</td>
      <td>hhttps://github.com/npinter/ROIsplitter</td>
    </tr>
    <tr>
      <td></td>
      <td>rawtools</td>
      <td></td>
      <td>2.0.4</td>
      <td>rawtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Raw Tools</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/kevinkovalchik/RawTools</td>
      <td>Proteomics</td>
      <td>rawtools</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/rawtools</td>
    </tr>
    <tr>
      <td></td>
      <td>regex_find_replace</td>
      <td>1.0.3</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Use python regular expressions to find and replace text</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>regex_find_replace</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/regex_find_replace</td>
    </tr>
    <tr>
      <td></td>
      <td>scaffold</td>
      <td>0.1.0</td>
      <td></td>
      <td>scaffold</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td>scaffold</td>
      <td>galaxyp</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>sixgill</td>
      <td>0.2.4</td>
      <td>0.2.4</td>
      <td>sixgill</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Six-frame Genome-Inferred Libraries for LC-MS/MS</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td></td>
      <td>Proteomics, MetaProteomics</td>
      <td>sixgill</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/sixgill</td>
    </tr>
    <tr>
      <td></td>
      <td>spectrast2spectrast_irt</td>
      <td>0.1.0</td>
      <td>0.11.0</td>
      <td>msproteomicstools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter from spectraST files to swath input files</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>spectrast2spectrast_irt</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/spectrast2spectrast_irt</td>
    </tr>
    <tr>
      <td></td>
      <td>spectrast2tsv</td>
      <td>0.1.0</td>
      <td>0.11.0</td>
      <td>msproteomicstools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter from spectraST files to swath input files</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>spectrast2tsv</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/spectrast2tsv</td>
    </tr>
    <tr>
      <td></td>
      <td>translate_bed_sequences</td>
      <td>0.2.0</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Perform 3 frame translation of BED file augmented with a sequence column</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>translate_bed_sequences</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/translate_bed_sequences</td>
    </tr>
    <tr>
      <td></td>
      <td>unipept</td>
      <td>4.5.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Unipept retrieves metaproteomics information</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp</td>
      <td>Proteomics</td>
      <td>unipept</td>
      <td>galaxyp</td>
      <td>https://unipept.ugent.be/apidocs</td>
    </tr>
    <tr>
      <td></td>
      <td>uniprotxml_downloader</td>
      <td>2.4.0</td>
      <td></td>
      <td>requests</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Download UniProt proteome in XML or fasta format</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>uniprotxml_downloader</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/uniprotxml_downloader</td>
    </tr>
    <tr>
      <td></td>
      <td>validate_fasta_database</td>
      <td>0.1.5</td>
      <td>1.0</td>
      <td>validate-fasta-database</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>runs Compomics database identification tool on any FASTA database, and separates valid and invalid entries based on a series of checks.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Fasta Manipulation, Proteomics</td>
      <td>validate_fasta_database</td>
      <td>galaxyp</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>bio3d</td>
      <td>2.4_1</td>
      <td>2.3_3</td>
      <td>r-bio3d</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Bio3d is a program that can be used to analyse molecular dynamics trajectories.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://thegrantlab.org/bio3d/index.php</td>
      <td>Computational chemistry</td>
      <td>bio3d</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/bio3d</td>
    </tr>
    <tr>
      <td></td>
      <td>biomoldyn</td>
      <td>1.5.2</td>
      <td></td>
      <td>scipy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tools for MD analysis</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/moldyn/</td>
      <td>Molecular Dynamics, Computational chemistry</td>
      <td>biomoldyn</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/</td>
    </tr>
    <tr>
      <td></td>
      <td>ambertools</td>
      <td>21.10</td>
      <td></td>
      <td>ambertools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Ambertools is a set of packages for preparing systems for molecular dynamics (MD) simulations and analyzing trajectories.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://ambermd.org/AmberTools.php</td>
      <td>Molecular Dynamics, Computational chemistry</td>
      <td>ambertools</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/</td>
    </tr>
    <tr>
      <td></td>
      <td>packmol</td>
      <td>18.169.1</td>
      <td></td>
      <td>packmol</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>PACKMOL is a package for creating starting structures for Molecular Dynamics simulations</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://m3g.iqm.unicamp.br/packmol/home.shtml</td>
      <td>Molecular Dynamics, Computational chemistry</td>
      <td>packmol</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem</td>
    </tr>
    <tr>
      <td></td>
      <td>topologyeditors</td>
      <td>0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Set of python scripts and associated tool files that can be used to modify topology files.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/buildtools/topologyeditors</td>
      <td>Molecular Dynamics, Computational chemistry</td>
      <td>topologyeditors</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/buildtools/topologyeditors</td>
    </tr>
    <tr>
      <td></td>
      <td>free_energy</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Free energy tools of BRIDGE.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Molecular Dynamics, Computational chemistry</td>
      <td>freeenergy</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tools/free_energy</td>
    </tr>
    <tr>
      <td></td>
      <td>gromacs</td>
      <td>2022</td>
      <td>2021.3</td>
      <td>gromacs</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>GROMACS is a package for performing molecular dynamics, primarily designed for biochemical molecules such as proteins, lipids and nucleic acids.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/gromacs</td>
      <td>Molecular Dynamics, Computational chemistry</td>
      <td>gromacs</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/gromacs</td>
    </tr>
    <tr>
      <td></td>
      <td>mdanalysis</td>
      <td>1.0.0</td>
      <td></td>
      <td>mdanalysis</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MDAnalysis is a package for analyzing trajectories from molecular dynamics (MD) simulations</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/MDAnalysis/mdanalysis</td>
      <td>Computational chemistry</td>
      <td>mdanalysis</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/</td>
    </tr>
    <tr>
      <td></td>
      <td>mdfileconverter</td>
      <td>1.9.7</td>
      <td></td>
      <td>mdtraj</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A tool for interconverting between different MD structure and trajectory file formats.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Molecular Dynamics, Computational chemistry</td>
      <td>md_converter</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tools/mdfileconverter</td>
    </tr>
    <tr>
      <td></td>
      <td>mdslicer</td>
      <td>1.9.9</td>
      <td></td>
      <td>mdtraj</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A tool for slicing trajectory files using MDTraj.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Molecular Dynamics, Computational chemistry</td>
      <td>md_converter</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tools/mdslicer</td>
    </tr>
    <tr>
      <td></td>
      <td>mdtraj</td>
      <td>1.9.7</td>
      <td></td>
      <td>mdtraj</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MDTraj is a python library that allows users to manipulate molecular dynamics (MD) trajectories</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/mdtraj/mdtraj</td>
      <td>Computational chemistry</td>
      <td>mdtraj</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/</td>
    </tr>
    <tr>
      <td></td>
      <td>openmm</td>
      <td>1.8.1</td>
      <td></td>
      <td>pdbfixer</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>OpenMM is a toolkit for molecular simulation using high performance GPU code.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/openmm</td>
      <td>Molecular Dynamics, Computational chemistry</td>
      <td>openmm</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/tree/master/tools/openmm</td>
    </tr>
    <tr>
      <td></td>
      <td>vmd</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>vmd is a package for  visualizing and analyzing trajectories from molecular dynamics (MD) simulations</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.ks.uiuc.edu/Research/vmd/</td>
      <td>Computational chemistry</td>
      <td>vmd</td>
      <td>chemteam</td>
      <td>https://github.com/thatchristoph/vmd-cvs-github/tree/master/vmd</td>
    </tr>
    <tr>
      <td></td>
      <td>artbio_bam_cleaning</td>
      <td>1.10+galaxy0</td>
      <td>1.19.2</td>
      <td>samtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>filter bam files before somatic-varscan or lumpy-smoove analysis</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>SAM, Variant Analysis</td>
      <td>artbio_bam_cleaning</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/artbio_bam_cleaning</td>
    </tr>
    <tr>
      <td></td>
      <td>bamparse</td>
      <td>4.1.1</td>
      <td>0.22.0</td>
      <td>pysam</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Generates hit count lists from bam alignments.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, Transcriptomics</td>
      <td>bamparse</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/bamparse</td>
    </tr>
    <tr>
      <td></td>
      <td>bigwig_to_bedgraph</td>
      <td>377+galaxy1</td>
      <td>448</td>
      <td>ucsc-bigwigtobedgraph</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Converts a bigWig file to bedGraph format</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Convert Formats</td>
      <td>bigwig_to_bedgraph</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/bigwig_to_bedgraph</td>
    </tr>
    <tr>
      <td></td>
      <td>bigwig_to_wig</td>
      <td>3+galaxy0</td>
      <td>377</td>
      <td>ucsc-bigwiginfo</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Converts a bigWig file to Wiggle (WIG) format</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://artbio.fr</td>
      <td>Convert Formats</td>
      <td>bigwig_to_wig</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/bigwig_to_wig</td>
    </tr>
    <tr>
      <td></td>
      <td>blast_to_scaffold</td>
      <td>1.1.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Generate DNA scaffold from blastn or tblastx alignments of Contigs</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, Sequence Analysis, Assembly</td>
      <td>blast_to_scaffold</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/blast_to_scaffold</td>
    </tr>
    <tr>
      <td></td>
      <td>blast_unmatched</td>
      <td>1.0.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Extract unmatched query sequences from blast</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Fasta Manipulation</td>
      <td>blast_unmatched</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/blast_unmatched</td>
    </tr>
    <tr>
      <td></td>
      <td>blastparser_and_hits</td>
      <td>2.7.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Parse blast outputs and compile hits</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Assembly, RNA</td>
      <td>blastparser_and_hits</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/blastparser_and_hits</td>
    </tr>
    <tr>
      <td></td>
      <td>blastx_to_scaffold</td>
      <td>1.1.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Generate DNA scaffold from blastx alignment of Contigs</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, Sequence Analysis, Assembly</td>
      <td>blastx_to_scaffold</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/blastx_to_scaffold</td>
    </tr>
    <tr>
      <td></td>
      <td>cap3</td>
      <td>2.0.1</td>
      <td>10.2011</td>
      <td>cap3</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>cap3 wrapper</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Assembly</td>
      <td>cap3</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/cap3</td>
    </tr>
    <tr>
      <td></td>
      <td>cherry_pick_fasta</td>
      <td>4.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Pick fasta sequence with specific header content</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Fasta Manipulation</td>
      <td>cherry_pick_fasta</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/cherry_pick_fasta</td>
    </tr>
    <tr>
      <td></td>
      <td>concat_multi_datasets</td>
      <td>1.4.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Concatenate multiple datasets tail-to-head, including collection datasets.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Text Manipulation</td>
      <td>concatenate_multiple_datasets</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/concat_multi_datasets</td>
    </tr>
    <tr>
      <td></td>
      <td>cpm_tpm_rpk</td>
      <td>0.5.2</td>
      <td>1.3.2</td>
      <td>r-optparse</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Generate CPM,TPM or RPK from raw counts</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Transcriptomics</td>
      <td>cpm_tpm_rpk</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/cpm_tpm_rpk</td>
    </tr>
    <tr>
      <td></td>
      <td>deseq2_normalization</td>
      <td>1.40.2+galaxy0</td>
      <td>1.42.0</td>
      <td>bioconductor-deseq2</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Normalizes gene hitlists</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, Transcriptomics, Sequence Analysis, Statistics</td>
      <td>deseq2_normalization</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/deseq2_normalization</td>
    </tr>
    <tr>
      <td></td>
      <td>embl2fa</td>
      <td>0.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Converts EMBL flat format to fasta format</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Text Manipulation</td>
      <td>embl2fa</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/embl2fa</td>
    </tr>
    <tr>
      <td></td>
      <td>ez_histograms</td>
      <td>3.4.4</td>
      <td>2.2.1</td>
      <td>r-ggplot2</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ggplot2 histograms and density plots</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/tidyverse/ggplot2</td>
      <td>Visualization, Statistics</td>
      <td>ez_histograms</td>
      <td>artbio</td>
      <td>https://github.com/artbio/tools-artbio/tree/main/tools/ez_histograms</td>
    </tr>
    <tr>
      <td></td>
      <td>fetch_fasta_from_ncbi</td>
      <td>3.1.0</td>
      <td>1.12</td>
      <td>urllib3</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Fetch fasta sequences from NCBI using eutils wrappers</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Fasta Manipulation, Data Source</td>
      <td>fetch_fasta_from_ncbi</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/fetch_fasta_from_ncbi</td>
    </tr>
    <tr>
      <td></td>
      <td>fisher_test</td>
      <td>2.32.0+galaxy0</td>
      <td>2.34.0</td>
      <td>bioconductor-qvalue</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Fisher's exact test on two-column hit lists.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, Statistics</td>
      <td>fishertest</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/fisher_test</td>
    </tr>
    <tr>
      <td></td>
      <td>gatk4</td>
      <td>4.1.7.0</td>
      <td>4.5.0.0</td>
      <td>gatk4</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Find somatic variations</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Variant Analysis</td>
      <td>gatk4</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gatk4</td>
    </tr>
    <tr>
      <td></td>
      <td>get_reference_fasta</td>
      <td>0.3.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Obtain reference genome sequence.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Data Source, Fasta Manipulation</td>
      <td>get_reference_fasta</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/get_reference_fasta</td>
    </tr>
    <tr>
      <td></td>
      <td>gsc_center_scale</td>
      <td>4.3.1+galaxy0</td>
      <td>1.3.2</td>
      <td>r-optparse</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Center or scale (standardize) data</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Statistics</td>
      <td>gsc_center_scale</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_center_scale</td>
    </tr>
    <tr>
      <td></td>
      <td>gsc_filter_cells</td>
      <td>4.3.1+galaxy0</td>
      <td>1.3.2</td>
      <td>r-optparse</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter single cell RNAseq data on libray depth and number of detected genes</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Transcriptomics</td>
      <td>gsc_filter_cells</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_filter_cells</td>
    </tr>
    <tr>
      <td></td>
      <td>gsc_filter_genes</td>
      <td>4.3.1+galaxy0</td>
      <td>1.3.2</td>
      <td>r-optparse</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter genes that are detected in less than a fraction of libraries in single cell RNAseq data</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Transcriptomics</td>
      <td>gsc_filter_genes</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_filter_genes</td>
    </tr>
    <tr>
      <td></td>
      <td>gsc_gene_expression_correlations</td>
      <td>4.3.1+galaxy0</td>
      <td>1.3.2</td>
      <td>r-optparse</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute single-cell paire-wise gene expressions correlations</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Transcriptomics</td>
      <td>gsc_gene_expression_correlations</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/gsc_gene_expression_correlations</td>
    </tr>
    <tr>
      <td></td>
      <td>gsc_high_dimensions_visualisation</td>
      <td>4.3+galaxy0</td>
      <td>1.3.2</td>
      <td>r-optparse</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Generates PCA, t-SNE and HCPC visualisation</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Transcriptomics, Visualization</td>
      <td>gsc_high_dimensions_visualisation</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_high_dimension_visualization</td>
    </tr>
    <tr>
      <td></td>
      <td>gsc_mannwhitney_de</td>
      <td>4.1.3+galaxy0</td>
      <td>1.3.2</td>
      <td>r-optparse</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Perform a mann-whitney differential testing between two sets of gene expression data</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Transcriptomics</td>
      <td>gsc_mannwhitney_de</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_mannwhitney_de</td>
    </tr>
    <tr>
      <td></td>
      <td>gsc_scran_normalize</td>
      <td>1.28.1+galaxy0</td>
      <td>1.30.0</td>
      <td>bioconductor-scran</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Normalize raw counts using scran</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Transcriptomics</td>
      <td>gsc_scran_normalize</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_scran_normalize</td>
    </tr>
    <tr>
      <td></td>
      <td>gsc_signature_score</td>
      <td>2.3.9+galaxy0</td>
      <td>1.3.2</td>
      <td>r-optparse</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute signature scores from single cell RNAseq data</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Transcriptomics</td>
      <td>gsc_signature_score</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/gsc_signature_score</td>
    </tr>
    <tr>
      <td></td>
      <td>guppy</td>
      <td>0.2.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A wrapper for the guppy basecaller tool from Oxford Nanopore Technologies</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Nanopore</td>
      <td>guppy_basecaller</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/guppy</td>
    </tr>
    <tr>
      <td></td>
      <td>high_dim_heatmap</td>
      <td>3.1.3+galaxy0</td>
      <td>2.17.0</td>
      <td>r-gplots</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>gplot heatmap.2 function adapted for plotting large heatmaps</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/cran/gplots</td>
      <td>Visualization</td>
      <td>high_dim_heatmap</td>
      <td>artbio</td>
      <td>https://github.com/artbio/tools-artbio/tree/main/tools/high_dim_heatmap</td>
    </tr>
    <tr>
      <td></td>
      <td>justdiff</td>
      <td>3.10+galaxy0</td>
      <td></td>
      <td>diffutils</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Unix diff</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Text Manipulation</td>
      <td>justdiff</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/justdiff</td>
    </tr>
    <tr>
      <td></td>
      <td>justgzip</td>
      <td>2.8+galaxy0</td>
      <td></td>
      <td>pigz</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compress fastq sequence files</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Convert Formats</td>
      <td>justgzip</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/justgzip</td>
    </tr>
    <tr>
      <td></td>
      <td>lumpy_smoove</td>
      <td>0.2.8+galaxy1</td>
      <td>0.7.1</td>
      <td>svtyper</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Galaxy wrapper of the lumpy-using smoove workflow</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Variant Analysis</td>
      <td>lumpy_smoove</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/lumpy_smoove</td>
    </tr>
    <tr>
      <td></td>
      <td>manta</td>
      <td>1.6</td>
      <td>1.6.0</td>
      <td>manta</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Structural variant and indel caller for mapped sequencing data</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Variant Analysis</td>
      <td>manta</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/manta</td>
    </tr>
    <tr>
      <td></td>
      <td>mapping_quality_stats</td>
      <td>0.22.0</td>
      <td>1.3.2</td>
      <td>r-optparse</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Collects and shows the distribution of MAPQ values in a BAM alignment file</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Sequence Analysis, Statistics</td>
      <td>mapping_quality_stats</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/mapping_quality_stats</td>
    </tr>
    <tr>
      <td></td>
      <td>mircounts</td>
      <td>1.6</td>
      <td></td>
      <td>tar</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Generates miRNA count lists from read alignments to mirBase.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, Transcriptomics</td>
      <td>mircounts</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/mircounts</td>
    </tr>
    <tr>
      <td></td>
      <td>mutational_patterns</td>
      <td>3.12.0</td>
      <td>3.12.0</td>
      <td>bioconductor-mutationalpatterns</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Mutational patterns and signatures in base substitution catalogs</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://artbio.fr</td>
      <td>Variant Analysis</td>
      <td>mutational_patterns</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/mutational_patterns</td>
    </tr>
    <tr>
      <td></td>
      <td>oases</td>
      <td>1.4.0</td>
      <td>0.2.09</td>
      <td>oases</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Short read assembler</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Assembly, RNA</td>
      <td>oases</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/oases</td>
    </tr>
    <tr>
      <td></td>
      <td>pathifier</td>
      <td>1.40.0</td>
      <td>1.40.0</td>
      <td>bioconductor-pathifier</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>pathifier</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://</td>
      <td>Transcriptomics, Statistics</td>
      <td>pathifier</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/pathifier</td>
    </tr>
    <tr>
      <td></td>
      <td>pindel</td>
      <td>0.2.5b9</td>
      <td>0.2.5b9</td>
      <td>pindel</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Pindel detects genome-wide structural variation.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://artbio.fr</td>
      <td>Variant Analysis</td>
      <td>pindel</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/pindel</td>
    </tr>
    <tr>
      <td></td>
      <td>probecoverage</td>
      <td>0.22.0</td>
      <td>0.22.0</td>
      <td>pysam</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>computes and plots read coverage of genomic regions by sequencing datasets</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://artbio.fr</td>
      <td>Sequence Analysis, Genomic Interval Operations, Graphics, Statistics</td>
      <td>probecoverage</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/probecoverage</td>
    </tr>
    <tr>
      <td></td>
      <td>repenrich</td>
      <td>1.5.2</td>
      <td>4.0.2</td>
      <td>bioconductor-edger</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Repeat element profiling</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/nskvir/RepEnrich</td>
      <td>Transcriptomics, Variant Analysis</td>
      <td>repenrich</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/repenrich</td>
    </tr>
    <tr>
      <td></td>
      <td>rsem</td>
      <td></td>
      <td>1.3.3</td>
      <td>rsem</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>transcript quantification from RNA-Seq data</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/deweylab/RSEM</td>
      <td>Transcriptomics, RNA</td>
      <td>rsem</td>
      <td>artbio</td>
      <td>https://github.com/artbio/tools-artbio/tree/master/tools/rsem</td>
    </tr>
    <tr>
      <td></td>
      <td>sambamba</td>
      <td>0.7.1+galaxy1</td>
      <td>1.0</td>
      <td>sambamba</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>filter BAM/SAM on flags, fields, tags, and region, or down-sample, or slice BAM/SAM</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>SAM</td>
      <td>sambamba</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/sambamba</td>
    </tr>
    <tr>
      <td></td>
      <td>sashimi_plot</td>
      <td>0.1.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Generates a sashimi plot from bam files.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, Transcriptomics, Graphics, Visualization</td>
      <td>sashimi_plot</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/sashimi_plot</td>
    </tr>
    <tr>
      <td></td>
      <td>sequence_format_converter</td>
      <td>2.2.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>various fasta to tabular conversions</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Convert Formats, Fasta Manipulation</td>
      <td>sequence_format_converter</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/sequence_format_converter</td>
    </tr>
    <tr>
      <td></td>
      <td>small_rna_clusters</td>
      <td>1.3.0</td>
      <td>0.22.0</td>
      <td>pysam</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>clusters small rna reads in alignment BAM files</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, SAM, Graphics, Next Gen Mappers</td>
      <td>small_rna_clusters</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/small_rna_clusters</td>
    </tr>
    <tr>
      <td></td>
      <td>small_rna_maps</td>
      <td>3.1.1</td>
      <td></td>
      <td>numpy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Generates small read maps from alignment BAM files</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, SAM, Graphics, Next Gen Mappers</td>
      <td>small_rna_maps</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/small_rna_maps</td>
    </tr>
    <tr>
      <td></td>
      <td>small_rna_signatures</td>
      <td>3.4.2</td>
      <td>0.22.0</td>
      <td>pysam</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Computes the tendency of small RNAs to overlap with each other.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA</td>
      <td>small_rna_signatures</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/small_rna_signatures</td>
    </tr>
    <tr>
      <td></td>
      <td>snvtocnv</td>
      <td>3.0.0+galaxy1</td>
      <td>3.0.0</td>
      <td>sequenza-utils</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>infer copy number variations from a vcf file with SNVs using R sequenza</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Variant Analysis</td>
      <td>snvtocnv</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/snvtocnv</td>
    </tr>
    <tr>
      <td></td>
      <td>sr_bowtie</td>
      <td>2.3.0</td>
      <td>1.3.1</td>
      <td>bowtie</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>bowtie wrapper tool to align small RNA sequencing reads</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, Next Gen Mappers</td>
      <td>sr_bowtie</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/sr_bowtie</td>
    </tr>
    <tr>
      <td></td>
      <td>sr_bowtie_dataset_annotation</td>
      <td>2.8</td>
      <td>1.3.1</td>
      <td>bowtie</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Maps iteratively small RNA sequencing datasets to reference sequences.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA</td>
      <td>sr_bowtie_dataset_annotation</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/sr_bowtie_dataset_annotation</td>
    </tr>
    <tr>
      <td></td>
      <td>tarfast5</td>
      <td>0.6.1</td>
      <td></td>
      <td>pigz</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>produces a tar.gz archive of fast5 sequence files</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Nanopore</td>
      <td>tarfast5</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/tarfast5</td>
    </tr>
    <tr>
      <td></td>
      <td>varscan_vaf</td>
      <td>0.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute variant allele frequency in vcf files generated by varscan.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Variant Analysis</td>
      <td>varscan_vaf</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/varscan_vaf</td>
    </tr>
    <tr>
      <td></td>
      <td>xpore</td>
      <td>2.1+galaxy0</td>
      <td>2.1</td>
      <td>xpore</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Identification and quantification of differential RNA modifications from direct RNA sequencing</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/GoekeLab/xpore</td>
      <td>Nanopore</td>
      <td>xpore</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/xpore</td>
    </tr>
    <tr>
      <td></td>
      <td>yac_clipper</td>
      <td>2.5.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Clips 3' adapters for small RNA sequencing reads.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, Fastq Manipulation</td>
      <td>yac_clipper</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/yac_clipper</td>
    </tr>
    <tr>
      <td></td>
      <td>EMLassemblyline</td>
      <td>0.1.1+galaxy0</td>
      <td></td>
      <td>r-emlassemblyline</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tools using EML Assembly Line R package to generate EML metadata from template metadata files and vice versa</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/EDIorg/EMLassemblyline</td>
      <td>Ecology</td>
      <td>emlassemblyline</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/EMLassemblyline</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecoregionalization_workflow</td>
      <td>0.1.0+galaxy0</td>
      <td></td>
      <td>r-base</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tools to compute ecoregionalization with BRT model predictions and clustering.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/PaulineSGN/Workflow_Galaxy</td>
      <td>Ecology</td>
      <td>ecoregionalization</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/Ecoregionalization_workflow</td>
    </tr>
    <tr>
      <td></td>
      <td>Geom_mean_workflow</td>
      <td>0.1.0+galaxy0</td>
      <td></td>
      <td>r-base</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tools to compute The evolution of the total volume of very large trees, standing dead wood and dead wood on the ground on an area and the rate of devolution of the volume of wood favorable to biodiversity by large ecological regions (France).</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/PaulineSGN/Galaxy_tool_moyenne_geom</td>
      <td>Ecology</td>
      <td>Geometric means (Dead wood)</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/Geom_mean_workflow</td>
    </tr>
    <tr>
      <td></td>
      <td>PAMPA</td>
      <td>0.0.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tools to compute and analyse biodiversity metrics</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Ecology</td>
      <td>pampa</td>
      <td>ecology</td>
      <td>https://github.com/ColineRoyaux/PAMPA-Galaxy</td>
    </tr>
    <tr>
      <td></td>
      <td>ab1_fastq</td>
      <td>1.20.0</td>
      <td>1.38.0</td>
      <td>bioconductor-sangerseqr</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tool to convert ab1 files into FASTQ files</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Convert Formats</td>
      <td>ab1fastq</td>
      <td>ecology</td>
      <td>https://github.com/ColineRoyaux/Galaxy_tool_projects/tree/main/ab1_fastq</td>
    </tr>
    <tr>
      <td></td>
      <td>champ_blocs</td>
      <td>0.0.0</td>
      <td></td>
      <td>r-base</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute indicators for turnover boulders fields</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Ecology</td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/Marie59/champ_blocs</td>
    </tr>
    <tr>
      <td></td>
      <td>consensus_from_alignments</td>
      <td>1.0.0</td>
      <td></td>
      <td>r-bioseq</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tool to compute a consensus sequence from several aligned fasta sequences</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>consalign</td>
      <td>ecology</td>
      <td>https://github.com/ColineRoyaux/Galaxy_tool_projects/tree/main/consensus_from_alignments</td>
    </tr>
    <tr>
      <td></td>
      <td>data_exploration</td>
      <td>0.0.0</td>
      <td></td>
      <td>r-tangles</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Explore data through multiple statistical tools</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Ecology</td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/Marie59/Data_explo_tools</td>
    </tr>
    <tr>
      <td></td>
      <td>xarray</td>
      <td>2022.3.0</td>
      <td></td>
      <td>xarray</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>xarray (formerly xray) is an open source project and Python package that makes working withlabelled multi-dimensional arrays simple, efficient, and fun!xarray integrates with Dask to support parallel computations and streaming computation on datasetsthat dont fit into memory.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://xarray.pydata.org</td>
      <td>Ecology</td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/data_manipulation/xarray/</td>
    </tr>
    <tr>
      <td></td>
      <td>gdal</td>
      <td>3.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Geospatial Data Abstraction Library tools are all dedicated to manipulate raster and vector geospatial data formats.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.gdal.org</td>
      <td>Ecology</td>
      <td>gdal</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/gdal</td>
    </tr>
    <tr>
      <td></td>
      <td>interpolation</td>
      <td>1.0</td>
      <td></td>
      <td>r-getopt</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Run IDW interpolation based on a .csv and .geojson file</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/AquaINFRA/galaxy</td>
      <td>Ecology</td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/interpolation</td>
    </tr>
    <tr>
      <td></td>
      <td>medenv</td>
      <td>0.0.1</td>
      <td></td>
      <td>pandas</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Retrieve environmental data from etopo, cmems and woa</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/jeremyfix/medenv</td>
      <td>Ecology, Data Source</td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/jeremyfix/medenv</td>
    </tr>
    <tr>
      <td></td>
      <td>obisindicators</td>
      <td>0.0.2</td>
      <td></td>
      <td>r-base</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute biodiveristy indicators for marine data from obis</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/Marie59/obisindicators</td>
      <td>Ecology</td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/obisindicators</td>
    </tr>
    <tr>
      <td></td>
      <td>ocean</td>
      <td>0.1.15</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Access, process, visualise oceanographic data for the Earth System</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/Marie59/FE-ft-ESG/tree/main/argo</td>
      <td>Ecology</td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/ocean</td>
    </tr>
    <tr>
      <td></td>
      <td>regionalgam</td>
      <td>1.5</td>
      <td></td>
      <td>r-mgcv</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/RetoSchmucki/regionalGAM</td>
      <td>Ecology</td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/regionalgam</td>
    </tr>
    <tr>
      <td></td>
      <td>spocc</td>
      <td>1.2.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Get species occurences data</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://cran.r-project.org/web/packages/spocc/index.html</td>
      <td>Ecology</td>
      <td>spocc_occ</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/spocc</td>
    </tr>
    <tr>
      <td></td>
      <td>srs_tools</td>
      <td>0.0.1</td>
      <td></td>
      <td>r-base</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute biodiversity indicators for remote sensing data from Sentinel 2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Ecology</td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/Marie59/Sentinel_2A/srs_tools</td>
    </tr>
    <tr>
      <td></td>
      <td>stoc</td>
      <td>0.0.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tools to analyse STOC data.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Ecology</td>
      <td>stoceps</td>
      <td>ecology</td>
      <td>https://github.com/Alanamosse/Galaxy-E/tree/stoctool/tools/stoc</td>
    </tr>
    <tr>
      <td></td>
      <td>vigiechiro</td>
      <td>0.1.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tools created by the vigiechiro team to analyses and identify chiro sounds files.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.vigienature-ecole.fr/les-observatoires/le-protocole-vigie-chiro</td>
      <td>Ecology</td>
      <td>vigiechiro</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/vigiechiro</td>
    </tr>
    <tr>
      <td></td>
      <td>xmlstarlet</td>
      <td>1.6.1</td>
      <td></td>
      <td>xmlstarlet</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tool to convert a xml file from one metadata standard to another</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Convert Formats</td>
      <td>xmlstarlet</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/main/tools-ecology/tools/xmlstarlet</td>
    </tr>
    <tr>
      <td></td>
      <td>c3s</td>
      <td>0.2.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Copernicus Climate Change Service (C3S)</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://cds.climate.copernicus.eu/cdsapp#!/search?type=dataset</td>
      <td>Climate Analysis</td>
      <td>c3s</td>
      <td>climate</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/c3s</td>
    </tr>
    <tr>
      <td></td>
      <td>cads</td>
      <td>0.1.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Copernicus Atmosphere Data Store (ADS)</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://ads.atmosphere.copernicus.eu/#!/home</td>
      <td>Climate Analysis</td>
      <td>cads</td>
      <td>climate</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/cads</td>
    </tr>
    <tr>
      <td></td>
      <td>cdo</td>
      <td>2.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>CDO (Climate Data Operators) is a collection of command line Operators to manipulate and analyse Climate and NWP model Data.Supported data formats are GRIB 1/2, netCDF 3/4, SERVICE, EXTRA and IEG. There are more than 600 operators available.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://code.mpimet.mpg.de/projects/cdo/</td>
      <td>Climate Analysis</td>
      <td></td>
      <td>climate</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/cdo</td>
    </tr>
    <tr>
      <td></td>
      <td>cesm</td>
      <td>2.1.3</td>
      <td>2.1.3</td>
      <td>cesm</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Community Earth System Model (CESM)</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://www.cesm.ucar.edu/</td>
      <td>Climate Analysis</td>
      <td>cesm</td>
      <td>climate</td>
      <td>https://github.com/ESCOMP/CESM</td>
    </tr>
    <tr>
      <td></td>
      <td>climate-stripes</td>
      <td>1.0.2</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Create climate stripes from a tabular input file</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.climate-lab-book.ac.uk/2018/warming-stripes/</td>
      <td>Climate Analysis, Visualization</td>
      <td>climate_stripes</td>
      <td>climate</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/climate-stripes</td>
    </tr>
    <tr>
      <td></td>
      <td>eodie</td>
      <td>1.0.2</td>
      <td></td>
      <td>eodie</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Earth Observation Data Information Extractor</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://eodie.readthedocs.io/</td>
      <td>Climate Analysis</td>
      <td>eodie</td>
      <td>climate</td>
      <td>https://gitlab.com/eetun-tiimi/EODIE</td>
    </tr>
    <tr>
      <td></td>
      <td>essential_climate_variables</td>
      <td>0.2.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Get Copernicus Essential Climate Variables for assessing climate variability</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://cds.climate.copernicus.eu/cdsapp#!/dataset/ecv-for-climate-change?tab=overview</td>
      <td>Climate Analysis, Data Source</td>
      <td>cds_essential_variability</td>
      <td>climate</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/essential_climate_variables</td>
    </tr>
    <tr>
      <td></td>
      <td>fates-emerald</td>
      <td>2.0</td>
      <td></td>
      <td>fates-emerald</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>EMERALD version of the Functionally Assembled Terrestrial Ecosystem Simulator (FATES) with Community Terrestrial Systems Model as host model</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/NordicESMhub/ctsm/blob/fates_emerald_api/README_fates_emerald_api</td>
      <td>Climate Analysis</td>
      <td>ctsm_fates</td>
      <td>climate</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/fates-emerald</td>
    </tr>
    <tr>
      <td></td>
      <td>mean-per-zone</td>
      <td>0.2.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Creates a png image showing statistic over areas as defined in the vector file</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/blob/master/tools/mean-per-zone/</td>
      <td>Visualization, GIS, Climate Analysis</td>
      <td>mean_per_zone</td>
      <td>climate</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/mean-per-zone</td>
    </tr>
    <tr>
      <td></td>
      <td>psy-maps</td>
      <td>1.2.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Visualization of regular geographical data on a map with psyplot</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/Chilipp/psy-maps</td>
      <td>Visualization, Climate Analysis</td>
      <td>psy_maps</td>
      <td>climate</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/psy-maps</td>
    </tr>
    <tr>
      <td></td>
      <td>shift-longitudes</td>
      <td>0.1.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Shift longitudes ranging from 0. and 360 degrees to -180. and 180. degrees</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/blob/master/tools/shift-longitudes/</td>
      <td>Climate Analysis</td>
      <td>shift_longitudes</td>
      <td>climate</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/shift-longitudes</td>
    </tr>
    <tr>
      <td></td>
      <td>smithsonian-volcanoes</td>
      <td>0.1.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Retrieve data from Volcanoes of the World (VOTW) Database</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://volcano.si.edu/gvp_votw.cfm</td>
      <td>Retrieve Data</td>
      <td>smithsonian_volcanoes</td>
      <td>climate</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/smithsonian-volcanoes</td>
    </tr>
    <tr>
      <td></td>
      <td>droplet-barcode-plot</td>
      <td>1.6.1+galaxy2</td>
      <td>0.0.1</td>
      <td>scxa-plots</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Make a cell barcode plot for droplet single-cell RNA-seq QC</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary</td>
      <td>Sequence Analysis</td>
      <td>droplet_barcode_plot</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/develop/tools/droplet-rank-plot/.shed.yml</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_provider</td>
      <td>0.4.4</td>
      <td>0.4.7</td>
      <td>atlas-fastq-provider</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Retrieval and download of FASTQ files from ENA and other repositories such as HCA.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ebi-gene-expression-group/atlas-fastq-provider</td>
      <td>Data Source, RNA, Transcriptomics</td>
      <td>atlas_fastq_provider</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/develop/tools/fastq_provider</td>
    </tr>
    <tr>
      <td></td>
      <td>gtf-2-gene-list</td>
      <td>1.52.0+galaxy0</td>
      <td>1.1.0</td>
      <td>atlas-gene-annotation-manipulation</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Utility to extract annotations from Ensembl GTF files.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary</td>
      <td>Sequence Analysis</td>
      <td>gtf2gene_list</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/develop/tools/gtf-2-gene-list/.shed.yml</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_pair</td>
      <td>1.0+galaxy0</td>
      <td>1.0</td>
      <td>fastq-pair</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Paired-end fastq pairer</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/linsalrob/fastq-pair</td>
      <td>Fastq Manipulation</td>
      <td>fastq_pair</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/develop/tools/qc/fastq_pair</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_quality_trimmer</td>
      <td>0.0.14+galaxy0</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FASTQ trimmer based on quality</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/agordon/fastx_toolkit</td>
      <td>Fastq Manipulation</td>
      <td>fastq_quality_trimmer</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/develop/tools/qc/fastq_quality_trimmer</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_utils</td>
      <td>0.25.1+galaxy0</td>
      <td>0.25.2</td>
      <td>fastq_utils</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Set of tools for handling fastq files</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/nunofonseca/fastq_utils</td>
      <td>Transcriptomics, RNA</td>
      <td>fastq_utils</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/develop/tools/qc/fastq_utils</td>
    </tr>
    <tr>
      <td></td>
      <td>salmon-kallisto-mtx-to-10x</td>
      <td>0.0.1+galaxy6</td>
      <td></td>
      <td>scipy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Transforms .mtx matrix and associated labels into a format compatible with tools expecting old-style 10X data</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary</td>
      <td>Sequence Analysis</td>
      <td>salmon_kallisto_mtx_to_10x</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/develop/tools/salmon-kallisto-mtx-to-10x/.shed.yml</td>
    </tr>
    <tr>
      <td></td>
      <td>cell-types-analysis</td>
      <td>1.1.1</td>
      <td>0.1.11</td>
      <td>cell-types-analysis</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tools for analysis of predictions from scRNAseq cell type classification tools, see https://github.com/ebi-gene-expression-group/cell-types-analysis</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>suite_cell_types_analysis</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
    </tr>
    <tr>
      <td></td>
      <td>data-hca</td>
      <td>v0.0.4+galaxy0</td>
      <td>0.0.4</td>
      <td>hca-matrix-downloader</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tools for interacting with the Human Cell Atlas resource https://prod.data.humancellatlas.org/explore/projects</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, Sequence Analysis</td>
      <td>suite_human_cell_atlas_tools</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
    </tr>
    <tr>
      <td></td>
      <td>data-scxa</td>
      <td>v0.0.2+galaxy2</td>
      <td></td>
      <td>wget</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tools for interacting with the EMBL-EBI Expression Atlas resource https://www.ebi.ac.uk/gxa/home https://www.ebi.ac.uk/gxa/sc/home</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, Sequence Analysis</td>
      <td>suite_ebi_expression_atlas</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
    </tr>
    <tr>
      <td></td>
      <td>decoupler</td>
      <td>1.4.0+galaxy1</td>
      <td>1.5.0</td>
      <td>decoupler</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>decoupler - Ensemble of methods to infer biological activities</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://decoupler-py.readthedocs.io/en/latest/</td>
      <td>Transcriptomics</td>
      <td>suite_decoupler</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
    </tr>
    <tr>
      <td></td>
      <td>dropletutils</td>
      <td>1.0.4</td>
      <td>0.0.5</td>
      <td>dropletutils-scripts</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>De-composed DropletUtils functionality tools, based on https://github.com/ebi-gene-expression-group/dropletutils-scripts and DropletUtils 1.0.3</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_dropletutils</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
    </tr>
    <tr>
      <td></td>
      <td>garnett</td>
      <td>0.2.8</td>
      <td>0.0.5</td>
      <td>garnett-cli</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>De-composed Garnett functionality tools, see https://github.com/ebi-gene-expression-group/garnett-cli and r-garnett 0.2.8</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_garnett</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
    </tr>
    <tr>
      <td></td>
      <td>monocle3</td>
      <td>0.1.4</td>
      <td>0.0.9</td>
      <td>monocle3-cli</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>De-composed monocle3 functionality tools, based on https://github.com/ebi-gene-expression-group/monocle-scripts and monocle3 0.1.2.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_monocle3</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
    </tr>
    <tr>
      <td></td>
      <td>sc3</td>
      <td>1.8.0</td>
      <td>0.0.6</td>
      <td>sc3-scripts</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>De-composed SC3 functionality tools, based on https://github.com/ebi-gene-expression-group/bioconductor-sc3-scripts and SC3 1.8.0.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_sc3</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
    </tr>
    <tr>
      <td></td>
      <td>scanpy</td>
      <td>1.9.3</td>
      <td>1.9.301</td>
      <td>scanpy-scripts</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>scanpy-scripts, command-line wrapper scripts around Scanpy.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://scanpy.readthedocs.io</td>
      <td>Transcriptomics, Sequence Analysis, RNA</td>
      <td>scanpy_scripts</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/develop/tools/tertiary-analysis/scanpy</td>
    </tr>
    <tr>
      <td></td>
      <td>scater</td>
      <td>1.10.0</td>
      <td>0.0.5</td>
      <td>scater-scripts</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>De-composed Scater functionality tools, based on https://github.com/ebi-gene-expression-group/bioconductor-scater-scripts and Scater 1.8.4.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_scater</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
    </tr>
    <tr>
      <td></td>
      <td>sccaf</td>
      <td>0.0.9</td>
      <td>0.0.10</td>
      <td>sccaf</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SCCAF: Single Cell Clustering Assessment Framework.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/sccaf/sccaf</td>
      <td>Transcriptomics</td>
      <td>SCCAF</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/develop/tools/tertiary-analysis/sccaf</td>
    </tr>
    <tr>
      <td></td>
      <td>sceasy</td>
      <td>0.0.5</td>
      <td>0.0.7</td>
      <td>r-sceasy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert scRNA data object between popular formats</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics</td>
      <td>sceasy</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
    </tr>
    <tr>
      <td></td>
      <td>scmap</td>
      <td>1.6.3</td>
      <td>0.1.0</td>
      <td>scmap-cli</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>De-composed scmap functionality tools, based on https://github.com/ebi-gene-expression-group/scmap-cli and scmap 1.6.0.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_scmap</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
    </tr>
    <tr>
      <td></td>
      <td>scpred</td>
      <td>1.0.2</td>
      <td>0.1.0</td>
      <td>scpred-cli</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>De-composed scPred functionality tools, see https://github.com/ebi-gene-expression-group/scpred-cli and r-scPred 1.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_scpred</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
    </tr>
    <tr>
      <td></td>
      <td>seurat</td>
      <td>4.0.0</td>
      <td>4.0.0</td>
      <td>seurat-scripts</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>De-composed Seurat functionality tools, based on https://github.com/ebi-gene-expression-group/r-seurat-scripts and Seurat 2.3.1</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_seurat</td>
      <td>ebi-gxa</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>ucsc-cell-browser</td>
      <td>1.0.0+galaxy1</td>
      <td>1.2.5</td>
      <td>ucsc-cell-browser</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Python pipeline and Javascript scatter plot library for single-cell datasets</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://cells.ucsc.edu/</td>
      <td>Transcriptomics</td>
      <td>ucsc_cell_browser</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/develop/tools/tertiary-analysis/ucsc-cell-browser/.shed.yml</td>
    </tr>
    <tr>
      <td></td>
      <td>biotransformer</td>
      <td>3.0.20230403</td>
      <td>3.0.20230403</td>
      <td>biotransformer</td>
      <td>Up-to-date</td>
      <td>biotransformer</td>
      <td>BioTransformer</td>
      <td>Metabolic pathway prediction, PTM site prediction, Natural product identification</td>
      <td>Small molecules, Endocrinology and metabolism, Metabolomics, Carbohydrates, NMR</td>
      <td>BioTransformer is a tool for prediction of small molecule metabolism in mammals.</td>
      <td>BioTransformer is a freely available web server that supports accurate, rapid and comprehensive in silico metabolism prediction.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://bitbucket.org/djoumbou/biotransformerjar/src/master/</td>
      <td>Metabolomics</td>
      <td>biotransformer</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/biotransformer</td>
    </tr>
    <tr>
      <td></td>
      <td>filter_compounds</td>
      <td>3.1.1</td>
      <td>2.3.90dev7d621d9</td>
      <td>openbabel</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tool for filtering organometallics/anorganic compounds from a list of compounds.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/RECETOX/galaxytools/</td>
      <td>Metabolomics</td>
      <td>filter_compounds</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/filter_compounds</td>
    </tr>
    <tr>
      <td></td>
      <td>matchms</td>
      <td>0.24.0</td>
      <td>0.24.2</td>
      <td>matchms</td>
      <td>To update</td>
      <td>matchms</td>
      <td>Matchms</td>
      <td>Spectral library search, Format validation, Filtering</td>
      <td>Metabolomics</td>
      <td>Searching, filtering and converting mass spectral libraries.</td>
      <td>Tool to import, process, clean, and compare mass spectrometry data.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/matchms/matchms</td>
      <td>Metabolomics</td>
      <td>matchms</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/matchms</td>
    </tr>
    <tr>
      <td></td>
      <td>misc</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/RECETOX/galaxytools</td>
      <td>Metabolomics</td>
      <td></td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/misc</td>
    </tr>
    <tr>
      <td></td>
      <td>msmetaenhancer</td>
      <td>0.3.0</td>
      <td>0.3.0</td>
      <td>msmetaenhancer</td>
      <td>Up-to-date</td>
      <td>msmetaenhancer</td>
      <td>MSMetaEnhancer</td>
      <td>Annotation, Standardisation and normalisation</td>
      <td>Metabolomics, Compound libraries and screening, Data submission, annotation and curation</td>
      <td></td>
      <td>Tool for mass spectra metadata annotation.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/RECETOX/MSMetaEnhancer</td>
      <td>Metabolomics</td>
      <td></td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/msmetaenhancer</td>
    </tr>
    <tr>
      <td></td>
      <td>msp_merge</td>
      <td>0.1.0</td>
      <td>0.24.2</td>
      <td>matchms</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/RECETOX/galaxytools</td>
      <td>Metabolomics</td>
      <td></td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/msp_merge</td>
    </tr>
    <tr>
      <td></td>
      <td>mzml_validator</td>
      <td>0.1.0+galaxy2</td>
      <td></td>
      <td>lxml</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>mzML Validator checks if mzML file validates against XML Schema Definition of HUPO Proteomics Standard Initiative.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/RECETOX/galaxytools</td>
      <td>Metabolomics, Proteomics</td>
      <td></td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/mzml_validator</td>
    </tr>
    <tr>
      <td></td>
      <td>qcxms</td>
      <td>5.2.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>QCxMS is a quantum chemical (QC) based program that enables users to calculate mass spectra (MS) using Born-Oppenheimer Molecular Dynamics (MD).</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/grimme-lab/QCxMS</td>
      <td>Computational chemistry, Molecular Dynamics</td>
      <td>QCxMS</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/qcxms</td>
    </tr>
    <tr>
      <td></td>
      <td>query</td>
      <td>0.2</td>
      <td></td>
      <td>click</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Execute an SQL statement on a set of tables</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>query</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/query</td>
    </tr>
    <tr>
      <td></td>
      <td>ramclustr</td>
      <td>1.3.0</td>
      <td>1.3.1</td>
      <td>r-ramclustr</td>
      <td>To update</td>
      <td>ramclustr</td>
      <td>RAMClustR</td>
      <td>Imputation, Standardisation and normalisation, Clustering, Correlation</td>
      <td>Metabolomics</td>
      <td></td>
      <td>A feature clustering algorithm for non-targeted mass spectrometric metabolomics data.</td>
      <td></td>
      <td>To update</td>
      <td>https://rdrr.io/cran/RAMClustR/</td>
      <td>Metabolomics</td>
      <td></td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/ramclustr</td>
    </tr>
    <tr>
      <td></td>
      <td>recetox_aplcms</td>
      <td>0.12.0</td>
      <td>0.12.0</td>
      <td>r-recetox-aplcms</td>
      <td>Up-to-date</td>
      <td>recetox-aplcms</td>
      <td>recetox-aplcms</td>
      <td>Chromatographic alignment, Quantification, Peak detection, Feature extraction, Alignment</td>
      <td>Metabolomics</td>
      <td>Peak detection tool for HRMS profile data.</td>
      <td>recetox-aplcms is a tool for peak detection in mass spectrometry data. The tool performs (1) noise removal, (2) peak detection, (3) retention time drift correction, (4) peak alignment and (5) weaker signal recovery as well as (6) suspect screening.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/RECETOX/recetox-aplcms</td>
      <td>Metabolomics</td>
      <td>recetox-aplcms</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/recetox_aplcms</td>
    </tr>
    <tr>
      <td></td>
      <td>recetox_msfinder</td>
      <td>v3.5.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>recetox-msfinder</td>
      <td>recetox-msfinder</td>
      <td>Annotation</td>
      <td>Metabolomics</td>
      <td></td>
      <td>This is a modified copy of MS-FINDER with source code modifications to make the tool accessible in Galaxy.MS-FINDER - software for structure elucidation of unknown spectra with hydrogen rearrangement (HR) rulesThe program supports molecular formula prediction, metabolie class prediction, and structure elucidation for EI-MS and MS/MS spectra, and the assembly is licensed under the CC-BY 4.0.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/RECETOX/recetox-msfinder</td>
      <td>Metabolomics</td>
      <td></td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/recetox_msfinder</td>
    </tr>
    <tr>
      <td></td>
      <td>recetox_xmsannotator</td>
      <td>0.10.0</td>
      <td>0.10.0</td>
      <td>r-recetox-xmsannotator</td>
      <td>Up-to-date</td>
      <td>recetox-xmsannotator</td>
      <td>recetox-xMSannotator</td>
      <td>Expression profile pathway mapping, Structure comparison, Isotopic distributions calculation, Annotation</td>
      <td></td>
      <td></td>
      <td>Annotation tool for untargeted LCMS1 data. Uses a database and adduct list for compound annotation and intensity networks, isotopic patterns and pathways for annotation scoring.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/RECETOX/recetox-xMSannotator</td>
      <td>Metabolomics</td>
      <td></td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/recetox-xmsannotator</td>
    </tr>
    <tr>
      <td></td>
      <td>rem_complex</td>
      <td>1.0.0</td>
      <td></td>
      <td>pandas</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Removes molecular coordination complexes.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/RECETOX/galaxytools</td>
      <td>Metabolomics</td>
      <td>rem_complex</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/rem_complex</td>
    </tr>
    <tr>
      <td></td>
      <td>retip</td>
      <td>0.5.4</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>retip</td>
      <td>Retip</td>
      <td>Retention time prediction, Spectrum calculation, Deisotoping, Formatting, Deposition</td>
      <td>Metabolomics, Proteomics experiment, Machine learning, Cheminformatics, Chemistry</td>
      <td></td>
      <td>Retention Time Prediction for Compound Annotation in Untargeted Metabolomics.Retip is an R package for predicting Retention Time (RT) for small molecules in a high pressure liquid chromatography (HPLC) Mass Spectrometry analysis.Retip - Retention Time prediction for Metabolomics.Retip: Retention Time Prediction for Compound Annotation in Untargeted Metabolomics Paolo Bonini, Tobias Kind, Hiroshi Tsugawa, Dinesh Kumar Barupal, and Oliver Fiehn Analytical Chemistry 2020 92 (11), 7515-7522 DOI: 10.1021/acs.analchem.9b05765.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/PaoloBnn/Retip</td>
      <td>Metabolomics</td>
      <td></td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/retip</td>
    </tr>
    <tr>
      <td></td>
      <td>riassigner</td>
      <td>0.4.0</td>
      <td>0.4.0</td>
      <td>riassigner</td>
      <td>Up-to-date</td>
      <td>riassigner</td>
      <td>RIAssigner</td>
      <td>Standardisation and normalisation</td>
      <td>Metabolomics, Compound libraries and screening, Data submission, annotation and curation</td>
      <td></td>
      <td>RIAssigner is a python tool for retention index (RI) computation for GC-MS data.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/RECETOX/RIAssigner</td>
      <td>Metabolomics</td>
      <td></td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/riassigner</td>
    </tr>
    <tr>
      <td></td>
      <td>rmassbank</td>
      <td>3.0.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>RMassBank is an R package for processing tandem MS files and building of MassBank records.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/MassBank/RMassBank</td>
      <td>Metabolomics</td>
      <td>rmassbank</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/rmassbank</td>
    </tr>
    <tr>
      <td></td>
      <td>spec2vec</td>
      <td>0.8.0</td>
      <td>0.8.0</td>
      <td>spec2vec</td>
      <td>Up-to-date</td>
      <td>spec2vec</td>
      <td>Spec2Vec</td>
      <td>Spectrum calculation, Spectral library search, Database search, Natural product identification</td>
      <td>Proteomics experiment, Metabolomics, Natural language processing, Proteomics</td>
      <td>Mass spectra similarity scoring using a trained Spec2Vec model.</td>
      <td>Improved mass spectral similarity scoring through learning of structural relationships.Spec2vec is a novel spectral similarity score inspired by a natural language processing algorithm -- Word2Vec. Where Word2Vec learns relationships between words in sentences, spec2vec does so for mass fragments and neutral losses in MS/MS spectra. The spectral similarity score is based on spectral embeddings learnt from the fragmental relationships within a large set of spectral data.Analysis and benchmarking of mass spectra similarity measures using gnps data set.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/iomega/spec2vec</td>
      <td>Metabolomics</td>
      <td>spec2vec</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/spec2vec</td>
    </tr>
    <tr>
      <td></td>
      <td>waveica</td>
      <td>0.2.0</td>
      <td>0.2.0</td>
      <td>r-recetox-waveica</td>
      <td>Up-to-date</td>
      <td>waveica</td>
      <td>WaveICA</td>
      <td>Standardisation and normalisation</td>
      <td>Metabolomics</td>
      <td>Removal of batch effects for large-scale untargeted metabolomics data based on wavelet analysis.</td>
      <td>Removal of batch effects for large-scale untargeted metabolomics data based on wavelet transform.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/RECETOX/WaveICA</td>
      <td>Metabolomics</td>
      <td></td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/waveica</td>
    </tr>
    <tr>
      <td></td>
      <td>xtb</td>
      <td>6.6.1</td>
      <td></td>
      <td>xtb</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Performs semiempirical molecular optimization.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/grimme-lab/xtb</td>
      <td>Metabolomics</td>
      <td>xtb_molecular_optimization</td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/xtb</td>
    </tr>
    <tr>
      <td></td>
      <td>consolidate_vcfs</td>
      <td>1.8.2</td>
      <td>1.8.2</td>
      <td>snvphyl-tools</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Combines freebayes and mpileup files for use by vcf2snvalignment</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://snvphyl.readthedocs.io/en/latest/</td>
      <td>Sequence Analysis</td>
      <td>consolidate_vcfs</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy</td>
    </tr>
    <tr>
      <td></td>
      <td>filter_density</td>
      <td>1.8.2</td>
      <td>1.8.2</td>
      <td>snvphyl-tools</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter out position based on distance between SNVs</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://snvphyl.readthedocs.io/en/latest/</td>
      <td>Sequence Analysis</td>
      <td>filter_density</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy</td>
    </tr>
    <tr>
      <td></td>
      <td>filter_stats</td>
      <td>1.8.2</td>
      <td>1.8.2</td>
      <td>snvphyl-tools</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SNVPhyl filter_stats</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://snvphyl.readthedocs.io/en/latest/</td>
      <td>Sequence Analysis</td>
      <td>filter_stats</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy</td>
    </tr>
    <tr>
      <td></td>
      <td>filter_vcf</td>
      <td>1.8.2</td>
      <td>1.8.2</td>
      <td>snvphyl-tools</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SNVPhyl filter_vcf</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://snvphyl.readthedocs.io/en/latest/</td>
      <td>Sequence Analysis</td>
      <td>filter_vcf</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy</td>
    </tr>
    <tr>
      <td></td>
      <td>find_repeats</td>
      <td>1.8.2</td>
      <td>1.8.2</td>
      <td>snvphyl-tools</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Find repetitive regions on a reference genome using MUMMer</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://snvphyl.readthedocs.io/en/latest/</td>
      <td>Sequence Analysis</td>
      <td>find_repeats</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy</td>
    </tr>
    <tr>
      <td></td>
      <td>positions2snv_alignment</td>
      <td>1.8.2</td>
      <td>1.8.2</td>
      <td>snvphyl-tools</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Generate alignment of SNVs from SNVPhyl variant table.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://snvphyl.readthedocs.io/en/latest/</td>
      <td>Variant Analysis</td>
      <td>positions2snv_alignment</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy</td>
    </tr>
    <tr>
      <td></td>
      <td>positions2snv_invariant_alignment</td>
      <td>1.8.2</td>
      <td>1.8.2</td>
      <td>snvphyl-tools</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Generate alignment of SNVs and non-variant positions from SNVPhyl variant table.</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://snvphyl.readthedocs.io/en/latest/</td>
      <td>Variant Analysis</td>
      <td>positions2snv_invariant_alignment</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy</td>
    </tr>
    <tr>
      <td></td>
      <td>snv_matrix</td>
      <td>1.8.2</td>
      <td>1.8.2</td>
      <td>snvphyl-tools</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Generate matrix of SNV distances</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://snvphyl.readthedocs.io/en/latest/</td>
      <td>Sequence Analysis</td>
      <td>snv_matrix</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy</td>
    </tr>
    <tr>
      <td></td>
      <td>vcf2snvalignment</td>
      <td>1.8.2</td>
      <td>1.8.2</td>
      <td>snvphyl-tools</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Generates multiple alignment of variant calls</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://snvphyl.readthedocs.io/en/latest/</td>
      <td>Sequence Analysis</td>
      <td>vcf2snvalignment</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy</td>
    </tr>
    <tr>
      <td></td>
      <td>verify_map</td>
      <td>1.8.2</td>
      <td>1.8.2</td>
      <td>snvphyl-tools</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Checks the mapping quality of all BAM(s)</td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://snvphyl.readthedocs.io/en/latest/</td>
      <td>Sequence Analysis</td>
      <td>verify_map</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy</td>
    </tr>
    <tr>
      <td></td>
      <td>suite_snvphyl</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SNVPhyl suite defining all dependencies for SNVPhyl</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>suite_snvphyl_1_2_3</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy</td>
    </tr>
    <tr>
      <td></td>
      <td>cooler</td>
      <td>0.9.3</td>
      <td>1.19.1</td>
      <td>htslib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>cooler different tools to process Hi-C from mirnylab</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/open2c/cooler</td>
      <td>Epigenetics</td>
      <td>cooler</td>
      <td>lldelisle</td>
      <td>https://github.com/lldelisle/tools-lldelisle/blob/master/tools/cooler/.shed.yml</td>
    </tr>
    <tr>
      <td></td>
      <td>fromHicupToJuicebox</td>
      <td>0.0.2</td>
      <td>0.22.0</td>
      <td>pysam</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert the output of hicup (as sam or bam) to the input of juicebox.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Epigenetics</td>
      <td>from_hicup_to_juicebox</td>
      <td>lldelisle</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>fromgtfTobed12</td>
      <td>0.11.1+galaxy1</td>
      <td>0.12</td>
      <td>gffutils</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert GTF files to BED12 format</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://pythonhosted.org/gffutils/contents.html</td>
      <td>Convert Formats</td>
      <td>fromgtftobed12</td>
      <td>lldelisle</td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/fromgtfTobed12</td>
    </tr>
    <tr>
      <td></td>
      <td>getTn5ExtendedCoverage</td>
      <td>0.0.2</td>
      <td>0.22.0</td>
      <td>pysam</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Take an input bam from ATAC-seq and generate a bedgraph using the center of the Tn5 insertion with an extension</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Epigenetics</td>
      <td>gettn5extendedcoverage</td>
      <td>lldelisle</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>hyperstack_to_bleach_corrected_movie</td>
      <td>20230328</td>
      <td>20231211</td>
      <td>Fiji</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Generate blach corrected movie from hyperstack</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Imaging</td>
      <td>hyperstack_to_bleach_corrected_movie</td>
      <td>lldelisle</td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/hyperstack_to_bleach_corrected_movie</td>
    </tr>
    <tr>
      <td></td>
      <td>incucyte_stack_and_upload_omero</td>
      <td>20231221</td>
      <td>20231211</td>
      <td>Fiji</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Combine images to stack and upload to the omero server</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Imaging</td>
      <td>incucyte_stack_and_upload_omero</td>
      <td>lldelisle</td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/incucyte_stack_and_upload_omero</td>
    </tr>
    <tr>
      <td></td>
      <td>measure_gastruloids</td>
      <td>20221216</td>
      <td>20231211</td>
      <td>fiji</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Get the ROI coordinates around the gastruloids as well as measurements like Area, elongation index</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Imaging</td>
      <td>measure_gastruloids</td>
      <td>lldelisle</td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/measure_gastruloids</td>
    </tr>
    <tr>
      <td></td>
      <td>omero_clean_rois_tables</td>
      <td>20230623</td>
      <td>20231211</td>
      <td>fiji</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Remove all ROIs and all tables on OMERO associated to an omero object and recursively up and down</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Imaging</td>
      <td>omero_clean_rois_tables</td>
      <td>lldelisle</td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/omero_clean_rois_tables</td>
    </tr>
    <tr>
      <td></td>
      <td>omero_get_children_ids</td>
      <td>0.1.0</td>
      <td>5.11.1</td>
      <td>omero-py</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Get omero id of children of an omero object id</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Imaging</td>
      <td>omero_get_children_ids</td>
      <td>lldelisle</td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/omero_get_children_ids</td>
    </tr>
    <tr>
      <td></td>
      <td>omero_hyperstack_to_fluo_measurements_on_gastruloid</td>
      <td>20230809</td>
      <td>20231211</td>
      <td>fiji</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Analyse Hyperstack on OMERO server to measure fluorescence levels</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Imaging</td>
      <td>omero_hyperstack_to_fluo_measurements_on_gastruloid</td>
      <td>lldelisle</td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/omero_hyperstack_to_fluo_measurements_on_gastruloid</td>
    </tr>
    <tr>
      <td></td>
      <td>omero_hyperstack_to_gastruloid_measurements</td>
      <td>20240214</td>
      <td>20231211</td>
      <td>fiji</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Analyse Hyperstack on OMERO server to segment gastruloid and compute measurements</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Imaging</td>
      <td>omero_hyperstack_to_gastruloid_measurements</td>
      <td>lldelisle</td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/omero_hyperstack_to_gastruloid_measurements</td>
    </tr>
    <tr>
      <td></td>
      <td>revertR2orientationInBam</td>
      <td>0.0.2</td>
      <td>1.19.2</td>
      <td>samtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Revert the mapped orientation of R2 mates in a bam.</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>SAM</td>
      <td>revertr2orientationinbam</td>
      <td>lldelisle</td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/revertR2orientationInBam</td>
    </tr>
    <tr>
      <td></td>
      <td>upload_roi_and_measures_to_omero</td>
      <td>0.0.5</td>
      <td>5.11.1</td>
      <td>omero-py</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Upload the ROI coordinates and the measurements to the omero server</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Imaging</td>
      <td>upload_roi_and_measures_to_omero</td>
      <td>lldelisle</td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/upload_roi_and_measures_to_omero</td>
    </tr>
    <tr>
      <td></td>
      <td>blast2go</td>
      <td>0.0.11</td>
      <td></td>
      <td>b2g4pipe</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Maps BLAST results to GO annotation terms</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/blast2go</td>
      <td>Ontology Manipulation, Sequence Analysis</td>
      <td>blast2go</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/blast2go</td>
    </tr>
    <tr>
      <td></td>
      <td>blast_rbh</td>
      <td>0.3.0</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>BLAST Reciprocal Best Hits (RBH) from two FASTA files</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/blast_rbh</td>
      <td>Fasta Manipulation, Sequence Analysis</td>
      <td>blast_rbh</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/blast_rbh</td>
    </tr>
    <tr>
      <td></td>
      <td>blastxml_to_top_descr</td>
      <td>0.1.2</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Make table of top BLAST match descriptions</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/blastxml_to_top_descr</td>
      <td>Convert Formats, Sequence Analysis, Text Manipulation</td>
      <td>blastxml_to_top_descr</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/blastxml_to_top_descr</td>
    </tr>
    <tr>
      <td></td>
      <td>make_nr</td>
      <td>0.0.2</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Make a FASTA file non-redundant</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/make_nr</td>
      <td>Fasta Manipulation, Sequence Analysis</td>
      <td>make_nr</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/make_nr</td>
    </tr>
    <tr>
      <td></td>
      <td>ncbi_blast_plus</td>
      <td>2.14.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>NCBI BLAST+</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://blast.ncbi.nlm.nih.gov/</td>
      <td>Sequence Analysis</td>
      <td>ncbi_blast_plus</td>
      <td>devteam</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/ncbi_blast_plus</td>
    </tr>
  </tbody>
</table>

</div>