<script src="https://code.jquery.com/jquery-3.7.0.js"></script>
<script src="https://cdn.datatables.net/1.13.6/js/jquery.dataTables.min.js"></script>
<script src="https://cdn.datatables.net/searchbuilder/1.6.0/js/dataTables.searchBuilder.min.js"></script>
<script src="https://cdn.datatables.net/datetime/1.5.1/js/dataTables.dateTime.min.js"></script>

<script src="https://cdn.datatables.net/responsive/2.5.0/js/dataTables.responsive.min.js"></script>

<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/responsive/2.2.3/css/responsive.dataTables.min.css">

<link rel="stylesheet" href="https://cdn.datatables.net/1.13.6/css/jquery.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/searchbuilder/1.6.0/css/searchBuilder.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/datetime/1.5.1/css/dataTables.dateTime.min.css">

<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" />
<script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>

<style type="text/css">
  .tooltip {
    font-family: Georgia;
    font-size: 18px;
  }
  .tooltip .tooltip-inner {
    background-color: #00FFFF;
    color: #000000;
    min-width: 400px;
  }
  </style>

<script>
    $(document).ready(function() {
    $('#dataframe').DataTable( {
        "sScrollX": "100%",
        "dom": "Qlfrtip",

        responsive: {
            details: {
                type: 'column'
            }
        },
        "bAutoWidth": false,
        columnDefs: [ {
            className: 'control',
            orderable: false,
            targets:   [ 0 ],
            width: 100,
        },
        { type: 'natural', targets: 1 } // allow normal sorting
        ],

    });
});
</script>

<div class="display nowrap" style="width:80%" width="100%">

<table class="dataframe display nowrap" id="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th>Expand</th>
      <th>Galaxy wrapper id</th>
      <th>Galaxy wrapper version</th>
      <th>Conda version</th>
      <th>Conda id</th>
      <th>Status</th>
      <th>bio.tool id</th>
      <th>bio.tool name</th>
      <th>EDAM operation</th>
      <th>EDAM topic</th>
      <th>Description</th>
      <th>bio.tool description</th>
      <th>Status</th>
      <th>Source</th>
      <th>ToolShed categories</th>
      <th>ToolShed id</th>
      <th>Galaxy wrapper owner</th>
      <th>Galaxy wrapper source</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td></td>
      <td>apoc</td>
      <td>1.0+galaxy1</td>
      <td>1b16</td>
      <td>apoc</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Large-scale structural comparison of protein pockets</td>
      <td></td>
      <td>To update</td>
      <td>http://cssb.biology.gatech.edu/APoc</td>
      <td>Computational chemistry</td>
      <td>apoc</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/apoc/</td>
    </tr>
    <tr>
      <td></td>
      <td>blast_parser</td>
      <td>0.1.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert 12- or 24-column BLAST output into 3-column hcluster_sg input</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/TGAC/earlham-galaxytools/</td>
      <td>Phylogenetics</td>
      <td>blast_parser</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/blast_parser</td>
    </tr>
    <tr>
      <td></td>
      <td>braker</td>
      <td>2.1.6</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>BRAKER is a pipeline for fully automated prediction of protein coding gene structures with GeneMark-ES/ET and AUGUSTUS in novel eukaryotic genomes .</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/Gaius-Augustus/BRAKER</td>
      <td>Genome annotation</td>
      <td>braker</td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/braker</td>
    </tr>
    <tr>
      <td></td>
      <td>braker3</td>
      <td>3.0.6</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>BRAKER3 is a pipeline for fully automated prediction of protein coding gene structures with GeneMark-ES/ET and AUGUSTUS in novel eukaryotic genomes .</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/Gaius-Augustus/BRAKER</td>
      <td>Genome annotation</td>
      <td>braker3</td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/braker</td>
    </tr>
    <tr>
      <td></td>
      <td>curl_post</td>
      <td>0.0.2</td>
      <td></td>
      <td>curl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Send file via cURL POST</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Data Export, Web Services</td>
      <td>curl_post</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/curl_post</td>
    </tr>
    <tr>
      <td></td>
      <td>ectyper</td>
      <td>1.0.0</td>
      <td>1.0.0</td>
      <td>ectyper</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>EC-Typer - in silico serotyping of Escherichia coli species</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/phac-nml/ecoli_serotyping</td>
      <td>Sequence Analysis</td>
      <td>ectyper</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/ecoli_serotyping</td>
    </tr>
    <tr>
      <td></td>
      <td>gstf_preparation</td>
      <td>0.4.3</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>GeneSeqToFamily preparation converts data for the workflow</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/TGAC/earlham-galaxytools/</td>
      <td>Convert Formats</td>
      <td>gstf_preparation</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/gstf_preparation</td>
    </tr>
    <tr>
      <td></td>
      <td>hcluster_sg</td>
      <td>0.5.1.1</td>
      <td>0.5.1</td>
      <td>hcluster_sg</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Hierarchically clustering on a sparse graph</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/douglasgscofield/hcluster</td>
      <td>Phylogenetics</td>
      <td>hcluster_sg</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/hcluster_sg</td>
    </tr>
    <tr>
      <td></td>
      <td>hcluster_sg_parser</td>
      <td>0.2.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Converts hcluster_sg 3-column output into lists of ids</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/TGAC/earlham-galaxytools/</td>
      <td>Phylogenetics</td>
      <td>hcluster_sg_parser</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/hcluster_sg_parser</td>
    </tr>
    <tr>
      <td></td>
      <td>helixer</td>
      <td>0.3.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Gene calling with Deep Neural Networks</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/weberlab-hhu/Helixer</td>
      <td>Genome annotation</td>
      <td>helixer</td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/helixer</td>
    </tr>
    <tr>
      <td></td>
      <td>imagecoordinates_flipaxis</td>
      <td>0.1-2</td>
      <td></td>
      <td>pandas</td>
      <td>To update</td>
      <td>galaxy_image_analysis</td>
      <td>Galaxy Image Analysis</td>
      <td>Image analysis</td>
      <td>Imaging, Bioinformatics</td>
      <td>Flip coordinate axes</td>
      <td>Developed within the Biomedical Computer Vision (BMCV) Group Heidelberg.</td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Imaging</td>
      <td>imagecoordinates_flipaxis</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/imagecoordinates_flipaxis/</td>
    </tr>
    <tr>
      <td></td>
      <td>isoplot</td>
      <td>1.3.0+galaxy1</td>
      <td>1.3.1</td>
      <td>isoplot</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Isoplot is a software for the visualisation of MS data from C13 labelling experiments</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Metabolomics, Visualization</td>
      <td>isoplot</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/llegregam/Isoplot/tree/main</td>
    </tr>
    <tr>
      <td></td>
      <td>jbrowse</td>
      <td></td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A tool allowing to export a JBrowse dataset into a JBrowse docker container</td>
      <td></td>
      <td>To update</td>
      <td>https://jbrowse.org</td>
      <td>Web Services</td>
      <td>jbrowse_to_container</td>
      <td>gga</td>
      <td>https://github.com/galaxy-genome-annotation/galaxy-tools/tree/master/tools/jbrowse</td>
    </tr>
    <tr>
      <td></td>
      <td>lotus2</td>
      <td>2.30</td>
      <td>2.31</td>
      <td>lotus2</td>
      <td>To update</td>
      <td>lotus2</td>
      <td>lotus2</td>
      <td>Sequence feature detection</td>
      <td>Metagenomics</td>
      <td>LotuS2 OTU processing pipeline</td>
      <td>LotuS2 is a lightweight and user-friendly pipeline that is fast, precise, and streamlined, using extensive pre- and post-ASV/OTU clustering steps to further increase data quality. High data usage rates and reliability enable high-throughput microbiome analysis in minutes.</td>
      <td>To update</td>
      <td>http://lotus2.earlham.ac.uk/</td>
      <td>Metagenomics</td>
      <td>lotus2</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/lotus2</td>
    </tr>
    <tr>
      <td></td>
      <td>miranda</td>
      <td>3.3a+galaxy1</td>
      <td>3.3a</td>
      <td>miranda</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Finds potential target sites for miRNAs in genomic sequences</td>
      <td></td>
      <td>To update</td>
      <td>http://www.microrna.org/</td>
      <td>RNA</td>
      <td>miranda</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/miranda</td>
    </tr>
    <tr>
      <td></td>
      <td>normalization</td>
      <td>1.0.7</td>
      <td>1.1_4</td>
      <td>r-batch</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>[Metabolomics][W4M][ALL] Normalization (operation applied on each individual spectrum) of preprocessed data</td>
      <td></td>
      <td>To update</td>
      <td>http://workflow4metabolomics.org</td>
      <td>Metabolomics</td>
      <td>normalization</td>
      <td>marie-tremblay-metatoul</td>
      <td>https://github.com/workflow4metabolomics/normalization</td>
    </tr>
    <tr>
      <td></td>
      <td>pangolin</td>
      <td>1.1.14</td>
      <td>4.3</td>
      <td>pangolin</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Phylogenetic Assignment of Named Global Outbreak LINeages</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/hCoV-2019/pangolin</td>
      <td>Sequence Analysis</td>
      <td>pangolin</td>
      <td>nml</td>
      <td>https://github.com/hCoV-2019/pangolin</td>
    </tr>
    <tr>
      <td></td>
      <td>replace_chromosome_names</td>
      <td>0.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Replace chromosome names</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>replace_chromosome_names</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/replace_chromosome_names/</td>
    </tr>
    <tr>
      <td></td>
      <td>sistr_cmd</td>
      <td>1.1.1</td>
      <td>1.1.1</td>
      <td>sistr_cmd</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SISTR in silico serotyping tool</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/phac-nml/sistr_cmd</td>
      <td>Sequence Analysis</td>
      <td>sistr_cmd</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>smina</td>
      <td>1.0</td>
      <td>2017.11.9</td>
      <td>smina</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>smina is a fork of Autodock Vina (http://vina.scripps.edu/) that focuses on improving scoring and minimization</td>
      <td></td>
      <td>To update</td>
      <td>https://sourceforge.net/projects/smina/</td>
      <td>Computational chemistry</td>
      <td>smina</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/smina/</td>
    </tr>
    <tr>
      <td></td>
      <td>srst2</td>
      <td>0.3.7</td>
      <td>0.2.0</td>
      <td>srst2</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Short Read Sequence Typing for Bacterial Pathogens</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>srst2</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>t_coffee</td>
      <td>13.45.0.4846264</td>
      <td>13.46.0.919e8c6b</td>
      <td>t-coffee</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>T-Coffee</td>
      <td></td>
      <td>To update</td>
      <td>http://www.tcoffee.org/</td>
      <td>Sequence Analysis</td>
      <td>t_coffee</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/t_coffee</td>
    </tr>
    <tr>
      <td></td>
      <td>unzip</td>
      <td>6.0</td>
      <td></td>
      <td>unzip</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Unzip file</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bmcv</td>
      <td>Convert Formats</td>
      <td>unzip</td>
      <td>imgteam</td>
      <td>https://github.com/BMCV/galaxy-image-analysis/tree/master/tools/unzip/</td>
    </tr>
    <tr>
      <td></td>
      <td>w4mclassfilter</td>
      <td>0.98.19</td>
      <td></td>
      <td>r-base</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter W4M data by values or metadata</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/HegemanLab/w4mclassfilter_galaxy_wrapper</td>
      <td>Metabolomics</td>
      <td>w4mclassfilter</td>
      <td>eschen42</td>
      <td>https://github.com/HegemanLab/w4mclassfilter_galaxy_wrapper/tree/master</td>
    </tr>
    <tr>
      <td></td>
      <td>w4mcorcov</td>
      <td>0.98.18</td>
      <td></td>
      <td>r-base</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>OPLS-DA Contrasts of Univariate Results</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/HegemanLab/w4mcorcov_galaxy_wrapper</td>
      <td>Metabolomics</td>
      <td>w4mcorcov</td>
      <td>eschen42</td>
      <td>https://github.com/HegemanLab/w4mcorcov_galaxy_wrapper/tree/master</td>
    </tr>
    <tr>
      <td></td>
      <td>w4mjoinpn</td>
      <td>0.98.2</td>
      <td>8.25</td>
      <td>coreutils</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Join positive- and negative-mode W4M datasets</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/HegemanLab/w4mjoinpn_galaxy_wrapper</td>
      <td>Metabolomics</td>
      <td>w4mjoinpn</td>
      <td>eschen42</td>
      <td>https://github.com/HegemanLab/w4mjoinpn_galaxy_wrapper/tree/master</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy wrapper id</td>
      <td>Galaxy wrapper version</td>
      <td>Conda version</td>
      <td>Conda id</td>
      <td>Status</td>
      <td>bio.tool id</td>
      <td>bio.tool name</td>
      <td>EDAM operation</td>
      <td>EDAM topic</td>
      <td>Description</td>
      <td>bio.tool description</td>
      <td>Status</td>
      <td>Source</td>
      <td>ToolShed categories</td>
      <td>ToolShed id</td>
      <td>Galaxy wrapper owner</td>
      <td>Galaxy wrapper source</td>
    </tr>
    <tr>
      <td></td>
      <td>10x_bamtofastq</td>
      <td>1.4.1</td>
      <td>1.4.1</td>
      <td>10x_bamtofastq</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Converts 10x Genomics BAM to FASTQ</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/10XGenomics/bamtofastq</td>
      <td>Convert Formats</td>
      <td>10x_bamtofastq</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/10x_bamtofastq</td>
    </tr>
    <tr>
      <td></td>
      <td>add_line_to_file</td>
      <td>0.1.0</td>
      <td>8.25</td>
      <td>coreutils</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Adds a text line to the beginning or end of a file.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>add_line_to_file</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/text_processing/add_line_to_file</td>
    </tr>
    <tr>
      <td></td>
      <td>agat</td>
      <td>1.2.0</td>
      <td>1.2.0</td>
      <td>agat</td>
      <td>Up-to-date</td>
      <td>agat</td>
      <td>AGAT</td>
      <td>Data handling, Genome annotation</td>
      <td>Genomics</td>
      <td>GTF/GFF analysis toolkit</td>
      <td>Another Gff Analysis Toolkit (AGAT)Suite of tools to handle gene annotations in any GTF/GFF format.</td>
      <td>Up-to-date</td>
      <td>https://github.com/NBISweden/AGAT</td>
      <td>Convert Formats, Statistics, Fasta Manipulation</td>
      <td>agat</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/agat</td>
    </tr>
    <tr>
      <td></td>
      <td>align_back_trans</td>
      <td>0.0.10</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Thread nucleotides onto a protein alignment (back-translation)</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/align_back_trans</td>
      <td>Fasta Manipulation, Sequence Analysis</td>
      <td>align_back_trans</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/align_back_trans</td>
    </tr>
    <tr>
      <td></td>
      <td>antarna</td>
      <td>1.1</td>
      <td>2.0.1.2</td>
      <td>antarna</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>antaRNA uses ant colony optimization to solve the inverse folding problem in RNA research .</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>antarna</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/antarna/</td>
    </tr>
    <tr>
      <td></td>
      <td>antismash</td>
      <td>6.1.1</td>
      <td>6.1.1</td>
      <td>antismash</td>
      <td>Up-to-date</td>
      <td>antismash</td>
      <td>antiSMASH</td>
      <td>Sequence clustering, Gene prediction, Differential gene expression analysis</td>
      <td>Molecular interactions, pathways and networks, Gene and protein families</td>
      <td>Antismash allows the genome-wide identification, annotation and analysis of secondary metabolite biosynthesis gene clusters</td>
      <td>Rapid genome-wide identification, annotation and analysis of secondary metabolite biosynthesis gene clusters in bacterial and fungal genomes. It integrates and cross-links with a large number of in silico secondary metabolite analysis tools that have been published earlier.</td>
      <td>Up-to-date</td>
      <td>https://antismash.secondarymetabolites.org</td>
      <td>Sequence Analysis</td>
      <td>antismash</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/antismash</td>
    </tr>
    <tr>
      <td></td>
      <td>atactk_trim_adapters</td>
      <td>0.1.6</td>
      <td>0.1.9</td>
      <td>atactk</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Trim adapters from paired-end HTS reads.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ParkerLab/atactk/</td>
      <td>Fastq Manipulation</td>
      <td>atactk_trim_adapters</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/blob/master/tools/trim_adapters</td>
    </tr>
    <tr>
      <td></td>
      <td>augustus</td>
      <td>3.1.0</td>
      <td>3.5.0</td>
      <td>augustus</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences.</td>
      <td></td>
      <td>To update</td>
      <td>http://bioinf.uni-greifswald.de/augustus/</td>
      <td>Sequence Analysis</td>
      <td>augustus</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/augustus</td>
    </tr>
    <tr>
      <td></td>
      <td>bamhash</td>
      <td>1.1</td>
      <td>1.1</td>
      <td>bamhash</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Hash BAM and FASTQ files to verify data integrity</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/DecodeGenetics/BamHash</td>
      <td>Sequence Analysis</td>
      <td>bamhash</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/bamhash</td>
    </tr>
    <tr>
      <td></td>
      <td>bigwig_to_bedgraph</td>
      <td>0.1.0</td>
      <td></td>
      <td>ucsc_tools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert from bigWig to bedGraph format</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Convert Formats</td>
      <td>bigwig_to_bedgraph</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/bigwig_to_bedgraph</td>
    </tr>
    <tr>
      <td></td>
      <td>blobtoolkit</td>
      <td>4.0.7</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Identification and isolation non-target data in draft and publicly available genome assemblies.</td>
      <td></td>
      <td>To update</td>
      <td>https://blobtoolkit.genomehubs.org/</td>
      <td>Sequence Analysis, Assembly</td>
      <td>blobtoolkit</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/blobtoolkit</td>
    </tr>
    <tr>
      <td></td>
      <td>blockbuster</td>
      <td>0.1.2</td>
      <td>0.0.1.1</td>
      <td>blockbuster</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Blockbuster detects blocks of overlapping reads using a gaussian-distribution approach.</td>
      <td></td>
      <td>To update</td>
      <td>http://hoffmann.bioinf.uni-leipzig.de/LIFE/blockbuster.html</td>
      <td>RNA, Sequence Analysis</td>
      <td>blockbuster</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/blockbuster</td>
    </tr>
    <tr>
      <td></td>
      <td>blockclust</td>
      <td>1.1.1</td>
      <td>1.1.1</td>
      <td>blockclust</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>BlockClust detects transcripts with similar processing patterns.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/workflows/blockclust</td>
      <td>RNA</td>
      <td>blockclust</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/blockclust</td>
    </tr>
    <tr>
      <td></td>
      <td>canu</td>
      <td>2.2</td>
      <td>2.2</td>
      <td>canu</td>
      <td>Up-to-date</td>
      <td>canu</td>
      <td>CANU</td>
      <td>De-novo assembly</td>
      <td>Genomics</td>
      <td>Canu is a hierarchical assembly pipeline designed for high-noise single-molecule sequencing (such as the PacBio RS II/Sequel or Oxford Nanopore MinION).</td>
      <td>De-novo assembly tool for long read chemistry like Nanopore data and PacBio data.</td>
      <td>Up-to-date</td>
      <td>https://github.com/marbl/canu</td>
      <td></td>
      <td>canu</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/canu</td>
    </tr>
    <tr>
      <td></td>
      <td>categorize_elements_satisfying_criteria</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Categorize Elements satisfying criteria.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>categorize_elements_satisfying_criteria</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/categorize_elements_satisfying_criteria</td>
    </tr>
    <tr>
      <td></td>
      <td>chipseeker</td>
      <td>1.32.0</td>
      <td>1.38.0</td>
      <td>bioconductor-chipseeker</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A tool for ChIP peak annotation and visualization</td>
      <td></td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/ChIPseeker.html</td>
      <td>ChIP-seq, Genome annotation</td>
      <td>chipseeker</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/chipseeker</td>
    </tr>
    <tr>
      <td></td>
      <td>circexplorer</td>
      <td>1.1.9.0</td>
      <td>1.1.10</td>
      <td>circexplorer</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A combined strategy to identify circular RNAs (circRNAs and ciRNAs)</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/YangLab/CIRCexplorer</td>
      <td>Sequence Analysis, RNA</td>
      <td>circexplorer</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/circexplorer</td>
    </tr>
    <tr>
      <td></td>
      <td>cmsearch_deoverlap</td>
      <td>0.08+galaxy0</td>
      <td></td>
      <td>perl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>removes lower scoring overlaps from cmsearch results.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/EBI-Metagenomics/pipeline-v5/blob/master/tools/RNA_prediction/cmsearch-deoverlap/cmsearch-deoverlap.pl</td>
      <td>RNA</td>
      <td>cmsearch_deoverlap</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/cmsearch_deoverlap</td>
    </tr>
    <tr>
      <td></td>
      <td>cmv</td>
      <td>1.0.8</td>
      <td>1.0.8</td>
      <td>cmv</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>cmv is a collection of tools for the visualisation of Hidden Markov Models and RNA-family models.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/eggzilla/cmv</td>
      <td>RNA</td>
      <td>cmv</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/cmv</td>
    </tr>
    <tr>
      <td></td>
      <td>cofold</td>
      <td>2.0.4.0</td>
      <td>2.0.4</td>
      <td>cofold</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Cofold predicts RNA secondary structures that takes co-transcriptional folding into account.</td>
      <td></td>
      <td>To update</td>
      <td>http://www.e-rna.org/cofold/</td>
      <td>RNA</td>
      <td>cofold</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/cofold</td>
    </tr>
    <tr>
      <td></td>
      <td>combine_metaphlan_humann</td>
      <td>0.3.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td>combine_metaphlan_and_humann</td>
      <td>Combine Metaphlan and HUMAnN</td>
      <td>Aggregation</td>
      <td>Metagenomics, Molecular interactions, pathways and networks</td>
      <td>Combine MetaPhlAn2 and HUMAnN2 outputs to relate genus/species abundances and gene families/pathways abundances</td>
      <td>"This tool combine MetaPhlAn outputs and HUMANnN outputs." - Galaxy tool wrapper</td>
      <td>To update</td>
      <td></td>
      <td>Metagenomics</td>
      <td>combine_metaphlan2_humann2</td>
      <td>bebatut</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/combine_metaphlan2_humann2</td>
    </tr>
    <tr>
      <td></td>
      <td>compare_humann2_output</td>
      <td>0.2.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>compare_humann2_outputs</td>
      <td>Compare HUMAnN2 outputs</td>
      <td>Comparison</td>
      <td>Metagenomics, Gene and protein families</td>
      <td>Compare outputs of HUMAnN2 for several samples and extract similar and specific information</td>
      <td>"This tool compare HUMANnN2 outputs with gene families or pathways and their relative abundances between several samples." - Galaxy tool wrapper</td>
      <td>To update</td>
      <td></td>
      <td>Metagenomics</td>
      <td>compare_humann2_output</td>
      <td>bebatut</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/compare_humann2_output</td>
    </tr>
    <tr>
      <td></td>
      <td>compute_motif_frequencies_for_all_motifs</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute Motif Frequencies For All Motifs, motif by motif.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Statistics</td>
      <td>compute_motif_frequencies_for_all_motifs</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/compute_motif_frequencies_for_all_motifs</td>
    </tr>
    <tr>
      <td></td>
      <td>count_gff_features</td>
      <td>0.2</td>
      <td>1.1.0</td>
      <td>galaxy-ops</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Count GFF Features</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>count_gff_features</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/count_gff_features</td>
    </tr>
    <tr>
      <td></td>
      <td>cpat</td>
      <td>3.0.4</td>
      <td>3.0.4</td>
      <td>cpat</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Coding-potential assessment tool using an alignment-free logistic regression model.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/liguowang/cpat</td>
      <td>Transcriptomics</td>
      <td>cpat</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/cpat</td>
    </tr>
    <tr>
      <td></td>
      <td>delete_overlapping_indels</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Delete Overlapping Indels from a chromosome indels file</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>delete_overlapping_indels</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/delete_overlapping_indels</td>
    </tr>
    <tr>
      <td></td>
      <td>dewseq</td>
      <td>0.1.0+galaxy0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>DEWSeq is a sliding window based peak caller for eCLIP/iCLIP data</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/EMBL-Hentze-group/DEWSeq_analysis_helpers</td>
      <td>Sequence Analysis, RNA, CLIP-seq</td>
      <td>dewseq</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/dewseq</td>
    </tr>
    <tr>
      <td></td>
      <td>dgidb_annotator</td>
      <td>0.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Annotates a tabular file with information from the Drug-Gene Interaction Database (http://dgidb.genome.wustl.edu/)</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Systems Biology, Variant Analysis</td>
      <td>dgidb_annotator</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/dgidb_annotator</td>
    </tr>
    <tr>
      <td></td>
      <td>diff</td>
      <td>3.7</td>
      <td></td>
      <td>diffutils</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>GNU diff tool that calculates the differences between two files.</td>
      <td></td>
      <td>To update</td>
      <td>http://www.gnu.org/software/diffutils/</td>
      <td>Text Manipulation</td>
      <td>diff</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/diff</td>
    </tr>
    <tr>
      <td></td>
      <td>diffbind</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Diffbind provides functions for  processing  ChIP-Seq  data.</td>
      <td></td>
      <td>To update</td>
      <td>http://bioconductor.org/packages/release/bioc/html/DiffBind.html</td>
      <td>ChIP-seq</td>
      <td>diffbind</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/diffbind</td>
    </tr>
    <tr>
      <td></td>
      <td>dotknot</td>
      <td>1.3.1</td>
      <td></td>
      <td>vienna_rna</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>DotKnot is a heuristic method for pseudoknot prediction in a given RNA sequence</td>
      <td></td>
      <td>To update</td>
      <td>http://dotknot.csse.uwa.edu.au/</td>
      <td>RNA, Proteomics</td>
      <td>dotknot</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna/dotknot</td>
    </tr>
    <tr>
      <td></td>
      <td>draw_stacked_barplots</td>
      <td>1.0.0</td>
      <td></td>
      <td>R</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Draw Stacked Bar Plots for different categories and different criteria</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Graphics, Statistics</td>
      <td>draw_stacked_barplots</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/draw_stacked_barplots</td>
    </tr>
    <tr>
      <td></td>
      <td>edta</td>
      <td></td>
      <td>2.1.0</td>
      <td>edta</td>
      <td>To update</td>
      <td>edta</td>
      <td>The Extensive de novo TE Annotator (EDTA)</td>
      <td>De-novo assembly, Deisotoping, Genome annotation</td>
      <td>Workflows, Mobile genetic elements, Plant biology, Transcription factors and regulatory sites, Model organisms</td>
      <td>The EDTA package was designed to filter out false discoveries in raw TE candidates and generate a high-quality non-redundant TE library for whole-genome TE annotations.</td>
      <td>The EDTA package was designed to filter out false discoveries in raw TE candidates and generate a high-quality non-redundant TE library for whole-genome TE annotations. Selection of initial search programs were based on benckmarkings on the annotation performance using a manually curated TE library in the rice genome.</td>
      <td>To update</td>
      <td>https://github.com/oushujun/EDTA</td>
      <td>Variant Analysis</td>
      <td>edta</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/edta</td>
    </tr>
    <tr>
      <td></td>
      <td>express</td>
      <td>1.1.1</td>
      <td>1.5.1</td>
      <td>eXpress</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Quantify the abundances of a set of target sequences from sampled subsequences</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>express</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/express</td>
    </tr>
    <tr>
      <td></td>
      <td>fasta_compute_length</td>
      <td>1.0.3</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute sequence length</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Fasta Manipulation</td>
      <td>fasta_compute_length</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/fasta_compute_length</td>
    </tr>
    <tr>
      <td></td>
      <td>fasta_filter_by_length</td>
      <td>1.2</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter sequences by length</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Fasta Manipulation</td>
      <td>fasta_filter_by_length</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/fasta_filter_by_length</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_info</td>
      <td>0.25.1</td>
      <td>0.25.2</td>
      <td>fastq_utils</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FASTQ info allows to validate single or paired fastq files</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/nunofonseca/fastq_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_info</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/fastq_info</td>
    </tr>
    <tr>
      <td></td>
      <td>find_diag_hits</td>
      <td>1.0.0</td>
      <td>0.9.0</td>
      <td>taxonomy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Find diagnostic hits</td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>Metagenomics</td>
      <td>find_diag_hits</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/taxonomy/find_diag_hits</td>
    </tr>
    <tr>
      <td></td>
      <td>flye</td>
      <td>2.9.1</td>
      <td>2.9.3</td>
      <td>flye</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Assembly of long and error-prone reads.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/fenderglass/Flye/</td>
      <td>Assembly</td>
      <td>flye</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/flye</td>
    </tr>
    <tr>
      <td></td>
      <td>format_cd_hit_output</td>
      <td>1.0.0+galaxy1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Format CD-hit output to rename representative sequences with cluster name and/or extract distribution inside clusters given a mapping file</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Fasta Manipulation</td>
      <td>format_cd_hit_output</td>
      <td>bebatut</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/format_cd_hit_output/</td>
    </tr>
    <tr>
      <td></td>
      <td>format_metaphlan2_output</td>
      <td>0.2.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>format_metaphlan2_output</td>
      <td>Format metaphlan2 output</td>
      <td>Formatting</td>
      <td>Taxonomy, Metagenomics</td>
      <td>Format MetaPhlAn2 output to extract abundance at different taxonomic levels</td>
      <td>"This tool format output file of MetaPhlan2 containing community content (abundance) at all taxonomic levels (from kingdom to strains)." - Galaxy tool wrapper</td>
      <td>To update</td>
      <td></td>
      <td>Metagenomics</td>
      <td>format_metaphlan2_output</td>
      <td>bebatut</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/format_metaphlan2_output/</td>
    </tr>
    <tr>
      <td></td>
      <td>get_orfs_or_cdss</td>
      <td>0.2.3</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Search nucleotide sequences for open reading frames (ORFs), or coding sequences (CDSs)</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/get_orfs_or_cdss</td>
      <td>Sequence Analysis</td>
      <td>get_orfs_or_cdss</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/get_orfs_or_cdss</td>
    </tr>
    <tr>
      <td></td>
      <td>gfastats</td>
      <td>1.3.6</td>
      <td>1.3.6</td>
      <td>gfastats</td>
      <td>Up-to-date</td>
      <td>gfastats</td>
      <td>gfastats</td>
      <td>Data handling</td>
      <td>Computational biology</td>
      <td>Tool for generating sequence statistics and simultaneous genome assembly file manipulation.</td>
      <td>gfastats is a single fast and exhaustive tool for summary statistics and simultaneous genome assembly file manipulation. gfastats also allows seamless fasta/fastq/gfa conversion.</td>
      <td>Up-to-date</td>
      <td>https://github.com/vgl-hub/gfastats</td>
      <td>Sequence Analysis</td>
      <td>gfastats</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/gfastats</td>
    </tr>
    <tr>
      <td></td>
      <td>hifiasm</td>
      <td>0.19.8</td>
      <td>0.19.8</td>
      <td>hifiasm</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A fast haplotype-resolved de novo assembler</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/chhylp123/hifiasm</td>
      <td>Assembly</td>
      <td>hifiasm</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/hifiasm</td>
    </tr>
    <tr>
      <td></td>
      <td>instagraal</td>
      <td>0.1.6</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>instagraal</td>
      <td>instaGRAAL</td>
      <td>Genome assembly, Mapping assembly, Genetic mapping, Scaffolding</td>
      <td>Sequence assembly, Mapping, Metagenomics, Statistics and probability, DNA binding sites</td>
      <td>Large genome reassembly based on Hi-C data</td>
      <td>Chromosome-level quality scaffolding of brown algal genomes using InstaGRAAL.Large genome reassembly based on Hi-C data, continuation of GRAAL.Large genome reassembly based on Hi-C data (continuation and partial rewrite of GRAAL) and post-scaffolding polishing libraries.This work is under continuous development/improvement - see GRAAL for information about the basic principles.sudo pip3 install -e git+https://github.com/koszullab/instagraal.git@master#egg=instagraal.Note to OS X users: There is currently no CUDA support on Mojave (10.14) and it is unclear when it is going to be added, if it is to be added at all. This means instaGRAAL (or indeed any CUDA-based application) will not work on Mojave. If you wish to run it on OS X, the only solution for now is to downgrade to High Sierra (10.13)</td>
      <td>To update</td>
      <td>https://github.com/koszullab/instaGRAAL</td>
      <td>Assembly</td>
      <td>instagraal</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/instagraal</td>
    </tr>
    <tr>
      <td></td>
      <td>intarna</td>
      <td>3.3.2</td>
      <td>3.3.2</td>
      <td>intarna</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Efficient RNA-RNA interaction prediction incorporating accessibility and seeding of interaction sites.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/BackofenLab/IntaRNA</td>
      <td>RNA</td>
      <td>intarna</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/intarna</td>
    </tr>
    <tr>
      <td></td>
      <td>itsx</td>
      <td>1.1.3</td>
      <td>1.1.3</td>
      <td>itsx</td>
      <td>Up-to-date</td>
      <td>ITSx</td>
      <td>ITSx</td>
      <td>Sequence feature detection</td>
      <td>Functional, regulatory and non-coding RNA, Microbiology</td>
      <td>ITSx is an open source software utility to extract the highly variable ITS1 and ITS2 subregions from ITS sequences.</td>
      <td>TSx is an open source software utility to extract the highly variable ITS1 and ITS2 subregions from ITS sequences, which is commonly used as a molecular barcode for e.g. fungi. As the inclusion of parts of the neighbouring, very conserved, ribosomal genes (SSU, 5S and LSU rRNA sequences) in the sequence identification process can lead to severely misleading results, ITSx identifies and extracts only the ITS regions themselves.</td>
      <td>Up-to-date</td>
      <td>https://microbiology.se/software/itsx/</td>
      <td>Metagenomics</td>
      <td>itsx</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/itsx</td>
    </tr>
    <tr>
      <td></td>
      <td>join_files_on_column_fuzzy</td>
      <td>1.0.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Join two files on a common column, allowing a certain difference.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>join_files_on_column_fuzzy</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/text_processing/join_files_on_column_fuzzy</td>
    </tr>
    <tr>
      <td></td>
      <td>lighter</td>
      <td>1.0</td>
      <td>1.1.2</td>
      <td>lighter</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Lighter is a kmer-based error correction method for whole genome sequencing data</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Fasta Manipulation</td>
      <td>lighter</td>
      <td>bgruening</td>
      <td>https://github.com/mourisl/Lighter</td>
    </tr>
    <tr>
      <td></td>
      <td>mcl</td>
      <td>14.137</td>
      <td>22.282</td>
      <td>mcl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Markov Cluster Algorithm</td>
      <td></td>
      <td>To update</td>
      <td>http://micans.org/mcl/</td>
      <td>Sequence Analysis, Metagenomics</td>
      <td>mcl</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/mcl</td>
    </tr>
    <tr>
      <td></td>
      <td>mea</td>
      <td>0.6.4.1</td>
      <td>0.6.4</td>
      <td>mea</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Maximum expected accuracy prediction</td>
      <td></td>
      <td>To update</td>
      <td>http://www.bioinf.uni-leipzig.de/Software/mea</td>
      <td>RNA</td>
      <td>mea</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/mea</td>
    </tr>
    <tr>
      <td></td>
      <td>megablast_xml_parser</td>
      <td>1.0.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Parse blast XML output</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Next Gen Mappers, Convert Formats</td>
      <td>megablast_xml_parser</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/megablast_xml_parser</td>
    </tr>
    <tr>
      <td></td>
      <td>metilene</td>
      <td>0.2.6.1</td>
      <td>0.2.8</td>
      <td>metilene</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Differential DNA methylation calling</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA, Statistics</td>
      <td>metilene</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/metilene</td>
    </tr>
    <tr>
      <td></td>
      <td>minced</td>
      <td>0.2.0</td>
      <td>0.4.2</td>
      <td>minced</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MinCED - Mining CRISPRs in Environmental Datasets</td>
      <td></td>
      <td>To update</td>
      <td>http://bioweb2.pasteur.fr/docs/modules/minced/0.1.5/_README</td>
      <td>Sequence Analysis</td>
      <td>minced</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/minced</td>
    </tr>
    <tr>
      <td></td>
      <td>minipolish</td>
      <td>0.1.3</td>
      <td>0.1.3</td>
      <td>minipolish</td>
      <td>Up-to-date</td>
      <td>minipolish</td>
      <td>minipolish</td>
      <td>Localised reassembly, Read depth analysis</td>
      <td>Sequence assembly, Sequencing</td>
      <td>Polishing miniasm assemblies</td>
      <td>A tool that bridges the output of miniasm (long-read assembly) and racon (assembly polishing) together to polish a draft assembly. It also provides read depth information in contigs.</td>
      <td>Up-to-date</td>
      <td>https://github.com/rrwick/Minipolish</td>
      <td>Sequence Analysis</td>
      <td>minipolish</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/minipolish</td>
    </tr>
    <tr>
      <td></td>
      <td>mitohifi</td>
      <td>3</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Assembly mitogenomes from Pacbio HiFi read.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/marcelauliano/MitoHiFi/tree/mitohifi_v2</td>
      <td>Assembly</td>
      <td>mitohifi</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/mitohifi</td>
    </tr>
    <tr>
      <td></td>
      <td>mqc</td>
      <td>1.9</td>
      <td>1.10</td>
      <td>mqc</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Ribosome profiling mapping quality control tool</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/Biobix/mQC</td>
      <td>Sequence Analysis</td>
      <td>mqc</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/mqc/</td>
    </tr>
    <tr>
      <td></td>
      <td>music_deconvolution</td>
      <td>0.1.1</td>
      <td>0.1.1</td>
      <td>music-deconvolution</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Multi-subject Single Cell deconvolution (MuSiC)</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/xuranw/MuSiC</td>
      <td>Transcriptomics</td>
      <td>music</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/music/</td>
    </tr>
    <tr>
      <td></td>
      <td>netboxr</td>
      <td>1.6.0</td>
      <td>1.9.0</td>
      <td>bioconductor-netboxr</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>netboxr enables automated discovery of biological process modules by network analysis</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Systems Biology</td>
      <td>netboxr</td>
      <td>bgruening</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>nextdenovo</td>
      <td>2.5.0</td>
      <td>2.5.2</td>
      <td>nextdenovo</td>
      <td>To update</td>
      <td>nextdenovo</td>
      <td>NextDenovo</td>
      <td>De-novo assembly, Genome assembly</td>
      <td>Sequencing, Sequence assembly</td>
      <td>String graph-based de novo assembler for long reads</td>
      <td>NextDenovo is a string graph-based de novo assembler for long reads (CLR, HiFi and ONT). It uses a "correct-then-assemble" strategy similar to canu (no correction step for PacBio Hifi reads), but requires significantly less computing resources and storages.</td>
      <td>To update</td>
      <td>https://github.com/Nextomics/NextDenovo</td>
      <td>Assembly</td>
      <td>nextdenovo</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/nextdenovo</td>
    </tr>
    <tr>
      <td></td>
      <td>pandas_rolling_window</td>
      <td>0.1</td>
      <td></td>
      <td>numpy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Rolling window calculations</td>
      <td></td>
      <td>To update</td>
      <td>https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.rolling.html</td>
      <td>Statistics</td>
      <td>pandas_rolling_window</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/pandas_rolling_window</td>
    </tr>
    <tr>
      <td></td>
      <td>paralyzer</td>
      <td>1.5</td>
      <td>1.5</td>
      <td>paralyzer</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A method to generate a high resolution map of interaction sites between RNA-binding proteins and their targets.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://ohlerlab.mdc-berlin.de/software/PARalyzer_85/</td>
      <td>RNA</td>
      <td>paralyzer</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/paralyzer</td>
    </tr>
    <tr>
      <td></td>
      <td>peakachu</td>
      <td>0.2.0+galaxy1</td>
      <td>0.2.0</td>
      <td>peakachu</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>PEAKachu is a peak-caller for CLIP- and RIP-Seq data</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, RNA</td>
      <td>peakachu</td>
      <td>rnateam</td>
      <td>https://github.com/tbischler/PEAKachu</td>
    </tr>
    <tr>
      <td></td>
      <td>pfamscan</td>
      <td>1.6</td>
      <td>1.6</td>
      <td>pfam_scan</td>
      <td>Up-to-date</td>
      <td>pfamscan</td>
      <td>PfamScan</td>
      <td>Protein sequence analysis</td>
      <td>Sequence analysis</td>
      <td>Search a FASTA sequence against a library of Pfam HMM.</td>
      <td>This tool is used to search a FASTA sequence against a library of Pfam HMM.</td>
      <td>Up-to-date</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/Pfam/Tools/</td>
      <td>Sequence Analysis</td>
      <td>pfamscan</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/pfamscan</td>
    </tr>
    <tr>
      <td></td>
      <td>pharmcat</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Pharmacogenomics Clinical Annotation Tool</td>
      <td></td>
      <td>To update</td>
      <td>https://pharmcat.org/</td>
      <td>Variant Analysis</td>
      <td>pharmcat</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/pharmcat</td>
    </tr>
    <tr>
      <td></td>
      <td>pileup_interval</td>
      <td>1.0.3</td>
      <td>0.10.0</td>
      <td>bx-python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Pileup-to-Interval condenses pileup format into ranges of bases</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>SAM</td>
      <td>pileup_interval</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/pileup_interval</td>
    </tr>
    <tr>
      <td></td>
      <td>pileup_parser</td>
      <td>1.0.2</td>
      <td></td>
      <td>perl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter pileup on coverage and SNPs</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-devteam/</td>
      <td>SAM</td>
      <td>pileup_parser</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/pileup_parser</td>
    </tr>
    <tr>
      <td></td>
      <td>pipmir</td>
      <td>0.1.0</td>
      <td>1.1</td>
      <td>pipmir</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A method to identify novel plant miRNA.</td>
      <td></td>
      <td>To update</td>
      <td>https://ohlerlab.mdc-berlin.de/software/Pipeline_for_the_Identification_of_Plant_miRNAs_84/</td>
      <td>RNA</td>
      <td>pipmir</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/pipmir</td>
    </tr>
    <tr>
      <td></td>
      <td>piranha</td>
      <td>1.2.1.0</td>
      <td>1.2.1</td>
      <td>piranha</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Piranha is a peak-caller for CLIP- and RIP-Seq data</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, RNA</td>
      <td>piranha</td>
      <td>rnateam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/piranha</td>
    </tr>
    <tr>
      <td></td>
      <td>plotly_ml_performance_plots</td>
      <td>0.2</td>
      <td></td>
      <td>pandas</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>performance plots for machine learning problems</td>
      <td></td>
      <td>To update</td>
      <td>http://scikit-learn.org/stable/modules/classes.html#module-sklearn.metrics</td>
      <td>Visualization</td>
      <td>plotly_ml_performance_plots</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/plotly_ml_performance_plots</td>
    </tr>
    <tr>
      <td></td>
      <td>plotly_parallel_coordinates_plot</td>
      <td>0.2</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>parallel coordinates plot produced with plotly</td>
      <td></td>
      <td>To update</td>
      <td>https://plot.ly/python/parallel-coordinates-plot/</td>
      <td>Visualization</td>
      <td>plotly_parallel_coordinates_plot</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/plotly_parallel_coordinates_plot</td>
    </tr>
    <tr>
      <td></td>
      <td>plotly_regression_performance_plots</td>
      <td>0.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>performance plots for regression problems</td>
      <td></td>
      <td>To update</td>
      <td>http://scikit-learn.org/stable/supervised_learning.html#supervised-learning</td>
      <td>Visualization</td>
      <td>plotly_regression_performance_plots</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/plotly_regression_performance_plots</td>
    </tr>
    <tr>
      <td></td>
      <td>poisson2test</td>
      <td>1.0.0</td>
      <td>0.9.0</td>
      <td>taxonomy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Poisson two-sample test</td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>Statistics, Metagenomics</td>
      <td>poisson2test</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/taxonomy/poisson2test</td>
    </tr>
    <tr>
      <td></td>
      <td>racon</td>
      <td>1.5.0</td>
      <td>1.5.0</td>
      <td>racon</td>
      <td>Up-to-date</td>
      <td>Racon</td>
      <td>Racon</td>
      <td>Genome assembly, Mapping assembly, Sequence trimming</td>
      <td>Whole genome sequencing, Sequence assembly, Plant biology</td>
      <td>Consensus module for raw de novo DNA assembly of long uncorrected reads.</td>
      <td>The Possibility to Use Oxford Nanopore Technology | Ultrafast consensus module for raw de novo genome assembly of long uncorrected reads. http://genome.cshlp.org/content/early/2017/01/18/gr.214270.116 Note: This was the original repository which will no longer be officially maintained. Please use the new official repository here: https://github.com/isovic/racon| Racon is intended as a standalone consensus module to correct raw contigs generated by rapid assembly methods which do not include a consensus step | Consensus module for raw de novo DNA assembly of long uncorrected reads</td>
      <td>Up-to-date</td>
      <td>https://github.com/isovic/racon</td>
      <td>Sequence Analysis</td>
      <td>racon</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/racon</td>
    </tr>
    <tr>
      <td></td>
      <td>rcas</td>
      <td>1.5.4</td>
      <td>1.28.2</td>
      <td>bioconductor-rcas</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>RCAS (RNA Centric Annotation System) for functional analysis of transcriptome-wide regions detected by high-throughput experiments</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/BIMSBbioinfo/RCAS</td>
      <td>RNA</td>
      <td>rcas</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/rcas/</td>
    </tr>
    <tr>
      <td></td>
      <td>remurna</td>
      <td>1.0.0</td>
      <td>1.0</td>
      <td>remurna</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>remuRNA - Measurement of Single Nucleotide Polymorphism induced Changes of RNA Conformation</td>
      <td></td>
      <td>To update</td>
      <td>https://www.ncbi.nlm.nih.gov/CBBresearch/Przytycka/index.cgi#remurna</td>
      <td>RNA</td>
      <td>remurna</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/remurna</td>
    </tr>
    <tr>
      <td></td>
      <td>rnasnp</td>
      <td>1.2.0</td>
      <td>1.2</td>
      <td>rnasnp</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>RNAsnp Efficient detection of local RNA secondary structure changes induced by SNPs</td>
      <td></td>
      <td>To update</td>
      <td>http://rth.dk/resources/rnasnp/</td>
      <td>RNA</td>
      <td>rnasnp</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rnasnp</td>
    </tr>
    <tr>
      <td></td>
      <td>rnaz</td>
      <td>2.1.1</td>
      <td>2.1.1</td>
      <td>rnaz</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>RNAz is a program for predicting structurally conserved and thermodynamically stable RNA secondary structures in multiple sequence alignments.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://www.tbi.univie.ac.at/~wash/RNAz/</td>
      <td>RNA</td>
      <td>rnaz</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_team/rnaz</td>
    </tr>
    <tr>
      <td></td>
      <td>sailfish</td>
      <td>0.10.1.1</td>
      <td></td>
      <td>bzip2</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Sailfish is a tool for transcript quantification from RNA-seq data</td>
      <td></td>
      <td>To update</td>
      <td>http://www.cs.cmu.edu/~ckingsf/software/sailfish/</td>
      <td>Sequence Analysis, RNA</td>
      <td>sailfish</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/sailfish</td>
    </tr>
    <tr>
      <td></td>
      <td>salmon</td>
      <td>1.10.1</td>
      <td>1.10.2</td>
      <td>salmon</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Salmon is a wicked-fast program to produce a highly-accurate, transcript-level quantification estimates from RNA-seq and single-cell data.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/COMBINE-lab/salmon</td>
      <td>Sequence Analysis, RNA, Transcriptomics</td>
      <td></td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/salmon</td>
    </tr>
    <tr>
      <td></td>
      <td>sam2interval</td>
      <td>1.0.2</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert SAM to interval.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>SAM</td>
      <td>sam2interval</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/sam2interval</td>
    </tr>
    <tr>
      <td></td>
      <td>sample_seqs</td>
      <td>0.2.6</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Sub-sample sequences files (e.g. to reduce coverage)</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/sample_seqs</td>
      <td>Assembly, Fasta Manipulation, Fastq Manipulation, Sequence Analysis</td>
      <td>sample_seqs</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/sample_seqs</td>
    </tr>
    <tr>
      <td></td>
      <td>samtools_depth</td>
      <td>0.0.3</td>
      <td>1.19</td>
      <td>samtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Coverage depth via samtools</td>
      <td></td>
      <td>To update</td>
      <td>http://www.htslib.org/</td>
      <td>Assembly, Sequence Analysis, SAM</td>
      <td>samtools_depth</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/samtools_depth</td>
    </tr>
    <tr>
      <td></td>
      <td>samtools_idxstats</td>
      <td>0.0.6</td>
      <td>1.19</td>
      <td>samtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>BAM mapping statistics (using samtools idxstats)</td>
      <td></td>
      <td>To update</td>
      <td>http://www.htslib.org/</td>
      <td>Assembly, Next Gen Mappers, SAM</td>
      <td>samtools_idxstats</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/samtools_idxstats</td>
    </tr>
    <tr>
      <td></td>
      <td>segemehl</td>
      <td>0.2.0.4</td>
      <td>0.3.4</td>
      <td>segemehl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>segemehl - short read mapping with gaps</td>
      <td></td>
      <td>To update</td>
      <td>http://www.bioinf.uni-leipzig.de/Software/segemehl/</td>
      <td>Next Gen Mappers</td>
      <td>segemehl</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/segemehl</td>
    </tr>
    <tr>
      <td></td>
      <td>selectsequencesfrommsa</td>
      <td>1.0.5</td>
      <td>1.0.5</td>
      <td>selectsequencesfrommsa</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SelectSequences - selects representative entries from a multiple sequence alignment in clustal format</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/eggzilla/SelectSequences</td>
      <td>RNA, Sequence Analysis</td>
      <td>selectsequencesfrommsa</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/selectsequencesfrommsa</td>
    </tr>
    <tr>
      <td></td>
      <td>seq_composition</td>
      <td>0.0.5</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Sequence composition</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_composition</td>
      <td>Sequence Analysis</td>
      <td>seq_composition</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_composition</td>
    </tr>
    <tr>
      <td></td>
      <td>seq_filter_by_id</td>
      <td>0.2.9</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter sequences by ID</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_filter_by_id</td>
      <td>Fasta Manipulation, Sequence Analysis, Text Manipulation</td>
      <td>seq_filter_by_id</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_filter_by_id</td>
    </tr>
    <tr>
      <td></td>
      <td>seq_filter_by_mapping</td>
      <td>0.0.8</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter sequencing reads using SAM/BAM mapping files</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_filter_by_mapping</td>
      <td>Assembly, Fasta Manipulation, Fastq Manipulation, SAM, Sequence Analysis</td>
      <td>seq_filter_by_mapping</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_filter_by_mapping</td>
    </tr>
    <tr>
      <td></td>
      <td>split_file_to_collection</td>
      <td>0.5.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Split tabular, MGF, FASTA, or FASTQ files to a dataset collection.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>split_file_to_collection</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/text_processing/split_file_to_collection</td>
    </tr>
    <tr>
      <td></td>
      <td>sshmm</td>
      <td>1.0.7</td>
      <td>1.0.7</td>
      <td>sshmm</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ssHMM is an RNA sequence-structure motif finder for RNA-binding protein data, such as CLIP-Seq data</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.molgen.mpg.de/heller/ssHMM</td>
      <td>Sequence Analysis, RNA</td>
      <td>sshmm</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/sshmm/</td>
    </tr>
    <tr>
      <td></td>
      <td>t2t_report</td>
      <td>1.0.0</td>
      <td>0.9.0</td>
      <td>taxonomy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Summarize taxonomy</td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>Metagenomics</td>
      <td>t2t_report</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/taxonomy/t2t_report</td>
    </tr>
    <tr>
      <td></td>
      <td>t_test_two_samples</td>
      <td>1.0.1</td>
      <td></td>
      <td>R</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>T Test for Two Samples</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Statistics</td>
      <td>t_test_two_samples</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/t_test_two_samples</td>
    </tr>
    <tr>
      <td></td>
      <td>tables_arithmetic_operations</td>
      <td>1.0.0</td>
      <td></td>
      <td>perl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Arithmetic Operations on tables</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/gops</td>
      <td>Genomic Interval Operations</td>
      <td>tables_arithmetic_operations</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/gops/tables_arithmetic_operations</td>
    </tr>
    <tr>
      <td></td>
      <td>targetfinder</td>
      <td>1.7</td>
      <td>1.7</td>
      <td>targetfinder</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Plant small RNA target prediction tool</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/carringtonlab/TargetFinder.git</td>
      <td>RNA</td>
      <td>targetfinder</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/targetfinder/</td>
    </tr>
    <tr>
      <td></td>
      <td>tgsgapcloser</td>
      <td>1.0.3</td>
      <td>1.2.1</td>
      <td>tgsgapcloser</td>
      <td>To update</td>
      <td>TGS-GapCloser</td>
      <td>TGS-GapCloser</td>
      <td>Genome assembly, Read mapping, Scaffolding, Localised reassembly</td>
      <td>Sequencing, Sequence assembly, Phylogeny, Transcription factors and regulatory sites, Mapping</td>
      <td>TGS-GapCloser uses error-prone long reads or preassembled contigs to fill N-gap in the genome assembly.</td>
      <td>TGS-GapCloser is a fast and accurately passing through the Bermuda in large genome using error-prone third-generation long reads.</td>
      <td>To update</td>
      <td>https://github.com/BGI-Qingdao/TGS-GapCloser</td>
      <td>Assembly</td>
      <td>tgsgapcloser</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/tgsgapcloser</td>
    </tr>
    <tr>
      <td></td>
      <td>tophat2</td>
      <td>2.1.1</td>
      <td>2.5.2</td>
      <td>bowtie2</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tophat - fast splice junction mapper for RNA-Seq reads</td>
      <td></td>
      <td>To update</td>
      <td>http://ccb.jhu.edu/software/tophat/index.shtml</td>
      <td>RNA, Next Gen Mappers</td>
      <td>tophat2</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/tophat2</td>
    </tr>
    <tr>
      <td></td>
      <td>tophat_fusion_post</td>
      <td>0.1</td>
      <td></td>
      <td>blast+</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Wrapper for Tophat-Fusion post step</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics</td>
      <td>tophat_fusion_post</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/tophat_fusion_post</td>
    </tr>
    <tr>
      <td></td>
      <td>trim_galore</td>
      <td>0.6.7</td>
      <td>0.6.10</td>
      <td>trim-galore</td>
      <td>To update</td>
      <td>trim_galore</td>
      <td>Trim Galore</td>
      <td>Sequence trimming</td>
      <td>Sequence analysis</td>
      <td>Trim Galore adaptive quality and adapter trimmer</td>
      <td>A wrapper tool around Cutadapt and FastQC to consistently apply quality and adapter trimming to FastQ files, with some extra functionality for MspI-digested RRBS-type (Reduced Representation Bisufite-Seq) libraries.</td>
      <td>To update</td>
      <td>http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/</td>
      <td>Sequence Analysis, Fastq Manipulation</td>
      <td>trim_galore</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/trim_galore</td>
    </tr>
    <tr>
      <td></td>
      <td>trimmer</td>
      <td>0.0.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Trim leading or trailing characters.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>trimmer</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/trimmer</td>
    </tr>
    <tr>
      <td></td>
      <td>vcf_filter</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter a VCF file</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Variant Analysis</td>
      <td>vcf_filter</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/vcf_filter</td>
    </tr>
    <tr>
      <td></td>
      <td>vcftools_annotate</td>
      <td>0.1</td>
      <td></td>
      <td>echo</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Annotate VCF using custom/user-defined annotations</td>
      <td></td>
      <td>To update</td>
      <td>https://vcftools.github.io/</td>
      <td>Variant Analysis</td>
      <td>vcftools_annotate</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/vcftools/vcftools_annotate</td>
    </tr>
    <tr>
      <td></td>
      <td>vcftools_compare</td>
      <td>0.1</td>
      <td>1.11</td>
      <td>tabix</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compare VCF files to get overlap and uniqueness statistics</td>
      <td></td>
      <td>To update</td>
      <td>https://vcftools.github.io/</td>
      <td>Variant Analysis</td>
      <td>vcftools_compare</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/vcftools/vcftools_compare</td>
    </tr>
    <tr>
      <td></td>
      <td>vcftools_isec</td>
      <td>0.1.1</td>
      <td>1.11</td>
      <td>tabix</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Intersect multiple VCF datasets</td>
      <td></td>
      <td>To update</td>
      <td>https://vcftools.github.io/</td>
      <td>Variant Analysis</td>
      <td>vcftools_isec</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/vcftools/vcftools_isec</td>
    </tr>
    <tr>
      <td></td>
      <td>vcftools_merge</td>
      <td>0.1.11</td>
      <td>1.11</td>
      <td>tabix</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Merge multiple VCF datasets into a single dataset</td>
      <td></td>
      <td>To update</td>
      <td>https://vcftools.github.io/</td>
      <td>Variant Analysis</td>
      <td>vcftools_merge</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/vcftools/vcftools_merge</td>
    </tr>
    <tr>
      <td></td>
      <td>vcftools_slice</td>
      <td>0.1</td>
      <td></td>
      <td>echo</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Subset VCF dataset by genomic regions</td>
      <td></td>
      <td>To update</td>
      <td>https://vcftools.github.io/</td>
      <td>Variant Analysis</td>
      <td>vcftools_slice</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/vcftools/vcftools_slice</td>
    </tr>
    <tr>
      <td></td>
      <td>vcftools_subset</td>
      <td>0.1</td>
      <td>1.11</td>
      <td>tabix</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Select samples from a VCF dataset</td>
      <td></td>
      <td>To update</td>
      <td>https://vcftools.github.io/</td>
      <td>Variant Analysis</td>
      <td>vcftools_subset</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/vcftools/vcftools_subset</td>
    </tr>
    <tr>
      <td></td>
      <td>venn_list</td>
      <td>0.1.2</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Draw Venn Diagram (PDF) from lists, FASTA files, etc</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/venn_list</td>
      <td>Graphics, Sequence Analysis, Visualization</td>
      <td>venn_list</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/venn_list</td>
    </tr>
    <tr>
      <td></td>
      <td>wtdbg</td>
      <td>2.5</td>
      <td>2.5</td>
      <td>wtdbg</td>
      <td>Up-to-date</td>
      <td>wtdbg2</td>
      <td>wtdbg2</td>
      <td>Genome assembly, De-novo assembly</td>
      <td>Sequence assembly, Sequencing</td>
      <td>WTDBG is a fuzzy Bruijn graph (FBG) approach to long noisy reads assembly.</td>
      <td>Wtdbg2 is a de novo sequence assembler for long noisy reads produced by PacBio or Oxford Nanopore Technologies (ONT). It assembles raw reads without error correction and then builds the consensus from intermediate assembly output. Wtdbg2 is able to assemble the human and even the 32Gb Axolotl genome at a speed tens of times faster than CANU and FALCON while producing contigs of comparable base accuracy.</td>
      <td>Up-to-date</td>
      <td>https://github.com/ruanjue/wtdbg2</td>
      <td>Assembly</td>
      <td>wtdbg</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/wtdbg</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy wrapper id</td>
      <td>Galaxy wrapper version</td>
      <td>Conda version</td>
      <td>Conda id</td>
      <td>Status</td>
      <td>bio.tool id</td>
      <td>bio.tool name</td>
      <td>EDAM operation</td>
      <td>EDAM topic</td>
      <td>Description</td>
      <td>bio.tool description</td>
      <td>Status</td>
      <td>Source</td>
      <td>ToolShed categories</td>
      <td>ToolShed id</td>
      <td>Galaxy wrapper owner</td>
      <td>Galaxy wrapper source</td>
    </tr>
    <tr>
      <td></td>
      <td>abricate</td>
      <td>1.0.1</td>
      <td>1.0.1</td>
      <td>abricate</td>
      <td>Up-to-date</td>
      <td>ABRicate</td>
      <td>ABRicate</td>
      <td>Antimicrobial resistance prediction</td>
      <td>Genomics, Microbiology</td>
      <td>Mass screening of contigs for antiobiotic resistance genes</td>
      <td>Mass screening of contigs for antimicrobial resistance or virulence genes.</td>
      <td>Up-to-date</td>
      <td>https://github.com/tseemann/abricate</td>
      <td>Sequence Analysis</td>
      <td>abricate</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/abricate/</td>
    </tr>
    <tr>
      <td></td>
      <td>adapter_removal</td>
      <td>2.3.3</td>
      <td>2.3.3</td>
      <td>adapterremoval</td>
      <td>Up-to-date</td>
      <td>adapterremoval</td>
      <td>AdapterRemoval</td>
      <td>Sequence trimming, Sequence merging, Primer removal</td>
      <td></td>
      <td>Removes residual adapter sequences from single-end (SE) or paired-end (PE) FASTQ reads.</td>
      <td>AdapterRemoval searches for and removes adapter sequences from High-Throughput Sequencing (HTS) data and (optionally) trims low quality bases from the 3' end of reads following adapter removal. AdapterRemoval can analyze both single end and paired end data, and can be used to merge overlapping paired-ended reads into (longer) consensus sequences. Additionally, AdapterRemoval can construct a consensus adapter sequence for paired-ended reads, if which this information is not available.</td>
      <td>Up-to-date</td>
      <td>https://github.com/MikkelSchubert/adapterremoval</td>
      <td>Fasta Manipulation, Sequence Analysis</td>
      <td>adapter_removal</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/adapter_removal/</td>
    </tr>
    <tr>
      <td></td>
      <td>aldex2</td>
      <td>1.26.0</td>
      <td>1.34.0</td>
      <td>bioconductor-aldex2</td>
      <td>To update</td>
      <td>aldex2</td>
      <td>ALDEx2</td>
      <td>Statistical inference</td>
      <td>Gene expression, Statistics and probability</td>
      <td>Performs analysis Of differential abundance taking sample variation into account</td>
      <td>A differential abundance analysis for the comparison of two or more conditions. It uses a Dirichlet-multinomial model to infer abundance from counts, that has been optimized for three or more experimental replicates. Infers sampling variation and calculates the expected FDR given the biological and sampling variation using the Wilcox rank test and Welches t-test, or the glm and Kruskal Wallis tests. Reports both P and fdr values calculated by the Benjamini Hochberg correction.</td>
      <td>To update</td>
      <td>https://github.com/ggloor/ALDEx_bioc</td>
      <td>Metagenomics</td>
      <td>aldex2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/aldex2</td>
    </tr>
    <tr>
      <td></td>
      <td>amplican</td>
      <td>1.14.0</td>
      <td>1.24.0</td>
      <td>bioconductor-amplican</td>
      <td>To update</td>
      <td>amplican</td>
      <td>amplican</td>
      <td>Alignment, Standardisation and normalisation</td>
      <td>PCR experiment, Statistics and probability</td>
      <td>AmpliCan is an analysis tool for genome editing.</td>
      <td>It performs alignment of the amplicon reads, normalizes gathered data, calculates multiple statistics (e.g. cut rates, frameshifts) and presents results in form of aggregated reports. Data and statistics can be broken down by experiments, barcodes, user defined groups, guides and amplicons allowing for quick identification of potential problems.</td>
      <td>To update</td>
      <td>https://github.com/valenlab/amplican</td>
      <td>Sequence Analysis</td>
      <td>amplican</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/amplican</td>
    </tr>
    <tr>
      <td></td>
      <td>ancombc</td>
      <td>1.4.0</td>
      <td>2.4.0</td>
      <td>bioconductor-ancombc</td>
      <td>To update</td>
      <td>ancombc</td>
      <td>ANCOMBC</td>
      <td>DNA barcoding</td>
      <td>Microbial ecology, Metagenomics</td>
      <td>Performs analysis of compositions of microbiomes with bias correction.</td>
      <td>Determine taxa whose absolute abundances, per unit volume, of the ecosystem (e.g. gut) are significantly different with changes in the covariate of interest (e.g. group). The current version of ancombc function implements Analysis of Compositions of Microbiomes with Bias Correction (ANCOM-BC) in cross-sectional data while allowing for covariate adjustment.</td>
      <td>To update</td>
      <td>https://github.com/FrederickHuangLin/ANCOMBC</td>
      <td>Metagenomics</td>
      <td>ancombc</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ancombc</td>
    </tr>
    <tr>
      <td></td>
      <td>annotatemyids</td>
      <td>3.17.0</td>
      <td>3.18.0</td>
      <td>bioconductor-org.hs.eg.db</td>
      <td>To update</td>
      <td>annotatemyids</td>
      <td>annotatemyids</td>
      <td>Annotation</td>
      <td></td>
      <td>annotateMyIDs: get annotation for a set of IDs using the Bioconductor annotation packages</td>
      <td>This tool can get annotation for a generic set of IDs, using the Bioconductor annotation data packages. Supported organisms are human, mouse, rat, fruit fly and zebrafish. The org.db packages that are used here are primarily based on mapping using Entrez Gene identifiers. More information on the annotation packages can be found at the Bioconductor website, for example, information on the human annotation package (org.Hs.eg.db) can be found here.</td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/annotatemyids</td>
      <td>Genome annotation</td>
      <td>annotatemyids</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/annotatemyids</td>
    </tr>
    <tr>
      <td></td>
      <td>arriba</td>
      <td>2.4.0</td>
      <td>2.4.0</td>
      <td>arriba</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Arriba detects fusion genes in RNA-Seq data after running RNA-STAR</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/suhrig/arriba</td>
      <td>Sequence Analysis, Transcriptomics</td>
      <td>arriba</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/arriba</td>
    </tr>
    <tr>
      <td></td>
      <td>augustus</td>
      <td>3.4.0</td>
      <td>3.5.0</td>
      <td>augustus</td>
      <td>To update</td>
      <td>augustus</td>
      <td>AUGUSTUS</td>
      <td>Gene prediction, Ab-initio gene prediction, Homology-based gene prediction, Homology-based gene prediction, Operation</td>
      <td>Gene transcripts, Gene and protein families</td>
      <td>AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences.</td>
      <td>AUGUSTUS is a eukaryotic gene prediction tool. It can integrate evidence, e.g. from RNA-Seq, ESTs, proteomics, but can also predict genes ab initio. The PPX extension to AUGUSTUS can take a protein sequence multiple sequence alignment as input to find new members of the family in a genome. It can be run through a web interface (see https://bio.tools/webaugustus), or downloaded and run locally.</td>
      <td>To update</td>
      <td>http://bioinf.uni-greifswald.de/augustus/</td>
      <td>Sequence Analysis</td>
      <td>augustus</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/augustus</td>
    </tr>
    <tr>
      <td></td>
      <td>bakta</td>
      <td>1.8.2</td>
      <td>1.9.1</td>
      <td>bakta</td>
      <td>To update</td>
      <td>Bakta</td>
      <td>Bakta</td>
      <td>Genome annotation</td>
      <td>Genomics, Data submission, annotation and curation, Sequence analysis</td>
      <td>"Bakta is a tool for the rapid &amp; standardized annotation of bacterial genomes and plasmids from both isolates and MAGs.It provides dbxref-rich and sORF-including annotations in machine-readable JSON &amp; bioinformatics standard file formats for automatic downstream analysis."</td>
      <td>Rapid &amp; standardized annotation of bacterial genomes, MAGs &amp; plasmids</td>
      <td>To update</td>
      <td>https://github.com/oschwengers/bakta</td>
      <td>Sequence Analysis</td>
      <td>bakta</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/bakta</td>
    </tr>
    <tr>
      <td></td>
      <td>bam_to_scidx</td>
      <td>1.0.1</td>
      <td></td>
      <td>openjdk</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Contains a tool that converts a BAM file to an ScIdx file.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/seqcode/cegr-tools/tree/master/src/org/seqcode/cegrtools/bamtoscidx</td>
      <td>Convert Formats</td>
      <td>bam_to_scidx</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bam_to_scidx</td>
    </tr>
    <tr>
      <td></td>
      <td>bamtools</td>
      <td>2.5.2</td>
      <td>2.5.2</td>
      <td>bamtools</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Operate on and transform BAM datasets in various ways using bamtools</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/pezmaster31/bamtools</td>
      <td>Sequence Analysis, SAM</td>
      <td>bamtools</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/bamtools/bamtools</td>
    </tr>
    <tr>
      <td></td>
      <td>bamtools_split</td>
      <td>2.5.2</td>
      <td>2.5.2</td>
      <td>bamtools</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/pezmaster31/bamtools</td>
      <td>Sequence Analysis, SAM</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/bamtools/bamtools_split</td>
    </tr>
    <tr>
      <td></td>
      <td>barrnap</td>
      <td>1.2.2</td>
      <td>0.9</td>
      <td>barrnap</td>
      <td>To update</td>
      <td>barrnap</td>
      <td>Barrnap</td>
      <td>Gene prediction</td>
      <td>Genomics, Model organisms, Model organisms</td>
      <td>Contains the Barrnap tool for finding ribosomal RNAs in FASTA sequences.</td>
      <td>Predict the location of ribosomal RNA genes in genomes. It supports bacteria (5S,23S,16S), archaea (5S,5.8S,23S,16S), mitochondria (12S,16S) and eukaryotes (5S,5.8S,28S,18S).</td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>barrnap</td>
      <td>iuc</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>basil</td>
      <td>1.2.0</td>
      <td>1.2.0</td>
      <td>anise_basil</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Breakpoint detection, including large insertions</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/seqan/anise_basil</td>
      <td>Variant Analysis</td>
      <td>basil</td>
      <td>iuc</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>bax2bam</td>
      <td>0.0.11</td>
      <td>0.0.11</td>
      <td>bax2bam</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>BAX to BAM converter</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/pacificbiosciences/bax2bam/</td>
      <td>Convert Formats, Sequence Analysis</td>
      <td>bax2bam</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pax2bam</td>
    </tr>
    <tr>
      <td></td>
      <td>beagle</td>
      <td>5.2_21Apr21.304</td>
      <td>5.4_22Jul22.46e</td>
      <td>beagle</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Beagle is a program for phasing and imputing missing genotypes.</td>
      <td></td>
      <td>To update</td>
      <td>https://faculty.washington.edu/browning/beagle/beagle.html</td>
      <td>Variant Analysis</td>
      <td>beagle</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/beagle</td>
    </tr>
    <tr>
      <td></td>
      <td>bellerophon</td>
      <td>1.0</td>
      <td>1.0</td>
      <td>bellerophon</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter mapped reads where the mapping spans a junction, retaining the 5-prime read.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/davebx/bellerophon</td>
      <td>Sequence Analysis</td>
      <td>bellerophon</td>
      <td>iuc</td>
      <td>https://github.com/davebx/bellerophon</td>
    </tr>
    <tr>
      <td></td>
      <td>biscot</td>
      <td>2.3.3</td>
      <td>2.3.3</td>
      <td>biscot</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Bionano scaffolding correction tool</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/institut-de-genomique/biscot</td>
      <td>Assembly</td>
      <td>biscot</td>
      <td>iuc</td>
      <td>https://github.com/bgruening/iuc/tree/master/tools/biscot</td>
    </tr>
    <tr>
      <td></td>
      <td>blastxml_to_gapped_gff3</td>
      <td>1.1</td>
      <td>0.6.6</td>
      <td>bcbiogff</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>BlastXML to gapped GFF3</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Convert Formats, Sequence Analysis</td>
      <td>blastxml_to_gapped_gff3</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/blastxml_to_gapped_gff3</td>
    </tr>
    <tr>
      <td></td>
      <td>bowtie2</td>
      <td>2.5.0</td>
      <td>2.5.2</td>
      <td>bowtie2</td>
      <td>To update</td>
      <td>bowtie2</td>
      <td>Bowtie 2</td>
      <td>Read mapping</td>
      <td>Mapping, Genomics, Mapping</td>
      <td>Bowtie2: Fast and sensitive read alignment</td>
      <td>Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes. Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome, its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes.</td>
      <td>To update</td>
      <td>http://bowtie-bio.sourceforge.net/bowtie2</td>
      <td>Next Gen Mappers</td>
      <td>bowtie2</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bowtie2</td>
    </tr>
    <tr>
      <td></td>
      <td>breseq</td>
      <td>0.35.5</td>
      <td>0.38.2</td>
      <td>breseq</td>
      <td>To update</td>
      <td>breseq</td>
      <td>breseq</td>
      <td>Polymorphism detection</td>
      <td>Sequencing, Sequence analysis, DNA mutation</td>
      <td>Predicts mutations in microbial genomes</td>
      <td>Runs Breseq software on a set of fastq files.</td>
      <td>To update</td>
      <td>https://github.com/barricklab/breseq</td>
      <td>Variant Analysis</td>
      <td>breseq</td>
      <td>iuc</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>busco</td>
      <td>5.5.0</td>
      <td>5.5.0</td>
      <td>busco</td>
      <td>Up-to-date</td>
      <td>busco</td>
      <td>BUSCO</td>
      <td>Sequence assembly validation, Scaffolding, Genome assembly, Transcriptome assembly</td>
      <td>Sequence assembly, Genomics, Transcriptomics, Sequence analysis</td>
      <td>BUSCO assess genome and annotation completeness</td>
      <td>Provides measures for quantitative assessment of genome assembly, gene set, and transcriptome completeness based on evolutionarily informed expectations of gene content from near-universal single-copy orthologs.</td>
      <td>Up-to-date</td>
      <td>https://gitlab.com/ezlab/busco/-/releases</td>
      <td>Sequence Analysis</td>
      <td>busco</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/busco/</td>
    </tr>
    <tr>
      <td></td>
      <td>bwa</td>
      <td>0.7.17</td>
      <td>0.7.17</td>
      <td>bwa</td>
      <td>Up-to-date</td>
      <td>bwa</td>
      <td>BWA</td>
      <td>Genome indexing, Sequence alignment, Read mapping, Sequence alignment, Generation, Sequence alignment, Generation, Sequence alignment, Sequence alignment</td>
      <td>Mapping</td>
      <td>Wrapper for bwa mem, aln, sampe, and samse</td>
      <td>Fast, accurate, memory-efficient aligner for short and long sequencing reads</td>
      <td>Up-to-date</td>
      <td>http://bio-bwa.sourceforge.net/</td>
      <td>Next Gen Mappers</td>
      <td>bwa</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bwa</td>
    </tr>
    <tr>
      <td></td>
      <td>bwa_mem2</td>
      <td>2.2.1</td>
      <td>2.2.1</td>
      <td>bwa-mem2</td>
      <td>Up-to-date</td>
      <td>bwa-mem2</td>
      <td>Bwa-mem2</td>
      <td>Sequence alignment</td>
      <td>Mapping</td>
      <td>Bwa-mem2 is the next version of the bwa-mem algorithm in bwa.</td>
      <td>Bwa-mem2 is the next version of the bwa-mem algorithm in bwa. It produces alignment identical to bwa and is ~1.3-3.1x faster depending on the use-case,  dataset and the running machine.</td>
      <td>Up-to-date</td>
      <td>https://github.com/bwa-mem2/bwa-mem2</td>
      <td>Next Gen Mappers</td>
      <td>bwa_mem2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bwa_mem2</td>
    </tr>
    <tr>
      <td></td>
      <td>bwameth</td>
      <td>0.2.6</td>
      <td>0.2.7</td>
      <td>bwameth</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Fast and accurate alignment of BS-seq reads</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/brentp/bwa-meth</td>
      <td>Sequence Analysis, Next Gen Mappers</td>
      <td>bwameth</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bwameth</td>
    </tr>
    <tr>
      <td></td>
      <td>calculate_numeric_param</td>
      <td>0.1.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Calculate a numeric parameter value using integer and float values.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>calculate_numeric_param</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/calculate_numeric_param</td>
    </tr>
    <tr>
      <td></td>
      <td>cemitool</td>
      <td>1.18.1</td>
      <td>1.26.0</td>
      <td>bioconductor-cemitool</td>
      <td>To update</td>
      <td>cemitool</td>
      <td>CEMiTool</td>
      <td>Enrichment analysis, Pathway or network analysis</td>
      <td>Gene expression, Transcriptomics, Microarray experiment</td>
      <td>Gene co-expression network analysis tool</td>
      <td>It unifies the discovery and the analysis of coexpression gene modules in a fully automatic manner, while providing a user-friendly html report with high quality graphs. Our tool evaluates if modules contain genes that are over-represented by specific pathways or that are altered in a specific sample group. Additionally, CEMiTool is able to integrate transcriptomic data with interactome information, identifying the potential hubs on each network.</td>
      <td>To update</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/CEMiTool.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>cemitool</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cemitool</td>
    </tr>
    <tr>
      <td></td>
      <td>charts</td>
      <td>1.0.1</td>
      <td></td>
      <td>r-getopt</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Enables advanced visualization options in Galaxy Charts</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Visualization</td>
      <td>charts</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/charts/</td>
    </tr>
    <tr>
      <td></td>
      <td>chromeister</td>
      <td>1.5.a</td>
      <td>1.5.a</td>
      <td>chromeister</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ultra-fast pairwise genome comparisons</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/estebanpw/chromeister</td>
      <td>Sequence Analysis</td>
      <td>chromeister</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/chromeister</td>
    </tr>
    <tr>
      <td></td>
      <td>circexplorer2</td>
      <td>2.3.8</td>
      <td>2.3.8</td>
      <td>circexplorer2</td>
      <td>Up-to-date</td>
      <td>circexplorer2</td>
      <td>CIRCexplorer2</td>
      <td></td>
      <td>RNA splicing, Gene transcripts, Literature and language</td>
      <td>Comprehensive and integrative circular RNA analysis toolset.</td>
      <td>Genome-wide annotation of circRNAs and their alternative back-splicing/splicing.</td>
      <td>Up-to-date</td>
      <td>https://github.com/YangLab/CIRCexplorer2</td>
      <td>RNA, Assembly</td>
      <td>circexplorer2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/circexplorer2</td>
    </tr>
    <tr>
      <td></td>
      <td>circos</td>
      <td>0.69.8</td>
      <td>0.69.9</td>
      <td>circos</td>
      <td>To update</td>
      <td>galactic_circos</td>
      <td>Galactic Circos</td>
      <td>Sequence visualisation</td>
      <td></td>
      <td>Build Circos Plots in Galaxy</td>
      <td>Galactic Circos is a Galaxy wrapper providing a GUI for the Circos tool. Circos allows visualizing data in a circular format.</td>
      <td>To update</td>
      <td>http://circos.ca/</td>
      <td>Graphics</td>
      <td>circos</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/circos</td>
    </tr>
    <tr>
      <td></td>
      <td>cite_seq_count</td>
      <td>1.4.4</td>
      <td>1.4.4</td>
      <td>cite-seq-count</td>
      <td>Up-to-date</td>
      <td>CITE-seq-Count</td>
      <td>CITE-seq-Count</td>
      <td>RNA-Seq quantification</td>
      <td>Transcriptomics, Immunoproteins and antigens</td>
      <td>Count CMO/HTO</td>
      <td>Tool for counting antibody TAGS from a CITE-seq and/or cell hashing experiment.</td>
      <td>Up-to-date</td>
      <td>https://github.com/Hoohm/CITE-seq-Count</td>
      <td>Transcriptomics</td>
      <td>cite_seq_count</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/cite_seq_count</td>
    </tr>
    <tr>
      <td></td>
      <td>clair3</td>
      <td>0.1.12</td>
      <td>1.0.5</td>
      <td>clair3</td>
      <td>To update</td>
      <td>clair3</td>
      <td>Clair3</td>
      <td>Variant calling</td>
      <td>Molecular genetics</td>
      <td>Symphonizing pileup and full-alignment for high-performance long-read variant calling</td>
      <td>Clair3 is a germline small variant caller for long-reads. Clair3 makes the best of two major method categories: pileup calling handles most variant candidates with speed, and full-alignment tackles complicated candidates to maximize precision and recall. Clair3 runs fast and has superior performance, especially at lower coverage. Clair3 is simple and modular for easy deployment and integration.</td>
      <td>To update</td>
      <td>https://github.com/HKU-BAL/Clair3</td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>clair3</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/clair3</td>
    </tr>
    <tr>
      <td></td>
      <td>clustalw</td>
      <td>2.1</td>
      <td>2.1</td>
      <td>clustalw</td>
      <td>Up-to-date</td>
      <td>\n                clustal2\n</td>
      <td></td>
      <td></td>
      <td></td>
      <td>ClustalW multiple sequence alignment program for DNA or proteins</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://www.clustal.org/clustal2/</td>
      <td>Phylogenetics, Sequence Analysis</td>
      <td>clustalw</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/clustalw</td>
    </tr>
    <tr>
      <td></td>
      <td>codeml</td>
      <td>4.9</td>
      <td>4.10.7</td>
      <td>paml</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Detects positive selection</td>
      <td></td>
      <td>To update</td>
      <td>http://abacus.gene.ucl.ac.uk/software/paml.html</td>
      <td>Phylogenetics</td>
      <td>codeml</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/codeml</td>
    </tr>
    <tr>
      <td></td>
      <td>collection_column_join</td>
      <td>0.0.3</td>
      <td>8.25</td>
      <td>coreutils</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Column Join on Collections</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>collection_column_join</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/collection_column_join</td>
    </tr>
    <tr>
      <td></td>
      <td>collection_element_identifiers</td>
      <td>0.0.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Extract element identifiers of a collection</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>collection_element_identifiers</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/collection_element_identifiers</td>
    </tr>
    <tr>
      <td></td>
      <td>column_order_header_sort</td>
      <td>0.0.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Sort Column Order by heading</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>column_order_header_sort</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/column_order_header_sort</td>
    </tr>
    <tr>
      <td></td>
      <td>column_remove_by_header</td>
      <td>1.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Remove columns by header</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>column_remove_by_header</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/column_remove_by_header</td>
    </tr>
    <tr>
      <td></td>
      <td>compose_text_param</td>
      <td>0.1.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compose a text parameter value using text, integer and float values</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>compose_text_param</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/compose_text_param</td>
    </tr>
    <tr>
      <td></td>
      <td>compress_file</td>
      <td>0.1.0</td>
      <td></td>
      <td>gzip</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compress files.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>compress_file</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/compress_file</td>
    </tr>
    <tr>
      <td></td>
      <td>concoct</td>
      <td>1.1.0</td>
      <td>1.1.0</td>
      <td>concoct</td>
      <td>Up-to-date</td>
      <td>concoct</td>
      <td>CONCOCT</td>
      <td>Sequence clustering, Read binning</td>
      <td>Metagenomics</td>
      <td>CONCOCT (Clustering cONtigs with COverage and ComposiTion) does unsupervised binning of metagenomic contigs byusing nucleotide composition - kmer frequencies - and coverage data for multiple samples. CONCOCT can accurately(up to species level) bin metagenomic contigs.</td>
      <td>A program for unsupervised binning of metagenomic contigs by using nucleotide composition, coverage data in multiple samples and linkage data from paired end reads.</td>
      <td>Up-to-date</td>
      <td>https://github.com/BinPro/CONCOCT</td>
      <td>Metagenomics</td>
      <td>concoct</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/concoct</td>
    </tr>
    <tr>
      <td></td>
      <td>crispr_studio</td>
      <td>1+galaxy0</td>
      <td>1</td>
      <td>crispr_studio</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>CRISPR Studio is a program developed to facilitate and accelerate CRISPR array visualization.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/moineaulab/CRISPRStudio</td>
      <td>Sequence Analysis</td>
      <td>crispr_studio</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/crispr_studio/</td>
    </tr>
    <tr>
      <td></td>
      <td>crosscontamination_barcode_filter</td>
      <td>0.3</td>
      <td>2.2.1</td>
      <td>r-ggplot2</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Barcode contamination discovery tool</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, Visualization</td>
      <td>crosscontamination_barcode_filter</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/crosscontamination_barcode_filter</td>
    </tr>
    <tr>
      <td></td>
      <td>cuffcompare</td>
      <td>2.2.1</td>
      <td>2.2.1</td>
      <td>cufflinks</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Galaxy wrappers for the Cuffcompare tool.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://cole-trapnell-lab.github.io/cufflinks/</td>
      <td>Transcriptomics</td>
      <td>cuffcompare</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/cufflinks/cuffcompare</td>
    </tr>
    <tr>
      <td></td>
      <td>cuffdiff</td>
      <td>2.2.1</td>
      <td>2.2.1</td>
      <td>cufflinks</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Galaxy wrappers for the Cuffdiff tool.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://cole-trapnell-lab.github.io/cufflinks/</td>
      <td>Transcriptomics</td>
      <td>cuffdiff</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/cufflinks/cuffdiff</td>
    </tr>
    <tr>
      <td></td>
      <td>cufflinks</td>
      <td>2.2.1</td>
      <td>2.2.1</td>
      <td>cufflinks</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Galaxy wrappers for the Cufflinks tool.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://cole-trapnell-lab.github.io/cufflinks/</td>
      <td>Transcriptomics</td>
      <td>cufflinks</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/cufflinks/cufflinks</td>
    </tr>
    <tr>
      <td></td>
      <td>cuffmerge</td>
      <td>2.2.1</td>
      <td>2.2.1</td>
      <td>cufflinks</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Galaxy wrappers for the Cuffmerge tool.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://cole-trapnell-lab.github.io/cufflinks/</td>
      <td>Transcriptomics</td>
      <td>cuffmerge</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/cufflinks/cuffmerge</td>
    </tr>
    <tr>
      <td></td>
      <td>cuffnorm</td>
      <td>2.2.1</td>
      <td>2.2.1</td>
      <td>cufflinks</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>The Cuffnorm tool</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://cole-trapnell-lab.github.io/cufflinks/</td>
      <td>Transcriptomics</td>
      <td>cuffnorm</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/cufflinks/cuffnorm</td>
    </tr>
    <tr>
      <td></td>
      <td>cuffquant</td>
      <td>2.2.1</td>
      <td>2.2.1</td>
      <td>cufflinks</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>The Cuffquant tool</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://cole-trapnell-lab.github.io/cufflinks/</td>
      <td>Transcriptomics</td>
      <td>cuffquant</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/cufflinks/cuffquant</td>
    </tr>
    <tr>
      <td></td>
      <td>cutadapt</td>
      <td>4.6</td>
      <td>4.6</td>
      <td>cutadapt</td>
      <td>Up-to-date</td>
      <td>cutadapt</td>
      <td>Cutadapt</td>
      <td>Sequence trimming</td>
      <td>Genomics, Probes and primers, Sequencing</td>
      <td>Flexible tool to remove adapter sequences (and quality trim) high throughput sequencing reads (fasta/fastq).</td>
      <td>Find and remove adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.</td>
      <td>Up-to-date</td>
      <td>https://cutadapt.readthedocs.org/en/stable/</td>
      <td>Fasta Manipulation, Fastq Manipulation, Sequence Analysis</td>
      <td>cutadapt</td>
      <td>lparsons</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cutadapt</td>
    </tr>
    <tr>
      <td></td>
      <td>cutesv</td>
      <td>1.0.8</td>
      <td>2.1.0</td>
      <td>cutesv</td>
      <td>To update</td>
      <td>cuteSV</td>
      <td>cuteSV</td>
      <td>Split read mapping, Genotyping, Structural variation detection</td>
      <td>DNA structural variation, Sequencing, Computer science</td>
      <td>Long-read sequencing enables the comprehensive discovery of structural variations (SVs). However, it is still non-trivial to achieve high sensitivity and performance simultaneously due to the complex SV characteristics implied by noisy long reads. Therefore, we propose cuteSV, a sensitive, fast and scalable long-read-based SV detection approach. cuteSV uses tailored methods to collect the signatures of various types of SVs and employs a clustering-and-refinement method to analyze the signatures to implement sensitive SV detection. Benchmarks on real Pacific Biosciences (PacBio) and Oxford Nanopore Technology (ONT) datasets demonstrate that cuteSV has better yields and scalability than state-of-the-art tools.</td>
      <td>Long Read based Human Genomic Structural Variation Detection with cuteSV | Long-read sequencing technologies enable to comprehensively discover structural variations (SVs). However, it is still non-trivial for state-of-the-art approaches to detect SVs with high sensitivity or high performance or both. Herein, we propose cuteSV, a sensitive, fast and lightweight SV detection approach. cuteSV uses tailored methods to comprehensively collect various types of SV signatures, and a clustering-and-refinement method to implement a stepwise SV detection, which enables to achieve high sensitivity without loss of accuracy. Benchmark results demonstrate that cuteSV has better yields on real datasets. Further, its speed and scalability are outstanding and promising to large-scale data analysis</td>
      <td>To update</td>
      <td>https://github.com/tjiangHIT/cuteSV</td>
      <td>Variant Analysis</td>
      <td>cutesv</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cutesv</td>
    </tr>
    <tr>
      <td></td>
      <td>das_tool</td>
      <td>1.1.6</td>
      <td>1.1.7</td>
      <td>das_tool</td>
      <td>To update</td>
      <td>dastool</td>
      <td>dastool</td>
      <td>Read binning</td>
      <td>Metagenomics</td>
      <td>DAS Tool for genome resolved metagenomics</td>
      <td>DAS Tool is an automated method that integrates the results of a flexible number of binning algorithms to calculate an optimized, non-redundant set of bins from a single assembly.</td>
      <td>To update</td>
      <td>https://github.com/cmks/DAS_Tool</td>
      <td>Metagenomics</td>
      <td>das_tool</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/das_tool</td>
    </tr>
    <tr>
      <td></td>
      <td>decontaminator</td>
      <td>1.0.0</td>
      <td></td>
      <td>numpy</td>
      <td>To update</td>
      <td>decontaminator</td>
      <td></td>
      <td></td>
      <td></td>
      <td>Deep Learning method for novel virus detection in sequencing data</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/cbib/decontaminator</td>
      <td>Machine Learning</td>
      <td>decontaminator</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/decontaminator</td>
    </tr>
    <tr>
      <td></td>
      <td>deepmicro</td>
      <td>1.4</td>
      <td>1.4</td>
      <td>deepmicro</td>
      <td>Up-to-date</td>
      <td>\n                DeepMicro\n</td>
      <td></td>
      <td></td>
      <td></td>
      <td>Representation learning and classification framework</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/paulzierep/DeepMicro</td>
      <td>Machine Learning</td>
      <td>deepmicro</td>
      <td>iuc</td>
      <td>https://github.com/paulzierep/DeepMicro</td>
    </tr>
    <tr>
      <td></td>
      <td>deepvariant</td>
      <td>1.5.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>DeepVariant is a deep learning-based variant caller</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/google/deepvariant</td>
      <td>Variant Analysis</td>
      <td>deepvariant</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/deepvariant</td>
    </tr>
    <tr>
      <td></td>
      <td>deg_annotate</td>
      <td>1.1.0</td>
      <td>2.31.1</td>
      <td>bedtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Annotate DESeq2/DEXSeq output tables</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics</td>
      <td>deg_annotate</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/deg_annotate</td>
    </tr>
    <tr>
      <td></td>
      <td>deseq2</td>
      <td>2.11.40.8</td>
      <td>1.42.0</td>
      <td>bioconductor-deseq2</td>
      <td>To update</td>
      <td>DESeq2</td>
      <td>DESeq2</td>
      <td>Differential gene expression analysis</td>
      <td>Transcriptomics</td>
      <td>Differential gene expression analysis based on the negative binomial distribution</td>
      <td>R/Bioconductor package for differential gene expression analysis based on the negative binomial distribution. Estimate variance-mean dependence in count data from high-throughput sequencing assays and test for differential expression based on a model using the negative binomial distribution.</td>
      <td>To update</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/DESeq2.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>deseq2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/deseq2</td>
    </tr>
    <tr>
      <td></td>
      <td>dexseq</td>
      <td>1.44</td>
      <td>1.48.0</td>
      <td>bioconductor-dexseq</td>
      <td>To update</td>
      <td>dexseq</td>
      <td>DEXSeq</td>
      <td>Enrichment analysis, Exonic splicing enhancer prediction</td>
      <td>RNA-Seq</td>
      <td>Inference of differential exon usage in RNA-Seq</td>
      <td>The package is focused on finding differential exon usage using RNA-seq exon counts between samples with different experimental designs. It provides functions that allows the user to make the necessary statistical tests based on a model that uses the negative binomial distribution to estimate the variance between biological replicates and generalized linear models for testing. The package also provides functions for the visualization and exploration of the results.</td>
      <td>To update</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/DEXSeq.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>dexseq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/dexseq</td>
    </tr>
    <tr>
      <td></td>
      <td>diffbind</td>
      <td>2.10.0</td>
      <td>3.12.0</td>
      <td>bioconductor-diffbind</td>
      <td>To update</td>
      <td>diffbind</td>
      <td>DiffBind</td>
      <td>Differential binding analysis</td>
      <td>ChIP-seq</td>
      <td>Diffbind provides functions for  processing  ChIP-Seq  data.</td>
      <td>Compute differentially bound sites from multiple ChIP-seq experiments using affinity (quantitative) data. Also enables occupancy (overlap) analysis and plotting functions.</td>
      <td>To update</td>
      <td>http://bioconductor.org/packages/release/bioc/html/DiffBind.html</td>
      <td>ChIP-seq</td>
      <td>diffbind</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/diffbind</td>
    </tr>
    <tr>
      <td></td>
      <td>disco</td>
      <td></td>
      <td>1.2</td>
      <td>disco</td>
      <td>To update</td>
      <td>disco</td>
      <td>DISCO</td>
      <td>Protein sequence analysis</td>
      <td>Structure determination</td>
      <td>DISCO is a overlap-layout-consensus (OLC) metagenome assembler</td>
      <td>DISCO is software to perform structure determination of protein homo-oligomers with cyclic symmetry.DISCO computes oligomeric protein structures using geometric constraints derived from RDCs and intermolecular distance restraints such as NOEs or disulfide bonds. When a reliable subunit structure can be calculated from intramolecular restraints, DISCO guarantees that all satisfying oligomer structures will be discovered, yet can run in minutes to hours on only a single desktop-class computer.</td>
      <td>To update</td>
      <td>http://disco.omicsbio.org/</td>
      <td>Metagenomics, Assembly</td>
      <td>disco</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/disco/</td>
    </tr>
    <tr>
      <td></td>
      <td>dropletutils</td>
      <td>1.10.0</td>
      <td>1.22.0</td>
      <td>bioconductor-dropletutils</td>
      <td>To update</td>
      <td>dropletutils</td>
      <td>DropletUtils</td>
      <td>Loading, Community profiling</td>
      <td>Gene expression, RNA-seq, Sequencing, Transcriptomics</td>
      <td>DropletUtils - Utilities for handling droplet-based single-cell RNA-seq data</td>
      <td>Provides a number of utility functions for handling single-cell (RNA-seq) data from droplet technologies such as 10X Genomics. This includes data loading, identification of cells from empty droplets, removal of barcode-swapped pseudo-cells, and downsampling of the count matrix.</td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/devel/bioc/html/DropletUtils.html</td>
      <td>Transcriptomics, Sequence Analysis</td>
      <td>dropletutils</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/dropletutils/</td>
    </tr>
    <tr>
      <td></td>
      <td>edger</td>
      <td>3.36.0</td>
      <td>4.0.2</td>
      <td>bioconductor-edger</td>
      <td>To update</td>
      <td>edger</td>
      <td>edgeR</td>
      <td>Differential gene expression analysis</td>
      <td>Genetics, RNA-Seq, ChIP-seq</td>
      <td>Perform RNA-Seq differential expression analysis using edgeR pipeline</td>
      <td>Differential expression analysis of RNA-seq expression profiles with biological replication. Implements a range of statistical methodology based on the negative binomial distributions, including empirical Bayes estimation, exact tests, generalized linear models and quasi-likelihood tests. As well as RNA-seq, it be applied to differential signal analysis of other types of genomic data that produce counts, including ChIP-seq, SAGE and CAGE.</td>
      <td>To update</td>
      <td>http://bioconductor.org/packages/release/bioc/html/edgeR.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>edger</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/edger</td>
    </tr>
    <tr>
      <td></td>
      <td>egsea</td>
      <td>1.20.0</td>
      <td>1.28.0</td>
      <td>bioconductor-egsea</td>
      <td>To update</td>
      <td>egsea</td>
      <td>EGSEA</td>
      <td>Gene set testing</td>
      <td>Systems biology</td>
      <td>This tool implements the Ensemble of Gene Set Enrichment Analyses (EGSEA) method for gene set testing</td>
      <td>This package implements the Ensemble of Gene Set Enrichment Analyses method for gene set testing.</td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/EGSEA.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>egsea</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/egsea</td>
    </tr>
    <tr>
      <td></td>
      <td>ena_upload</td>
      <td>0.6.3</td>
      <td>0.7.0</td>
      <td>ena-upload-cli</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Submits experimental data and respective metadata to the European Nucleotide Archive (ENA).</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/usegalaxy-eu/ena-upload-cli</td>
      <td>Data Export</td>
      <td>ena_upload</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ena_upload</td>
    </tr>
    <tr>
      <td></td>
      <td>ensembl_vep</td>
      <td>110.1</td>
      <td>110.1</td>
      <td>ensembl-vep</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Ensembl VEP: Annotate VCFs with variant effect predictions</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/Ensembl/ensembl-vep</td>
      <td>Variant Analysis</td>
      <td>ensembl_vep</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ensembl_vep</td>
    </tr>
    <tr>
      <td></td>
      <td>exomedepth</td>
      <td>1.1.0</td>
      <td>1.1.16</td>
      <td>r-exomedepth</td>
      <td>To update</td>
      <td>exomedepth</td>
      <td>ExomeDepth</td>
      <td>Sequence analysis, Variant calling, Genotyping, Copy number estimation</td>
      <td>Exome sequencing, Gene transcripts, Mapping, Sequencing, Genetic variation, Rare diseases</td>
      <td>ExomeDepth: Calls copy number variants (CNVs) from targeted sequence data</td>
      <td>Copy number variant (CNV) calling algorithm designed to control technical variability between samples. It calls CNVs from targeted sequence data, typically exome sequencing experiments designed to identify the genetic basis of Mendelian disorders.</td>
      <td>To update</td>
      <td>https://cran.r-project.org/package=ExomeDepth</td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>exomedepth</td>
      <td>crs4</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/exomedepth</td>
    </tr>
    <tr>
      <td></td>
      <td>exonerate</td>
      <td>2.4.0</td>
      <td>2.4.0</td>
      <td>exonerate</td>
      <td>Up-to-date</td>
      <td>exonerate</td>
      <td>Exonerate</td>
      <td>Pairwise sequence alignment, Protein threading, Genome alignment</td>
      <td>Sequence analysis, Sequence sites, features and motifs, Molecular interactions, pathways and networks</td>
      <td>Exonerate is a generic tool for pairwise sequence comparison.</td>
      <td>A tool for pairwise sequence alignment. It enables alignment for DNA-DNA and DNA-protein pairs and also gapped and ungapped alignment.</td>
      <td>Up-to-date</td>
      <td>https://www.ebi.ac.uk/about/vertebrate-genomics/software/exonerate</td>
      <td>Sequence Analysis</td>
      <td>exonerate</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/exonerate</td>
    </tr>
    <tr>
      <td></td>
      <td>export2graphlan</td>
      <td>0.20</td>
      <td>0.22</td>
      <td>export2graphlan</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>export2graphlan is a conversion software tool for producing both annotation and tree file for GraPhlAn</td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/CibioCM/export2graphlan/overview</td>
      <td>Metagenomics</td>
      <td>export2graphlan</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/export2graphlan/</td>
    </tr>
    <tr>
      <td></td>
      <td>fargene</td>
      <td>0.1</td>
      <td>0.1</td>
      <td>fargene</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>fARGene (Fragmented Antibiotic Resistance Gene iENntifiEr )</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/fannyhb/fargene</td>
      <td>Sequence Analysis</td>
      <td>fargene</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fargene</td>
    </tr>
    <tr>
      <td></td>
      <td>fasplit</td>
      <td>377</td>
      <td>377</td>
      <td>ucsc-fasplit</td>
      <td>Up-to-date</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td>UCSC Genome Browser Utilities</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>faSplit is a tool to split a single FASTA file into several files</td>
      <td>Utilities for handling sequences and assemblies from the UCSC Genome Browser project.</td>
      <td>Up-to-date</td>
      <td>http://hgdownload.cse.ucsc.edu/admin/exe/</td>
      <td>Fasta Manipulation</td>
      <td>ucsc_fasplit</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ucsc-tools/fasplit</td>
    </tr>
    <tr>
      <td></td>
      <td>fasta_nucleotide_color_plot</td>
      <td>1.0.1</td>
      <td></td>
      <td>openjdk</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Contains a tool that produces a graphical representation of FASTA data with each nucleotide represented by a selected color.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/seqcode/cegr-tools/tree/master/src/org/seqcode/cegrtools/fourcolorplot</td>
      <td>Visualization</td>
      <td>fasta_nucleotide_color_plot</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fasta_nucleotide_color_plot</td>
    </tr>
    <tr>
      <td></td>
      <td>fastani</td>
      <td>1.3</td>
      <td>1.34</td>
      <td>fastani</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Fast alignment-free computation of whole-genome Average Nucleotide Identity</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ParBLiSS/FastANI</td>
      <td>Sequence Analysis</td>
      <td>fastani</td>
      <td>iuc</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>fastp</td>
      <td></td>
      <td>0.23.4</td>
      <td>fastp</td>
      <td>To update</td>
      <td>\n                fastp\n</td>
      <td></td>
      <td></td>
      <td></td>
      <td>Fast all-in-one preprocessing for FASTQ files</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/OpenGene/fastp</td>
      <td>Sequence Analysis</td>
      <td>fastp</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fastp</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_combiner</td>
      <td>1.1.5</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Combine FASTA and QUAL into FASTQ</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation, Fasta Manipulation</td>
      <td>fastq_combiner</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_combiner</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_filter</td>
      <td>1.1.5</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter FASTQ reads by quality score and length</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_filter</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_filter</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_groomer</td>
      <td>1.1.5</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert between various FASTQ quality formats.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_groomer</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_groomer</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_manipulation</td>
      <td>1.1.5</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Manipulate FASTQ reads on various attributes.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_manipulation</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_manipulation</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_masker_by_quality</td>
      <td>1.1.5</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FASTQ Masker by quality score</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_masker_by_quality</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_masker_by_quality</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_paired_end_deinterlacer</td>
      <td>1.1.5</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FASTQ de-interlacer on paired end reads.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_paired_end_deinterlacer</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_paired_end_deinterlacer</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_paired_end_interlacer</td>
      <td>1.1.5</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FASTQ interlacer on paired end reads</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_paired_end_interlacer</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_paired_end_interlacer</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_paired_end_joiner</td>
      <td>1.1.5</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FASTQ joiner on paired end reads</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_paired_end_joiner</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_paired_end_joiner</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_paired_end_splitter</td>
      <td>1.1.5</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FASTQ splitter on joined paired end reads</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_paired_end_splitter</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_paired_end_splitter</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_stats</td>
      <td>1.1.5</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FASTQ Summary Statistics by column</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_stats</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_stats</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_to_tabular</td>
      <td>1.1.5</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FASTQ to Tabular converter</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_to_tabular</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_to_tabular</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_trimmer</td>
      <td>1.1.5</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FASTQ Trimmer by quality</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>fastq_trimmer</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/fastq_trimmer</td>
    </tr>
    <tr>
      <td></td>
      <td>fastqc</td>
      <td>0.74+galaxy0</td>
      <td>0.12.1</td>
      <td>fastqc</td>
      <td>To update</td>
      <td>fastqc</td>
      <td>FastQC</td>
      <td>Sequence composition calculation, Sequencing quality control, Statistical calculation</td>
      <td>Sequencing, Data quality management, Sequence analysis</td>
      <td>Read QC reports using FastQC</td>
      <td>This tool aims to provide a QC report which can spot problems or biases which originate either in the sequencer or in the starting library material. It can be run in one of two modes. It can either run as a stand alone interactive application for the immediate analysis of small numbers of FastQ files, or it can be run in a non-interactive mode where it would be suitable for integrating into a larger analysis pipeline for the systematic processing of large numbers of files.</td>
      <td>To update</td>
      <td>http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</td>
      <td>Fastq Manipulation</td>
      <td>fastqc</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fastqc</td>
    </tr>
    <tr>
      <td></td>
      <td>fastqe</td>
      <td>0.2.6+galaxy2</td>
      <td>0.3.1</td>
      <td>fastqe</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FASTQE</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fastqe</td>
      <td>Sequence Analysis</td>
      <td>fastqe</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fastqe</td>
    </tr>
    <tr>
      <td></td>
      <td>fasttree</td>
      <td>2.1.10</td>
      <td>2.1.11</td>
      <td>fasttree</td>
      <td>To update</td>
      <td>fasttree</td>
      <td>FastTree</td>
      <td>Phylogenetic tree generation (from molecular sequences), Phylogenetic tree generation (maximum likelihood and Bayesian methods)</td>
      <td>Phylogenetics, Sequence analysis</td>
      <td>FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide or protein sequences - GVL</td>
      <td>Infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide or protein sequences.</td>
      <td>To update</td>
      <td>http://www.microbesonline.org/fasttree/</td>
      <td>Phylogenetics</td>
      <td>fasttree</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fasttree</td>
    </tr>
    <tr>
      <td></td>
      <td>fatovcf</td>
      <td>448</td>
      <td>448</td>
      <td>ucsc-fatovcf</td>
      <td>Up-to-date</td>
      <td>UCSC_Genome_Browser_Utilities</td>
      <td>UCSC Genome Browser Utilities</td>
      <td></td>
      <td>Sequence analysis</td>
      <td>Convert a FASTA alignment file to Variant Call Format (VCF) single-nucleotide diffs</td>
      <td>Utilities for handling sequences and assemblies from the UCSC Genome Browser project.</td>
      <td>Up-to-date</td>
      <td>http://hgdownload.cse.ucsc.edu/admin/exe/</td>
      <td>Convert Formats</td>
      <td>ucsc_fatovcf</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ucsc-tools/fatovcf</td>
    </tr>
    <tr>
      <td></td>
      <td>featurecounts</td>
      <td>2.0.3</td>
      <td>2.0.6</td>
      <td>subread</td>
      <td>To update</td>
      <td>featurecounts</td>
      <td>FeatureCounts</td>
      <td>Read summarisation</td>
      <td>Sequencing</td>
      <td>featureCounts counts the number of reads aligned to defined masked regions in a reference genome</td>
      <td>featureCounts is a very efficient read quantifier. It can be used to summarize RNA-seq reads and gDNA-seq reads to a variety of genomic features such as genes, exons, promoters, gene bodies and genomic bins. It is included in the Bioconductor Rsubread package and also in the SourceForge Subread package.</td>
      <td>To update</td>
      <td>http://bioinf.wehi.edu.au/featureCounts</td>
      <td>RNA, Transcriptomics, Sequence Analysis</td>
      <td>featurecounts</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/featurecounts</td>
    </tr>
    <tr>
      <td></td>
      <td>feelnc</td>
      <td>0.2.1</td>
      <td>0.2</td>
      <td>feelnc</td>
      <td>To update</td>
      <td>feelnc</td>
      <td>FEELnc</td>
      <td>Annotation, Classification</td>
      <td>RNA-seq, Functional, regulatory and non-coding RNA</td>
      <td>Galaxy wrapper for FEELnc</td>
      <td>A tool to annotate long non-coding RNAs from RNA-seq assembled transcripts.</td>
      <td>To update</td>
      <td>https://github.com/tderrien/FEELnc</td>
      <td>Sequence Analysis</td>
      <td>feelnc</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/feelnc</td>
    </tr>
    <tr>
      <td></td>
      <td>fgsea</td>
      <td>1.8.0+galaxy1</td>
      <td>1.28.0</td>
      <td>bioconductor-fgsea</td>
      <td>To update</td>
      <td>fgsea</td>
      <td>fgsea</td>
      <td>Gene-set enrichment analysis</td>
      <td>Genetics</td>
      <td>Perform gene set testing using fgsea</td>
      <td>The package implements an algorithm for fast gene set enrichment analysis. Using the fast algorithm allows to make more permutations and get more fine grained p-values, which allows to use accurate stantard approaches to multiple hypothesis correction.</td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/fgsea.html</td>
      <td>Visualization, Transcriptomics, Statistics</td>
      <td>fgsea</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fgsea</td>
    </tr>
    <tr>
      <td></td>
      <td>filtlong</td>
      <td>0.2.1</td>
      <td>0.2.1</td>
      <td>filtlong</td>
      <td>Up-to-date</td>
      <td>\n                filtlong\n</td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filtlong - Filtering long reads by quality</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/rrwick/Filtlong</td>
      <td>Fastq Manipulation, Sequence Analysis</td>
      <td>filtlong</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/filtlong</td>
    </tr>
    <tr>
      <td></td>
      <td>flash</td>
      <td>1.2.11</td>
      <td>1.2.11</td>
      <td>flash</td>
      <td>Up-to-date</td>
      <td>flash</td>
      <td>FLASH</td>
      <td>Read pre-processing, Sequence merging, Sequence assembly</td>
      <td>Sequencing, Sequence assembly</td>
      <td>Fast Length Adjustment of SHort reads</td>
      <td>Identifies paired-end reads which overlap in the middle, converting them to single long reads</td>
      <td>Up-to-date</td>
      <td>https://ccb.jhu.edu/software/FLASH/</td>
      <td>Assembly, Fastq Manipulation</td>
      <td>flash</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/flash</td>
    </tr>
    <tr>
      <td></td>
      <td>fraggenescan</td>
      <td></td>
      <td>1.31</td>
      <td>fraggenescan</td>
      <td>To update</td>
      <td>fraggenescan</td>
      <td>FragGeneScan</td>
      <td>Gene prediction</td>
      <td>Genetics, Sequence analysis</td>
      <td>Tool for finding (fragmented) genes in short read</td>
      <td>Application for finding (fragmented) genes in short reads</td>
      <td>To update</td>
      <td>https://sourceforge.net/projects/fraggenescan/</td>
      <td>Sequence Analysis</td>
      <td>fraggenescan</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fraggenescan/</td>
    </tr>
    <tr>
      <td></td>
      <td>freebayes</td>
      <td>1.3.6</td>
      <td>1.3.7</td>
      <td>freebayes</td>
      <td>To update</td>
      <td>freebayes</td>
      <td>FreeBayes</td>
      <td>Variant calling, Statistical calculation</td>
      <td>Genomics, Genetic variation, Rare diseases</td>
      <td>Galaxy Freebayes Bayesian genetic variant detector tool</td>
      <td>Bayesian genetic variant detector designed to find small polymorphisms, specifically SNPs, indels, multi-nucleotide polymorphisms, and complex events (composite insertion and substitution events) smaller than the length of a short-read sequencing alignment.</td>
      <td>To update</td>
      <td>https://github.com/ekg/freebayes</td>
      <td>Variant Analysis</td>
      <td>freebayes</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/freebayes</td>
    </tr>
    <tr>
      <td></td>
      <td>gecko</td>
      <td>1.2</td>
      <td>1.2</td>
      <td>gecko</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Ungapped genome comparison</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/otorreno/gecko</td>
      <td>Sequence Analysis</td>
      <td>gecko</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gecko</td>
    </tr>
    <tr>
      <td></td>
      <td>genomescope</td>
      <td>2.0</td>
      <td>2.0</td>
      <td>genomescope2</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Analyze unassembled short reads</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/tbenavi1/genomescope2.0</td>
      <td>Statistics</td>
      <td>genomescope</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/genomescope</td>
    </tr>
    <tr>
      <td></td>
      <td>genomic_super_signature</td>
      <td>1.2.0</td>
      <td>1.10.0</td>
      <td>bioconductor-genomicsupersignature</td>
      <td>To update</td>
      <td>genomicsupersignature</td>
      <td>GenomicSuperSignature</td>
      <td>Gene-set enrichment analysis, Essential dynamics, Deposition, Principal component visualisation, Dimensionality reduction</td>
      <td>RNA-Seq, Transcriptomics, Microbial ecology, Genotype and phenotype, Microarray experiment</td>
      <td>Interpretation of RNAseq experiments through robust, efficient comparison to public databases</td>
      <td>GenomicSuperSignature is a package for the interpretation of RNA-seq experiments through robust, efficient comparison to public databases.</td>
      <td>To update</td>
      <td>https://github.com/shbrief/GenomicSuperSignature</td>
      <td>Sequence Analysis, RNA, Transcriptomics</td>
      <td>genomic_super_signature</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/genomic_super_signature</td>
    </tr>
    <tr>
      <td></td>
      <td>genrich</td>
      <td>0.5+galaxy2</td>
      <td>0.6.1</td>
      <td>genrich</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Genrich is a peak-caller for genomic enrichment assays (e.g. ChIP-seq, ATAC-seq).</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/jsh58/Genrich</td>
      <td>ChIP-seq</td>
      <td>genrich</td>
      <td>iuc</td>
      <td>https://github.com/jsh58/Genrich</td>
    </tr>
    <tr>
      <td></td>
      <td>get_hrun</td>
      <td>0.5.9.2</td>
      <td>0.8.0</td>
      <td>pyfaidx</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Annotate indel variants with homopolymer context</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/get_hrun</td>
      <td>Variant Analysis</td>
      <td>get_hrun</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/get_hrun</td>
    </tr>
    <tr>
      <td></td>
      <td>gfa_to_fa</td>
      <td>0.1.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>gfa_to_fa - Converting GFA format to Fasta format</td>
      <td></td>
      <td>To update</td>
      <td>http://gfa-spec.github.io/GFA-spec/</td>
      <td>Convert Formats</td>
      <td>gfa_to_fa</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gfa_to_fa</td>
    </tr>
    <tr>
      <td></td>
      <td>gffcompare</td>
      <td>0.12.6</td>
      <td>0.12.6</td>
      <td>gffcompare</td>
      <td>Up-to-date</td>
      <td>gffcompare</td>
      <td>gffcompare</td>
      <td>Sequence annotation</td>
      <td>Nucleic acids, Sequence analysis</td>
      <td>Galaxy wrappers for Geo Pertea's GffCompare package.</td>
      <td>Program for comparing, annotating, merging and tracking transcripts in GFF files.</td>
      <td>Up-to-date</td>
      <td>https://github.com/gpertea/gffcompare/</td>
      <td>Transcriptomics</td>
      <td>gffcompare</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gffcompare</td>
    </tr>
    <tr>
      <td></td>
      <td>gffread</td>
      <td>0.12.7</td>
      <td>0.12.7</td>
      <td>gffread</td>
      <td>Up-to-date</td>
      <td>gffread</td>
      <td>gffread</td>
      <td>Sequence annotation</td>
      <td>Nucleic acids, Sequence analysis</td>
      <td>gffread filters and/or converts GFF3/GTF2 records</td>
      <td>program for filtering, converting and manipulating GFF files</td>
      <td>Up-to-date</td>
      <td>http://ccb.jhu.edu/software/stringtie/gff.shtml#gffread/</td>
      <td>Sequence Analysis</td>
      <td>gffread</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gffread</td>
    </tr>
    <tr>
      <td></td>
      <td>goenrichment</td>
      <td>2.0.1</td>
      <td>2.0.1</td>
      <td>goenrichment</td>
      <td>Up-to-date</td>
      <td>goenrichment</td>
      <td>GOEnrichment</td>
      <td>Gene-set enrichment analysis</td>
      <td>Transcriptomics</td>
      <td>Performs GO Enrichment analysis.</td>
      <td>GOEnrichment is a tool for performing GO enrichment analysis of gene sets, such as those obtained from RNA-seq or Microarray experiments, to help characterize them at the functional level. It is available in Galaxy Europe and as a stand-alone tool.GOEnrichment is flexible in that it allows the user to use any version of the Gene Ontology and any GO annotation file they desire. To enable the use of GO slims, it is accompanied by a sister tool GOSlimmer, which can convert annotation files from full GO to any specified GO slim.The tool features an optional graph clustering algorithm to reduce the redundancy in the set of enriched GO terms and simplify its output.It was developed by the BioData.pt / ELIXIR-PT team at the Instituto Gulbenkian de Ciência.</td>
      <td>Up-to-date</td>
      <td>https://github.com/DanFaria/GOEnrichment</td>
      <td>Genome annotation</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/goenrichment</td>
    </tr>
    <tr>
      <td></td>
      <td>goseq</td>
      <td>1.50.0</td>
      <td>1.54.0</td>
      <td>bioconductor-goseq</td>
      <td>To update</td>
      <td>goseq</td>
      <td>GOseq</td>
      <td>Gene functional annotation</td>
      <td>RNA-Seq</td>
      <td>goseq does selection-unbiased testing for category enrichment amongst differentially expressed (DE) genes for RNA-seq data</td>
      <td>Detect Gene Ontology and/or other user defined categories which are over/under represented in RNA-seq data.</td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/goseq.html</td>
      <td>Statistics, RNA, Micro-array Analysis</td>
      <td>goseq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/goseq</td>
    </tr>
    <tr>
      <td></td>
      <td>graphembed</td>
      <td>2.4</td>
      <td>2.4</td>
      <td>graph_embed</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute a 2D embedding of a data matrix given supervised class information</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/fabriziocosta/GraphEmbed</td>
      <td>Statistics, Graphics</td>
      <td>graphembed</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/graphembed/</td>
    </tr>
    <tr>
      <td></td>
      <td>graphlan</td>
      <td></td>
      <td>1.1.3</td>
      <td>graphlan</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>GraPhlAn is a software tool for producing high-quality circular representations of taxonomic and phylogenetic trees</td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/nsegata/graphlan/overview</td>
      <td>Metagenomics, Graphics, Phylogenetics</td>
      <td>graphlan</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/humann2/</td>
    </tr>
    <tr>
      <td></td>
      <td>gubbins</td>
      <td>3.2.1</td>
      <td>3.3.1</td>
      <td>gubbins</td>
      <td>To update</td>
      <td>gubbins</td>
      <td>Gubbins</td>
      <td>Genotyping, Phylogenetic inference, Ancestral reconstruction</td>
      <td>Phylogeny, Genotype and phenotype, Whole genome sequencing</td>
      <td>Gubbins - bacterial recombination detection</td>
      <td>Gubbins is a tool for rapid phylogenetic analysis of large samples of recombinant bacterial whole genome sequences.</td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>gubbins</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gubbins</td>
    </tr>
    <tr>
      <td></td>
      <td>hapog</td>
      <td>1.3.6</td>
      <td>1.3.7</td>
      <td>hapog</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Hapo-G - Haplotype-Aware Polishing of Genomes</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/institut-de-genomique/HAPO-G</td>
      <td>Assembly</td>
      <td>hapog</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hapog</td>
    </tr>
    <tr>
      <td></td>
      <td>heinz</td>
      <td>1.0</td>
      <td>1.62.0</td>
      <td>bioconductor-bionet</td>
      <td>To update</td>
      <td>bionet</td>
      <td>BioNet</td>
      <td>Protein interaction analysis</td>
      <td>Molecular interactions, pathways and networks, Protein interactions</td>
      <td>An algorithm for identification of the optimal scoring subnetwork.</td>
      <td>This package provides functions for the integrated analysis of protein-protein interaction networks and the detection of functional modules. Different datasets can be integrated into the network by assigning p-values of statistical tests to the nodes of the network. By fitting a beta-uniform mixture model and calculating scores from these p-values, overall scores of network regions can be calculated and an integer linear programming algorithm identifies the maximum scoring subnetwork.</td>
      <td>To update</td>
      <td>https://github.com/ls-cwi/heinz</td>
      <td>Transcriptomics, Visualization, Statistics</td>
      <td>heinz</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/heinz</td>
    </tr>
    <tr>
      <td></td>
      <td>hifiasm_meta</td>
      <td>0.3.1</td>
      <td>hamtv0.3.1</td>
      <td>hifiasm_meta</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A hifiasm fork for metagenome assembly using Hifi reads.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/xfengnefx/hifiasm-meta</td>
      <td>Metagenomics</td>
      <td>hifiasm_meta</td>
      <td>galaxy-australia</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hifiasm_meta</td>
    </tr>
    <tr>
      <td></td>
      <td>hisat2</td>
      <td>2.2.1</td>
      <td>2.2.1</td>
      <td>hisat2</td>
      <td>Up-to-date</td>
      <td>hisat2</td>
      <td>HISAT2</td>
      <td>Sequence alignment</td>
      <td>RNA-seq</td>
      <td>HISAT2 is a fast and sensitive spliced alignment program.</td>
      <td>Alignment program for mapping next-generation sequencing reads (both DNA and RNA) to a population of human genomes (as well as to a single reference genome).</td>
      <td>Up-to-date</td>
      <td>http://ccb.jhu.edu/software/hisat2/</td>
      <td>Assembly</td>
      <td>hisat2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hisat2</td>
    </tr>
    <tr>
      <td></td>
      <td>htseq_count</td>
      <td>0.9.1+galaxy1</td>
      <td>2.0.4</td>
      <td>htseq</td>
      <td>To update</td>
      <td>htseq</td>
      <td>HTSeq</td>
      <td>Nucleic acid sequence analysis</td>
      <td>Sequence analysis</td>
      <td>Count aligned reads (SAM/BAM) that overlap genomic features (GFF)</td>
      <td>Python framework to process and analyse high-throughput sequencing (HTS) data</td>
      <td>To update</td>
      <td>https://readthedocs.org/projects/htseq/</td>
      <td>Genomic Interval Operations, SAM, Sequence Analysis, RNA</td>
      <td>htseq_count</td>
      <td>lparsons</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/htseq_count</td>
    </tr>
    <tr>
      <td></td>
      <td>humann</td>
      <td>3.8</td>
      <td>3.8</td>
      <td>humann</td>
      <td>Up-to-date</td>
      <td>humann</td>
      <td>humann</td>
      <td>Species frequency estimation, Taxonomic classification, Phylogenetic analysis</td>
      <td>Metagenomics, Phylogenomics</td>
      <td>HUMAnN for functionally profiling metagenomes and metatranscriptomes at species-level resolution</td>
      <td>HUMAnN is a pipeline for efficiently and accurately profiling the presence/absence and abundance of microbial pathways in a community from metagenomic or metatranscriptomic sequencing data (typically millions of short DNA/RNA reads). This process, referred to as functional profiling, aims to describe the metabolic potential of a microbial community and its members. More generally, functional profiling answers the question “What are the microbes in my community-of-interest doing (or are capable of doing)?”</td>
      <td>Up-to-date</td>
      <td>http://huttenhower.sph.harvard.edu/humann</td>
      <td>Metagenomics</td>
      <td>humann</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/humann</td>
    </tr>
    <tr>
      <td></td>
      <td>hypo</td>
      <td>1.0.3</td>
      <td>1.0.3</td>
      <td>hypo</td>
      <td>Up-to-date</td>
      <td>HyPo</td>
      <td>HyPo</td>
      <td>Optimisation and refinement, Genome assembly</td>
      <td>Sequence assembly, Genomics</td>
      <td>Super Fast &amp; Accurate Polisher for Long Read Genome Assemblies</td>
      <td>HyPo, a Hybrid Polisher, utilizes short as well as long reads within a single run to polish a long reads assembly of small and large genomes.</td>
      <td>Up-to-date</td>
      <td>https://github.com/kensung-lab/hypo</td>
      <td>Assembly</td>
      <td>hypo</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hypo</td>
    </tr>
    <tr>
      <td></td>
      <td>idba_ud</td>
      <td></td>
      <td>1.1.3</td>
      <td>idba</td>
      <td>To update</td>
      <td>idba</td>
      <td>IDBA</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly</td>
      <td>Wrappers for the idba assembler variants.</td>
      <td>A short read assembler based on iterative De Bruijn graph. It is developed under 64-bit Linux, but should be suitable for all unix-like system.</td>
      <td>To update</td>
      <td>https://i.cs.hku.hk/~alse/hkubrg/projects/index.html</td>
      <td>Assembly</td>
      <td>idba</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/idba_ud</td>
    </tr>
    <tr>
      <td></td>
      <td>idr</td>
      <td>2.0.3</td>
      <td>2.0.4.2</td>
      <td>idr</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Galaxy wrappers for the IDR package from Nathan Boleu</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/nboley/idr</td>
      <td>Sequence Analysis</td>
      <td>idr</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/idr/</td>
    </tr>
    <tr>
      <td></td>
      <td>iedb_api</td>
      <td>2.15.2</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Get epitope binding predictions from IEDB-API</td>
      <td></td>
      <td>To update</td>
      <td>http://tools.immuneepitope.org/main/tools-api/</td>
      <td>Data Source, Sequence Analysis</td>
      <td>iedb_api</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/iedb_api</td>
    </tr>
    <tr>
      <td></td>
      <td>integron_finder</td>
      <td>2.0.2</td>
      <td>3.4</td>
      <td>hmmer</td>
      <td>To update</td>
      <td>integron_finder</td>
      <td>Integron Finder</td>
      <td>Nucleic acid feature detection, Sequence motif recognition, Protein feature detection, Genome annotation</td>
      <td>Functional genomics, Mobile genetic elements, Molecular biology, Sequence analysis</td>
      <td>"IntegronFinder identify integrons with high accuracy and sensitivity.It searches for attC sites using covariance models, for integron-integrases using HMM profiles, and for other features (promoters, attI site) using pattern matching"</td>
      <td>A tool to detect Integron in DNA sequences.</td>
      <td>To update</td>
      <td>https://github.com/gem-pasteur/Integron_Finder</td>
      <td>Sequence Analysis</td>
      <td>integronfinder</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/integron_finder</td>
    </tr>
    <tr>
      <td></td>
      <td>interproscan</td>
      <td>5.59-91.0</td>
      <td>5.59_91.0</td>
      <td>interproscan</td>
      <td>To update</td>
      <td>interproscan_ebi</td>
      <td>InterProScan (EBI)</td>
      <td>Sequence motif recognition, Protein feature detection</td>
      <td>Gene and protein families, Sequence analysis</td>
      <td>Interproscan queries the interpro database and provides annotations.</td>
      <td>Scan sequences against the InterPro protein signature databases.</td>
      <td>To update</td>
      <td>http://www.ebi.ac.uk/Tools/pfa/iprscan5/</td>
      <td>Sequence Analysis</td>
      <td>interproscan</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/interproscan</td>
    </tr>
    <tr>
      <td></td>
      <td>iqtree</td>
      <td>2.1.2</td>
      <td>2.2.6</td>
      <td>iqtree</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Efficient phylogenomic software by maximum likelihood</td>
      <td></td>
      <td>To update</td>
      <td>http://www.iqtree.org/</td>
      <td>Phylogenetics</td>
      <td>iqtree</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/iqtree/</td>
    </tr>
    <tr>
      <td></td>
      <td>irissv</td>
      <td>1.0.4</td>
      <td>1.19</td>
      <td>samtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Refine insertion sequences</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/mkirsche/Iris</td>
      <td>Variant Analysis</td>
      <td>irissv</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tools/irissv/</td>
    </tr>
    <tr>
      <td></td>
      <td>isescan</td>
      <td>1.7.2.3</td>
      <td>1.7.2.1</td>
      <td>isescan</td>
      <td>To update</td>
      <td>ISEScan</td>
      <td>ISEScan</td>
      <td>Structural variation detection</td>
      <td>Genomics, DNA structural variation, Sequence analysis, Genetic variation</td>
      <td>"ISEScan is a pipeline to identify IS (Insertion Sequence) elements in genome and metagenomebased on profile hidden Markov models constructed from manually curated IS elements."</td>
      <td>Automated identification of insertion sequence elements in prokaryotic genomes.</td>
      <td>To update</td>
      <td>https://github.com/xiezhq/ISEScan</td>
      <td>Sequence Analysis</td>
      <td>ISEScan</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/isescan</td>
    </tr>
    <tr>
      <td></td>
      <td>isoformswitchanalyzer</td>
      <td>1.20.0</td>
      <td>2.2.0</td>
      <td>bioconductor-isoformswitchanalyzer</td>
      <td>To update</td>
      <td>IsoformSwitchAnalyzeR</td>
      <td>IsoformSwitchAnalyzeR</td>
      <td>Sequence comparison, Sequence analysis</td>
      <td>Computational biology, Gene transcripts</td>
      <td>Statistical identification of isoform switching from RNA-seq derived quantification of novel and/or annotated full-length isoforms.</td>
      <td>Enables identification of isoform switches with predicted functional consequences from RNA-seq data. Consequences can be chosen from a long list but includes protein domains gain/loss changes in NMD sensitivity etc. It directly supports import of data from Cufflinks/Cuffdiff, Kallisto, Salmon and RSEM but other transcript qunatification tools are easy to import as well.</td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/devel/bioc/html/IsoformSwitchAnalyzeR.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>isoformswitchanalyzer</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/isoformswitchanalyzer</td>
    </tr>
    <tr>
      <td></td>
      <td>jasminesv</td>
      <td>1.0.11</td>
      <td>1.1.5</td>
      <td>jasminesv</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Merge structural variants across samples</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/mkirsche/Jasmine/</td>
      <td>Variant Analysis</td>
      <td>jasminesv</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/jasminesv/</td>
    </tr>
    <tr>
      <td></td>
      <td>jbrowse</td>
      <td>1.16.11</td>
      <td>1.16.11</td>
      <td>jbrowse</td>
      <td>Up-to-date</td>
      <td>jbrowse</td>
      <td>JBrowse</td>
      <td>Genome visualisation</td>
      <td>Genomics</td>
      <td>JBrowse Genome Browser integrated as a Galaxy Tool</td>
      <td>Slick, speedy genome browser with a responsive and dynamic AJAX interface for visualization of genome data. Being developed by the GMOD project as a successor to GBrowse.</td>
      <td>Up-to-date</td>
      <td>https://jbrowse.org</td>
      <td>Sequence Analysis</td>
      <td>jbrowse</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/jbrowse</td>
    </tr>
    <tr>
      <td></td>
      <td>jcvi_gff_stats</td>
      <td>0.8.4</td>
      <td>1.3.9</td>
      <td>jcvi</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute statistics from a genome annotation in GFF3 format (using JCVI Python utilities)</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/tanghaibao/jcvi</td>
      <td>Sequence Analysis</td>
      <td>jcvi_gff_stats</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/jcvi_gff_stats</td>
    </tr>
    <tr>
      <td></td>
      <td>jellyfish</td>
      <td></td>
      <td>2.3.0</td>
      <td>kmer-jellyfish</td>
      <td>To update</td>
      <td>Jellyfish</td>
      <td>Jellyfish</td>
      <td>k-mer counting</td>
      <td>Sequence analysis, Genomics</td>
      <td>Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA</td>
      <td>A command-line algorithm for counting k-mers in DNA sequence.</td>
      <td>To update</td>
      <td>https://github.com/gmarcais/Jellyfish</td>
      <td>Assembly</td>
      <td>jellyfish</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/jellyfish</td>
    </tr>
    <tr>
      <td></td>
      <td>jq</td>
      <td>1.0</td>
      <td>1.5</td>
      <td>jq</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>JQ is a lightweight and flexible command-line JSON processor</td>
      <td></td>
      <td>To update</td>
      <td>https://stedolan.github.io/jq/</td>
      <td>Text Manipulation</td>
      <td>jq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/jq</td>
    </tr>
    <tr>
      <td></td>
      <td>kc-align</td>
      <td>1.0.2</td>
      <td>1.0.2</td>
      <td>kcalign</td>
      <td>Up-to-date</td>
      <td>kc-align</td>
      <td>kc-align</td>
      <td>Multiple sequence alignment</td>
      <td>Mapping</td>
      <td>Kc-Align custom tool</td>
      <td>A fast and accurate tool for performing codon-aware multiple sequence alignments</td>
      <td>Up-to-date</td>
      <td>https://github.com/davebx/kc-align</td>
      <td>Sequence Analysis</td>
      <td>kc_align</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/kc-align</td>
    </tr>
    <tr>
      <td></td>
      <td>king</td>
      <td>2.2.7</td>
      <td>2.2.7</td>
      <td>king</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Kinship-based INference for Gwas</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://people.virginia.edu/~wc9c/KING/</td>
      <td>Variant Analysis</td>
      <td>king</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/king/</td>
    </tr>
    <tr>
      <td></td>
      <td>kleborate</td>
      <td>2.3.2</td>
      <td>2.3.2</td>
      <td>kleborate</td>
      <td>Up-to-date</td>
      <td>kleborate</td>
      <td>Kleborate</td>
      <td>Multilocus sequence typing, Genome assembly, Virulence prediction</td>
      <td>Public health and epidemiology, Metagenomics, Population genomics, Sequence assembly, Whole genome sequencing</td>
      <td>Screen genome assemblies of Klebsiella pneumoniae and the Klebsiella pneumoniae species complex (KpSC)</td>
      <td>Genomic surveillance framework and global population structure for Klebsiella pneumoniae.Kleborate is a tool to screen genome assemblies of Klebsiella pneumoniae and the Klebsiella pneumoniae species complex (KpSC) for:.A manuscript describing the Kleborate software in full is currently in preparation. In the meantime, if you use Kleborate, please cite the preprint: Lam, MMC. et al. Genomic surveillance framework and global population structure for Klebsiella pneumoniae. bioRxiv (2020).</td>
      <td>Up-to-date</td>
      <td>https://github.com/katholt/Kleborate/wiki</td>
      <td>Metagenomics</td>
      <td>kleborate</td>
      <td>iuc</td>
      <td>https://github.com/katholt/Kleborate</td>
    </tr>
    <tr>
      <td></td>
      <td>kofamscan</td>
      <td>1.3.0</td>
      <td>1.3.0</td>
      <td>kofamscan</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Gene function annotation tool based on KEGG Orthology and hidden Markov model</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/takaram/kofam_scan</td>
      <td>Sequence Analysis</td>
      <td>kofamscan</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/kofamscan</td>
    </tr>
    <tr>
      <td></td>
      <td>kraken</td>
      <td></td>
      <td>1.1.1</td>
      <td>kraken</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Kraken is a system for assigning taxonomic labels to short DNAsequences, usually obtained through metagenomic studies. Previous attempts by otherbioinformatics software to accomplish this task have often used sequence alignmentor machine learning techniques that were quite slow, leading to the developmentof less sensitive but much faster abundance estimation programs. Kraken aims toachieve high sensitivity and high speed by utilizing exact alignments of k-mersand a novel classification algorithm.</td>
      <td></td>
      <td>To update</td>
      <td>http://ccb.jhu.edu/software/kraken/</td>
      <td>Metagenomics</td>
      <td>kraken</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tool_collections/kraken/</td>
    </tr>
    <tr>
      <td></td>
      <td>kraken2</td>
      <td>2.1.1</td>
      <td>2.1.3</td>
      <td>kraken2</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Kraken2 for taxonomic designation.</td>
      <td></td>
      <td>To update</td>
      <td>http://ccb.jhu.edu/software/kraken/</td>
      <td>Metagenomics</td>
      <td>kraken2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tool_collections/kraken2/kraken2/</td>
    </tr>
    <tr>
      <td></td>
      <td>kraken_biom</td>
      <td>1.2.0</td>
      <td>1.2.0</td>
      <td>kraken-biom</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Create BIOM-format tables (http://biom-format.org) from Kraken output (http://ccb.jhu.edu/software/kraken/)</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/smdabdoub/kraken-biom</td>
      <td>Metagenomics</td>
      <td>kraken_biom</td>
      <td>iuc</td>
      <td>https://github.com/smdabdoub/kraken-biom</td>
    </tr>
    <tr>
      <td></td>
      <td>kraken_taxonomy_report</td>
      <td>0.0.3</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Kraken taxonomy report</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/blankenberg/Kraken-Taxonomy-Report</td>
      <td>Metagenomics</td>
      <td>kraken_taxonomy_report</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/kraken_taxonomy_report</td>
    </tr>
    <tr>
      <td></td>
      <td>legsta</td>
      <td>0.5.1</td>
      <td>0.5.1</td>
      <td>legsta</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Performs in silico Legionella pneumophila sequence based typing.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/tseemann/legsta</td>
      <td>Sequence Analysis</td>
      <td>legsta</td>
      <td>iuc</td>
      <td>https://github.com/tseemann/legsta</td>
    </tr>
    <tr>
      <td></td>
      <td>length_and_gc_content</td>
      <td>0.1.2</td>
      <td>1.3.2</td>
      <td>r-optparse</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Gets gene length and gc content from a fasta and a GTF file</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/length_and_gc_content</td>
      <td>Fasta Manipulation, Statistics, RNA, Micro-array Analysis</td>
      <td>length_and_gc_content</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/length_and_gc_content</td>
    </tr>
    <tr>
      <td></td>
      <td>limma_voom</td>
      <td>3.50.1</td>
      <td>3.58.1</td>
      <td>bioconductor-limma</td>
      <td>To update</td>
      <td>limma</td>
      <td>limma</td>
      <td>RNA-Seq analysis</td>
      <td>Molecular biology, Genetics</td>
      <td>Perform RNA-Seq differential expression analysis using limma voom pipeline</td>
      <td>Data analysis, linear models and differential expression for microarray data.</td>
      <td>To update</td>
      <td>http://bioconductor.org/packages/release/bioc/html/limma.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>limma_voom</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/limma_voom</td>
    </tr>
    <tr>
      <td></td>
      <td>lineagespot</td>
      <td>1.4.0</td>
      <td></td>
      <td>r-base</td>
      <td>To update</td>
      <td>lineagespot</td>
      <td>lineagespot</td>
      <td>Variant calling</td>
      <td>Metagenomics, Gene transcripts, Evolutionary biology, Sequencing, Genetic variation</td>
      <td>Identification of SARS-CoV-2 related metagenomic mutations based on a single (or a list of) variant(s) file(s)</td>
      <td>Lineagespot is a framework written in R, and aims to identify and assign different SARS-CoV-2 lineages based on a single variant file (i.e., variant calling format).</td>
      <td>To update</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/lineagespot.html</td>
      <td>Metagenomics, Sequence Analysis</td>
      <td>lineagespot</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/lineagespot</td>
    </tr>
    <tr>
      <td></td>
      <td>links</td>
      <td>2.0.1</td>
      <td>2.0.1</td>
      <td>links</td>
      <td>Up-to-date</td>
      <td>links</td>
      <td>LINKS</td>
      <td>Scaffolding, Genome assembly, Read mapping, Read pre-processing, Sequence trimming</td>
      <td>Sequence assembly, Mapping, Sequencing</td>
      <td>Scaffold genome assemblies with long reads.</td>
      <td>LINKS (Long Interval Nucleotide K-mer Scaffolder) is a genomics application for scaffolding genome assemblies with long reads, such as those produced by Oxford Nanopore Technologies Ltd. It can be used to scaffold high-quality draft genome assemblies with any long sequences (eg. ONT reads, PacBio reads, other draft genomes, etc). It is also used to scaffold contig pairs linked by ARCS/ARKS.</td>
      <td>Up-to-date</td>
      <td>https://github.com/bcgsc/LINKS</td>
      <td>Assembly</td>
      <td>links</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/links</td>
    </tr>
    <tr>
      <td></td>
      <td>lumpy_sv</td>
      <td>0.3.1</td>
      <td>0.3.1</td>
      <td>lumpy-sv</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>LUMPY - a general probabilistic framework for structural variant discovery</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://layerlab.org/software/</td>
      <td>Variant Analysis</td>
      <td>lumpy_sv</td>
      <td>iuc</td>
      <td>https://github.com/arq5x/lumpy-sv</td>
    </tr>
    <tr>
      <td></td>
      <td>maaslin2</td>
      <td>0.99.12</td>
      <td>0.99.12</td>
      <td>maaslin2</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MaAsLin2 is comprehensive R package for efficiently determining multivariable association between microbial meta'omic features and clinical metadata.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://huttenhower.sph.harvard.edu/maaslin</td>
      <td>Metagenomics</td>
      <td>maaslin2</td>
      <td>iuc</td>
      <td>https://github.com/biobakery/Maaslin2</td>
    </tr>
    <tr>
      <td></td>
      <td>maker</td>
      <td>2.31.11</td>
      <td>3.01.03</td>
      <td>maker</td>
      <td>To update</td>
      <td>maker</td>
      <td>MAKER</td>
      <td>Genome annotation</td>
      <td>Genomics, DNA, Sequence analysis</td>
      <td>MAKER is a portable and easily configurable genome annotation pipeline.Its purpose is to allow smaller eukaryotic and prokaryotic genome projects to independently annotate their genomes and to create genome databases.</td>
      <td>Portable and easily configurable genome annotation pipeline. It’s purpose is to allow smaller eukaryotic and prokaryotic genome projects to independently annotate their genomes and to create genome databases.</td>
      <td>To update</td>
      <td>http://www.yandell-lab.org/software/maker.html</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/maker</td>
    </tr>
    <tr>
      <td></td>
      <td>map_param_value</td>
      <td>0.2.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Map a parameter value to another value</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>map_param_value</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/map_param_value</td>
    </tr>
    <tr>
      <td></td>
      <td>mapseq</td>
      <td>2.1.1</td>
      <td></td>
      <td>perl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>fast and accurate sequence read classification tool designed to assign taxonomy and OTU classifications to ribosomal RNA sequences.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/jfmrod/MAPseq</td>
      <td>Metagenomics</td>
      <td>mapseq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mapseq</td>
    </tr>
    <tr>
      <td></td>
      <td>masigpro</td>
      <td>1.49.3</td>
      <td>8.25</td>
      <td>coreutils</td>
      <td>To update</td>
      <td>masigpro</td>
      <td>maSigPro</td>
      <td>Regression analysis</td>
      <td>Gene expression, Molecular genetics, Microarray experiment, RNA-Seq</td>
      <td>Identify significantly differential expression profiles in time-course microarray experiments</td>
      <td>Regression based approach to find genes for which there are significant gene expression profile differences between experimental groups in time course microarray and RNA-Seq experiments.</td>
      <td>To update</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/maSigPro.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>masigpro</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/masigpro</td>
    </tr>
    <tr>
      <td></td>
      <td>maxbin2</td>
      <td></td>
      <td>2.2.7</td>
      <td>maxbin2</td>
      <td>To update</td>
      <td>maxbin</td>
      <td>MaxBin</td>
      <td>Sequence assembly</td>
      <td>Metagenomics, Sequence assembly, Microbiology</td>
      <td>clusters metagenomic contigs into bins</td>
      <td>Software for binning assembled metagenomic sequences based on an Expectation-Maximization algorithm.</td>
      <td>To update</td>
      <td>https://downloads.jbei.org/data/microbial_communities/MaxBin/MaxBin.html</td>
      <td>Metagenomics</td>
      <td>maxbin2</td>
      <td>mbernt</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/maxbin2/</td>
    </tr>
    <tr>
      <td></td>
      <td>mcl</td>
      <td>22.282</td>
      <td>22.282</td>
      <td>mcl</td>
      <td>Up-to-date</td>
      <td>mcl</td>
      <td>MCL</td>
      <td>Clustering, Network analysis, Gene regulatory network analysis</td>
      <td>Molecular interactions, pathways and networks</td>
      <td>The Markov Cluster Algorithm, a cluster algorithm for graphs</td>
      <td>MCL is a clustering algorithm widely used in bioinformatics and gaining traction in other fields.</td>
      <td>Up-to-date</td>
      <td>https://micans.org/mcl/man/mcl.html</td>
      <td>Sequence Analysis</td>
      <td>mcl</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/mcl</td>
    </tr>
    <tr>
      <td></td>
      <td>megahit</td>
      <td>1.2.9</td>
      <td>1.2.9</td>
      <td>megahit</td>
      <td>Up-to-date</td>
      <td>megahit</td>
      <td>MEGAHIT</td>
      <td>Genome assembly</td>
      <td>Metagenomics, Sequencing, Ecology, Sequence assembly</td>
      <td>An ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph.</td>
      <td>Single node assembler for large and complex metagenomics NGS reads, such as soil. It makes use of succinct de Bruijn graph to achieve low memory usage, whereas its goal is not to make memory usage as low as possible.</td>
      <td>Up-to-date</td>
      <td>https://github.com/voutcn/megahit</td>
      <td>Sequence Analysis, Assembly, Metagenomics</td>
      <td>megahit</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/megahit</td>
    </tr>
    <tr>
      <td></td>
      <td>megahit_contig2fastg</td>
      <td>1.1.3</td>
      <td>1.2.9</td>
      <td>megahit</td>
      <td>To update</td>
      <td>megahit</td>
      <td>MEGAHIT</td>
      <td>Genome assembly</td>
      <td>Metagenomics, Sequencing, Ecology, Sequence assembly</td>
      <td>A subprogram within the Megahit toolkit for converting contigs to assembly graphs (fastg)</td>
      <td>Single node assembler for large and complex metagenomics NGS reads, such as soil. It makes use of succinct de Bruijn graph to achieve low memory usage, whereas its goal is not to make memory usage as low as possible.</td>
      <td>To update</td>
      <td>https://github.com/voutcn/megahit/blob/master/tools/toolkit.cpp</td>
      <td>Sequence Analysis, Assembly, Metagenomics</td>
      <td>megahit_contig2fastg</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/megahit_contig2fastg</td>
    </tr>
    <tr>
      <td></td>
      <td>meme_chip</td>
      <td>4.11.2</td>
      <td>1.3.26</td>
      <td>graphicsmagick</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Performs motif discovery, motif enrichment analysis and clustering on large nucleotide datasets.</td>
      <td></td>
      <td>To update</td>
      <td>http://meme-suite.org/</td>
      <td>ChIP-seq</td>
      <td>meme_chip</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/meme_chip</td>
    </tr>
    <tr>
      <td></td>
      <td>merqury</td>
      <td>1.3</td>
      <td>1.3</td>
      <td>merqury</td>
      <td>Up-to-date</td>
      <td>merqury</td>
      <td>Merqury</td>
      <td>Genome assembly, k-mer counting, Scaffolding, Phasing, De-novo assembly</td>
      <td>Sequence assembly, Whole genome sequencing, Plant biology</td>
      <td>Merqury is a tool for evaluating genomes assemblies based of k-mer operations.</td>
      <td>Reference-free quality, completeness, and phasing assessment for genome assemblies.Evaluate genome assemblies with k-mers and more.Often, genome assembly projects have illumina whole genome sequencing reads available for the assembled individual.Merqury provides a set of tools for this purpose.</td>
      <td>Up-to-date</td>
      <td>https://github.com/marbl/merqury</td>
      <td>Assembly</td>
      <td>merqury</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/merqury</td>
    </tr>
    <tr>
      <td></td>
      <td>meryl</td>
      <td>1.3</td>
      <td>1.3</td>
      <td>merqury</td>
      <td>Up-to-date</td>
      <td>meryl</td>
      <td>Meryl</td>
      <td>k-mer counting</td>
      <td>Whole genome sequencing, Genomics, Sequence analysis, Sequencing</td>
      <td>Meryl a k-mer counter.</td>
      <td>Meryl is a tool for counting and working with sets of k-mers that was originally developed for use in the Celera Assembler and has since been migrated and maintained as part of Canu.</td>
      <td>Up-to-date</td>
      <td>https://github.com/marbl/meryl</td>
      <td>Assembly</td>
      <td>meryl</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/meryl</td>
    </tr>
    <tr>
      <td></td>
      <td>metabat2</td>
      <td>2.15</td>
      <td>2.15</td>
      <td>metabat2</td>
      <td>Up-to-date</td>
      <td>MetaBAT_2</td>
      <td>MetaBAT 2</td>
      <td>Read binning, Sequence assembly, Genome annotation</td>
      <td>Metagenomics, Sequence assembly, Metagenomic sequencing</td>
      <td>MetaBAT2 (Metagenome Binning based on Abundance and Tetranucleotide frequency) is an automated metagenome binningsoftware that integrates empirical probabilistic distances of genome abundance and tetranucleotide frequency.</td>
      <td>an adaptive binning algorithm for robust and efficient genome reconstruction from metagenome assemblies | MetaBAT2 clusters metagenomic contigs into different "bins", each of which should correspond to a putative genome | MetaBAT2 uses nucleotide composition information and source strain abundance (measured by depth-of-coverage by aligning the reads to the contigs) to perform binning</td>
      <td>Up-to-date</td>
      <td>https://bitbucket.org/berkeleylab/metabat/src/master/</td>
      <td>Metagenomics</td>
      <td>metabat2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/metabat2/</td>
    </tr>
    <tr>
      <td></td>
      <td>metaphlan</td>
      <td>4.0.6</td>
      <td>4.0.6</td>
      <td>metaphlan</td>
      <td>Up-to-date</td>
      <td>metaphlan</td>
      <td>MetaPhlAn</td>
      <td>Nucleic acid sequence analysis, Phylogenetic tree analysis</td>
      <td>Metagenomics, Phylogenomics</td>
      <td>MetaPhlAn for Metagenomic Phylogenetic Analysis</td>
      <td>Computational tool for profiling the composition of microbial communities from metagenomic shotgun sequencing data.</td>
      <td>Up-to-date</td>
      <td>https://github.com/biobakery/MetaPhlAn</td>
      <td>Metagenomics</td>
      <td>metaphlan</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/metaphlan/</td>
    </tr>
    <tr>
      <td></td>
      <td>migmap</td>
      <td>1.0.3</td>
      <td>1.0.3</td>
      <td>migmap</td>
      <td>Up-to-date</td>
      <td>MiGMAP</td>
      <td>MiGMAP</td>
      <td>Sequence analysis, Read mapping</td>
      <td>Immunoproteins, genes and antigens, Sequence analysis</td>
      <td>mapper for full-length T- and B-cell repertoire sequencing</td>
      <td>Mapper for full-length T- and B-cell repertoire sequencing.</td>
      <td>Up-to-date</td>
      <td>https://github.com/mikessh/migmap</td>
      <td>RNA, Sequence Analysis</td>
      <td>migmap</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/migmap</td>
    </tr>
    <tr>
      <td></td>
      <td>minia</td>
      <td>3.2.6</td>
      <td>3.2.6</td>
      <td>minia</td>
      <td>Up-to-date</td>
      <td>minia</td>
      <td>Minia</td>
      <td>Genome assembly</td>
      <td>Sequence assembly</td>
      <td>Short-read assembler based on a de Bruijn graph</td>
      <td>Short-read assembler based on a de Bruijn graph, capable of assembling a human genome on a desktop computer in a day.</td>
      <td>Up-to-date</td>
      <td>https://gatb.inria.fr/software/minia/</td>
      <td>Assembly</td>
      <td>minia</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/minia</td>
    </tr>
    <tr>
      <td></td>
      <td>miniasm</td>
      <td>0.3_r179</td>
      <td>0.3</td>
      <td>miniasm</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Miniasm - Ultrafast de novo assembly for long noisy reads (though having no consensus step)</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/lh3/miniasm</td>
      <td>Assembly</td>
      <td>miniasm</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/miniasm</td>
    </tr>
    <tr>
      <td></td>
      <td>minimap2</td>
      <td>2.26</td>
      <td>2.26</td>
      <td>minimap2</td>
      <td>Up-to-date</td>
      <td>minimap2</td>
      <td>Minimap2</td>
      <td>Pairwise sequence alignment</td>
      <td>Mapping</td>
      <td>A fast pairwise aligner for genomic and spliced nucleotide sequences</td>
      <td>Pairwise aligner for genomic and spliced nucleotide sequences</td>
      <td>Up-to-date</td>
      <td>https://github.com/lh3/minimap2</td>
      <td>Next Gen Mappers</td>
      <td>minimap2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/minimap2</td>
    </tr>
    <tr>
      <td></td>
      <td>miniprot</td>
      <td>0.12</td>
      <td>0.12</td>
      <td>miniprot</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Align a protein sequence against a genome with affine gap penalty, splicing and frameshift.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/lh3/miniprot</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/miniprot</td>
    </tr>
    <tr>
      <td></td>
      <td>mirnature</td>
      <td>1.1</td>
      <td>1.1</td>
      <td>mirnature</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Computational detection of canonical microRNAs</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/Bierinformatik/miRNAture</td>
      <td>RNA, Sequence Analysis</td>
      <td>mirnature</td>
      <td>iuc</td>
      <td>https://github.com/Bierinformatik/miRNAture</td>
    </tr>
    <tr>
      <td></td>
      <td>mitobim</td>
      <td>1.9.1</td>
      <td>1.9.1</td>
      <td>mitobim</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>assemble mitochondrial genomes</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/chrishah/MITObim</td>
      <td>Assembly</td>
      <td>mitobim</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mitobim</td>
    </tr>
    <tr>
      <td></td>
      <td>mitos</td>
      <td>1.1.1</td>
      <td>2.1.6</td>
      <td>mitos</td>
      <td>To update</td>
      <td>mitos</td>
      <td>MITOS</td>
      <td>Genome annotation</td>
      <td>Zoology, Whole genome sequencing</td>
      <td>de-novo annotation of metazoan mitochondrial genomes</td>
      <td>De novo metazoan mitochondrial genome annotation.</td>
      <td>To update</td>
      <td>http://mitos.bioinf.uni-leipzig.de/</td>
      <td>Sequence Analysis</td>
      <td>mitos</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mitos</td>
    </tr>
    <tr>
      <td></td>
      <td>mlst</td>
      <td>2.22.0</td>
      <td>2.23.0</td>
      <td>mlst</td>
      <td>To update</td>
      <td>mlst</td>
      <td>MLST</td>
      <td>Taxonomic classification</td>
      <td>Immunoproteins, genes and antigens</td>
      <td>Scan contig files against PubMLST typing schemes</td>
      <td>Multi Locus Sequence Typing from an assembled genome or from a set of reads.</td>
      <td>To update</td>
      <td>https://github.com/tseemann/mlst</td>
      <td>Sequence Analysis</td>
      <td>mlst</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mlst</td>
    </tr>
    <tr>
      <td></td>
      <td>moabs</td>
      <td>1.3.4.6</td>
      <td>1.3.9.0</td>
      <td>moabs</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MOABS for differential methylation analysis on Bisulfite sequencing data.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/sunnyisgalaxy/moabs</td>
      <td>Epigenetics</td>
      <td>moabs</td>
      <td>iuc</td>
      <td>https://github.com/sunnyisgalaxy/moabs</td>
    </tr>
    <tr>
      <td></td>
      <td>mosdepth</td>
      <td>0.3.5</td>
      <td>0.3.6</td>
      <td>mosdepth</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>fast and flexible BAM/CRAM depth calculation</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/brentp/mosdepth</td>
      <td>SAM</td>
      <td>mosdepth</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mosdepth</td>
    </tr>
    <tr>
      <td></td>
      <td>msaboot</td>
      <td>0.1.2</td>
      <td>0.1.2</td>
      <td>msaboot</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A multiple sequences alignment bootstrapping tool.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/phac-nml/msaboot</td>
      <td>Fasta Manipulation</td>
      <td>msaboot</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/msaboot</td>
    </tr>
    <tr>
      <td></td>
      <td>multigsea</td>
      <td>1.8.0</td>
      <td>1.12.0</td>
      <td>bioconductor-multigsea</td>
      <td>To update</td>
      <td>multiGSEA</td>
      <td>multiGSEA</td>
      <td>Gene-set enrichment analysis, Aggregation, Pathway analysis</td>
      <td>Metabolomics, Molecular interactions, pathways and networks, Proteomics, Transcriptomics, Small molecules</td>
      <td>GSEA-based pathway enrichment analysis for multi-omics data</td>
      <td>A GSEA-based pathway enrichment analysis for multi-omics data.multiGSEA: a GSEA-based pathway enrichment analysis for multi-omics data, BMC Bioinformatics 21, 561 (2020).Combining GSEA-based pathway enrichment with multi omics data integration.</td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/devel/bioc/html/multiGSEA.html</td>
      <td>Transcriptomics, Proteomics, Statistics</td>
      <td>multigsea</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/multigsea</td>
    </tr>
    <tr>
      <td></td>
      <td>multiqc</td>
      <td>1.11</td>
      <td>1.19</td>
      <td>multiqc</td>
      <td>To update</td>
      <td>multiqc</td>
      <td>MultiQC</td>
      <td>Validation</td>
      <td>Sequencing, Bioinformatics</td>
      <td>MultiQC aggregates results from bioinformatics analyses across many samples into a single report</td>
      <td>MultiQC aggregates results from multiple bioinformatics analyses across many samples into a single report. It searches a given directory for analysis logs and compiles a HTML report. It's a general use tool, perfect for summarising the output from numerous bioinformatics tools.</td>
      <td>To update</td>
      <td>http://multiqc.info/</td>
      <td>Fastq Manipulation, Statistics, Visualization</td>
      <td>multiqc</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/multiqc</td>
    </tr>
    <tr>
      <td></td>
      <td>nanoplot</td>
      <td>1.41.0</td>
      <td>1.42.0</td>
      <td>nanoplot</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Plotting tool for long read sequencing data and alignments</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/wdecoster/NanoPlot</td>
      <td>Visualization</td>
      <td>nanoplot</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/nanoplot/</td>
    </tr>
    <tr>
      <td></td>
      <td>ncbi_acc_download</td>
      <td>0.2.8</td>
      <td>0.2.8</td>
      <td>ncbi-acc-download</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Download sequences from GenBank/RefSeq by accession through the NCBI ENTREZ API</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/kblin/ncbi-acc-download</td>
      <td>Data Source</td>
      <td>ncbi_acc_download</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ncbi_acc_download</td>
    </tr>
    <tr>
      <td></td>
      <td>necat</td>
      <td>0.0.1_update20200803</td>
      <td>0.0.1_update20200803</td>
      <td>necat</td>
      <td>Up-to-date</td>
      <td>necat</td>
      <td>NECAT</td>
      <td>De-novo assembly</td>
      <td>Sequence assembly</td>
      <td>Error correction and de-novo assembly for ONT Nanopore reads</td>
      <td>NECAT is an error correction and de-novo assembly tool for Nanopore long noisy reads.</td>
      <td>Up-to-date</td>
      <td>https://github.com/xiaochuanle/NECAT</td>
      <td>Assembly</td>
      <td>necat</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/necat</td>
    </tr>
    <tr>
      <td></td>
      <td>nextclade</td>
      <td>2.7.0</td>
      <td>2.14.0</td>
      <td>nextalign</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Identify differences between your sequences and a reference sequence used by Nextstrain</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/nextstrain/nextclade</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/nextclade</td>
    </tr>
    <tr>
      <td></td>
      <td>ngmlr</td>
      <td>0.2.7</td>
      <td>0.2.7</td>
      <td>ngmlr</td>
      <td>Up-to-date</td>
      <td>ngmlr</td>
      <td>NGMLR</td>
      <td>DNA mapping, Sequence alignment, Genetic variation analysis</td>
      <td>Sequencing, Mapping, DNA structural variation</td>
      <td>CoNvex Gap-cost alignMents for Long Reads</td>
      <td>An algorithm to map third generation long-read sequencing data (PacBio and Oxford Nanopore) to a reference genome with a focus on reads that span structural variation.</td>
      <td>Up-to-date</td>
      <td>https://github.com/philres/ngmlr</td>
      <td>Next Gen Mappers</td>
      <td>ngmlr</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ngmlr</td>
    </tr>
    <tr>
      <td></td>
      <td>nonpareil</td>
      <td>3.1.1</td>
      <td>3.4.1</td>
      <td>nonpareil</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Estimate average coverage in metagenomic datasets</td>
      <td></td>
      <td>To update</td>
      <td>http://nonpareil.readthedocs.io</td>
      <td>Metagenomics</td>
      <td>nonpareil</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/nonpareil</td>
    </tr>
    <tr>
      <td></td>
      <td>novoplasty</td>
      <td>4.3.1</td>
      <td>4.3.3</td>
      <td>novoplasty</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>NOVOPlasty is a de novo assembler and heteroplasmy/variance caller for short circular genomes.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ndierckx/NOVOPlasty</td>
      <td>Assembly</td>
      <td>novoplasty</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/novoplasty</td>
    </tr>
    <tr>
      <td></td>
      <td>ococo</td>
      <td>0.1.2.6</td>
      <td>0.1.2.6</td>
      <td>ococo</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Variant detection of SNVs</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/karel-brinda/ococo</td>
      <td>Variant Analysis</td>
      <td>ococo</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ococo</td>
    </tr>
    <tr>
      <td></td>
      <td>optitype</td>
      <td>1.3.5</td>
      <td>1.3.5</td>
      <td>optitype</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Precision HLA typing from NGS data</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/FRED-2/OptiType</td>
      <td>Sequence Analysis</td>
      <td>optitype</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/optitype1</td>
    </tr>
    <tr>
      <td></td>
      <td>orfipy</td>
      <td>0.0.4</td>
      <td>0.0.4</td>
      <td>orfipy</td>
      <td>Up-to-date</td>
      <td>orfipy</td>
      <td>orfipy</td>
      <td>Coding region prediction, Database search, Transcriptome assembly, De-novo assembly</td>
      <td>Computer science, RNA-Seq, Transcriptomics, Small molecules</td>
      <td>Galaxy wrapper for ORFIPY</td>
      <td>A fast and flexible tool for extracting ORFs.orfipy is a tool written in python/cython to extract ORFs in extremely an fast and flexible manner. Other popular ORF searching tools are OrfM and getorf. Compared to OrfM and getorf, orfipy provides the most options to fine tune ORF searches. orfipy uses multiple CPU cores and is particularly faster for data containing multiple smaller fasta sequences such as de-novo transcriptome assemblies. Please read the preprint here.</td>
      <td>Up-to-date</td>
      <td>https://github.com/urmi-21/orfipy</td>
      <td>Sequence Analysis</td>
      <td>orfipy</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/orfipy</td>
    </tr>
    <tr>
      <td></td>
      <td>pangolin</td>
      <td>4.3</td>
      <td>4.3</td>
      <td>pangolin</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Pangolin assigns SARS-CoV-2 genome sequences their most likely lineages under the Pango nomenclature system.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/cov-lineages/pangolin</td>
      <td>Sequence Analysis</td>
      <td>pangolin</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pangolin</td>
    </tr>
    <tr>
      <td></td>
      <td>parse_mito_blast</td>
      <td>1.0.2</td>
      <td>1.0.2</td>
      <td>parse_mito_blast</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filtering blast out from querying assembly against mitochondrial database.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://raw.githubusercontent.com/VGP/vgp-assembly/master/galaxy_tools/parse_mito_blast/parse_mito_blast.py</td>
      <td>Sequence Analysis</td>
      <td>parse_mito_blast</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/parse_mito_blast</td>
    </tr>
    <tr>
      <td></td>
      <td>pathview</td>
      <td>1.34.0</td>
      <td>1.42.0</td>
      <td>bioconductor-pathview</td>
      <td>To update</td>
      <td>pathview</td>
      <td>pathview</td>
      <td>Pathway or network analysis, Pathway or network visualisation</td>
      <td>Molecular interactions, pathways and networks, Systems biology, Data visualisation</td>
      <td>Pathview is a tool set for pathway based data integration and visualization.</td>
      <td>Tool set for pathway based data integration and visualization that maps and renders a wide variety of biological data on relevant pathway graphs. It downloads the pathway graph data, parses the data file, maps user data to the pathway, and render pathway graph with the mapped data. In addition, it integrates with pathway and gene set (enrichment) analysis tools for large-scale and fully automated analysis.</td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/pathview.html</td>
      <td>Statistics, RNA, Micro-array Analysis</td>
      <td>pathview</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pathview</td>
    </tr>
    <tr>
      <td></td>
      <td>pe_histogram</td>
      <td>1.0.1</td>
      <td></td>
      <td>openjdk</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Contains a tool that produces an insert size histogram for a paired-end BAM file.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/seqcode/cegr-tools/tree/master/src/org/seqcode/cegrtools/pehistogram</td>
      <td>Graphics</td>
      <td>pe_histogram</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pe_histogram</td>
    </tr>
    <tr>
      <td></td>
      <td>pharokka</td>
      <td>1.3.2</td>
      <td></td>
      <td>\n                pharokka\n</td>
      <td>To update</td>
      <td>\n                pharokka\n</td>
      <td></td>
      <td></td>
      <td></td>
      <td>rapid standardised annotation tool for bacteriophage genomes and metagenomes</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/gbouras13/pharokka</td>
      <td>Genome annotation</td>
      <td>pharokka</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pharokka</td>
    </tr>
    <tr>
      <td></td>
      <td>phyml</td>
      <td>3.3.20220408</td>
      <td>3.3.20220408</td>
      <td>phyml</td>
      <td>Up-to-date</td>
      <td>phyml</td>
      <td>PhyML</td>
      <td>Phylogenetic tree generation (maximum likelihood and Bayesian methods)</td>
      <td>Phylogenetics, Bioinformatics, Phylogenetics</td>
      <td>PhyML is a phylogeny software based on the maximum-likelihood principle.</td>
      <td>Phylogenetic estimation software using Maximum Likelihood</td>
      <td>Up-to-date</td>
      <td>http://www.atgc-montpellier.fr/phyml/</td>
      <td>Phylogenetics</td>
      <td>phyml</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/phyml</td>
    </tr>
    <tr>
      <td></td>
      <td>pick_value</td>
      <td>0.1.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compose a text parameter value using text, integer and float values</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>pick_value</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pick_value</td>
    </tr>
    <tr>
      <td></td>
      <td>pilon</td>
      <td>1.20.1</td>
      <td>1.24</td>
      <td>pilon</td>
      <td>To update</td>
      <td>pilon</td>
      <td>pilon</td>
      <td>Sequence assembly, Analysis, Read alignment</td>
      <td>Assembly</td>
      <td>pilon is a tool for assembly improvement and variant analysis in bacteria</td>
      <td>Read alignment analysis to diagnose, report, and automatically improve de novo genome assemblies.</td>
      <td>To update</td>
      <td></td>
      <td>Variant Analysis</td>
      <td>pilon</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pilon</td>
    </tr>
    <tr>
      <td></td>
      <td>pipelign</td>
      <td>0.2</td>
      <td>0.2</td>
      <td>pipelign</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Multipe sequence alignment</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/asmmhossain/pipelign/</td>
      <td>Next Gen Mappers</td>
      <td>pipelign</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pipelign</td>
    </tr>
    <tr>
      <td></td>
      <td>pizzly</td>
      <td>0.37.3.1</td>
      <td>0.37.3</td>
      <td>pizzly</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Pizzly is a program for detecting gene fusions from RNA-Seq data of cancer samples.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/pmelsted/pizzly/</td>
      <td>Transcriptomics</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pizzly/</td>
    </tr>
    <tr>
      <td></td>
      <td>plasmidfinder</td>
      <td>2.1.6</td>
      <td>2.1.6</td>
      <td>plasmidfinder</td>
      <td>Up-to-date</td>
      <td>PlasmidFinder</td>
      <td>PlasmidFinder</td>
      <td>Genome assembly, Scaffolding, Multilocus sequence typing</td>
      <td>Whole genome sequencing, Sequence assembly, Mapping, Probes and primers</td>
      <td>"PlasmidFinder provides the detection of replicons in the WGSand assigns the plasmids under study to lineages that trace backthe information to the existing knowledge on Inc groups and suggestspossible reference plasmids for each lineage"</td>
      <td>PlasmidFinder is a tool for the identification and typing of Plasmid Replicons in Whole-Genome Sequencing (WGS).</td>
      <td>Up-to-date</td>
      <td>https://bitbucket.org/genomicepidemiology/plasmidfinder/src/master/</td>
      <td>Sequence Analysis</td>
      <td>plasmidfinder</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/plasmidfinder</td>
    </tr>
    <tr>
      <td></td>
      <td>plink</td>
      <td>1.90b6.21</td>
      <td>1.90b6.21</td>
      <td>plink</td>
      <td>Up-to-date</td>
      <td>plink</td>
      <td>PLINK</td>
      <td>Genetic variation analysis</td>
      <td>GWAS study</td>
      <td>PLINK is a free, open-source whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner.</td>
      <td>Free, open-source whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner.</td>
      <td>Up-to-date</td>
      <td>https://www.cog-genomics.org/plink</td>
      <td>Genome-Wide Association Study</td>
      <td>plink</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/plink</td>
    </tr>
    <tr>
      <td></td>
      <td>polypolish</td>
      <td>0.5.0</td>
      <td>0.5.0</td>
      <td>polypolish</td>
      <td>Up-to-date</td>
      <td>Polypolish</td>
      <td>Polypolish</td>
      <td>Genome assembly, Read mapping, Mapping assembly, Sequencing error detection</td>
      <td>Sequence assembly, Sequence composition, complexity and repeats, Mapping</td>
      <td>"Polypolish is a tool for polishing genome assemblies with short reads.Polypolish uses SAM files where each read has been aligned to all possible locations (not just a single best location).This allows it to repair errors in repeat regions that other alignment-based polishers cannot fix."</td>
      <td>Polypolish is a tool for polishing genome assemblies with short reads. Unlike other tools in this category, Polypolish uses SAM files where each read has been aligned to all possible locations (not just a single best location). This allows it to repair errors in repeat regions that other alignment-based polishers cannot fix.</td>
      <td>Up-to-date</td>
      <td>https://github.com/rrwick/Polypolish</td>
      <td>Sequence Analysis</td>
      <td>polypolish</td>
      <td>iuc</td>
      <td>https://github.com/mesocentre-clermont-auvergne/galaxy-tools/tree/master/tools/polypolish</td>
    </tr>
    <tr>
      <td></td>
      <td>porechop</td>
      <td></td>
      <td>0.2.4</td>
      <td>porechop</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Porechop - Finding and removing adapters from Oxford Nanopore reads</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/rrwick/Porechop</td>
      <td>Fasta Manipulation, Fastq Manipulation</td>
      <td>porechop</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/porechop</td>
    </tr>
    <tr>
      <td></td>
      <td>prinseq</td>
      <td>@TOOL_VERSION+galaxy2</td>
      <td>0.20.4</td>
      <td>prinseq</td>
      <td>To update</td>
      <td>prinseq</td>
      <td>PRINSEQ</td>
      <td>Read pre-processing, Sequence trimming, Sequence contamination filtering</td>
      <td>Transcriptomics, Metagenomics, Genomics</td>
      <td>PRINSEQ is a tool for easy and rapid quality control and data processing of metagenomic and metatranscriptomic datasets</td>
      <td>PRINSEQ is a sequence processing tool that can be used to filter, reformat and trim genomic and metagenomic sequence data. It generates summary statistics of the input in graphical and tabular formats that can be used for quality control steps. PRINSEQ is available as both standalone and web-based versions.</td>
      <td>To update</td>
      <td>http://prinseq.sourceforge.net/manual.html</td>
      <td>Fastq Manipulation, Metagenomics</td>
      <td>prinseq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/prinseq/</td>
    </tr>
    <tr>
      <td></td>
      <td>progressivemauve</td>
      <td></td>
      <td>snapshot_2015_02_13</td>
      <td>progressivemauve</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Mauve/ProgressiveMauve Multiple Sequence Aligner</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>progressivemauve</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/progressivemauve</td>
    </tr>
    <tr>
      <td></td>
      <td>prokka</td>
      <td>1.14.6</td>
      <td>1.14.6</td>
      <td>prokka</td>
      <td>Up-to-date</td>
      <td>prokka</td>
      <td>Prokka</td>
      <td>Gene prediction, Coding region prediction, Genome annotation</td>
      <td>Genomics, Model organisms, Virology</td>
      <td>Rapid annotation of prokaryotic genomes</td>
      <td>Software tool to annotate bacterial, archaeal and viral genomes quickly and produce standards-compliant output files.</td>
      <td>Up-to-date</td>
      <td>http://github.com/tseemann/prokka</td>
      <td>Sequence Analysis</td>
      <td>prokka</td>
      <td>crs4</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/prokka/</td>
    </tr>
    <tr>
      <td></td>
      <td>proteinortho</td>
      <td>6.2.3</td>
      <td>6.3.1</td>
      <td>proteinortho</td>
      <td>To update</td>
      <td>proteinortho</td>
      <td>Proteinortho</td>
      <td>Homology-based gene prediction</td>
      <td>Phylogeny</td>
      <td>Proteinortho is a tool to detect orthologous proteins/genes within different species.</td>
      <td>Proteinortho is a tool to detect orthologous genes within different species</td>
      <td>To update</td>
      <td>https://gitlab.com/paulklemm_PHD/proteinortho</td>
      <td>Proteomics</td>
      <td>proteinortho</td>
      <td>iuc</td>
      <td>https://gitlab.com/paulklemm_PHD/proteinortho</td>
    </tr>
    <tr>
      <td></td>
      <td>psiclass</td>
      <td>1.0.3</td>
      <td>1.0.3</td>
      <td>psiclass</td>
      <td>Up-to-date</td>
      <td>psiclass</td>
      <td></td>
      <td></td>
      <td></td>
      <td>PsiCLASS is a reference-based transcriptome assembler for single or multiple RNA-seq samples.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/splicebox/PsiCLASS</td>
      <td>Transcriptomics</td>
      <td>psiclass</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/psiclass</td>
    </tr>
    <tr>
      <td></td>
      <td>pureclip</td>
      <td>1.0.4</td>
      <td>1.3.1</td>
      <td>pureclip</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>PureCLIP is an HMM based peak caller specifically designed for eCLIP/iCLIP data</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/skrakau/PureCLIP</td>
      <td>Sequence Analysis, RNA, CLIP-seq</td>
      <td>pureclip</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pureclip</td>
    </tr>
    <tr>
      <td></td>
      <td>purge_dups</td>
      <td>1.2.6</td>
      <td>1.2.6</td>
      <td>purge_dups</td>
      <td>Up-to-date</td>
      <td>purge_dups</td>
      <td>purge_dups</td>
      <td>Genome assembly, Read binning, Scaffolding</td>
      <td>Sequence assembly</td>
      <td>Purge haplotigs and overlaps in an assembly based on read depth</td>
      <td>Identifying and removing haplotypic duplication in primary genome assemblies | haplotypic duplication identification tool | scripts/pd_config.py: script to generate a configuration file used by run_purge_dups.py | purge haplotigs and overlaps in an assembly based on read depth | Given a primary assembly pri_asm and an alternative assembly hap_asm (optional, if you have one), follow the steps shown below to build your own purge_dups pipeline, steps with same number can be run simultaneously. Among all the steps, although step 4 is optional, we highly recommend our users to do so, because assemblers may produce overrepresented seqeuences. In such a case, The final step 4 can be applied to remove those seqeuences</td>
      <td>Up-to-date</td>
      <td>https://github.com/dfguan/purge_dups</td>
      <td>Assembly</td>
      <td>purge_dups</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/purge_dups</td>
    </tr>
    <tr>
      <td></td>
      <td>pycoqc</td>
      <td>2.5.2</td>
      <td>2.5.2</td>
      <td>pycoqc</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>QC metrics for ONT Basecalling</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/tleonardi/pycoQC</td>
      <td>Nanopore</td>
      <td>pycoqc</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pycoqc</td>
    </tr>
    <tr>
      <td></td>
      <td>pyega3</td>
      <td>5.0.2</td>
      <td>5.1.0</td>
      <td>pyega3</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>EGA python client uses the EGA REST API to download authorized datasets and files.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/EGA-archive/ega-download-client</td>
      <td>Data Source</td>
      <td>ega_download_client</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pyega3</td>
    </tr>
    <tr>
      <td></td>
      <td>quast</td>
      <td>5.2.0</td>
      <td>5.2.0</td>
      <td>quast</td>
      <td>Up-to-date</td>
      <td>quast</td>
      <td>QUAST</td>
      <td>Visualisation, Sequence assembly validation</td>
      <td>Sequence assembly</td>
      <td>Quast (Quality ASsessment Tool) evaluates genome assemblies.</td>
      <td>QUAST stands for QUality ASsessment Tool.  It evaluates a quality of genome assemblies by computing various metrics and providing nice reports.</td>
      <td>Up-to-date</td>
      <td>http://quast.bioinf.spbau.ru/</td>
      <td>Assembly</td>
      <td>quast</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/quast</td>
    </tr>
    <tr>
      <td></td>
      <td>query_tabular</td>
      <td>3.3.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Loads tabular files into a SQLite DB to perform a SQL query producing a tabular output</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/query_tabular</td>
    </tr>
    <tr>
      <td></td>
      <td>ragtag</td>
      <td>2.1.0</td>
      <td>2.1.0</td>
      <td>ragtag</td>
      <td>Up-to-date</td>
      <td>ragtag</td>
      <td>ragtag</td>
      <td>Genome assembly</td>
      <td>Sequence assembly</td>
      <td>Reference-guided scaffolding of draft genomes tool.</td>
      <td>RagTag is a collection of software tools for scaffolding and improving modern genome assemblies.</td>
      <td>Up-to-date</td>
      <td>https://github.com/malonge/RagTag</td>
      <td>Assembly</td>
      <td>ragtag</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ragtag</td>
    </tr>
    <tr>
      <td></td>
      <td>rapidnj</td>
      <td>2.3.2</td>
      <td>2.3.2</td>
      <td>rapidnj</td>
      <td>Up-to-date</td>
      <td>rapidnj</td>
      <td>RapidNJ</td>
      <td>Phylogenetic tree generation</td>
      <td>Phylogeny</td>
      <td>Galaxy wrapper for the RapidNJ tool</td>
      <td>A tool for fast canonical neighbor-joining tree construction.</td>
      <td>Up-to-date</td>
      <td>https://birc.au.dk/software/rapidnj/</td>
      <td>Phylogenetics</td>
      <td>rapidnj</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/rapidnj</td>
    </tr>
    <tr>
      <td></td>
      <td>raven</td>
      <td>1.8.0</td>
      <td>1.8.3</td>
      <td>raven-assembler</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Raven is a de novo genome assembler for long uncorrected reads.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/lbcb-sci/raven</td>
      <td>Assembly</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/raven</td>
    </tr>
    <tr>
      <td></td>
      <td>raxml</td>
      <td>8.2.12</td>
      <td>8.2.13</td>
      <td>raxml</td>
      <td>To update</td>
      <td>raxml</td>
      <td>RAxML</td>
      <td>Sequence analysis, Phylogenetic tree analysis</td>
      <td>Phylogenetics, Sequence analysis</td>
      <td>RAxML - A Maximum Likelihood based phylogenetic inference</td>
      <td>A tool for Phylogenetic Analysis and Post-Analysis of Large Phylogenies.</td>
      <td>To update</td>
      <td>http://www.exelixis-lab.org/web/software/raxml/</td>
      <td>Phylogenetics</td>
      <td>raxml</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/raxml</td>
    </tr>
    <tr>
      <td></td>
      <td>rcorrector</td>
      <td>1.0.3+galaxy1</td>
      <td>1.0.6</td>
      <td>rcorrector</td>
      <td>To update</td>
      <td>rcorrector</td>
      <td>Rcorrector</td>
      <td>Sequencing error detection</td>
      <td>RNA, RNA-Seq, Sequencing</td>
      <td>Rcorrector (RNA-seq error CORRECTOR) is a kmer-based error correction method for RNA-seq data.</td>
      <td>This is a kmer-based error correction method for RNA-seq data. It can also be applied to other types of sequencing data where the read coverage is non-uniform, such as single-cell sequencing.</td>
      <td>To update</td>
      <td>https://github.com/mourisl/Rcorrector</td>
      <td>Fastq Manipulation</td>
      <td>rcorrector</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/rcorrector</td>
    </tr>
    <tr>
      <td></td>
      <td>read_it_and_keep</td>
      <td>0.2.2</td>
      <td>0.3.0</td>
      <td>read-it-and-keep</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Rapid decontamination of SARS-CoV-2 sequencing reads</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/GenomePathogenAnalysisService/read-it-and-keep</td>
      <td>Sequence Analysis</td>
      <td>read_it_and_keep</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/read-it-and-keep</td>
    </tr>
    <tr>
      <td></td>
      <td>recentrifuge</td>
      <td>1.12.1</td>
      <td>1.13.1</td>
      <td>recentrifuge</td>
      <td>To update</td>
      <td>Recentrifuge</td>
      <td>Recentrifuge</td>
      <td>Taxonomic classification, Expression analysis, Cross-assembly</td>
      <td>Metagenomics, Microbial ecology, Metagenomic sequencing</td>
      <td>"With Recentrifuge, researchers can analyze results from taxonomic classifiers using interactive charts with emphasis on the confidence level of the classifications.In addition to contamination-subtracted samples.Recentrifuge provides shared and exclusive taxa per sample,thus enabling robust contamination removal and comparative analysis in environmental and clinical metagenomics."</td>
      <td>Robust comparative analysis and contamination removal for metagenomics.</td>
      <td>To update</td>
      <td>https://github.com/khyox/recentrifuge</td>
      <td>Metagenomics</td>
      <td>recentrifuge</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/recentrifuge</td>
    </tr>
    <tr>
      <td></td>
      <td>red</td>
      <td>2018.09.10</td>
      <td>2018.09.10</td>
      <td>red</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Red (REpeat Detector)</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/BioinformaticsToolsmith/Red</td>
      <td>Sequence Analysis</td>
      <td>red</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/red</td>
    </tr>
    <tr>
      <td></td>
      <td>repeatmodeler</td>
      <td>2.0.5</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>RepeatModeler - Model repetitive DNA</td>
      <td></td>
      <td>To update</td>
      <td>https://www.repeatmasker.org/RepeatModeler/</td>
      <td>Genome annotation</td>
      <td>repeatmodeler</td>
      <td>csbl</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/repeatmodeler</td>
    </tr>
    <tr>
      <td></td>
      <td>resize_coordinate_window</td>
      <td>1.0.2</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Contains a tool that modifies the start and stop coordinates of GFF data, expanding the coordinate windowby a specified size.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Genomic Interval Operations</td>
      <td>resize_coordinate_window</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/resize_coordinate_window</td>
    </tr>
    <tr>
      <td></td>
      <td>rgcca</td>
      <td>3.0.2</td>
      <td>3.0.3</td>
      <td>rgccacmd</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>multi-block analysis</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/rgcca-factory/RGCCA</td>
      <td>Statistics</td>
      <td>rgcca</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/rgcca</td>
    </tr>
    <tr>
      <td></td>
      <td>roary</td>
      <td>3.13.0</td>
      <td>3.13.0</td>
      <td>roary</td>
      <td>Up-to-date</td>
      <td>roary</td>
      <td>Roary</td>
      <td>Genome assembly</td>
      <td>DNA, Genomics, Mapping</td>
      <td>Roary the pangenome pipeline</td>
      <td>A high speed stand alone pan genome pipeline, which takes annotated assemblies in GFF3 format (produced by Prokka (Seemann, 2014)) and calculates the pan genome.</td>
      <td>Up-to-date</td>
      <td>https://sanger-pathogens.github.io/Roary/</td>
      <td>Sequence Analysis</td>
      <td>roary</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/roary</td>
    </tr>
    <tr>
      <td></td>
      <td>ruvseq</td>
      <td>1.26.0</td>
      <td>1.36.0</td>
      <td>bioconductor-ruvseq</td>
      <td>To update</td>
      <td>ruvseq</td>
      <td>RUVSeq</td>
      <td>Differential gene expression analysis</td>
      <td>Gene expression, RNA-seq</td>
      <td>Remove Unwanted Variation from RNA-Seq Data</td>
      <td>This package implements the remove unwanted variation (RUV) methods for the normalization of RNA-Seq read counts between samples.</td>
      <td>To update</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/DESeq2.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>ruvseq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ruvseq</td>
    </tr>
    <tr>
      <td></td>
      <td>sarscov2formatter</td>
      <td>1.0</td>
      <td>1.0</td>
      <td>sarscov2formatter</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>sarscov2formatter custom script</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/nickeener/sarscov2formatter</td>
      <td>Sequence Analysis</td>
      <td>sarscov2formatter</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/sarscov2formatter</td>
    </tr>
    <tr>
      <td></td>
      <td>sarscov2summary</td>
      <td>0.1</td>
      <td>0.5</td>
      <td>sarscov2summary</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>sarscov2summary custom script</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/nickeener/sarscov2summary</td>
      <td>Sequence Analysis</td>
      <td>sarscov2summary</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/sarscov2summary</td>
    </tr>
    <tr>
      <td></td>
      <td>scoary</td>
      <td>1.6.16</td>
      <td>1.6.16</td>
      <td>scoary</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Scoary calculates the assocations between all genes in the accessory genome and the traits.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/AdmiralenOla/Scoary</td>
      <td>Metagenomics</td>
      <td>scoary</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/scoary</td>
    </tr>
    <tr>
      <td></td>
      <td>scpipe</td>
      <td>1.0.0+galaxy2</td>
      <td>2.2.0</td>
      <td>bioconductor-scpipe</td>
      <td>To update</td>
      <td>scpipe</td>
      <td>scPipe</td>
      <td>Genome annotation, Validation, Alignment, Visualisation</td>
      <td>Gene expression, RNA-seq, Sequencing</td>
      <td>A flexible preprocessing pipeline for single-cell RNA-sequencing data</td>
      <td>A preprocessing pipeline for single cell RNA-seq data that starts from the fastq files and produces a gene count matrix with associated quality control information. It can process fastq data generated by CEL-seq, MARS-seq, Drop-seq, Chromium 10x and SMART-seq protocols.</td>
      <td>To update</td>
      <td>http://bioconductor.org/packages/release/bioc/html/scPipe.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>scpipe</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/scpipe</td>
    </tr>
    <tr>
      <td></td>
      <td>seacr</td>
      <td>1.3</td>
      <td>1.3</td>
      <td>seacr</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SEACR is intended to call peaks and enriched regions from sparse CUT&amp;RUN or chromatin profiling data in which background is dominated by zeroes.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/FredHutch/SEACR</td>
      <td>Epigenetics, ChIP-seq</td>
      <td>seacr</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/seacr</td>
    </tr>
    <tr>
      <td></td>
      <td>semibin</td>
      <td>2.0.2</td>
      <td>2.0.2</td>
      <td>semibin</td>
      <td>Up-to-date</td>
      <td>semibin</td>
      <td>SemiBin</td>
      <td>Sequence assembly, Read binning</td>
      <td>Metagenomics, Machine learning, Microbial ecology, Sequence assembly</td>
      <td>SemiBin: Semi-supervised Metagenomic Binning Using Siamese Neural Networks</td>
      <td>Command tool for metagenomic binning with semi-supervised deep learning using information from reference genomes.</td>
      <td>Up-to-date</td>
      <td>https://semibin.readthedocs.io/en/latest/</td>
      <td>Metagenomics</td>
      <td>semibin</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/semibin</td>
    </tr>
    <tr>
      <td></td>
      <td>seq2hla</td>
      <td>2.3</td>
      <td>2.3</td>
      <td>seq2hla</td>
      <td>Up-to-date</td>
      <td>seq2hla</td>
      <td>Seq2HLA</td>
      <td>Read mapping, Genetic variation analysis</td>
      <td>Transcriptomics, Mapping</td>
      <td>Precision HLA typing and expression from RNAseq data</td>
      <td>seq2HLA is a computational tool to determine Human Leukocyte Antigen (HLA) directly from existing and future short RNA-Seq reads. It takes standard RNA-Seq sequence reads in fastq format as input, uses a bowtie index comprising known HLA alleles and outputs the most likely HLA class I and class II types, a p-value for each call, and the expression of each class.</td>
      <td>Up-to-date</td>
      <td>https://github.com/TRON-Bioinformatics/seq2HLA</td>
      <td>Sequence Analysis</td>
      <td>seq2hla</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/seq2hla</td>
    </tr>
    <tr>
      <td></td>
      <td>seqcomplexity</td>
      <td>0.1.2</td>
      <td>0.1.2</td>
      <td>seqcomplexity</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Sequence complexity for raw reads</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/stevenweaver/seqcomplexity</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/stephenshank/tools-iuc/tree/seqcomplexity/tools/seqcomplexity/</td>
    </tr>
    <tr>
      <td></td>
      <td>seurat</td>
      <td>4.3.0.1</td>
      <td>3.0.2</td>
      <td>r-seurat</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A toolkit for quality control, analysis, and exploration of single cell RNA sequencing data</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/satijalab/seurat</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>seurat</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/seurat</td>
    </tr>
    <tr>
      <td></td>
      <td>shasta</td>
      <td>0.6.0</td>
      <td>0.11.1</td>
      <td>shasta</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Fast de novo assembly of long read sequencing data</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/chanzuckerberg/shasta</td>
      <td>Assembly, Nanopore</td>
      <td>shasta</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/shasta</td>
    </tr>
    <tr>
      <td></td>
      <td>shovill</td>
      <td>1.1.0</td>
      <td>1.1.0</td>
      <td>shovill</td>
      <td>Up-to-date</td>
      <td>shovill</td>
      <td>shovill</td>
      <td>Genome assembly</td>
      <td>Genomics, Microbiology, Sequence assembly</td>
      <td>Faster de novo assembly pipeline based around Spades</td>
      <td>Shovill is a pipeline for assembly of bacterial isolate genomes from Illumina paired-end reads.  Shovill uses SPAdes at its core, but alters the steps before and after the primary assembly step to get similar results in less time. Shovill also supports other assemblers like SKESA, Velvet and Megahit, so you can take advantage of the pre- and post-processing the Shovill provides with those too.</td>
      <td>Up-to-date</td>
      <td>https://github.com/tseemann/shovill</td>
      <td>Assembly</td>
      <td>shovill</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/shovill</td>
    </tr>
    <tr>
      <td></td>
      <td>sickle</td>
      <td>1.33.2</td>
      <td>1.33</td>
      <td>sickle-trim</td>
      <td>To update</td>
      <td>sickle</td>
      <td>sickle</td>
      <td>Sequence trimming</td>
      <td>Data quality management</td>
      <td>A windowed adaptive trimming tool for FASTQ files using quality</td>
      <td>A  tool that uses sliding windows along with quality and length thresholds to determine when quality is sufficiently low to trim the 3'-end of reads and also determines when the quality is sufficiently high enough to trim the 5'-end of reads.</td>
      <td>To update</td>
      <td>https://github.com/najoshi/sickle</td>
      <td>Fastq Manipulation, Sequence Analysis</td>
      <td>sickle</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/sickle</td>
    </tr>
    <tr>
      <td></td>
      <td>sina</td>
      <td>1.7.2</td>
      <td>1.7.2</td>
      <td>sina</td>
      <td>Up-to-date</td>
      <td>sina</td>
      <td>SINA</td>
      <td>Sequence alignment analysis, Multiple sequence alignment, Taxonomic classification, Structure-based sequence alignment</td>
      <td>Sequencing, RNA, Nucleic acid structure analysis, Taxonomy, Sequence analysis, Taxonomy</td>
      <td>SINA reference based multiple sequence alignment</td>
      <td>Aligns and optionally taxonomically classifies your rRNA gene sequences.Reference based multiple sequence alignment</td>
      <td>Up-to-date</td>
      <td>https://sina.readthedocs.io/en/latest/</td>
      <td>Sequence Analysis</td>
      <td>sina</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/sina</td>
    </tr>
    <tr>
      <td></td>
      <td>slamdunk</td>
      <td>0.4.3</td>
      <td>0.4.3</td>
      <td>slamdunk</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Slamdunk maps and quantifies SLAMseq reads</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://t-neumann.github.io/slamdunk</td>
      <td>RNA, Transcriptomics, Sequence Analysis, Next Gen Mappers</td>
      <td>slamdunk</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/slamdunk</td>
    </tr>
    <tr>
      <td></td>
      <td>sleuth</td>
      <td>0.30.1</td>
      <td>0.30.1</td>
      <td>r-sleuth</td>
      <td>Up-to-date</td>
      <td>sleuth</td>
      <td>sleuth</td>
      <td>Expression data visualisation, Differential gene expression analysis, Gene expression profiling, Statistical calculation</td>
      <td>RNA-seq, Gene expression, Statistics and probability</td>
      <td>Sleuth is a program for differential analysis of RNA-Seq data.</td>
      <td>A statistical model and software application for RNA-seq differential expression analysis.</td>
      <td>Up-to-date</td>
      <td>https://github.com/pachterlab/sleuth</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>sleuth</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/sleuth</td>
    </tr>
    <tr>
      <td></td>
      <td>smudgeplot</td>
      <td>0.2.5</td>
      <td>0.2.5</td>
      <td>smudgeplot</td>
      <td>Up-to-date</td>
      <td>smudgeplots</td>
      <td>Smudgeplots</td>
      <td>Sequence trimming, Genotyping, k-mer counting</td>
      <td>Sequence assembly, Genetic variation, Mathematics</td>
      <td>Inference of ploidy and heterozygosity structure using whole genome sequencing</td>
      <td>Reference-free profiling of polyploid genomes | Inference of ploidy and heterozygosity structure using whole genome sequencing data | Smudgeplots are computed from raw or even better from trimmed reads and show the haplotype structure using heterozygous kmer pairs. For example: | This tool extracts heterozygous kmer pairs from kmer dump files and performs gymnastics with them. We are able to disentangle genome structure by comparing the sum of kmer pair coverages (CovA + CovB) to their relative coverage (CovA / (CovA + CovB)). Such an approach also allows us to analyze obscure genomes with duplications, various ploidy levels, etc | GenomeScope 2.0 and Smudgeplots: Reference-free profiling of polyploid genomes Timothy Rhyker Ranallo-Benavidez, Kamil S. Jaron, Michael C. Schatz bioRxiv 747568; doi: https://doi.org/10.1101/747568</td>
      <td>Up-to-date</td>
      <td>https://github.com/KamilSJaron/smudgeplot</td>
      <td>Assembly</td>
      <td>smudgeplot</td>
      <td>galaxy-australia</td>
      <td>https://github.com/galaxyproject/tools-iuc</td>
    </tr>
    <tr>
      <td></td>
      <td>sniffles</td>
      <td>1.0.12</td>
      <td>2.2</td>
      <td>sniffles</td>
      <td>To update</td>
      <td>sniffles</td>
      <td>Sniffles</td>
      <td>Sequence analysis, Structural variation detection</td>
      <td>DNA structural variation, Sequencing</td>
      <td>Galaxy wrapper for sniffles</td>
      <td>An algorithm for structural variation detection from third generation sequencing alignment.</td>
      <td>To update</td>
      <td>https://github.com/fritzsedlazeck/Sniffles</td>
      <td>Sequence Analysis</td>
      <td>sniffles</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc</td>
    </tr>
    <tr>
      <td></td>
      <td>snipit</td>
      <td>1.0.7</td>
      <td>1.2</td>
      <td>snipit</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Summarise snps relative to a reference sequence</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/aineniamh/snipit</td>
      <td>Variant Analysis, Sequence Analysis</td>
      <td>snipit</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/snipit</td>
    </tr>
    <tr>
      <td></td>
      <td>snippy</td>
      <td></td>
      <td>4.6.0</td>
      <td>snippy</td>
      <td>To update</td>
      <td>snippy</td>
      <td>snippy</td>
      <td>Phylogenetic tree visualisation, Phylogenetic tree generation, Variant calling</td>
      <td>Genomics, Model organisms, DNA polymorphism, Phylogenetics</td>
      <td>Contains the snippy tool for characterising microbial snps</td>
      <td>Rapid haploid variant calling and core SNP phylogeny generation.</td>
      <td>To update</td>
      <td>https://github.com/tseemann/snippy</td>
      <td>Sequence Analysis</td>
      <td>snippy</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/snippy</td>
    </tr>
    <tr>
      <td></td>
      <td>snpfreqplot</td>
      <td>1.0</td>
      <td></td>
      <td>r-base</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Generates a heatmap of allele frequencies grouped by variant type for SARS-CoV-2 data</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/snpfreqplot/</td>
      <td>Variant Analysis</td>
      <td>snpfreqplot</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/snpfreqplot/</td>
    </tr>
    <tr>
      <td></td>
      <td>socru</td>
      <td>2.1.7</td>
      <td>2.2.4</td>
      <td>socru</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Order and orientation of complete bacterial genomes</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/quadram-institute-bioscience/socru</td>
      <td>Sequence Analysis</td>
      <td>socru</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/socru</td>
    </tr>
    <tr>
      <td></td>
      <td>sonneityping</td>
      <td>20210201</td>
      <td>20210201</td>
      <td>sonneityping</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Scripts for parsing Mykrobe predict results for Shigella sonnei.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/katholt/sonneityping</td>
      <td>Sequence Analysis</td>
      <td>sonneityping</td>
      <td>iuc</td>
      <td>https://github.com/katholt/sonneityping</td>
    </tr>
    <tr>
      <td></td>
      <td>spades</td>
      <td>3.15.5</td>
      <td>3.15.5</td>
      <td>spades</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SPAdes is an assembly toolkit containing various assembly pipelines. It implements the following 4 stages: assembly graph construction, k-bimer adjustment, construction of paired assembly graph and contig construction.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/ablab/spades</td>
      <td>Assembly, RNA, Metagenomics</td>
      <td>spades</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/spades</td>
    </tr>
    <tr>
      <td></td>
      <td>spaln</td>
      <td>2.4.9</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Spaln (space-efficient spliced alignment) maps and aligns a set of cDNA or protein sequences onto a whole genomic sequence.</td>
      <td></td>
      <td>To update</td>
      <td>http://www.genome.ist.i.kyoto-u.ac.jp/~aln_user/spaln/</td>
      <td>Sequence Analysis, Genome annotation</td>
      <td>spaln</td>
      <td>iuc</td>
      <td>https://github.com/ogotoh/spaln</td>
    </tr>
    <tr>
      <td></td>
      <td>spotyping</td>
      <td>2.1</td>
      <td>2.1</td>
      <td>spotyping</td>
      <td>Up-to-date</td>
      <td>spotyping</td>
      <td>SpoTyping</td>
      <td>Variant pattern analysis</td>
      <td>Microbiology, Sequencing, Sequence composition, complexity and repeats, Genetic variation</td>
      <td>SpoTyping allows fast and accurate in silico Mycobacterium spoligotyping from sequence reads</td>
      <td>Fast and accurate in silico Mycobacterium spoligotyping from sequence reads.</td>
      <td>Up-to-date</td>
      <td>https://github.com/xiaeryu/SpoTyping-v2.0</td>
      <td>Sequence Analysis</td>
      <td>spotyping</td>
      <td>iuc</td>
      <td>https://github.com/xiaeryu/SpoTyping-v2.0/tree/master/SpoTyping-v2.0-commandLine</td>
    </tr>
    <tr>
      <td></td>
      <td>spyboat</td>
      <td>0.1.2</td>
      <td></td>
      <td>spyboat</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Wavelet analysis for 3d-image stacks</td>
      <td></td>
      <td>To update</td>
      <td>http://github.com/tensionhead/spyboat</td>
      <td>Imaging, Graphics</td>
      <td>spyboat</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/packages/spyboat</td>
    </tr>
    <tr>
      <td></td>
      <td>srst2</td>
      <td>0.2.0</td>
      <td>1.19</td>
      <td>samtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SRST2 Short Read Sequence Typing for Bacterial Pathogens</td>
      <td></td>
      <td>To update</td>
      <td>http://katholt.github.io/srst2/</td>
      <td>Metagenomics</td>
      <td>srst2</td>
      <td>iuc</td>
      <td>https://github.com/katholt/srst2</td>
    </tr>
    <tr>
      <td></td>
      <td>star_fusion</td>
      <td>0.5.4-3+galaxy1</td>
      <td>1.12.0</td>
      <td>star-fusion</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>STAR Fusion detects fusion genes in RNA-Seq data after running RNA-STAR</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Transcriptomics</td>
      <td>star_fusion</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/star_fusion</td>
    </tr>
    <tr>
      <td></td>
      <td>stringtie</td>
      <td>2.2.1</td>
      <td>2.2.1</td>
      <td>stringtie</td>
      <td>Up-to-date</td>
      <td>stringtie</td>
      <td>StringTie</td>
      <td>Transcriptome assembly, RNA-Seq analysis</td>
      <td>Transcriptomics, RNA-seq</td>
      <td>StringTie is a fast and highly efficient assembler of RNA-Seq alignments into potential transcripts.</td>
      <td>Fast and highly efficient assembler of RNA-Seq alignments into potential transcripts. It uses a novel network flow algorithm as well as an optional de novo assembly step to assemble and quantitate full-length transcripts representing multiple splice variants for each gene locus.</td>
      <td>Up-to-date</td>
      <td>http://ccb.jhu.edu/software/stringtie/</td>
      <td>Transcriptomics</td>
      <td>stringtie</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/stringtie</td>
    </tr>
    <tr>
      <td></td>
      <td>structure</td>
      <td>2.3.4</td>
      <td>2.3.4</td>
      <td>structure</td>
      <td>Up-to-date</td>
      <td>structure</td>
      <td>Structure</td>
      <td>Genetic variation analysis</td>
      <td>Population genetics</td>
      <td>for using multi-locus genotype data to investigate population structure.</td>
      <td>The program structureis a free software package for using multi-locus genotype data to investigate population structure. Its uses include inferring the presence of distinct populations, assigning individuals to populations, studying hybrid zones, identifying migrants and admixed individuals, and estimating population allele frequencies in situations where many individuals are migrants or admixed.</td>
      <td>Up-to-date</td>
      <td></td>
      <td>Phylogenetics, Variant Analysis</td>
      <td>structure</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/structure</td>
    </tr>
    <tr>
      <td></td>
      <td>syndiva</td>
      <td>1.0</td>
      <td>1.2.4</td>
      <td>clustalo</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SynDivA was developed to analyze the diversity of synthetic libraries of a Fibronectin domain.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>syndiva</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/SynDivA</td>
    </tr>
    <tr>
      <td></td>
      <td>table_compute</td>
      <td>1.2.4</td>
      <td></td>
      <td>pandas</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Perform general-purpose table operations</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>table_compute</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/table_compute</td>
    </tr>
    <tr>
      <td></td>
      <td>tabular_to_fastq</td>
      <td>1.1.5</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tabular to FASTQ converter</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproject/sequence_utils</td>
      <td>Fastq Manipulation</td>
      <td>tabular_to_fastq</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/galaxy_sequence_utils/tabular_to_fastq</td>
    </tr>
    <tr>
      <td></td>
      <td>tag_pileup_frequency</td>
      <td>1.0.2</td>
      <td></td>
      <td>openjdk</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Contains a tool that generates a frequency pileup of the 5' ends of aligned reads in a BAM filerelative to reference points in a BED file.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/seqcode/cegr-tools/tree/master/src/org/seqcode/cegrtools/tagpileup</td>
      <td>Statistics, SAM, Genomic Interval Operations</td>
      <td>tag_pileup_frequency</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/tag_pileup_frequency</td>
    </tr>
    <tr>
      <td></td>
      <td>taxonomy_krona_chart</td>
      <td>2.7.1+galaxy0</td>
      <td>2.8.1</td>
      <td>krona</td>
      <td>To update</td>
      <td>krona</td>
      <td>Krona</td>
      <td>Visualisation</td>
      <td>Metagenomics</td>
      <td>Krona pie chart from taxonomic profile</td>
      <td>Krona creates interactive HTML5 charts of hierarchical data (such as taxonomic abundance in a metagenome).</td>
      <td>To update</td>
      <td>http://sourceforge.net/projects/krona/</td>
      <td>Assembly</td>
      <td>taxonomy_krona_chart</td>
      <td>crs4</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/taxonomy_krona_chart</td>
    </tr>
    <tr>
      <td></td>
      <td>tb_variant_filter</td>
      <td>0.4.0</td>
      <td>0.4.0</td>
      <td>tb_variant_filter</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>M. tuberculosis H37Rv VCF filter</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/COMBAT-TB/tb_variant_filter</td>
      <td>Variant Analysis</td>
      <td>tb_variant_filter</td>
      <td>iuc</td>
      <td>https://github.com/COMBAT-TB/tb_variant_filter</td>
    </tr>
    <tr>
      <td></td>
      <td>tbl2gff3</td>
      <td>1.2</td>
      <td>0.6.6</td>
      <td>bcbiogff</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Table to GFF3</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/tbl2gff3</td>
      <td>Convert Formats, Sequence Analysis</td>
      <td>tbl2gff3</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/tbl2gff3</td>
    </tr>
    <tr>
      <td></td>
      <td>tbvcfreport</td>
      <td>0.1.10</td>
      <td>0.1.10</td>
      <td>tbvcfreport</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Generate HTML report from SnpEff M.tuberculosis VCF(s)</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/COMBAT-TB/tbvcfreport</td>
      <td>Variant Analysis</td>
      <td>tbvcfreport</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/tbvcfreport</td>
    </tr>
    <tr>
      <td></td>
      <td>te_finder</td>
      <td>1.0.1</td>
      <td>1.19</td>
      <td>samtools</td>
      <td>To update</td>
      <td>tefinder</td>
      <td>TEfinder</td>
      <td>Genome indexing, Variant calling, PCR primer design</td>
      <td>Sequencing, Mobile genetic elements, Workflows, Evolutionary biology, Genetic variation</td>
      <td>Transposable element insertions finder</td>
      <td>A Bioinformatics Pipeline for Detecting New Transposable Element Insertion Events in Next-Generation Sequencing Data.A bioinformatics tool for detecting novel transposable element insertions.TEfinder uses discordant reads to detect novel transposable element insertion events in paired-end sample sequencing data.</td>
      <td>To update</td>
      <td>https://github.com/VistaSohrab/TEfinder</td>
      <td>Sequence Analysis</td>
      <td>te_finder</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/te_finder/</td>
    </tr>
    <tr>
      <td></td>
      <td>tetyper</td>
      <td>1.1</td>
      <td>1.1</td>
      <td>tetyper</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Type a specific transposable element (TE) of interest from paired-end sequencing data.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/aesheppard/TETyper</td>
      <td>Sequence Analysis</td>
      <td>tetyper</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/tetyper</td>
    </tr>
    <tr>
      <td></td>
      <td>tn93</td>
      <td>1.0.6</td>
      <td>1.0.13</td>
      <td>tn93</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute distances between sequences</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/veg/tn93/</td>
      <td>Sequence Analysis</td>
      <td>tn93</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/tn93/</td>
    </tr>
    <tr>
      <td></td>
      <td>transdecoder</td>
      <td>5.5.0</td>
      <td>5.7.1</td>
      <td>transdecoder</td>
      <td>To update</td>
      <td>TransDecoder</td>
      <td>TransDecoder</td>
      <td>Coding region prediction, de Novo sequencing, De-novo assembly</td>
      <td>Genomics, Gene transcripts, RNA-Seq, Gene expression, Sequence assembly, Whole genome sequencing</td>
      <td>TransDecoder finds coding regions within transcripts</td>
      <td>TransDecoder identifies candidate coding regions within transcript sequences, such as those generated by de novo RNA-Seq transcript assembly using Trinity, or constructed based on RNA-Seq alignments to the genome using Tophat and Cufflinks.</td>
      <td>To update</td>
      <td>https://transdecoder.github.io/</td>
      <td>Transcriptomics, RNA</td>
      <td>transdecoder</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/transdecoder</td>
    </tr>
    <tr>
      <td></td>
      <td>transtermhp</td>
      <td></td>
      <td>2.09</td>
      <td>transtermhp</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Finds rho-independent transcription terminators in bacterial genomes</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>transtermhp</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/transtermhp</td>
    </tr>
    <tr>
      <td></td>
      <td>trinity</td>
      <td>2.15.1</td>
      <td>2.15.1</td>
      <td>trinity</td>
      <td>Up-to-date</td>
      <td>trinity</td>
      <td>Trinity</td>
      <td>Transcriptome assembly</td>
      <td>Transcriptomics, Gene expression, Gene transcripts</td>
      <td>Trinity represents a method for the efficient and robust de novo reconstruction of transcriptomes from RNA-seq datahttps://github.com/trinityrnaseq/trinityrnaseq</td>
      <td>Trinity is a transcriptome assembler which relies on three different tools, inchworm an assembler, chrysalis which pools contigs and butterfly which amongst others compacts a graph resulting from butterfly with reads.</td>
      <td>Up-to-date</td>
      <td>https://github.com/trinityrnaseq/trinityrnaseq</td>
      <td>Transcriptomics, RNA</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/trinity</td>
    </tr>
    <tr>
      <td></td>
      <td>trinotate</td>
      <td>3.2.2</td>
      <td>4.0.2</td>
      <td>trinotate</td>
      <td>To update</td>
      <td>trinotate</td>
      <td>Trinotate</td>
      <td>Gene functional annotation</td>
      <td>Gene expression, Transcriptomics</td>
      <td>Trinotate is a comprehensive annotation suite designed for automatic functional annotation of de novo transcriptomes.</td>
      <td>Comprehensive annotation suite designed for automatic functional annotation of transcriptomes, particularly de novo assembled transcriptomes, from model or non-model organisms.</td>
      <td>To update</td>
      <td>https://trinotate.github.io/</td>
      <td>Transcriptomics, RNA</td>
      <td>trinotate</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/trinotate</td>
    </tr>
    <tr>
      <td></td>
      <td>tsne</td>
      <td>0.0.2</td>
      <td>0.13</td>
      <td>r-rtsne</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>T-Distributed Stochastic Neighbor Embedding using a Barnes-Hut Implementation</td>
      <td></td>
      <td>To update</td>
      <td>https://cran.r-project.org/web/packages/Rtsne/</td>
      <td>Text Manipulation</td>
      <td>tsne</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/tsne</td>
    </tr>
    <tr>
      <td></td>
      <td>tximport</td>
      <td>1.22.0</td>
      <td>1.30.0</td>
      <td>bioconductor-tximport</td>
      <td>To update</td>
      <td>tximport</td>
      <td>tximport</td>
      <td>Pathway or network analysis, Formatting, RNA-Seq analysis</td>
      <td>Transcriptomics, Gene transcripts, Workflows</td>
      <td>Wrapper for the Bioconductor package tximport</td>
      <td>An R/Bioconductor package that imports transcript-level abundance, estimated counts and transcript lengths, and summarizes into matrices for use with downstream gene-level analysis packages.</td>
      <td>To update</td>
      <td>http://bioconductor.org/packages/tximport/</td>
      <td>Transcriptomics</td>
      <td>tximport</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/tximport</td>
    </tr>
    <tr>
      <td></td>
      <td>unicycler</td>
      <td>0.5.0</td>
      <td>0.5.0</td>
      <td>unicycler</td>
      <td>Up-to-date</td>
      <td>unicycler</td>
      <td>Unicycler</td>
      <td>Genome assembly, Aggregation</td>
      <td>Microbiology, Genomics, Sequencing, Sequence assembly</td>
      <td>Unicycler is a hybrid assembly pipeline for bacterial genomes.</td>
      <td>A tool for assembling bacterial genomes from a combination of short (2nd generation) and long (3rd generation) sequencing reads.</td>
      <td>Up-to-date</td>
      <td>https://github.com/rrwick/Unicycler</td>
      <td>Assembly</td>
      <td>unicycler</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/unicycler</td>
    </tr>
    <tr>
      <td></td>
      <td>usher</td>
      <td>0.2.1</td>
      <td>0.6.3</td>
      <td>usher</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>UShER toolkit wrappers</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/yatisht/usher</td>
      <td>Phylogenetics</td>
      <td>usher</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/usher</td>
    </tr>
    <tr>
      <td></td>
      <td>valet</td>
      <td></td>
      <td>1.0</td>
      <td>valet</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A pipeline for detecting mis-assemblies in metagenomic assemblies.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/marbl/VALET</td>
      <td>Metagenomics</td>
      <td>valet</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/valet</td>
    </tr>
    <tr>
      <td></td>
      <td>vapor</td>
      <td>1.0.2</td>
      <td>1.0.2</td>
      <td>vapor</td>
      <td>Up-to-date</td>
      <td>vapor</td>
      <td>VAPOR</td>
      <td>Data retrieval, De-novo assembly, Read mapping</td>
      <td>Whole genome sequencing, Mapping, Sequence assembly</td>
      <td>Classify Influenza samples from raw short read sequence data</td>
      <td>VAPOR is a tool for classification of Influenza samples from raw short read sequence data for downstream bioinformatics analysis. VAPOR is provided with a fasta file of full-length sequences (&gt; 20,000) for a given segment, a set of reads, and attempts to retrieve a reference that is closest to the sample strain.</td>
      <td>Up-to-date</td>
      <td>https://github.com/connor-lab/vapor</td>
      <td>Sequence Analysis</td>
      <td>vapor</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/vapor</td>
    </tr>
    <tr>
      <td></td>
      <td>varscan</td>
      <td>2.4.3</td>
      <td>2.4.6</td>
      <td>varscan</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>VarScan is a variant caller for high-throughput sequencing data</td>
      <td></td>
      <td>To update</td>
      <td>https://dkoboldt.github.io/varscan/</td>
      <td>Variant Analysis</td>
      <td>varscan</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/iuc/tree/master/tools/varscan</td>
    </tr>
    <tr>
      <td></td>
      <td>vcf2maf</td>
      <td>1.6.21</td>
      <td>1.6.21</td>
      <td>vcf2maf</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>vcf2maf: Convert VCF into MAF</td>
      <td></td>
      <td>Up-to-date</td>
      <td></td>
      <td>Convert Formats</td>
      <td>vcf2maf</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/vcf2maf</td>
    </tr>
    <tr>
      <td></td>
      <td>vcf2tsv</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Converts VCF files into tab-delimited format</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis, Convert Formats</td>
      <td>vcf2tsv</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcf2tsv</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfaddinfo</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Adds info fields from the second dataset which are not present in the first dataset.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfaddinfo</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfaddinfo</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfallelicprimitives</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Splits alleleic primitives (gaps or mismatches) into multiple VCF lines</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfallelicprimitives</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfallelicprimitives</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfanno</td>
      <td>0.3.3</td>
      <td>0.3.5</td>
      <td>vcfanno</td>
      <td>To update</td>
      <td>vcfanno</td>
      <td>vcfanno</td>
      <td>SNP annotation</td>
      <td>Genetic variation, Data submission, annotation and curation</td>
      <td>Annotate VCF files</td>
      <td>Fast, flexible annotation of genetic variants.</td>
      <td>To update</td>
      <td>https://github.com/brentp/vcfanno</td>
      <td>Variant Analysis</td>
      <td>vcfanno</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/vcfanno/</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfannotate</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Intersect VCF records with BED annotations</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfannotate</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfannotate</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfannotategenotypes</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Annotate genotypes in a VCF dataset using genotypes from another VCF dataset.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfannotategenotypes</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfannotategenotypes</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfbedintersect</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Intersect VCF and BED datasets</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfbedintersect</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfbedintersect</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfbreakcreatemulti</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Break multiple alleles into multiple records, or combine overallpoing alleles into a single record</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfbreakcreatemulti</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfbreakcreatemulti</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfcheck</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Verify that the reference allele matches the reference genome</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfcheck</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfcheck</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfcombine</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Combine multiple VCF datasets</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfcombine</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfcombine</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfcommonsamples</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Output records belonging to samples commong between two datasets.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfcommonsamples</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfcommonsamples</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfdistance</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Calculate distance to the nearest variant.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfdistance</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfdistance</td>
    </tr>
    <tr>
      <td></td>
      <td>vcffixup</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Count the allele frequencies across alleles present in each record in the VCF file.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcffixup</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcffixup</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfgeno2haplo</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert genotype-based phased alleles into haplotype alleles</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfgeno2haplo</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfgeno2haplo</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfgenotypes</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert numerical representation of genotypes to allelic.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfgenotypes</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfgenotypes</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfhethom</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Count the number of heterozygotes and alleles, compute het/hom ratio.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfhethom</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfhethom</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfleftalign</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Left-align indels and complex variants in VCF dataset</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfleftalign</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfleftalign</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfprimers</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Extract flanking sequences for each VCF record</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfprimers</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfprimers</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfrandomsample</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Randomly sample sites from VCF dataset</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfrandomsample</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfrandomsample</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfselectsamples</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Select samples from a VCF file</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfselectsamples</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfselectsamples</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfsort</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Sort VCF dataset by coordinate</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfsort</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfsort</td>
    </tr>
    <tr>
      <td></td>
      <td>vcfvcfintersect</td>
      <td></td>
      <td>1.0.9</td>
      <td>vcflib</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Intersect two VCF datasets</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/vcflib</td>
      <td>Variant Analysis</td>
      <td>vcfvcfintersect</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/vcflib/vcfvcfintersect</td>
    </tr>
    <tr>
      <td></td>
      <td>verkko</td>
      <td>1.3.1</td>
      <td>1.4.1</td>
      <td>verkko</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Telomere-to-telomere assembly pipeline</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/marbl/verkko</td>
      <td>Assembly</td>
      <td>verkko</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/verkko</td>
    </tr>
    <tr>
      <td></td>
      <td>virhunter</td>
      <td>1.0.0</td>
      <td></td>
      <td>numpy</td>
      <td>To update</td>
      <td>virhunter</td>
      <td>VirHunter</td>
      <td>Sequence classification</td>
      <td>Virology</td>
      <td>Deep Learning method for novel virus detection in sequencing data</td>
      <td>VirHunter is a deep learning method that uses Convolutional Neural Networks (CNNs) and a Random Forest Classifier to identify viruses in sequencing datasets. More precisely, VirHunter classifies previously assembled contigs as viral, host, and bacterial (contamination).</td>
      <td>To update</td>
      <td>https://github.com/cbib/virhunter</td>
      <td>Machine Learning</td>
      <td>virhunter</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/VirHunter</td>
    </tr>
    <tr>
      <td></td>
      <td>volcanoplot</td>
      <td>0.0.5</td>
      <td>2.2.1</td>
      <td>r-ggplot2</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tool to create a Volcano Plot</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Visualization, Transcriptomics, Statistics</td>
      <td>volcanoplot</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/volcanoplot</td>
    </tr>
    <tr>
      <td></td>
      <td>winnowmap</td>
      <td>2.03</td>
      <td>2.03</td>
      <td>winnowmap</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A long-read mapping tool optimized for mapping ONT and PacBio reads to repetitive reference sequences.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/marbl/Winnowmap</td>
      <td>Next Gen Mappers</td>
      <td>winnowmap</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/winnowmap</td>
    </tr>
    <tr>
      <td></td>
      <td>xpath</td>
      <td></td>
      <td>1.47</td>
      <td>perl-xml-xpath</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>XPath XML querying tool</td>
      <td></td>
      <td>To update</td>
      <td>http://search.cpan.org/dist/XML-XPath/</td>
      <td>Text Manipulation</td>
      <td>xpath</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/xpath</td>
    </tr>
    <tr>
      <td></td>
      <td>yahs</td>
      <td>1.2a.2</td>
      <td>1.2a.2</td>
      <td>yahs</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Yet Another Hi-C scaffolding tool</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/c-zhou/yahs</td>
      <td>Assembly</td>
      <td>yahs</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/yahs</td>
    </tr>
    <tr>
      <td></td>
      <td>zerone</td>
      <td>1.0</td>
      <td>1.0</td>
      <td>zerone</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ChIP-seq discretization and quality control</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/nanakiksc/zerone</td>
      <td>ChIP-seq</td>
      <td>zerone</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/zerone</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy wrapper id</td>
      <td>Galaxy wrapper version</td>
      <td>Conda version</td>
      <td>Conda id</td>
      <td>Status</td>
      <td>bio.tool id</td>
      <td>bio.tool name</td>
      <td>EDAM operation</td>
      <td>EDAM topic</td>
      <td>Description</td>
      <td>bio.tool description</td>
      <td>Status</td>
      <td>Source</td>
      <td>ToolShed categories</td>
      <td>ToolShed id</td>
      <td>Galaxy wrapper owner</td>
      <td>Galaxy wrapper source</td>
    </tr>
    <tr>
      <td></td>
      <td>bed_to_protein_map</td>
      <td>0.2.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Converts a BED file to a tabular list of exon locations</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>bed_to_protein_map</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/bed_to_protein_map</td>
    </tr>
    <tr>
      <td></td>
      <td>bigwig_to_bedgraph</td>
      <td>377+galaxy1</td>
      <td>448</td>
      <td>ucsc-bigwigtobedgraph</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Converts a bigWig file to bedGraph format</td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Convert Formats</td>
      <td>bigwig_to_bedgraph</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/bigwig_to_bedgraph</td>
    </tr>
    <tr>
      <td></td>
      <td>blast2go</td>
      <td>0.0.11</td>
      <td></td>
      <td>b2g4pipe</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Maps BLAST results to GO annotation terms</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/blast2go</td>
      <td>Ontology Manipulation, Sequence Analysis</td>
      <td>blast2go</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/blast2go</td>
    </tr>
    <tr>
      <td></td>
      <td>blastxml_to_top_descr</td>
      <td>0.1.2</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Make table of top BLAST match descriptions</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/blastxml_to_top_descr</td>
      <td>Convert Formats, Sequence Analysis, Text Manipulation</td>
      <td>blastxml_to_top_descr</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/blastxml_to_top_descr</td>
    </tr>
    <tr>
      <td></td>
      <td>c3s</td>
      <td>0.1.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Copernicus Climate Change Service (C3S)</td>
      <td></td>
      <td>To update</td>
      <td>https://cds.climate.copernicus.eu/cdsapp#!/search?type=dataset</td>
      <td>Climate Analysis</td>
      <td>c3s</td>
      <td>climate</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/c3s</td>
    </tr>
    <tr>
      <td></td>
      <td>cads</td>
      <td>0.1.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Copernicus Atmosphere Data Store (ADS)</td>
      <td></td>
      <td>To update</td>
      <td>https://ads.atmosphere.copernicus.eu/#!/home</td>
      <td>Climate Analysis</td>
      <td>cads</td>
      <td>climate</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/cads</td>
    </tr>
    <tr>
      <td></td>
      <td>cap3</td>
      <td>2.0.1</td>
      <td>10.2011</td>
      <td>cap3</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>cap3 wrapper</td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Assembly</td>
      <td>cap3</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/cap3</td>
    </tr>
    <tr>
      <td></td>
      <td>cesm</td>
      <td>2.1.3</td>
      <td>2.1.3</td>
      <td>cesm</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Community Earth System Model (CESM)</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://www.cesm.ucar.edu/</td>
      <td>Climate Analysis</td>
      <td>cesm</td>
      <td>climate</td>
      <td>https://github.com/ESCOMP/CESM</td>
    </tr>
    <tr>
      <td></td>
      <td>custom_pro_db</td>
      <td>1.22.0</td>
      <td>1.37.1</td>
      <td>bioconductor-rgalaxy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>CustomProDB</td>
      <td></td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/customProDB.html</td>
      <td>Proteomics</td>
      <td>custom_pro_db</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tools/bumbershoot/custom_pro_db</td>
    </tr>
    <tr>
      <td></td>
      <td>dbbuilder</td>
      <td>0.3.4</td>
      <td></td>
      <td>wget</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Protein Database Downloader</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/dbbuilder</td>
      <td>Proteomics</td>
      <td>dbbuilder</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/dbbuilder</td>
    </tr>
    <tr>
      <td></td>
      <td>decoyfasta</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Galaxy tool wrapper for the transproteomic pipeline decoyFASTA tool.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>decoyfasta</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/decoyfasta</td>
    </tr>
    <tr>
      <td></td>
      <td>dialignr</td>
      <td>1.2.0</td>
      <td>2.10.0</td>
      <td>bioconductor-dialignr</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>DIAlignR is an R package for retention time alignment of targeted mass spectrometric data, including DIA and SWATH-MS data. This tool works with MS2 chromatograms directly and uses dynamic programming for alignment of raw chromatographic traces. DIAlignR uses a hybrid approach of global (feature-based) and local (raw data-based) alignment to establish correspondence between peaks.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/shubham1637/DIAlignR</td>
      <td>Proteomics</td>
      <td>dialignr</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/dialignr</td>
    </tr>
    <tr>
      <td></td>
      <td>diann</td>
      <td>1.8.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>DiaNN (DIA-based Neural Networks) is a software for DIA/SWATH data processing.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/vdemichev/DiaNN</td>
      <td>Proteomics</td>
      <td>diann</td>
      <td>galaxyp</td>
      <td>https://github.com/vdemichev/DiaNN</td>
    </tr>
    <tr>
      <td></td>
      <td>diapysef</td>
      <td>0.3.5.0</td>
      <td>1.0.10</td>
      <td>diapysef</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>diapysef is a convenience package for working with DIA-PASEF data</td>
      <td></td>
      <td>To update</td>
      <td>https://pypi.org/project/diapysef/</td>
      <td>Proteomics</td>
      <td>diapysef</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/diapysef</td>
    </tr>
    <tr>
      <td></td>
      <td>diffacto</td>
      <td>1.0.6</td>
      <td>1.0.6</td>
      <td>diffacto</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Diffacto comparative protein abundance estimation</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/statisticalbiotechnology/diffacto</td>
      <td>Proteomics</td>
      <td>diffacto</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/diffacto</td>
    </tr>
    <tr>
      <td></td>
      <td>dropletutils</td>
      <td>1.0.4</td>
      <td>0.0.5</td>
      <td>dropletutils-scripts</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>De-composed DropletUtils functionality tools, based on https://github.com/ebi-gene-expression-group/dropletutils-scripts and DropletUtils 1.0.3</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_dropletutils</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
    </tr>
    <tr>
      <td></td>
      <td>eggnog_mapper</td>
      <td>2.1.8</td>
      <td>2.1.12</td>
      <td>eggnog-mapper</td>
      <td>To update</td>
      <td>eggnog-mapper-v2</td>
      <td>eggNOG-mapper v2</td>
      <td>Homology-based gene prediction, Genome annotation, Fold recognition, Information extraction, Query and retrieval</td>
      <td>Metagenomics, Phylogeny, Transcriptomics, Workflows, Sequence analysis</td>
      <td>eggnog-mapper fast functional annotation of novel sequences</td>
      <td>EggNOG-mapper is a tool for fast functional annotation of novel sequences. It uses precomputed orthologous groups and phylogenies from the eggNOG database (http://eggnog5.embl.de) to transfer functional information from fine-grained orthologs only.</td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>eggnog_mapper</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/eggnog_mapper/eggnog_mapper</td>
    </tr>
    <tr>
      <td></td>
      <td>eodie</td>
      <td>1.0.2</td>
      <td></td>
      <td>eodie</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Earth Observation Data Information Extractor</td>
      <td></td>
      <td>To update</td>
      <td>https://eodie.readthedocs.io/</td>
      <td>Climate Analysis</td>
      <td>eodie</td>
      <td>climate</td>
      <td>https://gitlab.com/eetun-tiimi/EODIE</td>
    </tr>
    <tr>
      <td></td>
      <td>fasta_merge_files_and_filter_unique_sequences</td>
      <td>1.2.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Concatenate FASTA database files together</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/</td>
      <td>Fasta Manipulation</td>
      <td>fasta_merge_files_and_filter_unique_sequences</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/fasta_merge_files_and_filter_unique_sequences</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_quality_trimmer</td>
      <td>0.0.14+galaxy0</td>
      <td>0.0.14</td>
      <td>fastx_toolkit</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FASTQ trimmer based on quality</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/agordon/fastx_toolkit</td>
      <td>Fastq Manipulation</td>
      <td>fastq_quality_trimmer</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/develop/tools/qc/fastq_quality_trimmer</td>
    </tr>
    <tr>
      <td></td>
      <td>feature_alignment</td>
      <td>0.11.0</td>
      <td>0.11.0</td>
      <td>msproteomicstools</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>TRIC integrates information from all available runs via a graph-based alignment strategy</td>
      <td></td>
      <td>Up-to-date</td>
      <td></td>
      <td>Proteomics</td>
      <td>feature_alignment</td>
      <td>galaxyp</td>
      <td>https://github.com/msproteomicstools/msproteomicstools/blob/master/TRIC-README.md</td>
    </tr>
    <tr>
      <td></td>
      <td>filter_by_fasta_ids</td>
      <td>2.3</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter FASTA on the headers and/or the sequences</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Fasta Manipulation, Proteomics</td>
      <td>filter_by_fasta_ids</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/filter_by_fasta_ids</td>
    </tr>
    <tr>
      <td></td>
      <td>flashlfq</td>
      <td>1.0.3.1</td>
      <td>1.2.6</td>
      <td>flashlfq</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FlashLFQ mass-spectrometry proteomics label-free quantification</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/smith-chem-wisc/FlashLFQ</td>
      <td>Proteomics</td>
      <td>flashlfq</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/flashlfq</td>
    </tr>
    <tr>
      <td></td>
      <td>gffcompare_to_bed</td>
      <td>0.2.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter and convert a gffCompare GTF to BED</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/gpertea/gffcompare/</td>
      <td>Convert Formats</td>
      <td>gffcompare_to_bed</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/gffcompare_to_bed</td>
    </tr>
    <tr>
      <td></td>
      <td>hardklor</td>
      <td>2.30.1+galaxy1</td>
      <td>2.3.2</td>
      <td>hardklor</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Hardklör</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>hardklor</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tools/hardklor</td>
    </tr>
    <tr>
      <td></td>
      <td>idconvert</td>
      <td></td>
      <td>3_0_9992</td>
      <td>proteowizard</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert mass spectrometry identification files on linux or MacOSX</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>idconvert</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msconvert</td>
    </tr>
    <tr>
      <td></td>
      <td>lfq_protein_quant</td>
      <td>1.0</td>
      <td>2.28.1</td>
      <td>bioconductor-msnbase</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Enable protein summarisation and quantitation</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/compomics/LFQ_galaxy_p</td>
      <td>Proteomics</td>
      <td>lfq_protein_quant</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/lfq_protein_quant</td>
    </tr>
    <tr>
      <td></td>
      <td>lumpy_sv</td>
      <td>1.2.2</td>
      <td>0.3.1</td>
      <td>lumpy-sv</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Find structural variations</td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Variant Analysis</td>
      <td>lumpy_sv</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/lumpy_sv</td>
    </tr>
    <tr>
      <td></td>
      <td>manta</td>
      <td>1.6</td>
      <td>1.19</td>
      <td>samtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Structural variant and indel caller for mapped sequencing data</td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Variant Analysis</td>
      <td>manta</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/manta</td>
    </tr>
    <tr>
      <td></td>
      <td>map_peptides_to_bed</td>
      <td>0.2</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Map peptides to a reference genome for display by a genome browser</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>map_peptides_to_bed</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/map_peptides_to_bed</td>
    </tr>
    <tr>
      <td></td>
      <td>maxquant</td>
      <td>2.0.3.0</td>
      <td>2.0.3.0</td>
      <td>maxquant</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>wrapper for MaxQuant</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/maxquant</td>
      <td>Proteomics</td>
      <td>maxquant</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/maxquant</td>
    </tr>
    <tr>
      <td></td>
      <td>meta_proteome_analyzer</td>
      <td>2.0.0</td>
      <td>2.0.0</td>
      <td>mpa-portable</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MetaProteomeAnalyzer</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/compomics/meta-proteome-analyzer/</td>
      <td>Proteomics</td>
      <td>meta_proteome_analyzer</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/meta_proteome_analyzer</td>
    </tr>
    <tr>
      <td></td>
      <td>metagene_annotator</td>
      <td>1.0.0</td>
      <td>1.0</td>
      <td>metagene_annotator</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MetaGeneAnnotator gene-finding program for prokaryote and phage</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>metagene_annotator</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/metagene_annotator</td>
    </tr>
    <tr>
      <td></td>
      <td>metanovo</td>
      <td>1.9.4</td>
      <td>1.9.4</td>
      <td>metanovo</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Produce targeted databases for mass spectrometry analysis.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/uct-cbio/proteomics-pipelines</td>
      <td>Proteomics</td>
      <td>metanovo</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/metanovo</td>
    </tr>
    <tr>
      <td></td>
      <td>morpheus</td>
      <td>2.255.0</td>
      <td>287</td>
      <td>morpheus</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Morpheus MS Search Application</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>morpheus</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/morpheus</td>
    </tr>
    <tr>
      <td></td>
      <td>msconvert</td>
      <td>3.0.20287</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>msconvert Convert and/or filter mass spectrometry files (including vendor formats) using the official Docker container</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>msconvert</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msconvert</td>
    </tr>
    <tr>
      <td></td>
      <td>msgfplus</td>
      <td>0.5</td>
      <td>2023.01.1202</td>
      <td>msgf_plus</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MSGF+</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>msgfplus</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msgfplus</td>
    </tr>
    <tr>
      <td></td>
      <td>msmetaenhancer</td>
      <td>0.3.0</td>
      <td>0.3.0</td>
      <td>msmetaenhancer</td>
      <td>Up-to-date</td>
      <td>msmetaenhancer</td>
      <td>MSMetaEnhancer</td>
      <td>Annotation, Standardisation and normalisation</td>
      <td>Metabolomics, Compound libraries and screening, Data submission, annotation and curation</td>
      <td></td>
      <td>Tool for mass spectra metadata annotation.</td>
      <td>Up-to-date</td>
      <td>https://github.com/RECETOX/MSMetaEnhancer</td>
      <td>Metabolomics</td>
      <td></td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/msmetaenhancer</td>
    </tr>
    <tr>
      <td></td>
      <td>msms_extractor</td>
      <td>1.0.0</td>
      <td>3_0_9992</td>
      <td>proteowizard</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Extract MS/MS scans from the mzML file(s) based on PSM report.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>msms_extractor</td>
      <td>galaxyp</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>msstats</td>
      <td>4.0.0</td>
      <td>4.10.0</td>
      <td>bioconductor-msstats</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MSstats tool for analyzing mass spectrometry proteomic datasets</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/MeenaChoi/MSstats</td>
      <td>Proteomics</td>
      <td></td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msstats</td>
    </tr>
    <tr>
      <td></td>
      <td>msstatstmt</td>
      <td>2.0.0</td>
      <td>2.10.0</td>
      <td>bioconductor-msstatstmt</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MSstatsTMT protein significance analysis in shotgun mass spectrometry-based proteomic experiments with tandem mass tag (TMT) labeling</td>
      <td></td>
      <td>To update</td>
      <td>http://msstats.org/msstatstmt/</td>
      <td>Proteomics</td>
      <td>msstatstmt</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msstatstmt</td>
    </tr>
    <tr>
      <td></td>
      <td>mt2mq</td>
      <td>1.1.0</td>
      <td></td>
      <td>r-tidyverse</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tool to prepare metatranscriptomic outputs from ASaiM for Metaquantome</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>mt2mq</td>
      <td>galaxyp</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>mz_to_sqlite</td>
      <td>2.1.1+galaxy0</td>
      <td>2.1.1</td>
      <td>mztosqlite</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Creates a SQLite database for proteomics data</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>mz_to_sqlite</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/mz_to_sqlite</td>
    </tr>
    <tr>
      <td></td>
      <td>obisindicators</td>
      <td>0.0.2</td>
      <td></td>
      <td>r-base</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute biodiveristy indicators for marine data from obis</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Ecology</td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/obisindicators</td>
    </tr>
    <tr>
      <td></td>
      <td>packmol</td>
      <td>18.169.1</td>
      <td></td>
      <td>packmol</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>PACKMOL is a package for creating starting structures for Molecular Dynamics simulations</td>
      <td></td>
      <td>To update</td>
      <td>http://m3g.iqm.unicamp.br/packmol/home.shtml</td>
      <td>Molecular Dynamics, Computational chemistry</td>
      <td>packmol</td>
      <td>chemteam</td>
      <td>https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem</td>
    </tr>
    <tr>
      <td></td>
      <td>pathifier</td>
      <td>1.0.2</td>
      <td>1.3.2</td>
      <td>r-optparse</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>pathifier</td>
      <td></td>
      <td>To update</td>
      <td>https://</td>
      <td>Transcriptomics, Statistics</td>
      <td>pathifier</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/pathifier</td>
    </tr>
    <tr>
      <td></td>
      <td>pep_pointer</td>
      <td>0.1.3+galaxy1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>PepPointer categorizes peptides by their genomic coordinates.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Genomic Interval Operations, Proteomics</td>
      <td>pep_pointer</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pep_pointer</td>
    </tr>
    <tr>
      <td></td>
      <td>pepquery</td>
      <td>1.6.2</td>
      <td>2.0.2</td>
      <td>pepquery</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A peptide-centric MS search engine for novel peptide identification and validation.</td>
      <td></td>
      <td>To update</td>
      <td>https://pepquery.org</td>
      <td>Proteomics</td>
      <td>pepquery</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pepquery</td>
    </tr>
    <tr>
      <td></td>
      <td>pepquery2</td>
      <td>2.0.2</td>
      <td>2.0.2</td>
      <td>pepquery</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>PepQuery2 peptide-centric MS search for peptide identification and validation</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://pepquery.org</td>
      <td>Proteomics</td>
      <td>pepquery2</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pepquery2</td>
    </tr>
    <tr>
      <td></td>
      <td>peptide_genomic_coordinate</td>
      <td>1.0.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Gets genomic coordinate of peptides based on the information in mzsqlite and genomic mapping sqlite files</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>peptide_genomic_coordinate</td>
      <td>galaxyp</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>percolator</td>
      <td>3.5</td>
      <td>3.5</td>
      <td>percolator</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Percolator</td>
      <td></td>
      <td>Up-to-date</td>
      <td></td>
      <td>Proteomics</td>
      <td>percolator</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tools/percolator</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_joinquantpepionswithproteins</td>
      <td>0.0.1</td>
      <td>0.0.2</td>
      <td>proteomiqon-joinquantpepionswithproteins</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>The tool JoinQuantPepIonsWithProteins combines results from ProteinInference and PSMBasedQuantification.</td>
      <td></td>
      <td>To update</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/JoinQuantPepIonsWithProteins.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_joinquantpepionswithproteins</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_joinquantpepionswithproteins</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_labeledproteinquantification</td>
      <td>0.0.1</td>
      <td>0.0.3</td>
      <td>proteomiqon-labeledproteinquantification</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>The tool LabeledProteinQuantification estimates protein abundances using quantified peptide ions.</td>
      <td></td>
      <td>To update</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/LabeledProteinQuantification.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_labeledproteinquantification</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_labeledproteinquantification</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_labelfreeproteinquantification</td>
      <td>0.0.1</td>
      <td>0.0.3</td>
      <td>proteomiqon-labelfreeproteinquantification</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>The tool LabelFreeProteinQuantification estimates protein abundances using quantified peptide ions.</td>
      <td></td>
      <td>To update</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/LabelfreeProteinQuantification.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_labelfreeproteinquantification</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_labelfreeproteinquantification</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_mzmltomzlite</td>
      <td>0.0.8</td>
      <td>0.0.8</td>
      <td>proteomiqon-mzmltomzlite</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>The tool MzMLToMzLite allows to convert mzML files to mzLite files.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/MzMLToMzLite.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_mzmltomzlite</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomIQon_MzMLToMzLite</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_peptidedb</td>
      <td>0.0.7</td>
      <td>0.0.7</td>
      <td>proteomiqon-peptidedb</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>The tool ProteomIQon PeptideDB creates a peptide database in the SQLite format.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/PeptideDB.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_peptidedb</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_peptidedb</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_peptidespectrummatching</td>
      <td>0.0.7</td>
      <td>0.0.7</td>
      <td>proteomiqon-peptidespectrummatching</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Given raw an MS run in the mzLite format, this tool iterates across all MS/MS scans, determines precursor charge states and possible peptide spectrum matches using reimplementations of SEQUEST,Andromeda and XTandem.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/PeptideSpectrumMatching.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_peptidespectrummatching</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_peptidespectrummatching</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_proteininference</td>
      <td>0.0.7</td>
      <td>0.0.7</td>
      <td>proteomiqon-proteininference</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MS-based shotgun proteomics estimates protein abundances using a proxy: peptides. The process of 'Protein Inference' is concerned with the mapping of identified peptides to the proteins they putatively originated from.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/ProteinInference.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_proteininference</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_proteininference</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_psmbasedquantification</td>
      <td>0.0.8</td>
      <td>0.0.9</td>
      <td>proteomiqon-psmbasedquantification</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>The PSMBasedQuantification tool was designed to allow label-free quantification as well as quantification of full metabolic labeled samples.</td>
      <td></td>
      <td>To update</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/PSMBasedQuantification.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_psmbasedquantification</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_psmbasedquantification</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_psmstatistics</td>
      <td>0.0.8</td>
      <td>0.0.8</td>
      <td>proteomiqon-psmstatistics</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>The PSMStatistics tool utilizes semi supervised machine learning techniques to integrate search engine scores as well as the mentioned quality scores into one single consensus score.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/PSMStatistics.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_psmstatistics</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_psmstatistics</td>
    </tr>
    <tr>
      <td></td>
      <td>quantp</td>
      <td>1.1.2</td>
      <td>1.11.6</td>
      <td>r-data.table</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Correlation between protein and transcript abundance</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>quantp</td>
      <td>galaxyp</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>quantwiz_iq</td>
      <td>2.0</td>
      <td>2.0</td>
      <td>quantwiz-iq</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Isobaric Quantitation using QuantWiz-IQ</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://sourceforge.net/projects/quantwiz/</td>
      <td>Proteomics</td>
      <td>quantwiz_iq</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/quantwiz_iq</td>
    </tr>
    <tr>
      <td></td>
      <td>qupath_roi_splitter</td>
      <td>0.1.0+galaxy1</td>
      <td></td>
      <td>geojson</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Split ROI coordinates of QuPath TMA annotation by cell type</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/npinter/ROIsplitter</td>
      <td>Imaging</td>
      <td>qupath_roi_splitter</td>
      <td>galaxyp</td>
      <td>hhttps://github.com/npinter/ROIsplitter</td>
    </tr>
    <tr>
      <td></td>
      <td>ramclustr</td>
      <td>1.3.0</td>
      <td>1.3.1</td>
      <td>r-ramclustr</td>
      <td>To update</td>
      <td>ramclustr</td>
      <td>RAMClustR</td>
      <td>Imputation, Standardisation and normalisation, Clustering, Correlation</td>
      <td>Metabolomics</td>
      <td></td>
      <td>A feature clustering algorithm for non-targeted mass spectrometric metabolomics data.</td>
      <td>To update</td>
      <td>https://rdrr.io/cran/RAMClustR/</td>
      <td>Metabolomics</td>
      <td></td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/ramclustr</td>
    </tr>
    <tr>
      <td></td>
      <td>rawtools</td>
      <td></td>
      <td>2.0.4</td>
      <td>rawtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Raw Tools</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/kevinkovalchik/RawTools</td>
      <td>Proteomics</td>
      <td>rawtools</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/rawtools</td>
    </tr>
    <tr>
      <td></td>
      <td>revertR2orientationInBam</td>
      <td>0.0.2</td>
      <td>1.19</td>
      <td>samtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Revert the mapped orientation of R2 mates in a bam.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>SAM</td>
      <td>revertr2orientationinbam</td>
      <td>lldelisle</td>
      <td>https://github.com/lldelisle/tools-lldelisle/tree/master/tools/revertR2orientationInBam</td>
    </tr>
    <tr>
      <td></td>
      <td>riassigner</td>
      <td>0.3.4</td>
      <td>0.3.4</td>
      <td>riassigner</td>
      <td>Up-to-date</td>
      <td>riassigner</td>
      <td>RIAssigner</td>
      <td>Standardisation and normalisation</td>
      <td>Metabolomics, Compound libraries and screening, Data submission, annotation and curation</td>
      <td></td>
      <td>RIAssigner is a python tool for retention index (RI) computation for GC-MS data.</td>
      <td>Up-to-date</td>
      <td>https://github.com/RECETOX/RIAssigner</td>
      <td>Metabolomics</td>
      <td></td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/riassigner</td>
    </tr>
    <tr>
      <td></td>
      <td>seurat</td>
      <td>0.3.0</td>
      <td>4.0.0</td>
      <td>seurat-scripts</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>De-composed Seurat functionality tools, based on https://github.com/ebi-gene-expression-group/r-seurat-scripts and Seurat 2.3.1</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_seurat</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
    </tr>
    <tr>
      <td></td>
      <td>translate_bed</td>
      <td>0.1.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Translate BED transcript CDS or cDNA in 3 frames</td>
      <td></td>
      <td>To update</td>
      <td>http://rest.ensembl.org/</td>
      <td>Proteomics</td>
      <td>translate_bed</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteogenomics/translate_bed</td>
    </tr>
    <tr>
      <td></td>
      <td>translate_bed_sequences</td>
      <td>0.2.0</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Perform 3 frame translation of BED file augmented with a sequence column</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>translate_bed_sequences</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/translate_bed_sequences</td>
    </tr>
    <tr>
      <td></td>
      <td>unipept</td>
      <td>4.5.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Unipept retrieves metaproteomics information</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp</td>
      <td>Proteomics</td>
      <td>unipept</td>
      <td>galaxyp</td>
      <td>https://unipept.ugent.be/apidocs</td>
    </tr>
    <tr>
      <td></td>
      <td>uniprotxml_downloader</td>
      <td>2.4.0</td>
      <td></td>
      <td>requests</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Download UniProt proteome in XML or fasta format</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>uniprotxml_downloader</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/uniprotxml_downloader</td>
    </tr>
    <tr>
      <td></td>
      <td>validate_fasta_database</td>
      <td>0.1.5</td>
      <td>1.0</td>
      <td>validate-fasta-database</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>runs Compomics database identification tool on any FASTA database, and separates valid and invalid entries based on a series of checks.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Fasta Manipulation, Proteomics</td>
      <td>validate_fasta_database</td>
      <td>galaxyp</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>xmlstarlet</td>
      <td>1.6.1</td>
      <td></td>
      <td>xmlstarlet</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tool to convert a xml file from one metadata standard to another</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Convert Formats</td>
      <td>xmlstarlet</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/main/tools-ecology/tools/xmlstarlet</td>
    </tr>
  </tbody>
</table>

</div>