<script src="https://code.jquery.com/jquery-3.7.0.js"></script>
<script src="https://cdn.datatables.net/1.13.6/js/jquery.dataTables.min.js"></script>
<script src="https://cdn.datatables.net/searchbuilder/1.6.0/js/dataTables.searchBuilder.min.js"></script>
<script src="https://cdn.datatables.net/datetime/1.5.1/js/dataTables.dateTime.min.js"></script>

<script src="https://cdn.datatables.net/responsive/2.5.0/js/dataTables.responsive.min.js"></script>

<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/responsive/2.2.3/css/responsive.dataTables.min.css">

<link rel="stylesheet" href="https://cdn.datatables.net/1.13.6/css/jquery.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/searchbuilder/1.6.0/css/searchBuilder.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/datetime/1.5.1/css/dataTables.dateTime.min.css">

<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" />
<script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>

<style type="text/css">
  .tooltip {
    font-family: Georgia;
    font-size: 18px;
  }
  .tooltip .tooltip-inner {
    background-color: #00FFFF;
    color: #000000;
    min-width: 400px;
  }
  </style>

<script>
    $(document).ready(function() {
    $('#dataframe').DataTable( {
        "sScrollX": "100%",
        "dom": "Qlfrtip",

        responsive: {
            details: {
                type: 'column'
            }
        },
        "bAutoWidth": false,
        columnDefs: [ {
            className: 'control',
            orderable: false,
            targets:   [ 0 ],
            width: 100,
        },
        { type: 'natural', targets: 1 } // allow normal sorting
        ],

    });
});
</script>

<div class="display nowrap" style="width:80%" width="100%">

<table class="dataframe display nowrap" id="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th>Expand</th>
      <th>Galaxy wrapper id</th>
      <th>Galaxy wrapper version</th>
      <th>Conda version</th>
      <th>Conda id</th>
      <th>Status</th>
      <th>bio.tool id</th>
      <th>bio.tool name</th>
      <th>EDAM operation</th>
      <th>EDAM topic</th>
      <th>Description</th>
      <th>bio.tool description</th>
      <th>Status</th>
      <th>Source</th>
      <th>ToolShed categories</th>
      <th>ToolShed id</th>
      <th>Galaxy wrapper owner</th>
      <th>Galaxy wrapper source</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td></td>
      <td>Ensembl-REST</td>
      <td>0.1.2</td>
      <td></td>
      <td>requests</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A suite of Galaxy tools designed to work with Ensembl REST API.</td>
      <td></td>
      <td>To update</td>
      <td>https://rest.ensembl.org</td>
      <td>Data Source</td>
      <td></td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/Ensembl-REST</td>
    </tr>
    <tr>
      <td></td>
      <td>GAFA</td>
      <td>0.3.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Gene Align and Family Aggregator</td>
      <td></td>
      <td>To update</td>
      <td>http://aequatus.tgac.ac.uk</td>
      <td>Visualization</td>
      <td>gafa</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/GAFA/</td>
    </tr>
    <tr>
      <td></td>
      <td>TreeBest</td>
      <td>1.9.2.post0</td>
      <td>1.9.2_ep78</td>
      <td>treebest</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>TreeBeST best</td>
      <td></td>
      <td>To update</td>
      <td>http://treesoft.sourceforge.net/treebest.shtml</td>
      <td>Phylogenetics</td>
      <td>treebest_best</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/TreeBest</td>
    </tr>
    <tr>
      <td></td>
      <td>apoc</td>
      <td>1.0+galaxy1</td>
      <td>1b16</td>
      <td>apoc</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Large-scale structural comparison of protein pockets</td>
      <td></td>
      <td>To update</td>
      <td>http://cssb.biology.gatech.edu/APoc</td>
      <td>Computational chemistry</td>
      <td>apoc</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/apoc/</td>
    </tr>
    <tr>
      <td></td>
      <td>blast_parser</td>
      <td>0.1.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert 12- or 24-column BLAST output into 3-column hcluster_sg input</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/TGAC/earlham-galaxytools/</td>
      <td>Phylogenetics</td>
      <td>blast_parser</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/blast_parser</td>
    </tr>
    <tr>
      <td></td>
      <td>ete</td>
      <td>3.1.2</td>
      <td>3.1.1</td>
      <td>ete3</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Analyse phylogenetic trees using the ETE Toolkit</td>
      <td></td>
      <td>To update</td>
      <td>http://etetoolkit.org/</td>
      <td>Phylogenetics</td>
      <td>ete</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/ete</td>
    </tr>
    <tr>
      <td></td>
      <td>export_to_cluster</td>
      <td>0.0.2</td>
      <td></td>
      <td>EXPORT_DIR_PREFIX</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Export datasets to cluster</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/TGAC/earlham-galaxytools/</td>
      <td>Data Export</td>
      <td>export_to_cluster</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/export_to_cluster/</td>
    </tr>
    <tr>
      <td></td>
      <td>gblocks</td>
      <td>0.91b</td>
      <td>0.91b</td>
      <td>gblocks</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Gblocks</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://molevol.cmima.csic.es/castresana/Gblocks.html</td>
      <td>Sequence Analysis</td>
      <td>gblocks</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/gblocks</td>
    </tr>
    <tr>
      <td></td>
      <td>gstf_preparation</td>
      <td>0.4.3</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>GeneSeqToFamily preparation converts data for the workflow</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/TGAC/earlham-galaxytools/</td>
      <td>Convert Formats</td>
      <td>gstf_preparation</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/gstf_preparation</td>
    </tr>
    <tr>
      <td></td>
      <td>hcluster_sg</td>
      <td>0.5.1.1</td>
      <td>0.5.1</td>
      <td>hcluster_sg</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Hierarchically clustering on a sparse graph</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/douglasgscofield/hcluster</td>
      <td>Phylogenetics</td>
      <td>hcluster_sg</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/hcluster_sg</td>
    </tr>
    <tr>
      <td></td>
      <td>hcluster_sg_parser</td>
      <td>0.2.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Converts hcluster_sg 3-column output into lists of ids</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/TGAC/earlham-galaxytools/</td>
      <td>Phylogenetics</td>
      <td>hcluster_sg_parser</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/hcluster_sg_parser</td>
    </tr>
    <tr>
      <td></td>
      <td>lotus2</td>
      <td>2.28.1</td>
      <td>2.30</td>
      <td>lotus2</td>
      <td>To update</td>
      <td>lotus2</td>
      <td>lotus2</td>
      <td>Sequence feature detection</td>
      <td>Metagenomics</td>
      <td>LotuS2 OTU processing pipeline</td>
      <td>LotuS2 is a lightweight and user-friendly pipeline that is fast, precise, and streamlined, using extensive pre- and post-ASV/OTU clustering steps to further increase data quality. High data usage rates and reliability enable high-throughput microbiome analysis in minutes.</td>
      <td>To update</td>
      <td>http://lotus2.earlham.ac.uk/</td>
      <td>Metagenomics</td>
      <td>lotus2</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/lotus2</td>
    </tr>
    <tr>
      <td></td>
      <td>miranda</td>
      <td>3.3a+galaxy1</td>
      <td>3.3a</td>
      <td>miranda</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Finds potential target sites for miRNAs in genomic sequences</td>
      <td></td>
      <td>To update</td>
      <td>http://www.microrna.org/</td>
      <td>RNA</td>
      <td>miranda</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/miranda</td>
    </tr>
    <tr>
      <td></td>
      <td>plotheatmap</td>
      <td>1.0</td>
      <td>1.64.0</td>
      <td>bioconductor-preprocesscore</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>This tool can be used to plot heatmap of gene expression data. The genes are chosen based on p-value, FDR, log FC and log CPM from edgeR output.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Computational chemistry</td>
      <td>plotheatmap</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/plotheatmap</td>
    </tr>
    <tr>
      <td></td>
      <td>rdock</td>
      <td>1.0</td>
      <td>2013.1</td>
      <td>rDock</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Docking ligands to proteins and nucleic acids</td>
      <td></td>
      <td>To update</td>
      <td>http://rdock.sourceforge.net/</td>
      <td>Computational chemistry</td>
      <td>rdock</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/rdock/</td>
    </tr>
    <tr>
      <td></td>
      <td>replace_chromosome_names</td>
      <td>0.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Replace chromosome names</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Text Manipulation</td>
      <td>replace_chromosome_names</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/replace_chromosome_names/</td>
    </tr>
    <tr>
      <td></td>
      <td>rsat_filter_snps</td>
      <td>0.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter SNPs in RSAT Matrix Scan output</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/TGAC/earlham-galaxytools/</td>
      <td>ChIP-seq, Systems Biology</td>
      <td>rsat_filter_snps</td>
      <td>earlham</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/rsat_filter_snps</td>
    </tr>
    <tr>
      <td></td>
      <td>smart_domains</td>
      <td>0.1.0</td>
      <td>1.7.8</td>
      <td>perl-bioperl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SMART domains</td>
      <td></td>
      <td>To update</td>
      <td>http://smart.embl.de/</td>
      <td>Sequence Analysis</td>
      <td>smart_domains</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/smart_domains</td>
    </tr>
    <tr>
      <td></td>
      <td>smina</td>
      <td>1.0</td>
      <td>2017.11.9</td>
      <td>smina</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>smina is a fork of Autodock Vina (http://vina.scripps.edu/) that focuses on improving scoring and minimization</td>
      <td></td>
      <td>To update</td>
      <td>https://sourceforge.net/projects/smina/</td>
      <td>Computational chemistry</td>
      <td>smina</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/smina/</td>
    </tr>
    <tr>
      <td></td>
      <td>t_coffee</td>
      <td>13.45.0.4846264</td>
      <td>13.46.0.919e8c6b</td>
      <td>t-coffee</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>T-Coffee</td>
      <td></td>
      <td>To update</td>
      <td>http://www.tcoffee.org/</td>
      <td>Sequence Analysis</td>
      <td>t_coffee</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/t_coffee</td>
    </tr>
  </tbody>
</table>

</div>