<script src="https://code.jquery.com/jquery-3.7.0.js"></script>
<script src="https://cdn.datatables.net/1.13.6/js/jquery.dataTables.min.js"></script>
<script src="https://cdn.datatables.net/searchbuilder/1.6.0/js/dataTables.searchBuilder.min.js"></script>
<script src="https://cdn.datatables.net/datetime/1.5.1/js/dataTables.dateTime.min.js"></script>

<script src="https://cdn.datatables.net/responsive/2.5.0/js/dataTables.responsive.min.js"></script>

<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/responsive/2.2.3/css/responsive.dataTables.min.css">

<link rel="stylesheet" href="https://cdn.datatables.net/1.13.6/css/jquery.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/searchbuilder/1.6.0/css/searchBuilder.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/datetime/1.5.1/css/dataTables.dateTime.min.css">

<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" />
<script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>

<style type="text/css">
  .tooltip {
    font-family: Georgia;
    font-size: 18px;
  }
  .tooltip .tooltip-inner {
    background-color: #00FFFF;
    color: #000000;
    min-width: 400px;
  }
  </style>

<script>
    $(document).ready(function() {
    $('#dataframe').DataTable( {
        "sScrollX": "100%",
        "dom": "Qlfrtip",

        responsive: {
            details: {
                type: 'column'
            }
        },
        "bAutoWidth": false,
        columnDefs: [ {
            className: 'control',
            orderable: false,
            targets:   [ 0 ],
            width: 100,
        },
        { type: 'natural', targets: 1 } // allow normal sorting
        ],

    });
});
</script>

<div class="display nowrap" style="width:80%" width="100%">

<table class="dataframe display nowrap" id="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th>Expand</th>
      <th>Suite ID</th>
      <th>Tool IDs</th>
      <th>Description</th>
      <th>Suite first commit date</th>
      <th>Homepage</th>
      <th>Suite version</th>
      <th>Suite conda package</th>
      <th>Latest suite conda package version</th>
      <th>Suite version status</th>
      <th>ToolShed categories</th>
      <th>EDAM operations</th>
      <th>EDAM reduced operations</th>
      <th>EDAM topics</th>
      <th>EDAM reduced topics</th>
      <th>Suite owner</th>
      <th>Suite source</th>
      <th>Suite parsed folder</th>
      <th>bio.tool ID</th>
      <th>bio.tool name</th>
      <th>bio.tool description</th>
      <th>biii ID</th>
      <th>Number of tools on UseGalaxy.org (Main)</th>
      <th>Number of tools on UseGalaxy.org.au</th>
      <th>Number of tools on UseGalaxy.eu</th>
      <th>Number of tools on UseGalaxy.fr</th>
      <th>Number of tools on APOSTL</th>
      <th>Number of tools on BioBix</th>
      <th>Number of tools on CIRM-CFBP</th>
      <th>Number of tools on ChemFlow</th>
      <th>Number of tools on Coloc-stats</th>
      <th>Number of tools on CorGAT</th>
      <th>Number of tools on FreeBioinfo</th>
      <th>Number of tools on GASLINI</th>
      <th>Number of tools on Galaxy@AuBi</th>
      <th>Number of tools on Galaxy@Pasteur</th>
      <th>Number of tools on GalaxyTrakr</th>
      <th>Number of tools on Genomic Hyperbrowser</th>
      <th>Number of tools on HyPhy HIV NGS Tools</th>
      <th>Number of tools on IPK Galaxy Blast Suite</th>
      <th>Number of tools on ImmPort Galaxy</th>
      <th>Number of tools on InteractoMIX</th>
      <th>Number of tools on Lebanese University Galaxy</th>
      <th>Number of tools on MISSISSIPPI</th>
      <th>Number of tools on Mandoiu Lab</th>
      <th>Number of tools on MiModD NacreousMap</th>
      <th>Number of tools on Palfinder</th>
      <th>Number of tools on PepSimili</th>
      <th>Number of tools on PhagePromotor</th>
      <th>Number of tools on UseGalaxy.be</th>
      <th>Number of tools on UseGalaxy.cz</th>
      <th>Number of tools on UseGalaxy.no</th>
      <th>Number of tools on Viral Variant Visualizer (VVV)</th>
      <th>Suite users (last 5 years) (usegalaxy.eu)</th>
      <th>Suite users (usegalaxy.eu)</th>
      <th>Suite runs (last 5 years) (usegalaxy.eu)</th>
      <th>Suite runs (usegalaxy.eu)</th>
      <th>Suite users (last 5 years) (usegalaxy.org)</th>
      <th>Suite users (usegalaxy.org)</th>
      <th>Suite runs (last 5 years) (usegalaxy.org)</th>
      <th>Suite runs (usegalaxy.org)</th>
      <th>Suite users (last 5 years) (usegalaxy.org.au)</th>
      <th>Suite users (usegalaxy.org.au)</th>
      <th>Suite runs (last 5 years) (usegalaxy.org.au)</th>
      <th>Suite runs (usegalaxy.org.au)</th>
      <th>Suite users (last 5 years) (usegalaxy.fr)</th>
      <th>Suite users (usegalaxy.fr)</th>
      <th>Suite runs (last 5 years) (usegalaxy.fr)</th>
      <th>Suite runs (usegalaxy.fr)</th>
      <th>Suite runs on main servers</th>
      <th>Suite runs (last 5 years) on main servers</th>
      <th>Suite users on main servers</th>
      <th>Suite users (last 5 years) on main servers</th>
      <th>Related Workflows</th>
      <th>Related Tutorials</th>
      <th>To keep</th>
      <th>Deprecated</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td></td>
      <td>CRSConverter</td>
      <td>crsconverter</td>
      <td>Tools to transforms geographical vector data to a specified coordinate reference system.</td>
      <td>2025-05-20</td>
      <td>https://github.com/Morphy427/phylodiversity_workflow/CRSConverter</td>
      <td>1.1</td>
      <td>r-sf</td>
      <td></td>
      <td>To update</td>
      <td>Ecology</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/Phylodiversity_workflow</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/CRSConverter</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>47</td>
      <td>47</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>47</td>
      <td>47</td>
      <td>3</td>
      <td>3</td>
      <td>https://usegalaxy.eu/published/workflow?id=89525918b2a32023</td>
      <td>ecology/phylodiversity_workflow</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>EstimEndem</td>
      <td>estimate_endem</td>
      <td>Tools to estimates the endemism of each cell in a spatial grid.</td>
      <td>2025-05-20</td>
      <td>https://github.com/Morphy427/phylodiversity_workflow/EstimEndem</td>
      <td>0.1.1+galaxy0</td>
      <td>r-phyloregion</td>
      <td></td>
      <td>To update</td>
      <td>Ecology</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/Phylodiversity_workflow</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/EstimEndem</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>40</td>
      <td>40</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>40</td>
      <td>40</td>
      <td>3</td>
      <td>3</td>
      <td>https://usegalaxy.eu/published/workflow?id=89525918b2a32023</td>
      <td>ecology/phylodiversity_workflow</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Geometric means (Dead wood)</td>
      <td>Map_shp, Mean_geom, bar_plot</td>
      <td>Tools to compute The evolution of the total volume of very large trees, standing dead wood and dead wood on the ground on an area and the rate of devolution of the volume of wood favorable to biodiversity by large ecological regions (France).</td>
      <td>2023-11-10</td>
      <td>https://github.com/PaulineSGN/Galaxy_tool_moyenne_geom</td>
      <td>0.1.0+galaxy0</td>
      <td>r-base</td>
      <td></td>
      <td>To update</td>
      <td>Ecology</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/Geom_mean_workflow</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/Geom_mean_workflow</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>60</td>
      <td>60</td>
      <td>163</td>
      <td>163</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>20</td>
      <td>20</td>
      <td>183</td>
      <td>183</td>
      <td>62</td>
      <td>62</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>PylOccuMatcher</td>
      <td>phylogenetic_occupancy_matcher</td>
      <td>Tools to matches phylogenetic data with occupancy data.</td>
      <td>2025-05-20</td>
      <td>https://github.com/Morphy427/phylodiversity_workflow/PhylOccuMatcher</td>
      <td>1.0</td>
      <td>r-phyloregion</td>
      <td></td>
      <td>To update</td>
      <td>Ecology</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/Phylodiversity_workflow</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/PhylOccuMatcher</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>22</td>
      <td>22</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>22</td>
      <td>22</td>
      <td>4</td>
      <td>4</td>
      <td>https://usegalaxy.eu/published/workflow?id=89525918b2a32023</td>
      <td>ecology/phylodiversity_workflow</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>PyloIndex</td>
      <td>phylo_index</td>
      <td>Tools to computes the phylodiversity index.</td>
      <td>2025-05-20</td>
      <td>https://github.com/Morphy427/phylodiversity_workflow/PyloIndex</td>
      <td>1.0</td>
      <td>r-phyloregion</td>
      <td></td>
      <td>To update</td>
      <td>Ecology</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/Phylodiversity_workflow</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/PhyloIndex</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>18</td>
      <td>18</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>18</td>
      <td>18</td>
      <td>5</td>
      <td>5</td>
      <td>https://usegalaxy.eu/published/workflow?id=89525918b2a32023</td>
      <td>ecology/phylodiversity_workflow</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>WormsMeasurements</td>
      <td>WormsMeasurements</td>
      <td>This tool uses WoRMS (World Register of Marine Species) to add measurement values to a dataset based on scientific names.</td>
      <td>2025-05-07</td>
      <td>https://github.com/jeanlecras/tools-ecology</td>
      <td>0.1.2</td>
      <td>r-base</td>
      <td></td>
      <td>To update</td>
      <td>Ecology</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/jeanlecras/tools-ecology/tree/master/tools/WormsMeasurements</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/WormsMeasurements</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>97</td>
      <td>97</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>97</td>
      <td>97</td>
      <td>4</td>
      <td>4</td>
      <td>https://usegalaxy.eu/published/workflow?id=749bb05ec8674bec</td>
      <td>ecology/ecoregiolifetraits</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>abacas</td>
      <td>abacas</td>
      <td>Order and Orientate Contigs</td>
      <td>2019-11-20</td>
      <td>https://github.com/phac-nml/abacas</td>
      <td>1.1</td>
      <td>mummer</td>
      <td>3.23</td>
      <td>To update</td>
      <td>Assembly</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>nml</td>
      <td>https://github.com/phac-nml/abacas</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/abacas</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>2</td>
      <td>2</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>abyss</td>
      <td>abyss-pe</td>
      <td>Assembly By Short Sequences - a de novo, parallel, paired-end sequence assembler</td>
      <td>2015-04-14</td>
      <td>http://www.bcgsc.ca/platform/bioinfo/software/abyss</td>
      <td>2.3.10</td>
      <td>abyss</td>
      <td>2.3.10</td>
      <td>Up-to-date</td>
      <td>Assembly</td>
      <td>Genome assembly, De-novo assembly, Scaffolding</td>
      <td>Genome assembly, De-novo assembly, Scaffolding</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/abyss</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/abyss</td>
      <td>abyss</td>
      <td>ABySS</td>
      <td>De novo genome sequence assembler using short reads.</td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1258</td>
      <td>1368</td>
      <td>7148</td>
      <td>7779</td>
      <td>183</td>
      <td>414</td>
      <td>505</td>
      <td>1305</td>
      <td>398</td>
      <td>398</td>
      <td>2608</td>
      <td>2608</td>
      <td>11</td>
      <td>11</td>
      <td>16</td>
      <td>16</td>
      <td>11708</td>
      <td>10277</td>
      <td>2191</td>
      <td>1850</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>True</td>
    </tr>
    <tr>
      <td></td>
      <td>assemblystats</td>
      <td>assemblystats</td>
      <td>Summarise an assembly (e.g. N50 metrics)</td>
      <td>2017-10-13</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>1.1.0</td>
      <td>perl-bioperl</td>
      <td>1.7.8</td>
      <td>To update</td>
      <td>Assembly</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/assemblystats</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>True</td>
    </tr>
    <tr>
      <td></td>
      <td>biotradis</td>
      <td>bacteria_tradis, tradis_essentiality, tradis_gene_insert_sites</td>
      <td>Bio-Tradis is a tool suite dedicated to essentiality analyses with TraDis data.</td>
      <td>2020-01-17</td>
      <td>https://www.sanger.ac.uk/science/tools/bio-tradis</td>
      <td>1.4.5</td>
      <td>biotradis</td>
      <td>1.4.5</td>
      <td>Up-to-date</td>
      <td>Genome annotation</td>
      <td>Sequence analysis</td>
      <td>Sequence analysis</td>
      <td>Mobile genetic elements, Workflows</td>
      <td>Mobile genetic elements, Workflows</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/biotradis</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/biotradis</td>
      <td>biotradis</td>
      <td>biotradis</td>
      <td>The Bio::TraDIS pipeline provides software utilities for the processing, mapping, and analysis of transposon insertion sequencing data. The pipeline was designed with the data from the TraDIS sequencing protocol in mind, but should work with a variety of transposon insertion sequencing protocols as long as they produce data in the expected format.</td>
      <td></td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>165</td>
      <td>176</td>
      <td>6369</td>
      <td>6769</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6769</td>
      <td>6369</td>
      <td>176</td>
      <td>165</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>bracken</td>
      <td>est_abundance</td>
      <td>Bayesian Reestimation of Abundance with KrakEN</td>
      <td>2019-10-15</td>
      <td>https://github.com/jenniferlu717/Bracken/releases</td>
      <td>3.1</td>
      <td>bracken</td>
      <td>3.1</td>
      <td>Up-to-date</td>
      <td>Sequence Analysis, Metagenomics</td>
      <td>Statistical calculation</td>
      <td>Statistical calculation</td>
      <td>Metagenomics, Microbial ecology</td>
      <td>Metagenomics, Microbial ecology</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bracken</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bracken</td>
      <td>bracken</td>
      <td>Bracken</td>
      <td>Bracken is a companion program to Kraken 1, KrakenUniq, or Kraken 2 While Kraken classifies reads to multiple levels in the taxonomic tree, Bracken allows estimation of abundance at a single level using those classifications (e.g. Bracken can estimate abundance of species within a sample).</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1714</td>
      <td>1714</td>
      <td>108307</td>
      <td>108307</td>
      <td>804</td>
      <td>804</td>
      <td>27569</td>
      <td>27569</td>
      <td>265</td>
      <td>265</td>
      <td>8873</td>
      <td>8873</td>
      <td>59</td>
      <td>59</td>
      <td>12432</td>
      <td>12432</td>
      <td>157181</td>
      <td>157181</td>
      <td>2842</td>
      <td>2842</td>
      <td>https://usegalaxy.eu/published/workflow?id=0a82eb8b44d9d502, https://usegalaxy.eu/published/workflow?id=0abd32dbc5207b88, https://usegalaxy.eu/published/workflow?id=0c47571d20535426, https://usegalaxy.eu/published/workflow?id=2fa2f67603772413, https://usegalaxy.eu/published/workflow?id=39d36a7ce0ef268d, https://usegalaxy.eu/published/workflow?id=47e14222073e7192, https://usegalaxy.eu/published/workflow?id=4f0daf8e9dd54127, https://usegalaxy.eu/published/workflow?id=6f878d8b43ef1521, https://usegalaxy.eu/published/workflow?id=7491883694fff308, https://usegalaxy.eu/published/workflow?id=884a4415b669690c, https://usegalaxy.eu/published/workflow?id=aafcfcb585c5ca75, https://usegalaxy.eu/published/workflow?id=b14845359b702444, https://usegalaxy.eu/published/workflow?id=bb63b5bb983fd49c, https://usegalaxy.eu/published/workflow?id=c14d2e8b3c191bc4, https://usegalaxy.eu/published/workflow?id=c930a54a30dcc7e3, https://usegalaxy.eu/published/workflow?id=cb2df493f5e334f7, https://usegalaxy.eu/published/workflow?id=ebb131f2a6172081, https://usegalaxy.fr/published/workflow?id=3d3b40398d1cd1ae, https://usegalaxy.org.au/published/workflow?id=92fcdd490831c6c5, https://usegalaxy.org.au/published/workflow?id=a335b9c9103b57ad, https://usegalaxy.org/published/workflow?id=1be5c982c2d36a6c, https://usegalaxy.org/published/workflow?id=2a782ad24271f570, https://usegalaxy.org/published/workflow?id=323c651ca583c9e5, https://usegalaxy.org/published/workflow?id=4f698eaeef30e364, https://usegalaxy.org/published/workflow?id=623fc80c7f91a33a, https://usegalaxy.org/published/workflow?id=9ba6da7dae465d3e, https://usegalaxy.org/published/workflow?id=9f68718f025bed30, https://usegalaxy.org/published/workflow?id=b6a795834009ce06, https://usegalaxy.org/published/workflow?id=bdd7011a908bc1aa, https://usegalaxy.org/published/workflow?id=de9bf4c1024f8d53, https://usegalaxy.org/published/workflow?id=dfd73857c0eb2a21, https://usegalaxy.org/published/workflow?id=f378960ee55f567e, https://workflowhub.eu/workflows/1052?version=11, https://workflowhub.eu/workflows/1199?version=2, https://workflowhub.eu/workflows/1470?version=2, https://workflowhub.eu/workflows/1644?version=1, https://workflowhub.eu/workflows/1674?version=2, https://workflowhub.eu/workflows/1881?version=2, https://workflowhub.eu/workflows/1882?version=3, https://workflowhub.eu/workflows/2043?version=1</td>
      <td>ecology/bacterial-isolate-species-contamination-checking, microbiome/taxonomic-profiling, sequence-analysis/quality-contamination-control</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>braker3</td>
      <td>braker3</td>
      <td>BRAKER3 is a pipeline for fully automated prediction of protein coding gene structures with GeneMark-ES/ET and AUGUSTUS in novel eukaryotic genomes .</td>
      <td>2023-06-13</td>
      <td>https://github.com/Gaius-Augustus/BRAKER</td>
      <td>3.0.8</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>Genome annotation</td>
      <td>Genome annotation, Gene prediction</td>
      <td>Genome annotation, Gene prediction</td>
      <td>RNA-Seq, Genomics, Structure prediction, Sequence analysis</td>
      <td>RNA-Seq, Genomics, Structure prediction, Sequence analysis</td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/braker</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/braker3</td>
      <td>braker3</td>
      <td>BRAKER3</td>
      <td>BRAKER3 is a pipeline for fully automated prediction of protein coding gene structures with GeneMark-ES/ET and AUGUSTUS in novel eukaryotic genomes</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>711</td>
      <td>711</td>
      <td>5354</td>
      <td>5354</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>135</td>
      <td>135</td>
      <td>987</td>
      <td>987</td>
      <td>26</td>
      <td>26</td>
      <td>275</td>
      <td>275</td>
      <td>6616</td>
      <td>6616</td>
      <td>872</td>
      <td>872</td>
      <td>https://workflowhub.eu/workflows/1509?version=1, https://workflowhub.eu/workflows/1746?version=1, https://workflowhub.eu/workflows/2029?version=1</td>
      <td>genome-annotation/braker3, genome-annotation/comparison-braker-helixer-annotation</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>champ_blocs</td>
      <td>cb_dissim, cb_ivr, cb_div</td>
      <td>Compute indicators for turnover boulders fields</td>
      <td>2022-10-28</td>
      <td></td>
      <td>0.0.0</td>
      <td>r-base</td>
      <td></td>
      <td>To update</td>
      <td>Ecology</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/Marie59/champ_blocs</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/champ_blocs</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>9</td>
      <td>216</td>
      <td>216</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>11</td>
      <td>11</td>
      <td>227</td>
      <td>227</td>
      <td>11</td>
      <td>11</td>
      <td>https://usegalaxy.eu/published/workflow?id=a3ad27817f77061b, https://workflowhub.eu/workflows/1690?version=1, https://workflowhub.eu/workflows/661?version=1</td>
      <td>ecology/champs-blocs</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>dada2</td>
      <td>dada2_assignTaxonomyAddspecies, dada2_dada, dada2_filterAndTrim, dada2_learnErrors, dada2_makeSequenceTable, dada2_mergePairs, dada2_plotComplexity, dada2_plotQualityProfile, dada2_primerCheck, dada2_removeBimeraDenovo, dada2_seqCounts</td>
      <td>DADA2 wrappers</td>
      <td>2019-07-05</td>
      <td>https://benjjneb.github.io/dada2/index.html</td>
      <td>1.34.0</td>
      <td>bioconductor-dada2</td>
      <td>1.34.0</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Variant calling, DNA barcoding</td>
      <td>Variant calling, DNA barcoding</td>
      <td>Sequencing, Genetic variation, Microbial ecology, Metagenomics</td>
      <td>Sequencing, Genetic variation, Microbial ecology, Metagenomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/dada2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/dada2</td>
      <td>dada2</td>
      <td>dada2</td>
      <td>This package infers exact sequence variants (SVs) from amplicon data, replacing the commonly used and coarser OTU clustering approach. This pipeline inputs demultiplexed fastq files, and outputs the sequence variants and their sample-wise abundances after removing substitution and chimera errors. Taxonomic classification is available via a native implementation of the RDP naive Bayesian classifier.</td>
      <td></td>
      <td>10</td>
      <td>10</td>
      <td>11</td>
      <td>11</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>865</td>
      <td>885</td>
      <td>205808</td>
      <td>209081</td>
      <td>1301</td>
      <td>1344</td>
      <td>254475</td>
      <td>270135</td>
      <td>179</td>
      <td>179</td>
      <td>52639</td>
      <td>52639</td>
      <td>72</td>
      <td>72</td>
      <td>38131</td>
      <td>38131</td>
      <td>569986</td>
      <td>551053</td>
      <td>2480</td>
      <td>2417</td>
      <td>https://usegalaxy.eu/published/workflow?id=48d0dc424e71d8c2, https://usegalaxy.eu/published/workflow?id=642f4c47028031d7, https://usegalaxy.eu/published/workflow?id=7b557efda89bb998, https://usegalaxy.eu/published/workflow?id=b483cf6a138d2372, https://usegalaxy.eu/published/workflow?id=e132370adf8f2838, https://usegalaxy.eu/published/workflow?id=e3918808605cf8cd, https://usegalaxy.fr/published/workflow?id=b5d86d505c09d170, https://usegalaxy.org.au/published/workflow?id=1016f17b77a8216d, https://usegalaxy.org.au/published/workflow?id=2d05f15a9616cc47, https://usegalaxy.org.au/published/workflow?id=400fbd96fb04c60e, https://usegalaxy.org/published/workflow?id=0942f2f3386e6b41, https://usegalaxy.org/published/workflow?id=38f9fc867c4d1f40, https://usegalaxy.org/published/workflow?id=4faff660f6defa10, https://usegalaxy.org/published/workflow?id=5df593510c7d944d, https://usegalaxy.org/published/workflow?id=78e60413358273bc, https://usegalaxy.org/published/workflow?id=806ef8816168916f, https://usegalaxy.org/published/workflow?id=e067c4e66f0af362, https://usegalaxy.org/published/workflow?id=ecf3a7f9da7409ef, https://workflowhub.eu/workflows/1395?version=2, https://workflowhub.eu/workflows/790?version=3</td>
      <td>microbiome/dada-16S</td>
      <td>True</td>
      <td>True</td>
    </tr>
    <tr>
      <td></td>
      <td>data_exploration</td>
      <td>tool_anonymization, ecology_homogeneity_normality, ecology_beta_diversity, ecology_link_between_var, ecology_presence_abs_abund, ecology_stat_presence_abs</td>
      <td>Explore data through multiple statistical tools</td>
      <td>2021-06-23</td>
      <td></td>
      <td>0.0.0</td>
      <td>r-tangles</td>
      <td></td>
      <td>To update</td>
      <td>Ecology</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/Marie59/Data_explo_tools</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/data_exploration</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>122</td>
      <td>122</td>
      <td>1286</td>
      <td>1286</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>44</td>
      <td>44</td>
      <td>1330</td>
      <td>1330</td>
      <td>128</td>
      <td>128</td>
      <td>https://usegalaxy.eu/published/workflow?id=7b576a562031ff7e, https://usegalaxy.eu/published/workflow?id=a0404f6291874c01, https://workflowhub.eu/workflows/1693?version=1, https://workflowhub.eu/workflows/656?version=1</td>
      <td>ecology/biodiversity-data-exploration</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>earth</td>
      <td>landcover_subindicator</td>
      <td>Access, process, visualise oceanographic data for the Earth System</td>
      <td>2024-10-21</td>
      <td>https://github.com/Marie59/FE-ft-ESG/tree/main/earth</td>
      <td>0.1.0</td>
      <td>numpy</td>
      <td></td>
      <td>To update</td>
      <td>Ecology</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/earth</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/earth</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>ecoregionalization</td>
      <td>ecoregion_brt_analysis, ecoregion_GeoNearestNeighbor, ecoregion_cluster_estimate, ClaraGuess, ecoregion_clara_cluster, ecoregion_eco_map, indicspecies, ecoregion_taxa_seeker</td>
      <td>Tools to compute ecoregionalization with BRT model predictions and clustering.</td>
      <td>2023-07-26</td>
      <td>https://github.com/PaulineSGN/Workflow_Galaxy</td>
      <td>0.1.2+galaxy0</td>
      <td>r-base</td>
      <td></td>
      <td>To update</td>
      <td>Ecology</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/Ecoregionalization_workflow</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/Ecoregionalization_workflow</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>13</td>
      <td>13</td>
      <td>987</td>
      <td>987</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>987</td>
      <td>987</td>
      <td>13</td>
      <td>13</td>
      <td>https://usegalaxy.eu/published/workflow?id=749bb05ec8674bec, https://usegalaxy.eu/published/workflow?id=89ff2a1143954d83, https://usegalaxy.eu/published/workflow?id=b727b64f37238fff, https://usegalaxy.eu/published/workflow?id=c5de0b794849e1b1, https://workflowhub.eu/workflows/1208?version=1, https://workflowhub.eu/workflows/1209?version=1, https://workflowhub.eu/workflows/1680?version=1, https://workflowhub.eu/workflows/658?version=1</td>
      <td>ecology/Ecoregionalization_tutorial, ecology/ecoregiolifetraits</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>emlassemblyline</td>
      <td>annotations_template, eal_table_template, eal_templates, eml2eal, eml_validate, entities_template, geo_cov_template, makeeml, raster_template, taxo_cov_template, vector_template</td>
      <td>Tools using EML Assembly Line R package to generate EML metadata from template metadata files and vice versa</td>
      <td>2023-07-21</td>
      <td>https://github.com/EDIorg/EMLassemblyline</td>
      <td>0.1.0+galaxy0</td>
      <td>r-base</td>
      <td></td>
      <td>To update</td>
      <td>Ecology</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/EMLassemblyline</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/EMLassemblyline</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>11</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>40</td>
      <td>40</td>
      <td>646</td>
      <td>646</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>2</td>
      <td>2</td>
      <td>648</td>
      <td>648</td>
      <td>41</td>
      <td>41</td>
      <td>https://usegalaxy.eu/published/workflow?id=a30d511e3dda0895, https://usegalaxy.eu/published/workflow?id=ab89c61db3d7abbb</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>export2graphlan</td>
      <td>export2graphlan</td>
      <td>export2graphlan is a conversion software tool for producing both annotation and tree file for GraPhlAn</td>
      <td>2017-03-03</td>
      <td>https://bitbucket.org/CibioCM/export2graphlan/overview</td>
      <td>0.20</td>
      <td>export2graphlan</td>
      <td>0.22</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Conversion</td>
      <td>Conversion</td>
      <td>Taxonomy, Metabolomics, Biomarkers</td>
      <td>Taxonomy, Metabolomics, Biomarkers</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/export2graphlan</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/export2graphlan</td>
      <td>export2graphlan</td>
      <td>export2graphlan</td>
      <td>export2graphlan is a conversion software tool for producing both annotation and tree file for GraPhlAn. In particular, the annotation file tries to highlight specific sub-trees deriving automatically from input file what nodes are important.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>589</td>
      <td>710</td>
      <td>6276</td>
      <td>7303</td>
      <td>193</td>
      <td>213</td>
      <td>712</td>
      <td>791</td>
      <td>122</td>
      <td>123</td>
      <td>616</td>
      <td>632</td>
      <td>16</td>
      <td>16</td>
      <td>43</td>
      <td>43</td>
      <td>8769</td>
      <td>7647</td>
      <td>1062</td>
      <td>920</td>
      <td>https://usegalaxy.eu/published/workflow?id=05f5f040337786bf, https://usegalaxy.eu/published/workflow?id=096b75501c8e0888, https://usegalaxy.eu/published/workflow?id=19de5cbae7f48103, https://usegalaxy.eu/published/workflow?id=2cf09adb1ed6c3ec, https://usegalaxy.eu/published/workflow?id=4d32b1b3ac0a7161, https://usegalaxy.eu/published/workflow?id=4f21205e71cabd37, https://usegalaxy.eu/published/workflow?id=5f0faac42b0a5062, https://usegalaxy.eu/published/workflow?id=713a5485e2007f90, https://usegalaxy.eu/published/workflow?id=79b4ae4403772098, https://usegalaxy.eu/published/workflow?id=7ab70660e6235cf0, https://usegalaxy.eu/published/workflow?id=87cc0cd143a12c41, https://usegalaxy.eu/published/workflow?id=994e5ff1df7d6c9e, https://usegalaxy.eu/published/workflow?id=9cbba5de2cbeeb0f, https://usegalaxy.eu/published/workflow?id=a8f829deb69b30ab, https://usegalaxy.eu/published/workflow?id=b5f848dc8e96663c, https://usegalaxy.eu/published/workflow?id=c01ef8354b585a1f, https://usegalaxy.eu/published/workflow?id=c3aa516e78ca4191, https://usegalaxy.eu/published/workflow?id=d05c6f0f3205fbd2, https://usegalaxy.eu/published/workflow?id=dfc2c53ac3103eff, https://usegalaxy.eu/published/workflow?id=ed0ad728cf21702b, https://usegalaxy.eu/published/workflow?id=fdca6c9d394986bf, https://usegalaxy.fr/published/workflow?id=09e8293456ee6d99, https://usegalaxy.fr/published/workflow?id=c4f6062e51d0f45e, https://workflowhub.eu/workflows/1451?version=2, https://workflowhub.eu/workflows/1466?version=2</td>
      <td>microbiome/metatranscriptomics, microbiome/metatranscriptomics-short</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>fairy</td>
      <td>fairy_cov, fairy_sketch</td>
      <td>Fast approximate contig coverage for metagenomic binning</td>
      <td>2024-12-20</td>
      <td>https://github.com/bluenote-1577/fairy</td>
      <td>0.5.8</td>
      <td>fairy</td>
      <td>0.5.8</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Read binning, Sequence assembly validation</td>
      <td>Read binning, Sequence assembly validation</td>
      <td>Metagenomics</td>
      <td>Metagenomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fairy</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fairy</td>
      <td>fairy</td>
      <td>fairy</td>
      <td>fast approximate contig coverage for metagenomic binningFairy computes multi-sample contig coverage for metagenome-assembled genome (MAG) binning.Fairy is used after metagenomic assembly and before binning. It can- Calculate coverage 100x-1000x faster than read alignment (e.g. BWA)- Give comparable bins for multi-sample binning (short read or nanopore reads)- Output formats that are compatible with MetaBAT2, MaxBin2, SemiBin2, and more</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>148</td>
      <td>148</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>148</td>
      <td>148</td>
      <td>5</td>
      <td>5</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>fastk</td>
      <td>fastk_fastk, fastk_histex, fastk_logex</td>
      <td>FastK: A K-mer counter (for HQ assembly data sets)</td>
      <td>2024-05-03</td>
      <td>https://github.com/thegenemyers/FASTK</td>
      <td>1.1.0</td>
      <td>fastk</td>
      <td>1.2</td>
      <td>To update</td>
      <td>Assembly</td>
      <td>k-mer counting, Sequence analysis</td>
      <td>k-mer counting, Sequence analysis</td>
      <td>Genomics, Sequence analysis</td>
      <td>Genomics, Sequence analysis</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fastk</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fastk</td>
      <td>fastk</td>
      <td>FASTK</td>
      <td>FastK is a kmer counter that is optimized for processing high quality DNA assembly data sets such as those produced with an Illumina instrument or a PacBio run in HiFi mode.</td>
      <td></td>
      <td>3</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>46</td>
      <td>46</td>
      <td>742</td>
      <td>742</td>
      <td>45</td>
      <td>45</td>
      <td>389</td>
      <td>389</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1131</td>
      <td>1131</td>
      <td>91</td>
      <td>91</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>frogs</td>
      <td>FROGS_affiliation_filters, FROGS_affiliation_postprocess, FROGS_affiliation_stats, FROGS_biom_to_stdBiom, FROGS_biom_to_tsv, FROGS_cluster_filters, FROGS_cluster_stats, FROGS_clustering, FROGS_demultiplex, FROGSSTAT_DESeq2_Preprocess, FROGSSTAT_DESeq2_Visualisation, FROGSFUNC_step2_functions, FROGSFUNC_step3_pathways, FROGSFUNC_step1_placeseqs, FROGS_itsx, FROGS_normalisation, FROGSSTAT_Phyloseq_Alpha_Diversity, FROGSSTAT_Phyloseq_Beta_Diversity, FROGSSTAT_Phyloseq_Sample_Clustering, FROGSSTAT_Phyloseq_Composition_Visualisation, FROGSSTAT_Phyloseq_Import_Data, FROGSSTAT_Phyloseq_Multivariate_Analysis_Of_Variance, FROGSSTAT_Phyloseq_Structure_Visualisation, FROGS_preprocess, FROGS_remove_chimera, FROGS_taxonomic_affiliation, FROGS_Tree, FROGS_tsv_to_biom</td>
      <td>Suite for metabarcoding analysis</td>
      <td>2021-08-16</td>
      <td>http://frogs.toulouse.inrae.fr/</td>
      <td>4.2.0</td>
      <td>frogs</td>
      <td>5.1.0</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>frogs</td>
      <td>https://github.com/geraldinepascal/FROGS-wrappers/</td>
      <td>https://github.com/geraldinepascal/FROGS-wrappers/tree/master/tools/frogs</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>28</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>18</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1271</td>
      <td>1272</td>
      <td>73623</td>
      <td>73661</td>
      <td>73661</td>
      <td>73623</td>
      <td>1272</td>
      <td>1271</td>
      <td>https://usegalaxy.fr/published/workflow?id=048cb90ec1ac7120, https://usegalaxy.fr/published/workflow?id=2ac84b2bd45876cf, https://usegalaxy.fr/published/workflow?id=a5f8c5d37ddfcf0f, https://usegalaxy.fr/published/workflow?id=b46954345641a3b8, https://usegalaxy.fr/published/workflow?id=d68b4aa5337f016b</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>funannotate</td>
      <td>funannotate_annotate, funannotate_clean, funannotate_compare, funannotate_predict, funannotate_sort</td>
      <td>Funannotate is a genome prediction, annotation, and comparison software package.</td>
      <td>2021-08-26</td>
      <td>https://funannotate.readthedocs.io</td>
      <td>1.8.17</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>Genome annotation</td>
      <td>Genome annotation</td>
      <td>Genome annotation</td>
      <td>Genomics</td>
      <td>Genomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/funannotate</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/funannotate</td>
      <td>funannotate</td>
      <td>funannotate</td>
      <td>funannotate is a pipeline for genome annotation (built specifically for fungi, but will also work with higher eukaryotes).</td>
      <td></td>
      <td>3</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>891</td>
      <td>891</td>
      <td>14118</td>
      <td>14118</td>
      <td>209</td>
      <td>209</td>
      <td>1453</td>
      <td>1453</td>
      <td>310</td>
      <td>310</td>
      <td>8597</td>
      <td>8597</td>
      <td>88</td>
      <td>88</td>
      <td>1505</td>
      <td>1505</td>
      <td>25673</td>
      <td>25673</td>
      <td>1498</td>
      <td>1498</td>
      <td>https://usegalaxy.cz/published/workflow?id=932f4901960a595a, https://usegalaxy.eu/published/workflow?id=07c92110c3fffcf8, https://usegalaxy.eu/published/workflow?id=e5becc6d66b2568c, https://usegalaxy.org.au/published/workflow?id=4e161a8f5098cc49, https://usegalaxy.org.au/published/workflow?id=52065a4100b11184, https://usegalaxy.org.au/published/workflow?id=a9068ea858222e2b, https://usegalaxy.org/published/workflow?id=7251c6e0dad8f927, https://workflowhub.eu/workflows/1525?version=1, https://workflowhub.eu/workflows/1656?version=1, https://workflowhub.eu/workflows/358?version=1, https://workflowhub.eu/workflows/754?version=1</td>
      <td>ecology/phylogeny-data-prep, genome-annotation/funannotate</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>gdal</td>
      <td>gdal_gdal_merge, gdal_gdal_translate, gdal_gdaladdo, gdal_gdalbuildvrt, gdal_gdalinfo, gdal_gdalwarp, gdal_ogr2ogr, gdal_ogrinfo</td>
      <td>Geospatial Data Abstraction Library tools are all dedicated to manipulate raster and vector geospatial data formats.</td>
      <td>2019-02-25</td>
      <td>https://www.gdal.org</td>
      <td>3.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>Ecology</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/gdal</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/gdal</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>11</td>
      <td>15</td>
      <td>724</td>
      <td>1525</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>3</td>
      <td>3</td>
      <td>1528</td>
      <td>727</td>
      <td>17</td>
      <td>13</td>
      <td>https://usegalaxy.eu/published/workflow?id=3b0803542deff026, https://usegalaxy.eu/published/workflow?id=b0b6c8756070650b, https://workflowhub.eu/workflows/1670?version=1, https://workflowhub.eu/workflows/404?version=1</td>
      <td>ecology/gbif_cleaning</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>genomad</td>
      <td>genomad_end_to_end</td>
      <td>Identify virus and plasmid genomes from nucleotide sequences</td>
      <td>2024-06-17</td>
      <td>https://github.com/apcamargo/genomad/</td>
      <td>1.11.1</td>
      <td>genomad</td>
      <td>1.11.2</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Sequence annotation, Taxonomic classification</td>
      <td>Sequence annotation, Taxonomic classification</td>
      <td>Sequence analysis</td>
      <td>Sequence analysis</td>
      <td>ufz</td>
      <td>https://github.com/Helmholtz-UFZ/ufz-galaxy-tools/blob/main/tools/genomad/</td>
      <td>https://github.com/Helmholtz-UFZ/galaxy-tools/tree/main/tools/genomad</td>
      <td>genomad</td>
      <td>geNomad</td>
      <td>geNomad is a tool that identifies virus and plasmid genomes from nucleotide sequences. It provides state-of-the-art classification performance and can be used to quickly find mobile genetic elements from genomes, metagenomes, or metatranscriptomes.</td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>119</td>
      <td>119</td>
      <td>2576</td>
      <td>2576</td>
      <td>20</td>
      <td>20</td>
      <td>71</td>
      <td>71</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2647</td>
      <td>2647</td>
      <td>139</td>
      <td>139</td>
      <td>https://usegalaxy.eu/published/workflow?id=36de05d50c703164, https://usegalaxy.eu/published/workflow?id=62392daea25a3980, https://usegalaxy.eu/published/workflow?id=c62d65832377e376</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>getorganelle</td>
      <td>get_annotated_regions_from_gb, get_organelle_from_reads</td>
      <td>GetOrganelle - This toolkit assembles organelle genomes from genomic skimming data.</td>
      <td>2022-03-15</td>
      <td>https://github.com/Kinggerm/GetOrganelle</td>
      <td>1.7.7.1</td>
      <td>getorganelle</td>
      <td>1.7.7.1</td>
      <td>Up-to-date</td>
      <td>Assembly</td>
      <td>De-novo assembly, Genome assembly, Mapping assembly, Mapping, Sequence trimming</td>
      <td>De-novo assembly, Genome assembly, Mapping assembly, Mapping, Sequence trimming</td>
      <td>Cell biology, Sequence assembly, Whole genome sequencing, Plant biology, Model organisms</td>
      <td>Cell biology, Sequence assembly, Whole genome sequencing, Plant biology, Model organisms</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/getorganelle</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/getorganelle</td>
      <td>getorganelle</td>
      <td>GetOrganelle</td>
      <td>A fast and versatile toolkit for accurate de novo assembly of organelle genomes.This toolkit assemblies organelle genome from genomic skimming data.</td>
      <td></td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>403</td>
      <td>403</td>
      <td>9423</td>
      <td>9423</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>108</td>
      <td>108</td>
      <td>1588</td>
      <td>1588</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>11011</td>
      <td>11011</td>
      <td>511</td>
      <td>511</td>
      <td>https://usegalaxy.org.au/published/workflow?id=f2541030f6d70b59</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>graphlan</td>
      <td>graphlan, graphlan_annotate</td>
      <td>GraPhlAn is a software tool for producing high-quality circular representations of taxonomic and phylogenetic trees</td>
      <td>2017-03-03</td>
      <td>https://github.com/biobakery/graphlan</td>
      <td></td>
      <td>graphlan</td>
      <td>1.1.3</td>
      <td>To update</td>
      <td>Metagenomics, Graphics, Phylogenetics</td>
      <td>Phylogenetic inference, Phylogenetic tree visualisation, Phylogenetic tree editing, Taxonomic classification</td>
      <td>Phylogenetic inference, Phylogenetic tree visualisation, Phylogenetic tree editing, Taxonomic classification</td>
      <td>Metagenomics, Phylogenetics, Phylogenomics, Cladistics</td>
      <td>Metagenomics, Phylogenetics, Phylogenomics, Cladistics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/graphlan</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/graphlan</td>
      <td>graphlan</td>
      <td>GraPhlAn</td>
      <td>GraPhlAn is a software tool for producing high-quality circular representations of taxonomic and phylogenetic trees. GraPhlAn focuses on concise, integrative, informative, and publication-ready representations of phylogenetically- and taxonomically-driven investigation.</td>
      <td></td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>799</td>
      <td>940</td>
      <td>11872</td>
      <td>13928</td>
      <td>325</td>
      <td>325</td>
      <td>1773</td>
      <td>1773</td>
      <td>202</td>
      <td>207</td>
      <td>1432</td>
      <td>1470</td>
      <td>21</td>
      <td>21</td>
      <td>90</td>
      <td>90</td>
      <td>17261</td>
      <td>15167</td>
      <td>1493</td>
      <td>1347</td>
      <td>https://usegalaxy.eu/published/workflow?id=05f5f040337786bf, https://usegalaxy.eu/published/workflow?id=096b75501c8e0888, https://usegalaxy.eu/published/workflow?id=19de5cbae7f48103, https://usegalaxy.eu/published/workflow?id=2cf09adb1ed6c3ec, https://usegalaxy.eu/published/workflow?id=4d32b1b3ac0a7161, https://usegalaxy.eu/published/workflow?id=4f21205e71cabd37, https://usegalaxy.eu/published/workflow?id=5f0faac42b0a5062, https://usegalaxy.eu/published/workflow?id=713a5485e2007f90, https://usegalaxy.eu/published/workflow?id=79b4ae4403772098, https://usegalaxy.eu/published/workflow?id=7ab70660e6235cf0, https://usegalaxy.eu/published/workflow?id=87cc0cd143a12c41, https://usegalaxy.eu/published/workflow?id=994e5ff1df7d6c9e, https://usegalaxy.eu/published/workflow?id=9cbba5de2cbeeb0f, https://usegalaxy.eu/published/workflow?id=a8f829deb69b30ab, https://usegalaxy.eu/published/workflow?id=b5f848dc8e96663c, https://usegalaxy.eu/published/workflow?id=c01ef8354b585a1f, https://usegalaxy.eu/published/workflow?id=c3aa516e78ca4191, https://usegalaxy.eu/published/workflow?id=d05c6f0f3205fbd2, https://usegalaxy.eu/published/workflow?id=dfc2c53ac3103eff, https://usegalaxy.eu/published/workflow?id=ec50dd6d56f50db1, https://usegalaxy.eu/published/workflow?id=ed0ad728cf21702b, https://usegalaxy.eu/published/workflow?id=fdca6c9d394986bf, https://usegalaxy.fr/published/workflow?id=09e8293456ee6d99, https://usegalaxy.fr/published/workflow?id=c4f6062e51d0f45e, https://workflowhub.eu/workflows/1451?version=2, https://workflowhub.eu/workflows/1466?version=2</td>
      <td>microbiome/metatranscriptomics, microbiome/metatranscriptomics-short</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>gtdb_to_taxdump</td>
      <td>gtdb_to_taxdump</td>
      <td>Convert GTDB taxonomy to NCBI taxdump format</td>
      <td>2024-08-25</td>
      <td>https://github.com/nick-youngblut/gtdb_to_taxdump</td>
      <td>0.1.9</td>
      <td>gtdb_to_taxdump</td>
      <td>0.1.9</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Data handling, Mapping, Generation</td>
      <td>Data handling, Mapping, Generation</td>
      <td>Computational biology</td>
      <td>Computational biology</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/gtdb_to_taxdump</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/gtdb_to_taxdump</td>
      <td>gtdb_to_taxdump</td>
      <td>gtdb_to_taxdump</td>
      <td>Tool with multiple functions. Main functions are to create a DIAMOND database from the GTDB taxonomy data or create a NCBI taxdump format out of this data. This tool can also create a mapping between the taxonomy classification between GTDB and NCBI.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>27</td>
      <td>27</td>
      <td>1127</td>
      <td>1127</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1127</td>
      <td>1127</td>
      <td>27</td>
      <td>27</td>
      <td>https://usegalaxy.eu/published/workflow?id=cf0159c14418525e, https://usegalaxy.eu/published/workflow?id=e4dabaf57668dbd1</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>hapcut2</td>
      <td>hapcut2</td>
      <td>Robust and accurate haplotype assembly for diverse sequencing technologies</td>
      <td>2022-10-19</td>
      <td>https://github.com/vibansal/HapCUT2</td>
      <td>1.3.4</td>
      <td>hapcut2</td>
      <td>1.3.4</td>
      <td>Up-to-date</td>
      <td>Assembly</td>
      <td>Haplotype mapping, Variant classification</td>
      <td>Haplotype mapping, Variant classification</td>
      <td></td>
      <td></td>
      <td>galaxy-australia</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hapcut2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hapcut2</td>
      <td>hapcut2</td>
      <td>HapCUT2</td>
      <td>HapCUT2 is a maximum-likelihood-based tool for assembling haplotypes from DNA sequence reads, designed to "just work" with excellent speed and accuracy across a range of long- and short-read sequencing technologies.The output is in Haplotype block format described here: https://github.com/vibansal/HapCUT2/blob/master/outputformat.md</td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>131</td>
      <td>131</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>131</td>
      <td>131</td>
      <td>6</td>
      <td>6</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>hapog</td>
      <td>hapog</td>
      <td>Hapo-G - Haplotype-Aware Polishing of Genomes</td>
      <td>2022-09-16</td>
      <td>https://github.com/institut-de-genomique/HAPO-G</td>
      <td>1.3.8</td>
      <td>hapog</td>
      <td>1.3.8</td>
      <td>Up-to-date</td>
      <td>Assembly</td>
      <td>Genome assembly, Optimisation and refinement</td>
      <td>Genome assembly, Optimisation and refinement</td>
      <td>Sequence assembly, Genomics</td>
      <td>Sequence assembly, Genomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hapog</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hapog</td>
      <td>hapog</td>
      <td>Hapo-G</td>
      <td>Hapo-G is a tool that aims to improve the quality of genome assemblies by polishing the consensus with accurate reads. It capable of incorporating phasing information from high-quality reads (short or long-reads) to polish genome assemblies and in particular assemblies of diploid and heterozygous genomes.</td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>94</td>
      <td>94</td>
      <td>677</td>
      <td>677</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>36</td>
      <td>36</td>
      <td>392</td>
      <td>392</td>
      <td>1069</td>
      <td>1069</td>
      <td>130</td>
      <td>130</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>helixer</td>
      <td>helixer</td>
      <td>Gene calling with Deep Neural Networks</td>
      <td>2023-06-14</td>
      <td>https://github.com/weberlab-hhu/Helixer</td>
      <td>0.3.3</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>Genome annotation</td>
      <td>Gene prediction, Genome annotation</td>
      <td>Gene prediction, Genome annotation</td>
      <td>Sequence analysis, Gene transcripts</td>
      <td>Sequence analysis, Gene transcripts</td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/helixer</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/helixer</td>
      <td>helixer</td>
      <td>Helixer</td>
      <td>Deep Learning to predict gene annotations</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>161</td>
      <td>161</td>
      <td>1024</td>
      <td>1024</td>
      <td>56</td>
      <td>56</td>
      <td>184</td>
      <td>184</td>
      <td>51</td>
      <td>51</td>
      <td>385</td>
      <td>385</td>
      <td>25</td>
      <td>25</td>
      <td>363</td>
      <td>363</td>
      <td>1956</td>
      <td>1956</td>
      <td>293</td>
      <td>293</td>
      <td>https://usegalaxy.eu/published/workflow?id=31bc2a44fb89aa89, https://usegalaxy.eu/published/workflow?id=327beb1edda305ff, https://usegalaxy.eu/published/workflow?id=3d604dad4f1f5b3d, https://usegalaxy.eu/published/workflow?id=4077e77b6af919c4, https://usegalaxy.eu/published/workflow?id=6550f72070b24082, https://usegalaxy.eu/published/workflow?id=8976041ef89e4548, https://usegalaxy.fr/published/workflow?id=ae05ff8487f58fd6, https://usegalaxy.org.au/published/workflow?id=a9068ea858222e2b, https://usegalaxy.org.au/published/workflow?id=dd9272826e6dd98d, https://usegalaxy.org.au/published/workflow?id=e263fcb6c98e231b, https://usegalaxy.org.au/published/workflow?id=ec7efff3d3249177, https://usegalaxy.org/published/workflow?id=4605309321799e9b, https://usegalaxy.org/published/workflow?id=80a7124c40438cc9, https://usegalaxy.org/published/workflow?id=f4111a169255535c, https://workflowhub.eu/workflows/1255?version=3, https://workflowhub.eu/workflows/1500?version=1, https://workflowhub.eu/workflows/1746?version=1</td>
      <td>genome-annotation/comparison-braker-helixer-annotation, genome-annotation/helixer</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>hifiasm</td>
      <td>hifiasm</td>
      <td>A fast haplotype-resolved de novo assembler</td>
      <td>2024-07-11</td>
      <td>https://github.com/chhylp123/hifiasm</td>
      <td>0.25.0</td>
      <td>hifiasm</td>
      <td>0.25.0</td>
      <td>Up-to-date</td>
      <td>Assembly</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hifiasm</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hifiasm</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>803</td>
      <td>803</td>
      <td>2713</td>
      <td>2713</td>
      <td>585</td>
      <td>585</td>
      <td>2136</td>
      <td>2136</td>
      <td>187</td>
      <td>187</td>
      <td>3817</td>
      <td>3817</td>
      <td>51</td>
      <td>51</td>
      <td>183</td>
      <td>183</td>
      <td>8849</td>
      <td>8849</td>
      <td>1626</td>
      <td>1626</td>
      <td>https://usegalaxy.eu/published/workflow?id=002a51bdef00e14d, https://usegalaxy.eu/published/workflow?id=01fe4c590e608103, https://usegalaxy.eu/published/workflow?id=023a0ee62dd1f593, https://usegalaxy.eu/published/workflow?id=03be9d62e8e4a129, https://usegalaxy.eu/published/workflow?id=085c3a4c89d137b5, https://usegalaxy.eu/published/workflow?id=089e5e83b1de0511, https://usegalaxy.eu/published/workflow?id=1151052f4393852a, https://usegalaxy.eu/published/workflow?id=1483534f49b826bf, https://usegalaxy.eu/published/workflow?id=17b0a6d3379a4392, https://usegalaxy.eu/published/workflow?id=194a05d42687b524, https://usegalaxy.eu/published/workflow?id=19bb57f8cfa48bb9, https://usegalaxy.eu/published/workflow?id=1ac0a15758277730, https://usegalaxy.eu/published/workflow?id=1bb084f1bc102734, https://usegalaxy.eu/published/workflow?id=1cba6eb4aff8dc61, https://usegalaxy.eu/published/workflow?id=1da7ac8eaf891f39, https://usegalaxy.eu/published/workflow?id=1e02a7292e9974b1, https://usegalaxy.eu/published/workflow?id=1f2a1438e41188f2, https://usegalaxy.eu/published/workflow?id=21a3f72851f29925, https://usegalaxy.eu/published/workflow?id=2671a5f85b2cccfe, https://usegalaxy.eu/published/workflow?id=270b6f82efab28b2, https://usegalaxy.eu/published/workflow?id=2e418c5810012e5a, https://usegalaxy.eu/published/workflow?id=342a5df69951e3d7, https://usegalaxy.eu/published/workflow?id=379642036b6f5a66, https://usegalaxy.eu/published/workflow?id=3a3e2d50fe7a9ee3, https://usegalaxy.eu/published/workflow?id=3e255a5b7026d26e, https://usegalaxy.eu/published/workflow?id=3eb6663a66f8e69a, https://usegalaxy.eu/published/workflow?id=3f5414bb38b47fcf, https://usegalaxy.eu/published/workflow?id=416141b245073406, https://usegalaxy.eu/published/workflow?id=45ae834d842302c3, https://usegalaxy.eu/published/workflow?id=471eb8b4c54f897d, https://usegalaxy.eu/published/workflow?id=475ea16f943cc013, https://usegalaxy.eu/published/workflow?id=4856e3522b2ee059, https://usegalaxy.eu/published/workflow?id=497b82d3b92dc076, https://usegalaxy.eu/published/workflow?id=4c3a9b574f37eebb, https://usegalaxy.eu/published/workflow?id=4d41ee83fc0040d4, https://usegalaxy.eu/published/workflow?id=5115299e77df8104, https://usegalaxy.eu/published/workflow?id=54590ccd11ffbf65, https://usegalaxy.eu/published/workflow?id=56523b41ff7eec76, https://usegalaxy.eu/published/workflow?id=57cc33ed64d71982, https://usegalaxy.eu/published/workflow?id=5ad406442ec21345, https://usegalaxy.eu/published/workflow?id=5e1906290f5b4d8a, https://usegalaxy.eu/published/workflow?id=6573f3b1606b7f09, https://usegalaxy.eu/published/workflow?id=69cdd404592535ec, https://usegalaxy.eu/published/workflow?id=7137e45ca8dc6e4c, https://usegalaxy.eu/published/workflow?id=71a6d128a09210f0, https://usegalaxy.eu/published/workflow?id=7337d15d121596da, https://usegalaxy.eu/published/workflow?id=79b5afb125e3c3da, https://usegalaxy.eu/published/workflow?id=83a1b8949d87e3da, https://usegalaxy.eu/published/workflow?id=847403b72dd346ff, https://usegalaxy.eu/published/workflow?id=8be366ad6a42408b, https://usegalaxy.eu/published/workflow?id=9207e74a5bf7d8e1, https://usegalaxy.eu/published/workflow?id=92760fc19ef120ce, https://usegalaxy.eu/published/workflow?id=92dafb3d1c06e074, https://usegalaxy.eu/published/workflow?id=95f420d8856bf725, https://usegalaxy.eu/published/workflow?id=9c0b67496dfe52b2, https://usegalaxy.eu/published/workflow?id=9f2d18f0d0896e8b, https://usegalaxy.eu/published/workflow?id=a6dd62ab4b9a0725, https://usegalaxy.eu/published/workflow?id=a9c1dca5da7170dd, https://usegalaxy.eu/published/workflow?id=aa6dc57f5c67fb4b, https://usegalaxy.eu/published/workflow?id=ac6eb99f70d35999, https://usegalaxy.eu/published/workflow?id=ae74f60cd20d56bc, https://usegalaxy.eu/published/workflow?id=b211fceaecf41c9e, https://usegalaxy.eu/published/workflow?id=b4948afacccfefdb, https://usegalaxy.eu/published/workflow?id=b8baade87e733fce, https://usegalaxy.eu/published/workflow?id=bc87309a8763d0b4, https://usegalaxy.eu/published/workflow?id=bcf99cc0469db276, https://usegalaxy.eu/published/workflow?id=bef0a7d604f2ebef, https://usegalaxy.eu/published/workflow?id=c4c2e669b86cffbc, https://usegalaxy.eu/published/workflow?id=c508c00e81018dd8, https://usegalaxy.eu/published/workflow?id=caba6d3d7db648e9, https://usegalaxy.eu/published/workflow?id=d67299c34e63f3d9, https://usegalaxy.eu/published/workflow?id=dc27453c438577f6, https://usegalaxy.eu/published/workflow?id=de9cf573314f416c, https://usegalaxy.eu/published/workflow?id=dfe3b555856c7b44, https://usegalaxy.eu/published/workflow?id=e138640778b7414c, https://usegalaxy.eu/published/workflow?id=e1947b059ef131d2, https://usegalaxy.eu/published/workflow?id=e2f936f1aa43a629, https://usegalaxy.eu/published/workflow?id=e7aebe58411d97ed, https://usegalaxy.eu/published/workflow?id=e8f9e1d911683b22, https://usegalaxy.eu/published/workflow?id=e935d2c8ffb52588, https://usegalaxy.eu/published/workflow?id=ec79f14ab91cd933, https://usegalaxy.eu/published/workflow?id=f1ca4289ee4f67bb, https://usegalaxy.eu/published/workflow?id=f38745208494d4ab, https://usegalaxy.eu/published/workflow?id=f578a097b4339025, https://usegalaxy.eu/published/workflow?id=f6a625946dea60fa, https://usegalaxy.eu/published/workflow?id=fb015e398373ea8a, https://usegalaxy.eu/published/workflow?id=fba1671d835b6dd3, https://usegalaxy.eu/published/workflow?id=fc25c1f8c6667c87, https://usegalaxy.eu/published/workflow?id=fd649ac4d4872c81, https://usegalaxy.org.au/published/workflow?id=02eebe3d909206ab, https://usegalaxy.org.au/published/workflow?id=044b6b475b4db27a, https://usegalaxy.org.au/published/workflow?id=08fac3b4c128e597, https://usegalaxy.org.au/published/workflow?id=0af0ea50f1237952, https://usegalaxy.org.au/published/workflow?id=0cc8c8875df6bccb, https://usegalaxy.org.au/published/workflow?id=10355b2680374a03, https://usegalaxy.org.au/published/workflow?id=1eb3e1452c672480, https://usegalaxy.org.au/published/workflow?id=2470cb61ba51288c, https://usegalaxy.org.au/published/workflow?id=2d45d7138d8715b4, https://usegalaxy.org.au/published/workflow?id=319e9d693fac8dce, https://usegalaxy.org.au/published/workflow?id=3604307ec2835628, https://usegalaxy.org.au/published/workflow?id=36604d0ab0dee988, https://usegalaxy.org.au/published/workflow?id=36f54eda8568355d, https://usegalaxy.org.au/published/workflow?id=371c9892d261cee2, https://usegalaxy.org.au/published/workflow?id=3c30c822058187be, https://usegalaxy.org.au/published/workflow?id=3df7f45b7b388f8c, https://usegalaxy.org.au/published/workflow?id=40666ceb68d101ac, https://usegalaxy.org.au/published/workflow?id=41795303a88d9e94, https://usegalaxy.org.au/published/workflow?id=4267bd1e4cfd1901, https://usegalaxy.org.au/published/workflow?id=43343fa1896f8278, https://usegalaxy.org.au/published/workflow?id=434127690544d0e9, https://usegalaxy.org.au/published/workflow?id=447d8daefdefafee, https://usegalaxy.org.au/published/workflow?id=47cd0a90609fd557, https://usegalaxy.org.au/published/workflow?id=4b5f80656fc94b88, https://usegalaxy.org.au/published/workflow?id=51bad198f1fd1215, https://usegalaxy.org.au/published/workflow?id=5550585ffc3d8d0e, https://usegalaxy.org.au/published/workflow?id=5b0c0e34394afed1, https://usegalaxy.org.au/published/workflow?id=5f4f874e49515555, https://usegalaxy.org.au/published/workflow?id=60c0dec52cdca708, https://usegalaxy.org.au/published/workflow?id=63789b6f82172679, https://usegalaxy.org.au/published/workflow?id=68b2961117e62890, https://usegalaxy.org.au/published/workflow?id=6c27019d03dd49d8, https://usegalaxy.org.au/published/workflow?id=6c9129acb9d98afe, https://usegalaxy.org.au/published/workflow?id=6ec0031406b10635, https://usegalaxy.org.au/published/workflow?id=8926e592f2e60863, https://usegalaxy.org.au/published/workflow?id=8a6233015db6dd24, https://usegalaxy.org.au/published/workflow?id=8bf8c9929249f37d, https://usegalaxy.org.au/published/workflow?id=8dd86497bc15d981, https://usegalaxy.org.au/published/workflow?id=8e8a5cce9f3f2f67, https://usegalaxy.org.au/published/workflow?id=91ee716be8bfd508, https://usegalaxy.org.au/published/workflow?id=9642aadb83c4f752, https://usegalaxy.org.au/published/workflow?id=970dc0d8bd74707f, https://usegalaxy.org.au/published/workflow?id=9804030358c3f844, https://usegalaxy.org.au/published/workflow?id=98d3def7da131b4c, https://usegalaxy.org.au/published/workflow?id=98f660971010d11d, https://usegalaxy.org.au/published/workflow?id=9e0bae0505ee7c07, https://usegalaxy.org.au/published/workflow?id=a02c102e179596cb, https://usegalaxy.org.au/published/workflow?id=a345bf84eb9bf6b2, https://usegalaxy.org.au/published/workflow?id=a525966251895f8a, https://usegalaxy.org.au/published/workflow?id=ad10cd704268dcf0, https://usegalaxy.org.au/published/workflow?id=afab89bc468da8b3, https://usegalaxy.org.au/published/workflow?id=b19fa49405117317, https://usegalaxy.org.au/published/workflow?id=b1f45d2f8d8343e9, https://usegalaxy.org.au/published/workflow?id=b5131e3a0ef2676e, https://usegalaxy.org.au/published/workflow?id=b9faf6a3f48fc11b, https://usegalaxy.org.au/published/workflow?id=be96bb47b6d6435a, https://usegalaxy.org.au/published/workflow?id=c43744d7dd9c0c0b, https://usegalaxy.org.au/published/workflow?id=c6a71573a299eb3a, https://usegalaxy.org.au/published/workflow?id=c7e1f666a3366823, https://usegalaxy.org.au/published/workflow?id=c7f2d6490128dc3c, https://usegalaxy.org.au/published/workflow?id=d298edda13b382e7, https://usegalaxy.org.au/published/workflow?id=d42d7b1aa47e870f, https://usegalaxy.org.au/published/workflow?id=d78879cfaa5948b3, https://usegalaxy.org.au/published/workflow?id=d7f3fc810261e419, https://usegalaxy.org.au/published/workflow?id=d9c181fd954b629a, https://usegalaxy.org.au/published/workflow?id=df4e186069552649, https://usegalaxy.org.au/published/workflow?id=e031af90daa2cadf, https://usegalaxy.org.au/published/workflow?id=e28ca83f14f4ede5, https://usegalaxy.org.au/published/workflow?id=e5f1ce106ea7af12, https://usegalaxy.org.au/published/workflow?id=ea687c5613fcce27, https://usegalaxy.org.au/published/workflow?id=f7a866159f1eaecf, https://usegalaxy.org.au/published/workflow?id=f83a3cda33131fb8, https://usegalaxy.org.au/published/workflow?id=fa37223950a91f34, https://usegalaxy.org.au/published/workflow?id=fe1e7fa0bc437ed8, https://usegalaxy.org.au/published/workflow?id=fff2e49bc90e38ca, https://usegalaxy.org/published/workflow?id=07e873a6d75dfe9f, https://usegalaxy.org/published/workflow?id=0993daadf2db5f51, https://usegalaxy.org/published/workflow?id=1245e9435cbc3985, https://usegalaxy.org/published/workflow?id=137d0f05a71d7acb, https://usegalaxy.org/published/workflow?id=1d24cc949c52e475, https://usegalaxy.org/published/workflow?id=1eec2586d1ff908b, https://usegalaxy.org/published/workflow?id=25a9f0693ff6d327, https://usegalaxy.org/published/workflow?id=283bc048cc7a80f5, https://usegalaxy.org/published/workflow?id=284150bb1f296378, https://usegalaxy.org/published/workflow?id=2b41752b8518a87b, https://usegalaxy.org/published/workflow?id=2c5c627c08563398, https://usegalaxy.org/published/workflow?id=2d994cc02ec05ede, https://usegalaxy.org/published/workflow?id=305fb9f84b053713, https://usegalaxy.org/published/workflow?id=31a6145b0832fa48, https://usegalaxy.org/published/workflow?id=3ba73f679558017c, https://usegalaxy.org/published/workflow?id=3db4eb1ecb588d44, https://usegalaxy.org/published/workflow?id=3f7550d5d2bf0b96, https://usegalaxy.org/published/workflow?id=408c799822cbf6ea, https://usegalaxy.org/published/workflow?id=435f0340bf00c940, https://usegalaxy.org/published/workflow?id=4403ca7fe447a5d7, https://usegalaxy.org/published/workflow?id=44781c8b0bd667ed, https://usegalaxy.org/published/workflow?id=484f8fcf21d67376, https://usegalaxy.org/published/workflow?id=498a8c6329ca5728, https://usegalaxy.org/published/workflow?id=4a0ad87aa5aa558a, https://usegalaxy.org/published/workflow?id=4bf73376661baafd, https://usegalaxy.org/published/workflow?id=4d5cc5010537f172, https://usegalaxy.org/published/workflow?id=578e634b63923e9b, https://usegalaxy.org/published/workflow?id=57e6345fda3e2682, https://usegalaxy.org/published/workflow?id=597c4d54f8a8ebe7, https://usegalaxy.org/published/workflow?id=5dfec2a3ae5fa765, https://usegalaxy.org/published/workflow?id=653d553ac2fa51c8, https://usegalaxy.org/published/workflow?id=67252039bce060b2, https://usegalaxy.org/published/workflow?id=683e725f672b7d9b, https://usegalaxy.org/published/workflow?id=6b511b47176893e3, https://usegalaxy.org/published/workflow?id=7072731b51e16923, https://usegalaxy.org/published/workflow?id=7dba74c915f63dc1, https://usegalaxy.org/published/workflow?id=7f6900852a81d121, https://usegalaxy.org/published/workflow?id=81f1f0fd6d9235fd, https://usegalaxy.org/published/workflow?id=84161f4f8bf77474, https://usegalaxy.org/published/workflow?id=864da6858d34df08, https://usegalaxy.org/published/workflow?id=8cfbaa14cff5a633, https://usegalaxy.org/published/workflow?id=8d605fbd33003d26, https://usegalaxy.org/published/workflow?id=8f7c9461910b08f1, https://usegalaxy.org/published/workflow?id=90f5b512997c70f8, https://usegalaxy.org/published/workflow?id=932b3bc10186f73e, https://usegalaxy.org/published/workflow?id=936ed7447c8e0a2d, https://usegalaxy.org/published/workflow?id=93c1fef4472aa7a8, https://usegalaxy.org/published/workflow?id=961dac8100c6c70c, https://usegalaxy.org/published/workflow?id=9938a81fb6911412, https://usegalaxy.org/published/workflow?id=9ddfdc9d6cacea88, https://usegalaxy.org/published/workflow?id=a8e0f77cd6ef49f9, https://usegalaxy.org/published/workflow?id=ab0be4897d5355ac, https://usegalaxy.org/published/workflow?id=acada5044d64a30a, https://usegalaxy.org/published/workflow?id=b11c6268704dcc1b, https://usegalaxy.org/published/workflow?id=b5644a5dcf237ef9, https://usegalaxy.org/published/workflow?id=b661fb850c4b4b01, https://usegalaxy.org/published/workflow?id=b6d535f210efa18e, https://usegalaxy.org/published/workflow?id=bb1cb6abfe84a646, https://usegalaxy.org/published/workflow?id=bbc894a79eb0e769, https://usegalaxy.org/published/workflow?id=bd6bfd338ffb9637, https://usegalaxy.org/published/workflow?id=bfd2e7e63d3e6856, https://usegalaxy.org/published/workflow?id=bfef75672b9c3ea4, https://usegalaxy.org/published/workflow?id=c0a9a2d53b8660de, https://usegalaxy.org/published/workflow?id=c983ea8a32662cd6, https://usegalaxy.org/published/workflow?id=ce04658f05029be2, https://usegalaxy.org/published/workflow?id=d08135fa13d1745e, https://usegalaxy.org/published/workflow?id=d36765ac28c11e1a, https://usegalaxy.org/published/workflow?id=d3f35215e6f45d79, https://usegalaxy.org/published/workflow?id=daaded719dcfd53b, https://usegalaxy.org/published/workflow?id=dae5d5e5df5c4dd7, https://usegalaxy.org/published/workflow?id=e4dd39237df58884, https://usegalaxy.org/published/workflow?id=f1a54b10d4afce8f, https://usegalaxy.org/published/workflow?id=f60027545d40b82d, https://usegalaxy.org/published/workflow?id=fa26fd51caea53c7, https://usegalaxy.org/published/workflow?id=fc7b14aafa0b421e, https://usegalaxy.org/published/workflow?id=fd40e50c9f1545ce, https://workflowhub.eu/workflows/1162?version=2, https://workflowhub.eu/workflows/221?version=3, https://workflowhub.eu/workflows/605?version=1, https://workflowhub.eu/workflows/612?version=25, https://workflowhub.eu/workflows/641?version=32, https://workflowhub.eu/workflows/642?version=25</td>
      <td>assembly/hifi-assembly, assembly/vgp_genome_assembly</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>hifiasm_meta</td>
      <td>hifiasm_meta</td>
      <td>A hifiasm fork for metagenome assembly using Hifi reads.</td>
      <td>2023-01-18</td>
      <td>https://github.com/xfengnefx/hifiasm-meta</td>
      <td>0.3.1</td>
      <td>hifiasm_meta</td>
      <td>hamtv0.3.1</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly, Metagenomics</td>
      <td>Sequence assembly, Metagenomics</td>
      <td>galaxy-australia</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hifiasm_meta</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hifiasm_meta</td>
      <td>hifiasm-meta</td>
      <td>Hifiasm-meta</td>
      <td>Hifiasm_meta - de novo metagenome assembler, based on hifiasm, a haplotype-resolved de novo assembler for PacBio Hifi reads.</td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>40</td>
      <td>40</td>
      <td>229</td>
      <td>229</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>16</td>
      <td>16</td>
      <td>368</td>
      <td>368</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>597</td>
      <td>597</td>
      <td>56</td>
      <td>56</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>hirondelle_crim_ogc_api_processes</td>
      <td>hirondelle_crim</td>
      <td>This tool is a wrapper for OGC API Processes coming from https://osf.io/gfbws/.</td>
      <td>2024-06-07</td>
      <td>https://github.com/AquaINFRA/galaxy</td>
      <td>0.2.0</td>
      <td>r-base</td>
      <td></td>
      <td>To update</td>
      <td>Ecology</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/ogc_api_processes_wrapper</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/hirondelle_crim_ogc_api_processes</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>237</td>
      <td>237</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>237</td>
      <td>237</td>
      <td>4</td>
      <td>4</td>
      <td>https://usegalaxy.eu/published/workflow?id=6173008772bdba09, https://usegalaxy.eu/published/workflow?id=b1fa4a3561988704</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>hisat2</td>
      <td>hisat2</td>
      <td>HISAT2 is a fast and sensitive spliced alignment program.</td>
      <td>2015-09-29</td>
      <td>http://ccb.jhu.edu/software/hisat2/</td>
      <td>2.2.1</td>
      <td>hisat2</td>
      <td>2.2.1</td>
      <td>Up-to-date</td>
      <td>Assembly</td>
      <td>Sequence alignment</td>
      <td>Sequence alignment</td>
      <td>RNA-seq</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hisat2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hisat2</td>
      <td>hisat2</td>
      <td>HISAT2</td>
      <td>Alignment program for mapping next-generation sequencing reads (both DNA and RNA) to a population of human genomes (as well as to a single reference genome).</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>12585</td>
      <td>14119</td>
      <td>420921</td>
      <td>467443</td>
      <td>28716</td>
      <td>40390</td>
      <td>729970</td>
      <td>1057729</td>
      <td>5266</td>
      <td>6509</td>
      <td>88398</td>
      <td>111027</td>
      <td>1273</td>
      <td>1277</td>
      <td>52049</td>
      <td>52136</td>
      <td>1688335</td>
      <td>1291338</td>
      <td>62295</td>
      <td>47840</td>
      <td>https://usegalaxy.eu/published/workflow?id=0de150b19102cb7e, https://usegalaxy.eu/published/workflow?id=135e8888872e8413, https://usegalaxy.eu/published/workflow?id=1502fd3fa90d32d8, https://usegalaxy.eu/published/workflow?id=17916f0229e3c868, https://usegalaxy.eu/published/workflow?id=17ad465d9515cd25, https://usegalaxy.eu/published/workflow?id=1842a61fe5f6965f, https://usegalaxy.eu/published/workflow?id=18c9095fbf5c4517, https://usegalaxy.eu/published/workflow?id=1b49eda539658e53, https://usegalaxy.eu/published/workflow?id=1bc7b5eabea972fd, https://usegalaxy.eu/published/workflow?id=1c3e7b3af6ff79fa, https://usegalaxy.eu/published/workflow?id=1ccc0c9e1c6821bb, https://usegalaxy.eu/published/workflow?id=1cd85fe90839c10d, https://usegalaxy.eu/published/workflow?id=1fe02012c96b731f, https://usegalaxy.eu/published/workflow?id=212c213f1d8e0154, https://usegalaxy.eu/published/workflow?id=2287afbf248d9c89, https://usegalaxy.eu/published/workflow?id=26185e8fa5a9b988, https://usegalaxy.eu/published/workflow?id=29a855d928337f8f, https://usegalaxy.eu/published/workflow?id=2d64d6571e01809d, https://usegalaxy.eu/published/workflow?id=300301c23327f797, https://usegalaxy.eu/published/workflow?id=3069c17b52decab4, https://usegalaxy.eu/published/workflow?id=318be2b30bf7884c, https://usegalaxy.eu/published/workflow?id=31d9eb6a120411d6, https://usegalaxy.eu/published/workflow?id=3550384e7c062a60, https://usegalaxy.eu/published/workflow?id=39876e9c04aad543, https://usegalaxy.eu/published/workflow?id=39afbf843b5513bf, https://usegalaxy.eu/published/workflow?id=3a8150c81b90a3a7, https://usegalaxy.eu/published/workflow?id=3d5b39e454df584b, https://usegalaxy.eu/published/workflow?id=47a0407867a90f95, https://usegalaxy.eu/published/workflow?id=487929c7b5be8dc3, https://usegalaxy.eu/published/workflow?id=4dd7f6e1c1e7f96a, https://usegalaxy.eu/published/workflow?id=50758631a0e81a7a, https://usegalaxy.eu/published/workflow?id=52cafb62f5bf6e25, https://usegalaxy.eu/published/workflow?id=534b81d48df6b507, https://usegalaxy.eu/published/workflow?id=535f777e5e42e98c, https://usegalaxy.eu/published/workflow?id=54467870b7c635f2, https://usegalaxy.eu/published/workflow?id=56868483be18f1a1, https://usegalaxy.eu/published/workflow?id=5aba0f20d994b1a4, https://usegalaxy.eu/published/workflow?id=5b00ef046439d757, https://usegalaxy.eu/published/workflow?id=60a45de522b8b695, https://usegalaxy.eu/published/workflow?id=60e4d0db3164f037, https://usegalaxy.eu/published/workflow?id=61e592bafd9d2b58, https://usegalaxy.eu/published/workflow?id=625aafb9cf16b03f, https://usegalaxy.eu/published/workflow?id=63749bd3045ccd4a, https://usegalaxy.eu/published/workflow?id=63964ba7d9edce74, https://usegalaxy.eu/published/workflow?id=67f75ed02a03bf93, https://usegalaxy.eu/published/workflow?id=68e8042f11d5a3fd, https://usegalaxy.eu/published/workflow?id=6d87481be47902e1, https://usegalaxy.eu/published/workflow?id=6e6cf59bfeb938bf, https://usegalaxy.eu/published/workflow?id=71a9224fa6f3da9d, https://usegalaxy.eu/published/workflow?id=733650962d379da2, https://usegalaxy.eu/published/workflow?id=762c64a9f588a420, https://usegalaxy.eu/published/workflow?id=766ebba7a2aee494, https://usegalaxy.eu/published/workflow?id=7734928ebc0a2654, https://usegalaxy.eu/published/workflow?id=7baae0775071c7ec, https://usegalaxy.eu/published/workflow?id=7c9b182da21748c7, https://usegalaxy.eu/published/workflow?id=82ef590b5821805d, https://usegalaxy.eu/published/workflow?id=88b382a6f56a6329, https://usegalaxy.eu/published/workflow?id=8b1e2fe9f4a07de0, https://usegalaxy.eu/published/workflow?id=910570a2caafe462, https://usegalaxy.eu/published/workflow?id=9141c6449595c6a1, https://usegalaxy.eu/published/workflow?id=927d24dcde139b9c, https://usegalaxy.eu/published/workflow?id=98d4adf5774e274e, https://usegalaxy.eu/published/workflow?id=9a1d17c12b139bda, https://usegalaxy.eu/published/workflow?id=9a60ecf98d4f8134, https://usegalaxy.eu/published/workflow?id=9e01e24c5e749365, https://usegalaxy.eu/published/workflow?id=9e31c51911987687, https://usegalaxy.eu/published/workflow?id=a2f36d4dd48d8ec2, https://usegalaxy.eu/published/workflow?id=a35e0c2fcba4e95f, https://usegalaxy.eu/published/workflow?id=a6c666f3028ea440, https://usegalaxy.eu/published/workflow?id=a72d2667a53830f7, https://usegalaxy.eu/published/workflow?id=a7638e7f908fd90d, https://usegalaxy.eu/published/workflow?id=ab6e6ffaaebab45b, https://usegalaxy.eu/published/workflow?id=ab8924d78fa5ef8f, https://usegalaxy.eu/published/workflow?id=ae1ca223d6667070, https://usegalaxy.eu/published/workflow?id=b30c294a094c90bc, https://usegalaxy.eu/published/workflow?id=b41ff6d8fb5a0878, https://usegalaxy.eu/published/workflow?id=b6eb5fe7e37f680a, https://usegalaxy.eu/published/workflow?id=b801e745205892f5, https://usegalaxy.eu/published/workflow?id=bbe2a1615c8faaae, https://usegalaxy.eu/published/workflow?id=bc32937d505135ab, https://usegalaxy.eu/published/workflow?id=bcb8a8c5bff790ab, https://usegalaxy.eu/published/workflow?id=bebcbd6f0ffa55c5, https://usegalaxy.eu/published/workflow?id=bfb762c96328e3c1, https://usegalaxy.eu/published/workflow?id=c885eaabae612d6d, https://usegalaxy.eu/published/workflow?id=cae527ce845e770e, https://usegalaxy.eu/published/workflow?id=cbc16cb5db98ab21, https://usegalaxy.eu/published/workflow?id=cc37667c2d1f9056, https://usegalaxy.eu/published/workflow?id=d086340422009ba9, https://usegalaxy.eu/published/workflow?id=d1338e2b4002a364, https://usegalaxy.eu/published/workflow?id=d3dfe7ba11508c72, https://usegalaxy.eu/published/workflow?id=d41b52a15ba56f54, https://usegalaxy.eu/published/workflow?id=d503dee6f9b35a18, https://usegalaxy.eu/published/workflow?id=d5333903ba947adf, https://usegalaxy.eu/published/workflow?id=d72edb9f8c529f47, https://usegalaxy.eu/published/workflow?id=d9f8c5d575da02b1, https://usegalaxy.eu/published/workflow?id=daf47855afa73de8, https://usegalaxy.eu/published/workflow?id=dbcf7af09e036754, https://usegalaxy.eu/published/workflow?id=de3eafbcf42c75bb, https://usegalaxy.eu/published/workflow?id=e62b659a5da9734e, https://usegalaxy.eu/published/workflow?id=e6c5fb1d37d29ec0, https://usegalaxy.eu/published/workflow?id=e8202b7b6338005d, https://usegalaxy.eu/published/workflow?id=e85c9385870aadcc, https://usegalaxy.eu/published/workflow?id=eb8168bc01249596, https://usegalaxy.eu/published/workflow?id=ebaa33538e7ff300, https://usegalaxy.eu/published/workflow?id=ecab5b2c3ff3df9d, https://usegalaxy.eu/published/workflow?id=f026c4b8341ff94c, https://usegalaxy.eu/published/workflow?id=f0ac222d3e965ab0, https://usegalaxy.eu/published/workflow?id=f12fb4a95954c448, https://usegalaxy.eu/published/workflow?id=fb18c5d14acb1b36, https://usegalaxy.eu/published/workflow?id=fe8a1a970fd0d4cd, https://usegalaxy.fr/published/workflow?id=0f6714dbdb90b6aa, https://usegalaxy.fr/published/workflow?id=334189184455d9a2, https://usegalaxy.fr/published/workflow?id=5d83e0886f2a7465, https://usegalaxy.fr/published/workflow?id=5e49933a687c2b46, https://usegalaxy.fr/published/workflow?id=6d2945d35fa956c8, https://usegalaxy.fr/published/workflow?id=72b531d980acaa6c, https://usegalaxy.fr/published/workflow?id=74b4d11c07f7f0a4, https://usegalaxy.fr/published/workflow?id=826cffb2458e4067, https://usegalaxy.fr/published/workflow?id=aa96487c5a1b29bf, https://usegalaxy.fr/published/workflow?id=e0bad0b97ee71577, https://usegalaxy.org.au/published/workflow?id=000cdc969e9f0886, https://usegalaxy.org.au/published/workflow?id=01997cb92a2ce2d5, https://usegalaxy.org.au/published/workflow?id=0220e60c66de9065, https://usegalaxy.org.au/published/workflow?id=02b8d033d9b1fa05, https://usegalaxy.org.au/published/workflow?id=046a1c600a6b77c2, https://usegalaxy.org.au/published/workflow?id=05950987aa2eba5b, https://usegalaxy.org.au/published/workflow?id=05b17b9d70c1db8b, https://usegalaxy.org.au/published/workflow?id=0614f24ebeb680de, https://usegalaxy.org.au/published/workflow?id=068f85b8b137bfd7, https://usegalaxy.org.au/published/workflow?id=07c97b9e59c0218f, https://usegalaxy.org.au/published/workflow?id=0c567180448d8774, https://usegalaxy.org.au/published/workflow?id=0cfc1408fd9ea4e7, https://usegalaxy.org.au/published/workflow?id=0d1db28665d19bda, https://usegalaxy.org.au/published/workflow?id=0ebaaecf61dfc921, https://usegalaxy.org.au/published/workflow?id=0fc7c14c4d1ac8f5, https://usegalaxy.org.au/published/workflow?id=10acf4f497172400, https://usegalaxy.org.au/published/workflow?id=114fb9589bb9db92, https://usegalaxy.org.au/published/workflow?id=14184bc2580a131f, https://usegalaxy.org.au/published/workflow?id=157e7614ae18849b, https://usegalaxy.org.au/published/workflow?id=16a2c2e8b3c1ffe6, https://usegalaxy.org.au/published/workflow?id=19315e6573c58239, https://usegalaxy.org.au/published/workflow?id=1ab66d382630afa7, https://usegalaxy.org.au/published/workflow?id=1c886f008a36b0b8, https://usegalaxy.org.au/published/workflow?id=20e5e6b233ffb951, https://usegalaxy.org.au/published/workflow?id=210ca013cc1cf185, https://usegalaxy.org.au/published/workflow?id=25ac9b4f2e39af97, https://usegalaxy.org.au/published/workflow?id=2999a1891ffe201b, https://usegalaxy.org.au/published/workflow?id=2a0046582f5f15a1, https://usegalaxy.org.au/published/workflow?id=2c58509f80897972, https://usegalaxy.org.au/published/workflow?id=2f7c0e3221cbca2f, https://usegalaxy.org.au/published/workflow?id=301fdb29c513ffbe, https://usegalaxy.org.au/published/workflow?id=30b2cd025e0300da, https://usegalaxy.org.au/published/workflow?id=31449596b22f81f4, https://usegalaxy.org.au/published/workflow?id=35384818ee1cbac2, https://usegalaxy.org.au/published/workflow?id=374311292d72c9f9, https://usegalaxy.org.au/published/workflow?id=383ce49fa7a367e9, https://usegalaxy.org.au/published/workflow?id=39d56981a6854d82, https://usegalaxy.org.au/published/workflow?id=3bc45aaed5da1058, https://usegalaxy.org.au/published/workflow?id=3c727d5cd349a846, https://usegalaxy.org.au/published/workflow?id=3ce9dc804780e4a1, https://usegalaxy.org.au/published/workflow?id=3ef07022656067b1, https://usegalaxy.org.au/published/workflow?id=410d0d19b6589c78, https://usegalaxy.org.au/published/workflow?id=421adea2f259ec26, https://usegalaxy.org.au/published/workflow?id=45c8789533c45abf, https://usegalaxy.org.au/published/workflow?id=4e096a68187ce652, https://usegalaxy.org.au/published/workflow?id=4efed79608f85604, https://usegalaxy.org.au/published/workflow?id=4f2818b6620037af, https://usegalaxy.org.au/published/workflow?id=4f562265cbb14cd6, https://usegalaxy.org.au/published/workflow?id=525612313de7591e, https://usegalaxy.org.au/published/workflow?id=53010c2f1f207429, https://usegalaxy.org.au/published/workflow?id=5314a7a4732a0ef1, https://usegalaxy.org.au/published/workflow?id=544cb12a16f283b8, https://usegalaxy.org.au/published/workflow?id=55c4d97a08a5ca09, https://usegalaxy.org.au/published/workflow?id=56599572e0364e2a, https://usegalaxy.org.au/published/workflow?id=5878a54446a26449, https://usegalaxy.org.au/published/workflow?id=5926a7669eabb1c6, https://usegalaxy.org.au/published/workflow?id=5a4e49ef8693d7c4, https://usegalaxy.org.au/published/workflow?id=5be983b903469f07, https://usegalaxy.org.au/published/workflow?id=5cbee52cf03b6f9a, https://usegalaxy.org.au/published/workflow?id=5ce00d3eab862f4e, https://usegalaxy.org.au/published/workflow?id=5ea74382fb77eb10, https://usegalaxy.org.au/published/workflow?id=608f3ad8f485e7a8, https://usegalaxy.org.au/published/workflow?id=6204bbcfe56931eb, https://usegalaxy.org.au/published/workflow?id=625f3c52832b0591, https://usegalaxy.org.au/published/workflow?id=6448b82959076a29, https://usegalaxy.org.au/published/workflow?id=6469ade8f6f7c718, https://usegalaxy.org.au/published/workflow?id=67191ea9fe977e08, https://usegalaxy.org.au/published/workflow?id=677ef3322366c953, https://usegalaxy.org.au/published/workflow?id=69233fb6edfcdde9, https://usegalaxy.org.au/published/workflow?id=6b34c95a247d15e3, https://usegalaxy.org.au/published/workflow?id=6cdbcd1f667d0f16, https://usegalaxy.org.au/published/workflow?id=6f0801fecbba5d3f, https://usegalaxy.org.au/published/workflow?id=723513b09d6690b8, https://usegalaxy.org.au/published/workflow?id=73ff1aa9acc0ad2a, https://usegalaxy.org.au/published/workflow?id=74186b006665daf8, https://usegalaxy.org.au/published/workflow?id=76323f103844f75b, https://usegalaxy.org.au/published/workflow?id=79522a1f7b0c53cb, https://usegalaxy.org.au/published/workflow?id=7c866ed4a3552d0d, https://usegalaxy.org.au/published/workflow?id=7cd1acb1da823cdc, https://usegalaxy.org.au/published/workflow?id=7cd4fcc8e06ce4fd, https://usegalaxy.org.au/published/workflow?id=7ea616c652d08fb6, https://usegalaxy.org.au/published/workflow?id=7ef245f5ae3b916d, https://usegalaxy.org.au/published/workflow?id=7febc6ac6f106e57, https://usegalaxy.org.au/published/workflow?id=811bbada546850bd, https://usegalaxy.org.au/published/workflow?id=83680c1c6dffb05b, https://usegalaxy.org.au/published/workflow?id=83a535840b388980, https://usegalaxy.org.au/published/workflow?id=8483ae422665a6c1, https://usegalaxy.org.au/published/workflow?id=8483e8d3e8d0ddc9, https://usegalaxy.org.au/published/workflow?id=8597166abda0a375, https://usegalaxy.org.au/published/workflow?id=863eee1b041b6c58, https://usegalaxy.org.au/published/workflow?id=879d4fb110a61f7a, https://usegalaxy.org.au/published/workflow?id=88c0d875eea2ee49, https://usegalaxy.org.au/published/workflow?id=897dfa78c3b25356, https://usegalaxy.org.au/published/workflow?id=8b8a5b50f2edc72c, https://usegalaxy.org.au/published/workflow?id=8c40b4a911517cc5, https://usegalaxy.org.au/published/workflow?id=8cab31db0bd3ff52, https://usegalaxy.org.au/published/workflow?id=8d196c8c62530dc0, https://usegalaxy.org.au/published/workflow?id=8d6010d07cc2e170, https://usegalaxy.org.au/published/workflow?id=91cfd96f697ba034, https://usegalaxy.org.au/published/workflow?id=925c167abd5610ae, https://usegalaxy.org.au/published/workflow?id=95a9070469a895c3, https://usegalaxy.org.au/published/workflow?id=95efb414f5984735, https://usegalaxy.org.au/published/workflow?id=975ddb0eb436f1fd, https://usegalaxy.org.au/published/workflow?id=97628b0604d5ec77, https://usegalaxy.org.au/published/workflow?id=978953f492b2b914, https://usegalaxy.org.au/published/workflow?id=981d7b02416b7734, https://usegalaxy.org.au/published/workflow?id=9898348d95033af2, https://usegalaxy.org.au/published/workflow?id=9988bc2402f32dc1, https://usegalaxy.org.au/published/workflow?id=9b1afa59c087eacd, https://usegalaxy.org.au/published/workflow?id=9cf3860659285656, https://usegalaxy.org.au/published/workflow?id=9df67170c9260539, https://usegalaxy.org.au/published/workflow?id=9e8d9b7055b7c863, https://usegalaxy.org.au/published/workflow?id=9ec9fe2715d7cfdc, https://usegalaxy.org.au/published/workflow?id=a1c59e9f5c7327d2, https://usegalaxy.org.au/published/workflow?id=a35e5a238a70176a, https://usegalaxy.org.au/published/workflow?id=a69db7f914c06407, https://usegalaxy.org.au/published/workflow?id=a7346c25dfdf2e3e, https://usegalaxy.org.au/published/workflow?id=a7726d18d2a5fcf2, https://usegalaxy.org.au/published/workflow?id=a7a30d970584f9ac, https://usegalaxy.org.au/published/workflow?id=a7d1cf2bb636e695, https://usegalaxy.org.au/published/workflow?id=a9fa6101f1fb13e2, https://usegalaxy.org.au/published/workflow?id=aa94c77b720a2810, https://usegalaxy.org.au/published/workflow?id=aaaf580eb00459b6, https://usegalaxy.org.au/published/workflow?id=aae07f802002fdd0, https://usegalaxy.org.au/published/workflow?id=ad13b4a61e8987b1, https://usegalaxy.org.au/published/workflow?id=ae5dca8108d1a8a6, https://usegalaxy.org.au/published/workflow?id=ae6892f1df022097, https://usegalaxy.org.au/published/workflow?id=aed93da6fd6e4e13, https://usegalaxy.org.au/published/workflow?id=af3f40cba4d7afa6, https://usegalaxy.org.au/published/workflow?id=af7b95f46485308c, https://usegalaxy.org.au/published/workflow?id=af990cca72794142, https://usegalaxy.org.au/published/workflow?id=afd9a8bbc07f9c3f, https://usegalaxy.org.au/published/workflow?id=b57bf134896c30ea, https://usegalaxy.org.au/published/workflow?id=b6db525b1b73394b, https://usegalaxy.org.au/published/workflow?id=b8c661ff6b79383b, https://usegalaxy.org.au/published/workflow?id=bda98c5a0ea70dd0, https://usegalaxy.org.au/published/workflow?id=c1a0a3888d9ee3ac, https://usegalaxy.org.au/published/workflow?id=c296b3848708f6f1, https://usegalaxy.org.au/published/workflow?id=c2e5a43356042b30, https://usegalaxy.org.au/published/workflow?id=c3b3717502ff6f76, https://usegalaxy.org.au/published/workflow?id=c3bb21f20a3ce689, https://usegalaxy.org.au/published/workflow?id=c3e198bc2146b4ad, https://usegalaxy.org.au/published/workflow?id=c65a5af6c0383f8c, https://usegalaxy.org.au/published/workflow?id=cb960edf95d2dc35, https://usegalaxy.org.au/published/workflow?id=cef68481b305a516, https://usegalaxy.org.au/published/workflow?id=d126940d33e63036, https://usegalaxy.org.au/published/workflow?id=d2073964dbe9acfa, https://usegalaxy.org.au/published/workflow?id=d5ecfdbc04b24227, https://usegalaxy.org.au/published/workflow?id=d7a65f577fe5f16b, https://usegalaxy.org.au/published/workflow?id=db0bc7357c181222, https://usegalaxy.org.au/published/workflow?id=db3370a412c4de78, https://usegalaxy.org.au/published/workflow?id=dce29c35dfac48b1, https://usegalaxy.org.au/published/workflow?id=de1c04a2caf70647, https://usegalaxy.org.au/published/workflow?id=e045a6064ad5fdc4, https://usegalaxy.org.au/published/workflow?id=e08ef6b519f5fba0, https://usegalaxy.org.au/published/workflow?id=e3cda847fe727b61, https://usegalaxy.org.au/published/workflow?id=e4ddf33ee5b5e259, https://usegalaxy.org.au/published/workflow?id=e52a3479c8216291, https://usegalaxy.org.au/published/workflow?id=e62f743342851151, https://usegalaxy.org.au/published/workflow?id=e826584d78b69467, https://usegalaxy.org.au/published/workflow?id=e8d3651c07ce0bec, https://usegalaxy.org.au/published/workflow?id=eaa841e6b7384633, https://usegalaxy.org.au/published/workflow?id=eb3a4535204383fa, https://usegalaxy.org.au/published/workflow?id=ec79c2fab8eac15d, https://usegalaxy.org.au/published/workflow?id=ed2f72e3537e8a4a, https://usegalaxy.org.au/published/workflow?id=efd39a4be0a9a4e6, https://usegalaxy.org.au/published/workflow?id=f0ced9a5d34b2aff, https://usegalaxy.org.au/published/workflow?id=f19d63d7c9a48160, https://usegalaxy.org.au/published/workflow?id=f268c181a47d0cab, https://usegalaxy.org.au/published/workflow?id=f5c0b0bc5e023e42, https://usegalaxy.org.au/published/workflow?id=f944f2d3fbd1f062, https://usegalaxy.org.au/published/workflow?id=fb644a9649f36f40, https://usegalaxy.org.au/published/workflow?id=fc64a77321752a3c, https://usegalaxy.org.au/published/workflow?id=fc6f8afdee980f0e, https://usegalaxy.org.au/published/workflow?id=fd477917a3d27f61, https://usegalaxy.org.au/published/workflow?id=fd8fb237f8dc24c3, https://usegalaxy.org.au/published/workflow?id=fe1de81a97578368, https://usegalaxy.org.au/published/workflow?id=fe893d72526eba71, https://usegalaxy.org.au/published/workflow?id=ff42213dda7d3e59, https://usegalaxy.org/published/workflow?id=0052e9b1b7bfec69, https://usegalaxy.org/published/workflow?id=00a501a1fada4104, https://usegalaxy.org/published/workflow?id=0292971d1a87abba, https://usegalaxy.org/published/workflow?id=02ff4efb583aaf49, https://usegalaxy.org/published/workflow?id=037c3a9d872be3de, https://usegalaxy.org/published/workflow?id=0a009cf9e88171f1, https://usegalaxy.org/published/workflow?id=0f43fdfe1824e9d7, https://usegalaxy.org/published/workflow?id=13460c791ca97689, https://usegalaxy.org/published/workflow?id=173ecc0c88090d0e, https://usegalaxy.org/published/workflow?id=1dd60e57d1d00406, https://usegalaxy.org/published/workflow?id=204d0da8a023d7da, https://usegalaxy.org/published/workflow?id=2135b0255284752c, https://usegalaxy.org/published/workflow?id=2375a915a5b33b39, https://usegalaxy.org/published/workflow?id=27144c7e44d3f56b, https://usegalaxy.org/published/workflow?id=271b97e5c00c4da4, https://usegalaxy.org/published/workflow?id=2a7fbfc1c17e38bc, https://usegalaxy.org/published/workflow?id=2d44e2b020bf94f8, https://usegalaxy.org/published/workflow?id=303af6cec326947c, https://usegalaxy.org/published/workflow?id=30883502ec515fe2, https://usegalaxy.org/published/workflow?id=31c35b9a93237235, https://usegalaxy.org/published/workflow?id=35209d092e636cd8, https://usegalaxy.org/published/workflow?id=385ac7ec85e37e3e, https://usegalaxy.org/published/workflow?id=3f8e0395007c8e9e, https://usegalaxy.org/published/workflow?id=4061e061f659449b, https://usegalaxy.org/published/workflow?id=454d7310d001a4d0, https://usegalaxy.org/published/workflow?id=4a50ce0e9f4d4bb8, https://usegalaxy.org/published/workflow?id=4aae99b2ca56a36b, https://usegalaxy.org/published/workflow?id=4c0785fe74ab0a24, https://usegalaxy.org/published/workflow?id=4e6005819e57ed35, https://usegalaxy.org/published/workflow?id=52e20808e37c8afd, https://usegalaxy.org/published/workflow?id=544ce8ff6c9f6d4c, https://usegalaxy.org/published/workflow?id=5944c5effd56b0d6, https://usegalaxy.org/published/workflow?id=5c4ce4a781317ebb, https://usegalaxy.org/published/workflow?id=5cb86cdc33296cc2, https://usegalaxy.org/published/workflow?id=63d590e8567f7157, https://usegalaxy.org/published/workflow?id=69af7dd51aff958a, https://usegalaxy.org/published/workflow?id=6a3e47124e32e260, https://usegalaxy.org/published/workflow?id=6c49d9c25ba4ddf5, https://usegalaxy.org/published/workflow?id=6d925811544f594b, https://usegalaxy.org/published/workflow?id=7022c4d9af725e5c, https://usegalaxy.org/published/workflow?id=73ef57dfa532af3d, https://usegalaxy.org/published/workflow?id=7551458d929355c2, https://usegalaxy.org/published/workflow?id=75f14618020c922a, https://usegalaxy.org/published/workflow?id=7a203cf1678df56d, https://usegalaxy.org/published/workflow?id=7b19452fe35b8b17, https://usegalaxy.org/published/workflow?id=7e81153760fb4c90, https://usegalaxy.org/published/workflow?id=812bc1b7a52f4f6f, https://usegalaxy.org/published/workflow?id=8700f593a3d642f5, https://usegalaxy.org/published/workflow?id=8d692df647e2aef3, https://usegalaxy.org/published/workflow?id=8e32b0a5a141543e, https://usegalaxy.org/published/workflow?id=8e9a94fe8e3fef9d, https://usegalaxy.org/published/workflow?id=8ec186ed3f0470e2, https://usegalaxy.org/published/workflow?id=92875b13082b5568, https://usegalaxy.org/published/workflow?id=93bfbd7db42db543, https://usegalaxy.org/published/workflow?id=9c022f4028de0bc6, https://usegalaxy.org/published/workflow?id=9efb061a87bc9f4d, https://usegalaxy.org/published/workflow?id=a743746151101352, https://usegalaxy.org/published/workflow?id=a86b1f1682bb4a62, https://usegalaxy.org/published/workflow?id=ac8b74b911778fbc, https://usegalaxy.org/published/workflow?id=b0e19e7315b64c87, https://usegalaxy.org/published/workflow?id=b2e81f82ccf3afd6, https://usegalaxy.org/published/workflow?id=b300d028da9b8396, https://usegalaxy.org/published/workflow?id=b3823d276917baf0, https://usegalaxy.org/published/workflow?id=b77fa73d4fbde131, https://usegalaxy.org/published/workflow?id=b882703ca6fea3a8, https://usegalaxy.org/published/workflow?id=bb2ac86b0440ae86, https://usegalaxy.org/published/workflow?id=bdbfd1206c59469a, https://usegalaxy.org/published/workflow?id=c24cfddfc88a97a3, https://usegalaxy.org/published/workflow?id=cd431e5f912d3616, https://usegalaxy.org/published/workflow?id=cf3f0760611f22f9, https://usegalaxy.org/published/workflow?id=d5514e327878b531, https://usegalaxy.org/published/workflow?id=d761637d43ee0324, https://usegalaxy.org/published/workflow?id=d8afd017905d6702, https://usegalaxy.org/published/workflow?id=d97a88a8f9061e25, https://usegalaxy.org/published/workflow?id=dab54002b3c0e351, https://usegalaxy.org/published/workflow?id=e22ea88bc0fdc146, https://usegalaxy.org/published/workflow?id=e26fabceece3b858, https://usegalaxy.org/published/workflow?id=e336bca259abd474, https://usegalaxy.org/published/workflow?id=e40c2e7f6b73fb11, https://usegalaxy.org/published/workflow?id=e5a01f746bc9e9b5, https://usegalaxy.org/published/workflow?id=eb01b539d17580c3, https://usegalaxy.org/published/workflow?id=ef2c1fb7d6a51b3a, https://usegalaxy.org/published/workflow?id=efb10012b8448b21, https://usegalaxy.org/published/workflow?id=f47f0b25c1405128, https://usegalaxy.org/published/workflow?id=f5d3d6fe05271c4b, https://usegalaxy.org/published/workflow?id=f961fc46d4fddecc, https://usegalaxy.org/published/workflow?id=fb903de4aa4d5090, https://workflowhub.eu/workflows/1407?version=1, https://workflowhub.eu/workflows/1437?version=1, https://workflowhub.eu/workflows/1673?version=1, https://workflowhub.eu/workflows/1675?version=1, https://workflowhub.eu/workflows/1685?version=1, https://workflowhub.eu/workflows/1692?version=1, https://workflowhub.eu/workflows/1788?version=1, https://workflowhub.eu/workflows/39?version=1, https://workflowhub.eu/workflows/688?version=1, https://workflowhub.eu/workflows/877?version=1</td>
      <td>proteomics/neoantigen-non-reference-database-generation, proteomics/proteogenomics-dbcreation, transcriptomics/de-novo, transcriptomics/minerva-pathways, transcriptomics/rna-seq-reads-to-counts, transcriptomics/srna</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>hypo</td>
      <td>hypo</td>
      <td>Super Fast &amp; Accurate Polisher for Long Read Genome Assemblies</td>
      <td>2021-11-15</td>
      <td>https://github.com/kensung-lab/hypo</td>
      <td>1.0.3</td>
      <td>hypo</td>
      <td>1.0.3</td>
      <td>Up-to-date</td>
      <td>Assembly</td>
      <td>Optimisation and refinement, Genome assembly</td>
      <td>Optimisation and refinement, Genome assembly</td>
      <td>Sequence assembly, Genomics</td>
      <td>Sequence assembly, Genomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hypo</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hypo</td>
      <td>HyPo</td>
      <td>HyPo</td>
      <td>HyPo, a Hybrid Polisher, utilizes short as well as long reads within a single run to polish a long reads assembly of small and large genomes.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>107</td>
      <td>107</td>
      <td>575</td>
      <td>575</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>575</td>
      <td>575</td>
      <td>107</td>
      <td>107</td>
      <td>https://usegalaxy.eu/published/workflow?id=83a1b8949d87e3da, https://workflowhub.eu/workflows/788?version=1, https://workflowhub.eu/workflows/789?version=1</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>interpolation</td>
      <td>interpolation_run_idw_interpolation</td>
      <td>Run IDW interpolation based on a .csv and .geojson file</td>
      <td>2024-01-05</td>
      <td>https://github.com/AquaINFRA/galaxy</td>
      <td>1.0</td>
      <td>r-getopt</td>
      <td></td>
      <td>To update</td>
      <td>Ecology</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/interpolation</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/interpolation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>70</td>
      <td>70</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>70</td>
      <td>70</td>
      <td>7</td>
      <td>7</td>
      <td>https://usegalaxy.eu/published/workflow?id=0b2304aa2083bd1a, https://workflowhub.eu/workflows/807?version=1</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>iphop</td>
      <td>iphop_predict</td>
      <td>Integrated Phage Host Prediction</td>
      <td>2024-10-14</td>
      <td>https://bitbucket.org/srouxjgi/iphop/</td>
      <td>1.3.3</td>
      <td>iphop</td>
      <td>1.4.2</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Sample comparison</td>
      <td>Sample comparison</td>
      <td></td>
      <td></td>
      <td>ufz</td>
      <td>https://github.com/Helmholtz-UFZ/ufz-galaxy-tools/blob/main/tools/iphop/</td>
      <td>https://github.com/Helmholtz-UFZ/galaxy-tools/tree/main/tools/iphop</td>
      <td>iphop</td>
      <td>iPHoP</td>
      <td>An automated command-line pipeline for predicting host genus of novel bacteriophages and archaeoviruses based on their genome sequences.</td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>9</td>
      <td>26</td>
      <td>26</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>26</td>
      <td>26</td>
      <td>9</td>
      <td>9</td>
      <td>https://usegalaxy.eu/published/workflow?id=62392daea25a3980, https://usegalaxy.eu/published/workflow?id=c62d65832377e376</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>jellyfish</td>
      <td>jellyfish</td>
      <td>Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA</td>
      <td>2021-04-07</td>
      <td>https://github.com/gmarcais/Jellyfish</td>
      <td></td>
      <td>kmer-jellyfish</td>
      <td>2.3.1</td>
      <td>To update</td>
      <td>Assembly</td>
      <td>k-mer counting</td>
      <td>k-mer counting</td>
      <td>Sequence analysis, Genomics</td>
      <td>Sequence analysis, Genomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/jellyfish</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/jellyfish</td>
      <td>Jellyfish</td>
      <td>Jellyfish</td>
      <td>A command-line algorithm for counting k-mers in DNA sequence.</td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>369</td>
      <td>369</td>
      <td>4901</td>
      <td>4901</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>87</td>
      <td>87</td>
      <td>1492</td>
      <td>1492</td>
      <td>35</td>
      <td>35</td>
      <td>1519</td>
      <td>1519</td>
      <td>7912</td>
      <td>7912</td>
      <td>491</td>
      <td>491</td>
      <td>https://usegalaxy.eu/published/workflow?id=bc32937d505135ab</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>kaiju</td>
      <td>kaiju_addtaxonnames, kaiju_mergeoutputs, kaiju_kaiju, kaiju_kaiju2krona, kaiju_kaiju2table</td>
      <td>taxonomic classification of high-throughput sequencing reads</td>
      <td>2025-04-22</td>
      <td>https://github.com/bioinformatics-centre/kaiju</td>
      <td>1.10.1</td>
      <td>kaiju</td>
      <td>1.10.1</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Taxonomic classification</td>
      <td>Taxonomic classification</td>
      <td>Metagenomics</td>
      <td>Metagenomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/kaiju</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/kaiju</td>
      <td>kaiju</td>
      <td>Kaiju</td>
      <td>Program for the taxonomic assignment of high-throughput sequencing reads, e.g., Illumina or Roche/454, from whole-genome sequencing of metagenomic DNA. Reads are directly assigned to taxa using the NCBI taxonomy and a reference database of protein sequences from Bacteria, Archaea, Fungi, microbial eukaryotes and viruses.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>kraken2</td>
      <td>kraken2</td>
      <td>Kraken2 for taxonomic designation.</td>
      <td>2019-03-05</td>
      <td>http://ccb.jhu.edu/software/kraken/</td>
      <td>2.17.1</td>
      <td>kraken2</td>
      <td>2.17.1</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Taxonomic classification</td>
      <td>Taxonomic classification</td>
      <td>Taxonomy, Metagenomics, Microbial ecology, Microbiology</td>
      <td>Taxonomy, Metagenomics, Microbial ecology</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/kraken2/kraken2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/kraken2/kraken2</td>
      <td>kraken2</td>
      <td>kraken2</td>
      <td>Kraken 2 is the newest version of Kraken, a taxonomic classification system using exact k-mer matches to achieve high accuracy and fast classification speeds. This classifier matches each k-mer within a query sequence to the lowest common ancestor (LCA) of all genomes containing the given k-mer. The k-mer assignments inform the classification algorithm.Any assumption that Krakens raw read assignments can be directly translated into species or strain-level abundance estimates is flawed. Bracken (Bayesian Reestimation of Abundance after Classification with KrakEN), estimates species abundances in metagenomics samples by probabilistically re-distributing reads in the taxonomic tree. (Lu, Jennifer et al. Bracken: estimating species abundance in metagenomics data.)</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>9806</td>
      <td>9999</td>
      <td>519669</td>
      <td>530279</td>
      <td>7561</td>
      <td>7562</td>
      <td>224146</td>
      <td>224149</td>
      <td>2412</td>
      <td>2459</td>
      <td>87806</td>
      <td>88898</td>
      <td>216</td>
      <td>216</td>
      <td>26628</td>
      <td>26628</td>
      <td>869954</td>
      <td>858249</td>
      <td>20236</td>
      <td>19995</td>
      <td>https://usegalaxy.eu/published/workflow?id=007ce6405d196df3, https://usegalaxy.eu/published/workflow?id=012762cb4acb068b, https://usegalaxy.eu/published/workflow?id=04ed3f21de17aa1d, https://usegalaxy.eu/published/workflow?id=0590554db9794d13, https://usegalaxy.eu/published/workflow?id=08eb5e55109c4d9c, https://usegalaxy.eu/published/workflow?id=0a82eb8b44d9d502, https://usegalaxy.eu/published/workflow?id=0abd32dbc5207b88, https://usegalaxy.eu/published/workflow?id=0bafeb17c21ebc9c, https://usegalaxy.eu/published/workflow?id=0c47571d20535426, https://usegalaxy.eu/published/workflow?id=0f044c9f62773a88, https://usegalaxy.eu/published/workflow?id=10101558b211a782, https://usegalaxy.eu/published/workflow?id=10150b1a188f6d07, https://usegalaxy.eu/published/workflow?id=12a018217ee146ca, https://usegalaxy.eu/published/workflow?id=14714876dda7186d, https://usegalaxy.eu/published/workflow?id=181e751be8dd9abc, https://usegalaxy.eu/published/workflow?id=2497292e98937a5f, https://usegalaxy.eu/published/workflow?id=2b25eaebd3987612, https://usegalaxy.eu/published/workflow?id=2c2f0b4312bae4f4, https://usegalaxy.eu/published/workflow?id=2c79eee5ca5a3b09, https://usegalaxy.eu/published/workflow?id=2e0075791f7fa384, https://usegalaxy.eu/published/workflow?id=2e5e6291cd112225, https://usegalaxy.eu/published/workflow?id=2fa2f67603772413, https://usegalaxy.eu/published/workflow?id=309fc4fc2e0da89c, https://usegalaxy.eu/published/workflow?id=33a052f9a3d5551d, https://usegalaxy.eu/published/workflow?id=344cd3a3c2ce5302, https://usegalaxy.eu/published/workflow?id=39d36a7ce0ef268d, https://usegalaxy.eu/published/workflow?id=3dec7c9b02c3d1ee, https://usegalaxy.eu/published/workflow?id=3ff4dbcc7a602e6b, https://usegalaxy.eu/published/workflow?id=44e16f0782f54111, https://usegalaxy.eu/published/workflow?id=461d71caea871db8, https://usegalaxy.eu/published/workflow?id=47e14222073e7192, https://usegalaxy.eu/published/workflow?id=49c46d7653d6c349, https://usegalaxy.eu/published/workflow?id=4a39d38fde81e35d, https://usegalaxy.eu/published/workflow?id=4a764275d4bf9f31, https://usegalaxy.eu/published/workflow?id=4f0daf8e9dd54127, https://usegalaxy.eu/published/workflow?id=57cfe5062c8e01bf, https://usegalaxy.eu/published/workflow?id=57d3109f32defa3b, https://usegalaxy.eu/published/workflow?id=5826e573b6f93762, https://usegalaxy.eu/published/workflow?id=628062a8cf93261f, https://usegalaxy.eu/published/workflow?id=6a5c39c1d38a10c2, https://usegalaxy.eu/published/workflow?id=6f878d8b43ef1521, https://usegalaxy.eu/published/workflow?id=7491883694fff308, https://usegalaxy.eu/published/workflow?id=7d84b0419296857d, https://usegalaxy.eu/published/workflow?id=884a4415b669690c, https://usegalaxy.eu/published/workflow?id=889f2782ad82bd69, https://usegalaxy.eu/published/workflow?id=8a14cda6140acab7, https://usegalaxy.eu/published/workflow?id=8b2f53745e720d0f, https://usegalaxy.eu/published/workflow?id=8c7e972020799d57, https://usegalaxy.eu/published/workflow?id=8d64614d1971065a, https://usegalaxy.eu/published/workflow?id=8deb08d4f228e198, https://usegalaxy.eu/published/workflow?id=8ee3b635c1c380cd, https://usegalaxy.eu/published/workflow?id=94c3d329c31c712f, https://usegalaxy.eu/published/workflow?id=9588311136ac8794, https://usegalaxy.eu/published/workflow?id=98bfad0c684011e9, https://usegalaxy.eu/published/workflow?id=a0ea6fc756dd17ea, https://usegalaxy.eu/published/workflow?id=a337bfb2eb1e583f, https://usegalaxy.eu/published/workflow?id=a403c661bf1bc4fa, https://usegalaxy.eu/published/workflow?id=a459c311f461adce, https://usegalaxy.eu/published/workflow?id=a705370bc2c13d5c, https://usegalaxy.eu/published/workflow?id=a84bac86ec183d46, https://usegalaxy.eu/published/workflow?id=aafcfcb585c5ca75, https://usegalaxy.eu/published/workflow?id=ad8b6d73c9654305, https://usegalaxy.eu/published/workflow?id=b14845359b702444, https://usegalaxy.eu/published/workflow?id=b2b46e71046b1999, https://usegalaxy.eu/published/workflow?id=b37800d990cceb83, https://usegalaxy.eu/published/workflow?id=b4809f13c8f0c0be, https://usegalaxy.eu/published/workflow?id=b659069edd9b1d4e, https://usegalaxy.eu/published/workflow?id=b67b5ecd3305f830, https://usegalaxy.eu/published/workflow?id=bb63b5bb983fd49c, https://usegalaxy.eu/published/workflow?id=bd4e3cdc7f29049b, https://usegalaxy.eu/published/workflow?id=bdc7cfaac0db2eab, https://usegalaxy.eu/published/workflow?id=c09159d7aad0f264, https://usegalaxy.eu/published/workflow?id=c14d2e8b3c191bc4, https://usegalaxy.eu/published/workflow?id=c38e3ac59d8b86d6, https://usegalaxy.eu/published/workflow?id=c4fc3adf8f7ff2bc, https://usegalaxy.eu/published/workflow?id=c930a54a30dcc7e3, https://usegalaxy.eu/published/workflow?id=ca80039c4933aa0a, https://usegalaxy.eu/published/workflow?id=cb2df493f5e334f7, https://usegalaxy.eu/published/workflow?id=cfd37e9e307e5d3d, https://usegalaxy.eu/published/workflow?id=d0ae255151450dad, https://usegalaxy.eu/published/workflow?id=d88145e7475e0b7b, https://usegalaxy.eu/published/workflow?id=e69b555faa6bb962, https://usegalaxy.eu/published/workflow?id=ebb131f2a6172081, https://usegalaxy.eu/published/workflow?id=ec6b2c926fffab5f, https://usegalaxy.eu/published/workflow?id=ed2df13cfbc86b99, https://usegalaxy.eu/published/workflow?id=ee0005a5487653ba, https://usegalaxy.eu/published/workflow?id=ef6556d65ce5d72c, https://usegalaxy.eu/published/workflow?id=f97e6fe1fcbc53a7, https://usegalaxy.eu/published/workflow?id=fde21b5f5c169114, https://usegalaxy.fr/published/workflow?id=3d3b40398d1cd1ae, https://usegalaxy.org.au/published/workflow?id=0edb9c8b767fa74b, https://usegalaxy.org.au/published/workflow?id=131636a795bac485, https://usegalaxy.org.au/published/workflow?id=25d52afddaa3451b, https://usegalaxy.org.au/published/workflow?id=29ce205d98faf47e, https://usegalaxy.org.au/published/workflow?id=3d47e0de658b3304, https://usegalaxy.org.au/published/workflow?id=4047f9f242542716, https://usegalaxy.org.au/published/workflow?id=4c7e0703aace6aa3, https://usegalaxy.org.au/published/workflow?id=52065a4100b11184, https://usegalaxy.org.au/published/workflow?id=5693811cd933d2a2, https://usegalaxy.org.au/published/workflow?id=59618ed15510f531, https://usegalaxy.org.au/published/workflow?id=5fb44f257e445f56, https://usegalaxy.org.au/published/workflow?id=69d91340fc7effa2, https://usegalaxy.org.au/published/workflow?id=7a0c4804e4d4c905, https://usegalaxy.org.au/published/workflow?id=8034bd0690b4cbe8, https://usegalaxy.org.au/published/workflow?id=83441e3de699d157, https://usegalaxy.org.au/published/workflow?id=897a1255cef5c310, https://usegalaxy.org.au/published/workflow?id=92fcdd490831c6c5, https://usegalaxy.org.au/published/workflow?id=97e55a900db3863b, https://usegalaxy.org.au/published/workflow?id=a335b9c9103b57ad, https://usegalaxy.org.au/published/workflow?id=a61d4bda54de7cd1, https://usegalaxy.org.au/published/workflow?id=c92fead7c30c7497, https://usegalaxy.org.au/published/workflow?id=c997a34b55425155, https://usegalaxy.org.au/published/workflow?id=d2dcd3b54e5082a6, https://usegalaxy.org.au/published/workflow?id=d417e920ff97e4c1, https://usegalaxy.org.au/published/workflow?id=d720c98f7227b6d5, https://usegalaxy.org.au/published/workflow?id=d9ba165e6ae55417, https://usegalaxy.org.au/published/workflow?id=e20172eb0f30d6cd, https://usegalaxy.org.au/published/workflow?id=ed3e48a73b5aaa56, https://usegalaxy.org/published/workflow?id=1473cdcbd664d147, https://usegalaxy.org/published/workflow?id=1b80302437db5937, https://usegalaxy.org/published/workflow?id=1be5c982c2d36a6c, https://usegalaxy.org/published/workflow?id=21161075002d6778, https://usegalaxy.org/published/workflow?id=2a782ad24271f570, https://usegalaxy.org/published/workflow?id=3011f5dbf884c9bc, https://usegalaxy.org/published/workflow?id=323c651ca583c9e5, https://usegalaxy.org/published/workflow?id=4b5f6b5d4f36e38b, https://usegalaxy.org/published/workflow?id=4f698eaeef30e364, https://usegalaxy.org/published/workflow?id=574e42683dc3961b, https://usegalaxy.org/published/workflow?id=5b74b780d3955eb3, https://usegalaxy.org/published/workflow?id=5c259623a010fe77, https://usegalaxy.org/published/workflow?id=5e38c7b36e4918ad, https://usegalaxy.org/published/workflow?id=605160c81d098703, https://usegalaxy.org/published/workflow?id=60f0d0d5d6e93941, https://usegalaxy.org/published/workflow?id=623fc80c7f91a33a, https://usegalaxy.org/published/workflow?id=7321c6ad5425d347, https://usegalaxy.org/published/workflow?id=7c9a55db0b0eb178, https://usegalaxy.org/published/workflow?id=8f5904693b5f74f4, https://usegalaxy.org/published/workflow?id=9ba6da7dae465d3e, https://usegalaxy.org/published/workflow?id=9f68718f025bed30, https://usegalaxy.org/published/workflow?id=a63927dd19759714, https://usegalaxy.org/published/workflow?id=a8b9252fad5fe06a, https://usegalaxy.org/published/workflow?id=b68ec42db8953569, https://usegalaxy.org/published/workflow?id=b6a795834009ce06, https://usegalaxy.org/published/workflow?id=b7817a12dbee583c, https://usegalaxy.org/published/workflow?id=b7fe3873ee5e3a02, https://usegalaxy.org/published/workflow?id=badd4b56dbdeaba7, https://usegalaxy.org/published/workflow?id=bdd7011a908bc1aa, https://usegalaxy.org/published/workflow?id=c3ea8fa3bb202dc6, https://usegalaxy.org/published/workflow?id=c8f9e048ddae7d03, https://usegalaxy.org/published/workflow?id=cace9f1359874352, https://usegalaxy.org/published/workflow?id=d2c408df40dc311b, https://usegalaxy.org/published/workflow?id=d4f0a7ccb09f1e55, https://usegalaxy.org/published/workflow?id=d5436cee11a34de9, https://usegalaxy.org/published/workflow?id=ddf7a594b9609d68, https://usegalaxy.org/published/workflow?id=de9bf4c1024f8d53, https://usegalaxy.org/published/workflow?id=dfd73857c0eb2a21, https://usegalaxy.org/published/workflow?id=e12324f5e3b930fd, https://usegalaxy.org/published/workflow?id=e7ce46762faccaac, https://usegalaxy.org/published/workflow?id=e887adf3afc3253d, https://usegalaxy.org/published/workflow?id=ea206593698adbfd, https://usegalaxy.org/published/workflow?id=f378960ee55f567e, https://usegalaxy.org/published/workflow?id=fc2b48600853f3ea, https://workflowhub.eu/workflows/100?version=1, https://workflowhub.eu/workflows/101?version=1, https://workflowhub.eu/workflows/1035?version=1, https://workflowhub.eu/workflows/1052?version=11, https://workflowhub.eu/workflows/1059?version=1, https://workflowhub.eu/workflows/1061?version=1, https://workflowhub.eu/workflows/1199?version=2, https://workflowhub.eu/workflows/124?version=1, https://workflowhub.eu/workflows/1439?version=1, https://workflowhub.eu/workflows/1470?version=2, https://workflowhub.eu/workflows/1473?version=1, https://workflowhub.eu/workflows/1483?version=1, https://workflowhub.eu/workflows/1492?version=2, https://workflowhub.eu/workflows/1594?version=1, https://workflowhub.eu/workflows/1644?version=1, https://workflowhub.eu/workflows/1647?version=1, https://workflowhub.eu/workflows/1674?version=2, https://workflowhub.eu/workflows/1881?version=2, https://workflowhub.eu/workflows/1882?version=3, https://workflowhub.eu/workflows/2043?version=1, https://workflowhub.eu/workflows/53?version=1</td>
      <td>assembly/assembly-decontamination, ecology/bacterial-isolate-species-contamination-checking, microbiome/beer-data-analysis, microbiome/nanopore-16S-metagenomics, microbiome/pathogen-detection-from-nanopore-foodborne-data, microbiome/taxonomic-profiling, sequence-analysis/quality-contamination-control, variant-analysis/tb-variant-analysis</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>lca_wrapper</td>
      <td>lca1</td>
      <td>Find lowest diagnostic rank</td>
      <td>2014-01-27</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>1.0.1</td>
      <td>taxonomy</td>
      <td>0.10.2</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/taxonomy/lca_wrapper</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tool_collections/taxonomy/lca_wrapper</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>3</td>
      <td>20</td>
      <td>11</td>
      <td>137</td>
      <td>14</td>
      <td>1112</td>
      <td>112</td>
      <td>6136</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6273</td>
      <td>123</td>
      <td>1132</td>
      <td>17</td>
      <td>https://usegalaxy.org/published/workflow?id=19c9c8cc3c813bda, https://usegalaxy.org/published/workflow?id=2f7638035fb889ac, https://usegalaxy.org/published/workflow?id=55aa743125c2465d, https://usegalaxy.org/published/workflow?id=59347cc210c4cdbc, https://usegalaxy.org/published/workflow?id=7294f75ad01ca7ed, https://usegalaxy.org/published/workflow?id=7b67ffcf1b7c2232, https://usegalaxy.org/published/workflow?id=85d825931edf4a64, https://usegalaxy.org/published/workflow?id=896d29573ea3e574, https://usegalaxy.org/published/workflow?id=d851f8436497dd1c</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>links</td>
      <td>links</td>
      <td>Scaffold genome assemblies with long reads.</td>
      <td>2022-02-23</td>
      <td>https://github.com/bcgsc/LINKS</td>
      <td>2.0.1</td>
      <td>links</td>
      <td>2.0.1</td>
      <td>Up-to-date</td>
      <td>Assembly</td>
      <td>Scaffolding, Genome assembly, Read mapping, Read pre-processing, Sequence trimming</td>
      <td>Scaffolding, Genome assembly, Read mapping, Read pre-processing, Sequence trimming</td>
      <td>Sequence assembly, Mapping, Sequencing</td>
      <td>Sequence assembly, Mapping, Sequencing</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/links</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/links</td>
      <td>links</td>
      <td>LINKS</td>
      <td>LINKS (Long Interval Nucleotide K-mer Scaffolder) is a genomics application for scaffolding genome assemblies with long reads, such as those produced by Oxford Nanopore Technologies Ltd. It can be used to scaffold high-quality draft genome assemblies with any long sequences (eg. ONT reads, PacBio reads, other draft genomes, etc). It is also used to scaffold contig pairs linked by ARCS/ARKS.</td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>181</td>
      <td>181</td>
      <td>940</td>
      <td>940</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>87</td>
      <td>87</td>
      <td>778</td>
      <td>778</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1718</td>
      <td>1718</td>
      <td>268</td>
      <td>268</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>lotus2</td>
      <td>lotus2</td>
      <td>LotuS2 OTU processing pipeline</td>
      <td>2021-05-13</td>
      <td>https://lotus2.earlham.ac.uk/</td>
      <td>2.32</td>
      <td>lotus2</td>
      <td>2.34.1</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Sequence feature detection, DNA barcoding</td>
      <td>Sequence feature detection, DNA barcoding</td>
      <td>Metagenomics, Taxonomy, Microbial ecology</td>
      <td>Metagenomics, Taxonomy, Microbial ecology</td>
      <td>earlhaminst</td>
      <td>https://github.com/EarlhamInst/earlham-galaxytools/tree/main/tools/lotus2</td>
      <td>https://github.com/EarlhamInst/earlham-galaxytools/tree/main/tools/lotus2</td>
      <td>lotus2</td>
      <td>lotus2</td>
      <td>LotuS2 is a lightweight and user-friendly pipeline that is fast, precise, and streamlined, using extensive pre- and post-ASV/OTU clustering steps to further increase data quality. High data usage rates and reliability enable high-throughput microbiome analysis in minutes.</td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>249</td>
      <td>249</td>
      <td>1979</td>
      <td>1979</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>22</td>
      <td>22</td>
      <td>417</td>
      <td>417</td>
      <td>2</td>
      <td>2</td>
      <td>5</td>
      <td>5</td>
      <td>2401</td>
      <td>2401</td>
      <td>273</td>
      <td>273</td>
      <td>https://usegalaxy.eu/published/workflow?id=466bdd8ba7b67264, https://workflowhub.eu/workflows/1460?version=1</td>
      <td>microbiome/lotus2-identifying-fungi, microbiome/mgnify-amplicon</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>mapseq</td>
      <td>mapseq</td>
      <td>fast and accurate sequence read classification tool designed to assign taxonomy and OTU classifications to ribosomal RNA sequences.</td>
      <td>2023-08-02</td>
      <td>https://github.com/jfmrod/MAPseq</td>
      <td>2.1.1b</td>
      <td>perl</td>
      <td></td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>k-mer counting</td>
      <td>k-mer counting</td>
      <td>Functional, regulatory and non-coding RNA, Sequence analysis, Sequence sites, features and motifs</td>
      <td>Functional, regulatory and non-coding RNA, Sequence analysis, Sequence sites, features and motifs</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mapseq</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mapseq</td>
      <td>mapseq</td>
      <td>MAPseq</td>
      <td>Highly efficient k-mer search with confidence estimates, for rRNA sequence analysis .</td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>64</td>
      <td>64</td>
      <td>9612</td>
      <td>9612</td>
      <td>24</td>
      <td>24</td>
      <td>434</td>
      <td>434</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10046</td>
      <td>10046</td>
      <td>88</td>
      <td>88</td>
      <td>https://usegalaxy.eu/published/workflow?id=08e25b733cd796e6, https://usegalaxy.eu/published/workflow?id=437704898229dfb6, https://usegalaxy.eu/published/workflow?id=fb806b88850f3427, https://usegalaxy.org/published/workflow?id=45f959d5bf505b15, https://usegalaxy.org/published/workflow?id=8151be18aab3a3c6, https://workflowhub.eu/workflows/1270?version=2, https://workflowhub.eu/workflows/1273?version=2, https://workflowhub.eu/workflows/1842?version=1, https://workflowhub.eu/workflows/1856?version=1</td>
      <td>microbiome/mgnify-amplicon</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>marine_omics</td>
      <td>sanntis_marine</td>
      <td>The Sanntis tool identify biosynthetic gene clusters (BGCs) in genomic &amp; metagenomic data</td>
      <td>2024-07-26</td>
      <td>https://github.com/Finn-Lab/SanntiS</td>
      <td>0.9.3.5</td>
      <td>sanntis</td>
      <td>0.9.4.1</td>
      <td>To update</td>
      <td>Ecology</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/marine_omics</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/marine_omics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>22</td>
      <td>22</td>
      <td>198</td>
      <td>198</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>112</td>
      <td>112</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>310</td>
      <td>310</td>
      <td>27</td>
      <td>27</td>
      <td>https://usegalaxy.eu/published/workflow?id=a448400f69594135, https://usegalaxy.eu/published/workflow?id=b9c938d1af08124b, https://usegalaxy.eu/published/workflow?id=cc59ae1a9878942c, https://workflowhub.eu/workflows/1663?version=1</td>
      <td>ecology/marine_omics_bgc</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>mash</td>
      <td>mash_dist, mash_paste, mash_screen, mash_sketch</td>
      <td>Fast genome and metagenome distance estimation using MinHash</td>
      <td>2018-12-07</td>
      <td>https://github.com/marbl/Mash</td>
      <td>2.3</td>
      <td>mash</td>
      <td>2.3</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Sequence distance matrix generation</td>
      <td>Sequence distance matrix generation</td>
      <td>Genomics, Metagenomics, Statistics and probability, Sequence analysis, DNA mutation</td>
      <td>Metagenomics, Statistics and probability, Sequence analysis, DNA mutation</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mash</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mash</td>
      <td>mash</td>
      <td>Mash</td>
      <td>Fast genome and metagenome distance estimation using MinHash.</td>
      <td></td>
      <td>4</td>
      <td>2</td>
      <td>4</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>4</td>
      <td>2</td>
      <td>0</td>
      <td>49</td>
      <td>57</td>
      <td>10768</td>
      <td>10867</td>
      <td>18</td>
      <td>18</td>
      <td>922</td>
      <td>922</td>
      <td>18</td>
      <td>18</td>
      <td>3187</td>
      <td>3187</td>
      <td>3</td>
      <td>3</td>
      <td>29</td>
      <td>29</td>
      <td>15005</td>
      <td>14906</td>
      <td>96</td>
      <td>88</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>mean_per_zone</td>
      <td>mean_per_zone</td>
      <td>Creates a png image showing statistic over areas as defined in the vector file</td>
      <td>2019-03-15</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/blob/master/tools/mean-per-zone/</td>
      <td>0.2.0</td>
      <td>python</td>
      <td></td>
      <td>To update</td>
      <td>Visualization, GIS, Climate Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>climate</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/mean-per-zone</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/mean-per-zone</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>3</td>
      <td>1</td>
      <td>25</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>25</td>
      <td>1</td>
      <td>3</td>
      <td>1</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>medenv</td>
      <td>iabiodiv_smartbiodiv_med_environ</td>
      <td>Retrieve environmental data from etopo, cmems and woa</td>
      <td>2023-11-30</td>
      <td>https://github.com/jeremyfix/medenv</td>
      <td>0.1.1</td>
      <td>pandas</td>
      <td></td>
      <td>To update</td>
      <td>Ecology, Data Source</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/jeremyfix/medenv</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/medenv</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>11</td>
      <td>11</td>
      <td>38</td>
      <td>38</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>38</td>
      <td>38</td>
      <td>11</td>
      <td>11</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>merqury</td>
      <td>merqury, merquryplot</td>
      <td>Merqury is a tool for evaluating genomes assemblies based of k-mer operations.</td>
      <td>2021-04-18</td>
      <td>https://github.com/marbl/merqury</td>
      <td>1.3</td>
      <td>merqury</td>
      <td>1.3</td>
      <td>Up-to-date</td>
      <td>Assembly</td>
      <td>Genome assembly, k-mer counting, Scaffolding, Phasing, De-novo assembly</td>
      <td>Genome assembly, k-mer counting, Scaffolding, Phasing, De-novo assembly</td>
      <td>Sequence assembly, Whole genome sequencing, Plant biology</td>
      <td>Sequence assembly, Whole genome sequencing, Plant biology</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/merqury</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/merqury</td>
      <td>merqury</td>
      <td>Merqury</td>
      <td>Reference-free quality, completeness, and phasing assessment for genome assemblies.Evaluate genome assemblies with k-mers and more.Often, genome assembly projects have illumina whole genome sequencing reads available for the assembled individual.Merqury provides a set of tools for this purpose.</td>
      <td></td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>661</td>
      <td>661</td>
      <td>4657</td>
      <td>4657</td>
      <td>264</td>
      <td>264</td>
      <td>1645</td>
      <td>1645</td>
      <td>243</td>
      <td>243</td>
      <td>1945</td>
      <td>1945</td>
      <td>77</td>
      <td>77</td>
      <td>440</td>
      <td>440</td>
      <td>8687</td>
      <td>8687</td>
      <td>1245</td>
      <td>1245</td>
      <td>https://usegalaxy.eu/published/workflow?id=002a51bdef00e14d, https://usegalaxy.eu/published/workflow?id=01fe4c590e608103, https://usegalaxy.eu/published/workflow?id=023a0ee62dd1f593, https://usegalaxy.eu/published/workflow?id=032a7281a86eba0d, https://usegalaxy.eu/published/workflow?id=03bb4c20a1f4b4de, https://usegalaxy.eu/published/workflow?id=03be9d62e8e4a129, https://usegalaxy.eu/published/workflow?id=0444c98a3ca2a3ca, https://usegalaxy.eu/published/workflow?id=04844dddcc4e8aa7, https://usegalaxy.eu/published/workflow?id=085c3a4c89d137b5, https://usegalaxy.eu/published/workflow?id=089e5e83b1de0511, https://usegalaxy.eu/published/workflow?id=0a3ebee985c26060, https://usegalaxy.eu/published/workflow?id=0f207a15b6f7fa10, https://usegalaxy.eu/published/workflow?id=1151052f4393852a, https://usegalaxy.eu/published/workflow?id=122da0054fd40adf, https://usegalaxy.eu/published/workflow?id=12b0bd336c76a874, https://usegalaxy.eu/published/workflow?id=1483534f49b826bf, https://usegalaxy.eu/published/workflow?id=17b0a6d3379a4392, https://usegalaxy.eu/published/workflow?id=194a05d42687b524, https://usegalaxy.eu/published/workflow?id=19bb57f8cfa48bb9, https://usegalaxy.eu/published/workflow?id=1ac0a15758277730, https://usegalaxy.eu/published/workflow?id=1af73dc855d0a5ed, https://usegalaxy.eu/published/workflow?id=1bb084f1bc102734, https://usegalaxy.eu/published/workflow?id=1cba6eb4aff8dc61, https://usegalaxy.eu/published/workflow?id=1d1ae39386b5c318, https://usegalaxy.eu/published/workflow?id=1da7ac8eaf891f39, https://usegalaxy.eu/published/workflow?id=1e02a7292e9974b1, https://usegalaxy.eu/published/workflow?id=1e7f4ba6261f979c, https://usegalaxy.eu/published/workflow?id=1f2a1438e41188f2, https://usegalaxy.eu/published/workflow?id=21a3f72851f29925, https://usegalaxy.eu/published/workflow?id=2294ef9515bcf972, https://usegalaxy.eu/published/workflow?id=2671a5f85b2cccfe, https://usegalaxy.eu/published/workflow?id=270b6f82efab28b2, https://usegalaxy.eu/published/workflow?id=29c97d43eeb8ed63, https://usegalaxy.eu/published/workflow?id=2e418c5810012e5a, https://usegalaxy.eu/published/workflow?id=3069c17b52decab4, https://usegalaxy.eu/published/workflow?id=3192f92ac4d389b8, https://usegalaxy.eu/published/workflow?id=342a5df69951e3d7, https://usegalaxy.eu/published/workflow?id=3480204a99bf4a35, https://usegalaxy.eu/published/workflow?id=379642036b6f5a66, https://usegalaxy.eu/published/workflow?id=399048afb4acef3d, https://usegalaxy.eu/published/workflow?id=3a3e2d50fe7a9ee3, https://usegalaxy.eu/published/workflow?id=3e255a5b7026d26e, https://usegalaxy.eu/published/workflow?id=3eb6663a66f8e69a, https://usegalaxy.eu/published/workflow?id=3f5414bb38b47fcf, https://usegalaxy.eu/published/workflow?id=416141b245073406, https://usegalaxy.eu/published/workflow?id=45ae834d842302c3, https://usegalaxy.eu/published/workflow?id=471eb8b4c54f897d, https://usegalaxy.eu/published/workflow?id=475ea16f943cc013, https://usegalaxy.eu/published/workflow?id=4856e3522b2ee059, https://usegalaxy.eu/published/workflow?id=497b82d3b92dc076, https://usegalaxy.eu/published/workflow?id=4c3a9b574f37eebb, https://usegalaxy.eu/published/workflow?id=4d41ee83fc0040d4, https://usegalaxy.eu/published/workflow?id=5115299e77df8104, https://usegalaxy.eu/published/workflow?id=54590ccd11ffbf65, https://usegalaxy.eu/published/workflow?id=56523b41ff7eec76, https://usegalaxy.eu/published/workflow?id=57cc33ed64d71982, https://usegalaxy.eu/published/workflow?id=5ad406442ec21345, https://usegalaxy.eu/published/workflow?id=5bf6718e7d6158d2, https://usegalaxy.eu/published/workflow?id=5c51f26ef8b20633, https://usegalaxy.eu/published/workflow?id=5e1906290f5b4d8a, https://usegalaxy.eu/published/workflow?id=6573f3b1606b7f09, https://usegalaxy.eu/published/workflow?id=6865f33cffca4922, https://usegalaxy.eu/published/workflow?id=69be0fb8276753ec, https://usegalaxy.eu/published/workflow?id=69cdd404592535ec, https://usegalaxy.eu/published/workflow?id=6c14b08a7ef7e79a, https://usegalaxy.eu/published/workflow?id=700fa3ab11a8f3e2, https://usegalaxy.eu/published/workflow?id=7137e45ca8dc6e4c, https://usegalaxy.eu/published/workflow?id=71a6d128a09210f0, https://usegalaxy.eu/published/workflow?id=7337d15d121596da, https://usegalaxy.eu/published/workflow?id=79b5afb125e3c3da, https://usegalaxy.eu/published/workflow?id=847403b72dd346ff, https://usegalaxy.eu/published/workflow?id=8be366ad6a42408b, https://usegalaxy.eu/published/workflow?id=8f105de48f7f1d9d, https://usegalaxy.eu/published/workflow?id=915ffef5b14c871e, https://usegalaxy.eu/published/workflow?id=9207e74a5bf7d8e1, https://usegalaxy.eu/published/workflow?id=92760fc19ef120ce, https://usegalaxy.eu/published/workflow?id=92dafb3d1c06e074, https://usegalaxy.eu/published/workflow?id=95f420d8856bf725, https://usegalaxy.eu/published/workflow?id=9667bf36217083b7, https://usegalaxy.eu/published/workflow?id=9c0b67496dfe52b2, https://usegalaxy.eu/published/workflow?id=9d209bcdae1a475e, https://usegalaxy.eu/published/workflow?id=9f2d18f0d0896e8b, https://usegalaxy.eu/published/workflow?id=a3f082ff30357040, https://usegalaxy.eu/published/workflow?id=a6dd62ab4b9a0725, https://usegalaxy.eu/published/workflow?id=a9c1dca5da7170dd, https://usegalaxy.eu/published/workflow?id=aa6dc57f5c67fb4b, https://usegalaxy.eu/published/workflow?id=ac6eb99f70d35999, https://usegalaxy.eu/published/workflow?id=ade6f36f6a035a73, https://usegalaxy.eu/published/workflow?id=ae74f60cd20d56bc, https://usegalaxy.eu/published/workflow?id=aea88bfca6d1d476, https://usegalaxy.eu/published/workflow?id=aef62f4d0bba003b, https://usegalaxy.eu/published/workflow?id=b211fceaecf41c9e, https://usegalaxy.eu/published/workflow?id=b4948afacccfefdb, https://usegalaxy.eu/published/workflow?id=b5de94e9cde071ab, https://usegalaxy.eu/published/workflow?id=b8baade87e733fce, https://usegalaxy.eu/published/workflow?id=bac84a7033c3423f, https://usegalaxy.eu/published/workflow?id=bc32937d505135ab, https://usegalaxy.eu/published/workflow?id=bc87309a8763d0b4, https://usegalaxy.eu/published/workflow?id=bcf99cc0469db276, https://usegalaxy.eu/published/workflow?id=bef0a7d604f2ebef, https://usegalaxy.eu/published/workflow?id=c4c2e669b86cffbc, https://usegalaxy.eu/published/workflow?id=c508c00e81018dd8, https://usegalaxy.eu/published/workflow?id=ca39822372847670, https://usegalaxy.eu/published/workflow?id=caba6d3d7db648e9, https://usegalaxy.eu/published/workflow?id=d02458f1d7d9c3f4, https://usegalaxy.eu/published/workflow?id=d172f927a489e9f8, https://usegalaxy.eu/published/workflow?id=d17d6029914d666b, https://usegalaxy.eu/published/workflow?id=d1bd5d8597c1d31d, https://usegalaxy.eu/published/workflow?id=d261bbd05889ce75, https://usegalaxy.eu/published/workflow?id=d67299c34e63f3d9, https://usegalaxy.eu/published/workflow?id=d924fafd8e042952, https://usegalaxy.eu/published/workflow?id=dc27453c438577f6, https://usegalaxy.eu/published/workflow?id=de9cf573314f416c, https://usegalaxy.eu/published/workflow?id=dfe3b555856c7b44, https://usegalaxy.eu/published/workflow?id=e138640778b7414c, https://usegalaxy.eu/published/workflow?id=e1947b059ef131d2, https://usegalaxy.eu/published/workflow?id=e2f936f1aa43a629, https://usegalaxy.eu/published/workflow?id=e7aebe58411d97ed, https://usegalaxy.eu/published/workflow?id=e8f9e1d911683b22, https://usegalaxy.eu/published/workflow?id=e935d2c8ffb52588, https://usegalaxy.eu/published/workflow?id=ec79f14ab91cd933, https://usegalaxy.eu/published/workflow?id=f08b29e4cef76cd5, https://usegalaxy.eu/published/workflow?id=f1ca4289ee4f67bb, https://usegalaxy.eu/published/workflow?id=f38745208494d4ab, https://usegalaxy.eu/published/workflow?id=f3e17c8dd80def27, https://usegalaxy.eu/published/workflow?id=f578a097b4339025, https://usegalaxy.eu/published/workflow?id=f6a625946dea60fa, https://usegalaxy.eu/published/workflow?id=f7cdaf862e6599c0, https://usegalaxy.eu/published/workflow?id=f882d14fc5dabbb9, https://usegalaxy.eu/published/workflow?id=f91245e580319e65, https://usegalaxy.eu/published/workflow?id=fb015e398373ea8a, https://usegalaxy.eu/published/workflow?id=fba1671d835b6dd3, https://usegalaxy.eu/published/workflow?id=fc25c1f8c6667c87, https://usegalaxy.eu/published/workflow?id=fd649ac4d4872c81, https://usegalaxy.org.au/published/workflow?id=000c2f1884bc4687, https://usegalaxy.org.au/published/workflow?id=02eebe3d909206ab, https://usegalaxy.org.au/published/workflow?id=06564b19b0655c19, https://usegalaxy.org.au/published/workflow?id=0810be3d7d2b0322, https://usegalaxy.org.au/published/workflow?id=08fac3b4c128e597, https://usegalaxy.org.au/published/workflow?id=094655df3ccd0b00, https://usegalaxy.org.au/published/workflow?id=0af0ea50f1237952, https://usegalaxy.org.au/published/workflow?id=0c11e1ba5e8fa21b, https://usegalaxy.org.au/published/workflow?id=0cc8c8875df6bccb, https://usegalaxy.org.au/published/workflow?id=0df25e2dbd5e8ed5, https://usegalaxy.org.au/published/workflow?id=10355b2680374a03, https://usegalaxy.org.au/published/workflow?id=1905ec04d5841f09, https://usegalaxy.org.au/published/workflow?id=1ce2a725df3adb74, https://usegalaxy.org.au/published/workflow?id=1dcc1b25c9091fbf, https://usegalaxy.org.au/published/workflow?id=1eb3e1452c672480, https://usegalaxy.org.au/published/workflow?id=1f49c65625b9d2de, https://usegalaxy.org.au/published/workflow?id=2356584eac7fa234, https://usegalaxy.org.au/published/workflow?id=2470cb61ba51288c, https://usegalaxy.org.au/published/workflow?id=269b42605a2a87d2, https://usegalaxy.org.au/published/workflow?id=2d45d7138d8715b4, https://usegalaxy.org.au/published/workflow?id=319e9d693fac8dce, https://usegalaxy.org.au/published/workflow?id=328adfa7d90b1d54, https://usegalaxy.org.au/published/workflow?id=3604307ec2835628, https://usegalaxy.org.au/published/workflow?id=36f54eda8568355d, https://usegalaxy.org.au/published/workflow?id=371c9892d261cee2, https://usegalaxy.org.au/published/workflow?id=3c30c822058187be, https://usegalaxy.org.au/published/workflow?id=3df7f45b7b388f8c, https://usegalaxy.org.au/published/workflow?id=40666ceb68d101ac, https://usegalaxy.org.au/published/workflow?id=40740c32b9a550c9, https://usegalaxy.org.au/published/workflow?id=41282d51f9112552, https://usegalaxy.org.au/published/workflow?id=41795303a88d9e94, https://usegalaxy.org.au/published/workflow?id=4267bd1e4cfd1901, https://usegalaxy.org.au/published/workflow?id=43343fa1896f8278, https://usegalaxy.org.au/published/workflow?id=434127690544d0e9, https://usegalaxy.org.au/published/workflow?id=447d8daefdefafee, https://usegalaxy.org.au/published/workflow?id=478c79365fc9677f, https://usegalaxy.org.au/published/workflow?id=47cd0a90609fd557, https://usegalaxy.org.au/published/workflow?id=4a404aa3e9f29e5a, https://usegalaxy.org.au/published/workflow?id=4b5f80656fc94b88, https://usegalaxy.org.au/published/workflow?id=4cb69c84b73ea9dd, https://usegalaxy.org.au/published/workflow?id=4e7930c7286c6e9e, https://usegalaxy.org.au/published/workflow?id=4ebf1297793870e7, https://usegalaxy.org.au/published/workflow?id=51bad198f1fd1215, https://usegalaxy.org.au/published/workflow?id=543fb2d9431a7e13, https://usegalaxy.org.au/published/workflow?id=5550585ffc3d8d0e, https://usegalaxy.org.au/published/workflow?id=5b0c0e34394afed1, https://usegalaxy.org.au/published/workflow?id=5cd804644a34c8fd, https://usegalaxy.org.au/published/workflow?id=5f4f874e49515555, https://usegalaxy.org.au/published/workflow?id=60c0dec52cdca708, https://usegalaxy.org.au/published/workflow?id=63789b6f82172679, https://usegalaxy.org.au/published/workflow?id=68b2961117e62890, https://usegalaxy.org.au/published/workflow?id=6c27019d03dd49d8, https://usegalaxy.org.au/published/workflow?id=6c9129acb9d98afe, https://usegalaxy.org.au/published/workflow?id=6ec0031406b10635, https://usegalaxy.org.au/published/workflow?id=6f76c43e6d2d7f74, https://usegalaxy.org.au/published/workflow?id=7168c01be9997823, https://usegalaxy.org.au/published/workflow?id=7909da83448d8bf4, https://usegalaxy.org.au/published/workflow?id=79bf7b0f5e9052d7, https://usegalaxy.org.au/published/workflow?id=81a5b96452c71a52, https://usegalaxy.org.au/published/workflow?id=82d0876385b5ac60, https://usegalaxy.org.au/published/workflow?id=8926e592f2e60863, https://usegalaxy.org.au/published/workflow?id=8a6233015db6dd24, https://usegalaxy.org.au/published/workflow?id=8bf8c9929249f37d, https://usegalaxy.org.au/published/workflow?id=8dd86497bc15d981, https://usegalaxy.org.au/published/workflow?id=8e8a5cce9f3f2f67, https://usegalaxy.org.au/published/workflow?id=91ee716be8bfd508, https://usegalaxy.org.au/published/workflow?id=9642aadb83c4f752, https://usegalaxy.org.au/published/workflow?id=970dc0d8bd74707f, https://usegalaxy.org.au/published/workflow?id=9804030358c3f844, https://usegalaxy.org.au/published/workflow?id=98d3def7da131b4c, https://usegalaxy.org.au/published/workflow?id=98f660971010d11d, https://usegalaxy.org.au/published/workflow?id=9e0bae0505ee7c07, https://usegalaxy.org.au/published/workflow?id=a02c102e179596cb, https://usegalaxy.org.au/published/workflow?id=a1015b83baa0eb6d, https://usegalaxy.org.au/published/workflow?id=a209f43d8a3cc853, https://usegalaxy.org.au/published/workflow?id=a345bf84eb9bf6b2, https://usegalaxy.org.au/published/workflow?id=a40dadc943a0db1e, https://usegalaxy.org.au/published/workflow?id=a525966251895f8a, https://usegalaxy.org.au/published/workflow?id=a740543c4e9808f1, https://usegalaxy.org.au/published/workflow?id=ac8bdc5a0b2d70cd, https://usegalaxy.org.au/published/workflow?id=ad10cd704268dcf0, https://usegalaxy.org.au/published/workflow?id=ad556db35858e7ec, https://usegalaxy.org.au/published/workflow?id=afab89bc468da8b3, https://usegalaxy.org.au/published/workflow?id=b19fa49405117317, https://usegalaxy.org.au/published/workflow?id=b1f45d2f8d8343e9, https://usegalaxy.org.au/published/workflow?id=b310af289ada88bc, https://usegalaxy.org.au/published/workflow?id=b5131e3a0ef2676e, https://usegalaxy.org.au/published/workflow?id=b9faf6a3f48fc11b, https://usegalaxy.org.au/published/workflow?id=be96bb47b6d6435a, https://usegalaxy.org.au/published/workflow?id=c43744d7dd9c0c0b, https://usegalaxy.org.au/published/workflow?id=c556c2270200801d, https://usegalaxy.org.au/published/workflow?id=c6a71573a299eb3a, https://usegalaxy.org.au/published/workflow?id=c7e1f666a3366823, https://usegalaxy.org.au/published/workflow?id=c7f2d6490128dc3c, https://usegalaxy.org.au/published/workflow?id=c853aa707fcddf4e, https://usegalaxy.org.au/published/workflow?id=cbf022e349d726d9, https://usegalaxy.org.au/published/workflow?id=d06665e4bc7fff6d, https://usegalaxy.org.au/published/workflow?id=d298edda13b382e7, https://usegalaxy.org.au/published/workflow?id=d42d7b1aa47e870f, https://usegalaxy.org.au/published/workflow?id=d678e57c7f1baf5d, https://usegalaxy.org.au/published/workflow?id=d78879cfaa5948b3, https://usegalaxy.org.au/published/workflow?id=d7f3fc810261e419, https://usegalaxy.org.au/published/workflow?id=d83d5ecf31c20490, https://usegalaxy.org.au/published/workflow?id=d957bfeb3cd03db0, https://usegalaxy.org.au/published/workflow?id=d9c181fd954b629a, https://usegalaxy.org.au/published/workflow?id=df11f09771e73e04, https://usegalaxy.org.au/published/workflow?id=df4e186069552649, https://usegalaxy.org.au/published/workflow?id=e031af90daa2cadf, https://usegalaxy.org.au/published/workflow?id=e28ca83f14f4ede5, https://usegalaxy.org.au/published/workflow?id=e5f1ce106ea7af12, https://usegalaxy.org.au/published/workflow?id=ea687c5613fcce27, https://usegalaxy.org.au/published/workflow?id=ecfb8a08e5a40e15, https://usegalaxy.org.au/published/workflow?id=f06370394b061e08, https://usegalaxy.org.au/published/workflow?id=f7a866159f1eaecf, https://usegalaxy.org.au/published/workflow?id=f895914e322c8b77, https://usegalaxy.org.au/published/workflow?id=fa37223950a91f34, https://usegalaxy.org.au/published/workflow?id=fe1e7fa0bc437ed8, https://usegalaxy.org.au/published/workflow?id=fea0e6c02c2f6274, https://usegalaxy.org.au/published/workflow?id=fff2e49bc90e38ca, https://usegalaxy.org/published/workflow?id=07e873a6d75dfe9f, https://usegalaxy.org/published/workflow?id=0993daadf2db5f51, https://usegalaxy.org/published/workflow?id=10d5e195501603bc, https://usegalaxy.org/published/workflow?id=1245e9435cbc3985, https://usegalaxy.org/published/workflow?id=137d0f05a71d7acb, https://usegalaxy.org/published/workflow?id=1b434bfc3aac0022, https://usegalaxy.org/published/workflow?id=1d24cc949c52e475, https://usegalaxy.org/published/workflow?id=1eec2586d1ff908b, https://usegalaxy.org/published/workflow?id=237ca9f36ca266e7, https://usegalaxy.org/published/workflow?id=242ef2e9dee35702, https://usegalaxy.org/published/workflow?id=25a9f0693ff6d327, https://usegalaxy.org/published/workflow?id=283bc048cc7a80f5, https://usegalaxy.org/published/workflow?id=284150bb1f296378, https://usegalaxy.org/published/workflow?id=2acbb8c5f72b9f1c, https://usegalaxy.org/published/workflow?id=2b41752b8518a87b, https://usegalaxy.org/published/workflow?id=2c5c627c08563398, https://usegalaxy.org/published/workflow?id=2d994cc02ec05ede, https://usegalaxy.org/published/workflow?id=2f1469dab87a3649, https://usegalaxy.org/published/workflow?id=305fb9f84b053713, https://usegalaxy.org/published/workflow?id=31387ed5ebc88cd2, https://usegalaxy.org/published/workflow?id=3179c157fa5a2fdd, https://usegalaxy.org/published/workflow?id=31a6145b0832fa48, https://usegalaxy.org/published/workflow?id=3753dcca4100e976, https://usegalaxy.org/published/workflow?id=3ba73f679558017c, https://usegalaxy.org/published/workflow?id=3db4eb1ecb588d44, https://usegalaxy.org/published/workflow?id=3f7550d5d2bf0b96, https://usegalaxy.org/published/workflow?id=4029a0c8fa23d375, https://usegalaxy.org/published/workflow?id=408c799822cbf6ea, https://usegalaxy.org/published/workflow?id=426358bd988cb39d, https://usegalaxy.org/published/workflow?id=435f0340bf00c940, https://usegalaxy.org/published/workflow?id=4403ca7fe447a5d7, https://usegalaxy.org/published/workflow?id=44781c8b0bd667ed, https://usegalaxy.org/published/workflow?id=45c0b677c4920d03, https://usegalaxy.org/published/workflow?id=484f8fcf21d67376, https://usegalaxy.org/published/workflow?id=498a8c6329ca5728, https://usegalaxy.org/published/workflow?id=49c9e8161a46667f, https://usegalaxy.org/published/workflow?id=4a0ad87aa5aa558a, https://usegalaxy.org/published/workflow?id=4bf38f5b45faa8a1, https://usegalaxy.org/published/workflow?id=4bf73376661baafd, https://usegalaxy.org/published/workflow?id=4d5cc5010537f172, https://usegalaxy.org/published/workflow?id=578e634b63923e9b, https://usegalaxy.org/published/workflow?id=57e6345fda3e2682, https://usegalaxy.org/published/workflow?id=597c4d54f8a8ebe7, https://usegalaxy.org/published/workflow?id=5dfec2a3ae5fa765, https://usegalaxy.org/published/workflow?id=5edabfc62f58d8cf, https://usegalaxy.org/published/workflow?id=653d553ac2fa51c8, https://usegalaxy.org/published/workflow?id=65ccf48b7872e36c, https://usegalaxy.org/published/workflow?id=67252039bce060b2, https://usegalaxy.org/published/workflow?id=683e725f672b7d9b, https://usegalaxy.org/published/workflow?id=6a27c0d224683f73, https://usegalaxy.org/published/workflow?id=6b511b47176893e3, https://usegalaxy.org/published/workflow?id=6fc7497c86ebd8ee, https://usegalaxy.org/published/workflow?id=7072731b51e16923, https://usegalaxy.org/published/workflow?id=73c20d784b7ef81a, https://usegalaxy.org/published/workflow?id=771345c66d5b9700, https://usegalaxy.org/published/workflow?id=7adf904dea83e699, https://usegalaxy.org/published/workflow?id=7dba74c915f63dc1, https://usegalaxy.org/published/workflow?id=7f6900852a81d121, https://usegalaxy.org/published/workflow?id=81f1f0fd6d9235fd, https://usegalaxy.org/published/workflow?id=84161f4f8bf77474, https://usegalaxy.org/published/workflow?id=864da6858d34df08, https://usegalaxy.org/published/workflow?id=86c67855acbf6124, https://usegalaxy.org/published/workflow?id=8ccb42012dcb5db2, https://usegalaxy.org/published/workflow?id=8cfbaa14cff5a633, https://usegalaxy.org/published/workflow?id=8d605fbd33003d26, https://usegalaxy.org/published/workflow?id=8f7c9461910b08f1, https://usegalaxy.org/published/workflow?id=90f5b512997c70f8, https://usegalaxy.org/published/workflow?id=932b3bc10186f73e, https://usegalaxy.org/published/workflow?id=936ed7447c8e0a2d, https://usegalaxy.org/published/workflow?id=93c1fef4472aa7a8, https://usegalaxy.org/published/workflow?id=950c7245a6c3f6e9, https://usegalaxy.org/published/workflow?id=961dac8100c6c70c, https://usegalaxy.org/published/workflow?id=96bd0007dceec123, https://usegalaxy.org/published/workflow?id=9938a81fb6911412, https://usegalaxy.org/published/workflow?id=9ab68c65337b893b, https://usegalaxy.org/published/workflow?id=9ddfdc9d6cacea88, https://usegalaxy.org/published/workflow?id=a8e0f77cd6ef49f9, https://usegalaxy.org/published/workflow?id=ab0be4897d5355ac, https://usegalaxy.org/published/workflow?id=acada5044d64a30a, https://usegalaxy.org/published/workflow?id=b11c6268704dcc1b, https://usegalaxy.org/published/workflow?id=b12f99e2233fcca6, https://usegalaxy.org/published/workflow?id=b5644a5dcf237ef9, https://usegalaxy.org/published/workflow?id=b661fb850c4b4b01, https://usegalaxy.org/published/workflow?id=b6d535f210efa18e, https://usegalaxy.org/published/workflow?id=bb1cb6abfe84a646, https://usegalaxy.org/published/workflow?id=bbc894a79eb0e769, https://usegalaxy.org/published/workflow?id=bd6bfd338ffb9637, https://usegalaxy.org/published/workflow?id=bfd2e7e63d3e6856, https://usegalaxy.org/published/workflow?id=bfef75672b9c3ea4, https://usegalaxy.org/published/workflow?id=c09b8a0c96123cbb, https://usegalaxy.org/published/workflow?id=c0a9a2d53b8660de, https://usegalaxy.org/published/workflow?id=c694856c01ce7cc5, https://usegalaxy.org/published/workflow?id=c983ea8a32662cd6, https://usegalaxy.org/published/workflow?id=cb4ccd3068819418, https://usegalaxy.org/published/workflow?id=cc50da71cdf6f244, https://usegalaxy.org/published/workflow?id=ce04658f05029be2, https://usegalaxy.org/published/workflow?id=cfd5dc3c7bc1e832, https://usegalaxy.org/published/workflow?id=d08135fa13d1745e, https://usegalaxy.org/published/workflow?id=d2e8a6b39fe88f4c, https://usegalaxy.org/published/workflow?id=d36765ac28c11e1a, https://usegalaxy.org/published/workflow?id=d3f35215e6f45d79, https://usegalaxy.org/published/workflow?id=d4b2b85db47c405e, https://usegalaxy.org/published/workflow?id=d4f0a7ccb09f1e55, https://usegalaxy.org/published/workflow?id=daaded719dcfd53b, https://usegalaxy.org/published/workflow?id=dae5d5e5df5c4dd7, https://usegalaxy.org/published/workflow?id=e12e642b88a0fe08, https://usegalaxy.org/published/workflow?id=e4dd39237df58884, https://usegalaxy.org/published/workflow?id=eaf96fa3c39c210b, https://usegalaxy.org/published/workflow?id=f1a54b10d4afce8f, https://usegalaxy.org/published/workflow?id=f60027545d40b82d, https://usegalaxy.org/published/workflow?id=f93a4f6b5f3aaaed, https://usegalaxy.org/published/workflow?id=fa26fd51caea53c7, https://usegalaxy.org/published/workflow?id=fc7b14aafa0b421e, https://usegalaxy.org/published/workflow?id=fd155fcba9bed7ff, https://usegalaxy.org/published/workflow?id=fd40e50c9f1545ce, https://workflowhub.eu/workflows/1103?version=3, https://workflowhub.eu/workflows/1104?version=2, https://workflowhub.eu/workflows/1162?version=2, https://workflowhub.eu/workflows/1163?version=2, https://workflowhub.eu/workflows/1164?version=2, https://workflowhub.eu/workflows/1607?version=1, https://workflowhub.eu/workflows/1641?version=1, https://workflowhub.eu/workflows/1646?version=1, https://workflowhub.eu/workflows/403?version=9, https://workflowhub.eu/workflows/605?version=1, https://workflowhub.eu/workflows/606?version=2, https://workflowhub.eu/workflows/608?version=1, https://workflowhub.eu/workflows/612?version=25, https://workflowhub.eu/workflows/633?version=23, https://workflowhub.eu/workflows/641?version=32, https://workflowhub.eu/workflows/642?version=25, https://workflowhub.eu/workflows/701?version=1, https://workflowhub.eu/workflows/702?version=1, https://workflowhub.eu/workflows/751?version=18, https://workflowhub.eu/workflows/788?version=1, https://workflowhub.eu/workflows/789?version=1, https://workflowhub.eu/workflows/794?version=2</td>
      <td>assembly/ERGA-post-assembly-QC, assembly/assembly-quality-control, assembly/vgp_genome_assembly</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>merquryfk</td>
      <td>merquryfk_asmplot, merquryfk_cnplot, merquryfk_hapmaker, merquryfk_happlot, merquryfk_merquryfk</td>
      <td>FastK based version of Merqury</td>
      <td>2025-02-11</td>
      <td>https://github.com/thegenemyers/MERQURY.FK</td>
      <td>1.1.1</td>
      <td>merquryfk</td>
      <td>1.2</td>
      <td>To update</td>
      <td>Sequence Analysis, Assembly</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/merquryfk</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/merquryfk</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>metabuli</td>
      <td>metabuli_classify</td>
      <td>Classifying metagenomes by jointly analysing both DNA and amino acid (AA) sequences</td>
      <td>2024-06-04</td>
      <td>https://github.com/steineggerlab/Metabuli</td>
      <td>1.0.5</td>
      <td>metabuli</td>
      <td>1.1.0</td>
      <td>To update</td>
      <td>Sequence Analysis, Metagenomics</td>
      <td>Taxonomic classification</td>
      <td>Taxonomic classification</td>
      <td>Taxonomy</td>
      <td>Taxonomy</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/metabuli</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/metabuli</td>
      <td>metabuli</td>
      <td>metabuli</td>
      <td>Metabuli: specific and sensitive metagenomic classification via joint analysis of DNA and amino acid</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>metagenomeseq_normalization</td>
      <td>metagenomeseq_normalizaton</td>
      <td>metagenomeSeq Normalization</td>
      <td>2017-03-27</td>
      <td>https://bioconductor.org/packages/3.18/bioc/html/metagenomeSeq.html</td>
      <td>1.16.0-0.0.1</td>
      <td>bioconductor-metagenomeseq</td>
      <td>1.43.0</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td>Sequence visualisation, Statistical calculation</td>
      <td>Sequence visualisation, Statistical calculation</td>
      <td>Metagenomics, Sequencing</td>
      <td>Metagenomics, Sequencing</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/metagenomeseq</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/metagenomeseq</td>
      <td>metagenomeseq</td>
      <td>metagenomeSeq</td>
      <td>Designed to determine features (be it Operational Taxanomic Unit (OTU), species, etc.) that are differentially abundant between two or more groups of multiple samples. It is designed to address the effects of both normalization and under-sampling of microbial communities on disease association detection and the testing of feature correlations.</td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>15</td>
      <td>23</td>
      <td>521</td>
      <td>763</td>
      <td>28</td>
      <td>56</td>
      <td>261</td>
      <td>530</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1293</td>
      <td>782</td>
      <td>79</td>
      <td>43</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>metawrapmg_binning</td>
      <td>metawrapmg_binning, metawrapmg_bin_refinement</td>
      <td>A flexible pipeline for genome-resolved metagenomic data analysis</td>
      <td>2024-04-11</td>
      <td>https://github.com/bxlab/metaWRAP</td>
      <td>1.3.0</td>
      <td>metawrap-mg</td>
      <td>1.3.0</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Read binning, Sequence assembly, Genome annotation, Sequence trimming, Demultiplexing</td>
      <td>Read binning, Sequence assembly, Genome annotation, Sequence trimming, Demultiplexing</td>
      <td>Whole genome sequencing, Metagenomic sequencing, Metagenomics</td>
      <td>Whole genome sequencing, Metagenomic sequencing, Metagenomics</td>
      <td>galaxy-australia</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/metawrapmg</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/metawrapmg</td>
      <td>metawrap</td>
      <td>MetaWRAP</td>
      <td>MetaWRAP aims to be an easy-to-use metagenomic wrapper suite that accomplishes the core tasks of metagenomic analysis from start to finish: read quality control, assembly, visualization, taxonomic profiling, extracting draft genomes (binning), and functional annotation.</td>
      <td></td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>100</td>
      <td>100</td>
      <td>1596</td>
      <td>1596</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>67</td>
      <td>67</td>
      <td>587</td>
      <td>587</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2183</td>
      <td>2183</td>
      <td>167</td>
      <td>167</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>mg_toolkit</td>
      <td>mg_toolkit_bulk_download, mg_toolkit_original_metadata</td>
      <td>Metagenomics toolkit enables scientists to download all of the sample metadata for a given study or sequence to a single csv file.</td>
      <td>2024-07-12</td>
      <td>https://github.com/EBI-Metagenomics/emg-toolkit</td>
      <td>0.10.4</td>
      <td>mg-toolkit</td>
      <td>0.10.4</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Data retrieval</td>
      <td>Data retrieval</td>
      <td>Metagenomics</td>
      <td>Metagenomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mg_toolkit</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mg_toolkit</td>
      <td>mg-toolkit</td>
      <td>mg-toolkit</td>
      <td>Metagenomics toolkit enables scientists to download all of the sample metadata for a given study or sequence to a single csv file.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>10</td>
      <td>21</td>
      <td>21</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>21</td>
      <td>21</td>
      <td>10</td>
      <td>10</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>miniasm</td>
      <td>miniasm</td>
      <td>Miniasm - Ultrafast de novo assembly for long noisy reads (though having no consensus step)</td>
      <td>2019-06-18</td>
      <td>https://github.com/lh3/miniasm</td>
      <td>0.3_r179</td>
      <td>miniasm</td>
      <td>0.3</td>
      <td>To update</td>
      <td>Assembly</td>
      <td>De-novo assembly</td>
      <td>De-novo assembly</td>
      <td>Genomics, Sequence assembly</td>
      <td>Genomics, Sequence assembly</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/miniasm</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/miniasm</td>
      <td>miniasm</td>
      <td>miniasm</td>
      <td>Miniasm is a very fast OLC-based de novo assembler for noisy long reads. It takes all-vs-all read self-mappings (typically by minimap) as input and outputs an assembly graph in the GFA format.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>571</td>
      <td>687</td>
      <td>11506</td>
      <td>14702</td>
      <td>474</td>
      <td>538</td>
      <td>9124</td>
      <td>10016</td>
      <td>88</td>
      <td>96</td>
      <td>1235</td>
      <td>1277</td>
      <td>16</td>
      <td>16</td>
      <td>166</td>
      <td>166</td>
      <td>26161</td>
      <td>22031</td>
      <td>1337</td>
      <td>1149</td>
      <td>https://usegalaxy.eu/published/workflow?id=249c115413f17f28, https://usegalaxy.eu/published/workflow?id=32d2f5340098bcbd, https://usegalaxy.eu/published/workflow?id=39837ca09374598b, https://usegalaxy.eu/published/workflow?id=44e53ea319e07daa, https://usegalaxy.eu/published/workflow?id=b4de76c0a4d0e23a, https://usegalaxy.eu/published/workflow?id=c48807045e258dd8, https://usegalaxy.eu/published/workflow?id=dc7107896a3f2964, https://usegalaxy.org.au/published/workflow?id=d9571b3dfef34e5e, https://workflowhub.eu/workflows/1477?version=1, https://workflowhub.eu/workflows/406?version=1</td>
      <td>microbiome/plasmid-metagenomics-nanopore</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>mitobim</td>
      <td>mitobim</td>
      <td>assemble mitochondrial genomes</td>
      <td>2020-12-29</td>
      <td>https://github.com/chrishah/MITObim</td>
      <td>1.9.1</td>
      <td>mitobim</td>
      <td>1.9.1</td>
      <td>Up-to-date</td>
      <td>Assembly</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mitobim</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mitobim</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>168</td>
      <td>168</td>
      <td>1463</td>
      <td>1463</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>39</td>
      <td>39</td>
      <td>533</td>
      <td>533</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1996</td>
      <td>1996</td>
      <td>207</td>
      <td>207</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>mmseqs2</td>
      <td>mmseqs2_easy_linclust_clustering, mmseqs2_taxonomy_assignment</td>
      <td>MMseqs2 is an ultra fast and sensitive sequence search and clustering suite</td>
      <td>2025-03-27</td>
      <td>https://github.com/soedinglab/MMseqs2</td>
      <td>17-b804f</td>
      <td>mmseqs2</td>
      <td>17.b804f</td>
      <td>To update</td>
      <td>Sequence Analysis, Metagenomics</td>
      <td>Sequence similarity search, Sequence alignment, Sequence clustering, Taxonomic classification</td>
      <td>Sequence similarity search, Sequence alignment, Sequence clustering, Taxonomic classification</td>
      <td>Metagenomics, Sequence analysis, Proteins, Nucleic acids, Gene and protein families, Taxonomy</td>
      <td>Metagenomics, Sequence analysis, Nucleic acids, Gene and protein families, Taxonomy</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mmseqs2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mmseqs2</td>
      <td>MMseqs2</td>
      <td>MMseqs2</td>
      <td>MMseqs2 (Many-against-Many sequence searching) is a software suite to search and cluster huge protein and nucleotide sequence sets. MMseqs2 is open source software implemented in C++ for Linux, MacOS, and (as beta version, via cygwin) Windows. The software is designed to run on multiple cores and servers and exhibits very good scalability. MMseqs2 can run 10000 times faster than BLAST. At 100 times its speed it achieves almost the same sensitivity. It can perform profile searches with the same sensitivity as PSI-BLAST at over 400 times its speed.MMseqs2 includes Linclust, the first clustering algorithm whose runtime scales linearly With Linclust we clustered 1.6 billion metagenomic sequence fragments in 10h on a single server to 50% sequence identity.</td>
      <td></td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>42</td>
      <td>42</td>
      <td>260</td>
      <td>260</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>29</td>
      <td>29</td>
      <td>289</td>
      <td>289</td>
      <td>45</td>
      <td>45</td>
      <td>https://usegalaxy.eu/published/workflow?id=62392daea25a3980, https://workflowhub.eu/workflows/2024?version=2</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>ncbi_egapx</td>
      <td>ncbi_egapx</td>
      <td>Eukaryotic Genome Annotation Pipeline - External (EGAPx)</td>
      <td>2024-08-19</td>
      <td>https://github.com/ncbi/egapx</td>
      <td>0.4.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>Genome annotation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>richard-burhans</td>
      <td>https://github.com/richard-burhans/galaxytools/tree/main/tools/ncbi_egapx</td>
      <td>https://github.com/richard-burhans/galaxytools/tree/main/tools/ncbi_egapx</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>7</td>
      <td>7</td>
      <td>49</td>
      <td>49</td>
      <td>1037</td>
      <td>1037</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1044</td>
      <td>1044</td>
      <td>51</td>
      <td>51</td>
      <td>https://usegalaxy.org/published/workflow?id=7251c6e0dad8f927, https://usegalaxy.org/published/workflow?id=951575e9a352301c, https://usegalaxy.org/published/workflow?id=ce488732866ba48f, https://usegalaxy.org/published/workflow?id=e93a0522ec3a35e6</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>necat</td>
      <td>necat</td>
      <td>Error correction and de-novo assembly for ONT Nanopore reads</td>
      <td>2021-11-29</td>
      <td>https://github.com/xiaochuanle/NECAT</td>
      <td>0.0.1_update20200803</td>
      <td>necat</td>
      <td>0.0.1_update20200803</td>
      <td>Up-to-date</td>
      <td>Assembly</td>
      <td>De-novo assembly</td>
      <td>De-novo assembly</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/necat</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/necat</td>
      <td>necat</td>
      <td>NECAT</td>
      <td>NECAT is an error correction and de-novo assembly tool for Nanopore long noisy reads.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>228</td>
      <td>228</td>
      <td>1106</td>
      <td>1106</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1106</td>
      <td>1106</td>
      <td>228</td>
      <td>228</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>newick_utils</td>
      <td>newick_display</td>
      <td>Perform operations on Newick trees</td>
      <td>2018-10-01</td>
      <td>https://github.com/tjunier/newick_utils</td>
      <td>1.6+galaxy1</td>
      <td>newick_utils</td>
      <td>1.6</td>
      <td>To update</td>
      <td>Visualization, Metagenomics</td>
      <td>Phylogenetic tree generation, Phylogenetic tree analysis, Phylogenetic tree reconstruction</td>
      <td></td>
      <td>Phylogeny, Genomics, Computer science</td>
      <td>Phylogeny, Genomics, Computer science</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/newick_utils</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/newick_utils</td>
      <td>newick_utilities</td>
      <td>Newick Utilities</td>
      <td>The Newick Utilities are a set of command-line tools for processing phylogenetic trees. They can process arbitrarily large amounts of data and do not require user interaction, which makes them suitable for automating phylogeny processing tasks.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1756</td>
      <td>1893</td>
      <td>51682</td>
      <td>53373</td>
      <td>2196</td>
      <td>2332</td>
      <td>18983</td>
      <td>20195</td>
      <td>1114</td>
      <td>1145</td>
      <td>9635</td>
      <td>9912</td>
      <td>48</td>
      <td>49</td>
      <td>310</td>
      <td>312</td>
      <td>83792</td>
      <td>80610</td>
      <td>5419</td>
      <td>5114</td>
      <td>https://usegalaxy.eu/published/workflow?id=36de05d50c703164, https://usegalaxy.eu/published/workflow?id=376119528377a3ae, https://usegalaxy.eu/published/workflow?id=974cc681d07ec788, https://usegalaxy.eu/published/workflow?id=b1a799497c366eef, https://usegalaxy.eu/published/workflow?id=c1e26abe61c69688, https://usegalaxy.eu/published/workflow?id=c67d2e75eeb4fd19, https://usegalaxy.eu/published/workflow?id=c8bc4af6dd3b5d02, https://usegalaxy.eu/published/workflow?id=dadca1020f586338, https://usegalaxy.eu/published/workflow?id=dc57118a95b10a80, https://usegalaxy.eu/published/workflow?id=e16e524c557d7928, https://usegalaxy.eu/published/workflow?id=ec50dd6d56f50db1, https://usegalaxy.eu/published/workflow?id=f21d6a40a10a8ae8, https://usegalaxy.eu/published/workflow?id=f6a763951d815944, https://usegalaxy.org.au/published/workflow?id=2717e90438c7fb8c, https://usegalaxy.org.au/published/workflow?id=3fd2195b6928d5e0, https://usegalaxy.org.au/published/workflow?id=a01c3004bcef9dd3, https://usegalaxy.org.au/published/workflow?id=a52609119b30e64d, https://usegalaxy.org.au/published/workflow?id=b60922a253df6654, https://usegalaxy.org.au/published/workflow?id=eda40b58616a0fe4, https://usegalaxy.org/published/workflow?id=005924662f68a480, https://usegalaxy.org/published/workflow?id=0596162a499b4074, https://usegalaxy.org/published/workflow?id=2d3063882d8239ff, https://usegalaxy.org/published/workflow?id=3ef08553982f6ffc, https://usegalaxy.org/published/workflow?id=603e5c61c57bcf3f, https://usegalaxy.org/published/workflow?id=6a1eb7e7bececba1, https://usegalaxy.org/published/workflow?id=6f090b2c259847d2, https://usegalaxy.org/published/workflow?id=775335812a6b7a61, https://usegalaxy.org/published/workflow?id=80565301ada0d3eb, https://usegalaxy.org/published/workflow?id=9c4038fa15af6995, https://usegalaxy.org/published/workflow?id=bc01b8b203ecc754, https://usegalaxy.org/published/workflow?id=d653eaefcf6b4697, https://workflowhub.eu/workflows/1060?version=1, https://workflowhub.eu/workflows/1418?version=1, https://workflowhub.eu/workflows/1465?version=1, https://workflowhub.eu/workflows/1487?version=2, https://workflowhub.eu/workflows/653?version=1</td>
      <td>evolution/abc_intro_phylo, microbiome/mothur-miseq-sop, microbiome/mothur-miseq-sop-short, microbiome/pathogen-detection-from-nanopore-foodborne-data</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>nonpareil</td>
      <td>nonpareil</td>
      <td>Estimate average coverage in metagenomic datasets</td>
      <td>2017-11-03</td>
      <td>http://nonpareil.readthedocs.io</td>
      <td>3.5.5</td>
      <td>nonpareil</td>
      <td>3.5.5</td>
      <td>Up-to-date</td>
      <td>Metagenomics</td>
      <td>Operation</td>
      <td>Operation</td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/nonpareil</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/nonpareil</td>
      <td>nonpareil</td>
      <td>nonpareil</td>
      <td>Estimate metagenomic coverage and sequence diversity</td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>16</td>
      <td>19</td>
      <td>105</td>
      <td>195</td>
      <td>23</td>
      <td>26</td>
      <td>122</td>
      <td>168</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>363</td>
      <td>227</td>
      <td>45</td>
      <td>39</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>obisindicators</td>
      <td>obisindicators, obis_data</td>
      <td>Compute biodiveristy indicators for marine data from obis</td>
      <td>2022-11-04</td>
      <td>https://github.com/Marie59/obisindicators</td>
      <td>0.0.2</td>
      <td>r-base</td>
      <td></td>
      <td>To update</td>
      <td>Ecology</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/obisindicators</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/obisindicators</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>2</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>29</td>
      <td>29</td>
      <td>399</td>
      <td>399</td>
      <td>15</td>
      <td>15</td>
      <td>25</td>
      <td>25</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>424</td>
      <td>424</td>
      <td>44</td>
      <td>44</td>
      <td>https://usegalaxy.eu/published/workflow?id=4f51080ffe7d57e0, https://usegalaxy.eu/published/workflow?id=9e05bd65faf58ab3, https://workflowhub.eu/workflows/1698?version=1, https://workflowhub.eu/workflows/662?version=1, https://workflowhub.eu/workflows/758?version=1</td>
      <td>climate/earth_system, ecology/obisindicators</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>ocean</td>
      <td>argo_getdata, copernicusmarine, divand_full_analysis</td>
      <td>Access, process, and visualise oceanographic data for the Earth System</td>
      <td>2023-11-17</td>
      <td>https://github.com/Marie59/FE-ft-ESG/tree/main/ocean</td>
      <td>0.1.15</td>
      <td>copernicusmarine</td>
      <td></td>
      <td>To update</td>
      <td>Ecology</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/ocean</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/ocean</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>14</td>
      <td>14</td>
      <td>189</td>
      <td>189</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>189</td>
      <td>189</td>
      <td>14</td>
      <td>14</td>
      <td>https://usegalaxy.eu/published/workflow?id=26fe6b0fb6ca2892, https://usegalaxy.eu/published/workflow?id=a80f9b926ba43892, https://usegalaxy.eu/published/workflow?id=fc33ea1679e5f1b7, https://workflowhub.eu/workflows/1436?version=1, https://workflowhub.eu/workflows/1445?version=1</td>
      <td>climate/argo_pangeo, climate/earth_system, climate/ocean-variables</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>ogcProcess_otb_bandmath</td>
      <td>otb_band_math</td>
      <td>Outputs a monoband image which is the result of a mathematical operation on several multi-band images.</td>
      <td>2024-03-12</td>
      <td>https://github.com/AquaINFRA/galaxy</td>
      <td>1.0</td>
      <td>r-base</td>
      <td></td>
      <td>To update</td>
      <td>Ecology</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/OtbBandMath</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/ogcProcess_otb_bandmath</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>100</td>
      <td>100</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>100</td>
      <td>100</td>
      <td>8</td>
      <td>8</td>
      <td>https://usegalaxy.eu/published/workflow?id=b1fa4a3561988704, https://usegalaxy.eu/published/workflow?id=e8244d1c80ea0470</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>ogcProcess_otb_meanShiftSmoothing</td>
      <td>otb_mean_shift_smoothing</td>
      <td>This application smooths an image using the MeanShift algorithm.</td>
      <td>2024-03-12</td>
      <td>https://github.com/AquaINFRA/galaxy</td>
      <td>1.0</td>
      <td>r-base</td>
      <td></td>
      <td>To update</td>
      <td>Ecology</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/interpolation</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/ogcProcess_otb_meanShiftSmoothing</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>57</td>
      <td>57</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>57</td>
      <td>57</td>
      <td>4</td>
      <td>4</td>
      <td>https://usegalaxy.eu/published/workflow?id=e8244d1c80ea0470</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>pampa</td>
      <td>pampa_communitymetrics, pampa_presabs, pampa_glmcomm, pampa_glmsp, pampa_plotglm</td>
      <td>Tools to compute and analyse biodiversity metrics</td>
      <td>2020-06-06</td>
      <td></td>
      <td>0.0.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>Ecology</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/ColineRoyaux/PAMPA-Galaxy</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/PAMPA</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>4</td>
      <td>0</td>
      <td>50</td>
      <td>53</td>
      <td>1997</td>
      <td>2045</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>26</td>
      <td>26</td>
      <td>1141</td>
      <td>1141</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>3187</td>
      <td>3139</td>
      <td>80</td>
      <td>77</td>
      <td>https://usegalaxy.eu/published/workflow?id=428be4c7d1fd828f, https://usegalaxy.eu/published/workflow?id=7c45c90fffe40aca, https://usegalaxy.eu/published/workflow?id=d33ab07e4cf44f8c, https://workflowhub.eu/workflows/1696?version=1</td>
      <td>ecology/PAMPA-toolsuite-tutorial</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>pharokka</td>
      <td>pharokka</td>
      <td>rapid standardised annotation tool for bacteriophage genomes and metagenomes</td>
      <td>2023-02-14</td>
      <td>https://github.com/gbouras13/pharokka</td>
      <td>1.3.2</td>
      <td>\n                pharokka\n</td>
      <td></td>
      <td>To update</td>
      <td>Genome annotation</td>
      <td>Genome annotation, Antimicrobial resistance prediction, tRNA gene prediction, Formatting, Sequence assembly</td>
      <td>Genome annotation, Antimicrobial resistance prediction, tRNA gene prediction, Formatting, Sequence assembly</td>
      <td>Metagenomics, Sequence sites, features and motifs, Workflows, Functional, regulatory and non-coding RNA</td>
      <td>Metagenomics, Sequence sites, features and motifs, Workflows, Functional, regulatory and non-coding RNA</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pharokka</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pharokka</td>
      <td>pharokka</td>
      <td>Pharokka</td>
      <td>Pharokka is a rapid standardised annotation tool for bacteriophage genomes and metagenomes.</td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>766</td>
      <td>766</td>
      <td>16933</td>
      <td>16933</td>
      <td>1</td>
      <td>1</td>
      <td>21</td>
      <td>21</td>
      <td>117</td>
      <td>117</td>
      <td>1202</td>
      <td>1202</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>18156</td>
      <td>18156</td>
      <td>884</td>
      <td>884</td>
      <td>https://usegalaxy.eu/published/workflow?id=62392daea25a3980</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>phi_toolkit_report</td>
      <td>phi_toolkit_report</td>
      <td>Phage host interaction toolkit report generator</td>
      <td>2025-06-04</td>
      <td>https://git.ufz.de/borimcor/phage-host-analysis</td>
      <td>0.2.0</td>
      <td>bioconductor-biostrings</td>
      <td>2.74.0</td>
      <td>To update</td>
      <td>Metagenomics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ufz</td>
      <td>https://github.com/Helmholtz-UFZ/ufz-galaxy-tools/blob/main/tools/phi-toolkit</td>
      <td>https://github.com/Helmholtz-UFZ/galaxy-tools/tree/main/tools/phi-toolkit</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>https://usegalaxy.eu/published/workflow?id=c62d65832377e376</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>plasmidspades</td>
      <td>plasmidspades</td>
      <td>Genome assembler for assemblying plasmid</td>
      <td>2016-06-06</td>
      <td></td>
      <td>1.1</td>
      <td>spades</td>
      <td>4.2.0</td>
      <td>To update</td>
      <td>Assembly</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>nml</td>
      <td></td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/plasmidspades</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>prinseq</td>
      <td>prinseq</td>
      <td>PRINSEQ is a tool for easy and rapid quality control and data processing of metagenomic and metatranscriptomic datasets</td>
      <td>2016-05-30</td>
      <td>http://prinseq.sourceforge.net/manual.html</td>
      <td>@TOOL_VERSION+galaxy2</td>
      <td>prinseq</td>
      <td>0.20.4</td>
      <td>To update</td>
      <td>Fastq Manipulation, Metagenomics</td>
      <td>Read pre-processing, Sequence trimming, Sequence contamination filtering</td>
      <td>Read pre-processing, Sequence trimming, Sequence contamination filtering</td>
      <td>Transcriptomics, Metagenomics, Genomics</td>
      <td>Transcriptomics, Metagenomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/prinseq</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/prinseq</td>
      <td>prinseq</td>
      <td>PRINSEQ</td>
      <td>PRINSEQ is a sequence processing tool that can be used to filter, reformat and trim genomic and metagenomic sequence data. It generates summary statistics of the input in graphical and tabular formats that can be used for quality control steps. PRINSEQ is available as both standalone and web-based versions.</td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>345</td>
      <td>412</td>
      <td>15643</td>
      <td>16651</td>
      <td>1317</td>
      <td>1501</td>
      <td>10523</td>
      <td>12474</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>11</td>
      <td>11</td>
      <td>17</td>
      <td>17</td>
      <td>29142</td>
      <td>26183</td>
      <td>1924</td>
      <td>1673</td>
      <td>https://usegalaxy.eu/published/workflow?id=0609a4b1941d1a68, https://usegalaxy.eu/published/workflow?id=76e9cd837ea5e7da, https://usegalaxy.eu/published/workflow?id=8f415b0c409f56f7, https://usegalaxy.eu/published/workflow?id=9fddc44294236e8f, https://usegalaxy.eu/published/workflow?id=d6ff6be33b8bfe85, https://usegalaxy.eu/published/workflow?id=da19c0eaa757fa39, https://usegalaxy.org/published/workflow?id=20b7ae4b517a80fd, https://usegalaxy.org/published/workflow?id=46f184a0e95f3c1c, https://usegalaxy.org/published/workflow?id=5f878c4cc3bff68c, https://usegalaxy.org/published/workflow?id=a71fcbc60caae8b6, https://usegalaxy.org/published/workflow?id=afb67b4b9952f57d, https://workflowhub.eu/workflows/1271?version=3, https://workflowhub.eu/workflows/1272?version=2, https://workflowhub.eu/workflows/1850?version=1, https://workflowhub.eu/workflows/1854?version=1</td>
      <td>microbiome/mgnify-amplicon</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>purge_dups</td>
      <td>purge_dups</td>
      <td>Purge haplotigs and overlaps in an assembly based on read depth</td>
      <td>2021-02-03</td>
      <td>https://github.com/dfguan/purge_dups</td>
      <td>1.2.6</td>
      <td>purge_dups</td>
      <td>1.2.6</td>
      <td>Up-to-date</td>
      <td>Assembly</td>
      <td>Genome assembly, Read binning, Scaffolding</td>
      <td>Genome assembly, Read binning, Scaffolding</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/purge_dups</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/purge_dups</td>
      <td>purge_dups</td>
      <td>purge_dups</td>
      <td>Identifying and removing haplotypic duplication in primary genome assemblies | haplotypic duplication identification tool | scripts/pd_config.py: script to generate a configuration file used by run_purge_dups.py | purge haplotigs and overlaps in an assembly based on read depth | Given a primary assembly pri_asm and an alternative assembly hap_asm (optional, if you have one), follow the steps shown below to build your own purge_dups pipeline, steps with same number can be run simultaneously. Among all the steps, although step 4 is optional, we highly recommend our users to do so, because assemblers may produce overrepresented seqeuences. In such a case, The final step 4 can be applied to remove those seqeuences</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>365</td>
      <td>365</td>
      <td>19303</td>
      <td>19303</td>
      <td>97</td>
      <td>97</td>
      <td>1776</td>
      <td>1776</td>
      <td>80</td>
      <td>80</td>
      <td>3001</td>
      <td>3001</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>24080</td>
      <td>24080</td>
      <td>542</td>
      <td>542</td>
      <td>https://usegalaxy.eu/published/workflow?id=032a7281a86eba0d, https://usegalaxy.eu/published/workflow?id=03bb4c20a1f4b4de, https://usegalaxy.eu/published/workflow?id=0444c98a3ca2a3ca, https://usegalaxy.eu/published/workflow?id=04844dddcc4e8aa7, https://usegalaxy.eu/published/workflow?id=0a3ebee985c26060, https://usegalaxy.eu/published/workflow?id=0f207a15b6f7fa10, https://usegalaxy.eu/published/workflow?id=122da0054fd40adf, https://usegalaxy.eu/published/workflow?id=12b0bd336c76a874, https://usegalaxy.eu/published/workflow?id=1af73dc855d0a5ed, https://usegalaxy.eu/published/workflow?id=1d1ae39386b5c318, https://usegalaxy.eu/published/workflow?id=1e7f4ba6261f979c, https://usegalaxy.eu/published/workflow?id=2294ef9515bcf972, https://usegalaxy.eu/published/workflow?id=29c97d43eeb8ed63, https://usegalaxy.eu/published/workflow?id=3192f92ac4d389b8, https://usegalaxy.eu/published/workflow?id=3480204a99bf4a35, https://usegalaxy.eu/published/workflow?id=399048afb4acef3d, https://usegalaxy.eu/published/workflow?id=5bf6718e7d6158d2, https://usegalaxy.eu/published/workflow?id=5c51f26ef8b20633, https://usegalaxy.eu/published/workflow?id=6865f33cffca4922, https://usegalaxy.eu/published/workflow?id=69be0fb8276753ec, https://usegalaxy.eu/published/workflow?id=6c14b08a7ef7e79a, https://usegalaxy.eu/published/workflow?id=700fa3ab11a8f3e2, https://usegalaxy.eu/published/workflow?id=83a1b8949d87e3da, https://usegalaxy.eu/published/workflow?id=8f105de48f7f1d9d, https://usegalaxy.eu/published/workflow?id=915ffef5b14c871e, https://usegalaxy.eu/published/workflow?id=9667bf36217083b7, https://usegalaxy.eu/published/workflow?id=9d209bcdae1a475e, https://usegalaxy.eu/published/workflow?id=a3f082ff30357040, https://usegalaxy.eu/published/workflow?id=ade6f36f6a035a73, https://usegalaxy.eu/published/workflow?id=aea88bfca6d1d476, https://usegalaxy.eu/published/workflow?id=aef62f4d0bba003b, https://usegalaxy.eu/published/workflow?id=b5de94e9cde071ab, https://usegalaxy.eu/published/workflow?id=bac84a7033c3423f, https://usegalaxy.eu/published/workflow?id=ca39822372847670, https://usegalaxy.eu/published/workflow?id=d02458f1d7d9c3f4, https://usegalaxy.eu/published/workflow?id=d172f927a489e9f8, https://usegalaxy.eu/published/workflow?id=d17d6029914d666b, https://usegalaxy.eu/published/workflow?id=d1bd5d8597c1d31d, https://usegalaxy.eu/published/workflow?id=d261bbd05889ce75, https://usegalaxy.eu/published/workflow?id=d67299c34e63f3d9, https://usegalaxy.eu/published/workflow?id=d924fafd8e042952, https://usegalaxy.eu/published/workflow?id=f08b29e4cef76cd5, https://usegalaxy.eu/published/workflow?id=f3e17c8dd80def27, https://usegalaxy.eu/published/workflow?id=f7cdaf862e6599c0, https://usegalaxy.eu/published/workflow?id=f882d14fc5dabbb9, https://usegalaxy.eu/published/workflow?id=f91245e580319e65, https://usegalaxy.org.au/published/workflow?id=000c2f1884bc4687, https://usegalaxy.org.au/published/workflow?id=06564b19b0655c19, https://usegalaxy.org.au/published/workflow?id=0810be3d7d2b0322, https://usegalaxy.org.au/published/workflow?id=094655df3ccd0b00, https://usegalaxy.org.au/published/workflow?id=0c11e1ba5e8fa21b, https://usegalaxy.org.au/published/workflow?id=0df25e2dbd5e8ed5, https://usegalaxy.org.au/published/workflow?id=1905ec04d5841f09, https://usegalaxy.org.au/published/workflow?id=1ce2a725df3adb74, https://usegalaxy.org.au/published/workflow?id=1dcc1b25c9091fbf, https://usegalaxy.org.au/published/workflow?id=1f49c65625b9d2de, https://usegalaxy.org.au/published/workflow?id=2356584eac7fa234, https://usegalaxy.org.au/published/workflow?id=269b42605a2a87d2, https://usegalaxy.org.au/published/workflow?id=328adfa7d90b1d54, https://usegalaxy.org.au/published/workflow?id=40740c32b9a550c9, https://usegalaxy.org.au/published/workflow?id=41282d51f9112552, https://usegalaxy.org.au/published/workflow?id=434127690544d0e9, https://usegalaxy.org.au/published/workflow?id=4a404aa3e9f29e5a, https://usegalaxy.org.au/published/workflow?id=4cb69c84b73ea9dd, https://usegalaxy.org.au/published/workflow?id=4e7930c7286c6e9e, https://usegalaxy.org.au/published/workflow?id=4ebf1297793870e7, https://usegalaxy.org.au/published/workflow?id=543fb2d9431a7e13, https://usegalaxy.org.au/published/workflow?id=6f76c43e6d2d7f74, https://usegalaxy.org.au/published/workflow?id=7168c01be9997823, https://usegalaxy.org.au/published/workflow?id=7909da83448d8bf4, https://usegalaxy.org.au/published/workflow?id=79bf7b0f5e9052d7, https://usegalaxy.org.au/published/workflow?id=81a5b96452c71a52, https://usegalaxy.org.au/published/workflow?id=82d0876385b5ac60, https://usegalaxy.org.au/published/workflow?id=a1015b83baa0eb6d, https://usegalaxy.org.au/published/workflow?id=a209f43d8a3cc853, https://usegalaxy.org.au/published/workflow?id=a40dadc943a0db1e, https://usegalaxy.org.au/published/workflow?id=a740543c4e9808f1, https://usegalaxy.org.au/published/workflow?id=ac8bdc5a0b2d70cd, https://usegalaxy.org.au/published/workflow?id=ad556db35858e7ec, https://usegalaxy.org.au/published/workflow?id=b310af289ada88bc, https://usegalaxy.org.au/published/workflow?id=c556c2270200801d, https://usegalaxy.org.au/published/workflow?id=c853aa707fcddf4e, https://usegalaxy.org.au/published/workflow?id=cbf022e349d726d9, https://usegalaxy.org.au/published/workflow?id=d678e57c7f1baf5d, https://usegalaxy.org.au/published/workflow?id=d83d5ecf31c20490, https://usegalaxy.org.au/published/workflow?id=e5dfeff4a3068f36, https://usegalaxy.org.au/published/workflow?id=ecfb8a08e5a40e15, https://usegalaxy.org.au/published/workflow?id=f06370394b061e08, https://usegalaxy.org.au/published/workflow?id=f895914e322c8b77, https://usegalaxy.org.au/published/workflow?id=fea0e6c02c2f6274, https://usegalaxy.org/published/workflow?id=10d5e195501603bc, https://usegalaxy.org/published/workflow?id=1b434bfc3aac0022, https://usegalaxy.org/published/workflow?id=237ca9f36ca266e7, https://usegalaxy.org/published/workflow?id=242ef2e9dee35702, https://usegalaxy.org/published/workflow?id=2acbb8c5f72b9f1c, https://usegalaxy.org/published/workflow?id=2f1469dab87a3649, https://usegalaxy.org/published/workflow?id=31387ed5ebc88cd2, https://usegalaxy.org/published/workflow?id=3179c157fa5a2fdd, https://usegalaxy.org/published/workflow?id=3753dcca4100e976, https://usegalaxy.org/published/workflow?id=426358bd988cb39d, https://usegalaxy.org/published/workflow?id=45c0b677c4920d03, https://usegalaxy.org/published/workflow?id=49c9e8161a46667f, https://usegalaxy.org/published/workflow?id=4bf38f5b45faa8a1, https://usegalaxy.org/published/workflow?id=5edabfc62f58d8cf, https://usegalaxy.org/published/workflow?id=65ccf48b7872e36c, https://usegalaxy.org/published/workflow?id=6a27c0d224683f73, https://usegalaxy.org/published/workflow?id=6fc7497c86ebd8ee, https://usegalaxy.org/published/workflow?id=73c20d784b7ef81a, https://usegalaxy.org/published/workflow?id=771345c66d5b9700, https://usegalaxy.org/published/workflow?id=7adf904dea83e699, https://usegalaxy.org/published/workflow?id=86c67855acbf6124, https://usegalaxy.org/published/workflow?id=8ccb42012dcb5db2, https://usegalaxy.org/published/workflow?id=950c7245a6c3f6e9, https://usegalaxy.org/published/workflow?id=96bd0007dceec123, https://usegalaxy.org/published/workflow?id=9ab68c65337b893b, https://usegalaxy.org/published/workflow?id=b12f99e2233fcca6, https://usegalaxy.org/published/workflow?id=c09b8a0c96123cbb, https://usegalaxy.org/published/workflow?id=c694856c01ce7cc5, https://usegalaxy.org/published/workflow?id=cb4ccd3068819418, https://usegalaxy.org/published/workflow?id=cc50da71cdf6f244, https://usegalaxy.org/published/workflow?id=cfd5dc3c7bc1e832, https://usegalaxy.org/published/workflow?id=d2e8a6b39fe88f4c, https://usegalaxy.org/published/workflow?id=d4b2b85db47c405e, https://usegalaxy.org/published/workflow?id=e12e642b88a0fe08, https://usegalaxy.org/published/workflow?id=eaf96fa3c39c210b, https://usegalaxy.org/published/workflow?id=f93a4f6b5f3aaaed, https://usegalaxy.org/published/workflow?id=fd155fcba9bed7ff, https://workflowhub.eu/workflows/1163?version=2, https://workflowhub.eu/workflows/1607?version=1, https://workflowhub.eu/workflows/237?version=2, https://workflowhub.eu/workflows/606?version=2, https://workflowhub.eu/workflows/633?version=23, https://workflowhub.eu/workflows/701?version=1, https://workflowhub.eu/workflows/751?version=18</td>
      <td>assembly/vgp_genome_assembly</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>quast</td>
      <td>quast</td>
      <td>Quast (Quality ASsessment Tool) evaluates genome assemblies.</td>
      <td>2016-09-07</td>
      <td>http://quast.bioinf.spbau.ru/</td>
      <td>5.3.0</td>
      <td>quast</td>
      <td>5.3.0</td>
      <td>Up-to-date</td>
      <td>Assembly</td>
      <td>Visualisation, Sequence assembly validation</td>
      <td>Visualisation, Sequence assembly validation</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/quast</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/quast</td>
      <td>quast</td>
      <td>QUAST</td>
      <td>QUAST stands for QUality ASsessment Tool.  It evaluates a quality of genome assemblies by computing various metrics and providing nice reports.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>11028</td>
      <td>11672</td>
      <td>120605</td>
      <td>126603</td>
      <td>13078</td>
      <td>14773</td>
      <td>78457</td>
      <td>86986</td>
      <td>4849</td>
      <td>5353</td>
      <td>46644</td>
      <td>50987</td>
      <td>613</td>
      <td>619</td>
      <td>14902</td>
      <td>14963</td>
      <td>279539</td>
      <td>260608</td>
      <td>32417</td>
      <td>29568</td>
      <td>https://usegalaxy.eu/published/workflow?id=007ce6405d196df3, https://usegalaxy.eu/published/workflow?id=01dc0619e050dc91, https://usegalaxy.eu/published/workflow?id=0404013d5c27210d, https://usegalaxy.eu/published/workflow?id=0563f58718be932d, https://usegalaxy.eu/published/workflow?id=085c227c9a56fcae, https://usegalaxy.eu/published/workflow?id=0ebfab0e957b7267, https://usegalaxy.eu/published/workflow?id=1524b189069b42a5, https://usegalaxy.eu/published/workflow?id=165089132248a0b0, https://usegalaxy.eu/published/workflow?id=1860e02bcb894a64, https://usegalaxy.eu/published/workflow?id=1e5f1d9a51032e44, https://usegalaxy.eu/published/workflow?id=1ea8bfbe6cb04531, https://usegalaxy.eu/published/workflow?id=2e0075791f7fa384, https://usegalaxy.eu/published/workflow?id=309fc4fc2e0da89c, https://usegalaxy.eu/published/workflow?id=30b514cfb9cce524, https://usegalaxy.eu/published/workflow?id=3303a04887a47675, https://usegalaxy.eu/published/workflow?id=34a07bf6407510ef, https://usegalaxy.eu/published/workflow?id=368d1ac0150ea0b6, https://usegalaxy.eu/published/workflow?id=374797e58377c40f, https://usegalaxy.eu/published/workflow?id=37da7b42cb063a3a, https://usegalaxy.eu/published/workflow?id=3d937648809ca4e7, https://usegalaxy.eu/published/workflow?id=4216424f032c055f, https://usegalaxy.eu/published/workflow?id=45c0933b5c522628, https://usegalaxy.eu/published/workflow?id=4ddbffe4b3fef275, https://usegalaxy.eu/published/workflow?id=56edd22d14392691, https://usegalaxy.eu/published/workflow?id=579ff83177c5ee09, https://usegalaxy.eu/published/workflow?id=5826e573b6f93762, https://usegalaxy.eu/published/workflow?id=5b940b283d91d99d, https://usegalaxy.eu/published/workflow?id=6158c3b5fc12044e, https://usegalaxy.eu/published/workflow?id=628062a8cf93261f, https://usegalaxy.eu/published/workflow?id=62c47e3c9d4f3aa3, https://usegalaxy.eu/published/workflow?id=667bac8d7453e5f1, https://usegalaxy.eu/published/workflow?id=69be0fb8276753ec, https://usegalaxy.eu/published/workflow?id=6d87481be47902e1, https://usegalaxy.eu/published/workflow?id=723e718f37e9ab91, https://usegalaxy.eu/published/workflow?id=7aa32aba3fde0708, https://usegalaxy.eu/published/workflow?id=7e48134082dab0a3, https://usegalaxy.eu/published/workflow?id=7ecd92cb382d8168, https://usegalaxy.eu/published/workflow?id=842e0ea7b1eb56af, https://usegalaxy.eu/published/workflow?id=8b2f53745e720d0f, https://usegalaxy.eu/published/workflow?id=8b8e89b9b8801f2f, https://usegalaxy.eu/published/workflow?id=8deb08d4f228e198, https://usegalaxy.eu/published/workflow?id=8f3ce55d44d9f369, https://usegalaxy.eu/published/workflow?id=9219557de92e0203, https://usegalaxy.eu/published/workflow?id=94c3d329c31c712f, https://usegalaxy.eu/published/workflow?id=968f0ac2617aa524, https://usegalaxy.eu/published/workflow?id=97312d273b6e8bd9, https://usegalaxy.eu/published/workflow?id=98bfad0c684011e9, https://usegalaxy.eu/published/workflow?id=9b9e1e28f3683f92, https://usegalaxy.eu/published/workflow?id=9ddb703e1db13504, https://usegalaxy.eu/published/workflow?id=a337bfb2eb1e583f, https://usegalaxy.eu/published/workflow?id=a847773b563b2328, https://usegalaxy.eu/published/workflow?id=a84bac86ec183d46, https://usegalaxy.eu/published/workflow?id=b1d9e56aef03672b, https://usegalaxy.eu/published/workflow?id=b2b46e71046b1999, https://usegalaxy.eu/published/workflow?id=b627aee461e50756, https://usegalaxy.eu/published/workflow?id=b7c2440621968526, https://usegalaxy.eu/published/workflow?id=b93333efdab316d8, https://usegalaxy.eu/published/workflow?id=ba1fda2318046543, https://usegalaxy.eu/published/workflow?id=c9ef2d2ce92e2dfb, https://usegalaxy.eu/published/workflow?id=cf8591391a17a914, https://usegalaxy.eu/published/workflow?id=d57d41e306241396, https://usegalaxy.eu/published/workflow?id=d67299c34e63f3d9, https://usegalaxy.eu/published/workflow?id=e532d98f4b368666, https://usegalaxy.eu/published/workflow?id=e8eb80ee09159bac, https://usegalaxy.eu/published/workflow?id=ea485dbabc76488d, https://usegalaxy.eu/published/workflow?id=eb5ab8bc82569eee, https://usegalaxy.eu/published/workflow?id=ecf48fa50af560a0, https://usegalaxy.eu/published/workflow?id=fba1671d835b6dd3, https://usegalaxy.eu/published/workflow?id=fca458178b3c794a, https://usegalaxy.eu/published/workflow?id=fde21b5f5c169114, https://usegalaxy.eu/published/workflow?id=fe43cf59e48bf556, https://usegalaxy.org.au/published/workflow?id=02c74095a7128c9d, https://usegalaxy.org.au/published/workflow?id=1ad4c47cee851dda, https://usegalaxy.org.au/published/workflow?id=243c67fa4312855e, https://usegalaxy.org.au/published/workflow?id=2e29b21cdec7d0cc, https://usegalaxy.org.au/published/workflow?id=30430b75d05f54eb, https://usegalaxy.org.au/published/workflow?id=39c724282ee008e3, https://usegalaxy.org.au/published/workflow?id=4023bca282ca9315, https://usegalaxy.org.au/published/workflow?id=434127690544d0e9, https://usegalaxy.org.au/published/workflow?id=478c79365fc9677f, https://usegalaxy.org.au/published/workflow?id=533dbb3242ee4255, https://usegalaxy.org.au/published/workflow?id=5b32e2f0d390a4ba, https://usegalaxy.org.au/published/workflow?id=5cd804644a34c8fd, https://usegalaxy.org.au/published/workflow?id=68f5c371adbe5490, https://usegalaxy.org.au/published/workflow?id=7fa13ccbf044edc5, https://usegalaxy.org.au/published/workflow?id=83e3ceceb1a92493, https://usegalaxy.org.au/published/workflow?id=874492eb776ba813, https://usegalaxy.org.au/published/workflow?id=8a40059343cf4b87, https://usegalaxy.org.au/published/workflow?id=8b6739a000780d99, https://usegalaxy.org.au/published/workflow?id=8d47811c6154d347, https://usegalaxy.org.au/published/workflow?id=8f7d5ebead46a6d0, https://usegalaxy.org.au/published/workflow?id=92fcdd490831c6c5, https://usegalaxy.org.au/published/workflow?id=9b75924c483b5f6b, https://usegalaxy.org.au/published/workflow?id=aee5b5645c51cac3, https://usegalaxy.org.au/published/workflow?id=b6bad1f167257248, https://usegalaxy.org.au/published/workflow?id=b7aa22291322e505, https://usegalaxy.org.au/published/workflow?id=d06665e4bc7fff6d, https://usegalaxy.org.au/published/workflow?id=d832905fbaf2ca31, https://usegalaxy.org.au/published/workflow?id=d957bfeb3cd03db0, https://usegalaxy.org.au/published/workflow?id=df11f09771e73e04, https://usegalaxy.org.au/published/workflow?id=ee7a1615f187391d, https://usegalaxy.org.au/published/workflow?id=f01be6a7685ac476, https://usegalaxy.org.au/published/workflow?id=f25a05070bf97ded, https://usegalaxy.org.au/published/workflow?id=fb0986a952c35e0a, https://usegalaxy.org.au/published/workflow?id=fc664a96d8099652, https://usegalaxy.org/published/workflow?id=00596e862edb23f2, https://usegalaxy.org/published/workflow?id=0602c14704dbe1ff, https://usegalaxy.org/published/workflow?id=0e30324072570abb, https://usegalaxy.org/published/workflow?id=109092d8dbda4398, https://usegalaxy.org/published/workflow?id=1c7af480caf150bd, https://usegalaxy.org/published/workflow?id=3cf46c167ccb374f, https://usegalaxy.org/published/workflow?id=4029a0c8fa23d375, https://usegalaxy.org/published/workflow?id=4161cfd67ed94b5d, https://usegalaxy.org/published/workflow?id=4d1279007d0137fa, https://usegalaxy.org/published/workflow?id=4e1ae78405034ad3, https://usegalaxy.org/published/workflow?id=51c485af11d90700, https://usegalaxy.org/published/workflow?id=524602abe59b18f7, https://usegalaxy.org/published/workflow?id=547f86db467bf620, https://usegalaxy.org/published/workflow?id=562bcaa9b4d2e8a4, https://usegalaxy.org/published/workflow?id=5cd56232e873beab, https://usegalaxy.org/published/workflow?id=69d8863d40524120, https://usegalaxy.org/published/workflow?id=7a33bc2d0970b502, https://usegalaxy.org/published/workflow?id=7dd6e4150bb81ad1, https://usegalaxy.org/published/workflow?id=80565301ada0d3eb, https://usegalaxy.org/published/workflow?id=80ac6adead9f97d8, https://usegalaxy.org/published/workflow?id=8111bb9bb26bb665, https://usegalaxy.org/published/workflow?id=88bc9aa7090875c1, https://usegalaxy.org/published/workflow?id=9ac1e4efe310b3b3, https://usegalaxy.org/published/workflow?id=9c0fbd9f51a9857f, https://usegalaxy.org/published/workflow?id=9e5c5ddb60a24889, https://usegalaxy.org/published/workflow?id=a67afbea4b2eb26b, https://usegalaxy.org/published/workflow?id=b76a9ce44ea019f1, https://usegalaxy.org/published/workflow?id=c30f6ca171896908, https://usegalaxy.org/published/workflow?id=c454c930ccfb0d3f, https://usegalaxy.org/published/workflow?id=c65bec6c332bf740, https://usegalaxy.org/published/workflow?id=c8d74054c1820087, https://usegalaxy.org/published/workflow?id=cdcf91a8ddb5dcd1, https://usegalaxy.org/published/workflow?id=db08887d7d6dc147, https://usegalaxy.org/published/workflow?id=e2ab3d3219a648a7, https://usegalaxy.org/published/workflow?id=e745c45651d829b8, https://usegalaxy.org/published/workflow?id=f4d623a19b32370e, https://usegalaxy.org/published/workflow?id=f926e70bee242785, https://usegalaxy.org/published/workflow?id=fb01e1a49793c8be, https://workflowhub.eu/workflows/1352?version=4, https://workflowhub.eu/workflows/1390?version=2, https://workflowhub.eu/workflows/1580?version=1, https://workflowhub.eu/workflows/1595?version=1, https://workflowhub.eu/workflows/1598?version=1, https://workflowhub.eu/workflows/1602?version=2, https://workflowhub.eu/workflows/1607?version=1, https://workflowhub.eu/workflows/1613?version=2, https://workflowhub.eu/workflows/1621?version=1, https://workflowhub.eu/workflows/1622?version=1, https://workflowhub.eu/workflows/1634?version=1, https://workflowhub.eu/workflows/1646?version=1, https://workflowhub.eu/workflows/1882?version=3, https://workflowhub.eu/workflows/2011?version=1, https://workflowhub.eu/workflows/2024?version=2, https://workflowhub.eu/workflows/225?version=1, https://workflowhub.eu/workflows/229?version=1, https://workflowhub.eu/workflows/403?version=9, https://workflowhub.eu/workflows/750?version=3, https://workflowhub.eu/workflows/794?version=2</td>
      <td>assembly/assembly-quality-control, assembly/general-introduction, assembly/hifi-assembly, assembly/hybrid_denovo_assembly, assembly/largegenome, assembly/metagenomics-assembly, assembly/mrsa-illumina, assembly/mrsa-nanopore, assembly/unicycler-assembly, assembly/vgp_genome_assembly, assembly/vgp_workflow_training, microbiome/metagenomics-assembly</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>quickmerge</td>
      <td>quickmerge</td>
      <td>Merge long-read and hybrid assemblies to increase contiguity</td>
      <td>2022-07-08</td>
      <td>https://github.com/mahulchak/quickmerge</td>
      <td>0.3</td>
      <td>quickmerge</td>
      <td>0.3</td>
      <td>Up-to-date</td>
      <td>Assembly</td>
      <td>Genome assembly, Scaffolding, De-novo assembly, Genotyping</td>
      <td>Genome assembly, Scaffolding, De-novo assembly, Genotyping</td>
      <td>Structural variation, Sequence assembly, DNA polymorphism, Whole genome sequencing, Genotype and phenotype</td>
      <td>Structural variation, Sequence assembly, DNA polymorphism, Whole genome sequencing, Genotype and phenotype</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/quickmerge</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/quickmerge</td>
      <td>quickmerge</td>
      <td>quickmerge</td>
      <td>Quickmerge is a program that uses complementary information from genomes assembled with long reads in order to improve contiguity, and works with assemblies derived from both Pacific Biosciences or Oxford Nanopore. Quickmerge will even work with hybrid assemblies made by combining long reads and Illumina short reads.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>ragtag</td>
      <td>ragtag</td>
      <td>Reference-guided scaffolding of draft genomes tool.</td>
      <td>2021-11-10</td>
      <td>https://github.com/malonge/RagTag</td>
      <td>2.1.0</td>
      <td>ragtag</td>
      <td>2.1.0</td>
      <td>Up-to-date</td>
      <td>Assembly</td>
      <td>Genome assembly</td>
      <td>Genome assembly</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ragtag</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ragtag</td>
      <td>ragtag</td>
      <td>ragtag</td>
      <td>RagTag is a collection of software tools for scaffolding and improving modern genome assemblies.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>774</td>
      <td>774</td>
      <td>8106</td>
      <td>8106</td>
      <td>6</td>
      <td>6</td>
      <td>59</td>
      <td>59</td>
      <td>108</td>
      <td>108</td>
      <td>2145</td>
      <td>2145</td>
      <td>32</td>
      <td>32</td>
      <td>416</td>
      <td>416</td>
      <td>10726</td>
      <td>10726</td>
      <td>920</td>
      <td>920</td>
      <td>https://usegalaxy.org.au/published/workflow?id=7f37c5366bce5489</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>raven</td>
      <td>raven</td>
      <td>Raven is a de novo genome assembler for long uncorrected reads.</td>
      <td>2020-11-04</td>
      <td>https://github.com/lbcb-sci/raven</td>
      <td>1.8.3</td>
      <td>raven-assembler</td>
      <td>1.8.3</td>
      <td>Up-to-date</td>
      <td>Assembly</td>
      <td>De-novo assembly, Genome assembly, Read pre-processing</td>
      <td>De-novo assembly, Genome assembly, Read pre-processing</td>
      <td>Sequence assembly, Whole genome sequencing, Genomics</td>
      <td>Sequence assembly, Whole genome sequencing, Genomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/raven</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/raven</td>
      <td>raven</td>
      <td>Raven</td>
      <td>a de novo genome assembler for long reads.Raven is a de novo genome assembler for long uncorrected reads.</td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>610</td>
      <td>610</td>
      <td>15837</td>
      <td>15837</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>276</td>
      <td>276</td>
      <td>13041</td>
      <td>13041</td>
      <td>24</td>
      <td>24</td>
      <td>217</td>
      <td>217</td>
      <td>29095</td>
      <td>29095</td>
      <td>910</td>
      <td>910</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>regionalgam</td>
      <td>regionalgam_ab_index, regionalgam_autocor_acf, regionalgam_flight_curve, regionalgam_glmmpql, regionalgam_gls_adjusted, regionalgam_gls, regionalgam_plot_trend</td>
      <td></td>
      <td>2019-06-18</td>
      <td>https://github.com/RetoSchmucki/regionalGAM</td>
      <td>1.5</td>
      <td>r-mgcv</td>
      <td></td>
      <td>To update</td>
      <td>Ecology</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/regionalgam</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/regionalgam</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>18</td>
      <td>39</td>
      <td>129</td>
      <td>547</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>547</td>
      <td>129</td>
      <td>39</td>
      <td>18</td>
      <td>https://usegalaxy.eu/published/workflow?id=8e92a65d3307d87e, https://workflowhub.eu/workflows/1707?version=1</td>
      <td>ecology/regionalGAM</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>repeatexplorer2</td>
      <td>repeatexplorer_clustering</td>
      <td>Tool for annotation of repeats from unassembled shotgun reads.</td>
      <td>2023-11-01</td>
      <td>https://github.com/repeatexplorer/repex_tarean</td>
      <td>2.3.8</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>Genome annotation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>gga</td>
      <td>https://github.com/galaxy-genome-annotation/galaxy-tools/tree/master/tools/repeatexplorer2</td>
      <td>https://github.com/galaxy-genome-annotation/galaxy-tools/tree/master/tools/repeatexplorer2</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>83</td>
      <td>83</td>
      <td>369</td>
      <td>369</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>369</td>
      <td>369</td>
      <td>83</td>
      <td>83</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>repeatmodeler</td>
      <td>repeatmodeler</td>
      <td>RepeatModeler - Model repetitive DNA</td>
      <td>2021-08-26</td>
      <td>https://www.repeatmasker.org/RepeatModeler/</td>
      <td>2.0.5</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>Genome annotation</td>
      <td>Repeat sequence detection</td>
      <td>Repeat sequence detection</td>
      <td>Sequence composition, complexity and repeats, Sequence composition, complexity and repeats</td>
      <td>Sequence composition, complexity and repeats, Sequence composition, complexity and repeats</td>
      <td>csbl</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/repeatmodeler</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/repeatmodeler</td>
      <td>repeatmodeler</td>
      <td>RepeatModeler</td>
      <td>De-novo repeat family identification and modeling package. At the heart of RepeatModeler are two de-novo repeat finding programs ( RECON and RepeatScout ) which employ complementary computational methods for identifying repeat element boundaries and family relationships from sequence data. RepeatModeler assists in automating the runs of RECON and RepeatScout given a genomic database and uses the output to build, refine and classify consensus models of putative interspersed repeats.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>725</td>
      <td>725</td>
      <td>9423</td>
      <td>9423</td>
      <td>424</td>
      <td>424</td>
      <td>2473</td>
      <td>2473</td>
      <td>240</td>
      <td>240</td>
      <td>8995</td>
      <td>8995</td>
      <td>30</td>
      <td>30</td>
      <td>270</td>
      <td>270</td>
      <td>21161</td>
      <td>21161</td>
      <td>1419</td>
      <td>1419</td>
      <td>https://usegalaxy.eu/published/workflow?id=09c534f738426a6d, https://usegalaxy.eu/published/workflow?id=327beb1edda305ff, https://usegalaxy.eu/published/workflow?id=4077e77b6af919c4, https://usegalaxy.eu/published/workflow?id=f9279495683a8ac3, https://usegalaxy.fr/published/workflow?id=ae05ff8487f58fd6, https://usegalaxy.org.au/published/workflow?id=a71349ba0bff403a, https://usegalaxy.org.au/published/workflow?id=a9068ea858222e2b, https://usegalaxy.org.au/published/workflow?id=ffb1117077f68a9f, https://usegalaxy.org/published/workflow?id=effc17e170263ae6, https://workflowhub.eu/workflows/575?version=1, https://workflowhub.eu/workflows/875?version=3</td>
      <td>genome-annotation/repeatmasker</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>retrieve_bold</td>
      <td>retrieve_bold</td>
      <td>Search a list of sequences in BOLD (Barcode of Life Data System) from specified taxa list and markers</td>
      <td>2024-06-21</td>
      <td>https://anaconda.org/conda-forge/r-bold</td>
      <td>1.3.0</td>
      <td>r-base</td>
      <td></td>
      <td>To update</td>
      <td>Ecology</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/wpearman1996/MARES_database_pipeline/tree/master</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/retrieve_bold</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>15</td>
      <td>15</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>15</td>
      <td>15</td>
      <td>8</td>
      <td>8</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>rnaquast</td>
      <td>rna_quast</td>
      <td>rnaQuast (RNA Quality Assessment Tool) evaluates genome assemblies.</td>
      <td>2020-10-05</td>
      <td>https://github.com/ablab/rnaquast</td>
      <td>2.3.0</td>
      <td>rnaquast</td>
      <td>2.3.0</td>
      <td>Up-to-date</td>
      <td>Assembly, RNA</td>
      <td>De-novo assembly, Transcriptome assembly, Sequence assembly validation</td>
      <td>De-novo assembly, Transcriptome assembly, Sequence assembly validation</td>
      <td>Sequence assembly, Transcriptomics, RNA-Seq</td>
      <td>Sequence assembly, Transcriptomics, RNA-Seq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/rnaquast</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/rnaquast</td>
      <td>rnaQUAST</td>
      <td>rnaQUAST</td>
      <td>Quality assessment tool for de novo transcriptome assemblies.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>375</td>
      <td>375</td>
      <td>1887</td>
      <td>1887</td>
      <td>196</td>
      <td>196</td>
      <td>767</td>
      <td>767</td>
      <td>31</td>
      <td>31</td>
      <td>326</td>
      <td>326</td>
      <td>11</td>
      <td>11</td>
      <td>103</td>
      <td>103</td>
      <td>3083</td>
      <td>3083</td>
      <td>613</td>
      <td>613</td>
      <td>https://usegalaxy.eu/published/workflow?id=47a0407867a90f95, https://usegalaxy.eu/published/workflow?id=9b025479fade745d, https://usegalaxy.eu/published/workflow?id=b7a3cbcd32734e8b, https://workflowhub.eu/workflows/1688?version=1</td>
      <td>transcriptomics/differential-isoform-expression</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>salsa</td>
      <td>salsa</td>
      <td>A tool to scaffold long read assemblies with Hi-C</td>
      <td>2021-01-14</td>
      <td>https://github.com/marbl/SALSA</td>
      <td>2.3</td>
      <td>salsa2</td>
      <td>2.3</td>
      <td>Up-to-date</td>
      <td>Assembly</td>
      <td>Genome assembly, De-novo assembly, Scaffolding</td>
      <td>Genome assembly, De-novo assembly, Scaffolding</td>
      <td>Sequence assembly, DNA binding sites, Mapping</td>
      <td>Sequence assembly, DNA binding sites, Mapping</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/salsa2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/salsa2</td>
      <td>SALSA</td>
      <td>SALSA</td>
      <td>&gt; VERY_LOW CONFIDENCE! | &gt; CORRECT NAME OF TOOL COULD ALSO BE 'chromosome-scale', 'reference-quality', 'Hi-C', 'scaffolder' | Integrating Hi-C links with assembly graphs for chromosome-scale assembly | SALSA: A tool to scaffold long read assemblies with Hi-C data | SALSA: A tool to scaffold long read assemblies with Hi-C | This code is used to scaffold your assemblies using Hi-C data. This version implements some improvements in the original SALSA algorithm. If you want to use the old version, it can be found in the old_salsa branch</td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>130</td>
      <td>130</td>
      <td>495</td>
      <td>495</td>
      <td>103</td>
      <td>103</td>
      <td>1074</td>
      <td>1074</td>
      <td>24</td>
      <td>24</td>
      <td>410</td>
      <td>410</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1979</td>
      <td>1979</td>
      <td>257</td>
      <td>257</td>
      <td>https://usegalaxy.org.au/published/workflow?id=434127690544d0e9, https://usegalaxy.org.au/published/workflow?id=e07866576cc2f467, https://workflowhub.eu/workflows/1607?version=1</td>
      <td>assembly/vgp_genome_assembly</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>sample_seqs</td>
      <td>sample_seqs</td>
      <td>Sub-sample sequences files (e.g. to reduce coverage)</td>
      <td>2014-02-18</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/sample_seqs</td>
      <td>0.2.6</td>
      <td>biopython</td>
      <td>1.70</td>
      <td>To update</td>
      <td>Assembly, Fasta Manipulation, Fastq Manipulation, Sequence Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/sample_seqs</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/sample_seqs</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>376</td>
      <td>457</td>
      <td>3760</td>
      <td>4585</td>
      <td>274</td>
      <td>274</td>
      <td>2029</td>
      <td>2029</td>
      <td>54</td>
      <td>54</td>
      <td>2601</td>
      <td>2601</td>
      <td>9</td>
      <td>9</td>
      <td>113</td>
      <td>113</td>
      <td>9328</td>
      <td>8503</td>
      <td>794</td>
      <td>713</td>
      <td></td>
      <td>transcriptomics/mirna-target-finder</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>samtools_depad</td>
      <td>samtools_depad</td>
      <td>Re-align a SAM/BAM file with a padded reference (using samtools depad)</td>
      <td>2014-04-14</td>
      <td>http://www.htslib.org/</td>
      <td>0.0.5</td>
      <td>samtools</td>
      <td>1.23</td>
      <td>To update</td>
      <td>Assembly, SAM, Sequence Analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/samtools_depad</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/samtools_depad</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>samtools_depth</td>
      <td>samtools_depth</td>
      <td>Coverage depth via samtools</td>
      <td>2014-11-19</td>
      <td>http://www.htslib.org/</td>
      <td>0.0.3</td>
      <td>samtools</td>
      <td>1.23</td>
      <td>To update</td>
      <td>Assembly, Sequence Analysis, SAM</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/samtools_depth</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/samtools_depth</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1071</td>
      <td>1130</td>
      <td>8403</td>
      <td>9175</td>
      <td>1771</td>
      <td>1964</td>
      <td>11692</td>
      <td>12814</td>
      <td>99</td>
      <td>99</td>
      <td>761</td>
      <td>761</td>
      <td>77</td>
      <td>77</td>
      <td>151</td>
      <td>151</td>
      <td>22901</td>
      <td>21007</td>
      <td>3270</td>
      <td>3018</td>
      <td>https://usegalaxy.eu/published/workflow?id=09c7069ae409c362, https://usegalaxy.eu/published/workflow?id=4a764275d4bf9f31, https://usegalaxy.eu/published/workflow?id=60e0b2a28a900178, https://usegalaxy.eu/published/workflow?id=61fe08fb3d5e83f8, https://usegalaxy.eu/published/workflow?id=64abcddeb516a712, https://usegalaxy.eu/published/workflow?id=6aa2891bd2b41f32, https://usegalaxy.eu/published/workflow?id=cdf2f02428f5f83f, https://usegalaxy.eu/published/workflow?id=d215a2e554fd83bc, https://usegalaxy.eu/published/workflow?id=d88145e7475e0b7b, https://usegalaxy.eu/published/workflow?id=fc0b9efa7d2e61e6, https://usegalaxy.org.au/published/workflow?id=0e11619979c2830c, https://usegalaxy.org.au/published/workflow?id=244ea5e94237ebad, https://usegalaxy.org.au/published/workflow?id=38f57cffaeb0e39f, https://usegalaxy.org.au/published/workflow?id=76a0a3af8de11622, https://usegalaxy.org.au/published/workflow?id=857e482acfcb2dba, https://usegalaxy.org/published/workflow?id=06d6dcd2655592af, https://usegalaxy.org/published/workflow?id=10ac77144ced73ad, https://usegalaxy.org/published/workflow?id=18899840e49d89c3, https://usegalaxy.org/published/workflow?id=29ba77708cdafe0f, https://usegalaxy.org/published/workflow?id=2ded909e7e3a9962, https://usegalaxy.org/published/workflow?id=357d644e99be5e13, https://usegalaxy.org/published/workflow?id=38911ba6f66d80f6, https://usegalaxy.org/published/workflow?id=3e703abaeb870f6f, https://usegalaxy.org/published/workflow?id=3f2d1003ff959c7c, https://usegalaxy.org/published/workflow?id=3ffb59671b58d5ce, https://usegalaxy.org/published/workflow?id=44b5161868a34561, https://usegalaxy.org/published/workflow?id=6d778eeb176d563c, https://usegalaxy.org/published/workflow?id=71f8a4090877f171, https://usegalaxy.org/published/workflow?id=84dfa09b472ed3f2, https://usegalaxy.org/published/workflow?id=8b4572ac274176c0, https://usegalaxy.org/published/workflow?id=8daa91d54040ba95, https://usegalaxy.org/published/workflow?id=b561d3d9645a2726, https://usegalaxy.org/published/workflow?id=c4af6c5f5dfaae82, https://usegalaxy.org/published/workflow?id=cb810cdaa4915f4b, https://usegalaxy.org/published/workflow?id=e215018e24b84197, https://usegalaxy.org/published/workflow?id=efb62e8178ecdcaf, https://usegalaxy.org/published/workflow?id=f3ed710e251833a7, https://usegalaxy.org/published/workflow?id=fcd98ca3abcf6c61, https://workflowhub.eu/workflows/1063?version=5, https://workflowhub.eu/workflows/1479?version=1, https://workflowhub.eu/workflows/1738?version=1, https://workflowhub.eu/workflows/1739?version=1, https://workflowhub.eu/workflows/1740?version=1, https://workflowhub.eu/workflows/1741?version=1, https://workflowhub.eu/workflows/2036?version=1, https://workflowhub.eu/workflows/2038?version=1</td>
      <td>microbiome/pathogen-detection-from-nanopore-foodborne-data</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>sdmpredictors</td>
      <td>sdmpredictors_list_layers</td>
      <td>Terrestrial and marine predictors for species distribution modelling.</td>
      <td>2024-03-29</td>
      <td>https://cran.r-project.org/web/packages/sdmpredictors/index.html</td>
      <td>0.2.15</td>
      <td>r-base</td>
      <td></td>
      <td>To update</td>
      <td>Ecology</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/sdmpredictors</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/sdmpredictors</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>10</td>
      <td>10</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>10</td>
      <td>4</td>
      <td>4</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>seq_primer_clip</td>
      <td>seq_primer_clip</td>
      <td>Trim off 5' or 3' primers</td>
      <td>2013-09-26</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_primer_clip</td>
      <td>0.0.18</td>
      <td>galaxy_sequence_utils</td>
      <td>1.2.0</td>
      <td>To update</td>
      <td>Assembly, Fasta Manipulation, Text Manipulation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_primer_clip</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_primer_clip</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>shasta</td>
      <td>shasta</td>
      <td>Fast de novo assembly of long read sequencing data</td>
      <td>2020-11-11</td>
      <td>https://github.com/chanzuckerberg/shasta</td>
      <td>0.6.0</td>
      <td>shasta</td>
      <td>0.14.0</td>
      <td>To update</td>
      <td>Assembly, Nanopore</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/shasta</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/shasta</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>399</td>
      <td>399</td>
      <td>1291</td>
      <td>1291</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>98</td>
      <td>98</td>
      <td>791</td>
      <td>791</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2082</td>
      <td>2082</td>
      <td>497</td>
      <td>497</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>shovill</td>
      <td>shovill</td>
      <td>Faster de novo assembly pipeline based around Spades</td>
      <td>2017-10-24</td>
      <td>https://github.com/tseemann/shovill</td>
      <td>1.4.2</td>
      <td>shovill</td>
      <td>1.4.2</td>
      <td>Up-to-date</td>
      <td>Assembly</td>
      <td>Genome assembly</td>
      <td>Genome assembly</td>
      <td>Genomics, Microbiology, Sequence assembly</td>
      <td>Genomics, Microbiology, Sequence assembly</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/shovill</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/shovill</td>
      <td>shovill</td>
      <td>shovill</td>
      <td>Shovill is a pipeline for assembly of bacterial isolate genomes from Illumina paired-end reads.  Shovill uses SPAdes at its core, but alters the steps before and after the primary assembly step to get similar results in less time. Shovill also supports other assemblers like SKESA, Velvet and Megahit, so you can take advantage of the pre- and post-processing the Shovill provides with those too.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>3269</td>
      <td>3491</td>
      <td>70801</td>
      <td>76107</td>
      <td>4933</td>
      <td>5212</td>
      <td>51724</td>
      <td>55190</td>
      <td>1425</td>
      <td>1669</td>
      <td>30312</td>
      <td>35794</td>
      <td>83</td>
      <td>83</td>
      <td>9136</td>
      <td>9136</td>
      <td>176227</td>
      <td>161973</td>
      <td>10455</td>
      <td>9710</td>
      <td>https://usegalaxy.eu/published/workflow?id=0404013d5c27210d, https://usegalaxy.eu/published/workflow?id=1ea8bfbe6cb04531, https://usegalaxy.eu/published/workflow?id=309fc4fc2e0da89c, https://usegalaxy.eu/published/workflow?id=3303a04887a47675, https://usegalaxy.eu/published/workflow?id=3d937648809ca4e7, https://usegalaxy.eu/published/workflow?id=4216424f032c055f, https://usegalaxy.eu/published/workflow?id=42c0da2e86318b37, https://usegalaxy.eu/published/workflow?id=4f245ac304ab76d9, https://usegalaxy.eu/published/workflow?id=5b893b69a1b6748f, https://usegalaxy.eu/published/workflow?id=5b940b283d91d99d, https://usegalaxy.eu/published/workflow?id=628062a8cf93261f, https://usegalaxy.eu/published/workflow?id=723e718f37e9ab91, https://usegalaxy.eu/published/workflow?id=7aa32aba3fde0708, https://usegalaxy.eu/published/workflow?id=7e48134082dab0a3, https://usegalaxy.eu/published/workflow?id=94c3d329c31c712f, https://usegalaxy.eu/published/workflow?id=a727e22dc0431614, https://usegalaxy.eu/published/workflow?id=b5ffba506152b1a1, https://usegalaxy.eu/published/workflow?id=b90585626a793502, https://usegalaxy.eu/published/workflow?id=b93333efdab316d8, https://usegalaxy.eu/published/workflow?id=ba1fda2318046543, https://usegalaxy.eu/published/workflow?id=cbe3769f1abb9fa5, https://usegalaxy.eu/published/workflow?id=e16e524c557d7928, https://usegalaxy.eu/published/workflow?id=e532d98f4b368666, https://usegalaxy.eu/published/workflow?id=fca458178b3c794a, https://usegalaxy.eu/published/workflow?id=fde21b5f5c169114, https://usegalaxy.org.au/published/workflow?id=52065a4100b11184, https://usegalaxy.org.au/published/workflow?id=7a0c4804e4d4c905, https://usegalaxy.org.au/published/workflow?id=92fcdd490831c6c5, https://usegalaxy.org.au/published/workflow?id=d78f885b4b86635f, https://usegalaxy.org.au/published/workflow?id=f2541030f6d70b59, https://usegalaxy.org.au/published/workflow?id=fa76b66785ce38b8, https://usegalaxy.org/published/workflow?id=0e30324072570abb, https://usegalaxy.org/published/workflow?id=109092d8dbda4398, https://usegalaxy.org/published/workflow?id=1c7af480caf150bd, https://usegalaxy.org/published/workflow?id=3cf46c167ccb374f, https://usegalaxy.org/published/workflow?id=4161cfd67ed94b5d, https://usegalaxy.org/published/workflow?id=4d1279007d0137fa, https://usegalaxy.org/published/workflow?id=524602abe59b18f7, https://usegalaxy.org/published/workflow?id=5cd56232e873beab, https://usegalaxy.org/published/workflow?id=8ebc2b1ae24ed884, https://usegalaxy.org/published/workflow?id=9ac1e4efe310b3b3, https://usegalaxy.org/published/workflow?id=9e5c5ddb60a24889, https://usegalaxy.org/published/workflow?id=b76a9ce44ea019f1, https://usegalaxy.org/published/workflow?id=c454c930ccfb0d3f, https://usegalaxy.org/published/workflow?id=d4c790a45978bb9e, https://usegalaxy.org/published/workflow?id=d4f0a7ccb09f1e55, https://usegalaxy.org/published/workflow?id=db08887d7d6dc147, https://usegalaxy.org/published/workflow?id=e2ab3d3219a648a7, https://usegalaxy.org/published/workflow?id=e75bd7d392e65209, https://usegalaxy.org/published/workflow?id=f4d623a19b32370e, https://workflowhub.eu/workflows/100?version=1, https://workflowhub.eu/workflows/102?version=1, https://workflowhub.eu/workflows/103?version=1, https://workflowhub.eu/workflows/1043?version=12, https://workflowhub.eu/workflows/1189?version=2, https://workflowhub.eu/workflows/1602?version=2, https://workflowhub.eu/workflows/1767?version=1, https://workflowhub.eu/workflows/407?version=1, https://workflowhub.eu/workflows/470?version=1</td>
      <td>assembly/mrsa-illumina</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>smudgeplot</td>
      <td>smudgeplot</td>
      <td>Inference of ploidy and heterozygosity structure using whole genome sequencing</td>
      <td>2022-06-30</td>
      <td>https://github.com/KamilSJaron/smudgeplot</td>
      <td>0.2.5</td>
      <td>smudgeplot</td>
      <td>0.5.3</td>
      <td>To update</td>
      <td>Assembly</td>
      <td>Sequence trimming, Genotyping, k-mer counting</td>
      <td>Sequence trimming, Genotyping, k-mer counting</td>
      <td>Sequence assembly, Genetic variation, Mathematics</td>
      <td>Sequence assembly, Genetic variation, Mathematics</td>
      <td>galaxy-australia</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/smudgeplot</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/smudgeplot</td>
      <td>smudgeplots</td>
      <td>Smudgeplots</td>
      <td>Reference-free profiling of polyploid genomes | Inference of ploidy and heterozygosity structure using whole genome sequencing data | Smudgeplots are computed from raw or even better from trimmed reads and show the haplotype structure using heterozygous kmer pairs. For example: | This tool extracts heterozygous kmer pairs from kmer dump files and performs gymnastics with them. We are able to disentangle genome structure by comparing the sum of kmer pair coverages (CovA + CovB) to their relative coverage (CovA / (CovA + CovB)). Such an approach also allows us to analyze obscure genomes with duplications, various ploidy levels, etc | GenomeScope 2.0 and Smudgeplots: Reference-free profiling of polyploid genomes Timothy Rhyker Ranallo-Benavidez, Kamil S. Jaron, Michael C. Schatz bioRxiv 747568; doi: https://doi.org/10.1101/747568</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>146</td>
      <td>146</td>
      <td>805</td>
      <td>805</td>
      <td>90</td>
      <td>90</td>
      <td>242</td>
      <td>242</td>
      <td>78</td>
      <td>78</td>
      <td>1315</td>
      <td>1315</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2362</td>
      <td>2362</td>
      <td>314</td>
      <td>314</td>
      <td>https://usegalaxy.eu/published/workflow?id=bc32937d505135ab, https://workflowhub.eu/workflows/1103?version=3, https://workflowhub.eu/workflows/1104?version=2, https://workflowhub.eu/workflows/1641?version=1, https://workflowhub.eu/workflows/603?version=2, https://workflowhub.eu/workflows/698?version=1</td>
      <td>assembly/ERGA-post-assembly-QC</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>spades</td>
      <td>spades_biosyntheticspades, spades_coronaspades, spades_metaplasmidspades, metaspades, spades_metaviralspades, spades_plasmidspades, rnaspades, spades_rnaviralspades, spades</td>
      <td>SPAdes is an assembly toolkit containing various assembly pipelines. It implements the following 4 stages: assembly graph construction, k-bimer adjustment, construction of paired assembly graph and contig construction.</td>
      <td>2016-12-21</td>
      <td>https://github.com/ablab/spades</td>
      <td>4.2.0</td>
      <td>spades</td>
      <td>4.2.0</td>
      <td>Up-to-date</td>
      <td>Assembly, RNA, Metagenomics</td>
      <td>Genome assembly</td>
      <td>Genome assembly</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/spades</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/spades</td>
      <td>spades</td>
      <td>SPAdes</td>
      <td>St. Petersburg genome assembler  is intended for both standard isolates and single-cell MDA bacteria assemblies. SPAdes 3.9 works with Illumina or IonTorrent reads and is capable of providing hybrid assemblies using PacBio, Oxford Nanopore and Sanger reads. Additional contigs can be provided and can be used as long reads.</td>
      <td></td>
      <td>9</td>
      <td>9</td>
      <td>9</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>9</td>
      <td>3</td>
      <td>0</td>
      <td>8250</td>
      <td>8882</td>
      <td>104429</td>
      <td>116176</td>
      <td>14023</td>
      <td>14731</td>
      <td>147425</td>
      <td>154112</td>
      <td>5211</td>
      <td>6722</td>
      <td>62902</td>
      <td>76811</td>
      <td>373</td>
      <td>380</td>
      <td>2677</td>
      <td>2762</td>
      <td>349861</td>
      <td>317433</td>
      <td>30715</td>
      <td>27857</td>
      <td>https://usegalaxy.eu/published/workflow?id=007ce6405d196df3, https://usegalaxy.eu/published/workflow?id=02631612a43a4d9b, https://usegalaxy.eu/published/workflow?id=1860e02bcb894a64, https://usegalaxy.eu/published/workflow?id=37da7b42cb063a3a, https://usegalaxy.eu/published/workflow?id=49c46d7653d6c349, https://usegalaxy.eu/published/workflow?id=579ff83177c5ee09, https://usegalaxy.eu/published/workflow?id=62c47e3c9d4f3aa3, https://usegalaxy.eu/published/workflow?id=7b755497d168cb13, https://usegalaxy.eu/published/workflow?id=7ecd92cb382d8168, https://usegalaxy.eu/published/workflow?id=87231a6a7938cb95, https://usegalaxy.eu/published/workflow?id=8c7958cb05fba814, https://usegalaxy.eu/published/workflow?id=9382f9bbbdddb5ad, https://usegalaxy.eu/published/workflow?id=93d1035a307fe63d, https://usegalaxy.eu/published/workflow?id=98bfad0c684011e9, https://usegalaxy.eu/published/workflow?id=9a97c6eb4fe015e7, https://usegalaxy.eu/published/workflow?id=a8417c63e9e22afb, https://usegalaxy.eu/published/workflow?id=a847773b563b2328, https://usegalaxy.eu/published/workflow?id=bebcbd6f0ffa55c5, https://usegalaxy.eu/published/workflow?id=c702c76cadebac30, https://usegalaxy.eu/published/workflow?id=d2f75f70c33394ee, https://usegalaxy.eu/published/workflow?id=d4a95588e6e50893, https://usegalaxy.eu/published/workflow?id=db10c092fad70d69, https://usegalaxy.eu/published/workflow?id=ef83f7fd2176a244, https://usegalaxy.fr/published/workflow?id=5bb882218afda132, https://usegalaxy.org.au/published/workflow?id=05f3abecd4ddd4a1, https://usegalaxy.org.au/published/workflow?id=38d992baf9a2a2bc, https://usegalaxy.org.au/published/workflow?id=39c724282ee008e3, https://usegalaxy.org.au/published/workflow?id=813a4938397cf6ee, https://usegalaxy.org.au/published/workflow?id=87cdeb05b1ebeab0, https://usegalaxy.org.au/published/workflow?id=8abcfe166f1a6cd8, https://usegalaxy.org.au/published/workflow?id=8f7d5ebead46a6d0, https://usegalaxy.org.au/published/workflow?id=914330feb75ea9ab, https://usegalaxy.org.au/published/workflow?id=fa76b66785ce38b8, https://usegalaxy.org/published/workflow?id=15cca0e249f40aea, https://usegalaxy.org/published/workflow?id=18899840e49d89c3, https://usegalaxy.org/published/workflow?id=33d90e718ce500ef, https://usegalaxy.org/published/workflow?id=5039ae2c81d0aba8, https://usegalaxy.org/published/workflow?id=71f8a4090877f171, https://usegalaxy.org/published/workflow?id=7251c6e0dad8f927, https://usegalaxy.org/published/workflow?id=87b3305ea433b4e2, https://usegalaxy.org/published/workflow?id=8a350e0d96e937a5, https://usegalaxy.org/published/workflow?id=8d229c0f58a07ebe, https://usegalaxy.org/published/workflow?id=9ba6da7dae465d3e, https://usegalaxy.org/published/workflow?id=9c0fbd9f51a9857f, https://usegalaxy.org/published/workflow?id=ae81cc0e8263d03e, https://usegalaxy.org/published/workflow?id=e0c33057ac2670ba, https://usegalaxy.org/published/workflow?id=fb01e1a49793c8be, https://workflowhub.eu/workflows/1390?version=2, https://workflowhub.eu/workflows/1626?version=1, https://workflowhub.eu/workflows/1634?version=1, https://workflowhub.eu/workflows/434?version=1, https://workflowhub.eu/workflows/52?version=1, https://workflowhub.eu/workflows/68?version=1</td>
      <td>assembly/debruijn-graph-assembly, assembly/metagenomics-assembly, microbiome/metagenomics-assembly</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>spaln</td>
      <td>list_spaln_tables, spaln</td>
      <td>Spaln (space-efficient spliced alignment) maps and aligns a set of cDNA or protein sequences onto a whole genomic sequence.</td>
      <td>2019-01-11</td>
      <td>https://github.com/ogotoh/spaln</td>
      <td>2.4.9</td>
      <td>python</td>
      <td></td>
      <td>To update</td>
      <td>Sequence Analysis, Genome annotation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/spaln</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/spaln</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>110</td>
      <td>132</td>
      <td>399</td>
      <td>612</td>
      <td>267</td>
      <td>318</td>
      <td>4249</td>
      <td>4506</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5118</td>
      <td>4648</td>
      <td>450</td>
      <td>377</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>spocc_occ</td>
      <td>spocc_occ</td>
      <td>Get species occurences data</td>
      <td>2019-05-23</td>
      <td>https://cran.r-project.org/web/packages/spocc/index.html</td>
      <td>1.2.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>Ecology</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/spocc</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/spocc</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>123</td>
      <td>146</td>
      <td>706</td>
      <td>976</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>12</td>
      <td>12</td>
      <td>28</td>
      <td>28</td>
      <td>1004</td>
      <td>734</td>
      <td>158</td>
      <td>135</td>
      <td>https://usegalaxy.eu/published/workflow?id=3b0803542deff026, https://usegalaxy.eu/published/workflow?id=b0b6c8756070650b, https://workflowhub.eu/workflows/1670?version=1, https://workflowhub.eu/workflows/1687?version=1, https://workflowhub.eu/workflows/404?version=1</td>
      <td>climate/earth_system, ecology/gbif_cleaning, ecology/species-distribution-modeling</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>srs_tools</td>
      <td>srs_diversity_maps, srs_global_indices, srs_process_data, srs_spectral_indices, srs_pca, srs_preprocess_s2, srs_metadata</td>
      <td>Compute biodiversity indicators for remote sensing data from Sentinel 2</td>
      <td>2022-11-16</td>
      <td></td>
      <td>0.0.1</td>
      <td>r-base</td>
      <td></td>
      <td>To update</td>
      <td>Ecology</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/Marie59/Sentinel_2A/srs_tools</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/srs_tools</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>4</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>274</td>
      <td>274</td>
      <td>5</td>
      <td>5</td>
      <td>48</td>
      <td>48</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>22</td>
      <td>22</td>
      <td>344</td>
      <td>344</td>
      <td>16</td>
      <td>16</td>
      <td>https://usegalaxy.eu/published/workflow?id=1d2cd0af01a30a46, https://usegalaxy.eu/published/workflow?id=da88448ab15d167d, https://workflowhub.eu/workflows/1669?version=2, https://workflowhub.eu/workflows/657?version=1</td>
      <td>ecology/remote-sensing</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>stoceps</td>
      <td>stoceps_filteringsp, stoceps_glm, stoceps_glm_group, stoceps_maketablecarrer, stoceps_trend_indic</td>
      <td>Tools to analyse STOC data.</td>
      <td>2019-09-26</td>
      <td></td>
      <td>0.0.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>Ecology</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/Alanamosse/Galaxy-E/tree/stoctool/tools/stoc</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/stoc</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>24</td>
      <td>26</td>
      <td>180</td>
      <td>354</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>354</td>
      <td>180</td>
      <td>26</td>
      <td>24</td>
      <td>https://usegalaxy.eu/published/workflow?id=01274dfdbdfcaa88</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>taxonomy_filter_refseq</td>
      <td>taxonomy_filter_refseq</td>
      <td>Filter RefSeq by taxonomy</td>
      <td>2019-01-13</td>
      <td>https://github.com/pvanheus/ncbitaxonomy</td>
      <td>0.3.0</td>
      <td>rust-ncbitaxonomy</td>
      <td>1.0.7</td>
      <td>To update</td>
      <td>Sequence Analysis, Genome annotation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/taxonomy_filter_refseq</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/taxonomy_filter_refseq</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>taxonomy_krona_chart</td>
      <td>taxonomy_krona_chart</td>
      <td>Krona pie chart from taxonomic profile</td>
      <td>2015-08-06</td>
      <td>http://sourceforge.net/projects/krona/</td>
      <td>2.7.1+galaxy0</td>
      <td>krona</td>
      <td>2.8.1</td>
      <td>To update</td>
      <td>Assembly</td>
      <td>Visualisation</td>
      <td>Visualisation</td>
      <td>Metagenomics</td>
      <td>Metagenomics</td>
      <td>crs4</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/taxonomy_krona_chart</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/taxonomy_krona_chart</td>
      <td>krona</td>
      <td>Krona</td>
      <td>Krona creates interactive HTML5 charts of hierarchical data (such as taxonomic abundance in a metagenome).</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>7406</td>
      <td>7874</td>
      <td>48617</td>
      <td>51653</td>
      <td>6438</td>
      <td>7659</td>
      <td>33268</td>
      <td>41191</td>
      <td>2005</td>
      <td>2042</td>
      <td>11003</td>
      <td>11192</td>
      <td>162</td>
      <td>163</td>
      <td>565</td>
      <td>586</td>
      <td>104622</td>
      <td>93453</td>
      <td>17738</td>
      <td>16011</td>
      <td>https://usegalaxy.eu/published/workflow?id=007ce6405d196df3, https://usegalaxy.eu/published/workflow?id=0590554db9794d13, https://usegalaxy.eu/published/workflow?id=05f5f040337786bf, https://usegalaxy.eu/published/workflow?id=08e25b733cd796e6, https://usegalaxy.eu/published/workflow?id=08eb5e55109c4d9c, https://usegalaxy.eu/published/workflow?id=096b75501c8e0888, https://usegalaxy.eu/published/workflow?id=0bafeb17c21ebc9c, https://usegalaxy.eu/published/workflow?id=10101558b211a782, https://usegalaxy.eu/published/workflow?id=14714876dda7186d, https://usegalaxy.eu/published/workflow?id=181e751be8dd9abc, https://usegalaxy.eu/published/workflow?id=19de5cbae7f48103, https://usegalaxy.eu/published/workflow?id=2cf09adb1ed6c3ec, https://usegalaxy.eu/published/workflow?id=309fc4fc2e0da89c, https://usegalaxy.eu/published/workflow?id=39d36a7ce0ef268d, https://usegalaxy.eu/published/workflow?id=437704898229dfb6, https://usegalaxy.eu/published/workflow?id=44e16f0782f54111, https://usegalaxy.eu/published/workflow?id=4a764275d4bf9f31, https://usegalaxy.eu/published/workflow?id=4d32b1b3ac0a7161, https://usegalaxy.eu/published/workflow?id=4f0daf8e9dd54127, https://usegalaxy.eu/published/workflow?id=4f21205e71cabd37, https://usegalaxy.eu/published/workflow?id=57cfe5062c8e01bf, https://usegalaxy.eu/published/workflow?id=5f0faac42b0a5062, https://usegalaxy.eu/published/workflow?id=628062a8cf93261f, https://usegalaxy.eu/published/workflow?id=7491883694fff308, https://usegalaxy.eu/published/workflow?id=79b4ae4403772098, https://usegalaxy.eu/published/workflow?id=7ab70660e6235cf0, https://usegalaxy.eu/published/workflow?id=86465932556bc96c, https://usegalaxy.eu/published/workflow?id=87cc0cd143a12c41, https://usegalaxy.eu/published/workflow?id=8deb08d4f228e198, https://usegalaxy.eu/published/workflow?id=8ee3b635c1c380cd, https://usegalaxy.eu/published/workflow?id=9472be79da012999, https://usegalaxy.eu/published/workflow?id=94c3d329c31c712f, https://usegalaxy.eu/published/workflow?id=994e5ff1df7d6c9e, https://usegalaxy.eu/published/workflow?id=9cbba5de2cbeeb0f, https://usegalaxy.eu/published/workflow?id=a337bfb2eb1e583f, https://usegalaxy.eu/published/workflow?id=a403c661bf1bc4fa, https://usegalaxy.eu/published/workflow?id=a459c311f461adce, https://usegalaxy.eu/published/workflow?id=a84bac86ec183d46, https://usegalaxy.eu/published/workflow?id=a8f829deb69b30ab, https://usegalaxy.eu/published/workflow?id=ad8b6d73c9654305, https://usegalaxy.eu/published/workflow?id=b14845359b702444, https://usegalaxy.eu/published/workflow?id=b2b46e71046b1999, https://usegalaxy.eu/published/workflow?id=b37800d990cceb83, https://usegalaxy.eu/published/workflow?id=b66e1d8171e6dac2, https://usegalaxy.eu/published/workflow?id=b86f24473a282ef9, https://usegalaxy.eu/published/workflow?id=b91a380ceaa0342b, https://usegalaxy.eu/published/workflow?id=bd4e3cdc7f29049b, https://usegalaxy.eu/published/workflow?id=c01ef8354b585a1f, https://usegalaxy.eu/published/workflow?id=c1e26abe61c69688, https://usegalaxy.eu/published/workflow?id=c2b7ec5aacfc6107, https://usegalaxy.eu/published/workflow?id=c3aa516e78ca4191, https://usegalaxy.eu/published/workflow?id=c67d2e75eeb4fd19, https://usegalaxy.eu/published/workflow?id=ca80039c4933aa0a, https://usegalaxy.eu/published/workflow?id=cb2df493f5e334f7, https://usegalaxy.eu/published/workflow?id=ce21961260fe4351, https://usegalaxy.eu/published/workflow?id=cfd37e9e307e5d3d, https://usegalaxy.eu/published/workflow?id=d05c6f0f3205fbd2, https://usegalaxy.eu/published/workflow?id=d0ae255151450dad, https://usegalaxy.eu/published/workflow?id=d88145e7475e0b7b, https://usegalaxy.eu/published/workflow?id=dadca1020f586338, https://usegalaxy.eu/published/workflow?id=dfc2c53ac3103eff, https://usegalaxy.eu/published/workflow?id=ed0ad728cf21702b, https://usegalaxy.eu/published/workflow?id=ee0005a5487653ba, https://usegalaxy.eu/published/workflow?id=ef6556d65ce5d72c, https://usegalaxy.eu/published/workflow?id=f21d6a40a10a8ae8, https://usegalaxy.eu/published/workflow?id=fb806b88850f3427, https://usegalaxy.eu/published/workflow?id=fdca6c9d394986bf, https://usegalaxy.eu/published/workflow?id=fde21b5f5c169114, https://usegalaxy.fr/published/workflow?id=09e8293456ee6d99, https://usegalaxy.fr/published/workflow?id=c4f6062e51d0f45e, https://usegalaxy.org.au/published/workflow?id=131636a795bac485, https://usegalaxy.org.au/published/workflow?id=2717e90438c7fb8c, https://usegalaxy.org.au/published/workflow?id=4c7e0703aace6aa3, https://usegalaxy.org.au/published/workflow?id=6b7e565258a9da2a, https://usegalaxy.org.au/published/workflow?id=a335b9c9103b57ad, https://usegalaxy.org.au/published/workflow?id=d9ba165e6ae55417, https://usegalaxy.org.au/published/workflow?id=ed3e48a73b5aaa56, https://usegalaxy.org/published/workflow?id=21161075002d6778, https://usegalaxy.org/published/workflow?id=3011f5dbf884c9bc, https://usegalaxy.org/published/workflow?id=45f959d5bf505b15, https://usegalaxy.org/published/workflow?id=4b5f6b5d4f36e38b, https://usegalaxy.org/published/workflow?id=5b74b780d3955eb3, https://usegalaxy.org/published/workflow?id=60f0d0d5d6e93941, https://usegalaxy.org/published/workflow?id=6f090b2c259847d2, https://usegalaxy.org/published/workflow?id=7321c6ad5425d347, https://usegalaxy.org/published/workflow?id=775335812a6b7a61, https://usegalaxy.org/published/workflow?id=8151be18aab3a3c6, https://usegalaxy.org/published/workflow?id=8f5904693b5f74f4, https://usegalaxy.org/published/workflow?id=9ba6da7dae465d3e, https://usegalaxy.org/published/workflow?id=a63927dd19759714, https://usegalaxy.org/published/workflow?id=b68ec42db8953569, https://usegalaxy.org/published/workflow?id=b7817a12dbee583c, https://usegalaxy.org/published/workflow?id=bc01b8b203ecc754, https://usegalaxy.org/published/workflow?id=c3ea8fa3bb202dc6, https://usegalaxy.org/published/workflow?id=cade0c5c11978323, https://usegalaxy.org/published/workflow?id=d5436cee11a34de9, https://usegalaxy.org/published/workflow?id=d653eaefcf6b4697, https://usegalaxy.org/published/workflow?id=e12324f5e3b930fd, https://workflowhub.eu/workflows/100?version=1, https://workflowhub.eu/workflows/101?version=1, https://workflowhub.eu/workflows/1059?version=1, https://workflowhub.eu/workflows/1199?version=2, https://workflowhub.eu/workflows/124?version=1, https://workflowhub.eu/workflows/1270?version=2, https://workflowhub.eu/workflows/1273?version=2, https://workflowhub.eu/workflows/1439?version=1, https://workflowhub.eu/workflows/1451?version=2, https://workflowhub.eu/workflows/1465?version=1, https://workflowhub.eu/workflows/1466?version=2, https://workflowhub.eu/workflows/1470?version=2, https://workflowhub.eu/workflows/1472?version=1, https://workflowhub.eu/workflows/1473?version=1, https://workflowhub.eu/workflows/1483?version=1, https://workflowhub.eu/workflows/1842?version=1, https://workflowhub.eu/workflows/1856?version=1, https://workflowhub.eu/workflows/2024?version=2, https://workflowhub.eu/workflows/53?version=1, https://workflowhub.eu/workflows/624?version=1</td>
      <td>microbiome/beer-data-analysis, microbiome/general-tutorial, microbiome/metatranscriptomics, microbiome/metatranscriptomics-short, microbiome/mgnify-amplicon, microbiome/mothur-miseq-sop, microbiome/nanopore-16S-metagenomics, microbiome/pathogen-detection-from-nanopore-foodborne-data, microbiome/taxonomic-profiling</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>terrabyte_ogc_api_processes</td>
      <td>terrabyte_ogc_api_processes</td>
      <td>This tool is a wrapper for OGC API Processes (OTB) coming from terrabyte (https://docs.terrabyte.lrz.de/).</td>
      <td>2024-11-11</td>
      <td>https://github.com/AquaINFRA/galaxy</td>
      <td>0.1.0</td>
      <td>r-base</td>
      <td></td>
      <td>To update</td>
      <td>Ecology</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/AquaINFRA/tools-ecology/tree/terrabyte_ogc_api_process</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/terrabyte_ogc_api_processes</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>12</td>
      <td>12</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>12</td>
      <td>12</td>
      <td>2</td>
      <td>2</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>trimns</td>
      <td>trimns</td>
      <td>TrimNs is used to trim and remove fake cut sites from bionano hybrid scaffold data in the VGP pipeline</td>
      <td>2021-01-29</td>
      <td>https://github.com/VGP/vgp-assembly/tree/master/pipeline/bionano/trimNs</td>
      <td>0.1.0</td>
      <td>trimns_vgp</td>
      <td>1.0</td>
      <td>To update</td>
      <td>Assembly</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/TrimNs</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/TrimNs</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>25</td>
      <td>25</td>
      <td>49</td>
      <td>49</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>49</td>
      <td>49</td>
      <td>25</td>
      <td>25</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>trycycler</td>
      <td>trycycler_cluster, trycycler_consensus, trycycler_partition, trycycler_reconcile_msa, trycycler_subsample</td>
      <td>Trycycler toolkit wrappers</td>
      <td>2021-02-11</td>
      <td>https://github.com/rrwick/Trycycler</td>
      <td>0.5.5</td>
      <td>trycycler</td>
      <td>0.5.6</td>
      <td>To update</td>
      <td>Assembly</td>
      <td>Genome assembly, Sequence trimming, Scaffold gap completion, Sequence assembly validation</td>
      <td>Genome assembly, Sequence trimming, Scaffold gap completion, Sequence assembly validation</td>
      <td>Sequence assembly, DNA, Human biology, Whole genome sequencing, Genomics</td>
      <td>Sequence assembly, DNA, Human biology, Whole genome sequencing, Genomics</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/trycycler</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/trycycler</td>
      <td>trycycler</td>
      <td>Trycycler</td>
      <td>Trycycler: consensus long-read assemblies for bacterial genomes</td>
      <td></td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>122</td>
      <td>122</td>
      <td>15743</td>
      <td>15743</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>57</td>
      <td>57</td>
      <td>6233</td>
      <td>6233</td>
      <td>3</td>
      <td>3</td>
      <td>27</td>
      <td>27</td>
      <td>22003</td>
      <td>22003</td>
      <td>182</td>
      <td>182</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>tsebra</td>
      <td>tsebra</td>
      <td>This tool has been developed to combine BRAKER predictions.</td>
      <td>2023-10-19</td>
      <td>https://github.com/Gaius-Augustus/TSEBRA</td>
      <td>1.1.2.5</td>
      <td>tsebra</td>
      <td>1.1.2.5</td>
      <td>Up-to-date</td>
      <td>Genome annotation</td>
      <td>Homology-based gene prediction, Alternative splicing prediction</td>
      <td>Homology-based gene prediction, Alternative splicing prediction</td>
      <td>Gene expression, RNA-Seq, Gene transcripts, Model organisms</td>
      <td>Gene expression, RNA-Seq, Gene transcripts, Model organisms</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/tsebra</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/tsebra</td>
      <td>tsebra</td>
      <td>TSEBRA</td>
      <td>TSEBRA is a combiner tool that selects transcripts from gene predictions based on the support by extrisic evidence in form of introns and start/stop codons. It was developed to combine BRAKER1 and BRAKER2 predicitons to increase their accuracies.</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>16</td>
      <td>16</td>
      <td>51</td>
      <td>51</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>51</td>
      <td>51</td>
      <td>16</td>
      <td>16</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>unicycler</td>
      <td>unicycler</td>
      <td>Unicycler is a hybrid assembly pipeline for bacterial genomes.</td>
      <td>2017-05-08</td>
      <td>https://github.com/rrwick/Unicycler</td>
      <td>0.5.1</td>
      <td>unicycler</td>
      <td>0.5.1</td>
      <td>Up-to-date</td>
      <td>Assembly</td>
      <td>Genome assembly, Aggregation</td>
      <td>Genome assembly, Aggregation</td>
      <td>Microbiology, Genomics, Sequencing, Sequence assembly</td>
      <td>Microbiology, Genomics, Sequencing, Sequence assembly</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/unicycler</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/unicycler</td>
      <td>unicycler</td>
      <td>Unicycler</td>
      <td>A tool for assembling bacterial genomes from a combination of short (2nd generation) and long (3rd generation) sequencing reads.</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>4697</td>
      <td>5202</td>
      <td>83521</td>
      <td>96874</td>
      <td>5770</td>
      <td>8769</td>
      <td>65067</td>
      <td>104850</td>
      <td>2392</td>
      <td>2613</td>
      <td>22717</td>
      <td>26831</td>
      <td>272</td>
      <td>276</td>
      <td>2160</td>
      <td>2183</td>
      <td>230738</td>
      <td>173465</td>
      <td>16860</td>
      <td>13131</td>
      <td>https://usegalaxy.eu/published/workflow?id=165089132248a0b0, https://usegalaxy.eu/published/workflow?id=1ea8bfbe6cb04531, https://usegalaxy.eu/published/workflow?id=249c115413f17f28, https://usegalaxy.eu/published/workflow?id=24eaa682897379c8, https://usegalaxy.eu/published/workflow?id=2e0075791f7fa384, https://usegalaxy.eu/published/workflow?id=346b46e6316c6950, https://usegalaxy.eu/published/workflow?id=374797e58377c40f, https://usegalaxy.eu/published/workflow?id=39837ca09374598b, https://usegalaxy.eu/published/workflow?id=44e53ea319e07daa, https://usegalaxy.eu/published/workflow?id=69be516a48761365, https://usegalaxy.eu/published/workflow?id=7b755497d168cb13, https://usegalaxy.eu/published/workflow?id=842e0ea7b1eb56af, https://usegalaxy.eu/published/workflow?id=8b8e89b9b8801f2f, https://usegalaxy.eu/published/workflow?id=8f3ce55d44d9f369, https://usegalaxy.eu/published/workflow?id=9219557de92e0203, https://usegalaxy.eu/published/workflow?id=b2b46e71046b1999, https://usegalaxy.eu/published/workflow?id=b2dfa74f55a73409, https://usegalaxy.eu/published/workflow?id=c48807045e258dd8, https://usegalaxy.eu/published/workflow?id=c702c76cadebac30, https://usegalaxy.eu/published/workflow?id=dc7107896a3f2964, https://usegalaxy.eu/published/workflow?id=e8eb80ee09159bac, https://usegalaxy.eu/published/workflow?id=ea485dbabc76488d, https://usegalaxy.eu/published/workflow?id=ecf48fa50af560a0, https://usegalaxy.eu/published/workflow?id=fe43cf59e48bf556, https://usegalaxy.fr/published/workflow?id=5bb882218afda132, https://usegalaxy.org.au/published/workflow?id=38d992baf9a2a2bc, https://usegalaxy.org.au/published/workflow?id=8a40059343cf4b87, https://usegalaxy.org.au/published/workflow?id=8f7d5ebead46a6d0, https://usegalaxy.org.au/published/workflow?id=9c60b83feaa22415, https://usegalaxy.org.au/published/workflow?id=b59fada651043735, https://usegalaxy.org.au/published/workflow?id=b7aa22291322e505, https://usegalaxy.org.au/published/workflow?id=c9b7de574fb2f2d5, https://usegalaxy.org.au/published/workflow?id=d9571b3dfef34e5e, https://usegalaxy.org.au/published/workflow?id=f01be6a7685ac476, https://usegalaxy.org.au/published/workflow?id=fc3c83b11f527977, https://usegalaxy.org/published/workflow?id=5039ae2c81d0aba8, https://usegalaxy.org/published/workflow?id=69514a75e4c9af08, https://usegalaxy.org/published/workflow?id=8111bb9bb26bb665, https://usegalaxy.org/published/workflow?id=a67afbea4b2eb26b, https://usegalaxy.org/published/workflow?id=b43b72dcf5f7694d, https://usegalaxy.org/published/workflow?id=bc72dcbca89ca35d, https://usegalaxy.org/published/workflow?id=c65bec6c332bf740, https://usegalaxy.org/published/workflow?id=db0671862a57968d, https://workflowhub.eu/workflows/1477?version=1, https://workflowhub.eu/workflows/1598?version=1, https://workflowhub.eu/workflows/1612?version=1, https://workflowhub.eu/workflows/1616?version=1, https://workflowhub.eu/workflows/2011?version=1, https://workflowhub.eu/workflows/38?version=1, https://workflowhub.eu/workflows/406?version=1, https://workflowhub.eu/workflows/52?version=1, https://workflowhub.eu/workflows/68?version=1</td>
      <td>assembly/assembly-with-preprocessing, assembly/hybrid_denovo_assembly, assembly/unicycler-assembly, microbiome/plasmid-metagenomics-nanopore</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>velvet</td>
      <td>velvetg, velveth</td>
      <td>de novo genomic assembler specially designed for short read sequencing technologies</td>
      <td>2020-06-09</td>
      <td>https://www.ebi.ac.uk/~zerbino/velvet/</td>
      <td></td>
      <td>velvet</td>
      <td>1.2.10</td>
      <td>To update</td>
      <td>Assembly</td>
      <td>Formatting, De-novo assembly</td>
      <td>Formatting, De-novo assembly</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/velvet</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/velvet</td>
      <td>velvet</td>
      <td>Velvet</td>
      <td>A de novo genomic assembler specially designed for short read sequencing technologies, such as Solexa or 454 or SOLiD.</td>
      <td></td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>2567</td>
      <td>2888</td>
      <td>27621</td>
      <td>32158</td>
      <td>4951</td>
      <td>5263</td>
      <td>52855</td>
      <td>57273</td>
      <td>2097</td>
      <td>2221</td>
      <td>30746</td>
      <td>33121</td>
      <td>109</td>
      <td>109</td>
      <td>1097</td>
      <td>1097</td>
      <td>123649</td>
      <td>112319</td>
      <td>10481</td>
      <td>9724</td>
      <td>https://usegalaxy.eu/published/workflow?id=01dc0619e050dc91, https://usegalaxy.eu/published/workflow?id=56edd22d14392691, https://usegalaxy.eu/published/workflow?id=6d87481be47902e1, https://usegalaxy.eu/published/workflow?id=842e0ea7b1eb56af, https://usegalaxy.eu/published/workflow?id=b7c2440621968526, https://usegalaxy.eu/published/workflow?id=ef83f7fd2176a244, https://usegalaxy.org.au/published/workflow?id=30430b75d05f54eb, https://usegalaxy.org.au/published/workflow?id=533dbb3242ee4255, https://usegalaxy.org.au/published/workflow?id=7fa13ccbf044edc5, https://usegalaxy.org.au/published/workflow?id=83e3ceceb1a92493, https://usegalaxy.org.au/published/workflow?id=874492eb776ba813, https://usegalaxy.org.au/published/workflow?id=8b6739a000780d99, https://usegalaxy.org.au/published/workflow?id=94c7fdcbbc7b428a, https://usegalaxy.org.au/published/workflow?id=9b75924c483b5f6b, https://usegalaxy.org.au/published/workflow?id=aee5b5645c51cac3, https://usegalaxy.org.au/published/workflow?id=dccc8fb49d61ac33, https://usegalaxy.org.au/published/workflow?id=ee7a1615f187391d, https://usegalaxy.org.au/published/workflow?id=f25a05070bf97ded, https://usegalaxy.org.au/published/workflow?id=fb0986a952c35e0a, https://usegalaxy.org/published/workflow?id=109092d8dbda4398, https://usegalaxy.org/published/workflow?id=1c7af480caf150bd, https://usegalaxy.org/published/workflow?id=3cf46c167ccb374f, https://usegalaxy.org/published/workflow?id=4e1ae78405034ad3, https://usegalaxy.org/published/workflow?id=51c485af11d90700, https://usegalaxy.org/published/workflow?id=7a33bc2d0970b502, https://usegalaxy.org/published/workflow?id=c30f6ca171896908, https://usegalaxy.org/published/workflow?id=e2ab3d3219a648a7, https://usegalaxy.org/published/workflow?id=e745c45651d829b8, https://usegalaxy.org/published/workflow?id=f6bd33dedc7c6c12, https://usegalaxy.org/published/workflow?id=f926e70bee242785, https://workflowhub.eu/workflows/1622?version=1</td>
      <td>assembly/debruijn-graph-assembly, assembly/general-introduction</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>verkko</td>
      <td>verkko</td>
      <td>Telomere-to-telomere assembly pipeline</td>
      <td>2023-01-24</td>
      <td>https://github.com/marbl/verkko</td>
      <td>1.3.1</td>
      <td>verkko</td>
      <td>2.3</td>
      <td>To update</td>
      <td>Assembly</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/verkko</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/verkko</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>43</td>
      <td>43</td>
      <td>105</td>
      <td>105</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>13</td>
      <td>13</td>
      <td>136</td>
      <td>136</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>241</td>
      <td>241</td>
      <td>56</td>
      <td>56</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>vigiechiro</td>
      <td>vigiechiro_bilanenrichipf, vigiechiro_bilanenrichirp, vigiechiro_idcorrect_2ndlayer, vigiechiro_idvalid</td>
      <td>Tools created by the vigiechiro team to analyses and identify chiro sounds files.</td>
      <td>2019-03-13</td>
      <td>https://www.vigienature-ecole.fr/les-observatoires/le-protocole-vigie-chiro</td>
      <td>0.1.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>Ecology</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/vigiechiro</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/vigiechiro</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>258</td>
      <td>284</td>
      <td>14141</td>
      <td>17115</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>17115</td>
      <td>14141</td>
      <td>284</td>
      <td>258</td>
      <td>https://usegalaxy.eu/published/workflow?id=596b073ab7249aa2, https://usegalaxy.eu/published/workflow?id=c0888e0a0fb53ed3</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>yahs</td>
      <td>yahs</td>
      <td>Yet Another Hi-C scaffolding tool</td>
      <td>2022-06-28</td>
      <td>https://github.com/c-zhou/yahs</td>
      <td>1.2a.2</td>
      <td>yahs</td>
      <td>1.2.2</td>
      <td>To update</td>
      <td>Assembly</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/yahs</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/yahs</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>291</td>
      <td>291</td>
      <td>2516</td>
      <td>2516</td>
      <td>107</td>
      <td>107</td>
      <td>750</td>
      <td>750</td>
      <td>41</td>
      <td>41</td>
      <td>1062</td>
      <td>1062</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4328</td>
      <td>4328</td>
      <td>439</td>
      <td>439</td>
      <td>https://usegalaxy.eu/published/workflow?id=069d99391a69baf4, https://usegalaxy.eu/published/workflow?id=07c92110c3fffcf8, https://usegalaxy.eu/published/workflow?id=082bfd9b36e921e8, https://usegalaxy.eu/published/workflow?id=0c3ce9feb83f7920, https://usegalaxy.eu/published/workflow?id=0cf3c2114ab65858, https://usegalaxy.eu/published/workflow?id=3010ecad1a342331, https://usegalaxy.eu/published/workflow?id=31f8c7c46207432f, https://usegalaxy.eu/published/workflow?id=3b57fc81001667b5, https://usegalaxy.eu/published/workflow?id=44c92c04f1d14631, https://usegalaxy.eu/published/workflow?id=4949751217c0e211, https://usegalaxy.eu/published/workflow?id=5395e76fa828c343, https://usegalaxy.eu/published/workflow?id=652433160b04b232, https://usegalaxy.eu/published/workflow?id=792b24fc23bd3071, https://usegalaxy.eu/published/workflow?id=7b4a9aebb09b51ca, https://usegalaxy.eu/published/workflow?id=7e383fe34031a9d0, https://usegalaxy.eu/published/workflow?id=7e68cf8a67efa8e9, https://usegalaxy.eu/published/workflow?id=84236d781f7dd2ad, https://usegalaxy.eu/published/workflow?id=a2daeb488bee0152, https://usegalaxy.eu/published/workflow?id=a37d1e0db16cfb67, https://usegalaxy.eu/published/workflow?id=a53f78aa6866b811, https://usegalaxy.eu/published/workflow?id=b287bb779cf686ef, https://usegalaxy.eu/published/workflow?id=bc8ccd13a07bd9ec, https://usegalaxy.eu/published/workflow?id=c8f94a2e838fc0f1, https://usegalaxy.eu/published/workflow?id=cab2aa791a814d55, https://usegalaxy.eu/published/workflow?id=cf4638fbca7c460b, https://usegalaxy.eu/published/workflow?id=d649ec485c4b3994, https://usegalaxy.eu/published/workflow?id=f1e2577699125607, https://usegalaxy.eu/published/workflow?id=f63127675a07e00a, https://usegalaxy.eu/published/workflow?id=feeb1fa358ccf328, https://usegalaxy.org.au/published/workflow?id=0aa316f8c6cff444, https://usegalaxy.org.au/published/workflow?id=0dbb85f727302a72, https://usegalaxy.org.au/published/workflow?id=29f0c07c8a24b542, https://usegalaxy.org.au/published/workflow?id=3a64c1031340d860, https://usegalaxy.org.au/published/workflow?id=3b6e9f03b088bf01, https://usegalaxy.org.au/published/workflow?id=455e68f185fec41b, https://usegalaxy.org.au/published/workflow?id=466809eb3392f7fb, https://usegalaxy.org.au/published/workflow?id=4ea1049371de111c, https://usegalaxy.org.au/published/workflow?id=6c1c11a81a2ab1b8, https://usegalaxy.org.au/published/workflow?id=6de8899c40a90b98, https://usegalaxy.org.au/published/workflow?id=6ec5728eec9c26d7, https://usegalaxy.org.au/published/workflow?id=7028436ef03171cc, https://usegalaxy.org.au/published/workflow?id=77b2fcf8c805ad5d, https://usegalaxy.org.au/published/workflow?id=7a97e8ccc332c1e8, https://usegalaxy.org.au/published/workflow?id=868775ef66d174a5, https://usegalaxy.org.au/published/workflow?id=86eb0ea9dc230f04, https://usegalaxy.org.au/published/workflow?id=ae2a397837b232b1, https://usegalaxy.org.au/published/workflow?id=ae6a03240680712a, https://usegalaxy.org.au/published/workflow?id=b53f82aaf32fbaa9, https://usegalaxy.org.au/published/workflow?id=b9e89401313a0cca, https://usegalaxy.org.au/published/workflow?id=be35847680078c90, https://usegalaxy.org.au/published/workflow?id=cac0b8a09d59044b, https://usegalaxy.org.au/published/workflow?id=ea98be0f99aaceb9, https://usegalaxy.org.au/published/workflow?id=f38f610e0d95915f, https://usegalaxy.org.au/published/workflow?id=f53d8ade897b3f46, https://usegalaxy.org.au/published/workflow?id=fdec6e2fcd5e2cf5, https://usegalaxy.org/published/workflow?id=155839aeb6340f54, https://usegalaxy.org/published/workflow?id=1ea88ca80a7f0038, https://usegalaxy.org/published/workflow?id=23eeee48217654d4, https://usegalaxy.org/published/workflow?id=28dbbfb3d480f4b1, https://usegalaxy.org/published/workflow?id=28ddea356ae57da1, https://usegalaxy.org/published/workflow?id=2ff2961f044cac0f, https://usegalaxy.org/published/workflow?id=325783f21bc5fef6, https://usegalaxy.org/published/workflow?id=349f1bcdbc84045f, https://usegalaxy.org/published/workflow?id=3a92253f57e553bd, https://usegalaxy.org/published/workflow?id=3b0d42747f598f82, https://usegalaxy.org/published/workflow?id=3d805d7e0f34c727, https://usegalaxy.org/published/workflow?id=3db51822f04f4ff7, https://usegalaxy.org/published/workflow?id=49a07807fcea8158, https://usegalaxy.org/published/workflow?id=6341c384ceaf47ed, https://usegalaxy.org/published/workflow?id=74049914e4a6c791, https://usegalaxy.org/published/workflow?id=7b8ee08ba291d231, https://usegalaxy.org/published/workflow?id=7fd68212ce020e6a, https://usegalaxy.org/published/workflow?id=88327cb2e0fe3c25, https://usegalaxy.org/published/workflow?id=903e35b62946ed76, https://usegalaxy.org/published/workflow?id=91a4daecbb97d03e, https://usegalaxy.org/published/workflow?id=9f4c4038d1c519e2, https://usegalaxy.org/published/workflow?id=a4a6d6846c1692f9, https://usegalaxy.org/published/workflow?id=a8cd62e1924a64e1, https://usegalaxy.org/published/workflow?id=afa89bb0e8966753, https://usegalaxy.org/published/workflow?id=c03ada2070e9ca8a, https://usegalaxy.org/published/workflow?id=c95e9116e4b7982b, https://usegalaxy.org/published/workflow?id=f09033420c6267ab, https://usegalaxy.org/published/workflow?id=fa36d9b50e956917, https://usegalaxy.org/published/workflow?id=fb98ef955b3dbd4b, https://workflowhub.eu/workflows/1054?version=1, https://workflowhub.eu/workflows/1164?version=2, https://workflowhub.eu/workflows/608?version=1, https://workflowhub.eu/workflows/625?version=29, https://workflowhub.eu/workflows/702?version=1</td>
      <td>assembly/vgp_genome_assembly</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>zoo_project_ogc_api_processes</td>
      <td>zoo_project_ogc_api_processes</td>
      <td>This tool is a wrapper for OGC API Processes (OTB) coming from the Zoo Project (https://zoo-project.github.io/docs/intro.html) and was created using the OGC-API-Process2Galaxy tool (https://github.com/AquaINFRA/OGC-API-Process2Galaxy). Check the README in the repository for more information.</td>
      <td>2024-05-03</td>
      <td>https://github.com/AquaINFRA/galaxy</td>
      <td>0.1.0</td>
      <td>r-base</td>
      <td></td>
      <td>To update</td>
      <td>Ecology</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/ogc_api_processes_wrapper</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/zoo_project_ogc_api_processes</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>28</td>
      <td>28</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>28</td>
      <td>28</td>
      <td>5</td>
      <td>5</td>
      <td>https://usegalaxy.eu/published/workflow?id=e95bfab07fdec632</td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
  </tbody>
</table>

</div>