<script src="https://code.jquery.com/jquery-3.7.0.js"></script>
<script src="https://cdn.datatables.net/1.13.6/js/jquery.dataTables.min.js"></script>
<script src="https://cdn.datatables.net/searchbuilder/1.6.0/js/dataTables.searchBuilder.min.js"></script>
<script src="https://cdn.datatables.net/datetime/1.5.1/js/dataTables.dateTime.min.js"></script>

<script src="https://cdn.datatables.net/responsive/2.5.0/js/dataTables.responsive.min.js"></script>

<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/responsive/2.2.3/css/responsive.dataTables.min.css">

<link rel="stylesheet" href="https://cdn.datatables.net/1.13.6/css/jquery.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/searchbuilder/1.6.0/css/searchBuilder.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/datetime/1.5.1/css/dataTables.dateTime.min.css">

<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" />
<script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>

<style type="text/css">
  .tooltip {
    font-family: Georgia;
    font-size: 18px;
  }
  .tooltip .tooltip-inner {
    background-color: #00FFFF;
    color: #000000;
    min-width: 400px;
  }
  </style>

<script>
    $(document).ready(function() {
    $('#dataframe').DataTable( {
        "sScrollX": "100%",
        "dom": "Qlfrtip",

        responsive: {
            details: {
                type: 'column'
            }
        },
        "bAutoWidth": false,
        columnDefs: [ {
            className: 'control',
            orderable: false,
            targets:   [ 0 ],
            width: 100,
        },
        { type: 'natural', targets: 1 } // allow normal sorting
        ],

    });
});
</script>

<div class="display nowrap" style="width:80%" width="100%">

<table class="dataframe display nowrap" id="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th>Expand</th>
      <th>Galaxy wrapper id</th>
      <th>Galaxy tool ids</th>
      <th>Description</th>
      <th>bio.tool id</th>
      <th>bio.tool ids</th>
      <th>biii</th>
      <th>bio.tool name</th>
      <th>bio.tool description</th>
      <th>EDAM operation</th>
      <th>EDAM topic</th>
      <th>Status</th>
      <th>Source</th>
      <th>ToolShed categories</th>
      <th>ToolShed id</th>
      <th>Galaxy wrapper owner</th>
      <th>Galaxy wrapper source</th>
      <th>Galaxy wrapper parsed folder</th>
      <th>Galaxy wrapper version</th>
      <th>Conda id</th>
      <th>Conda version</th>
      <th>EDAM operation (no superclasses)</th>
      <th>EDAM topic (no superclasses)</th>
      <th>Available on UseGalaxy.org (Main)</th>
      <th>Available on UseGalaxy.org.au</th>
      <th>Available on UseGalaxy.eu</th>
      <th>Available on UseGalaxy.fr</th>
      <th>Tools available on UseGalaxy.org (Main)</th>
      <th>Tools available on UseGalaxy.org.au</th>
      <th>Tools available on UseGalaxy.eu</th>
      <th>Tools available on UseGalaxy.fr</th>
      <th>Tools available on APOSTL</th>
      <th>Tools available on BF2I-MAP</th>
      <th>Tools available on BioBix</th>
      <th>Tools available on CIRM-CFBP</th>
      <th>Tools available on Center for Phage Technology (CPT)</th>
      <th>Tools available on ChemFlow</th>
      <th>Tools available on Coloc-stats</th>
      <th>Tools available on CoralSNP</th>
      <th>Tools available on CropGalaxy</th>
      <th>Tools available on Dintor</th>
      <th>Tools available on FreeBioinfo</th>
      <th>Tools available on GASLINI</th>
      <th>Tools available on Galaxy@AuBi</th>
      <th>Tools available on Galaxy@Pasteur</th>
      <th>Tools available on GalaxyTrakr</th>
      <th>Tools available on Genomic Hyperbrowser</th>
      <th>Tools available on GigaGalaxy</th>
      <th>Tools available on HyPhy HIV NGS Tools</th>
      <th>Tools available on IPK Galaxy Blast Suite</th>
      <th>Tools available on ImmPort Galaxy</th>
      <th>Tools available on InteractoMIX</th>
      <th>Tools available on MISSISSIPPI</th>
      <th>Tools available on Mandoiu Lab</th>
      <th>Tools available on MiModD NacreousMap</th>
      <th>Tools available on Oqtans</th>
      <th>Tools available on Palfinder</th>
      <th>Tools available on PepSimili</th>
      <th>Tools available on PhagePromotor</th>
      <th>Tools available on UseGalaxy.be</th>
      <th>Tools available on UseGalaxy.cz</th>
      <th>Tools available on UseGalaxy.no</th>
      <th>Tools available on Viral Variant Visualizer (VVV)</th>
      <th>No. of tool users (2022-2023) (usegalaxy.eu)</th>
      <th>Total tool usage (usegalaxy.eu)</th>
      <th>Deprecated</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td></td>
      <td>EMLassemblyline</td>
      <td>eal_table_template, eal_templates, eml2eal, entities_template, geo_cov_template, makeeml, raster_template, taxo_cov_template, vector_template</td>
      <td>Tools using EML Assembly Line R package to generate EML metadata from template metadata files and vice versa</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/EDIorg/EMLassemblyline</td>
      <td>Ecology</td>
      <td>emlassemblyline</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/EMLassemblyline</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/EMLassemblyline</td>
      <td>0.1.1+galaxy0</td>
      <td>r-emlassemblyline</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>102</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Ecoregionalization_workflow</td>
      <td>ecoregion_brt_analysis, ecoregion_GeoNearestNeighbor, ecoregion_cluster_estimate, ecoregion_clara_cluster, ecoregion_eco_map, ecoregion_taxa_seeker</td>
      <td>Tools to compute ecoregionalization with BRT model predictions and clustering.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/PaulineSGN/Workflow_Galaxy</td>
      <td>Ecology</td>
      <td>ecoregionalization</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/Ecoregionalization_workflow</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/Ecoregionalization_workflow</td>
      <td>0.1.0+galaxy0</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>82</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Geom_mean_workflow</td>
      <td>Map_shp, Mean_geom, bar_plot</td>
      <td>Tools to compute The evolution of the total volume of very large trees, standing dead wood and dead wood on the ground on an area and the rate of devolution of the volume of wood favorable to biodiversity by large ecological regions (France).</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/PaulineSGN/Galaxy_tool_moyenne_geom</td>
      <td>Ecology</td>
      <td>Geometric means (Dead wood)</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/Geom_mean_workflow</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/Geom_mean_workflow</td>
      <td>0.1.0+galaxy0</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>PAMPA</td>
      <td>pampa_communitymetrics, pampa_presabs, pampa_glmcomm, pampa_glmsp, pampa_plotglm</td>
      <td>Tools to compute and analyse biodiversity metrics</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Ecology</td>
      <td>pampa</td>
      <td>ecology</td>
      <td>https://github.com/ColineRoyaux/PAMPA-Galaxy</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/PAMPA</td>
      <td>0.0.2</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>4</td>
      <td>0</td>
      <td>73</td>
      <td>1004</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>TrimNs</td>
      <td>trimns</td>
      <td>TrimNs is used to trim and remove fake cut sites from bionano hybrid scaffold data in the VGP pipeline</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/VGP/vgp-assembly/tree/master/pipeline/bionano/trimNs</td>
      <td>Assembly</td>
      <td>trimns</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/trimN</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/TrimNs</td>
      <td>0.1.0</td>
      <td>trimns_vgp</td>
      <td>1.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>42</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>abacas</td>
      <td>abacas</td>
      <td>Order and Orientate Contigs</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/abacas</td>
      <td>Assembly</td>
      <td>abacas</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/abacas</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/abacas</td>
      <td>1.1</td>
      <td>mummer</td>
      <td>3.23</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>abyss</td>
      <td>abyss-pe</td>
      <td>Assembly By Short Sequences - a de novo, parallel, paired-end sequence assembler</td>
      <td>abyss</td>
      <td>abyss</td>
      <td></td>
      <td>ABySS</td>
      <td>De novo genome sequence assembler using short reads.</td>
      <td>Genome assembly, De-novo assembly, Scaffolding</td>
      <td>Sequence assembly</td>
      <td>To update</td>
      <td>http://www.bcgsc.ca/platform/bioinfo/software/abyss</td>
      <td>Assembly</td>
      <td>abyss</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/abyss</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/abyss</td>
      <td>2.3.7</td>
      <td>abyss</td>
      <td>2.3.8</td>
      <td>Genome assembly, De-novo assembly, Scaffolding</td>
      <td>Sequence assembly</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>391</td>
      <td>4278</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>annotatemyids</td>
      <td>annotatemyids</td>
      <td>annotateMyIDs: get annotation for a set of IDs using the Bioconductor annotation packages</td>
      <td>annotatemyids</td>
      <td>annotatemyids</td>
      <td></td>
      <td>annotatemyids</td>
      <td>This tool can get annotation for a generic set of IDs, using the Bioconductor annotation data packages. Supported organisms are human, mouse, rat, fruit fly and zebrafish. The org.db packages that are used here are primarily based on mapping using Entrez Gene identifiers. More information on the annotation packages can be found at the Bioconductor website, for example, information on the human annotation package (org.Hs.eg.db) can be found here.</td>
      <td>Annotation</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/annotatemyids</td>
      <td>Genome annotation</td>
      <td>annotatemyids</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/annotatemyids</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/annotatemyids</td>
      <td>3.18.0</td>
      <td>bioconductor-org.hs.eg.db</td>
      <td>3.18.0</td>
      <td>Annotation</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1175</td>
      <td>25975</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>aquainfra_importer</td>
      <td>aquainfra_importer</td>
      <td>A data source tool for downloading datasets via the AquaINFRA Interaction Platform.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/AquaINFRA/galaxy</td>
      <td>Ecology</td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/AquaINFRA/tools-ecology/tree/aquainfra_importer</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/aquainfra_importer</td>
      <td>1.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>argnorm</td>
      <td>argnorm</td>
      <td>argNorm is a tool to normalize antibiotic resistance genes (ARGs) by mapping them to the antibiotic resistance ontology (ARO) created by the CARD database</td>
      <td>argnorm</td>
      <td>argnorm</td>
      <td></td>
      <td>argNorm</td>
      <td>argNorm is a tool to normalize antibiotic resistance genes (ARGs) by mapping them to the antibiotic resistance ontology (ARO) created by the CARD database.argNorm also enhances antibiotic resistance gene annotations by providing drug categorization of the drugs that antibiotic resistance genes confer resistance to.</td>
      <td>Gene functional annotation</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/BigDataBiology/argNorm</td>
      <td>Genome annotation</td>
      <td>argnorm</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/argnorm</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/argnorm</td>
      <td>0.6.0</td>
      <td>argnorm</td>
      <td>0.6.0</td>
      <td>Gene functional annotation</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>assembly-stats</td>
      <td>assembly_stats</td>
      <td>Assembly metric visualisations to facilitate rapid assessment and comparison of assembly quality.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/rjchallis/assembly-stats</td>
      <td>Assembly</td>
      <td>assembly_stats</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/assembly-stats</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/assembly-stats</td>
      <td>17.02</td>
      <td>rjchallis-assembly-stats</td>
      <td>17.02</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>assemblystats</td>
      <td>assemblystats</td>
      <td>Summarise an assembly (e.g. N50 metrics)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>Assembly</td>
      <td>assemblystats</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/assemblystats</td>
      <td>1.1.0</td>
      <td>perl-bioperl</td>
      <td>1.7.8</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>bam2mappingstats</td>
      <td>bam2mappingstats</td>
      <td>Generates mapping stats from a bam file.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>Assembly</td>
      <td>bam2mappingstats</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/bam2mappingstats</td>
      <td>1.1.0</td>
      <td>perl</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>baseline_calculator</td>
      <td>tt_baseline</td>
      <td>Toxicity prediction using QSAR models</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bernt-matthias/mb-galaxy-tools</td>
      <td>Ecology, Text Manipulation</td>
      <td>baseline_toxicity_calculator</td>
      <td>mbernt</td>
      <td>https://github.com/bernt-matthias/mb-galaxy-tools/tools/tox_tools/baseline_calculator</td>
      <td>https://github.com/Helmholtz-UFZ/galaxy-tools/tree/main/tools/tox_tools/baseline_calculator</td>
      <td>0.1.0+galaxy0</td>
      <td>pandas</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>bionano</td>
      <td>bionano_scaffold</td>
      <td>Bionano Solve is a set of tools for analyzing Bionano data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bionanogenomics.com/</td>
      <td>Assembly</td>
      <td>bionano</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/bionano</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/bionano</td>
      <td>3.7.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>112</td>
      <td>531</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>biotradis</td>
      <td>bacteria_tradis, tradis_essentiality, tradis_gene_insert_sites</td>
      <td>Bio-Tradis is a tool suite dedicated to essentiality analyses with TraDis data.</td>
      <td>biotradis</td>
      <td>biotradis</td>
      <td></td>
      <td>biotradis</td>
      <td>The Bio::TraDIS pipeline provides software utilities for the processing, mapping, and analysis of transposon insertion sequencing data. The pipeline was designed with the data from the TraDIS sequencing protocol in mind, but should work with a variety of transposon insertion sequencing protocols as long as they produce data in the expected format.</td>
      <td>Sequence analysis</td>
      <td>Mobile genetic elements, Workflows</td>
      <td>Up-to-date</td>
      <td>https://www.sanger.ac.uk/science/tools/bio-tradis</td>
      <td>Genome annotation</td>
      <td>biotradis</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/biotradis</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/biotradis</td>
      <td>1.4.5</td>
      <td>biotradis</td>
      <td>1.4.5</td>
      <td>Sequence analysis</td>
      <td>Mobile genetic elements, Workflows</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>biscot</td>
      <td>biscot</td>
      <td>Bionano scaffolding correction tool</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/institut-de-genomique/biscot</td>
      <td>Assembly</td>
      <td>biscot</td>
      <td>iuc</td>
      <td>https://github.com/bgruening/iuc/tree/master/tools/biscot</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/biscot</td>
      <td>2.3.3</td>
      <td>biscot</td>
      <td>2.3.3</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>3</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>blast_to_scaffold</td>
      <td>blast2scaffold</td>
      <td>Generate DNA scaffold from blastn or tblastx alignments of Contigs</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, Sequence Analysis, Assembly</td>
      <td>blast_to_scaffold</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/blast_to_scaffold</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/blast_to_scaffold</td>
      <td>1.1.0</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>blastparser_and_hits</td>
      <td>BlastParser_and_hits</td>
      <td>Parse blast outputs and compile hits</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Assembly, RNA</td>
      <td>blastparser_and_hits</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/blastparser_and_hits</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/blastparser_and_hits</td>
      <td>2.7.1</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>blastx_to_scaffold</td>
      <td>blastx2scaffold</td>
      <td>Generate DNA scaffold from blastx alignment of Contigs</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, Sequence Analysis, Assembly</td>
      <td>blastx_to_scaffold</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/blastx_to_scaffold</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/blastx_to_scaffold</td>
      <td>1.1.1</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>blobtoolkit</td>
      <td>blobtoolkit</td>
      <td>Identification and isolation non-target data in draft and publicly available genome assemblies.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://blobtoolkit.genomehubs.org/</td>
      <td>Sequence Analysis, Assembly</td>
      <td>blobtoolkit</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/blobtoolkit</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/blobtoolkit</td>
      <td>4.0.7</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>21</td>
      <td>685</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>braker</td>
      <td>braker</td>
      <td>BRAKER is a pipeline for fully automated prediction of protein coding gene structures with GeneMark-ES/ET and AUGUSTUS in novel eukaryotic genomes .</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/Gaius-Augustus/BRAKER</td>
      <td>Genome annotation</td>
      <td>braker</td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/braker</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/braker</td>
      <td>2.1.6</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>17</td>
      <td>109</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>braker3</td>
      <td>braker3</td>
      <td>BRAKER3 is a pipeline for fully automated prediction of protein coding gene structures with GeneMark-ES/ET and AUGUSTUS in novel eukaryotic genomes .</td>
      <td>braker3</td>
      <td>braker3</td>
      <td></td>
      <td>BRAKER3</td>
      <td>BRAKER3 is a pipeline for fully automated prediction of protein coding gene structures with GeneMark-ES/ET and AUGUSTUS in novel eukaryotic genomes</td>
      <td>Genome annotation, Gene prediction</td>
      <td>RNA-Seq, Genomics, Structure prediction, Sequence analysis</td>
      <td>To update</td>
      <td>https://github.com/Gaius-Augustus/BRAKER</td>
      <td>Genome annotation</td>
      <td>braker3</td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/braker</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/braker3</td>
      <td>3.0.8</td>
      <td></td>
      <td></td>
      <td>Genome annotation, Gene prediction</td>
      <td>RNA-Seq, Genomics, Structure prediction, Sequence analysis</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>10</td>
      <td>567</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>cap3</td>
      <td>cap3</td>
      <td>cap3 wrapper</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Assembly</td>
      <td>cap3</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/cap3</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/cap3</td>
      <td>2.0.1</td>
      <td>cap3</td>
      <td>10.2011</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>101</td>
      <td>7766</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>champ_blocs</td>
      <td>cb_dissim, cb_ivr, cb_div</td>
      <td>Compute indicators for turnover boulders fields</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Ecology</td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/Marie59/champ_blocs</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/champ_blocs</td>
      <td>0.0.0</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>19</td>
      <td>174</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>chipseeker</td>
      <td>chipseeker</td>
      <td>A tool for ChIP peak annotation and visualization</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/ChIPseeker.html</td>
      <td>ChIP-seq, Genome annotation</td>
      <td>chipseeker</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/chipseeker</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/chipseeker</td>
      <td>1.32.0</td>
      <td>bioconductor-chipseeker</td>
      <td>1.38.0</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>418</td>
      <td>15690</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>circexplorer2</td>
      <td>circexplorer2</td>
      <td>Comprehensive and integrative circular RNA analysis toolset.</td>
      <td>circexplorer2</td>
      <td>circexplorer2</td>
      <td></td>
      <td>CIRCexplorer2</td>
      <td>Genome-wide annotation of circRNAs and their alternative back-splicing/splicing.</td>
      <td></td>
      <td>RNA splicing, Gene transcripts, Literature and language</td>
      <td>Up-to-date</td>
      <td>https://github.com/YangLab/CIRCexplorer2</td>
      <td>RNA, Assembly</td>
      <td>circexplorer2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/circexplorer2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/circexplorer2</td>
      <td>2.3.8</td>
      <td>circexplorer2</td>
      <td>2.3.8</td>
      <td></td>
      <td>RNA splicing, Gene transcripts, Literature and language</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>16</td>
      <td>269</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>clc_assembly_cell</td>
      <td>clc_assembler, clc_mapper</td>
      <td>Galaxy wrapper for the CLC Assembly Cell suite from CLCBio</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/clc_assembly_cell</td>
      <td>Assembly, Next Gen Mappers, SAM</td>
      <td>clc_assembly_cell</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/clc_assembly_cell</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/clc_assembly_cell</td>
      <td>0.0.7</td>
      <td>samtools</td>
      <td>1.20</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>combine_assembly_stats</td>
      <td>combine_stats</td>
      <td>Combine multiple Assemblystats datasets into a single tabular report</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>Assembly</td>
      <td>combine_assemblystats</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/combine_assembly_stats</td>
      <td>1.0</td>
      <td>perl-getopt-long</td>
      <td>2.58</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>count_roi_variants</td>
      <td>count_roi_variants</td>
      <td>Count sequence variants in region of interest in BAM file</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/count_roi_variants</td>
      <td>Assembly, SAM</td>
      <td>count_roi_variants</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/count_roi_variants</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/count_roi_variants</td>
      <td>0.0.6</td>
      <td>samtools</td>
      <td>1.20</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>coverage_stats</td>
      <td>coverage_stats</td>
      <td>BAM coverage statistics using samtools idxstats and depth</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/coverage_stats</td>
      <td>Assembly, SAM</td>
      <td>coverage_stats</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/coverage_stats</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/coverage_stats</td>
      <td>0.1.0</td>
      <td>samtools</td>
      <td>1.20</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>data_exploration</td>
      <td>tool_anonymization, ecology_homogeneity_normality, ecology_beta_diversity, ecology_link_between_var, ecology_presence_abs_abund, ecology_stat_presence_abs</td>
      <td>Explore data through multiple statistical tools</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Ecology</td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/Marie59/Data_explo_tools</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/data_exploration</td>
      <td>0.0.0</td>
      <td>r-tangles</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>0</td>
      <td>0</td>
      <td>146</td>
      <td>658</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>deepsig</td>
      <td>deepsig</td>
      <td>Predictor of signal peptides in proteins based on deep learning</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/BolognaBiocomp/deepsig</td>
      <td>Genome annotation</td>
      <td>deepsig</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/deepsig</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/deepsig</td>
      <td>1.2.5</td>
      <td>deepsig</td>
      <td>1.2.5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>disco</td>
      <td>disco</td>
      <td>DISCO is a overlap-layout-consensus (OLC) metagenome assembler</td>
      <td>disco</td>
      <td>disco</td>
      <td></td>
      <td>DISCO</td>
      <td>DISCO is software to perform structure determination of protein homo-oligomers with cyclic symmetry.DISCO computes oligomeric protein structures using geometric constraints derived from RDCs and intermolecular distance restraints such as NOEs or disulfide bonds. When a reliable subunit structure can be calculated from intramolecular restraints, DISCO guarantees that all satisfying oligomer structures will be discovered, yet can run in minutes to hours on only a single desktop-class computer.</td>
      <td>Protein sequence analysis</td>
      <td>Structure determination</td>
      <td>To update</td>
      <td>http://disco.omicsbio.org/</td>
      <td>Metagenomics, Assembly</td>
      <td>disco</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/disco/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/disco</td>
      <td></td>
      <td>disco</td>
      <td>1.2</td>
      <td>Protein sequence analysis</td>
      <td>Structure determination</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>42</td>
      <td>369</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>dose_responses</td>
      <td>dr_curve</td>
      <td>A  tool for analyzing and visualizing the relationship between various doses and their corresponding biological responses</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/bernt-matthias/mb-galaxy-tools</td>
      <td>Ecology</td>
      <td>dose_response_analysis_tool</td>
      <td>ufz</td>
      <td>https://github.com/bernt-matthias/mb-galaxy-tools/tools/tox_tools/baseline_calculator</td>
      <td>https://github.com/Helmholtz-UFZ/galaxy-tools/tree/main/tools/tox_tools/dose_responses</td>
      <td>3.0_1</td>
      <td>r-drc</td>
      <td>3.0_1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>ear</td>
      <td>make_ear</td>
      <td>A tool to compile assembly reports and stastics from assembly pipeline</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ERGA-consortium/EARs/tree/main</td>
      <td>Sequence Analysis, Assembly</td>
      <td>erga_ear</td>
      <td>bgruening</td>
      <td>https://github.com/ERGA-consortium/EARs/tree/main</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/ear</td>
      <td>24.08.26</td>
      <td>reportlab</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>fastk</td>
      <td>fastk_fastk</td>
      <td>FastK: A K-mer counter (for HQ assembly data sets)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/thegenemyers/FASTK</td>
      <td>Assembly</td>
      <td>fastk</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fastk</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fastk</td>
      <td>1.0.0</td>
      <td>fastk</td>
      <td>1.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>fermikit</td>
      <td>fermi2, fermikit_variants</td>
      <td>FermiKit is a de novo assembly based variant calling pipeline for deep Illumina resequencing data.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/lh3/fermikit</td>
      <td>Assembly, Variant Analysis</td>
      <td>fermikit</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fermikit</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/fermikit</td>
      <td>r193</td>
      <td>fermi2</td>
      <td>r193</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>filter_spades_repeats</td>
      <td>filter_spades_repeat</td>
      <td>Remove short and repeat contigs/scaffolds</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/galaxy_tools/</td>
      <td>Assembly</td>
      <td>filter_spades_repeats</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools/</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/filter_spades_repeats</td>
      <td>1.0.1</td>
      <td>perl-bioperl</td>
      <td>1.7.8</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>flash</td>
      <td>flash</td>
      <td>Fast Length Adjustment of SHort reads</td>
      <td>flash</td>
      <td>flash</td>
      <td></td>
      <td>FLASH</td>
      <td>Identifies paired-end reads which overlap in the middle, converting them to single long reads</td>
      <td>Read pre-processing, Sequence merging, Sequence assembly</td>
      <td>Sequencing, Sequence assembly</td>
      <td>Up-to-date</td>
      <td>https://ccb.jhu.edu/software/FLASH/</td>
      <td>Assembly, Fastq Manipulation</td>
      <td>flash</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/flash</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/flash</td>
      <td>1.2.11</td>
      <td>flash</td>
      <td>1.2.11</td>
      <td>Read pre-processing, Sequence merging, Sequence assembly</td>
      <td>Sequencing, Sequence assembly</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>74</td>
      <td>13759</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>flye</td>
      <td>flye</td>
      <td>Assembly of long and error-prone reads.</td>
      <td>Flye</td>
      <td>Flye</td>
      <td></td>
      <td>Flye</td>
      <td>Flye is a de novo assembler for single molecule sequencing reads, such as those produced by PacBio and Oxford Nanopore Technologies. It is designed for a wide range of datasets, from small bacterial projects to large mammalian-scale assemblies. The package represents a complete pipeline: it takes raw PB / ONT reads as input and outputs polished contigs.</td>
      <td>Genome assembly, De-novo assembly, Mapping assembly, Cross-assembly</td>
      <td>Sequence assembly, Metagenomics, Whole genome sequencing, Genomics</td>
      <td>To update</td>
      <td>https://github.com/fenderglass/Flye/</td>
      <td>Assembly</td>
      <td>flye</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/flye</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/flye</td>
      <td>2.9.4</td>
      <td>flye</td>
      <td>2.9.5</td>
      <td>Genome assembly, De-novo assembly, Mapping assembly, Cross-assembly</td>
      <td>Sequence assembly, Metagenomics, Whole genome sequencing</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1499</td>
      <td>20904</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>funannotate</td>
      <td>funannotate_annotate, funannotate_clean, funannotate_compare, funannotate_predict, funannotate_sort</td>
      <td>Funannotate is a genome prediction, annotation, and comparison software package.</td>
      <td>funannotate</td>
      <td>funannotate</td>
      <td></td>
      <td>funannotate</td>
      <td>funannotate is a pipeline for genome annotation (built specifically for fungi, but will also work with higher eukaryotes).</td>
      <td>Genome annotation</td>
      <td>Genomics</td>
      <td>To update</td>
      <td>https://funannotate.readthedocs.io</td>
      <td>Genome annotation</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/funannotate</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/funannotate</td>
      <td>1.8.15</td>
      <td></td>
      <td></td>
      <td>Genome annotation</td>
      <td>Genomics</td>
      <td>3</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>3</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>900</td>
      <td>5906</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>gdal</td>
      <td>gdal_gdal_merge, gdal_gdal_translate, gdal_gdaladdo, gdal_gdalbuildvrt, gdal_gdalinfo, gdal_gdalwarp, gdal_ogr2ogr, gdal_ogrinfo</td>
      <td>Geospatial Data Abstraction Library tools are all dedicated to manipulate raster and vector geospatial data formats.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.gdal.org</td>
      <td>Ecology</td>
      <td>gdal</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/gdal</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/gdal</td>
      <td>3.0.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>10</td>
      <td>1475</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>getorganelle</td>
      <td>get_annotated_regions_from_gb, get_organelle_from_reads</td>
      <td>GetOrganelle - This toolkit assembles organelle genomes from genomic skimming data.</td>
      <td>getorganelle</td>
      <td>getorganelle</td>
      <td></td>
      <td>GetOrganelle</td>
      <td>A fast and versatile toolkit for accurate de novo assembly of organelle genomes.This toolkit assemblies organelle genome from genomic skimming data.</td>
      <td>De-novo assembly, Genome assembly, Mapping assembly, Mapping, Sequence trimming</td>
      <td>Cell biology, Sequence assembly, Whole genome sequencing, Plant biology, Model organisms</td>
      <td>Up-to-date</td>
      <td>https://github.com/Kinggerm/GetOrganelle</td>
      <td>Assembly</td>
      <td>getorganelle</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/getorganelle</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/getorganelle</td>
      <td>1.7.7.1</td>
      <td>getorganelle</td>
      <td>1.7.7.1</td>
      <td>De-novo assembly, Genome assembly, Mapping assembly, Mapping, Sequence trimming</td>
      <td>Cell biology, Sequence assembly, Whole genome sequencing, Plant biology, Model organisms</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>18</td>
      <td>495</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>goenrichment</td>
      <td>goenrichment, goslimmer</td>
      <td>Performs GO Enrichment analysis.</td>
      <td>goenrichment</td>
      <td>goenrichment</td>
      <td></td>
      <td>GOEnrichment</td>
      <td>GOEnrichment is a tool for performing GO enrichment analysis of gene sets, such as those obtained from RNA-seq or Microarray experiments, to help characterize them at the functional level. It is available in Galaxy Europe and as a stand-alone tool.GOEnrichment is flexible in that it allows the user to use any version of the Gene Ontology and any GO annotation file they desire. To enable the use of GO slims, it is accompanied by a sister tool GOSlimmer, which can convert annotation files from full GO to any specified GO slim.The tool features an optional graph clustering algorithm to reduce the redundancy in the set of enriched GO terms and simplify its output.It was developed by the BioData.pt / ELIXIR-PT team at the Instituto Gulbenkian de Ciência.</td>
      <td>Gene-set enrichment analysis</td>
      <td>Transcriptomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/DanFaria/GOEnrichment</td>
      <td>Genome annotation</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/goenrichment</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/goenrichment</td>
      <td>2.0.1</td>
      <td>goenrichment</td>
      <td>2.0.1</td>
      <td>Gene-set enrichment analysis</td>
      <td>Transcriptomics</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>408</td>
      <td>5484</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>graphmap</td>
      <td>graphmap_align, graphmap_overlap</td>
      <td>Mapper for long, error-prone reads.</td>
      <td>graphmap</td>
      <td>graphmap</td>
      <td></td>
      <td>graphmap</td>
      <td>Splice-aware RNA-seq mapper for long reads | GraphMap - A highly sensitive and accurate mapper for long, error-prone reads http://www.nature.com/ncomms/2016/160415/ncomms11307/full/ncomms11307.html https://www.biorxiv.org/content/10.1101/720458v1</td>
      <td>Sequence trimming, EST assembly, Read mapping</td>
      <td>Gene transcripts, RNA-Seq, RNA splicing</td>
      <td>To update</td>
      <td>https://github.com/isovic/graphmap/</td>
      <td>Assembly</td>
      <td>graphmap</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/graphmap</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/graphmap</td>
      <td>0.5.2</td>
      <td>graphmap</td>
      <td>0.6.3</td>
      <td>Sequence trimming, EST assembly, Read mapping</td>
      <td>Gene transcripts, RNA-Seq, RNA splicing</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>136</td>
      <td>6651</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>hapcut2</td>
      <td>hapcut2</td>
      <td>Robust and accurate haplotype assembly for diverse sequencing technologies</td>
      <td>hapcut2</td>
      <td>hapcut2</td>
      <td></td>
      <td>HapCUT2</td>
      <td>HapCUT2 is a maximum-likelihood-based tool for assembling haplotypes from DNA sequence reads, designed to "just work" with excellent speed and accuracy across a range of long- and short-read sequencing technologies.The output is in Haplotype block format described here: https://github.com/vibansal/HapCUT2/blob/master/outputformat.md</td>
      <td>Haplotype mapping, Variant classification</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/vibansal/HapCUT2</td>
      <td>Assembly</td>
      <td>hapcut2</td>
      <td>galaxy-australia</td>
      <td>https://github.com/galaxyproject/tools-iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hapcut2</td>
      <td>1.3.4</td>
      <td>hapcut2</td>
      <td>1.3.4</td>
      <td>Haplotype mapping, Variant classification</td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>hapog</td>
      <td>hapog</td>
      <td>Hapo-G - Haplotype-Aware Polishing of Genomes</td>
      <td>hapog</td>
      <td>hapog</td>
      <td></td>
      <td>Hapo-G</td>
      <td>Hapo-G is a tool that aims to improve the quality of genome assemblies by polishing the consensus with accurate reads. It capable of incorporating phasing information from high-quality reads (short or long-reads) to polish genome assemblies and in particular assemblies of diploid and heterozygous genomes.</td>
      <td>Genome assembly, Optimisation and refinement</td>
      <td>Sequence assembly, Genomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/institut-de-genomique/HAPO-G</td>
      <td>Assembly</td>
      <td>hapog</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hapog</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hapog</td>
      <td>1.3.8</td>
      <td>hapog</td>
      <td>1.3.8</td>
      <td>Genome assembly, Optimisation and refinement</td>
      <td>Sequence assembly, Genomics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>36</td>
      <td>295</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>helixer</td>
      <td>helixer</td>
      <td>Gene calling with Deep Neural Networks</td>
      <td>helixer</td>
      <td>helixer</td>
      <td></td>
      <td>Helixer</td>
      <td>Deep Learning to predict gene annotations</td>
      <td>Gene prediction, Genome annotation</td>
      <td>Sequence analysis, Gene transcripts</td>
      <td>To update</td>
      <td>https://github.com/weberlab-hhu/Helixer</td>
      <td>Genome annotation</td>
      <td>helixer</td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/helixer</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/helixer</td>
      <td>0.3.3</td>
      <td></td>
      <td></td>
      <td>Gene prediction, Genome annotation</td>
      <td>Sequence analysis, Gene transcripts</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>93</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>hifiasm</td>
      <td>hifiasm</td>
      <td>A fast haplotype-resolved de novo assembler</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/chhylp123/hifiasm</td>
      <td>Assembly</td>
      <td>hifiasm</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/hifiasm</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hifiasm</td>
      <td>0.19.9</td>
      <td>hifiasm</td>
      <td>0.19.9</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>297</td>
      <td>1410</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>hirondelle_crim_ogc_api_processes</td>
      <td>hirondelle_crim</td>
      <td>This tool is a wrapper for OGC API Processes coming from https://osf.io/gfbws/.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/AquaINFRA/galaxy</td>
      <td>Ecology</td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/ogc_api_processes_wrapper</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/hirondelle_crim_ogc_api_processes</td>
      <td>0.1.0</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>hisat</td>
      <td>hisat</td>
      <td>HISAT is a fast and sensitive spliced alignment program.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://ccb.jhu.edu/software/hisat/index.shtml</td>
      <td>Assembly</td>
      <td>hisat</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/hisat</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/hisat</td>
      <td>1.0.3</td>
      <td>hisat</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>228</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>hisat2</td>
      <td>hisat2</td>
      <td>HISAT2 is a fast and sensitive spliced alignment program.</td>
      <td>hisat2</td>
      <td>hisat2</td>
      <td></td>
      <td>HISAT2</td>
      <td>Alignment program for mapping next-generation sequencing reads (both DNA and RNA) to a population of human genomes (as well as to a single reference genome).</td>
      <td>Sequence alignment</td>
      <td>RNA-seq</td>
      <td>Up-to-date</td>
      <td>http://ccb.jhu.edu/software/hisat2/</td>
      <td>Assembly</td>
      <td>hisat2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hisat2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hisat2</td>
      <td>2.2.1</td>
      <td>hisat2</td>
      <td>2.2.1</td>
      <td>Sequence alignment</td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>4183</td>
      <td>299104</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>hypo</td>
      <td>hypo</td>
      <td>Super Fast &amp; Accurate Polisher for Long Read Genome Assemblies</td>
      <td>HyPo</td>
      <td>HyPo</td>
      <td></td>
      <td>HyPo</td>
      <td>HyPo, a Hybrid Polisher, utilizes short as well as long reads within a single run to polish a long reads assembly of small and large genomes.</td>
      <td>Optimisation and refinement, Genome assembly</td>
      <td>Sequence assembly, Genomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/kensung-lab/hypo</td>
      <td>Assembly</td>
      <td>hypo</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hypo</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/hypo</td>
      <td>1.0.3</td>
      <td>hypo</td>
      <td>1.0.3</td>
      <td>Optimisation and refinement, Genome assembly</td>
      <td>Sequence assembly, Genomics</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>39</td>
      <td>354</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>icescreen</td>
      <td>icescreen</td>
      <td>ICEscreen identifies Integrative Conjugative Elements (ICEs) and Integrative Mobilizable Elements (IMEs) in Bacillota genomes.</td>
      <td>icescreen</td>
      <td>icescreen</td>
      <td></td>
      <td>ICEscreen</td>
      <td>A tool to detect Firmicute ICEs and IMEs, isolated or enclosed in composite structures.</td>
      <td>Database search, Protein feature detection</td>
      <td>Mobile genetic elements, Sequence sites, features and motifs, Genomics, Molecular interactions, pathways and networks, Structural variation</td>
      <td>To update</td>
      <td>https://icescreen.migale.inrae.fr/</td>
      <td>Genome annotation</td>
      <td>icescreen</td>
      <td>iuc</td>
      <td>https://forgemia.inra.fr/ices_imes_analysis/icescreen</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/icescreen</td>
      <td>1.3.1</td>
      <td>icescreen</td>
      <td>1.3.2</td>
      <td>Database search, Protein feature detection</td>
      <td>Mobile genetic elements, Sequence sites, features and motifs, Genomics, Molecular interactions, pathways and networks, Structural variation</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>idba_ud</td>
      <td>idba_hybrid, idba_tran, idba_ud</td>
      <td>Wrappers for the idba assembler variants.</td>
      <td>idba</td>
      <td>idba</td>
      <td></td>
      <td>IDBA</td>
      <td>A short read assembler based on iterative De Bruijn graph. It is developed under 64-bit Linux, but should be suitable for all unix-like system.</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly</td>
      <td>To update</td>
      <td>https://i.cs.hku.hk/~alse/hkubrg/projects/index.html</td>
      <td>Assembly</td>
      <td>idba</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/idba_ud</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/idba_ud</td>
      <td></td>
      <td>idba</td>
      <td>1.1.3</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly</td>
      <td>3</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>3</td>
      <td>0</td>
      <td>69</td>
      <td>1127</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>instagraal</td>
      <td>instagraal</td>
      <td>Large genome reassembly based on Hi-C data</td>
      <td>instagraal</td>
      <td>instagraal</td>
      <td></td>
      <td>instaGRAAL</td>
      <td>Chromosome-level quality scaffolding of brown algal genomes using InstaGRAAL.Large genome reassembly based on Hi-C data, continuation of GRAAL.Large genome reassembly based on Hi-C data (continuation and partial rewrite of GRAAL) and post-scaffolding polishing libraries.This work is under continuous development/improvement - see GRAAL for information about the basic principles.sudo pip3 install -e git+https://github.com/koszullab/instagraal.git@master#egg=instagraal.Note to OS X users: There is currently no CUDA support on Mojave (10.14) and it is unclear when it is going to be added, if it is to be added at all. This means instaGRAAL (or indeed any CUDA-based application) will not work on Mojave. If you wish to run it on OS X, the only solution for now is to downgrade to High Sierra (10.13)</td>
      <td>Genome assembly, Mapping assembly, Genetic mapping, Scaffolding</td>
      <td>Sequence assembly, Mapping, Metagenomics, Statistics and probability, DNA binding sites</td>
      <td>To update</td>
      <td>https://github.com/koszullab/instaGRAAL</td>
      <td>Assembly</td>
      <td>instagraal</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/instagraal</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/instagraal</td>
      <td>0.1.6</td>
      <td></td>
      <td></td>
      <td>Genome assembly, Mapping assembly, Genetic mapping, Scaffolding</td>
      <td>Sequence assembly, Mapping, Metagenomics, Statistics and probability, DNA binding sites</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>14</td>
      <td>139</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>interpolation</td>
      <td>interpolation_run_idw_interpolation</td>
      <td>Run IDW interpolation based on a .csv and .geojson file</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/AquaINFRA/galaxy</td>
      <td>Ecology</td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/interpolation</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/interpolation</td>
      <td>1.0</td>
      <td>r-getopt</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>jellyfish</td>
      <td>jellyfish</td>
      <td>Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA</td>
      <td>Jellyfish</td>
      <td>Jellyfish</td>
      <td></td>
      <td>Jellyfish</td>
      <td>A command-line algorithm for counting k-mers in DNA sequence.</td>
      <td>k-mer counting</td>
      <td>Sequence analysis, Genomics</td>
      <td>To update</td>
      <td>https://github.com/gmarcais/Jellyfish</td>
      <td>Assembly</td>
      <td>jellyfish</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/jellyfish</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/jellyfish</td>
      <td></td>
      <td>kmer-jellyfish</td>
      <td>2.3.1</td>
      <td>k-mer counting</td>
      <td>Sequence analysis, Genomics</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>91</td>
      <td>1138</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>khmer</td>
      <td>khmer_abundance_distribution_single, khmer_abundance_distribution, khmer_count_median, khmer_partition, khmer_extract_partitions, khmer_filter_abundance, khmer_filter_below_abundance_cutoff, khmer_normalize_by_median</td>
      <td>In-memory nucleotide sequence k-mer counting, filtering, graph traversal and more</td>
      <td>khmer</td>
      <td>khmer</td>
      <td></td>
      <td>khmer</td>
      <td>khmer is a set of command-line tools for working with DNA shotgun sequencing data from genomes, transcriptomes, metagenomes, and single cells. khmer can make de novo assemblies faster, and sometimes better. khmer can also identify (and fix) problems with shotgun data.</td>
      <td>Standardisation and normalisation, De-novo assembly</td>
      <td>Sequence assembly</td>
      <td>Up-to-date</td>
      <td>https://khmer.readthedocs.org/</td>
      <td>Assembly, Next Gen Mappers</td>
      <td>khmer</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/khmer</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/khmer</td>
      <td>3.0.0a3</td>
      <td>khmer</td>
      <td>3.0.0a3</td>
      <td>Standardisation and normalisation, De-novo assembly</td>
      <td>Sequence assembly</td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>8</td>
      <td>8</td>
      <td>0</td>
      <td>63</td>
      <td>2022</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>links</td>
      <td>links</td>
      <td>Scaffold genome assemblies with long reads.</td>
      <td>links</td>
      <td>links</td>
      <td></td>
      <td>LINKS</td>
      <td>LINKS (Long Interval Nucleotide K-mer Scaffolder) is a genomics application for scaffolding genome assemblies with long reads, such as those produced by Oxford Nanopore Technologies Ltd. It can be used to scaffold high-quality draft genome assemblies with any long sequences (eg. ONT reads, PacBio reads, other draft genomes, etc). It is also used to scaffold contig pairs linked by ARCS/ARKS.</td>
      <td>Scaffolding, Genome assembly, Read mapping, Read pre-processing, Sequence trimming</td>
      <td>Sequence assembly, Mapping, Sequencing</td>
      <td>Up-to-date</td>
      <td>https://github.com/bcgsc/LINKS</td>
      <td>Assembly</td>
      <td>links</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/links</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/links</td>
      <td>2.0.1</td>
      <td>links</td>
      <td>2.0.1</td>
      <td>Scaffolding, Genome assembly, Read mapping, Read pre-processing, Sequence trimming</td>
      <td>Sequence assembly, Mapping, Sequencing</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>77</td>
      <td>405</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>marine_omics</td>
      <td>sanntis_marine</td>
      <td>The Sanntis tool identify biosynthetic gene clusters (BGCs) in genomic &amp; metagenomic data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/Finn-Lab/SanntiS</td>
      <td>Ecology</td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/marine_omics</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/marine_omics</td>
      <td>0.9.3.5</td>
      <td>sanntis</td>
      <td>0.9.3.5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>mean-per-zone</td>
      <td>mean_per_zone</td>
      <td>Creates a png image showing statistic over areas as defined in the vector file</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/blob/master/tools/mean-per-zone/</td>
      <td>Visualization, GIS, Climate Analysis</td>
      <td>mean_per_zone</td>
      <td>climate</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/mean-per-zone</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/mean-per-zone</td>
      <td>0.2.0</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>24</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>medenv</td>
      <td>iabiodiv_smartbiodiv_med_environ</td>
      <td>Retrieve environmental data from etopo, cmems and woa</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/jeremyfix/medenv</td>
      <td>Ecology, Data Source</td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/jeremyfix/medenv</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/medenv</td>
      <td>0.1.0</td>
      <td>pandas</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>megahit</td>
      <td>megahit</td>
      <td>An ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph.</td>
      <td>megahit</td>
      <td>megahit</td>
      <td></td>
      <td>MEGAHIT</td>
      <td>Single node assembler for large and complex metagenomics NGS reads, such as soil. It makes use of succinct de Bruijn graph to achieve low memory usage, whereas its goal is not to make memory usage as low as possible.</td>
      <td>Genome assembly</td>
      <td>Metagenomics, Sequencing, Ecology, Sequence assembly</td>
      <td>Up-to-date</td>
      <td>https://github.com/voutcn/megahit</td>
      <td>Sequence Analysis, Assembly, Metagenomics</td>
      <td>megahit</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/megahit</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/megahit</td>
      <td>1.2.9</td>
      <td>megahit</td>
      <td>1.2.9</td>
      <td>Genome assembly</td>
      <td>Metagenomics, Sequencing, Sequence assembly</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>548</td>
      <td>9530</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>megahit_contig2fastg</td>
      <td>megahit_contig2fastg</td>
      <td>A subprogram within the Megahit toolkit for converting contigs to assembly graphs (fastg)</td>
      <td>megahit</td>
      <td>megahit</td>
      <td></td>
      <td>MEGAHIT</td>
      <td>Single node assembler for large and complex metagenomics NGS reads, such as soil. It makes use of succinct de Bruijn graph to achieve low memory usage, whereas its goal is not to make memory usage as low as possible.</td>
      <td>Genome assembly</td>
      <td>Metagenomics, Sequencing, Ecology, Sequence assembly</td>
      <td>To update</td>
      <td>https://github.com/voutcn/megahit/blob/master/tools/toolkit.cpp</td>
      <td>Sequence Analysis, Assembly, Metagenomics</td>
      <td>megahit_contig2fastg</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/megahit_contig2fastg</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/megahit_contig2fastg</td>
      <td>1.1.3</td>
      <td>megahit</td>
      <td>1.2.9</td>
      <td>Genome assembly</td>
      <td>Metagenomics, Sequencing, Sequence assembly</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>54</td>
      <td>475</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>merqury</td>
      <td>merqury, merquryplot</td>
      <td>Merqury is a tool for evaluating genomes assemblies based of k-mer operations.</td>
      <td>merqury</td>
      <td>merqury</td>
      <td></td>
      <td>Merqury</td>
      <td>Reference-free quality, completeness, and phasing assessment for genome assemblies.Evaluate genome assemblies with k-mers and more.Often, genome assembly projects have illumina whole genome sequencing reads available for the assembled individual.Merqury provides a set of tools for this purpose.</td>
      <td>Genome assembly, k-mer counting, Scaffolding, Phasing, De-novo assembly</td>
      <td>Sequence assembly, Whole genome sequencing, Plant biology</td>
      <td>Up-to-date</td>
      <td>https://github.com/marbl/merqury</td>
      <td>Assembly</td>
      <td>merqury</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/merqury</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/merqury</td>
      <td>1.3</td>
      <td>merqury</td>
      <td>1.3</td>
      <td>Genome assembly, k-mer counting, Scaffolding, Phasing, De-novo assembly</td>
      <td>Sequence assembly, Whole genome sequencing, Plant biology</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>244</td>
      <td>2483</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>meryl</td>
      <td>meryl_arithmetic_kmers, meryl_count_kmers, meryl_filter_kmers, meryl_groups_kmers, meryl_histogram_kmers, meryl_print, meryl_trio_mode</td>
      <td>Meryl a k-mer counter.</td>
      <td>meryl</td>
      <td>meryl</td>
      <td></td>
      <td>Meryl</td>
      <td>Meryl is a tool for counting and working with sets of k-mers that was originally developed for use in the Celera Assembler and has since been migrated and maintained as part of Canu.</td>
      <td>k-mer counting</td>
      <td>Whole genome sequencing, Genomics, Sequence analysis, Sequencing</td>
      <td>Up-to-date</td>
      <td>https://github.com/marbl/meryl</td>
      <td>Assembly</td>
      <td>meryl</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/meryl</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/meryl</td>
      <td>1.3</td>
      <td>merqury</td>
      <td>1.3</td>
      <td>k-mer counting</td>
      <td>Whole genome sequencing, Genomics, Sequence analysis</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>metaeuk</td>
      <td>metaeuk_easy_predict</td>
      <td>MetaEuk is a modular toolkit designed for large-scale gene discovery andannotation in eukaryotic metagenomic contigs. Metaeuk combines the fast andsensitive homology search capabilities of MMseqs2 with a dynamic programmingprocedure to recover optimal exons sets. It reduces redundancies in multiplediscoveries of the same gene and resolves conflicting gene predictions onthe same strand.</td>
      <td>MetaEuk</td>
      <td>MetaEuk</td>
      <td></td>
      <td>MetaEuk</td>
      <td>MetaEuk - sensitive, high-throughput gene discovery and annotation for large-scale eukaryotic metagenomics</td>
      <td>Homology-based gene prediction</td>
      <td>Metagenomics, Gene and protein families</td>
      <td>To update</td>
      <td>https://github.com/soedinglab/metaeuk</td>
      <td>Sequence Analysis, Genome annotation</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/metaeuk</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/metaeuk</td>
      <td>5.34c21f2</td>
      <td>metaeuk</td>
      <td>6.a5d39d9</td>
      <td>Homology-based gene prediction</td>
      <td>Metagenomics, Gene and protein families</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>16</td>
      <td>184</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>minia</td>
      <td>minia</td>
      <td>Short-read assembler based on a de Bruijn graph</td>
      <td>minia</td>
      <td>minia</td>
      <td></td>
      <td>Minia</td>
      <td>Short-read assembler based on a de Bruijn graph, capable of assembling a human genome on a desktop computer in a day.</td>
      <td>Genome assembly</td>
      <td>Sequence assembly</td>
      <td>Up-to-date</td>
      <td>https://gatb.inria.fr/software/minia/</td>
      <td>Assembly</td>
      <td>minia</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/minia</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/minia</td>
      <td>3.2.6</td>
      <td>minia</td>
      <td>3.2.6</td>
      <td>Genome assembly</td>
      <td>Sequence assembly</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>109</td>
      <td>2206</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>miniasm</td>
      <td>miniasm</td>
      <td>Miniasm - Ultrafast de novo assembly for long noisy reads (though having no consensus step)</td>
      <td>miniasm</td>
      <td>miniasm</td>
      <td></td>
      <td>miniasm</td>
      <td>Miniasm is a very fast OLC-based de novo assembler for noisy long reads. It takes all-vs-all read self-mappings (typically by minimap) as input and outputs an assembly graph in the GFA format.</td>
      <td>De-novo assembly</td>
      <td>Genomics, Sequence assembly</td>
      <td>To update</td>
      <td>https://github.com/lh3/miniasm</td>
      <td>Assembly</td>
      <td>miniasm</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/miniasm</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/miniasm</td>
      <td>0.3_r179</td>
      <td>miniasm</td>
      <td>0.3</td>
      <td>De-novo assembly</td>
      <td>Genomics, Sequence assembly</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>178</td>
      <td>11938</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>mitobim</td>
      <td>mitobim</td>
      <td>assemble mitochondrial genomes</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/chrishah/MITObim</td>
      <td>Assembly</td>
      <td>mitobim</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mitobim</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/mitobim</td>
      <td>1.9.1</td>
      <td>mitobim</td>
      <td>1.9.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>66</td>
      <td>881</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>mitohifi</td>
      <td>mitohifi</td>
      <td>Assembly mitogenomes from Pacbio HiFi read.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/marcelauliano/MitoHiFi/tree/mitohifi_v2</td>
      <td>Assembly</td>
      <td>mitohifi</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/mitohifi</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/mitohifi</td>
      <td>3</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>56</td>
      <td>613</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>ncbi_egapx</td>
      <td>ncbi_egapx</td>
      <td>Eukaryotic Genome Annotation Pipeline - External (EGAPx)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ncbi/egapx</td>
      <td>Genome annotation</td>
      <td>ncbi_egapx</td>
      <td>richard-burhans</td>
      <td>https://github.com/richard-burhans/galaxytools/tree/main/tools/ncbi_egapx</td>
      <td>https://github.com/richard-burhans/galaxytools/tree/main/tools/ncbi_egapx</td>
      <td>0.2-alpha</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>necat</td>
      <td>necat</td>
      <td>Error correction and de-novo assembly for ONT Nanopore reads</td>
      <td>necat</td>
      <td>necat</td>
      <td></td>
      <td>NECAT</td>
      <td>NECAT is an error correction and de-novo assembly tool for Nanopore long noisy reads.</td>
      <td>De-novo assembly</td>
      <td>Sequence assembly</td>
      <td>Up-to-date</td>
      <td>https://github.com/xiaochuanle/NECAT</td>
      <td>Assembly</td>
      <td>necat</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/necat</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/necat</td>
      <td>0.0.1_update20200803</td>
      <td>necat</td>
      <td>0.0.1_update20200803</td>
      <td>De-novo assembly</td>
      <td>Sequence assembly</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>95</td>
      <td>667</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>nextdenovo</td>
      <td>nextdenovo</td>
      <td>String graph-based de novo assembler for long reads</td>
      <td>nextdenovo</td>
      <td>nextdenovo</td>
      <td></td>
      <td>NextDenovo</td>
      <td>NextDenovo is a string graph-based de novo assembler for long reads (CLR, HiFi and ONT). It uses a "correct-then-assemble" strategy similar to canu (no correction step for PacBio Hifi reads), but requires significantly less computing resources and storages.</td>
      <td>De-novo assembly, Genome assembly</td>
      <td>Sequencing, Sequence assembly</td>
      <td>To update</td>
      <td>https://github.com/Nextomics/NextDenovo</td>
      <td>Assembly</td>
      <td>nextdenovo</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/nextdenovo</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/nextdenovo</td>
      <td>2.5.0</td>
      <td>nextdenovo</td>
      <td>2.5.2</td>
      <td>De-novo assembly, Genome assembly</td>
      <td>Sequencing, Sequence assembly</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>84</td>
      <td>268</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>novoplasty</td>
      <td>novoplasty</td>
      <td>NOVOPlasty is a de novo assembler and heteroplasmy/variance caller for short circular genomes.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ndierckx/NOVOPlasty</td>
      <td>Assembly</td>
      <td>novoplasty</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/novoplasty</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/novoplasty</td>
      <td>4.3.1</td>
      <td>novoplasty</td>
      <td>4.3.5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>162</td>
      <td>6384</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>oases</td>
      <td>oasesoptimiserv</td>
      <td>Short read assembler</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Assembly, RNA</td>
      <td>oases</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/oases</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/oases</td>
      <td>1.4.0</td>
      <td>oases</td>
      <td>0.2.09</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>obisindicators</td>
      <td>obisindicators, obis_data</td>
      <td>Compute biodiveristy indicators for marine data from obis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/Marie59/obisindicators</td>
      <td>Ecology</td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/obisindicators</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/obisindicators</td>
      <td>0.0.2</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>0</td>
      <td>2</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>2</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>45</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>ocean</td>
      <td>argo_getdata, divand_full_analysis</td>
      <td>Access, process, visualise oceanographic data for the Earth System</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/Marie59/FE-ft-ESG/tree/main/argo</td>
      <td>Ecology</td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/ocean</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/ocean</td>
      <td>0.1.15</td>
      <td>julia</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>ogcProcess_otb_bandmath</td>
      <td>otb_band_math</td>
      <td>Outputs a monoband image which is the result of a mathematical operation on several multi-band images.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/AquaINFRA/galaxy</td>
      <td>Ecology</td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/OtbBandMath</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/ogcProcess_otb_bandmath</td>
      <td>1.0</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>ogcProcess_otb_meanShiftSmoothing</td>
      <td>otb_mean_shift_smoothing</td>
      <td>This application smooths an image using the MeanShift algorithm.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/AquaINFRA/galaxy</td>
      <td>Ecology</td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/interpolation</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/ogcProcess_otb_meanShiftSmoothing</td>
      <td>1.0</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>pharokka</td>
      <td>pharokka</td>
      <td>rapid standardised annotation tool for bacteriophage genomes and metagenomes</td>
      <td>pharokka</td>
      <td>pharokka</td>
      <td></td>
      <td>Pharokka</td>
      <td>Pharokka is a rapid standardised annotation tool for bacteriophage genomes and metagenomes.</td>
      <td>Genome annotation, Antimicrobial resistance prediction, tRNA gene prediction, Formatting, Sequence assembly</td>
      <td>Metagenomics, Sequence sites, features and motifs, Workflows, Functional, regulatory and non-coding RNA</td>
      <td>To update</td>
      <td>https://github.com/gbouras13/pharokka</td>
      <td>Genome annotation</td>
      <td>pharokka</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pharokka</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pharokka</td>
      <td>1.3.2</td>
      <td>\n                pharokka\n</td>
      <td></td>
      <td>Genome annotation, Antimicrobial resistance prediction, tRNA gene prediction, Formatting, Sequence assembly</td>
      <td>Metagenomics, Sequence sites, features and motifs, Workflows, Functional, regulatory and non-coding RNA</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>74</td>
      <td>2565</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>plasmidspades</td>
      <td>plasmidspades</td>
      <td>Genome assembler for assemblying plasmid</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Assembly</td>
      <td>plasmidspades</td>
      <td>nml</td>
      <td></td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/plasmidspades</td>
      <td>1.1</td>
      <td>spades</td>
      <td>4.0.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>prodigal</td>
      <td>prodigal</td>
      <td>A protein-coding gene prediction software tool for bacterial and archaeal genomes</td>
      <td>prodigal</td>
      <td>prodigal</td>
      <td></td>
      <td>Prodigal</td>
      <td>Fast, reliable protein-coding gene prediction for prokaryotic genomes.</td>
      <td>Genome annotation</td>
      <td>Genomics, Sequence analysis</td>
      <td>Up-to-date</td>
      <td>https://github.com/hyattpd/Prodigal</td>
      <td>Genome annotation</td>
      <td>prodigal</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/prodigal</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/prodigal</td>
      <td>2.6.3</td>
      <td>prodigal</td>
      <td>2.6.3</td>
      <td>Genome annotation</td>
      <td>Genomics, Sequence analysis</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>promer</td>
      <td>promer4_substitutions</td>
      <td>Aligns two sets of contigs and reports amino acid substitutions between them</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/promer</td>
      <td>Assembly</td>
      <td>promer</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/promer</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/promer</td>
      <td>1.2</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>purge_dups</td>
      <td>purge_dups</td>
      <td>Purge haplotigs and overlaps in an assembly based on read depth</td>
      <td>purge_dups</td>
      <td>purge_dups</td>
      <td></td>
      <td>purge_dups</td>
      <td>Identifying and removing haplotypic duplication in primary genome assemblies | haplotypic duplication identification tool | scripts/pd_config.py: script to generate a configuration file used by run_purge_dups.py | purge haplotigs and overlaps in an assembly based on read depth | Given a primary assembly pri_asm and an alternative assembly hap_asm (optional, if you have one), follow the steps shown below to build your own purge_dups pipeline, steps with same number can be run simultaneously. Among all the steps, although step 4 is optional, we highly recommend our users to do so, because assemblers may produce overrepresented seqeuences. In such a case, The final step 4 can be applied to remove those seqeuences</td>
      <td>Genome assembly, Read binning, Scaffolding</td>
      <td>Sequence assembly</td>
      <td>Up-to-date</td>
      <td>https://github.com/dfguan/purge_dups</td>
      <td>Assembly</td>
      <td>purge_dups</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/purge_dups</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/purge_dups</td>
      <td>1.2.6</td>
      <td>purge_dups</td>
      <td>1.2.6</td>
      <td>Genome assembly, Read binning, Scaffolding</td>
      <td>Sequence assembly</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>167</td>
      <td>16800</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>quast</td>
      <td>quast</td>
      <td>Quast (Quality ASsessment Tool) evaluates genome assemblies.</td>
      <td>quast</td>
      <td>quast</td>
      <td></td>
      <td>QUAST</td>
      <td>QUAST stands for QUality ASsessment Tool.  It evaluates a quality of genome assemblies by computing various metrics and providing nice reports.</td>
      <td>Visualisation, Sequence assembly validation</td>
      <td>Sequence assembly</td>
      <td>Up-to-date</td>
      <td>http://quast.bioinf.spbau.ru/</td>
      <td>Assembly</td>
      <td>quast</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/quast</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/quast</td>
      <td>5.2.0</td>
      <td>quast</td>
      <td>5.2.0</td>
      <td>Visualisation, Sequence assembly validation</td>
      <td>Sequence assembly</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>3567</td>
      <td>51567</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>quickmerge</td>
      <td>quickmerge</td>
      <td>Merge long-read and hybrid assemblies to increase contiguity</td>
      <td>quickmerge</td>
      <td>quickmerge</td>
      <td></td>
      <td>quickmerge</td>
      <td>Quickmerge is a program that uses complementary information from genomes assembled with long reads in order to improve contiguity, and works with assemblies derived from both Pacific Biosciences or Oxford Nanopore. Quickmerge will even work with hybrid assemblies made by combining long reads and Illumina short reads.</td>
      <td>Genome assembly, Scaffolding, De-novo assembly, Genotyping</td>
      <td>Structural variation, Sequence assembly, DNA polymorphism, Whole genome sequencing, Genotype and phenotype</td>
      <td>Up-to-date</td>
      <td>https://github.com/mahulchak/quickmerge</td>
      <td>Assembly</td>
      <td>quickmerge</td>
      <td>galaxy-australia</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/quickmerge</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/quickmerge</td>
      <td>0.3</td>
      <td>quickmerge</td>
      <td>0.3</td>
      <td>Genome assembly, Scaffolding, De-novo assembly, Genotyping</td>
      <td>Structural variation, Sequence assembly, DNA polymorphism, Whole genome sequencing, Genotype and phenotype</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>ragtag</td>
      <td>ragtag</td>
      <td>Reference-guided scaffolding of draft genomes tool.</td>
      <td>ragtag</td>
      <td>ragtag</td>
      <td></td>
      <td>ragtag</td>
      <td>RagTag is a collection of software tools for scaffolding and improving modern genome assemblies.</td>
      <td>Genome assembly</td>
      <td>Sequence assembly</td>
      <td>Up-to-date</td>
      <td>https://github.com/malonge/RagTag</td>
      <td>Assembly</td>
      <td>ragtag</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ragtag</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/ragtag</td>
      <td>2.1.0</td>
      <td>ragtag</td>
      <td>2.1.0</td>
      <td>Genome assembly</td>
      <td>Sequence assembly</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>237</td>
      <td>2833</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>raven</td>
      <td>raven</td>
      <td>Raven is a de novo genome assembler for long uncorrected reads.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/lbcb-sci/raven</td>
      <td>Assembly</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/raven</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/raven</td>
      <td>1.8.3</td>
      <td>raven-assembler</td>
      <td>1.8.3</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>262</td>
      <td>6902</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>regionalgam</td>
      <td>regionalgam_ab_index, regionalgam_autocor_acf, regionalgam_flight_curve, regionalgam_glmmpql, regionalgam_gls_adjusted, regionalgam_gls, regionalgam_plot_trend</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/RetoSchmucki/regionalGAM</td>
      <td>Ecology</td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/regionalgam</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/regionalgam</td>
      <td>1.5</td>
      <td>r-mgcv</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>22</td>
      <td>526</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>repeatexplorer2</td>
      <td>repeatexplorer_clustering</td>
      <td>Tool for annotation of repeats from unassembled shotgun reads.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/repeatexplorer/repex_tarean</td>
      <td>Genome annotation</td>
      <td>repeatexplorer2</td>
      <td>gga</td>
      <td>https://github.com/galaxy-genome-annotation/galaxy-tools/tree/master/tools/repeatexplorer2</td>
      <td>https://github.com/galaxy-genome-annotation/galaxy-tools/tree/master/tools/repeatexplorer2</td>
      <td>2.3.8</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>16</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>repeatmodeler</td>
      <td>repeatmodeler</td>
      <td>RepeatModeler - Model repetitive DNA</td>
      <td>repeatmodeler</td>
      <td>repeatmodeler</td>
      <td></td>
      <td>RepeatModeler</td>
      <td>De-novo repeat family identification and modeling package. At the heart of RepeatModeler are two de-novo repeat finding programs ( RECON and RepeatScout ) which employ complementary computational methods for identifying repeat element boundaries and family relationships from sequence data. RepeatModeler assists in automating the runs of RECON and RepeatScout given a genomic database and uses the output to build, refine and classify consensus models of putative interspersed repeats.</td>
      <td>Repeat sequence detection</td>
      <td>Sequence composition, complexity and repeats, Sequence composition, complexity and repeats</td>
      <td>To update</td>
      <td>https://www.repeatmasker.org/RepeatModeler/</td>
      <td>Genome annotation</td>
      <td>repeatmodeler</td>
      <td>csbl</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/repeatmodeler</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/repeatmodeler</td>
      <td>2.0.5</td>
      <td></td>
      <td></td>
      <td>Repeat sequence detection</td>
      <td>Sequence composition, complexity and repeats, Sequence composition, complexity and repeats</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>217</td>
      <td>1177</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>retrieve_bold</td>
      <td>retrieve_bold</td>
      <td>Search a list of sequences in BOLD (Barcode of Life Data System) from specified taxa list and markers</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://anaconda.org/conda-forge/r-bold</td>
      <td>Ecology</td>
      <td>retrieve_bold</td>
      <td>ecology</td>
      <td>https://github.com/wpearman1996/MARES_database_pipeline/tree/master</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/retrieve_bold</td>
      <td>1.3.0</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>rnaquast</td>
      <td>rna_quast</td>
      <td>rnaQuast (RNA Quality Assessment Tool) evaluates genome assemblies.</td>
      <td>rnaQUAST</td>
      <td>rnaQUAST</td>
      <td></td>
      <td>rnaQUAST</td>
      <td>Quality assessment tool for de novo transcriptome assemblies.</td>
      <td>De-novo assembly, Transcriptome assembly, Sequence assembly validation</td>
      <td>Sequence assembly, Transcriptomics, RNA-seq</td>
      <td>Up-to-date</td>
      <td>https://github.com/ablab/rnaquast</td>
      <td>Assembly, RNA</td>
      <td>rnaquast</td>
      <td>iuc</td>
      <td>https://git.ufz.de/lehmanju/rnaquast</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/rnaquast</td>
      <td>2.3.0</td>
      <td>rnaquast</td>
      <td>2.3.0</td>
      <td>De-novo assembly, Transcriptome assembly, Sequence assembly validation</td>
      <td>Sequence assembly, Transcriptomics</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>109</td>
      <td>1110</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>salsa2</td>
      <td>salsa</td>
      <td>A tool to scaffold long read assemblies with Hi-C</td>
      <td>SALSA</td>
      <td>SALSA</td>
      <td></td>
      <td>SALSA</td>
      <td>&gt; VERY_LOW CONFIDENCE! | &gt; CORRECT NAME OF TOOL COULD ALSO BE 'chromosome-scale', 'reference-quality', 'Hi-C', 'scaffolder' | Integrating Hi-C links with assembly graphs for chromosome-scale assembly | SALSA: A tool to scaffold long read assemblies with Hi-C data | SALSA: A tool to scaffold long read assemblies with Hi-C | This code is used to scaffold your assemblies using Hi-C data. This version implements some improvements in the original SALSA algorithm. If you want to use the old version, it can be found in the old_salsa branch</td>
      <td>Genome assembly, De-novo assembly, Scaffolding</td>
      <td>Sequence assembly, DNA binding sites, Mapping</td>
      <td>Up-to-date</td>
      <td>https://github.com/marbl/SALSA</td>
      <td>Assembly</td>
      <td>salsa</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/salsa2</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/salsa2</td>
      <td>2.3</td>
      <td>salsa2</td>
      <td>2.3</td>
      <td>Genome assembly, De-novo assembly, Scaffolding</td>
      <td>Sequence assembly, DNA binding sites, Mapping</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>84</td>
      <td>454</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>sample_seqs</td>
      <td>sample_seqs</td>
      <td>Sub-sample sequences files (e.g. to reduce coverage)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/sample_seqs</td>
      <td>Assembly, Fasta Manipulation, Fastq Manipulation, Sequence Analysis</td>
      <td>sample_seqs</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/sample_seqs</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/sample_seqs</td>
      <td>0.2.6</td>
      <td>biopython</td>
      <td>1.70</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>149</td>
      <td>3765</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>samtools_depad</td>
      <td>samtools_depad</td>
      <td>Re-align a SAM/BAM file with a padded reference (using samtools depad)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.htslib.org/</td>
      <td>Assembly, SAM, Sequence Analysis</td>
      <td>samtools_depad</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/samtools_depad</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/samtools_depad</td>
      <td>0.0.5</td>
      <td>samtools</td>
      <td>1.20</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>samtools_depth</td>
      <td>samtools_depth</td>
      <td>Coverage depth via samtools</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.htslib.org/</td>
      <td>Assembly, Sequence Analysis, SAM</td>
      <td>samtools_depth</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/samtools_depth</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/samtools_depth</td>
      <td>0.0.3</td>
      <td>samtools</td>
      <td>1.20</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>296</td>
      <td>4948</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>samtools_idxstats</td>
      <td>samtools_idxstats</td>
      <td>BAM mapping statistics (using samtools idxstats)</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.htslib.org/</td>
      <td>Assembly, Next Gen Mappers, SAM</td>
      <td>samtools_idxstats</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/samtools_idxstats</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/samtools_idxstats</td>
      <td>0.0.6</td>
      <td>samtools</td>
      <td>1.20</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1450</td>
      <td>48426</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>sdmpredictors</td>
      <td>sdmpredictors_list_layers</td>
      <td>Terrestrial and marine predictors for species distribution modelling.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://cran.r-project.org/web/packages/sdmpredictors/index.html</td>
      <td>Ecology</td>
      <td>sdmpredictors</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/sdmpredictors</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/sdmpredictors</td>
      <td>0.2.15</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>seq_filter_by_mapping</td>
      <td>seq_filter_by_mapping</td>
      <td>Filter sequencing reads using SAM/BAM mapping files</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_filter_by_mapping</td>
      <td>Assembly, Fasta Manipulation, Fastq Manipulation, SAM, Sequence Analysis</td>
      <td>seq_filter_by_mapping</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_filter_by_mapping</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_filter_by_mapping</td>
      <td>0.0.8</td>
      <td>biopython</td>
      <td>1.70</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>82</td>
      <td>3784</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>seq_primer_clip</td>
      <td>seq_primer_clip</td>
      <td>Trim off 5' or 3' primers</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_primer_clip</td>
      <td>Assembly, Fasta Manipulation, Text Manipulation</td>
      <td>seq_primer_clip</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_primer_clip</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_primer_clip</td>
      <td>0.0.18</td>
      <td>galaxy_sequence_utils</td>
      <td>1.1.5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>shasta</td>
      <td>shasta</td>
      <td>Fast de novo assembly of long read sequencing data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/chanzuckerberg/shasta</td>
      <td>Assembly, Nanopore</td>
      <td>shasta</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/shasta</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/shasta</td>
      <td>0.6.0</td>
      <td>shasta</td>
      <td>0.12.0</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>154</td>
      <td>763</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>shovill</td>
      <td>shovill</td>
      <td>Faster de novo assembly pipeline based around Spades</td>
      <td>shovill</td>
      <td>shovill</td>
      <td></td>
      <td>shovill</td>
      <td>Shovill is a pipeline for assembly of bacterial isolate genomes from Illumina paired-end reads.  Shovill uses SPAdes at its core, but alters the steps before and after the primary assembly step to get similar results in less time. Shovill also supports other assemblers like SKESA, Velvet and Megahit, so you can take advantage of the pre- and post-processing the Shovill provides with those too.</td>
      <td>Genome assembly</td>
      <td>Genomics, Microbiology, Sequence assembly</td>
      <td>Up-to-date</td>
      <td>https://github.com/tseemann/shovill</td>
      <td>Assembly</td>
      <td>shovill</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/shovill</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/shovill</td>
      <td>1.1.0</td>
      <td>shovill</td>
      <td>1.1.0</td>
      <td>Genome assembly</td>
      <td>Genomics, Microbiology, Sequence assembly</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1008</td>
      <td>41600</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>smudgeplot</td>
      <td>smudgeplot</td>
      <td>Inference of ploidy and heterozygosity structure using whole genome sequencing</td>
      <td>smudgeplots</td>
      <td>smudgeplots</td>
      <td></td>
      <td>Smudgeplots</td>
      <td>Reference-free profiling of polyploid genomes | Inference of ploidy and heterozygosity structure using whole genome sequencing data | Smudgeplots are computed from raw or even better from trimmed reads and show the haplotype structure using heterozygous kmer pairs. For example: | This tool extracts heterozygous kmer pairs from kmer dump files and performs gymnastics with them. We are able to disentangle genome structure by comparing the sum of kmer pair coverages (CovA + CovB) to their relative coverage (CovA / (CovA + CovB)). Such an approach also allows us to analyze obscure genomes with duplications, various ploidy levels, etc | GenomeScope 2.0 and Smudgeplots: Reference-free profiling of polyploid genomes Timothy Rhyker Ranallo-Benavidez, Kamil S. Jaron, Michael C. Schatz bioRxiv 747568; doi: https://doi.org/10.1101/747568</td>
      <td>Sequence trimming, Genotyping, k-mer counting</td>
      <td>Sequence assembly, Genetic variation, Mathematics</td>
      <td>Up-to-date</td>
      <td>https://github.com/KamilSJaron/smudgeplot</td>
      <td>Assembly</td>
      <td>smudgeplot</td>
      <td>galaxy-australia</td>
      <td>https://github.com/galaxyproject/tools-iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/smudgeplot</td>
      <td>0.2.5</td>
      <td>smudgeplot</td>
      <td>0.2.5</td>
      <td>Sequence trimming, Genotyping, k-mer counting</td>
      <td>Sequence assembly, Genetic variation, Mathematics</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>22</td>
      <td>203</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>spades</td>
      <td>spades_biosyntheticspades, spades_coronaspades, spades_metaplasmidspades, metaspades, spades_metaviralspades, spades_plasmidspades, rnaspades, spades_rnaviralspades, spades</td>
      <td>SPAdes is an assembly toolkit containing various assembly pipelines. It implements the following 4 stages: assembly graph construction, k-bimer adjustment, construction of paired assembly graph and contig construction.</td>
      <td>spades</td>
      <td>coronaspades, biosyntheticspades, metaplasmidspades, plasmidspades, metaspades, rnaspades, metaviralspades, rnaviralspades, spades</td>
      <td></td>
      <td>SPAdes</td>
      <td>St. Petersburg genome assembler – is intended for both standard isolates and single-cell MDA bacteria assemblies. SPAdes 3.9 works with Illumina or IonTorrent reads and is capable of providing hybrid assemblies using PacBio, Oxford Nanopore and Sanger reads. Additional contigs can be provided and can be used as long reads.</td>
      <td>Genome assembly</td>
      <td>Sequence assembly</td>
      <td>To update</td>
      <td>https://github.com/ablab/spades</td>
      <td>Assembly, RNA, Metagenomics</td>
      <td>spades</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/spades</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/spades</td>
      <td>3.15.5</td>
      <td>spades</td>
      <td>4.0.0</td>
      <td>Genome assembly</td>
      <td>Sequence assembly</td>
      <td>9</td>
      <td>9</td>
      <td>9</td>
      <td>9</td>
      <td>9</td>
      <td>9</td>
      <td>9</td>
      <td>9</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>3</td>
      <td>8</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>3</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>3</td>
      <td>0</td>
      <td>3547</td>
      <td>72953</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>spaln</td>
      <td>list_spaln_tables, spaln</td>
      <td>Spaln (space-efficient spliced alignment) maps and aligns a set of cDNA or protein sequences onto a whole genomic sequence.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://www.genome.ist.i.kyoto-u.ac.jp/~aln_user/spaln/</td>
      <td>Sequence Analysis, Genome annotation</td>
      <td>spaln</td>
      <td>iuc</td>
      <td>https://github.com/ogotoh/spaln</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/spaln</td>
      <td>2.4.9</td>
      <td>python</td>
      <td></td>
      <td></td>
      <td></td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>34</td>
      <td>446</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>spocc</td>
      <td>spocc_occ</td>
      <td>Get species occurences data</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://cran.r-project.org/web/packages/spocc/index.html</td>
      <td>Ecology</td>
      <td>spocc_occ</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/spocc</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/spocc</td>
      <td>1.2.2</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>56</td>
      <td>769</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>srs_tools</td>
      <td>srs_diversity_maps, srs_global_indices, srs_process_data, srs_spectral_indices, srs_pca, srs_preprocess_s2, srs_metadata</td>
      <td>Compute biodiversity indicators for remote sensing data from Sentinel 2</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Ecology</td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/Marie59/Sentinel_2A/srs_tools</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/srs_tools</td>
      <td>0.0.1</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>4</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>4</td>
      <td>0</td>
      <td>7</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>7</td>
      <td>0</td>
      <td>0</td>
      <td>21</td>
      <td>225</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>stoc</td>
      <td>stoceps_filteringsp, stoceps_glm, stoceps_glm_group, stoceps_maketablecarrer, stoceps_trend_indic</td>
      <td>Tools to analyse STOC data.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Ecology</td>
      <td>stoceps</td>
      <td>ecology</td>
      <td>https://github.com/Alanamosse/Galaxy-E/tree/stoctool/tools/stoc</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/stoc</td>
      <td>0.0.2</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>10</td>
      <td>325</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>taxonomy_filter_refseq</td>
      <td>taxonomy_filter_refseq</td>
      <td>Filter RefSeq by taxonomy</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/pvanheus/ncbitaxonomy</td>
      <td>Sequence Analysis, Genome annotation</td>
      <td>taxonomy_filter_refseq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/taxonomy_filter_refseq</td>
      <td>0.3.0</td>
      <td>rust-ncbitaxonomy</td>
      <td>1.0.7</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>taxonomy_krona_chart</td>
      <td>taxonomy_krona_chart</td>
      <td>Krona pie chart from taxonomic profile</td>
      <td>krona</td>
      <td>krona</td>
      <td></td>
      <td>Krona</td>
      <td>Krona creates interactive HTML5 charts of hierarchical data (such as taxonomic abundance in a metagenome).</td>
      <td>Visualisation</td>
      <td>Metagenomics</td>
      <td>To update</td>
      <td>http://sourceforge.net/projects/krona/</td>
      <td>Assembly</td>
      <td>taxonomy_krona_chart</td>
      <td>crs4</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/taxonomy_krona_chart</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/taxonomy_krona_chart</td>
      <td>2.7.1+galaxy0</td>
      <td>krona</td>
      <td>2.8.1</td>
      <td>Visualisation</td>
      <td>Metagenomics</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1801</td>
      <td>27426</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>telescope</td>
      <td>telescope_assign</td>
      <td>Single locus resolution of Transposable ELEment expression.</td>
      <td>Telescope-expression</td>
      <td>Telescope-expression</td>
      <td></td>
      <td>Telescope</td>
      <td>Telescope is a tool for the characterization of the retrotranscriptome by accurate estimation of transposable element expression and the quantification of transposable element expression using RNA-seq.It can be used for Statistical Performance of TE Quantification Methods.All scripts needed to examine the sensitivity and biases of computational approaches for quantifying TE expression: 1) unique counts, 2) best counts, 3) RepEnrich, 4) TEtranscripts, 5) RSEM, 6) SalmonTE, and 7) Telescope.</td>
      <td>Essential dynamics, Sequence trimming, RNA-Seq quantification, Expression analysis, Read mapping</td>
      <td>RNA-Seq, Transcriptomics, Mapping, Gene transcripts, Sequence assembly</td>
      <td>Up-to-date</td>
      <td>https://github.com/mlbendall/telescope/</td>
      <td>Genome annotation</td>
      <td>telescope_assign</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/telescope</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/telescope</td>
      <td>1.0.3</td>
      <td>telescope</td>
      <td>1.0.3</td>
      <td>Essential dynamics, Sequence trimming, RNA-Seq quantification, Expression analysis, Read mapping</td>
      <td>RNA-Seq, Transcriptomics, Mapping, Gene transcripts, Sequence assembly</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>tgsgapcloser</td>
      <td>tgsgapcloser</td>
      <td>TGS-GapCloser uses error-prone long reads or preassembled contigs to fill N-gap in the genome assembly.</td>
      <td>TGS-GapCloser</td>
      <td>TGS-GapCloser</td>
      <td></td>
      <td>TGS-GapCloser</td>
      <td>TGS-GapCloser is a fast and accurately passing through the Bermuda in large genome using error-prone third-generation long reads.</td>
      <td>Genome assembly, Read mapping, Scaffolding, Localised reassembly</td>
      <td>Sequencing, Sequence assembly, Phylogeny, Transcription factors and regulatory sites, Mapping</td>
      <td>To update</td>
      <td>https://github.com/BGI-Qingdao/TGS-GapCloser</td>
      <td>Assembly</td>
      <td>tgsgapcloser</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/tgsgapcloser</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/tgsgapcloser</td>
      <td>1.0.3</td>
      <td>tgsgapcloser</td>
      <td>1.2.1</td>
      <td>Genome assembly, Read mapping, Scaffolding, Localised reassembly</td>
      <td>Sequencing, Sequence assembly, Phylogeny, Transcription factors and regulatory sites, Mapping</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>36</td>
      <td>460</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>transit</td>
      <td>gff_to_prot, transit_gumbel, transit_hmm, transit_resampling, transit_tn5gaps</td>
      <td>TRANSIT</td>
      <td>transit</td>
      <td>transit</td>
      <td></td>
      <td>TRANSIT</td>
      <td>A tool for the analysis of Tn-Seq data. It provides an easy to use graphical interface and access to three different analysis methods that allow the user to determine essentiality in a single condition as well as between conditions.</td>
      <td>Transposon prediction</td>
      <td>DNA, Sequencing, Mobile genetic elements</td>
      <td>To update</td>
      <td>https://github.com/mad-lab/transit/</td>
      <td>Genome annotation</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/transit/</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/transit</td>
      <td>3.0.2</td>
      <td>transit</td>
      <td>3.2.3</td>
      <td>Transposon prediction</td>
      <td>DNA, Sequencing, Mobile genetic elements</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>84</td>
      <td>907</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>trycycler</td>
      <td>trycycler_cluster, trycycler_consensus, trycycler_partition, trycycler_reconcile_msa, trycycler_subsample</td>
      <td>Trycycler toolkit wrappers</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/rrwick/Trycycler</td>
      <td>Assembly</td>
      <td>trycycler</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/trycycler</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/trycycler</td>
      <td>0.5.5</td>
      <td>trycycler</td>
      <td>0.5.5</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>5</td>
      <td>5</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>147</td>
      <td>8942</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>tsebra</td>
      <td>tsebra</td>
      <td>This tool has been developed to combine BRAKER predictions.</td>
      <td>tsebra</td>
      <td>tsebra</td>
      <td></td>
      <td>TSEBRA</td>
      <td>TSEBRA is a combiner tool that selects transcripts from gene predictions based on the support by extrisic evidence in form of introns and start/stop codons. It was developed to combine BRAKER1 and BRAKER2 predicitons to increase their accuracies.</td>
      <td>Homology-based gene prediction, Alternative splicing prediction</td>
      <td>Gene expression, RNA-Seq, Gene transcripts, Model organisms</td>
      <td>To update</td>
      <td>https://github.com/Gaius-Augustus/TSEBRA</td>
      <td>Genome annotation</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/tsebra</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/tsebra</td>
      <td>1.1.2.4</td>
      <td>tsebra</td>
      <td>1.1.2.5</td>
      <td>Homology-based gene prediction, Alternative splicing prediction</td>
      <td>Gene expression, RNA-Seq, Gene transcripts, Model organisms</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>5</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>unicycler</td>
      <td>unicycler</td>
      <td>Unicycler is a hybrid assembly pipeline for bacterial genomes.</td>
      <td>unicycler</td>
      <td>unicycler</td>
      <td></td>
      <td>Unicycler</td>
      <td>A tool for assembling bacterial genomes from a combination of short (2nd generation) and long (3rd generation) sequencing reads.</td>
      <td>Genome assembly, Aggregation</td>
      <td>Microbiology, Genomics, Sequencing, Sequence assembly</td>
      <td>Up-to-date</td>
      <td>https://github.com/rrwick/Unicycler</td>
      <td>Assembly</td>
      <td>unicycler</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/unicycler</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/unicycler</td>
      <td>0.5.1</td>
      <td>unicycler</td>
      <td>0.5.1</td>
      <td>Genome assembly, Aggregation</td>
      <td>Microbiology, Genomics, Sequencing, Sequence assembly</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1558</td>
      <td>65732</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>velvet</td>
      <td>velvetg, velveth</td>
      <td>de novo genomic assembler specially designed for short read sequencing technologies</td>
      <td>velvet</td>
      <td>velvet</td>
      <td></td>
      <td>Velvet</td>
      <td>A de novo genomic assembler specially designed for short read sequencing technologies, such as Solexa or 454 or SOLiD.</td>
      <td>Formatting, De-novo assembly</td>
      <td>Sequence assembly</td>
      <td>To update</td>
      <td>https://www.ebi.ac.uk/~zerbino/velvet/</td>
      <td>Assembly</td>
      <td>velvet</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/velvet</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/velvet</td>
      <td></td>
      <td>velvet</td>
      <td>1.2.10</td>
      <td>Formatting, De-novo assembly</td>
      <td>Sequence assembly</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
      <td>0</td>
      <td>2083</td>
      <td>25420</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>velvet_optimiser</td>
      <td>velvetoptimiser</td>
      <td>Automatically optimize Velvet assemblies</td>
      <td>velvetoptimiser</td>
      <td>velvetoptimiser</td>
      <td></td>
      <td>VelvetOptimiser</td>
      <td>This tool is designed to run as a wrapper script for the Velvet assembler (Daniel Zerbino, EBI UK) and to assist with optimising the assembly.</td>
      <td>Optimisation and refinement, Sequence assembly</td>
      <td>Genomics, Sequence assembly</td>
      <td>To update</td>
      <td></td>
      <td>Assembly</td>
      <td>velvetoptimiser</td>
      <td>simon-gladman</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/velvetoptimiser</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/velvet_optimiser</td>
      <td>2.2.6+galaxy2</td>
      <td>velvet</td>
      <td>1.2.10</td>
      <td>Optimisation and refinement, Sequence assembly</td>
      <td>Genomics, Sequence assembly</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>255</td>
      <td>4741</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>verkko</td>
      <td>verkko</td>
      <td>Telomere-to-telomere assembly pipeline</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/marbl/verkko</td>
      <td>Assembly</td>
      <td>verkko</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/verkko</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/verkko</td>
      <td>1.3.1</td>
      <td>verkko</td>
      <td>2.1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>9</td>
      <td>22</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>vigiechiro</td>
      <td>vigiechiro_bilanenrichipf, vigiechiro_bilanenrichirp, vigiechiro_idcorrect_2ndlayer, vigiechiro_idvalid</td>
      <td>Tools created by the vigiechiro team to analyses and identify chiro sounds files.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://www.vigienature-ecole.fr/les-observatoires/le-protocole-vigie-chiro</td>
      <td>Ecology</td>
      <td>vigiechiro</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/vigiechiro</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/vigiechiro</td>
      <td>0.1.1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>4</td>
      <td>4</td>
      <td>0</td>
      <td>351</td>
      <td>12203</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>wtdbg</td>
      <td>wtdbg</td>
      <td>WTDBG is a fuzzy Bruijn graph (FBG) approach to long noisy reads assembly.</td>
      <td>wtdbg2</td>
      <td>wtdbg2</td>
      <td></td>
      <td>wtdbg2</td>
      <td>Wtdbg2 is a de novo sequence assembler for long noisy reads produced by PacBio or Oxford Nanopore Technologies (ONT). It assembles raw reads without error correction and then builds the consensus from intermediate assembly output. Wtdbg2 is able to assemble the human and even the 32Gb Axolotl genome at a speed tens of times faster than CANU and FALCON while producing contigs of comparable base accuracy.</td>
      <td>Genome assembly, De-novo assembly</td>
      <td>Sequence assembly, Sequencing</td>
      <td>Up-to-date</td>
      <td>https://github.com/ruanjue/wtdbg2</td>
      <td>Assembly</td>
      <td>wtdbg</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/wtdbg</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/wtdbg</td>
      <td>2.5</td>
      <td>wtdbg</td>
      <td>2.5</td>
      <td>Genome assembly, De-novo assembly</td>
      <td>Sequence assembly, Sequencing</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>116</td>
      <td>1660</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>xarray</td>
      <td>timeseries_extraction, xarray_coords_info, xarray_mapplot, xarray_metadata_info, xarray_netcdf2netcdf, xarray_select</td>
      <td>xarray (formerly xray) is an open source project and Python package that makes working withlabelled multi-dimensional arrays simple, efficient, and fun!xarray integrates with Dask to support parallel computations and streaming computation on datasetsthat don’t fit into memory.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://xarray.pydata.org</td>
      <td>Ecology</td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/data_manipulation/xarray/</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/data_manipulation/xarray</td>
      <td>2022.3.0</td>
      <td>xarray</td>
      <td></td>
      <td></td>
      <td></td>
      <td>5</td>
      <td>2</td>
      <td>6</td>
      <td>5</td>
      <td>5</td>
      <td>2</td>
      <td>6</td>
      <td>5</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>6</td>
      <td>2</td>
      <td>0</td>
      <td>175</td>
      <td>5876</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>yahs</td>
      <td>yahs</td>
      <td>Yet Another Hi-C scaffolding tool</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/c-zhou/yahs</td>
      <td>Assembly</td>
      <td>yahs</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/yahs</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/yahs</td>
      <td>1.2a.2</td>
      <td>yahs</td>
      <td>1.2a.2</td>
      <td></td>
      <td></td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>1</td>
      <td>1</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>64</td>
      <td>344</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>zoo_project_ogc_api_processes</td>
      <td>zoo_project_ogc_api_processes</td>
      <td>This tool is a wrapper for OGC API Processes (OTB) coming from the Zoo Project (https://zoo-project.github.io/docs/intro.html) and was created using the OGC-API-Process2Galaxy tool (https://github.com/AquaINFRA/OGC-API-Process2Galaxy). Check the README in the repository for more information.</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/AquaINFRA/galaxy</td>
      <td>Ecology</td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/ogc_api_processes_wrapper</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/zoo_project_ogc_api_processes</td>
      <td>0.1.0</td>
      <td>r-base</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>1</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
      <td></td>
    </tr>
  </tbody>
</table>

</div>