<script src="https://code.jquery.com/jquery-3.7.0.js"></script>
<script src="https://cdn.datatables.net/1.13.6/js/jquery.dataTables.min.js"></script>
<script src="https://cdn.datatables.net/searchbuilder/1.6.0/js/dataTables.searchBuilder.min.js"></script>
<script src="https://cdn.datatables.net/datetime/1.5.1/js/dataTables.dateTime.min.js"></script>

<script src="https://cdn.datatables.net/responsive/2.5.0/js/dataTables.responsive.min.js"></script>

<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/responsive/2.2.3/css/responsive.dataTables.min.css">

<link rel="stylesheet" href="https://cdn.datatables.net/1.13.6/css/jquery.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/searchbuilder/1.6.0/css/searchBuilder.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/datetime/1.5.1/css/dataTables.dateTime.min.css">

<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" />
<script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>

<style type="text/css">
  .tooltip {
    font-family: Georgia;
    font-size: 18px;
  }
  .tooltip .tooltip-inner {
    background-color: #00FFFF;
    color: #000000;
    min-width: 400px;
  }
  </style>

<script>
    $(document).ready(function() {
    $('#dataframe').DataTable( {
        "sScrollX": "100%",
        "dom": "Qlfrtip",

        responsive: {
            details: {
                type: 'column'
            }
        },
        "bAutoWidth": false,
        columnDefs: [ {
            className: 'control',
            orderable: false,
            targets:   [ 0 ],
            width: 100,
        },
        { type: 'natural', targets: 1 } // allow normal sorting
        ],

    });
});
</script>

<div class="display nowrap" style="width:80%" width="100%">

<table class="dataframe display nowrap" id="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th>Expand</th>
      <th>Topic</th>
      <th>Title</th>
      <th>Link</th>
      <th>EDAM topic</th>
      <th>EDAM operation</th>
      <th>Creation</th>
      <th>Last modification</th>
      <th>Version</th>
      <th>Tutorial</th>
      <th>Slides</th>
      <th>Video</th>
      <th>Workflows</th>
      <th>Tools</th>
      <th>Servers with precise tool versions</th>
      <th>Servers with tool but different versions</th>
      <th>Feedback number</th>
      <th>Feedback mean note</th>
      <th>Visitors</th>
      <th>Page views</th>
      <th>Visit duration</th>
      <th>Video views</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Genome Assembly Quality Control</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/assembly-quality-control/tutorial.html</td>
      <td>Biodiversity, Sequence assembly, Data quality management</td>
      <td>De-novo assembly, Scaffolding, Sequence assembly validation, Phasing, k-mer counting, Visualisation, Genome assembly</td>
      <td>2022-08-26</td>
      <td>2025-06-04</td>
      <td>6</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>meryl, busco, chromeister, merqury, quast</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>2148863</td>
      <td>4898160</td>
      <td>4.400000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Large genome assembly and polishing</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/largegenome/tutorial.html</td>
      <td>Genomics, Sequence assembly, Biodiversity</td>
      <td>De-novo assembly, Sequence assembly visualisation, Sequence assembly validation, Sequence assembly, Variant calling, Cross-assembly, Base-calling, Sequencing quality control, Sequence contamination filtering, Statistical calculation, Scatter plot plotting, Visualisation, Pairwise sequence alignment, Sequence composition calculation, Box-Whisker plot plotting, Validation, Mapping assembly, Genome assembly</td>
      <td>2023-03-06</td>
      <td>2025-06-04</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>medaka_consensus_pipeline, meryl, busco, flye, fastp, multiqc, fasta-stats, racon, fastqc, nanoplot, genomescope, bandage_image, barchart_gnuplot, quast, minimap2</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>7444</td>
      <td>9656</td>
      <td>2.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>ERGA post-assembly QC</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/ERGA-post-assembly-QC/tutorial.html</td>
      <td>Genomics, Sequence assembly, Biodiversity</td>
      <td>De-novo assembly, Data handling, Phasing, Sequence assembly validation, Pairwise sequence alignment, Visualisation, Sequence alignment, Data parsing, Scaffolding, Genotyping, Sequence alignment analysis, Data deposition, k-mer counting, Genome assembly, Sequence trimming, Sequence assembly visualisation</td>
      <td>2023-06-23</td>
      <td>2025-06-04</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>samtools_stats, merqury, bandage_image, busco, pretext_snapshot, gfastats, pretext_map, collapse_dataset, CONVERTER_gz_to_uncompressed, tp_find_and_replace, __EXTRACT_DATASET__, genomescope, bellerophon, bwa_mem2, meryl, blobtoolkit, interactive_tool_blobtoolkit, smudgeplot, bg_diamond, minimap2</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>1986</td>
      <td>2566</td>
      <td>2.033333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Chloroplast genome assembly</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/chloroplast-assembly/tutorial.html</td>
      <td></td>
      <td>Coding region prediction, Gene prediction, Scatter plot plotting, Sequence alignment, Generation, Analysis, Genome indexing, Genome visualisation, Genome assembly, De-novo assembly, Sequence assembly visualisation, Sequence assembly, Cross-assembly, Genome annotation, Read alignment, Box-Whisker plot plotting, Read mapping, Mapping assembly</td>
      <td>2020-12-04</td>
      <td>2025-05-15</td>
      <td>21</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>jbrowse, nanoplot, upload1, fasta-stats, pilon, bwa_mem, prokka, bandage_image, flye, bandage_info</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>8083</td>
      <td>12576</td>
      <td>2.816667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Assembly of the mitochondrial genome from PacBio HiFi reads</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/mitochondrion-assembly/tutorial.html</td>
      <td>Genomics, Sequence assembly, Biodiversity, Mitochondria</td>
      <td></td>
      <td>2024-09-03</td>
      <td>2025-06-04</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>compress_file, mitohifi</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>882</td>
      <td>1072</td>
      <td>1.866667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Decontamination of a genome assembly</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/assembly-decontamination/tutorial.html</td>
      <td>Genomics, Sequence assembly, Biodiversity</td>
      <td>Data handling, Taxonomic classification</td>
      <td>2024-09-04</td>
      <td>2025-06-04</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>kraken2, parse_mito_blast, Filter1, ncbi_dustmasker_wrapper, Cut1, ncbi_blastn_wrapper, tp_sed_tool, gfastats, tp_cat</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>1675</td>
      <td>2069</td>
      <td>2.450000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Unicycler Assembly</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/unicycler-assembly/tutorial.html</td>
      <td>Sequence assembly, Genomics, Microbiology</td>
      <td>Coding region prediction, Gene prediction, Statistical calculation, Sequence assembly validation, Sequencing quality control, Visualisation, Genome annotation, Aggregation, Sequence composition calculation, Validation, Genome assembly</td>
      <td>2017-10-11</td>
      <td>2025-05-15</td>
      <td>24</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>unicycler, multiqc, fastqc, prokka, quast</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>Galaxy@AuBi, GalaxyTrakr, MISSISSIPPI, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>3062576</td>
      <td>7874211</td>
      <td>4.416667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Unicycler assembly of SARS-CoV-2 genome with preprocessing to remove human genome reads</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/assembly-with-preprocessing/tutorial.html</td>
      <td></td>
      <td>Data handling, Sequence assembly visualisation, Sequence contamination filtering, Sequencing quality control, Pairwise sequence alignment, Scatter plot plotting, Aggregation, Sequence file editing, Box-Whisker plot plotting, Validation, Read mapping, Genome assembly</td>
      <td>2020-04-21</td>
      <td>2025-05-15</td>
      <td>14</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>unicycler, samtools_stats, collapse_dataset, fastp, fasta_filter_by_length, multiqc, bowtie2, seqtk_sample, __ZIP_COLLECTION__, samtools_fastx, nanoplot, fasterq_dump, bandage_image, samtools_view, bandage_info, minimap2</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td>Galaxy@AuBi, MISSISSIPPI, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>3055563</td>
      <td>7861983</td>
      <td>5.550000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Vertebrate genome assembly using HiFi, Bionano and Hi-C data - Step by Step</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/vgp_genome_assembly/tutorial.html</td>
      <td>Genomics, Sequence assembly, Biodiversity</td>
      <td>De-novo assembly, Data handling, Mapping, Read binning, Phasing, Sequence assembly validation, Pairwise sequence alignment, Read pre-processing, Visualisation, Sequence alignment, Scaffolding, Primer removal, k-mer counting, Sequence trimming, Genome assembly</td>
      <td>2021-06-04</td>
      <td>2025-06-04</td>
      <td>57</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>purge_dups, yahs, gfa_to_fa, merqury, tp_grep_tool, tp_cut_tool, busco, collection_column_join, bionano_scaffold, pretext_snapshot, sort1, hifiasm, gfastats, pretext_map, bedtools_bamtobed, collapse_dataset, cat1, tp_find_and_replace, bellerophon, genomescope, salsa, bwa_mem2, meryl, Convert characters1, cutadapt, Add_a_column1, quast, minimap2</td>
      <td></td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>15966</td>
      <td>24071</td>
      <td>1.966667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Genome assembly using PacBio data</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/hifi-assembly/tutorial.html</td>
      <td>Genomics, Sequence assembly, Biodiversity</td>
      <td>De-novo assembly, Sequence assembly validation, Cross-assembly, Visualisation, Mapping assembly, Genome assembly</td>
      <td>2021-11-29</td>
      <td>2025-06-04</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>busco, flye, gfa_to_fa, hifiasm, fasta-stats, quast</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>844</td>
      <td>992</td>
      <td>1.500000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Using the VGP workflows to assemble a vertebrate genome with HiFi and Hi-C data</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/vgp_workflow_training/tutorial.html</td>
      <td>Sequence assembly, Chromosome conformation capture</td>
      <td>Visualisation, Sequence assembly validation</td>
      <td>2022-04-06</td>
      <td>2025-09-09</td>
      <td>26</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>quast</td>
      <td>Galaxy@AuBi, GalaxyTrakr, MISSISSIPPI, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3541</td>
      <td>4887</td>
      <td>2.666667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Climate</td>
      <td>Getting your hands-on earth data</td>
      <td>https://training.galaxyproject.org//topics/climate/tutorials/earth_system/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-10-04</td>
      <td>2025-05-10</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>c3s, spocc_occ, argo_getdata, obis_data</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>109</td>
      <td>143</td>
      <td>2.100000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Sentinel 2 biodiversity</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/remote-sensing/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-04-13</td>
      <td>2025-06-03</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>srs_global_indices, srs_diversity_maps, srs_preprocess_s2, srs_pca, srs_spectral_indices, srs_process_data</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>616</td>
      <td>790</td>
      <td>1.616667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Compute and analyze biodiversity metrics with PAMPA toolsuite</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/PAMPA-toolsuite-tutorial/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-11-19</td>
      <td>2025-09-03</td>
      <td>23</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>tp_cut_tool, Filter1, datamash_transpose, pampa_plotglm, pampa_presabs, pampa_glmcomm, regexColumn1, regex1, mergeCols1, tp_sort_header_tool, pampa_communitymetrics, pampa_glmsp, tp_easyjoin_tool, Count1, tp_cat</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>1107</td>
      <td>1470</td>
      <td>1.933333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>RAD-Seq to construct genetic maps</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/genetic-map-rad-seq/tutorial.html</td>
      <td></td>
      <td>Data handling</td>
      <td>2017-02-14</td>
      <td>2025-06-03</td>
      <td>13</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>stacks_genotypes, stacks_denovomap</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>892</td>
      <td>1251</td>
      <td>1.333333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Champs blocs indicators</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/champs-blocs/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-01-04</td>
      <td>2025-06-03</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>cb_qecb, cb_dissim, cb_eco, cb_ivr, cb_div</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>180</td>
      <td>329</td>
      <td>1.816667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Species distribution modeling</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/species-distribution-modeling/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-07-04</td>
      <td>2025-06-03</td>
      <td>18</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tp_cut_tool, spocc_occ, interactive_tool_wallace, Filter1, tabular_to_csv, tp_find_and_replace</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3201</td>
      <td>4829</td>
      <td>1.150000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>RAD-Seq de-novo data analysis</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/de-novo-rad-seq/tutorial.html</td>
      <td></td>
      <td>Data handling, Sequence composition calculation, Statistical calculation, Sequencing quality control</td>
      <td>2017-02-14</td>
      <td>2025-06-03</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>stacks_procrad, stacks_populations, fastqc, stacks_denovomap</td>
      <td>UseGalaxy.be, UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>2695</td>
      <td>4287</td>
      <td>2.083333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Regional GAM</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/regionalGAM/tutorial.html</td>
      <td>Ecology</td>
      <td>Visualisation</td>
      <td>2019-07-04</td>
      <td>2025-06-03</td>
      <td>18</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>regionalgam_plot_trend, Filter1, tab2csv_R, regionalgam_ab_index, regexColumn1, regionalgam_glmmpql, regionalgam_autocor_acf, mergeCols1, Remove beginning1, Paste1, tp_awk_tool, regionalgam_flight_curve, Count1, ggplot2_point, regionalgam_gls, flight-curve, csv2tab_R</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>350</td>
      <td>466</td>
      <td>3.400000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Metabarcoding/eDNA through Obitools</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/Obitools-metabarcoding/tutorial.html</td>
      <td></td>
      <td>Sequence analysis, Sequence composition calculation, Statistical calculation, Sequencing quality control</td>
      <td>2022-02-28</td>
      <td>2025-06-03</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>obi_ngsfilter, Filter1, unzip, seq_filter_by_id, obi_illumina_pairend, Cut1, join1, ncbi_blastn_wrapper, obi_grep, obi_stat, obi_tab, fastq_groomer, fastqc, wc_gnu, obi_uniq, obi_annotate, obi_clean</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>4532</td>
      <td>6289</td>
      <td>1.933333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Obis marine indicators</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/obisindicators/tutorial.html</td>
      <td>Biodiversity, Ecology, Data mining</td>
      <td></td>
      <td>2023-04-11</td>
      <td>2025-06-03</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tp_cut_tool, obisindicators, obis_data</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>318</td>
      <td>398</td>
      <td>2.016667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Preparing genomic data for phylogeny reconstruction</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/phylogeny-data-prep/tutorial.html</td>
      <td></td>
      <td>Genetic variation analysis, Sequence clustering, Sequence analysis, Sequence assembly validation, Multiple sequence alignment, Genome annotation</td>
      <td>2022-10-21</td>
      <td>2025-06-03</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>proteinortho, Filter1, busco, collapse_dataset, clustalw, clipkit, glimmer_gbk_to_orf, regex1, tp_replace_in_line, proteinortho_grab_proteins, funannotate_predict, phykit_alignment_based, repeatmasker_wrapper</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2228</td>
      <td>3071</td>
      <td>2.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Ecoregionalization workflow tutorial</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/Ecoregionalization_tutorial/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-09-16</td>
      <td>2025-06-03</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tp_cut_tool, ecoregion_brt_analysis, ecoregion_clara_cluster, ecoregion_cluster_estimate, ecoregion_taxa_seeker, unzip, regexColumn1, ecoregion_eco_map, ecoregion_GeoNearestNeighbor, mergeCols1, filter_tabular, interactive_tool_jupyter_notebook</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>265</td>
      <td>352</td>
      <td>2.900000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Life Traits Ecoregionalization workflow</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/ecoregiolifetraits/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-06-02</td>
      <td>2025-06-10</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>tp_cut_tool, ecoregion_brt_analysis, Filter1, ecoregion_taxa_seeker, unzip, regexColumn1, ecoregion_eco_map, ecoregion_GeoNearestNeighbor, column_remove_by_header, WormsMeasurements, ClaraGuess</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>42</td>
      <td>45</td>
      <td>2.183333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>RAD-Seq Reference-based data analysis</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/ref-based-rad-seq/tutorial.html</td>
      <td></td>
      <td>Data handling, Statistical calculation, Sequencing quality control, Sequence alignment, Sequence composition calculation, Generation, Validation, Genome indexing, Read mapping</td>
      <td>2017-02-14</td>
      <td>2025-11-06</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>stacks_populations, Filter1, bwa_wrapper, cat1, multiqc, regex_replace, Summary_Statistics1, stacks_refmap, Cut1, tp_replace_in_line, Convert characters1, tp_sort_header_tool, fastqc, stacks_procrad, addValue, Count1, Grep1, bwa</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3150</td>
      <td>4904</td>
      <td>1.816667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Creating FAIR Quality assessment reports and draft of Data Papers from EML metadata with MetaShRIMPS</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/Metashrimps_tutorial/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-01-22</td>
      <td>2025-06-03</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>interactive_tool_metashrimps</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>111</td>
      <td>121</td>
      <td>1.550000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Taxonomic Analysis of eDNA</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/eDNA-taxonomic-analysis/tutorial.html</td>
      <td></td>
      <td>Sequencing quality control, Sequence contamination filtering</td>
      <td>2025-06-02</td>
      <td>2025-06-03</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>fastp, Count1, cshl_fastq_to_fasta, ncbi_blastn_wrapper</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>Galaxy@AuBi, GalaxyTrakr, MISSISSIPPI, UseGalaxy.be, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>467</td>
      <td>588</td>
      <td>0.716667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Cleaning GBIF data for the use in Ecology</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/gbif_cleaning/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-10-28</td>
      <td>2025-06-03</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>spocc_occ, Filter1, gdal_ogr2ogr, Summary_Statistics1, Count1</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.no</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>650</td>
      <td>805</td>
      <td>0.916667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Marine Omics identifying biosynthetic gene clusters</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/marine_omics_bgc/tutorial.html</td>
      <td></td>
      <td>Protein feature detection, Sequence motif recognition, Genome annotation</td>
      <td>2024-08-19</td>
      <td>2025-06-03</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>regex1, interproscan, SanntiS_marine, sanntis_marine, prodigal</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>184</td>
      <td>217</td>
      <td>2.483333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Biodiversity data exploration</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/biodiversity-data-exploration/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-03-01</td>
      <td>2025-06-03</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>tp_cut_tool, ecology_stat_presence_abs, regexColumn1, ecology_beta_diversity, ecology_link_between_var, Remove beginning1, ecology_presence_abs_abund, ecology_homogeneity_normality, tool_anonymization</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1630</td>
      <td>2279</td>
      <td>2.733333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Audio data annotation with NEAL (Nature + Energy Audio labeler)</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/NEAL_tutorial/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-10-14</td>
      <td>2025-11-06</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>13</td>
      <td>16</td>
      <td>0.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Cleaning GBIF data using OpenRefine</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/openrefine_gbif/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-01-21</td>
      <td>2025-06-03</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>interactive_tool_openrefine, regex1</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>160</td>
      <td>196</td>
      <td>1.083333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>QGIS Web Feature Services</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/QGIS_Web_Feature_Services/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-10-11</td>
      <td>2025-06-03</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>interactive_tool_qgis</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>278</td>
      <td>319</td>
      <td>1.650000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Comparison of two annotation tools - Helixer and Braker3</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/comparison-braker-helixer-annotation/tutorial.html</td>
      <td>Sequence assembly, Genomics</td>
      <td>Sequence assembly validation, Gene prediction, Differential protein expression profiling, Sequence annotation, Sequence alignment, Genome annotation, Genome visualisation</td>
      <td>2025-06-20</td>
      <td>2025-06-25</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>jbrowse, braker3, busco, rna_star, omark, gffread, helixer</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>349</td>
      <td>421</td>
      <td>1.816667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Masking repeats with RepeatMasker</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/repeatmasker/tutorial.html</td>
      <td>Sequence composition, complexity and repeats, Genomics</td>
      <td>Mapping, Editing, RNA-Seq analysis, Repeat sequence detection, Genome annotation</td>
      <td>2021-11-29</td>
      <td>2025-06-03</td>
      <td>22</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>repeatmodeler, red, repeatmasker_wrapper, bedtools_maskfastabed</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>10075</td>
      <td>12836</td>
      <td>1.383333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Refining Genome Annotations with Apollo (prokaryotes)</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/apollo/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis</td>
      <td>Genome visualisation</td>
      <td>2021-06-04</td>
      <td>2025-05-15</td>
      <td>22</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>jbrowse, iframe, list_organism, create_account, create_or_update</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2824498</td>
      <td>7075516</td>
      <td>5.133333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Genome annotation with Braker3</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/braker3/tutorial.html</td>
      <td>Sequence assembly, Genomics, Biodiversity, Zoology, Plant biology</td>
      <td>Sequence annotation, Sequence assembly validation, Gene prediction, Differential protein expression profiling, Genome visualisation, Sequence alignment, Genome annotation</td>
      <td>2025-03-17</td>
      <td>2025-06-20</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>jbrowse, busco, rna_star, omark, fasta-stats, gffread, braker3</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1348</td>
      <td>1670</td>
      <td>1.866667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Genome annotation with Helixer</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/helixer/tutorial.html</td>
      <td>Biodiversity, Genomics, Zoology, Plant biology</td>
      <td>Genome visualisation, Sequence assembly validation, Gene prediction, Differential protein expression profiling, Sequence annotation, Genome annotation</td>
      <td>2024-08-21</td>
      <td>2025-06-04</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>jbrowse, busco, omark, jcvi_gff_stats, gffread, helixer</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1215</td>
      <td>1451</td>
      <td>1.716667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Genome annotation with Funannotate</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/funannotate/tutorial.html</td>
      <td>Gene and protein families</td>
      <td>Formatting, Sequence assembly validation, Fold recognition, Homology-based gene prediction, Sequence annotation, Query and retrieval, Information extraction, Sequence alignment, Genome annotation, Sequence motif recognition, Protein feature detection, Genome visualisation</td>
      <td>2021-11-29</td>
      <td>2025-06-10</td>
      <td>26</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>jbrowse, busco, aegean_parseval, eggnog_mapper, rna_star, interproscan, funannotate_predict, picard_MergeSamFiles, funannotate_compare, funannotate_annotate</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>10705</td>
      <td>15155</td>
      <td>1.950000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Comparative gene analysis in unannotated genomes</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/gene-centric/tutorial.html</td>
      <td>Genomics, Gene and protein families, Sequence analysis, Phylogeny, Comparative genomics</td>
      <td>De-novo assembly, Coding region prediction, Sequence alignment analysis, Multiple sequence alignment, Transcriptome assembly, Phylogenetic tree generation, Database search</td>
      <td>2022-09-08</td>
      <td>2025-06-03</td>
      <td>10</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>bg_diamond_view, bg_diamond_makedb, Filter1, collapse_dataset, cat1, tab2fasta, rapidnj, regexColumn1, Cut1, join1, tp_split_on_column, rbc_mafft, gops_intersect_1, orfipy, Add_a_column1, bg_diamond</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>1850</td>
      <td>2601</td>
      <td>3.066667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Functional annotation of protein sequences</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/functional/tutorial.html</td>
      <td>Function analysis, Protein sites, features and motifs</td>
      <td>Sequence alignment analysis, Protein feature detection, Fold recognition, Homology-based gene prediction, Sequence motif recognition, Query and retrieval, Information extraction, Genome annotation</td>
      <td>2022-07-20</td>
      <td>2025-06-03</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>eggnog_mapper, bg_diamond, interproscan, ncbi_blastp_wrapper</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>4256</td>
      <td>5710</td>
      <td>1.683333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Refining Genome Annotations with Apollo (eukaryotes)</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/apollo-euk/tutorial.html</td>
      <td>Genomics</td>
      <td>Genome visualisation</td>
      <td>2022-08-22</td>
      <td>2025-06-03</td>
      <td>16</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>jbrowse, iframe, create_or_update</td>
      <td>UseGalaxy.no</td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td>0</td>
      <td></td>
      <td>1454</td>
      <td>2213</td>
      <td>2.416667</td>
      <td>0</td>
    </tr>
  </tbody>
</table>

</div>