<script src="https://code.jquery.com/jquery-3.7.0.js"></script>
<script src="https://cdn.datatables.net/1.13.6/js/jquery.dataTables.min.js"></script>
<script src="https://cdn.datatables.net/searchbuilder/1.6.0/js/dataTables.searchBuilder.min.js"></script>
<script src="https://cdn.datatables.net/datetime/1.5.1/js/dataTables.dateTime.min.js"></script>

<script src="https://cdn.datatables.net/responsive/2.5.0/js/dataTables.responsive.min.js"></script>

<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/responsive/2.2.3/css/responsive.dataTables.min.css">

<link rel="stylesheet" href="https://cdn.datatables.net/1.13.6/css/jquery.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/searchbuilder/1.6.0/css/searchBuilder.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/datetime/1.5.1/css/dataTables.dateTime.min.css">

<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" />
<script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>

<style type="text/css">
  .tooltip {
    font-family: Georgia;
    font-size: 18px;
  }
  .tooltip .tooltip-inner {
    background-color: #00FFFF;
    color: #000000;
    min-width: 400px;
  }
  </style>

<script>
    $(document).ready(function() {
    $('#dataframe').DataTable( {
        "sScrollX": "100%",
        "dom": "Qlfrtip",

        responsive: {
            details: {
                type: 'column'
            }
        },
        "bAutoWidth": false,
        columnDefs: [ {
            className: 'control',
            orderable: false,
            targets:   [ 0 ],
            width: 100,
        },
        { type: 'natural', targets: 1 } // allow normal sorting
        ],

    });
});
</script>

<div class="display nowrap" style="width:80%" width="100%">

<table class="dataframe display nowrap" id="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th>Expand</th>
      <th>Name</th>
      <th>Source</th>
      <th>ID</th>
      <th>Link</th>
      <th>Creators</th>
      <th>Tags</th>
      <th>Creation time</th>
      <th>Update time</th>
      <th>Latest version</th>
      <th>Versions</th>
      <th>Number of steps</th>
      <th>Tools</th>
      <th>EDAM operations</th>
      <th>EDAM topics</th>
      <th>License</th>
      <th>DOI</th>
      <th>Projects</th>
      <th>To keep</th>
      <th>Deprecated</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td></td>
      <td>Genome assembly workflow for nanopore reads, for TSI</td>
      <td>WorkflowHub</td>
      <td>1114</td>
      <td>https://workflowhub.eu/workflows/1114?version=1</td>
      <td>Anna Syme</td>
      <td>tsi, genome_assembly, nanopore</td>
      <td>2024-09-03</td>
      <td>2024-09-03</td>
      <td>1</td>
      <td>1</td>
      <td>10.0</td>
      <td>split_file_to_collection, bandage_image, nanofilt, collapse_dataset, fasta-stats, bandage_info, porechop, fastqc, flye</td>
      <td>De-novo assembly, Sequence composition calculation, Sequencing quality control, Statistical calculation, Cross-assembly, Sequence assembly visualisation, Genome assembly, Mapping assembly</td>
      <td></td>
      <td>GPL-3.0-or-later</td>
      <td>10.48546/workflowhub.workflow.1114.1</td>
      <td>Australian BioCommons, Galaxy Australia</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>ERGA-BGE Genome Report ANNOT analyses</td>
      <td>WorkflowHub</td>
      <td>1096</td>
      <td>https://workflowhub.eu/workflows/1096?version=2</td>
      <td>Diego De Panis</td>
      <td>annotation, biodiversity, erga, genomics, qc</td>
      <td>2025-02-05</td>
      <td>2025-02-24</td>
      <td>2</td>
      <td>2</td>
      <td>13.0</td>
      <td>omark, collapse_dataset, lftp, tp_text_file_with_recurring_lines, pick_value, agat, busco</td>
      <td>Data handling, Differential protein expression profiling, Genome annotation, Sequence assembly validation</td>
      <td>Genomics</td>
      <td>MIT</td>
      <td></td>
      <td>ERGA Annotation</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>ERGA Profiling Long Reads v2601 (WF1)</td>
      <td>WorkflowHub</td>
      <td>603</td>
      <td>https://workflowhub.eu/workflows/603?version=2</td>
      <td>Diego De Panis</td>
      <td>erga, profiling</td>
      <td>2024-09-18</td>
      <td>2026-01-20</td>
      <td>2</td>
      <td>2</td>
      <td>17.0</td>
      <td>genomescope, tp_grep_tool, param_value_from_file, Convert characters1, tp_cut_tool, smudgeplot, Cut1, tp_find_and_replace, meryl, Add_a_column1</td>
      <td>Genotyping, k-mer counting, Sequence trimming</td>
      <td>Whole genome sequencing</td>
      <td>MIT</td>
      <td></td>
      <td>ERGA Assembly</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>ERGA DataQC HiFi v2601 (WF0)</td>
      <td>WorkflowHub</td>
      <td>602</td>
      <td>https://workflowhub.eu/workflows/602?version=2</td>
      <td>Diego De Panis</td>
      <td>dataqc, erga, hifi</td>
      <td>2024-09-18</td>
      <td>2026-01-20</td>
      <td>2</td>
      <td>2</td>
      <td>5.0</td>
      <td>collapse_dataset, seqkit_stats, cutadapt, fastqc, multiqc</td>
      <td>Sequence composition calculation, Sequence trimming, Sequencing quality control, Primer removal, Statistical calculation, Read pre-processing, Validation, Sequence conversion, DNA translation, DNA transcription</td>
      <td>Whole genome sequencing</td>
      <td>MIT</td>
      <td></td>
      <td>ERGA Assembly</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>ERGA DataQC Illumina v2601 (WF0)</td>
      <td>WorkflowHub</td>
      <td>601</td>
      <td>https://workflowhub.eu/workflows/601?version=2</td>
      <td>Diego De Panis</td>
      <td>dataqc, erga, illumina</td>
      <td>2024-09-18</td>
      <td>2026-01-20</td>
      <td>2</td>
      <td>2</td>
      <td>7.0</td>
      <td>__FLATTEN__, collapse_dataset, seqkit_stats, fastp, fastqc, multiqc</td>
      <td>Sequence composition calculation, Sequence trimming, Sequencing quality control, Statistical calculation, Sequence contamination filtering, Validation, Sequence conversion, DNA translation, DNA transcription</td>
      <td>Whole genome sequencing</td>
      <td>MIT</td>
      <td></td>
      <td>ERGA Assembly</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>ERGA DataQC ONT v2505 (WF0)</td>
      <td>WorkflowHub</td>
      <td>697</td>
      <td>https://workflowhub.eu/workflows/697?version=1</td>
      <td>Diego De Panis</td>
      <td>dataqc, erga, ont</td>
      <td>2024-01-08</td>
      <td>2025-06-01</td>
      <td>1</td>
      <td>1</td>
      <td>3.0</td>
      <td>collapse_dataset, nanoplot, seqkit_stats</td>
      <td>Sequence conversion, Box-Whisker plot plotting, DNA translation, Scatter plot plotting, Sequence trimming, DNA transcription</td>
      <td>Whole genome sequencing</td>
      <td>MIT</td>
      <td></td>
      <td>ERGA Assembly</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>ERGA Long Reads PriAlt Purge+QC v2505 (WF3)</td>
      <td>WorkflowHub</td>
      <td>1163</td>
      <td>https://workflowhub.eu/workflows/1163?version=2</td>
      <td>Diego De Panis</td>
      <td>assembly, erga, hifi, qc</td>
      <td>2025-06-01</td>
      <td>2025-06-01</td>
      <td>2</td>
      <td>2</td>
      <td>25.0</td>
      <td>param_value_from_file, purge_dups, cat1, merqury, gfa_to_fa, busco, gfastats, minimap2</td>
      <td>De-novo assembly, Pairwise sequence alignment, Read binning, Phasing, Genome assembly, Data handling, Sequence assembly validation, Scaffolding, k-mer counting</td>
      <td>Whole genome sequencing</td>
      <td>MIT</td>
      <td></td>
      <td>ERGA Assembly</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>ERGA HiC Pri Scaffolding+QC YaHS v2505 (WF4)</td>
      <td>WorkflowHub</td>
      <td>1164</td>
      <td>https://workflowhub.eu/workflows/1164?version=2</td>
      <td>Diego De Panis</td>
      <td>assembly, erga, hic, qc</td>
      <td>2025-02-03</td>
      <td>2025-06-01</td>
      <td>2</td>
      <td>2</td>
      <td>33.0</td>
      <td>pretext_map, pick_value, gfa_to_fa, pairtools_split, pairtools_parse, pairtools_sort, CONVERTER_fasta_to_fai, bwa_mem2, sambamba_flagstat, Cut1, merqury, pretext_snapshot, gfastats, pairtools_dedup, param_value_from_file, yahs, __EXTRACT_DATASET__, sambamba_merge, busco</td>
      <td>De-novo assembly, Data handling, Sequence alignment, Sequence assembly validation, Scaffolding, k-mer counting, Phasing, Genome assembly</td>
      <td>Whole genome sequencing</td>
      <td>MIT</td>
      <td></td>
      <td>ERGA Assembly</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>ERGA Long reads-only Assembly+QC Hifiasm v2505 (WF2)</td>
      <td>WorkflowHub</td>
      <td>1162</td>
      <td>https://workflowhub.eu/workflows/1162?version=2</td>
      <td>Diego De Panis</td>
      <td>assembly, erga, hifi, ont, qc</td>
      <td>2025-05-30</td>
      <td>2025-09-09</td>
      <td>2</td>
      <td>2</td>
      <td>22.0</td>
      <td>param_value_from_file, hifiasm, merqury, gfa_to_fa, busco, bandage_image, gfastats</td>
      <td>De-novo assembly, Data handling, Sequence assembly validation, Scaffolding, k-mer counting, Sequence assembly visualisation, Phasing, Genome assembly</td>
      <td>Whole genome sequencing</td>
      <td>MIT</td>
      <td></td>
      <td>ERGA Assembly</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>ERGA-BGE Genome Report ASM analyses (one-asm HiFi + HiC)</td>
      <td>WorkflowHub</td>
      <td>1104</td>
      <td>https://workflowhub.eu/workflows/1104?version=2</td>
      <td>Diego De Panis</td>
      <td>genome assembly, genomics, qc</td>
      <td>2024-11-04</td>
      <td>2024-12-05</td>
      <td>2</td>
      <td>2</td>
      <td>45.0</td>
      <td>genomescope, rseqc_bam_stat, pick_value, meryl, sam_merge2, cutadapt, fastp, pairtools_split, pairtools_parse, blobtoolkit, pairtools_sort, tp_text_file_with_recurring_lines, datasets_download_genome, CONVERTER_fasta_to_fai, fasterq_dump, hicexplorer_hicmergematrixbins, bwa_mem2, minimap2, bg_diamond, sambamba_flagstat, Cut1, merqury, cooler_cload_tabix, cooler_csort_tabix, gfastats, pairtools_dedup, __FLATTEN__, collapse_dataset, smudgeplot, lftp, __EXTRACT_DATASET__, hicexplorer_hicplotmatrix, sambamba_merge, busco, bedtools_makewindowsbed</td>
      <td>Pairwise sequence alignment, Sequence trimming, Mapping, Read pre-processing, Phasing, Data parsing, Genome assembly, Sequence alignment analysis, Sequence database search, Visualisation, Scaffolding, k-mer counting, De-novo assembly, Sequencing quality control, Primer removal, Genotyping, Sequence contamination filtering, Data handling, Data deposition, Sequence alignment, Sequence assembly validation, Data retrieval</td>
      <td>Genomics</td>
      <td>MIT</td>
      <td></td>
      <td>ERGA Assembly</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>ERGA HiC Collapsed Scaffolding+QC YaHS v2311 (WF4)</td>
      <td>WorkflowHub</td>
      <td>702</td>
      <td>https://workflowhub.eu/workflows/702?version=1</td>
      <td>Diego De Panis</td>
      <td>erga, hic, name:assembly+qc</td>
      <td>2024-01-09</td>
      <td>2024-01-09</td>
      <td>1</td>
      <td>1</td>
      <td>16.0</td>
      <td>param_value_from_file, pretext_map, yahs, bellerophon, merqury, gfa_to_fa, busco, bwa_mem2, gfastats, pretext_snapshot</td>
      <td>De-novo assembly, Data handling, Sequence alignment, Sequence assembly validation, Scaffolding, k-mer counting, Phasing, Genome assembly</td>
      <td>Whole genome sequencing</td>
      <td>MIT</td>
      <td></td>
      <td>ERGA Assembly</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>ERGA ONT+Illumina Collapsed Purge+QC v2311 (WF3)</td>
      <td>WorkflowHub</td>
      <td>701</td>
      <td>https://workflowhub.eu/workflows/701?version=1</td>
      <td>Diego De Panis</td>
      <td>erga, illumina, name:assembly+qc</td>
      <td>2024-01-09</td>
      <td>2024-01-09</td>
      <td>1</td>
      <td>1</td>
      <td>14.0</td>
      <td>param_value_from_file, purge_dups, merqury, gfa_to_fa, busco, gfastats, minimap2</td>
      <td>De-novo assembly, Pairwise sequence alignment, Read binning, Phasing, Genome assembly, Data handling, Sequence assembly validation, Scaffolding, k-mer counting</td>
      <td>Whole genome sequencing</td>
      <td>MIT</td>
      <td></td>
      <td>ERGA Assembly</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>ERGA HiC Hap1Hap2 Scaffolding+QC YaHS v2309 (WF4)</td>
      <td>WorkflowHub</td>
      <td>608</td>
      <td>https://workflowhub.eu/workflows/608?version=1</td>
      <td>Diego De Panis</td>
      <td>assembly+qc, erga, hi-c</td>
      <td>2023-10-11</td>
      <td>2024-03-13</td>
      <td>1</td>
      <td>1</td>
      <td>30.0</td>
      <td>param_value_from_file, pretext_map, yahs, bellerophon, merqury, gfa_to_fa, busco, bwa_mem2, gfastats, pretext_snapshot</td>
      <td>De-novo assembly, Data handling, Sequence alignment, Sequence assembly validation, Scaffolding, k-mer counting, Phasing, Genome assembly</td>
      <td>Whole genome sequencing</td>
      <td>MIT</td>
      <td>10.48546/workflowhub.workflow.608.1</td>
      <td>ERGA Assembly</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>ERGA HiFi+HiC Assembly+QC Hifiasm v2309 (WF2)</td>
      <td>WorkflowHub</td>
      <td>605</td>
      <td>https://workflowhub.eu/workflows/605?version=1</td>
      <td>Diego De Panis</td>
      <td>assembly+qc, erga, hi-c, hifi</td>
      <td>2023-10-09</td>
      <td>2024-03-13</td>
      <td>1</td>
      <td>1</td>
      <td>21.0</td>
      <td>param_value_from_file, hifiasm, merqury, gfa_to_fa, busco, bandage_image, gfastats</td>
      <td>De-novo assembly, Data handling, Sequence assembly validation, Scaffolding, k-mer counting, Sequence assembly visualisation, Phasing, Genome assembly</td>
      <td>Whole genome sequencing</td>
      <td>MIT</td>
      <td>10.48546/workflowhub.workflow.605.1</td>
      <td>ERGA Assembly</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Refining Genome Annotations with Apollo (prokaryotes)</td>
      <td>WorkflowHub</td>
      <td>749</td>
      <td>https://workflowhub.eu/workflows/749?version=1</td>
      <td>Anthony Bretaudeau</td>
      <td>genome-annotation</td>
      <td>2024-02-14</td>
      <td>2025-11-04</td>
      <td>1</td>
      <td>1</td>
      <td>5.0</td>
      <td>iframe, create_or_update, create_account, jbrowse, list_organism</td>
      <td>Genome visualisation</td>
      <td></td>
      <td>CC-BY-4.0</td>
      <td>10.48546/workflowhub.workflow.749.1</td>
      <td>EuroScienceGateway</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>GigaScience_Peptide_Annotation_demonstration_STS26T_neoantigen_candidates_workflow</td>
      <td>WorkflowHub</td>
      <td>1790</td>
      <td>https://workflowhub.eu/workflows/1790?version=1</td>
      <td></td>
      <td>gtn, galaxy, name:neoantigen</td>
      <td>2025-07-07</td>
      <td>2025-07-07</td>
      <td>1</td>
      <td>1</td>
      <td>11.0</td>
      <td>regexColumn1, converting pipes to columns\nConvert characters1, query_tabular, tab2fasta, pep_pointer, converting colons to columns\nConvert characters1</td>
      <td></td>
      <td></td>
      <td>GPL-3.0-or-later</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow 'Biodiversity data exploration tuto'</td>
      <td>WorkflowHub</td>
      <td>1693</td>
      <td>https://workflowhub.eu/workflows/1693?version=1</td>
      <td></td>
      <td>ecology, gtn, galaxy</td>
      <td>2025-06-02</td>
      <td>2025-06-02</td>
      <td>1</td>
      <td>1</td>
      <td>6.0</td>
      <td>ecology_beta_diversity, ecology_link_between_var, ecology_presence_abs_abund, tool_anonymization, ecology_stat_presence_abs, ecology_homogeneity_normality</td>
      <td></td>
      <td></td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Metagenomics assembly tutorial workflow</td>
      <td>WorkflowHub</td>
      <td>1634</td>
      <td>https://workflowhub.eu/workflows/1634?version=1</td>
      <td></td>
      <td>gtn, galaxy</td>
      <td>2025-06-02</td>
      <td>2025-06-02</td>
      <td>1</td>
      <td>1</td>
      <td>8.0</td>
      <td>megahit, bowtie2, megahit_contig2fastg, bandage_info, collection_column_join, quast, metaspades, bandage_image</td>
      <td>Sequence assembly validation, Read mapping, Visualisation, Sequence assembly visualisation, Genome assembly</td>
      <td></td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation with Prokka</td>
      <td>WorkflowHub</td>
      <td>1497</td>
      <td>https://workflowhub.eu/workflows/1497?version=1</td>
      <td></td>
      <td>gtn, galaxy, genome-annotation</td>
      <td>2025-06-02</td>
      <td>2025-06-02</td>
      <td>1</td>
      <td>1</td>
      <td>2.0</td>
      <td>prokka, jbrowse</td>
      <td>Coding region prediction, Genome visualisation, Gene prediction, Genome annotation</td>
      <td></td>
      <td>GPL-3.0-or-later</td>
      <td></td>
      <td>Galaxy Training Network</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>ONT -- Metagenomics-Kraken2-Krona</td>
      <td>WorkflowHub</td>
      <td>53</td>
      <td>https://workflowhub.eu/workflows/53?version=1</td>
      <td>Milad Miladi</td>
      <td>ont</td>
      <td>2020-08-05</td>
      <td>2023-02-13</td>
      <td>1</td>
      <td>1</td>
      <td>4.0</td>
      <td>tp_replace_in_line, datamash_reverse, taxonomy_krona_chart, tp_replace_in_line, kraken2</td>
      <td>Visualisation, Taxonomic classification</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>NanoGalaxy</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Biodiversity data exploration tutorial</td>
      <td>WorkflowHub</td>
      <td>656</td>
      <td>https://workflowhub.eu/workflows/656?version=1</td>
      <td>Yvan Le Bras, Coline Royaux, Marie Joss√©</td>
      <td>biodiversity, ecology</td>
      <td>2023-11-09</td>
      <td>2024-12-09</td>
      <td>1</td>
      <td>1</td>
      <td>6.0</td>
      <td>ecology_beta_diversity, ecology_link_between_var, ecology_presence_abs_abund, tool_anonymization, ecology_stat_presence_abs, ecology_homogeneity_normality</td>
      <td></td>
      <td>Biodiversity, Ecology</td>
      <td>CC-BY-4.0</td>
      <td>10.48546/workflowhub.workflow.656.1</td>
      <td>PNDB</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Population and community metrics calculation from Biodiversity data</td>
      <td>WorkflowHub</td>
      <td>49</td>
      <td>https://workflowhub.eu/workflows/49?version=2</td>
      <td>Yvan Le Bras, Coline Royaux</td>
      <td>biodiversity, community_metrics, ecology, glm, modeling, presence_absence, species abundance, statistics</td>
      <td>2020-07-24</td>
      <td>2024-12-09</td>
      <td>2</td>
      <td>1</td>
      <td>4.0</td>
      <td></td>
      <td></td>
      <td></td>
      <td>MIT</td>
      <td>10.48546/workflowhub.workflow.49.2</td>
      <td>PNDB</td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow CESAB Biodiversity data training 2025 - data access exercise bonus</td>
      <td>https://usegalaxy.eu</td>
      <td>39366728f462b4f1</td>
      <td>https://usegalaxy.eu/published/workflow?id=39366728f462b4f1</td>
      <td></td>
      <td>cesab, frb, gbif, pndb, biodiversity, data, occurences, map, gis, spatial, training</td>
      <td>2025-11-06</td>
      <td>2025-11-06</td>
      <td>12</td>
      <td>12</td>
      <td></td>
      <td>lftp, interactive_tool_jupyter_notebook, collapse_dataset, lftp, tp_text_file_with_recurring_lines, cat1, Remove beginning1</td>
      <td></td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow CESAB Biodiversity data training 2025 - data access exercise 1</td>
      <td>https://usegalaxy.eu</td>
      <td>a52ea1bae6683dce</td>
      <td>https://usegalaxy.eu/published/workflow?id=a52ea1bae6683dce</td>
      <td></td>
      <td>cesab, frb, gbif, pndb, biodiversity, data, download, training</td>
      <td>2025-11-05</td>
      <td>2025-11-06</td>
      <td>15</td>
      <td>15</td>
      <td></td>
      <td>collapse_dataset, lftp, interactive_tool_jupyter_notebook, tp_text_file_with_recurring_lines</td>
      <td></td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow CESAB Biodiversity data training 2025 - data access exercise 2</td>
      <td>https://usegalaxy.eu</td>
      <td>ca0775b90e12a784</td>
      <td>https://usegalaxy.eu/published/workflow?id=ca0775b90e12a784</td>
      <td></td>
      <td>cesab, gbif, pndb, biodiversity, data, occurences, training</td>
      <td>2025-11-05</td>
      <td>2025-11-06</td>
      <td>9</td>
      <td>9</td>
      <td></td>
      <td>collapse_dataset, lftp, interactive_tool_jupyter_notebook, tp_text_file_with_recurring_lines</td>
      <td></td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Obitools eDNA metabarcoding</td>
      <td>https://usegalaxy.eu</td>
      <td>afad9a17a80bb26d</td>
      <td>https://usegalaxy.eu/published/workflow?id=afad9a17a80bb26d</td>
      <td></td>
      <td>ecology, biodiversity, edna</td>
      <td>2023-11-09</td>
      <td>2025-09-12</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>unzip, fastq_groomer, obi_annotate, obi_tab, ncbi_blastn_wrapper, Filter1, seq_filter_by_id, obi_illumina_pairend, wc_gnu, Cut1, obi_ngsfilter, join1, obi_grep, obi_clean, obi_uniq, fastqc, obi_stat</td>
      <td>Sequence composition calculation, Sequence analysis, Sequencing quality control, Statistical calculation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Satellite data for biodiversity</td>
      <td>https://usegalaxy.eu</td>
      <td>1d2cd0af01a30a46</td>
      <td>https://usegalaxy.eu/published/workflow?id=1d2cd0af01a30a46</td>
      <td>Marie Josse</td>
      <td>ecology, earth-system, satellite</td>
      <td>2023-03-01</td>
      <td>2024-05-27</td>
      <td>5</td>
      <td>5</td>
      <td></td>
      <td>srs_diversity_maps, srs_preprocess_s2, srs_global_indices, srs_spectral_indices</td>
      <td></td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Obis biodiversity indicator on Asian pacific</td>
      <td>https://usegalaxy.eu</td>
      <td>9e05bd65faf58ab3</td>
      <td>https://usegalaxy.eu/published/workflow?id=9e05bd65faf58ab3</td>
      <td></td>
      <td></td>
      <td>2023-03-13</td>
      <td>2023-11-10</td>
      <td>2</td>
      <td>2</td>
      <td></td>
      <td>csv_to_tabular, Filter1, obisindicators, csv_to_tabular, tp_cut_tool</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Remote sensing Sentinel 2 data analysis to produce biodiversity metrics</td>
      <td>https://usegalaxy.eu</td>
      <td>da88448ab15d167d</td>
      <td>https://usegalaxy.eu/published/workflow?id=da88448ab15d167d</td>
      <td></td>
      <td></td>
      <td>2023-11-09</td>
      <td>2023-11-09</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>srs_diversity_maps, srs_preprocess_s2, srs_global_indices, srs_spectral_indices</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>Compute and analyze biodiversity metrics with PAMPA toolsuite</td>
      <td>https://usegalaxy.eu</td>
      <td>d33ab07e4cf44f8c</td>
      <td>https://usegalaxy.eu/published/workflow?id=d33ab07e4cf44f8c</td>
      <td></td>
      <td>ecology</td>
      <td>2021-02-24</td>
      <td>2021-02-24</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>tp_sort_header_tool, mergeCols1, tp_cat, datamash_transpose, Filter1, regexColumn1, tp_cut_tool, pampa_plotglm, pampa_communitymetrics, pampa_glmcomm, pampa_presabs, Count1, pampa_glmsp, regex1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>16S_biodiversity_BIOM</td>
      <td>https://usegalaxy.org.au</td>
      <td>ca6f29110df8dadc</td>
      <td>https://usegalaxy.org.au/published/workflow?id=ca6f29110df8dadc</td>
      <td></td>
      <td>metadegalaxy</td>
      <td>2019-03-21</td>
      <td>2021-04-01</td>
      <td>9</td>
      <td>9</td>
      <td></td>
      <td>biom_convert, phyloseq_taxonomy, symmetricPlot, phyloseq_net, phyloseq_DESeq2, phyloseq_abundance, phyloseq_richness, biom_add_metadata</td>
      <td>Formatting</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>16S_biodiversity_for_nonoverlap_paired_end</td>
      <td>https://usegalaxy.org.au</td>
      <td>869440d35a618836</td>
      <td>https://usegalaxy.org.au/published/workflow?id=869440d35a618836</td>
      <td></td>
      <td>metadegalaxy</td>
      <td>2019-03-21</td>
      <td>2021-04-01</td>
      <td>18</td>
      <td>18</td>
      <td></td>
      <td>biom_convert, phyloseq_taxonomy, cat_multi_datasets, addValue, symmetricPlot, phyloseq_net, phyloseq_abundance, fastqc, vsearch_clustering, cat1, bwa_mem, biom_add_metadata, samtools_fastx, trimmomatic, Cut1, uclust2otutable, picard_FilterSamReads, phyloseq_DESeq2, vsearch_chimera_detection, phyloseq_richness, vsearch_dereplication, vsearch_search</td>
      <td>Formatting, Sequence composition calculation, DNA mapping, Read mapping, Sequencing quality control, Statistical calculation, Generation, Chimera detection, Genome indexing, Sequence alignment</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
    <tr>
      <td></td>
      <td>16S_biodiversity_for_overlap_paired_end</td>
      <td>https://usegalaxy.org.au</td>
      <td>14e80b7f9f68ec2a</td>
      <td>https://usegalaxy.org.au/published/workflow?id=14e80b7f9f68ec2a</td>
      <td></td>
      <td>metadegalaxy</td>
      <td>2019-03-21</td>
      <td>2021-04-01</td>
      <td>16</td>
      <td>16</td>
      <td></td>
      <td>biom_convert, phyloseq_taxonomy, addValue, symmetricPlot, phyloseq_net, phyloseq_abundance, fastqc, vsearch_clustering, iuc_pear, cat1, picard_MergeSamFiles, bwa_mem, biom_add_metadata, samtools_fastx, trimmomatic, Cut1, uclust2otutable, picard_FilterSamReads, phyloseq_DESeq2, vsearch_chimera_detection, phyloseq_richness, vsearch_dereplication, vsearch_search</td>
      <td>Formatting, Sequence composition calculation, DNA mapping, Read mapping, Sequencing quality control, Statistical calculation, Generation, Chimera detection, Genome indexing, Sequence alignment, Sequence merging</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>True</td>
      <td>False</td>
    </tr>
  </tbody>
</table>

</div>